<!DOCTYPE html>
<html dir="ltr" xmlns="http://www.w3.org/1999/xhtml" xmlns:mml="http://www.w3.org/1998/Math/MathML" class="modernizr-svg modernizr-mutationobserver modernizr-localstorage modernizr-inlinesvg modernizr-svgclippaths modernizr-matchmedia modernizr-flexbox js" style="--grove-client-width:500px;" lang="en"><head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#">
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<link rel="dns-prefetch" href="https://fonts.gstatic.com/">
<link rel="dns-prefetch" href="https://fonts.googleapis.com/">
<link rel="dns-prefetch" href="https://www.googletagservices.com/">
<link rel="dns-prefetch" href="https://stats.g.doubleclick.net/">
<link rel="dns-prefetch" href="https://www.google-analytics.com/">
<link rel="dns-prefetch" href="https://scholar.google.com/">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes">
<link rel="shortcut icon" href="https://www.genetics.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon">
<link rel="canonical" href="https://www.genetics.org/content/177/2/861">
<meta name="Generator" content="Drupal 7 (http://drupal.org)">
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="https://www.genetics.org/content/177/2/861.full.pdf">
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="https://www.genetics.org/content/177/2/861.full.txt">
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="https://www.genetics.org/content/177/2/861.ppt">
<meta name="issue_cover_image" content="https://www.genetics.org/sites/default/files/highwire/genetics/177/2.cover.gif">
<meta name="article_thumbnail" content="https://www.genetics.org/content/genetics/177/2/861/embed/tex-math-1.tex">
<meta name="type" content="article">
<meta name="category" content="research-article">
<meta name="HW.identifier" content="/genetics/177/2/861.atom">
<meta name="HW.pisa" content="genetics;177/2/861">
<meta name="DC.Format" content="text/html">
<meta name="DC.Language" content="en">
<meta name="DC.Title" content="Empirical Bayes Inference of Pairwise FST and Its Distribution in the Genome">
<meta name="DC.Identifier" content="10.1534/genetics.107.077263">
<meta name="DC.Date" content="2007-10-01">
<meta name="DC.Publisher" content="Genetics">
<meta name="DC.Rights" content="Copyright Â© 2007 by the Genetics Society of America">
<meta name="DC.AccessRights" content="restricted">
<meta name="DC.Description" content="Populations often have very complex hierarchical structure. Therefore, it is crucial in genetic monitoring and conservation biology to have a reliable estimate of the pattern of population subdivision. F ST's for pairs of sampled localities or subpopulations are crucial statistics for the exploratory analysis of population structures, such as cluster analysis and multidimensional scaling. However, the estimation of F ST is not precise enough to reliably estimate the population structure and the extent of heterogeneity. This article proposes an empirical Bayes procedure to estimate locus-specific pairwise F ST's. The posterior mean of the pairwise F ST can be interpreted as a shrinkage estimator, which reduces the variance of conventional estimators largely at the expense of a small bias. The global F ST of a population generally varies among loci in the genome. Our maximum-likelihood estimates of global F ST's can be used as sufficient statistics to estimate the distribution of F ST in the genome. We demonstrate the efficacy and robustness of our model by simulation and by an analysis of the microsatellite allele frequencies of the Pacific herring. The heterogeneity of the global F ST in the genome is discussed on the basis of the estimated distribution of the global F ST for the herring and examples of human single nucleotide polymorphisms (SNPs).">
<meta name="DC.Contributor" content="Shuichi Kitada">
<meta name="DC.Contributor" content="Toshihide Kitakado">
<meta name="DC.Contributor" content="Hirohisa Kishino">
<meta name="article:published_time" content="2007-10-01">
<meta name="article:section" content="Investigations">
<meta name="citation_title" content="Empirical Bayes Inference of Pairwise FST and Its Distribution in the Genome">
<meta name="citation_abstract" content="&lt;p&gt;Populations often have very complex hierarchical structure. Therefore, it is crucial in genetic monitoring and conservation biology to have a reliable estimate of the pattern of population subdivision. &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt;9s for pairs of sampled localities or subpopulations are crucial statistics for the exploratory analysis of population structures, such as cluster analysis and multidimensional scaling. However, the estimation of &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt; is not precise enough to reliably estimate the population structure and the extent of heterogeneity. This article proposes an empirical Bayes procedure to estimate locus-specific pairwise &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt;9s. The posterior mean of the pairwise &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt; can be interpreted as a shrinkage estimator, which reduces the variance of conventional estimators largely at the expense of a small bias. The global &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt; of a population generally varies among loci in the genome. Our maximum-likelihood estimates of global &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt;9s can be used as sufficient statistics to estimate the distribution of &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt; in the genome. We demonstrate the efficacy and robustness of our model by simulation and by an analysis of the microsatellite allele frequencies of the Pacific herring. The heterogeneity of the global &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt; in the genome is discussed on the basis of the estimated distribution of the global &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt; for the herring and examples of human single nucleotide polymorphisms (SNPs).&lt;/p&gt;" lang="en">
<meta name="citation_journal_title" content="Genetics">
<meta name="citation_publisher" content="Genetics">
<meta name="citation_publication_date" content="2007/10/01">
<meta name="citation_mjid" content="genetics;177/2/861">
<meta name="citation_id" content="177/2/861">
<meta name="citation_public_url" content="https://www.genetics.org/content/177/2/861">
<meta name="citation_abstract_html_url" content="https://www.genetics.org/content/177/2/861.abstract">
<meta name="citation_full_html_url" content="https://www.genetics.org/content/177/2/861.full">
<meta name="citation_pdf_url" content="https://www.genetics.org/content/genetics/177/2/861.full.pdf">
<meta name="citation_issn" content="0016-6731">
<meta name="citation_issn" content="1943-2631">
<meta name="citation_doi" content="10.1534/genetics.107.077263">
<meta name="citation_pmid" content="17660541">
<meta name="citation_volume" content="177">
<meta name="citation_issue" content="2">
<meta name="citation_article_type" content="Research Article">
<meta name="citation_section" content="Investigations">
<meta name="citation_firstpage" content="861">
<meta name="citation_lastpage" content="873">
<meta name="citation_author" content="Shuichi Kitada">
<meta name="citation_author" content="Toshihide Kitakado">
<meta name="citation_author" content="Hirohisa Kishino">
<meta name="citation_reference" content="citation_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_title=Likelihood-based inference for genetic correlation coefficients.;citation_pages=221-230;citation_volume=63;citation_year=2003;citation_issue=3;citation_pmid=12689793;citation_doi=10.1016/S0040-5809(03)00007-8">
<meta name="citation_reference" content="citation_title=Heredity;citation_pages=583;citation_volume=78;citation_year=1997;citation_pmid=9203353;citation_doi=10.1038/sj.hdy.6881750">
<meta name="citation_reference" content="citation_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_title=The estimation of population differentiation with microsatellite markers.;citation_pages=155-165;citation_volume=11;citation_year=2002;citation_issue=2;citation_pmid=11856418;citation_doi=10.1046/j.0962-1083.2001.01436.x">
<meta name="citation_reference" content="citation_title=Evolution;citation_pages=72;citation_volume=23;citation_year=1969;citation_doi=10.2307/2406485">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=ANALYSES OF GENE FREQUENCIES;citation_pages=679-700;citation_volume=74;citation_year=1973;citation_issue=4;citation_issn=0016-6731;citation_pmid=17248636">
<meta name="citation_reference" content="citation_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_title=Estimation of inbreeding parameters in stratified populations.;citation_pages=271-281;citation_volume=50;citation_year=1986;citation_issue=Pt 3;citation_pmid=3446014">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=Bayesian Analysis of Genetic Differentiation Between Populations;citation_pages=367-374;citation_volume=163;citation_year=2003;citation_issue=1;citation_issn=0016-6731;citation_pmid=12586722">
<meta name="citation_reference" content="citation_year=1995">
<meta name="citation_reference" content="citation_year=2004">
<meta name="citation_reference" content="citation_year=2003">
<meta name="citation_reference" content="citation_title=Evol. Bioinform. Online;citation_pages=47;citation_volume=1;citation_year=2005;citation_pmid=19325852">
<meta name="citation_reference" content="citation_title=American journal of human biology :  the official journal of the Human Biology Council;citation_journal_abbrev=Am J Hum Biol;citation_title=Locus-specific genetic diversity between human populations: an analysis of the literature.;citation_pages=814-823;citation_volume=15;citation_year=2003;citation_issue=6;citation_pmid=14595873;citation_doi=10.1002/ajhb.10215">
<meta name="citation_reference" content="citation_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_title=Islands of linkage disequilibrium.;citation_pages=109-111;citation_volume=29;citation_year=2001;citation_issue=2;citation_pmid=11586289;citation_doi=10.1038/ng1001-109">
<meta name="citation_reference" content="citation_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_title=FSTAT (Version 1.2): A Computer Program to Calculate F-Statistics;citation_pages=485-486;citation_volume=86;citation_year=1995;citation_issue=6;citation_issn=0022-1503">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=Testing Differentiation in Diploid Populations;citation_pages=1933-1940;citation_volume=144;citation_year=1996;citation_issue=4;citation_issn=0016-6731;citation_pmid=8978076">
<meta name="citation_reference" content="citation_year=2004">
<meta name="citation_reference" content="citation_title=Science;citation_journal_abbrev=Science;citation_title=Demographic Histories and Patterns of Linkage Disequilibrium in Chinese and Indian Rhesus Macaques;citation_pages=240-243;citation_volume=316;citation_year=2007;citation_issue=5822;citation_issn=0036-8075;citation_pmid=17431170;citation_doi=10.1126/science.1140462">
<meta name="citation_reference" content="citation_title=Science;citation_journal_abbrev=Science;citation_title=Whole-Genome Patterns of Common DNA Variation in Three Human Populations;citation_pages=1072-1079;citation_volume=307;citation_year=2005;citation_issue=5712;citation_issn=0036-8075;citation_pmid=15718463;citation_doi=10.1126/science.1105436">
<meta name="citation_reference" content="citation_title=Hereditas;citation_pages=245;citation_volume=130;citation_year=1999;citation_doi=10.1111/j.1601-5223.1999.00245.x">
<meta name="citation_reference" content="citation_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_title=A Bayesian approach to inferring population structure from dominant markers.;citation_pages=1157-1164;citation_volume=11;citation_year=2002;citation_issue=7;citation_pmid=12074723;citation_doi=10.1046/j.1365-294X.2002.01512.x">
<meta name="citation_reference" content="citation_title=Genetics;citation_volume=175;citation_year=2007;citation_pmid=17237522;citation_doi=10.1534/genetics.106.061317">
<meta name="citation_reference" content="citation_year=1965">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=Simultaneous Detection of Linkage Disequilibrium and Genetic Differentiation of Subdivided Populations;citation_pages=2003-2013;citation_volume=167;citation_year=2004;citation_issue=4;citation_issn=0016-6731;citation_pmid=15342536;citation_doi=10.1534/genetics.103.023044">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=Empirical Bayes Procedure for Estimating Genetic Distance Between Populations and Effective Population Size;citation_pages=2063-2079;citation_volume=156;citation_year=2000;citation_issue=4;citation_issn=0016-6731;citation_pmid=11102396">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=An integrated-likelihood method for estimating genetic differentiation between populations.;citation_pages=2073-2082;citation_volume=173;citation_year=2006;citation_issue=4;citation_issn=0016-6731;citation_pmid=16751669;citation_doi=10.1534/genetics.106.055350">
<meta name="citation_reference" content="citation_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_title=Prospects for whole-genome linkage disequilibrium mapping of common disease genes.;citation_pages=139-144;citation_volume=22;citation_year=1999;citation_issue=2;citation_pmid=10369254;citation_doi=10.1038/9642">
<meta name="citation_reference" content="citation_title=Genetica;citation_journal_abbrev=Genetica;citation_title=Applications of the Dirichlet distribution to forensic match probabilities.;citation_pages=107-117;citation_volume=96;citation_year=1995;citation_issue=1-2;citation_pmid=7607447;citation_doi=10.1007/BF01441156">
<meta name="citation_reference" content="citation_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_title=A Bayesian hierarchical model for allele frequencies.;citation_pages=17-33;citation_volume=20;citation_year=2001;citation_issue=1;citation_pmid=11119294;citation_doi=10.1002/1098-2272(200101)20:1&lt;17::AID-GEPI3&gt;3.0.CO;2-Q">
<meta name="citation_reference" content="citation_year=1989">
<meta name="citation_reference" content="citation_title=Science;citation_journal_abbrev=Science;citation_title=The Fine-Scale Structure of Recombination Rate Variation in the Human Genome;citation_pages=581-584;citation_volume=304;citation_year=2004;citation_issue=5670;citation_issn=0036-8075;citation_pmid=15105499;citation_doi=10.1126/science.1092500">
<meta name="citation_reference" content="citation_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_title=Analysis of Gene Diversity in Subdivided Populations;citation_pages=3321-3323;citation_volume=70;citation_year=1973;citation_issue=12;citation_issn=0027-8424;citation_pmid=4519626;citation_doi=10.1073/pnas.70.12.3321">
<meta name="citation_reference" content="citation_title=Annals of human genetics;citation_journal_abbrev=Ann Hum Genet;citation_title=Estimation of fixation indices and gene diversities.;citation_pages=253-259;citation_volume=47;citation_year=1983;citation_issue=Pt 3;citation_pmid=6614868;citation_doi=10.1111/j.1469-1809.1983.tb00993.x">
<meta name="citation_reference" content="citation_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_title=Microsatellite analysis of population structure in Canadian polar bears.;citation_pages=347-354;citation_volume=4;citation_year=1995;citation_issue=3;citation_pmid=7663752;citation_doi=10.1111/j.1365-294X.1995.tb00227.x">
<meta name="citation_reference" content="citation_title=Trends in Ecology &amp; Evolution;citation_journal_abbrev=Trends in Ecology &amp; Evolution;citation_title=Identification of management units using population genetic data.;citation_pages=11-16;citation_volume=22;citation_year=2007;citation_issue=1;citation_pmid=16982114;citation_doi=10.1016/j.tree.2006.09.003">
<meta name="citation_reference" content="citation_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_title=Effects of metapopulation processes on measures of genetic diversity;citation_pages=1851-1864;citation_volume=355;citation_year=2000;citation_issue=1404;citation_issn=0080-4622;citation_pmid=11205346;citation_doi=10.1098/rstb.2000.0740">
<meta name="citation_reference" content="citation_title=Genetics;citation_pages=945;citation_volume=156;citation_year=2000;citation_issn=0016-6731">
<meta name="citation_reference" content="citation_title=Genetical research;citation_journal_abbrev=Genet Res;citation_title=Estimating gene flow in island populations.;citation_pages=147-158;citation_volume=67;citation_year=1996;citation_issue=2;citation_pmid=8801187">
<meta name="citation_reference" content="citation_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_title=Properties of bias and variance of two multiallelic estimators of F(ST).;citation_pages=285-296;citation_volume=57;citation_year=2000;citation_issue=3;citation_pmid=10828220;citation_doi=10.1006/tpbi.2000.1457">
<meta name="citation_reference" content="citation_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_title=GENEPOP (Version 1.2): Population Genetics Software for Exact Tests and Ecumenicism;citation_pages=248-249;citation_volume=86;citation_year=1995;citation_issue=3;citation_issn=0022-1503">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=ESTIMATION OF THE COANCESTRY COEFFICIENT: BASIS FOR A SHORT-TERM GENETIC DISTANCE;citation_pages=767-779;citation_volume=105;citation_year=1983;citation_issue=3;citation_issn=0016-6731;citation_pmid=17246175">
<meta name="citation_reference" content="citation_title=Genetics;citation_journal_abbrev=Genetics;citation_title=DEVIATIONS FROM HARDY-WEINBERG PROPORTIONS: SAMPLING VARIANCES AND USE IN ESTIMATION OF INBREEDING COEFFICIENTS;citation_pages=703-718;citation_volume=107;citation_year=1984;citation_issue=4;citation_issn=0016-6731;citation_pmid=6745643">
<meta name="citation_reference" content="citation_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_title=Informativeness of genetic markers for inference of ancestry.;citation_pages=1402-1422;citation_volume=73;citation_year=2003;citation_issue=6;citation_pmid=14631557;citation_doi=10.1086/380416">
<meta name="citation_reference" content="citation_year=2003">
<meta name="citation_reference" content="citation_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_title=Statistical power when testing for genetic differentiation.;citation_pages=2361-2373;citation_volume=10;citation_year=2001;citation_issue=10;citation_pmid=11703649;citation_doi=10.1046/j.0962-1083.2001.01345.x">
<meta name="citation_reference" content="citation_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_title=Power for detecting genetic divergence: differences between statistical methods and marker loci.;citation_pages=2031-2045;citation_volume=15;citation_year=2006;citation_issue=8;citation_pmid=16780422;citation_doi=10.1111/j.1365-294X.2006.02839.x">
<meta name="citation_reference" content="citation_year=1982">
<meta name="citation_reference" content="citation_year=1956">
<meta name="citation_reference" content="citation_title=Psychometrika;citation_pages=401;citation_volume=17;citation_year=1952;citation_doi=10.1007/BF02288916">
<meta name="citation_reference" content="citation_title=Human genetics;citation_journal_abbrev=Hum Genet;citation_title=Searching for signals of evolutionary selection in 168 genes related to immune function.;citation_pages=92-102;citation_volume=119;citation_year=2006;citation_issue=1-2;citation_pmid=16362345;citation_doi=10.1007/s00439-005-0090-0">
<meta name="citation_reference" content="citation_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_title=What is a population? An empirical evaluation of some genetic methods for identifying the number of gene pools and their degree of connectivity.;citation_pages=1419-1439;citation_volume=15;citation_year=2006;citation_issue=6;citation_pmid=16629801;citation_doi=10.1111/j.1365-294X.2006.02890.x">
<meta name="citation_reference" content="citation_title=Evolution;citation_journal_abbrev=Evolution;citation_title=Estimating the unbiased estimator theta for population genetic survey data.;citation_pages=2601-2605;citation_volume=55;citation_year=2001;citation_issue=12;citation_pmid=11831672">
<meta name="citation_reference" content="citation_year=1996">
<meta name="citation_reference" content="citation_title=Evolution;citation_pages=1358;citation_volume=38;citation_year=1984;citation_doi=10.2307/2408641">
<meta name="citation_reference" content="citation_title=Annu. Rev. Genet.;citation_volume=36;citation_year=2002;citation_pmid=12359738;citation_doi=10.1146/annurev.genet.36.050802.093940">
<meta name="citation_reference" content="citation_title=Genome Research;citation_journal_abbrev=Genome Res;citation_title=Measures of human population structure show heterogeneity among genomic regions;citation_pages=1468-1476;citation_volume=15;citation_year=2005;citation_issue=11;citation_issn=1088-9051;citation_pmid=16251456;citation_doi=10.1101/gr.4398405">
<meta name="citation_reference" content="citation_title=Ann. Eugen.;citation_pages=323;citation_volume=15;citation_year=1951;citation_pmid=24540312">
<meta name="citation_reference" content="citation_year=1969">
<meta name="citation_reference" content="citation_year=1987">
<meta name="twitter:title" content="Empirical Bayes Inference of Pairwise FST and Its Distribution in the Genome">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://www.genetics.org/sites/default/files/highwire/genetics/177/2.cover.gif">
<meta name="twitter:description" content="Populations often have very complex hierarchical structure. Therefore, it is crucial in genetic monitoring and conservation biology to have a reliable estimate of the pattern of population subdivision. F ST's for pairs of sampled localities or subpopulations are crucial statistics for the exploratory analysis of population structures, such as cluster analysis and multidimensional scaling. However, the estimation of F ST is not precise enough to reliably estimate the population structure and the extent of heterogeneity. This article proposes an empirical Bayes procedure to estimate locus-specific pairwise F ST's. The posterior mean of the pairwise F ST can be interpreted as a shrinkage estimator, which reduces the variance of conventional estimators largely at the expense of a small bias. The global F ST of a population generally varies among loci in the genome. Our maximum-likelihood estimates of global F ST's can be used as sufficient statistics to estimate the distribution of F ST in the genome. We demonstrate the efficacy and robustness of our model by simulation and by an analysis of the microsatellite allele frequencies of the Pacific herring. The heterogeneity of the global F ST in the genome is discussed on the basis of the estimated distribution of the global F ST for the herring and examples of human single nucleotide polymorphisms (SNPs).">
<meta name="og-title" property="og:title" content="Empirical Bayes Inference of Pairwise FST and Its Distribution in the Genome">
<meta name="og-url" property="og:url" content="https://www.genetics.org/content/177/2/861">
<meta name="og-site-name" property="og:site_name" content="Genetics">
<meta name="og-description" property="og:description" content="Populations often have very complex hierarchical structure. Therefore, it is crucial in genetic monitoring and conservation biology to have a reliable estimate of the pattern of population subdivision. F ST's for pairs of sampled localities or subpopulations are crucial statistics for the exploratory analysis of population structures, such as cluster analysis and multidimensional scaling. However, the estimation of F ST is not precise enough to reliably estimate the population structure and the extent of heterogeneity. This article proposes an empirical Bayes procedure to estimate locus-specific pairwise F ST's. The posterior mean of the pairwise F ST can be interpreted as a shrinkage estimator, which reduces the variance of conventional estimators largely at the expense of a small bias. The global F ST of a population generally varies among loci in the genome. Our maximum-likelihood estimates of global F ST's can be used as sufficient statistics to estimate the distribution of F ST in the genome. We demonstrate the efficacy and robustness of our model by simulation and by an analysis of the microsatellite allele frequencies of the Pacific herring. The heterogeneity of the global F ST in the genome is discussed on the basis of the estimated distribution of the global F ST for the herring and examples of human single nucleotide polymorphisms (SNPs).">
<meta name="og-type" property="og:type" content="article">
<meta name="og-image" property="og:image" content="https://www.genetics.org/sites/default/files/highwire/genetics/177/2.cover.gif"><script src="groveMain" type="text/javascript" async="" data-api-base="https://remarqable.com/api"></script><script src="config.html" type="text/javascript" async=""></script><script type="application/json" id="user">[]</script>
    <title>Empirical Bayes Inference of Pairwise FST and Its Distribution in the Genome | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6.css" media="all">
<link type="text/css" rel="stylesheet" href="css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe.css" media="screen">
<link type="text/css" rel="stylesheet" href="css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmd.css" media="all">
<link type="text/css" rel="stylesheet" href="highwire.css" media="all">
<link type="text/css" rel="stylesheet" href="css__KPgiwEdjo5O9HdRryHqQs1TpsV-PeBb9B7sGBvwM9FQ__opzZWgPNgQ.css" media="all">
<link type="text/css" rel="stylesheet" href="jquery.css" media="all">
<link type="text/css" rel="stylesheet" href="css__mIkAuGAo6TFLpoaRWYEuzNQcz6D1DbG5Z-G8snSEY5s__41DdnEfT7Y.css" media="all">

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.genetics.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__GtdprKW0c9maP8pNmDN8lnTjFlFWbDQP16bdrcOxYA4.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpq.css" media="all">
<link type="text/css" rel="stylesheet" href="css__TBRBnZ_CSnwH5nFCOjVMUdHFY2J9a3XLrQhc3QHqOYQ___Y5TkohVwd.css" media="all">

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.genetics.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__GtdprKW0c9maP8pNmDN8lnTjFlFWbDQP16bdrcOxYA4.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMp.css" media="all">
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="css__gkuPHl1ejB3uV_AecA9P1bZo2cGViQ9XAq80DTA1Mw4__8vVMy5xbjw.css" media="all">
<link type="text/css" rel="stylesheet" href="css__5D9zCmvXApgXpLXGr0ULqiPtjjNtE_xoKXnf5a3vlzQ__e1EjSALHc_.css" media="all">
    <script src="1390.js" async="" type="text/javascript"></script><script src="gtm.js" type="text/javascript" async=""></script><script async="" src="analytics.js"></script><script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="modernizr.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0Q.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="gpt.js"></script><script src="pubads_impl_2019121002.js" id="gpt-impl-0.9426751804833358"></script><link rel="preload" href="integrator.js" as="script"><script src="integrator.js"></script><script>processGoogleTokenSync({"newToken":"FBS"},5);</script>
<script type="text/javascript" src="js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBs.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7B.js"></script>
<script type="text/javascript" async="async" src="casa.js"></script>
<script type="text/javascript" src="js__3uCzzYE5jr6ly9nvsM8PRBRkhh4p0NWCMDja4EKBy4M__dmGGVYcB8Lhn.js"></script>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="js__0zdUME0GLauYYyEch2tGPydIw-IG7Uizip0Qb4wUsig__3RgszrCgk4bO.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="js__fhq_kmKS3zjIxlsAa3GE1ubLajyXUgXXTXULzAtiqfg__Lm1_Ls-bdpbW.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35992623-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYv.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DuQEwzHzJ_nfc1nBz6XlpNaJfPyYzkLJrAijEgMlVYM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_nps_survey\/js\/nps_survey.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"334013","apath":"\/genetics\/177\/2\/861.atom","pisa":"genetics;177\/2\/861","ac":{"\/genetics\/177\/2\/861.atom":{"access":{"full":true},"pisa_id":"","apath":"\/genetics\/177\/2\/861.atom","jcode":"genetics"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"genetics;177\/2\/861"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"genetics;177\/2\/861"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/177\/2\/861":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.genetics.org\/sites\/default\/files\/GENturq_0.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v8YJBBvDzmvLsetdvnK0plB7ggWlI\\\\\\\/R9YwA\\\\\\\/ITcMbC\\\\\\\/Px181p7H+EPdb53E+0QS8XqATvP0TXIyPdXniKUBgZVC8oEVRJJ4qdiuteF0J+y9enHPEkWjvG6NE+prXm+UVWAUpo+4nlfDTBIDjQPm\\\\\\\/KVJujE5idu6qhcmKHaz2EwwvQnHnQlwdhtQTQONL7XNlTUznDEiUqMziHYdBrZe5391kvZLlYOq3WdotZF8a4eWArJ4yR5rrGla\\\\\\\/YSvEUB46rtojyIidZ0WLEW3pPGLVNjFyFJBrq6l9QV57ysE5ZNo3cxOPFiiYyB3v5YFfzZLxRF\\\\\\\/drea2QGxNjXCteftn+ltlUqSUBfA\\\\\\\/QixVfrF+KBXjhHB25ILtl2MxW4F3rb6efWL9KdwUGlGze8Xxtm55diFVJEHOs43f22n7IRoLhfBQtu7FOp0Og40UW+q4PVhj4Tpi01x3KVYSL\\\\\\\/yB5z8D1C5JSvtKBGCKWR+uvbxNnM2WmEk4hDueeEqLZSXElpMyx5D\\\\\\\/9kKAdeB\\\\\\\/klUcv49NakxbhFYVERXZ4Z9MxGihrBtR6o4n2Q+q\\\\\\\/tHVOOeJmOnD8ZvKFqLWr4i8eBeIX9KDZ6GxQAyBXvuhTI5\\\\\\\/cti5czCZ4nANYUZOpQSaJS+mgWXVRgPeJYcKQXKhiZSD\\\\\\\/L1YtS8N3zfLjyUkqo49g3czS7eY7UHm82bMJ9EixXaGrK63TP+l8Q2jg0f+Wtxr34yiS1MiynpX0aMpmkZmvOhpNrYOVUbFIk7CnJUFJSn9z1Z1MQS8EusFbN6ggDuSZU1bsS2m\\\\\\\/m3TmFL4HjGPkcJyvEquLN8loPT7NMThuvmRPV2IjyOPvn3KoqEqEXHwOCt8WBJRGAeuR5dkb4mYGkPj6RJtzV1FdusS1YwERT5q+m7Xjad81up\\\\\\\/Ob3kx8kkpmnpMYAZ53ruawnO9lUWEue1nBbKVgklasSjqhqsGS66lkvkQDH\\\\\\\/WvK2t680lCOdC2f1VLFr5Ts348ZbUl+hw\\\\\\\/LcMxu9BRIZcf\\\\\\\/aDd\\\\\\\/ugPMPlGH2H3jWXqnlkbhx1mkmKAoFBWl1BQdtZq\\\\\\\/sMK86AZuvR6pBjyvFX7R3hysInquHLeOmlxuDtOL2F3ulq5T71uqKpDgDMcRBglEDyzzEa4uBnodilwx0je3l96ptGQMRIo3QYMk0Ggekc2rQWWqczky0xeoT+5xR7kbK\\\\\\\/qrAwxqC7N1ANjN7R39wV7Oz9J6zQg6\\\\\\\/z7PfO1izqSsqixDjxepSAvCJml+i4QCnMUvEonPpGIMYSySnN0GPXxp4XU0pNZ\\\\\\\/z\\\u0022,\\\u0022iv\\\u0022:\\\u00226QNY\\\\\\\/CLgcdYfokaqusXeRw==\\\u0022,\\\u0022salt\\\u0022:\\\u00222be6b58ce6e14e3c3058432947d8ecf7\\\u0022}\u0022,\u0022hmac\u0022:\u00229c3f1e8107797efa97e5549e3b8fe65cb63f89fb7cc1d817079d81c2a16105c1\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YGE84JZtCId1mGnD2ZeN9go+6JxQsWCV0i\\\\\\\/Ub2TR4xZne6NALQZIlYsi95lJlZljeMjyw4XBKgEWxYwPMKyBcLd6KuxaEv2\\\\\\\/O8VvdOu7aJkdlO60LPOiV7czwijFmozcrmBzHjyR5DC8crVOurB6MMI5APFJqnVhYGLHbxxYUEhp5vgprf+lHmHi54hZ8RzhBsU47A+RdmPYw3OaTFopZwROKDSxJ5kNx52izfbx8l8WkT8GpdTxEJ9XNO4O1ei\\\\\\\/yY4p1Zh0BEiVTCcKQ\\\\\\\/LmjzF0OkHbDCCkO4FUjZ1zESxxaL+TSYvvrLgbylN+2yoiR7yUN8VP9zhBin8GVrg8JHBLPrchevtLTS4ifCqXHzkGAiw4SyjQlN5lg1Lff+tyznldSzRixjxynQCuc6XuodVrl55S0ytNFR4chTdu8y3gsw6NbWX5j0w\\\\\\\/UB\\\\\\\/cKq\\\\\\\/oYyAE\\\\\\\/2hN5VTrWCNEIhNVf4ugjXWzUpH2g3hGHanpWX8LRTwvSmdUrBj6+QpydN8rFyp01cJa45G7FswO7lhwZhV2EoSLOmNDg6oma8dM40XFQIVxTczdlqM+FC7cisg3e\\\\\\\/Ugo+bNKI9PRCSunQdNLPBQRwah\\\\\\\/WJAiwrkILctkb1m46lx00BOVB6E63H0sYjQsmjka5nH8tH3tPtQ9Pf9uEx4Zi36DJN3ChSAQUIdNg0SME0P9uh11GidcVGlrTbeR76Tqe7Gc5Cub++m9NwYfKIPUkNiz1aJfzhMkJ5OZgguQAtBwLAJeSs83IEtg1UVK5UQDIM\\\\\\\/FyiXsoSs4vUE7HAXl92EWuNourovUU52f+lp857KdYh8v9GDmLIkSOAAXM0UhKwfBv1YWsUyMSZ2GR60AyWwaHmo8hdmK0PAm4qICcuW96hcjj5BcgG+233ywnQaFEIx9Hh3+EsVfYFWx\\\\\\\/7vSFWdyAXqHA8Xm4KVUPtd1KrosJG0OmLLmpXnG2oDEK+HPGyYZe3e1ypx\\\\\\\/Qfb+bHNPEqL1RhUBUjrriD3fy5hKm26\\\\\\\/9OXRQRLyI4X\\\\\\\/qtUbyTmvl7+aawPJhRIAr+GmCBAL3KztBUzgSA\\\\\\\/EQGGkg5WTLpQxqfPtkIDl37\\\\\\\/C2JAGDy\\\\\\\/PmbnTs47dEY7Bh5+jeIRrlw1QF8d35fmBaB4LV6ZVhad7d0xxg3nhdgvcRWC98CbV+SCO66OCbR6QW+\\\\\\\/tarUz7bECKwk3HNP8B0A\\\\\\\/TgfNhI4UsJPtThdtx8ZyVGHwQQ+3RMvf1A9azr8vtPR81AUl053x3lVdA3+sz7BzTgyhdCCJcPRlQ2bI0jT02moVSp+CXHrhokGfU5LLDMojqGyhEvPq9wcUctd4047SEX7TzrjcdUa3wB3ps7Wm0Gr47+5\\\\\\\/9ppvsHezrzMoQiAihKRQGNXjZLMigiX3X4KF+r8q8U+haI5gz\\\\\\\/6vzQu4zDZ03UimNKP5FxUa\\\\\\\/UgOcJ6KD9I2HW2yJnX0J1HSwBNAYenORjve0Za0+rVAI\\\\\\\/1Zioa8fuHaUNlCgjCY9XtBjeqtS9+Deb4\\\\\\\/ThR8dg1pEZO\\\\\\\/qzeQoi\\\\\\\/rQZoSaOyLVDGcdjhY4PsCTQumw+X5qjjNsXrJxARlWynzZ1hThdCobPgqROV0ujwA1XAninIgTsaqHAYOh8et54vNfTeaesGGhFi6o8exlMdhB4W01ZY3tjlg+DUKTdOWO\\\\\\\/uqfz2Aut75dKib9mG8TetW\\\\\\\/gSOK5I\\\\\\\/IimiUtQJw2Bmb0V4XCAbHx2xaB4NyQiAMnfw1BQ\\\\\\\/EUcEEZD+K668kR6YaqkbVlZcoZKKNEXgM6K1pfO1ZIYZIe7eFERE4B+zMvUOPrre\\\\\\\/pjdQz\\\\\\\/GzToUFdYgTUoHbMTdRMLQSSc3qTSAMkfqoVKPexcYKuR8EojTW\\\\\\\/zOBOghZ+oKytY70mpe6YDBVADtlE9eQsrfSXAWeI5TZk9WHEtOCTZQXvjHDjdyXr+q1NPQvahL1iL\\\\\\\/BOQFuOg7tkfUKB0OSscCbxlipfEv3h43N+ZpsROKBAdOsW9XbfHkKDPbdsp6sM0LzYXvjxkZEpqWBQ\\\\\\\/QNd0b0YFQ5dwUhDO3sg1b1luX2Zbtit5KvFjjkgMvk1h9yfzvEdXpl4Lv0T2ewqcYvtAjpoWdLtMkW+FxaBJ0mXZ+nyuhbDBgj\\\\\\\/oRllHucVgauQGqBMwTXeL4L68FKuf2yxmizASWOaYk3oSM9TLgKU1vW4cwIrtdciO+zeUK2mZypGYxiP3Ltsc1wEhwfTdL3ilNgsAiCXxWF1FK9fDtSWhuIamd+7erMeZyHVssBYJ0MsbU4M0p3lP2HadYRDBVESkqyDf\\\\\\\/WtLM6EgQwT+yclG+xS6WJfyp89spHaCa\\\\\\\/DIanRhnGT2jvxKy1SOn7lviLAj2CDAmslezTwzd6LisJyN5u759dv51dnMg1QDRKBh35vYIjbA\\\\\\\/l8\\\\\\\/kmJrH52WvOES8VFwRs4X150F0Jr3ruD4OXcCMNdOVlPDTKja0AZzx+GpIBBpA3MVeTWWN6kVUhq9CCEeuuhpv00ag8WKpo1CmzreGAXrIXtMpXizybb30qHWaoa0+1eBbnnNEoIGgwVWaKlJE5+2ZeT3Ej+LsxWpAcO6kLUQSo3O2b+04owULXDL0e3VtYE9jFG4qAJDIWP+2k4NUYioLWJ8YZREJA8ZEqdHkX785Dvh1RGP+xa\\\\\\\/RAb+U6wS0e5oBO4no2NjQELiPQGJCOpEsXMyyIweD6yX+E83wb5NHq0Pst8YMivF7MvhSGbbhTh8a5uwbwTNcrkS4B6RFVOgIWU+wMzwbW9oMCgzgCsSh+wFzctEvhQazMg+VMuSpD\\\\\\\/T0CWiDO0O36nVfzKdAyfakVimOw3NPKKHaIvbzZvW1IRHZQa+JWFx9zlWL4KtlQ5eeT12Awew2eECpTnjIKkTDCM2wBjhvujXzAvXnRDr4toQ1j9yVHiS9RZ00JPAwBIb92P+SxIp4R1T79e7mPO6LXp29ryoKZsKZv7MdDBoPhf2KQAew3+96erWruvNCw0pcLXTqYZT8yoAKOPsXm1cADQSOpJisqvGXQ3fuvukTY\\\\\\\/IxQjHyhZH6orYyQPNic66lc6mDMOFHILeTgVLEP8rLbzR2dxehXEBVMKdO3hPjLhUaDs5071fEZTdXV\\\\\\\/bGEh5GTDzDiMtg84YslTbYtDgplGgE6iVS+B7CTva639N98FdlOmMC+KrgsRNK\\\\\\\/scQQX11f+QsUbk\\\\\\\/F39r8AuAp3bO\\\\\\\/sm\\\\\\\/\\\\\\\/avYIuxRtQfsLH555owC9+dsY5vDeQFmveCasdHSVnsEQIbDWzAfXZWd\\\\\\\/aviyqFSTVhMJkARN7Hh5QAEI0uPgan6fStt2qyuS8L\\\\\\\/YJBfEUNrUWlJx7hcBIbagCAcC3mVma6dA8p7Od1z+MD28QuCVUbqQLwsEHB08WTjeibjETmTpULWmAGk0OFesCoN\\\\\\\/4TthugrPsx7VJbcOW9ZM27lwK6fEvxOhBF2MR8z0vzg5VFeIu3HAU6pz2GBqY28KrhiRD9ree+RqiQ1Ofj9yyH1YzJkDZgUWk4+qwcgN0GkRTNO5FwYseFv3qYZHrRyIjCyHrfwBF2XdyL5hfOgfVTELuxon7Lkrp22GwtWguMGqI6e6KPw6F6VSZTXkuXxf\\\\\\\/AnMUBCpzww12VU17mIxv3w1qitoLulO7jqIh+Bq+MUAFK7NwZcJdL6M8YWU+oge88uscaBAuupk8byDtiTdGvqJo\\\\\\\/9suc2IZ+YA49N6tHvoeLzscjkdKZ5DGnl8uX72D\\\\\\\/ez\\\\\\\/9KRyP2y7WmS8+6FMbU6fkFWe7rh8Y0486T7INb8Ee2prRdqNm4UubpliDji\\\\\\\/8rkQ\\\\\\\/ZvfnAVrC3fycHEQsth3gPpR6do+E3jr1svCOIsTjmYT3AzIbNDNg2qOdcTN2iwalaGcPVMTunM9cbsyns3AdW9B+4Y8v3EHOWjHQtym2ZB7zDQf7L7uwHGFlisvL9xoeWDYwevIEVnyWglHsWKn6Jq2adQcRL0vbXeJRw+JEmcEO9m5hgMu9+VJLT0xuERd++gFAgpH2c2PnTJFQAJGjxrya+B5Kco5Q0e+vniU7PHeSrCRJQaaxfMb7eGUsg7LJjK7QGS8uWCKXl6cKCXsdW0jxwuDrSJc6Ic2KypgbWwTG0m0noCYxWPHgf9JHTJVI5\\\\\\\/4dSVy+EisMFahkjfsa57xrzQaNyGSQTMYIDs2bJCDxBoT+c3tajnR6Ajfpw0Va2FfPJygOqf1t6vCHuK1QT878IbcSDpq+JlxLkA7F0J3XusjuftXrlwUv876q\\\\\\\/pEXxm53hQK6BMs3lbC5YfxfNBEEBa2GVX8d194jFjfkzaEDg0hvNQk69zpUdZcPh\\\\\\\/iyEGntM9i\\\\\\\/lR75zWP4GTy9cU7KAJqMXHLtWuYFC8X4u8QaSJekaIHuol90VhrXIk0WVHLpcI4U4khxa+dTplp+LjRc9oBKIc8oaEA6l\\\\\\\/ixwtZgId8j1VWqhRB0Jj7ileEDqXQspjqv79jLz3C1AoPPkQV014jaZpn\\\\\\\/9Zlfb0gL8wsTpNKDxGPYkBtfINM6kn6EBA48VwXblISNbbFv9wSdbL7pPj7fzLtPrULp1L08a5mFVcSJoyUP9dzQOpy9QD9\\\\\\\/WVXmoJ2cQkktDK+InP5HU3h7BcIh6y9yImoJATPfAaZUmTEpdNPtlAXe8p0klod62cOczJamd9n\\\\\\\/ph10ROu+3ldbXoFfId4iIu2RbVFAcn6CnZlcIOAlkhqzHds3v2LBy4aIQ7\\\\\\\/C\\\\\\\/QpHLhuYnOPEuwbQgxFzCDCToJPctDrtKzCp8w0i6Y9lZEmKUBWIoLuEkRiX9WZvi8ZNvSSAP\\\\\\\/8m8wTlBYzBKXKyZmMfu6GJ9903KgUB8yLFXumnHbXT3TB2LDP4+7Gmr57FQ+QgoS2ET+zGvzdNe5sLxes\\\\\\\/rILhEEXcpwyX6V\\\\\\\/xY\\\\\\\/XbcQMMxRmhZZXa6hEmYmyXbX9aWjoLnF0BTUjTgkyyWRD1LHxGfBZy9wHtFuldmbFRTYcvnoMHUvbQvBApnySMEopbZOSJUwZT9wLzB7n0cz+XgMh65eXcgKWvoxz5g3Ko2o\\\\\\\/ygQ+6w80PJK+JqD+jU2XjjpymVzVMCPO7saRJUMnpFqtoBRmyrzjPb0s5NAYLEbsRgAQo025yFvVqbXQ3ts5g6eZl0meS6Ui0lyPEGDH3wKwXwIzckCAOkb1NzVXKpJyP+Z\\\\\\\/pmpGYNbtQVjfvndX5jwOuJiHXSAPTnUqoVhR1fqJx5w\\\\\\\/uzwzFcD+kBX2W2RkuiGJDOdtL9vaJfFX3AK2JNTX4ydm8cgUbRTbz\\\\\\\/JoMSMaHz7OvryexVH8liRkwhrTy51PiCA+4hDLXC3YEsUBukKebvrs31m1fFIWZjj3HweHr7OjhJMR3Lff0\\\\\\\/zHdc5CVkJuOUN\\\\\\\/TxCV8EbvVGW0hQ12mVvhrpp7D+a1qzlrOPQ7WyU8eUb+18UCltunCYaS3zSudi72FMG70v3JfGFfN10gHjaF64SsErNO8iiGBdDz+9hI\\\\\\\/KlNX6Sf3QA2uIbupNKHNx7kaVVtYmdDSl9FA6tUFvE\\\\\\\/8NRzAfuocLZ73w5s+XAhhHmgWsKKh5naj4w3zibckvIQAA\\\\\\\/TqIQ\\\\\\\/YB+TiV4x3Wu4fSkDiNBUX2gUrrlTTyTWYFLiwifKBNeOIXdaf5fBcir07JTLviKdRcr+jRNR2iW1UhpvQuRsHDUDMHZ548PDL7N3rcvS35PnM1zo5aPxWb1YoYvKybLKXxpIt9dtyIBSUtsMrbRXC0rRcWOvyiZPH9fLLeZVlJYBra3X2byS\\\\\\\/6qt4rqkfhVmLbzpp6bAETDxxdirMvlfgemvJdYT43rLTKc+6oAkwXWcBJRgPcX2TylwcIQGPEFhQ6O\\\\\\\/PcW5GESB5iBbUco7GCPuFTNCjrHFegd0\\\\\\\/A3DglV32XgdAS7IBdZLYjCHOvA+9PzGC8OP12SAjpri9tG2S93Pr6bVxQ9E6vbeJTWXyP1NY9ojIKziK9QJPN7WJQHuE0EqL\\\\\\\/HcDbIW9PM90DqWo0txeA8HmhGT+8P3\\\\\\\/dEG8P4YppCMjXXQU0NQuH\\\\\\\/hf2RSb1+JWqNHD+xxLeTOqtqzADSZUyXeYTTENu2p1aoq14faYf7adi2\\\\\\\/GzhjnBCl7a08BXJoRTAoxlV0UkRlzALEMU5uykTbX\\\\\\\/jGvJrbngzojeq7VKz4rEXl5CiAuhOSkFAttQgDzmMqaw8SArp+sPZGUnz4U28z6x+5Ukjcd6EzN\\\\\\\/Z01Oo+UlkTIQ8NvZXs8ks7BwqgU4HpfI3JfH5labyHIRbhxGCmPxkeBjVwPk0D3qx+x\\\\\\\/1MWOmcvNVgLC1TvB+7GbsktJunhfBNl0P4IKU8wp0rMEwtSSuoOhPJ6jb4Bco9iAErX0TUSfAIka67X7lS75ZR053\\\\\\\/YZ3RTGfB8zpG6SfCeaawoX44chfenEmoeDtv9OoMvaLhpueHHetfd48xMvGJgwkAjVVWwIzafyEg97MTqYOXP+Y4r3OByQDNgE9LUjy8gpu8GiDeyG3Zf63roB1ZYyxkyzQMd0v3\\\\\\\/kh0c1jJq+Pxjx5TIsykYkftJpohXNoZO3CyfksBjgzlxcSBd8b0X1yO+csnTd59CwSY9dVZAAMYIJL3SjaiokkiGhafL2vavjE2Zs2zZp5+tfLLYTIYm4zcvPqVjcuhzWPb8\\\\\\\/AwyMIRhDH3AkzNfjLMcuEoMNdFIM7JeDjC7jTaB7owQImtN4IvZ9TgZ+Qcvpjor64wamkkpxz4hvlc6MUV5J2R6wTi1kTHYgOHVrOq0QgNPnaqs3+JRiRrgj34tNAmMLwwEVaKEHoRGZ6DENEEIB19FLRWaE8+zSGnhYdtvgkrnXfeeT4yCIXN3E5u6r\\\\\\\/KSl2EGUKxksp7Z2pitlAQnBaj7poXxn3Xe\\\\\\\/yJumogS8j7aMZs3tL+7216jmKkRx3NLkhW9FhAmffvDel40lgPI6x\\\\\\\/\\\\\\\/JB5GjaS8Gspl+VNAiGbsZAbhutrsx6nPdWdxzw\\\\\\\/KnhyQEq0yXrUIo9u98kS8J+LRfebkfG34wR3KAJHklJiHTL+nJL5LM1ewX5F51aokIABUbOfGDxSGxF5Dj6mXpcbf914UA6H\\\\\\\/dIOrjz1Qs\\\\\\\/LOGr05EhorgwHNvxGapQU2Ay8caBo2FeKi9XnIKiCDf2FD23RugewvMzfkjupWBNqMXAoOanF3mFaba\\\\\\\/oxLcXamVCDG8P9lm8xPPUlRNcPSNK3a9D2aAF2zapO6cng9deK9HU+NgCJ3gkEtfkrIJmDifoo5AzmvEQxiyN8Sqa+o5M2ld2E0ZsM1LUbG8s9vKv70X+mIzNbz26Lx8b5aMtTAeOwXRRxibzPK6fU3WGH3s9q6T8rA0MpfHaHXEt7IE2OfF2MHWbFrvmXheAwd80lLKAb3g9B+DALklAQRTVUufyku2DVSOYQCl2BbU5kX4SxOHv9T0h4\\\\\\\/Fcrcoden3xSasDh1xZvVWhcGtGWyJf6c+eiRGgHTf1UpFcU4TvkIslgAD3glWPHZGX7GmJUUhKA14ahjrvn6rqltE0cf8nlu\\\\\\\/6bAhYI2K4XPYTEV8v4mMXu6aUvmqB\\\\\\\/ALjyFmX74Fchp3sGPm2CnfB0qmxiVWef6yX5wTaZehxrxhPvRHdAADsrXCTCjxR8tDusG5loDeCtIh6bDHU48SmKfP\\\\\\\/v\\\\\\\/WSMfFA2veh6TWDjW8tG3OvhS9vBHlAaeTmHuhxjMYqd4NtogIoKe\\\\\\\/FsloICkBo64IABFsiigxuRYbaes7irudV2kgeqsY1MMKfmyaAZXyqVBlZLemsNhMOAIkIUfNzJvaHimj4CiPok8ph9mjbhCpvqRKlsp4ow3OXDgvs8P6kB1NEGcQpLjU9grX4sbryF4ZsrSrO+rNb2HFx4MonnXxD41NMmmgsGRnf4tiIdAyGMHqIqmZ6XD3B2+qXFnWANDDCoOWHMYpDr4OM8r0IzFlidEG2U8VT6Iqdcls0Ttxf\\\\\\\/uKv69z\\\\\\\/uCTqZIMCeZa8yXsIKxX0TiZ5jmyp0QCJHtwvZFPBwoFufbKUvT6dDqxltTAiZP3ME87HnkIV7obizeo2FzEDfDSHHQSQIriHa6XBKjJ4AR0w9eqjUjLrV5Ta7Ilwh+YzsItE+SLkZmIm30d3OAIHE0MNmld2CAvJgSWZeUq4WSPH5e5O1KX1r1o\\\\\\\/SN\\\\\\\/t8qiPqoLRNJbyipND0XhBKWhoTs1T1HLLK2+r\\\\\\\/W1+leS8HIgiODk5NrrnNl655p8XvOOMOYVoogTx6ZxtKoLCKh+Sn99KyYvf15bHopn8kBDxFoqe4KDfSfQTR3v6xvp9v7TJF9D8GZtHydIj6y\\\\\\\/zaAhII0egb1Pjkc24bGuRm482607CQFHqhjeZt2keJaUK3JJEF7KgFp3BwnvPawJcBZABEAUlZ6xdKXa4Yp1WORpbySb2O67tbE8lfxTfirgDcHW0k16JjA0YOYRyjeuOFBWWG92VJviIpSnCPbAGEvTxvawu8qNDT8jnbtP+QoPQgmioMGdTrB\\\\\\\/2124zqsU4owR\\\\\\\/rUvAQbPUntfHX1tic5CC1Iu+MNoWp2A2qCcdDeGukQtNnOg7HoiI8cgGjQb+687I\\\\\\\/HPn06R8IqMW6eQ8fC+6cKzVZnhBIf8z1GpXfZLGpZ3HpDbtLte2Ji21lGFnYCWq0GG8EQSkU1WvOQRN1hMpkZUFY\\\\\\\/KPKxQyO4a4JZO5MrE65eAdGXxbQWNcj7YnOXyg1f\\\\\\\/Xga2u9AyocyaFyuxs5Hyo9cm2l3EfliguCkcKTp1Y0sVrn5EZpmUVNaq6zhQ8bqRgkJil8VGecn5Ysc2t1ahR59DPW8zJnQOxL4Kv0BRqfU68got59L4JPW0LVZJOdT9lbv+gcRqMX5bHNzuR8QmHhQdcjihBQyvDRolVf4YXRa+862Ka+Jsi5TIHOuNW0oB3L1WnPds9ItXS+gxj3m\\\\\\\/tbsXGaGPDoBtj\\\\\\\/eEomN2P0dmytVVbmYoMFiIg0htqfDp1GnxJwZqtANGveCVN8mLeIeePoFJ1C281Bnz870V5YtZegKIc4KWdowx++bPx63VGT\\\\\\\/Mo0w4adtfP\\\\\\\/ygOWuv553\\\\\\\/lpXuqt1psEwvnnODUhgwGND6K2n+pel8Pzj6Se6MwrxrYn04s2cCudrJ57ftw4ud1shRQxtfVC8H049+u\\\\\\\/HgGVPswggeZV+jIKAZ3zPRQ\\\\\\\/nDvi60UTD\\\\\\\/1XM8PEQlC9xbreZtu4ZUeVXBNzRrJg+Gs2i0v4rOBvDYABKpg53yDcfNW2TSFc9WAzklzlp0xZlM0G9N8Ii2e1xvQs62fiQzsOl\\\\\\\/wIf4COxbrzlX\\\\\\\/gisQiWHcOY\\\\\\\/HKGEX5k82XwiEuqfWV6BioiMrtr2c3PiOolioHmk3UOJL4LLKiIbOp4nKg4vMb1Z0\\\\\\\/EPbFTH4\\\\\\\/38v3dYTAVzL+0qtJIk9BMP0uwGXzRWPaLqgr8hbgD6vnXk4\\\\\\\/60ef9XDxKLhUi+uupAwVUuwzIsFgoPkkvaIpY4f+ex6aUiG9jQ7dMZPWHnxm0R7hPQaiNTod1wO40aKvS9Vil53L7Xg0aQolfI2YlH2XvNMAnlRIRqXj0gOrF76tKsbWzHZbRoYkSEvI9r0tzXnUQJjMMZRklBb\\\\\\\/kkWut659v\\\\\\\/i\\\\\\\/H3jQMqfDw5HfcTiny+q8OPyvgfnm+A2bfzVSyRh6Bq+oIp2t3Y5X3AGW\\\\\\\/11a5UuyFm00iY6Kct9hKqGU1wgkZR+xEeQgqyuh\\\\\\\/Prv5T6VLv0JlC5dvH3GtQx2q3fpiVJ7u\\\\\\\/9BoMqvJlNRuEfhmlnCnt06JrXdeeO7LhZj\\\\\\\/ZPGsQr7HbkmmUGEoOPvsLPM6TaXV+XYqZW4CgeLzAL1Iww6KB7WYb7jKzCQMnYZLzWnNg\\\\\\\/H\\\\\\\/CAUFwT4ddqVFl4v\\\\\\\/tUAVmVtYn26sSB1GmQr1PpszhIH8+MZD3MiNI3wL6G0lBYXgLVxve3mSkLP+gYzroc+lotS0jpTysN9ajzgY\\\\\\\/RvB6pLnIY8Dv2lrCwKWqZzqduF7ycz\\\\\\\/Cukq8GoOTeOIGt3+M9PPJOackW3+BoGsMjQdjWriQGDnXI29bbJVmngHZZ0B9ZC9ghtnS6xudxRpbEcOy\\\\\\\/PyLaXmfzw3fcLkNjPebsogbcSsJJLzn8RbFDLcv0Q6H5hbu+Zs\\\\\\\/oQQW4PVuD\\\\\\\/V8L+rvgCVlC7QIYXX\\\\\\\/TJDB5xvlHtyZpwz6TmfpC9rtYp7WU+ByC5x9BkkToMeJjg7d1fIxCUt19\\\\\\\/HYtVC8w6BAAO8lFgQsBw\\\\\\\/8mvDjx4mX9ifMo4sUmzTZcaxCAGsCHa2YNE5Y+nIG\\\\\\\/40AGHfr4tiXFLF8EAItJaSAHIJMy1BDzPt0AT0z7HiB3Ws1Lj4WAiZ26RcA2BP4MfhARJUrwHcuaMeSE0VWTovEDBxNbwt+JGnPniGvMGSA9IGKv0tZXCSa7i+58vDOBZS0FHP3BOIFTW2bQgGwNlDFfrEF82\\\\\\\/DcLh9PTGGHFbCWKQRS3J9aFImbLtdP24xvIJC1+ViD4zwZrXX3XFhvRW\\\\\\\/96YC+0BtAjenlFc9bfHCJetJp4Ar\\\\\\\/NsUCkyv6InivP2agLRlTUldCZVijFvjKUAGrelQWWvFthKSUVaE+n0sddJh\\\\\\\/qisHpAa5zqCHdgQ5Ve\\\\\\\/wlGczTGmw4DRWduGP7fmuySWDS+9dhacVY7nouXJmW5R1noym5r9LQFEpOT7qhrs\\\\\\\/WmISdOeQbhOCsB1tBYlewnOCwLcPNMFoPF4ZJdIesuqxPuthDdGZvWNbK1oNY\\\\\\\/SL7e46x823XZzKNAba1LQ7JTofHscC\\\\\\\/WusnEPF3BuGjG+7oayu2ddTpqDnaUHhgovQrPb2ZMm7vaBYyxhyYIK7r5PqZdTMoEZxmMrgIRP4dSrBqhj1+SPZcijx31dsECJb9Ofuap4TUxdehiWBDAevHfUwrMZNz\\\\\\\/r+cwMfj91gNfka3o8c022uucbHavmmG7bumsUxJsTQvhPcEwrkMvUi5tRnVwcXTvCAc8ce4lecKQSXAc+bhiA6NGAcZNwMvcAK+7rBA9g9JhrPovbfJ8F9sXmNKIIDPHPVPoXUeVJChmJf9iw7sPZLLjJR764FXIxyx+uZZJpae4NdGyszknoHL0GV3bimBC2GY5w8tSmI8\\\\\\\/3giRQovcmeCxDNz8JhJK\\\\\\\/wmGyq05juH0rfF5yuqYixUjJrVr6zSGJA4yzxXuXm+xxva\\\\\\\/4\\\\\\\/i2uEb5umPBKNt\\\\\\\/JgB91pj55rENjIyOx0pvrIsrYFrl0FaKn594lvFz2YLomnTQc7vKYart7Ghkk310JeMK5JBjTfXWmVQq6dkkG4ymUnffftjRXQ8HMNjpLo+KgDMnPj28B\\\\\\\/MJ+\\\\\\\/OFSndItLhs2HA1DODFGuCoZ2NFSpBVZRTn93oTV0PM8x0g3qo+DFxTpWA5uluYuA4MdR6\\\\\\\/J0plTQtB0O5JTsVrjk1ygjgXpyZ6SLgm8ygP8vJoFtFH8ev53JCK5QTR2yzRX2nyYl75I7FA0zXJK\\\\\\\/xyxhy7Q++xyIlKANorXmibHZPmH9oWydl6KA8VX0QAwZKWa1RwwV02W9tw2hTay2P0JW\\\\\\\/i34fLzvTzjY7Nn1nuaCc6LOwpAmqGb6BGDeirjNbzXvIRzeqJU5YMdDgzLfdPdyvWgjzp9X0eaoirb7sA3cdNKy2pFD73q\\\\\\\/fZ9ZXHQNo\\\\\\\/zF1xa6qzwjFobAEKGlKkXBbvPp8dnP\\\\\\\/6XsJkdNNdvmxY\\\\\\\/IKhd2nHy0p1qi9UvMy7JMGlPHDOXqmUI3\\\\\\\/wn0EOpgLQ52jRrLZ58RUB1vkuvd2hRAWrd6ol8AZuaaiUxdlwF+TSDF0cV+WloVaUvUCq2YoyUlpmMO+WzUJNLZglsNp8w1xr0GMczBs+hn22hKPhUjY2MznqadDvI9FIF3pRytAQTyrdixLES\\\\\\\/\\\\\\\/yosgvNI\\\\\\\/RvBJpvI1xU+QvYcY7eGIH9AYeJt7qzAFHDANB6JLsxN84ONqX4HK4yqJJ8yBqDdJmcf9nU8VYh04CDLq3XL0M6QChZ766ayfhWMikuEG0adm+HnuUrnDp5HwybsYveorugAuxqbpIhPhfVTRDiLs\\\\\\\/Fvlqs6wPyDScCAcZa8zXO2DEpSMMlvPLDCjXSQYP14fG62cBAUzKU3zr1hytjb2+KlD4lTW4PG28T+cj44OWfY66LFNz0JhEXdPFZv0NhWhf2mX7X6bFpUQ+1rmQZ4wFpgLWyFcZNbK12MOD3CLwV9dDBWmL++7UhjglxH0OdZzlZYVnsoaTB3LAybBsYUwMSNUjlk75lLKK67BppM5iI3doAJezDbSPEFHtzwNoHmMY5DHcMqPejqzA6MecWl5nh6Qv2x53DEMr+5UJ7VG7FGe7ZafMCXHB\\\\\\\/cEDNORhNJlYGYRhroASGGLmiMW87gc\\\\\\\/O071fpKFQ+BXnmKPQzkyZo45U74T1z+mqsyZDqLaPw5oX\\\\\\\/gxpDElO57OxzjZSWODJ26DYVjSfQV7XQnF5nmS6gl1zZJMCAoTLVpDNHV2FHGQyGPv9uC3n+6Yz4GdA8FrCDR1CuDhT8QME7A8ctnhmdHI06QfxErm\\\\\\\/EsmQmB2uS0yNyqtkCykyclV4kn1mZu9zVKzwmPf1pBODpQRjSm5ee1EuoSgrcEbOkEWw+zeUZwgwx83ZdsHZKFiKO6uvNezs\\\\\\\/wSlJ0oJc9FST7Mc4jqOrllAOCLwW5XtnSMzQhDmEGP\\\\\\\/Y8ZAAMlXG4+L5RL0pSb8cAVpwuZ8IugPOkUMqxdl3ZQP1V3fqBNfd43POU39T9INlk22p6Vi+Cwk2WTbXGVNyRyUDkrNdfntSQkVvkdrZCnh29NYW97MX58Hwox1UEB2vQq8PGHV4af4cgTtExtiSA6g3v0rkpIkjLMKcRkp7mrcRyrd5A2vcVhoBxym30hx83p3jCWvmgfpZEOsC3HGdH4ZgxTOFeKSkVh8N\\\\\\\/MalRiwPODfd0vsFhN6KjbYn+PGCMvhQq4TQEOFdvULlYtRZMaGE4eFyD\\\\\\\/L7glov9ce4W0j8Xo2nklrnKBu82GO4NvHUO8fCD1\\\\\\\/xb0NHnJi2GMh35J2dHeBxcaGaaBRFhWq4SwCHMh+5e5MCk5n1m++gvVRZwUjOp0oJvbYZ3rvSOMQV8UPY7IaXzCQGiglRFA5vYcblECKQ5xF1vGoS0CB5ITWpwMQmxRgg1KXM894uYZ8q318\\\\\\\/lGDDG9AaoD8hf5dOX1JIIxFH0sgnkBuOoOZv3JXRbxr+RhmZgFQtgXdhAF1fYRxeN4GeLOvlCliLMrAXdjoBFPSywJhtUCGglYxh7DX5A66J6gOO+IlEQrS1Q9uxZJCDnaqvaIkRVGDgAr2CI3u6V3Q7SF4CsdWsWoiZ4WIV6hWUYcFea7bxp2fiqVLlWfROO1WltsezAS5HeTpgWJR66mBv0W+Tl7VhP5l0WDrIWZ\\\\\\\/ZN9DAfS1d82wINcjlya5CBNmJhwemaGAT1FU\\\\\\\/jF8XLODAf9h9PrUntTUwZ\\\\\\\/+pzmvWQrl\\\\\\\/gwz3FeBN6Zh2eBZI6lcaA9KxBaBAs1D43NmvGbjbJkYb5ele\\\\\\\/B\\\\\\\/H1lndeIopcMnVHCelsXWX7XET5numCAaeubh3U\\\\\\\/v5ffgkWZHTt5g6HIo+VMD6DeiCj5VNw3mKWPDI1KWnNJm3frnePT\\\\\\\/LSn0eIcF3XKhqpxoCOCT+9u3oTCIvLbIVIiM5TUxPuGxKatFKyJE1riomVB+im7UePaSmIBDr5jjkfw8o5nIIn+U+ENMWDt0SAXk9kAk6NuTDo14xZQAp1KcC2QpudevecxrzGSxK5g6YTPgTm6+ezVtaTdveAqgxUTZ\\\\\\\/M+r1Ce6IE5q5r+k5ow3zomeINDz57bOpimF08mhFBQzTZkIktQCIb2UIXosqfJiNtOv+Ix1Gj9sNiPfavtfv1ajIoEqT8\\\\\\\/0qjkfgbDjAT\\\\\\\/KErYwH27IATFiAp6WkiWLGzO1p1NtULbBCmrRboT6TUMCGgoKvSwk6llLgg7GtQ6NysJUQjBTEYrK\\\\\\\/wN8nvRCk8OAlKkOOmYkM9qehPGuUxHCIlVbtQacpKr7bUfRtOUsEMx6yMG0af\\\\\\\/FTNRHA+iE9Fe5QcyiTEb8LCjOSOPN\\\\\\\/nkhnv+hshxk0ubO8fBWLRyQ6R+pl846gsSpGPOEg7iK7xfruAhgvq8byNruxF72uwwsF+9Sh7ooXBLXPIv0m9iO6+JoWcCIL2GZTvpmhxf91KyREjhq21IQsSh2Y3NHVs3KHrrFKz5Xu5jKxwOz9kSQ3A+sjghrcqjnX+p0wxtmhjM9g2aXnKn1XKW9aC41MCrmO5BA4WnXHcScvnFHXktPkfMEELQk+a8LQ3nUeqe1k7MByQRFgiGIUte8M9IR+eyxvjjUqsc5SBX62kw40H8ti\\\\\\\/cMx2p9D6gCkcKNuIgIp31NQOzVpoIpQ1RTPjTgLc4fQnryTobRyX2lS3J0iilolOAfvR8ZPM9N\\\\\\\/VDG8LhY5mWOYh2sfDAUgExqn9XeNbj10M8jXiAV6do6po3E35wWoK4nsxRcYzKTe0FjCf7IoK4k9zrJmlrJV\\\\\\\/3h0cIyaxoZTLjm4mPSasEFZQGc52bFoL6xJdeMbbShumMX4eD0LzKWHAvB0+Ln04\\\\\\\/hCNCx52Sx+4S2RrLCXqfT\\\\\\\/MhRD0Pr2Oxjzvf2CXlpyqDwgR8ANg+\\\\\\\/5rvneviXKTxwGSEU8jB3MaWt19kL21Xb9wsllAGYojkH\\\\\\\/8L5xpzoNQzr2StIA4cHcsQ4iaErseWQ5JB\\\\\\\/fUUyvhJX5Q59V3MAL4rnX83u0BA9KPYRBvAzob3xs+uIL8zbfAXVR4A25hKXQ9e6NHe7KJpgJLmkR5ONUQLoPA1kc6WzA61fxjFFgwx0TwAj1LTJukBeyds3s40lQVfEOtYsyvItpGAzZGWEAdS46RE45KHcVptjPv5pQipYTR+NiPPl8TiE6AToTuJoXkxwsLac3eAGi4AxUpqZP02X6mAfEEUG28dEnZOFN4hMYA7ukfwyzq8n+knG9VM4xE0vZOMdu64hmYeIpobDiz35G2TflWBCoIdPSgzExk1WEzGSe61PApysyDKamf+\\\\\\\/47LBmUA4FEd4ehUJTgAGnq2hIkdk\\\\\\\/fqompWzfQUMIgm3i5fll1ctKgqmEhEyc4aRTVqyWdKJ3PAI8VMyFdVei0ywPueGvvb6MCi+aeld2fk97bw0jVNYZ3yTYBF5ogCYxpErSBc9wwh8pKy7qp45F4q08WEp+Oqvws5A56gcCBhYwctKvIavu7K1dwrvioWrobk1klTGdy6pw4+TgzJ3ATJhAy6tVZR8Fxdjgh7MTyxJ\\\\\\\/j1rxwsCppxLqQE8bcB8WqOHkQjQycCmrdcLVQUJ3+EK\\\\\\\/iIgi+qypQxIyjfzIekIXUrKQZyZcpZrimuQ7MsonluPWkpAz\\\\\\\/RyZ8GFKX\\\\\\\/gU4aVthHsyIJPKv8f+YgbnbSL3nmcCQr5Z6pyCKWgzLa\\\\\\\/BUcA8jlaBIP\\\\\\\/fjuLjmVBLikycj+kXwrL\\\\\\\/1IMpwici2BTqvRs\\\\\\\/izldeNlKKHOcgrc2mCJ\\\\\\\/+0GZHS6y9loCTl7njzAhhFBdxL7qsn4QSl0eytI3a7rpISI88KEcWC6eONFq+tFJUbc3+i\\\\\\\/K7jLGLD1t6K7gwiuMPqnocqAAN8jhzSdeTld1vt36V9pPRavBPt\\\\\\\/jiCAxoMqWYFp8TXhjlnxwnfDp1Y1O0dg4R8trRkoCRXOnIkT3spvSfy9TK4F4NNU3tNR\\\\\\\/Y0cb2fpE30JIv+qqYV5JGOrTlv3un3wyMI5a4qrHAGzUAcKkhXod0DmdBIHgCPr9Jb0toyJPDJYgiRoiQnAz92WWToRTftjzy1+azEfO5ckAsrLUM\\\\\\\/IKwOfhK1hD1innxYeQMYIhhC06vCDfVTN8F4SK8RPH+BOAAGmTO4gbqss3S8O2F6bgPE+o9lU9dpp2xPmXEmXHsT\\\\\\\/AoSHEZ\\\\\\\/m2HIcJmP\\\\\\\/Yf4gvou7ydoqDn1HMe8\\\\\\\/MymnT3G1doRV91tDcMa8fwJSAPo1t8FLLOSB7XzSouxb04NugNaAqPGT1og1bkZ0WqI41wALQ5zLcJzSi0MhVioAdwDWmGBJctEWRAWzxUjZYo4MXEaqYa6oW9cZlOLjcke9eK5UZ28yUX2MftBqcLtAOL+3YJUkiiAs7BktslI1Cwcr1A1JRqBvGOshkvQG8vE522D82kjGm4EAbwvsX61nnmk2bOn3z5KsFOKs72cYGakdBrGM\\\\\\\/3ASwG3DqBnVtRIWuX7NO1phtaAbpc66ovPScm4yuwZoDg2bZiAoFsfEv6vqRte86qpy2TVMUmPKG62O10yWIMvIvAssJKV6niVp1Z8kP13oX5UEvV6lNclTJcc4cDoTmm75s8EU8ipK0YXiV3w3Mkca87rjzoonnNFNMmxJchMy1tqYYHA7tBG9dpsyHmgqoAcWD4ZphPOZtbeJAbE6Ikx0g+sdRcAvmO8uD0oV\\\\\\\/ztmsHsLOKnsMi1LUsZZdfN+\\\\\\\/bkOaAdCrD\\\\\\\/e34m+AtkgTUxlFNmxXBt6tz\\\\\\\/\\\\\\\/sUL4CoP0zkqxqkArm\\\\\\\/bPiV8osgdQns\\\\\\\/6gMJmdq3vTdLS0+mBS7hptTe5\\\\\\\/OodW2FZjUP\\\\\\\/QPHX3XKBHtMlUwZswd4TenDxPyQCXoNS\\\\\\\/BCtVmYNzb9fe\\\\\\\/8ZfccivmhzyrvJ035G6ubUW3aUiEKr6i3SiqT20JLHyHkKN2jSkzUU66hTVpTmn7A2cACrY0K0OFXYQh7EdmZlwRRN0PekCFuEuD8SxUDNun8jS\\\\\\\/WUAY1Qj7SI6nglCfg+bNw7drEUFIuo+Wn1IOeBNoPsNSNhe6CINRgL7j7snRrJ4AS6Opda\\\\\\\/typhtF6vmAHu\\\\\\\/Qc8B1RRZEluRtewWefLuvE\\\\\\\/C9y7eHeey8T8Lk65jJ8sMOyv5b0E1A4zDb42TDQeeRI6gqjtV1IcBUhyenfjWFdV+drOKVj2YWkJMhGiIfWZT80KETVBZeQAON20W9PYdH6O+UNiDg66n7+cgyCJvwdknSHD48xz37x\\\\\\\/2EDlaNc8WbJTq1zJuWVLT904LYI0TzF0ZxrjfBEN+vy6eXgzYvro6VZStb11BlKtnwGNYlwtm9Uh7QFZ+kMNvzpzpvL39e90GFEBkIslrSvbkeauGkYLCWW\\\\\\\/el9VzNWV0A9n\\\\\\\/nfBGRYZfPKOqTh3NXPSgz7gMoHjPQYL52Ma+hg5LdXUjJMMOpTgDykrfBgbC06mRJ016h9gX4+xlVGgeTLqfVuVe\\\\\\\/hoH6tT7qxHUdIt9fBwCSS2ZYA1r1H4RPbdqwyzWxU8dFJpXlfEbK7+bt1SHcOyfs6VjEGeP9mmRj9rNrxPggeBSCIllLlI9HwTF5vVoiXFyA+2BIZl3lyi9+q1zOmER9Dmoany4ds6JF+oQuJQLl3s\\\\\\\/SQYmuV2W+GLJFhRk2Ny4vy2Xddo8egNSbypAeJVNPvv+vyn06YdyckluJqPwbSU22c6+YfZZNUy3XAHJcimsbbp8Eej4h7G2TzaaZ6bPhdBk5MhislCLO0RyHVym3L\\\\\\\/wNQMRPsN3zOuzZy1gaU8M62LyJwH1YDEkuxLgKEk\\\\\\\/8YoPgqgOESZmuY0lGIEpCO\\\\\\\/knnUhZmMZfWAqA5uzM9YZQeIuEK8LYP+LzUXXHRvVjA2Iy8sHccNwKD5qO39afWmRmkxtjhvFIUuzrUyOANIgDvP11zyLOwfTv66ptwNAWcN+Kf70YOAGAItSeQKVmXHuvyKQQ4OvS7mopxSuPCgFFQk\\\\\\\/6IRKROvNbIyqV4rb9ryL62x5ZMjoYsjnWJprA9mlIXy1FqeEpimp3nNhDF2WoVa5\\\\\\\/mlkIA\\\\\\\/DfR1J\\\\\\\/kF0Jn23GIy3LOCzJ0Y6kY84iw0HIVYZQD5AjO+H+Off4CBZzFkHSxJRC3LEH\\\\\\\/UFh08Zh0V0k86isC6Orap+cpcFjdzTbHUReLjfd4xWRAVCShP8fJ6phqzaGDokwo4IPfsDdf6coaiscls1B7Lh3BtAr7i2XoUNFBGDhndjR9a3s6vHS1+aBVgGV9yOs1wUOgzMPF7phJ2e9ZWdgERYWnriyD2v5+qPFFFVKq7CKYGvmbhfzDXf5bO8Nc74GDmpHvbOScsVR77pxp\\\\\\\/oKfOB7gIAKxidk96ypzjbX0QNEG1\\\\\\\/0E8JQ3\\\\\\\/76vRXyi+\\\\\\\/8gkTTb9tF75rSg7GFvF6HrTvpFUSMbfpmD+meLG38u2DwsYkG\\\\\\\/I8WZJRlWGSMHdOqQXtvFr3ZX7lbGa8t\\\\\\\/0TIN\\\\\\\/zXLoID3nHnTCgbx+rkKwWDTByc1QZMJnWT\\\\\\\/D6BR1gH5n2Kfkyh6NYUxdDqxNE8nx8UG13Ukl4Y1Go6\\\\\\\/lp3AesvtrKSX0gv2wctMdAWSBF6BiQwn3N9KTIgCMEEy5L6wiGtvLUsy9fliAJPbhujSXZdR8kF7UFfx+W1WUnUImeLgXfJynZ5bp1YDd8zyshxjNgfiLtzM8tptb5rcAcJfZbVPiX9o4llty\\\\\\\/eb+BeGWDvgdkB\\\\\\\/aMWj7uweo3rjUVOvR+mjN4JD8j2cswrLRSC8XkC1hmt6rwBHyQyuuLez9TlOzDXvsyxcXSdJ442UkmovKHchUSzjplNI0RdPZNIiqLJMsImjprxgF0u4Hqj\\\\\\\/FVTKYoLAoLhDUXTZ8pPQ9w+EjhKmUwl0tjarpWKuNRFpUoonzdqoaSXnxfutYLnIL15dzKTuc9w844XXf\\\\\\\/4fIgmqKpW9zjt2gGxcUB3vAM2Y9aG+8Glv0iLJ5MfvNTuS888qVmgtif27H4ZpqqNAaBmnznhFs73awXbv20GXCIxYsv4agu3xLgHebA9JJI1f2oWkxLRr3FqTkfZgsbEue+y+HB2eTPpEftJvQbs\\\\\\\/2iv3jvPrMYIQwla6OjDEwVBQD\\\\\\\/\\\\\\\/kRZavl7BVSZ43e9CIDp3Ce9FhJgg\\\\\\\/6NqNzrctk+6omuYLj8NTIZvLHsrFkHSVOs7d5u+DvBWm\\\\\\\/Lh3yBPVB9c+RvyCFQuf31dsEGjqdpwNs1Y68MRuUpvV+1dTpX2t+dYJzHtMMXfS+lnCmSgIH6Tmj5nYMAROuxsJ4urP4LGHoaTHkfuzStWNpOeAxInWwMhGyauouJzmyb1Ks4tvXx4X2D3x9QLLhI87LDDTMg9AcvBMvZ4tEGo0th1vA1d34XeztCfyKJH6JdVPyRhvVx9cQ8g1W2aUmA+1eeP7AtHOSQAhxIvGcHIzjk7xYImNPqWnBQMX4Hm7FDi0ntpkFhzRDv\\\\\\\/v3xNKvLSVmKWaVoKO4u3WWCw0l0+uT1zM+2qnzKg5Evqk3fkHH132\\\\\\\/8+craqw5Wd8FOULa3msksk9pdaje77VGCJ4gtfqveQUvpAJG0Y0xRNG2xp5ZOVobFhdKwFvdnk58A57tZS0PiRfHRrPgDProQvMnSMsD0fO\\\\\\\/bKBGgx+5bi3xEv1xgoOupn7RgdNqjrc2q0KkOGZqHeGmREakxe8zgSBH96E38O9eC0cb8\\\\\\\/4L8nHEvs1oJ+AbSxT2weDPQaANP62Z5FuSTQL0Yy4Od9iNwFt0iWHPFYRH4ENR69j\\\\\\\/I+xwwYmXld6qHgH0kx\\\\\\\/RfSszBbVQ7lJewBPTi6KHARoC8JNk+B5S21bvLYlcIksp8P3oTu70CMBx+eY843IqtRAVOV\\\\\\\/7MdFnC28DRnqhlnjzG6YZGhde96cZ2gxK4gZD46y0xQhd12+DuLmO6H7F3IYmyD+wMbiZP0lDPKgjzAoJ8Jvot\\\\\\\/BKJW\\\\\\\/D6MxejZLT6tz3SXKq4ys5YITS4NVowNm888C0WyoW0zoK0gVPqXkI35prRq4ySpPDFfDH\\\\\\\/t6gR\\\\\\\/NdI4ZqwvvKujrF2UNXRq+wbhd8vm8u2K6hiQp6WCythIa3sN2iGfqm16NxXcUfdzGtFxdSR6Jz5HAdhutkAlPGUelN03mgOH\\\\\\\/2w9Kqr6WURJ3bONYD2cIwuO8HoULJz6zPuVcXkQCi+xBKldOZWzoo4gYBr31f\\\\\\\/TiJDM3gEd4SrZa+b+hkTL1WMnGFu2r3WVJsNsARX3bXw+E6EDqKe0Pgrgw+BLC9bGV5IWRf7EM1XJ98wj8rJ74qtbLHjhILGo9Raoom27VGZlXbaH9oCeW3JV9ME7gQ5jv8TDjnZeJ7NEQfDOk3qJgJ0vhD1gTz83u\\\\\\\/ajzJobRjwAhk8kl4g\\\\\\\/0XFkz5PCkU4WkLdsMet9DfCa+Y8fOPV4DKKbJ6OW+zdzGdOyg9T5cZsknu5J40VrQ0nInj1j7qt+SJcM6h2HliI7+gCud2Ah28PmKGodeR0uAtn0Kk7a4\\\\\\\/39PHDegl5\\\\\\\/Zuv885JcKnyFMMvpAZuJoxnp2N5\\\\\\\/MqoUQf+1kPJHjqTlCU4gC74HBPDu6KhpF3yRvOjJXkgR5LN\\\\\\\/dcKaPGxljenqNFUN599jZf+HPaAqXGovyDda16466\\\\\\\/CTFxs+Mn1xPYaz0qd9E2Y\\\\\\\/7rJlOM5zm\\\\\\\/9B6kV+tLgcWc50iGyjNba+vsUkMn8xC1hNai3Da2IpKB8x2FdsznuCCMZgS+gBAi+Zwb+mc9dpzH1jzKHr+36+52BSR6fFaWM7z0Ala0BCJoqM+MOsEbfGBpqgO5RXUqp9XTyXRMW0+lKH4cAy57aU021d+i9oZEYsMVgAxE9jQv+fHYdxGIScyZYxu9SNMpy8aM68e1vf5kz7piupVBNOpvH52JozfF69UoUHCcHdWBI2GHSEQb+Q7zK3jsOQyiO+sm6vlq72Itet2909wdl6JIum8MgAyOkF+Hc1j0Tpt6RCmIhscL9zZsfI6LuL5RLqlNDduYygfgcbpm0z663BKA3X6iy2PYRKsS5Sxh9CjcNugjGdqy7AH0vq7UO9eU5rfZKhktt7HQa30k1OzL5qXI5o52Zt\\\\\\\/LsJCQampqEYIc0+DfxDXRKpmfZTWsnHDDquGpGjK1YiJ7IzDxhqH7WkG+fpEoNzutwxf5PaHY30Sb2zc0zeFqfEgPx\\\\\\\/ctQ4SqKZ35qmgtoDplWQxIZy0POhgHmW4DmfEc19bIIRn\\\\\\\/LtDV8yOZNoiZCX1B9wGrUdFGqq0jO7dD2b7mdFPXldEA9meaBbL36+NNN0YuMFfAg0SnBHcHhgjmMYc7B7M\\\\\\\/LBZZrj5tNjKspVHyCT6+C\\\\\\\/powbpVHZJLc+4PY\\\\\\\/s79WmCIQHpibwibvxGDBHAR\\\\\\\/uGq6Y6x8fSR2RSsFI0nhAkNwzKsbB8ArVvE\\\\\\\/YM659oixekbZjPw9qwDBA3b4+zZ0waJQIcWwcuHdRqbQsDBYZ3puBIF\\\\\\\/SmcPHqZ5femQTt27\\\\\\\/N7iS+xheoCx8DQBLHqRZ+jQOE7iHBSE0yVEaAu1gW60mq9\\\\\\\/S1Y\\\\\\\/kd6V4n9uI+wEWscNRSK4yLt+f1aeHyPbewcbZ1XdddyXTjqvJnJKSCFAcKXE22NsDzHqNOSJQMycW5HoyYtekH+ZmkBcOJGXWu1CUzE+JgN6u2wSJEHhEAzeL+Uv9OuYAKw+g5HiruTSm3O9feKOjMGBxBjcpT69Au1my80LDXozPfafby1VQFP+WgOUPSs7LlyIJ7T6m8FLXW8EL\\\\\\\/2bf3Nqqq55tdAnocqIHzOn1Kc4AprVffJCL66YQdCXBDxglCuFkwiqLQPueMkXJnfrl0JP2Pj6Mxg6rFjA0SQZ64zRGTdVp3yDdkKGGjiZ3+lZ+2PGZlp1BISUa3G1llJ8+4TVEc24WdrZztkdmhtvbHfTQ67J90TEt8A4tWmq\\\\\\\/WEWcWyjoqMaj9qD1z58cq6zhrvukoq8tN1hK7wwX960DR27XDiOHIFxTgbkiXlk4MxiKTRHsv7CpzncQKcKCi6ULl\\\\\\\/FBL7fSxiMknF5X3ICjFNha\\\\\\\/\\\\\\\/Rp2SWVHPTgtvHjFakvOQC4rxcQH7SuGvDe7ELDOnXQrC4DESv6tPTRQjffimCkxTCFPnYgfKB8tLrUlpQtyBlZVcgfwK4Yc2ZcP\\\\\\\/CWi1RFDakVa4EDvIOaN0tyWB1wJM0UscW8TpGGGFRPI+EY1U8ZFjaNXWNuEex1YPFYCPrHJonbQ2pZUGAyeyp4bbPELjEptr9fTNNqNAS5PeBkceqvaArsft+WAaqIs\\\\\\\/BHm53kbGDAQrBF0XsqurcsJ8927mxk9S2mVlKghorAQD9\\\\\\\/elIM3K3UULGbekDwXaEN\\\\\\\/Woo4WeSb0lb5xBzrRW4rFCovCH4gM\\\\\\\/tJk487rWkQIgkPOsEUONhoZe1uoEr2LkUf5LdOhqijGy6HhKKZILT\\\\\\\/CDpAVl5Yg5yDYqaOAzf0blByu4LeIFrtEUxnFecaOV2\\\\\\\/AP8GTmuKOdbDBBzD00\\\\\\\/vo6V\\\\\\\/HXvQ3SarC0rjzPY59yQ4l1ZOLqYsdpANzBPx4HMEo6zxYqr4k+odgIUXAtUJIxw68tYT9YVAunk6XUPo7QBeqfiI8McBo2prw1o+\\\\\\\/Jwi87vfwrnLdZpfGmivbOKk+\\\\\\\/HrlR2IwJoTMnLHbZsV+MXS5Kd3w3wDh2XZgG18u2P0UfIrZ7KDmtOJMB6+2VrPnFUmqvxrD\\\\\\\/FKB2EnEt\\\\\\\/vhXNW7yIooILaR5pb4YgYYaP2VvPr2pdwhjUuPmABCw0t31AeY0QTHa6XpERiYhp6qIGIUZT\\\\\\\/6UDXns6KhEGCLTEVNtjBPlkkf2F7GoJSMvBh9vebN+72NGptZBTttsyPu5LijemMjo4v3DUcykAM+IG7rY5JavMaUfda4TdPzk0HZ4LWrDqCNhvmVbBUjSTjm4vlDT34Eb0bj2ch6J7hm+3HJBEkQO53IsbVRxNxwY0qyHGwcE3LvhNoHe7kbrbQxGkY5TJrxSMbZ7GIUSr+uXrwiFZ2gxGc\\\\\\\/zSsDeixMhzrxXhGjpbe2zOL3OGsu02k5k3LUdeZbTa1Jl9NCBfnz74TKJMOJIxXoM+5gkgPXXa5JyK6KmQ2AoKM\\\\\\\/0Q5Huw4ma1ltPfMHtWWeK4bRkr32Lv5bXwxlqoSsfy3L4+e9X0GQPZHhWq7UfXGWXOUEYy3FFcxAw6utC6pmDmU4xCfdmmn41fzcV75xsyDGWtIevNTfJzogeSygVPHOpM9JsgXbqMYvYGHXPmZVxrqtN7Ra\\\\\\\/jCLuGHPoyLTm5\\\\\\\/W4xpuovd4E8WxQkXQWGz3pUU5KKUUDARdt\\\\\\\/3o3V0Bi5iuAD8O7+2Nj1u+2AFsbrgRXxI7R\\\\\\\/nY3dvvrzSoh+UMX7IFPNVhCwLM1F1CsXEBhfOIXSUqIrP3kUVSe8jo6bWIcnNTbXD+wvOW3qyws41xyJbXBm6TFPQb8nf139GzPECeuMEL\\\\\\\/idyvNbBIPJQSJ7p0cmTswkcG+ZN1r98Mrmcv2fpAgDCgdDS7QSpKnCen3PhjeEE\\\\\\\/lvutCXW8hlSYitvY302kS1muPG2Mid0C+erq1FkDFzlHDMjhYXtt\\\\\\\/oveR37DM4NYa4SEuHXD1AQFW1ylEdfR5VF+DfPa15yD6bBYrDYdPj4PuLt2MdmeQQvlWXl4sBd9gWEZANKZxW+1f9QrCKbuk+CTEEIsMqQw7FBG\\\\\\\/aEXG0wDu+AkGYl8V8fncG\\\\\\\/TALfdtET3ISWZeRH+l1BomN3Sr0NHLWx7jutVencc9t1yg8v\\\\\\\/m1T7xpj2TAOTaNCre8i+onwm0GRIUpQr+gNlHyRFux54qjKagcQdavZCpoST7m8rgHeQCSOLh20T275ajNjfz0BotUxuaeHcqzKUdE\\\\\\\/W4m4cPFyDTskknH7xNqzz6fny\\\\\\\/j4A4iaspsKOzSN621W6bbsFCu+CkevJcGcJwGFjre8ULhFdd222EL5h6IAGcjIOl2MGsoBWH0B6lTlYYcBLSiY+OfOowWH\\\\\\\/JLSLOEZ7khziX5KA4wfkof1GYhe2vSn5iD4Bd+rjk4MbM9YlFCG5j2AAm68oppfnN9OP0CHIHy5uaAKiaHimh0D\\\\\\\/ICEtfCCnp+ZXhSRBUx0dpz5M94zJNrhONJq12InG1qyiiW823k++mGUut5XLpKQ6U1Rz2P6jIuzcgKoZsm9RCq+mI0uB7nPH1nP2CIMckeYwKTZ0T9i4Sv03LwrCB2BBUCIfjmdkh2ftMTAeNJSNUm6ad3FAndz3rAFRFWZjMO6hdmq7JymolXjkQqTeqmg4U5EJ1ROC+SULmm4ghTBoFtErteroQTsC6XYIlxeJRgBaLAi\\\\\\\/mv7nAT9N7g0+lioxfsQPzjzhixuvF9dC8adDT4t34l294N5jrT0AWgarlf1oEoVkbpRiTkylDLa7k7903ZF7U4VlN9HugI1X70aidgcWS37uWUpU+jZsf\\\\\\\/PM887Cfx7kP3J4Pi4VY1KIpLSqFJFIaumtTfp+fItq1uec2LPrviEK5mQ3CaC0jovMK0ZEwDh5cPrFrOiYyocffCMOhCcadIJnpuEpTcPfzbj0rKqNnmLUcdfujwmrw64yCeo1Jcb4UKPBJ4\\\\\\\/ERvVhqBjV88qVFxdHHtxbCvf1i8C6bW1V2UDDvgB\\\\\\\/5LRqy+DG10iqZDgcyVjFK\\\\\\\/479sf7UHqZ99Ci1x+UphnmRal4G329felq9AUNQeysynoDTfjnt+gQfizkyxDVGe0WWbsbOId0HCmDKuVsdC7t9P18lLixzlWeB7NvdC5RBpjRovdt\\\\\\\/OGe0jkq6OwHTLi+9uFvvsIhf+U83Q9uSn\\\\\\\/vBjWM1ykYAUadaNTYPRxe+W9KdcUVmQsubVa8yxgbtDjQjAxV0yr3gChJu+MZe2bDYgLrdGSLajYY91LWgrK\\\\\\\/c4kbGtT\\\\\\\/Gu3TgO6KSLGGYPcTPOv6bRLbOJXF3xXY0gRiv2vx3aaEyLdtysWXKIfqgcQ6ohaLLqZ7LGdF4UEEFlpVg6LCtc+lb95ltd3674J6zyIqPdrRM\\\\\\\/hQdJ8RS+UfhV2vm76IX+z+9wM75UpjqqDiRGI0oMKihqNiel30zCkJI\\\\\\\/7YbQLhU6lT4cqoxAIRSvRvaHz7Te3rDmCTVJQT9aoC2kPEWDJq0GC6sbkrdNxNrnB\\\\\\\/yp0CshNeLqFdfyXJJoi6ZqhfVW5HzakH7UnpoeQk7KFh1fRK2wa4gyjUah3g\\\\\\\/tf7WePsPxRIctxVAqORyS3wA6j1pEhWBgcyiXJVzm0FZPir0E03Lqp76Zqt1lQW79RBv+mFKoR1+4wffk0s+DYH6KLY5qyjzgHN26maVN4uO3Iz4cC+IzSXvFULANMxkdXmqAroClFxWVI1OVaXOBzYQihNR9R2nJ7CagfV0ZxDNUVqry3O83bD1gsPl03LFXEsKw4BKE0ys9ibXQjL5H8tCS3pvgEe4uUBoEPfTLNW9P1qr+IRIF7GOXCO6PY1xgt6zSIJDQeXT055S1R3XEQE7amIdoUVvtQMWYRr9UKZWeN\\\\\\\/OJuYGmhm\\\\\\\/0XIoFTNwgWNt1Y64NofWW85UoAlmJiAJRd15AL0LGiyj0c7zdYiwi3dpovWD03gOPFbfmB36BBhSwZtUDy2nYyTSRx0M7IH6RGVQUK9WEj3TFQkQdyplIqMIlDjL01uM7PI1Gwp0eJ3dcCtzQ1QRMXt5BXQX0TyTnhG331adejBpBo4RniNNblnzk8dWFVqqQCNqDRFZkslgGaVbr2l\\\\\\\/DePrXPeptn7J7E48VNKvEOaKOPsMLUL7mpXVj4TxTlUnCxD8U1NZcA5dv\\\\\\\/5i7biY1qTcgg7VuI+k8myN41LcPJTVyo89uTNKM+VRDBrp3XL\\\\\\\/4+V+gf7464gDpvLElPOmdyTIC5xRm2nbFT1a5DrbCNqFyI1\\\\\\\/leaA1uOrXVhHNZoPT+qIrAK+Sa9IlO3vbIrM0wuXanKdmpaakCZws0SKRgeepp1fY+3KnMuk4R8zqmwSMwrmFe+tLrH2TF32K7OajWW8VO\\\\\\\/Czpl\\\\\\\/2Lkyqu7wc3diEvEega1jg7E00\\\\\\\/pJT6EVXIy5dv6FfsjLZCH\\\\\\\/4HMVpUN0D28VpugjL1kZ0cq94OL7B4W4aIJOekv6KZs53ujgWCEQM18YTHIRXjQHXcESTHnt4c3jTbKr8e18OLqUdrSV8v4Eyzsf\\\\\\\/iZ8AvR2Qw3MW6a2twKtITM58jHlMDQh+5nC8HLkUO2JO9nGLcnko\\\\\\\/Dd7naioKViYbtcarYjpefaEppjR3qdHtqlJUd3lphu1X+Sw6hvWXaod153EZQSkvNfAop69aLlbhbK5+X8Zauo8L0Wk9sK6pCwB1AAC2foDs69\\\\\\\/DiGmoMdMpwGavFl1sCgTnNFYl7HcatfnwZ\\\\\\\/i4myjPEMcdM0GxP6k4LnRG1ALxPWQ6z4A5nBRoFuZ9CNr8PWCbYWnX\\\\\\\/0ytsnrbbqJPMfLSFY0wICioz1cHfNHSg4LJ\\\\\\\/tBr0\\\\\\\/G3DjW059vyg8v9Mg7xnfRdT6iAgLu61vEOIWeJ6nJllX3UQnt8airUrhQggHX0MIMTjMcKB7+vIJQha5ByDjLl1ROR1\\\\\\\/eeCpW5e1lr0HcJ6enPeQHckQfqNzu6+S9Tnsx48LPL1W+eq5r3kprCS8UYGGdld5wAgO0Fsa5YZ+WsGTAfl5o93kRp+FjQ+o\\\\\\\/NdHXr1uZiLcV81IkC+HX8WWxmQH7vSgWlEqTfVUR0hU5XJ7A8ghpJ\\\\\\\/E8gVyk0d6d9uMFR\\\\\\\/NhM\\\\\\\/XGq5LbAKoi1vvaPJpv7IP1olv+gVFhhpL0EdC8NBsHzXvQ6IFhTKpl+sJ70e7IzVY1qNjUmYOMHBoK\\\\\\\/uAr3B37RzwQFOPyU\\\\\\\/hZHD3H6BV5CJHPKbeBS\\\\\\\/Tf11pqt\\\\\\\/1HLRWCI8rA\\\\\\\/KJxwICpFjboNm0eyiX\\\\\\\/XUYl5pKr+6O653YOnRYDCmVnMf7r0YUzAbHbZATmY6gZ5v9wxKVAbyOW6NKBCvDD\\\\\\\/3FEwQQPf9lTVx1iviPRZ\\\\\\\/ZipBw\\\\\\\/dQ1fzos2+\\\\\\\/QJJhYUr5tvGn2SAHZjlpP5h1iSA2UFZ9M4GIykLGCPEZIuE5PHdLn1paR1AjGFuxWhJZzleocUqDlBropbab+x1ZE7Y5Ym3S0SpL\\\\\\\/v4ooUh2gG+KD1ZECfujyLDY3K09WPmzVy6V4micfn\\\\\\\/hm5e5nQrVzCCSs3jqbCEUgZ8k57uRLe0STH9sMCIjeJaFDq9lnx1opsVZj0jtq\\\\\\\/lYEdE4N6Vrpda9Ze9KgdSgWLC3ZF74CgT2sA\\\\\\\/04jXsgep3AS45Wsk3LtcKFY51ln8yrQNxh2OMopf1yUdI8Tfo9tAs+admPehFDFC91XZxdTgQ3P+lMYh4LZUBHYGhlzg3e9Qxvo9Tsn9JTVSNLgEuLiC0z+FSbvLhBRv8iAl5vpvmlIBcw2Qpdcjj0J2gNIfnks4nVBYXMB6aG+bsjERKROLpa5KkU0FAilflPCQHiAq6RskghRYOM7pKjNTEMRmcyFSbd32nf6hE1zAkGThCXlqDXCm\\\\\\\/Gll8jWgktADnynkb03wbCB9iNRDzvz7kHdPRl5LUpuaPpmry7shFFyqQ2GYcb0LtE\\\\\\\/FYW8jG3IX1+apJAYt+pF0UBuePSGGouGbwRwJZbRY0imBzDpyu+Sazjh+7HbER+hYj1nXND57aRlG85QOseSd7cKXSS8lvRrVI1RW+hP3ZmVj0aN+gJawm2Ip83LUSjV28e12005qypnLjeqOcAuUjEAyH\\\\\\\/TyGFozKfJhYYjRSL4VLnX7lYggiL42qx6njFUSbzRAXKzZ8of1Fbf3nx4vELA6prwaKnYCwx6yk6jkA2tacWAwIJyyxObeaPDq6d3Ta3zGu1J\\\\\\\/eQ\\\\\\\/etJ6rOVzqaMz3WQgnPZG5srHcZ1gcbonnL1R6bmMFNWSUb+368JHoDffvVnC0EwKzO9vJPn8u\\\\\\\/3nD0Rq6QDxvjYAhpqCN0vtAuf4zZKEUyo5pbSIPg46PKy2aCSd1hgtHunYV6xzj4n\\\\\\\/17\\\\\\\/J1pAe3u674pyco\\\\\\\/qKOfyTxkjeHzNk8Zgt6MZv7JeG87TnRUFQP4QzJHXXvz8fqaNXefxYAwwIWVA1lFi1Za7I7BLVq4T1ILFyZfSooirV1AvhE4rPYfiKZ+tc7Jp7ioQrG8zY8\\\\\\\/C4ZeYcfqcWgohp2qNYpdFixwgsgM4AjhK9kW7ZeR5k9jeSBMNhFLM4Bx5G4xccnyBn7FViAkY+6T4CqicL0YBuJ0TyhVajzPg91H6UwJkJnM7R+xcvUIxuQCTj+X7zG+toJRONENQidmRB6QP+UyZfll3GiUdvOw4XyStA3kqLiTyucSvfqD8NqYshvIPktAK7SbWq8nc6TaQ3reHlevR0A+BSvNfKWDKmeOjfqhn8I53dds1Pu7c9VZLCwI74adLw+fjtADXwvtd3bQ\\\\\\\/QYZPFJIGd38jecdsQdyYtVNDZ5V\\\\\\\/LJPubK4pC6CSoU4QD1TC3pN0vd5YkAz0WC2OmZAWorOomz5BumlmOddw7M9kLxmm7gjBYmukkarcDb5AAQ9tURsZdkMrycsIh5++OlpKLdYU\\\\\\\/iwaYX8iECfGCccRlR27w4uDq5HQjlhkx9\\\\\\\/N5zB9Jol4p3vo\\\\\\\/5bdsSQdOM6p33Zsdzkn94PDPqhMb\\\\\\\/paHTPLvLFU8k2NG4aIctTAznEcZaIFLB2RTfmDB8BPLAGABxThhxsOWNCLvcwHGZaX357kLddTcF\\\\\\\/9C+fPiJUbr+9ThWu0SiS3wSHHe1rxB\\\\\\\/Ig==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ljrY8LL1GRN19weVEYAXvw==\\\u0022,\\\u0022salt\\\u0022:\\\u00222be6b58ce6e14e3c3058432947d8ecf7\\\u0022}\u0022,\u0022hmac\u0022:\u0022f96526d94de2fac4cf1a53e0255672e905e3a7eab3dda18dea0897bdf32a6d11\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v0TeCALwpAp8Bbgpyjo95E4iEUotHg6az1a+15F8Pf4V5dpGlz6ZXhIJLExAtqB2acqziNN060mUCsKV9mkqOHSciGYkAEyRXGbq5BVjT+oIovs6lNNSLhMNKm38ZMYuM7K\\\\\\\/PC\\\\\\\/wKh2yJYTsKgPPqYXQRHqZIszpGjCVE\\\\\\\/H271s5F+8VqzwCMvL3bBbtld1ndfQO7\\\\\\\/GKKfna5S8sYJXFkb8giKwJmqlUIURw9BzGAKnuViZDPxb99e5+X5v7KiHP73Edo2a3q5QhfJ4xOQ3Q0ra+Vn6ocezsmhurXvmvC\\\\\\\/\\\\\\\/fm0SLatlSeoovFknemB5yiIq5Mh6JVXyqDiL3Um0+WWbounzAFVZPhC65GqbNpHO6W2GEDhNZkcmk9T4FgtyQWDBF46aBA1nos94+WoeofXUoIV8mxpyuLMRsCLrICWCutHmZu7YWuHZCBmUK53PaHRbtV5KqFAl+Kz\\\\\\\/5bo\\\\\\\/iBBacxI8nQmozIKx+msle1Vr34wJv1SUucROP9gV8q7J4ad6vi\\\\\\\/de3IjHFgXum6r5sR+gOZA9kUDuAjBkgwGZs\\\\\\\/HEfdG+jlUs3IDZ4Og5y2UBp+7YWWYmyfLFyIiPPWoNpeAnwG2p\\\\\\\/0oyH16lQI8dzfgRySm7Vojq+GX\\\\\\\/N5Gdye+di6k3HzGENiBF27FMBH9+RJtW0jugApSbi98ohImIpdS8P+JYVZlR7e8DBkgQZOUgFNiK+p3f6LHO3p0GneSi1kfsLBBPBN7zK7DZ0JNG4mEg+mLUn9olcEvkgAVIp2sICEUXE2a0OMjkpfJHYJUGBWoelVNBX2PICjN0OMJXsOl8ejrAZtcYTQrU5mFMhho5UZSwLKVybsfOn0zhFVhWO\\\\\\\/P7XYlF12oQJYROS9QuYwEf7v1ya5wNYVqs5IEgToOmUE2PTl0AcMyq\\\\\\\/phcnOLRrCYuA0OOElnlu73N5Kqk3wLYG38rRIjM8UVt5in+89XmrKsvswwYIXuZCdyN6dF7tqEXr5PQhXL4YlKNKnES4cHca8SYfHQ1VFLHewCRN77ZdrMxeiY3WT3PGK47KPf2WmFUGryunyeO8OWalzzg7PKCyWqerHY\\\\\\\/dsixUtzXFmTUTvgDaEIQQzl3aZ+p7EWb13IjNBL9aUOBsfQZetFkEwuVZ2iqkDG1BFOxIH67OaSYjpxBRWZjnyxXcC7fLtIdY64MHHwjYDTDdVbMik0hzGSR32VnmX4L+A6JhLnkgpaKIddRETcurJLIjvlpU5kZ\\\\\\\/kr37CJYzfclhx3hCkMPw+Z3xtXr+N5\\\\\\\/hhfI4aq5mNdatwPJSwKhSnU0wz4wkzOgL0TNv3i1elRzqeVGd4\\\\\\\/dlw\\\\\\\/cnqsm\\\\\\\/KwLY\\\\\\\/fyHdUGYIVBGfqeRKbkHzSSfWUfRzuArEHpdoHKm0BKDtec5pozZsGUmxlZ\\\\\\\/XfjG3fH0OWtx4apWkkwMHd6mmJ48oyxg0sJuRQ8dksMcSFNisUfWPrOrObtvyLbbbfhhcM4KcXMvSTHVYJHh61pvoYaQOwqTdbVUql38yeq3xH7tYwH6SVjOMNc4x0rD6g\\\\\\\/ZK2CxMyr\\\\\\\/h7dcW5l420+Yv2fTV7rBN8tfp5EVlpti54iKY2yJzUwXkoyTdIbe+6tBLNZlWhdI0B7RS+T095MU4VTa6DQgYaFUpzm2who4S6y9U\\\\\\\/E\\\\\\\/hly5mOf9W80dOWdndSzkF6++x3yRjw\\\\\\\/kvuiftRXINfjmiFCd+MtH6GHCZ6FFPp55IwIN4qsNfzyjLHaP9tS+KfwEctBkCC27osLZtHqttYplf5thJjjx0cLdGU2OKzKu4HcM1aKDvr1I6onOrsJjzucno5Vjl9K\\\\\\\/YAXmI1Y4Tc6FwBJNK3rpBl96aSlArErVDihlsRCyBuzeNRYi6G1NEJl0LeJZ9m2+LVm0q2RIvGlD1F7z7Ubu0wWf2Z5EWHRPemHtCGduxaLrAHmzxwfN3QAywwc3j4fzU7\\\\\\\/zhstejYVRtHxh\\\\\\\/ae+Z0+Ct\\\\\\\/tJzTcxMtSRma7usXPgFtz21Yge9f2Zi4SZsDUR2QY7GimqNi8TGwZIw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022LXFuOw7ozPj5VllByGOazA==\\\u0022,\\\u0022salt\\\u0022:\\\u00222be6b58ce6e14e3c3058432947d8ecf7\\\u0022}\u0022,\u0022hmac\u0022:\u0022d613247e1c39e08b7dc3207f04bdbab4161ce993e944111645d0d7f93be783a0\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  <script type="text/javascript" charset="utf-8" async="" src="groveMainModule.js"></script><link id="grove-annotator-css" type="text/css" rel="stylesheet" href="https://remarqable.com/main/annotator.css"><link id="grove-main-css" type="text/css" rel="stylesheet" href="https://remarqable.com/main/grove_main.css"><link id="grove-main-2" type="text/css" rel="stylesheet" href="https://remarqable.com/main/grove_main_vars.css"><link id="grove-roboto-css" rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic"><style id="annotator-dynamic-style">.annotator-adder, .annotator-outer, .annotator-notice {
  z-index: 100019;
}
.annotator-filter {
  z-index: 100009;
}</style></head>
  <body class="html not-front not-logged-in page-node page-node- page-node-334013 node-type-highwire-article context-content hw-default-jcode-genetics hw-article-type-research-article omega-mediaqueries-processed omega-equalheights-processed hw-responsive-layout-normal responsive-layout-normal"><div id="omega-media-query-dummy"><style media="all">#omega-media-query-dummy { position: relative; z-index: -1; }</style><!--[if (lt IE 9)&(!IEMobile)]><style media="all">#omega-media-query-dummy { z-index: 1; }</style><![endif]--><style media="all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)">#omega-media-query-dummy { z-index: 0; }</style><style media="all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)">#omega-media-query-dummy { z-index: 1; }</style><style media="all and (min-width: 1220px)">#omega-media-query-dummy { z-index: 2; }</style></div>
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-MFS4DXP" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-MFS4DXP');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first" style="height: 79px;">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="https://www.genetics.org/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-g3journalorg  even " role="menuitem"><a href="http://www.g3journal.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">G3: Genes | Genomes | Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links">
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated">
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="https://www.genetics.org/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo">
  
      
  
  <div class="pane-content">
    <a href="https://www.genetics.org/" class="" data-icon-position="" data-hide-link-title="0"><img src="GENturq.png" alt="Genetics" title="Genetics"></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><ul class="highwire-responsive-toggle-triggers"><li class="trigger-0"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></li><li class="trigger-1"><i class="icon-gear"></i><span class="element-invisible">User menu</span></li><li class="trigger-2"><i class="icon-search"></i><span class="element-invisible">Search</span></li></ul><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu">
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu">
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="https://www.genetics.org/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="https://www.genetics.org/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first menu-mlid-663" role="menuitem"><a href="https://www.genetics.org/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="https://www.genetics.org/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="https://www.genetics.org/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="https://www.genetics.org/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="https://www.genetics.org/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="https://www.genetics.org/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="https://www.genetics.org/content/editorial-board-section" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="leaf menu-mlid-1397" role="menuitem"><a href="http://www.genetics.org/content/early-career-reviewers" data-hide-link-title="0" class="" data-icon-position="">Early Career Reviewers</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="https://www.genetics.org/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="https://www.genetics.org/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1404" role="menuitem"><a href="http://www.genetics.org/content/gsa-journals-series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1224" role="menuitem"><a href="http://www.genetics.org/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="leaf menu-mlid-899" role="menuitem"><a href="https://www.genetics.org/content/multiparental_populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
<li class="leaf menu-mlid-801" role="menuitem"><a href="https://www.genetics.org/content/flybook" data-hide-link-title="0" class="" data-icon-position="">FlyBook</a></li>
<li class="leaf menu-mlid-1187" role="menuitem"><a href="http://www.genetics.org/content/wormbook" data-hide-link-title="0" class="" data-icon-position="">WormBook</a></li>
<li class="last leaf menu-mlid-803" role="menuitem"><a href="https://www.genetics.org/content/yeastbook" data-hide-link-title="0" class="" data-icon-position="">YeastBook</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1245" role="menuitem"><a href="http://www.genetics.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1246" role="menuitem"><a href="http://www.genetics.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1228" role="menuitem"><a href="http://www.genetics.org/collection/commentary" data-hide-link-title="0" class="" data-icon-position="">Commentaries</a></li>
<li class="leaf menu-mlid-811" role="menuitem"><a href="https://www.genetics.org/category/editorial" data-hide-link-title="0" class="" data-icon-position="">Editorials</a></li>
<li class="leaf menu-mlid-1195" role="menuitem"><a href="http://www.genetics.org/collection/gsa-honors-and-awards" data-hide-link-title="0" class="" data-icon-position="">GSA Honors and Awards</a></li>
<li class="leaf menu-mlid-1222" role="menuitem"><a href="http://www.genetics.org/collection/methods-technology-resources" data-hide-link-title="0" class="" data-icon-position="">Methods, Technology &amp; Resources</a></li>
<li class="leaf menu-mlid-895" role="menuitem"><a href="https://www.genetics.org/category/perspectives" data-hide-link-title="0" class="" data-icon-position="">Perspectives</a></li>
<li class="leaf menu-mlid-1221" role="menuitem"><a href="http://www.genetics.org/collection/primer" data-hide-link-title="0" class="" data-icon-position="">Primers</a></li>
<li class="leaf menu-mlid-1220" role="menuitem"><a href="http://www.genetics.org/collection/reviews" data-hide-link-title="0" class="" data-icon-position="">Reviews</a></li>
<li class="last leaf menu-mlid-1227" role="menuitem"><a href="http://www.genetics.org/collection/genetic-toolbox-review" data-hide-link-title="0" class="" data-icon-position="">Toolbox Reviews</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="https://www.genetics.org/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="https://www.genetics.org/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="https://www.genetics.org/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="https://www.genetics.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="https://www.genetics.org/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="https://www.genetics.org/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="https://www.genetics.org/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="https://www.genetics.org/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="https://www.genetics.org/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-891" role="menuitem"><a href="https://www.genetics.org/content/why-subscribe" data-hide-link-title="0" class="" data-icon-position="">Why subscribe?</a></li>
<li class="leaf menu-mlid-794" role="menuitem"><a href="https://www.genetics.org/content/institutional" data-hide-link-title="0" class="" data-icon-position="">For institutions</a></li>
<li class="leaf menu-mlid-795" role="menuitem"><a href="https://www.genetics.org/content/individual-subscriptions" data-hide-link-title="0" class="" data-icon-position="">For individuals</a></li>
<li class="leaf menu-mlid-679" role="menuitem"><a href="https://www.genetics.org/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="https://www.genetics.org/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu">
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="https://www.genetics.org/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.g3journal.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">G3: Genes | Genomes | Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu">
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities">
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message highwire-user-message-once-processed" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search">
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext">
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/177/2/861" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." id="edit-keywords" name="keywords" size="60" maxlength="128" class="form-text" type="text">
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" id="edit-submit--5" name="op" value="Search" class="form-submit" type="submit"></div><input name="form_build_id" value="form-boRBXoHuJvu5Dqqzg5T4pX12VeFvjpvIhU6SbQiuzmI" type="hidden">
<input name="form_id" value="highwire_search_qsearch_nocontext_form" type="hidden">
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree">
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="https://www.genetics.org/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="https://www.genetics.org/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="Genetics" src="GENturq_0.png"></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom">
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/177/2/861" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" id="edit-keywords--2" name="keywords" size="60" maxlength="128" class="form-text" type="text">
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" id="edit-submit--6" name="op" value="Search" class="form-submit" type="submit"></div><input name="form_build_id" value="form-1kZtyX3IReunxRkR8JuN5IjrKDVA8hCNeHLCWFq_e4Y" type="hidden">
<input name="form_id" value="highwire_search_quicksearch_form_0" type="hidden">
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2">
  
      
  
  <div class="pane-content">
    <p><a href="https://www.genetics.org/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="https://www.genetics.org/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even" role="menuitem"><a href="https://www.genetics.org/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="https://www.genetics.org/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="https://www.genetics.org/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="https://www.genetics.org/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="https://www.genetics.org/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT</a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="https://www.genetics.org/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="https://www.genetics.org/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-429231  odd " role="menuitem"><a href="https://www.genetics.org/content/editorial-board-section" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-1397 menu-path-geneticsorg-content-early-career-reviewers  even " role="menuitem"><a href="http://www.genetics.org/content/early-career-reviewers" data-hide-link-title="0" class="" data-icon-position="">Early Career Reviewers</a></li>
<li class="menu-672 menu-path-node-10  odd last" role="menuitem"><a href="https://www.genetics.org/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="https://www.genetics.org/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1404 menu-path-geneticsorg-content-gsa-journals-series first odd " role="menuitem"><a href="http://www.genetics.org/content/gsa-journals-series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1224 menu-path-geneticsorg-content-genomic-prediction  even " role="menuitem"><a href="http://www.genetics.org/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-899 menu-path-node-433204  odd " role="menuitem"><a href="https://www.genetics.org/content/multiparental_populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
<li class="menu-801 menu-path-node-429330  even " role="menuitem"><a href="https://www.genetics.org/content/flybook" data-hide-link-title="0" class="" data-icon-position="">FlyBook</a></li>
<li class="menu-1187 menu-path-geneticsorg-content-wormbook  odd " role="menuitem"><a href="http://www.genetics.org/content/wormbook" data-hide-link-title="0" class="" data-icon-position="">WormBook</a></li>
<li class="menu-803 menu-path-node-429331  even last" role="menuitem"><a href="https://www.genetics.org/content/yeastbook" data-hide-link-title="0" class="" data-icon-position="">YeastBook</a></li>
</ul></li>
<li class="menu-1245 menuparent  menu-path-geneticsorg-content-article-types  odd " role="menuitem"><a href="http://www.genetics.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1246 menu-path-geneticsorg-content-article-types first odd " role="menuitem"><a href="http://www.genetics.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1228 menu-path-geneticsorg-collection-commentary  even " role="menuitem"><a href="http://www.genetics.org/collection/commentary" data-hide-link-title="0" class="" data-icon-position="">Commentaries</a></li>
<li class="menu-811 menu-path-taxonomy-term-12973  odd " role="menuitem"><a href="https://www.genetics.org/category/editorial" data-hide-link-title="0" class="" data-icon-position="">Editorials</a></li>
<li class="menu-1195 menu-path-geneticsorg-collection-gsa-honors-and-awards  even " role="menuitem"><a href="http://www.genetics.org/collection/gsa-honors-and-awards" data-hide-link-title="0" class="" data-icon-position="">GSA Honors and Awards</a></li>
<li class="menu-1222 menu-path-geneticsorg-collection-methods-technology-resources  odd " role="menuitem"><a href="http://www.genetics.org/collection/methods-technology-resources" data-hide-link-title="0" class="" data-icon-position="">Methods, Technology &amp; Resources</a></li>
<li class="menu-895 menu-path-taxonomy-term-12918  even " role="menuitem"><a href="https://www.genetics.org/category/perspectives" data-hide-link-title="0" class="" data-icon-position="">Perspectives</a></li>
<li class="menu-1221 menu-path-geneticsorg-collection-primer  odd " role="menuitem"><a href="http://www.genetics.org/collection/primer" data-hide-link-title="0" class="" data-icon-position="">Primers</a></li>
<li class="menu-1220 menu-path-geneticsorg-collection-reviews  even " role="menuitem"><a href="http://www.genetics.org/collection/reviews" data-hide-link-title="0" class="" data-icon-position="">Reviews</a></li>
<li class="menu-1227 menu-path-geneticsorg-collection-genetic-toolbox-review  odd last" role="menuitem"><a href="http://www.genetics.org/collection/genetic-toolbox-review" data-hide-link-title="0" class="" data-icon-position="">Toolbox Reviews</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="https://www.genetics.org/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="https://www.genetics.org/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="https://www.genetics.org/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="https://www.genetics.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="https://www.genetics.org/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="https://www.genetics.org/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="https://www.genetics.org/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="https://www.genetics.org/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="https://www.genetics.org/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-891 menu-path-node-428899 first odd " role="menuitem"><a href="https://www.genetics.org/content/why-subscribe" data-hide-link-title="0" class="" data-icon-position="">Why subscribe?</a></li>
<li class="menu-794 menu-path-node-3  even " role="menuitem"><a href="https://www.genetics.org/content/institutional" data-hide-link-title="0" class="" data-icon-position="">For institutions</a></li>
<li class="menu-795 menu-path-node-428900  odd " role="menuitem"><a href="https://www.genetics.org/content/individual-subscriptions" data-hide-link-title="0" class="" data-icon-position="">For individuals</a></li>
<li class="menu-679 menu-path-alerts  even " role="menuitem"><a href="https://www.genetics.org/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  odd last" role="menuitem"><a href="https://www.genetics.org/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout">
	  
  <div class="panel-row-wrapper clearfix">
		
		<iframe id="Sticky" src="https://remarqable.com/web/sticky.html" scrolling="no" style="overflow: hidden; visibility: hidden; position: fixed;" data-expanded="normal" data-pd-id="gsa" title="Remarq Sticky widget" frameborder="0"></iframe><div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager">
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="https://www.genetics.org/content/177/2/847" title="The Flightless I Homolog, fli-1, Regulates Anterior/Posterior Polarity, Asymmetric Cell Division and Ovulation During Caenorhabditis elegans Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="https://www.genetics.org/content/177/2/875" title="Reconciling Strong Stabilizing Selection with the Maintenance of Genetic Variation in a Natural Population of Black Field Crickets (Teleogryllus commodus)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation">
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="334013" id="node334013" data-pisa="genetics;177/2/861" data-pisa-master="genetics;genetics.107.077263" data-apath="/genetics/177/2/861.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Empirical Bayes Inference of Pairwise <em>F</em><sub>ST</sub> and Its Distribution in the Genome</h1>  
    	<div class="highwire-cite-authors"><span class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shuichi Kitada</span>, <span class="highwire-citation-author" data-delta="1">Toshihide Kitakado</span> and <span class="highwire-citation-author" data-delta="2">Hirohisa Kishino</span></span></div>
  
    	<div class="highwire-cite-metadata"><span class="highwire-cite-metadata-journal highwire-cite-metadata">Genetics </span><span class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2007 </span><span class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 177 </span><span class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span class="highwire-cite-metadata-pages highwire-cite-metadata">861-873;  </span><span class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/genetics.107.077263 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node334013" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shuichi Kitada</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuichi%2BKitada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kitada%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AShuichi%2BKitada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Toshihide Kitakado</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toshihide%2BKitakado" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kitakado%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AToshihide%2BKitakado" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hirohisa Kishino</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hirohisa%2BKishino" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kishino%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AHirohisa%2BKishino" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab panels-ajax-tabs-once-processed">
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first active"><a href="https://www.genetics.org/content/177/2/861" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed panels-ajax-tabs-first-loaded hw-panels-ajax-tabs-once-processed" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:334013" data-trigger="" data-url-enabled="1" style="cursor: pointer;">Article</a><a href="https://www.genetics.org/panels_ajax_tab/jnl_genetics_tab_art/node:334013/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="https://www.genetics.org/content/177/2/861.figures-only" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed hw-panels-ajax-tabs-once-processed" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:334013" data-trigger="figures-only" data-url-enabled="1" style="cursor: pointer;">Figures &amp; Data</a><a href="https://www.genetics.org/panels_ajax_tab/jnl_genetics_tab_data/node:334013/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="https://www.genetics.org/content/177/2/861.supplemental" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed hw-panels-ajax-tabs-once-processed" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:334013" data-trigger="supplemental" data-url-enabled="1" style="cursor: pointer;">Supplemental</a><a href="https://www.genetics.org/panels_ajax_tab/jnl_genetics_tab_supp/node:334013/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="https://www.genetics.org/content/177/2/861.article-info" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed hw-panels-ajax-tabs-once-processed" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:334013" data-trigger="article-info" data-url-enabled="1" style="cursor: pointer;">Info &amp; Metrics</a><a href="https://www.genetics.org/panels_ajax_tab/jnl_genetics_tab_info/node:334013/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="https://www.genetics.org/content/genetics/177/2/861.full.pdf" class="panels-ajax-tab-tab panels-ajax-tabs-once-processed hw-panels-ajax-tabs-once-processed" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:334013" data-trigger="full.pdf+html" data-url-enabled="1" style="cursor: pointer;" target="_blank"><i class="icon-file-alt"></i></a><a href="https://www.genetics.org/panels_ajax_tab/jnl_genetics_tab_pdf/node:334013/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab">
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style="display:none"><img class="loading" src="loading.gif" alt="Loading" title="Loading"></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire rmq-annotator-wrapper">
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view"><span class="highwire-journal-article-marker-start"></span><iframe id="TopWidget" src="https://remarqable.com/web/topwidget.html" scrolling="no" style="overflow: hidden; visibility: hidden; position: fixed;" data-expanded="normal" data-pd-id="gsa" title="Remarq Top Widget" frameborder="0"></iframe><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-1">Populations
 often have very complex hierarchical structure. Therefore, it is 
crucial in genetic monitoring and conservation biology to have a 
reliable estimate of the pattern of population subdivision. <em>F</em><sub>ST</sub>'s
 for pairs of sampled localities or subpopulations are crucial 
statistics for the exploratory analysis of population structures, such 
as cluster analysis and multidimensional scaling. However, the 
estimation of <em>F</em><sub>ST</sub> is not precise enough to reliably 
estimate the population structure and the extent of heterogeneity. This 
article proposes an empirical Bayes procedure to estimate locus-specific
 pairwise <em>F</em><sub>ST</sub>'s. The posterior mean of the pairwise <em>F</em><sub>ST</sub>
 can be interpreted as a shrinkage estimator, which reduces the variance
 of conventional estimators largely at the expense of a small bias. The 
global <em>F</em><sub>ST</sub> of a population generally varies among loci in the genome. Our maximum-likelihood estimates of global <em>F</em><sub>ST</sub>'s can be used as sufficient statistics to estimate the distribution of <em>F</em><sub>ST</sub>
 in the genome. We demonstrate the efficacy and robustness of our model 
by simulation and by an analysis of the microsatellite allele 
frequencies of the Pacific herring. The heterogeneity of the global <em>F</em><sub>ST</sub> in the genome is discussed on the basis of the estimated distribution of the global <em>F</em><sub>ST</sub> for the herring and examples of human single nucleotide polymorphisms (SNPs).</p></div><p id="p-2">INFERRING
 genetic population structure has been a major theme in population 
biology, ecology, and human genetics. The fixation index <em>F</em><sub>ST</sub>, introduced by <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">W<span class="sc">right</span> (1951)</a>, is a key parameter for such studies and is most commonly used to measure genetic divergence among subpopulations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">P<span class="sc">alsbÃ¸ll</span> <em>et al</em>. 2007</a>).
 It is defined as the correlation between random gametes drawn from the 
same subpopulation relative to the total population. Another measure 
used frequently is <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">C<span class="sc">ockerham</span>'s (1969</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">1973</a>)
 coancestry coefficient, which is the probability that two random genes 
from different individuals are identical by descent, and the average 
overall pairs of individuals within the same subpopulation equal 
Wright's <em>F</em><sub>ST</sub> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">E<span class="sc">xcoffier</span> 2003</a>). We use the notation Î¸<sub>WC</sub> for the average coancestry coefficient and Î¸<sub>WC</sub> = <em>F</em><sub>ST</sub> as shown by <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">W<span class="sc">eir</span> and C<span class="sc">ockerham</span> (1984)</a>. <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">N<span class="sc">ei</span>'s (1973)</a> <em>G</em><sub>ST</sub> is analogous to <em>F</em><sub>ST</sub> and identical to <em>F</em><sub>ST</sub> for diploid random-mating populations (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">E<span class="sc">xcoffier</span> 2003</a>).</p><p id="p-3"><a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">N<span class="sc">ei</span> and C<span class="sc">hesser</span> (1983)</a> proposed an estimator for <em>F</em><sub>ST</sub> and <em>G</em><sub>ST</sub>.
 The estimation of these parameters accounts only for the sampling error
 within subpopulations and therefore assumes that all subpopulations 
have been sampled (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">C<span class="sc">ockerham</span> and W<span class="sc">eir</span> 1986</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">E<span class="sc">xcoffier</span> 2003</a>). <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">W<span class="sc">eir</span> and C<span class="sc">ockerham</span> (1984)</a> developed the moment estimator <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-1.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-1.gif"/></noscript></span></span> for the coancestry coefficient Î¸<sub>WC</sub>,
 which takes the sampling error for the subpopulations into account. 
Several moment estimators with different weighting schemes have also 
been derived (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">R<span class="sc">obertson</span> and H<span class="sc">ill</span> 1984</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">W<span class="sc">eir</span> and C<span class="sc">ockerham</span> 1984</a>). An alternative estimation has been discussed using the method of ordinary least squares (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">R<span class="sc">eynolds</span> <em>et al</em>. 1983</a>). <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> (2002)</a> extended Î¸<sub>WC</sub>
 to a population-specific parameter to allow different levels of 
coancestry for different populations. They also derived an estimator for
 Î¸<sub>WC</sub> with confidence intervals using a normal theory approach.</p><p id="p-4">Despite the development of methods for assigning individuals to populations (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">P<span class="sc">aetkau</span> <em>et al</em>. 1995</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">P<span class="sc">ritchard</span> <em>et al</em>. 2000</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">H<span class="sc">uelsenbeck</span> and A<span class="sc">ndolfatto</span> 2007</a>), the differentiation estimators remain the most commonly used tools for describing population structure (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">B<span class="sc">alloux</span> and L<span class="sc">ugon</span>-M<span class="sc">oulin</span> 2002</a>). <a id="xref-ref-53-4" class="xref-bibr" href="#ref-53">W<span class="sc">eir</span> and C<span class="sc">ockerham</span> (1984)</a> showed that their estimator <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-2.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-2.gif"/></noscript></span></span> provides the smallest bias among the moment estimators. <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">G<span class="sc">oudet</span> <em>et al</em>. (1996)</a> confirmed this using simulations and showed that <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-3.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-3.gif"/></noscript></span></span> generates the least-biased estimate of <em>F</em><sub>ST</sub> but has the largest variance when <em>F</em><sub>ST</sub> is small. <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">R<span class="sc">aufaste</span> and B<span class="sc">onhomme</span> (2000)</a> showed that <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-4.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-4.gif"/></noscript></span></span> is nearly unbiased, with minimal variance for large <em>F</em><sub>ST</sub>, and that the estimator of <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">R<span class="sc">obertson</span> and H<span class="sc">ill</span> (1984)</a> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-5.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-5.gif"/></noscript></span></span> is negatively biased, with minimal variance for small <em>F</em><sub>ST</sub>. They proposed a correction for the bias of <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-6.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-6.gif"/></noscript></span></span> but this cannot be corrected properly in the range of [0.05, 0.1]. Therefore, a precise estimate of <em>F</em><sub>ST</sub> is crucial, especially for small and moderate levels of genetic differentiation.</p><p id="p-5">In addition to the estimation of <em>F</em><sub>ST</sub> over all subpopulations in a metapopulation (hereafter, we call this global <em>F</em><sub>ST</sub>), <em>F</em><sub>ST</sub>'s for pairs of sampled localities or subpopulations (pairwise <em>F</em><sub>ST</sub>) are usually estimated in conservation biology and ecology. In fact, the computer programs Arlequin (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">E<span class="sc">xcoffier</span> <em>et al</em>. 2005</a>), FSTAT (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">G<span class="sc">oudet</span> 1995</a>), and Genepop (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">R<span class="sc">aymond</span> and R<span class="sc">ousset</span> 1995</a>)
 estimate these parameters and are used widely in ecological studies. 
These three software packages produce the same or similar values for 
pairwise <em>F</em><sub>ST</sub> estimates and provide the basic 
statistics for exploratory analyses of population structure, such as 
cluster analysis and multidimensional scaling. They are also used as a 
criterion for population differentiation (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">W<span class="sc">aples</span> and G<span class="sc">aggiotti</span> 2006</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">P<span class="sc">alsbÃ¸ll</span> <em>et al</em>. 2007</a>). However, the estimation of <em>F</em><sub>ST</sub>'s
 is not precise enough to reliably estimate the population structure and
 the extent of heterogeneity, especially for large gene flow species.</p><p id="p-6">Small numbers of individuals taken from each locality should also affect the precision of <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-7.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-7.gif"/></noscript></span></span>
 Populations often have very complex hierarchical structures, and 
geographical samples are usually taken from many localities to include a
 wide area. Therefore, the numbers of individuals from each locality are
 frequently limited by the large number of sampling points. Small sample
 sizes can result in biased estimates of the allele frequencies of each 
subpopulation. This bias may be larger for cases with larger numbers of 
alleles, such as microsatellite DNA. Uncertainty in the estimates of 
allele frequencies should affect the estimation of <em>F</em><sub>ST</sub>'s. The Bayesian approach provides better estimates of allele frequencies by taking uncertainty into account (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">L<span class="sc">ange</span> 1995</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">L<span class="sc">ockwood</span> <em>et al</em>. 2001</a>). Posterior distributions of global <em>F</em><sub>ST</sub> were simulated from posterior distributions of allele frequencies, assuming common hyperparameters across all loci (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">H<span class="sc">olsinger</span> 1999</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">H<span class="sc">olsinger</span> <em>et al</em>. 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">C<span class="sc">orander</span> <em>et al</em>. 2003</a>). However, accurate estimation of pairwise <em>F</em><sub>ST</sub>, the essential parameter in ecological studies, has not been fully investigated.</p><p id="p-7">In this article, we propose an empirical Bayes procedure to estimate locus-specific pairwise <em>F</em><sub>ST</sub>'s,
 taking into account the uncertainty of the allele frequencies of 
subpopulations. The estimation procedure has two stages. First, the 
hyperparameters of Dirichlet prior distributions for allele frequencies 
at each locus are estimated from observed allele counts by a 
maximum-likelihood method. The global <em>F</em><sub>ST</sub> is then 
estimated at each locus. Second, on the basis of the estimates of the 
hyperparameters, and given the allele counts, posterior distributions of
 the allele frequencies are generated for each locus, from which the 
posterior distributions of locus-specific pairwise <em>F</em><sub>ST</sub>'s are simulated. The posterior mean of our empirical Bayes pairwise <em>F</em><sub>ST</sub> estimates can be interpreted as a shrinkage estimator (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">S<span class="sc">tein</span> 1956</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">M<span class="sc">aritz</span> and L<span class="sc">ewin</span> 1989</a>)
 anchored to the average of the true values among pairs. It performs 
better than conventional differentiation estimators and robustly 
estimates the population structure, even for non-Dirichlet cases, as 
stepping-stone models. The posterior distribution of pairwise <em>F</em><sub>ST</sub>'s can be used to calculate a criterion of population differentiation. Our maximum-likelihood estimates of the global <em>F</em><sub>ST</sub>'s can also be used as sufficient statistics to estimate the distribution of <em>F</em><sub>ST</sub>
 among loci in the genome. Our model assumes random mating or random 
sampling of alleles at each locality and that linkage equilibrium holds 
between loci. It also assumes that allele counts at each locus, given 
the true allele frequencies, are independent among populations. Our 
method can be applied to frequency data for common genetic markers, 
including isozymes, mitochondrial DNA, microsatellites, and single 
nucleotide polymorphisms (SNPs). We show the efficacy of our model by 
simulation and by an analysis of microsatellite allele frequencies of 
the Pacific herring. The heterogeneity of <em>F</em><sub>ST</sub> in the genome is discussed on the basis of the estimated distribution of global <em>F</em><sub>ST</sub> for the herring and examples of human SNPs.</p><div class="section" id="sec-1"><h2>MODELS AND METHODS</h2><div id="sec-2" class="subsection"><div id="sec-3" class="subsection"><h4>The model:</h4><p id="p-8">Consider a simple random sampling from multiple localities in a metapopulation. Suppose that <em>K</em> random-mating demes or subpopulations are drawn from the metapopulation. Let <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-8.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-8.gif"/></noscript></span></span> be a vector of the true allele frequencies at locus <em>l</em> in subpopulation <em>k</em>, where <em>J<sub>l</sub></em> is the number of different alleles at the locus, and <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-9.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-9.gif"/></noscript></span></span> We assume a Dirichlet distribution as the prior distribution of <em>p<sub>kl</sub></em>. The probability density function is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-10.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-10.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-11.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-11.gif"/></noscript></span></span> are the hyperparameters and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-12.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-12.gif"/></noscript></span></span>
 is a scale parameter that is specific for the locus. This model 
describes well a metapopulation that has a continuous structure and 
consists of an infinite number of subpopulations or demes (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">P<span class="sc">annell</span> and C<span class="sc">harlesworth</span> 2000</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">R<span class="sc">ousset</span> 2003</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">H<span class="sc">anski</span> and G<span class="sc">aggiotti</span> 2004</a>). Let <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-13.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-13.gif"/></noscript></span></span> be the mean allele frequency for the metapopulation at the locus satisfying <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-14.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-14.gif"/></noscript></span></span> Hence, we have the relation Î²<sub><em>lj</em></sub> = Î±<sub><em>lj</em></sub>/Î¸<sub><em>l</em></sub>.</p><p id="p-9">Under this model, the global <em>F</em><sub>ST</sub> (hereafter denoted as <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-15.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-15.gif"/></noscript></span></span>) at each locus is expressed simply by the scale parameter, as<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-16.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-16.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>as given by <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">W<span class="sc">right</span> (1969)</a>, <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">R<span class="sc">annala</span> and H<span class="sc">artigan</span> (1996)</a>, <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">B<span class="sc">alding</span> and N<span class="sc">ichols</span> (1997)</a>, <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">L<span class="sc">ockwood</span> <em>et al</em>. (2001)</a>, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">B<span class="sc">alding</span> (2003)</a>, and <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">K<span class="sc">itada</span> and K<span class="sc">ishino</span> (2004)</a>. In this model, the variance of the <em>j</em>th allele frequency for the locus, <em>p<sub>klj</sub></em>, is expressed by<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-17.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-17.gif"/></noscript></span></span>as given by <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">W<span class="sc">eir</span> (1996)</a>, <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">H<span class="sc">olsinger</span> <em>et al</em>. (2002)</a>, <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">B<span class="sc">alding</span> (2003)</a>, and <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">K<span class="sc">itakado</span> <em>et al</em>. (2006)</a>. The Dirichlet distribution assumes an evolutionary equilibrium and an equal mutation rate for all alleles (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> 2002</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">E<span class="sc">wens</span> 2004</a>). Under this assumption, the scale parameter Î¸<sub><em>l</em></sub> refers to the rate of gene flow, as given by <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">R<span class="sc">annala</span> and H<span class="sc">artigan</span> (1996)</a>. We use the symbol Î¸ for the scale parameter, following <a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">R<span class="sc">annala</span> and H<span class="sc">artigan</span> (1996)</a> and <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">B<span class="sc">alding</span> and N<span class="sc">ichols</span> (1997)</a>. <a id="xref-ref-53-5" class="xref-bibr" href="#ref-53">W<span class="sc">eir</span> and C<span class="sc">ockerham</span> (1984)</a> also used the same symbol Î¸ for the coancestry coefficient (= <em>F</em><sub>ST</sub>), so we use Î¸<sub>WC</sub> for their Î¸. Our <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-18.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-18.gif"/></noscript></span></span> is equivalent to Î¸<sub>WC</sub> (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">W<span class="sc">eir</span> 1996</a>, pp. 47â48) and Holsinger's Î¸<sup>B</sup> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">H<span class="sc">olsinger</span> 1999</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">H<span class="sc">olsinger</span> <em>et al</em>. 2002</a>).</p></div><div id="sec-4" class="subsection"><h4>Maximum-likelihood estimation of hyperparameters and global <em>F</em><sub>ST</sub>:</h4><p id="p-10">The maximum-likelihood estimation of the hyperparameters has been discussed by <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">L<span class="sc">ange</span> (1995)</a>, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">K<span class="sc">itada</span> <em>et al</em>. (2000)</a>, and <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">B<span class="sc">alding</span> (2003)</a>. A pseudo-likelihood approach was also taken by <a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">R<span class="sc">annala</span> and H<span class="sc">artigan</span> (1996)</a>. In the maximum-likelihood framework, a method for the simultaneous estimation of <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-19.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-19.gif"/></noscript></span></span> and the linkage disequilibrium coefficient between two SNPs has been proposed (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">K<span class="sc">itada</span> and K<span class="sc">ishino</span> 2004</a>). <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">K<span class="sc">itakado</span> <em>et al</em>. (2006)</a> proposed an integrated-likelihood approach to reduce the negative bias of <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-20.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-20.gif"/></noscript></span></span> particularly for cases with few sampling points.</p><p id="p-11">Suppose that <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-21.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-21.gif"/></noscript></span></span> alleles of diploid organisms (<em>N<sub>k</sub></em>/2 individuals) are counted at locus <em>l</em> and <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-22.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-22.gif"/></noscript></span></span> denotes a vector of observed allele counts at the locus in subpopulation <em>k</em>. We assume that all individuals are successfully genotyped at all loci, so <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-23.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-23.gif"/></noscript></span></span> The marginal likelihood of the observed allele counts at a locus <em>n<sub>kl</sub></em> has a Dirichlet-multinomial distribution (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">L<span class="sc">ange</span> 1995</a>; <a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">R<span class="sc">annala</span> and H<span class="sc">artigan</span> 1996</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">W<span class="sc">eir</span> 1996</a>; <a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">B<span class="sc">alding</span> and N<span class="sc">ichols</span> 1997</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">K<span class="sc">itada</span> <em>et al</em>. 2000</a>; <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">B<span class="sc">alding</span> 2003</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">R<span class="sc">ousset</span> 2003</a>). The parameters to be estimated are <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-24.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-24.gif"/></noscript></span></span>
 Because we assume the independence of subpopulations, the overall 
likelihood for these parameters is given by the product of the 
likelihood functions for <em>K</em> samples, as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-25.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-25.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>The hyperparameters Î±<sub><em>l</em></sub> are estimated by maximizing this marginal likelihood (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">L<span class="sc">ange</span> 1995</a>; <a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">K<span class="sc">itada</span> <em>et al</em>. 2000</a>).
 Our method can be used for both allele and haplotype counts without 
modification, but some notations differ slightly. For haploid organisms,
 <em>N<sub>k</sub></em> refers to the individuals genotyped; and <em>n<sub>kl</sub></em> should be <em>n<sub>k</sub></em> and Î±<sub><em>lj</em></sub> should be Î±<sub><em>j</em></sub>. Henceforth, for simplicity, we focus on diploid organisms.</p><p id="p-12">The locus-specific <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-26.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-26.gif"/></noscript></span></span> can be estimated by substituting <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-27.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-27.gif"/></noscript></span></span> for Î¸<sub><em>l</em></sub> in <a id="xref-disp-formula-2-1" class="xref-disp-formula" href="#disp-formula-2">Equation 1</a>. The variance estimator for <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-28.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-28.gif"/></noscript></span></span> is calculated from the Fisher information matrix for Î±<sub><em>l</em></sub>, as <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-29.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-29.gif"/></noscript></span></span> The asymptotic variance for <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-30.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-30.gif"/></noscript></span></span> is estimated using the Delta method (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">S<span class="sc">eber</span> 1982</a>) as<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-31.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-31.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>In
 our metapopulation model or infinite-island model, the sampled 
localities are regarded as a sample from all possible demes or 
subpopulations, including those not sampled. Hence, <a id="xref-disp-formula-4-1" class="xref-disp-formula" href="#disp-formula-4">Equation 2</a> estimates the locus-specific genetic differentiation under the random-effect model of population sampling (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">W<span class="sc">eir</span> 1996</a>). The average estimate of <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-32.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-32.gif"/></noscript></span></span> for all loci is calculated as an arithmetic mean across the loci.</p></div><div id="sec-5" class="subsection"><h4>Empirical Bayes estimation of pairwise <em>F</em><sub>ST</sub>:</h4><p id="p-13">The posterior distribution of allele frequencies <em>p<sub>kl</sub></em> at locus <em>l</em> in subpopulation <em>k</em> is again a Dirichlet distribution, with parameters modified by the sample allele counts<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-33.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-33.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>(<a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">L<span class="sc">ange</span> 1995</a>; <a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">W<span class="sc">eir</span> 1996</a>). Given the estimates of the hyperparameters and the sampled allele counts, random numbers of <em>p<sub>kl</sub></em> can be generated through this posterior distribution. The posterior distributions for any parametric functions of <em>p<sub>kl</sub></em> can then be simulated by the empirical Bayes procedure (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">K<span class="sc">itada</span> <em>et al</em>. 2000</a>).</p><p id="p-14">When
 population differentiation between or among specific subpopulations is 
of interest, the selected populations can be regarded as the entire set 
of populations. Hence, applying the fixed-effect model of population 
sampling (<a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">W<span class="sc">eir</span> 1996</a>) is appropriate. Therefore, we use Nei's <em>G</em><sub>ST</sub> formula (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">N<span class="sc">ei</span> and C<span class="sc">hesser</span> 1983</a>), which defines quantities with respect to fixed extant populations (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">C<span class="sc">ockerham</span> and W<span class="sc">eir</span> 1986</a>), to estimate the posterior distributions of pairwise <em>F</em><sub>ST</sub>'s (hereafter denoted as <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-34.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-34.gif"/></noscript></span></span>), as did <a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">H<span class="sc">olsinger</span> (1999)</a> and <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">C<span class="sc">orander</span> <em>et al</em>. (2003)</a> in estimating global <em>F</em><sub>ST</sub>. Nei's gene diversity analysis compares expected heterozygosities under HardyâWeinberg equilibrium (HWE), and the <em>G</em><sub>ST</sub> estimator is expressed as a function of allele frequencies. Therefore the posterior distribution of <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-35.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-35.gif"/></noscript></span></span> at each locus can easily be generated on the basis of the <em>G</em><sub>ST</sub> estimator, without using genotype frequencies. We set the number of each simulation to 10,000, so 10,000 <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-36.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-36.gif"/></noscript></span></span>'s are calculated at each locus from the 10,000 sets of allele frequencies <em>p<sub>kl</sub></em> between a set of two populations. From the posterior distribution of <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-37.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-37.gif"/></noscript></span></span>
 the posterior mean and 95% credible interval are calculated. We use the
 posterior mean as the empirical Bayes estimator of locus-specific <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-38.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-38.gif"/></noscript></span></span> We can also calculate the probability that <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-39.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-39.gif"/></noscript></span></span> is smaller than an arbitrary value [<em>e.g</em>., <em>P</em>(<span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-40.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-40.gif"/></noscript></span></span>)], which can be used as the criterion for population differentiation (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">W<span class="sc">aples</span> and G<span class="sc">aggiotti</span> 2006</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">P<span class="sc">alsbÃ¸ll</span> <em>et al</em>. 2007</a>). The average estimate of <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-41.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-41.gif"/></noscript></span></span> for overall loci is calculated as an arithmetic mean across the loci.</p><p id="p-15"><a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">R<span class="sc">osenberg</span> <em>et al</em>. (2003)</a>
 proposed a general measure for determining the amount of information on
 individual ancestry on the basis of the KullbackâLeibler information. 
The informativeness for assignment <em>I<sub>n</sub></em> is defined as<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-42.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-42.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-43.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-43.gif"/></noscript></span></span> The authors showed that <em>I<sub>n</sub></em> and <em>F</em><sub>ST</sub> are very closely correlated but that <em>I<sub>n</sub></em> is more informative than the standard SNP-specific pairwise <em>F</em><sub>ST</sub>. In an additional analysis, we examine how our empirical Bayes method works to measure this under the same simulation protocol.</p></div><div id="sec-6" class="subsection"><h4>Inferring heterogeneity of global <em>F</em><sub>ST</sub> among loci:</h4><p id="p-16">We estimate locus-specific <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-44.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-44.gif"/></noscript></span></span>'s on the basis of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-45.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-45.gif"/></noscript></span></span>
 estimated at each locus. Evolutionary forces may differ among sites in 
the genome. Therefore, it is important to investigate the heterogeneity 
of <em>F</em><sub>ST</sub> among loci. One practical analysis is to test the null hypothesis H<sub>0</sub>, the homogeneity of <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-46.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-46.gif"/></noscript></span></span> among <em>L</em> loci, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-47.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-47.gif"/></noscript></span></span>) against the alternative hypothesis H<sub>1</sub>, the heterogeneity of <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-48.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-48.gif"/></noscript></span></span> among loci, on the basis of estimates of <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-49.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-49.gif"/></noscript></span></span> When a large number of subpopulations are sampled, the maximum-likelihood estimate <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-50.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-50.gif"/></noscript></span></span> follows a normal distribution of <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-51.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-51.gif"/></noscript></span></span> The maximum likelihood under H<sub>0</sub> is then given as <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-52.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-52.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-53.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-53.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-54.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-54.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-55.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-55.gif"/></noscript></span></span> The maximum likelihood under H<sub>1</sub> is <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-56.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-56.gif"/></noscript></span></span> maximizing <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-57.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-57.gif"/></noscript></span></span> by <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-58.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-58.gif"/></noscript></span></span> The negative twice-log-likelihood ratio is then <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-59.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-59.gif"/></noscript></span></span> which follows the Ï<sup>2</sup>-distribution with (<em>L</em> â 1) d.f. under the null hypothesis. We can test the heterogeneity of <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-60.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-60.gif"/></noscript></span></span> on the basis of the test statistics.</p><p id="p-17">The other approach to investigate the heterogeneity of <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-61.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-61.gif"/></noscript></span></span> is to estimate the distribution of <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-62.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-62.gif"/></noscript></span></span>
 in the genome. In recent years, the number of loci analyzed has been 
increasing in ecological studies, but is still smaller than those used 
for human SNPs. For such cases, it would be difficult to directly 
estimate the specific distribution of <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-63.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-63.gif"/></noscript></span></span> from the data. Here, we estimate the distribution of <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-64.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-64.gif"/></noscript></span></span> in the genome from estimates of randomly selected loci, <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-65.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-65.gif"/></noscript></span></span> When the distribution is expressed by the parametric model <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-66.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-66.gif"/></noscript></span></span> the unknown parameter <span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-67.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-67.gif"/></noscript></span></span> which defines the distribution of <span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-68.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-68.gif"/></noscript></span></span> is estimated by maximizing the log marginal likelihood<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-69.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-69.gif"/></noscript></span></span>where <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-70.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-70.gif"/></noscript></span></span> is the distribution of <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-71.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-71.gif"/></noscript></span></span> Here, we assume the independence of loci <span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-72.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-72.gif"/></noscript></span></span> Because the maximum-likelihood estimator <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-73.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-73.gif"/></noscript></span></span> is a sufficient statistic, it is possible to estimate the distribution of <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-74.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-74.gif"/></noscript></span></span> in the genome on the basis of the estimates <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-75.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-75.gif"/></noscript></span></span> for randomly sampled loci, instead of using a direct estimation from the data.</p><p id="p-18">For the preliminary discussion here, we assume that <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-76.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-76.gif"/></noscript></span></span> is normally distributed in the genome. When the distribution of <span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-77.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-77.gif"/></noscript></span></span> is expected to be different from 0, a simple approximation may be a normal distribution. We then assume <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-78.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-78.gif"/></noscript></span></span> follows <em>N</em>(Î¼, Ï<sup>2</sup>) as a first step in estimating the distribution of <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-79.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-79.gif"/></noscript></span></span> in the genome under the limited number of loci analyzed. In this case, the parameter vector Ï refers to Î¼ and Ï<sup>2</sup>. The general form of the log marginal likelihood given above becomes<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-80.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-80.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>Here, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-81.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-81.gif"/></noscript></span></span> is the variance of the estimates, <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-82.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-82.gif"/></noscript></span></span> We estimate Î¼ and Ï<sup>2</sup> numerically, regarding <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-83.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-83.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-84.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-84.gif"/></noscript></span></span> The distribution of <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-85.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-85.gif"/></noscript></span></span>'s can also be calculated with slight modifications to the above procedure.</p></div></div></div><div class="section" id="sec-7"><h2>RESULTS</h2><div id="sec-8" class="subsection"><div id="sec-9" class="subsection"><h4>Improved precision of our empirical Bayes estimator for pairwise <em>F</em><sub>ST</sub>:</h4><p id="p-19">We investigated the performance of our method of estimating pairwise <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-86.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-86.gif"/></noscript></span></span> using numerical simulations. Random vectors of allele frequencies at locus <em>l</em> in subpopulation <em>k</em>, <em>p<sub>kl</sub></em>'s, were generated independently from the Dirichlet distribution with the parameter Î±<sub><em>l</em></sub>(= Î¸<sub><em>l</em></sub>âÎ²<sub><em>l</em></sub>). Here, the number of sampling localities (<em>K</em>) was set at 5, 10, and 50, and the mean allele frequencies at a locus were assumed <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-87.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-87.gif"/></noscript></span></span> with <em>J<sub>l</sub></em> = 50. As the true values of the global <em>F</em><sub>ST,<em>l</em></sub>'s (= 1/(1 + Î¸<sub><em>l</em></sub>)), we chose four different levels: <span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-88.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-88.gif"/></noscript></span></span> = 0.01, 0.05, 0.1, and 0.2. The sample size (<em>N<sub>k</sub></em>/2) was deemed to be common to all the localities and was set at 20, 30, and 50 individuals. Then, allele counts <em>n<sub>kl</sub></em>'s were drawn independently from the multinomial distribution Multi(<em>N<sub>k</sub></em>, <em>p<sub>kl</sub></em>) for <em>K</em> localities. The pairwise <span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-89.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-89.gif"/></noscript></span></span> values between the first and second localities were evaluated by the conventional Nei's <em>G</em><sub>ST</sub> estimator and the empirical Bayes method. In the latter procedure, 500 <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-90.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-90.gif"/></noscript></span></span>'s were simulated on the basis of Nei's <em>G</em><sub>ST</sub> formula to save computation time, and the posterior mean was calculated as the estimator of the pairwise <em>F</em><sub>ST</sub>. The point estimate for the conventional <em>G</em><sub>ST</sub> estimator and the posterior mean of the empirical Bayes estimates were compared with the true <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-91.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-91.gif"/></noscript></span></span> These procedures were iterated <em>R</em> = 1000 times.</p><p id="p-20"><a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and supplemental Figure S1 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a> show the general features of the empirical Bayes estimator compared with those of the conventional <em>G</em><sub>ST</sub>
 estimator. The former examines the case of a relatively large number of
 sampling points and a limited sample size from each sampling point (<em>K</em> = 50, <em>N<sub>k</sub></em>/2 = 20). The latter investigates the case of moderately large samples from a small number of sampling points (<em>K</em> = 10, <em>N<sub>k</sub></em>/2 = 50), which is common in ecological studies. It is clearly apparent that the conventional <em>G</em><sub>ST</sub> estimator greatly overestimates the true values and has a large variance, especially for small <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-92.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-92.gif"/></noscript></span></span> In contrast, the empirical Bayes estimates of <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-93.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-93.gif"/></noscript></span></span>'s shrink toward the average of the true <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-94.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-94.gif"/></noscript></span></span>
 and reduce the positive bias of the conventional estimator. This is 
reasonable because our empirical Bayes estimator can be interpreted as a
 shrinkage estimator (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">S<span class="sc">tein</span> 1956</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">M<span class="sc">aritz</span> and L<span class="sc">ewin</span> 1989</a>).
</p><div id="F1" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The conventional GST (A) and empirical Bayes (B) estimates of pairwise \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} from 1000 simulations under the infinite-island model at various levels of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}\) \end{document} (0.01, 0.05, 0.1, 0.2) over subpopulations. The mean allele frequencies assumed were \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(\mathrm{{\beta}}{=}(1,2,{\ldots},{\,}J)/(J(J{+}1)/2)\) \end{document} with J = 50. The number of sampling localities (K) was set at 50. The sample size Nk/2 (individuals) was common to all localities and was set at 20 individuals." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The conventional GST (A) and empirical Bayes (B) estimates of pairwise \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} from 1000 simulations under the infinite-island model at various levels of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}\) \end{document} (0.01, 0.05, 0.1, 0.2) over subpopulations. The mean allele frequencies assumed were \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(\mathrm{{\beta}}{=}(1,2,{\ldots},{\,}J)/(J(J{+}1)/2)\) \end{document} with J = 50. The number of sampling localities (K) was set at 50. The sample size Nk/2 (individuals) was common to all localities and was set at 20 individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1.â" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F1.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="Figure 1.&#x2014;" src="https://www.genetics.org/content/genetics/177/2/861/F1.medium.gif"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1.â" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381439" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">F<span class="sc">igure</span> 1.â</span> <p id="p-21" class="first-child">The conventional <em>G</em><sub>ST</sub> (A) and empirical Bayes (B) estimates of pairwise <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F1/embed/tex-math-95.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F1/embed/tex-math-95.gif"/></noscript></span></span> from 1000 simulations under the infinite-island model at various levels of <span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F1/embed/tex-math-96.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F1/embed/tex-math-96.gif"/></noscript></span></span> (0.01, 0.05, 0.1, 0.2) over subpopulations. The mean allele frequencies assumed were <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F1/embed/tex-math-97.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F1/embed/tex-math-97.gif"/></noscript></span></span> with <em>J</em> = 50. The number of sampling localities (<em>K</em>) was set at 50. The sample size <em>N<sub>k</sub></em>/2 (individuals) was common to all localities and was set at 20 individuals.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The effects of the number of sampling points <em>K</em> and sampled individuals <em>N<sub>k</sub></em>/2 are also shown in the supplemental material at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>. The numbers of sampling points did not affect the positive bias of the conventional <em>G</em><sub>ST</sub> estimator because <em>K</em>
 was fixed at 2. The empirical Bayes procedure provided larger variation
 for smaller numbers of subpopulations, especially for small <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-98.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-98.gif"/></noscript></span></span> values, and vice versa (supplemental Figure S2 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>). Conversely, larger numbers of sampled individuals reduced the positive bias of the conventional <em>G</em><sub>ST</sub> estimator and the variation of the empirical Bayes estimator (supplemental Figure S3 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>).</p><p id="p-23">For a quantitative comparison of the two estimators, we used the following two measures: <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-99.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-99.gif"/></noscript></span></span> (relative mean bias) and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-100.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-100.gif"/></noscript></span></span> (root relative mean squared error), where <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-101.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-101.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-102.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-102.gif"/></noscript></span></span> are an estimate and the true value, respectively, for pairwise <span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-103.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-103.gif"/></noscript></span></span> in the <em>r</em>âth iteration. As shown in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, smaller <em>K</em> and sample sizes (<em>N<sub>k</sub></em>/2) resulted in larger positive biases for the conventional estimator for various levels of true <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-104.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-104.gif"/></noscript></span></span> values. The bias and variation became smaller as <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-105.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-105.gif"/></noscript></span></span> became larger. In contrast, the empirical Bayes estimator provided smaller biases and variations for all cases of <span class="inline-formula" id="inline-formula-97"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-106.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-106.gif"/></noscript></span></span> although smaller <em>K</em> and sample sizes resulted in a slight negative bias.
</p><div class="table pos-float" id="T1"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/381464/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="https://www.genetics.org/highwire/markup/381464/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label"><strong>TABLE 1</strong></span> <p id="p-24" class="first-child"><strong>Relative mean bias and root relative mean squared error (in parentheses) of the conventional <em>G</em><sub>ST</sub> and empirical Bayes estimators of pairwise <em>F</em><sub>ST</sub> from 1000 simulations at various levels of global <em>F</em><sub>ST</sub> (see text)</strong></p><div class="sb-div caption-clear"></div></div></div><p id="p-25">The negative bias of the empirical Bayes estimator of pairwise <em>F</em><sub>ST</sub>
 is large, especially when gene flow is large and the estimation is 
based on a sample from a few sampling points. Underestimation of 
population differences should lead to optimistic management strategies. 
Therefore, it is recommended in the conservation genetics of birds and 
fish that samples be collected from as many sampling points as possible.
 It is noted that the bias of the empirical Bayes estimator is reduced 
by increasing the number of sampling points, whereas the bias of the 
conventional <em>G</em><sub>ST</sub> estimator is not.</p><p id="p-26">We estimated Rosenberg <em>et al</em>.'s informativeness of assignment <em>I<sub>n</sub></em> between the first and second localities using <a id="xref-disp-formula-7-1" class="xref-disp-formula" href="#disp-formula-7">Equation 5</a> (<em>K</em> = 2) and the empirical Bayes method based on the <em>I<sub>n</sub></em> estimator for the case of <span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-108.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-108.gif"/></noscript></span></span> The mean allele frequencies were assumed to be <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-109.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-109.gif"/></noscript></span></span> with <em>J</em> = 50. The number of sampling localities (<em>K</em>) was set at 50 and the sample size (<em>N<sub>k</sub></em>/2 individuals) was set at 20 individuals for each sampling point. <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows that the conventional estimator for <em>I<sub>n</sub></em> was positively biased, whereas the empirical Bayes estimator of <em>I<sub>n</sub></em> performed much better, consistent with the fact that <em>I<sub>n</sub></em> produces upwardly biased estimates in small samples (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">R<span class="sc">osenberg</span> <em>et al</em>. 2003</a>).
</p><div id="F2" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The conventional informativeness of assignment In (Rosenberg et al. 2003) (A) and empirical Bayes (B) estimates of In from 1000 simulations for the case of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}{=}0.01.\) \end{document} The mean allele frequencies assumed were \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(\mathrm{{\beta}}{=}(1,2,{\ldots},{\,}J)/(J(J{+}1)/2)\) \end{document} with J = 50. The number of sampling localities (K) was set at 50 and the sample size Nk/2 (individuals) was common to all localities and was set at 20 individuals." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The conventional informativeness of assignment In (Rosenberg et al. 2003) (A) and empirical Bayes (B) estimates of In from 1000 simulations for the case of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}{=}0.01.\) \end{document} The mean allele frequencies assumed were \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(\mathrm{{\beta}}{=}(1,2,{\ldots},{\,}J)/(J(J{+}1)/2)\) \end{document} with J = 50. The number of sampling localities (K) was set at 50 and the sample size Nk/2 (individuals) was common to all localities and was set at 20 individuals.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2.â" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F2.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="Figure 2.&#x2014;" src="https://www.genetics.org/content/genetics/177/2/861/F2.medium.gif"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2.â" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381443" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">F<span class="sc">igure</span> 2.â</span> <p id="p-27" class="first-child">The conventional informativeness of assignment <em>I<sub>n</sub></em> (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">R<span class="sc">osenberg</span> <em>et al</em>. 2003</a>) (A) and empirical Bayes (B) estimates of <em>I<sub>n</sub></em> from 1000 simulations for the case of <span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F2/embed/tex-math-110.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F2/embed/tex-math-110.gif"/></noscript></span></span> The mean allele frequencies assumed were <span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F2/embed/tex-math-111.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F2/embed/tex-math-111.gif"/></noscript></span></span> with <em>J</em> = 50. The number of sampling localities (<em>K</em>) was set at 50 and the sample size <em>N<sub>k</sub></em>/2 (individuals) was common to all localities and was set at 20 individuals.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h4>Robustness of our shrinkage estimator for pairwise <em>F</em><sub>ST</sub>:</h4><p id="p-28">The robustness of our <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-112.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-112.gif"/></noscript></span></span>
 estimator was explored using numerical simulations for non-Dirichlet 
distributions of the allele frequencies. We considered cases in which 
genetic differentiation becomes larger with geographic distance. Such a 
stepping-stone model is biologically realistic and may be common (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">P<span class="sc">alsbÃ¸ll</span> <em>et al</em>. 2007</a>). We set the number of sampled subpopulations (<em>K</em>) to 15 and the <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-113.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-113.gif"/></noscript></span></span>
 between two adjacent populations to 0.001 (case 1) or 0.0005 (case 2). 
We considered biallelic cases and set the allele frequencies to (0.5, 
0.5) at a locus for the middle population. We then calculated the allele
 frequencies <em>p<sub>kl</sub></em>'s at the locus for another 14 subpopulations by numerical optimization. The sample size (<em>N<sub>k</sub></em>/2) was deemed to be common to all localities and was set at 20 individuals. Then, allele counts <em>n<sub>k</sub></em>'s were drawn independently from the multinomial distribution Multi(<em>N<sub>k</sub></em>, <em>p<sub>kl</sub></em>) for 15 localities. A total of 105 pairwise <em>F</em><sub>ST</sub> values between all sets of the two localities were evaluated by the conventional <em>G</em><sub>ST</sub> and the empirical Bayes estimator following the simulation protocol described above.</p><p id="p-29">As shown in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> (case 1) and supplemental Figure S4 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a> (case 2, top left), our empirical Bayes procedure provided better estimates than those of the conventional <em>G</em><sub>ST</sub> estimator for smaller <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-114.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-114.gif"/></noscript></span></span>'s
 (â¼ &lt;0.06 for case 1 and 0.04 for case 2). In conservation, 
management units should be defined among subpopulations with little 
genetic differentiation. <a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">P<span class="sc">alsbÃ¸ll</span> <em>et al</em>. (2007)</a> concluded that <span class="inline-formula" id="inline-formula-106"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-115.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-115.gif"/></noscript></span></span>
 could be used as the criterion for deciding the separate management 
units of sockeye salmon spawning sites. For such cases with very small 
genetic differentiation, our method performs more efficiently, even for 
stepping-stone models. On the contrary, the conventional <em>G</em><sub>ST</sub> estimator displayed a positive bias for the whole range of <span class="inline-formula" id="inline-formula-107"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-116.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-116.gif"/></noscript></span></span>'s in both cases. Our method reduced the positive bias for small <span class="inline-formula" id="inline-formula-108"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-117.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-117.gif"/></noscript></span></span>'s and the bias became negative for larger <span class="inline-formula" id="inline-formula-109"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-118.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-118.gif"/></noscript></span></span>'s.
 Reflecting the characteristics of the shrinkage estimator, the relative
 mean bias of the empirical Bayes estimator, which was the average of 
the 105 pairwise <em>F</em><sub>ST</sub> estimates, was slightly (1.06 times) larger than that of the conventional estimator for case 1 (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The precision of our estimates was much better for the whole range of <span class="inline-formula" id="inline-formula-110"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-119.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-119.gif"/></noscript></span></span> [<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, top right (case 1) and supplemental Figure S4 (case 2)].

</p><div id="F3" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean (top left) and root relative mean squared error (top right) of the conventional GST (red circle) and empirical Bayes estimators (blue circle) of pairwise \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} from 1000 simulations under the stepping-stone models. Means and root MSEs were plotted on the true \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document}'s which fluctuated very slightly when small uniform random variables were added to prevent the points overlapping heavily. The number of subpopulations (K) was set at 15 and the pairwise \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} between two adjacent populations was set at 0.001 (case 1) and 0.0005 (case 2). The sample size Nk/2 (individuals) was common to all localities and set at 20 individuals. Only the results for case 1 are shown. The results of the MDS analysis of two data sets are given in the bottom section; black circles show the true population structure, and the estimated population structure based on the conventional (red â*â) and empirical Bayes estimates (blue â+â) of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} is shown." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean (top left) and root relative mean squared error (top right) of the conventional GST (red circle) and empirical Bayes estimators (blue circle) of pairwise \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} from 1000 simulations under the stepping-stone models. Means and root MSEs were plotted on the true \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document}'s which fluctuated very slightly when small uniform random variables were added to prevent the points overlapping heavily. The number of subpopulations (K) was set at 15 and the pairwise \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} between two adjacent populations was set at 0.001 (case 1) and 0.0005 (case 2). The sample size Nk/2 (individuals) was common to all localities and set at 20 individuals. Only the results for case 1 are shown. The results of the MDS analysis of two data sets are given in the bottom section; black circles show the true population structure, and the estimated population structure based on the conventional (red â*â) and empirical Bayes estimates (blue â+â) of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} is shown.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3.â" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F3.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="Figure 3.&#x2014;" src="https://www.genetics.org/content/genetics/177/2/861/F3.medium.gif"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3.â" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381447" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">F<span class="sc">igure</span> 3.â</span> <p id="p-30" class="first-child">Mean (top left) and root relative mean squared error (top right) of the conventional <em>G</em><sub>ST</sub> (red circle) and empirical Bayes estimators (blue circle) of pairwise <span class="inline-formula" id="inline-formula-111"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-120.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-120.gif"/></noscript></span></span> from 1000 simulations under the stepping-stone models. Means and root MSEs were plotted on the true <span class="inline-formula" id="inline-formula-112"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-121.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-121.gif"/></noscript></span></span>'s
 which fluctuated very slightly when small uniform random variables were
 added to prevent the points overlapping heavily. The number of 
subpopulations (<em>K</em>) was set at 15 and the pairwise <span class="inline-formula" id="inline-formula-113"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-122.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-122.gif"/></noscript></span></span> between two adjacent populations was set at 0.001 (case 1) and 0.0005 (case 2). The sample size <em>N<sub>k</sub></em>/2
 (individuals) was common to all localities and set at 20 individuals. 
Only the results for case 1 are shown. The results of the MDS analysis 
of two data sets are given in the bottom section; black circles show the
 true population structure, and the estimated population structure based
 on the conventional (red â*â) and empirical Bayes estimates (blue â+â) 
of <span class="inline-formula" id="inline-formula-114"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-123.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F3/embed/tex-math-123.gif"/></noscript></span></span> is shown.</p><div class="sb-div caption-clear"></div></div></div><div class="table pos-float" id="T2"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/381467/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="https://www.genetics.org/highwire/markup/381467/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label"><strong>TABLE 2</strong></span> <p id="p-31" class="first-child"><strong>Relative mean bias and root relative mean squared error of the conventional <em>G</em><sub>ST</sub> and empirical Bayes estimators of pairwise <em>F</em><sub>ST</sub> from 1000 simulations under the stepping-stone models (see text)</strong></p><div class="sb-div caption-clear"></div></div></div><p id="p-33">We also investigated the robustness of estimating population structure on the basis of estimated <span class="inline-formula" id="inline-formula-116"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-125.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-125.gif"/></noscript></span></span>'s. The results of the multidimensional scaling (MDS) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">T<span class="sc">orgerson</span> 1952</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Y<span class="sc">oung</span> and H<span class="sc">amer</span> 1987</a>)
 analysis of two data sets in the simulation showed that our method 
describes the true population structure well. The conventional <em>G</em><sub>ST</sub> estimator also worked, despite the larger positive bias and variance (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, bottom).</p></div><div id="sec-11" class="subsection"><h4>The case of Pacific herring:</h4><p id="p-34">We analyzed six geographical samples of the Pacific herring <em>Clupea pallasii</em>
 from spawning areas in Lake Akkeshi (AK), Yudonuma Lake (YD), and Funka
 Bay (FK), which are located off the east coast of Hokkaido in Japan, 
and Obuchinuma Lake (OB), Miyako Bay (MY), and Matsushima Bay (MT), 
located off the northern Pacific coast of Honshu. Hatchery fish, 
released and recaptured in Lake Akkeshi (AKH) and Miyako Bay (MYH), were
 also distinguished from wild fish on the basis of otoliths stained with
 alizarin complexon. A total of 2055 mature individuals were genotyped 
at five microsatellite loci. Allele frequencies are given in 
supplemental Tables S1âS5 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>.
 HWE was satisfied in each sample except at four localities for three 
loci, and the assumption of our metapopulation model was considered to 
be satisfied (supplemental Figure S5). On the basis of the estimates of 
the hyperparameters (supplemental Figure S6), the scale parameter Î¸<sub><em>l</em></sub> and <span class="inline-formula" id="inline-formula-117"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-126.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-126.gif"/></noscript></span></span> were estimated over all subpopulations (<a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>).
</p><div class="table pos-float" id="T3"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/381471/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="https://www.genetics.org/highwire/markup/381471/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label"><strong>TABLE 3</strong></span> <p id="p-35" class="first-child"><strong>Estimated locus-specific scale parameters Î¸ and <span class="inline-formula" id="inline-formula-118"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/T3/embed/tex-math-127.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/T3/embed/tex-math-127.gif"/></noscript></span></span> over subpopulations of the Pacific herring, with standard errors in parentheses</strong></p><div class="sb-div caption-clear"></div></div></div><p id="p-38">We estimated the posterior distributions of <span class="inline-formula" id="inline-formula-120"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-129.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-129.gif"/></noscript></span></span>
 for all sets of subpopulations at all loci. As an example, the 
posterior distributions between FK and MY at the five loci are shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The posterior means of <span class="inline-formula" id="inline-formula-121"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-130.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-130.gif"/></noscript></span></span> varied from 0.0064 to 0.0245, with the 95% credible intervals in parentheses (<a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a>). The <em>P</em>-values in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">5</a> are for the homogeneity contingency test performed with Genepop (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">R<span class="sc">aymond</span> and R<span class="sc">ousset</span> 1995</a>). At all loci, the allelic differences between FK and MY were highly significant (<em>P</em> &lt; 0.0000). We used <span class="inline-formula" id="inline-formula-122"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-131.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-131.gif"/></noscript></span></span> of 0.01 as a population criterion and defined the probability of <span class="inline-formula" id="inline-formula-123"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-132.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-132.gif"/></noscript></span></span> â¤ 0.01 as <em>P</em>*. Here, <span class="inline-formula" id="inline-formula-124"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-133.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-133.gif"/></noscript></span></span> = 0.01 refers to <span class="inline-formula" id="inline-formula-125"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-134.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-134.gif"/></noscript></span></span> which means that the effective number of migrants is 25 individuals per generation (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">W<span class="sc">aples</span> and G<span class="sc">aggiotti</span> 2006</a>), where <em>N</em><sub>e</sub> is the effective population size and <em>m</em> is the migration rate. The <em>P</em>*-value was near 1.0 for <em>Cha</em> 63, indicating that the genetic differentiation at the locus was small. In contrast, <em>P</em>*-values were 0 or near 0 for <em>Cha</em> 17, <em>Cha</em> 113, and <em>Cha</em> 123 and 0.5318 for <em>Cha</em> 20 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). The posterior distribution of the average <span class="inline-formula" id="inline-formula-126"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-135.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-135.gif"/></noscript></span></span> over all the loci was calculated as the mean of the posterior distributions at five loci (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>, bottom right). Both the <em>P-</em> and <em>P</em>*-values coincided at 0, showing significant genetic differentiation between Funka Bay and Miyako Bay.


</p><div id="F4" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior distributions of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} for the Pacific herring between Funka Bay and Miyako Bay at each locus, and over all loci, which were averaged over \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} at five loci." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Posterior distributions of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} for the Pacific herring between Funka Bay and Miyako Bay at each locus, and over all loci, which were averaged over \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} at five loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4.â" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F4.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="Figure 4.&#x2014;" src="https://www.genetics.org/content/genetics/177/2/861/F4.medium.gif"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4.â" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381452" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">F<span class="sc">igure</span> 4.â</span> <p id="p-39" class="first-child">Posterior distributions of <span class="inline-formula" id="inline-formula-127"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F4/embed/tex-math-136.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F4/embed/tex-math-136.gif"/></noscript></span></span> for the Pacific herring between Funka Bay and Miyako Bay at each locus, and over all loci, which were averaged over <span class="inline-formula" id="inline-formula-128"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F4/embed/tex-math-137.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F4/embed/tex-math-137.gif"/></noscript></span></span> at five loci.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Posterior distributions of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} for the Pacific herring over all loci, which were averaged over \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} for five loci: AK, Lake Akkeshi; YD, Yudonuma Lake; FK, Funka Bay; OB, Obuchinuma Lake; MY, Miyako Bay; and MT, Matsushima Bay." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Posterior distributions of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} for the Pacific herring over all loci, which were averaged over \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{P}}\) \end{document} for five loci: AK, Lake Akkeshi; YD, Yudonuma Lake; FK, Funka Bay; OB, Obuchinuma Lake; MY, Miyako Bay; and MT, Matsushima Bay.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5.â" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F5.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="Figure 5.&#x2014;" src="https://www.genetics.org/content/genetics/177/2/861/F5.medium.gif"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5.â" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381456" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">F<span class="sc">igure</span> 5.â</span> <p id="p-40" class="first-child">Posterior distributions of <span class="inline-formula" id="inline-formula-129"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F5/embed/tex-math-138.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F5/embed/tex-math-138.gif"/></noscript></span></span> for the Pacific herring over all loci, which were averaged over <span class="inline-formula" id="inline-formula-130"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F5/embed/tex-math-139.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F5/embed/tex-math-139.gif"/></noscript></span></span> for five loci: AK, Lake Akkeshi; YD, Yudonuma Lake; FK, Funka Bay; OB, Obuchinuma Lake; MY, Miyako Bay; and MT, Matsushima Bay.</p><div class="sb-div caption-clear"></div></div></div><div class="table pos-float" id="T4"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline highwireTablesMarkupProcessor-processed" data-table-url="/highwire/markup/381475/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="https://www.genetics.org/highwire/markup/381475/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup init-colorbox-processed cboxElement" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label"><strong>TABLE 4</strong></span> <p id="p-41" class="first-child"><strong>Posterior means and 95% credible and confidence intervals for pairwise <em>F</em><sub>ST</sub> of the Pacific herring between Funka Bay and Miyako Bay at each locus</strong></p><div class="sb-div caption-clear"></div></div></div><p id="p-43">The posterior distributions of average <span class="inline-formula" id="inline-formula-131"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-140.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-140.gif"/></noscript></span></span>
 over all loci for all pairs of wild subpopulations were also simulated 
as simple means of the posterior distributions at five loci (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). The allelic differences were highly significant with <em>P</em> = 0.0000, except between MY and MT. The criterion <em>P</em>* resulted in a very different evaluation of population differentiation, even for the same <em>P</em>-value
 of 0.0000. This result clearly shows the difficulty in the 
hypothesis-testing framework in evaluating the genetic differentiation 
between subpopulations (<em>e.g</em>., <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">D<span class="sc">izon</span> <em>et al</em>. 1995</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">R<span class="sc">yman</span> and J<span class="sc">orde</span> 2001</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">R<span class="sc">yman</span> <em>et al</em>. 2006</a>). The <em>P</em>*-based criterion works well in this case and is recommended for use in hypothesis testing.</p><p id="p-44">The variation in <span class="inline-formula" id="inline-formula-132"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-141.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-141.gif"/></noscript></span></span> over five loci was not trivial, with a coefficient of variation (CV) of 23% (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). However, the negative twice log-likelihood ratio Î» was calculated to be 6.9264 and the hypothesis of constant <span class="inline-formula" id="inline-formula-133"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-142.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-142.gif"/></noscript></span></span> for all loci was not rejected (<em>P</em> = 0.8602, d.f. = 4). We estimated the distribution of <span class="inline-formula" id="inline-formula-134"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-143.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-143.gif"/></noscript></span></span> in the genome, assuming normality based on <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation 6</a>.
 The maximum-likelihood estimates (MLEs) for Î¼ and Ï were obtained with 
90, 95, and 99% confidence intervals, which define the distribution of <span class="inline-formula" id="inline-formula-135"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-144.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-144.gif"/></noscript></span></span> in the genome via the likelihood profile (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). The MLE for Î¼, with the 95% confidence interval, was 0.0160 (0.0128, 0.0206) [<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a> (a, e)] and for Ï was 0 (0, 0.00716) [<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6A</a> (0, c)]. The weighted mean <span class="inline-formula" id="inline-formula-136"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-145.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-145.gif"/></noscript></span></span> coincided with the MLE of Î¼ (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>) and the distribution of the weighted mean <span class="inline-formula" id="inline-formula-137"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-146.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-146.gif"/></noscript></span></span> shown as the blue line in <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6B</a>, described well the 95% confidence interval of Î¼ [<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6A</a> (a, e)]. Here, <span class="inline-formula" id="inline-formula-138"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-147.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-147.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-139"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-148.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-148.gif"/></noscript></span></span> were estimated with <a id="xref-disp-formula-5-2" class="xref-disp-formula" href="#disp-formula-5">Equation 3</a>.
 The log-likelihood profile for Ï was monotonic and decreasing, 
indicating that the point estimate of Ï was 0 (supplemental Figure S7 at
 <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>). Hence, the point estimate of the distribution of <span class="inline-formula" id="inline-formula-140"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-149.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-149.gif"/></noscript></span></span> for the Pacific herring is constant, supporting the hypothesis of constant <span class="inline-formula" id="inline-formula-141"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-150.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-150.gif"/></noscript></span></span> throughout the genome. However, the distribution of <span class="inline-formula" id="inline-formula-142"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-151.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-151.gif"/></noscript></span></span> has uncertainty, which accounts for the confidence regions of Î¼ and Ï (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6B</a>).
</p><div id="F6" class="fig pos-float odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confidence regions of the distribution of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}\) \end{document} in the genome of the Pacific herring, assuming a normal distribution (see the text). (A) The confidence regions of Î¼ and Ï2, which specify the distribution. (B) The MLE distribution (red line, the delta distribution) and the representative distributions on the boundary of the confidence regions (aâe), which correspond to the points in A. The distribution of the weighted mean of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \({\widehat{F_{\mathrm{ST}}^{\mathrm{G}}}}\) \end{document} is superimposed (blue line)." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confidence regions of the distribution of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}\) \end{document} in the genome of the Pacific herring, assuming a normal distribution (see the text). (A) The confidence regions of Î¼ and Ï2, which specify the distribution. (B) The MLE distribution (red line, the delta distribution) and the representative distributions on the boundary of the confidence regions (aâe), which correspond to the points in A. The distribution of the weighted mean of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \({\widehat{F_{\mathrm{ST}}^{\mathrm{G}}}}\) \end{document} is superimposed (blue line).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-6.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-6.medium.gif"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381460" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-inline-img"><a href="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Confidence regions of the distribution of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}\) \end{document} in the genome of the Pacific herring, assuming a normal distribution (see the text). (A) The confidence regions of Î¼ and Ï2, which specify the distribution. (B) The MLE distribution (red line, the delta distribution) and the representative distributions on the boundary of the confidence regions (aâe), which correspond to the points in A. The distribution of the weighted mean of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \({\widehat{F_{\mathrm{ST}}^{\mathrm{G}}}}\) \end{document} is superimposed (blue line)." class="highwire-fragment fragment-images colorbox-load highwireFiguresMarkupProcessor-processed cboxElement" rel="gallery-fragment-images-240626172" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Confidence regions of the distribution of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \(F_{\mathrm{ST}}^{\mathrm{G}}\) \end{document} in the genome of the Pacific herring, assuming a normal distribution (see the text). (A) The confidence regions of Î¼ and Ï2, which specify the distribution. (B) The MLE distribution (red line, the delta distribution) and the representative distributions on the boundary of the confidence regions (aâe), which correspond to the points in A. The distribution of the weighted mean of \batchmode \documentclass[fleqn,10pt,legalpaper]{article} \usepackage{amssymb} \usepackage{amsfonts} \usepackage{amsmath} \pagestyle{empty} \begin{document} \({\widehat{F_{\mathrm{ST}}^{\mathrm{G}}}}\) \end{document} is superimposed (blue line).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-7.medium.gif"><noscript><img class="highwire-fragment fragment-image" alt="" src="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-7.medium.gif"/></noscript></span></a><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download " data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.genetics.org/content/genetics/177/2/861/F6/graphic-7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="https://www.genetics.org/highwire/powerpoint/381460" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div></div></div><div class="fig-caption"><span class="fig-label">F<span class="sc">igure</span> 6.â</span> <p id="p-45" class="first-child">Confidence regions of the distribution of <span class="inline-formula" id="inline-formula-143"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F6/embed/tex-math-152.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F6/embed/tex-math-152.gif"/></noscript></span></span> in the genome of the Pacific herring, assuming a normal distribution (see the text). (A) The confidence regions of Î¼ and Ï<sup>2</sup>,
 which specify the distribution. (B) The MLE distribution (red line, the
 delta distribution) and the representative distributions on the 
boundary of the confidence regions (aâe), which correspond to the points
 in A. The distribution of the weighted mean of <span class="inline-formula" id="inline-formula-144"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F6/embed/tex-math-153.gif"><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/F6/embed/tex-math-153.gif"/></noscript></span></span> is superimposed (blue line).</p><div class="sb-div caption-clear"></div></div></div></div></div></div><div class="section" id="sec-12"><h2>DISCUSSION</h2><p id="p-46">Our empirical Bayes estimator of <span class="inline-formula" id="inline-formula-145"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-154.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-154.gif"/></noscript></span></span> performed better than the conventional <em>G</em><sub>ST</sub> estimator for various levels of <span class="inline-formula" id="inline-formula-146"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-155.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-155.gif"/></noscript></span></span>
 and various sampling conditions under the infinite-island model. Even 
for non-Dirichlet distributions of allele frequencies, such as 
stepping-stone models, our method provided better estimates of <span class="inline-formula" id="inline-formula-147"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-156.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-156.gif"/></noscript></span></span> than did the conventional <em>G</em><sub>ST</sub>, especially for cases with large gene flow.</p><div id="sec-13" class="subsection"><div id="sec-14" class="subsection"><h4>Integrated likelihood method:</h4><p id="p-47">The empirical Bayes estimator of <span class="inline-formula" id="inline-formula-148"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-157.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-157.gif"/></noscript></span></span>'s
 is negatively biased, especially when the population has large gene 
flow and the estimation is based on a sample from only a few sampling 
points (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, supplemental Figure S2 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>).
 With a small number of sampling points, the MLE of Î¸ is not precise. 
Therefore, it is recommended in the conservation genetic analysis of 
such a population that the samples be collected from as many sampling 
points as possible. When sampling from many localities is not feasible, 
the integrated-likelihood method (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">K<span class="sc">itakado</span> <em>et al</em>. 2006</a>) can reduce the negative bias of <span class="inline-formula" id="inline-formula-149"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-158.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-158.gif"/></noscript></span></span> We estimated pairwise <span class="inline-formula" id="inline-formula-150"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-159.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-159.gif"/></noscript></span></span>'s by the empirical Bayes method based on integrated-likelihood estimates (ILEs) of Î¸ for cases with <span class="inline-formula" id="inline-formula-151"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-160.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-160.gif"/></noscript></span></span> = 0.01, <em>K</em> = 5, and <em>N<sub>k</sub></em>/2 = 20, 30, or 50 with the same simulation protocol used for <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> (supplemental Figure S8 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>). The relative mean biases of the <span class="inline-formula" id="inline-formula-152"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-161.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-161.gif"/></noscript></span></span>
 estimates, with root relative mean squared errors in parentheses, were 
â0.238 (0.506), â0.132 (0.365), and â0.060 (0.272), respectively. These 
values were much smaller than those estimated on the basis of the MLE of
 Î¸, given in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> (top three rows). The negative bias of <span class="inline-formula" id="inline-formula-153"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-162.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-162.gif"/></noscript></span></span>'s based on the MLE was reduced to 32.8, 24.4, and 18.2% for <em>N<sub>k</sub></em>/2 = 20, 30, and 50, respectively. The integrated-likelihood method uses a uniform prior for the mean allele frequency Î²<sub><em>l</em></sub> and eliminates Î²<sub><em>l</em></sub>
 by integration regarding it as a nuisance parameter. By using the ILE 
of Î¸ instead of the MLE, the empirical Bayes method proposed here 
provides nearly unbiased estimates of <span class="inline-formula" id="inline-formula-154"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-163.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-163.gif"/></noscript></span></span>'s when the sample sizes (<em>N<sub>k</sub></em>/2
 individuals) are large and works more efficiently, even for cases with a
 small number of sampling points (supplemental Figure S8).</p></div><div id="sec-15" class="subsection"><h4>Weir and Cockerham's Î¸Ì<sub>WC</sub>:</h4><p id="p-48">When
 we estimate genetic differentiation between two specific 
subpopulations, selected subpopulations can be regarded as the entire 
set of populations. Nei's <em>G</em><sub>ST</sub> formula defines quantities with respect to fixed extant populations (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">C<span class="sc">ockerham</span> and W<span class="sc">eir</span> 1986</a>). In addition, <em>G</em><sub>ST</sub>
 is a function of allele frequencies under HWE, and the posterior 
distribution can easily be simulated from only allele frequencies. 
Therefore, we used the <em>G</em><sub>ST</sub> estimator to estimate the posterior distribution of pairwise <em>F</em><sub>ST</sub>.</p><p id="p-49">The citation record suggests that the most widely used estimator for Wright's <em>F</em><sub>ST</sub> is Weir and Cockerham's <span class="inline-formula" id="inline-formula-155"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-164.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-164.gif"/></noscript></span></span> (<a id="xref-ref-54-4" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> 2002</a>). This moment estimator takes the sampling error for subpopulations into account and essentially estimates the global <em>F</em><sub>ST</sub>, <span class="inline-formula" id="inline-formula-156"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-165.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-165.gif"/></noscript></span></span> The estimator is also widely used to estimate pairwise <em>F</em><sub>ST</sub>'s among fixed pairs of populations. With the assumption of no local inbreeding, Î¸<sub>WC</sub> is estimated only from sample allele frequencies, but these need to be inferred from sample genotype frequencies (<a id="xref-ref-54-5" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> 2002</a>). With <em>J</em> alleles at a locus, the number of possible genotypes is <em>J</em>(<em>J</em> + 1)/2. In microsatellite DNA analyses, <em>J</em> is generally large. If <em>J</em>
 = 50, the number of genotypes is 1225. Such a situation makes our 
simulation more complicated and the uncertainty of the genotype counts 
becomes large under small or moderate sample sizes.</p><p id="p-50">Here, we investigated the properties of <span class="inline-formula" id="inline-formula-157"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-166.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-166.gif"/></noscript></span></span> on estimating pairwise <em>F</em><sub>ST</sub> on the basis of the relationship between <em>G</em><sub>ST</sub> and Î¸<sub>WC</sub>. Weir and Cockerham's estimates <span class="inline-formula" id="inline-formula-158"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-167.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-167.gif"/></noscript></span></span> can be approximated as a function of <em>G</em><sub>ST</sub> by <a id="xref-disp-formula-4-2" class="xref-disp-formula" href="#disp-formula-4">Equation 2</a> in <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">W<span class="sc">eicker</span> <em>et al</em>. (2001)</a>: <span class="inline-formula" id="inline-formula-159"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-168.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-168.gif"/></noscript></span></span> Using this equation, we calculated the conventional estimates of pairwise Weir and Cockerham's Î¸<sub>WC</sub> from <em>G</em><sub>ST</sub> estimates with the simulation protocol described in <span class="sc">results</span>. We examined cases with a global <em>F</em><sub>ST</sub> of <span class="inline-formula" id="inline-formula-160"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-169.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-169.gif"/></noscript></span></span> = 0.01, 0.05, 0.1, and 0.2. The mean allele frequencies were assumed <span class="inline-formula" id="inline-formula-161"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-170.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-170.gif"/></noscript></span></span> with <em>J</em> = 50. The number of sampling localities <em>K</em> was set at 10 and the sample size <em>N<sub>k</sub></em>/2
 (individuals) was deemed to be common to all the localities and was set
 at 50 individuals. As shown in supplemental Figure S9 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>, the two estimators have linear relationships. Hence, our simulation results on the conventional <em>G</em><sub>ST</sub> estimator can be extended straightforwardly to pairwise Î¸<sub>WC</sub>. In fact, the pairwise <span class="inline-formula" id="inline-formula-162"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-171.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-171.gif"/></noscript></span></span>-values calculated for the Pacific herring with Genepop (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">R<span class="sc">aymond</span> and R<span class="sc">ousset</span> 1995</a>) were 1.93 Â± 0.47 times larger than the posterior means of <span class="inline-formula" id="inline-formula-163"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-172.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-172.gif"/></noscript></span></span> (supplemental Figure S10 at <a href="http://www.genetics.org/supplemental/">http://www.genetics.org/supplemental/</a>), which coincides with our simulations for small <span class="inline-formula" id="inline-formula-164"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-173.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-173.gif"/></noscript></span></span>'s (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, supplemental Figures S1 and S11).</p><p id="p-51">Weir and Cockerham's estimator <span class="inline-formula" id="inline-formula-165"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-174.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-174.gif"/></noscript></span></span> is nearly unbiased (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">R<span class="sc">aufaste</span> and B<span class="sc">onhomme</span> 2000</a>), although it has a negative bias for the two-allele case (<a id="xref-ref-54-6" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> 2002</a>). Nevertheless, when estimating the pairwise <em>F</em><sub>ST</sub>, <span class="inline-formula" id="inline-formula-166"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-175.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-175.gif"/></noscript></span></span> is considered to have a large positive bias, especially for species with large gene flows. Nei's <em>G</em><sub>ST</sub> and Rosenberg <em>et al</em>.'s informativeness of assignment <em>I<sub>n</sub></em>
 also showed the same phenomenon, suggesting the positive bias is 
irrelevant to the estimators. This positive bias of the conventional 
estimators was larger for smaller genetic differentiation. This might be
 caused by the large variation in the sample allele frequencies, which 
is larger for smaller sample sizes (individuals) and largely exceeded 
the real variation between subpopulations. The shrinkage estimator 
stabilizes such variation and provides better estimates.</p></div><div id="sec-16" class="subsection"><h4>Weir and Hill's normal theory:</h4><p id="p-52"><a id="xref-ref-54-7" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill'</span>s (2002)</a> normal theory approach has the same variance as a Dirichlet distribution when <em>i</em> â  <em>i</em>â² in their notation. Hence, their estimator <span class="inline-formula" id="inline-formula-167"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-176.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-176.gif"/></noscript></span></span> is equivalent to our <span class="inline-formula" id="inline-formula-168"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-177.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-177.gif"/></noscript></span></span> <span class="inline-formula" id="inline-formula-169"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-178.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-178.gif"/></noscript></span></span> is the variance of the allele frequencies among subpopulations relative to the total population. Hence, <span class="inline-formula" id="inline-formula-170"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-179.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-179.gif"/></noscript></span></span> refers to a sample variance of allele frequencies, and therefore <span class="inline-formula" id="inline-formula-171"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-180.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-180.gif"/></noscript></span></span> follows a Ï<sup>2</sup>-distribution when the number of sampling points <em>K</em> is sufficiently large, as shown by <a id="xref-ref-54-8" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> (2002)</a>. The shape of the posterior distribution of <span class="inline-formula" id="inline-formula-172"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-181.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-181.gif"/></noscript></span></span> at each locus was unimodal and slightly right tailed, which reminded us of Ï<sup>2</sup>-distributions (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>). We estimated Weir and Hill's confidence intervals by substituting the posterior mean of <span class="inline-formula" id="inline-formula-173"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-182.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-182.gif"/></noscript></span></span> with <span class="inline-formula" id="inline-formula-174"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-183.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-183.gif"/></noscript></span></span> as <span class="inline-formula" id="inline-formula-175"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-184.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-184.gif"/></noscript></span></span> where d.f. = (<em>K</em> â 1)(<em>J</em> â 1) is the degrees of freedom. The confidence intervals of <span class="inline-formula" id="inline-formula-176"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-185.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-185.gif"/></noscript></span></span> obtained with the Ï<sup>2</sup>-approximation
 coincided well with the credible intervals calculated from the 
posterior distributions, although our credible intervals were narrower 
than the confidence intervals of the Ï<sup>2</sup>-approximation, which were slightly right tailed (<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a>). The slight difference in the intervals might have been the effect of the small <em>K</em>(= 8) on the Ï<sup>2</sup>-approximation,
 although it was not substantial. On the contrary, the confidence 
interval for the sample mean over all loci was narrower than our 
credible interval (<a id="xref-table-wrap-4-3" class="xref-table" href="#T4">Table 4</a>).
 The distribution of a sample mean is normal when the sample size is 
large with the variance reduced by the central limit theorem. A Ï<sup>2</sup>-distribution approaches a normal distribution as the degrees of freedom become larger. For our case of 1309 d.f. [<span class="inline-formula" id="inline-formula-177"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-186.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-186.gif"/></noscript></span></span> as given in <a id="xref-ref-54-9" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> 2002</a>],
 the two distributions are equal. The property of the sample mean should
 cause the narrower confidence interval of the normal theory approach. 
The result shows that the posterior distribution of <span class="inline-formula" id="inline-formula-178"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-187.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-187.gif"/></noscript></span></span> describes the distribution of <span class="inline-formula" id="inline-formula-179"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-188.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-188.gif"/></noscript></span></span> well, both for each locus and for the average over all loci.</p></div><div id="sec-17" class="subsection"><h4>LD among loci:</h4><p id="p-53">The case study of the Pacific herring, based on a few microsatellite markers, did not detect significant variation in <span class="inline-formula" id="inline-formula-180"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-189.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-189.gif"/></noscript></span></span> among loci in the genome (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6</a>). The assumption of normality for the MLE of <span class="inline-formula" id="inline-formula-181"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-190.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-190.gif"/></noscript></span></span>
 is valid when more than a few sampling points are surveyed. This 
assumption might be violated when the data are collected from only a few
 sampling points or <span class="inline-formula" id="inline-formula-182"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-191.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-191.gif"/></noscript></span></span> is close to 0 or 1.0. We also assumed the independence of loci <span class="inline-formula" id="inline-formula-183"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-192.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-192.gif"/></noscript></span></span>
 However, it is necessary to take into account the linkage 
disequilibrium (LD) among loci, when the molecular markers are tightly 
linked.</p><p id="p-54">Recent progress in whole-genome analysis of human populations provides a new perspective on the inference of <em>F</em><sub>ST</sub> and its distribution in genomes (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">G<span class="sc">arte</span> 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">H<span class="sc">inds</span> <em>et al</em>. 2005</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">W<span class="sc">eir</span> <em>et al</em>. 2005</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">W<span class="sc">alsh</span> <em>et al</em>. 2006</a>). In their <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">W<span class="sc">eir</span> <em>et al</em>. (2005)</a> showed that values for the single-locus marker <span class="inline-formula" id="inline-formula-184"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-193.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-193.gif"/></noscript></span></span> over the whole human genome for three (Perlegen) or four (HapMap) populations had a distribution very much like the Ï<sup>2</sup>-distributions with 2 or 3 d.f. and suggested that values of <span class="inline-formula" id="inline-formula-185"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-194.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-194.gif"/></noscript></span></span> are genome-region specific. However, <span class="inline-formula" id="inline-formula-186"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-195.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-195.gif"/></noscript></span></span> follows a Ï<sup>2</sup>-distribution under constant <span class="inline-formula" id="inline-formula-187"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-196.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-196.gif"/></noscript></span></span> as reported by <a id="xref-ref-54-10" class="xref-bibr" href="#ref-54">W<span class="sc">eir</span> and H<span class="sc">ill</span> (2002)</a>
 and as demonstrated in our analysis of the Pacific herring. Therefore, 
the distributions of the values for the single-locus marker <span class="inline-formula" id="inline-formula-188"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-197.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-197.gif"/></noscript></span></span> in <a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">W<span class="sc">eir</span> <em>et al</em>. (2005)</a> do not necessarily support the genome-region-specific <em>F</em><sub>ST</sub> hypothesis in the human genome.</p><p id="p-55">Weir <em>et al</em>.'s 5-Mb window average values for <span class="inline-formula" id="inline-formula-189"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-198.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-198.gif"/></noscript></span></span>
 were close to normal because of the property of the sample mean. The 
standard deviations (SDs) of the 5-Mb window average values of <span class="inline-formula" id="inline-formula-190"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-199.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-199.gif"/></noscript></span></span> decreased substantially from single-locus estimates of 0.12 to 0.02 for HapMap and from 0.11 to 0.02 for Perlegen (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a> and <a id="xref-table-wrap-3-4" class="xref-table" href="#T3">3</a> in <a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">W<span class="sc">eir</span> <em>et al</em>. 2005</a>). The average number of markers in a 5-Mb window was 1114 for HapMap and 1834 for Perlegen (calculated from <a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> in <a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">W<span class="sc">eir</span> <em>et al</em>. 2005</a>). Hence, these SDs are expected to be <span class="inline-formula" id="inline-formula-191"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-200.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-200.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-192"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-201.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-201.gif"/></noscript></span></span> if all SNPs are independent and distributed identically in the genome. The effective sample size (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">K<span class="sc">ish</span> 1965</a>, p. 259) could be much smaller than the real number of SNPs in the 5-Mb windows if the <span class="inline-formula" id="inline-formula-193"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-202.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-202.gif"/></noscript></span></span>'s
 are correlated, because of LD between the SNPs. A coalescent simulation
 of human population history implies that linkage equilibrium holds for 
SNPs separated by &gt;10â100 kb (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> in <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">K<span class="sc">ruglyak</span> 1999</a>).
 Therefore, we can estimate the effective number of human SNPs per 
100-kb window to be â¼1â20. If we assumed it to be â¼10, the effective 
number of SNPs per 5-Mb window becomes 500. Therefore, the SDs of the 
5-Mb window <span class="inline-formula" id="inline-formula-194"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-203.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-203.gif"/></noscript></span></span> are expected to be <span class="inline-formula" id="inline-formula-195"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-204.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-204.gif"/></noscript></span></span> for HapMap and <span class="inline-formula" id="inline-formula-196"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed tex lazyload" alt="Math" data-src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-205.gif"><noscript><img class="highwire-embed tex" alt="Math" src="https://www.genetics.org/sites/default/files/highwire/genetics/177/2/861/embed/tex-math-205.gif"/></noscript></span></span>
 for Perlegen. The actual value (0.02) is much larger than these, even 
when the LD among the SNPs is taken into account. This discrepancy can 
be explained by the large-scale heterogeneity of <em>F</em><sub>ST</sub>
 between the 5-Mb windows. New data show that LD is highly structured 
into discrete blocks of sequences separated by hot spots of 
recombination (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">G<span class="sc">oldstein</span> 2001</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">M<span class="sc">c</span>V<span class="sc">ean</span> <em>et al</em>. 2004</a>) and differs among species (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">H<span class="sc">ernandez</span> <em>et al</em>. 2007</a>). The simultaneous estimation of <em>F</em><sub>ST</sub>'s of SNPs and the LD between the SNPs should give us an accurate picture of the distribution of <em>F</em><sub>ST</sub> in genomes.</p></div></div></div><div class="section ack" id="ack-1"><h2>Acknowledgments</h2><p id="p-57">We
 are grateful to Bruce Weir for valuable comments on the manuscript and 
encouragement with this work. We also thank the anonymous reviewers for 
their constructive comments, which improved the earlier version of the 
manuscript. This work was supported by grants from the Japan Society for
 the Promotion of Science.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-other fn-group-arthw-fm" id="fn-5"><p id="p-56">Communicating editor: R. W. D<span class="sc">oerge</span></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2007-06-11"><span class="received-label">Received </span>June 11, 2007.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2007-07-17"><span class="accepted-label">Accepted </span>July 17, 2007.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2007 by the Genetics Society of America</li></ul><div class="section ref-list" id="ref-list-1"><h2>References</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.1" data-doi="10.1016/S0040-5809(03)00007-8"><div class="cit-metadata"><cite>B<span class="sc">alding</span>, D. J., <span class="cit-pub-date">2003</span> Likelihood-based inference for genetic correlation coefficients. <span class="cit-source">Theor. Popul. Biol.</span> <span class="cit-vol">63</span><strong>:</strong> <span class="cit-fpage">221</span>â230.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DBalding%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D63%26rft.issue%253D3%26rft.spage%253D221%26rft.epage%253D230%26rft.atitle%253DLikelihood-based%2Binference%2Bfor%2Bgenetic%2Bcorrelation%2Bcoefficients.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0040-5809%252803%252900007-8%26rft_id%253Dinfo%253Apmid%252F12689793%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1016/S0040-5809(03)00007-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=12689793&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000182420200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Balding%2C+D.+J.%2C+2003+Likelihood-based+inference+for+genetic+correlation+coefficients.+Theor.+Popul.+Biol.+63%3A+221%E2%80%93230." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.2" data-doi="10.1038/sj.hdy.6881750"><div class="cit-metadata"><cite>B<span class="sc">alding</span>, D. J., and R. A. N<span class="sc">ichols</span>, <span class="cit-pub-date">1997</span> Significant genetic correlations among Caucasians at forensic DNA loci. <span class="cit-source">Heredity</span> <span class="cit-vol">78</span><strong>:</strong> <span class="cit-fpage">583</span>â589.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DHeredity%26rft.volume%253D78%26rft.spage%253D583%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.hdy.6881750%26rft_id%253Dinfo%253Apmid%252F9203353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1038/sj.hdy.6881750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=9203353&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1997XE87300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Balding%2C+D.+J.%2C+and+R.+A.+Nichols%2C+1997+Significant+genetic+correlations+among+Caucasians+at+forensic+DNA+loci.+Heredity+78%3A+583%E2%80%93589." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.3" data-doi="10.1046/j.0962-1083.2001.01436.x"><div class="cit-metadata"><cite>B<span class="sc">alloux</span>, F., and N. L<span class="sc">ugon</span>-M<span class="sc">oulin</span>, <span class="cit-pub-date">2002</span> The estimation of population differentiation with microsatellite markers. <span class="cit-source">Mol. Ecol.</span> <span class="cit-vol">11</span><strong>:</strong> <span class="cit-fpage">155</span>â165.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBalloux%26rft.auinit1%253DF.%26rft.volume%253D11%26rft.issue%253D2%26rft.spage%253D155%26rft.epage%253D165%26rft.atitle%253DThe%2Bestimation%2Bof%2Bpopulation%2Bdifferentiation%2Bwith%2Bmicrosatellite%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.0962-1083.2001.01436.x%26rft_id%253Dinfo%253Apmid%252F11856418%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1046/j.0962-1083.2001.01436.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=11856418&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Balloux%2C+F.%2C+and+N.+Lugon-Moulin%2C+2002+The+estimation+of+population+differentiation+with+microsatellite+markers.+Mol.+Ecol.+11%3A+155%E2%80%93165." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.4" data-doi="10.2307/2406485"><div class="cit-metadata"><cite>C<span class="sc">ockerham</span>, C. C., <span class="cit-pub-date">1969</span> Variance of gene frequencies. <span class="cit-source">Evolution</span> <span class="cit-vol">23</span><strong>:</strong> <span class="cit-fpage">72</span>â83.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DEvolution%26rft.volume%253D23%26rft.spage%253D72%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2406485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.2307/2406485&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1969D082300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Cockerham%2C+C.+C.%2C+1969+Variance+of+gene+frequencies.+Evolution+23%3A+72%E2%80%9383." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.5"><div class="cit-metadata"><cite>C<span class="sc">ockerham</span>, C. C., <span class="cit-pub-date">1973</span> Analysis of gene frequencies. <span class="cit-source">Genetics</span> <span class="cit-vol">74</span><strong>:</strong> <span class="cit-fpage">679</span>â700.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D74%26rft.issue%253D4%26rft.spage%253D679%26rft.epage%253D700%26rft.atitle%253DANALYSES%2BOF%2BGENE%2BFREQUENCIES%26rft_id%253Dinfo%253Apmid%252F17248636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvNC82NzkiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Cockerham%2C+C.+C.%2C+1973+Analysis+of+gene+frequencies.+Genetics+74%3A+679%E2%80%93700." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.6"><div class="cit-metadata"><cite>C<span class="sc">ockerham</span>, C. C., and B. S. W<span class="sc">eir</span>, <span class="cit-pub-date">1986</span> Estimation of inbreeding parameters in stratified populations. <span class="cit-source">Ann. Hum. Genet.</span> <span class="cit-vol">50</span><strong>:</strong> <span class="cit-fpage">271</span>â281.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DCockerham%26rft.auinit1%253DC.%2BC.%26rft.volume%253D50%26rft.issue%253DPt%2B3%26rft.spage%253D271%26rft.epage%253D281%26rft.atitle%253DEstimation%2Bof%2Binbreeding%2Bparameters%2Bin%2Bstratified%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F3446014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=3446014&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1986D598000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Cockerham%2C+C.+C.%2C+and+B.+S.+Weir%2C+1986+Estimation+of+inbreeding+parameters+in+stratified+populations.+Ann.+Hum.+Genet.+50%3A+271%E2%80%93281." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.7"><div class="cit-metadata"><cite>C<span class="sc">orander</span>, J., P. W<span class="sc">aldmann</span> and M. J. S<span class="sc">illanpÃ¤Ã¤</span>, <span class="cit-pub-date">2003</span> Bayesian analysis of genetic differentiation between populations. <span class="cit-source">Genetics</span> <span class="cit-vol">163</span><strong>:</strong> <span class="cit-fpage">367</span>â374.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DCorander%26rft.auinit1%253DJ.%26rft.volume%253D163%26rft.issue%253D1%26rft.spage%253D367%26rft.epage%253D374%26rft.atitle%253DBayesian%2BAnalysis%2Bof%2BGenetic%2BDifferentiation%2BBetween%2BPopulations%26rft_id%253Dinfo%253Apmid%252F12586722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzEvMzY3IjtzOjQ6ImF0b20iO3M6MjQ6Ii9nZW5ldGljcy8xNzcvMi84NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Corander%2C+J.%2C+P.+Waldmann+and+M.+J.+Sillanp%C3%A4%C3%A4%2C+2003+Bayesian+analysis+of+genetic+differentiation+between+populations.+Genetics+163%3A+367%E2%80%93374." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.8"><div class="cit-metadata"><cite>D<span class="sc">izon</span>, A. E., B. L. T<span class="sc">aylor</span> and G. M. O'C<span class="sc">orry</span>-C<span class="sc">rowe</span>, <span class="cit-pub-date">1995</span>
 Why statistical power is necessary to link analyses of molecular 
variation to decisions about population structure, pp. 288â294 in <em>Evolution and the Aquatic Ecosystem</em>, edited by J. L. N<span class="sc">ielsen</span> and D. A. P<span class="sc">owers</span>. American Fisheries Society Symposium 17, Bethesda, MD.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Dizon%2C+A.+E.%2C+B.+L.+Taylor+and+G.+M.+O%27Corry-Crowe%2C+1995+Why+statistical+power+is+necessary+to+link+analyses+of+molecular+variation+to+decisions+about+population+structure%2C+pp.+288%E2%80%93294+in+Evolution+and+the+Aquatic+Ecosystem%2C+edited+by+J.+L.+Nielsen+and+D.+A.+Powers.+American+Fisheries+Society+Symposium+17%2C+Bethesda%2C+MD." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.9"><div class="cit-metadata"><cite>E<span class="sc">wens</span>, W. J., <span class="cit-pub-date">2004</span> <em>Mathematical Population Genetics, I. Theoretical Introduction</em>, Ed. 2. Springer, Berlin/Heidelberg, Germany/New York.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Ewens%2C+W.+J.%2C+2004+Mathematical+Population+Genetics%2C+I.+Theoretical+Introduction%2C+Ed.+2.+Springer%2C+Berlin%2FHeidelberg%2C+Germany%2FNew+York." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.10"><div class="cit-metadata"><cite>E<span class="sc">xcoffier</span>, L., <span class="cit-pub-date">2003</span> Analysis of population subdivision, pp. 713â750 in <em>Handbook of Statistical Genetics</em>, Ed. 2, edited by D. J. B<span class="sc">alding</span>, M. B<span class="sc">ishop</span> and C. C<span class="sc">annings</span>. Wiley, Chichester, UK.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Excoffier%2C+L.%2C+2003+Analysis+of+population+subdivision%2C+pp.+713%E2%80%93750+in+Handbook+of+Statistical+Genetics%2C+Ed.+2%2C+edited+by+D.+J.+Balding%2C+M.+Bishop+and+C.+Cannings.+Wiley%2C+Chichester%2C+UK." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.11"><div class="cit-metadata"><cite>E<span class="sc">xcoffier</span>, L., G. L<span class="sc">aval</span> and S. S<span class="sc">chneider</span>, <span class="cit-pub-date">2005</span> Arlequin ver. 3.0: an integrated software package for population genetics data analysis. <span class="cit-source">Evol. Bioinform. Online</span> <span class="cit-vol">1</span><strong>:</strong> <span class="cit-fpage">47</span>â50.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DEvol.%2BBioinform.%2BOnline%26rft.volume%253D1%26rft.spage%253D47%26rft_id%253Dinfo%253Apmid%252F19325852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=19325852&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Excoffier%2C+L.%2C+G.+Laval+and+S.+Schneider%2C+2005+Arlequin+ver.+3.0%3A+an+integrated+software+package+for+population+genetics+data+analysis.+Evol.+Bioinform.+Online+1%3A+47%E2%80%9350." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.12" data-doi="10.1002/ajhb.10215"><div class="cit-metadata"><cite>G<span class="sc">arte</span>, S., <span class="cit-pub-date">2003</span> Locus-specific genetic diversity between human populations: an analysis of the literature. <span class="cit-source">Am. J. Hum. Biol.</span> <span class="cit-vol">15</span><strong>:</strong> <span class="cit-fpage">814</span>â823.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bbiology%2B%253A%2B%2Bthe%2Bofficial%2Bjournal%2Bof%2Bthe%2BHuman%2BBiology%2BCouncil%26rft.stitle%253DAm%2BJ%2BHum%2BBiol%26rft.aulast%253DGarte%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D814%26rft.epage%253D823%26rft.atitle%253DLocus-specific%2Bgenetic%2Bdiversity%2Bbetween%2Bhuman%2Bpopulations%253A%2Ban%2Banalysis%2Bof%2Bthe%2Bliterature.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fajhb.10215%26rft_id%253Dinfo%253Apmid%252F14595873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1002/ajhb.10215&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=14595873&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000186302700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Garte%2C+S.%2C+2003+Locus-specific+genetic+diversity+between+human+populations%3A+an+analysis+of+the+literature.+Am.+J.+Hum.+Biol.+15%3A+814%E2%80%93823." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.13" data-doi="10.1038/ng1001-109"><div class="cit-metadata"><cite>G<span class="sc">oldstein</span>, D. B., <span class="cit-pub-date">2001</span> Islands of linkage disequilibrium. <span class="cit-source">Nat. Genet.</span> <span class="cit-vol">29</span><strong>:</strong> <span class="cit-fpage">109</span>â111.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DGoldstein%26rft.auinit1%253DD.%2BB.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D109%26rft.epage%253D111%26rft.atitle%253DIslands%2Bof%2Blinkage%2Bdisequilibrium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1001-109%26rft_id%253Dinfo%253Apmid%252F11586289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1038/ng1001-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=11586289&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000171374100007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Goldstein%2C+D.+B.%2C+2001+Islands+of+linkage+disequilibrium.+Nat.+Genet.+29%3A+109%E2%80%93111." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.14"><div class="cit-metadata"><cite>G<span class="sc">oudet</span>, J., <span class="cit-pub-date">1995</span> FSTAT (version 1.2): a computer program to calculate F-statistics. <span class="cit-source">J. Hered.</span> <span class="cit-vol">86</span><strong>:</strong> <span class="cit-fpage">485</span>â486.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.issn%253D0022-1503%26rft.aulast%253DGoudet%26rft.auinit1%253DJ.%26rft.volume%253D86%26rft.issue%253D6%26rft.spage%253D485%26rft.epage%253D486%26rft.atitle%253DFSTAT%2B%2528Version%2B1.2%2529%253A%2BA%2BComputer%2BProgram%2Bto%2BCalculate%2BF-Statistics%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6ODoiODYvNi80ODUiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Goudet%2C+J.%2C+1995+FSTAT+(version+1.2)%3A+a+computer+program+to+calculate+F-statistics.+J.+Hered.+86%3A+485%E2%80%93486." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.15"><div class="cit-metadata"><cite>G<span class="sc">oudet</span>, J., M. R<span class="sc">aymond</span>, T. <span class="sc">de</span> M<span class="sc">eeÃ¼s</span> and F. R<span class="sc">ousset</span>, <span class="cit-pub-date">1996</span> Testing differentiation in diploid populations. <span class="cit-source">Genetics</span> <span class="cit-vol">144</span><strong>:</strong> <span class="cit-fpage">1933</span>â1940.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DGoudet%26rft.auinit1%253DJ.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1933%26rft.epage%253D1940%26rft.atitle%253DTesting%2BDifferentiation%2Bin%2BDiploid%2BPopulations%26rft_id%253Dinfo%253Apmid%252F8978076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzE5MzMiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Goudet%2C+J.%2C+M.+Raymond%2C+T.+de+Mee%C3%BCs+and+F.+Rousset%2C+1996+Testing+differentiation+in+diploid+populations.+Genetics+144%3A+1933%E2%80%931940." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.16"><div class="cit-metadata"><cite>H<span class="sc">anski</span>, I., and O. E. G<span class="sc">aggiotti</span>, <span class="cit-pub-date">2004</span> <em>Ecology, Genetics, and Evolution of Metapopulations</em>. Academic Press, San Diego.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Hanski%2C+I.%2C+and+O.+E.+Gaggiotti%2C+2004+Ecology%2C+Genetics%2C+and+Evolution+of+Metapopulations.+Academic+Press%2C+San+Diego." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.17" data-doi="10.1126/science.1140462"><div class="cit-metadata"><cite>H<span class="sc">ernandez</span>, R. D., M. J. H<span class="sc">ubisz</span>, D. A. W<span class="sc">heeler</span>, D. G. S<span class="sc">mith</span>, B. F<span class="sc">erguson</span> <em>et al</em>., <span class="cit-pub-date">2007</span> Demographic histories and patterns of linkage disequilibrium in Chinese and Indian rhesus macaques. <span class="cit-source">Science</span> <span class="cit-vol">316</span><strong>:</strong> <span class="cit-fpage">240</span>â243.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.issn%253D0036-8075%26rft.aulast%253DHernandez%26rft.auinit1%253DR.%2BD.%26rft.volume%253D316%26rft.issue%253D5822%26rft.spage%253D240%26rft.epage%253D243%26rft.atitle%253DDemographic%2BHistories%2Band%2BPatterns%2Bof%2BLinkage%2BDisequilibrium%2Bin%2BChinese%2Band%2BIndian%2BRhesus%2BMacaques%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1140462%26rft_id%253Dinfo%253Apmid%252F17431170%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTYvNTgyMi8yNDAiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Hernandez%2C+R.+D.%2C+M.+J.+Hubisz%2C+D.+A.+Wheeler%2C+D.+G.+Smith%2C+B.+Ferguson+et+al.%2C+2007+Demographic+histories+and+patterns+of+linkage+disequilibrium+in+Chinese+and+Indian+rhesus+macaques.+Science+316%3A+240%E2%80%93243." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.18" data-doi="10.1126/science.1105436"><div class="cit-metadata"><cite>H<span class="sc">inds</span>, D. A., L. L. S<span class="sc">tuve</span>, G. B. N<span class="sc">elsen</span>, E. H<span class="sc">alperin</span>, E. E<span class="sc">skin</span> <em>et al</em>., <span class="cit-pub-date">2005</span> Whole-genome patterns of common DNA variation in three human populations. <span class="cit-source">Science</span> <span class="cit-vol">307</span><strong>:</strong> <span class="cit-fpage">1072</span>â1079.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.issn%253D0036-8075%26rft.aulast%253DHinds%26rft.auinit1%253DD.%2BA.%26rft.volume%253D307%26rft.issue%253D5712%26rft.spage%253D1072%26rft.epage%253D1079%26rft.atitle%253DWhole-Genome%2BPatterns%2Bof%2BCommon%2BDNA%2BVariation%2Bin%2BThree%2BHuman%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1105436%26rft_id%253Dinfo%253Apmid%252F15718463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDcvNTcxMi8xMDcyIjtzOjQ6ImF0b20iO3M6MjQ6Ii9nZW5ldGljcy8xNzcvMi84NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Hinds%2C+D.+A.%2C+L.+L.+Stuve%2C+G.+B.+Nelsen%2C+E.+Halperin%2C+E.+Eskin+et+al.%2C+2005+Whole-genome+patterns+of+common+DNA+variation+in+three+human+populations.+Science+307%3A+1072%E2%80%931079." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.19" data-doi="10.1111/j.1601-5223.1999.00245.x"><div class="cit-metadata"><cite>H<span class="sc">olsinger</span>, K. E., <span class="cit-pub-date">1999</span> Analysis of genetic diversity in geographically structured populations: a Bayesian perspective. <span class="cit-source">Hereditas</span> <span class="cit-vol">130</span><strong>:</strong> <span class="cit-fpage">245</span>â255.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DHereditas%26rft.volume%253D130%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1601-5223.1999.00245.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1111/j.1601-5223.1999.00245.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000082590200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Holsinger%2C+K.+E.%2C+1999+Analysis+of+genetic+diversity+in+geographically+structured+populations%3A+a+Bayesian+perspective.+Hereditas+130%3A+245%E2%80%93255." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.20" data-doi="10.1046/j.1365-294X.2002.01512.x"><div class="cit-metadata"><cite>H<span class="sc">olsinger</span>, K. E., P. O. L<span class="sc">ewis</span> and D. K. D<span class="sc">ey</span>, <span class="cit-pub-date">2002</span> A Bayesian approach to inferring population structure from dominant markers. <span class="cit-source">Mol. Ecol.</span> <span class="cit-vol">11</span><strong>:</strong> <span class="cit-fpage">1157</span>â1164.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DHolsinger%26rft.auinit1%253DK.%2BE.%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D1157%26rft.epage%253D1164%26rft.atitle%253DA%2BBayesian%2Bapproach%2Bto%2Binferring%2Bpopulation%2Bstructure%2Bfrom%2Bdominant%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-294X.2002.01512.x%26rft_id%253Dinfo%253Apmid%252F12074723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1046/j.1365-294X.2002.01512.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=12074723&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000176321300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Holsinger%2C+K.+E.%2C+P.+O.+Lewis+and+D.+K.+Dey%2C+2002+A+Bayesian+approach+to+inferring+population+structure+from+dominant+markers.+Mol.+Ecol.+11%3A+1157%E2%80%931164." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.21" data-doi="10.1534/genetics.106.061317"><div class="cit-metadata"><cite>H<span class="sc">uelsenbeck</span>, J. P., and P. A<span class="sc">ndolfatto</span>, <span class="cit-pub-date">2007</span> Inference of population structure under a Dirichlet process model. <span class="cit-source">Genetics</span> <span class="cit-vol">175</span><strong>:</strong> <span class="cit-fpage">1787</span>â1802.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061317%26rft_id%253Dinfo%253Apmid%252F17237522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NS80LzE3ODciO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Huelsenbeck%2C+J.+P.%2C+and+P.+Andolfatto%2C+2007+Inference+of+population+structure+under+a+Dirichlet+process+model.+Genetics+175%3A+1787%E2%80%931802." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.22"><div class="cit-metadata"><cite>K<span class="sc">ish</span>, L., <span class="cit-pub-date">1965</span> <em>Survey Sampling</em>. Wiley, New York.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Kish%2C+L.%2C+1965+Survey+Sampling.+Wiley%2C+New+York." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.23" data-doi="10.1534/genetics.103.023044"><div class="cit-metadata"><cite>K<span class="sc">itada</span>, S., and H. K<span class="sc">ishino</span>, <span class="cit-pub-date">2004</span> Simultaneous detection of linkage disequilibrium and genetic differentiation of subdivided populations. <span class="cit-source">Genetics</span> <span class="cit-vol">167</span><strong>:</strong> <span class="cit-fpage">2003</span>â2013.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DKitada%26rft.auinit1%253DS.%26rft.volume%253D167%26rft.issue%253D4%26rft.spage%253D2003%26rft.epage%253D2013%26rft.atitle%253DSimultaneous%2BDetection%2Bof%2BLinkage%2BDisequilibrium%2Band%2BGenetic%2BDifferentiation%2Bof%2BSubdivided%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.103.023044%26rft_id%253Dinfo%253Apmid%252F15342536%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ny80LzIwMDMiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Kitada%2C+S.%2C+and+H.+Kishino%2C+2004+Simultaneous+detection+of+linkage+disequilibrium+and+genetic+differentiation+of+subdivided+populations.+Genetics+167%3A+2003%E2%80%932013." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.24"><div class="cit-metadata"><cite>K<span class="sc">itada</span>, S., T. H<span class="sc">ayashi</span> and H. K<span class="sc">ishino</span>, <span class="cit-pub-date">2000</span> Empirical Bayes procedure for estimating genetic distance between populations and effective population size. <span class="cit-source">Genetics</span> <span class="cit-vol">156</span><strong>:</strong> <span class="cit-fpage">2063</span>â2079.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DKitada%26rft.auinit1%253DS.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D2063%26rft.epage%253D2079%26rft.atitle%253DEmpirical%2BBayes%2BProcedure%2Bfor%2BEstimating%2BGenetic%2BDistance%2BBetween%2BPopulations%2Band%2BEffective%2BPopulation%2BSize%26rft_id%253Dinfo%253Apmid%252F11102396%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzIwNjMiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Kitada%2C+S.%2C+T.+Hayashi+and+H.+Kishino%2C+2000+Empirical+Bayes+procedure+for+estimating+genetic+distance+between+populations+and+effective+population+size.+Genetics+156%3A+2063%E2%80%932079." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.25" data-doi="10.1534/genetics.106.055350"><div class="cit-metadata"><cite>K<span class="sc">itakado</span>, T., S. K<span class="sc">itada</span>, H. K<span class="sc">ishino</span> and H. J. S<span class="sc">kaug</span>, <span class="cit-pub-date">2006</span> An integrated-likelihood method for estimating genetic differentiation between populations. <span class="cit-source">Genetics</span> <span class="cit-vol">173</span><strong>:</strong> <span class="cit-fpage">2073</span>â2082.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.volume%253D173%26rft.issue%253D4%26rft.spage%253D2073%26rft.epage%253D2082%26rft.atitle%253DAn%2Bintegrated-likelihood%2Bmethod%2Bfor%2Bestimating%2Bgenetic%2Bdifferentiation%2Bbetween%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.055350%26rft_id%253Dinfo%253Apmid%252F16751669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My80LzIwNzMiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Kitakado%2C+T.%2C+S.+Kitada%2C+H.+Kishino+and+H.+J.+Skaug%2C+2006+An+integrated-likelihood+method+for+estimating+genetic+differentiation+between+populations.+Genetics+173%3A+2073%E2%80%932082." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.26" data-doi="10.1038/9642"><div class="cit-metadata"><cite>K<span class="sc">ruglyak</span>, L., <span class="cit-pub-date">1999</span> Prospects for whole-genome linkage disequilibrium mapping of common disease genes. <span class="cit-source">Nat. Genet.</span> <span class="cit-vol">22</span><strong>:</strong> <span class="cit-fpage">139</span>â144.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKruglyak%26rft.auinit1%253DL.%26rft.volume%253D22%26rft.issue%253D2%26rft.spage%253D139%26rft.epage%253D144%26rft.atitle%253DProspects%2Bfor%2Bwhole-genome%2Blinkage%2Bdisequilibrium%2Bmapping%2Bof%2Bcommon%2Bdisease%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F9642%26rft_id%253Dinfo%253Apmid%252F10369254%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1038/9642&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10369254&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000080680900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Kruglyak%2C+L.%2C+1999+Prospects+for+whole-genome+linkage+disequilibrium+mapping+of+common+disease+genes.+Nat.+Genet.+22%3A+139%E2%80%93144." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference in text" id="ref-27">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.27" data-doi="10.1007/BF01441156"><div class="cit-metadata"><cite>L<span class="sc">ange</span>, K., <span class="cit-pub-date">1995</span> Application of the Dirichlet distribution to forensic match probabilities. <span class="cit-source">Genetica</span> <span class="cit-vol">96</span><strong>:</strong> <span class="cit-fpage">107</span>â117.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DLange%26rft.auinit1%253DK.%26rft.volume%253D96%26rft.issue%253D1-2%26rft.spage%253D107%26rft.epage%253D117%26rft.atitle%253DApplications%2Bof%2Bthe%2BDirichlet%2Bdistribution%2Bto%2Bforensic%2Bmatch%2Bprobabilities.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01441156%26rft_id%253Dinfo%253Apmid%252F7607447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1007/BF01441156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=7607447&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Lange%2C+K.%2C+1995+Application+of+the+Dirichlet+distribution+to+forensic+match+probabilities.+Genetica+96%3A+107%E2%80%93117." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference in text" id="ref-28">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.28" data-doi="10.1002/1098-2272(200101)20:1&lt;17::AID-GEPI3&gt;3.0.CO;2-Q"><div class="cit-metadata"><cite>L<span class="sc">ockwood</span>, J. R., K. R<span class="sc">oeder</span> and B. D<span class="sc">evlin</span>, <span class="cit-pub-date">2001</span> A Bayesian hierarchical model for allele frequencies. <span class="cit-source">Genet. Epidemiol.</span> <span class="cit-vol">20</span><strong>:</strong> <span class="cit-fpage">17</span>â33.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLockwood%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D20%26rft.issue%253D1%26rft.spage%253D17%26rft.epage%253D33%26rft.atitle%253DA%2BBayesian%2Bhierarchical%2Bmodel%2Bfor%2Ballele%2Bfrequencies.%26rft_id%253Dinfo%253Adoi%252F10.1002%252F1098-2272%2528200101%252920%253A1%253C17%253A%253AAID-GEPI3%253E3.0.CO%253B2-Q%26rft_id%253Dinfo%253Apmid%252F11119294%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1002/1098-2272(200101)20:1%3C17::AID-GEPI3%3E3.0.CO;2-Q&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=11119294&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000165982700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Lockwood%2C+J.+R.%2C+K.+Roeder+and+B.+Devlin%2C+2001+A+Bayesian+hierarchical+model+for+allele+frequencies.+Genet.+Epidemiol.+20%3A+17%E2%80%9333." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference in text" id="ref-29">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.29"><div class="cit-metadata"><cite>M<span class="sc">aritz</span>, J. S., and T. L. L<span class="sc">ewin</span>, <span class="cit-pub-date">1989</span> <em>Empirical Bayes Methods</em>, Ed. 2. Chapman &amp; Hall, London.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Maritz%2C+J.+S.%2C+and+T.+L.+Lewin%2C+1989+Empirical+Bayes+Methods%2C+Ed.+2.+Chapman+%26+Hall%2C+London." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference in text" id="ref-30">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.30" data-doi="10.1126/science.1092500"><div class="cit-metadata"><cite>M<span class="sc">c</span>V<span class="sc">ean</span>, G. A. T., S. R. M<span class="sc">yers</span>, S. H<span class="sc">unt</span>, P. D<span class="sc">eloukas</span>, D. R. B<span class="sc">entley</span> <em>et al</em>., <span class="cit-pub-date">2004</span> The fine-scale structure of recombination rate variation in the human genome. <span class="cit-source">Science</span> <span class="cit-vol">304</span><strong>:</strong> <span class="cit-fpage">581</span>â584.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.issn%253D0036-8075%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%2BA.%2BT.%26rft.volume%253D304%26rft.issue%253D5670%26rft.spage%253D581%26rft.epage%253D584%26rft.atitle%253DThe%2BFine-Scale%2BStructure%2Bof%2BRecombination%2BRate%2BVariation%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1092500%26rft_id%253Dinfo%253Apmid%252F15105499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMDQvNTY3MC81ODEiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=McVean%2C+G.+A.+T.%2C+S.+R.+Myers%2C+S.+Hunt%2C+P.+Deloukas%2C+D.+R.+Bentley+et+al.%2C+2004+The+fine-scale+structure+of+recombination+rate+variation+in+the+human+genome.+Science+304%3A+581%E2%80%93584." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference in text" id="ref-31">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.31" data-doi="10.1073/pnas.70.12.3321"><div class="cit-metadata"><cite>N<span class="sc">ei</span>, M., <span class="cit-pub-date">1973</span> Analysis of gene diversity in subdivided populations. <span class="cit-source">Proc. Natl. Acad. Sci. USA</span> <span class="cit-vol">70</span><strong>:</strong> <span class="cit-fpage">3321</span>â3323.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.issn%253D0027-8424%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D70%26rft.issue%253D12%26rft.spage%253D3321%26rft.epage%253D3323%26rft.atitle%253DAnalysis%2Bof%2BGene%2BDiversity%2Bin%2BSubdivided%2BPopulations%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.70.12.3321%26rft_id%253Dinfo%253Apmid%252F4519626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzAvMTIvMzMyMSI7czo0OiJhdG9tIjtzOjI0OiIvZ2VuZXRpY3MvMTc3LzIvODYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Nei%2C+M.%2C+1973+Analysis+of+gene+diversity+in+subdivided+populations.+Proc.+Natl.+Acad.+Sci.+USA+70%3A+3321%E2%80%933323." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference in text" id="ref-32">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.32" data-doi="10.1111/j.1469-1809.1983.tb00993.x"><div class="cit-metadata"><cite>N<span class="sc">ei</span>, M., and R. K. C<span class="sc">hesser</span>, <span class="cit-pub-date">1983</span> Estimation of fixation indices and gene diversities. <span class="cit-source">Ann. Hum. Genet.</span> <span class="cit-vol">47</span><strong>:</strong> <span class="cit-fpage">253</span>â259.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DAnnals%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAnn%2BHum%2BGenet%26rft.aulast%253DNei%26rft.auinit1%253DM.%26rft.volume%253D47%26rft.issue%253DPt%2B3%26rft.spage%253D253%26rft.epage%253D259%26rft.atitle%253DEstimation%2Bof%2Bfixation%2Bindices%2Band%2Bgene%2Bdiversities.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-1809.1983.tb00993.x%26rft_id%253Dinfo%253Apmid%252F6614868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1111/j.1469-1809.1983.tb00993.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=6614868&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1983RB52000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Nei%2C+M.%2C+and+R.+K.+Chesser%2C+1983+Estimation+of+fixation+indices+and+gene+diversities.+Ann.+Hum.+Genet.+47%3A+253%E2%80%93259." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference in text" id="ref-33">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.33" data-doi="10.1111/j.1365-294X.1995.tb00227.x"><div class="cit-metadata"><cite>P<span class="sc">aetkau</span>, D., W. C<span class="sc">alvert</span>, I. S<span class="sc">tirling</span> and C. S<span class="sc">trobeck</span>, <span class="cit-pub-date">1995</span> Microsatellite analysis of population structure in Canadian polar bears. <span class="cit-source">Mol. Ecol.</span> <span class="cit-vol">4</span><strong>:</strong> <span class="cit-fpage">347</span>â354.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DPaetkau%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D347%26rft.epage%253D354%26rft.atitle%253DMicrosatellite%2Banalysis%2Bof%2Bpopulation%2Bstructure%2Bin%2BCanadian%2Bpolar%2Bbears.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.1995.tb00227.x%26rft_id%253Dinfo%253Apmid%252F7663752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1111/j.1365-294X.1995.tb00227.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=7663752&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1995RF43700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Paetkau%2C+D.%2C+W.+Calvert%2C+I.+Stirling+and+C.+Strobeck%2C+1995+Microsatellite+analysis+of+population+structure+in+Canadian+polar+bears.+Mol.+Ecol.+4%3A+347%E2%80%93354." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference in text" id="ref-34">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.34" data-doi="10.1016/j.tree.2006.09.003"><div class="cit-metadata"><cite>P<span class="sc">alsbÃ¸ll</span>, P. J., M. B<span class="sc">Ã©rubÃ©</span> and F. W. A<span class="sc">llendorf</span>, <span class="cit-pub-date">2007</span> Identification of management units using population genetic data. <span class="cit-source">Trends Ecol. Evol.</span> <span class="cit-vol">22</span><strong>:</strong> <span class="cit-fpage">11</span>â16.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.stitle%253DTrends%2Bin%2BEcology%2B%2526%2BEvolution%26rft.aulast%253DPalsboll%26rft.auinit1%253DP.%2BJ.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D16%26rft.atitle%253DIdentification%2Bof%2Bmanagement%2Bunits%2Busing%2Bpopulation%2Bgenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2006.09.003%26rft_id%253Dinfo%253Apmid%252F16982114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1016/j.tree.2006.09.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=16982114&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Palsb%C3%B8ll%2C+P.+J.%2C+M.+B%C3%A9rub%C3%A9+and+F.+W.+Allendorf%2C+2007+Identification+of+management+units+using+population+genetic+data.+Trends+Ecol.+Evol.+22%3A+11%E2%80%9316." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference in text" id="ref-35">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.35" data-doi="10.1098/rstb.2000.0740"><div class="cit-metadata"><cite>P<span class="sc">annell</span>, J. R., and B. C<span class="sc">harlesworth</span>, <span class="cit-pub-date">2000</span> Effects of metapopulation processes on measures of genetic diversity. <span class="cit-source">Philos. Trans. R. Soc. Lond. B</span> <span class="cit-vol">355</span><strong>:</strong> <span class="cit-fpage">1851</span>â1864.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.issn%253D0080-4622%26rft.aulast%253DPannell%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D355%26rft.issue%253D1404%26rft.spage%253D1851%26rft.epage%253D1864%26rft.atitle%253DEffects%2Bof%2Bmetapopulation%2Bprocesses%2Bon%2Bmeasures%2Bof%2Bgenetic%2Bdiversity%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2000.0740%26rft_id%253Dinfo%253Apmid%252F11205346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNTUvMTQwNC8xODUxIjtzOjQ6ImF0b20iO3M6MjQ6Ii9nZW5ldGljcy8xNzcvMi84NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Pannell%2C+J.+R.%2C+and+B.+Charlesworth%2C+2000+Effects+of+metapopulation+processes+on+measures+of+genetic+diversity.+Philos.+Trans.+R.+Soc.+Lond.+B+355%3A+1851%E2%80%931864." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference in text" id="ref-36">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.36"><div class="cit-metadata"><cite>P<span class="sc">ritchard</span>, J. K., M. S<span class="sc">tephens</span> and P. D<span class="sc">onnelly</span>, <span class="cit-pub-date">2000</span> Inference of population structure using multilocus genotype data. <span class="cit-source">Genetics</span> <span class="cit-vol">156</span><strong>:</strong> <span class="cit-fpage">945</span>â959.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.issn%253D0016-6731%26rft.volume%253D156%26rft.spage%253D945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Pritchard%2C+J.+K.%2C+M.+Stephens+and+P.+Donnelly%2C+2000+Inference+of+population+structure+using+multilocus+genotype+data.+Genetics+156%3A+945%E2%80%93959." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference in text" id="ref-37">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.37"><div class="cit-metadata"><cite>R<span class="sc">annala</span>, B., and J. A. H<span class="sc">artigan</span>, <span class="cit-pub-date">1996</span> Estimating gene flow in island populations. <span class="cit-source">Genet. Res.</span> <span class="cit-vol">67</span><strong>:</strong> <span class="cit-fpage">147</span>â158.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DRannala%26rft.auinit1%253DB.%26rft.volume%253D67%26rft.issue%253D2%26rft.spage%253D147%26rft.epage%253D158%26rft.atitle%253DEstimating%2Bgene%2Bflow%2Bin%2Bisland%2Bpopulations.%26rft_id%253Dinfo%253Apmid%252F8801187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=8801187&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1996UK49900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Rannala%2C+B.%2C+and+J.+A.+Hartigan%2C+1996+Estimating+gene+flow+in+island+populations.+Genet.+Res.+67%3A+147%E2%80%93158." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference in text" id="ref-38">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.38" data-doi="10.1006/tpbi.2000.1457"><div class="cit-metadata"><cite>R<span class="sc">aufaste</span>, N., and F. B<span class="sc">onhomme</span>, <span class="cit-pub-date">2000</span> Properties of bias and variance of two multiallelic estimators of <em>F<sub>ST</sub></em>. <span class="cit-source">Theor. Popul. Biol.</span> <span class="cit-vol">57</span><strong>:</strong> <span class="cit-fpage">285</span>â296.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DRaufaste%26rft.auinit1%253DN.%26rft.volume%253D57%26rft.issue%253D3%26rft.spage%253D285%26rft.epage%253D296%26rft.atitle%253DProperties%2Bof%2Bbias%2Band%2Bvariance%2Bof%2Btwo%2Bmultiallelic%2Bestimators%2Bof%2BF%2528ST%2529.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2000.1457%26rft_id%253Dinfo%253Apmid%252F10828220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1006/tpbi.2000.1457&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10828220&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000087411400008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Raufaste%2C+N.%2C+and+F.+Bonhomme%2C+2000+Properties+of+bias+and+variance+of+two+multiallelic+estimators+of+FST.+Theor.+Popul.+Biol.+57%3A+285%E2%80%93296." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference in text" id="ref-39">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.39"><div class="cit-metadata"><cite>R<span class="sc">aymond</span>, M., and F. R<span class="sc">ousset</span>, <span class="cit-pub-date">1995</span> GENEPOP (version 3.4): population genetics software for exact tests and ecumenicism. <span class="cit-source">J. Hered.</span> <span class="cit-vol">86</span><strong>:</strong> <span class="cit-fpage">248</span>â249.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.issn%253D0022-1503%26rft.aulast%253DRaymond%26rft.auinit1%253DM.%26rft.volume%253D86%26rft.issue%253D3%26rft.spage%253D248%26rft.epage%253D249%26rft.atitle%253DGENEPOP%2B%2528Version%2B1.2%2529%253A%2BPopulation%2BGenetics%2BSoftware%2Bfor%2BExact%2BTests%2Band%2BEcumenicism%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo2OiJqaGVyZWQiO3M6NToicmVzaWQiO3M6ODoiODYvMy8yNDgiO3M6NDoiYXRvbSI7czoyNDoiL2dlbmV0aWNzLzE3Ny8yLzg2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Raymond%2C+M.%2C+and+F.+Rousset%2C+1995+GENEPOP+(version+3.4)%3A+population+genetics+software+for+exact+tests+and+ecumenicism.+J.+Hered.+86%3A+248%E2%80%93249." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference in text" id="ref-40">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.40"><div class="cit-metadata"><cite>R<span class="sc">eynolds</span>, J., B. S. W<span class="sc">eirs</span> and C. C. C<span class="sc">ockerham</span>, <span class="cit-pub-date">1983</span> Estimation of the coancestry coefficient: basis for a short-term genetic distance. <span class="cit-source">Genetics</span> <span class="cit-vol">105</span><strong>:</strong> <span class="cit-fpage">767</span>â779.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DReynolds%26rft.auinit1%253DJ.%26rft.volume%253D105%26rft.issue%253D3%26rft.spage%253D767%26rft.epage%253D779%26rft.atitle%253DESTIMATION%2BOF%2BTHE%2BCOANCESTRY%2BCOEFFICIENT%253A%2BBASIS%2BFOR%2BA%2BSHORT-TERM%2BGENETIC%2BDISTANCE%26rft_id%253Dinfo%253Apmid%252F17246175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA1LzMvNzY3IjtzOjQ6ImF0b20iO3M6MjQ6Ii9nZW5ldGljcy8xNzcvMi84NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Reynolds%2C+J.%2C+B.+S.+Weirs+and+C.+C.+Cockerham%2C+1983+Estimation+of+the+coancestry+coefficient%3A+basis+for+a+short-term+genetic+distance.+Genetics+105%3A+767%E2%80%93779." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference in text" id="ref-41">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.41"><div class="cit-metadata"><cite>R<span class="sc">obertson</span>, A., and W. G. H<span class="sc">ill</span>, <span class="cit-pub-date">1984</span> Deviation from HardyâWeinberg proportions: sample variances and use in estimation of inbreeding coefficients. <span class="cit-source">Genetics</span> <span class="cit-vol">107</span><strong>:</strong> <span class="cit-fpage">703</span>â718.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.issn%253D0016-6731%26rft.aulast%253DRobertson%26rft.auinit1%253DA.%26rft.volume%253D107%26rft.issue%253D4%26rft.spage%253D703%26rft.epage%253D718%26rft.atitle%253DDEVIATIONS%2BFROM%2BHARDY-WEINBERG%2BPROPORTIONS%253A%2BSAMPLING%2BVARIANCES%2BAND%2BUSE%2BIN%2BESTIMATION%2BOF%2BINBREEDING%2BCOEFFICIENTS%26rft_id%253Dinfo%253Apmid%252F6745643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA3LzQvNzAzIjtzOjQ6ImF0b20iO3M6MjQ6Ii9nZW5ldGljcy8xNzcvMi84NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Robertson%2C+A.%2C+and+W.+G.+Hill%2C+1984+Deviation+from+Hardy%E2%80%93Weinberg+proportions%3A+sample+variances+and+use+in+estimation+of+inbreeding+coefficients.+Genetics+107%3A+703%E2%80%93718." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference in text" id="ref-42">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.42" data-doi="10.1086/380416"><div class="cit-metadata"><cite>R<span class="sc">osenberg</span>, N. A., L. M. L<span class="sc">i</span>, R. W<span class="sc">ard</span> and J. K. P<span class="sc">ritchard</span>, <span class="cit-pub-date">2003</span> Informativeness of genetic markers for inference of ancestry. <span class="cit-source">Am. J. Hum. Genet.</span> <span class="cit-vol">73</span><strong>:</strong> <span class="cit-fpage">1402</span>â1422.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DRosenberg%26rft.auinit1%253DN.%2BA.%26rft.volume%253D73%26rft.issue%253D6%26rft.spage%253D1402%26rft.epage%253D1422%26rft.atitle%253DInformativeness%2Bof%2Bgenetic%2Bmarkers%2Bfor%2Binference%2Bof%2Bancestry.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F380416%26rft_id%253Dinfo%253Apmid%252F14631557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1086/380416&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=14631557&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000187491100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Rosenberg%2C+N.+A.%2C+L.+M.+Li%2C+R.+Ward+and+J.+K.+Pritchard%2C+2003+Informativeness+of+genetic+markers+for+inference+of+ancestry.+Am.+J.+Hum.+Genet.+73%3A+1402%E2%80%931422." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference in text" id="ref-43">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.43"><div class="cit-metadata"><cite>R<span class="sc">ousset</span>, F., <span class="cit-pub-date">2003</span> Inferences from spatial population genetics, pp. 681â712 in <em>Handbook of Statistical Genetics</em>, Ed. 2, edited by D. J. B<span class="sc">alding</span>, M. B<span class="sc">ishop</span> and C. C<span class="sc">annings</span>. John Wiley &amp; Sons, Chichester, UK.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Rousset%2C+F.%2C+2003+Inferences+from+spatial+population+genetics%2C+pp.+681%E2%80%93712+in+Handbook+of+Statistical+Genetics%2C+Ed.+2%2C+edited+by+D.+J.+Balding%2C+M.+Bishop+and+C.+Cannings.+John+Wiley+%26+Sons%2C+Chichester%2C+UK." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference in text" id="ref-44">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.44" data-doi="10.1046/j.0962-1083.2001.01345.x"><div class="cit-metadata"><cite>R<span class="sc">yman</span>, N., and P. E. J<span class="sc">orde</span>, <span class="cit-pub-date">2001</span> Statistical power when testing for genetic differentiation. <span class="cit-source">Mol. Ecol.</span> <span class="cit-vol">10</span><strong>:</strong> <span class="cit-fpage">2361</span>â2373.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRyman%26rft.auinit1%253DN.%26rft.volume%253D10%26rft.issue%253D10%26rft.spage%253D2361%26rft.epage%253D2373%26rft.atitle%253DStatistical%2Bpower%2Bwhen%2Btesting%2Bfor%2Bgenetic%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.0962-1083.2001.01345.x%26rft_id%253Dinfo%253Apmid%252F11703649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1046/j.0962-1083.2001.01345.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=11703649&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000171693700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Ryman%2C+N.%2C+and+P.+E.+Jorde%2C+2001+Statistical+power+when+testing+for+genetic+differentiation.+Mol.+Ecol.+10%3A+2361%E2%80%932373." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference in text" id="ref-45">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.45" data-doi="10.1111/j.1365-294X.2006.02839.x"><div class="cit-metadata"><cite>R<span class="sc">yman</span>, N., S. P<span class="sc">alm</span>, C. A<span class="sc">ndrÃ©</span>, G. R. C<span class="sc">arvalho</span>, T. G. D<span class="sc">ahlgren</span> <em>et al</em>., <span class="cit-pub-date">2006</span> Power for detecting genetic divergence: differences between statistical methods and marker loci. <span class="cit-source">Mol. Ecol.</span> <span class="cit-vol">15</span><strong>:</strong> <span class="cit-fpage">2031</span>â2045.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DRyman%26rft.auinit1%253DN.%26rft.volume%253D15%26rft.issue%253D8%26rft.spage%253D2031%26rft.epage%253D2045%26rft.atitle%253DPower%2Bfor%2Bdetecting%2Bgenetic%2Bdivergence%253A%2Bdifferences%2Bbetween%2Bstatistical%2Bmethods%2Band%2Bmarker%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.02839.x%26rft_id%253Dinfo%253Apmid%252F16780422%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.02839.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=16780422&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Ryman%2C+N.%2C+S.+Palm%2C+C.+Andr%C3%A9%2C+G.+R.+Carvalho%2C+T.+G.+Dahlgren+et+al.%2C+2006+Power+for+detecting+genetic+divergence%3A+differences+between+statistical+methods+and+marker+loci.+Mol.+Ecol.+15%3A+2031%E2%80%932045." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference in text" id="ref-46">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.46"><div class="cit-metadata"><cite>S<span class="sc">eber</span>, G. A. F., <span class="cit-pub-date">1982</span> <em>The Estimation of Animal Abundance and Related Parameters</em>, Ed. 2. Griffin, London.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Seber%2C+G.+A.+F.%2C+1982+The+Estimation+of+Animal+Abundance+and+Related+Parameters%2C+Ed.+2.+Griffin%2C+London." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference in text" id="ref-47">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.47"><div class="cit-metadata"><cite>S<span class="sc">tein</span>, C., <span class="cit-pub-date">1956</span>
 Inadmissibility of the usual estimator for the mean of a multivariate 
distribution. Proceedings of the Third Berkeley Symposium on 
Mathematical Statistics and Probability, Berkeley, California, Vol. 1, 
pp. 197â206. University of California Press, Berkeley, CA.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Stein%2C+C.%2C+1956+Inadmissibility+of+the+usual+estimator+for+the+mean+of+a+multivariate+distribution.+Proceedings+of+the+Third+Berkeley+Symposium+on+Mathematical+Statistics+and+Probability%2C+Berkeley%2C+California%2C+Vol.+1%2C+pp.+197%E2%80%93206.+University+of+California+Press%2C+Berkeley%2C+CA." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference in text" id="ref-48">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.48" data-doi="10.1007/BF02288916"><div class="cit-metadata"><cite>T<span class="sc">orgerson</span>, W. S., <span class="cit-pub-date">1952</span> Multidimensional scaling: 1. theory and methods. <span class="cit-source">Psychometrika</span> <span class="cit-vol">17</span><strong>:</strong> <span class="cit-fpage">401</span>â419.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DPsychometrika%26rft.volume%253D17%26rft.spage%253D401%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02288916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1007/BF02288916&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1952XT51100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Torgerson%2C+W.+S.%2C+1952+Multidimensional+scaling%3A+1.+theory+and+methods.+Psychometrika+17%3A+401%E2%80%93419." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference in text" id="ref-49">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.49" data-doi="10.1007/s00439-005-0090-0"><div class="cit-metadata"><cite>W<span class="sc">alsh</span>, R. D., P. S<span class="sc">abeti</span>, H. B. H<span class="sc">utcheson</span>, B. F<span class="sc">ry</span>, S. F. S<span class="sc">chaffner</span> <em>et al</em>., <span class="cit-pub-date">2006</span> Searching for signals of evolutionary selection in 168 genes related to immune function. <span class="cit-source">Hum. Genet.</span> <span class="cit-vol">119</span><strong>:</strong> <span class="cit-fpage">92</span>â102.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DHuman%2Bgenetics%26rft.stitle%253DHum%2BGenet%26rft.aulast%253DWalsh%26rft.auinit1%253DE.%2BC.%26rft.volume%253D119%26rft.issue%253D1-2%26rft.spage%253D92%26rft.epage%253D102%26rft.atitle%253DSearching%2Bfor%2Bsignals%2Bof%2Bevolutionary%2Bselection%2Bin%2B168%2Bgenes%2Brelated%2Bto%2Bimmune%2Bfunction.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00439-005-0090-0%26rft_id%253Dinfo%253Apmid%252F16362345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1007/s00439-005-0090-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=16362345&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000235635000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Walsh%2C+R.+D.%2C+P.+Sabeti%2C+H.+B.+Hutcheson%2C+B.+Fry%2C+S.+F.+Schaffner+et+al.%2C+2006+Searching+for+signals+of+evolutionary+selection+in+168+genes+related+to+immune+function.+Hum.+Genet.+119%3A+92%E2%80%93102." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference in text" id="ref-50">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.50" data-doi="10.1111/j.1365-294X.2006.02890.x"><div class="cit-metadata"><cite>W<span class="sc">aples</span>, R. S., and O. G<span class="sc">aggiotti</span>, <span class="cit-pub-date">2006</span>
 What is a population? An empirical evaluation of some genetic methods 
for identifying the number of gene pools and their degree of 
connectivity. <span class="cit-source">Mol. Ecol.</span> <span class="cit-vol">15</span><strong>:</strong> <span class="cit-fpage">1419</span>â1439.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DWaples%26rft.auinit1%253DR.%2BS.%26rft.volume%253D15%26rft.issue%253D6%26rft.spage%253D1419%26rft.epage%253D1439%26rft.atitle%253DWhat%2Bis%2Ba%2Bpopulation%253F%2BAn%2Bempirical%2Bevaluation%2Bof%2Bsome%2Bgenetic%2Bmethods%2Bfor%2Bidentifying%2Bthe%2Bnumber%2Bof%2Bgene%2Bpools%2Band%2Btheir%2Bdegree%2Bof%2Bconnectivity.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2006.02890.x%26rft_id%253Dinfo%253Apmid%252F16629801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1111/j.1365-294X.2006.02890.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=16629801&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000236798200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Waples%2C+R.+S.%2C+and+O.+Gaggiotti%2C+2006+What+is+a+population%3F+An+empirical+evaluation+of+some+genetic+methods+for+identifying+the+number+of+gene+pools+and+their+degree+of+connectivity.+Mol.+Ecol.+15%3A+1419%E2%80%931439." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference in text" id="ref-51">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.51"><div class="cit-metadata"><cite>W<span class="sc">eicker</span>, J. J., R. T. B<span class="sc">rumfield</span> and K. W<span class="sc">inker</span>, <span class="cit-pub-date">2001</span> Estimating the unbiased estimator Î¸ for population genetic survey data. <span class="cit-source">Evolution</span> <span class="cit-vol">55</span><strong>:</strong> <span class="cit-fpage">2601</span>â2605.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DWeicker%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D55%26rft.issue%253D12%26rft.spage%253D2601%26rft.epage%253D2605%26rft.atitle%253DEstimating%2Bthe%2Bunbiased%2Bestimator%2Btheta%2Bfor%2Bpopulation%2Bgenetic%2Bsurvey%2Bdata.%26rft_id%253Dinfo%253Apmid%252F11831672%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=11831672&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Weicker%2C+J.+J.%2C+R.+T.+Brumfield+and+K.+Winker%2C+2001+Estimating+the+unbiased+estimator+%CE%B8+for+population+genetic+survey+data.+Evolution+55%3A+2601%E2%80%932605." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference in text" id="ref-52">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.52"><div class="cit-metadata"><cite>W<span class="sc">eir</span>, B. S., <span class="cit-pub-date">1996</span> <em>Genetic Data Analysis II</em>. Sinauer Associates, Sunderland, MA.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Weir%2C+B.+S.%2C+1996+Genetic+Data+Analysis+II.+Sinauer+Associates%2C+Sunderland%2C+MA." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference in text" id="ref-53">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.53" data-doi="10.2307/2408641"><div class="cit-metadata"><cite>W<span class="sc">eir</span>, B. S., and C. C. C<span class="sc">ockerham</span>, <span class="cit-pub-date">1984</span> Estimating F-statistics for the analysis of population structure. <span class="cit-source">Evolution</span> <span class="cit-vol">38</span><strong>:</strong> <span class="cit-fpage">1358</span>â1370.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Weir%2C+B.+S.%2C+and+C.+C.+Cockerham%2C+1984+Estimating+F-statistics+for+the+analysis+of+population+structure.+Evolution+38%3A+1358%E2%80%931370." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference in text" id="ref-54">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.54" data-doi="10.1146/annurev.genet.36.050802.093940"><div class="cit-metadata"><cite>W<span class="sc">eir</span>, B. S., and W. G. H<span class="sc">ill</span>, <span class="cit-pub-date">2002</span> Estimating F-statistics. <span class="cit-source">Annu. Rev. Genet.</span> <span class="cit-vol">36</span><strong>:</strong> <span class="cit-fpage">721</span>â750.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.genet.36.050802.093940%26rft_id%253Dinfo%253Apmid%252F12359738%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=10.1146/annurev.genet.36.050802.093940&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=12359738&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=000180365100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Weir%2C+B.+S.%2C+and+W.+G.+Hill%2C+2002+Estimating+F-statistics.+Annu.+Rev.+Genet.+36%3A+721%E2%80%93750." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference in text" id="ref-55">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.55" data-doi="10.1101/gr.4398405"><div class="cit-metadata"><cite>W<span class="sc">eir</span>, B. S., L. R. C<span class="sc">ardon</span>, A. D. A<span class="sc">nderson</span>, D. M. N<span class="sc">ielsen</span> and W. G. H<span class="sc">ill</span>, <span class="cit-pub-date">2005</span> Measures of human population structure show heterogeneity among genomic regions. <span class="cit-source">Genome Res.</span> <span class="cit-vol">15</span><strong>:</strong> <span class="cit-fpage">1468</span>â1476.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.issn%253D1088-9051%26rft.aulast%253DWeir%26rft.auinit1%253DB.%2BS.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1468%26rft.epage%253D1476%26rft.atitle%253DMeasures%2Bof%2Bhuman%2Bpopulation%2Bstructure%2Bshow%2Bheterogeneity%2Bamong%2Bgenomic%2Bregions%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4398405%26rft_id%253Dinfo%253Apmid%252F16251456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNDY4IjtzOjQ6ImF0b20iO3M6MjQ6Ii9nZW5ldGljcy8xNzcvMi84NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Weir%2C+B.+S.%2C+L.+R.+Cardon%2C+A.+D.+Anderson%2C+D.+M.+Nielsen+and+W.+G.+Hill%2C+2005+Measures+of+human+population+structure+show+heterogeneity+among+genomic+regions.+Genome+Res.+15%3A+1468%E2%80%931476." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference in text" id="ref-56">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.56"><div class="cit-metadata"><cite>W<span class="sc">right</span>, S., <span class="cit-pub-date">1951</span> The general structure of populations. <span class="cit-source">Ann. Eugen.</span> <span class="cit-vol">15</span><strong>:</strong> <span class="cit-fpage">323</span>â354.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/content/177/2/%7Bopenurl%7D?query=rft.jtitle%253DAnn.%2BEugen.%26rft.volume%253D15%26rft.spage%253D323%26rft_id%253Dinfo%253Apmid%252F24540312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url" style="display: none;"><span>OpenUrl</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=24540312&amp;link_type=MED&amp;atom=%2Fgenetics%2F177%2F2%2F861.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="https://www.genetics.org/lookup/external-ref?access_num=A1951XY09200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Wright%2C+S.%2C+1951+The+general+structure+of+populations.+Ann.+Eugen.+15%3A+323%E2%80%93354." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference in text" id="ref-57">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.57"><div class="cit-metadata"><cite>W<span class="sc">right</span>, S., <span class="cit-pub-date">1969</span> <em>Evolution and the Genetics of Populations: The Theory of Gene Frequencies</em>, Vol. 2. University of Chicago Press, Chicago.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Wright%2C+S.%2C+1969+Evolution+and+the+Genetics+of+Populations%3A+The+Theory+of+Gene+Frequencies%2C+Vol.+2.+University+of+Chicago+Press%2C+Chicago." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference in text" id="ref-58">âµ</a><div class="cit ref-cit ref-other google_scholar_link-processed custom-js-processed" id="cit-177.2.861.58"><div class="cit-metadata"><cite>Y<span class="sc">oung</span>, F. W., and R. M. H<span class="sc">amer</span>, <span class="cit-pub-date">1987</span> <em>Multidimensional Scaling: History, Theory and Applications</em>. Erlbaum, New York.</cite></div><div class="cit-extra"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=article&amp;q_txt=Young%2C+F.+W.%2C+and+R.+M.+Hamer%2C+1987+Multidimensional+Scaling%3A+History%2C+Theory+and+Applications.+Erlbaum%2C+New+York." target="_blank" class="cit-ref-sprinkles cit-ref-sprinkles-google-scholar cit-ref-sprinkles-google-scholar"><span>Google Scholar</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.genetics.org/content/177/2/861.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  <div class="rmq-annotator-adder rmq_tooltipCustom" aria-modal="true" style="display: none;"><div class="rmq_tooltiptext rmq_pdfTooltipbg"><button id="highlight"><div class="rmq_adderIcon"><svg width="17px" height="17px" viewBox="0 0 17 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g><g transform="translate(3.000000, 2.000000)"><polygon fill="#f8f8f8" points="0 13 11 13 11 12 0 12"></polygon><path fill="#f8f8f8" d="M1.41222388,7.76602119 L0,9.22181369 L2.70562744,11.4613037 L3.90055197,10.2714002 L6.10399263,9.49181108 C6.27186931,9.45774448 6.42432144,9.36721231 6.53455341,9.23296783 L10.7121763,4.10573462 C11.4966018,3.14240008 11.4456458,1.73074241 10.5635903,0.876360432 C9.5470617,-0.108570284 8.17015245,0.00985276586 7.37327375,0.762518151 L2.4184515,5.05239021 C2.28857681,5.16579913 2.20289355,5.31750614 2.17245618,5.48314331 L1.41222388,7.76602119 Z M9.87368038,3.42274977 L5.69746197,8.54826531 C5.73746826,8.49954004 5.78863949,8.46405676 5.84509535,8.44410079 L5.78998674,8.45576148 L3.3008194,9.33644225 L2.66821289,10.0626831 L1.61816406,9.09716797 L2.27437774,8.43044769 L2.39929014,8.22479354 L3.2141979,5.77773238 L3.22462418,5.72323352 C3.20611601,5.77886906 3.17300281,5.82922848 3.1280883,5.86845046 L8.09853212,1.56439002 C8.5203635,1.16665716 9.24476428,1.10435404 9.81111104,1.65309641 C10.2821702,2.10937658 10.3102266,2.88663752 9.87368038,3.42274977 Z"></path><polygon fill="#f8f8f8" points="3 5.75518453 6.34423899 9.18321503 7.11834502 8.4280305 3.77410604 5"></polygon><polygon fill="#f8f8f8" points="1.61553955 8.29144287 3.62811279 10.0869141 4.58270264 9.5645752 1.77410604 7"></polygon></g></g></g></svg></div><div class="rmq_adderText">HIGHLIGHT</div></button><button id="note"><div class="rmq_adderIcon"><svg width="17px" height="17px" viewBox="0 0 17 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" style="margin-left: 4px"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g><path fill="#f8f8f8" d="M1.98730469,0 L1.98730469,5 L0,5 L0,15 L10.999,15 L10.999,5 L8.9675293,5.03501211 L8.9675293,0.0350121094 L1.98730469,0 Z M3,0.999 L8.031,0.999 L7.99909961,5 L3,5 L3,0.999 Z M0.999,14 L9.999,14 L9.999,6 L0.999,6 L0.999,14 Z M5,12.001 L6,12.001 L6,8 L5,8 L5,12.001 Z"></path></g></g></svg></div><div class="rmq_adderText">NOTE</div></button><button id="comment"><div class="rmq_adderIcon"><svg width="17px" height="17px" viewBox="0 0 17 17" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g><g transform="translate(2.000000, 3.000000)"><polygon fill="#f8f8f8" points="1.67657143 3.04367885 10.248 3.04367885 10.248 2.197525 1.67657143 2.197525"></polygon><polygon fill="#f8f8f8" points="1.67657143 5.15906346 10.248 5.15906346 10.248 4.31290962 1.67657143 4.31290962"></polygon><polygon fill="#f8f8f8" points="1.67657143 7.27444808 7.42564286 7.27444808 7.42564286 6.42829423 1.67657143 6.42829423"></polygon><polyline stroke="#f8f8f8" stroke-linecap="square" points="11.8175786 0.105769231 0.107078571 0.105769231 0.107078571 9.07182692 7.14550714 9.07182692 10.2899357 10.9306154 10.2708643 9.06759615 11.8175786 9.06759615 11.8175786 0.105769231"></polyline></g></g></g></svg></div><div class="rmq_adderText">COMMENT</div></button><button id="conversation"><div class="rmq_adderIcon"><svg width="21px" height="21px" viewBox="0 0 21 21" version="1.1" xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink"><g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"><g><path fill="#f8f8f8" id="a" d="M9 17.988V10h11.262v7.984h-1.35l.02 2.014-3.401-2.01H9zm1-1h5.804l2.11 1.247-.013-1.25h1.36V11H10v5.988zM15.64 10V4H2v10.047h1.664L3.64 16.44l4.04-2.388H9v-1H7.406l-2.748 1.624.017-1.629H3V5h11.64v5h1z"></path></g></g></svg></div><div class="rmq_adderText">DISCUSS</div></button></div></div></div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager">
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="https://www.genetics.org/content/177/2/847" title="The Flightless I Homolog, fli-1, Regulates Anterior/Posterior Polarity, Asymmetric Cell Division and Ovulation During Caenorhabditis elegans Development" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="https://www.genetics.org/content/177/2/875" title="Reconciling Strong Stabilizing Selection with the Maintenance of Genetic Variation in a Natural Population of Black Field Crickets (Teleogryllus commodus)" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top">
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin">
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1">
  
      
  
  <div class="pane-content">
    <p><a href="https://www.genetics.org/content/177/2"><span>Volume 177 Issue 2, October 2007</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover">
  
      
  
  <div class="pane-content">
    <div class="cover-issue"><div class="cover-issue-image"><a href="https://www.genetics.org/content/177/2.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="2.gif" alt="Genetics: 177 (2)" width="248" height="330"></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch">
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/177/2/861" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" id="edit-keywords" name="keywords" size="60" maxlength="128" class="form-text" type="text">
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" id="edit-submit" name="op" value="Search" class="form-submit" type="submit"></div><input name="form_build_id" value="form-w2Xr6z4Sn5mVfpKwsbW13-Gl4tkwj8TGgp2LXuWz4eg" type="hidden">
<input name="form_id" value="highwire_search_quicksearch_form_0" type="hidden">
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin">
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="https://www.genetics.org/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">INVESTIGATIONS</a></div><div class="field-item odd"><a href="https://www.genetics.org/category/population-and-evolutionary-genetics" class="" data-icon-position="" data-hide-link-title="0">Population and evolutionary genetics</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin">
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="https://www.genetics.org/lens/genetics/177/2/861" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin">
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email">
  
      
  
  <div class="pane-content">
    <div class="minipanel-dialog-wrapper"><div class="minipanel-dialog-link-link"><a href="https://www.genetics.org/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger minipanel-dialog-link-trigger-processed" title="" data-icon-position="" data-hide-link-title="0"><i class="icon-envelope"></i> Email</a></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link">
  
      
  
  <div class="pane-content">
    <a href="https://www.genetics.org/content/genetics/177/2/861.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert">
  
      
  
  <div class="pane-content">
    <div class="minipanel-dialog-wrapper"><div class="minipanel-dialog-link-link"><a href="https://www.genetics.org/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon minipanel-dialog-link-trigger-processed" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link">
  
      
  
  <div class="pane-content">
    <a href="https://www.genetics.org/content/genetics/177/2/861.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax" id="shareit">
  
      
  
  <div class="pane-content">
    <a href="https://www.genetics.org/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit">
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation">
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="334013" id="node334013--2" data-pisa="genetics;177/2/861" data-pisa-master="genetics;genetics.107.077263" data-apath="/genetics/177/2/861.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Empirical Bayes Inference of Pairwise <em>F</em><sub>ST</sub> and Its Distribution in the Genome</h1>  
    	<div class="highwire-cite-authors"><span class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shuichi Kitada</span>, <span class="highwire-citation-author" data-delta="1">Toshihide Kitakado</span> and <span class="highwire-citation-author" data-delta="2">Hirohisa Kishino</span></span></div>
  
    	<div class="highwire-cite-metadata"><span class="highwire-cite-metadata-journal highwire-cite-metadata">Genetics </span><span class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2007 </span><span class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 177 </span><span class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span class="highwire-cite-metadata-pages highwire-cite-metadata">861-873;  </span><span class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/genetics.107.077263 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node334013--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shuichi Kitada</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuichi%2BKitada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kitada%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AShuichi%2BKitada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Toshihide Kitakado</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toshihide%2BKitakado" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kitakado%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AToshihide%2BKitakado" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hirohisa Kishino</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hirohisa%2BKishino" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kishino%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AHirohisa%2BKishino" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy">
  
      
  
  <div class="pane-content">
    <div class="clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input id="dynamic" value="https://www.genetics.org/content/177/2/861" size="50" type="text">
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links">
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="https://www.genetics.org/highwire_log/share/delicious?link=http%3A%2F%2Fdel.icio.us%2Fpost%3Furl%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26title%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="delicious-32.png" alt="del.icio.us logo"></a> <a href="https://www.genetics.org/highwire_log/share/digg?link=http%3A%2F%2Fdigg.com%2Fsubmit%3Fphase%3D2%26url%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26title%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="digg.png" alt="Digg logo"></a> <a href="https://www.genetics.org/highwire_log/share/reddit?link=http%3A%2F%2Freddit.com%2Fsubmit%3Furl%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26title%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="reddit.png" alt="Reddit logo"></a> <a href="https://www.genetics.org/highwire_log/share/twitter?link=http%3A%2F%2Ftwitter.com%2Fshare%3Furl%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26text%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="twitter.png" alt="Twitter logo"></a> <a href="https://www.genetics.org/highwire_log/share/citeulike?link=http%3A%2F%2Fwww.citeulike.org%2Fposturl%3Furl%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26title%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="citeyoulike.png" alt="CiteULike logo"></a> <a href="https://www.genetics.org/highwire_log/share/facebook?link=http%3A%2F%2Fwww.facebook.com%2Fsharer.php%3Fu%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26t%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="fb-blue.png" alt="Facebook logo"></a> <a href="https://www.genetics.org/highwire_log/share/google?link=http%3A%2F%2Fwww.google.com%2Fbookmarks%2Fmark%3Fop%3Dadd%26bkmk%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26title%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="google-32.png" alt="Google logo"></a> <a href="https://www.genetics.org/highwire_log/share/mendeley?link=http%3A%2F%2Fwww.mendeley.com%2Fimport%2F%3Furl%3Dhttps%253A%2F%2Fwww.genetics.org%2Fcontent%2F177%2F2%2F861%26title%3DEmpirical%2520Bayes%2520Inference%2520of%2520Pairwise%2520FST%2520and%2520Its%2520Distribution%2520in%2520the%2520Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0" data-log-redirect-set="true"><img src="mendeley.png" alt="Mendeley logo"></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool">
  
      
  
  <div class="pane-content">
    <div class="minipanel-dialog-wrapper"><div class="minipanel-dialog-link-link"><a href="https://www.genetics.org/highwire/citation/334013/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon minipanel-dialog-link-trigger-processed" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-ajax-pane-content pane-highwire-permission-link">
  
        
    
  
  <div class="pane-content">
    <div class="panels-ajax-pane panels-ajax-pane-new-37 panels-ajax-pane-once-processed" data-pid="new-37"><!--?xml version="1.0" encoding="UTF-8" ?-->
    <div class="panels-ajax-pane-html" version="HTML+RDFa+MathML 1.1" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:dc="http://purl.org/dc/terms/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:og="http://ogp.me/ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:sioc="http://rdfs.org/sioc/ns#" xmlns:sioct="http://rdfs.org/sioc/types#" xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <div class="panels-ajax-pane-head"></div><div class="panels-ajax-pane-body"><div class="panels-ajax-pane-panel panels-ajax-pane-panel-"><a href="https://www.copyright.com/openurl.do?contentIdType=doi&amp;issn=19432631&amp;contentId=10.1534/genetics.107.077263" target="_blank" class="highwire-permission-link link-icon"><i class="icon-copyright"></i> <span class="title">Request Permissions</span></a></div></div></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin">
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container" class="ui-accordion ui-widget ui-helper-reset" role="tablist"><h3 class="ui-accordion-header ui-helper-reset ui-state-default ui-accordion-header-active ui-state-active ui-corner-top ui-accordion-icons" role="tab" id="ui-accordion-highwire_article_accordion_container-header-0" aria-controls="ui-accordion-highwire_article_accordion_container-panel-0" aria-selected="true" tabindex="0"><span class="ui-accordion-header-icon ui-icon ui-icon-triangle-1-s"></span>Related Articles</h3><div class="ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom ui-accordion-content-active" style="display: block;" id="ui-accordion-highwire_article_accordion_container-panel-0" aria-labelledby="ui-accordion-highwire_article_accordion_container-header-0" role="tabpanel" aria-expanded="true" aria-hidden="false"><div class="panels-ajax-pane panels-ajax-pane-new-5 panels-ajax-pane-once-processed" data-pid="new-5"><!--?xml version="1.0" encoding="UTF-8" ?-->
    <div class="panels-ajax-pane-html" version="HTML+RDFa+MathML 1.1" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:dc="http://purl.org/dc/terms/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:og="http://ogp.me/ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:sioc="http://rdfs.org/sioc/ns#" xmlns:sioct="http://rdfs.org/sioc/types#" xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <div class="panels-ajax-pane-head"><link type="text/css" rel="stylesheet" href="css__Wz1GudemGmdBDJBhh7ASJQYBkEzH25n5pSrde4GjjJE__qEyxHMyhzA.css" media="all">
</div><div class="panels-ajax-pane-body"><div class="panels-ajax-pane-panel panels-ajax-pane-panel-"><div class="highwire-list-footer"><div class="highwire-related-articles-footer"><ul class="links inline"><li class="related-scopus first"><a href="https://www.genetics.org/lookup/external-ref?access_num=genetics%3B177/2/861&amp;link_type=SCOPUS_CITING&amp;external_url=http%3A//www.scopus.com/scopus/inward/citedby.url%3FpartnerID%3Dc9DrA512%26rel%3DR6.0.0%26eid%3D2-s2.0-35549011367%26md5%3D6f8f66db792c64015eb1922a92159011" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Scopus</a></li><li class="related-pubmed"><a href="https://www.genetics.org/lookup/external-ref?link_type=MED_NBRS&amp;access_num=17660541" target="_blank" class="" data-icon-position="" data-hide-link-title="0">PubMed</a></li><li class="related-google-scholar last"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=related&amp;author%5B0%5D=Shuichi%2BKitada&amp;author%5B1%5D=Toshihide%2BKitakado&amp;author%5B2%5D=Hirohisa%2BKishino&amp;title=Empirical%2BBayes%2BInference%2Bof%2BPairwise%2BFST%2Band%2BIts%2BDistribution%2Bin%2Bthe%2BGenome&amp;publication_year=2007&amp;path=content/177/2/861" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Google Scholar</a></li></ul></div></div></div></div></div></div></div><h3 class="ui-accordion-header ui-helper-reset ui-state-default ui-corner-all ui-accordion-icons" role="tab" id="ui-accordion-highwire_article_accordion_container-header-1" aria-controls="ui-accordion-highwire_article_accordion_container-panel-1" aria-selected="false" tabindex="-1"><span class="ui-accordion-header-icon ui-icon ui-icon-triangle-1-e"></span>Cited By</h3><div class="ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom" style="display: none;" id="ui-accordion-highwire_article_accordion_container-panel-1" aria-labelledby="ui-accordion-highwire_article_accordion_container-header-1" role="tabpanel" aria-expanded="false" aria-hidden="true"><div class="panels-ajax-pane panels-ajax-pane-new-6 panels-ajax-pane-once-processed" data-pid="new-6"><!--?xml version="1.0" encoding="UTF-8" ?-->
    <div class="panels-ajax-pane-html" version="HTML+RDFa+MathML 1.1" xmlns:content="http://purl.org/rss/1.0/modules/content/" xmlns:dc="http://purl.org/dc/terms/" xmlns:foaf="http://xmlns.com/foaf/0.1/" xmlns:og="http://ogp.me/ns#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:sioc="http://rdfs.org/sioc/ns#" xmlns:sioct="http://rdfs.org/sioc/types#" xmlns:skos="http://www.w3.org/2004/02/skos/core#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <div class="panels-ajax-pane-head"><link type="text/css" rel="stylesheet" href="css__ihXGyedBMMukVyzCJzHHUJeXKG5HcvLf3xbFFSS0IYI__nyUuCD1BMV.css" media="all">
</div><div class="panels-ajax-pane-body"><div class="panels-ajax-pane-panel panels-ajax-pane-panel-"><div class="highwire-list highwire-cited-by"><ul><li class="first odd"><a href="http://biorxiv.org/cgi/content/full/083915v2" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">FST and kinship for arbitrary population structures I: Generalized definitions</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://biorxiv.org/cgi/content/abstract/083915v2" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://biorxiv.org/cgi/content/full/083915v2" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://biorxiv.org/cgi/reprint/083915v2" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li><li class="last even"><a href="http://www.genetics.org/cgi/content/full/189/2/621" class="" data-icon-position="" data-hide-link-title="0"><div class="highwire-cite-title">A New Method to Uncover Signatures of Divergent and Stabilizing Selection in Quantitative Traits</div></a><div class="variant-links"><ul class="links inline"><li class="variant-abstract first"><a href="http://www.genetics.org/cgi/content/abstract/189/2/621" class="" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="variant-fulltext"><a href="http://www.genetics.org/cgi/content/full/189/2/621" class="" data-icon-position="" data-hide-link-title="0">Fulltext</a></li><li class="variant-pdf last"><a href="http://www.genetics.org/cgi/reprint/189/2/621" class="" data-icon-position="" data-hide-link-title="0">PDF</a></li></ul></div></li></ul></div><div class="highwire-list-footer"><ul class="links inline"><li class="citing-scopus first"><a href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=c9DrA512&amp;rel=R6.0.0&amp;eid=2-s2.0-35549011367&amp;md5=6f8f66db792c64015eb1922a92159011" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Scopus <span class="counter">(24)</span></a></li><li class="citing-google-scholar last"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=citedby&amp;path=content/177/2/861" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Google Scholar</a></li></ul></div></div></div></div></div></div><h3 class="ui-accordion-header ui-helper-reset ui-state-default ui-corner-all ui-accordion-icons" role="tab" id="ui-accordion-highwire_article_accordion_container-header-2" aria-controls="ui-accordion-highwire_article_accordion_container-panel-2" aria-selected="false" tabindex="-1"><span class="ui-accordion-header-icon ui-icon ui-icon-triangle-1-e"></span>More in this TOC Section</h3><div class="ui-accordion-content ui-helper-reset ui-widget-content ui-corner-bottom" style="display: none;" id="ui-accordion-highwire_article_accordion_container-panel-2" aria-labelledby="ui-accordion-highwire_article_accordion_container-header-2" role="tabpanel" aria-expanded="false" aria-hidden="true"><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="447045" id="node447045" data-pisa="genetics;213/4/1545" data-pisa-master="genetics;genetics.119.302749" data-apath="/genetics/213/4/1545.atom"><cite class="highwire-cite highwire-citation-jcore-title-only">
  <div class="highwire-cite-title"><a href="https://www.genetics.org/content/213/4/1545" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Comparative Genomics and Transcriptomics To Analyze Fruiting Body Development in Filamentous Ascomycetes</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="447035" id="node447035" data-pisa="genetics;213/4/1531" data-pisa-master="genetics;genetics.119.302733" data-apath="/genetics/213/4/1531.atom"><cite class="highwire-cite highwire-citation-jcore-title-only">
  <div class="highwire-cite-title"><a href="https://www.genetics.org/content/213/4/1531" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Fate of Deleterious Variants in a Barley Genomic Prediction Population</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="447044" id="node447044" data-pisa="genetics;213/4/1401" data-pisa-master="genetics;genetics.119.302601" data-apath="/genetics/213/4/1401.atom"><cite class="highwire-cite highwire-citation-jcore-title-only">
  <div class="highwire-cite-title"><a href="https://www.genetics.org/content/213/4/1401" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Comprehensive Scanning Mutagenesis of Human Retrotransposon LINE-1 Identifies Motifs Essential for Function</span></a></div>
</cite>
</div></li></ul></div><div class="highwire-list-footer"><a href="https://www.genetics.org/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article">
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float highwire-article-nav-processed highwire-nav-float-processed"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="https://www.genetics.org/content/177/2/861" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">MODELS AND METHODS</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">RESULTS</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">DISCUSSION</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">References</a></li></ul></div></li><li class="odd"><a href="https://www.genetics.org/content/177/2/861.figures-only" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures &amp; Data</a></li><li class="even"><a href="https://www.genetics.org/content/177/2/861.supplemental" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="https://www.genetics.org/content/177/2/861.article-info" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info &amp; Metrics</a></li><li class="last even"><a href="https://www.genetics.org/content/177/2/861.full.pdf" class="highwire-article-nav-jumplink hw-panels-ajax-tabs-once-processed" data-icon-position="" data-hide-link-title="0" target="_blank"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa">
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="GSALogoWhite_lg.png" style="width: 225px; height: 95px;"></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the 
professional membership organization for scientific researchers and 
educators in the field of genetics. Our members work to advance 
knowledge in the basic mechanisms of inheritance, from the molecular to 
the population level.</p>

<p>Online ISSN: 1943-2631</p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.genetics.org/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="https://www.genetics.org/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="https://www.genetics.org/content/why-subscribe" data-hide-link-title="0" class="" data-icon-position="">For Subscribers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.genetics.org/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="https://www.genetics.org/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="https://www.genetics.org/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected">
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <p><script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
    <style type="text/css">
<!--{cke_protected}{C}%3C!%2D%2D%20p.sniphang%20%7B%20padding-left%3A%2030px%3B%20text-indent%3A%20-30px%3B%20%7D%20%2D%2D%3E--<img alt="" data-cke-saved-src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png">
</style>
</p>
<p><a href="http://byscientistsforscience.org/" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" src="gen-sspa-logo2x.png" width="150"></a></p>
<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>
<p>
    <style type="text/css">
</style>
</p>
<p class="sniphang"><a href="https://www.genetics.org/content/rss"><img alt="RSS" src="rss-square.png" style="width: 24px; height: auto;"> &nbsp;<strong>Subscribe with RSS.</strong></a></p>
<p class="sniphang" style="padding-top:10px;"><a href="https://www.genetics.org/alerts"><img alt="email" src="emailsubscribe.png" style="width: 24px; height: auto;"> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu">
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right">
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright Â© 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu">
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.genetics.org/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About GENETICS</a></li>
<li class="leaf" role="menuitem"><a href="https://www.genetics.org/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="https://www.genetics.org/content/advertising" class="" data-icon-position="" data-hide-link-title="0">Advertising</a></li>
<li class="leaf" role="menuitem"><a href="https://www.genetics.org/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="https://www.genetics.org/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org/" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHM.js"></script>
<script type="text/javascript" src="js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jy.js" async="async"></script>
<script type="text/javascript" src="js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVH.js"></script>
<script type="text/javascript" src="js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOh.js" defer="defer"></script>
<script type="text/javascript" src="js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1s.js"></script>
  

<div id="cboxOverlay" style="display: none;"></div><div id="colorbox" class="" role="dialog" tabindex="-1" style="display: none;"><div id="cboxWrapper"><div><div id="cboxTopLeft" style="float: left;"></div><div id="cboxTopCenter" style="float: left;"></div><div id="cboxTopRight" style="float: left;"></div></div><div style="clear: left;"><div id="cboxMiddleLeft" style="float: left;"></div><div id="cboxContent" style="float: left;"><div id="cboxTitle" style="float: left;" class=""></div><div id="cboxCurrent" style="float: left;"></div><button type="button" id="cboxPrevious"></button><button type="button" id="cboxNext"></button><button id="cboxSlideshow"></button><div id="cboxLoadingOverlay" style="float: left;"></div><div id="cboxLoadingGraphic" style="float: left;"></div></div><div id="cboxMiddleRight" style="float: left;"></div></div><div style="clear: left;"><div id="cboxBottomLeft" style="float: left;"></div><div id="cboxBottomCenter" style="float: left;"></div><div id="cboxBottomRight" style="float: left;"></div></div></div><div style="position: absolute; width: 9999px; visibility: hidden; display: none; max-width: none;"></div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="ui-id-1" aria-labelledby="ui-id-2"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-2" class="ui-dialog-title">&nbsp;</span><button class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" aria-disabled="false" title="close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">close</span></button></div><div class="minipanel-dialog-link-mini ui-dialog-content ui-widget-content" style="" id="ui-id-1"><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward">
  
      
  
  <div class="pane-content">
    <form action="/content/177/2/861" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this Genetics article.</p><p>NOTE: We request 
your email address only to inform the recipient that it was you who 
recommended this article, and that it is not junk mail. We do not retain
 these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-email" name="email" size="58" maxlength="256" class="form-text required" type="text">
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-name" name="name" size="58" maxlength="256" class="form-text required" type="text">
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea><div class="grippie"></div></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="https://www.genetics.org/content/177/2/861" class="active" data-icon-position="" data-hide-link-title="0">Empirical Bayes Inference of Pairwise FST and Its Distribution in the Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable textarea-processed resizable-textarea"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea><div class="grippie"></div></div>
</div>
<input name="path" value="node/334013" type="hidden">
<input name="path_cid" value="" type="hidden">
<input name="forward_footer" value=" " type="hidden">
<input name="form_build_id" value="form-eZcJqDP1vGlaA9Kq-Y0vqY32Db2nrMTy8DDoIE6Bpe4" type="hidden">
<input name="form_id" value="forward_form" type="hidden">
<div class="form-actions form-wrapper" id="edit-actions"><input id="edit-submit" name="op" value="Send Message" class="form-submit" type="submit"></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="ui-id-3" aria-labelledby="ui-id-4"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-4" class="ui-dialog-title">&nbsp;</span><button class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" aria-disabled="false" title="close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">close</span></button></div><div class="minipanel-dialog-link-mini ui-dialog-content ui-widget-content" style="" id="ui-id-3"><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form">
  
      
  
  <div class="pane-content">
    <div id="alerts-form-wrapper"><form action="/content/177/2/861" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-name--2" name="name" size="60" maxlength="128" class="form-text required" type="text">
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" type="password">
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input id="edit-highwire-alerts-email" name="highwire_alerts_email" size="60" maxlength="128" class="form-text required" type="text">
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input id="edit-submit--4" name="op" value="Submit" class="form-submit ajax-processed" type="submit"><input name="form_build_id" value="form-QcdwYKeH0hxZkjGp9Mm5heeSyUzFV_6pYCWbdSr0BsU" type="hidden">
<input name="form_id" value="highwire_alerts_article_sign_up_form" type="hidden">
<input name="current_path" value="content/177/2/861" type="hidden">
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div><div class="ui-dialog ui-widget ui-widget-content ui-corner-all ui-front" style="display: none;" tabindex="-1" role="dialog" aria-describedby="ui-id-5" aria-labelledby="ui-id-6"><div class="ui-dialog-titlebar ui-widget-header ui-corner-all ui-helper-clearfix"><span id="ui-id-6" class="ui-dialog-title">&nbsp;</span><button class="ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only ui-dialog-titlebar-close" role="button" aria-disabled="false" title="close"><span class="ui-button-icon-primary ui-icon ui-icon-closethick"></span><span class="ui-button-text">close</span></button></div><div class="minipanel-dialog-link-mini ui-dialog-content ui-widget-content" style="" id="ui-id-5"><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export">
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="334013" id="node334013--2" data-pisa="genetics;177/2/861" data-pisa-master="genetics;genetics.107.077263" data-apath="/genetics/177/2/861.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-highwire-article-top-a">
  
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Empirical Bayes Inference of Pairwise <em>F</em><sub>ST</sub> and Its Distribution in the Genome</h1>  
    	<div class="highwire-cite-authors"><span class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Shuichi Kitada</span>, <span class="highwire-citation-author" data-delta="1">Toshihide Kitakado</span> and <span class="highwire-citation-author" data-delta="2">Hirohisa Kishino</span></span></div>
  
    	<div class="highwire-cite-metadata"><span class="highwire-cite-metadata-journal highwire-cite-metadata">Genetics </span><span class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2007 </span><span class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 177 </span><span class="highwire-cite-metadata-issue highwire-cite-metadata">no. 2 </span><span class="highwire-cite-metadata-pages highwire-cite-metadata">861-873;  </span><span class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/genetics.107.077263 </span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node334013--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Shuichi Kitada</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shuichi%2BKitada" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kitada%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AShuichi%2BKitada" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Toshihide Kitakado</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Toshihide%2BKitakado" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kitakado%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AToshihide%2BKitakado" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hirohisa Kishino</div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="https://www.genetics.org/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hirohisa%2BKishino" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="https://www.genetics.org/lookup/external-ref?access_num=Kishino%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="https://www.genetics.org/search/author1%3AHirohisa%2BKishino" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=861&amp;rft.epage=873&amp;rft.atitle=Empirical%20Bayes%20Inference%20of%20Pairwise%20FST%20and%20Its%20Distribution%20in%20the%20Genome&amp;rft.volume=177&amp;rft.issue=2&amp;rft.date=2007-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=Genetics&amp;rft.au=Kitada%2C+Shuichi&amp;rft.au=Kitakado%2C+Toshihide&amp;rft.au=Kishino%2C+Hirohisa"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="https://www.genetics.org/highwire/citation/334013/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="https://www.genetics.org/highwire/citation/334013/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="https://www.genetics.org/highwire/citation/334013/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="https://www.genetics.org/highwire/citation/334013/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="https://www.genetics.org/highwire/citation/334013/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="https://www.genetics.org/highwire/citation/334013/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="https://www.genetics.org/highwire/citation/334013/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="https://www.genetics.org/highwire/citation/334013/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="https://www.genetics.org/highwire/citation/334013/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="https://www.genetics.org/highwire/citation/334013/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="https://www.genetics.org/highwire/citation/334013/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="https://www.genetics.org/highwire/citation/334013/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div><script type="text/javascript" id="" src="hermes.js"></script><div id="rmq_pdfTooltip" class="rmq_tooltipCustom rmq_tooltipPosition rmq_pdfTooltipbg" data-init="true" style="display: none"><div class="rmq_tooltiptext rmq_annotatedpdf rmq_adderPdfExplanatoryText">Select REMARQ PDF to open the article including all annotations and conversations.</div><span class="rmq_tooltiptext"><button id="rmq_annotatedpdf"><div class="rmq_adderIcon"><svg xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" width="15" height="19" viewBox="0 0 15 19"><defs><path id="rmqpdf_a" d="M.655 1.39h4.79V.45H.655z"></path><path id="rmqpdf_c" d="M0 18.611h14.101V.001H0v18.61zm1.5-1.5h11.102V1.498H1.5V17.11z"></path><path id="rmqpdf_e" d="M3.046.22c1.31 0 2.4.812 2.4 2.132v2.224c0 1.181-.822 1.947-1.93 2.086v.147c0 .526.148.647.729.647h.139v.83a2.751 2.751 0 0 1-.37.028c-.895 0-1.466-.425-1.466-1.246v-.406C1.44 6.514.655 5.748.655 4.576V2.361c0-1.319 1.072-2.14 2.39-2.14zm1.338 4.218v-1.78c0-.223-.027-.426-.083-.592-.147-.618-.683-.868-1.255-.868-.701 0-1.33.38-1.33 1.31V4.29c0 .221.028.416.075.59.156.61.692.86 1.255.86.7 0 1.338-.362 1.338-1.302z"></path></defs><g fill="none" fill-rule="evenodd"><g transform="translate(4 13)"><mask id="rmqpdf_b" fill="#fff"><use xlink:href="#rmqpdf_a"></use></mask><use fill="#333" xlink:href="#rmqpdf_a"></use><g fill="#F8F8F8" mask="url(#rmqpdf_b)"><path d="M-5-14h16V6H-5z"></path></g></g><mask id="rmqpdf_d" fill="#fff"><use xlink:href="#rmqpdf_c"></use></mask><use fill="#333" xlink:href="#rmqpdf_c"></use><g fill="#F8F8F8" mask="url(#rmqpdf_d)"><path d="M-1-1h16v20H-1z"></path></g><g transform="translate(4 4)"><mask id="rmqpdf_f" fill="#fff"><use xlink:href="#rmqpdf_e"></use></mask><use fill="#333" xlink:href="#rmqpdf_e"></use><g fill="#F8F8F8" mask="url(#rmqpdf_f)"><path d="M-5-5h16v20H-5z"></path></g></g></g></svg></div><div class="rmq_adderText">REMARQ PDF</div></button><button id="rmq_originalpdf"><div class="rmq_adderIcon"><svg xmlns="http://www.w3.org/2000/svg" xlink="http://www.w3.org/1999/xlink" width="14" height="19" viewBox="0 0 14 19"><defs><path id="pdf_a" d="M0 18.611h14.101V.001H0v18.61zm1.5-1.5h11.102V1.498H1.5V17.11z"></path><path id="pdf_c" d="M3.16 5.85l.017-.035c.103-.277.103-.277.172-.449.035-.12.121-.363.311-.914.086-.225.138-.432.19-.604.258.448.465.793.776 1.225.12.155.207.293.31.414l-.155.051c-.12.052-.259.086-.449.156l-.24.086c-.432.138-.674.224-1.07.38.051-.104.085-.207.137-.31M3.556.845a.399.399 0 0 1 .052-.12c.017.017.017.052.034.102.07.38.07.501-.034 1.001-.017-.05-.035-.103-.035-.137-.086-.38-.103-.57-.017-.846m2.916 5.037c.172 0 .344 0 .585.052.173.035.225.103.225.121v.121s-.018.017-.103.034a.847.847 0 0 1-.345.018c-.086-.035-.225-.087-.518-.328.052-.018.104-.018.156-.018M1.417 7.781c-.07.121-.12.242-.172.328-.173.328-.207.396-.31.5-.07.052-.156.017-.208-.017C.71 8.575.658 8.54.78 8.23c.068-.173.397-.38.69-.569 0 .051-.035.085-.052.12m1.26-2.622c-.052.155-.052.155-.156.414l-.017.035a8.76 8.76 0 0 1-.466.982c-.207.104-.414.207-.707.38-.62.362-1.053.621-1.208 1.035-.086.225-.293.777.155 1.14a.846.846 0 0 0 1.105 0c.207-.19.293-.363.465-.692.052-.085.104-.189.172-.327.277-.535.45-.897.484-.949 0-.017.017-.034.034-.052.121-.05.26-.103.414-.172.57-.242.811-.31 1.328-.483l.242-.086c.19-.069.327-.103.466-.155.172-.069.311-.104.448-.155.121.103.242.223.397.344.328.276.553.414.794.483.207.052.5.035.69-.017.604-.155.62-.603.638-.794.017-.259-.138-.707-.81-.828-.31-.052-.518-.052-.708-.052-.19 0-.345.018-.587.052a.983.983 0 0 1-.207.035c-.172-.173-.293-.345-.483-.604-.362-.518-.586-.88-.948-1.518-.052-.086-.104-.189-.156-.276.035-.155.07-.327.104-.534l.052-.243c.137-.655.172-.879.068-1.414-.12-.587-.517-.724-.724-.707-.103 0-.517.07-.69.673-.12.413-.103.724 0 1.155.07.311.225.691.483 1.157-.086.362-.19.69-.379 1.241-.173.57-.259.794-.293.932"></path></defs><g fill="none" fill-rule="evenodd"><mask id="pdf_b" fill="#fff"><use xlink:href="#pdf_a"></use></mask><use fill="#333" xlink:href="#pdf_a"></use><g fill="#F8F8F8" mask="url(#pdf_b)"><path d="M-1-1h16v20H-1z"></path></g><g><g transform="translate(3 5)"><mask id="pdf_d" fill="#fff"><use xlink:href="#pdf_c"></use></mask><use fill="#333" xlink:href="#pdf_c"></use><g fill="#F8F8F8" mask="url(#pdf_d)"><path d="M-1 0H9v10H-1z"></path></g></g></g></g></svg></div><div class="rmq_adderText">PDF</div></button></span></div><iframe id="CornerWidget" src="https://remarqable.com/web/cornerWidget.html" scrolling="no" style="overflow: hidden; visibility: visible;" data-expanded="false" data-signedin="false" data-expl="false" data-pd-id="gsa" data-mobile-user-agent="false" title="Remarq Trigger Frame" frameborder="0"></iframe><iframe id="GroveSide" src="https://remarqable.com/web/leftside.html" scrolling="yes" data-expanded="false" data-pd-id="gsa" data-mobile-user-agent="false" title="Remarq Widget Main" frameborder="0"></iframe></body></html>