<!DOCTYPE html>
<html lang="en"><head><script type="text/javascript" src="https://bam.nr-data.net/1/8d76ca6f26?a=220513093&amp;v=1118.0c07c19&amp;to=ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw%3D%3D&amp;rst=6740&amp;ref=https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbx108/4106928&amp;ap=2352&amp;be=2972&amp;fe=6488&amp;dc=5486&amp;perf=%7B%22timing%22:%7B%22of%22:1555594925718,%22n%22:0,%22f%22:0,%22dn%22:0,%22dne%22:0,%22c%22:0,%22ce%22:0,%22rq%22:0,%22rp%22:2910,%22rpe%22:2910,%22dl%22:2914,%22di%22:5440,%22ds%22:5486,%22de%22:5764,%22dc%22:6488,%22l%22:6488,%22le%22:6526%7D,%22navigation%22:%7B%7D%7D&amp;jsonp=NREUM.setToken"></script><script src="https://js-agent.newrelic.com/nr-1118.min.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"><meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><script async="" src="gtm.js"></script><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"220513093","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2352,"agent":"","atts":""}</script><script type="text/javascript">window.NREUM||(NREUM={}),__nr_require=function(e,n,t){function r(t){if(!n[t]){var o=n[t]={exports:{}};e[t][0].call(o.exports,function(n){var o=e[t][1][n];return r(o||n)},o,o.exports)}return n[t].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<t.length;o++)r(t[o]);return r}({1:[function(e,n,t){function r(){}function o(e,n,t){return function(){return i(e,[c.now()].concat(u(arguments)),n?null:this,t),n?void 0:this}}var i=e("handle"),a=e(3),u=e(4),f=e("ee").get("tracer"),c=e("loader"),s=NREUM;"undefined"==typeof window.newrelic&&(newrelic=s);var p=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],d="api-",l=d+"ixn-";a(p,function(e,n){s[n]=o(d+n,!0,"api")}),s.addPageAction=o(d+"addPageAction",!0),s.setCurrentRouteName=o(d+"routeName",!0),n.exports=newrelic,s.interaction=function(){return(new r).get()};var m=r.prototype={createTracer:function(e,n){var t={},r=this,o="function"==typeof n;return i(l+"tracer",[c.now(),e,t],r),function(){if(f.emit((o?"":"no-")+"fn-start",[c.now(),r,o],t),o)try{return n.apply(this,arguments)}catch(e){throw f.emit("fn-err",[arguments,this,e],t),e}finally{f.emit("fn-end",[c.now()],t)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(e,n){m[n]=o(l+n)}),newrelic.noticeError=function(e,n){"string"==typeof e&&(e=new Error(e)),i("err",[e,c.now(),!1,n])}},{}],2:[function(e,n,t){function r(e,n){if(!o)return!1;if(e!==o)return!1;if(!n)return!0;if(!i)return!1;for(var t=i.split("."),r=n.split("."),a=0;a<r.length;a++)if(r[a]!==t[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var u=navigator.userAgent,f=u.match(a);f&&u.indexOf("Chrome")===-1&&u.indexOf("Chromium")===-1&&(o="Safari",i=f[1])}n.exports={agent:o,version:i,match:r}},{}],3:[function(e,n,t){function r(e,n){var t=[],r="",i=0;for(r in e)o.call(e,r)&&(t[i]=n(r,e[r]),i+=1);return t}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],4:[function(e,n,t){function r(e,n,t){n||(n=0),"undefined"==typeof t&&(t=e?e.length:0);for(var r=-1,o=t-n||0,i=Array(o<0?0:o);++r<o;)i[r]=e[n+r];return i}n.exports=r},{}],5:[function(e,n,t){n.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],ee:[function(e,n,t){function r(){}function o(e){function n(e){return e&&e instanceof r?e:e?f(e,u,i):i()}function t(t,r,o,i){if(!d.aborted||i){e&&e(t,r,o);for(var a=n(o),u=v(t),f=u.length,c=0;c<f;c++)u[c].apply(a,r);var p=s[y[t]];return p&&p.push([b,t,r,a]),a}}function l(e,n){h[e]=v(e).concat(n)}function m(e,n){var t=h[e];if(t)for(var r=0;r<t.length;r++)t[r]===n&&t.splice(r,1)}function v(e){return h[e]||[]}function g(e){return p[e]=p[e]||o(t)}function w(e,n){c(e,function(e,t){n=n||"feature",y[t]=n,n in s||(s[n]=[])})}var h={},y={},b={on:l,addEventListener:l,removeEventListener:m,emit:t,get:g,listeners:v,context:n,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(s.api||s.feature)&&(d.aborted=!0,s=d.backlog={})}var u="nr@context",f=e("gos"),c=e(3),s={},p={},d=n.exports=o();d.backlog=s},{}],gos:[function(e,n,t){function r(e,n,t){if(o.call(e,n))return e[n];var r=t();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(e,n,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return e[n]=r,r}var o=Object.prototype.hasOwnProperty;n.exports=r},{}],handle:[function(e,n,t){function r(e,n,t,r){o.buffer([e],r),o.emit(e,n,t)}var o=e("ee").get("handle");n.exports=r,r.ee=o},{}],id:[function(e,n,t){function r(e){var n=typeof e;return!e||"object"!==n&&"function"!==n?-1:e===window?0:a(e,i,function(){return o++})}var o=1,i="nr@id",a=e("gos");n.exports=r},{}],loader:[function(e,n,t){function r(){if(!E++){var e=x.info=NREUM.info,n=l.getElementsByTagName("script")[0];if(setTimeout(s.abort,3e4),!(e&&e.licenseKey&&e.applicationID&&n))return s.abort();c(y,function(n,t){e[n]||(e[n]=t)}),f("mark",["onload",a()+x.offset],null,"api");var t=l.createElement("script");t.src="https://"+e.agent,n.parentNode.insertBefore(t,n)}}function o(){"complete"===l.readyState&&i()}function i(){f("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(u=Math.max((new Date).getTime(),u))-x.offset}var u=(new Date).getTime(),f=e("handle"),c=e(3),s=e("ee"),p=e(2),d=window,l=d.document,m="addEventListener",v="attachEvent",g=d.XMLHttpRequest,w=g&&g.prototype;NREUM.o={ST:setTimeout,SI:d.setImmediate,CT:clearTimeout,XHR:g,REQ:d.Request,EV:d.Event,PR:d.Promise,MO:d.MutationObserver};var h=""+location,y={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-1118.min.js"},b=g&&w&&w[m]&&!/CriOS/.test(navigator.userAgent),x=n.exports={offset:u,now:a,origin:h,features:{},xhrWrappable:b,userAgent:p};e(1),l[m]?(l[m]("DOMContentLoaded",i,!1),d[m]("load",r,!1)):(l[v]("onreadystatechange",o),d[v]("onload",r)),f("mark",["firstbyte",u],null,"api");var E=0,O=e(5)},{}]},{},["loader"]);</script><link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">
    <link href="style.css" rel="stylesheet" type="text/css">

    
    <link href="jquery.css" rel="stylesheet" type="text/css">

    
    <link href="jscrollpane.css" rel="stylesheet" type="text/css">
    <link href="theme.css" rel="stylesheet" type="text/css">
    
    <link href="css.css" rel="stylesheet" type="text/css">
        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site976621378.css" rel="stylesheet" type="text/css">
    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'bib';
    </script>

    
    <script type="text/javascript" src="jquery-2.js"></script>

    
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">

        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbx108/4106928","@type":"ScholarlyArticle","name":"MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization","datePublished":"2017-09-06","isPartOf":{"@id":"https://academic.oup.com/bib","@type":"Periodical","name":"Briefings in Bioinformatics","issn":["1477-4054"]},"url":"http://dx.doi.org/10.1093/bib/bbx108","keywords":["multiple sequence alignment","sequence analysis","phylogenetic tree"],"inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbx108/4106928","author":[{"name":"Katoh, Kazutaka","@type":"Person","sameAs":"http://orcid.org/0000-0003-4133-8393"},{"name":"Rozewicki, John","@type":"Person"},{"name":"Yamada, Kazunori D.","@type":"Person"}],"description":"Abstract.  This article describes several features in the MAFFT online service for multiple sequence alignment (MSA). As a result of recent advances in sequenci","pageStart":"bbx108","pageEnd":"","thumbnailURL":"https://academic.oup.com/UI/app/img/generic-cover.png","headline":"MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization","image":"https://academic.oup.com/UI/app/img/generic-cover.png"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization">
    <meta property="og:description" content="Abstract.  This article describes several features in the MAFFT online service for multiple sequence alignment (MSA). As a result of recent advances in sequenci">
    <meta property="og:url" content="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbx108/4106928">
    <meta property="og:image" content="https://academic.oup.com/UI/app/img/generic-cover.png">
    <meta property="og:updated_time" content="2017-09-06">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Katoh, Kazutaka"><meta name="citation_author" content="Rozewicki, John"><meta name="citation_author" content="Yamada, Kazunori D."><meta name="citation_title" content="MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization"><meta name="citation_doi" content="10.1093/bib/bbx108"><meta name="citation_journal_title" content="Briefings in Bioinformatics"><meta name="citation_journal_abbrev" content="Brief Bioinform"><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-66; "><meta name="citation_reference" content="citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability;  citation_journal_title=Mol Biol Evol;  citation_year=2013;  citation_volume=30;  citation_issue=4;  citation_pages=772-80; "><meta name="citation_reference" content="citation_title=Using de novo protein structure predictions to measure the quality of very large multiple sequence alignments;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_issue=6;  citation_pages=814-20; "><meta name="citation_reference" content="citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega;  citation_journal_title=Mol Syst Biol;  citation_year=2011;  citation_volume=7;  citation_pages=539."><meta name="citation_reference" content="citation_title=FastSP: linear time calculation of alignment accuracy;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=3250-8; "><meta name="citation_reference" content="citation_title=PartTree: an algorithm to build an approximate tree from a large number of unaligned sequences;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=372-4; "><meta name="citation_reference" content="citation_title=CLUSTAL: a package for performing multiple sequence alignment on a microcomputer;  citation_journal_title=Gene;  citation_year=1988;  citation_volume=73;  citation_pages=237-44; "><meta name="citation_reference" content="citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins;  citation_journal_title=J Mol Biol;  citation_year=1970;  citation_volume=48;  citation_pages=443-53; "><meta name="citation_reference" content="citation_title=The alignment of sets of sequences and the construction of phyletic trees: an integrated method;  citation_journal_title=J Mol Evol;  citation_year=1984;  citation_volume=20;  citation_pages=175-86; "><meta name="citation_reference" content="citation_title=Progressive sequence alignment as a prerequisite to correct phylogenetic trees;  citation_journal_title=J Mol Evol;  citation_year=1987;  citation_volume=25;  citation_pages=351-60; "><meta name="citation_reference" content="citation_publisher=National Biomedical Research Foundation, Maryland;  citation_title=Atlas of Protein Sequence and Structure;  citation_year=1978;  citation_pages=345-52; "><meta name="citation_reference" content="citation_title=Application of the mafft sequence alignment program to large data-reexamination of the usefulness of chained guide trees;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_issue=21;  citation_pages=3246-51; "><meta name="citation_reference" content="citation_title=Simple chained guide trees give high-quality protein multiple sequence alignments;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2014;  citation_volume=111;  citation_pages=10556-61; "><meta name="citation_reference" content="citation_title=A strategy for the rapid multiple alignment of protein sequences. confidence levels from tertiary structure comparisons;  citation_journal_title=J Mol Biol;  citation_year=1987;  citation_volume=198;  citation_pages=327-37; "><meta name="citation_reference" content="citation_title=A novel randomized iterative strategy for aligning multiple protein sequences;  citation_journal_title=Comput Appl Biosci;  citation_year=1991;  citation_volume=7;  citation_pages=479-84; "><meta name="citation_reference" content="citation_title=Optimal alignment between groups of sequences and its application to multiple sequence alignment;  citation_journal_title=Comput Appl Biosci;  citation_year=1993;  citation_volume=9;  citation_pages=361-70; "><meta name="citation_reference" content="citation_title=Adding unaligned sequences into an existing alignment using MAFFT and LAST;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3144-6; "><meta name="citation_reference" content="citation_title=Protein multiple sequence alignment benchmarking through secondary structure prediction;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=33;  citation_issue=9;  citation_pages=1331-7; "><meta name="citation_reference" content="citation_title=COFFEE: an objective function for multiple sequence alignments;  citation_journal_title=Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=407-22; "><meta name="citation_reference" content="citation_title=Systematic exploration of guide-tree topology effects for small protein alignments;  citation_journal_title=BMC Bioinformatics;  citation_year=2014;  citation_volume=15;  citation_pages=338."><meta name="citation_reference" content="citation_title=Simple chained guide trees give poorer multiple sequence alignments than inferred trees in simulation and phylogenetic benchmarks;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2015;  citation_volume=112;  citation_pages=E99-100; "><meta name="citation_reference" content="citation_title=Ultra-large alignments using phylogeny-aware profiles;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=124."><meta name="citation_reference" content="citation_title=Sequence embedding for fast construction of guide trees for multiple sequence alignment;  citation_journal_title=Algorithms Mol Biol;  citation_year=2010;  citation_volume=5;  citation_pages=21."><meta name="citation_reference" content="citation_title=PASTA: ultra-large multiple sequence alignment for nucleotide and amino-acid sequences;  citation_journal_title=J Comput Biol;  citation_year=2015;  citation_volume=22;  citation_issue=5;  citation_pages=377-86; "><meta name="citation_reference" content="citation_title=Hmmer web server: interactive sequence similarity searching;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W29-37; "><meta name="citation_reference" content="citation_title=Aligning short reads to reference alignments and trees;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2068-75; "><meta name="citation_reference" content="citation_title=Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1684-91; "><meta name="citation_reference" content="citation_title=Assessment and refinement of eukaryotic gene structure prediction with gene-structure-aware multiple protein sequence alignment;  citation_journal_title=BMC Bioinformatics;  citation_year=2014;  citation_volume=15;  citation_pages=189."><meta name="citation_reference" content="citation_title=MisPred: a resource for identification of erroneous protein sequences in public databases;  citation_journal_title=Database;  citation_year=2013;  citation_volume=2013;  citation_pages=bat053."><meta name="citation_reference" content="citation_title=A beginner’s guide to eukaryotic genome annotation;  citation_journal_title=Nat Rev Genet;  citation_year=2012;  citation_volume=13;  citation_pages=329-42; "><meta name="citation_reference" content="citation_title=aLeaves facilitates on-demand exploration of metazoan gene family trees on mafft sequence alignment server with enhanced interactivity;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=W22-8; "><meta name="citation_reference" content="citation_title=Clustering of highly homologous sequences to reduce the size of large protein databases;  citation_journal_title=Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_issue=3;  citation_pages=282-3; "><meta name="citation_reference" content="citation_title=MaxAlign: maximizing usable data in an alignment;  citation_journal_title=BMC Bioinformatics;  citation_year=2007;  citation_volume=8;  citation_pages=312."><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;  citation_journal_title=Mol Biol Evol;  citation_year=1987;  citation_volume=4;  citation_issue=4;  citation_pages=406-25; "><meta name="citation_reference" content="citation_title=A statistical method for evaluating systematic relationships;  citation_journal_title=University of Kansas Scientific Bulletin;  citation_year=1958;  citation_volume=28;  citation_pages=1409-38; "><meta name="citation_reference" content="citation_title=Phylo.io: interactive viewing and comparison of large phylogenetic trees on the web;  citation_journal_title=Mol Biol Evol;  citation_year=2016;  citation_volume=33;  citation_issue=8;  citation_pages=2163-6; "><meta name="citation_reference" content="citation_title=phyloXML: XML for evolutionary biology and comparative genomics;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=356."><meta name="citation_reference" content="citation_title=Jalview version 2–a multiple sequence alignment editor and analysis workbench;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_issue=9;  citation_pages=1189-91; "><meta name="citation_reference" content="citation_title=MSAViewer: interactive JavaScript visualization of multiple sequence alignments;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_issue=22;  citation_pages=3501-3; "><meta name="citation_reference" content="citation_title=Making automated multiple alignments of very large numbers of protein sequences;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_issue=8;  citation_pages=989-95; "><meta name="citation_reference" content="citation_title=Assessing the utility of coevolution-based residue-residue contact predictions in a sequence- and structure-rich era;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2013;  citation_volume=110;  citation_pages=15674-9; "><meta name="citation_reference" content="citation_title=Protein structure prediction from sequence variation;  citation_journal_title=Nat Biotechnol;  citation_year=2012;  citation_volume=30;  citation_issue=11;  citation_pages=1072-80; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/bib/advance-article-pdf/doi/10.1093/bib/bbx108/19706293/bbx108.pdf"><meta name="description" content="Abstract.  This article describes several features in the MAFFT online service for multiple sequence alignment (MSA). As a result of recent advances in sequenci">

        <meta name="citation_xml_url" content="https://academic.oup.com/bib/article-xml/doi/10.1093/bib/bbx108/4106928">



    <link rel="canonical" href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbx108/4106928">


    <script async="async" src="gpt.js"></script>
    <script>
      var googletag = googletag || {};
      googletag.cmd = googletag.cmd || [];
    </script>

    

    <script type="text/javascript">
      var gptAdSlots = [];
      googletag.cmd.push(function() {
        var pathChar = '/';
        
        var mapping_ad1 = googletag.sizeMapping().
        addSize([1024, 0], [[970, 90], [728, 90]]).
        addSize([768, 0], [728, 90]).
        addSize([0, 0], [320, 50]).build();
        gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'bib_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
        defineSizeMapping(mapping_ad1).
        addService(googletag.pubads());
        
        var mapping_ad2 = googletag.sizeMapping().
	    addSize([0, 0], []).
        addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
        gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'bib_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
        defineSizeMapping(mapping_ad2).
        addService(googletag.pubads());
        
        var mapping_ad3 = googletag.sizeMapping().
	    addSize([0, 0], []).
        addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
        gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'bib_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
        defineSizeMapping(mapping_ad3).
        addService(googletag.pubads());
        
        var mapping_ad4 = googletag.sizeMapping().
	    addSize([0, 0], []).
        addSize([768, 0], [728, 90]).build();
        gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'bib_Behind_Ad4', [728, 90], 'adBlockFooter').
        defineSizeMapping(mapping_ad4).
        addService(googletag.pubads());
        
        googletag.pubads().setTargeting("jnlspage", "advance-article");
        googletag.pubads().setTargeting("jnlsurl", "bib/advance-article/doi/10.1093/bib/bbx108/4106928");
        googletag.pubads().enableSingleRequest();
        googletag.pubads().collapseEmptyDivs();
        
      });
    </script>
    

    <script>
        var oupcookiepolicy_siteid = "journals";	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=bib&authentication_method=IP&authzrequired=false&account_id=2783535&account_list=2783535,2785688&authnips=197.248.44.39&doi=10.1093/bib/bbx108';
    </script>
<title>
	MAFFT online service: multiple sequence alignment, interactive sequence
 choice and visualization | Briefings in Bioinformatics | Oxford 
Academic
</title><script id="altmetric-embed-js" src="altmetric_badges-71f30fc33cff2fd0a5de8e407f36bda8.js"></script><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019041101.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-f6c32e9d70390ae112e1f1b71326f9b4.css"><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax_LineBox {display: table!important}
.MathJax_LineBox span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>

<body data-sitename="briefingsinbioinformatics" class="off-canvas pg_article" theme-bib="" style="cursor: default;"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div>
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4106928" id="webform">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUIODQwOTY1MjUPZBYCZg9kFgJmD2QWBGYPZBYCAg4PZBYCAgEPZBYCZg8WAh4EVGV4dAXbWzxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0b2gsIEthenV0YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm96ZXdpY2tpLCBKb2huIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYWRhLCBLYXp1bm9yaSBELiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTUFGRlQgb25saW5lIHNlcnZpY2U6IG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCwgaW50ZXJhY3RpdmUgc2VxdWVuY2UgY2hvaWNlIGFuZCB2aXN1YWxpemF0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9iaWIvYmJ4MTA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkJyaWVmaW5ncyBpbiBCaW9pbmZvcm1hdGljcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iQnJpZWYgQmlvaW5mb3JtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFGRlQ6IGEgbm92ZWwgbWV0aG9kIGZvciByYXBpZCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYmFzZWQgb24gZmFzdCBGb3VyaWVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zMDU5LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgc29mdHdhcmUgdmVyc2lvbiA3OiBpbXByb3ZlbWVudHMgaW4gcGVyZm9ybWFuY2UgYW5kIHVzYWJpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sIEJpb2wgRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX2lzc3VlPTQ7ICBjaXRhdGlvbl9wYWdlcz03NzItODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIGRlIG5vdm8gcHJvdGVpbiBzdHJ1Y3R1cmUgcHJlZGljdGlvbnMgdG8gbWVhc3VyZSB0aGUgcXVhbGl0eSBvZiB2ZXJ5IGxhcmdlIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25faXNzdWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTgxNC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCwgc2NhbGFibGUgZ2VuZXJhdGlvbiBvZiBoaWdoLXF1YWxpdHkgcHJvdGVpbiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzIHVzaW5nIENsdXN0YWwgT21lZ2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbCBTeXN0IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTUzOS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0U1A6IGxpbmVhciB0aW1lIGNhbGN1bGF0aW9uIG9mIGFsaWdubWVudCBhY2N1cmFjeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zMjUwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhcnRUcmVlOiBhbiBhbGdvcml0aG0gdG8gYnVpbGQgYW4gYXBwcm94aW1hdGUgdHJlZSBmcm9tIGEgbGFyZ2UgbnVtYmVyIG9mIHVuYWxpZ25lZCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MzcyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUw6IGEgcGFja2FnZSBmb3IgcGVyZm9ybWluZyBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgb24gYSBtaWNyb2NvbXB1dGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9MjM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmVyYWwgbWV0aG9kIGFwcGxpY2FibGUgdG8gdGhlIHNlYXJjaCBmb3Igc2ltaWxhcml0aWVzIGluIHRoZSBhbWlubyBhY2lkIHNlcXVlbmNlIG9mIHR3byBwcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SiBNb2wgQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk3MDsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTQ0My01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFsaWdubWVudCBvZiBzZXRzIG9mIHNlcXVlbmNlcyBhbmQgdGhlIGNvbnN0cnVjdGlvbiBvZiBwaHlsZXRpYyB0cmVlczogYW4gaW50ZWdyYXRlZCBtZXRob2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUogTW9sIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2dyZXNzaXZlIHNlcXVlbmNlIGFsaWdubWVudCBhcyBhIHByZXJlcXVpc2l0ZSB0byBjb3JyZWN0IHBoeWxvZ2VuZXRpYyB0cmVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SiBNb2wgRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTM1MS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPU5hdGlvbmFsIEJpb21lZGljYWwgUmVzZWFyY2ggRm91bmRhdGlvbiwgTWFyeWxhbmQ7ICBjaXRhdGlvbl90aXRsZT1BdGxhcyBvZiBQcm90ZWluIFNlcXVlbmNlIGFuZCBTdHJ1Y3R1cmU7ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl9wYWdlcz0zNDUtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIHRoZSBtYWZmdCBzZXF1ZW5jZSBhbGlnbm1lbnQgcHJvZ3JhbSB0byBsYXJnZSBkYXRhLXJlZXhhbWluYXRpb24gb2YgdGhlIHVzZWZ1bG5lc3Mgb2YgY2hhaW5lZCBndWlkZSB0cmVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9pc3N1ZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTMyNDYtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBjaGFpbmVkIGd1aWRlIHRyZWVzIGdpdmUgaGlnaC1xdWFsaXR5IHByb3RlaW4gbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYyBOYXRsIEFjYWQgU2NpIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xMDU1Ni02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdHJhdGVneSBmb3IgdGhlIHJhcGlkIG11bHRpcGxlIGFsaWdubWVudCBvZiBwcm90ZWluIHNlcXVlbmNlcy4gY29uZmlkZW5jZSBsZXZlbHMgZnJvbSB0ZXJ0aWFyeSBzdHJ1Y3R1cmUgY29tcGFyaXNvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUogTW9sIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MTk4OyAgY2l0YXRpb25fcGFnZXM9MzI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIHJhbmRvbWl6ZWQgaXRlcmF0aXZlIHN0cmF0ZWd5IGZvciBhbGlnbmluZyBtdWx0aXBsZSBwcm90ZWluIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHV0IEFwcGwgQmlvc2NpOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NzktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltYWwgYWxpZ25tZW50IGJldHdlZW4gZ3JvdXBzIG9mIHNlcXVlbmNlcyBhbmQgaXRzIGFwcGxpY2F0aW9uIHRvIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHV0IEFwcGwgQmlvc2NpOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkZGluZyB1bmFsaWduZWQgc2VxdWVuY2VzIGludG8gYW4gZXhpc3RpbmcgYWxpZ25tZW50IHVzaW5nIE1BRkZUIGFuZCBMQVNUOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTMxNDQtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYmVuY2htYXJraW5nIHRocm91Z2ggc2Vjb25kYXJ5IHN0cnVjdHVyZSBwcmVkaWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX2lzc3VlPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNPRkZFRTogYW4gb2JqZWN0aXZlIGZ1bmN0aW9uIGZvciBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQwNy0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBleHBsb3JhdGlvbiBvZiBndWlkZS10cmVlIHRvcG9sb2d5IGVmZmVjdHMgZm9yIHNtYWxsIHByb3RlaW4gYWxpZ25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MzM4LiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBjaGFpbmVkIGd1aWRlIHRyZWVzIGdpdmUgcG9vcmVyIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudHMgdGhhbiBpbmZlcnJlZCB0cmVlcyBpbiBzaW11bGF0aW9uIGFuZCBwaHlsb2dlbmV0aWMgYmVuY2htYXJrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYyBOYXRsIEFjYWQgU2NpIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz1FOTktMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYS1sYXJnZSBhbGlnbm1lbnRzIHVzaW5nIHBoeWxvZ2VueS1hd2FyZSBwcm9maWxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMjQuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgZW1iZWRkaW5nIGZvciBmYXN0IGNvbnN0cnVjdGlvbiBvZiBndWlkZSB0cmVlcyBmb3IgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbGdvcml0aG1zIE1vbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yMS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQVNUQTogdWx0cmEtbGFyZ2UgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IGZvciBudWNsZW90aWRlIGFuZCBhbWluby1hY2lkIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SiBDb21wdXQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX2lzc3VlPTU7ICBjaXRhdGlvbl9wYWdlcz0zNzctODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhtbWVyIHdlYiBzZXJ2ZXI6IGludGVyYWN0aXZlIHNlcXVlbmNlIHNpbWlsYXJpdHkgc2VhcmNoaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPVcyOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxpZ25pbmcgc2hvcnQgcmVhZHMgdG8gcmVmZXJlbmNlIGFsaWdubWVudHMgYW5kIHRyZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIwNjgtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY3VyYXRlIGV4dGVuc2lvbiBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzIHVzaW5nIGEgcGh5bG9nZW55LWF3YXJlIGdyYXBoIGFsZ29yaXRobTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNjg0LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IGFuZCByZWZpbmVtZW50IG9mIGV1a2FyeW90aWMgZ2VuZSBzdHJ1Y3R1cmUgcHJlZGljdGlvbiB3aXRoIGdlbmUtc3RydWN0dXJlLWF3YXJlIG11bHRpcGxlIHByb3RlaW4gc2VxdWVuY2UgYWxpZ25tZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xODkuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWlzUHJlZDogYSByZXNvdXJjZSBmb3IgaWRlbnRpZmljYXRpb24gb2YgZXJyb25lb3VzIHByb3RlaW4gc2VxdWVuY2VzIGluIHB1YmxpYyBkYXRhYmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURhdGFiYXNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwMTM7ICBjaXRhdGlvbl9wYWdlcz1iYXQwNTMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBiZWdpbm5lcuKAmXMgZ3VpZGUgdG8gZXVrYXJ5b3RpYyBnZW5vbWUgYW5ub3RhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IFJldiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTMyOS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9YUxlYXZlcyBmYWNpbGl0YXRlcyBvbi1kZW1hbmQgZXhwbG9yYXRpb24gb2YgbWV0YXpvYW4gZ2VuZSBmYW1pbHkgdHJlZXMgb24gbWFmZnQgc2VxdWVuY2UgYWxpZ25tZW50IHNlcnZlciB3aXRoIGVuaGFuY2VkIGludGVyYWN0aXZpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9VzIyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0ZXJpbmcgb2YgaGlnaGx5IGhvbW9sb2dvdXMgc2VxdWVuY2VzIHRvIHJlZHVjZSB0aGUgc2l6ZSBvZiBsYXJnZSBwcm90ZWluIGRhdGFiYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9pc3N1ZT0zOyAgY2l0YXRpb25fcGFnZXM9MjgyLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1heEFsaWduOiBtYXhpbWl6aW5nIHVzYWJsZSBkYXRhIGluIGFuIGFsaWdubWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMTIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG5laWdoYm9yLWpvaW5pbmcgbWV0aG9kOiBhIG5ldyBtZXRob2QgZm9yIHJlY29uc3RydWN0aW5nIHBoeWxvZ2VuZXRpYyB0cmVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sIEJpb2wgRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25faXNzdWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdGF0aXN0aWNhbCBtZXRob2QgZm9yIGV2YWx1YXRpbmcgc3lzdGVtYXRpYyByZWxhdGlvbnNoaXBzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Vbml2ZXJzaXR5IG9mIEthbnNhcyBTY2llbnRpZmljIEJ1bGxldGluOyAgY2l0YXRpb25feWVhcj0xOTU4OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTQwOS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG8uaW86IGludGVyYWN0aXZlIHZpZXdpbmcgYW5kIGNvbXBhcmlzb24gb2YgbGFyZ2UgcGh5bG9nZW5ldGljIHRyZWVzIG9uIHRoZSB3ZWI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbCBCaW9sIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9pc3N1ZT04OyAgY2l0YXRpb25fcGFnZXM9MjE2My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1waHlsb1hNTDogWE1MIGZvciBldm9sdXRpb25hcnkgYmlvbG9neSBhbmQgY29tcGFyYXRpdmUgZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTM1Ni4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1KYWx2aWV3IHZlcnNpb24gMuKAk2EgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IGVkaXRvciBhbmQgYW5hbHlzaXMgd29ya2JlbmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX2lzc3VlPTk7ICBjaXRhdGlvbl9wYWdlcz0xMTg5LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NU0FWaWV3ZXI6IGludGVyYWN0aXZlIEphdmFTY3JpcHQgdmlzdWFsaXphdGlvbiBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX2lzc3VlPTIyOyAgY2l0YXRpb25fcGFnZXM9MzUwMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWtpbmcgYXV0b21hdGVkIG11bHRpcGxlIGFsaWdubWVudHMgb2YgdmVyeSBsYXJnZSBudW1iZXJzIG9mIHByb3RlaW4gc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX2lzc3VlPTg7ICBjaXRhdGlvbl9wYWdlcz05ODktOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc2luZyB0aGUgdXRpbGl0eSBvZiBjb2V2b2x1dGlvbi1iYXNlZCByZXNpZHVlLXJlc2lkdWUgY29udGFjdCBwcmVkaWN0aW9ucyBpbiBhIHNlcXVlbmNlLSBhbmQgc3RydWN0dXJlLXJpY2ggZXJhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jIE5hdGwgQWNhZCBTY2kgVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTE1Njc0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3RlaW4gc3RydWN0dXJlIHByZWRpY3Rpb24gZnJvbSBzZXF1ZW5jZSB2YXJpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdCBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25faXNzdWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDcyLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vYmliL2FkdmFuY2UtYXJ0aWNsZS1wZGYvZG9pLzEwLjEwOTMvYmliL2JieDEwOC8xOTcwNjI5My9iYngxMDgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoaXMgYXJ0aWNsZSBkZXNjcmliZXMgc2V2ZXJhbCBmZWF0dXJlcyBpbiB0aGUgTUFGRlQgb25saW5lIHNlcnZpY2UgZm9yIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCAoTVNBKS4gQXMgYSByZXN1bHQgb2YgcmVjZW50IGFkdmFuY2VzIGluIHNlcXVlbmNpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFfy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MTA2OTI4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQmc2l0ZUlkPTUxNDMeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZE2mWCEg2F0dFfzm5e0CEfQpNFSD" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5143" type="hidden">
        <input id="hdnAdDelaySeconds" value="3000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="default" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/bib_Behind_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">International Institue of Tropical Agriculture (IITA) <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5143&amp;returnUrl=%2fbib%2fadvance-article%2fdoi%2f10.1093%2fbib%2fbbx108%2f4106928" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Briefings in Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            International Institue of Tropical Agriculture (IITA)
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5143&amp;returnUrl=%2fbib%2fadvance-article%2fdoi%2f10.1093%2fbib%2fbbx108%2f4106928" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224036">
            <a href="https://academic.oup.com/bib/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224037">
            <a href="https://academic.oup.com/bib/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224034">
            <a href="https://academic.oup.com/bib/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224038">
            <a href="https://academic.oup.com/bib/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224039">
            <a href="http://mc.manuscriptcentral.com/bib" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224040">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224041">
            <a href="https://academic.oup.com/bib/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224042">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224035">
            <a href="https://academic.oup.com/bib/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224043">
            <a href="https://academic.oup.com/bib/pages/About" class="nav-link ">About Briefings in Bioinformatics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224044">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;search=&amp;t732=470054&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224045">
            <a href="https://academic.oup.com/bib/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224046">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224047">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/bib" class=" journal-logo-container">

            <img id="logo-BriefingsinBioinformatics" class="journal-logo" src="bib_title526563102.svg" alt="Briefings in Bioinformatics">

                </a>
            <div class="society-logo-block">
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224036">
            <a href="https://academic.oup.com/bib/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224037">
            <a href="https://academic.oup.com/bib/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224034">
            <a href="https://academic.oup.com/bib/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224038">
            <a href="https://academic.oup.com/bib/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224039">
            <a href="http://mc.manuscriptcentral.com/bib" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224040">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224041">
            <a href="https://academic.oup.com/bib/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224042">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1224035">
            <a href="https://academic.oup.com/bib/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224043">
            <a href="https://academic.oup.com/bib/pages/About" class="nav-link ">About Briefings in Bioinformatics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224044">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;search=&amp;t732=470054&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224045">
            <a href="https://academic.oup.com/bib/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224046">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1224047">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Briefings in Bioinformatics</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bib/advanced-search?page=1&amp;fl_SiteID=5143&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5143" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/bib" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="bib" type="hidden"> 
    </div>

        </section>
        <main id="main" class="js-main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                    

                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="96239289">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239289">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239292">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239292">MSA of large data</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239304">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239304">Selecting suitable strategies</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239311">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239311">Use of existing MSA</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239316">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239316">Interactive sequence choice and visualization</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239322">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239322">Necessity of large MSAs</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239324">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239324">Command-line options</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239367">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239367">Acknowledgement</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="96239371">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#96239371">References</a>
            </div>
        </li>
</ul>
 
    </div>

	                </div>
            
                    
                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
                    

            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazutaka Katoh</div>
    </div>


                <div class="info-card-note">
                    <a reveal-id="bbx108-cor1" data-open="bbx108-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>Corresponding author: Kazutaka Katoh, 3-1 Yamadaoka, Suita, Osaka 565-0871, JAPAN. E-mail: <a href="mailto:katoh@ifrec.osaka-u.ac.jp" target="">katoh@ifrec.osaka-u.ac.jp</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-4133-8393">
                <img class="orchid-icon" alt="" src="mini-icon.png">&nbsp;&nbsp;http://orcid.org/0000-0003-4133-8393
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bib/search-results?f_Authors=Kazutaka+Katoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katoh%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Katoh%20Kazutaka%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazutaka Katoh<i class="icon-general-mail"></i></a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John Rozewicki</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bib/search-results?f_Authors=John+Rozewicki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rozewicki%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Rozewicki%20John%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">John Rozewicki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazunori D. Yamada</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bib/search-results?f_Authors=Kazunori+D.+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yamada%20Kazunori%20D.%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazunori D. Yamada</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Briefings in Bioinformatics</em>, bbx108, <a href="https://doi.org/10.1093/bib/bbx108">https://doi.org/10.1093/bib/bbx108</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 June 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">27 July 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap">
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4106928" href="https://academic.oup.com/bib/advance-article-pdf/doi/10.1093/bib/bbx108/19706293/bbx108.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Kazutaka Katoh, John Rozewicki, Kazunori D. Yamada,  MAFFT online
 service: multiple sequence alignment, interactive sequence choice and 
visualization, <em>Briefings in Bioinformatics</em>, , bbx108, <a href="https://doi.org/10.1093/bib/bbx108">https://doi.org/10.1093/bib/bbx108</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4106928&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
        <div class="module-widget">
    <i class="icon-menu_permissions">
        <span class="screenreader-text">Permissions Icon</span>
    </i>
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1477-4054&amp;title=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&amp;publicationDate=2017-09-06&amp;volumeNum=&amp;issueNum=&amp;author=Katoh%2C%20Kazutaka%3B%20Rozewicki%2C%20John&amp;startPage=1&amp;endPage=7&amp;contentId=10.1093%2Fbib%2Fbbx108&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
</div>
 
    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                    <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Briefings in Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            International Institue of Tropical Agriculture (IITA)
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5143&amp;returnUrl=%2fbib%2fadvance-article%2fdoi%2f10.1093%2fbib%2fbbx108%2f4106928" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Briefings in Bioinformatics</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bib/advanced-search?page=1&amp;fl_SiteID=5143&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="bib" type="hidden"> 
    </div>

                        </div>
                    </div>                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="96239289" id="96239289" class="abstract-title">Abstract</h2>
<section class="abstract"><p>This article describes several features in 
the MAFFT online service for multiple sequence alignment (MSA). As a 
result of recent advances in sequencing technologies, huge numbers of 
biological sequences are available and the need for MSAs with large 
numbers of sequences is increasing. To extract biologically relevant 
information from such data, sophistication of algorithms is necessary 
but not sufficient. Intuitive and interactive tools for experimental 
biologists to semiautomatically handle large data are becoming 
important. We are working on development of MAFFT toward these two 
directions. Here, we explain (i) the Web interface for recently 
developed options for large data and (ii) interactive usage to refine 
sequence data sets and MSAs.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;multiple sequence alignment&quot;">multiple sequence alignment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence analysis&quot;">sequence analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic tree&quot;">phylogenetic tree</a></div><div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/bib/search-results?f_TocHeadingTitle=Paper">Paper</a>
        </div></div>
<p>Multiple sequence alignment (MSA) is an important step in comparative
 analyses of biological sequences. We provide an online service for 
computing MSAs on the Web using MAFFT [<a reveal-id="bbx108-B1" data-open="bbx108-B1" class="link link-ref link-reveal xref-bibr">1</a>, <a reveal-id="bbx108-B2" data-open="bbx108-B2" class="link link-ref link-reveal xref-bibr">2</a>].
 MAFFT has several different options for computing large MSAs consisting
 of thousands of sequences. Our service also has some additional 
functions (interactive sequence selection and phylogenetic inference) 
for preprocessing and postprocessing MSA. Moreover, these processes can 
be circularly performed as necessary. Here, we describe usage of these 
functions, including recently added ones, and several tips for using our
 online service.</p>                            <h2 scrollto-destination="96239292" id="96239292" class="section-title">MSA of large data</h2>
<p>The demand for MSAs with a large number of sequences is increasing 
along with the advance of sequencing technologies. The default option of
 MAFFT, FFT-NS-2, is applicable to most cases, but MAFFT has more 
options for constructing large MSAs. They can be selected in a 
designated page for large alignment on the MAFFT server: <a class="link link-uri" href="http://mafft.cbrc.jp/alignment/server/large.html" target="">http://mafft.cbrc.jp/alignment/server/large.html</a>. Below, we briefly explain the options available on this page. Headings (A)–(G) correspond to those in <a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1</a>. Benchmark results of these options are shown in <a reveal-id="bbx108-T1" data-open="bbx108-T1" class="link link-reveal link-table xref-fig">Table 1</a>. Commands for locally running those options are available in the last section. </p><div content-id="bbx108-T1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="bbx108-T1" data-id="bbx108-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Results of two different benchmarks, ContTest (136 entries, 1467–43 912 sequences) [<a reveal-id="bbx108-B3" data-open="bbx108-B3" class="link link-ref link-reveal xref-bibr">3</a>] and HomFam (89 entries; 93–93 681 sequences) [<a reveal-id="bbx108-B4" data-open="bbx108-B4" class="link link-ref link-reveal xref-bibr">4</a>], for some MAFFT options available on our online server</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="2">Method&nbsp;</th><th colspan="2">ContTest<hr>&nbsp;</th><th colspan="2">HomFam<hr>&nbsp;</th></tr><tr><th>Accuracy score&nbsp;</th><th>CPU time (minutes)&nbsp;</th><th><strong>Accuracy score (SP/TC)</strong>&nbsp;</th><th>CPU time (minutes)&nbsp;</th></tr></thead><tbody><tr><td rowspan="4"><strong>A</strong>&nbsp;</td><td>PartTree (partsize = 50)&nbsp;</td><td>0.4103&nbsp;</td><td>61&nbsp;</td><td>0.7862/0.5658&nbsp;</td><td>47&nbsp;</td></tr><tr><td>PartTree (partsize = 1000)&nbsp;</td><td>0.4364&nbsp;</td><td>140&nbsp;</td><td>0.8258/0.6377&nbsp;</td><td>94&nbsp;</td></tr><tr><td>DPPartTree (partsize = 50)&nbsp;</td><td>0.4424&nbsp;</td><td>210&nbsp;</td><td>0.8413/0.6597&nbsp;</td><td>160&nbsp;</td></tr><tr><td>DPPartTree (partsize = 1000)&nbsp;</td><td>0.4632&nbsp;</td><td>1000&nbsp;</td><td>0.8541/0.6934&nbsp;</td><td>820&nbsp;</td></tr><tr><td><strong>B</strong>&nbsp;</td><td>FFT-NS-1&nbsp;</td><td>0.4856&nbsp;</td><td>170&nbsp;</td><td>0.8491/0.6669&nbsp;</td><td>160&nbsp;</td></tr><tr><td><strong>B</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-1 (memsavetree)&nbsp;</td><td>0.4835&nbsp;</td><td>280&nbsp;</td><td>0.8416/0.6667&nbsp;</td><td>260&nbsp;</td></tr><tr><td><strong>D</strong>&nbsp;</td><td>FFT-NS-2&nbsp;</td><td>0.4998&nbsp;</td><td>500&nbsp;</td><td>0.8759/0.7162&nbsp;</td><td>460&nbsp;</td></tr><tr><td><strong>D</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-2 (memsavetree)&nbsp;</td><td>0.5099&nbsp;</td><td>1100&nbsp;</td><td>0.8611/0.7023&nbsp;</td><td>990&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100)&nbsp;</td><td>0.5153&nbsp;</td><td>730&nbsp;</td><td>0.8821/0.7274&nbsp;</td><td>650&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500)&nbsp;</td><td>0.5361&nbsp;</td><td>1200&nbsp;</td><td>0.8970/0.7586&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000)&nbsp;</td><td>0.5440&nbsp;</td><td>3400&nbsp;</td><td>0.9075/0.7810&nbsp;</td><td>4400&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>+<strong>C</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100, memsavetree)&nbsp;</td><td>0.5298&nbsp;</td><td>1500&nbsp;</td><td>0.8845/0.7416&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500, memsavetree)&nbsp;</td><td>0.5438&nbsp;</td><td>2000&nbsp;</td><td>0.8995/0.7638&nbsp;</td><td>2000&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000, memsavetree)&nbsp;</td><td>0.5428&nbsp;</td><td>4200&nbsp;</td><td>0.9052/0.7826&nbsp;</td><td>5000&nbsp;</td></tr><tr><td><strong>F</strong>&nbsp;</td><td>G-INS-1&nbsp;</td><td>0.5696&nbsp;</td><td>55 000&nbsp;</td><td>0.9306/0.8288&nbsp;</td><td>49000&nbsp;</td></tr><tr><td><strong>G</strong>&nbsp;</td><td>Randomchain&nbsp;</td><td>0.5425&nbsp;</td><td>100&nbsp;</td><td>0.8349/0.6681&nbsp;</td><td>88&nbsp;</td></tr></tbody></table></div><div content-id="bbx108-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="2">Method&nbsp;</th><th colspan="2">ContTest<hr>&nbsp;</th><th colspan="2">HomFam<hr>&nbsp;</th></tr><tr><th>Accuracy score&nbsp;</th><th>CPU time (minutes)&nbsp;</th><th><strong>Accuracy score (SP/TC)</strong>&nbsp;</th><th>CPU time (minutes)&nbsp;</th></tr></thead><tbody><tr><td rowspan="4"><strong>A</strong>&nbsp;</td><td>PartTree (partsize = 50)&nbsp;</td><td>0.4103&nbsp;</td><td>61&nbsp;</td><td>0.7862/0.5658&nbsp;</td><td>47&nbsp;</td></tr><tr><td>PartTree (partsize = 1000)&nbsp;</td><td>0.4364&nbsp;</td><td>140&nbsp;</td><td>0.8258/0.6377&nbsp;</td><td>94&nbsp;</td></tr><tr><td>DPPartTree (partsize = 50)&nbsp;</td><td>0.4424&nbsp;</td><td>210&nbsp;</td><td>0.8413/0.6597&nbsp;</td><td>160&nbsp;</td></tr><tr><td>DPPartTree (partsize = 1000)&nbsp;</td><td>0.4632&nbsp;</td><td>1000&nbsp;</td><td>0.8541/0.6934&nbsp;</td><td>820&nbsp;</td></tr><tr><td><strong>B</strong>&nbsp;</td><td>FFT-NS-1&nbsp;</td><td>0.4856&nbsp;</td><td>170&nbsp;</td><td>0.8491/0.6669&nbsp;</td><td>160&nbsp;</td></tr><tr><td><strong>B</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-1 (memsavetree)&nbsp;</td><td>0.4835&nbsp;</td><td>280&nbsp;</td><td>0.8416/0.6667&nbsp;</td><td>260&nbsp;</td></tr><tr><td><strong>D</strong>&nbsp;</td><td>FFT-NS-2&nbsp;</td><td>0.4998&nbsp;</td><td>500&nbsp;</td><td>0.8759/0.7162&nbsp;</td><td>460&nbsp;</td></tr><tr><td><strong>D</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-2 (memsavetree)&nbsp;</td><td>0.5099&nbsp;</td><td>1100&nbsp;</td><td>0.8611/0.7023&nbsp;</td><td>990&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100)&nbsp;</td><td>0.5153&nbsp;</td><td>730&nbsp;</td><td>0.8821/0.7274&nbsp;</td><td>650&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500)&nbsp;</td><td>0.5361&nbsp;</td><td>1200&nbsp;</td><td>0.8970/0.7586&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000)&nbsp;</td><td>0.5440&nbsp;</td><td>3400&nbsp;</td><td>0.9075/0.7810&nbsp;</td><td>4400&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>+<strong>C</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100, memsavetree)&nbsp;</td><td>0.5298&nbsp;</td><td>1500&nbsp;</td><td>0.8845/0.7416&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500, memsavetree)&nbsp;</td><td>0.5438&nbsp;</td><td>2000&nbsp;</td><td>0.8995/0.7638&nbsp;</td><td>2000&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000, memsavetree)&nbsp;</td><td>0.5428&nbsp;</td><td>4200&nbsp;</td><td>0.9052/0.7826&nbsp;</td><td>5000&nbsp;</td></tr><tr><td><strong>F</strong>&nbsp;</td><td>G-INS-1&nbsp;</td><td>0.5696&nbsp;</td><td>55 000&nbsp;</td><td>0.9306/0.8288&nbsp;</td><td>49000&nbsp;</td></tr><tr><td><strong>G</strong>&nbsp;</td><td>Randomchain&nbsp;</td><td>0.5425&nbsp;</td><td>100&nbsp;</td><td>0.8349/0.6681&nbsp;</td><td>88&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>Note</em>: The sum-of-pairs (SP) and total-column (TC) scores for HomFam were calculated by the FastSP program [<a reveal-id="bbx108-B5" data-open="bbx108-B5" class="link link-ref link-reveal xref-bibr">5</a>]. (<strong>A</strong>–<strong>G</strong>)
 correspond to the techniques explained in the main text. Command-line 
arguments are displayed after performing the calculation on the online 
service and also listed in the main text. Random numbers are used in (<strong>A</strong>), (<strong>E</strong>) and (<strong>G</strong>). In this test, only one set of random numbers was used for each method. For (<strong>E</strong>) and (<strong>G</strong>),
 seed of random numbers can be specified in the download version (see 
the last section in the main text) but cannot be specified in the online
 version. See <a class="link link-uri" href="https://mafft.sb.ecei.tohoku.ac.jp/" target="">https://mafft.sb.ecei.tohoku.ac.jp/</a> for detailed results.</p></div></div></div></div></div><div class="table-wrap"><div id="bbx108-T1" data-id="bbx108-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Results of two different benchmarks, ContTest (136 entries, 1467–43 912 sequences) [<a reveal-id="bbx108-B3" data-open="bbx108-B3" class="link link-ref link-reveal xref-bibr">3</a>] and HomFam (89 entries; 93–93 681 sequences) [<a reveal-id="bbx108-B4" data-open="bbx108-B4" class="link link-ref link-reveal xref-bibr">4</a>], for some MAFFT options available on our online server</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="2">Method&nbsp;</th><th colspan="2">ContTest<hr>&nbsp;</th><th colspan="2">HomFam<hr>&nbsp;</th></tr><tr><th>Accuracy score&nbsp;</th><th>CPU time (minutes)&nbsp;</th><th><strong>Accuracy score (SP/TC)</strong>&nbsp;</th><th>CPU time (minutes)&nbsp;</th></tr></thead><tbody><tr><td rowspan="4"><strong>A</strong>&nbsp;</td><td>PartTree (partsize = 50)&nbsp;</td><td>0.4103&nbsp;</td><td>61&nbsp;</td><td>0.7862/0.5658&nbsp;</td><td>47&nbsp;</td></tr><tr><td>PartTree (partsize = 1000)&nbsp;</td><td>0.4364&nbsp;</td><td>140&nbsp;</td><td>0.8258/0.6377&nbsp;</td><td>94&nbsp;</td></tr><tr><td>DPPartTree (partsize = 50)&nbsp;</td><td>0.4424&nbsp;</td><td>210&nbsp;</td><td>0.8413/0.6597&nbsp;</td><td>160&nbsp;</td></tr><tr><td>DPPartTree (partsize = 1000)&nbsp;</td><td>0.4632&nbsp;</td><td>1000&nbsp;</td><td>0.8541/0.6934&nbsp;</td><td>820&nbsp;</td></tr><tr><td><strong>B</strong>&nbsp;</td><td>FFT-NS-1&nbsp;</td><td>0.4856&nbsp;</td><td>170&nbsp;</td><td>0.8491/0.6669&nbsp;</td><td>160&nbsp;</td></tr><tr><td><strong>B</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-1 (memsavetree)&nbsp;</td><td>0.4835&nbsp;</td><td>280&nbsp;</td><td>0.8416/0.6667&nbsp;</td><td>260&nbsp;</td></tr><tr><td><strong>D</strong>&nbsp;</td><td>FFT-NS-2&nbsp;</td><td>0.4998&nbsp;</td><td>500&nbsp;</td><td>0.8759/0.7162&nbsp;</td><td>460&nbsp;</td></tr><tr><td><strong>D</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-2 (memsavetree)&nbsp;</td><td>0.5099&nbsp;</td><td>1100&nbsp;</td><td>0.8611/0.7023&nbsp;</td><td>990&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100)&nbsp;</td><td>0.5153&nbsp;</td><td>730&nbsp;</td><td>0.8821/0.7274&nbsp;</td><td>650&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500)&nbsp;</td><td>0.5361&nbsp;</td><td>1200&nbsp;</td><td>0.8970/0.7586&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000)&nbsp;</td><td>0.5440&nbsp;</td><td>3400&nbsp;</td><td>0.9075/0.7810&nbsp;</td><td>4400&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>+<strong>C</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100, memsavetree)&nbsp;</td><td>0.5298&nbsp;</td><td>1500&nbsp;</td><td>0.8845/0.7416&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500, memsavetree)&nbsp;</td><td>0.5438&nbsp;</td><td>2000&nbsp;</td><td>0.8995/0.7638&nbsp;</td><td>2000&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000, memsavetree)&nbsp;</td><td>0.5428&nbsp;</td><td>4200&nbsp;</td><td>0.9052/0.7826&nbsp;</td><td>5000&nbsp;</td></tr><tr><td><strong>F</strong>&nbsp;</td><td>G-INS-1&nbsp;</td><td>0.5696&nbsp;</td><td>55 000&nbsp;</td><td>0.9306/0.8288&nbsp;</td><td>49000&nbsp;</td></tr><tr><td><strong>G</strong>&nbsp;</td><td>Randomchain&nbsp;</td><td>0.5425&nbsp;</td><td>100&nbsp;</td><td>0.8349/0.6681&nbsp;</td><td>88&nbsp;</td></tr></tbody></table></div><div content-id="bbx108-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="2">Method&nbsp;</th><th colspan="2">ContTest<hr>&nbsp;</th><th colspan="2">HomFam<hr>&nbsp;</th></tr><tr><th>Accuracy score&nbsp;</th><th>CPU time (minutes)&nbsp;</th><th><strong>Accuracy score (SP/TC)</strong>&nbsp;</th><th>CPU time (minutes)&nbsp;</th></tr></thead><tbody><tr><td rowspan="4"><strong>A</strong>&nbsp;</td><td>PartTree (partsize = 50)&nbsp;</td><td>0.4103&nbsp;</td><td>61&nbsp;</td><td>0.7862/0.5658&nbsp;</td><td>47&nbsp;</td></tr><tr><td>PartTree (partsize = 1000)&nbsp;</td><td>0.4364&nbsp;</td><td>140&nbsp;</td><td>0.8258/0.6377&nbsp;</td><td>94&nbsp;</td></tr><tr><td>DPPartTree (partsize = 50)&nbsp;</td><td>0.4424&nbsp;</td><td>210&nbsp;</td><td>0.8413/0.6597&nbsp;</td><td>160&nbsp;</td></tr><tr><td>DPPartTree (partsize = 1000)&nbsp;</td><td>0.4632&nbsp;</td><td>1000&nbsp;</td><td>0.8541/0.6934&nbsp;</td><td>820&nbsp;</td></tr><tr><td><strong>B</strong>&nbsp;</td><td>FFT-NS-1&nbsp;</td><td>0.4856&nbsp;</td><td>170&nbsp;</td><td>0.8491/0.6669&nbsp;</td><td>160&nbsp;</td></tr><tr><td><strong>B</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-1 (memsavetree)&nbsp;</td><td>0.4835&nbsp;</td><td>280&nbsp;</td><td>0.8416/0.6667&nbsp;</td><td>260&nbsp;</td></tr><tr><td><strong>D</strong>&nbsp;</td><td>FFT-NS-2&nbsp;</td><td>0.4998&nbsp;</td><td>500&nbsp;</td><td>0.8759/0.7162&nbsp;</td><td>460&nbsp;</td></tr><tr><td><strong>D</strong>+<strong>C</strong>&nbsp;</td><td>FFT-NS-2 (memsavetree)&nbsp;</td><td>0.5099&nbsp;</td><td>1100&nbsp;</td><td>0.8611/0.7023&nbsp;</td><td>990&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100)&nbsp;</td><td>0.5153&nbsp;</td><td>730&nbsp;</td><td>0.8821/0.7274&nbsp;</td><td>650&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500)&nbsp;</td><td>0.5361&nbsp;</td><td>1200&nbsp;</td><td>0.8970/0.7586&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000)&nbsp;</td><td>0.5440&nbsp;</td><td>3400&nbsp;</td><td>0.9075/0.7810&nbsp;</td><td>4400&nbsp;</td></tr><tr><td rowspan="3"><strong>E</strong>+<strong>C</strong>&nbsp;</td><td>mafft-sparsecore (<em>p</em> = 100, memsavetree)&nbsp;</td><td>0.5298&nbsp;</td><td>1500&nbsp;</td><td>0.8845/0.7416&nbsp;</td><td>1300&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 500, memsavetree)&nbsp;</td><td>0.5438&nbsp;</td><td>2000&nbsp;</td><td>0.8995/0.7638&nbsp;</td><td>2000&nbsp;</td></tr><tr><td>mafft-sparsecore (<em>p</em> = 1000, memsavetree)&nbsp;</td><td>0.5428&nbsp;</td><td>4200&nbsp;</td><td>0.9052/0.7826&nbsp;</td><td>5000&nbsp;</td></tr><tr><td><strong>F</strong>&nbsp;</td><td>G-INS-1&nbsp;</td><td>0.5696&nbsp;</td><td>55 000&nbsp;</td><td>0.9306/0.8288&nbsp;</td><td>49000&nbsp;</td></tr><tr><td><strong>G</strong>&nbsp;</td><td>Randomchain&nbsp;</td><td>0.5425&nbsp;</td><td>100&nbsp;</td><td>0.8349/0.6681&nbsp;</td><td>88&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><em>Note</em>: The sum-of-pairs (SP) and total-column (TC) scores for HomFam were calculated by the FastSP program [<a reveal-id="bbx108-B5" data-open="bbx108-B5" class="link link-ref link-reveal xref-bibr">5</a>]. (<strong>A</strong>–<strong>G</strong>)
 correspond to the techniques explained in the main text. Command-line 
arguments are displayed after performing the calculation on the online 
service and also listed in the main text. Random numbers are used in (<strong>A</strong>), (<strong>E</strong>) and (<strong>G</strong>). In this test, only one set of random numbers was used for each method. For (<strong>E</strong>) and (<strong>G</strong>),
 seed of random numbers can be specified in the download version (see 
the last section in the main text) but cannot be specified in the online
 version. See <a class="link link-uri" href="https://mafft.sb.ecei.tohoku.ac.jp/" target="">https://mafft.sb.ecei.tohoku.ac.jp/</a> for detailed results.</p></div></div></div></div><p></p>                        <a id="96239294" scrollto-destination="96239294"></a>
<div id="bbx108-F1" data-id="bbx108-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="m_bbx108f1.png" alt="Screenshot of input page for large MSAs in MAFFT online service. (A–G) are explained in the main text. " path-from-xml="bbx108f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/96239294/bbx108f1.tif" path-from-xml="bbx108f1.tif" target="_blank">View large</a><a section="96239294" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/PAP/10.1093_bib_bbx108/1/bbx108f1.png?Expires=1555688528&amp;Signature=KcuMTDcs8v6R7qOWQI1e4qG9dsyXyPANANjAxgtwrIzcuPMyhtw8IHecAF1SNV8lo%7ES0Z1PMkhgfQ%7EYvme1wm0%7ENfoLv-RDZ-HzS1cofZfnHd2DbbGIfmjwepQCSMOnBR0mDP9RnXJCIdQg1GtIBezf9eZhCvuR3Jm%7E2t9l4iiVUUx%7E%7EUW3UcPeTCdBaeoOu9RU2G2ecy7q5Dz8Tq2BfMWV%7EMhe17HeaZwh0PNm3D58U5Y%7ENlScW5Qs42LHlYglDdHGnP7l8R%7Eo5IggTgsYGikfJaTC%7ExdnOZ%7ECqVdykwaY95IncDkHfkzyy%7ElfhiBJeYURq53sMheVMVHfQJzwhmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=96239294&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5143" class="download-slide" path-from-xml="bbx108f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshot of input page for large MSAs in MAFFT online service. (<strong>A</strong>–<strong>G</strong>) are explained in the main text. </p></div></div><div content-id="bbx108-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="m_bbx108f1.png" alt="Screenshot of input page for large MSAs in MAFFT online service. (A–G) are explained in the main text. " path-from-xml="bbx108f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/96239294/bbx108f1.tif" path-from-xml="bbx108f1.tif" target="_blank">View large</a><a section="96239294" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/PAP/10.1093_bib_bbx108/1/bbx108f1.png?Expires=1555688528&amp;Signature=KcuMTDcs8v6R7qOWQI1e4qG9dsyXyPANANjAxgtwrIzcuPMyhtw8IHecAF1SNV8lo%7ES0Z1PMkhgfQ%7EYvme1wm0%7ENfoLv-RDZ-HzS1cofZfnHd2DbbGIfmjwepQCSMOnBR0mDP9RnXJCIdQg1GtIBezf9eZhCvuR3Jm%7E2t9l4iiVUUx%7E%7EUW3UcPeTCdBaeoOu9RU2G2ecy7q5Dz8Tq2BfMWV%7EMhe17HeaZwh0PNm3D58U5Y%7ENlScW5Qs42LHlYglDdHGnP7l8R%7Eo5IggTgsYGikfJaTC%7ExdnOZ%7ECqVdykwaY95IncDkHfkzyy%7ElfhiBJeYURq53sMheVMVHfQJzwhmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=96239294&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5143" class="download-slide" path-from-xml="bbx108f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Screenshot of input page for large MSAs in MAFFT online service. (<strong>A</strong>–<strong>G</strong>) are explained in the main text. </p></div></div><p>A. PartTree and DPPartTree (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1A</a>) [<a reveal-id="bbx108-B6" data-open="bbx108-B6" class="link link-ref link-reveal xref-bibr">6</a>]
 are highly approximate options. These methods recursively cluster 
sequences and simultaneously compute a distance between the clusters, 
each of which is represented by a single sequence. The order of the 
computational time is <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mi&gt;O&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mtext&gt;log&lt;/mtext&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mo&gt;&amp;#x2061;&lt;/mo&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-1" style="width: 7.091em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.357em; height: 0px; font-size: 132%;"><span style="position: absolute; clip: rect(1.36em, 1005.26em, 2.685em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math; font-style: italic;">O</span><span class="mo" id="MathJax-Span-5" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-6" style="font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mo" id="MathJax-Span-7" style="font-family: MathJax_Main;"> </span><span class="mtext" id="MathJax-Span-8" style="font-family: MathJax_Main; padding-left: 0.167em;">log</span><span class="mo" id="MathJax-Span-9" style="font-family: MathJax_Main; padding-left: 0.167em;"> </span><span class="mo" id="MathJax-Span-10"></span><span class="mi" id="MathJax-Span-11" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mo" id="MathJax-Span-12" style="font-family: MathJax_Main;">)</span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.401em; border-left: 0px solid; width: 0px; height: 1.463em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>O</mi><mo stretchy="false">(</mo><mi>N</mi><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mi>N</mi><mo stretchy="false">)</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-1"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>O</mi><mo stretchy="false">(</mo><mi>N</mi><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mi>N</mi><mo stretchy="false">)</mo></mrow></math></script>⁠</span>, where <em>N</em>
 is the number of sequences. They are fast and applicable to large MSAs,
 but accuracy is sacrificed because of the approximation of guide tree 
calculation (<a reveal-id="bbx108-T1" data-open="bbx108-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The PartTree and DPPartTree options share a basic design, but the former uses <em>k</em><em>-</em>mer-based distance to estimate the similarity between sequences [<a reveal-id="bbx108-B7" data-open="bbx108-B7" class="link link-ref link-reveal xref-bibr">7</a>], while the latter uses dynamic programming (DP) [<a reveal-id="bbx108-B8" data-open="bbx108-B8" class="link link-ref link-reveal xref-bibr">8</a>]
 to estimate the similarity. Accordingly, the latter is slower but more 
accurate. In the command-line version, the balance between accuracy and 
speed can also be adjusted by a parameter, partsize, but this parameter 
is fixed to 1000 in the online service.</p><div class="block-child-p">B. FFT-NS-1 (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1B</a>): This is another approximate method. Its accuracy is higher than PartTree and DPPartTree in benchmark tests (<a reveal-id="bbx108-T1" data-open="bbx108-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The input sequences are progressively aligned using a guide tree [<a reveal-id="bbx108-B6" data-open="bbx108-B6" class="link link-ref link-reveal xref-bibr">6</a>, <a reveal-id="bbx108-B9" data-open="bbx108-B9" class="link link-ref link-reveal xref-bibr">9</a>, <a reveal-id="bbx108-B10" data-open="bbx108-B10" class="link link-ref link-reveal xref-bibr">10</a>]. For constructing the guide tree, pairwise distances are computed based on the number of shared <em>k</em><em>-</em>mers. The length of <em>k</em><em>-</em>mer is 6 for both protein and nucleotide data, but 20 amino acids are grouped into six physicochemical groups [<a reveal-id="bbx108-B11" data-open="bbx108-B11" class="link link-ref link-reveal xref-bibr">11</a>],
 and an amino acid sequence is converted to a sequence composed of six 
letters. The current version of MAFFT uses the following formula to 
compute distance <em>D<sub>ij</sub></em> between sequences <em>i</em> and <em>j</em>: <a id="E1" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;msub&gt;&lt;mi&gt;D&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mo&gt;{&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;S&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mtext&gt;min&lt;/mtext&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;S&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;mi&gt;i&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;S&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;mi&gt;j&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;}&lt;/mo&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mi&gt;f&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mi&gt;y&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-13" style="width: 23.216em; display: inline-block;"><span style="display: inline-block; position: relative; width: 16.691em; height: 0px; font-size: 139%;"><span style="position: absolute; clip: rect(1.408em, 1016.62em, 2.74em, -1000em); top: -2.302em; left: 0em;"><span class="mrow" id="MathJax-Span-14"><span class="mrow" id="MathJax-Span-15"><span class="msub" id="MathJax-Span-16"><span style="display: inline-block; position: relative; width: 1.438em; height: 0px;"><span style="position: absolute; clip: rect(3.154em, 1000.8em, 4.125em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-17" style="font-family: MathJax_Math; font-style: italic;">D</span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.831em; left: 0.828em;"><span class="mrow" id="MathJax-Span-18"><span class="mi" id="MathJax-Span-19" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-20" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-21" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-22" style="padding-left: 0.278em;"><span style="font-family: MathJax_Main;">{</span></span><span class="mn" id="MathJax-Span-23" style="font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-24" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="msub" id="MathJax-Span-25" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.223em; height: 0px;"><span style="position: absolute; clip: rect(3.132em, 1000.65em, 4.147em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-26" style="font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.831em; left: 0.613em;"><span class="mrow" id="MathJax-Span-27"><span class="mi" id="MathJax-Span-28" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-29" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-30" style="font-family: MathJax_Main;">/</span><span class="mtext" id="MathJax-Span-31" style="font-family: MathJax_Main;">min</span><span class="mo" id="MathJax-Span-32" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-33"><span style="display: inline-block; position: relative; width: 1.176em; height: 0px;"><span style="position: absolute; clip: rect(3.132em, 1000.65em, 4.147em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-34" style="font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.831em; left: 0.613em;"><span class="mrow" id="MathJax-Span-35"><span class="mi" id="MathJax-Span-36" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mi" id="MathJax-Span-37" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-38" style="font-family: MathJax_Main;">,</span><span class="msub" id="MathJax-Span-39" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.271em; height: 0px;"><span style="position: absolute; clip: rect(3.132em, 1000.65em, 4.147em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-40" style="font-family: MathJax_Math; font-style: italic;">S<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.831em; left: 0.613em;"><span class="mrow" id="MathJax-Span-41"><span class="mi" id="MathJax-Span-42" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span><span class="mi" id="MathJax-Span-43" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">j</span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-44" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-45" style=""><span style="font-family: MathJax_Main;">}</span></span><span class="mo" id="MathJax-Span-46" style="font-family: MathJax_Main;"> </span><span class="mo" id="MathJax-Span-47" style="font-family: MathJax_Main; padding-left: 0.167em;">/</span><span class="mo" id="MathJax-Span-48" style="font-family: MathJax_Main;"> </span><span class="mi" id="MathJax-Span-49" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.06em;"></span></span><span class="mo" id="MathJax-Span-50" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-51" style="font-family: MathJax_Math; font-style: italic;">x</span><span class="mo" id="MathJax-Span-52" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-53" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.006em;"></span></span><span class="mo" id="MathJax-Span-54" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-55" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.302em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.476em; border-left: 0px solid; width: 0px; height: 1.585em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>D</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mo>{</mo><mn>1</mn><mo>−</mo><msub><mi>S</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>/</mo><mtext>min</mtext><mo stretchy="false">(</mo><msub><mi>S</mi><mrow><mi>i</mi><mi>i</mi></mrow></msub><mo>,</mo><msub><mi>S</mi><mrow><mi>j</mi><mi>j</mi></mrow></msub><mo stretchy="false">)</mo><mo>}</mo><mo> </mo><mo>/</mo><mo> </mo><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo stretchy="false">)</mo><mo>,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-2"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>D</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mo>{</mo><mn>1</mn><mo>−</mo><msub><mi>S</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>/</mo><mtext>min</mtext><mo stretchy="false">(</mo><msub><mi>S</mi><mrow><mi>i</mi><mi>i</mi></mrow></msub><mo>,</mo><msub><mi>S</mi><mrow><mi>j</mi><mi>j</mi></mrow></msub><mo stretchy="false">)</mo><mo>}</mo><mo> </mo><mo>/</mo><mo> </mo><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo stretchy="false">)</mo><mo>,</mo></mrow></math></script></div></div></div> where <em>S<sub>ij</sub></em> is alignment score between sequences <em>i</em> and <em>j</em>. <em>f</em>(<em>x</em>, <em>y</em>)
 adjusts the distance to avoid a case where the distance between 
unrelated sequences happens to become zero when a long sequence and a 
short sequence are compared. <a id="E2" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mi&gt;f&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mi&gt;y&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mi&gt;a&lt;/mi&gt;&lt;mi&gt;y&lt;/mi&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mi&gt;b&lt;/mi&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;x&lt;/mi&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mi&gt;b&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mi&gt;c&lt;/mi&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-56" style="width: 18.42em; display: inline-block;"><span style="display: inline-block; position: relative; width: 13.237em; height: 0px; font-size: 139%;"><span style="position: absolute; clip: rect(1.408em, 1013.17em, 2.696em, -1000em); top: -2.302em; left: 0em;"><span class="mrow" id="MathJax-Span-57"><span class="mrow" id="MathJax-Span-58"><span class="mi" id="MathJax-Span-59" style="font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.06em;"></span></span><span class="mo" id="MathJax-Span-60" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-61" style="font-family: MathJax_Math; font-style: italic;">x</span><span class="mo" id="MathJax-Span-62" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-63" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.006em;"></span></span><span class="mo" id="MathJax-Span-64" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-65" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-66" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">a</span><span class="mi" id="MathJax-Span-67" style="font-family: MathJax_Math; font-style: italic;">y<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.006em;"></span></span><span class="mo" id="MathJax-Span-68" style="font-family: MathJax_Main;">/</span><span class="mi" id="MathJax-Span-69" style="font-family: MathJax_Math; font-style: italic;">x</span><span class="mo" id="MathJax-Span-70" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-71" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">b</span><span class="mo" id="MathJax-Span-72" style="font-family: MathJax_Main;">/</span><span class="mo" id="MathJax-Span-73" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-74" style="font-family: MathJax_Math; font-style: italic;">x</span><span class="mo" id="MathJax-Span-75" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-76" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">b</span><span class="mo" id="MathJax-Span-77" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-78" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-79" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">c</span><span class="mo" id="MathJax-Span-80" style="font-family: MathJax_Main;">,</span></span></span><span style="display: inline-block; width: 0px; height: 2.302em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.414em; border-left: 0px solid; width: 0px; height: 1.523em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo stretchy="false">)</mo><mo>=</mo><mi>a</mi><mi>y</mi><mo>/</mo><mi>x</mi><mo>+</mo><mi>b</mi><mo>/</mo><mo stretchy="false">(</mo><mi>x</mi><mo>+</mo><mi>b</mi><mo stretchy="false">)</mo><mo>+</mo><mi>c</mi><mo>,</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-3"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><mi>y</mi><mo stretchy="false">)</mo><mo>=</mo><mi>a</mi><mi>y</mi><mo>/</mo><mi>x</mi><mo>+</mo><mi>b</mi><mo>/</mo><mo stretchy="false">(</mo><mi>x</mi><mo>+</mo><mi>b</mi><mo stretchy="false">)</mo><mo>+</mo><mi>c</mi><mo>,</mo></mrow></math></script></div></div></div> where <em>x</em> and <em>y</em> are the lengths of the longer and the shorter sequence <em>i</em> or <em>j</em>, respectively. <em>a</em>, <em>b</em> and <em>c</em> are empirically determined parameters; <em>a </em>=<em> </em>0.1, <em>b </em>= 10 000 (nucleotide), 2500 (amino acid) and <em>c</em> = 0.01. As <em>D<sub>ij</sub></em> is computed for all sequence pairs, the computational time is proportional to <em>N</em><sup>2</sup>, where <em>N</em> is the number of sequences. The space complexity is also <em>O</em>(<em>N</em><sup>2</sup>) by default.</div><div class="block-child-p">To build a guide from distances, MAFFT uses a UPGMA-like method with a small modification [<a reveal-id="bbx108-B12" data-open="bbx108-B12" class="link link-ref link-reveal xref-bibr">12</a>]. When merging clusters <em>L</em> and <em>R</em> into a new cluster <em>P</em>, distance <em>D<sub>PC</sub></em> from <em>P</em> to a third cluster <em>C</em> is calculated with: <a id="E3" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-4-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;msub&gt;&lt;mi&gt;D&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;P&lt;/mi&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;D&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;L&lt;/mi&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;D&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;R&lt;/mi&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;/&lt;/mo&gt;&lt;mn&gt;2&lt;/mn&gt;&lt;mo&gt;+&lt;/mo&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mo&gt;&amp;#x2212;&lt;/mo&gt;&lt;mi&gt;s&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mtext&gt;min&lt;/mtext&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;D&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;L&lt;/mi&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;msub&gt;&lt;mi&gt;D&lt;/mi&gt;&lt;mrow&gt;&lt;mi&gt;R&lt;/mi&gt;&lt;mi&gt;C&lt;/mi&gt;&lt;/mrow&gt;&lt;/msub&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;mo&gt;.&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-81" style="width: 31.273em; display: inline-block;"><span style="display: inline-block; position: relative; width: 22.494em; height: 0px; font-size: 139%;"><span style="position: absolute; clip: rect(1.408em, 1022.41em, 2.696em, -1000em); top: -2.302em; left: 0em;"><span class="mrow" id="MathJax-Span-82"><span class="mrow" id="MathJax-Span-83"><span class="msub" id="MathJax-Span-84"><span style="display: inline-block; position: relative; width: 1.971em; height: 0px;"><span style="position: absolute; clip: rect(3.154em, 1000.8em, 4.125em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-85" style="font-family: MathJax_Math; font-style: italic;">D</span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.828em;"><span class="mrow" id="MathJax-Span-86"><span class="mi" id="MathJax-Span-87" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">P<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.077em;"></span></span><span class="mi" id="MathJax-Span-88" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-89" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-90" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">s</span><span class="mo" id="MathJax-Span-91" style="font-family: MathJax_Main;"> </span><span class="mo" id="MathJax-Span-92" style="font-family: MathJax_Main; padding-left: 0.167em;">(</span><span class="msub" id="MathJax-Span-93"><span style="display: inline-block; position: relative; width: 1.922em; height: 0px;"><span style="position: absolute; clip: rect(3.154em, 1000.8em, 4.125em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-94" style="font-family: MathJax_Math; font-style: italic;">D</span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.828em;"><span class="mrow" id="MathJax-Span-95"><span class="mi" id="MathJax-Span-96" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">L</span><span class="mi" id="MathJax-Span-97" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-98" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="msub" id="MathJax-Span-99" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 1.977em; height: 0px;"><span style="position: absolute; clip: rect(3.154em, 1000.8em, 4.125em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-100" style="font-family: MathJax_Math; font-style: italic;">D</span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.828em;"><span class="mrow" id="MathJax-Span-101"><span class="mi" id="MathJax-Span-102" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">R</span><span class="mi" id="MathJax-Span-103" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-104" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-105" style="font-family: MathJax_Main;">/</span><span class="mn" id="MathJax-Span-106" style="font-family: MathJax_Main;">2</span><span class="mo" id="MathJax-Span-107" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mo" id="MathJax-Span-108" style="font-family: MathJax_Main; padding-left: 0.222em;">(</span><span class="mn" id="MathJax-Span-109" style="font-family: MathJax_Main;">1</span><span class="mo" id="MathJax-Span-110" style="font-family: MathJax_Main; padding-left: 0.222em;">−</span><span class="mi" id="MathJax-Span-111" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">s</span><span class="mo" id="MathJax-Span-112" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-113" style="font-family: MathJax_Main;"> </span><span class="mtext" id="MathJax-Span-114" style="font-family: MathJax_Main; padding-left: 0.167em;">min</span><span class="mo" id="MathJax-Span-115" style="font-family: MathJax_Main;">(</span><span class="msub" id="MathJax-Span-116"><span style="display: inline-block; position: relative; width: 1.922em; height: 0px;"><span style="position: absolute; clip: rect(3.154em, 1000.8em, 4.125em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-117" style="font-family: MathJax_Math; font-style: italic;">D</span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.828em;"><span class="mrow" id="MathJax-Span-118"><span class="mi" id="MathJax-Span-119" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">L</span><span class="mi" id="MathJax-Span-120" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-121" style="font-family: MathJax_Main;">,</span><span class="msub" id="MathJax-Span-122" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.977em; height: 0px;"><span style="position: absolute; clip: rect(3.154em, 1000.8em, 4.125em, -1000em); top: -3.981em; left: 0em;"><span class="mi" id="MathJax-Span-123" style="font-family: MathJax_Math; font-style: italic;">D</span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span><span style="position: absolute; top: -3.827em; left: 0.828em;"><span class="mrow" id="MathJax-Span-124"><span class="mi" id="MathJax-Span-125" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">R</span><span class="mi" id="MathJax-Span-126" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">C<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.032em;"></span></span></span><span style="display: inline-block; width: 0px; height: 3.981em;"></span></span></span></span><span class="mo" id="MathJax-Span-127" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-128" style="font-family: MathJax_Main;">.</span></span></span><span style="display: inline-block; width: 0px; height: 2.302em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.414em; border-left: 0px solid; width: 0px; height: 1.523em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>D</mi><mrow><mi>P</mi><mi>C</mi></mrow></msub><mo>=</mo><mi>s</mi><mo> </mo><mo stretchy="false">(</mo><msub><mi>D</mi><mrow><mi>L</mi><mi>C</mi></mrow></msub><mo>+</mo><msub><mi>D</mi><mrow><mi>R</mi><mi>C</mi></mrow></msub><mo stretchy="false">)</mo><mo>/</mo><mn>2</mn><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>s</mi><mo stretchy="false">)</mo><mo> </mo><mtext>min</mtext><mo stretchy="false">(</mo><msub><mi>D</mi><mrow><mi>L</mi><mi>C</mi></mrow></msub><mo>,</mo><msub><mi>D</mi><mrow><mi>R</mi><mi>C</mi></mrow></msub><mo stretchy="false">)</mo><mo>.</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-4"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>D</mi><mrow><mi>P</mi><mi>C</mi></mrow></msub><mo>=</mo><mi>s</mi><mo> </mo><mo stretchy="false">(</mo><msub><mi>D</mi><mrow><mi>L</mi><mi>C</mi></mrow></msub><mo>+</mo><msub><mi>D</mi><mrow><mi>R</mi><mi>C</mi></mrow></msub><mo stretchy="false">)</mo><mo>/</mo><mn>2</mn><mo>+</mo><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>s</mi><mo stretchy="false">)</mo><mo> </mo><mtext>min</mtext><mo stretchy="false">(</mo><msub><mi>D</mi><mrow><mi>L</mi><mi>C</mi></mrow></msub><mo>,</mo><msub><mi>D</mi><mrow><mi>R</mi><mi>C</mi></mrow></msub><mo stretchy="false">)</mo><mo>.</mo></mrow></math></script></div></div></div></div><p>The resulting tree becomes more imbalanced [<a reveal-id="bbx108-B13" data-open="bbx108-B13" class="link link-ref link-reveal xref-bibr">13</a>] with smaller values of parameter <em>s</em> (0 ≤ <em>s</em> ≤1). The default <em>s</em>
 value has been unchanged from 0.1 since the initial release in 2002, 
but can be specified with the --mixedlinkage flag in the download 
version.</p><p>C. To compute a guide tree with less RAM, a low-memory mode is available but not enabled by default (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1C</a>).
 If a calculation in the online service requires more RAM than a 
threshold, then the calculation is terminated and an error message is 
returned instructing the user to select the low-memory mode. In this 
mode, instead of storing a full distance matrix in RAM, distances are 
calculated two times during the tree building step. Accordingly, the 
calculation time is longer than the normal mode.</p><p>D. FFT-NS-2 (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1D</a>):
 This is the default option of MAFFT. In this method, after performing 
FFT-NS-1, a new distance matrix and guide tree are recalculated based on
 the MSA, and then the final MSA is built using the new guide tree. In 
benchmark tests, the accuracy is generally improved by the recalculation
 of the guide tree as shown in <a reveal-id="bbx108-T1" data-open="bbx108-T1" class="link link-reveal link-table xref-fig">Table 1</a>. This method is at least two times slower than FFT-NS-1. The low-memory mode (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1C</a>) is also available for FFT-NS-2.</p><p>E. mafft-sparsecore (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1E</a>) [<a reveal-id="bbx108-B12" data-open="bbx108-B12" class="link link-ref link-reveal xref-bibr">12</a>] is a combination of the iterative refinement method [<a reveal-id="bbx108-B14" data-open="bbx108-B14" class="link link-ref link-reveal xref-bibr">14–16</a>]
 and the progressive method. It aims to improve the alignment accuracy 
by partly applying the iterative refinement method, which is known to be
 more accurate than the progressive method. The procedure was described 
in Yamada <em>et al</em>. (2016) [<a reveal-id="bbx108-B12" data-open="bbx108-B12" class="link link-ref link-reveal xref-bibr">12</a>]: (i) the input sequences are sorted by length. From the upper <em>n</em>% of the sorted sequences, <em>p</em> sequences are randomly selected as ‘core’ sequences. The default values of <em>n</em> and <em>p</em> are 50 and 500, respectively. (ii) An MSA of the <em>p</em>
 core sequences is constructed by an iterative refinement option, 
G-INS-i. (iii) The remaining sequences are added to the core MSA using 
the –add option [<a reveal-id="bbx108-B17" data-open="bbx108-B17" class="link link-ref link-reveal xref-bibr">17</a>], which uses the progressive alignment method. The accuracy and speed are controlled by the parameter <em>p</em>. With larger <em>p</em>, the accuracy is improved, but computational cost becomes higher (<a reveal-id="bbx108-T1" data-open="bbx108-T1" class="link link-reveal link-table xref-fig">Table 1</a>),
 as more sequences are subjected to the iterative refinement 
calculation. The memory usage is mainly determined by the progressive 
alignment stage (iii). The low-memory mode (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1C</a>) is also available for mafft-sparsecore.</p><p>F. G-INS-1 (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1F</a>): This gives more accurate MSAs [<a reveal-id="bbx108-B12" data-open="bbx108-B12" class="link link-ref link-reveal xref-bibr">12</a>, <a reveal-id="bbx108-B18" data-open="bbx108-B18" class="link link-ref link-reveal xref-bibr">18</a>]
 but takes a longer computational time and requires more RAM than other 
methods. This method uses an accurate guide tree based on all-to-all DP 
calculation and a scoring function similar to COFFEE [<a reveal-id="bbx108-B19" data-open="bbx108-B19" class="link link-ref link-reveal xref-bibr">19</a>]
 in progressive alignment. We are developing a memory-efficient version 
of G-INS-1, which runs in parallel on distributed memory systems or 
shared memory systems (manuscript in preparation). This option is 
experimentally supported at <a class="link link-uri" href="http://mafft.cbrc.jp/alignment/server/large-lsf.html" target="">http://mafft.cbrc.jp/alignment/server/large-lsf.html</a>.</p><p>G. Pileup (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1G</a>):
 This is the simplest strategy. The first and the second sequences are 
first aligned. Then, the other sequences are added to the alignment in 
the order in the input file. Random chain: This is similar to Pileup, 
but the order of sequences is randomized. The usefulness of this 
strategy is controversial [<a reveal-id="bbx108-B3" data-open="bbx108-B3" class="link link-ref link-reveal xref-bibr">3</a>, <a reveal-id="bbx108-B12" data-open="bbx108-B12" class="link link-ref link-reveal xref-bibr">12</a>, <a reveal-id="bbx108-B13" data-open="bbx108-B13" class="link link-ref link-reveal xref-bibr">13</a>, <a reveal-id="bbx108-B20" data-open="bbx108-B20" class="link link-ref link-reveal xref-bibr">20</a>, <a reveal-id="bbx108-B21" data-open="bbx108-B21" class="link link-ref link-reveal xref-bibr">21</a>]. However, because these methods have an advantage in computational simplicity, we have made them available in our service.</p>                            <h2 scrollto-destination="96239304" id="96239304" class="section-title">Selecting suitable strategies</h2>
<p>To select suitable MAFFT options for specific problems, consider the following factors.</p><p>For
 aligning a small number of sequences, the iterative refinement method 
is known to effectively improve the accuracy, as noted above. However, 
for large-scale MSAs (the subject of this article), the effect of 
iterative refinement was recently assessed to be small. More 
specifically, in <a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1</a> in Le <em>et al</em>. [<a reveal-id="bbx108-B18" data-open="bbx108-B18" class="link link-ref link-reveal xref-bibr">18</a>],
 the advantage of MAFFT-L-INS-i (an iterative refinement method) over 
MAFFT-L-INS-1 (a progressive method) was clearly observed for a small 
number of sequences but not for thousands of sequences. Moreover, a 
direct application of the iterative refinement method to large sequence 
data sets is difficult in terms of computational resources.</p><p>In benchmarks with ∼1000–100 000 sequences, G-INS-1 outperforms other methods in accuracy as shown in <a reveal-id="bbx108-T1" data-open="bbx108-T1" class="link link-reveal link-table xref-fig">Table 1</a>.
 The difference is statistically significant in several cases. Thus, 
this method is first recommended if computational resources allow. We 
are making an effort to decrease the computational resources required by
 this method. If it is difficult to apply G-INS-1, then the next 
candidate would be mafft-sparsecore, which uses the advantage of 
iterative refinement for small MSAs.</p><p>These two methods can be 
applied to typical protein sequences with &lt;10 000 sites, but cannot 
be applied to long DNA sequences. In such a case, FFT-NS-2 or FFT-NS-1 
can be useful, as the computational time is proportional to <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-5-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mi&gt;L&lt;/mi&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mtext&gt;log&lt;/mtext&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mo&gt;&amp;#x2061;&lt;/mo&gt;&lt;mi&gt;L&lt;/mi&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-129" style="width: 4.602em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.463em; height: 0px; font-size: 132%;"><span style="position: absolute; clip: rect(1.416em, 1003.43em, 2.641em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-130"><span class="mrow" id="MathJax-Span-131"><span class="mi" id="MathJax-Span-132" style="font-family: MathJax_Math; font-style: italic;">L</span><span class="mo" id="MathJax-Span-133" style="font-family: MathJax_Main;"> </span><span class="mtext" id="MathJax-Span-134" style="font-family: MathJax_Main; padding-left: 0.167em;">log</span><span class="mo" id="MathJax-Span-135" style="font-family: MathJax_Main; padding-left: 0.167em;"> </span><span class="mo" id="MathJax-Span-136"></span><span class="mi" id="MathJax-Span-137" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">L</span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.343em; border-left: 0px solid; width: 0px; height: 1.331em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>L</mi><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mi>L</mi></mrow></math></span></span><script type="math/mml" id="MathJax-Element-5"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>L</mi><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mi>L</mi></mrow></math></script>⁠</span>, where <em>L</em> is sequence length, because of the FFT approximation [<a reveal-id="bbx108-B1" data-open="bbx108-B1" class="link link-ref link-reveal xref-bibr">1</a>].
 However, this is only when the input sequences share global homology 
(from 5′ end to 3′end), and the similarity level is high. MAFFT cannot 
handle data with genomic rearrangements, such as inversions and 
translocations. Also note that an MSA can be built only when the 
sequences are all homologous. It does not make sense to construct an MSA
 of nonhomologous sequences.</p><p>For a data set with much &gt;100 000 
sequences, PartTree and DPPartTree, can be applied if the sequences are 
homologous, as their time complexity is <em>O</em>(<em>N</em>log<em>N</em>), where <em>N</em> is the number of sequences. However, there are also other popular programs, such as Clustal Omega [<a reveal-id="bbx108-B4" data-open="bbx108-B4" class="link link-ref link-reveal xref-bibr">4</a>] and UPP [<a reveal-id="bbx108-B22" data-open="bbx108-B22" class="link link-ref link-reveal xref-bibr">22</a>],
 for this purpose. The PartTree algorithm contributed to these programs 
theoretically and/or practically. Clustal Omega uses the mBed algorithm [<a reveal-id="bbx108-B23" data-open="bbx108-B23" class="link link-ref link-reveal xref-bibr">23</a>] to build a guide tree with a time complexity of <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-6-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mi&gt;O&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;(&lt;/mo&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mtext&gt;log&lt;/mtext&gt;&lt;mo&gt;&amp;#x2009;&lt;/mo&gt;&lt;mo&gt;&amp;#x2061;&lt;/mo&gt;&lt;mi&gt;N&lt;/mi&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-138" style="width: 7.091em; display: inline-block;"><span style="display: inline-block; position: relative; width: 5.357em; height: 0px; font-size: 132%;"><span style="position: absolute; clip: rect(1.36em, 1005.26em, 2.685em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-139"><span class="mrow" id="MathJax-Span-140"><span class="mi" id="MathJax-Span-141" style="font-family: MathJax_Math; font-style: italic;">O</span><span class="mo" id="MathJax-Span-142" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-143" style="font-family: MathJax_Math; font-style: italic;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mo" id="MathJax-Span-144" style="font-family: MathJax_Main;"> </span><span class="mtext" id="MathJax-Span-145" style="font-family: MathJax_Main; padding-left: 0.167em;">log</span><span class="mo" id="MathJax-Span-146" style="font-family: MathJax_Main; padding-left: 0.167em;"> </span><span class="mo" id="MathJax-Span-147"></span><span class="mi" id="MathJax-Span-148" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">N<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.085em;"></span></span><span class="mo" id="MathJax-Span-149" style="font-family: MathJax_Main;">)</span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.401em; border-left: 0px solid; width: 0px; height: 1.463em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>O</mi><mo stretchy="false">(</mo><mi>N</mi><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mi>N</mi><mo stretchy="false">)</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-6"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>O</mi><mo stretchy="false">(</mo><mi>N</mi><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mi>N</mi><mo stretchy="false">)</mo></mrow></math></script>⁠</span>. UPP uses PASTA [<a reveal-id="bbx108-B24" data-open="bbx108-B24" class="link link-ref link-reveal xref-bibr">24</a>] to build a backbone MSA of a small number of sequences and then adds the remaining sequences using hmmalign [<a reveal-id="bbx108-B25" data-open="bbx108-B25" class="link link-ref link-reveal xref-bibr">25</a>] with the time complexity of <em>O</em>(<em>N</em>).
 PASTA uses MAFFT-PartTree to generate the initial MSA and MAFFT-L-INS-i
 (an iterative refinement option for small data) to generate sub-MSAs of
 closely related sequences. Performance comparison including these 
methods can be seen on <a class="link link-uri" href="https://mafft.sb.ecei.tohoku.ac.jp/" target="">https://mafft.sb.ecei.tohoku.ac.jp/</a>, which also includes detailed benchmark results for subsets with different data sizes.</p><p>Similarity
 level and difference in sequence lengths also should be considered. If 
the sequences are highly similar to each other and their lengths are 
also similar, then fast methods, such as FFT-NS-1 or even Pileup should 
result in a useful MSA. If the input data have fragmentary sequences and
 full-length sequences, then a two-step strategy sometimes works well. 
That is (i) align the full-length sequences first and then (ii) add the 
fragmentary sequences to the full-length MSA using the --addfragments 
option (see next section).</p>                            <h2 scrollto-destination="96239311" id="96239311" class="section-title">Use of existing MSA</h2>
<p>Each step of the calculation of mafft-sparsecore (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1E</a>) can be separately or manually performed. If a reliable MSA and a set of unaligned sequences are given to <a class="link link-uri" href="http://mafft.cbrc.jp/alignment/server/add_sequences.html" target="">http://mafft.cbrc.jp/alignment/server/add_sequences.html</a>, then an MSA of all the sequences is returned, in which the existing MSA is preserved as the original one.</p><p>Several
 variants, --add, --addfull, --addfragments and --addlong, are 
available. They can be selected according to the relative length of new 
sequences to the existing MSA as illustrated in <a reveal-id="bbx108-F2" data-open="bbx108-F2" class="link link-reveal link-table xref-fig">Figure 2</a>.
 The four options work similarly to each other. However, sequences added
 with the --add option are subjected to distance calculation with time 
complexity of <em>O</em>(<em>N</em><sup>2</sup>), where <em>N</em> is 
the number of sequences. In the other three options, distances between 
the sequences in the existing alignment are computed with a time 
complexity of <em>O</em>(<em>M</em><sup>2</sup>), where <em>M</em> is the number of sequences in the existing MSA, to build a tree of the <em>M</em> sequences using the UPGMA-like method (see above). For each of (<em>N</em>−<em>M</em>) sequences to be added, distances to the <em>M</em> sequences are computed to locate the position of the sequence in the tree, followed by the building of an alignment of (<em>M</em> + 1) sequences. Then, a full MSA is built from the (<em>N</em>−<em>M</em>)
 MSAs. The latter strategy is useful when the new sequences do not 
overlap with each other (as in the case of fragmentary sequences) and 
when the phylogenetic relationship between new sequences is not 
necessary to consider. There are several other tools, such as hmmalign [<a reveal-id="bbx108-B25" data-open="bbx108-B25" class="link link-ref link-reveal xref-bibr">25</a>], PaPaRa [<a reveal-id="bbx108-B26" data-open="bbx108-B26" class="link link-ref link-reveal xref-bibr">26</a>] and PAGAN [<a reveal-id="bbx108-B27" data-open="bbx108-B27" class="link link-ref link-reveal xref-bibr">27</a>], to add sequences to an existing MSA. </p>                        <a id="96239314" scrollto-destination="96239314"></a>
<div id="bbx108-F2" data-id="bbx108-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="m_bbx108f2.png" alt="Variants of --add option." path-from-xml="bbx108f2.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/96239314/bbx108f2.tif" path-from-xml="bbx108f2.tif" target="_blank">View large</a><a section="96239314" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/PAP/10.1093_bib_bbx108/1/bbx108f2.png?Expires=1555688528&amp;Signature=HHfMX9Nb-5BsGY6DxqitwcebECNqwvdcCH1jtyXMFxpnkGgtSSaSljnf45j99GIYaBonwlpRmkX6%7EJ0glOv%7E2ZZVP5rJyvQZSMEklhNeR97job1qk5vIPTXiBynUiLGAiB2ijzGb7Hlvqdn31Y5dahSpTWuKlw22L94Ffxstufyu7VCQaW2DxUzYICDRa71iF3nhXgkZbYk%7Eyo-kxyF%7Es-w7gDD1D63mbkg5vjg8jgzRKJsHFQaLsd2N7nVcWqVncHynx%7EplipGPBKjAcJa1KKDoG9hBoZrKvum0WJz2qiFpveiXD2NlumAdwcFUKD4Rv-wKhD7Z-D59fynWeVSFhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=96239314&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5143" class="download-slide" path-from-xml="bbx108f2.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Variants of --add option.</p></div></div><div content-id="bbx108-F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="m_bbx108f2.png" alt="Variants of --add option." path-from-xml="bbx108f2.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/96239314/bbx108f2.tif" path-from-xml="bbx108f2.tif" target="_blank">View large</a><a section="96239314" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/PAP/10.1093_bib_bbx108/1/bbx108f2.png?Expires=1555688528&amp;Signature=HHfMX9Nb-5BsGY6DxqitwcebECNqwvdcCH1jtyXMFxpnkGgtSSaSljnf45j99GIYaBonwlpRmkX6%7EJ0glOv%7E2ZZVP5rJyvQZSMEklhNeR97job1qk5vIPTXiBynUiLGAiB2ijzGb7Hlvqdn31Y5dahSpTWuKlw22L94Ffxstufyu7VCQaW2DxUzYICDRa71iF3nhXgkZbYk%7Eyo-kxyF%7Es-w7gDD1D63mbkg5vjg8jgzRKJsHFQaLsd2N7nVcWqVncHynx%7EplipGPBKjAcJa1KKDoG9hBoZrKvum0WJz2qiFpveiXD2NlumAdwcFUKD4Rv-wKhD7Z-D59fynWeVSFhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=96239314&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5143" class="download-slide" path-from-xml="bbx108f2.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Variants of --add option.</p></div></div><p>Note
 that the length of the resulting MSA can differ from that of the 
original MSA. This is because additional gaps are necessary when new 
sequences have insertions. All-gap sites, if any, in the original MSA 
are deleted. As such changes in length are not useful in some cases, we 
have implemented a new option, --keeplength, in which (1) insertions in 
the new sequences are deleted and (2) all-gap sites in the original MSA 
are reinserted as shown at the right end in <a reveal-id="bbx108-F2" data-open="bbx108-F2" class="link link-reveal link-table xref-fig">Figure 2</a>. This option is selectable in the online version and sometimes useful for mapping new sequences to a reference MSA.</p>                            <h2 scrollto-destination="96239316" id="96239316" class="section-title">Interactive sequence choice and visualization</h2>
<p>Recently, we have access to huge amounts of sequence data from widely
 divergent organisms, but the quality of the data is not always high 
because of the limitations of sequencing technologies. In the case of 
amino acid sequence data, the difficulty in eukaryotic gene prediction [<a reveal-id="bbx108-B28" data-open="bbx108-B28" class="link link-ref link-reveal xref-bibr">28–30</a>]
 also results in errors in data. It might be possible to automatically 
exclude such problematic data in certain cases, but sometimes, 
biologically important information is in low-quality sequences, 
especially when interest is in nonmodel organisms.</p><p>For such cases,
 it is necessary to manually choose sequences, but this is becoming 
difficult because of increasing data size. Therefore, an interactive 
tool to help this process is necessary. Our service has some functions 
for this purpose as explained in Kuraku <em>et al</em>. [<a reveal-id="bbx108-B31" data-open="bbx108-B31" class="link link-ref link-reveal xref-bibr">31</a>]. Sequences can be selected/unselected one by one in the sequence selection window (<a reveal-id="bbx108-F3" data-open="bbx108-F3" class="link link-reveal link-table xref-fig">Figure 3B</a>).
 Moreover, a group of sequences in a single phylogenetic cluster can be 
selected or unselected in a tree viewer. If you click on a node in a 
tree (<a reveal-id="bbx108-F3" data-open="bbx108-F3" class="link link-reveal link-table xref-fig">Figure 3A</a>), the descendant sequences under the node are selected or unselected together in the list of sequences (<a reveal-id="bbx108-F3" data-open="bbx108-F3" class="link link-reveal link-table xref-fig">Figure 3B</a>). Automated tools for sequence selection, such as CD-HIT [<a reveal-id="bbx108-B32" data-open="bbx108-B32" class="link link-ref link-reveal xref-bibr">32</a>] and MaxAlign [<a reveal-id="bbx108-B33" data-open="bbx108-B33" class="link link-ref link-reveal xref-bibr">33</a>],
 can also run on our service. The selected sequences are subjected to 
phylogenetic tree inference using the neighbor-joining method [<a reveal-id="bbx108-B34" data-open="bbx108-B34" class="link link-ref link-reveal xref-bibr">34</a>] or UPGMA [<a reveal-id="bbx108-B35" data-open="bbx108-B35" class="link link-ref link-reveal xref-bibr">35</a>] with several options, such as distance measure and the number of bootstrap cycles (<a reveal-id="bbx108-F3" data-open="bbx108-F3" class="link link-reveal link-table xref-fig">Figure 3C</a>).
 Then, the data set can be further refined using the new tree. The 
maximum-likelihood method is not supported because of the high 
computational costs. It must be performed locally or using other online 
services. </p>                        <a id="96239319" scrollto-destination="96239319"></a>
<div id="bbx108-F3" data-id="bbx108-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="m_bbx108f3.png" alt="Interactive sequence selection. A group of sequences in guide tree (A) is selected at a time in sequence selection window (B). Several options for tree estimation can be selected (C). MSA can be visually checked using MSAViewer (D)." path-from-xml="bbx108f3.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/96239319/bbx108f3.tif" path-from-xml="bbx108f3.tif" target="_blank">View large</a><a section="96239319" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/PAP/10.1093_bib_bbx108/1/bbx108f3.png?Expires=1555688528&amp;Signature=VfSIjImqhWoKEV99YK9PBT%7EQdWblq60KaKQz3ogh7hw6%7EwNQEy5R2ejJWYUtrnRZou4pbFA2-CwUy9xqKFVy7qfubwV7I9FpmxfCgt1BDxm0FgTBFIs52tfUDaFMDNaV45H4SFCYayMcI5%7EFeXS7-ovK2ngwAWD5bN9y5T1rBAPvrHqX01sKfzAZUc8WNYToqXSlDFWaDbvMY0u2x5dCjxIu%7E2DYK8K3NwbhvAhCZY8YwWeEPiHQRDlF97v9iRXLDkxqSt3kX2s39zdjZIatcP2cHzSCqDvRAoOnPxAXTQSXXtP-3LTiWCdyWJJRJSc-UO8Qm59VIXOcgwiuOXaSTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=96239319&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5143" class="download-slide" path-from-xml="bbx108f3.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Interactive sequence selection. A group of sequences in guide tree (<strong>A</strong>) is selected at a time in sequence selection window (<strong>B</strong>). Several options for tree estimation can be selected (<strong>C</strong>). MSA can be visually checked using MSAViewer (<strong>D</strong>).</p></div></div><div content-id="bbx108-F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="m_bbx108f3.png" alt="Interactive sequence selection. A group of sequences in guide tree (A) is selected at a time in sequence selection window (B). Several options for tree estimation can be selected (C). MSA can be visually checked using MSAViewer (D)." path-from-xml="bbx108f3.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/96239319/bbx108f3.tif" path-from-xml="bbx108f3.tif" target="_blank">View large</a><a section="96239319" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bib/PAP/10.1093_bib_bbx108/1/bbx108f3.png?Expires=1555688528&amp;Signature=VfSIjImqhWoKEV99YK9PBT%7EQdWblq60KaKQz3ogh7hw6%7EwNQEy5R2ejJWYUtrnRZou4pbFA2-CwUy9xqKFVy7qfubwV7I9FpmxfCgt1BDxm0FgTBFIs52tfUDaFMDNaV45H4SFCYayMcI5%7EFeXS7-ovK2ngwAWD5bN9y5T1rBAPvrHqX01sKfzAZUc8WNYToqXSlDFWaDbvMY0u2x5dCjxIu%7E2DYK8K3NwbhvAhCZY8YwWeEPiHQRDlF97v9iRXLDkxqSt3kX2s39zdjZIatcP2cHzSCqDvRAoOnPxAXTQSXXtP-3LTiWCdyWJJRJSc-UO8Qm59VIXOcgwiuOXaSTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=96239319&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5143" class="download-slide" path-from-xml="bbx108f3.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Interactive sequence selection. A group of sequences in guide tree (<strong>A</strong>) is selected at a time in sequence selection window (<strong>B</strong>). Several options for tree estimation can be selected (<strong>C</strong>). MSA can be visually checked using MSAViewer (<strong>D</strong>).</p></div></div><p>Two tree viewers, Phylo.io [<a reveal-id="bbx108-B36" data-open="bbx108-B36" class="link link-ref link-reveal xref-bibr">36</a>] and Archaeopteryx [<a reveal-id="bbx108-B37" data-open="bbx108-B37" class="link link-ref link-reveal xref-bibr">37</a>],
 are used for sequence selection and visualization of phylogenetic 
trees. Originally, we used Archaeopteryx Java plugin, but modern 
browsers no longer support Java plugin for security reasons. Thus, we 
recently adopted Phylo.io, which is written in JavaScript and works with
 most modern browsers. With the addition of Phylo.io to our service, we 
have added some new features: </p><ul class="bullet"><li><p>Coloring of sequence title corresponding to the databases in aLeaves [<a reveal-id="bbx108-B29" data-open="bbx108-B29" class="link link-ref link-reveal xref-bibr">29</a>].</p></li><li><p>Interactive sequence selection (see above).</p></li><li><p>Automatic
 rooting similar to mid-point rooting. This is just for visualization 
without any biological basis. To estimate the position of root, outgroup
 or other additional information is necessary.</p></li></ul><p></p><p>A 
JavaScript version of Archaeopteryx is being developed (C. Zmasek, 
personal communication), and we are planning to use this viewer, too. To
 visualize MSAs, two tools, Jalview [<a reveal-id="bbx108-B38" data-open="bbx108-B38" class="link link-ref link-reveal xref-bibr">38</a>] (as Java plugin) and MSAViewer [<a reveal-id="bbx108-B39" data-open="bbx108-B39" class="link link-ref link-reveal xref-bibr">39</a>] (written in JavaScript; <a reveal-id="bbx108-F3" data-open="bbx108-F3" class="link link-reveal link-table xref-fig">Figure 3C</a>), are available on our service.</p>                            <h2 scrollto-destination="96239322" id="96239322" class="section-title">Necessity of large MSAs</h2>
<p>The relationship between alignment accuracy and data size is still 
unclear. It is naively expected that the accuracy of an MSA is improved 
with the number of input sequences. However, highly accurate methods 
cannot be applied to large data because of computational costs. Useful 
information related to this issue has recently been reported by Le <em>et al</em>. [<a reveal-id="bbx108-B18" data-open="bbx108-B18" class="link link-ref link-reveal xref-bibr">18</a>].
 In their tests, the accuracy of downstream analysis (protein secondary 
structure prediction in this case) is improved with the increase of 
sequences for medium-scale data (&lt;1000 sequences), but with more 
sequences, the accuracy reaches a sort of plateau. Thus, there may be 
optimal data size. Their test also suggested that the accuracy of MSA 
itself hits a maximum point at a smaller number of sequences (around 
200) and that the accuracy of MSA decreases with an increase in the 
number of sequences. This observation is consistent with Sievers <em>et al</em>. [<a reveal-id="bbx108-B40" data-open="bbx108-B40" class="link link-ref link-reveal xref-bibr">40</a>].
 Such optimal data sizes can differ for different problems. For example,
 in the case of prediction of contact residues based on co-evolution, 
larger MSAs are generally thought to be necessary [<a reveal-id="bbx108-B41" data-open="bbx108-B41" class="link link-ref link-reveal xref-bibr">41</a>, <a reveal-id="bbx108-B42" data-open="bbx108-B42" class="link link-ref link-reveal xref-bibr">42</a>].</p>                            <h2 scrollto-destination="96239324" id="96239324" class="section-title">Command-line options</h2>
<p>Each method also runs locally. In the current version (7.310; August 2017), the corresponding commands are as follows:</p><p>PartTree (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1A</a>)</p><p><span class="monospace">mafft --parttree --partsize</span><span class="monospace">1000</span><span class="monospace"><em>input</em></span> <span class="monospace">&gt;</span> <span class="monospace"><em>output</em></span></p><p>DPPartTree (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1A</a>)</p><p><span class="monospace">mafft --dpparttree --partsize</span><span class="monospace">1000</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p>FFT-NS-1 (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1B</a>)</p><p><span class="monospace">mafft --retree 1</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft --retree 1 --memsavetree</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (low-memory mode)</p><p><span class="monospace">mafft --retree 1 --thread -1</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (multithread mode)</p><p>With thread -1, the number of physical cores is automatically counted and all cores are used. See <a class="link link-uri" href="http://mafft.cbrc.jp/alignment/software/multithreading.html" target="">http://mafft.cbrc.jp/alignment/software/multithreading.html</a> for detailed information on multithreading.</p><p>FFT-NS-2 (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1D</a>)</p><p><span class="monospace">mafft</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft --memsavetree</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (low-memory mode)</p><p><span class="monospace">mafft --thread -1</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (multithread mode)</p><p>mafft-sparsecore (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1E</a>)</p><p><span class="monospace">mafft-sparsecore.rb -</span><span class="monospace">p</span><span class="monospace"><em>p</em></span><span class="monospace">-n</span><span class="monospace"><em>n</em></span><span class="monospace">-s</span><span class="monospace"><em>s</em></span><span class="monospace">-i</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft-sparsecore.rb -p</span><span class="monospace"><em>p</em></span><span class="monospace">-n</span><span class="monospace"><em>n</em></span><span class="monospace">-s</span><span class="monospace"><em>s</em></span><span class="monospace">-A ”--memsavetree” -i</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (low-memory mode)</p><p><span class="monospace">mafft-sparsecore.rb -p</span><span class="monospace"><em>p</em></span><span class="monospace">-n</span><span class="monospace"><em>n</em></span><span class="monospace">-s</span><span class="monospace"><em>s</em></span><span class="monospace">-A ”--thread -1” -C ”--thread -1” -i</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (multithread mode)</p><p><em>p</em> and <em>n</em> are as explained above, and <em>s</em>
 is seed for random numbers. Flags for the iterative refinement stage 
and those for the progressive stage can be specified after -C and -A, 
respectively. See <a class="link link-uri" href="http://mafft.cbrc.jp/alignment/software/sparsecore.html" target="">http://mafft.cbrc.jp/alignment/software/sparsecore.html</a> for detailed information.</p><p>G-INS-1 (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1F</a>)</p><p><span class="monospace">mafft --globalpair</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft --globalpair --thread -1</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span> (multithread mode)</p><p>Pileup (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1G</a>)</p><p><span class="monospace">mafft --pileup</span><span class="monospace"><em>input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p>Random chain (<a reveal-id="bbx108-F1" data-open="bbx108-F1" class="link link-reveal link-table xref-fig">Figure 1G</a>)</p><p><span class="monospace">mafft --randomchain --randomseed</span><span class="monospace"><em>s input </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><em>s</em> is seed for random numbers.</p><p>Adding new sequences to an MSA</p><p><span class="monospace">mafft --add</span><span class="monospace"><em>newSequences</em></span><span class="monospace"><em>existingMSA </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft --addfull</span><span class="monospace"><em>newSequences</em></span><span class="monospace"><em>existingMSA </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft --addlong</span><span class="monospace"><em>newSequences</em></span><span class="monospace"><em>existingMSA </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p><span class="monospace">mafft --addfragments</span><span class="monospace">newSequences</span><span class="monospace"><em>existingMSA </em></span><span class="monospace">&gt;</span><span class="monospace"><em> output</em></span></p><p>The --keeplength flag can be added to each command (see above). Add <span class="monospace">--thread −1</span> to enable multithreading.</p><p></p><div class="boxed-text"><div class="caption"><p>Key Points</p></div><ul class="bullet"><li><p>MSA is an important step in phylogeny inference, functional prediction and many other analyses.</p></li><li><p>The demand for MSAs with a large number of sequences is increasing.</p></li><li><p>MAFFT
 has different options for computing large MSAs in both the local and 
online versions. The online version has additional features for 
preprocessing and postprocessing MSAs.</p></li></ul></div><p></p><p><strong>Kazutaka Katoh</strong>
 is an associate professor in the Department of Genome Informatics, 
Research Institute for Microbial Diseases, Osaka University. His 
research interests are in bioinformatics and molecular evolution.</p><p><strong>John Rozewicki</strong>
 is a researcher in the Department of Genome Informatics, Research 
Institute for Microbial Diseases, Osaka University. His research focus 
is designing systems to conduct bioinformatics research using 
high-performance computing and networked server infrastructure.</p><p><strong>Kazunori D. Yamada</strong>
 is an assistant professor in graduate school of information sciences, 
Tohoku University. His research interests include development of amino 
acid sequence alignment methods and machine learning.</p><p>Research 
Institute for Microbial Diseases, Osaka University, conducts basic 
research in the areas of infectious disease, immunology and oncology.</p>                            <h2 scrollto-destination="96239367" id="96239367" class="backacknowledgements-title">Acknowledgement</h2>
<p>The authors thank Daron M. Standley, RIMD, Osaka University, for inspiring discussion.</p>                            <h3 scrollto-destination="96239369" id="96239369" class="section-title">Funding</h3>
<p>Japan Society for the Promotion of Science (JSPS) KAKENHI (grant 
number JP16K07464) and the Platform Project for Supporting Drug 
Discovery and Life Science Research from Japan Agency for Medical 
Research and Development (AMED).</p>                            <h2 scrollto-destination="96239371" id="96239371" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="bbx108-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Misawa</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K</div></div></div>, et al.&nbsp; <div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</div>. <div class="source">Nucleic Acids Res</div>&nbsp; <div class="year">2002</div>;<div class="volume">30</div>:<div class="fpage">3059</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K%20Katoh&amp;author=K%20Misawa&amp;author=K%20Kuma&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Standley</div> <div class="given-names">DM.</div></div></div> <div class="article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</div>. <div class="source">Mol Biol Evol</div>&nbsp; <div class="year">2013</div>;<div class="volume">30</div>(<div class="issue">4</div>):<div class="fpage">772</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;author=K%20Katoh&amp;author=DM.%20Standley&amp;publication_year=2013&amp;journal=Mol%20Biol%20Evol&amp;volume=30&amp;pages=772-80" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst010" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst010" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmst010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fox</div> <div class="given-names">G</div></div>, <div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">DG.</div></div></div> <div class="article-title">Using de novo protein structure predictions to measure the quality of very large multiple sequence alignments</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2016</div>;<div class="volume">32</div>(<div class="issue">6</div>):<div class="fpage">814</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20de%20novo%20protein%20structure%20predictions%20to%20measure%20the%20quality%20of%20very%20large%20multiple%20sequence%20alignments&amp;author=G%20Fox&amp;author=F%20Sievers&amp;author=DG.%20Higgins&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=814-20" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv592" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv592" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtv592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26568625" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Wilm</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Dineen</div> <div class="given-names">D</div></div></div>, et al.&nbsp; <div class="article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</div>. <div class="source">Mol Syst Biol</div>&nbsp; <div class="year">2011</div>;<div class="volume">7</div>:<div class="fpage">539.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%2C%20scalable%20generation%20of%20high-quality%20protein%20multiple%20sequence%20alignments%20using%20Clustal%20Omega&amp;author=F%20Sievers&amp;author=A%20Wilm&amp;author=D%20Dineen&amp;publication_year=2011&amp;journal=Mol%20Syst%20Biol&amp;volume=7&amp;pages=539." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2011.75" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.75" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fmsb.2011.75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21988835" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mirarab</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Warnow</div> <div class="given-names">T.</div></div></div> <div class="article-title">FastSP: linear time calculation of alignment accuracy</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2011</div>;<div class="volume">27</div>:<div class="fpage">3250</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FastSP%3A%20linear%20time%20calculation%20of%20alignment%20accuracy&amp;author=S%20Mirarab&amp;author=T.%20Warnow&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=3250-8" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr553" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr553" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21984754" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Toh</div> <div class="given-names">H.</div></div></div> <div class="article-title">PartTree: an algorithm to build an approximate tree from a large number of unaligned sequences</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2007</div>;<div class="volume">23</div>:<div class="fpage">372</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PartTree%3A%20an%20algorithm%20to%20build%20an%20approximate%20tree%20from%20a%20large%20number%20of%20unaligned%20sequences&amp;author=K%20Katoh&amp;author=H.%20Toh&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=372-4" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl592" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl592" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17118958" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">PM.</div></div></div> <div class="article-title">CLUSTAL: a package for performing multiple sequence alignment on a microcomputer</div>. <div class="source">Gene</div>&nbsp; <div class="year">1988</div>;<div class="volume">73</div>:<div class="fpage">237</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%3A%20a%20package%20for%20performing%20multiple%20sequence%20alignment%20on%20a%20microcomputer&amp;author=DG%20Higgins&amp;author=PM.%20Sharp&amp;publication_year=1988&amp;journal=Gene&amp;volume=73&amp;pages=237-44" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119%2888%2990330-7" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119%2888%2990330-7" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0378-1119(88)90330-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3243435" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Needleman</div> <div class="given-names">SB</div></div>, <div class="name string-name"><div class="surname">Wunsch</div> <div class="given-names">CD.</div></div></div> <div class="article-title">A general method applicable to the search for similarities in the amino acid sequence of two proteins</div>. <div class="source">J Mol Biol</div>&nbsp; <div class="year">1970</div>;<div class="volume">48</div>:<div class="fpage">443</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20method%20applicable%20to%20the%20search%20for%20similarities%20in%20the%20amino%20acid%20sequence%20of%20two%20proteins&amp;author=SB%20Needleman&amp;author=CD.%20Wunsch&amp;publication_year=1970&amp;journal=J%20Mol%20Biol&amp;volume=48&amp;pages=443-53" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836%2870%2990057-4" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836%2870%2990057-4" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0022-2836(70)90057-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5420325" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hogeweg</div> <div class="given-names">P</div></div>, <div class="name string-name"><div class="surname">Hesper</div> <div class="given-names">B.</div></div></div> <div class="article-title">The alignment of sets of sequences and the construction of phyletic trees: an integrated method</div>. <div class="source">J Mol Evol</div>&nbsp; <div class="year">1984</div>;<div class="volume">20</div>:<div class="fpage">175</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20alignment%20of%20sets%20of%20sequences%20and%20the%20construction%20of%20phyletic%20trees%3A%20an%20integrated%20method&amp;author=P%20Hogeweg&amp;author=B.%20Hesper&amp;publication_year=1984&amp;journal=J%20Mol%20Evol&amp;volume=20&amp;pages=175-86" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02257378" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02257378" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02257378"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6433036" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feng</div> <div class="given-names">DF</div></div>, <div class="name string-name"><div class="surname">Doolittle</div> <div class="given-names">RF.</div></div></div> <div class="article-title">Progressive sequence alignment as a prerequisite to correct phylogenetic trees</div>. <div class="source">J Mol Evol</div>&nbsp; <div class="year">1987</div>;<div class="volume">25</div>:<div class="fpage">351</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Progressive%20sequence%20alignment%20as%20a%20prerequisite%20to%20correct%20phylogenetic%20trees&amp;author=DF%20Feng&amp;author=RF.%20Doolittle&amp;publication_year=1987&amp;journal=J%20Mol%20Evol&amp;volume=25&amp;pages=351-60" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02603120" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02603120" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02603120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3118049" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dayhoff</div> <div class="given-names">MO</div></div>, <div class="name string-name"><div class="surname">Schwartz</div> <div class="given-names">RM</div></div>, <div class="name string-name"><div class="surname">Orcutt</div> <div class="given-names">BC.</div></div></div> A model of evolutionary change in proteins. In <div class="person-group"><div class="name string-name"><div class="surname">Dayhoff</div> <div class="given-names">MO</div></div>, <div class="name string-name"><div class="surname">Ech</div> <div class="given-names">RV</div></div></div> (eds), <div class="source">Atlas of Protein Sequence and Structure</div>&nbsp;. <div class="publisher-loc">Maryland</div>: <div class="publisher-name">National Biomedical Research Foundation</div>; <div class="year">1978</div>, <div class="fpage">345</div>–<div class="lpage">52</div>.</div></div></div></div><div content-id="bbx108-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">KD</div></div>, <div class="name string-name"><div class="surname">Tomii</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div></div> <div class="article-title">Application of the mafft sequence alignment program to large data-reexamination of the usefulness of chained guide trees</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2016</div>;<div class="volume">32</div>(<div class="issue">21</div>):<div class="fpage">3246</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20the%20mafft%20sequence%20alignment%20program%20to%20large%20data-reexamination%20of%20the%20usefulness%20of%20chained%20guide%20trees&amp;author=KD%20Yamada&amp;author=K%20Tomii&amp;author=K.%20Katoh&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=3246-51" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btw412" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtw412" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtw412"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27378296" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boyce</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">DG.</div></div></div> <div class="article-title">Simple chained guide trees give high-quality protein multiple sequence alignments</div>. <div class="source">Proc Natl Acad Sci USA</div>&nbsp; <div class="year">2014</div>;<div class="volume">111</div>:<div class="fpage">10556</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20chained%20guide%20trees%20give%20high-quality%20protein%20multiple%20sequence%20alignments&amp;author=K%20Boyce&amp;author=F%20Sievers&amp;author=DG.%20Higgins&amp;publication_year=2014&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=111&amp;pages=10556-61" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1405628111" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1405628111" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1405628111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25002495" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barton</div> <div class="given-names">GJ</div></div>, <div class="name string-name"><div class="surname">Sternberg</div> <div class="given-names">MJ.</div></div></div> <div class="article-title">A strategy for the rapid multiple alignment of protein sequences. confidence levels from tertiary structure comparisons</div>. <div class="source">J Mol Biol</div>&nbsp; <div class="year">1987</div>;<div class="volume">198</div>:<div class="fpage">327</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20strategy%20for%20the%20rapid%20multiple%20alignment%20of%20protein%20sequences.%20confidence%20levels%20from%20tertiary%20structure%20comparisons&amp;author=GJ%20Barton&amp;author=MJ.%20Sternberg&amp;publication_year=1987&amp;journal=J%20Mol%20Biol&amp;volume=198&amp;pages=327-37" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836%2887%2990316-0" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836%2887%2990316-0" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0022-2836(87)90316-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3430611" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berger</div> <div class="given-names">MP</div></div>, <div class="name string-name"><div class="surname">Munson</div> <div class="given-names">PJ.</div></div></div> <div class="article-title">A novel randomized iterative strategy for aligning multiple protein sequences</div>. <div class="source">Comput Appl Biosci</div>&nbsp; <div class="year">1991</div>;<div class="volume">7</div>:<div class="fpage">479</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20randomized%20iterative%20strategy%20for%20aligning%20multiple%20protein%20sequences&amp;author=MP%20Berger&amp;author=PJ.%20Munson&amp;publication_year=1991&amp;journal=Comput%20Appl%20Biosci&amp;volume=7&amp;pages=479-84" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1747779" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gotoh</div> <div class="given-names">O.</div></div></div> <div class="article-title">Optimal alignment between groups of sequences and its application to multiple sequence alignment</div>. <div class="source">Comput Appl Biosci</div>&nbsp; <div class="year">1993</div>;<div class="volume">9</div>:<div class="fpage">361</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimal%20alignment%20between%20groups%20of%20sequences%20and%20its%20application%20to%20multiple%20sequence%20alignment&amp;author=O.%20Gotoh&amp;publication_year=1993&amp;journal=Comput%20Appl%20Biosci&amp;volume=9&amp;pages=361-70" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8324637" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Frith</div> <div class="given-names">MC.</div></div></div> <div class="article-title">Adding unaligned sequences into an existing alignment using MAFFT and LAST</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2012</div>;<div class="volume">28</div>:<div class="fpage">3144</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adding%20unaligned%20sequences%20into%20an%20existing%20alignment%20using%20MAFFT%20and%20LAST&amp;author=K%20Katoh&amp;author=MC.%20Frith&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3144-6" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts578" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts578" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23023983" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le</div> <div class="given-names">Q</div></div>, <div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">DG.</div></div></div> <div class="article-title">Protein multiple sequence alignment benchmarking through secondary structure prediction</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2017</div>;<div class="volume">33</div>(<div class="issue">9</div>):<div class="fpage">1331</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20multiple%20sequence%20alignment%20benchmarking%20through%20secondary%20structure%20prediction&amp;author=Q%20Le&amp;author=F%20Sievers&amp;author=DG.%20Higgins&amp;publication_year=2017&amp;journal=Bioinformatics&amp;volume=33&amp;pages=1331-7" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28093407" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Notredame</div> <div class="given-names">C</div></div>, <div class="name string-name"><div class="surname">Holm</div> <div class="given-names">L</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">DG.</div></div></div> <div class="article-title">COFFEE: an objective function for multiple sequence alignments</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">1998</div>;<div class="volume">14</div>:<div class="fpage">407</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=COFFEE%3A%20an%20objective%20function%20for%20multiple%20sequence%20alignments&amp;author=C%20Notredame&amp;author=L%20Holm&amp;author=DG.%20Higgins&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=407-22" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.5.407" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.5.407" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F14.5.407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9682054" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">GM</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">DG.</div></div></div> <div class="article-title">Systematic exploration of guide-tree topology effects for small protein alignments</div>. <div class="source">BMC Bioinformatics</div>&nbsp; <div class="year">2014</div>;<div class="volume">15</div>:<div class="fpage">338.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20exploration%20of%20guide-tree%20topology%20effects%20for%20small%20protein%20alignments&amp;author=F%20Sievers&amp;author=GM%20Hughes&amp;author=DG.%20Higgins&amp;publication_year=2014&amp;journal=BMC%20Bioinformatics&amp;volume=15&amp;pages=338." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-15-338" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-15-338" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-15-338"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25282640" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tan</div> <div class="given-names">G</div></div>, <div class="name string-name"><div class="surname">Gil</div> <div class="given-names">M</div></div>, <div class="name string-name"><div class="surname">Löytynoja</div> <div class="given-names">AP</div></div></div>, et al.&nbsp; <div class="article-title">Simple
 chained guide trees give poorer multiple sequence alignments than 
inferred trees in simulation and phylogenetic benchmarks</div>. <div class="source">Proc Natl Acad Sci USA</div>&nbsp; <div class="year">2015</div>;<div class="volume">112</div>:<div class="fpage">E99</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20chained%20guide%20trees%20give%20poorer%20multiple%20sequence%20alignments%20than%20inferred%20trees%20in%20simulation%20and%20phylogenetic%20benchmarks&amp;author=G%20Tan&amp;author=M%20Gil&amp;author=AP%20L%C3%B6ytynoja&amp;publication_year=2015&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=112&amp;pages=E99-100" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1417526112" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1417526112" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1417526112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25564672" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">NPD</div></div>, <div class="name string-name"><div class="surname">Mirarab</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">K</div></div></div>, et al.&nbsp; <div class="article-title">Ultra-large alignments using phylogeny-aware profiles</div>. <div class="source">Genome Biol</div>&nbsp; <div class="year">2015</div>;<div class="volume">16</div>:<div class="fpage">124.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultra-large%20alignments%20using%20phylogeny-aware%20profiles&amp;author=NPD%20Nguyen&amp;author=S%20Mirarab&amp;author=K%20Kumar&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=124." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0688-z" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0688-z" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs13059-015-0688-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26076734" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blackshields</div> <div class="given-names">G</div></div>, <div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">W</div></div></div>, et al.&nbsp; <div class="article-title">Sequence embedding for fast construction of guide trees for multiple sequence alignment</div>. <div class="source">Algorithms Mol Biol</div>&nbsp; <div class="year">2010</div>;<div class="volume">5</div>:<div class="fpage">21.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20embedding%20for%20fast%20construction%20of%20guide%20trees%20for%20multiple%20sequence%20alignment&amp;author=G%20Blackshields&amp;author=F%20Sievers&amp;author=W%20Shi&amp;publication_year=2010&amp;journal=Algorithms%20Mol%20Biol&amp;volume=5&amp;pages=21." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1748-7188-5-21" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1748-7188-5-21" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1748-7188-5-21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20470396" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mirarab</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">N</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">S</div></div></div>, et al.&nbsp; <div class="article-title">PASTA: ultra-large multiple sequence alignment for nucleotide and amino-acid sequences</div>. <div class="source">J Comput Biol</div>&nbsp; <div class="year">2015</div>;<div class="volume">22</div>(<div class="issue">5</div>):<div class="fpage">377</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PASTA%3A%20ultra-large%20multiple%20sequence%20alignment%20for%20nucleotide%20and%20amino-acid%20sequences&amp;author=S%20Mirarab&amp;author=N%20Nguyen&amp;author=S%20Guo&amp;publication_year=2015&amp;journal=J%20Comput%20Biol&amp;volume=22&amp;pages=377-86" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2014.0156" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2014.0156" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1089%2Fcmb.2014.0156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25549288" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">RD</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">SR.</div></div></div> <div class="article-title">Hmmer web server: interactive sequence similarity searching</div>. <div class="source">Nucleic Acids Res</div>&nbsp; <div class="year">2011</div>;<div class="volume">39</div>:<div class="fpage">W29</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hmmer%20web%20server%3A%20interactive%20sequence%20similarity%20searching&amp;author=RD%20Finn&amp;author=J%20Clements&amp;author=SR.%20Eddy&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W29-37" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr367" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr367" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593126" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berger</div> <div class="given-names">SA</div></div>, <div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> <div class="article-title">Aligning short reads to reference alignments and trees</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2011</div>;<div class="volume">27</div>:<div class="fpage">2068</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aligning%20short%20reads%20to%20reference%20alignments%20and%20trees&amp;author=SA%20Berger&amp;author=A.%20Stamatakis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2068-75" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr320" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr320" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21636595" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Löytynoja</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Vilella</div> <div class="given-names">AJ</div></div>, <div class="name string-name"><div class="surname">Goldman</div> <div class="given-names">N.</div></div></div> <div class="article-title">Accurate extension of multiple sequence alignments using a phylogeny-aware graph algorithm</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2012</div>;<div class="volume">28</div>:<div class="fpage">1684</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20extension%20of%20multiple%20sequence%20alignments%20using%20a%20phylogeny-aware%20graph%20algorithm&amp;author=A%20L%C3%B6ytynoja&amp;author=AJ%20Vilella&amp;author=N.%20Goldman&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1684-91" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts198" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts198" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22531217" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gotoh</div> <div class="given-names">O</div></div>, <div class="name string-name"><div class="surname">Morita</div> <div class="given-names">M</div></div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">DR.</div></div></div> <div class="article-title">Assessment and refinement of eukaryotic gene structure prediction with gene-structure-aware multiple protein sequence alignment</div>. <div class="source">BMC Bioinformatics</div>&nbsp; <div class="year">2014</div>;<div class="volume">15</div>:<div class="fpage">189.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20and%20refinement%20of%20eukaryotic%20gene%20structure%20prediction%20with%20gene-structure-aware%20multiple%20protein%20sequence%20alignment&amp;author=O%20Gotoh&amp;author=M%20Morita&amp;author=DR.%20Nelson&amp;publication_year=2014&amp;journal=BMC%20Bioinformatics&amp;volume=15&amp;pages=189." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-15-189" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-15-189" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-15-189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24927652" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagy</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Patthy</div> <div class="given-names">L.</div></div></div> <div class="article-title">MisPred: a resource for identification of erroneous protein sequences in public databases</div>. <div class="source">Database</div>&nbsp; <div class="year">2013</div>;<div class="volume">2013</div>:<div class="fpage">bat053.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MisPred%3A%20a%20resource%20for%20identification%20of%20erroneous%20protein%20sequences%20in%20public%20databases&amp;author=A%20Nagy&amp;author=L.%20Patthy&amp;publication_year=2013&amp;journal=Database&amp;volume=2013&amp;pages=bat053." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/database/bat053" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdatabase%2Fbat053" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fdatabase%2Fbat053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23864220" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yandell</div> <div class="given-names">M</div></div>, <div class="name string-name"><div class="surname">Ence</div> <div class="given-names">D.</div></div></div> <div class="article-title">A beginner’s guide to eukaryotic genome annotation</div>. <div class="source">Nat Rev Genet</div>&nbsp; <div class="year">2012</div>;<div class="volume">13</div>:<div class="fpage">329</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20beginner%E2%80%99s%20guide%20to%20eukaryotic%20genome%20annotation&amp;author=M%20Yandell&amp;author=D.%20Ence&amp;publication_year=2012&amp;journal=Nat%20Rev%20Genet&amp;volume=13&amp;pages=329-42" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3174" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3174" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrg3174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22510764" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kuraku</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Zmasek</div> <div class="given-names">CM</div></div>, <div class="name string-name"><div class="surname">Nishimura</div> <div class="given-names">O</div></div></div>, et al.&nbsp; <div class="article-title">aLeaves
 facilitates on-demand exploration of metazoan gene family trees on 
mafft sequence alignment server with enhanced interactivity</div>. <div class="source">Nucleic Acids Res</div>&nbsp; <div class="year">2013</div>;<div class="volume">41</div>:<div class="fpage">W22</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=aLeaves%20facilitates%20on-demand%20exploration%20of%20metazoan%20gene%20family%20trees%20on%20mafft%20sequence%20alignment%20server%20with%20enhanced%20interactivity&amp;author=S%20Kuraku&amp;author=CM%20Zmasek&amp;author=O%20Nishimura&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=W22-8" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt389" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt389" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkt389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23677614" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">W</div></div>, <div class="name string-name"><div class="surname">Jaroszewski</div> <div class="given-names">L</div></div>, <div class="name string-name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div></div> <div class="article-title">Clustering of highly homologous sequences to reduce the size of large protein databases</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2001</div>;<div class="volume">17</div>(<div class="issue">3</div>):<div class="fpage">282</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustering%20of%20highly%20homologous%20sequences%20to%20reduce%20the%20size%20of%20large%20protein%20databases&amp;author=W%20Li&amp;author=L%20Jaroszewski&amp;author=A.%20Godzik&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=282-3" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.3.282" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.3.282" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F17.3.282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11294794" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gouveia-Oliveira</div> <div class="given-names">R</div></div>, <div class="name string-name"><div class="surname">Sackett</div> <div class="given-names">PW</div></div>, <div class="name string-name"><div class="surname">Pedersen</div> <div class="given-names">AG.</div></div></div> <div class="article-title">MaxAlign: maximizing usable data in an alignment</div>. <div class="source">BMC Bioinformatics</div>&nbsp; <div class="year">2007</div>;<div class="volume">8</div>:<div class="fpage">312.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MaxAlign%3A%20maximizing%20usable%20data%20in%20an%20alignment&amp;author=R%20Gouveia-Oliveira&amp;author=PW%20Sackett&amp;author=AG.%20Pedersen&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics&amp;volume=8&amp;pages=312." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-312" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-312" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-8-312"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17725821" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saitou</div> <div class="given-names">N</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div></div> <div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>. <div class="source">Mol Biol Evol</div>&nbsp; <div class="year">1987</div>;<div class="volume">4</div>(<div class="issue">4</div>):<div class="fpage">406</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol%20Biol%20Evol&amp;volume=4&amp;pages=406-25" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sokal</div> <div class="given-names">RR</div></div>, <div class="name string-name"><div class="surname">Michener</div> <div class="given-names">CD.</div></div></div> <div class="article-title">A statistical method for evaluating systematic relationships</div>. <div class="source">University of Kansas Scientific Bulletin</div>&nbsp; <div class="year">1958</div>;<div class="volume">28</div>:<div class="fpage">1409</div>–<div class="lpage">38</div>.</div></div></div></div><div content-id="bbx108-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">O</div></div>, <div class="name string-name"><div class="surname">Dylus</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Dessimoz</div> <div class="given-names">C.</div></div></div> <div class="article-title">Phylo.io: interactive viewing and comparison of large phylogenetic trees on the web</div>. <div class="source">Mol Biol Evol</div>&nbsp; <div class="year">2016</div>;<div class="volume">33</div>(<div class="issue">8</div>):<div class="fpage">2163</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylo.io%3A%20interactive%20viewing%20and%20comparison%20of%20large%20phylogenetic%20trees%20on%20the%20web&amp;author=O%20Robinson&amp;author=D%20Dylus&amp;author=C.%20Dessimoz&amp;publication_year=2016&amp;journal=Mol%20Biol%20Evol&amp;volume=33&amp;pages=2163-6" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw080" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw080" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsw080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27189561" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">MV</div></div>, <div class="name string-name"><div class="surname">Zmasek</div> <div class="given-names">CM.</div></div></div> <div class="article-title">phyloXML: XML for evolutionary biology and comparative genomics</div>. <div class="source">BMC Bioinformatics</div>&nbsp; <div class="year">2009</div>;<div class="volume">10</div>:<div class="fpage">356.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=phyloXML%3A%20XML%20for%20evolutionary%20biology%20and%20comparative%20genomics&amp;author=MV%20Han&amp;author=CM.%20Zmasek&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=356." target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-356" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-356" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-10-356"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19860910" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">AM</div></div>, <div class="name string-name"><div class="surname">Procter</div> <div class="given-names">JB</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">DM</div></div></div>, et al.&nbsp; <div class="article-title">Jalview version 2–a multiple sequence alignment editor and analysis workbench</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2009</div>;<div class="volume">25</div>(<div class="issue">9</div>):<div class="fpage">1189</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Jalview%20version%202%E2%80%93a%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench&amp;author=AM%20Waterhouse&amp;author=JB%20Procter&amp;author=DM%20Martin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1189-91" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp033" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp033" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151095" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yachdav</div> <div class="given-names">G</div></div>, <div class="name string-name"><div class="surname">Wilzbach</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Rauscher</div> <div class="given-names">B</div></div></div>, et al.&nbsp; <div class="article-title">MSAViewer: interactive JavaScript visualization of multiple sequence alignments</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2016</div>;<div class="volume">32</div>(<div class="issue">22</div>):<div class="fpage">3501</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MSAViewer%3A%20interactive%20JavaScript%20visualization%20of%20multiple%20sequence%20alignments&amp;author=G%20Yachdav&amp;author=S%20Wilzbach&amp;author=B%20Rauscher&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=3501-3" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27412096" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Dineen</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Wilm</div> <div class="given-names">A</div></div></div>, et al.&nbsp; <div class="article-title">Making automated multiple alignments of very large numbers of protein sequences</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="year">2013</div>;<div class="volume">29</div>(<div class="issue">8</div>):<div class="fpage">989</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Making%20automated%20multiple%20alignments%20of%20very%20large%20numbers%20of%20protein%20sequences&amp;author=F%20Sievers&amp;author=D%20Dineen&amp;author=A%20Wilm&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=989-95" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btt093" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt093" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23428640" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kamisetty</div> <div class="given-names">H</div></div>, <div class="name string-name"><div class="surname">Ovchinnikov</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Baker</div> <div class="given-names">D.</div></div></div> <div class="article-title">Assessing the utility of coevolution-based residue-residue contact predictions in a sequence- and structure-rich era</div>. <div class="source">Proc Natl Acad Sci USA</div>&nbsp; <div class="year">2013</div>;<div class="volume">110</div>:<div class="fpage">15674</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20utility%20of%20coevolution-based%20residue-residue%20contact%20predictions%20in%20a%20sequence-%20and%20structure-rich%20era&amp;author=H%20Kamisetty&amp;author=S%20Ovchinnikov&amp;author=D.%20Baker&amp;publication_year=2013&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=110&amp;pages=15674-9" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1314045110" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1314045110" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1314045110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24009338" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bbx108-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bbx108-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marks</div> <div class="given-names">DS</div></div>, <div class="name string-name"><div class="surname">Hopf</div> <div class="given-names">TA</div></div>, <div class="name string-name"><div class="surname">Sander</div> <div class="given-names">C.</div></div></div> <div class="article-title">Protein structure prediction from sequence variation</div>. <div class="source">Nat Biotechnol</div>&nbsp; <div class="year">2012</div>;<div class="volume">30</div>(<div class="issue">11</div>):<div class="fpage">1072</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20structure%20prediction%20from%20sequence%20variation&amp;author=DS%20Marks&amp;author=TA%20Hopf&amp;author=C.%20Sander&amp;publication_year=2012&amp;journal=Nat%20Biotechnol&amp;volume=30&amp;pages=1072-80" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2419" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2419" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.2419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23138306" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="">http://creativecommons.org/licenses/by-nc/4.0/</a>),
 which permits non-commercial re-use, distribution, and reproduction in 
any medium, provided the original work is properly cited. For commercial
 re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget --> 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=4106928&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5143" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div>
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/bib_Behind_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">7,993</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/bib/bbx108" data-uuid="ebb4da0c-440a-66e6-1066-75c2bb4fb9e8"><a target="" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=24893038" style="display:inline-block;">
        <img alt="Article has an altmetric score of 3" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4106928"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fbib%2fadvance-article%2fdoi%2f10.1093%2fbib%2fbbx108%2f4106928">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fbib%2Fadvance-article%2Fdoi%2F10.1093%2Fbib%2Fbbx108%2F4106928" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/bib/CrossRef-CitedBy/4106928">CrossRef</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz038/5436939?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   100 Years of evolving gene–disease complexities and scientific debutants
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz039/5432309?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   DNA methylation analysis in plants: review of computational tools and future perspectives
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz026/5428023?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Validation strategies for target prediction methods
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz037/5424984?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   A feature-based approach to predict hot spots in protein–DNA binding interfaces
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bbz035/5424165?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Molecular Biology Information 
Service: an innovative medical library-based bioinformatics support 
service for biomedical researchers
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/bib_Behind_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

        </div> 
    </div>
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="MAFFT online service: multiple sequence alignment, interactive sequence choice and visualization" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    


        
        <script type="text/javascript" src="MathJax.js"></script>
        
        <script type="text/javascript" src="footer.js"></script>

        
    
    <script type="text/javascript" src="addthis_widget.js"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh870" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.6072316804029534&amp;iit=1555594930656&amp;tmr=load%3D1555594930624%26core%3D1555594930650%26main%3D1555594930652%26ifr%3D1555594930660&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fbib%2Fadvance-article%2Fdoi%2F10.1093%2Fbib%2Fbbx108%2F4106928&amp;href=https%3A%2F%2Facademic.oup.com%2Fbib%2Fadvance-article%2Fdoi%2F10.1093%2Fbib%2Fbbx108%2F4106928&amp;dt=MAFFT%20online%20service%3A%20multiple%20sequence%20alignment%2C%20interactive%20sequence%20choice%20and%20visualization&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5cb87eb2878edc2d&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DMAFFT%2520online%2520service%253A%2520multiple%2520sequence%2520alignment%252C%2520interactive%2520sequence%2520choice%2520and%2520visualization%26description%3DAbstract.%2520%2520This%2520article%2520describes%2520several%2520features%2520in%2520the%2520MAFFT%2520online%2520service%2520for%2520multiple%2520sequence%2520alignment%2520%28MSA%29.%2520As%2520a%2520result%2520of%2520recent%2520advances%2520in%2520sequenci%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fbib%252Fadvance-article%252Fdoi%252F10.1093%252Fbib%252Fbbx108%252F4106928%26image%3Dhttps%253A%252F%252Facademic.oup.com%252FUI%252Fapp%252Fimg%252Fgeneric-cover.png%26updated_time%3D2017-09-06%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.4.0-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/bib/bbx108");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/bib_Behind_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/bib/pages/About">About Briefings in Bioinformatics</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/bib/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/bib/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
                            <li class="link-5">
                                <a href="https://twitter.com/oxfordjournals">Twitter</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://academic.oup.com/bib/subscribe">Purchase</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-4">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;search=&amp;t732=470054&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-BriefingsinBioinformatics" class="journal-footer-affiliations-logo" src="bib_f1526563102.svg" alt="Briefings in Bioinformatics">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1477-4054</li>
                                        <li>Copyright © 2019 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5143&amp;returnUrl=%2fbib%2fadvance-article%2fdoi%2f10.1093%2fbib%2fbbx108%2f4106928">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ"/>
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="bbx108.js"></script><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size4,sans-serif;"></div></div></body></html>