<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script>

        <title>IQ-TREE: A Fast and Effective Stochastic Algorithm for 
Estimating Maximum-Likelihood Phylogenies | Molecular Biology and 
Evolution | Oxford Academic</title>

        <script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>




    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site-1495373344.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'mbe';
            App.SiteURL = 'academic.oup.com/mbe';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/mbe/article/32/1/268/2925592","@type":"ScholarlyArticle","name":"IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies","datePublished":"2014-11-03","isPartOf":{"@id":"https://academic.oup.com/mbe/issue/32/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-01-01","isPartOf":{"@id":"https://academic.oup.com/mbe","@type":"Periodical","name":"Molecular Biology and Evolution","issn":["1537-1719"]}},"url":"http://dx.doi.org/10.1093/molbev/msu300","keywords":["phylogenetic inference","phylogeny","maximum likelihood","stochastic algorithm"],"inLanguage":"en","copyrightHolder":"Society for Molecular Biology and Evolution","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/mbe/article/32/1/268/2925592","author":[{"name":"Nguyen, Lam-Tung","affiliation":"Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria  Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria","@type":"Person"},{"name":"Schmidt, Heiko A.","affiliation":"Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria","@type":"Person"},{"name":"von Haeseler, Arndt","affiliation":"Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria  Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria","@type":"Person"},{"name":"Minh, Bui Quang","affiliation":"Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria","@type":"Person"}],"description":"Abstract.  Large phylogenomics data sets require fast tree inference methods, especially for maximum-likelihood (ML) phylogenies. Fast programs exist, but due t","pageStart":"268","pageEnd":"274","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/32/1/0/m_mbe32_1.cover.gif?Expires=1637255441&Signature=KpMVQfbTV4w5LSS9zy~MpzaUuBLvehoxmA46PzIPbOrLpe5KqxuA-ibzV5cyLEuGZC-3~iDob5efQA38ML7ypFCkkLNUsCBZb~vFHZj-kHo1uL9TeD3PuWZ7oTdJchQCR~JqCunxxOGgqiOJiglfao-9Nkg8mMleZN5B4yKDIojyoHp1rSsJS1x1XhrTdSiX-S2wjSGxbOnKGc1w-E5gFXu3Ls7HgfhAgHj3WLRgnq3XylZ-1OTRIF9eYYCVLgmH~IR437y43FZXTdACu8iejn82KWxuM2rf7pD9fVHOkvEMXMcU5k8SSV-HxJSRpAAs3D7umGd6NuvYnZ3K-Y-eAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/32/1/0/m_mbe32_1.cover.gif?Expires=1637255441&Signature=KpMVQfbTV4w5LSS9zy~MpzaUuBLvehoxmA46PzIPbOrLpe5KqxuA-ibzV5cyLEuGZC-3~iDob5efQA38ML7ypFCkkLNUsCBZb~vFHZj-kHo1uL9TeD3PuWZ7oTdJchQCR~JqCunxxOGgqiOJiglfao-9Nkg8mMleZN5B4yKDIojyoHp1rSsJS1x1XhrTdSiX-S2wjSGxbOnKGc1w-E5gFXu3Ls7HgfhAgHj3WLRgnq3XylZ-1OTRIF9eYYCVLgmH~IR437y43FZXTdACu8iejn82KWxuM2rf7pD9fVHOkvEMXMcU5k8SSV-HxJSRpAAs3D7umGd6NuvYnZ3K-Y-eAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies">
    <meta property="og:description" content="Abstract.  Large phylogenomics data sets require fast tree inference methods, especially for maximum-likelihood (ML) phylogenies. Fast programs exist, but due t">
    <meta property="og:url" content="https://academic.oup.com/mbe/article/32/1/268/2925592">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/32/1/0/m_mbe32_1.cover.gif?Expires=1637255441&amp;Signature=KpMVQfbTV4w5LSS9zy~MpzaUuBLvehoxmA46PzIPbOrLpe5KqxuA-ibzV5cyLEuGZC-3~iDob5efQA38ML7ypFCkkLNUsCBZb~vFHZj-kHo1uL9TeD3PuWZ7oTdJchQCR~JqCunxxOGgqiOJiglfao-9Nkg8mMleZN5B4yKDIojyoHp1rSsJS1x1XhrTdSiX-S2wjSGxbOnKGc1w-E5gFXu3Ls7HgfhAgHj3WLRgnq3XylZ-1OTRIF9eYYCVLgmH~IR437y43FZXTdACu8iejn82KWxuM2rf7pD9fVHOkvEMXMcU5k8SSV-HxJSRpAAs3D7umGd6NuvYnZ3K-Y-eAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2014-11-03">
<meta name="citation_author" content="Nguyen, Lam-Tung"><meta name="citation_author_institution" content="Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria"><meta name="citation_author_institution" content="Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria"><meta name="citation_author" content="Schmidt, Heiko A."><meta name="citation_author_institution" content="Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria"><meta name="citation_author" content="von Haeseler, Arndt"><meta name="citation_author_institution" content="Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria"><meta name="citation_author_institution" content="Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria"><meta name="citation_author" content="Minh, Bui Quang"><meta name="citation_author_institution" content="Center for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, University of Vienna, Medical University of Vienna, Vienna, Austria"><meta name="citation_title" content="IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies"><meta name="citation_firstpage" content="268"><meta name="citation_lastpage" content="274"><meta name="citation_doi" content="10.1093/molbev/msu300"><meta name="citation_journal_title" content="Molecular Biology and Evolution"><meta name="citation_journal_abbrev" content="Mol Biol Evol"><meta name="citation_volume" content="32"><meta name="citation_issue" content="1"><meta name="citation_publication_date" content="2015/01/01"><meta name="citation_issn" content="0737-4038"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Maximum likelihood of evolutionary trees is hard; Lect Notes Comput Sci.;  citation_year=2005;  citation_volume=3500;  citation_pages=296-310; "><meta name="citation_reference" content="citation_title=Methods for computing Wagner trees; Syst Zool.;  citation_year=1970;  citation_volume=19;  citation_pages=83-92; "><meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach; J Mol Evol.;  citation_year=1981;  citation_volume=17;  citation_pages=368-376; "><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland (MA); Inferring phylogenies;  citation_year=2004; "><meta name="citation_reference" content="citation_title=Toward defining course of evolution—minimum change for a specific tree topology; Syst Zool.;  citation_year=1971;  citation_volume=20;  citation_pages=406-416; "><meta name="citation_reference" content="citation_title=The phylogenetic likelihood library; citation_text=Advance Access published October 30, 2014, doi:10.1093/sysbio/syu084; Syst Biol.;  citation_year=2014; "><meta name="citation_reference" content="citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data; Mol Biol Evol.;  citation_year=1997;  citation_volume=14;  citation_pages=685-695; "><meta name="citation_reference" content="citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0; Syst Biol.;  citation_year=2010;  citation_volume=59;  citation_pages=307-321; "><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood; Syst Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=696-704; "><meta name="citation_reference" content="citation_title=MetaPIGA v2.0: maximum likelihood large phylogeny estimation using the metapopulation genetic algorithm and other stochastic heuristics; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=379"><meta name="citation_reference" content="citation_title=Improving the efficiency of SPR moves in phylogenetic tree search methods based on maximum likelihood; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=4338-4347; "><meta name="citation_reference" content="citation_title=A new method for calculating evolutionary substitution rates; J Mol Evol.;  citation_year=1984;  citation_volume=20;  citation_pages=86-93; "><meta name="citation_reference" content="citation_title=A genetic algorithm for maximum-likelihood phylogeny inference using nucleotide sequence data; Mol Biol Evol.;  citation_year=1998;  citation_volume=15;  citation_pages=277-283; "><meta name="citation_reference" content="citation_title=Ultrafast approximation for phylogenetic bootstrap; Mol Biol Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=1188-1195; "><meta name="citation_reference" content="citation_title=pIQPNNI: parallel reconstruction of large maximum likelihood phylogenies; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3794-3796; "><meta name="citation_reference" content="citation_title=Characterizing the phylogenetic tree-search problem; Syst Biol.;  citation_year=2012;  citation_volume=61;  citation_pages=228-239; "><meta name="citation_reference" content="citation_title=Increasing the efficiency of searches for the maximum likelihood tree in a phylogenetic analysis of up to 150 nucleotide sequences; Syst Biol.;  citation_year=2007;  citation_volume=56;  citation_pages=988-1010; "><meta name="citation_reference" content="citation_publisher=Frommann-Holzboog Verlag, Stuttgart (Germany); Evolutionsstrategie—optimierung technischer systeme nach prinzipien der biologischen evolution;  citation_year=1973; "><meta name="citation_reference" content="citation_title=Stochastic search strategy for estimation of maximum likelihood phylogenetic trees; Syst Biol.;  citation_year=2001;  citation_volume=50;  citation_pages=7-17; "><meta name="citation_reference" content="citation_title=TreeBASE: a prototype database of phylogenetic analyses and an interactive tool for browsing the phylogeny of life; Am J Bot.;  citation_year=1994;  citation_volume=81;  citation_pages=183"><meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2688-2690; "><meta name="citation_reference" content="citation_title=PAUP*. Phylogenetic analysis using parsimony (* and other methods); citation_text=Version 4. Sunderland (MA): Sinauer Associates;  citation_year=2003; "><meta name="citation_reference" content="citation_title=Phylogeny reconstruction; citation_publisher=Sunderland (MA), Sinauer Associates; Molecular systematics;  citation_year=1990;  citation_pages=411-501; "><meta name="citation_reference" content="citation_title=IQPNNI: moving fast through tree space and stopping in time; Mol Biol Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=1565-1571; "><meta name="citation_reference" content="citation_title=Accelerated likelihood surface exploration: the likelihood ratchet; Syst Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=368-373; "><meta name="citation_reference" content="citation_title=A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach; Mol Biol Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=691-699; "><meta name="citation_reference" content="citation_title=The prevalence of multifurcations in tree-space and their implications for tree-search; Mol Biol Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=2674-2677; "><meta name="citation_reference" content="citation_title=Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods; J Mol Evol.;  citation_year=1994;  citation_volume=39;  citation_pages=306-314; "><meta name="citation_reference" content="citation_title=Maximum likelihood estimation on large phylogenies and analysis of adaptive evolution in human influenza virus A; J Mol Evol.;  citation_year=2000;  citation_volume=51;  citation_pages=423-432; "><meta name="citation_reference" content="citation_title=Genetic algorithm approaches for the phylogenetic analysis of large biological sequence data sets under the maximum likelihood criterion; citation_text=[Ph.D. dissertation]. Austin (TX): The University of Texas at Austin;  citation_year=2006; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/mbe/article-pdf/32/1/268/13171186/msu300.pdf"><meta name="description" content="Abstract.  Large phylogenomics data sets require fast tree inference methods, especially for maximum-likelihood (ML) phylogenies. Fast programs exist, but due t">    <script id="altmetric-embed-js" src="altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019111401.js" async=""></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2.css"><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head><body data-sitestyletemplate="Journal" theme-mbe="" class="off-canvas pg_Article pg_article" data-sitename="molecularbiologyandevolution"><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/mbe/article-xml/32/1/268/2925592">



    <link rel="canonical" href="https://academic.oup.com/mbe/article/32/1/268/2925592">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "mbe/article/32/1/268/2925592");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    
    

    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=molbev&authzrequired=false&doi=10.1093/molbev/msu300';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5325" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f32%2f1%2f268%2f2925592" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f32%2f1%2f268%2f2925592" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429235">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429232">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">More content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429236">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429237">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%E2%80%93+Newest+First&amp;f_TocHeadingTitle=Fast+TracksANDFAST+TRACK" class="nav-link ">Fast Track</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429238">
            <a href="https://academic.oup.com/mbe/search-results?allJournals=1&amp;f_TocHeadingTitle=Perspective&amp;fl_SiteID=5325&amp;page=1&amp;sort=Date+%e2%80%93+Newest+First" class="nav-link ">Perspectives</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429239">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Discoveries&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Discoveries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429240">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=METHODS&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Methods</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429241">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Resources&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Resources</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429242">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;f_TocHeadingTitle=PROTOCOLSANDProtocol&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Protocols</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429243">
            <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=News" class="nav-link ">News</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429233">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429244">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429245">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429246">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429247">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429248">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429249">
            <a href="https://academic.oup.com/mbe/pages/Open_Access" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429250">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429251">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429234">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429252">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429253">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429254">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429255">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429256">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429257">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429258">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429259">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429260">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/mbe" class="journal-logo-container">
                    <img id="logo-MolecularBiologyandEvolution" class="journal-logo" src="mbe_title188616415.svg" alt="Molecular Biology and Evolution">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.smbe.org/smbe/" target="" class="society-logo-container">
                                <img id="logo-SocietyforMolecularBiologyandEvolution" class="society-logo" src="mbe_h1-1277340080.svg" alt="Society for Molecular Biology and Evolution">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429235">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429232">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">More content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429236">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429237">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%E2%80%93+Newest+First&amp;f_TocHeadingTitle=Fast+TracksANDFAST+TRACK" class="nav-link ">Fast Track</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429238">
            <a href="https://academic.oup.com/mbe/search-results?allJournals=1&amp;f_TocHeadingTitle=Perspective&amp;fl_SiteID=5325&amp;page=1&amp;sort=Date+%e2%80%93+Newest+First" class="nav-link ">Perspectives</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429239">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Discoveries&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Discoveries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429240">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=METHODS&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Methods</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429241">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Resources&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Resources</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429242">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;f_TocHeadingTitle=PROTOCOLSANDProtocol&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Protocols</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429243">
            <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=News" class="nav-link ">News</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429233">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429244">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429245">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429246">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429247">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429248">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429249">
            <a href="https://academic.oup.com/mbe/pages/Open_Access" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429250">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429251">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429234">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429252">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429253">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429254">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429255">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429256">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429257">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429258">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429259">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429260">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option>
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="SMBE Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/smbe" type="hidden">
    <input name="siteID" id="hfsiteID" value="5325" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5538" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=45913" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/mbe" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/mbe/issue/32/1"><img id="issueImage" class="fb-featured-image" src="m_mbe32_1.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/mbe/issue/32/1">
                        <div class="volume trailing-comma">Volume 32</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
January 2015        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="74402238">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402238">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74402240">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402240">Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74402244">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402244">New Approaches</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74402246">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402246">Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74402266">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402266">Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74402274">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402274">Materials and Methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74402289">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402289">Acknowledgments</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="74402291">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74402291">References</a>
            </div>
        </li>
<li class="section-jump-link head-1"><div class="section-jump-link__link-wrap"><a class="jumplink js-jumplink scrollTo" href="#authorNotesSectionTitle">Author notes</a></div></li><li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/mbe/article/32/1/258/2925575">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/mbe/article/32/1/275/2925559">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lam-Tung Nguyen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center
 for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, 
University of Vienna, Medical University of Vienna, Vienna, Austria</div><div class="aff"><div class="label"><sup>2</sup></div>Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Lam-Tung+Nguyen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nguyen%20L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nguyen%20Lam-Tung%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Lam-Tung Nguyen</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Heiko A. Schmidt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center
 for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, 
University of Vienna, Medical University of Vienna, Vienna, Austria</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Heiko+A.+Schmidt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schmidt%20H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Schmidt%20Heiko%20A.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Heiko A. Schmidt</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arndt von Haeseler</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center
 for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, 
University of Vienna, Medical University of Vienna, Vienna, Austria</div><div class="aff"><div class="label"><sup>2</sup></div>Bioinformatics and Computational Biology, Faculty of Computer Science, University of Vienna, Vienna, Austria</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Arndt+von+Haeseler">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=von%20Haeseler%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22von%20Haeseler%20Arndt%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Arndt von Haeseler</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bui Quang Minh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center
 for Integrative Bioinformatics Vienna, Max F. Perutz Laboratories, 
University of Vienna, Medical University of Vienna, Vienna, Austria</div>
        </div>


        <div class="info-author-correspondence">
            <strong>*Corresponding author:</strong> E-mail: <a href="mailto:minh.bui@univie.ac.at" target="">minh.bui@univie.ac.at</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Bui+Quang+Minh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minh%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Minh%20Bui%20Quang%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Bui Quang Minh</a><span class="delimiter"><i class="icon-general-mail"></i></span>
                        </span>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="footnote-content"><strong>Associate editor:</strong> Barbara Holland</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Molecular Biology and Evolution</em>, Volume 32, Issue 1, January 2015, Pages 268–274, <a href="https://doi.org/10.1093/molbev/msu300">https://doi.org/10.1093/molbev/msu300</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">03 November 2014</div>
            </div>
        </div>
    </div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2925592" href="https://academic.oup.com/mbe/article-pdf/32/1/268/13171186/msu300.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Lam-Tung Nguyen, Heiko A. Schmidt, Arndt von Haeseler, Bui Quang 
Minh,  IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating
 Maximum-Likelihood Phylogenies, <em>Molecular Biology and Evolution</em>, Volume 32, Issue 1, January 2015, Pages 268–274, <a href="https://doi.org/10.1093/molbev/msu300">https://doi.org/10.1093/molbev/msu300</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925592&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1537-1719&amp;title=IQ-TREE%3A%20A%20Fast%20and%20Effective%20Stochastic%20Algorithm%20for%20Estimating%20Maximum-Likelihood%20Phylogenies&amp;publicationDate=2014-11-03&amp;volumeNum=32&amp;issueNum=1&amp;author=Nguyen%2C%20Lam-Tung%3B%20Schmidt%2C%20Heiko%20A.&amp;startPage=268&amp;endPage=274&amp;contentId=10.1093%2Fmolbev%2Fmsu300&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f32%2f1%2f268%2f2925592" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option>
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="74402238" id="74402238" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Large phylogenomics data sets require fast 
tree inference methods, especially for maximum-likelihood (ML) 
phylogenies. Fast programs exist, but due to inherent heuristics to find
 optimal trees, it is not clear whether the best tree is found. Thus, 
there is need for additional approaches that employ different search 
strategies to find ML trees and that are at the same time as fast as 
currently available ML programs. We show that a combination of 
hill-climbing approaches and a stochastic perturbation method can be 
time-efficiently implemented. If we allow the same CPU time as RAxML and
 PhyML, then our software IQ-TREE found higher likelihoods between 62.2%
 and 87.1% of the studied alignments, thus efficiently exploring the 
tree-space. If we use the IQ-TREE stopping rule, RAxML and PhyML are 
faster in 75.7% and 47.1% of the DNA alignments and 42.2% and 100% of 
the protein alignments, respectively. However, the range of obtaining 
higher likelihoods with IQ-TREE improves to 73.3–97.1%. IQ-TREE is 
freely available at <a class="link link-uri" href="http://www.cibiv.at/software/iqtree" target="">http://www.cibiv.at/software/iqtree</a>.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic inference&quot;">phylogenetic inference</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogeny">phylogeny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;maximum likelihood&quot;">maximum likelihood</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;stochastic algorithm&quot;">stochastic algorithm</a></div><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=METHODS">METHODS</a>
                </div></div>
                    <h2 scrollto-destination="74402240" id="74402240" class="section-title">Introduction</h2>
<p>Phylogenetic inference by maximum likelihood (ML) is widely used in molecular systematics (<a reveal-id="msu300-B3" data-open="msu300-B3" class="link link-ref link-reveal xref-bibr">Felsenstein 1981</a>, <a reveal-id="msu300-B4" data-open="msu300-B4" class="link link-ref link-reveal xref-bibr">2004</a>).
 It involves the estimation of substitution model parameters, branch 
lengths and tree topology. These parameters are usually estimated one 
after another with the tree topology being the main parameter of 
interest. While efficient numerical methods for estimating substitution 
model parameters and branch lengths on a fixed tree exist (<a reveal-id="msu300-B29" data-open="msu300-B29" class="link link-ref link-reveal xref-bibr">Yang 2000</a>), finding the optimal tree topology is an NP-hard combinatorial optimization problem (<a reveal-id="msu300-B1" data-open="msu300-B1" class="link link-ref link-reveal xref-bibr">Chor and Tuller 2005</a>). Therefore, one has to rely on search heuristics to find the “best” tree.</p><p>ML
 tree searches apply inter alia local tree rearrangements such as 
nearest neighbor interchange (NNI), subtree pruning and regrafting 
(SPR), or tree bisection and reconnection (TBR) to improve the current 
tree (<a reveal-id="msu300-B9" data-open="msu300-B9" class="link link-ref link-reveal xref-bibr">Guindon and Gascuel 2003</a>; <a reveal-id="msu300-B22" data-open="msu300-B22" class="link link-ref link-reveal xref-bibr">Swofford 2003</a>; <a reveal-id="msu300-B21" data-open="msu300-B21" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>).
 Here, only modifications that increase the tree likelihood (“uphill” 
moves) are allowed. Such approaches are prone to be stuck in local 
optima (e.g., <a reveal-id="msu300-B23" data-open="msu300-B23" class="link link-ref link-reveal xref-bibr">Swofford and Olsen 1990</a>).
 The problem becomes more severe if the local tree rearrangement method 
can only generate a small number of trees in neighborhood of the current
 tree. As a result, SPR algorithms often find trees with higher 
likelihoods than those that are based on NNI (<a reveal-id="msu300-B17" data-open="msu300-B17" class="link link-ref link-reveal xref-bibr">Morrison 2007</a>; <a reveal-id="msu300-B27" data-open="msu300-B27" class="link link-ref link-reveal xref-bibr">Whelan and Money 2010</a>; <a reveal-id="msu300-B16" data-open="msu300-B16" class="link link-ref link-reveal xref-bibr">Money and Whelan 2012</a>). TBR is not often used due to its high computational demand.</p><p>Stochastic
 algorithms were developed to overcome the problem of local optima 
encountered by hill-climbing algorithms. Current ML implementations of 
stochastic algorithms allow “downhill” moves (<a reveal-id="msu300-B19" data-open="msu300-B19" class="link link-ref link-reveal xref-bibr">Salter and Pearl 2001</a>; <a reveal-id="msu300-B25" data-open="msu300-B25" class="link link-ref link-reveal xref-bibr">Vos 2003</a>; <a reveal-id="msu300-B24" data-open="msu300-B24" class="link link-ref link-reveal xref-bibr">Vinh and von Haeseler 2004</a>) or maintain a population of candidate trees (<a reveal-id="msu300-B13" data-open="msu300-B13" class="link link-ref link-reveal xref-bibr">Lewis 1998</a>; <a reveal-id="msu300-B30" data-open="msu300-B30" class="link link-ref link-reveal xref-bibr">Zwickl 2006</a>; <a reveal-id="msu300-B10" data-open="msu300-B10" class="link link-ref link-reveal xref-bibr">Helaers and Milinkovitch 2010</a>)
 to avoid local optima. However, in terms of both likelihood 
maximization and computation time such implementations have been found 
not to perform as well as SPR-based hill-climbing algorithms (<a reveal-id="msu300-B21" data-open="msu300-B21" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>; <a reveal-id="msu300-B17" data-open="msu300-B17" class="link link-ref link-reveal xref-bibr">Morrison 2007</a>).
 The large variety of techniques makes it difficult to combine them into
 effective and efficient stochastic algorithms. While the possibilities 
to enhance a hill-climbing algorithm are limited, the potential to 
improve the effectiveness and efficiency of stochastic algorithms is not
 yet fully explored.</p>                    <h2 scrollto-destination="74402244" id="74402244" class="section-title">New Approaches</h2>
<p>This article presents a fast and effective stochastic algorithm for 
finding ML trees. The core idea is to perform an efficient sampling of 
local optima in the tree space. Here, the best local optimum found 
represents the reported ML tree. To this end, we combine elements of 
hill-climbing algorithms, random perturbation of current best trees, and
 a broad sampling of initial starting trees. Comparative analyses for 
many large DNA and amino acid (AA) multiple sequence alignments 
retrieved from TreeBASE (<a reveal-id="msu300-B20" data-open="msu300-B20" class="link link-ref link-reveal xref-bibr">Sanderson et al. 1994</a>) showed that our new search strategy often achieves higher likelihoods compared with RAxML (<a reveal-id="msu300-B21" data-open="msu300-B21" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>) and PhyML (<a reveal-id="msu300-B8" data-open="msu300-B8" class="link link-ref link-reveal xref-bibr">Guindon et al. 2010</a>).</p>                    <h2 scrollto-destination="74402246" id="74402246" class="section-title">Results</h2>
                    <h3 scrollto-destination="74402247" id="74402247" class="section-title">Benchmark Setup</h3>
<p>Here, we compared the performance of our approach (implemented in 
IQ-TREE 1.0) with the default tree searches implemented in PhyML 3.1 and
 RAxML 7.3.5. To that end, we downloaded multiple sequence alignments 
from TreeBASE (<a reveal-id="msu300-B20" data-open="msu300-B20" class="link link-ref link-reveal xref-bibr">Sanderson et al. 1994</a>;
 accessed December 1, 2012) fulfilling the following criteria. First, 
the number of sequences must be between 200 and 800 for DNA and between 
50 and 600 for AA alignments. Second, the alignment length must be at 
least four (or two) times the number of sequences in DNA (or AA) 
alignments. Third, the proportion of gaps/unknown characters must be 
less or equal than 70%. Identical sequences were discarded from the 
alignments keeping only one. We obtained 70 DNA and 45 AA alignments 
(see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary tables S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online). The DNA alignment lengths range from 976 to 61,199 sites. The AA alignment lengths were between 126 and 22,426 sites.</p><p>For all programs, we used GTR (general time reversible; <a reveal-id="msu300-B12" data-open="msu300-B12" class="link link-ref link-reveal xref-bibr">Lanave et al. 1984</a>) and WAG (<a reveal-id="msu300-B26" data-open="msu300-B26" class="link link-ref link-reveal xref-bibr">Whelan and Goldman 2001</a>) models for DNA and AA alignments, respectively. Rate heterogeneity followed the discrete Γ model (<a reveal-id="msu300-B28" data-open="msu300-B28" class="link link-ref link-reveal xref-bibr">Yang 1994</a>)
 with four rate categories, where relative rates are computed as the 
mean of the portion of the Γ distribution falling in the respective 
category. To avoid numerical discrepancies between different likelihood 
implementations, we used PhyML to compute the log-likelihoods of the 
final trees based on parameters produced by each program. We note that, 
for 92% of the trees the differences in log-likelihoods recomputed by 
IQ-TREE and PhyML are smaller than 0.01 and the maximal difference is 
0.05 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig S0</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online). All analyses were performed on the Vienna Scientific Cluster (VSC-2, vsc.ac.at).</p>                    <h3 scrollto-destination="74402250" id="74402250" class="section-title">Comparison with Equal Running Times</h3>
<p>As RAxML and PhyML are considered as the two high-performance ML 
tree-inference programs, we first benchmarked IQ-TREE by restricting the
 running time of IQ-TREE to that required by each RAxML and PhyML run. 
This is done to study how efficiently IQ-TREE uses its search time 
compared with the other programs. For each alignment, we ran RAxML ten 
times and PhyML once (because the default tree search in PhyML is 
deterministic). Subsequently, we ran IQ-TREE ten times for each 
alignment with restricted CPU time. Then, we compared for each alignment
 the average log-likelihood of trees produced by IQ-TREE with those by 
the other two programs.</p><p><a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">Figure 1</a> (and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig. S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online) displays the pairwise log-likelihood difference distributions for IQ-TREE versus RAxML (<a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>a</em> and <em>b</em>) and PhyML (<a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>c</em> and <em>d</em>).
 Trees inferred with IQ-TREE for DNA alignments had in 87.1% of the 
instances a higher likelihood than RAxML-trees or PhyML-trees (<a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>a</em> and <em>c</em>).
 Although these percentages are identical, the alignments for which 
IQ-TREE found better trees if compared with RAxML or PhyML are not the 
same (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig. S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online). For 12.9% of the alignments, RAxML or PhyML found better trees. </p>                        <a id="74402253" scrollto-destination="74402253"></a>
<div id="msu300-F1" data-id="msu300-F1" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_msu300f1p.jpeg" alt="Performance of IQ-TREE for fixed CPU times: (a, b) Display frequencies of log-likelihood differences for IQ-TREE minus RAxML for 70 DNA (a) and 45 AA (b) alignments. (c) and (d) show the same if IQ-TREE is compared with PhyML. IQ-TREE’s CPU times were limited to those required by RAxML and PhyML, respectively. The percentages on the dashed line in (b) and (d) represent the fraction of alignments where log-likelihood differences are smaller than 0.01." path-from-xml="msu300f1p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74402253/msu300f1p.jpeg" path-from-xml="msu300f1p.jpeg" target="_blank">Open in new tab</a><a section="74402253" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300f1p.jpeg?Expires=1577219882&amp;Signature=EhNy0c1nAj5ogP55%7ETPkXjS3BSv1ZTx4PxOdeSsD9NO1D4XegnqhagqAikyMNSgTMdDVUPrIUYGvwulIcd1JeGpFZ7ti1rIfzEX2OEcrO-c8mGvnMgKXUGMZ%7EFaJeAie1sdOvVFbXVG3rGfUipfvEDcAbm7Y0jaKm-ZvyXTCzbUJapJ21250d%7EqdyGZ-r9x4mamuVwr6LMQdI4wdgrp%7Err-tlwkETiArSJZx2d8qxAxi0ZYffb0FUC-wsSZgao2n%7Ef2oJGNQjZdpLUvh1QhIeU40HFRhIqlm03IB3aZT1FM9P096Hv00wkh9zUTCKucEVbvIQ8DQpkf%7E6te7F5jXhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74402253&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu300f1p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of IQ-TREE for fixed CPU times: (<em>a</em>, <em>b</em>) Display frequencies of log-likelihood differences for IQ-TREE minus RAxML for 70 DNA (<em>a</em>) and 45 AA (<em>b</em>) alignments. (<em>c</em>) and (<em>d</em>)
 show the same if IQ-TREE is compared with PhyML. IQ-TREE’s CPU times 
were limited to those required by RAxML and PhyML, respectively. The 
percentages on the dashed line in (<em>b</em>) and (<em>d</em>) represent the fraction of alignments where log-likelihood differences are smaller than 0.01.</p></div></div><div content-id="msu300-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_msu300f1p.jpeg" alt="Performance of IQ-TREE for fixed CPU times: (a, b) Display frequencies of log-likelihood differences for IQ-TREE minus RAxML for 70 DNA (a) and 45 AA (b) alignments. (c) and (d) show the same if IQ-TREE is compared with PhyML. IQ-TREE’s CPU times were limited to those required by RAxML and PhyML, respectively. The percentages on the dashed line in (b) and (d) represent the fraction of alignments where log-likelihood differences are smaller than 0.01." path-from-xml="msu300f1p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74402253/msu300f1p.jpeg" path-from-xml="msu300f1p.jpeg" target="_blank">Open in new tab</a><a section="74402253" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300f1p.jpeg?Expires=1577219882&amp;Signature=EhNy0c1nAj5ogP55%7ETPkXjS3BSv1ZTx4PxOdeSsD9NO1D4XegnqhagqAikyMNSgTMdDVUPrIUYGvwulIcd1JeGpFZ7ti1rIfzEX2OEcrO-c8mGvnMgKXUGMZ%7EFaJeAie1sdOvVFbXVG3rGfUipfvEDcAbm7Y0jaKm-ZvyXTCzbUJapJ21250d%7EqdyGZ-r9x4mamuVwr6LMQdI4wdgrp%7Err-tlwkETiArSJZx2d8qxAxi0ZYffb0FUC-wsSZgao2n%7Ef2oJGNQjZdpLUvh1QhIeU40HFRhIqlm03IB3aZT1FM9P096Hv00wkh9zUTCKucEVbvIQ8DQpkf%7E6te7F5jXhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74402253&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu300f1p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of IQ-TREE for fixed CPU times: (<em>a</em>, <em>b</em>) Display frequencies of log-likelihood differences for IQ-TREE minus RAxML for 70 DNA (<em>a</em>) and 45 AA (<em>b</em>) alignments. (<em>c</em>) and (<em>d</em>)
 show the same if IQ-TREE is compared with PhyML. IQ-TREE’s CPU times 
were limited to those required by RAxML and PhyML, respectively. The 
percentages on the dashed line in (<em>b</em>) and (<em>d</em>) represent the fraction of alignments where log-likelihood differences are smaller than 0.01.</p></div></div><p>For the AA alignments, IQ-TREE found higher likelihoods in 62.2% if compared with RAxML (<a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>b</em>) and in 66.7% if compared with PhyML (<a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>d</em>).
 Contrary to DNA we observed 22.2% of the alignments where RAxML and 
IQ-TREE found trees with negligible log-likelihood differences 
(&lt;0.01). This number is 13.3% when comparing PhyML with IQ-TREE. In 
only 15.6% and 20% of the AA alignments, RAxML and PhyML performed 
better (with respect to tree log-likelihoods) than IQ-TREE, 
respectively.</p><p>We note that the distributions in <a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">figure 1</a><em>a</em>, <em>c</em>, and <em>d</em> are skewed to the right. Thus, our tree search strategy sometimes leads to substantially better likelihoods.</p><p>In
 summary, based on the analysis of a large collection of alignments, we 
demonstrate that IQ-TREE shows higher likelihoods in approximately 
three-quarters of the analyzed data. The improvement is almost the same 
compared with RAxML or PhyML. Because we fixed IQ-TREE’s running time to
 the time the other programs needed, we conclude that the employed 
search strategy explores tree-space more efficiently.</p>                    <h3 scrollto-destination="74402257" id="74402257" class="section-title">Comparison with Different Running Time</h3>
<p>We now discuss the performance of IQ-TREE if the CPU time is not 
determined by RAxML or PhyML but rather by the default stopping rule 
(see Materials and Methods). Thus, we compared the differences in CPU 
times and the differences in log-likelihoods (<a reveal-id="msu300-F2" data-open="msu300-F2" class="link link-reveal link-table xref-fig">fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig. S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online). Again, the analyses were based on average of ten independent IQ-TREE and RAxML runs. <a reveal-id="msu300-F2" data-open="msu300-F2" class="link link-reveal link-table xref-fig">Figure 2</a> is organized like <a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">figure 1</a>
 (RAxML vs. IQ-TREE results in the first row, PhyML vs. IQ-TREE results 
in the second row, DNA alignments left column, and AA alignments right 
column). </p>                        <a id="74402259" scrollto-destination="74402259"></a>
<div id="msu300-F2" data-id="msu300-F2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_msu300f2p.jpeg" alt="Performance of IQ-TREE for variable CPU times: The upper plots (a, b) show the performance of IQ-TREE against RAxML using the 70 DNA (a) and 45 AA (b) alignments. The lower plots (c, d) show the same against PhyML. Each dot in the main diagrams represents for one alignment the mean differences of the CPU times (y axis) and of the mean differences of log-likelihoods (x axis) of the reconstructed trees by the programs compared. The whiskers at each point show the standard errors of the differences. The histograms at the top and the side present the marginal frequencies. Dots to the right of the vertical dashed line represent alignments where IQ-TREE found a higher likelihood. If a dot is below the horizontal dashed line, the reconstruction by IQ-TREE was faster. Percentages in the quadrants of histograms denote the fraction of alignments in that region. Percentages on the dashed line reflect the number of alignments where log-likelihood differences are smaller than 0.01 (see [b] and [d])." path-from-xml="msu300f2p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74402259/msu300f2p.jpeg" path-from-xml="msu300f2p.jpeg" target="_blank">Open in new tab</a><a section="74402259" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300f2p.jpeg?Expires=1577219882&amp;Signature=WXW6KdpePaO-E9HsfxaY0FkAgD3-4uQxeU65dz-28kTvyJEXXdLXaP9l8GevSa6hUhb3sisbmloomSw3yKTOEleoBvcFPvitldEwWcMv3X%7E8AFo0dN3SS3z3x662r6047xjwuNXKd1a5JZuyhlxOrp9piNE2O6NVCU-Qk9WwMCXrI2l-Ewz-e1-i17-CCHosG-IPPuIdpMLxVZrFu5XYenAI1Wt9NxCF6TY-3i0kgaVs0JmEpdEYWcuSwlCmV5BUJOGdLLdlFDaRhXfv6ixJHyzuB2AiwzhlCsN5TqpDvqKGw2nP9vFyfDt5-ik7EmEZGHUwyfTpqxZ5KElMtJbDRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74402259&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu300f2p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of IQ-TREE for variable CPU times: The upper plots (<em>a</em>, <em>b</em>) show the performance of IQ-TREE against RAxML using the 70 DNA (<em>a</em>) and 45 AA (<em>b</em>) alignments. The lower plots (<em>c</em>, <em>d</em>)
 show the same against PhyML. Each dot in the main diagrams represents 
for one alignment the mean differences of the CPU times (<em>y</em> axis) and of the mean differences of log-likelihoods (<em>x</em>
 axis) of the reconstructed trees by the programs compared. The whiskers
 at each point show the standard errors of the differences. The 
histograms at the top and the side present the marginal frequencies. 
Dots to the right of the vertical dashed line represent alignments where
 IQ-TREE found a higher likelihood. If a dot is below the horizontal 
dashed line, the reconstruction by IQ-TREE was faster. Percentages in 
the quadrants of histograms denote the fraction of alignments in that 
region. Percentages on the dashed line reflect the number of alignments 
where log-likelihood differences are smaller than 0.01 (see [<em>b</em>] and [<em>d</em>]).</p></div></div><div content-id="msu300-F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_msu300f2p.jpeg" alt="Performance of IQ-TREE for variable CPU times: The upper plots (a, b) show the performance of IQ-TREE against RAxML using the 70 DNA (a) and 45 AA (b) alignments. The lower plots (c, d) show the same against PhyML. Each dot in the main diagrams represents for one alignment the mean differences of the CPU times (y axis) and of the mean differences of log-likelihoods (x axis) of the reconstructed trees by the programs compared. The whiskers at each point show the standard errors of the differences. The histograms at the top and the side present the marginal frequencies. Dots to the right of the vertical dashed line represent alignments where IQ-TREE found a higher likelihood. If a dot is below the horizontal dashed line, the reconstruction by IQ-TREE was faster. Percentages in the quadrants of histograms denote the fraction of alignments in that region. Percentages on the dashed line reflect the number of alignments where log-likelihood differences are smaller than 0.01 (see [b] and [d])." path-from-xml="msu300f2p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74402259/msu300f2p.jpeg" path-from-xml="msu300f2p.jpeg" target="_blank">Open in new tab</a><a section="74402259" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300f2p.jpeg?Expires=1577219882&amp;Signature=WXW6KdpePaO-E9HsfxaY0FkAgD3-4uQxeU65dz-28kTvyJEXXdLXaP9l8GevSa6hUhb3sisbmloomSw3yKTOEleoBvcFPvitldEwWcMv3X%7E8AFo0dN3SS3z3x662r6047xjwuNXKd1a5JZuyhlxOrp9piNE2O6NVCU-Qk9WwMCXrI2l-Ewz-e1-i17-CCHosG-IPPuIdpMLxVZrFu5XYenAI1Wt9NxCF6TY-3i0kgaVs0JmEpdEYWcuSwlCmV5BUJOGdLLdlFDaRhXfv6ixJHyzuB2AiwzhlCsN5TqpDvqKGw2nP9vFyfDt5-ik7EmEZGHUwyfTpqxZ5KElMtJbDRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74402259&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu300f2p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of IQ-TREE for variable CPU times: The upper plots (<em>a</em>, <em>b</em>) show the performance of IQ-TREE against RAxML using the 70 DNA (<em>a</em>) and 45 AA (<em>b</em>) alignments. The lower plots (<em>c</em>, <em>d</em>)
 show the same against PhyML. Each dot in the main diagrams represents 
for one alignment the mean differences of the CPU times (<em>y</em> axis) and of the mean differences of log-likelihoods (<em>x</em>
 axis) of the reconstructed trees by the programs compared. The whiskers
 at each point show the standard errors of the differences. The 
histograms at the top and the side present the marginal frequencies. 
Dots to the right of the vertical dashed line represent alignments where
 IQ-TREE found a higher likelihood. If a dot is below the horizontal 
dashed line, the reconstruction by IQ-TREE was faster. Percentages in 
the quadrants of histograms denote the fraction of alignments in that 
region. Percentages on the dashed line reflect the number of alignments 
where log-likelihood differences are smaller than 0.01 (see [<em>b</em>] and [<em>d</em>]).</p></div></div><p>By
 allowing variable CPU time, the number of the alignments for which 
IQ-TREE found higher-likelihood trees than RAxML or PhyML increases. For
 97.1% of the DNA alignments, the likelihood is improved compared with 
RAxML (<a reveal-id="msu300-F2" data-open="msu300-F2" class="link link-reveal link-table xref-fig">fig. 2</a><em>a</em>)
 with the maximal average log-likelihood difference of 109.5 (TreeBASE 
ID: M7964). For two DNA alignments, IQ-TREE obtained trees with lower 
likelihoods than RAxML with log-likelihood differences up to −8.9 
(M2534).</p><p>This success in finding higher likelihoods comes at a 
cost; IQ-TREE required longer CPU times than RAxML for 75.7% of the DNA 
alignments. However, the situation is complicated; the differences in 
average CPU times are highly variable, and for some alignments, one 
program is much faster than the other. For example, for the alignment 
M7024 IQ-TREE needed 4.2 h more than RAxML to finish, whereas RAxML 
required 8.3 h more to find an optimal tree for M14582. To finish all 
ten repetitions for the 70 DNA alignments, IQ-TREE needed 2,020 CPU 
hours (∼87 CPU days), whereas RAxML needed 1,870 CPU hours (∼78 CPU 
days). This is an average CPU time difference of less than 13 min per 
run.</p><p><a reveal-id="msu300-F2" data-open="msu300-F2" class="link link-reveal link-table xref-fig">Figure 2</a><em>b</em> displays the results for the 45 AA alignments. For ten AA alignments (22.2%, cf. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig. S2b</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>
 online), IQ-TREE and RAxML inferred trees with likelihood differences 
smaller than 0.01 for all ten runs. For 73.3% of the AA alignments, 
IQ-TREE obtained higher likelihoods than RAxML with a maximal 
log-likelihood difference of 21.9 (M11012). And for only 4.4% of the AA 
alignments, the results were in favor of RAxML, with a maximum 
log-likelihood difference of −8.9 (M3114). In terms of computing time, 
IQ-TREE obtained the result faster than RAxML in 57.8% of the AA 
alignments, whereas RAxML was faster in 42.2%. In total, IQ-TREE needed 
2,042 CPU hours to complete all 450 runs, whereas RAxML required 2,380 
CPU hours. This is an excess of 16.6% compared with the CPU time of 
IQ-TREE. Thus, on average RAxML needed 45 CPU minutes more per run. The 
runtime ratios between IQ-TREE and RAxML range from 0.6 to 3.2 for DNA 
and from 0.5 to 1.9 for protein alignments.</p><p>Finally, <a reveal-id="msu300-F2" data-open="msu300-F2" class="link link-reveal link-table xref-fig">figure 2</a><em>c</em> and <em>d</em>
 displays the results of IQ-TREE and PhyML for the DNA and AA 
alignments, respectively. IQ-TREE obtained higher likelihoods than PhyML
 for 91.4% of the DNA and 77.8% of the AA alignments. PhyML obtained 
higher likelihoods in 8.6% and 2.2% for DNA and AA, respectively. 
Notably, the maximal log-likelihood differences in favor of IQ-TREE were
 280.5 (M4794) and 621.1 (M8630) for DNA and AA, respectively. The 
maximal differences in favor of PhyML were −6.3 (M9143) and −0.27 
(M8175) for DNA and AA, respectively.</p><p>With respect to computing 
times, PhyML was faster in 47.1% of the DNA and for all AA alignments, 
whereas IQ-TREE was faster in 52.9% of the DNA alignments. PhyML spent 
357 and 61 CPU hours for all DNA and AA alignments, respectively, 
compared with the average CPU times for IQ-TREE of 202 and 204 h for all
 DNA and AA alignments, respectively. However, in the shorter runtime 
PhyML produced lower likelihoods for 77.8% of the AA alignments. The 
runtime ratios between IQ-TREE and PhyML range from 0.3 to 2.5 (DNA) and
 from 2 to 7.5 (protein).</p><p>In addition, we ran PhyML ten times per alignment using a random starting tree and SPR search strategy. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary figure S3</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>
 online, shows the results. In terms of computing time, PhyML ran slower
 than IQ-TREE for 98.6% DNA alignments but faster than IQ-TREE for 100% 
AA alignments. With respect to log-likelihoods, IQ-TREE produced higher 
likelihoods than PhyML for 88.6% DNA and 93.3% AA alignments (an 
increase by 2.8% for PhyML on DNA, but a decrease by 15.5% on AA). 
Hence, IQ-TREE performed better than PhyML under both the default and 
random starting tree options.</p>                    <h2 scrollto-destination="74402266" id="74402266" class="section-title">Discussion</h2>
<p>We have combined well-known phylogenetic and combinatorial 
optimization techniques into a fast and effective tree search algorithm.
 The success of IQ-TREE results from two factors: The new tree search 
strategy helps to escape local optima and, thus, leads to trees with 
high likelihood and the phylogenetic likelihood library (<a reveal-id="msu300-B6" data-open="msu300-B6" class="link link-ref link-reveal xref-bibr">Flouri et al. 2014</a>)
 reduces the time for the likelihood computation. Given the same amount 
of computing time, the efficient IQ-TREE implementation of hill-climbing
 and stochastic NNI operations (see Materials and Methods) computed 
trees with higher likelihood than RAxML or PhyML in the majority of 
cases (up to 87.1% of the benchmark data). This improvement is further 
boosted if the internal stopping rule was used (up to 97.1%). The 
success of IQ-TREE in finding trees with higher likelihoods is somehow 
at odds with the discussion in the literature about the effectiveness of
 NNI compared with SPR (<a reveal-id="msu300-B11" data-open="msu300-B11" class="link link-ref link-reveal xref-bibr">Hordijk and Gascuel 2005</a>; <a reveal-id="msu300-B8" data-open="msu300-B8" class="link link-ref link-reveal xref-bibr">Guindon et al. 2010</a>; <a reveal-id="msu300-B27" data-open="msu300-B27" class="link link-ref link-reveal xref-bibr">Whelan and Money 2010</a>).
 One explanation for the very good performance of IQ-TREE is possibly 
the introduction of the stochastic NNI. This random perturbation of 
locally optimal trees helps to escape local optima. The perturbed trees 
are then optimized by hill-climbing NNI, thus allowing for the 
possibility to find new and higher local optima. The combination of 
random and deterministic elements for ML searches has been previously 
proposed in <a reveal-id="msu300-B25" data-open="msu300-B25" class="link link-ref link-reveal xref-bibr">Vos (2003)</a> and <a reveal-id="msu300-B24" data-open="msu300-B24" class="link link-ref link-reveal xref-bibr">Vinh and von Haeseler (2004)</a>.</p><p>In addition, we employed some elements of evolution strategies (<a reveal-id="msu300-B18" data-open="msu300-B18" class="link link-ref link-reveal xref-bibr">Rechenberg 1973</a>)
 to allow for a broader exploration of the tree space. To this end, we 
maintain a small population (candidate tree set) of locally optimal 
trees initially generated from a large number of maximum parsimony (MP) 
trees. Throughout the ML tree search, we continuously update the 
candidate tree set with better trees. This extension of the tree-search 
also contributes to the performance of IQ-TREE.</p><p>It would be 
interesting to evaluate the relative importance of all these heuristic 
optimization techniques implemented in IQ-TREE, but this is beyond the 
scope of this article and will be discussed in forthcoming technical 
publications.</p><p>We also observed that the improvement in 
log-likelihood differences is positively skewed for all comparative 
analyses with the exception of the time constrained IQ-TREE versus RAxML
 for AA data (<a reveal-id="msu300-F1" data-open="msu300-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>b</em>).
 Thus, not only the average log-likelihood improved with IQ-TREE but 
also for some alignments, the improvement is substantial. It would be 
interesting to find out the characteristics of such alignments to 
further improve ML tree reconstruction methods. Moreover, by using many 
large alignments we show that the very good performance of our search 
strategy is not limited to a few alignments. Based on our benchmark 
results, we are confident that IQ-TREE will generally work very well.</p><p>We
 would like to point out that it is not enough to run phylogenetic 
programs with a stochastic component only once. RAxML and IQ-TREE showed
 some variation in the log-likelihoods, if they were run several times 
(here ten times) on the same alignment (<a reveal-id="msu300-F2" data-open="msu300-F2" class="link link-reveal link-table xref-fig">fig. 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig. S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577219882&amp;Signature=yVbRduEMlNg5gkJBBTFfzWb3pjXKqLrSHDoARMFrlKVMgHGywBsu8aI8AkDgXtnVVxERkU1MMpqJmtBpC169dgv2J0R894-1QSZKvEAEhbxpoa9qh5OHoCXhKkJAvsF24UO9EzxvvKOesg70tP6oJe5EiEOIS-hVzGQZi9sBTDfqnfq8-JC%7EMlNNOayw7EF7gJOzKnk%7E59KGrg177-VPrnYYR5ysbW%7EIJMKVvTx2tpjugsXb-XEUVUvCek60lw78LxSX9U23S%7E7fjKX-Z-YV5LBSAPPwXCOdHT0g3NQJPJn3MfUUIKJ5TDe2VbtaN6XdbztDb5ed4mwc%7E56CZgebPw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>
 online). This observation implies that both programs still finish 
sometimes in local optima and one should rerun the programs as many 
times as possible. In addition, we also offer the possibility to run 
IQ-TREE longer by adjusting the corresponding parameter of the stopping 
rule or by applying the statistical stopping rule suggested by <a reveal-id="msu300-B24" data-open="msu300-B24" class="link link-ref link-reveal xref-bibr">Vinh and von Haeseler (2004)</a>.
 Compared with other programs, our results show that with the standard 
setting of the stopping rule IQ-TREE can produce very good results, with
 a moderate increase in running time.</p><p>The proposed tree search in 
IQ-TREE also improves the accuracy of the recently introduced ultrafast 
bootstrap approximation approach (<a reveal-id="msu300-B14" data-open="msu300-B14" class="link link-ref link-reveal xref-bibr">Minh et al. 2013</a>).
 To further facilitate large phylogenetic analyses, we also consider 
future development of IQ-TREE for distributed computing platforms. The 
highly independent components of our stochastic search algorithm would 
allow us to implement an efficient parallelization strategy (cf. <a reveal-id="msu300-B15" data-open="msu300-B15" class="link link-ref link-reveal xref-bibr">Minh et al. 2005</a>) with near-optimal speedup. Thus, the running time of very large phylogenetic analyses would then be greatly reduced.</p><p>In
 conclusion, IQ-TREE is a time and search efficient ML-tree 
reconstruction program. It complements the collection of available 
ML-programs and shows a better performance with respect to the ML search
 than RAxML or PhyML. However, as IQ-TREE is not always better than the 
other programs, we recommend using all three programs.</p>                    <h2 scrollto-destination="74402274" id="74402274" class="section-title">Materials and Methods</h2>
<p>In the following, we describe the ingredients of the fast tree 
reconstruction method that are implemented in IQ-TREE. We used the 
phylogenetic likelihood library (<a reveal-id="msu300-B6" data-open="msu300-B6" class="link link-ref link-reveal xref-bibr">Flouri et al. 2014</a>)
 for likelihood and parsimony computations. We first describe our fast 
hill-climbing NNI algorithm that is repeatedly used throughout the tree 
search. Subsequently, we will explain the initial tree generation and 
the stochastic NNI process.</p>                    <h3 scrollto-destination="74402276" id="74402276" class="section-title">Hill-Climbing NNI</h3>
<p>For the determination of locally optimal trees, we implemented a fast
 hill-climbing NNI search. Our approach is based on the work of <a reveal-id="msu300-B9" data-open="msu300-B9" class="link link-ref link-reveal xref-bibr">Guindon and Gascuel (2003)</a> where they applied several NNIs simultaneously. The hill-climbing NNI is a central element of the search strategy (<a reveal-id="msu300-F3" data-open="msu300-F3" class="link link-reveal link-table xref-fig">fig. 3</a> box c). </p>                        <a id="74402278" scrollto-destination="74402278"></a>
<div id="msu300-F3" data-id="msu300-F3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_msu300f3p.jpeg" alt="Flowchart for the stochastic search algorithm. The variable count counts the number of random perturbations (box b and box c) as a new best tree was found." path-from-xml="msu300f3p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74402278/msu300f3p.jpeg" path-from-xml="msu300f3p.jpeg" target="_blank">Open in new tab</a><a section="74402278" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300f3p.jpeg?Expires=1577219882&amp;Signature=x6j3yTX-HF3UX%7E-k3Q8vkqUtUZo10rlQr0lp2FO5SoTKdVf43-b9iWzjrUYIVC2pG-MxHz57UFf%7EeRLEr7UdZy-QLWnPCkFf4CMDDFcjwMa2n2b57qInzUNvbrRDjaN%7EkZfijjyY9t6GjhPJ1PyhMkE1lf%7E8pDAAnnTlWcvB%7ENT3inz8CHk3k4o9n3nPg-RfNglfptYS3ZD1g4Fq4pwl20dq0OAo1V7WXbcZGi9MDkCDOz9myer%7EL%7EUJsOiK8HSw9WH18vOKYaQoVPcr80j-AVEoVPv4F2afIxkg6Xi4jgDT%7EPtKWAfIs38DGW7nX4yVeXql3Uzf5sbI4qKodZ4O4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74402278&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu300f3p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart for the stochastic search algorithm. The variable <em>count</em> counts the number of random perturbations (box b and box c) as a new best tree was found.</p></div></div><div content-id="msu300-F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_msu300f3p.jpeg" alt="Flowchart for the stochastic search algorithm. The variable count counts the number of random perturbations (box b and box c) as a new best tree was found." path-from-xml="msu300f3p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74402278/msu300f3p.jpeg" path-from-xml="msu300f3p.jpeg" target="_blank">Open in new tab</a><a section="74402278" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300f3p.jpeg?Expires=1577219882&amp;Signature=x6j3yTX-HF3UX%7E-k3Q8vkqUtUZo10rlQr0lp2FO5SoTKdVf43-b9iWzjrUYIVC2pG-MxHz57UFf%7EeRLEr7UdZy-QLWnPCkFf4CMDDFcjwMa2n2b57qInzUNvbrRDjaN%7EkZfijjyY9t6GjhPJ1PyhMkE1lf%7E8pDAAnnTlWcvB%7ENT3inz8CHk3k4o9n3nPg-RfNglfptYS3ZD1g4Fq4pwl20dq0OAo1V7WXbcZGi9MDkCDOz9myer%7EL%7EUJsOiK8HSw9WH18vOKYaQoVPcr80j-AVEoVPv4F2afIxkg6Xi4jgDT%7EPtKWAfIs38DGW7nX4yVeXql3Uzf5sbI4qKodZ4O4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74402278&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu300f3p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart for the stochastic search algorithm. The variable <em>count</em> counts the number of random perturbations (box b and box c) as a new best tree was found.</p></div></div><p>NNI
 is a local tree rearrangement operation that swaps two subtrees across 
an internal branch. Each inner branch defines two distinct NNIs. Thus, 
for an unrooted bifurcating tree with <em>n</em> taxa, there are 2(<em>n</em>-3) NNI-trees in the NNI-neighborhood of that tree.</p><p>For
 a given tree (current tree), we first compute the approximate 
likelihoods of each NNI-tree by optimizing the respective inner branch 
and the four adjacent branches. In the following, we only consider NNIs 
that increase the tree likelihood compared with the current tree. We 
then create a list of nonconflicting NNIs. Two NNIs are considered 
conflicting if they operate on the same inner branch or adjacent 
branches. The list is initialized with the best NNI. We then add the 
next best NNI to the list if it does not conflict with any existing NNI 
in the list otherwise we discard it. We repeat this procedure until all 
NNIs have been processed.</p><p>Afterwards, we simultaneously apply all 
NNIs in the list to the current tree and compute the likelihood of the 
resulting tree by doing one tree traversal of ML branch length 
optimization. If the likelihood of the resulting tree is worse than the 
likelihood of the best NNI-tree, we discard all topological 
modifications except that of the best NNI in the list. Thus, if the list
 is not empty, a new tree with higher likelihood will be found. This 
tree will replace the current tree. Furthermore, we tag the inner 
branches on the new current tree on which NNIs were applied and perform 
the remaining procedure as follows.</p><p>Instead of computing the full 
NNI-neighborhood we conduct a reduced NNI search on the new current 
tree, applying the following heuristic to find a locally optimal tree. 
We only compute the NNI-trees on inner branches that are at most two 
branches away from the tagged branches. For such admissible branches, we
 compute the list as described. If the list is empty, a locally optimal 
tree has been found and the hill-climbing search is finished. Otherwise,
 we continue the reduced NNI search with the better tree as described 
above.</p>                    <h3 scrollto-destination="74402283" id="74402283" class="section-title">Initial Tree Generation</h3>
<p>Tree search heuristics typically start with a quickly built initial 
tree that is subsequently improved. For example, PhyML starts with a 
BIONJ tree (<a reveal-id="msu300-B7" data-open="msu300-B7" class="link link-ref link-reveal xref-bibr">Gascuel 1997</a>) whereas RAxML starts with a stepwise addition tree (<a reveal-id="msu300-B2" data-open="msu300-B2" class="link link-ref link-reveal xref-bibr">Farris 1970</a>) using MP (<a reveal-id="msu300-B5" data-open="msu300-B5" class="link link-ref link-reveal xref-bibr">Fitch 1971</a>) where the MP tree is further optimized with lazy subtree rearrangement (<a reveal-id="msu300-B21" data-open="msu300-B21" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>).
 To get a representative sample of plausible initial trees, we generate 
100 parsimony trees using the same strategy as RAxML. From the 100 
trees, we collect all unique topologies and compute their approximate 
likelihoods by doing one tree traversal of ML branch length 
optimization. From the ranked list of ML values, we select the top 20 
trees and perform hill-climbing NNI on each tree to obtain the locally 
optimal ML trees. We then retain the top five topologies with highest 
likelihood in the so-called candidate tree set ℂ for further 
optimization (<a reveal-id="msu300-F3" data-open="msu300-F3" class="link link-reveal link-table xref-fig">fig. 3</a> box a).</p>                    <h3 scrollto-destination="74402285" id="74402285" class="section-title">A Stochastic NNI Step</h3>
<p>The locally optimal trees in the candidate set ℂ are randomly 
perturbed to allow the escape from local optima. To this end, we 
introduce a so-called stochastic NNI step (<a reveal-id="msu300-F3" data-open="msu300-F3" class="link link-reveal link-table xref-fig">fig. 3</a> box b). Here, we perform 0.5(<em>n</em> − 3) random NNIs on a tree <em>T</em> randomly drawn from ℂ, where <em>n</em>
 − 3 is the number of inner branches. Then, we apply hill-climbing NNI 
to the perturbed tree to obtain a new locally optimal tree <em>T</em>* (<a reveal-id="msu300-F3" data-open="msu300-F3" class="link link-reveal link-table xref-fig">fig. 3</a> box c).</p><p>If <em>T</em>* has a higher likelihood than the best tree in ℂ, we replace that tree by <em>T</em>*. Moreover, the stochastic NNI successfully found a better tree, thus the number (<em>count</em>) of perturbations after a new better tree was found is set to zero. If <em>T</em>*’s likelihood is higher than the likelihood for the worst tree in ℂ, then that tree is replaced by <em>T</em>*. Finally, ℂ does not change if the likelihood of <em>T</em>*
 is smaller than the smallest likelihood for the trees in ℂ. In the last
 two cases, the tree with the highest likelihood did not change and <em>count</em> is increased by one.</p><p>The tree search stops, if the current best tree has not changed for <em>count</em> = 100 random perturbations. The flowchart of our stochastic tree search is summarized in <a reveal-id="msu300-F3" data-open="msu300-F3" class="link link-reveal link-table xref-fig">figure 3</a>.</p>                    <h2 scrollto-destination="74402289" id="74402289" class="backacknowledgements-title">Acknowledgments</h2>
<p>The authors thank Tomas Flouri, Fernando Izquierdo-Carrasco, and 
Alexandros Stamatakis for help with the phylogenetic likelihood library,
 Pablo A. Goloboff and an anonymous reviewer for helpful comments on the
 manuscript. The computational results presented have been achieved 
using the Vienna Scientific Cluster (VSC-2). This work was supported by 
the Austrian Science Fund—FWF (grant number I760-B17) to B.Q.M. and 
A.v.H., and the University of Vienna (Initiativkolleg I059-N) to L.-T.N.
 and A.v.H.</p>                    <h2 scrollto-destination="74402291" id="74402291" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="msu300-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chor</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Tuller</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">Maximum likelihood of evolutionary trees is hard</div>,&nbsp;<div class="source">Lect Notes Comput Sci.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">3500</div>&nbsp;(pg.&nbsp;<div class="fpage">296</div>-<div class="lpage">310</div>)<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Maximum%20likelihood%20of%20evolutionary%20trees%20is%20hard&amp;aulast=Chor&amp;date=2005&amp;spage=296&amp;epage=310&amp;volume=3500" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Maximum%20likelihood%20of%20evolutionary%20trees%20is%20hard&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Farris</div> <div class="given-names">JS</div></div>.&nbsp;</div><div class="article-title">Methods for computing Wagner trees</div>,&nbsp;<div class="source">Syst Zool.</div>&nbsp;,&nbsp;<div class="year">1970</div>,&nbsp;vol.&nbsp;<div class="volume">19</div>&nbsp;(pg.&nbsp;<div class="fpage">83</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Methods%20for%20computing%20Wagner%20trees&amp;author=JS%20Farris&amp;publication_year=1970&amp;journal=Syst%20Zool.&amp;volume=19&amp;pages=83-92" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2307/2412028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2412028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F2412028"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Methods%20for%20computing%20Wagner%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach</div>,&nbsp;<div class="source">J Mol Evol.</div>&nbsp;,&nbsp;<div class="year">1981</div>,&nbsp;vol.&nbsp;<div class="volume">17</div>&nbsp;(pg.&nbsp;<div class="fpage">368</div>-<div class="lpage">376</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;author=J%20Felsenstein&amp;publication_year=1981&amp;journal=J%20Mol%20Evol.&amp;volume=17&amp;pages=368-376" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF01734359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01734359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF01734359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7288891" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div>,&nbsp;<div class="source">Inferring phylogenies</div>&nbsp;,&nbsp;<div class="year">2004</div><div class="publisher-loc">Sunderland (MA)</div><div class="publisher-name">Sinauer Associates</div><div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Inferring%20phylogenies&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Inferring%20phylogenies&amp;aulast=Felsenstein&amp;date=2004" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inferring%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Inferring%20phylogenies">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fitch</div> <div class="given-names">WM</div></div>.&nbsp;</div><div class="article-title">Toward defining course of evolution—minimum change for a specific tree topology</div>,&nbsp;<div class="source">Syst Zool.</div>&nbsp;,&nbsp;<div class="year">1971</div>,&nbsp;vol.&nbsp;<div class="volume">20</div>&nbsp;(pg.&nbsp;<div class="fpage">406</div>-<div class="lpage">416</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Toward%20defining%20course%20of%20evolution%E2%80%94minimum%20change%20for%20a%20specific%20tree%20topology&amp;author=WM%20Fitch&amp;publication_year=1971&amp;journal=Syst%20Zool.&amp;volume=20&amp;pages=406-416" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2307/2412116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2412116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F2412116"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Toward%20defining%20course%20of%20evolution%E2%80%94minimum%20change%20for%20a%20specific%20tree%20topology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flouri</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Izquierdo-Carrasco</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Darriba</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Aberer</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Nguyen</div> <div class="given-names">L-T</div></div>,&nbsp; <div class="name"><div class="surname">Minh</div> <div class="given-names">BQ</div></div>,&nbsp; <div class="name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">The phylogenetic likelihood library</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2014</div>&nbsp;<div class="comment">Advance Access published October 30, 2014, doi:10.1093/sysbio/syu084</div><div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=The%20phylogenetic%20likelihood%20library&amp;aulast=Flouri&amp;date=2014" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20phylogenetic%20likelihood%20library&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">685</div>-<div class="lpage">695</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;author=O%20Gascuel&amp;publication_year=1997&amp;journal=Mol%20Biol%20Evol.&amp;volume=14&amp;pages=685-695" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a025808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254330" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guindon</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Dufayard</div> <div class="given-names">JF</div></div>,&nbsp; <div class="name"><div class="surname">Lefort</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Anisimova</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Hordijk</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">59</div>&nbsp;(pg.&nbsp;<div class="fpage">307</div>-<div class="lpage">321</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S%20Guindon&amp;author=JF%20Dufayard&amp;author=V%20Lefort&amp;author=M%20Anisimova&amp;author=W%20Hordijk&amp;author=O%20Gascuel&amp;publication_year=2010&amp;journal=Syst%20Biol.&amp;volume=59&amp;pages=307-321" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syq010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guindon</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">52</div>&nbsp;(pg.&nbsp;<div class="fpage">696</div>-<div class="lpage">704</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=S%20Guindon&amp;author=O%20Gascuel&amp;publication_year=2003&amp;journal=Syst%20Biol.&amp;volume=52&amp;pages=696-704" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150390235520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Helaers</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Milinkovitch</div> <div class="given-names">MC</div></div>.&nbsp;</div><div class="article-title">MetaPIGA
 v2.0: maximum likelihood large phylogeny estimation using the 
metapopulation genetic algorithm and other stochastic heuristics</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;pg.&nbsp;<div class="fpage">379</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MetaPIGA%20v2.0%3A%20maximum%20likelihood%20large%20phylogeny%20estimation%20using%20the%20metapopulation%20genetic%20algorithm%20and%20other%20stochastic%20heuristics&amp;author=R%20Helaers&amp;author=MC%20Milinkovitch&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=379" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-11-379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-11-379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20633263" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MetaPIGA%20v2.0%3A%20maximum%20likelihood%20large%20phylogeny%20estimation%20using%20the%20metapopulation%20genetic%20algorithm%20and%20other%20stochastic%20heuristics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hordijk</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">Improving the efficiency of SPR moves in phylogenetic tree search methods based on maximum likelihood</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">4338</div>-<div class="lpage">4347</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20efficiency%20of%20SPR%20moves%20in%20phylogenetic%20tree%20search%20methods%20based%20on%20maximum%20likelihood&amp;author=W%20Hordijk&amp;author=O%20Gascuel&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=4338-4347" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bti713" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti713" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbti713"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16234323" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Improving%20the%20efficiency%20of%20SPR%20moves%20in%20phylogenetic%20tree%20search%20methods%20based%20on%20maximum%20likelihood&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lanave</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Preparata</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Saccone</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Serio</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">A new method for calculating evolutionary substitution rates</div>,&nbsp;<div class="source">J Mol Evol.</div>&nbsp;,&nbsp;<div class="year">1984</div>,&nbsp;vol.&nbsp;<div class="volume">20</div>&nbsp;(pg.&nbsp;<div class="fpage">86</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20new%20method%20for%20calculating%20evolutionary%20substitution%20rates&amp;author=C%20Lanave&amp;author=G%20Preparata&amp;author=C%20Saccone&amp;author=G%20Serio&amp;publication_year=1984&amp;journal=J%20Mol%20Evol.&amp;volume=20&amp;pages=86-93" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF02101990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02101990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02101990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6429346" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20new%20method%20for%20calculating%20evolutionary%20substitution%20rates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lewis</div> <div class="given-names">PO</div></div>.&nbsp;</div><div class="article-title">A genetic algorithm for maximum-likelihood phylogeny inference using nucleotide sequence data</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">15</div>&nbsp;(pg.&nbsp;<div class="fpage">277</div>-<div class="lpage">283</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20algorithm%20for%20maximum-likelihood%20phylogeny%20inference%20using%20nucleotide%20sequence%20data&amp;author=PO%20Lewis&amp;publication_year=1998&amp;journal=Mol%20Biol%20Evol.&amp;volume=15&amp;pages=277-283" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025924" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025924" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a025924"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9501494" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20genetic%20algorithm%20for%20maximum-likelihood%20phylogeny%20inference%20using%20nucleotide%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minh</div> <div class="given-names">BQ</div></div>,&nbsp; <div class="name"><div class="surname">Nguyen</div> <div class="given-names">MA</div></div>,&nbsp; <div class="name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Ultrafast approximation for phylogenetic bootstrap</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">30</div>&nbsp;(pg.&nbsp;<div class="fpage">1188</div>-<div class="lpage">1195</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20approximation%20for%20phylogenetic%20bootstrap&amp;author=BQ%20Minh&amp;author=MA%20Nguyen&amp;author=A%20von%20Haeseler&amp;publication_year=2013&amp;journal=Mol%20Biol%20Evol.&amp;volume=30&amp;pages=1188-1195" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/mst024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmst024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23418397" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ultrafast%20approximation%20for%20phylogenetic%20bootstrap&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minh</div> <div class="given-names">BQ</div></div>,&nbsp; <div class="name"><div class="surname">Vinh</div> <div class="given-names">LS</div></div>,&nbsp; <div class="name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Schmidt</div> <div class="given-names">HA</div></div>.&nbsp;</div><div class="article-title">pIQPNNI: parallel reconstruction of large maximum likelihood phylogenies</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">3794</div>-<div class="lpage">3796</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=pIQPNNI%3A%20parallel%20reconstruction%20of%20large%20maximum%20likelihood%20phylogenies&amp;author=BQ%20Minh&amp;author=LS%20Vinh&amp;author=A%20von%20Haeseler&amp;author=HA%20Schmidt&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3794-3796" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bti594" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbti594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16046495" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:pIQPNNI%3A%20parallel%20reconstruction%20of%20large%20maximum%20likelihood%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Money</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Whelan</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Characterizing the phylogenetic tree-search problem</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">61</div>&nbsp;(pg.&nbsp;<div class="fpage">228</div>-<div class="lpage">239</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Characterizing%20the%20phylogenetic%20tree-search%20problem&amp;author=D%20Money&amp;author=S%20Whelan&amp;publication_year=2012&amp;journal=Syst%20Biol.&amp;volume=61&amp;pages=228-239" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syr097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyr097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyr097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22076302" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Characterizing%20the%20phylogenetic%20tree-search%20problem&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morrison</div> <div class="given-names">DA</div></div>.&nbsp;</div><div class="article-title">Increasing
 the efficiency of searches for the maximum likelihood tree in a 
phylogenetic analysis of up to 150 nucleotide sequences</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">56</div>&nbsp;(pg.&nbsp;<div class="fpage">988</div>-<div class="lpage">1010</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Increasing%20the%20efficiency%20of%20searches%20for%20the%20maximum%20likelihood%20tree%20in%20a%20phylogenetic%20analysis%20of%20up%20to%20150%20nucleotide%20sequences&amp;author=DA%20Morrison&amp;publication_year=2007&amp;journal=Syst%20Biol.&amp;volume=56&amp;pages=988-1010" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150701779808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150701779808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150701779808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18066931" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Increasing%20the%20efficiency%20of%20searches%20for%20the%20maximum%20likelihood%20tree%20in%20a%20phylogenetic%20analysis%20of%20up%20to%20150%20nucleotide%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rechenberg</div> <div class="given-names">I</div></div>.&nbsp;</div>,&nbsp;<div class="source">Evolutionsstrategie—optimierung technischer systeme nach prinzipien der biologischen evolution</div>&nbsp;,&nbsp;<div class="year">1973</div><div class="publisher-loc">Stuttgart (Germany)</div><div class="publisher-name">Frommann-Holzboog Verlag</div><div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Evolutionsstrategie%E2%80%94optimierung%20technischer%20systeme%20nach%20prinzipien%20der%20biologischen%20evolution&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Evolutionsstrategie%E2%80%94optimierung%20technischer%20systeme%20nach%20prinzipien%20der%20biologischen%20evolution&amp;aulast=Rechenberg&amp;date=1973" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evolutionsstrategie%E2%80%94optimierung%20technischer%20systeme%20nach%20prinzipien%20der%20biologischen%20evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Evolutionsstrategie%E2%80%94optimierung%20technischer%20systeme%20nach%20prinzipien%20der%20biologischen%20evolution">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salter</div> <div class="given-names">LA</div></div>,&nbsp; <div class="name"><div class="surname">Pearl</div> <div class="given-names">DK</div></div>.&nbsp;</div><div class="article-title">Stochastic search strategy for estimation of maximum likelihood phylogenetic trees</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2001</div>,&nbsp;vol.&nbsp;<div class="volume">50</div>&nbsp;(pg.&nbsp;<div class="fpage">7</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Stochastic%20search%20strategy%20for%20estimation%20of%20maximum%20likelihood%20phylogenetic%20trees&amp;author=LA%20Salter&amp;author=DK%20Pearl&amp;publication_year=2001&amp;journal=Syst%20Biol.&amp;volume=50&amp;pages=7-17" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/106351501750107413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F106351501750107413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F106351501750107413"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12116596" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Stochastic%20search%20strategy%20for%20estimation%20of%20maximum%20likelihood%20phylogenetic%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sanderson</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">Donoghue</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">Piel</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Eriksson</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">TreeBASE: a prototype database of phylogenetic analyses and an interactive tool for browsing the phylogeny of life</div>,&nbsp;<div class="source">Am J Bot.</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">81</div>&nbsp;pg.&nbsp;<div class="fpage">183</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=TreeBASE%3A%20a%20prototype%20database%20of%20phylogenetic%20analyses%20and%20an%20interactive%20tool%20for%20browsing%20the%20phylogeny%20of%20life&amp;author=MJ%20Sanderson&amp;author=MJ%20Donoghue&amp;author=W%20Piel&amp;author=T%20Eriksson&amp;publication_year=1994&amp;journal=Am%20J%20Bot.&amp;volume=81&amp;pages=183" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2307/2445299" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2445299" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F2445299"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:TreeBASE%3A%20a%20prototype%20database%20of%20phylogenetic%20analyses%20and%20an%20interactive%20tool%20for%20browsing%20the%20phylogeny%20of%20life&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">2688</div>-<div class="lpage">2690</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A%20Stamatakis&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2688-2690" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swofford</div> <div class="given-names">DL</div></div>.&nbsp;</div><div class="article-title">PAUP*. Phylogenetic analysis using parsimony (* and other methods)</div>,&nbsp;<div class="year">2003</div>&nbsp;<div class="comment">Version 4. Sunderland (MA): Sinauer Associates</div><div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=&amp;aulast=Swofford&amp;date=2003" href="#"></a></span></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swofford</div> <div class="given-names">DL</div></div>,&nbsp; <div class="name"><div class="surname">Olsen</div> <div class="given-names">GJ</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Hillis</div> <div class="given-names">DM</div></div>,&nbsp; <div class="name"><div class="surname">Moritz</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">Phylogeny reconstruction</div>,&nbsp;<div class="source">Molecular systematics</div>&nbsp;,&nbsp;<div class="year">1990</div><div class="publisher-loc">Sinauer Associates</div><div class="publisher-name">Sunderland (MA)</div>(pg.&nbsp;<div class="fpage">411</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Molecular%20systematics&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Molecular%20systematics&amp;aulast=Swofford&amp;date=1990&amp;spage=411&amp;epage=501" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20systematics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Molecular%20systematics">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vinh</div> <div class="given-names">LS</div></div>,&nbsp; <div class="name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">IQPNNI: moving fast through tree space and stopping in time</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">1565</div>-<div class="lpage">1571</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=IQPNNI%3A%20moving%20fast%20through%20tree%20space%20and%20stopping%20in%20time&amp;author=LS%20Vinh&amp;author=A%20von%20Haeseler&amp;publication_year=2004&amp;journal=Mol%20Biol%20Evol.&amp;volume=21&amp;pages=1565-1571" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msh176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsh176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15163768" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:IQPNNI%3A%20moving%20fast%20through%20tree%20space%20and%20stopping%20in%20time&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">RA</div></div>.&nbsp;</div><div class="article-title">Accelerated likelihood surface exploration: the likelihood ratchet</div>,&nbsp;<div class="source">Syst Biol.</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">52</div>&nbsp;(pg.&nbsp;<div class="fpage">368</div>-<div class="lpage">373</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accelerated%20likelihood%20surface%20exploration%3A%20the%20likelihood%20ratchet&amp;author=RA%20Vos&amp;publication_year=2003&amp;journal=Syst%20Biol.&amp;volume=52&amp;pages=368-373" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12775525" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Accelerated%20likelihood%20surface%20exploration%3A%20the%20likelihood%20ratchet&amp;aulast=Vos&amp;date=2003&amp;spage=368&amp;epage=373&amp;volume=52" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accelerated%20likelihood%20surface%20exploration%3A%20the%20likelihood%20ratchet&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whelan</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Goldman</div> <div class="given-names">N</div></div>.&nbsp;</div><div class="article-title">A general empirical model of protein evolution derived from multiple protein families using a maximum-likelihood approach</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2001</div>,&nbsp;vol.&nbsp;<div class="volume">18</div>&nbsp;(pg.&nbsp;<div class="fpage">691</div>-<div class="lpage">699</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20general%20empirical%20model%20of%20protein%20evolution%20derived%20from%20multiple%20protein%20families%20using%20a%20maximum-likelihood%20approach&amp;author=S%20Whelan&amp;author=N%20Goldman&amp;publication_year=2001&amp;journal=Mol%20Biol%20Evol.&amp;volume=18&amp;pages=691-699" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a003851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11319253" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20general%20empirical%20model%20of%20protein%20evolution%20derived%20from%20multiple%20protein%20families%20using%20a%20maximum-likelihood%20approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whelan</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Money</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">The prevalence of multifurcations in tree-space and their implications for tree-search</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">2674</div>-<div class="lpage">2677</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20prevalence%20of%20multifurcations%20in%20tree-space%20and%20their%20implications%20for%20tree-search&amp;author=S%20Whelan&amp;author=D%20Money&amp;publication_year=2010&amp;journal=Mol%20Biol%20Evol.&amp;volume=27&amp;pages=2674-2677" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msq163" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq163" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsq163"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20584772" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20prevalence%20of%20multifurcations%20in%20tree-space%20and%20their%20implications%20for%20tree-search&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z</div></div>.&nbsp;</div><div class="article-title">Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods</div>,&nbsp;<div class="source">J Mol Evol.</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">39</div>&nbsp;(pg.&nbsp;<div class="fpage">306</div>-<div class="lpage">314</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20phylogenetic%20estimation%20from%20DNA%20sequences%20with%20variable%20rates%20over%20sites%3A%20approximate%20methods&amp;author=Z%20Yang&amp;publication_year=1994&amp;journal=J%20Mol%20Evol.&amp;volume=39&amp;pages=306-314" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF00160154" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00160154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00160154"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7932792" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Maximum%20likelihood%20phylogenetic%20estimation%20from%20DNA%20sequences%20with%20variable%20rates%20over%20sites%3A%20approximate%20methods&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z</div></div>.&nbsp;</div><div class="article-title">Maximum likelihood estimation on large phylogenies and analysis of adaptive evolution in human influenza virus A</div>,&nbsp;<div class="source">J Mol Evol.</div>&nbsp;,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">51</div>&nbsp;(pg.&nbsp;<div class="fpage">423</div>-<div class="lpage">432</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20estimation%20on%20large%20phylogenies%20and%20analysis%20of%20adaptive%20evolution%20in%20human%20influenza%20virus%20A&amp;author=Z%20Yang&amp;publication_year=2000&amp;journal=J%20Mol%20Evol.&amp;volume=51&amp;pages=423-432" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s002390010105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs002390010105"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11080365" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Maximum%20likelihood%20estimation%20on%20large%20phylogenies%20and%20analysis%20of%20adaptive%20evolution%20in%20human%20influenza%20virus%20A&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu300-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu300-B30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zwickl</div> <div class="given-names">DJ</div></div>.&nbsp;</div><div class="article-title">Genetic
 algorithm approaches for the phylogenetic analysis of large biological 
sequence data sets under the maximum likelihood criterion</div>,&nbsp;<div class="year">2006</div>&nbsp;<div class="comment"> [Ph.D. dissertation]. Austin (TX): The University of Texas at Austin</div></div></div></div></div></div>        <!-- /foreach -->



        

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="FN1"><div class="fn"><div class="footnote-content"><strong>Associate editor:</strong> Barbara Holland</div></div></div><div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution Non-Commercial License 
(http://creativecommons.org/licenses/by-nc/4.0/), which permits 
non-commercial re-use, distribution, and reproduction in any medium, 
provided the original work is properly cited. For commercial re-use, 
please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2925592&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5325">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/32/1/10.1093_molbev_msu300/1/msu300_Supplementary_Data.zip?Expires=1577272928&amp;Signature=PzGyVmYjdTmCFH-F4RbUIsdpWlHhML1CzCDjNsM2ygjLAf7rU3NkZyU4RVBLbE2VvgX3ZpQQxL7m7%7E03VFVm6EsLn7zDAkqZ3QsKgApLxRvJDzDuK5pWP8s%7E4ri7-7E%7EhajqYTCW5wC4caJLWrHnq8REgwXLopq%7EDBWg2o4sNbbaSj5zDmBUgfCFXi7qAVPrI-h5kbiMnJXC7z0UXycPaOGngOMeqM4T26t3gsyCnmp0zzJljosFJdPzvNN2Zkq4iUdGReJYRS6i%7EQmu7O2JX-YMZsO%7Epu%7EB65Zimc-uyL96oAO9VcXEAbhdZpTU8Q%7E67KzzAe4KH4crMYawR8iVEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">13,505</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350050200025&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">1,931</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/molbev/msu300" data-uuid="0cf8e3c6-3689-21e1-d02c-1441a0453a92"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=2852032" style="display:inline-block;">
        <img alt="Article has an altmetric score of 21" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2925592"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350050200025&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fmbe%2farticle%2f32%2f1%2f268%2f2925592">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350050200025&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (1931)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F32%2F1%2F268%2F2925592" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/mbe/crossref-citedby/2925592">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz271/5626191?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Recurrent Collection of  Drosophila 
melanogaster  from Wild African Environments and Genomic Insights into 
Species History
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz234/5613640?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Dissecting the transcriptomic basis of phenotypic evolution in an aquatic keystone grazer
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz270/5625159?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Somatic mutations and genome stability maintenance in clonal coral colonies
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz241/5601619?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Living Trees: High-Quality Reproducible 
and Reusable Construction of Bacterial Phylogenetic Trees
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz268/5618730?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Developmental systems drift and the drivers of sex chromosome evolution
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="IQ-TREE: A Fast and Effective Stochastic Algorithm for Estimating Maximum-Likelihood Phylogenies | Molecular Biology and Evolution | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/molbev/msu300");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/mbe_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/mbe/pages/About">About Molecular Biology and Evolution</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/mbe/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/mbe/pages/Policies">Policies</a>
                                </li>
                                <li class="link-4">
                                    <a href="https://academic.oup.com/mbe/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://www.smbe.org/smbe/">Contact Us</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://twitter.com/OfficialSMBE">Twitter</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/mbe/subscribe">Purchase</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1537-1719</li>
                                            <li>Print ISSN 0737-4038</li>
                    <li>Copyright © 2019 Society for Molecular Biology and Evolution</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh166" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1574254654662&amp;tmr=load%3D1574254654626%26core%3D1574254654654%26main%3D1574254654658%26ifr%3D1574254654664&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F32%2F1%2F268%2F2925592&amp;href=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F32%2F1%2F268%2F2925592&amp;dt=IQ-TREE%3A%20A%20Fast%20and%20Effective%20Stochastic%20Algorithm%20for%20Estimating%20Maximum-Likelihood%20Phylogenies&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5dd5383e081d09f6&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3DIQ-TREE%253A%2520A%2520Fast%2520and%2520Effective%2520Stochastic%2520Algorithm%2520for%2520Estimating%2520Maximum-Likelihood%2520Phylogenies%26description%3DAbstract.%2520%2520Large%2520phylogenomics%2520data%2520sets%2520require%2520fast%2520tree%2520inference%2520methods%252C%2520especially%2520for%2520maximum-likelihood%2520%28ML%29%2520phylogenies.%2520Fast%2520programs%2520exist%252C%2520but%2520due%2520t%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fmbe%252Farticle%252F32%252F1%252F268%252F2925592%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fmbe%252FIssue%252F32%252F1%252F0%252Fm_mbe32_1.cover.gif%253FExpires%253D1637255441%2526Signature%253DKpMVQfbTV4w5LSS9zy%7EMpzaUuBLvehoxmA46PzIPbOrLpe5KqxuA-ibzV5cyLEuGZC-3%7EiDob5efQA38ML7ypFCkkLNUsCBZb%7EvFHZj-kHo1uL9TeD3PuWZ7oTdJchQCR%7EJqCunxxOGgqiOJiglfao-9Nkg8mMleZN5B4yKDIojyoHp1rSsJS1x1XhrTdSiX-S2wjSGxbOnKGc1w-E5gFXu3Ls7HgfhAgHj3WLRgnq3XylZ-1OTRIF9eYYCVLgmH%7EIR437y43FZXTdACu8iejn82KWxuM2rf7pD9fVHOkvEMXMcU5k8SSV-HxJSRpAAs3D7umGd6NuvYnZ3K-Y-eAw__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2014-11-03&amp;csi=undefined&amp;rev=v8.28.1-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f32%2f1%2f268%2f2925592">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>

    <script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="msu300.js"></script></body></html>