<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script>

        <title>RAxML-NG: a fast, scalable and user-friendly tool for 
maximum likelihood phylogenetic inference | Bioinformatics | Oxford 
Academic</title>

        <script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>




    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site2065275106.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'bioinformatics';
            App.SiteURL = 'academic.oup.com/bioinformatics';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/bioinformatics/article/35/21/4453/5487384","@type":"ScholarlyArticle","name":"RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference","datePublished":"2019-05-09","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/issue/35/21","@type":"PublicationIssue","issueNumber":"21","datePublished":"2019-11-01","isPartOf":{"@id":"https://academic.oup.com/bioinformatics","@type":"Periodical","name":"Bioinformatics","issn":["1460-2059"]}},"url":"http://dx.doi.org/10.1093/bioinformatics/btz305","inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/bioinformatics/article/35/21/4453/5487384","author":[{"name":"Kozlov, Alexey M","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person","sameAs":"http://orcid.org/0000-0001-7394-2718"},{"name":"Darriba, Diego","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person","sameAs":"http://orcid.org/0000-0001-5482-3932"},{"name":"Flouri, Tomáš","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person"},{"name":"Morel, Benoit","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person"},{"name":"Stamatakis, Alexandros","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany   Institute of Theoretical Informatics, Karlsruhe Institute of Technology , Karlsruhe, Germany","@type":"Person"}],"description":"AbstractMotivation.  Phylogenies are important for fundamental biological research, but also have numerous applications in biotechnology, agriculture and medici","pageStart":"4453","pageEnd":"4455","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/35/21/1/m_bioinfo_35_21cover.png?Expires=1636237385&Signature=V~1cgtIBht1v9aG9x6ARgS7VkgEhw-ZYH5bSCrZFo3lGs5z4EoCeulfT2sIPe7J2gSgSbL1MFOdQHt5Q9Uasr~zUTpXWzZoLlm~14yOjAgYhxwB8dCnpT6eKmjTPgdr7xNFuD87iBMPGBgs~o3An0aarYJsLRsGmYtXnq4ld6aN4LXL-pwIyT-6fWTLL3CY1BXtPlKW-xYzJSRjkQkrODfuy3lSDNuVRlVTMySo-D2xPchzyU727S-c17icFLodTEjsYWwS5IqRpQB9Ghnoci0HGwIHyJYyZYZ8ZC7NTOBQSQeN0rCsBxVhEICdQSz1Pt0a0hc5kdbX-u9QEarmvwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/35/21/1/m_bioinfo_35_21cover.png?Expires=1636237385&Signature=V~1cgtIBht1v9aG9x6ARgS7VkgEhw-ZYH5bSCrZFo3lGs5z4EoCeulfT2sIPe7J2gSgSbL1MFOdQHt5Q9Uasr~zUTpXWzZoLlm~14yOjAgYhxwB8dCnpT6eKmjTPgdr7xNFuD87iBMPGBgs~o3An0aarYJsLRsGmYtXnq4ld6aN4LXL-pwIyT-6fWTLL3CY1BXtPlKW-xYzJSRjkQkrODfuy3lSDNuVRlVTMySo-D2xPchzyU727S-c17icFLodTEjsYWwS5IqRpQB9Ghnoci0HGwIHyJYyZYZ8ZC7NTOBQSQeN0rCsBxVhEICdQSz1Pt0a0hc5kdbX-u9QEarmvwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference">
    <meta property="og:description" content="AbstractMotivation.  Phylogenies are important for fundamental biological research, but also have numerous applications in biotechnology, agriculture and medici">
    <meta property="og:url" content="https://academic.oup.com/bioinformatics/article/35/21/4453/5487384">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/35/21/1/m_bioinfo_35_21cover.png?Expires=1636237385&amp;Signature=V~1cgtIBht1v9aG9x6ARgS7VkgEhw-ZYH5bSCrZFo3lGs5z4EoCeulfT2sIPe7J2gSgSbL1MFOdQHt5Q9Uasr~zUTpXWzZoLlm~14yOjAgYhxwB8dCnpT6eKmjTPgdr7xNFuD87iBMPGBgs~o3An0aarYJsLRsGmYtXnq4ld6aN4LXL-pwIyT-6fWTLL3CY1BXtPlKW-xYzJSRjkQkrODfuy3lSDNuVRlVTMySo-D2xPchzyU727S-c17icFLodTEjsYWwS5IqRpQB9Ghnoci0HGwIHyJYyZYZ8ZC7NTOBQSQeN0rCsBxVhEICdQSz1Pt0a0hc5kdbX-u9QEarmvwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2019-05-09">
<meta name="citation_author" content="Kozlov, Alexey M"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Darriba, Diego"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Flouri, Tomáš"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Morel, Benoit"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Stamatakis, Alexandros"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author_institution" content="Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany"><meta name="citation_title" content="RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference"><meta name="citation_firstpage" content="4453"><meta name="citation_lastpage" content="4455"><meta name="citation_doi" content="10.1093/bioinformatics/btz305"><meta name="citation_journal_title" content="Bioinformatics"><meta name="citation_journal_abbrev" content="Bioinformatics"><meta name="citation_volume" content="35"><meta name="citation_issue" content="21"><meta name="citation_publication_date" content="2019/11/01"><meta name="citation_issn" content="1367-4803"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Using non-reversible context-dependent evolutionary models to study substitution patterns in primate non-coding sequences;  citation_journal_title=J. Mol. Evol;  citation_year=2010;  citation_volume=71;  citation_pages=34-50; "><meta name="citation_reference" content="citation_title=EPA-ng: massively parallel evolutionary placement of genetic sequences;  citation_journal_title=Syst. Biol;  citation_year=2018;  citation_volume=68;  citation_pages=365-369; "><meta name="citation_reference" content="citation_title=Two C++ libraries for counting trees on a phylogenetic terrace;  citation_journal_title=Bioinformatics;  citation_year=2017;  citation_volume=34;  citation_pages=3399-3401; "><meta name="citation_reference" content=" citation_title=Practical Methods of Optimization;  citation_year=1987; "><meta name="citation_reference" content="citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0;  citation_journal_title=Syst. Biol;  citation_year=2010;  citation_volume=59;  citation_pages=307-321; "><meta name="citation_reference" content="citation_publisher=Springer, Berlin Heidelberg;  citation_title=Algorithms in Bioinformatics, Volume 8701 of Lecture Notes in Computer Science;  citation_year=2014;  citation_pages=204-216; "><meta name="citation_reference" content="citation_title=Efficient detection of repeating sites to accelerate phylogenetic likelihood calculations;  citation_journal_title=Syst. Biol;  citation_year=2017;  citation_volume=66;  citation_pages=205-217; "><meta name="citation_reference" content="citation_title=ExaML version 3: a tool for phylogenomic analyses on supercomputers;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=2577-2579; "><meta name="citation_reference" content="citation_title=Empirical profile mixture models for phylogenetic reconstruction;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2317-2323; "><meta name="citation_reference" content="citation_title=Modeling protein evolution with several amino acid replacement matrices depending on site rates;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=2921-2936; "><meta name="citation_reference" content="citation_title=Renewing Felsensteinen phylogenetic bootstrap in the era of big data;  citation_journal_title=Nature;  citation_year=2018;  citation_volume=556;  citation_pages=452-456; "><meta name="citation_reference" content="citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=268-274; "><meta name="citation_reference" content="citation_title=FastTree 2 – approximately maximum-likelihood trees for large alignments;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=1-10; "><meta name="citation_reference" content="citation_title=Terraces in phylogenetic tree space;  citation_journal_title=Science;  citation_year=2011;  citation_volume=333;  citation_pages=448-450; "><meta name="citation_reference" content="Stamatakis A. ( 2006 ) Phylogenetic models of rate heterogeneity: a high performance computing perspective. In: Proceedings of IPDPS2006, HICOMB Workshop , Proceedings on CD , IEEE, Rhodos, Greece. "><meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1312-1313; "><meta name="citation_reference" content="citation_title=A space-time process model for the evolution of DNA sequences;  citation_journal_title=Genetics;  citation_year=1995;  citation_volume=139;  citation_pages=993-1005; "><meta name="citation_reference" content="citation_title=Evaluating fast maximum likelihood-based phylogenetic programs using empirical phylogenomic data sets;  citation_journal_title=Mol. Biol. Evol;  citation_year=2018;  citation_volume=35;  citation_pages=486-503; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/bioinformatics/article-pdf/35/21/4453/30330793/btz305.pdf"><meta name="description" content="AbstractMotivation.  Phylogenies are important for fundamental biological research, but also have numerous applications in biotechnology, agriculture and medici">    <script id="altmetric-embed-js" src="altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019111401.js" async=""></script><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2.css"></head><body data-sitestyletemplate="Journal" theme-bioinformatics="" class="off-canvas pg_Article pg_article" data-sitename="bioinformatics"><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/bioinformatics/article-xml/35/21/4453/5487384">



    <link rel="canonical" href="https://academic.oup.com/bioinformatics/article/35/21/4453/5487384">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "bioinformatics/article/35/21/4453/5487384");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    
    

    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=bioinformatics&authzrequired=false&doi=10.1093/bioinformatics/btz305';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5139" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f35%2f21%2f4453%2f5487384" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f35%2f21%2f4453%2f5487384" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387512">
            <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387513">
            <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387510">
            <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387514">
            <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387515">
            <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387516">
            <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387517">
            <a href="https://academic.oup.com/bioinformatics/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387518">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387511">
            <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387519">
            <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link ">About Bioinformatics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387520">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387521">
            <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387522">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387523">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387524">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/bioinformatics" class="journal-logo-container">
                    <img id="logo-Bioinformatics" class="journal-logo" src="bioinformatics_title-205396838.svg" alt="Bioinformatics">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.iscb.org/" target="" class="society-logo-container">
                                <img id="logo-InternationalSocietyforComputationalBiology" class="society-logo" src="bioinformatics_h11820233911.svg" alt="International Society for Computational Biology">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387512">
            <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387513">
            <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387510">
            <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387514">
            <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387515">
            <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387516">
            <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387517">
            <a href="https://academic.oup.com/bioinformatics/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387518">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387511">
            <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387519">
            <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link ">About Bioinformatics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387520">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387521">
            <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387522">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387523">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387524">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Bioinformatics</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5139" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=117130" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/bioinformatics" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="bioinformatics" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/bioinformatics/issue/35/21"><img id="issueImage" class="fb-featured-image" src="m_bioinfo_35_21cover.png" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/bioinformatics/issue/35/21">
                        <div class="volume trailing-comma">Volume 35</div>
                        <div class="issue">Issue 21</div>

            </a>

        </div>
        <div class="ii-pub-date">
1 November 2019        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="165818420">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818420">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="165818421">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818421">1 Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="165818424">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818424">2 New features and optimizations</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="165818439">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818439">3 Evaluation</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="165818441">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818441">4 Availability and user support</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="165818443">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818443">5 Future work</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="165818445">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818445">Acknowledgements</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="165818450">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#165818450">References</a>
            </div>
        </li>
<li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/bioinformatics/article/35/21/4451/5490857">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/bioinformatics/article/35/21/4456/5421160">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexey M Kozlov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. E-mail: <a href="mailto:alexey.kozlov@h-its.org" target="">alexey.kozlov@h-its.org</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0001-7394-2718">
                <img class="orchid-icon" alt="" src="mini-icon.png">&nbsp;&nbsp;http://orcid.org/0000-0001-7394-2718
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Alexey+M+Kozlov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kozlov%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kozlov%20Alexey%20M%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Alexey M Kozlov</a><span class="delimiter"><i class="icon-general-mail"></i>, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Diego Darriba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0001-5482-3932">
                <img class="orchid-icon" alt="" src="mini-icon.png">&nbsp;&nbsp;http://orcid.org/0000-0001-5482-3932
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Diego+Darriba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Darriba%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Darriba%20Diego%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Diego Darriba</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomáš Flouri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Tom%c3%a1%c5%a1+Flouri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Flouri%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Flouri%20Tom%C3%A1%C5%A1%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Tomáš Flouri</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Benoit Morel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Benoit+Morel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morel%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Morel%20Benoit%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Benoit Morel</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexandros Stamatakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div><div class="aff"><div class="insititution">Institute of Theoretical Informatics, Karlsruhe Institute of Technology</div>, Karlsruhe, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Alexandros+Stamatakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stamatakis%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stamatakis%20Alexandros%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Alexandros Stamatakis</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Bioinformatics</em>, Volume 35, Issue 21, 1 November 2019, Pages 4453–4455, <a href="https://doi.org/10.1093/bioinformatics/btz305">https://doi.org/10.1093/bioinformatics/btz305</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">09 May 2019</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">22 October 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Revision received:</div>
                    <div class="wi-date">16 April 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">24 April 2019</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">09 May 2019</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5487384" href="https://academic.oup.com/bioinformatics/article-pdf/35/21/4453/30330793/btz305.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Alexey M Kozlov, Diego Darriba, Tomáš Flouri, Benoit Morel, 
Alexandros Stamatakis,  RAxML-NG: a fast, scalable and user-friendly 
tool for maximum likelihood phylogenetic inference, <em>Bioinformatics</em>, Volume 35, Issue 21, 1 November 2019, Pages 4453–4455, <a href="https://doi.org/10.1093/bioinformatics/btz305">https://doi.org/10.1093/bioinformatics/btz305</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5487384&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1460-2059&amp;title=RAxML-NG%3A%20a%20fast%2C%20scalable%20and%20user-friendly%20tool%20for%20maximum%20likelihood%20phylogenetic%20inference&amp;publicationDate=2019-05-09&amp;volumeNum=35&amp;issueNum=21&amp;author=Kozlov%2C%20Alexey%20M%3B%20Darriba%2C%20Diego&amp;startPage=4453&amp;endPage=4455&amp;contentId=10.1093%2Fbioinformatics%2Fbtz305&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f35%2f21%2f4453%2f5487384" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Bioinformatics</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="bioinformatics" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="165818420" id="165818420" class="abstract-title">Abstract</h2>
<section class="abstract"><section class="sec"><div class="title">Motivation</div><p>Phylogenies
 are important for fundamental biological research, but also have 
numerous applications in biotechnology, agriculture and medicine. 
Finding the optimal tree under the popular maximum likelihood (ML) 
criterion is known to be NP-hard. Thus, highly optimized and scalable 
codes are needed to analyze constantly growing empirical datasets.</p></section><section class="sec"><div class="title">Results</div><p>We
 present RAxML-NG, a from-scratch re-implementation of the established 
greedy tree search algorithm of RAxML/ExaML. RAxML-NG offers improved 
accuracy, flexibility, speed, scalability, and usability compared with 
RAxML/ExaML. On taxon-rich datasets, RAxML-NG typically finds 
higher-scoring trees than IQTree, an increasingly popular recent tool 
for ML-based phylogenetic inference (although IQ-Tree shows better 
stability). Finally, RAxML-NG introduces several new features, such as 
the detection of terraces in tree space and the recently introduced 
transfer bootstrap support metric.</p></section><section class="sec"><div class="title">Availability and implementation</div><p>The code is available under GNU GPL at <span class="monospace"><a class="link link-uri" href="https://github.com/amkozlov/raxml-ng" target="">https://github.com/amkozlov/raxml-ng</a></span>. RAxML-NG web service (maintained by Vital-IT) is available at <span class="monospace"><a class="link link-uri" href="https://raxml-ng.vital-it.ch/" target="">https://raxml-ng.vital-it.ch/</a></span>.</p></section><section class="sec"><div class="title">Supplementary information</div><p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/35/21/10.1093_bioinformatics_btz305/1/btz305_supplementary_data.pdf?Expires=1576575697&amp;Signature=TciCuoomYP8pdzfo9bTTBeMSxPqHL0rgPwpL6Z9Thn19O6SpyzIbmRSP-rsZNZjlTZ%7EeZt2xUlTEhfPExIAFbgA682sOQx9APc6aRDsAk1o69AulniQ9-QwjjFxvTs2BsDREHomoUb3SM9azUlja0X%7E3ikvKilVpznLYl4NIpvwsuToDOkvgEvx50E3MRfz-OnuY1ywiiljAxoXYFi-ukkkDepuU1ws0BCx7mDv9llB0SLejj8Qn7UDAff0JFDL0DBu0fEfKsCjIFL0ditwm5B%7EdJtnqSPKJbrptJ0qE5dvTn059%7EHjDT1Sxe3XBgFZUp%7EKX6Fu7g3jx00Pz7Qmdlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>Bioinformatics</em> online.</p></section></section>                        <div class="article-metadata-panel clearfix"><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/bioinformatics/search-results?f_TocHeadingTitle=PHYLOGENETICS">PHYLOGENETICS</a>
                </div><div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Associate Editor:
                            <span class="al-author-name">
                                <span id="info-21532242" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jonathan Wren</div>
            <div class="info-card-role">
                Associate Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Jonathan+Wren">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wren%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wren%20Jonathan%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Jonathan Wren</a>
                            </span>
                        <br>
                    </div>
            </div>
        </div></div>
                    <h2 scrollto-destination="165818421" id="165818421" class="section-title">1 Introduction</h2>
<p>RAxML (<a reveal-id="btz305-B16" data-open="btz305-B16" class="link link-ref link-reveal xref-bibr">Stamatakis, 2014</a>)
 is a popular maximum likelihood (ML) tree inference tool which has been
 developed and supported by our group for the last 15 years. More 
recently, we also released ExaML (<a reveal-id="btz305-B8" data-open="btz305-B8" class="link link-ref link-reveal xref-bibr">Kozlov <em>et al.</em>, 2015</a>),
 a dedicated code for analyzing genome-scale datasets on supercomputers.
 ExaML implements the core tree search functionality of RAxML and scales
 to thousands of CPU cores. Other widely used ML inference tools are, 
for instance, IQ-Tree (<a reveal-id="btz305-B12" data-open="btz305-B12" class="link link-ref link-reveal xref-bibr">Nguyen <em>et al.</em>, 2015</a>), PhyML (<a reveal-id="btz305-B5" data-open="btz305-B5" class="link link-ref link-reveal xref-bibr">Guindon <em>et al.</em>, 2010</a>) and FastTree (<a reveal-id="btz305-B13" data-open="btz305-B13" class="link link-ref link-reveal xref-bibr">Price <em>et al.</em>, 2010</a>).</p><p>Here, we introduce our new code called RAxML-NG (RAxML Next Generation). It combines the strengths and concepts of RAxML <em>and</em> ExaML, and offers several additional improvements which we describe in the next section.</p>                    <h2 scrollto-destination="165818424" id="165818424" class="section-title">2 New features and optimizations</h2>
                    <h3 scrollto-destination="165818425" id="165818425" class="section-title">2.1 Evolutionary model extensions</h3>
<p>While RAxML/ExaML only fully supported the General Time Reversible 
(GTR) model of DNA substitution, RAxML-NG now supports all 22 
‘classical’ GTR-derived models. All model parameters (including branch 
lengths) can be either optimized or fixed to user-specified values. 
RAxML-NG also offers the following features: </p><ol class="number"><li><p>edge-proportional branch length estimation for multi-gene alignments,</p></li><li><p>FreeRate model of rate heterogeneity (<a reveal-id="btz305-B17" data-open="btz305-B17" class="link link-ref link-reveal xref-bibr">Yang, 1995</a>),</p></li><li><p>per-rate scalers in the Γ model of rate heterogeneity to prevent numerical underflow on large trees.</p></li></ol><p></p>                    <h3 scrollto-destination="165818427" id="165818427" class="section-title">2.2 Search algorithm modifications</h3>
<p>The subtree enumeration method used in RAxML/ExaML occasionally 
skipped promising topological moves; this has now been fixed in RAxML-NG
 (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/35/21/10.1093_bioinformatics_btz305/1/btz305_supplementary_data.pdf?Expires=1576575697&amp;Signature=TciCuoomYP8pdzfo9bTTBeMSxPqHL0rgPwpL6Z9Thn19O6SpyzIbmRSP-rsZNZjlTZ%7EeZt2xUlTEhfPExIAFbgA682sOQx9APc6aRDsAk1o69AulniQ9-QwjjFxvTs2BsDREHomoUb3SM9azUlja0X%7E3ikvKilVpznLYl4NIpvwsuToDOkvgEvx50E3MRfz-OnuY1ywiiljAxoXYFi-ukkkDepuU1ws0BCx7mDv9llB0SLejj8Qn7UDAff0JFDL0DBu0fEfKsCjIFL0ditwm5B%7EdJtnqSPKJbrptJ0qE5dvTn059%7EHjDT1Sxe3XBgFZUp%7EKX6Fu7g3jx00Pz7Qmdlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> for details). Further, RAxML-NG employs a two-step L-BFGS-B method (<a reveal-id="btz305-B4" data-open="btz305-B4" class="link link-ref link-reveal xref-bibr">Fletcher, 1987</a>) to optimize the parameters of the LG4X model (<a reveal-id="btz305-B10" data-open="btz305-B10" class="link link-ref link-reveal xref-bibr">Le <em>et al.</em>, 2012</a>).
 This approach (first introduced in IQ-Tree) is usually faster and more 
stable than the sequential optimization using Brent’s method in 
RAxML/ExaML.</p>                    <h3 scrollto-destination="165818429" id="165818429" class="section-title">2.3 Transfer bootstrap</h3>
<p>RAxML-NG can compute the novel branch support metric called transfer bootstrap expectation (TBE) recently proposed in (<a reveal-id="btz305-B11" data-open="btz305-B11" class="link link-ref link-reveal xref-bibr">Lemoine <em>et al.</em>, 2018</a>).
 When compared with the classic Felsenstein bootstrap, TBE is less 
sensitive to individual misplaced taxa in replicate trees, and thus 
better suited to reveal well-supported deep splits in large trees with 
thousands of taxa.</p>                    <h3 scrollto-destination="165818431" id="165818431" class="section-title">2.4 Phylogenetic terraces</h3>
<p>Certain patterns of missing data in multi-gene alignments can yield 
multiple tree topologies with identical likelihood scores—a phenomenon 
known as <em>terraces</em> in tree space (<a reveal-id="btz305-B14" data-open="btz305-B14" class="link link-ref link-reveal xref-bibr">Sanderson <em>et al.</em>, 2011</a>). RAxML-NG employs the recently released <em>terraphast</em> library (<a reveal-id="btz305-B3" data-open="btz305-B3" class="link link-ref link-reveal xref-bibr">Biczok <em>et al.</em>, 2017</a>) to assess if the inferred best-scoring ML tree resides on a terrace, and report the size of that terrace.</p>                    <h3 scrollto-destination="165818433" id="165818433" class="section-title">2.5 Performance and scalability</h3>
<p>In RAxML-NG, we further optimized the vectorized likelihood 
computation kernels and eliminated known sequential bottlenecks of 
RAxML. We also integrated an optimization technique for likelihood 
calculations known as <em>site repeats</em> (<a reveal-id="btz305-B7" data-open="btz305-B7" class="link link-ref link-reveal xref-bibr">Kobert <em>et al.</em>, 2017</a>)
 which yields runtime improvements of 10–60%. Finally, RAxML-NG 
implements several features for enhancing parallel efficiency, 
previously only available in ExaML: </p><ol class="number"><li><p>efficient fine-grained parallelization with MPI or MPI+pthreads,</p></li><li><p>binary input file format (compressed alignment),</p></li><li><p>restart from a checkpoint,</p></li><li><p>improved load balancing for multi-gene alignments (<a reveal-id="btz305-B6" data-open="btz305-B6" class="link link-ref link-reveal xref-bibr">Kobert <em>et al.</em>, 2014</a>)</p></li></ol><p></p>                    <h3 scrollto-destination="165818435" id="165818435" class="section-title">2.6 Usability</h3>
<p>Several RAxML-NG features aim to improve usability and avoid common 
pitfalls: auto-detection of CPU instruction set and number of cores, 
recommendation for the optimal number of threads, automatic restart from
 the last checkpoint after program interruption, search progress 
reporting in the log file etc.</p>                    <h3 scrollto-destination="165818437" id="165818437" class="section-title">2.7 Modularization</h3>
<p>RAxML and ExaML are large monolithic codes. This hindered 
maintenance, extension and code reuse. In RAxML-NG, we encapsulated the 
phylogenetic likelihood kernels and numerical optimization routines in 
two libraries: <em>libpll</em> (<a class="link link-uri" href="https://github.com/xflouris/libpll-2" target="">https://github.com/xflouris/libpll-2</a>) and <em>pll-modules</em> (<a class="link link-uri" href="https://github.com/ddarriba/pll-modules" target="">https://github.com/ddarriba/pll-modules</a>),
 respectively. Both libraries include unit tests and are also being used
 by other software tools developed in our lab such as ModelTest-NG and 
EPA-NG (<a reveal-id="btz305-B2" data-open="btz305-B2" class="link link-ref link-reveal xref-bibr">Barbera <em>et al.</em>, 2018</a>). This yields our likelihood computation code more error-proof than in RAxML/ExaML.</p>                    <h2 scrollto-destination="165818439" id="165818439" class="section-title">3 Evaluation</h2>
<p>A recent evaluation of fast ML-based methods (<a reveal-id="btz305-B18" data-open="btz305-B18" class="link link-ref link-reveal xref-bibr">Zhou <em>et al.</em>, 2018</a>)
 showed that IQTree yields the best tree inference accuracy, closely 
followed by RAxML/ExaML. Thus, we benchmarked RAxML-NG against these 
three programs on the collection of empirical datasets used by Zhou <em>et al.</em>
 RAxML-NG found the best-scoring tree for the highest number of datasets
 (19/21) among all programs tested, while being 1.3× to 4.5× faster. 
Furthermore, it scales to the large number of cores with a parallel 
efficiency of up to 125% (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/35/21/10.1093_bioinformatics_btz305/1/btz305_supplementary_data.pdf?Expires=1576575697&amp;Signature=TciCuoomYP8pdzfo9bTTBeMSxPqHL0rgPwpL6Z9Thn19O6SpyzIbmRSP-rsZNZjlTZ%7EeZt2xUlTEhfPExIAFbgA682sOQx9APc6aRDsAk1o69AulniQ9-QwjjFxvTs2BsDREHomoUb3SM9azUlja0X%7E3ikvKilVpznLYl4NIpvwsuToDOkvgEvx50E3MRfz-OnuY1ywiiljAxoXYFi-ukkkDepuU1ws0BCx7mDv9llB0SLejj8Qn7UDAff0JFDL0DBu0fEfKsCjIFL0ditwm5B%7EdJtnqSPKJbrptJ0qE5dvTn059%7EHjDT1Sxe3XBgFZUp%7EKX6Fu7g3jx00Pz7Qmdlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>
 for details). In summary, RAxML-NG is clearly superior to RAxML/ExaML, 
and thus we recommend that the users of these codes upgrade as soon as 
possible. Comparison to IQTree yielded mixed results: although RAxML-NG 
is generally faster and returns higher-scoring trees on taxon-rich 
alignments, IQTree results show much lower variance. Hence, on 
alignments with strong phylogenetic signal, IQTree may require fewer 
replicate searches than RAxML-NG to find the best-scoring tree.</p>                    <h2 scrollto-destination="165818441" id="165818441" class="section-title">4 Availability and user support</h2>
<p>The RAxML-NG source code as well as pre-compiled binaries for Linux and MacOS are available at <a class="link link-uri" href="https://github.com/amkozlov/raxml-ng" target="">https://github.com/amkozlov/raxml-ng</a>. RAxML-NG is also available as a web service (maintained by the Vital-IT unit of the Swiss Institute of Bioinformatics) at <a class="link link-uri" href="https://raxml-ng.vital-it.ch/" target="">https://raxml-ng.vital-it.ch/</a>. An up-to-date user manual is available at <a class="link link-uri" href="https://github.com/amkozlov/raxml-ng/wiki" target="">https://github.com/amkozlov/raxml-ng/wiki</a>. User support is provided via the RAxML Google group at: <a class="link link-uri" href="https://groups.google.com/forum/#%21forum/raxml" target="">https://groups.google.com/forum/#!forum/raxml</a>.</p>                    <h2 scrollto-destination="165818443" id="165818443" class="section-title">5 Future work</h2>
<p>In future versions of RAxML-NG, we plan to add site heterogeneity models such as RAxML-CAT (<a reveal-id="btz305-B15" data-open="btz305-B15" class="link link-ref link-reveal xref-bibr">Stamatakis, 2006</a>) and PhyloBayes-CAT (<a reveal-id="btz305-B9" data-open="btz305-B9" class="link link-ref link-reveal xref-bibr">Le <em>et al.</em>, 2008</a>), as well as non-reversible context-dependent models of evolution (<a reveal-id="btz305-B1" data-open="btz305-B1" class="link link-ref link-reveal xref-bibr">Baele <em>et al.</em>, 2010</a>).
 Furthermore, we plan to explore orthogonal parallelization schemes 
(across tree nodes and/or topological moves), for leveraging the 
capabilities of modern parallel hardware and more efficiently analyzing 
datasets with thousands of taxa.</p>                    <h2 scrollto-destination="165818445" id="165818445" class="backacknowledgements-title" data-legacy-id="ack1">Acknowledgements</h2>
<p>We thank Lucas Czech, Pierre Barbera and members of the RAxML google 
group for helpful suggestions and testing the beta version of this 
software. We also thank Fabio Lehmann and Heinz Stockinger for the 
implementation and support of the RAxML-NG web server. Fast TBE 
computation code was contributed by Sarah Lutteropp.</p>                    <h3 scrollto-destination="165818447" id="165818447" class="section-title">Funding</h3>
<p>This work was financially supported by the Klaus Tschira Foundation.</p><p><em>Conflict of Interest</em>: none declared.</p>                    <h2 scrollto-destination="165818450" id="165818450" class="backreferences-title" data-legacy-id="ref1">References</h2>
<div class="ref-list"><div content-id="btz305-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B1" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baele</div> <div class="given-names">G.</div></div></div> et al.&nbsp; (<div class="year">2010</div>) 
<div class="article-title">Using non-reversible context-dependent evolutionary models to study substitution patterns in primate non-coding sequences</div>. <div class="source">J. Mol. Evol</div>&nbsp;., <div class="volume">71</div>, <div class="fpage">34</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Using%20non-reversible%20context-dependent%20evolutionary%20models%20to%20study%20substitution%20patterns%20in%20primate%20non-coding%20sequences&amp;author=G.%20Baele&amp;publication_year=2010&amp;journal=J.%20Mol.%20Evol&amp;volume=71&amp;pages=34-50" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/s00239-010-9362-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-010-9362-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs00239-010-9362-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20623275" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Using%20non-reversible%20context-dependent%20evolutionary%20models%20to%20study%20substitution%20patterns%20in%20primate%20non-coding%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B2" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barbera</div> <div class="given-names">P.</div></div></div> et al.&nbsp; (<div class="year">2018</div>) 
<div class="article-title">EPA-ng: massively parallel evolutionary placement of genetic sequences</div>. <div class="source">Syst. Biol</div>&nbsp;., <div class="volume">68</div>, <div class="fpage">365</div>–<div class="lpage">369</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=EPA-ng%3A%20massively%20parallel%20evolutionary%20placement%20of%20genetic%20sequences&amp;author=P.%20Barbera&amp;publication_year=2018&amp;journal=Syst.%20Biol&amp;volume=68&amp;pages=365-369" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syy054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyy054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyy054"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:EPA-ng%3A%20massively%20parallel%20evolutionary%20placement%20of%20genetic%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B3" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Biczok</div> <div class="given-names">R.</div></div></div> et al.&nbsp; (<div class="year">2017</div>) 
<div class="article-title">Two C++ libraries for counting trees on a phylogenetic terrace</div>. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">34</div>, <div class="fpage">3399</div>–<div class="lpage">3401</div>.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Two%20C%2B%2B%20libraries%20for%20counting%20trees%20on%20a%20phylogenetic%20terrace&amp;aulast=Biczok&amp;date=2017&amp;spage=3399&amp;epage=3401&amp;volume=34" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Two%20C%2B%2B%20libraries%20for%20counting%20trees%20on%20a%20phylogenetic%20terrace&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B4" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fletcher</div> <div class="given-names">R.</div></div></div> (<div class="year">1987</div>) <div class="source">Practical Methods of Optimization</div>&nbsp;. Vol. 1. John Wiley &amp; Sons, Chichester, New York.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Practical%20Methods%20of%20Optimization&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Practical%20Methods%20of%20Optimization&amp;aulast=Fletcher&amp;date=1987" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Practical%20Methods%20of%20Optimization&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Practical%20Methods%20of%20Optimization">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B5" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div></div> et al.&nbsp; (<div class="year">2010</div>) 
<div class="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</div>. <div class="source">Syst. Biol</div>&nbsp;., <div class="volume">59</div>, <div class="fpage">307</div>–<div class="lpage">321</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;publication_year=2010&amp;journal=Syst.%20Biol&amp;volume=59&amp;pages=307-321" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syq010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B6" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobert</div> <div class="given-names">K.</div></div></div> et al.&nbsp; (<div class="year">2014</div>) The divisible load balance problem and its application to phylogenetic inference. In: <div class="person-group"><div class="name string-name"><div class="surname">Brown</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div></div> (eds) <div class="source">Algorithms in Bioinformatics, Volume 8701 of Lecture Notes in Computer Science</div>&nbsp;. 
<div class="publisher-name">Springer</div>, 
<div class="publisher-loc">Berlin Heidelberg</div>, pp. <div class="fpage">204</div>–<div class="lpage">216</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Algorithms%20in%20Bioinformatics%2C%20Volume%208701%20of%20Lecture%20Notes%20in%20Computer%20Science&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Algorithms%20in%20Bioinformatics%2C%20Volume%208701%20of%20Lecture%20Notes%20in%20Computer%20Science&amp;aulast=Kobert&amp;date=2014&amp;spage=204&amp;epage=216" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Algorithms%20in%20Bioinformatics%2C%20Volume%208701%20of%20Lecture%20Notes%20in%20Computer%20Science&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Algorithms%20in%20Bioinformatics%2C%20Volume%208701%20of%20Lecture%20Notes%20in%20Computer%20Science">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B7" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobert</div> <div class="given-names">K.</div></div></div> et al.&nbsp; (<div class="year">2017</div>) 
<div class="article-title">Efficient detection of repeating sites to accelerate phylogenetic likelihood calculations</div>. <div class="source">Syst. Biol</div>&nbsp;., <div class="volume">66</div>, <div class="fpage">205</div>–<div class="lpage">217</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Efficient%20detection%20of%20repeating%20sites%20to%20accelerate%20phylogenetic%20likelihood%20calculations&amp;author=K.%20Kobert&amp;publication_year=2017&amp;journal=Syst.%20Biol&amp;volume=66&amp;pages=205-217" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27576546" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Efficient%20detection%20of%20repeating%20sites%20to%20accelerate%20phylogenetic%20likelihood%20calculations&amp;aulast=Kobert&amp;date=2017&amp;spage=205&amp;epage=217&amp;volume=66" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Efficient%20detection%20of%20repeating%20sites%20to%20accelerate%20phylogenetic%20likelihood%20calculations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B8" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kozlov</div> <div class="given-names">A.M.</div></div></div> et al.&nbsp; (<div class="year">2015</div>) 
<div class="article-title">ExaML version 3: a tool for phylogenomic analyses on supercomputers</div>. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">31</div>, <div class="fpage">2577</div>–<div class="lpage">2579</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ExaML%20version%203%3A%20a%20tool%20for%20phylogenomic%20analyses%20on%20supercomputers&amp;author=A.M.%20Kozlov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=2577-2579" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btv184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtv184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25819675" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ExaML%20version%203%3A%20a%20tool%20for%20phylogenomic%20analyses%20on%20supercomputers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B9" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le</div> <div class="given-names">S.Q.</div></div></div> et al.&nbsp; (<div class="year">2008</div>) 
<div class="article-title">Empirical profile mixture models for phylogenetic reconstruction</div>. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">24</div>, <div class="fpage">2317</div>–<div class="lpage">2323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Empirical%20profile%20mixture%20models%20for%20phylogenetic%20reconstruction&amp;author=S.Q.%20Le&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2317-2323" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btn445" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn445" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtn445"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18718941" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Empirical%20profile%20mixture%20models%20for%20phylogenetic%20reconstruction&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B10" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le</div> <div class="given-names">S.Q.</div></div></div> et al.&nbsp; (<div class="year">2012</div>) 
<div class="article-title">Modeling protein evolution with several amino acid replacement matrices depending on site rates</div>. <div class="source">Mol. Biol. Evol</div>&nbsp;., <div class="volume">29</div>, <div class="fpage">2921</div>–<div class="lpage">2936</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Modeling%20protein%20evolution%20with%20several%20amino%20acid%20replacement%20matrices%20depending%20on%20site%20rates&amp;author=S.Q.%20Le&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=2921-2936" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/mss112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmss112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22491036" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Modeling%20protein%20evolution%20with%20several%20amino%20acid%20replacement%20matrices%20depending%20on%20site%20rates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B11" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lemoine</div> <div class="given-names">F.</div></div></div> et al.&nbsp; (<div class="year">2018</div>) 
<div class="article-title">Renewing Felsensteinen phylogenetic bootstrap in the era of big data</div>. <div class="source">Nature</div>&nbsp;, <div class="volume">556</div>, <div class="fpage">452</div>–<div class="lpage">456</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Renewing%20Felsensteinen%20phylogenetic%20bootstrap%20in%20the%20era%20of%20big%20data&amp;author=F.%20Lemoine&amp;publication_year=2018&amp;journal=Nature&amp;volume=556&amp;pages=452-456" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41586-018-0043-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41586-018-0043-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41586-018-0043-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29670290" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Renewing%20Felsensteinen%20phylogenetic%20bootstrap%20in%20the%20era%20of%20big%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B12" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">L.-T.</div></div></div> et al.&nbsp; (<div class="year">2015</div>) 
<div class="article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</div>. <div class="source">Mol. Biol. Evol</div>&nbsp;., <div class="volume">32</div>, <div class="fpage">268</div>–<div class="lpage">274</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;author=L.-T.%20Nguyen&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=268-274" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msu300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsu300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371430" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B13" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Price</div> <div class="given-names">M.N.</div></div></div> et al.&nbsp; (<div class="year">2010</div>) 
<div class="article-title">FastTree 2 – approximately maximum-likelihood trees for large alignments</div>. <div class="source">PLoS One</div>&nbsp;, <div class="volume">5</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=FastTree%202%20%E2%80%93%20approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;aulast=Price&amp;date=2010&amp;spage=1&amp;epage=10&amp;volume=5" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FastTree%202%20%E2%80%93%20approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B14" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sanderson</div> <div class="given-names">M.J.</div></div></div> et al.&nbsp; (<div class="year">2011</div>) 
<div class="article-title">Terraces in phylogenetic tree space</div>. <div class="source">Science</div>&nbsp;, <div class="volume">333</div>, <div class="fpage">448</div>–<div class="lpage">450</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Terraces%20in%20phylogenetic%20tree%20space&amp;author=M.J.%20Sanderson&amp;publication_year=2011&amp;journal=Science&amp;volume=333&amp;pages=448-450" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1206357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1206357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1206357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21680810" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Terraces%20in%20phylogenetic%20tree%20space&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B15" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> (<div class="year">2006</div>) Phylogenetic models of rate heterogeneity: a high performance computing perspective. In: <em>Proceedings of IPDPS2006, HICOMB Workshop</em>, <em>Proceedings on CD</em>, IEEE, Rhodos, Greece.</div></div></div></div><div content-id="btz305-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B16" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> (<div class="year">2014</div>) 
<div class="article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</div>. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">30</div>, <div class="fpage">1312</div>–<div class="lpage">1313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1312-1313" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B17" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> (<div class="year">1995</div>) 
<div class="article-title">A space-time process model for the evolution of DNA sequences</div>. <div class="source">Genetics</div>&nbsp;, <div class="volume">139</div>, <div class="fpage">993</div>–<div class="lpage">1005</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20space-time%20process%20model%20for%20the%20evolution%20of%20DNA%20sequences&amp;author=Z.%20Yang&amp;publication_year=1995&amp;journal=Genetics&amp;volume=139&amp;pages=993-1005" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7713447" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20space-time%20process%20model%20for%20the%20evolution%20of%20DNA%20sequences&amp;aulast=Yang&amp;date=1995&amp;spage=993&amp;epage=1005&amp;volume=139" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20space-time%20process%20model%20for%20the%20evolution%20of%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btz305-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btz305-B18" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div></div> et al.&nbsp; (<div class="year">2018</div>) 
<div class="article-title">Evaluating fast maximum likelihood-based phylogenetic programs using empirical phylogenomic data sets</div>. <div class="source">Mol. Biol. Evol</div>&nbsp;., <div class="volume">35</div>, <div class="fpage">486</div>–<div class="lpage">503</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluating%20fast%20maximum%20likelihood-based%20phylogenetic%20programs%20using%20empirical%20phylogenomic%20data%20sets&amp;author=X.%20Zhou&amp;publication_year=2018&amp;journal=Mol.%20Biol.%20Evol&amp;volume=35&amp;pages=486-503" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msx302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsx302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsx302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29177474" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluating%20fast%20maximum%20likelihood-based%20phylogenetic%20programs%20using%20empirical%20phylogenomic%20data%20sets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="">http://creativecommons.org/licenses/by-nc/4.0/</a>),
 which permits non-commercial re-use, distribution, and reproduction in 
any medium, provided the original work is properly cited. For commercial
 re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5487384&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5139">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/35/21/10.1093_bioinformatics_btz305/1/btz305_supplementary_data.pdf?Expires=1577264505&amp;Signature=Qh%7EkMCrNAUR80EIIMXAO0dRnNJ5XswCS-vlA-mNHso129HOlUA-TdY0g8gR15p4W2Tggu557f014jEzycNqCpgJT4FU41HsgO5nAOgDWOqNc0p%7ERDw6Aoz21B-9Ur7NpbFqG55PSjeILrPyVB7sqPklB1oNBgswGNV71pcYTg0wXgkklm0s7WzCeT-lUNsF%7ENCf5M5JvwDqdd8E6Iu-2OltDaJO3hL1etwFSKaSondpdA2sCrwjAra60VrdgeR10MFa8psqjXOg99BLXYXskDaXYkvusLQXcpEng3Zxc%7EaAcWVm5D1GTygAnyiw%7E8GbIkzRi9Q--MLO641VFnTpuhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">btz305_Supplementary_Data</div></a> - pdf file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="11"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,268</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/bioinformatics/btz305" data-uuid="803dbc40-d5e4-6a86-6fb0-9837b4c92ddf"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=60126899" style="display:inline-block;">
        <img alt="Article has an altmetric score of 26" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5487384"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fbioinformatics%2farticle%2f35%2f21%2f4453%2f5487384">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F35%2F21%2F4453%2F5487384" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/bioinformatics/crossref-citedby/5487384">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz861/5631910?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                BRM: A statistical method for QTL mapping based on bulked segregant analysis by deep sequencing
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz841/5631909?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                HypercubeME: two hundred million combinatorially complete datasets from a single experiment
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz843/5631908?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                An Integrative Approach for Fine-Mapping Chromatin Interactions
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz859/5631907?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                KofamKOALA: KEGG ortholog assignment based on profile HMM and adaptive score threshold
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz826/5631906?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                SCOT: Rethinking the Classification of Secondary Structure Elements
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>
    <div class="widget widget-JobTarget widget-instance-Article_RightRailB0Article_RightRail_JobTarget">
        <div class="wrapper">
    <div class="header">
        <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/">Looking for your next opportunity?</a>
    </div>

        <div id="jobs" class="jobs-list">
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/specialist-cardiologist-echocardiography/51691723/20102/">Specialist Cardiologist - echocardiography</a></div>
                    <div class="job-location">
                        Hamilton, Other / Non US
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/specialist-orthopaedic-surgeon-subspecialty-spines/51691492/20102/">Specialist Orthopaedic Surgeon - Subspecialty spines</a></div>
                    <div class="job-location">
                        Hamilton, Other / Non US
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/assistant-clinical-investigator/51691386/20102/">Assistant Clinical Investigator</a></div>
                    <div class="job-location">
                        Bethesda, Maryland
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/assistant-clinical-investigator/51665158/20102/">Assistant Clinical Investigator</a></div>
                    <div class="job-location">
                        bethesda, Maryland
                    </div>
                </div>
        </div>
        <div class="all-jobs"><a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/">View all jobs</a></div>

    <div class="sponsor-and-jobnetwork-links">
        <div class="job-sponsor"><a href="http://global.oup.com/"><img src="oup-logo_002.svg" alt="OXFORD University Press"></a></div>
        <div class="job-network"><a href="http://globaljobs-careernetwork.oxfordjournals.org/home/index.cfm?site_id=20101"><img src="journals-careers-network.svg" alt="Journals Career Network"></a></div>
    </div>
</div>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference | Bioinformatics | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/bioinformatics/btz305");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/bioinformatics/pages/About">About Bioinformatics</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://twitter.com/oxfordjournals">Twitter</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://academic.oup.com/bioinformatics/subscribe">Purchase</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/bioinformatics" target="">
                        <img id="footer-logo-Bioinformatics" class="journal-footer-affiliations-logo" src="bioinformatics_f1-205396838.svg" alt="Bioinformatics">
                    </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1460-2059</li>
                                            <li>Print ISSN 1367-4803</li>
                    <li>Copyright © 2019 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh651" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1574241514154&amp;tmr=load%3D1574241514118%26core%3D1574241514146%26main%3D1574241514150%26ifr%3D1574241514160&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F35%2F21%2F4453%2F5487384&amp;href=https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F35%2F21%2F4453%2F5487384&amp;dt=RAxML-NG%3A%20a%20fast%2C%20scalable%20and%20user-friendly%20tool%20for%20maximum%20likelihood%20phylogenetic%20inference&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5dd504ead28d0d3b&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3DRAxML-NG%253A%2520a%2520fast%252C%2520scalable%2520and%2520user-friendly%2520tool%2520for%2520maximum%2520likelihood%2520phylogenetic%2520inference%26description%3DAbstractMotivation.%2520%2520Phylogenies%2520are%2520important%2520for%2520fundamental%2520biological%2520research%252C%2520but%2520also%2520have%2520numerous%2520applications%2520in%2520biotechnology%252C%2520agriculture%2520and%2520medici%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fbioinformatics%252Farticle%252F35%252F21%252F4453%252F5487384%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fbioinformatics%252FIssue%252F35%252F21%252F1%252Fm_bioinfo_35_21cover.png%253FExpires%253D1636237385%2526Signature%253DV%7E1cgtIBht1v9aG9x6ARgS7VkgEhw-ZYH5bSCrZFo3lGs5z4EoCeulfT2sIPe7J2gSgSbL1MFOdQHt5Q9Uasr%7EzUTpXWzZoLlm%7E14yOjAgYhxwB8dCnpT6eKmjTPgdr7xNFuD87iBMPGBgs%7Eo3An0aarYJsLRsGmYtXnq4ld6aN4LXL-pwIyT-6fWTLL3CY1BXtPlKW-xYzJSRjkQkrODfuy3lSDNuVRlVTMySo-D2xPchzyU727S-c17icFLodTEjsYWwS5IqRpQB9Ghnoci0HGwIHyJYyZYZ8ZC7NTOBQSQeN0rCsBxVhEICdQSz1Pt0a0hc5kdbX-u9QEarmvwA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2019-05-09&amp;csi=undefined&amp;rev=v8.28.1-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f35%2f21%2f4453%2f5487384">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>

    <script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="btz305.js"></script></body></html>