<!DOCTYPE html>
<html lang="en"><head><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <!-- Meta -->
    <meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">

    <!-- Stylesheets -->
    
<link href="css.css" type="text/css" rel="stylesheet">
<link href="globalCSS.css" type="text/css" rel="stylesheet">
<link href="cookiePolicyCSS.css" type="text/css" rel="stylesheet">
<link href="chartist.css" type="text/css" rel="stylesheet">

        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-1071638357.css" rel="stylesheet" type="text/css">
    

    <!-- Scripts -->
    
<script async="" src="analytics.js"></script><script src="jquery-2.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [0] 
        };

        App.CurrentSubdomain = 'sysbio';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/sysbio/article/61/1/90/1680002","@type":"ScholarlyArticle","name":"SATé-II: Very Fast and Accurate Simultaneous Estimation of Multiple Sequence Alignments and Phylogenetic Trees","datePublished":"2011-12-01","isPartOf":{"@id":"https://academic.oup.com/sysbio/issue/61/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2012-01-01","isPartOf":{"@id":"https://academic.oup.com/sysbio","@type":"Periodical","name":"Systematic Biology","issn":["1076-836X"]}},"url":"http://dx.doi.org/10.1093/sysbio/syr095","keywords":["Alignment","maximum likelihood","phylogenetics","SATé"],"inLanguage":"en","copyrightHolder":"Society of Systematic Biologists","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/sysbio/article/61/1/90/1680002","author":[{"name":"Liu, Kevin","affiliation":"1 Department of Computer Science, University of Texas at Austin, Austin, TX 78712, USA","@type":"Person"},{"name":"Warnow, Tandy J.","affiliation":"1 Department of Computer Science, University of Texas at Austin, Austin, TX 78712, USA","@type":"Person"},{"name":"Holder, Mark T.","affiliation":"2 Department of Ecology and Evolutionary Biology, University of Kansas at Lawrence, Lawrence, KS 66045, USA","@type":"Person"},{"name":"Nelesen, Serita M.","affiliation":"3 Department of Computer Science, Calvin College, Grand Rapids, MI 49546, USA","@type":"Person"},{"name":"Yu, Jiaye","affiliation":"2 Department of Ecology and Evolutionary Biology, University of Kansas at Lawrence, Lawrence, KS 66045, USA","@type":"Person"},{"name":"Stamatakis, Alexandros P.","affiliation":"4 Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany","@type":"Person"},{"name":"Linder, C. Randal","affiliation":"5 Section of Integrative Biology, School of Biological Sciences, University of Texas at Austin, Austin, TX, USA","@type":"Person"}],"description":"Abstract.  Highly accurate estimation of phylogenetic trees for large data sets is difficult, in part because multiple sequence alignments must be accurate for ","pageStart":"90","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/61/1/0/m_sysbio61_1.cover.gif?Expires=2147483647&Signature=34VyRoXr-doQ21LWWePdNaq3T-X4B1yR3E4ZiE~voVMMe4AekzweCCOybgo~DYuLWruG1d6JceoEaif8OZPRV8c4UEVgu6wuQOTnrTaC8VV3-wMKI67Bk99xtGgTlSTY1x2Na942~QumosAM2Zo~TzMCIYZEeXwhObNj48UxpKYGtjl~bULDvUrK-kMZmvuuiWhbbmDxLGCoiG-nbv-JAflMQGbOO0cjDfW3u9H1dXRrIY-eIVaQxSWM2pwOPO0U9xi65lYyaxIUQ-BZd1CRHebH6B5QtCLv3IoPDVl7j26u5LAiFBeDejvkYANtLfGkpGCPueegyf7t31oN959RmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"SATé-II: Very Fast and Accurate Simultaneous Estimation of Multiple Sequence Alignments and Phylogenetic Trees","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/61/1/0/m_sysbio61_1.cover.gif?Expires=2147483647&Signature=34VyRoXr-doQ21LWWePdNaq3T-X4B1yR3E4ZiE~voVMMe4AekzweCCOybgo~DYuLWruG1d6JceoEaif8OZPRV8c4UEVgu6wuQOTnrTaC8VV3-wMKI67Bk99xtGgTlSTY1x2Na942~QumosAM2Zo~TzMCIYZEeXwhObNj48UxpKYGtjl~bULDvUrK-kMZmvuuiWhbbmDxLGCoiG-nbv-JAflMQGbOO0cjDfW3u9H1dXRrIY-eIVaQxSWM2pwOPO0U9xi65lYyaxIUQ-BZd1CRHebH6B5QtCLv3IoPDVl7j26u5LAiFBeDejvkYANtLfGkpGCPueegyf7t31oN959RmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="SATé-II: Very Fast and Accurate Simultaneous Estimation of Multiple Sequence Alignments and Phylogenetic Trees | Systematic Biology | Oxford Academic">
    <meta property="og:description" content="Abstract.  Highly accurate estimation of phylogenetic trees for large data sets is difficult, in part because multiple sequence alignments must be accurate for ">
    <meta property="og:url" content="https://academic.oup.com/sysbio/article/61/1/90/1680002">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/67/5/2/m_sysbio_67_5cover.png?Expires=2147483647&amp;Signature=iMgOj~ozkXgiMv5ypBZDzosyeLzAPALO2wmmKLje0Ck54FjJm347WeTb3PKZWVrxYH9T2vN47duSgGc4dLVp5tBbPeDOQV99~fC0QoiuNqiJsKVtDjUdYShshHjSJhQalWrhtfVODxOV3i~okx0xHvcYKoh0XaVEgY8howRgKsjKDokUzxP8A-IlCyuRe7~RCRWcLvUokOnZjMW6dIfPEHfg4NKlHFUTrgzwYmZcyCm1Idbn11KibNLt9Vs51khpe8c8~eRqAFQvD8TZZR2syjseNNvaC62SMaVI7aravRbjdk14RleGCLdGvpguCHY1RYMY5C~wSqHtyxn-d0GGQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2011-12-01">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Liu, Kevin"><meta name="citation_author_institution" content="Department of Computer Science, University of Texas at Austin, Austin, TX 78712, USA"><meta name="citation_author" content="Warnow, Tandy J."><meta name="citation_author_institution" content="Department of Computer Science, University of Texas at Austin, Austin, TX 78712, USA"><meta name="citation_author" content="Holder, Mark T."><meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas at Lawrence, Lawrence, KS 66045, USA"><meta name="citation_author" content="Nelesen, Serita M."><meta name="citation_author_institution" content="Department of Computer Science, Calvin College, Grand Rapids, MI 49546, USA"><meta name="citation_author" content="Yu, Jiaye"><meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas at Lawrence, Lawrence, KS 66045, USA"><meta name="citation_author" content="Stamatakis, Alexandros P."><meta name="citation_author_institution" content="Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany"><meta name="citation_author" content="Linder, C. Randal"><meta name="citation_author_institution" content="Section of Integrative Biology, School of Biological Sciences, University of Texas at Austin, Austin, TX, USA"><meta name="citation_title" content="SATé-II: Very Fast and Accurate Simultaneous Estimation of Multiple Sequence Alignments and Phylogenetic Trees"><meta name="citation_firstpage" content="90"><meta name="citation_lastpage" content="90"><meta name="citation_doi" content="10.1093/sysbio/syr095"><meta name="citation_journal_title" content="Systematic Biology"><meta name="citation_journal_abbrev" content="Syst Biol"><meta name="citation_volume" content="61"><meta name="citation_issue" content="1"><meta name="citation_publication_date" content="2012/01/01"><meta name="citation_issn" content="1063-5157"><meta name="citation_publisher" content="Oxford University Press"><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; J. R. Stat. Soc. Series B Met;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; "><meta name="citation_reference" content="citation_title=The comparative RNA web (CRW) site: an online database of comparative sequence and structure information for ribosomal, intron and other RNAs; citation_text=Available from: http://www.rna.ccbb.utexas.edu. Accessed 4 October 2011; BMC Bioinformatics;  citation_year=2002;  citation_volume=3;  citation_pages=2"><meta name="citation_reference" content="citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity; BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=113"><meta name="citation_reference" content="citation_title=MUSCLE: a multiple sequence alignment with high accuracy and high throughput; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1792-1797; "><meta name="citation_reference" content="citation_title=Simultaneous statistical multiple alignment and phylogeny reconstruction; Syst. Biol.;  citation_year=2005;  citation_volume=54;  citation_pages=548-561; "><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood; Syst. Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=696-704; "><meta name="citation_reference" content="citation_title=Lessons from an evolving ribosomal-RNA—16S and 23S ribosomal RNA structures from a comparative perspective; Microbiol. Rev.;  citation_year=1994;  citation_volume=58;  citation_pages=10-26; "><meta name="citation_reference" content="citation_title=The accuracy of ribosomal RNA comparative structure models. Curr. Opin; Struct. Biol.;  citation_year=2002;  citation_volume=12;  citation_pages=301-310; "><meta name="citation_reference" content="citation_title=Unified approach to alignment and phylogenies; Methods Enzymol;  citation_year=1990;  citation_volume=183;  citation_pages=626-645; "><meta name="citation_reference" content="citation_title=An empirical test of bootstrapping as a method for assessing confidence in phylogenetic analysis; Syst. Biol.;  citation_year=1993;  citation_volume=42;  citation_pages=182-192; "><meta name="citation_reference" content="citation_title=Evolution of protein molecules; citation_publisher=Academic Press, New York; Mammalian protein metabolism;  citation_year=1969;  citation_pages=21-132; "><meta name="citation_reference" content="citation_title=MAFFT version 5: improvement in accuracy of multiple sequence alignment; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=511-518; "><meta name="citation_reference" content="citation_title=PartTree: an algorithm to build an approximate tree from a large number of unaligned sequences; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=372-374; "><meta name="citation_reference" content="citation_title=Recent developments in the MAFFT multiple sequence alignment program; Brief. Bioinform;  citation_year=2008;  citation_volume=9;  citation_pages=286-298; "><meta name="citation_reference" content="citation_title=Aligning alignments; citation_publisher=Springer, Berlin (Germany); Proceedings of CPM 1998 Lecture Notes in Computer Science;  citation_year=1998;  citation_pages=189-208; "><meta name="citation_reference" content="citation_title=A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences; J. Mol. Evol.;  citation_year=1980;  citation_volume=16;  citation_pages=111-120; "><meta name="citation_reference" content="citation_title=GESTALT: Genomic Steiner alignments; Lect. Notes Comput. Sci.;  citation_year=1999;  citation_volume=1645;  citation_pages=101-114; "><meta name="citation_reference" content="citation_title=Remote Unix—turning idle workstations into cycle servers. Usenix summer conference. USENIX Association;  citation_year=1987;  citation_pages=381-384; "><meta name="citation_reference" content="citation_title=Multiple sequence alignment: a major challenge to large-scale phylogenetics. PLoS Curr. Tree of Life; citation_text=Available from: http://knol.google.com/k/kevin-liu/multiple-sequence-alignment-a-major/ectabesw3uba/9. Accessed 4 October 2011;  citation_year=2010; "><meta name="citation_reference" content="citation_title=Barking up the wrong treelength: the impact of gap penalty on alignment and tree accuracy; IEEE/ACM Trans. Comput. Biol. Bioinform;  citation_year=2009;  citation_volume=6;  citation_pages=7-21; "><meta name="citation_reference" content="citation_title=Rapid and accurate large-scale coestimation of sequence alignments and phylogenetic trees; Science;  citation_year=2009;  citation_volume=324;  citation_pages=1561-1564; "><meta name="citation_reference" content="citation_title=Bayesian coestimation of phylogeny and sequence alignment; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=83"><meta name="citation_reference" content="citation_text=2002 January 3–7; Lihue, Hawaii. p. 211–222; The accuracy of fast phylogenetic methods for large datasets. Proceedings of the 7th Pacific Symposium on BioComputing (PSB02);  citation_year=2002; "><meta name="citation_reference" content="citation_title=StatAlign: an extendable software package for joint Bayesian estimation of alignments and evolutionary trees; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2403-2404; "><meta name="citation_reference" content="citation_title=FastTree 2—approximately maximum-likelihood trees for large alignments; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e9490"><meta name="citation_reference" content="citation_title=Joint Bayesian estimation of alignment and phylogeny; Syst. Biol.;  citation_year=2005;  citation_volume=54;  citation_pages=401-418; "><meta name="citation_reference" content="citation_title=The general stochastic model of nucleotide substitution; J. Theor. Biol.;  citation_year=1990;  citation_volume=142;  citation_pages=485-501; "><meta name="citation_reference" content="citation_title=r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=301-302; "><meta name="citation_reference" content="citation_title=Minimal mutation trees of sequences; SIAM J. Appl. Math.;  citation_year=1975;  citation_volume=28;  citation_pages=35-42; "><meta name="citation_reference" content="citation_title=Simultaneous comparison of three or more sequences related by a tree; citation_text=Time warps, string edits, and macromolecules: the theory and practice of sequence comparison. Chapter 2. Reading (MA): Addison-Wesley. p. 253–264;  citation_year=1983; "><meta name="citation_reference" content="citation_title=Mega-phylogeny approach for comparative biology: an alternative to supertree and supermatrix approaches; BMC Evol. Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=37"><meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2688-2690; "><meta name="citation_reference" content="citation_title=Rose: generating sequence families; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=157-163; "><meta name="citation_reference" content="citation_title=BAli-Phy: simultaneous Bayesian inference of alignment and phylogeny; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2047-2048; "><meta name="citation_reference" content="citation_title=PAUP*: phylogenetic analysis using parsimony (*and other methods). version 4; citation_publisher=Sinauer Associates, Sunderland (MA);  citation_year=2003; "><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-4680; "><meta name="citation_reference" content="citation_title=A comprehensive comparison of multiple sequence alignment programs; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=2682-2690; "><meta name="citation_reference" content="citation_title=An evolutionary model for maximum likelihood alignment of DNA sequences; J. Mol. Evol.;  citation_year=1991;  citation_volume=33;  citation_pages=114-124; "><meta name="citation_reference" content="citation_title=Inching toward reality: an improved likelihood model of sequence evolution; J. Mol. Evol.;  citation_year=1992;  citation_volume=34;  citation_pages=3-16; "><meta name="citation_reference" content="citation_title=Links between maximum likelihood and maximum parsimony under a simple model of site substitution; Bull. Math. Biol.;  citation_year=1997;  citation_volume=59;  citation_pages=581-607; "><meta name="citation_reference" content="citation_title=POY version 4: phylogenetic analysis using dynamic homologies; Cladistics;  citation_year=2010;  citation_volume=26;  citation_pages=72-85; "><meta name="citation_reference" content="citation_title=Sequence alignment, parameter sensitivity, and the phylogenetic analysis of molecular data; Syst. Biol.;  citation_year=1995;  citation_volume=44;  citation_pages=321-331; "><meta name="citation_reference" content="citation_title=Optimization alignment: the end of multiple sequence alignment in phylogenetics?; Cladistics;  citation_year=1996;  citation_volume=12;  citation_pages=1-9; "><meta name="citation_reference" content="citation_title=MALIGN: a multiple sequence alignment program; J. Hered;  citation_year=1994;  citation_volume=85;  citation_pages=417-418; "><meta name="citation_reference" content="citation_title=Multiple alignment by aligning alignments; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=i559-i568; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/sysbio/article-pdf/61/1/90/24562924/syr095.pdf"><meta name="description" content="Abstract.  Highly accurate estimation of phylogenetic trees for large data sets is difficult, in part because multiple sequence alignments must be accurate for ">
        <meta name="citation_xml_url" content="https://academic.oup.com/sysbio/article-xml/61/1/90/1680002">



    <link rel="canonical" href="https://academic.oup.com/sysbio/article/61/1/90/1680002">


        <!-- Start: GPT Async -->

<script async="async" src="gpt.js"></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type="text/javascript">
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'sysbio_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "sysbio/article/61/1/90/1680002");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=sysbio&authzrequired=false&doi=10.1093/sysbio/syr095';
    </script>
<title>
	SATé-II: Very Fast and Accurate Simultaneous Estimation of Multiple 
Sequence Alignments and Phylogenetic Trees | Systematic Biology | Oxford
 Academic
</title><script id="altmetric-embed-js" src="altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_264.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>

<body data-sitename="systematicbiology" class="off-canvas pg_article" theme-sysbio=""><div id="MathJax_Message" style="display: none;"></div>
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 100});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1680002" id="webform">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYtlPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaXUsIEtldmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRlciBTY2llbmNlLCBVbml2ZXJzaXR5IG9mIFRleGFzIGF0IEF1c3RpbiwgQXVzdGluLCBUWCA3ODcxMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2Fybm93LCBUYW5keSBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0ZXIgU2NpZW5jZSwgVW5pdmVyc2l0eSBvZiBUZXhhcyBhdCBBdXN0aW4sIEF1c3RpbiwgVFggNzg3MTIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhvbGRlciwgTWFyayBULiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRWNvbG9neSBhbmQgRXZvbHV0aW9uYXJ5IEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgS2Fuc2FzIGF0IExhd3JlbmNlLCBMYXdyZW5jZSwgS1MgNjYwNDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5lbGVzZW4sIFNlcml0YSBNLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0ZXIgU2NpZW5jZSwgQ2FsdmluIENvbGxlZ2UsIEdyYW5kIFJhcGlkcywgTUkgNDk1NDYsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1LCBKaWF5ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRWNvbG9neSBhbmQgRXZvbHV0aW9uYXJ5IEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgS2Fuc2FzIGF0IExhd3JlbmNlLCBMYXdyZW5jZSwgS1MgNjYwNDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN0YW1hdGFraXMsIEFsZXhhbmRyb3MgUC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2llbnRpZmljIENvbXB1dGluZyBHcm91cCwgSGVpZGVsYmVyZyBJbnN0aXR1dGUgZm9yIFRoZW9yZXRpY2FsIFN0dWRpZXMsIFNjaGxvc3MtV29sZnNicnVubmVud2VnIDM1LCBELTY5MTE4IEhlaWRlbGJlcmcsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaW5kZXIsIEMuIFJhbmRhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNlY3Rpb24gb2YgSW50ZWdyYXRpdmUgQmlvbG9neSwgU2Nob29sIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgVGV4YXMgYXQgQXVzdGluLCBBdXN0aW4sIFRYLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNBVMOpLUlJOiBWZXJ5IEZhc3QgYW5kIEFjY3VyYXRlIFNpbXVsdGFuZW91cyBFc3RpbWF0aW9uIG9mIE11bHRpcGxlIFNlcXVlbmNlIEFsaWdubWVudHMgYW5kIFBoeWxvZ2VuZXRpYyBUcmVlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI5MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvc3lzYmlvL3N5cjA5NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJTeXN0ZW1hdGljIEJpb2xvZ3kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IlN5c3QgQmlvbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTA2My01MTU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sbGluZyB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGU6IGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyBKLiBSLiBTdGF0LiBTb2MuIFNlcmllcyBCIE1ldDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT01NzsgIGNpdGF0aW9uX3BhZ2VzPTI4OS0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wYXJhdGl2ZSBSTkEgd2ViIChDUlcpIHNpdGU6IGFuIG9ubGluZSBkYXRhYmFzZSBvZiBjb21wYXJhdGl2ZSBzZXF1ZW5jZSBhbmQgc3RydWN0dXJlIGluZm9ybWF0aW9uIGZvciByaWJvc29tYWwsIGludHJvbiBhbmQgb3RoZXIgUk5BczsgY2l0YXRpb25fdGV4dD1BdmFpbGFibGUgZnJvbTogaHR0cDovL3d3dy5ybmEuY2NiYi51dGV4YXMuZWR1LiBBY2Nlc3NlZCA0IE9jdG9iZXIgMjAxMTsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVVTQ0xFOiBhIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBtZXRob2Qgd2l0aCByZWR1Y2VkIHRpbWUgYW5kIHNwYWNlIGNvbXBsZXhpdHk7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVVTQ0xFOiBhIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIGhpZ2ggYWNjdXJhY3kgYW5kIGhpZ2ggdGhyb3VnaHB1dDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTc5Mi0xNzk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW11bHRhbmVvdXMgc3RhdGlzdGljYWwgbXVsdGlwbGUgYWxpZ25tZW50IGFuZCBwaHlsb2dlbnkgcmVjb25zdHJ1Y3Rpb247IFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NTQ4LTU2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUsIGZhc3QsIGFuZCBhY2N1cmF0ZSBhbGdvcml0aG0gdG8gZXN0aW1hdGUgbGFyZ2UgcGh5bG9nZW5pZXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTY5Ni03MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlc3NvbnMgZnJvbSBhbiBldm9sdmluZyByaWJvc29tYWwtUk5B4oCUMTZTIGFuZCAyM1Mgcmlib3NvbWFsIFJOQSBzdHJ1Y3R1cmVzIGZyb20gYSBjb21wYXJhdGl2ZSBwZXJzcGVjdGl2ZTsgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTAtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhY2N1cmFjeSBvZiByaWJvc29tYWwgUk5BIGNvbXBhcmF0aXZlIHN0cnVjdHVyZSBtb2RlbHMuIEN1cnIuIE9waW47IFN0cnVjdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zMDEtMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlmaWVkIGFwcHJvYWNoIHRvIGFsaWdubWVudCBhbmQgcGh5bG9nZW5pZXM7IE1ldGhvZHMgRW56eW1vbDsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz02MjYtNjQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlbXBpcmljYWwgdGVzdCBvZiBib290c3RyYXBwaW5nIGFzIGEgbWV0aG9kIGZvciBhc3Nlc3NpbmcgY29uZmlkZW5jZSBpbiBwaHlsb2dlbmV0aWMgYW5hbHlzaXM7IFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTgyLTE5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHByb3RlaW4gbW9sZWN1bGVzOyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIE5ldyBZb3JrOyBNYW1tYWxpYW4gcHJvdGVpbiBtZXRhYm9saXNtOyAgY2l0YXRpb25feWVhcj0xOTY5OyAgY2l0YXRpb25fcGFnZXM9MjEtMTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVCB2ZXJzaW9uIDU6IGltcHJvdmVtZW50IGluIGFjY3VyYWN5IG9mIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTExLTUxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFydFRyZWU6IGFuIGFsZ29yaXRobSB0byBidWlsZCBhbiBhcHByb3hpbWF0ZSB0cmVlIGZyb20gYSBsYXJnZSBudW1iZXIgb2YgdW5hbGlnbmVkIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0zNzItMzc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQgZGV2ZWxvcG1lbnRzIGluIHRoZSBNQUZGVCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgcHJvZ3JhbTsgQnJpZWYuIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9Mjg2LTI5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxpZ25pbmcgYWxpZ25tZW50czsgY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW4gKEdlcm1hbnkpOyBQcm9jZWVkaW5ncyBvZiBDUE0gMTk5OCBMZWN0dXJlIE5vdGVzIGluIENvbXB1dGVyIFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl9wYWdlcz0xODktMjA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtZXRob2QgZm9yIGVzdGltYXRpbmcgZXZvbHV0aW9uYXJ5IHJhdGVzIG9mIGJhc2Ugc3Vic3RpdHV0aW9ucyB0aHJvdWdoIGNvbXBhcmF0aXZlIHN0dWRpZXMgb2YgbnVjbGVvdGlkZSBzZXF1ZW5jZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HRVNUQUxUOiBHZW5vbWljIFN0ZWluZXIgYWxpZ25tZW50czsgTGVjdC4gTm90ZXMgQ29tcHV0LiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE2NDU7ICBjaXRhdGlvbl9wYWdlcz0xMDEtMTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZW1vdGUgVW5peOKAlHR1cm5pbmcgaWRsZSB3b3Jrc3RhdGlvbnMgaW50byBjeWNsZSBzZXJ2ZXJzLiBVc2VuaXggc3VtbWVyIGNvbmZlcmVuY2UuIFVTRU5JWCBBc3NvY2lhdGlvbjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3BhZ2VzPTM4MS0zODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudDogYSBtYWpvciBjaGFsbGVuZ2UgdG8gbGFyZ2Utc2NhbGUgcGh5bG9nZW5ldGljcy4gUExvUyBDdXJyLiBUcmVlIG9mIExpZmU7IGNpdGF0aW9uX3RleHQ9QXZhaWxhYmxlIGZyb206IGh0dHA6Ly9rbm9sLmdvb2dsZS5jb20vay9rZXZpbi1saXUvbXVsdGlwbGUtc2VxdWVuY2UtYWxpZ25tZW50LWEtbWFqb3IvZWN0YWJlc3czdWJhLzkuIEFjY2Vzc2VkIDQgT2N0b2JlciAyMDExOyAgY2l0YXRpb25feWVhcj0yMDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXJraW5nIHVwIHRoZSB3cm9uZyB0cmVlbGVuZ3RoOiB0aGUgaW1wYWN0IG9mIGdhcCBwZW5hbHR5IG9uIGFsaWdubWVudCBhbmQgdHJlZSBhY2N1cmFjeTsgSUVFRS9BQ00gVHJhbnMuIENvbXB1dC4gQmlvbC4gQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBhbmQgYWNjdXJhdGUgbGFyZ2Utc2NhbGUgY29lc3RpbWF0aW9uIG9mIHNlcXVlbmNlIGFsaWdubWVudHMgYW5kIHBoeWxvZ2VuZXRpYyB0cmVlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTYxLTE1NjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJheWVzaWFuIGNvZXN0aW1hdGlvbiBvZiBwaHlsb2dlbnkgYW5kIHNlcXVlbmNlIGFsaWdubWVudDsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz04MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RleHQ9MjAwMiBKYW51YXJ5IDPigJM3OyBMaWh1ZSwgSGF3YWlpLiBwLiAyMTHigJMyMjI7IFRoZSBhY2N1cmFjeSBvZiBmYXN0IHBoeWxvZ2VuZXRpYyBtZXRob2RzIGZvciBsYXJnZSBkYXRhc2V0cy4gUHJvY2VlZGluZ3Mgb2YgdGhlIDd0aCBQYWNpZmljIFN5bXBvc2l1bSBvbiBCaW9Db21wdXRpbmcgKFBTQjAyKTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdEFsaWduOiBhbiBleHRlbmRhYmxlIHNvZnR3YXJlIHBhY2thZ2UgZm9yIGpvaW50IEJheWVzaWFuIGVzdGltYXRpb24gb2YgYWxpZ25tZW50cyBhbmQgZXZvbHV0aW9uYXJ5IHRyZWVzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI0MDMtMjQwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdFRyZWUgMuKAlGFwcHJveGltYXRlbHkgbWF4aW11bS1saWtlbGlob29kIHRyZWVzIGZvciBsYXJnZSBhbGlnbm1lbnRzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTk0OTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Kb2ludCBCYXllc2lhbiBlc3RpbWF0aW9uIG9mIGFsaWdubWVudCBhbmQgcGh5bG9nZW55OyBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTQwMS00MTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5lcmFsIHN0b2NoYXN0aWMgbW9kZWwgb2YgbnVjbGVvdGlkZSBzdWJzdGl0dXRpb247IEouIFRoZW9yLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDI7ICBjaXRhdGlvbl9wYWdlcz00ODUtNTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1yOHM6IGluZmVycmluZyBhYnNvbHV0ZSByYXRlcyBvZiBtb2xlY3VsYXIgZXZvbHV0aW9uIGFuZCBkaXZlcmdlbmNlIHRpbWVzIGluIHRoZSBhYnNlbmNlIG9mIGEgbW9sZWN1bGFyIGNsb2NrOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTMwMS0zMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmltYWwgbXV0YXRpb24gdHJlZXMgb2Ygc2VxdWVuY2VzOyBTSUFNIEouIEFwcGwuIE1hdGguOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MzUtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXVsdGFuZW91cyBjb21wYXJpc29uIG9mIHRocmVlIG9yIG1vcmUgc2VxdWVuY2VzIHJlbGF0ZWQgYnkgYSB0cmVlOyBjaXRhdGlvbl90ZXh0PVRpbWUgd2FycHMsIHN0cmluZyBlZGl0cywgYW5kIG1hY3JvbW9sZWN1bGVzOiB0aGUgdGhlb3J5IGFuZCBwcmFjdGljZSBvZiBzZXF1ZW5jZSBjb21wYXJpc29uLiBDaGFwdGVyIDIuIFJlYWRpbmcgKE1BKTogQWRkaXNvbi1XZXNsZXkuIHAuIDI1M+KAkzI2NDsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVnYS1waHlsb2dlbnkgYXBwcm9hY2ggZm9yIGNvbXBhcmF0aXZlIGJpb2xvZ3k6IGFuIGFsdGVybmF0aXZlIHRvIHN1cGVydHJlZSBhbmQgc3VwZXJtYXRyaXggYXBwcm9hY2hlczsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJBeE1MLVZJLUhQQzogbWF4aW11bSBsaWtlbGlob29kLWJhc2VkIHBoeWxvZ2VuZXRpYyBhbmFseXNlcyB3aXRoIHRob3VzYW5kcyBvZiB0YXhhIGFuZCBtaXhlZCBtb2RlbHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjY4OC0yNjkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb3NlOiBnZW5lcmF0aW5nIHNlcXVlbmNlIGZhbWlsaWVzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE1Ny0xNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJBbGktUGh5OiBzaW11bHRhbmVvdXMgQmF5ZXNpYW4gaW5mZXJlbmNlIG9mIGFsaWdubWVudCBhbmQgcGh5bG9nZW55OyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTIwNDctMjA0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFVUCo6IHBoeWxvZ2VuZXRpYyBhbmFseXNpcyB1c2luZyBwYXJzaW1vbnkgKCphbmQgb3RoZXIgbWV0aG9kcykuIHZlcnNpb24gNDsgY2l0YXRpb25fcHVibGlzaGVyPVNpbmF1ZXIgQXNzb2NpYXRlcywgU3VuZGVybGFuZCAoTUEpOyAgY2l0YXRpb25feWVhcj0yMDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMIFc6IGltcHJvdmluZyB0aGUgc2Vuc2l0aXZpdHkgb2YgcHJvZ3Jlc3NpdmUgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHRocm91Z2ggc2VxdWVuY2Ugd2VpZ2h0aW5nLCBwb3NpdGlvbi1zcGVjaWZpYyBnYXAgcGVuYWx0aWVzIGFuZCB3ZWlnaHQgbWF0cml4IGNob2ljZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NDY3My00NjgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXByZWhlbnNpdmUgY29tcGFyaXNvbiBvZiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI2ODItMjY5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZvbHV0aW9uYXJ5IG1vZGVsIGZvciBtYXhpbXVtIGxpa2VsaWhvb2QgYWxpZ25tZW50IG9mIEROQSBzZXF1ZW5jZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xMTQtMTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNoaW5nIHRvd2FyZCByZWFsaXR5OiBhbiBpbXByb3ZlZCBsaWtlbGlob29kIG1vZGVsIG9mIHNlcXVlbmNlIGV2b2x1dGlvbjsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTMtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmtzIGJldHdlZW4gbWF4aW11bSBsaWtlbGlob29kIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSB1bmRlciBhIHNpbXBsZSBtb2RlbCBvZiBzaXRlIHN1YnN0aXR1dGlvbjsgQnVsbC4gTWF0aC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz01ODEtNjA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QT1kgdmVyc2lvbiA0OiBwaHlsb2dlbmV0aWMgYW5hbHlzaXMgdXNpbmcgZHluYW1pYyBob21vbG9naWVzOyBDbGFkaXN0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NzItODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFsaWdubWVudCwgcGFyYW1ldGVyIHNlbnNpdGl2aXR5LCBhbmQgdGhlIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBtb2xlY3VsYXIgZGF0YTsgU3lzdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0zMjEtMzMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6YXRpb24gYWxpZ25tZW50OiB0aGUgZW5kIG9mIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBpbiBwaHlsb2dlbmV0aWNzPzsgQ2xhZGlzdGljczsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFMSUdOOiBhIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBwcm9ncmFtOyBKLiBIZXJlZDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTQxNy00MTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIGFsaWdubWVudCBieSBhbGlnbmluZyBhbGlnbm1lbnRzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPWk1NTktaTU2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL3N5c2Jpby9hcnRpY2xlLXBkZi82MS8xLzkwLzI0NTYyOTI0L3N5cjA5NS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSGlnaGx5IGFjY3VyYXRlIGVzdGltYXRpb24gb2YgcGh5bG9nZW5ldGljIHRyZWVzIGZvciBsYXJnZSBkYXRhIHNldHMgaXMgZGlmZmljdWx0LCBpbiBwYXJ0IGJlY2F1c2UgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50cyBtdXN0IGJlIGFjY3VyYXRlIGZvciAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTE2ODAwMDImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkcSr65CNnLUwuRa83t0zczOSm2B4=" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5349" type="hidden">
        <input id="hdnAdDelaySeconds" value="5000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="sticky" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad1_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Supplement_Ad1_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Supplement_Ad1_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f61%2f1%2f90%2f1680002" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f61%2f1%2f90%2f1680002" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126564">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126565">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126562">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126566">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126567">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126568">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126569">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126570">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126563">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126571">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126572">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126573">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126574">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126575">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126576">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126577">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126578">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126579">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126580">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/sysbio" class=" journal-logo-container">

            <img id="logo-SystematicBiology" class="journal-logo" src="sysbio_title1175497244.svg" alt="Systematic Biology">

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.systbio.org/" target="" class="society-logo-container">
                                <img id="logo-SocietyofSystematicBiologists" class="society-logo" src="sysbio_h1-360035926.svg" alt="Society of Systematic Biologists">
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126564">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126565">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126562">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126566">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126567">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126568">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126569">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126570">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126563">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126571">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126572">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126573">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126574">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126575">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126576">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126577">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126578">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1126579">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1126580">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5349" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=67248" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/sysbio" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="sysbio" type="hidden">
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left mobile">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/sysbio/issue/61/1"><img id="issueImage" class="fb-featured-image" src="m_sysbio61_1.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/sysbio/issue/61/1">
                        <div class="volume trailing-comma">Volume 61</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
January 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115126981">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115126981">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115126990">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115126990">M<span class="small-caps">ATERIALS AND</span> M<span class="small-caps">ETHODS</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115127029">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115127029">R<span class="small-caps">ESULTS</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115127060">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115127060">D<span class="small-caps">ISCUSSION</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115127065">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115127065">S<span class="small-caps">UPPLEMENTARY</span> M<span class="small-caps">ATERIAL</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115127067">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115127067">F<span class="small-caps">UNDING</span></a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115127132">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115127132">References</a>
                </div>
            </li>
    <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/sysbio/article/61/1/80/1676942">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/sysbio/article/61/1/107/1680251">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">SATé-II: Very Fast and Accurate Simultaneous Estimation of Multiple Sequence Alignments and Phylogenetic Trees
<i class="icon-availability_free" title="Free"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kevin Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Computer Science, University of Texas at Austin, Austin, TX 78712, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Kevin+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Liu%20K%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kevin Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tandy J. Warnow</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Computer Science, University of Texas at Austin, Austin, TX 78712, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Tandy+J.+Warnow">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Warnow%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Warnow%20T%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tandy J. Warnow</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mark T. Holder</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Ecology and Evolutionary Biology, University of Kansas at Lawrence, Lawrence, KS 66045, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Mark+T.+Holder">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Holder%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Holder%20M%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mark T. Holder</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Serita M. Nelesen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Department of Computer Science, Calvin College, Grand Rapids, MI 49546, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Serita+M.+Nelesen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nelesen%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Nelesen%20S%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Serita M. Nelesen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiaye Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Ecology and Evolutionary Biology, University of Kansas at Lawrence, Lawrence, KS 66045, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Jiaye+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Yu%20J%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jiaye Yu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexandros P. Stamatakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Scientific
 Computing Group, Heidelberg Institute for Theoretical Studies, 
Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Alexandros+P.+Stamatakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stamatakis%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stamatakis%20A%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alexandros P. Stamatakis</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C. Randal Linder</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>Section of Integrative Biology, School of Biological Sciences, University of Texas at Austin, Austin, TX, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>Correspondence to be sent to: The University of Texas at Austin, Austin, TX 78712, USA; E-mail: <a href="mailto:rlinder@mail.utexas.edu" target="">rlinder@mail.utexas.edu</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=C.+Randal+Linder">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Linder%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Linder%20C%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">C. Randal Linder<i class="icon-general-mail"></i></a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Systematic Biology</em>, Volume 61, Issue 1, 1 January 2012, Pages 90, <a href="https://doi.org/10.1093/sysbio/syr095">https://doi.org/10.1093/sysbio/syr095</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 December 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 August 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">09 November 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 June 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1680002" href="https://academic.oup.com/sysbio/article-pdf/61/1/90/24562924/syr095.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Kevin Liu, Tandy J. Warnow, Mark T. Holder, Serita M. Nelesen, 
Jiaye Yu, Alexandros P. Stamatakis, C. Randal Linder;  SATé-II: Very 
Fast and Accurate Simultaneous Estimation of Multiple Sequence 
Alignments and Phylogenetic Trees, <em>Systematic Biology</em>, Volume 61, Issue 1, 1 January 2012, Pages 90, <a href="https://doi.org/10.1093/sysbio/syr095">https://doi.org/10.1093/sysbio/syr095</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=ris">RIS (Zotero)</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=ris">EndNote</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=bibtex">BibTex</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=txt">Medlars</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=ris">ProCite</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=txt">RefWorks</a></li>
        <li><a href="https://academic.oup.com/sysbio/downloadcitation/1680002?format=ris">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1076-836X&amp;title=SAT%C3%A9-II%3A%20Very%20Fast%20and%20Accurate%20Simultaneous%20Estimation%20of%20Multiple%20Sequence%20Alignments%20and%20Phylogenetic%20Trees&amp;publicationDate=2011-12-01&amp;volumeNum=61&amp;issueNum=1&amp;author=Liu%2C%20Kevin%3B%20Warnow%2C%20Tandy%20J.&amp;startPage=90&amp;endPage=&amp;contentId=10.1093%2Fsysbio%2Fsyr095&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f61%2f1%2f90%2f1680002" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="sysbio" type="hidden">
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="115126981" id="115126981" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Highly accurate estimation of phylogenetic 
trees for large data sets is difficult, in part because multiple 
sequence alignments must be accurate for phylogeny estimation methods to
 be accurate. Coestimation of alignments and trees has been attempted 
but currently only SATé estimates reasonably accurate trees and 
alignments for large data sets in practical time frames (Liu K., 
Raghavan S., Nelesen S., Linder C.R., Warnow T. 2009b. Rapid and 
accurate large-scale coestimation of sequence alignments and 
phylogenetic trees. Science. 324:1561–1564). Here, we present a 
modification to the original SATé algorithm that improves upon SATé 
(which we now call SATé-I) in terms of speed and of phylogenetic and 
alignment accuracy. SATé-II uses a different divide-and-conquer strategy
 than SATé-I and so produces smaller more closely related subsets than 
SATé-I; as a result, SATé-II produces more accurate alignments and 
trees, can analyze larger data sets, and runs more efficiently than 
SATé-I. Generally, SATé is a metamethod that takes an existing multiple 
sequence alignment method as an input parameter and boosts the quality 
of that alignment method. SATé-II-boosted alignment methods are 
significantly more accurate than their unboosted versions, and trees 
based upon these improved alignments are more accurate than trees based 
upon the original alignments. Because SATé-I used maximum likelihood 
(ML) methods that treat gaps as missing data to estimate trees and 
because we found a correlation between the quality of tree/alignment 
pairs and ML scores, we explored the degree to which SATé's performance 
depends on using ML with gaps treated as missing data to determine the 
best tree/alignment pair. We present two lines of evidence that using ML
 with gaps treated as missing data to optimize the alignment and tree 
produces very poor results. First, we show that the optimization problem
 where a set of unaligned DNA sequences is given and the output is the 
tree and alignment of those sequences that maximize likelihood under the
 Jukes–Cantor model is uninformative in the worst possible sense. For 
all inputs, all trees optimize the likelihood score. Second, we show 
that a greedy heuristic that uses GTR+Gamma ML to optimize the alignment
 and the tree can produce very poor alignments and trees. Therefore, the
 excellent performance of SATé-II and SATé-I is not because ML is used 
as an optimization criterion for choosing the best tree/alignment pair 
but rather due to the particular divide-and-conquer realignment 
techniques employed.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Alignment">Alignment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;maximum likelihood&quot;">maximum likelihood</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phylogenetics">phylogenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SATé">SATé</a></div><div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/sysbio/search-results?f_TocHeadingTitle=Regular%20articles">Regular articles</a>
        </div></div>
<p>Phylogenetic estimation using nucleotide and amino acid sequences 
typically proceeds in two phases: a multiple sequence alignment is 
produced for a set of unaligned sequences and then a tree is estimated 
using the multiple sequence alignment. Given sufficient computational 
resources, these “two-phase” methods can analyze data sets of thousands 
of taxa and aligned sites but can have poor accuracy on data sets that 
have evolved under high substitution and indel (insertion and deletion) 
rates (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. 2009</a>, <a reveal-id="bib19" data-open="bib19" class="link link-ref link-reveal xref-bibr">Liu et al. 2010</a>).</p><p>Methods
 that simultaneously estimate a multiple sequence alignment and a 
phylogenetic tree from unaligned sequences have been developed over the 
course of two decades, starting in the 1990s (<a reveal-id="bib9" data-open="bib9" class="link link-ref link-reveal xref-bibr">Hein 1990</a>,
 Wheeler09011994, Wheeler95, wheeler96.) Today's methods are broadly 
classified into two categories: methods based upon parametric 
statistical models of sequence evolution that incorporate substitution 
and indel events (e.g., TKF1 (<a reveal-id="bib38" data-open="bib38" class="link link-ref link-reveal xref-bibr">Thorne et al. 1991</a>) and TKF2 (<a reveal-id="bib39" data-open="bib39" class="link link-ref link-reveal xref-bibr">Thorne et al. 1992</a>))
 and nonparametric methods that resemble maximum parsimony. Current 
methods that fall into the parametric category include BAli-Phy (<a reveal-id="bib26" data-open="bib26" class="link link-ref link-reveal xref-bibr">Redelings and Suchard 2005</a>, <a reveal-id="bib34" data-open="bib34" class="link link-ref link-reveal xref-bibr">Suchard and Redelings 2006</a>), StatAlign (Novák et al. 2008), the method published by <a reveal-id="bib22" data-open="bib22" class="link link-ref link-reveal xref-bibr">Lunter et al. (2005)</a>, and ALIFRITZ (<a reveal-id="bib5" data-open="bib5" class="link link-ref link-reveal xref-bibr">Fleissner et al. 2005</a>).
 Methods in the second category, by contrast, typically attempt to 
minimize the total number of evolutionary events or the total cost of 
these events. The most frequently used method of this type is POY (<a reveal-id="bib41" data-open="bib41" class="link link-ref link-reveal xref-bibr">Varón et al. 2010</a>) but the basic approach was introduced by <a reveal-id="bib29" data-open="bib29" class="link link-ref link-reveal xref-bibr">Sankoff (1975)</a>, and there are other methods of this type (<a reveal-id="bib30" data-open="bib30" class="link link-ref link-reveal xref-bibr">Sankoff and Cedergren 1983</a>, <a reveal-id="bib17" data-open="bib17" class="link link-ref link-reveal xref-bibr">Lancia and Ravi 1999</a>, <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Nelesen et al. 2009</a>). Of all these methods for simultaneous estimation of alignments and trees, only SATé (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al., 2009</a>)
 has been demonstrated to produce alignments and trees that are more 
accurate than two-phase methods on large data sets. The Mega-phylogeny 
method (<a reveal-id="bib31" data-open="bib31" class="link link-ref link-reveal xref-bibr">Smith et al. 2009</a>) can analyze very large data sets and is potentially promising; however, it has not yet been compared with other methods.</p><p>SATé uses an iterative procedure to compute a series of alignment/tree pairs. The first tree is produced by running RAxML (<a reveal-id="bib32" data-open="bib32" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>) on a MAFFT (<a reveal-id="bib12" data-open="bib12" class="link link-ref link-reveal xref-bibr">Katoh et al. 2005</a>, <a reveal-id="bib14" data-open="bib14" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2008</a>)
 alignment, running each method in its default (and most accurate) 
setting. To compute a new alignment, the current tree is used to produce
 a division of the sequence data set into disjoint subsets; MAFFT 
alignments are computed on each subset, and these alignments are merged 
into an alignment on the entire data set using Muscle (Edgar 2004a, <a reveal-id="bib4" data-open="bib4" class="link link-ref link-reveal xref-bibr">Edgar 2004b</a>); finally, a GTR+Gamma (<a reveal-id="bib27" data-open="bib27" class="link link-ref link-reveal xref-bibr">Rodriguez et al. 1990</a>)
 ML tree is computed using RAxML on the new alignment. If the ML score 
improves, the new alignment/tree pair is accepted; otherwise, SATé 
realigns the sequences using a somewhat different technique. This 
process repeats until a stopping criterion is satisfied, and the 
alignment/tree pair is returned that had the best GTR+Gamma ML score. On
 moderate to difficult data sets (sequences with high levels of 
substitution and/or indels), SATé produces much more accurate alignments
 and trees than existing two-phase techniques and does so fairly rapidly
 (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. 2009</a>).
 However, the topological accuracy of trees estimated using SATé on data
 simulated under difficult model conditions differs significantly from 
the accuracy of trees produced using maximum likelihood (ML) estimation 
on the true alignment, suggesting that further improvement might still 
be obtained through a re-design of the algorithm.</p><p>Because the 
sequence evolution model for ML phylogeny estimation used in SATé is 
GTR+Gamma with gaps treated as missing data, SATé is not attempting to 
solve ML under a sequence evolution model that includes indels. This 
suggests that SATé is not likely to be statistically consistent under a 
model in which sequences evolve with indels. Even so, SATé has very good
 performance on simulated data under a wide range of model conditions (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. 2009</a>).
 Therefore, although SATé uses ML estimation to select among its 
generated alignment/tree pairs, it remains an open question whether the 
good performance of SATé is due to its ability to optimize alignments 
and trees under GTR+Gamma or to some other aspect of its algorithm 
design.</p><p>In this paper, we make progress on understanding why SATé 
produces excellent results. We show definitively that the reason SATé is
 highly accurate is <em>not</em> due to the use of the ML criterion to 
select among the alignment/tree pairs it generates. We show empirically 
that whether SATé uses ML to choose among tree/alignment pairs or simply
 takes the last tree alignment pair generated after at least three to 
five iterations has little or no effect on the topological accuracy of 
the tree produced. In addition, we provide a mathematical proof that <em>all trees are optimal solutions for all inputs</em> when optimizing the Jukes–Cantor (JC) (<a reveal-id="bib11" data-open="bib11" class="link link-ref link-reveal xref-bibr">Jukes and Cantor 1969</a>) ML score—treating gaps as missing data—while allowing the alignment and tree to change arbitrarily (&nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>).
 Finally, we show that a greedy heuristic that seeks to optimize the 
GTR+Gamma ML score (again treating gaps as missing data) can produce 
very poor alignments and trees (&nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>).</p><p>These
 insights led us to explore the algorithmic techniques in SATé and to 
the design of a new method, SATé-II. We focused on the decomposition 
technique used in SATé and observed that (i) on some large data sets, it
 produced very large subsets and (ii) these subsets sometimes contained 
highly dissimilar sequences. We therefore changed the decomposition 
technique so that all subsets were small (at most 200 sequences), which 
improved both the algorithm's speed and the accuracy of the alignments. 
We also explored other modifications (see below) that improved the 
method but to a lesser degree.</p><p>We show that SATé-II produces more 
accurate alignments and trees than SATé (which we now call SATé-I) and 
two-phase methods. Furthermore, SATé-II is designed so it can be used 
with any alignment method—not just MAFFT—so it is now a metamethod that 
takes an existing multiple sequence alignment method as an input 
parameter. We show that alignment methods boosted by SATé-II are 
significantly more accurate than their unboosted versions and that trees
 based upon these improved alignments are more accurate than trees based
 upon the unboosted alignments. Importantly, SATé-II can be used to 
analyze much larger data sets than SATé-I, and we demonstrate its 
scalability and accuracy on biological data sets with up to almost 
28,000 sequences. Thus, SATé-II provides, for the first time, an 
automated method that is capable of producing highly accurate sequence 
alignments and trees for very large molecular data sets.</p>                            <h2 scrollto-destination="115126990" id="115126990" class="section-title">M<span class="small-caps">ATERIALS AND</span> M<span class="small-caps">ETHODS</span></h2>
                            <h3 scrollto-destination="115126991" id="115126991" class="section-title">Algorithm and Study Design</h3>
<p>We had two objectives in this study. The first objective was to 
understand which aspects of SATé-I's algorithm design contributed the 
most to its tree and alignment accuracy and, in particular, to 
understand the extent to which using ML to select the best 
alignment/tree pair contributed to SATé-I's accuracy. The second 
objective was to use what we learned about the impact of the algorithmic
 design on phylogenetic accuracy to produce a more accurate coestimation
 method.</p>                            <h4 scrollto-destination="115126993" id="115126993" class="section-title">Designing SATé-II.—</h4>
<p>In designing SATé-II, we sought to improve the quality of its 
alignments, the topological accuracy of its trees, and its speed over 
SATé-I. To make the changes clear, we first describe SATé-I and then 
show how SATé-II differs.</p>                            <h4 scrollto-destination="115126995" id="115126995" class="section-title">SATé-I algorithm.—</h4>
<p>SATé-I produces a starting tree by estimating a set of alignments on a
 data set, computing GTR+Gamma ML trees using RAxML on each alignment 
and then keeping the tree with the best GTR+Gamma ML score. SATé-I then 
iterates, where each iteration begins using a divide-and-conquer 
strategy, guided by the current tree, to produce nonoverlapping subsets 
of taxa. These subsets are realigned using MAFFT (<a reveal-id="bib12" data-open="bib12" class="link link-ref link-reveal xref-bibr">Katoh et al. 2005</a>, <a reveal-id="bib14" data-open="bib14" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2008</a>) and then merged using Muscle (Edgar 2004a, <a reveal-id="bib4" data-open="bib4" class="link link-ref link-reveal xref-bibr">2004b</a>)
 to produce a new alignment on the full set of taxa. The iteration 
finishes by computing a GTR+Gamma ML tree on the new alignment using 
RAxML.</p><p>The divide-and-conquer strategy used in SATé-I is the CT-5 (center tree-5) decomposition (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. 2009</a>). The CT-5 decomposition takes a branch <em>e</em> in the input tree <em>T</em> and extends outwards from this branch to define a binary tree <em>t</em> within <em>T</em> with up to 32 (i.e., 2<sup>5</sup>) leaves by including all nodes in <em>T</em> that are at most 4 branches distant from <em>e</em>. (More generally, a CT-<em>k</em> decomposition takes an internal branch and extends outwards, creating a subtree that contains all nodes at most <em>k</em> − 1 branches distant from <em>e</em>.)
 Once the subtree based upon a CT-5 decomposition is defined, removing 
it from the original tree separates the leaf set into at most 32 
subsets. SATé-I then computes alignments on each subset using MAFFT and 
merges these alignments into a single alignment on the entire data set 
using Muscle. Finally, SATé-I computes a GTR+Gamma ML tree on this 
alignment using RAxML. This method of producing subsets of taxa does not
 constrain the maximum number of taxa in a subset.</p>                            <h4 scrollto-destination="115126998" id="115126998" class="section-title">SATé-II algorithm.—</h4>
<p>SATé-II differs from SATé-I in three ways, the first two of which 
have the greatest effect on alignment and tree accuracy. Most 
importantly, SATé-II uses a different divide-and-conquer strategy. The 
CT-5 decomposition used in SATé-I sometimes produces very large subsets,
 and these subsets sometimes contain very dissimilar sequences, often 
negatively impacting alignment and tree accuracy. In contrast, SATé-II 
divides the sequences into two sets by removing the longest branch in 
the current tree (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>),
 where branch lengths are given by the ML tree on the previously 
computed alignment. Then SATé-II recurses, selecting the longest branch 
in each subset to further subdivide the taxa, until each subset has at 
most 200 taxa. (Note that this strategy always performs at least one 
realignment on all data sets because the first decomposition is always 
performed, no matter how small the tree is.) This decomposition 
technique is guaranteed to produce small subsets and ones that are less 
likely to contain very dissimilar sequences. Second, for merging 
subalignments, we determined that Opal (<a reveal-id="bib45" data-open="bib45" class="link link-ref link-reveal xref-bibr">Wheeler and Kececioglu 2007</a>)
 produced more accurate merges than Muscle, leading to more accurate 
alignments and trees. Finally, SATé-II uses only the RAxML tree on the 
MAFFT alignment as its starting tree, rather than computing several 
possible starting trees. This improves running time without affecting 
the final accuracy of the final alignment and tree.</p>                        <a id="115127000" scrollto-destination="115127000"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 1.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f01_lw.gif" alt="A cartoon of a SATé-II iteration. The tree T is divided into two subtrees T1 and T2 using a longest branch e in T, and this process is repeated until each subtree has no more than the maximum allowable number of leaves; here, we show this process producing four subtrees T11,T12,T21, and T22. Then the sequences at the leaves in each subtree are aligned, and the resultant alignments on subsets are merged, pairwise, in the reverse order in which they were created. Finally, a new ML tree T′ is estimated on the resultant alignment." path-from-xml="sysbiosyr095f01_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127000/sysbiosyr095f01_lw.gif" path-from-xml="sysbiosyr095f01_lw.gif" target="_blank">View large</a><a section="115127000" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f01_lw.gif?Expires=2147483647&amp;Signature=H9Zb-18p33SmXFJH-IdvQTMUyPNDjoZEhDdP9ASpJUZgXthC8bUMgRCgkuI40p6TkMhxk2IluBtOO7Wlpof5U6HXwvr6qDZ0M-Q3k-YulRScmu9Nlox4qtMlisaSUUmI5o3BNQ1Gcsu%7Esk1gogepteAuUi053UaUbu0tPQwkpDQ5%7Ej1t62SCavMZhsd8Xmk8oOfumfdy0dhuqL6zoAKM2-yxiAQVvnlmMq1mDubckHTcns5lJrfUGqWgOYqQq-0GzaXbbpF0zcsedH4Lka4pkaR%7Eo5Yb5A-nlYDQyNJXQR6LWLdOJ4lgxxGmpwD65wsaw7jBv3Ah-UqAz5K7e7rysA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127000&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f01_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>A cartoon of a SATé-II iteration. The tree <em>T</em> is divided into two subtrees <em>T</em><sub>1</sub> and <em>T</em><sub>2</sub> using a longest branch <em>e</em> in <em>T</em>,
 and this process is repeated until each subtree has no more than the 
maximum allowable number of leaves; here, we show this process producing
 four subtrees <em>T</em><sub>11</sub>,<em>T</em><sub>12</sub>,<em>T</em><sub>21</sub>, and <em>T</em><sub>22</sub>.
 Then the sequences at the leaves in each subtree are aligned, and the 
resultant alignments on subsets are merged, pairwise, in the reverse 
order in which they were created. Finally, a new ML tree <em>T</em><sup>′</sup> is estimated on the resultant alignment.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 1.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f01_lw.gif" alt="A cartoon of a SATé-II iteration. The tree T is divided into two subtrees T1 and T2 using a longest branch e in T, and this process is repeated until each subtree has no more than the maximum allowable number of leaves; here, we show this process producing four subtrees T11,T12,T21, and T22. Then the sequences at the leaves in each subtree are aligned, and the resultant alignments on subsets are merged, pairwise, in the reverse order in which they were created. Finally, a new ML tree T′ is estimated on the resultant alignment." path-from-xml="sysbiosyr095f01_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127000/sysbiosyr095f01_lw.gif" path-from-xml="sysbiosyr095f01_lw.gif" target="_blank">View large</a><a section="115127000" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f01_lw.gif?Expires=2147483647&amp;Signature=H9Zb-18p33SmXFJH-IdvQTMUyPNDjoZEhDdP9ASpJUZgXthC8bUMgRCgkuI40p6TkMhxk2IluBtOO7Wlpof5U6HXwvr6qDZ0M-Q3k-YulRScmu9Nlox4qtMlisaSUUmI5o3BNQ1Gcsu%7Esk1gogepteAuUi053UaUbu0tPQwkpDQ5%7Ej1t62SCavMZhsd8Xmk8oOfumfdy0dhuqL6zoAKM2-yxiAQVvnlmMq1mDubckHTcns5lJrfUGqWgOYqQq-0GzaXbbpF0zcsedH4Lka4pkaR%7Eo5Yb5A-nlYDQyNJXQR6LWLdOJ4lgxxGmpwD65wsaw7jBv3Ah-UqAz5K7e7rysA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127000&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f01_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>A cartoon of a SATé-II iteration. The tree <em>T</em> is divided into two subtrees <em>T</em><sub>1</sub> and <em>T</em><sub>2</sub> using a longest branch <em>e</em> in <em>T</em>,
 and this process is repeated until each subtree has no more than the 
maximum allowable number of leaves; here, we show this process producing
 four subtrees <em>T</em><sub>11</sub>,<em>T</em><sub>12</sub>,<em>T</em><sub>21</sub>, and <em>T</em><sub>22</sub>.
 Then the sequences at the leaves in each subtree are aligned, and the 
resultant alignments on subsets are merged, pairwise, in the reverse 
order in which they were created. Finally, a new ML tree <em>T</em><sup>′</sup> is estimated on the resultant alignment.</p></div></div><p>Using
 this basic algorithm as a foundation, we designed three variants of 
SATé-II (SATé-II simple, SATé-II ML, and SATé-II fast) to determine the 
fastest and most accurate versions, as well as to directly test the 
impact of using the ML score under GTR+Gamma (treating gaps as missing 
data) to select among the tree/alignment pairs generated during the 
SATé-II iterations. The variants differ in the number of iterations 
performed and the criterion used to select the tree/alignment pair to be
 returned from the set of generated tree/alignment pairs. However, each 
method uses the same iterative strategy, with each iteration producing a
 new alignment and then an ML tree on that new alignment. SATé-II simple
 runs until a time-limited or iteration-limited stopping rule of the 
user's choosing is satisfied, returning the final tree/alignment pair 
produced by the iterative strategy. The use of ML within SATé-II simple 
is limited to the generation of trees from estimated alignments and is 
not used as a criterion to determine the final tree/alignment pair. 
SATé-II ML performs the same iterations as SATé-II simple but compares 
all tree/alignment pairs produced and returns the tree with the best 
GTR+Gamma ML score and the alignment from which it was estimated. The 
third variant, SATé-II fast, runs only until an iteration produces a 
worse ML score than the previous iteration; it then stops and returns 
the penultimate tree/alignment pair.</p>                            <h3 scrollto-destination="115127002" id="115127002" class="section-title">Data Sets for the Study</h3>
<p>We used simulated data sets to explore the performance of SATé-II's 
variants relative to one another and in comparison to SATé-I and 
two-phase methods. We also used simulated data sets to evaluate the 
heuristic we developed to directly optimize ML on both the alignment and
 the tree. We used biological data sets to evaluate the final SATé-II 
algorithm in comparison to SATé-I and two-phase methods.</p>                            <h4 scrollto-destination="115127004" id="115127004" class="section-title">Simulated data sets.—</h4>
<p>Our simulated data sets included the data sets originally generated to evaluate SATé-I (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al., 2009</a>),
 and ranged from 100 to 1000 taxa. We varied rates of substitutions and 
indels, and included different gap length distributions, to produce a 
diverse set of data sets of varying levels of difficulty. Since the 
100-taxon data sets in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>
 were relatively easy to align, we added some more difficult 100-taxon 
models that were produced using the same methodology as in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a> (Table S1 in online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>, available from <a class="link link-uri" href="http://www.sysbio.oxfordjournals.org/" target="">http://www.sysbio.oxfordjournals.org/</a>). We used r8s (<a reveal-id="bib28" data-open="bib28" class="link link-ref link-reveal xref-bibr">Sanderson 2003</a>) version 1.7 to generate nonultrametric birth–death model trees and ROSE (<a reveal-id="bib33" data-open="bib33" class="link link-ref link-reveal xref-bibr">Stoye et al. 1998</a>) version 1.3 to simulate evolution of sequences down the model trees (see &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a> for specific commands).</p><p>All simulated data sets were produced by simulating evolution with indels and substitutions, under the GTR+Gamma+Gap model (see <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a> and &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a>).
 Therefore, substitutions were drawn from the GTR+Gamma model, and gap 
probabilities were scaled with substitution probabilities. We simulated 
under three different gap length distributions (short, medium, and 
long), three different numbers of taxa (100, 500, and 1000), and five 
different combinations of tree height and gap probability. In total, we 
simulated under 45 model conditions. For each model condition, we 
generated 20 model trees, and we then generated one data set for each 
model tree. Thus, we produced 900 data sets overall. Some of these model
 conditions are “easy” because SATé-I and all the two-phase methods 
produced trees with missing branch rates (i.e., the false-negative rate,
 which is the percentage of the branches in the true tree that are not 
present in the estimated tree) essentially identical to that of RAxML on
 the true alignment (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. 2009</a>). For the other model conditions, the two-phase methods differed in their abilities to produce accurate trees.</p><p>For
 each simulated data set, we reported various values, as follows. We 
compared each estimated alignment to the true alignment using the 
alignment SP-FN error rate, which is the percentage of the number of 
homologous pairs of nucleotides in the true alignment that are missing 
in the estimated alignment (<a reveal-id="bib37" data-open="bib37" class="link link-ref link-reveal xref-bibr">Thompson et al. 1999</a>). The topology of each estimated tree was compared to that of the <em>potentially inferable model tree</em>
 (PIMT), which consisted of the model tree with all zero-event branches 
(i.e., branches on which no sequence change occurred) contracted; using 
PIMTs prevents methods from being penalized for failing to recover 
branches that cannot be inferred from the data. We report the missing 
branch rates (the percentage of branches in the true tree that do not 
appear in the estimated tree) and false-positive rates (the percentage 
of branches in the estimated tree that do not appear in the true tree) 
of the estimated trees with respect to their PIMTs. Finally, we report 
the likelihood scores returned by RAxML, normalized by the likelihood 
score produced by RAxML on the MAFFT alignment for that data set. 
Normalized likelihood scores less than 1.0 indicate alignments for which
 RAxML produced a tree with a better likelihood than it produced using 
an MAFFT alignment, whereas normalized likelihood scores greater than 
1.0 indicate alignments for which RAxML produced a tree with a 
likelihood score worse than that it produced using a MAFFT alignment.</p>                            <h4 scrollto-destination="115127008" id="115127008" class="section-title">Biological data sets.—</h4>
<p>We explored performance on the six data sets studied in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>
 and two much larger data sets from Cannone et al. (2002): the 16S.T 
data set (rRNA sequences from the three domains of life and two 
organelles) and the 16S.B.ALL data set (rRNA sequences from the Bacteria
 domain). All these data sets are from Robin Gutell's rRNA Web site (<a class="link link-uri" href="http://www.rna.ccbb.utexas.edu/DAT/3C/Alignment/" target="">http://www.rna.ccbb.utexas.edu/DAT/3C/Alignment/</a>)
 and have the virtue of having their curated alignments inferred using 
sequence covariation, secondary structure, and higher level structural 
features (<a reveal-id="bib7" data-open="bib7" class="link link-ref link-reveal xref-bibr">Gutell et al. 1994</a>, <a reveal-id="bib2" data-open="bib2" class="link link-ref link-reveal xref-bibr">Cannone et al. 2002</a>, <a reveal-id="bib8" data-open="bib8" class="link link-ref link-reveal xref-bibr">Gutell et al. 2002</a>).
 While no alignment of biological data can be guaranteed to be 
completely accurate, because of the approaches employed by Gutell, the 
biological data sets we used are considered some of the most reliable 
curated nucleic acid sequence alignments.</p><p>We “cleaned” the 16S.T and 16S.B.ALL data sets using the same techniques employed in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>
 to remove sites occupied entirely by gaps and taxa containing more than
 50% unsequenced characters. Cleaning removed 3% of the 16S.T sites and 
11.8% of the 16S.B.ALL sites. No taxa were removed from the 16S.T data 
set, but 23.2% of the taxa in the 16S.B.ALL data set were removed. Table
 S2 lists cleaning statistics. The cleaned 16S.T curated alignment had 
7350 sequences, 11,856 aligned sites, 34.5% average <em>p</em>-distance, 90.1% maximum <em>p</em>-distance,
 87.4% indels, and average gap length of 12.1. The cleaned 16S.B.ALL 
data set had 27,643 sequences, 6857 aligned sites, 21.0% average <em>p</em>-distance, 76.9% maximum <em>p</em>-distance,
 80.0% indels, and average gap length of 4.9. Thus, both the 16S.T and 
16S.B.ALL data sets present substantial challenges to alignment 
estimators, due to the curated alignments' substitution saturation and 
high gappiness.</p><p>We computed curation-based trees on the biological
 data sets as follows. First, we ran RAxML on each curated alignment to 
produce an initial topology. We then determined the support on each edge
 using a bootstrap analysis (RAxML) on the curated alignments. The 
number of bootstrap replicates depended upon the data set: 346 
replicates on the 16S.T data set, 444 replicates on the 16S.B.ALL data 
set, and 500 bootstrap replicates on the other data sets. Branches with 
at least 75% bootstrap support were retained in the curation-based tree (<a reveal-id="bib10" data-open="bib10" class="link link-ref link-reveal xref-bibr">Hillis and Bull 1993</a>). Table S3 in the online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a> gives these statistics for all the biological data sets.</p><p>To
 evaluate estimated alignments on biological data sets, we compared them
 to the curated alignments using the alignment SP-FN error rate. To 
evaluate estimated trees, we used the missing branch rate to compare 
their topologies to the curation-based trees. We recognize that 
comparison of estimated trees with the curation-based trees is not a 
measure of the true accuracy of the estimated trees; however, given the 
high quality of the rRNA-curated alignments we used, the comparison is 
likely to be informative about the quality of the estimated trees, 
particularly because low-support branches are excluded from the 
comparison.</p>                            <h3 scrollto-destination="115127013" id="115127013" class="section-title">Two-Phase Methods</h3>
<p>We aligned all the simulated and biological data sets—with the 
exception of the two largest biological data sets, 16S.T and 
16S.B.ALL—using ClustalW (<a reveal-id="bib36" data-open="bib36" class="link link-ref link-reveal xref-bibr">Thompson et al. 1994</a>) version 2.0.4, MAFFT version 6.240, Muscle version 3.7, Prank+GT (as described in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>),
 and Opal version 1.0.2. Each alignment method was run in its default 
setting. For each resulting alignment, we ran RAxML to estimate an ML 
tree.</p><p>Due to the high number of taxa in the 16S.T and 16S.B.ALL data sets, we used the PartTree (<a reveal-id="bib13" data-open="bib13" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2007</a>) algorithm in MAFFT and the Quicktree (<a reveal-id="bib36" data-open="bib36" class="link link-ref link-reveal xref-bibr">Thompson et al. 1994</a>)
 option in ClustalW to estimate multiple sequence alignments and RAxML 
to estimate ML trees on each alignment. On the 16S.B.ALL data set, we 
also used FastTree (<a reveal-id="bib25" data-open="bib25" class="link link-ref link-reveal xref-bibr">Price et al. 2010</a>) to estimate an ML tree on the PartTree alignment. &nbsp;<a class="js-jumplink" href="#app3">Appendix 3</a> provides complete details, including program versions and commands, for our experiments with these methods.</p>                            <h3 scrollto-destination="115127016" id="115127016" class="section-title">Variations of SATé-I and SATé-II Studied</h3>
<p>Except for the two largest data sets (16S.T and 16S.B.ALL), we ran 
SATé-I and SATé-II in their default settings. The default settings for 
SATé-I follow those given in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>, running SATé-I until 24 h had passed, at which point we returned the last tree.</p><p>Both
 SATé-I and SATé-II analyses of the two largest data sets were modified 
to enable them to complete. First, instead of the starting tree being 
based upon an MAFFT alignment, we based it on the PartTree (<a reveal-id="bib13" data-open="bib13" class="link link-ref link-reveal xref-bibr">Katoh and Toh 2007</a>)
 alignment because MAFFT cannot run on data sets of these sizes. Of the 
alignment methods that can handle very large numbers of taxa, the 
PartTree method yields among the most accurate alignments in our 
experience (<a reveal-id="bib19" data-open="bib19" class="link link-ref link-reveal xref-bibr">Liu et al. 2010</a>).</p><p>To
 compute the starting tree using SATé-I, we used RAxML on the PartTree 
alignment. For the SATé-II analysis of the 16S.T data set, we used RAxML
 to compute the ML tree on the PartTree alignment. For the SATé-II 
analysis of the 16S.B.ALL data set, we used FastTree (a less accurate 
but faster ML method) (<a reveal-id="bib25" data-open="bib25" class="link link-ref link-reveal xref-bibr">Price et al. 2010</a>)
 to estimate an ML tree on the PartTree alignment. Thus, for the 16S.T 
data set, SATé-I and SATé-II started with the same starting tree, 
whereas for the 16S.B.ALL data set, SATé-II started with a less accurate
 (but more quickly computed) starting tree. We made additional 
modifications to SATé-I and SATé-II on the 16S.B.ALL data set, due to 
its large size. We used Muscle to merge subalignments produced by both 
SATé-I and SATé-II since Opal failed to run for such large data sets. We
 also used FastTree instead of RAxML inside each iteration of SATé-II to
 compute trees on each newly estimated alignment.</p><p>The 
modifications to SATé-II for the largest data sets had the effect of 
making SATé-II run faster and use less memory than SATé-I but might also
 have reduced topological and alignment accuracy. For example, on the 
largest data set (16S.B.ALL), SATé-II started with a less accurate 
starting tree and used a less accurate ML estimation technique within 
its iterative strategy than SATé-I. Since our objective was to compare 
SATé-II to SATé-I primarily with respect to tree and alignment error, 
the effect of these modifications likely biases the comparison in favor 
of SATé-I rather than in favor of SATé-II.</p>                            <h3 scrollto-destination="115127021" id="115127021" class="section-title">Understanding the Role of ML in SATé</h3>
<p>ML has two roles in SATé-I: It is used in each iteration to compute a
 tree on each new alignment and also used to select an alignment/tree 
pair from the sequence of such pairs produced during the iterative 
strategy. Within the iterative procedure where the tree is inferred, ML 
is used rather than other phylogenetic method estimators—for example, 
maximum parsimony and neighbor joining—because the alternatives would 
likely produce less accurate trees, particularly on data sets with high 
rates of substitution. RAxML, in particular, is used because it is able 
to analyze large data sets and is known to do a good job of optimizing 
likelihood scores.</p><p>With respect to the second use of ML—selecting a
 single alignment/tree pair from the sequence of such pairs—it is less 
clear that it is helping to find the best alignment/tree pair. We 
therefore, undertook two studies to test whether optimizing likelihood 
on a tree/alignment pair will produce better alignments and trees. We 
provide a mathematical proof that seeking to maximize the JC likelihood 
score (treating gaps as missing data) and allowing the alignment to 
change is a poor optimality criterion because all trees are optimal 
solutions (see &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>). It is still unknown whether this statement is true or false for the GTR model (see &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>).</p><p>We
 also explored whether directly seeking to produce an alignment and tree
 that optimized the GTR+Gamma ML score, treating gaps as missing data, 
was a poor optimization approach for phylogeny estimation. We designed a
 simple, greedy heuristic to optimize GTR+Gamma ML scores, and tested it
 on some of our 1000 taxon simulated data sets (see online &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a>).</p>                            <h3 scrollto-destination="115127025" id="115127025" class="section-title">Computational Resources</h3>
<p>To perform simulations and run the two-phase methods on simulated and
 biological data sets—other than the 16S.T and 16S.B.ALL data sets—we 
used a heterogeneous Condor cluster (<a reveal-id="bib18" data-open="bib18" class="link link-ref link-reveal xref-bibr">Litzkow 1987</a>)
 at the University of Texas at Austin, consisting of computers generally
 comparable to a 3.0 Ghz Intel 32-bit CPU with between 512 MB and 4 GB 
of main memory. For the SATé-I and SATé-II runs on simulated and 
biological data sets—other than the 16S.T and 16S.B.ALL data sets—we 
used the Mastodon computing cluster at the University of Texas at 
Austin, consisting of Intel 32-bit CPUs running at 2.6, 2.66, or 3.06 
Ghz with 4 GB main memory per CPU.</p><p>To run SATé-II on the 16S.T and
 16S.B.ALL data sets and to run SATé-I on the 16S.T data set, we used a 
64-bit computing cluster at the University of Texas at Austin, 
consisting of machines with 8-core 2.83 Ghz Intel Xeon 64-bit CPUs with 
32 GB main memory per CPU. Due to the memory requirements of SATé-I and 
the two-phase methods on the 16S.B.ALL data sets, we used two machines 
with very large shared memory, each having a 16-core 64-bit AMD Opteron 
CPU running at 2.5 Ghz and with either 128 GB or 256 GB main memory. 
This machine was also used to construct the curation-based tree for the 
16S.T and 16S.B.ALL data set.</p><p>For the experiment with the greedy 
heuristic, we used the SSE3-based SIMD sequential version of RAxML to 
conduct all computational experiments on a Sun x4600 system with 32 
cores and 64 GB of main memory. Finally, to assess support on the 
curation-based tree on the curated alignment for the 16S.B.ALL data set,
 we performed a bootstrap analysis using the Ranger supercomputer at the
 Texas Advanced Computing Center (TACC) at the University of Texas at 
Austin. The Ranger supercomputer has 3936 SunBlade x6420 16-core 2.3 Ghz
 compute nodes for a total of 62,976 processing cores and 123 TB of main
 memory.</p>                            <h2 scrollto-destination="115127029" id="115127029" class="section-title">R<span class="small-caps">ESULTS</span></h2>
                            <h3 scrollto-destination="115127030" id="115127030" class="section-title">Simulated Data</h3>
                            <h4 scrollto-destination="115127031" id="115127031" class="section-title">Role of ML.—</h4>
<p>We began by assessing SATé-II simple and SATé-II ML to determine 
whether using ML to choose the best tree/alignment pair was the cause of
 SATé's excellent performance. SATé-II ML and SATé-II simple were run 
for 1 week on the 1000-taxon data sets and 3 days on the 500- and 
100-taxon data sets. We tested for statistically significant differences
 (false-discovery rate corrected (<a reveal-id="bib1" data-open="bib1" class="link link-ref link-reveal xref-bibr">Benjamini and Hochberg 1995</a>) pairwise <em>t</em>-tests)
 in the final trees and alignments on the moderate to difficult data 
sets. Although SATé-II ML had statistically better ML scores than 
SATé-II simple (as would be expected since SATé-II ML is explicitly 
optimizing the ML score), the differences in their final trees' 
topological accuracy was not statistically significant (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> for the 1000-taxon moderate-to-difficult data sets and Fig. S1 in online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>
 for the 100- and 500-taxon data sets of similar difficulty). This 
result shows clearly that using ML to choose the best tree/alignment 
pair is not responsible for the quality of the tree produced by SATé. 
Interestingly, SATé-II ML did produce alignments that had lower SP-FN 
errors than SATé-II simple, and although these improvements were 
statistically significant, they were quite small. At this time, it is 
not clear why choosing the best ML score affects alignment accuracy, but
 evidently, the small improvement in alignment accuracy is insufficient 
to produce an improvement in the tree topology.</p>                        <a id="115127033" scrollto-destination="115127033"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 2.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f02_ht.gif" alt="Comparison of 1-week runs of SATé-II simple and SATé-II ML on simulated 1000-taxon moderate-to-difficult data sets. Graphs from top to bottom are (a) normalized ML scores, (b) missing branch rates, and (c) alignment SP-FN errors. For each data point, n = 20; bars are standard errors. For normalized ML score, missing branch rate and alignment SP-FN error charts, model conditions marked with an asterisk (*) indicate that SATé-II ML was significantly better than SATé-II simple by Benjamini and Hochberg (1995) corrected pairwise t-tests (n = 40, α = 0.05)." path-from-xml="sysbiosyr095f02_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127033/sysbiosyr095f02_ht.gif" path-from-xml="sysbiosyr095f02_ht.gif" target="_blank">View large</a><a section="115127033" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f02_ht.gif?Expires=2147483647&amp;Signature=r5xm3RctTM6y-8nfV7Ht%7EiLac97Q3B4hQmOqocMKGSKNOz0rQNa6s79oi6H4570m3e-qzH8jfOBjSPyU-oY0qtTNBGdc0B-Otxv5YTvsSFjycO0zbhc28jcSELt7WSlm7SRbXSFKTU9YdwxDFtfWPdiuOhNfwEi0ih3rlfgry0zoHDXKZYIpHZ1zw4GnUMTcmxYf5WnUcULno1aaoshmTkGLdBIo%7E2Q0LJCJ6QXVMp-p-Tu6FJCijX-zXfzK4XMR262XWs4EJZW3s9JgP8hj%7E41FgE%7EPeFrsi%7E30akgd0id8tpvOeU-nbZj4TclhejkPsAp-kR6YVh3OPmvzaMS8eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127033&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f02_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison
 of 1-week runs of SATé-II simple and SATé-II ML on simulated 1000-taxon
 moderate-to-difficult data sets. Graphs from top to bottom are (a) 
normalized ML scores, (b) missing branch rates, and (c) alignment SP-FN 
errors. For each data point, <em>n</em> = 20; bars are standard errors. 
For normalized ML score, missing branch rate and alignment SP-FN error 
charts, model conditions marked with an asterisk (*) indicate that 
SATé-II ML was significantly better than SATé-II simple by <a reveal-id="bib1" data-open="bib1" class="link link-ref link-reveal xref-bibr">Benjamini and Hochberg (1995)</a> corrected pairwise <em>t</em>-tests (<em>n</em> = 40, <em>α</em> = 0.05).</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 2.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f02_ht.gif" alt="Comparison of 1-week runs of SATé-II simple and SATé-II ML on simulated 1000-taxon moderate-to-difficult data sets. Graphs from top to bottom are (a) normalized ML scores, (b) missing branch rates, and (c) alignment SP-FN errors. For each data point, n = 20; bars are standard errors. For normalized ML score, missing branch rate and alignment SP-FN error charts, model conditions marked with an asterisk (*) indicate that SATé-II ML was significantly better than SATé-II simple by Benjamini and Hochberg (1995) corrected pairwise t-tests (n = 40, α = 0.05)." path-from-xml="sysbiosyr095f02_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127033/sysbiosyr095f02_ht.gif" path-from-xml="sysbiosyr095f02_ht.gif" target="_blank">View large</a><a section="115127033" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f02_ht.gif?Expires=2147483647&amp;Signature=r5xm3RctTM6y-8nfV7Ht%7EiLac97Q3B4hQmOqocMKGSKNOz0rQNa6s79oi6H4570m3e-qzH8jfOBjSPyU-oY0qtTNBGdc0B-Otxv5YTvsSFjycO0zbhc28jcSELt7WSlm7SRbXSFKTU9YdwxDFtfWPdiuOhNfwEi0ih3rlfgry0zoHDXKZYIpHZ1zw4GnUMTcmxYf5WnUcULno1aaoshmTkGLdBIo%7E2Q0LJCJ6QXVMp-p-Tu6FJCijX-zXfzK4XMR262XWs4EJZW3s9JgP8hj%7E41FgE%7EPeFrsi%7E30akgd0id8tpvOeU-nbZj4TclhejkPsAp-kR6YVh3OPmvzaMS8eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127033&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f02_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison
 of 1-week runs of SATé-II simple and SATé-II ML on simulated 1000-taxon
 moderate-to-difficult data sets. Graphs from top to bottom are (a) 
normalized ML scores, (b) missing branch rates, and (c) alignment SP-FN 
errors. For each data point, <em>n</em> = 20; bars are standard errors. 
For normalized ML score, missing branch rate and alignment SP-FN error 
charts, model conditions marked with an asterisk (*) indicate that 
SATé-II ML was significantly better than SATé-II simple by <a reveal-id="bib1" data-open="bib1" class="link link-ref link-reveal xref-bibr">Benjamini and Hochberg (1995)</a> corrected pairwise <em>t</em>-tests (<em>n</em> = 40, <em>α</em> = 0.05).</p></div></div><p>To
 further understand the relationship between ML score and tree and 
alignment accuracy, we examined the improvement over time of ML score, 
tree accuracy, and alignment accuracy of SATé-II simple and SATé-II ML (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> for the 1000 taxon data sets, online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a> Fig. S2 for the 500-taxon data sets and online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>
 Fig. S3 for the 100-taxon data sets). ML scores and alignment and 
topological accuracy for SATé-II simple and SATé-II ML tended to improve
 in parallel, and the rate of improvement dramatically slowed or stopped
 within a small number of iterations. Thus, whether ML is used to guide 
the search for the best tree/alignment pair or not, the initial 
iterations have a large impact on tree and alignment accuracy, and the 
magnitude of improvement decreases rapidly.</p>                        <a id="115127035" scrollto-destination="115127035"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 3.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f03_ht.gif" alt="Timeplots are shown for the three SATé-II variants on the three most difficult 1000-taxon simulated model conditions. From top to bottom, normalized ML scores, missing branch rates, and alignment SP-FN errors are shown. The zero iteration is the starting alignment and tree. For each model condition, only iterations that were finished by all SATé-II runs on all 20 data sets are included. n = 20 for each datapoint." path-from-xml="sysbiosyr095f03_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127035/sysbiosyr095f03_ht.gif" path-from-xml="sysbiosyr095f03_ht.gif" target="_blank">View large</a><a section="115127035" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f03_ht.gif?Expires=2147483647&amp;Signature=E19k%7EGjPk8XKwyaPS8eftw0k6reauU6qS6VTB%7E5lo77pYRP%7E18Qo%7E799UPIHWcwfo5-RCXv38InXIP-98aQnADfRs2g6wls2QeHmRvScAMpDC1GnGYhK1S2rZo4VrKpmx8VnL-eOTmvJumpEOo12MNhxaJYzLgHsaUi29ZEVe65JeYRcW%7EQYtLOac3LvnYmNdVpdxrAcEphfLTnQBGo8o03UThFkVQkuOct3umh9i2gifsaayUW1lqhCwTdoRYrtCYv8hv1BNoQKetE%7E3Hc5MNx7vumF88zxNZmcj4UXvzYIiVmK4Z77zGy1X4mBiQBfvyGRa1zXlUPVSzjJHfKT6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127035&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f03_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Timeplots
 are shown for the three SATé-II variants on the three most difficult 
1000-taxon simulated model conditions. From top to bottom, normalized ML
 scores, missing branch rates, and alignment SP-FN errors are shown. The
 zero iteration is the starting alignment and tree. For each model 
condition, only iterations that were finished by all SATé-II runs on all
 20 data sets are included. <em>n</em> = 20 for each datapoint.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 3.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f03_ht.gif" alt="Timeplots are shown for the three SATé-II variants on the three most difficult 1000-taxon simulated model conditions. From top to bottom, normalized ML scores, missing branch rates, and alignment SP-FN errors are shown. The zero iteration is the starting alignment and tree. For each model condition, only iterations that were finished by all SATé-II runs on all 20 data sets are included. n = 20 for each datapoint." path-from-xml="sysbiosyr095f03_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127035/sysbiosyr095f03_ht.gif" path-from-xml="sysbiosyr095f03_ht.gif" target="_blank">View large</a><a section="115127035" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f03_ht.gif?Expires=2147483647&amp;Signature=E19k%7EGjPk8XKwyaPS8eftw0k6reauU6qS6VTB%7E5lo77pYRP%7E18Qo%7E799UPIHWcwfo5-RCXv38InXIP-98aQnADfRs2g6wls2QeHmRvScAMpDC1GnGYhK1S2rZo4VrKpmx8VnL-eOTmvJumpEOo12MNhxaJYzLgHsaUi29ZEVe65JeYRcW%7EQYtLOac3LvnYmNdVpdxrAcEphfLTnQBGo8o03UThFkVQkuOct3umh9i2gifsaayUW1lqhCwTdoRYrtCYv8hv1BNoQKetE%7E3Hc5MNx7vumF88zxNZmcj4UXvzYIiVmK4Z77zGy1X4mBiQBfvyGRa1zXlUPVSzjJHfKT6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127035&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f03_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Timeplots
 are shown for the three SATé-II variants on the three most difficult 
1000-taxon simulated model conditions. From top to bottom, normalized ML
 scores, missing branch rates, and alignment SP-FN errors are shown. The
 zero iteration is the starting alignment and tree. For each model 
condition, only iterations that were finished by all SATé-II runs on all
 20 data sets are included. <em>n</em> = 20 for each datapoint.</p></div></div><p>We also explored the performance of a heuristic (see online &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a>)
 that builds an alignment greedily, one sequence at a time while 
explicitly optimizing the GTR+Gamma ML score during each sequence 
addition. We tested this heuristic on a collection of 1000 taxon data 
sets generated in this study. Very poor trees and alignments were 
obtained for the moderate and difficult model conditions as a result of 
optimizing GTR+Gamma ML, treating gaps as missing data (online &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a>).
 Although it is possible that this result is due in part to the 
operation of our heuristic, the fact that it consistently produced worse
 alignments and trees by optimizing ML on data sets of moderate and 
difficult model conditions suggests that the problem lay in using ML 
treating gaps as missing data.</p>                            <h4 scrollto-destination="115127037" id="115127037" class="section-title">Impact of additional iterations on SATé-II's performance.—</h4>
<p>Having shown that using ML scores to pick the best tree produced by 
SATé-II is unnecessary, we next explored how to make SATé-II as fast as 
possible, without sacrificing too much accuracy. We compared the 
starting tree and alignment to the tree and alignment produced by a 
single iteration of SATé-II (Fig. S4). Under hard 1000-taxon model 
conditions, very substantial improvements in both alignment and tree 
accuracy arose from a single iteration through the SATé-II loop. In 
fact, the trees resulting from the first iteration of SATé-II were more 
accurate than the result of a 24-h analysis using SATé-I. In particular,
 SATé-II fast, which stops searching for a better tree/alignment pair 
once the ML score gets worse, does as well as the other SATé-II variants
 for easy models (data not shown) and results in only a small decrease 
in tree accuracy for the harder models (Fig. S4 for the 1000-taxon 
moderate-to-difficult data sets) data sets.</p>                            <h4 scrollto-destination="115127039" id="115127039" class="section-title">Other aspects of algorithm design.—</h4>
<p>We explored other changes to SATé-II's algorithm design, including 
the choice of starting tree, using different alignment methods to align 
subsets, the use of Muscle instead of Opal as the alignment merger, the 
maximum data set size for the decomposition strategy, and the edge 
around which the decomposition occurs. These experiments (Table S4) 
showed that SATé-II fast is robust to the choice of starting tree and 
that three changes to the algorithm relative to SATé-I yielded 
improvements in alignment and/or tree accuracy: SATé-II was improved by 
using Opal as the alignment merger instead of Muscle, by decomposing the
 data into smaller subsets, and by decomposing around the longest branch
 instead of other branches.</p><p>Of these algorithm design issues, the 
technique used to merge subalignments (i.e., Muscle or Opal) and the 
size of the decomposition tended to have very large impacts, whereas the
 choice of decomposition edge had a relatively small impact (e.g., Table
 S4). At present, we do not know what algorithmic differences between 
Muscle and Opal cause Opal to produce more accurate merges. Both Muscle 
and Opal use profile–profile alignment to merge subalignments but they 
differ in the details. One possible significant difference is their 
scoring functions for estimating the number of gaps: Muscle uses a 
log-expectation scoring function (Edgar 2004a, <a reveal-id="bib4" data-open="bib4" class="link link-ref link-reveal xref-bibr">2004b</a>) with optimistic gap counts (<a reveal-id="bib15" data-open="bib15" class="link link-ref link-reveal xref-bibr">Kececioglu and Zhang 1998</a>), whereas Opal uses a sum-of-pairs scoring function with pessimistic gap counts (<a reveal-id="bib15" data-open="bib15" class="link link-ref link-reveal xref-bibr">Kececioglu and Zhang 1998</a>, <a reveal-id="bib45" data-open="bib45" class="link link-ref link-reveal xref-bibr">Wheeler and Kececioglu 2007</a>).
 With respect to decomposition size, decomposing several times produced 
much bigger improvements than decomposing only once, and decomposing to 
data sets of size at most 200 produced better results than decomposing 
to larger data sets (e.g., Table S4 which includes comparison of 
decomposition variants of SATé-II on one difficult 1000-taxon model 
condition). The choice of method for aligning subsets also had a large 
impact on the alignment and tree accuracy and is discussed below.</p>                            <h4 scrollto-destination="115127042" id="115127042" class="section-title">Comparison of SATé-II fast with SATé-I and two-phase methods.—</h4>
<p>For the simulated data sets, SATé-II fast matched or improved upon 
SATé-I and all the two-phase methods with respect to ML scores, 
false-negative rates, false-positive rates, and alignment errors. With 
the exception of false-positive topological error rates, <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Figure 4</a> shows these results for the 1000-taxon data sets, and Figure S5 in online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>
 shows these results for smaller data sets. The false-positive 
topological error rates are shown in Figure S6. For the difficult 500- 
and 1000-taxon data sets, the accuracy of the SATé-II fast trees was 
substantially greater than that of the next most accurate method, which 
was usually SATé-I and almost as accurate as RAxML trees produced on the
 true alignments. Finally, although SATé-I was reasonably efficient 
(even on the 1000-taxon data sets), SATé-II fast was <em>much</em> faster than SATé-I (which by default runs for 24 h after the starting tree is computed).</p>                        <a id="115127044" scrollto-destination="115127044"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 4.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f04_3c.gif" alt="Comparisons of SATé-II fast to SATé-I and two-phase methods on “moderate-to-difficult” simulated 1000-taxon data sets. Panels from top to bottom are (a) ML scores (normalized by scores obtained by ML [MAFFT]), (b) missing branch rates (%), (c) alignment SP-FN error (%), and (d) runtime (h); n = 20 for each model condition. SATé runtimes include the time to compute starting trees. Model conditions marked with an asterisk (*) indicate significant improvement by SATé-II fast over the next best estimated method, as measured by Benjamini and Hochberg (1995) corrected pairwise t-tests (n = 40, α = 0.05). Q values for these statistical tests are reported in Table S6." path-from-xml="sysbiosyr095f04_3c.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127044/sysbiosyr095f04_3c.gif" path-from-xml="sysbiosyr095f04_3c.gif" target="_blank">View large</a><a section="115127044" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f04_3c.gif?Expires=2147483647&amp;Signature=OZpNcvuSk-5hQfdID3pdk2-XDm%7Eh65UVtH%7EU643WExRHbeMpN8NCGmCiwHBnEVCnvxSvHDEPH%7EtHbqLvxaI2soWmMAZ9AnGfHnST8HNuQxLoWfuz%7E79MsCqDtJGYmXLafPTfhAr-85A-2Wy56yzCYotKyco8o35aC0W5uHsZi4H7E0AnFtlfr08ZxxW6vUpXx4c26fnbao4QxxiQJ469ZeRtgQq7-ysvDSERWQ-afBXm8p04fV-JerIpDmBRI2Ls4ibL0VUmk4WJUfVflrYFfF3E9IJrJHifru-fEBYGX-YlmPqCnWLdTUPcdYINrqPR%7EN-2wnlPw5FOweA838hF2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127044&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f04_3c.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparisons
 of SATé-II fast to SATé-I and two-phase methods on 
“moderate-to-difficult” simulated 1000-taxon data sets. Panels from top 
to bottom are (a) ML scores (normalized by scores obtained by ML 
[MAFFT]), (b) missing branch rates (%), (c) alignment SP-FN error (%), 
and (d) runtime (h); <em>n</em> = 20 for each model condition. SATé 
runtimes include the time to compute starting trees. Model conditions 
marked with an asterisk (*) indicate significant improvement by SATé-II 
fast over the next best estimated method, as measured by <a reveal-id="bib1" data-open="bib1" class="link link-ref link-reveal xref-bibr">Benjamini and Hochberg (1995)</a> corrected pairwise <em>t</em>-tests (<em>n</em> = 40, <em>α</em> = 0.05). <em>Q</em> values for these statistical tests are reported in Table S6.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 4.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f04_3c.gif" alt="Comparisons of SATé-II fast to SATé-I and two-phase methods on “moderate-to-difficult” simulated 1000-taxon data sets. Panels from top to bottom are (a) ML scores (normalized by scores obtained by ML [MAFFT]), (b) missing branch rates (%), (c) alignment SP-FN error (%), and (d) runtime (h); n = 20 for each model condition. SATé runtimes include the time to compute starting trees. Model conditions marked with an asterisk (*) indicate significant improvement by SATé-II fast over the next best estimated method, as measured by Benjamini and Hochberg (1995) corrected pairwise t-tests (n = 40, α = 0.05). Q values for these statistical tests are reported in Table S6." path-from-xml="sysbiosyr095f04_3c.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127044/sysbiosyr095f04_3c.gif" path-from-xml="sysbiosyr095f04_3c.gif" target="_blank">View large</a><a section="115127044" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f04_3c.gif?Expires=2147483647&amp;Signature=OZpNcvuSk-5hQfdID3pdk2-XDm%7Eh65UVtH%7EU643WExRHbeMpN8NCGmCiwHBnEVCnvxSvHDEPH%7EtHbqLvxaI2soWmMAZ9AnGfHnST8HNuQxLoWfuz%7E79MsCqDtJGYmXLafPTfhAr-85A-2Wy56yzCYotKyco8o35aC0W5uHsZi4H7E0AnFtlfr08ZxxW6vUpXx4c26fnbao4QxxiQJ469ZeRtgQq7-ysvDSERWQ-afBXm8p04fV-JerIpDmBRI2Ls4ibL0VUmk4WJUfVflrYFfF3E9IJrJHifru-fEBYGX-YlmPqCnWLdTUPcdYINrqPR%7EN-2wnlPw5FOweA838hF2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127044&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f04_3c.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparisons
 of SATé-II fast to SATé-I and two-phase methods on 
“moderate-to-difficult” simulated 1000-taxon data sets. Panels from top 
to bottom are (a) ML scores (normalized by scores obtained by ML 
[MAFFT]), (b) missing branch rates (%), (c) alignment SP-FN error (%), 
and (d) runtime (h); <em>n</em> = 20 for each model condition. SATé 
runtimes include the time to compute starting trees. Model conditions 
marked with an asterisk (*) indicate significant improvement by SATé-II 
fast over the next best estimated method, as measured by <a reveal-id="bib1" data-open="bib1" class="link link-ref link-reveal xref-bibr">Benjamini and Hochberg (1995)</a> corrected pairwise <em>t</em>-tests (<em>n</em> = 40, <em>α</em> = 0.05). <em>Q</em> values for these statistical tests are reported in Table S6.</p></div></div>                            <h4 scrollto-destination="115127045" id="115127045" class="section-title">SATé-II as an alignment method booster.—</h4>
<p>We explored using SATé-II fast to “boost” the performance of 
alignment methods, by substituting a given alignment method (e.g., 
Muscle) for MAFFT to estimate the initial tree and to align the subsets.
 These “SATé-II-boosted” methods produced substantially more accurate 
trees and alignments than their base methods on large difficult to align
 data sets (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a>).
 Particularly, dramatic improvements were obtained when the base method 
was ClustalW or Muscle. Furthermore, SATé-II-boosted versions of Opal 
and Prank+GT were close to SATé-II fast in terms of topological 
accuracy. Thus, SATé-II can be considered a “metamethod” that can 
improve the accuracy of alignment methods, enabling them to produce more
 accurate alignments and enabling RAxML to produce more accurate trees. 
The magnitude of the improvement in alignment accuracy depends on the 
accuracy of the original alignment method on the data set, with 
improvements greatest on the harder-to-align data sets.</p>                        <a id="115127047" scrollto-destination="115127047"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 5.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f05_3c.gif" alt="Comparisons of SATé-II-boosted MSA methods to the unboosted MSA methods with respect to missing branch rates on simulated moderate-to-difficult 1000-taxon data sets. Panels from top to bottom display results for alignment methods (a) Muscle, (b) Opal, (c) ClustalW, and (d) Prank+GT. For each panel, the unboosted MSA methods are shown in blue, whereas the boosted SATé-II fast variants are shown in green. SATé-II fast in its default mode is shown in red, and RAxML on the true alignment is shown in black." path-from-xml="sysbiosyr095f05_3c.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127047/sysbiosyr095f05_3c.gif" path-from-xml="sysbiosyr095f05_3c.gif" target="_blank">View large</a><a section="115127047" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f05_3c.gif?Expires=2147483647&amp;Signature=2eNmL9agp6JTYbFLJbbawnS8tbSgJpZzh0-lLZZM2aYLa08g3hyNURqcFgnhreRQMemVRmhAe%7EULmotd-vVV7NJVHOQbfQ4nQDT7IC2pTJfy1dsHX143qokHaPIVVSL3T8KmNm7nDUHfOA0z0YTp5h9t%7E9sj5IaVYXgF88jTRheMCMiP0UN0Eud66pr3ctvMB3QX3VTV1Xd0r3RcDtu0k3Ko37efEUrZoasuD%7ErrW2tvV4vJm982yw8vsjuMju54KGgpMzcM7G8pbNwv8Dss-8sDj7nwWjecRPpkuOxXgSkHJlJNhoiasoo5FjKJJrucdP65WjnKZzAY-A6xfE0vIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127047&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f05_3c.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparisons
 of SATé-II-boosted MSA methods to the unboosted MSA methods with 
respect to missing branch rates on simulated moderate-to-difficult 
1000-taxon data sets. Panels from top to bottom display results for 
alignment methods (a) Muscle, (b) Opal, (c) ClustalW, and (d) Prank+GT. 
For each panel, the unboosted MSA methods are shown in blue, whereas the
 boosted SATé-II fast variants are shown in green. SATé-II fast in its 
default mode is shown in red, and RAxML on the true alignment is shown 
in black.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 5.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f05_3c.gif" alt="Comparisons of SATé-II-boosted MSA methods to the unboosted MSA methods with respect to missing branch rates on simulated moderate-to-difficult 1000-taxon data sets. Panels from top to bottom display results for alignment methods (a) Muscle, (b) Opal, (c) ClustalW, and (d) Prank+GT. For each panel, the unboosted MSA methods are shown in blue, whereas the boosted SATé-II fast variants are shown in green. SATé-II fast in its default mode is shown in red, and RAxML on the true alignment is shown in black." path-from-xml="sysbiosyr095f05_3c.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127047/sysbiosyr095f05_3c.gif" path-from-xml="sysbiosyr095f05_3c.gif" target="_blank">View large</a><a section="115127047" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f05_3c.gif?Expires=2147483647&amp;Signature=2eNmL9agp6JTYbFLJbbawnS8tbSgJpZzh0-lLZZM2aYLa08g3hyNURqcFgnhreRQMemVRmhAe%7EULmotd-vVV7NJVHOQbfQ4nQDT7IC2pTJfy1dsHX143qokHaPIVVSL3T8KmNm7nDUHfOA0z0YTp5h9t%7E9sj5IaVYXgF88jTRheMCMiP0UN0Eud66pr3ctvMB3QX3VTV1Xd0r3RcDtu0k3Ko37efEUrZoasuD%7ErrW2tvV4vJm982yw8vsjuMju54KGgpMzcM7G8pbNwv8Dss-8sDj7nwWjecRPpkuOxXgSkHJlJNhoiasoo5FjKJJrucdP65WjnKZzAY-A6xfE0vIg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127047&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f05_3c.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparisons
 of SATé-II-boosted MSA methods to the unboosted MSA methods with 
respect to missing branch rates on simulated moderate-to-difficult 
1000-taxon data sets. Panels from top to bottom display results for 
alignment methods (a) Muscle, (b) Opal, (c) ClustalW, and (d) Prank+GT. 
For each panel, the unboosted MSA methods are shown in blue, whereas the
 boosted SATé-II fast variants are shown in green. SATé-II fast in its 
default mode is shown in red, and RAxML on the true alignment is shown 
in black.</p></div></div>                            <h3 scrollto-destination="115127048" id="115127048" class="section-title">Biological Data</h3>
<p>For the six smallest data sets, RAxML on a MAFFT alignment, SATé-I, 
and SATé-II had essentially the same accuracy (Table S5), and all the 
other two-phase methods did quite poorly. On the two larger biological 
data sets, 16S.T and 16S.B.ALL, we observed substantial differences 
between SATé-I, SATé-II and all the two-phase methods. First, on the 
largest of these data sets, 16S.B.ALL, the only two-phase methods that 
completed were RAxML and FastTree on the PartTree alignment. SATé-I 
failed to complete even one alignment because the decomposition produced
 several very large subsets of sequences (the largest having 8666 
sequences), and MAFFT failed when analyzing these subsets, even given 
256 GB of memory. In contrast, SATé-II ran well on a 32 GB machine and 
completed several iterations.</p><p>On the 16S.B.ALL data set, using a 
FastTree ML tree estimated on the PartTree alignment as its starting 
tree, SATé-II produced a more accurate tree than any two-phase methods 
after just one iteration (428.9 h, about 17.9 days), and additional 
iterations continued to improve the topological accuracy (<a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6</a>).
 By the eighth iteration, SATé-II's missing branch rate was less than 
half of that of RAxML on the PartTree alignment. SATé-II's trees were 
calculated reasonably efficiently. For example, RAxML on the PartTree 
required 1418 h runtime (almost 59 days) to compute, whereas, including 
the time to compute the starting tree, four iterations of SATé-II 
required 2067.9 h (about 86 days), and eight iterations required 4083.6 h
 (about 170 days).</p>                        <a id="115127051" scrollto-destination="115127051"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 6.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f06_lw.gif" alt="Missing branch rates of two-phase methods on the PartTree alignment and SATé-II on the 16S.B.ALL data set. The two-phase methods are shown in white, and the SATé-II iterations are shown in black. The SATé-II analyses start from FastTree on the PartTree alignment. The missing branch rates are computed with respect to the curation-based tree, which is an ML tree estimated using RAxML on the curated alignment after contraction of branches with less than 75% support from a RAxML bootstrapping analysis with 444 replicates. Each iteration of SATé-II computes subalignments using MAFFT, merges subalignments using Muscle, and computes a tree using FastTree; these iterations take between 7 to 35 days each and run on a 32 GB machine. By comparison, SATé-I failed to run on this data set. n = 1 for all reported values." path-from-xml="sysbiosyr095f06_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127051/sysbiosyr095f06_lw.gif" path-from-xml="sysbiosyr095f06_lw.gif" target="_blank">View large</a><a section="115127051" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f06_lw.gif?Expires=2147483647&amp;Signature=lsWJCNiJJy1OVj8wC6HC7blOwu%7EG5D8xTOkpNKkyngssexsjG4bJuHqUhC1NRq9DQayivjBhCPhE-Nvy0F9n7mylh52J8%7EmmGQWlHfHQRFxS0vx658vRicVhpggCiD7YE47Dxl00Aft7kptNp6jsstSL68LG6i1gmkBqFSwjDJul9xvgPgrESrZ1aPBh2eTGOylcno4Gjku5tMuGi8pPCkCed24SpmGaU09NMurZ0nwnBGsp28Gj3KeaYEmIK9LUzIIGbNBe2tz42GMoLeYgR3lVHGtflcjKMNvaedaBI1A%7E6uKG-y7qIzfhx3jCFtfv-IlzFYa5sL2pnyVDxfCODg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127051&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f06_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Missing
 branch rates of two-phase methods on the PartTree alignment and SATé-II
 on the 16S.B.ALL data set. The two-phase methods are shown in white, 
and the SATé-II iterations are shown in black. The SATé-II analyses 
start from FastTree on the PartTree alignment. The missing branch rates 
are computed with respect to the curation-based tree, which is an ML 
tree estimated using RAxML on the curated alignment after contraction of
 branches with less than 75% support from a RAxML bootstrapping analysis
 with 444 replicates. Each iteration of SATé-II computes subalignments 
using MAFFT, merges subalignments using Muscle, and computes a tree 
using FastTree; these iterations take between 7 to 35 days each and run 
on a 32 GB machine. By comparison, SATé-I failed to run on this data 
set. <em>n</em> = 1 for all reported values.</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 6.</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr095f06_lw.gif" alt="Missing branch rates of two-phase methods on the PartTree alignment and SATé-II on the 16S.B.ALL data set. The two-phase methods are shown in white, and the SATé-II iterations are shown in black. The SATé-II analyses start from FastTree on the PartTree alignment. The missing branch rates are computed with respect to the curation-based tree, which is an ML tree estimated using RAxML on the curated alignment after contraction of branches with less than 75% support from a RAxML bootstrapping analysis with 444 replicates. Each iteration of SATé-II computes subalignments using MAFFT, merges subalignments using Muscle, and computes a tree using FastTree; these iterations take between 7 to 35 days each and run on a 32 GB machine. By comparison, SATé-I failed to run on this data set. n = 1 for all reported values." path-from-xml="sysbiosyr095f06_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115127051/sysbiosyr095f06_lw.gif" path-from-xml="sysbiosyr095f06_lw.gif" target="_blank">View large</a><a section="115127051" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/sysbiosyr095f06_lw.gif?Expires=2147483647&amp;Signature=lsWJCNiJJy1OVj8wC6HC7blOwu%7EG5D8xTOkpNKkyngssexsjG4bJuHqUhC1NRq9DQayivjBhCPhE-Nvy0F9n7mylh52J8%7EmmGQWlHfHQRFxS0vx658vRicVhpggCiD7YE47Dxl00Aft7kptNp6jsstSL68LG6i1gmkBqFSwjDJul9xvgPgrESrZ1aPBh2eTGOylcno4Gjku5tMuGi8pPCkCed24SpmGaU09NMurZ0nwnBGsp28Gj3KeaYEmIK9LUzIIGbNBe2tz42GMoLeYgR3lVHGtflcjKMNvaedaBI1A%7E6uKG-y7qIzfhx3jCFtfv-IlzFYa5sL2pnyVDxfCODg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115127051&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="sysbiosyr095f06_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Missing
 branch rates of two-phase methods on the PartTree alignment and SATé-II
 on the 16S.B.ALL data set. The two-phase methods are shown in white, 
and the SATé-II iterations are shown in black. The SATé-II analyses 
start from FastTree on the PartTree alignment. The missing branch rates 
are computed with respect to the curation-based tree, which is an ML 
tree estimated using RAxML on the curated alignment after contraction of
 branches with less than 75% support from a RAxML bootstrapping analysis
 with 444 replicates. Each iteration of SATé-II computes subalignments 
using MAFFT, merges subalignments using Muscle, and computes a tree 
using FastTree; these iterations take between 7 to 35 days each and run 
on a 32 GB machine. By comparison, SATé-I failed to run on this data 
set. <em>n</em> = 1 for all reported values.</p></div></div><p>Performance
 on the 16S.T data set, which has 7350 taxa, showed similar advantages. 
SATé-I and SATé-II gave dramatic improvements over the best two-phase 
methods, RAxML on the PartTree alignment, which had 16.5% missing branch
 rate (Fig. S7). One iteration of SATé-II reduced the error to 11.2% 
error, and subsequent iterations reduced the error to 8.2% error.</p>                            <h3 scrollto-destination="115127053" id="115127053" class="section-title">Relative Scalability of SATé-I and SATé-II</h3>
<p>A significant advantage of SATé-II over SATé-I was running time and 
memory, and this advantage increases with the data set size. On the 
smaller data sets (i.e., the simulated data sets with up to 1000 
sequences and the six biological data sets in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>), SATé-II finished in a fraction of the time of a SATé-I analysis (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>
 and Fig. S5 for the simulated data sets, and Table S5 for the 
biological data sets). Improvements in speed and memory usage were even 
larger on the largest data sets we examined (the 7350 taxon 16S.T data 
set, and the 27,643 taxon 16S.B.ALL data set). On the 16S.T data set 
(Fig. S7), the realignment step within each iteration of SATé-I took 
almost 20 times as long as the realignment step of SATé-II (150.5 h, on 
average for SATé-I compared to 8.3 h for each realignment done by 
SATé-II). The difference on the 16S.B.ALL data set was especially 
significant, since SATé-II finished each iteration in approximately 1 
week, but SATé-I failed to complete its first realignment of the 
16S.B.ALL data set.</p><p>The memory requirements of SATé-I and SATé-II 
are also worth noting. On the simulated data sets and all biological 
data sets except for the two largest, 4 GB of main memory was sufficient
 to run both SATé-I and SATé-II. On 16S.T, the second largest biological
 data set, SATé-I and SATé-II required machines with 32 GB of main 
memory to run to completion. For the 16S.B.ALL data set, with 27,643 
taxa, although SATé-I failed to complete its first alignment, SATé-II 
finished several iterations on a machine with 32 GB of main memory.</p>                            <h3 scrollto-destination="115127056" id="115127056" class="section-title">Methods that Optimize ML, Treating Gaps as Missing Data</h3>
<p>Our theoretical (&nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a>) and empirical results (Fig. S8 in online &nbsp;<a class="js-jumplink" href="#app1">Appendix 1</a> and online &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a>)
 regarding approaches that aggressively seek the alignment/tree pair 
that optimizes ML scores while treating gaps as missing data, suggest 
that this is a very poor approach for some models of evolution and may 
be generally inadvisable.</p><p>However, the way SATé-II utilizes ML is 
not subject to this problem. SATé-II uses ML in two ways—to compute 
trees on estimated alignments and to select a tree/alignment pair from 
among a set of alignment/tree pairs generated by SATé-II's iterative 
procedure. The alignments estimated by SATé-II are produced by a 
combination of MAFFT and Opal or Muscle, methods that use algorithmic 
criteria that are independent of ML. Thus, while ML is an integral 
component of SATé-II's algorithmic strategy, the technique SATé-II uses 
to produce alignments is independent of ML. This may be why the use of 
ML within SATé-II is not detrimental.</p><p>Interestingly, our 
experiments showed that SATé-II ML produced alignments that were 
slightly more accurate than SATé-II simple alignments, and trees that 
were statistically of equal accuracy to SATé-II simple trees. Why this 
use of ML (to select from a set of tree/alignment pairs) provides this 
small advantage is still unclear.</p>                            <h2 scrollto-destination="115127060" id="115127060" class="section-title">D<span class="small-caps">ISCUSSION</span></h2>
<p>SATé-II is a novel and highly accurate method for simultaneously 
estimating phylogenetic alignments and trees. Furthermore, its 
computationally most efficient method, SATé-II fast, offers dramatic 
improvements over SATé-I and two-phase methods on large 
difficult-to-align data sets. Also, SATé-II is a metamethod that can be 
used to improve the accuracy of a given alignment method and in turn 
improve tree estimations based upon these alignments.</p><p>The evidence
 presented here strongly suggests that the most important part of the 
algorithm design for SATé-II is its iterative use of a 
divide-and-conquer realignment strategy. The intuition behind this 
algorithm design is that even the best alignment methods have difficulty
 aligning sequence data sets when they are large and have evolved with 
many substitutions and indels. The SATé-II decomposition technique seeks
 to break the large data set into smaller subsets, so that each of the 
smaller subsets has a smaller maximum evolutionary distance between its 
taxa and is fairly densely sampled within that subset. This enables the 
subset alignments produced by MAFFT (or another alignment method) to be 
more accurate. Furthermore, since the merger technique does not undo the
 alignments on subsets, the result of merging the improved subset 
alignments is able to maintain the improved accuracy. Therefore, for 
imperfect initial alignments, performing the SATé-II realignment 
technique on an ML tree estimated on the initial alignment will produce a
 more accurate alignment, and RAxML on the new alignment will produce a 
more accurate tree. Furthermore, subsequent iterations will tend to 
provide additional improvements, until the alignment/tree pairs it 
produces have become fairly accurate.</p><p>SATé-II provides a very 
substantial improvement in scalability and accuracy over SATé-I because 
of the specific divide-and-conquer design used in SATé-II, which ensures
 that the subsets it creates for realignment are quite small (at most 
200 sequences); by comparison, the subsets produced by the SATé-I 
decomposition can be unboundedly large. More generally, this means that 
very large data sets can be analyzed using SATé-II that cannot be 
analyzed by SATé-I in the same timeframes.</p><p>As we have 
demonstrated, SATé-II trees approach the accuracy of GTR+Gamma ML trees 
(estimated using RAxML) on the true alignment. However, RAxML and other 
standard ML methods (e.g., PhyML (<a reveal-id="bib6" data-open="bib6" class="link link-ref link-reveal xref-bibr">Guindon and Gascuel 2003</a>), PAUP* (<a reveal-id="bib35" data-open="bib35" class="link link-ref link-reveal xref-bibr">Swofford 2003</a>), FastTree (<a reveal-id="bib25" data-open="bib25" class="link link-ref link-reveal xref-bibr">Price et al. 2010</a>),
 etc.) treat gaps as missing data and are based upon statistical models 
of evolution that do not include indel events. There are two 
consequences of this observation. The first is that SATé-II is not 
likely to be statistically consistent under models that include indel 
events, because standard ML methods are themselves not statistically 
consistent, even if given the true alignment. That is, treating gaps as 
missing data is not a statistically consistent way of estimating 
phylogenies from alignments that have gaps. The second consequence is 
that while SATé-II is a substantial advance, it seems likely that even 
better results could be obtained when <em>scalable</em> algorithms and 
software are developed that estimate phylogenies under statistical 
models that include indels as well as substitutions. Unfortunately, 
existing methods that are based upon statistical models that include 
indel events (e.g., StatAlign (Novák et al. 2008), BAli-Phy (<a reveal-id="bib34" data-open="bib34" class="link link-ref link-reveal xref-bibr">Suchard and Redelings 2006</a>), and ALIFRITZ (<a reveal-id="bib5" data-open="bib5" class="link link-ref link-reveal xref-bibr">Fleissner et al. 2005</a>)) are not yet able to analyze even moderately large data sets.</p>                            <h2 scrollto-destination="115127065" id="115127065" class="section-title">S<span class="small-caps">UPPLEMENTARY</span> M<span class="small-caps">ATERIAL</span></h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/syr095_supplementary_data.zip?Expires=2147483647&amp;Signature=1xOaW3LVtkjXCgetiu2CgiuTBjaE2aFbExDo-ZriyJDw1qsmAzYyV0d18wr9L74PTUeVvUuOU-o6HOHEz1qQurKHynduI5LgBKUPPB8DtLi%7ESbyvUvq6V%7ErvbaBYpvJgzvuyN1m3iZUJBZyMf6tnzgpAUymDhB9SRMYnMhBfUzL8BJ-qKcmDUw90Aflfyn8tUIEYecM5yCYCCwC9odH2%7EZpfghYX4nZcuIqzajjrV42P-4L9JRsEtDhjwwd%7E7UIkKeHo7DerZFsLEKFyTG%7EfU0kw3s33tm%7ED8psWcPNeF7GRXFl4tP8ZVKV%7EZ%7EkHqPNmUVWfE4GFcaPmCSEGwUfYuQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span>.</p>                            <h2 scrollto-destination="115127067" id="115127067" class="section-title">F<span class="small-caps">UNDING</span></h2>
<p>A.S. was supported under the auspices of the Emmy Noether program by 
the German Science Foundation (Deutsche Forschungsgemeinschaft). T.J.W.,
 K.L., and S.N. were supported by the US National Science Foundation 
under ITR-0331453 (CIPRES) and T.J.W., K.L., and C.R.L. were supported 
by the US National Science Foundation grant ATOL-0733029. M.T.H. and 
J.Y. were supported by the US National Science Foundation through grant 
ATOL-0732920. T.J.W. also received support from the Guggenheim 
Foundation and from Microsoft Research New England. This research was 
supported in part by the US National Science Foundation Research 
Infrastructure Grant EIA-0303609.</p><p>The authors would like to acknowledge the Texas Advanced Computing Center (TACC) (<a class="link link-uri" href="http://www.tacc.utexas.edu/" target="">http://www.tacc.utexas.edu</a>)
 at The University of Texas at Austin for providing HPC resources that 
have contributed to the research results reported within this paper. We 
also wish to thank John Cazes and Dan Stanzione for their assistance 
with the analyses on the TACC HPC resources. The authors also thank 
Steve Evans for helpful discussions regarding the theoretical results, 
as well as Ron DeBry, Laura Kubatko, and two anonymous reviewers for 
useful suggestions.</p>                            <h4 scrollto-destination="115127073" id="115127073" class="section-title">Role Of Likelihood Optimization, Treating Gaps As Missing Data</h4>
<p>In this section, we provide the results regarding the use of 
optimizing likelihood while treating gaps as missing data. We present a 
theoretical result for an extended version of the JC model (a special 
case of the GTR model), in which we allow branches of the model tree to 
have substitution probabilities set to 0. We call this extension the EJC
 model. We show that optimizing likelihood under the EJC model, allowing
 the alignment to change arbitrarily, is uninformative in that there is 
some alignment for which all trees are optimal under this criterion. 
Then, by continuity, it follows that the same result is true for the JC 
model, which requires that all branches have nonzero substitution 
probability. We show as well that the proof technique does not apply to 
the Kimura 2-parameter (K2P) model (a submodel of GTR in which 
transitions and transversions each have a fixed probability, but these 
can differ). Finally, online &nbsp;<a class="js-jumplink" href="#app2">Appendix 2</a>
 provides the results of a simple heuristic we designed that produces a 
greedy alignment and tree and which shows that improving the GTR + Gamma
 ML score can lead to very poor alignments and trees.</p><div class="block-child-p"><strong>Lemma 1</strong> Let <em>A</em> be a DNA alignment in which no site is entirely gapped, and let <em>r</em><sub><em>i</em></sub> denote the number of states present in site <em>i</em>. Let <em>A</em><sup>′</sup> be the alignment obtained by splitting every site <em>i</em> in <em>A</em> into <em>r</em><sub><em>i</em></sub> sites, so that each of these <em>r</em><sub><em>i</em></sub> sites has only one state. For example, if the site is given by<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx1_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx1_ht.gif"></div></div>then we would obtain three sites, as follows:<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx2_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx2_ht.gif"></div></div>Then, for all trees <em>T</em>, if gaps are treated as missing data, ML<sub>EJC</sub>(<em>A</em>,<em>T</em>) ≤ ML<sub>EJC</sub>(<em>A</em><sup>′</sup>,<em>T</em>) = 1/4<sup><em>R</em></sup>, where <em>R</em> is the number of sites in alignment <em>A</em><sup>′</sup>.</div><p><em>Proof.</em> We begin by noting that ML<sub>EJC</sub>(<em>A</em>,<em>T</em>) ≤ ML<sub>NCM</sub>(<em>A</em>,<em>T</em>), where NCM denotes the No Common Mechanism model (<a reveal-id="bib40" data-open="bib40" class="link link-ref link-reveal xref-bibr">Tuffley and Steel 1997</a>). Under the NCM model, sites evolve under a JC model down a common tree, and for every site <em>i</em> and edge <em>e</em> combination, there is a probability of change <em>p</em>(<em>e</em>,<em>i</em>) such that 0 ≤ <em>p</em>(<em>e</em>,<em>i</em>) &lt; 3/4. Thus, site <em>i</em> changes on edge <em>e</em> with probability <em>p</em>(<em>e</em>,<em>i</em>). Clearly, the NCM model contains the EJC model. However, the NCM model does not contain more general models such as K2P (<a reveal-id="bib16" data-open="bib16" class="link link-ref link-reveal xref-bibr">Kimura 1980</a>). Note also that treating gaps as missing data within the NCM model yields ML<sub><em> N C M</em></sub> (<em>A</em>,<em>T</em>) = 1/4<sup><em>k</em> + <em>M</em><em>P</em>(<em>T</em>,<em>A</em>)</sup>, where <em>k</em> is the number of sites in <em>A</em>, and <em>M</em><em>P</em>(<em>T</em>,<em>A</em>) denotes the parsimony score of alignment <em>A</em> on tree <em>T</em>.</p><div class="block-child-p">We also observe that ML<sub>EJC</sub>(<em>T</em>,<em>A</em><sup>′</sup>) is obtained by setting all substitution probabilities to 0 and so ML<sub>EJC</sub>(<em>T</em>,<em>A</em><sup>′</sup>) = 1/4<sup><em>R</em></sup>, where <em>R</em> is the number of sites in the alignment <em>A</em><sup>′</sup>. Note that <em>R</em> = ∑<sub><em>i</em></sub><em>r</em><sub><em>i</em></sub> (where <em>r</em><sub><em>i</em></sub> is the number of states that appear in site <em>i</em> in alignment <em>A</em>), and that <em>r</em><sub><em>i</em></sub> ≤ <em>M</em><em>P</em>(<em>T</em>,<em>i</em>) + 1, where <em>M</em><em>P</em>(<em>T</em>,<em>i</em>) is the number of times site <em>i</em> changes on tree <em>T</em>. Therefore <em>R</em> ≤ <em>M</em><em>P</em>(<em>T</em>,<em>A</em>) + <em>k</em>. Putting all this together, we obtain<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx3_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx3_ht.gif"></div></div>∥</div><p><strong>Theorem 1</strong> Let Φ(<em>S</em>) be a method that takes as input a set <em>S</em> of unaligned sequences and returns all pairs (T<sup>opt</sup>,A<sup>opt</sup>), where T<sup>opt</sup> is a phylogenetic tree and A<sup>opt</sup> is an alignment on <em>S</em>, such that for some set <em>θ</em><sup>opt</sup> of EJC parameters, Pr[A<sup>opt</sup>|T<sup>opt</sup>,<em>θ</em><sup>opt</sup>] is maximum among all possible alignments of <em>S</em>, trees on <em>S</em>, and EJC parameter values. (Recall that gaps are treated as missing data under this calculation.) Then for any set <em>S</em> of unaligned sequences, there is an alignment A<sup>opt</sup> such that Φ(<em>S</em>) = {(<em>T</em>∈A<sup>opt</sup>):<em>T</em>∈<em>T</em><sub><em>S</em></sub>}, where <em>T</em><sub><em>S</em></sub> is the set of all trees with leaf set <em>S</em>. In particular, the tree with all substitution probabilities set to zero is an optimal solution.</p><p><em>Proof.</em> Let <em>S</em> be an arbitrary set of <em>n</em> DNA sequences. Let (<em>T</em>,<em>A</em>,<em>θ</em>) be an optimal solution to EJC ML, so that <em>A</em> is an alignment on <em>S</em>, (<em>T</em>,<em>θ</em>) is an EJC model tree, and <em>P</em><em>r</em>[<em>A</em>|(<em>T</em>,<em>θ</em>)] is maximized among all possible alignments <em>A</em> of <em>S</em> and model trees (<em>T</em>,<em>θ</em>). Without loss of generality, we will assume that <em>A</em> has no entirely gapped sites. Then by Lemma 1, we can assume that <em>A</em> has only one type of nucleotide appearing in any site, that <em>θ</em> sets all branch substitution probabilities to zero, and that <em>P</em><em>r</em>[<em>A</em>|(<em>T</em>,<em>θ</em>)] = 1/4<sup><em>R</em></sup>, where <em>R</em> is the number of sites in alignment <em>A</em>. Note that <em>P</em><em>r</em>[<em>A</em>|(<em>T</em>,<em>θ</em>)] therefore does not depend upon the tree <em>T</em>, so that <em>ϕ</em>(<em>S</em>) contains all trees <em>T</em> on the leaf set <em>S</em>. ∥</p><p>We
 now show that we can extend this theorem to the JC model, in which all 
branch substitution probabilities are nonzero. We will work with a 
modification of the ML problem, in order to be able to address the fact 
that optimum ML solutions cannot be achieved without permitting 
substitution probabilities to be zero.</p><p><em>Supremum Likelihood:</em></p><ul class="bullet"><li><p>Input: unaligned sequences <em>S</em></p></li><li><p>Output: Tree <em>T</em> and alignment <em>A</em> for <em>S</em> such that <em>s</em><em>u</em><em>p</em><sub><em>θ</em></sub>{<em>P</em><em>r</em>[<em>A</em>|(<em>T</em>,<em>θ</em>)]} is maximized.</p></li></ul><p><strong>Theorem 2</strong> Let <em>S</em> = {<em>s</em><sub>1</sub>,<em>s</em><sub>2</sub>,…,<em>s</em><sub><em>n</em></sub>} be a set of unaligned sequences. For <em>T</em> a tree on {<em>s</em><sub>1</sub>,<em>s</em><sub>2</sub>,…,<em>s</em><sub><em>n</em></sub>}, let <em>f</em>(<em>S</em>,<em>T</em>) = <em>s</em><em>u</em><em>p</em><sub><em>θ</em>,<em>A</em></sub>{<em>P</em><em>r</em>[<em>A</em>|<em>T</em>,<em>θ</em>]}, where <em>A</em> is an alignment on <em>S</em> and <em>θ</em> the JC parameters (edge substitution probabilities). Then for all sets <em>S</em> of unaligned sequences and all trees <em>T</em> leaf labeled by the taxon set of <em>S</em>, <em>f</em>(<em>S</em>,<em>T</em>) = <em>f</em>(<em>S</em>,<em>T</em><sup>′</sup>). Hence, all trees are optimal solutions to supremum likelihood for JC, if the alignment is allowed to change arbitrarily.</p><p><em>Proof.</em> Let <em>S</em> be an arbitrary set of sequences. By Theorem 1, there is an alignment <em>A</em><sup>*</sup> on <em>S</em> optimizing the EJC ML score, such that for all bifurcating trees <em>T</em>, by setting all branch substitution probabilities on the branches of <em>T</em> to zero, we obtain an optimal EJC ML score. Let <em>M</em> = <em>P</em><em>r</em>[<em>A</em><sup>*</sup>|<em>T</em>,<em>θ</em>], where <em>θ</em>
 sets all substitution probabilities to zero. Note therefore that the 
optimal solution to Supremum Likelihood under the JC model will be <em>at most M</em>. We now show that all trees can achieve <em>M</em> (i.e., for all trees <em>T</em>, <em>s</em><em>u</em><em>p</em><sub><em>θ</em>,<em>A</em></sub>{<em>P</em><em>r</em>[<em>A</em>|<em>T</em>,<em>θ</em>]} = <em>M</em>); the result then follows.</p><p>Let <em>ϵ</em> &gt; 0 be given. By continuity of likelihood scores, for small enough (but all positive) branch substitution probabilities, <em>θ</em><sup>′</sup>, on <em>T</em>, <em>P</em><em>r</em>[<em>A</em><sup>*</sup>|<em>T</em>,<em>θ</em><sup>′</sup>] ≥ <em>M</em> − <em>ϵ</em>. Since <em>ϵ</em> &gt; 0 was arbitrary, this shows that <em>s</em><em>u</em><em>p</em><sub><em>θ</em>,<em>A</em></sub>{<em>P</em><em>r</em>[<em>A</em>|<em>T</em>,<em>θ</em>]} = <em>M</em>. Since the tree <em>T</em> was arbitrary, this shows that all trees achieve the optimal supremum likelihood score and the result follows. ∥</p><p>Interestingly,
 the conclusion of Theorem 1 does not apply to models in which the 
relative rates associated with different substitution classes vary. For 
example, even under EK2P (the “extended” K2P model), it is <em>not</em> the case that <em>M</em><em>L</em><sub><em>E</em><em>K</em>2<em>P</em></sub>(<em>T</em>,<em>A</em>) ≤ <em>M</em><em>L</em><sub><em>E</em><em>K</em>2<em>P</em></sub>(<em>T</em>,<em>A</em><sup>′</sup>) for any tree <em>T</em> and alignment <em>A</em>.</p><div class="block-child-p">Consider the following alignment:<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx4_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx4_ht.gif"></div></div>Because the data can be explained without any transversions, the ML estimate of the transition/transversion rate ratio (<em>κ</em>) is infinity. <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>
 shows the tree topologies in Newick notation and the maximum (really 
“supremum”, as discussed above) likelihood score for each tree. We note 
that the ML score for these trees is obtained in the limit, by allowing <em>ν</em>→<em>∞</em> (where <em>ν</em>
 denotes the lengths of certain branches in the trees below). Note that 
on the tree topology that requires only one change, the likelihood can 
approach 1/8, whereas on the other two tree topologies, the ML score 
cannot exceed 1/16. The root state frequency accounts for a factor of 
1/4 in the K2P model. Transversions are disallowed when <em>κ</em> = <em>∞</em>, so<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx5_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx5_ht.gif"></div></div></div>                        <a id="115127088" scrollto-destination="115127088"></a>
<div content-id="tbl1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label">T<span class="small-caps">ABLE</span> 1.</div><div class="caption"><p>Example trees and ML scores</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Treea&nbsp;</td><td>K2P ML scoreb&nbsp;</td></tr></thead><tbody><tr><td>(1:0,2:0,(3:0,4:0):<em>ν</em>)&nbsp;</td><td>1/8&nbsp;</td></tr><tr><td>(1:0,3:<em>ν</em>,(2:0,4:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,3:0,(2:<em>ν</em>,4:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:0,4:<em>ν</em>,(2:0,3:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,4:0,(2:<em>ν</em>,3:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><td>Treea&nbsp;</td><td>K2P ML scoreb&nbsp;</td></tr></thead><tbody><tr><td>(1:0,2:0,(3:0,4:0):<em>ν</em>)&nbsp;</td><td>1/8&nbsp;</td></tr><tr><td>(1:0,3:<em>ν</em>,(2:0,4:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,3:0,(2:<em>ν</em>,4:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:0,4:<em>ν</em>,(2:0,3:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,4:0,(2:<em>ν</em>,3:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p><em>ν</em> denotes the lengths of certain branches in the trees. All other branch lengths are zero.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">b</div><p>The ML score is attained as <em>ν</em> approaches infinity.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115127088" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label">T<span class="small-caps">ABLE</span> 1.</div><div class="caption"><p>Example trees and ML scores</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Treea&nbsp;</td><td>K2P ML scoreb&nbsp;</td></tr></thead><tbody><tr><td>(1:0,2:0,(3:0,4:0):<em>ν</em>)&nbsp;</td><td>1/8&nbsp;</td></tr><tr><td>(1:0,3:<em>ν</em>,(2:0,4:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,3:0,(2:<em>ν</em>,4:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:0,4:<em>ν</em>,(2:0,3:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,4:0,(2:<em>ν</em>,3:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><td>Treea&nbsp;</td><td>K2P ML scoreb&nbsp;</td></tr></thead><tbody><tr><td>(1:0,2:0,(3:0,4:0):<em>ν</em>)&nbsp;</td><td>1/8&nbsp;</td></tr><tr><td>(1:0,3:<em>ν</em>,(2:0,4:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,3:0,(2:<em>ν</em>,4:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:0,4:<em>ν</em>,(2:0,3:<em>ν</em>):0)&nbsp;</td><td>1/16&nbsp;</td></tr><tr><td>(1:<em>ν</em>,4:0,(2:<em>ν</em>,3:0):0)&nbsp;</td><td>1/16&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p><em>ν</em> denotes the lengths of certain branches in the trees. All other branch lengths are zero.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">b</div><p>The ML score is attained as <em>ν</em> approaches infinity.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115127088" target="_blank">View Large</a></div></div><p>Thus, the ML score can be obtained for this data set by multiplying 1/4 by 1/2<sup><em>M</em><em>P</em>(<em>T</em>,<em>A</em>)</sup>, where <em>M</em><em>P</em>(<em>T</em>,<em>A</em>) is the parsimony score of tree <em>T</em>.</p><div class="block-child-p">The trivial alignment, <em>A</em><sup>′</sup>, for this data set would be,<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx6_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx6_ht.gif"></div></div>and
 the ML score (attainable on any tree with all branch lengths set to 
zero) would be 1/16 (the product of the state frequencies at each 
column).</div><p>This demonstrates that, when relative rates of 
substitution are not all equal, it is possible for selection of the 
optimal tree and alignment pair to favor one tree topology over another.
 Thus, for some models, tree inference is technically possible under ML 
while jointly estimating the alignment and treating gaps as missing 
data. Despite this, we emphasize that it will almost never be advisable 
to optimize the alignment without penalizing gaps.</p><p>The following methodology and program commands are identical to the simulation study performed in <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>.</p><p>We simulated data sets for our simulation study in two steps. In Step 1, we generated model trees using r8s (<a reveal-id="bib28" data-open="bib28" class="link link-ref link-reveal xref-bibr">Sanderson 2003</a>), and in Step 2, we simulated sequences on the model trees using ROSE (<a reveal-id="bib33" data-open="bib33" class="link link-ref link-reveal xref-bibr">Stoye et al. 1998</a>).</p>                            <h4 scrollto-destination="115127095" id="115127095" class="section-title">Step 1: Generation of model trees using r8s.</h4>
<p>We used r8s version 1.7 to generate birth–death model trees with 100,
 500, or 1000 taxa. r8s used the following script as input to perform 
this step:</p><ul class="list-simple"><li><p>begin rates;</p></li><li><p>simulate diversemodel=bdback seed = &lt; random seed&gt;</p></li><li><p>ntaxa = &lt; 100 or 500 or 1000&gt; T=0;</p></li><li><p>describe tree=0 plot=tree_description;</p></li><li><p>end;</p></li></ul><p>For each model tree, we deviated the branch lengths away from ultrametricity according to the calculation from <a reveal-id="bib23" data-open="bib23" class="link link-ref link-reveal xref-bibr">Nakhleh et al. (2002)</a> with deviation factor <em>c</em> = 2.0. To perform this calculation, we used the custom program from <a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Liu, Raghavan et al. (2009)</a>, obtained from <a class="link link-uri" href="http://www.cs.utexas.edu/users/tandy/science-paper.html" target="">http://www.cs.utexas.edu/users/tandy/science-paper.html</a>.
 Next, for each model tree, we multiplied the branch lengths by a factor
 which we varied as the “tree height” parameter of simulation. Finally, 
due to ROSE's inability to handle fractional branch lengths in its model
 of evolution, we multiplied all branch lengths by a factor of 100 to 
ensure that branch lengths were generally not less than 1.</p>                            <h4 scrollto-destination="115127099" id="115127099" class="section-title">Step 2: Simulation of sequences using ROSE.</h4>
<p>We used ROSE version 1.3 to simulate the evolution of DNA sequences on each model tree under a GTR+Gamma+Gap model.</p><ul class="bullet"><li><p>GTR+Gamma model of substitution—the GTR model (<a reveal-id="bib27" data-open="bib27" class="link link-ref link-reveal xref-bibr">Rodriguez et al. 1990</a>)
 is parameterized by the frequencies of nucleotides in the root sequence
 and the instantaneous rates of change between nucleotides. For these 
parameters, we used the estimates produced by PAUP* on the NemATOL 
alignment (obtained from the NemATOL website at <a class="link link-uri" href="http://nematol.unh.edu/tree/tree1/vIch20ct682_c1w1.aln" target="">http://nematol.unh.edu/tree/tree1/vIch20ct682_c1w1.aln</a>) of 682 species of nematodes. We used the instantaneous rate matrix to compute the transitional probability matrix <em>P</em>(<em>t</em>) = <em>e</em><sup><em>A</em><em>t</em></sup> which is required as input to ROSE, where <em>A</em> is the instantaneous rate matrix and <em>t</em> = 0.001.</p></li></ul><div class="block-child-p">We used a sequence length of 1000 nucleotides for the root sequence. The frequencies of nucleotides in the root sequence <em>f</em><sub><em>X</em></sub>, where X is a nucleotide, were<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx7_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx7_ht.gif"></div></div></div><div class="block-child-p">The instantaneous rates of change between nucleotides were<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="m_sysbiosyr095fx8_ht.gif" alt="graphic" path-from-xml="sysbiosyr095fx8_ht.gif"></div></div></div><p>We set the shape parameter α of the Gamma distribution, which controls the rate of variation across sites, to 1.0.</p><ul class="bullet"><li><p>Probability
 of a gap event—we set the probability of insertions equal to the 
probability of deletions for each model condition and each model 
condition used a different setting for this parameter.</p></li><li><p>Gap
 length distribution—once an insertion or deletion event occurs, the 
length of that event was drawn from the gap length distribution. We used
 three geometric gap length distributions with finite tails: long, 
medium, and short. The gap length distributions used in our study are 
given by the following lists, where the first element of a list is the 
probability of a gap of length one given that a gap event occurs, the 
second element is the probability of a gap of length two given that a 
gap event occurs, and so forth.</p></li></ul><p>Long gap length distribution:</p><blockquote><p>[0.1028, 0.0899, 0.0792, 0.0702, 0.0627, 0.0565,</p><p>0.0514, 0.0470, 0.0433, 0.0400, 0.0369, 0.0341,</p><p>0.0314, 0.0289, 0.0266, 0.0245, 0.0225, 0.0206,</p><p>0.0188, 0.0171, 0.0155, 0.0141, 0.0127, 0.0114,</p><p>0.0100, 0.0087, 0.0075, 0.0063, 0.0052, 0.0042]</p></blockquote><p></p><p>Medium gap length distribution:</p><blockquote><p>[0.2012, 0.1600, 0.1280, 0.1024, 0.0819,</p><p>0.0655, 0.0524, 0.0419, 0.0336, 0.0268,</p><p>0.0215, 0.0172, 0.0137, 0.0110, 0.0088,</p><p>0.0070, 0.0056, 0.0045, 0.0036, 0.0029,</p><p>0.0023, 0.0018, 0.0015, 0.0012, 0.0009,</p><p>0.0008, 0.0006, 0.0005, 0.0004, 0.0003, 0.0002]</p></blockquote><p></p><p>Short gap length distribution:[0.4613,0.2527,0.1545,0.0896,0.0419]</p><p>The
 long gap length distribution had expected gap length of 9.2 and median 
gap length of 7; the medium gap distribution had expected gap length of 
5.0 and median gap length of 4, and the short gap distribution had 
expected gap length of 2.0 and median gap length of 2.</p><p>In summary,
 there are a total of four parameters that we varied in our simulation: 
the number of taxa, the tree height, the probability of a gap event, and
 the gap length distribution. Table S7 provides the parameter settings 
that constitute each of the fifteen 1000-taxon model conditions, and 
similarly for Table S8 for the fifteen 500-taxon model conditions and 
Table S1 for the fifteen 100-taxon model conditions.</p><p>For each 
model condition, we repeated Step 1 to obtain 20 different model trees, 
and for each model tree, we performed Step 2 once to simulate one data 
set. This produced 20 replicate data sets per model condition.</p><p>The
 empirical statistics of the true alignments and true trees from each 
model condition are listed in Table S7 for the 1000-taxon model 
conditions, Table S8 for the 500-taxon model conditions, and Table S1 
for the 100-taxon model conditions.</p><p>The complete script used as input to ROSE for Step 2 was</p><ul class="list-simple"><li><p>TheInsFunc =&lt;Insert event gap length distribution - long, medium or short&gt;</p></li><li><p>TheDelFunc =&lt;Delete event gap length distribution - long, medium or short&gt;</p></li><li><p>InputType = 4</p></li><li><p>TheAlphabet = “ACGT”</p></li><li><p>TheFreq =[.300414,.191363,.196748,.311475]</p></li><li><p>ThePAMMatrix =</p></li><li><p>[[0.9948, 0.0012, 0.0035, 0.0005],</p></li><li><p>[0.0012, 0.9933, 0.0011, 0.0044],</p></li><li><p>[0.0035, 0.0011, 0.9944, 0.0010],</p></li><li><p>[0.0005, 0.0044, 0.0010, 0.9941]]</p></li><li><p>TheInsertThreshold =&lt;Insertion event probability&gt;</p></li><li><p>TheDeleteThreshold =&lt;Deletion event probability&gt;</p></li><li><p>SequenceLen = 1000</p></li><li><p>TheTree = &lt; birth-death model tree in Newick format with branch lengths, deviated from ultrametricity&gt;</p></li><li><p>ChooseFromLeaves = False</p></li><li><p>AlignmentWithAncestors = True</p></li><li><p>TreeWithAncestors = True</p></li><li><p>SequenceNum =&lt;99 or 499 or 999&gt;</p></li><li><p>SeedVal =&lt;random seed integer&gt;</p></li><li><p>TheMutationProbability = &lt; site-by-site vector listing rate multipliers for that site&gt;</p></li></ul><p>Several
 programs were invoked during the course of our work to estimate 
multiple sequence alignments. These programs were ClustalW version 
2.0.4, MAFFT version 6.240, Muscle version 3.7, Prank+GT using Prank 
version 080904 and using RAxML (MAFFT) as a guide tree, and Opal version
 1.0.2, which required Java version 1.6.0_02. Below are the commands 
used for each program, where &lt;input&gt; is a FASTA-formatted input 
file containing unaligned sequences and &lt;output&gt; is the resulting 
FASTA-formatted output file. Note that we ran MAFFT and ClustalW each in
 two ways: one uses the default, and more accurate, command, and the 
other uses the command that can run on large data sets.</p><ul class="bullet"><li><p>ClustalW (default): clustalw2 -align -infile=&lt;input&gt; -outfile=&lt;output&gt; -output=fasta</p></li><li><p>ClustalW-Quicktree: clustalw2 -align -infile=&lt;input&gt; -outfile=&lt;output&gt; -output=fasta -quicktree</p></li><li><p>MAFFT (default) mafft –localpair –maxiterate 1000 –quiet &lt;input&gt; &gt;&lt;output&gt;</p></li><li><p>MAFFT-PartTree: mafft –parttree –retree 2 –partsize 1000 &lt;input&gt; &gt;&lt;output&gt;</p></li><li><p>Muscle: muscle -in &lt;input&gt; -out &lt;output&gt; -quiet</p></li><li><p>Prank+GT:
 prank -d=&lt;input&gt; -o=&lt;output&gt; -t=&lt;RAxML(MAFFT) guide 
tree&gt; -noxml -notree -nopost +F -matinitsize=5 -uselogs</p></li><li><p>Opal: java -jar Opal.1.0.2.jar –in &lt;input&gt; –out &lt;output&gt;</p></li></ul><p>We
 also used Muscle version 3.7 and Opal version 1.0.2 to merge two input 
alignments into an output alignment. We list the commands used for each 
program below, where &lt;input 1&gt; and &lt;input 2&gt; are 
FASTA-formatted input files each containing a multiple sequence 
alignment and &lt;output&gt; is the resulting FASTA-formatted output 
file:</p><ul class="bullet"><li><p>Opal merge: java -jar Opal.1.0.2.jar –in &lt;input 1&gt; –in2 &lt;input 2&gt; –out &lt;output&gt; –align_method profile</p></li><li><p>Muscle merge: muscle -profile -in1 &lt;input 1&gt; -in2 &lt;input 2&gt; -out &lt;output&gt; -quiet</p></li></ul><p>ML
 trees for all data sets other than the two largest biological data sets
 (16S.T and 16S.B.ALL) were estimated using RAxML version 7.0.4. All 
two-phase tree estimations on alignments generated by the above MSA 
methods used the default RAxML command below. All two-phase RAxML 
estimations were run to completion. &lt;input&gt; is a PHYLIP-formatted 
input alignment file.</p><ul class="bullet"><li><p>RAxML default: raxmlHPC -m GTRMIX -w &lt;work dir&gt; -n &lt;identifying suffix&gt; -s &lt;input&gt;</p></li></ul><p>On
 the 16S.T data set, we used several different versions and commands for
 RAxML, as listed below. We ran an alpha-version 7.2.4_sse3 of RAxML 
which makes use of a stopping rule that permits faster runs than the 
default RAxML version 7.0.4.</p><ul class="bullet"><li><p>RAxML default 
version 7.0.4, used for two-phase ML tree estimation: raxmlHPC -m GTRMIX
 -w &lt;work dir&gt; -n &lt;identifying suffix&gt; -s &lt;input&gt;</p></li><li><p>RAxML
 alpha-version 7.2.4_sse3 fast search, used for SATé-I and SATé-II: 
raxmlHPC-SSE3 -F -D -m GTRCAT -n &lt;identifying suffix&gt; -s 
&lt;input&gt;</p></li><li><p>RAxML alpha-version 7.2.4_sse3 score and 
report branch lengths, used for SATé-I and SATé-II: raxmlHPC-SSE3 -m 
GTRGAMMA -n &lt;identifying suffix&gt; -s &lt;input alignment&gt; -t 
&lt;input tree&gt; -f e</p></li></ul><p>On the 16S.B.ALL data set, we 
used several different versions and commands for ML tree estimation, as 
listed below. Due to the time and space requirements for RAxML on this 
data set, SATé-II used FastTree (<a reveal-id="bib25" data-open="bib25" class="link link-ref link-reveal xref-bibr">Price et al. 2010</a>)
 in place of RAxML as an ML tree estimator on a fixed alignment. Our 
attempts to run SATé-I on this data set failed prior to any ML tree 
estimation.</p><ul class="bullet"><li><p>RAxML default version 7.0.4, 
used for two-phase ML tree estimation: raxmlHPC -m GTRMIX -w &lt;work 
dir&gt; -n &lt;identifying suffix&gt; -s &lt;input&gt;</p></li><li><p>FastTree
 version 2.1.3, used for SATé-II's starting tree and during search by 
SATé-II: FastTree -nt -gtr -nosupport -log &lt;log file&gt;&lt;input 
alignment&gt; &gt;&lt;output tree&gt;</p></li></ul><p>To construct a 
curation-based tree on the biological data sets other than 16S.T and 
16S.B.ALL, we used the following command in RAxML version 7.0.4 to 
estimate an ML tree on the curated alignment and assign support values 
to the branches of the ML tree using a rapid bootstrapping analysis with
 500 bootstrap replicates:</p><ul class="bullet"><li><p>RAxML rapid 
bootstrap: raxmlHPC -f a -m GTRGAMMA -s &lt;input&gt; -n &lt;identifying
 suffix&gt; -x &lt;random number&gt; -p &lt;random number&gt; -N 
&lt;number of replicates&gt;</p></li></ul><p>To construct a 
curation-based tree on the 16S.T data set, we estimated an ML tree on 
the curated alignment using RAxML version 7.2.6 with the RAxML default 
command above. Then, we performed a rapid bootstrap analysis with 346 
bootstrap replicates using RAxML version 7.0.4 and the RAxML rapid 
bootstrap command above to assign support values to the branches of the 
ML tree.</p><p>To construct a curation-based tree on the 16S.B.ALL data 
set, we estimated an ML tree on the curated alignment using RAxML 
version 7.2.6 with the RAxML default command listed above. Next, a 
standard bootstrap analysis with 444 bootstrap replicates was performed 
using RAxML version 7.2.5 on TACC's Ranger supercomputing cluster to 
assign support values to the branches of the ML tree. The RAxML analysis
 of each bootstrap replicate was parallelized using the 4 cores of a 
single compute node on Ranger via the following command.</p><ul class="bullet"><li><p>RAxML analysis of a single standard bootstrap replicate:</p><blockquote><p>raxmlHPC-PTHREADS-SSE3 -m GTRCAT -s &lt;input&gt;</p><p>-n &lt;identifying suffix&gt; -b &lt;random number&gt;</p><p>-p &lt;random number&gt; -N 1 -T 4'</p></blockquote><p></p></li></ul><p>When
 necessary, we enabled multithreading for the RAxML runs either by 
recompiling with PTHREADS and running with an additional flag -T 
&lt;number of threads&gt; or by recompiling with MPI for the rapid 
bootstrapping analysis; this parallelization does not otherwise change 
the RAxML commands. The RAxML outputs are unaffected by parallelization,
 and all reported runtimes are for serialized execution.</p>                            <h2 scrollto-destination="115127132" id="115127132" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="bib1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Hochberg</div> <div class="given-names">Y</div></div>.&nbsp;</div><div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>,&nbsp;<div class="source">J. R. Stat. Soc. Series B Met</div>&nbsp;,&nbsp;<div class="year">1995</div>,&nbsp;vol.&nbsp;<div class="volume">57</div>&nbsp;(pg.&nbsp;<div class="fpage">289</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="bib2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannone</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Subramanian</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Schnare</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Collett</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">D'Souza</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Du</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Feng</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Lin</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Madabusi</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Muller</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Pande</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Shang</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Yu</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Gutell</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">The
 comparative RNA web (CRW) site: an online database of comparative 
sequence and structure information for ribosomal, intron and other RNAs</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;pg.&nbsp;<div class="fpage">2</div>&nbsp;&nbsp;<div class="comment">Available from: <a class="link link-uri" href="http://www.rna.ccbb.utexas.edu/" target="">http://www.rna.ccbb.utexas.edu</a>. Accessed 4 October 2011</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20comparative%20RNA%20web%20%28CRW%29%20site%3A%20an%20online%20database%20of%20comparative%20sequence%20and%20structure%20information%20for%20ribosomal%2C%20intron%20and%20other%20RNAs&amp;author=J%20Cannone&amp;author=S%20Subramanian&amp;author=M%20Schnare&amp;author=J%20Collett&amp;author=L%20D%27Souza&amp;author=Y%20Du&amp;author=B%20Feng&amp;author=N%20Lin&amp;author=L%20Madabusi&amp;author=K%20Muller&amp;author=N%20Pande&amp;author=Z%20Shang&amp;author=N%20Yu&amp;author=R%20Gutell&amp;publication_year=2002&amp;journal=BMC%20Bioinformatics&amp;volume=3&amp;pages=2" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-3-2" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-3-2" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-3-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11869452" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>.&nbsp;</div><div class="article-title">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">5</div>&nbsp;pg.&nbsp;<div class="fpage">113</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;author=RC%20Edgar&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=113" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-113" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-113" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-5-113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15318951" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>.&nbsp;</div><div class="article-title">MUSCLE: a multiple sequence alignment with high accuracy and high throughput</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">32</div>&nbsp;(pg.&nbsp;<div class="fpage">1792</div>-<div class="lpage">1797</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1792-1797" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fleissner</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Metzler</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Simultaneous statistical multiple alignment and phylogeny reconstruction</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">54</div>&nbsp;(pg.&nbsp;<div class="fpage">548</div>-<div class="lpage">561</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simultaneous%20statistical%20multiple%20alignment%20and%20phylogeny%20reconstruction&amp;author=R%20Fleissner&amp;author=D%20Metzler&amp;author=A%20von%20Haeseler&amp;publication_year=2005&amp;journal=Syst.%20Biol.&amp;volume=54&amp;pages=548-561" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150590950371" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150590950371" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150590950371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085574" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guindon</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">52</div>&nbsp;(pg.&nbsp;<div class="fpage">696</div>-<div class="lpage">704</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=S%20Guindon&amp;author=O%20Gascuel&amp;publication_year=2003&amp;journal=Syst.%20Biol.&amp;volume=52&amp;pages=696-704" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150390235520" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutell</div> <div class="given-names">RR</div></div>,&nbsp; <div class="name"><div class="surname">Larsen</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Woese</div> <div class="given-names">CR</div></div>.&nbsp;</div><div class="article-title">Lessons from an evolving ribosomal-RNA—16S and 23S ribosomal RNA structures from a comparative perspective</div>,&nbsp;<div class="source">Microbiol. Rev.</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">58</div>&nbsp;(pg.&nbsp;<div class="fpage">10</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lessons%20from%20an%20evolving%20ribosomal-RNA%E2%80%9416S%20and%2023S%20ribosomal%20RNA%20structures%20from%20a%20comparative%20perspective&amp;author=RR%20Gutell&amp;author=N%20Larsen&amp;author=CR%20Woese&amp;publication_year=1994&amp;journal=Microbiol.%20Rev.&amp;volume=58&amp;pages=10-26" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8177168" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutell</div> <div class="given-names">RR</div></div>,&nbsp; <div class="name"><div class="surname">Lee</div> <div class="given-names">JC</div></div>,&nbsp; <div class="name"><div class="surname">Cannone</div> <div class="given-names">JJ</div></div>.&nbsp;</div><div class="article-title">The accuracy of ribosomal RNA comparative structure models. Curr. Opin</div>,&nbsp;<div class="source">Struct. Biol.</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">301</div>-<div class="lpage">310</div>)</div></div></div></div><div content-id="bib9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Unified approach to alignment and phylogenies</div>,&nbsp;<div class="source">Methods Enzymol</div>&nbsp;,&nbsp;<div class="year">1990</div>,&nbsp;vol.&nbsp;<div class="volume">183</div>&nbsp;(pg.&nbsp;<div class="fpage">626</div>-<div class="lpage">645</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unified%20approach%20to%20alignment%20and%20phylogenies&amp;author=J%20Hein&amp;publication_year=1990&amp;journal=Methods%20Enzymol&amp;volume=183&amp;pages=626-645" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0076-6879%2890%2983041-7" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0076-6879%2890%2983041-7" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0076-6879(90)83041-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2314296" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hillis</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Bull</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">An empirical test of bootstrapping as a method for assessing confidence in phylogenetic analysis</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">42</div>&nbsp;(pg.&nbsp;<div class="fpage">182</div>-<div class="lpage">192</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20empirical%20test%20of%20bootstrapping%20as%20a%20method%20for%20assessing%20confidence%20in%20phylogenetic%20analysis&amp;author=D%20Hillis&amp;author=J%20Bull&amp;publication_year=1993&amp;journal=Syst.%20Biol.&amp;volume=42&amp;pages=182-192" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/42.2.182" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F42.2.182" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F42.2.182"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jukes</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Cantor</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Munro</div> <div class="given-names">HN</div></div>,&nbsp; <div class="name"><div class="surname">Allison</div> <div class="given-names">JB</div></div>.&nbsp;</div><div class="article-title">Evolution of protein molecules</div>,&nbsp;<div class="source">Mammalian protein metabolism</div>&nbsp;,&nbsp;<div class="year">1969</div><div class="publisher-loc">New York</div><div class="publisher-name">Academic Press</div>(pg.&nbsp;<div class="fpage">21</div>-<div class="lpage">132</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20protein%20molecules&amp;author=T%20Jukes&amp;author=C%20Cantor&amp;author=HN%20Munro&amp;author=JB%20Allison&amp;publication_year=1969&amp;book=Mammalian%20protein%20metabolism" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/B978-1-4832-3211-9.50009-7" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FB978-1-4832-3211-9.50009-7" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FB978-1-4832-3211-9.50009-7"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Kuma</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Miyata</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">MAFFT version 5: improvement in accuracy of multiple sequence alignment</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">511</div>-<div class="lpage">518</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%20version%205%3A%20improvement%20in%20accuracy%20of%20multiple%20sequence%20alignment&amp;author=K%20Katoh&amp;author=K%20Kuma&amp;author=H%20Toh&amp;author=T%20Miyata&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=511-518" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki198" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki198" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661851" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">PartTree: an algorithm to build an approximate tree from a large number of unaligned sequences</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;(pg.&nbsp;<div class="fpage">372</div>-<div class="lpage">374</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PartTree%3A%20an%20algorithm%20to%20build%20an%20approximate%20tree%20from%20a%20large%20number%20of%20unaligned%20sequences&amp;author=K%20Katoh&amp;author=H%20Toh&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=372-374" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl592" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl592" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17118958" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Recent developments in the MAFFT multiple sequence alignment program</div>,&nbsp;<div class="source">Brief. Bioinform</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;(pg.&nbsp;<div class="fpage">286</div>-<div class="lpage">298</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%20developments%20in%20the%20MAFFT%20multiple%20sequence%20alignment%20program&amp;author=K%20Katoh&amp;author=H%20Toh&amp;publication_year=2008&amp;journal=Brief.%20Bioinform&amp;volume=9&amp;pages=286-298" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn013" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn013" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbib%2Fbbn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18372315" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kececioglu</div> <div class="given-names">JD</div></div>,&nbsp; <div class="name"><div class="surname">Zhang</div> <div class="given-names">W</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Farach</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Aligning alignments</div>,&nbsp;<div class="source">Proceedings of CPM 1998 Lecture Notes in Computer Science</div>&nbsp;,&nbsp;<div class="year">1998</div><div class="publisher-loc">Berlin (Germany)</div><div class="publisher-name">Springer</div>(pg.&nbsp;<div class="fpage">189</div>-<div class="lpage">208</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aligning%20alignments&amp;author=JD%20Kececioglu&amp;author=W%20Zhang&amp;author=M%20Farach&amp;publication_year=1998&amp;book=Proceedings%20of%20CPM%201998%20Lecture%20Notes%20in%20Computer%20Science" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BFb0030776" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBFb0030776" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBFb0030776"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences</div>,&nbsp;<div class="source">J. Mol. Evol.</div>&nbsp;,&nbsp;<div class="year">1980</div>,&nbsp;vol.&nbsp;<div class="volume">16</div>&nbsp;(pg.&nbsp;<div class="fpage">111</div>-<div class="lpage">120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20estimating%20evolutionary%20rates%20of%20base%20substitutions%20through%20comparative%20studies%20of%20nucleotide%20sequences&amp;author=M%20Kimura&amp;publication_year=1980&amp;journal=J.%20Mol.%20Evol.&amp;volume=16&amp;pages=111-120" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01731581" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01731581" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF01731581"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7463489" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lancia</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Ravi</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">GESTALT: Genomic Steiner alignments</div>,&nbsp;<div class="source">Lect. Notes Comput. Sci.</div>&nbsp;,&nbsp;<div class="year">1999</div>,&nbsp;vol.&nbsp;<div class="volume">1645</div>&nbsp;(pg.&nbsp;<div class="fpage">101</div>-<div class="lpage">114</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GESTALT%3A%20Genomic%20Steiner%20alignments&amp;author=G%20Lancia&amp;author=R%20Ravi&amp;publication_year=1999&amp;journal=Lect.%20Notes%20Comput.%20Sci.&amp;volume=1645&amp;pages=101-114" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/3-540-48452-3" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F3-540-48452-3" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2F3-540-48452-3"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Litzkow</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Remote Unix—turning idle workstations into cycle servers. Usenix summer conference. USENIX Association</div>,&nbsp;<div class="year">1987</div>(pg.&nbsp;<div class="fpage">381</div>-<div class="lpage">384</div>)</div></div></div></div><div content-id="bib19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Linder</div> <div class="given-names">CR</div></div>,&nbsp; <div class="name"><div class="surname">Warnow</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">Multiple sequence alignment: a major challenge to large-scale phylogenetics. PLoS Curr. Tree of Life</div>,&nbsp;<div class="year">2010</div>&nbsp;<div class="comment">Available from: <a class="link link-uri" href="http://knol.google.com/k/kevin-liu/multiple-sequence-alignment-a-major/ectabesw3uba/9" target="">http://knol.google.com/k/kevin-liu/multiple-sequence-alignment-a-major/ectabesw3uba/9</a>. Accessed 4 October 2011</div></div></div></div></div><div content-id="bib20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Nelesen</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Raghavan</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Linder</div> <div class="given-names">CR</div></div>,&nbsp; <div class="name"><div class="surname">Warnow</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">Barking up the wrong treelength: the impact of gap penalty on alignment and tree accuracy</div>,&nbsp;<div class="source">IEEE/ACM Trans. Comput. Biol. Bioinform</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">7</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Barking%20up%20the%20wrong%20treelength%3A%20the%20impact%20of%20gap%20penalty%20on%20alignment%20and%20tree%20accuracy&amp;author=K%20Liu&amp;author=S%20Nelesen&amp;author=S%20Raghavan&amp;author=CR%20Linder&amp;author=T%20Warnow&amp;publication_year=2009&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol.%20Bioinform&amp;volume=6&amp;pages=7-21" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TCBB.2008.63" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTCBB.2008.63" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1109%2FTCBB.2008.63"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19179695" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Raghavan</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Nelesen</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Linder</div> <div class="given-names">CR</div></div>,&nbsp; <div class="name"><div class="surname">Warnow</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">Rapid and accurate large-scale coestimation of sequence alignments and phylogenetic trees</div>,&nbsp;<div class="source">Science</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">324</div>&nbsp;(pg.&nbsp;<div class="fpage">1561</div>-<div class="lpage">1564</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20accurate%20large-scale%20coestimation%20of%20sequence%20alignments%20and%20phylogenetic%20trees&amp;author=K%20Liu&amp;author=S%20Raghavan&amp;author=S%20Nelesen&amp;author=CR%20Linder&amp;author=T%20Warnow&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=1561-1564" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1171243" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1171243" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1171243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541996" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lunter</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Miklós</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Drummond</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Jensen</div> <div class="given-names">JL</div></div>,&nbsp; <div class="name"><div class="surname">Hein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Bayesian coestimation of phylogeny and sequence alignment</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;pg.&nbsp;<div class="fpage">83</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bayesian%20coestimation%20of%20phylogeny%20and%20sequence%20alignment&amp;author=G%20Lunter&amp;author=I%20Mikl%C3%B3s&amp;author=A%20Drummond&amp;author=JL%20Jensen&amp;author=J%20Hein&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=83" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-83" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-83" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-6-83"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15804354" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakhleh</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Moret</div> <div class="given-names">BME</div></div>,&nbsp; <div class="name"><div class="surname">Roshan</div> <div class="given-names">U</div></div>,&nbsp; <div class="name"><div class="surname">Sun J</div> <div class="given-names">St. John K.</div></div>,&nbsp; <div class="name"><div class="surname">Warnow</div> <div class="given-names">T</div></div>.&nbsp;</div>,&nbsp;<div class="source">The accuracy of fast phylogenetic methods for large datasets. Proceedings of the 7th Pacific Symposium on BioComputing (PSB02)</div>&nbsp;,&nbsp;<div class="year">2002</div>&nbsp;<div class="comment">2002 January 3–7; Lihue, Hawaii. p. 211–222</div></div></div></div></div><div content-id="bib24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Novák Á</div> <div class="given-names">Miklós I</div></div>,&nbsp; <div class="name"><div class="surname">Lyngsoe</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Hein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">StatAlign: an extendable software package for joint Bayesian estimation of alignments and evolutionary trees</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">24</div>&nbsp;(pg.&nbsp;<div class="fpage">2403</div>-<div class="lpage">2404</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=StatAlign%3A%20an%20extendable%20software%20package%20for%20joint%20Bayesian%20estimation%20of%20alignments%20and%20evolutionary%20trees&amp;author=Mikl%C3%B3s%20I%20Nov%C3%A1k%20%C3%81&amp;author=R%20Lyngsoe&amp;author=J%20Hein&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2403-2404" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn457" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn457" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtn457"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18753153" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Dehal</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Arkin</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">FastTree 2—approximately maximum-likelihood trees for large alignments</div>,&nbsp;<div class="source">PLoS One</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">5</div>&nbsp;pg.&nbsp;<div class="fpage">e9490</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FastTree%202%E2%80%94approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;author=M%20Price&amp;author=P%20Dehal&amp;author=A%20Arkin&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e9490" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009490" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009490" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0009490"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20224823" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Redelings</div> <div class="given-names">BD</div></div>,&nbsp; <div class="name"><div class="surname">Suchard</div> <div class="given-names">MA</div></div>.&nbsp;</div><div class="article-title">Joint Bayesian estimation of alignment and phylogeny</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">54</div>&nbsp;(pg.&nbsp;<div class="fpage">401</div>-<div class="lpage">418</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Joint%20Bayesian%20estimation%20of%20alignment%20and%20phylogeny&amp;author=BD%20Redelings&amp;author=MA%20Suchard&amp;publication_year=2005&amp;journal=Syst.%20Biol.&amp;volume=54&amp;pages=401-418" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150590947041" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150590947041" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150590947041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16012107" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rodriguez</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Oliver</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Marin</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Medina</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">The general stochastic model of nucleotide substitution</div>,&nbsp;<div class="source">J. Theor. Biol.</div>&nbsp;,&nbsp;<div class="year">1990</div>,&nbsp;vol.&nbsp;<div class="volume">142</div>&nbsp;(pg.&nbsp;<div class="fpage">485</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20general%20stochastic%20model%20of%20nucleotide%20substitution&amp;author=F%20Rodriguez&amp;author=J%20Oliver&amp;author=A%20Marin&amp;author=J%20Medina&amp;publication_year=1990&amp;journal=J.%20Theor.%20Biol.&amp;volume=142&amp;pages=485-501" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-5193%2805%2980104-3" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-5193%2805%2980104-3" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0022-5193(05)80104-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2338834" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sanderson</div> <div class="given-names">MJ</div></div>.&nbsp;</div><div class="article-title">r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">19</div>&nbsp;(pg.&nbsp;<div class="fpage">301</div>-<div class="lpage">302</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=r8s%3A%20inferring%20absolute%20rates%20of%20molecular%20evolution%20and%20divergence%20times%20in%20the%20absence%20of%20a%20molecular%20clock&amp;author=MJ%20Sanderson&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=301-302" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.301" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F19.2.301" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F19.2.301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12538260" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sankoff</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">Minimal mutation trees of sequences</div>,&nbsp;<div class="source">SIAM J. Appl. Math.</div>&nbsp;,&nbsp;<div class="year">1975</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">35</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimal%20mutation%20trees%20of%20sequences&amp;author=D%20Sankoff&amp;publication_year=1975&amp;journal=SIAM%20J.%20Appl.%20Math.&amp;volume=28&amp;pages=35-42" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1137/0128004" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1137%2F0128004" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1137%2F0128004"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sankoff</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Cedergren</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Sankoff</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Kruskal</div> <div class="given-names">JB</div></div>.&nbsp;</div><div class="article-title">Simultaneous comparison of three or more sequences related by a tree</div>,&nbsp;<div class="year">1983</div>&nbsp;<div class="comment">Time
 warps, string edits, and macromolecules: the theory and practice of 
sequence comparison. Chapter 2. Reading (MA): Addison-Wesley. p. 253–264</div></div></div></div></div><div content-id="bib31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib31" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Beaulieu</div> <div class="given-names">JM</div></div>,&nbsp; <div class="name"><div class="surname">Donoghue</div> <div class="given-names">MJ</div></div>.&nbsp;</div><div class="article-title">Mega-phylogeny approach for comparative biology: an alternative to supertree and supermatrix approaches</div>,&nbsp;<div class="source">BMC Evol. Biol.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;pg.&nbsp;<div class="fpage">37</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mega-phylogeny%20approach%20for%20comparative%20biology%3A%20an%20alternative%20to%20supertree%20and%20supermatrix%20approaches&amp;author=S%20Smith&amp;author=JM%20Beaulieu&amp;author=MJ%20Donoghue&amp;publication_year=2009&amp;journal=BMC%20Evol.%20Biol.&amp;volume=9&amp;pages=37" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-9-37" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-9-37" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2148-9-37"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19210768" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib32" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">2688</div>-<div class="lpage">2690</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A%20Stamatakis&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2688-2690" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl446" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib33" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoye</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Evers</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Meyer</div> <div class="given-names">F</div></div>.&nbsp;</div><div class="article-title">Rose: generating sequence families</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">157</div>-<div class="lpage">163</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rose%3A%20generating%20sequence%20families&amp;author=J%20Stoye&amp;author=D%20Evers&amp;author=F%20Meyer&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=157-163" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.2.157" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.2.157" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F14.2.157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9545448" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib34" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suchard</div> <div class="given-names">MA</div></div>,&nbsp; <div class="name"><div class="surname">Redelings</div> <div class="given-names">BD</div></div>.&nbsp;</div><div class="article-title">BAli-Phy: simultaneous Bayesian inference of alignment and phylogeny</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">2047</div>-<div class="lpage">2048</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BAli-Phy%3A%20simultaneous%20Bayesian%20inference%20of%20alignment%20and%20phylogeny&amp;author=MA%20Suchard&amp;author=BD%20Redelings&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2047-2048" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl175" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl175" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16679334" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib35" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swofford</div> <div class="given-names">DL</div></div>.&nbsp;</div><div class="article-title">PAUP*: phylogenetic analysis using parsimony (*and other methods). version 4</div>,&nbsp;<div class="year">2003</div><div class="publisher-loc">Sunderland (MA)</div><div class="publisher-name">Sinauer Associates</div></div></div></div></div><div content-id="bib36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib36" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Higgins</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Gibson</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">CLUSTAL
 W: improving the sensitivity of progressive multiple sequence alignment
 through sequence weighting, position-specific gap penalties and weight 
matrix choice</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">4673</div>-<div class="lpage">4680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J%20Thompson&amp;author=D%20Higgins&amp;author=T%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-4680" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib37" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">JD</div></div>,&nbsp; <div class="name"><div class="surname">Plewniak</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Poch</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">A comprehensive comparison of multiple sequence alignment programs</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1999</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">2682</div>-<div class="lpage">2690</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20comparison%20of%20multiple%20sequence%20alignment%20programs&amp;author=JD%20Thompson&amp;author=F%20Plewniak&amp;author=O%20Poch&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=2682-2690" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.13.2682" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.13.2682" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F27.13.2682"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10373585" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib38" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thorne</div> <div class="given-names">JL</div></div>,&nbsp; <div class="name"><div class="surname">Kishino</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">An evolutionary model for maximum likelihood alignment of DNA sequences</div>,&nbsp;<div class="source">J. Mol. Evol.</div>&nbsp;,&nbsp;<div class="year">1991</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">114</div>-<div class="lpage">124</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20model%20for%20maximum%20likelihood%20alignment%20of%20DNA%20sequences&amp;author=JL%20Thorne&amp;author=H%20Kishino&amp;author=J%20Felsenstein&amp;publication_year=1991&amp;journal=J.%20Mol.%20Evol.&amp;volume=33&amp;pages=114-124" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02193625" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02193625" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02193625"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1920447" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib39" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thorne</div> <div class="given-names">JL</div></div>,&nbsp; <div class="name"><div class="surname">Kishino</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Inching toward reality: an improved likelihood model of sequence evolution</div>,&nbsp;<div class="source">J. Mol. Evol.</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">34</div>&nbsp;(pg.&nbsp;<div class="fpage">3</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inching%20toward%20reality%3A%20an%20improved%20likelihood%20model%20of%20sequence%20evolution&amp;author=JL%20Thorne&amp;author=H%20Kishino&amp;author=J%20Felsenstein&amp;publication_year=1992&amp;journal=J.%20Mol.%20Evol.&amp;volume=34&amp;pages=3-16" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00163848" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00163848" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF00163848"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1556741" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib40" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuffley</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Steel</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Links between maximum likelihood and maximum parsimony under a simple model of site substitution</div>,&nbsp;<div class="source">Bull. Math. Biol.</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">59</div>&nbsp;(pg.&nbsp;<div class="fpage">581</div>-<div class="lpage">607</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Links%20between%20maximum%20likelihood%20and%20maximum%20parsimony%20under%20a%20simple%20model%20of%20site%20substitution&amp;author=C%20Tuffley&amp;author=M%20Steel&amp;publication_year=1997&amp;journal=Bull.%20Math.%20Biol.&amp;volume=59&amp;pages=581-607" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02459467" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02459467" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02459467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9172826" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib41" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varón</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Vinh</div> <div class="given-names">LS</div></div>,&nbsp; <div class="name"><div class="surname">Wheeler</div> <div class="given-names">WC</div></div>.&nbsp;</div><div class="article-title">POY version 4: phylogenetic analysis using dynamic homologies</div>,&nbsp;<div class="source">Cladistics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">72</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=POY%20version%204%3A%20phylogenetic%20analysis%20using%20dynamic%20homologies&amp;author=A%20Var%C3%B3n&amp;author=LS%20Vinh&amp;author=WC%20Wheeler&amp;publication_year=2010&amp;journal=Cladistics&amp;volume=26&amp;pages=72-85" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cla.2009.26.issue-1" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcla.2009.26.issue-1" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fcla.2009.26.issue-1"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib42" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">WC</div></div>.&nbsp;</div><div class="article-title">Sequence alignment, parameter sensitivity, and the phylogenetic analysis of molecular data</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">1995</div>,&nbsp;vol.&nbsp;<div class="volume">44</div>&nbsp;(pg.&nbsp;<div class="fpage">321</div>-<div class="lpage">331</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20alignment%2C%20parameter%20sensitivity%2C%20and%20the%20phylogenetic%20analysis%20of%20molecular%20data&amp;author=WC%20Wheeler&amp;publication_year=1995&amp;journal=Syst.%20Biol.&amp;volume=44&amp;pages=321-331" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/44.3.321" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F44.3.321" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F44.3.321"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib43" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">WC</div></div>.&nbsp;</div><div class="article-title">Optimization alignment: the end of multiple sequence alignment in phylogenetics?</div>,&nbsp;<div class="source">Cladistics</div>&nbsp;,&nbsp;<div class="year">1996</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">1</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20alignment%3A%20the%20end%20of%20multiple%20sequence%20alignment%20in%20phylogenetics%3F&amp;author=WC%20Wheeler&amp;publication_year=1996&amp;journal=Cladistics&amp;volume=12&amp;pages=1-9" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cla.1996.12.issue-1" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcla.1996.12.issue-1" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fcla.1996.12.issue-1"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib44" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">WC</div></div>,&nbsp; <div class="name"><div class="surname">Gladstein</div> <div class="given-names">DS</div></div>.&nbsp;</div><div class="article-title">MALIGN: a multiple sequence alignment program</div>,&nbsp;<div class="source">J. Hered</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">85</div>&nbsp;(pg.&nbsp;<div class="fpage">417</div>-<div class="lpage">418</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MALIGN%3A%20a%20multiple%20sequence%20alignment%20program&amp;author=WC%20Wheeler&amp;author=DS%20Gladstein&amp;publication_year=1994&amp;journal=J.%20Hered&amp;volume=85&amp;pages=417-418" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jhered.a111492" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jhered.a111492" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.jhered.a111492"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib45" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheeler</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Kececioglu</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Multiple alignment by aligning alignments</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;(pg.&nbsp;<div class="fpage">i559</div>-<div class="lpage">i568</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20alignment%20by%20aligning%20alignments&amp;author=T%20Wheeler&amp;author=J%20Kececioglu&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=i559-i568" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm226" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm226" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtm226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646343" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">© The Author(s) 2011. 
Published by Oxford University Press, on behalf of the Society of 
Systematic Biologists. All rights reserved. For Permissions, please 
email: journals.permissions@oup.com</div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=1680002&amp;xsltPath=%7E/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/61/1/10.1093_sysbio_syr095/4/syr095_supplementary_data.zip?Expires=2147483647&amp;Signature=1xOaW3LVtkjXCgetiu2CgiuTBjaE2aFbExDo-ZriyJDw1qsmAzYyV0d18wr9L74PTUeVvUuOU-o6HOHEz1qQurKHynduI5LgBKUPPB8DtLi%7ESbyvUvq6V%7ErvbaBYpvJgzvuyN1m3iZUJBZyMf6tnzgpAUymDhB9SRMYnMhBfUzL8BJ-qKcmDUw90Aflfyn8tUIEYecM5yCYCCwC9odH2%7EZpfghYX4nZcuIqzajjrV42P-4L9JRsEtDhjwwd%7E7UIkKeHo7DerZFsLEKFyTG%7EfU0kw3s33tm%7ED8psWcPNeF7GRXFl4tP8ZVKV%7EZ%7EkHqPNmUVWfE4GFcaPmCSEGwUfYuQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad2_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Supplement_Ad2_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Supplement_Ad2_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">907</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000299195700007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">169</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/sysbio/syr095" data-uuid="8d2b7338-6b8c-d0fc-072c-29f02b615e8a"><a style="cursor: pointer; display: inline-block; background-image: url(&quot;https://badges.altmetric.com/?size=64&amp;score=?&amp;types=????????&amp;style=donut&quot;); background-repeat: no-repeat; width: 64px; height: 64px;" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;doi=10.1093%2Fsysbio%2Fsyr095" target="_self"> </a></div>
         <script type="text/javascript" src="embed.js"></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1680002"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000299195700007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fsysbio%2farticle%2f61%2f1%2f90%2f1680002">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000299195700007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (169)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F61%2F1%2F90%2F1680002" target="">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href="https://academic.oup.com/sysbio/CrossRef-CitedBy/1680002">CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/sysbio/advance-article/doi/10.1093/sysbio/syy057/5107025">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy057/5107025">Detecting the Dependence of Diversification on Multiple Traits from Phylogenetic Trees and Trait Data</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy057/5107025">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/sysbio/advance-article/doi/10.1093/sysbio/syy061/5104882">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy061/5104882">Long-Branch Attraction in Species Tree Estimation: Inconsistency of Partitioned Likelihood and Topology-Based Summary Methods</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy061/5104882">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/sysbio/advance-article/doi/10.1093/sysbio/syy054/5079844">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy054/5079844">EPA-ng: Massively Parallel Evolutionary Placement of Genetic Sequences</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy054/5079844">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/sysbio/advance-article/doi/10.1093/sysbio/syy056/5100880">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy056/5100880">Book Review</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy056/5100880">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/sysbio/advance-article/doi/10.1093/sysbio/syy064/5100879">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy064/5100879">Impacts
 Of Inference Method And Dataset Filtering On Phylogenomic Resolution In
 A Rapid Radiation of Ground Squirrels (Xerinae: Marmotini)</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syy064/5100879">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad3_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Supplement_Ad3_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Supplement_Ad3_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/sysbio/syr095");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad4_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/sysbio_Supplement_Ad4_0" title="3rd party ad content" name="google_ads_iframe_/116097782/sysbio_Supplement_Ad4_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/sysbio/pages/About">About Systematic Biology</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/sysbio/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/sysbio/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.systbio.org/">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/systbiol">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/sysbio/subscribe">Purchase</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-5">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-SystematicBiology" class="journal-footer-affiliations-logo" src="sysbio_f1-178201843.svg" alt="Systematic Biology">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1076-836X</li>
                                            <li>Print ISSN 1063-5157</li>
                    <li>Copyright © 2018 Society of Systematic Biologists</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f61%2f1%2f90%2f1680002">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="footerJS" type="text/javascript"></script>
<script src="OupCookiePolicyJS" type="text/javascript"></script>
<script src="MathJax.js" type="text/javascript"></script>
<script src="chartist.js" type="text/javascript"></script>
<script src="ArticlePageJS" type="text/javascript"></script>
<script src="addthis_widget.js" type="text/javascript"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh285" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.8296555419636504&amp;iit=1539611365032&amp;tmr=load%3D1539611364900%26core%3D1539611364998%26main%3D1539611365002%26ifr%3D1539611365136&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F61%2F1%2F90%2F1680002&amp;href=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F61%2F1%2F90%2F1680002&amp;dt=SAT%C3%A9-II%3A%20Very%20Fast%20and%20Accurate%20Simultaneous%20Estimation%20of%20Multiple%20Sequence%20Alignments%20and%20Phylogenetic%20Trees%20%7C%20Systematic%20Biology%20%7C%20Oxford%20Academic&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5bc49ae4c19bedc7&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DSAT%25C3%25A9-II%253A%2520Very%2520Fast%2520and%2520Accurate%2520Simultaneous%2520Estimation%2520of%2520Multiple%2520Sequence%2520Alignments%2520and%2520Phylogenetic%2520Trees%2520%257C%2520Systematic%2520Biology%2520%257C%2520Oxford%2520Academic%26description%3DAbstract.%2520%2520Highly%2520accurate%2520estimation%2520of%2520phylogenetic%2520trees%2520for%2520large%2520data%2520sets%2520is%2520difficult%252C%2520in%2520part%2520because%2520multiple%2520sequence%2520alignments%2520must%2520be%2520accurate%2520for%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fsysbio%252Farticle%252F61%252F1%252F90%252F1680002%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fsysbio%252FIssue%252F67%252F5%252F2%252Fm_sysbio_67_5cover.png%253FExpires%253D2147483647%2526Signature%253DiMgOj%7EozkXgiMv5ypBZDzosyeLzAPALO2wmmKLje0Ck54FjJm347WeTb3PKZWVrxYH9T2vN47duSgGc4dLVp5tBbPeDOQV99%7EfC0QoiuNqiJsKVtDjUdYShshHjSJhQalWrhtfVODxOV3i%7Eokx0xHvcYKoh0XaVEgY8howRgKsjKDokUzxP8A-IlCyuRe7%7ERCRWcLvUokOnZjMW6dIfPEHfg4NKlHFUTrgzwYmZcyCm1Idbn11KibNLt9Vs51khpe8c8%7EeRqAFQvD8TZZR2syjseNNvaC62SMaVI7aravRbjdk14RleGCLdGvpguCHY1RYMY5C%7EwSqHtyxn-d0GGQA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2011-12-01%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.3.7-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
<script src="api.js" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="syr095.txt"></script></body></html>