<!DOCTYPE html>
<html class="js webfonts-loaded" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>phyloXML: XML for evolutionary biology and comparative genomics | BMC Bioinformatics | Full Text</title>

        

    <meta name="citation_abstract" content="Evolutionary trees are central to a wide range of biological studies. In many of these studies, tree nodes and branches need to be associated (or annotated) with various attributes. For example, in studies concerned with organismal relationships, tree nodes are associated with taxonomic names, whereas tree branches have lengths and oftentimes support values. Gene trees used in comparative genomics or phylogenomics are usually annotated with taxonomic information, genome-related data, such as gene names and functional annotations, as well as events such as gene duplications, speciations, or exon shufflings, combined with information related to the evolutionary tree itself. The data standards currently used for evolutionary trees have limited capacities to incorporate such annotations of different data types. We developed a XML language, named phyloXML, for describing evolutionary trees, as well as various associated data items. PhyloXML provides elements for commonly used items, such as branch lengths, support values, taxonomic names, and gene names and identifiers. By using " property"="" elements,="" phyloxml="" can="" be="" adapted="" to="" novel="" and="" unforeseen="" use="" cases.="" we="" also="" developed="" various="" software="" tools="" for="" reading,="" writing,="" conversion,="" visualization="" of="" formatted="" data.="" is="" an="" xml="" language="" defined="" by="" a="" complete="" schema="" in="" xsd="" that="" allows="" storing="" exchanging="" the="" structures="" evolutionary="" trees="" as="" well="" associated="" more="" information="" about="" itself,="" schema,="" implementing="" supporting="" phyloxml,="" available="" at="" http:="" www.phyloxml.org="" ."="">

    <meta name="description" content="Evolutionary trees are central to a wide range of biological studies. In many of these studies, tree nodes and branches need to be associated (or annotated) with various attributes. For example, in studies concerned with organismal relationships, tree nodes are associated with taxonomic names, whereas tree branches have lengths and oftentimes support values. Gene trees used in comparative genomics or phylogenomics are usually annotated with taxonomic information, genome-related data, such as gene names and functional annotations, as well as events such as gene duplications, speciations, or exon shufflings, combined with information related to the evolutionary tree itself. The data standards currently used for evolutionary trees have limited capacities to incorporate such annotations of different data types. We developed a XML language, named phyloXML, for describing evolutionary trees, as well as various associated data items. PhyloXML provides elements for commonly used items, such as branch lengths, support values, taxonomic names, and gene names and identifiers. By using " property"="" elements,="" phyloxml="" can="" be="" adapted="" to="" novel="" and="" unforeseen="" use="" cases.="" we="" also="" developed="" various="" software="" tools="" for="" reading,="" writing,="" conversion,="" visualization="" of="" formatted="" data.="" is="" an="" xml="" language="" defined="" by="" a="" complete="" schema="" in="" xsd="" that="" allows="" storing="" exchanging="" the="" structures="" evolutionary="" trees="" as="" well="" associated="" more="" information="" about="" itself,="" schema,="" implementing="" supporting="" phyloxml,="" available="" at="" http:="" www.phyloxml.org="" ."="">

    <meta name="journal_id" content="12859">

    <meta name="dc.title" content="phyloXML: XML for evolutionary biology and comparative genomics">

    <meta name="dc.source" content="BMC Bioinformatics 2009 10:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2009-10-27">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2009 Han and Zmasek; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="Evolutionary trees are central to a wide range of biological studies. In many of these studies, tree nodes and branches need to be associated (or annotated) with various attributes. For example, in studies concerned with organismal relationships, tree nodes are associated with taxonomic names, whereas tree branches have lengths and oftentimes support values. Gene trees used in comparative genomics or phylogenomics are usually annotated with taxonomic information, genome-related data, such as gene names and functional annotations, as well as events such as gene duplications, speciations, or exon shufflings, combined with information related to the evolutionary tree itself. The data standards currently used for evolutionary trees have limited capacities to incorporate such annotations of different data types. We developed a XML language, named phyloXML, for describing evolutionary trees, as well as various associated data items. PhyloXML provides elements for commonly used items, such as branch lengths, support values, taxonomic names, and gene names and identifiers. By using &quot;property&quot; elements, phyloXML can be adapted to novel and unforeseen use cases. We also developed various software tools for reading, writing, conversion, and visualization of phyloXML formatted data. PhyloXML is an XML language defined by a complete schema in XSD that allows storing and exchanging the structures of evolutionary trees as well as associated data. More information about phyloXML itself, the XSD schema, as well as tools implementing and supporting phyloXML, is available at 
                    http://www.phyloxml.org
                    
                  .">

    <meta name="prism.issn" content="1471-2105">

    <meta name="prism.publicationName" content="BMC Bioinformatics">

    <meta name="prism.publicationDate" content="2009-10-27">

    <meta name="prism.volume" content="10">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="356">

    <meta name="prism.copyright" content="2009 Han and Zmasek; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-356">

    <meta name="prism.doi" content="doi:10.1186/1471-2105-10-356">

    <meta name="citation_pdf_url" content="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-356">

    <meta name="citation_fulltext_html_url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-356">

    <meta name="citation_journal_title" content="BMC Bioinformatics">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2105">

    <meta name="citation_title" content="phyloXML: XML for evolutionary biology and comparative genomics">

    <meta name="citation_volume" content="10">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2009/12">

    <meta name="citation_online_date" content="2009/10/27">

    <meta name="citation_firstpage" content="356">

    <meta name="citation_article_type" content="Software">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/1471-2105-10-356">

    <meta name="DOI" content="10.1186/1471-2105-10-356">

    <meta name="citation_doi" content="10.1186/1471-2105-10-356">

    <meta name="dc.creator" content="Mira V Han">

    <meta name="dc.creator" content="Christian M Zmasek">

    <meta name="dc.subject" content="Bioinformatics">

    <meta name="dc.subject" content="Microarrays">

    <meta name="dc.subject" content="Computational Biology/Bioinformatics">

    <meta name="dc.subject" content="Computer Appl. in Life Sciences">

    <meta name="dc.subject" content="Combinatorial Libraries">

    <meta name="dc.subject" content="Algorithms">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Phylogenomics: intersection of evolution and genomics; citation_author=JA Eisen, CM Fraser; citation_volume=300; citation_issue=5626; citation_publication_date=2003; citation_pages=1706-1707; citation_doi=10.1126/science.1086292; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Omics; citation_title=Taking the first steps towards a standard for reporting on phylogenies: Minimum Information About a Phylogenetic Analysis (MIAPA); citation_author=J Leebens-Mack, T Vision, E Brenner, JE Bowers, S Cannon, MJ Clement, CW Cunningham, C dePamphilis, R deSalle, JJ Doyle; citation_volume=10; citation_issue=2; citation_publication_date=2006; citation_pages=231-237; citation_doi=10.1089/omi.2006.10.231; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=A simple algorithm to infer gene duplication and speciation events on a gene tree; citation_author=CM Zmasek, SR Eddy; citation_volume=17; citation_issue=9; citation_publication_date=2001; citation_pages=821-828; citation_doi=10.1093/bioinformatics/17.9.821; citation_id=CR3">

    <meta name="citation_reference" content="citation_inbook_title=Phylogeography: The History and Formation of Species; citation_publication_date=2000; citation_id=CR4; citation_author=JC Avise; citation_publisher=Harvard University Press">

    <meta name="citation_reference" content="citation_journal_title=Systematic Biology; citation_title=NEXUS: An Extensible File Format for Systematic Information; citation_author=DR Maddison, DL Swofford, WP Maddison; citation_volume=46; citation_issue=4; citation_publication_date=1997; citation_pages=590-621; citation_doi=10.1093/sysbio/46.4.590; citation_id=CR5">

    <meta name="citation_reference" content="NHX[
                    http://www.phylosoft.org/NHX
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ATV: display and manipulation of annotated phylogenetic trees; citation_author=CM Zmasek, SR Eddy; citation_volume=17; citation_issue=4; citation_publication_date=2001; citation_pages=383-384; citation_doi=10.1093/bioinformatics/17.4.383; citation_id=CR7">

    <meta name="citation_reference" content="NeXML[
                    http://www.nexml.org
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Cladistics; citation_title=PHYLIP - phylogeny inference package; citation_author=J Felsenstein; citation_volume=5; citation_publication_date=1989; citation_pages=164-166; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Taxonomic markup language: applying XML to systematic data; citation_author=R Gilmour; citation_volume=16; citation_issue=4; citation_publication_date=2000; citation_pages=406-407; citation_doi=10.1093/bioinformatics/16.4.406; citation_id=CR10">

    <meta name="citation_reference" content="citation_inbook_title=Extensible Markup Language (XML) 1.0; citation_publication_date=1998; citation_id=CR11; citation_author=T Bray; citation_author=J Paoli; citation_author=CM Sperberg-McQueen">

    <meta name="citation_reference" content="citation_inbook_title=Inferring Phylogenies; citation_publication_date=2004; citation_id=CR12; citation_author=J Felsenstein; citation_publisher=Sinauer Associates">

    <meta name="citation_reference" content="Archaeopteryx[
                    http://www.phylosoft.org/archaeopteryx
                    
                  ]">

    <meta name="citation_reference" content="citation_journal_title=Source Code for Biology and Medicine; citation_title=TreeViewJ: An application for viewing and analyzing phylogenetic trees; citation_author=MW Peterson, ME Colosimo; citation_volume=2; citation_issue=1; citation_publication_date=2007; citation_pages=7; citation_doi=10.1186/1751-0473-2-7; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=The Bioperl toolkit: Perl modules for the life sciences; citation_author=JE Stajich, D Block, K Boulez, SE Brenner, SA Chervitz, C Dagdigian, G Fuellen, JG Gilbert, I Korf, H Lapp; citation_volume=12; citation_publication_date=2002; citation_pages=1611-1618; citation_doi=10.1101/gr.361602; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=Genome Informatics; citation_title=BioRuby: Open-Source Bioinformatics Library; citation_author=N Goto, MC Nakao, S Kawashima, T Katayama, M Kanehisa; citation_volume=14; citation_publication_date=2003; citation_pages=629-630; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Biopython: freely available Python tools for computational molecular biology and bioinformatics; citation_author=PJ Cock, T Antao, JT Chang, BA Chapman, CJ Cox, A Dalke, I Friedberg, T Hamelryck, F Kauff, B Wilczynski; citation_volume=25; citation_issue=11; citation_publication_date=2009; citation_pages=1422-1423; citation_doi=10.1093/bioinformatics/btp163; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=A Semantic Web for bioinformatics: goals, tools, systems, applications; citation_author=N Cannata, M Schroder, R Marangoni, P Romano; citation_volume=9; citation_issue=Suppl 4; citation_publication_date=2008; citation_pages=S1; citation_doi=10.1186/1471-2105-9-S4-S1; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Brief Bioinform; citation_title=Biological knowledge management: the emerging role of the Semantic Web technologies; citation_author=E Antezana, M Kuiper, V Mironov; citation_volume=10; citation_issue=4; citation_publication_date=2009; citation_pages=392-407; citation_doi=10.1093/bib/bbp024; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Genome Biology; citation_title=Novel genes dramatically alter regulatory network topology in amphioxus; citation_author=Q Zhang, CM Zmasek, LJ Dishaw, MG Mueller, Y Ye, GW Litman, A Godzik; citation_volume=9; citation_issue=8; citation_publication_date=2008; citation_pages=R123; citation_doi=10.1186/gb-2008-9-8-r123; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=Genome Biology; citation_title=Surprising complexity of the ancestral apoptosis network; citation_author=CM Zmasek, Q Zhang, Y Ye, A Godzik; citation_volume=8; citation_issue=10; citation_publication_date=2007; citation_pages=R226; citation_doi=10.1186/gb-2007-8-10-r226; citation_id=CR21">

    <meta name="citation_author" content="Mira V Han">

    <meta name="citation_author_institution" content="School of Informatics, Indiana University, Bloomington, USA">

    <meta name="citation_author" content="Christian M Zmasek">

    <meta name="citation_author_institution" content="Bioinformatics &amp; Systems Biology, Burnham Institute for Medical Research, La Jolla, USA">



        <meta name="format-detection" content="telephone=no">

        <link rel="shortcut icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-9ff1bf1161.ico">
<link rel="apple-touch-icon" sizes="114x114" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/app-icon-114x114-aa3514ee38.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/app-icon-180x180-230f795aee.png">
<link rel="icon" type="image/png" sizes="16x16" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-16x16-25a0239eec.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-32x32-637712e609.png">
<link rel="icon" type="image/png" sizes="96x96" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-96x96-9f5dc535c7.png">
<link rel="icon" type="image/png" sizes="194x194" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-194x194-d2a2617de2.png">
<meta name="msapplication-TileColor" content="#e6e6e6">
<meta name="msapplication-TileImage" content="/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png">
<meta name="theme-color" content="#1b3051">

        <script async="" src="//cdn.krxd.net/ctjs/controltag.js.f0b61617fbc713883207cc130147aee2"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        

<link rel="stylesheet" media="screen" href="core-a9ea426f56.css">
<link rel="stylesheet" media="print" href="print-7680b1a80b.css">

<link rel="stylesheet" id="js-mustard" href="enhanced-3e5b67d6cd.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">



        

        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcbioinformatics.biomedcentral.com',
            siteWithPath: 'bmcbioinformatics.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcbioinformatics',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1471-2105-10-356',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            hasFigshareInvoked: false,
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>

        <!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script> 
        
    <script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/1471-2105-10-356","articleType":"Software","peerReviewType":"Closed","supplement":null,"keywords":"Evolutionary Tree;Tree Node;Complete Schema;Open Source Library;Common Data Element"},"contentInfo":{"imprint":"BioMed Central","title":"phyloXML: XML for evolutionary biology and comparative genomics","publishedAt":1256601600000,"publishedAtDate":"2009-10-27","author":["Mira V Han","Christian M Zmasek"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"bmcbioinformatics.biomedcentral.com","volume":"10","issue":"1","title":"BMC Bioinformatics","type":"SERIES","journalID":12859,"gaCode":"UA-62065396-3","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Software","publishingSegment":"BMC Series - Life and Health"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"},"attributes":{"featureFlags":[{"name":"ab_test_bmc_shared_article_renderer","active":false}]}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script id="springerlink-event-tracker" src="eventTracker.asc"></script>


        <script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script>
        <script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    d.className += ' webfonts-loaded';
                }
            })(document.documentElement);
        </script>
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle button { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle button { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style><script type="text/javascript" src="jquery-3.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
        
    
        
    

        
    
        
            <!-- Google Tag Manager -->
            <script>
                (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
            </script>
            <!-- End Google Tag Manager -->
        
    


        
    
    <link rel="canonical" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-356">
    

        
        
        <meta property="og:url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-356">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Bioinformatics">
        <meta property="og:title" content="phyloXML: XML for evolutionary biology and comparative genomics">
        <meta property="og:description" content="Evolutionary trees are central to a wide range of biological studies. In many of these studies, tree nodes and branches need to be associated (or annotated) with various attributes. For example, in studies concerned with organismal relationships, tree nodes are associated with taxonomic names, whereas tree branches have lengths and oftentimes support values. Gene trees used in comparative genomics or phylogenomics are usually annotated with taxonomic information, genome-related data, such as gene names and functional annotations, as well as events such as gene duplications, speciations, or exon shufflings, combined with information related to the evolutionary tree itself. The data standards currently used for evolutionary trees have limited capacities to incorporate such annotations of different data types. We developed a XML language, named phyloXML, for describing evolutionary trees, as well as various associated data items. PhyloXML provides elements for commonly used items, such as branch lengths, support values, taxonomic names, and gene names and identifiers. By using &quot;property&quot; elements, phyloXML can be adapted to novel and unforeseen use cases. We also developed various software tools for reading, writing, conversion, and visualization of phyloXML formatted data. PhyloXML is an XML language defined by a complete schema in XSD that allows storing and exchanging the structures of evolutionary trees as well as associated data. More information about phyloXML itself, the XSD schema, as well as tools implementing and supporting phyloXML, is available at http://www.phyloxml.org .">
        <meta property="og:image" content="https://media.springernature.com/w110/springer-static/cover/journal/12859.jpg">
    
        
    <script type="text/javascript" src="jquery.js"></script><script src="figshare.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019062401.js" async=""></script></head>

    <body class="journal journal-fulltext"><div id="MathJax_Message" style="display: none;"></div>
        
    
        
            <!-- Google Tag Manager (noscript) -->
            <noscript>
                <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                        height="0" width="0" style="display:none;visibility:hidden"></iframe>
            </noscript>
            <!-- End Google Tag Manager (noscript) -->
        
    

        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-alert"><path d="M1 14h14v-.172a1 1 0 0 0-.293-.707L13 11.414V8A5 5 0 0 0 3 8v3h2.5a.5.5 0 1 1 0 1H2.414l-1.121 1.121a1 1 0 0 0-.293.707V14zm5 1a2 2 0 1 0 4 0H6zm-5 0a1 1 0 0 1-1-1v-.172a2 2 0 0 1 .586-1.414L2 11V8a6 6 0 1 1 12 0v3l1.414 1.414A2 2 0 0 1 16 13.828V14a1 1 0 0 1-1 1h-4a3 3 0 0 1-6 0H1zM8.997 2.082a1 1 0 1 0-1.993 0c-.336.057-.662.14-.977.25a2 2 0 1 1 3.945 0 5.958 5.958 0 0 0-.975-.25z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-article"><path d="M14 14c0 .6.4 1 1 1h1c.5 0 1-.5 1-1V5h-3v9zm-1 0V1H1v13c0 .5.4 1 1 1h11.3c-.2-.3-.3-.6-.3-1zM0 14V1c0-.6.5-1 1-1h12c.6 0 1 .5 1 1v3h4v10c0 1.1-.9 2-2 2H2c-1.1 0-2-.9-2-2zM4 4v2h6V4H4zM3 3h8v4H3V3zm0 5h8v1H3V8zm0 2h8v1H3v-1zm0 2h8v1H3v-1z"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"></path></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-editor"><path d="M10.2 16.9c1.3-.2 2.5-.7 3.5-1.5-.9-2.5-2.1-3.9-3.8-4.3V9.8c1.2-.4 2-1.5 2-2.8 0-1.7-1.3-3-3-3S6 5.3 6 7c0 1.3.8 2.4 2 2.8v1.3c-1.7.4-2.9 1.8-3.8 4.3 1 .8 2.2 1.3 3.5 1.5.5.1.9.1 1.3.1s.8 0 1.2-.1zm4.4-2.2C16.1 13.3 17 11.2 17 9c0-4.4-3.6-8-8-8S1 4.6 1 9c0 2.2.9 4.3 2.4 5.7.8-2 1.8-3.6 3.5-4.3C5.8 9.7 5 8.4 5 7c0-2.2 1.8-4 4-4s4 1.8 4 4c0 1.4-.8 2.7-1.9 3.4 1.6.7 2.7 2.3 3.5 4.3zM9 18c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9z"></path></symbol><symbol id="icon-explore"><path d="M9 17c4.4 0 8-3.6 8-8s-3.6-8-8-8-8 3.6-8 8 3.6 8 8 8zm0 1c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm0-2.5c-.3 0-.5-.2-.5-.5s.2-.5.5-.5c3 0 5.4-2.4 5.5-5.3 0-.3.2-.5.5-.5s.5.2.5.5c-.1 3.5-3 6.3-6.5 6.3zm-.3-13c.3 0 .5.2.5.5s-.2.5-.4.5c-3 .1-5.3 2.6-5.3 5.5 0 .3-.2.5-.5.5s-.5-.2-.5-.5c0-3.5 2.7-6.3 6.2-6.5zm1.1 8.1L8.6 9.4c-.2-.2-.2-.5 0-.7.2-.2.5-.2.7 0l1.2 1.2 1.6-4-4.5 1.7-1.8 4.5 4-1.5zm4.1-6.5l-2.7 6.7c-.1.3-.3.5-.6.6l-6.7 2.7 2.7-6.7c.2-.4.4-.6.7-.7l6.6-2.6z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-alert"><path d="M1 14h14v-.172a1 1 0 0 0-.293-.707L13 11.414V8A5 5 0 0 0 3 8v3h2.5a.5.5 0 1 1 0 1H2.414l-1.121 1.121a1 1 0 0 0-.293.707V14zm5 1a2 2 0 1 0 4 0H6zm-5 0a1 1 0 0 1-1-1v-.172a2 2 0 0 1 .586-1.414L2 11V8a6 6 0 1 1 12 0v3l1.414 1.414A2 2 0 0 1 16 13.828V14a1 1 0 0 1-1 1h-4a3 3 0 0 1-6 0H1zM8.997 2.082a1 1 0 1 0-1.993 0c-.336.057-.662.14-.977.25a2 2 0 1 1 3.945 0 5.958 5.958 0 0 0-.975-.25z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-article"><path d="M14 14c0 .6.4 1 1 1h1c.5 0 1-.5 1-1V5h-3v9zm-1 0V1H1v13c0 .5.4 1 1 1h11.3c-.2-.3-.3-.6-.3-1zM0 14V1c0-.6.5-1 1-1h12c.6 0 1 .5 1 1v3h4v10c0 1.1-.9 2-2 2H2c-1.1 0-2-.9-2-2zM4 4v2h6V4H4zM3 3h8v4H3V3zm0 5h8v1H3V8zm0 2h8v1H3v-1zm0 2h8v1H3v-1z"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"></path></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-editor"><path d="M10.2 16.9c1.3-.2 2.5-.7 3.5-1.5-.9-2.5-2.1-3.9-3.8-4.3V9.8c1.2-.4 2-1.5 2-2.8 0-1.7-1.3-3-3-3S6 5.3 6 7c0 1.3.8 2.4 2 2.8v1.3c-1.7.4-2.9 1.8-3.8 4.3 1 .8 2.2 1.3 3.5 1.5.5.1.9.1 1.3.1s.8 0 1.2-.1zm4.4-2.2C16.1 13.3 17 11.2 17 9c0-4.4-3.6-8-8-8S1 4.6 1 9c0 2.2.9 4.3 2.4 5.7.8-2 1.8-3.6 3.5-4.3C5.8 9.7 5 8.4 5 7c0-2.2 1.8-4 4-4s4 1.8 4 4c0 1.4-.8 2.7-1.9 3.4 1.6.7 2.7 2.3 3.5 4.3zM9 18c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9z"></path></symbol><symbol id="icon-explore"><path d="M9 17c4.4 0 8-3.6 8-8s-3.6-8-8-8-8 3.6-8 8 3.6 8 8 8zm0 1c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm0-2.5c-.3 0-.5-.2-.5-.5s.2-.5.5-.5c3 0 5.4-2.4 5.5-5.3 0-.3.2-.5.5-.5s.5.2.5.5c-.1 3.5-3 6.3-6.5 6.3zm-.3-13c.3 0 .5.2.5.5s-.2.5-.4.5c-3 .1-5.3 2.6-5.3 5.5 0 .3-.2.5-.5.5s-.5-.2-.5-.5c0-3.5 2.7-6.3 6.2-6.5zm1.1 8.1L8.6 9.4c-.2-.2-.2-.5 0-.7.2-.2.5-.2.7 0l1.2 1.2 1.6-4-4.5 1.7-1.8 4.5 4-1.5zm4.1-6.5l-2.7 6.7c-.1.3-.3.5-.6.6l-6.7 2.7 2.7-6.7c.2-.4.4-.6.7-.7l6.6-2.6z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner">Skip to main content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcbioinformatics/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1471-2105-10-356;kwrd=Evolutionary Tree,Tree Node,Complete Schema,Open Source Library,Common Data Element;pmc=L15001,B12050,I23050,L17004,M14018,M14018;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/bmcbioinformatics/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2105-10-356&amp;kwrd=Evolutionary Tree,Tree Node,Complete Schema,Open Source Library,Common Data Element&amp;pmc=L15001,B12050,I23050,L17004,M14018,M14018&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/bmcbioinformatics/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2105-10-356&amp;kwrd=Evolutionary Tree,Tree Node,Complete Schema,Open Source Library,Common Data Element&amp;pmc=L15001,B12050,I23050,L17004,M14018,M14018&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            <div id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
        </div>
    </div>

        
         
    
        <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
    
    
        <header class="c-popup-search" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="c-navbar__container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img alt="BMC" itemprop="logo" role="img" src="logo-bmc-white-series-1da6525f4e.svg" width="76" height="18">
        <div class="c-logo__strapline">
            <img alt="Part of Springer Nature" role="img" src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" width="173" height="16">
        </div>
    </a>
</div>

            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav c-navbar__nav--right">
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true" focusable="false">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text">
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true" focusable="false">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>

    

        
            <header class="c-journal-header c-journal-header--bmc-bioinformatics ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcbioinformatics.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Bioinformatics</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="c-navbar__container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcbioinformatics.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-page-layout c-page-layout--fulltext" data-component="sticky-container">
                <div class="c-page-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Implementation">Implementation</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec3" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and Discussion">Results and Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec4" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec5" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Availability and requirements">Availability and requirements</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-page-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-list-reset">
            <li class="c-article-identifiers__item" data-test="article-category">Software</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>
    

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">phyloXML: XML for evolutionary biology and comparative genomics</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-list-reset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Mira&nbsp;V&nbsp;Han</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Christian&nbsp;M&nbsp;Zmasek</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup><a href="mailto:czmasek@burnham.org" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Bioinformatics</span></span><span class="ArticleCitation_Year">2009</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">10</strong>:356</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/1471-2105-10-356">https://doi.org/10.1186/1471-2105-10-356</a></p><p class="Copyright">© &nbsp;Han and Zmasek; licensee BioMed Central Ltd.&nbsp;2009</p></div></div><ul class="ArticleHistory u-list-reset"><li class="History HistoryReceived"><strong>Received: </strong><span>8&nbsp;April&nbsp;2009</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>27&nbsp;October&nbsp;2009</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">27&nbsp;October&nbsp;2009</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">Evolutionary trees are central to a wide
 range of biological studies. In many of these studies, tree nodes and 
branches need to be associated (or annotated) with various attributes. 
For example, in studies concerned with organismal relationships, tree 
nodes are associated with taxonomic names, whereas tree branches have 
lengths and oftentimes support values. Gene trees used in comparative 
genomics or phylogenomics are usually annotated with taxonomic 
information, genome-related data, such as gene names and functional 
annotations, as well as events such as gene duplications, speciations, 
or exon shufflings, combined with information related to the 
evolutionary tree itself. The data standards currently used for 
evolutionary trees have limited capacities to incorporate such 
annotations of different data types.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">We developed a XML language, named 
phyloXML, for describing evolutionary trees, as well as various 
associated data items. PhyloXML provides elements for commonly used 
items, such as branch lengths, support values, taxonomic names, and gene
 names and identifiers. By using "property" elements, phyloXML can be 
adapted to novel and unforeseen use cases. We also developed various 
software tools for reading, writing, conversion, and visualization of 
phyloXML formatted data.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p class="Para">PhyloXML is an XML language defined by a
 complete schema in XSD that allows storing and exchanging the 
structures of evolutionary trees as well as associated data. More 
information about phyloXML itself, the XSD schema, as well as tools 
implementing and supporting phyloXML, is available at <span class="ExternalRef"><a href="http://www.phyloxml.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.phyloxml.org</span></a></span>.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Evolutionary Tree</li><li class="c-keywords__item" itemprop="about">Tree Node</li><li class="c-keywords__item" itemprop="about">Complete Schema</li><li class="c-keywords__item" itemprop="about">Open Source Library</li><li class="c-keywords__item" itemprop="about">Common Data Element</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Information
 that can be interpreted in a phylogenetic context is growing rapidly in
 both types and quantities, due to the advancement of large-scale 
studies such as metagenomics and phylogenomics [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].
 Current formats for describing evolutionary trees are becoming 
increasingly inappropriate. The main limitation of present formats is 
the lack of standardized means to annotate tree nodes and branches with 
distinct attributes. In the case of species trees, these attributes are 
taxonomic names, branch lengths, and often (possibly multiple) support 
values (such as bootstrap values or posterior probabilities). Gene trees
 used in comparative genomics and phylogenomics applications 
additionally require fields for gene identifiers and potentially gene 
duplication events [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>], whereas trees used in phylogeographic [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>] applications require fields for geographic data. While some existing formats such as Nexus [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] or NHX (New Hampshire eXtended) [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]
 allow describing additional information associated with phylogenetic 
trees, these formats have been shown to be problematic in the 
extensibility or the interoperability as a standard. The complexity of 
the Nexus format has led to different parsers that only understand a 
subset of the format, and different programs that produce poorly formed 
outputs (although a XML based replacement for the Nexus format, named 
"NeXML", is being developed and is expected to alleviate problems 
stemming from the complexity of the Nexus format [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]). The NHX format, built as an <em xmlns="" class="EmphasisTypeItalic">adhoc</em> extension to the Newick (New Hampshire) standard [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]
 has limits in the types of information it can incorporate, since it has
 been developed with one primary use case in mind - representing gene 
trees with inferred gene duplication events [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. Previous proposals for a XML format for systematic data [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>] never gained popularity, possibly due to a lack of supporting software.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Here we describe phyloXML, a new standardized format for phylogenetic documents that is based on the formal language of XML [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] and which is inspired by the XML tree representation described in [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>] (this XML format is used as output format by the "Retree" program from the PHYLIP package [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]).</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Implementation" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Implementation">Implementation</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Implementation" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Along
 with the complete schema in XSD that defines the format of phyloXML, a 
number of tools have been implemented to support the reading and writing
 of phyloXML. The Java command-line tools "phyloxml_converter" can 
convert existing formats (Nexus, Newick/New Hampshire, and NHX) into 
phyloXML, and "decorator" helps the users insert various data types into
 a phyloXML tree. There are multiple tree-viewing programs that support 
the format, including Archaeopteryx [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] (the successor to the tree display tool ATV [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]) and TreeViewJ [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].
 Furthermore, Archaeopteryx allows the user to easily convert phyloXML 
to Nexus, Newick/New Hampshire, and NHX and vice versa. So far, phyloXML
 support has been developed for three open source libraries for 
computational molecular biology and bioinformatics, namely BioPerl [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>] (module Bio::TreeIO::phyloxml), BioRuby (module Bio::PhyloXML) [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], and Biopython (module Bio.Tree.PhyloXML) [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. The XSD schema and links to supporting applications, together with more complex examples of phyloXML can be found at <span class="ExternalRef"><a href="http://www.phyloxml.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.phyloxml.org</span></a></span>.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec3" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#ResultsandDiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="ResultsandDiscussion">Results and Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="ResultsandDiscussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">PhyloXML
 is general, with over 20 different elements that encompass an extensive
 range of information (such as confidence values, sequence, and 
taxonomic data) that could be added to phylogenies. PhyloXML is 
extensible, containing legitimate grammar for user-defined contents, 
while it is also easy to expand the vocabulary of the schema without 
disrupting existing usage. Because the format is defined by a XML 
schema, phyloXML is also easy to validate and process. The structure of 
the document is readily parsed by any existing XML parser, while 
interpreting the content needs to be implemented depending on the use 
case. Because of the restrictive nature of the XML schema, unambiguous 
"well-formed" and "valid" documents will facilitate greater data 
exchange among users and programs that was not feasible before.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Similar
 to NHX, and unlike Nexus, the structure of phyloXML is phylogeny 
oriented rather than character oriented. The basic structure of a 
phyloXML document is a hierarchical cluster of recursive clades. Each 
clade corresponds to a node, and the set of clades that congregate at 
the root compose a phylogeny. Each clade element can also enclose nested
 elements that are annotations to the containing clade. This kind of 
hierarchical representation of the phylogeny and its corresponding 
annotations in each level is not only intuitive, but also naturally 
suitable for a description by XML. The following is an example of a 
phyloXML document describing a simple gene tree with three external 
nodes (for more examples, [see Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>]).</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&lt;phylogeny rooted="true"&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&lt;name&gt;Alcohol dehydrogenases&lt;/name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&lt;description&gt;contains examples of commonly used elements&lt;/description&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&lt;clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;events&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;speciations&gt;1&lt;/speciations&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/events&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;taxonomy&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;id provider = "ncbi"&gt;6645&lt;/id&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;scientific_name&gt;Octopus
 vulgaris&lt;/scientific_name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/taxonomy&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;sequence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;accession
 source="UniProtKB"&gt;P81431&lt;/accession&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;name&gt;Alcohol dehydrogenase class-3&lt;/name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/sequence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;confidence type="bootstrap"&gt;100&lt;/confidence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;events&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;speciations&gt;1&lt;/speciations&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/events&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;taxonomy&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;id
 provider = "ncbi"&gt;1423&lt;/id&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;scientific_name&gt;Bacillus
 subtilis&lt;/scientific_name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/taxonomy&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;sequence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;accession
 source="UniProtKB"&gt;P71017&lt;/accession&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;name&gt;Alcohol
 dehydrogenase&lt;/name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/sequence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;taxonomy&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;id
 provider = "ncbi"&gt;562&lt;/id&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;scientific_name&gt;Escherichia
 coli&lt;/scientific_name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/taxonomy&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;sequence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;accession
 source="UniProtKB"&gt;Q46856&lt;/accession&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;name&gt;Alcohol
 dehydrogenase&lt;/name&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/sequence&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&nbsp;&nbsp;&nbsp;&lt;/clade&gt;</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">&lt;/phylogeny&gt;</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Application
 specific data types that are not covered by the schema are supported by
 phyloXML, explicitly as reserved &lt;property&gt; elements, as well as 
extensions that can be defined by the user. &lt;property&gt; provides an
 interface for custom typed and referenced data. The &lt;property&gt; 
elements can be applied to the &lt;phylogeny&gt; itself, the 
&lt;clade&gt;, or the parent branch inherent in each clade. The 
recursive structure has no bound for depth or breadth, and the same 
element can be attached to a node multiple times with different values 
as long as it complies with the schema. For example, phylogenies that 
are built as a consensus of multiple approaches can have multiple 
&lt;confidence&gt; elements attached to the same clade to describe the 
support values resulting from the different methods. Molecular sequence 
can be easily associated with a certain &lt;clade&gt; by the 
&lt;sequence&gt; element, and not only the raw sequence data but also 
complex annotations can be added to the sequence using 
&lt;annotation&gt;, &lt;sequence_relation&gt; (used to describe 
orthologous and paralogous relations, for example) and &lt;domain 
architecture&gt;. Table <span class="InternalRef"><a href="#Tab1">1</a></span> describes some of the elements that are supported to annotate evolutionary trees.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                      <p xmlns="" class="SimplePara">phyloXML elements and attributes summary</p>
                    </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Element/Attribute</p></th><th><p xmlns="" class="SimplePara">Description and Sub-elements</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">phylogeny</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Represents a phylogeny, contains clades.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">clade</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Used recursively to represent node of a phylogeny.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">taxonomy</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Represents taxonomic information.</p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">id</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">code</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">scientific_name</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">common_name</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">synonym</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">authority</strong>
                          </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">sequence</strong>
                          </p></td><td><p xmlns="" class="SimplePara">A gene or protein associated with a clade.</p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">symbol</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">accession</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">name</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">location</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">mol_seq</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">uri</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">annotation</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">domain_architecture</strong>
                          </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">events</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Events at a clade.</p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">type</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">duplications</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">speciations</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">losses</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">confidence</strong>
                          </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">annotation</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Annotation of sequence.</p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">desc</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">confidence</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">property</strong>
                          </p></td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">uri</strong>
                          </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">property</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Typed and referenced mixed (free text) content.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">uri</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Uniform resource identifier (e.g. a URL).</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">confidence</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Statistical confidence.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">distribution</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Geographic distribution of the items of a clade.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">date</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Date associated with a clade.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">sequence_relation</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Typed relationship between two sequences.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">clade_relation</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Typed relationship between two clades.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">id_ref</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Attribute, used together with id_source to describe relations between various elements.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                            <strong class="EmphasisTypeBold">id_source</strong>
                          </p></td><td><p xmlns="" class="SimplePara">Attribute, used together with id_ref to describe relations between various elements.</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                    <p xmlns="" class="SimplePara">Representative elements and attributes of phyloXML (version 1.10) are shown. XML element/attribute names are in bold letters.</p>
                  </div></figure></div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">While
 the most straightforward structure of the document is the hierarchy of 
nested clades that automatically describes the topology of the 
phylogeny, it is also possible to describe the topology in a flat manner
 using the attributes id_ref and id_source. Id_source is an optional 
attribute that assigns a unique id to a &lt;clade&gt;, &lt;taxonomy&gt; 
or a &lt;sequence&gt;. Elements with an attribute of id_ref will be 
associated with the specific element that has the same value of 
id_source identifier. Using id_ref and id_source provides much 
flexibility in the structure of the document, and allows the 
representation of network topologies that cannot be represented with a 
hierarchical structure.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 the following, we compare and contrast key features of the phyloXML 
standard with those of the NeXML format currently being developed [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>].
 One significant difference between phyloXML and NeXML is that phyloXML 
provides predefined elements for data elements commonly used in 
phylogenetics, phylogenomics, and comparative genomics (such as elements
 for taxonomic and sequence information). In contrast, NeXML (in its 
most current version as of this writing) approaches this by providing 
meta elements which are intended to be compliant with RDFa 
recommendations so that they can be expanded to RDF triples by an XSL 
stylesheet [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].
 This mechanism essentially allows expandable key/value attachments for 
various elements of an evolutionary tree, which are mediated by 
ontologies and which can be expanded to RDF. The obvious advantage of 
such an approach lies in its flexibility and in the fact that it allows 
representing unforeseen types of data and lends itself well to knowledge
 integration. On the other hand, different producers of NeXML formatted 
data might represent common data elements differently, in particular if 
they rely on different ontologies or if no commonly used ontology has 
yet been established for the problem domain, thus hampering the stated 
goal of interoperability for documents containing phylogenetic trees 
annotated with more than just basic OTUs (NeXML)/clade names (phyloXML).
 Due to the fact that phyloXML provides predefined elements, 
interoperability for documents containing commonly used types of data is
 guaranteed. Another advantage of explicitly modelling common elements 
in the XSD schema versus relying on key/value attachments mediated by 
ontologies is that this approach does not introduce dependencies on 
additional resources. To achieve the flexibility to represent data not 
modelled in the current XSD schema, phyloXML employs a two pronged 
strategy. Firstly, &lt;property&gt; elements can be used to store data 
not covered by the current schema. In fact, the "ref" attribute of 
&lt;property&gt; elements allows for mediation by ontologies, very 
similar to the approach used in NeXML. Secondly, XML is inherently 
extensible thus allowing the incorporation of data from other XML 
languages as well as extension of the phyloXML standard itself (in the 
form of future versions). Another difference between phyloXML and NeXML 
is in the structure of data representation. In NeXML, the data 
associated with the nodes are separated out of the tree into a tabulated
 structure; while in PhyloXML all the data associated with the nodes are
 within the tree structure itself. For this reason, the NeXML lends 
itself well to statistical approaches where the user treats the data as a
 list of samples. PhyloXML is convenient to use in algorithmic 
procedures where the data associated with the nodes are updated through 
multiple iterations of tree traversing, e.g. Expectation-Maximization 
across the tree. This difference is more of interest to the software 
developer who needs to think of how the data should be structured. Of 
course, both formats can ultimately be parsed and stored into any type 
of data structure, but one may be more suited for and easier to handle 
in certain approaches than the other. A third key difference between the
 two formats is that NeXML, like NEXUS, attempts to model all elements 
associated with phylogenetic inference, such as characters (molecular 
sequences, categorical data or continuous data), substitution models, 
and evolutionary trees. PhyloXML, in contrast, focuses on evolutionary 
trees with associated data only, thus simplifying the creation of 
compliant parsers and corresponding data structures.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Since
 phyloXML was not devised as input format for phylogeny inference 
software, we decided not to provide the means to store multiple sequence
 alignments as separate elements. In its current version (1.10, as of 
this writing), phyloXML only allows storing aligned molecular sequences 
via the &lt;molecular_sequence&gt; sub-element of &lt;sequence&gt; (with
 the "is_aligned" attribute set to "true"). Due to the extensible nature
 of XML, it is straightforward to add additional elements and 
sub-elements in future versions of phyloXML depending on user needs, 
without running into compatibility issues with existing software 
implementations.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 complete schema in XSD defining the phyloXML format, as well as 
software to visualize phyloXML formatted data is available at <span class="ExternalRef"><a href="http://www.phyloxml.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.phyloxml.org</span></a></span> under the open source LGPL license.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec4" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 developed phyloXML, an XML language designed to describe phylogenetic 
trees and associated data. PhyloXML provides elements for commonly used 
features, such as taxonomic information, gene names and identifiers, 
branch lengths, support values, and gene duplication and speciation 
events. Using these standardized elements allows interoperability 
between various applications and databases. Furthermore, both due to 
extensible nature of XML itself and the provision of &lt;property&gt; 
elements by phyloXML, extensibility as well as domain specific 
applications are ensured.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 also developed a number of software applications to read, write, 
convert to and from, and visualize phyloXML formatted data. Furthermore,
 phyloXML is supported by the BioPerl [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], BioRuby [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>], and Biopython [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>] open source libraries. In practice, phyloXML has already proven valuable in research on regulatory network evolution [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>, <span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>].
 In these studies evolutionary tree nodes were associated with such 
distinct data fields as taxonomic information, protein names, protein 
functions, domain-architectures, and gene duplications. PhyloXML 
provided a convenient and transparent means to store, visualize, and 
analyze these data in a phylogenetic context, leading to new biological 
insights.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec5" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Availabilityandrequirements" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Availabilityandrequirements">Availability and requirements</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Availabilityandrequirements" data-test="collapse-section"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">
                <div class="UnorderedList"><ul class="UnorderedListMarkBullet"><li>
                    <p class="Para"><strong xmlns="" class="EmphasisTypeBold">Project name:</strong> phyloXML</p>
                  </li><li>
                    <p class="Para">
                      <strong xmlns="" class="EmphasisTypeBold">Project home page:</strong>
                      <span class="ExternalRef"><a href="http://www.phyloxml.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.phyloxml.org</span></a></span>
                    </p>
                  </li><li>
                    <p class="Para"><strong xmlns="" class="EmphasisTypeBold">Operating system(s):</strong> Platform independent</p>
                  </li><li>
                    <p class="Para"><strong xmlns="" class="EmphasisTypeBold">Programming language:</strong> XML, Java, Perl (BioPerl), Python (Biopython), Ruby (BioRuby)</p>
                  </li><li>
                    <p class="Para"><strong xmlns="" class="EmphasisTypeBold">License:</strong> GNU LGPL</p>
                  </li><li>
                    <p class="Para"><strong xmlns="" class="EmphasisTypeBold">Any restrictions to use by non-academics:</strong> none</p>
                  </li></ul></div>
              </div></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Listofabbreviations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Listofabbreviations">List of abbreviations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Listofabbreviations"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">NHX</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">New Hampshire eXtended</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">OTU</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Operational Taxonomic Unit</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">RDF</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Resource Description Framework</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">RDFa</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Resource Description Framework in attributes</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">XML</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Extensible Markup Language</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">XSD</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">XML Schema Definition</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>
                    <strong class="EmphasisTypeBold">XSL</strong>
                  :&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Extensible Stylesheet Language.</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">The
 authors would like to thank Ethalinda Cannon, Chris Fields, Adam 
Godzik, Hilmar Lapp, Chris Mungall, and Rutger Vos for their suggestions
 and discussions; and Diana Jaunzeikare and Eric Talevich for 
implementing phyloXML support in BioRuby and Biopython, respectively 
[supported by Google Summer of Code 2009 in collaboration with the 
National Evolutionary Synthesis Center (NESCent), NSF #EF-0423641].</p><p xmlns="" class="SimplePara">Funding: CMZ was partly supported by P20 GM076221 (JCMM) from the National Institute of General Medical Sciences <span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.nigms.nih.gov/" target="_blank" class="ext-link"><span class="RefSource">http://www.nigms.nih.gov</span></a></span>.
 MVH was supported by Google Summer of Code 2008 in collaboration with 
the National Evolutionary Synthesis Center (NESCent), NSF #EF-0423641.</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Electronic supplementary material</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-10-356/MediaObjects/12859_2009_3086_MOESM1_ESM.XML">12859_2009_3086_MOESM1_ESM.XML</a>
                          <span xmlns="" class="SimplePara">Additional file 1: <strong class="EmphasisTypeBold">phyloXML examples</strong>.
 A file containing some simple evolutionary trees demonstrating select 
phyloXML features (the phylogenies can be displayed with Archaeopteryx, 
available at <span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.phylosoft.org/archaeopteryx" target="_blank" class="ext-link"><span class="RefSource">http://www.phylosoft.org/archaeopteryx</span></a></span>; the XML itself can be viewed with any text or XML editor). (XML 15 KB)</span>
                        </div></div>
                  </div>
                </div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors' contributions</h3><p xmlns="" class="SimplePara">MVH
 developed the phyloXML support in BioPerl and drafted the manuscript; 
CMZ developed the phyloXML format and its implementation in Java and 
helped to draft the manuscript. Both authors read and approved the final
 manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">School of Informatics, Indiana University, Bloomington, IN&nbsp;47408, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Bioinformatics &amp; Systems Biology, Burnham Institute for Medical Research, La Jolla, CA&nbsp;92037, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Eisen JA, Fraser CM: Phylogenomics: intersection of evolution and genomics. <em class="EmphasisTypeItalic">Science</em> 2003, 300(5626):1706–1707. 10.1126/science.1086292</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1086292"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12805538"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenomics%3A%20intersection%20of%20evolution%20and%20genomics&amp;author=JA.%20Eisen&amp;author=CM.%20Fraser&amp;journal=Science&amp;volume=300&amp;issue=5626&amp;pages=1706-1707&amp;publication_year=2003&amp;doi=10.1126%2Fscience.1086292"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Leebens-Mack J, Vision T, Brenner E, Bowers JE, Cannon S, Clement MJ, Cunningham CW, dePamphilis C, deSalle R, Doyle JJ, <em class="EmphasisTypeItalic">et al</em>.:
 Taking the first steps towards a standard for reporting on phylogenies:
 Minimum Information About a Phylogenetic Analysis (MIAPA). <em class="EmphasisTypeItalic">Omics</em> 2006, 10(2):231–237. 10.1089/omi.2006.10.231</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3167193"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/omi.2006.10.231"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16901231"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taking%20the%20first%20steps%20towards%20a%20standard%20for%20reporting%20on%20phylogenies%3A%20Minimum%20Information%20About%20a%20Phylogenetic%20Analysis%20%28MIAPA%29&amp;author=J.%20Leebens-Mack&amp;author=T.%20Vision&amp;author=E.%20Brenner&amp;author=JE.%20Bowers&amp;author=S.%20Cannon&amp;author=MJ.%20Clement&amp;author=CW.%20Cunningham&amp;author=C.%20dePamphilis&amp;author=R.%20deSalle&amp;author=JJ.%20Doyle&amp;journal=Omics&amp;volume=10&amp;issue=2&amp;pages=231-237&amp;publication_year=2006&amp;doi=10.1089%2Fomi.2006.10.231"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zmasek CM, Eddy SR: A simple algorithm to infer gene duplication and speciation events on a gene tree. <em class="EmphasisTypeItalic">Bioinformatics</em> 2001, 17(9):821–828. 10.1093/bioinformatics/17.9.821</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/17.9.821"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11590098"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20simple%20algorithm%20to%20infer%20gene%20duplication%20and%20speciation%20events%20on%20a%20gene%20tree&amp;author=CM.%20Zmasek&amp;author=SR.%20Eddy&amp;journal=Bioinformatics&amp;volume=17&amp;issue=9&amp;pages=821-828&amp;publication_year=2001&amp;doi=10.1093%2Fbioinformatics%2F17.9.821"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Avise JC: <em class="EmphasisTypeItalic">Phylogeography: The History and Formation of Species.</em> Cambridge, Massachusetts: Harvard University Press; 2000.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=JC.%20Avise&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maddison DR, Swofford DL, Maddison WP: NEXUS: An Extensible File Format for Systematic Information. <em class="EmphasisTypeItalic">Systematic Biology</em> 1997, 46(4):590–621.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/46.4.590"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11975335"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NEXUS%3A%20An%20Extensible%20File%20Format%20for%20Systematic%20Information&amp;author=DR.%20Maddison&amp;author=DL.%20Swofford&amp;author=WP.%20Maddison&amp;journal=Systematic%20Biology&amp;volume=46&amp;issue=4&amp;pages=590-621&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">NHX[<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.phylosoft.org/NHX" target="_blank" class="ext-link"><span class="RefSource">http://www.phylosoft.org/NHX</span></a></span>]</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zmasek CM, Eddy SR: ATV: display and manipulation of annotated phylogenetic trees. <em class="EmphasisTypeItalic">Bioinformatics</em> 2001, 17(4):383–384. 10.1093/bioinformatics/17.4.383</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/17.4.383"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11301314"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ATV%3A%20display%20and%20manipulation%20of%20annotated%20phylogenetic%20trees&amp;author=CM.%20Zmasek&amp;author=SR.%20Eddy&amp;journal=Bioinformatics&amp;volume=17&amp;issue=4&amp;pages=383-384&amp;publication_year=2001&amp;doi=10.1093%2Fbioinformatics%2F17.4.383"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">NeXML[<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.nexml.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.nexml.org</span></a></span>]</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein J: PHYLIP - phylogeny inference package. <em class="EmphasisTypeItalic">Cladistics</em> 1989, 5: 164–166.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PHYLIP%20-%20phylogeny%20inference%20package&amp;author=J.%20Felsenstein&amp;journal=Cladistics&amp;volume=5&amp;pages=164-166&amp;publication_year=1989"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gilmour R: Taxonomic markup language: applying XML to systematic data. <em class="EmphasisTypeItalic">Bioinformatics</em> 2000, 16(4):406–407. 10.1093/bioinformatics/16.4.406</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/16.4.406"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10869042"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taxonomic%20markup%20language%3A%20applying%20XML%20to%20systematic%20data&amp;author=R.%20Gilmour&amp;journal=Bioinformatics&amp;volume=16&amp;issue=4&amp;pages=406-407&amp;publication_year=2000&amp;doi=10.1093%2Fbioinformatics%2F16.4.406"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bray T, Paoli J, Sperberg-McQueen CM: Extensible Markup Language (XML) 1.0. 1998.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=T.%20Bray&amp;author=J.%20Paoli&amp;author=CM.%20Sperberg-McQueen&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein J: <em class="EmphasisTypeItalic">Inferring Phylogenies.</em> Sunderland: Sinauer Associates; 2004.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=J.%20Felsenstein&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Archaeopteryx[<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.phylosoft.org/archaeopteryx" target="_blank" class="ext-link"><span class="RefSource">http://www.phylosoft.org/archaeopteryx</span></a></span>]</span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Peterson MW, Colosimo ME: TreeViewJ: An application for viewing and analyzing phylogenetic trees. <em class="EmphasisTypeItalic">Source Code for Biology and Medicine</em> 2007, 2(1):7. 10.1186/1751-0473-2-7</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2170439"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1751-0473-2-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17974028"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=TreeViewJ%3A%20An%20application%20for%20viewing%20and%20analyzing%20phylogenetic%20trees&amp;author=MW.%20Peterson&amp;author=ME.%20Colosimo&amp;journal=Source%20Code%20for%20Biology%20and%20Medicine&amp;volume=2&amp;issue=1&amp;pages=7&amp;publication_year=2007&amp;doi=10.1186%2F1751-0473-2-7"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stajich
 JE, Block D, Boulez K, Brenner SE, Chervitz SA, Dagdigian C, Fuellen G,
 Gilbert JG, Korf I, Lapp H: The Bioperl toolkit: Perl modules for the 
life sciences. <em class="EmphasisTypeItalic">Genome Res</em> 2002, 12: 1611–1618. 10.1101/gr.361602</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC187536"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.361602"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12368254"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Bioperl%20toolkit%3A%20Perl%20modules%20for%20the%20life%20sciences&amp;author=JE.%20Stajich&amp;author=D.%20Block&amp;author=K.%20Boulez&amp;author=SE.%20Brenner&amp;author=SA.%20Chervitz&amp;author=C.%20Dagdigian&amp;author=G.%20Fuellen&amp;author=JG.%20Gilbert&amp;author=I.%20Korf&amp;author=H.%20Lapp&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1611-1618&amp;publication_year=2002&amp;doi=10.1101%2Fgr.361602"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goto N, Nakao MC, Kawashima S, Katayama T, Kanehisa M: BioRuby: Open-Source Bioinformatics Library. <em class="EmphasisTypeItalic">Genome Informatics</em> 2003, 14: 629–630.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BioRuby%3A%20Open-Source%20Bioinformatics%20Library&amp;author=N.%20Goto&amp;author=MC.%20Nakao&amp;author=S.%20Kawashima&amp;author=T.%20Katayama&amp;author=M.%20Kanehisa&amp;journal=Genome%20Informatics&amp;volume=14&amp;pages=629-630&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cock PJ, Antao T, Chang JT, Chapman BA, Cox CJ, Dalke A, Friedberg I, Hamelryck T, Kauff F, Wilczynski B, <em class="EmphasisTypeItalic">et al</em>.: Biopython: freely available Python tools for computational molecular biology and bioinformatics. <em class="EmphasisTypeItalic">Bioinformatics</em> 2009, 25(11):1422–1423. 10.1093/bioinformatics/btp163</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2682512"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp163"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19304878"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biopython%3A%20freely%20available%20Python%20tools%20for%20computational%20molecular%20biology%20and%20bioinformatics&amp;author=PJ.%20Cock&amp;author=T.%20Antao&amp;author=JT.%20Chang&amp;author=BA.%20Chapman&amp;author=CJ.%20Cox&amp;author=A.%20Dalke&amp;author=I.%20Friedberg&amp;author=T.%20Hamelryck&amp;author=F.%20Kauff&amp;author=B.%20Wilczynski&amp;journal=Bioinformatics&amp;volume=25&amp;issue=11&amp;pages=1422-1423&amp;publication_year=2009&amp;doi=10.1093%2Fbioinformatics%2Fbtp163"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cannata N, Schroder M, Marangoni R, Romano P: A Semantic Web for bioinformatics: goals, tools, systems, applications. <em class="EmphasisTypeItalic">BMC Bioinformatics</em> 2008, 9(Suppl 4):S1. 10.1186/1471-2105-9-S4-S1</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2367628"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-9-S4-S1"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18460170"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20Semantic%20Web%20for%20bioinformatics%3A%20goals%2C%20tools%2C%20systems%2C%20applications&amp;author=N.%20Cannata&amp;author=M.%20Schroder&amp;author=R.%20Marangoni&amp;author=P.%20Romano&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;issue=Suppl%204&amp;pages=S1&amp;publication_year=2008&amp;doi=10.1186%2F1471-2105-9-S4-S1"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Antezana E, Kuiper M, Mironov V: Biological knowledge management: the emerging role of the Semantic Web technologies. <em class="EmphasisTypeItalic">Brief Bioinform</em> 2009, 10(4):392–407. 10.1093/bib/bbp024</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/bbp024"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19457869"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biological%20knowledge%20management%3A%20the%20emerging%20role%20of%20the%20Semantic%20Web%20technologies&amp;author=E.%20Antezana&amp;author=M.%20Kuiper&amp;author=V.%20Mironov&amp;journal=Brief%20Bioinform&amp;volume=10&amp;issue=4&amp;pages=392-407&amp;publication_year=2009&amp;doi=10.1093%2Fbib%2Fbbp024"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zhang
 Q, Zmasek CM, Dishaw LJ, Mueller MG, Ye Y, Litman GW, Godzik A: Novel 
genes dramatically alter regulatory network topology in amphioxus. <em class="EmphasisTypeItalic">Genome Biology</em> 2008, 9(8):R123. 10.1186/gb-2008-9-8-r123</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2575513"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2008-9-8-r123"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18680598"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Novel%20genes%20dramatically%20alter%20regulatory%20network%20topology%20in%20amphioxus&amp;author=Q.%20Zhang&amp;author=CM.%20Zmasek&amp;author=LJ.%20Dishaw&amp;author=MG.%20Mueller&amp;author=Y.%20Ye&amp;author=GW.%20Litman&amp;author=A.%20Godzik&amp;journal=Genome%20Biology&amp;volume=9&amp;issue=8&amp;pages=R123&amp;publication_year=2008&amp;doi=10.1186%2Fgb-2008-9-8-r123"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zmasek CM, Zhang Q, Ye Y, Godzik A: Surprising complexity of the ancestral apoptosis network. <em class="EmphasisTypeItalic">Genome Biology</em> 2007, 8(10):R226. 10.1186/gb-2007-8-10-r226</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2246300"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2007-8-10-r226"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17958905"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Surprising%20complexity%20of%20the%20ancestral%20apoptosis%20network&amp;author=CM.%20Zmasek&amp;author=Q.%20Zhang&amp;author=Y.%20Ye&amp;author=A.%20Godzik&amp;journal=Genome%20Biology&amp;volume=8&amp;issue=10&amp;pages=R226&amp;publication_year=2007&amp;doi=10.1186%2Fgb-2007-8-10-r226"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#HanandZmaseklicenseeBioMedCentralLtd" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="HanandZmaseklicenseeBioMedCentralLtd">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="HanandZmaseklicenseeBioMedCentralLtd"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Han and Zmasek; licensee BioMed Central Ltd.&nbsp;2009<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published
 under license to BioMed Central Ltd. This is an Open Access article 
distributed under the terms of the Creative Commons Attribution License (<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-page-layout__side u-text-sm u-margin-top-md-at-lt-md">
                    <aside>
                        <div class="c-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-10-356" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/1471-2105-10-356">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/1471-2105-10-356" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-10-356?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="c-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    16190
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 283 <a href="http://citations.springer.com/item?doi=10.1186/1471-2105-10-356" rel="noopener" target="_blank">more information</a>
            </li>
        
                                
    <li class="c-list-group__item">
        <p>Altmetric Attention Score: 3</p>
        <div class="altmetric">
            <a class="altmetric__link" href="http://www.altmetric.com/details.php?citation_id=3578530&amp;domain=www.biomedcentral.com" target="_blank" rel="noopener" data-track="click" data-track-category="Article Page" data-track-action="Click Altmetric" data-track-label="10.1186/1471-2105-10-356">
                <img class="altmetric__img" src="a.png" alt="Altmetric Attention Score">
            </a>
        </div>
    </li>

                            </ul>
                        </div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
                <li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-10-356">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-10-356">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-10-356">
                            <use xlink:href="#icon-twitter"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-10-356">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-10-356">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-10-356">
                            <use xlink:href="#icon-facebook"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-10-356">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-10-356">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-10-356">
                            <use xlink:href="#icon-linkedin"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-10-356">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-10-356">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-10-356">
                            <use xlink:href="#icon-weibo"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-10-356">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-10-356">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-10-356">
                            <use xlink:href="#icon-reddit"></use>
                        </svg>
                    </button>
                </li>
            </ul>
        </div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-10-356/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/1471-2105-10-356">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
 
                        <div class="c-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/1471-2105-10-356" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2F1471-2105-10-356&amp;volume=10&amp;issue=1&amp;title=phyloXML%3A+XML+for+evolutionary+biology+and+comparative+genomics&amp;author_name=Mira+V+Han&amp;start_page=1&amp;end_page=6">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Bioinformatics</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1471-2105</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:bmcbioinformatics@biomedcentral.com">bmcbioinformatics@biomedcentral.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1471-2105-10-356.png" alt="">

         
    <footer>
        
            <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" data-test="publisher-footer-menu">
            <div class="c-flex">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://support.biomedcentral.com/support/home">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-list-reset" data-test="footer-social-links">
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="https://www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        
        <div class="c-sn-footer">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature" itemprop="logo" role="img">
        <p data-test="copyright"> © 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="https://www.springernature.com/" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

        
    </footer>

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            
            appScript.src = '/static/js/app-bundle-1dabaf4e47.js';
            
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script><script src="app-bundle-1dabaf4e47.js"></script>






    


<script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(27);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var f=[],g="",h=[],k=a.split(";");for(a=0;a<k.length;++a){void 0!==l&&(g=l);var d=k[a].split("\x3d");var l=d[0];l!==g&&(0<g.length&&0<f.length&&h.push([g,f]),f=[]);if(2===d.length&&""!==d[0]&&""!==d[1]){var m=d[1].split(",");for(d=0;d<m.length;++d)f.push(m[d])}}0<l.length&&0<f.length&&h.push([l,f]);return h}
function splitSizes(a){var f=[];if(null!==a){var g=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<g.length;++a){var h=g[a].split("x");var k=parseInt(h[0],10);var d=parseInt(h[1],10);2===h.length&&!isNaN(k)&&!isNaN(d)&&0<=k&&0<=d&&f.push([k,d])}}return f}function debounce(a,f){var g=null,h=null;return function(){var k=this,d=Number(new Date),l=arguments;g&&d<g+f?(clearTimeout(h),h=setTimeout(function(){g=d;a.apply(k,l)},f)):(g=d,a.apply(k,l))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,f){function g(e){var b="test";b=b.replace(/[\[\]]/g,"\\$\x26");b=new RegExp("[?\x26]"+b+"(\x3d([^\x26#]*)|\x26|#|$)");b=(b=b.exec(window.location.href))?b[2]?decodeURIComponent(b[2].replace(/\+/g," ")):"":null;b=b&&"ads"===b?"adtype\x3dtest;":"";e=e.getAttribute("data-gpt-targeting");var a=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var c=isLoggedIn();b&&-1===e.indexOf(b)&&(e+=b);a&&
(e+=a);return e+c+q}function h(e,b){for(var a=e.length,c=[];a--;)b(e[a],a)&&(c.push(e[a].slot),e.splice(a,1));c.length&&googletag.pubads().refresh(c);return e}function k(e){var b=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return h(e,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return b>a})}function d(){googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=k(c)});var a=debounce(function(){googletag.cmd.push(function(){c=
k(c);c.length||removeScrollEvent(a)})},250);addScrollEvent(a)}function l(a){var e=document.createElement("script");e.onload=function(){a(window.PubGrade)};e.async=!0;e.src="https://cdn.pbgrd.com/core-bmc.js";document.head.appendChild(e)}function m(a){!a||a&&a.fired?setTimeout(d,1):document.addEventListener("pbgrdFinished",d)}var q,r=getAdContainers(),c=[];f&&(q=f.dfppKeyValues?f.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var p=0;r[p];++p){var n=r[p];splitKeys(g(n));
c.push({divId:n.getAttribute("id"),adUnitPath:n.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(n.getAttribute("data-gpt-sizes")),targeting:splitKeys(g(n))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(31)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());for(var b=
0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(t){console.log("failed to create slot for",c[a])}});-1!==window.location.href.indexOf("shared-article-renderer")?l(m):m(null)})(window,window.Krux);</script><script type="text/javascript" id="ga-data-track-event-listeners">var setupHandler=google_tag_manager["GTM-TDGJHK"].macro(50);setupHandler("click");setupHandler("submit");</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2292/generated/client.html?source=https://bmcbioinformatics.biomedcentral.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe></body></html>