<!DOCTYPE html>
<html class="js" style="--font-family-sans: Europa; --font-family-serif: Noto Serif;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <title>Bio.Phylo: A unified toolkit for processing, analyzing 
and visualizing phylogenetic trees in Biopython | BMC Bioinformatics | 
Full Text</title>

        

    <meta name="citation_abstract" content="Ongoing innovation in phylogenetics and evolutionary biology has been accompanied by a proliferation of software tools, data formats, analytical techniques and web servers. This brings with it the challenge of integrating phylogenetic and other related biological data found in a wide variety of formats, and underlines the need for reusable software that can read, manipulate and transform this information into the various forms required to build computational pipelines. We built a Python software library for working with phylogenetic data that is tightly integrated with Biopython, a broad-ranging toolkit for computational biology. Our library, Bio.Phylo, is highly interoperable with existing libraries, tools and standards, and is capable of parsing common file formats for phylogenetic trees, performing basic transformations and manipulations, attaching rich annotations, and visualizing trees. We unified the modules for working with the standard file formats Newick, NEXUS and phyloXML behind a consistent and simple API, providing a common set of functionality independent of the data source. Bio.Phylo meets a growing need in bioinformatics for working with heterogeneous types of phylogenetic data. By supporting interoperability with multiple file formats and leveraging existing Biopython features, this library simplifies the construction of phylogenetic workflows. We also provide examples of the benefits of building a community around a shared open-source project. Bio.Phylo is included with Biopython, available through the Biopython website, 
                    http://biopython.org
                    
                  .">

    <meta name="description" content="Ongoing innovation in phylogenetics and evolutionary biology has been accompanied by a proliferation of software tools, data formats, analytical techniques and web servers. This brings with it the challenge of integrating phylogenetic and other related biological data found in a wide variety of formats, and underlines the need for reusable software that can read, manipulate and transform this information into the various forms required to build computational pipelines. We built a Python software library for working with phylogenetic data that is tightly integrated with Biopython, a broad-ranging toolkit for computational biology. Our library, Bio.Phylo, is highly interoperable with existing libraries, tools and standards, and is capable of parsing common file formats for phylogenetic trees, performing basic transformations and manipulations, attaching rich annotations, and visualizing trees. We unified the modules for working with the standard file formats Newick, NEXUS and phyloXML behind a consistent and simple API, providing a common set of functionality independent of the data source. Bio.Phylo meets a growing need in bioinformatics for working with heterogeneous types of phylogenetic data. By supporting interoperability with multiple file formats and leveraging existing Biopython features, this library simplifies the construction of phylogenetic workflows. We also provide examples of the benefits of building a community around a shared open-source project. Bio.Phylo is included with Biopython, available through the Biopython website, 
                    http://biopython.org
                    
                  .">

    <meta name="journal_id" content="12859">

    <meta name="dc.title" content="Bio.Phylo: A unified toolkit for processing, analyzing and visualizing phylogenetic trees in Biopython">

    <meta name="dc.source" content="BMC Bioinformatics 2012 13:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2012-08-21">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2012 Talevich et al.; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="Ongoing innovation in phylogenetics and evolutionary biology has been accompanied by a proliferation of software tools, data formats, analytical techniques and web servers. This brings with it the challenge of integrating phylogenetic and other related biological data found in a wide variety of formats, and underlines the need for reusable software that can read, manipulate and transform this information into the various forms required to build computational pipelines. We built a Python software library for working with phylogenetic data that is tightly integrated with Biopython, a broad-ranging toolkit for computational biology. Our library, Bio.Phylo, is highly interoperable with existing libraries, tools and standards, and is capable of parsing common file formats for phylogenetic trees, performing basic transformations and manipulations, attaching rich annotations, and visualizing trees. We unified the modules for working with the standard file formats Newick, NEXUS and phyloXML behind a consistent and simple API, providing a common set of functionality independent of the data source. Bio.Phylo meets a growing need in bioinformatics for working with heterogeneous types of phylogenetic data. By supporting interoperability with multiple file formats and leveraging existing Biopython features, this library simplifies the construction of phylogenetic workflows. We also provide examples of the benefits of building a community around a shared open-source project. Bio.Phylo is included with Biopython, available through the Biopython website, 
                    http://biopython.org
                    
                  .">

    <meta name="prism.issn" content="1471-2105">

    <meta name="prism.publicationName" content="BMC Bioinformatics">

    <meta name="prism.publicationDate" content="2012-08-21">

    <meta name="prism.volume" content="13">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="209">

    <meta name="prism.copyright" content="2012 Talevich et al.; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-209">

    <meta name="prism.doi" content="doi:10.1186/1471-2105-13-209">

    <meta name="citation_pdf_url" content="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-209">

    <meta name="citation_fulltext_html_url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-209">

    <meta name="citation_journal_title" content="BMC Bioinformatics">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2105">

    <meta name="citation_title" content="Bio.Phylo: A unified toolkit for processing, analyzing and visualizing phylogenetic trees in Biopython">

    <meta name="citation_volume" content="13">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2012/12">

    <meta name="citation_online_date" content="2012/08/21">

    <meta name="citation_firstpage" content="209">

    <meta name="citation_article_type" content="Software">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/1471-2105-13-209">

    <meta name="DOI" content="10.1186/1471-2105-13-209">

    <meta name="citation_doi" content="10.1186/1471-2105-13-209">

    <meta name="dc.creator" content="Eric Talevich">

    <meta name="dc.creator" content="Brandon M Invergo">

    <meta name="dc.creator" content="Peter JA Cock">

    <meta name="dc.creator" content="Brad A Chapman">

    <meta name="dc.subject" content="Bioinformatics">

    <meta name="dc.subject" content="Microarrays">

    <meta name="dc.subject" content="Computational Biology/Bioinformatics">

    <meta name="dc.subject" content="Computer Appl. in Life Sciences">

    <meta name="dc.subject" content="Combinatorial Libraries">

    <meta name="dc.subject" content="Algorithms">

    <meta name="citation_reference" content="citation_journal_title=American Naturalist; citation_title=Phylogenies and the comparative method; citation_author=J Felsenstein; citation_volume=125; citation_publication_date=1985; citation_pages=1-15; citation_doi=10.1086/284325; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=Bio:Phylo - phyloinformatic analysis using Perl; citation_author=RA Vos, J Caravas, K Hartmann, MA Jensen, C Miller; citation_volume=12; citation_publication_date=2011; citation_pages=63; citation_doi=10.1186/1471-2105-12-63; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Omics: J Integr Biol; citation_title=Taking the first steps towards a standard for reporting on phylogenies: Minimum Information About a Phylogenetic Analysis (MIAPA); citation_author=J Leebens-Mack, T Vision, E Brenner, JE Bowers, S Cannon, MJ Clement, CW Cunningham, C DePamphilis, R DeSalle, JJ Doyle, Ja Eisen, X Gu, J Harshman, RK Jansen, Ea Kellogg, EV Koonin, BD Mishler, H Philippe, JC Pires, YL Qiu, SY Rhee, K Sjölander, DE Soltis, PS Soltis, DW Stevenson, K Wall, T Warnow, C Zmasek; citation_volume=10; citation_issue=2; citation_publication_date=2006; citation_pages=231-237; citation_doi=10.1089/omi.2006.10.231; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Evolutionary Bioinf; citation_title=Initial implementation of a comparative data analysis ontology; citation_author=F Prosdocimi, B Chisham, E Pontelli, JD Thompson, A Stoltzfus; citation_volume=5; citation_publication_date=2009; citation_pages=47-66; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=phyloXML: XML for evolutionary biology and comparative genomics; citation_author=MV Han, CM Zmasek; citation_volume=10; citation_publication_date=2009; citation_pages=356; citation_doi=10.1186/1471-2105-10-356; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=NeXML: rich, extensible, and verifiable representation of comparative data and metadata; citation_author=Ra Vos, JP Balhoff, Ja Caravas, MT Holder, H Lapp, WP Maddison, PE Midford, A Priyam, J Sukumaran, X Xia, A Stoltzfus; citation_volume=61; citation_issue=4; citation_publication_date=2012; citation_pages=675-689; citation_doi=10.1093/sysbio/sys025; citation_id=CR6">

    <meta name="citation_reference" content="citation_inbook_title=Package ‘ape’; citation_publication_date=2010; citation_id=CR7; citation_author=E Paradis; citation_author=B Bolker; citation_author=J Claude; citation_author=HS Cuong; citation_author=R Desper; citation_author=J Dutheil; citation_author=O Gascuel; citation_author=C Heibl; citation_author=D Lawson; citation_author=V Lefort; citation_author=J Lemon; citation_author=Y Noel; citation_author=J Nylander; citation_author=R Opgen-rhein; citation_author=DD Vienne">

    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=The Bioperl toolkit: Perl modules for the life sciences; citation_author=JE Stajich, D Block, K Boulez, SE Brenner, Sa Chervitz, C Dagdigian, G Fuellen, JGR Gilbert, I Korf, H Lapp, H Lehväslaiho, C Matsalla, CJ Mungall, BI Osborne, MR Pocock, P Schattner, M Senger, LD Stein, E Stupka, MD Wilkinson, E Birney; citation_volume=12; citation_issue=10; citation_publication_date=2002; citation_pages=1611-1618; citation_doi=10.1101/gr.361602; citation_id=CR8">

    <meta name="citation_reference" content="citation_inbook_title=Mesquite: A modular system for evolutionary analysis; citation_publication_date=2011; citation_id=CR9; citation_author=W Maddison; citation_author=D Maddison">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Biopython: freely available Python tools for computational molecular biology and bioinformatics; citation_author=PJA Cock, T Antao, JT Chang, BA Chapman, CJ Cox, A Dalke, I Friedberg, T Hamelryck, F Kauff, B Wilczynski, MJL de Hoon; citation_volume=25; citation_issue=11; citation_publication_date=2009; citation_pages=1422-1423; citation_doi=10.1093/bioinformatics/btp163; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=BioJava: an open-source framework for bioinformatics; citation_author=RCG Holland, TA Down, M Pocock, A Prlić, D Huen, K James, S Foisy, A Dräger, A Yates, M Heuer, MJ Schreiber; citation_volume=24; citation_issue=18; citation_publication_date=2008; citation_pages=2096-2097; citation_doi=10.1093/bioinformatics/btn397; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=BioRuby: bioinformatics software for the Ruby programming language; citation_author=N Goto, P Prins, M Nakao, R Bonnal, J Aerts, T Katayama; citation_volume=26; citation_issue=20; citation_publication_date=2010; citation_pages=2617-2619; citation_doi=10.1093/bioinformatics/btq475; citation_id=CR12">

    <meta name="citation_reference" content="citation_inbook_title=BioLib; citation_publication_date=2008; citation_id=CR13; citation_author=P Prins">

    <meta name="citation_reference" content="citation_inbook_title=BioSQL; citation_publication_date=2003; citation_id=CR14; citation_author=H Lapp">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Entrez Gene: gene-centered information at NCBI; citation_author=D Maglott, J Ostell, KD Pruitt, T Tatusova; citation_volume=33; citation_issue=Database issue; citation_publication_date=2005; citation_pages=D54-D58; citation_doi=10.1093/nar/gki031; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; citation_author=SF Altschul, TL Madden, aa Schäffer, J Zhang, Z Zhang, W Miller, DJ Lipman; citation_volume=25; citation_issue=17; citation_publication_date=1997; citation_pages=3389-3402; citation_doi=10.1093/nar/25.17.3389; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=BLAST+: architecture and applications; citation_author=C Camacho, G Coulouris, V Avagyan, N Ma, J Papadopoulos, K Bealer, TL Madden; citation_volume=10; citation_publication_date=2009; citation_pages=421; citation_doi=10.1186/1471-2105-10-421; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Comput Sci Eng; citation_title=Python for Scientific Computing; citation_author=TE Oliphant; citation_volume=9; citation_issue=3; citation_publication_date=2007; citation_pages=10-20; citation_doi=10.1109/MCSE.2007.58; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=PyCogent: a toolkit for making sense from sequence; citation_author=R Knight, P Maxwell, A Birmingham, J Carnes, JG Caporaso, BC Easton, M Eaton, M Hamady, H Lindsay, Z Liu, C Lozupone, D McDonald, M Robeson, R Sammut, S Smit, MJ Wakefield, J Widmann, S Wikman, S Wilson, H Ying, GA Huttley; citation_volume=8; citation_issue=8; citation_publication_date=2007; citation_pages=R171; citation_doi=10.1186/gb-2007-8-8-r171; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=DendroPy: a Python library for phylogenetic computing; citation_author=J Sukumaran, MT Holder; citation_volume=26; citation_issue=12; citation_publication_date=2010; citation_pages=1569-1571; citation_doi=10.1093/bioinformatics/btq228; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=ETE: a python Environment for Tree Exploration; citation_author=J Huerta-Cepas, J Dopazo, T Gabaldón; citation_volume=11; citation_publication_date=2010; citation_pages=24; citation_doi=10.1186/1471-2105-11-24; citation_id=CR21">

    <meta name="citation_reference" content="citation_title=p4: A Python package for phylogenetics; citation_publication_date=2003; citation_id=CR22; citation_author=PG Foster">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=NEXUS: an extensible file format for systematic information; citation_author=DR Maddison, DL Swofford, WP Maddison; citation_volume=46; citation_issue=4; citation_publication_date=1997; citation_pages=590-621; citation_doi=10.1093/sysbio/46.4.590; citation_id=CR23">

    <meta name="citation_reference" content="citation_title=The Newick tree format; citation_publication_date=1986; citation_id=CR24; citation_author=J Archie; citation_author=WH Day; citation_author=W Maddison; citation_author=C Meacham; citation_author=FJ Rohlf; citation_author=D Swofford; citation_author=J Felsenstein">

    <meta name="citation_reference" content="citation_journal_title=Software Pract Experience; citation_title=An open graph visualization system and its applications to software engineering; citation_author=E Gansner; citation_volume=00; citation_issue=S1; citation_publication_date=2000; citation_pages=1-5; citation_id=CR25">

    <meta name="citation_reference" content="citation_title=Exploring network structure, dynamics, and function using NetworkX; citation_inbook_title=Proceedings of the 7th Python in Science Conference (SciPy2008), SciPy; citation_publication_date=2008; citation_pages=11; citation_id=CR26; citation_author=AA Hagberg; citation_author=DA Schult; citation_author=PJ Swart">

    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Surprising complexity of the ancestral apoptosis network; citation_author=CM Zmasek, Q Zhang, Y Ye, A Godzik; citation_volume=8; citation_issue=10; citation_publication_date=2007; citation_pages=R226; citation_doi=10.1186/gb-2007-8-10-r226; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Comput Sci Eng; citation_title=Matplotlib: A 2D Graphics Environment; citation_author=JD Hunter; citation_volume=9; citation_issue=3; citation_publication_date=2007; citation_pages=90-95; citation_doi=10.1109/MCSE.2007.55; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=ATV: display and manipulation of annotated phylogenetic trees; citation_author=CM Zmasek, SR Eddy; citation_volume=17; citation_issue=4; citation_publication_date=2001; citation_pages=383-384; citation_doi=10.1093/bioinformatics/17.4.383; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=Trends Genet; citation_title=EMBOSS: The European Molecular Biology Open Software Suite; citation_author=P Rice, I Longden, A Bleasby; citation_volume=16; citation_issue=6; citation_publication_date=2000; citation_pages=276-277; citation_doi=10.1016/S0168-9525(00)02024-2; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0; citation_author=S Guindon, JF Dufayard, V Lefort, M Anisimova, W Hordijk, O Gascuel; citation_volume=59; citation_issue=3; citation_publication_date=2010; citation_pages=307-321; citation_doi=10.1093/sysbio/syq010; citation_id=CR31">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; citation_author=A Stamatakis; citation_volume=22; citation_issue=21; citation_publication_date=2006; citation_pages=2688-2690; citation_doi=10.1093/bioinformatics/btl446; citation_id=CR32">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=PAML 4: phylogenetic analysis by maximum likelihood; citation_author=Z Yang; citation_volume=24; citation_issue=8; citation_publication_date=2007; citation_pages=1586-1591; citation_doi=10.1093/molbev/msm088; citation_id=CR33">

    <meta name="citation_reference" content="citation_journal_title=Systematic Biol; citation_title=WASABI: an automated sequence processing system for multigene phylogenies; citation_author=F Kauff, C Cox, F Lutzoni; citation_volume=56; citation_issue=3; citation_publication_date=2007; citation_pages=523; citation_doi=10.1080/10635150701395340; citation_id=CR34">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=GenomeDiagram: a python package for the visualization of large-scale genomic data; citation_author=L Pritchard, JA White, PR Birch, IK Toth; citation_volume=22; citation_issue=5; citation_publication_date=2006; citation_pages=616-617; citation_doi=10.1093/bioinformatics/btk021; citation_id=CR35">

    <meta name="citation_reference" content="citation_journal_title=Biol Direct; citation_title=Telling the Whole Story in a 10,000-Genome World; citation_author=RG Beiko; citation_volume=6; citation_publication_date=2011; citation_pages=34; citation_doi=10.1186/1745-6150-6-34; citation_id=CR36">

    <meta name="citation_reference" content="citation_journal_title=BMC Evolutionary Biol; citation_title=Structural and evolutionary divergence of eukaryotic protein kinases in Apicomplexa; citation_author=E Talevich, A Mirza, N Kannan; citation_volume=11; citation_publication_date=2011; citation_pages=321; citation_doi=10.1186/1471-2148-11-321; citation_id=CR37">

    <meta name="citation_reference" content="citation_journal_title=Am J Bot; citation_title=Understanding angiosperm diversification using small and large phylogenetic trees; citation_author=SA Smith, JM Beaulieu, A Stamatakis, MJ Donoghue; citation_volume=98; citation_issue=3; citation_publication_date=2011; citation_pages=404-414; citation_doi=10.3732/ajb.1000481; citation_id=CR38">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Darwin’s abominable mystery: Insights from a supertree of the angiosperms; citation_author=TJ Davies, TG Barraclough, MW Chase, PS Soltis, DE Soltis, V Savolainen; citation_volume=101; citation_issue=7; citation_publication_date=2004; citation_pages=1904-1909; citation_doi=10.1073/pnas.0308127100; citation_id=CR39">

    <meta name="citation_author" content="Eric Talevich">

    <meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, USA">

    <meta name="citation_author" content="Brandon M Invergo">

    <meta name="citation_author_institution" content="Institute of Evolutionary Biology (CSIC-UPF), CEXS-UPF-PRBB, Barcelona, Spain">

    <meta name="citation_author" content="Peter JA Cock">

    <meta name="citation_author_institution" content="James Hutton Institute, Invergowrie, Dundee, UK">

    <meta name="citation_author" content="Brad A Chapman">

    <meta name="citation_author_institution" content="Harvard School of Public Health Bioinformatics Core, Boston, USA">



        <meta name="format-detection" content="telephone=no">

        <link rel="shortcut icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-9ff1bf1161.ico">
<link rel="apple-touch-icon" sizes="114x114" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/app-icon-114x114-aa3514ee38.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png">
<link rel="apple-touch-icon" sizes="180x180" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/app-icon-180x180-230f795aee.png">
<link rel="icon" type="image/png" sizes="16x16" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-16x16-25a0239eec.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-32x32-637712e609.png">
<link rel="icon" type="image/png" sizes="96x96" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-96x96-9f5dc535c7.png">
<link rel="icon" type="image/png" sizes="194x194" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-194x194-d2a2617de2.png">
<meta name="msapplication-TileColor" content="#e6e6e6">
<meta name="msapplication-TileImage" content="/static/images/favicons/bmc/app-icon-144x144-67f5b60136.png">
<meta name="theme-color" content="#1b3051">

        <script type="text/javascript" src="https://consumer.krxd.net/consent/set/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;dc=1&amp;al=1&amp;tg=1&amp;cd=0&amp;sh=0&amp;re=0&amp;callback=Krux.ns.macmillan.kxjsonp_consent_set_1"></script><script type="text/javascript" src="https://consumer.krxd.net/consent/get/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;callback=Krux.ns.macmillan.kxjsonp_consent_get_0"></script><script async="" src="//cdn.krxd.net/ctjs/controltag.js.f0b61617fbc713883207cc130147aee2"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        

<link rel="stylesheet" media="screen" href="core-18eed0985c.css">
<link rel="stylesheet" media="print" href="print-7680b1a80b.css">

<link rel="stylesheet" id="js-mustard" href="app-f4a9218f4d.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">


        

        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>

        
    <script type="text/javascript">
        config = {
            env: 'live-snpaas',
            site: 'bmcbioinformatics.biomedcentral.com',
            siteWithPath: 'bmcbioinformatics.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcbioinformatics',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1471-2105-13-209',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            hasFigshareInvoked: false,
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>

        <!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script> 
        
    <script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/1471-2105-13-209","articleType":"Software","peerReviewType":"Closed","supplement":null,"keywords":"Application Programming Interface;Configuration File;Tree Object;Computational Pipeline;Standard File Format"},"contentInfo":{"imprint":"BioMed Central","title":"Bio.Phylo: A unified toolkit for processing, analyzing and visualizing phylogenetic trees in Biopython","publishedAt":1345507200000,"publishedAtDate":"2012-08-21","author":["Eric Talevich","Brandon M Invergo","Peter JA Cock","Brad A Chapman"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"bmcbioinformatics.biomedcentral.com","volume":"13","issue":"1","title":"BMC Bioinformatics","type":"SERIES","journalID":12859,"gaCode":"UA-62065396-3","section":[{"sectionId":"AC_25f68238fdecde1484666136803a0470","sectionName":"Comparative genomics"}]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Software","publishingSegment":"BMC Series - Life and Health"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script id="springerlink-event-tracker" src="eventTracker.asc"></script>


        <script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script>
        <script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script>
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle button { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle button { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style><script type="text/javascript" src="jquery-3.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl + '&gtm_auth=eWcsn65X4UbU5hx6f5HaEA&gtm_preview=env-59&gtm_cookies_win=x';
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-209">
    

        
        
        <meta property="og:url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-209">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Bioinformatics">
        <meta property="og:title" content="Bio.Phylo: A unified toolkit for processing, analyzing and visualizing phylogenetic trees in Biopython">
        <meta property="og:description" content="Ongoing innovation in phylogenetics and evolutionary biology has been accompanied by a proliferation of software tools, data formats, analytical techniques and web servers. This brings with it the challenge of integrating phylogenetic and other related biological data found in a wide variety of formats, and underlines the need for reusable software that can read, manipulate and transform this information into the various forms required to build computational pipelines. We built a Python software library for working with phylogenetic data that is tightly integrated with Biopython, a broad-ranging toolkit for computational biology. Our library, Bio.Phylo, is highly interoperable with existing libraries, tools and standards, and is capable of parsing common file formats for phylogenetic trees, performing basic transformations and manipulations, attaching rich annotations, and visualizing trees. We unified the modules for working with the standard file formats Newick, NEXUS and phyloXML behind a consistent and simple API, providing a common set of functionality independent of the data source. Bio.Phylo meets a growing need in bioinformatics for working with heterogeneous types of phylogenetic data. By supporting interoperability with multiple file formats and leveraging existing Biopython features, this library simplifies the construction of phylogenetic workflows. We also provide examples of the benefits of building a community around a shared open-source project. Bio.Phylo is included with Biopython, available through the Biopython website, http://biopython.org .">
        <meta property="og:image" content="https://media.springernature.com/w110/springer-static/cover/journal/12859.jpg">
    
        
    <script src="figshare.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019061101.js" async=""></script><script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script></head>

    <body class="journal journal-fulltext"><div id="MathJax_Message" style="display: none;"></div>
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK&gtm_auth=eWcsn65X4UbU5hx6f5HaEA&gtm_preview=env-59&gtm_cookies_win=x"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->

            
        
    

        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-alert"><path d="M1 14h14v-.172a1 1 0 0 0-.293-.707L13 11.414V8A5 5 0 0 0 3 8v3h2.5a.5.5 0 1 1 0 1H2.414l-1.121 1.121a1 1 0 0 0-.293.707V14zm5 1a2 2 0 1 0 4 0H6zm-5 0a1 1 0 0 1-1-1v-.172a2 2 0 0 1 .586-1.414L2 11V8a6 6 0 1 1 12 0v3l1.414 1.414A2 2 0 0 1 16 13.828V14a1 1 0 0 1-1 1h-4a3 3 0 0 1-6 0H1zM8.997 2.082a1 1 0 1 0-1.993 0c-.336.057-.662.14-.977.25a2 2 0 1 1 3.945 0 5.958 5.958 0 0 0-.975-.25z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-article"><path d="M14 14c0 .6.4 1 1 1h1c.5 0 1-.5 1-1V5h-3v9zm-1 0V1H1v13c0 .5.4 1 1 1h11.3c-.2-.3-.3-.6-.3-1zM0 14V1c0-.6.5-1 1-1h12c.6 0 1 .5 1 1v3h4v10c0 1.1-.9 2-2 2H2c-1.1 0-2-.9-2-2zM4 4v2h6V4H4zM3 3h8v4H3V3zm0 5h8v1H3V8zm0 2h8v1H3v-1zm0 2h8v1H3v-1z"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"></path></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-editor"><path d="M10.2 16.9c1.3-.2 2.5-.7 3.5-1.5-.9-2.5-2.1-3.9-3.8-4.3V9.8c1.2-.4 2-1.5 2-2.8 0-1.7-1.3-3-3-3S6 5.3 6 7c0 1.3.8 2.4 2 2.8v1.3c-1.7.4-2.9 1.8-3.8 4.3 1 .8 2.2 1.3 3.5 1.5.5.1.9.1 1.3.1s.8 0 1.2-.1zm4.4-2.2C16.1 13.3 17 11.2 17 9c0-4.4-3.6-8-8-8S1 4.6 1 9c0 2.2.9 4.3 2.4 5.7.8-2 1.8-3.6 3.5-4.3C5.8 9.7 5 8.4 5 7c0-2.2 1.8-4 4-4s4 1.8 4 4c0 1.4-.8 2.7-1.9 3.4 1.6.7 2.7 2.3 3.5 4.3zM9 18c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9z"></path></symbol><symbol id="icon-explore"><path d="M9 17c4.4 0 8-3.6 8-8s-3.6-8-8-8-8 3.6-8 8 3.6 8 8 8zm0 1c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm0-2.5c-.3 0-.5-.2-.5-.5s.2-.5.5-.5c3 0 5.4-2.4 5.5-5.3 0-.3.2-.5.5-.5s.5.2.5.5c-.1 3.5-3 6.3-6.5 6.3zm-.3-13c.3 0 .5.2.5.5s-.2.5-.4.5c-3 .1-5.3 2.6-5.3 5.5 0 .3-.2.5-.5.5s-.5-.2-.5-.5c0-3.5 2.7-6.3 6.2-6.5zm1.1 8.1L8.6 9.4c-.2-.2-.2-.5 0-.7.2-.2.5-.2.7 0l1.2 1.2 1.6-4-4.5 1.7-1.8 4.5 4-1.5zm4.1-6.5l-2.7 6.7c-.1.3-.3.5-.6.6l-6.7 2.7 2.7-6.7c.2-.4.4-.6.7-.7l6.6-2.6z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-alert"><path d="M1 14h14v-.172a1 1 0 0 0-.293-.707L13 11.414V8A5 5 0 0 0 3 8v3h2.5a.5.5 0 1 1 0 1H2.414l-1.121 1.121a1 1 0 0 0-.293.707V14zm5 1a2 2 0 1 0 4 0H6zm-5 0a1 1 0 0 1-1-1v-.172a2 2 0 0 1 .586-1.414L2 11V8a6 6 0 1 1 12 0v3l1.414 1.414A2 2 0 0 1 16 13.828V14a1 1 0 0 1-1 1h-4a3 3 0 0 1-6 0H1zM8.997 2.082a1 1 0 1 0-1.993 0c-.336.057-.662.14-.977.25a2 2 0 1 1 3.945 0 5.958 5.958 0 0 0-.975-.25z"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-article"><path d="M14 14c0 .6.4 1 1 1h1c.5 0 1-.5 1-1V5h-3v9zm-1 0V1H1v13c0 .5.4 1 1 1h11.3c-.2-.3-.3-.6-.3-1zM0 14V1c0-.6.5-1 1-1h12c.6 0 1 .5 1 1v3h4v10c0 1.1-.9 2-2 2H2c-1.1 0-2-.9-2-2zM4 4v2h6V4H4zM3 3h8v4H3V3zm0 5h8v1H3V8zm0 2h8v1H3v-1zm0 2h8v1H3v-1z"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 6 9"><path d="M1.14.854a.515.515 0 0 1 0-.708.467.467 0 0 1 .68 0l3.847 4.008a.5.5 0 0 1 0 .692L1.82 8.854a.467.467 0 0 1-.68 0 .515.515 0 0 1 0-.708L4.643 4.5 1.141.854z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><g filter="url(#a)"><path fill="#FFF" d="M0 31.01h60.643V0H0z"></path></g></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download-rounded"><path d="M0 13c0-.556.449-1 1.002-1h9.996a.999.999 0 1 1 0 2H1.002A1.006 1.006 0 0 1 0 13zM7 1v6.8l2.482-2.482c.392-.392 1.022-.4 1.403-.02a1.001 1.001 0 0 1 0 1.417l-4.177 4.177a1.001 1.001 0 0 1-1.416 0L1.115 6.715a.991.991 0 0 1-.016-1.4 1 1 0 0 1 1.42.003L5 7.8V1c0-.55.444-.996 1-.996.552 0 1 .445 1 .996z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-editor"><path d="M10.2 16.9c1.3-.2 2.5-.7 3.5-1.5-.9-2.5-2.1-3.9-3.8-4.3V9.8c1.2-.4 2-1.5 2-2.8 0-1.7-1.3-3-3-3S6 5.3 6 7c0 1.3.8 2.4 2 2.8v1.3c-1.7.4-2.9 1.8-3.8 4.3 1 .8 2.2 1.3 3.5 1.5.5.1.9.1 1.3.1s.8 0 1.2-.1zm4.4-2.2C16.1 13.3 17 11.2 17 9c0-4.4-3.6-8-8-8S1 4.6 1 9c0 2.2.9 4.3 2.4 5.7.8-2 1.8-3.6 3.5-4.3C5.8 9.7 5 8.4 5 7c0-2.2 1.8-4 4-4s4 1.8 4 4c0 1.4-.8 2.7-1.9 3.4 1.6.7 2.7 2.3 3.5 4.3zM9 18c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9z"></path></symbol><symbol id="icon-explore"><path d="M9 17c4.4 0 8-3.6 8-8s-3.6-8-8-8-8 3.6-8 8 3.6 8 8 8zm0 1c-5 0-9-4-9-9s4-9 9-9 9 4 9 9-4 9-9 9zm0-2.5c-.3 0-.5-.2-.5-.5s.2-.5.5-.5c3 0 5.4-2.4 5.5-5.3 0-.3.2-.5.5-.5s.5.2.5.5c-.1 3.5-3 6.3-6.5 6.3zm-.3-13c.3 0 .5.2.5.5s-.2.5-.4.5c-3 .1-5.3 2.6-5.3 5.5 0 .3-.2.5-.5.5s-.5-.2-.5-.5c0-3.5 2.7-6.3 6.2-6.5zm1.1 8.1L8.6 9.4c-.2-.2-.2-.5 0-.7.2-.2.5-.2.7 0l1.2 1.2 1.6-4-4.5 1.7-1.8 4.5 4-1.5zm4.1-6.5l-2.7 6.7c-.1.3-.3.5-.6.6l-6.7 2.7 2.7-6.7c.2-.4.4-.6.7-.7l6.6-2.6z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.6 10.6c2.2 0 4-1.7 4-4s-1.7-4-4-4-4 1.7-4 4 1.8 4 4 4zm4 1.4l1.5-1.5 3.7 3.6c.4.5.4 1.2 0 1.6-.4.4-1.1.4-1.6 0L10.6 12zm-4 1.2C2.9 13.2 0 10.3 0 6.6S2.9 0 6.6 0s6.6 2.9 6.6 6.6-2.9 6.6-6.6 6.6z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcbioinformatics/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1471-2105-13-209;kwrd=Application Programming Interface,Configuration File,Tree Object,Computational Pipeline,Standard File Format;pmc=L15001,B12050,I23050,L17004,M14018,M14018;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/bmcbioinformatics/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2105-13-209&amp;kwrd=Application Programming Interface,Configuration File,Tree Object,Computational Pipeline,Standard File Format&amp;pmc=L15001,B12050,I23050,L17004,M14018,M14018&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/bmcbioinformatics/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2105-13-209&amp;kwrd=Application Programming Interface,Configuration File,Tree Object,Computational Pipeline,Standard File Format&amp;pmc=L15001,B12050,I23050,L17004,M14018,M14018&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            <div id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
        </div>
    </div>

        
         
    
        <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>
    
    
        <header class="c-popup-search" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="c-navbar__container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img alt="BMC" itemprop="logo" role="img" src="logo-bmc-white-series-1da6525f4e.svg" width="76" height="18">
        <div class="c-logo__strapline">
            <img alt="Part of Springer Nature" role="img" src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" width="173" height="16">
        </div>
    </a>
</div>

            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav c-navbar__nav--right">
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true" focusable="false">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text">
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true" focusable="false">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>

    

        
            <header class="c-journal-header c-journal-header--bmc-bioinformatics ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcbioinformatics.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Bioinformatics</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="c-navbar__container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcbioinformatics.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-page-layout c-page-layout--fulltext" data-component="sticky-container">
                <div class="c-page-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Implementation">Implementation</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec15" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Discussion">Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec17" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec18" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Availability and requirements">Availability and requirements</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-page-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Software</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>
    

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Bio.Phylo: A unified toolkit for processing, analyzing and visualizing phylogenetic trees in Biopython</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Eric&nbsp;Talevich</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:etal@uga.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Brandon&nbsp;M&nbsp;Invergo</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Peter&nbsp;JA&nbsp;Cock</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Brad&nbsp;A&nbsp;Chapman</span><sup><a href="#Aff4" class="AffiliationID">4</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Bioinformatics</span></span><span class="ArticleCitation_Year">2012</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">13</strong>:209</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/1471-2105-13-209">https://doi.org/10.1186/1471-2105-13-209</a></p><p class="Copyright">© &nbsp;Talevich et al.; licensee BioMed Central Ltd.&nbsp;2012</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>6&nbsp;March&nbsp;2012</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>8&nbsp;August&nbsp;2012</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">21&nbsp;August&nbsp;2012</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">Ongoing innovation in phylogenetics and 
evolutionary biology has been accompanied by a proliferation of software
 tools, data formats, analytical techniques and web servers. This brings
 with it the challenge of integrating phylogenetic and other related 
biological data found in a wide variety of formats, and underlines the 
need for reusable software that can read, manipulate and transform this 
information into the various forms required to build computational 
pipelines.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">We built a Python software library for 
working with phylogenetic data that is tightly integrated with 
Biopython, a broad-ranging toolkit for computational biology. Our 
library, Bio.Phylo, is highly interoperable with existing libraries, 
tools and standards, and is capable of parsing common file formats for 
phylogenetic trees, performing basic transformations and manipulations, 
attaching rich annotations, and visualizing trees. We unified the 
modules for working with the standard file formats Newick, NEXUS and 
phyloXML behind a consistent and simple API, providing a common set of 
functionality independent of the data source.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p class="Para">Bio.Phylo meets a growing need in 
bioinformatics for working with heterogeneous types of phylogenetic 
data. By supporting interoperability with multiple file formats and 
leveraging existing Biopython features, this library simplifies the 
construction of phylogenetic workflows. We also provide examples of the 
benefits of building a community around a shared open-source project. 
Bio.Phylo is included with Biopython, available through the Biopython 
website, <span class="ExternalRef"><a href="http://biopython.org/" target="_blank" class="ext-link"><span class="RefSource">http://biopython.org</span></a></span>.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Application Programming Interface</li><li class="c-keywords__item" itemprop="about">Configuration File</li><li class="c-keywords__item" itemprop="about">Tree Object</li><li class="c-keywords__item" itemprop="about">Computational Pipeline</li><li class="c-keywords__item" itemprop="about">Standard File Format</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Comparative
 methods in biology have been an active area of scientific software 
development since the wider availability of computing resources made 
such large-scale quantitative analyses feasible [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>].
 In recent years, the number and range of tools for working with 
phylogenetic information alone has expanded dramatically, creating new 
opportunities as well as challenges in data integration [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]. At the same time, we have seen new efforts to create standards for data exchange and storage [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span><span>, <span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>, <span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>, </span><span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 There is thus a growing need for modular software toolkits that can 
integrate cleanly into workflows for bioinformatics analyses that 
include phylogenetic data, as well as facilitate tool development by 
providing a high-level application programming interface (API) 
independent of data storage formats.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Freely
 available, open-source software libraries now play a major role in 
integrating software components for bioinformatics analyses. Generalized
 toolkits for working with phylogenetic information are already 
available and in wide use, including the Ape package for the R 
statistical programming language [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], BioPerl [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>] and Bio::Phylo [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>] for Perl, and Mesquite [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]
 for Java. The Open Bioinformatics Foundation in particular supports an 
ecosystem of broadly aimed bioinformatics libraries, collectively 
referred to as the Bio* or “Bio-star” projects. These include BioPerl [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>], Biopython [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>], BioJava [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], BioRuby [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>], BioLib [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] and BioSQL [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].
 All of these libraries provide common functionality to each of their 
target programming languages or environments: read and write a variety 
of file formats used in bioinformatics, communicate with public web 
services such as NCBI Entrez [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], access widely used stand-alone programs such as BLAST [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>],
 and define fundamental data types for storing biological information, 
such as annotated sequence records, protein structures and phylogenetic 
trees.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We chose Python as the implementation language for our library due to its increasing usage in scientific work [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>].
 It allows for particularly concise, easy-to-read code, has extensive 
library support, and enables the same code to be run on all major 
operating systems. In addition to Biopython, several other high-quality 
Python libraries are available for phylogenetics: PyCogent [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>], DendroPy [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>], ETE [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] and p4 [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]
 each serve specific problem domains well. However, they focus 
specificially on phylogenetics and are not intended to be 
general-purpose frameworks for processing biological data.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 this article we present a framework for phylogenetics in the Python 
programming language, fully integrated with the Biopython toolkit. We 
supplemented existing support for the NEXUS [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] and Newick [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]
 standards in Biopython with a full-featured phylogenetic module, 
Bio.Phylo, incorporating new input/output support for the phyloXML 
standard [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>].
 We then integrated the new I/O infrastructure with related code in 
Biopython and added more features that would be useful for researchers, 
with a focus on easy interoperability and rapid script development.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Our
 contributions include an I/O framework with parsers and serializers for
 the standard Newick, NEXUS and phyloXML formats for phylogenetic trees,
 integrated through a consistent and simple API that is familiar to 
users of Biopython and BioPerl, as well as a set of common functions to 
perform on trees independently of the source data format. This framework
 includes comprehensive class definitions for the rich annotation types 
which can be serialized in the standard phyloXML format. We also provide
 several options for visualizing trees, and convenient integration with 
popular third-party tools.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Implementation" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Implementation">Implementation</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Implementation" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Bio.Phylo
 is written as a sub-package within the Biopython code base, and is 
installed along with the Biopython distribution. It has been available 
as part of the Biopython library since version 1.54, released in May 
2010.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 library can be used in Python versions 2.5 through 2.7 and 3.1 or 
later, without any external libraries required at the time of 
installation. Functions that rely on external libraries are written 
within a separate module of the code base, and import their dependencies
 at run-time. This design makes it possible to install Biopython and use
 the rest of Bio.Phylo without having installed the dependencies for 
these specific functions. Because Bio.Phylo is written entirely in 
Python, it also runs on alternative implementations of Python: Jython 
2.5 and Pypy 1.6 through 1.9 in particular pass the module’s unit 
testing suite.</p><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">I/O functions for standard file formats</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A
 unified API for input and output is provided for the Newick, NEXUS and 
phyloXML formats with the same underlying object structures. The simple 
API style is shared with Bio.SeqIO, AlignIO, Motif and other modules 
within Biopython; the API also resembles that of BioPerl. The <span xmlns="" class="EmphasisFontCategoryNonProportional">read</span> and <span xmlns="" class="EmphasisFontCategoryNonProportional">write</span>
 functions accept a filename or handle, so they also work with Unix 
pipes, network handles or other file-like objects. As in Biopython’s 
SeqIO and AlignIO, a <span xmlns="" class="EmphasisFontCategoryNonProportional">convert</span> function is available to convert between any two of the supported formats with a single call.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 I/O code is designed for simple addition of other file formats without 
disturbing the existing code for Newick, NEXUS and phyloXML. Parsing and
 serialization code is separated from the internal tree object 
representation. All parsers return a common object type, the tree, 
independently of the source data format, and the parsed tree objects all
 support a common set of operations.</p>
              </section><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Common tree representation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A
 phylogeny is represented by a Tree object which contains global 
information about the phylogeny, such as whether it is rooted, and a 
reference to the root Clade. Each Clade contains a reference to its 
child clades, a simple Python list of further Clade objects, nested 
recursively. The Clade object also contains information about the node 
occuring at the split or tip it represents, such as the length of the 
branch leading to it and the name of the node. There is no additional 
complex data structure operating “under the hood”, and trees are not 
required to be bifurcating, although the functions in Bio.Phylo 
currently assume that each Clade has a single parent, i.e., the topology
 of the Tree is indeed a tree and not a network. This straightforward 
design is conducive to implementing algorithms in a form that is easy to
 read and understand, with minimal need for management of the object’s 
internal data representation.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 basic Tree and Clade objects store the intersection of the information 
that can be represented in the Newick and phyloXML formats. To store 
additional format-specific attributes, we defined separate Newick and 
PhyloXML sub-classes which inherit from the basic Tree and Clade 
classes. Classes for each element type defined in the phyloXML 
specification have been implemented in the Bio.Phylo.PhyloXML module, 
allowing richer annotation types to be attached to Tree objects. For 
convenience in adding graphical cues, the Clade class also has 
attributes for the displayed color and width of a branch. The properties
 of the branch color and width attributes follow the phyloXML 
specification, and are available on the common Clade class due to their 
usefulness, particularly during interactive work. In accordance with the
 phyloXML specification, these attributes apply in a cascading manner 
down the clade: for example, if the root Clade object is assigned the 
color blue, the entire tree will be displayed as blue unless a child 
clade overrides this attribute.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Methods for tree inspection and manipulation</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 Tree and Clade objects also implement common methods for tree 
manipulation and simple analyses that might be used routinely in 
bioinformatics workflows. These include methods for tree search and 
traversal, extracting basic information, and modifying or manipulating 
the tree. A listing of these methods and other functions available in 
Bio.Phylo is given in Table <span class="InternalRef"><a href="#Tab1">1</a></span>.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">
                          <strong class="EmphasisTypeBold">Built-in functions and tree methods</strong>
                        </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Source</p></th><th><p xmlns="" class="SimplePara">Function</p></th><th><p xmlns="" class="SimplePara">Description</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">read</span>
                            </p></td><td><p xmlns="" class="SimplePara">Parse a file in the given format and return a single tree.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">parse</span>
                            </p></td><td><p xmlns="" class="SimplePara">Iteratively parse a file and return each of the trees it contains.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">write</span>
                            </p></td><td><p xmlns="" class="SimplePara">Write a sequence of trees to file in the given format.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">convert</span>
                            </p></td><td><p xmlns="" class="SimplePara">Convert between two tree file formats.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">draw</span>
                            </p></td><td><p xmlns="" class="SimplePara">Plot the given tree using matplotlib (or pylab).</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">draw_ascii</span>
                            </p></td><td><p xmlns="" class="SimplePara">Draw an ascii-art phylogram of the given tree.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">draw_graphviz</span>
                            </p></td><td><p xmlns="" class="SimplePara">Display a tree or clade as a graph, using the graphviz engine.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Bio.Phylo</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">to_networkx</span>
                            </p></td><td><p xmlns="" class="SimplePara">Convert a Tree object to a NetworkX graph object.</p></td></tr><tr class="noclass"><td colspan="3"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bio.Phylo.BaseTree</em>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">find_elements</span>
                            </p></td><td><p xmlns="" class="SimplePara">Find all tree elements matching the given attributes.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">find_clades</span>
                            </p></td><td><p xmlns="" class="SimplePara">Find each clade containing a matching element.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">find_any</span>
                            </p></td><td><p xmlns="" class="SimplePara">Return the first matching element found by <span class="EmphasisFontCategoryNonProportional">find_elements</span>, if any.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">get_path</span>
                            </p></td><td><p xmlns="" class="SimplePara">List the clades directly between the current node and the target.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">get_nonterminals</span>
                            </p></td><td><p xmlns="" class="SimplePara">List of all of the tree or clade’s internal nodes.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">get_terminals</span>
                            </p></td><td><p xmlns="" class="SimplePara">List of all of the tree or clade’s “leaf” nodes.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">trace</span>
                            </p></td><td><p xmlns="" class="SimplePara">List of all clade object between two targets in the tree/clade.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">common_ancestor</span>
                            </p></td><td><p xmlns="" class="SimplePara">Most recent common ancestor (clade) of all the given targets.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">count_terminals</span>
                            </p></td><td><p xmlns="" class="SimplePara">Count the number of terminal nodes within the tree.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">depths</span>
                            </p></td><td><p xmlns="" class="SimplePara">Create a mapping of tree clades to depths (by branch length).</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">distance</span>
                            </p></td><td><p xmlns="" class="SimplePara">Calculate the sum of the branch lengths between two targets.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">is_bifurcating</span>
                            </p></td><td><p xmlns="" class="SimplePara">Return True if tree downstream of node is strictly bifurcating.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">is_monophyletic</span>
                            </p></td><td><p xmlns="" class="SimplePara">If the given terminals comprise a complete subclade, return the MRCA.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">is_parent_of</span>
                            </p></td><td><p xmlns="" class="SimplePara">True if target is a descendent of the tree.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">is_preterminal</span>
                            </p></td><td><p xmlns="" class="SimplePara">True if all direct descendents are terminal.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">total_branch_length</span>
                            </p></td><td><p xmlns="" class="SimplePara">Calculate the sum of all the branch lengths in the tree.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">collapse</span>
                            </p></td><td><p xmlns="" class="SimplePara">Deletes target from the tree, relinking its children to its parent.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">collapse_all</span>
                            </p></td><td><p xmlns="" class="SimplePara">Collapse all the descendents of the tree, leaving only terminals.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">ladderize</span>
                            </p></td><td><p xmlns="" class="SimplePara">Sort clades in-place according to the number of terminal nodes.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">prune</span>
                            </p></td><td><p xmlns="" class="SimplePara">Prunes a terminal clade from the tree.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">TreeMixin</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">split</span>
                            </p></td><td><p xmlns="" class="SimplePara">Generate <em class="EmphasisTypeItalic">n</em> (default 2) new descendants.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><span class="EmphasisFontCategoryNonProportional">Tree</span>, <span class="EmphasisFontCategoryNonProportional">Clade</span></p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">is_terminal</span>
                            </p></td><td><p xmlns="" class="SimplePara">True if the node has no descendents.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Tree</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">root_with_outgroup</span>
                            </p></td><td><p xmlns="" class="SimplePara">Reroot the tree with the specified outgroup clade.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Tree</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">root_at_midpoint</span>
                            </p></td><td><p xmlns="" class="SimplePara">Reroot the tree at the midpoint between the two most distant terminals.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Tree</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">format</span>
                            </p></td><td><p xmlns="" class="SimplePara">Serialize the tree as a string in the specified file format.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Tree</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">as_phyloxml</span>
                            </p></td><td><p xmlns="" class="SimplePara">Convert the tree to its PhyloXML subclass equivalent.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Tree</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">from_clade</span>
                            </p></td><td><p xmlns="" class="SimplePara">Create a new Tree object given a clade.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Tree</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">randomized</span>
                            </p></td><td><p xmlns="" class="SimplePara">Create a randomized bifurcating tree, given a list of taxa.</p></td></tr><tr class="noclass"><td colspan="3"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bio.Phylo.PhyloXML</em>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">Phylogeny</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">to_alignment</span>
                            </p></td><td><p xmlns="" class="SimplePara">Construct an alignment from the aligned sequences in this tree.</p></td></tr><tr class="noclass"><td colspan="3"><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">Bio.Phylo.PAML</em>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">chi2</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">cdf_chi2</span>
                            </p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">χ</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> cumulative distribution function, for log-likelihood ratio tests.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">baseml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">read</span>
                            </p></td><td><p xmlns="" class="SimplePara">Parse a BASEML results file.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">codeml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">read</span>
                            </p></td><td><p xmlns="" class="SimplePara">Parse a CODEML results file.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">yn00</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">read</span>
                            </p></td><td><p xmlns="" class="SimplePara">Parse a yn00 results file.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">write_ctl_file</span>
                            </p></td><td><p xmlns="" class="SimplePara">Dynamically build a program-specific control file.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">read_ctl_file</span>
                            </p></td><td><p xmlns="" class="SimplePara">Parse a control file to create a program-specific class instance.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">print_options</span>
                            </p></td><td><p xmlns="" class="SimplePara">Print all of the program options and their current settings.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">set_options</span>
                            </p></td><td><p xmlns="" class="SimplePara">Set the value of a program option.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">get_option</span>
                            </p></td><td><p xmlns="" class="SimplePara">Return the value of a program option.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">get_all_options</span>
                            </p></td><td><p xmlns="" class="SimplePara">Return the current values of all the program options.</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">_paml.Paml</span>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisFontCategoryNonProportional">run</span>
                            </p></td><td><p xmlns="" class="SimplePara">Run a PAML program and parse the results.</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Public methods and 
functions provided in the Bio.Phylo module and sub-modules. An 
up-to-date version of this information is available at <span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://biopython.org/DIST/docs/api/" target="_blank" class="ext-link"><span class="RefSource">http://biopython.org/DIST/docs/api/</span></a></span>.</p>
                    </div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Both
 the Tree and Clade classes inherit from a third class, TreeMixin, which
 implements these common tree methods. Therefore, most of the same 
methods are available on both the Tree object and any of the Clade 
objects it contains; in practice, one can usually ignore the distinction
 between the global Tree object and the root Clade.</p>
              </section><section id="Sec6" class="Section2 RenderAsSection2">
                <h3 class="Heading">Visualization</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We implemented several mechanisms for displaying trees. The <span xmlns="" class="EmphasisFontCategoryNonProportional">draw</span> function displays a rooted phylogram (Figure <span class="InternalRef"><a href="#Fig1">1</a></span>), in the style of Phylip’s <span xmlns="" class="EmphasisFontCategoryNonProportional">drawgram</span> program, while <span xmlns="" class="EmphasisFontCategoryNonProportional">draw_graphviz</span> displays an unrooted cladogram using the Graphviz programs [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>] for layout and the NetworkX library [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>] as an intermediate graph representation.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_Article_5387_Fig1_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2012_Article_5387_Fig1_HTML.jpg" alt="Figure 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_Article_5387_Fig1_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_Article_5387_Fig1_HTML.jpg" alt="Figure 1"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Tree visualization.</strong>
 An example tree with the code to generate each plot shown below each 
plot. First, a phyloXML tree of the Apaf-1 protein family [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]
 is downloaded, read by Bio.Phylo, and plotted with default settings. 
The tree is then rerooted at the midpoint of its two most divergent tips
 and ladderized such that sibling clades with a larger number of 
descendents are listed first. The clade of genes belonging to vertebrate
 species is identified as the common ancestor of the human and 
zebrafish, after inspection of the original tree. The vertebrate clade 
is highlighted with the color fuchsia and an increased branch width, and
 the rest of the tree is colored gray. Finally, the tree is plotted 
again</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The generated plots can be further modified using the built-in functions of matplotlib or PyLab [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>].
 In case no visualization libraries are available on a system, 
plain-text representations of a tree are also possible. The function <span xmlns="" class="EmphasisFontCategoryNonProportional">draw_ascii</span> draws an “ASCII-art” tree to standard output or a given file handle. Using the built-in Python <span xmlns="" class="EmphasisFontCategoryNonProportional">print</span>
 statement on a Tree object shows the nested object hierarchy, including
 class names and the values of attributes such as branch length.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Since
 the complete phyloXML specification is implemented, files can be saved 
by Bio.Phylo with graphical annotations and then rendered with popular 
visualization tools such as Archaeopteryx [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].
 The cascading behavior of clade colors and branch widths is shared by 
Archaeopteryx and other tree visualization software that implements 
phyloXML.</p>
              </section><section id="Sec7" class="Section2 RenderAsSection2">
                <h3 class="Heading">Wrappers for third-party programs</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Biopython
 includes a common framework for building wrappers for external 
programs. This framework allows us to leverage the functionality of 
widely used third-party programs from within Biopython, rather than 
reimplement those functions. For example, Biopython supports the Phylip 
suite of programs via EMBOSS [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>], with wrappers implemented in the Bio.Emboss.Applications module. We follow this rationale in Bio.Phylo.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Within the sub-module Bio.Phylo.Applications, we currently provide wrappers for the tree inference programs PhyML [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] and RAxML [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]. The code scaffolding is in place to add more, using the same common Bio.Application framework.</p>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">PAML integration</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Bio.Phylo also includes wrappers for the PAML suite of programs [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>], specifically the analysis programs <span xmlns="" class="EmphasisFontCategoryNonProportional">yn00</span>, <span xmlns="" class="EmphasisFontCategoryNonProportional">baseml</span>, and <span xmlns="" class="EmphasisFontCategoryNonProportional">codeml</span>. Additionally, we created a pure-Python re-implementation of the program <span xmlns="" class="EmphasisFontCategoryNonProportional">chi2</span>
 as a simple means to perform likelihood ratio tests. Since the analysis
 programs are operated through the use of configuration files rather 
than command-line arguments, a different approach was required than what
 Bio.Application enables. We therefore provide wrappers for these 
programs in a separate sub-module, Bio.Phylo.PAML.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 each of the three analysis programs, the wrapper defines a 
corresponding class that the user instantiates to store the 
configuration state. The programs’ standard configuration options are 
set through the <span xmlns="" class="EmphasisFontCategoryNonProportional">set_options</span> method and are automatically written to a configuration file when the <span xmlns="" class="EmphasisFontCategoryNonProportional">run</span> method is invoked or through the <span xmlns="" class="EmphasisFontCategoryNonProportional">write_ctl_file</span> method. Existing configuration files may be parsed via the method <span xmlns="" class="EmphasisFontCategoryNonProportional">read_ctl_file</span>. Finally, each module of Bio.Phylo.PAML provides a <span xmlns="" class="EmphasisFontCategoryNonProportional">read</span>
 method, which parses the output files of the respective programs. The 
results of an analysis are stored hierarchically in a set of nested 
dictionary objects, allowing quick access by keywords.</p>
                </section>
              </section><section id="Sec9" class="Section2 RenderAsSection2">
                <h3 class="Heading">Biopython integration</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Bio.Phylo
 integrates cleanly with other parts of the Biopython toolkit. We reuse 
existing Biopython API conventions, including classes for exceptions and
 warnings, as well as the packaging and installation mechanisms and the 
testing framework. As mentioned above, the wrappers for running external
 programs use a common Biopython framework, Bio.Application. NEXUS and 
Newick support were obtained through a refactoring of the Bio.Nexus 
module, originally written by Frank Kauff and Cymon J. Cox [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Where
 appropriate, classes in the PhyloXML module support methods for 
conversion to and from instances of the general-purpose Biopython 
classes for molecular sequences, annotated sequence regions and multiple
 sequence alignments. For example, phyloXML defines a ProteinDomain 
element type which represents a functional domain within a protein 
sequence that appears in the tree; our implementation of the 
ProteinDomain class includes a <span xmlns="" class="EmphasisFontCategoryNonProportional">to_seqfeature</span>
 method to convert a ProteinDomain instance to an instance of the 
Biopython class SeqFeature, a generalized representation of an annotated
 sequence region. These SeqFeature objects could then be used with the 
Biopython’s GenomeDiagram module [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]
 to create a diagram of the protein domain architecture of each sequence
 appearing in the phyloXML tree. The ProteinDomain class also provides a
 complementary method, <span xmlns="" class="EmphasisFontCategoryNonProportional">from_seqfeature</span>,
 which could be used to add domain architecture annotations to the 
sequences in a phyloXML record, given the corresponding GenBank files.</p>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Validation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A
 complete suite of unit tests to verify correct functioning of each 
module is included with the Biopython distribution. This includes the 
round-trip parsing and serialization of example files in each supported 
file format; conversion between formats; proper construction, behavior 
and serialization of all phyloXML element types; methods for tree 
inspection, traversal and manipulation; and succesful loading of 
optional third-party libraries, if available.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Use of Bio.Phylo in published studies</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Since
 the Bio.Phylo module has been made available to the community 
throughout its development, researchers have had the opportunity to use 
it in studies that have since been published.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A
 recent study of microbial phylogenomics used Bio.Phylo for large-scale 
processing of microbial gene trees, generating permutations of tree 
topologies by rerooting over 100,000 gene trees at each internal and 
external node [<span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]. Additional file <span class="InternalRef"><a href="#MOESM1">1</a></span> shows a simplified example of how such rerooting could be performed.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 another study of the eukaryotic protein kinase superfamily in the 
protozoan phylum Apicomplexa, Bio.Phylo was used to identify putative 
lineage-specific ortholog groups by selecting clades with significant 
bootstrap support from gene trees [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>].
 The corresponding sequences were then automatically extracted from a 
matching FASTA file, using Biopython’s SeqIO module, for further 
analysis in a computational pipeline.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Performance</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Despite
 the stark simplification of the underlying data structures, the 
majority of the built-in tree methods have a run time that is 
theoretically linear or constant in proportion to the number of nodes in
 the tree. The tree traversal methods <span xmlns="" class="EmphasisFontCategoryNonProportional">find_clades</span> and <span xmlns="" class="EmphasisFontCategoryNonProportional">find_elements</span>
 are Python generator functions which evaluate and emit values 
incrementally, as needed by the caller; this “lazy evaluation” approach 
uses computer memory more efficiently and avoids performing more 
computation than necessary; for example, the method <span xmlns="" class="EmphasisFontCategoryNonProportional">find_any</span>
 uses this property to stop traversal after finding the first element 
matching the user’s query and thus avoid traversing the remainder of the
 tree structure or evaluating any further comparisons.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We timed several benchmark operations on large tree files to evaluate the speed of Bio.Phylo on several different tasks (Table <span class="InternalRef"><a href="#Tab2">2</a></span>; Additional file <span class="InternalRef"><a href="#MOESM2">2</a></span>). On more modestly sized trees and input files, these operations typically complete in a small fraction of a second.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">
                          <strong class="EmphasisTypeBold">Performance</strong>
                        </p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col><col><col><col><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Task</p></th><th><p xmlns="" class="SimplePara">Input tree</p></th><th><p xmlns="" class="SimplePara">Python 2.7</p></th><th><p xmlns="" class="SimplePara">Python 3.2</p></th><th><p xmlns="" class="SimplePara">PyPy 1.9</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Read a very large Newick tree</p></td><td><p xmlns="" class="SimplePara">Smith 2011 angiosperm supertree</p></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">(55473 terminal nodes) [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]</p></td><td><p xmlns="" class="SimplePara">17.45</p></td><td><p xmlns="" class="SimplePara">16.85</p></td><td><p xmlns="" class="SimplePara">1.214</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Read the same large tree in phyloXML</p></td><td><p xmlns="" class="SimplePara">Smith 2011, converted to phyloXML with</p></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara"><span class="EmphasisFontCategoryNonProportional">phylo_converter</span> (<span xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://phylosoft.org/" target="_blank" class="ext-link"><span class="RefSource">http://phylosoft.org/</span></a></span>)</p></td><td><p xmlns="" class="SimplePara">3.805</p></td><td><p xmlns="" class="SimplePara">4.318</p></td><td><p xmlns="" class="SimplePara">3.937</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Write the same large tree as Newick</p></td><td><p xmlns="" class="SimplePara">Smith 2011</p></td><td><p xmlns="" class="SimplePara">0.5238</p></td><td><p xmlns="" class="SimplePara">0.7704</p></td><td><p xmlns="" class="SimplePara">0.4378</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Write the same large tree as phyloXML</p></td><td><p xmlns="" class="SimplePara">Smith 2011</p></td><td><p xmlns="" class="SimplePara">10.39</p></td><td><p xmlns="" class="SimplePara">10.85</p></td><td><p xmlns="" class="SimplePara">24.17</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Read a medium-sized Newick tree</p></td><td><p xmlns="" class="SimplePara">Davies 2004 angiosperm supertree</p></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">(440 terminal nodes) [<span xmlns="http://www.w3.org/1999/xhtml" class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]</p></td><td><p xmlns="" class="SimplePara">0.1097</p></td><td><p xmlns="" class="SimplePara">0.1087</p></td><td><p xmlns="" class="SimplePara">0.007312</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Parse many Newick trees</p></td><td><p xmlns="" class="SimplePara">Davies 2004, copies rerooted at</p></td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">each node (816 trees)</p></td><td><p xmlns="" class="SimplePara">84.91</p></td><td><p xmlns="" class="SimplePara">84.29</p></td><td><p xmlns="" class="SimplePara">6.812</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Reroot at each node</p></td><td><p xmlns="" class="SimplePara">Davies 2004</p></td><td><p xmlns="" class="SimplePara">1.347</p></td><td><p xmlns="" class="SimplePara">1.167</p></td><td><p xmlns="" class="SimplePara">0.3450</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Collapse all splits with bootstrap values less than 50</p></td><td><p xmlns="" class="SimplePara">Davies 2004</p></td><td><p xmlns="" class="SimplePara">2.266</p></td><td><p xmlns="" class="SimplePara">2.312</p></td><td><p xmlns="" class="SimplePara">2.411</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Total branch length</p></td><td><p xmlns="" class="SimplePara">Davies 2004</p></td><td><p xmlns="" class="SimplePara">0.01322</p></td><td><p xmlns="" class="SimplePara">0.01310</p></td><td><p xmlns="" class="SimplePara">0.01448</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Ladderize the tree</p></td><td><p xmlns="" class="SimplePara">Davies 2004</p></td><td><p xmlns="" class="SimplePara">0.1274</p></td><td><p xmlns="" class="SimplePara">0.1190</p></td><td><p xmlns="" class="SimplePara">0.1127</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Count terminal nodes</p></td><td><p xmlns="" class="SimplePara">Davies 2004</p></td><td><p xmlns="" class="SimplePara">0.006838</p></td><td><p xmlns="" class="SimplePara">0.006323</p></td><td><p xmlns="" class="SimplePara">0.005914</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Performance of 
Bio.Phylo functions and tree methods under different Python versions on 
several benchmark tasks. Reported execution times are the median of 101 
replications of each task, in seconds (Additional file <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#MOESM2">2</a></span>).
 Benchmarks were evaluated with Python versions 2.7.3 and 3.2.3 and PyPy
 version 1.9 on an Intel Xeon E5405 2.00 GHz processor with 8 GB memory,
 running under 64-bit Ubuntu Linux 12.04 with Biopython 1.60 installed.</p>
                    </div></figure></div>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">Cookbook and additional documentation</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 keeping the Bio.Phylo module general-purpose and simple to begin using,
 we have chosen not to include niche functions, or approaches that are 
still the topic of active research, in the Biopython distribution. 
However, we nonetheless anticipate that other users of the Bio.Phylo 
module will want to use these features, and there is a benefit to 
sharing this code. We resolve this by maintaining an online “cookbook” 
on the Biopython wiki (<span class="ExternalRef"><a href="http://biopython.org/wiki/Phylo_cookbook" target="_blank" class="ext-link"><span class="RefSource">http://biopython.org/wiki/Phylo_cookbook</span></a></span>).
 This cookbook contains working code samples for common usage patterns, 
as well as exporting to object types of other libraries. In particular, 
we provide functions to convert a Bio.Phylo tree object to a distance or
 adjacency matrix using the NumPy module for Python, and to export a 
Bio.Phylo tree to the native tree objects used by the R package Ape [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], via the Rpy2 module, and the Python package PyCogent [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The main Biopython tutorial, included with the Biopython distribution and available online at <span class="ExternalRef"><a href="http://biopython.org/" target="_blank" class="ext-link"><span class="RefSource">http://biopython.org/</span></a></span>, contains a chapter on the Bio.Phylo module describing its use in detail.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec15" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Discussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Discussion">Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Discussion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Bio.Phylo
 organizes phylogenetic trees as a primary data type, filling a 
previously underserved area of data handling within Biopython. The 
module directly implements tree parsing and serialization in three 
standard formats, as well as navigation, visualization and manipulation 
of phylogenetic trees, and conversion of tree data to other data types. 
Other important aspects of phylogenetics, including phylogenetic tree 
reconstruction and analysis of rate variation and ancestral character 
states, exist as optimized stand-alone programs by other authors; RAxML 
and PAML are examples of such programs that are already well accepted by
 the phylogenetics community. Rather than re-implement the functionality
 of these established third-party programs ourselves, we have opted to 
provide wrappers for these, and focus on providing “glue” utilities to 
ease the process of assembling computational workflows that involve 
phylogenetic data.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">By
 building our work into an existing, popular library, we were able to 
take advantage of both the software infrastructure and the community of 
developers and users associated with Biopython. By reusing core objects 
and maintaining common API conventions for file parsing, the resulting 
software package has a familiar feel to new users who have prior 
experience with Biopython or BioPerl. To ensure code correctness and 
minimize errors introduced during additional development, we reuse the 
existing test framework and packaging mechanisms in Biopython. A 
continuous integration server <span class="ExternalRef"><a href="http://testing.open-bio.org/biopython/" target="_blank" class="ext-link"><span class="RefSource">http://testing.open-bio.org/biopython/</span></a></span>
 runs an automated test suite nightly on all supported Python versions, 
operating systems and implementations, including the Jython Java-based 
port of Python.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Our
 work also provides an example of the short- and long-term benefits of 
building a community of developers and users around a shared open-source
 project. By integrating with Biopython from the beginning, we gained 
access to an existing community of developers and users who have 
continually tested the software under a variety of environments and use 
cases, reported bugs, requested new features, and provided new code. A 
notable example of a contribution from the Biopython user community is 
Bio.Phylo.PAML, which began as an independent project, pypaml. We 
successfully integrated the pypaml source code into Biopython, with 
further enhancements based on feedback from Biopython developers and 
users.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 permissive open-source license that governs both Biopython and 
Bio.Phylo allows this code to be reused freely in other software, which 
could help overcome the pervasive problem of incompatible software 
support for widely used file formats.</p><section id="Sec16" class="Section2 RenderAsSection2">
                <h3 class="Heading">Future development</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 future releases we intend to provide support for another recently 
standardized XML-based format, NeXML, an XML-based successor to NEXUS [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 Since the core classes for phylogenies have already been defined and 
implemented in Bio.Phylo, the implementation of NeXML I/O is expected to
 be straightforward.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Another
 area for future growth is the addition of wrappers for other 
third-party applications in Bio.Phylo.Applications. Using the existing 
Bio.Application framework, it is straightforward to add other tools that
 can be run with a standard command-line interface. Other widely used 
programs that require specialized input, notably MrBayes and PhyloBayes,
 may be implemented with an approach similar to Bio.Phylo.PAML, 
providing a module containing both the functions to generate the 
configuration file and to run the program itself with the generated 
configuration.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Smaller
 features may appear first on the Cookbook wiki page, implemented as 
stand-alone functions, and may later be added to the official Biopython 
distribution.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec17" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Bio.Phylo
 meets a growing need in bioinformatics for working with heterogeneous 
types of biological and phylogenetic data. By supporting 
interoperability with multiple file formats and leveraging existing 
Biopython features, this library simplifies the construction of 
phylogenetic workflows and computational pipelines, addressing practical
 issues of data integration and exchange in the bioinformatics 
community.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec18" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Availabilityandrequirements" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Availabilityandrequirements">Availability and requirements</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Availabilityandrequirements" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><strong xmlns="" class="EmphasisTypeBold">Project name:</strong> Biopython<strong xmlns="" class="EmphasisTypeBold">Project home page:</strong><span class="ExternalRef"><a href="http://biopython.org/" target="_blank" class="ext-link"><span class="RefSource">http://biopython.org/</span></a></span><strong xmlns="" class="EmphasisTypeBold">Operating system:</strong> Platform independent<strong xmlns="" class="EmphasisTypeBold">Programming language:</strong> Python<strong xmlns="" class="EmphasisTypeBold">Other requirements:</strong> Python 2.5 or higher; optional libraries matplotlib, Graphviz, NetworkX<strong xmlns="" class="EmphasisTypeBold">License:</strong> Biopython License<strong xmlns="" class="EmphasisTypeBold">Any restrictions to use by non-academics:</strong> None</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We
 acknowledge the role of Google Summer of Code 2009 and the National 
Evolutionary Synthesis Center (NESCent) Phyloinformatics program in 
sponsoring and initiating this project. We also thank the Biopython 
community and Open Bioinformatics Foundation for ongoing advice and 
support. Several individuals made direct contributions to Bio.Phylo: 
Aaron Gallagher, Connor McCoy, Joel Berendzen, Robert Beiko and Tanya 
Golubchik provided bug fixes, test cases, feature requests and example 
code; Frank Kauff wrote most of the tree parsing code in the Bio.Nexus 
module, which Bio.Phylo builds upon, and gave helpful comments on a 
draft of the manuscript; Cymon J. Cox gave advice on the port of 
Bio.Nexus code; and Christian M. Zmasek co-mentored the original Google 
Summer of Code project with Brad A. Chapman, guiding the implementation 
of phyloXML. We also thank the peer reviewers for helpful suggestions on
 this manuscript.</p><p xmlns="" class="SimplePara">BI is supported by a
 FI-DGR from AGAUR (Catalonia, Spain). Harvard Bioinformatics Core 
provided for publication costs and moral support.</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Electronic supplementary material</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_5387_MOESM1_ESM.py">12859_2012_5387_MOESM1_ESM.py</a>
                          <span xmlns="" class="SimplePara">Additional 
file 1: Tree rerooting. Example code to read a tree from a file and 
write copies of the tree rerooted at every internal and external node. 
The method <span class="EmphasisFontCategoryNonProportional">find_clades</span>
 produces an iterator over the tree rather than a new list, in order to 
make more efficient use of memory in general. In this case, because the 
method <span class="EmphasisFontCategoryNonProportional">root_with_outgroup</span> modifies the tree in-place, which could change the ordering of nodes during traversal, <span class="EmphasisFontCategoryNonProportional">list</span> is used to create an unchanging copy of references to the original nodes in the tree. (PY 359 bytes)</span>
                        </div></div>
                  </div>
                  <div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_5387_MOESM2_ESM.py">12859_2012_5387_MOESM2_ESM.py</a>
                          <span xmlns="" class="SimplePara">Additional file 2: Performance benchmark script. Python script to time the execution of the benchmark tasks shown in Table <span xmlns="http://www.w3.org/1999/xhtml" class="InternalRef"><a href="#Tab2">2</a></span>. (PY 5 KB)</span>
                        </div></div>
                  </div>
                </div>
              
              
                
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authors’ original submitted files for images.<div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_5387_MOESM3_ESM.pdf">12859_2012_5387_MOESM3_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 1</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-13-209/MediaObjects/12859_2012_5387_MOESM4_ESM.pdf">12859_2012_5387_MOESM4_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 2</span>
                        </div></div>
                  </div></div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Competing interests</h3><p xmlns="" class="SimplePara">The authors declare that they have no competing interests.</p></aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors’ contributions</h3><p xmlns="" class="SimplePara">ET
 wrote the code for Bio.Phylo. BI wrote the code for Bio.Phylo.PAML. BAC
 and PJAC mentored and supervised the development of Bio.Phylo and 
contributed enhancements and bug fixes. ET, BI, PJAC and BAC wrote the 
manuscript. All authors read and approved the final manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Institute of Bioinformatics, University of Georgia, 120 Green Street, Athens, GA&nbsp;30602, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Institute of Evolutionary Biology (CSIC-UPF), CEXS-UPF-PRBB, C/ Doctor Aiguader 88, 08003&nbsp;Barcelona, Spain</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">James Hutton Institute, Invergowrie, Dundee, DD2 5DA, UK</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff4"><span class="AffiliationNumber">(4)</span><div class="AffiliationText">Harvard School of Public Health Bioinformatics Core, 655 Huntington Ave, Boston, MA&nbsp;02115, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein J: Phylogenies and the comparative method. <em class="EmphasisTypeItalic">American Naturalist</em> 1985, 125: 1–15. 10.1086/284325</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1086/284325"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenies%20and%20the%20comparative%20method&amp;author=J.%20Felsenstein&amp;journal=American%20Naturalist&amp;volume=125&amp;pages=1-15&amp;publication_year=1985&amp;doi=10.1086%2F284325"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vos RA, Caravas J, Hartmann K, Jensen MA, Miller C: Bio:Phylo - phyloinformatic analysis using Perl. <em class="EmphasisTypeItalic">BMC Bioinformatics</em> 2011, 12: 63. 10.1186/1471-2105-12-63</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3056726"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-12-63"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21352572"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bio%3APhylo%20-%20phyloinformatic%20analysis%20using%20Perl&amp;author=RA.%20Vos&amp;author=J.%20Caravas&amp;author=K.%20Hartmann&amp;author=MA.%20Jensen&amp;author=C.%20Miller&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=63&amp;publication_year=2011&amp;doi=10.1186%2F1471-2105-12-63"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Leebens-Mack
 J, Vision T, Brenner E, Bowers JE, Cannon S, Clement MJ, Cunningham CW,
 DePamphilis C, DeSalle R, Doyle JJ, Eisen Ja, Gu X, Harshman J, Jansen 
RK, Kellogg Ea, Koonin EV, Mishler BD, Philippe H, Pires JC, Qiu YL, 
Rhee SY, Sjölander K, Soltis DE, Soltis PS, Stevenson DW, Wall K, Warnow
 T, Zmasek C: Taking the first steps towards a standard for reporting on
 phylogenies: Minimum Information About a Phylogenetic Analysis (MIAPA).
 <em class="EmphasisTypeItalic">Omics: J Integr Biol</em> 2006, 10(2):231–237. 10.1089/omi.2006.10.231</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/omi.2006.10.231"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Taking%20the%20first%20steps%20towards%20a%20standard%20for%20reporting%20on%20phylogenies%3A%20Minimum%20Information%20About%20a%20Phylogenetic%20Analysis%20%28MIAPA%29&amp;author=J.%20Leebens-Mack&amp;author=T.%20Vision&amp;author=E.%20Brenner&amp;author=JE.%20Bowers&amp;author=S.%20Cannon&amp;author=MJ.%20Clement&amp;author=CW.%20Cunningham&amp;author=C.%20DePamphilis&amp;author=R.%20DeSalle&amp;author=JJ.%20Doyle&amp;author=Ja.%20Eisen&amp;author=X.%20Gu&amp;author=J.%20Harshman&amp;author=RK.%20Jansen&amp;author=Ea.%20Kellogg&amp;author=EV.%20Koonin&amp;author=BD.%20Mishler&amp;author=H.%20Philippe&amp;author=JC.%20Pires&amp;author=YL.%20Qiu&amp;author=SY.%20Rhee&amp;author=K.%20Sj%C3%B6lander&amp;author=DE.%20Soltis&amp;author=PS.%20Soltis&amp;author=DW.%20Stevenson&amp;author=K.%20Wall&amp;author=T.%20Warnow&amp;author=C.%20Zmasek&amp;journal=Omics%3A%20J%20Integr%20Biol&amp;volume=10&amp;issue=2&amp;pages=231-237&amp;publication_year=2006&amp;doi=10.1089%2Fomi.2006.10.231"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Prosdocimi F, Chisham B, Pontelli E, Thompson JD, Stoltzfus A: Initial implementation of a comparative data analysis ontology. <em class="EmphasisTypeItalic">Evolutionary Bioinf</em> 2009, 5: 47–66.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Initial%20implementation%20of%20a%20comparative%20data%20analysis%20ontology&amp;author=F.%20Prosdocimi&amp;author=B.%20Chisham&amp;author=E.%20Pontelli&amp;author=JD.%20Thompson&amp;author=A.%20Stoltzfus&amp;journal=Evolutionary%20Bioinf&amp;volume=5&amp;pages=47-66&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Han MV, Zmasek CM: phyloXML: XML for evolutionary biology and comparative genomics. <em class="EmphasisTypeItalic">BMC Bioinformatics</em> 2009, 10: 356. 10.1186/1471-2105-10-356</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2774328"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-10-356"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19860910"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=phyloXML%3A%20XML%20for%20evolutionary%20biology%20and%20comparative%20genomics&amp;author=MV.%20Han&amp;author=CM.%20Zmasek&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=356&amp;publication_year=2009&amp;doi=10.1186%2F1471-2105-10-356"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vos
 Ra, Balhoff JP, Caravas Ja, Holder MT, Lapp H, Maddison WP, Midford PE,
 Priyam A, Sukumaran J, Xia X, Stoltzfus A: NeXML: rich, extensible, and
 verifiable representation of comparative data and metadata. <em class="EmphasisTypeItalic">Syst Biol</em> 2012, 61(4):675–689. 10.1093/sysbio/sys025</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3376374"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/sys025"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22357728"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NeXML%3A%20rich%2C%20extensible%2C%20and%20verifiable%20representation%20of%20comparative%20data%20and%20metadata&amp;author=Ra.%20Vos&amp;author=JP.%20Balhoff&amp;author=Ja.%20Caravas&amp;author=MT.%20Holder&amp;author=H.%20Lapp&amp;author=WP.%20Maddison&amp;author=PE.%20Midford&amp;author=A.%20Priyam&amp;author=J.%20Sukumaran&amp;author=X.%20Xia&amp;author=A.%20Stoltzfus&amp;journal=Syst%20Biol&amp;volume=61&amp;issue=4&amp;pages=675-689&amp;publication_year=2012&amp;doi=10.1093%2Fsysbio%2Fsys025"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Paradis
 E, Bolker B, Claude J, Cuong HS, Desper R, Dutheil J, Gascuel O, Heibl 
C, Lawson D, Lefort V, Lemon J, Noel Y, Nylander J, Opgen-rhein R, 
Vienne DD: Package ‘ape’. 2010.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://ape.mpl.ird.fr/" target="_blank" class="ext-link"><span class="RefSource">http://ape.mpl.ird.fr/</span></a></span> []</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=E.%20Paradis&amp;author=B.%20Bolker&amp;author=J.%20Claude&amp;author=HS.%20Cuong&amp;author=R.%20Desper&amp;author=J.%20Dutheil&amp;author=O.%20Gascuel&amp;author=C.%20Heibl&amp;author=D.%20Lawson&amp;author=V.%20Lefort&amp;author=J.%20Lemon&amp;author=Y.%20Noel&amp;author=J.%20Nylander&amp;author=R.%20Opgen-rhein&amp;author=DD.%20Vienne&amp;publication_year=2010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stajich
 JE, Block D, Boulez K, Brenner SE, Chervitz Sa, Dagdigian C, Fuellen G,
 Gilbert JGR, Korf I, Lapp H, Lehväslaiho H, Matsalla C, Mungall CJ, 
Osborne BI, Pocock MR, Schattner P, Senger M, Stein LD, Stupka E, 
Wilkinson MD, Birney E: The Bioperl toolkit: Perl modules for the life 
sciences. <em class="EmphasisTypeItalic">Genome Res</em> 2002, 12(10):1611–1618. 10.1101/gr.361602</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC187536"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.361602"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12368254"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Bioperl%20toolkit%3A%20Perl%20modules%20for%20the%20life%20sciences&amp;author=JE.%20Stajich&amp;author=D.%20Block&amp;author=K.%20Boulez&amp;author=SE.%20Brenner&amp;author=Sa.%20Chervitz&amp;author=C.%20Dagdigian&amp;author=G.%20Fuellen&amp;author=JGR.%20Gilbert&amp;author=I.%20Korf&amp;author=H.%20Lapp&amp;author=H.%20Lehv%C3%A4slaiho&amp;author=C.%20Matsalla&amp;author=CJ.%20Mungall&amp;author=BI.%20Osborne&amp;author=MR.%20Pocock&amp;author=P.%20Schattner&amp;author=M.%20Senger&amp;author=LD.%20Stein&amp;author=E.%20Stupka&amp;author=MD.%20Wilkinson&amp;author=E.%20Birney&amp;journal=Genome%20Res&amp;volume=12&amp;issue=10&amp;pages=1611-1618&amp;publication_year=2002&amp;doi=10.1101%2Fgr.361602"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maddison W, Maddison D: Mesquite: A modular system for evolutionary analysis. 2011.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://mesquiteproject.org/" target="_blank" class="ext-link"><span class="RefSource">http://mesquiteproject.org</span></a></span> []</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=W.%20Maddison&amp;author=D.%20Maddison&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Cock
 PJA, Antao T, Chang JT, Chapman BA, Cox CJ, Dalke A, Friedberg I, 
Hamelryck T, Kauff F, Wilczynski B, de Hoon MJL: Biopython: freely 
available Python tools for computational molecular biology and 
bioinformatics. <em class="EmphasisTypeItalic">Bioinformatics</em> 2009, 25(11):1422–1423. 10.1093/bioinformatics/btp163</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2682512"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btp163"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19304878"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Biopython%3A%20freely%20available%20Python%20tools%20for%20computational%20molecular%20biology%20and%20bioinformatics&amp;author=PJA.%20Cock&amp;author=T.%20Antao&amp;author=JT.%20Chang&amp;author=BA.%20Chapman&amp;author=CJ.%20Cox&amp;author=A.%20Dalke&amp;author=I.%20Friedberg&amp;author=T.%20Hamelryck&amp;author=F.%20Kauff&amp;author=B.%20Wilczynski&amp;author=MJL.%20de%20Hoon&amp;journal=Bioinformatics&amp;volume=25&amp;issue=11&amp;pages=1422-1423&amp;publication_year=2009&amp;doi=10.1093%2Fbioinformatics%2Fbtp163"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Holland
 RCG, Down TA, Pocock M, Prlić A, Huen D, James K, Foisy S, Dräger A, 
Yates A, Heuer M, Schreiber MJ: BioJava: an open-source framework for 
bioinformatics. <em class="EmphasisTypeItalic">Bioinformatics</em> 2008, 24(18):2096–2097. 10.1093/bioinformatics/btn397</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2530884"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btn397"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18689808"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BioJava%3A%20an%20open-source%20framework%20for%20bioinformatics&amp;author=RCG.%20Holland&amp;author=TA.%20Down&amp;author=M.%20Pocock&amp;author=A.%20Prli%C4%87&amp;author=D.%20Huen&amp;author=K.%20James&amp;author=S.%20Foisy&amp;author=A.%20Dr%C3%A4ger&amp;author=A.%20Yates&amp;author=M.%20Heuer&amp;author=MJ.%20Schreiber&amp;journal=Bioinformatics&amp;volume=24&amp;issue=18&amp;pages=2096-2097&amp;publication_year=2008&amp;doi=10.1093%2Fbioinformatics%2Fbtn397"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goto N, Prins P, Nakao M, Bonnal R, Aerts J, Katayama T: BioRuby: bioinformatics software for the Ruby programming language. <em class="EmphasisTypeItalic">Bioinformatics</em> 2010, 26(20):2617–2619. 10.1093/bioinformatics/btq475</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2951089"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btq475"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20739307"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BioRuby%3A%20bioinformatics%20software%20for%20the%20Ruby%20programming%20language&amp;author=N.%20Goto&amp;author=P.%20Prins&amp;author=M.%20Nakao&amp;author=R.%20Bonnal&amp;author=J.%20Aerts&amp;author=T.%20Katayama&amp;journal=Bioinformatics&amp;volume=26&amp;issue=20&amp;pages=2617-2619&amp;publication_year=2010&amp;doi=10.1093%2Fbioinformatics%2Fbtq475"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Prins P, Contributors: BioLib. 2008.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://biolib.open-bio.org/" target="_blank" class="ext-link"><span class="RefSource">http://biolib.open-bio.org/</span></a></span> []</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=P.%20Prins&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lapp H, Contributors: BioSQL. 2003.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.biosql.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.biosql.org</span></a></span> []</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=H.%20Lapp&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maglott D, Ostell J, Pruitt KD, Tatusova T: Entrez Gene: gene-centered information at NCBI. <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2005, 33(Database issue):D54-D58.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC539985"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gki031"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15608257"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Entrez%20Gene%3A%20gene-centered%20information%20at%20NCBI&amp;author=D.%20Maglott&amp;author=J.%20Ostell&amp;author=KD.%20Pruitt&amp;author=T.%20Tatusova&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;issue=Database%20issue&amp;pages=D54-D58&amp;publication_year=2005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Altschul
 SF, Madden TL, Schäffer aa, Zhang J, Zhang Z, Miller W, Lipman DJ: 
Gapped BLAST and PSI-BLAST: a new generation of protein database search 
programs. <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1997, 25(17):3389–3402. 10.1093/nar/25.17.3389</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC146917"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/25.17.3389"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9254694"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=SF.%20Altschul&amp;author=TL.%20Madden&amp;author=aa.%20Sch%C3%A4ffer&amp;author=J.%20Zhang&amp;author=Z.%20Zhang&amp;author=W.%20Miller&amp;author=DJ.%20Lipman&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;issue=17&amp;pages=3389-3402&amp;publication_year=1997&amp;doi=10.1093%2Fnar%2F25.17.3389"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Camacho C, Coulouris G, Avagyan V, Ma N, Papadopoulos J, Bealer K, Madden TL: BLAST+: architecture and applications. <em class="EmphasisTypeItalic">BMC Bioinformatics</em> 2009, 10: 421. 10.1186/1471-2105-10-421</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2803857"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-10-421"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20003500"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BLAST%2B%3A%20architecture%20and%20applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;author=N.%20Ma&amp;author=J.%20Papadopoulos&amp;author=K.%20Bealer&amp;author=TL.%20Madden&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=421&amp;publication_year=2009&amp;doi=10.1186%2F1471-2105-10-421"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Oliphant TE: Python for Scientific Computing. <em class="EmphasisTypeItalic">Comput Sci Eng</em> 2007, 9(3):10–20.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1109/MCSE.2007.58"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Python%20for%20Scientific%20Computing&amp;author=TE.%20Oliphant&amp;journal=Comput%20Sci%20Eng&amp;volume=9&amp;issue=3&amp;pages=10-20&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Knight
 R, Maxwell P, Birmingham A, Carnes J, Caporaso JG, Easton BC, Eaton M, 
Hamady M, Lindsay H, Liu Z, Lozupone C, McDonald D, Robeson M, Sammut R,
 Smit S, Wakefield MJ, Widmann J, Wikman S, Wilson S, Ying H, Huttley 
GA: PyCogent: a toolkit for making sense from sequence. <em class="EmphasisTypeItalic">Genome Biol</em> 2007, 8(8):R171. 10.1186/gb-2007-8-8-r171</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2375001"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2007-8-8-r171"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17708774"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PyCogent%3A%20a%20toolkit%20for%20making%20sense%20from%20sequence&amp;author=R.%20Knight&amp;author=P.%20Maxwell&amp;author=A.%20Birmingham&amp;author=J.%20Carnes&amp;author=JG.%20Caporaso&amp;author=BC.%20Easton&amp;author=M.%20Eaton&amp;author=M.%20Hamady&amp;author=H.%20Lindsay&amp;author=Z.%20Liu&amp;author=C.%20Lozupone&amp;author=D.%20McDonald&amp;author=M.%20Robeson&amp;author=R.%20Sammut&amp;author=S.%20Smit&amp;author=MJ.%20Wakefield&amp;author=J.%20Widmann&amp;author=S.%20Wikman&amp;author=S.%20Wilson&amp;author=H.%20Ying&amp;author=GA.%20Huttley&amp;journal=Genome%20Biol&amp;volume=8&amp;issue=8&amp;pages=R171&amp;publication_year=2007&amp;doi=10.1186%2Fgb-2007-8-8-r171"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sukumaran J, Holder MT: DendroPy: a Python library for phylogenetic computing. <em class="EmphasisTypeItalic">Bioinformatics</em> 2010, 26(12):1569–1571. 10.1093/bioinformatics/btq228</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btq228"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20421198"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=DendroPy%3A%20a%20Python%20library%20for%20phylogenetic%20computing&amp;author=J.%20Sukumaran&amp;author=MT.%20Holder&amp;journal=Bioinformatics&amp;volume=26&amp;issue=12&amp;pages=1569-1571&amp;publication_year=2010&amp;doi=10.1093%2Fbioinformatics%2Fbtq228"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Huerta-Cepas J, Dopazo J, Gabaldón T: ETE: a python Environment for Tree Exploration. <em class="EmphasisTypeItalic">BMC Bioinformatics</em> 2010, 11: 24. 10.1186/1471-2105-11-24</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2820433"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2105-11-24"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20070885"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ETE%3A%20a%20python%20Environment%20for%20Tree%20Exploration&amp;author=J.%20Huerta-Cepas&amp;author=J.%20Dopazo&amp;author=T.%20Gabald%C3%B3n&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=24&amp;publication_year=2010&amp;doi=10.1186%2F1471-2105-11-24"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Foster PG: <em class="EmphasisTypeItalic">p4: A Python package for phylogenetics</em>. 2003. [<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://code.google.com/p/p4-phylogenetics/" target="_blank" class="ext-link"><span class="RefSource">http://code.google.com/p/p4-phylogenetics/</span></a></span>] []</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=p4%3A%20A%20Python%20package%20for%20phylogenetics&amp;author=PG.%20Foster&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Maddison DR, Swofford DL, Maddison WP: NEXUS: an extensible file format for systematic information. <em class="EmphasisTypeItalic">Syst Biol</em> 1997, 46(4):590–621. 10.1093/sysbio/46.4.590</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/46.4.590"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11975335"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NEXUS%3A%20an%20extensible%20file%20format%20for%20systematic%20information&amp;author=DR.%20Maddison&amp;author=DL.%20Swofford&amp;author=WP.%20Maddison&amp;journal=Syst%20Biol&amp;volume=46&amp;issue=4&amp;pages=590-621&amp;publication_year=1997&amp;doi=10.1093%2Fsysbio%2F46.4.590"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Archie J, Day WH, Maddison W, Meacham C, Rohlf FJ, Swofford D, Felsenstein J: <em class="EmphasisTypeItalic">The Newick tree format</em>. 1986. [<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://evolution.genetics.washington.edu/phylip/newicktree.html" target="_blank" class="ext-link"><span class="RefSource">http://evolution.genetics.washington.edu/phylip/newicktree.html</span></a></span>] []</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Newick%20tree%20format&amp;author=J.%20Archie&amp;author=WH.%20Day&amp;author=W.%20Maddison&amp;author=C.%20Meacham&amp;author=FJ.%20Rohlf&amp;author=D.%20Swofford&amp;author=J.%20Felsenstein&amp;publication_year=1986"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gansner E: An open graph visualization system and its applications to software engineering. <em class="EmphasisTypeItalic">Software Pract Experience</em> 2000, 00(S1):1–5.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=An%20open%20graph%20visualization%20system%20and%20its%20applications%20to%20software%20engineering&amp;author=E.%20Gansner&amp;journal=Software%20Pract%20Experience&amp;volume=00&amp;issue=S1&amp;pages=1-5&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hagberg AA, Schult DA, Swart PJ: Exploring network structure, dynamics, and function using NetworkX. In <em class="EmphasisTypeItalic">Proceedings of the 7th Python in Science Conference (SciPy2008), SciPy</em> Edited by: Varoquaux G, Vaught T, Millman J. 2008, -11. Pasadena, CA USA–15 Pasadena, CA USA–15</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Exploring%20network%20structure%2C%20dynamics%2C%20and%20function%20using%20NetworkX&amp;author=AA.%20Hagberg&amp;author=DA.%20Schult&amp;author=PJ.%20Swart&amp;pages=11&amp;publication_year=2008"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zmasek CM, Zhang Q, Ye Y, Godzik A: Surprising complexity of the ancestral apoptosis network. <em class="EmphasisTypeItalic">Genome Biol</em> 2007, 8(10):R226. 10.1186/gb-2007-8-10-r226</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2246300"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/gb-2007-8-10-r226"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17958905"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Surprising%20complexity%20of%20the%20ancestral%20apoptosis%20network&amp;author=CM.%20Zmasek&amp;author=Q.%20Zhang&amp;author=Y.%20Ye&amp;author=A.%20Godzik&amp;journal=Genome%20Biol&amp;volume=8&amp;issue=10&amp;pages=R226&amp;publication_year=2007&amp;doi=10.1186%2Fgb-2007-8-10-r226"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hunter JD: Matplotlib: A 2D Graphics Environment. <em class="EmphasisTypeItalic">Comput Sci Eng</em> 2007, 9(3):90–95.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1109/MCSE.2007.55"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Matplotlib%3A%20A%202D%20Graphics%20Environment&amp;author=JD.%20Hunter&amp;journal=Comput%20Sci%20Eng&amp;volume=9&amp;issue=3&amp;pages=90-95&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zmasek CM, Eddy SR: ATV: display and manipulation of annotated phylogenetic trees. <em class="EmphasisTypeItalic">Bioinformatics</em> 2001, 17(4):383–384. 10.1093/bioinformatics/17.4.383</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/17.4.383"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11301314"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=ATV%3A%20display%20and%20manipulation%20of%20annotated%20phylogenetic%20trees&amp;author=CM.%20Zmasek&amp;author=SR.%20Eddy&amp;journal=Bioinformatics&amp;volume=17&amp;issue=4&amp;pages=383-384&amp;publication_year=2001&amp;doi=10.1093%2Fbioinformatics%2F17.4.383"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rice P, Longden I, Bleasby A: EMBOSS: The European Molecular Biology Open Software Suite. <em class="EmphasisTypeItalic">Trends Genet</em> 2000, 16(6):276–277. 10.1016/S0168-9525(00)02024-2</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0168-9525%2800%2902024-2"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10827456"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20The%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;journal=Trends%20Genet&amp;volume=16&amp;issue=6&amp;pages=276-277&amp;publication_year=2000&amp;doi=10.1016%2FS0168-9525%2800%2902024-2"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guindon
 S, Dufayard JF, Lefort V, Anisimova M, Hordijk W, Gascuel O: New 
algorithms and methods to estimate maximum-likelihood phylogenies: 
assessing the performance of PhyML 3.0. <em class="EmphasisTypeItalic">Syst Biol</em> 2010, 59(3):307–321. [<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://www.atgc-montpellier.fr/phyml/" target="_blank" class="ext-link"><span class="RefSource">http://www.atgc-montpellier.fr/phyml/</span></a></span>] [] 10.1093/sysbio/syq010</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/syq010"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525638"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=JF.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;author=O.%20Gascuel&amp;journal=Syst%20Biol&amp;volume=59&amp;issue=3&amp;pages=307-321&amp;publication_year=2010&amp;doi=10.1093%2Fsysbio%2Fsyq010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis A: RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models. <em class="EmphasisTypeItalic">Bioinformatics</em> 2006, 22(21):2688–2690. 10.1093/bioinformatics/btl446</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btl446"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16928733"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A.%20Stamatakis&amp;journal=Bioinformatics&amp;volume=22&amp;issue=21&amp;pages=2688-2690&amp;publication_year=2006&amp;doi=10.1093%2Fbioinformatics%2Fbtl446"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang Z: PAML 4: phylogenetic analysis by maximum likelihood. <em class="EmphasisTypeItalic">Mol Biol Evol</em> 2007, 24(8):1586–1591. 10.1093/molbev/msm088</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msm088"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17483113"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;issue=8&amp;pages=1586-1591&amp;publication_year=2007&amp;doi=10.1093%2Fmolbev%2Fmsm088"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kauff F, Cox C, Lutzoni F: WASABI: an automated sequence processing system for multigene phylogenies. <em class="EmphasisTypeItalic">Systematic Biol</em> 2007, 56(3):523. 10.1080/10635150701395340</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/10635150701395340"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=WASABI%3A%20an%20automated%20sequence%20processing%20system%20for%20multigene%20phylogenies&amp;author=F.%20Kauff&amp;author=C.%20Cox&amp;author=F.%20Lutzoni&amp;journal=Systematic%20Biol&amp;volume=56&amp;issue=3&amp;pages=523&amp;publication_year=2007&amp;doi=10.1080%2F10635150701395340"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pritchard L, White JA, Birch PR, Toth IK: GenomeDiagram: a python package for the visualization of large-scale genomic data. <em class="EmphasisTypeItalic">Bioinformatics</em> 2006, 22(5):616–617. 10.1093/bioinformatics/btk021</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btk021"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16377612"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=GenomeDiagram%3A%20a%20python%20package%20for%20the%20visualization%20of%20large-scale%20genomic%20data&amp;author=L.%20Pritchard&amp;author=JA.%20White&amp;author=PR.%20Birch&amp;author=IK.%20Toth&amp;journal=Bioinformatics&amp;volume=22&amp;issue=5&amp;pages=616-617&amp;publication_year=2006&amp;doi=10.1093%2Fbioinformatics%2Fbtk021"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Beiko RG: Telling the Whole Story in a 10,000-Genome World. <em class="EmphasisTypeItalic">Biol Direct</em> 2011, 6: 34. 10.1186/1745-6150-6-34</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3158115"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1745-6150-6-34"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21714939"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Telling%20the%20Whole%20Story%20in%20a%2010%2C000-Genome%20World&amp;author=RG.%20Beiko&amp;journal=Biol%20Direct&amp;volume=6&amp;pages=34&amp;publication_year=2011&amp;doi=10.1186%2F1745-6150-6-34"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Talevich E, Mirza A, Kannan N: Structural and evolutionary divergence of eukaryotic protein kinases in Apicomplexa. <em class="EmphasisTypeItalic">BMC Evolutionary Biol</em> 2011, 11: 321. 10.1186/1471-2148-11-321</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1186/1471-2148-11-321"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Structural%20and%20evolutionary%20divergence%20of%20eukaryotic%20protein%20kinases%20in%20Apicomplexa&amp;author=E.%20Talevich&amp;author=A.%20Mirza&amp;author=N.%20Kannan&amp;journal=BMC%20Evolutionary%20Biol&amp;volume=11&amp;pages=321&amp;publication_year=2011&amp;doi=10.1186%2F1471-2148-11-321"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 SA, Beaulieu JM, Stamatakis A, Donoghue MJ: Understanding angiosperm 
diversification using small and large phylogenetic trees. <em class="EmphasisTypeItalic">Am J Bot</em> 2011, 98(3):404–414. 10.3732/ajb.1000481</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.3732/ajb.1000481"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21613134"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Understanding%20angiosperm%20diversification%20using%20small%20and%20large%20phylogenetic%20trees&amp;author=SA.%20Smith&amp;author=JM.%20Beaulieu&amp;author=A.%20Stamatakis&amp;author=MJ.%20Donoghue&amp;journal=Am%20J%20Bot&amp;volume=98&amp;issue=3&amp;pages=404-414&amp;publication_year=2011&amp;doi=10.3732%2Fajb.1000481"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Davies
 TJ, Barraclough TG, Chase MW, Soltis PS, Soltis DE, Savolainen V: 
Darwin’s abominable mystery: Insights from a supertree of the 
angiosperms. <em class="EmphasisTypeItalic">Proc Natl Acad Sci USA</em> 2004, 101(7):1904–1909. 10.1073/pnas.0308127100</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC357025"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0308127100"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14766971"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Darwin%E2%80%99s%20abominable%20mystery%3A%20Insights%20from%20a%20supertree%20of%20the%20angiosperms&amp;author=TJ.%20Davies&amp;author=TG.%20Barraclough&amp;author=MW.%20Chase&amp;author=PS.%20Soltis&amp;author=DE.%20Soltis&amp;author=V.%20Savolainen&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=101&amp;issue=7&amp;pages=1904-1909&amp;publication_year=2004&amp;doi=10.1073%2Fpnas.0308127100"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#TalevichetallicenseeBioMedCentralLtd" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="TalevichetallicenseeBioMedCentralLtd">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="TalevichetallicenseeBioMedCentralLtd"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Talevich et al.; licensee BioMed Central Ltd.&nbsp;2012<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published
 under license to BioMed Central Ltd. This is an Open Access article 
distributed under the terms of the Creative Commons Attribution License (<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-page-layout__side u-text-sm u-margin-top-md-at-lt-md">
                    <aside>
                        <div class="c-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-13-209" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/1471-2105-13-209">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/1471-2105-13-209" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-13-209?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
    <div id="articleSections" class="c-separator">
        <h3 class="p-section-title">Section</h3>
        <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
            
                <li class="c-list-group__item">
                    <a href="https://bmcbioinformatics.biomedcentral.com/articles/sections/comparative-genomics" data-track="click" data-track-category="Article Page" data-track-action="Click section link" data-track-label="10.1186/1471-2105-13-209">Comparative genomics</a>
                </li>
            
        </ul>
    </div>
    
 
                        
    
 
                        <div class="c-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    12888
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 43 <a href="http://citations.springer.com/item?doi=10.1186/1471-2105-13-209" rel="noopener" target="_blank">more information</a>
            </li>
        
                                
    <li class="c-list-group__item">
        <p>Altmetric Attention Score: 19</p>
        <div class="altmetric">
            <a class="altmetric__link" href="http://www.altmetric.com/details.php?citation_id=896626&amp;domain=www.biomedcentral.com" target="_blank" rel="noopener" data-track="click" data-track-category="Article Page" data-track-action="Click Altmetric" data-track-label="10.1186/1471-2105-13-209">
                <img class="altmetric__img" src="a.png" alt="Altmetric Attention Score">
            </a>
        </div>
    </li>

                            </ul>
                        </div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
                <li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-13-209">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-13-209">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-13-209">
                            <use xlink:href="#icon-twitter"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-13-209">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-13-209">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-13-209">
                            <use xlink:href="#icon-facebook"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-13-209">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-13-209">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-13-209">
                            <use xlink:href="#icon-linkedin"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-13-209">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-13-209">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-13-209">
                            <use xlink:href="#icon-weibo"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-13-209">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-13-209">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-13-209">
                            <use xlink:href="#icon-reddit"></use>
                        </svg>
                    </button>
                </li>
            </ul>
        </div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-209/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/1471-2105-13-209">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
 
                        <div class="c-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/1471-2105-13-209" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2F1471-2105-13-209&amp;volume=13&amp;issue=1&amp;title=Bio.Phylo%3A+A+unified+toolkit+for+processing%2C+analyzing+and+visualizing+phylogenetic+trees+in+Biopython&amp;author_name=Eric+Talevich&amp;start_page=1&amp;end_page=9">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Bioinformatics</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1471-2105</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:bmcbioinformatics@biomedcentral.com">bmcbioinformatics@biomedcentral.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1471-2105-13-209.png" alt="">

         
    <footer>
        
            <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" data-test="publisher-footer-menu">
            <div class="c-flex">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://support.biomedcentral.com/support/home">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="https://www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        
        <div class="c-sn-footer">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature" itemprop="logo" role="img">
        <p data-test="copyright"> © 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="https://www.springernature.com/" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

        
    </footer>

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            
            appScript.src = '/static/js/app-bundle-62e4f009a3.js';
            
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script><script src="app-bundle-62e4f009a3.js"></script>






    


<iframe src="https://event-tracker.springernature.com" title="EventTracker" id="event-tracker" style="display:none; visibility:hidden" aria-hidden="true" tabindex="-1" width="0" height="0"></iframe><script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(27);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var d=[],e="",g=[],h=a.split(";");for(a=0;a<h.length;++a){void 0!==k&&(e=k);var b=h[a].split("\x3d");var k=b[0];k!==e&&(0<e.length&&0<d.length&&g.push([e,d]),d=[]);if(2===b.length&&""!==b[0]&&""!==b[1]){var c=b[1].split(",");for(b=0;b<c.length;++b)d.push(c[b])}}0<k.length&&0<d.length&&g.push([k,d]);return g}
function splitSizes(a){var d=[];if(null!==a){var e=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<e.length;++a){var g=e[a].split("x");var h=parseInt(g[0],10);var b=parseInt(g[1],10);2===g.length&&!isNaN(h)&&!isNaN(b)&&0<=h&&0<=b&&d.push([h,b])}}return d}function debounce(a,d){var e=null,g=null;return function(){var h=this,b=Number(new Date),k=arguments;e&&b<e+d?(clearTimeout(g),g=setTimeout(function(){e=b;a.apply(h,k)},d)):(e=b,a.apply(h,k))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,d){function e(f){var a="test";a=a.replace(/[\[\]]/g,"\\$\x26");a=new RegExp("[?\x26]"+a+"(\x3d([^\x26#]*)|\x26|#|$)");a=(a=a.exec(window.location.href))?a[2]?decodeURIComponent(a[2].replace(/\+/g," ")):"":null;a=a&&"ads"===a?"adtype\x3dtest;":"";f=f.getAttribute("data-gpt-targeting");var c=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var d=isLoggedIn();a&&-1===f.indexOf(a)&&(f+=a);c&&
(f+=c);return f+d+b}function g(a,c){for(var f=a.length,b=[];f--;)c(a[f],f)&&(b.push(a[f].slot),a.splice(f,1));b.length&&googletag.pubads().refresh(b);return a}function h(f){var c=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return g(f,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})}var b,k=getAdContainers(),c=[];d&&(b=d.dfppKeyValues?d.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var m=
0;k[m];++m){var l=k[m];splitKeys(e(l));c.push({divId:l.getAttribute("id"),adUnitPath:l.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(l.getAttribute("data-gpt-sizes")),targeting:splitKeys(e(l))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(31)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());
for(var b=0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(p){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=h(c)});var n=debounce(function(){googletag.cmd.push(function(){c=h(c);c.length||removeScrollEvent(n)})},250);addScrollEvent(n)})(window,
window.Krux);</script><script type="text/javascript" id="ga-data-track-event-listeners">var setupHandler=google_tag_manager["GTM-TDGJHK"].macro(39);setupHandler("click");setupHandler("submit");</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Fbmcbioinformatics.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>