<!DOCTYPE html>
<html class="js" style="--font-family-sans: Europa; --font-family-serif: Noto Serif;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <title>Algorithms, data structures, and numerics for 
likelihood-based phylogenetic inference of huge trees | BMC 
Bioinformatics | Full Text</title>

        

    <meta name="citation_abstract" content="The rapid accumulation of molecular sequence data, driven by novel wet-lab sequencing technologies, poses new challenges for large-scale maximum likelihood-based phylogenetic analyses on trees with more than 30,000 taxa and several genes. The three main computational challenges are: numerical stability, the scalability of search algorithms, and the high memory requirements for computing the likelihood. We introduce methods for solving these three key problems and provide respective proof-of-concept implementations in RAxML. The mechanisms presented here are not RAxML-specific and can thus be applied to any likelihood-based (Bayesian or maximum likelihood) tree inference program. We develop a new search strategy that can reduce the time required for tree inferences by more than 50% while yielding equally good trees (in the statistical sense) for well-chosen starting trees. We present an adaptation of the Subtree Equality Vector technique for phylogenomic datasets with missing data (already available in RAxML v728) that can reduce execution times and memory requirements by up to 50%. Finally, we discuss issues pertaining to the numerical stability of the Γ model of rate heterogeneity on very large trees and argue in favor of rate heterogeneity models that use a single rate or rate category for each site to resolve these problems. We address three major issues pertaining to large scale tree reconstruction under maximum likelihood and propose respective solutions. Respective proof-of-concept/production-level implementations of our ideas are made available as open-source code.">

    <meta name="description" content="The rapid accumulation of molecular sequence data, driven by novel wet-lab sequencing technologies, poses new challenges for large-scale maximum likelihood-based phylogenetic analyses on trees with more than 30,000 taxa and several genes. The three main computational challenges are: numerical stability, the scalability of search algorithms, and the high memory requirements for computing the likelihood. We introduce methods for solving these three key problems and provide respective proof-of-concept implementations in RAxML. The mechanisms presented here are not RAxML-specific and can thus be applied to any likelihood-based (Bayesian or maximum likelihood) tree inference program. We develop a new search strategy that can reduce the time required for tree inferences by more than 50% while yielding equally good trees (in the statistical sense) for well-chosen starting trees. We present an adaptation of the Subtree Equality Vector technique for phylogenomic datasets with missing data (already available in RAxML v728) that can reduce execution times and memory requirements by up to 50%. Finally, we discuss issues pertaining to the numerical stability of the Γ model of rate heterogeneity on very large trees and argue in favor of rate heterogeneity models that use a single rate or rate category for each site to resolve these problems. We address three major issues pertaining to large scale tree reconstruction under maximum likelihood and propose respective solutions. Respective proof-of-concept/production-level implementations of our ideas are made available as open-source code.">

    <meta name="journal_id" content="12859">

    <meta name="dc.title" content="Algorithms, data structures, and numerics for likelihood-based phylogenetic inference of huge trees">

    <meta name="dc.source" content="BMC Bioinformatics 2011 12:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2011-12-13">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2011 Izquierdo-Carrasco et al; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="The rapid accumulation of molecular sequence data, driven by novel wet-lab sequencing technologies, poses new challenges for large-scale maximum likelihood-based phylogenetic analyses on trees with more than 30,000 taxa and several genes. The three main computational challenges are: numerical stability, the scalability of search algorithms, and the high memory requirements for computing the likelihood. We introduce methods for solving these three key problems and provide respective proof-of-concept implementations in RAxML. The mechanisms presented here are not RAxML-specific and can thus be applied to any likelihood-based (Bayesian or maximum likelihood) tree inference program. We develop a new search strategy that can reduce the time required for tree inferences by more than 50% while yielding equally good trees (in the statistical sense) for well-chosen starting trees. We present an adaptation of the Subtree Equality Vector technique for phylogenomic datasets with missing data (already available in RAxML v728) that can reduce execution times and memory requirements by up to 50%. Finally, we discuss issues pertaining to the numerical stability of the Γ model of rate heterogeneity on very large trees and argue in favor of rate heterogeneity models that use a single rate or rate category for each site to resolve these problems. We address three major issues pertaining to large scale tree reconstruction under maximum likelihood and propose respective solutions. Respective proof-of-concept/production-level implementations of our ideas are made available as open-source code.">

    <meta name="prism.issn" content="1471-2105">

    <meta name="prism.publicationName" content="BMC Bioinformatics">

    <meta name="prism.publicationDate" content="2011-12-13">

    <meta name="prism.volume" content="12">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="470">

    <meta name="prism.copyright" content="2011 Izquierdo-Carrasco et al; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-470">

    <meta name="prism.doi" content="doi:10.1186/1471-2105-12-470">

    <meta name="citation_pdf_url" content="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-470">

    <meta name="citation_fulltext_html_url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-470">

    <meta name="citation_journal_title" content="BMC Bioinformatics">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2105">

    <meta name="citation_title" content="Algorithms, data structures, and numerics for likelihood-based phylogenetic inference of huge trees">

    <meta name="citation_volume" content="12">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2011/12">

    <meta name="citation_online_date" content="2011/12/13">

    <meta name="citation_firstpage" content="470">

    <meta name="citation_article_type" content="Research article">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/1471-2105-12-470">

    <meta name="DOI" content="10.1186/1471-2105-12-470">

    <meta name="citation_doi" content="10.1186/1471-2105-12-470">

    <meta name="dc.creator" content="Fernando Izquierdo-Carrasco">

    <meta name="dc.creator" content="Stephen A Smith">

    <meta name="dc.creator" content="Alexandros Stamatakis">

    <meta name="dc.subject" content="Bioinformatics">

    <meta name="dc.subject" content="Microarrays">

    <meta name="dc.subject" content="Computational Biology/Bioinformatics">

    <meta name="dc.subject" content="Computer Appl. in Life Sciences">

    <meta name="dc.subject" content="Combinatorial Libraries">

    <meta name="dc.subject" content="Algorithms">

    <meta name="citation_reference" content="citation_journal_title=Genome Research; citation_title=Pyrosequencing Sheds Light on DNA Sequencing; citation_author=M Ronaghi; citation_volume=11; citation_publication_date=2001; citation_pages=3-11; citation_doi=10.1101/gr.11.1.3; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach; citation_author=J Felsenstein; citation_volume=17; citation_publication_date=1981; citation_pages=368-376; citation_doi=10.1007/BF01734359; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=pIQPNNI: parallel reconstruction of large maximum likelihood phylogenies; citation_author=B Minh, L Vinh, A Haeseler, H Schmidt; citation_volume=21; citation_issue=19; citation_publication_date=2005; citation_pages=3794-3796; citation_doi=10.1093/bioinformatics/bti594; citation_id=CR3">

    <meta name="citation_reference" content="citation_title=Genetic Algorithm Approaches for the Phylogenetic Analysis of Large Biological Sequence Datasets under the Maximum Likelihood Criterion; citation_inbook_title=PhD thesis; citation_publication_date=2006; citation_id=CR4; citation_author=D Zwickl">

    <meta name="citation_reference" content="citation_journal_title=Systematic biology; citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0; citation_author=S Guindon, J Dufayard, V Lefort, M Anisimova, W Hordijk, O Gascuel; citation_volume=59; citation_issue=3; citation_publication_date=2010; citation_pages=307; citation_doi=10.1093/sysbio/syq010; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=FastTree 2- Approximately Maximum-Likelihood Trees for Large Alignments; citation_author=M Price, P Dehal, A Arkin; citation_volume=5; citation_issue=3; citation_publication_date=2010; citation_pages=e9490; citation_doi=10.1371/journal.pone.0009490; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models; citation_author=F Ronquist, J Huelsenbeck; citation_volume=19; citation_issue=12; citation_publication_date=2003; citation_pages=1572-1574; citation_doi=10.1093/bioinformatics/btg180; citation_id=CR7">

    <meta name="citation_reference" content="citation_inbook_title=PhyloBayes. v2. 3; citation_publication_date=2007; citation_id=CR8; citation_author=N Lartillot; citation_author=S Blanquart; citation_author=T Lepage">

    <meta name="citation_reference" content="citation_journal_title=BMC Evol Biol; citation_title=BEAST: Bayesian evolutionary analysis by sampling trees; citation_author=A Drummond, A Rambaut; citation_volume=7; citation_issue=214; citation_publication_date=2007; citation_pages=1471-2148; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Rates of Molecular Evolution Are Linked to Life History in Flowering Plants; citation_author=S Smith, M Donoghue; citation_volume=322; citation_issue=5898; citation_publication_date=2008; citation_pages=86-89; citation_doi=10.1126/science.1163197; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Cladistics; citation_title=Phylogenetic analysis of 73060 taxa corroborates major eukaryotic groups; citation_author=PA Goloboff, SA Catalano, JM Mirande, CA Szumik, JS Arias, M Källersjö, JS Farris; citation_volume=25; citation_publication_date=2009; citation_pages=1-20; citation_doi=10.1111/j.1096-0031.2009.00255.x; citation_id=CR11">

    <meta name="citation_reference" content="citation_title=Mega-phylogeny approach for comparative biology: an alternative to supertree and supermatrix approaches; citation_inbook_title=BMC Evolutionary Biology; citation_publication_date=2009; citation_id=CR12; citation_author=SA Smith; citation_author=JM Beaulieu; citation_author=MJ Donoghue">

    <meta name="citation_reference" content="citation_title=Accelerating Parallel Maximum Likelihood-based Phylogenetic Tree Calculations using Subtree Equality Vectors; citation_inbook_title=Proc of IEEE/ACM Supercomputing Conference 2002 (SC2002); citation_publication_date=2002; citation_id=CR13; citation_author=A Stamatakis; citation_author=T Ludwig; citation_author=H Meier; citation_author=MJ Wolf">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites; citation_author=Z Yang; citation_volume=39; citation_publication_date=1994; citation_pages=306-314; citation_doi=10.1007/BF00160154; citation_id=CR14">

    <meta name="citation_reference" content="citation_title=PhyNav: A novel approach to reconstruct large phylogenies; citation_inbook_title=Proc of GfKl conference; citation_publication_date=2004; citation_id=CR15; citation_author=S Le; citation_author=H Schmidt; citation_author=A Haeseler">

    <meta name="citation_reference" content="Schmidt B (Ed): Bioinformatics: High Performance Parallel Computer Architectures. Taylor &amp; Francis 2010 chap. Orchestrating the Phylogenetic Likelihood Function on Emerging Parallel Architectures; 85–115.">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Open source clustering software; citation_author=MJL de Hoon, JN S Imoto, S Miyano; citation_volume=20; citation_issue=9; citation_publication_date=2004; citation_pages=1453-1454; citation_doi=10.1093/bioinformatics/bth078; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; citation_author=A Stamatakis; citation_volume=22; citation_issue=21; citation_publication_date=2006; citation_pages=2688-2690; citation_doi=10.1093/bioinformatics/btl446; citation_id=CR18">

    <meta name="citation_reference" content="citation_title=AxML: A Fast Program for Sequential and Parallel Phylo-genetic Tree Calculations Based on the Maximum Likelihood Method; citation_inbook_title=Proceedings of 1st IEEE Computer Society Bioinformatics Conference (CSB2002); citation_publication_date=2002; citation_pages=21-28; citation_id=CR19; citation_author=A Stamatakis; citation_author=T Ludwig; citation_author=H Meier; citation_author=MJ Wolf">

    <meta name="citation_reference" content="citation_journal_title=Systematic biology; citation_title=Column sorting: Rapid calculation of the phylogenetic likelihood function; citation_author=S Pond, S Muse; citation_volume=53; citation_issue=5; citation_publication_date=2004; citation_pages=685-692; citation_doi=10.1080/10635150490522269; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=Journal of theoretical biology; citation_title=Phylogenetic estimation with partial likelihood tensors; citation_author=J Sumner, M Charleston; citation_volume=262; citation_issue=3; citation_publication_date=2010; citation_pages=413-424; citation_doi=10.1016/j.jtbi.2009.09.037; citation_id=CR21">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Time and memory efficient likelihood-based tree searches on gappy phylogenomic alignments; citation_author=A Stamatakis, N Alachiotis; citation_volume=26; citation_issue=12; citation_publication_date=2010; citation_pages=i132-i139; citation_doi=10.1093/bioinformatics/btq205; citation_id=CR22">

    <meta name="citation_reference" content="citation_journal_title=Syst Biol; citation_title=Does Choice in Model Selection Affect Maximum Likelihood Analysis?; citation_author=J Ripplinger, J Sullivan; citation_volume=57; citation_publication_date=2008; citation_pages=76-85; citation_doi=10.1080/10635150801898920; citation_id=CR23">

    <meta name="citation_reference" content="citation_title=Phylogenetic Models of Rate Heterogeneity: A High Performance Computing Perspective; citation_inbook_title=Proc. of IPDPS2006, HICOMB Workshop, Proceedings on CD, Rhodos, Greece; citation_publication_date=2006; citation_id=CR24; citation_author=A Stamatakis">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=A Bayesian Mixture Model for Across-Site Heterogeneities in the AminoAcid Replacement Process; citation_author=N Lartillot, H Philippe; citation_volume=21; citation_issue=6; citation_publication_date=2004; citation_pages=1095-1109; citation_doi=10.1093/molbev/msh112; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=Briefings in Bioinformatics; citation_title=Recent developments in the MAFFT multiple sequence alignment program; citation_author=K Katoh, H Toh; citation_volume=9; citation_issue=4; citation_publication_date=2008; citation_pages=286-298; citation_doi=10.1093/bib/bbn013; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research; citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; citation_author=RC Edgar; citation_volume=32; citation_issue=5; citation_publication_date=2004; citation_pages=1792-1797; citation_doi=10.1093/nar/gkh340; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Phyutility: a phyloinformatics tool for trees, alignments and molecular data; citation_author=SA Smith, CW Dunn; citation_volume=24; citation_issue=5; citation_publication_date=2008; citation_pages=715-716; citation_doi=10.1093/bioinformatics/btm619; citation_id=CR28">

    <meta name="citation_reference" content="citation_title=Understanding angiosperm diversification using small and large phylogenetic trees; citation_inbook_title=American Journal of Botany; citation_publication_date=2011; citation_id=CR29; citation_author=S Smith; citation_author=J Beaulieu; citation_author=A Stamatakis; citation_author=M Donoghue">

    <meta name="citation_reference" content="citation_journal_title=J VLSI Sig Proc Sys; citation_title=Exploring new Search Algorithms and Hardware for Phylogenetics: RAxML meets the IBM Cell; citation_author=A Stamatakis, F Blagojevic, CD Antonopoulos, DS Nikolopoulos; citation_volume=48; citation_issue=3; citation_publication_date=2007; citation_pages=271-286; citation_doi=10.1007/s11265-007-0067-4; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=Phil Trans R Soc series B Biol Sci; citation_title=Efficient computation of the phylogenetic likelihood function on multi-gene alignments and multi-core architectures; citation_author=A Stamatakis, M Ott; citation_volume=363; citation_publication_date=2008; citation_pages=3977-3984; citation_doi=10.1098/rstb.2008.0163; citation_id=CR31">

    <meta name="citation_reference" content="citation_journal_title=Molecular biology and evolution; citation_title=INDELible: a flexible simulator of biological sequence evolution; citation_author=W Fletcher, Z Yang; citation_volume=26; citation_issue=8; citation_publication_date=2009; citation_pages=1879-1888; citation_doi=10.1093/molbev/msp098; citation_id=CR32">

    <meta name="citation_author" content="Fernando Izquierdo-Carrasco">

    <meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany">

    <meta name="citation_author" content="Stephen A Smith">

    <meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany">

    <meta name="citation_author_institution" content="2 Smith Lab, Dept. Ecology and Evolutionary Biology, University of Michigan, Ann Arbor, USA">

    <meta name="citation_author" content="Alexandros Stamatakis">

    <meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany">



        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">


        <script type="text/javascript" src="https://consumer.krxd.net/consent/set/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;dc=1&amp;al=1&amp;tg=1&amp;cd=0&amp;sh=0&amp;re=0&amp;callback=Krux.ns.macmillan.kxjsonp_consent_set_1"></script><script type="text/javascript" src="https://consumer.krxd.net/consent/get/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;callback=Krux.ns.macmillan.kxjsonp_consent_get_0"></script><script async="" src="//cdn.krxd.net/ctjs/controltag.js.09ebc571a022d419934530eaf14c8a78"></script><script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/javascript" async="" src="https://cdn.krxd.net/controltag/KDqylSLE.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <link rel="stylesheet" media="screen" href="basic-0600efa430.css">
<link rel="stylesheet" media="print" href="print-7680b1a80b.css">

<link rel="stylesheet" id="js-mustard" href="main-4cf21094a7.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>

        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcbioinformatics.biomedcentral.com',
            siteWithPath: 'bmcbioinformatics.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcbioinformatics',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1471-2105-12-470',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central',
            mustardcut: false
        };
    </script>

        <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('dataLayerCreated', function() {
    var script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function () {
        var dL;
        var doi;
        var imprint;
        var bpids;
        var publisher;
        var pageType;
        var eventObject;

        for (var i = 0; i < window.dataLayer.length; i++) {
            if (window.dataLayer[i].event === "dataLayerCreated") {
                dL = window.dataLayer[i];
            }
        }

        if (!dL || !dL.content.article || !dL.content.article.doi) { return false;}

        doi = dL.content.article.doi;
        imprint = dL.content.contentInfo.imprint;
        bpids = dL.session.authentication.authenticationID;
        pageType = dL.page.category.pageType;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);
        }

        eventObject = {'content_type': pageType, 'doi': doi};

        if (imprint === 'BioMed Central') {
            publisher = 'BMC';
        }
        else if (imprint === 'SpringerOpen') {
            publisher = 'SpringerOpen';
        }
        else {
            throw new Error('Event Tracker Error: The publisher has an unexpected value');
        }

        if (bpids.length > 0) {
            eventObject['business_partner_ids'] = bpids;
        }

        new EventTracker({'platform': publisher}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>

        <!--Polyfills CustomEvent constructor in IE. Allows us to use events to manage race conditions in client side js-->
<script>
    (function () {
        if (typeof window.CustomEvent === "function") { return false; } // If IE polyfill not needed just return

        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: undefined };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>
        
    <script type="text/javascript">
        window.dataLayer = [{"content":{"article":{"doi":"10.1186/1471-2105-12-470","articleType":"Research article","peerReviewType":"Closed","supplement":null,"keywords":"Branch Length;Rate Heterogeneity;Ancestral Node;Memory Saving;Numerical Scaling"},"contentInfo":{"imprint":"BioMed Central","title":"Algorithms, data structures, and numerics for likelihood-based phylogenetic inference of huge trees","publishedAt":1323734400000,"publishedAtDate":"2011-12-13","author":["Fernando Izquierdo-Carrasco","Stephen A Smith","Alexandros Stamatakis"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"bmcbioinformatics.biomedcentral.com","volume":"12","issue":"1","title":"BMC Bioinformatics","type":"SERIES","journalID":12859,"gaCode":"UA-62065396-3","section":[{"sectionId":"AC_e2ddcdcc432a37d7d0b89c589f2db3bc","sectionName":"Transcriptome analysis"}]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Research article","publishingSegment":"BMC-series Biology"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}},"event":"dataLayerCreated"}];
        var event = new CustomEvent('dataLayerCreated');
        document.dispatchEvent(event);
    </script><script id="springerlink-event-tracker" src="eventTracker.asc"></script>


        <script>
            
            var mustardcutlink = document.getElementById('js-mustard');
            if (mustardcutlink && window.matchMedia && window.matchMedia(mustardcutlink.media).matches) {
                window.config.mustardcut = true;
            }
        </script>
        <script>
            (function(d){
                var config = window.config;
                if (config && config.mustardcut && config.publisherBrand && sessionStorage.fontsLoaded === 'true') {
                    if (window.config.publisherBrand === 'BioMed Central') {
                        d.style.setProperty('--font-family-sans', 'Europa');
                        d.style.setProperty('--font-family-serif', 'Noto Serif')
                    } else if (window.config.publisherBrand === 'SpringerOpen' || window.config.publisherBrand === 'Springer') {
                        d.style.setProperty('--font-family-sans', 'Source Sans Pro');
                    }
                }
            })(document.documentElement);
        </script>
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><script type="text/javascript" src="jquery.js"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle a { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle a { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-470">
    

        
        
        <meta property="og:url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-470">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Bioinformatics">
        <meta property="og:title" content="Algorithms, data structures, and numerics for likelihood-based phylogenetic inference of huge trees">
        <meta property="og:description" content="The rapid accumulation of molecular sequence data, driven by novel wet-lab sequencing technologies, poses new challenges for large-scale maximum likelihood-based phylogenetic analyses on trees with more than 30,000 taxa and several genes. The three main computational challenges are: numerical stability, the scalability of search algorithms, and the high memory requirements for computing the likelihood. We introduce methods for solving these three key problems and provide respective proof-of-concept implementations in RAxML. The mechanisms presented here are not RAxML-specific and can thus be applied to any likelihood-based (Bayesian or maximum likelihood) tree inference program. We develop a new search strategy that can reduce the time required for tree inferences by more than 50% while yielding equally good trees (in the statistical sense) for well-chosen starting trees. We present an adaptation of the Subtree Equality Vector technique for phylogenomic datasets with missing data (already available in RAxML v728) that can reduce execution times and memory requirements by up to 50%. Finally, we discuss issues pertaining to the numerical stability of the Γ model of rate heterogeneity on very large trees and argue in favor of rate heterogeneity models that use a single rate or rate category for each site to resolve these problems. We address three major issues pertaining to large scale tree reconstruction under maximum likelihood and propose respective solutions. Respective proof-of-concept/production-level implementations of our ideas are made available as open-source code.">
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12859.jpg">
    
    <script type="text/javascript" src="jquery_002.js"></script><script src="figshare.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><style type="text/css">.MathJax_Display {text-align: center; margin: 0; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none; box-sizing: content-box}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_test {font-style: normal; font-weight: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-transform: none; letter-spacing: normal; word-spacing: normal; overflow: hidden; height: 1px}
.MathJax_test.mjx-test-display {display: table!important}
.MathJax_test.mjx-test-inline {display: inline!important; margin-right: -1px}
.MathJax_test.mjx-test-default {display: block!important; clear: both}
.MathJax_ex_box {display: inline-block!important; position: absolute; overflow: hidden; min-height: 0; max-height: none; padding: 0; border: 0; margin: 0; width: 1px; height: 60ex}
.MathJax_em_box {display: inline-block!important; position: absolute; overflow: hidden; min-height: 0; max-height: none; padding: 0; border: 0; margin: 0; width: 1px; height: 60em}
.mjx-test-inline .MathJax_left_box {display: inline-block; width: 0; float: left}
.mjx-test-inline .MathJax_right_box {display: inline-block; width: 0; float: right}
.mjx-test-display .MathJax_right_box {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
#MathJax_Message {margin: 0}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link rel="preload" href="https://adservice.google.com/adsid/integrator.js?domain=bmcbioinformatics.biomedcentral.com"><script type="text/javascript" src="https://adservice.google.com/adsid/integrator.js?domain=bmcbioinformatics.biomedcentral.com"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_319.js" async=""></script></head>

    <body class="journal journal-fulltext"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="">Typesetting math: 22%</div>
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcbioinformatics/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1471-2105-12-470;kwrd=Branch Length,Rate Heterogeneity,Ancestral Node,Memory Saving,Numerical Scaling;pmc=L15001,B12050,I23050,L17004,M14018,M14018;">
                <noscript>
                    <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/270604982/bmc/bmcbioinformatics/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2105-12-470&amp;kwrd=Branch Length,Rate Heterogeneity,Ancestral Node,Memory Saving,Numerical Scaling&amp;pmc=L15001,B12050,I23050,L17004,M14018,M14018&amp;">
                        <img data-test="gpt-advert-fallback-img"
                             src="//pubads.g.doubleclick.net/gampad/ad?iu=/270604982/bmc/bmcbioinformatics/articles&amp;sz=728x90,970x90&amp;pos=LB1&amp;doi=10.1186/1471-2105-12-470&amp;kwrd=Branch Length,Rate Heterogeneity,Ancestral Node,Memory Saving,Numerical Scaling&amp;pmc=L15001,B12050,I23050,L17004,M14018,M14018&amp;"
                             alt="Advertisement"
                             width="728"
                             height="90">
                    </a>
                </noscript>
            <div id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img src="logo-bmc-white-series-1da6525f4e.svg" alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav c-navbar__nav--right">
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="c-form-field__label">Search all BMC articles</label>
        <div class="c-flex">
            <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" type="text">
            <div>
                <button class="c-button" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


            </div>
        </div>
        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--bmc-bioinformatics ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcbioinformatics.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Bioinformatics</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcbioinformatics.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Methods">Methods</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec11" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and Discussion">Results and Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec16" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Research article</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">Algorithms, data structures, and numerics for likelihood-based phylogenetic inference of huge trees</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Fernando&nbsp;Izquierdo-Carrasco</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:Fernando.Izquierdo@h-its.org" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Stephen&nbsp;A&nbsp;Smith</span><sup><a href="#Aff1" class="AffiliationID">1</a>, <a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Alexandros&nbsp;Stamatakis</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:Alexandros.Stamatakis@h-its.org" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Bioinformatics</span></span><span class="ArticleCitation_Year">2011</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">12</strong>:470</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/1471-2105-12-470">https://doi.org/10.1186/1471-2105-12-470</a></p><p class="Copyright">© &nbsp;Izquierdo-Carrasco et al; licensee BioMed Central Ltd.&nbsp;2011</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>23&nbsp;April&nbsp;2011</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>13&nbsp;December&nbsp;2011</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">13&nbsp;December&nbsp;2011</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">The rapid accumulation of molecular 
sequence data, driven by novel wet-lab sequencing technologies, poses 
new challenges for large-scale maximum likelihood-based phylogenetic 
analyses on trees with more than 30,000 taxa and several genes. The 
three main computational challenges are: numerical stability, the 
scalability of search algorithms, and the high memory requirements for 
computing the likelihood.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">We introduce methods for solving these 
three key problems and provide respective proof-of-concept 
implementations in RAxML. The mechanisms presented here are not 
RAxML-specific and can thus be applied to any likelihood-based (Bayesian
 or maximum likelihood) tree inference program. We develop a new search 
strategy that can reduce the time required for tree inferences by more 
than 50% while yielding equally good trees (in the statistical sense) 
for well-chosen starting trees. We present an adaptation of the Subtree 
Equality Vector technique for phylogenomic datasets with missing data 
(already available in RAxML v728) that can reduce execution times <em xmlns="" class="EmphasisTypeItalic">and</em>
 memory requirements by up to 50%. Finally, we discuss issues pertaining
 to the numerical stability of the Γ model of rate heterogeneity on very
 large trees and argue in favor of rate heterogeneity models that use a 
single rate or rate category for each site to resolve these problems.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p class="Para">We address three major issues pertaining
 to large scale tree reconstruction under maximum likelihood and propose
 respective solutions. Respective proof-of-concept/production-level 
implementations of our ideas are made available as open-source code.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Branch Length</li><li class="c-keywords__item" itemprop="about">Rate Heterogeneity</li><li class="c-keywords__item" itemprop="about">Ancestral Node</li><li class="c-keywords__item" itemprop="about">Memory Saving</li><li class="c-keywords__item" itemprop="about">Numerical Scaling</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The rapid accumulation of molecular sequence data that is driven by novel wet-lab sequencing techniques such as pyrosequencing [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>] and collaborative whole-genome sequencing projects such as the 10 K vertebrate genome project <span class="ExternalRef"><a href="http://www.genome10k.org/" target="_blank" class="ext-link"><span class="RefSource">http://www.genome10k.org/</span></a></span>
 pose unprecedented challenges with respect to the scalability and 
numerical stability of phylogenetic inference programs. Datasets are 
continuously growing with respect to the number of base-pairs and/or the
 number of taxa. For likelihood-based [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>] (Bayesian and Maximum Likelihood) codes with their extremely high computational requirements in terms of memory <em xmlns="" class="EmphasisTypeItalic">and</em>
 floating point operations, improving scalability for large datasets is 
particularly challenging. Here, we focus on algorithm design, 
improvement of numerical stability, and technical solutions for 
accelerating the likelihood function and reducing the memory 
requirements on phylogenomic datasets with missing data that contain 
more than 10,000 taxa. The concepts we introduce are generic, that is, 
they can be applied to other likelihood-based programs such as IQPNNI [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>], GARLI [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], PHYML 3.0 [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], FastTree 2.0 [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], MrBayes [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>], PhyloBayes [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>], and BEAST [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>] or to libraries for computing the phylogenetic likelihood such as BEAGLE <span class="ExternalRef"><a href="http://code.google.com/p/beagle-lib/" target="_blank" class="ext-link"><span class="RefSource">http://code.google.com/p/beagle-lib/</span></a></span>.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The largest published maximum likelihood tree to date contained approximately 13,000 taxa [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. FastTree 2 has been used to infer approximate maximum likelihood trees of approximately 200,000 taxa [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>], and the largest published tree using parsimony contained approximately 73,000 taxa [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>]. With novel alignment assembly methods such as, for instance, PHLAWD [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>],
 increasing data availability, and collaborative projects for 
reconstructing huge trees (e.g., the iPlant plant tree of life grand 
challenge project <span class="ExternalRef"><a href="http://www.iplantcollaborative.org/challenge/iplant-tree-life" target="_blank" class="ext-link"><span class="RefSource">http://www.iplantcollaborative.org/challenge/iplant-tree-life</span></a></span>) there exists a need to infer even larger trees exceeding 100,000 taxa. RAxML version 7.2.8 alpha (available at <span class="ExternalRef"><a href="http://wwwkramer.in.tum.de/exelixis/software.html" target="_blank" class="ext-link"><span class="RefSource">http://wwwkramer.in.tum.de/exelixis/software.html</span></a></span>)
 already incorporates some of the mechanisms presented here. It has been
 successfully deployed --without crashing-- to infer biologically 
reasonable Maximum Likelihood (ML [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>]) trees on phylogenomic datasets with 10-20 genes for 38,000, 56,000, and 116,000 taxa.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Methods" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Methods">Methods</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Methods" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 the following we discuss three different topics: (i) the design of a 
new search algorithm for large datasets, (ii) an appropriately adapted 
re-implementation of the Subtree Equality Vector (SEVs [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>]) technique in RAxML, and (iii) numerical issues that arise with the widely used Γ model of rate heterogeneity [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].</p><section id="Sec3" class="Section2 RenderAsSection2">
                <h3 class="Heading">PhyNav Revisited: Constraining the tree search to a backbone tree</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">PhyNav (Phylogenetic Navigator [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>])
 first introduced the idea to reduce the dimension of the tree (and 
potentially also the memory footprint of the tree) on which the search 
is conducted, by identifying subtrees of closely related taxa whose root
 may be represented by a single <em xmlns="" class="EmphasisTypeItalic">virtual tip</em>.
 The rationale is that in large alignments there may exist many taxa 
that are closely related to each other which can therefore be clustered 
together into a single virtual tip (which we also denote as 
super-taxon). By clustering taxa into virtual tips, the dimension of the
 tree can be reduced allowing for a tree search on the backbone tree 
that is induced by the virtual tips.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Given
 a hypothetical perfectly balanced tree, a reduction of 50% could 
correspond to collapsing each pair of taxa into a single virtual tip. 
Thus, for each pair of tips, there would be one less inner node to 
operate on, and the total number of inner nodes would have been halved.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We henceforth denote such a reduction of the tree dimension as <em xmlns="" class="EmphasisTypeItalic">reduction factor</em> and denote the reduced unrooted tree that is induced by the virtual tips as <em xmlns="" class="EmphasisTypeItalic">backbone</em> tree.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Once
 an appropriate backbone tree has been computed (see below), a SPR-based
 (Subtree Pruning Re-Grafting) search, or any other heuristic search 
strategy using, for instance, NNI (Nearest Neighbor Interchange) or TBR 
(Tree Bisection Reconnection) moves, can be restricted to operate within
 this backbone tree. In other words, the virtual tips are interpreted as
 tips in the backbone tree on which we conduct the tree search. In our 
RAxML proof-of-concept implementation that deploys SPR moves, only 
subtrees that form part of the backbone tree are pruned and will 
exclusively be re-inserted into branches that lie within the backbone.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Despite
 restricting the tree search to the backbone, in our setup, we always 
compute the log likelihood score of the comprehensive tree during the 
backbone tree search. The log likelihood score of the comprehensive tree
 can be easily computed, because virtual tips are ancestral probability 
vectors that summarize the signal of the (excluded) real tips situated 
below the respective virtual tip. Note that, memory requirements for 
storing the ancestral probability vector representing a virtual tip are 
significantly higher than for storing a terminal taxon. For terminal 
taxa, it suffices to store the molecular sequence as an array of single 
bytes and to use a lookup table for obtaining the corresponding tip 
probability vector (see [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] for details).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">One
 way to implement a PhyNav-like method comprises the following 
computational steps: Initially, generate a reasonable starting tree, 
using, for instance, parsimony. Then, determine an appropriate backbone 
tree and optimize branch lengths and model parameters on this 
comprehensive tree under ML. Thereafter, determine and mark the 
ancestral probability vectors that will become virtual tips in the 
backbone. Finally, conduct a tree search on the backbone tree.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 also achieve a memory footprint reduction (not implemented), one can 
write a multiple sequence alignment for the backbone to file that will 
partially consist of nucleotide sequences and partially of ancestral 
probability vectors representing virtual tips. This reduced alignment 
can then be parsed together with the backbone tree for conducting a tree
 search. By deploying PhyNav-like algorithms, one can save memory, if 
inner nodes (ancestral probability vectors) are excluded from the 
backbone, since ancestral probability vectors largely dominate the 
memory requirements of likelihood-based programs (see [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] for details).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 terms of algorithm design, the issue that predominantly affects 
performance is the computation of the backbone tree, that is: How do we 
determine "good" virtual tips?</p>
                <section id="Sec4" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Building the Backbone</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To build a backbone, we assume that a reasonable (i.e., non-random) fully resolved comprehensive tree <em xmlns="" class="EmphasisTypeItalic">T</em> comprising all taxa (e.g., obtained via parsimony using TNT [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>] or RAxML) is provided as input. This comprehensive <em xmlns="" class="EmphasisTypeItalic">n</em>-taxon tree has <em xmlns="" class="EmphasisTypeItalic">n</em> tips and <em xmlns="" class="EmphasisTypeItalic">n</em>-2 ancestral (inner) nodes.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The second parameter for the backbone tree algorithm is the desired tree size reduction factor <em xmlns="" class="EmphasisTypeItalic">R</em>, where 0.0 &lt; <em xmlns="" class="EmphasisTypeItalic">R</em> &lt; 1.0. This parameter denotes to which fraction of <em xmlns="" class="EmphasisTypeItalic">n</em> the backbone tree shall be reduced in size. Ideally, the backbone tree will then comprise <em xmlns="" class="EmphasisTypeItalic">n</em>·<em xmlns="" class="EmphasisTypeItalic">R</em>-2 ancestral nodes and <em xmlns="" class="EmphasisTypeItalic">n</em>·<em xmlns="" class="EmphasisTypeItalic">R</em> backbone tips. Backbone tips may either be virtual tips (ancestral nodes) or real tips. Evidently, choosing very low values of <em xmlns="" class="EmphasisTypeItalic">R</em>
 may significantly impact the quality of the inference, specially if 
very short branch lengths are present. According to our experience (see 
results section), using <em xmlns="" class="EmphasisTypeItalic">R</em> &gt; 0.25 is a safe lower bound.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Our
 backbone construction algorithm executes two main computational steps 
that are described in more detail below. Initially, we assign the <em xmlns="" class="EmphasisTypeItalic">n</em> tips to <em xmlns="" class="EmphasisTypeItalic">n</em>·<em xmlns="" class="EmphasisTypeItalic">R</em> clusters, that is, <em xmlns="" class="EmphasisTypeItalic">c</em> = <span class="stix">⌈</span><em xmlns="" class="EmphasisTypeItalic">n</em>·<em xmlns="" class="EmphasisTypeItalic">R</em><span class="stix">⌉</span>, where <em xmlns="" class="EmphasisTypeItalic">c</em>
 is the total number of clusters obtained. For each tip we store a 
cluster identifier that denotes to which cluster the tip has been 
assigned. Thereafter, we traverse the tree and use the cluster 
identifiers to label all ancestral nodes as residing inside, outside or 
on the boundary of the backbone.</p>
                  <section id="Sec5" class="Section4 RenderAsSection4">
                    <h5 class="Heading">Tip clustering</h5>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">There
 are plenty of possible approaches to cluster tips. First, the available
 topology itself (for example, the optimized parsimony starting tree) 
can be used directly as a hierachical tree (hierachical clustering). 
Another alternative may be to compute parsimony scores of subtrees, and 
then cluster together according to a threshold. Here, we only present an
 approach based on computing a distance matrix and applying 
average-linkage hierarchical clustering [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>]. Our assesment indicated this approach yields significantly better results than the others.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 standard hierarchical clustering, the first step consists of 
calculating a distance matrix that contains the pair-wise distances 
between all items (tips) to be clustered. However, given a comprehensive
 tree <em xmlns="" class="EmphasisTypeItalic">T</em> with ML estimates 
of branch lengths, we can directly obtain this distance matrix from the 
tree by calculating the pair-wise patristic distances. The patristic 
distance between two taxa is the sum of branch lengths on the path in 
the tree connecting the two taxa. Thus, the distance matrix is 
symmetric. The space requirements for storing such a patristic distance 
matrix are in <em xmlns="" class="EmphasisTypeItalic">O</em>(<em xmlns="" class="EmphasisTypeItalic">n</em><sup>2</sup>) which can become prohibitive for large alignments with <em xmlns="" class="EmphasisTypeItalic">n</em>
 ≥ 30, 000 tips. We observe that, the pair-wise patristic distances 
between most tips will be very large and hence these tips will be 
assigned to different clusters anyway. Therefore, to save memory, one 
can decompose this process into computing several smaller, partial 
distance matrices, since the comprehensive starting tree already induces
 a hierarchical clustering structure. If we subdivide the problem into 
computing <em xmlns="" class="EmphasisTypeItalic">p</em> partial pair-wise distance matrices, and each partial matrix defines <span class="InlineEquation" id="IEq1"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-1"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="semantics" id="MathJax-Span-3"><span class="mrow" id="MathJax-Span-4"><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo MathClass-rel" id="MathJax-Span-6" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-7" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.279em; height: 0px;"><span style="position: absolute; clip: rect(3.107em, 1001em, 4.397em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-8"><span class="mo MathClass-op" id="MathJax-Span-9" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.367em, 1000.44em, 4.154em, -1000em); top: -4.479em; left: 1.056em;"><span class="mrow" id="MathJax-Span-10"><span class="mi" id="MathJax-Span-11" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.386em, 1001.22em, 4.162em, -1000em); top: -3.717em; left: 1.056em;"><span class="mrow" id="MathJax-Span-12"><span class="mi" id="MathJax-Span-13" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-rel" id="MathJax-Span-14" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-15" style="font-size: 70.7%; font-family: MathJax_Main;">0</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="msub" id="MathJax-Span-16" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.752em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.43em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-17"><span class="mi" id="MathJax-Span-18" style="font-family: MathJax_Math; font-style: italic;">c</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.433em;"><span class="mrow" id="MathJax-Span-19"><span class="mi" id="MathJax-Span-20" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-21" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfenced" id="MathJax-Span-22"><span class="mo" id="MathJax-Span-23" style=""><span style="font-family: MathJax_Main;">⌈</span></span><span class="mrow" id="MathJax-Span-24"><span class="mi" id="MathJax-Span-25" style="font-family: MathJax_Math; font-style: italic;">n</span><span class="mo MathClass-bin" id="MathJax-Span-26" style="font-family: MathJax_Main; padding-left: 0.222em;">⋅</span><span class="mi" id="MathJax-Span-27" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">R</span></span><span class="mo" id="MathJax-Span-28" style=""><span style="font-family: MathJax_Main;">⌉</span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <mi>c</mi>
                            <mo class="MathClass-rel">=</mo>
                            <msubsup>
                              <mrow>
                                <mo class="MathClass-op">∑</mo>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                                <mo class="MathClass-rel">=</mo>
                                <mn>0</mn>
                              </mrow>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                            </msubsup>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                            <mo class="MathClass-rel">=</mo>
                            <mfenced separators="" open="⌈" close="⌉">
                              <mrow>
                                <mi>n</mi>
                                <mo class="MathClass-bin">⋅</mo>
                                <mi>R</mi>
                              </mrow>
                            </mfenced>
                          </mrow></semantics></math></script></span>, so
 that the total number of clusters, we need to ensure that desired 
clusters still corresponds to the specified reduction factor <em xmlns="" class="EmphasisTypeItalic">R</em>. To achieve this, we do not fix the number of partial matrices <em xmlns="" class="EmphasisTypeItalic">k</em> a priori. Instead, we define a threshold value <em xmlns="" class="EmphasisTypeItalic">m</em> that represents an upper bound for the number of tips contained in each partial matrix. Let <em xmlns="" class="EmphasisTypeItalic">n</em> be the total number of taxa, <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> the number of tips in a partial matrix, where <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> ≤ <em xmlns="" class="EmphasisTypeItalic">m</em> and <span class="InlineEquation" id="IEq2"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-29"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-30"><span class="semantics" id="MathJax-Span-31"><span class="mrow" id="MathJax-Span-32"><span class="mi" id="MathJax-Span-33" style="font-family: MathJax_Math; font-style: italic;">n</span><span class="mo MathClass-rel" id="MathJax-Span-34" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-35" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.279em; height: 0px;"><span style="position: absolute; clip: rect(3.107em, 1001em, 4.397em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-36"><span class="mo MathClass-op" id="MathJax-Span-37" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.367em, 1000.44em, 4.154em, -1000em); top: -4.479em; left: 1.056em;"><span class="mrow" id="MathJax-Span-38"><span class="mi" id="MathJax-Span-39" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.386em, 1001.22em, 4.162em, -1000em); top: -3.717em; left: 1.056em;"><span class="mrow" id="MathJax-Span-40"><span class="mi" id="MathJax-Span-41" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-rel" id="MathJax-Span-42" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-43" style="font-size: 70.7%; font-family: MathJax_Main;">0</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="msub" id="MathJax-Span-44" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.919em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.58em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-45"><span class="mi" id="MathJax-Span-46" style="font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.6em;"><span class="mrow" id="MathJax-Span-47"><span class="mi" id="MathJax-Span-48" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <mi>n</mi>
                            <mo class="MathClass-rel">=</mo>
                            <msubsup>
                              <mrow>
                                <mo class="MathClass-op">∑</mo>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                                <mo class="MathClass-rel">=</mo>
                                <mn>0</mn>
                              </mrow>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                            </msubsup>
                            <msub>
                              <mrow>
                                <mi>n</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow></semantics></math></script></span>. From each partial matrix, we extract an amount of clusters proportional to its size, that is, <span class="InlineEquation" id="IEq3"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-49"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-50"><span class="semantics" id="MathJax-Span-51"><span class="mrow" id="MathJax-Span-52"><span class="msub" id="MathJax-Span-53"><span style="display: inline-block; position: relative; width: 0.752em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.43em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-54"><span class="mi" id="MathJax-Span-55" style="font-family: MathJax_Math; font-style: italic;">c</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.433em;"><span class="mrow" id="MathJax-Span-56"><span class="mi" id="MathJax-Span-57" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo MathClass-rel" id="MathJax-Span-58" style="font-family: MathJax_Main; padding-left: 0.278em;">∝</span><span class="mi" id="MathJax-Span-59" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">c</span><span class="mo MathClass-bin" id="MathJax-Span-60" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mfrac" id="MathJax-Span-61" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.77em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.545em, 1000.65em, 4.258em, -1000em); top: -4.514em; left: 50%; margin-left: -0.325em;"><span class="mrow" id="MathJax-Span-62"><span class="msub" id="MathJax-Span-63"><span style="display: inline-block; position: relative; width: 0.65em; height: 0px;"><span style="position: absolute; clip: rect(3.545em, 1000.41em, 4.154em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-64"><span class="mi" id="MathJax-Span-65" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.896em; left: 0.424em;"><span class="mrow" id="MathJax-Span-66"><span class="mi" id="MathJax-Span-67" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.545em, 1000.41em, 4.154em, -1000em); top: -3.657em; left: 50%; margin-left: -0.212em;"><span class="mrow" id="MathJax-Span-68"><span class="mi" id="MathJax-Span-69" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(0.856em, 1000.77em, 1.205em, -1000em); top: -1.281em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.77em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                            <mo class="MathClass-rel">∝</mo>
                            <mi>c</mi>
                            <mo class="MathClass-bin">×</mo>
                            <mfrac>
                              <mrow>
                                <msub>
                                  <mrow>
                                    <mi>n</mi>
                                  </mrow>
                                  <mrow>
                                    <mi>i</mi>
                                  </mrow>
                                </msub>
                              </mrow>
                              <mrow>
                                <mi>n</mi>
                              </mrow>
                            </mfrac>
                          </mrow></semantics></math></script></span>.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">This
 is implemented as follows: First, we find a set of subtrees such that 
(i) each subtree has as many tips as possible and at most <em xmlns="" class="EmphasisTypeItalic">m</em>
 tree tips and (ii) each tree tip is included in exactly one subtree, 
that is, all tree tips are included in one subtree and no tip forms part
 of more than one subtree.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For each such subtree <em xmlns="" class="EmphasisTypeItalic">i</em>, we then build a (partial) patristic distance matrix for all <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> subtree tips. Thereafter, we cluster them, 
by generating a hierarchical cluster tree. This hierarchical tree may be
 cut at different levels to generate a varying number of subtree tip 
groups. We choose to cut the the tree such that it generates <em xmlns="" class="EmphasisTypeItalic">c</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> clusters of subtree tips. If required, the number of desired clusters <em xmlns="" class="EmphasisTypeItalic">c</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> will have been iteratively adjusted beforehand (for further details see below) for each partial matrix <em xmlns="" class="EmphasisTypeItalic">i</em> to ensure that <span class="InlineEquation" id="IEq4"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-70"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-71"><span class="semantics" id="MathJax-Span-72"><span class="mrow" id="MathJax-Span-73"><span class="mi" id="MathJax-Span-74" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo MathClass-rel" id="MathJax-Span-75" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="msubsup" id="MathJax-Span-76" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.279em; height: 0px;"><span style="position: absolute; clip: rect(3.107em, 1001em, 4.397em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-77"><span class="mo MathClass-op" id="MathJax-Span-78" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.367em, 1000.44em, 4.154em, -1000em); top: -4.479em; left: 1.056em;"><span class="mrow" id="MathJax-Span-79"><span class="mi" id="MathJax-Span-80" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.386em, 1001.22em, 4.162em, -1000em); top: -3.717em; left: 1.056em;"><span class="mrow" id="MathJax-Span-81"><span class="mi" id="MathJax-Span-82" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-rel" id="MathJax-Span-83" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-84" style="font-size: 70.7%; font-family: MathJax_Main;">0</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="msub" id="MathJax-Span-85" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.752em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.43em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-86"><span class="mi" id="MathJax-Span-87" style="font-family: MathJax_Math; font-style: italic;">c</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.433em;"><span class="mrow" id="MathJax-Span-88"><span class="mi" id="MathJax-Span-89" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <mi>c</mi>
                            <mo class="MathClass-rel">=</mo>
                            <msubsup>
                              <mrow>
                                <mo class="MathClass-op">∑</mo>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                                <mo class="MathClass-rel">=</mo>
                                <mn>0</mn>
                              </mrow>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                            </msubsup>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow></semantics></math></script></span>.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 example, consider a 40, 000-taxon tree, a reduction factor of 0.5 
(corresponding to 20, 000 clusters), and a partial matrix threshold of 
32, 000 taxa. In this example, we may obtain distance matrices of 10, 
000 and 30, 000 taxa respectively. Then we will need to extract 15, 000 
clusters from the 30, 000 taxa distance matrix and 5, 000 clusters from 
the 10, 000 taxa distance matrix.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 be able to apply this method and compute partial patristic distance 
matrices, we need to devise an algorithm that selects subtrees from the 
comprehensive phylogeny such that they contain at most <em xmlns="" class="EmphasisTypeItalic">m</em> taxa. We start by selecting the innermost node of the tree (see below). Consider that, each inner node <em xmlns="" class="EmphasisTypeItalic">i</em> of an unrooted binary tree <em xmlns="" class="EmphasisTypeItalic">T</em> can be regarded as a trifurcation that defines three subtrees <em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i, a</em>
                      </sub>, <em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i, b</em>
                      </sub>and <em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i, c</em>
                      </sub>. We define subtree length <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> ) as the sum of all branch lengths in subtree <em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> . Thus, <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i, a</em>
                      </sub>) + <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i, b</em>
                      </sub>) + <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i, c</em>
                      </sub>) = <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em>) holds for any inner node <em xmlns="" class="EmphasisTypeItalic">i</em>, where <em xmlns="" class="EmphasisTypeItalic">T</em> is the comprehensive tree.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In our current default implementation, we select the innermost node <em xmlns="" class="EmphasisTypeItalic">j</em> that maximizes <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em>) - max{<em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">j, a</em>
                      </sub>), <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">j, b</em>
                      </sub>), <em xmlns="" class="EmphasisTypeItalic">stl</em>(<em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">j, c</em>
                      </sub>)}. An alternative criterion for selecting 
the innermost node is to determine the node that minimizes the variance 
of the three outgoing subtree lengths. Other possible criteria, that are
 not based on subtree length may be defined, for instance, as finding 
the node that minimizes the variance of the node-to-tip distance or 
finding the node with the highest minimum node-to-tip distance. The 
node-to-tip distance is defined as the sum of branch lengths on the path
 in the tree leading from an ancestral node to a tip.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 conducted an empirical assessment (based on our collection of large 
real-world trees) of these alternative approaches for determining the 
innermost node of a tree. The outcome (results in additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>)
 was that the respective innermost nodes (as identified by the above 
criteria) are either identical or close neighbors, that is, located in 
the same region of the tree.</p>
                  </section>
                  <section id="Sec6" class="Section4 RenderAsSection4">
                    <h5 class="Heading">Backbone construction</h5>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Once
 we have determined the innermost node, we conduct a depth-first tree 
traversal starting at this node and descend into each of the three 
subtrees. The depth-first traversal terminates, when a subtree root is 
encountered that comprises ≤ <em xmlns="" class="EmphasisTypeItalic">m</em> tips. All subtree roots that contain ≤ <em xmlns="" class="EmphasisTypeItalic">m</em> tips are stored in a list for further processing. Thus, when the depth-first traversal has been completed, this list of <em xmlns="" class="EmphasisTypeItalic">k</em> subtree roots can be used to generate the <em xmlns="" class="EmphasisTypeItalic">k</em> partial patristic distance matrices of maximum size <em xmlns="" class="EmphasisTypeItalic">O</em>(<em xmlns="" class="EmphasisTypeItalic">m</em><sup>2</sup>). In our implementation, we set <em xmlns="" class="EmphasisTypeItalic">m</em> := 1024. This a suitable value, since only a few seconds are required for processing partial distance matrices.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 each subtree root (i.e., each partial patristic distance matrix), we 
determine how many clusters should approximately be extracted, via <span class="InlineEquation" id="IEq5"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-5-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-90"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-91"><span class="semantics" id="MathJax-Span-92"><span class="mrow" id="MathJax-Span-93"><span class="msub" id="MathJax-Span-94"><span style="display: inline-block; position: relative; width: 0.875em; height: 0px;"><span style="position: absolute; clip: rect(3.249em, 1000.49em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-95"><span class="mover" id="MathJax-Span-96"><span style="display: inline-block; position: relative; width: 0.556em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.43em, 4.158em, -1000em); top: -4.002em; left: 0.034em;"><span class="mrow" id="MathJax-Span-97"><span class="mi" id="MathJax-Span-98" style="font-family: MathJax_Math; font-style: italic;">c</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.267em, 1000.43em, 3.603em, -1000em); top: -4.02em; left: 0.056em;"><span class="mo MathClass-op" id="MathJax-Span-99" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.556em;"><span class="mrow" id="MathJax-Span-100"><span class="mi" id="MathJax-Span-101" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="mo MathClass-punc" id="MathJax-Span-102" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="mo MathClass-rel" id="MathJax-Span-103" style="font-family: MathJax_Main;">=</span><span class="mfenced" id="MathJax-Span-104"><span class="mo" id="MathJax-Span-105" style="vertical-align: 0em;"><span style="font-family: MathJax_Size1;">⌊</span></span><span class="mrow" id="MathJax-Span-106"><span class="mfrac" id="MathJax-Span-107"><span style="display: inline-block; position: relative; width: 0.474em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.386em, 1000.3em, 4.147em, -1000em); top: -4.402em; left: 50%; margin-left: -0.177em;"><span class="mrow" id="MathJax-Span-108"><span class="mn" id="MathJax-Span-109" style="font-size: 70.7%; font-family: MathJax_Main;">1</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.386em, 1000.32em, 4.147em, -1000em); top: -3.631em; left: 50%; margin-left: -0.177em;"><span class="mrow" id="MathJax-Span-110"><span class="mn" id="MathJax-Span-111" style="font-size: 70.7%; font-family: MathJax_Main;">2</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(0.856em, 1000.47em, 1.205em, -1000em); top: -1.281em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.474em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span><span class="mo MathClass-bin" id="MathJax-Span-112" style="font-family: MathJax_Main; padding-left: 0.222em;">+</span><span class="mi" id="MathJax-Span-113" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">c</span><span class="mo MathClass-bin" id="MathJax-Span-114" style="font-family: MathJax_Main; padding-left: 0.222em;">⋅</span><span class="mfrac" id="MathJax-Span-115" style="padding-left: 0.222em;"><span style="display: inline-block; position: relative; width: 0.77em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.545em, 1000.65em, 4.258em, -1000em); top: -4.514em; left: 50%; margin-left: -0.325em;"><span class="mrow" id="MathJax-Span-116"><span class="msub" id="MathJax-Span-117"><span style="display: inline-block; position: relative; width: 0.65em; height: 0px;"><span style="position: absolute; clip: rect(3.545em, 1000.41em, 4.154em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-118"><span class="mi" id="MathJax-Span-119" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.896em; left: 0.424em;"><span class="mrow" id="MathJax-Span-120"><span class="mi" id="MathJax-Span-121" style="font-size: 50%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.545em, 1000.41em, 4.154em, -1000em); top: -3.657em; left: 50%; margin-left: -0.212em;"><span class="mrow" id="MathJax-Span-122"><span class="mi" id="MathJax-Span-123" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">n</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(0.856em, 1000.77em, 1.205em, -1000em); top: -1.281em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.77em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.061em;"></span></span></span></span></span><span class="mo" id="MathJax-Span-124" style="vertical-align: 0em;"><span style="font-family: MathJax_Size1;">⌋</span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-5"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <msub>
                              <mrow>
                                <mover accent="true">
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-op">¯</mo>
                                </mover>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                            <mo class="MathClass-punc">:</mo>
                            <mo class="MathClass-rel">=</mo>
                            <mfenced separators="" open="⌊" close="⌋">
                              <mrow>
                                <mfrac>
                                  <mrow>
                                    <mn>1</mn>
                                  </mrow>
                                  <mrow>
                                    <mn>2</mn>
                                  </mrow>
                                </mfrac>
                                <mo class="MathClass-bin">+</mo>
                                <mi>c</mi>
                                <mo class="MathClass-bin">⋅</mo>
                                <mfrac>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>n</mi>
                                      </mrow>
                                      <mrow>
                                        <mi>i</mi>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                  <mrow>
                                    <mi>n</mi>
                                  </mrow>
                                </mfrac>
                              </mrow>
                            </mfenced>
                          </mrow></semantics></math></script></span>, where <em xmlns="" class="EmphasisTypeItalic">i</em> is the cluster (subtree) number, <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> is the number of tips in the respective cluster/subtree, <em xmlns="" class="EmphasisTypeItalic">c</em> = <em xmlns="" class="EmphasisTypeItalic">n</em>·<em xmlns="" class="EmphasisTypeItalic">R</em> is the total number of desired clusters, and <span class="InlineEquation" id="IEq6"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-6-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-125"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-126"><span class="semantics" id="MathJax-Span-127"><span class="mrow" id="MathJax-Span-128"><span class="msub" id="MathJax-Span-129"><span style="display: inline-block; position: relative; width: 0.875em; height: 0px;"><span style="position: absolute; clip: rect(3.249em, 1000.49em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-130"><span class="mover" id="MathJax-Span-131"><span style="display: inline-block; position: relative; width: 0.556em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.43em, 4.158em, -1000em); top: -4.002em; left: 0.034em;"><span class="mrow" id="MathJax-Span-132"><span class="mi" id="MathJax-Span-133" style="font-family: MathJax_Math; font-style: italic;">c</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.267em, 1000.43em, 3.603em, -1000em); top: -4.02em; left: 0.056em;"><span class="mo MathClass-op" id="MathJax-Span-134" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.556em;"><span class="mrow" id="MathJax-Span-135"><span class="mi" id="MathJax-Span-136" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-6"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <msub>
                              <mrow>
                                <mover accent="true">
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-op">¯</mo>
                                </mover>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow></semantics></math></script></span> is the number of clusters for subtree <em xmlns="" class="EmphasisTypeItalic">i</em>. In general, <span class="InlineEquation" id="IEq7"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processed" id="MathJax-Element-7-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-137"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 122%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-138"><span class="semantics" id="MathJax-Span-139"><span class="mrow" id="MathJax-Span-140"><span class="mi" id="MathJax-Span-141" style="font-family: MathJax_Math; font-style: italic;">c</span><span class="mo MathClass-rel" id="MathJax-Span-142" style="font-family: MathJax_Main; padding-left: 0.278em;">≠</span><span class="msubsup" id="MathJax-Span-143" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 2.279em; height: 0px;"><span style="position: absolute; clip: rect(3.107em, 1001em, 4.397em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-144"><span class="mo MathClass-op" id="MathJax-Span-145" style="font-family: MathJax_Size1; vertical-align: 0em;">∑</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.367em, 1000.44em, 4.154em, -1000em); top: -4.479em; left: 1.056em;"><span class="mrow" id="MathJax-Span-146"><span class="mi" id="MathJax-Span-147" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">k</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.386em, 1001.22em, 4.162em, -1000em); top: -3.717em; left: 1.056em;"><span class="mrow" id="MathJax-Span-148"><span class="mi" id="MathJax-Span-149" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span><span class="mo MathClass-rel" id="MathJax-Span-150" style="font-size: 70.7%; font-family: MathJax_Main;">=</span><span class="mn" id="MathJax-Span-151" style="font-size: 70.7%; font-family: MathJax_Main;">0</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span><span class="msub" id="MathJax-Span-152" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.875em; height: 0px;"><span style="position: absolute; clip: rect(3.249em, 1000.49em, 4.158em, -1000em); top: -4.002em; left: 0em;"><span class="mrow" id="MathJax-Span-153"><span class="mover" id="MathJax-Span-154"><span style="display: inline-block; position: relative; width: 0.556em; height: 0px;"><span style="position: absolute; clip: rect(3.415em, 1000.43em, 4.158em, -1000em); top: -4.002em; left: 0.034em;"><span class="mrow" id="MathJax-Span-155"><span class="mi" id="MathJax-Span-156" style="font-family: MathJax_Math; font-style: italic;">c</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; clip: rect(3.267em, 1000.43em, 3.603em, -1000em); top: -4.02em; left: 0.056em;"><span class="mo MathClass-op" id="MathJax-Span-157" style=""><span style="font-family: MathJax_Main;">¯</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span><span style="position: absolute; top: -3.852em; left: 0.556em;"><span class="mrow" id="MathJax-Span-158"><span class="mi" id="MathJax-Span-159" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">i</span></span><span style="display: inline-block; width: 0px; height: 4.002em;"></span></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-7"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <mi>c</mi>
                            <mo class="MathClass-rel">≠</mo>
                            <msubsup>
                              <mrow>
                                <mo class="MathClass-op">∑</mo>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                                <mo class="MathClass-rel">=</mo>
                                <mn>0</mn>
                              </mrow>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                            </msubsup>
                            <msub>
                              <mrow>
                                <mover accent="true">
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-op">¯</mo>
                                </mover>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow></semantics></math></script></span>. The overhead, or deficit for that matter, of clusters, that is given by <span class="InlineEquation" id="IEq8"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-8-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-8"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <mi mathvariant="normal">Δ</mi>
                            <mi>c</mi>
                            <mo class="MathClass-rel">=</mo>
                            <mi>c</mi>
                            <mo class="MathClass-bin">-</mo>
                            <msubsup>
                              <mrow>
                                <mo class="MathClass-op">∑</mo>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                                <mo class="MathClass-rel">=</mo>
                                <mn>0</mn>
                              </mrow>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                            </msubsup>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow></semantics></math></script></span>, is
 then proportionally distributed across all remaining partial matrices. 
This process is repeated iteratively until no overhead (or deficit) 
remains. In each iteration, we reassign <span class="InlineEquation" id="IEq9"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-9-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-9"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                            <mo class="MathClass-punc">:</mo>
                            <mo class="MathClass-rel">=</mo>
                            <mfenced separators="" open="⌈" close="⌉">
                              <mrow>
                                <msub>
                                  <mrow>
                                    <mover accent="true">
                                      <mrow>
                                        <mi>c</mi>
                                      </mrow>
                                      <mo class="MathClass-op">¯</mo>
                                    </mover>
                                  </mrow>
                                  <mrow>
                                    <mi>i</mi>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-bin">+</mo>
                                <mi mathvariant="normal">Δ</mi>
                                <mi>c</mi>
                                <mo class="MathClass-bin">⋅</mo>
                                <mfrac>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mover accent="true">
                                          <mrow>
                                            <mi>c</mi>
                                          </mrow>
                                          <mo class="MathClass-op">¯</mo>
                                        </mover>
                                      </mrow>
                                      <mrow>
                                        <mi>i</mi>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                </mfrac>
                              </mrow>
                            </mfenced>
                          </mrow></semantics></math></script></span> until <span class="InlineEquation" id="IEq10"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-10-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-10"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                            <mi>c</mi>
                            <mo class="MathClass-rel">=</mo>
                            <msubsup>
                              <mrow>
                                <mo class="MathClass-op">∑</mo>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                                <mo class="MathClass-rel">=</mo>
                                <mn>0</mn>
                              </mrow>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                            </msubsup>
                            <msub>
                              <mrow>
                                <mi>c</mi>
                              </mrow>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                            </msub>
                          </mrow></semantics></math></script></span> for every <em xmlns="" class="EmphasisTypeItalic">i</em>.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Then, for each subtree <em xmlns="" class="EmphasisTypeItalic">i</em> = 1...<em xmlns="" class="EmphasisTypeItalic">k</em> we proceed as follows:</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For all tips in subtree <em xmlns="" class="EmphasisTypeItalic">i</em>, calculate the patristic distances to all other tips in this subtree and save them in the respective distance matrix.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Apply pairwise average clustering to generate a hierarchical tree of joins from the distance matrix.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Cut the tree, such that exactly <em xmlns="" class="EmphasisTypeItalic">c</em><sub>
                        <em xmlns="" class="EmphasisTypeItalic">i</em>
                      </sub> clusters are generated.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Add those clusters to a global list of clusters. Maintain a list that keeps track to which cluster a tip belongs.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">When all subtrees have been processed, we have a list of <em xmlns="" class="EmphasisTypeItalic">c</em> clusters. Note that, each cluster contains <em xmlns="" class="EmphasisTypeItalic">x</em> tips, where 1 ≤ <em xmlns="" class="EmphasisTypeItalic">x</em> ≤ <em xmlns="" class="EmphasisTypeItalic">m</em>
 and that each tip is assigned to exactly one cluster. The step to build
 the backbone from the clusters is not trivial. We use labels (<span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span>, <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span> and <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span>) to identify which nodes belong to the backbone and which ones do not.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The backbone tree is defined by nodes marked as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span> and <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span>. Once the clusters have been computed, we build the backbone as follows: Initially, we label each inner node in the tree as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span>, tip nodes which belong to clusters of size one as <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span>, and all remaining terminal nodes as <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span>.
 In addition, we maintain a list for storing the cluster identifiers of 
ancestral nodes that will not form part of the backbone.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Once
 this is done, we update/adapt the backbone assignment for ancestral 
nodes: The nodes of the comprehensive tree that represent the <em xmlns="" class="EmphasisTypeItalic">k</em> subtree roots will remain inside the backbone. On each of the <em xmlns="" class="EmphasisTypeItalic">k</em>
 subtree roots, we initiate a post-order traversal to relabel the 
ancestral nodes, if required, according the following rule set:</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">If the two child nodes are labeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span> or <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span>, the ancestral node remains labeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span>.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">If one child is labeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span> or <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span> and the other child as <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span>, the ancestral node is relabeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span> and the <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span> child node is relabeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span>.</p>
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">If both children are labeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span>, we need to check to which cluster they belong. If they belong to the same cluster, the parent node is labeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span>
 and the shared cluster identifier of the child nodes is propagated to 
the parent node. If the two children do not belong to the same cluster, 
the parent node is labeled as inside and both children are relabeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span>.</p>
                    <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">When the post-order traversal is about to be completed, we arrive at the subtree root <em xmlns="" class="EmphasisTypeItalic">i</em> again, which was originally labelled as <span xmlns="" class="EmphasisFontCategoryNonProportional">inside</span>. At this point, we check whether the adjacent backbone node of the subtree root <em xmlns="" class="EmphasisTypeItalic">i</em> has been labeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">outside</span>. Whenever this is the case (see Figure <span class="InternalRef"><a href="#Fig1">1</a></span> for an example), the adjacent backbone node is relabeled as <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span> for consistency.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                          <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig1_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2011_Article_5012_Fig1_HTML.jpg" alt="Figure 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig1_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig1_HTML.jpg" alt="Figure 1"></noscript></a>
                        </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                            <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Consistency of labels at the backbone boundaries</strong>.
 At first (left) an initial backbone exists (thick branches), all inner 
nodes are labelled as inside (I) and each tip node has a cluster id. 
After completion of the post-order traversal (right), each inner node 
has been relabelled accordingly, if required. Here, cluster 2 is 
monophyletic, hence the cluster id was inherited propagated back to the 
initial backbone node. This produced a branch (edge) with an inside and 
an outside node; therefore the outside(O) node is relabelled (arrow) as 
boundary(B) node.</p>
                          </div></figcaption></figure></div>
                    <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Given
 a set of tips that form part of the same cluster, it may occur that 
these tips also form a monophyletic group. In this case, during the 
postorder traversal, all ancestral nodes will be grouped together under 
the same cluster identifier and the common ancestral node will become a 
backbone <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span> (virtual tip). However, if the tips in a cluster are not monophyletic (see for instance, in Figure <span class="InternalRef"><a href="#Fig2">2</a></span>), the application of the above rules requires some additional <span xmlns="" class="EmphasisFontCategoryNonProportional">boundary</span> relabelling.<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                          <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig2_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2011_Article_5012_Fig2_HTML.jpg" alt="Figure 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig2_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig2_HTML.jpg" alt="Figure 2"></noscript></a>
                        </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 2</span>
                            <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Increase of in backbone tips due to topology conflicts</strong>.
 At first (left) an initial backbone exists (thick branches), all inner 
nodes are labelled as inside (I) and each tip node has a cluster id. 
Upon completion of the post-order traversal (right), each inner node has
 been relabelled accordingly. Here, cluster 2 is not monophyletic. 
Hence, an additional virtual tip is created, that is, cluster 2 
generates 2 boundary tips.</p>
                          </div></figcaption></figure></div>
                    <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Based
 on the prolegomena, a single cluster may thus induce more than a single
 virtual tip. As a consequence, the number of virtual tips may actually 
be higher than the number of clusters. In turn, the reduction of tree 
size that can be achieved will be smaller than specified by <em xmlns="" class="EmphasisTypeItalic">R</em>.
 The impact and frequency of occurrence of this phenomenon 
(non-monophyletic clusters) depends on the shape of the tree and the 
branch lengths. In Table <span class="InternalRef"><a href="#Tab1">1</a></span>,
 we outline this effect for trees with 37,831 and 55,593 taxa. We 
computed the average number of virtual tips generated by our algorithm 
on 10 distinct trees per dataset and reduction factors of 0.25 and 0.5 
respectively.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                            <p xmlns="" class="SimplePara">Average number of computed backbone tips over 10 distinct trees</p>
                          </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="3"><p xmlns="" class="SimplePara">Average number of computed backbone tips</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Reduction Factor R</p></td><td><p xmlns="" class="SimplePara">37831 (expected)</p></td><td><p xmlns="" class="SimplePara">55593 (expected)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">0.25</p></td><td><p xmlns="" class="SimplePara">12668.0 (9457.75)</p></td><td><p xmlns="" class="SimplePara">19366.7 (13898.25)</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">0.50</p></td><td><p xmlns="" class="SimplePara">22340.0 (18915.5)</p></td><td><p xmlns="" class="SimplePara">33501.5 (27796.5)</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                          <p xmlns="" class="SimplePara">The average number of backbone tips is higher than the expected number <em class="EmphasisTypeItalic">n</em>·<em class="EmphasisTypeItalic">R</em></p>
                        </div></figure></div>
                  </section>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Tree Searches on the Backbone</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We have implemented the above algorithm in a dedicated RAxML version that is available for download at <span class="ExternalRef"><a href="http://wwwkramer.in.tum.de/exelixis/software/BackboneSearch.zip" target="_blank" class="ext-link"><span class="RefSource">http://wwwkramer.in.tum.de/exelixis/software/BackboneSearch.zip</span></a></span>.
 Initially, RAxML will generate a comprehensive randomized stepwise 
addition order parsimony tree, or read in a user specified tree via <span xmlns="" class="EmphasisFontCategoryNonProportional">-t</span>.
 Then it will optimize ML model parameters--including branch lengths--on
 the comprehensive tree. Thereafter, it will execute the backbone 
algorithm as described above. The tree searches on the backbone are 
based on the standard RAxML hill-climbing algorithm. Lazy SPR moves are 
only conducted within the backbone. After each cycle of SPR moves (see [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]
 for details), the backbone tree will be re-computed based on the 
currently best tree. Also, the branch lengths of the entire tree 
(including those branches not forming part of the backbone) will be 
re-optimized once after each SPR cycle.</p>
                </section>
              </section><section id="Sec8" class="Section2 RenderAsSection2">
                <h3 class="Heading">Subtree Equality Vectors Re-Visited</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 introduced and implemented the concept of Subtree Equality Vectors 
(SEVs) to accelerate likelihood computations by reducing the number of 
required floating point operations in 2002 [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. Conceptually similar approaches were presented in 2004 [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] and 2010 [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The underlying idea is based on the following observation: Given two identical alignment sites <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em> that evolve under the same evolutionary model (GTR parameters, <em xmlns="" class="EmphasisTypeItalic">α</em>
 shape parameter of the Γ function, etc.) and for which a joint branch 
length has been estimated, their per-site log likelihoods <em xmlns="" class="EmphasisTypeItalic">LnL</em>(<em xmlns="" class="EmphasisTypeItalic">i</em>) and <em xmlns="" class="EmphasisTypeItalic">LnL</em>(<em xmlns="" class="EmphasisTypeItalic">j</em>)
 will be identical. Hence, to save computations, one can compress the 
identical sites into a single site pattern and assign a respective site 
pattern count (weight) to this site pattern. Thus, for two identical 
sites <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em>, we can compute the per-site log likelihood as 2·<em xmlns="" class="EmphasisTypeItalic">LnL</em>(<em xmlns="" class="EmphasisTypeItalic">i</em>).
 This global compression of alignments (executed prior to conducting 
likelihood computations) is implemented in all current likelihood-based 
codes.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">This
 basic idea of site compression can be extended to the subtree level, by
 using SEVs for instance, to save additional computations. Consider the 
equation [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>] for computing the ancestral probability vector entry for observing nucleotide A at site <em xmlns="" class="EmphasisTypeItalic">i</em> of a parent node node <em xmlns="" class="EmphasisTypeItalic">p</em>, with two child nodes <em xmlns="" class="EmphasisTypeItalic">l</em> and <em xmlns="" class="EmphasisTypeItalic">r</em> given the respective branch lengths <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">l</em>
                  </sub> and <em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">r</em>
                  </sub> and transition probability matrices <em xmlns="" class="EmphasisTypeItalic">P</em>(<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">l</em>
                  </sub> ) and <em xmlns="" class="EmphasisTypeItalic">P</em>(<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">r</em>
                  </sub> ):<div id="Equ1" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-11-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-11"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msubsup>
                            <mrow>
                              <mover accent="true">
                                <mrow>
                                  <mi>L</mi>
                                </mrow>
                                <mo class="MathClass-op">→</mo>
                              </mover>
                            </mrow>
                            <mrow>
                              <mi>A</mi>
                            </mrow>
                            <mrow>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <mi>p</mi>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                            </mrow>
                          </msubsup>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mi>i</mi>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                          <mo class="MathClass-rel">=</mo>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <munderover accentunder="false" accent="false">
                                <mrow>
                                  <mo mathsize="big">∑</mo>
                                </mrow>
                                <mrow>
                                  <mi>S</mi>
                                  <mo class="MathClass-rel">=</mo>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mi>T</mi>
                                </mrow>
                              </munderover>
                              <msub>
                                <mrow>
                                  <mi>P</mi>
                                </mrow>
                                <mrow>
                                  <mi>A</mi>
                                  <mi>S</mi>
                                </mrow>
                              </msub>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>l</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                              <msubsup>
                                <mrow>
                                  <mover accent="true">
                                    <mrow>
                                      <mi>L</mi>
                                    </mrow>
                                    <mo>→</mo>
                                  </mover>
                                </mrow>
                                <mrow>
                                  <mi>S</mi>
                                </mrow>
                                <mrow>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <mi>l</mi>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                </mrow>
                              </msubsup>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <mi>i</mi>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <munderover accentunder="false" accent="false">
                                <mrow>
                                  <mo mathsize="big">∑</mo>
                                </mrow>
                                <mrow>
                                  <mi>S</mi>
                                  <mo class="MathClass-rel">=</mo>
                                  <mi>A</mi>
                                </mrow>
                                <mrow>
                                  <mi>T</mi>
                                </mrow>
                              </munderover>
                              <msub>
                                <mrow>
                                  <mi>P</mi>
                                </mrow>
                                <mrow>
                                  <mi>A</mi>
                                  <mi>S</mi>
                                </mrow>
                              </msub>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>b</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>r</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                              <msubsup>
                                <mrow>
                                  <mover accent="true">
                                    <mrow>
                                      <mi>L</mi>
                                    </mrow>
                                    <mo>→</mo>
                                  </mover>
                                </mrow>
                                <mrow>
                                  <mi>S</mi>
                                </mrow>
                                <mrow>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <mi>r</mi>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                </mrow>
                              </msubsup>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <mi>i</mi>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(1)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We observe that, if the site patterns in the subtree (and hence sub-alignment) rooted at <em xmlns="" class="EmphasisTypeItalic">p</em> at sites <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em> are identical, <em xmlns="" class="EmphasisTypeItalic">and</em> if the transition probability matrices <em xmlns="" class="EmphasisTypeItalic">P</em>(<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">l</em>
                  </sub> ) and <em xmlns="" class="EmphasisTypeItalic">P</em>(<em xmlns="" class="EmphasisTypeItalic">b</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">r</em>
                  </sub> ) are identical at sites <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em> (implying identical branch lengths and model parameters at sites <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em>), then <span class="InlineEquation" id="IEq11"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-12-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-12"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msubsup>
                          <mrow>
                            <mi>L</mi>
                          </mrow>
                          <mrow>
                            <mi>S</mi>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>p</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msubsup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>i</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                        <mo class="MathClass-rel">=</mo>
                        <msubsup>
                          <mrow>
                            <mi>L</mi>
                          </mrow>
                          <mrow>
                            <mi>S</mi>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>p</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msubsup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>j</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                      </mrow></semantics></math></script></span> for all states <em xmlns="" class="EmphasisTypeItalic">S</em> (e.g., <span xmlns="" class="EmphasisFontCategoryNonProportional">A, C, G, T</span>). Thus, we can avoid re-computing all ancestral states for site <em xmlns="" class="EmphasisTypeItalic">j</em> if we have already computed the ancestral states for site <em xmlns="" class="EmphasisTypeItalic">i</em>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 key technical challenge with this approach is that it requires a large 
amount of bookkeeping, to keep track of identical subtree site patterns 
(for details see [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]).
 Moreover, SEVs require additional data structures and a case switch in 
the innermost loop of the likelihood function that iterates over the 
sites of the ancestral probability vectors, which may lead to cache 
misses and incorrectly predicted conditional jumps by the processor 
hardware. Because of these observations we had abandoned this approach 
completely in RAxML.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">However,
 the advent of gappy phylogenomic alignments, that is alignments that 
contain a large amount of structured (non-randomly distributed across 
the alignment) missing data regions per gene for the taxa under study, 
motivated us to re-assess SEVs in a simpler and thus more efficient 
setting.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Gaps
 and undetermined characters are mathematically equivalent in the 
standard ML framework. Since structured patches of missing data dominate
 current phylogenomic datasets (typically the amount of missing data 
ranges between 50% to 90%), we only track subtree site patterns that 
entirely consist of gaps/undetermined characters (e.g., we are only 
interested in subtree sites of the from: ---- in a subtree of size 4). 
Thereby, we avoid the more complex task (see [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>] and [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]) of tracking <em xmlns="" class="EmphasisTypeItalic">all</em> identical subtree site patterns (e.g., detecting all sites of the form: <span xmlns="" class="EmphasisFontCategoryNonProportional">ACCT</span>
 in a subtree of size 4). This restriction simplifies the required 
bookkeeping procedure and data structure significantly, because we only 
need to know whether a subtree site consists entirely of gaps or not. 
Thus, given an alignment with <em xmlns="" class="EmphasisTypeItalic">n</em> sites, it suffices to enhance the data structures for storing tips and inner nodes by a simple bit vector with <em xmlns="" class="EmphasisTypeItalic">n</em> bits. If all-gap sites are represented by 1 and non-gap sites by 0, we simply need to execute a bit-wise <span xmlns="" class="EmphasisFontCategoryNonProportional">AND</span> on the respective bit vectors of the child nodes <em xmlns="" class="EmphasisTypeItalic">l</em> and <em xmlns="" class="EmphasisTypeItalic">r</em> in conjunction with the tree traversal for computing the likelihood to determine the all-gap sites at the ancestral node <em xmlns="" class="EmphasisTypeItalic">p</em> (see Figure <span class="InternalRef"><a href="#Fig3">3</a></span>). We can then use this bit vector at <em xmlns="" class="EmphasisTypeItalic">p</em> to determine if we need to compute something at a site <em xmlns="" class="EmphasisTypeItalic">i</em> or not.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig3_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2011_Article_5012_Fig3_HTML.jpg" alt="Figure 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig3_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig3_HTML.jpg" alt="Figure 3"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 3</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Using Subtree Equality Vectors to save computations for all-gap alignment sites in subtrees</strong>.</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 have implemented this method for DNA and protein data under the Γ model
 of rate heterogeneity in RAxML v728 (alpha) available at <span class="ExternalRef"><a href="http://wwwkramer.in.tum.de/exelixis/software.html" target="_blank" class="ext-link"><span class="RefSource">http://wwwkramer.in.tum.de/exelixis/software.html</span></a></span>.
 Evidently, the efficiency of this approach depends on the proportion of
 gaps/missing data and the distribution of gaps in the input alignment. 
Since areas of missing data are typically well-structured in current 
phylogenomic datasets, this approach is expected to work well with this 
kind of input data. To facilitate the deployment of the SEV-based 
version of the likelihood function, we have integrated an automatic 
performance test that decides whether to use the SEV-based or the 
standard likelihood implementation. When the starting tree has been 
computed or parsed by RAxML, the program will execute a full tree 
traversal (re-compute all ancestral probability vectors) for the 
standard <em xmlns="" class="EmphasisTypeItalic">and</em> the SEV-based 
likelihood function implementation and measure the respective execution 
times. If the execution time of the SEV-based approach is 20% smaller 
than that of the standard implementation, RAxML will automatically use 
the SEV-based implementation for all subsequent likelihood computations.
 The threshold of 20% is based on empirical observations. While SEVs can
 speed-up ancestral probability vector computations, SEVs slightly slow 
down the branch length optimization and likelihood computation (at the 
root) functions because of the memory accesses to the bit vectors.</p>
                <section id="Sec9" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Saving Memory with SEVs</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">SEVs as implemented here, can also be deployed to reduce memory requirements. As mentioned above, if, at an ancestral node <em xmlns="" class="EmphasisTypeItalic">p</em>
 we encounter an all-gap site, we completely omit its computation. In 
order to accomplish this, we need to maintain only one additional 
ancestral probability vector site, that contains the signal for all-gap 
sites. Consider an ancestral probability vector where 50% of the entries
 in the all-gap site bit-vector are set to 1, that is, where we only 
need to compute 50% of the ancestral probability vector entries with 
respect to the total alignment length.</p>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 can observe that, in addition to saving 50% of the computations 
required for this ancestral probability vector, we can also save 50% of 
the memory space required for storing the ancestral probability vector 
(see Figure <span class="InternalRef"><a href="#Fig4">4</a></span>). 
Thus, the memory requirements for each ancestral node can be determined 
on-the-fly as we traverse the tree, by subtracting the number of entries
 that are set to 1 in the bit vector from the input alignment length. 
Remember that, the bit vectors we deploy are always as long as the input
 alignment.<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig4_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2011_Article_5012_Fig4_HTML.jpg" alt="Figure 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig4_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig4_HTML.jpg" alt="Figure 4"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 4</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Using Subtree Equality Vectors to save computations and memory for all-gap alignment sites in subtrees</strong>.</p>
                        </div></figcaption></figure></div>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 key technical problem that arises is that, the required ancestral 
probability vector lengths at inner nodes will change dynamically when 
the tree topology changes or even when the tree is just re-rooted. Given
 a rooting of the tree, one may think of this as ancestral probability 
vectors becoming longer while one approaches the root of the tree. At 
present we have implemented this by dynamically freeing and allocating 
memory (using <span xmlns="" class="EmphasisFontCategoryNonProportional">free()</span> and <span xmlns="" class="EmphasisFontCategoryNonProportional">malloc()</span>)
 at each ancestral node. The reallocation only takes place when the 
all-gap bit-vector count (number of bits set to 1) corresponding to the 
required ancestral probability vector does not equal the all-gap 
bit-vector count of the current ancestral probability vector at an 
ancestral node.</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Note
 that, the concepts presented here can also be applied to phylogenomic 
datasets with joint branch length estimates across partitions, while the
 conceptually different ideas presented in [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>] can only be applied to partitioned phylogenomic datasets with per-partition branch length estimates.</p>
                </section>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Numerical Problems of the Γ Model of Rate Heterogeneity</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Numerical
 scaling of the entries in the ancestral (inner) probability vectors 
during likelihood computations on trees, for avoiding numerical 
underflow has become a standard technique that is implemented in most 
likelihood-based programs (PHYML, Mr-Bayes, GARLI, RAxML, BEAST, etc.). 
For an overview of numerical scaling techniques, please refer to [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>].
 A numerical problem that arises for very large trees with more than 
approximately 50,000 taxa in RAxML (and probably all other 
likelihood-based programs as well) is associated with the widely used [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>] Γ model of rate heterogeneity [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>].</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 the Γ model, a discrete approximation (typically using 4 discrete 
rates) is used to approximate the integral of the likelihood over the Γ 
curve at each site. That is, instead of computing the ancestral 
probabilities <em xmlns="" class="EmphasisTypeItalic">L</em>(<em xmlns="" class="EmphasisTypeItalic">A</em>), <em xmlns="" class="EmphasisTypeItalic">L</em>(<em xmlns="" class="EmphasisTypeItalic">C</em>), <em xmlns="" class="EmphasisTypeItalic">L</em>(<em xmlns="" class="EmphasisTypeItalic">G</em>), <em xmlns="" class="EmphasisTypeItalic">L</em>(<em xmlns="" class="EmphasisTypeItalic">T</em>) for the 4 nucleotides <span xmlns="" class="EmphasisFontCategoryNonProportional">A, C, G, T</span> at a specific site of an ancestral node in the tree (see Equation <span class="InternalRef"><a href="#Equ1">1</a></span>), one needs to compute those probabilities for all 4 discrete Γ rates <em xmlns="" class="EmphasisTypeItalic">r</em><sub>0</sub>, <em xmlns="" class="EmphasisTypeItalic">r</em><sub>1</sub>, <em xmlns="" class="EmphasisTypeItalic">r</em><sub>2</sub>, <em xmlns="" class="EmphasisTypeItalic">r</em><sub>3</sub>. Thus, every site of an ancestral probability vector comprises 16 values:<div id="Equ2" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-13-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-13"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mtable class="gathered">
                            <mtr>
                              <mtd>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>A</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>0</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>C</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>0</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>G</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>0</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>T</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>0</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                              </mtd>
                            </mtr>
                            <mtr>
                              <mtd>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>A</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>3</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>C</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>3</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>G</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>3</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>T</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>3</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mo class="MathClass-punc">.</mo>
                              </mtd>
                            </mtr>
                          </mtable>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(2)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 numerical problem that arises with the Γ model on very large trees is 
that those 16 values need to be jointly scaled numerically (all 16 
values are multiplied by a large number) to avoid numerical underflow 
(see below).</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Scaling of the probability vector entries may be conducted as follows: At a specific site <em xmlns="" class="EmphasisTypeItalic">c</em> of an ancestral probability vector for DNA data <span class="InlineEquation" id="IEq12"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-14-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-14"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mover accent="true">
                          <mrow>
                            <mi>L</mi>
                          </mrow>
                          <mo class="MathClass-op">→</mo>
                        </mover>
                      </mrow></semantics></math></script></span> we scale the entries if, for instance,<div id="Equ3" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-15-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-15"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mtable class="gathered">
                            <mtr>
                              <mtd>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>A</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>0</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mo class="MathClass-rel">&lt;</mo>
                                <mi>ε</mi>
                                <mo class="MathClass-bin">∧</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>C</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>0</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mo class="MathClass-rel">&lt;</mo>
                                <mi>ε</mi>
                                <mo class="MathClass-punc">,</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                              </mtd>
                            </mtr>
                            <mtr>
                              <mtd>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">,</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>G</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>3</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mo class="MathClass-rel">&lt;</mo>
                                <mi>ε</mi>
                                <mo class="MathClass-bin">∧</mo>
                                <mi>L</mi>
                                <msub>
                                  <mrow>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>T</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mrow>
                                    <msub>
                                      <mrow>
                                        <mi>r</mi>
                                      </mrow>
                                      <mrow>
                                        <mn>3</mn>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                </msub>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mi>c</mi>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mo class="MathClass-rel">&lt;</mo>
                                <mi>ε</mi>
                              </mtd>
                            </mtr>
                          </mtable>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(3)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">where <em xmlns="" class="EmphasisTypeItalic">ε</em> can be set to <em xmlns="" class="EmphasisTypeItalic">ε</em> := 1/2<sup>256</sup> under double precision arithmetics. Thus, we decide to scale up all ancestral probability vector entries at a site <em xmlns="" class="EmphasisTypeItalic">c</em>, when <em xmlns="" class="EmphasisTypeItalic">all</em> unscaled entries for <em xmlns="" class="EmphasisTypeItalic">all</em> discrete rates are smaller than some pre-defined <em xmlns="" class="EmphasisTypeItalic">ε</em>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Other
 options for scaling exist. For instance, one calculates a scaling 
factor such that the largest of the 16 ancestral probability values at a
 site is scaled to 1.0. One can also conduct this type of scaling at 
every ancestral probability vector without checking that all values are 
smaller than some <em xmlns="" class="EmphasisTypeItalic">ε</em>. We 
also experimented with such alternative implementations for numerical 
scaling in RAxML, but were not able to solve the general scaling problem
 for the Γ model of rate heterogeneity (see below). With alternative 
scaling implementations, the fundamental numerical problem occurred 
again for slightly larger tree sizes.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">If according to Equation <span class="InternalRef"><a href="#Equ3">3</a></span> a probability vector column <em xmlns="" class="EmphasisTypeItalic">c</em> at vector <span class="InlineEquation" id="IEq13"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-16-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-16"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mover accent="true">
                          <mrow>
                            <mi>L</mi>
                          </mrow>
                          <mo class="MathClass-op">→</mo>
                        </mover>
                      </mrow></semantics></math></script></span> needs to be scaled, we simply multiply all entries<div id="Equ4" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-17-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-17"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mrow>
                              <mover class="overrightarrow">
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>L</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mo class="MathClass-op">⃗</mo>
                              </mover>
                            </mrow>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>r</mi>
                                </mrow>
                                <mrow>
                                  <mn>0</mn>
                                </mrow>
                              </msub>
                            </mrow>
                          </msub>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mi>c</mi>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                          <mo class="MathClass-punc">,</mo>
                          <msub>
                            <mrow>
                              <mover class="overrightarrow">
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>L</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>C</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mo class="MathClass-op">⃗</mo>
                              </mover>
                            </mrow>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>r</mi>
                                </mrow>
                                <mrow>
                                  <mn>0</mn>
                                </mrow>
                              </msub>
                            </mrow>
                          </msub>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mi>c</mi>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                          <mo class="MathClass-punc">,</mo>
                          <mo class="MathClass-punc">.</mo>
                          <mo class="MathClass-punc">.</mo>
                          <mo class="MathClass-punc">.</mo>
                          <mo class="MathClass-punc">,</mo>
                          <msub>
                            <mrow>
                              <mover class="overrightarrow">
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>L</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>G</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mo class="MathClass-op">⃗</mo>
                              </mover>
                            </mrow>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>r</mi>
                                </mrow>
                                <mrow>
                                  <mn>3</mn>
                                </mrow>
                              </msub>
                            </mrow>
                          </msub>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mi>c</mi>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                          <mo class="MathClass-punc">,</mo>
                          <msub>
                            <mrow>
                              <mover class="overrightarrow">
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>L</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>T</mi>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mo class="MathClass-op">⃗</mo>
                              </mover>
                            </mrow>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>r</mi>
                                </mrow>
                                <mrow>
                                  <mn>3</mn>
                                </mrow>
                              </msub>
                            </mrow>
                          </msub>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mi>c</mi>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(4)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">by 2<sup>256</sup>.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 order to correct (undo) the scaling multiplications (accumulated during
 a tree traversal) at the virtual root, we need to keep track of the 
total number of scaling operations conducted per column. For this, we 
use integer vectors <span class="InlineEquation" id="IEq14"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-18-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-18"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mover accent="true">
                          <mrow>
                            <mi>U</mi>
                          </mrow>
                          <mo class="MathClass-op">→</mo>
                        </mover>
                      </mrow></semantics></math></script></span> that 
maintain the scaling events and correspond to the respective probability
 vectors at inner nodes. As we traverse the tree to compute an ancestral
 vector <span class="InlineEquation" id="IEq15"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-19-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-19"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>L</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span> from two child vectors <span class="InlineEquation" id="IEq16"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-20-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-20"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>L</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span> and <span class="InlineEquation" id="IEq17"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-21-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-21"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>L</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>j</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span> the scaling vector is initially updated as follows <span class="InlineEquation" id="IEq18"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-22-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-22"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                        <mo class="MathClass-punc">:</mo>
                        <mo class="MathClass-rel">=</mo>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                        <mo class="MathClass-bin">+</mo>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>j</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                      </mrow></semantics></math></script></span>. Then, if an entry of <span class="InlineEquation" id="IEq19"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-23-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-23"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>L</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span> needs to be scaled at position <em xmlns="" class="EmphasisTypeItalic">c</em> we increment <span class="InlineEquation" id="IEq20"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-24-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-24"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                        <mo class="MathClass-punc">:</mo>
                        <mo class="MathClass-rel">=</mo>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>k</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                        <mrow>
                          <mo class="MathClass-open">(</mo>
                          <mrow>
                            <mi>c</mi>
                          </mrow>
                          <mo class="MathClass-close">)</mo>
                        </mrow>
                        <mo class="MathClass-bin">+</mo>
                        <mn>1</mn>
                      </mrow></semantics></math></script></span>. The scaling vectors at the tips of the tree are not allocated, but implicitly initialized with 0.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">At the virtual root, given <span class="InlineEquation" id="IEq21"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-25-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-25"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>L</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span>, <span class="InlineEquation" id="IEq22"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-26-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-26"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>L</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>j</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span> and the corresponding scaling vectors <span class="InlineEquation" id="IEq23"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-27-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-27"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>i</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span>, <span class="InlineEquation" id="IEq24"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-28-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-28"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <msup>
                          <mrow>
                            <mover accent="true">
                              <mrow>
                                <mi>U</mi>
                              </mrow>
                              <mo class="MathClass-op">→</mo>
                            </mover>
                          </mrow>
                          <mrow>
                            <mrow>
                              <mo class="MathClass-open">(</mo>
                              <mrow>
                                <mi>j</mi>
                              </mrow>
                              <mo class="MathClass-close">)</mo>
                            </mrow>
                          </mrow>
                        </msup>
                      </mrow></semantics></math></script></span>, we can compute the likelihood under the Γ model as follows:<div id="Equ5" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-29-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-29"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mi>l</mi>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mi>c</mi>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                          <mo class="MathClass-rel">=</mo>
                          <msup>
                            <mrow>
                              <mi>ε</mi>
                            </mrow>
                            <mrow>
                              <msup>
                                <mrow>
                                  <mi>U</mi>
                                </mrow>
                                <mrow>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <mi>i</mi>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                </mrow>
                              </msup>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <mi>c</mi>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                              <mo class="MathClass-bin">+</mo>
                              <msup>
                                <mrow>
                                  <mi>U</mi>
                                </mrow>
                                <mrow>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <mi>j</mi>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                </mrow>
                              </msup>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <mi>c</mi>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                            </mrow>
                          </msup>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <mfrac>
                                <mrow>
                                  <mn>1</mn>
                                </mrow>
                                <mrow>
                                  <mn>4</mn>
                                </mrow>
                              </mfrac>
                              <mo class="MathClass-bin">⋅</mo>
                              <msub>
                                <mrow>
                                  <mi>Q</mi>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>r</mi>
                                    </mrow>
                                    <mrow>
                                      <mn>0</mn>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </msub>
                              <mo class="MathClass-bin">+</mo>
                              <mfrac>
                                <mrow>
                                  <mn>1</mn>
                                </mrow>
                                <mrow>
                                  <mn>4</mn>
                                </mrow>
                              </mfrac>
                              <mo class="MathClass-bin">⋅</mo>
                              <msub>
                                <mrow>
                                  <mi>Q</mi>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>r</mi>
                                    </mrow>
                                    <mrow>
                                      <mn>1</mn>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </msub>
                              <mo class="MathClass-bin">+</mo>
                              <mfrac>
                                <mrow>
                                  <mn>1</mn>
                                </mrow>
                                <mrow>
                                  <mn>4</mn>
                                </mrow>
                              </mfrac>
                              <mo class="MathClass-bin">⋅</mo>
                              <msub>
                                <mrow>
                                  <mi>Q</mi>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>r</mi>
                                    </mrow>
                                    <mrow>
                                      <mn>2</mn>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </msub>
                              <mo class="MathClass-bin">+</mo>
                              <mfrac>
                                <mrow>
                                  <mn>1</mn>
                                </mrow>
                                <mrow>
                                  <mn>4</mn>
                                </mrow>
                              </mfrac>
                              <mo class="MathClass-bin">⋅</mo>
                              <msub>
                                <mrow>
                                  <mi>Q</mi>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>r</mi>
                                    </mrow>
                                    <mrow>
                                      <mn>3</mn>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </msub>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(5)</div></div></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">where, for instance,<div id="Equ6" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-30-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-30"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <msub>
                            <mrow>
                              <mi>Q</mi>
                            </mrow>
                            <mrow>
                              <msub>
                                <mrow>
                                  <mi>r</mi>
                                </mrow>
                                <mrow>
                                  <mn>0</mn>
                                </mrow>
                              </msub>
                            </mrow>
                          </msub>
                          <mo class="MathClass-punc">:</mo>
                          <mo class="MathClass-rel">=</mo>
                          <mrow>
                            <mo class="MathClass-open">(</mo>
                            <mrow>
                              <munderover accentunder="false" accent="false">
                                <mrow>
                                  <mo mathsize="big">∑</mo>
                                </mrow>
                                <mrow>
                                  <mi>R</mi>
                                  <mo class="MathClass-rel">=</mo>
                                  <msub>
                                    <mrow>
                                      <mi>A</mi>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>r</mi>
                                        </mrow>
                                        <mrow>
                                          <mn>0</mn>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </msub>
                                </mrow>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>T</mi>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>r</mi>
                                        </mrow>
                                        <mrow>
                                          <mn>0</mn>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </msub>
                                </mrow>
                              </munderover>
                              <mrow>
                                <mo class="MathClass-open">(</mo>
                                <mrow>
                                  <msub>
                                    <mrow>
                                      <mi>π</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                    </mrow>
                                  </msub>
                                  <msubsup>
                                    <mrow>
                                      <mover accent="true">
                                        <mrow>
                                          <mi>L</mi>
                                        </mrow>
                                        <mo>→</mo>
                                      </mover>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>R</mi>
                                        </mrow>
                                        <mrow>
                                          <msub>
                                            <mrow>
                                              <mi>r</mi>
                                            </mrow>
                                            <mrow>
                                              <mn>0</mn>
                                            </mrow>
                                          </msub>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                    <mrow>
                                      <mrow>
                                        <mo class="MathClass-open">(</mo>
                                        <mrow>
                                          <mi>i</mi>
                                        </mrow>
                                        <mo class="MathClass-close">)</mo>
                                      </mrow>
                                    </mrow>
                                  </msubsup>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                  <munderover accentunder="false" accent="false">
                                    <mrow>
                                      <mo mathsize="big">∑</mo>
                                    </mrow>
                                    <mrow>
                                      <mi>S</mi>
                                      <mo class="MathClass-rel">=</mo>
                                      <msub>
                                        <mrow>
                                          <mi>A</mi>
                                        </mrow>
                                        <mrow>
                                          <msub>
                                            <mrow>
                                              <mi>r</mi>
                                            </mrow>
                                            <mrow>
                                              <mn>0</mn>
                                            </mrow>
                                          </msub>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>T</mi>
                                        </mrow>
                                        <mrow>
                                          <msub>
                                            <mrow>
                                              <mi>r</mi>
                                            </mrow>
                                            <mrow>
                                              <mn>0</mn>
                                            </mrow>
                                          </msub>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                  </munderover>
                                  <msub>
                                    <mrow>
                                      <mi>P</mi>
                                    </mrow>
                                    <mrow>
                                      <mi>R</mi>
                                      <mi>S</mi>
                                    </mrow>
                                  </msub>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>b</mi>
                                        </mrow>
                                        <mrow>
                                          <mi>v</mi>
                                          <mi>r</mi>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                  <msubsup>
                                    <mrow>
                                      <mover accent="true">
                                        <mrow>
                                          <mi>L</mi>
                                        </mrow>
                                        <mo>→</mo>
                                      </mover>
                                    </mrow>
                                    <mrow>
                                      <msub>
                                        <mrow>
                                          <mi>S</mi>
                                        </mrow>
                                        <mrow>
                                          <msub>
                                            <mrow>
                                              <mi>r</mi>
                                            </mrow>
                                            <mrow>
                                              <mn>0</mn>
                                            </mrow>
                                          </msub>
                                        </mrow>
                                      </msub>
                                    </mrow>
                                    <mrow>
                                      <mrow>
                                        <mo class="MathClass-open">(</mo>
                                        <mrow>
                                          <mi>j</mi>
                                        </mrow>
                                        <mo class="MathClass-close">)</mo>
                                      </mrow>
                                    </mrow>
                                  </msubsup>
                                  <mrow>
                                    <mo class="MathClass-open">(</mo>
                                    <mrow>
                                      <mi>c</mi>
                                    </mrow>
                                    <mo class="MathClass-close">)</mo>
                                  </mrow>
                                </mrow>
                                <mo class="MathClass-close">)</mo>
                              </mrow>
                            </mrow>
                            <mo class="MathClass-close">)</mo>
                          </mrow>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(6)</div></div></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">If we take the logarithm of <em xmlns="" class="EmphasisTypeItalic">l</em>(<em xmlns="" class="EmphasisTypeItalic">c</em>) this can be rewritten as:<div id="Equ7" class="Equation EquationMathjax"><div class="EquationContent"><span class="MathJax_Preview" style="">MathML</span><span class="MathJax MathJax_Processing" id="MathJax-Element-31-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-31"><math xmlns="http://www.w3.org/1998/Math/MathML"><semantics><mrow>
                        <mrow>
                          <mtable class="gathered">
                            <mtr>
                              <mtd>
                                <mi>l</mi>
                                <mi>o</mi>
                                <mi>g</mi>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mi>l</mi>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>c</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mo class="MathClass-rel">=</mo>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <msup>
                                      <mrow>
                                        <mi>U</mi>
                                      </mrow>
                                      <mrow>
                                        <mrow>
                                          <mo class="MathClass-open">(</mo>
                                          <mrow>
                                            <mi>i</mi>
                                          </mrow>
                                          <mo class="MathClass-close">)</mo>
                                        </mrow>
                                      </mrow>
                                    </msup>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>c</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                    <mo class="MathClass-bin">+</mo>
                                    <msup>
                                      <mrow>
                                        <mi>U</mi>
                                      </mrow>
                                      <mrow>
                                        <mrow>
                                          <mo class="MathClass-open">(</mo>
                                          <mrow>
                                            <mi>j</mi>
                                          </mrow>
                                          <mo class="MathClass-close">)</mo>
                                        </mrow>
                                      </mrow>
                                    </msup>
                                    <mrow>
                                      <mo class="MathClass-open">(</mo>
                                      <mrow>
                                        <mi>c</mi>
                                      </mrow>
                                      <mo class="MathClass-close">)</mo>
                                    </mrow>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mi>l</mi>
                                <mi>o</mi>
                                <mi>g</mi>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mi>ε</mi>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                                <mo class="MathClass-bin">+</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                                <mo class="MathClass-punc">.</mo>
                              </mtd>
                            </mtr>
                            <mtr>
                              <mtd>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mspace width="1em" class="quad"></mspace>
                                <mo class="MathClass-bin">+</mo>
                                <mi>l</mi>
                                <mi>o</mi>
                                <mi>g</mi>
                                <mrow>
                                  <mo class="MathClass-open">(</mo>
                                  <mrow>
                                    <mfrac>
                                      <mrow>
                                        <mn>1</mn>
                                      </mrow>
                                      <mrow>
                                        <mn>4</mn>
                                      </mrow>
                                    </mfrac>
                                    <mo class="MathClass-bin">⋅</mo>
                                    <msub>
                                      <mrow>
                                        <mi>Q</mi>
                                      </mrow>
                                      <mrow>
                                        <msub>
                                          <mrow>
                                            <mi>r</mi>
                                          </mrow>
                                          <mrow>
                                            <mn>0</mn>
                                          </mrow>
                                        </msub>
                                      </mrow>
                                    </msub>
                                    <mo class="MathClass-bin">+</mo>
                                    <mfrac>
                                      <mrow>
                                        <mn>1</mn>
                                      </mrow>
                                      <mrow>
                                        <mn>4</mn>
                                      </mrow>
                                    </mfrac>
                                    <mo class="MathClass-bin">⋅</mo>
                                    <msub>
                                      <mrow>
                                        <mi>Q</mi>
                                      </mrow>
                                      <mrow>
                                        <msub>
                                          <mrow>
                                            <mi>r</mi>
                                          </mrow>
                                          <mrow>
                                            <mn>1</mn>
                                          </mrow>
                                        </msub>
                                      </mrow>
                                    </msub>
                                    <mo class="MathClass-bin">+</mo>
                                    <mfrac>
                                      <mrow>
                                        <mn>1</mn>
                                      </mrow>
                                      <mrow>
                                        <mn>4</mn>
                                      </mrow>
                                    </mfrac>
                                    <mo class="MathClass-bin">⋅</mo>
                                    <msub>
                                      <mrow>
                                        <mi>Q</mi>
                                      </mrow>
                                      <mrow>
                                        <msub>
                                          <mrow>
                                            <mi>r</mi>
                                          </mrow>
                                          <mrow>
                                            <mn>2</mn>
                                          </mrow>
                                        </msub>
                                      </mrow>
                                    </msub>
                                    <mo class="MathClass-bin">+</mo>
                                    <mfrac>
                                      <mrow>
                                        <mn>1</mn>
                                      </mrow>
                                      <mrow>
                                        <mn>4</mn>
                                      </mrow>
                                    </mfrac>
                                    <mo class="MathClass-bin">⋅</mo>
                                    <msub>
                                      <mrow>
                                        <mi>Q</mi>
                                      </mrow>
                                      <mrow>
                                        <msub>
                                          <mrow>
                                            <mi>r</mi>
                                          </mrow>
                                          <mrow>
                                            <mn>3</mn>
                                          </mrow>
                                        </msub>
                                      </mrow>
                                    </msub>
                                  </mrow>
                                  <mo class="MathClass-close">)</mo>
                                </mrow>
                              </mtd>
                            </mtr>
                          </mtable>
                        </mrow>
                      </mrow></semantics></math></script></div> <div class="EquationNumber">(7)</div></div></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">As
 can be observed, if all 16 values are scaled jointly, the scaling 
multiplications can be easily undone numerically at the virtual root, 
when the overall likelihood of the tree is computed (see [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] for more details). This is not the case, if one intends to scale the ancestral probability values individually on a per-rate (<em xmlns="" class="EmphasisTypeItalic">r</em><sub>0</sub>, ..., <em xmlns="" class="EmphasisTypeItalic">r</em><sub>3</sub>) basis.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 problem that arises with using Γ on very large trees is that, the 16 
ancestral probability values (using four discrete Γ rates for DNA data),
 may have such highly divergent numerical values because of the 4 
discrete rates <em xmlns="" class="EmphasisTypeItalic">r</em><sub>0</sub>, <em xmlns="" class="EmphasisTypeItalic">r</em><sub>1</sub>, <em xmlns="" class="EmphasisTypeItalic">r</em><sub>2</sub>, <em xmlns="" class="EmphasisTypeItalic">r</em><sub>3</sub>,
 that scaling across all 16 of them will still not prevent numerical 
under-flow. In other words, the smallest value of those 16 will be too 
small and the largest too large to fit into the representable machine 
number range between 0.0 and 1.0. Scaling values above 1.0 will yield 
numerical overflow and does hence not provide a solution either. At 
present, we are not aware how scaling multiplications can be undone 
(reversed) in a numerically stable way at the root, if one scales the 
probability values for each discrete rate category individually. This 
phenomenon could also occur for models other than Γ that only use four 
values per site (see below), but will probably occur on significantly 
larger trees. While one could use extended precision libraries, as 
provided for instance by the GNU Scientific Library, the negative 
performance impact will be such, that the computation of large trees 
also becomes prohibitive.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">It
 is however possible to address this scaling problem by discarding the 
per-rate/category likelihoods that contribute least to the overall 
likelihood and thereby approximate the GAMMA-based likelihood score of a
 site. This approach (currently unpublished) has been implemented in 
PhyML [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 this end, we advocate the usage of per-site rate categories as proposed
 and implemented in RAxML (CAT approximation of rate heterogeneity [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]), PhyloBayes [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>], or FastTree 2.0 [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 While methods that model among-site rate heterogeneity by using one 
rate per site can help to significantly reduce computational 
requirements (memory utilization and floating point operations are 
reduced by approximately a factor of four compared to a Γ model with 
four discrete rates [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>]),
 they can also help to resolve the aforementioned numerical problems 
with Γ on very large trees. Note that, the CAT approximation as 
implemented in RAxML should not be confused with the substantially 
different CAT model implemented in PhyloBayes. The unfortunate fact that
 an identical acronym is used is because at time of publication, the 
author of RAxML was not aware of the PhyloBayes CAT model that was 
introduced earlier.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">One key issue with the so-called CAT approximation of rate heterogeneity [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] in RAxML was that branch length values were meaningless. This has been corrected in RAxML version 7.2.9 (available at <span class="ExternalRef"><a href="http://wwwkramer.in.tum.de/exelixis/software/RAxML-7.2.9.tar.bz2" target="_blank" class="ext-link"><span class="RefSource">http://wwwkramer.in.tum.de/exelixis/software/RAxML-7.2.9.tar.bz2</span></a></span>)
 by appropriately re-scaling the per-site rate categories such that the 
mean substitution rate is 1.0. While, as we show, the correctly scaled 
CAT-based branch lengths are highly correlated (see results section) 
with the branch lengths obtained from the Γ model, the overall tree 
length obtained for Γ and CAT-based branch lengths can vary 
significantly. Analogous results were obtained for FastTree 2.0 [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 This does not represent a problem as long as post-analysis tools for 
trees (e.g, divergence-time estimation, ancestral state reconstruction) 
do not rely on absolute branch length values.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Another
 issue that needs to be addressed is that CAT-based log likelihood 
scores across different runs (e.g., two ML searches on the original 
alignment using different starting trees) can not be compared directly, 
because the estimates and assignments of rate categories to sites may be
 slightly different for each search/tree topology. Therefore, for 
comparing likelihood scores of trees under the CAT approximation of rate
 heterogeneity, we need to score all alternative trees under the same 
assignment of rate categories to sites. RAxML v729 implements the <span xmlns="" class="EmphasisFontCategoryNonProportional">-f n</span> option to score a set of fixed trees under the same rate category to site assignment.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Finally,
 one also needs to assess how differently trees are ranked (ordered) 
with respect to their log likelihood scores, if scored under Γ or under 
CAT. While one would not expect a perfect rank correlation (because CAT 
has more ML model parameters than Γ), because both models accomodate 
rate heterogeneity, the correlation should not be too low either.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec11" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#ResultsandDiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="ResultsandDiscussion">Results and Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="ResultsandDiscussion" data-test="collapse-section"><section id="Sec12" class="Section2 RenderAsSection2">
                <h3 class="Heading">Test Datasets</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To assess our methods, we used two large multi-gene datasets of plants.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 first dataset comprises 37,831 taxa and 9,028 sites and was obtained as
 follows: We assembled a DNA sequence matrix of 37,831 seed plant taxa 
consisting of the chloroplast regions atpB (1,861 taxa, &gt; 2.6 
Megabases [Mb]), matK (10,886 taxa, &gt; 14.3 Mb), rbcL (7,319 taxa, 
&gt; 9.7 Mb), trnK (4,163 taxa, &gt; 7.5 Mb), and trnL-trnF (17,618 
taxa, &gt; 13 Mb), and the internal transcribed spacer (ITS; 26,038 
taxa, &gt; 14.3 Mb), using the Phylogeny Assembly with Databases tool 
(PHLAWD [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>]<span class="ExternalRef"><a href="http://code.google.com/p/phlawd" target="_blank" class="ext-link"><span class="RefSource">http://code.google.com/p/phlawd</span></a></span>). All sequence alignments were conducted using MAFFT version 6 [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>] for initial alignments and MUSCLE for profile alignments [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>]. Alignment matrix manipulations were performed with Phyutility [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 second dataset comprises 55,593 taxa and 9,853 sites and was obtained 
using the same pipeline as described above. The gene regions used were 
atpB (2,346 taxa, &gt; 3.6 Megabases [Mb]), matK (14,848 taxa, &gt; 33.6
 Mb), rbcL (10,269 taxa, &gt; 14.9 Mb), trnK (5,859 taxa, &gt; 15.3 Mb),
 and trnL-trnF (25,346 taxa, &gt; 30.1 Mb), and the internal transcribed
 spacer (ITS; 37,492 taxa, &gt; 30.9 Mb).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 ease of reference we henceforth denote the 37,831 taxon datasets as 38 K
 and the 55,593 taxon as 56 K. Trees computed on the 56 K dataset have 
recently been published [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>] and the alignment is available at <span class="ExternalRef"><a href="http://datadryad.org/" target="_blank" class="ext-link"><span class="RefSource">http://datadryad.org/</span></a></span>. The 38 K dataset is currently unpublished, but will be made available immediately upon publication.</p>
              </section><section id="Sec13" class="Section2 RenderAsSection2">
                <h3 class="Heading">Backbone Algorithm</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To test the backbone algorithm we executed the dedicated RAxML version (available at <span class="ExternalRef"><a href="http://wwwkramer.in.tum.de/exelixis/software/BackboneSearch.zip" target="_blank" class="ext-link"><span class="RefSource">http://wwwkramer.in.tum.de/exelixis/software/BackboneSearch.zip</span></a></span>) with the experimental <span xmlns="" class="EmphasisFontCategoryNonProportional">-L</span>
 command line option. This option initially builds a backbone tree and 
then deploys the CAT approximation of rate heterogeneity [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>] with the standard RAxML hill-climbing search algorithm [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>] to apply lazy SPR moves (see [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>])
 within the backbone only. We used tree size reduction factors of 0.25 
and 0.5. As starting trees, we used randomized stepwise addition order 
parsimony starting trees generated with RAxML v727 (<span xmlns="" class="EmphasisFontCategoryNonProportional">-y</span>
 option). For each dataset, we inferred 10 ML trees for each of the 10 
parsimony starting trees. RAxML was executed using the Pthreads-based 
parallel version [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>] with 16 threads on unloaded Quad-Core AMD Opteron nodes with 16 cores and 128 GB RAM each.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 computed average runtimes over 10 runs for the 38 K and 56 K datasets 
respectively. For each backbone tree, we also computed the theoretical 
minimum number of bytes (denoted as <em xmlns="" class="EmphasisTypeItalic">Memory for Backbone</em>)
 required to store the ancestral probability vectors at the virtual tips
 and the inner nodes which dominate memory requirements. If the branch 
length optimization process, unlike in our current implementation, is 
limited to optimizing branches within the backbone, this theoretical 
minimum value represents a good estimate of the memory footprint for a 
backbone tree search. We also computed the respective memory 
requirements for the comprehensive tree (denoted as <em xmlns="" class="EmphasisTypeItalic">Memory for Full tree</em>), which reflects the 'standard' memory requirements when no reduction factor is applied.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">These values (see Tables <span class="InternalRef"><a href="#Tab2">2</a></span> and <span class="InternalRef"><a href="#Tab3">3</a></span>) provide a notion of the potential memory savings that can be achieved by the backbone approach. In Tables <span class="InternalRef"><a href="#Tab2">2</a></span> and <span class="InternalRef"><a href="#Tab3">3</a></span> we also provide the respective execution times and average log likelihood scores obtained by using the backbone algorithm (<em xmlns="" class="EmphasisTypeItalic">R</em> := 0.25, <em xmlns="" class="EmphasisTypeItalic">R</em> := 0.5) and a comprehensive search on the full tree (<em xmlns="" class="EmphasisTypeItalic">R</em>
 := 1.0). Those values have been averaged over 10 runs (10 starting 
trees). While execution times can be reduced by the backbone approach, 
log likelihood scores obtained by conducting searches on a backbone are 
slightly worse than those obtained by searching on the full tree. Also 
note that, for unfavorable tree shapes, that is, tree shapes where a 
substantial part of the phylogenetic signal is located at or near the 
tips, a too aggressive setting of <em xmlns="" class="EmphasisTypeItalic">R</em>
 may potentially generate unfavorable results since this signal can be 
lost in the backbone. However, some exploratory tests with simulated 
data (results not shown) did not show such an effect for backbone 
searches.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Average runtimes, memory requirements, and log likelihood scores (over 10 runs) for the 38 K dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">37831 taxa</p></th></tr></thead><tbody><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">R = 0.25</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">R = 0.5</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">R = 1</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Runtime (h)</p></td><td><p xmlns="" class="SimplePara">30.41</p></td><td><p xmlns="" class="SimplePara">38.60</p></td><td><p xmlns="" class="SimplePara">54.03</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Memory for Backbone (GB)</p></td><td><p xmlns="" class="SimplePara">4.90</p></td><td><p xmlns="" class="SimplePara">7.70</p></td><td><p xmlns="" class="SimplePara">N/A</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Memory for Full tree (GB)</p></td><td><p xmlns="" class="SimplePara">10.33</p></td><td><p xmlns="" class="SimplePara">10.33</p></td><td><p xmlns="" class="SimplePara">10.33</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">LogLikelihood (Avg)</p></td><td><p xmlns="" class="SimplePara">-5531436</p></td><td><p xmlns="" class="SimplePara">-5530051</p></td><td><p xmlns="" class="SimplePara">-5529406</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">LogLikelihood (Std Dev)</p></td><td><p xmlns="" class="SimplePara">943.26</p></td><td><p xmlns="" class="SimplePara">770.47</p></td><td><p xmlns="" class="SimplePara">1307.16</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Avg (logLH - logLH(R = 1))</p></td><td><p xmlns="" class="SimplePara">2030.24</p></td><td><p xmlns="" class="SimplePara">645.31</p></td><td><p xmlns="" class="SimplePara">0.0</p></td></tr></tbody></table></div></figure><figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Average runtimes, memory requirements, and log likelihood scores (over 10 runs) for the 56 K dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">55593 taxa</p></th></tr></thead><tbody><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">R = 0.25</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">R = 0.5</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">R = 1</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Runtime (h)</p></td><td><p xmlns="" class="SimplePara">50.17</p></td><td><p xmlns="" class="SimplePara">63.22</p></td><td><p xmlns="" class="SimplePara">85.89</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Memory for Backbone (GB)</p></td><td><p xmlns="" class="SimplePara">8.22</p></td><td><p xmlns="" class="SimplePara">12.72</p></td><td><p xmlns="" class="SimplePara">N/A</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Memory for Full tree (GB)</p></td><td><p xmlns="" class="SimplePara">16.82</p></td><td><p xmlns="" class="SimplePara">16.82</p></td><td><p xmlns="" class="SimplePara">16.82</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">LogLikelihood</p></td><td><p xmlns="" class="SimplePara">-7063342</p></td><td><p xmlns="" class="SimplePara">-7061516</p></td><td><p xmlns="" class="SimplePara">-7060488</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">LogLikelihood (Std Dev)</p></td><td><p xmlns="" class="SimplePara">1727.90</p></td><td><p xmlns="" class="SimplePara">1761.27</p></td><td><p xmlns="" class="SimplePara">1718.47</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Avg (logLH - logLH(R = 1))</p></td><td><p xmlns="" class="SimplePara">2853.41</p></td><td><p xmlns="" class="SimplePara">1028.04</p></td><td><p xmlns="" class="SimplePara">0.0</p></td></tr></tbody></table></div></figure></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In Figures <span class="InternalRef"><a href="#Fig5">5</a></span> and <span class="InternalRef"><a href="#Fig6">6</a></span> we show that the choice of the random number seed (<span xmlns="" class="EmphasisFontCategoryNonProportional">-p</span>
 option in RAxML), that determines the shape of the starting trees, has a
 significant impact on the final log likelihood score (computed under 
GTR+Γ), irrespective of the search strategy that is used. On average, 
searches on the full tree yield better likelihood scores than searches 
on backbone trees. However, the variance of the likelihood score as a 
function of the starting tree (parsimony random number seed) is 
analogous to the score variance between full and backbone tree searches.
 For example, on the 38 K dataset, the log likelihood scores on 10 final
 trees obtained for full searches show a standard deviation of 1307 log 
likelihood units. The average difference in log likelihood scores per 
starting tree between the full search and a backbone search with <em xmlns="" class="EmphasisTypeItalic">R</em> := 0.50 is only 645 log likelihood units and 2030 log likelihood units for backbone searches with <em xmlns="" class="EmphasisTypeItalic">R</em> := 0.25, respectively.<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig5_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2011_Article_5012_Fig5_HTML.jpg" alt="Figure 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig5_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig5_HTML.jpg" alt="Figure 5"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 5</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Log Likelihood scores for different Reduction factors (38 k dataset)</strong>.
 Plot of log likelihood scores under GTR+Γ of the final trees obtained 
by each method as a function of the starting tree (random number seed) 
for the 38 K dataset. Each LH score (point) results from an independent 
search. The lines linking the points are only guiding the eye.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig6_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2011_Article_5012_Fig6_HTML.jpg" alt="Figure 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig6_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_Article_5012_Fig6_HTML.jpg" alt="Figure 6"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 6</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Log Likelihood scores for different Reduction factors (56 k dataset)</strong>.
 Plot of log likelihood scores under GTR+Γ of the final trees obtained 
by each method as a function of the starting tree (random number seed) 
for the 56 K dataset. Each LH score (point) results from an independent 
search. The lines linking the points are only guiding the eye.</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Given
 the runtimes savings that can be achieved by the backbone approach, 
backbone tree searches can be used, for instance, to explore a larger 
number of parsimony starting trees which substantially influence the 
final log likelihood scores. A reasonable strategy for finding 
best-known ML trees may consist in starting many fast searches with a 
relatively aggressive setting of <em xmlns="" class="EmphasisTypeItalic">R</em>
 := 0.25 to identify/determine a set of 'good' starting trees that yield
 the best final log likelihood scores. In a second step, full tree 
searches can be conducted on those promising starting trees to find 
trees with even better scores.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 used simulated datasets in order to better understand the impact of the
 backbone algorithm on topological accuracy. We ran indelible [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>]
 to generate simulated MSAs of 1500 taxa (575 bp) and 5000 taxa (1074 
bp). We compared the symmetric difference (number of bipartitions that 
differ between two topologies) between the true tree and the topologies 
from the starndard full search and the backbone-based ones. For each 
dataset, the full search and the backbone search with <em xmlns="" class="EmphasisTypeItalic">R</em> := 0.25 and <em xmlns="" class="EmphasisTypeItalic">R</em> := 0.5 were ran 5 different times with different starting trees. Table <span class="InternalRef"><a href="#Tab4">4</a></span>
 shows the average symetric differences among all approaches for the 
dataset with 1500 taxa. We see that, in terms of topological accuracy, 
applying the reductions of <em xmlns="" class="EmphasisTypeItalic">R</em> := 0.25 and <em xmlns="" class="EmphasisTypeItalic">R</em>
 := 0.5 yield topologies that remain close to the standard full search. 
Furthermore, the distance to the true tree is not increased by the 
reduction.<figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">Average symmetric differences (over 5 runs) for the 1500 dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">Average Symmetric Difference</p></th></tr></thead><tbody><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeBoldItalic">R</span>
                              <strong class="EmphasisTypeBold">:= 0.25</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeBoldItalic">R</span>
                              <strong class="EmphasisTypeBold">:= 0.5</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <span class="EmphasisTypeBoldItalic">R</span>
                              <strong class="EmphasisTypeBold">:= 1</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">R</em> := 0.25</p></td><td><p xmlns="" class="SimplePara">182.6</p></td><td><p xmlns="" class="SimplePara">169.9</p></td><td><p xmlns="" class="SimplePara">188.0</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">R</em> := 0.5</p></td><td><p xmlns="" class="SimplePara">169.9</p></td><td><p xmlns="" class="SimplePara">152.8</p></td><td><p xmlns="" class="SimplePara">146.2</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">R</em> := 1</p></td><td><p xmlns="" class="SimplePara">188.0</p></td><td><p xmlns="" class="SimplePara">146.2</p></td><td><p xmlns="" class="SimplePara">133.0</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">True Tree</p></td><td><p xmlns="" class="SimplePara">398.8</p></td><td><p xmlns="" class="SimplePara">382.0</p></td><td><p xmlns="" class="SimplePara">388.0</p></td></tr></tbody></table></div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 likelihood scores for both simulated datasets follow the same pattern 
as in the case of real data. These results, details on how the 
simulation datasets were generated, as well as the symetric difference 
for the 5000 taxa dataset have been included in the additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>.</p>
              </section><section id="Sec14" class="Section2 RenderAsSection2">
                <h3 class="Heading">SEV Performance</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 also used the 38 K and 56 K datasets to test memory savings and 
speedups achieved by applying the adapted SEV technique to phylogenomic 
datasets. The gappyness (percentage of missing data in the alignments) 
is 81.53% for 38 K and 83.40% for 56 K, respectively.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 each alignment, we computed a parsimony starting tree with RAxML that 
was then evaluated (model parameter and branch length optimization 
without tree search, RAxML <span xmlns="" class="EmphasisFontCategoryNonProportional">-f e</span>
 option) with RAxML under the GTR+Γ model using the SEV reimplementation
 (with and without memory saving) and using the standard likelihood 
implementation.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 standard implementation required 41 GB of memory on the 38 K dataset 
and 66 GB of memory on the 56 K dataset. The SEV technique with the 
memory saving option enabled (<span xmlns="" class="EmphasisFontCategoryNonProportional">-U</span>
 option, available as of RAxML v727) reduced memory footprints under Γ 
to 14 GB (38 K) and 21 GB (56 K) respectively. The log likelihood scores
 for all three implementations were exactly identical. As shown in 
Tables <span class="InternalRef"><a href="#Tab5">5</a></span> and <span class="InternalRef"><a href="#Tab6">6</a></span>,
 the runtimes of the SEV-based versions are 25-40% faster than for the 
standard implementation. The runtime differences between the SEV-based 
implementation with memory saving enabled and the plain SEV version 
without memory saving, can be attributed to differences in memory access
 patterns. While both versions conduct the same number of computations, 
the memory-saving version needs to make millions of calls to OS routines
 (<span xmlns="" class="EmphasisFontCategoryNonProportional">free()</span> and <span xmlns="" class="EmphasisFontCategoryNonProportional">malloc()</span>) while the plain SEV version exhibits a higher memory footprint and thereby, potentially, a higher cache miss rate.<figure xmlns="" id="Tab5" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 5</span>
                        <p xmlns="" class="SimplePara">SEV evaluation for the 38 k dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">37831 taxa</p></th></tr></thead><tbody><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">SEVs</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">SEVs with memory saving</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">standard</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Runtime (s)</p></td><td><p xmlns="" class="SimplePara">4125.1</p></td><td><p xmlns="" class="SimplePara">4116.8</p></td><td><p xmlns="" class="SimplePara">6541.1</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Memory (GB)</p></td><td><p xmlns="" class="SimplePara">42</p></td><td><p xmlns="" class="SimplePara">15</p></td><td><p xmlns="" class="SimplePara">41</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">LogLikelihood</p></td><td><p xmlns="" class="SimplePara">-5528590</p></td><td><p xmlns="" class="SimplePara">-5528590</p></td><td><p xmlns="" class="SimplePara">-5528590</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Execution times and
 memory requirements for optimizing model parameters and branch lengths 
under on the 38 K dataset using SEVs, SEVs with memory saving, and the 
standard likelihood implementation.</p>
                    </div></figure><figure xmlns="" id="Tab6" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 6</span>
                        <p xmlns="" class="SimplePara">SEV evaluation for the 56 k dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="4"><p xmlns="" class="SimplePara">55593 taxa</p></th></tr></thead><tbody><tr class="noclass"><td>&nbsp;</td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">SEVs</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">SEVs with memory saving</strong>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <strong class="EmphasisTypeBold">standard</strong>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Runtime (s)</p></td><td><p xmlns="" class="SimplePara">7145.2</p></td><td><p xmlns="" class="SimplePara">8095.1</p></td><td><p xmlns="" class="SimplePara">11181.4</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Memory (GB)</p></td><td><p xmlns="" class="SimplePara">67</p></td><td><p xmlns="" class="SimplePara">29</p></td><td><p xmlns="" class="SimplePara">67</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">log likelihood</p></td><td><p xmlns="" class="SimplePara">-7059556</p></td><td><p xmlns="" class="SimplePara">-7059556</p></td><td><p xmlns="" class="SimplePara">-7059556</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Execution times and
 memory requirements for optimizing model parameters and branch lengths 
under on the 56 K dataset using SEVs, SEVs with memory saving, and the 
standard likelihood implementation.</p>
                    </div></figure></div>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Estimating branch lengths and computing likelihood scores with CAT</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We optimized branch lengths and model parameters under CAT and Γ using RAxML v730 (<span xmlns="" class="EmphasisFontCategoryNonProportional">-f n</span>
 option) on collections of 32 and 22 final ML trees for the 38 K and 56 K
 partitioned datasets, respectively. For the CAT model, we also assessed
 the impact of using, 8, 16, 25 (default), and 40 per-site rate 
categories.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 each tree, we computed the average Pearson correlation coefficient 
between the branch lengths obtained under CAT (for 8, 16, 25, and 40 
per-site rate categories) and the branch lengths as estimated under Γ. 
We also computed the average tree length ratio over all 32 trees.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 determine if the trees are ranked in the same order by their respective
 Γ and CAT log likelihood scores, we computed the Spearman rank 
correlation of the CAT- and Γ-based tree rankings. As shown in Tables <span class="InternalRef"><a href="#Tab7">7</a></span> and <span class="InternalRef"><a href="#Tab8">8</a></span>,
 the Spearman correlation was above 0.99 in all cases. This indicates 
that, trees are ordered in almost the same way, regardless of whether 
they are scored under CAT or Γ.<figure xmlns="" id="Tab7" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 7</span>
                        <p xmlns="" class="SimplePara">Correlations between CAT and Γ models for the 38 k dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="5"><p xmlns="" class="SimplePara">37831 taxa, 32 ML trees</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of per-site rate categories</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">25</p></td><td><p xmlns="" class="SimplePara">40</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average BL correlation with Γ</p></td><td><p xmlns="" class="SimplePara">0.994</p></td><td><p xmlns="" class="SimplePara">0.995</p></td><td><p xmlns="" class="SimplePara">0.995</p></td><td><p xmlns="" class="SimplePara">0.995</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average Tree length ratio (Γ/CAT)</p></td><td><p xmlns="" class="SimplePara">1.743</p></td><td><p xmlns="" class="SimplePara">1.739</p></td><td><p xmlns="" class="SimplePara">1.739</p></td><td><p xmlns="" class="SimplePara">1.739</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Spearman rank correlation(Γ, CAT)</p></td><td><p xmlns="" class="SimplePara">0.994</p></td><td><p xmlns="" class="SimplePara">0.992</p></td><td><p xmlns="" class="SimplePara">0.992</p></td><td><p xmlns="" class="SimplePara">0.992</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Correlation between
 CAT and Γ-based ML branch length estimates, total tree length ratios, 
and Spearman rank correlation coefficients between likelihood-induced 
tree rankings obtained from CAT and Γ for dataset 38 K.</p>
                    </div></figure><figure xmlns="" id="Tab8" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 8</span>
                        <p xmlns="" class="SimplePara">Correlations between CAT and Γ models for the 56 k dataset</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th colspan="5"><p xmlns="" class="SimplePara">55593 taxa, 22 ML trees</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Number of per-site rate categories</p></td><td><p xmlns="" class="SimplePara">8</p></td><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">25</p></td><td><p xmlns="" class="SimplePara">40</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average BL correlation with Γ</p></td><td><p xmlns="" class="SimplePara">0.877</p></td><td><p xmlns="" class="SimplePara">0.877</p></td><td><p xmlns="" class="SimplePara">0.877</p></td><td><p xmlns="" class="SimplePara">0.877</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Average Tree length ratio (Γ/CAT)</p></td><td><p xmlns="" class="SimplePara">1.569</p></td><td><p xmlns="" class="SimplePara">1.567</p></td><td><p xmlns="" class="SimplePara">1.567</p></td><td><p xmlns="" class="SimplePara">1.608</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Spearman rank correlation(Γ, CAT)</p></td><td><p xmlns="" class="SimplePara">1.0</p></td><td><p xmlns="" class="SimplePara">1.0</p></td><td><p xmlns="" class="SimplePara">1.0</p></td><td><p xmlns="" class="SimplePara">1.0</p></td></tr></tbody></table></div><div xmlns="http://www.w3.org/1999/xhtml" class="TableFooter">
                      <p xmlns="" class="SimplePara">Correlation between
 CAT and Γ-based ML branch length estimates, total tree length ratios, 
and Spearman rank correlation coefficients between likelihood-induced 
tree rankings obtained from CAT and Γ for dataset 55 K.</p>
                    </div></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We also used FastTree 2 (with options <span xmlns="" class="EmphasisFontCategoryNonProportional">-gamma -nt -nome -mllen</span>) to score both collections of trees under the hybrid CAT/Gamma20 model [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>].
 Once again, the Spearman rank correlation of FastTree CAT/Gamma20 and 
RAxML Γbased tree rankings remained above 0.99 in all cases.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 average branch length correlation between CAT and Γ optimized branches 
was above 0.87. On those two large datasets, the absolute length of 
Γ-based branch length estimates was larger than for CAT as shown by the 
average tree length ratios.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 also executed analogous analyses on 10 smaller single-gene (and 
non-partitioned) datasets with 1481 up to 4114 taxa. In addition, we 
evaluated significantly larger ML tree collections (160 ML trees per 
dataset) for those smaller datasets. These additional experiments 
confirmed our observations for the 38 K and 55 K datasets and also 
revealed that the total tree length under CAT, can also be larger than 
the Γ-based tree length. Respective plots for all datasets are provided 
in the additional file <span class="InternalRef"><a href="#MOESM1">1</a></span>.</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec16" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 have explored several techniques, addressed problems, and proposed some
 solutions for phylogenetic tree inference with likelihood-based methods
 on trees with several tens of thousands of taxa.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Initially,
 we revisit and re-assess techniques for reducing the tree size, 
inspired by earlier work on a program called Phylogenetic Navigator 
(Phy-Nav). Significant effort was invested in exploring different 
backbone construction techniques (results/experiments not shown). Here, 
we describe the method that worked best with respect to final log 
likelihood scores. Such backbone-based techniques can help to reduce 
memory footprints and execution times. However, in almost all cases they
 yield final trees with worse likelihoods compared to comprehensive tree
 searches on a full, unreduced tree. We find that likelihood scores of 
final trees heavily depend on the respective starting trees, and 
conclude that backbone approaches can be deployed for identifying 'good'
 starting trees, that can then be further refined using a comprehensive 
tree search.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 have adapted and re-implemented the SEV technique for phylogenomic 
datasets with missing data and enhanced it by a novel memory-saving 
option. This new technique, is generally applicable to all 
likelihood-based codes and can reduce execution times by 25-40% on 
sufficiently 'gappy' datasets by omitting redundant computations. More 
importantly, the revised SEV technique can be deployed to achieve 
significant memory savings that are almost proportional to the amount of
 missing data in the test datasets. This technique has already been 
fully integrated into the standard RAxML distribution (as of v727). 
Moreover, RAxML will automatically determine whether to use the standard
 likelihood implementation or the SEV-based likelihood implementation.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Finally,
 we analyze problems associated to numerical scaling for avoiding 
underflow, that can occur when using the Γ model of rate heterogeneity 
on very large datasets. While for the 38 K and 55 K datasets we were 
still able to evaluate trees under Γ, on some even larger datasets that 
we are currently analyzing (e.g., 116,408 taxa 18,692 sites) numerical 
scaling under Γ appears to be impossible using 64-bit floating point 
arithmetics. To this end, we advocate the usage of models that rely on 
per-site rate categories for accommodating rate heterogeneity among 
sites. Clearly, further research is required in this area to devise 
statistically robust and meaningful models. Nonetheless, we provide an 
empirical assessment of branch length estimates as obtained under Γ and 
the RAxML-specific implementation for estimating and assigning per-site 
evolutionary rate categories. We find that, given proper scaling of 
per-site rates, branch lengths between CAT and Γ based trees are highly 
correlated, despite the fact that absolute branch length values can 
differ substantially. We also find that, ordering tree collections using
 Γbased and CAT-based log likelihood scores induces very similar 
rankings of trees as determined be the Spearman rank correlation 
coefficient.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 work presented here has a clear exploratory flavor and we hope that it 
will be useful to the community for identifying future research 
directions pertaining to large-scale phylogenetic inference using 
likelihood-based methods. The problems and solutions we discuss in this 
paper, emerged within the framework of the plant tree of life grand 
challenge project that aims at reconstructing the plant tree of life 
comprising approximately 500,000 taxa.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">FIC
 is funded by the German Science Foundation (DFG), AS is funded by the 
Heidelberg Institute for Theoretical Studies, SAS is funded by the 
National Science Foundation (NSF). The authors would like to thank 
Bernard Moret for granting access to his AMD Barcelona nodes and Morgan 
Price for useful discussions regarding branch length issues under CAT.</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Electronic supplementary material</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM1_ESM.PDF">12859_2011_5012_MOESM1_ESM.PDF</a>
                          <span xmlns="" class="SimplePara">Additional file 1:<strong class="EmphasisTypeBold">Supplementary Material</strong>.
 Assesment of alternative criteria to identify the innermost node of a 
tree. Evaluation of the backbone algorithm with simulated data: 
Simulation details and symmetric difference for the 5000 taxa dataset, 
log likelihood scores for ML trees on simulated datasets(1500 and 5000 
species). Evaluation of the backbone algorithm with real data and 
comparison with FastTree 2. Correlation between CAT and Γ-based ML 
branch length estimates, total tree length ratios, and Spearman rank 
correlation coefficients between likelihood-induced tree rankings 
obtained from CAT and Γ for 12 different datasets ranging from 1481 up 
to 4114 number of taxa. Correlations between log likelihood scores under
 the RAxML CAT model and Γ model for the 38 k and 56 k dataset. 
Correlations between log likelihood scores under the RAxML CAT model and
 the FastTree 2 CAT/Gamma20 model for the 38 k and 56 k dataset. (PDF 
984 KB)</span>
                        </div></div>
                  </div>
                </div>
              
              
                
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authors’ original submitted files for images.<div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM2_ESM.jpeg">12859_2011_5012_MOESM2_ESM.jpeg</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 1</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM3_ESM.jpeg">12859_2011_5012_MOESM3_ESM.jpeg</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 2</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM4_ESM.jpeg">12859_2011_5012_MOESM4_ESM.jpeg</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 3</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM5_ESM.jpeg">12859_2011_5012_MOESM5_ESM.jpeg</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 4</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM6_ESM.jpeg">12859_2011_5012_MOESM6_ESM.jpeg</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 5</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-12-470/MediaObjects/12859_2011_5012_MOESM7_ESM.jpeg">12859_2011_5012_MOESM7_ESM.jpeg</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 6</span>
                        </div></div>
                  </div></div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors' contributions</h3><p xmlns="" class="SimplePara">FIC
 developed and implemented the backbone algorithm and conducted all 
computational experiments. AS developed and implemented the SEV-based 
techniques and implemented the CAT re-scaling procedure. SAS assembled 
the large test datasets. FIC, AS, and SAS wrote and edited the 
manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">The
 Exelixis Lab, Scientific Computing Group, Heidelberg Institute for 
Theoretical Studies, Schloss-Wolfsbrunnenweg 35, 
D-69118&nbsp;Heidelberg, Germany</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">2
 Smith Lab, Dept. Ecology and Evolutionary Biology, University of 
Michigan, 2005 Kraus Natural Science Building, Ann Arbor, 
MI&nbsp;48109-1048, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ronaghi M: Pyrosequencing Sheds Light on DNA Sequencing. Genome Research 2001, 11: 3–11. 10.1101/gr.11.1.3</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1101/gr.11.1.3"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11156611"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Pyrosequencing%20Sheds%20Light%20on%20DNA%20Sequencing&amp;author=M.%20Ronaghi&amp;journal=Genome%20Research&amp;volume=11&amp;pages=3-11&amp;publication_year=2001&amp;doi=10.1101%2Fgr.11.1.3"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein
 J: Evolutionary trees from DNA sequences: a maximum likelihood 
approach. J Mol Evol 1981, 17: 368–376. 10.1007/BF01734359</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF01734359"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7288891"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;author=J.%20Felsenstein&amp;journal=J%20Mol%20Evol&amp;volume=17&amp;pages=368-376&amp;publication_year=1981&amp;doi=10.1007%2FBF01734359"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Minh
 B, Vinh L, Haeseler A, Schmidt H: pIQPNNI: parallel reconstruction of 
large maximum likelihood phylogenies. Bioinformatics 2005, 
21(19):3794–3796. 10.1093/bioinformatics/bti594</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bti594"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16046495"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=pIQPNNI%3A%20parallel%20reconstruction%20of%20large%20maximum%20likelihood%20phylogenies&amp;author=B.%20Minh&amp;author=L.%20Vinh&amp;author=A.%20Haeseler&amp;author=H.%20Schmidt&amp;journal=Bioinformatics&amp;volume=21&amp;issue=19&amp;pages=3794-3796&amp;publication_year=2005&amp;doi=10.1093%2Fbioinformatics%2Fbti594"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Zwickl
 D: Genetic Algorithm Approaches for the Phylogenetic Analysis of Large 
Biological Sequence Datasets under the Maximum Likelihood Criterion. PhD
 thesis. University of Texas at Austin; 2006.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20Algorithm%20Approaches%20for%20the%20Phylogenetic%20Analysis%20of%20Large%20Biological%20Sequence%20Datasets%20under%20the%20Maximum%20Likelihood%20Criterion&amp;author=D.%20Zwickl&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Guindon
 S, Dufayard J, Lefort V, Anisimova M, Hordijk W, Gascuel O: New 
algorithms and methods to estimate maximum-likelihood phylogenies: 
assessing the performance of PhyML 3.0. Systematic biology 2010, 
59(3):307. 10.1093/sysbio/syq010</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/sysbio/syq010"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20525638"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=J.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;author=O.%20Gascuel&amp;journal=Systematic%20biology&amp;volume=59&amp;issue=3&amp;pages=307&amp;publication_year=2010&amp;doi=10.1093%2Fsysbio%2Fsyq010"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Price
 M, Dehal P, Arkin A: FastTree 2- Approximately Maximum-Likelihood Trees
 for Large Alignments. PLoS ONE 2010, 5(3):e9490. 
10.1371/journal.pone.0009490</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835736"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1371/journal.pone.0009490"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20224823"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=FastTree%202-%20Approximately%20Maximum-Likelihood%20Trees%20for%20Large%20Alignments&amp;author=M.%20Price&amp;author=P.%20Dehal&amp;author=A.%20Arkin&amp;journal=PLoS%20ONE&amp;volume=5&amp;issue=3&amp;pages=e9490&amp;publication_year=2010&amp;doi=10.1371%2Fjournal.pone.0009490"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ronquist
 F, Huelsenbeck J: MrBayes 3: Bayesian phylogenetic inference under 
mixed models. Bioinformatics 2003, 19(12):1572–1574. 
10.1093/bioinformatics/btg180</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg180"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12912839"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MrBayes%203%3A%20Bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;author=F.%20Ronquist&amp;author=J.%20Huelsenbeck&amp;journal=Bioinformatics&amp;volume=19&amp;issue=12&amp;pages=1572-1574&amp;publication_year=2003&amp;doi=10.1093%2Fbioinformatics%2Fbtg180"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lartillot N, Blanquart S, Lepage T: PhyloBayes. v2. 3. 2007.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=N.%20Lartillot&amp;author=S.%20Blanquart&amp;author=T.%20Lepage&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Drummond A, Rambaut A: BEAST: Bayesian evolutionary analysis by sampling trees. BMC Evol Biol 2007, 7(214):1471–2148.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BEAST%3A%20Bayesian%20evolutionary%20analysis%20by%20sampling%20trees&amp;author=A.%20Drummond&amp;author=A.%20Rambaut&amp;journal=BMC%20Evol%20Biol&amp;volume=7&amp;issue=214&amp;pages=1471-2148&amp;publication_year=2007"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 S, Donoghue M: Rates of Molecular Evolution Are Linked to Life History 
in Flowering Plants. Science 2008, 322(5898):86–89. 
10.1126/science.1163197</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.1163197"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18832643"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Rates%20of%20Molecular%20Evolution%20Are%20Linked%20to%20Life%20History%20in%20Flowering%20Plants&amp;author=S.%20Smith&amp;author=M.%20Donoghue&amp;journal=Science&amp;volume=322&amp;issue=5898&amp;pages=86-89&amp;publication_year=2008&amp;doi=10.1126%2Fscience.1163197"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Goloboff
 PA, Catalano SA, Mirande JM, Szumik CA, Arias JS, Källersjö M, Farris 
JS: Phylogenetic analysis of 73060 taxa corroborates major eukaryotic 
groups. Cladistics 2009, 25: 1–20. 10.1111/j.1096-0031.2008.00233.x</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1096-0031.2009.00255.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%2073060%20taxa%20corroborates%20major%20eukaryotic%20groups&amp;author=PA.%20Goloboff&amp;author=SA.%20Catalano&amp;author=JM.%20Mirande&amp;author=CA.%20Szumik&amp;author=JS.%20Arias&amp;author=M.%20K%C3%A4llersj%C3%B6&amp;author=JS.%20Farris&amp;journal=Cladistics&amp;volume=25&amp;pages=1-20&amp;publication_year=2009&amp;doi=10.1111%2Fj.1096-0031.2008.00233.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 SA, Beaulieu JM, Donoghue MJ: Mega-phylogeny approach for comparative 
biology: an alternative to supertree and supermatrix approaches. BMC 
Evolutionary Biology 2009., 9(37):</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mega-phylogeny%20approach%20for%20comparative%20biology%3A%20an%20alternative%20to%20supertree%20and%20supermatrix%20approaches&amp;author=SA.%20Smith&amp;author=JM.%20Beaulieu&amp;author=MJ.%20Donoghue&amp;publication_year=2009"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A, Ludwig T, Meier H, Wolf MJ: Accelerating Parallel Maximum 
Likelihood-based Phylogenetic Tree Calculations using Subtree Equality 
Vectors. Proc of IEEE/ACM Supercomputing Conference 2002 (SC2002) 2002. 
[Proceedings on CD] [Proceedings on CD]</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Accelerating%20Parallel%20Maximum%20Likelihood-based%20Phylogenetic%20Tree%20Calculations%20using%20Subtree%20Equality%20Vectors&amp;author=A.%20Stamatakis&amp;author=T.%20Ludwig&amp;author=H.%20Meier&amp;author=MJ.%20Wolf&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yang
 Z: Maximum likelihood phylogenetic estimation from DNA sequences with 
variable rates over sites. J Mol Evol 1994, 39: 306–314. 
10.1007/BF00160154</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF00160154"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7932792"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20phylogenetic%20estimation%20from%20DNA%20sequences%20with%20variable%20rates%20over%20sites&amp;author=Z.%20Yang&amp;journal=J%20Mol%20Evol&amp;volume=39&amp;pages=306-314&amp;publication_year=1994&amp;doi=10.1007%2FBF00160154"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Le S, Schmidt H, Haeseler A: PhyNav: A novel approach to reconstruct large phylogenies. Proc of GfKl conference 2004.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=PhyNav%3A%20A%20novel%20approach%20to%20reconstruct%20large%20phylogenies&amp;author=S.%20Le&amp;author=H.%20Schmidt&amp;author=A.%20Haeseler&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schmidt
 B (Ed): Bioinformatics: High Performance Parallel Computer 
Architectures. Taylor &amp; Francis 2010 chap. Orchestrating the 
Phylogenetic Likelihood Function on Emerging Parallel Architectures; 
85–115.<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="https://scholar.google.com/scholar?hl=en&amp;q=Schmidt%20B%20%28Ed%29%3A%20Bioinformatics%3A%20High%20Performance%20Parallel%20Computer%20Architectures.%20Taylor%20%26%20Francis%202010%20chap.%20Orchestrating%20the%20Phylogenetic%20Likelihood%20Function%20on%20Emerging%20Parallel%20Architectures%3B%2085%E2%80%93115."><span><span>Google Scholar</span></span></a></span></span></span><span class="Occurrences"></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">de
 Hoon MJL, S Imoto JN, Miyano S: Open source clustering software. 
Bioinformatics 2004, 20(9):1453–1454. 10.1093/bioinformatics/bth078</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/bth078"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14871861"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Open%20source%20clustering%20software&amp;author=MJL.%20de%20Hoon&amp;author=JN.%20S%20Imoto&amp;author=S.%20Miyano&amp;journal=Bioinformatics&amp;volume=20&amp;issue=9&amp;pages=1453-1454&amp;publication_year=2004&amp;doi=10.1093%2Fbioinformatics%2Fbth078"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A: RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with 
thousands of taxa and mixed models. Bioinformatics 2006, 
22(21):2688–2690. 10.1093/bioinformatics/btl446</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btl446"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16928733"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A.%20Stamatakis&amp;journal=Bioinformatics&amp;volume=22&amp;issue=21&amp;pages=2688-2690&amp;publication_year=2006&amp;doi=10.1093%2Fbioinformatics%2Fbtl446"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A, Ludwig T, Meier H, Wolf MJ: AxML: A Fast Program for Sequential and 
Parallel Phylo-genetic Tree Calculations Based on the Maximum Likelihood
 Method. Proceedings of 1st IEEE Computer Society Bioinformatics 
Conference (CSB2002) 2002, 21–28.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1109/CSB.2002.1039325"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=AxML%3A%20A%20Fast%20Program%20for%20Sequential%20and%20Parallel%20Phylo-genetic%20Tree%20Calculations%20Based%20on%20the%20Maximum%20Likelihood%20Method&amp;author=A.%20Stamatakis&amp;author=T.%20Ludwig&amp;author=H.%20Meier&amp;author=MJ.%20Wolf&amp;pages=21-28&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pond
 S, Muse S: Column sorting: Rapid calculation of the phylogenetic 
likelihood function. Systematic biology 2004, 53(5):685–692. 
10.1080/10635150490522269</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/10635150490522269"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Column%20sorting%3A%20Rapid%20calculation%20of%20the%20phylogenetic%20likelihood%20function&amp;author=S.%20Pond&amp;author=S.%20Muse&amp;journal=Systematic%20biology&amp;volume=53&amp;issue=5&amp;pages=685-692&amp;publication_year=2004&amp;doi=10.1080%2F10635150490522269"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sumner
 J, Charleston M: Phylogenetic estimation with partial likelihood 
tensors. Journal of theoretical biology 2010, 262(3):413–424. 
10.1016/j.jtbi.2009.09.037</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/j.jtbi.2009.09.037"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19822153"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20estimation%20with%20partial%20likelihood%20tensors&amp;author=J.%20Sumner&amp;author=M.%20Charleston&amp;journal=Journal%20of%20theoretical%20biology&amp;volume=262&amp;issue=3&amp;pages=413-424&amp;publication_year=2010&amp;doi=10.1016%2Fj.jtbi.2009.09.037"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A, Alachiotis N: Time and memory efficient likelihood-based tree 
searches on gappy phylogenomic alignments. Bioinformatics 2010, 
26(12):i132-i139. 10.1093/bioinformatics/btq205</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2881390"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btq205"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20529898"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Time%20and%20memory%20efficient%20likelihood-based%20tree%20searches%20on%20gappy%20phylogenomic%20alignments&amp;author=A.%20Stamatakis&amp;author=N.%20Alachiotis&amp;journal=Bioinformatics&amp;volume=26&amp;issue=12&amp;pages=i132-i139&amp;publication_year=2010&amp;doi=10.1093%2Fbioinformatics%2Fbtq205"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ripplinger
 J, Sullivan J: Does Choice in Model Selection Affect Maximum Likelihood
 Analysis? Syst Biol 2008, 57: 76–85. 10.1080/10635150801898920</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/10635150801898920"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18275003"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Does%20Choice%20in%20Model%20Selection%20Affect%20Maximum%20Likelihood%20Analysis%3F&amp;author=J.%20Ripplinger&amp;author=J.%20Sullivan&amp;journal=Syst%20Biol&amp;volume=57&amp;pages=76-85&amp;publication_year=2008&amp;doi=10.1080%2F10635150801898920"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A: Phylogenetic Models of Rate Heterogeneity: A High Performance 
Computing Perspective. Proc. of IPDPS2006, HICOMB Workshop, Proceedings 
on CD, Rhodos, Greece 2006.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetic%20Models%20of%20Rate%20Heterogeneity%3A%20A%20High%20Performance%20Computing%20Perspective&amp;author=A.%20Stamatakis&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lartillot
 N, Philippe H: A Bayesian Mixture Model for Across-Site Heterogeneities
 in the AminoAcid Replacement Process. Mol Biol Evol 2004, 
21(6):1095–1109. 10.1093/molbev/msh112</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msh112"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15014145"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20Bayesian%20Mixture%20Model%20for%20Across-Site%20Heterogeneities%20in%20the%20AminoAcid%20Replacement%20Process&amp;author=N.%20Lartillot&amp;author=H.%20Philippe&amp;journal=Mol%20Biol%20Evol&amp;volume=21&amp;issue=6&amp;pages=1095-1109&amp;publication_year=2004&amp;doi=10.1093%2Fmolbev%2Fmsh112"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Katoh
 K, Toh H: Recent developments in the MAFFT multiple sequence alignment 
program. Briefings in Bioinformatics 2008, 9(4):286–298. 
10.1093/bib/bbn013</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bib/bbn013"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18372315"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recent%20developments%20in%20the%20MAFFT%20multiple%20sequence%20alignment%20program&amp;author=K.%20Katoh&amp;author=H.%20Toh&amp;journal=Briefings%20in%20Bioinformatics&amp;volume=9&amp;issue=4&amp;pages=286-298&amp;publication_year=2008&amp;doi=10.1093%2Fbib%2Fbbn013"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edgar
 RC: MUSCLE: multiple sequence alignment with high accuracy and high 
throughput. Nucleic Acids Research 2004, 32(5):1792–1797. 
10.1093/nar/gkh340</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkh340"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15034147"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC.%20Edgar&amp;journal=Nucleic%20Acids%20Research&amp;volume=32&amp;issue=5&amp;pages=1792-1797&amp;publication_year=2004&amp;doi=10.1093%2Fnar%2Fgkh340"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 SA, Dunn CW: Phyutility: a phyloinformatics tool for trees, alignments 
and molecular data. Bioinformatics 2008, 24(5):715–716. 
10.1093/bioinformatics/btm619</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btm619"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18227120"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phyutility%3A%20a%20phyloinformatics%20tool%20for%20trees%2C%20alignments%20and%20molecular%20data&amp;author=SA.%20Smith&amp;author=CW.%20Dunn&amp;journal=Bioinformatics&amp;volume=24&amp;issue=5&amp;pages=715-716&amp;publication_year=2008&amp;doi=10.1093%2Fbioinformatics%2Fbtm619"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Smith
 S, Beaulieu J, Stamatakis A, Donoghue M: Understanding angiosperm 
diversification using small and large phylogenetic trees. American 
Journal of Botany 2011. ajb-1000481v1</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Understanding%20angiosperm%20diversification%20using%20small%20and%20large%20phylogenetic%20trees&amp;author=S.%20Smith&amp;author=J.%20Beaulieu&amp;author=A.%20Stamatakis&amp;author=M.%20Donoghue&amp;publication_year=2011"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A, Blagojevic F, Antonopoulos CD, Nikolopoulos DS: Exploring new Search
 Algorithms and Hardware for Phylogenetics: RAxML meets the IBM Cell. J 
VLSI Sig Proc Sys 2007, 48(3):271–286. 10.1007/s11265-007-0067-4</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/s11265-007-0067-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Exploring%20new%20Search%20Algorithms%20and%20Hardware%20for%20Phylogenetics%3A%20RAxML%20meets%20the%20IBM%20Cell&amp;author=A.%20Stamatakis&amp;author=F.%20Blagojevic&amp;author=CD.%20Antonopoulos&amp;author=DS.%20Nikolopoulos&amp;journal=J%20VLSI%20Sig%20Proc%20Sys&amp;volume=48&amp;issue=3&amp;pages=271-286&amp;publication_year=2007&amp;doi=10.1007%2Fs11265-007-0067-4"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Stamatakis
 A, Ott M: Efficient computation of the phylogenetic likelihood function
 on multi-gene alignments and multi-core architectures. Phil Trans R Soc
 series B Biol Sci 2008, 363: 3977–3984. 10.1098/rstb.2008.0163</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1098/rstb.2008.0163"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Efficient%20computation%20of%20the%20phylogenetic%20likelihood%20function%20on%20multi-gene%20alignments%20and%20multi-core%20architectures&amp;author=A.%20Stamatakis&amp;author=M.%20Ott&amp;journal=Phil%20Trans%20R%20Soc%20series%20B%20Biol%20Sci&amp;volume=363&amp;pages=3977-3984&amp;publication_year=2008&amp;doi=10.1098%2Frstb.2008.0163"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Fletcher
 W, Yang Z: INDELible: a flexible simulator of biological sequence 
evolution. Molecular biology and evolution 2009, 26(8):1879–1888. 
10.1093/molbev/msp098</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2712615"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/molbev/msp098"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19423664"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=INDELible%3A%20a%20flexible%20simulator%20of%20biological%20sequence%20evolution&amp;author=W.%20Fletcher&amp;author=Z.%20Yang&amp;journal=Molecular%20biology%20and%20evolution&amp;volume=26&amp;issue=8&amp;pages=1879-1888&amp;publication_year=2009&amp;doi=10.1093%2Fmolbev%2Fmsp098"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#IzquierdoCarrascoetallicenseeBioMedCentralLtd" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="IzquierdoCarrascoetallicenseeBioMedCentralLtd">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="IzquierdoCarrascoetallicenseeBioMedCentralLtd"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Izquierdo-Carrasco et al; licensee BioMed Central Ltd.&nbsp;2011<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published
 under license to BioMed Central Ltd. This is an Open Access article 
distributed under the terms of the Creative Commons Attribution License (<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0" target="_blank" class="ext-link"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-12-470" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/1471-2105-12-470">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/1471-2105-12-470" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-12-470?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
    <div id="articleSections" class="c-separator">
        <h3 class="p-section-title">Section</h3>
        <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
            
                <li class="c-list-group__item">
                    <a href="https://bmcbioinformatics.biomedcentral.com/articles/sections/transcriptome-analysis" data-track="click" data-track-category="Article Page" data-track-action="Click section link" data-track-label="10.1186/1471-2105-12-470">Transcriptome analysis</a>
                </li>
            
        </ul>
    </div>
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    7237
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 34 <a href="http://citations.springer.com/item?doi=10.1186/1471-2105-12-470" rel="noopener" target="_blank">more information</a>
            </li>
        
                                
    <li class="c-list-group__item">
        <p>Altmetric Attention Score: 1</p>
        <div class="altmetric">
            <a class="altmetric__link" href="http://www.altmetric.com/details.php?citation_id=491880&amp;domain=www.biomedcentral.com" target="_blank" rel="noopener" data-track="click" data-track-category="Article Page" data-track-action="Click Altmetric" data-track-label="10.1186/1471-2105-12-470">
                <img class="altmetric__img" src="a.png" alt="Altmetric Attention Score">
            </a>
        </div>
    </li>

                            </ul>
                        </div>
                        
    <div class="c-separator">
        <div class="js-shown">
            <h3 class="p-section-title">Share this article</h3>
            <ul data-test="social-media-share-buttons" class="c-social-links">
                <li class="c-social-links__item">
                    <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-12-470">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-12-470">Share on Twitter</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-12-470">
                            <use xlink:href="#icon-twitter"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-12-470">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-12-470">Share on Facebook</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-12-470">
                            <use xlink:href="#icon-facebook"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-12-470">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-12-470">Share on LinkedIn</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-12-470">
                            <use xlink:href="#icon-linkedin"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-12-470">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-12-470">Share on Weibo</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-12-470">
                            <use xlink:href="#icon-weibo"></use>
                        </svg>
                    </button>
                </li>
                <li class="c-social-links__item">
                    <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-12-470">
                        <span class="u-visually-hidden" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-12-470">Share on Reddit</span>
                        <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-12-470">
                            <use xlink:href="#icon-reddit"></use>
                        </svg>
                    </button>
                </li>
            </ul>
        </div>
        
            <div data-sharedit="container">
                <a class="c-button u-margin-top-sm" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-12-470/sharedit" rel="nofollow" data-sharedit="button" data-track="click" data-track-category="sharedit" data-track-action="click article sharedit button" data-track-label="10.1186/1471-2105-12-470">Get shareable link</a>
            </div>
        
    </div>
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/1471-2105-12-470" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2F1471-2105-12-470&amp;volume=12&amp;issue=1&amp;title=Algorithms%2C+data+structures%2C+and+numerics+for+likelihood-based+phylogenetic+inference+of+huge+trees&amp;author_name=Fernando+Izquierdo-Carrasco&amp;start_page=1&amp;end_page=14">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Bioinformatics</span>
                        </h4>
                         <p class="c-journal-footer__issn">ISSN: 1471-2105</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:bmcbioinformatics@biomedcentral.com">bmcbioinformatics@biomedcentral.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1471-2105-12-470.png" alt="">

         
    <footer>
        <div class="c-publisher-footer" data-test="publisher-footer">
    <div class="u-container">
        
        <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
            <div class="c-flex" aria-label="Publisher footer links">
                
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                </li>
                            
                        </ul>
                    
                        <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/contact-us">Support and Contact</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                </li>
                            
                                <li class="c-list-group__item">
                                    <a class="u-gray-link" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                </li>
                            
                        </ul>
                    
                
            </div>
            <div class="u-margin-bottom-lg">
                <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                <ul class="c-flex u-reset-list" data-test="footer-social-links">
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://twitter.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Twitter page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-twitter-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="https://www.facebook.com/BioMedCentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Facebook page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-facebook-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                        <li class="u-margin-top-xs u-margin-right-xs">
                            <a href="http://www.weibo.com/biomedcentral" class="u-gray-link">
                                <span class="u-visually-hidden">BMC Weibo page</span>
                                <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                    <use xlink:href="#icon-weibo-bordered"></use>
                                </svg>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </div>
        <p class="u-reset-margin">
            By using this website, you agree to our
            <a class="u-gray-link" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
            <a class="u-gray-link" href="https://www.biomedcentral.com/privacy-statement">Privacy
                statement</a> and
            <a class="u-gray-link" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
            
                <a class="optanon-toggle-display u-gray-link" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
            
        </p>
    </div>
</div>

        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo">
        <p data-test="copyright">© 2019 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com/" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

        
<script>
    (function() {
        if (window.config && window.config.mustardcut) {
            var appScript = document.createElement('script');
            appScript.src = '/static/js/app-bundle-9819d71f7a.js';
            appScript.async = false;
            document.body.appendChild(appScript);
        }
    })();
</script><script src="app-bundle-9819d71f7a.js"></script>

    


<iframe src="https://event-tracker.springernature.com" title="EventTracker" id="event-tracker" style="display:none; visibility:hidden" aria-hidden="true" tabindex="-1" width="0" height="0"></iframe><script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="https://www.googletagservices.com/tag/js/gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(27);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var d=[],e="",g=[],h=a.split(";");for(a=0;a<h.length;++a){void 0!==k&&(e=k);var b=h[a].split("\x3d");var k=b[0];k!==e&&(0<e.length&&0<d.length&&g.push([e,d]),d=[]);if(2===b.length&&""!==b[0]&&""!==b[1]){var c=b[1].split(",");for(b=0;b<c.length;++b)d.push(c[b])}}0<k.length&&0<d.length&&g.push([k,d]);return g}
function splitSizes(a){var d=[];if(null!==a){var e=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<e.length;++a){var g=e[a].split("x");var h=parseInt(g[0],10);var b=parseInt(g[1],10);2===g.length&&!isNaN(h)&&!isNaN(b)&&0<=h&&0<=b&&d.push([h,b])}}return d}function debounce(a,d){var e=null,g=null;return function(){var h=this,b=Number(new Date),k=arguments;e&&b<e+d?(clearTimeout(g),g=setTimeout(function(){e=b;a.apply(h,k)},d)):(e=b,a.apply(h,k))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,d){function e(f){var a="test";a=a.replace(/[\[\]]/g,"\\$\x26");a=new RegExp("[?\x26]"+a+"(\x3d([^\x26#]*)|\x26|#|$)");a=(a=a.exec(window.location.href))?a[2]?decodeURIComponent(a[2].replace(/\+/g," ")):"":null;a=a&&"ads"===a?"adtype\x3dtest;":"";f=f.getAttribute("data-gpt-targeting");var c=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var d=isLoggedIn();a&&-1===f.indexOf(a)&&(f+=a);c&&
(f+=c);return f+d+b}function g(a,c){for(var f=a.length,b=[];f--;)c(a[f],f)&&(b.push(a[f].slot),a.splice(f,1));b.length&&googletag.pubads().refresh(b);return a}function h(f){var c=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return g(f,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})}var b,k=getAdContainers(),c=[];d&&(b=d.dfppKeyValues?d.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var m=
0;k[m];++m){var l=k[m];splitKeys(e(l));c.push({divId:l.getAttribute("id"),adUnitPath:l.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(l.getAttribute("data-gpt-sizes")),targeting:splitKeys(e(l))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(31)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());
for(var b=0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(p){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=h(c)});var n=debounce(function(){googletag.cmd.push(function(){c=h(c);c.length||removeScrollEvent(n)})},250);addScrollEvent(n)})(window,
window.Krux);</script><script type="text/javascript" id="ga-data-track-event-listeners">var setupHandler=google_tag_manager["GTM-TDGJHK"].macro(39);setupHandler("click");setupHandler("submit");</script><script type="text/javascript" id="crossmark-script" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="recommended-script" src="https://recommended.springernature.com/latest/entry-point.js"></script><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; min-width: 0px; max-width: none; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size4,sans-serif;"></div></div></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Fbmcbioinformatics.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>