<!DOCTYPE html>
<html class="no-js" lang="en"><head><script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">

        <title>general species delimitation method with applications to phylogenetic placements | Bioinformatics | Oxford Academic</title>

        <script type="text/javascript" async="" src="recaptcha__en.js"></script><script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>
<script src="jquery-migrate-1.js"></script>



    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842209231/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site2065275106.css" rel="stylesheet" type="text/css">

        <script>
        (function (w, d, s, l, i) {
          w[l] = w[l] || []; w[l].push({
            'gtm.start':
              new Date().getTime(), event: 'gtm.js'
          }); var f = d.getElementsByTagName(s)[0],
            j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
            'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
        })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    

        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'bioinformatics';
            App.SiteURL = 'academic.oup.com/bioinformatics';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/bioinformatics/article/29/22/2869/314968","@type":"ScholarlyArticle","name":"A general species delimitation method with applications to phylogenetic placements","datePublished":"2013-08-29","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/bioinformatics/issue/29/22","@type":"PublicationIssue","issueNumber":"22","datePublished":"2013-11-15","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/bioinformatics","@type":"Periodical","name":"Bioinformatics","issn":["1460-2059"]}},"url":"http://dx.doi.org/10.1093/bioinformatics/btt499","inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2020","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/bioinformatics/article/29/22/2869/314968","author":[{"name":"Zhang, Jiajie","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece","@type":"Person"},{"name":"Kapli, Paschalia","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece","@type":"Person"},{"name":"Pavlidis, Pavlos","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece","@type":"Person"},{"name":"Stamatakis, Alexandros","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece","@type":"Person"}],"description":"Abstract.  Motivation: Sequence-based methods to delimit species are central to DNA taxonomy, microbial community surveys and DNA metabarcoding studies. Current","pageStart":"2869","pageEnd":"2876","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/29/22/0/m_bioinformatics29_22.cover.gif?Expires=1645104693&Signature=i0itJzJUtxG91QzNsQ69vcG3sJsDbKoCYO-Fa6VIVJu5i4wIS2vQGVYIpSFvRUI5eSFP~n0QvjpbhXzwcqfOCyHiJQhdKqBZiKZz5y6BHV0H-26d3DWDYAzUnGQB41EVxBgEnjgLknnLHomSxslt6Vp7Z~2jO7EanX-hiuVLFJFMOmIAZ8heGUGsBgK7TPqbVF9YLyF8yBMripVpR4jhcQ~3hHgud-uVXhunnKf6fvtz0VV4FXWmeISrBssY3nXj79yP5wgloUpKzMeR28AcP8~rC9QlMXrYE4BQKNatC9R~hxxnVUJjYifP1SDC4bT3kDPkhGywxzmS~7LH~4HgUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A general species delimitation method with applications to phylogenetic placements","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/29/22/0/m_bioinformatics29_22.cover.gif?Expires=1645104693&Signature=i0itJzJUtxG91QzNsQ69vcG3sJsDbKoCYO-Fa6VIVJu5i4wIS2vQGVYIpSFvRUI5eSFP~n0QvjpbhXzwcqfOCyHiJQhdKqBZiKZz5y6BHV0H-26d3DWDYAzUnGQB41EVxBgEnjgLknnLHomSxslt6Vp7Z~2jO7EanX-hiuVLFJFMOmIAZ8heGUGsBgK7TPqbVF9YLyF8yBMripVpR4jhcQ~3hHgud-uVXhunnKf6fvtz0VV4FXWmeISrBssY3nXj79yP5wgloUpKzMeR28AcP8~rC9QlMXrYE4BQKNatC9R~hxxnVUJjYifP1SDC4bT3kDPkhGywxzmS~7LH~4HgUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="general species delimitation method with applications to phylogenetic placements">
    <meta property="og:description" content="Abstract.  Motivation: Sequence-based methods to delimit species are central to DNA taxonomy, microbial community surveys and DNA metabarcoding studies. Current">
    <meta property="og:url" content="https://academic.oup.com/bioinformatics/article/29/22/2869/314968">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/29/22/0/m_bioinformatics29_22.cover.gif?Expires=1645104693&amp;Signature=i0itJzJUtxG91QzNsQ69vcG3sJsDbKoCYO-Fa6VIVJu5i4wIS2vQGVYIpSFvRUI5eSFP~n0QvjpbhXzwcqfOCyHiJQhdKqBZiKZz5y6BHV0H-26d3DWDYAzUnGQB41EVxBgEnjgLknnLHomSxslt6Vp7Z~2jO7EanX-hiuVLFJFMOmIAZ8heGUGsBgK7TPqbVF9YLyF8yBMripVpR4jhcQ~3hHgud-uVXhunnKf6fvtz0VV4FXWmeISrBssY3nXj79yP5wgloUpKzMeR28AcP8~rC9QlMXrYE4BQKNatC9R~hxxnVUJjYifP1SDC4bT3kDPkhGywxzmS~7LH~4HgUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2013-08-29">
<meta name="citation_author" content="Zhang, Jiajie"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, 2Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, 3Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, 4Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and 5Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece"><meta name="citation_author" content="Kapli, Paschalia"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, 2Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, 3Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, 4Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and 5Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece"><meta name="citation_author" content="Pavlidis, Pavlos"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, 2Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, 3Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, 4Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and 5Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece"><meta name="citation_author" content="Stamatakis, Alexandros"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, 2Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, 3Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, 4Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and 5Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece"><meta name="citation_title" content="A general species delimitation method with applications to phylogenetic placements"><meta name="citation_firstpage" content="2869"><meta name="citation_lastpage" content="2876"><meta name="citation_doi" content="10.1093/bioinformatics/btt499"><meta name="citation_journal_title" content="Bioinformatics"><meta name="citation_journal_abbrev" content="Bioinformatics"><meta name="citation_volume" content="29"><meta name="citation_issue" content="22"><meta name="citation_publication_date" content="2013/11/15"><meta name="citation_issn" content="1367-4803"><meta name="citation_publisher" content="Oxford Academic"><meta name="citation_reference" content="citation_title=Phylogenetics and speciation; Trends Ecol. Evol.;  citation_year=2001;  citation_volume=16;  citation_pages=391-399; "><meta name="citation_reference" content="citation_title=Choosing among alternative ‘phylogenetic’ species concepts; Syst. Bot.;  citation_year=1995;  citation_volume=20;  citation_pages=560-573; "><meta name="citation_reference" content="citation_title=Genealogical perspectives on the species problem; citation_publisher=Missouri Botanical Garden, ; Experimental and Molecular Approaches to Plant Biosystematics;  citation_year=1995;  citation_pages=289-303; "><meta name="citation_reference" content="citation_title=Performance, accuracy, and Web server for evolutionary placement of short sequence reads under maximum likelihood; Syst. Biol.;  citation_year=2011;  citation_volume=60;  citation_pages=291-302; "><meta name="citation_reference" content="citation_title=Sequencing our way towards understanding global eukaryotic biodiversity; Trends Ecol. Evol.;  citation_year=2012;  citation_volume=27;  citation_pages=233-243; "><meta name="citation_reference" content="citation_title=ESPRIT-Tree: hierarchical clustering analysis of millions of 16S rRNA pyrosequences in quasilinear computational time; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=e95"><meta name="citation_reference" content="citation_title=Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample; Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=4516-4522; "><meta name="citation_reference" content="citation_title=Species delimitation using a combined coalescent and information-theoretic approach: an example from North American Myotis bats; Syst. Biol.;  citation_year=2010;  citation_volume=59;  citation_pages=400-414; "><meta name="citation_reference" content="citation_title=Bioinformatic challenges for DNA metabarcoding of plants and animals; Mol. Ecol.;  citation_year=2012;  citation_volume=21;  citation_pages=1834-1847; "><meta name="citation_reference" content="citation_title=The Ribosomal Database Project: improved alignments and new tools for rRNA analysis; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D141-D145; "><meta name="citation_reference" content="citation_title=Divergence times and colonization of the Canary Islands by Gallotia lizards; Mol. Phylogenet. Evol.;  citation_year=2010;  citation_volume=56;  citation_pages=747-757; "><meta name="citation_reference" content="citation_title=Species concepts and speciation analysis; Curr. Ornithol.;  citation_year=1983;  citation_volume=1;  citation_pages=159-187; "><meta name="citation_reference" content="citation_title=Populations, genetic variation, and the delimitation of phylogenetic species; Syst. Biol.;  citation_year=1992;  citation_volume=41;  citation_pages=421-435; "><meta name="citation_reference" content="citation_title=BEAST: bayesian evolutionary analysis by sampling trees; BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=214"><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference; Genome Inform.;  citation_year=2009;  citation_volume=23;  citation_pages=205-211; "><meta name="citation_reference" content="citation_title=Search and clustering orders of magnitude faster than BLAST; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2460-2461; "><meta name="citation_reference" content="citation_title=UCHIME improves sensitivity and speed of chimera detection; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2194-2200; "><meta name="citation_reference" content="citation_publisher=Columbia Univ Press, New York; Phylogenetic Patterns and the Evolutionary Process: Method and Theory in Comparative Biology;  citation_year=1980; "><meta name="citation_reference" content="citation_title=SpedeSTEM: a rapid and accurate method for species delimitation; Mol. Ecol. Resour.;  citation_year=2011;  citation_volume=11;  citation_pages=473-480; "><meta name="citation_reference" content="citation_title=Single-locus species delimitation: a test of the mixed Yule-coalescent model, with an empirical application to Philippine round-leaf bats; Proc. Biol. Sci.;  citation_year=2012;  citation_volume=279;  citation_pages=3678-3686; "><meta name="citation_reference" content="citation_title=INDELible: a flexible simulator of biological sequence evolution; Mol. Biol. Evol.;  citation_year=2009;  citation_volume=26;  citation_pages=1879-1888; "><meta name="citation_reference" content="citation_title=Molecular barcodes for soil nematode identification; Mol. Ecol.;  citation_year=2002;  citation_volume=11;  citation_pages=839-850; "><meta name="citation_reference" content="citation_title=Independently evolving species in asexual bdelloid rotifers; PLoS Biol.;  citation_year=2007;  citation_volume=5;  citation_pages=e87"><meta name="citation_reference" content="citation_title=CD-HIT: accelerated for clustering the next-generation sequencing data; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3150-3152; "><meta name="citation_reference" content="citation_title=Delimiting species using single-locus data and the generalized mixed yule coalescent (GMYC) approach: a revised method and evaluation on simulated datasets; Syst. Biol.;  citation_year=2013;  citation_volume=62;  citation_pages=707-724; "><meta name="citation_reference" content="citation_title=Coalescent-based species delimitation in an integrative taxonomy; Trends Ecol. Evol.;  citation_year=2012;  citation_volume=27;  citation_pages=480-488; "><meta name="citation_reference" content="citation_title=Conservation genetics at the species boundary; Conserv. Biol.;  citation_year=2000;  citation_volume=14;  citation_pages=120-131; "><meta name="citation_reference" content="citation_title=Integrating DNA barcode data and taxonomic practice: determination, discovery, and description; Bioessays;  citation_year=2011;  citation_volume=33;  citation_pages=135-147; "><meta name="citation_reference" content="citation_title=Clustering 16S rRNA for OTU prediction: a method of unsupervised Bayesian clustering; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=611-618; "><meta name="citation_reference" content="citation_title=A dirichlet process prior for estimating lineage-specific substitution rates; Mol. Biol. Evol.;  citation_year=2012;  citation_volume=29;  citation_pages=939-955; "><meta name="citation_reference" content="citation_title=Generating samples under a Wright-Fisher neutral model of genetic variation; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=337-338; "><meta name="citation_reference" content="citation_title=ETE: a python environment for tree exploration; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=24"><meta name="citation_reference" content="citation_title=Accurate taxonomy assignments from 16S rRNA sequences produced by highly parallel pyrosequencers; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=e120"><meta name="citation_reference" content="citation_title=pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=538"><meta name="citation_reference" content="citation_title=DNA barcoding: error rates based on comprehensive sampling; PLoS Biol.;  citation_year=2005;  citation_volume=3;  citation_pages=e422"><meta name="citation_reference" content="citation_title=Accelerated species inventory on Madagascar using coalescent-based models of species delineation; Syst. Biol.;  citation_year=2009;  citation_volume=58;  citation_pages=298-311; "><meta name="citation_reference" content="citation_title=An amplification of the phylogenetic species concept; Cladistics;  citation_year=1990;  citation_volume=6;  citation_pages=211-223; "><meta name="citation_reference" content="citation_title=The integrative future of taxonomy; Front. Zool.;  citation_year=2010;  citation_volume=7;  citation_pages=16"><meta name="citation_reference" content="citation_title=Revisiting the insect mitochondrial molecular clock: the mid-Aegean trench calibration; Mol. Biol. Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=1659-1672; "><meta name="citation_reference" content="citation_title=Sequence-based species delimitation for the DNA taxonomy of undescribed insects; Syst. Biol.;  citation_year=2006;  citation_volume=55;  citation_pages=595-609; "><meta name="citation_reference" content="citation_title=Accounting for uncertainty in species delineation during the analysis of environmental DNA sequence data; Methods Ecol. Evol.;  citation_year=2012;  citation_volume=3;  citation_pages=1-11; "><meta name="citation_reference" content="citation_title=ABGD, Automatic Barcode Gap Discovery for primary species delimitation; Mol. Ecol.;  citation_year=2012;  citation_volume=21;  citation_pages=1864-1877; "><meta name="citation_reference" content="citation_title=r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=301-302; "><meta name="citation_reference" content="citation_title=A comparison of DNA-based methods for delimiting species in a Cretan land snail radiation reveals shortcomings of exclusively molecular taxonomy; Cladistics;  citation_year=2012;  citation_volume=28;  citation_pages=300-316; "><meta name="citation_reference" content="citation_title=Assessing and improving methods used in operational taxonomic unit-based approaches for 16S rRNA gene sequence analysis; App. Environ. Microbiol.;  citation_year=2011;  citation_volume=77;  citation_pages=3219-3226; "><meta name="citation_reference" content="citation_title=Delimiting species: a Renaissance issue in systematic biology; Trends Ecol. Evol.;  citation_year=2003;  citation_volume=18;  citation_pages=462-470; "><meta name="citation_reference" content="citation_title=Operational criteria for delimiting species; Annu. Rev. Ecol. Evol. Syst.;  citation_year=2004;  citation_volume=35;  citation_pages=199-227; "><meta name="citation_reference" content="citation_title=Taxonomic note: a place for DNA-DNA reassociation and 16S rRNA sequence analysis in the present species definition in bacteriology; Int.J. Syst. Bacteriol.;  citation_year=1994;  citation_volume=44;  citation_pages=846-849; "><meta name="citation_reference" content="citation_title=The Bioperl toolkit: perl modules for the life sciences; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1611-1618; "><meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2688-90; "><meta name="citation_reference" content="citation_title=A large-scale benchmark study of existing algorithms for taxonomy-independent microbial community analysis; Briefings Bioinform.;  citation_year=2012;  citation_volume=13;  citation_pages=107-121; "><meta name="citation_reference" content="citation_title=Information theoretic measures for clusterings comparison: Variants, properties, normalization and correction for chance; J. Mach. Learn. Res.;  citation_year=2010;  citation_volume=11;  citation_pages=2837-2854; "><meta name="citation_reference" content="citation_title=Recent advances in DNA taxonomy; J. Zool. Syst. Evol. Res.;  citation_year=2007;  citation_volume=45;  citation_pages=1-10; "><meta name="citation_reference" content="citation_title=Toward a DNA taxonomy of Alpine Rhithrogena (Ephemeroptera: Heptageniidae) using a mixed Yule-coalescent analysis of mitochondrial and nuclear DNA; PloS One;  citation_year=2011;  citation_volume=6;  citation_pages=e19728"><meta name="citation_reference" content="citation_title=Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy; App. Environ. Microbiol.;  citation_year=2007;  citation_volume=73;  citation_pages=5261-5267; "><meta name="citation_reference" content="citation_title=Bayesian species delimitation using multilocus sequence data; Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=9264-9269; "><meta name="citation_reference" content="citation_title=Biodiversity soup: metabarcoding of arthropods for rapid biodiversity assessment and biomonitoring; Methods Ecol. Evol.;  citation_year=2012;  citation_volume=3;  citation_pages=613-623; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/bioinformatics/article-pdf/29/22/2869/17100395/btt499.pdf"><meta name="description" content="Abstract.  Motivation: Sequence-based methods to delimit species are central to DNA taxonomy, microbial community surveys and DNA metabarcoding studies. Current">    <script id="altmetric-embed-js" src="altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1.js"></script><link rel="preload" href="integrator.js" as="script"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2020013001.js" async=""></script><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2667a43520a.css"></head><body data-sitestyletemplate="Journal" theme-bioinformatics="" class="off-canvas pg_Article pg_article" data-sitename="bioinformatics"><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
<meta name="citation_xml_url" content="https://academic.oup.com/bioinformatics/article-xml/29/22/2869/314968">



    <link rel="canonical" href="https://academic.oup.com/bioinformatics/article/29/22/2869/314968">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "bioinformatics/article/29/22/2869/314968");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    


    



    



    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=bioinformatics&authzrequired=false&doi=10.1093/bioinformatics/btt499';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5139" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search Menu</span></i>
                    </a>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </a>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </a>
                </li>

                
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f29%2f22%2f2869%2f314968" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <a href="javascript:;" id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f29%2f22%2f2869%2f314968" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387512">
                <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link ">Issues</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387513">
                <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link ">Advance articles</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387510">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">Submit</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387514">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387515">
                <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link ">Submission Site</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387516">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Open Access</a>
                
            </li>
    </ul>

            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387517">
                <a href="https://academic.oup.com/bioinformatics/subscribe" class="nav-link ">Purchase</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387518">
                <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387511">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">About</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387519">
                <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link ">About Bioinformatics</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387520">
                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387521">
                <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387522">
                <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387523">
                <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387524">
                <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
                
            </li>
    </ul>

            </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/bioinformatics" class="journal-logo-container">
                    <img id="logo-Bioinformatics" class="journal-logo" src="bioinformatics_title-205396838.svg" alt="Bioinformatics">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.iscb.org/" target="" class="society-logo-container">
                                <img id="logo-InternationalSocietyforComputationalBiology" class="society-logo" src="bioinformatics_h11820233911.svg" alt="International Society for Computational Biology">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387512">
                <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link ">Issues</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387513">
                <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link ">Advance articles</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387510">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">Submit</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387514">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387515">
                <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link ">Submission Site</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387516">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Open Access</a>
                
            </li>
    </ul>

            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387517">
                <a href="https://academic.oup.com/bioinformatics/subscribe" class="nav-link ">Purchase</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387518">
                <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387511">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">About</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387519">
                <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link ">About Bioinformatics</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387520">
                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387521">
                <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387522">
                <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387523">
                <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387524">
                <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
                
            </li>
    </ul>

            </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <a href="javascript:;" class="navbar-search-close js_close-navsearch">Close</a>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Bioinformatics</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <a href="javascript:;" id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch"><span class="screenreader-text">Search Menu</span></a></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5139" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=11777" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/bioinformatics" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="bioinformatics" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/bioinformatics/issue/29/22"><img id="issueImage" class="fb-featured-image" src="m_bioinformatics29_22.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/bioinformatics/issue/29/22">
                        <div class="volume trailing-comma">Volume 29</div>
                        <div class="issue">Issue 22</div>

            </a>

        </div>
        <div class="ii-pub-date">
15 November 2013        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="84627098">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627098">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="84627099">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627099">1 INTRODUCTION</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="84627109">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627109">2 METHODS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="84627124">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627124">3 EXPERIMENTAL SETUP</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="84627141">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627141">4 RESULTS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="84627156">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627156">5 DISCUSSION</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="84627170">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627170">ACKNOWLEDGEMENTS</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="84627174">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#84627174">REFERENCES</a>
            </div>
        </li>
<li class="section-jump-link head-1"><div class="section-jump-link__link-wrap"><a class="jumplink js-jumplink scrollTo" href="#authorNotesSectionTitle">Author notes</a></div></li><li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/bioinformatics/article/29/22/2859/316554">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/bioinformatics/article/29/22/2877/313226">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    A general species delimitation method with applications to phylogenetic placements
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiajie Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, <sup>2</sup>Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, <sup>3</sup>Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, <sup>4</sup>Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and <sup>5</sup>Institute
 of Molecular Biology and Biotechnology, Foundation for Research and 
Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Jiajie+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20Jiajie%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Jiajie Zhang</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paschalia Kapli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, <sup>2</sup>Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, <sup>3</sup>Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, <sup>4</sup>Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and <sup>5</sup>Institute
 of Molecular Biology and Biotechnology, Foundation for Research and 
Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Paschalia+Kapli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kapli%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kapli%20Paschalia%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Paschalia Kapli</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pavlos Pavlidis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, <sup>2</sup>Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, <sup>3</sup>Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, <sup>4</sup>Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and <sup>5</sup>Institute
 of Molecular Biology and Biotechnology, Foundation for Research and 
Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Pavlos+Pavlidis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pavlidis%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Pavlidis%20Pavlos%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Pavlos Pavlidis</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexandros Stamatakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, D-68159 Heidelberg, Germany, <sup>2</sup>Graduate School for Computing in Medicine and Life Sciences, University of Lübeck, <sup>3</sup>Institut für Neuro- und Bioinformatik, University of Lübeck, 23538 Lübeck, Germany, <sup>4</sup>Natural History Museum of Crete, University of Crete, GR-71409 Irakleio, Crete, Greece and <sup>5</sup>Institute
 of Molecular Biology and Biotechnology, Foundation for Research and 
Technology-Hellas-FORTH, GR-70013 Heraklion, Crete, Greece</div>
        </div>


        <div class="info-author-correspondence">
            *To whom correspondence should be addressed.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Alexandros+Stamatakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stamatakis%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stamatakis%20Alexandros%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Alexandros Stamatakis</a><span class="delimiter"><i class="icon-general-mail"></i></span>
                        </span>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Associate Editor: David Posada</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Bioinformatics</em>, Volume 29, Issue 22, 15 November 2013, Pages 2869–2876, <a href="https://doi.org/10.1093/bioinformatics/btt499">https://doi.org/10.1093/bioinformatics/btt499</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">29 August 2013</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">09 April 2013</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Revision received:</div>
                    <div class="wi-date">20 August 2013</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">21 August 2013</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">29 August 2013</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body js-content-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="314968" href="https://academic.oup.com/bioinformatics/article-pdf/29/22/2869/17100395/btt499.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        <a href="#" class="js-cite-button" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <h3 class="modal-title">Cite</h3>
        <div class="oxford-citation-text">
            <p>Jiajie Zhang, Paschalia Kapli, Pavlos Pavlidis, 
Alexandros Stamatakis,  A general species delimitation method with 
applications to phylogenetic placements, <em>Bioinformatics</em>, Volume 29, Issue 22, 15 November 2013, Pages 2869–2876, <a href="https://doi.org/10.1093/bioinformatics/btt499">https://doi.org/10.1093/bioinformatics/btt499</a></p>
        </div>

    <div class="citation-download-wrap">
        <form action="/Citation/Download" method="get" id="citationModal">
            <input name="resourceId" value="314968" type="hidden">
            <input name="resourceType" value="3" type="hidden">
            <label for="selectFormat" class="hide">Select Format</label>
            <select required="" name="citationFormat" class="citation-download-format" id="selectFormat">
                <option selected="selected" disabled="disabled">Select format</option>
                        <option value="0">.ris (Mendeley, Papers, Zotero)</option>
                        <option value="1">.enw (EndNote)</option>
                        <option value="2">.bibtex (BibTex)</option>
                        <option value="3">.txt (Medlars, RefWorks)</option>

            </select>
            <button class="btn citation-download-link disabled" type="submit">Download citation</button>
        </form>
    </div>

    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1460-2059&amp;title=A%20general%20species%20delimitation%20method%20with%20applications%20to%20phylogenetic%20placements&amp;publicationDate=2013-08-29&amp;volumeNum=29&amp;issueNum=22&amp;author=Zhang%2C%20Jiajie%3B%20Kapli%2C%20Paschalia&amp;startPage=2869&amp;endPage=2876&amp;contentId=10.1093%2Fbioinformatics%2Fbtt499&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <a href="javascript:;" id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f29%2f22%2f2869%2f314968" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <a href="javascript:;" class="navbar-search-close js_close-navsearch">Close</a>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Bioinformatics</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <a href="javascript:;" id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch"><span class="screenreader-text">Search Menu</span></a></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="bioinformatics" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="84627098" id="84627098" class="abstract-title">Abstract</h2>
<section class="abstract"><p><strong>Motivation:</strong> Sequence-based
 methods to delimit species are central to DNA taxonomy, microbial 
community surveys and DNA metabarcoding studies. Current approaches 
either rely on simple sequence similarity thresholds (OTU-picking) or on
 complex and compute-intensive evolutionary models. The OTU-picking 
methods scale well on large datasets, but the results are highly 
sensitive to the similarity threshold. Coalescent-based species 
delimitation approaches often rely on Bayesian statistics and Markov 
Chain Monte Carlo sampling, and can therefore only be applied to small 
datasets.</p><p><strong>Results:</strong> We introduce the Poisson tree 
processes (PTP) model to infer putative species boundaries on a given 
phylogenetic input tree. We also integrate PTP with our evolutionary 
placement algorithm (EPA-PTP) to count the number of species in 
phylogenetic placements. We compare our approaches with popular 
OTU-picking methods and the General Mixed Yule Coalescent (GMYC) model. 
For <em>de novo</em> species delimitation, the stand-alone PTP model 
generally outperforms GYMC as well as OTU-picking methods when 
evolutionary distances between species are small. PTP neither requires 
an ultrametric input tree nor a sequence similarity threshold as input. 
In the open reference species delimitation approach, EPA-PTP yields more
 accurate results than <em>de novo</em> species delimitation methods. 
Finally, EPA-PTP scales on large datasets because it relies on the 
parallel implementations of the EPA and RAxML, thereby allowing to 
delimit species in high-throughput sequencing data.</p><p><strong>Availability and implementation:</strong> The code is freely available at <a class="link link-uri openInAnotherWindow" href="http://www.exelixis-lab.org/software.html" target="_blank">www.exelixis-lab.org/software.html</a>.</p><p><strong>Contact:</strong><a href="mailto:Alexandros.Stamatakis@h-its.org" target="">Alexandros.Stamatakis@h-its.org</a></p><p><strong>Supplementary information:</strong><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056691&amp;Signature=a-EKrpvYhcGf6JW9KYopncPN3nvK3FPqeI~mBN7tsb3VQVs4NR23qq8h10DvMspohveDdhqeaCxg3SlcebXvd~JECTMPBQOmnUzq3BzrE7CWzqQaKCt2zEFzrlX4C41xoVrCrjR1azMJwN-0iVE4ixJD5JL~HEOti8Ti2ebqAxt8SiBehHH9H1qLGdpw2QQGR4Pfp9IDO9qMKzjyGTRFHkhdEpUGOpgmDH9Q5KiVBPeeqknMcGawztAFBPPEx3YlWxgTcZipx0a4WhqaGYUqaicybGyb6SZ93kbuDna91QYQsFD94KE8fevErnyL9UJZk~sKg0eI5ww-U9nXDCNl~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>Bioinformatics</em> online.</p></section>                        <div class="article-metadata-panel clearfix"></div>
                    <h2 scrollto-destination="84627099" id="84627099" class="section-title">1 INTRODUCTION</h2>
<p>DNA barcoding studies mostly rely on a single marker gene and are widely used for <em>DNA taxonomy</em> (<a reveal-id="btt499-B28" data-open="btt499-B28" class="link link-ref link-reveal xref-bibr">Goldstein and DeSalle, 2011</a>; <a reveal-id="btt499-B53" data-open="btt499-B53" class="link link-ref link-reveal xref-bibr">Vogler and Monaghan, 2007</a>).
 More recently, high-throughput sequencing of barcoding genes has been 
deployed to disentangle the structure of microbial communities (<a reveal-id="btt499-B7" data-open="btt499-B7" class="link link-ref link-reveal xref-bibr">Caporaso <em>et al.</em>, 2011</a>) and in <em>metabarcoding</em> biodiversity (<a reveal-id="btt499-B9" data-open="btt499-B9" class="link link-ref link-reveal xref-bibr">Coissac <em>et al.</em>, 2012</a>)
 studies. A central analytical task in such studies is to classify 
molecular sequences into entities that correspond to species; this is 
commonly denoted as OTU-picking in metagenomic studies (<a reveal-id="btt499-B51" data-open="btt499-B51" class="link link-ref link-reveal xref-bibr">Sun <em>et al.</em>, 2012</a>). The main goals of such methods are to identify known species and delimit new species (<a reveal-id="btt499-B53" data-open="btt499-B53" class="link link-ref link-reveal xref-bibr">Vogler and Monaghan, 2007</a>).</p><p>Numerous
 approaches exist for associating anonymous reads/query sequences with 
known species, for instance, nearest-neighbor BLAST (<a reveal-id="btt499-B33" data-open="btt499-B33" class="link link-ref link-reveal xref-bibr">Liu <em>et al.</em>, 2008</a>) or the naïve Bayesian classifier (<a reveal-id="btt499-B55" data-open="btt499-B55" class="link link-ref link-reveal xref-bibr">Wang <em>et al.</em>, 2007</a>).
 These methods use sequence similarity to associate reads with taxonomic
 ranks. Phylogeny-aware methods for identifying reads were introduced 
independently and simultaneously with the evolutionary placement 
algorithm (EPA; <a reveal-id="btt499-B4" data-open="btt499-B4" class="link link-ref link-reveal xref-bibr">Berger <em>et al.</em>, 2011</a>) and pplacer (<a reveal-id="btt499-B34" data-open="btt499-B34" class="link link-ref link-reveal xref-bibr">Matsen <em>et al.</em>, 2010</a>). Instead of sequence similarity, they use the phylogenetic signal in the reference <em>and</em>
 query sequences to attain higher classification accuracy. Note that 
obtaining a taxonomic classification from phylogenetic placements 
represents a difficult task because phylogenies and taxonomies are 
frequently incongruent (<a reveal-id="btt499-B10" data-open="btt499-B10" class="link link-ref link-reveal xref-bibr">Cole <em>et al.</em>, 2009</a>). Placement methods are similar to <em>closed-reference</em> OTU-picking (<a reveal-id="btt499-B5" data-open="btt499-B5" class="link link-ref link-reveal xref-bibr">Bik <em>et al.</em>, 2012</a>) or <em>taxonomy-dependent</em> methods (<a reveal-id="btt499-B45" data-open="btt499-B45" class="link link-ref link-reveal xref-bibr">Schloss and Westcott, 2011</a>).
 Their ability to associate query sequences with species depends on the 
completeness of the taxon sampling in the reference data (<a reveal-id="btt499-B35" data-open="btt499-B35" class="link link-ref link-reveal xref-bibr">Meyer and Paulay, 2005</a>).
 Closed-reference or taxonomy-dependent methods generally lack the 
ability to delimit new species; consequently, they may underestimate the
 number of species and hence the diversity in the query sequences (see 
example in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056691&amp;Signature=a-EKrpvYhcGf6JW9KYopncPN3nvK3FPqeI~mBN7tsb3VQVs4NR23qq8h10DvMspohveDdhqeaCxg3SlcebXvd~JECTMPBQOmnUzq3BzrE7CWzqQaKCt2zEFzrlX4C41xoVrCrjR1azMJwN-0iVE4ixJD5JL~HEOti8Ti2ebqAxt8SiBehHH9H1qLGdpw2QQGR4Pfp9IDO9qMKzjyGTRFHkhdEpUGOpgmDH9Q5KiVBPeeqknMcGawztAFBPPEx3YlWxgTcZipx0a4WhqaGYUqaicybGyb6SZ93kbuDna91QYQsFD94KE8fevErnyL9UJZk~sKg0eI5ww-U9nXDCNl~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).</p><p>To identify new species, <em>taxonomy-independent</em> methods or <em>de novo</em> OTU-picking approaches are used to initially cluster sequences into so-called molecular operational taxonomic units (MOTUs) (<a reveal-id="btt499-B22" data-open="btt499-B22" class="link link-ref link-reveal xref-bibr">Floyd <em>et al.</em>, 2002</a>; <a reveal-id="btt499-B28" data-open="btt499-B28" class="link link-ref link-reveal xref-bibr">Goldstein and DeSalle, 2011</a>; <a reveal-id="btt499-B53" data-open="btt499-B53" class="link link-ref link-reveal xref-bibr">Vogler and Monaghan, 2007</a>).
 Then, one can use a representative sequence from each MOTU cluster and 
assign a taxonomic rank via taxonomy-dependent methods. Although 
taxonomic assignments may still be inaccurate due to incomplete 
reference data, coarse-grain biodiversity estimates can be accurate when
 MOTUs are assigned to higher taxonomic ranks. <em>De novo</em> OTU-picking usually relies on unsupervised machine learning methods (<a reveal-id="btt499-B6" data-open="btt499-B6" class="link link-ref link-reveal xref-bibr">Cai and Sun, 2011</a>; <a reveal-id="btt499-B16" data-open="btt499-B16" class="link link-ref link-reveal xref-bibr">Edgar, 2010</a>; <a reveal-id="btt499-B24" data-open="btt499-B24" class="link link-ref link-reveal xref-bibr">Fu <em>et al.</em>, 2012</a>) that cluster sequences based on, mostly arbitrary, sequence similarity thresholds (<a reveal-id="btt499-B42" data-open="btt499-B42" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>; <a reveal-id="btt499-B45" data-open="btt499-B45" class="link link-ref link-reveal xref-bibr">Schloss and Westcott, 2011</a>). However, it is currently unclear how MOTUs correspond to species (<a reveal-id="btt499-B53" data-open="btt499-B53" class="link link-ref link-reveal xref-bibr">Vogler and Monaghan, 2007</a>).</p><p>To
 delimit species using molecular sequences, we initially need to define 
our species concept. The phylogenetic species concept (PSC) was 
initially introduced by <a reveal-id="btt499-B18" data-open="btt499-B18" class="link link-ref link-reveal xref-bibr">Eldredge and Cracraft (1980)</a> and subsequently refined by <a reveal-id="btt499-B2" data-open="btt499-B2" class="link link-ref link-reveal xref-bibr">Baum and Donoghue (1995)</a>; <a reveal-id="btt499-B12" data-open="btt499-B12" class="link link-ref link-reveal xref-bibr">Cracraft (1983)</a>; <a reveal-id="btt499-B13" data-open="btt499-B13" class="link link-ref link-reveal xref-bibr">Davis and Nixon (1992)</a>; and <a reveal-id="btt499-B37" data-open="btt499-B37" class="link link-ref link-reveal xref-bibr">Nixon and Wheeler (1990)</a>. For a review of PSCs definitions please refer to <a reveal-id="btt499-B3" data-open="btt499-B3" class="link link-ref link-reveal xref-bibr">Baum and Shaw (1995)</a>.
 In general, phylogenetic species are the smallest units for which 
phylogenetic relationships can be reliably inferred. The PSC, in 
particular, from the genealogical point of view (<a reveal-id="btt499-B3" data-open="btt499-B3" class="link link-ref link-reveal xref-bibr">Baum and Shaw, 1995</a>),
 states that species reside at the transition point between evolutionary
 relationships that are best represented phylogenetically and 
relationships that are best reflected by reticulating genealogical 
connections (<a reveal-id="btt499-B27" data-open="btt499-B27" class="link link-ref link-reveal xref-bibr">Goldstein and Desalle, 2000</a>).</p><p>There already exist several PSC-based species delimitation approaches (e.g. see reviews in <a reveal-id="btt499-B26" data-open="btt499-B26" class="link link-ref link-reveal xref-bibr">Fujita <em>et al.</em>, 2012</a>; <a reveal-id="btt499-B46" data-open="btt499-B46" class="link link-ref link-reveal xref-bibr">Sites and Marshall, 2003</a>, <a reveal-id="btt499-B47" data-open="btt499-B47" class="link link-ref link-reveal xref-bibr">2004</a>). The General Mixed Yule Coalescent (GMYC) model (<a reveal-id="btt499-B25" data-open="btt499-B25" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>; <a reveal-id="btt499-B40" data-open="btt499-B40" class="link link-ref link-reveal xref-bibr">Pons <em>et al.</em>, 2006</a>) for delimiting species on single genes is frequently used in empirical studies (<a reveal-id="btt499-B8" data-open="btt499-B8" class="link link-ref link-reveal xref-bibr">Carstens and Dewey, 2010</a>; <a reveal-id="btt499-B23" data-open="btt499-B23" class="link link-ref link-reveal xref-bibr">Fontaneto <em>et al.</em>, 2007</a>; <a reveal-id="btt499-B36" data-open="btt499-B36" class="link link-ref link-reveal xref-bibr">Monaghan <em>et al.</em>, 2009</a>; <a reveal-id="btt499-B41" data-open="btt499-B41" class="link link-ref link-reveal xref-bibr">Powell, 2012</a>; <a reveal-id="btt499-B54" data-open="btt499-B54" class="link link-ref link-reveal xref-bibr">Vuataz <em>et al.</em>, 2011</a>).</p><p>The
 GMYC method models speciation (among-species branching events) via a 
pure birth process and within-species branching events as neutral 
coalescent processes. GMYC identifies the transition points between 
inter- and intra-species branching rates on a time-calibrated 
ultrametric tree by maximizing the likelihood score of the model. It 
assumes that all lineages leading from the root to the transition points
 are different species. GMYC has been shown to work well for 
comparatively small population sizes and low birth rates (<a reveal-id="btt499-B20" data-open="btt499-B20" class="link link-ref link-reveal xref-bibr">Esselstyn <em>et al.</em>, 2012</a>).
 One drawback of GMYC is that it depends on the accuracy of the 
ultrametric input tree. Obtaining an ultrametric tree from a given 
phylogeny is a compute-intensive <em>and</em> potentially error-prone process. Most state-of-the-art likelihood-based tree calibration methods such as BEAST (<a reveal-id="btt499-B14" data-open="btt499-B14" class="link link-ref link-reveal xref-bibr">Drummond and Rambaut, 2007</a>) or DPPDIV (<a reveal-id="btt499-B30" data-open="btt499-B30" class="link link-ref link-reveal xref-bibr">Heath <em>et al.</em>, 2012</a>)
 rely on Bayesian sampling using MCMC (Markov Chain Monte Carlo) 
methods. When delimiting species in phylogenetic placements, which 
requires calibrating thousands of trees, it is almost impossible to 
deploy these methods in an automated pipeline, given the difficulties to
 assess MCMC chain convergence, for instance.</p><p>Inspired by the PSC,
 we introduce the PTP model that can delimit species using 
non-ultrametric phylogenies. Ultrametricity is not required because we 
model speciation rate by directly using the number of substitutions. The
 PSC implies that deploying phylogenetic reconstruction methods within a
 species is inappropriate. A hierarchical relationship can nonetheless 
be inferred for intra-species sequences using phylogenetic methods. 
However, we expect to observe significant (in the statistical sense) 
differences between the relationships reconstructed among and within 
species. These differences are reflected by branch lengths that 
represent the mean expected number of substitutions per site between two
 branching events. Thus, our fundamental assumption is that the number 
of substitutions between species is significantly higher than the number
 of substitutions within species. In a sense, this is analogous to the 
GMYC approach that intends to identify significant changes in the pace 
of branching events on the tree. However, GMYC uses time to identify 
branching rate transition points, whereas, in contrast, PTP directly 
uses the number of substitutions.</p><p>PTP is simple, fast and robust. 
Thus, it can easily be integrated with the EPA to calculate the number 
of species in a set of query sequences that have been placed into a 
specific branch of the reference phylogeny. We implemented an open 
reference species delimitation pipeline by integrating PTP with the EPA 
to identify known and new species.</p><p>Initially, we assess the performance of GMYC and the PTP approach as general <em>de novo</em>
 species delimitation methods using real and simulated data. We then 
compare PTP and GMYC with two representative OTU-picking methods UCLUST (<a reveal-id="btt499-B16" data-open="btt499-B16" class="link link-ref link-reveal xref-bibr">Edgar, 2010</a>) and CROP (<a reveal-id="btt499-B29" data-open="btt499-B29" class="link link-ref link-reveal xref-bibr">Hao <em>et al.</em>, 2011</a>).
 UCLUST represents a fixed threshold OTU-picking approach, whereas CROP 
is a soft threshold method that attempts to detect sequence clusters 
using a Gaussian mixture model. Finally, we evaluate the performance of 
our open reference approach EPA-PTP. For a fair comparison, we also 
integrated CROP with the EPA (EPA-CROP). </p>                    <h2 scrollto-destination="84627109" id="84627109" class="section-title">2 METHODS</h2>
                    <h3 scrollto-destination="84627110" id="84627110" class="section-title" data-legacy-id="SEC2.1">2.1 The Poisson tree processes model</h3>
<p>Classic speciation models such as the birth–death process (BDP) 
assume that new species will emerge and current species will become 
extinct at certain rates that are measured in unit time (<a reveal-id="btt499-B1" data-open="btt499-B1" class="link link-ref link-reveal xref-bibr">Barraclough and Nee, 2001</a>).
 Usually, a time-calibrated tree is required as an input. Thus, for 
molecular sequence data, a molecular clock model must be applied to 
calibrate the tree. Coalescent theory also relies on unit time to 
describe the relationships among ancestors and descendants in a 
population.</p><p>Instead, we may consider the number of substitutions 
between branching and/or speciation events, by modeling speciations 
using the number of substitutions instead of the time. The underlying 
assumption is that each substitution has a small probability of 
generating a speciation. Note that the substitutions are independent of 
each other. If we consider one substitution at a time in discrete steps,
 the probability of observing η speciations for κ substitutions is given
 by a binomial distribution. Because we assume that each substitution 
has a small probability ρ of generating a speciation, and the number of 
substitutions in a population of size η is large, the process follows a 
Poisson distribution in continuous time with rate <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i1.gif" alt="forumla" path-from-xml="btt499i1.gif">⁠</span>. Therefore, the number of substitutions until the next speciation event follows an exponential distribution.</p><p>Comparing
 this with the assumptions of a BDP, we observe that each generation 
(e.g. with a generation time of 20 years) on a time-calibrated 
ultrametric tree has a small probability of speciation. The BDP does not
 model substitutions, thus, substitutions are superimposed onto the BDP,
 whereas PTP explicitly models substitutions. Substitution information 
can easily be obtained by using the branch lengths of the phylogenetic 
input tree. Thus, in our model, the underlying assumptions for observing
 a branching event are consistent with the assumptions made for 
phylogenetic tree inference.</p><p>We can now consider two independent 
classes of Poisson processes. One process class describes speciation 
such that the average number of substitutions until the next speciation 
event follows an exponential distribution. Given the species tree, we 
can estimate the rate of speciations per substitution in a 
straightforward way. The second Poisson process class describes 
within-species branching events that are analogous to coalescent events.
 We assume that the number of substitutions until the next 
within-species branching event also follows an exponential distribution.
 Thus, our model assumes that the branch lengths of the input tree have 
been generated by two independent Poisson process classes.</p><p>In the following step, we assign/fit the Poisson processes to the tree. Let <em>T</em> be a rooted tree, and <em>P<sub>i</sub></em> be a path from the root to leaf <em>i</em>, where <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i2.gif" alt="forumla" path-from-xml="btt499i2.gif"></span> and <em>l</em> is the number of leaves. Let <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i3.gif" alt="forumla" path-from-xml="btt499i3.gif"></span> be the edge lengths of <em>P<sub>i</sub></em>, representing the number of substitutions. We further assume that <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i4.gif" alt="forumla" path-from-xml="btt499i4.gif"></span> are independent exponentially distributed random variables with parameter λ. Let <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i5.gif" alt="forumla" path-from-xml="btt499i5.gif"></span> be the sum over the edge lengths for <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i6.gif" alt="forumla" path-from-xml="btt499i6.gif">⁠</span>. We further define <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i7.gif" alt="forumla" path-from-xml="btt499i7.gif">⁠</span>. <em>B<sub>ik</sub></em> is the number of substitutions of the <em>k</em>th branching event, and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i8.gif" alt="forumla" path-from-xml="btt499i8.gif"></span> is the number of branching events below <em>B<sub>ik</sub></em>. Note that <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i9.gif" alt="forumla" path-from-xml="btt499i9.gif"></span> constitutes a Poisson process. Thereby, <em>T</em> and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i10.gif" alt="forumla" path-from-xml="btt499i10.gif"></span> together form a tree of Poisson processes, which we denote as <em>Poisson Tree Processes (PTP)</em>. To a rooted phylogeny with <em>m</em> species, we apply/fit one among-species PTP and at most <em>m</em> within-species PTPs. An example is shown in <a reveal-id="btt499-F1" data-open="btt499-F1" class="link link-reveal link-table xref-fig">Figure 1</a>. </p>                        <a id="84627116" scrollto-destination="84627116"></a>
<div id="btt499-F1" data-id="btt499-F1" class="fig fig-section"><div class="label fig-label">Fig. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_btt499f1p.gif" alt="Illustration of the PTP. The example tree contains 6 speciation events: R, A, B, D, E, F, and 4 species: C, D, E, F. Species C consists of one individual; species D, E, F have two individuals each. The thick lines represent among-species PTP, and the thin lines represent within-species PTPs. The Newick representation of this tree is ((C:0.14, (d1:0.01, d2:0.02)D:0.1)A:0.15, ((e1:0.015, e2:0.014)E:0.1, (f1:0.03, f2:0.02)F:0.12)B:0.11)R. The tree has a total of 16 different possible species delimitations. The maximum likelihood search returned the depicted species delimitation with a log-likelihood score of 24.77, and  = 8.33 and  = 55.05" path-from-xml="btt499f1p.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/84627116/btt499f1p.gif" path-from-xml="btt499f1p.gif" target="_blank">Open in new tab</a><a section="84627116" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499f1p.gif?Expires=1585056692&amp;Signature=z8g29~AQOqA5X~ohOlGjgUoJWeWwEt3rzURtA51PT-XvjvQztXJCHpQr3mvn4ofXQvwbNKuldywDPjU2sbPcsKOLhsY0im9FDjwqoBENIZC59WG0ejAqYtGUK7XefXmVybty8G~~SkKCJ76tN9Afg5hfFbeCH-jpMLmu0PzylFGMunS~jhF0T5sdHGAkLTyNff601V~IhjF4i52MwhkigD3akbTcMdG0L4Vi52HSuZmCLkujltnIYDEUfnQQB70W5hpcbylvW7rxr0np~-RF7UEhfXaySJdEiEr4GGllNiiH8P6u82XdZ5qwdEwaIvyYupRmiYZixeEjdJ6~ff2H7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=84627116&amp;ar=314968&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btt499f1p.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of the <em>PTP</em>. The example tree contains 6 speciation events: <em>R, A, B, D, E, F</em>, and 4 species: <em>C, D, E, F</em>. Species <em>C</em> consists of one individual; species <em>D, E, F</em> have two individuals each. The thick lines represent among-species <em>PTP</em>, and the thin lines represent within-species <em>PTPs</em>. The Newick representation of this tree is ((<em>C:0.14, (d1:0.01, d2:0.02)D:0.1)A:0.15, ((e1:0.015, e2:0.014)E:0.1, (f1:0.03, f2:0.02)F:0.12)B:0.11)R</em>.
 The tree has a total of 16 different possible species delimitations. 
The maximum likelihood search returned the depicted species delimitation
 with a log-likelihood score of 24.77, and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i11.gif" alt="forumla" path-from-xml="btt499i11.gif"></span> = 8.33 and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i12.gif" alt="forumla" path-from-xml="btt499i12.gif"></span> = 55.05</p></div></div><div content-id="btt499-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_btt499f1p.gif" alt="Illustration of the PTP. The example tree contains 6 speciation events: R, A, B, D, E, F, and 4 species: C, D, E, F. Species C consists of one individual; species D, E, F have two individuals each. The thick lines represent among-species PTP, and the thin lines represent within-species PTPs. The Newick representation of this tree is ((C:0.14, (d1:0.01, d2:0.02)D:0.1)A:0.15, ((e1:0.015, e2:0.014)E:0.1, (f1:0.03, f2:0.02)F:0.12)B:0.11)R. The tree has a total of 16 different possible species delimitations. The maximum likelihood search returned the depicted species delimitation with a log-likelihood score of 24.77, and  = 8.33 and  = 55.05" path-from-xml="btt499f1p.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/84627116/btt499f1p.gif" path-from-xml="btt499f1p.gif" target="_blank">Open in new tab</a><a section="84627116" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499f1p.gif?Expires=1585056692&amp;Signature=z8g29~AQOqA5X~ohOlGjgUoJWeWwEt3rzURtA51PT-XvjvQztXJCHpQr3mvn4ofXQvwbNKuldywDPjU2sbPcsKOLhsY0im9FDjwqoBENIZC59WG0ejAqYtGUK7XefXmVybty8G~~SkKCJ76tN9Afg5hfFbeCH-jpMLmu0PzylFGMunS~jhF0T5sdHGAkLTyNff601V~IhjF4i52MwhkigD3akbTcMdG0L4Vi52HSuZmCLkujltnIYDEUfnQQB70W5hpcbylvW7rxr0np~-RF7UEhfXaySJdEiEr4GGllNiiH8P6u82XdZ5qwdEwaIvyYupRmiYZixeEjdJ6~ff2H7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=84627116&amp;ar=314968&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btt499f1p.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of the <em>PTP</em>. The example tree contains 6 speciation events: <em>R, A, B, D, E, F</em>, and 4 species: <em>C, D, E, F</em>. Species <em>C</em> consists of one individual; species <em>D, E, F</em> have two individuals each. The thick lines represent among-species <em>PTP</em>, and the thin lines represent within-species <em>PTPs</em>. The Newick representation of this tree is ((<em>C:0.14, (d1:0.01, d2:0.02)D:0.1)A:0.15, ((e1:0.015, e2:0.014)E:0.1, (f1:0.03, f2:0.02)F:0.12)B:0.11)R</em>.
 The tree has a total of 16 different possible species delimitations. 
The maximum likelihood search returned the depicted species delimitation
 with a log-likelihood score of 24.77, and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i11.gif" alt="forumla" path-from-xml="btt499i11.gif"></span> = 8.33 and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i12.gif" alt="forumla" path-from-xml="btt499i12.gif"></span> = 55.05</p></div></div><div class="block-child-p">In analogy to BP&amp;P (<a reveal-id="btt499-B56" data-open="btt499-B56" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2010</a>) and GMYC (<a reveal-id="btt499-B40" data-open="btt499-B40" class="link link-ref link-reveal xref-bibr">Pons <em>et al.</em>, 2006</a>),
 we conduct a search for the transition points where the branching 
pattern changes from an among-species to a within-species branching 
pattern. The total number of possible delimitations on a rooted binary 
tree with <em>m</em> tips ranges between <em>m</em> (caterpillar tree) and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i13.gif" alt="forumla" path-from-xml="btt499i13.gif">⁠</span>, depending on the actual tree shape (<a reveal-id="btt499-B25" data-open="btt499-B25" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>).
 Because the search space is generally too large for an exhaustive 
search, we need to devise heuristic search strategies. Given a fixed 
species delimitation, we fit two exponential distributions to the 
respective two branch length classes (among- and within-species 
branching events). We calculate the log-likelihood as follows: <a id="btt499-M1" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="btt499-M1"><div class="fig-graphic"><a class="jumplink-placeholder" id="btt499m1.gif"><img class="contentFigures" src="m_btt499m1.gif" alt="formula" path-from-xml="btt499m1.gif"></a></div></div><div class="label">(1)</div></div> where <em>x</em><sub>1</sub> to <em>x<sub>k</sub></em> are the branch lengths generated by among-species PTPs, <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i14.gif" alt="forumla" path-from-xml="btt499i14.gif"></span> to <em>x<sub>n</sub></em> are the branch lengths of within-species PTPs, <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i15.gif" alt="forumla" path-from-xml="btt499i15.gif"></span> is the speciation rate per substitution and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i16.gif" alt="forumla" path-from-xml="btt499i16.gif"></span> is the rate of within-species branching events per substitution. The rates <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i17.gif" alt="forumla" path-from-xml="btt499i17.gif"></span> and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i18.gif" alt="forumla" path-from-xml="btt499i18.gif"></span> can be obtained via the inverse of the average branch lengths that belong to the respective processes. Based on <a href="#btt499-M1" class="sectionLink xref-sec">Equation (1)</a>, we search for the species delimitation that maximizes <em>L</em>.
 A standard likelihood-ratio test with one degree of freedom can be used
 to test if there are indeed two classes of branch lengths. Large <em>P</em>-value indicates that either all sequences are one species or that every sequence represents a single species.</div><p>We
 developed and assessed three heuristic search strategies for finding 
species delimitations with ‘good’ likelihood scores, which are described
 in the online supplement. For the experimental results presented here, 
we used the heuristic that performed best, based on our preliminary 
experiments.</p>                    <h3 scrollto-destination="84627119" id="84627119" class="section-title" data-legacy-id="SEC2.2">2.2 Species delimitation using phylogenetic placements</h3>
<p>In the following, we describe the open reference species delimitation
 pipeline that combines the EPA with the PTP (EPA-PTP). The EPA 
initially places a large number of query sequences (short reads) into 
the branches of a given reference phylogeny. Thereafter, we execute PTP 
separately and independently for the query sequences assigned to each 
branch. This allows to annotate the branches of the reference tree by 
the number of species induced by the query sequences that were placed 
into each branch. The input of our pipeline is a reference alignment 
where each sequence represents one species and a reference phylogeny for
 that alignment. The PTP method and the pipeline are implemented in 
Python and rely on the python Environment for Tree Exploration package (<a reveal-id="btt499-B32" data-open="btt499-B32" class="link link-ref link-reveal xref-bibr">Huerta-Cepas <em>et al.</em>, 2010</a>) for tree manipulation and visualization.</p><p>Our pipeline executes the following steps: </p><ol class="order"><li><p>Run UCHIME (<a reveal-id="btt499-B17" data-open="btt499-B17" class="link link-ref link-reveal xref-bibr">Edgar <em>et al.</em>, 2011</a>) against the reference alignment to remove chimeric query sequences. </p></li><li><p>Use
 EPA to place the query sequences onto the reference tree. Sequences 
that have a maximum placement likelihood weight of &lt;0.5 (i.e. an 
uncertain placement, see <a reveal-id="btt499-B4" data-open="btt499-B4" class="link link-ref link-reveal xref-bibr">Berger <em>et al.</em>, 2011</a> for details) are discarded.</p></li><li><p>For
 each branch in the reference tree, we extract the set of query 
sequences that have been placed into that branch and infer a tree on 
them using RAxML (<a reveal-id="btt499-B50" data-open="btt499-B50" class="link link-ref link-reveal xref-bibr">Stamatakis, 2006</a>).
 Because the PTP method requires a correctly rooted tree, we use the 
following two rooting strategies: if the branch leads to a tip, apart 
from the query sequences, we extend the alignment by including the 
reference tree tip sequence and that reference sequence that is furthest
 away from the current tip. The most distant sequence is used as 
outgroup. Keep in mind, that thereby the tree will be rooted at the 
longest branch (see the discussion below). To analyze query sequence 
placements at internal branches, we use the RAxML −g constraint tree 
option to obtain a rooted tree of the query sequences. The constraint 
tree consists of the bifurcating reference tree and a polytomy 
comprising the query sequences attached to the reference tree branch 
under consideration. The result of this constrained ML tree search is a 
resolved tree of query sequences that are attached to the reference tree
 branch. The attachment point is used as root.</p></li><li><p>Because we
 assume that the reference phylogeny is a species tree that reflects our
 knowledge about the speciation process and rate, we initially estimate <span class="inline-formula no-formula-id"><img class="content-image" src="m_btt499i19.gif" data-src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/m_btt499i19.gif?Expires=1585056692&amp;Signature=ji68PlPB0EbwxMh4br9Eg6YFmW~atyaSmBeAgm96zxSlxRZu2cbkN7h9bP6Cx69C8viKwRvIO2TFfQIKKQkacdiy5UDSPAva5j10tPYUspmfXNhuIFy1E~esLL~g1GjQ9xvNEM0OKD~3zeeYZuN0woC7x668RzWdvA390fIh6RoLVKM3~LWrJYZlZIOL13l9J8w6fcbjSgDEbhuj7dMl9up0CKcX51nfgKzTPOnqR6GvgCGW80MMqfGVv4HFRyhvBgtq-3GUq5QonttQcsfc-POLvTpFZw4zH1TOok6wQ-nPvYnGnGs10fawPXfcu9D6OLyT1~CDKfOl2rpYD3SY7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="" path-from-xml="btt499i19.gif"></span>
 only once on the reference phylogeny. Thereafter, we apply PTP to each 
query sequence (one for each branch of the reference phylogeny) tree to 
delimit species. Note that in this scenario we will only need to 
estimate <span class="inline-formula no-formula-id"><img class="content-image" src="m_btt499i20.gif" data-src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/m_btt499i20.gif?Expires=1585056692&amp;Signature=C~qZmpW14OAsfag8BQlf8v5wZ0wIjBL3620NZpe82YQxtUMdUeGrNux71uvgnjvxqu1fu9YT6tNY2~QI~KRDFI5KealdowP2~nsuNIn3jHYT26B0P2ggSqbFoyvFgh9xG2qpICy-7r431~F-0COPFRsNLkkVuL~Eg96Va4I-ANLWTV-kv1qaojYxx9AXbavAhO-9P8GB9C6fWg1ltKd0WPiwCqJ63q1ns9oh~Ennrts4fhqos1YdV8kgf63pctJn-buVGxgN5ot-5qSHdM~MSZ2BZ1vY4fhrLDdbKY6pgHjeolzn2EBT3zQoeVhEZC~Me1tyfufuhqVOUrLuSOYUhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="" path-from-xml="btt499i20.gif">⁠</span>, as <span class="inline-formula no-formula-id"><img class="content-image" src="m_btt499i21.gif" data-src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/m_btt499i21.gif?Expires=1585056692&amp;Signature=k938-iXpg3K8AAikKbYQ~tAQyihLu4ZiD4INQw9TLE8zfdhkF~A-pZ9onhHuJP2YLx3E1R6gFpT6LLgFnRfI6qYPuxC9xMrBiH45SBz3GnSlFY64kzrA-HnbvIqhCPOEICv5Njc6F4wRGcVG~Lqgu26mQ9LlMGCkuxLuusbDW4-sCPyWMRbqACQYOAyiC43Dzgq-dk3wCyVgy3XKMRVrOotXsW56eqzDvQgVH5PyCWmY8RpkUqhzGQuU9NuO-MUZWHAz~BsR3nY5U9fYoKLJ5JIBgfK9e4b8eNr79dNpzc24sBHo3XwQp72XSlyRI5V7xlWtNEySs0t4R96Nl4EVqg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="" path-from-xml="btt499i21.gif"></span> remains fixed.</p></li><li><p>When
 PTP is applied to a placement of query sequences on a terminal branch, 
those queries that are delimited as one population with the reference 
sequence at the tip will be assigned taxonomically to the species 
represented by this reference sequence. Otherwise, they are identified 
as new species in the reference tree.</p></li></ol><p>As mentioned 
previously, we also combined EPA with CROP (EPA-CROP). The method works 
as EPA-PTP, with the only difference that CROP is used instead of PTP to
 calculate the number of MOTUs for each placement. </p>                    <h2 scrollto-destination="84627124" id="84627124" class="section-title" data-legacy-id="SEC3">3 EXPERIMENTAL SETUP</h2>
<p>We initially tested stand-alone PTP for general species delimitation 
and compared it with the single-threshold GMYC model. The 
single-threshold GMYC model infers a single cutoff time <em>T</em> where all nodes above <em>T</em> represent species. Although the more advanced multiple-threshold GMYC allows for several threshold times <em>T<sub>i</sub></em>, the single-threshold GMYC is usually more accurate than the multi-threshold version (see <a reveal-id="btt499-B26" data-open="btt499-B26" class="link link-ref link-reveal xref-bibr">Fujita <em>et al.</em>, 2012</a> for details). For simulated data, we used RAxML (<a reveal-id="btt499-B50" data-open="btt499-B50" class="link link-ref link-reveal xref-bibr">Stamatakis, 2006</a>) to infer phylogenetic trees, and then used them as input to PTP. Subsequently, these phylogenies were made <em>ultrametric</em> by r8s (<a reveal-id="btt499-B43" data-open="btt499-B43" class="link link-ref link-reveal xref-bibr">Sanderson, 2003</a>)
 to test GMYC. For UCLUST and CROP, only molecular sequences are needed 
as input. In both programs, we set the sequence dissimilarity threshold 
to 97%, a widely accepted threshold for bacterial sequences (<a reveal-id="btt499-B48" data-open="btt499-B48" class="link link-ref link-reveal xref-bibr">Stackebrandt and Goebel, 1994</a>).
 For real datasets, we used the phylogenetic tree and ultrametric tree 
from the original publications whenever possible, otherwise we used the 
same settings as described previously.</p><p>We then tested our open 
reference species delimitation approaches (EPA-PTP and EPA-CROP). We 
also assessed the impact of incomplete taxon sampling on the accuracy of
 these approaches, by removing up to 50% of the reference sequences.</p>                    <h3 scrollto-destination="84627127" id="84627127" class="section-title" data-legacy-id="SEC3.1">3.1 Empirical datasets</h3>
                    <h4 scrollto-destination="84627128" id="84627128" class="section-title" data-legacy-id="SEC3.1.1">3.1.1 Arthropod datasets</h4>
<p>The <em>Rivancidella</em> dataset comprises three genes (cyt b, COI, 16S) and was originally used in <a reveal-id="btt499-B40" data-open="btt499-B40" class="link link-ref link-reveal xref-bibr">Pons <em>et al.</em> (2006)</a>.
 The total number of sequences is 472, which represents 24 morphological
 species and 4 outgroup taxa. The estimated number of putative species 
for the genus as inferred by GMYC was 48 (with confidence limits ranging
 between 46 and 52 species). Alternative methods (see <a reveal-id="btt499-B40" data-open="btt499-B40" class="link link-ref link-reveal xref-bibr">Pons <em>et al.</em>, 2006</a> for details) used in this study yielded 46 and 47 putative species, respectively.</p><p>We also used COI marker datasets (<a reveal-id="btt499-B39" data-open="btt499-B39" class="link link-ref link-reveal xref-bibr">Papadopoulou <em>et al.</em>, 2010</a>) of the genera <em>Dendarus</em>, <em>Pimellia</em> and <em>Tentyria</em>.
 The datasets comprise 51, 56 and 59 sequences, respectively. The number
 of species that were attributed to each taxon using morphological 
criteria was seven, one and one.</p>                    <h4 scrollto-destination="84627131" id="84627131" class="section-title" data-legacy-id="SEC3.1.2">3.1.2 Gallotia dataset</h4>
<p>The lizard genus <em>Gallotia</em> comprises seven species (based on genetic <em>and</em>
 morphological markers) that are endemic to the Canary islands. The 
taxonomic species tree and the molecular phylogeny for this dataset are 
fully congruent. The data (<a reveal-id="btt499-B11" data-open="btt499-B11" class="link link-ref link-reveal xref-bibr">Cox <em>et al.</em>, 2010</a>) comprises four mitochondrial genes (cyt b, COI, 12S, 16S) and a total of 90 sequences (76 representing <em>Gallotia</em> and 14 outgroup sequences). </p>                    <h4 scrollto-destination="84627133" id="84627133" class="section-title" data-legacy-id="SEC3.1.3">3.1.3 Arthropod metabarcoding dataset</h4>
<p>This dataset contains 673 full-length COI arthropod sequences with a 
length of 658 bp. The sequence was obtained via polymerase chain 
reaction amplification and Sanger sequencing. Subsequently, these 673 
sequences were resequenced with a 454 sequencer to generate a total of 
133 057 short reads (<a reveal-id="btt499-B57" data-open="btt499-B57" class="link link-ref link-reveal xref-bibr">Yu <em>et al.</em>, 2012</a>). Using the Sanger data as reference, Yu <em>et al.</em>
 developed metabarcoding protocols that use the 454 reads to unravel the
 diversity in the reference data. The authors use a multistep 
OTU-picking procedure with different similarity thresholds for 
clustering the 454 reads and the full-length reference sequences. The 
method clustered the 673 sequences into 547 MOTUs. The OTU-picking 
results for the 454 data are summarized in <a reveal-id="btt499-T4" data-open="btt499-T4" class="link link-reveal link-table xref-fig">Table 4</a>.
 Our PTP model finds 545 putative species in the 673 full-length 
sequences when directly applied to the phylogenetic reference tree. To 
ensure comparability of results, we used the 547 MOTUs identified in the
 original study to build a reference tree and reference alignment for 
testing the EPA-PTP and EPA-CROP pipelines. Initially, we aligned 454 
sequences with a length exceeding 100 bp to these 547 reference 
sequences with HMMER (<a reveal-id="btt499-B15" data-open="btt499-B15" class="link link-ref link-reveal xref-bibr">Eddy, 2009</a>). Yu <em>et al.</em>
 initially blasted the 454-MOTU (obtained via three alternative 
clustering methods) to the Sanger-MOTUs using a threshold of 1e-10 and 
97% minimum similarity. The Sanger-MOTUs that did not match any of the 
454-MOTUs are called ‘dropouts’ by the authors. Inversely, 454-MOTUs 
that did not match Sanger-MOTUs are called ‘no-matches’.</p><p>Analogously,
 in our pipelines, when the delimited species from 454 sequence 
placements contain one of the full-length reference sequences (see step 4
 in 2.2), we consider this as a ‘match’. Further, we denote a 
full-length reference sequence that is not included in any short read 
placement delimitation as ‘dropout’. Finally, we call a short read 
placement that is delimited as a new species (i.e. does not contain a 
reference sequence) as ‘no-match’.</p>                    <h3 scrollto-destination="84627136" id="84627136" class="section-title" data-legacy-id="SEC3.2">3.2 Simulations</h3>
<p>We generated simulated datasets using a Yule coalescent model. We used ms (<a reveal-id="btt499-B31" data-open="btt499-B31" class="link link-ref link-reveal xref-bibr">Hudson, 2002</a>) and BioPerl (<a reveal-id="btt499-B49" data-open="btt499-B49" class="link link-ref link-reveal xref-bibr">Stajich <em>et al.</em>, 2002</a>) in combination with INDELible (<a reveal-id="btt499-B21" data-open="btt499-B21" class="link link-ref link-reveal xref-bibr">Fletcher and Yang, 2009</a>)
 to simulate sequences. Using a modified version of the BioPerl module 
Bio::Phylo that allows to vary the birth rate value in the simulations, 
we initially generated a set of random <em>species</em> trees <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i22.gif" alt="forumla" path-from-xml="btt499i22.gif">⁠</span>. The leaves of each tree <em>T<sub>i</sub></em> (<span class="inline-formula no-formula-id">⁠<img class="contentFigures" src="m_btt499i23.gif" alt="forumla" path-from-xml="btt499i23.gif">⁠</span>)
 represent extant species. All 600 simulated datasets we generated 
contain 30 species. In the next step, we used ms to generate a 
structured coalescent gene tree. The node ages of the phylogenetic tree <em>T<sub>i</sub></em>
 are interpreted as divergence times between populations. In other 
words, we treat species as diverged populations that were completely 
isolated from each other after they diverged from their common ancestor.
 Thus, using ms we simulated a multispecies coalescent gene tree with 30
 species and 100 individuals per species. For each species, we randomly 
selected 10 individuals to generate evenly sampled (in terms of the 
number of individuals per species) datasets. We also generated unevenly 
sampled datasets containing 2 species with 100 individuals, 4 with 50 
individuals, 8 with 10 individuals and 16 with 2 individuals. Finally, 
we used INDELible to simulate DNA alignments of 250, 500 and 1000 bp on 
the previously mentioned multispecies coalescent trees.</p><p>We generated datasets with a scaled birth rate (<span class="inline-formula no-formula-id">⁠<img class="contentFigures" src="m_btt499i24.gif" alt="forumla" path-from-xml="btt499i24.gif">⁠</span>); small values generate large evolutionary distances between species. For details on the simulations and on the scaled rate <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i25.gif" alt="forumla" path-from-xml="btt499i25.gif">⁠</span>, please refer to the online supplement.</p><p>We used the normalized mutual information (NMI) criterion (<a reveal-id="btt499-B52" data-open="btt499-B52" class="link link-ref link-reveal xref-bibr">Vinh <em>et al.</em>, 2010</a>)
 to asses how the delimitation accuracy of the different algorithms 
agrees with the ground truth. The mutual information (MI) of two 
distinct partitions of the same dataset quantifies how much information 
is shared by these; NMI scales MI to values between 0.0 and 1.0. In our 
case, NMI = 1 means that the delimitation is identical to the ground 
truth, whereas NMI = 0 means that the delimited species are randomly 
partitioned compared with the ground truth.</p><p>Finally, we also 
tested the EPA-PTP and EPA-CROP pipelines on simulated data. In each 
simulated alignment, we randomly selected one individual sequence per 
species as reference sequence and treated the remaining sequences (of 
that species) as query sequences. To assess the impact of incomplete 
reference trees on species delimitations, we randomly removed up to 50% 
of the reference sequences. We deployed the same metrics as mentioned 
previously to quantify delimitation accuracy.</p>                    <h2 scrollto-destination="84627141" id="84627141" class="section-title">4 RESULTS</h2>
                    <h3 scrollto-destination="84627142" id="84627142" class="section-title" data-legacy-id="SEC4.1">4.1 General species delimitation</h3>
<p>The number of putative species delimited for <em>Dendarus</em>, <em>Pimelia</em>, <em>Tentyria</em> and <em>Gallotia</em> are comparable for all four methods (<a reveal-id="btt499-T1" data-open="btt499-T1" class="link link-reveal link-table xref-fig">Table 1</a>). For the <em>Gallotia</em> dataset, GMYC and PTP yield identical results. Three of the <em>Gallotia</em>
 species were split into two separate groups according to geographical 
isolation of the corresponding populations on different islands (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). </p>                        <a id="84627144" scrollto-destination="84627144"></a>
<div content-id="btt499-T1" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="btt499-T1" class="table-wrap-title" data-id="btt499-T1"><div class="label">Table 1.</div><div class="caption"><p>Number of species delimitated on real data</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Taxon&nbsp;</th><th>Morphological&nbsp;</th><th>GMYC&nbsp;</th><th>PTP&nbsp;</th><th>CROP&nbsp;</th><th>UCLUST&nbsp;</th></tr></thead><tbody><tr><td><em>Rivacindela</em>&nbsp;</td><td>24&nbsp;</td><td>48&nbsp;</td><td>27/44<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>6&nbsp;</td><td>82&nbsp;</td></tr><tr><td><em>Dendarus</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>9/11<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>11&nbsp;</td></tr><tr><td><em>Pimelia</em>&nbsp;</td><td>1&nbsp;</td><td>10&nbsp;</td><td>9/15<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td></tr><tr><td><em>Tentyria</em>&nbsp;</td><td>1&nbsp;</td><td>2&nbsp;</td><td>2/2<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>1&nbsp;</td><td>3&nbsp;</td></tr><tr><td><em>Gallotia</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>10/10<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>9&nbsp;</td><td>15&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Taxon&nbsp;</th><th>Morphological&nbsp;</th><th>GMYC&nbsp;</th><th>PTP&nbsp;</th><th>CROP&nbsp;</th><th>UCLUST&nbsp;</th></tr></thead><tbody><tr><td><em>Rivacindela</em>&nbsp;</td><td>24&nbsp;</td><td>48&nbsp;</td><td>27/44<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>6&nbsp;</td><td>82&nbsp;</td></tr><tr><td><em>Dendarus</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>9/11<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>11&nbsp;</td></tr><tr><td><em>Pimelia</em>&nbsp;</td><td>1&nbsp;</td><td>10&nbsp;</td><td>9/15<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td></tr><tr><td><em>Tentyria</em>&nbsp;</td><td>1&nbsp;</td><td>2&nbsp;</td><td>2/2<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>1&nbsp;</td><td>3&nbsp;</td></tr><tr><td><em>Gallotia</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>10/10<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>9&nbsp;</td><td>15&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="btt499-TF1"><div class="fn"><p><sup>a</sup>Using the ultrametric tree as an input for PTP.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/[QueryString]" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="btt499-T1" class="table-wrap-title" data-id="btt499-T1"><div class="label">Table 1.</div><div class="caption"><p>Number of species delimitated on real data</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Taxon&nbsp;</th><th>Morphological&nbsp;</th><th>GMYC&nbsp;</th><th>PTP&nbsp;</th><th>CROP&nbsp;</th><th>UCLUST&nbsp;</th></tr></thead><tbody><tr><td><em>Rivacindela</em>&nbsp;</td><td>24&nbsp;</td><td>48&nbsp;</td><td>27/44<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>6&nbsp;</td><td>82&nbsp;</td></tr><tr><td><em>Dendarus</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>9/11<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>11&nbsp;</td></tr><tr><td><em>Pimelia</em>&nbsp;</td><td>1&nbsp;</td><td>10&nbsp;</td><td>9/15<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td></tr><tr><td><em>Tentyria</em>&nbsp;</td><td>1&nbsp;</td><td>2&nbsp;</td><td>2/2<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>1&nbsp;</td><td>3&nbsp;</td></tr><tr><td><em>Gallotia</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>10/10<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>9&nbsp;</td><td>15&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Taxon&nbsp;</th><th>Morphological&nbsp;</th><th>GMYC&nbsp;</th><th>PTP&nbsp;</th><th>CROP&nbsp;</th><th>UCLUST&nbsp;</th></tr></thead><tbody><tr><td><em>Rivacindela</em>&nbsp;</td><td>24&nbsp;</td><td>48&nbsp;</td><td>27/44<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>6&nbsp;</td><td>82&nbsp;</td></tr><tr><td><em>Dendarus</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>9/11<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>11&nbsp;</td></tr><tr><td><em>Pimelia</em>&nbsp;</td><td>1&nbsp;</td><td>10&nbsp;</td><td>9/15<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td></tr><tr><td><em>Tentyria</em>&nbsp;</td><td>1&nbsp;</td><td>2&nbsp;</td><td>2/2<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>1&nbsp;</td><td>3&nbsp;</td></tr><tr><td><em>Gallotia</em>&nbsp;</td><td>7&nbsp;</td><td>10&nbsp;</td><td>10/10<a reveal-id="btt499-TF1" data-open="btt499-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</td><td>9&nbsp;</td><td>15&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="btt499-TF1"><div class="fn"><p><sup>a</sup>Using the ultrametric tree as an input for PTP.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/[QueryString]" target="_blank">Open in new tab</a></div></div></div><p>On the <em>Rivacindela</em>
 dataset PTP yields a more conservative delimitation than GMYC. PTP 
identifies 27 putative species (GMYC: 48), which is closer to the number
 of morphological species (24) and the number of independent networks 
(25) obtained via statistical parsimony (<a reveal-id="btt499-B40" data-open="btt499-B40" class="link link-ref link-reveal xref-bibr">Pons <em>et al.</em>, 2006</a>).
 This pronounced difference may be associated with the construction of 
the ultrametric tree. According to the r8s manual, the presence of many 
short (close to zero) branches in the tree can yield inaccurate results.
 When applying PTP to the ultrametric tree, the resulting estimate is 
substantially closer to the GMYC estimate (see <a reveal-id="btt499-T1" data-open="btt499-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Thus, we believe that the overestimation of the <em>Rivacidela</em>
 species by GMYC is most probably because of an erroneous ultrametric 
tree reconstruction. CROP and UCLUST yield dissimilar results; CROP only
 detects 6 clusters, whereas UCLUST detects 82 clusters. </p><p>The results on evenly sampled simulated data are summarized in <a reveal-id="btt499-T2" data-open="btt499-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.
 On average, PTP shows the best performance and outperforms GMYC in all 
the test scenarios. OTU-picking methods work well on datasets with small
 <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i26.gif" alt="forumla" path-from-xml="btt499i26.gif"></span> values that is when the evolutionary distances between species are large. For <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i27.gif" alt="forumla" path-from-xml="btt499i27.gif">⁠</span>, UCLUST generally outperforms PTP and yields the best overall results. However, with increasing <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i28.gif" alt="forumla" path-from-xml="btt499i28.gif"></span>
 the accuracy of OTU-picking methods decreases steeply. As expected, for
 shorter sequence lengths (250 and 500 bp), accuracy deteriorates for 
all methods and in a more pronounced way for PTP and GMYC. However, even
 with sequence lengths of 250 bp, PTP still yields best results on 
datasets with <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i29.gif" alt="forumla" path-from-xml="btt499i29.gif">⁠</span>. </p>                        <a id="84627147" scrollto-destination="84627147"></a>
<div content-id="btt499-T2" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="btt499-T2" class="table-wrap-title" data-id="btt499-T2"><div class="label">Table 2.</div><div class="caption"><p>Species delimitation accuracy (measured in NMI) on simulated evenly sampled data</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.969&nbsp;</td><td>0.959&nbsp;</td><td>0.938&nbsp;</td><td>0.892&nbsp;</td><td>0.782&nbsp;</td><td>0.575&nbsp;</td><td>0.852 (0.023)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.964&nbsp;</td><td>0.930&nbsp;</td><td>0.848&nbsp;</td><td>0.646&nbsp;</td><td>0.232&nbsp;</td><td>0.038&nbsp;</td><td>0.609 (0.151)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.924&nbsp;</td><td>0.914&nbsp;</td><td>0.907&nbsp;</td><td>0.886&nbsp;</td><td>0.834&nbsp;</td><td>0.697&nbsp;</td><td>0.860 (0.007)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.944&nbsp;</td><td>0.935&nbsp;</td><td>0.922&nbsp;</td><td>0.905&nbsp;</td><td>0.882&nbsp;</td><td>0.857&nbsp;</td><td>0.907 (0.001)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.967&nbsp;</td><td>0.954&nbsp;</td><td>0.930&nbsp;</td><td>0.871&nbsp;</td><td>0.735&nbsp;</td><td>0.522&nbsp;</td><td>0.829 (0.029)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.961&nbsp;</td><td>0.917&nbsp;</td><td>0.800&nbsp;</td><td>0.545&nbsp;</td><td>0.152&nbsp;</td><td>0.024&nbsp;</td><td>0.566 (0.159)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.892&nbsp;</td><td>0.620&nbsp;</td><td>0.484&nbsp;</td><td>0.464&nbsp;</td><td>0.550&nbsp;</td><td>0.503&nbsp;</td><td>0.585 (0.025)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.946&nbsp;</td><td>0.927&nbsp;</td><td>0.907&nbsp;</td><td>0.881&nbsp;</td><td>0.833&nbsp;</td><td>0.780&nbsp;</td><td>0.879 (0.003)&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.969&nbsp;</td><td>0.959&nbsp;</td><td>0.938&nbsp;</td><td>0.892&nbsp;</td><td>0.782&nbsp;</td><td>0.575&nbsp;</td><td>0.852 (0.023)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.964&nbsp;</td><td>0.930&nbsp;</td><td>0.848&nbsp;</td><td>0.646&nbsp;</td><td>0.232&nbsp;</td><td>0.038&nbsp;</td><td>0.609 (0.151)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.924&nbsp;</td><td>0.914&nbsp;</td><td>0.907&nbsp;</td><td>0.886&nbsp;</td><td>0.834&nbsp;</td><td>0.697&nbsp;</td><td>0.860 (0.007)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.944&nbsp;</td><td>0.935&nbsp;</td><td>0.922&nbsp;</td><td>0.905&nbsp;</td><td>0.882&nbsp;</td><td>0.857&nbsp;</td><td>0.907 (0.001)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.967&nbsp;</td><td>0.954&nbsp;</td><td>0.930&nbsp;</td><td>0.871&nbsp;</td><td>0.735&nbsp;</td><td>0.522&nbsp;</td><td>0.829 (0.029)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.961&nbsp;</td><td>0.917&nbsp;</td><td>0.800&nbsp;</td><td>0.545&nbsp;</td><td>0.152&nbsp;</td><td>0.024&nbsp;</td><td>0.566 (0.159)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.892&nbsp;</td><td>0.620&nbsp;</td><td>0.484&nbsp;</td><td>0.464&nbsp;</td><td>0.550&nbsp;</td><td>0.503&nbsp;</td><td>0.585 (0.025)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.946&nbsp;</td><td>0.927&nbsp;</td><td>0.907&nbsp;</td><td>0.881&nbsp;</td><td>0.833&nbsp;</td><td>0.780&nbsp;</td><td>0.879 (0.003)&nbsp;</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/[QueryString]" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="btt499-T2" class="table-wrap-title" data-id="btt499-T2"><div class="label">Table 2.</div><div class="caption"><p>Species delimitation accuracy (measured in NMI) on simulated evenly sampled data</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.969&nbsp;</td><td>0.959&nbsp;</td><td>0.938&nbsp;</td><td>0.892&nbsp;</td><td>0.782&nbsp;</td><td>0.575&nbsp;</td><td>0.852 (0.023)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.964&nbsp;</td><td>0.930&nbsp;</td><td>0.848&nbsp;</td><td>0.646&nbsp;</td><td>0.232&nbsp;</td><td>0.038&nbsp;</td><td>0.609 (0.151)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.924&nbsp;</td><td>0.914&nbsp;</td><td>0.907&nbsp;</td><td>0.886&nbsp;</td><td>0.834&nbsp;</td><td>0.697&nbsp;</td><td>0.860 (0.007)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.944&nbsp;</td><td>0.935&nbsp;</td><td>0.922&nbsp;</td><td>0.905&nbsp;</td><td>0.882&nbsp;</td><td>0.857&nbsp;</td><td>0.907 (0.001)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.967&nbsp;</td><td>0.954&nbsp;</td><td>0.930&nbsp;</td><td>0.871&nbsp;</td><td>0.735&nbsp;</td><td>0.522&nbsp;</td><td>0.829 (0.029)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.961&nbsp;</td><td>0.917&nbsp;</td><td>0.800&nbsp;</td><td>0.545&nbsp;</td><td>0.152&nbsp;</td><td>0.024&nbsp;</td><td>0.566 (0.159)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.892&nbsp;</td><td>0.620&nbsp;</td><td>0.484&nbsp;</td><td>0.464&nbsp;</td><td>0.550&nbsp;</td><td>0.503&nbsp;</td><td>0.585 (0.025)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.946&nbsp;</td><td>0.927&nbsp;</td><td>0.907&nbsp;</td><td>0.881&nbsp;</td><td>0.833&nbsp;</td><td>0.780&nbsp;</td><td>0.879 (0.003)&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.969&nbsp;</td><td>0.959&nbsp;</td><td>0.938&nbsp;</td><td>0.892&nbsp;</td><td>0.782&nbsp;</td><td>0.575&nbsp;</td><td>0.852 (0.023)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.964&nbsp;</td><td>0.930&nbsp;</td><td>0.848&nbsp;</td><td>0.646&nbsp;</td><td>0.232&nbsp;</td><td>0.038&nbsp;</td><td>0.609 (0.151)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.924&nbsp;</td><td>0.914&nbsp;</td><td>0.907&nbsp;</td><td>0.886&nbsp;</td><td>0.834&nbsp;</td><td>0.697&nbsp;</td><td>0.860 (0.007)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.944&nbsp;</td><td>0.935&nbsp;</td><td>0.922&nbsp;</td><td>0.905&nbsp;</td><td>0.882&nbsp;</td><td>0.857&nbsp;</td><td>0.907 (0.001)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    UCLUST&nbsp;</td><td>0.967&nbsp;</td><td>0.954&nbsp;</td><td>0.930&nbsp;</td><td>0.871&nbsp;</td><td>0.735&nbsp;</td><td>0.522&nbsp;</td><td>0.829 (0.029)&nbsp;</td></tr><tr><td>    CROP&nbsp;</td><td>0.961&nbsp;</td><td>0.917&nbsp;</td><td>0.800&nbsp;</td><td>0.545&nbsp;</td><td>0.152&nbsp;</td><td>0.024&nbsp;</td><td>0.566 (0.159)&nbsp;</td></tr><tr><td>    GMYC&nbsp;</td><td>0.892&nbsp;</td><td>0.620&nbsp;</td><td>0.484&nbsp;</td><td>0.464&nbsp;</td><td>0.550&nbsp;</td><td>0.503&nbsp;</td><td>0.585 (0.025)&nbsp;</td></tr><tr><td>    PTP&nbsp;</td><td>0.946&nbsp;</td><td>0.927&nbsp;</td><td>0.907&nbsp;</td><td>0.881&nbsp;</td><td>0.833&nbsp;</td><td>0.780&nbsp;</td><td>0.879 (0.003)&nbsp;</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/[QueryString]" target="_blank">Open in new tab</a></div></div></div><p>On the unevenly sampled simulated datasets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>),
 the delimitation accuracy decreases for UCLUST and PTP. CROP and GMYC 
yield higher NMI scores than on evenly sampled dataset. On average, PTP 
yields the best results over all (evenly and unevenly sampled) simulated
 datasets.</p>                    <h3 scrollto-destination="84627149" id="84627149" class="section-title" data-legacy-id="SEC4.2">4.2 Species delimitation with phylogenetic placements</h3>
<p>By combining EPA with PTP (or CROP) and applying it to simulated data
 as described in Section 3.2, we can substantially improve the 
delimitation accuracy on simulated data (<a reveal-id="btt499-T3" data-open="btt499-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). </p>                        <a id="84627151" scrollto-destination="84627151"></a>
<div content-id="btt499-T3" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="btt499-T3" class="table-wrap-title" data-id="btt499-T3"><div class="label">Table 3.</div><div class="caption"><p>Species delimitation accuracy (measured in NMI) on simulated evenly sampled data using the EPA-PTP pipeline</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.989&nbsp;</td><td>0.978&nbsp;</td><td>0.962&nbsp;</td><td>0.933&nbsp;</td><td>0.884&nbsp;</td><td>0.836&nbsp;</td><td>0.930 (0.003)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.984&nbsp;</td><td>0.972&nbsp;</td><td>0.955&nbsp;</td><td>0.925&nbsp;</td><td>0.876&nbsp;</td><td>0.830&nbsp;</td><td>0.923 (0.003)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.976&nbsp;</td><td>0.966&nbsp;</td><td>0.949&nbsp;</td><td>0.921&nbsp;</td><td>0.872&nbsp;</td><td>0.823&nbsp;</td><td>0.917 (0.003)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.971&nbsp;</td><td>0.959&nbsp;</td><td>0.943&nbsp;</td><td>0.912&nbsp;</td><td>0.868&nbsp;</td><td>0.816&nbsp;</td><td>0.911 (0.003)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.966&nbsp;</td><td>0.956&nbsp;</td><td>0.939&nbsp;</td><td>0.908&nbsp;</td><td>0.860&nbsp;</td><td>0.805&nbsp;</td><td>0.905 (0.003)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.962&nbsp;</td><td>0.950&nbsp;</td><td>0.934&nbsp;</td><td>0.904&nbsp;</td><td>0.853&nbsp;</td><td>0.787&nbsp;</td><td>0.898 (0.004)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.978&nbsp;</td><td>0.968&nbsp;</td><td>0.949&nbsp;</td><td>0.918&nbsp;</td><td>0.863&nbsp;</td><td>0.811&nbsp;</td><td>0.914 (0.004)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.967&nbsp;</td><td>0.955&nbsp;</td><td>0.935&nbsp;</td><td>0.907&nbsp;</td><td>0.854&nbsp;</td><td>0.800&nbsp;</td><td>0.903 (0.004)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.956&nbsp;</td><td>0.944&nbsp;</td><td>0.926&nbsp;</td><td>0.895&nbsp;</td><td>0.846&nbsp;</td><td>0.786&nbsp;</td><td>0.892 (0.004)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.942&nbsp;</td><td>0.926&nbsp;</td><td>0.912&nbsp;</td><td>0.880&nbsp;</td><td>0.830&nbsp;</td><td>0.773&nbsp;</td><td>0.877 (0.004)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.927&nbsp;</td><td>0.911&nbsp;</td><td>0.893&nbsp;</td><td>0.861&nbsp;</td><td>0.813&nbsp;</td><td>0.755&nbsp;</td><td>0.860 (0.004)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.909&nbsp;</td><td>0.891&nbsp;</td><td>0.871&nbsp;</td><td>0.838&nbsp;</td><td>0.784&nbsp;</td><td>0.732&nbsp;</td><td>0.837 (0.004)&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.989&nbsp;</td><td>0.978&nbsp;</td><td>0.962&nbsp;</td><td>0.933&nbsp;</td><td>0.884&nbsp;</td><td>0.836&nbsp;</td><td>0.930 (0.003)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.984&nbsp;</td><td>0.972&nbsp;</td><td>0.955&nbsp;</td><td>0.925&nbsp;</td><td>0.876&nbsp;</td><td>0.830&nbsp;</td><td>0.923 (0.003)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.976&nbsp;</td><td>0.966&nbsp;</td><td>0.949&nbsp;</td><td>0.921&nbsp;</td><td>0.872&nbsp;</td><td>0.823&nbsp;</td><td>0.917 (0.003)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.971&nbsp;</td><td>0.959&nbsp;</td><td>0.943&nbsp;</td><td>0.912&nbsp;</td><td>0.868&nbsp;</td><td>0.816&nbsp;</td><td>0.911 (0.003)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.966&nbsp;</td><td>0.956&nbsp;</td><td>0.939&nbsp;</td><td>0.908&nbsp;</td><td>0.860&nbsp;</td><td>0.805&nbsp;</td><td>0.905 (0.003)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.962&nbsp;</td><td>0.950&nbsp;</td><td>0.934&nbsp;</td><td>0.904&nbsp;</td><td>0.853&nbsp;</td><td>0.787&nbsp;</td><td>0.898 (0.004)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.978&nbsp;</td><td>0.968&nbsp;</td><td>0.949&nbsp;</td><td>0.918&nbsp;</td><td>0.863&nbsp;</td><td>0.811&nbsp;</td><td>0.914 (0.004)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.967&nbsp;</td><td>0.955&nbsp;</td><td>0.935&nbsp;</td><td>0.907&nbsp;</td><td>0.854&nbsp;</td><td>0.800&nbsp;</td><td>0.903 (0.004)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.956&nbsp;</td><td>0.944&nbsp;</td><td>0.926&nbsp;</td><td>0.895&nbsp;</td><td>0.846&nbsp;</td><td>0.786&nbsp;</td><td>0.892 (0.004)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.942&nbsp;</td><td>0.926&nbsp;</td><td>0.912&nbsp;</td><td>0.880&nbsp;</td><td>0.830&nbsp;</td><td>0.773&nbsp;</td><td>0.877 (0.004)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.927&nbsp;</td><td>0.911&nbsp;</td><td>0.893&nbsp;</td><td>0.861&nbsp;</td><td>0.813&nbsp;</td><td>0.755&nbsp;</td><td>0.860 (0.004)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.909&nbsp;</td><td>0.891&nbsp;</td><td>0.871&nbsp;</td><td>0.838&nbsp;</td><td>0.784&nbsp;</td><td>0.732&nbsp;</td><td>0.837 (0.004)&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="btt499-TF3"><div class="fn"><p><em>Note</em>: ref. indicates reference sequences</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/[QueryString]" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="btt499-T3" class="table-wrap-title" data-id="btt499-T3"><div class="label">Table 3.</div><div class="caption"><p>Species delimitation accuracy (measured in NMI) on simulated evenly sampled data using the EPA-PTP pipeline</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.989&nbsp;</td><td>0.978&nbsp;</td><td>0.962&nbsp;</td><td>0.933&nbsp;</td><td>0.884&nbsp;</td><td>0.836&nbsp;</td><td>0.930 (0.003)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.984&nbsp;</td><td>0.972&nbsp;</td><td>0.955&nbsp;</td><td>0.925&nbsp;</td><td>0.876&nbsp;</td><td>0.830&nbsp;</td><td>0.923 (0.003)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.976&nbsp;</td><td>0.966&nbsp;</td><td>0.949&nbsp;</td><td>0.921&nbsp;</td><td>0.872&nbsp;</td><td>0.823&nbsp;</td><td>0.917 (0.003)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.971&nbsp;</td><td>0.959&nbsp;</td><td>0.943&nbsp;</td><td>0.912&nbsp;</td><td>0.868&nbsp;</td><td>0.816&nbsp;</td><td>0.911 (0.003)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.966&nbsp;</td><td>0.956&nbsp;</td><td>0.939&nbsp;</td><td>0.908&nbsp;</td><td>0.860&nbsp;</td><td>0.805&nbsp;</td><td>0.905 (0.003)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.962&nbsp;</td><td>0.950&nbsp;</td><td>0.934&nbsp;</td><td>0.904&nbsp;</td><td>0.853&nbsp;</td><td>0.787&nbsp;</td><td>0.898 (0.004)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.978&nbsp;</td><td>0.968&nbsp;</td><td>0.949&nbsp;</td><td>0.918&nbsp;</td><td>0.863&nbsp;</td><td>0.811&nbsp;</td><td>0.914 (0.004)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.967&nbsp;</td><td>0.955&nbsp;</td><td>0.935&nbsp;</td><td>0.907&nbsp;</td><td>0.854&nbsp;</td><td>0.800&nbsp;</td><td>0.903 (0.004)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.956&nbsp;</td><td>0.944&nbsp;</td><td>0.926&nbsp;</td><td>0.895&nbsp;</td><td>0.846&nbsp;</td><td>0.786&nbsp;</td><td>0.892 (0.004)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.942&nbsp;</td><td>0.926&nbsp;</td><td>0.912&nbsp;</td><td>0.880&nbsp;</td><td>0.830&nbsp;</td><td>0.773&nbsp;</td><td>0.877 (0.004)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.927&nbsp;</td><td>0.911&nbsp;</td><td>0.893&nbsp;</td><td>0.861&nbsp;</td><td>0.813&nbsp;</td><td>0.755&nbsp;</td><td>0.860 (0.004)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.909&nbsp;</td><td>0.891&nbsp;</td><td>0.871&nbsp;</td><td>0.838&nbsp;</td><td>0.784&nbsp;</td><td>0.732&nbsp;</td><td>0.837 (0.004)&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>NMI&nbsp;</th><th colspan="6">b′<hr>&nbsp;</th><th>Mean (variance)&nbsp;</th></tr><tr><th>&nbsp;</th><th>5&nbsp;</th><th>10&nbsp;</th><th>20&nbsp;</th><th>40&nbsp;</th><th>80&nbsp;</th><th>160&nbsp;</th><th>&nbsp;</th></tr></thead><tbody><tr><td>1000 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.989&nbsp;</td><td>0.978&nbsp;</td><td>0.962&nbsp;</td><td>0.933&nbsp;</td><td>0.884&nbsp;</td><td>0.836&nbsp;</td><td>0.930 (0.003)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.984&nbsp;</td><td>0.972&nbsp;</td><td>0.955&nbsp;</td><td>0.925&nbsp;</td><td>0.876&nbsp;</td><td>0.830&nbsp;</td><td>0.923 (0.003)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.976&nbsp;</td><td>0.966&nbsp;</td><td>0.949&nbsp;</td><td>0.921&nbsp;</td><td>0.872&nbsp;</td><td>0.823&nbsp;</td><td>0.917 (0.003)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.971&nbsp;</td><td>0.959&nbsp;</td><td>0.943&nbsp;</td><td>0.912&nbsp;</td><td>0.868&nbsp;</td><td>0.816&nbsp;</td><td>0.911 (0.003)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.966&nbsp;</td><td>0.956&nbsp;</td><td>0.939&nbsp;</td><td>0.908&nbsp;</td><td>0.860&nbsp;</td><td>0.805&nbsp;</td><td>0.905 (0.003)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.962&nbsp;</td><td>0.950&nbsp;</td><td>0.934&nbsp;</td><td>0.904&nbsp;</td><td>0.853&nbsp;</td><td>0.787&nbsp;</td><td>0.898 (0.004)&nbsp;</td></tr><tr><td>250 bp&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>    Full ref.&nbsp;</td><td>0.978&nbsp;</td><td>0.968&nbsp;</td><td>0.949&nbsp;</td><td>0.918&nbsp;</td><td>0.863&nbsp;</td><td>0.811&nbsp;</td><td>0.914 (0.004)&nbsp;</td></tr><tr><td>    90% ref.&nbsp;</td><td>0.967&nbsp;</td><td>0.955&nbsp;</td><td>0.935&nbsp;</td><td>0.907&nbsp;</td><td>0.854&nbsp;</td><td>0.800&nbsp;</td><td>0.903 (0.004)&nbsp;</td></tr><tr><td>    80% ref.&nbsp;</td><td>0.956&nbsp;</td><td>0.944&nbsp;</td><td>0.926&nbsp;</td><td>0.895&nbsp;</td><td>0.846&nbsp;</td><td>0.786&nbsp;</td><td>0.892 (0.004)&nbsp;</td></tr><tr><td>    70% ref.&nbsp;</td><td>0.942&nbsp;</td><td>0.926&nbsp;</td><td>0.912&nbsp;</td><td>0.880&nbsp;</td><td>0.830&nbsp;</td><td>0.773&nbsp;</td><td>0.877 (0.004)&nbsp;</td></tr><tr><td>    60% ref.&nbsp;</td><td>0.927&nbsp;</td><td>0.911&nbsp;</td><td>0.893&nbsp;</td><td>0.861&nbsp;</td><td>0.813&nbsp;</td><td>0.755&nbsp;</td><td>0.860 (0.004)&nbsp;</td></tr><tr><td>    50% ref.&nbsp;</td><td>0.909&nbsp;</td><td>0.891&nbsp;</td><td>0.871&nbsp;</td><td>0.838&nbsp;</td><td>0.784&nbsp;</td><td>0.732&nbsp;</td><td>0.837 (0.004)&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="btt499-TF3"><div class="fn"><p><em>Note</em>: ref. indicates reference sequences</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/[QueryString]" target="_blank">Open in new tab</a></div></div></div><p>When
 the reference phylogeny includes &gt;70% of the reference data, EPA-PTP
 outperforms all competing approaches, including stand-alone PTP. 
EPA-PTP outperforms PTP even when the reference phylogeny contains only 
50% of the simulated reference data for <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i33.gif" alt="forumla" path-from-xml="btt499i33.gif">⁠</span>. With increasing <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i34.gif" alt="forumla" path-from-xml="btt499i34.gif">⁠</span>, the reference data need to be more complete for EPA-PTP to outperform PTP. This is because with increasing <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i35.gif" alt="forumla" path-from-xml="btt499i35.gif">⁠</span>,
 internal branch lengths tend to get shorter and the EPA placement 
accuracy decreases. Hence, more data are needed to obtain accurate 
placements. Note that under extremely high speciation rates, EPA-PTP 
performs worse than PTP. The estimation errors may also because of (i) 
discarding sequences with low likelihood weights (see Section 2.2) (ii) 
errors in phylogenetic inferences or (iii) PTP heuristics failing to 
find the maximum likelihood species delimitation.</p><p>The results for the EPA-CROP pipeline are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S4</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. EPA-CROP outperforms the stand-alone version of CROP, but the results are worse than for EPA-PTP. </p><p>On the <em>Arthropod</em>
 metabarcoding data, the EPA-PTP pipeline yields substantially better 
results than the multistep OTU-picking pipeline used in the original 
publication (<a reveal-id="btt499-T4" data-open="btt499-T4" class="link link-reveal link-table xref-fig">Table 4</a>).
 When the complete full-length reference sequence tree is used, the 
EPA-PTP pipeline shows substantially lower ‘dropout’ and ‘no-match’ 
rates. It recovers 12.5% more species with respect to the reference data
 that represent an improvement of over 50%. Here, we apply an analogous 
criterion as in the original study where at least two reads need to be 
contained in an OTU cluster for it to be considered. In our case, <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i36.gif" alt="forumla" path-from-xml="btt499i36.gif"></span>
 reads need to be contained in a species delimitation. If an OTU cluster
 or species delimitation only contains one read, it is highly likely 
that it represents a sequencing error. However, the availability of the 
complete reference dataset is not granted for most metabarcoding 
analyses. Thus, as for the simulated data, we randomly removed up to 50%
 of the reference sequences and reran our pipelines. We then calculated 
the ratios between the number of species estimated on the reduced 
reference data relative to the number of species estimated on the 
complete reference data. The results are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S5</a></span>
 in the online supplement. When species are delimited with 
taxonomy-dependent approaches, such as the EPA, the number of estimated 
species is expected to decrease with the number of species in the 
reference data. When combined with PTP (using <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i37.gif" alt="forumla" path-from-xml="btt499i37.gif"></span>
 reads per delimitation as cutoff), EPA-PTP yields stable diversity 
estimates, irrespective of the completeness of the reference phylogeny. 
EPA-CROP also yields better results than the multistep OTU-picking 
pipeline and stand-alone CROP. The results are slightly worse than for 
EPA-PTP (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). </p>                        <a id="84627155" scrollto-destination="84627155"></a>
<div content-id="btt499-T4" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="btt499-T4" class="table-wrap-title" data-id="btt499-T4"><div class="label">Table 4.</div><div class="caption"><p><em>Arthropod</em> dataset: number of estimated MOTUs and species for the complete reference data and tree</p></div></div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>No. reads&nbsp;</th><th colspan="3">OTU-picking<hr>&nbsp;</th><th colspan="3">EPA-PTP<hr>&nbsp;</th></tr><tr><th>&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th></tr></thead><tbody><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i30.gif" alt="forumla" path-from-xml="btt499i30.gif"></span> reads&nbsp;</td><td>973&nbsp;</td><td>19&nbsp;</td><td>42.8&nbsp;</td><td>587&nbsp;</td><td>7.3&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i31.gif" alt="forumla" path-from-xml="btt499i31.gif"></span> reads&nbsp;</td><td>602&nbsp;</td><td>24&nbsp;</td><td>25.4&nbsp;</td><td>516&nbsp;</td><td>11.5&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i32.gif" alt="forumla" path-from-xml="btt499i32.gif"></span> reads&nbsp;</td><td>—&nbsp;</td><td>36&nbsp;</td><td>—&nbsp;</td><td>441&nbsp;</td><td>21.9&nbsp;</td><td>3.2&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T4" class="table-modal"><table><thead name="thead" align="left"><tr><th>No. reads&nbsp;</th><th colspan="3">OTU-picking<hr>&nbsp;</th><th colspan="3">EPA-PTP<hr>&nbsp;</th></tr><tr><th>&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th></tr></thead><tbody><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i30.gif" alt="forumla" path-from-xml="btt499i30.gif"></span> reads&nbsp;</td><td>973&nbsp;</td><td>19&nbsp;</td><td>42.8&nbsp;</td><td>587&nbsp;</td><td>7.3&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i31.gif" alt="forumla" path-from-xml="btt499i31.gif"></span> reads&nbsp;</td><td>602&nbsp;</td><td>24&nbsp;</td><td>25.4&nbsp;</td><td>516&nbsp;</td><td>11.5&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i32.gif" alt="forumla" path-from-xml="btt499i32.gif"></span> reads&nbsp;</td><td>—&nbsp;</td><td>36&nbsp;</td><td>—&nbsp;</td><td>441&nbsp;</td><td>21.9&nbsp;</td><td>3.2&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="btt499-TF2"><div class="fn"><p><em>Note</em>:
 Sanger data (the reference dataset) has a total of 547 MOTUs. The ‘—’ 
indicates that the number is not available in the original publication.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/84627155" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="btt499-T4" class="table-wrap-title" data-id="btt499-T4"><div class="label">Table 4.</div><div class="caption"><p><em>Arthropod</em> dataset: number of estimated MOTUs and species for the complete reference data and tree</p></div></div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>No. reads&nbsp;</th><th colspan="3">OTU-picking<hr>&nbsp;</th><th colspan="3">EPA-PTP<hr>&nbsp;</th></tr><tr><th>&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th></tr></thead><tbody><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i30.gif" alt="forumla" path-from-xml="btt499i30.gif"></span> reads&nbsp;</td><td>973&nbsp;</td><td>19&nbsp;</td><td>42.8&nbsp;</td><td>587&nbsp;</td><td>7.3&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i31.gif" alt="forumla" path-from-xml="btt499i31.gif"></span> reads&nbsp;</td><td>602&nbsp;</td><td>24&nbsp;</td><td>25.4&nbsp;</td><td>516&nbsp;</td><td>11.5&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i32.gif" alt="forumla" path-from-xml="btt499i32.gif"></span> reads&nbsp;</td><td>—&nbsp;</td><td>36&nbsp;</td><td>—&nbsp;</td><td>441&nbsp;</td><td>21.9&nbsp;</td><td>3.2&nbsp;</td></tr></tbody></table></div><div content-id="btt499-T4" class="table-modal"><table><thead name="thead" align="left"><tr><th>No. reads&nbsp;</th><th colspan="3">OTU-picking<hr>&nbsp;</th><th colspan="3">EPA-PTP<hr>&nbsp;</th></tr><tr><th>&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th><th>Number of cluster&nbsp;</th><th>Drop-out (%)&nbsp;</th><th>No-match (%)&nbsp;</th></tr></thead><tbody><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i30.gif" alt="forumla" path-from-xml="btt499i30.gif"></span> reads&nbsp;</td><td>973&nbsp;</td><td>19&nbsp;</td><td>42.8&nbsp;</td><td>587&nbsp;</td><td>7.3&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i31.gif" alt="forumla" path-from-xml="btt499i31.gif"></span> reads&nbsp;</td><td>602&nbsp;</td><td>24&nbsp;</td><td>25.4&nbsp;</td><td>516&nbsp;</td><td>11.5&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i32.gif" alt="forumla" path-from-xml="btt499i32.gif"></span> reads&nbsp;</td><td>—&nbsp;</td><td>36&nbsp;</td><td>—&nbsp;</td><td>441&nbsp;</td><td>21.9&nbsp;</td><td>3.2&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="btt499-TF2"><div class="fn"><p><em>Note</em>:
 Sanger data (the reference dataset) has a total of 547 MOTUs. The ‘—’ 
indicates that the number is not available in the original publication.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/84627155" target="_blank">Open in new tab</a></div></div></div>                    <h2 scrollto-destination="84627156" id="84627156" class="section-title">5 DISCUSSION</h2>
<p>We introduced, implemented and made available a new model (PTP) for 
species delimitation that is mainly intended for delimiting species in 
single-locus molecular phylogenies. PTP can propose species boundaries 
that are consistent with the PSC. An important advantage of our method 
is that it models speciation in terms of the number of substitutions. 
Thereby, it circumvents the potentially error-prone and 
compute-intensive process of generating time-calibrated ultrametric 
trees, which are required as an input for GMYC.</p><p>Using real 
datasets, we show that delimitations inferred with PTP are comparable 
with delimitations obtained via GMYC. Simulations suggest PTP 
outperforms GMYC.</p><p>In addition, it is more straightforward to use 
because it only requires a standard phylogenetic tree as input and 
because it also is substantially faster. On the 673-taxon metabarcoding 
dataset (using a modern Intel desktop processor), for instance, r8s 
requires 3 days to complete, whereas RAxML in combination with PTP only 
requires a total of about 20 min to return a species delimitation.</p><p>We
 also compared GMYC and PTP with two clustering algorithms: CROP and 
UCLUST. From our point of view, the problem of species delimitation 
needs to incoporate data from various sources (e.g. sequences <em>and</em>
 trees) and also depends heavily on the species definition used. Thus, 
GMYC and PTP yield comparable results on real data because they are 
based on the PSC. In contrast, by their very definition, CROP and UCLUST
 simply identify sequence clusters. The fact that there <em>is</em> a difference between sequence clusters and PSC-based species delimitation is underpinned by our simulations.</p><p>We simulate the data in accordance with the GMYC model that essentially adopts the PSC. To demonstrate the impact of the <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i38.gif" alt="forumla" path-from-xml="btt499i38.gif"></span>
 parameter on clustering-based delimitation accuracy, we plotted the 
pairwise sequence distances within species and between directly adjacent
 species in the simulated tree, for <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i39.gif" alt="forumla" path-from-xml="btt499i39.gif"></span> and <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i40.gif" alt="forumla" path-from-xml="btt499i40.gif"></span> in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figures S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> of the online supplement. Lower <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i41.gif" alt="forumla" path-from-xml="btt499i41.gif"></span> values lead to larger evolutionary distances between species, that is, the so-called barcoding gap (<a reveal-id="btt499-B42" data-open="btt499-B42" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>) is present. Increasing <span class="inline-formula no-formula-id"><img class="contentFigures" src="m_btt499i42.gif" alt="forumla" path-from-xml="btt499i42.gif"></span> reduces the evolutionary distances between species and the barcoding gap disappears (see <a reveal-id="btt499-B42" data-open="btt499-B42" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>
 for examples of this phenomenon on real data). Therefore, our 
simulations show that clustering algorithms work on datasets with the 
barcoding gap because phylogenetic species are mostly consistent with 
sequence clusters in this case. However, clustering methods are prone to
 fail when the barcoding gap is not present because sequences cannot be 
told apart any more via sequence similarity alone. As we show, GMYC and 
PTP delimitation performance is more robust to the absence of barcoding 
gap. Thus, when no prior information (barcoding gap presence) about the 
dataset is available and the goal is to delimit phylogenetic species, 
GMYC and PTP should be preferred.</p><p>Apart from the stand-alone PTP code, we also introduced the EPA-PTP pipeline that combines the EPA with PTP.</p><p>The
 EPA-PTP pipeline represents the first integrated approach for analyzing
 metagenomic data that combines the phylogenetic placement approach with
 an explicit statistical criterion for species delimitation. On a 
representative empirical dataset, our pipeline yields a substantially 
more accurate diversity estimate than traditional OTU-picking methods. 
Using simulated data, we show that, open reference-based approaches can 
improve delimitation accuracy compared with <em>de novo</em> approaches.
 More importantly, the EPA-PTP pipeline allows for deploying a widely 
accepted species concept to metagenomic data, where millions of 
sequences need to be processed. EPA-CROP (with the default setting of 
2000 MCMC generations) is approximately twice as fast as EPA-PTP on the 
metabarcoding dataset. Note that 2000 generations may not be sufficient,
 and that CROP does not offer a built-in MCMC convergence assessment 
criterion. </p><p>In the following, we discuss the current limitations of our approach.</p><p>Readers
 should keep in mind that entities delimited by PTP are putative species
 only. The phylogenetic trees inferred on single-gene molecular 
sequences are gene trees rather than species trees, albeit the 
hierarchical relationships above the species boundaries are expected to 
be mostly consistent with the species tree. However, the boundaries 
inferred by PTP are only approximate. Additional data need to be 
integrated to further validate the delimitations, such as morphological 
characters and multi-gene sequence data (<a reveal-id="btt499-B19" data-open="btt499-B19" class="link link-ref link-reveal xref-bibr">Ence and Carstens, 2011</a>) within an integrative taxonomy framework (<a reveal-id="btt499-B38" data-open="btt499-B38" class="link link-ref link-reveal xref-bibr">Padial <em>et al.</em>, 2010</a>; <a reveal-id="btt499-B44" data-open="btt499-B44" class="link link-ref link-reveal xref-bibr">Sauer and Hausdorf, 2012</a>).
 The putative species delimited by PTP, can, for instance, be used as 
initial hypothesis that can be further scrutinized with multilocus 
coalescent-based methods such as BP&amp;P (<a reveal-id="btt499-B56" data-open="btt499-B56" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2010</a>).
 BP&amp;P requires prior knowledge of species boundaries, and it 
represents a validation method, rather than a delimitation method. Owing
 to its computational complexity, BP&amp;P can currently only handle up 
to 20 species. </p><p>Compared with OTU-picking methods, PTP and EPA-PTP
 require substantially more CPU time because of the phylogenetic 
calculations. Although most OTU-picking methods can run on an 
off-the-shelf desktop computer, the EPA-PTP pipeline requires a 
multicore server for analyzing large metagenomic datasets. </p><p>Because
 PTP initiates the search for the maximum likelihood delimitation at the
 root of the input phylogeny, the tree has to be correctly rooted to 
obtain accurate estimates. Also, PTP should be used with caution on 
datasets where the number of individuals sampled per species is 
unbalanced and where the over-sampled species exhibit small 
within-species variation (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1</a></span> through 4).</p><p>In
 such cases, the inferred phylogeny will comprise both, subtrees 
(comprising one species and many individuals) with a large number of 
extremely short branches, and subtrees (comprising one species but only 
few individuals) with short, but not extremely short branches. Such 
unbalanced samples may require the introduction of a third λ parameter 
class of branches to accommodate (i) over-sampled within-species 
branches, (ii) within-species branches and (iii) among-species branches.
 Otherwise, the species that are not over-sampled cannot be delimited 
properly, that is, each individual is likely to be identified as a 
separate species. Hence, we either need a criterion for removing 
over-sampled sequences, or a criterion to decide when and how many 
additional classes of <em>PTP</em> (λ parameters) need to be introduced.</p><p>However, a major drawback of introducing additional <em>PTP</em>
 classes is that the delimitation search space becomes significantly 
larger. Hence, finding the maximum likelihood delimitation or a best 
known delimitation represents a challenging task. Thus, before extending
 the number of classes, we feel that more work on the design and 
performance of heuristic search strategies for species delimitation is 
required to better characterize and understand the problem. This also 
applies to the heuristics used in GMYC, given that the underlying 
optimization problems are similar.</p>                    <h2 scrollto-destination="84627170" id="84627170" class="backacknowledgements-title">ACKNOWLEDGEMENTS</h2>
<p>The authors thank T. Barraclough and T. Fujisawa for help on 
re-implementing GMYC in Python and T. Flouri for a bug fix in r8s. They 
also thank R.P. Brown, A. Papadopoulou and J. Pons for providing the 
empirical datasets. They acknowledge the support of Graduate School for 
Computing in Medicine and Life Sciences, University of Lübeck.</p><p><em>Funding</em>: Jiajie Zhang is supported by a HITS scholarship.</p><p><em>Conflict of Interest</em>: none declared.</p>                    <h2 scrollto-destination="84627174" id="84627174" class="backreferences-title">REFERENCES</h2>
<div class="ref-list"><div content-id="btt499-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barraclough</div> <div class="given-names">TG</div></div>,&nbsp; <div class="name"><div class="surname">Nee</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Phylogenetics and speciation</div>,&nbsp;<div class="source ">Trends Ecol. Evol.</div>,&nbsp;<div class="year">2001</div>,&nbsp;vol.&nbsp;<div class="volume">16</div>&nbsp;(pg.&nbsp;<div class="fpage">391</div>-<div class="lpage">399</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogenetics%20and%20speciation&amp;author=TG%20Barraclough&amp;author=S%20Nee&amp;publication_year=2001&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=16&amp;pages=391-399" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0169-5347(01)02161-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(01)02161-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0169-5347(01)02161-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11403872" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenetics%20and%20speciation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baum</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Donoghue</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Choosing among alternative ‘phylogenetic’ species concepts</div>,&nbsp;<div class="source ">Syst. Bot.</div>,&nbsp;<div class="year">1995</div>,&nbsp;vol.&nbsp;<div class="volume">20</div>&nbsp;(pg.&nbsp;<div class="fpage">560</div>-<div class="lpage">573</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Choosing%20among%20alternative%20%E2%80%98phylogenetic%E2%80%99%20species%20concepts&amp;author=D%20Baum&amp;author=M%20Donoghue&amp;publication_year=1995&amp;journal=Syst.%20Bot.&amp;volume=20&amp;pages=560-573" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2307/2419810" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2419810" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F2419810"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Choosing%20among%20alternative%20%E2%80%98phylogenetic%E2%80%99%20species%20concepts&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baum</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Shaw</div> <div class="given-names">K</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Hoch</div> <div class="given-names">PC</div></div>,&nbsp; <div class="name"><div class="surname">Stephenson</div> <div class="given-names">AG</div></div>.&nbsp;</div><div class="article-title">Genealogical perspectives on the species problem</div>,&nbsp;<div class="source ">Experimental and Molecular Approaches to Plant Biosystematics</div>,&nbsp;<div class="year">1995</div><div class="publisher-name">Missouri Botanical Garden</div>(pg.&nbsp;<div class="fpage">289</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Experimental%20and%20Molecular%20Approaches%20to%20Plant%20Biosystematics&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Experimental%20and%20Molecular%20Approaches%20to%20Plant%20Biosystematics&amp;aulast=Baum&amp;date=1995&amp;spage=289&amp;epage=303" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Experimental%20and%20Molecular%20Approaches%20to%20Plant%20Biosystematics&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Experimental%20and%20Molecular%20Approaches%20to%20Plant%20Biosystematics">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">SA</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Performance, accuracy, and Web server for evolutionary placement of short sequence reads under maximum likelihood</div>,&nbsp;<div class="source ">Syst. Biol.</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">60</div>&nbsp;(pg.&nbsp;<div class="fpage">291</div>-<div class="lpage">302</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Performance%2C%20accuracy%2C%20and%20Web%20server%20for%20evolutionary%20placement%20of%20short%20sequence%20reads%20under%20maximum%20likelihood&amp;author=SA%20Berger&amp;publication_year=2011&amp;journal=Syst.%20Biol.&amp;volume=60&amp;pages=291-302" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syr010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyr010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyr010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21436105" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Performance%2C%20accuracy%2C%20and%20Web%20server%20for%20evolutionary%20placement%20of%20short%20sequence%20reads%20under%20maximum%20likelihood&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bik</div> <div class="given-names">HM</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Sequencing our way towards understanding global eukaryotic biodiversity</div>,&nbsp;<div class="source ">Trends Ecol. Evol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">233</div>-<div class="lpage">243</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequencing%20our%20way%20towards%20understanding%20global%20eukaryotic%20biodiversity&amp;author=HM%20Bik&amp;publication_year=2012&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=27&amp;pages=233-243" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tree.2011.11.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2011.11.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tree.2011.11.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22244672" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequencing%20our%20way%20towards%20understanding%20global%20eukaryotic%20biodiversity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cai</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Sun</div> <div class="given-names">Y</div></div>.&nbsp;</div><div class="article-title">ESPRIT-Tree: hierarchical clustering analysis of millions of 16S rRNA pyrosequences in quasilinear computational time</div>,&nbsp;<div class="source ">Nucleic Acids Res.</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">39</div>&nbsp;pg.&nbsp;<div class="fpage">e95</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ESPRIT-Tree%3A%20hierarchical%20clustering%20analysis%20of%20millions%20of%2016S%20rRNA%20pyrosequences%20in%20quasilinear%20computational%20time&amp;author=Y%20Cai&amp;author=Y%20Sun&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=e95" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21596775" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ESPRIT-Tree%3A%20hierarchical%20clustering%20analysis%20of%20millions%20of%2016S%20rRNA%20pyrosequences%20in%20quasilinear%20computational%20time&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Caporaso</div> <div class="given-names">JG</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Global patterns of 16S rRNA diversity at a depth of millions of sequences per sample</div>,&nbsp;<div class="source ">Proc. Natl Acad. Sci. USA</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">108</div>&nbsp;<div class="supplement">Suppl. 1</div>(pg.&nbsp;<div class="fpage">4516</div>-<div class="lpage">4522</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Global%20patterns%20of%2016S%20rRNA%20diversity%20at%20a%20depth%20of%20millions%20of%20sequences%20per%20sample&amp;author=JG%20Caporaso&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=4516-4522" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1000080107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1000080107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1000080107"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Global%20patterns%20of%2016S%20rRNA%20diversity%20at%20a%20depth%20of%20millions%20of%20sequences%20per%20sample&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carstens</div> <div class="given-names">BC</div></div>,&nbsp; <div class="name"><div class="surname">Dewey</div> <div class="given-names">TA</div></div>.&nbsp;</div><div class="article-title">Species delimitation using a combined coalescent and information-theoretic approach: an example from North American Myotis bats</div>,&nbsp;<div class="source ">Syst. Biol.</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">59</div>&nbsp;(pg.&nbsp;<div class="fpage">400</div>-<div class="lpage">414</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Species%20delimitation%20using%20a%20combined%20coalescent%20and%20information-theoretic%20approach%3A%20an%20example%20from%20North%20American%20Myotis%20bats&amp;author=BC%20Carstens&amp;author=TA%20Dewey&amp;publication_year=2010&amp;journal=Syst.%20Biol.&amp;volume=59&amp;pages=400-414" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20547777" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Species%20delimitation%20using%20a%20combined%20coalescent%20and%20information-theoretic%20approach%3A%20an%20example%20from%20North%20American%20Myotis%20bats&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coissac</div> <div class="given-names">E</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Bioinformatic challenges for DNA metabarcoding of plants and animals</div>,&nbsp;<div class="source ">Mol. Ecol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">1834</div>-<div class="lpage">1847</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20challenges%20for%20DNA%20metabarcoding%20of%20plants%20and%20animals&amp;author=E%20Coissac&amp;publication_year=2012&amp;journal=Mol.%20Ecol.&amp;volume=21&amp;pages=1834-1847" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1365-294X.2012.05550.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2012.05550.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1365-294X.2012.05550.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22486822" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bioinformatic%20challenges%20for%20DNA%20metabarcoding%20of%20plants%20and%20animals&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">The Ribosomal Database Project: improved alignments and new tools for rRNA analysis</div>,&nbsp;<div class="source ">Nucleic Acids Res.</div>,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">37</div>&nbsp;(pg.&nbsp;<div class="fpage">D141</div>-<div class="lpage">D145</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Ribosomal%20Database%20Project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;author=JR%20Cole&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D141-D145" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkn879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19004872" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Ribosomal%20Database%20Project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cox</div> <div class="given-names">SC</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Divergence times and colonization of the Canary Islands by Gallotia lizards</div>,&nbsp;<div class="source ">Mol. Phylogenet. Evol.</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">56</div>&nbsp;(pg.&nbsp;<div class="fpage">747</div>-<div class="lpage">757</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Divergence%20times%20and%20colonization%20of%20the%20Canary%20Islands%20by%20Gallotia%20lizards&amp;author=SC%20Cox&amp;publication_year=2010&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=56&amp;pages=747-757" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ympev.2010.03.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2010.03.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2010.03.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20307675" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Divergence%20times%20and%20colonization%20of%20the%20Canary%20Islands%20by%20Gallotia%20lizards&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cracraft</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Species concepts and speciation analysis</div>,&nbsp;<div class="source ">Curr. Ornithol.</div>,&nbsp;<div class="year">1983</div>,&nbsp;vol.&nbsp;<div class="volume">1</div>&nbsp;(pg.&nbsp;<div class="fpage">159</div>-<div class="lpage">187</div>)<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Species%20concepts%20and%20speciation%20analysis&amp;aulast=Cracraft&amp;date=1983&amp;spage=159&amp;epage=187&amp;volume=1" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Species%20concepts%20and%20speciation%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davis</div> <div class="given-names">JI</div></div>,&nbsp; <div class="name"><div class="surname">Nixon</div> <div class="given-names">KC</div></div>.&nbsp;</div><div class="article-title">Populations, genetic variation, and the delimitation of phylogenetic species</div>,&nbsp;<div class="source ">Syst. Biol.</div>,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">41</div>&nbsp;(pg.&nbsp;<div class="fpage">421</div>-<div class="lpage">435</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Populations%2C%20genetic%20variation%2C%20and%20the%20delimitation%20of%20phylogenetic%20species&amp;author=JI%20Davis&amp;author=KC%20Nixon&amp;publication_year=1992&amp;journal=Syst.%20Biol.&amp;volume=41&amp;pages=421-435" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/41.4.421" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F41.4.421" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F41.4.421"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Populations%2C%20genetic%20variation%2C%20and%20the%20delimitation%20of%20phylogenetic%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Drummond</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Rambaut</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">BEAST: bayesian evolutionary analysis by sampling trees</div>,&nbsp;<div class="source ">BMC Evol. Biol.</div>,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">7</div>&nbsp;pg.&nbsp;<div class="fpage">214</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BEAST%3A%20bayesian%20evolutionary%20analysis%20by%20sampling%20trees&amp;author=AJ%20Drummond&amp;author=A%20Rambaut&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=214" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2148-7-214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2148-7-214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996036" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BEAST%3A%20bayesian%20evolutionary%20analysis%20by%20sampling%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">SR</div></div>.&nbsp;</div><div class="article-title">A new generation of homology search tools based on probabilistic inference</div>,&nbsp;<div class="source ">Genome Inform.</div>,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;(pg.&nbsp;<div class="fpage">205</div>-<div class="lpage">211</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;author=SR%20Eddy&amp;publication_year=2009&amp;journal=Genome%20Inform.&amp;volume=23&amp;pages=205-211" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20180275" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;aulast=Eddy&amp;date=2009&amp;spage=205&amp;epage=211&amp;volume=23" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>.&nbsp;</div><div class="article-title">Search and clustering orders of magnitude faster than BLAST</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">2460</div>-<div class="lpage">2461</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;author=RC%20Edgar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2460-2461" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">UCHIME improves sensitivity and speed of chimera detection</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">2194</div>-<div class="lpage">2200</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;author=RC%20Edgar&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2194-2200" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eldredge</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Cracraft</div> <div class="given-names">J</div></div>.&nbsp;</div>,&nbsp;<div class="source ">Phylogenetic Patterns and the Evolutionary Process: Method and Theory in Comparative Biology</div>,&nbsp;<div class="year">1980</div><div class="publisher-loc">New York</div><div class="publisher-name">Columbia Univ Press</div><div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Phylogenetic%20Patterns%20and%20the%20Evolutionary%20Process%3A%20Method%20and%20Theory%20in%20Comparative%20Biology&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Phylogenetic%20Patterns%20and%20the%20Evolutionary%20Process%3A%20Method%20and%20Theory%20in%20Comparative%20Biology&amp;aulast=Eldredge&amp;date=1980" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenetic%20Patterns%20and%20the%20Evolutionary%20Process%3A%20Method%20and%20Theory%20in%20Comparative%20Biology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Phylogenetic%20Patterns%20and%20the%20Evolutionary%20Process%3A%20Method%20and%20Theory%20in%20Comparative%20Biology">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ence</div> <div class="given-names">DD</div></div>,&nbsp; <div class="name"><div class="surname">Carstens</div> <div class="given-names">BC</div></div>.&nbsp;</div><div class="article-title">SpedeSTEM: a rapid and accurate method for species delimitation</div>,&nbsp;<div class="source ">Mol. Ecol. Resour.</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;(pg.&nbsp;<div class="fpage">473</div>-<div class="lpage">480</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=SpedeSTEM%3A%20a%20rapid%20and%20accurate%20method%20for%20species%20delimitation&amp;author=DD%20Ence&amp;author=BC%20Carstens&amp;publication_year=2011&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=11&amp;pages=473-480" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/men.2011.11.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2011.11.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmen.2011.11.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21481205" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:SpedeSTEM%3A%20a%20rapid%20and%20accurate%20method%20for%20species%20delimitation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Esselstyn</div> <div class="given-names">JA</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Single-locus
 species delimitation: a test of the mixed Yule-coalescent model, with 
an empirical application to Philippine round-leaf bats</div>,&nbsp;<div class="source ">Proc. Biol. Sci.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">279</div>&nbsp;(pg.&nbsp;<div class="fpage">3678</div>-<div class="lpage">3686</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Single-locus%20species%20delimitation%3A%20a%20test%20of%20the%20mixed%20Yule-coalescent%20model%2C%20with%20an%20empirical%20application%20to%20Philippine%20round-leaf%20bats&amp;author=JA%20Esselstyn&amp;publication_year=2012&amp;journal=Proc.%20Biol.%20Sci.&amp;volume=279&amp;pages=3678-3686" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rspb.2012.0705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2012.0705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frspb.2012.0705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22764163" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Single-locus%20species%20delimitation%3A%20a%20test%20of%20the%20mixed%20Yule-coalescent%20model%2C%20with%20an%20empirical%20application%20to%20Philippine%20round-leaf%20bats&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fletcher</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Yang</div> <div class="given-names">Z</div></div>.&nbsp;</div><div class="article-title">INDELible: a flexible simulator of biological sequence evolution</div>,&nbsp;<div class="source ">Mol. Biol. Evol.</div>,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">1879</div>-<div class="lpage">1888</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=INDELible%3A%20a%20flexible%20simulator%20of%20biological%20sequence%20evolution&amp;author=W%20Fletcher&amp;author=Z%20Yang&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=26&amp;pages=1879-1888" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msp098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsp098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19423664" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:INDELible%3A%20a%20flexible%20simulator%20of%20biological%20sequence%20evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Floyd</div> <div class="given-names">R</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Molecular barcodes for soil nematode identification</div>,&nbsp;<div class="source ">Mol. Ecol.</div>,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;(pg.&nbsp;<div class="fpage">839</div>-<div class="lpage">850</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20barcodes%20for%20soil%20nematode%20identification&amp;author=R%20Floyd&amp;publication_year=2002&amp;journal=Mol.%20Ecol.&amp;volume=11&amp;pages=839-850" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1365-294X.2002.01485.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294X.2002.01485.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1365-294X.2002.01485.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11972769" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20barcodes%20for%20soil%20nematode%20identification&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fontaneto</div> <div class="given-names">D</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Independently evolving species in asexual bdelloid rotifers</div>,&nbsp;<div class="source ">PLoS Biol.</div>,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">5</div>&nbsp;pg.&nbsp;<div class="fpage">e87</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Independently%20evolving%20species%20in%20asexual%20bdelloid%20rotifers&amp;author=D%20Fontaneto&amp;publication_year=2007&amp;journal=PLoS%20Biol.&amp;volume=5&amp;pages=e87" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pbio.0050087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pbio.0050087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17373857" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Independently%20evolving%20species%20in%20asexual%20bdelloid%20rotifers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">L</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">CD-HIT: accelerated for clustering the next-generation sequencing data</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">3150</div>-<div class="lpage">3152</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CD-HIT%3A%20accelerated%20for%20clustering%20the%20next-generation%20sequencing%20data&amp;author=L%20Fu&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3150-3152" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bts565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CD-HIT%3A%20accelerated%20for%20clustering%20the%20next-generation%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujisawa</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Barraclough</div> <div class="given-names">TG</div></div>.&nbsp;</div><div class="article-title">Delimiting
 species using single-locus data and the generalized mixed yule 
coalescent (GMYC) approach: a revised method and evaluation on simulated
 datasets</div>,&nbsp;<div class="source ">Syst. Biol.</div>,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">62</div>&nbsp;(pg.&nbsp;<div class="fpage">707</div>-<div class="lpage">724</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Delimiting%20species%20using%20single-locus%20data%20and%20the%20generalized%20mixed%20yule%20coalescent%20(GMYC)%20approach%3A%20a%20revised%20method%20and%20evaluation%20on%20simulated%20datasets&amp;author=T%20Fujisawa&amp;author=TG%20Barraclough&amp;publication_year=2013&amp;journal=Syst.%20Biol.&amp;volume=62&amp;pages=707-724" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syt033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyt033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyt033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23681854" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Delimiting%20species%20using%20single-locus%20data%20and%20the%20generalized%20mixed%20yule%20coalescent%20(GMYC)%20approach%3A%20a%20revised%20method%20and%20evaluation%20on%20simulated%20datasets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujita</div> <div class="given-names">MK</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Coalescent-based species delimitation in an integrative taxonomy</div>,&nbsp;<div class="source ">Trends Ecol. Evol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">480</div>-<div class="lpage">488</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Coalescent-based%20species%20delimitation%20in%20an%20integrative%20taxonomy&amp;author=MK%20Fujita&amp;publication_year=2012&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=27&amp;pages=480-488" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tree.2012.04.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2012.04.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tree.2012.04.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22633974" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Coalescent-based%20species%20delimitation%20in%20an%20integrative%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldstein</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Desalle</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Conservation genetics at the species boundary</div>,&nbsp;<div class="source ">Conserv. Biol.</div>,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">120</div>-<div class="lpage">131</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Conservation%20genetics%20at%20the%20species%20boundary&amp;author=P%20Goldstein&amp;author=R%20Desalle&amp;publication_year=2000&amp;journal=Conserv.%20Biol.&amp;volume=14&amp;pages=120-131" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1523-1739.2000.98122.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1523-1739.2000.98122.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1523-1739.2000.98122.x"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Conservation%20genetics%20at%20the%20species%20boundary&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goldstein</div> <div class="given-names">PZ</div></div>,&nbsp; <div class="name"><div class="surname">DeSalle</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Integrating DNA barcode data and taxonomic practice: determination, discovery, and description</div>,&nbsp;<div class="source ">Bioessays</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">135</div>-<div class="lpage">147</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Integrating%20DNA%20barcode%20data%20and%20taxonomic%20practice%3A%20determination%2C%20discovery%2C%20and%20description&amp;author=PZ%20Goldstein&amp;author=R%20DeSalle&amp;publication_year=2011&amp;journal=Bioessays&amp;volume=33&amp;pages=135-147" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1002/bies.v33.2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbies.v33.2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1002%2Fbies.v33.2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21184470" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Integrating%20DNA%20barcode%20data%20and%20taxonomic%20practice%3A%20determination%2C%20discovery%2C%20and%20description&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hao</div> <div class="given-names">X</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Clustering 16S rRNA for OTU prediction: a method of unsupervised Bayesian clustering</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">611</div>-<div class="lpage">618</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Clustering%2016S%20rRNA%20for%20OTU%20prediction%3A%20a%20method%20of%20unsupervised%20Bayesian%20clustering&amp;author=X%20Hao&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=611-618" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21233169" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Clustering%2016S%20rRNA%20for%20OTU%20prediction%3A%20a%20method%20of%20unsupervised%20Bayesian%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heath</div> <div class="given-names">TA</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">A dirichlet process prior for estimating lineage-specific substitution rates</div>,&nbsp;<div class="source ">Mol. Biol. Evol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">29</div>&nbsp;(pg.&nbsp;<div class="fpage">939</div>-<div class="lpage">955</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20dirichlet%20process%20prior%20for%20estimating%20lineage-specific%20substitution%20rates&amp;author=TA%20Heath&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=29&amp;pages=939-955" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msr255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsr255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22049064" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20dirichlet%20process%20prior%20for%20estimating%20lineage-specific%20substitution%20rates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B31" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">RR</div></div>.&nbsp;</div><div class="article-title">Generating samples under a Wright-Fisher neutral model of genetic variation</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">18</div>&nbsp;(pg.&nbsp;<div class="fpage">337</div>-<div class="lpage">338</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Generating%20samples%20under%20a%20Wright-Fisher%20neutral%20model%20of%20genetic%20variation&amp;author=RR%20Hudson&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=337-338" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/18.2.337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.2.337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F18.2.337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847089" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Generating%20samples%20under%20a%20Wright-Fisher%20neutral%20model%20of%20genetic%20variation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B32" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huerta-Cepas</div> <div class="given-names">J</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">ETE: a python environment for tree exploration</div>,&nbsp;<div class="source ">BMC Bioinformatics</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;pg.&nbsp;<div class="fpage">24</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ETE%3A%20a%20python%20environment%20for%20tree%20exploration&amp;author=J%20Huerta-Cepas&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=24" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-11-24" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-11-24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20070885" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ETE%3A%20a%20python%20environment%20for%20tree%20exploration&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B33" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Z</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Accurate taxonomy assignments from 16S rRNA sequences produced by highly parallel pyrosequencers</div>,&nbsp;<div class="source ">Nucleic Acids Res.</div>,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">36</div>&nbsp;pg.&nbsp;<div class="fpage">e120</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accurate%20taxonomy%20assignments%20from%2016S%20rRNA%20sequences%20produced%20by%20highly%20parallel%20pyrosequencers&amp;author=Z%20Liu&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=e120" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkn491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18723574" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accurate%20taxonomy%20assignments%20from%2016S%20rRNA%20sequences%20produced%20by%20highly%20parallel%20pyrosequencers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B34" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsen</div> <div class="given-names">FA</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree</div>,&nbsp;<div class="source ">BMC Bioinformatics</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;pg.&nbsp;<div class="fpage">538</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=pplacer%3A%20linear%20time%20maximum-likelihood%20and%20Bayesian%20phylogenetic%20placement%20of%20sequences%20onto%20a%20fixed%20reference%20tree&amp;author=FA%20Matsen&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=538" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-11-538" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-538" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-11-538"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21034504" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:pplacer%3A%20linear%20time%20maximum-likelihood%20and%20Bayesian%20phylogenetic%20placement%20of%20sequences%20onto%20a%20fixed%20reference%20tree&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B35" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyer</div> <div class="given-names">CP</div></div>,&nbsp; <div class="name"><div class="surname">Paulay</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">DNA barcoding: error rates based on comprehensive sampling</div>,&nbsp;<div class="source ">PLoS Biol.</div>,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;pg.&nbsp;<div class="fpage">e422</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DNA%20barcoding%3A%20error%20rates%20based%20on%20comprehensive%20sampling&amp;author=CP%20Meyer&amp;author=G%20Paulay&amp;publication_year=2005&amp;journal=PLoS%20Biol.&amp;volume=3&amp;pages=e422" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pbio.0030422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pbio.0030422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16336051" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DNA%20barcoding%3A%20error%20rates%20based%20on%20comprehensive%20sampling&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B36" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Monaghan</div> <div class="given-names">MT</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Accelerated species inventory on Madagascar using coalescent-based models of species delineation</div>,&nbsp;<div class="source ">Syst. Biol.</div>,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">58</div>&nbsp;(pg.&nbsp;<div class="fpage">298</div>-<div class="lpage">311</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accelerated%20species%20inventory%20on%20Madagascar%20using%20coalescent-based%20models%20of%20species%20delineation&amp;author=MT%20Monaghan&amp;publication_year=2009&amp;journal=Syst.%20Biol.&amp;volume=58&amp;pages=298-311" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syp027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyp027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyp027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525585" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accelerated%20species%20inventory%20on%20Madagascar%20using%20coalescent-based%20models%20of%20species%20delineation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B37" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nixon</div> <div class="given-names">KC</div></div>,&nbsp; <div class="name"><div class="surname">Wheeler</div> <div class="given-names">QD</div></div>.&nbsp;</div><div class="article-title">An amplification of the phylogenetic species concept</div>,&nbsp;<div class="source ">Cladistics</div>,&nbsp;<div class="year">1990</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">211</div>-<div class="lpage">223</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=An%20amplification%20of%20the%20phylogenetic%20species%20concept&amp;author=KC%20Nixon&amp;author=QD%20Wheeler&amp;publication_year=1990&amp;journal=Cladistics&amp;volume=6&amp;pages=211-223" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/cla.1990.6.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcla.1990.6.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fcla.1990.6.issue-3"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20amplification%20of%20the%20phylogenetic%20species%20concept&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B38" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Padial</div> <div class="given-names">J</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">The integrative future of taxonomy</div>,&nbsp;<div class="source ">Front. Zool.</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">7</div>&nbsp;pg.&nbsp;<div class="fpage">16</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20integrative%20future%20of%20taxonomy&amp;author=J%20Padial&amp;publication_year=2010&amp;journal=Front.%20Zool.&amp;volume=7&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1742-9994-7-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1742-9994-7-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1742-9994-7-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20500846" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20integrative%20future%20of%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B39" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Papadopoulou</div> <div class="given-names">A</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Revisiting the insect mitochondrial molecular clock: the mid-Aegean trench calibration</div>,&nbsp;<div class="source ">Mol. Biol. Evol.</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">1659</div>-<div class="lpage">1672</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Revisiting%20the%20insect%20mitochondrial%20molecular%20clock%3A%20the%20mid-Aegean%20trench%20calibration&amp;author=A%20Papadopoulou&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=1659-1672" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msq051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsq051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20167609" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Revisiting%20the%20insect%20mitochondrial%20molecular%20clock%3A%20the%20mid-Aegean%20trench%20calibration&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B40" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pons</div> <div class="given-names">J</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Sequence-based species delimitation for the DNA taxonomy of undescribed insects</div>,&nbsp;<div class="source ">Syst. Biol.</div>,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">55</div>&nbsp;(pg.&nbsp;<div class="fpage">595</div>-<div class="lpage">609</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20species%20delimitation%20for%20the%20DNA%20taxonomy%20of%20undescribed%20insects&amp;author=J%20Pons&amp;publication_year=2006&amp;journal=Syst.%20Biol.&amp;volume=55&amp;pages=595-609" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150600852011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150600852011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150600852011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16967577" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequence-based%20species%20delimitation%20for%20the%20DNA%20taxonomy%20of%20undescribed%20insects&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B41" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Powell</div> <div class="given-names">JR</div></div>.&nbsp;</div><div class="article-title">Accounting for uncertainty in species delineation during the analysis of environmental DNA sequence data</div>,&nbsp;<div class="source ">Methods Ecol. Evol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;(pg.&nbsp;<div class="fpage">1</div>-<div class="lpage">11</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accounting%20for%20uncertainty%20in%20species%20delineation%20during%20the%20analysis%20of%20environmental%20DNA%20sequence%20data&amp;author=JR%20Powell&amp;publication_year=2012&amp;journal=Methods%20Ecol.%20Evol.&amp;volume=3&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.2041-210X.2011.00122.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.2041-210X.2011.00122.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.2041-210X.2011.00122.x"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accounting%20for%20uncertainty%20in%20species%20delineation%20during%20the%20analysis%20of%20environmental%20DNA%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B42" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Puillandre</div> <div class="given-names">N</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">ABGD, Automatic Barcode Gap Discovery for primary species delimitation</div>,&nbsp;<div class="source ">Mol. Ecol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">1864</div>-<div class="lpage">1877</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ABGD%2C%20Automatic%20Barcode%20Gap%20Discovery%20for%20primary%20species%20delimitation&amp;author=N%20Puillandre&amp;publication_year=2012&amp;journal=Mol.%20Ecol.&amp;volume=21&amp;pages=1864-1877" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1365-294X.2011.05239.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2011.05239.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1365-294X.2011.05239.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21883587" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ABGD%2C%20Automatic%20Barcode%20Gap%20Discovery%20for%20primary%20species%20delimitation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B43" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sanderson</div> <div class="given-names">MJ</div></div>.&nbsp;</div><div class="article-title">r8s: inferring absolute rates of molecular evolution and divergence times in the absence of a molecular clock</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">19</div>&nbsp;(pg.&nbsp;<div class="fpage">301</div>-<div class="lpage">302</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=r8s%3A%20inferring%20absolute%20rates%20of%20molecular%20evolution%20and%20divergence%20times%20in%20the%20absence%20of%20a%20molecular%20clock&amp;author=MJ%20Sanderson&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=301-302" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/19.2.301" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F19.2.301" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F19.2.301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12538260" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:r8s%3A%20inferring%20absolute%20rates%20of%20molecular%20evolution%20and%20divergence%20times%20in%20the%20absence%20of%20a%20molecular%20clock&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B44" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sauer</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Hausdorf</div> <div class="given-names">B</div></div>.&nbsp;</div><div class="article-title">A
 comparison of DNA-based methods for delimiting species in a Cretan land
 snail radiation reveals shortcomings of exclusively molecular taxonomy</div>,&nbsp;<div class="source ">Cladistics</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">300</div>-<div class="lpage">316</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20DNA-based%20methods%20for%20delimiting%20species%20in%20a%20Cretan%20land%20snail%20radiation%20reveals%20shortcomings%20of%20exclusively%20molecular%20taxonomy&amp;author=J%20Sauer&amp;author=B%20Hausdorf&amp;publication_year=2012&amp;journal=Cladistics&amp;volume=28&amp;pages=300-316" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/cla.2012.28.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcla.2012.28.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fcla.2012.28.issue-3"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20comparison%20of%20DNA-based%20methods%20for%20delimiting%20species%20in%20a%20Cretan%20land%20snail%20radiation%20reveals%20shortcomings%20of%20exclusively%20molecular%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B45" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schloss</div> <div class="given-names">PD</div></div>,&nbsp; <div class="name"><div class="surname">Westcott</div> <div class="given-names">SL</div></div>.&nbsp;</div><div class="article-title">Assessing and improving methods used in operational taxonomic unit-based approaches for 16S rRNA gene sequence analysis</div>,&nbsp;<div class="source ">App. Environ. Microbiol.</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">77</div>&nbsp;(pg.&nbsp;<div class="fpage">3219</div>-<div class="lpage">3226</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Assessing%20and%20improving%20methods%20used%20in%20operational%20taxonomic%20unit-based%20approaches%20for%2016S%20rRNA%20gene%20sequence%20analysis&amp;author=PD%20Schloss&amp;author=SL%20Westcott&amp;publication_year=2011&amp;journal=App.%20Environ.%20Microbiol.&amp;volume=77&amp;pages=3219-3226" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.02810-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02810-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.02810-10"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Assessing%20and%20improving%20methods%20used%20in%20operational%20taxonomic%20unit-based%20approaches%20for%2016S%20rRNA%20gene%20sequence%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B46" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sites</div> <div class="given-names">JW</div></div>,&nbsp; <div class="name"><div class="surname">Marshall</div> <div class="given-names">JC</div></div>.&nbsp;</div><div class="article-title">Delimiting species: a Renaissance issue in systematic biology</div>,&nbsp;<div class="source ">Trends Ecol. Evol.</div>,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">18</div>&nbsp;(pg.&nbsp;<div class="fpage">462</div>-<div class="lpage">470</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Delimiting%20species%3A%20a%20Renaissance%20issue%20in%20systematic%20biology&amp;author=JW%20Sites&amp;author=JC%20Marshall&amp;publication_year=2003&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=18&amp;pages=462-470" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0169-5347(03)00184-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(03)00184-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0169-5347(03)00184-8"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Delimiting%20species%3A%20a%20Renaissance%20issue%20in%20systematic%20biology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B47" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sites</div> <div class="given-names">JW</div></div>,&nbsp; <div class="name"><div class="surname">Marshall</div> <div class="given-names">JC</div></div>.&nbsp;</div><div class="article-title">Operational criteria for delimiting species</div>,&nbsp;<div class="source ">Annu. Rev. Ecol. Evol. Syst.</div>,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">35</div>&nbsp;(pg.&nbsp;<div class="fpage">199</div>-<div class="lpage">227</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Operational%20criteria%20for%20delimiting%20species&amp;author=JW%20Sites&amp;author=JC%20Marshall&amp;publication_year=2004&amp;journal=Annu.%20Rev.%20Ecol.%20Evol.%20Syst.&amp;volume=35&amp;pages=199-227" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev.ecolsys.35.112202.130128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ecolsys.35.112202.130128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev.ecolsys.35.112202.130128"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Operational%20criteria%20for%20delimiting%20species&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B48" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stackebrandt</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Goebel</div> <div class="given-names">BM</div></div>.&nbsp;</div><div class="article-title">Taxonomic
 note: a place for DNA-DNA reassociation and 16S rRNA sequence analysis 
in the present species definition in bacteriology</div>,&nbsp;<div class="source ">Int.J. Syst. Bacteriol.</div>,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume">44</div>&nbsp;(pg.&nbsp;<div class="fpage">846</div>-<div class="lpage">849</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Taxonomic%20note%3A%20a%20place%20for%20DNA-DNA%20reassociation%20and%2016S%20rRNA%20sequence%20analysis%20in%20the%20present%20species%20definition%20in%20bacteriology&amp;author=E%20Stackebrandt&amp;author=BM%20Goebel&amp;publication_year=1994&amp;journal=Int.J.%20Syst.%20Bacteriol.&amp;volume=44&amp;pages=846-849" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1099/00207713-44-4-846" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-44-4-846" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1099%2F00207713-44-4-846"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Taxonomic%20note%3A%20a%20place%20for%20DNA-DNA%20reassociation%20and%2016S%20rRNA%20sequence%20analysis%20in%20the%20present%20species%20definition%20in%20bacteriology&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B49" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stajich</div> <div class="given-names">JE</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">The Bioperl toolkit: perl modules for the life sciences</div>,&nbsp;<div class="source ">Genome Res.</div>,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">1611</div>-<div class="lpage">1618</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Bioperl%20toolkit%3A%20perl%20modules%20for%20the%20life%20sciences&amp;author=JE%20Stajich&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1611-1618" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.361602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.361602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.361602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12368254" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Bioperl%20toolkit%3A%20perl%20modules%20for%20the%20life%20sciences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B50" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</div>,&nbsp;<div class="source ">Bioinformatics</div>,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">2688</div>-<div class="lpage">90</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A%20Stamatakis&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2688-90" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B51" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">Y</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">A large-scale benchmark study of existing algorithms for taxonomy-independent microbial community analysis</div>,&nbsp;<div class="source ">Briefings Bioinform.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">13</div>&nbsp;(pg.&nbsp;<div class="fpage">107</div>-<div class="lpage">121</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20large-scale%20benchmark%20study%20of%20existing%20algorithms%20for%20taxonomy-independent%20microbial%20community%20analysis&amp;author=Y%20Sun&amp;publication_year=2012&amp;journal=Briefings%20Bioinform.&amp;volume=13&amp;pages=107-121" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bib/bbr009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbr009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbib%2Fbbr009"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20large-scale%20benchmark%20study%20of%20existing%20algorithms%20for%20taxonomy-independent%20microbial%20community%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B52" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vinh</div> <div class="given-names">N</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Information theoretic measures for clusterings comparison: Variants, properties, normalization and correction for chance</div>,&nbsp;<div class="source ">J. Mach. Learn. Res.</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;(pg.&nbsp;<div class="fpage">2837</div>-<div class="lpage">2854</div>)<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Information%20theoretic%20measures%20for%20clusterings%20comparison%3A%20Variants%2C%20properties%2C%20normalization%20and%20correction%20for%20chance&amp;aulast=Vinh&amp;date=2010&amp;spage=2837&amp;epage=2854&amp;volume=11" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Information%20theoretic%20measures%20for%20clusterings%20comparison%3A%20Variants%2C%20properties%2C%20normalization%20and%20correction%20for%20chance&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B53" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vogler</div> <div class="given-names">AP</div></div>,&nbsp; <div class="name"><div class="surname">Monaghan</div> <div class="given-names">MT</div></div>.&nbsp;</div><div class="article-title">Recent advances in DNA taxonomy</div>,&nbsp;<div class="source ">J. Zool. Syst. Evol. Res.</div>,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">45</div>&nbsp;(pg.&nbsp;<div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20in%20DNA%20taxonomy&amp;author=AP%20Vogler&amp;author=MT%20Monaghan&amp;publication_year=2007&amp;journal=J.%20Zool.%20Syst.%20Evol.%20Res.&amp;volume=45&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/jzs.2007.45.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjzs.2007.45.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fjzs.2007.45.issue-1"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Recent%20advances%20in%20DNA%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B54" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vuataz</div> <div class="given-names">L</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Toward
 a DNA taxonomy of Alpine Rhithrogena (Ephemeroptera: Heptageniidae) 
using a mixed Yule-coalescent analysis of mitochondrial and nuclear DNA</div>,&nbsp;<div class="source ">PloS One</div>,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;pg.&nbsp;<div class="fpage">e19728</div>&nbsp;<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20DNA%20taxonomy%20of%20Alpine%20Rhithrogena%20(Ephemeroptera%3A%20Heptageniidae)%20using%20a%20mixed%20Yule-coalescent%20analysis%20of%20mitochondrial%20and%20nuclear%20DNA&amp;author=L%20Vuataz&amp;publication_year=2011&amp;journal=PloS%20One&amp;volume=6&amp;pages=e19728" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0019728" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019728" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0019728"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21611178" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Toward%20a%20DNA%20taxonomy%20of%20Alpine%20Rhithrogena%20(Ephemeroptera%3A%20Heptageniidae)%20using%20a%20mixed%20Yule-coalescent%20analysis%20of%20mitochondrial%20and%20nuclear%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B55" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</div>,&nbsp;<div class="source ">App. Environ. Microbiol.</div>,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">73</div>&nbsp;(pg.&nbsp;<div class="fpage">5261</div>-<div class="lpage">5267</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Naive%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;author=Q%20Wang&amp;publication_year=2007&amp;journal=App.%20Environ.%20Microbiol.&amp;volume=73&amp;pages=5261-5267" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.00062-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00062-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.00062-07"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Naive%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B56" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Rannala</div> <div class="given-names">B</div></div>.&nbsp;</div><div class="article-title">Bayesian species delimitation using multilocus sequence data</div>,&nbsp;<div class="source ">Proc. Natl Acad. Sci. USA</div>,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">107</div>&nbsp;(pg.&nbsp;<div class="fpage">9264</div>-<div class="lpage">9269</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bayesian%20species%20delimitation%20using%20multilocus%20sequence%20data&amp;author=Z%20Yang&amp;author=B%20Rannala&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=9264-9269" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0913022107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0913022107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0913022107"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bayesian%20species%20delimitation%20using%20multilocus%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btt499-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btt499-B57" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">DW</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Biodiversity soup: metabarcoding of arthropods for rapid biodiversity assessment and biomonitoring</div>,&nbsp;<div class="source ">Methods Ecol. Evol.</div>,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;(pg.&nbsp;<div class="fpage">613</div>-<div class="lpage">623</div>)<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Biodiversity%20soup%3A%20metabarcoding%20of%20arthropods%20for%20rapid%20biodiversity%20assessment%20and%20biomonitoring&amp;author=DW%20Yu&amp;publication_year=2012&amp;journal=Methods%20Ecol.%20Evol.&amp;volume=3&amp;pages=613-623" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/mee3.2012.3.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmee3.2012.3.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmee3.2012.3.issue-4"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biodiversity%20soup%3A%20metabarcoding%20of%20arthropods%20for%20rapid%20biodiversity%20assessment%20and%20biomonitoring&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix" style="display: none;"></div>

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id=""><div class="fn"><div class="footnote-content">Associate Editor: David Posada</div></div></div><div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press. All rights reserved.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution License 
(http://creativecommons.org/licenses/by/3.0/), which permits 
unrestricted reuse, distribution, and reproduction in any medium, 
provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    <div class="article-metadata-panel solr-resource-metadata">
                <div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/bioinformatics/search-results?f_TocHeadingTitle=Phylogenetics">Phylogenetics</a>
                </div>

    </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=314968&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5139">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/29/22/10.1093_bioinformatics_btt499/2/btt499_Supplementary_Data.zip?Expires=1585056692&amp;Signature=o1gwyuubOdMGRQLnWRGerJ2G1pek4n0~1F7HqK1NEx6UKsg9oDwElhVY-VhNMVOAp28uS7vnADHATBumWl6tnNuUiTwKGeAPr5Dq52wyyo9LJc9y~YStvSXoEaAeL1H0eYbUGJNI2zFncI5BIXGCA31FqMBzonLw-k6ujtK1rP-o~qu394A406fJFpeTTepQsMqfBgSakVHOF-NOFYZI-HDXCwAqwX7omqG2cs4V6Qakh7ZFLSXr9dtSWkbo6OxEagiOk4xIFuaSuDCRwQD24iwT9XSbboVaqqa8c055vkGswBMAqCCraBo7yl77858kJ6cZeQUBVgBPkXivY2CRxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="11"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        



    <div class="artmet-wrapper horizontal-artmet">

        <div class="artmet-condensed-wrap clearfix">

            <div class="artmet-condensed-stats clearfix">
                    <div class="artmet-item artmet-views">
                        <span class="artmet-number">9,424</span>
                        <span class="artmet-text">Views</span>
                    </div>

                <div class="artmet-item artmet-citations">
                    <span class="artmet-number">
                            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000326643600008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">749</a>
                    </span>
                    <span class="artmet-text">Citations</span>
                </div>
                    <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/bioinformatics/btt499" data-uuid="1eb8cbaa-a856-1c07-6a9b-dc2fea0e3646"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=1724949" style="display:inline-block;">
        <img alt="Article has an altmetric score of 9" src="https://badges.altmetric.com/?size=128&amp;score=9&amp;types=tttttttg" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

                    </div>



            </div>

                <div class="artmet-modal-trigger-wrap clearfix">
                    <a class="artmet-modal-trigger" data-article-id="314968"><i class="icon-metrics"></i><span>View Metrics</span></a>
                </div>
        </div>
            <div class="artmet-modal" id="MetricsModal">
                <div class="artmet-modal-contents">
                    <a class="artmet-close-modal">×</a>
                </div>
            </div>
    </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000326643600008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fbioinformatics%2farticle%2f29%2f22%2f2869%2f314968">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000326643600008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (749)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F29%2F22%2F2869%2F314968" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/bioinformatics/crossref-citedby/314968">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa105/5739438?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
SPsimSeq: semi-parametric simulation of bulk and single cell RNA sequencing data                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa106/5739437?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
SSIF: Subsumption-based Sub-term Inference Framework to Audit Gene Ontology                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa104/5736548?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
Michelanglo: sculpting protein views on web pages without coding                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa100/5736131?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
A Unified ILP Framework for Core Ancestral Genome Reconstruction Problems                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa099/5736130?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
GeneSwitches : Ordering gene-expression and functional events in single-cell experiments                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>
    <div class="widget widget-JobTarget widget-instance-Article_RightRailB0Article_RightRail_JobTarget">
        <div class="wrapper">
    <div class="header">
        <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/">Looking for your next opportunity?</a>
    </div>

        <div id="jobs" class="jobs-list">
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/specialist-physician-for-older-people/43778538/20102/">Specialist Physician for Older People</a></div>
                    <div class="job-location">
                        Whangarei, Northland
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/physician/53048008/20102/">Physician</a></div>
                    <div class="job-location">
                        Silver Spring, Maryland
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/sleep-medicine-opportunity/53031087/20102/">Sleep Medicine Opportunity</a></div>
                    <div class="job-location">
                        Columbus, Ohio
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/assistant-professor-and-associatefull-professor/52019081/20102/">Assistant Professor and Associate/Full Professor</a></div>
                    <div class="job-location">
                        Hershey, Pennsylvania
                    </div>
                </div>
        </div>
        <div class="all-jobs"><a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/">View all jobs</a></div>

    <div class="sponsor-and-jobnetwork-links">
        <div class="job-sponsor"><a href="http://global.oup.com/"><img src="oup-logo_002.svg" alt="OXFORD University Press"></a></div>
        <div class="job-network"><a href="http://globaljobs-careernetwork.oxfordjournals.org/home/index.cfm?site_id=20101"><img src="journals-careers-network.svg" alt="Journals Career Network"></a></div>
    </div>
</div>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="general species delimitation method with applications to phylogenetic placements | Bioinformatics | Oxford Academic" type="hidden">
<input id="hfLeftNavStickyOffset" name="hfLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="api.js"></script>
    
    
    <script type="text/javascript" src="chartist.js"></script>

    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/bioinformatics/btt499");
            googletag.enableServices();
        });
    </script>



        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/bioinformatics/pages/About">About Bioinformatics</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://twitter.com/oxfordjournals">Twitter</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://academic.oup.com/bioinformatics/subscribe">Purchase</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/bioinformatics" target="">
                        <img id="footer-logo-Bioinformatics" class="journal-footer-affiliations-logo" src="bioinformatics_f1-205396838.svg" alt="Bioinformatics">
                    </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1460-2059</li>
                                            <li>Print ISSN 1367-4803</li>
                    <li>Copyright © 2020 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2020 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
    .sf-facet-list .sf-facet label, .sf-facet-list .taxonomy-label-wrap label {font-size: .9375rem;}
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f29%2f22%2f2869%2f314968">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script src="MathJax.js"></script>


<script src="polyfill.js"></script>

    <script type="text/javascript" src="footer.js"></script>
    <script async="async" src="addthis_widget.js"></script>


    

    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>





<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="btt499.js"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh371" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1582032696894&amp;tmr=load%3D1582032696854%26core%3D1582032696887%26main%3D1582032696891%26ifr%3D1582032696897&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F29%2F22%2F2869%2F314968&amp;href=https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F29%2F22%2F2869%2F314968&amp;dt=general%20species%20delimitation%20method%20with%20applications%20to%20phylogenetic%20placements&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5e4be738c002722b&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3Dgeneral%2520species%2520delimitation%2520method%2520with%2520applications%2520to%2520phylogenetic%2520placements%26description%3DAbstract.%2520%2520Motivation%253A%2520Sequence-based%2520methods%2520to%2520delimit%2520species%2520are%2520central%2520to%2520DNA%2520taxonomy%252C%2520microbial%2520community%2520surveys%2520and%2520DNA%2520metabarcoding%2520studies.%2520Current%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fbioinformatics%252Farticle%252F29%252F22%252F2869%252F314968%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fbioinformatics%252FIssue%252F29%252F22%252F0%252Fm_bioinformatics29_22.cover.gif%253FExpires%253D1645104693%2526Signature%253Di0itJzJUtxG91QzNsQ69vcG3sJsDbKoCYO-Fa6VIVJu5i4wIS2vQGVYIpSFvRUI5eSFP~n0QvjpbhXzwcqfOCyHiJQhdKqBZiKZz5y6BHV0H-26d3DWDYAzUnGQB41EVxBgEnjgLknnLHomSxslt6Vp7Z~2jO7EanX-hiuVLFJFMOmIAZ8heGUGsBgK7TPqbVF9YLyF8yBMripVpR4jhcQ~3hHgud-uVXhunnKf6fvtz0VV4FXWmeISrBssY3nXj79yP5wgloUpKzMeR28AcP8~rC9QlMXrYE4BQKNatC9R~hxxnVUJjYifP1SDC4bT3kDPkhGywxzmS~7LH~4HgUQ__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2013-08-29&amp;csi=undefined&amp;rev=v8.28.3-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style></body></html>