<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="64566caa-5890-4546-b6d3-a6721ecb0500" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="https://cdn.krxd.net/userdata/get?pub=5c9d7251-7b10-4260-8df4-d859510ca122&amp;callback=Krux.ns._default.kxjsonp_userdata"></script><script type="text/javascript" src="https://beacon.krxd.net/optout_check?callback=Krux.ns._default.kxjsonp_optOutCheck"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";issue:issue:doi\:10.1111/mec.1998.7.issue-4;page:string:Article/Chapter View;website:website:pericles;article:article:doi\:10.1046/j.1365-294x.1998.00370.x;journal:journal:1365294x;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text"><meta data-pb-head="head-widgets-start">
<script async="" src="prum.js"></script><script src="iframe_api"></script><script async="" src="controltag.93f35b9b9881afd24d882ee537cf5cdd"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script type="text/javascript">var $DoubleClickZone = "molec-eco_mec";var $DoubleClickSite =  "wly.ecology_000046";</script><script>digitalData = {"site":{"ip":"197.248.44.39","environment":"LIVE","website":"onlinelibrary.wiley.com"},"page":{"pageGroup":"Publication Pages","page":"Article/Chapter View","subPage":"Full Text","title":"title","errorCode":200},"identities":[{"type":"BasicGroup","uuid":"be20949b-b372-471e-96a5-9963c7266a9d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000054870"}]},{"type":"BasicGroup","uuid":"98ba5e39-8ca0-4d0a-addd-395777fbf515","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000034166"}]},{"type":"BasicGroup","uuid":"e623deab-c83b-41e3-84a2-d9f1d46c5f17"},{"type":"RegionUser","uuid":"53f1676b-6f75-446d-8831-5915e1edce59"},{"type":"InstitutionUser","uuid":"dab9a169-ec74-4502-8066-0f76387b0f4e","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASP"}]},{"type":"BasicGroup","uuid":"40e3f768-07c0-45e5-aa81-97aef6eb686c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000005407"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"5196cf05-b385-4de3-91a2-2d5ffd18abc5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASPKEN"}]},{"type":"InstitutionUser","uuid":"480e4dba-ca42-475f-a7ca-5b6fbfad49f7","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000014668"}]}],"publication":{"item":{"title":"To tree or not to tree","doi":"10.1046/j.1365-294x.1998.00370.x","access":"full","earliestDate":"2002-02-28","formats":["Abstract","Full text","PDF"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"group":{"doi":"10.1111/mec.1998.7.issue-4","access":"full","volume":"7","issue":"4","coverDate":"1998-04-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"series":{"title":"Molecular Ecology","doi":"10.1111/(ISSN)1365-294X","access":"no","code":"1365294x","issn":"0962-1083","eissn":"1365-294X","topics":[{"taxonomyUri":"gpt-zonename","topicLabel":"molec-eco_MEC","topicUri":"molec-eco_mec"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.ecology_000046","topicUri":"wly.ecology_000046"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=1">







    









    

    <meta name="citation_doi" content="10.1046/j.1365-294x.1998.00370.x">

    <meta name="dc.identifier" content="10.1046/j.1365-294x.1998.00370.x">




<meta name="citation_journal_title" content="Molecular Ecology">
<meta name="citation_title" content="To tree or not to tree">
<meta name="citation_keywords" content="homoplasy">
<meta name="citation_keywords" content="phylogeography">
<meta name="citation_keywords" content="population structure">
<meta name="citation_keywords" content="spanning trees">
<meta name="citation_volume" content="7">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="399">
<meta name="citation_lastpage" content="412">
<meta name="citation_publication_date" content="1998/02/01">
<meta name="citation_online_date" content="28 February 2002">
<meta name="citation_issn" content="1365-294X">
<meta name="citation_language" content="en">
<meta name="citation_author" content="PETER E. SMOUSE">
<meta name="citation_author_institution" content="Center for Theoretical and Applied Genetics, and Department of Ecology, Evolution and Natural Resources, Rutgers University, New Brunswick, NJ 08903–0231, USA">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1046/j.1365-294x.1998.00370.x">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-294x.1998.00370.x">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1046/j.1365-294x.1998.00370.x">
<meta name="citation_publisher" content="Wiley/Blackwell (10.1111)">
<meta name="article_references" content="SMOUSE, P. E. (1998), To tree or not to tree. Molecular Ecology, 7: 399-412. doi:10.1046/j.1365-294x.1998.00370.x">
<meta name="epdf_available" content="true">


<title>To tree or not to tree - SMOUSE - 1998 - Molecular Ecology - Wiley Online Library</title>




<meta data-pb-head="head-widgets-end">

<link rel="stylesheet" type="text/css" href="f8d8product.css"><link rel="stylesheet" type="text/css" href="build.css"><link rel="stylesheet" type="text/css" href="default.css" id="pb-css" data-pb-css-id="t-1-v0/default.css">




<meta name="referrer" content="origin-when-cross-origin">

<link rel="stylesheet" type="text/css" href="hub-1529318462147.css">
<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1525191732107.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* HACK TO CHANGE LABEL IN LOGIN FORM */
.login-popup .login-form label[for="login"]:before {
    content: "Email or Customer ID";
    color: black;
}

.login-popup .login-form label[for="login"] {
    color: white;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents, 
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY STYLE TO HIDE NUMBER OF DOWNLOADS ON ARTICLE PAGES */
.info-tab-wrapper .metrics .section__body div:first-child {
	display: none;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;	
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}




</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript" xml:space="preserve">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>
	
<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	switch("journal_1365294x") {
		case "journal_23798068":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19343663":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19388969":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19349262":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_25725637":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19343647":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348533":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_25725602":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19342616":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_1934340x":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_21604762":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19483430":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19349300":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348258":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_1934368x":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_21612617":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348576":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19349289":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348290":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_15518833":
			var $DoubleClickSite = "wly.civ.eng.con_000034";
			break;
		case "journal_25738461":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_15372197":
			var $DoubleClickSite = "wly.plant.sci_000100";
			break;
		case "journal_18832148":
			var $DoubleClickSite = "wly.cardio.dis_000029";
			break;
		case "journal_21576580":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_15494918":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_24750360":
			var $DoubleClickSite = "wly.geriat.ed_000067";
			break;
		case "journal_18780369":
			var $DoubleClickSite = "wly.nursing_000088";
			break;
		case "journal_19412444":
			var $DoubleClickSite = "wly.nutrition_000089";
			break;
		case "journal_17582652":
			var $DoubleClickSite = "wly.inf.dis.mi_000072";
			break;
		case "journal_25721143":
			var $DoubleClickSite = "wly.sport.med_000112";
			break;
		case "journal_15518701":
			var $DoubleClickSite = "wly.civ.eng.con_000034";
			break;
		case "journal_14781239":
			var $DoubleClickSite = "wly.gastro.hep_000059";
			break;
		case "journal_19383673":
			var $DoubleClickSite = "wly.basic.med_000026";
			break;
		case "journal_24758817":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_24725390":
			var $DoubleClickSite = "wly.emerge.med_000050";
			break;
		case "journal_15589293":
			var $DoubleClickSite = "wly.indus.chem_000124";
			break;
		case "journal_15589307":
			var $DoubleClickSite = "wly.food.sci_000058";
			break;
		case "journal_15589331":
			var $DoubleClickSite = "wly.chemistry_000033";
			break;
		case "journal_18735924":
			var $DoubleClickSite = "wly.finan.inves_000057";
			break;
		case "journal_25770470":
			var $DoubleClickSite = "wly.elec.e.eng_000049";
			break;
		case "journal_25738615":
			var $DoubleClickSite = "wly.finan.inves_000057";
	};

	switch("journal_1365294x") {
		case "journal_23798068":
			var $DoubleClickZone = "cp-plant-bio_CPPB";
			break;
		case "journal_19343663":
			var $DoubleClickZone = "cp-protein-sci_CPPS";
			break;
		case "journal_19388969":
			var $DoubleClickZone = "cp-stem-cell-bio_CPSC";
			break;
		case "journal_19349262":
			var $DoubleClickZone = "cp-toxicology_CPTX";
			break;
		case "journal_25725637":
			var $DoubleClickZone = "cp-mri_CPMR";
			break;
		case "journal_19343647":
			var $DoubleClickZone = "cp-molecular-bio_CPMB";
			break;
		case "jorunal_19348533":
			var $DoubleClickZone = "cp-microbio_CPMI";
			break;
		case "journal_25725602":
			var $DoubleClickZone = "cp-food-a-chem_CPFA";
			break;
		case "journal_19342616":
			var $DoubleClickZone = "cp-cell-bio_CPCL";
			break;
		case "journal_1934340x":
			var $DoubleClickZone = "cp-bioinfo_CPBI";
			break;
		case "journal_21604762":
			var $DoubleClickZone = "cp-chem-bio_CPCB";
			break;
		case "journal_19483430":
			var $DoubleClickZone = "cp-ess-lab-tech_CPLT";
			break;
		case "journal_19349300":
			var $DoubleClickZone = "cp-cytometry_CPCY";
			break;
		case "journal_19348258":
			var $DoubleClickZone = "cp-human-gen_CPHG";
			break;
		case "journal_1934368x":
			var $DoubleClickZone = "cp-immun_CPIM";
			break;
		case "journal_21612617":
			var $DoubleClickZone = "cp-mouse-b_CPMO";
			break;
		case "journal_19348576":
			var $DoubleClickZone = "cp-neurosci_CPNE";
			break;
		case "journal_19349289":
			var $DoubleClickZone = "cp-n-acid-chem_CPNA";
			break;
		case "journal_19348290":
			var $DoubleClickZone = "cp-pharma_CPPH";
			break;
		case "journal_15518833":
			var $DoubleClickZone = "jrnl-american-water-works-assc_AWWA";
			break;
		case "journal_25738461":
			var $DoubleClickZone = "adv-in-cell-and-gene_ACG2";
			break;
		case "journal_15372197":
			var $DoubleClickZone = "am-j-o-botany_AJB2";
			break;
		case "journal_18832148":
			var $DoubleClickZone = "j-o-arrhythmia_JOA3";
			break;
		case "journal_21576580":
			var $DoubleClickZone = "stem-cells-trans-medicine_STC3";
			break;
		case "journal_15494918":
			var $DoubleClickZone = "stem-cells_STEM";
			break;
		case "journal_24750360":
			var $DoubleClickZone = "aging-medicine_AGM2";
			break;
		case "journal_18780369":
			var $DoubleClickZone = "aorn_AORN";
			break;
		case "journal_19412444":
			var $DoubleClickZone = "j-o-parent-and-enteral-nutrition_JPEN";
			break;
		case "journal_17582652":
			var $DoubleClickZone = "j-international-aids-s_JIA2";
			break;
		case "journal_25721143":
			var $DoubleClickZone = "j-orthoped-res_JOR";
			break;
		case "journal_15518701":
			var $DoubleClickZone = "opflow_OPFL";
			break;
		case "journal_14781239":
			var $DoubleClickZone = "j-gastro-hepatology_JGH";
			break;
		case "journal_19383673":
			var $DoubleClickZone = "jrnl-leukocyte-biology_JLB";
			break;
		case "journal_24758817":
			var $DoubleClickZone = "peptide-science_PEP2";
			break;
		case "journal_24725390":
			var $DoubleClickZone = "academic-emer-med_acem_ET";
			break;
		case "journal_15589293":
			var $DoubleClickZone = "jrnl-surfact-detergents_JSDE";
			break;
		case "journal_15589307":
			var $DoubleClickZone = "lipids_LIPD";
			break;
		case "journal_15589331":
			var $DoubleClickZone = "j-amer-oil_chem";
			break;
		case "journal_18735924":
			var $DoubleClickZone = "rev-finan-econ_RFE";
			break;
		case "journal_25770470":
			var $DoubleClickZone = "j-quant-eng_QUE2";
			break;
		case "journal_25738615":
			var $DoubleClickZone = "Fin-plan-review";
	};

	if (!$DoubleClickSite) {
		var $DoubleClickSite = "undefined-site";
	};

	if (!$DoubleClickZone) {
		var $DoubleClickZone = "undefined-zone";
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};

		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		); 	// Krux

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );
				
		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ()) 
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ()) 
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>









    
        
    








    




<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "To tree or not to tree - SMOUSE - 1998 - Molecular Ecology - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker.js"></script><script async="" type="text/javascript" src="epdf_linker_002.js"></script>
<style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_stumbleupon_badge,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_239.js" async=""></script><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML">MathJax.Hub.Config({displayAlign: "left", "HTML-CSS": { linebreaks: {automatic: true,width: "690px"}, undefinedFamily: "STIXGeneral, Arial Unicode MS, icomoon, serif"},"fast-preview": { disabled: true },tex2jax: {inlineMath: [["$", "$"], ["\\(", "\\)"]],processEscapes: true},styles: {"span.MathJax": {"padding-right": "2px"}}});MathJax.Hub.Startup.onload();</script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
span.MathJax {padding-right: 2px}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script></head>
<body class="pb-ui"><div id="MathJax_Message" style="display: none;"></div>

    <a href="#article__content" id="skipNavigationLink" class="skip" tabindex="0">Skip to Main Content</a>




<script type="text/javascript">

    if(true) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('article__content');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('article__content').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader" style="position: fixed;">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        


        
    

</div></div></div>

        </div>
    




        
        




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg hidden-md padded-content">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm institution--block">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access Provided by</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper" style="overflow: hidden;"><div class="tab clearfix"><input placeholder="Enter Your Search Term" class="fakeQSInput" type="search"><div class="dropBlock" data-db-parent-of="dbTab-0"><a href="#" class="dropBlock__link" data-db-target-for="dbTab-0"><span>This Journal</span><i class="icon-arrow_d_n" aria-hidden="true"></i></a><ul role="tablist" class="rlist tab__nav dropblock--tab" data-db-target-of="dbTab-0"><li class="active"><a href="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" role="tab" data-toggle="tab" id="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" aria-expanded="true" data-search-in="thisJournal">This Journal</a></li><li><a href="#pane-67065c09-4a88-49cd-934c-ac707951d35c1" role="tab" data-toggle="tab" id="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" aria-expanded="false" data-search-in="default">Anywhere</a></li></ul></div><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Enter Your Search Term" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="1365294x" type="hidden"><div class="search-options-wrapper"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced#citation" title="" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" class="btn quick-search__button icon-search"><span>Search</span></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Enter Your Search Term" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced#citation" title="" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" class="btn quick-search__button icon-search"><span>Search</span></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    








<div class="loginBar login-open">
    <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x" class="show-login">
        <span class="icon-profile_empty"></span>
        <span class="sign-in-label">Login / Register</span>
    </a>
    
</div>


        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    




        
        




        
        




        
        <!-- [Tag{id&#x3D;103071, uri&#x3D;molec-eco_mec, name&#x3D;molec-eco_MEC}, Tag{id&#x3D;101738, uri&#x3D;wly.ecology_000046, name&#x3D;wly.ecology_000046}, Tag{id&#x3D;320, uri&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;688, uri&#x3D;ch00, name&#x3D;General Chemistry}, Tag{id&#x3D;1184, uri&#x3D;ls2f, name&#x3D;Ecological Genetics}, Tag{id&#x3D;323, uri&#x3D;ls40, name&#x3D;Evolution}, Tag{id&#x3D;1206, uri&#x3D;ls43, name&#x3D;Molecular Evolution}, Tag{id&#x3D;1207, uri&#x3D;ls41, name&#x3D;Evolutionary Biology}, Tag{id&#x3D;60, uri&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;1180, uri&#x3D;ls2g, name&#x3D;Molecular Ecology}, Tag{id&#x3D;93, uri&#x3D;ch, name&#x3D;CHEMISTRY}, Tag{id&#x3D;317, uri&#x3D;ls60, name&#x3D;Genetics}] -->                          <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 












        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet" style="padding-top: 71px;">
        



        
        <!--begin pagefulltext-->
<div class="page-body pagefulltext">
    <div data-pb-dropzone="main">
    



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        


        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x&pubId=50930488&placeholderId=1015&productId=101018" target="_blank">
					<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_0" title="3rd party ad content" name="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="100%" frameborder="0" height="0"></iframe></div></div>
	</aside>
        </div>
    


        </div>
    




        
        
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.1998.7.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1046%2Fj.1365-294x.1998.00370.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.1998.7.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1046%2Fj.1365-294x.1998.00370.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.1998.7.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1046%2Fj.1365-294x.1998.00370.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=global-subject-codes" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.1998.7.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1046%2Fj.1365-294x.1998.00370.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=global-subject-codes" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone"></div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      



        
        <div data-pb-dropzone="publicaitonContent-series-title" class="pb-dropzone">
  <h1><a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage">Molecular Ecology</a></h1>
  <a href="https://onlinelibrary.wiley.com/toc/1365294x/7/4" class="volume-issue">Volume 7, Issue 4</a>
  <a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage" class="citation--logo"><img class="journal-banner-image" test="test-value" alt="Molecular Ecology banner" src="1365294x-1525974682817.jpg"></a>
</div>

   </div>
   <div class="doi-access-container clearfix">
      <div class="doi-access-wrapper"><div class="full-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Full Access</div></div></div></div>
   </div>
   <h2 class="citation__title">To tree or not to tree</h2>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=SMOUSE%2C+PETER+E" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl"><span>PETER E. SMOUSE</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" id="am1">
            <p>Center for Theoretical and Applied Genetics, and 
Department of Ecology, Evolution and Natural Resources, Rutgers 
University, New Brunswick, NJ 08903–0231, USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=SMOUSE%2C+PETER+E">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=SMOUSE%2C+PETER+E" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl"><span>PETER E. SMOUSE</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" id="a1" style="display: none;">
                  <p>Center for Theoretical and Applied Genetics, and 
Department of Ecology, Evolution and Natural Resources, Rutgers 
University, New Brunswick, NJ 08903–0231, USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=SMOUSE%2C+PETER+E">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections">
      <div class="epub-section"><span class="epub-state">First published:&nbsp;</span><span class="epub-date">28 February 2002</span></div>
      <div class="epub-section"><a class="epub-doi" href="https://doi.org/10.1046/j.1365-294x.1998.00370.x">https://doi.org/10.1046/j.1365-294x.1998.00370.x</a></div>
      <div class="epub-section cited-by-count"><span>Cited by: <a href="#citedby-section">47</a></span></div>
   </div>
   <div class="article-header__correspondence-to"><i aria-hidden="true" class="at-icon-wrapper icon-mail_outline"></i>
P. E. Smouse Tel.: +01‐732‐932‐1064; Fax: +01‐732‐932‐8746; E‐mail <a title="
                        Link to email address
                    " href="mailto:Smouse@AESOP.Rutgers.EDU">Smouse@AESOP.Rutgers.EDU</a></div>
   <div class="epub-sections"></div>
</div></div><nav class="stickybar coolBar trans"><div class="stickybar__wrapper coolBar__wrapper clearfix" style="width: 690px; top: 0px;"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><div class="transplanted-clone"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-294x.1998.00370.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1046/j.1365-294x.1998.00370.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></div></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d34467600"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss1-title"><span> Introduction</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss2-title"><span> Turning phenetic data into a tree</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss8-title"><span> Analysing biogeography</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss13-title"><span> A tree is a picture</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss14-1-title"><span> Acknowledgements</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><ul data-cb-group="Article" data-cb-group-icon="icon-toc" class="rlist coolBar__first"></ul><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1046/j.1365-294x.1998.00370.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1046/j.1365-294x.1998.00370.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1046%2Fj.1365-294x.1998.00370.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1046%2Fj.1365-294x.1998.00370.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1046%2Fj.1365-294x.1998.00370.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" src="addthis_widget.js"></script><div class="relative"><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1046/j.1365-294x.1998.00370.x/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="http://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="http://www.wiley.com/WileyCDA/Section/id-829333.html" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email to a friend</a></li><li><a class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linkedin</a></li><li><a class="addthis_button_google_plusone_share" target="_blank" title="Google+" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-google-plus"></i>Google+</a></li><li><a class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><li><a class="addthis_button_citeulike" target="_blank" title="CiteULike" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-citeulike"></i>CiteULike</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav><div class="article__body"><style>
            .inline-equation {
            overflow-x:auto;
            }
        </style><article>
   
   
   <div class="abstract-group">
      <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
         <h2 id="d34467600" class="article-section__header section__title main abstractlang_en main">Abstract</h2>
         <div class="article-section__content en main">
            
            <p>The practice of tracking geographical divergence along a phylogenetic tree has added
               an evolutionary perspective to biogeographic analysis within single species. In spite
               of the popularity of phylogeography, there is an emerging problem. Recurrent mutation
               and recombination both create homoplasy, multiple evolutionary occurrences of the
               same character that are identical in state but not identical by descent. Homoplasic
               molecular data are phylogenetically ambiguous. Converting homoplasic molecular data
               into a tree represents an extrapolation, and there can be myriad candidate trees among
               which to choose. Derivative biogeographic analyses of ‘the tree’ are analyses of that
               extrapolation, and the results depend on the tree chosen. I explore the informational
               aspects of converting a multicharacter data set into a phylogenetic tree, and then
               explore what happens when that tree is used for population analysis. Three conclusions
               follow: (i) some trees are better than others; good trees are true to the data, whereas
               bad trees are not; (ii) for biogeographic analysis, we should use only good trees,
               which yield the same biogeographic inference as the phenetic data, but little more;
               and (iii) the reliable biogeographic inference is inherent in the phenetic data, not
               the trees.
            </p>
            
         </div>
      </section>
   </div>
   
   <section class="article-section article-section__full">
      
      <div class="article-section__content" id="ss1">
         
         <h2 class="article-section__title section__title section1" id="ss1-title"> Introduction</h2>
         
         <p>The molecular revolution has made it feasible to attack problems of population genetics
            and systematics with a vast array of variable genetic markers. While a single marker
            can yield idiosyncratic results, a large collection of independent markers should
            exhibit a coherent pattern, perhaps best viewed as a molecular manifestation of the
            Central Limit Theorem. We often have more genetic markers than we have individuals,
            however, and we are in a poor position to make any assumptions about so many markers.
            For any substantial number of markers, the standard assumption of independent segregation
            is difficult to accept, and we are certainly in no position to test it, given the
            limited number of individuals at our disposal. Rather than assume independent segregation
            for all the markers, we assume that the mutations that have given rise to the different
            states of each character were (evolutionarily) independent and sequential. We can
            then relate multicharacter genotypes to each other, either in terms of the numbers
            of substitutional differences that separate them (phenetic distances) or in terms
            of their pathways of evolutionary divergence (patristic distances).
         </p>
         
         <p>The comparison of multicharacter genotypes over geographical space, using phenetic
            methods, is a form of biogeographic analysis. Traditional population genetics and
            systematics both have voluminous literatures on the subject. The practice of tracking
            biogeographic divergence along a phylogenetic tree, an enterprise for which <span><a href="#b2" class="bibLink tab-link" data-tab="pane-pcw-references">2</a></span>) have coined the term phylogeography, has added an overtly evolutionary perspective
            to biogeographic studies. The practice of intraspecific phylogeography has spread
            so rapidly (e.g. <span><a href="#b30" class="bibLink tab-link" data-tab="pane-pcw-references">30</a></span>; <span><a href="#b1" class="bibLink tab-link" data-tab="pane-pcw-references">1</a></span>; <span><a href="#b33" class="bibLink tab-link" data-tab="pane-pcw-references">33</a></span>; <span><a href="#b44" class="bibLink tab-link" data-tab="pane-pcw-references">44</a></span>; <span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>; <span><a href="#b25" class="bibLink tab-link" data-tab="pane-pcw-references">25</a></span>; <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>; <span><a href="#b21" class="bibLink tab-link" data-tab="pane-pcw-references">21</a></span>; <span><a href="#b9" class="bibLink tab-link" data-tab="pane-pcw-references">9</a></span>) that we find ourselves celebrating its 10th anniversary with a special issue of
            <i>Molecular Ecology</i>.
         </p>
         
         <p>At the risk of casting a pall over the celebration, I must point out that something
            is amiss. In many cases, the genetic data on which the enterprise depends are phylogenetically
            ambiguous, and that has implications for the biogeographic analyses that follow. There
            are two obvious sources of such ambiguity.
         </p>
         
         <p>(i) Recurrent mutation. In our never‐ending quest for highly informative (highly polymorphic)
            markers within a single species, we are (almost by default) selecting genetic markers
            with recurrent mutation rates high enough to yield multiple mutations over the phylogenetic
            timescale of interest. The net consequence is a large amount of mutational homoplasy
            in the raw data, multiple evolutionary substitutions that are identical in state but
            not identical by descent (<span><a href="#b42" class="bibLink tab-link" data-tab="pane-pcw-references">42</a></span>; <span><a href="#b18" class="bibLink tab-link" data-tab="pane-pcw-references">18</a></span>; <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>; <span><a href="#b3" class="bibLink tab-link" data-tab="pane-pcw-references">3</a></span>). We have always known that allozyme alleles could be homoplasic, but the same is
            obviously true for RFLP markers, single nucleotide sites, RAPD markers, minisatellites,
            and microsatellites; homoplasy is almost inherent in the exercise. Homoplasy is a
            serious problem with many molecular data sets, and it leads to phylogenetic ambiguity.
            Many of our molecular data sets are less than compellingly phyletic.
         </p>
         
         <p>(ii) Recombination. Within a single species, even closely linked genes will become
            decoupled over evolutionary time, and multicharacter genetic profiles cannot be related
            to each other in a strictly phyletic fashion. In a sense, all character states become
            homoplasic. The phylogenetic tree derived from all the data (species tree) will be
            some average that does not conform well to any particular gene tree. Many authors
            have made the obvious distinction between gene trees and a species trees, but what
            we should do about the duality, other than acknowledge it, remains unclear.
         </p>
         
         <p>The usual strategy is to concentrate on one or a few loci, deriving a separate gene
            tree for each; this strategy has the virtue of avoiding the complication of recombination
            among them. To the extent that the data are convincingly tree‐like, the tree will
            tell the phylogenetic story for the gene, but even for closely linked markers within
            a single gene, intragenic recombination can occur often enough (over evolutionary
            time) to confound a strictly phyletic interpretation. <span><a href="#b41" class="bibLink tab-link" data-tab="pane-pcw-references">41</a></span>) and <span><a href="#b29" class="bibLink tab-link" data-tab="pane-pcw-references">29</a></span>) showed that if intragenic recombination occurs no more often than point mutation
            within the gene, surely a plausible assumption, then a very substantial fraction of
            the allelic variation encountered will be due to recombination. As an unavoidable
            consequence, individual molecular characters can become highly homoplasic. A striking
            example is provided by the β‐globin region in humans (<span><a href="#b23" class="bibLink tab-link" data-tab="pane-pcw-references">23</a></span>), the evolution of which has as much to do with recombination as it does phylesis.
            To avoid recombinational complications, many have turned to nonrecombining organellar
            genomes that are uniparentally inherited, mtDNA or cpDNA (<span><a href="#b7" class="bibLink tab-link" data-tab="pane-pcw-references">7</a></span>; <span><a href="#b5" class="bibLink tab-link" data-tab="pane-pcw-references">5</a></span>; <span><a href="#b16" class="bibLink tab-link" data-tab="pane-pcw-references">16</a></span>; <span><a href="#b45" class="bibLink tab-link" data-tab="pane-pcw-references">45</a></span>; <span><a href="#b40" class="bibLink tab-link" data-tab="pane-pcw-references">40</a></span>; <span><a href="#b22" class="bibLink tab-link" data-tab="pane-pcw-references">22</a></span>; <span><a href="#b26" class="bibLink tab-link" data-tab="pane-pcw-references">26</a></span>). If the mutation rate is high enough and the time frame is large enough, mutational
            homoplasy will remain a problem, and many molecular character sets are chosen precisely
            because the mutation rate is high enough to generate substantial variation.
         </p>
         
         <p>The more homoplasic ambiguity we encounter in the data, the larger the degree of phylogenetic
            uncertainty we have. Notwithstanding this difficulty, the usual practice is to use
            some convenient tree‐making algorithm to choose a tree, and then to proceed with biogeographic
            analysis. Where there exists a considerable range of alternatives from among which
            to choose, the choice itself can become an issue. The 'African Eve' exchange (<span><a href="#b24" class="bibLink tab-link" data-tab="pane-pcw-references">24</a></span>; <span><a href="#b44" class="bibLink tab-link" data-tab="pane-pcw-references">44</a></span>; <span><a href="#b17" class="bibLink tab-link" data-tab="pane-pcw-references">17</a></span>; <span><a href="#b43" class="bibLink tab-link" data-tab="pane-pcw-references">43</a></span>) is a case in point.
         </p>
         
         <p>There are data sets which are compellingly tree‐like and, for them, the choice of
            tree is easy, and the resulting biogeographic interpretation is entirely plausible.
            Many of our data sets are not credibly tree‐like, however, and to the extent that
            a tree is not inherent in the raw data, there will be an element of extrapolation
            involved in the resulting phylogeny. To the same extent that phylogenetic extrapolation
            is uncertain, any derivative population structure or biogeographic analyses are also
            uncertain. Attention has recently been devoted to minimizing the impact of that phylogenetic
            ambiguity (<span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>; <span><a href="#b8" class="bibLink tab-link" data-tab="pane-pcw-references">8</a></span>; <span><a href="#b10" class="bibLink tab-link" data-tab="pane-pcw-references">10</a></span>; <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>; <span><a href="#b3" class="bibLink tab-link" data-tab="pane-pcw-references">3</a></span>), but there are important questions that need attention. Is there any necessity to
            impose a phylogenetic tree for population analysis? Is there anything to be gained
            by doing so? Basically, we need to decide whether to tree or not to tree.
         </p>
         
         <p>The objectives of this study are to explore the informational aspects of converting
            a multicharacter data set into a phylogenetic tree, and then to explore what happens
            when that tree is used for population analysis. In the process of exploration, using
            a simple example, I will address two proximal questions: (i) how much phylogenetic
            ambiguity is inherent in the molecular data set, and how can we choose among multiple
            candidate trees; and (ii) how does our choice of tree impact on subsequent biogeographic
            analyses? Buttressing the example with results from the literature, I will argue that
            while some trees are harmless, in biogeographic context, others are problematic. Inasmuch
            as we can extract the reliable biogeographic inference from the phenetic data themselves,
            I will argue that we might be better served by using tree‐free methods for biogeographic
            analysis.
         </p>
         
      </div>
      
      <div class="article-section__content" id="ss2">
         
         <h2 class="article-section__title section__title section1" id="ss2-title"> Turning phenetic data into a tree</h2>
         
         <div class="article-section__sub-content" id="ss3">
            
            <h3 class="article-section__sub-title section2"> Homoplasy and the hypercube</h3>
            
            <p>Our problem is best understood if we have a clear sense of what we are actually doing
               when we make a tree. What follows is applicable to almost any type of multicharacter
               data set, but I will concentrate on uniparentally inherited haploid genomes, mitochondria
               or chloroplasts. For organelle genomes, any particular combination of multiple character
               states is inherited as an indivisible unit, called a haplotype. In the absence of
               recombination, the primary problem becomes one of mutational homoplasy. I will concentrate
               on haplotypic marker sets, because recombining markers can only make the situation
               worse. My comments thus pertain to a best‐case analysis but, as we shall see, the
               best may not be very good.
            </p>
            
            <p>It is also convenient to illustrate with binary characters, because they are sufficient
               to show the nature and scope of the problem. Multistate characters can also be accommodated,
               but they require additional mathematical elaboration, and they change neither the
               essential questions nor the answers. So, imagine a string of M binary characters,
               each coded (0, 1). We might be talking about a set of restriction sites, each coded
               as either present (1) or absent (0), a set of deletions/insertions/inversions, each
               coded as present (1) or absent (0), or even a sequence of polymorphic nucleotides
               that, by virtue of strong transitional bias, exhibit alternative states A and G (or
               C and T), each site coded arbitrarily as (0, 1).
            </p>
            
            <p>For each individual in the study, we have a binary vector of length M (a string of
               1s and 0s), representing the M polymorphic character states for that individual. The
               different states of that vector (haplotypes) could be treated simply as distinguishable
               alleles (as with allozyme alleles), without regard to their levels of divergence,
               because they are inherited as indivisible units. While that is a perfectly legitimate
               treatment, it sometimes makes poor use of our hard‐won molecular data (cf. <span><a href="#b11" class="bibLink tab-link" data-tab="pane-pcw-references">11</a></span>). If we use the degree of divergence among the haplotypes, we can extract additional
               (potentially useful) information (<span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>).
            </p>
            
            <p>We begin by defining a measure of phenetic divergence (distance) between any two haplotypes.
               There are many different distance metrics in common usage, but it is convenient to
               formulate the problem with a Manhattan (city‐block) metric. To set the notation and
               begin the illustration, imagine a set of M = 3 binary characters, denoted the 1st,
               2nd and 3rd (<a href="#f1" class="scrollableLink">Fig. 1a</a>). With M = 3 characters, there are 2<sup>3</sup> = 8 possible haplotypes, denoted by a set of 3 vectors (<i>υ<sub>i</sub></i>) = (A = 000, B = 100, C = 010, D = 001, E = 110, F = 101, G = 011, H = 111), as in
               <a href="#f1" class="scrollableLink"></a>Fig. 1a, and collectively occupying all the vertices of a three‐dimensional cube (<a href="#f1" class="scrollableLink">Fig. 1b</a>). The Manhattan distance between adjacent haplotypes (e.g. A–B, B–E, G–H) is 1; that
               between haplotypes at opposite corners in the same plane <i>(</i>e.g. A–E, B–D, F–G) is 2; and that between haplotypes at opposite vertices of the
               three‐dimensional cube (e.g. A–H, B–G, C–F) is 3. In a perfectly tree‐like (strictly
               phyletic) data set, we would have one more haplotype (L) than we have characters (M),
               and there would be no gaps and no closed loops (i.e. L = M + 1). We prefer the sort
               of data set represented in <a href="#f1" class="scrollableLink"></a>Fig. 1c which, although small, is phylogenetically unambiguous. Unfortunately, we
               all too often encounter data sets more similar to those represented in <a href="#f1" class="scrollableLink"></a>Fig. 1d,e. Many data sets have portions of the character space that even show the
               pattern in <a href="#f1" class="scrollableLink"></a>Fig. 1b. The illustration is a mere three‐dimensional version of a much larger problem.
               With multiple (M &gt; 4) characters, we can always place the L haplotypes at the vertices
               of an M‐dimensional analogue of our cube (a hypercube). Multicharacter data sets frequently
               exhibit substantial homoplasy, many closed (sometimes interlocking) loops, and nontrivial
               phylogenetic ambiguity.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f1"><a target="_blank" href="#">
                     <!--EMPTY SOURCE--></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f1&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>
                        
                        <p>A homoplasic octet of three‐character, binary haplotypes, and their representation
                           as the vertices of a three‐dimensional unit cube. (a) vector representation of the
                           eight haplotypes; (b) cubic representation of the full octet; (c) a tree‐like set
                           of four haplotypes; (d) an ambiguous set of five haplotypes; (e) an ambiguous set
                           of six haplotypes; solid lines represent single‐step connecting edges between observed
                           haplotypes.
                        </p>
                        
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="faf1" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="faf1" class="accordion__content" style="display: none;">
                           A homoplasic octet of three‐character, binary haplotypes, and their representation
                           as the vertices of a three‐dimensional unit cube. (a) vector representation of the
                           eight haplotypes; (b) cubic representation of the full octet; (c) a tree‐like set
                           of four haplotypes; (d) an ambiguous set of five haplotypes; (e) an ambiguous set
                           of six haplotypes; solid lines represent single‐step connecting edges between observed
                           haplotypes.
                           
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p> <b>Message 1.</b> At the heart of our problem is the fact that many of the available data sets are
               homoplasic for the most informative (most polymorphic) portion of the character set,
               and phylogenetic ambiguity is necessarily severe.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss4">
            
            <h3 class="article-section__sub-title section2"> How many characters are enough?</h3>
            
            <p>Keeping in mind that the entire exercise is being pursued in the interests of population
               structure and/or biogeographic analysis within a single species, we will have a survey
               of <i>P</i> populations, and we will have sampled <i>N</i> individuals. As we anticipate variation within populations, we should have <i>N</i> &gt; <i>P</i>. I have commented elsewhere on the need for adequate population subsampling (<span><a href="#b36" class="bibLink tab-link" data-tab="pane-pcw-references">36</a></span>; <span><a href="#b35" class="bibLink tab-link" data-tab="pane-pcw-references">35</a></span>). These <i>N</i> individuals will exhibit L differentiated haplotypes, occupying L of the 2<sup>M</sup> vertices of our M‐dimensional hypercube. The natural tendency in molecular systematics
               is to use a large battery of characters, so that each individual is genetically unique,
               but how many characters do we need to characterize <i>N</i> individuals? Is it really necessary to have many more molecular characters than individuals?
            </p>
            
            <p>There are three cases of interest. Case (i) is that of a perfectly tree‐like data
               set (L = M + 1); there are no closed loops and no missing intermediates. An example
               would be a data set consisting of haplotypes A, B, C and D, as in <a href="#f1" class="scrollableLink"></a>Fig. 1c. There are three single‐step substitutional changes needed to connect those
               haplotypes (A–B, A–C and A–D), and each involves a separate character. The ideal outcome
               is a ‘perfectly tree‐like data set’, because there is then only one acceptable candidate.
               Using that tree for subsequent population structure or biogeographic analyses is utterly
               unobjectionable; there is a very real sense in which the data are the tree. It is
               unfortunate that we seldom encounter this case in molecular phylogenetic practice.
            </p>
            
            <p>Case (ii) is that where we have more characters than we need for the number of haplotypes
               recovered (L &lt; M + 1). Some of the characters are redundant, and there will be gaps
               in the connecting network. An example would be a data set consisting of haplotypes
               A = (000) and F = (101) in <a href="#f1" class="scrollableLink"></a>Fig. 1a. The 2nd character is monomorphic within the data set, but the 1st and 3rd
               characters are polymorphic; they are also perfectly correlated, being either both
               0 or both 1. What this implies is that either the two characters have experienced
               simultaneous evolutionary substitutions, possibly indicating a recombination event,
               or that a mutationally intermediate haplotype is absent from the data set. That missing
               haplotype may now be extinct, or we may just have missed it in our (usually less than
               exhaustive) sampling. The missing haplotype might be either B or D, and although there
               is some ambiguity of pathway, we can say that the path between A and F requires two
               steps. In the absence of other haplotypes that might confuse the issue, whether we
               invoke B or D does not affect the tree or subsequent analysis. This case occurs most
               often when the number of characters is quite large, relative to the number of individuals
               sampled, i.e. when M &gt; <i>N</i>. This is a common problem, for example, with DNA sequence data from the hypervariable
               region of the mitochondrial D‐<span class="smallCaps">L</span>oop. Finding the optimal tree can be a challenge, because the hypercube is sparsely
               occupied, and the critical information on the links between the observed haplotypes
               is contained in a large number of missing haplotypes. As long as the proper links
               are unambiguous, there is no problem, but difficulty arises wherever there are myriad
               alternative connections that change the nature of the tree. The number of alternative
               trees can be huge, and there is sometimes not much to choose from among very divergent
               candidates. The dictates of good practice suggest that we should avoid informational
               overkill by using a smaller number of cladistically informative characters from the
               outset, but we have limited ability to choose among characters a priori, on the basis
               of their probable cladistic behaviour. The alternative is somehow to remove the homoplasy
               <i>post hoc</i>, and <span><a href="#b48" class="bibLink tab-link" data-tab="pane-pcw-references">48</a></span>) has recently attempted this, in connection with the ‘African Eve’ story. How well
               a <i>post hoc</i> strategy will perform in routine practice is a matter that needs further exploration.
            </p>
            
            <p>Case (iii) is that where there are more haplotypes than can be accounted for by unique
               evolutionary substitutions (L &gt; M + 1), frequently encountered with RFLP haplotypes.
               An example would be a data set consisting of haplotypes A, B, C, D and E from <a href="#f1" class="scrollableLink"></a>Fig. 1d. The A–D link is obvious enough, but there is a closed loop (A–B–E–C–A) that
               implies two substitutions (homoplasy) for either the 1st or 2nd character. We can
               devise four trees, very different, but all equally good. A more complicated example
               is provided by the haplotypic array in <a href="#f1" class="scrollableLink"></a>Fig. 1e, where there are two homoplasies and two closed loops. The array in <a href="#f1" class="scrollableLink"></a>Fig. 1b has five homoplasies. For more characters and haplotypes, the problem can
               become overwhelming. An obvious finesse is to increase the number of characters, but
               the empirical reality is that the homoplasy already present in a data set of M characters
               cannot decrease. My own experience has been that an expanded character set will generally
               contain additional homoplasies. We convert a homoplasic Case (iii) situation into
               an even more homoplasic Case (ii) situation.
            </p>
            
            <p>The point of phylogeographic analysis is to use an unambiguous tree to describe how
               the individuals are connected, and the molecular characters represent evolutionary
               replication. On the other hand, if we are forced to use the operational taxonomic
               units (OTUs) to convert the character arrays into a tree, because the data are phylogenetically
               ambiguous, the individuals become the replication, and we cannot credibly characterize
               more characters than we have individuals. Moreover, there is an element of circularity
               involved in using the OTUs to order character changes into a tree, and then using
               the tree to order the OTUs. We really should not be doing both.
            </p>
            
            <p> <b>Message 2.</b> Using a vast number of characters to make a tree for small numbers of individuals
               is informational overkill. For biogeographic analyses, we would do better to characterize
               more individuals for fewer characters.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss5">
            
            <h3 class="article-section__sub-title section2"> Converting hypercubic data into a tree</h3>
            
            <p>By virtue of employing orthogonal axes and a Manhattan metric, distances between haplotypes
               obey the Pythagorean Theorem, and we conveniently define the squared phenetic distance
               between the <i>i</i>th and <i>j</i>th haplotypes (<i>υ<sub>i</sub></i> and <i>υ<sub>j</sub></i>) as
            </p>
            
            <p> 
               </p><div class="inline-equation"><span class="inline-equation__construct">
                     <!--EMPTY SOURCE--></span><span class="inline-equation__label" id="m1"></span></div> 
            <p></p>
            
            <p>where Δ<i><sub>ij</sub></i> is the number of (0 vs. 1) site‐by‐site differences between the two haplotypes. Our
               distance metric (δ<sup>2</sup><i><sub>ij</sub></i>)<i><sub>p</sub></i> is thus both Manhattan and Euclidean, which is analytically ideal for what follows.
               For example, haplotypes A and B in <a href="#f1" class="scrollableLink"></a>Fig. 1a differ only for the 1st character, so (δ<sup>2</sup><sub>AB</sub>)<i><sub>P</sub></i> = 1; similarly, haplotypes C and H differ for both the 1st and 3rd characters, so
               (δ<sup>2</sup><sub>CH</sub>)<i><sub>P</sub></i> = 2, etc. It is convenient to represent the total collection of squared distances
               between all possible pairs of L haplotypes in the form of an L × L phenetic distance
               matrix, henceforth denoted <b><i>D</i></b><i><sub>p</sub></i> = (δ<sup>2</sup><i><sub>ij</sub></i>)<i><sub>p</sub></i>. The matrix <b><i>D</i></b><i><sub>p</sub></i> makes no assumptions about treeness of the data, being a simple tally of the differences
               between pairs of haplotypes. The data are explicitly embodied in <b><i>D</i></b><i><sub>p</sub></i>, and we should always remember that <b><i>D</i></b><i><sub>p</sub></i> is only as tree‐like as are the raw data.
            </p>
            
            <p>It is not absolutely essential to use a Manhattan or a Euclidean metric, and many
               other choices are available, but the choice of metric is not the issue here. Once
               a metric (describing the degree of divergence) has been chosen to represent the divergence
               between haplotypes, the pairwise interhaplotypic distances contain all the information
               available. Whether we use the raw data or a phenetic distance matrix derived from
               those data, a derivative tree represents an imposition of additional assumptions on
               the data. To the extent that ‘the tree’ is an extrapolation beyond the data, analysis
               of the tree is unavoidably analysis of those extraneous assumptions.
            </p>
            
            <p>So how do we make/choose a tree? For purposes of exposition, I will concentrate on
               the six‐haplotype data set in <a href="#f1" class="scrollableLink"></a>Fig. 1e, which is homoplasic enough to illustrate, yet simple enough to permit exhaustive
               examination. We have six haplotypes and seven connecting (single‐step) edges. To make
               a tree, we have to cut both the (A–B–E–C–A) and (A–B–F–D–A) loops. To do that, we
               must remove two edges, while maintaining connectedness of all six haplotypes. There
               are precisely 15 ways to do that, each yielding a different tree (<a href="#f2" class="scrollableLink">Fig. 2</a>). These trees are represented as straight‐line segments, portrayed in two dimensions,
               but it is important to remember that the trees are actually wending their way through
               five‐dimensional space in city‐block fashion, with each step at right angles to all
               steps that come before and all steps that follow. By careful construction, the interhaplotype
               distances, measured along any particular tree (patristic distances) are both Manhattan
               and Euclidean, facilitating subsequent comparison of the trees with the data and with
               each other.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f2"><a target="_blank" href="#">
                     <!--EMPTY SOURCE--></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 2</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f2&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>
                        
                        <p>The 15 minimum spanning trees (MSTs) that can be extracted from the sextet of haplotypes
                           in <a href="#f1" class="scrollableLink"></a>Fig. 1d, each depicted in two‐dimensional form; the connected sextet from which the
                           trees are derived is shown in the upper‐right corner; the trees are broken into classes,
                           based on topological similarity.
                        </p>
                        
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="faf2" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="faf2" class="accordion__content" style="display: none;">
                           The 15 minimum spanning trees (MSTs) that can be extracted from the sextet of haplotypes
                           in Fig. 1d, each depicted in two‐dimensional form; the connected sextet from which
                           the trees are derived is shown in the upper‐right corner; the trees are broken into
                           classes, based on topological similarity.
                           
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>The trees listed in <a href="#f2" class="scrollableLink">Fig. 2</a> are minimum spanning trees (MSTs). Spanning trees have OTUs (haplotypes) as both
               nodes and branch tips. Steiner trees, with which most of us are more familiar, have
               OTUs only as branch tips, with the nodes representing (now) extinct ancestral OTUs.
               For interspecific work, the ancestral intermediates are commonly absent from the data
               set, and Steiner trees capture the essence of the situation. For intraspecific work,
               the intermediates are often still present, and our trees resemble strawberry plants,
               spreading across the landscape as an articulating network; MSTs have their attractions.
               I will use MSTs, because they are sufficient to illustrate the point, and because
               the critical messages do not depend on the particular tree‐building algorithm in any
               event. It is possible to compute the exact number of competing MSTs from such a molecular
               data set, and (in principle) to delineate them, using Kirchoff methods (<span><a href="#b32" class="bibLink tab-link" data-tab="pane-pcw-references">32</a></span>; <span><a href="#b15" class="bibLink tab-link" data-tab="pane-pcw-references">15</a></span>). Recall that the haplotypes shown in <a href="#f1" class="scrollableLink"></a>Fig. 1c yield only one MST, while those shown in <a href="#f1" class="scrollableLink"></a>Fig. 1d yield four, and those shown in <a href="#f1" class="scrollableLink"></a>Fig. 1e yield 15. Those shown in <a href="#f1" class="scrollableLink"></a>Fig. 1b yield 384 MSTs; the greater the degree of homoplasy, the greater the number
               of MSTs. If the degree of homoplasy is substantial (L &gt; M), the number of MSTs can
               become vast. As an empirical illustration, <span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>) reported L = 56 haplotypes and M = 34 RFLP sites in a collection of human mtDNA
               genotypes. There were in excess of 10<sup>9</sup> (equally) maximum parsimony MSTs. Those trees were topologically diverse and, at
               least in terms of total tree length, there was nothing to choose among them. I shall
               have more to say about choosing among equally parsimonious trees later.
            </p>
            
            <p>A homoplasy is created wherever the same character has changed more than once, yielding
               parallel edges for our M‐dimensional hypercube. Breaking a loop is tantamount to transforming
               one such character, mutating twice, into a pair of independent characters, changing
               once each. That amounts to adding orthogonal (right‐angle) axes to our character space,
               one per homoplasy. Making a tree is thus tantamount to imposing additional dimensions
               on the data. If there are <i>m</i> = L – (M + 1) homoplasies in the data set, we have to add <i>m</i> dimensions to our character set. Those extra dimensions were a real part of the evolutionary
               process, but their identities have long since become obscured. All we have are the
               phenetic data. Any tree we construct represents an extrapolation from the data. Derivative
               analyses are, by default, analyses of that extrapolation. There are multiple sets
               of <i>m</i> loops we can cut, sometimes a huge number, and subsequent analyses depend on the
               tree chosen. If we insist on analysing a tree, we cannot escape the questions of which
               extrapolations we choose to analyse, and why we have chosen that set?
            </p>
            
            <p> <b>Message 3.</b> Converting the phenetic data into a tree represents an extrapolation from the data;
               alternative trees are alternative extrapolations. Analysis of any tree is an analysis
               of the corresponding extrapolation.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss6">
            
            <h3 class="article-section__sub-title section2"> Choosing among equally parsimonious trees</h3>
            
            <p>Setting aside, for the moment, the question of whether we should even be using a tree
               for subsequent analysis, it has to be said that some of these equal‐length trees are
               better than others. Return to the illustration (<a href="#f2" class="scrollableLink">Fig. 2</a>). Each of the possible trees yields a derivative patristic distance matrix. A tree
               and its patristic distance matrix represent a 1<b>:</b>1 mapping. Each of these matrices has <i>M</i> dimensions, where <i>M</i> is the number of dimensions in our augmented character space (<i>M</i> = M + <i>m</i>), defined by breaking the <i>m</i> loops. A patristic distance matrix defines and is defined by its tree; the augmented
               data are perfectly (and artificially) tree‐like. By inserting an additional orthogonal
               axis for each broken loop, we preserve the Manhattan/ Euclidean geometry of our patristic
               distance metric during tree construction. Three example MSTs (one from each topological
               class) and their matching patristic distance matrices (<b><i>D</i></b><i><sub>1</sub></i>, <b><i>D</i></b><i><sub>3</sub></i> and <b><i>D</i></b><i><sub>10</sub></i>) are presented in <a href="#t1" class="scrollableLink">Table 1</a>, for contrast with the phenetic distance matrix, <b><i>D</i></b><i><sub>p</sub></i>.
            </p>
            
            <div class="article-table-content" id="t1">
               <header class="article-table-caption"><span class="table-caption__label">Table 1.
                     </span>The phenetic data of <a href="#f2" class="scrollableLink">Fig. 2</a>, and an example trio of minimum spanning trees that can be extracted from those same
                  data, their patristic distance matrices, their sums of squares, Q<i><sub>t</sub></i>, and their cophenetic correlations with the phenetic distance matrix
               </header>
               
               
               <img class="table__image" src="">
               
            </div>
            
            <p>As total tree length (L – 1) is not a discriminating criterion among MSTs, we must
               invoke a 2nd criterion if we wish to narrow the search any further. It is possible
               to compare competing MSTs by means of the sum of patristic distances between each
               of the L (L – 1)/2 pairs of haplotypes, divided by the total number of those haplotypes
            </p>
            
            <p> 
               </p><div class="inline-equation"><span class="inline-equation__construct">
                     <!--EMPTY SOURCE--></span><span class="inline-equation__label" id="m2"></span></div> 
            <p></p>
            
            <p>where the subscripts <i>i</i> and <i>j</i> index the different haplotypes. Basically, we sum the elements of the patristic distance
               matrix, below the diagonal, and divide by the total number of haplotypes (L = 6 in
               this case). While all MSTs are of the same total length (L – 1), they vary considerably
               in their <i>Q<sub>t</sub></i> values.
            </p>
            
            <p>It develops that <i>Q</i><sub>t</sub> is also the sum of squared deviations of the OTUs from the centroid of the <i>M</i>‐dimensional augmented character space containing the <i>t</i>th tree, which will be useful when we return to population structure analysis. If
               we divide <i>Q<sub>t</sub></i> by its degrees of freedom (L – 1), we convert the sum of squared deviations into
               a corresponding variance. To be specific,
            </p>
            
            <p> 
               </p><div class="inline-equation"><span class="inline-equation__construct">
                     <!--EMPTY SOURCE--></span><span class="inline-equation__label" id="m3"></span></div> 
            <p></p>
            
            <p> <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>) have suggested ranking trees on the basis of Var(<b><i>D</i></b><i><sub>t</sub></i>) or (equivalently) <i>Q<sub>t</sub></i> and choosing that tree with the minimum value, an optimization strategy they labelled
               molecular variance parsimony. The sum of patristic distances between all pairs of
               OTUs serves as a powerful 2nd level sorting criterion.
            </p>
            
            <p>For the trees in <a href="#t1" class="scrollableLink">Table 1</a>, <i>Q<sub>1</sub></i> = 4.833, <i>Q<sub>3</sub></i> = 5.167 and <i>Q<sub>10</sub></i> = 5.833. Note that the sum of squares for the phenetic distance matrix, representing
               the raw data, is <i>Q<sub>p</sub></i> = 4.167. The patristic distance between the <i>i</i>th and <i>j</i>th haplotypes, measured along any particular tree, cannot be shorter than the phenetic
               distance between them, because broken loops lengthen some of the patristic distances.
               Hence (δ<sup>2</sup><i><sub>ij</sub></i>)<i><sub>P</sub></i>≤ (δ<sup>2</sup><i><sub>ij</sub></i>)<i><sub>t</sub></i> for all pairs of haplotypes (<i>i</i> and <i>j</i>), and it follows that <i>Q<sub>p</sub></i>≤<i>Q<sub>t</sub></i> for all trees (<i>t</i> = 1, …, T). The MST which least increases the variation among haplotypes is that
               tree which represents minimal extrapolation from the raw data. As a practical matter,
               the trees with smallest <i>Q<sub>t</sub></i> are topologically the most tightly packed (most articulated).
            </p>
            
            <p>It is also possible to compare the details of each tree (<i>t</i> = 1, …, T) with the raw data, in point by point fashion, by computing the cophenetic
               correlation (<i>r<sub>pt</sub></i>) between <b><i>D</i></b><i><sub>p</sub></i> and each of the <b><i>D</i></b><i><sub>t</sub></i> values (<span><a href="#b38" class="bibLink tab-link" data-tab="pane-pcw-references">38</a></span>; <span><a href="#b39" class="bibLink tab-link" data-tab="pane-pcw-references">39</a></span>; <span><a href="#b4" class="bibLink tab-link" data-tab="pane-pcw-references">4</a></span>; <span><a href="#b37" class="bibLink tab-link" data-tab="pane-pcw-references">37</a></span>). The computed values for the three example trees are presented in <a href="#t1" class="scrollableLink">Table 1</a>. <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>) have shown that trees with small <i>Q<sub>t</sub></i> values are highly correlated with the data, whereas trees with larger <i>Q<sub>t</sub></i> values are generally less so. All else being equal, trees with small <i>Q<sub>t</sub></i> are preferable, and among a set of equally parsimonious MSTs. In fact, we can use
               the cophenetic correlation (<i>r<sub>pt</sub></i>) as a gauge of the ‘inherent treeness’ of the data set. None of the example trees
               in <a href="#t1" class="scrollableLink">Table 1</a> is credibly tree‐like, because <i>r<sub>pt</sub></i> &lt; 0.58 for the best of them, a consequence of the inherent homoplasy in the example
               data set. A practitioner who finds a tree with <i>r<sub>pt</sub></i> &gt; 0.90 or 0.95 could be reasonably confident that the data were reliably tree‐like
               (nonhomoplasic). Any tree with <i>r<sub>pt</sub></i> &lt; 0.90 should probably be viewed with a degree of healthy skepticism.
            </p>
            
            <p>There is no guarantee that we have the ‘true tree’, even if <i>Q<sub>t</sub></i> is almost as small as <i>Q<sub>p</sub></i> and <i>r<sub>pt</sub></i> is close to unity. A close fit of model (tree) to data should not be interpreted
               as phylogenetic truth. On the other hand, phylogenetic truth is generally unknowable,
               and a close fit to the data is about the best we can do; most of us will accept it.
               Other optimality criteria could be used, of course, and other tree‐making algorithms
               employ them, but the details of alternative optimization algorithms are not the issue
               here. The central idea with all of the available algorithms is to choose that tree
               which is as closest to the data, given the constraints imposed by the algorithm/model.
            </p>
            
            <p> <b>Message 4.</b> We want a tree that is as true to the data as we can manage. Thus, good trees are
               those that mimic the phenetic data closely, whereas bad trees are those that do not
               conform closely to the data.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss7">
            
            <h3 class="article-section__sub-title section2"> Frequency weighting – a 3rd level criterion</h3>
            
            <p>The use of <i>Q<sub>t</sub></i> provides us with a telling 2nd level criterion for choosing among MST topologies,
               but any given topological type can be represented by different trees. To illustrate,
               our simple example (<a href="#f2" class="scrollableLink">Fig. 2</a>) exhibits three different topological classes: (a) Tree 1, (b) Trees 2–7, and (c)
               Trees 8–15. Within a topological class, all trees have the same <i>Q<sub>t</sub></i> value. It would be useful to have a 3rd level sorting criterion to distinguish among
               topological ties. <span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>), <span><a href="#b8" class="bibLink tab-link" data-tab="pane-pcw-references">8</a></span>), <span><a href="#b10" class="bibLink tab-link" data-tab="pane-pcw-references">10</a></span>), and <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>) have all argued that the frequencies of different haplotypes provide useful information
               about tree structure. The basic tenet, traceable to <span><a href="#b46" class="bibLink tab-link" data-tab="pane-pcw-references">46</a></span>), is that (under neutrality), the most frequent haplotypes in the population are
               probably the oldest, everything else being equal. High‐frequency haplotypes have probably
               been present for a long time, having had ample time to achieve substantial copy numbers.
               New variants are probably derived from the more common haplotypes and, because they
               are new, will be rarer. It is possible to imagine a number of scenarios for which
               this pattern will not hold (e.g. <span><a href="#b34" class="bibLink tab-link" data-tab="pane-pcw-references">34</a></span>), but as a first approximation, the two assertions imply that the most frequent haplotypes
               should cluster relatively close together, near the centroid of the phylogenetic space,
               whereas the rarer variants should occupy the outer fringes of the tree. Those expectations
               favour a frequency‐weighted <i>Q</i>‐criterion, as even those trees sharing similar topologies will have different distributions
               of mass.
            </p>
            
            <p>For a sample of <i>N</i> individuals in the study, exhibiting L different haplotypes, we have <i>n<sub>i</sub></i> replicates of the <i>i</i>th haplotype. We can either increase the dimension of the distance matrices to <i>N</i>×<i>N</i>, reporting the genetic distance between each pair of individuals, or (equivalently)
               we can leave the (L × L) interhaplotypic distance matrices as is, but weight the (δ<sup>2</sup><i><sub>ij</sub></i>)<i><sub>p</sub></i> and (δ<sup>2</sup><i><sub>ij</sub></i>)<i><sub>t</sub></i> by (<i>n<sub>i</sub> n<sub>j</sub></i>), yielding a weighted version of <i>Q</i>, denoted
            </p>
            
            <p> 
               </p><div class="inline-equation"><span class="inline-equation__construct">
                     <!--EMPTY SOURCE--></span><span class="inline-equation__label" id="m4"></span></div> 
            <p></p>
            
            <p>for the phenetic distance matrix, <b>D</b><i><sub>p</sub></i>, and by
            </p>
            
            <p> 
               </p><div class="inline-equation"><span class="inline-equation__construct">
                     <!--EMPTY SOURCE--></span><span class="inline-equation__label" id="m5"></span></div> 
            <p></p>
            
            <p>for the <i>t</i>th patristic distance matrix, <b><i>D</i></b><i><sub>t</sub></i>. In other words, we compute the sum of the elements below the diagonal of the <i>N</i>×<i>N</i> interindividual distance matrix, and divide by <i>N</i>. Any MST that connects the more common haplotypes has an Θ<i><sub>t</sub></i> advantage. By virtue of that same frequency weighting, the placement of rare haplotypes
               has relatively little impact on Θ<i><sub>t</sub></i>. Good trees generally have the common haplotypes near the centre of mass, where they
               define the central branching order, with the rare haplotypes being relegated to the
               periphery.
            </p>
            
            <p>Again to illustrate, imagine that we had sampled <i>N</i> = 100 individuals, which had replicate numbers for the six haplotypes of <i>n</i><sub>A</sub> = 40, <i>n</i><sub>B</sub> = 30, <i>n</i><sub>C</sub> = 15, <i>n</i><sub>D</sub> = 2, <i>n</i><sub>E</sub> = 3 and <i>n</i><sub>F</sub> = 10. Each of the trees in <a href="#f2" class="scrollableLink">Fig. 2</a> has a different Θ<i><sub>t</sub></i> value, listed as Θ<i><sub>t</sub></i> (total) in <a href="#t2" class="scrollableLink">Table 2</a>. It develops that most of the Class III trees (trees 10–15) disconnect haplotypes
               A and B, and have severely inflated Θ<i><sub>t</sub></i> values as a consequence. The Class II trees are better, in general, but those candidates
               that disconnect either haplotypes A and C or haplotypes B and F (trees 2, 4, 6 and
               7) have modestly elevated Θ<i><sub>t</sub></i> values. Tree 1 remains the best, with tree 5 and tree 3 close behind; even tree 9,
               from Class III, is a credible contender, because it preserves the most critical links
               (C–A–B–F). All other trees are poorer contenders, to one degree or another.
            </p>
            
            <div class="article-table-content" id="t2">
               <header class="article-table-caption"><span class="table-caption__label">Table 2.
                     </span><span class="smallCaps">AMOVA</span> for the 15 candidate trees of <a href="#f2" class="scrollableLink">Fig. 2</a>, and a comparison with the phenetic data, when haplotypic replication is unequal
                  (<i>n</i><sub>A</sub> = 40, <i>n</i><sub>B</sub> = 30, <i>n</i><sub>C</sub> = 15, <i>n</i><sub>D</sub> = 2, <i>n</i><sub>E</sub> = 3 and <i>n</i><sub>F</sub> = 10)
               </header>
               
               
               <img class="table__image" src="">
               
            </div>
            
            <p> <span><a href="#b19" class="bibLink tab-link" data-tab="pane-pcw-references">19</a></span>) have shown, at least under some circumstances, that using haplotype frequency information
               can provide more power for detecting population subdivision. It is important to note
               here that we have assumed something nontrivial about the evolutionary process in order
               to fine‐tune our choice of tree, and the refinement is only as reliable as that assumption.
               Frequency should thus only be used as a 3rd level criterion, but it does narrow the
               range of tree choices dramatically, and it certainly helps us distinguish between
               different members of the same topological class. It is also completely compatible
               with the population and biogeographic analyses that motivate the larger enterprise,
               and to which I will turn next. If we are to use frequency weighting, however, it is
               absolutely imperative that we refrain from sampling individuals differentially, based
               on their genotypes. We need a well‐balanced and unbiased sample of the variation that
               is present in our <i>P</i> populations.
            </p>
            
            <p>Before leaving the question of how to make and compare trees, one final comment is
               in order. There are many optimization criteria other than total tree length (L), the
               sum of pairwise distances between haplotypes (<i>Q<sub>t</sub></i>), or even the weighted sum of pairwise distances (Θ<i><sub>t</sub></i>), and my choices above are not intended to be proscriptive. Each of the standard
               tree‐making algorithms in common usage has its own particular criterion, and each
               algorithm makes assumptions about the nature of the evolutionary process, assumptions
               that we can check only poorly. In practice, we construct the best tree we can from
               the data, using whatever criterion is explicit (or implicit) in our favourite tree‐making
               algorithm, and we then proceed with the population structure or biogeographic analyses
               of interest. Any inference we choose to draw is contingent on the assumptions we have
               made. The important point is not so much the choice of that criterion as it is the
               preference we have for a tree that mimics the data as closely as possible, given the
               constraints.
            </p>
            
            <p> <b>Message 5.</b> If we are going to make a tree from phenetic data, especially if we are going to
               use it for subsequent population or biogeographic analysis, we want a tree that is
               as ‘true to the data’ as we can make it.
            </p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="ss8">
         
         <h2 class="article-section__title section__title section1" id="ss8-title"> Analysing biogeography</h2>
         
         <div class="article-section__sub-content" id="ss9">
            
            <h3 class="article-section__sub-title section2"> Phenetic or patristic distances?</h3>
            
            <p>At this juncture, it is timely to remind ourselves that our initial motivation for
               making a tree was to facilitate a geographical analysis of the pattern of variation
               within the species, couched in overtly evolutionary terms. Having discovered that
               there are competing trees we can use, and having chosen one, we move on to biogeographic
               analysis. There are many multivariate methods we can use, in pursuit of biogeographic
               pattern, but the choice of method is not the issue here. Suffice that virtually all
               of these analyses can be accessed either from the raw character data on the <i>N</i> individuals, or equivalently from an <i>N</i>×<i>N</i> distance matrix among those <i>N</i> individuals. The question of larger interest is whether we should analyse the phenetic
               data themselves, embedded in the phenetic distance matrix (<b><i>D</i></b><i><sub>p</sub></i>), or whether we should analyse the tree, represented by a patristic distance matrix
               (<b><i>D</i></b><i><sub>t</sub></i>)? I will pursue that question in the context of population structure analysis, with
               which my colleagues and I have some experience, but the answer is fairly general.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss10">
            
            <h3 class="article-section__sub-title section2"> Population structure analysis</h3>
            
            <p>The first task is to determine the degree to which populations have different haplotypes
               or different frequencies of shared haplotypes, a problem conveniently pursued with
               the use of a molecular analysis of variance (<span class="smallCaps">AMOVA</span>). <span class="smallCaps">AMOVA</span>, which is a generic analogue of analysis of variance, begins with an <i>N</i>×<i>N</i> interindividual distance matrix, particularly convenient for molecular data (<span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>). <span class="smallCaps">AMOVA</span> has been used for inversion polymorphisms (<span><a href="#b12" class="bibLink tab-link" data-tab="pane-pcw-references">12</a></span>), for multiallelic codominant allozyme data (<span><a href="#b31" class="bibLink tab-link" data-tab="pane-pcw-references">31</a></span>), for multilocus dominant/recessive RAPD data (<span><a href="#b20" class="bibLink tab-link" data-tab="pane-pcw-references">20</a></span>), for haplotypic RFLP data (<span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>), and it can easily be extended to microsatellites (<span><a href="#b28" class="bibLink tab-link" data-tab="pane-pcw-references">28</a></span>) and multistate sequence data. It can be used for just about any type of genetic
               data imaginable. The basic idea is to partition the variation among individuals, Θ<i><sub>t</sub></i> (total), into separate components for variation within and divergence among populations.
               We can easily extend the model to multiple nesting levels, should we have several
               layers of subdivision (e.g. <span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>; <span><a href="#b20" class="bibLink tab-link" data-tab="pane-pcw-references">20</a></span>; <span><a href="#b11" class="bibLink tab-link" data-tab="pane-pcw-references">11</a></span>; <span><a href="#b31" class="bibLink tab-link" data-tab="pane-pcw-references">31</a></span>). We can even use cross‐classified analysis where individuals can be stratified in
               more than one way (<span><a href="#b6" class="bibLink tab-link" data-tab="pane-pcw-references">6</a></span>).
            </p>
            
            <p>For our immediate purposes, it is sufficient to compute the sums of squares of interindividual
               distances, Θ<i><sub>t</sub></i> (total), for a collection of <i>N</i> individuals in <i>P</i> populations, and partition it into within‐population and among‐population components.
               Using the classic <span class="smallCaps">ANOVA</span>‐type extraction procedures, we estimate the corresponding variance components, and
               compute the fraction of the total variance accounted for by divergence among populations,
               defined as
            </p>
            
            <p> 
               </p><div class="inline-equation"><span class="inline-equation__construct">
                     <!--EMPTY SOURCE--></span><span class="inline-equation__label" id="m6"></span></div> 
            <p></p>
            
            <p>a haplotypic analogue of the <i>F</i> statistic used to describe population structure for allozyme loci (<span><a href="#b47" class="bibLink tab-link" data-tab="pane-pcw-references">47</a></span>). We can treat the haplotypes as simply different, as for multiallelic allozymes
               (<span><a href="#b31" class="bibLink tab-link" data-tab="pane-pcw-references">31</a></span>), or we can use phenetic distances (e.g. <span><a href="#b20" class="bibLink tab-link" data-tab="pane-pcw-references">20</a></span>; <span><a href="#b11" class="bibLink tab-link" data-tab="pane-pcw-references">11</a></span>; <span><a href="#b6" class="bibLink tab-link" data-tab="pane-pcw-references">6</a></span>). We can even use the patristic distances for any specified tree (<span><a href="#b14" class="bibLink tab-link" data-tab="pane-pcw-references">14</a></span>; <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>). I will use <span class="smallCaps">AMOVA</span> to compare the population structure analysis derivable from the phenetic data with
               that obtained by using each of the phylogenetic trees.
            </p>
            
            <p>This is best illustrated with an extension of our contrived example. Earlier, I listed
               a set of unequal replicate numbers for the six haplotypes. Imagine that the <i>N</i> = 100 individuals were actually sampled from <i>P</i> = 5 populations (20 individuals each), with the observed haplotypic replicate numbers
               shown in <a href="#f3" class="scrollableLink">Fig. 3</a>. The five sampled populations are drawn from two ill‐defined subspecies, populations
               (a) and (b) from subspecies I, populations (d) and (e) from subspecies II, and population
               (c) from a bridging intermediate type. Our suspicion is that there has been enough
               gene flow between the two subspecies, through the intermediate type, to blur the edges,
               and we hope to use molecular data to shed some evolutionary light on the taxonomic
               situation.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f3"><a target="_blank" href="#">
                     <!--EMPTY SOURCE--></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 3</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f3&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>
                        
                        <p>Schematic representation of the population distribution of the example sextet of haplotypes
                           in <a href="#f1" class="scrollableLink"></a>Fig. 1e, with sample haplotype replicate numbers attached; populations (a) and (b)
                           are found in subspecies I; populations (d) and (e) are found in subspecies II; population
                           (c) represents an intermediate type.
                        </p>
                        
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="faf3" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="faf3" class="accordion__content" style="display: none;">
                           Schematic representation of the population distribution of the example sextet of haplotypes
                           in Fig. 1e, with sample haplotype replicate numbers attached; populations (a) and
                           (b) are found in subspecies I; populations (d) and (e) are found in subspecies II;
                           population (c) represents an intermediate type.
                           
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>For the phenetic distance matrix, and for each of the 15 separate patristic distance
               matrices, we conducted a singly nested <span class="smallCaps">AMOVA</span>; the results are tallied in <a href="#t2" class="scrollableLink">Table 2</a>. As pointed out earlier, Θ<i><sub>t</sub></i> (total) – which ignores the geographical pattern, favours tree 1, but trees 5 and
               3 are also strong contenders, as is tree 9. The Θ<i><sub>t</sub></i> (within)‐values yield a slightly different order, with trees 5 and 9 both better
               than trees 1 and 3. A close examination of <a href="#f3" class="scrollableLink">Fig. 3</a> will show that all four trees share the C–A–B–F pathway, both within populations
               and among them, but that trees 5 and 9 provide slightly clearer connections to the
               peripheral haplotypes D and E. On the other hand, there is nothing much to choose
               among these trees, and the phenetic data tell the essential story. The Φ<i><sub>st</sub></i> criteria are erratic, changing substantially with small changes in the tree. Tree
               9 had the largest value (Φ<i><sub>st</sub></i> = 0.233) and tree 1 the smallest (Φ<i><sub>st</sub></i> = 0.150), among the four trees, with the phenetic data in between (Φ<i><sub>st</sub></i> = 0.184). We should probably not use Φ<i><sub>st</sub></i> to choose a tree (<span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>), and even the results with Θ<i><sub>t</sub></i> (total) and Θ<i><sub>t</sub></i> (within) do not yield a compelling choice. None of these good trees creates a problem
               with the population structure analysis, but we might as well just let the data speak
               for themselves! What the data say is that the two subspecies are significantly divergent,
               with subspecies I favouring the A–C–E–B portion of the haplotype space, and subspecies
               II favouring the A–B–F–D portion of the haplotype space, although having nontrivial
               frequencies of haplotype C as well. The major difference is the frequency split between
               haplotypes A and B. The intermediate type seems to be a blend, possibly caused by
               gene flow, with haplotypes C, A, B and F being the only ones recovered. In terms of
               a possible tree, only the C–A–B–F path is compatible with all the evidence, and it
               is compelling. The precise connections of haplotypes E and D are difficult to discern,
               and the four alternative choices represent the alternative trees 1, 3, 5 and 9.
            </p>
            
            <p> <b>Message 6.</b> For biogeographic analysis: (i) good trees are harmless, but they do not actually
               help much; (ii) bad trees mimic the data poorly and should be avoided; and (iii) the
               most reliable inference is that present in the phenetic data themselves.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss11">
            
            <h3 class="article-section__sub-title section2"> Support from the literature</h3>
            
            <p>These results are entirely in accord with those obtained by <span><a href="#b13" class="bibLink tab-link" data-tab="pane-pcw-references">13</a></span>). Working with human mtDNA from <i>N</i> = 672 people from <i>P</i> = 10 populations, nested within five regional groups, a set of five restriction enzymes
               uncovered M = 34 RFLPs, defining L = 56 distinguishable haplotypes. As I mentioned
               earlier, there were in excess of 10<sup>9</sup> MSTs, all of the same minimal length. It was not possible to examine all 10<sup>9</sup> competing MSTs, but we did examine a random sample of 10 000 with <span class="smallCaps">AMOVA</span>. Phenetic treatment indicated substantial population structure, measured both as
               differences in haplotype frequencies and partially nonoverlapping arrays of haplotypes
               in different populations. Good trees yielded virtually the same inference as the phenetic
               data. Bad trees had large Θ<i><sub>t</sub></i>‐values, both within and among populations, and they are such poor matches for the
               phenetic data that their inference was unreliable. The Φ<i><sub>st</sub></i> criterion was extremely sensitive to small changes in the choice of tree, and it
               was best viewed as consequential analytical output, rather than as tree‐defining input.
            </p>
            
            <p>Even in those cases where a phenetic treatment of biogeographic pattern is productive,
               a patristic treatment may yield no meaningful improvement. That seems surprising at
               first, but a little reflection shows that it makes sense. To the extent that the data
               are inherently tree‐like, phyletic information is explicitly embedded in the raw data
               and will be automatically incorporated into the phenetic distance matrix. Having expressed
               a preference for trees that faithfully mimic the phenetic data, it should come as
               no surprise that such trees yield virtually the same inference as the data themselves.
               In short, the inference that can be trusted the most is that derived directly from
               the phenetic data. A good tree will probably do no damage, but it will probably not
               help much, and a bad tree cannot be trusted at all.
            </p>
            
            <p>Any inference drawn from a particular tree that is strongly divergent from the inference
               available from the phenetic data should be ‘cause for pause’. We either have a ‘bad
               tree’ or the extraneous information (the evolutionary assumptions inherent in our
               choice of algorithm) are driving the analysis, notwithstanding phenetic data that
               tell a different story. It is possible to encounter situations like this, but the
               investigator should be very confident of the assumptions before overruling the data.
            </p>
            
            <p>I have said little about the difficulty of actually discerning an optimal tree. My
               example was contrived to be small. There were only 15 candidate trees, and it was
               possible to examine them all. For the human mtDNA example just above, 10 000 random
               MSTs was still less than one in 100 000 of those available. There were almost surely
               better trees than the best we found, but there can be no guarantee that we will ever
               find the best tree. All tree‐making algorithms are subject to the same dilemma, in
               the face of phenetic data that are phylogenetically ambiguous to any considerable
               extent. The number of tree choices is beyond our ability to enumerate, much less to
               analyse. We can find a good tree, but we will never know if we have found the best.
               In view of the subsequent inference limitations for our biogeographic survey, even
               were we able to identify an optimal tree, the entire exercise may be rather pointless.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss12">
            
            <h3 class="article-section__sub-title section2"> Can we use the geography to choose a tree?</h3>
            
            <p>If we cannot use a tree to say something incremental about biogeographic pattern,
               can we at least use biogeographic pattern to say something incremental about a tree?
               Imagine a situation where the phenetic treatment yields convincing evidence of biogeographic
               structure or pattern, in spite of some homoplasic ambiguity in the connection pattern
               of peripheral haplotypes. Just to illustrate, return to the geographical array shown
               in <a href="#f3" class="scrollableLink">Fig. 3</a>. I have already commented that trees 1, 3, 5 and 9 are all serious contenders, but
               that there is not much to choose among them. A slightly different array of population
               samples could tip the balance in favour of one or the other, which says that we are
               relying on the nuances of how the rare haplotypes D and E are connected, from population
               to population, rather than on the strong phyletic signal inherent in the C–A–B–F pathway,
               which is already obvious from the phenetic data. To infer anything more is ‘reaching’.
               In general, we should expect those features of the data set that are inherently tree‐like
               to emerge naturally from the analysis, but they will already be obvious in the phenetic
               data. It is possible that one might encounter a biogeographic pattern that would provide
               compelling support for a particular tree, among a modest set of very good trees, but
               as good trees all mimic the phenetic data closely, I am not sanguine about the prospects
               for fine‐tuning. A little gene flow and a little sampling variation go a long way
               toward confusing the issue. In closing this section, it is imperative to offer a cautionary
               note. If we insist on using biogeographic pattern to choose a tree, we are not also
               entitled to use that derived tree for biogeographic analysis. To do so would be to
               assume the conclusion, because the argument is circular.
            </p>
            
            <p> <b>Message 7.</b> We might be able to use a good tree to say something incremental about biogeographic
               pattern. We might be able to use biogeographic pattern to say something incremental
               about the choice of tree. We cannot legitimately do both.
            </p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="ss13">
         
         <h2 class="article-section__title section__title section1" id="ss13-title"> A tree is a picture</h2>
         
         <p>Multivariate statistical analysis of a voluminous biogeographic data set is definitely
            the way to proceed, but none of us thinks well in hyperdimensional space. A picture
            is worth a 1000 words (or molecular markers), and whatever a tree is not, it is certainly
            a picture. When all is said and done, we will need a picture to convey the story,
            so why not use a tree? Several comments are in order. First, there are myriad pictorial
            methods available for hyperdimensional data, most of which perform quite nicely without
            any assumptions about phyletic radiation. I have commented above on the fact that
            reticulation can be an important source of evolutionary pattern, and (mutational homoplasy
            aside) a strictly radiating tree can actually be misleading. Second, for the analysis
            of biogeographic pattern, it is obvious that ordination analyses, spatial autocorrelation
            analyses, and cluster analyses all have much to offer, and they all yield a picture.
            They all begin with either the raw data or the phenetic distance matrix, and the philosophical
            and practical difficulties inherent in using a tree instead are the same as those
            we have encountered with <span class="smallCaps">AMOVA</span>. Third, most multivariate methods allow a dimensional reduction of the full data
            set, stressing the central tendencies and suppressing the exquisite peripheral detail.
            By contrast, phylogenetic trees actually increase dimensionality, beyond what is explicit
            in the data. While we can draw a tree in two dimensions, we should never forget that
            it is wandering through a hyperspace of higher dimension than are the data themselves.
            We are (to some extent) analysing/depicting nondata. Fourth, we need to distinguish
            between a tree used as a pictorial representation of an analytical result, and a tree
            used as an object of formal analysis. I am not averse to artful display, but we need
            to recognize it for what it is. A tree is one of several useful ways to draw a picture,
            and I have no objections, but my preference would be to analyse the data.
         </p>
         
      </div>
      
      <div class="article-section__content" id="ss14-1">
         
         <h2 class="article-section__title section__title section1" id="ss14-1-title"> Acknowledgements</h2>
         
         <p>I am especially grateful to David Gordon, Rod Peakall, and three reviewers for voluminous
            commentary on the manuscript. Any remaining errors of omission or commission remain
            my own responsibility. This work was undertaken while I was a sabbatical visitor at
            the Division of Botany and Zoology, Australian National University, and I wish to
            thank the faculty, students and staff at ANU for their unfailing collegiality during
            my visit. The work was funded by the New Jersey Agricultural Experiment Station, NJAES‐32106/USDA.
         </p>
         
      </div>
      
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion"><a href="#" class="accordion__control" aria-controls="idss15-1" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;pbContext=%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.1998.7.issue-4%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Barticle%3Aarticle%3Adoi%5C%3A10.1046%2Fj.1365-294x.1998.00370.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;onlyLog=true"><span class="section__title" id="references-section-1">References</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;" id="ss15-1">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="b1"><span class="bullet">1</span>
                     Avise JC (<span class="pubYear">1989</span>) <span class="articleTitle">Gene trees and organismal histories: a phylogenetic approach to population biology</span>. 
                     <i>Evolution</i>, <span class="vol">43</span>, 1192–1208.
                     <div class="extra-links" id="e_1_2_6_2_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.1989.tb02568.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=28564500" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1989AN91500004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Evolution&amp;publication_year=1989&amp;pages=%0AAvise+JC+%281989%29+Gene+trees+and+organismal+histories%3A+a+phylogenetic+approach+to+population+biology.+%0AEvolution%2C+43%2C+1192%E2%80%931208.&amp;journal=1192-1208.&amp;issue=43&amp;author=JC+Avise&amp;title=Gene+trees+and+organismal+histories%3A+a+phylogenetic+approach+to+population+biology&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b2"><span class="bullet">2</span>
                     Avise JC, 
                     Arnold J, 
                     Ball RM
                     <i>et al.</i>  (<span class="pubYear">1987</span>) <span class="articleTitle">Intraspecific phylogeography: the mitochondrial DNA bridge between population genetics
                        and systematics</span>. 
                     <i>Annual Review of Ecology and Systematics</i>, <span class="vol">18</span>, 489–522.
                     <div class="extra-links" id="e_1_2_6_3_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2005.02655.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVGltLnO" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=16101761" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1987K958800020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Annual+Review+of+Ecology+and+Systematics&amp;publication_year=1987&amp;pages=%0AAvise+JC%2C+%0AArnold+J%2C+%0ABall+RM%0A++++++++++++et+al.++%281987%29+Intraspecific+phylogeography%3A+the+mitochondrial+DNA+bridge+between+population+genetics+and+systematics.+%0AAnnual+Review+of+Ecology+and+Systematics%2C+18%2C+489%E2%80%93522.&amp;journal=489-522.&amp;issue=18&amp;author=JC+Aviseauthor=J+Arnoldauthor=RM+Ball&amp;title=Intraspecific+phylogeography%3A+the+mitochondrial+DNA+bridge+between+population+genetics+and+systematics&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b3"><span class="bullet">3</span>
                     Bandelt H‐J, 
                     Forster P, 
                     Sykes BC, 
                     Richards MB (<span class="pubYear">1995</span>) <span class="articleTitle">Mitochondrial portraits of human populations using median networks</span>. 
                     <i>Genetics</i>, <span class="vol">141</span>, 743–755.
                     <div class="extra-links" id="e_1_2_6_4_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK287gvFKitQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=8647407" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1995RX01900026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1995&amp;pages=%0ABandelt+H%E2%80%90J%2C+%0AForster+P%2C+%0ASykes+BC%2C+%0ARichards+MB+%281995%29+Mitochondrial+portraits+of+human+populations+using+median+networks.+%0AGenetics%2C+141%2C+743%E2%80%93755.&amp;journal=743-755.&amp;issue=141&amp;author=H%E2%80%90J+Bandeltauthor=P+Forsterauthor=BC+Sykesauthor=MB+Richards&amp;title=Mitochondrial+portraits+of+human+populations+using+median+networks&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b4"><span class="bullet">4</span>
                     Barrantes R, 
                     Smouse PE, 
                     Mohrenweiser HW
                     <i>et al.</i>  (<span class="pubYear">1990</span>) <span class="articleTitle">Microevolution in lower Central America. I. Genetic characterization of the Chibcha‐speaking
                        groups of Costa Rica and Panama, and a taxonomy based on genetics, linguistics, and
                        geography</span>. 
                     <i>American Journal of Human Genetics</i>, <span class="vol">46</span>, 63–84.
                     <div class="extra-links" id="e_1_2_6_5_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK3c7gt1Cnsg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=2294755" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1990CJ04300009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=American+Journal+of+Human+Genetics&amp;publication_year=1990&amp;pages=%0ABarrantes+R%2C+%0ASmouse+PE%2C+%0AMohrenweiser+HW%0A++++++++++++et+al.++%281990%29+Microevolution+in+lower+Central+America.+I.+Genetic+characterization+of+the+Chibcha%E2%80%90speaking+groups+of+Costa+Rica+and+Panama%2C+and+a+taxonomy+based+on+genetics%2C+linguistics%2C+and+geography.+%0AAmerican+Journal+of+Human+Genetics%2C+46%2C+63%E2%80%9384.&amp;journal=63-84.&amp;issue=46&amp;author=R+Barrantesauthor=PE+Smouseauthor=HW+Mohrenweiser&amp;title=Microevolution+in+lower+Central+America.+I.+Genetic+characterization+of+the+Chibcha%E2%80%90speaking+groups+of+Costa+Rica+and+Panama%2C+and+a+taxonomy+based+on+genetics%2C+linguistics%2C+and+geography&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b5"><span class="bullet">5</span>
                     Birky CWJ (<span class="pubYear">1988</span>) Evolution and variation in plant chloroplast and mitochondrial genomes. In: <i>Plant Evolutionary Biology</i> (eds Gottlieb L, Jain S), pp. 25–53. Chapman and Hall, New York.
                     <div class="extra-links" id="e_1_2_6_6_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ABirky+CWJ+%281988%29+Evolution+and+variation+in+plant+chloroplast+and+mitochondrial+genomes.+In%3A+Plant+Evolutionary+Biology+%28eds+Gottlieb+L%2C+Jain+S%29%2C+pp.+25%E2%80%9353.+Chapman+and+Hall%2C+New+York." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b6"><span class="bullet">6</span>
                     Brown BL, 
                     Epifanio JM, 
                     Smouse PE, 
                     Kobak CJ (<span class="pubYear">1996</span>) <span class="articleTitle">Temporal stability of mtDNA haplotype frequencies in American shad stocks: to pool
                        or not to pool across years?</span><i>Canadian Journal of Fisheries and Aquatic Science</i>, <span class="vol">53</span>, 2274–2283. 
                     <div class="extra-links" id="e_1_2_6_7_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1139%2Fcjfas-53-10-2274" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1996WE26000012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Canadian+Journal+of+Fisheries+and+Aquatic+Science&amp;publication_year=1996&amp;pages=%0ABrown+BL%2C+%0AEpifanio+JM%2C+%0ASmouse+PE%2C+%0AKobak+CJ+%281996%29+Temporal+stability+of+mtDNA+haplotype+frequencies+in+American+shad+stocks%3A+to+pool+or+not+to+pool+across+years%3FCanadian+Journal+of+Fisheries+and+Aquatic+Science%2C+53%2C+2274%E2%80%932283.+&amp;journal=2274-2283.+&amp;issue=53&amp;author=BL+Brownauthor=JM+Epifanioauthor=PE+Smouseauthor=CJ+Kobak&amp;title=Temporal+stability+of+mtDNA+haplotype+frequencies+in+American+shad+stocks%3A+to+pool+or+not+to+pool+across+years%3F&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b7"><span class="bullet">7</span>
                     Cann RL, 
                     Stoneking M, 
                     Wilson AC (<span class="pubYear">1987</span>) <span class="articleTitle">Mitochondrial DNA and human evolution</span>. 
                     <i>Nature</i>, <span class="vol">325</span>, 31–36.
                     <div class="extra-links" id="e_1_2_6_8_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1038%2F325031a0" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaL2sXpslagtQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=3025745" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1987F481900041" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature&amp;publication_year=1987&amp;pages=%0ACann+RL%2C+%0AStoneking+M%2C+%0AWilson+AC+%281987%29+Mitochondrial+DNA+and+human+evolution.+%0ANature%2C+325%2C+31%E2%80%9336.&amp;journal=31-36.&amp;issue=325&amp;author=RL+Cannauthor=M+Stonekingauthor=AC+Wilson&amp;title=Mitochondrial+DNA+and+human+evolution&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b8"><span class="bullet">8</span>
                     Crandall KA &amp; 
                     Templeton AR (<span class="pubYear">1993</span>) <span class="articleTitle">Empirical tests and some predictions from coalescent theory with applications to intraspecific
                        phylogeny reconstruction</span>. 
                     <i>Genetics</i>, <span class="vol">134</span>, 959–969.
                     <div class="extra-links" id="e_1_2_6_9_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK3szltFyltw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=8349118" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1993LJ51300025" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1993&amp;pages=%0ACrandall+KA+%26+%0ATempleton+AR+%281993%29+Empirical+tests+and+some+predictions+from+coalescent+theory+with+applications+to+intraspecific+phylogeny+reconstruction.+%0AGenetics%2C+134%2C+959%E2%80%93969.&amp;journal=959-969.&amp;issue=134&amp;author=KA+Crandallauthor=AR+Templeton&amp;title=Empirical+tests+and+some+predictions+from+coalescent+theory+with+applications+to+intraspecific+phylogeny+reconstruction&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b9"><span class="bullet">9</span>
                     Crandall KA &amp; 
                     Templeton AR (<span class="pubYear">1996</span>) Applications of intraspecific phylogenetics. In<i>: New Uses for New Phylogenies</i> (eds Harvey PH, Leigh Brown AJ, Maynard Smith J, Nee S), pp. 81–99. Oxford Press,
                     Oxford.
                     <div class="extra-links" id="e_1_2_6_10_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ACrandall+KA+%26+%0ATempleton+AR+%281996%29+Applications+of+intraspecific+phylogenetics.+In%3A+New+Uses+for+New+Phylogenies+%28eds+Harvey+PH%2C+Leigh+Brown+AJ%2C+Maynard+Smith+J%2C+Nee+S%29%2C+pp.+81%E2%80%9399.+Oxford+Press%2C+Oxford." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b10"><span class="bullet">10</span>
                     Crandall KA, 
                     Templeton AR, 
                     Sing CF (<span class="pubYear">1994</span>) Intraspecific cladogram estimation: Problems and solutions. In<b>:</b><i>Models in Phylogeny Reconstruction</i> (eds Scotland RW, Siebert DJ, Williams DM), pp. 273–297. Clarendon Press, Oxford.
                     <div class="extra-links" id="e_1_2_6_11_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ACrandall+KA%2C+%0ATempleton+AR%2C+%0ASing+CF+%281994%29+Intraspecific+cladogram+estimation%3A+Problems+and+solutions.+In%3AModels+in+Phylogeny+Reconstruction+%28eds+Scotland+RW%2C+Siebert+DJ%2C+Williams+DM%29%2C+pp.+273%E2%80%93297.+Clarendon+Press%2C+Oxford." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b11"><span class="bullet">11</span>
                     Epifanio JM, 
                     Smouse PE, 
                     Kobak CJ, 
                     Brown BL (<span class="pubYear">1995</span>) <span class="articleTitle">Mitochondrial DNA divergence among populations of American shad (<i>Alosa sapidissima</i>): How much variation is enough for mixed stock analysis? </span><i>Canadian Journal of Fisheries and Aquatic Science</i>, <span class="vol">52</span>, 1688–1702.
                     <div class="extra-links" id="e_1_2_6_12_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1139%2Ff95-761" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK28XktFSltg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1995TE92400009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Canadian+Journal+of+Fisheries+and+Aquatic+Science&amp;publication_year=1995&amp;pages=%0AEpifanio+JM%2C+%0ASmouse+PE%2C+%0AKobak+CJ%2C+%0ABrown+BL+%281995%29+Mitochondrial+DNA+divergence+among+populations+of+American+shad+%28Alosa+sapidissima%29%3A+How+much+variation+is+enough+for+mixed+stock+analysis%3F+Canadian+Journal+of+Fisheries+and+Aquatic+Science%2C+52%2C+1688%E2%80%931702.&amp;journal=1688-1702.&amp;issue=52&amp;author=JM+Epifanioauthor=PE+Smouseauthor=CJ+Kobakauthor=BL+Brown&amp;title=Mitochondrial+DNA+divergence+among+populations+of+American+shad+%28Alosa+sapidissima%29%3A+How+much+variation+is+enough+for+mixed+stock+analysis%3F+&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b12"><span class="bullet">12</span>
                     Etges WJ, 
                     Johnson WR, 
                     Duncan GA, 
                     Huckins G, 
                     Heed WB (<span class="pubYear">1998</span>) Ecological genetics of cactophilic Drosophila. In: <i>Ecology and Conservation of the Sonoran Desert Flora</i> (ed. Robichaux R). University of Arizona Press, Tucson, in press.
                     <div class="extra-links" id="e_1_2_6_13_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0AEtges+WJ%2C+%0AJohnson+WR%2C+%0ADuncan+GA%2C+%0AHuckins+G%2C+%0AHeed+WB+%281998%29+Ecological+genetics+of+cactophilic+Drosophila.+In%3A+Ecology+and+Conservation+of+the+Sonoran+Desert+Flora+%28ed.+Robichaux+R%29.+University+of+Arizona+Press%2C+Tucson%2C+in+press." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b13"><span class="bullet">13</span>
                     Excoffier L &amp; 
                     Smouse PE (<span class="pubYear">1994</span>) <span class="articleTitle">Using allele frequencies and geographic subdivision to reconstruct gene trees within
                        a species: Molecular variance parsimony</span>. 
                     <i>Genetics</i>, <span class="vol">136</span>, 343–359.
                     <div class="extra-links" id="e_1_2_6_14_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK2c7os1Smug%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=8138170" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1994MP22200031" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1994&amp;pages=%0AExcoffier+L+%26+%0ASmouse+PE+%281994%29+Using+allele+frequencies+and+geographic+subdivision+to+reconstruct+gene+trees+within+a+species%3A+Molecular+variance+parsimony.+%0AGenetics%2C+136%2C+343%E2%80%93359.&amp;journal=343-359.&amp;issue=136&amp;author=L+Excoffierauthor=PE+Smouse&amp;title=Using+allele+frequencies+and+geographic+subdivision+to+reconstruct+gene+trees+within+a+species%3A+Molecular+variance+parsimony&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b14"><span class="bullet">14</span>
                     Excoffier L, 
                     Smouse PE, 
                     Quattro JM (<span class="pubYear">1992</span>) <span class="articleTitle">Analysis of molecular variance inferred from metric distances among DNA haplotypes:
                        Application to human mitochondrial DNA restriction sites</span>. 
                     <i>Genetics</i>, <span class="vol">131</span>, 479–491.
                     <div class="extra-links" id="e_1_2_6_15_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1016%2F0277-3791%2895%2900068-2" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK38XlsVCntro%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=64&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1995QSRv...14..867M" onclick="newWindow(this.href);return false" target="_blank">ADS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1644282" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1992HW75900021" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1992&amp;pages=%0AExcoffier+L%2C+%0ASmouse+PE%2C+%0AQuattro+JM+%281992%29+Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+Application+to+human+mitochondrial+DNA+restriction+sites.+%0AGenetics%2C+131%2C+479%E2%80%93491.&amp;journal=479-491.&amp;issue=131&amp;author=L+Excoffierauthor=PE+Smouseauthor=JM+Quattro&amp;title=Analysis+of+molecular+variance+inferred+from+metric+distances+among+DNA+haplotypes%3A+Application+to+human+mitochondrial+DNA+restriction+sites&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b15"><span class="bullet">15</span>
                     Gibbons A (<span class="pubYear">1985</span>) <i>Algorithmic Graph Theory</i>. Cambridge University Press, Cambridge.
                     <div class="extra-links" id="e_1_2_6_16_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0AGibbons+A+%281985%29+Algorithmic+Graph+Theory.+Cambridge+University+Press%2C+Cambridge." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b16"><span class="bullet">16</span>
                     Harris SA &amp; 
                     Ingram R (<span class="pubYear">1991</span>) <span class="articleTitle">Chloroplast DNA and biosystematics: the effects of intraspecific diversity and plasmid
                        transmission</span>. 
                     <i>Taxon</i>, <span class="vol">40</span>, 393–412.
                     <div class="extra-links" id="e_1_2_6_17_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.2307%2F1223218" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991GD09100002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Taxon&amp;publication_year=1991&amp;pages=%0AHarris+SA+%26+%0AIngram+R+%281991%29+Chloroplast+DNA+and+biosystematics%3A+the+effects+of+intraspecific+diversity+and+plasmid+transmission.+%0ATaxon%2C+40%2C+393%E2%80%93412.&amp;journal=393-412.&amp;issue=40&amp;author=SA+Harrisauthor=R+Ingram&amp;title=Chloroplast+DNA+and+biosystematics%3A+the+effects+of+intraspecific+diversity+and+plasmid+transmission&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b17"><span class="bullet">17</span>
                     Hedges SB, 
                     Kumar S, 
                     Tamura K, 
                     Stoneking M (<span class="pubYear">1992</span>) <span class="articleTitle">Human origins and analysis of mitochondrial DNA sequences</span>. 
                     <i>Nature</i>, <span class="vol">255</span>, 737–739.
                     <div class="extra-links" id="e_1_2_6_18_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK387ltFWjsg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1738849" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1992HC50600046" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature&amp;publication_year=1992&amp;pages=%0AHedges+SB%2C+%0AKumar+S%2C+%0ATamura+K%2C+%0AStoneking+M+%281992%29+Human+origins+and+analysis+of+mitochondrial+DNA+sequences.+%0ANature%2C+255%2C+737%E2%80%93739.&amp;journal=737-739.&amp;issue=255&amp;author=SB+Hedgesauthor=S+Kumarauthor=K+Tamuraauthor=M+Stoneking&amp;title=Human+origins+and+analysis+of+mitochondrial+DNA+sequences&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b18"><span class="bullet">18</span>
                     Hudson RR (<span class="pubYear">1989</span>) <span class="articleTitle">How often are polymorphic restriction sites due to a single mutation?</span><i>Theoretical Population Biology</i>, <span class="vol">36</span>, 23–33.
                     <div class="extra-links" id="e_1_2_6_19_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1016%2F0040-5809%2889%2990021-X" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK3c%252FitVSrtA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=2572069" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1989AQ68300002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Theoretical+Population+Biology&amp;publication_year=1989&amp;pages=%0AHudson+RR+%281989%29+How+often+are+polymorphic+restriction+sites+due+to+a+single+mutation%3FTheoretical+Population+Biology%2C+36%2C+23%E2%80%9333.&amp;journal=23-33.&amp;issue=36&amp;author=RR+Hudson&amp;title=How+often+are+polymorphic+restriction+sites+due+to+a+single+mutation%3F&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b19"><span class="bullet">19</span>
                     Hudson RR, 
                     Boos DD, 
                     Kaplan NL (<span class="pubYear">1992</span>) <span class="articleTitle">A statistical test for detecting geographic subdivision</span>. 
                     <i>Molecular Biology and Evolution</i>, <span class="vol">9</span>, 138–151.
                     <div class="extra-links" id="e_1_2_6_20_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK383gtl2rtg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1552836" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1992GX79400010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Biology+and+Evolution&amp;publication_year=1992&amp;pages=%0AHudson+RR%2C+%0ABoos+DD%2C+%0AKaplan+NL+%281992%29+A+statistical+test+for+detecting+geographic+subdivision.+%0AMolecular+Biology+and+Evolution%2C+9%2C+138%E2%80%93151.&amp;journal=138-151.&amp;issue=9&amp;author=RR+Hudsonauthor=DD+Boosauthor=NL+Kaplan&amp;title=A+statistical+test+for+detecting+geographic+subdivision&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b20"><span class="bullet">20</span>
                     Huff DR, 
                     Peakall R, 
                     Smouse PE (<span class="pubYear">1993</span>) <span class="articleTitle">RAPD variation within and among natural populations of outcrossing Buffalograss (<i>Buchloë dactyloides</i> (Nutt.) Englem.)</span>. 
                     <i>Theoretical and Applied Genetics</i>, <span class="vol">86</span>, 927–934.
                     <div class="extra-links" id="e_1_2_6_21_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1007%2FBF00211043" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK2cXitFaktrw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=24193999" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1993MA72300006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Theoretical+and+Applied+Genetics&amp;publication_year=1993&amp;pages=%0AHuff+DR%2C+%0APeakall+R%2C+%0ASmouse+PE+%281993%29+RAPD+variation+within+and+among+natural+populations+of+outcrossing+Buffalograss+%28Buchlo%C3%AB+dactyloides+%28Nutt.%29+Englem.%29.+%0ATheoretical+and+Applied+Genetics%2C+86%2C+927%E2%80%93934.&amp;journal=927-934.&amp;issue=86&amp;author=DR+Huffauthor=R+Peakallauthor=PE+Smouse&amp;title=RAPD+variation+within+and+among+natural+populations+of+outcrossing+Buffalograss+%28Buchlo%C3%AB+dactyloides+%28Nutt.%29+Englem.%29&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b21"><span class="bullet">21</span>
                     Joseph L &amp; 
                     Moritz C (<span class="pubYear">1995</span>) <span class="articleTitle">Mitochondrial phylogeography of birds from eastern Australian rainforests: first fragments</span>. 
                     <i>Australian Journal of Zoology</i>, <span class="vol">42</span>, 385–403.
                     <div class="extra-links" id="e_1_2_6_22_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1071%2FZO9940385" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1Krsrw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1994PG12400011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Australian+Journal+of+Zoology&amp;publication_year=1995&amp;pages=%0AJoseph+L+%26+%0AMoritz+C+%281995%29+Mitochondrial+phylogeography+of+birds+from+eastern+Australian+rainforests%3A+first+fragments.+%0AAustralian+Journal+of+Zoology%2C+42%2C+385%E2%80%93403.&amp;journal=385-403.&amp;issue=42&amp;author=L+Josephauthor=C+Moritz&amp;title=Mitochondrial+phylogeography+of+birds+from+eastern+Australian+rainforests%3A+first+fragments&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b22"><span class="bullet">22</span>
                     Kolman CJ, 
                     Bermingham E, 
                     Cooke R, 
                     Ward RH, 
                     Arias TD, 
                     Guionneau‐Sinclair F (<span class="pubYear">1995</span>) <span class="articleTitle">Reduced mtDNA diversity in the Ngöbé Amerinds of Panamá</span>. 
                     <i>Genetics</i>, <span class="vol">140</span>, 275–283.
                     <div class="extra-links" id="e_1_2_6_23_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK28Xht1Cgu70%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=7635293" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1995QV52100023" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1995&amp;pages=%0AKolman+CJ%2C+%0ABermingham+E%2C+%0ACooke+R%2C+%0AWard+RH%2C+%0AArias+TD%2C+%0AGuionneau%E2%80%90Sinclair+F+%281995%29+Reduced+mtDNA+diversity+in+the+Ng%C3%B6b%C3%A9+Amerinds+of+Panam%C3%A1.+%0AGenetics%2C+140%2C+275%E2%80%93283.&amp;journal=275-283.&amp;issue=140&amp;author=CJ+Kolmanauthor=E+Berminghamauthor=R+Cookeauthor=RH+Wardauthor=TD+Ariasauthor=F+Guionneau%E2%80%90Sinclair&amp;title=Reduced+mtDNA+diversity+in+the+Ng%C3%B6b%C3%A9+Amerinds+of+Panam%C3%A1&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b23"><span class="bullet">23</span>
                     Long JC, 
                     Chakravarti A, 
                     Boehm CD, 
                     Antonarakis S, 
                     Kazazian HH (<span class="pubYear">1990</span>) <span class="articleTitle">Phylogeny of human β‐globin haplotypes and its implications for recent human evolution</span>. 
                     <i>American Journal of Physical Anthropology</i>, <span class="vol">81</span>, 113–130.
                     <div class="extra-links" id="e_1_2_6_24_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1002/ajpa.1330810112" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK3c7ksV2rtA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1967905" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1990CH23800011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=American+Journal+of+Physical+Anthropology&amp;publication_year=1990&amp;pages=%0ALong+JC%2C+%0AChakravarti+A%2C+%0ABoehm+CD%2C+%0AAntonarakis+S%2C+%0AKazazian+HH+%281990%29+Phylogeny+of+human+%CE%B2%E2%80%90globin+haplotypes+and+its+implications+for+recent+human+evolution.+%0AAmerican+Journal+of+Physical+Anthropology%2C+81%2C+113%E2%80%93130.&amp;journal=113-130.&amp;issue=81&amp;author=JC+Longauthor=A+Chakravartiauthor=CD+Boehmauthor=S+Antonarakisauthor=HH+Kazazian&amp;title=Phylogeny+of+human+%CE%B2%E2%80%90globin+haplotypes+and+its+implications+for+recent+human+evolution&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b24"><span class="bullet">24</span>
                     Maddison DR (<span class="pubYear">1991</span>) <span class="articleTitle">African origin of human mitochondrial DNA reexamined</span>. 
                     <i>Systematic Zoology</i>, <span class="vol">40</span>, 355–362.
                     <div class="extra-links" id="e_1_2_6_25_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.2307%2F2992327" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991GH60400007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Systematic+Zoology&amp;publication_year=1991&amp;pages=%0AMaddison+DR+%281991%29+African+origin+of+human+mitochondrial+DNA+reexamined.+%0ASystematic+Zoology%2C+40%2C+355%E2%80%93362.&amp;journal=355-362.&amp;issue=40&amp;author=DR+Maddison&amp;title=African+origin+of+human+mitochondrial+DNA+reexamined&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b25"><span class="bullet">25</span>
                     Maddison DR, 
                     Ruvolo M, 
                     Swofford DL (<span class="pubYear">1992</span>) <span class="articleTitle">Geographic origins of human mitochondrial DNA: phylogenetic evidence from control
                        region sequences</span>. 
                     <i>Systematic Biology</i>, <span class="vol">41</span>, 111–124.
                     <div class="extra-links" id="e_1_2_6_26_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1093%2Fsysbio%2F41.1.111" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1992JL28900010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Systematic+Biology&amp;publication_year=1992&amp;pages=%0AMaddison+DR%2C+%0ARuvolo+M%2C+%0ASwofford+DL+%281992%29+Geographic+origins+of+human+mitochondrial+DNA%3A+phylogenetic+evidence+from+control+region+sequences.+%0ASystematic+Biology%2C+41%2C+111%E2%80%93124.&amp;journal=111-124.&amp;issue=41&amp;author=DR+Maddisonauthor=M+Ruvoloauthor=DL+Swofford&amp;title=Geographic+origins+of+human+mitochondrial+DNA%3A+phylogenetic+evidence+from+control+region+sequences&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b26"><span class="bullet">26</span>
                     Mason‐Gamer RJ, 
                     Holsinger KE, 
                     Jansen RK (<span class="pubYear">1995</span>) <span class="articleTitle">Chloroplast DNA haplotype variation within and among populations of <i>Coreopsis grandiflora</i> (Asteraceae<i>)</i></span>. 
                     <i>Molecular Biology and Evolution</i>, <span class="vol">12</span>, 371–381.
                     <div class="extra-links" id="e_1_2_6_27_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK2MXltF2ntLc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1995QU32000002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Biology+and+Evolution&amp;publication_year=1995&amp;pages=%0AMason%E2%80%90Gamer+RJ%2C+%0AHolsinger+KE%2C+%0AJansen+RK+%281995%29+Chloroplast+DNA+haplotype+variation+within+and+among+populations+of+Coreopsis+grandiflora+%28Asteraceae%29.+%0AMolecular+Biology+and+Evolution%2C+12%2C+371%E2%80%93381.&amp;journal=371-381.&amp;issue=12&amp;author=RJ+Mason%E2%80%90Gamerauthor=KE+Holsingerauthor=RK+Jansen&amp;title=Chloroplast+DNA+haplotype+variation+within+and+among+populations+of+Coreopsis+grandiflora+%28Asteraceae%29&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b27"><span class="bullet">27</span>
                     Merriwether DA, 
                     Clark AG, 
                     Ballinger SW
                     <i>et al.</i>  (<span class="pubYear">1991</span>) <span class="articleTitle">The structure of human mitochondrial DNA variation</span>. 
                     <i>Journal of Molecular Evolution</i>, <span class="vol">33</span>, 543–555.
                     <div class="extra-links" id="e_1_2_6_28_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1007%2FBF02102807" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK38Xls1Wltg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1685753" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991GT50800009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Journal+of+Molecular+Evolution&amp;publication_year=1991&amp;pages=%0AMerriwether+DA%2C+%0AClark+AG%2C+%0ABallinger+SW%0A++++++++++++et+al.++%281991%29+The+structure+of+human+mitochondrial+DNA+variation.+%0AJournal+of+Molecular+Evolution%2C+33%2C+543%E2%80%93555.&amp;journal=543-555.&amp;issue=33&amp;author=DA+Merriwetherauthor=AG+Clarkauthor=SW+Ballinger&amp;title=The+structure+of+human+mitochondrial+DNA+variation&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b28"><span class="bullet">28</span>
                     Michalakis Y &amp; 
                     Excoffier L (<span class="pubYear">1996</span>) <span class="articleTitle">A generic estimate of population subdivision using distances between alleles with
                        special reference for microsatellite loci</span>. 
                     <i>Genetics</i>, <span class="vol">142</span>, 1061–1064.
                     <div class="extra-links" id="e_1_2_6_29_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK2s%252Fgt1Gltw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=8849912" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1996TX45500037" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1996&amp;pages=%0AMichalakis+Y+%26+%0AExcoffier+L+%281996%29+A+generic+estimate+of+population+subdivision+using+distances+between+alleles+with+special+reference+for+microsatellite+loci.+%0AGenetics%2C+142%2C+1061%E2%80%931064.&amp;journal=1061-1064.&amp;issue=142&amp;author=Y+Michalakisauthor=L+Excoffier&amp;title=A+generic+estimate+of+population+subdivision+using+distances+between+alleles+with+special+reference+for+microsatellite+loci&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b29"><span class="bullet">29</span>
                     Morgan K &amp; 
                     Strobeck C (<span class="pubYear">1979</span>) <span class="articleTitle">Is intragenic recombination a factor in the maintenance of genetic variation in natural
                        populations?</span><i>Nature</i>, <span class="vol">277</span>, 383–384.
                     <div class="extra-links" id="e_1_2_6_30_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1038%2F277383a0" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaL3M%252Fht1SgtA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=551258" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1979GG38600044" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature&amp;publication_year=1979&amp;pages=%0AMorgan+K+%26+%0AStrobeck+C+%281979%29+Is+intragenic+recombination+a+factor+in+the+maintenance+of+genetic+variation+in+natural+populations%3FNature%2C+277%2C+383%E2%80%93384.&amp;journal=383-384.&amp;issue=277&amp;author=K+Morganauthor=C+Strobeck&amp;title=Is+intragenic+recombination+a+factor+in+the+maintenance+of+genetic+variation+in+natural+populations%3F&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b30"><span class="bullet">30</span>
                     Moritz C, 
                     Dowling TE, 
                     Brown WM (<span class="pubYear">1987</span>) <span class="articleTitle">Evolution of animal mitochondrial DNA: relevance for population biology and systematics</span>. 
                     <i>Annual Review of Ecology and Systematics</i>, <span class="vol">18</span>, 269–292.
                     <div class="extra-links" id="e_1_2_6_31_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2005.02773.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADC%252BD28XitVGktrY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=16367830" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1987K958800012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Annual+Review+of+Ecology+and+Systematics&amp;publication_year=1987&amp;pages=%0AMoritz+C%2C+%0ADowling+TE%2C+%0ABrown+WM+%281987%29+Evolution+of+animal+mitochondrial+DNA%3A+relevance+for+population+biology+and+systematics.+%0AAnnual+Review+of+Ecology+and+Systematics%2C+18%2C+269%E2%80%93292.&amp;journal=269-292.&amp;issue=18&amp;author=C+Moritzauthor=TE+Dowlingauthor=WM+Brown&amp;title=Evolution+of+animal+mitochondrial+DNA%3A+relevance+for+population+biology+and+systematics&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b31"><span class="bullet">31</span>
                     Peakall R, 
                     Smouse PE, 
                     Huff DR (<span class="pubYear">1995</span>) <span class="articleTitle">Evolutionary implications of allozyme and RAPD variation in diploid populations of
                        dioecious buffalograss <i>Buchloë dactyloides</i></span>. 
                     <i>Molecular Ecology</i>, <span class="vol">4</span>, 135–147.
                     <div class="extra-links" id="e_1_2_6_32_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.1995.tb00203.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK2MXls1CjsL0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=11555250" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1995QR94900001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology&amp;publication_year=1995&amp;pages=%0APeakall+R%2C+%0ASmouse+PE%2C+%0AHuff+DR+%281995%29+Evolutionary+implications+of+allozyme+and+RAPD+variation+in+diploid+populations+of+dioecious+buffalograss+Buchlo%C3%AB+dactyloides.+%0AMolecular+Ecology%2C+4%2C+135%E2%80%93147.&amp;journal=135-147.&amp;issue=4&amp;author=R+Peakallauthor=PE+Smouseauthor=DR+Huff&amp;title=Evolutionary+implications+of+allozyme+and+RAPD+variation+in+diploid+populations+of+dioecious+buffalograss+Buchlo%C3%AB+dactyloides&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b32"><span class="bullet">32</span>
                     Prim RC (<span class="pubYear">1957</span>) <span class="articleTitle">Shortest connection networks and some generalizations</span>. 
                     <i>Bell System Technology Journal</i>, <span class="vol">36</span>, 1389–1401.
                     <div class="extra-links" id="e_1_2_6_33_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1002/j.1538-7305.1957.tb01515.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1957WV13500003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Bell+System+Technology+Journal&amp;publication_year=1957&amp;pages=%0APrim+RC+%281957%29+Shortest+connection+networks+and+some+generalizations.+%0ABell+System+Technology+Journal%2C+36%2C+1389%E2%80%931401.&amp;journal=1389-1401.&amp;issue=36&amp;author=RC+Prim&amp;title=Shortest+connection+networks+and+some+generalizations&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b33"><span class="bullet">33</span>
                     Quattro JM, 
                     Avise JC, 
                     Vrijenhoek RC (<span class="pubYear">1991</span>) <span class="articleTitle">Molecular evidence for multiple origins of hybridogenetic fish clones (Poeciliidae:
                        Poeciliopsis)</span>. 
                     <i>Genetics</i>, <span class="vol">127</span>, 391–398.
                     <div class="extra-links" id="e_1_2_6_34_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaK3M7mvFaruw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=2004710" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991EV66400014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1991&amp;pages=%0AQuattro+JM%2C+%0AAvise+JC%2C+%0AVrijenhoek+RC+%281991%29+Molecular+evidence+for+multiple+origins+of+hybridogenetic+fish+clones+%28Poeciliidae%3A+Poeciliopsis%29.+%0AGenetics%2C+127%2C+391%E2%80%93398.&amp;journal=391-398.&amp;issue=127&amp;author=JM+Quattroauthor=JC+Aviseauthor=RC+Vrijenhoek&amp;title=Molecular+evidence+for+multiple+origins+of+hybridogenetic+fish+clones+%28Poeciliidae%3A+Poeciliopsis%29&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b34"><span class="bullet">34</span>
                     Slatkin M &amp; 
                     Hudson RR (<span class="pubYear">1991</span>) <span class="articleTitle">Pairwise comparisons of mitochondrial DNA sequences in stable and exponentially growing
                        populations</span>. 
                     <i>Genetics</i>, <span class="vol">129</span>, 555–562.
                     <div class="extra-links" id="e_1_2_6_35_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK38Xhs12mtr8%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1743491" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991GH44500020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1991&amp;pages=%0ASlatkin+M+%26+%0AHudson+RR+%281991%29+Pairwise+comparisons+of+mitochondrial+DNA+sequences+in+stable+and+exponentially+growing+populations.+%0AGenetics%2C+129%2C+555%E2%80%93562.&amp;journal=555-562.&amp;issue=129&amp;author=M+Slatkinauthor=RR+Hudson&amp;title=Pairwise+comparisons+of+mitochondrial+DNA+sequences+in+stable+and+exponentially+growing+populations&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b35"><span class="bullet">35</span>
                     Smouse PE &amp; 
                     Chevillon C (<span class="pubYear">1998</span>) Analytical aspects of population‐specific DNA‐fingerprinting for individuals. <i>Journal of Heredity</i>, in press.
                     <div class="extra-links" id="e_1_2_6_36_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ASmouse+PE+%26+%0AChevillon+C+%281998%29+Analytical+aspects+of+population%E2%80%90specific+DNA%E2%80%90fingerprinting+for+individuals.+Journal+of+Heredity%2C+in+press." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b36"><span class="bullet">36</span>
                     Smouse PE, 
                     Dowling TE, 
                     Tworek JA, 
                     Hoeh WR, 
                     Brown WM (<span class="pubYear">1991</span>) <span class="articleTitle">Effects of intraspecific variation on phylogenetic inference: A likelihood analysis
                        of mtDNA restriction site data in cyprinid fishes</span>. 
                     <i>Systematic Zoology</i>, <span class="vol">40</span>, 393–409.
                     <div class="extra-links" id="e_1_2_6_37_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.2307%2F2992235" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991JE50000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Systematic+Zoology&amp;publication_year=1991&amp;pages=%0ASmouse+PE%2C+%0ADowling+TE%2C+%0ATworek+JA%2C+%0AHoeh+WR%2C+%0ABrown+WM+%281991%29+Effects+of+intraspecific+variation+on+phylogenetic+inference%3A+A+likelihood+analysis+of+mtDNA+restriction+site+data+in+cyprinid+fishes.+%0ASystematic+Zoology%2C+40%2C+393%E2%80%93409.&amp;journal=393-409.&amp;issue=40&amp;author=PE+Smouseauthor=TE+Dowlingauthor=JA+Tworekauthor=WR+Hoehauthor=WM+Brown&amp;title=Effects+of+intraspecific+variation+on+phylogenetic+inference%3A+A+likelihood+analysis+of+mtDNA+restriction+site+data+in+cyprinid+fishes&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b37"><span class="bullet">37</span>
                     Smouse PE &amp; 
                     Long JC (<span class="pubYear">1992</span>) <span class="articleTitle">Matrix correlation analysis in anthropology and genetics</span>. 
                     <i>Yearbook of Physical Anthropology</i>, <span class="vol">35</span>, 187–213.
                     <div class="extra-links" id="e_1_2_6_38_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1002/ajpa.1330350608" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1992KC96700006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Yearbook+of+Physical+Anthropology&amp;publication_year=1992&amp;pages=%0ASmouse+PE+%26+%0ALong+JC+%281992%29+Matrix+correlation+analysis+in+anthropology+and+genetics.+%0AYearbook+of+Physical+Anthropology%2C+35%2C+187%E2%80%93213.&amp;journal=187-213.&amp;issue=35&amp;author=PE+Smouseauthor=JC+Long&amp;title=Matrix+correlation+analysis+in+anthropology+and+genetics&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b38"><span class="bullet">38</span>
                     Sokal RR &amp; 
                     Rohlf FJ (<span class="pubYear">1962</span>) <span class="articleTitle">The comparison of dendrograms by objective methods</span>. 
                     <i>Taxon</i>, <span class="vol">11</span>, 33–40.
                     <div class="extra-links" id="e_1_2_6_39_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.2307%2F1217208" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Taxon&amp;publication_year=1962&amp;pages=%0ASokal+RR+%26+%0ARohlf+FJ+%281962%29+The+comparison+of+dendrograms+by+objective+methods.+%0ATaxon%2C+11%2C+33%E2%80%9340.&amp;journal=33-40.&amp;issue=11&amp;author=RR+Sokalauthor=FJ+Rohlf&amp;title=The+comparison+of+dendrograms+by+objective+methods&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b39"><span class="bullet">39</span>
                     Sokal RR, 
                     Smouse PE, 
                     Neel JV (<span class="pubYear">1986</span>) <span class="articleTitle">The genetic structure of a tribal population. XV. Patterns inferred by autocorrelation
                        analysis</span>. 
                     <i>Genetics</i>, <span class="vol">114</span>, 259–287.
                     <div class="extra-links" id="e_1_2_6_40_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaL2s%252FjtFGhsA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=3770468" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1986E052900016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1986&amp;pages=%0ASokal+RR%2C+%0ASmouse+PE%2C+%0ANeel+JV+%281986%29+The+genetic+structure+of+a+tribal+population.+XV.+Patterns+inferred+by+autocorrelation+analysis.+%0AGenetics%2C+114%2C+259%E2%80%93287.&amp;journal=259-287.&amp;issue=114&amp;author=RR+Sokalauthor=PE+Smouseauthor=JV+Neel&amp;title=The+genetic+structure+of+a+tribal+population.+XV.+Patterns+inferred+by+autocorrelation+analysis&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b40"><span class="bullet">40</span>
                     Soltis DE, 
                     Soltis PS, 
                     Milligan BG (<span class="pubYear">1992</span>) Intraspecific chloroplast DNA variation: systematic and phylogenetic implications.
                     In: <i>Molecular Systematics of Plants</i> (eds Soltis P, Soltis D, Doyle J), pp. 117–150. Chapman and Hall, New York.
                     <div class="extra-links" id="e_1_2_6_41_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ASoltis+DE%2C+%0ASoltis+PS%2C+%0AMilligan+BG+%281992%29+Intraspecific+chloroplast+DNA+variation%3A+systematic+and+phylogenetic+implications.+In%3A+Molecular+Systematics+of+Plants+%28eds+Soltis+P%2C+Soltis+D%2C+Doyle+J%29%2C+pp.+117%E2%80%93150.+Chapman+and+Hall%2C+New+York." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b41"><span class="bullet">41</span>
                     Strobeck C &amp; 
                     Morgan K (<span class="pubYear">1978</span>) <span class="articleTitle">The effect of intragenic recombination on the number of alleles in a finite population</span>. 
                     <i>Genetics</i>, <span class="vol">88</span>, 829–844.
                     <div class="extra-links" id="e_1_2_6_42_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=17248821" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1978FA50200012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genetics&amp;publication_year=1978&amp;pages=%0AStrobeck+C+%26+%0AMorgan+K+%281978%29+The+effect+of+intragenic+recombination+on+the+number+of+alleles+in+a+finite+population.+%0AGenetics%2C+88%2C+829%E2%80%93844.&amp;journal=829-844.&amp;issue=88&amp;author=C+Strobeckauthor=K+Morgan&amp;title=The+effect+of+intragenic+recombination+on+the+number+of+alleles+in+a+finite+population&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b42"><span class="bullet">42</span>
                     Templeton AR (<span class="pubYear">1983</span>) Convergent evolution and non‐parametric inferences from restriction data and DNA
                     sequences. In<b>:</b><i>Statistical Analysis of DNA Sequence Data</i> (ed. Weir BS), pp. 151–179. Marcel Dekker, New York.
                     <div class="extra-links" id="e_1_2_6_43_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ATempleton+AR+%281983%29+Convergent+evolution+and+non%E2%80%90parametric+inferences+from+restriction+data+and+DNA+sequences.+In%3AStatistical+Analysis+of+DNA+Sequence+Data+%28ed.+Weir+BS%29%2C+pp.+151%E2%80%93179.+Marcel+Dekker%2C+New+York." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b43"><span class="bullet">43</span>
                     Templeton AR (<span class="pubYear">1992</span>) Human origins and analysis of mitochondrial DNA sequences. <i>Nature</i>, <b>255</b>, 737.
                     <div class="extra-links" id="e_1_2_6_44_2"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ATempleton+AR+%281992%29+Human+origins+and+analysis+of+mitochondrial+DNA+sequences.+Nature%2C+255%2C+737." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b44"><span class="bullet">44</span>
                     Vigilant L, 
                     Stoneking M, 
                     Harpending H, 
                     Hawkes K, 
                     Wilson AC (<span class="pubYear">1991</span>) <span class="articleTitle">African populations and the evolution of human mitochondrial DNA</span>. 
                     <i>Science</i>, <span class="vol">253</span>, 1503–1507.
                     <div class="extra-links" id="e_1_2_6_45_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1126%2Fscience.1840702" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK3MXmt1Ghu7g%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1840702" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991GG65100032" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Science&amp;publication_year=1991&amp;pages=%0AVigilant+L%2C+%0AStoneking+M%2C+%0AHarpending+H%2C+%0AHawkes+K%2C+%0AWilson+AC+%281991%29+African+populations+and+the+evolution+of+human+mitochondrial+DNA.+%0AScience%2C+253%2C+1503%E2%80%931507.&amp;journal=1503-1507.&amp;issue=253&amp;author=L+Vigilantauthor=M+Stonekingauthor=H+Harpendingauthor=K+Hawkesauthor=AC+Wilson&amp;title=African+populations+and+the+evolution+of+human+mitochondrial+DNA&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b45"><span class="bullet">45</span>
                     Ward RH, 
                     Frazier BI, 
                     Dew‐Jager K, 
                     Pääbo S (<span class="pubYear">1991</span>) <span class="articleTitle">Extensive mitochondrial diversity within a single Amerindian tribe</span>. 
                     <i>Proceedings of the National Academy of Sciences (USA)</i>, <span class="vol">88</span>, 8720–8724.
                     <div class="extra-links" id="e_1_2_6_46_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1073%2Fpnas.88.19.8720" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK38Xkt1ag" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1681540" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1991GH80500090" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Proceedings+of+the+National+Academy+of+Sciences+%28USA%29&amp;publication_year=1991&amp;pages=%0AWard+RH%2C+%0AFrazier+BI%2C+%0ADew%E2%80%90Jager+K%2C+%0AP%C3%A4%C3%A4bo+S+%281991%29+Extensive+mitochondrial+diversity+within+a+single+Amerindian+tribe.+%0AProceedings+of+the+National+Academy+of+Sciences+%28USA%29%2C+88%2C+8720%E2%80%938724.&amp;journal=8720-8724.&amp;issue=88&amp;author=RH+Wardauthor=BI+Frazierauthor=K+Dew%E2%80%90Jagerauthor=S+P%C3%A4%C3%A4bo&amp;title=Extensive+mitochondrial+diversity+within+a+single+Amerindian+tribe&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b46"><span class="bullet">46</span>
                     Watterson GA &amp; 
                     Guess HA (<span class="pubYear">1977</span>) <span class="articleTitle">Is the most frequent allele the oldest?</span><i>Theoretical Population Biology</i>, <span class="vol">11</span>, 141–160.
                     <div class="extra-links" id="e_1_2_6_47_2"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=10.1016%2F0040-5809%2877%2990023-5" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ASTN%3A280%3ADyaE2s7ps1Siug%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=867285" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1977DG64500001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Theoretical+Population+Biology&amp;publication_year=1977&amp;pages=%0AWatterson+GA+%26+%0AGuess+HA+%281977%29+Is+the+most+frequent+allele+the+oldest%3FTheoretical+Population+Biology%2C+11%2C+141%E2%80%93160.&amp;journal=141-160.&amp;issue=11&amp;author=GA+Wattersonauthor=HA+Guess&amp;title=Is+the+most+frequent+allele+the+oldest%3F&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b47"><span class="bullet">47</span>
                     Weir BS &amp; 
                     Cockerham CC (<span class="pubYear">1984</span>) <span class="articleTitle">Estimating <i>F</i>‐statistics for the analysis of population structure</span>. 
                     <i>Evolution</i>, <span class="vol">38</span>, 1358–1370.
                     <div class="extra-links" id="e_1_2_6_48_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.1984.tb05657.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=1%3ACAS%3A528%3ADyaK2cXivFWisL4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=28563791" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1984TY40400017" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Evolution&amp;publication_year=1984&amp;pages=%0AWeir+BS+%26+%0ACockerham+CC+%281984%29+Estimating+F%E2%80%90statistics+for+the+analysis+of+population+structure.+%0AEvolution%2C+38%2C+1358%E2%80%931370.&amp;journal=1358-1370.&amp;issue=38&amp;author=BS+Weirauthor=CC+Cockerham&amp;title=Estimating+F%E2%80%90statistics+for+the+analysis+of+population+structure&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b48"><span class="bullet">48</span>
                     Wills C (<span class="pubYear">1996</span>) <span class="articleTitle">Topiary pruning and weighting reinforce an African origin for the human mitochondrial
                        DNA tree</span>. 
                     <i>Evolution</i>, <span class="vol">50</span>, 977–989.
                     <div class="extra-links" id="e_1_2_6_49_2"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.1996.tb02340.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x&amp;key=A1996UY55500002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Evolution&amp;publication_year=1996&amp;pages=%0AWills+C+%281996%29+Topiary+pruning+and+weighting+reinforce+an+African+origin+for+the+human+mitochondrial+DNA+tree.+%0AEvolution%2C+50%2C+977%E2%80%93989.&amp;journal=977-989.&amp;issue=50&amp;author=C+Wills&amp;title=Topiary+pruning+and+weighting+reinforce+an+African+origin+for+the+human+mitochondrial+DNA+tree&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
      
   </section>
   <section id="citedBy" class="article-section article-section__citedBy">
      <div class="accordion article-accordion"><a href="#" class="accordion__control" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" id="idss15" style="display: none;"><h3 class="border-bottom">Number of times cited: 47</h3><ul class="rlist separator"><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Alessio
            Maccagni, </span><span class="hlFld-ContribAuthor">Christian Parisod and </span><span class="hlFld-ContribAuthor">Jason R. Grant</span></span>, <span>Phylogeography of
            the moonwort fern Botrychium lunaria (Ophioglossaceae) based on chloroplast DNA in
            the Central-European Mountain System</span><span class="seperator">, </span><span class="seriesTitle"><i>Alpine Botany</i></span><span class="seperator">, </span><span class="volume"><b>127</b></span><span class="issue">, 2</span><span class="seperator">,
            </span><span class="page-range"> (185)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1007/s00035-017-0188-8">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">A.R.
            Templeton</span></span>, <span>World Dispersals and Genetic Diversity of Mankind</span>,
            <span class="seriesTitle"><span>On Human Nature</span></span><span class="seperator">,
            </span><span class="doi">10.1016/B978-0-12-420190-3.00005-3</span><span class="seperator">,
            </span><span class="page-range"> (65-83)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1016/B978-0-12-420190-3.00005-3">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Peter
            E. Smouse, </span><span class="hlFld-ContribAuthor">Michael R. Whitehead and </span><span class="hlFld-ContribAuthor">Rod Peakall</span></span>, <span>An informational diversity
            framework, illustrated with sexually deceptive orchids in early stages of speciation</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology Resources</i></span><span class="seperator">, </span><span class="volume"><b>15</b></span><span class="issue">,
            6</span><span class="seperator">, </span><span class="page-range"> (1375-1384)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2015)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12422">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Lisa
            N. Barrow, </span><span class="hlFld-ContribAuthor">Alyssa T. Bigelow, </span><span class="hlFld-ContribAuthor">Christopher A. Phillips and </span><span class="hlFld-ContribAuthor">Emily
            Moriarty Lemmon</span></span>, <span>Phylogeographic inference using Bayesian model
            comparison across a fragmented chorus frog species complex</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">,
            </span><span class="volume"><b>24</b></span><span class="issue">, 18</span><span class="seperator">,
            </span><span class="page-range"> (4739-4758)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2015)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.13343">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Raul Ruiz-Arce, </span><span class="hlFld-ContribAuthor">Christopher L. Owen, </span><span class="hlFld-ContribAuthor">Donald
            B. Thomas, </span><span class="hlFld-ContribAuthor">Norman B. Barr and </span><span class="hlFld-ContribAuthor">Bruce A. McPheron</span></span>, <span>Phylogeographic
            Structure in Anastrepha ludens (Diptera: Tephritidae) Populations Inferred With mtDNA
            Sequencing</span><span class="seperator">, </span><span class="seriesTitle"><i>Journal
            of Economic Entomology</i></span><span class="seperator">, </span><span class="volume"><b>108</b></span><span class="issue">, 3</span><span class="seperator">, </span><span class="page-range">
            (1324)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2015)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/jee/tov082">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Diego
            Mallo, </span><span class="hlFld-ContribAuthor">Agustín Sánchez‐Cobos and </span><span class="hlFld-ContribAuthor">Miguel Arenas</span></span>, <span>Diverse Considerations
            for Successful Phylogenetic Tree Reconstruction: Impacts from Model Misspecification,
            Recombination, Homoplasy, and Pattern Recognition</span>, <span class="seriesTitle"><span>Pattern
            Recognition in Computational Molecular Biology</span></span><span class="seperator">,
            </span><span class="page-range"> (439-456)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2015)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1002/9781119078845.ch23">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Fernanda
            F. Caniato, </span><span class="hlFld-ContribAuthor">Martha T. Hamblin, </span><span class="hlFld-ContribAuthor">Claudia T. Guimaraes, </span><span class="hlFld-ContribAuthor">Zhiwu
            Zhang, </span><span class="hlFld-ContribAuthor">Robert E. Schaffert, </span><span class="hlFld-ContribAuthor">Leon V. Kochian, </span><span class="hlFld-ContribAuthor">Jurandir
            V. Magalhaes and </span><span class="hlFld-ContribAuthor">James C. Nelson</span></span>,
            <span>Association Mapping Provides Insights into the Origin and the Fine Structure
            of the Sorghum Aluminum Tolerance Locus, AltSB</span><span class="seperator">, </span><span class="seriesTitle"><i>PLoS ONE</i></span><span class="seperator">, </span><span class="volume"><b>9</b></span><span class="issue">, 1</span><span class="seperator">, </span><span class="page-range">
            (e87438)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2014)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1371/journal.pone.0087438">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Alan
            R. Templeton</span></span>, <span>Biological races in humans</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Studies in History and Philosophy of Science Part
            C: Studies in History and Philosophy of Biological and Biomedical Sciences</i></span><span class="seperator">, </span><span class="doi">10.1016/j.shpsc.2013.04.010</span><span class="seperator">, </span><span class="volume"><b>44</b></span><span class="issue">,
            3</span><span class="seperator">, </span><span class="page-range"> (262-271)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2013)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1016/j.shpsc.2013.04.010">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Amanda
            J. Zellmer, </span><span class="hlFld-ContribAuthor">Margaret M. Hanes, </span><span class="hlFld-ContribAuthor">Sarah M. Hird and </span><span class="hlFld-ContribAuthor">Bryan
            C. Carstens</span></span>, <span>Deep Phylogeographic Structure and Environmental
            Differentiation in the Carnivorous Plant Sarracenia alata</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Systematic Biology</i></span><span class="seperator">,
            </span><span class="volume"><b>61</b></span><span class="issue">, 5</span><span class="seperator">,
            </span><span class="page-range"> (763)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2012)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/sysbio/sys048">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Emily K. Croteau, </span><span class="hlFld-ContribAuthor">Edward J. Heist, </span><span class="hlFld-ContribAuthor">Clayton
            K. Nielsen, </span><span class="hlFld-ContribAuthor">Jennifer R. Hutchinson and </span><span class="hlFld-ContribAuthor">Eric C. Hellgren</span></span>, <span>Microsatellites
            and mitochondrial DNA reveal regional population structure in bobcats (Lynx rufus)
            of North America</span><span class="seperator">, </span><span class="seriesTitle"><i>Conservation
            Genetics</i></span><span class="seperator">, </span><span class="volume"><b>13</b></span><span class="issue">, 6</span><span class="seperator">, </span><span class="page-range">
            (1637)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2012)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1007/s10592-012-0416-0">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">DARREN
            P. MARTIN, </span><span class="hlFld-ContribAuthor">PHILIPPE LEMEY and </span><span class="hlFld-ContribAuthor">DAVID POSADA</span></span>, <span>Analysing recombination
            in nucleotide sequences</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology Resources</i></span><span class="seperator">, </span><span class="volume"><b>11</b></span><span class="issue">, 6</span><span class="seperator">, </span><span class="page-range">
            (943-955)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2011)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1755-0998.2011.03026.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Yizhong Duan, </span><span class="hlFld-ContribAuthor">Qingbo
            Gao, </span><span class="hlFld-ContribAuthor">Faqi Zhang, </span><span class="hlFld-ContribAuthor">Yinghu
            Li, </span><span class="hlFld-ContribAuthor">Pengcheng Fu and </span><span class="hlFld-ContribAuthor">Shilong
            Chen</span></span>, <span>Phylogeographic analysis of the endemic species Sibiraea
            angustata reveals a marginal refugium in the Qinghai–Tibet Plateau</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Nordic Journal of Botany</i></span><span class="seperator">,
            </span><span class="volume"><b>29</b></span><span class="issue">, 5</span><span class="seperator">,
            </span><span class="page-range"> (615-624)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2011)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1756-1051.2011.00999.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Laurent
            Excoffier, </span><span class="hlFld-ContribAuthor">Matthieu Foll and </span><span class="hlFld-ContribAuthor">Rémy J. Petit</span></span>, <span>Genetic Consequences
            of Range Expansions</span><span class="seperator">, </span><span class="seriesTitle"><i>Annual
            Review of Ecology, Evolution, and Systematics</i></span><span class="seperator">,
            </span><span class="volume"><b>40</b></span><span class="issue">, 1</span><span class="seperator">,
            </span><span class="page-range"> (481)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2009)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1146/annurev.ecolsys.39.110707.173414">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">BRYAN
            C. CARSTENS, </span><span class="hlFld-ContribAuthor">HOLLY N. STOUTE and </span><span class="hlFld-ContribAuthor">NOAH M. REID</span></span>, <span>An information‐theoretical
            approach to phylogeography</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>18</b></span><span class="issue">, 20</span><span class="seperator">, </span><span class="page-range">
            (4270-4282)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2009)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2009.04327.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">RÉMY J. PETIT</span></span>, <span>The coup de grâce for
            the nested clade phylogeographic analysis?</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">, </span><span class="volume"><b>17</b></span><span class="issue">, 2</span><span class="seperator">,
            </span><span class="page-range"> (516-518)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2007)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2007.03589.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">GIORGOS
            TRYFONOPOULOS, </span><span class="hlFld-ContribAuthor">EVANTHIA THANOU, </span><span class="hlFld-ContribAuthor">BASILIOS CHONDROPOULOS and </span><span class="hlFld-ContribAuthor">STELLA
            FRAGUEDAKIS‐TSOLIS</span></span>, <span>mtDNA analysis reveals the ongoing speciation
            on Greek populations of Microtus (Terricola) thomasi (Arvicolidae, Rodentia)</span><span class="seperator">, </span><span class="seriesTitle"><i>Biological Journal of the
            Linnean Society</i></span><span class="seperator">, </span><span class="volume"><b>95</b></span><span class="issue">, 1</span><span class="seperator">, </span><span class="page-range">
            (117-130)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2008)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.2008.01025.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">O. HAMARSHEH, </span><span class="hlFld-ContribAuthor">W.
            PRESBER, </span><span class="hlFld-ContribAuthor">Z. ABDEEN, </span><span class="hlFld-ContribAuthor">S.
            SAWALHA, </span><span class="hlFld-ContribAuthor">A. AL‐LAHEM and </span><span class="hlFld-ContribAuthor">G.
            SCHÖNIAN</span></span>, <span>Genetic structure of Mediterranean populations of the
            sandfly Phlebotomus papatasi by mitochondrial cytochrome b haplotype analysis</span><span class="seperator">, </span><span class="seriesTitle"><i>Medical and Veterinary Entomology</i></span><span class="seperator">, </span><span class="volume"><b>21</b></span><span class="issue">,
            3</span><span class="seperator">, </span><span class="page-range"> (270-277)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2007)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2915.2007.00695.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Y. NACIRI and </span><span class="hlFld-ContribAuthor">M.
            GAUDEUL</span></span>, <span>Phylogeography of the endangered Eryngium alpinum L.
            (Apiaceae) in the European Alps</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>16</b></span><span class="issue">, 13</span><span class="seperator">, </span><span class="page-range">
            (2721-2733)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2007)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2007.03269.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Fabrícia F. Do Nascimento, </span><span class="hlFld-ContribAuthor">Cibele
            R. Bonvicino and </span><span class="hlFld-ContribAuthor">Héctor N. Seuánez</span></span>,
            <span>Population genetic studies of Alouatta caraya (Alouattinae, Primates): inferences
            on geographic distribution and ecology</span><span class="seperator">, </span><span class="seriesTitle"><i>American Journal of Primatology</i></span><span class="seperator">,
            </span><span class="volume"><b>69</b></span><span class="issue">, 10</span><span class="seperator">,
            </span><span class="page-range"> (1093-1104)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2007)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1002/ajp.20423">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">T. Kavar, </span><span class="hlFld-ContribAuthor">P.
            Pavlovčič, </span><span class="hlFld-ContribAuthor">S. Sušnik, </span><span class="hlFld-ContribAuthor">V.
            Meglič and </span><span class="hlFld-ContribAuthor">M. Virant-Doberlet</span></span>,
            <span>Genetic differentiation of geographically separated populations of the southern
            green stink bug Nezara viridula (Hemiptera: Pentatomidae)</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Bulletin of Entomological Research</i></span><span class="seperator">, </span><span class="volume"><b>96</b></span><span class="issue">,
            02</span><span class="seperator">, </span><span class="page-range"> (117)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2006)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1079/BER2005406">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Marchelli
            Paula and </span><span class="hlFld-ContribAuthor">Gallo Leonardo</span></span>, <span>Multiple
            ice-age refugia in a southern beech of South America as evidenced by chloroplast DNA
            markers</span><span class="seperator">, </span><span class="seriesTitle"><i>Conservation
            Genetics</i></span><span class="seperator">, </span><span class="volume"><b>7</b></span><span class="issue">, 4</span><span class="seperator">, </span><span class="page-range">
            (591)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2006)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1007/s10592-005-9069-6">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">JEF
            R. JAEGER, </span><span class="hlFld-ContribAuthor">BRETT R. RIDDLE and </span><span class="hlFld-ContribAuthor">DAVID F. BRADFORD</span></span>, <span>Cryptic Neogene
            vicariance and Quaternary dispersal of the red‐spotted toad (Bufo punctatus): insights
            on the evolution of North American warm desert biotas</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">,
            </span><span class="volume"><b>14</b></span><span class="issue">, 10</span><span class="seperator">,
            </span><span class="page-range"> (3033-3048)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2005)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2005.02645.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">F.F.
            Nascimento, </span><span class="hlFld-ContribAuthor">C.R. Bonvicino, </span><span class="hlFld-ContribAuthor">F.C.D. da Silva, </span><span class="hlFld-ContribAuthor">M.P.C.
            Schneider and </span><span class="hlFld-ContribAuthor">H.N. Seuánez</span></span>,
            <span>Cytochrome <i>b</i> polymorphisms and population structure of two species of
            <i>Alouatta</i> (Primates)</span><span class="seperator">, </span><span class="seriesTitle"><i>Cytogenetic
            and Genome Research</i></span><span class="seperator">, </span><span class="volume"><b>108</b></span><span class="issue">, 1-3</span><span class="seperator">, </span><span class="page-range">
            (106)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2005)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1159/000080808">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">John
            D. Bussell, </span><span class="hlFld-ContribAuthor">Michelle Waycott and </span><span class="hlFld-ContribAuthor">Jennifer A. Chappill</span></span>, <span>Arbitrarily
            amplified DNA markers as characters for phylogenetic inference</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Perspectives in Plant Ecology, Evolution and Systematics</i></span><span class="seperator">, </span><span class="volume"><b>7</b></span><span class="issue">,
            1</span><span class="seperator">, </span><span class="page-range"> (3)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2005)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1016/j.ppees.2004.07.001">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">M.
            E. DORKEN and </span><span class="hlFld-ContribAuthor">S. C. H. BARRETT</span></span>,
            <span>Chloroplast haplotype variation among monoecious and dioecious populations of
            Sagittaria latifolia (Alismataceae) in eastern North America</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">,
            </span><span class="volume"><b>13</b></span><span class="issue">, 9</span><span class="seperator">,
            </span><span class="page-range"> (2699-2707)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2004)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2004.02246.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">E.
            RANDI, </span><span class="hlFld-ContribAuthor">P. C. ALVES, </span><span class="hlFld-ContribAuthor">J.
            CARRANZA, </span><span class="hlFld-ContribAuthor">S. MILOŠEVIĆ‐ZLATANOVIĆ, </span><span class="hlFld-ContribAuthor">A. SFOUGARIS and </span><span class="hlFld-ContribAuthor">N.
            MUCCI</span></span>, <span>Phylogeography of roe deer (Capreolus capreolus) populations:
            the effects of historical genetic subdivisions and recent nonequilibrium dynamics</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">, </span><span class="volume"><b>13</b></span><span class="issue">,
            10</span><span class="seperator">, </span><span class="page-range"> (3071-3083)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2004)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2004.02279.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">T. T. T. NGUYEN, </span><span class="hlFld-ContribAuthor">C.
            M. AUSTIN, </span><span class="hlFld-ContribAuthor">M. M. MEEWAN, </span><span class="hlFld-ContribAuthor">M.
            B. SCHULTZ and </span><span class="hlFld-ContribAuthor">D. R. JERRY</span></span>,
            <span>Phylogeography of the freshwater crayfish Cherax destructor Clark (Parastacidae)
            in inland Australia: historical fragmentation and recent range expansion</span><span class="seperator">, </span><span class="seriesTitle"><i>Biological Journal of the
            Linnean Society</i></span><span class="seperator">, </span><span class="volume"><b>83</b></span><span class="issue">, 4</span><span class="seperator">, </span><span class="page-range">
            (539-550)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2004)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.2004.00410.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">De‐Xing Zhang and </span><span class="hlFld-ContribAuthor">Godfrey
            M. Hewitt</span></span>, <span>Nuclear DNA analyses in genetic studies of populations:
            practice, problems and prospects</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>12</b></span><span class="issue">, 3</span><span class="seperator">, </span><span class="page-range">
            (563-584)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2003)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.2003.01773.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">A. S. Schultheis, </span><span class="hlFld-ContribAuthor">L.
            A. Weigt and </span><span class="hlFld-ContribAuthor">A. C. Hendricks</span></span>,
            <span>Gene flow, dispersal, and nested clade analysis among populations of the stonefly
            Peltoperla tarteri in the southern Appalachians</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">, </span><span class="volume"><b>11</b></span><span class="issue">, 3</span><span class="seperator">,
            </span><span class="page-range"> (317-327)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2002)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.2002.01445.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">A.
            S. Schultheis, </span><span class="hlFld-ContribAuthor">L. A. Weigt and </span><span class="hlFld-ContribAuthor">A. C. Hendricks</span></span>, <span>Arrangement and structural
            conservation of the mitochondrial control region of two species of Plecoptera: utility
            of tandem repeat‐containing regions in studies of population genetics and evolutionary
            history</span><span class="seperator">, </span><span class="seriesTitle"><i>Insect
            Molecular Biology</i></span><span class="seperator">, </span><span class="volume"><b>11</b></span><span class="issue">, 6</span><span class="seperator">, </span><span class="page-range">
            (605-610)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2002)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2583.2002.00371.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">David M. Althoff and </span><span class="hlFld-ContribAuthor">Olle
            Pellmyr</span></span>, <span>EXAMINING GENETIC STRUCTURE IN A BOGUS YUCCA MOTH: A
            SEQUENTIAL APPROACH TO PHYLOGEOGRAPHY</span><span class="seperator">, </span><span class="seriesTitle"><i>Evolution</i></span><span class="seperator">, </span><span class="volume"><b>56</b></span><span class="issue">, 8</span><span class="seperator">,
            </span><span class="page-range"> (1632-1643)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2007)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.0014-3820.2002.tb01475.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">David
            M. Althoff and </span><span class="hlFld-ContribAuthor">Olle Pellmyr</span></span>,
            <span>EXAMINING GENETIC STRUCTURE IN A BOGUS YUCCA MOTH: A SEQUENTIAL APPROACH TO
            PHYLOGEOGRAPHY</span><span class="seperator">, </span><span class="seriesTitle"><i>Evolution</i></span><span class="seperator">, </span><span class="volume"><b>56</b></span><span class="issue">,
            8</span><span class="seperator">, </span><span class="page-range"> (1632)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2002)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1554/0014-3820%282002%29056[1632:EGSIAB]2.0.CO;2">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">I.
            Stehlik, </span><span class="hlFld-ContribAuthor">J. J. Schneller and </span><span class="hlFld-ContribAuthor">K. Bachmann</span></span>, <span>Resistance or emigration:
            response of the high‐alpine plant Eritrichium nanum (L.) Gaudin to the ice age within
            the Central Alps</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>10</b></span><span class="issue">, 2</span><span class="seperator">, </span><span class="page-range">
            (357-370)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2008)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.2001.01179.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">P. Kotlík and </span><span class="hlFld-ContribAuthor">P.
            Berrebi</span></span>, <span>Phylogeography of the barbel (Barbus barbus) assessed
            by mitochondrial DNA variation</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>10</b></span><span class="issue">, 9</span><span class="seperator">, </span><span class="page-range">
            (2177-2185)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2001)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.0962-1083.2001.01344.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">S. Creer, </span><span class="hlFld-ContribAuthor">A.
            Malhotra, </span><span class="hlFld-ContribAuthor">R. S. Thorpe and </span><span class="hlFld-ContribAuthor">W.‐H.
            Chou</span></span>, <span>Multiple causation of phylogeographical pattern as revealed
            by nested clade analysis of the bamboo viper (Trimeresurus stejnegeri) within Taiwan</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">, </span><span class="volume"><b>10</b></span><span class="issue">,
            8</span><span class="seperator">, </span><span class="page-range"> (1967-1981)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2001)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.0962-1083.2001.01332.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Eduardo Tarazona-Santos, </span><span class="hlFld-ContribAuthor">Denise
            R. Carvalho-Silva, </span><span class="hlFld-ContribAuthor">Davide Pettener, </span><span class="hlFld-ContribAuthor">Donata Luiselli, </span><span class="hlFld-ContribAuthor">Gian
            Franco De Stefano, </span><span class="hlFld-ContribAuthor">Cristina Martinez Labarga,
            </span><span class="hlFld-ContribAuthor">Olga Rickards, </span><span class="hlFld-ContribAuthor">Chris
            Tyler-Smith, </span><span class="hlFld-ContribAuthor">Sérgio D.J. Pena and </span><span class="hlFld-ContribAuthor">Fabrício R. Santos</span></span>, <span>Genetic Differentiation
            in South Amerindians Is Related to Environmental and Cultural Diversity: Evidence
            from the Y Chromosome</span><span class="seperator">, </span><span class="seriesTitle"><i>The
            American Journal of Human Genetics</i></span><span class="seperator">, </span><span class="volume"><b>68</b></span><span class="issue">, 6</span><span class="seperator">,
            </span><span class="page-range"> (1485)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2001)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1086/320601">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Jakob Müller</span></span>,
            <span>Invasion history and genetic population structure of riverine macroinvertebrates</span><span class="seperator">, </span><span class="seriesTitle"><i>Zoology</i></span><span class="seperator">,
            </span><span class="volume"><b>104</b></span><span class="issue">, 3-4</span><span class="seperator">, </span><span class="page-range"> (346)</span><span class="seperator">,
            </span><span class="pub-date"><span class="pub-year">(2001)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1078/0944-2006-00040">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Jianping Xu, </span><span class="hlFld-ContribAuthor">Rytas
            Vilgalys and </span><span class="hlFld-ContribAuthor">Thomas G. Mitchell</span></span>,
            <span>Multiple gene genealogies reveal recent dispersion and hybridization in the
            human pathogenic fungus Cryptococcus neoformans</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">, </span><span class="volume"><b>9</b></span><span class="issue">, 10</span><span class="seperator">,
            </span><span class="page-range"> (1471-1481)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2001)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.2000.01021.x">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Bruno
            Guinand and </span><span class="hlFld-ContribAuthor">Henri Tachet</span></span>, <span>Population
            structure of Hydropsyche exocellata. Genetic homogeneity in a zone of fragmented distribution</span><span class="seperator">, </span><span class="seriesTitle"><i>Comptes Rendus de l'Académie
            des Sciences - Series III - Sciences de la Vie</i></span><span class="seperator">,
            </span><span class="volume"><b>323</b></span><span class="issue">, 9</span><span class="seperator">,
            </span><span class="page-range"> (757)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2000)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1016/S0764-4469%2800%2901215-4">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">B.
            A. Schaal and </span><span class="hlFld-ContribAuthor">K. M. Olsen</span></span>,
            <span>Gene genealogies and population variation in plants</span><span class="seperator">,
            </span><span class="seriesTitle"><i>Proceedings of the National Academy of Sciences</i></span><span class="seperator">, </span><span class="volume"><b>97</b></span><span class="issue">,
            13</span><span class="seperator">, </span><span class="page-range"> (7024)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2000)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1073/pnas.97.13.7024">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Eva
            Lutz, </span><span class="hlFld-ContribAuthor">J. Jakob Schneller and </span><span class="hlFld-ContribAuthor">Rolf Holderegger</span></span>, <span>Understanding population
            history for conservation purposes: population genetics of Saxifraga aizoides (Saxifragaceae)
            in the lowlands and lower mountains north of the Alps </span><span class="seperator">,
            </span><span class="seriesTitle"><i>American Journal of Botany</i></span><span class="seperator">,
            </span><span class="volume"><b>87</b></span><span class="issue">, 4</span><span class="seperator">,
            </span><span class="page-range"> (583-590)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2000)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.2307/2656602">Wiley Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">George F. Barrowclough, </span><span class="hlFld-ContribAuthor">R. J. Gutiérrez and </span><span class="hlFld-ContribAuthor">Jeffrey
            G. Groth</span></span>, <span>PHYLOGEOGRAPHY OF SPOTTED OWL (STRIX OCCIDENTALIS) POPULATIONS
            BASED ON MITOCHONDRIAL DNA SEQUENCES: GENE FLOW, GENETIC STRUCTURE, AND A NOVEL BIOGEOGRAPHIC
            PATTERN</span><span class="seperator">, </span><span class="seriesTitle"><i>Evolution</i></span><span class="seperator">, </span><span class="volume"><b>53</b></span><span class="issue">,
            3</span><span class="seperator">, </span><span class="page-range"> (919-931)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.1999.tb05385.x">Wiley
            Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">G. G. Vendramin, </span><span class="hlFld-ContribAuthor">B.
            Degen, </span><span class="hlFld-ContribAuthor">R. J. Petit, </span><span class="hlFld-ContribAuthor">M.
            Anzidei, </span><span class="hlFld-ContribAuthor">A. Madaghiele and </span><span class="hlFld-ContribAuthor">
            B. Ziegenhagen</span></span>, <span>High level of variation at Abies alba chloroplast
            microsatellite loci in Europe</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>8</b></span><span class="issue">, 7</span><span class="seperator">, </span><span class="page-range">
            (1117-1126)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2002)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.1999.00666.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">JEAN‐DOMINIQUE DURAND, </span><span class="hlFld-ContribAuthor">BRUNO
            GUINAND and </span><span class="hlFld-ContribAuthor">YVETTE BOUVET</span></span>,
            <span>Local and global multivariate analysis of geographical mitochondrial DNA variation
            in Leuciscus cephalus L. 1758 (Pisces: Cyprinidae) in the Balkan Peninsula</span><span class="seperator">, </span><span class="seriesTitle"><i>Biological Journal of the
            Linnean Society</i></span><span class="seperator">, </span><span class="volume"><b>67</b></span><span class="issue">, 1</span><span class="seperator">, </span><span class="page-range">
            (19-42)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2008)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.1999.tb01927.x">Wiley
            Online Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">J. García‐martínez, </span><span class="hlFld-ContribAuthor">A.
            Moya, </span><span class="hlFld-ContribAuthor">J. A. Raga and </span><span class="hlFld-ContribAuthor">A.
            Latorre</span></span>, <span>Genetic differentiation in the striped dolphin Stenella
            coeruleoalba from European waters according to mitochondrial DNA (mtDNA) restriction
            analysis</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Ecology</i></span><span class="seperator">, </span><span class="volume"><b>8</b></span><span class="issue">, 6</span><span class="seperator">, </span><span class="page-range">
            (1069-1073)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2002)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.1999.00672.x">Wiley Online
            Library</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Jean Dominique Durand, </span><span class="hlFld-ContribAuthor">Alan
            R. Templeton, </span><span class="hlFld-ContribAuthor">Bruno Guinand, </span><span class="hlFld-ContribAuthor">Anastasia Imsiridou and </span><span class="hlFld-ContribAuthor">Yvette
            Bouvet</span></span>, <span>Nested Clade and Phylogeographic Analyses of the Chub,
            Leuciscus cephalus (Teleostei, Cyprinidae), in Greece: Implications for Balkan Peninsula
            Biogeography</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular
            Phylogenetics and Evolution</i></span><span class="seperator">, </span><span class="volume"><b>13</b></span><span class="issue">, 3</span><span class="seperator">, </span><span class="page-range">
            (566)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(1999)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1006/mpev.1999.0671">Crossref</a></span></div></li><li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">E.
            BERMINGHAM and </span><span class="hlFld-ContribAuthor">C. MORITZ</span></span>, <span>Comparative
            phylogeography: concepts and applications</span><span class="seperator">, </span><span class="seriesTitle"><i>Molecular Ecology</i></span><span class="seperator">, </span><span class="volume"><b>7</b></span><span class="issue">, 4</span><span class="seperator">,
            </span><span class="page-range"> (367-369)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2002)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.1998.00424.x">Wiley Online Library</a></span></div></li></ul>
         </div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md sticko__parent article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/1365294x/1998/7/4"><img src="mec.jpg" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/1365294x/1998/7/4"><span>Volume</span><span class="val">7</span>, <span>Issue</span><span class="val">4</span></a></p><p>April 1998</p><p class="page-range"><span>Pages</span><span> 399-412</span></p>



        
        
</div>
</div>



        
        


        <div class="pb-ad">
            		<div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_1__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_1" title="3rd party ad content" name="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_1" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x&pubId=50930488&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		</div>
        </div>
    

<div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko js--sticko" style="width: 329.983px; top: 81px; position: static; height: 0px;"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list cloned"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><ul role="tablist" class="rlist tab__content sticko__child" style="height: 0px; overflow-y: auto;"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tab" class="tab__pane article-row-right__panes active"><div class="contentTab"><!-- XSLT: InfoTab.xsl, Revision: 9055584, Interface: urn:atypon.com:wiley:interface:atypon-wiley-prime, was cached: Tue Aug 07 06:48:45 PDT 2018-->
<section class="info-tab-wrapper">
   <section class="metrics">
      <p class="info-header">Metrics</p>
      <div class="section__body">
         <div>
            <label>Downloads: </label>4
         </div>
         <div class="cited-by-count">
            <label>Citations: </label>47
         </div>
         <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-info-tab-metrics">
            



        
        <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
<div data-badge-popover="left" data-badge-type="1" data-doi="10.1046/j.1365-294x.1998.00370.x" data-hide-no-mentions="false" class="altmetric-embed" data-uuid="89e8d673-3df2-821d-7db4-5ead9e721a04"><a style="cursor: pointer; display: inline-block; background-image: url(&quot;https://d1uo4w7k31k5mn.cloudfront.net/v1/0.png&quot;); background-repeat: no-repeat; width: 110px; height: 20px;" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;doi=10.1046%2Fj.1365-294x.1998.00370.x" target="_self"> </a></div> 


         </div>
      </div>
   </section>
   <section class="details">
      <p class="info-header">Details</p>
      <p class="copyright">1998 Blackwell Science Ltd</p>
      <p></p>
      <ul class="cross-mark rlist"></ul>
   </section>
   <section class="keywords">
      <p class="info-header">Keywords</p>
      <ul class="rlist rlist--inline">
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=homoplasy">homoplasy</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=phylogeography">phylogeography</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=population%20structure">population structure</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=spanning%20trees">spanning trees</a>
         </li>
      </ul>
   </section>
   <section class="publication-history">
      <p class="info-header">Publication History</p>
      <ul class="rlist">
         <li>
            <label>Issue Online: </label>28 February 2002
         </li>
         <li>
            <label>Version of Record online: </label>28 February 2002
         </li>
      </ul>
   </section>
</section>
<!-- END OF XSLT -->
</div><div class="emptyTab">



        
        


        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_2__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_2" title="3rd party ad content" name="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_2" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x&pubId=50930488&placeholderId=1013&productId=101018" target="_blank">
				<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	</div>
</aside>
        </div>
    

</div></li><li class="tab__spinner" style="display: none;"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a title="Close Figure Viewer" href="#" data-ux3-role="controller" class="figure-viewer__ctrl__close"><span class="icon-close_thin"></span></a></div><div class="figure-viewer__top__left"><a title="Browse All Figures" href="#" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"></span></a><a title="Return to Figure" href="#" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: 100%;"><div class="figure-viewer__cent__left"><a title="Previous Figure" href="#" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"></span></a><a title="Next Figure" href="#" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
    </div>
</div><!--end pagefulltext-->

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">









        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h3 class="hidden accessibility-enabled">
                Additional links
            </h3>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="collapsible-menu">
  <h3 data-bind="toggle" class="collapsible-menu-header">About Wiley Online Library<span class="icon icon-chevron-down"></span></h3>
  <ul class="rlist collapsible-menu-body">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="collapsible-menu">
    <h3 data-bind="toggle" class="collapsible-menu-header">Help &amp; Support<span class="icon icon-chevron-down"></span></h3>
    <ul class="rlist collapsible-menu-body">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="collapsible-menu">
    <h3 data-bind="toggle" class="collapsible-menu-header">Opportunities<span class="icon icon-chevron-down"></span></h3>
    <ul class="rlist collapsible-menu-body">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="collapsible-menu">
    <h3 data-bind="toggle" class="collapsible-menu-header">Connect with Wiley<span class="icon icon-chevron-down"></span></h3>
    <ul class="rlist collapsible-menu-body">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2018 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="063fa3e6-c0a3-4672-a2e6-62aa1b09ed11" class="footer-logo pull-right">
        



        
        <a href="http://www.wiley.com/" title=""><img alt="" src="footer-logo-wiley-1510029248417.png"></a>

        </div>
    
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        














<div class="popup login-popup hidden">



    <div class="content">
        <a href="#" class="close"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2>Log in to Wiley Online Library</h2>

            



























<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        





<input name="redirectUri" value="/doi/10.1046/j.1365-294x.1998.00370.x" type="hidden">
<input name="loginUri" value="/doi/10.1046/j.1365-294x.1998.00370.x" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        
        
                


















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="login">Email or Customer ID</label>
                
        </div>
        <input id="login" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="text">
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                
















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" type="password">
        <span class="password-eye-icon icon-eye hidden"></span>
        
                    
                        
                            <div class="actions">
                                <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a>
                            </div>
                        
                    
                
    </div>

            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            <div class="submit" disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom" name="submit" value="Log In" disabled="disabled" type="submit">
                            </div>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            <div class="login-container clearfix form-btn">
                <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                    
                    NEW USER &gt;
                </a>

                
                    <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x" class="linkBtn float-right instituation-login">
                        
                        INSTITUTIONAL LOGIN &gt;
                    </a>
                
            </div>
        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="modal__header">


            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
            <h2>Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
            </div>
            <div class="success-template hidden">
                <h2>Congrats!</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        <div></div>




        
        





<div class="popup registration-popup hidden">



    <div class="content">
        <a href="#" class="close"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2>Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/10.1046/j.1365-294x.1998.00370.x" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login">
                    
                    Returning user
                </a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>

            <h2>Forgot your password?</h2>
            <p class="sub">Enter your email address below. If your address has been previously registered, you will
                receive an email with instructions on how to reset your password. If you don't receive an email,
                you should register as a new user</p>
        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                


















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">
                <p>Please check your email for your password reset instructions.</p>
            </div>
        </div>
    </div>
</div>





        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <a href="#" class="close"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script><script src="satellite-5aaec36164746d79c1001d14.js"></script>
        </div>
    </div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh955" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.5941208592118427&amp;iit=1534497491674&amp;tmr=load%3D1534497487950%26core%3D1534497487994%26main%3D1534497491670%26ifr%3D1534497491710&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x&amp;dt=To%20tree%20or%20not%20to%20tree%20-%20SMOUSE%20-%201998%20-%20Molecular%20Ecology%20-%20Wiley%20Online%20Library&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5b7692cffd0c4c8a&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=&amp;csi=undefined&amp;rev=v8.3.27-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>


<script type="text/javascript" src="main.js"></script>
<script type="text/javascript" src="f8d8product.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>

<div class="kxhead" data-id="JMOONjNh" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20226"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20227"></span><span class="kxtag kxinvisible" data-id="20229"><script>
(function() {
  // adadvisor redirects to http://logger... so it's not https safe
  if (location.protocol == "http:") {
     var u = "https://aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e";
     (new Image()).src = u;
   }
})();
</script></span><span class="kxtag kxinvisible" data-id="20234"><script type="text/javascript">
var findPh = document.getElementsByTagName("h1")[0]
if(findPh != null) { var pageheader = document.getElementsByTagName("h1")[0].innerHTML }

// Using Omniture prop1 to produce page attribute Page Type
Krux('set', 'page_attr_omniture_prop1', typeof window.s =='object' && window.s['prop1']);

// Using Omniture prop2 to produce page attribute Page Subtype
Krux('set', 'page_attr_omniture_prop2', typeof window.s =='object' && window.s['prop2']);

// Using Omniture prop3 to produce page attribute Title
Krux('set', 'page_attr_omniture_prop3', typeof window.s =='object' && window.s['prop3']);

// Using Omniture prop4 to produce page attribute Subject Codes
Krux('set', 'page_attr_omniture_prop4', typeof window.s =='object' && window.s['prop4']);

// Using Omniture prop7 to produce page attribute Article Title
Krux('set', 'page_attr_omniture_prop7', typeof window.s =='object' && window.s['prop7']);

// Using Omniture prop8 to produce page attribute Journal Volume and Issue
Krux('set', 'page_attr_omniture_prop8', typeof window.s =='object' && window.s['prop8']);

// Using Omniture prop9 to produce page attribute Article Access State
Krux('set', 'page_attr_omniture_prop9', typeof window.s =='object' && window.s['prop9']);

// Using Omniture prop10 to produce page attribute Publication Access Status
Krux('set', 'page_attr_omniture_prop10', typeof window.s =='object' && window.s['prop10']);

// Using Omniture prop13 to produce page attribute Journal DOI
Krux('set', 'page_attr_omniture_prop13', typeof window.s =='object' && window.s['prop13']);

// Using Omniture prop14 to produce page attribute Article / Chapter DOI
Krux('set', 'page_attr_omniture_prop14', typeof window.s =='object' && window.s['prop14']);

// Using Omniture prop15 to produce page attribute Issue DOI
Krux('set', 'page_attr_omniture_prop15', typeof window.s =='object' && window.s['prop15']);

// Using Omniture prop16 to produce page attribute MRW Chapter
Krux('set', 'page_attr_omniture_prop16', typeof window.s =='object' && window.s['prop16']);

// Using Omniture prop20 to produce page attribute Site Search Query
Krux('set', 'page_attr_omniture_prop20', typeof window.s =='object' && window.s['prop20']);

// Using Omniture prop26 to produce page attribute Logged In or Out
Krux('set', 'user_attr_omniture_prop26', typeof window.s =='object' && window.s['prop26']);

// Using Omniture channel to produce page attribute Channel
Krux('set', 'page_attr_omniture_channel', typeof window.s =='object' && window.s['channel']);

// Using Omniture prop33 to produce page attribute Primary Subject Code
Krux('set', 'page_attr_omniture_prop33', typeof window.s =='object' && window.s['prop33']);

// Using pageheader to produce page attribute Page Header
Krux('scrape', { 'page_attr_wileyonlinelib_pageheader': {javascript: 'pageheader'}});

</script></span><span class="kxtag kxinvisible" data-id="20238"></span><span class="kxtag kxinvisible" data-id="20251"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8da8b14d-5569-4bec-bcea-722864ee8d06&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20252"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bef9f122-393d-4c45-8d8d-32d8be7ac433&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iyzu39iyzud95&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20253"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';
 
            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
         
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8bf57916-aac8-4f01-a386-4baf103b3e1f&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=bckw15bckwu20&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20254"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };

        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20255"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=536f0daa-aaaa-4d9e-9a25-dde40646786a&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20256"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=7c6392c9-e878-492c-8b14-bf06e3828ebd&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rowp70rowpu60&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20257"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
        
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bb8ae0e2-9cd7-45b2-ad37-7737269627d8&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rsxs71rsxsk73&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><a target="_blank" href="http://www.crossref.org/crossmark"><img class="crossmark-reset crossmark-popup__logo"></a><button class="crossmark-reset crossmark-popup__btn-close"></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><div class="w-slide padding-top" style="top: 71px;"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1046/j.1365-294x.1998.00370.x?callback=_altmetric.embed_callback&amp;domain=onlinelibrary.wiley.com&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=12-17"></script><div id="SI_29r6u67C30RWd01"></div><script type="text/javascript" src="//zn8wxbnuhlfeu1aqt-wiley.siteintercept.qualtrics.com/WRSiteInterceptEngine/?Q_SIID=SI_29r6u67C30RWd01&amp;Q_LOC=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2F10.1046%2Fj.1365-294x.1998.00370.x&amp;t=1534497494690"></script><script src="https://siteintercept.qualtrics.com/WRSiteInterceptEngine/Asset.php?Module=Core&amp;Version=SIE.2018.17&amp;Q_CLIENTVERSION=SIE.2018.17&amp;Q_CLIENTTYPE=web" defer="defer"></script></body><iframe src="https://injections.readcube.com/html/track_epdf.3307bb36.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe><iframe id="kx-proxy-JMOONjNh" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=JMOONjNh&amp;kxt=https%3A%2F%2Fonlinelibrary.wiley.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>