<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script>

        <title>HyDe: A Python Package for Genome-Scale Hybridization Detection | Systematic Biology | Oxford Academic</title>

        <script type="text/javascript" async="" src="analytics.js"></script><script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>




    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site57462798.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'sysbio';
            App.SiteURL = 'academic.oup.com/sysbio';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/sysbio/article/67/5/821/4944070","@type":"ScholarlyArticle","name":"HyDe: A Python Package for Genome-Scale Hybridization Detection","datePublished":"2018-03-19","isPartOf":{"@id":"https://academic.oup.com/sysbio/issue/67/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-09-01","isPartOf":{"@id":"https://academic.oup.com/sysbio","@type":"Periodical","name":"Systematic Biology","issn":["1076-836X"]}},"url":"http://dx.doi.org/10.1093/sysbio/syy023","keywords":["ABBA-BABA","coalescence","gene flow","hybridization","phylogenetic invariants"],"inLanguage":"en","copyrightHolder":"Society of Systematic Biologists","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/sysbio/article/67/5/821/4944070","author":[{"name":"Blischak, Paul D","affiliation":"Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA","@type":"Person"},{"name":"Chifman, Julia","affiliation":"Department of Mathematics and Statistics, American University, Washington, DC 20016, USA","@type":"Person"},{"name":"Wolfe, Andrea D","affiliation":"Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA","@type":"Person"},{"name":"Kubatko, Laura S","affiliation":"Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA  Department of Statistics, The Ohio State University, Columbus, OH 43210, USA","@type":"Person"}],"description":"Abstract.  The analysis of hybridization and gene flow among closely related taxa is a common goal for researchers studying speciation and phylogeography. Many ","pageStart":"821","pageEnd":"829","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/67/5/3/m_sysbio_67_5cover.png?Expires=1637529744&Signature=LY1AEsR30WVdAO0HTeXklECp8-0DhPQIFHZ3V2S5jDE-uSD38h4EsVKg8CsX9ga7e-RdDgzgG1ahw3jvXmS4mPIkxIpXJ3mjEDD6epnNAG1GNMd4OIOyAWSfJPfRzSViTCjihKmBWWWHoeNxrM4foB0u-1Inyw7et6ZP5N6Oxzd9Xb~c~kAjk3sLmsL0aqvvICo2hpUGZh0Jp28HxHQSxbiz5g2xG3PXVStKWRcTkTatZ64FDDn8-SJ3Hr9aTeY5fo-OnaEUBxKTKbX-QSf8Q7I78cGi7SQ6bPQh7y-NnEI8Vv1PYM6pPqbv7P3Cw8SEXfVl~0CiDGF83gCc-7MRag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"HyDe: A Python Package for Genome-Scale Hybridization Detection","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/67/5/3/m_sysbio_67_5cover.png?Expires=1637529744&Signature=LY1AEsR30WVdAO0HTeXklECp8-0DhPQIFHZ3V2S5jDE-uSD38h4EsVKg8CsX9ga7e-RdDgzgG1ahw3jvXmS4mPIkxIpXJ3mjEDD6epnNAG1GNMd4OIOyAWSfJPfRzSViTCjihKmBWWWHoeNxrM4foB0u-1Inyw7et6ZP5N6Oxzd9Xb~c~kAjk3sLmsL0aqvvICo2hpUGZh0Jp28HxHQSxbiz5g2xG3PXVStKWRcTkTatZ64FDDn8-SJ3Hr9aTeY5fo-OnaEUBxKTKbX-QSf8Q7I78cGi7SQ6bPQh7y-NnEI8Vv1PYM6pPqbv7P3Cw8SEXfVl~0CiDGF83gCc-7MRag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="HyDe: A Python Package for Genome-Scale Hybridization Detection">
    <meta property="og:description" content="Abstract.  The analysis of hybridization and gene flow among closely related taxa is a common goal for researchers studying speciation and phylogeography. Many ">
    <meta property="og:url" content="https://academic.oup.com/sysbio/article/67/5/821/4944070">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/67/5/3/m_sysbio_67_5cover.png?Expires=1637529744&amp;Signature=LY1AEsR30WVdAO0HTeXklECp8-0DhPQIFHZ3V2S5jDE-uSD38h4EsVKg8CsX9ga7e-RdDgzgG1ahw3jvXmS4mPIkxIpXJ3mjEDD6epnNAG1GNMd4OIOyAWSfJPfRzSViTCjihKmBWWWHoeNxrM4foB0u-1Inyw7et6ZP5N6Oxzd9Xb~c~kAjk3sLmsL0aqvvICo2hpUGZh0Jp28HxHQSxbiz5g2xG3PXVStKWRcTkTatZ64FDDn8-SJ3Hr9aTeY5fo-OnaEUBxKTKbX-QSf8Q7I78cGi7SQ6bPQh7y-NnEI8Vv1PYM6pPqbv7P3Cw8SEXfVl~0CiDGF83gCc-7MRag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2018-03-19">
<meta name="citation_author" content="Blischak, Paul D"><meta name="citation_author_institution" content="Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA"><meta name="citation_author" content="Chifman, Julia"><meta name="citation_author_institution" content="Department of Mathematics and Statistics, American University, Washington, DC 20016, USA"><meta name="citation_author" content="Wolfe, Andrea D"><meta name="citation_author_institution" content="Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA"><meta name="citation_author" content="Kubatko, Laura S"><meta name="citation_author_institution" content="Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA"><meta name="citation_author_institution" content="Department of Statistics, The Ohio State University, Columbus, OH 43210, USA"><meta name="citation_title" content="HyDe: A Python Package for Genome-Scale Hybridization Detection"><meta name="citation_firstpage" content="821"><meta name="citation_lastpage" content="829"><meta name="citation_doi" content="10.1093/sysbio/syy023"><meta name="citation_journal_title" content="Systematic Biology"><meta name="citation_journal_abbrev" content="Syst Biol"><meta name="citation_volume" content="67"><meta name="citation_issue" content="5"><meta name="citation_publication_date" content="2018/09/01"><meta name="citation_issn" content="1063-5157"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Split scores: a tool to quantify phylogenetic signal in genome-scale data.;  citation_journal_title=Syst. Biol.;  citation_year=2016;  citation_volume=66;  citation_pages=620-636; "><meta name="citation_reference" content="citation_title=Adaptive genetic exchange: a tangled history of admixture and evolutionary innovation.;  citation_journal_title=Trends Ecol. Evol.;  citation_year=2017;  citation_volume=32;  citation_pages=601-611; "><meta name="citation_reference" content="citation_title=A genomic view of introgression and hybrid speciation.;  citation_journal_title=Curr. Opin. Genet. Dev.;  citation_year=2007;  citation_volume=17;  citation_pages=513-518; "><meta name="citation_reference" content="citation_title=Cython: the best of both worlds.;  citation_journal_title=Comput. Sci. Eng.;  citation_year=2010;  citation_volume=13;  citation_pages=31-39; "><meta name="citation_reference" content="citation_title=Invariants of phylogenies in a simple case with discrete states.;  citation_journal_title=J. Classification;  citation_year=1987;  citation_volume=4;  citation_pages=57-71; "><meta name="citation_reference" content="citation_title=Quartet inference from SNP data under the coalescent model.;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3317-3324; "><meta name="citation_reference" content="citation_title=Identifiability of the unrooted species tree topology under the coalescent model with time-reversible substitution processes, site-specific rate variation, and invariable sites.;  citation_journal_title=J. Theor. Biol.;  citation_year=2015;  citation_volume=374;  citation_pages=35-47; "><meta name="citation_reference" content="citation_title=Phylogenomics reveals extensive reticulate evolution in Xiphophorus fishes.;  citation_journal_title=Evolution;  citation_year=2013;  citation_volume=67;  citation_pages=2166-2179; "><meta name="citation_reference" content="citation_title=Testing for ancient admixture between closely related populations.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2239-2252; "><meta name="citation_reference" content="citation_title=Inferring phylogeny and introgression using RADseq data: an example from the flowering plants (Pedicularis: Orobanchaceae).;  citation_journal_title=Syst. Biol.;  citation_year=2013;  citation_volume=62;  citation_pages=689-706; "><meta name="citation_reference" content="citation_title=Estimating hybridization in the presence of coalescence using phylogenetic intraspecific sampling.;  citation_journal_title=BMC Evol. Biol.;  citation_year=2011;  citation_volume=11;  citation_pages=291"><meta name="citation_reference" content="citation_title=A draft sequence of the Neandertal genome.;  citation_journal_title=Science;  citation_year=2010;  citation_volume=328;  citation_pages=710-722; "><meta name="citation_reference" content="citation_title=Dating of human-ape splitting by a molecular clock of mitochondrial DNA.;  citation_journal_title=J. Mol. Evol.;  citation_year=1985;  citation_volume=22;  citation_pages=160-174; "><meta name="citation_reference" content="citation_title=Generating samples under a Wright-Fisher neutral model of genetic variation.;  citation_journal_title=Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=337-338; "><meta name="citation_reference" content="citation_title=A statistical approach for distinguishing hybridization and incomplete lineage sorting.;  citation_journal_title=Am. Nat.;  citation_year=2009;  citation_volume=174;  citation_pages=E54-E70; "><meta name="citation_reference" content="citation_title=Evolution of protein molecules.; citation_publisher=Academic Press., New York;  citation_title=Mammalian protein metabolism.;  citation_year=1969;  citation_pages=21-132; "><meta name="citation_reference" content="citation_title=Hybridization can promote adaptive radiation by means of transgressive segregation.;  citation_journal_title=Ecol. Lett.;  citation_year=2017;  citation_volume=21;  citation_pages=264-274; "><meta name="citation_reference" content="citation_title=Evaluating allopolyploid origins in strawberries (fragaria) using haplotypes generated from target capture sequencing.;  citation_journal_title=BMC Evol. Biol.;  citation_year=2017;  citation_volume=17;  citation_pages=180"><meta name="citation_reference" content="citation_title=On the genealogy of large populations.;  citation_journal_title=J. Appl. Prob.;  citation_year=1982;  citation_volume=19;  citation_pages=27-43; "><meta name="citation_reference" content="citation_title=Identifying hybridization events in the presence of coalescence via model selection.;  citation_journal_title=Syst. Biol.;  citation_year=2009;  citation_volume=58;  citation_pages=478-488; "><meta name="citation_reference" content="citation_title=An invariants-based method for hybridization detection from genome-scale sequence data.;  citation_year=2015; "><meta name="citation_reference" content="citation_title=A rate-independent technique for analysis of nucleic acid sequences: evolutionary parsimony.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=167-191; "><meta name="citation_reference" content="citation_title=Gene trees in species trees.;  citation_journal_title=Syst. Biol.;  citation_year=1997;  citation_volume=46;  citation_pages=523-536; "><meta name="citation_reference" content="citation_title=Hybridization as an invasion of the genome.;  citation_journal_title=Trends Ecol. Evol.;  citation_year=2005;  citation_volume=20;  citation_pages=229-237; "><meta name="citation_reference" content="citation_title=Hybrid speciation.;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=446;  citation_pages=279-283; "><meta name="citation_reference" content="citation_title=Genome-wide evidence for speciation with gene flow in Heliconius butterflies.;  citation_journal_title=Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=1817-1828; "><meta name="citation_reference" content="citation_title=Evaluating the use of ABBA-BABA statistics to locate introgressed loci.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2014;  citation_volume=32;  citation_pages=244-257; "><meta name="citation_reference" content="citation_title=Detecting hybrid speciation in the presence of incomplete lineage sorting using gene tree incongruence: a model.;  citation_journal_title=Theor. Popul. Biol.;  citation_year=2009;  citation_volume=75;  citation_pages=35-45; "><meta name="citation_reference" content="citation_title=Polyploid incidence and evolution.;  citation_journal_title=Ann. Rev. Genet.;  citation_year=2000;  citation_volume=34;  citation_pages=401-437; "><meta name="citation_reference" content="citation_title=Ancient admixture in human history.;  citation_journal_title=Genetics;  citation_year=2012;  citation_volume=192;  citation_pages=1065-1093; "><meta name="citation_reference" content="citation_title=Detection and polarization of introgression in a five-taxon phylogeny.;  citation_journal_title=Syst. Biol.;  citation_year=2015;  citation_volume=64;  citation_pages=651-662; "><meta name="citation_reference" content="citation_title=Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees.;  citation_journal_title=Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_pages=235-238; "><meta name="citation_reference" content="citation_title=Hybridization and adaptive radiation.;  citation_journal_title=Trends Ecol. Evol.;  citation_year=2004;  citation_volume=19;  citation_pages=198-207; "><meta name="citation_reference" content="citation_title=Inferring phylogenetic networks with maximum pseudolikelihood under incomplete lineage sorting.;  citation_journal_title=PLoS Genet.;  citation_year=2016;  citation_volume=12;  citation_pages=e1005896"><meta name="citation_reference" content="citation_title=Some probabilistic and statistical problems in the analysis of DNA sequences.;  citation_journal_title=Lectures on Mathematics in the Life Sciences;  citation_year=1986;  citation_volume=17;  citation_pages=57-86; "><meta name="citation_reference" content="citation_title=Rooting phylogenetic trees under the coalescent model using site pattern probabilities.;  citation_journal_title=BMC Evol. Biol.;  citation_year=2017;  citation_volume=17;  citation_pages=263"><meta name="citation_reference" content="citation_title=Parsimonious inference of hybridization in the presence of incomplete lineage sorting.;  citation_journal_title=Syst. Biol.;  citation_year=2013;  citation_volume=62;  citation_pages=738-751; "><meta name="citation_reference" content="citation_title=The probability of a gene tree topology within a phylogenetic network with applications to hybridization detection.;  citation_journal_title=PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002660"><meta name="citation_reference" content="citation_title=Maximum likelihood inference of reticulate evolutionary histories.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=16448-16453; "><meta name="citation_reference" content="citation_title=Coalescent histories on phylogenetic networks and detection of hybridization despite incomplete lineage sorting.;  citation_journal_title=Syst. Biol.;  citation_year=2011;  citation_volume=60;  citation_pages=138-149; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/sysbio/article-pdf/67/5/821/25517390/syy023.pdf"><meta name="description" content="Abstract.  The analysis of hybridization and gene flow among closely related taxa is a common goal for researchers studying speciation and phylogeography. Many ">    <link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019111801.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style></style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2.css"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/extensions/TeX/boldsymbol.js?V=2.7.1"></script></head><body data-sitestyletemplate="Journal" theme-sysbio="" class="off-canvas pg_Article pg_article" data-sitename="systematicbiology"><div id="MathJax_Message" style="">Processing math: 19%</div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/sysbio/article-xml/67/5/821/4944070">



    <link rel="canonical" href="https://academic.oup.com/sysbio/article/67/5/821/4944070">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/sysbio_Behind_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/sysbio_Behind_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/sysbio_Behind_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/sysbio_Behind_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "sysbio/article/67/5/821/4944070");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    
    

    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=sysbio&authzrequired=false&doi=10.1093/sysbio/syy023';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5349" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f67%2f5%2f821%2f4944070" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f67%2f5%2f821%2f4944070" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425934">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425935">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425932">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425936">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425937">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425938">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425939">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425933">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425940">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425941">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425942">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425943">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425944">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425945">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425946">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425947">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425948">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425949">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/sysbio" class="journal-logo-container">
                    <img id="logo-SystematicBiology" class="journal-logo" src="sysbio_title1175497244.svg" alt="Systematic Biology">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.systbio.org/" target="" class="society-logo-container">
                                <img id="logo-SocietyofSystematicBiologists" class="society-logo" src="sysbio_h1-360035926.svg" alt="Society of Systematic Biologists">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425934">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425935">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425932">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425936">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425937">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425938">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425939">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425933">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425940">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425941">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425942">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425943">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425944">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425945">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425946">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425947">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1425948">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1425949">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5349" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109722" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/sysbio" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="sysbio" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/sysbio/issue/67/5"><img id="issueImage" class="fb-featured-image" src="m_sysbio_67_5cover.png" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/sysbio/issue/67/5">
                        <div class="volume trailing-comma">Volume 67</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
September 2018        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="120230336">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230336">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230341">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230341">DESCRIPTION</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230371">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230371">BENCHMARKS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230389">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230389">BIOLOGICAL EXAMPLES</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230402">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230402">CONCLUSIONS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230404">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230404">AVAILABILITY</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230406">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230406">SUPPLEMENTARY MATERIALS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230408">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230408">FUNDING</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="120230410">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230410">ACKNOWLEDGEMENTS</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="120230412">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#120230412">References</a>
            </div>
        </li>
</ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/sysbio/article/67/5/800/4985806">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/sysbio/article/67/5/830/4866060">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    HyDe: A Python Package for Genome-Scale Hybridization Detection
<i class="icon-availability_free" title="Free"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paul D Blischak</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA</div>
        </div>


        <div class="info-author-correspondence">
            Correspondence to be sent: Department of Evolution, Ecology,
 and Organismal Biology, The Ohio State University, 318 W. 12th Avenue, 
Columbus, OH 43210, USA; E-mail: <a href="mailto:blischak.4@osu.edu" target="">blischak.4@osu.edu</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Paul+D+Blischak">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Blischak%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Blischak%20Paul%20D%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Paul D Blischak</a><span class="delimiter"><i class="icon-general-mail"></i>, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Julia Chifman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Mathematics and Statistics, American University, Washington, DC 20016, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Julia+Chifman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chifman%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Chifman%20Julia%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Julia Chifman</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrea D Wolfe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Andrea+D+Wolfe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wolfe%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wolfe%20Andrea%20D%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Andrea D Wolfe</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Laura S Kubatko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Evolution, Ecology, and Organismal Biology, The Ohio State University, Columbus, OH 43210, USA</div><div class="aff">Department of Statistics, The Ohio State University, Columbus, OH 43210, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Laura+S+Kubatko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kubatko%20L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kubatko%20Laura%20S%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Laura S Kubatko</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Systematic Biology</em>, Volume 67, Issue 5, September 2018, Pages 821829, <a href="https://doi.org/10.1093/sysbio/syy023">https://doi.org/10.1093/sysbio/syy023</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">19 March 2018</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">14 September 2017</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Revision received:</div>
                    <div class="wi-date">07 March 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">15 March 2018</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">19 March 2018</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4944070" href="https://academic.oup.com/sysbio/article-pdf/67/5/821/25517390/syy023.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Paul D Blischak, Julia Chifman, Andrea D Wolfe, Laura S Kubatko, 
 HyDe: A Python Package for Genome-Scale Hybridization Detection, <em>Systematic Biology</em>, Volume 67, Issue 5, September 2018, Pages 821829, <a href="https://doi.org/10.1093/sysbio/syy023">https://doi.org/10.1093/sysbio/syy023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=4944070&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p> 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1076-836X&amp;title=HyDe%3A%20A%20Python%20Package%20for%20Genome-Scale%20Hybridization%20Detection&amp;publicationDate=2018-03-19&amp;volumeNum=67&amp;issueNum=5&amp;author=Blischak%2C%20Paul%20D%3B%20Chifman%2C%20Julia&amp;startPage=821&amp;endPage=829&amp;contentId=10.1093%2Fsysbio%2Fsyy023&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f67%2f5%2f821%2f4944070" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="sysbio" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="120230336" id="120230336" class="abstract-title">Abstract</h2>
<section class="abstract"><p>The analysis of hybridization and gene flow
 among closely related taxa is a common goal for researchers studying 
speciation and phylogeography. Many methods for hybridization detection 
use simple site pattern frequencies from observed genomic data and 
compare them to null models that predict an absence of gene flow. The 
theory underlying the detection of hybridization using these site 
pattern probabilities exploits the relationship between the coalescent 
process for gene trees within population trees and the process of 
mutation along the branches of the gene trees. For certain models, site 
patterns are predicted to occur in equal frequency (i.e., their 
difference is 0), producing a set of functions called <em>phylogenetic invariants</em>.
 In this article, we introduce HyDe, a software package for detecting 
hybridization using phylogenetic invariants arising under the coalescent
 model with hybridization. HyDe is written in Python and can be used 
interactively or through the command line using pre-packaged scripts. We
 demonstrate the use of HyDe on simulated data, as well as on two 
empirical data sets from the literature. We focus in particular on 
identifying individual hybrids within population samples and on 
distinguishing between hybrid speciation and gene flow. HyDe is freely 
available as an open source Python package under the GNU GPL v3 on both 
GitHub (<a class="link link-uri" href="https://github.com/pblischak/HyDe" target="">https://github.com/pblischak/HyDe</a>) and the Python Package Index (PyPI: <a class="link link-uri" href="https://pypi.python.org/pypi/phyde" target="">https://pypi.python.org/pypi/phyde</a>).</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="ABBA-BABA">ABBA-BABA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="coalescence">coalescence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene flow&quot;">gene flow</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="hybridization">hybridization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic invariants&quot;">phylogenetic invariants</a></div><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/sysbio/search-results?f_TocHeadingTitle=Software%20for%20Systematics%20and%20Evolution">Software for Systematics and Evolution</a>
                </div><div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Associate Editor:
                            <span class="al-author-name">
                                <span id="info-18386989" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David Posada</div>
            <div class="info-card-role">
                Associate Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=David+Posada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Posada%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Posada%20David%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">David Posada</a>
                            </span>
                        <br>
                    </div>
            </div>
        </div></div>
<p>It is increasingly recognized that a strictly bifurcating model of 
population descent is inadequate to describe the evolutionary history of
 many species and is particularly common among plants, as well as some 
groups of animals (<a reveal-id="B24" data-open="B24" class="link link-ref link-reveal xref-bibr">Mallet 2005</a>; <a reveal-id="B3" data-open="B3" class="link link-ref link-reveal xref-bibr">Baack and Rieseberg 2007</a>; <a reveal-id="B25" data-open="B25" class="link link-ref link-reveal xref-bibr">Mallet 2007</a>).
 Hybridization and gene flow are processes that obscure this simple 
model, but separating the signal of admixture from other sources of 
incongruence, such as incomplete lineage sorting (ILS), is especially 
difficult (<a reveal-id="B23" data-open="B23" class="link link-ref link-reveal xref-bibr">Maddison 1997</a>).
 Early methods for detecting hybridization in the presence of ILS used 
estimated gene trees to detect deviations from the coalescent model 
under the expectation of a bifurcating species tree (<a reveal-id="B15" data-open="B15" class="link link-ref link-reveal xref-bibr">Joly et al. 2009</a>; <a reveal-id="B20" data-open="B20" class="link link-ref link-reveal xref-bibr">Kubatko 2009</a>; <a reveal-id="B28" data-open="B28" class="link link-ref link-reveal xref-bibr">Meng and Kubatko 2009</a>; <a reveal-id="B11" data-open="B11" class="link link-ref link-reveal xref-bibr">Gerard et al. 2011</a>). This work was later extended to include searches over network space to infer species networks with reticulate edges (<a reveal-id="B40" data-open="B40" class="link link-ref link-reveal xref-bibr">Yu et al. 2011</a>, <a reveal-id="B38" data-open="B38" class="link link-ref link-reveal xref-bibr">2012</a>, <a reveal-id="B37" data-open="B37" class="link link-ref link-reveal xref-bibr">2013</a>, <a reveal-id="B39" data-open="B39" class="link link-ref link-reveal xref-bibr">2014</a>; <a reveal-id="B34" data-open="B34" class="link link-ref link-reveal xref-bibr">Sols-Lumis and An 2016</a>).
 Because these network inference methods do not always scale to large 
numbers of species, other approaches for detecting hybridization that 
use genome-wide single nucleotide polymorphism (SNP) data to test for 
admixture on rooted, four- or five-taxon trees have often been employed 
(ABBA-BABA-like methods; <a reveal-id="B12" data-open="B12" class="link link-ref link-reveal xref-bibr">Green et al. 2010</a>; <a reveal-id="B9" data-open="B9" class="link link-ref link-reveal xref-bibr">Durand et al. 2011</a>; <a reveal-id="B30" data-open="B30" class="link link-ref link-reveal xref-bibr">Patterson et al. 2012</a>; <a reveal-id="B27" data-open="B27" class="link link-ref link-reveal xref-bibr">Martin et al. 2014</a>; <a reveal-id="B10" data-open="B10" class="link link-ref link-reveal xref-bibr">Eaton and Ree 2013</a>; <a reveal-id="B31" data-open="B31" class="link link-ref link-reveal xref-bibr">Pease and Hahn 2015</a>).
 A common feature of these genome-wide methods for hybridization 
detection is their use of site patterns to test for deviations from the 
expected frequency under a neutral coalescent model with no 
introgression (<a reveal-id="B12" data-open="B12" class="link link-ref link-reveal xref-bibr">Green et al. 2010</a>; <a reveal-id="B9" data-open="B9" class="link link-ref link-reveal xref-bibr">Durand et al. 2011</a>).</p><p>The
 theoretical underpinnings of these site pattern-based inference methods
 originate from the study of invariants: functions of site pattern 
probabilities that contain information about the underlying 
relationships of the sampled species. For example, the D-statistic (<a reveal-id="B30" data-open="B30" class="link link-ref link-reveal xref-bibr">Patterson et al. 2012</a>)
 is based on invariants and uses the site patterns ABBA and BABA, which 
should occur in equal frequency in the absence of admixture. The 
earliest use of invariants to infer phylogenies was a pair of articles 
from <a reveal-id="B5" data-open="B5" class="link link-ref link-reveal xref-bibr">Cavender and Felsenstein (1987)</a> and <a reveal-id="B22" data-open="B22" class="link link-ref link-reveal xref-bibr">Lake (1987)</a>,
 who separately derived functions to determine the correct topology for 
an unrooted quartet using binary and nucleotide data, respectively. 
Recent applications of phylogenetic invariants include proving 
identifiability (<a reveal-id="B7" data-open="B7" class="link link-ref link-reveal xref-bibr">Chifman and Kubatko 2015</a>), coalescent-based species tree inference (SVDquartets; <a reveal-id="B6" data-open="B6" class="link link-ref link-reveal xref-bibr">Chifman and Kubatko 2014</a>), sliding-window analyses of phylogenetic bipartitions (SplitSup; <a reveal-id="B1" data-open="B1" class="link link-ref link-reveal xref-bibr">Allman et al. 2016</a>), and the detection of hybridization (<a reveal-id="B12" data-open="B12" class="link link-ref link-reveal xref-bibr">Green et al. 2010</a>; <a reveal-id="B9" data-open="B9" class="link link-ref link-reveal xref-bibr">Durand et al. 2011</a>; <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman 2015</a>).
 The increased interest in methods using invariants is concomitant with 
the ability to collect genomic sequence data, allowing for accurate 
estimates of site pattern frequencies. Furthermore, because these 
methods are based on site pattern frequencies, they offer 
computationally tractable approaches for analyzing genome-scale data 
sets.</p><p>In this article, we introduce HyDe, a Python package for the
 detection of hybridization using phylogenetic invariants. HyDe 
automates the detection of hybridization across large numbers of species
 and can conduct hypothesis tests at both the population and individual 
levels. A particular advantage of HyDe over other methods is its ability
 to assess individual-level variation in the amount of hybridization 
using per-individual hypothesis tests and individual bootstrap 
resampling in putative hybrid populations. The programming philosophy 
behind HyDe is to provide a low-level library of data structures and 
methods for computing site pattern probabilities and conducting 
hypothesis tests, along with a core set of Python scripts that use this 
library to automatically parse and analyze genomic data. This setup 
allows our software to be easily extended as new methods based on site 
pattern probabilities are developed. We describe the available features 
of the software in detail below and demonstrate the use of HyDe on both 
simulated and empirical data sets. To facilitate the use of HyDe for 
other researchers, we have provided all of the code for processing, 
plotting, and interpreting the results of these analyses.</p>                    <h2 scrollto-destination="120230341" id="120230341" class="section-title" data-legacy-id="SEC1">DESCRIPTION</h2>
                    <h3 scrollto-destination="120230342" id="120230342" class="section-title" data-legacy-id="SEC1.1">Model Background</h3>
<p>In this section, we provide a brief overview of the model used by 
HyDe for detecting hybridization. The theory behind the model, the 
derivation of the test statistics and corresponding asymptotic 
distributions, as well as assessments of statistical power and 
performance for different simulation settings can be found in <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman (2015)</a>.</p><p>Consider
 a rooted, four-taxon network consisting of an outgroup and a triplet of
 ingroup populations: two parental populations (P1 and P2) and a hybrid 
population (Hyb) that is a mixture of P1 and P2 (<a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Under this model, gene trees arise within the parental population trees following the coalescent process (<a reveal-id="B19" data-open="B19" class="link link-ref link-reveal xref-bibr">Kingman 1982</a>), where the hybrid population is either (1) sister to P1 with probability <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-1"><span class="MJXp-mn" id="MJXp-Span-2">1</span><span class="MJXp-mo" id="MJXp-Span-3" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-4"></span></span></span><script type="math/tex" id="MathJax-Element-1">1-\gamma</script></span> or (2) sister to P2 with probability <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-5"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-6"></span></span></span><script type="math/tex" id="MathJax-Element-2">\gamma</script></span> (<a reveal-id="B28" data-open="B28" class="link link-ref link-reveal xref-bibr">Meng and Kubatko 2009</a>).
 Mutations are then placed on these gene trees according to the standard
 Markov models of nucleotide substitution [e.g., JC69 (<a reveal-id="B16" data-open="B16" class="link link-ref link-reveal xref-bibr">Jukes and Cantor 1969</a>), HKY85 (<a reveal-id="B13" data-open="B13" class="link link-ref link-reveal xref-bibr">Hasegawa et al. 1985</a>), GTR (<a reveal-id="B35" data-open="B35" class="link link-ref link-reveal xref-bibr">Tavar 1986</a>)].
 Given one sampled individual within each population, we can describe a 
probability distribution on the possible site patterns observed at the 
tips of the gene trees: <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-7"><span class="MJXp-msubsup" id="MJXp-Span-8"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-9" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-10" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-11">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-12">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-13">k</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-14">l</span></span></span><span class="MJXp-mo" id="MJXp-Span-15" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-16">P</span><span class="MJXp-mo" id="MJXp-Span-17" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mtext" id="MJXp-Span-18">O</span><span class="MJXp-mo" id="MJXp-Span-19" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-20">i</span><span class="MJXp-mo" id="MJXp-Span-21" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mtext" id="MJXp-Span-22">P1</span><span class="MJXp-mo" id="MJXp-Span-23" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-24">j</span><span class="MJXp-mo" id="MJXp-Span-25" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mtext" id="MJXp-Span-26">Hyb</span><span class="MJXp-mo" id="MJXp-Span-27" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-28">k</span><span class="MJXp-mo" id="MJXp-Span-29" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mtext" id="MJXp-Span-30">P2</span><span class="MJXp-mo" id="MJXp-Span-31" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-32">l</span><span class="MJXp-mo" id="MJXp-Span-33" style="margin-left: 0em; margin-right: 0em;">)</span></span></span><script type="math/tex" id="MathJax-Element-3">p_{ijkl} = P(\text{O} = i, \text{P1} = j, \text{Hyb} = k, \text{P2} = l)</script></span>, with <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-34"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-35">i</span><span class="MJXp-mo" id="MJXp-Span-36" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-37">j</span><span class="MJXp-mo" id="MJXp-Span-38" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-39">k</span><span class="MJXp-mo" id="MJXp-Span-40" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-41">l</span><span class="MJXp-mo" id="MJXp-Span-42" style="margin-left: 0.333em; margin-right: 0.333em;"></span><span class="MJXp-mo" id="MJXp-Span-43" style="margin-left: 0em; margin-right: 0em;">{</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-44">A</span><span class="MJXp-mo" id="MJXp-Span-45" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-46">G</span><span class="MJXp-mo" id="MJXp-Span-47" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-48">C</span><span class="MJXp-mo" id="MJXp-Span-49" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-50">T</span><span class="MJXp-mo" id="MJXp-Span-51" style="margin-left: 0em; margin-right: 0em;">}</span></span></span><script type="math/tex" id="MathJax-Element-4">i,j,k,l \in \{A,G,C,T\}</script></span>. Using nucleotide sequence data, we can also estimate these probabilities using the observed site patterns to get <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-52"><span class="MJXp-msubsup" id="MJXp-Span-53"><span class="MJXp-mrow" id="MJXp-Span-54" style="margin-right: 0.05em;"><span class="MJXp-munderover" id="MJXp-Span-55"><span><span class="MJXp-over"><span class="" style="margin-bottom: -1.17em;"><span class="MJXp-mo" id="MJXp-Span-57" style="margin-left: 0px; margin-right: 0px;"></span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-56">p</span></span></span></span></span></span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-58" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-59">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-60">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-61">k</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-62">l</span></span></span><span class="MJXp-mo" id="MJXp-Span-63" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-64" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-msubsup" id="MJXp-Span-65"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-66" style="margin-right: 0.05em;">X</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-67" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-68">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-69">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-70">k</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-71">l</span></span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-72">S</span></span></span></span></span></span></span></span><script type="math/tex" id="MathJax-Element-5">\hat{p}_{ijkl} = \frac{X_{ijkl}}{S}</script></span>, where <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-73"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-74">S</span></span></span><script type="math/tex" id="MathJax-Element-6">S</script></span> is the total number of observed sites and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-75"><span class="MJXp-msubsup" id="MJXp-Span-76"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-77" style="margin-right: 0.05em;">X</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-78" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-79">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-80">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-81">k</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-82">l</span></span></span></span></span><script type="math/tex" id="MathJax-Element-7">X_{ijkl}</script></span> is the number of times that site pattern <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-83"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-84">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-85">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-86">k</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-87">l</span></span></span><script type="math/tex" id="MathJax-Element-8">ijkl</script></span>
 was observed. Implicit in this model is the assumption that each site 
evolves along its own gene tree, making unlinked sites the most 
appropriate input data. However, when the number of genes is large, it 
has been shown through simulations and empirical analyses that 
multilocus data provide a good approximation for the model and can be 
used to calculate site pattern probabilities (<a reveal-id="B6" data-open="B6" class="link link-ref link-reveal xref-bibr">Chifman and Kubatko 2014</a>; <a reveal-id="B36" data-open="B36" class="link link-ref link-reveal xref-bibr">Tian and Kubatko 2017</a>).</p>                        <a id="120230345" scrollto-destination="120230345"></a>
<div id="F1" data-id="F1" class="fig fig-section"><div class="label fig-label">Fig. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f1.png" alt="Illustration of the model for detecting hybridization using HyDe. The hybrid population (Hyb) is modeled as a mixture between two parental populations, P1 (1-$\gamma$) and P2 ($\gamma$)." path-from-xml="syy023f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230345/syy023f1.tif" path-from-xml="syy023f1.tif" target="_blank">Open in new tab</a><a section="120230345" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f1.png?Expires=1577481743&amp;Signature=xaWtorwAsE0g9-FJj%7Eu7Y8jNMjyG274s7JpzmdfOCrhSUkLGsvSaC3RtKZEYlMGngDxO2eCC-4ng3K1aI1oSOAmIsVd0MQqfSxh2MzLPxgCrCcfRUqZs2FzVjv45bjI8CxKmxKag1FPb1Zw1pMBh7fFEQnjmCoXJPGn8fzykWiM9ZMzku--TdJq6MQ5hWf-4kiqW5UPc%7EK5IQ6pqzxPUAG4BGI8ggWOl2cgz82dqRGbBwWTKRUlb7h1vvfVQSBgpBlXdyT7rzdL-D%7EYVhyI4mnH2AlO1PcKoqRcFBy8HOSPb1mo%7EwrRMsRfYMKVEFRVVAkQqbPNjgKeEHLGiNC9STA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230345&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration
 of the model for detecting hybridization using HyDe. The hybrid 
population (Hyb) is modeled as a mixture between two parental 
populations, P1 (1-<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-88"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-89"></span></span></span><script type="math/tex" id="MathJax-Element-9">\gamma</script></span>) and P2 (<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-90"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-91"></span></span></span><script type="math/tex" id="MathJax-Element-10">\gamma</script></span>).</p></div></div><div content-id="F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f1.png" alt="Illustration of the model for detecting hybridization using HyDe. The hybrid population (Hyb) is modeled as a mixture between two parental populations, P1 (1-$\gamma$) and P2 ($\gamma$)." path-from-xml="syy023f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230345/syy023f1.tif" path-from-xml="syy023f1.tif" target="_blank">Open in new tab</a><a section="120230345" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f1.png?Expires=1577481743&amp;Signature=xaWtorwAsE0g9-FJj%7Eu7Y8jNMjyG274s7JpzmdfOCrhSUkLGsvSaC3RtKZEYlMGngDxO2eCC-4ng3K1aI1oSOAmIsVd0MQqfSxh2MzLPxgCrCcfRUqZs2FzVjv45bjI8CxKmxKag1FPb1Zw1pMBh7fFEQnjmCoXJPGn8fzykWiM9ZMzku--TdJq6MQ5hWf-4kiqW5UPc%7EK5IQ6pqzxPUAG4BGI8ggWOl2cgz82dqRGbBwWTKRUlb7h1vvfVQSBgpBlXdyT7rzdL-D%7EYVhyI4mnH2AlO1PcKoqRcFBy8HOSPb1mo%7EwrRMsRfYMKVEFRVVAkQqbPNjgKeEHLGiNC9STA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230345&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration
 of the model for detecting hybridization using HyDe. The hybrid 
population (Hyb) is modeled as a mixture between two parental 
populations, P1 (1-<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-92"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-93"></span></span></span><script type="math/tex" id="MathJax-Element-11">\gamma</script></span>) and P2 (<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-94"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-95"></span></span></span><script type="math/tex" id="MathJax-Element-12">\gamma</script></span>).</p></div></div><div class="block-child-p">These
 site pattern probabilities form the basis of the test for hybridization
 that is implemented in HyDe, as well as several other methods. As we 
noted before, the D-statistic (<a reveal-id="B30" data-open="B30" class="link link-ref link-reveal xref-bibr">Patterson et al. 2012</a>) is based on invariants and uses the site pattern probabilities <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-96"><span class="MJXp-msubsup" id="MJXp-Span-97"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-98" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-99" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-100">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-101">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-102">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-103">i</span></span></span></span></span><script type="math/tex" id="MathJax-Element-13">p_{ijji}</script></span> and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-104"><span class="MJXp-msubsup" id="MJXp-Span-105"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-106" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-107" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-108">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-109">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-110">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-111">j</span></span></span></span></span><script type="math/tex" id="MathJax-Element-14">p_{ijij}</script></span> to test for hybridization assuming a null model of no admixture: <a id="syy023M1" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syy023M1"><div class="tex-math display-math"> <span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math MJXp-display" id="MJXp-Span-112"><span class="MJXp-mrow" id="MJXp-Span-113"><span class="MJXp-mi MJXp-bold" id="MJXp-Span-114">D</span></span><span class="MJXp-mo" id="MJXp-Span-115" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-116" style="vertical-align: 0.25em;"><span class="MJXp-box"><span class="MJXp-msubsup" id="MJXp-Span-117"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-118" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-119" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-120">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-121">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-122">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-123">i</span></span></span><span class="MJXp-mo" id="MJXp-Span-124" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-125"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-126" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-127" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-128">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-129">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-130">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-131">j</span></span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box"><span class="MJXp-msubsup" id="MJXp-Span-132"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-133" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-134" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-135">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-136">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-137">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-138">i</span></span></span><span class="MJXp-mo" id="MJXp-Span-139" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-msubsup" id="MJXp-Span-140"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-141" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-142" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-143">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-144">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-145">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-146">j</span></span></span></span></span></span></span></span><span class="MJXp-mspace" id="MJXp-Span-147" style="width: 0.167em; height: 0em;"></span><span class="MJXp-mo" id="MJXp-Span-148" style="margin-left: 0em; margin-right: 0.222em;">.</span></span></span><script type="math/tex; mode=display" id="MathJax-Element-15">\begin{equation}
\mathbf{D} = \frac{p_{ijji} - p_{ijij}}{p_{ijji} + p_{ijij}}\,.
\end{equation}</script></div></div><div class="label">(1)</div></div></div><p>When admixture is absent, the expected value of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-149"><span class="MJXp-mrow" id="MJXp-Span-150"><span class="MJXp-mi MJXp-bold" id="MJXp-Span-151">D</span></span></span></span><script type="math/tex" id="MathJax-Element-16">\mathbf{D}</script></span> is 0, indicating that <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-152"><span class="MJXp-msubsup" id="MJXp-Span-153"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-154" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-155" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-156">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-157">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-158">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-159">i</span></span></span></span></span><script type="math/tex" id="MathJax-Element-17">p_{ijji}</script></span> and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-160"><span class="MJXp-msubsup" id="MJXp-Span-161"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-162" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-163" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-164">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-165">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-166">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-167">j</span></span></span></span></span><script type="math/tex" id="MathJax-Element-18">p_{ijij}</script></span> should occur in equal frequency (i.e., they are invariant).</p><div class="block-child-p">In a similar way, <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman (2015)</a>
 presented several different invariants-based test statistics that 
detect hybridization using linear phylogenetic invariants. Among the 
invariants that they tested, the ratio of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-168"><span class="MJXp-msubsup" id="MJXp-Span-169"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-170" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-171" style="vertical-align: -0.4em;">1</span></span><span class="MJXp-mo" id="MJXp-Span-172" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-msubsup" id="MJXp-Span-173"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-174" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-175" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-176">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-177">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-178">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-179">j</span></span></span><span class="MJXp-mo" id="MJXp-Span-180" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-181"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-182" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-183" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-184">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-185">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-186">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-187">j</span></span></span></span></span><script type="math/tex" id="MathJax-Element-19">f_1 = p_{iijj} - p_{ijij}</script></span> and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-188"><span class="MJXp-msubsup" id="MJXp-Span-189"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-190" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-191" style="vertical-align: -0.4em;">2</span></span><span class="MJXp-mo" id="MJXp-Span-192" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-msubsup" id="MJXp-Span-193"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-194" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-195" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-196">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-197">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-198">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-199">i</span></span></span><span class="MJXp-mo" id="MJXp-Span-200" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-201"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-202" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-203" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-204">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-205">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-206">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-207">j</span></span></span></span></span><script type="math/tex" id="MathJax-Element-20">f_2 = p_{ijji} - p_{ijij}</script></span>
 provided the most statistical power and is used to form the test 
statistic that is used by HyDe to detect hybridization. When the model 
holds, it can be shown that <a id="syy023M2" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syy023M2"><div class="tex-math display-math"> <span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math MJXp-display" id="MJXp-Span-208"><span class="MJXp-mfrac" id="MJXp-Span-209" style="vertical-align: 0.25em;"><span class="MJXp-box"><span class="MJXp-msubsup" id="MJXp-Span-210"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-211" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-212" style="vertical-align: -0.4em;">1</span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box"><span class="MJXp-msubsup" id="MJXp-Span-213"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-214" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-215" style="vertical-align: -0.4em;">2</span></span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-216" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-217" style="vertical-align: 0.25em;"><span class="MJXp-box"><span class="MJXp-msubsup" id="MJXp-Span-218"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-219" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-220" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-221">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-222">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-223">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-224">j</span></span></span><span class="MJXp-mo" id="MJXp-Span-225" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-226"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-227" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-228" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-229">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-230">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-231">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-232">j</span></span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box"><span class="MJXp-msubsup" id="MJXp-Span-233"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-234" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-235" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-236">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-237">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-238">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-239">i</span></span></span><span class="MJXp-mo" id="MJXp-Span-240" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-241"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-242" style="margin-right: 0.05em;">p</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-243" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-244">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-245">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-246">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-247">j</span></span></span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-248" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-249" style="vertical-align: 0.25em;"><span class="MJXp-box"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-250"></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box"><span class="MJXp-mn" id="MJXp-Span-251">1</span><span class="MJXp-mo" id="MJXp-Span-252" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-253"></span></span></span></span></span></span><span class="MJXp-mspace" id="MJXp-Span-254" style="width: 0.167em; height: 0em;"></span><span class="MJXp-mo" id="MJXp-Span-255" style="margin-left: 0em; margin-right: 0.222em;">.</span></span></span><script type="math/tex; mode=display" id="MathJax-Element-21">\begin{equation}
\frac{f_1}{f_2} = \frac{p_{iijj} - p_{ijij}}{p_{ijji} - p_{ijij}} = \frac{\gamma}{1 - \gamma}\,.
\end{equation}</script></div></div><div class="label">(2)</div></div></div><div class="block-child-p">Deviation
 of the observed frequencies from this expectation is used to form a 
test statistic that asymptotically follows a standard Normal 
distribution, allowing formal hypothesis tests to be conducted (<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-256"><span class="MJXp-msubsup" id="MJXp-Span-257"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-258" style="margin-right: 0.05em;">H</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-259" style="vertical-align: -0.4em;">0</span></span><span class="MJXp-mo" id="MJXp-Span-260" style="margin-left: 0.111em; margin-right: 0.167em;">:</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-261"></span><span class="MJXp-mo" id="MJXp-Span-262" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-263">0</span></span></span><script type="math/tex" id="MathJax-Element-22">H_0: \gamma=0</script></span> vs. <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-264"><span class="MJXp-msubsup" id="MJXp-Span-265"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-266" style="margin-right: 0.05em;">H</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-267" style="vertical-align: -0.4em;">1</span></span><span class="MJXp-mo" id="MJXp-Span-268" style="margin-left: 0.111em; margin-right: 0.167em;">:</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-269"></span><span class="MJXp-mo" id="MJXp-Span-270" style="margin-left: 0.333em; margin-right: 0.333em;">&gt;</span><span class="MJXp-mn" id="MJXp-Span-271">0</span></span></span><script type="math/tex" id="MathJax-Element-23">H_1: \gamma > 0</script></span>). Furthermore, because the ratio of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-272"><span class="MJXp-msubsup" id="MJXp-Span-273"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-274" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-275" style="vertical-align: -0.4em;">1</span></span></span></span><script type="math/tex" id="MathJax-Element-24">f_1</script></span> and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-276"><span class="MJXp-msubsup" id="MJXp-Span-277"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-278" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-279" style="vertical-align: -0.4em;">2</span></span></span></span><script type="math/tex" id="MathJax-Element-25">f_2</script></span> is a function of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-280"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-281"></span></span></span><script type="math/tex" id="MathJax-Element-26">\gamma</script></span>, we can estimate the amount of admixture directly from the observed site pattern frequencies: <a id="syy023M3" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syy023M3"><div class="tex-math display-math"> <span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math MJXp-display" id="MJXp-Span-282"><span class="MJXp-mrow" id="MJXp-Span-283"><span class="MJXp-munderover" id="MJXp-Span-284"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-286" style="margin-left: 0px; margin-right: 0px;"></span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-285"></span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-287" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-288" style="vertical-align: 0.25em;"><span class="MJXp-box"><span class="MJXp-mrow" id="MJXp-Span-289"><span class="MJXp-munderover" id="MJXp-Span-290"><span><span class="MJXp-over"><span class=""><span class="MJXp-mo" id="MJXp-Span-294" style="margin-left: 0px; margin-right: 0px;">^</span></span><span class=""><span class="MJXp-msubsup" id="MJXp-Span-291"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-292" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-293" style="vertical-align: -0.4em;">1</span></span></span></span></span></span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box"><span class="MJXp-mrow" id="MJXp-Span-295"><span class="MJXp-munderover" id="MJXp-Span-296"><span><span class="MJXp-over"><span class=""><span class="MJXp-mo" id="MJXp-Span-300" style="margin-left: 0px; margin-right: 0px;">^</span></span><span class=""><span class="MJXp-msubsup" id="MJXp-Span-297"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-298" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-299" style="vertical-align: -0.4em;">1</span></span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-301" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mrow" id="MJXp-Span-302"><span class="MJXp-munderover" id="MJXp-Span-303"><span><span class="MJXp-over"><span class=""><span class="MJXp-mo" id="MJXp-Span-307" style="margin-left: 0px; margin-right: 0px;">^</span></span><span class=""><span class="MJXp-msubsup" id="MJXp-Span-304"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-305" style="margin-right: 0.05em;">f</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-306" style="vertical-align: -0.4em;">2</span></span></span></span></span></span></span></span></span></span></span></span><span class="MJXp-mspace" id="MJXp-Span-308" style="width: 0.167em; height: 0em;"></span><span class="MJXp-mo" id="MJXp-Span-309" style="margin-left: 0em; margin-right: 0.222em;">.</span></span></span><script type="math/tex; mode=display" id="MathJax-Element-27">\begin{equation}
\hat{\gamma} = \frac{\hat{f_1}}{\hat{f_1} + \hat{f_2}} \,.
\end{equation}</script></div></div><div class="label">(3)</div></div></div>                    <h3 scrollto-destination="120230350" id="120230350" class="section-title" data-legacy-id="SEC1.2">Software Features</h3>
<p>In this section, we outline the main features of the HyDe software. Previously, <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman (2015)</a>
 used C code to test their new methods and distributed it in accordance 
with standard publishing practices. However, the code was only intended 
to evaluate performance for their method of hybridization detection, and
 not as a distribution-level piece of software. The new Python package
 that we present here serves that purpose, and provides additional 
functionality to accommodate more complex hybridization scenarios and 
sampling designs.</p>                    <h4 scrollto-destination="120230352" id="120230352" class="section-title" data-legacy-id="SEC1.2.1">Multiple individuals per population.</h4>
<p>We have modified the calculation of the site pattern probabilities originally presented in <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman (2015)</a>
 to accommodate multiple individuals per population, rather than testing
 each sampled individual separately. We do this by considering all 
permutations of the individuals in the four populations involved in the 
hypothesis test. For example, if there are <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-310"><span class="MJXp-msubsup" id="MJXp-Span-311"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-312" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-313" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-314">O</span></span></span></span></span><script type="math/tex" id="MathJax-Element-28">N_{O}</script></span> individuals in the outgroup, <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-315"><span class="MJXp-msubsup" id="MJXp-Span-316"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-317" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-318" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-319">P</span><span class="MJXp-mn" id="MJXp-Span-320">1</span></span></span></span></span><script type="math/tex" id="MathJax-Element-29">N_{P1}</script></span> individuals in parental population one, <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-321"><span class="MJXp-msubsup" id="MJXp-Span-322"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-323" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-324" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-325">H</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-326">y</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-327">b</span></span></span></span></span><script type="math/tex" id="MathJax-Element-30">N_{Hyb}</script></span> individuals in the putative hybrid population, and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-328"><span class="MJXp-msubsup" id="MJXp-Span-329"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-330" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-331" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-332">P</span><span class="MJXp-mn" id="MJXp-Span-333">2</span></span></span></span></span><script type="math/tex" id="MathJax-Element-31">N_{P2}</script></span> individuals in parental population two, then a total of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-334"><span class="MJXp-msubsup" id="MJXp-Span-335"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-336" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-337" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-338">O</span></span></span><span class="MJXp-mo" id="MJXp-Span-339" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-340"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-341" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-342" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-343">P</span><span class="MJXp-mn" id="MJXp-Span-344">1</span></span></span><span class="MJXp-mo" id="MJXp-Span-345" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-346"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-347" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-348" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-349">H</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-350">y</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-351">b</span></span></span><span class="MJXp-mo" id="MJXp-Span-352" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-msubsup" id="MJXp-Span-353"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-354" style="margin-right: 0.05em;">N</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-355" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-356">P</span><span class="MJXp-mn" id="MJXp-Span-357">2</span></span></span></span></span><script type="math/tex" id="MathJax-Element-32">N_{O} \times N_{P1} \times N_{Hyb} \times N_{P2}</script></span>
 quartets are used to calculate the site patterns for the hypothesis 
test. Including multiple individuals per population increases the sample
 size used to calculate the site pattern probabilities and can therefore
 lead to more accurate detection of hybridization.</p>                    <h4 scrollto-destination="120230354" id="120230354" class="section-title" data-legacy-id="SEC1.2.2">Identifying individual hybrids.</h4>
<p>An underlying assumption of the coalescent with hybridization is that
 all the individuals in the hybrid population are admixed (i.e., hybrid 
speciation; <a reveal-id="B28" data-open="B28" class="link link-ref link-reveal xref-bibr">Meng and Kubatko 2009</a>).
 However, when gene flow, rather than hybrid speciation, is responsible 
for the introgression of genetic material into the admixed population, 
it is possible that not all individuals will contain these introgressed 
alleles. When hybridization detection is conducted on populations with a
 mix of hybrids and non-hybrids, it is possible for invariant-based 
tests to report significant results, even though the underlying 
assumption of uniform admixture is violated. To help with the detection 
of non-uniform introgression into the hybrid population, we have 
included two methods in HyDe that aim to detect variation in the amount 
of hybridization in the individuals sampled.</p><p>If not all of the 
individuals in a putative hybrid population are admixed, bootstrap 
resampling can reveal heterogeneity in the process of introgression when
 more or fewer hybrid individuals are included in each replicate. For 
example, if we are testing four diploid individuals in a putative hybrid
 and only two of them are 50:50 hybrids (<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-358"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-359"></span></span></span><script type="math/tex" id="MathJax-Element-33">\gamma</script></span> = 0.5 for each), then the value of gamma for the whole population will be <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-360"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-361"></span><span class="MJXp-mo" id="MJXp-Span-362" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-363">0.25</span></span></span><script type="math/tex" id="MathJax-Element-34">\gamma = 0.25</script></span>. When we resample individuals with replacement and recalculate <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-364"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-365"></span></span></span><script type="math/tex" id="MathJax-Element-35">\gamma</script></span>, we will get different values depending on how many times the hybrids are resampled (0 times: <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-366"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-367"></span><span class="MJXp-mo" id="MJXp-Span-368" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-369">0.0</span></span></span><script type="math/tex" id="MathJax-Element-36">\gamma = 0.0</script></span>; 1 time: <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-370"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-371"></span><span class="MJXp-mo" id="MJXp-Span-372" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-373">0.125</span></span></span><script type="math/tex" id="MathJax-Element-37">\gamma = 0.125</script></span>; 2 times: <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-374"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-375"></span><span class="MJXp-mo" id="MJXp-Span-376" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-377">0.25</span></span></span><script type="math/tex" id="MathJax-Element-38">\gamma = 0.25</script></span>; 3 times: <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-378"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-379"></span><span class="MJXp-mo" id="MJXp-Span-380" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-381">0.375</span></span></span><script type="math/tex" id="MathJax-Element-39">\gamma = 0.375</script></span>; 4 times: <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-382"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-383"></span><span class="MJXp-mo" id="MJXp-Span-384" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-385">0.5</span></span></span><script type="math/tex" id="MathJax-Element-40">\gamma = 0.5</script></span>). Because the process of hybridization is not uniform, the value of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-386"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-387"></span></span></span><script type="math/tex" id="MathJax-Element-41">\gamma</script></span> jumps between different values. This can also be visualized by plotting the distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-388"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-389"></span></span></span><script type="math/tex" id="MathJax-Element-42">\gamma</script></span> across the bootstrap replicates (e.g., <a reveal-id="F1" data-open="F1" class="link link-reveal link-table xref-fig">Fig. 2</a>).</p><p>We
 have also implemented methods that test all individuals in a putative 
hybrid separately while treating the parents and outgroup as 
populations. This approach allows hypothesis tests and estimates of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-390"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-391"></span></span></span><script type="math/tex" id="MathJax-Element-43">\gamma</script></span>
 to be calculated for every individual to see if it is a hybrid. A 
caveat with testing each individual is that the number of sites sampled 
must be enough to have statistical power to detect that hybridization 
has occurred (<a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman 2015</a>).</p>                    <h4 scrollto-destination="120230358" id="120230358" class="section-title" data-legacy-id="SEC1.2.3">Ambiguity codes and missing data.</h4>
<p>To allow more data to be used, and to account for the potentially 
large amounts of missing data that are common in high throughput 
sequencing data sets, we have implemented approaches that integrate over
 missing or ambiguous nucleotides by considering the possible 
resolutions of the observed bases into site patterns. We note that this 
approach is also appropriate for high coverage data sets when an 
individual is heterozygous at a particular locus, but a consensus 
sequence with an ambiguity code is used to represent the different bases
 of the heterozygous genotype. As an example, consider the site pattern 
AGRG. There are two possible resolutions for the ambiguity code R: AGGG 
and AGAG. To account for this, we add 0.5 to the site pattern counts <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-392"><span class="MJXp-msubsup" id="MJXp-Span-393"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-394" style="margin-right: 0.05em;">X</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-395" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-396">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-397">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-398">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-399">j</span></span></span></span></span><script type="math/tex" id="MathJax-Element-44">X_{ijjj}</script></span> and <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-400"><span class="MJXp-msubsup" id="MJXp-Span-401"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-402" style="margin-right: 0.05em;">X</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-403" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-404">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-405">j</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-406">i</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-407">j</span></span></span></span></span><script type="math/tex" id="MathJax-Element-45">X_{ijij}</script></span>.
 In general, for any site pattern containing ambiguous or missing bases 
(but not gaps), we find all possible resolutions and add one divided by 
the total number of resolutions to the corresponding site pattern 
counts.</p>                    <h3 scrollto-destination="120230360" id="120230360" class="section-title" data-legacy-id="SEC1.3">Software Design and Implementation</h3>
<p>HyDe is implemented in Python and also uses Cython, a superset of the
 Python language that allows C/C++ capabilities to be incorporated into 
Python for better efficiency (<a reveal-id="B4" data-open="B4" class="link link-ref link-reveal xref-bibr">Behnel et al. 2010</a>). Installing HyDe requires several external Python modules [<span class="monospace">numpy</span>, <span class="monospace">cython</span>, <span class="monospace">matplotlib</span>, <span class="monospace">multiprocess</span>, and <span class="monospace">seaborn</span>].
 The goal behind our implementation of HyDe was to provide both 
pre-packaged scripts to conduct standard hybridization detection 
analyses, as well as a library of functions and data structures that 
researchers could use to customize their analyses or implement new tests
 based on site pattern probabilities. Below, we describe the 
functionality of both of these interfaces, as well as the input files 
required to run HyDe.</p>                    <h4 scrollto-destination="120230362" id="120230362" class="section-title" data-legacy-id="SEC1.3.1">Input files.</h4>
<p>The input files for running HyDe are plain text files containing the 
DNA sequence data and the map of individuals to populations. The DNA 
sequence data are expected to be in sequential Phylip format and the 
population map is a two-column table with individual names in the first 
column and the name of the population that it belongs to in the second 
column. The third input file that is required for individual-level 
testing and bootstrapping (optional for running a normal HyDe analysis) 
is a three column table of triplets. A hypothesis test is then run for 
each triplet using the first column as parent one, the second column as 
the putative hybrid, and the third column as parent two. The outgroup 
for each test is always the same and is specified separately at the 
command line. Output files from previous HyDe analyses can also be used 
to specify which triplets are to be tested.</p>                    <h4 scrollto-destination="120230364" id="120230364" class="section-title" data-legacy-id="SEC1.3.2">Command line interface.</h4>
<p>The command line interface for HyDe consists of a set of Python 
scripts that can be used to detect hybridization, filter results, 
conduct individual bootstrapping, and test for hybridization at the 
individual level. The three main scripts for detecting hybridization 
with HyDe provide the majority of the basic functionality that is needed
 to detect hybridization in an empirical data set and to assess if 
individuals in putative hybrids are all equally admixed. Each of these 
scripts also has a multithreaded version for parallelizing hypothesis 
tests across triplets (in square brackets below).</p><p></p><ul class="bullet"><li><p><strong><em>run_hyde.py</em></strong> [<em>run_hyde_mp.py</em>]: The <em>run_hyde.py</em>
 script detects hybridization by running a hypothesis test on all 
possible triplets in a data set in all directions (i.e., a full HyDe 
analysis). There is also an option to supply a list of specific triplets
 (three column text file) to test. The script outputs two files: one 
with all of the hypothesis tests that were conducted and one with only 
those hypothesis tests that were significant at an overall <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-408"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-409"></span><span class="MJXp-mo" id="MJXp-Span-410" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-411">0.05</span></span></span><script type="math/tex" id="MathJax-Element-46">\alpha=0.05</script></span> level (after incorporating a Bonferonni correction) with estimates of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-412"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-413"></span></span></span><script type="math/tex" id="MathJax-Element-47">\gamma</script></span> between 0 and 1.</p></li><li><p><strong><em>individual_hyde.py</em></strong> [<em>individual_hyde_mp.py</em>]: The <em>individual_hyde.py</em>
 script runs separate hybridization detection analyses on each 
individual within a putative hybrid lineage. The only additional input 
needed for the script is a three column list of triplets or a filtered 
results file from the <em>run_hyde.py</em> script.</p></li><li><p><strong><em>bootstrap_hyde.py</em></strong> [<em>bootstrap_hyde_mp.py</em>]: The <em>bootstrap_hyde.py</em>
 script performs bootstrap resampling of the individuals within a 
putative hybrid lineage and conducts a hypothesis test for each 
bootstrap replicate. Similar to the <em>individual_hyde.py</em> script, 
the script uses as input a table of triplets or a filtered results file 
from a previous hybridization detection analysis.</p></li></ul><p></p><p>The workflow that we envision for these scripts starts with using the <em>run_hyde.py</em>
 script to test for hybridization on all triplets in all possible 
directions, and to filter out only those triplets that have significant 
evidence for hybridization. Then, using the filtered output file, users 
can either test all of the individuals in each hybrid lineage or perform
 bootstrap resampling of the individuals using the <em>individual_hyde.py</em> and <em>bootstrap_hyde.py</em>
 scripts, respectively. Users with specific hypotheses that they want to
 test can simply create a text file listing the triplets of interest to 
be run through any of the three scripts.</p>                    <h4 scrollto-destination="120230368" id="120230368" class="section-title" data-legacy-id="SEC1.3.3">Python API.</h4>
<p>Each of the command line scripts described above makes use of an 
underlying Python library with built-in data structures for processing 
the hypothesis tests and bootstrapping analyses conducted by HyDe. This 
library is exposed through the Python module <span class="monospace">phyde</span> (<strong>P</strong>ythonic <strong>Hy</strong>bridization <strong>De</strong>tection). The main data structures that are part of this module are listed below: </p><ul class="bullet"><li><p><strong><em>HydeData</em></strong>: The <em>HydeData</em>
 class is the primary data structure for reading in and storing DNA 
sequence data and maps assigning individuals to populations. This class 
also implements three of the main methods for detecting hybridization at
 the population level [<span class="monospace">test_triple()</span>] and the individual level [<span class="monospace">test_individuals()</span>], as well as bootstrapping individuals within populations [<span class="monospace">bootstrap_triple()</span>] (see <a href="#box1" class="sectionLink xref-sec">Box 1</a>).</p></li><li><p><strong><em>HydeResult</em></strong>: The <em>HydeResult</em>
 class parses the results file output by a hybridization detection 
analysis and stores the results as a Python dictionary (key-value pair).
 Values stored by the <em>HydeResult</em> class can be accessed by 
providing the name of the desired value and the names of the taxa in the
 triplet of interest as arguments to the variable used when reading in 
the results. For example, if we read the results of a hybridization 
detection analysis into a variable named <span class="monospace">results</span>, we would access the estimated value of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-414"><span class="MJXp-mrow" id="MJXp-Span-415"><span class="MJXp-munderover" id="MJXp-Span-416"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-418" style="margin-left: 0px; margin-right: 0px;"></span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-417"></span></span></span></span></span></span></span></span><script type="math/tex" id="MathJax-Element-48">\hat{\gamma}</script></span> for the triplet (<span class="monospace">sp1</span>, <span class="monospace">sp2</span>, <span class="monospace">sp3</span>) using the following code: results("Gamma", "sp1", "sp2", "sp3")!.</p></li><li><p><strong><em>Bootstrap</em></strong>: The <em>Bootstrap</em> class is similar to the <em>HydeResult</em> class except that it has built-in methods for parsing the format of the bootstrap output file written by the <em>bootstrap_hyde.py</em> script. Each bootstrap replicate is printed with a single line containing four pound symbols separating each tested triplet (<span class="monospace">####\n</span>).
 These results are parsed into a Python dictionary that can be used to 
access the bootstrap replicates for particular triplets using the 
variable name, just like the <em>HydeResult</em> class [e.g., <span class="monospace">bootstraps("Gamma", "sp1", "sp2", "sp3")</span>].</p></li><li><p><strong><em>phyde.viz</em></strong>: The <em>viz</em> submodule uses the <span class="monospace">matplotlib</span> and <span class="monospace">seaborn</span>
 libraries to implement basic functions for plotting the distribution of
 bootstrap replicates for any quantity calculated by the <em>bootstrap_hyde.py</em> script (Z-score, p-value, <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-419"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-420"></span></span></span><script type="math/tex" id="MathJax-Element-49">\gamma</script></span>, etc.) for a specified triplet. It does this by taking the name of a <em>Bootstrap</em> object variable, the name of the value to be plotted, and the names of the taxa in the triplet of interest (<a class="link link-uri" href="https://academic.oup.com/sysbio/article-lookup/doi/10.1093/sysbio/syy023#supplementary-data" target="">Supplemental Materials</a>: HyDe_SysBio.ipynb).</p></li></ul><p></p><div class="boxed-text boxed-matter"><div class="label">Box 1.</div><div class="caption"><p>Example code to run hybridization detection analyses with HyDe using the Python API. The <span class="monospace">phyde</span> module is loaded using the <span class="monospace">import</span> command:</p></div><div class="clearfix"></div><p><span class="monospace"># import the phyde module</span></p><p><span class="monospace">import phyde</span></p><p>Data are read in and stored using the <em>HydeData</em>
 class by passing the names of the input file, map file, and outgroup, 
as well as the number of individuals, the number of populations, and the
 number of sites.</p><p><span class="monospace"># read in data using HydeData class</span></p><p><span class="monospace">data = phyde.HydeData("data.txt", "map.txt",</span></p><p><span class="monospace">"out", 16, 4, 50000)</span></p><p>With the data read in, we can use methods implemented in the <em>HydeData</em> class to test for hybridization at the population level [<span class="monospace">test_triple(p1, hyb, p2)</span>], at the individual level [<span class="monospace">test_individuals(p1, hyb, p2)</span>], and can bootstrap resample individuals [<span class="monospace">bootstrap_triple(p1, hyb, p2, nreps)</span>].</p><p><span class="monospace"># test for hybridization in the "sp2" population</span></p><p><span class="monospace">res1 = data.test_triple("sp1", "sp2", "sp3")</span></p><p><span class="monospace"># test all individuals in the "sp2" population</span></p><p><span class="monospace">res2 = data.test_individuals("sp1", "sp2", "sp3")</span></p><p><span class="monospace"># bootstrap individuals in the "sp2" population 200 times</span></p><p><span class="monospace">res3 = data.bootstrap_triple("sp1", "sp2", "sp3", 200)</span></p></div>                    <h2 scrollto-destination="120230371" id="120230371" class="section-title" data-legacy-id="SEC2">BENCHMARKS</h2>
<p>All code used to complete the simulations and example analyses are available as a Jupyter Notebook on Dryad (<a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.372sq" target="">http://dx.doi.org/10.5061/dryad.372sq</a>).
 We have also included all output files needed to reproduce the tables 
and figures. Note that in this section we use the names <span class="monospace">out</span>, <span class="monospace">sp1</span>, <span class="monospace">sp2</span>, and <span class="monospace">sp3</span>
 for the names of the populations, rather than O, P1, Hyb, and P2. We do
 this to denote that these taxa have not yet been tested for 
hybridization, whereas the model is explicitly constructed to contain a 
hybrid species and its parents.</p>                    <h3 scrollto-destination="120230373" id="120230373" class="section-title" data-legacy-id="SEC2.1">Non-Uniform Hybridization Simulations</h3>
<div class="block-child-p">To demonstrate the use of the methods in HyDe
 that are designed to identify individual hybrids, we set up a simulated
 example for four taxa (<span class="monospace">out</span>, <span class="monospace">sp1</span>, <span class="monospace">sp2</span>, <span class="monospace">sp3</span>)
 that intentionally violated the assumption of hybrid speciation by 
including a single hybrid individual in the population being tested for 
admixture (<span class="monospace">sp2</span>). We simulated gene trees using the program <span class="monospace">ms</span> (<a reveal-id="B14" data-open="B14" class="link link-ref link-reveal xref-bibr">Hudson 2002</a>)
 with population branch lengths of 1.0 coalescent unit assuming a model 
of coalescent independent sites (i.e., one gene tree per site; <a reveal-id="B21" data-open="B21" class="link link-ref link-reveal xref-bibr">Kubatko and Chifman 2015</a>)
 for 10,000, 50,000, 100,000, 250,000, and 500,000 sites. The parental 
populations were simulated with five individuals each and the outgroup 
had only one individual. The admixed population (<span class="monospace">sp2</span>) had four non-admixed individuals that were most closely related to parental population <span class="monospace">sp1</span>, and a single hybrid individual that was a 50:50 (<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-421"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-422"></span><span class="MJXp-mo" id="MJXp-Span-423" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-424">0.5</span></span></span><script type="math/tex" id="MathJax-Element-50">\gamma=0.5</script></span>) mix between <span class="monospace">sp1</span> and <span class="monospace">sp3</span>. A single DNA base was simulated on each gene tree for the different numbers of sites using the program <span class="monospace">Seq-Gen</span> (<a reveal-id="B32" data-open="B32" class="link link-ref link-reveal xref-bibr">Rambaut and Grassly 1997</a>). We rescaled gene tree branch lengths from coalescent units to mutation units using a population scaled mutation rate (<span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-425"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-426"></span><span class="MJXp-mo" id="MJXp-Span-427" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-428">4</span><span class="MJXp-msubsup" id="MJXp-Span-429"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-430" style="margin-right: 0.05em;">N</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-431" style="vertical-align: -0.4em;">0</span></span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-432"></span></span></span><script type="math/tex" id="MathJax-Element-51">\theta=4N_0\mu</script></span>) of 0.01 per site and used a GTR+Gamma model of nucleotide substitution with three rate categories: <a id="syy023UM1" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="syy023UM1"><div class="tex-math display-math"> <span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math MJXp-display" id="MJXp-Span-433"><span class="MJXp-mtable" id="MJXp-Span-434"><span><span class="MJXp-mtr" id="MJXp-Span-435" style="vertical-align: baseline;"><span class="MJXp-mtd" id="MJXp-Span-436" style="text-align: right;"><span class="MJXp-mrow" id="MJXp-Span-437"><span class="MJXp-mi MJXp-mono" id="MJXp-Span-438">s</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-439">e</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-440">q</span><span class="MJXp-mo MJXp-mono" id="MJXp-Span-441" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-442">g</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-443">e</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-444">n</span><span class="MJXp-mo MJXp-mono" id="MJXp-Span-445" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-446">m</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-447">G</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-448">T</span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-449">R</span><span class="MJXp-mo MJXp-mono" id="MJXp-Span-450" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-451">s</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-452">0.01</span><span class="MJXp-mo MJXp-mono" id="MJXp-Span-453" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-454">l</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-455">1</span></span></span></span><span class="MJXp-mtr" id="MJXp-Span-456" style="vertical-align: baseline;"><span class="MJXp-mtd" id="MJXp-Span-457" style="padding-top: 0.3em; text-align: right;"><span class="MJXp-mrow" id="MJXp-Span-458"><span class="MJXp-mo MJXp-mono" id="MJXp-Span-459" style="margin-left: 0em; margin-right: 0.111em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-460">r</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-461">1.0</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-462">0.2</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-463">10.0</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-464">0.75</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-465">3.2</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-466">1.6</span></span></span></span><span class="MJXp-mtr" id="MJXp-Span-467" style="vertical-align: baseline;"><span class="MJXp-mtd" id="MJXp-Span-468" style="padding-top: 0.3em; text-align: right;"><span class="MJXp-mrow" id="MJXp-Span-469"><span class="MJXp-mo MJXp-mono" id="MJXp-Span-470" style="margin-left: 0em; margin-right: 0.111em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-471">f</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-472">0.15</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-473">0.35</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-474">0.15</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-475">0.35</span><span class="MJXp-mo MJXp-mono" id="MJXp-Span-476" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-477">i</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-478">0.2</span></span></span></span><span class="MJXp-mtr" id="MJXp-Span-479" style="vertical-align: baseline;"><span class="MJXp-mtd" id="MJXp-Span-480" style="padding-top: 0.3em; text-align: right;"><span class="MJXp-mrow" id="MJXp-Span-481"><span class="MJXp-mo MJXp-mono" id="MJXp-Span-482" style="margin-left: 0em; margin-right: 0.111em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-483">a</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-484">5.0</span><span class="MJXp-mo MJXp-mono" id="MJXp-Span-485" style="margin-left: 0.267em; margin-right: 0.267em;"></span><span class="MJXp-mi MJXp-mono" id="MJXp-Span-486">g</span><span class="MJXp-mn MJXp-mono" id="MJXp-Span-487">3</span></span></span></span></span></span></span></span><script type="math/tex; mode=display" id="MathJax-Element-52">\begin{eqnarray*}
\tt{seq-gen -mGTR -s 0.01 -l 1}\\
\tt{-r 1.0 0.2 10.0 0.75 3.2 1.6}\\
\tt{-f 0.15 0.35 0.15 0.35 -i 0.2}\\
\tt{-a 5.0 -g 3}
\end{eqnarray*}</script> </div></div></div></div><p>Output from <span class="monospace">Seq-Gen</span> was formatted for input to HyDe using a Python script (<em>seqgen2hyde.py</em>; available on Dryad). Hybridization detection was completed for the simulated data sets with HyDe v0.4.1 using the <em>run_hyde.py</em>, <em>bootstrap_hyde.py</em> (500 bootstrap replicates), and <em>individual_hyde.py</em> scripts. Output files were processed and plotted in Python v2.7 using the <span class="monospace">phyde</span>, <span class="monospace">matplotlib</span>, and <span class="monospace">seaborn</span> modules.</p>                    <h4 scrollto-destination="120230376" id="120230376" class="section-title" data-legacy-id="SEC2.1.1">Results.</h4>
<p>Testing for hybridization at the population level using the <em>run_hyde.py</em> script produced a test statistic indicating that there was significant admixture in the <span class="monospace">sp2</span> population at the <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-488"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-489"></span><span class="MJXp-mo" id="MJXp-Span-490" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-491">0.05</span></span></span><script type="math/tex" id="MathJax-Element-53">\alpha=0.05</script></span> level when the number of sites was at least 50,000 (<a reveal-id="T1" data-open="T1" class="link link-reveal link-table xref-fig">Table 1</a>). The estimated values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-492"><span class="MJXp-mrow" id="MJXp-Span-493"><span class="MJXp-munderover" id="MJXp-Span-494"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-496" style="margin-left: 0px; margin-right: 0px;"></span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-495"></span></span></span></span></span></span></span></span><script type="math/tex" id="MathJax-Element-54">\hat{\gamma}</script></span> for these tests were close to 0.9, which is inconsistent with the data that we simulated. The value of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-497"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-498"></span></span></span><script type="math/tex" id="MathJax-Element-55">\gamma</script></span>
 that we would expect is 0.1. However, because this is an explicit 
violation of the coalescent model with hybridization, the formula for 
the estimation of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-499"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-500"></span></span></span><script type="math/tex" id="MathJax-Element-56">\gamma</script></span> is no longer valid.</p><p></p><div content-id="T1" class="table-modal" style="display: block;"><div class="table-wrap" content-id="T1"><div id="T1" data-id="T1" class="table-wrap-title"><div class="label">Table 1.</div><div class="caption"><p>Results of the population-level hybridization detection analyses for the non-uniform hybridization simulation using the <em>run_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>1.100&nbsp;</td><td>0.136&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.234&nbsp;</td><td>0.013&nbsp;</td><td>0.897&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.274&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.310 \times 10^{\text{-}4}</span><script type="math/tex">5.310 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.900&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.420&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.973 \times 10^{\text{-}8}</span><script type="math/tex">2.973 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>6.808&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.966 \times 10^{\text{-}12}</span><script type="math/tex">4.966 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.905&nbsp;</td></tr></tbody></table></div><div content-id="T1" class="table-modal"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>1.100&nbsp;</td><td>0.136&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.234&nbsp;</td><td>0.013&nbsp;</td><td>0.897&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.274&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.310 \times 10^{\text{-}4}</span><script type="math/tex">5.310 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.900&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.420&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.973 \times 10^{\text{-}8}</span><script type="math/tex">2.973 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>6.808&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.966 \times 10^{\text{-}12}</span><script type="math/tex">4.966 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.905&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn1"><div class="fn"><p>Note that the result for the simulation with 10,000 sites was not significant at the <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\alpha=0.05</span><script type="math/tex">\alpha=0.05</script></span> level (marked with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">*</span><script type="math/tex">*</script></span>).</p></div></div></div></div></div><div class="table-wrap" content-id="T1"><div id="T1" data-id="T1" class="table-wrap-title"><div class="label">Table 1.</div><div class="caption"><p>Results of the population-level hybridization detection analyses for the non-uniform hybridization simulation using the <em>run_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>1.100&nbsp;</td><td>0.136&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.234&nbsp;</td><td>0.013&nbsp;</td><td>0.897&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.274&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.310 \times 10^{\text{-}4}</span><script type="math/tex">5.310 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.900&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.420&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.973 \times 10^{\text{-}8}</span><script type="math/tex">2.973 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>6.808&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.966 \times 10^{\text{-}12}</span><script type="math/tex">4.966 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.905&nbsp;</td></tr></tbody></table></div><div content-id="T1" class="table-modal"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>1.100&nbsp;</td><td>0.136&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.234&nbsp;</td><td>0.013&nbsp;</td><td>0.897&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.274&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.310 \times 10^{\text{-}4}</span><script type="math/tex">5.310 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.900&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.420&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.973 \times 10^{\text{-}8}</span><script type="math/tex">2.973 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.896&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>6.808&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.966 \times 10^{\text{-}12}</span><script type="math/tex">4.966 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.905&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn1"><div class="fn"><p>Note that the result for the simulation with 10,000 sites was not significant at the <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\alpha=0.05</span><script type="math/tex">\alpha=0.05</script></span> level (marked with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">*</span><script type="math/tex">*</script></span>).</p></div></div></div></div><p></p><p>If we did not already know that not all of the individuals were hybrids, we would mistakenly infer that the <span class="monospace">sp2</span> population has <span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}90</span><script type="math/tex">{\sim}90</script></span>% of its genetic material inherited from population <span class="monospace">sp3</span>. However, when we test each individual using the <em>individual_hyde.py</em> script, we correctly infer that only one of the individuals is admixed (individual i10, <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}\approx0.5</span><script type="math/tex">\hat{\gamma}\approx0.5</script></span>) and that the rest of the individuals are not hybrids (<a reveal-id="T2" data-open="T2" class="link link-reveal link-table xref-fig">Table 2</a>). Bootstrap resampling of the individuals in population <span class="monospace">sp2</span> also indicates that hybridization is not uniform. <a reveal-id="F2" data-open="F2" class="link link-reveal link-table xref-fig">Figure 2</a> shows the distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> across all 500 bootstrap replicates and demonstrates that the value of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> jumps between different values depending on the number of times the hybrid individual is resampled.</p>                        <a id="120230380" scrollto-destination="120230380"></a>
<div id="F2" data-id="F2" class="fig fig-section"><div class="label fig-label">Fig. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f2.png" alt="Violin plots of the distribution of $\hat{\gamma}$ across 500 bootstrap replicates for 10,000, 50,000, 100,000, 250,000, and 500,000 sites for the non-uniform hybridization simulations. The black dots in each violin plot represent the actual values of $\hat{\gamma}$ that were estimated by each replicate (black dots are jittered). The pattern of jumping between distinct values of $\gamma$ with no estimates in between are a strong indication of non-uniform admixture." path-from-xml="syy023f2.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230380/syy023f2.tif" path-from-xml="syy023f2.tif" target="_blank">Open in new tab</a><a section="120230380" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f2.png?Expires=1577481743&amp;Signature=dnwz6QJPhH1z4mI6uTkfCVRSAt1E9bbVaNUQ1rpfmd3ssLJEXW5K3n2WkmfVdntQqaX2tvjJtQ1T86rv8ppVkGooVLVB6VX-9Dh6FLFbIlj-OE1aldTrko0q78HNz8xF32bWjlpyynlgYD2ikL2Pukk3ZcJpMWm9bWSk1R0TJ5nTY4zFkXNRvaoAO%7EknOnW4MdTFCVB8BsSLxUUvJw5wo0YX8LJaWWJIlOeRYUm5l1PXxX-%7E-riAv0rmDDfFTKP2%7E6HDlNQMUCF1NhWPDGo5ECDXI4WShiiTCF%7E%7EtlbXvxu1UtNwMvBpRPJJFnApfOzf9H2Xcomc8vEkRVACjgrdBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230380&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f2.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Violin plots of the distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 across 500 bootstrap replicates for 10,000, 50,000, 100,000, 250,000, 
and 500,000 sites for the non-uniform hybridization simulations. The 
black dots in each violin plot represent the actual values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> that were estimated by each replicate (black dots are jittered). The pattern of jumping between distinct values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\gamma</span><script type="math/tex">\gamma</script></span> with no estimates in between are a strong indication of non-uniform admixture.</p></div></div><div content-id="F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f2.png" alt="Violin plots of the distribution of $\hat{\gamma}$ across 500 bootstrap replicates for 10,000, 50,000, 100,000, 250,000, and 500,000 sites for the non-uniform hybridization simulations. The black dots in each violin plot represent the actual values of $\hat{\gamma}$ that were estimated by each replicate (black dots are jittered). The pattern of jumping between distinct values of $\gamma$ with no estimates in between are a strong indication of non-uniform admixture." path-from-xml="syy023f2.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230380/syy023f2.tif" path-from-xml="syy023f2.tif" target="_blank">Open in new tab</a><a section="120230380" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f2.png?Expires=1577481743&amp;Signature=dnwz6QJPhH1z4mI6uTkfCVRSAt1E9bbVaNUQ1rpfmd3ssLJEXW5K3n2WkmfVdntQqaX2tvjJtQ1T86rv8ppVkGooVLVB6VX-9Dh6FLFbIlj-OE1aldTrko0q78HNz8xF32bWjlpyynlgYD2ikL2Pukk3ZcJpMWm9bWSk1R0TJ5nTY4zFkXNRvaoAO%7EknOnW4MdTFCVB8BsSLxUUvJw5wo0YX8LJaWWJIlOeRYUm5l1PXxX-%7E-riAv0rmDDfFTKP2%7E6HDlNQMUCF1NhWPDGo5ECDXI4WShiiTCF%7E%7EtlbXvxu1UtNwMvBpRPJJFnApfOzf9H2Xcomc8vEkRVACjgrdBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230380&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f2.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Violin plots of the distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 across 500 bootstrap replicates for 10,000, 50,000, 100,000, 250,000, 
and 500,000 sites for the non-uniform hybridization simulations. The 
black dots in each violin plot represent the actual values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> that were estimated by each replicate (black dots are jittered). The pattern of jumping between distinct values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\gamma</span><script type="math/tex">\gamma</script></span> with no estimates in between are a strong indication of non-uniform admixture.</p></div></div><p></p><div content-id="T2" class="table-modal" style="display: block;"><div class="table-wrap" content-id="T2"><div id="T2" data-id="T2" class="table-wrap-title"><div class="label">Table 2.</div><div class="caption"><p>Results of the individual-level hybridization detection analyses for the non-uniform hybridization simulation using the <em>individual_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10,000&nbsp;</th><th colspan="3">50,000&nbsp;</th><th colspan="3">100,000&nbsp;</th><th colspan="3">250,000&nbsp;</th><th colspan="3">500,000&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.408&nbsp;</td><td>0.342&nbsp;</td><td>n.s.&nbsp;</td><td>0.054&nbsp;</td><td>0.478&nbsp;</td><td>n.s.&nbsp;</td><td>0.049&nbsp;</td><td>0.480&nbsp;</td><td>n.s.&nbsp;</td><td>0.387&nbsp;</td><td>0.349&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.015&nbsp;</td><td>0.506&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.241&nbsp;</td><td>0.405&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.295&nbsp;</td><td>0.616&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.289&nbsp;</td><td>0.614&nbsp;</td><td>n.s.&nbsp;</td><td>0.322&nbsp;</td><td>0.374&nbsp;</td><td>n.s.&nbsp;</td><td>0.012&nbsp;</td><td>0.496&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>1.128&nbsp;</td><td>0.130&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.994&nbsp;</td><td>0.840&nbsp;</td><td>n.s.&nbsp;</td><td>0.106&nbsp;</td><td>0.458&nbsp;</td><td>n.s.&nbsp;</td><td>0.773&nbsp;</td><td>0.220&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.275&nbsp;</td><td>0.609&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>0.711&nbsp;</td><td>0.239&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.445&nbsp;</td><td>0.672&nbsp;</td><td>n.s.&nbsp;</td><td>0.554&nbsp;</td><td>0.290&nbsp;</td><td>n.s.&nbsp;</td><td>0.734&nbsp;</td><td>0.231&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.130&nbsp;</td><td>0.552&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>2.267&nbsp;</td><td>0.012&nbsp;</td><td>0.562&nbsp;</td><td>6.464&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.108 \times 10^{\text{-}11}</span><script type="math/tex">5.108 \times 10^{\text{-}11}</script></span>&nbsp;</td><td>0.450&nbsp;</td><td>9.011&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>14.015&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>20.114&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.508&nbsp;</td></tr></tbody></table></div><div content-id="T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10,000&nbsp;</th><th colspan="3">50,000&nbsp;</th><th colspan="3">100,000&nbsp;</th><th colspan="3">250,000&nbsp;</th><th colspan="3">500,000&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.408&nbsp;</td><td>0.342&nbsp;</td><td>n.s.&nbsp;</td><td>0.054&nbsp;</td><td>0.478&nbsp;</td><td>n.s.&nbsp;</td><td>0.049&nbsp;</td><td>0.480&nbsp;</td><td>n.s.&nbsp;</td><td>0.387&nbsp;</td><td>0.349&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.015&nbsp;</td><td>0.506&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.241&nbsp;</td><td>0.405&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.295&nbsp;</td><td>0.616&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.289&nbsp;</td><td>0.614&nbsp;</td><td>n.s.&nbsp;</td><td>0.322&nbsp;</td><td>0.374&nbsp;</td><td>n.s.&nbsp;</td><td>0.012&nbsp;</td><td>0.496&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>1.128&nbsp;</td><td>0.130&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.994&nbsp;</td><td>0.840&nbsp;</td><td>n.s.&nbsp;</td><td>0.106&nbsp;</td><td>0.458&nbsp;</td><td>n.s.&nbsp;</td><td>0.773&nbsp;</td><td>0.220&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.275&nbsp;</td><td>0.609&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>0.711&nbsp;</td><td>0.239&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.445&nbsp;</td><td>0.672&nbsp;</td><td>n.s.&nbsp;</td><td>0.554&nbsp;</td><td>0.290&nbsp;</td><td>n.s.&nbsp;</td><td>0.734&nbsp;</td><td>0.231&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.130&nbsp;</td><td>0.552&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>2.267&nbsp;</td><td>0.012&nbsp;</td><td>0.562&nbsp;</td><td>6.464&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.108 \times 10^{\text{-}11}</span><script type="math/tex">5.108 \times 10^{\text{-}11}</script></span>&nbsp;</td><td>0.450&nbsp;</td><td>9.011&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>14.015&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>20.114&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.508&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn2"><div class="fn"><p>n.s. = not significant. Individual i10 was the only hybrid in the population and was simulated with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\gamma=0.5</span><script type="math/tex">\gamma=0.5</script></span>.</p></div></div></div></div></div><div class="table-wrap" content-id="T2"><div id="T2" data-id="T2" class="table-wrap-title"><div class="label">Table 2.</div><div class="caption"><p>Results of the individual-level hybridization detection analyses for the non-uniform hybridization simulation using the <em>individual_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10,000&nbsp;</th><th colspan="3">50,000&nbsp;</th><th colspan="3">100,000&nbsp;</th><th colspan="3">250,000&nbsp;</th><th colspan="3">500,000&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.408&nbsp;</td><td>0.342&nbsp;</td><td>n.s.&nbsp;</td><td>0.054&nbsp;</td><td>0.478&nbsp;</td><td>n.s.&nbsp;</td><td>0.049&nbsp;</td><td>0.480&nbsp;</td><td>n.s.&nbsp;</td><td>0.387&nbsp;</td><td>0.349&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.015&nbsp;</td><td>0.506&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.241&nbsp;</td><td>0.405&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.295&nbsp;</td><td>0.616&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.289&nbsp;</td><td>0.614&nbsp;</td><td>n.s.&nbsp;</td><td>0.322&nbsp;</td><td>0.374&nbsp;</td><td>n.s.&nbsp;</td><td>0.012&nbsp;</td><td>0.496&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>1.128&nbsp;</td><td>0.130&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.994&nbsp;</td><td>0.840&nbsp;</td><td>n.s.&nbsp;</td><td>0.106&nbsp;</td><td>0.458&nbsp;</td><td>n.s.&nbsp;</td><td>0.773&nbsp;</td><td>0.220&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.275&nbsp;</td><td>0.609&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>0.711&nbsp;</td><td>0.239&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.445&nbsp;</td><td>0.672&nbsp;</td><td>n.s.&nbsp;</td><td>0.554&nbsp;</td><td>0.290&nbsp;</td><td>n.s.&nbsp;</td><td>0.734&nbsp;</td><td>0.231&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.130&nbsp;</td><td>0.552&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>2.267&nbsp;</td><td>0.012&nbsp;</td><td>0.562&nbsp;</td><td>6.464&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.108 \times 10^{\text{-}11}</span><script type="math/tex">5.108 \times 10^{\text{-}11}</script></span>&nbsp;</td><td>0.450&nbsp;</td><td>9.011&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>14.015&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>20.114&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.508&nbsp;</td></tr></tbody></table></div><div content-id="T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10,000&nbsp;</th><th colspan="3">50,000&nbsp;</th><th colspan="3">100,000&nbsp;</th><th colspan="3">250,000&nbsp;</th><th colspan="3">500,000&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.408&nbsp;</td><td>0.342&nbsp;</td><td>n.s.&nbsp;</td><td>0.054&nbsp;</td><td>0.478&nbsp;</td><td>n.s.&nbsp;</td><td>0.049&nbsp;</td><td>0.480&nbsp;</td><td>n.s.&nbsp;</td><td>0.387&nbsp;</td><td>0.349&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.015&nbsp;</td><td>0.506&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.241&nbsp;</td><td>0.405&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.295&nbsp;</td><td>0.616&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.289&nbsp;</td><td>0.614&nbsp;</td><td>n.s.&nbsp;</td><td>0.322&nbsp;</td><td>0.374&nbsp;</td><td>n.s.&nbsp;</td><td>0.012&nbsp;</td><td>0.496&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>1.128&nbsp;</td><td>0.130&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.994&nbsp;</td><td>0.840&nbsp;</td><td>n.s.&nbsp;</td><td>0.106&nbsp;</td><td>0.458&nbsp;</td><td>n.s.&nbsp;</td><td>0.773&nbsp;</td><td>0.220&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.275&nbsp;</td><td>0.609&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>0.711&nbsp;</td><td>0.239&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.445&nbsp;</td><td>0.672&nbsp;</td><td>n.s.&nbsp;</td><td>0.554&nbsp;</td><td>0.290&nbsp;</td><td>n.s.&nbsp;</td><td>0.734&nbsp;</td><td>0.231&nbsp;</td><td>n.s.&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">-</span><script type="math/tex">-</script></span>0.130&nbsp;</td><td>0.552&nbsp;</td><td>n.s.&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>2.267&nbsp;</td><td>0.012&nbsp;</td><td>0.562&nbsp;</td><td>6.464&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.108 \times 10^{\text{-}11}</span><script type="math/tex">5.108 \times 10^{\text{-}11}</script></span>&nbsp;</td><td>0.450&nbsp;</td><td>9.011&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>14.015&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.497&nbsp;</td><td>20.114&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.508&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn2"><div class="fn"><p>n.s. = not significant. Individual i10 was the only hybrid in the population and was simulated with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\gamma=0.5</span><script type="math/tex">\gamma=0.5</script></span>.</p></div></div></div></div><p></p>                    <h3 scrollto-destination="120230382" id="120230382" class="section-title" data-legacy-id="SEC2.2">Validating Population-Level Hybridization Detection</h3>
<p>To contrast with the previous simulation, and to validate 
hybridization at the population level, we repeated the same procedure as
 above for simulating DNA sequence data for four taxa, but this time <em>all</em> individuals in the <span class="monospace">sp2</span>
 population were 10% admixed. This maintains the overall level of 10% 
admixture in the population, but now it is distributed evenly among the 
individuals. We then ran a hybridization detection analysis on these 
simulated data using the <em>run_hyde.py</em>, <em>bootstrap_hyde.py</em> (500 replicates), and <em>individual_hyde.py</em> scripts.</p>                    <h4 scrollto-destination="120230384" id="120230384" class="section-title" data-legacy-id="SEC2.2.1">Results.</h4>
<p>At the population level, the analysis with <em>run_hyde.py</em> detected significant hybridization (<span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}=0.118-0.125</span><script type="math/tex">\hat{\gamma}=0.118-0.125</script></span>) in all cases except for the simulation with 10,000 sites (<a reveal-id="T3" data-open="T3" class="link link-reveal link-table xref-fig">Table 3</a>). For this scenario, there was not enough data to detect the hybridization at the <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\alpha=0.05</span><script type="math/tex">\alpha=0.05</script></span> level. Analyzing each individual in the <span class="monospace">sp2</span> population correctly detected hybridization in all individuals, with estimates of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> slightly over the original level of 10% admixture (<a reveal-id="T4" data-open="T4" class="link link-reveal link-table xref-fig">Table 4</a>). Bootstrap resampling of the individuals in the <span class="monospace">sp2</span> population also demonstrated that hybridization was uniform, as can be seen by the unimodal distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> values across bootstrap replicates in <a reveal-id="F3" data-open="F3" class="link link-reveal link-table xref-fig">Fig. 3</a>.</p>                        <a id="120230386" scrollto-destination="120230386"></a>
<div id="F3" data-id="F3" class="fig fig-section"><div class="label fig-label">Fig. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f3.png" alt="Violin plots of the distribution of $\hat{\gamma}$ across 500 bootstrap replicates for 50,000, 100,000, 250,000, and 500,000 sites for the uniform hybridization simulations. The black dots in each violin plot represent the actual values of $\hat{\gamma}$ that were estimated by each replicate (black dots are jittered). The distribution of the estimates across bootstrap samples indicates that hybridization is uniform. The analyses with 10,000 sites did not have enough statistical power to detect hybridization and were therefore left out of the plot." path-from-xml="syy023f3.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230386/syy023f3.tif" path-from-xml="syy023f3.tif" target="_blank">Open in new tab</a><a section="120230386" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f3.png?Expires=1577481743&amp;Signature=sZx1kMGWt3Qbwoiiw4CEOZu4Iy9e6V4cWObxFISrt81aYpGlttDHxRTEaBwCoVGk%7EynHLtYIyydEOumeBIx6GEDazrDT%7EVqX0gedrxZ5eTa6-hVOSjgRjPjwpKykxp396VrD%7EEmwzAxnV0i29S1TlZqx2YMnO541%7Efp1dgqp-4LthLBURaO4q7Fp1cuplZ6GToBKmCUc81rZiT5bMh55OqbNlW5Oro%7E-mF65rIcFNYxU1AL6eX7zzlRCrKGsVPaDhQyvUithFKgs6aoCBjYgjXMl0a89kSw9wl5KM-renrIZdXi1REUDxKzVbRGtTz9WAWxkLclKXFnH2926JpKL7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230386&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f3.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Violin plots of the distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 across 500 bootstrap replicates for 50,000, 100,000, 250,000, and 
500,000 sites for the uniform hybridization simulations. The black dots 
in each violin plot represent the actual values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 that were estimated by each replicate (black dots are jittered). The 
distribution of the estimates across bootstrap samples indicates that 
hybridization is uniform. The analyses with 10,000 sites did not have 
enough statistical power to detect hybridization and were therefore left
 out of the plot.</p></div></div><div content-id="F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f3.png" alt="Violin plots of the distribution of $\hat{\gamma}$ across 500 bootstrap replicates for 50,000, 100,000, 250,000, and 500,000 sites for the uniform hybridization simulations. The black dots in each violin plot represent the actual values of $\hat{\gamma}$ that were estimated by each replicate (black dots are jittered). The distribution of the estimates across bootstrap samples indicates that hybridization is uniform. The analyses with 10,000 sites did not have enough statistical power to detect hybridization and were therefore left out of the plot." path-from-xml="syy023f3.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230386/syy023f3.tif" path-from-xml="syy023f3.tif" target="_blank">Open in new tab</a><a section="120230386" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f3.png?Expires=1577481743&amp;Signature=sZx1kMGWt3Qbwoiiw4CEOZu4Iy9e6V4cWObxFISrt81aYpGlttDHxRTEaBwCoVGk%7EynHLtYIyydEOumeBIx6GEDazrDT%7EVqX0gedrxZ5eTa6-hVOSjgRjPjwpKykxp396VrD%7EEmwzAxnV0i29S1TlZqx2YMnO541%7Efp1dgqp-4LthLBURaO4q7Fp1cuplZ6GToBKmCUc81rZiT5bMh55OqbNlW5Oro%7E-mF65rIcFNYxU1AL6eX7zzlRCrKGsVPaDhQyvUithFKgs6aoCBjYgjXMl0a89kSw9wl5KM-renrIZdXi1REUDxKzVbRGtTz9WAWxkLclKXFnH2926JpKL7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230386&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f3.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Violin plots of the distribution of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 across 500 bootstrap replicates for 50,000, 100,000, 250,000, and 
500,000 sites for the uniform hybridization simulations. The black dots 
in each violin plot represent the actual values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 that were estimated by each replicate (black dots are jittered). The 
distribution of the estimates across bootstrap samples indicates that 
hybridization is uniform. The analyses with 10,000 sites did not have 
enough statistical power to detect hybridization and were therefore left
 out of the plot.</p></div></div><p></p><div content-id="T3" class="table-modal" style="display: block;"><div class="table-wrap" content-id="T3"><div id="T3" data-id="T3" class="table-wrap-title"><div class="label">Table 3.</div><div class="caption"><p>Results of the population-level hybridization detection analyses for the 10% admixture simulation using the <em>run_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>0.071&nbsp;</td><td>0.472&nbsp;</td><td>0.008&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.363&nbsp;</td><td>0.009&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.677&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.180 \times 10^{\text{$-$}4}</span><script type="math/tex">1.180 \times 10^{\text{$-$}4}</script></span>&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.796&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.401 \times 10^{\text{-}9}</span><script type="math/tex">3.401 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>8.425&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.118&nbsp;</td></tr></tbody></table></div><div content-id="T3" class="table-modal"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>0.071&nbsp;</td><td>0.472&nbsp;</td><td>0.008&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.363&nbsp;</td><td>0.009&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.677&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.180 \times 10^{\text{$-$}4}</span><script type="math/tex">1.180 \times 10^{\text{$-$}4}</script></span>&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.796&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.401 \times 10^{\text{-}9}</span><script type="math/tex">3.401 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>8.425&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.118&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn3"><div class="fn"><p>Note that the result for the simulation with 10,000 sites was not significant at the <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\alpha=0.05</span><script type="math/tex">\alpha=0.05</script></span> level (marked with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">*</span><script type="math/tex">*</script></span>).</p></div></div></div></div></div><div class="table-wrap" content-id="T3"><div id="T3" data-id="T3" class="table-wrap-title"><div class="label">Table 3.</div><div class="caption"><p>Results of the population-level hybridization detection analyses for the 10% admixture simulation using the <em>run_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>0.071&nbsp;</td><td>0.472&nbsp;</td><td>0.008&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.363&nbsp;</td><td>0.009&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.677&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.180 \times 10^{\text{$-$}4}</span><script type="math/tex">1.180 \times 10^{\text{$-$}4}</script></span>&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.796&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.401 \times 10^{\text{-}9}</span><script type="math/tex">3.401 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>8.425&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.118&nbsp;</td></tr></tbody></table></div><div content-id="T3" class="table-modal"><table><thead name="thead" align="left"><tr><th># sites&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>10,000<span class="inline-formula no-formula-id"><span class="MathJax_Preview">^*</span><script type="math/tex">^*</script></span>&nbsp;</td><td>0.071&nbsp;</td><td>0.472&nbsp;</td><td>0.008&nbsp;</td></tr><tr><td>50,000&nbsp;</td><td>2.363&nbsp;</td><td>0.009&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>100,000&nbsp;</td><td>3.677&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.180 \times 10^{\text{$-$}4}</span><script type="math/tex">1.180 \times 10^{\text{$-$}4}</script></span>&nbsp;</td><td>0.125&nbsp;</td></tr><tr><td>250,000&nbsp;</td><td>5.796&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.401 \times 10^{\text{-}9}</span><script type="math/tex">3.401 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>500,000&nbsp;</td><td>8.425&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.118&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn3"><div class="fn"><p>Note that the result for the simulation with 10,000 sites was not significant at the <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\alpha=0.05</span><script type="math/tex">\alpha=0.05</script></span> level (marked with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">*</span><script type="math/tex">*</script></span>).</p></div></div></div></div><p></p><p></p><div content-id="T4" class="table-modal" style="display: block;"><div class="table-wrap" content-id="T4"><div id="T4" data-id="T4" class="table-wrap-title"><div class="label">Table 4.</div><div class="caption"><p>Results of the individual-level hybridization detection analyses for the 10% admixture simulation using the <em>individual_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10 000&nbsp;</th><th colspan="3">50 000&nbsp;</th><th colspan="3">100 000&nbsp;</th><th colspan="3">250 000&nbsp;</th><th colspan="3">500 000&nbsp;</th><th>&nbsp;&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.092&nbsp;</td><td>0.463&nbsp;</td><td>n.s.&nbsp;</td><td>2.141&nbsp;</td><td>0.016&nbsp;</td><td>0.117&nbsp;</td><td>3.967&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.645 \times 10^{\text{$-$}5}</span><script type="math/tex">3.645 \times 10^{\text{$-$}5}</script></span>&nbsp;</td><td>0.133&nbsp;</td><td>5.545&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.476 \times 10^{\text{-}8}</span><script type="math/tex">1.476 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.116&nbsp;</td><td>8.151&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.116&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.089&nbsp;</td><td>0.464&nbsp;</td><td>n.s.&nbsp;</td><td>2.542&nbsp;</td><td>0.006&nbsp;</td><td>0.132&nbsp;</td><td>4.397&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.498 \times 10^{\text{-}6}</span><script type="math/tex">5.498 \times 10^{\text{-}6}</script></span>&nbsp;</td><td>0.144&nbsp;</td><td>6.795&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.450 \times 10^{\text{-}12}</span><script type="math/tex">5.450 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.136&nbsp;</td><td>8.092&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.115&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>0.2457&nbsp;</td><td>0.399&nbsp;</td><td>n.s.&nbsp;</td><td>2.155&nbsp;</td><td>0.016&nbsp;</td><td>0.113&nbsp;</td><td>3.326&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.403 \times 10^{\text{-}4}</span><script type="math/tex">4.403 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.117&nbsp;</td><td>5.421&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.975 \times 10^{\text{-}8}</span><script type="math/tex">2.975 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.110&nbsp;</td><td>8.682&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.122&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.663&nbsp;</td><td>0.004&nbsp;</td><td>0.132&nbsp;</td><td>2.458&nbsp;</td><td>0.007&nbsp;</td><td>0.091&nbsp;</td><td>5.538&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.536 \times 10^{\text{-}8}</span><script type="math/tex">1.536 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.115&nbsp;</td><td>8.637&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.315&nbsp;</td><td>0.010&nbsp;</td><td>0.128&nbsp;</td><td>4.217&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.241 \times 10^{\text{-}5}</span><script type="math/tex">1.241 \times 10^{\text{-}5}</script></span>&nbsp;</td><td>0.135&nbsp;</td><td>5.664&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">7.426 \times 10^{\text{-}9}</span><script type="math/tex">7.426 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.118&nbsp;</td><td>8.563&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr></tbody></table></div><div content-id="T4" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10 000&nbsp;</th><th colspan="3">50 000&nbsp;</th><th colspan="3">100 000&nbsp;</th><th colspan="3">250 000&nbsp;</th><th colspan="3">500 000&nbsp;</th><th>&nbsp;&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.092&nbsp;</td><td>0.463&nbsp;</td><td>n.s.&nbsp;</td><td>2.141&nbsp;</td><td>0.016&nbsp;</td><td>0.117&nbsp;</td><td>3.967&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.645 \times 10^{\text{$-$}5}</span><script type="math/tex">3.645 \times 10^{\text{$-$}5}</script></span>&nbsp;</td><td>0.133&nbsp;</td><td>5.545&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.476 \times 10^{\text{-}8}</span><script type="math/tex">1.476 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.116&nbsp;</td><td>8.151&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.116&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.089&nbsp;</td><td>0.464&nbsp;</td><td>n.s.&nbsp;</td><td>2.542&nbsp;</td><td>0.006&nbsp;</td><td>0.132&nbsp;</td><td>4.397&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.498 \times 10^{\text{-}6}</span><script type="math/tex">5.498 \times 10^{\text{-}6}</script></span>&nbsp;</td><td>0.144&nbsp;</td><td>6.795&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.450 \times 10^{\text{-}12}</span><script type="math/tex">5.450 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.136&nbsp;</td><td>8.092&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.115&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>0.2457&nbsp;</td><td>0.399&nbsp;</td><td>n.s.&nbsp;</td><td>2.155&nbsp;</td><td>0.016&nbsp;</td><td>0.113&nbsp;</td><td>3.326&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.403 \times 10^{\text{-}4}</span><script type="math/tex">4.403 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.117&nbsp;</td><td>5.421&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.975 \times 10^{\text{-}8}</span><script type="math/tex">2.975 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.110&nbsp;</td><td>8.682&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.122&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.663&nbsp;</td><td>0.004&nbsp;</td><td>0.132&nbsp;</td><td>2.458&nbsp;</td><td>0.007&nbsp;</td><td>0.091&nbsp;</td><td>5.538&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.536 \times 10^{\text{-}8}</span><script type="math/tex">1.536 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.115&nbsp;</td><td>8.637&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.315&nbsp;</td><td>0.010&nbsp;</td><td>0.128&nbsp;</td><td>4.217&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.241 \times 10^{\text{-}5}</span><script type="math/tex">1.241 \times 10^{\text{-}5}</script></span>&nbsp;</td><td>0.135&nbsp;</td><td>5.664&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">7.426 \times 10^{\text{-}9}</span><script type="math/tex">7.426 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.118&nbsp;</td><td>8.563&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn4"><div class="fn"><p>n.s. = not significant. All individuals were equally admixed with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\gamma=0.1</span><script type="math/tex">\gamma=0.1</script></span>.</p></div></div></div></div></div><div class="table-wrap" content-id="T4"><div id="T4" data-id="T4" class="table-wrap-title"><div class="label">Table 4.</div><div class="caption"><p>Results of the individual-level hybridization detection analyses for the 10% admixture simulation using the <em>individual_hyde.py</em> script</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10 000&nbsp;</th><th colspan="3">50 000&nbsp;</th><th colspan="3">100 000&nbsp;</th><th colspan="3">250 000&nbsp;</th><th colspan="3">500 000&nbsp;</th><th>&nbsp;&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.092&nbsp;</td><td>0.463&nbsp;</td><td>n.s.&nbsp;</td><td>2.141&nbsp;</td><td>0.016&nbsp;</td><td>0.117&nbsp;</td><td>3.967&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.645 \times 10^{\text{$-$}5}</span><script type="math/tex">3.645 \times 10^{\text{$-$}5}</script></span>&nbsp;</td><td>0.133&nbsp;</td><td>5.545&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.476 \times 10^{\text{-}8}</span><script type="math/tex">1.476 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.116&nbsp;</td><td>8.151&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.116&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.089&nbsp;</td><td>0.464&nbsp;</td><td>n.s.&nbsp;</td><td>2.542&nbsp;</td><td>0.006&nbsp;</td><td>0.132&nbsp;</td><td>4.397&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.498 \times 10^{\text{-}6}</span><script type="math/tex">5.498 \times 10^{\text{-}6}</script></span>&nbsp;</td><td>0.144&nbsp;</td><td>6.795&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.450 \times 10^{\text{-}12}</span><script type="math/tex">5.450 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.136&nbsp;</td><td>8.092&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.115&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>0.2457&nbsp;</td><td>0.399&nbsp;</td><td>n.s.&nbsp;</td><td>2.155&nbsp;</td><td>0.016&nbsp;</td><td>0.113&nbsp;</td><td>3.326&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.403 \times 10^{\text{-}4}</span><script type="math/tex">4.403 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.117&nbsp;</td><td>5.421&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.975 \times 10^{\text{-}8}</span><script type="math/tex">2.975 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.110&nbsp;</td><td>8.682&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.122&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.663&nbsp;</td><td>0.004&nbsp;</td><td>0.132&nbsp;</td><td>2.458&nbsp;</td><td>0.007&nbsp;</td><td>0.091&nbsp;</td><td>5.538&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.536 \times 10^{\text{-}8}</span><script type="math/tex">1.536 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.115&nbsp;</td><td>8.637&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.315&nbsp;</td><td>0.010&nbsp;</td><td>0.128&nbsp;</td><td>4.217&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.241 \times 10^{\text{-}5}</span><script type="math/tex">1.241 \times 10^{\text{-}5}</script></span>&nbsp;</td><td>0.135&nbsp;</td><td>5.664&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">7.426 \times 10^{\text{-}9}</span><script type="math/tex">7.426 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.118&nbsp;</td><td>8.563&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr></tbody></table></div><div content-id="T4" class="table-modal"><table><thead name="thead" align="left"><tr><th>&nbsp;&nbsp;</th><th colspan="3">10 000&nbsp;</th><th colspan="3">50 000&nbsp;</th><th colspan="3">100 000&nbsp;</th><th colspan="3">250 000&nbsp;</th><th colspan="3">500 000&nbsp;</th><th>&nbsp;&nbsp;</th></tr><tr><th>Individual&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th><th><em>Z</em>-score&nbsp;</th><th><em>P</em>-value&nbsp;</th><th><span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\boldsymbol{\gamma}}</span><script type="math/tex">\hat{\boldsymbol{\gamma}}</script></span>&nbsp;</th></tr></thead><tbody><tr><td>i6&nbsp;</td><td>0.092&nbsp;</td><td>0.463&nbsp;</td><td>n.s.&nbsp;</td><td>2.141&nbsp;</td><td>0.016&nbsp;</td><td>0.117&nbsp;</td><td>3.967&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">3.645 \times 10^{\text{$-$}5}</span><script type="math/tex">3.645 \times 10^{\text{$-$}5}</script></span>&nbsp;</td><td>0.133&nbsp;</td><td>5.545&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.476 \times 10^{\text{-}8}</span><script type="math/tex">1.476 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.116&nbsp;</td><td>8.151&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.116&nbsp;</td></tr><tr><td>i7&nbsp;</td><td>0.089&nbsp;</td><td>0.464&nbsp;</td><td>n.s.&nbsp;</td><td>2.542&nbsp;</td><td>0.006&nbsp;</td><td>0.132&nbsp;</td><td>4.397&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.498 \times 10^{\text{-}6}</span><script type="math/tex">5.498 \times 10^{\text{-}6}</script></span>&nbsp;</td><td>0.144&nbsp;</td><td>6.795&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">5.450 \times 10^{\text{-}12}</span><script type="math/tex">5.450 \times 10^{\text{-}12}</script></span>&nbsp;</td><td>0.136&nbsp;</td><td>8.092&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.220 \times 10^{\text{-}16}</span><script type="math/tex">2.220 \times 10^{\text{-}16}</script></span>&nbsp;</td><td>0.115&nbsp;</td></tr><tr><td>i8&nbsp;</td><td>0.2457&nbsp;</td><td>0.399&nbsp;</td><td>n.s.&nbsp;</td><td>2.155&nbsp;</td><td>0.016&nbsp;</td><td>0.113&nbsp;</td><td>3.326&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">4.403 \times 10^{\text{-}4}</span><script type="math/tex">4.403 \times 10^{\text{-}4}</script></span>&nbsp;</td><td>0.117&nbsp;</td><td>5.421&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">2.975 \times 10^{\text{-}8}</span><script type="math/tex">2.975 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.110&nbsp;</td><td>8.682&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.122&nbsp;</td></tr><tr><td>i9&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.663&nbsp;</td><td>0.004&nbsp;</td><td>0.132&nbsp;</td><td>2.458&nbsp;</td><td>0.007&nbsp;</td><td>0.091&nbsp;</td><td>5.538&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.536 \times 10^{\text{-}8}</span><script type="math/tex">1.536 \times 10^{\text{-}8}</script></span>&nbsp;</td><td>0.115&nbsp;</td><td>8.637&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr><tr><td>i10&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td><td>2.315&nbsp;</td><td>0.010&nbsp;</td><td>0.128&nbsp;</td><td>4.217&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">1.241 \times 10^{\text{-}5}</span><script type="math/tex">1.241 \times 10^{\text{-}5}</script></span>&nbsp;</td><td>0.135&nbsp;</td><td>5.664&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">7.426 \times 10^{\text{-}9}</span><script type="math/tex">7.426 \times 10^{\text{-}9}</script></span>&nbsp;</td><td>0.118&nbsp;</td><td>8.563&nbsp;</td><td><span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\sim}0.0</span><script type="math/tex">{\sim}0.0</script></span>&nbsp;</td><td>0.119&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="fn4"><div class="fn"><p>n.s. = not significant. All individuals were equally admixed with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\gamma=0.1</span><script type="math/tex">\gamma=0.1</script></span>.</p></div></div></div></div><p></p>                    <h2 scrollto-destination="120230389" id="120230389" class="section-title" data-legacy-id="SEC3">BIOLOGICAL EXAMPLES</h2>
                    <h3 scrollto-destination="120230390" id="120230390" class="section-title" data-legacy-id="SEC3.1">Heliconius Butterflies</h3>
<p>DNA sequence data from <a reveal-id="B26" data-open="B26" class="link link-ref link-reveal xref-bibr">Martin et al. (2013)</a> was downloaded for four populations of <em>Heliconius</em>
 butterflies (248,822,400 sites; available on Dryad). The number of 
individuals per population was as follows: four individuals of <em>H. melpomene rosina</em>, four individuals of <em>H. melpomene timareta</em>, four individuals of <em>H. cydno</em>, and one individual of <em>H. hecale</em>. We tested the hypothesis that <em>H. cydno</em> is a hybrid between <em>H. melpomene rosina</em> and <em>H. melpomene timareta</em> with <em>H. hecale</em> as an outgroup using the <em>run_hyde.py</em> script. We then conducted bootstrapping (500 replicates) of the <em>H. cydno</em> individuals, as well as testing each individual separately using the <em>bootstrap_hyde.py</em> and <em>individual_hyde.py</em> scripts, respectively. As a last comparison, we reanalyzed the data with the <em>run_hyde.py</em>
 script, but changed the underlying code so that all missing or 
ambiguous sites were ignored in order to test the effect of 
including/excluding such data.</p>                    <h4 scrollto-destination="120230392" id="120230392" class="section-title" data-legacy-id="SEC3.1.1">Results.</h4>
<p>Significant hybridization was detected in <em>H. cydno</em> at the population level (<span class="inline-formula no-formula-id"><span class="MathJax_Preview">\text{Z-score} = 472.599</span><script type="math/tex">\text{Z-score} = 472.599</script></span>, <span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\textit{P}}-value = {\sim}0.0</span><script type="math/tex">{\textit{P}}-value = {\sim}0.0</script></span>, <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma} = 0.342</span><script type="math/tex">\hat{\gamma} = 0.342</script></span>). Each individual also showed significant levels of hybridization, with <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> ranging from 0.324 to 0.393, indicating that hybridization in <em>H. cydno</em> is mostly uniform across the individuals sampled. <a reveal-id="F4" data-open="F4" class="link link-reveal link-table xref-fig">Figure 4</a> shows a density plot for the estimated values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>
 across the 500 bootstrap replicates. Although the distribution does 
exhibit some amount of jumping between different values they are still 
between the lower and upper bounds for the individual level estimates of
 <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span>, providing corroborating evidence that all the individuals in the <em>H. cydno</em>
 population are admixed. When we ignored missing or ambiguous sites, we 
still detected significant hybridization, but the value of the test 
statistic was less than half of the value from the original analysis (<span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\textit{Z}-score} = 189.073</span><script type="math/tex">{\textit{Z}-score} = 189.073</script></span>, <span class="inline-formula no-formula-id"><span class="MathJax_Preview">{\textit{P}}-value = {\sim}0.0</span><script type="math/tex">{\textit{P}}-value = {\sim}0.0</script></span>, <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma} = 0.395</span><script type="math/tex">\hat{\gamma} = 0.395</script></span>). This demonstrates the increase in power that can result from including <em>all</em> relevant data in the analysis.</p>                        <a id="120230394" scrollto-destination="120230394"></a>
<div id="F4" data-id="F4" class="fig fig-section"><div class="label fig-label">Fig. 4.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f4.png" alt="Density plot of the estimated values of $\hat{\gamma}$ across 500 bootstrap replicates testing for hybridization in Heliconius cydno between H. melpomene rosina and H. melpomene timareta." path-from-xml="syy023f4.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230394/syy023f4.tif" path-from-xml="syy023f4.tif" target="_blank">Open in new tab</a><a section="120230394" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f4.png?Expires=1577481743&amp;Signature=NNhonh-3d4MTsFZADZ4Q-mRpEWZKgIA73dXJ45GjdD0hImNvaUcf42eu%7ErxrLdnu1gCWrgYEMX8JHtGXgdoFpWhIzWjybJ6pjjM2IFP6d1d9KQ5y3Wf1P9rMkkP7UaDYPVg-Cfhl%7Edxsy19ExjKNsKbce6vfItBqIp88wf%7ETnwrfB-ngtxlxGRgTBdv0sFqGMB8FciAGxreKzewDB022aKZbioVXbzQ-Klu-VWrX8INXGUhX-Y2xWWEwnym3hgqTXH1GArhvzUZ83ivW8XyyJqesUvY%7EpKSg7NTw5pZOdF7FuCIMxJRiXTwq8BMygjPV%7EFdcI8VzNJp91VOeqW3RzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230394&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f4.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Density plot of the estimated values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> across 500 bootstrap replicates testing for hybridization in <em>Heliconius cydno</em> between <em>H. melpomene rosina</em> and <em>H. melpomene timareta</em>.</p></div></div><div content-id="F4" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 4.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f4.png" alt="Density plot of the estimated values of $\hat{\gamma}$ across 500 bootstrap replicates testing for hybridization in Heliconius cydno between H. melpomene rosina and H. melpomene timareta." path-from-xml="syy023f4.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230394/syy023f4.tif" path-from-xml="syy023f4.tif" target="_blank">Open in new tab</a><a section="120230394" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f4.png?Expires=1577481743&amp;Signature=NNhonh-3d4MTsFZADZ4Q-mRpEWZKgIA73dXJ45GjdD0hImNvaUcf42eu%7ErxrLdnu1gCWrgYEMX8JHtGXgdoFpWhIzWjybJ6pjjM2IFP6d1d9KQ5y3Wf1P9rMkkP7UaDYPVg-Cfhl%7Edxsy19ExjKNsKbce6vfItBqIp88wf%7ETnwrfB-ngtxlxGRgTBdv0sFqGMB8FciAGxreKzewDB022aKZbioVXbzQ-Klu-VWrX8INXGUhX-Y2xWWEwnym3hgqTXH1GArhvzUZ83ivW8XyyJqesUvY%7EpKSg7NTw5pZOdF7FuCIMxJRiXTwq8BMygjPV%7EFdcI8VzNJp91VOeqW3RzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230394&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f4.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Density plot of the estimated values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> across 500 bootstrap replicates testing for hybridization in <em>Heliconius cydno</em> between <em>H. melpomene rosina</em> and <em>H. melpomene timareta</em>.</p></div></div>                    <h3 scrollto-destination="120230395" id="120230395" class="section-title" data-legacy-id="SEC3.2">Swordtail Fish <em>(Xiphophorous)</em></h3>
<p>Transcriptome data from <a reveal-id="B8" data-open="B8" class="link link-ref link-reveal xref-bibr">Cui et al. (2013)</a> was obtained from the authors for 26 species of swordtail fish (25 species of <em>Xiphophorous</em> + 1 outgroup from the genus <em>Priapella</em>).
 The data (3,706,285 sites) were first put in sequential Phylip format 
for analysis with HyDe. Each taxon was then analyzed at the individual 
level, resulting in <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\binom{25}{3} \times 3 = 6900</span><script type="math/tex">\binom{25}{3} \times 3 = 6900</script></span> hypothesis tests. Because there were not multiple individuals per taxon, we only analyzed these data using the <em>run_hyde.py</em> script. As with the <em>Heliconius</em> example, we also reanalyzed these data ignoring all sites with missing or ambiguous bases.</p>                    <h4 scrollto-destination="120230397" id="120230397" class="section-title" data-legacy-id="SEC3.2.1">Results.</h4>
<p>Out of 6900 tests, 2199 reported significant levels of hybridization (Bonferroni corrected <em>P</em>-value of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\frac{0.05}{6900}=7.246\times10^{-6}</span><script type="math/tex">\frac{0.05}{6900}=7.246\times10^{-6}</script></span>).
 When ambiguous or missing bases were ignored, only 867 tests were 
reported as significant, again demonstrating the increase in power to 
detect hybridization when including these sites. The distribution of the
 <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> values for the 2199 significant tests is plotted in <a reveal-id="F5" data-open="F5" class="link link-reveal link-table xref-fig">Fig. 5</a>. This plot shows that most of the admixture occurring in <em>Xiphophorous</em> is at low levels (<span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> close to either 0.0 or 1.0). However, there are many instances of higher levels of admixture, which matches the findings of <a reveal-id="B8" data-open="B8" class="link link-ref link-reveal xref-bibr">Cui et al. (2013)</a> regarding the extensive amount of gene flow and introgression in this group. <a reveal-id="B34" data-open="B34" class="link link-ref link-reveal xref-bibr">Sols-Lumis and An (2016)</a> also analyzed these data and found evidence for an ancestral hybridization event between <em>X. xiphidium</em> and the northern swordtail clade (denoted NS in <a reveal-id="B34" data-open="B34" class="link link-ref link-reveal xref-bibr">Sols-Lumis and An (2016)</a>, Fig. 10), a pattern we recover as well (259 significant hypothesis tests involving <em>X. xiphidium</em>).</p>                        <a id="120230399" scrollto-destination="120230399"></a>
<div id="F5" data-id="F5" class="fig fig-section"><div class="label fig-label">Fig. 5.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f5.png" alt="Density plot of the estimated values of $\hat{\gamma}$ across the 2199 hypothesis tests that detected significant levels of hybridization in Xiphophorous." path-from-xml="syy023f5.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230399/syy023f5.tif" path-from-xml="syy023f5.tif" target="_blank">Open in new tab</a><a section="120230399" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f5.png?Expires=1577481743&amp;Signature=Wo-qaQfudHgLya8uvPLTe3Wcz4w1FUW-fQM5H2oOGKOXY1qLxC2AbBE42o5ei7cT65T3UrGJ7P0mPb0EN8tvWSR11P8OK6TDIEgHFFbSSqyWrQLk9IJEOjKBUT1KxfYxvx7D3fwu0Z6JP6qkTKnpvJcbXDus7vBtgUZKBI%7Em%7EFbQMPFh9sovwCpvjvfBDbJ90FLzYhJEIBv2I-4n%7EkvnYS1%7ECBsrpIOP6wE0Nv8t1hOYhMxGvBq8QhQgX-HuZhYEOvbjI2wrWEmRXM2DKUZCPRvq5evbGkZg4-6mp%7EnOtGTHl7Zwy2RmiMwqvQgosO30792F2EBnSZEwnNuKnoIWMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230399&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f5.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Density plot of the estimated values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> across the 2199 hypothesis tests that detected significant levels of hybridization in <em>Xiphophorous</em>.</p></div></div><div content-id="F5" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 5.</div><div class="graphic-wrap"><img class="content-image" src="m_syy023f5.png" alt="Density plot of the estimated values of $\hat{\gamma}$ across the 2199 hypothesis tests that detected significant levels of hybridization in Xiphophorous." path-from-xml="syy023f5.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/120230399/syy023f5.tif" path-from-xml="syy023f5.tif" target="_blank">Open in new tab</a><a section="120230399" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/67/5/10.1093_sysbio_syy023/2/syy023f5.png?Expires=1577481743&amp;Signature=Wo-qaQfudHgLya8uvPLTe3Wcz4w1FUW-fQM5H2oOGKOXY1qLxC2AbBE42o5ei7cT65T3UrGJ7P0mPb0EN8tvWSR11P8OK6TDIEgHFFbSSqyWrQLk9IJEOjKBUT1KxfYxvx7D3fwu0Z6JP6qkTKnpvJcbXDus7vBtgUZKBI%7Em%7EFbQMPFh9sovwCpvjvfBDbJ90FLzYhJEIBv2I-4n%7EkvnYS1%7ECBsrpIOP6wE0Nv8t1hOYhMxGvBq8QhQgX-HuZhYEOvbjI2wrWEmRXM2DKUZCPRvq5evbGkZg4-6mp%7EnOtGTHl7Zwy2RmiMwqvQgosO30792F2EBnSZEwnNuKnoIWMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=120230399&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="syy023f5.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Density plot of the estimated values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview">\hat{\gamma}</span><script type="math/tex">\hat{\gamma}</script></span> across the 2199 hypothesis tests that detected significant levels of hybridization in <em>Xiphophorous</em>.</p></div></div>                    <h3 scrollto-destination="120230400" id="120230400" class="section-title" data-legacy-id="SEC3.3">Timing</h3>
<p>We timed all analyses for the <em>Heliconius</em> and swordtail data using the Unix <span class="monospace">time</span> command. For <em>Heliconius</em>, the analysis using the <em>run_hyde.py</em>
 script took 11m 49.377s, and the analysis that ignored missing or 
ambiguous sites took 4m 47.667s. Analyzing the four individuals of <em>H. cydno</em> separately with the <em>individual_hyde.py</em>
 script took 10m 5.120s. Completing 500 bootstrap replicates for this 
data set ran the longest of any analysis that we conducted, taking a 
total of 4105m 33.312s (2.851 days). For the swordtail data, the 
analysis with the <em>run_hyde.py</em> script took 22m 40.673s, almost twice as long as the analysis for the <em>Heliconius</em>
 data. This indicates that analyzing more triplets is more 
computationally intensive than analyzing more sites. When ambiguous or 
missing sites were ignored, the analysis took 5m 27.889s.</p>                    <h2 scrollto-destination="120230402" id="120230402" class="section-title" data-legacy-id="SEC4">CONCLUSIONS</h2>
<p>Hybridization and gene flow are increasingly recognized as important forces in the evolution of taxa across the Tree of Life (<a reveal-id="B33" data-open="B33" class="link link-ref link-reveal xref-bibr">Seehausen 2004</a>; <a reveal-id="B2" data-open="B2" class="link link-ref link-reveal xref-bibr">Arnold and Kunte 2017</a>; <a reveal-id="B17" data-open="B17" class="link link-ref link-reveal xref-bibr">Kagawa and Takimoto 2017</a>).
 In addition, for groups such as plants, fish, and amphibians, 
hybridization can also be accompanied by whole genome duplication 
(allopolyploidy; <a reveal-id="B29" data-open="B29" class="link link-ref link-reveal xref-bibr">Otto and Whitton 2000</a>). <a reveal-id="B18" data-open="B18" class="link link-ref link-reveal xref-bibr">Kamneva et al. (2017)</a>
 recently used methods based on the coalescent with hybridization to 
model haplotypes in allopolyploid strawberries, and we anticipate that 
approaches using site pattern probabilities should be equally applicable
 in such situations. Gaining an understanding of when these complex 
processes are occurring and how they may vary among individuals within a
 population are important steps to understand the spatial distribution 
of shared genetic variation among diverging lineages. The methods we 
have implemented in HyDe to detect hybridization at the population and 
individual level provide a set of computationally efficient methods for 
researchers to assess patterns of admixture in natural populations. HyDe
 also provides an open source Python library for the calculation of site
 pattern probabilities that can easily be extended to calculate 
additional statistics that may be developed in the future. As access to 
genomic data continues to grow, we anticipate that methods such as HyDe 
that use phylogenetic invariants will play an important role for 
phylogenomic inferences in non-model species.</p>                    <h2 scrollto-destination="120230404" id="120230404" class="section-title" data-legacy-id="SEC5">AVAILABILITY</h2>
<p>HyDe is available as an open source Python package distributed under 
the GNU General Public License v3 for both Python 2.7 and 3.6. 
Developmental code can be found on GitHub (<a class="link link-uri" href="https://github.com/pblischak/HyDe" target="">https://github.com/pblischak/HyDe</a>) with official code releases uploaded to the Python Package Index (PyPI: <a class="link link-uri" href="https://pypi.python.org/pypi/phyde" target="">https://pypi.python.org/pypi/phyde</a>). Documentation for installing and running HyDe can be found on ReadTheDocs (<a class="link link-uri" href="http://hybridization-detection.readthedocs.io/" target="">http://hybridization-detection.readthedocs.io</a>).</p>                    <h2 scrollto-destination="120230406" id="120230406" class="section-title" data-legacy-id="SEC6">SUPPLEMENTARY MATERIALS</h2>
<p>Data available from the Dryad Digital Repository: <a class="link link-uri" href="http://dx.doi.org/10.5061/dryad.372sq" target="">http://dx.doi.org/10.5061/dryad.372sq</a>.</p>                    <h2 scrollto-destination="120230408" id="120230408" class="backsection-title">FUNDING</h2>
<p>This work was supported in part by grants from the National Science 
Foundation under the awards DMS-1106706 (J.C. and L.S.K.) and 
DEB-1455399 (A.D.W. and L.S.K.), and National Institutes of Health 
Cancer Biology Training Grant T32-CA079448 at Wake Forest School of 
Medicine (J.C.).</p>                    <h2 scrollto-destination="120230410" id="120230410" class="backacknowledgements-title">ACKNOWLEDGEMENTS</h2>
<p>The authors thank Ccile An and Frank Burbink for the invitation to 
be a part of this special issue on the impact of gene flow and 
reticulation in phylogenetics, as well as David Posada and three 
anonymous reviewers for comments that improved the manuscript. We also 
thank R. Cui, C. Sols-Lemus, and C. An for their help with the <em>Xiphophorous</em> data set.</p>                    <h2 scrollto-destination="120230412" id="120230412" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B1" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Allman</div>  <div class="given-names">E.S.,</div></div> <div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.,</div></div> <div class="name string-name"><div class="surname">Rhodes</div>  <div class="given-names">J.A.</div></div></div> <div class="year">2016</div>. <div class="article-title">Split scores: a tool to quantify phylogenetic signal in genome-scale data.</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">66</div>:<div class="fpage">620</div><div class="lpage">636</div>.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Split%20scores%3A%20a%20tool%20to%20quantify%20phylogenetic%20signal%20in%20genome-scale%20data.&amp;aulast=Allman&amp;date=2016&amp;spage=620&amp;epage=636&amp;volume=66" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Split%20scores%3A%20a%20tool%20to%20quantify%20phylogenetic%20signal%20in%20genome-scale%20data.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B2" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arnold</div>  <div class="given-names">M.J.,</div></div> <div class="name string-name"><div class="surname">Kunte</div>  <div class="given-names">K.</div></div></div> <div class="year">2017</div>. <div class="article-title">Adaptive genetic exchange: a tangled history of admixture and evolutionary innovation.</div> <div class="source">Trends Ecol. Evol.</div>&nbsp; <div class="volume">32</div>:<div class="fpage">601</div><div class="lpage">611</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Adaptive%20genetic%20exchange%3A%20a%20tangled%20history%20of%20admixture%20and%20evolutionary%20innovation.&amp;author=M.J.%2C%20Arnold&amp;author=K.%20Kunte&amp;publication_year=2017&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=32&amp;pages=601-611" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tree.2017.05.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2017.05.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tree.2017.05.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28645486" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Adaptive%20genetic%20exchange%3A%20a%20tangled%20history%20of%20admixture%20and%20evolutionary%20innovation.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B3" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baack</div>  <div class="given-names">E.J.,</div></div> <div class="name string-name"><div class="surname">Rieseberg</div>  <div class="given-names">L.H.</div></div></div> <div class="year">2007</div>. <div class="article-title">A genomic view of introgression and hybrid speciation.</div> <div class="source">Curr. Opin. Genet. Dev.</div>&nbsp; <div class="volume">17</div>:<div class="fpage">513</div><div class="lpage">518</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20view%20of%20introgression%20and%20hybrid%20speciation.&amp;author=E.J.%2C%20Baack&amp;author=L.H.%20Rieseberg&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=17&amp;pages=513-518" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.gde.2007.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2007.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.gde.2007.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17933508" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20genomic%20view%20of%20introgression%20and%20hybrid%20speciation.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B4" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Behnel</div>  <div class="given-names">S.,</div></div> <div class="name string-name"><div class="surname">Bradshaw</div>  <div class="given-names">R.,</div></div> <div class="name string-name"><div class="surname">Citro</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Dalcin</div>  <div class="given-names">L.,</div></div> <div class="name string-name"><div class="surname">Seljebotn</div>  <div class="given-names">D.,</div></div> <div class="name string-name"><div class="surname">Smith</div>  <div class="given-names">K.</div></div></div> <div class="year">2010</div>. <div class="article-title">Cython: the best of both worlds.</div> <div class="source">Comput. Sci. Eng.</div>&nbsp; <div class="volume">13</div>:<div class="fpage">31</div><div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Cython%3A%20the%20best%20of%20both%20worlds.&amp;author=S.%2C%20Behnel&amp;author=R.%2C%20Bradshaw&amp;author=C.%2C%20Citro&amp;author=L.%2C%20Dalcin&amp;author=D.%2C%20Seljebotn&amp;author=K.%20Smith&amp;publication_year=2010&amp;journal=Comput.%20Sci.%20Eng.&amp;volume=13&amp;pages=31-39" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1109/MCSE.2010.118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FMCSE.2010.118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1109%2FMCSE.2010.118"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Cython%3A%20the%20best%20of%20both%20worlds.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B5" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cavender</div>  <div class="given-names">J.A.,</div></div> <div class="name string-name"><div class="surname">Felsenstein</div>  <div class="given-names">J.</div></div></div> <div class="year">1987</div>. <div class="article-title">Invariants of phylogenies in a simple case with discrete states.</div> <div class="source">J. Classification</div>&nbsp; <div class="volume">4</div>:<div class="fpage">57</div><div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Invariants%20of%20phylogenies%20in%20a%20simple%20case%20with%20discrete%20states.&amp;author=J.A.%2C%20Cavender&amp;author=J.%20Felsenstein&amp;publication_year=1987&amp;journal=J.%20Classification&amp;volume=4&amp;pages=57-71" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF01890075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01890075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF01890075"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Invariants%20of%20phylogenies%20in%20a%20simple%20case%20with%20discrete%20states.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B6" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chifman</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.</div></div></div> <div class="year">2014</div>. <div class="article-title">Quartet inference from SNP data under the coalescent model.</div> <div class="source">Bioinformatics</div>&nbsp; <div class="volume">30</div>:<div class="fpage">3317</div><div class="lpage">3324</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Quartet%20inference%20from%20SNP%20data%20under%20the%20coalescent%20model.&amp;author=J.%2C%20Chifman&amp;author=L.S.%20Kubatko&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3317-3324" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu530" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu530" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu530"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25104814" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Quartet%20inference%20from%20SNP%20data%20under%20the%20coalescent%20model.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B7" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chifman</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.</div></div></div> <div class="year">2015</div>. <div class="article-title">Identifiability
 of the unrooted species tree topology under the coalescent model with 
time-reversible substitution processes, site-specific rate variation, 
and invariable sites.</div> <div class="source">J. Theor. Biol.</div>&nbsp; <div class="volume">374</div>:<div class="fpage">35</div><div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identifiability%20of%20the%20unrooted%20species%20tree%20topology%20under%20the%20coalescent%20model%20with%20time-reversible%20substitution%20processes%2C%20site-specific%20rate%20variation%2C%20and%20invariable%20sites.&amp;author=J.%2C%20Chifman&amp;author=L.S.%20Kubatko&amp;publication_year=2015&amp;journal=J.%20Theor.%20Biol.&amp;volume=374&amp;pages=35-47" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.jtbi.2015.03.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2015.03.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.jtbi.2015.03.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25791286" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identifiability%20of%20the%20unrooted%20species%20tree%20topology%20under%20the%20coalescent%20model%20with%20time-reversible%20substitution%20processes%2C%20site-specific%20rate%20variation%2C%20and%20invariable%20sites.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B8" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cui</div>  <div class="given-names">R.,</div></div> <div class="name string-name"><div class="surname">Schumer</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Kruesi</div>  <div class="given-names">K.,</div></div> <div class="name string-name"><div class="surname">Walter</div>  <div class="given-names">R.,</div></div> <div class="name string-name"><div class="surname">Andolfatto</div>  <div class="given-names">P.,</div></div> <div class="name string-name"><div class="surname">Rosenthal</div>  <div class="given-names">G.G.</div></div></div> <div class="year">2013</div>. <div class="article-title">Phylogenomics reveals extensive reticulate evolution in <em>Xiphophorus</em> fishes.</div> <div class="source">Evolution</div>&nbsp; <div class="volume">67</div>:<div class="fpage">2166</div><div class="lpage">2179</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogenomics%20reveals%20extensive%20reticulate%20evolution%20in%20Xiphophorus%20fishes.&amp;author=R.%2C%20Cui&amp;author=M.%2C%20Schumer&amp;author=K.%2C%20Kruesi&amp;author=R.%2C%20Walter&amp;author=P.%2C%20Andolfatto&amp;author=G.G.%20Rosenthal&amp;publication_year=2013&amp;journal=Evolution&amp;volume=67&amp;pages=2166-2179" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/evo.12099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.12099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fevo.12099"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23888843" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogenomics%20reveals%20extensive%20reticulate%20evolution%20in%20Xiphophorus%20fishes.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B9" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Durand</div>  <div class="given-names">E.Y.,</div></div> <div class="name string-name"><div class="surname">Patterson</div>  <div class="given-names">N.,</div></div> <div class="name string-name"><div class="surname">Reich</div>  <div class="given-names">D.,</div></div> <div class="name string-name"><div class="surname">Slatkin</div>  <div class="given-names">M.</div></div></div> <div class="year">2011</div>. <div class="article-title">Testing for ancient admixture between closely related populations.</div> <div class="source">Mol. Biol. Evol.</div>&nbsp; <div class="volume">28</div>:<div class="fpage">2239</div><div class="lpage">2252</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Testing%20for%20ancient%20admixture%20between%20closely%20related%20populations.&amp;author=E.Y.%2C%20Durand&amp;author=N.%2C%20Patterson&amp;author=D.%2C%20Reich&amp;author=M.%20Slatkin&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2239-2252" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msr048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsr048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21325092" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Testing%20for%20ancient%20admixture%20between%20closely%20related%20populations.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B10" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eaton</div>  <div class="given-names">D.A.R.,</div></div> <div class="name string-name"><div class="surname">Ree</div>  <div class="given-names">R.H.</div></div></div> <div class="year">2013</div>. <div class="article-title">Inferring phylogeny and introgression using RADseq data: an example from the flowering plants (<em>Pedicularis</em>: Orobanchaceae).</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">62</div>:<div class="fpage">689</div><div class="lpage">706</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Inferring%20phylogeny%20and%20introgression%20using%20RADseq%20data%3A%20an%20example%20from%20the%20flowering%20plants%20%28Pedicularis%3A%20Orobanchaceae%29.&amp;author=D.A.R.%2C%20Eaton&amp;author=R.H.%20Ree&amp;publication_year=2013&amp;journal=Syst.%20Biol.&amp;volume=62&amp;pages=689-706" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syt032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyt032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyt032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23652346" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inferring%20phylogeny%20and%20introgression%20using%20RADseq%20data%3A%20an%20example%20from%20the%20flowering%20plants%20%28Pedicularis%3A%20Orobanchaceae%29.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B11" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gerard</div>  <div class="given-names">D.,</div></div> <div class="name string-name"><div class="surname">Gibbs</div>  <div class="given-names">H.L.,</div></div> <div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.</div></div></div> <div class="year">2011</div>. <div class="article-title">Estimating hybridization in the presence of coalescence using phylogenetic intraspecific sampling.</div> <div class="source">BMC Evol. Biol.</div>&nbsp; <div class="volume">11</div>:<div class="fpage">291</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Estimating%20hybridization%20in%20the%20presence%20of%20coalescence%20using%20phylogenetic%20intraspecific%20sampling.&amp;author=D.%2C%20Gerard&amp;author=H.L.%2C%20Gibbs&amp;author=L.S.%20Kubatko&amp;publication_year=2011&amp;journal=BMC%20Evol.%20Biol.&amp;volume=11&amp;pages=291" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2148-11-291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-11-291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2148-11-291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21978073" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Estimating%20hybridization%20in%20the%20presence%20of%20coalescence%20using%20phylogenetic%20intraspecific%20sampling.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B12" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Green</div>  <div class="given-names">R.E.,</div></div> <div class="name string-name"><div class="surname">Krause,</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Briggs</div>  <div class="given-names">A.W.,</div></div> <div class="name string-name"><div class="surname">Maricic</div>  <div class="given-names">T.,</div></div> <div class="name string-name"><div class="surname">Stenzel</div>  <div class="given-names">U.,</div></div> <div class="name string-name"><div class="surname">Kircher</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Patterson</div>  <div class="given-names">N.,</div></div> <div class="name string-name"><div class="surname">Li</div>  <div class="given-names">H.,</div></div> <div class="name string-name"><div class="surname">Zhai</div>  <div class="given-names">W.,</div></div> <div class="name string-name"><div class="surname">Fritz</div>  <div class="given-names">M.H.-Y.,</div></div> <div class="name string-name"><div class="surname">Hansen</div>  <div class="given-names">N.F.,</div></div> <div class="name string-name"><div class="surname">Durand</div>  <div class="given-names">E.Y.,</div></div> <div class="name string-name"><div class="surname">Malaspinas</div>  <div class="given-names">A.-S.,</div></div> <div class="name string-name"><div class="surname">Jensen</div>  <div class="given-names">J.D.,</div></div> <div class="name string-name"><div class="surname">Marques-Bonet</div>  <div class="given-names">T.,</div></div> <div class="name string-name"><div class="surname">Alkan</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Prfer</div>  <div class="given-names">K.,</div></div> <div class="name string-name"><div class="surname">Meyer</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Burbano</div>  <div class="given-names">H.A.,</div></div> <div class="name string-name"><div class="surname">Good</div>  <div class="given-names">J.M.,</div></div> <div class="name string-name"><div class="surname">Schultz</div>  <div class="given-names">R.,</div></div> <div class="name string-name"><div class="surname">Aximu-Petri</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Butthof</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Hber</div>  <div class="given-names">B.,</div></div> <div class="name string-name"><div class="surname">Hffner</div>  <div class="given-names">B.,</div></div> <div class="name string-name"><div class="surname">Siegemund</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Weihmann</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Nusbaum</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Lander</div>  <div class="given-names">E.S.,</div></div> <div class="name string-name"><div class="surname">Russ</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Novod</div>  <div class="given-names">N.,</div></div> <div class="name string-name"><div class="surname">Affourtit</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Egholm</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Verna</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Rudan</div>  <div class="given-names">P.,</div></div> <div class="name string-name"><div class="surname">Brajkovic</div>  <div class="given-names">D.,</div></div> <div class="name string-name"><div class="surname">Kucan</div>  <div class="given-names">,</div></div> <div class="name string-name"><div class="surname">Guic</div>  <div class="given-names">I.,</div></div> <div class="name string-name"><div class="surname">Doronichev</div>  <div class="given-names">V.B.,</div></div> <div class="name string-name"><div class="surname">Golovanova</div>  <div class="given-names">L.V.,</div></div> <div class="name string-name"><div class="surname">Lalueza-Fox</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">de la Rasilla</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Fortea</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Rosas</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Schmitz</div>  <div class="given-names">R.W.,</div></div> <div class="name string-name"><div class="surname">Johnson</div>  <div class="given-names">P.L.F.,</div></div> <div class="name string-name"><div class="surname">Eichler</div>  <div class="given-names">E.E.,</div></div> <div class="name string-name"><div class="surname">Falush</div>  <div class="given-names">D.,</div></div> <div class="name string-name"><div class="surname">Birney</div>  <div class="given-names">E.,</div></div> <div class="name string-name"><div class="surname">Mullikin</div>  <div class="given-names">J.C.,</div></div> <div class="name string-name"><div class="surname">Slatkin</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Nielsen</div>  <div class="given-names">R.,</div></div> <div class="name string-name"><div class="surname">Kelso</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Lachmann</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Reich</div>  <div class="given-names">D.,</div></div> <div class="name string-name"><div class="surname">Pbo</div>  <div class="given-names">S.</div></div></div> <div class="year">2010</div>. <div class="article-title">A draft sequence of the Neandertal genome.</div> <div class="source">Science</div>&nbsp; <div class="volume">328</div>:<div class="fpage">710</div><div class="lpage">722</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20of%20the%20Neandertal%20genome.&amp;author=R.E.%2C%20Green&amp;author=J.%2C%20Krause%2C&amp;author=A.W.%2C%20Briggs&amp;author=T.%2C%20Maricic&amp;author=U.%2C%20Stenzel&amp;author=M.%2C%20Kircher&amp;author=N.%2C%20Patterson&amp;author=H.%2C%20Li&amp;author=W.%2C%20Zhai&amp;author=M.H.-Y.%2C%20Fritz&amp;author=N.F.%2C%20Hansen&amp;author=E.Y.%2C%20Durand&amp;author=A.-S.%2C%20Malaspinas&amp;author=J.D.%2C%20Jensen&amp;author=T.%2C%20Marques-Bonet&amp;author=C.%2C%20Alkan&amp;author=K.%2C%20Pr%C3%BCfer&amp;author=M.%2C%20Meyer&amp;author=H.A.%2C%20Burbano&amp;author=J.M.%2C%20Good&amp;author=R.%2C%20Schultz&amp;author=A.%2C%20Aximu-Petri&amp;author=A.%2C%20Butthof&amp;author=B.%2C%20H%C3%B6ber&amp;author=B.%2C%20H%C3%B6ffner&amp;author=M.%2C%20Siegemund&amp;author=A.%2C%20Weihmann&amp;author=C.%2C%20Nusbaum&amp;author=E.S.%2C%20Lander&amp;author=C.%2C%20Russ&amp;author=N.%2C%20Novod&amp;author=J.%2C%20Affourtit&amp;author=M.%2C%20Egholm&amp;author=C.%2C%20Verna&amp;author=P.%2C%20Rudan&amp;author=D.%2C%20Brajkovic&amp;author=%C5%BD%2C%20Kucan&amp;author=I.%2C%20Gu%C5%A1ic&amp;author=V.B.%2C%20Doronichev&amp;author=L.V.%2C%20Golovanova&amp;author=C.%2C%20Lalueza-Fox&amp;author=M.%2C%20de%20la%20Rasilla&amp;author=J.%2C%20Fortea&amp;author=A.%2C%20Rosas&amp;author=R.W.%2C%20Schmitz&amp;author=P.L.F.%2C%20Johnson&amp;author=E.E.%2C%20Eichler&amp;author=D.%2C%20Falush&amp;author=E.%2C%20Birney&amp;author=J.C.%2C%20Mullikin&amp;author=M.%2C%20Slatkin&amp;author=R.%2C%20Nielsen&amp;author=J.%2C%20Kelso&amp;author=M.%2C%20Lachmann&amp;author=D.%2C%20Reich&amp;author=S.%20P%C3%A4%C3%A4bo&amp;publication_year=2010&amp;journal=Science&amp;volume=328&amp;pages=710-722" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1188021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1188021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1188021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20448178" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20draft%20sequence%20of%20the%20Neandertal%20genome.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B13" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hasegawa</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Kishino</div>  <div class="given-names">H.,</div></div> <div class="name string-name"><div class="surname">Yano</div>  <div class="given-names">T.</div></div></div> <div class="year">1985</div>. <div class="article-title">Dating of human-ape splitting by a molecular clock of mitochondrial DNA.</div> <div class="source">J. Mol. Evol.</div>&nbsp; <div class="volume">22</div>:<div class="fpage">160</div><div class="lpage">174</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Dating%20of%20human-ape%20splitting%20by%20a%20molecular%20clock%20of%20mitochondrial%20DNA.&amp;author=M.%2C%20Hasegawa&amp;author=H.%2C%20Kishino&amp;author=T.%20Yano&amp;publication_year=1985&amp;journal=J.%20Mol.%20Evol.&amp;volume=22&amp;pages=160-174" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF02101694" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02101694" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02101694"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3934395" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Dating%20of%20human-ape%20splitting%20by%20a%20molecular%20clock%20of%20mitochondrial%20DNA.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B14" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hudson</div>  <div class="given-names">R.R.</div></div></div> <div class="year">2002</div>. <div class="article-title">Generating samples under a Wright-Fisher neutral model of genetic variation.</div> <div class="source">Bioinformatics</div>&nbsp; <div class="volume">18</div>:<div class="fpage">337</div><div class="lpage">338</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Generating%20samples%20under%20a%20Wright-Fisher%20neutral%20model%20of%20genetic%20variation.&amp;author=R.R.%20Hudson&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=337-338" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/18.2.337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.2.337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F18.2.337"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847089" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Generating%20samples%20under%20a%20Wright-Fisher%20neutral%20model%20of%20genetic%20variation.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B15" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Joly</div>  <div class="given-names">S.,</div></div> <div class="name string-name"><div class="surname">McLenachan</div>  <div class="given-names">P.A.,</div></div> <div class="name string-name"><div class="surname">Lockhart</div>  <div class="given-names">P.J.</div></div></div> <div class="year">2009</div>. <div class="article-title">A statistical approach for distinguishing hybridization and incomplete lineage sorting.</div> <div class="source">Am. Nat.</div>&nbsp; <div class="volume">174</div>:<div class="fpage">E54</div><div class="lpage">E70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20approach%20for%20distinguishing%20hybridization%20and%20incomplete%20lineage%20sorting.&amp;author=S.%2C%20Joly&amp;author=P.A.%2C%20McLenachan&amp;author=P.J.%20Lockhart&amp;publication_year=2009&amp;journal=Am.%20Nat.&amp;volume=174&amp;pages=E54-E70" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1086/600082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F600082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1086%2F600082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19519219" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20statistical%20approach%20for%20distinguishing%20hybridization%20and%20incomplete%20lineage%20sorting.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B16" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jukes</div>  <div class="given-names">T.H.,</div></div> <div class="name string-name"><div class="surname">Cantor</div>  <div class="given-names">C.R.</div></div></div> <div class="year">1969</div>. <div class="article-title">Evolution of protein molecules.</div> In: <div class="person-group"><div class="name string-name"><div class="surname">Monroe</div>  <div class="given-names">H.,</div></div></div> editor. <div class="source">Mammalian protein metabolism.</div>&nbsp; <div class="publisher-loc">New York</div>: <div class="publisher-name">Academic Press.</div> p. <div class="fpage">21</div><div class="lpage">132</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Mammalian%20protein%20metabolism.&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Mammalian%20protein%20metabolism.&amp;aulast=Jukes&amp;date=1969&amp;spage=21&amp;epage=132" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mammalian%20protein%20metabolism.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Mammalian%20protein%20metabolism.">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B17" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kagawa</div>  <div class="given-names">K.,</div></div> <div class="name string-name"><div class="surname">Takimoto,</div>  <div class="given-names">G.</div></div></div> <div class="year">2017</div>. <div class="article-title">Hybridization can promote adaptive radiation by means of transgressive segregation.</div> <div class="source">Ecol. Lett.</div>&nbsp; <div class="volume">21</div>:<div class="fpage">264</div><div class="lpage">274</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hybridization%20can%20promote%20adaptive%20radiation%20by%20means%20of%20transgressive%20segregation.&amp;author=K.%2C%20Kagawa&amp;author=G.%20Takimoto%2C&amp;publication_year=2017&amp;journal=Ecol.%20Lett.&amp;volume=21&amp;pages=264-274" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/ele.12891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fele.12891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fele.12891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29243294" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hybridization%20can%20promote%20adaptive%20radiation%20by%20means%20of%20transgressive%20segregation.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B18" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kamneva</div>  <div class="given-names">O.K.,</div></div> <div class="name string-name"><div class="surname">Syring</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Liston</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Rosenberg</div>  <div class="given-names">N.A.</div></div></div> <div class="year">2017</div>. <div class="article-title">Evaluating allopolyploid origins in strawberries (fragaria) using haplotypes generated from target capture sequencing.</div> <div class="source">BMC Evol. Biol.</div>&nbsp; <div class="volume">17</div>:<div class="fpage">180</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluating%20allopolyploid%20origins%20in%20strawberries%20%28fragaria%29%20using%20haplotypes%20generated%20from%20target%20capture%20sequencing.&amp;author=O.K.%2C%20Kamneva&amp;author=J.%2C%20Syring&amp;author=A.%2C%20Liston&amp;author=N.A.%20Rosenberg&amp;publication_year=2017&amp;journal=BMC%20Evol.%20Biol.&amp;volume=17&amp;pages=180" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12862-017-1019-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-017-1019-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12862-017-1019-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28778145" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluating%20allopolyploid%20origins%20in%20strawberries%20%28fragaria%29%20using%20haplotypes%20generated%20from%20target%20capture%20sequencing.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B19" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kingman</div>  <div class="given-names">J.F.C.</div></div></div> <div class="year">1982</div>. <div class="article-title">On the genealogy of large populations.</div> <div class="source">J. Appl. Prob.</div>&nbsp; <div class="volume">19</div>:<div class="fpage">27</div><div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=On%20the%20genealogy%20of%20large%20populations.&amp;author=J.F.C.%20Kingman&amp;publication_year=1982&amp;journal=J.%20Appl.%20Prob.&amp;volume=19&amp;pages=27-43" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.2307/3213548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F3213548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F3213548"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:On%20the%20genealogy%20of%20large%20populations.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B20" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.</div></div></div> <div class="year">2009</div>. <div class="article-title">Identifying hybridization events in the presence of coalescence via model selection.</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">58</div>:<div class="fpage">478</div><div class="lpage">488</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identifying%20hybridization%20events%20in%20the%20presence%20of%20coalescence%20via%20model%20selection.&amp;author=L.S.%20Kubatko&amp;publication_year=2009&amp;journal=Syst.%20Biol.&amp;volume=58&amp;pages=478-488" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syp055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyp055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyp055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525602" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identifying%20hybridization%20events%20in%20the%20presence%20of%20coalescence%20via%20model%20selection.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B21" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.,</div></div> <div class="name string-name"><div class="surname">Chifman</div>  <div class="given-names">J.</div></div></div> <div class="year">2015</div>. <div class="article-title">An invariants-based method for hybridization detection from genome-scale sequence data.</div> bioRxiv doi.org/10.1101/034348.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=An%20invariants-based%20method%20for%20hybridization%20detection%20from%20genome-scale%20sequence%20data.&amp;aulast=Kubatko&amp;date=2015" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:An%20invariants-based%20method%20for%20hybridization%20detection%20from%20genome-scale%20sequence%20data.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B22" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lake</div>  <div class="given-names">J.A.</div></div></div> <div class="year">1987</div>. <div class="article-title">A rate-independent technique for analysis of nucleic acid sequences: evolutionary parsimony.</div> <div class="source">Mol. Biol. Evol.</div>&nbsp; <div class="volume">4</div>:<div class="fpage">167</div><div class="lpage">191</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20rate-independent%20technique%20for%20analysis%20of%20nucleic%20acid%20sequences%3A%20evolutionary%20parsimony.&amp;author=J.A.%20Lake&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=167-191" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447007" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20rate-independent%20technique%20for%20analysis%20of%20nucleic%20acid%20sequences%3A%20evolutionary%20parsimony.&amp;aulast=Lake&amp;date=1987&amp;spage=167&amp;epage=191&amp;volume=4" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20rate-independent%20technique%20for%20analysis%20of%20nucleic%20acid%20sequences%3A%20evolutionary%20parsimony.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B23" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maddison</div>  <div class="given-names">W.P.</div></div></div> <div class="year">1997</div>. <div class="article-title">Gene trees in species trees.</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">46</div>:<div class="fpage">523</div><div class="lpage">536</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20trees%20in%20species%20trees.&amp;author=W.P.%20Maddison&amp;publication_year=1997&amp;journal=Syst.%20Biol.&amp;volume=46&amp;pages=523-536" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/46.3.523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F46.3.523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F46.3.523"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20trees%20in%20species%20trees.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B24" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mallet</div>  <div class="given-names">J.</div></div></div> <div class="year">2005</div>. <div class="article-title">Hybridization as an invasion of the genome.</div> <div class="source">Trends Ecol. Evol.</div>&nbsp; <div class="volume">20</div>:<div class="fpage">229</div><div class="lpage">237</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hybridization%20as%20an%20invasion%20of%20the%20genome.&amp;author=J.%20Mallet&amp;publication_year=2005&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=20&amp;pages=229-237" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tree.2005.02.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2005.02.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tree.2005.02.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701374" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hybridization%20as%20an%20invasion%20of%20the%20genome.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B25" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mallet</div>  <div class="given-names">J.</div></div></div> <div class="year">2007</div>. <div class="article-title">Hybrid speciation.</div> <div class="source">Nature</div>&nbsp; <div class="volume">446</div>:<div class="fpage">279</div><div class="lpage">283</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hybrid%20speciation.&amp;author=J.%20Mallet&amp;publication_year=2007&amp;journal=Nature&amp;volume=446&amp;pages=279-283" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nature05706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature05706"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17361174" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hybrid%20speciation.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B26" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div>  <div class="given-names">S.H.,</div></div> <div class="name string-name"><div class="surname">Dasmahapatra</div>  <div class="given-names">K.K.,</div></div> <div class="name string-name"><div class="surname">Nadeau</div>  <div class="given-names">N.J.,</div></div> <div class="name string-name"><div class="surname">Salazar</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Walters</div>  <div class="given-names">J.R.,</div></div> <div class="name string-name"><div class="surname">Simpson</div>  <div class="given-names">F.,</div></div> <div class="name string-name"><div class="surname">Blaxter</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Manica</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Mallet</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Jiggins</div>  <div class="given-names">C.D.</div></div></div> <div class="year">2013</div>. <div class="article-title">Genome-wide evidence for speciation with gene flow in <em>Heliconius</em> butterflies.</div> <div class="source">Genome Res.</div>&nbsp; <div class="volume">23</div>:<div class="fpage">1817</div><div class="lpage">1828</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20evidence%20for%20speciation%20with%20gene%20flow%20in%20Heliconius%20butterflies.&amp;author=S.H.%2C%20Martin&amp;author=K.K.%2C%20Dasmahapatra&amp;author=N.J.%2C%20Nadeau&amp;author=C.%2C%20Salazar&amp;author=J.R.%2C%20Walters&amp;author=F.%2C%20Simpson&amp;author=M.%2C%20Blaxter&amp;author=A.%2C%20Manica&amp;author=J.%2C%20Mallet&amp;author=C.D.%20Jiggins&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=1817-1828" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.159426.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.159426.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.159426.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24045163" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20evidence%20for%20speciation%20with%20gene%20flow%20in%20Heliconius%20butterflies.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B27" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div>  <div class="given-names">S.H.,</div></div> <div class="name string-name"><div class="surname">Davey</div>  <div class="given-names">J.W.,</div></div> <div class="name string-name"><div class="surname">Jiggins</div>  <div class="given-names">C.D.</div></div></div> <div class="year">2014</div>. <div class="article-title">Evaluating the use of ABBA-BABA statistics to locate introgressed loci.</div> <div class="source">Mol. Biol. Evol.</div>&nbsp; <div class="volume">32</div>:<div class="fpage">244</div><div class="lpage">257</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Evaluating%20the%20use%20of%20ABBA-BABA%20statistics%20to%20locate%20introgressed%20loci.&amp;author=S.H.%2C%20Martin&amp;author=J.W.%2C%20Davey&amp;author=C.D.%20Jiggins&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=32&amp;pages=244-257" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msu269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsu269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25246699" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Evaluating%20the%20use%20of%20ABBA-BABA%20statistics%20to%20locate%20introgressed%20loci.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B28" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meng</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.</div></div></div> <div class="year">2009</div>. <div class="article-title">Detecting hybrid speciation in the presence of incomplete lineage sorting using gene tree incongruence: a model.</div> <div class="source">Theor. Popul. Biol.</div>&nbsp; <div class="volume">75</div>:<div class="fpage">35</div><div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Detecting%20hybrid%20speciation%20in%20the%20presence%20of%20incomplete%20lineage%20sorting%20using%20gene%20tree%20incongruence%3A%20a%20model.&amp;author=C.%2C%20Meng&amp;author=L.S.%20Kubatko&amp;publication_year=2009&amp;journal=Theor.%20Popul.%20Biol.&amp;volume=75&amp;pages=35-45" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tpb.2008.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tpb.2008.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tpb.2008.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19038278" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Detecting%20hybrid%20speciation%20in%20the%20presence%20of%20incomplete%20lineage%20sorting%20using%20gene%20tree%20incongruence%3A%20a%20model.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B29" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Otto</div>  <div class="given-names">S.P.,</div></div> <div class="name string-name"><div class="surname">Whitton</div>  <div class="given-names">J.</div></div></div> <div class="year">2000</div>. <div class="article-title">Polyploid incidence and evolution.</div> <div class="source">Ann. Rev. Genet.</div>&nbsp; <div class="volume">34</div>:<div class="fpage">401</div><div class="lpage">437</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Polyploid%20incidence%20and%20evolution.&amp;author=S.P.%2C%20Otto&amp;author=J.%20Whitton&amp;publication_year=2000&amp;journal=Ann.%20Rev.%20Genet.&amp;volume=34&amp;pages=401-437" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev.genet.34.1.401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.34.1.401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev.genet.34.1.401"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Polyploid%20incidence%20and%20evolution.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B30" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Patterson</div>  <div class="given-names">N.,</div></div> <div class="name string-name"><div class="surname">Moorjani</div>  <div class="given-names">P.,</div></div> <div class="name string-name"><div class="surname">Luo</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Swapan</div>  <div class="given-names">M.,</div></div> <div class="name string-name"><div class="surname">Rohland</div>  <div class="given-names">N.,</div></div> <div class="name string-name"><div class="surname">Zhan</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Genschoreck</div>  <div class="given-names">T.,</div></div> <div class="name string-name"><div class="surname">Webster</div>  <div class="given-names">T.,</div></div> <div class="name string-name"><div class="surname">Reich</div>  <div class="given-names">D.</div></div></div> <div class="year">2012</div>. <div class="article-title">Ancient admixture in human history.</div> <div class="source">Genetics</div>&nbsp; <div class="volume">192</div>:<div class="fpage">1065</div><div class="lpage">1093</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ancient%20admixture%20in%20human%20history.&amp;author=N.%2C%20Patterson&amp;author=P.%2C%20Moorjani&amp;author=Y.%2C%20Luo&amp;author=M.%2C%20Swapan&amp;author=N.%2C%20Rohland&amp;author=Y.%2C%20Zhan&amp;author=T.%2C%20Genschoreck&amp;author=T.%2C%20Webster&amp;author=D.%20Reich&amp;publication_year=2012&amp;journal=Genetics&amp;volume=192&amp;pages=1065-1093" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.112.145037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.112.145037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.112.145037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22960212" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ancient%20admixture%20in%20human%20history.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B31" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pease</div>  <div class="given-names">J.B.,</div></div> <div class="name string-name"><div class="surname">Hahn</div>  <div class="given-names">M.W.</div></div></div> <div class="year">2015</div>. <div class="article-title">Detection and polarization of introgression in a five-taxon phylogeny.</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">64</div>:<div class="fpage">651</div><div class="lpage">662</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20polarization%20of%20introgression%20in%20a%20five-taxon%20phylogeny.&amp;author=J.B.%2C%20Pease&amp;author=M.W.%20Hahn&amp;publication_year=2015&amp;journal=Syst.%20Biol.&amp;volume=64&amp;pages=651-662" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syv023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyv023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyv023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25888025" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Detection%20and%20polarization%20of%20introgression%20in%20a%20five-taxon%20phylogeny.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B32" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rambaut</div>  <div class="given-names">A.,</div></div> <div class="name string-name"><div class="surname">Grassly</div>  <div class="given-names">N.C.</div></div></div> <div class="year">1997</div>. <div class="article-title">Seq-Gen: an application for the Monte Carlo simulation of DNA sequence evolution along phylogenetic trees.</div> <div class="source">Comput. Appl. Biosci.</div>&nbsp; <div class="volume">13</div>:<div class="fpage">235</div><div class="lpage">238</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Seq-Gen%3A%20an%20application%20for%20the%20Monte%20Carlo%20simulation%20of%20DNA%20sequence%20evolution%20along%20phylogenetic%20trees.&amp;author=A.%2C%20Rambaut&amp;author=N.C.%20Grassly&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=235-238" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9183526" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Seq-Gen%3A%20an%20application%20for%20the%20Monte%20Carlo%20simulation%20of%20DNA%20sequence%20evolution%20along%20phylogenetic%20trees.&amp;aulast=Rambaut&amp;date=1997&amp;spage=235&amp;epage=238&amp;volume=13" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Seq-Gen%3A%20an%20application%20for%20the%20Monte%20Carlo%20simulation%20of%20DNA%20sequence%20evolution%20along%20phylogenetic%20trees.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B33" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seehausen</div>  <div class="given-names">O.</div></div></div> <div class="year">2004</div>. <div class="article-title">Hybridization and adaptive radiation.</div> <div class="source">Trends Ecol. Evol.</div>&nbsp; <div class="volume">19</div>:<div class="fpage">198</div><div class="lpage">207</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hybridization%20and%20adaptive%20radiation.&amp;author=O.%20Seehausen&amp;publication_year=2004&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=19&amp;pages=198-207" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tree.2004.01.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2004.01.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tree.2004.01.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16701254" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hybridization%20and%20adaptive%20radiation.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B34" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sols-Lumis</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">An</div>  <div class="given-names">C.</div></div></div> <div class="year">2016</div>. <div class="article-title">Inferring phylogenetic networks with maximum pseudolikelihood under incomplete lineage sorting.</div> <div class="source">PLoS Genet.</div>&nbsp; <div class="volume">12</div>:<div class="fpage">e1005896</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Inferring%20phylogenetic%20networks%20with%20maximum%20pseudolikelihood%20under%20incomplete%20lineage%20sorting.&amp;author=C.%2C%20Sol%C3%ADs-Lumis&amp;author=C.%20An%C3%A9&amp;publication_year=2016&amp;journal=PLoS%20Genet.&amp;volume=12&amp;pages=e1005896" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1005896" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005896" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1005896"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26950302" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Inferring%20phylogenetic%20networks%20with%20maximum%20pseudolikelihood%20under%20incomplete%20lineage%20sorting.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B35" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tavar</div>  <div class="given-names">S.</div></div></div> <div class="year">1986</div>. <div class="article-title">Some probabilistic and statistical problems in the analysis of DNA sequences.</div> <div class="source">Lectures on Mathematics in the Life Sciences</div>&nbsp; <div class="volume">17</div>:<div class="fpage">57</div><div class="lpage">86</div>.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Some%20probabilistic%20and%20statistical%20problems%20in%20the%20analysis%20of%20DNA%20sequences.&amp;aulast=Tavar%C3%A9&amp;date=1986&amp;spage=57&amp;epage=86&amp;volume=17" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Some%20probabilistic%20and%20statistical%20problems%20in%20the%20analysis%20of%20DNA%20sequences.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B36" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tian</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Kubatko</div>  <div class="given-names">L.S.</div></div></div> <div class="year">2017</div>. <div class="article-title">Rooting phylogenetic trees under the coalescent model using site pattern probabilities.</div> <div class="source">BMC Evol. Biol.</div>&nbsp; <div class="volume">17</div>:<div class="fpage">263</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Rooting%20phylogenetic%20trees%20under%20the%20coalescent%20model%20using%20site%20pattern%20probabilities.&amp;author=Y.%2C%20Tian&amp;author=L.S.%20Kubatko&amp;publication_year=2017&amp;journal=BMC%20Evol.%20Biol.&amp;volume=17&amp;pages=263" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12862-017-1108-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-017-1108-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12862-017-1108-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/29258427" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Rooting%20phylogenetic%20trees%20under%20the%20coalescent%20model%20using%20site%20pattern%20probabilities.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B37" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Barnett</div>  <div class="given-names">R.M.,</div></div> <div class="name string-name"><div class="surname">Nakhleh</div>  <div class="given-names">L.</div></div></div> <div class="year">2013</div>. <div class="article-title">Parsimonious inference of hybridization in the presence of incomplete lineage sorting.</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">62</div>:<div class="fpage">738</div><div class="lpage">751</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Parsimonious%20inference%20of%20hybridization%20in%20the%20presence%20of%20incomplete%20lineage%20sorting.&amp;author=Y.%2C%20Yu&amp;author=R.M.%2C%20Barnett&amp;author=L.%20Nakhleh&amp;publication_year=2013&amp;journal=Syst.%20Biol.&amp;volume=62&amp;pages=738-751" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syt037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyt037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyt037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23736104" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Parsimonious%20inference%20of%20hybridization%20in%20the%20presence%20of%20incomplete%20lineage%20sorting.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B38" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Degnan</div>  <div class="given-names">J.H.,</div></div> <div class="name string-name"><div class="surname">Nakhleh</div>  <div class="given-names">L.</div></div></div> <div class="year">2012</div>. <div class="article-title">The probability of a gene tree topology within a phylogenetic network with applications to hybridization detection.</div> <div class="source">PLoS Genet.</div>&nbsp; <div class="volume">8</div>:<div class="fpage">e1002660</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20probability%20of%20a%20gene%20tree%20topology%20within%20a%20phylogenetic%20network%20with%20applications%20to%20hybridization%20detection.&amp;author=Y.%2C%20Yu&amp;author=J.H.%2C%20Degnan&amp;author=L.%20Nakhleh&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002660" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pgen.1002660" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002660" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pgen.1002660"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22536161" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20probability%20of%20a%20gene%20tree%20topology%20within%20a%20phylogenetic%20network%20with%20applications%20to%20hybridization%20detection.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B39" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Dong</div>  <div class="given-names">J.,</div></div> <div class="name string-name"><div class="surname">Liu</div>  <div class="given-names">K.J.,</div></div> <div class="name string-name"><div class="surname">Nakhleh</div>  <div class="given-names">L.</div></div></div> <div class="year">2014</div>. <div class="article-title">Maximum likelihood inference of reticulate evolutionary histories.</div> <div class="source">Proc. Natl. Acad. Sci. USA</div>&nbsp; <div class="volume">111</div>:<div class="fpage">16448</div><div class="lpage">16453</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Maximum%20likelihood%20inference%20of%20reticulate%20evolutionary%20histories.&amp;author=Y.%2C%20Yu&amp;author=J.%2C%20Dong&amp;author=K.J.%2C%20Liu&amp;author=L.%20Nakhleh&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=16448-16453" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1407950111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1407950111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1407950111"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Maximum%20likelihood%20inference%20of%20reticulate%20evolutionary%20histories.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="B40" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div>  <div class="given-names">Y.,</div></div> <div class="name string-name"><div class="surname">Than</div>  <div class="given-names">C.,</div></div> <div class="name string-name"><div class="surname">Degnan</div>  <div class="given-names">J.H.,</div></div> <div class="name string-name"><div class="surname">Nakhleh</div>  <div class="given-names">L.</div></div></div> <div class="year">2011</div>. <div class="article-title">Coalescent histories on phylogenetic networks and detection of hybridization despite incomplete lineage sorting.</div> <div class="source">Syst. Biol.</div>&nbsp; <div class="volume">60</div>:<div class="fpage">138</div><div class="lpage">149</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Coalescent%20histories%20on%20phylogenetic%20networks%20and%20detection%20of%20hybridization%20despite%20incomplete%20lineage%20sorting.&amp;author=Y.%2C%20Yu&amp;author=C.%2C%20Than&amp;author=J.H.%2C%20Degnan&amp;author=L.%20Nakhleh&amp;publication_year=2011&amp;journal=Syst.%20Biol.&amp;volume=60&amp;pages=138-149" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syq084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyq084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21248369" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Coalescent%20histories%20on%20phylogenetic%20networks%20and%20detection%20of%20hybridization%20despite%20incomplete%20lineage%20sorting.&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement"> The Author(s) 2018. 
Published by Oxford University Press, on behalf of the Society of 
Systematic Biologists. All rights reserved. For permissions, please 
email: <a href="mailto:journals.permissions@oup.com" target="">journals.permissions@oup.com</a></div><div class="license"><div class="license-p">This article is published and distributed under the terms of the Oxford University Press, Standard Journals Publication Model (<a class="link link-uri" href="https://academic.oup.com/journals/pages/about_us/legal/notices" target="">https://academic.oup.com/journals/pages/about_us/legal/notices</a>)</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=4944070&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5349">Download all figures</a>
                </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,324</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000443580600006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/sysbio/syy023" data-uuid="0ec285a2-c9af-6668-c97b-0acf98efb05a"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=34708223" style="display:inline-block;">
        <img alt="Article has an altmetric score of 11" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4944070"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal"></a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-SeeAlso widget-instance-Article_RightRailB0Article_RightRail_SeeAlso">
            <div class="vt-see-also rail-widget_wrap">
        <h3 class="seeAlso-title">See also</h3>
                <ul class="seeAlso-articleLinks">
                <li>
                        <h4 class="seeAlso-group-header">Companion Articles</h4>
                    <ul>
                        <li>
                            <a href="https://academic.oup.com/sysbio/article/67/5/800/4985806">Phylogenetic Comparative Methods on Phylogenetic Networks with Reticulations</a>
                        </li>
                    </ul>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000443580600006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fsysbio%2farticle%2f67%2f5%2f821%2f4944070">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000443580600006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (8)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F67%2F5%2F821%2F4944070" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/sysbio/crossref-citedby/4944070">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz078/5634158?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                PHYLOGENETIC CONFLICTS, COMBINABILITY, AND DEEP PHYLOGENOMICS IN PLANTS
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz076/5626195?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Species Selection Regime and Phylogenetic Tree Shape
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz075/5626194?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                A phenotype-genotype codon model for detecting adaptive evolution
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz071/5613902?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Impossibility Results on Stability of Phylogenetic Consensus Methods
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz073/5613901?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Reticulate evolution helps explain 
apparent homoplasy in floral biology and pollination in baobabs ( 
Adansonia ; Bombacoideae; Malvaceae)
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Behind_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="HyDe: A Python Package for Genome-Scale Hybridization Detection | Systematic Biology | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/sysbio/syy023");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/sysbio_Behind_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/sysbio/pages/About">About Systematic Biology</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/sysbio/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/sysbio/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.systbio.org/">Contact Us</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://twitter.com/systbiol">Twitter</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/sysbio/subscribe">Purchase</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="" target="">
                        <img id="footer-logo-SystematicBiology" class="journal-footer-affiliations-logo" src="sysbio_f1-178201843.svg" alt="Systematic Biology">
                    </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1076-836X</li>
                                            <li>Print ISSN 1063-5157</li>
                    <li>Copyright  2019 Society of Systematic Biologists</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright  2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh766" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1574683639548&amp;tmr=load%3D1574683639498%26core%3D1574683639534%26main%3D1574683639540%26ifr%3D1574683639554&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F67%2F5%2F821%2F4944070&amp;href=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F67%2F5%2F821%2F4944070&amp;dt=HyDe%3A%20A%20Python%20Package%20for%20Genome-Scale%20Hybridization%20Detection&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5ddbc3f7f7ac0ed8&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3DHyDe%253A%2520A%2520Python%2520Package%2520for%2520Genome-Scale%2520Hybridization%2520Detection%26description%3DAbstract.%2520%2520The%2520analysis%2520of%2520hybridization%2520and%2520gene%2520flow%2520among%2520closely%2520related%2520taxa%2520is%2520a%2520common%2520goal%2520for%2520researchers%2520studying%2520speciation%2520and%2520phylogeography.%2520Many%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fsysbio%252Farticle%252F67%252F5%252F821%252F4944070%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fsysbio%252FIssue%252F67%252F5%252F3%252Fm_sysbio_67_5cover.png%253FExpires%253D1637529744%2526Signature%253DLY1AEsR30WVdAO0HTeXklECp8-0DhPQIFHZ3V2S5jDE-uSD38h4EsVKg8CsX9ga7e-RdDgzgG1ahw3jvXmS4mPIkxIpXJ3mjEDD6epnNAG1GNMd4OIOyAWSfJPfRzSViTCjihKmBWWWHoeNxrM4foB0u-1Inyw7et6ZP5N6Oxzd9Xb%7Ec%7EkAjk3sLmsL0aqvvICo2hpUGZh0Jp28HxHQSxbiz5g2xG3PXVStKWRcTkTatZ64FDDn8-SJ3Hr9aTeY5fo-OnaEUBxKTKbX-QSf8Q7I78cGi7SQ6bPQh7y-NnEI8Vv1PYM6pPqbv7P3Cw8SEXfVl%7E0CiDGF83gCc-7MRag__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2018-03-19&amp;csi=undefined&amp;rev=v8.28.1-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f67%2f5%2f821%2f4944070">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>

    <script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>



<script type="text/javascript" src="syy023.js"></script><div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div></body></html>