_altmetric.embed_callback({"title":"HyDe: A Python Package for Genome-Scale Hybridization Detection","doi":"10.1093/sysbio/syy023","pmid":"29562307","altmetric_jid":"4f6fa5153cf058f610003603","issns":["1063-5157","1076-836X"],"journal":"Systematic Biology","cohorts":{"pub":9,"sci":10},"abstract":"The analysis of hybridization and gene flow among closely related taxa is a common goal for researchers studying speciation and phylogeography. Many methods for hybridization detection use simple site pattern frequencies from observed genomic data and compare them to null models that predict an absence of gene flow. The theory underlying the detection of hybridization using these site pattern probabilities exploits the relationship between the coalescent process for gene trees within population trees and the process of mutation along the branches of the gene trees. For certain models, site patterns are predicted to occur in equal frequency (i.e., their difference is 0), producing a set of functions called phylogenetic invariants. In this paper we introduce HyDe, a software package for detecting hybridization using phylogenetic invariants arising under the coalescent model with hybridization. HyDe is written in Python, and can be used interactively or through the command line using pre-packaged scripts. We demonstrate the use of HyDe on simulated data, as well as on two empirical data sets from the literature. We focus in particular on identifying individual hybrids within population samples and on distinguishing between hybrid speciation and gene flow. HyDe is freely available as an open source Python package under the GNU GPL v3 on both GitHub (https://github.com/pblischak/HyDe) and the Python Package Index (PyPI: https://pypi.python.org/pypi/phyde).","context":{"all":{"count":13791430,"mean":8.2282410952401,"rank":1531764,"pct":88,"higher_than":12248604},"journal":{"count":1189,"mean":10.461589225589,"rank":294,"pct":75,"higher_than":895},"similar_age_3m":{"count":274750,"mean":14.533790230355,"rank":50897,"pct":81,"higher_than":223563},"similar_age_journal_3m":{"count":27,"mean":16.480615384615,"rank":19,"pct":29,"higher_than":8}},"authors":["Paul D Blischak","Julia Chifman","Andrea D Wolfe","Laura S Kubatko"],"type":"article","altmetric_id":34708223,"schema":"1.5.4","is_oa":false,"publisher_subjects":[{"name":"Evolutionary Biology","scheme":"era"},{"name":"Genetics","scheme":"era"}],"cited_by_posts_count":20,"cited_by_tweeters_count":19,"cited_by_accounts_count":19,"last_updated":1559658901,"score":10.25,"history":{"1y":3.1,"6m":2.85,"3m":0,"1m":0,"1w":0,"6d":0,"5d":0,"4d":0,"3d":0,"2d":0,"1d":0,"at":10.25},"url":"http://dx.doi.org/10.1093/sysbio/syy023","added_on":1521714442,"published_on":1521417600,"subjects":["biology"],"scopus_subjects":["Biochemistry, Genetics and Molecular Biology","Agricultural and Biological Sciences","Life Sciences"],"readers":{"citeulike":"0","mendeley":"73","connotea":"0"},"readers_count":73,"images":{"small":"https://badges.altmetric.com/?size=64&score=11&types=tttttttt","medium":"https://badges.altmetric.com/?size=100&score=11&types=tttttttt","large":"https://badges.altmetric.com/?size=180&score=11&types=tttttttt"},"details_url":"http://www.altmetric.com/details.php?citation_id=34708223"})