<!DOCTYPE html>
<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><iframe src="javascript:false" title="" style="width: 0px; height: 0px; border: 0px none; display: none;"></iframe><script type="text/javascript" async="" src="geo2.js"></script>
      <meta name="citation_pii" content="S1055790313000808">
<meta name="citation_issn" content="1055-7903">
<meta name="citation_volume" content="67">
<meta name="citation_lastpage" content="607">
<meta name="citation_issue" content="3">
<meta name="citation_publisher" content="Academic Press">
<meta name="citation_firstpage" content="600">
<meta name="citation_journal_title" content="Molecular Phylogenetics and Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.ympev.2013.02.023">
<meta name="dc.identifier" content="10.1016/j.ympev.2013.02.023">
<meta name="citation_article_type" content="Full-length article">
<meta property="og:image" content="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-fx1.jpg">
<meta property="og:description" content="New Guinea is one of five high biodiversity wilderness areas, and frog diversity is exceptionally large, with more than 400 species described to date.…">
<meta name="citation_title" content="Phylogeny and phylogeography of Mantophryne (Anura: Microhylidae) reveals cryptic diversity in New Guinea">
<meta property="og:title" content="Phylogeny and phylogeography of Mantophryne (Anura: Microhylidae) reveals cryptic diversity in New Guinea">
<meta name="citation_publication_date" content="2013/06/01">
<meta name="citation_online_date" content="2013/03/14">
<meta name="citation_pdf_url" content="https://www.sciencedirect.com/science/article/pii/S1055790313000808/pdfft?md5=337c3f2b299046c8ae64cfe97b0b9f35&amp;pid=1-s2.0-S1055790313000808-main.pdf">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR">
      <title>Phylogeny and phylogeography of Mantophryne (Anura: Microhylidae) reveals cryptic diversity in New Guinea - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S1055790313000808">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel="shortcut icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="arp.css">
      <link rel="dns-prefetch" href="https://w.usabilla.com/">
      <link rel="dns-prefetch" href="https://www.deepdyve.com/">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <script>
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Molecular Phylogenetics and Evolution"};
        window.pageData = {"content":[{"entitlementType":"package","format":"MIME-XHTML","id":"sd:article:pii:S1055790313000808","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"1055-7903","volumeNumber":"67","issueNumber":"3"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1531921034255,"loadTime":""},"visitor":{"accessType":"ae:ANON_IP","accountId":"ae:262591","accountName":"ae:International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"ae:12289512","ipAddress":"197.248.44.39","appSessionId":"fdf01f62447ee3481f3b4045afa46b236f2bgxrqa"}};
        window.arp = {
          config: {"reduxLogging":false,"assetsBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/d459717ce4bedcd05c18402c0b015aff6be02806","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","googleMapsApiKey":"AIzaSyAV0ZB1XWtRlzJ06pJAW74Oan87lQVgBHY","bosUrl":"https://feedback.recs.d.elsevier.com/raw/events","bosTimeOut":60000,"strictMode":false},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
      </script>
      <script type="text/javascript" src="204774041.js"></script>
      <!-- begin mPulse embed code -->
      <script>
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
    <script src="satellite-5b3b560664746d57b70018c6.js"></script><script src="s-code-contents-9c0358adbc3b5986e210099b3bf1d427fc5bd286.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_233.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link id="plx-css-summary" type="text/css" rel="stylesheet" href="https://d39af2mgp1pqhg.cloudfront.net/summary.css"><script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script><script type="text/javascript" src="//d39af2mgp1pqhg.cloudfront.net/extjs/xss.js"></script></head>
    <body><div style="display: none;" id="lightningjs-usabilla_live"><div><iframe id="lightningjs-frame-usabilla_live" frameborder="0"></iframe></div></div>
      <a class="sr-only sr-only-focusable" href="#app">Skip to main content</a>
      <!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://service.elsevier.com/app/answers/detail/a_id/9831">this support page</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
      <div data-iso-key="_0"><div class="App" id="app" role="application" data-reactroot=""><div class="page"><section><div id="header"><div class="u-bg-white" style="overflow:visible" role="banner"><div class="els-header" style="min-height:80px"><a id="els-main-title-link" href="https://www.sciencedirect.com/"><svg id="els-header-wordmark" class="u-fill-orange u-margin-l-top u-margin-s-left u-margin-l-left-from-sm" viewBox="-3334 3439.4 163 26" style="width:163px;height:24px"><title>ScienceDirect</title><g transform="scale(.8125,.8125)"><path d="M-4099.8,4240.4c0-1.8,1-3.6,4.4-3.6c1.7,0,3.7,0.5,5.5,1.6l0.2-3.1c-1.7-0.7-3.5-1.1-5.6-1.1 c-5.5,0-7.8,2.9-7.8,6.4c0,6.6,10.4,7.2,10.4,11.7c0,1.8-1.4,3.6-4.6,3.6c-2,0-4-0.7-5.6-1.6l-0.4,3.1c1.8,0.9,4.2,1.2,6.1,1.2 c5,0,7.9-2.9,7.9-6.5C-4089.4,4245.8-4099.8,4244.9-4099.8,4240.4"></path><path id="c" d="M-4080.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4083.1,4244.3-4080.6,4243.1-4080.3,4242.9"></path><path id="i" d="M-4068,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-4066,4234.1-4067,4233.1-4068,4233.1 M-4069.5,4258.1h3v-17.7h-3V4258.1z"></path><path id="e" d="M-4057.1,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4058.7,4244.4-4057.4,4243.3-4057.1,4243.1z M-4047.5,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4 c0,5.8,3.5,9.2,7.9,9.2c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-4047.5,4248.9"></path><path d="M-4034.8,4240c-2.6,0-4.3,1.3-5.7,3.1l-0.5-2.6h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.5v12.8h3v-11.9 c0.8-1.1,2.5-2.9,2.9-3.1c0.3-0.2,1.5-0.5,2.5-0.5c2.7,0,2.9,1.4,3,4c0,1.4,0,3.7,0,3.7c0,3.5-0.1,7.6-0.1,7.6h3 c0,0,0.1-5.3,0.1-8.2c0-1.8,0-3.5-0.1-5.3C-4029.5,4241-4031.6,4240-4034.8,4240"></path><path d="M-3982.5,4255.6h-4.4v-18.6h4.8c6.4,0,8.2,5.2,8.2,9.2C-3973.8,4252.2-3976.5,4255.6-3982.5,4255.6z M-3981.6,4234.6h-8.4v23.5h8.1c8.6,0,11.5-6.7,11.5-11.9C-3970.4,4240.9-3973.2,4234.6-3981.6,4234.6"></path><path d="M-3950.5,4240c-1.9,0-3.4,1.7-4,3.2l-0.5-2.8h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.4v12.8h3v-11.2 c0.6-1.5,2-4.4,3.7-4.4c1.1,0,1.2,1.2,1.2,1.4l2.5-0.7v-0.2c0,0,0-0.2-0.1-0.5C-3947.4,4240.9-3948.5,4240-3950.5,4240"></path><path d="M-3903.5,4255.2c-1.1,0.4-2,0.8-3,0.8c-1.4,0-1.9-0.8-1.9-2.8v-10.4h4.6v-2.3h-4.6v-4.7h-2.9v4.7h-3.2v2.3h3.2 v11.4c0,3.1,1.6,4.4,3.9,4.4c1.4,0,3-0.5,4.1-0.9L-3903.5,4255.2"></path><g><path d="M-3923.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-3926.1,4244.3-3923.6,4243.1-3923.3,4242.9"></path></g><g><path d="M-3941.6,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-3943.2,4244.4-3941.8,4243.3-3941.6,4243.1z M-3932,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3932,4248.9"></path></g><g><path d="M-3964.5,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-3962.5,4234.1-3963.4,4233.1-3964.5,4233.1 M-3965.9,4258.1h3v-17.7h-3V4258.1z"></path></g><g><path d="M-4004.4,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4006,4244.4-4004.6,4243.3-4004.4,4243.1z M-3994.7,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3994.7,4248.9"></path></g><g><path d="M-4019.1,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4021.8,4244.3-4019.4,4243.1-4019.1,4242.9"></path></g></g></svg></a><div class="move-right u-display-inline-block"><nav class="u-clr-grey8 u-show-from-md"><div class="u-display-inline-block" style="margin-top:29px"><span><span class="u-margin-l-right"><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/science/journals" id="els-header-navigation-section-journal" style="border-bottom:"><span class="anchor-text">Journals</span></a><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/science/bookbshsrw" id="els-header-navigation-section-book" style="border-bottom:"><span class="anchor-text">Books</span></a></span><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790313000808" id="els-header-navigation-section-register"><span class="anchor-text">Register</span></a></span><a class="anchor qa-no-js-fallback-link anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790313000808" id="els-header-user-sign-in"><span class="anchor-text">Sign in</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a><li style="list-style:none;display:inline"><a class="anchor u-margin-l-hor help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="help" title="Help" target="_blank"><span class="anchor-text"><svg focusable="false" viewBox="0 0 114 128" style="margin-top:-10px" width="21.375" height="24" class="icon icon-help"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></span></a></li></div></nav></div><div class="u-hide-from-md move-right"><ul><li style="list-style:none;display:inline-block;margin-top:32px" class="u-margin-s-hor u-margin-l-hor-from-sm"><div class="hamburger-button"><button class="button-link button-link-primary" aria-label="Mobile menu" aria-expanded="false" type="button"><svg viewBox="0 0 40 18" width="40" height="18" y="52"><path d="M0 16h40v2H0zm0-8h40v2H0zm0-8h40v2H0z"></path></svg><span class="button-link-text"></span></button></div></li></ul><div style="bottom:0;left:0;position:fixed;top:0;width:100%;z-index:70;opacity:.8" class="u-bg-grey1 u-display-none"></div><div id="mobile-menu" style="overflow:auto;position:fixed;width:288px;right:0;top:0;z-index:1000;height:100%" aria-label="Mobile menu" class="u-bg-grey7 u-clr-grey1 u-display-none" role="navigation"><div class="u-bg-black panel-s"></div><div class="u-bg-grey8 panel-s"><ul><li style="list-style:none"><a class="anchor journals-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/science/journals" style="border-bottom:" id="mobile-journals-link"><span class="anchor-text">Journals</span></a></li><li style="list-style:none"><a class="anchor books-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/science/bookbshsrw" style="border-bottom:" id="mobile-books-link"><span class="anchor-text">Books</span></a></li><li style="list-style:none"><a class="anchor u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790313000808" id="mobile-register-link"><span class="anchor-text">Register</span></a></li></ul></div><div class="panel-s u-bg-grey7"><ul class="text-xs"></ul><ul class="u-margin-s-top text-s"><li style="list-style:none"><a class="anchor anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790313000808" id="mobile-sign-in-out-link" rel="nofollow"><span class="anchor-text">Sign In</span></a></li><li style="list-style:none;display:inline"><a class="anchor u-padding-xs-top text-s help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="mobile-help" title="Help" target="_blank"><span class="anchor-text">Help</span></a></li></ul></div></div><div class="move-bottom move-center"></div></div></div></div></div><div class="Article" id="mathjax-container"><div class="sticky-outer-wrapper"><div class="sticky-inner-wrapper" style="position: relative; z-index: 1; transform: translate3d(0px, 0px, 0px);"><div class="Toolbar" role="region" aria-label="download options and search"><div class="toolbar-container"><div class="u-show-from-lg col-lg-6">&nbsp;</div><div class="buttons pull-left pad-left"><button class="show-toc-button u-hide-from-lg"><svg focusable="false" viewBox="0 0 104 128" width="19.5" height="24" class="icon icon-list"><path d="m2e1 95a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm14 55h68v1e1h-68zm0-3e1h68v1e1h-68zm0-3e1h68v1e1h-68z"></path></svg><span class="label">Outline</span></button><div class="PdfDownloadButton"><button class="button-link button-link-secondary" id="pdfLink" aria-controls="Download PDF options" type="button"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="button-link-text"><span class="pdf-download-label u-show-inline-from-lg">Download PDF</span><span class="pdf-download-label-short u-hide-from-lg">Download</span></span></button></div><div class="ExportCitation" role="region" id="export-citation"><button class="button ExportCitationButton button-anchor" type="button"><span class="button-text">Export</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div><div class="pull-right pad-right u-show-from-md"><form id="quick-search" class="QuickSearch" action="/search/advanced" method="get"><input class="query" aria-label="Search ScienceDirect" name="qs" placeholder="Search ScienceDirect" type="text"><button class="button button-primary" type="submit" aria-label="Submit search"><span class="button-text"><svg focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><a class="advanced-search-link" href="https://www.sciencedirect.com/search/advanced">Advanced</a><input name="origin" value="article" type="hidden"><input name="zone" value="qSearch" type="hidden"></form></div></div></div></div></div><div class="article-wrapper grid row"><div class="u-show-from-lg col-lg-6"><div class="TableOfContents u-margin-l-bottom" lang="en"><div class="Outline" id="toc-outline"><h2>Outline</h2><ol class="u-padding-s-bottom"><li><a href="#aep-abstract-id14" title="Abstract">Abstract</a></li><li></li><li><a href="#aep-keywords-id16" title="Keywords">Keywords</a></li><li><a href="#s0005" title="1. Introduction">1. Introduction</a></li><li><a href="#s0010" title="2. Materials and methods">2. Materials and methods</a></li><li><a href="#s0035" title="3. Results">3. Results</a></li><li><a href="#s0050" title="4. Discussion">4. Discussion</a></li><li><a href="#aep-acknowledgment-id17" title="Acknowledgments">Acknowledgments</a></li><li><a href="#s0065" title="Appendix A. Supplementary material">Appendix A. Supplementary material</a></li><li><a href="#bi0005" title="References">References</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show full outline</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2>Figures (5)</h2><ol><li><a href="#f0030"><div><img alt="Unlabelled figure" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-fx1.sml" style="max-width: 219px; max-height: 154px;"></div></a></li><li><a href="#f0005"><div><img alt="Fig. 1. Distribution map of all putative Mantophryne putative species" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr1.sml" style="max-width: 158px; max-height: 163px;"></div></a></li><li><a href="#f0010"><div><img alt="Fig. 2. Biogeographic reconstruction of Mantophryne" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr2.sml" style="max-width: 150px; max-height: 163px;"></div></a></li><li><a href="#f0015"><div><img alt="Fig. 3. Concatenated phylogeny" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr3.sml" style="max-width: 130px; max-height: 164px;"></div></a></li><li><a href="#f0020"><div><img alt="Fig. 4. Species tree from *BEAST" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr4.sml" style="max-width: 219px; max-height: 154px;"></div></a></li></ol><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2>Tables (3)</h2><ol class="u-padding-s-bottom"><li><a href="#t0005" title="List of loci, PCR primers, and annealing temperatures."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 1</a></li><li><a href="#t0010" title="List of the length, number of variable sites, and partitioning scheme for each locus."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 2</a></li><li><a href="#t0015" title="Biogeographic reconstruction probabilities. 0-Northern New Guinea (accreted portions of the Outer Melanesian Island Arc), 1-Papuan Peninsula (East Papuan Composite Terrane), 2-D’Entrecasteaux Islands,..."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 3</a></li></ol><div class="PageDivider"></div></div><div class="Extras" id="toc-extras"><h2>Extras (1)</h2><ol class="u-padding-s-bottom"><li><a href="#m0005" title="Supplementary data 1"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-interactive-case-insights"><path d="m2 1e1v1.1e2h94v-1.1e2h-94zm84 1e2h-74v-9e1h74v9e1zm-64-28h14v16h-14zm2e1 -18h14v34h-14zm2e1 8h14v26h-14zm-6.04-16.3l22.04-11.86v-11.36l-21.96 11.82-19.06-10.58-16.98 18.52v14.8l19.02-20.76z"></path></svg>Supplementary data 1</a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right" role="main" lang="en"><noscript>JavaScript is disabled on your browser. Please enable JavaScript to use all the features on this page.</noscript><div class="Publication" id="publication"><div class="publication-brand u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903"><img class="publication-brand-image" src="elsevier-non-solus.png" alt="Elsevier"></a></div><div class="publication-volume u-text-center"><h2 class="publication-title" id="publication-title"><a class="publication-title-link" title="Go to Molecular Phylogenetics and Evolution on ScienceDirect" href="https://www.sciencedirect.com/science/journal/10557903">Molecular Phylogenetics and Evolution</a></h2><div class="text-xs"><a title="Go to table of contents for this volume/issue" href="https://www.sciencedirect.com/science/journal/10557903/67/3">Volume 67, Issue 3</a>, <!-- -->June 2013<!-- -->, Pages 600-607</div></div><div class="publication-cover u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903/67/3"><img class="publication-cover-image" src="1-s2.gif" alt="Molecular Phylogenetics and Evolution"></a></div></div><h1 class="Head"><span class="title-text">Phylogeny and phylogeography of <em>Mantophryne</em> (Anura: Microhylidae) reveals cryptic diversity in New Guinea</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><a class="author size-m workspace-trigger" name="baep-author-id5" href="#%21"><span class="content"><span class="text given-name">Lauren A.</span><span class="text surname">Oliver</span><span class="author-ref" id="baff1"><sup>a</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id6" href="#%21"><span class="content"><span class="text given-name">Eric N.</span><span class="text surname">Rittmeyer</span><span class="author-ref" id="baff1"><sup>a</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id7" href="#%21"><span class="content"><span class="text given-name">Fred</span><span class="text surname">Kraus</span><span class="author-ref" id="baff2"><sup>b</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id8" href="#%21"><span class="content"><span class="text given-name">Stephen J.</span><span class="text surname">Richards</span><span class="author-ref" id="baff3"><sup>c</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id9" href="#%21"><span class="content"><span class="text given-name">Christopher C.</span><span class="text surname">Austin</span><span class="author-ref" id="baff1"><sup>a</sup></span><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a></div></div></div><button class="show-hide-details"><svg viewBox="0 0 9 9" class="icon-expand"><path d="M5 7H4V5H2V4h2V2h1v2h2v1H5z"></path><path d="M0 0v9h9V0zm1 1h7v7H1z"></path></svg>Show more</button></div><div class="DoiLink" id="doi-link"><a class="doi" href="https://doi.org/10.1016/j.ympev.2013.02.023" target="_blank" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier">https://doi.org/10.1016/j.ympev.2013.02.023</a><a class="rights-and-content" target="_blank" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1055790313000808&amp;orderBeanReset=true">Get rights and content</a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts" id="abstracts"><div class="abstract author" id="aep-abstract-id14" lang="en"><h2 class="section-title">Abstract</h2><div id="aep-abstract-sec-id15"><p id="sp0015">New
 Guinea is one of five high biodiversity wilderness areas, and frog 
diversity is exceptionally large, with more than 400 species described 
to date. The microhylid frog genus <em>Mantophryne</em> is endemic to New Guinea and consists of four species, three of which have narrow geographic distributions and a fourth, <em>M. lateralis</em>, with a broad range that spans the eastern half of the island. Here, we sequence 104 <em>Mantophryne</em>
 samples for three mitochondrial and three nuclear loci to reconstruct 
the first phylogeny of the genus and to examine spatial patterns of 
diversity within <em>M. lateralis</em>. Results indicate that the wide-ranging <em>M. lateralis</em>
 is composed of at least nine geographically separated and 
well-supported lineages that represent putative species. Biogeographic 
analysis suggests that <em>Mantophryne</em> evolved on the eastern 
Papuan peninsula with subsequent dispersal westward, as well as 
overwater dispersal events to the Louisiade and D’Entrecasteaux 
archipelagos.</p></div></div><div class="abstract graphical" id="aep-abstract-id10" lang="en"><div id="aep-abstract-sec-id11"><h3>Graphical abstract</h3><p id="sp0055"><span class="display"><figure class="figure" id="f0030"><span id="download-image"><img src="1-s2.jpg" alt="" height="200"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-fx1.jpg" target="__blank">Download full-size image</a></li></ol></span></figure></span></p></div><div id="aep-abstract-sec-id13"><h3>Highlights</h3><p id="sp0060">►
 A widespread frog species in New Guinea is shown to be a complex of 9 
species. ► Biogeographic reconstructions indicate two overwater 
dispersal events. ► New Guinea could be a focal center for frog 
biodiversity. ► Evidence for other cryptic frog lineages on the island.</p></div></div></div><ul class="issue-navigation" id="issue-navigation"><li class="previous"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790313000778"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-left"><path d="m1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text"><span><strong>Previous </strong>article<span class="extra-detail"> in issue</span></span></span></a></li><li class="next"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790313000912"><span class="button-alternative-text"><strong>Next </strong>article<span class="extra-detail"> in issue</span></span><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords"><div id="aep-keywords-id16" class="keywords-section"><h2 class="section-title">Keywords</h2><div class="keyword"><span>Amphibian</span></div><div class="keyword"><span>Anura</span></div><div class="keyword"><span>Asterophryinae</span></div><div class="keyword"><span>Biodiversity</span></div><div class="keyword"><span>Microhylidae</span></div><div class="keyword"><span>Species tree</span></div></div></div><div class="Body" id="body"><div><section id="s0005"><h2>1. Introduction</h2><p id="p0005">New Guinea is the world’s largest and highest tropical island and is one of the most biodiverse regions on the planet (<a name="bb0045" href="#b0045" class="workspace-trigger">Beehler, 1993</a>, <a name="bb0080" href="#b0080" class="workspace-trigger">Dinerstein and Wikramanayake, 1993</a>, <a name="bb0160" href="#b0160" class="workspace-trigger">Myers et al., 2000</a>).
 The complex geological history of New Guinea has played a major role in
 the high level of biodiversity, especially in reptiles and amphibians (<a name="bb0105" href="#b0105" class="workspace-trigger">Heads, 2002</a>).
 Relatively recent tectonic activity, including island accretion, 
montane uplift, and volcanism, have created a complex landscape with 
extreme habitat heterogeneity and topographic relief along a steep 
elevational gradient from sea-level to over 5000&nbsp;m. Accretion of 
the Inner and Outer Melanesian Island Arcs onto the Australian plate 
formed the central mountain range approximately 5–10&nbsp;million years ago (<a name="bb0180" href="#b0180" class="workspace-trigger">Pigram and Davies, 1987</a>, <a name="bb0005" href="#b0005" class="workspace-trigger">Abbott et al., 1994</a>, <a name="bb0015" href="#b0015" class="workspace-trigger">Allison, 1996</a>, <a name="bb0100" href="#b0100" class="workspace-trigger">Hall, 1997</a>, <a name="bb0245" href="#b0245" class="workspace-trigger">Tregoning et al., 1999</a>, <a name="bb0105" href="#b0105" class="workspace-trigger">Heads, 2002</a>).</p><p id="p0010">The
 frog diversity on New Guinea represents approximately 8% of global 
diversity, despite accounting for only about 0.6% of global land area. 
However, frog biodiversity is extremely underestimated; species 
accumulation curves demonstrate that the true number on New Guinea is 
likely double the current 410 described species (<a name="bb0015" href="#b0015" class="workspace-trigger">Allison, 1996</a>, <a name="bb0025" href="#b0025" class="workspace-trigger">Austin et al., 2008</a>, <a name="bb0020" href="#b0020" class="workspace-trigger">Allison et al., 2010</a>).
 If this prediction is correct, it would mean that New Guinea would 
represent an astonishing biogeographic focal center of frog diversity. 
The family Microhylidae is the most diverse group of frogs in New Guinea
 with over 218 described species representing 53% of New Guinean 
amphibian diversity, almost all species being endemic (<a name="bb0020" href="#b0020" class="workspace-trigger">Allison et al., 2010</a>).</p><div><p id="p0015">Based
 predominantly on call structure and subtle differences in morphology, 
Günther recently described six new species from the previously monotypic
 microhylid <em>Hylophorbus</em>, showing that the widespread “<em>H. rufescens</em>” is actually composed of multiple geographically separated species (<a name="bb0095" href="#b0095" class="workspace-trigger">Günther, 2001</a>). However, only a handful of populations in the westernmost portion of the range were examined and <em>Hylophorbus</em> also extends into the eastern part of New Guinea. Another microhylid frog with a similarly broad range is <em>Mantophryne lateralis</em>, which occurs throughout the lowlands of eastern New Guinea (<a name="bb0055" href="#b0055" class="workspace-trigger">Boulenger, 1897</a>, <a name="bb0275" href="#b0275" class="workspace-trigger">Zweifel, 1972</a>, <a name="bb0060" href="#b0060" class="workspace-trigger">Burton, 1986</a>, <a name="bb0155" href="#b0155" class="workspace-trigger">Menzies, 2006</a>). This broad range is concordant with <em>Hylophorbus</em> and with many other amphibian species and makes <em>M. lateralis</em> a good candidate for examining patterns of genetic diversity in eastern New Guinea. <em>Mantophryne</em> currently includes only three other species, all with restricted geographic ranges (<a name="bf0005" href="#f0005" class="workspace-trigger">Fig. 1</a>): <em>M. axanthogaster</em> (Sudest Island) (<a name="bb0130" href="#b0130" class="workspace-trigger">Kraus and Allison, 2009</a>); <em>M. infulata</em> (Arau and the Huon Peninsula) (<a name="bb0275" href="#b0275" class="workspace-trigger">Zweifel, 1972</a>, <a name="bb0060" href="#b0060" class="workspace-trigger">Burton, 1986</a>); and <em>M. louisiadensis</em> (Rossel Island) (<a name="bb0175" href="#b0175" class="workspace-trigger">Parker, 1934</a>, <a name="bb0275" href="#b0275" class="workspace-trigger">Zweifel, 1972</a>, <a name="bb0060" href="#b0060" class="workspace-trigger">Burton, 1986</a>).</p><figure class="figure" id="f0005"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr1.jpg" alt="" height="552"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr1.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p id="sp0020"><span class="label">Fig. 1</span>. Distribution map of all putative <em>Mantophryne</em> putative species. Symbols correspond to those in <a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>. Two species occur in Amau, indicated by an open circle and a closed star.</p></span></div></figure></div><p id="p0020">Here we use data from multiple loci to reconstruct the phylogeny of the genus <em>Mantophryne</em>
 in order to examine spatial and topological patterns of diversity. In 
addition, we conduct a biogeographic analysis using ancestral 
reconstructions to decipher areas of origin and patterns of dispersal. 
Our analyses, while confined to a single genus, may provide a template 
for examining cryptic diversity in other frogs in New Guinea.</p></section><section id="s0010"><h2>2. Materials and methods</h2><section id="s0015"><h3>2.1. Sampling</h3><p id="p0025">A total of 104 ingroup samples were examined, representing all four described species of <em>Mantophryne</em>: 94 samples spanning the distribution of <em>M. lateralis</em>, four <em>M. louisiadensis</em>, five <em>M. axanthogaster</em>, and one <em>M. infulata</em> (<a name="bf0005" href="#f0005" class="workspace-trigger">Fig. 1</a>). To test the monophyly of <em>Mantophryne</em>, we also included 24 samples representing four described and multiple undescribed species of <em>Hylophorbus</em>, one sample of the monotypic genus <em>Pherohapsis</em> (<em>P. menziesi</em>) and eight samples of a putative new species of <em>Mantophryne</em> from Amau Village, Central Province, Papua New Guinea. Based on higher-level asterophryine phylogenetics (<a name="bb0125" href="#b0125" class="workspace-trigger">Köhler and Günther, 2008</a>), we also included outgroup samples of the genera <em>Austrochaperina, Callulops</em>, <em>Choerophryne</em>, and <em>Sphenophryne</em>. Voucher numbers and collection localities are provided in <a name="bs0065" href="#s0065" class="workspace-trigger">Supplementary Appendix 1</a>.</p></section><section id="s0020"><h3>2.2. DNA isolation, amplification, and sequencing</h3><div><p id="p0030">Whole
 genomic DNA was extracted from liver tissue using the Qiagen DNeasy 
Blood and Tissue Kit (Valencia, California, USA). Three mitochondrial 
gene regions (<em>12S</em>, <em>16S</em>, and <em>cytochrome b</em>) and three nuclear loci (<em>Tyrosinase, c-myc</em> exon 2, and <em>c-myc</em>
 exon 3) were selected based on their utility in previous studies that 
included microhylid frogs and sequenced using previously published 
primers (<a name="bt0005" href="#t0005" class="workspace-trigger">Table 1</a>) (<a name="bb0210" href="#b0210" class="workspace-trigger">Richards and Moore, 1996</a>, <a name="bb0050" href="#b0050" class="workspace-trigger">Bossuyt and Milinkovitch, 2000</a>, <a name="bb0255" href="#b0255" class="workspace-trigger">Wiens et al., 2005</a>, <a name="bb0125" href="#b0125" class="workspace-trigger">Köhler and Günther, 2008</a>). Target gene regions were amplified and as in <a name="bb0030" href="#b0030" class="workspace-trigger">Austin et al., 2010a</a>, <a name="bb0035" href="#b0035" class="workspace-trigger">Austin et al., 2010b</a> and purified as in <a name="bb0040" href="#b0040" class="workspace-trigger">Austin et al. (2011)</a>.
 Purified amplicons were then cycle sequenced in both directions with 
BigDye v. 3.1 Terminator Sequencing Kit (Applied Biosystems, Foster 
City, CA, USA) using previously published protocols and sequenced on an 
ABI 3100 automated capillary sequencer (<a name="bb0030" href="#b0030" class="workspace-trigger">Austin et al., 2010a</a>, <a name="bb0035" href="#b0035" class="workspace-trigger">Austin et al., 2010b</a>).</p><div class="tables frame-topbot rowsep-0 colsep-0" id="t0005"><div class="captions"><span><p id="sp0040"><span class="label">Table 1</span>. List of loci, PCR primers, and annealing temperatures.</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">Locus</th><th scope="col" class="align-left">Primer</th><th scope="col" class="align-left">Primer sequence 5′–3′</th><th scope="col" class="align-left">Temp. (°C)</th><th scope="col" class="align-left">References</th></tr></thead><tbody><tr class="valign-top"><td class="align-left" rowspan="2"><em>12S</em></td><td class="align-left"><em>L2519</em></td><td class="align-left">AAACTGGGATTAGATACCCCACTAT</td><td class="align-left" rowspan="2">55</td><td class="align-left"><a name="bb0210" href="#b0210" class="workspace-trigger">Richards and Moore, 1996</a></td></tr><tr class="valign-top"><td class="align-left"><em>H3296</em></td><td class="align-left">GCTAGACCATKATGCAAAAGGTA</td><td class="align-left"><a name="bb0210" href="#b0210" class="workspace-trigger">Richards and Moore, 1996</a></td></tr><tr class="valign-top"><td class="align-left" colspan="5"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2"><em>16S</em></td><td class="align-left"><em>16S-L</em></td><td class="align-left">TCGAACTTAGAGATAGCTGGTT</td><td class="align-left" rowspan="2">55</td><td class="align-left"></td></tr><tr class="valign-top"><td class="align-left"><em>16S-H</em></td><td class="align-left">GCGAATGTTTTTGGTAAACA</td><td class="align-left"></td></tr><tr class="valign-top"><td class="align-left" colspan="5"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2"><em>Cytb</em></td><td class="align-left"><em>CytbA</em></td><td class="align-left">CCATGAGGACAAATATCATTYTGRGG</td><td class="align-left" rowspan="2">46</td><td class="align-left"><a name="bb0050" href="#b0050" class="workspace-trigger">Bossuyt and Milinkovitch, 2000</a></td></tr><tr class="valign-top"><td class="align-left"><em>CytbB</em></td><td class="align-left">CTTCTACTGGTTGTCCTCCGATTCA</td><td class="align-left"><a name="bb0050" href="#b0050" class="workspace-trigger">Bossuyt and Milinkovitch, 2000</a></td></tr><tr class="valign-top"><td class="align-left" colspan="5"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2"><em>Tyrosinase</em></td><td class="align-left"><em>Tyr1A</em></td><td class="align-left">AGGTCCTCTTRAGCAAGGAATG</td><td class="align-left" rowspan="2">55</td><td class="align-left"><a name="bb0050" href="#b0050" class="workspace-trigger">Bossuyt and Milinkovitch, 2000</a></td></tr><tr class="valign-top"><td class="align-left"><em>Tyr1F</em></td><td class="align-left">TCATCTCCCGYCAYCTTCTGGAT</td><td class="align-left"><a name="bb0050" href="#b0050" class="workspace-trigger">Bossuyt and Milinkovitch, 2000</a></td></tr><tr class="valign-top"><td class="align-left" colspan="5"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2"><em>C-myc exon 2</em></td><td class="align-left"><em>cmyc2F</em></td><td class="align-left">ACVGARTTCCTGGGAGGGGACATGG</td><td class="align-left" rowspan="2">55</td><td class="align-left"><a name="bb0255" href="#b0255" class="workspace-trigger">Wiens et al., 2005</a></td></tr><tr class="valign-top"><td class="align-left"><em>cmyc-ex2d R</em></td><td class="align-left">TCATTCAATGGGTAAGGGAAGACC</td><td class="align-left"><a name="bb0255" href="#b0255" class="workspace-trigger">Wiens et al., 2005</a></td></tr><tr class="valign-top"><td class="align-left" colspan="5"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2"><em>C-myc exon 3</em></td><td class="align-left"><em>c-myc-ex3F</em></td><td class="align-left">CCCACCAGTCCAGACCTCACCACAG</td><td class="align-left" rowspan="2">48</td><td class="align-left"><a name="bb0255" href="#b0255" class="workspace-trigger">Wiens et al., 2005</a></td></tr><tr class="valign-top"><td class="align-left"><em>c-myc-ex3R</em></td><td class="align-left">GTTCTCTTTTGAGTTTTAACTGTTC</td><td class="align-left"><a name="bb0255" href="#b0255" class="workspace-trigger">Wiens et al., 2005</a></td></tr></tbody></table></div></div></div><p id="p0035">Complementary
 strands were assembled and visually edited in Sequencher v4.7 (Gene 
Codes Corp., Ann Arbor, MI, USA) and aligned in ClustalX2 (<a name="bb0135" href="#b0135" class="workspace-trigger">Larkin et al., 2007</a>). The number of variable sites for each locus was computed using Mega 5.0 (<a name="bb0240" href="#b0240" class="workspace-trigger">Tamura et al., 2011</a>). All sequences were deposited in GenBank; accession numbers are available in the <a name="bs0065" href="#s0065" class="workspace-trigger">Supplementary Appendix 1</a>.</p></section><section id="s0025"><h3>2.3. Phylogenetic analysis</h3><div><p id="p0040">To
 estimate phylogenetic relationships, Bayesian inference, maximum 
likelihood (ML), and maximum parsimony (MP) were implemented for 
individual loci as well as for a concatenated dataset of all six loci. 
Maximum parsimony analyses were conducted in PAUP<sup>*</sup> ver.4.0b10 (<a name="bb0235" href="#b0235" class="workspace-trigger">Swofford, 2003</a>) using PAUPrat (<a name="bb0230" href="#b0230" class="workspace-trigger">Sikes and Lewis, 2001</a>) to implement the parsimony ratchet (<a name="bb0165" href="#b0165" class="workspace-trigger">Nixon, 1999</a>).
 Fifty replicates of 5000 ratchet iterations and 20% character 
perturbation were conducted using tree bisection–reconstruction (TBR) 
branch swapping to find the most parsimonious trees using the CIPRES 
portal (<a name="bb0250" href="#b0250" class="workspace-trigger">Warnow, 2008</a>).
 Branch support for ML and MP was assessed with 1000 bootstrap (BS) 
pseudoreplicates. For Bayesian and ML analyses, a variety of 
partitioning strategies were tested, including partitioning by locus and
 by codon position (for the protein-coding cytochrome b gene). The 
best-fit model of sequence evolution for each partition was selected in 
jModeltest v0.1.1 (<a name="bb0185" href="#b0185" class="workspace-trigger">Posada, 2008</a>) using the corrected Akaike Information Criterion (AICc; <a name="bt0010" href="#t0010" class="workspace-trigger">Table 2</a>). ML analyses were implemented in Garli v2.0 (<a name="bb0280" href="#b0280" class="workspace-trigger">Zwickl, 2006</a>)
 using default parameters and five search replicates per partitioning 
strategy. The best-fit partitioning strategy was then selected using the
 AIC, and 50 total search replicates were conducted on the best-fit 
partitioning strategy. Bayesian analyses were implemented in MrBayes 
v.3.1.2 (<a name="bb0120" href="#b0120" class="workspace-trigger">Huelsenbeck and Ronquist, 2001</a>, <a name="bb0220" href="#b0220" class="workspace-trigger">Ronquist and Huelsenbeck, 2003</a>).
 As in ML analyses, a variety of partitioning strategies were tested and
 the best-fit strategy was selected by the AIC. For each partitioning 
strategy, Bayesian analyses consisted of two independent runs, each with
 four chains, run for 25&nbsp;million generations sampling every 1000 
generations. Substitution model parameters were unlinked among 
partitions, and rate priors were set to “variable” to allow differing 
substitution rates among partitions. The branch-length prior was set to 
an unconstrained exponential distribution with a mean of 50; all other 
priors were left at default settings. Convergence was assessed by 
examining the potential scale-reduction factors in MrBayes, as well as 
by examining the traces of all parameters and the effective sample sizes
 in Tracer v1.5 (<a name="bb0190" href="#b0190" class="workspace-trigger">Rambaut and Drummond, 2007</a>) and by comparing the Bayesian posterior probabilities (BPPs) of all splits in Are We There Yet (AWTY) (<a name="bb0170" href="#b0170" class="workspace-trigger">Nylander et al., 2008</a>).</p><div class="tables frame-topbot rowsep-0 colsep-0" id="t0010"><div class="captions"><span><p id="sp0045"><span class="label">Table 2</span>. List of the length, number of variable sites, and partitioning scheme for each locus.</p></span></div><div class="groups"><table><thead><tr class="valign-top"><th scope="col" class="align-left rowsep-1" rowspan="2">Partition</th><th scope="col" class="align-left rowsep-1" rowspan="2">Length</th><th scope="col" class="align-char">All samples</th><th scope="col" class="align-char">Ingroup samples</th><th scope="col" class="align-left rowsep-1" rowspan="2">Best-fit model</th></tr><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">No. variable sites (No. parsimony informative sites)</th><th scope="col" class="align-left">No. variable sites (No. parsimony informative sites)</th></tr></thead><tbody><tr class="valign-top"><td class="align-left"><em>12S</em></td><td class="align-left">687</td><td class="align-char">347 (301)</td><td class="align-char">218 (188)</td><td class="align-left">TIM3&nbsp;+&nbsp;G</td></tr><tr class="valign-top"><td class="align-left"><em>16S</em></td><td class="align-left">445</td><td class="align-char">298 (271)</td><td class="align-char">212 (184)</td><td class="align-left">HKY&nbsp;+&nbsp;G</td></tr><tr class="valign-top"><td class="align-left"><em>Cyt b pos. 1</em></td><td class="align-left">163</td><td class="align-char">65 (60)</td><td class="align-char">36 (32)</td><td class="align-left">TrN&nbsp;+&nbsp;G</td></tr><tr class="valign-top"><td class="align-left"><em>Cyt b pos. 2</em></td><td class="align-left">163</td><td class="align-char">28 (24)</td><td class="align-char">12 (11)</td><td class="align-left">TrN&nbsp;+&nbsp;G</td></tr><tr class="valign-top"><td class="align-left"><em>Cyt b pos. 3</em></td><td class="align-left">163</td><td class="align-char">160 (159)</td><td class="align-char">139 (132)</td><td class="align-left">TIM1&nbsp;+&nbsp;G</td></tr><tr class="valign-top"><td class="align-left"><em>Tyrosinase</em></td><td class="align-left">503</td><td class="align-char">155 (95)</td><td class="align-char">76 (45)</td><td class="align-left" rowspan="3">HKY&nbsp;+&nbsp;G</td></tr><tr class="valign-top"><td class="align-left"><em>c-myc exon 2</em></td><td class="align-left">330</td><td class="align-char">52 (28)</td><td class="align-char">20 (12)</td></tr><tr class="valign-top"><td class="align-left"><em>c-myc exon 3</em></td><td class="align-left">356</td><td class="align-char">70 (43)</td><td class="align-char">32 (16)</td></tr></tbody></table></div></div></div><p id="p0045">Because heterogeneity among gene genealogies may be substantial (<a name="bb0070" href="#b0070" class="workspace-trigger">Degnan and Rosenberg, 2006</a>, <a name="bb0075" href="#b0075" class="workspace-trigger">Degnan and Rosenberg, 2009</a>, <a name="bb0090" href="#b0090" class="workspace-trigger">Edwards et al., 2007</a>), in addition to concatenated phylogenies, we also estimated the underlying species tree using <sup>*</sup>BEAST (<a name="bb0110" href="#b0110" class="workspace-trigger">Heled and Drummond, 2010</a>), which accounts for heterogeneity among gene trees due to incomplete lineage sorting. Species tree analyses, such as <sup>*</sup>BEAST, require samples to be assigned <em>a priori</em> to species that accurately represent independently evolving evolutionary lineages. In this case, we suspected that <em>M. lateralis</em>
 represents a cryptic complex of multiple species. We generated four 
species trees assuming that well-supported (&gt;0.95 BPP, &gt;75 MLBS, 
&gt;75 MPBS) clades in the concatenated phylogenetic analyses with 3%, 
4%, 5%, and 6% mitochondrial divergence represented distinct lineages. 
Analyses were conducted using an uncorrelated lognormal molecular clock 
for 250&nbsp;million generations, sampling every 1000 generations. The 
first 50,000 samples were discarded as burn-in. Mitochondrial loci were 
partitioned as in the concatenated analysis, but were constrained to a 
single gene genealogy due because recombination within the mitochondrial
 genome is unexpected. Each nuclear locus was allowed to evolve 
independently under its own best-fit model. Convergence and stationarity
 were assessed by examining traces and effective sample sizes of all 
parameters in Tracer 1.5.</p></section><section id="s0030"><h3>2.4. Biogeographic reconstruction</h3><div><p id="p0050">To examine the biogeographic history of <em>Mantophryne,</em> we used maximum likelihood ancestral state reconstruction (MLASR) (<a name="bb0225" href="#b0225" class="workspace-trigger">Schluter et al., 1997</a>), Statistical Dispersal–Vicariance Analysis (S-DIVA) (<a name="bb0260" href="#b0260" class="workspace-trigger">Yu et al., 2010</a>), and Dispersal–Extinction Cladogenesis (DEC) (<a name="bb0200" href="#b0200" class="workspace-trigger">Ree et al., 2005</a>) analyses. These methods require discrete regions of occurrence, which we defined by geologic history (<a name="bf0010" href="#f0010" class="workspace-trigger">Fig. 2</a>):
 (0) North Coast (including accreted portions of Outer Melanesian Island
 Arc), (1) Papuan Peninsula (East Papuan Composite Terrane), (2) 
D’Entrecasteaux Islands, (3) Louisiade Islands (an extension of the Owen
 Stanley Terrane on the Papuan Peninsula), and (4) Southern and Western 
New Guinea (Australian Plate). MLASR was conducted in Mesquite v. 2.74 (<a name="bb0150" href="#b0150" class="workspace-trigger">Maddison and Maddison, 2010</a>) using the maximum clade credibility species tree estimated in <sup>*</sup>BEAST.
 However, this method assumes that each species (extant and ancestral) 
occurred in a single region, whereas, in reality, it is possible that 
some ancestral species spanned multiple regions. Thus, we also 
investigated biogeographic history using S-DIVA and DEC analyses. Both 
S-DIVA and DEC accomodate more realistic, though more complex, models by
 allowing distributions to span multiple regions and by limiting 
dispersal between distant regions. S-DIVA was implemented in RASP 
ver.1.107 (<a name="bb0260" href="#b0260" class="workspace-trigger">Yu et al., 2010</a>, <a name="bb0265" href="#b0265" class="workspace-trigger">Yu et al., 2011</a>) using a posterior distribution of 20,000 species trees from <sup>*</sup>BEAST
 analysis (thinned to one per 10,000 iterations) to integrate 
reconstructions over phylogenetic uncertainty. DEC was conducted using 
lagrange ver.2.0.1 (<a name="bb0205" href="#b0205" class="workspace-trigger">Ree and Smith, 2008</a>) in Python ver.2.72, using the maximum clade credibility species tree estimated in <sup>*</sup>BEAST.
 In both S-DIVA and DEC analyses, we constrained ancestral distributions
 to adjacent areas, i.e. 0–1, 1–2, 1–3, 1–2–3, or 1–4. We did not allow a
 0–4 distribution because unsuitable montane habitat occurs between 
regions 0 and 4, making dispersal directly between these regions highly 
unlikely. Similarly, we also constrained dispersal between island 
systems and either North Coast or southern/western populations, since 
these dispersals are unlikely without traversing the mainland Papuan 
peninsula.</p><figure class="figure" id="f0010"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr2.jpg" alt="" height="605"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr2.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p id="sp0025"><span class="label">Fig. 2</span>. Biogeographic reconstruction of <em>Mantophryne</em>.
 Biogeographic regions: 0-North Coast (includes accreted portions of 
Outer Melanesia Island Arc), 1-Papuan Peninsula (East Papuan Composite 
Terrane), 2-D’Entrecasteaux Islands, 3-Louisiade Islands, and 4-Southern
 and Western New Guinea (Australian Plate). Support values for each node
 are listed in <a name="bt0015" href="#t0015" class="workspace-trigger">Table 3</a>.</p></span></div></figure></div></section></section><section id="s0035"><h2>3. Results</h2><section id="s0040"><h3>3.1. Phylogenetic analyses</h3><div><p id="p0055">The final aligned length of the combined mitochondrial and nuclear dataset was 1621&nbsp;bp.
 The length of each partition, number of variable and parsimony 
informative sites, and models for nucleotide substitution are provided 
in <a name="bt0010" href="#t0010" class="workspace-trigger">Table 2</a>.
 Putative species groupings generated using 3% and 4% divergence were 
identical, and all analyses yielded identical topologies with similar 
support; thus, we present the results from the 4% divergence analysis. 
The phylogenetic reconstruction (<a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>) suggests that <em>M. lateralis</em>
 is monophyletic and split into nine clades: (A) Mt. Shungol, Morobe 
Province, (B) North Coast New Guinea – Huon Peninsula (Outer Melanesian 
Arc), (C) Northern Milne Bay Province, (D) Central and Oro Province, (E)
 Southwestern New Guinea and Southern Highlands Province, (F) Haia, Gulf
 Province (G) Sobo and Haia, Gulf Province, (H) Southern Milne Bay 
Province, (I) the D’Entrecasteaux islands. Each <em>M. lateralis</em> 
clade is strongly supported with 100 MPBS, &gt;89 MLBS and 1.0 BPP, and 
is geographically isolated, with one exception: samples from Haia, Gulf 
Province were recovered in both clades F and G. <em>Pherohapsis</em> and a putative new species of <em>Mantophryne</em> from Amau Village, Papua New Guinea (<em>M</em>. Amau sp.) are sister taxa and were recovered as embedded within <em>Mantophryne</em>, the sister clade to the <em>M. lateralis</em> complex. <em>Mantophryne louisiadensis</em>&nbsp;+&nbsp;<em>M. axanthogaster</em> were recovered as sister taxa and forming the sister group to <em>M. lateralis</em>&nbsp;+&nbsp;<em>Pherohapsis</em>. Furthermore, <em>M. infulata</em> was recovered within the sister genus <em>Hylophorbus</em>, rather than within <em>Mantophryne</em>.</p><figure class="figure" id="f0015"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr3.jpg" alt="" height="757"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr3.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p id="sp0030"><span class="label">Fig. 3</span>. 
Concatenated phylogeny. Branch support values are listed as maximum 
parsimony bootstrap/maximum likelihood bootstrap/Bayesian posterior 
probability (BPP). 100 bootstrap or 1.0 BPP are indicated by an *. 
Bootstraps less than 50, and BPP less than 0.75, are not listed. Symbols
 correspond to those in <a name="bf0005" href="#f0005" class="workspace-trigger">Fig. 1</a>.</p></span></div></figure></div><div><p id="p0060">The species tree estimates (<a name="bf0020" href="#f0020" class="workspace-trigger">Fig. 4</a>) largely corroborate the concatenated gene tree. The only exception is the relationships among <em>M. lateralis</em> clades E, F, and G. In the concatenated gene tree, E is the sister taxon to F&nbsp;+&nbsp;G. However, in the species tree, F is the sister-taxon to E&nbsp;+&nbsp;G. This E&nbsp;+&nbsp;F&nbsp;+&nbsp;G clade is strongly supported in both the species tree (BPP&nbsp;=&nbsp;0.99) and the concatenated gene tree (MPBS&nbsp;=&nbsp;90, MLBS&nbsp;=&nbsp;92, BPP&nbsp;=&nbsp;1.0). Although the sister taxon relationship of clades F&nbsp;+&nbsp;G is strongly supported in the concatenated phylogeny (MPBS&nbsp;=&nbsp;100, MLBS&nbsp;=&nbsp;89.7, BPP&nbsp;=&nbsp;1.0), the sister relationship of clades E&nbsp;+&nbsp;G is only weakly supported in the species tree (BPP&nbsp;=&nbsp;0.81). Lower-level relationships are often discordant between these two methods due to heterogeneity among gene trees (<a name="bb0070" href="#b0070" class="workspace-trigger">Degnan and Rosenberg, 2006</a>, <a name="bb0075" href="#b0075" class="workspace-trigger">Degnan and Rosenberg, 2009</a>, <a name="bb0090" href="#b0090" class="workspace-trigger">Edwards et al., 2007</a>).</p><figure class="figure" id="f0020"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr4.jpg" alt="" height="422"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-gr4.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p id="sp0035"><span class="label">Fig. 4</span>. Species tree from <sup>*</sup>BEAST.
 Node supports are listed as Bayesian posterior probabilities. Support 
values &lt;0.75 are not listed and values of 1.0 are listed as *.</p></span></div></figure></div></section><section id="s0045"><h3>3.2. Biogeography</h3><div><p id="p0065">MLASR, S-DIVA, and DEC produced similar results (<a name="bt0015" href="#t0015" class="workspace-trigger">Table 3</a>). MLSAR suggests that <em>Mantophryne</em> originated in the Papuan Peninsula (<em>p</em>&nbsp;=&nbsp;0.54) over the Louisiade Islands (<em>p</em>&nbsp;=&nbsp;0.19), while S-DIVA reconstructs a root distribution spanning the Papuan Peninsula and the Louisiade Islands (<em>p</em>&nbsp;=&nbsp;0.999). DEC (<em>p</em>&nbsp;=&nbsp;0.73) estimates the initial split within <em>Mantophryne</em> occurring between lineages in the Louisiade Islands and in the Papuan Peninsula (<em>p</em>&nbsp;=&nbsp;0.739). MLASR (<em>p</em>&nbsp;=&nbsp;0.75, 0.81), S-DIVA (<em>p</em>&nbsp;=&nbsp;0.99, 0.99), and DEC (<em>p</em>&nbsp;=&nbsp;0.79, 0.55) all suggest that mainland <em>Mantophryne</em> and the <em>M. lateralis</em>
 complex originated in the Papuan peninsula. Furthermore, all analyses 
suggest that the North Coast and South Coast populations each originated
 from single dispersal events out of the Papuan Peninsula region.</p><div class="tables frame-topbot rowsep-0 colsep-0" id="t0015"><div class="captions"><span><p id="sp0050"><span class="label">Table 3</span>. 
Biogeographic reconstruction probabilities. 0-Northern New Guinea 
(accreted portions of the Outer Melanesian Island Arc), 1-Papuan 
Peninsula (East Papuan Composite Terrane), 2-D’Entrecasteaux Islands, 
3-Louisiade Islands and 4-Southern and Western New Guinea (Australian 
Plate).</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">Node</th><th scope="col" class="align-left">Reconstructed distribution</th><th scope="col" class="align-left">MLASR support</th><th scope="col" class="align-left">Reconstructed distribution</th><th scope="col" class="align-left">S-DIVA support</th><th scope="col" class="align-left">Reconstructed distribution</th><th scope="col" class="align-left">DEC support</th></tr></thead><tbody><tr class="valign-top"><td class="align-left" rowspan="5">1</td><td class="align-left">[1]</td><td class="align-left">0.542</td><td class="align-left">[13]</td><td class="align-left">0.999</td><td class="align-left">[3|1]</td><td class="align-left">0.739</td></tr><tr class="valign-top"><td class="align-left">[3]</td><td class="align-left">0.198</td><td class="align-left"></td><td class="align-left"></td><td class="align-left">[3|12]</td><td class="align-left">0.131</td></tr><tr class="valign-top"><td class="align-left">[4]</td><td class="align-left">0.097</td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left">[2]</td><td class="align-left">0.087</td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left">[0]</td><td class="align-left">0.075</td><td class="align-left">&nbsp;</td><td class="align-left">&nbsp;</td><td class="align-left">&nbsp;</td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left">2</td><td class="align-left">[3]</td><td class="align-left">0.985</td><td class="align-left">[3]</td><td class="align-left">1.000</td><td class="align-left">[3|3]</td><td class="align-left">0.992</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="4">3</td><td class="align-left">[1]</td><td class="align-left">0.752</td><td class="align-left">[1]</td><td class="align-left">0.996</td><td class="align-left">[1|1]</td><td class="align-left">0.792</td></tr><tr class="valign-top"><td class="align-left">[3]</td><td class="align-left">0.079</td><td class="align-left"></td><td class="align-left"></td><td class="align-left">[12|1]</td><td class="align-left">0.143</td></tr><tr class="valign-top"><td class="align-left">[4]</td><td class="align-left">0.072</td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left">[2]</td><td class="align-left">0.058</td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left">4</td><td class="align-left">[1]</td><td class="align-left">0.994</td><td class="align-left">[1]</td><td class="align-left">1.000</td><td class="align-left">[1|1]</td><td class="align-left">0.996</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="3">5</td><td class="align-left">[1]</td><td class="align-left">0.817</td><td class="align-left">[1]</td><td class="align-left">0.998</td><td class="align-left">[1|1]</td><td class="align-left">0.559</td></tr><tr class="valign-top"><td class="align-left">[4]</td><td class="align-left">0.083</td><td class="align-left"></td><td class="align-left"></td><td class="align-left">[1|12]</td><td class="align-left">0.217</td></tr><tr class="valign-top"><td class="align-left">[2]</td><td class="align-left">0.055</td><td class="align-left"></td><td class="align-left"></td><td class="align-left">[14|1]</td><td class="align-left">0.192</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2">6</td><td class="align-left">[1]</td><td class="align-left">0.758</td><td class="align-left">[12]</td><td class="align-left">1.000</td><td class="align-left">[1|2]</td><td class="align-left">0.968</td></tr><tr class="valign-top"><td class="align-left">[2]</td><td class="align-left">0.196</td><td class="align-left">&nbsp;</td><td class="align-left">&nbsp;</td><td class="align-left">&nbsp;</td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2">7</td><td class="align-left">[1]</td><td class="align-left">0.817</td><td class="align-left">[14]</td><td class="align-left">1.000</td><td class="align-left">[4|1]</td><td class="align-left">0.942</td></tr><tr class="valign-top"><td class="align-left">[4]</td><td class="align-left">0.137</td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left">&nbsp;</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left">8</td><td class="align-left">[4]</td><td class="align-left">0.972</td><td class="align-left">[4]</td><td class="align-left">1.000</td><td class="align-left">[4|4]</td><td class="align-left">0.984</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left">9</td><td class="align-left">[4]</td><td class="align-left">0.999</td><td class="align-left">[4]</td><td class="align-left">1.000</td><td class="align-left">[4|4]</td><td class="align-left">0.998</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left" rowspan="2">10</td><td class="align-left">[1]</td><td class="align-left">0.977</td><td class="align-left">[01]</td><td class="align-left">0.518</td><td class="align-left">[1|1]</td><td class="align-left">0.757</td></tr><tr class="valign-top"><td class="align-left"></td><td class="align-left"></td><td class="align-left">[1]</td><td class="align-left">0.482</td><td class="align-left">[01|1]</td><td class="align-left">0.238</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left">11</td><td class="align-left">[1]</td><td class="align-left">0.997</td><td class="align-left">[1]</td><td class="align-left">1.000</td><td class="align-left">[1|1]</td><td class="align-left">0.999</td></tr><tr class="valign-top"><td class="align-left" colspan="7"><br></td></tr><tr class="valign-top"><td class="align-left">12</td><td class="align-left">[1]</td><td class="align-left">0.953</td><td class="align-left">[01]</td><td class="align-left">1.000</td><td class="align-left">[1|0]</td><td class="align-left">0.941</td></tr></tbody></table></div></div></div></section></section><section id="s0050"><h2>4. Discussion</h2><p id="p0070">This is the first phylogeographic study of any New Guinean frog. Our molecular data for the widespread <em>M. lateralis</em>
 suggest that there may be nine or more independently evolving lineages 
across its range in eastern New Guinea. These data, combined with the 
recent work by Günther and others on the genus <em>Hylophorbus</em> in 
central and western New Guinea, suggest that some, and possibly many, of
 the currently widespread frog species in New Guinea may instead 
represent cryptic complexes of multiple species (<a name="bb0095" href="#b0095" class="workspace-trigger">Günther, 2001</a>, <a name="bb0215" href="#b0215" class="workspace-trigger">Richards and Oliver, 2007</a>, <a name="bb0130" href="#b0130" class="workspace-trigger">Kraus and Allison, 2009</a>). While <a name="bb0095" href="#b0095" class="workspace-trigger">Günther (2001)</a> primarily focused his species analyses on <em>Hylophorbus</em> at the westernmost tip of New Guinea, our phylogenetic analyses included several populations of <em>Hylophorbus</em>
 from the central and eastern part of the island. These samples were 
deeply divergent, and may represent independent evolutionary lineages. <em>Hylophorbus</em> exhibits similar ecology to <em>Mantophryne</em>
 and could, therefore, display similar topological and spatial patterns.
 In addition, we included two disparate populations of the widespread <em>Sphenophryne cornuta</em>.
 These populations, sampled from north and south of the central 
cordillera, similarly show a large degree of sequence divergence (14.8% 
mtDNA), and may represent distinct species.</p><p id="p0075">Recent work on New Guinean reptiles has also revealed cryptic diversity in wide-ranging species, including two skinks (<em>Sphenomorphus jobiensis</em> and <em>Carlia fusca</em>) and three snakes (two <em>Aspidomorphus</em> species and <em>Morelia viridis</em>) (<a name="bb0085" href="#b0085" class="workspace-trigger">Donnellan and Aplin, 1989</a>, <a name="bb0195" href="#b0195" class="workspace-trigger">Rawlings and Donnellan, 2003</a>, <a name="bb0270" href="#b0270" class="workspace-trigger">Zug, 2004</a>, <a name="bb0040" href="#b0040" class="workspace-trigger">Austin et al., 2011</a>).
 Preliminary results from work done by others and us involving 
additional lizard “species” show similar patterns (unpublished data). 
These data, in concert with our findings, suggest that cryptic species 
might be more common in New Guinea than previously thought. Current 
understanding has been based in large part on studies of museum 
specimens, where details of color and pattern are often difficult to 
discern. Research involving living animals clearly supports the fact 
that many New Guinean taxa currently treated as single species represent
 complexes of multiple species (F. Kraus, unpubl. data).</p><p id="p0080">Biogeographic reconstructions suggest that <em>Mantophryne</em>
 most likely evolved in the Papuan peninsular region and subsequently 
dispersed outward. This would require at least two oceanic dispersal 
events to the Louisiade and D’Etrecasteaux archipelagos, as these 
islands have long remained isolated from mainland New Guinea (<a name="bb0145" href="#b0145" class="workspace-trigger">Luyendyk et al., 1973</a>, <a name="bb0065" href="#b0065" class="workspace-trigger">Davies and Warren, 1988</a>, <a name="bb0115" href="#b0115" class="workspace-trigger">Hill et al., 1992</a>, <a name="bb0010" href="#b0010" class="workspace-trigger">Abers et al., 1997</a>, <a name="bb0140" href="#b0140" class="workspace-trigger">Little et al., 2007</a>).
 While the three biogeographic analyses may initially appear 
incongruent, this reflects methodological differences, rather than truly
 conflicting signal. MLASR suggests that the peninsular origin is the 
most likely but also indicates a possible Louisiade islands origin, 
while S-DIVA reconstructs the basal node of <em>Mantophryne</em> as 
occurring in both the Louisiades and the Papuan peninsula. The results 
from DEC indicate that at the time of the basal split, <em>Mantophryne</em> occured on both the Papuan peninsula and the Louisiade islands. Interestingly, the genus <em>Hylophorbus</em>
 also occurs in both the Louisiade and D’Entrecasteaux archipelagos off 
the eastern tip of New Guinea. This is consistent with other frog 
genera, such as <em>Litoria, Hylarana, Copiula,</em> and <em>Callulops</em>, suggesting that the concordant distributions of these two genera may mirror common historical and ecological processes.</p><p id="p0085">Our
 results based on an improved understanding of phylogenetic 
relationships highlight the taxonomic confusion associated with the 
closely related genera <em>Mantophryne</em>, <em>Hylophorbus</em> and <em>Pherohapsis</em>. The monotypic <em>Pherohapsis</em> is embedded within <em>Mantophryne</em> and thus should be synonymized, changing <em>P. menziesi</em> to <em>M. menziesi</em>. Our results also show that <em>M. infulata</em> is nested within <em>Hylophorbus</em> and should be included in that genus as <em>H. infulatus.</em> We are currently analyzing call and morphological data to assess whether nine clades of the <em>lateralis</em> complex, as well as the putative new species of <em>Mantophryne</em> from Amau, are conspecific lineages or distinct species in an effort towards a comprehensive species delimitation analysis for <em>Mantophryne</em>.</p></section></div><section id="aep-acknowledgment-id17"><h2>Acknowledgments</h2><p id="p0115">We
 thank the many villagers on whose land we conducted fieldwork. We thank
 B. Roy, V. Kula, and B. Wilmot from the PNG Department of Environment 
and Conservation, J. Robins from the PNG National Research Institute, 
and J. Animiato, I. Bigilale, B. Iova, R. Singadan and P. Kei who have 
provided research assistance in Papua New Guinea. This manuscript was 
improved by comments from the Austin lab group, Bryan Carstens, Sabrina 
Taylor, Tara Pelletier, and two anonymous reviewers. All research was 
approved under LSU IACUC protocol 06-071. This research was funded by <span id="GS1">National Science Foundation</span> Grants <a href="#GS1">DEB 0445213, DBI 0400797, and DEB 1146033 to CCA; and DEB 0103794 and 0743890 to FK</a>.</p></section><div class="Appendices"><section id="s0065"><h2>Appendix A. Supplementary material</h2><p id="p0125"><span class="display"><div class="e-component e-component-mmc1" id="m0005"><div class="article-attachment"><a class="icon-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-mmc1.xls" title="Download spreadsheet (79KB)" target="_blank"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-interactive-case-insights"><path d="m2 1e1v1.1e2h94v-1.1e2h-94zm84 1e2h-74v-9e1h74v9e1zm-64-28h14v16h-14zm2e1 -18h14v34h-14zm2e1 8h14v26h-14zm-6.04-16.3l22.04-11.86v-11.36l-21.96 11.82-19.06-10.58-16.98 18.52v14.8l19.02-20.76z"></path></svg></a><a class="download-link" target="_blank" download="" title="Download spreadsheet (79KB)" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790313000808-mmc1.xls">Download spreadsheet (79KB)</a><a class="help-link right" title="Help with xls files (Opens in new window)" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank">Help with xls files</a></div><div class="captions"><span><p id="sp0065"><span class="label">Supplementary data 1</span>. List of all samples used in study with corresponding museum catalog numbers, localities, and Gen Bank accession numbers</p></span></div></div></span></p></section></div></div><div class="related-content-links u-hide-from-md"><button class="button button-anchor" type="button"><span class="button-text">Recommended articles</span></button><button class="button button-anchor" type="button"><span class="button-text">Citing articles (<!-- -->12<!-- -->)</span></button></div><div class="Tail"></div><section class="bibliography" id="bi0005"><h2 class="section-title">References</h2><section class="bibliography-sec" id="bs0005"><dl class="references"><dt class="label"><a href="#bb0005" id="ref-id-b0005">Abbott et al., 1994</a></dt><dd class="reference" id="h0220"><div class="contribution">L.D. Abbott, E.A. Silver, P.R. Thompson, M.V. Filewicz, C. Schnieder, Abdoerrias<strong class="title">Stratigraphic constraints on the development and timing of arc-continent collision in northern Papua New Guinea</strong></div><div class="host">J. Sediment. Res., 64 (2b) (1994), pp. 169-183</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0010" id="ref-id-b0010">Abers et al., 1997</a></dt><dd class="reference" id="h0225"><div class="contribution">G.A. Abers, C.Z. Mutter, J. Fang<strong class="title">Shallow dips of normal faults during rapid extension: earthquakes in the Woodlark-D’Entrecasteaux rift system, Papua New Guinea</strong></div><div class="host">J. Geophy. Res., 102 (B7) (1997)</div><div class="comment">15, 301–315, 327</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0015" id="ref-id-b0015">Allison, 1996</a></dt><dd class="reference" id="h0005"><div class="contribution">A. Allison<strong class="title">Zoogeography of amphibians and reptiles of New Guinea and the Pacific region</strong></div><div class="host">A. Keast, S.E. Miller (Eds.), The Origin and Evolution of Pacific Island Biotas, New Guinea to Eastern Polynesia: Patterns and Processes, SPB Academic Publishing, Amsterdam (1996), pp. 407-436</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0020" id="ref-id-b0020">Allison et al., 2010</a></dt><dd class="reference" id="j0015"><span>Allison, A., Kraus, F., Polhemus, D., Pratt, T., 2010. Papuan Herpetofauna: An Online Reference. &lt;<a href="http://pbs.bishopmuseum.org/papuanherps/" target="_blank">http://pbs.bishopmuseum.org/papuanherps/</a>&gt;.</span></dd><dt class="label"><a href="#bb0025" id="ref-id-b0025">Austin et al., 2008</a></dt><dd class="reference" id="h0010"><div class="contribution">C.C. Austin, C.J. Hayden, I. Bigilale, C. Dahl, J. Anaminiato<strong class="title">Checklist and comments on the terrestrial amphibian and reptile fauna from Utai, northwestern Papua New Guinea</strong></div><div class="host">Herpetol. Rev., 39 (1) (2008), pp. 40-46</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0030" id="ref-id-b0030">Austin et al., 2010a</a></dt><dd class="reference" id="h0015"><div class="contribution">C.C. Austin, E.N. Rittmeyer, S.J. Richards, G.R. Zug<strong class="title">Phylogeny, historical biogeography and body size evolution in Pacific Island Crocodile skinks <em>Tribolonotus</em> (Squamata; Scinicidae)</strong></div><div class="host">Mol. Phylogenet. Evol., 57 (2010), pp. 227-236</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0035" id="ref-id-b0035">Austin et al., 2010b</a></dt><dd class="reference" id="h0020"><div class="contribution">C.C. Austin, M. Spataro, S. Peterson, J. Jordan, J.D. McVay<strong class="title">Conservation genetics of Boelen’s python (<em>Morelia boeleni</em>) from New Guinea: reduced genetic diversity and divergence of captive and wild animals</strong></div><div class="host">Conserv. Genet., 11 (2010), pp. 889-896</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0040" id="ref-id-b0040">Austin et al., 2011</a></dt><dd class="reference" id="h0025"><div class="contribution">C.C. Austin, E.N. Rittmeyer, L.A. Oliver, J.O. Andermann, G.R. Zug, G.H. Rodda, N.D. Jackson<strong class="title">The bioinvasion of Guam: inferring geographic origin, pace, pattern and process of an invasive lizard (<em>Carlia</em>) in the Pacific using multi-locus genomic data</strong></div><div class="host">Biol. Invasions, 13 (2011), pp. 1951-1967</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0045" id="ref-id-b0045">Beehler, 1993</a></dt><dd class="reference" id="j0070"><span>Beehler,
 B.M., 1993. Mapping PNG’s biodiversity. Papua New Guinea Conservation 
Needs Assessment. B.M. Beehler. Washington, DC, The Biodiversity Support
 Program, vol. 2, pp. 193–209.</span></dd><dt class="label"><a href="#bb0050" id="ref-id-b0050">Bossuyt and Milinkovitch, 2000</a></dt><dd class="reference" id="h0035"><div class="contribution">F. Bossuyt, M.C. Milinkovitch<strong class="title">Convergent adaptive radiations in Madagascan and Asian ranid frogs reveal covariation between larval and adult traits</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 97 (12) (2000), pp. 6585-6590</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0055" id="ref-id-b0055">Boulenger, 1897</a></dt><dd class="reference" id="h0040"><div class="contribution">G.A. Boulenger<strong class="title">Description of New lizards and frogs from Mt. Victoria, Owen Stanley Range, New Guinea collected by Mr. A.S</strong></div><div class="host">Anthony. Ann. Mag. Nat. Hist., 19 (1897), pp. 6-13</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0060" id="ref-id-b0060">Burton, 1986</a></dt><dd class="reference" id="h0045"><div class="contribution">T.C. Burton<strong class="title">A reassessment of the Papuan subfamily Asterophryinae (Anura: Microhylidae)</strong></div><div class="host">Rec. South Aust. Mus., 19 (1986), pp. 405-450</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0065" id="ref-id-b0065">Davies and Warren, 1988</a></dt><dd class="reference" id="h0050"><div class="contribution">H.L. Davies, R.G. Warren<strong class="title">Origin
 of eclogite-bearing, domed, layered metamorphic complexes (core 
complexes) in the D’Entrecasteaux Islands, Papua New Guinea</strong></div><div class="host">Tecton, 7 (1) (1988), pp. 1-21</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0070" id="ref-id-b0070">Degnan and Rosenberg, 2006</a></dt><dd class="reference" id="h0055"><div class="contribution">J.H. Degnan, N.A. Rosenberg<strong class="title">Discordance of species trees with their most likely gene trees</strong></div><div class="host">PLoS Genet., 2 (2006), pp. 762-768</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0075" id="ref-id-b0075">Degnan and Rosenberg, 2009</a></dt><dd class="reference" id="h0060"><div class="contribution">J.H. Degnan, N.A. Rosenberg<strong class="title">Gene tree discordance, phylogenetic inference and the multispecies coalescent</strong></div><div class="host">Trends Ecol. Evol., 24 (6) (2009), pp. 332-340</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0080" id="ref-id-b0080">Dinerstein and Wikramanayake, 1993</a></dt><dd class="reference" id="h0065"><div class="contribution">E. Dinerstein, E.D. Wikramanayake<strong class="title">Beyond hotspots: how to prioritize investments to conserve biodiversity in the Indo-Pacific region</strong></div><div class="host">Conserv. Biol., 7 (1993), pp. 53-65</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0085" id="ref-id-b0085">Donnellan and Aplin, 1989</a></dt><dd class="reference" id="h0070"><div class="contribution">S.C. Donnellan, K.P. Aplin<strong class="title">Resolution of cryptic species in the New Guinean lizard, <em>Sphenomorphus jobiensis</em> (Scincidae) by electrophoresis</strong></div><div class="host">Copeia, 1989 (1) (1989), pp. 81-88</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0090" id="ref-id-b0090">Edwards et al., 2007</a></dt><dd class="reference" id="h0075"><div class="contribution">S.V. Edwards, L. Liu, D.K. Pearl<strong class="title">High-resolution species trees without concatenation</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 104 (14) (2007), pp. 5936-5941</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0095" id="ref-id-b0095">Günther, 2001</a></dt><dd class="reference" id="h0080"><div class="contribution">R. Günther<strong class="title">The Papuan frog genus <em>Hylophorbus</em> (Anura: Microhylidae) in not monospecific: descriptions of six new species</strong></div><div class="host">Russ. J. Herpetol., 8 (2) (2001), pp. 81-104</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0100" id="ref-id-b0100">Hall, 1997</a></dt><dd class="reference" id="j0020"><span>Hall, R., 1997. Cenozoic tectonics of SE Asia and Australasia. Proc. Pet. Syst, SE Asia Australas. Conf. pp. 47–62.</span></dd><dt class="label"><a href="#bb0105" id="ref-id-b0105">Heads, 2002</a></dt><dd class="reference" id="h0085"><div class="contribution">M. Heads<strong class="title">Birds of paradise, vicariance biogeography and terrane tectonics in New Guinea</strong></div><div class="host">J. Biogeogr., 29 (2002), pp. 261-283</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0110" id="ref-id-b0110">Heled and Drummond, 2010</a></dt><dd class="reference" id="h0090"><div class="contribution">J. Heled, A.J. Drummond<strong class="title">Bayesian inference of species trees from multilocus data</strong></div><div class="host">Mol. Biol. Evol., 27 (3) (2010), pp. 570-580</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0115" id="ref-id-b0115">Hill et al., 1992</a></dt><dd class="reference" id="h0095"><div class="contribution">E.J. Hill, S.L. Baldwin, G.S. Lister<strong class="title">Unroofing of active metamorphic core complexes in the D’Entrecasteaux Islands, Papua New Guinea</strong></div><div class="host">Geology, 20 (1992), pp. 907-910</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0120" id="ref-id-b0120">Huelsenbeck and Ronquist, 2001</a></dt><dd class="reference" id="h0100"><div class="contribution">J.P. Huelsenbeck, F. Ronquist<strong class="title">MRBAYES: Bayesian inference of phylogeny</strong></div><div class="host">Bioinforma, 17 (2001), pp. 754-755</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0125" id="ref-id-b0125">Köhler and Günther, 2008</a></dt><dd class="reference" id="h0105"><div class="contribution">F. Köhler, R. Günther<strong class="title">The
 radiation of microhylid frogs (Amphibia: Anura) on New Guinea: a 
mitochondrial phylogeny reveals parallel evolution of morphological and 
life history traits and disproves the current morphology-based 
classification</strong></div><div class="host">Mol. Phylogenet. Evol., 47 (2008), pp. 353-365</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0130" id="ref-id-b0130">Kraus and Allison, 2009</a></dt><dd class="reference" id="h0110"><div class="contribution">F. Kraus, A. Allison<strong class="title">New species of frogs from Papua New Guinea</strong></div><div class="host">Bish. Mus. Occas. Pap., 104 (2009), pp. 1-36</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0135" id="ref-id-b0135">Larkin et al., 2007</a></dt><dd class="reference" id="h0115"><div class="contribution">M.A. Larkin, G. Blackshields, N.P. Brown, R. Chenna, P.A. McGettigan, H. McWilliam, F. Valentin, I.M. Wallace, A. Wilm, R. Lopez, J.D. Thomson, T.J. Gibson, D.G. Higgins<strong class="title">Clustal W and Clustal X version 2.0</strong></div><div class="host">Bioinforma, 23 (2007), pp. 2947-2948</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0140" id="ref-id-b0140">Little et al., 2007</a></dt><dd class="reference" id="h0230"><div class="contribution">T.A. Little, S.L. Baldwin, P.G. Fitzgerald, B. Monteleone<strong class="title">Continental
 rifting and metamorphic core complex formation ahead of the Woodlark 
spreading ridge, D’Entrecasteaux Islands, Papua New Guinea</strong></div><div class="host">Tecton, 26 (2007), p. TC1002</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0145" id="ref-id-b0145">Luyendyk et al., 1973</a></dt><dd class="reference" id="h0120"><div class="contribution">B.P. Luyendyk, K.T. MacDonald, W.B. Bryan<strong class="title">Rifting history of the Woodlark basin in the southwest Pacific</strong></div><div class="host">Geol. Soc. Am. Bull., 84 (4) (1973), pp. 1125-1134</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0150" id="ref-id-b0150">Maddison and Maddison, 2010</a></dt><dd class="reference" id="j0030"><span>Maddison, W.P., Maddison, D.R., 2010. Mesquite: A Modular System for Evolutionary Analysis. Version 2.73. &lt;<a href="http://mesquiteproject.org/" target="_blank">http://mesquiteproject.org</a>&gt;.</span></dd><dt class="label"><a href="#bb0155" id="ref-id-b0155">Menzies, 2006</a></dt><dd class="reference" id="h0125"><div class="contribution">J. Menzies<strong class="title">The Frogs of New Guinea and the Solomon Islands</strong></div><div class="host">Pensoft Publishers, Bulgaria (2006)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0160" id="ref-id-b0160">Myers et al., 2000</a></dt><dd class="reference" id="h0130"><div class="contribution">N. Myers, R.A. Mittermeier, C.G. Mittermeier, G.A.B. da Fonesca, J. Kent<strong class="title">Biodiversity hotspots for conservation priorities</strong></div><div class="host">Nature, 403 (2000), pp. 853-858</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0165" id="ref-id-b0165">Nixon, 1999</a></dt><dd class="reference" id="h0235"><div class="contribution">K.C. Nixon<strong class="title">The parsimony ratchet, a new method for rapid parsimony analysis</strong></div><div class="host">Cladistics, 15 (1999), pp. 407-414</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0170" id="ref-id-b0170">Nylander et al., 2008</a></dt><dd class="reference" id="h0135"><div class="contribution">J.A.A. Nylander, J.C. Wilgenbusch, D.L. Warren, D.L. Swofford<strong class="title">AWTY (are we there yet?): a system for graphical exploration of MCMC convergence in Bayesian phylogenetics</strong></div><div class="host">Bioinforma, 24 (4) (2008), pp. 581-583</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0175" id="ref-id-b0175">Parker, 1934</a></dt><dd class="reference" id="h0140"><div class="contribution">H.W. Parker<strong class="title">A Monograph of the Frogs of the Family Microhylidae</strong></div><div class="host">The British Museum, London (1934)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0180" id="ref-id-b0180">Pigram and Davies, 1987</a></dt><dd class="reference" id="h0145"><div class="contribution">C.J. Pigram, H.L. Davies<strong class="title">Terranes and the accretion history of the New Guinea Orogen</strong></div><div class="host">BMR J. Aust. Geol. Geophys., 10 (1987), pp. 193-211</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0185" id="ref-id-b0185">Posada, 2008</a></dt><dd class="reference" id="h0150"><div class="contribution">D. Posada<strong class="title">JModelTest: Phylogenetic model averaging</strong></div><div class="host">Mol. Biol. Evol., 25 (7) (2008), pp. 1253-1256</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0190" id="ref-id-b0190">Rambaut and Drummond, 2007</a></dt><dd class="reference" id="j0040"><span>Rambaut, A., Drummond, A.J., 2007. Tracer v1.4. &lt;<a href="http://beast.bio.ed.ac.uk/Tracer" target="_blank">http://beast.bio.ed.ac.uk/Tracer</a>&gt;.</span></dd><dt class="label"><a href="#bb0195" id="ref-id-b0195">Rawlings and Donnellan, 2003</a></dt><dd class="reference" id="h0155"><div class="contribution">L.H. Rawlings, S.C. Donnellan<strong class="title">Phylogeographic analysis of the green python, <em>Morelia viridis</em>, reveals cryptic diversity</strong></div><div class="host">Mol. Phylogenet. Evol., 27 (1) (2003), pp. 36-44</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0200" id="ref-id-b0200">Ree et al., 2005</a></dt><dd class="reference" id="h0160"><div class="contribution">R.H. Ree, B.R. Moore, C.O. Webb, M.J. Donoghue<strong class="title">A likelihood framework for inferring the evolution of geographic range on phylogenetic trees</strong></div><div class="host">Evolution, 59 (11) (2005), pp. 2299-2311</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0205" id="ref-id-b0205">Ree and Smith, 2008</a></dt><dd class="reference" id="h0165"><div class="contribution">R.H. Ree, S. Smith<strong class="title">Maximum-likelihood inference of geographic range evolution by dispersal, local extinction, and cladogenesis</strong></div><div class="host">Syst. Biol., 57 (2008), pp. 4-14</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0210" id="ref-id-b0210">Richards and Moore, 1996</a></dt><dd class="reference" id="h0170"><div class="contribution">C.M. Richards, W.S. Moore<strong class="title">A phylogeny for the African Treefrog Hyperoliidae based on mitochondrial rDNA</strong></div><div class="host">Mol. Phylogenet. Evol., 5 (1996), pp. 522-532</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0215" id="ref-id-b0215">Richards and Oliver, 2007</a></dt><dd class="reference" id="h0175"><div class="contribution">S.J. Richards, P.M. Oliver<strong class="title">A new species of <em>Hylophorbus</em> (Anura, Microhylidae) from the Juon Peninsula, Papua New Guinea</strong></div><div class="host">Mitt. Mus. Natur. Be. Zool. Reihe, 83 (2007), pp. 83-89</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0220" id="ref-id-b0220">Ronquist and Huelsenbeck, 2003</a></dt><dd class="reference" id="h0180"><div class="contribution">F. Ronquist, J.P. Huelsenbeck<strong class="title">MRBAYES 3: Bayesian phylogenetic inference under mixed models</strong></div><div class="host">Bioinforma, 19 (2003), pp. 1572-1574</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0225" id="ref-id-b0225">Schluter et al., 1997</a></dt><dd class="reference" id="h0185"><div class="contribution">D. Schluter, T. Price, A. Mooers, D. Ludwig<strong class="title">Likelihood of ancestor states in adaptive radiation</strong></div><div class="host">Evolution, 51 (6) (1997), pp. 1699-1711</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0230" id="ref-id-b0230">Sikes and Lewis, 2001</a></dt><dd class="reference" id="j0045"><span>Sikes, D.S., Lewis, P.O., 2001. PAUPRat: PAUP<sup>*</sup>
 Implementation of the Parsimony Ratchet. Distributed by the Authors. 
Department of Ecology and Evolutionary Biology, University of 
Connecticut, Storrs, USA.</span></dd><dt class="label"><a href="#bb0235" id="ref-id-b0235">Swofford, 2003</a></dt><dd class="reference" id="j0050"><span>Swofford, D.L., 2003. PAUP<sup>*</sup>: Phylogenetic Analysis using Parsimony, version 4.0b10. S. Associates. Sunderland.</span></dd><dt class="label"><a href="#bb0240" id="ref-id-b0240">Tamura et al., 2011</a></dt><dd class="reference" id="h0190"><div class="contribution">K. Tamura, D. Peterson, N. Peterson, G. Stecher, M. Nei, S. Kumar<strong class="title">MEGA5: Molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</strong></div><div class="host">Mol. Biol. Evol., 28 (10) (2011), pp. 2731-2739</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0245" id="ref-id-b0245">Tregoning et al., 1999</a></dt><dd class="reference" id="h0195"><div class="contribution">P. Tregoning, R.J. Jackson, McQueen, K. Lambeck, C. Stevens, R.P. Little, R. Curley, R. Rosa<strong class="title">Motion of the South Bismarck Plate, Papua New Guinea</strong></div><div class="host">Geophys. Red. Lett., 26 (23) (1999), pp. 3517-3520</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0250" id="ref-id-b0250">Warnow, 2008</a></dt><dd class="reference" id="j0055"><span>Warnow, T., 2008. Cipres Portal v 1.10. &lt;<a href="http://www.phylo.org/sub_sections/portal/" target="_blank">http://www.phylo.org/sub_sections/portal/</a>&gt;.</span></dd><dt class="label"><a href="#bb0255" id="ref-id-b0255">Wiens et al., 2005</a></dt><dd class="reference" id="h0200"><div class="contribution">J.T. Wiens, J.W. Fetzner Jr., C.L. Parkinson, T.W. Reeder<strong class="title">Hylid frog phylogeny and sampling strategies for speciose clades</strong></div><div class="host">Syst. Biol., 54 (2005), pp. 778-807</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0260" id="ref-id-b0260">Yu et al., 2010</a></dt><dd class="reference" id="h0205"><div class="contribution">Y. Yu, A. Harris, X. He<strong class="title">S-DIVA (Statistical Disperal–Vicariance Analysis): a tool for inferring biogeographic histories</strong></div><div class="host">Mol. Phylogenet. Evol., 56 (2010), pp. 848-850</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0265" id="ref-id-b0265">Yu et al., 2011</a></dt><dd class="reference" id="j0060"><span>Yu, Y., Harris, A., He, X., 2011. RASP (Reconstruct Ancenstreal State in Phylogenies) ver. 1.107. &lt;<a href="http://mnh.scu.edu.cn/soft/blog/RASP" target="_blank">http://mnh.scu.edu.cn/soft/blog/RASP</a>&gt;.</span></dd><dt class="label"><a href="#bb0270" id="ref-id-b0270">Zug, 2004</a></dt><dd class="reference" id="h0210"><div class="contribution">G.R. Zug<strong class="title">Systematics of the <em>Carlia fusca</em> lizards of New Guinea and nearby islands</strong></div><div class="host">Bish. Mus. Press, Honolulu (2004)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0275" id="ref-id-b0275">Zweifel, 1972</a></dt><dd class="reference" id="h0215"><div class="contribution">R.G. Zweifel<strong class="title">Results of the Archbold Expeditions. No. 97 A revision to the frogs of the subfamily Asterophryinae family Microhylidae</strong></div><div class="host">Bull. Am. Mus. Nat. Hist., 148 (3) (1972), pp. 411-546</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0280" id="ref-id-b0280">Zwickl, 2006</a></dt><dd class="reference" id="j0065"><span>Zwickl,
 D.J., 2006. Genetic Algorithm Approaches for the Phylogenetic Analysis 
of Large Biological Sequence Datasets Under The Maximum Likelihood 
Criterion. Ph.D. Dissertation, The University of Texas, Austin. &lt;<a href="http://www.bio.utexas.edu/faculty/antisense/garli/Garli.html" target="_blank">http://www.bio.utexas.edu/faculty/antisense/garli/Garli.html</a>&gt;.</span></dd></dl></section></section><div class="Copyright"><span class="copyright-line">Copyright © 2013 Elsevier Inc. All rights reserved.</span></div></article><div class="u-show-from-md col-lg-6 col-md-8 pad-right"><aside class="RelatedContent" aria-label="Related content"><section class="SidePanel"><header id="recommended-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Recommended articles</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S105579031730297X"><h3 class="article-title ellipsis" id="recommended-articles-article0-title" title="Molecular phylogenetics and dating of the problematic New Guinea microhylid frogs (Amphibia: Anura) reveals elevated speciation rates and need for taxonomic reclassification"><span>Molecular
 phylogenetics and dating of the problematic New Guinea microhylid frogs
 (Amphibia: Anura) reveals elevated speciation rates and need for 
taxonomic reclassification</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 112, 2017, pp. 1-11</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S105579031730297X/pdfft?md5=40ef09bd61dc1bd2cc62f14d83c12320&amp;pid=1-s2.0-S105579031730297X-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article0-title" aria-controls="recommended-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790314004254"><h3 class="article-title ellipsis" id="recommended-articles-article1-title" title="Molecular evidence for cryptic candidate species in Iberian Pelodytes (Anura, Pelodytidae)"><span>Molecular evidence for cryptic candidate species in Iberian <em>Pelodytes</em> (Anura, Pelodytidae)</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 83, 2015, pp. 224-241</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790314004254/pdfft?md5=4dcf939233fa6e3dffbd88828eff5337&amp;pid=1-s2.0-S1055790314004254-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article1-title" aria-controls="recommended-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790311002570"><h3 class="article-title ellipsis" id="recommended-articles-article2-title" title="Systematic relationships of Oriental tiny frogs of the family Microhylidae (Amphibia, Anura) as revealed by mtDNA genealogy"><span>Systematic relationships of Oriental tiny frogs of the family Microhylidae (Amphibia, Anura) as revealed by mtDNA genealogy</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 61, Issue 1, 2011, pp. 167-176</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790311002570/pdfft?md5=a6c13a3282ef101b6c83b0b327f20718&amp;pid=1-s2.0-S1055790311002570-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article2-title" aria-controls="recommended-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article2" aria-hidden="true"></div></li></ul></div><div class="pagination u-position-relative u-padding-xs-bottom"><span class="u-position-absolute"></span><span class="pagination-pages-label"><span class="pagination-nav u-margin-xs-hor pagination-current underline-page-number">1</span><span class="pagination-nav u-margin-xs-hor">2</span></span><span class="u-position-absolute"><button class="button-link button-link-secondary next-button" type="button"><span class="button-link-text">Next</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></button></span></div></div></section><section class="SidePanel"><header id="citing-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle button-link-primary" aria-expanded="false" type="button"><span class="button-link-text"><h2 class="section-title">Citing articles (12)</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="u-display-none" aria-hidden="true" aria-describedby="citing-articles-header"><div id="citing-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317308035" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article0-title" title="A large-scale phylogeny of Microhylidae inferred from a combined dataset of 121 genes and 427 taxa">A large-scale phylogeny of Microhylidae inferred from a combined dataset of 121 genes and 427 taxa</h3></a><div class="article-source ellipsis">2018, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317308035/pdfft?md5=2223e51e3181ed88eeb776a6270b118b&amp;pid=1-s2.0-S1055790317308035-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article0-title" aria-controls="citing-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317304323" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article1-title" title="Early insularity and subsequent mountain uplift were complementary drivers of diversification in a Melanesian lizard radiation (Gekkonidae: Cyrtodactylus)">Early
 insularity and subsequent mountain uplift were complementary drivers of
 diversification in a Melanesian lizard radiation (Gekkonidae: 
Cyrtodactylus)</h3></a><div class="article-source ellipsis">2018, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317304323/pdfft?md5=f1d730a615a45fec5142a58e329d4833&amp;pid=1-s2.0-S1055790317304323-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article1-title" aria-controls="citing-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790316303414" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article2-title" title="An ant genus-group (Prenolepis) illuminates the biogeography and drivers of insect diversification in the Indo-Pacific">An ant genus-group (Prenolepis) illuminates the biogeography and drivers of insect diversification in the Indo-Pacific</h3></a><div class="article-source ellipsis">2018, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790316303414/pdfft?md5=f8659a4c030b0b6c134abe3f38527848&amp;pid=1-s2.0-S1055790316303414-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article2-title" aria-controls="citing-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article2" aria-hidden="true"></div></li></ul><a class="anchor side-panel-view-more" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-84876101153&amp;md5=ab4b26a743d221ba3bb349d336c440" target="_blank"><span class="anchor-text">View more articles</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></div></div></section><section class="SidePanel hidden"><header id="metrics-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Article Metrics</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><a href="https://plu.mx/plum/a/?doi=10.1016/j.ympev.2013.02.023" class="plumx-summary plum-sciencedirect-theme" data-pass-hidden-categories="true" data-hide-usage="true" data-orientation="vertical" data-hide-print="true" data-site="plum" data-on-success="onMetricsWidgetSuccess">View article metrics</a></div></section></aside></div></div><div></div></div><div id="footer"><div class="hor-line" style="border-color:#e9711c"></div><div class="panel-s u-padding-l-bottom u-bg-white u-clr-grey7" role="contentinfo"><a class="anchor move-left els-footer-elsevier anchor-has-inherit-color" href="https://www.elsevier.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg viewBox="-3345 3440.027 140.01 24.333" style="width:104px;height:30px"><title>Elsevier</title><path id="E" style="fill:#E9711C" d="M-3343.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3344L-3343.999,3461.698"></path><path style="fill:#E9711C" d="M-3325.448,3461.698c2.074-0.118,2.83-0.502,2.83-2.832v-13.511c0-2.33-0.757-2.715-2.83-2.832v-0.591h8.884 v0.591c-2.243,0-3.027,0.472-3.027,2.891v13.009c0,1.652,0.056,2.625,1.71,2.625h4.008c3,0,4.4-2,5.576-4.1l0.673,0.118 l-1.71,5.222h-16.114V3461.698"></path><path style="fill:#E9711C" d="M-3307.122,3456.27h0.561c1.12,2.596,2.886,5.4,5.94,5.4c2,0,3.672-1.3,3.672-3.334 c0-1.652-1.626-3.1-4.176-4.927c-3.28-2.36-5.41-3.746-5.41-6.43c0-3.422,2.55-5.517,5.633-5.517c2.214,0,3,0.944,4.204,0.944 c0.476,0,0.56-0.266,0.476-0.737h0.561l0.645,6.076h-0.561c-0.785-2.625-2.523-5.19-5.354-5.19c-1.737,0-3.138,1.356-3.138,3.185 c0,1.918,1.933,3.157,5.016,5.016c2.523,1.504,5,3.362,5,6.254c0,3.245-2.608,5.752-5.97,5.752c-2.02,0-4.148-0.855-4.68-0.855 c-0.336,0-0.7,0.207-0.756,0.649h-0.56l-1.094-6.282"></path><path style="fill:#E9711C" d="M-3293.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3294L-3293.999,3461.698"></path><path style="fill:#E9711C" d="M-3265.839,3462.524h-0.42l-5.41-12.538c-0.896-2.065-1.71-4.16-2.83-6.136 c-0.478-0.826-1.346-1.327-2.3-1.327v-0.591h8.323v0.591c-0.785,0-2.354,0-2.354,1.15c0,0.384,0.87,2.45,1.653,4.308l4.063,9.676 l4.877-11.359c0.588-1.356,0.757-2.094,0.757-2.713c0-0.618-0.673-0.974-2.13-1.062v-0.59h5.941v0.591 c-0.337,0.06-0.7,0.117-1.037,0.295c-1.066,0.56-2.13,3.57-2.635,4.749l-6.5,14.957"></path><path style="fill:#E9711C" d="M-3255.472,3461.698c2.24,0,3.025-0.473,3.025-2.892v-13.393c0-2.42-0.784-2.89-3.025-2.891v-0.59h9.078v0.591 c-2.24,0-3.025,0.472-3.025,2.891v13.393c0,2.42,0.784,2.892,3.025,2.892v0.59h-9.08v-0.59"></path><path id="E_2_" style="fill:#E9711C" d="M-3244.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3245L-3244.999,3461.698"></path><path style="fill:#E9711C" d="M-3206,3461.698c-1.26-0.354-1.71-0.68-2.466-1.623l-6.166-7.609c3.027-0.65,5.13-2.185,5.13-5.547 c0-4.75-4.26-4.986-7.764-4.986h-9.191v0.591c2.24,0,3.026,0.472,3.026,2.891v13.393c0,2.42-0.785,2.892-3.026,2.892v0.59h9.08 v-0.59c-2.242,0-3.027-0.473-3.027-2.892v-5.604h2.551l7.314,9.086h4.54L-3206,3461.698 M-3220.399,3444.499 c0-1.387,0.337-1.476,2.186-1.476c2.774,0,5.3,0.974,5.3,4.308c0,3.6-2.887,4.63-5.914,4.631h-1.569v-7.463H-3220.399z"></path></svg></span></a><div class="panel-s u-bg-white u-padding-0-hor-from-xs u-padding-s-hor-from-md u-padding-xs-ver text-xs u-clear-both-from-xs u-clear-none-from-md"><p class="u-margin-xs-bottom"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/solutions/sciencedirect" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">About ScienceDirect</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science/activateaccess" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Remote access</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science?_ob=ShoppingCartURL&amp;_method=display&amp;md5=3ff44acb300f01481824c54a2973d019" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Shopping cart</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Contact and support</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Terms and conditions</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/privacy-policy" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Privacy policy</span></a></p><p id="els-footer-cookie-message">Cookies are used by this site. For more information, visit the<!-- --> <a class="anchor u-margin-0-right" href="https://www.elsevier.com/solutions/sciencedirect/support/cookies" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">cookies page</span></a>.</p><p id="els-footer-copyright">Copyright © <!-- -->2018<!-- --> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</p></div><div><span style="margin-top:-10px" class="u-position-relative move-bottom u-float-left-from-xs u-float-right-from-md move-right u-padding-0-hor"><a class="anchor els-footer-relx anchor-has-inherit-color" href="https://www.relx.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg style="width:118px;height:28px" viewBox="-3248 3424.938 334.44 53.422"><title>RELX Group</title><path style="fill:#E9711C" d="M-3227.285,3448.329c10.948,0,24.15-2.576,24.15-13.124c0-7.474-7.63-10.267-14.6-10.267 c-10.647,0-30.265,5.926-30.265,26.96c0,10.555,8.034,16.643,20.778,16.643c13.474,0,23.266-8.768,26.26-20.535 c-7.912,13.648-17.972,17.896-26.18,17.896c-10.19,0-14.077-7.568-14.077-14.05c0-18.56,14.406-24.798,23.494-24.798 c6.135,0,9.727,3.936,9.727,8.253c0,11.918-17.17,11.578-25.008,11.578c-0.803,0-3.274,0.02-3.76-0.003l-1.654,1.714 c3.59,0.508,8.554,1.497,13.944,4.223c11.204,5.656,21.745,20.59,35.7,20.59c5.062,0,6.38-1.543,7.335-2.893 c-6.834,4.354-18.286-3.197-24.46-9.04c-5.082-4.806-9.57-10.26-21.387-13.146"></path><g><path style="fill:#666666" d="M-3158.73,3468.502c-0.423,0-0.636-0.164-0.794-0.547l-6.885-14.511h-6.197c-0.21,0-0.316,0.11-0.316,0.33 v14.182c0,0.328-0.213,0.547-0.532,0.547h-5.189c-0.32,0-0.53-0.22-0.53-0.547v-36.139c0-0.33,0.21-0.55,0.53-0.549h13.928 c6.62,0,11.492,4.545,11.492,11.117c0,4.873-2.7,8.65-6.833,10.238l7.575,15.167c0.21,0.385,0,0.713-0.37,0.713L-3158.73,3468.502 L-3158.73,3468.502z M-3159.42,3442.384c0-3.232-2.223-5.31-5.506-5.311h-7.68c-0.21,0-0.316,0.108-0.316,0.328v9.912 c0,0.217,0.104,0.328,0.316,0.328h7.68C-3161.644,3447.641-3159.42,3445.561-3159.42,3442.384"></path><path style="fill:#666666" d="M-3146.95,3431.817c0-0.33,0.21-0.55,0.53-0.549h21.865c0.317,0,0.53,0.22,0.53,0.549v4.709 c0,0.33-0.212,0.548-0.53,0.548h-15.827c-0.212,0-0.318,0.108-0.318,0.328v9.089c0,0.22,0.106,0.328,0.318,0.328h12.726 c0.317,0,0.53,0.22,0.53,0.547v4.709c0,0.33-0.213,0.548-0.53,0.548h-12.726c-0.212,0-0.318,0.11-0.318,0.329v9.418 c0,0.22,0.106,0.328,0.318,0.328h15.827c0.317,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.212,0.547-0.53,0.547h-21.865 c-0.32,0-0.53-0.22-0.53-0.547C-3146.95,3467.956-3146.95,3431.817-3146.95,3431.817z"></path><path style="fill:#666666" d="M-3118.2,3431.817c0-0.33,0.212-0.55,0.53-0.549h5.191c0.317,0,0.53,0.22,0.53,0.549v30.553 c0,0.22,0.106,0.328,0.317,0.328h14.722c0.318,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.213,0.547-0.53,0.547h-20.76 c-0.317,0-0.53-0.22-0.53-0.547C-3118.2,3467.956-3118.2,3431.817-3118.2,3431.817z"></path><path style="fill:#666666" d="M-3052.46,3449.885c0-6.46,0.423-8.926,1.006-10.787c1.747-5.53,5.878-8.377,11.758-8.377 c5.88,0,9.532,3.012,11.28,6.9c0.107,0.274,0.107,0.548-0.21,0.712l-1.695,0.876c-0.265,0.11-0.53,0.055-0.688-0.22 c-1.802-3.396-4.45-5.312-8.74-5.312c-4.45,0-7.468,2.192-8.792,6.3c-0.475,1.422-0.845,3.722-0.845,9.909s0.37,8.486,0.845,9.91 c1.324,4.107,4.343,6.3,8.792,6.3c4.342,0,7.468-2.137,8.79-6.19c0.477-1.422,0.85-3.56,0.85-7.117c0-0.22-0.106-0.328-0.318-0.328 h-8.155c-0.318,0-0.53-0.22-0.53-0.548v-1.807c0-0.33,0.212-0.548,0.53-0.548h10.962c0.317,0,0.53,0.22,0.53,0.548v2.243 c0,3.725-0.423,6.572-0.954,8.27c-1.694,5.53-5.88,8.432-11.65,8.432c-5.88,0-10.01-2.848-11.758-8.38 C-3052.036,3458.811-3052.459,3456.345-3052.46,3449.885"></path><path style="fill:#666666" d="M-3021.43,3468.502c-0.317,0-0.53-0.22-0.53-0.547v-25.023c0-0.33,0.213-0.548,0.53-0.548h1.801 c0.316,0,0.53,0.22,0.53,0.548v3.338h0.052c0.953-2.572,3.23-4.434,6.78-4.434c2.013,0,3.866,0.712,5.084,1.807 c0.265,0.164,0.317,0.438,0.107,0.713l-1.06,1.531c-0.212,0.274-0.48,0.274-0.795,0.11c-1.164-0.767-2.436-1.313-4.024-1.313 c-4.45,0-6.144,3.996-6.144,8.815v14.456c0,0.328-0.213,0.547-0.53,0.547H-3021.43L-3021.43,3468.502z"></path><path style="fill:#666666" d="M-3004.54,3462.26c-0.53-1.752-0.847-3.668-0.847-6.844c0-3.12,0.316-5.037,0.847-6.79 c1.378-4.327,4.818-6.79,9.426-6.79c4.662,0,8.104,2.464,9.48,6.79c0.53,1.752,0.848,3.668,0.848,6.79 c0,3.176-0.317,5.092-0.848,6.844c-1.377,4.326-4.818,6.79-9.48,6.79C-2999.722,3469.05-3003.162,3466.585-3004.54,3462.26 M-2988.387,3461.33c0.475-1.48,0.688-3.066,0.688-5.914c0-2.794-0.213-4.38-0.688-5.86c-1.007-3.12-3.442-4.873-6.728-4.873 c-3.23,0-5.666,1.752-6.672,4.873c-0.477,1.48-0.688,3.065-0.688,5.86c0,2.848,0.212,4.436,0.688,5.914 c1.006,3.12,3.442,4.873,6.672,4.873C-2991.829,3466.203-2989.395,3464.451-2988.387,3461.33"></path><path style="fill:#666666" d="M-2963.66,3468.502c-0.317,0-0.528-0.22-0.528-0.547v-3.066h-0.053c-1.272,2.574-3.92,4.162-7.36,4.162 c-5.615,0-8.74-3.616-8.74-9.913v-16.206c0-0.33,0.213-0.548,0.53-0.548h1.801c0.317,0,0.53,0.22,0.53,0.548v15.44 c0,5.257,2.118,7.83,6.515,7.83c3.812,0,6.78-2.74,6.78-7.284v-15.987c0-0.33,0.21-0.548,0.528-0.548h1.802 c0.317,0,0.53,0.22,0.53,0.548v25.023c0,0.328-0.213,0.547-0.53,0.547h-1.802L-2963.66,3468.502z"></path><path style="fill:#666666" d="M-2955.29,3478.36c-0.317,0-0.53-0.22-0.53-0.549v-34.879c0-0.33,0.213-0.548,0.53-0.548h1.8 c0.317,0,0.53,0.22,0.53,0.548v3.176h0.053c1.218-2.465,3.496-4.272,7.68-4.272c4.45,0,7.256,2.08,8.58,6.242 c0.638,2.137,0.9,4.435,0.9,7.393c0,2.9-0.264,5.2-0.9,7.336c-1.323,4.164-4.13,6.242-8.58,6.242c-4.183,0-6.46-1.807-7.68-4.27 h-0.053v13.031c0,0.33-0.213,0.55-0.53,0.549L-2955.29,3478.36L-2955.29,3478.36z M-2939.35,3461.602 c0.53-1.643,0.688-3.832,0.688-6.13c0-2.355-0.158-4.545-0.688-6.188c-0.952-2.957-3.124-4.6-6.46-4.6 c-3.178,0-5.507,1.533-6.46,4.6c-0.477,1.423-0.69,3.34-0.69,6.188c0,2.847,0.212,4.71,0.69,6.13 c0.954,3.067,3.283,4.602,6.46,4.602C-2942.473,3466.204-2940.3,3464.561-2939.35,3461.602"></path><path style="fill:#666666" d="M-3070.48,3468.545c-0.424,0-0.634-0.164-0.85-0.547l-7.254-12.649h-0.106l-7.307,12.649 c-0.213,0.383-0.424,0.547-0.85,0.547h-5.72c-0.37,0-0.528-0.328-0.318-0.711l10.76-18.562l-9.96-17.248 c-0.21-0.385-0.05-0.712,0.32-0.712h5.721c0.424,0,0.636,0.163,0.847,0.548l6.513,11.278h0.106l6.513-11.278 c0.213-0.385,0.425-0.548,0.85-0.548h5.72c0.37,0,0.53,0.327,0.317,0.712l-9.956,17.248l10.75,18.562 c0.21,0.383,0.052,0.71-0.32,0.711h-5.771H-3070.48z"></path><path style="fill:#666666" d="M-2932.08,3443.951c-0.108,0-0.18-0.075-0.18-0.189v-10.385c0-0.075-0.036-0.113-0.11-0.113h-3.202 c-0.11,0-0.184-0.073-0.184-0.186v-1.6c0-0.11,0.074-0.186,0.184-0.186h8.746c0.11,0,0.183,0.075,0.183,0.186v1.6 c0,0.113-0.07,0.186-0.183,0.186h-3.201c-0.073,0-0.11,0.038-0.11,0.113v10.385c0,0.113-0.07,0.19-0.18,0.189H-2932.08 L-2932.08,3443.951z"></path><path style="fill:#666666" d="M-2924.47,3431.478c0-0.112,0.07-0.186,0.178-0.186h1.604c0.163,0,0.252,0.056,0.307,0.186l3.367,7.8h0.07 l3.315-7.8c0.052-0.13,0.144-0.186,0.305-0.186h1.583c0.11,0,0.182,0.074,0.182,0.186v12.286c0,0.11-0.07,0.186-0.182,0.186h-1.564 c-0.11,0-0.18-0.075-0.18-0.186v-7.744h-0.073l-2.593,5.957c-0.07,0.168-0.18,0.242-0.343,0.242h-1.044 c-0.16,0-0.27-0.074-0.343-0.242l-2.59-5.957h-0.072v7.744c0,0.11-0.072,0.186-0.18,0.186h-1.567c-0.106,0-0.178-0.075-0.178-0.186 v-12.286H-2924.47z"></path></g></svg></span></a></span></div></div></div></section></div></div></div>
<script type="application/json" data-iso-key="_0">{"userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36 Zotero/5.0","abstracts":{"content":[{"#name":"abstract","$":{"xmlns:ce":true,"class":"graphical","lang":"en","id":"aep-abstract-id10"},"$$":[{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id11"},"$$":[{"#name":"section-title","_":"Graphical abstract"},{"#name":"simple-para","$":{"id":"sp0055","view":"all"},"$$":[{"#name":"display","$$":[{"#name":"figure","$":{"id":"f0030"},"$$":[{"#name":"link","$":{"locator":"fx1","id":"aep-link-id12"}}]}]}]}]},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id13"},"$$":[{"#name":"section-title","_":"Highlights"},{"#name":"simple-para","$":{"id":"sp0060","view":"all"},"_":"► A widespread frog species in New Guinea is shown to be a complex of 9 species. ► Biogeographic reconstructions indicate two overwater dispersal events. ► New Guinea could be a focal center for frog biodiversity. ► Evidence for other cryptic frog lineages on the island."}]}]},{"#name":"abstract","$":{"xmlns:ce":true,"class":"author","lang":"en","id":"aep-abstract-id14"},"$$":[{"#name":"section-title","_":"Abstract"},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id15"},"$$":[{"#name":"simple-para","$":{"id":"sp0015","view":"all"},"$$":[{"#name":"__text__","_":"New Guinea is one of five high biodiversity wilderness areas, and frog diversity is exceptionally large, with more than 400 species described to date. The microhylid frog genus "},{"#name":"italic","_":"Mantophryne"},{"#name":"__text__","_":" is endemic to New Guinea and consists of four species, three of which have narrow geographic distributions and a fourth, "},{"#name":"italic","_":"M. lateralis"},{"#name":"__text__","_":", with a broad range that spans the eastern half of the island. Here, we sequence 104 "},{"#name":"italic","_":"Mantophryne"},{"#name":"__text__","_":" samples for three mitochondrial and three nuclear loci to reconstruct the first phylogeny of the genus and to examine spatial patterns of diversity within "},{"#name":"italic","_":"M. lateralis"},{"#name":"__text__","_":". Results indicate that the wide-ranging "},{"#name":"italic","_":"M. lateralis"},{"#name":"__text__","_":" is composed of at least nine geographically separated and well-supported lineages that represent putative species. Biogeographic analysis suggests that "},{"#name":"italic","_":"Mantophryne"},{"#name":"__text__","_":" evolved on the eastern Papuan peninsula with subsequent dispersal westward, as well as overwater dispersal events to the Louisiade and D’Entrecasteaux archipelagos."}]}]}]}],"floats":[],"footnotes":[],"attachments":[{"attachment-eid":"1-s2.0-S1055790313000808-fx1.jpg","ucs-locator":"https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790313000808/fx1/DOWNSAMPLED/image/jpeg/05f58dd2632ecf35d397a9630c0664f5/fx1.jpg","file-basename":"fx1","abstract-attachment":"true","filename":"fx1.jpg","extension":"jpg","filesize":"20454","pixel-height":"200","pixel-width":"284","attachment-type":"IMAGE-DOWNSAMPLED"},{"attachment-eid":"1-s2.0-S1055790313000808-fx1.sml","ucs-locator":"https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790313000808/fx1/THUMBNAIL/image/gif/71ea7382152e8bcf552e3afd050cca69/fx1.sml","file-basename":"fx1","abstract-attachment":"true","filename":"fx1.sml","extension":"sml","filesize":"5753","pixel-height":"154","pixel-width":"219","attachment-type":"IMAGE-THUMBNAIL"}]},"biographies":{},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:ja":true,"xmlns":true,"class":"keyword","id":"aep-keywords-id16"},"$$":[{"#name":"section-title","_":"Keywords"},{"#name":"keyword","$$":[{"#name":"text","_":"Amphibian"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Anura"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Asterophryinae"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Biodiversity"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Microhylidae"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Species tree"}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"experiments":{},"rawtext":"","authors":{"content":[{"#name":"author-group","$":{"xmlns:ce":true},"$$":[{"#name":"author","$":{"id":"aep-author-id5"},"$$":[{"#name":"given-name","_":"Lauren A."},{"#name":"surname","_":"Oliver"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"aep-author-id6"},"$$":[{"#name":"given-name","_":"Eric N."},{"#name":"surname","_":"Rittmeyer"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"aep-author-id7"},"$$":[{"#name":"given-name","_":"Fred"},{"#name":"surname","_":"Kraus"},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"author","$":{"id":"aep-author-id8"},"$$":[{"#name":"given-name","_":"Stephen J."},{"#name":"surname","_":"Richards"},{"#name":"cross-ref","$":{"refid":"aff3"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"aep-author-id9"},"$$":[{"#name":"given-name","_":"Christopher C."},{"#name":"surname","_":"Austin"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"type":"email"},"_":"ccaustin@lsu.edu"}]},{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Department of Biological Sciences, Museum of Natural Science, 119 Foster Hall, Louisiana State University, Baton Rouge, LA 70803, USA"}]},{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Bernice P. Bishop Museum, 1525 Bernice Street, Honolulu, HI 96817, USA"}]},{"#name":"affiliation","$":{"id":"aff3"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Department of Terrestrial Vertebrates, Museum and Art Gallery of the Northern Territory, GPO Box 4646, Darwin, NT 0801, Australia"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author."}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Department of Biological Sciences, Museum of Natural Science, 119 Foster Hall, Louisiana State University, Baton Rouge, LA 70803, USA"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Bernice P. Bishop Museum, 1525 Bernice Street, Honolulu, HI 96817, USA"}]},"aff3":{"#name":"affiliation","$":{"id":"aff3"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Department of Terrestrial Vertebrates, Museum and Art Gallery of the Northern Territory, GPO Box 4646, Darwin, NT 0801, Australia"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"body":{},"exam":{},"article":{"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":119,"openArchiveStatus":false,"openArchiveArticleCount":0,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1055790313X00065-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790313X00065/cover/DOWNSAMPLED200/image/gif/6fcc24e5769bfbfc3ea16f00e098b6ab/cov200h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790313X00065/cover/DOWNSAMPLED200/image/gif/6fcc24e5769bfbfc3ea16f00e098b6ab/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"11347","pixel-height":"200","pixel-width":"149"},{"attachment-eid":"1-s2.0-S1055790313X00065-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790313X00065/cover/DOWNSAMPLED/image/gif/02ff5238f8926f8d92e4d5388032d3c2/cov150h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790313X00065/cover/DOWNSAMPLED/image/gif/02ff5238f8926f8d92e4d5388032d3c2/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"7297","pixel-height":"150","pixel-width":"112"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S10557903.gif","sourceOpenAccess":false,"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1055790313X00065-cov150h.gif"},"pii":"S1055790313000808","dates":{"Available online":"14 March 2013","Received":"4 October 2012","Revised":["21 February 2013"],"Accepted":"24 February 2013","Publication date":"1 June 2013"},"access":{},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"analyticsMetadata":{"accountId":"262591","accountName":"International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"12289512"},"cid":"272322","content-family":"serial","copyright-line":"Copyright © 2013 Elsevier Inc. All rights reserved.","cover-date-years":["2013"],"cover-date-start":"2013-06-01","cover-date-text":"June 2013","document-subtype":"fla","document-type":"article","entitledToken":"FAE68D3AA35B2C4B271656EFF72EAC3254EB92E1F309225A7C486A5B0404FDD9F49A9357A1850B65","eid":"1-s2.0-S1055790313000808","doi":"10.1016/j.ympev.2013.02.023","first-fp":"600","hub-eid":"1-s2.0-S1055790313X00065","issuePii":"S1055790313X00065","iss-first":"3","item-weight":"FULL-TEXT","language":"en","last-lp":"607","last-author":{"#name":"last-author","$":{"xmlns:dm":true},"$$":[{"#name":"author","$":{"xmlns:ce":true,"id":"aep-author-id9"},"$$":[{"#name":"given-name","_":"Christopher C."},{"#name":"surname","_":"Austin"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"type":"email"},"_":"ccaustin@lsu.edu"}]}]},"normalized-first-auth-initial":"L","normalized-first-auth-surname":"OLIVER","pages":[{"last-page":"607","first-page":"600"}],"srctitle":"Molecular Phylogenetics and Evolution","timestamp":"2015-05-15T04:50:19.967271-04:00","title":{"content":[{"#name":"title","$":{"xmlns:ce":true},"$$":[{"#name":"__text__","_":"Phylogeny and phylogeography of "},{"#name":"italic","_":"Mantophryne"},{"#name":"__text__","_":" (Anura: Microhylidae) reveals cryptic diversity in New Guinea"}]}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"67","vol-iss-suppl-text":"Volume 67, Issue 3","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":false,"issn":"10557903","issn-primary-formatted":"1055-7903","useEnhancedReader":false,"isCorpReq":false,"pdfDownload":{"linkType":"DOWNLOAD","linkToPdf":"/science/article/pii/S1055790313000808/pdfft?md5=337c3f2b299046c8ae64cfe97b0b9f35&pid=1-s2.0-S1055790313000808-main.pdf","isPdfFullText":false,"fileName":"1-s2.0-S1055790313000808-main.pdf"},"pdfUrlForCrawlers":"https://www.sciencedirect.com/science/article/pii/S1055790313000808/pdfft?md5=337c3f2b299046c8ae64cfe97b0b9f35&pid=1-s2.0-S1055790313000808-main.pdf","indexTag":true,"volRange":"67","issRange":"3","userProfile":{"departmentName":"General Access","accessType":"IPRANGE","accountId":"262591","webUserId":"12289512","accountName":"International Institute of Tropical Agriculture","departmentId":"267018","userType":"NORMAL","hasMultipleOrganizations":false},"entitlementReason":"package","articleEntitlement":{"entitled":true},"aipType":"none","downloadFullIssue":true,"headerConfig":{"helpUrl":"https://service.elsevier.com/app/home/supporthub/sciencedirect/","contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","orgName":"","webUserId":"12289512","libraryBanner":{},"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasMultiOrg":false,"userType":"IPRANGE","allowCart":false},"open-research":{},"titleString":"Phylogeny and phylogeography of Mantophryne (Anura: Microhylidae) reveals cryptic diversity in New Guinea","isAbstract":false,"onAbstractWhitelist":false,"isContentVisible":false,"ajaxLinks":{"citingArticles":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true}},"specialIssueArticles":{},"recommendations":{},"entitledRecommendations":{"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"citingArticles":{},"workspace":{"isOpen":false},"crossMark":{"isOpen":false},"userIdentity":{},"refersTo":{},"referredToBy":{},"downloadIssue":{"isOpen":false,"articles":[],"selected":[]},"references":{},"referenceLinks":{"internal":{},"external":{}},"glossary":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenRecommendations":true,"isOpenCitingArticles":false,"citingArticles":[false,false,false],"recommendations":[false,false,false,false,false,false],"specialIssueArticles":[false,false,false]},"banner":{"expanded":false},"transientError":{"isOpen":false},"pdfDropdown":{},"exportCitation":{"isOpen":false},"tableOfContents":{"showEntitledTocLinks":true},"chapters":{"toc":[],"isLoading":false},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"metrics":{"isOpen":true},"signOut":{"isOpen":false},"issueNavigation":{"previous":{},"next":{}},"tail":{}}</script>
      
      <script src="satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c.js"></script><script src="satellite-565e008964746d4385002642.js"></script>
      <script src="babel-polyfill.js"></script>
      <script src="react.js"></script>
      <script src="react-dom.js"></script>
      <script src="arp.js" async=""></script>
      <!-- begin usabilla live embed code -->
      <script type="text/javascript">
        window.lightningjs||function(c){function g(b,d){d&&(d+=(/\?/.test(d)?"&":"?")+"lv=1");c[b]||function(){var i=window,h=document,j=b,g=h.location.protocol,l="load",k=0;(function(){function b(){a.P(l);a.w=1;c[j]("_load")}c[j]=function(){function m(){m.id=e;return c[j].apply(m,arguments)}var b,e=++k;b=this&&this!=i?this.id||0:0;(a.s=a.s||[]).push([e,b,arguments]);m.then=function(b,c,h){var d=a.fh[e]=a.fh[e]||[],j=a.eh[e]=a.eh[e]||[],f=a.ph[e]=a.ph[e]||[];b&&d.push(b);c&&j.push(c);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"//"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.frameBorder="0";f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
        window.usabilla_live = lightningjs.require("usabilla_live", "https://w.usabilla.com/eb1c14a91932.js");
        var customData = {};

        if(window.pageData && pageData.content && pageData.content[0]) {
          customData.entitlementType = pageData.content[0].entitlementType;
        }
        if(window.pageData && pageData.visitor) {
          customData.accessType = pageData.visitor.accessType;
          customData.accountId = pageData.visitor.accountId;
          customData.loginStatus = pageData.visitor.loginStatus;
        }
        usabilla_live("data", {"custom": customData });
      </script>
      <!-- end usabilla live embed code -->
      <script type="text/x-mathjax-config;executed=true">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="MathJax.js"></script>
      <script async="" src="gpt.js"></script>
    
  <div class="js-react-modal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><script src="widget-summary.js" async=""></script><iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_elsevier_0" name="destination_publishing_iframe_elsevier_0_name" style="display: none; width: 0px; height: 0px;" src="https://elsevier.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS1055790313000808"></iframe></body></html>