<!DOCTYPE html>
<html lang="en"><head><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <!-- Meta -->
    <meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">

    <!-- Stylesheets -->
    
<link href="css.css" type="text/css" rel="stylesheet">
<link href="globalCSS.css" type="text/css" rel="stylesheet">
<link href="cookiePolicyCSS.css" type="text/css" rel="stylesheet">
<link href="chartist.css" type="text/css" rel="stylesheet">

        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-1131979791.css" rel="stylesheet" type="text/css">
    

    <!-- Scripts -->
    
<script async="" src="analytics.js"></script><script src="jquery-2.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [0] 
        };

        App.CurrentSubdomain = 'nar';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/nar/article/32/5/1792/2380623","@type":"ScholarlyArticle","name":"MUSCLE: multiple sequence alignment with high accuracy and high throughput","about":["benchmarking","computers","sequence alignment","software","trees (plant)","Computational Methods"],"datePublished":"2004-03-01","isPartOf":{"@id":"https://academic.oup.com/nar/issue/32/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2004-03-01","isPartOf":{"@id":"https://academic.oup.com/nar","@type":"Periodical","name":"Nucleic Acids Research","issn":["1362-4962"]}},"url":"http://dx.doi.org/10.1093/nar/gkh340","keywords":["benchmarking","computers","sequence alignment","software","trees (plant)"],"inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/nar/article/32/5/1792/2380623","author":[{"name":"Edgar, Robert C.","affiliation":"195 Roque Moraes Drive, Mill Valley, CA 94941, USA","@type":"Person"}],"description":"Abstract.   We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences. Elements of the algorithm include fast distance es","pageStart":"1792","pageEnd":"1797","thumbnailURL":"https://academic.oup.com/UI/app/img/generic-cover.png","headline":"MUSCLE: multiple sequence alignment with high accuracy and high throughput","image":"https://academic.oup.com/UI/app/img/generic-cover.png"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="MUSCLE: multiple sequence alignment with high accuracy and high throughput | Nucleic Acids Research | Oxford Academic">
    <meta property="og:description" content="Abstract.   We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences. Elements of the algorithm include fast distance es">
    <meta property="og:url" content="https://academic.oup.com/nar/article/32/5/1792/2380623">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/Issue/46/18/2/m_cover.gif?Expires=2147483647&amp;Signature=eV~hm087cge5g8xBrr8Ad3gSMmcSfY8S8fYMMxVMyebMOzGEOd-Q8KiPx0XalwBTcA6jZy29jYCCVng~gYiA2bRURSHYiJJmFdS3MB~zxLd3wDRhaCxH2ogC-EC~NIicXdJr6ACFFc~6Slb3Z4F3GPnTUmF1TfmETtWJ9ZljeOrPJCq00DIOHPwcSP5C6wnJSfquTOINmA8u8gutkWbmGOwPFaTa37wgWdMQ3sPlXeaTD7x3QVAvxU1cvpttJcvurNH6ZD1KkAoWB3fbxyqXHIEHlDf-ZLeY2k321uAAnOvcvm5UKznRhGSheuGJ5tbEO3eOOn2s4QmfHPn3G8VWYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2004-03-01">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Edgar, Robert C."><meta name="citation_author_institution" content="195 Roque Moraes Drive, Mill Valley, CA 94941, USA"><meta name="citation_title" content="MUSCLE: multiple sequence alignment with high accuracy and high throughput"><meta name="citation_firstpage" content="1792"><meta name="citation_lastpage" content="1797"><meta name="citation_doi" content="10.1093/nar/gkh340"><meta name="citation_keyword" content="sequence alignment"><meta name="citation_keyword" content="software"><meta name="citation_journal_title" content="Nucleic Acids Research"><meta name="citation_journal_abbrev" content="Nucleic Acids Res"><meta name="citation_volume" content="32"><meta name="citation_issue" content="5"><meta name="citation_publication_date" content="2004/03/01"><meta name="citation_issn" content="0305-1048"><meta name="citation_publisher" content="Oxford University Press"><meta name="citation_reference" content="J. Comput. Biol.;  citation_year=1994;  citation_volume=1;  citation_pages=337"><meta name="citation_reference" content="Adv. Math.;  citation_year=1976;  citation_volume=20;  citation_pages=367"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1984;  citation_volume=20;  citation_pages=175"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1987;  citation_volume=25;  citation_pages=351"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=2000;  citation_volume=302;  citation_pages=205"><meta name="citation_reference" content="Pharmacogenomics;  citation_year=2002;  citation_volume=3;  citation_pages=131"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1996;  citation_volume=264;  citation_pages=823"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=3059"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=380"><meta name="citation_reference" content=" citation_year=1983; "><meta name="citation_reference" content=" citation_year=1973; "><meta name="citation_reference" content="Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406"><meta name="citation_reference" content="Bioinformatics;  citation_year=2004; "><meta name="citation_reference" content="CABIOS;  citation_year=1996;  citation_volume=12;  citation_pages=327"><meta name="citation_reference" content=" citation_year=2001; "><meta name="citation_reference" content="Mol. Biol. Evol.;  citation_year=2002;  citation_volume=19;  citation_pages=8"><meta name="citation_reference" content="Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=721"><meta name="citation_reference" content="CABIOS;  citation_year=1995;  citation_volume=11;  citation_pages=13"><meta name="citation_reference" content="Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=87"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=323"><meta name="citation_reference" content="Bioinformatics;  citation_year=2004; "><meta name="citation_reference" content="Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=5857"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=231"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=229"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=254"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1995;  citation_volume=247;  citation_pages=536"><meta name="citation_reference" content="Protein Eng.;  citation_year=1995;  citation_volume=8;  citation_pages=647"><meta name="citation_reference" content="Protein Eng.;  citation_year=1998;  citation_volume=11;  citation_pages=739"><meta name="citation_reference" content="Proteins;  citation_year=1999;  citation_volume=3;  citation_pages=209"><meta name="citation_reference" content="J. Comput. Biol.;  citation_year=2000;  citation_volume=7;  citation_pages=685"><meta name="citation_reference" content="Proteins;  citation_year=2000;  citation_volume=40;  citation_pages=6"><meta name="citation_reference" content="Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=494"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=2003;  citation_volume=326;  citation_pages=317"><meta name="citation_reference" content="Bioinformatics;  citation_year=2004; "><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=316"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=2994"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=34"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=2682"><meta name="citation_reference" content="Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=I215"><meta name="citation_reference" content="J. Am. Stat. Assoc.;  citation_year=1937;  citation_volume=32;  citation_pages=675"><meta name="citation_reference" content="Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=157"><meta name="citation_reference" content="Proc. Int. Conf. Intell. Syst. Mol. Biol.;  citation_year=1998;  citation_volume=6;  citation_pages=165"><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/nar/article-pdf/32/5/1792/7055030/gkh340.pdf"><meta name="description" content="Abstract.   We describe MUSCLE, a new computer program for creating multiple alignments of protein sequences. Elements of the algorithm include fast distance es">
        <meta name="citation_xml_url" content="https://academic.oup.com/nar/article-xml/32/5/1792/2380623">



    <link rel="canonical" href="https://academic.oup.com/nar/article/32/5/1792/2380623">


        <!-- Start: GPT Async -->

<script async="async" src="gpt.js"></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type="text/javascript">
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "nar/article/32/5/1792/2380623");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=nar&authzrequired=false&account_id=&doi=10.1093/nar/gkh340';
    </script>
<title>
	MUSCLE: multiple sequence alignment with high accuracy and high throughput | Nucleic Acids Research | Oxford Academic
</title><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_266.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-2c47105b6381604898bbf8ae8a680350.css"></head>

<body data-sitename="nucleicacidsresearch" class="off-canvas pg_article" theme-nar="">
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 100});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2380623" id="webform">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbAyPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJFZGdhciwgUm9iZXJ0IEMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMTk1IFJvcXVlIE1vcmFlcyBEcml2ZSwgTWlsbCBWYWxsZXksIENBIDk0OTQxLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik1VU0NMRTogbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHdpdGggaGlnaCBhY2N1cmFjeSBhbmQgaGlnaCB0aHJvdWdocHV0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTc5MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTc5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvbmFyL2draDM0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9rZXl3b3JkIiBjb250ZW50PSJzZXF1ZW5jZSBhbGlnbm1lbnQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fa2V5d29yZCIgY29udGVudD0ic29mdHdhcmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA0LzAzLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjAzMDUtMTA0OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iSi4gQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MzM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQWR2LiBNYXRoLjsgIGNpdGF0aW9uX3llYXI9MTk3NjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTM2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz0yMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJQaGFybWFjb2dlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI2NDsgIGNpdGF0aW9uX3BhZ2VzPTgyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTMwNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0zODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25feWVhcj0xOTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25feWVhcj0xOTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQ0FCSU9TOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTcyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkNBQklPUzsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz04NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9NTg1NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTIzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQ2NzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI0NzsgIGNpdGF0aW9uX3BhZ2VzPTUzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlByb3RlaW4gRW5nLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iUHJvdGVpbiBFbmcuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NzM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iUHJvdGVpbnM7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTIwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTY4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlByb3RlaW5zOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDk0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0zMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MzE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI5OTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI2ODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPUkyMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBBbS4gU3RhdC4gQXNzb2MuOyAgY2l0YXRpb25feWVhcj0xOTM3OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9Njc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJQcm9jLiBJbnQuIENvbmYuIEludGVsbC4gU3lzdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL25hci9hcnRpY2xlLXBkZi8zMi81LzE3OTIvNzA1NTAzMC9na2gzNDAucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBXZSBkZXNjcmliZSBNVVNDTEUsIGEgbmV3IGNvbXB1dGVyIHByb2dyYW0gZm9yIGNyZWF0aW5nIG11bHRpcGxlIGFsaWdubWVudHMgb2YgcHJvdGVpbiBzZXF1ZW5jZXMuIEVsZW1lbnRzIG9mIHRoZSBhbGdvcml0aG0gaW5jbHVkZSBmYXN0IGRpc3RhbmNlIGVzIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yMzgwNjIzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDA5JWVBsKl+8jwKL2FwoJtDZx9d" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5127" type="hidden">
        <input id="hdnAdDelaySeconds" value="3000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="default" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad1_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad1_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad1_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f32%2f5%2f1792%2f2380623" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Nucleic Acids Research</option><option class="header-search-bar-filters-item" value="narese/24">Subject: Computational Methods</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f32%2f5%2f1792%2f2380623" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115593">
            <a href="https://academic.oup.com/nar/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115590">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Section browse</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115594">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDChemical+Biology+and+Nucleic+Acid+Chemistry" class="nav-link ">Chemical Biology and Nucleic Acid Chemistry</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115595">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=COMPUTATIONAL+BIOLOGY" class="nav-link ">Computational Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115596">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Data+Resources+and+Analyses" class="nav-link ">Data Resources and Analyses</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115597">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Gene+Regulation,+Chromatin+and+Epigenetics" class="nav-link ">Gene Regulation, Chromatin and Epigenetics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115598">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genome+integrity,+repair+and+replication" class="nav-link ">Genome Integrity, Repair and Replication</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115599">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genomics" class="nav-link ">Genomics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115600">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=MOLECULAR+BIOLOGY" class="nav-link ">Molecular Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115601">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Nucleic+Acid+Enzymes" class="nav-link ">Nucleic Acid Enzymes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115602">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=RNAANDRNA+and+RNA-protein+complexes" class="nav-link ">RNA and RNA-protein complexes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115603">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=STRUCTURAL+BIOLOGY" class="nav-link ">Structural Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115604">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDSynthetic+Biology+and+Bioengineering" class="nav-link ">Synthetic Biology and Bioengineering</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115605">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Methods+Online" class="nav-link ">Methods Online</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115606">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=SURVEY%20@@@%20SUMMARY" class="nav-link ">Surveys and Summaries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115607">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Database+Issue" class="nav-link ">Database</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115608">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Web+Server+issue" class="nav-link ">Web Server</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115609">
            <a href="https://academic.oup.com/nar/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115591">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115610">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115611">
            <a href="http://mc.manuscriptcentral.com/nar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115612">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115613">
            <a href="https://academic.oup.com/journals/pages/open_access" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115614">
            <a href="https://academic.oup.com/nar/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115592">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115615">
            <a href="https://academic.oup.com/nar/pages/About" class="nav-link ">About Nucleic Acids Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115616">
            <a href="https://academic.oup.com/nar/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115617">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115618">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115619">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115620">
            <a href="https://academic.oup.com/nar/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115621">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115622">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/nar" class=" journal-logo-container">

            <img id="logo-NucleicAcidsResearch" class="journal-logo" src="nar_title248941275.svg" alt="Nucleic Acids Research">

                </a>
            <div class="society-logo-block">
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115593">
            <a href="https://academic.oup.com/nar/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115590">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Section browse</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115594">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDChemical+Biology+and+Nucleic+Acid+Chemistry" class="nav-link ">Chemical Biology and Nucleic Acid Chemistry</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115595">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=COMPUTATIONAL+BIOLOGY" class="nav-link ">Computational Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115596">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Data+Resources+and+Analyses" class="nav-link ">Data Resources and Analyses</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115597">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Gene+Regulation,+Chromatin+and+Epigenetics" class="nav-link ">Gene Regulation, Chromatin and Epigenetics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115598">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genome+integrity,+repair+and+replication" class="nav-link ">Genome Integrity, Repair and Replication</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115599">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genomics" class="nav-link ">Genomics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115600">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=MOLECULAR+BIOLOGY" class="nav-link ">Molecular Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115601">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Nucleic+Acid+Enzymes" class="nav-link ">Nucleic Acid Enzymes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115602">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=RNAANDRNA+and+RNA-protein+complexes" class="nav-link ">RNA and RNA-protein complexes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115603">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=STRUCTURAL+BIOLOGY" class="nav-link ">Structural Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115604">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDSynthetic+Biology+and+Bioengineering" class="nav-link ">Synthetic Biology and Bioengineering</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115605">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Methods+Online" class="nav-link ">Methods Online</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115606">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=SURVEY%20@@@%20SUMMARY" class="nav-link ">Surveys and Summaries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115607">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Database+Issue" class="nav-link ">Database</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115608">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Web+Server+issue" class="nav-link ">Web Server</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115609">
            <a href="https://academic.oup.com/nar/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115591">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115610">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115611">
            <a href="http://mc.manuscriptcentral.com/nar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115612">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115613">
            <a href="https://academic.oup.com/journals/pages/open_access" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115614">
            <a href="https://academic.oup.com/nar/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115592">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115615">
            <a href="https://academic.oup.com/nar/pages/About" class="nav-link ">About Nucleic Acids Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115616">
            <a href="https://academic.oup.com/nar/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115617">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115618">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115619">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115620">
            <a href="https://academic.oup.com/nar/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115621">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115622">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Nucleic Acids Research</option><option class="header-search-bar-filters-item" value="narese/24">Subject: Computational Methods</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/nar/advanced-search?page=1&amp;fl_SiteID=5127&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5127" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=81744" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/nar" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="nar" type="hidden">
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left mobile">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/nar/issue/32/5"><img id="issueFallbackImage" class="fb-featured-image" src="generic-cover.png" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/nar/issue/32/5">
                        <div class="volume trailing-comma">Volume 32</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
1 March 2004        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="38266725">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266725">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="38266728">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266728">INTRODUCTION</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="38266730">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266730">MUSCLE algorithm</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="38266760">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266760">Assessment</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="38266774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266774">RESULTS</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="38266778">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266778">DISCUSSION</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="38266792">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#38266792">References</a>
                </div>
            </li>
    </ul>

    </div>

	                <div class="comments-jumplink">
        <a class="js-jumplink scrollTo" href="#usercomments"><i class="icon-menu_comments"></i>Comments (0)</a>
    </div></div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/nar/article/32/5/1783/2380627">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/nar/article/32/5/1798/2380664">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">MUSCLE: multiple sequence alignment with high accuracy and high throughput
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert C. Edgar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">195 Roque Moraes Drive, Mill Valley, CA 94941, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="COR1" data-open="COR1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
             *Email: <a href="mailto:bob@drive5.com">bob@drive5.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Robert+C.+Edgar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Edgar%20R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Edgar%20R%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robert C. Edgar<i class="icon-general-mail"></i></a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Nucleic Acids Research</em>, Volume 32, Issue 5, 1 March 2004, Pages 1792–1797, <a href="https://doi.org/10.1093/nar/gkh340">https://doi.org/10.1093/nar/gkh340</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 March 2004</div>
                        </div>
                    </div>
                            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2380623" href="https://academic.oup.com/nar/article-pdf/32/5/1792/7055030/gkh340.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Robert C. Edgar;  MUSCLE: multiple sequence alignment with high accuracy and high throughput, <em>Nucleic Acids Research</em>, Volume 32, Issue 5, 1 March 2004, Pages 1792–1797, <a href="https://doi.org/10.1093/nar/gkh340">https://doi.org/10.1093/nar/gkh340</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=ris">RIS (Zotero)</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=ris">EndNote</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=bibtex">BibTex</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=txt">Medlars</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=ris">ProCite</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=txt">RefWorks</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2380623?format=ris">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1362-4962&amp;title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;publicationDate=2004-03-01&amp;volumeNum=32&amp;issueNum=5&amp;author=Edgar%2C%20Robert%20C.&amp;startPage=1792&amp;endPage=1797&amp;contentId=10.1093%2Fnar%2Fgkh340&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Nucleic Acids Research</option><option class="header-search-bar-filters-item" value="narese/24">Subject: Computational Methods</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f32%2f5%2f1792%2f2380623" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Nucleic Acids Research</option><option class="header-search-bar-filters-item" value="narese/24">Subject: Computational Methods</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/nar/advanced-search?page=1&amp;fl_SiteID=5127&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="nar" type="hidden">
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="38266725" id="38266725" class="abstract-title">Abstract</h2>
<section class="abstract"><p> We describe MUSCLE, a new computer program
 for creating multiple alignments of protein sequences. Elements of the 
algorithm include fast distance estimation using <em>k</em> mer 
counting, progressive alignment using a new profile function we call the
 log‐expectation score, and refinement using tree‐dependent restricted 
partitioning. The speed and accuracy of MUSCLE are compared with 
T‐Coffee, MAFFT and CLUSTALW on four test sets of reference alignments: 
BAliBASE, SABmark, SMART and a new benchmark, PREFAB. MUSCLE achieves 
the highest, or joint highest, rank in accuracy on each of these sets. 
Without refinement, MUSCLE achieves average accuracy statistically 
indistinguishable from T‐Coffee and MAFFT, and is the fastest of the 
tested methods for large numbers of sequences, aligning 5000 sequences 
of average length 350 in 7 min on a current desktop computer. The MUSCLE
 program, source code and PREFAB test data are freely available at <a class="link link-uri" href="http://www.drive5.com/muscle" target="">http://www.drive5. com/muscle.</a></p></section>                        <div class="article-metadata-panel clearfix"></div>
<p>Received January 19, 2004; Revised January 30, 2004; Accepted February 24, 2004</p>                            <h2 scrollto-destination="38266728" id="38266728" class="section-title">INTRODUCTION</h2>
<p> Multiple alignments of protein sequences are important in many 
applications, including phylogenetic tree estimation, structure 
prediction and critical residue identification. The most natural 
formulation of the computational problem is to define a model of 
sequence evolution that assigns probabilities to elementary sequence 
edits and seeks a most probable directed graph in which edges represent 
edits and terminal nodes are the observed sequences. No tractable method
 for finding such a graph is known. A heuristic alternative is to seek a
 multiple alignment that optimizes the sum of pairs (SP) score, i.e. the
 sum of pairwise alignment scores. Optimizing the SP score is NP 
complete ( <a reveal-id="GKH340C1" data-open="GKH340C1" class="link link-ref link-reveal xref-default">1</a> ) and can be achieved by dynamic programming with time and space complexity O( <em>L</em><sup><em>N</em></sup> ) in the sequence length <em>L</em> and number of sequences <em>N</em> ( <a reveal-id="GKH340C2" data-open="GKH340C2" class="link link-ref link-reveal xref-default">2</a> ). A more popular strategy is the progressive method ( <a reveal-id="GKH340C3" data-open="GKH340C3" class="link link-ref link-reveal xref-default">3</a> , <a reveal-id="GKH340C4" data-open="GKH340C4" class="link link-ref link-reveal xref-default">4</a>
 ), which first estimates a tree and then constructs a pairwise 
alignment of the subtrees found at each internal node. A subtree is 
represented by its profile, a multiple alignment treated as a sequence 
by regarding each column as an alignable symbol. A variant on this 
strategy is used by T‐Coffee ( <a reveal-id="GKH340C5" data-open="GKH340C5" class="link link-ref link-reveal xref-default">5</a>
 ), which aligns profiles by optimizing a score derived from local and 
global alignments of all pairs of input sequences. Misalignments by 
progressive methods are sometimes readily apparent (Fig. <a reveal-id="GKH340F1" data-open="GKH340F1" class="link link-ref link-reveal xref-default">1</a> ), motivating further processing (refinement). For a recent review of multiple alignment methods, see Notredame ( <a reveal-id="GKH340C6" data-open="GKH340C6" class="link link-ref link-reveal xref-default">6</a>
 ). Here we describe MUSCLE (multiple sequence comparison by 
log‐expectation), a new computer program for multiple protein sequence 
alignment. </p>                            <h2 scrollto-destination="38266730" id="38266730" class="section-title">MUSCLE algorithm</h2>
<p> Here we give an overview of the algorithm; a more detailed 
discussion is given in Edgar (submitted). Following guide tree 
construction, the fundamental step is pairwise profile alignment, which 
is used first for progressive alignment and then for refinement. This is
 similar to the strategies used by PRRP ( <a reveal-id="GKH340C7" data-open="GKH340C7" class="link link-ref link-reveal xref-default">7</a> ) and MAFFT ( <a reveal-id="GKH340C8" data-open="GKH340C8" class="link link-ref link-reveal xref-default">8</a> ). </p>                            <h3 scrollto-destination="38266732" id="38266732" class="section-title">Distance measures and guide tree estimation</h3>
<p> MUSCLE uses two distance measures for a pair of sequences: a <em>k</em> mer distance (for an unaligned pair) and the Kimura distance (for an aligned pair). A <em>k</em> mer is a contiguous subsequence of length <em>k</em> , also known as a word or <em>k</em> ‐tuple. Related sequences tend to have more <em>k</em> mers in common than expected by chance. The <em>k</em> mer distance is derived from the fraction of <em>k</em> mers in common in a compressed alphabet, which we have previously shown to correlate well with fractional identity ( <a reveal-id="GKH340C9" data-open="GKH340C9" class="link link-ref link-reveal xref-default">9</a>
 ). This measure does not require an alignment, giving a significant 
speed advantage. Given an aligned pair of sequences, we compute the 
pairwise identity and convert to an additive distance estimate, applying
 the Kimura correction for multiple substitutions at a single site ( <a reveal-id="GKH340C10" data-open="GKH340C10" class="link link-ref link-reveal xref-default">10</a> ). Distance matrices are clustered using UPGMA ( <a reveal-id="GKH340C11" data-open="GKH340C11" class="link link-ref link-reveal xref-default">11</a> ), which we find to give slightly improved results over neighbor‐joining ( <a reveal-id="GKH340C12" data-open="GKH340C12" class="link link-ref link-reveal xref-default">12</a>
 ), despite the expectation that neighbor‐joining will give a more 
reliable estimate of the evolutionary tree. This can be explained by 
assuming that in progressive alignment, the best accuracy is obtained at
 each node by aligning the two profiles that have fewest differences, 
even if they are not evolutionary neighbors. </p>                            <h3 scrollto-destination="38266734" id="38266734" class="section-title">Profile alignment</h3>
<p> In order to apply pairwise alignment to profiles, a scoring function
 must be defined on an aligned pair of profile positions, i.e. a pair of
 multiple alignment columns [see, for example Edgar and Sjolander ( <a reveal-id="GKH340C13" data-open="GKH340C13" class="link link-ref link-reveal xref-default">13</a> )]. Let <em>i</em> and <em>j</em> be amino acid types, <em>p</em><sub><em>i</em></sub> the background probability of <em>i</em> , <em>p</em><sub><em>ij</em></sub> the joint probability of <em>i</em> and <em>j</em> being aligned to each other, <em>f</em><sup><em>x</em></sup><sub><em>i</em></sub> the observed frequency of <em>i</em> in column <em>x</em> of the first profile, and <em>f</em><sup><em>x</em></sup><sub>G</sub> the observed frequency of gaps in that column at position <em>x</em> in the family (similarly for position <em>y</em> in the second profile). The estimated probability α <sup><em>x</em></sup><sub><em>i</em></sub> of observing amino acid <em>i</em> in position <em>x</em> can be derived from <em>f</em><sup><em>x</em></sup> , typically by adding heuristic pseudo‐counts or by using Bayesian methods such as Dirichlet mixture priors ( <a reveal-id="GKH340C14" data-open="GKH340C14" class="link link-ref link-reveal xref-default">14</a> ). MUSCLE uses a new profile function we call the log‐expectation (LE) score: </p><p> LE <sup><em>xy</em></sup> = (1 – <em>f</em><sup><em>x</em></sup><sub>G</sub> ) (1 – <em>f</em><sup><em>y</em></sup><sub>G</sub> ) log Σ <sub><em>i</em></sub> Σ <sub><em>j</em></sub><em>f</em><sup><em>x</em></sup><sub><em>i</em></sub><em>f</em><sup><em>y</em></sup><sub><em>j</em></sub><em>p</em><sub><em>ij</em></sub> / <em>p</em><sub><em>i</em></sub><em>p</em><sub><em>j</em></sub><strong>1</strong></p><p> This is a modified version of the log‐average function ( <a reveal-id="GKH340C15" data-open="GKH340C15" class="link link-ref link-reveal xref-default">15</a> ): </p><p> LA <sup><em>xy</em></sup> = log Σ <sub><em>i</em></sub> Σ <sub><em>j</em></sub><em>α</em><sup><em>x</em></sup><sub><em>i</em></sub> α <sup><em>y</em></sup><sub><em>j</em></sub><em>p</em><sub><em>ij</em></sub> / <em>p</em><sub><em>i</em></sub><em>p</em><sub><em>j</em></sub><strong>2</strong></p><p> MUSCLE uses probabilities <em>p</em><sub><em>i</em></sub> and <em>p</em><sub><em>ij</em></sub> derived from the 240 PAM VTML matrix ( <a reveal-id="GKH340C16" data-open="GKH340C16" class="link link-ref link-reveal xref-default">16</a> ). Frequencies <em>f</em><sub><em>i</em></sub>
 are normalized to sum to 1 when indels are present (otherwise the 
logarithm becomes increasingly negative with increasing numbers of gaps 
even when aligning conserved or similar residues). The factor (1 – <em>f</em><sub>G</sub>
 ) is the occupancy of a column, introduced to encourage more highly 
occupied columns to align. Position‐specific gap penalties are used, 
employing heuristics similar to those found in MAFFT and LAGAN ( <a reveal-id="GKH340C17" data-open="GKH340C17" class="link link-ref link-reveal xref-default">17</a> ). </p>                            <h3 scrollto-destination="38266740" id="38266740" class="section-title">Algorithm</h3>
<p> The high‐level flow is depicted in Figure <a reveal-id="GKH340F2" data-open="GKH340F2" class="link link-ref link-reveal xref-default">2</a> . </p><p></p><p><em>Stage 1, Draft progressive</em> . The goal of the first stage is to produce a multiple alignment, emphasizing speed over accuracy. </p><p> 1.1 The <em>k</em> mer distance is computed for each pair of input sequences, giving distance matrix D1. </p><p>1.2 Matrix D1 is clustered by UPGMA, producing binary tree TREE1.</p><p>1.3
 A progressive alignment is constructed by following the branching order
 of TREE1. At each leaf, a profile is constructed from an input 
sequence. Nodes in the tree are visited in prefix order (children before
 their parent). At each internal node, a pairwise alignment is 
constructed of the two child profiles, giving a new profile which is 
assigned to that node. This produces a multiple alignment of all input 
sequences, MSA1, at the root.</p><p></p><p><em>Stage 2, Improved progressive</em> . The main source of error in the draft progressive stage is the approximate <em>k</em>
 mer distance measure, which results in a suboptimal tree. MUSCLE 
therefore re‐estimates the tree using the Kimura distance, which is more
 accurate but requires an alignment. </p><p>2.1 The Kimura distance for each pair of input sequences is computed from MSA1, giving distance matrix D2.</p><p>2.2 Matrix D2 is clustered by UPGMA, producing binary tree TREE2.</p><p>2.3
 A progressive alignment is produced following TREE2 (similar to 1.3), 
producing multiple alignment MSA2. This is optimized by computing 
alignments only for subtrees whose branching orders changed relative to 
TREE1.</p><p></p><p><em>Stage 3, Refinement.</em></p><p>3.1 An edge is chosen from TREE2 (edges are visited in order of decreasing distance from the root).</p><p>3.2 TREE2 is divided into two subtrees by deleting the edge. The profile of the multiple alignment in each subtree is computed.</p><p>3.3 A new multiple alignment is produced by re‐aligning the two profiles.</p><p>3.4 If the SP score is improved, the new alignment is kept, otherwise it is discarded.</p><p>
 Steps 3.1–3.4 are repeated until convergence or until a user‐defined 
limit is reached. This is a variant of tree‐dependent restricted 
partitioning ( <a reveal-id="GKH340C18" data-open="GKH340C18" class="link link-ref link-reveal xref-default">18</a> ). </p><p>
 Complete multiple alignments are available at steps 1.3, 2.3 and 3.4, 
at which points the algorithm may be terminated. We refer to the first 
two stages alone as MUSCLE‐p, which produces MSA2. MUSCLE‐p has time 
complexity O( <em>N</em><sup>2</sup><em>L</em> + <em>NL</em><sup>2</sup> ) and space complexity O( <em>N</em><sup>2</sup> + <em>NL + L</em><sup>2</sup> ). Refinement adds an O( <em>N</em><sup>3</sup><em>L</em> ) term to the time complexity. </p>                            <h2 scrollto-destination="38266760" id="38266760" class="section-title">Assessment</h2>
<p> We assessed the performance of MUSCLE on four sets of reference alignments: BAliBASE ( <a reveal-id="GKH340C19" data-open="GKH340C19" class="link link-ref link-reveal xref-default">19</a> , <a reveal-id="GKH340C20" data-open="GKH340C20" class="link link-ref link-reveal xref-default">20</a> ), SABmark ( <a reveal-id="GKH340C21" data-open="GKH340C21" class="link link-ref link-reveal xref-default">21</a> ), SMART ( <a reveal-id="GKH340C22" data-open="GKH340C22" class="link link-ref link-reveal xref-default">22</a> – <a reveal-id="GKH340C24" data-open="GKH340C24" class="link link-ref link-reveal xref-default">24</a> ) and a new benchmark, PREFAB. We compared these with four other methods: CLUSTALW ( <a reveal-id="GKH340C25" data-open="GKH340C25" class="link link-ref link-reveal xref-default">25</a>
 ), probably the most widely used program at the time of writing; 
T‐Coffee, which has the best BAliBASE score reported to date; and two 
MAFFT scripts: FFTNS1, the fastest previously published method known to 
the author (in which diagonal finding by fast Fourier transform is 
enabled and a progressive alignment constructed), and NWNSI, the slowest
 but most accurate of the MAFFT methods (in which fast Fourier transform
 is disabled and refinement is enabled). Tested versions were MUSCLE 
3.2, CLUSTALW 1.82, T‐Coffee 1.37 and MAFFT 3.82. We also evaluated 
MUSCLE‐p, in which the refinement stage is omitted. We also tried 
Align‐m 1.0 ( <a reveal-id="GKH340C21" data-open="GKH340C21" class="link link-ref link-reveal xref-default">21</a>
 ), but found in many cases that the program either aborted or was 
impractically slow on the larger alignments found in SMART and PREFAB. </p><p></p><p><em>BAliBASE</em>
 . We used version 2 of the BAliBASE benchmark, reference sets Ref 1–Ref
 5. Other reference sets contain repeats, inversions and transmembrane 
helices, for which none of the tested algorithms is designed. </p><p></p><p><em>SABmark.</em>
 We used version 1.63 of the SABmark reference alignments, which 
consists of two subsets: Superfamily and Twilight. All sequences have 
known structure. The Twilight set contains 1994 domains from the Astral 
database ( <a reveal-id="GKH340C26" data-open="GKH340C26" class="link link-ref link-reveal xref-default">26</a> ) with pairwise sequence similarity e‐values ≤1, divided into 236 folds according to the SCOP classification ( <a reveal-id="GKH340C27" data-open="GKH340C27" class="link link-ref link-reveal xref-default">27</a>
 ). The Superfamily set contains sequences of pairwise identity ≤50%, 
divided into 462 SCOP superfamilies. Each pair of structures was aligned
 with two structural aligners: SOFI ( <a reveal-id="GKH340C28" data-open="GKH340C28" class="link link-ref link-reveal xref-default">28</a> ) and CE ( <a reveal-id="GKH340C29" data-open="GKH340C29" class="link link-ref link-reveal xref-default">29</a>
 ), producing a sequence alignment from the consensus in which only 
high‐confidence regions are retained. Input sets range from three to 25 
sequences, with an average of eight and an average sequence length of 
179. </p><p></p><p><em>SMART</em> . SMART contains multiple alignments 
refined by experts, focusing primarily on signaling domains. While 
structures were considered where known, sequence methods were also used 
to aid construction of the database, so SMART is not suitable as a 
definitive benchmark. However, conventional wisdom [e.g. Fischer <em>et al.</em> ( <a reveal-id="GKH340C30" data-open="GKH340C30" class="link link-ref link-reveal xref-default">30</a>
 )] holds that machine‐assisted experts can produce superior alignments 
to automated methods, so performance on this set is of interest for 
comparison. We used a version of SMART downloaded in July 2000, before 
the first version of MUSCLE was made available; eliminating the 
possibility that MUSCLE was used to aid construction. We discarded 
alignments of more than 100 sequences in order to make the test 
tractable for T‐Coffee, leaving 267 alignments averaging 31 sequences of
 length 175. </p><p></p><p><em>PREFAB</em> . The methods used to create 
databases such as BAliBASE and SMART are time‐consuming and demand 
significant expertise, making a fully automated protocol desirable. 
Perhaps the most obvious approach is to generate sequence alignments 
from automated alignments of multiple structures, but this is fraught 
with difficulties; see for example Eidhammer <em>et al.</em> ( <a reveal-id="GKH340C31" data-open="GKH340C31" class="link link-ref link-reveal xref-default">31</a>
 ). With this in mind, we constructed a new test set, PREFAB (protein 
reference alignment benchmark) which exploits methodology ( <a reveal-id="GKH340C21" data-open="GKH340C21" class="link link-ref link-reveal xref-default">21</a> , <a reveal-id="GKH340C32" data-open="GKH340C32" class="link link-ref link-reveal xref-default">32</a> , <a reveal-id="GKH340C33" data-open="GKH340C33" class="link link-ref link-reveal xref-default">33</a> ), test data ( <a reveal-id="GKH340C13" data-open="GKH340C13" class="link link-ref link-reveal xref-default">13</a> , <a reveal-id="GKH340C34" data-open="GKH340C34" class="link link-ref link-reveal xref-default">34</a> , <a reveal-id="GKH340C35" data-open="GKH340C35" class="link link-ref link-reveal xref-default">35</a> ) and statistical methods ( <a reveal-id="GKH340C19" data-open="GKH340C19" class="link link-ref link-reveal xref-default">19</a>
 ) that have previously been applied to alignment accuracy assessment. 
The protocol is as follows. Two proteins are aligned by a structural 
method that does not incorporate sequence similarity. Each sequence is 
used to query a database, from which high‐scoring hits are collected. 
The queries and their hits are combined and aligned by a multiple 
sequence method. Accuracy is assessed on the original pair alone, by 
comparison with their structural alignment. Three test sets selected 
from the FSSP database ( <a reveal-id="GKH340C36" data-open="GKH340C36" class="link link-ref link-reveal xref-default">36</a> ) were used as described in Sadreyev and Grishin ( <a reveal-id="GKH340C34" data-open="GKH340C34" class="link link-ref link-reveal xref-default">34</a> ) (data kindly provided by Ruslan Sadreyev), and Edgar and Sjolander ( <a reveal-id="GKH340C13" data-open="GKH340C13" class="link link-ref link-reveal xref-default">13</a> , <a reveal-id="GKH340C35" data-open="GKH340C35" class="link link-ref link-reveal xref-default">35</a>
 ), which we call SG, PP1 and PP2, respectively. These three sets vary 
mainly in their selection criteria. PP1 and PP2 contain pairs with 
sequence identity ≤30%. PP1 was designed to select pairs that have high 
structural similarity, requiring a <em>z</em> ‐score of ≥15 and a root mean square deviation (r.m.s.d.) of ≤2.5 Å. PP2 selected more diverged pairs with a <em>z</em>
 ‐score of ≥8 and ≤12, and an r.m.s.d. of ≤3.5 Å. SG contains pairs 
sampled from three ranges of sequence identity: 0–15, 15–30 and 30–97%, 
with no <em>z</em> ‐score or r.m.s.d. limits. We re‐aligned each pair of structures using the CE aligner ( <a reveal-id="GKH340C29" data-open="GKH340C29" class="link link-ref link-reveal xref-default">29</a>
 ), and retained only those pairs for which FSSP and CE agreed on 50 or 
more positions. This was designed to minimize questionable and ambiguous
 structural alignments as done in SABmark and MaxBench ( <a reveal-id="GKH340C33" data-open="GKH340C33" class="link link-ref link-reveal xref-default">33</a> ). We used the full‐chain sequence of each structure to make a PSI‐BLAST ( <a reveal-id="GKH340C37" data-open="GKH340C37" class="link link-ref link-reveal xref-default">37</a> , <a reveal-id="GKH340C38" data-open="GKH340C38" class="link link-ref link-reveal xref-default">38</a> ) search of the NCBI non‐redundant protein sequence database ( <a reveal-id="GKH340C39" data-open="GKH340C39" class="link link-ref link-reveal xref-default">39</a>
 ), keeping locally aligned regions of hits with e‐values below 0.01. 
Hits were filtered to 80% maximum identity (including the query), and 24
 selected at random. Finally, each pair of structures and their 
remaining hits were combined to make sets of ≤50 sequences. The limit of
 50 was arbitrarily chosen to make the test tractable on a desktop 
computer for some of the more resource‐intensive methods, in particular 
T‐Coffee (which needed 10 CPU days, as noted in Table <a reveal-id="GKH340TB4" data-open="GKH340TB4" class="link link-ref link-reveal xref-default">4</a>
 ). The final set, PREFAB version 3.0, has 1932 alignments averaging 49 
sequences of length 240, of which 178 positions in the structure pair 
are found in the consensus of FSSP and CE. </p>                            <h3 scrollto-destination="38266770" id="38266770" class="section-title">Accuracy measurement</h3>
<p> We used three accuracy measures: <em>Q</em> , TC and APDB. <em>Q</em>
 (quality) is the number of correctly aligned residue pairs divided by 
the number of residue pairs in the reference alignment. This has 
previously been termed the developer score ( <a reveal-id="GKH340C32" data-open="GKH340C32" class="link link-ref link-reveal xref-default">32</a> ) and SPS ( <a reveal-id="GKH340C40" data-open="GKH340C40" class="link link-ref link-reveal xref-default">40</a>
 ). TC (total column score) is the number of correctly aligned columns 
divided by the number of columns in the reference alignment; this is 
Thompson <em>et al.</em> ’s CS and is equivalent to <em>Q</em> in the case of two sequences (as in PREFAB). APDB ( <a reveal-id="GKH340C41" data-open="GKH340C41" class="link link-ref link-reveal xref-default">41</a> ) is derived from structures alone; no reference alignment of the sequences or structures is needed. For BAliBASE, we use <em>Q</em> and TC, measured only on core blocks as annotated in the database. For PREFAB, we use <em>Q</em> , including only those positions on which CE and FSSP agree, and also APDB. For SMART, we use <em>Q</em> and TC computed for all columns. For SABmark, we average the <em>Q</em> score over each pair of sequences. TC score is not applicable to SABmark as the reference alignments are pairwise. </p>                            <h3 scrollto-destination="38266772" id="38266772" class="section-title">Statistical analysis</h3>
<p> Following Thompson <em>et al.</em> ( <a reveal-id="GKH340C19" data-open="GKH340C19" class="link link-ref link-reveal xref-default">19</a> ), statistical significance is measured by a Friedman rank test ( <a reveal-id="GKH340C42" data-open="GKH340C42" class="link link-ref link-reveal xref-default">42</a> ), which is more conservative than the Wilcoxon test that has also been used for alignment accuracy discrimination ( <a reveal-id="GKH340C5" data-open="GKH340C5" class="link link-ref link-reveal xref-default">5</a> , <a reveal-id="GKH340C7" data-open="GKH340C7" class="link link-ref link-reveal xref-default">7</a> , <a reveal-id="GKH340C8" data-open="GKH340C8" class="link link-ref link-reveal xref-default">8</a>
 ) as fewer assumptions are made about the population distribution. In 
particular, the Wilcoxon test assumes a symmetrical difference between 
two methods, but in practice we sometimes observe a significant skew. 
PREFAB and SABmark use automated structure alignment methods, which 
sometimes produce questionable results. Many low‐quality regions are 
eliminated by taking the consensus between two independent aligners, but
 some may remain. In PREFAB, assessment of a multiple alignment is made 
on a single pair of sequences, which may be more or less accurately 
aligned than the average over all pairs. In SABmark, the upper bound on <em>Q</em>
 is less than 1 to a varying degree because the pairwise reference 
alignments may not be mutually consistent. These effects can be viewed 
as introducing noise into the experiment, and a single accuracy 
measurement may be subject to error. However, as the structural aligners
 do not use primary sequence, these errors are unbiased with respect to 
sequence methods. A difference in accuracy between two sequence 
alignment methods can therefore be established by the Friedman test, and
 the measured difference in average accuracy will be approximately 
correct when measured over a sufficient number of samples. </p>                            <h2 scrollto-destination="38266774" id="38266774" class="section-title">RESULTS</h2>
<p> Quality scores and CPU times are summarized in Tables <a reveal-id="GKH340TB1" data-open="GKH340TB1" class="link link-ref link-reveal xref-default">1</a> – <a reveal-id="GKH340TB2" data-open="GKH340TB2" class="link link-ref link-reveal xref-default">2</a><a reveal-id="GKH340TB3" data-open="GKH340TB3" class="link link-ref link-reveal xref-default">3</a><a reveal-id="GKH340TB4" data-open="GKH340TB4" class="link link-ref link-reveal xref-default">4</a><a reveal-id="GKH340TB5" data-open="GKH340TB5" class="link link-ref link-reveal xref-default">5</a><a reveal-id="GKH340TB6" data-open="GKH340TB6" class="link link-ref link-reveal xref-default">6</a><a reveal-id="GKH340TB7" data-open="GKH340TB7" class="link link-ref link-reveal xref-default">7</a> ; rankings and statistical significance on PREFAB and BAliBASE for all pairs of methods are given in Table <a reveal-id="GKH340TB8" data-open="GKH340TB8" class="link link-ref link-reveal xref-default">8</a>
 . On all test sets and quality measures, MUSCLE achieves the highest 
ranking (in some cases jointly with other methods due to lack of 
statistical significance), and MUSCLE‐p is statistically 
indistinguishable from T‐Coffee and NWNSI. MUSCLE achieves the highest 
BAliBASE score reported to date, but the improvement of 1.6% in <em>Q</em> and 2.2% in TC over T‐Coffee has low significance ( <em>P</em> = 0.15). A similar result is found on SABmark, where MUSCLE achieves a 1.5% improvement over T‐Coffee in <em>Q</em> with <em>P</em> = 0.14. The <em>Q</em>
 score on PREFAB is best able to distinguish between methods, giving 
statistically significant rankings to MUSCLE &gt; MUSCLE‐p, MUSCLE &gt; 
T‐Coffee, MUSCLE &gt; NWNSI and MUSCLE‐p &gt; NWNSI. SMART also ranks 
MUSCLE highest. SMART cannot be considered definitive due to the use of 
sequence methods in construction of the database, although any bias from
 this source is likely to favor methods that were available to the SMART
 developers (i.e. to be against MUSCLE). The SMART results could be 
interpreted as suggesting that MUSCLE alignments are more consistent 
with refinements made by human experts. The APDB score appears to be 
relatively insensitive, showing no significant improvement due to the 
refinement stage of MUSCLE (similarly for MAFFT; not shown), and is not 
able to distinguish between the four highest scoring methods. We 
speculate that the scatter observed in the correlation between APDB and 
more conventional measures such as TC ( <a reveal-id="GKH340C40" data-open="GKH340C40" class="link link-ref link-reveal xref-default">40</a> ) injects sufficient noise to obscure meaningful differences in accuracy that can be resolved using <em>Q</em> . The low rank of Align‐m on SABmark differs from results quoted by Van Walle <em>et al.</em> ( <a reveal-id="GKH340C21" data-open="GKH340C21" class="link link-ref link-reveal xref-default">21</a>
 ), who assessed pairwise alignments produced by an intermediate step in
 the algorithm, whereas we used the final multiple alignment. </p>                            <h3 scrollto-destination="38266776" id="38266776" class="section-title">Resource requirements for large numbers of sequences</h3>
<p> To investigate resource requirements for increasing number of sequences <em>N</em> , we used the Rose sequence generator ( <a reveal-id="GKH340C43" data-open="GKH340C43" class="link link-ref link-reveal xref-default">43</a> ) (complete results not shown). In agreement with other studies, [e.g. Katoh <em>et al.</em> ( <a reveal-id="GKH340C8" data-open="GKH340C8" class="link link-ref link-reveal xref-default">8</a> )], we found that T‐Coffee was unable to align more than approximately 10 <sup>2</sup>
 sequences of typical length on a current desktop computer. CLUSTALW was
 able to align a few hundred sequences, with a practical limit around <em>N</em> = 10 <sup>3</sup> where CPU time begins to scale approximately as <em>N</em><sup>4</sup>
 . The largest set had 5000 sequences of average length 350. MUSCLE‐p 
completed this test in 7 min, compared with 10 min for FFTNS1; we 
estimate that CLUSTALW would need approximately 1 year. </p>                            <h2 scrollto-destination="38266778" id="38266778" class="section-title">DISCUSSION</h2>
<p> We have described a new multiple sequence alignment algorithm, 
MUSCLE, and presented evidence that it creates alignments with average 
accuracy comparable with or superior to the best current methods. It 
should be emphasized that performance differences between the better 
methods emerge only when averaged over a large number of test cases, 
even when alignments are considered trustworthy. For example, on 
BAliBASE, the lowest scoring of the tested methods (FFTNS1) achieved a 
higher <em>Q</em> than the highest scoring (MUSCLE) in 21 out of 141 
alignments and tied in 19 more; compared with T‐Coffee, MUSCLE scored 
higher or tied in 95 cases, but lower in 24. This suggests the use of 
multiple algorithms and careful inspection of the results. MUSCLE is 
comparable in speed with CLUSTALW, completing a test set (PREFAB) 
averaging 49 sequences of length 240 in about half the time. The 
progressive method MUSCLE‐p, which has average accuracy statistically 
indistinguishable from T‐Coffee and the most accurate MAFFT script, is 
the fastest algorithm known to the author for large numbers of 
sequences, able to align 5000 sequences of average length 350 in 7 min 
on a current desktop computer. The MUSCLE software, source code and test
 data are freely available at: <a class="link link-uri" href="http://www.drive5.com/muscle" target="">http://www.drive5. com/muscle.</a></p><p></p>                        <a id="38266782" scrollto-destination="38266782"></a>
<div id="GKH340F1" data-id="GKH340F1" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkh340f1.jpg" alt="Figure 1. Motifs misaligned by a progressive method. A set of 41 sequences containing SH2 domains ( 44 ) were aligned by the progressive method T‐Coffee (above), and by MUSCLE (below). The N‐terminal region of a subset of five sequences is shown. The highlighted columns (upper case) are conserved within this family but are misaligned by T‐Coffee. It should be noted that T‐Coffee aligns these motifs correctly when given these five sequences alone; the problem arises in the context of the other sequences. Complete alignments are available at http://www.drive5.com/muscle." path-from-xml="gkh340f1.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/38266782/gkh340f1.jpeg" path-from-xml="gkh340f1.jpeg" target="_blank">View large</a><a section="38266782" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/32/5/10.1093_nar_gkh340/2/gkh340f1.jpeg?Expires=2147483647&amp;Signature=lYIQsfzAR1s0c8V3o82BKYnIZLawi0wh1wnoEU9sMlHMiqlsW67l0nblB3A6n4sWRqYC2jo76jcwXj%7EROTXWDXKEgziJn1vb-HcqAMgyF8oXRx1z9xVTObm4-dxSIRkuwJmYoAVMF4yyBLaLtTG7sBdTveX7V621Zvc-Yskybmebx65kwPJ4%7E4mu6xRJ1H48PAm2seJg0qaBlhynKqkyK0QjvOxTGwsmztUStqPW4xeqgty-1HxlFttqvv8FfYbzCsvZhkmW89T5xEtrZUJ13zoZoa67G7mEtEqebr%7EYPddFubF%7Ex90XRhVsPrjz2al7T8yvGA9-0Fj9-tZQfAdVKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=38266782&amp;ar=2380623&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkh340f1.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 1.</strong> Motifs misaligned by a progressive method. A set of 41 sequences containing SH2 domains ( <a reveal-id="GKH340C44" data-open="GKH340C44" class="link link-ref link-reveal xref-default">44</a>
 ) were aligned by the progressive method T‐Coffee (above), and by 
MUSCLE (below). The N‐terminal region of a subset of five sequences is 
shown. The highlighted columns (upper case) are conserved within this 
family but are misaligned by T‐Coffee. It should be noted that T‐Coffee 
aligns these motifs correctly when given these five sequences alone; the
 problem arises in the context of the other sequences. Complete 
alignments are available at <a class="link link-uri" href="http://www.drive5.com/muscle" target="">http://www.drive5.com/muscle.</a></p></div></div><div content-id="GKH340F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkh340f1.jpg" alt="Figure 1. Motifs misaligned by a progressive method. A set of 41 sequences containing SH2 domains ( 44 ) were aligned by the progressive method T‐Coffee (above), and by MUSCLE (below). The N‐terminal region of a subset of five sequences is shown. The highlighted columns (upper case) are conserved within this family but are misaligned by T‐Coffee. It should be noted that T‐Coffee aligns these motifs correctly when given these five sequences alone; the problem arises in the context of the other sequences. Complete alignments are available at http://www.drive5.com/muscle." path-from-xml="gkh340f1.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/38266782/gkh340f1.jpeg" path-from-xml="gkh340f1.jpeg" target="_blank">View large</a><a section="38266782" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/32/5/10.1093_nar_gkh340/2/gkh340f1.jpeg?Expires=2147483647&amp;Signature=lYIQsfzAR1s0c8V3o82BKYnIZLawi0wh1wnoEU9sMlHMiqlsW67l0nblB3A6n4sWRqYC2jo76jcwXj%7EROTXWDXKEgziJn1vb-HcqAMgyF8oXRx1z9xVTObm4-dxSIRkuwJmYoAVMF4yyBLaLtTG7sBdTveX7V621Zvc-Yskybmebx65kwPJ4%7E4mu6xRJ1H48PAm2seJg0qaBlhynKqkyK0QjvOxTGwsmztUStqPW4xeqgty-1HxlFttqvv8FfYbzCsvZhkmW89T5xEtrZUJ13zoZoa67G7mEtEqebr%7EYPddFubF%7Ex90XRhVsPrjz2al7T8yvGA9-0Fj9-tZQfAdVKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=38266782&amp;ar=2380623&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkh340f1.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 1.</strong> Motifs misaligned by a progressive method. A set of 41 sequences containing SH2 domains ( <a reveal-id="GKH340C44" data-open="GKH340C44" class="link link-ref link-reveal xref-default">44</a>
 ) were aligned by the progressive method T‐Coffee (above), and by 
MUSCLE (below). The N‐terminal region of a subset of five sequences is 
shown. The highlighted columns (upper case) are conserved within this 
family but are misaligned by T‐Coffee. It should be noted that T‐Coffee 
aligns these motifs correctly when given these five sequences alone; the
 problem arises in the context of the other sequences. Complete 
alignments are available at <a class="link link-uri" href="http://www.drive5.com/muscle" target="">http://www.drive5.com/muscle.</a></p></div></div>                        <a id="38266783" scrollto-destination="38266783"></a>
<div id="GKH340F2" data-id="GKH340F2" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkh340f2.jpg" alt="Figure 2. This diagram summarizes the flow of the MUSCLE algorithm. There are three main stages: Stage 1 (draft progressive), Stage 2 (improved progressive) and Stage 3 (refinement). A multiple alignment is available at the completion of each stage, at which point the algorithm may terminate. " path-from-xml="gkh340f2.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/38266783/gkh340f2.jpeg" path-from-xml="gkh340f2.jpeg" target="_blank">View large</a><a section="38266783" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/32/5/10.1093_nar_gkh340/2/gkh340f2.jpeg?Expires=2147483647&amp;Signature=LvHhVK8aetvLATmKVK%7E4psR9nR%7EZuFRgQIAg55IK7xoOVR2qHELGOKkF4NT3vJESH7hMyn2REPP21oRR-TUIJ44dfPPJCn-9VfspQ55kOcSPUB4QkIFV0EjJxCmn%7EiM4I-q8kj1EWd55NsVWJKVzKI6o8I2OvQ9goWgj8B9VDKgSjqjdzrI5QOO87Q6wH38TIY0A%7EIV3ra2udVavEUjG-Nngu9pfTuXWKtN7tgO93Z1BZ0UJe-XMZ-qi-dDbNqK7%7EEKHfGdERnxO9uRLPpbjHuKuJDon9SVtzxaIlhwixxgFfwTlRwCC%7EQkE0rFawRRhMZYiwzbHcCaDjaojrF8OvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=38266783&amp;ar=2380623&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkh340f2.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 2.</strong>
 This diagram summarizes the flow of the MUSCLE algorithm. There are 
three main stages: Stage 1 (draft progressive), Stage 2 (improved 
progressive) and Stage 3 (refinement). A multiple alignment is available
 at the completion of each stage, at which point the algorithm may 
terminate. </p></div></div><div content-id="GKH340F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkh340f2.jpg" alt="Figure 2. This diagram summarizes the flow of the MUSCLE algorithm. There are three main stages: Stage 1 (draft progressive), Stage 2 (improved progressive) and Stage 3 (refinement). A multiple alignment is available at the completion of each stage, at which point the algorithm may terminate. " path-from-xml="gkh340f2.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/38266783/gkh340f2.jpeg" path-from-xml="gkh340f2.jpeg" target="_blank">View large</a><a section="38266783" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/32/5/10.1093_nar_gkh340/2/gkh340f2.jpeg?Expires=2147483647&amp;Signature=LvHhVK8aetvLATmKVK%7E4psR9nR%7EZuFRgQIAg55IK7xoOVR2qHELGOKkF4NT3vJESH7hMyn2REPP21oRR-TUIJ44dfPPJCn-9VfspQ55kOcSPUB4QkIFV0EjJxCmn%7EiM4I-q8kj1EWd55NsVWJKVzKI6o8I2OvQ9goWgj8B9VDKgSjqjdzrI5QOO87Q6wH38TIY0A%7EIV3ra2udVavEUjG-Nngu9pfTuXWKtN7tgO93Z1BZ0UJe-XMZ-qi-dDbNqK7%7EEKHfGdERnxO9uRLPpbjHuKuJDon9SVtzxaIlhwixxgFfwTlRwCC%7EQkE0rFawRRhMZYiwzbHcCaDjaojrF8OvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=38266783&amp;ar=2380623&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkh340f2.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 2.</strong>
 This diagram summarizes the flow of the MUSCLE algorithm. There are 
three main stages: Stage 1 (draft progressive), Stage 2 (improved 
progressive) and Stage 3 (refinement). A multiple alignment is available
 at the completion of each stage, at which point the algorithm may 
terminate. </p></div></div>                        <a id="38266784" scrollto-destination="38266784"></a>
<div content-id="GKH340TB1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB1" class="table-wrap-title" data-id="GKH340TB1"><div class="label -label"><strong>Table 1. </strong></div><div class="caption"><p>BAliBASE scores and times</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.896&nbsp;</td><td>0.747&nbsp;</td><td>97&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.883&nbsp;</td><td>0.727&nbsp;</td><td>52&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.882&nbsp;</td><td>0.731&nbsp;</td><td>1500&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.881&nbsp;</td><td>0.722&nbsp;</td><td>170&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.860&nbsp;</td><td>0.690&nbsp;</td><td>170&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.844&nbsp;</td><td>0.646&nbsp;</td><td>16&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB1" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.896&nbsp;</td><td>0.747&nbsp;</td><td>97&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.883&nbsp;</td><td>0.727&nbsp;</td><td>52&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.882&nbsp;</td><td>0.731&nbsp;</td><td>1500&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.881&nbsp;</td><td>0.722&nbsp;</td><td>170&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.860&nbsp;</td><td>0.690&nbsp;</td><td>170&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.844&nbsp;</td><td>0.646&nbsp;</td><td>16&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> Average <em>Q</em>
 and TC scores for each method on BAliBASE are shown, together with the 
total CPU time in seconds. Align‐m aborted on two alignments; average 
scores on the remainder were <em>Q</em> = 0.852 and TC = 0.670, requiring 2202 s. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266784" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB1" class="table-wrap-title" data-id="GKH340TB1"><div class="label -label"><strong>Table 1. </strong></div><div class="caption"><p>BAliBASE scores and times</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.896&nbsp;</td><td>0.747&nbsp;</td><td>97&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.883&nbsp;</td><td>0.727&nbsp;</td><td>52&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.882&nbsp;</td><td>0.731&nbsp;</td><td>1500&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.881&nbsp;</td><td>0.722&nbsp;</td><td>170&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.860&nbsp;</td><td>0.690&nbsp;</td><td>170&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.844&nbsp;</td><td>0.646&nbsp;</td><td>16&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB1" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.896&nbsp;</td><td>0.747&nbsp;</td><td>97&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.883&nbsp;</td><td>0.727&nbsp;</td><td>52&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.882&nbsp;</td><td>0.731&nbsp;</td><td>1500&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.881&nbsp;</td><td>0.722&nbsp;</td><td>170&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.860&nbsp;</td><td>0.690&nbsp;</td><td>170&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.844&nbsp;</td><td>0.646&nbsp;</td><td>16&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> Average <em>Q</em>
 and TC scores for each method on BAliBASE are shown, together with the 
total CPU time in seconds. Align‐m aborted on two alignments; average 
scores on the remainder were <em>Q</em> = 0.852 and TC = 0.670, requiring 2202 s. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266784" target="_blank">View Large</a></div></div>                        <a id="38266785" scrollto-destination="38266785"></a>
<div content-id="GKH340TB2" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB2" class="table-wrap-title" data-id="GKH340TB2"><div class="label -label"><strong>Table 2.</strong></div><div class="caption"><p>BAliBASE Q scores on subsets </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.887&nbsp;</td><td>0.935&nbsp;</td><td>0.823&nbsp;</td><td>0.876&nbsp;</td><td>0.968&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.871&nbsp;</td><td>0.928&nbsp;</td><td>0.813&nbsp;</td><td>0.857&nbsp;</td><td>0.974&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.866&nbsp;</td><td>0.934&nbsp;</td><td>0.787&nbsp;</td><td>0.917&nbsp;</td><td>0.957&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.867&nbsp;</td><td>0.923&nbsp;</td><td>0.787&nbsp;</td><td>0.904&nbsp;</td><td>0.963&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.861&nbsp;</td><td>0.932&nbsp;</td><td>0.751&nbsp;</td><td>0.823&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.838&nbsp;</td><td>0.908&nbsp;</td><td>0.708&nbsp;</td><td>0.793&nbsp;</td><td>0.947&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB2" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.887&nbsp;</td><td>0.935&nbsp;</td><td>0.823&nbsp;</td><td>0.876&nbsp;</td><td>0.968&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.871&nbsp;</td><td>0.928&nbsp;</td><td>0.813&nbsp;</td><td>0.857&nbsp;</td><td>0.974&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.866&nbsp;</td><td>0.934&nbsp;</td><td>0.787&nbsp;</td><td>0.917&nbsp;</td><td>0.957&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.867&nbsp;</td><td>0.923&nbsp;</td><td>0.787&nbsp;</td><td>0.904&nbsp;</td><td>0.963&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.861&nbsp;</td><td>0.932&nbsp;</td><td>0.751&nbsp;</td><td>0.823&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.838&nbsp;</td><td>0.908&nbsp;</td><td>0.708&nbsp;</td><td>0.793&nbsp;</td><td>0.947&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> The average <em>Q</em>
 score for each method on each BAliBASE subset is shown. Ref1 is the 
largest subset with 81 test sets, comprising almost 60% of the database.
 Other subsets are smaller. For example, Ref4 and Ref5 have 12 
alignments each, and there are large variances in the individual scores 
from which the averages are computed. In our opinion, it is not possible
 to draw meaningful conclusions about the relative performance of 
different methods on these subsets. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266785" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB2" class="table-wrap-title" data-id="GKH340TB2"><div class="label -label"><strong>Table 2.</strong></div><div class="caption"><p>BAliBASE Q scores on subsets </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.887&nbsp;</td><td>0.935&nbsp;</td><td>0.823&nbsp;</td><td>0.876&nbsp;</td><td>0.968&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.871&nbsp;</td><td>0.928&nbsp;</td><td>0.813&nbsp;</td><td>0.857&nbsp;</td><td>0.974&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.866&nbsp;</td><td>0.934&nbsp;</td><td>0.787&nbsp;</td><td>0.917&nbsp;</td><td>0.957&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.867&nbsp;</td><td>0.923&nbsp;</td><td>0.787&nbsp;</td><td>0.904&nbsp;</td><td>0.963&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.861&nbsp;</td><td>0.932&nbsp;</td><td>0.751&nbsp;</td><td>0.823&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.838&nbsp;</td><td>0.908&nbsp;</td><td>0.708&nbsp;</td><td>0.793&nbsp;</td><td>0.947&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB2" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.887&nbsp;</td><td>0.935&nbsp;</td><td>0.823&nbsp;</td><td>0.876&nbsp;</td><td>0.968&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.871&nbsp;</td><td>0.928&nbsp;</td><td>0.813&nbsp;</td><td>0.857&nbsp;</td><td>0.974&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.866&nbsp;</td><td>0.934&nbsp;</td><td>0.787&nbsp;</td><td>0.917&nbsp;</td><td>0.957&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.867&nbsp;</td><td>0.923&nbsp;</td><td>0.787&nbsp;</td><td>0.904&nbsp;</td><td>0.963&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.861&nbsp;</td><td>0.932&nbsp;</td><td>0.751&nbsp;</td><td>0.823&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.838&nbsp;</td><td>0.908&nbsp;</td><td>0.708&nbsp;</td><td>0.793&nbsp;</td><td>0.947&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> The average <em>Q</em>
 score for each method on each BAliBASE subset is shown. Ref1 is the 
largest subset with 81 test sets, comprising almost 60% of the database.
 Other subsets are smaller. For example, Ref4 and Ref5 have 12 
alignments each, and there are large variances in the individual scores 
from which the averages are computed. In our opinion, it is not possible
 to draw meaningful conclusions about the relative performance of 
different methods on these subsets. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266785" target="_blank">View Large</a></div></div>                        <a id="38266786" scrollto-destination="38266786"></a>
<div content-id="GKH340TB3" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB3" class="table-wrap-title" data-id="GKH340TB3"><div class="label -label"><strong>Table 3.</strong></div><div class="caption"><p>BAliBASE TC scores on subsets</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.815&nbsp;</td><td>0.574&nbsp;</td><td>0.577&nbsp;</td><td>0.627&nbsp;</td><td>0.902&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.795&nbsp;</td><td>0.558&nbsp;</td><td>0.550&nbsp;</td><td>0.598&nbsp;</td><td>0.891&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.780&nbsp;</td><td>0.573&nbsp;</td><td>0.510&nbsp;</td><td>0.751&nbsp;</td><td>0.903&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.788&nbsp;</td><td>0.514&nbsp;</td><td>0.514&nbsp;</td><td>0.742&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.782&nbsp;</td><td>0.579&nbsp;</td><td>0.470&nbsp;</td><td>0.542&nbsp;</td><td>0.638&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.732&nbsp;</td><td>0.496&nbsp;</td><td>0.350&nbsp;</td><td>0.451&nbsp;</td><td>0.831&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB3" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.815&nbsp;</td><td>0.574&nbsp;</td><td>0.577&nbsp;</td><td>0.627&nbsp;</td><td>0.902&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.795&nbsp;</td><td>0.558&nbsp;</td><td>0.550&nbsp;</td><td>0.598&nbsp;</td><td>0.891&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.780&nbsp;</td><td>0.573&nbsp;</td><td>0.510&nbsp;</td><td>0.751&nbsp;</td><td>0.903&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.788&nbsp;</td><td>0.514&nbsp;</td><td>0.514&nbsp;</td><td>0.742&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.782&nbsp;</td><td>0.579&nbsp;</td><td>0.470&nbsp;</td><td>0.542&nbsp;</td><td>0.638&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.732&nbsp;</td><td>0.496&nbsp;</td><td>0.350&nbsp;</td><td>0.451&nbsp;</td><td>0.831&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>The average TC score for each method on each BAliBASE subset is shown.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266786" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB3" class="table-wrap-title" data-id="GKH340TB3"><div class="label -label"><strong>Table 3.</strong></div><div class="caption"><p>BAliBASE TC scores on subsets</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.815&nbsp;</td><td>0.574&nbsp;</td><td>0.577&nbsp;</td><td>0.627&nbsp;</td><td>0.902&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.795&nbsp;</td><td>0.558&nbsp;</td><td>0.550&nbsp;</td><td>0.598&nbsp;</td><td>0.891&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.780&nbsp;</td><td>0.573&nbsp;</td><td>0.510&nbsp;</td><td>0.751&nbsp;</td><td>0.903&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.788&nbsp;</td><td>0.514&nbsp;</td><td>0.514&nbsp;</td><td>0.742&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.782&nbsp;</td><td>0.579&nbsp;</td><td>0.470&nbsp;</td><td>0.542&nbsp;</td><td>0.638&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.732&nbsp;</td><td>0.496&nbsp;</td><td>0.350&nbsp;</td><td>0.451&nbsp;</td><td>0.831&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB3" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>Ref1&nbsp;</td><td>Ref2&nbsp;</td><td>Ref3&nbsp;</td><td>Ref4&nbsp;</td><td>Ref5&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.815&nbsp;</td><td>0.574&nbsp;</td><td>0.577&nbsp;</td><td>0.627&nbsp;</td><td>0.902&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.795&nbsp;</td><td>0.558&nbsp;</td><td>0.550&nbsp;</td><td>0.598&nbsp;</td><td>0.891&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.780&nbsp;</td><td>0.573&nbsp;</td><td>0.510&nbsp;</td><td>0.751&nbsp;</td><td>0.903&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.788&nbsp;</td><td>0.514&nbsp;</td><td>0.514&nbsp;</td><td>0.742&nbsp;</td><td>0.859&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.782&nbsp;</td><td>0.579&nbsp;</td><td>0.470&nbsp;</td><td>0.542&nbsp;</td><td>0.638&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.732&nbsp;</td><td>0.496&nbsp;</td><td>0.350&nbsp;</td><td>0.451&nbsp;</td><td>0.831&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>The average TC score for each method on each BAliBASE subset is shown.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266786" target="_blank">View Large</a></div></div>                        <a id="38266787" scrollto-destination="38266787"></a>
<div content-id="GKH340TB4" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB4" class="table-wrap-title" data-id="GKH340TB4"><div class="label -label"><strong>Table 4.</strong></div><div class="caption"><p><em>Q</em> scores and times on PREFAB </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>0–20%&nbsp;</td><td>20–40%&nbsp;</td><td>40–70%&nbsp;</td><td>70–100%&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.645&nbsp;</td><td>0.473&nbsp;</td><td>0.813&nbsp;</td><td>0.937&nbsp;</td><td>0.980&nbsp;</td><td> 1.7 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.634&nbsp;</td><td>0.460&nbsp;</td><td>0.802&nbsp;</td><td>0.942&nbsp;</td><td>0.985&nbsp;</td><td> 2.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.615&nbsp;</td><td>0.464&nbsp;</td><td>0.795&nbsp;</td><td>0.935&nbsp;</td><td>0.976&nbsp;</td><td> 1.0 × 10 <sup>6</sup>&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.615&nbsp;</td><td>0.448&nbsp;</td><td>0.772&nbsp;</td><td>0.930&nbsp;</td><td>0.939&nbsp;</td><td> 1.4 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.591&nbsp;</td><td>0.423&nbsp;</td><td>0.756&nbsp;</td><td>0.931&nbsp;</td><td>0.938&nbsp;</td><td> 1.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.563&nbsp;</td><td>0.382&nbsp;</td><td>0.732&nbsp;</td><td>0.916&nbsp;</td><td>0.930&nbsp;</td><td> 3.3 × 10 <sup>4</sup>&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB4" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>0–20%&nbsp;</td><td>20–40%&nbsp;</td><td>40–70%&nbsp;</td><td>70–100%&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.645&nbsp;</td><td>0.473&nbsp;</td><td>0.813&nbsp;</td><td>0.937&nbsp;</td><td>0.980&nbsp;</td><td> 1.7 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.634&nbsp;</td><td>0.460&nbsp;</td><td>0.802&nbsp;</td><td>0.942&nbsp;</td><td>0.985&nbsp;</td><td> 2.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.615&nbsp;</td><td>0.464&nbsp;</td><td>0.795&nbsp;</td><td>0.935&nbsp;</td><td>0.976&nbsp;</td><td> 1.0 × 10 <sup>6</sup>&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.615&nbsp;</td><td>0.448&nbsp;</td><td>0.772&nbsp;</td><td>0.930&nbsp;</td><td>0.939&nbsp;</td><td> 1.4 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.591&nbsp;</td><td>0.423&nbsp;</td><td>0.756&nbsp;</td><td>0.931&nbsp;</td><td>0.938&nbsp;</td><td> 1.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.563&nbsp;</td><td>0.382&nbsp;</td><td>0.732&nbsp;</td><td>0.916&nbsp;</td><td>0.930&nbsp;</td><td> 3.3 × 10 <sup>4</sup>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> The average <em>Q</em>
 score for each method over all PREFAB alignments (All), and the total 
CPU time in seconds are given. The remaining columns show average <em>Q</em>
 scores on subsets in which the structure pairs fall within the given 
pairwise identity ranges. Note that T‐Coffee required 10 CPU days to 
complete the test, compared with &lt;5 h for MUSCLE and ∼30 min for 
MUSCLE‐p. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266787" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB4" class="table-wrap-title" data-id="GKH340TB4"><div class="label -label"><strong>Table 4.</strong></div><div class="caption"><p><em>Q</em> scores and times on PREFAB </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>0–20%&nbsp;</td><td>20–40%&nbsp;</td><td>40–70%&nbsp;</td><td>70–100%&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.645&nbsp;</td><td>0.473&nbsp;</td><td>0.813&nbsp;</td><td>0.937&nbsp;</td><td>0.980&nbsp;</td><td> 1.7 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.634&nbsp;</td><td>0.460&nbsp;</td><td>0.802&nbsp;</td><td>0.942&nbsp;</td><td>0.985&nbsp;</td><td> 2.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.615&nbsp;</td><td>0.464&nbsp;</td><td>0.795&nbsp;</td><td>0.935&nbsp;</td><td>0.976&nbsp;</td><td> 1.0 × 10 <sup>6</sup>&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.615&nbsp;</td><td>0.448&nbsp;</td><td>0.772&nbsp;</td><td>0.930&nbsp;</td><td>0.939&nbsp;</td><td> 1.4 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.591&nbsp;</td><td>0.423&nbsp;</td><td>0.756&nbsp;</td><td>0.931&nbsp;</td><td>0.938&nbsp;</td><td> 1.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.563&nbsp;</td><td>0.382&nbsp;</td><td>0.732&nbsp;</td><td>0.916&nbsp;</td><td>0.930&nbsp;</td><td> 3.3 × 10 <sup>4</sup>&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB4" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>0–20%&nbsp;</td><td>20–40%&nbsp;</td><td>40–70%&nbsp;</td><td>70–100%&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.645&nbsp;</td><td>0.473&nbsp;</td><td>0.813&nbsp;</td><td>0.937&nbsp;</td><td>0.980&nbsp;</td><td> 1.7 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.634&nbsp;</td><td>0.460&nbsp;</td><td>0.802&nbsp;</td><td>0.942&nbsp;</td><td>0.985&nbsp;</td><td> 2.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.615&nbsp;</td><td>0.464&nbsp;</td><td>0.795&nbsp;</td><td>0.935&nbsp;</td><td>0.976&nbsp;</td><td> 1.0 × 10 <sup>6</sup>&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.615&nbsp;</td><td>0.448&nbsp;</td><td>0.772&nbsp;</td><td>0.930&nbsp;</td><td>0.939&nbsp;</td><td> 1.4 × 10 <sup>4</sup>&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.591&nbsp;</td><td>0.423&nbsp;</td><td>0.756&nbsp;</td><td>0.931&nbsp;</td><td>0.938&nbsp;</td><td> 1.0 × 10 <sup>3</sup>&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.563&nbsp;</td><td>0.382&nbsp;</td><td>0.732&nbsp;</td><td>0.916&nbsp;</td><td>0.930&nbsp;</td><td> 3.3 × 10 <sup>4</sup>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> The average <em>Q</em>
 score for each method over all PREFAB alignments (All), and the total 
CPU time in seconds are given. The remaining columns show average <em>Q</em>
 scores on subsets in which the structure pairs fall within the given 
pairwise identity ranges. Note that T‐Coffee required 10 CPU days to 
complete the test, compared with &lt;5 h for MUSCLE and ∼30 min for 
MUSCLE‐p. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266787" target="_blank">View Large</a></div></div>                        <a id="38266788" scrollto-destination="38266788"></a>
<div content-id="GKH340TB5" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB5" class="table-wrap-title" data-id="GKH340TB5"><div class="label -label"><strong>Table 5.</strong></div><div class="caption"><p>APDB scores on PREFAB</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>APDB&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>62.0&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>61.4&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>60.8&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>59.1&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB5" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>APDB&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>62.0&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>61.4&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>60.8&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>59.1&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>
 The average APDB score of each method on the PREFAB reference 
alignments is given. There is no statistically significant difference 
between the four best methods. The top four are significantly better 
than FFTNS1 (MUSCLE‐p &gt; FFTNS1 with <em>P</em> = 0.009), and FFTNS1 is significantly better than CLUSTALW ( <em>P</em> = 3 × 10 <sup>–5</sup> ). </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266788" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB5" class="table-wrap-title" data-id="GKH340TB5"><div class="label -label"><strong>Table 5.</strong></div><div class="caption"><p>APDB scores on PREFAB</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>APDB&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>62.0&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>61.4&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>60.8&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>59.1&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB5" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>APDB&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>62.0&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>61.9&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>61.4&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>60.8&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>59.1&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>
 The average APDB score of each method on the PREFAB reference 
alignments is given. There is no statistically significant difference 
between the four best methods. The top four are significantly better 
than FFTNS1 (MUSCLE‐p &gt; FFTNS1 with <em>P</em> = 0.009), and FFTNS1 is significantly better than CLUSTALW ( <em>P</em> = 3 × 10 <sup>–5</sup> ). </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266788" target="_blank">View Large</a></div></div>                        <a id="38266789" scrollto-destination="38266789"></a>
<div content-id="GKH340TB6" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB6" class="table-wrap-title" data-id="GKH340TB6"><div class="label -label"><strong>Table 6.</strong></div><div class="caption"><p><em>Q</em> scores and CPU times on SABmark </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>Superfamily&nbsp;</td><td>Twilight&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.430&nbsp;</td><td>0.523&nbsp;</td><td>0.249&nbsp;</td><td>1886&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.424&nbsp;</td><td>0.519&nbsp;</td><td>0.237&nbsp;</td><td>5615&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.416&nbsp;</td><td>0.511&nbsp;</td><td>0.230&nbsp;</td><td>304&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.410&nbsp;</td><td>0.506&nbsp;</td><td>0.223&nbsp;</td><td>629&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.404&nbsp;</td><td>0.498&nbsp;</td><td>0.220&nbsp;</td><td>206&nbsp;</td></tr><tr><td>FFSNT1&nbsp;</td><td>0.373&nbsp;</td><td>0.467&nbsp;</td><td>0.190&nbsp;</td><td>75&nbsp;</td></tr><tr><td>Align‐m&nbsp;</td><td>0.348&nbsp;</td><td>0.445&nbsp;</td><td>0.172&nbsp;</td><td>8902&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB6" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>Superfamily&nbsp;</td><td>Twilight&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.430&nbsp;</td><td>0.523&nbsp;</td><td>0.249&nbsp;</td><td>1886&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.424&nbsp;</td><td>0.519&nbsp;</td><td>0.237&nbsp;</td><td>5615&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.416&nbsp;</td><td>0.511&nbsp;</td><td>0.230&nbsp;</td><td>304&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.410&nbsp;</td><td>0.506&nbsp;</td><td>0.223&nbsp;</td><td>629&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.404&nbsp;</td><td>0.498&nbsp;</td><td>0.220&nbsp;</td><td>206&nbsp;</td></tr><tr><td>FFSNT1&nbsp;</td><td>0.373&nbsp;</td><td>0.467&nbsp;</td><td>0.190&nbsp;</td><td>75&nbsp;</td></tr><tr><td>Align‐m&nbsp;</td><td>0.348&nbsp;</td><td>0.445&nbsp;</td><td>0.172&nbsp;</td><td>8902&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> All gives the average <em>Q</em> score over all SABmark alignments, Superfamily and Twilight are average <em>Q</em> scores on the two subsets. These are computed first by averaging <em>Q</em>
 for each pair in a single multiple alignment, then averaging over 
multiple alignments. This corrects for the lack of independence between 
pairs in a given multiple alignment. Align‐m aborted in nine cases; 
quoted averages for this program are for completed alignments. Selected <em>P</em> ‐values are: MUSCLE &gt; T‐Coffee <em>P</em> = 0.14, MUSCLE &gt; MUSCLE‐p <em>P</em> = 4×10 <sup>–5</sup> , MUSCLE &gt; NWNSI <em>P</em> = 6 × 10 <sup>–6</sup> , MUSCLE‐p &gt; NWNSI <em>P</em> = 0.03, T‐Coffee &gt; MUSCLE‐p <em>P</em> = 0.1, T‐Coffee &gt; Align‐m <em>P</em> &lt; 10 <sup>–10</sup> . </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266789" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB6" class="table-wrap-title" data-id="GKH340TB6"><div class="label -label"><strong>Table 6.</strong></div><div class="caption"><p><em>Q</em> scores and CPU times on SABmark </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>Superfamily&nbsp;</td><td>Twilight&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.430&nbsp;</td><td>0.523&nbsp;</td><td>0.249&nbsp;</td><td>1886&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.424&nbsp;</td><td>0.519&nbsp;</td><td>0.237&nbsp;</td><td>5615&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.416&nbsp;</td><td>0.511&nbsp;</td><td>0.230&nbsp;</td><td>304&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.410&nbsp;</td><td>0.506&nbsp;</td><td>0.223&nbsp;</td><td>629&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.404&nbsp;</td><td>0.498&nbsp;</td><td>0.220&nbsp;</td><td>206&nbsp;</td></tr><tr><td>FFSNT1&nbsp;</td><td>0.373&nbsp;</td><td>0.467&nbsp;</td><td>0.190&nbsp;</td><td>75&nbsp;</td></tr><tr><td>Align‐m&nbsp;</td><td>0.348&nbsp;</td><td>0.445&nbsp;</td><td>0.172&nbsp;</td><td>8902&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB6" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>All&nbsp;</td><td>Superfamily&nbsp;</td><td>Twilight&nbsp;</td><td>CPU&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.430&nbsp;</td><td>0.523&nbsp;</td><td>0.249&nbsp;</td><td>1886&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.424&nbsp;</td><td>0.519&nbsp;</td><td>0.237&nbsp;</td><td>5615&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.416&nbsp;</td><td>0.511&nbsp;</td><td>0.230&nbsp;</td><td>304&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.410&nbsp;</td><td>0.506&nbsp;</td><td>0.223&nbsp;</td><td>629&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.404&nbsp;</td><td>0.498&nbsp;</td><td>0.220&nbsp;</td><td>206&nbsp;</td></tr><tr><td>FFSNT1&nbsp;</td><td>0.373&nbsp;</td><td>0.467&nbsp;</td><td>0.190&nbsp;</td><td>75&nbsp;</td></tr><tr><td>Align‐m&nbsp;</td><td>0.348&nbsp;</td><td>0.445&nbsp;</td><td>0.172&nbsp;</td><td>8902&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> All gives the average <em>Q</em> score over all SABmark alignments, Superfamily and Twilight are average <em>Q</em> scores on the two subsets. These are computed first by averaging <em>Q</em>
 for each pair in a single multiple alignment, then averaging over 
multiple alignments. This corrects for the lack of independence between 
pairs in a given multiple alignment. Align‐m aborted in nine cases; 
quoted averages for this program are for completed alignments. Selected <em>P</em> ‐values are: MUSCLE &gt; T‐Coffee <em>P</em> = 0.14, MUSCLE &gt; MUSCLE‐p <em>P</em> = 4×10 <sup>–5</sup> , MUSCLE &gt; NWNSI <em>P</em> = 6 × 10 <sup>–6</sup> , MUSCLE‐p &gt; NWNSI <em>P</em> = 0.03, T‐Coffee &gt; MUSCLE‐p <em>P</em> = 0.1, T‐Coffee &gt; Align‐m <em>P</em> &lt; 10 <sup>–10</sup> . </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266789" target="_blank">View Large</a></div></div>                        <a id="38266790" scrollto-destination="38266790"></a>
<div content-id="GKH340TB7" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB7" class="table-wrap-title" data-id="GKH340TB7"><div class="label -label"><strong>Table 7.</strong></div><div class="caption"><p><em>Q</em> and TC scores on SMART </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>Significance&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.855&nbsp;</td><td>0.537&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.848&nbsp;</td><td>0.546&nbsp;</td><td>0.03&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.836&nbsp;</td><td>0.505&nbsp;</td><td>0.54&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.835&nbsp;</td><td>0.503&nbsp;</td><td>0.16&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.823&nbsp;</td><td>0.504&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.817&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB7" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>Significance&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.855&nbsp;</td><td>0.537&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.848&nbsp;</td><td>0.546&nbsp;</td><td>0.03&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.836&nbsp;</td><td>0.505&nbsp;</td><td>0.54&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.835&nbsp;</td><td>0.503&nbsp;</td><td>0.16&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.823&nbsp;</td><td>0.504&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.817&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>
 The average Q and TC accuracy scores over the 267 reference alignments 
in SMART that have no more than 100 sequences are given. The last column
 is the <em>P</em> ‐value of the difference between the method in a row and the method in the next row, measured on the <em>Q</em> score. The <em>P</em> ‐value for MUSCLE &gt; T‐Coffee is 0.0004 on <em>Q</em> and 0.01 on TC; the <em>P</em> ‐value for NSI &gt; T‐Coffee is 0.19 on <em>Q</em> and 0.0002 on TC. The difference between MUSCLE and NWNSI is only weakly significant on the <em>Q</em> score ( <em>P</em> = 0.07) and is not significant on the TC score ( <em>P</em> = 0.3). </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266790" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB7" class="table-wrap-title" data-id="GKH340TB7"><div class="label -label"><strong>Table 7.</strong></div><div class="caption"><p><em>Q</em> and TC scores on SMART </p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>Significance&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.855&nbsp;</td><td>0.537&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.848&nbsp;</td><td>0.546&nbsp;</td><td>0.03&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.836&nbsp;</td><td>0.505&nbsp;</td><td>0.54&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.835&nbsp;</td><td>0.503&nbsp;</td><td>0.16&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.823&nbsp;</td><td>0.504&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.817&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB7" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td><em>Q</em>&nbsp;</td><td>TC&nbsp;</td><td>Significance&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>0.855&nbsp;</td><td>0.537&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td>0.848&nbsp;</td><td>0.546&nbsp;</td><td>0.03&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td>0.836&nbsp;</td><td>0.505&nbsp;</td><td>0.54&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>0.835&nbsp;</td><td>0.503&nbsp;</td><td>0.16&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td>0.823&nbsp;</td><td>0.504&nbsp;</td><td>0.07&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td>0.817&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>
 The average Q and TC accuracy scores over the 267 reference alignments 
in SMART that have no more than 100 sequences are given. The last column
 is the <em>P</em> ‐value of the difference between the method in a row and the method in the next row, measured on the <em>Q</em> score. The <em>P</em> ‐value for MUSCLE &gt; T‐Coffee is 0.0004 on <em>Q</em> and 0.01 on TC; the <em>P</em> ‐value for NSI &gt; T‐Coffee is 0.19 on <em>Q</em> and 0.0002 on TC. The difference between MUSCLE and NWNSI is only weakly significant on the <em>Q</em> score ( <em>P</em> = 0.07) and is not significant on the TC score ( <em>P</em> = 0.3). </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266790" target="_blank">View Large</a></div></div>                        <a id="38266791" scrollto-destination="38266791"></a>
<div content-id="GKH340TB8" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKH340TB8" class="table-wrap-title" data-id="GKH340TB8"><div class="label -label"><strong>Table 8.</strong></div><div class="caption"><p>Ranks and statistical significance on BAliBASE and PREFAB</p></div> </div><div class="table-overflow"><table><tbody><tr><td>&nbsp;</td><td>MUSCLE&nbsp;</td><td>MUSCLE‐p&nbsp;</td><td>T‐Coffee&nbsp;</td><td>NWNSI&nbsp;</td><td>FFTNS1&nbsp;</td><td>CLUSTALW&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>&nbsp;</td><td>+0.001&nbsp;</td><td>(0.15)&nbsp;</td><td>+ 0.005&nbsp;</td><td> +2 × 10 <sup>–6</sup>&nbsp;</td><td>+0.0002&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td> –6 × 10 <sup>–6</sup>&nbsp;</td><td>&nbsp;</td><td>(0.3)&nbsp;</td><td>(0.7)&nbsp;</td><td>+0.0002&nbsp;</td><td>+0.02&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>–0.0002&nbsp;</td><td>(0.4)&nbsp;</td><td>&nbsp;</td><td>(0.55)&nbsp;</td><td> +7 × 10 <sup>–5</sup>&nbsp;</td><td>+0.01&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –10 <sup>–7</sup>&nbsp;</td><td>&nbsp;</td><td>+0.0001&nbsp;</td><td>(0.06)&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>&nbsp;</td><td>–0.04&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>–0.008&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB8" class="table-modal"><table><tbody><tr><td>&nbsp;</td><td>MUSCLE&nbsp;</td><td>MUSCLE‐p&nbsp;</td><td>T‐Coffee&nbsp;</td><td>NWNSI&nbsp;</td><td>FFTNS1&nbsp;</td><td>CLUSTALW&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>&nbsp;</td><td>+0.001&nbsp;</td><td>(0.15)&nbsp;</td><td>+ 0.005&nbsp;</td><td> +2 × 10 <sup>–6</sup>&nbsp;</td><td>+0.0002&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td> –6 × 10 <sup>–6</sup>&nbsp;</td><td>&nbsp;</td><td>(0.3)&nbsp;</td><td>(0.7)&nbsp;</td><td>+0.0002&nbsp;</td><td>+0.02&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>–0.0002&nbsp;</td><td>(0.4)&nbsp;</td><td>&nbsp;</td><td>(0.55)&nbsp;</td><td> +7 × 10 <sup>–5</sup>&nbsp;</td><td>+0.01&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –10 <sup>–7</sup>&nbsp;</td><td>&nbsp;</td><td>+0.0001&nbsp;</td><td>(0.06)&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>&nbsp;</td><td>–0.04&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>–0.008&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> Each entry in the table contains the <em>P</em>
 ‐value assigned by a Friedman rank test to the difference between a 
pair of methods. The upper‐right corner of the matrix is obtained from <em>Q</em> scores on BAliBASE, the lower‐left corner from <em>Q</em> scores on PREFAB. If the method to the left is ranked higher than the method above, the <em>P‐</em> value is preceded by +. If the method to the left is ranked lower, the <em>P</em> ‐value is preceded by –. If the <em>P</em>
 ‐value is &gt;0.05, the difference is not considered significant and is
 shown in parentheses. So, for example, MUSCLE ranks higher than 
T‐Coffee on PREFAB with <em>P</em> = 0.0002 and MUSCLE‐p higher than CLUSTALW on BAliBASE with <em>P</em> = 0.02. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266791" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKH340TB8" class="table-wrap-title" data-id="GKH340TB8"><div class="label -label"><strong>Table 8.</strong></div><div class="caption"><p>Ranks and statistical significance on BAliBASE and PREFAB</p></div> </div><div class="table-overflow"><table><tbody><tr><td>&nbsp;</td><td>MUSCLE&nbsp;</td><td>MUSCLE‐p&nbsp;</td><td>T‐Coffee&nbsp;</td><td>NWNSI&nbsp;</td><td>FFTNS1&nbsp;</td><td>CLUSTALW&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>&nbsp;</td><td>+0.001&nbsp;</td><td>(0.15)&nbsp;</td><td>+ 0.005&nbsp;</td><td> +2 × 10 <sup>–6</sup>&nbsp;</td><td>+0.0002&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td> –6 × 10 <sup>–6</sup>&nbsp;</td><td>&nbsp;</td><td>(0.3)&nbsp;</td><td>(0.7)&nbsp;</td><td>+0.0002&nbsp;</td><td>+0.02&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>–0.0002&nbsp;</td><td>(0.4)&nbsp;</td><td>&nbsp;</td><td>(0.55)&nbsp;</td><td> +7 × 10 <sup>–5</sup>&nbsp;</td><td>+0.01&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –10 <sup>–7</sup>&nbsp;</td><td>&nbsp;</td><td>+0.0001&nbsp;</td><td>(0.06)&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>&nbsp;</td><td>–0.04&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>–0.008&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div content-id="GKH340TB8" class="table-modal"><table><tbody><tr><td>&nbsp;</td><td>MUSCLE&nbsp;</td><td>MUSCLE‐p&nbsp;</td><td>T‐Coffee&nbsp;</td><td>NWNSI&nbsp;</td><td>FFTNS1&nbsp;</td><td>CLUSTALW&nbsp;</td></tr><tr><td>MUSCLE&nbsp;</td><td>&nbsp;</td><td>+0.001&nbsp;</td><td>(0.15)&nbsp;</td><td>+ 0.005&nbsp;</td><td> +2 × 10 <sup>–6</sup>&nbsp;</td><td>+0.0002&nbsp;</td></tr><tr><td>MUSCLE‐p&nbsp;</td><td> –6 × 10 <sup>–6</sup>&nbsp;</td><td>&nbsp;</td><td>(0.3)&nbsp;</td><td>(0.7)&nbsp;</td><td>+0.0002&nbsp;</td><td>+0.02&nbsp;</td></tr><tr><td>T‐Coffee&nbsp;</td><td>–0.0002&nbsp;</td><td>(0.4)&nbsp;</td><td>&nbsp;</td><td>(0.55)&nbsp;</td><td> +7 × 10 <sup>–5</sup>&nbsp;</td><td>+0.01&nbsp;</td></tr><tr><td>NWNSI&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –10 <sup>–7</sup>&nbsp;</td><td>&nbsp;</td><td>+0.0001&nbsp;</td><td>(0.06)&nbsp;</td></tr><tr><td>FFTNS1&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>&nbsp;</td><td>–0.04&nbsp;</td></tr><tr><td>CLUSTALW&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td> –&lt;10 <sup>–10</sup>&nbsp;</td><td>–0.008&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p> Each entry in the table contains the <em>P</em>
 ‐value assigned by a Friedman rank test to the difference between a 
pair of methods. The upper‐right corner of the matrix is obtained from <em>Q</em> scores on BAliBASE, the lower‐left corner from <em>Q</em> scores on PREFAB. If the method to the left is ranked higher than the method above, the <em>P‐</em> value is preceded by +. If the method to the left is ranked lower, the <em>P</em> ‐value is preceded by –. If the <em>P</em>
 ‐value is &gt;0.05, the difference is not considered significant and is
 shown in parentheses. So, for example, MUSCLE ranks higher than 
T‐Coffee on PREFAB with <em>P</em> = 0.0002 and MUSCLE‐p higher than CLUSTALW on BAliBASE with <em>P</em> = 0.02. </p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/38266791" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination="38266792" id="38266792" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="GKH340C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C1" class="ref-content "><div class="label -label">1.</div><div class="citation mixed-citation">
            Wang,L. and Jiang,T. (
            <div class="year">1994</div>
            ) On the complexity of multiple sequence alignment. 
            <div class="source">J. Comput. Biol.</div>&nbsp;
            , 
            <div class="volume">1</div>
            , 
            <div class="fpage">337</div>
            –348.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Wang,L.+and+Jiang,T.+%28+1994+%29+On+the+complexity+of+multiple+sequence+alignment.+J.+Comput.+Biol.+,+1+,+337+%E2%80%93348." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C2" class="ref-content "><div class="label -label">2.</div><div class="citation mixed-citation">
            Waterman,M.S., Smith,T.F. and Beyer,W.A. (
            <div class="year">1976</div>
            ) Some biological sequence metrics. 
            <div class="source">Adv. Math.</div>&nbsp;
            , 
            <div class="volume">20</div>
            , 
            <div class="fpage">367</div>
            –387.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Waterman,M.S.,+Smith,T.F.+and+Beyer,W.A.+%28+1976+%29+Some+biological+sequence+metrics.+Adv.+Math.+,+20+,+367+%E2%80%93387." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C3" class="ref-content "><div class="label -label">3.</div><div class="citation mixed-citation">
            Hogeweg,P. and Hesper,B. (
            <div class="year">1984</div>
            ) The alignment of sets of sequences and the construction of phyletic trees: an integrated method. 
            <div class="source">J. Mol. Evol.</div>&nbsp;
            , 
            <div class="volume">20</div>
            , 
            <div class="fpage">175</div>
            –186.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Hogeweg,P.+and+Hesper,B.+%28+1984+%29+The+alignment+of+sets+of+sequences+and+the+construction+of+phyletic+trees:+an+integrated+method.+J.+Mol.+Evol.+,+20+,+175+%E2%80%93186." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C4" class="ref-content "><div class="label -label">4.</div><div class="citation mixed-citation">
            Feng,D.F. and Doolittle,R.F. (
            <div class="year">1987</div>
            ) Progressive sequence alignment as a prerequisite to correct phylogenetic trees. 
            <div class="source">J. Mol. Evol.</div>&nbsp;
            , 
            <div class="volume">25</div>
            , 
            <div class="fpage">351</div>
            –360.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Feng,D.F.+and+Doolittle,R.F.+%28+1987+%29+Progressive+sequence+alignment+as+a+prerequisite+to+correct+phylogenetic+trees.+J.+Mol.+Evol.+,+25+,+351+%E2%80%93360." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C5" class="ref-content "><div class="label -label">5.</div><div class="citation mixed-citation">
            Notredame,C., Higgins,D.G. and Heringa,J. (
            <div class="year">2000</div>
            ) T‐Coffee: a novel method for fast and accurate multiple sequence alignment. 
            <div class="source">J. Mol. Biol.</div>&nbsp;
            , 
            <div class="volume">302</div>
            , 
            <div class="fpage">205</div>
            –217.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Notredame,C.,+Higgins,D.G.+and+Heringa,J.+%28+2000+%29+T%E2%80%90Coffee:+a+novel+method+for+fast+and+accurate+multiple+sequence+alignment.+J.+Mol.+Biol.+,+302+,+205+%E2%80%93217." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C6" class="ref-content "><div class="label -label">6.</div><div class="citation mixed-citation">
            Notredame,C. (
            <div class="year">2002</div>
            ) Recent progress in multiple sequence alignment: a survey. 
            <div class="source">Pharmacogenomics</div>&nbsp;
            , 
            <div class="volume">3</div>
            , 
            <div class="fpage">131</div>
            –144.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Notredame,C.+%28+2002+%29+Recent+progress+in+multiple+sequence+alignment:+a+survey.+Pharmacogenomics+,+3+,+131+%E2%80%93144." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C7" class="ref-content "><div class="label -label">7.</div><div class="citation mixed-citation">
            Gotoh,O. (
            <div class="year">1996</div>
            ) Significant improvement in accuracy of multiple protein 
sequence alignments by iterative refinement as assessed by reference to 
structural alignments. 
            <div class="source">J. Mol. Biol.</div>&nbsp;
            , 
            <div class="volume">264</div>
            , 
            <div class="fpage">823</div>
            –838.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Gotoh,O.+%28+1996+%29+Significant+improvement+in+accuracy+of+multiple+protein+sequence+alignments+by+iterative+refinement+as+assessed+by+reference+to+structural+alignments.+J.+Mol.+Biol.+,+264+,+823+%E2%80%93838." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C8" class="ref-content "><div class="label -label">8.</div><div class="citation mixed-citation">
            Katoh,K., Misawa,K., Kuma,K. and Miyata,T. (
            <div class="year">2002</div>
            ) MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">30</div>
            , 
            <div class="fpage">3059</div>
            –3066.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Katoh,K.,+Misawa,K.,+Kuma,K.+and+Miyata,T.+%28+2002+%29+MAFFT:+a+novel+method+for+rapid+multiple+sequence+alignment+based+on+fast+Fourier+transform.+Nucleic+Acids+Res.+,+30+,+3059+%E2%80%933066." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C9" class="ref-content "><div class="label -label">9.</div><div class="citation mixed-citation">
            Edgar,R.C. (
            <div class="year">2004</div>
            ) Local homology recognition and distance measures in linear time using compressed amino acid alphabets. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">32</div>
            , 
            <div class="fpage">380</div>
            –385.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Edgar,R.C.+%28+2004+%29+Local+homology+recognition+and+distance+measures+in+linear+time+using+compressed+amino+acid+alphabets.+Nucleic+Acids+Res.+,+32+,+380+%E2%80%93385." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C10" class="ref-content "><div class="label -label">10.</div><div class="citation mixed-citation">
            Kimura,M. (
            <div class="year">1983</div>
            ) 
            <em>The Neutral Theory of Molecular Evolution</em>
            . Cambridge University Press.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Kimura,M.+%28+1983+%29+The+Neutral+Theory+of+Molecular+Evolution+.+Cambridge+University+Press." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C11" class="ref-content "><div class="label -label">11.</div><div class="citation mixed-citation">
            Sneath,P.H.A. and Sokal,R.R. (
            <div class="year">1973</div>
            ) 
            <em>Numerical Taxonomy</em>
            . Freeman, San Francisco.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sneath,P.H.A.+and+Sokal,R.R.+%28+1973+%29+Numerical+Taxonomy+.+Freeman,+San+Francisco." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C12" class="ref-content "><div class="label -label">12.</div><div class="citation mixed-citation">
            Saitou,N. and Nei,M. (
            <div class="year">1987</div>
            ) The neighbor‐joining method: a new method for reconstructing phylogenetic trees. 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;
            , 
            <div class="volume">4</div>
            , 
            <div class="fpage">406</div>
            –425.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Saitou,N.+and+Nei,M.+%28+1987+%29+The+neighbor%E2%80%90joining+method:+a+new+method+for+reconstructing+phylogenetic+trees.+Mol.+Biol.+Evol.+,+4+,+406+%E2%80%93425." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C13" class="ref-content "><div class="label -label">13.</div><div class="citation mixed-citation">
            Edgar,R.C. and Sjolander,K. (
            <div class="year">2004</div>
            ) A comparison of scoring functions for protein sequence profile alignment. 
            <div class="source">Bioinformatics</div>&nbsp;
            , DOI: 10.1093/bioinformatics/bth090.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Edgar,R.C.+and+Sjolander,K.+%28+2004+%29+A+comparison+of+scoring+functions+for+protein+sequence+profile+alignment.+Bioinformatics+,+DOI:+10.1093/bioinformatics/bth090." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C14" class="ref-content "><div class="label -label">14.</div><div class="citation mixed-citation">
            Sjolander,K., Karplus,K., Brown,M., Hughey,R., Krogh,A., Mian,I.S. and Haussler,D. (
            <div class="year">1996</div>
            ) Dirichlet mixtures: a method for improved detection of weak but significant protein sequence homology. 
            <div class="source">CABIOS</div>&nbsp;
            , 
            <div class="volume">12</div>
            , 
            <div class="fpage">327</div>
            –345.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sjolander,K.,+Karplus,K.,+Brown,M.,+Hughey,R.,+Krogh,A.,+Mian,I.S.+and+Haussler,D.+%28+1996+%29+Dirichlet+mixtures:+a+method+for+improved+detection+of+weak+but+significant+protein+sequence+homology.+CABIOS+,+12+,+327+%E2%80%93345." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C15" class="ref-content "><div class="label -label">15.</div><div class="citation mixed-citation">
            von Ohsen,N. and Zimmer,R. (
            <div class="year">2001</div>
            ) Improving profile–profile alignment via log average scoring. In Gascuel,O. and Moret,B.M.E. (eds), 
            <em>Algorithms in Bioinformatics,</em>
             
            <em>First International Workshop</em>
            , 
            <em>WABI 2001</em>
            . Springer‐Verlag, Berlin, Germany, pp. 11–26.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=von+Ohsen,N.+and+Zimmer,R.+%28+2001+%29+Improving+profile%E2%80%93profile+alignment+via+log+average+scoring.+In+Gascuel,O.+and+Moret,B.M.E.+%28eds%29,+Algorithms+in+Bioinformatics,+First+International+Workshop+,+WABI+2001+.+Springer%E2%80%90Verlag,+Berlin,+Germany,+pp.+11%E2%80%9326." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C16" class="ref-content "><div class="label -label">16.</div><div class="citation mixed-citation">
            Muller,T., Spang,R. and Vingron,M. (
            <div class="year">2002</div>
            ) Estimating amino acid substitution models: a comparison of
 Dayhoff’s estimator, the resolvent approach and a maximum likelihood 
method. 
            <div class="source">Mol. Biol. Evol.</div>&nbsp;
            , 
            <div class="volume">19</div>
            , 
            <div class="fpage">8</div>
            –13.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Muller,T.,+Spang,R.+and+Vingron,M.+%28+2002+%29+Estimating+amino+acid+substitution+models:+a+comparison+of+Dayhoff%E2%80%99s+estimator,+the+resolvent+approach+and+a+maximum+likelihood+method.+Mol.+Biol.+Evol.+,+19+,+8+%E2%80%9313." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C17" class="ref-content "><div class="label -label">17.</div><div class="citation mixed-citation">
            Brudno,M., Do,C.B., Cooper,G.M., Kim,M.F., Davydov,E., Green,E.D., Sidow,A. and Batzoglou,S. (
            <div class="year">2003</div>
            ) LAGAN and Multi‐LAGAN: efficient tools for large‐scale multiple alignment of genomic DNA. 
            <div class="source">Genome Res.</div>&nbsp;
            , 
            <div class="volume">13</div>
            , 
            <div class="fpage">721</div>
            –731.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Brudno,M.,+Do,C.B.,+Cooper,G.M.,+Kim,M.F.,+Davydov,E.,+Green,E.D.,+Sidow,A.+and+Batzoglou,S.+%28+2003+%29+LAGAN+and+Multi%E2%80%90LAGAN:+efficient+tools+for+large%E2%80%90scale+multiple+alignment+of+genomic+DNA.+Genome+Res.+,+13+,+721+%E2%80%93731." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C18" class="ref-content "><div class="label -label">18.</div><div class="citation mixed-citation">
            Hirosawa,M., Totoki,Y., Hoshida,M. and Ishikawa,M. (
            <div class="year">1995</div>
            ) Comprehensive study on iterative algorithms of multiple sequence alignment. 
            <div class="source">CABIOS</div>&nbsp;
            , 
            <div class="volume">11</div>
            , 
            <div class="fpage">13</div>
            –18.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Hirosawa,M.,+Totoki,Y.,+Hoshida,M.+and+Ishikawa,M.+%28+1995+%29+Comprehensive+study+on+iterative+algorithms+of+multiple+sequence+alignment.+CABIOS+,+11+,+13+%E2%80%9318." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C19" class="ref-content "><div class="label -label">19.</div><div class="citation mixed-citation">
            Thompson,J.D., Plewniak,F. and Poch,O. (
            <div class="year">1999</div>
            ) BAliBASE: a benchmark alignment database for the evaluation of multiple alignment programs. 
            <div class="source">Bioinformatics</div>&nbsp;
            , 
            <div class="volume">15</div>
            , 
            <div class="fpage">87</div>
            –88.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Thompson,J.D.,+Plewniak,F.+and+Poch,O.+%28+1999+%29+BAliBASE:+a+benchmark+alignment+database+for+the+evaluation+of+multiple+alignment+programs.+Bioinformatics+,+15+,+87+%E2%80%9388." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C20" class="ref-content "><div class="label -label">20.</div><div class="citation mixed-citation">
            Bahr,A., Thompson,J.D., Thierry,J.C. and Poch,O. (
            <div class="year">2001</div>
            ) BAliBASE (Benchmark Alignment dataBASE): enhancements for 
repeats, transmembrane sequences and circular permutations. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">29</div>
            , 
            <div class="fpage">323</div>
            –326.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Bahr,A.,+Thompson,J.D.,+Thierry,J.C.+and+Poch,O.+%28+2001+%29+BAliBASE+%28Benchmark+Alignment+dataBASE%29:+enhancements+for+repeats,+transmembrane+sequences+and+circular+permutations.+Nucleic+Acids+Res.+,+29+,+323+%E2%80%93326." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C21" class="ref-content "><div class="label -label">21.</div><div class="citation mixed-citation">
            Van Walle,I., Lasters,I. and Wyns,L. (
            <div class="year">2004</div>
            ) Align‐m—a new algorithm for multiple alignment of highly divergent sequences. 
            <div class="source">Bioinformatics</div>&nbsp;
            , DOI: 10.1093/bioinformatics/bth116.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Van+Walle,I.,+Lasters,I.+and+Wyns,L.+%28+2004+%29+Align%E2%80%90m%E2%80%94a+new+algorithm+for+multiple+alignment+of+highly+divergent+sequences.+Bioinformatics+,+DOI:+10.1093/bioinformatics/bth116." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C22" class="ref-content "><div class="label -label">22.</div><div class="citation mixed-citation">
            Schultz,J., Milpetz,F., Bork,P. and Ponting,C.P. (
            <div class="year">1998</div>
            ) SMART, a simple modular architecture research tool: identification of signaling domains. 
            <div class="source">Proc. Natl Acad. Sci. USA</div>&nbsp;
            , 
            <div class="volume">95</div>
            , 
            <div class="fpage">5857</div>
            –5864.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Schultz,J.,+Milpetz,F.,+Bork,P.+and+Ponting,C.P.+%28+1998+%29+SMART,+a+simple+modular+architecture+research+tool:+identification+of+signaling+domains.+Proc.+Natl+Acad.+Sci.+USA+,+95+,+5857+%E2%80%935864." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C23" class="ref-content "><div class="label -label">23.</div><div class="citation mixed-citation">
            Schultz,J., Copley,R.R., Doerks,T., Ponting,C.P. and Bork,P. (
            <div class="year">2000</div>
            ) SMART: a web‐based tool for the study of genetically mobile domains. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">28</div>
            , 
            <div class="fpage">231</div>
            –234.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Schultz,J.,+Copley,R.R.,+Doerks,T.,+Ponting,C.P.+and+Bork,P.+%28+2000+%29+SMART:+a+web%E2%80%90based+tool+for+the+study+of+genetically+mobile+domains.+Nucleic+Acids+Res.+,+28+,+231+%E2%80%93234." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C24" class="ref-content "><div class="label -label">24.</div><div class="citation mixed-citation">
            Ponting,C.P., Schultz,J., Milpetz,F. and Bork,P. (
            <div class="year">1999</div>
            ) SMART: identification and annotation of domains from signalling and extracellular protein sequences. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">27</div>
            , 
            <div class="fpage">229</div>
            –332.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Ponting,C.P.,+Schultz,J.,+Milpetz,F.+and+Bork,P.+%28+1999+%29+SMART:+identification+and+annotation+of+domains+from+signalling+and+extracellular+protein+sequences.+Nucleic+Acids+Res.+,+27+,+229+%E2%80%93332." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C25" class="ref-content "><div class="label -label">25.</div><div class="citation mixed-citation">
            Thompson,J.D., Higgins,D.G. and Gibson,T.J. (
            <div class="year">1994</div>
            ) CLUSTAL W: improving the sensitivity of progressive 
multiple sequence alignment through sequence weighting, 
position‐specific gap penalties and weight matrix choice. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">22</div>
            , 
            <div class="fpage">4673</div>
            –4680.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Thompson,J.D.,+Higgins,D.G.+and+Gibson,T.J.+%28+1994+%29+CLUSTAL+W:+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting,+position%E2%80%90specific+gap+penalties+and+weight+matrix+choice.+Nucleic+Acids+Res.+,+22+,+4673+%E2%80%934680." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C26" class="ref-content "><div class="label -label">26.</div><div class="citation mixed-citation">
            Brenner,S.E., Koehl,P. and Levitt,M. (
            <div class="year">2000</div>
            ) The ASTRAL compendium for protein structure and sequence analysis. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">28</div>
            , 
            <div class="fpage">254</div>
            –256.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Brenner,S.E.,+Koehl,P.+and+Levitt,M.+%28+2000+%29+The+ASTRAL+compendium+for+protein+structure+and+sequence+analysis.+Nucleic+Acids+Res.+,+28+,+254+%E2%80%93256." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C27" class="ref-content "><div class="label -label">27.</div><div class="citation mixed-citation">
            Murzin,A.G., Brenner,S.E., Hubbard,T. and Chothia,C. (
            <div class="year">1995</div>
            ) SCOP: a structural classification of proteins database for the investigation of sequences and structures. 
            <div class="source">J. Mol. Biol.</div>&nbsp;
            , 
            <div class="volume">247</div>
            , 
            <div class="fpage">536</div>
            –540.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Murzin,A.G.,+Brenner,S.E.,+Hubbard,T.+and+Chothia,C.+%28+1995+%29+SCOP:+a+structural+classification+of+proteins+database+for+the+investigation+of+sequences+and+structures.+J.+Mol.+Biol.+,+247+,+536+%E2%80%93540." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C28" class="ref-content "><div class="label -label">28.</div><div class="citation mixed-citation">
            Boutonnet,N.S., Rooman,M.J., Ochagavia,M.E., Richelle,J. and Wodak,S.J. (
            <div class="year">1995</div>
            ) Optimal protein structure alignments by multiple linkage clustering: application to distantly related proteins. 
            <div class="source">Protein Eng.</div>&nbsp;
            , 
            <div class="volume">8</div>
            , 
            <div class="fpage">647</div>
            –662.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Boutonnet,N.S.,+Rooman,M.J.,+Ochagavia,M.E.,+Richelle,J.+and+Wodak,S.J.+%28+1995+%29+Optimal+protein+structure+alignments+by+multiple+linkage+clustering:+application+to+distantly+related+proteins.+Protein+Eng.+,+8+,+647+%E2%80%93662." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C29" class="ref-content "><div class="label -label">29.</div><div class="citation mixed-citation">
            Shindyalov,I.N. and Bourne,P.E. (
            <div class="year">1998</div>
            ) Protein structure alignment by incremental combinatorial extension (CE) of the optimal path. 
            <div class="source">Protein Eng.</div>&nbsp;
            , 
            <div class="volume">11</div>
            , 
            <div class="fpage">739</div>
            –747.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Shindyalov,I.N.+and+Bourne,P.E.+%28+1998+%29+Protein+structure+alignment+by+incremental+combinatorial+extension+%28CE%29+of+the+optimal+path.+Protein+Eng.+,+11+,+739+%E2%80%93747." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C30" class="ref-content "><div class="label -label">30.</div><div class="citation mixed-citation">
            Fischer,D., Barret,C., Bryson,K., Elofsson,A., Godzik,A., 
Jones,D., Karplus,K.J., Kelley,L.A., MacCallum,R.M., Pawowski,K., 
Rost,B., Rychlewski,L. and Sternberg,M. (
            <div class="year">1999</div>
            ) CAFASP‐1: critical assessment of fully automated structure prediction methods. 
            <div class="source">Proteins</div>&nbsp;
            , Suppl. 
            <div class="volume">3</div>
            , 
            <div class="fpage">209</div>
            –217.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Fischer,D.,+Barret,C.,+Bryson,K.,+Elofsson,A.,+Godzik,A.,+Jones,D.,+Karplus,K.J.,+Kelley,L.A.,+MacCallum,R.M.,+Pawowski,K.,+Rost,B.,+Rychlewski,L.+and+Sternberg,M.+%28+1999+%29+CAFASP%E2%80%901:+critical+assessment+of+fully+automated+structure+prediction+methods.+Proteins+,+Suppl.+3+,+209+%E2%80%93217." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C31" class="ref-content "><div class="label -label">31.</div><div class="citation mixed-citation">
            Eidhammer,I., Jonassen,I. and Taylor,W.R. (
            <div class="year">2000</div>
            ) Structure comparison and structure patterns. 
            <div class="source">J. Comput. Biol.</div>&nbsp;
            , 
            <div class="volume">7</div>
            , 
            <div class="fpage">685</div>
            –716.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Eidhammer,I.,+Jonassen,I.+and+Taylor,W.R.+%28+2000+%29+Structure+comparison+and+structure+patterns.+J.+Comput.+Biol.+,+7+,+685+%E2%80%93716." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C32" class="ref-content "><div class="label -label">32.</div><div class="citation mixed-citation">
            Sauder,J.M., Arthur,J.W. and Dunbrack,R.L.,Jr (
            <div class="year">2000</div>
            ) Large‐scale comparison of protein sequence alignment algorithms with structure alignments. 
            <div class="source">Proteins</div>&nbsp;
            , 
            <div class="volume">40</div>
            , 
            <div class="fpage">6</div>
            –22.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sauder,J.M.,+Arthur,J.W.+and+Dunbrack,R.L.,Jr+%28+2000+%29+Large%E2%80%90scale+comparison+of+protein+sequence+alignment+algorithms+with+structure+alignments.+Proteins+,+40+,+6+%E2%80%9322." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C33" class="ref-content "><div class="label -label">33.</div><div class="citation mixed-citation">
            Leplae,R. and Hubbard,T.J. (
            <div class="year">2002</div>
            ) MaxBench: evaluation of sequence and structure comparison methods. 
            <div class="source">Bioinformatics</div>&nbsp;
            , 
            <div class="volume">18</div>
            , 
            <div class="fpage">494</div>
            –495.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Leplae,R.+and+Hubbard,T.J.+%28+2002+%29+MaxBench:+evaluation+of+sequence+and+structure+comparison+methods.+Bioinformatics+,+18+,+494+%E2%80%93495." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C34" class="ref-content "><div class="label -label">34.</div><div class="citation mixed-citation">
            Sadreyev,R. and Grishin,N. (
            <div class="year">2003</div>
            ) COMPASS: a tool for comparison of multiple protein alignments with assessment of statistical significance. 
            <div class="source">J. Mol. Biol.</div>&nbsp;
            , 
            <div class="volume">326</div>
            , 
            <div class="fpage">317</div>
            –336.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sadreyev,R.+and+Grishin,N.+%28+2003+%29+COMPASS:+a+tool+for+comparison+of+multiple+protein+alignments+with+assessment+of+statistical+significance.+J.+Mol.+Biol.+,+326+,+317+%E2%80%93336." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C35" class="ref-content "><div class="label -label">35.</div><div class="citation mixed-citation">
            Edgar,R.C. and Sjolander,K. (
            <div class="year">2004</div>
            ) COACH: profile‐profile alignment of protein families using hidden Markov models. 
            <div class="source">Bioinformatics</div>&nbsp;
            , DOI: 10.1093/bioinformatics/bth091.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Edgar,R.C.+and+Sjolander,K.+%28+2004+%29+COACH:+profile%E2%80%90profile+alignment+of+protein+families+using+hidden+Markov+models.+Bioinformatics+,+DOI:+10.1093/bioinformatics/bth091." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C36" class="ref-content "><div class="label -label">36.</div><div class="citation mixed-citation">
            Holm,L. and Sander,C. (
            <div class="year">1998</div>
            ) Touring protein fold space with Dali/FSSP. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">26</div>
            , 
            <div class="fpage">316</div>
            –319.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Holm,L.+and+Sander,C.+%28+1998+%29+Touring+protein+fold+space+with+Dali/FSSP.+Nucleic+Acids+Res.+,+26+,+316+%E2%80%93319." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C37" class="ref-content "><div class="label -label">37.</div><div class="citation mixed-citation">
            Altschul,S.F., Madden,T.L., Schaffer,A.A., Zhang,J., Zhang,Z., Miller,W. and Lipman,D.J. (
            <div class="year">1997</div>
            ) Gapped BLAST and PSI‐BLAST: a new generation of protein database search programs. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">25</div>
            , 
            <div class="fpage">3389</div>
            –3402.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Altschul,S.F.,+Madden,T.L.,+Schaffer,A.A.,+Zhang,J.,+Zhang,Z.,+Miller,W.+and+Lipman,D.J.+%28+1997+%29+Gapped+BLAST+and+PSI%E2%80%90BLAST:+a+new+generation+of+protein+database+search+programs.+Nucleic+Acids+Res.+,+25+,+3389+%E2%80%933402." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C38" class="ref-content "><div class="label -label">38.</div><div class="citation mixed-citation">
            Schaffer,A.A., Aravind,L., Madden,T.L., Shavirin,S., Spouge,J.L., Wolf,Y.I., Koonin,E.V. and Altschul,S.F. (
            <div class="year">2001</div>
            ) Improving the accuracy of PSI‐BLAST protein database 
searches with composition‐based statistics and other refinements. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">29</div>
            , 
            <div class="fpage">2994</div>
            –3005.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Schaffer,A.A.,+Aravind,L.,+Madden,T.L.,+Shavirin,S.,+Spouge,J.L.,+Wolf,Y.I.,+Koonin,E.V.+and+Altschul,S.F.+%28+2001+%29+Improving+the+accuracy+of+PSI%E2%80%90BLAST+protein+database+searches+with+composition%E2%80%90based+statistics+and+other+refinements.+Nucleic+Acids+Res.+,+29+,+2994+%E2%80%933005." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C39" class="ref-content "><div class="label -label">39.</div><div class="citation mixed-citation">
            Pruitt,K.D., Tatusova,T. and Maglott,D.R. (
            <div class="year">2003</div>
            ) NCBI Reference Sequence project: update and current status. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">31</div>
            , 
            <div class="fpage">34</div>
            –37.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Pruitt,K.D.,+Tatusova,T.+and+Maglott,D.R.+%28+2003+%29+NCBI+Reference+Sequence+project:+update+and+current+status.+Nucleic+Acids+Res.+,+31+,+34+%E2%80%9337." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C40" class="ref-content "><div class="label -label">40.</div><div class="citation mixed-citation">
            Thompson,J.D., Plewniak,F. and Poch,O. (
            <div class="year">1999</div>
            ) A comprehensive comparison of multiple sequence alignment programs. 
            <div class="source">Nucleic Acids Res.</div>&nbsp;
            , 
            <div class="volume">27</div>
            , 
            <div class="fpage">2682</div>
            –2690.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Thompson,J.D.,+Plewniak,F.+and+Poch,O.+%28+1999+%29+A+comprehensive+comparison+of+multiple+sequence+alignment+programs.+Nucleic+Acids+Res.+,+27+,+2682+%E2%80%932690." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C41" class="ref-content "><div class="label -label">41.</div><div class="citation mixed-citation">
            O’Sullivan,O., Zehnder,M., Higgins,D., Bucher,P., Grosdidier,A. and Notredame,C. (
            <div class="year">2003</div>
            ) APDB: a novel measure for benchmarking sequence alignment methods without reference alignments. 
            <div class="source">Bioinformatics</div>&nbsp;
            , 
            <div class="volume">19</div>
             Suppl. 1, 
            <div class="fpage">I215</div>
            –I221.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=O%E2%80%99Sullivan,O.,+Zehnder,M.,+Higgins,D.,+Bucher,P.,+Grosdidier,A.+and+Notredame,C.+%28+2003+%29+APDB:+a+novel+measure+for+benchmarking+sequence+alignment+methods+without+reference+alignments.+Bioinformatics+,+19+Suppl.+1,+I215+%E2%80%93I221." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C42" class="ref-content "><div class="label -label">42.</div><div class="citation mixed-citation">
            Friedman,M. (
            <div class="year">1937</div>
            ) The use of ranks to avoid the assumption of normality implicit in the analysis of variance. 
            <div class="source">J. Am. Stat. Assoc.</div>&nbsp;
            , 
            <div class="volume">32</div>
            , 
            <div class="fpage">675</div>
            –701.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Friedman,M.+%28+1937+%29+The+use+of+ranks+to+avoid+the+assumption+of+normality+implicit+in+the+analysis+of+variance.+J.+Am.+Stat.+Assoc.+,+32+,+675+%E2%80%93701." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C43" class="ref-content "><div class="label -label">43.</div><div class="citation mixed-citation">
            Stoye,J., Evers,D. and Meyer,F. (
            <div class="year">1998</div>
            ) Rose: generating sequence families. 
            <div class="source">Bioinformatics</div>&nbsp;
            , 
            <div class="volume">14</div>
            , 
            <div class="fpage">157</div>
            –163.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Stoye,J.,+Evers,D.+and+Meyer,F.+%28+1998+%29+Rose:+generating+sequence+families.+Bioinformatics+,+14+,+157+%E2%80%93163." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKH340C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKH340C44" class="ref-content "><div class="label -label">44.</div><div class="citation mixed-citation">
            Sjolander,K. (
            <div class="year">1998</div>
            ) Phylogenetic inference in protein superfamilies: analysis of SH2 domains. 
            <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol.</div>&nbsp;
            , 
            <div class="volume">6</div>
            , 
            <div class="fpage">165</div>
            –174.
          <div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sjolander,K.+%28+1998+%29+Phylogenetic+inference+in+protein+superfamilies:+analysis+of+SH2+domains.+Proc.+Int.+Conf.+Intell.+Syst.+Mol.+Biol.+,+6+,+165+%E2%80%93174." target="">Google Scholar</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix" style="display: none;"></div>

        
<div class="copyright copyright-statement">Oxford University Press</div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-taxonomies">
        <div class="article-metadata">
            <div class="article-metadata-taxonomies-title">Subject</div>
                <a href="https://academic.oup.com/nar/search-results?page=1&amp;tax=narese/24">Computational Methods</a>
        </div>
        </div>
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Articles">Articles</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2380623&amp;xsltPath=%7E/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

<div class="comment-body__wrap">
    <div id="usercomments" class="comment-body__title">
        <h2>Comments</h2>
    </div>
    <div class="comment-body__number">
        0 Comments
    </div>
    <div class="submitCommentPlaceHolder"><!-- UserComment.cshtml placed here by JS --><div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
            <button id="comment-modal-opener" class="btn comment-body__button" data-reveal="">Add comment</button>
    <div id="divCommentModal" class="comment-form__modal medium reveal-modal" data-reveal="">

        <button class="comment-form__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close comment form modal</span>
        </button>

        <div class="preloader-wrap hide">
            <div class="preloader">
                <div class="spinner"></div>
            </div>
        </div>

        <div id="TermsAndConditionsContainer" class="comment-form__wrap" data-reveal=""></div>

            <div id="termsAgreeRow" class="comment-form__row--last terms-conditions">
                <input id="termsAndConditionsCheckbox" class="comment-form__input comment-form__checkbox" title="Terms and Conditions" required="required" type="checkbox">
                <label class="comment-form__required-statement required" for="termsAndConditionsCheckbox">I agree to the terms and conditions.</label>
                <label class="lblTermsAndConditions errors comment-form__error-label" for="termsAndConditionsCheckbox">
                    You must accept the terms and conditions.
                </label>
            </div>

        <div id="termsButtonRow" class="termsButtonRow comment-form__row">
            <button id="btnUserAgreeTerms" class="btnUserAgreeTerms btn btn-grey">Add comment</button>
            <button href="#" class="btnModalCancel btn btn-grey">Cancel</button>
        </div>


        <div id="divUserCommentForm" class="comment-form__wrap">

            <div class="comment-form__title">Submit a comment</div>



                <div class="comment-form__row">
                    <label class="comment-form__input-label required" for="Author">Name</label>

                    <input class="comment-form__input comment-form__textbox required" dataauthorvalue="" id="Author" maxlength="2000" name="Author" required="required" type="text">
                </div>


                <div class="comment-form__row">
                    <label class="comment-form__input-label required" for="AffInstitution">Affiliations</label>
                    <input class="comment-form__input comment-form__textbox required" id="AffInstitution" maxlength="200" name="AffInstitution" required="required" type="text">
                </div>



            <div class="comment-form__row">
                <label class="comment-form__input-label required" for="Title">Comment title</label>
                <input class="comment-form__input comment-form__textbox required" id="Title" maxlength="250" name="Title" required="required" type="text">
            </div>

            <div class="comment-form__row">
                <label class="comment-form__input-label required" for="Comment">Comment</label>
                <textarea class="comment-form__input comment-form__textarea required" cols="20" id="Comment" maxlength="@Model.Config.CommentBodyCharLimit" name="Comment" required="required" rows="2"></textarea>
                </div>


                <div class="captchaAlert comment-form__error-label">
                    You have entered an invalid code
                </div>

                <div class="submitRow comment-form__row">
                    <button id="btnUserCommentSubmit" class="btnUserCommentSubmit btn btn-grey">Submit</button>
                    <button href="#" class="btnModalCancel btn btn-grey">Cancel</button>
                </div>

            </div><!-- /.comment-form__wrap -->

            <div class="content-form__response comment-form__row">
                
                <div class="content-form__response--success">
                    <div class="alertmsg content-form__response-text" id="displayAlert">
                        Thank you for submitting a comment on this 
article. Your comment will be reviewed and published at the journal's 
discretion. Please check for further notifications by email.
                    </div>
                    <button class="btnCommentModalExit content-form__response-button btn btn-grey">Close</button>
                </div>
                
                            
                <div id="divError" class="errors content-form__response--error"></div>
            </div><!-- /.comment-form__response -->

        </div><!-- /.comment-form__modal -->
                        <script type="text/javascript">

                            /*******************************************************************************
                             * JS here is only being used to assign variables from values in the model
                             * logic should be implemented in external JS files, like client.script.js
                             *******************************************************************************/

                            var SCM = SCM || {};
                            SCM.UserCommentView = SCM.UserCommentView || {};

                            SCM.UserCommentView.articleId = '2380623';
                            SCM.UserCommentView.isUserLoggedIn = 'False';
                            SCM.UserCommentView.divId = 'WidgetNoAccessModalSmallOUP_UserComment_Article';
                            SCM.UserCommentView.bShowDisclosure = 'false';
                            SCM.UserCommentView.isNotModerated = 'false';
                            SCM.UserCommentView.showAffiliations = 'true';
                            SCM.UserCommentView.InstanceName = 'OUP_UserComment_Article';
                        </script>

    </div></div>
    
    
</div><!-- /.comment-body__wrap -->

    </div>
    

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad2_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad2_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad2_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">13,571</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000220487200025&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">16,900</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/nar/gkh340" data-uuid="a335b083-e548-ea14-c3af-ae7f12db2113"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=272285" style="display:inline-block;">
        <img alt="Article has an altmetric score of 31" src="https://badges.altmetric.com/?size=128&amp;score=31&amp;types=aaaabtwd" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2380623"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                <div class="userAlert alertType-33">
                    <a role="link" aria-label="Subject alert" data-userloggedin="False" data-alerttype="33">Subject alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
                <h3 class="article-widget-head">More on this topic</h3>
    <div class="widget-related-content">
            <div class="article-title"><a href="https://academic.oup.com/nar/article/30/14/3059/2904316">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/38/suppl_2/W29/1095729">SATCHMO-JS: a webserver for simultaneous protein multiple sequence alignment and phylogenetic tree construction</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/42/D1/D897/1057169">PhylomeDB v4: zooming into the plurality of evolutionary histories of a genome</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/46/14/e83/4990634">MeShClust: an intelligent tool for clustering DNA sequences</a></div>
    </div>

    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000220487200025&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fnar%2farticle%2f32%2f5%2f1792%2f2380623">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-RelatedPubMed widget-instance-Article_RightRailB0Article_RightRail_RelatedPubMed">
        <h3 class="article-widget-head">Related articles in PubMed</h3>
<div class="widget-related-pubmed">
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30269237" id="article_ArticleTitle" target="">The use of Electronic Health Records to Support Population Health: A Systematic Review of the Literature.</a></div>
                <!-- <div class="citation"><span>J Med Syst</span> 2018;42(11):214.</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30268604" id="article_ArticleTitle" target="">Symposium review: Omics in dairy and animal science-Promise, potential, and pitfalls.</a></div>
                <!-- <div class="citation"><span>J Dairy Sci</span> 2018.</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30268485" id="article_ArticleTitle" target="">Carving
 out another slice of the pie: Exceptional response to single agent 
imatinib in an asian female never-smoker with advanced NSCLC with a 
de-novo PDGFR-Î± N848â€‰K mutation.</a></div>
                <!-- <div class="citation"><span>Lung Cancer</span> 2018;124():86-89.</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30268356" id="article_ArticleTitle" target="">Analysis
 of impingement-free range of motion of the glenohumeral joint after 
reverse total shoulder arthroplasty using three different implant 
models.</a></div>
                <!-- <div class="citation"><span>J Orthop Sci</span> 2018.</div> -->
            </div>
        </div>
</div>
<!-- <div class="article-widget-foot">
    <div class="view-more">
            <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=((sequence+AND+alignment)+OR+(computer+AND+software)+OR+(benchmarking)+OR+(computers))" id="Config_ViewMoreLink" class="view-more-link" target="_blank">View More</a>
    </div>
</div> -->
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000220487200025&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (16900)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F32%2F5%2F1792%2F2380623" target="">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href="https://academic.oup.com/nar/CrossRef-CitedBy/2380623">CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky934/5133625">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky934/5133625">Structural
 basis for recognition and repair of the 3′-phosphate by NExo, a base 
excision DNA repair nuclease from  Neisseria meningitidis</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky934/5133625">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky912/5133664">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky912/5133664">Substrate specificity of human apurinic/apyrimidinic endonuclease APE1 in the nucleotide incision repair pathway</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky912/5133664">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky917/5133667">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky917/5133667">CLIPick: a sensitive peak caller for expression-based deconvolution of HITS-CLIP signals</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky917/5133667">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky921/5133660">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky921/5133660">Modulation of  MKNK2  alternative splicing by splice-switching oligonucleotides as a novel approach for glioblastoma treatment</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky921/5133660">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky924/5133661">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky924/5133661">Structural,
 functional and biological insights into the role of  Mycobacterium 
tuberculosis  VapBC11 toxin–antitoxin system: targeting a tRNase to 
tackle mycobacterial adaptation</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky924/5133661">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad3_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad3_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad3_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="MUSCLE: multiple sequence alignment with high accuracy and high throughput" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/nar/gkh340");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad4_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad4_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad4_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/nar/pages/About">About Nucleic Acids Research</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/nar/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/nar/pages/Policies">Policies</a>
                            </li>
                            <li class="link-4">
                                <a href="https://academic.oup.com/nar/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/nar_open">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="https://www.linkedin.com/in/nar-nucleic-acids-research-5297133b">LinkedIn</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-4">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-NucleicAcidsResearch" class="journal-footer-affiliations-logo" src="nar_f1248941275.svg" alt="Nucleic Acids Research">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1362-4962</li>
                                            <li>Print ISSN 0305-1048</li>
                    <li>Copyright © 2018 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f32%2f5%2f1792%2f2380623">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="footerJS" type="text/javascript"></script>
<script src="OupCookiePolicyJS" type="text/javascript"></script>
<script src="MathJax.js" type="text/javascript"></script>
<script src="chartist.js" type="text/javascript"></script>
<script src="ArticlePageJS" type="text/javascript"></script>
<script src="addthis_widget.js" type="text/javascript"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh821" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.8296555419636504&amp;iit=1539788477462&amp;tmr=load%3D1539788477428%26core%3D1539788477456%26main%3D1539788477458%26ifr%3D1539788477464&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F32%2F5%2F1792%2F2380623&amp;href=https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F32%2F5%2F1792%2F2380623&amp;dt=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput%20%7C%20Nucleic%20Acids%20Research%20%7C%20Oxford%20Academic&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5bc74ebd8a191783&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DMUSCLE%253A%2520multiple%2520sequence%2520alignment%2520with%2520high%2520accuracy%2520and%2520high%2520throughput%2520%257C%2520Nucleic%2520Acids%2520Research%2520%257C%2520Oxford%2520Academic%26description%3DAbstract.%2520%2520%2520We%2520describe%2520MUSCLE%252C%2520a%2520new%2520computer%2520program%2520for%2520creating%2520multiple%2520alignments%2520of%2520protein%2520sequences.%2520Elements%2520of%2520the%2520algorithm%2520include%2520fast%2520distance%2520es%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fnar%252Farticle%252F32%252F5%252F1792%252F2380623%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fnar%252FIssue%252F46%252F18%252F2%252Fm_cover.gif%253FExpires%253D2147483647%2526Signature%253DeV%7Ehm087cge5g8xBrr8Ad3gSMmcSfY8S8fYMMxVMyebMOzGEOd-Q8KiPx0XalwBTcA6jZy29jYCCVng%7EgYiA2bRURSHYiJJmFdS3MB%7EzxLd3wDRhaCxH2ogC-EC%7ENIicXdJr6ACFFc%7E6Slb3Z4F3GPnTUmF1TfmETtWJ9ZljeOrPJCq00DIOHPwcSP5C6wnJSfquTOINmA8u8gutkWbmGOwPFaTa37wgWdMQ3sPlXeaTD7x3QVAvxU1cvpttJcvurNH6ZD1KkAoWB3fbxyqXHIEHlDf-ZLeY2k321uAAnOvcvm5UKznRhGSheuGJ5tbEO3eOOn2s4QmfHPn3G8VWYA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2004-03-01%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.3.31-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
<script src="api.js" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="gkh340"></script></body></html>