<!DOCTYPE html>
<html style="height: 100%;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<meta name="viewport" content="width=device-width,  initial-scale=1">
<meta content="text/html; charset=utf-8">
<meta name="robots" content="all">
<meta name="alexaVerifyID" content="prVlK2AuSBfK43ZTfG7WLg5ergs">
<meta name="yandex-verification" content="0e61e7ac671ee7a3">
<link rel="shortcut icon" href="https://www.omicsonline.org/images/omics-international-animated-favicon.gif">
<link rel="stylesheet" href="bootstrap.css">
<link href="css.css" rel="stylesheet">
<link rel="stylesheet" href="all.css">
<link rel="stylesheet" href="global.css">
<link rel="stylesheet" href="journal-bootstrap.css">
<link rel="stylesheet" href="journal-external-style.css">
<link rel="stylesheet" href="journal-full-text.css">
<link rel="stylesheet" href="styles.css">
<link rel="stylesheet" type="text/css" href="main-coolautosuggest.css">
<script type="text/javascript" async="" src="https://d31qbv1cthcecs.cloudfront.net/atrk.js"></script><script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/javascript" async="" src="https://d31qbv1cthcecs.cloudfront.net/atrk.js"></script><script src="jquery.js" type="text/javascript"></script>
<script src="https://www.omicsonline.org/js/jquery.coolautosuggest.js" type="text/javascript"></script>
<script src="https://www.omicsonline.org/js/jquery.coolfieldset.js" type="text/javascript"></script>

<script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-29271993-2" type="text/javascript"></script>
<script type="text/javascript">
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-29271993-2');
</script>

<script type="text/javascript">
_atrk_opts = { atrk_acct:"AQs4j1a4ZP00a6", domain:"omicsonline.org",dynamic: true};
(function() { var as = document.createElement('script'); as.type = 'text/javascript'; as.async = true; as.src = "https://d31qbv1cthcecs.cloudfront.net/atrk.js"; var s = document.getElementsByTagName('script')[0];s.parentNode.insertBefore(as, s); })();
</script>
<noscript>
<img src="https://d5nxst8fruw4z.cloudfront.net/atrk.gif?account=AQs4j1a4ZP00a6" style="display:none" height="1" width="1" alt="alexa"/>
</noscript>

<script type="application/ld+json">
{
  "@context" : "http://schema.org",
  "@type" : "Article",
  "name" : "Molecular Markers in Phylogenetic Studies-A Review",
  "author" : {
    "@type" : "Person",
    "name" : "Anand Patwardhan, Samit Ray and Amit Roy"
  },
  "datePublished" : "2014-08-29",
  "url" : "https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.pdf"
}
</script>
<title>Molecular Markers in Phylogenetic Studies-A Review | OMICS International</title>
<meta name="keywords" content="Molecular phylogeny; Phylogenetic tree; Molecular marker; Molecular clock; Bar code of Life, omics, open access, omics publishing group, open access publisher, open access publishers, open access publications, open access journals, open access artcles, omics group, omicsonline">
<meta name="description" content="Uses of molecular markers in the phylogenetic studies of various organisms have become increasingly important in recent times. This review gives an overview of different ..">
<link rel="canonical" href="https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php?aid=30965"><meta name="citation_publisher" content="OMICS International">
<meta name="citation_journal_title" content="Journal of Phylogenetics &amp; Evolutionary Biology">
<meta name="citation_title" content="Molecular Markers in Phylogenetic Studies-A Review">
<meta name="citation_author" content="An">
<meta name="citation_author" content="Patwardhan">
<meta name="citation_author" content="Samit Ray">
<meta name="citation_author" content="Amit Roy">
<meta name="citation_year" content="2014">
<meta name="citation_volume" content="2">
<meta name="citation_issue" content="2">
<meta name="citation_doi" content="10.4172/2329-9002.1000131">
<meta name="citation_issn" content="2329-9002">
<meta name="citation_publication_date" content="2014/08/29">
<meta name="citation_firstpage" content="1">
<meta name="citation_lastpage" content="9">
<meta name="citation_fulltext_html_url" content="https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php?aid=30965">
<meta name="citation_pdf_url" content="https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.pdf">
<meta name="citation_abstract_html_url" content="https://www.omicsonline.org/peer-reviewed/molecular-markers-in-phylogenetic-studiesa-reviewp-30965.html">
<meta name="citation_abstract" content="Uses of molecular markers in the phylogenetic studies of various organisms have become increasingly important in recent times. This review gives an overview of different molecular markers employed by researchers for the purpose of phylogenetic studies. Availability of fast DNA sequencing techniques along with the development of robust statistical analysis methods, provided a new momentum to this field. In this context, utility of different nuclear encoded genes (like 16S rRNA, 5S rRNA, 28S rRNA) mitochondrial (cytochrome oxidase, mitochondrial 12S, cytochrome b, control region) and few chloroplast encoded genes (like rbcL, matK, rpl16) are discussed. Criteria for choosing suitable molecular markers and steps leading to the construction of phylogenetic trees have been discussed. Although widely practised even now, traditional morphology based systems of classification of organisms have some limitations. On the other hand it appears that the use of molecular markers, though relatively recent in popularity and are not free entirely of flaws, can complement the traditional morphology based method for phylogenetic studies.">
<meta name="dc.source" content="Journal of Phylogenetics &amp; Evolutionary Biology 2014 2:2">
<meta name="dc.format" content="text/html">
<meta name="dc.publisher" content="OMICS International">
<meta name="dc.date" content="2014-08-29">
<meta name="dc.type" content="OriginalPaper">
<meta name="dc.language" content="En">
<meta name="dc.copyright" content="2014 The Author(s).">
<meta name="prism.issn" content="
2329-9002">
<meta name="prism.publicationName" content="Journal of Phylogenetics &amp; Evolutionary Biology">
<meta name="prism.publicationDate" content="2014-08-29">
<meta name="prism.volume" content="2">
<meta name="prism.number" content="2">
<meta name="prism.section" content="OriginalPaper">
<meta name="prism.startingPage" content="1">
<meta name="prism.copyright" content="2014 The Author(s).">
<meta name="prism.url" content="https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php?aid=30965">
<meta name="citation_date" content=" 2014/08/29 ">
<meta name="dc.creator" content="An">
<meta name="dc.creator" content="Patwardhan">
<meta name="dc.creator" content="Samit Ray">
<meta name="dc.creator" content="Amit Roy">
<meta property="og:title" content="Molecular Markers in Phylogenetic Studies-A Review">
<meta property="og:url" content="https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php?aid=30965">
<meta property="og:image" content="images/journal-logos/Journal-of-Phylogenetics-&amp;-Evolutionary-Biology-logo.png">
<meta property="og:description" content="Uses of molecular markers in the phylogenetic studies of various organisms have become increasingly important in recent times. This review gives an overview of different molecular .. ">
<meta property="og:site_name" content="OMICS International">
<script type="text/javascript">
    function openimage(theURL, winName, features) {
        window.open(theURL, winName, features);
    }
</script>
<link type="text/css" rel="stylesheet" charset="UTF-8" href="https://translate.googleapis.com/translate_static/css/translateelement.css"><script type="text/javascript" charset="UTF-8" src="https://translate.googleapis.com/translate_static/js/element/main.js"></script><script type="text/javascript" charset="UTF-8" src="https://translate.googleapis.com/element/TE_20180625_00/e/js/element/element_main.js"></script></head>
<body class="body_class" style="position: relative; min-height: 100%; top: 0px;">
<div class="container container_style">
<style>
	.navbar-expand-lg .navbar-nav .nav-link {
			padding-right: 1.1rem !important;
			padding-left: 1.1rem !important;
	}
</style>
<div class="row" style="background-color:#185fa1;">
<div class="col-12 col-sm-6">
<nav class="nav divided-list justify-content-center justify-content-sm-start">
<a href="https://www.omicsonline.org/" title="Home" class="nav-link white">Home</a>
<a href="https://www.omicsonline.org/open-access-journals-list.php" title="OMICS Publishing Group Journals" class="nav-link white">Publications</a>
<a href="https://www.omicsonline.org/international-scientific-conferences/" target="_blank" title="OMICS Group Conferences" class="nav-link white">Conferences </a>
<a href="https://www.omicsonline.org/register.php" title="Register" class="nav-link white">Register</a>
<a href="https://www.omicsonline.org/contact.php" title="Contact" class="nav-link white">Contact</a>
</nav>
</div>
<div class="col-12 col-sm-6">
<nav class="nav justify-content-center justify-content-sm-end">
<a href="https://www.omicsonline.org/sitemap.xml" title="Click here" class="nav-link icon white bg-transparent"><i class="fas fa-sitemap"></i></a>
<a href="https://www.facebook.com/omicsinternational" target="_blank" title="Click here" class="nav-link icon white bg-transparent"><i class="fab fa-facebook-f"></i></a>
<a href="https://twitter.com/OMICSJournal" target="_blank" title="Click here" class="nav-link icon white bg-transparent"><i class="fab fa-twitter"></i></a>
<a href="https://www.linkedin.com/company/omics-publishing-group/" target="_blank" title="Click here" class="nav-link icon white bg-transparent"><i class="fab fa-linkedin-in"></i></a>
<a href="https://www.omicsonline.org/rss/journal-of-phylogenetics-evolutionary-biology.xml" title="Click here" class="nav-link icon white bg-transparent"><i class="fas fa-rss"></i></a>
<a href="https://plus.google.com/+OmicsgroupOrg/" target="_blank" title="Click here" class="nav-link icon white bg-transparent"><i class="fab fa-google-plus-g"></i></a>
<a href="https://www.slideshare.net/OMICSPublishingGroup" target="_blank" title="Click here" class="nav-link icon white bg-transparent"><i class="fab fa-slideshare"></i></a>
<a href="https://www.youtube.com/OMICSPublishingGroup" target="_blank" title="Click here" class="nav-link icon white bg-transparent"><i class="fab fa-youtube"></i></a>
</nav>
</div>
</div>
<div class="row align-items-center my-2">
<div class="col-12 col-md-2">
<div class="journal_logo"><a href="https://www.omicsonline.org/phylogenetics-evolutionary-biology.php" title="Journal of Phylogenetics &amp; Evolutionary Biology"><img src="Journal-of-Phylogenetics--Evolutionary-Biology-logo.png" alt="Journal of Phylogenetics &amp; Evolutionary Biology" width="100" height="100"></a></div>
<div class="issn"> ISSN: 2329-9002</div>
</div>
<div class="col-12 col-md-7">
<span class="journal_head_ft">Journal of Phylogenetics &amp; Evolutionary Biology</span>
</div>
<div class="col-12 col-sm-6 col-md-3 justify-content-center">
<form id="tfnewsearch" role="search" action="https://www.omicsonline.org/searchresult.php">
<div class="form-group">
<div class="input-group">
<input class="form-control" name="keyword" id="keyword" pattern=".{4,40}" required="" title="4 to 40 characters" placeholder="Search.." autocomplete="off" type="text"><div class="suggestions" id="suggestions_holder" style="display: none;"></div>
<div class="input-group-append">
<input class="btn btn-primary" value="search" type="submit">
</div>
</div>
</div>
<script language="javascript" type="text/javascript">
					
					$("#keyword").coolautosuggest({
							url: "https://www.omicsonline.org/author-names.php?chars=",
					});
			</script>
</form>
</div>
</div>
<div class="row">
<nav class="navbar navbar-expand-lg navbar-dark bg-dark border-0 rounded-0 py-0 w-100 navbar-inverse">
<button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#journalNavbar" aria-controls="journalNavbar" aria-expanded="false" aria-label="Toggle navigation">
<span class="navbar-toggler-icon"></span>
</button>

<div class="collapse navbar-collapse" id="journalNavbar">
<ul class="navbar-nav font-size-12">
<li class="nav-item"><a href="https://www.omicsonline.org/phylogenetics-evolutionary-biology.php" title="Home" class="nav-link white">Home</a></li>
<li class="nav-item dropdown">
<a href="#" class="dropdown-toggle nav-link white" data-toggle="dropdown" role="button" aria-expanded="false">Editorial Panel</a>
<div class="dropdown-menu" aria-labelledby="navbarDropdown">
<a href="https://www.omicsonline.org/editorinchief-phylogenetics-evolutionary-biology-open-access.php" title="Editor-in-Chief" class="dropdown-item">Editor-in-Chief</a>
<a href="https://www.omicsonline.org/editorialboard-phylogenetics-evolutionary-biology-open-access.php" title="Editorial Board" class="dropdown-item">Editorial Board</a>
</div>
</li>
<li class="nav-item"><a href="https://www.omicsonline.org/instructionsforauthors-phylogenetics-evolutionary-biology-open-access.php" title="Instructions for Authors" class="nav-link white">Instructions for Authors</a></li>
<li class="nav-item"><a href="https://www.omicsonline.org/submitmanuscript-phylogenetics-evolutionary-biology-open-access.php" title="Submit Manuscript" class="nav-link white">Submit Manuscript</a></li>
<li class="nav-item"><a href="https://www.omicsonline.org/ArchiveJPGEB/articleinpress-phylogenetics-evolutionary-biology-open-access.php" title="Articles in press" class="nav-link white">Articles in press</a></li>
<li class="nav-item"><a href="https://www.omicsonline.org/ArchiveJPGEB/currentissue-phylogenetics-evolutionary-biology-open-access.php" title="Current Issue" class="nav-link white">Current Issue</a></li>
<li class="nav-item"><a href="https://www.omicsonline.org/archive-phylogenetics-evolutionary-biology-open-access.php" title="Archive" class="nav-link white">Archive</a></li>
<li class="nav-item dropdown">
<a href="#" class="nav-link dropdown-toggle white" data-toggle="dropdown" role="button" aria-expanded="false">Special Issues</a>
<div class="dropdown-menu" aria-labelledby="navbarDropdown">
<a href="https://www.omicsonline.org/guidelines-phylogenetics-evolutionary-biology-open-access.php" title="Guidelines" class="dropdown-item">Guidelines</a>
<a href="https://www.omicsonline.org/specialissue-phylogenetics-evolutionary-biology-open-access.php" title="Upcoming Special Issues" class="dropdown-item">Upcoming Special Issues</a>
</div>
</li>
<li class="nav-item"><a href="https://www.omicsonline.org/journals-publishing-getting-readers-indexing.php" title="Metrics" class="nav-link white">Metrics</a></li>
<li class="nav-item"><a href="https://www.omicsonline.org/contact-phylogenetics-evolutionary-biology-open-access.php" title="Contact" class="nav-link white">Contact</a></li>
</ul>
</div>

</nav>
</div>
<div class="panel panel-default">
<div class="panel-body text-center text-primary" style="line-height:23px;">
<strong>Make the best use of Scientific Research and information from our 700+ peer reviewed, <a href="https://www.omicsonline.org/open-access-journals-list.php" class="text-warning" title="Open Access Journals">Open Access Journals</a> that operates with the help of 50,000+ Editorial Board Members and esteemed reviewers and 1000+ <a href="https://www.omicsonline.org/international-societies-world-fedarations-scientific-associations.php" class="text-warning" title="Scientific associations">Scientific associations</a> in <a href="https://www.omicsonline.org/medical-sciences-journals.php" class="text-warning" title="Medical">Medical,</a> <a href="https://www.omicsonline.org/clinical-journals.php" class="text-warning" title="Clinical">Clinical,</a> <a href="https://www.omicsonline.org/pharmaceutical-sciences-journals.php" class="text-warning" title="Pharmaceutical">Pharmaceutical,</a> <a href="https://www.omicsonline.org/engineering-journals.php" class="text-warning" title="Engineering">Engineering,</a> Technology and <a href="https://www.omicsonline.org/business-and-management-journals.php" class="text-warning" title="Management">Management</a> Fields.</strong>
</div>
</div>
<div class="panel panel-default ">
<div class="panel-body text-center text-primary" style="line-height:23px;">
<strong>Meet Inspiring Speakers and Experts at our 3000+ Global <a href="https://www.conferenceseries.com/" target="_blank" title="Conferenceseries" class="text-warning">Conferenceseries</a> Events with over 600+ Conferences, 1200+ Symposiums and 1200+ Workshops on <br>
Medical, Pharma, Engineering, Science, Technology and Business</strong>
</div>
</div>
<div class="page-header margin-t-20">
<h4 class="text-primary margin-b-0"> Research Article <span class="pull-right">Open Access</span> </h4>
</div>
<div class="row">
<div class="col-md-9 full-text">
<h1 class="fulltext_h1heading margin-t-0"> Molecular Markers in Phylogenetic Studies-A Review </h1>
<table class="table" border="0" cellspacing="0" cellpadding="0">
<tbody>
<tr>
<td colspan="2"><strong>Anand Patwardhan<sup><a href="#a1">1</a><a href="#b1">#</a></sup>, Samit Ray<sup><a href="#a2">2</a></sup> and Amit Roy<sup><a href="#a1">1</a><a href="#b1">#</a><a href="#corr">*</a></sup></strong></td>
</tr>
<tr>
<td colspan="2"><sup>1</sup><a name="a1" id="a1"></a>Department of Biotechnology, Siksha-Bhavana, Visva-Bharati University, Santiniketan 731 235, India</td>
</tr>
<tr>
<td colspan="2"><sup>2</sup><a name="a2" id="a2"></a>Department of Botany, Siksha-Bhavana, Visva-Bharati University, Santiniketan 731 235, India</td>
</tr>
<tr>
<td colspan="2"><sup>#</sup><a name="b1" id="b1"></a>Equally contributed for the manuscript</td>
</tr>
<tr>
<td class="body-content" width="144" valign="top"><strong>Corresponding Author : </strong></td>
<td class="body-content" width="590">Amit Roy<a name="corr" id="corr"></a><br>
Department of Biotechnology, Visva-Bharati University<br>
Santiniketan 731235, India<br>
<strong>Tel:</strong> +91-9433144948<br>
<strong>E-mail:</strong> amit.roy@visva-bharati.ac.in</td>
</tr>
<tr>
<td colspan="2" class="body-content"><strong>Received</strong> July 27, 2014; <strong>Accepted</strong> August 21, 2014; <strong>Published</strong> August 29, 2014</td>
</tr>
<tr>
<td colspan="3" class="body-content"><strong>Citation:</strong> 
Patwardhan A, Ray S, Roy A (2014) Molecular Markers in Phylogenetic 
Studies – A Review. J Phylogen Evolution Biol 2:131. doi: 
10.4172/2329-9002-2-131</td>
</tr>
<tr>
<td colspan="3" class="body-content"><strong>Copyright:</strong> © 2014 
Patwardhan A, et al. This is an open-access article distributed under 
the terms of the Creative Commons Attribution License, which permits 
unrestricted use, distribution, and reproduction in any medium, provided
 the original author and source are credited.</td>
</tr>
<tr>
<td colspan="3" class="body-content"><strong>Related article at</strong><br>
<img alt="Download" src="fulltext-arrow.png" align="absmiddle"><a href="https://www.ncbi.nlm.nih.gov/pubmed?term=Molecular+Markers+in+Phylogenetic+Studies-A+Review&amp;cmd=DetailsSearch" target="_blank">Pubmed </a><img alt="Download" src="fulltext-arrow.png" align="absmiddle"><a href="https://scholar.google.co.in/scholar?hl=en&amp;q=Molecular+Markers+in+Phylogenetic+Studies-A+Review&amp;btnG=" target="_blank">Scholar Google </a></td>
</tr>
</tbody>
</table> <p> <strong>Visit for more related articles at</strong> <a href="https://www.omicsonline.org/archive-phylogenetics-evolutionary-biology-open-access.php" title="Journal of Phylogenetics &amp; Evolutionary Biology"> Journal of Phylogenetics &amp; Evolutionary Biology </a> </p>
<div class="text-center">
<div class="btn-group btn-group-justified" role="group" aria-label="...">
<a href="https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.pdf" title="View PDF" class="btn btn-default text-warning"><i class="fa fa-file-text-o" aria-hidden="true"></i> View PDF</a> <a href="https://www.omicsonline.org/pdfdownload.php?download=open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.pdf&amp;aid=30965" title="Download PDF" class="btn btn-default text-warning"><i class="fa fa-file-pdf-o" aria-hidden="true"></i> Download PDF</a>
<a href="#tables_figures" title="Tables &amp; Figures" class="btn btn-default text-warning"><i class="fa fa-table" aria-hidden="true"></i> Tables &amp; Figures</a>
</div>
</div>
<h4>Abstract</h4>
<p> </p><p>Uses of molecular markers in the phylogenetic studies of 
various organisms have become increasingly important in recent times. 
This review gives an overview of different molecular markers employed by
 researchers for the purpose of phylogenetic studies. Availability of 
fast DNA sequencing techniques along with the development of robust 
statistical analysis methods, provided a new momentum to this field. In 
this context, utility of different nuclear encoded genes (like 16S rRNA,
 5S rRNA, 28S rRNA) mitochondrial (cytochrome oxidase, mitochondrial 
12S, cytochrome b, control region) and few chloroplast encoded genes 
(like rbcL, matK, rpl16) are discussed. Criteria for choosing suitable 
molecular markers and steps leading to the construction of phylogenetic 
trees have been discussed. Although widely practised even now, 
traditional morphology based systems of classification of organisms have
 some limitations. On the other hand it appears that the use of 
molecular markers, though relatively recent in popularity and are not 
free entirely of flaws, can complement the traditional morphology based 
method for phylogenetic studies.</p> <p></p>
<table class="table" border="0" cellspacing="0" cellpadding="0">
<tbody>
<tr>
<td><font class="fulltext-heading">Keywords</font></td>
</tr>
<tr>
<td colspan="2"><strong><a href="https://www.omicsonline.org/scholarly/molecular-phylogenetics-journals-articles-ppts-list.php">Molecular phylogeny</a></strong>; Phylogenetic tree; <strong><a href="https://www.omicsonline.org/scholarly/molecular-biomarkers-journals-articles-ppts-list.php">Molecular marker</a></strong>; Molecular clock; Bar code of Life</td>
</tr>
<tr>
<td><font class="fulltext-heading">Introduction</font></td>
</tr>
<tr>
<td colspan="2">Phylogeny is the history of descent of a group of taxa 
such as species from their common ancestors including the order of 
branching and sometimes the times of divergence. The term “Phylogeny” is
 derived from a combination of Greek words. Phylon stand for “tribe” or 
“clan” or “race” and genesis means “origin” or “source”. The term can 
also be applied to the genealogy of genes derived from a common 
ancestral gene. In molecular phylogeny, the relationships among 
organisms or genes are studied by comparing homologues of DNA or <a href="https://www.omicsonline.org/scholarly/protein-sequence-analysis-journals-articles-ppts-list.php"><strong>protein sequences</strong></a>. Dissimilarities among the sequences indicate genetic divergence as a result of <strong><a href="https://www.omicsonline.org/scholarly/evolutionary-developmental-biology-journals-articles-ppts-list.php">molecular evolution</a></strong>
 during the course of time. In brief, while classical phylogenetic 
approach relies on morphological characteristics of an organism, the 
molecular approaches depend on nucleotide sequences of RNA and DNA and 
sequences of amino acids of a protein which are determined using modern 
techniques. By comparing homologous molecules from different organisms 
it is possible to establish their degree of similarity thereby 
establishing or revealing a hierarchy of relationship a phylogenetic 
tree. Both the classical morphology based methods and molecular analysis
 based methods are of importance as the basic bio-molecular framework of
 all organisms are similar and morphology of an organism is actually the
 manifestations of its genome, <strong><a href="https://www.omicsonline.org/scholarly/proteome-journals-articles-ppts-list.php">proteome</a></strong> and <strong><a href="https://www.omicsonline.org/scholarly/transcripotme-journals-articles-ppts-list.php">transcriptome</a></strong>
 profiles. A combination of the morphological based methods and 
molecular analysis based methods thus strengthens the exercise of the 
determination of phylogenetic relationships of organisms to a great 
extent.</td>
</tr>
<tr>
<td colspan="2">The job of determination of phylogenetic relationship of
 various organisms is a difficult one as the living world exhibits 
unimaginable diversity with respect to its species content. This 
diversity is not only reflected in phenotypic characters but also in 
ultra-structural, biochemical and molecular features. Phenotypically 
similar organisms may have contrasting biochemical and molecular 
features. A rough estimate of the number of described species is 1.4 to 
1.8 million [<a href="#1">1</a>,<a href="#2">2</a>] of which arthropods,
 (especially insects), molluscs, and vascular plants account for more 
than 80%. Still there are millions of species which are unknown and 
unclassified. The field of taxonomy deals with classification, 
nomenclature and identification of unknown organisms i.e., the process 
of determining whether an organism belongs to one of the units defined 
previously, and if it does not belong to the any of the established 
taxonomic units, then categorize it as a new taxon. The task of 
describing, naming and classifying the organism is a part of 
systematics. Some terminologies related to molecular phylogeny are 
presented in Box 1.</td>
</tr>
<tr>
<td colspan="2">Since every organism is the result of an evolutionary 
process, one has to know its evolutionary history to understand and 
express it in biological terms. For the purpose of determination of 
evolutionary history, three types of information are necessary. The 
first one is phenotypic, i.e. the information gained from expressed 
features including both internal and external morphology, proteins and <strong><a href="https://www.omicsonline.org/open-access/bone-strength-skeletal-muscle-area-and-biochemical-markers-associatedwith-bone-metabolism-in-patients-with-fragility-distal-radius-2329-9509-1000167.php?aid=68739">biochemical markers</a></strong>.
 The second one is genotypic i.e. the knowledge obtained from the 
genetic material inside the cell. Lastly, when the homologies between 
DNA and proteins are compared, we get information about the phylogeny of
 that organism and the knowledge gained can be represented in the 
graphical form of a phylogenetic tree. It is to be noted, however, that 
phylogenetic trees have also been constructed in early days, long before
 the advent of techniques employing molecular markers, from studies on 
external morphology of organisms by noted evolutionary biologists.</td>
</tr>
<tr>
<td colspan="2">One of the most exciting developments in the past decade
 has been the application of powerful and ultra rapid nucleic acid 
sequencing techniques to the problems of phylogenetic studies. Rapid 
availability of large amounts of sequence data called for developments 
of robust mathematical and statistical analysis tools for explaining the
 process of evolution and this acute need ultimately gave rise to the 
science of molecular systematics. While molecular phylogeny, in a really
 broad way, may be a domain of the biology, the molecular systematics 
might be viewed as more of a statistical science in which powerful 
computation based simulation experiments are used to infer phylogenetic 
trees from these biological data obtained from a study of molecular 
markers. The idea of this review is mainly to focus on the molecular 
markers currently in use today and is divided into three sections; 1) 
the first section deals with history and general information on 
molecular phylogeny followed by 2) a section on typical molecular 
markers (e.g. 16S and 18S rRNA, matK etc.) used for this types of 
studies and 3) a very brief section on evolutionary tree building 
methods without which the review will remain incomplete. A general flow 
chart of various steps involved in studying molecular phylogeny using 
molecular markers is depicted in Figure 1.</td>
</tr>
<tr>
<td><font class="fulltext-heading">General Information on Molecular Phylogeny</font></td>
</tr>
<tr>
<td colspan="2"><strong>Classical and modern methods of phylogenetic studies</strong></td>
</tr>
<tr>
<td colspan="2">Long time back Aristotle (384-322 B.C.) did extensive morphological and embryological studies to classify <strong><a href="https://www.omicsonline.org/scholarly/marine-engineering-journals-articles-ppts-list.php">marine organisms</a></strong>. Following this, in the 18<sup>th</sup>
 century Linnaeus developed binomial system of nomenclature. He not only
 gave birth to the field of taxonomy but was the first to draw a 
phylogenetic tree. Later Charles Darwin added the occurrence of two 
important processes in phylogeny, mainly, branching and subsequent 
divergence. Early proponents of molecular phylogeny claimed that 
molecular data were more likely to reflect the true phylogeny than 
morphological data, chiefly because they reflected gene-level changes, 
which were thought to be less subject to convergence and parallelism 
than were morphological traits. This early theory now appears to be 
inaccurate and molecular data are in fact subject to scores of the same 
problems that morphological data are. Additionally, in case of 
unicellular organisms like bacteria morphology, physiology and many 
other properties are not informative enough to be used as phylogenetic 
markers. Thus, bacterial classification remained a determinative one, 
despite the efforts of microbiologists to figure out a natural bacterial
 classification. Moreover, there are many bacteria that cannot be 
cultured in the laboratory and their identification solely relies on 
molecular data. Recent adoption of polyphasic approaches (discussed in 
brief later) appear to have solved these difficulties.</td>
</tr>
<tr>
<td colspan="2">In recent years molecular phylogeny entered a rapidly 
expanding area with great improvements in the techniques and analyses of
 nucleic acid and protein sequencing. Early research using rRNA involved
 direct reverse transcriptase mediated sequencing of portion of both the
 small and large subunits of ribosome [<a href="#3">3</a>,<a href="#4">4</a>].
 As rRNA are the major portion of total cellular RNAs, it was relatively
 easy to obtain enough RNA for sequencing. It is to be noted, however, 
that sequences generated from direct sequencing of rRNA by reverse 
transcriptase have been found to be far more more error-prone than DNA 
sequences generated directly from the nuclear genes encoding ribosomal 
DNA (rDNA) [<a href="#5">5</a>]. In general, the methods utilizing DNA 
isolation, PCR, automated sequencing and then comparing these DNA or 
protein sequences are more preferred these days. In summary, molecular 
phylogenetic studies have been and remains technique driven and as a 
corollary, dominates the modern taxonomic studies.</td>
</tr>
<tr>
<td colspan="2"><strong>Molecular clock and the phylogenetics</strong></td>
</tr>
<tr>
<td colspan="2">Zuckerkandl and Pauling [<a href="#6">6</a>] were the 
first to study amino acid sequences of haemoglobin among different 
species and their results were remarkable. They found that haemoglobin 
molecules from horse and human differed by only 18 amino acids; mouse 
and human haemoglobins differed by 16 amino acids while mouse and horse 
hemoglobins differed only by 22 residues; but between humans and sharks 
there were differences in 79 amino acids in this molecule. These 
important observations seemed to suggest that there is a constant rate 
of amino acid substitution over time. To explain these results 
Zuckerkandl and Pauling [<a href="#6">6</a>] proposed the so called 
molecular clock hypothesis. The concept is based on a steady rate of 
change in DNA sequences over time and provided a basis for dating the 
time of divergence of lineages. It suggests that these amino acid 
differences correlate with the evolutionary time scale. As explained 
above, amino acid differences between mammals are less compared to that 
between mammals and shark. Thus, a biomolecule was acting like a 
molecular clock. Further they are distanced from each other in the 
evolutionary timescale, greater would be the differences in their 
molecular sequences and vice versa. Similarly the molecular clock 
hypothesis was used to propose that humans and apes diverged 
approximately 5 million years ago [<a href="#7">7</a>]. Although 
informative, the hypothesis has been questioned many times because 
biomolecules are subjected to changes at different rates.</td>
</tr>
<tr>
<td colspan="2">The phylogeny concluded from a single marker gene or 
protein sequence only reflects evolution of that particular gene. But 
use of a single marker can lead to interpretation problems, because 
other genes in the organism may show different rates of evolution or 
even show different evolutionary history if horizontal gene transfer has
 taken place. Vertical gene transfer is the normal passage of genes from
 parent to offspring. Horizontal or lateral gene transfer happens when 
genes transfer between unrelated organisms, a common phenomenon in 
bacteria e.g. acquired antibiotic resistance leading to multidrug 
resistant bacterial species. There have also been well-known cases of 
horizontal gene transfers between eukaryotes. Horizontal gene transfer 
has complicated the determination of phylogenies of organisms. 
Inconsistencies in phylogeny have been reported among specific groups of
 organisms depending on the marker genes used to construct evolutionary 
trees. The only way to determine which genes have been acquired 
vertically and which one horizontally is to assume that the largest set 
of genes that have been inherited together have been inherited 
vertically. This requires analyzing a large number of genes as opposed 
to studying a single marker gene. So only when one considers the 
evolution of multiple genes in a <strong><a href="https://www.omicsonline.org/scholarly/genome-biology-journals-articles-ppts-list.php">genome</a></strong>, one can get more convincing conclusions about the evolutionary status of an organism.</td>
</tr>
<tr>
<td colspan="2"><strong>Molecular markers are favoured over morphological data</strong></td>
</tr>
<tr>
<td colspan="2">The underlying fact useful for molecular systematics is 
that different genes accumulate mutations at different rates. This 
difference depends on how much change a gene can tolerate without losing
 its function. For example, histone molecules may become non-functional 
if some of its amino acids are replaced with different ones. On the 
other hand internal transcribed spacers (ITS) of ribosomal RNA can still
 fold properly if many of its nucleotides are changed. Thus, ITS can 
accumulate mutations more rapidly than histones, reflecting the 
different functional constraints on their gene product. The advantages 
of using molecular data is obvious - molecular data are more numerous 
than fossil records and easier to obtain. There is no sampling bias 
involved, which helps to correct the gaps in real fossil records. A more
 clear and robust phylogenetic tree can be constructed with the 
molecular data. On the other hand parameters for morphological data on 
many occasions are limited in number and become insufficient to 
distinguish two organisms at phyla, class, order and family levels. When
 variation in morphological data become insufficient to distinguish two 
organisms-at phyla class, order, family etc. levels, analysis of the 
biomolecules are considered, which are large in number and occur in 
various forms in organisms. Therefore, the biomolecular markers have 
become favourite and sometimes the only information available for 
researchers to reconstruct evolutionary history. The big difference is 
that there are simply many more molecular characters available, and 
their interpretation is generally easier. Another advantage of molecular
 data is that all known life forms are based on nucleic acids and, each 
nucleotide position, in theory, can be considered as a character and 
assumed to be independent. The morphological adaptations of an organism,
 in any case, are mirrored in its biomolecules and vice versa.</td>
</tr>
<tr>
<td colspan="2"><strong>Potential of a gene in resolving phylogenetic relationship</strong></td>
</tr>
<tr>
<td colspan="2">The biomolecule based reconstruction of ancient 
phylogenetic history first requires the discovery and analysis of slowly
 evolving nucleotide or amino acid sequences. Not all genes or 
macromolecules are suitable phylogenetic markers and not all marker 
molecules are useful for the analysis of a given group of organisms. The
 method of screening molecular sequences for their ability to resolve 
relationships within a particular group include studies which assess the
 ability of a gene to recover well-established phylogenetic 
relationships within clades of similar age and the construction of 
fossil-based pair wise difference curves, which estimate the rate of 
potentially informative character changes during the geological interval
 when a clade underwent phylogenetic divergence [<a href="#8">8</a>,<a href="#9">9</a>].
 For example, to establish the utility of mitochondrial COI and COII 
(cytochrome oxidase I &amp; II) genes for the purpose of phylogeny 
studies, Caterino and Sperling used these genes to study phylogeny of <em>Papilio</em>
 sp. and after that they examined the phylogenetic placements of several
 lineages which have proven difficult in previous studies [<a href="#10">10</a>].
 Such genes serve as molecular fossils and through comparative analysis 
of the molecular fossils from a number of related organisms, the 
evolutionary history of the genes and even the organisms can be 
revealed.</td>
</tr>
<tr>
<td colspan="2"><strong>Properties of ideal marker genes</strong></td>
</tr>
<tr>
<td colspan="2">The properties that should be possessed by an ideal marker are as follows [<a href="#11">11</a>]:</td>
</tr>
<tr>
<td colspan="2">(a) A single-copy gene may be more useful than 
multiple-copy gene; this condition is satisfied by the mitochondrial and
 nuclear genes; (b) As marker gene sequences are aligned prior to 
phylogenetic analysis, their alignment should be easy. The length of the
 same gene can vary among different members of taxa due to insertions or
 deletions because of which aligning their sequences may be difficult. 
However, regions with ambiguous alignments can be avoided specifically 
or secondary structure information may be applied [<a href="#12">12</a>];
 (c) The substitution rate should be optimum so as to provide enough 
informative sites. A gene evolving too fast may reach a state of 
saturation due to multiple substitutions. This problem can be enhanced 
by base composition bias since this makes it more likely that the second
 mutation at a particular site will be a reversion to the original 
state. For protein coding genes it may be the case that the synonymous 
substitution rate is too high even though very few non-substitutions 
have occurred; (d) Primers should be available to selectively amplify 
the marker gene. However, the primer should not be too universal as in 
that case it would lead to amplification of non-specific genes present 
as contaminants or contributed by symbionts [<a href="#13">13</a>]; (e) A too much of base variation among the taxa, is not preferable which may not reflect the true ancestry [<a href="#14">14</a>]. The breakthrough in the study of the phylogeny of prokaryotes was achieved by Carl Woese and co-workers in the seventies [<a href="#15">15</a>,<a href="#16">16</a>].
 They introduced rapid methods of comparative 16S rRNA sequence analysis
 and phylogenetic tree reconstruction. The results of these efforts 
provided, for the first time, insight into the phylogeny of prokaryotes 
and also established the three domains of life, popularly known as– “The
 Universal Tree of Life” – Archaea (formerly archaebacteria), Bacteria 
(formerly eubacteria) and Eukarya (eukaryotes) [<a href="#16">16</a>,<a href="#17">17</a>].
 So far, these molecular studies of divergence have drawn on DNA or 
amino acid sequence data for highly conserved genes, particularly the 
structural ribosomal genes 18S/16S/5S/28S, the nuclear protein-coding 
gene elongation factor-1a (EF-1α) and the slowly evolving mitochondrial 
gene cytochrome c oxidase I (COI), histone H3, U2 snRNA and many more 
genes which are widely distributed. Some of the very popular markers 
being used widely in phylogenetic studies are described below in some 
detail.</td>
</tr>
<tr>
<td><font class="fulltext-heading">Molecular Markers</font></td>
</tr>
<tr>
<td colspan="2"><strong>Nuclear ribosomal genes</strong></td>
</tr>
<tr>
<td colspan="2">Ribosomal RNA is considered as the best target for 
studying phylogenetic relationship because, it is universal and is 
composed of highly conserved as well as variable domains [<a href="#16">16</a>,<a href="#18">18</a>].
 The ribosomes consist of rRNA and proteins. In all organisms the 
ribosome consists of two subunits, the small ribosomal subunit (SSU) 
contains a single RNA species (the 18S rRNA in eukaryotes and the 16S 
rRNA in others). In Bacteria and Archaea, the large subunit (LSU) 
contains two rRNA species (the 5S and 23S rRNAs); in most eukaryotes the
 large subunit contains three RNA species (the 5S, 5.8S and 25S/28S 
rRNAs). The core structures of the SSU and LSU rRNAs contain 10 and 18 
such variable regions, respectively. Moreover, rRNA genes are evolving 
more slowly than protein encoding genes and are particularly important 
for the phylogenetic analysis of distantly related species [<a href="#19">19</a>]. In particular, secondary-structure models of RNA molecules have been based almost exclusively on comparative <strong><a href="https://www.omicsonline.org/scholarly/sequence-analysis-journals-articles-ppts-list.php">sequence analysis</a></strong> [<a href="#20">20</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>16S rRNA:</strong> It was in 1960s that Dubnau et al. observed the conservation in the 16S rRNA gene sequence among <em>Bacillus</em> species [<a href="#21">21</a>]. But, it was only after the classic work done by Woese, that these gene sequences were used for bacterial taxonomy [<a href="#16">16</a>].
 The 16S rRNA gene is conserved, which does not mean that it evolves at a
 same rate in all organisms. This important property helps researchers 
to distinguish among different bacterial groups [<a href="#16">16</a>,<a href="#20">20</a>,<a href="#22">22</a>].
 The 16S rRNA gene is about 1550 bp long and contains both variable and 
conserved regions with characteristic oligonucleotide signature 
sequences (unique to a particular phylogenetic group). Using primers of 
the conserved regions, the in-between variable region can be amplified. 
This is sufficient to differentiate organisms using statistically valid 
measurements [<a href="#23">23</a>,<a href="#24">24</a>]. As 16S gene is
 present in all bacteria, one can measure relationships among all 
bacterial species. Comparing 16S sequences of unknown bacteria with 
already deposited sequence will assist in marking those bacteria in a 
particular group [<a href="#22">22</a>]. Studying 16S and 23S rRNA are the backbone of bacterial taxonomy, especially for identification of nonculturable bacteria.</td>
</tr>
<tr>
<td colspan="2"><strong>5S rRNA: </strong>Ribosomal 5S RNA, a ~120 
nucleotide long RNA, is found in virtually all ribosomes with the 
exception of mitochondria of some fungi, higher animals and most 
protists [<a href="#25">25</a>]. The nucleotide sequence of 5S rRNA is 
highly conserved throughout nature and phylogenetic analysis alone 
provided an initial model for its secondary structure [<a href="#15">15</a>,<a href="#18">18</a>].
 The primary structure of these rRNA molecules are sufficiently 
constrained that on the whole they have not changed rapidly in time [<a href="#18">18</a>]. Some of the first molecular sequence data available for green algae came from nuclear 5S ribosomal RNA. Troitskii et al. [<a href="#26">26</a>]
 derived complete or partial nucleotide sequences of five different 
rRNAs from a number of seed plants and discussed the angiosperm origins 
and early stages of land plant evolution based on phylogenetic 
dendrograms using the compatibility [<a href="#27">27</a>] and parsimony methods from the PHYLIP package [<a href="#28">28</a>].
 However, the reliability of hypothesis based on this molecule were 
questioned because the 5S rRNA molecule is only 120 bases long with too 
few informative sites that can be used in analysis of close relatives. 
It is a rapidly evolving molecule, so that in the positions that do 
vary, there are so many substitutions that the number of potentially 
informative sites is too small to allow reliable analysis for studying 
ancient divergences. In fact, there are reports that 5S rRNA sequence 
data do not have sufficient resolving power to contribute significantly 
to our understanding of phylogenetic relationships at any taxonomic 
level [<a href="#29">29</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>28S rRNA:</strong> Phylogenetic analyses based on molecular sequences must come from genes encoding larger molecules than the 120 bp 5S rRNA [<a href="#29">29</a>].
 The 28S rRNA gene is about 811 bp in length. 28S rRNA gene sequences 
for many major metazoan groups have become available in the recent 
years. Also, efforts to align sequences according to the 
secondary-structure model for 28S rRNA of these organisms have become 
commonplace for the purpose of phylogenetic analyses. For example, <em>Encarsia</em>,
 which is a large genus of minute parasitic wasps, only a few of the 
species-groups are defined unambiguously on the basis of morphological 
characters alone. Phylogenetic relationships within this genus still are
 largely unresolved; only recently attempts have been made to use 
molecular data to underpin the taxonomy based on morphological 
characters and to resolve phylogenetic relationships. All molecular 
studies conducted so far have used the D2 expansion region of the 28S 
ribosomal RNA; there has been comparatively little information about the
 suitability of other gene regions to inferring phylogenetic 
relationships or to defining species limits in this group [<a href="#30">30</a>,<a href="#31">31</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Mitochondrial genes (mtDNA)</strong></td>
</tr>
<tr>
<td colspan="2">Mitochondrial DNA data can be very powerful in resolving
 species-level phylogenies. The order of genes in the mitochondrion is 
variable, and they are separated by large regions of noncoding DNA. The 
mitochondrial genome rearranges itself frequently so that many 
rearranged forms can occur in the same cell. The use of mtDNA has become
 increasingly popular in phylogenetics and population genetic studies 
because of i) developments in methodology for mtDNA isolation, ii) use 
of restriction enzymes to detect nucleotide differences, iii) the 
developments of PCR methodologies and iv) applicability of universal 
primers for amplification of DNA [<a href="#32">32</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Cytochrome oxidase I/II (COI/II):</strong> The 
enzyme cytochrome c oxidase is a very well known protein of electron 
transport chain and is found in both bacteria and mitochondria. The COI 
and COII genes code for two of seven polypeptide subunits in the 
cytochrome c oxidase complex. The COI gene consists of approximately 894
 bp. COI and/ or COII sequences have been applied to phylogenetic 
problems at a wide range of hierarchical levels in insects, from closely
 related species to genera and subfamilies, families, and even orders. 
The COI gene is slowly evolving compared to other protein coding 
mitochondrial genes and is widely used for estimating molecular 
phylogenies [<a href="#33">33</a>] and is a good performer in recovering an expected tree [<a href="#34">34</a>].
 So sequencing both the genes represents one of the largest sequence 
data sets generated for phylogenetic study of any group and also fulfils
 the putative phylogenetic accuracy. The combination of COI and 12S rRNA
 is appropriate to distinguish the taxa of interest at different 
taxonomic level. COI and COII have been used for species and population 
analyses of parasitoids and COI has recently been suggested as a 
potential ‘barcode’ for insect identification in general. Zhang and Sota
 reported that the COI sequence of mitochondrial data had higher 
sequence divergence than four other nuclear genes, in beetles [<a href="#35">35</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Mitochondrial 12S:</strong> Mitochondrial 12S 
rRNA gene sequence analysis is extensively used in molecular taxonomy 
and phylogeny. Earlier, mitochondrial 12S rRNA gene sequence was used 
for species determination in wild-life forensic biology. It has been 
postulated earlier that 12S gene sequences are useful for the 
determination of moderate to long divergence times. The length of this 
gene is about 450 bp and it can be amplified by universal primers. The 
355 bp sequence of this gene was used for identification, phylogenetic 
relationships and calculation of divergence time of the Indian leopards [<a href="#36">36</a>]. Chaolun et al. used the 12S gene to infer the evolutionary history of 28 species of certain coral groups [<a href="#37">37</a>].
 They found out that phylogenetic analyses using mitochondrial 12S rRNA 
gene data did not support the current view of phylogeny for this group 
of corals based upon skeletal morphology and fossil records. Allard and 
Honeycutt reported that the 12S rRNA gene is not evolving at a higher 
rate within certain rodent lineages [<a href="#38">38</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Cytochrome-b:</strong> Cytochrome-b gene (~1,143
 bp) is reported as the most useful marker in recovering phylogenetic 
relationships among closely related taxa but can lose resolution at 
deeper nodes. Although the Cytochrome-b gene has proven useful in 
recovering phylogenetically useful information at a variety of taxonomic
 levels, strength of its utility can be lineage-dependent and declines 
with evolutionary depth. Bradley et al. [<a href="#39">39</a>] concluded
 that, although the Cytochrome-b data contain considerable phylogenetic 
signal, definition of content and resolution of the phylogeny of genus <em>Peromyscus</em> (deer mice) needs other additional information [<a href="#39">39</a>]. The patterns of speciation and trait evolution in <em>Tragopan</em>,
 a genus of five Indo-Himalayan bird species, were examined using 
sequences of the mitochondrial cytochrome b gene (CYB) and its control 
region (CR) [<a href="#40">40</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Control region for replication of mitochondrial DNA:</strong>
 The only major non-coding area of the mtDNA is the control region, 
typically 1 kb, involved in the regulation and initiation of mtDNA 
replication and transcription and is responsible for the regulation of 
heavy (H) and light (L) strand transcription and of H-strand 
replication. The approximate mutation rate in mtDNA is 10<sup>-8</sup>/site/year compared to 10<sup>-9</sup>/site/year
 in nuclear genes. Most differences between mtDNA sequences are point 
mutations, with a strong bias for transitions over transversions [<a href="#32">32</a>].
 Rogaev et al. reported the presence of variable number of tandem 
repeats (VNTR) in the control region which are characterized by high 
somatic hypervariability in some mammoth [<a href="#41">41</a>]. The 
evolution of the control region of mammalian mtDNA shows some features 
such as strong rate heterogeneity among sites, the presence of tandem 
repeated elements, a high frequency of nucleotides insertion/ deletion, 
and lineage specificity [<a href="#42">42</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Chloroplast genes</strong></td>
</tr>
<tr>
<td colspan="2">Many plant phylogenetic studies are based on chloroplast
 DNA (cpDNA). In plants, cpDNA is smallest as compared to mitochondria 
and nuclear genome. It is assumed to be conserved in its evolution in 
terms of nucleotide substitution with very little rearrangements which 
permits the molecule to be used in resolving phylogenetic relationships 
especially at deep levels of evolution [<a href="#43">43</a>]. However, 
selection of a gene of sufficient length and appropriate substitution 
rate is a crucial step. Currently used cpDNA genes include rbcL, ndhF, 
rpl16, matK, atpB and many more (some of them are described below).</td>
</tr>
<tr>
<td colspan="2"><strong>rbcL:</strong> Ribulose 1, 5-bisphosphate 
carboxylase/oxygenase (rubisco) is the first enzyme of C3 cycle in 
plants. It is the most abundant and most important protein on the planet
 and central to the global carbon cycle [<a href="#44">44</a>]. The rbcL
 gene is located on cp genome as a single copy gene and has an enormous 
phylogenetic utility. The rbcL gene is ~1428 bp long and is universal to
 all plants (except in some parasites). It is very convenient to study, 
easy to align and its secondary structure is known and present in many 
copies with less insertions and deletions. The rbcL gene encodes the 
large subunit of rubisco, while the small subunit is encoded by rbcS 
gene in nucleus. The rbcL gene was one of the first plant genes to be 
sequenced [<a href="#45">45</a>] and is still among the most frequently 
sequenced segments of plant DNA. This gene has been used widely in 
systematic studies of land plants, angiosperms in particular [<a href="#44">44</a>].
 About 500 rbcL sequences were used to address phylogenetic 
relationships within angiosperms and secondarily among extant seed 
plants [<a href="#44">44</a>]. Although there is length variation 
between plants and algal genes, their alignment is easy. However many 
researchers prefer 18S rDNA for sampling than rbcL sequence because of 
the more rapid rate of evolution in the latter molecule. Although rbcL 
is conserved and readily alignable across divergent taxa, this molecule 
exhibits a higher substitution rate than the 18S rDNA. Mc Court et al. 
tentatively concluded that although rbcL sequences may be inappropriate 
in phylogenetic studies of ancient branching events (unless and until 
more thorough taxon sampling is possible), the use of this gene within 
green algal groups appears to be appropriate [<a href="#46">46</a>]. For
 example, rbcL does not contain enough information for resolving 
relationships between closely related genera e.g. Hordeum, Triticum, and
 Aegilops. In such cases the non-coding regions of chloroplast DNA, 
which are supposed to evolve more rapidly than coding regions are also 
analyzed. Palmer et al. have shown that the 16S rRNA gene as the most 
conserved of chloroplast genes followed by 23S rRNA [<a href="#47">47</a>]. So, they are more useful phylogenetically at the higher hierarchical levels than the rbcL gene, which codes for a protein.</td>
</tr>
<tr>
<td colspan="2"><strong>matK:</strong> The matK (maturase) gene is 
approximately 1500 base pairs (bp), located within the intron of the 
chloroplast gene trnK (lysine tRNA), and encodes a maturase involved in 
splicing type II introns from RNA transcripts [<a href="#48">48</a>,<a href="#49">49</a>].
 Recent studies have shown the usefulness of this gene in resolving 
intergeneric or interspecific relationships among flowering plants. The 
matK gene is known to have relatively high rates of substitution 
compared with other genes used in grass systematics, possesses high 
proportions of transversion mutations, and the 3 section of its coding 
region has been proven quite useful for constructing phylogenies at the 
subfamily level in the Poaceae [<a href="#47">47</a>]. Sequences from 
noncoding regions of the chloroplast genome are often used in 
systematics because such regions tend to evolve relatively rapidly.</td>
</tr>
<tr>
<td colspan="2"><strong>ndhF:</strong> This gene codes for subunit F of 
NADP dehydrogenase and is about 1100 bp in length and present in the 
small single-copy region. Givnish et al. used ndhF sequence variation to
 reconstruct relationships across 282 taxa representing 78 monocot 
families [<a href="#49">49</a>]. Moreover, they showed that 
relationships within orders are consistent with those based on rbcL, 
alone or in combination with atpB and 18S rDNA, and generally better 
supported and ndhF contributes more than twice as many informative 
characters as rbcL and nearly as many as rbcL, atpB, and 18S rDNA 
combined. Kim and Jansen did an extensive sequence comparison of the 
chloroplast ndhF gene from all major clades of the largest flowering 
plant family (Asteraceae) and showed that this gene provides ~3 times 
more phylogenetic information than rbcL [<a href="#50">50</a>]. This is 
because it is substantially longer and evolves twice as fast. The 5’ 
region (1380 bp) of ndhF is very different from the 3’ region (855 bp) 
and is similar to rbcL in both the rate and the pattern of sequence 
change.</td>
</tr>
<tr>
<td colspan="2"><strong>rpl16:</strong> Zhang used chloroplast noncoding rpl16 intron (1059 bp) sequences to reconstruct the phylogeny of the grass family [<a href="#51">51</a>].
 He reported that the rpl16 intron sequence data confirmed three 
traditional herbaceous bamboo tribes, Streptochaeteae, Anomochloeae, and
 Phareae, as the most basal lineages in the extant grasses. Zhang also 
showed that the comparisons of the nucleotide divergence and the genetic
 distance between the chloroplast noncoding rpl16 intron and the ndhF 
gene among the major groups of the grass family showed that the rpl16 
intron sequences had a lower transition/transversion ratio but higher 
nucleotide divergence and genetic distance [<a href="#51">51</a>]. 
Earlier studies indicated that noncoding sequences had a much more 
complicated evolution pattern and more frequent insertion and deletion 
events than to coding regions [<a href="#44">44</a>]. The rpl16 intron 
sequences show similar results in many reports. Comparison between the 
ndhF gene and the rpl16 intron sequences done by Zhang indicated that 
the sequence divergence in the rpl16 intron was 1.40 times of that in 
the ndhF gene [<a href="#51">51</a>]. Some other additional marker genes are mentioned in Table 1.</td>
</tr>
<tr>
<td><font class="fulltext-heading">Phylogenetic Tree Construction Methods</font></td>
</tr>
<tr>
<td colspan="2">The result of a molecular phylogenetic analysis can be 
represented in a diagram in the form of a phylogenetic tree. Phylogeny 
is an abstract phenomenon and it cannot be observed directly. It is 
something that happened in the past and must be reconstructed using 
available evidence. By studying a phylogenetic tree it is possible to 
obtain a quick overall idea about the given species and its relation to 
other species phylogenetically close to it. As large numbers of 
potential trees are possible, finding out a tree which perfectly 
reflects the evolutionary history is very difficult. A tree can also be 
rooted or unrooted. There is an exponential relationship between the 
possible number of trees for ‘n’ taxa, given by, for rooted tree</td>
</tr>
<tr>
<td colspan="2">N = (2n-3)! 2n<sup>-2</sup>(n-s)!</td>
</tr>
<tr>
<td colspan="2">and for unrooted tree,</td>
</tr>
<tr>
<td colspan="2">N=(2n-5)!/2n<sup>-3</sup>(n-3)!.</td>
</tr>
<tr>
<td colspan="2">Thus, even for ten taxa under study, there are millions 
of possible tree topologies available. So, there are various methods to 
select an optimal tree. The trees can be drawn in different ways such as
 cladogram or a phylogram. As depicted in Figure 1, a phylogenetic tree 
construction goes through essentially five steps: a) Selection of 
molecular markers; b) Performing multiple sequence alignments; c) 
Choosing an evolutionary model; d) Determining a tree building method 
and lastly e) Assessing tree reliability [<a href="#52">52</a>-<a href="#70">70</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Selection of molecular markers</strong></td>
</tr>
<tr>
<td colspan="2">The molecular data can either be obtained from 
nucleotide or protein sequence data. This often depends upon the 
closeness of the organisms under study. Nucleotide sequence is preferred
 while studying closely related organisms, slowly evolving genes are 
used for widely divergent groups, whereas non-coding mitochondrial DNA 
is a choice while studying individuals of a population. Protein 
sequences are more conserved due to codon degeneracy, while the third 
position of a codon in nucleotide sequence may show variation. Some of 
the widely used molecular markers preferred by the investigators engaged
 in molecular phylogenetic research have already been described in 
section 2.</td>
</tr>
<tr>
<td colspan="2"><strong>Multiple Sequence Alignment</strong></td>
</tr>
<tr>
<td colspan="2">Once the markers to be studied have been determined, the
 DNA sequence of the selected marker genes of the target organism needs 
to be experimentally determined. For this, total DNA is isolated from 
the appropriate tissue of the organism. In most instances total cellular
 DNA may be isolated using many of the well established DNA isolation 
protocols. The chosen markers are then amplified using the isolated DNA 
as template and marker specific oligonucleotides as primers by PCR 
method. For many of the markers discussed in this article, well known 
universal primers are already described in the literature. 
Alternatively, the primer can be designed depending upon the specific 
need of the project. The amplified PCR products are then sequenced. As 
the DNA sequence of the marker genes are obtained, wholly or in part, 
the next step is to align the sequence with the DNA sequence of the same
 markers of closely known species. Multiple alignment is possibly the 
most critical step in the procedure because it establishes positional 
correspondence in evolution [<a href="#70">70</a>]. Only a successful 
sequence alignment produces a genealogically related tree. Multiple 
alignments can be done using various very well known alignment programs 
like ClustalW, T-coffee, Multialin etc. to mention a few. Secondary 
structure information may also assist alignment. Praline is one such 
program which extracts the information of secondary structure for the 
purpose of alignment. Some programs (Rascal, NorMD, and Gblocks) can 
improve the alignment by correcting the errors or by removing poorly 
aligned positions.</td>
</tr>
<tr>
<td colspan="2"><strong>Choosing an evolutionary model</strong></td>
</tr>
<tr>
<td colspan="2">The next step is to select a proper substitution model 
that provides the researcher with ideas of the evolutionary process by 
taking into account multiple substitution events. However, the observed 
number of substitutions may not represent the true evolutionary process 
that actually occurred at the locus of interest. When a mutation is 
detected as G replaced by T, the nucleotides may have actually undergone
 a number of transitional steps to become T in the sequence G→A →C→ T. 
Similarly a back mutation could have taken place also when a mutated 
nucleotide changed back to the original nucleotide such that A→T→A. 
Additionally, an identical nucleotide observed in the alignment may be 
due to parallel mutations; such multiple substitutions and convergence 
at individual positions obscure the estimation of the true evolutionary 
distances between the sequences. This effect is known as homoplasy which
 needs to be corrected for the generation of a true evolutionary tree. 
To correct homoplasy, statistical models known as substitution models or
 evolutionary models, are needed to infer the true evolutionary 
distances between sequences. Following are the two important 
substitution models [<a href="#70">70</a>].</td>
</tr>
<tr>
<td colspan="2"><strong>Jukes-Cantor model:</strong> Jukes-Cantor model 
assumes that purines as well as pyrimidines are substituted with equal 
probability. This model can only analyse reasonably closely related 
sequences.</td>
</tr>
<tr>
<td colspan="2">Kimura model: In contrast, Kimura two-parameter model [<a href="#71">71</a>]
 assumes that transition mutations should occur more often that 
transversion. This is a model that takes in to account the differential 
mutation rates of transitions &amp; transversions and is more realistic.
 For protein sequences, the evolutionary distances from an alignment can
 be corrected using a PAM or JTT amino acid substation matrix. 
Alternatively, protein equivalents of Jukes-Cantor model &amp; Kimura 
models can be used to correct evolutionary distances.</td>
</tr>
<tr>
<td colspan="2"><strong>Tree building method:</strong> Next step is the evolutionary tree building. There are several methods available [<a href="#71">71</a>]
 and it is generally recommended to perform exhaustive experiments using
 one or more model. However, it may be a time consuming task when number
 of taxa increases drastically. Figure 2 shows the summary of different 
methods which are routinely used. Here we will discuss them in brief as 
detail explanation of each method is out of scope of this review.</td>
</tr>
<tr>
<td colspan="2"><strong>Methods based on characters:</strong> Such 
methods take into account the mutational events accumulated on the 
sequences and thus avoid loss of information. It easily provides 
information regarding homoplasy and ancestral states. It produces more 
accurate trees than the distance based methods. Two most popular 
character based methods are maximum parsimony and maximum likelihood.</td>
</tr>
<tr>
<td colspan="2"><strong>Methods based on distance:</strong> A true 
evolutionary distance between sequences can be calculated from observed 
distance after correction using different models. They are subdivided as
 optimality based and clustering based algorithms.</td>
</tr>
<tr>
<td colspan="2"><strong>Phylogenetic tree evaluation method</strong></td>
</tr>
<tr>
<td colspan="2">Having constructed the tree, its validity needs to be 
checked. Different statistical test are used to evaluate the reliability
 of the constructed tree. Bootstrapping and Jackknifing are employed to 
check the reliability of the tree while, Kishino-Hasegawa test, Bayesian
 analysis, and Shimodaira-Hasegawa test are used to confirm whether the 
tree is better than any other tree. In bootstrapping technique, randomly
 sized and positioned pieces of sequence form the same part of the 
molecule are sampled randomly and a new phylogenetic analysis is 
performed to produce a tree. To determine the robustness of the tree it 
is generally recommended that a phylogenetic tree should be bootstrapped
 500-1000 times, thus making the process time consuming. The bootstrap 
results are compared to the original approximated tree. Branch point 
scores around 90% suggest that the predicted tree is accurate. However, 
controversies can still arise. In Jackknifing half of the data set is 
subjected to phylogenetic tree construction using the same method as of 
original. The Bayesian simulation test uses Markov chain MonteCarlo 
(MCMC) procedure which is very fast and involved thousands of steps of 
resampling. Kishino-Hasegawa test is especially used for maximum 
parsimony trees, a t-value is calculated, which is used for evaluation 
against the t-distribution to see whether the values falls within the 
significant range (e.g. &lt;0.05),</td>
</tr>
<tr>
<td colspan="2">t=Pa-Pt/SD/√n</td>
</tr>
<tr>
<td colspan="2">where,</td>
</tr>
<tr>
<td colspan="2">n is the number of informative sites, the degree of 
freedom is n-1, t is the test statistical value, Pa is the average 
site-to-site difference between the two trees, SD is the standard 
deviation, and Pt is the total difference of branch lengths of the two 
trees. Shimodaira-Hasegawa (SH) test is frequently used for Maximum 
likelihood trees; it tests the goodness of fit using χ<sup>2</sup> test [<a href="#70">70</a>].</td>
</tr>
<tr>
<td colspan="2"><font class="fulltext-heading">DNA Barcode in Animals and Plants</font></td>
</tr>
<tr>
<td colspan="2">While in case of most animal species cytochrome oxidase 
(COI) has been described as a relatively accurate system for cost 
effective species identification purpose, even in the recent past there 
has not been a generally accepted DNA barcode standard for the plant 
kingdom as the performance of different loci combinations remains 
inadequate among different plant families. DNA barcoding, a relatively 
new term, is defined as a method for identifying species by using short 
DNA sequences, known as DNA barcodes, to facilitate biodiversity studies
 and enhance forensic analyses etc. So the researchers designed family 
specific primers and came closer to accepted phylogeny using this 
approach. In 2009, a large consortium of researchers, the “Consortium 
for the Barcode of Life (CBOL) Plant Working Group” proposed portions of
 two coding regions from the plastid (chloroplast) genome—molecular 
markers rbcL and matK—as a core barcode for plants, to be supplemented 
with additional regions as required. This recommendation was accepted by
 the international Consortium for the Barcode of Life, but with the 
rider that further sequencing of additional markers should be 
undertaken. This was driven by concerns that routine use of a third (or 
even a fourth) marker may be necessary to obtain adequate discriminatory
 power and to guard against sequencing failure for one of the markers [<a href="#69">69</a>,<a href="#72">72</a>].</td>
</tr>
<tr>
<td colspan="2"><font class="fulltext-heading">Polyphasic Approach for Bacterial Taxonomy</font></td>
</tr>
<tr>
<td colspan="2">Over the last 25 years, a much broader range of 
taxonomic studies of bacteria has gradually replaced the former reliance
 upon morphological, physiological, and biochemical characterization [<a href="#73">73</a>].
 The polyphasic taxonomy includes all available phenotypic and genotypic
 data and integrates them into a system of classification, derived from 
16S rRNA sequence analysis. It is conjectured that as more and more 
parameters become available in future, the polyphasic classification 
will gain increasing stability. Bacterial taxonomists did not have a 
clearly set array of rules for species definition, mainly because in 
unicellular organisms like bacteria morphology, physiology and many 
other properties are not informative enough to be used as phylogenetic 
markers. This has a telling effect on bacterial taxonomy problems. This 
problem is faced in polyphasic taxonomy, which does not depend on a 
theory, a hypothesis, or a set of rules and presents a pragmatic 
approach to a consensus type of taxonomy, integrating all available data
 maximally. In future, polyphasic taxonomy will have to cope with (i) 
enormous amounts of data, (ii) large numbers of strains, and (iii) data 
fusion (data aggregation), which will demand efficient and centralized 
data storage. Thus taxonomic studies will require collaborative efforts 
by specialized laboratories even more than now is the case [<a href="#73">73</a>,<a href="#74">74</a>].</td>
</tr>
<tr>
<td colspan="2"><font class="fulltext-heading">Discussion</font></td>
</tr>
<tr>
<td colspan="2">Although there are large numbers of phylogenetic markers
 available, the researcher should not be limited only to these genes. In
 fact, there is a need for developing additional markers for 
phylogenetic analysis. The number of genes used for phylogenetic 
analysis over plants, animals and microorganisms should be increased 
through nuclear genome sequencing and EST (expressed sequence tag) 
projects. Also, need of markers over large group of organisms is very 
crucial. Future effort should be directed towards improving the 
algorithms for various analysis softwares. The power of genes involved 
with the physiology of organisms such as the cell division (cdc) genes, 
salt tolerance genes, heat shock genes, homeotic genes, receptor genes 
etc. to mention a few, should also be explored as they show great 
homology over a large range of organisms. At the same time, efforts of 
classical biologists who have been basing their phylogeny analyses on 
morphological studies of both external and internal features of an 
organism should be encouraged. In combination with studies using 
molecular genetic markers and morphology, relatively full proof systems 
can be devised for the phylogenetic studies of Archaea and Eukarya 
groups, much in line with the polyphasic approaches described for 
bacteria.</td>
</tr>
<tr>
<td colspan="2">As time passes more data will become available, more 
novel organisms will be detected and software development will need to 
take into account the combination and linking of the different 
databases. We will also have increasing access to the genome and DNA 
sequences from many organisms will be available because of the repaid 
advances in the sequencing technologies. The most challenging task will 
definitely be to process this mass of information into a useful 
classification concept. Discovery of newer molecular markers for the 
purpose of phylogenetic studies will have to keep pace with the progress
 in the downstream techniques and analysis procedure as they generate 
the raw data on the basis of which the analyses are carried out.</td>
</tr>
<tr>
<td><font class="fulltext-heading">Acknowledgements</font></td>
</tr>
<tr>
<td colspan="2">This work has been supported in parts by grants from 
CSIR (India), UGC (India) to AR and DBT-HRD (India) grant to the 
Department of Biotechnology, Visva-Bharati University, India.</td>
</tr>
<tr>
<td colspan="2"><font class="fulltext-heading">References</font></td>
</tr>
<tr>
<td colspan="2">&nbsp;</td>
</tr>
<tr>
<td>
<ol style="padding-left: 35px">
<li value="1" id="Reference_Titile_Link"><a name="1" id="1"></a>Wilson E O (1994)<a href="http://www.hup.harvard.edu/catalog.php?isbn=9780674058170" target="_blank"> The diversity of life. Cambridge: Harvard University Press. pp. 205-260.</a> <br>
&nbsp;</li>
<li value="2" id="Reference_Titile_Link"><a name="2" id="2"></a>May RM (1990)<a href="http://rstb.royalsocietypublishing.org/content/330/1257/293" target="_blank"> How many species? Phil Trans R SocLond B 330: 293-304.</a> <br>
&nbsp;</li>
<li value="3" id="Reference_Titile_Link"><a name="3" id="3"></a>Zuckerkandl E, Pauling L (1965)<a href="http://rana.lbl.gov/290/papers/zuckerkandl_pauling.pdf" target="_blank">
 Evolutionary divergence and convergence in proteins. In: Bryson V, 
Vogel HJ. Evolving genes and proteins. Academic Press, New York. pp. 
97-166.</a> <br>
&nbsp;</li>
<li value="4" id="Reference_Titile_Link"><a name="4" id="4"></a>Sarich VM, Wilson AC (1967)<a href="https://www.ncbi.nlm.nih.gov/pubmed/4964406" target="_blank"> Immunological time scale for hominid evolution.Science 158: 1200-1203.</a> <br>
&nbsp;</li>
<li value="5" id="Reference_Titile_Link"><a name="5" id="5"></a>Zimmer 
EA, et al. (1989) Ribosomal RNA phylogenies and flowering plant 
evolution. In: Fernholm B, Bremer K, Jornvall H. The hierarchy of life. 
Elsevier, Amsterdam. <br>
&nbsp;</li>
<li value="6" id="Reference_Titile_Link"><a name="6" id="6"></a>Buchheim MA, Turmel M, Zimmer EA, Chapman RL (1990)<a href="http://onlinelibrary.wiley.com/doi/10.1111/j.0022-3646.1990.00689.x/abstract" target="_blank"> Phylogeny of Chlamydomonas (Chlorophyta) based on cladistic analysis of nuclear 18S rRNA sequence data. J Phycol 26: 689-699.</a> <br>
&nbsp;</li>
<li value="7" id="Reference_Titile_Link"><a name="7" id="7"></a>Medlin L, Elwood HJ, Stickel S, Sogin ML (1988)<a href="https://www.ncbi.nlm.nih.gov/pubmed/3224833" target="_blank"> The characterization of enzymatically amplified eukaryotic 16S-like rRNA-coding regions.Gene 71: 491-499.</a> <br>
&nbsp;</li>
<li value="8" id="Reference_Titile_Link"><a name="8" id="8"></a>Mindell DP, Honeycutt RL (1990)<a href="http://www.annualreviews.org/doi/abs/10.1146/annurev.es.21.110190.002545?journalCode=ecolsys.1" target="_blank"> Ribosomal RNA in vertebrates: Evolution and phylogenetic applications. Annu Rev EcolSyst 21: 541-566.</a> <br>
&nbsp;</li>
<li value="9" id="Reference_Titile_Link"><a name="9" id="9"></a>Graybeal A (1994)<a href="https://sysbio.oxfordjournals.org/content/43/2/174.abstract" target="_blank">
 Evaluating the phylogenetic utility of genes: A search for genes 
informative about deep divergences among vertebrates. SystBiol 43: 
174-193.</a> <br>
&nbsp;</li>
<li value="10" id="Reference_Titile_Link"><a name="10" id="10"></a>Caterino MS, Sperling FA (1999)<a href="https://www.ncbi.nlm.nih.gov/pubmed/10082616" target="_blank"> Papilio phylogeny based on mitochondrial cytochrome oxidase I and II genes.MolPhylogenetEvol 11: 122-137.</a> <br>
&nbsp;</li>
<li value="11" id="Reference_Titile_Link"><a name="11" id="11"></a>Cruickshank R (2002)<a href="http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.143.6296" target="_blank"> Molecular markers for the phylogenetic of mites and ticks. Sys App Acarol 7: 3-14.</a> <br>
&nbsp;</li>
<li value="12" id="Reference_Titile_Link"><a name="12" id="12"></a>Kjer KM (1995)<a href="https://www.ncbi.nlm.nih.gov/pubmed/8845967" target="_blank">
 Use of rRNA secondary structure in phylogenetic studies to identify 
homologous positions: an example of alignment and data presentation from
 the frogs.MolPhylogenetEvol 4: 314-330.</a> <br>
&nbsp;</li>
<li value="13" id="Reference_Titile_Link"><a name="13" id="13"></a>Yli-Mattila T, Paavanen-Huhtala S, Fenton B, Tuovinen T (2000)<a href="https://www.ncbi.nlm.nih.gov/pubmed/11345320" target="_blank">
 Species and strain identification of the predatory mite 
Euseiusfinlandicus by RAPD-PCR and ITS sequences.ExpApplAcarol 24: 
863-880.</a> <br>
&nbsp;</li>
<li value="14" id="Reference_Titile_Link"><a name="14" id="14"></a>Galtier N, Gouy M (1995)<a href="https://www.ncbi.nlm.nih.gov/pubmed/7479987" target="_blank"> Inferring phylogenies from DNA sequences of unequal base compositions.ProcNatlAcadSci U S A 92: 11317-11321.</a> <br>
&nbsp;</li>
<li value="15" id="Reference_Titile_Link"><a name="15" id="15"></a>Fox GE, Woese CR (1975)<a href="https://www.ncbi.nlm.nih.gov/pubmed/1185796" target="_blank"> The architecture of 5S rRNA and its relation to function.J MolEvol 6: 61-76.</a> <br>
&nbsp;</li>
<li value="16" id="Reference_Titile_Link"><a name="16" id="16"></a>Woese CR (1987)<a href="https://www.ncbi.nlm.nih.gov/pubmed/2439888" target="_blank"> Bacterial evolution.Microbiol Rev 51: 221-271.</a> <br>
&nbsp;</li>
<li value="17" id="Reference_Titile_Link"><a name="17" id="17"></a>Woese CR , Kandler O, Wheelis ML (1990)<a href="https://www.ncbi.nlm.nih.gov/pubmed/2112744" target="_blank">
 Towards a natural system of organisms: proposal for the domains 
Archaea, Bacteria, and Eucarya.ProcNatlAcadSci U S A 87: 4576-4579.</a> <br>
&nbsp;</li>
<li value="18" id="Reference_Titile_Link"><a name="18" id="18"></a>Goerge EF, Pechman CR, Woese CR (1977)<a href="http://ijs.sgmjournals.org/content/27/1/44" target="_blank">
 Comparative cataloging of 16S ribosomal ribonucleic acid: Molecular 
approach to prokaryotic systematics. Int J SystBacteriol 27: 44-57.</a> <br>
&nbsp;</li>
<li value="19" id="Reference_Titile_Link"><a name="19" id="19"></a>Moritz C, Dowling TE, Brown WM (1987)<a href="https://www.deepdyve.com/lp/annual-reviews/evolution-of-animal-mitochondrial-dna-relevance-for-population-biology-hRfm7o1Y0K" target="_blank"> Evolution of Animal mitochondrial DNA-revelence for population biology and systematic. Ann Rev EcolSyst 18: 269-292.</a> <br>
&nbsp;</li>
<li value="20" id="Reference_Titile_Link"><a name="20" id="20"></a>Gutell RR, Gray MW, Schnare MN (1993)<a href="https://www.ncbi.nlm.nih.gov/pubmed/8332527" target="_blank"> A compilation of large subunit (23S and 23S-like) ribosomal RNA structures: 1993.Nucleic Acids Res 21: 3055-3074.</a> <br>
&nbsp;</li>
<li value="21" id="Reference_Titile_Link"><a name="21" id="21"></a>Dubnau D, Smith I, Morell P, Marmur J (1965)<a href="https://www.ncbi.nlm.nih.gov/pubmed/4956287" target="_blank">
 Gene conservation in Bacillus species. I. Conserved genetic and nucleic
 acid base sequence homologies.ProcNatlAcadSci U S A 54: 491-498.</a> <br>
&nbsp;</li>
<li value="22" id="Reference_Titile_Link"><a name="22" id="22"></a>Clarridge JE (2004)<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC523561/" target="_blank">
 Impact of 16S rRNA Gene Sequence Analysis for identification of 
bacteria on clinical microbiology and infectious diseases. ClinMicrobiol
 Rev 17: 840–862.</a> <br>
&nbsp;</li>
<li value="23" id="Reference_Titile_Link"><a name="23" id="23"></a>Pfister P, Hobbie S, Vicens Q, Böttger EC, Westhof E (2003)<a href="https://www.ncbi.nlm.nih.gov/pubmed/14523926" target="_blank">
 The molecular basis for A-site mutations conferring aminoglycoside 
resistance: relationship between ribosomal susceptibility and X-ray 
crystal structures.Chembiochem 4: 1078-1088.</a> <br>
&nbsp;</li>
<li value="24" id="Reference_Titile_Link"><a name="24" id="24"></a>Chen K, Neimark H, Rumore P, Steinman CR (1989)<a href="https://www.ncbi.nlm.nih.gov/pubmed/2469620" target="_blank"> Broad range DNA probes for detecting and amplifying eubacterial nucleic acids.FEMS MicrobiolLett 48: 19-24.</a> <br>
&nbsp;</li>
<li value="25" id="Reference_Titile_Link"><a name="25" id="25"></a>Gray MW, Burger G, Lang BF (1999)<a href="https://www.ncbi.nlm.nih.gov/pubmed/10066161" target="_blank"> Mitochondrial evolution.Science 283: 1476-1481.</a> <br>
&nbsp;</li>
<li value="26" id="Reference_Titile_Link"><a name="26" id="26"></a>Troitsky AV, MelekhovetsYuF, Rakhimova GM, Bobrova VK, Valiejo-Roman KM, et al. (1991)<a href="https://www.ncbi.nlm.nih.gov/pubmed/1904502" target="_blank"> Angiosperm origin and early stages of seed plant evolution deduced from rRNA sequence comparisons.J MolEvol 32: 253-261.</a> <br>
&nbsp;</li>
<li value="27" id="Reference_Titile_Link"><a name="27" id="27"></a>Estabrook GF (1983)<a href="https://link.springer.com/chapter/10.1007%2F978-3-642-69024-2_32" target="_blank"> The causes of character incompatibility. In: Felsenstein J Numerical taxonomy. Springer-Verlag, Berlin. pp. 279-295.</a> <br>
&nbsp;</li>
<li value="28" id="Reference_Titile_Link"><a name="28" id="28"></a>Felsenstein J (1989)<a href="http://evolution.genetics.washington.edu/phylip.html" target="_blank"> Phylip-Phylogeny inference package version 3.2</a> <br>
&nbsp;</li>
<li value="29" id="Reference_Titile_Link"><a name="29" id="29"></a>Steele KP, HolsingerKE, Jansen RK, Taylor DW (1991)<a href="http://mbe.library.arizona.edu/data/1991/0802/6stee.pdf" target="_blank"> Assessing the reliability of 5S rRNA sequence data for phylogenetic analysis in green plants. MolBiolEvol 8: 240-248.</a> <br>
&nbsp;</li>
<li value="30" id="Reference_Titile_Link"><a name="30" id="30"></a>Manzari S, Polaszek A, Belshaw R, Quicke DL (2002)<a href="https://www.ncbi.nlm.nih.gov/pubmed/12020376" target="_blank">
 Morphometric and molecular analysis of the Encarsiainaron species-group
 (Hymenoptera: Aphelinidae), parasitoids of whiteflies (Hemiptera: 
Aleyrodidae).Bull Entomol Res 92: 165-176.</a> <br>
&nbsp;</li>
<li value="31" id="Reference_Titile_Link"><a name="31" id="31"></a>Schmidt S, Driver F, Barro PD (2006)<a href="https://www.sciencedirect.com/science/article/pii/S1439609206000055" target="_blank">
 The phylogenetic characteristics of three different 28S rRNA gene 
regions in Encarsia (Insecta, Hymenoptera, Aphelinidae). Org Divers Evol
 6: 127-139.</a> <br>
&nbsp;</li>
<li value="32" id="Reference_Titile_Link"><a name="32" id="32"></a>Brown WM, Prager EM, Wang A, Wilson AC (1982)<a href="https://www.ncbi.nlm.nih.gov/pubmed/6284948" target="_blank"> Mitochondrial DNA sequences of primates: tempo and mode of evolution.J MolEvol 18: 225-239.</a> <br>
&nbsp;</li>
<li value="33" id="Reference_Titile_Link"><a name="33" id="33"></a>Russo CA, Takezaki N, Nei M (1996)<a href="https://www.ncbi.nlm.nih.gov/pubmed/8742641" target="_blank">
 Efficiencies of different genes and different tree-building methods in 
recovering a known vertebrate phylogeny.MolBiolEvol 13: 525-536.</a> <br>
&nbsp;</li>
<li value="34" id="Reference_Titile_Link"><a name="34" id="34"></a>Zardoya R, Meyer A (1996)<a href="https://www.ncbi.nlm.nih.gov/pubmed/8752002" target="_blank">
 Phylogenetic performance of mitochondrial protein-coding genes in 
resolving relationships among vertebrates.MolBiolEvol 13: 933-942.</a> <br>
&nbsp;</li>
<li value="35" id="Reference_Titile_Link"><a name="35" id="35"></a>Zhang AB, Sota T (2007)<a href="https://www.ncbi.nlm.nih.gov/pubmed/17693098" target="_blank">
 Nuclear gene sequences resolve species phylogeny and mitochondrial 
introgression in Leptocarabus beetles showing trans-species 
polymorphisms. MolPhylEvol 45: 534-546.</a> <br>
&nbsp;</li>
<li value="36" id="Reference_Titile_Link"><a name="36" id="36"></a>Pandey PK, Dhotre DP, Dharne MS, Khadse AN, Hiremath UI, et al. (2007)<a href="https://www.academia.edu/5819708/Evaluation_of_mitochondrial_12S_rRNA_gene_in_the_identification_of_Panthera_pardus_fusca_Meyer_1794_from_field-collected_scat_samples_in_the_Western_Ghats_Maharashtra_India" target="_blank">
 Evaluation of mitochondrial 12S rRNA gene in the identification of 
Pantherapardusfusca (Meyer, 1794) from field-collected scat samples in 
the Western Ghats, Maharashtra, India. CurrSci 92: 1129-1133.</a> <br>
&nbsp;</li>
<li value="37" id="Reference_Titile_Link"><a name="37" id="37"></a>Chen CA, Wallace CC, Wolstenholme J (2002)<a href="https://www.ncbi.nlm.nih.gov/pubmed/12069546" target="_blank">
 Analysis of the mitochondrial 12S rRNA gene supports a two-clade 
hypothesis of the evolutionary history of scleractinian 
corals.MolPhylogenetEvol 23: 137-149.</a> <br>
&nbsp;</li>
<li value="38" id="Reference_Titile_Link"><a name="38" id="38"></a>Allard MW, Honeycutt RL (1992)<a href="https://www.ncbi.nlm.nih.gov/pubmed/1552839" target="_blank">
 Nucleotide sequence variation in the mitochondrial 12S rRNA gene and 
the phylogeny of African mole-rats (Rodentia: Bathyergidae).MolBiolEvol 
9: 27-40.</a> <br>
&nbsp;</li>
<li value="39" id="Reference_Titile_Link"><a name="39" id="39"></a>Bradley RD, Durish ND, Rogers DS, Miller JR, Engstrom MD, et al. (2007)<a href="https://www.ncbi.nlm.nih.gov/pubmed/19924266" target="_blank"> TOWARD A MOLECULAR PHYLOGENY FOR PEROMYSCUS: EVIDENCE FROM MITOCHONDRIAL CYTOCHROME-b SEQUENCES.J Mammal 88: 1146-1159.</a> <br>
&nbsp;</li>
<li value="40" id="Reference_Titile_Link"><a name="40" id="40"></a>Randi E, Lucchini V, Hennache A, Kimball RT, Braun EL, Ligon JD (2001)<a href="https://www.ncbi.nlm.nih.gov/pubmed/11341802" target="_blank">
 Evolution of the mitochondrial DNA control-region and cytochrome b 
genes, and the inference of phylogenetic relationships in the avian 
genus Lophura (Galliformes). MolPhylEvol 19: 187–201.</a> <br>
&nbsp;</li>
<li value="41" id="Reference_Titile_Link"><a name="41" id="41"></a>Rogaev EI,Moliaka YK, Malyarchuk BA, Kondrashov FA, Derenko MV, et al. (2006)<a href="https://www.ncbi.nlm.nih.gov/pubmed/16448217" target="_blank"> Complete mitochondrial genome and phylogeny of Pleistocene mammoth Mammuthusprimigenius.PLoSBiol 4: e73.</a> <br>
&nbsp;</li>
<li value="42" id="Reference_Titile_Link"><a name="42" id="42"></a>Pesole, Gissi C, De Chirico A, Saccone C (1999)<a href="https://www.ncbi.nlm.nih.gov/pubmed/10079281" target="_blank"> Nucleotide substitution rate of mammalian mitochondrial genomes.J MolEvol 48: 427-434.</a> <br>
&nbsp;</li>
<li value="43" id="Reference_Titile_Link"><a name="43" id="43"></a>Zurawski G, Clegg MT (1987)<a href="http://www.annualreviews.org/doi/abs/10.1146/annurev.pp.38.060187.002135" target="_blank">
 Evolution of higher plant chloroplast DNA-encoded genes: Implications 
for structure– function and phylogenetic studies. Annu Rev Plant Physiol
 38: 391–418.</a> <br>
&nbsp;</li>
<li value="44" id="Reference_Titile_Link"><a name="44" id="44"></a>Chase MW, Soltis DE, Olmstead RG, Morgan D, Les DH, et al. (1993)<a href="https://www.jstor.org/discover/10.2307/2399846?uid=3738256&amp;uid=2129&amp;uid=2&amp;uid=70&amp;uid=4&amp;sid=21104549479387" target="_blank"> Phylogenetics of seed plants: An analysis of nucleotide sequences from the plastid gene rbcL. Ann Miss Bot Gard 80: 528-580.</a> <br>
&nbsp;</li>
<li value="45" id="Reference_Titile_Link"><a name="45" id="45"></a>Zurawski G, Clegg MT, Brown AH (1984)<a href="https://www.ncbi.nlm.nih.gov/pubmed/17246206" target="_blank"> The Nature of Nucleotide Sequence Divergence between Barley and Maize Chloroplast DNA.Genetics 106: 735-749.</a> <br>
&nbsp;</li>
<li value="46" id="Reference_Titile_Link"><a name="46" id="46"></a>McCourt RM, Karol KG, Guerlesquin M, Feist M, et al. (1996)<a href="https://www.jstor.org/discover/10.2307/2445965?uid=3738256&amp;uid=2129&amp;uid=2&amp;uid=70&amp;uid=4&amp;sid=21104549479387" target="_blank">
 Phylogeny of extant genera in the family Characeae (Charales, 
Charophyceae) based on rbcL sequence and morphology. Am J Bot 83: 
125-131.</a> <br>
&nbsp;</li>
<li value="47" id="Reference_Titile_Link"><a name="47" id="47"></a>Palmer JD, Jansen RK, Michaels HJ, Chase MW, Manhart JR (1988)<a href="https://www.jstor.org/discover/10.2307/2399279?uid=3738256&amp;uid=2129&amp;uid=2&amp;uid=70&amp;uid=4&amp;sid=21104549479387" target="_blank"> Chloroplast DNA variation and plant phylogeny. Ann Miss Bot Gard 75: 1180-1206.</a> <br>
&nbsp;</li>
<li value="48" id="Reference_Titile_Link"><a name="48" id="48"></a>Liang H, Hili KW (1996)<a href="http://www.nrcresearchpress.com/doi/abs/10.1139/b96-017#.U_cIwaxingk" target="_blank"> Application of the matK gene sequences to grass systematics. Canadian J Bot 74: 125-134.</a> <br>
&nbsp;</li>
<li value="49" id="Reference_Titile_Link"><a name="49" id="49"></a>Givnish TJ, Pires JC, Graham SW, McPherson MA, Prince LM, et al. (2006)<a href="http://www.botany.wisc.edu/sytsma/pdf/ConcertedConvMonocots06.pdf" target="_blank">
 Phylogenetic relationships of monocots based on the highly informative 
plastid gene ndhF: Evidence for widespread concerted convergence. Aliso 
22: 28-51.</a> <br>
&nbsp;</li>
<li value="50" id="Reference_Titile_Link"><a name="50" id="50"></a>Kim KJ, Jansen RK (1995)<a href="https://www.ncbi.nlm.nih.gov/pubmed/7479788" target="_blank"> ndhF sequence evolution and the major clades in the sunflower family.ProcNatlAcadSci U S A 92: 10379-10383.</a> <br>
&nbsp;</li>
<li value="51" id="Reference_Titile_Link"><a name="51" id="51"></a>Zhang W (2000)<a href="https://www.ncbi.nlm.nih.gov/pubmed/10764541" target="_blank"> Phylogeny of the grass family (Poaceae) from rpl16 intron sequence data.MolPhylogenetEvol 15: 135-146.</a> <br>
&nbsp;</li>
<li value="52" id="Reference_Titile_Link"><a name="52" id="52"></a>Friendlander TP, Regier JC, Mitter C (1994)<a href="https://sysbio.oxfordjournals.org/content/43/4/511.abstract" target="_blank">
 Phylogenetic information content of five nuclear gene sequences in 
animals: Initial assessment of character sets from concordance and 
divergence studies. SystBiol 43: 511-525.</a> <br>
&nbsp;</li>
<li value="53" id="Reference_Titile_Link"><a name="53" id="53"></a>Fox MG, Sorhannus UM (2003)<a href="https://www.ncbi.nlm.nih.gov/pubmed/14733439" target="_blank"> RpoA: a useful gene for phylogenetic analysis in diatoms.J EukaryotMicrobiol 50: 471-475.</a> <br>
&nbsp;</li>
<li value="54" id="Reference_Titile_Link"><a name="54" id="54"></a>Hoot SB, Douglas AW (1998)<a href="http://www.publish.csiro.au/paper/SB98027.htm" target="_blank"> Phylogeny of the Proteaceae based on atpB and atpB-rbcLintergenic spacer region sequences. Australian Syst Bot 11: 301 – 320.</a> <br>
&nbsp;</li>
<li value="55" id="Reference_Titile_Link"><a name="55" id="55"></a>Mitra S, Stallions DR (1976)<a href="https://www.ncbi.nlm.nih.gov/pubmed/786624" target="_blank"> The role of Escherichia coli dna A gene and its integrative suppression in M13 coliphage DNA synthesis.Eur J Biochem 67: 37-45.</a> <br>
&nbsp;</li>
<li value="56" id="Reference_Titile_Link"><a name="56" id="56"></a>Tsagkarakou A, Guillemaud T, Rousset F (1996)<a href="https://www.ncbi.nlm.nih.gov/pubmed/8799741" target="_blank">
 Molecular identification of a Wolbachiaendosymbiont in a 
Tetranychusurticae strain (Acari: Tetranychidae). Insect MolBiol 5: 
217-221.</a> <br>
&nbsp;</li>
<li value="57" id="Reference_Titile_Link"><a name="57" id="57"></a>Lawrence JG, Ochman H, Hartl DL (1991)<a href="https://www.ncbi.nlm.nih.gov/pubmed/1955870" target="_blank"> Molecular and evolutionary relationships among enteric bacteria.J Gen Microbiol 137: 1911-1921.</a> <br>
&nbsp;</li>
<li value="58" id="Reference_Titile_Link"><a name="58" id="58"></a>Georgopoulos C, Ang D (1990)<a href="https://www.ncbi.nlm.nih.gov/pubmed/1983267" target="_blank"> The Escherichia coli groEchaperonins.Semin Cell Biol 1: 19-25.</a> <br>
&nbsp;</li>
<li value="59" id="Reference_Titile_Link"><a name="59" id="59"></a>Casiraghi M, Bordenstein SR, Baldo L, Lo N, Beninati T, et al. (2005)<a href="https://www.ncbi.nlm.nih.gov/pubmed/16339946" target="_blank">
 Phylogeny of Wolbachiapipientis based on gltA, groEL and ftsZ gene 
sequences: clustering of arthropod and nematode symbionts in the F 
supergroup, and evidence for further diversity in the Wolbachia 
tree.Microbiology 151: 4015-4022.</a> <br>
&nbsp;</li>
<li value="60" id="Reference_Titile_Link"><a name="60" id="60"></a>Persson C (2000)<a href="https://www.ncbi.nlm.nih.gov/pubmed/10898780" target="_blank">
 Phylogeny of the NeotropicalAlibertia group (Rubiaceae), with emphasis 
on the genus Alibertia, inferred from ITS and 5S ribosomal DNA 
sequences.Am J Bot 87: 1018-1028.</a> <br>
&nbsp;</li>
<li value="61" id="Reference_Titile_Link"><a name="61" id="61"></a>Urbanczyk H, Ast JC, Kaeding AJ, Oliver JD, Dunlap PV (2008)<a href="https://www.ncbi.nlm.nih.gov/pubmed/18359809" target="_blank"> Phylogenetic analysis of the incidence of lux gene horizontal transfer in Vibrionaceae.J Bacteriol 190: 3494-3504.</a> <br>
&nbsp;</li>
<li value="62" id="Reference_Titile_Link"><a name="62" id="62"></a>Leys R, Cooper SJB, Schwarz MP (2002)<a href="http://onlinelibrary.wiley.com/doi/10.1046/j.1095-8312.2002.00108.x/abstract" target="_blank">
 Molecular phylogeny and historical biogeography of the large carpenter 
bees, genus Xylocopa (Hymenoptera: Apidae). Biol J Linn Soc 77: 249 – 
266.</a> <br>
&nbsp;</li>
<li value="63" id="Reference_Titile_Link"><a name="63" id="63"></a>Sakamoto H, Landais S, Evrin C, Laurent-Winter C, Bârzu O, et al. (2004)<a href="https://www.ncbi.nlm.nih.gov/pubmed/15256558" target="_blank"> Structure-function relationships of UMP kinases from pyrH mutants of Gram-negative bacteria.Microbiology 150: 2153-2159.</a> <br>
&nbsp;</li>
<li value="64" id="Reference_Titile_Link"><a name="64" id="64"></a>Stine OC, Sozhamannan S, Gou Q, Zheng S, Morris JG Jr, et al. (2000)<a href="https://www.ncbi.nlm.nih.gov/pubmed/11083852" target="_blank"> Phylogeny of Vibrio cholerae based on recA sequence.Infect Immun 68: 7180-7185.</a> <br>
&nbsp;</li>
<li value="65" id="Reference_Titile_Link"><a name="65" id="65"></a>Edgecombe GD (2000)<a href="http://www.museunacional.ufrj.br/mndi/Aracnologia/pdfliteratura/Edgecombe%20et%20al%202000%20Arhtropod%20cladistics.pdf" target="_blank"> Arthropod cladistics: combined analysis of histone H3 and U2 snRNA sequences and morphology. Cladistics 16: 155–203.</a> <br>
&nbsp;</li>
<li value="66" id="Reference_Titile_Link"><a name="66" id="66"></a>Van Meer MM, Witteveldt J, Stouthamer R (1999)<a href="https://www.ncbi.nlm.nih.gov/pubmed/10469257" target="_blank"> Phylogeny of the arthropod endosymbiontWolbachia based on the wsp gene.Insect MolBiol 8: 399-408.</a> <br>
&nbsp;</li>
<li value="67" id="Reference_Titile_Link"><a name="67" id="67"></a>Kjer MK, Carle FL, Litman J, Ware J (2006)<a href="http://www.zin.ru/animalia/coleoptera/eng/ksnlibp7.htm" target="_blank"> A molecular phylogeny of Hexapoda. Arthropod SystPhyl 64: 35-44.</a> <br>
&nbsp;</li>
<li value="68" id="Reference_Titile_Link"><a name="68" id="68"></a>Kress WJ, Erickson DL (2007)<a href="https://www.ncbi.nlm.nih.gov/pubmed/17551588" target="_blank">
 A two-locus global DNA barcode for land plants: the coding rbcL gene 
complements the non-coding trnH-psbA spacer region.PLoS One 2: e508.</a> <br>
&nbsp;</li>
<li value="69" id="Reference_Titile_Link"><a name="69" id="69"></a>Chase et al. (2007)<a href="https://www.jstor.org/discover/10.2307/25065788?uid=3738256&amp;uid=2129&amp;uid=2&amp;uid=70&amp;uid=4&amp;sid=21104549479387" target="_blank"> A proposal for a standardised protocol to barcode all land plants. Taxon 56(2), 295 - 299.</a> <br>
 &nbsp;</li>
<li value="70" id="Reference_Titile_Link"><a name="70" id="70"></a>Xiong J (2006)<a href="http://www.cambridge.org/us/academic/subjects/life-sciences/genomics-bioinformatics-and-systems-biology/essential-bioinformatics" target="_blank"> Essential Bioinformatics. Cambridge University Press. pp. 127 – 168.</a> <br>
&nbsp;</li>
<li value="71" id="Reference_Titile_Link"><a name="71" id="71"></a>Kimura M (1980)<a href="https://www.ncbi.nlm.nih.gov/pubmed/7463489" target="_blank">
 A simple method for estimating evolutionary rates of base substitutions
 through comparative studies of nucleotide sequences.J MolEvol 16: 
111-120.</a> <br>
&nbsp;</li>
<li value="72" id="Reference_Titile_Link"><a name="72" id="72"></a>Vandamme P, Pot B, Gillis M, de Vos P, Kersters K, et al. (1996)<a href="https://www.ncbi.nlm.nih.gov/pubmed/8801440" target="_blank"> Polyphasic taxonomy, a consensus approach to bacterial systematics.Microbiol Rev 60: 407-438.</a> <br>
&nbsp;</li>
<li value="73" id="Reference_Titile_Link"><a name="73" id="73"></a>Gillis M, Vandamme P, De Vos P, Swings, J and Kersters K (2005)<a href="https://link.springer.com/chapter/10.1007%2F0-387-28021-9_7" target="_blank"> Polyphasic taxonomy. pp. 43 – 48</a> <br>
&nbsp;</li>
<li value="74" id="Reference_Titile_Link"><a name="74" id="74"></a>Hollingsworth PM (2011)<a href="https://www.ncbi.nlm.nih.gov/pubmed/22109553" target="_blank"> Refining the DNA barcode for land plants.ProcNatlAcadSci U S A 108: 19451-19452.</a> <br>
&nbsp;</li>
</ol>
</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p> <h4 class="fulltext-heading" id="tables_figures">Tables and Figures at a glance</h4>
<table class="table" border="0" cellspacing="0" cellpadding="0">
<tbody>
<tr>
<td width="32"><img src="table-icon.gif" class="table_img_class" alt="image" style="border:1px solid #CCCCCC; padding:3px;" onclick="openimage('https://www.omicsonline.org/JPGEBimages/2329-9002-2-131-t001.html','','scrollbars=yes,resizable=yes,width=700,height=520')" title="Click Here" width="90" height="70"></td>
</tr>
<tr>
<td><strong>Table 1</strong></td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<table width="500">
<tbody>
<tr>
<td width="32"><img src="table-icon.gif" class="table_img_class" alt="image" style="border:1px solid #CCCCCC; padding:3px;" onclick="openimage('https://www.omicsonline.org/JPGEBimages/2329-9002-2-131-b001.html','','scrollbars=yes,resizable=yes,width=700,height=430')" title="Click Here" width="90" height="70"></td>
</tr>
<tr>
<td><strong>Box 1</strong></td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<h4 class="fulltext-heading">Figures at a glance</h4>
<table class="table" border="0" cellspacing="0" cellpadding="0">
<tbody>
<tr>
<td width="32"><img src="2329-9002-2-131-g001" class="table_img_class" alt="image" style="border:1px solid #CCCCCC; padding:3px;" onclick="openimage('https://www.omicsonline.org/JPGEBimages/2329-9002-2-131-g001.html','','scrollbars=yes,resizable=yes,width=570,height=750')" title="Click Here" width="90" height="70"></td>
<td>&nbsp;</td>
<td width="32"><img src="2329-9002-2-131-g002" class="table_img_class" alt="image" style="border:1px solid #CCCCCC; padding:3px;" onclick="openimage('https://www.omicsonline.org/JPGEBimages/2329-9002-2-131-g002.html','','scrollbars=yes,resizable=yes,width=570,height=400')" title="Click Here" width="90" height="70"></td>
</tr>
<tr>
<td><strong>Figure 1</strong></td>
<td>&nbsp;</td>
 <td><strong>Figure 2</strong></td>
</tr>
</tbody>
</table> </div>

<div class="col-md-3 ">
<script type="text/javascript">
						function recomend_librarian() {
							var url=escape(document.location.href);
							//var str="Omicsonline | Full text |  ";
							var t=document.title;
							t=t.replace("OMICS International | Full text | ", "");
							window.open("https://www.omicsonline.org/recommend-to-librarian.php?title=" + t + "&url=" + url, "_blank");
						}
					</script>

<div class="panel panel-default">
<div class="panel-body text-primary"> Select your language of interest to view the total content in your interested language
<div id="google_translate_element" style="width:180px;" align="center"><div class="skiptranslate goog-te-gadget" dir="ltr" style=""><div id=":0.targetLanguage"><select class="goog-te-combo" aria-label="Language Translate Widget"><option value="" selected="selected">Select Language</option><option value="af">Afrikaans</option><option value="sq">Albanian</option><option value="am">Amharic</option><option value="ar">Arabic</option><option value="hy">Armenian</option><option value="az">Azerbaijani</option><option value="eu">Basque</option><option value="be">Belarusian</option><option value="bn">Bengali</option><option value="bs">Bosnian</option><option value="bg">Bulgarian</option><option value="ca">Catalan</option><option value="ceb">Cebuano</option><option value="ny">Chichewa</option><option value="zh-CN">Chinese (Simplified)</option><option value="zh-TW">Chinese (Traditional)</option><option value="co">Corsican</option><option value="hr">Croatian</option><option value="cs">Czech</option><option value="da">Danish</option><option value="nl">Dutch</option><option value="eo">Esperanto</option><option value="et">Estonian</option><option value="tl">Filipino</option><option value="fi">Finnish</option><option value="fr">French</option><option value="fy">Frisian</option><option value="gl">Galician</option><option value="ka">Georgian</option><option value="de">German</option><option value="el">Greek</option><option value="gu">Gujarati</option><option value="ht">Haitian Creole</option><option value="ha">Hausa</option><option value="haw">Hawaiian</option><option value="iw">Hebrew</option><option value="hi">Hindi</option><option value="hmn">Hmong</option><option value="hu">Hungarian</option><option value="is">Icelandic</option><option value="ig">Igbo</option><option value="id">Indonesian</option><option value="ga">Irish</option><option value="it">Italian</option><option value="ja">Japanese</option><option value="jw">Javanese</option><option value="kn">Kannada</option><option value="kk">Kazakh</option><option value="km">Khmer</option><option value="ko">Korean</option><option value="ku">Kurdish (Kurmanji)</option><option value="ky">Kyrgyz</option><option value="lo">Lao</option><option value="la">Latin</option><option value="lv">Latvian</option><option value="lt">Lithuanian</option><option value="lb">Luxembourgish</option><option value="mk">Macedonian</option><option value="mg">Malagasy</option><option value="ms">Malay</option><option value="ml">Malayalam</option><option value="mt">Maltese</option><option value="mi">Maori</option><option value="mr">Marathi</option><option value="mn">Mongolian</option><option value="my">Myanmar (Burmese)</option><option value="ne">Nepali</option><option value="no">Norwegian</option><option value="ps">Pashto</option><option value="fa">Persian</option><option value="pl">Polish</option><option value="pt">Portuguese</option><option value="pa">Punjabi</option><option value="ro">Romanian</option><option value="ru">Russian</option><option value="sm">Samoan</option><option value="gd">Scots Gaelic</option><option value="sr">Serbian</option><option value="st">Sesotho</option><option value="sn">Shona</option><option value="sd">Sindhi</option><option value="si">Sinhala</option><option value="sk">Slovak</option><option value="sl">Slovenian</option><option value="so">Somali</option><option value="es">Spanish</option><option value="su">Sundanese</option><option value="sw">Swahili</option><option value="sv">Swedish</option><option value="tg">Tajik</option><option value="ta">Tamil</option><option value="te">Telugu</option><option value="th">Thai</option><option value="tr">Turkish</option><option value="uk">Ukrainian</option><option value="ur">Urdu</option><option value="uz">Uzbek</option><option value="vi">Vietnamese</option><option value="cy">Welsh</option><option value="xh">Xhosa</option><option value="yi">Yiddish</option><option value="yo">Yoruba</option><option value="zu">Zulu</option></select></div>Powered by <span style="white-space:nowrap"><a class="goog-logo-link" href="https://translate.google.com/" target="_blank"><img src="https://www.gstatic.com/images/branding/googlelogo/1x/googlelogo_color_42x16dp.png" style="padding-right: 3px" alt="Google Translate" width="37px" height="14px">Translate</a></span></div></div>
<script type="text/javascript">
								function googleTranslateElementInit() {
									new google.translate.TranslateElement({
										pageLanguage: 'en'
									}, 'google_translate_element');
								}
							</script>
<script src="https://translate.google.com/translate_a/element.js?cb=googleTranslateElementInit" type="text/javascript"></script>
</div>
</div>


<a title="click here" href="#post" class="btn3d btn btn-success btn-block m-b-20">Post your comment</a>


<div class="panel panel-default full-text-right-panel">
<div class="panel-heading">
<h3 class="panel-title">Share This Article</h3>
</div>
<div class="panel-body">
<ul class="soc">
<li>
<a class="soc-facebook" href="http://rest.sharethis.com/v1/share/share?destination=facebook&amp;api_key=5ffe4e56-22bc-43ba-b939-67ecb59ff6c5&amp;url=https%3A%2F%2Fwww.omicsonline.org%2Fopen-access%2Fmolecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php%3Faid%3D30965&amp;title=Molecular+Markers+in+Phylogenetic+StudiesA+Review" target="_blank" rel="nofollow" title="Share with Facebook"></a> </li>
<li><a class="soc-twitter" href="http://rest.sharethis.com/v1/share/share?destination=twitter&amp;api_key=5ffe4e56-22bc-43ba-b939-67ecb59ff6c5&amp;url=https%3A%2F%2Fwww.omicsonline.org%2Fopen-access%2Fmolecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php%3Faid%3D30965&amp;title=Molecular+Markers+in+Phylogenetic+StudiesA+Review" target="_blank" rel="nofollow" title="Share with Twitter"></a> </li>
<li><a class="soc-linkedin" href="http://rest.sharethis.com/v1/share/share?destination=linkedin&amp;api_key=5ffe4e56-22bc-43ba-b939-67ecb59ff6c5&amp;url=https%3A%2F%2Fwww.omicsonline.org%2Fopen-access%2Fmolecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php%3Faid%3D30965&amp;title=Molecular+Markers+in+Phylogenetic+StudiesA+Review" target="_blank" rel="nofollow" title="Share with Linkdin"></a> </li>
<li><a class="soc-googleplus" href="http://rest.sharethis.com/v1/share/share?destination=googleplus&amp;api_key=5ffe4e56-22bc-43ba-b939-67ecb59ff6c5&amp;url=https%3A%2F%2Fwww.omicsonline.org%2Fopen-access%2Fmolecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php%3Faid%3D30965&amp;title=Molecular+Markers+in+Phylogenetic+StudiesA+Review" target="_blank" rel="nofollow" title="Share with Google+"></a> </li>
<li><a class="soc-pinterest" href="http://rest.sharethis.com/v1/share/share?destination=pinterest&amp;api_key=5ffe4e56-22bc-43ba-b939-67ecb59ff6c5&amp;url=https%3A%2F%2Fwww.omicsonline.org%2Fopen-access%2Fmolecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php%3Faid%3D30965&amp;title=Molecular+Markers+in+Phylogenetic+StudiesA+Review" rel="nofollow" target="_self" title="Share with Pintrest"></a> </li>
<li><a class="soc-blogger soc-icon-last" href="http://rest.sharethis.com/v1/share/share?destination=blogger&amp;api_key=5ffe4e56-22bc-43ba-b939-67ecb59ff6c5&amp;url=https%3A%2F%2Fwww.omicsonline.org%2Fopen-access%2Fmolecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php%3Faid%3D30965&amp;title=Molecular+Markers+in+Phylogenetic+StudiesA+Review" target="_blank" rel="nofollow" title="Share with Blogger"></a> </li>
</ul>
</div>
</div>


<div class="panel panel-default full-text-right-panel">
<div class="panel-heading">
<h3 class="panel-title">Relevant Topics</h3>
</div>
<div class="panel-body" style="height:300px; overflow-y:scroll;">


<ul class="list-unstyled caret-right-list">
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/agricultural-entomology-journals-articles-ppts-list.php" title="Agricultural Entomology" target="_blank"> Agricultural Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/agricultural-diversity-journals-articles-ppts-list.php" title="Agricultural diversity" target="_blank"> Agricultural diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/amphibian-journals-articles-ppts-list.php" title="Amphibian" target="_blank"> Amphibian </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/amphibian--lizards-journals-articles-ppts-list.php" title="Amphibian  Lizards" target="_blank"> Amphibian Lizards </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/amphibian-reptile-journals-articles-ppts-list.php" title="Amphibian Reptile" target="_blank"> Amphibian Reptile </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/applied-entomology-journals-articles-ppts-list.php" title="Applied Entomology" target="_blank"> Applied Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/behavioral-ecology-journals-articles-ppts-list.php" title="Behavioral ecology" target="_blank"> Behavioral ecology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biodiversity--ecosystem-journals-articles-ppts-list.php" title="Biodiversity &amp; ecosystem" target="_blank"> Biodiversity &amp; ecosystem </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biodiversity-hotspot-journals-articles-ppts-list.php" title="Biodiversity hotspot" target="_blank"> Biodiversity hotspot </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biodiversity-management-journals-articles-ppts-list.php" title="Biodiversity management" target="_blank"> Biodiversity management </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biodiversity-preservation-journals-articles-ppts-list.php" title="Biodiversity preservation" target="_blank"> Biodiversity preservation </a> </li>
 <li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biodiversity-restoration-journals-articles-ppts-list.php" title="Biodiversity restoration" target="_blank"> Biodiversity restoration </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biodiversity-solution-journals-articles-ppts-list.php" title="Biodiversity solution" target="_blank"> Biodiversity solution </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/biological-diversity-measure-journals-articles-ppts-list.php" title="Biological diversity measure" target="_blank"> Biological diversity measure </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-behavior-journals-articles-ppts-list.php" title="Bird Behavior" target="_blank"> Bird Behavior </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-biology-journals-articles-ppts-list.php" title="Bird Biology" target="_blank"> Bird Biology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-diseases-journals-articles-ppts-list.php" title="Bird Diseases" target="_blank"> Bird Diseases </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-ecology-journals-articles-ppts-list.php" title="Bird Ecology" target="_blank"> Bird Ecology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-evolution-journals-articles-ppts-list.php" title="Bird Evolution" target="_blank"> Bird Evolution </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-genomics-journals-articles-ppts-list.php" title="Bird Genomics" target="_blank"> Bird Genomics </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-guilds-journals-articles-ppts-list.php" title="Bird Guilds" target="_blank"> Bird Guilds </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-identification-journals-articles-ppts-list.php" title="Bird Identification" target="_blank"> Bird Identification </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-migration-journals-articles-ppts-list.php" title="Bird Migration" target="_blank"> Bird Migration </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-neurobiology-journals-articles-ppts-list.php" title="Bird Neurobiology" target="_blank"> Bird Neurobiology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-nische-journals-articles-ppts-list.php" title="Bird Nische" target="_blank"> Bird Nische </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-physiology-journals-articles-ppts-list.php" title="Bird Physiology" target="_blank"> Bird Physiology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-qtl-mapping-journals-articles-ppts-list.php" title="Bird QTL Mapping" target="_blank"> Bird QTL Mapping </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/bird-research-journals-articles-ppts-list.php" title="Bird Research" target="_blank"> Bird Research </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/birds-in-food-pyramid-journals-articles-ppts-list.php" title="Birds in Food Pyramid" target="_blank"> Birds in Food Pyramid </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/climate-change-effects-journals-articles-ppts-list.php" title="Climate change effects" target="_blank"> Climate change effects </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/conservational-biodiversity-journals-articles-ppts-list.php" title="Conservational biodiversity" target="_blank"> Conservational biodiversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/conservational-biology-journals-articles-ppts-list.php" title="Conservational biology" target="_blank"> Conservational biology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/diversity-and-food-security-journals-articles-ppts-list.php" title="Diversity and food security" target="_blank"> Diversity and food security </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/domestic-birds-journals-articles-ppts-list.php" title="Domestic Birds" target="_blank"> Domestic Birds </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/ecology-protection-journals-articles-ppts-list.php" title="Ecology protection" target="_blank"> Ecology protection </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/economic-entomology-journals-articles-ppts-list.php" title="Economic Entomology" target="_blank"> Economic Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/endangered-plants--animals-journals-articles-ppts-list.php" title="Endangered plants &amp; animals" target="_blank"> Endangered plants &amp; animals </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/entomology-journals-articles-ppts-list.php" title="Entomology" target="_blank"> Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/entomology-supplies-journals-articles-ppts-list.php" title="Entomology Supplies" target="_blank"> Entomology Supplies </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/environment-conservation-journals-articles-ppts-list.php" title="Environment conservation" target="_blank"> Environment conservation </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/environmental-deterioration-journals-articles-ppts-list.php" title="Environmental deterioration" target="_blank"> Environmental deterioration </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/environmental-diversity-journals-articles-ppts-list.php" title="Environmental diversity" target="_blank"> Environmental diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/evolutionary-developmental-biology-journals-articles-ppts-list.php" title="Evolutionary Developmental Biology" target="_blank"> Evolutionary Developmental Biology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/evolutionary-ecology-journals-articles-ppts-list.php" title="Evolutionary Ecology" target="_blank"> Evolutionary Ecology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/evolutionary-genomics-journals-articles-ppts-list.php" title="Evolutionary Genomics" target="_blank"> Evolutionary Genomics </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/evolutionary-systems-biology-journals-articles-ppts-list.php" title="Evolutionary Systems Biology" target="_blank"> Evolutionary Systems Biology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/field-ornithology-journals-articles-ppts-list.php" title="Field Ornithology" target="_blank"> Field Ornithology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/flea-behavior-journals-articles-ppts-list.php" title="Flea Behavior" target="_blank"> Flea Behavior </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/forensic-entomology-journals-articles-ppts-list.php" title="Forensic Entomology" target="_blank"> Forensic Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/forest-diversity-journals-articles-ppts-list.php" title="Forest diversity" target="_blank"> Forest diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/genetic-diversity-conservation-journals-articles-ppts-list.php" title="Genetic diversity conservation" target="_blank"> Genetic diversity conservation </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/global-warming-protection-journals-articles-ppts-list.php" title="Global warming protection" target="_blank"> Global warming protection </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/herpetological-conservation-and-biology-journals-articles-ppts-list.php" title="Herpetological Conservation and Biology" target="_blank"> Herpetological Conservation and Biology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/herpetological-medicine-journals-articles-ppts-list.php" title="Herpetological Medicine" target="_blank"> Herpetological Medicine </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/herpetology-journals-articles-ppts-list.php" title="Herpetology" target="_blank"> Herpetology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/human-phylogenetics-journals-articles-ppts-list.php" title="Human Phylogenetics" target="_blank"> Human Phylogenetics </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/insect-entomology-journals-articles-ppts-list.php" title="Insect Entomology" target="_blank"> Insect Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/insect-diversity-journals-articles-ppts-list.php" title="Insect diversity" target="_blank"> Insect diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/macroevolution-journals-articles-ppts-list.php" title="Macroevolution" target="_blank"> Macroevolution </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/marine-diversity-journals-articles-ppts-list.php" title="Marine diversity" target="_blank"> Marine diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/medical-entomology-journals-articles-ppts-list.php" title="Medical Entomology" target="_blank"> Medical Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/microbial-diversity-journals-articles-ppts-list.php" title="Microbial diversity" target="_blank"> Microbial diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/molecular-ecology-journals-articles-ppts-list.php" title="Molecular Ecology" target="_blank"> Molecular Ecology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/molecular-phylogenetics-journals-articles-ppts-list.php" title="Molecular Phylogenetics" target="_blank"> Molecular Phylogenetics </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/natural-environmental-protection-journals-articles-ppts-list.php" title="Natural Environmental protection" target="_blank"> Natural Environmental protection </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/natural-resources--fossils-journals-articles-ppts-list.php" title="Natural resources &amp; fossils" target="_blank"> Natural resources &amp; fossils </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/organismal-biology-journals-articles-ppts-list.php" title="Organismal Biology" target="_blank"> Organismal Biology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/organismic-evolutionary-ecology-journals-articles-ppts-list.php" title="Organismic evolutionary ecology" target="_blank"> Organismic evolutionary ecology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/ornithology-journals-articles-ppts-list.php" title="Ornithology" target="_blank"> Ornithology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/phylogenetic-analysis-journals-articles-ppts-list.php" title="Phylogenetic Analysis" target="_blank"> Phylogenetic Analysis </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/plant-phylogenetics-journals-articles-ppts-list.php" title="Plant Phylogenetics" target="_blank"> Plant Phylogenetics </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/plant-diversity-journals-articles-ppts-list.php" title="Plant diversity" target="_blank"> Plant diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/reptiles-journals-articles-ppts-list.php" title="Reptiles" target="_blank"> Reptiles </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/soil-diversity-journals-articles-ppts-list.php" title="Soil diversity" target="_blank"> Soil diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/species-diversity-journals-articles-ppts-list.php" title="Species diversity" target="_blank"> Species diversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/species-extinction-process-journals-articles-ppts-list.php" title="Species extinction process" target="_blank"> Species extinction process </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/systemic-entomology-journals-articles-ppts-list.php" title="Systemic Entomology" target="_blank"> Systemic Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/theoretical-phylogenetics-journals-articles-ppts-list.php" title="Theoretical Phylogenetics" target="_blank"> Theoretical Phylogenetics </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/threats-to-biodiversity-journals-articles-ppts-list.php" title="Threats to biodiversity" target="_blank"> Threats to biodiversity </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/urban-herpetology-journals-articles-ppts-list.php" title="Urban Herpetology" target="_blank"> Urban Herpetology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/veterinary-entomology-journals-articles-ppts-list.php" title="Veterinary Entomology" target="_blank"> Veterinary Entomology </a> </li>
<li class="right-content"> <a href="https://www.omicsonline.org/scholarly/wildlife-journals-articles-ppts-list.php" title="Wildlife" target="_blank"> Wildlife </a> </li>
</ul>
</div>
</div>


<div class="panel panel-default full-text-right-panel">
<div class="panel-heading">
<h3 class="panel-title">Recommended Journals</h3>
</div>
<div class="panel-body">
<ul class="list-unstyled caret-right-list">
<li> <a href="https://www.omicsonline.org/biodiversity-endangered-species.php" title="Biodiversity Journal" target="_self"> Biodiversity Journal </a> </li>
<li> <a href="https://www.omicsonline.org/entomology-ornithology-herpetology.php" title="Entomology Journal" target="_self"> Entomology Journal </a> </li>
</ul>
<div class="text-right"> <a title="Click here" class="btn btn-success btn-xs" href="https://www.omicsonline.org/open-access-journals-list.php" target="_blank">View More <i aria-hidden="true" class="fa fa-angle-double-right"></i></a> </div>
</div>
</div>


<div class="panel panel-default full-text-right-panel">
<div class="panel-heading">
<h3 class="panel-title">Recommended Conferences</h3>
</div>
<div class="panel-body" style="max-height:230px; overflow-y:scroll;">
<ul class="list-unstyled caret-right-list">
<li> International Conference on <a href="https://computational-biology.conferenceseries.com/"><strong>Computational Biology</strong></a> and <a href="https://computational-biology.conferenceseries.com/"><strong>Bioinformatics</strong></a><br>
Sep 05-06 2018 Tokyo, Japan </li>
<li> 12th&nbsp;International Conference on Advancements in <a href="https://bioinformatics.conferenceseries.com/speaker/2016/abstract/2016/fragments-fragmentome-and-fragmentomics-in-proteomics"><strong>Bioinformatics</strong></a> and <a href="https://bioinformatics.conferenceseries.com/speaker/2016/abstract/2016/fragments-fragmentome-and-fragmentomics-in-proteomics"><strong>Drug Discovery</strong></a><br>
November 26-27, 2018 Dublin, Ireland </li>
</ul>
<div class="text-right">
</div>
</div>
</div>


<div class="panel panel-default full-text-right-panel">
<div class="panel-heading">
<h3 class="panel-title">Article Tools</h3>
</div>
<div class="panel-body">
<ul class="list-unstyled caret-right-list">
<li><a href="https://www.omicsgroup.org/citations/export-citations.php?doi=10.4172/2329-9002.1000131" title="Export citation">Export citation</a> </li>
<li><a href="https://www.omicsgroup.org/blog-this-article/index.php?doi=10.4172/2329-9002.1000131" title="Share/Blog this article">Share/Blog this article</a> </li>
</ul>
</div>
</div>


<div class="panel panel-default full-text-right-panel">
<div class="panel-heading">
<h3 class="panel-title">Article Usage</h3>
</div>
<ul class="list-group">
<li class="list-group-item">Total views: <strong class="text-warning">
16998 </strong> </li>
<li class="list-group-item">[From(publication date): <br>
November-2014 - Jul 31, 2018]</li>
<li class="list-group-item list-group-item-info">Breakdown by view type</li>
<li class="list-group-item">HTML page views : <strong class="text-warning"> 12485 </strong> </li>
<li class="list-group-item">PDF downloads : <strong class="text-warning"> 4513 </strong> </li>
</ul>
</div>

</div>
</div>
</div>



<script language="javascript" type="text/javascript">
$(document).ready(function () {
$("#comment_form").submit(function () {
$(".error").hide();
var hasError = false;
//var emailReg = /^([\w-\.]+@([\w-]+\.)+[\w-]{2,4})?$/;

var com_name=$("#com_name").val();
var com_email=$("#com_email").val();
var comments=$("#comments").val();
var captcha=$("#captcha").val();
var journal=$("#journal").val();

$("#msgbox1").html('<img src="https://www.omicsonline.org/images/wait.gif" alt="Wait" width="128" height="15"/><br> <font color="green">Loading Please wait....</font>').fadeIn(1000);
//check the username exists or not from ajax
$.post("https://www.omicsonline.org/article-comment-postings.php", {
art_id: 30965,
journal: $('#journal').val(),
com_name: $('#com_name').val(),
com_email: $('#com_email').val(),
comments: $('#comments').val(),
captcha: $('#captcha').val(),
rand: Math.random()
}, function (data) {
if (data == 'yes') //if correct login detail
{
$("#msgbox1").fadeTo(200, 0.8, function () //start fading the messagebox
	{
		$(this).html('<font color="green">Thank you for submitting your comments. It will be displayed soon.</font>').addClass('messageboxok').fadeTo(3000, 1,
			//add message and change the class of the box and start fading
			//$(this).html('Logging in ,Please wait...').addClass('messageboxok').fadeTo(900,1,
			function () {
				//redirect to secure page
				$(this).html('<font color="green">Thank you for submitting your comments. It will be displayed soon.</font>').addClass('messageboxerror').fadeTo(900, 1);
								document.location = 'https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php?aid=30965';
			}
		);

	});
} else {
$("#msgbox1").fadeTo(200, 0.1, function () //start fading the messagebox
	{
		//add message and change the class of the box and start fading
		$(this).html('<font color="red">The Validation code does not match..!</font>').addClass('messageboxerror').fadeTo(900, 1);
	});
}

});
return false; //not to post the  form physically
});
});
</script>
<div class="container container_style">
<div class="row">
<a id="post"></a>
<div class="col-md-9">
</div>
<div class="clear">&nbsp;</div>
<div class="col-md-9">
<h2 class="fulltext-heading">Post your comment</h2>
<div id="msgbox1" style="font-size:13px; padding:3px; display:none;" align="center"></div>
<form role="form" method="post" id="comment_form" action="">
<div class="form-group">
<input class="form-control" name="journal" id="journal" value="jpgeb" type="hidden">
<label for="usr">Name: <font color="#FF0000">*</font></label>
<input class="form-control" required="" name="com_name" id="com_name" type="text">
</div>
<div class="form-group">
<label for="usr">E-mail:</label>
<input class="form-control" name="com_email" id="com_email" type="text">
</div>
<div class="form-group">
<label for="usr">Your comment: <font color="#FF0000">*</font></label>
<textarea class="form-control" id="comments" required="" name="comments" rows="5">	</textarea>
</div>
<div class="form-group">
<label for="usr">Anti Spam Code:</label>
<input id="captcha" name="captcha" required="" class="form-control" style="width:100px; font-weight:bold; color:#003399; text-align:center; font-size:14px;" maxlength="6" type="text">
<img src="captcha_code_file.jpg" id="captchaimg" alt="captcha" align="absmiddle"> &nbsp;&nbsp;<a href="javascript:%20refreshCaptcha();"><img src="refresh.jpg" alt="Reload" title="Reload" width="24" border="0" align="absmiddle" height="24"></a>&nbsp;&nbsp;Can't read the image? <a href="javascript:%20refreshCaptcha();" title="Click here">click here</a> to refresh
<p></p>
</div>
<input class="btn btn-success btn-group-sm" name="submit" value="Post your comment" type="submit">
</form>
</div>
</div>
<div class="row">
<div class="col-12" style="margin-top:5px;">
<div class="redheading" style="font-size:18px; font-weight:600; padding-top:0px;">Peer Reviewed Journals</div>
<div class="clear">&nbsp;</div>
<strong style="color:#000;">Make the best use of Scientific Research and information from our 700 + peer reviewed, <a href="https://www.omicsonline.org/open-access-journals-list.php" title="Click here">Open Access Journals</a></strong> </div>
</div>
<div class="row">

<div class="col-md-6">
<div class="panel panel-primary brd-orange-800">
<div class="panel-heading bg-orange-800 brd-orange-800">
<h3 class="panel-title font-size-18">Journals by Subject</h3>
</div>
<div class="panel-body equal-height-5 text-left font-size-12" style="height: 454.7px;">
<div class="row">
<div class="col-md-6">
<ul class="list-unstyled list-padding-xs">
<li><a title="Agri and Aquaculture" href="https://www.omicsonline.org/agri-and-aquaculture-journals-impact-factor-ranking.php">Agri and Aquaculture</a> </li>
<li><a title="Biochemistry" href="https://www.omicsonline.org/biochemistry-journals.php">Biochemistry</a> </li>
<li><a title="Bioinformatics &amp; Systems Biology" href="https://www.omicsonline.org/bioinformatics-and-systems-biology-journals-impact-factor-ranking.php">Bioinformatics &amp; Systems Biology</a> </li>
<li><a title="Biomedical Sciences" href="https://www.omicsonline.org/biomedical-sciences-journals-impact-factor-ranking.php">Biomedical Sciences</a> </li>
<li><a title="Business &amp; Management" href="https://www.omicsonline.org/business-and-management-journals.php">Business &amp; Management</a> </li>
<li><a title="Chemical Engineering" href="https://www.omicsonline.org/chemical-engineering-journals.php">Chemical Engineering</a> </li>
<li><a title="Chemistry" href="https://www.omicsonline.org/chemistry-journals.php">Chemistry</a> </li>
<li><a title="Clinical Sciences" href="https://www.omicsonline.org/clinical-journals.php">Clinical Sciences</a> </li>
<li><a title="Computer Science" href="https://www.omicsonline.org/computer-science-journals-impact-factor-ranking.php">Computer Science</a> </li>
<li><a title="Economics &amp; Accounting" href="https://www.omicsonline.org/economics-and-accounting-journals-impact-factor-ranking.php">Economics &amp; Accounting</a> </li>
<li><a title="Engineering" href="https://www.omicsonline.org/engineering-journals.php">Engineering</a> </li>
<li><a title="Environmental Sciences" href="https://www.omicsonline.org/environmental-sciences-journals-impact-factor-ranking.php">Environmental Sciences</a> </li>
<li><a title="Food &amp; Nutrition" href="https://www.omicsonline.org/food-and-nutrition-journals-impact-factor-ranking.php">Food &amp; Nutrition</a> </li>
<li><a title="General Science" href="https://www.omicsonline.org/general-science-journals.php">General Science</a> </li>
<li><a title="Genetics &amp; Molecular Biology" href="https://www.omicsonline.org/genetics-and-molecular-biology-journals.php">Genetics &amp; Molecular Biology</a> </li>
</ul>
</div>
<div class="col-md-6">
<ul class="list-unstyled list-padding-xs">
<li><a title="Geology &amp; Earth Science" href="https://www.omicsonline.org/geology-and-earth-science-journals-impact-factor-ranking.php">Geology &amp; Earth Science</a> </li>
<li><a title="Immunology &amp; Microbiology" href="https://www.omicsonline.org/immunology-microbiology-journals.php">Immunology &amp; Microbiology</a> </li>
<li><a title="Informatics" href="https://www.omicsonline.org/informatics-journals.php">Informatics</a> </li>
<li><a title="Materials Science" href="https://www.omicsonline.org/material-sciences-journals.php">Materials Science</a> </li>
<li><a title="Mathematics" href="https://www.omicsonline.org/mathematics-journals.php">Mathematics</a> </li>
<li><a title="Medical Sciences" href="https://www.omicsonline.org/medical-sciences-journals.php">Medical Sciences</a> </li>
<li><a title="Nanotechnology" href="https://www.omicsonline.org/nanotechnology-journals-impact-factor-ranking.php">Nanotechnology</a> </li>
<li><a title="Neuroscience &amp; Psychology" href="https://www.omicsonline.org/neurology-and-psychiatry-journals.php">Neuroscience &amp; Psychology</a> </li>
<li><a title="Nursing &amp; Health Care" href="https://www.omicsonline.org/nursing-and-health-journals.php">Nursing &amp; Health Care</a> </li>
<li><a title="Pharmaceutical Sciences" href="https://www.omicsonline.org/pharmaceutical-sciences-journals.php">Pharmaceutical Sciences</a> </li>
<li><a title="Physics" href="https://www.omicsonline.org/physics-journals.php">Physics</a> </li>
<li><a title="Plant Sciences" href="https://www.omicsonline.org/plant-sciences-journals-impact-factor-ranking.php">Plant Sciences</a> </li>
<li><a title="Social &amp; Political Sciences" href="https://www.omicsonline.org/social-and-political-science-journals.php">Social &amp; Political Sciences</a> </li>
<li><a title="Veterinary Sciences" href="https://www.omicsonline.org/veterinary-science-journals.php">Veterinary Sciences</a> </li>
</ul>
</div>
</div>
</div>
</div>
</div>

<div class="col-md-6">

<div class="panel panel-primary brd-orange-800">
<div class="panel-heading bg-orange-800 brd-orange-800">
<h3 class="panel-title font-size-18">Clinical &amp; Medical Journals</h3>
</div>
<div class="panel-body equal-height-5 text-left" style="height: 454.7px;">
<div class="row">
<div class="col-sm-4">
<ul class="list-unstyled list-padding-xs">
<li><a title="Anesthesiology" href="https://www.omicsonline.org/anesthesiology-journals.php">Anesthesiology</a> </li>
<li><a title="Cardiology" href="https://www.omicsonline.org/cardiology-journals.php">Cardiology</a> </li>
<li><a title="Clinical Research" href="https://www.omicsonline.org/clinical-research-journals.php">Clinical Research</a> </li>
<li><a title="Dentistry" href="https://www.omicsonline.org/dental-and-oral-health-journals.php">Dentistry</a> </li>
<li><a title="Dermatology" href="https://www.omicsonline.org/dermatology-journals.php">Dermatology</a> </li>
<li><a title="Diabetes &amp; Endocrinology" href="https://www.omicsonline.org/diabetes-and-endocrinology-journals.php">Diabetes &amp; Endocrinology</a> </li>
<li><a title="Gasteroenterology" href="https://www.omicsonline.org/gastroenterology-journals.php">Gasteroenterology</a> </li>
<li><a title="Genetics" href="https://www.omicsonline.org/genetics-journals-impact-factor-ranking.php">Genetics</a> </li>
<li><a title="Haematology" href="https://www.omicsonline.org/haematology-journals.php">Haematology</a> </li>
<li><a title="Healthcare " href="https://www.omicsonline.org/health-care-journals.php">Healthcare </a> </li>
<li><a title="Immunology" href="https://www.omicsonline.org/immunology-journals.php">Immunology</a> </li>
<li><a title="Infectious Diseases" href="https://www.omicsonline.org/infectious-diseases-journals.php">Infectious Diseases</a> </li>
<li><a title="Medicine" href="https://www.omicsonline.org/medicine-journals.php">Medicine</a> </li>
<li><a title="Microbiology" href="https://www.omicsonline.org/microbiology-journals.php">Microbiology</a> </li>
</ul>
</div>
<div class="col-sm-4">
<ul class="list-unstyled list-padding-xs">
<li><a title="Molecular Biology" href="https://www.omicsonline.org/molecular-biology-journals-impact-factor-ranking.php">Molecular Biology</a> </li>
<li><a title="Nephrology" href="https://www.omicsonline.org/nephrology-journals.php">Nephrology</a> </li>
<li><a title="Neurology" href="https://www.omicsonline.org/neurology-journals.php">Neurology</a> </li>
<li><a title="Nursing" href="https://www.omicsonline.org/nursing-journals.php">Nursing</a> </li>
<li><a title="Nutrition" href="https://www.omicsonline.org/nutrition-journals.php">Nutrition</a> </li>
<li><a title="Oncology" href="https://www.omicsonline.org/oncology-and-cancer-therapeutics-journals.php">Oncology</a> </li>
<li><a title="Ophthalmology" href="https://www.omicsonline.org/ophthalmology-journals.php">Ophthalmology</a> </li>
<li><a title="Orthopaedics" href="https://www.omicsonline.org/orthopaedics-journals.php">Orthopaedics</a> </li>
<li><a title="Pathology" href="https://www.omicsonline.org/pathology-journals.php">Pathology</a> </li>
<li><a title="Pediatrics" href="https://www.omicsonline.org/pediatrics-journals.php">Pediatrics</a> </li>
<li><a title="Physicaltherapy &amp; Rehabilitation" href="https://www.omicsonline.org/physicaltherapy-rehabilitation-journals.php">Physicaltherapy &amp; Rehabilitation</a> </li>
</ul>
</div>
<div class="col-sm-4">
<ul class="list-unstyled list-padding-xs">
<li><a title="Psychiatry" href="https://www.omicsonline.org/psychiatry-journals.php">Psychiatry</a> </li>
<li><a title="Pulmonology" href="https://www.omicsonline.org/pulmonology-journals.php">Pulmonology</a> </li>
<li><a title="Radiology" href="https://www.omicsonline.org/radiology-journals-impact-factor-ranking.php">Radiology</a> </li>
<li><a title="Reproductive Medicine" href="https://www.omicsonline.org/reproductive-medicine-journals.php">Reproductive Medicine</a> </li>
<li><a title="Surgery" href="https://www.omicsonline.org/surgery-journals.php">Surgery</a> </li>
<li><a title="Toxicology" href="https://www.omicsonline.org/toxicology-journals.php">Toxicology</a> </li>
</ul>
</div>
</div>
</div>
</div>

</div>
</div>
<div class="row">
<div class="col-md-9" style="margin-top:5px;">
<div class="redheading" style="font-size:18px; font-weight:600; padding-top:0px;">International Conferences 2018-19 </div>
<div class="clear">&nbsp;</div>
<strong style="color:#000;">Meet Inspiring Speakers and Experts at our 3000+ Global <a href="http://annualmeeting.conferenceseries.com/" target="_blank" title="Click here">Annual Meetings</a></strong> </div>
<div class="col-md-3"> <a class="sprite confrence-logo" title="OMICS International" href="https://www.conferenceseries.com/" target="_blank"></a> </div>
</div>
<div class="row">
<div class="col-md-4">

<div class="panel panel-primary brd-orange-800">
<div class="panel-heading bg-orange-800 brd-orange-800">
<h3 class="panel-title font-size-18">Conferences by Country</h3>
</div>
<div class="panel-body equal-height-6 text-left contries" style="height: 540.6px;">
<div class="row no-gutters">
<div class="col-4">
<ul class="list-unstyled list-padding">
<li><a title="USA" target="_blank" href="https://www.conferenceseries.com/usa-meetings/"><i class="us flag"></i> USA</a> </li>
<li><a title="Australia" target="_blank" href="https://www.conferenceseries.com/conference-destinations.php"><i class="au flag"></i> Australia</a> </li>
<li><a title="UAE" target="_blank" href="https://www.conferenceseries.com/uae-conferences.php"><i class="ae flag"></i> UAE</a> </li>
<li><a title="Italy" target="_blank" href="https://www.conferenceseries.com/italy-conferences.php"><i class="it flag"></i> Italy</a> </li>
<li><a title="Germany" target="_blank" href="https://www.conferenceseries.com/conference-destinations.php"><i class="de flag"></i> Germany</a> </li>
<li><a title="UK" target="_blank" href="https://www.conferenceseries.com/uk-conferences.php"><i class="gb flag"></i> UK</a> </li>
<li> <a title="Japan" target="_blank" href="https://www.conferenceseries.com/japan-conferences.php"><i class="jp flag"></i> Japan</a> </li>
<li> <a title="Brazil" target="_blank" href="https://www.conferenceseries.com/brazil-conferences.php"><i class="br flag"></i> Brazil</a> </li>
<li> <a title="South Korea" target="_blank" href="https://www.conferenceseries.com/south-korea-conferences.php"><i class="kr flag"></i> South Korea</a> </li>
<li> <a title="Netherlands" target="_blank" href="#"><i class="nl flag"></i> Netherlands</a> </li>
</ul>
</div>
<div class="col-4">
<ul class="list-unstyled list-padding">
<li> <a title="Spain" target="_blank" href="https://www.conferenceseries.com/spain-conferences.php"><i class="es flag"></i> Spain</a> </li>
<li> <a title="Canada" target="_blank" href="https://www.conferenceseries.com/conference-destinations.php"><i class="ca flag"></i> Canada</a> </li>
<li> <a title="China" target="_blank" href="https://www.conferenceseries.com/china-conferences.php"><i class="cn flag"></i> China</a> </li>
<li> <a title="France" target="_blank" href="https://www.conferenceseries.com/france-conferences.php"><i class="fr flag"></i> France</a> </li>
<li> <a title="India" target="_blank" href="https://www.conferenceseries.com/india-conferences.php"><i class="in flag"></i> India</a> </li>
<li> <a title="Malaysia" target="_blank" href="https://www.conferenceseries.com/malaysia-conferences.php"><i class="my flag"></i> Malaysia</a> </li>
<li> <a title="Singapore" target="_blank" href="https://www.conferenceseries.com/singapore-conferences.php"><i class="sg flag"></i> Singapore</a> </li>
<li> <a title="South Africa" target="_blank" href="https://www.conferenceseries.com/southafrica-conferences.php"><i class="za flag"></i> South Africa</a> </li>
<li> <a title="New Zealand" target="_blank" href="#"><i class="nz flag"></i> New Zealand</a> </li>
<li> <a title="Philippines" target="_blank" href="#"><i class="ph flag"></i> Philippines</a> </li>
</ul>
</div>
<div class="col-4">
<ul class="list-unstyled list-padding">
<li> <a title="Poland" target="_blank" href="https://www.conferenceseries.com/poland-meetings/"><i class="pl flag"></i> Poland</a> </li>
<li> <a title="Austria" target="_blank" href="https://www.conferenceseries.com/austria-meetings/"><i class="at flag"></i> Austria</a> </li>
<li> <a title="Turkey" target="_blank" href="https://www.conferenceseries.com/turkey-meetings/"><i class="tr flag"></i> Turkey</a> </li>
<li> <a title="Finland" target="_blank" href="https://www.conferenceseries.com/finland-meetings/"><i class="fi flag"></i> Finland</a> </li>
<li> <a title="Ukraine" target="_blank" href="https://www.conferenceseries.com/ukraine-meetings/"><i class="ua flag"></i> Ukraine</a> </li>
<li> <a title="Denmark" target="_blank" href="https://www.conferenceseries.com/denmark-meetings/"><i class="dk flag"></i> Denmark</a> </li>
<li> <a title="Mexico" target="_blank" href="https://www.conferenceseries.com/mexico-meetings/"><i class="mx flag"></i> Mexico</a> </li>
<li> <a title="Norway" target="_blank" href="https://www.conferenceseries.com/norway-meetings/"><i class="no flag"></i> Norway</a> </li>
<li> <a href="https://www.conferenceseries.com/romania-meetings" target="_blank" title="Romania"><i class="romania flag"></i> Romania</a> </li>
</ul>
</div>
</div>
</div>
</div>

</div>
<div class="col-md-5">

<div class="panel panel-primary brd-orange-800">
<div class="panel-heading bg-orange-800 brd-orange-800">
<h3 class="panel-title font-size-18">Medical &amp; Clinical Conferences</h3>
</div>
<div class="panel-body equal-height-6 text-left cbc" style="height: 540.6px;">
<div class="row">
<div class="col-6">
<ul class="list-unstyled list-padding">
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/microbiology-meetings/">Microbiology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/diabetes-endocrinology-meetings/">Diabetes &amp; Endocrinology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/nursing-meetings/">Nursing</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/healthcare-management-meetings/">Healthcare Management</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/neuroscience-meetings/">Neuroscience</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/immunology-meetings/">Immunology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/gastroenterology-meetings/">Gastroenterology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/genetics-molecular-biology-meetings/">Genetics &amp; Molecular Biology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/pathology-meetings/">Pathology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/alternative-healthcare-meetings/">Alternative Healthcare</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/pediatrics-meetings/">Pediatrics</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/ophthalmology-meetings/">Ophthalmology</a> </li>
</ul>
</div>
<div class="col-6">
<ul class="list-unstyled list-padding">
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/oncology-cancer-meetings/">Oncology &amp; Cancer</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/cardiology-meetings/">Cardiology</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/dentistry-meetings/">Dentistry</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/physical-therapy-rehabilitation-meetings/">Physical Therapy Rehabilitation</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/psychiatry-meetings/">Psychiatry</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/infectious-diseases-meetings/">Infectious Diseases</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/medical-ethics-health-policies-meetings/">Medical Ethics &amp; Health Policies</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/palliativecare-meetings/">Palliativecare</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/reproductive-medicine-women-healthcare-meetings/">Reproductive Medicine &amp; Women Healthcare</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/surgery-meetings/">Surgery</a> </li>
<li><a title="Click here" target="_blank" href="https://www.conferenceseries.com/radiology-meetings/">Radiology</a> </li>
</ul>
</div>
</div>
</div>
</div>

</div>
<div class="col-md-3">

<div class="panel panel-primary brd-orange-800">
<div class="panel-heading bg-orange-800 brd-orange-800">
<h3 class="panel-title font-size-18">Conferences By Subject</h3>
</div>
<ul class="list-group equal-height-6 text-left list-padding" style="height: 540.6px;">
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/pharmaceutical-sciences-meetings/">Pharmaceutical Sciences</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/pharma-marketing-industry-meetings/">Pharma Marketing &amp; Industry</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/agri-food-aqua-meetings/">Agri, Food &amp; Aqua</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/nutrition-meetings/">Nutrition</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/physics-materials-science-meetings/">Physics &amp; Materials Science</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/earth-environmental-science-meetings/">Environmental Science</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/eee-engineering-meetings/">EEE &amp; Engineering</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/veterinary-meetings/">Veterinary</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/chemical-engineering-meetings/">Chemical Engineering</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/business-management-meetings/">Business Management</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/massmedia-meetings/">Massmedia</a> </li>
<li class="list-group-item"><a title="Click here" target="_blank" href="https://www.conferenceseries.com/geology-earth-science-meetings/">Geology &amp; Earth science</a> </li>
</ul>
</div>

</div>
</div>
</div>

<div class="container-fluid footerBg">
<div class="container">
<div class="row ftrHrd">
<h3>Contact Us</h3>
</div>
<div class="row contact_list">
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Agri &amp; Aquaculture Journals
</p>
<p><span class="fa fa-user"></span> Dr. Krish </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:agriaquaculture@omicsonline.com"> agriaquaculture@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> +1-702-714-7001<span class="pding-lft">Extn: 9040</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Biochemistry Journals </p>
<p><span class="fa fa-user"></span> Datta A </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:biochemjournals@omicsonline.com"> biochemjournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9037</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Business &amp; Management Journals </p>
<p><span class="fa fa-user"></span> Ronald </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:business@omicsonline.com"> business@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9042</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Chemistry Journals </p>
<p><span class="fa fa-user"></span> Gabriel Shaw </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:chemistryjournals@omicsonline.com"> chemistryjournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9040</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 138px;">
<p> Clinical Journals </p>
<p><span class="fa fa-user"></span> Datta A </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:clinicaljournals@omicsonline.com"> clinicaljournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9037</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 138px;">
<p> Engineering Journals </p>
<p><span class="fa fa-user"></span> James Franklin </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:engineeringjournals@omicsonline.com"> engineeringjournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9042</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 138px;">
<p> Food &amp; Nutrition Journals </p>
<p><span class="fa fa-user"></span> Katie Wilson </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:nutritionjournals@omicsonline.com"> nutritionjournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9042</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 138px;">
<p> General Science </p>
<p><span class="fa fa-user"></span> Andrea Jason </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:generalscience@omicsonline.com"> generalscience@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9043</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Genetics &amp; Molecular Biology Journals </p>
<p><span class="fa fa-user"></span> Anna Melissa </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:geneticsmolbio@omicsonline.com"> geneticsmolbio@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9006</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Immunology &amp; Microbiology Journals </p>
<p><span class="fa fa-user"></span> David Gorantl </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:immunomicrobiol@omicsonline.com"> immunomicrobiol@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9014</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Materials Science Journals </p>
<p><span class="fa fa-user"></span> Rachle Green </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:materialsci@omicsonline.com"> materialsci@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9039</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Nursing &amp; Health Care Journals </p>
<p><span class="fa fa-user"></span> Stephanie Skinner </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:nursinghealthcare@omicsonline.com"> nursinghealthcare@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9039</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Medical Journals </p>
<p><span class="fa fa-user"></span> Nimmi Anna </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:medicaljournals@omicsonline.com"> medicaljournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9038</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Neuroscience &amp; Psychology Journals </p>
<p><span class="fa fa-user"></span> Nathan T </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:neuropsychology@omicsonline.com"> neuropsychology@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9041</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Pharmaceutical Sciences Journals </p>
<p><span class="fa fa-user"></span> Ann Jose </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:pharmajournals@omicsonline.com"> pharmajournals@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9007</span> </p>
</div>
<div class="col-md-3 content-grid-item equal-height-9 brd-dotted-after brd-dotted-before ftr-grid" style="height: 158px;">
<p> Social &amp; Political Science Journals </p>
<p><span class="fa fa-user"></span> Steve Harry </p>
<p> <span class="fa fa-envelope-o"></span> <a title="Click Here" href="mailto:social_politicalsci@omicsonline.com"> social_politicalsci@omicsonline.com </a> </p>
<p><span class="fa fa-phone"></span> 1-702-714-7001<span class="pding-lft">Extn: 9042</span> </p>
</div>
</div>
</div>
<div class="clear">&nbsp;</div>
<div class="container" style="color:#FFF; text-align:center;"> © 2008- 2018 <a href="https://www.omicsonline.com/" target="_blank" title="OMICS International" alt="OMICS International" style="color:#FFFFFF; text-decoration:underline;">OMICS International</a> - Open Access Publisher. Best viewed in <a href="https://mozilla.org/" target="_blank" style="color:#FFFFFF; text-decoration:underline;" title="Mozilla Firefox" rel="nofollow">Mozilla Firefox</a> | <a href="https://support.google.com/chrome/bin/answer.py?hl=en&amp;answer=95346" target="_blank" title="Google Chrome" style="color:#FFFFFF; text-decoration:underline;" rel="nofollow">Google Chrome</a> | <a href="https://www.microsoft.com/en-in/download/internet-explorer-7-details.aspx" title="Above IE 7.0 version" target="_blank" rel="nofollow" style="color:#FFFFFF; text-decoration:underline;">Above IE 7.0 version</a> </div>
</div>

<div class="scroll-top-wrapper"> <span class="scroll-top-inner"> <i class="fa fa-2x fa-arrow-circle-up"></i> </span> </div>

<script src="https://code.jquery.com/jquery-3.3.1.min.js" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" type="text/javascript"></script>
<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js" type="text/javascript"></script>

<script type="text/javascript">
function refreshCaptcha() {
var img = document.images[ 'captchaimg' ];
img.src = img.src.substring(0, img.src.lastIndexOf("?")) + "?rand=" + Math.random() * 1000;
}
</script>
<script type="text/javascript">
$(function () {
$('[data-toggle="popover"]').popover()
})
</script>
<script type="text/javascript" src="/js/grids.min.js"></script>
<script type="text/javascript">
// Equal Height
var EqualHeight = function () {
//"use strict";

// Handle Equal Height
var handleEqualHeight = function () {
$(function ($) {
for (var count = 1; count < 17; count++) {
$('.equal-height-' + count).responsiveEqualHeightGrid();
}
});
}

return {
init: function () {
handleEqualHeight(); // initial setup for equal height
}
}
}();

$(document).ready(function () {
EqualHeight.init();
});
</script>
<script type="text/javascript">
_atrk_opts = {
atrk_acct: "AQs4j1a4ZP00a6",
domain: "omicsonline.org",
dynamic: true
};
(function () {
var as = document.createElement('script');
as.type = 'text/javascript';
as.async = true;
as.src = "https://d31qbv1cthcecs.cloudfront.net/atrk.js";
var s = document.getElementsByTagName('script')[ 0 ];
s.parentNode.insertBefore(as, s);
})();
</script>
<noscript>
<img src="https://d5nxst8fruw4z.cloudfront.net/atrk.gif?account=AQs4j1a4ZP00a6" style="display:none" height="1" width="1" alt="alexa"/>
</noscript>

<script type="text/javascript">
$(function () {
$(document).on('scroll', function () {
if ($(window).scrollTop() > 100) {
$('.scroll-top-wrapper').addClass('show');
} else {
$('.scroll-top-wrapper').removeClass('show');
}
});
$('.scroll-top-wrapper').on('click', scrollToTop);
});

function scrollToTop() {
verticalOffset = typeof (verticalOffset) != 'undefined' ? verticalOffset : 0;
element=$('body');
offset = element.offset();
offsetTop = offset.top;
$('html, body').animate({
scrollTop: offsetTop
}, 500, 'linear');
}
</script>

<a id="chat" class="btn p-0 border-0 rounded-0 leave-your-message font-size-20 white d-flex flex-column text-center fweight-600" data-toggle="modal" data-target="#leaveAMessage" style="transform: rotate(90deg); transform-origin: 90% 30%;"> <span class="px-1" style="background-color: #185fa1; text-shadow: 1px 1px 1px #666;
">Leave Your Message</span> <span style="background-color: #ff5a00;
"><i class="fas fa-envelope"></i> 24x7 <i class="fas fa-phone-volume"></i></span> </a>


<div class="modal fade" id="leaveAMessage" tabindex="-1" role="dialog" aria-labelledby="leaveAMessagelLabel" aria-hidden="true">
<div class="modal-dialog modal-lg" role="document">
<form role="form" id="leavemessage_form" name="leavemessage_form" action="" method="post">
<div class="modal-content">
<div class="modal-header">
<h3 class="modal-title text-center" id="leaveAMessageLabel"> Journal of Phylogenetics &amp; Evolutionary Biology </h3>
<button type="button" class="close" data-dismiss="modal" aria-label="Close"> <span aria-hidden="true">×</span> </button>
</div>
<div class="modal-body">
<div class="container-fluid">
<div class="row">
<div class="col">
<p class="text-center text-success"><strong>Please leave a message, we will get back you shortly.</strong></p>
<div id="msgbox2" class="text-center text-success" style="display:none;" align="center"></div>
<input class="form-control" name="journal" id="journal" value="jpgeb" type="hidden">
<input class="form-control" name="art_id" id="art_id" value="30965" type="hidden">
<div class="form-group">
<label for="lm_name">Name</label>
<input class="form-control" name="lm_name" id="lm_name" placeholder="Enter Name" required="" type="name">
</div>
<div class="form-group">
<label for="lm_email">Email</label>
<input class="form-control" name="lm_email" id="lm_email" placeholder="Enter Email" required="" type="email">
</div>
<div class="form-group">
<label for="lm_phone">Mobile Number</label>
<input class="form-control" name="lm_phone" id="lm_phone" placeholder="Enter Mobile Number" required="" type="text">
</div>
<div class="form-group">
<label for="lm_country">Country</label>
<select class="form-control" name="lm_country" id="lm_country" required="">
<option value="" selected="selected">Select country</option>
<option value="Afghanistan"> Afghanistan</option>
<option value="Albania"> Albania</option>
<option value="Algeria"> Algeria</option>
<option value="American Samoa"> American Samoa</option>
<option value="Andorra"> Andorra</option>
<option value="Angola"> Angola</option>
<option value="Anguilla"> Anguilla</option>
<option value="Antarctica"> Antarctica</option>
<option value="Antigua and Barbuda"> Antigua and Barbuda</option>
<option value="Argentina"> Argentina</option>
<option value="Armenia"> Armenia</option>
<option value="Aruba"> Aruba</option>
<option value="Australia"> Australia</option>
<option value="Austria"> Austria</option>
<option value="AZ USA"> AZ USA</option>
<option value="Azerbaijan"> Azerbaijan</option>
<option value="Bahamas"> Bahamas</option>
<option value="Bahrain"> Bahrain</option>
<option value="Bangladesh"> Bangladesh</option>
<option value="Barbados"> Barbados</option>
<option value="Belarus"> Belarus</option>
<option value="Belgium"> Belgium</option>
<option value="Belize"> Belize</option>
<option value="Benin"> Benin</option>
<option value="BENIN Republic"> BENIN Republic</option>
<option value="Bermuda"> Bermuda</option>
<option value="Bhutan"> Bhutan</option>
<option value="Bolivia"> Bolivia</option>
<option value="Bosnia"> Bosnia</option>
<option value="Bosnia and Herzegovina"> Bosnia and Herzegovina</option>
<option value="Botswana"> Botswana</option>
<option value="Bouvet Island"> Bouvet Island</option>
<option value="Brasil"> Brasil</option>
<option value="Brazil"> Brazil</option>
<option value="British Indian Ocean Territory"> British Indian Ocean Territory</option>
<option value="British Virgin Islands"> British Virgin Islands</option>
<option value="Brunei"> Brunei</option>
<option value="Brunei Darussalam"> Brunei Darussalam</option>
<option value="Bulgaria"> Bulgaria</option>
<option value="Burkina Faso"> Burkina Faso</option>
<option value="Burma"> Burma</option>
<option value="Burundi"> Burundi</option>
<option value="Cambodia"> Cambodia</option>
<option value="Cameroon"> Cameroon</option>
<option value="Canada"> Canada</option>
<option value="Cape Verde"> Cape Verde</option>
<option value="Cayman Islands"> Cayman Islands</option>
<option value="Central Africa"> Central Africa</option>
<option value="Central African Republic"> Central African Republic</option>
<option value="Chad"> Chad</option>
<option value="Chile"> Chile</option>
<option value="China"> China</option>
<option value="Christmas Island"> Christmas Island</option>
<option value="CO USA"> CO USA</option>
<option value="Colombia"> Colombia</option>
<option value="Comoros"> Comoros</option>
<option value="Congo"> Congo</option>
<option value="Cook Islands"> Cook Islands</option>
<option value="Costa Rica"> Costa Rica</option>
<option value="Cote D'Ivoire"> Cote D'Ivoire</option>
<option value="Croatia"> Croatia</option>
<option value="Cuba"> Cuba</option>
<option value="Curacao"> Curacao</option>
<option value="Cyprus"> Cyprus</option>
<option value="Czech Republic"> Czech Republic</option>
<option value="Democratic Republic of Congo"> Democratic Republic of Congo</option>
<option value="Democratic Republic of the Congo"> Democratic Republic of the Congo</option>
<option value="Denmark"> Denmark</option>
<option value="Djibouti"> Djibouti</option>
<option value="Dominica"> Dominica</option>
<option value="Dominican Republic"> Dominican Republic</option>
<option value="DR Congo"> DR Congo</option>
<option value="Ecuador"> Ecuador</option>
<option value="Egypt"> Egypt</option>
<option value="El Salvador"> El Salvador</option>
<option value="England"> England</option>
<option value="Equatorial Guinea"> Equatorial Guinea</option>
<option value="Eritrea"> Eritrea</option>
<option value="Espana"> Espana</option>
<option value="Estonia"> Estonia</option>
<option value="Ethiopia"> Ethiopia</option>
<option value="Falkland Islands"> Falkland Islands</option>
<option value="Faroe Islands"> Faroe Islands</option>
<option value="Fiji"> Fiji</option>
<option value="Finland"> Finland</option>
<option value="France"> France</option>
<option value="French Guiana"> French Guiana</option>
<option value="French Polynesia"> French Polynesia</option>
<option value="French Southern Territories"> French Southern Territories</option>
<option value="Gabon"> Gabon</option>
<option value="Gambia"> Gambia</option>
<option value="Georgia"> Georgia</option>
<option value="Germany"> Germany</option>
<option value="Ghana"> Ghana</option>
<option value="Gibraltar"> Gibraltar</option>
<option value="Greece"> Greece</option>
<option value="Greenland"> Greenland</option>
<option value="Grenada"> Grenada</option>
<option value="Guadeloupe"> Guadeloupe</option>
<option value="Guam"> Guam</option>
<option value="Guatemala"> Guatemala</option>
<option value="Guinea"> Guinea</option>
<option value="Guinea-Bissau"> Guinea-Bissau</option>
<option value="Guyana"> Guyana</option>
<option value="Haiti"> Haiti</option>
<option value="Heard Island"> Heard Island</option>
 <option value="Honduras"> Honduras</option>
<option value="Hong Kong"> Hong Kong</option>
<option value="Hong Kong SAR"> Hong Kong SAR</option>
<option value="Hungary"> Hungary</option>
<option value="Iceland"> Iceland</option>
<option value="India"> India</option>
<option value="Indonesia"> Indonesia</option>
<option value="Iran"> Iran</option>
<option value="Iraq"> Iraq</option>
<option value="Ireland"> Ireland</option>
<option value="Islamic Republic of Iran"> Islamic Republic of Iran</option>
<option value="Islamic Republic of Pakistan"> Islamic Republic of Pakistan</option>
<option value="Israel"> Israel</option>
<option value="Isreal"> Isreal</option>
<option value="Italy"> Italy</option>
<option value="Ivory Coast"> Ivory Coast</option>
<option value="Jamaica"> Jamaica</option>
<option value="Japan"> Japan</option>
<option value="Jordan"> Jordan</option>
<option value="Kazakhstan"> Kazakhstan</option>
<option value="Kenya"> Kenya</option>
<option value="Kingdom of Bahrain"> Kingdom of Bahrain</option>
<option value="Kingdom of Saudi Arabia"> Kingdom of Saudi Arabia</option>
<option value="Kingdom of the Netherlands"> Kingdom of the Netherlands</option>
<option value="Kiribati"> Kiribati</option>
<option value="Korea"> Korea</option>
<option value="Korea"> Korea</option>
<option value="Korea, Rep"> Korea, Rep</option>
<option value="Kosova"> Kosova</option>
<option value="Kosovo"> Kosovo</option>
<option value="KSA"> KSA</option>
<option value="Kurdistan"> Kurdistan</option>
<option value="Kuwait"> Kuwait</option>
<option value="Kyrgyzstan"> Kyrgyzstan</option>
<option value="Laos"> Laos</option>
<option value="Latvia"> Latvia</option>
<option value="Lebanon"> Lebanon</option>
<option value="Lesotho"> Lesotho</option>
<option value="Liberia"> Liberia</option>
<option value="Libya"> Libya</option>
<option value="Libyan Arab Jamahiriya"> Libyan Arab Jamahiriya</option>
<option value="Liechtenstein"> Liechtenstein</option>
<option value="Lithuania"> Lithuania</option>
<option value="Luxembourg"> Luxembourg</option>
<option value="Macao"> Macao</option>
<option value="Macedonia"> Macedonia</option>
<option value="Madagascar"> Madagascar</option>
<option value="Malawi"> Malawi</option>
<option value="Malaysia"> Malaysia</option>
<option value="Maldives"> Maldives</option>
<option value="Mali"> Mali</option>
<option value="Malta"> Malta</option>
<option value="Marshall Islands"> Marshall Islands</option>
<option value="Martinique"> Martinique</option>
<option value="Mauritania"> Mauritania</option>
<option value="Mauritius"> Mauritius</option>
<option value="Mayotte"> Mayotte</option>
<option value="Mexico"> Mexico</option>
<option value="Micronesia"> Micronesia</option>
<option value="Monaco"> Monaco</option>
<option value="Mongolia"> Mongolia</option>
<option value="Montenegro"> Montenegro</option>
<option value="Montserrat"> Montserrat</option>
<option value="Morocco"> Morocco</option>
<option value="Mozambique"> Mozambique</option>
<option value="Myanmar"> Myanmar</option>
<option value="Namibia"> Namibia</option>
<option value="Nauru"> Nauru</option>
<option value="Nepal"> Nepal</option>
<option value="Netherlands"> Netherlands</option>
<option value="Netherlands Antilles"> Netherlands Antilles</option>
<option value="New Caledonia"> New Caledonia</option>
<option value="New Mexico"> New Mexico</option>
<option value="New Zealand"> New Zealand</option>
<option value="Nicaragua"> Nicaragua</option>
<option value="Niger"> Niger</option>
<option value="Nigeria"> Nigeria</option>
<option value="Niue"> Niue</option>
<option value="Norfolk Island"> Norfolk Island</option>
<option value="North Africa"> North Africa</option>
<option value="North Cyprus"> North Cyprus</option>
<option value="North Ethiopia"> North Ethiopia</option>
<option value="Northern Cyprus"> Northern Cyprus</option>
<option value="Northern Ethiopia"> Northern Ethiopia</option>
<option value="Northern Mariana Islands"> Northern Mariana Islands</option>
<option value="Norway"> Norway</option>
<option value="Oman"> Oman</option>
<option value="P.R. China"> P.R. China</option>
<option value="Pakistan"> Pakistan</option>
<option value="Palau"> Palau</option>
<option value="Palestine"> Palestine</option>
<option value="Palestinian Territory"> Palestinian Territory</option>
<option value="Panama"> Panama</option>
 <option value="Papua New Guinea"> Papua New Guinea</option>
<option value="Paraguay"> Paraguay</option>
<option value="People’s Republic of China"> People’s Republic of China</option>
<option value="Peru"> Peru</option>
<option value="Philippines"> Philippines</option>
<option value="Pitcairn"> Pitcairn</option>
<option value="Poland"> Poland</option>
<option value="Portugal"> Portugal</option>
<option value="PR China"> PR China</option>
<option value="Puerto Rico"> Puerto Rico</option>
<option value="Qatar"> Qatar</option>
<option value="République de Côte d’Ivoire"> République de Côte d’Ivoire</option>
<option value="R. Macedonia"> R. Macedonia</option>
<option value="Republic of Armenia"> Republic of Armenia</option>
<option value="Republic of Belarus"> Republic of Belarus</option>
<option value="Republic of Benin"> Republic of Benin</option>
<option value="Republic of China"> Republic of China</option>
<option value="Republic of Indonesia"> Republic of Indonesia</option>
<option value="Republic of Iraq"> Republic of Iraq</option>
<option value="Republic of Ireland"> Republic of Ireland</option>
<option value="Republic of Kazakhstan"> Republic of Kazakhstan</option>
<option value="Republic of Korea"> Republic of Korea</option>
<option value="Republic of Kosovo"> Republic of Kosovo</option>
<option value="Republic of Macedonia"> Republic of Macedonia</option>
<option value="Republic of Mauritius"> Republic of Mauritius</option>
<option value="Republic of Moldova"> Republic of Moldova</option>
<option value="Republic of Singapore"> Republic of Singapore</option>
<option value="Republic of South Africa"> Republic of South Africa</option>
<option value="Republic of South Korea"> Republic of South Korea</option>
<option value="Republic of the Philippines"> Republic of the Philippines</option>
<option value="Republic of Tunisia"> Republic of Tunisia</option>
<option value="Republic of Uzbekistan"> Republic of Uzbekistan</option>
<option value="Republic of Yemen"> Republic of Yemen</option>
<option value="Republica Argentina"> Republica Argentina</option>
<option value="Reunion"> Reunion</option>
<option value="Romania"> Romania</option>
<option value="Russia"> Russia</option>
<option value="Russian Federation"> Russian Federation</option>
<option value="Rwanda"> Rwanda</option>
<option value="Saint Helena"> Saint Helena</option>
<option value="Saint Kitts and Nevis"> Saint Kitts and Nevis</option>
<option value="Saint Lucia"> Saint Lucia</option>
<option value="Saint Pierre and Miquelon"> Saint Pierre and Miquelon</option>
<option value="Saint Vincent and the Grenadines"> Saint Vincent and the Grenadines</option>
<option value="Samoa"> Samoa</option>
<option value="San Marino"> San Marino</option>
<option value="Sao Tome and Principe"> Sao Tome and Principe</option>
<option value="Saudi Arabia"> Saudi Arabia</option>
<option value="SCOTLAND"> SCOTLAND</option>
<option value="Senegal"> Senegal</option>
<option value="Seoul Korea"> Seoul Korea</option>
<option value="Serbia"> Serbia</option>
<option value="Serbia and Montenegro"> Serbia and Montenegro</option>
<option value="Seychelles"> Seychelles</option>
<option value="Sierra Leone"> Sierra Leone</option>
<option value="Singapore"> Singapore</option>
<option value="Slovak Republic"> Slovak Republic</option>
<option value="Slovakia"> Slovakia</option>
<option value="Slovenia"> Slovenia</option>
<option value="Solomon Islands"> Solomon Islands</option>
<option value="Somalia"> Somalia</option>
<option value="South Africa"> South Africa</option>
<option value="South Georgia"> South Georgia</option>
<option value="South Korea"> South Korea</option>
<option value="South Sudan"> South Sudan</option>
<option value="South-West Ethiopia"> South-West Ethiopia</option>
<option value="Southern Ethiopia"> Southern Ethiopia</option>
<option value="Spain"> Spain</option>
<option value="Sri Lanka"> Sri Lanka</option>
<option value="State of Kuwait"> State of Kuwait</option>
<option value="State of Palestine"> State of Palestine</option>
<option value="Sudan"> Sudan</option>
<option value="Sultanate of Oman"> Sultanate of Oman</option>
<option value="Suriname"> Suriname</option>
<option value="Svalbard and Jan Mayen"> Svalbard and Jan Mayen</option>
<option value="Swaziland"> Swaziland</option>
<option value="Sweden"> Sweden</option>
<option value="Switzerland"> Switzerland</option>
<option value="Syria"> Syria</option>
<option value="Syrian Arab Republic"> Syrian Arab Republic</option>
<option value="Taiwan"> Taiwan</option>
<option value="Taiwan"> Taiwan</option>
<option value="Taiwan, R.O.C"> Taiwan, R.O.C</option>
<option value="Tajikistan"> Tajikistan</option>
<option value="Tanzania"> Tanzania</option>
<option value="Tanzania"> Tanzania</option>
<option value="Thailand"> Thailand</option>
<option value="The Czech Republic"> The Czech Republic</option>
<option value="The Kingdom of Saudi Arabia"> The Kingdom of Saudi Arabia</option>
<option value="The Netherlands"> The Netherlands</option>
<option value="The People’s Republic of China"> The People’s Republic of China</option>
<option value="The Republic of Kenya"> The Republic of Kenya</option>
<option value="The Republic of Korea"> The Republic of Korea</option>
<option value="Timor-Leste"> Timor-Leste</option>
<option value="Togo"> Togo</option>
<option value="Tokelau"> Tokelau</option>
<option value="Tonga"> Tonga</option>
<option value="Trinidad"> Trinidad</option>
<option value="Trinidad and Tobago"> Trinidad and Tobago</option>
<option value="Tunisia"> Tunisia</option>
<option value="Tunisie"> Tunisie</option>
<option value="Turkey"> Turkey</option>
<option value="Turkish Republic of Northern Cyprus"> Turkish Republic of Northern Cyprus</option>
<option value="Turkmenistan"> Turkmenistan</option>
<option value="Turks and Caicos Islands"> Turks and Caicos Islands</option>
<option value="Tuvalu"> Tuvalu</option>
<option value="UAE"> UAE</option>
<option value="Uganda"> Uganda</option>
<option value="UK"> UK</option>
<option value="Ukraine"> Ukraine</option>
<option value="United Arab Emirates"> United Arab Emirates</option>
<option value="United Kingdom"> United Kingdom</option>
<option value="United Republic of Tanzania"> United Republic of Tanzania</option>
<option value="United States"> United States</option>
<option value="United States Minor Outlying Islands"> United States Minor Outlying Islands</option>
<option value="United States of America"> United States of America</option>
<option value="United States Virgin Islands"> United States Virgin Islands</option>
<option value="Unites Arab Emirates"> Unites Arab Emirates</option>
<option value="Uruguay"> Uruguay</option>
<option value="USA"> USA</option>
<option value="Uzbekistan"> Uzbekistan</option>
<option value="Vanuatu"> Vanuatu</option>
<option value="Vatican City"> Vatican City</option>
<option value="Venezuela"> Venezuela</option>
<option value="Vietnam"> Vietnam</option>
<option value="Virginia"> Virginia</option>
<option value="Wallis and Futuna"> Wallis and Futuna</option>
<option value="West Africa"> West Africa</option>
<option value="West Indies"> West Indies</option>
<option value="Western Australia"> Western Australia</option>
<option value="Western Ethiopia"> Western Ethiopia</option>
<option value="Western Sahara"> Western Sahara</option>
<option value="Yemen"> Yemen</option>
<option value="Zambia"> Zambia</option>
<option value="Zimbabwe"> Zimbabwe</option>
</select>
</div>
<div class="form-group">
<label for="lm_ri">Research Interest</label>
<input class="form-control" name="lm_ri" id="lm_ri" placeholder="Enter Your Research Here" type="text">
</div>
<div class="form-group">
<label for="lm_msg">Message</label>
<textarea class="form-control" rows="3" name="lm_msg" id="lm_msg" placeholder="Enter Message "></textarea>
</div>
<div class="form-group">
<label for="usr">Anti Spam Code:</label>
<input id="captcha1" name="captcha1" required="" class="form-control" style="width:100px; font-weight:bold; color:#003399; text-align:center; font-size:14px;" maxlength="6" type="text">
<img src="captcha_code_file1.jpg" id="captchaimg1" alt="captcha" align="absmiddle"> &nbsp;&nbsp;<a href="javascript:%20refreshCaptcha1();"><img src="refresh.jpg" alt="Reload" title="Reload" width="24" border="0" align="absmiddle" height="24"></a>&nbsp;&nbsp;Can't read the image? <a href="javascript:%20refreshCaptcha1();" title="Click here">click here</a> to refresh
<p></p>
</div>
</div>
</div>
</div>
</div>
<div class="modal-footer">
<button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>

<input class="btn btn-primary" name="submitnew" value="Submit" type="submit">
</div>
</div>
</form>
</div>
</div>
<script type="text/javascript">
function refreshCaptcha1() {
var img = document.images[ 'captchaimg1' ];
img.src = img.src.substring(0, img.src.lastIndexOf("?")) + "?rand=" + Math.random() * 1000;
}
</script>
<script type="text/javascript">
$(document).ready(function () {
$("#leavemessage_form").submit(function () {

$(".error").hide();
var hasError = false;
//var emailReg = /^([\w-\.]+@([\w-]+\.)+[\w-]{2,4})?$/;
var lm_name=$("#lm_name").val();
var lm_email=$("#lm_email").val();
var lm_phone=$("#lm_phone").val();
var lm_country=$("#lm_country").val();
var lm_ri=$("#lm_ri").val();
var lm_msg=$("#lm_msg").val();
var captcha1=$("#captcha1").val();
var journal=$("#journal").val();
var art_id=$("#art_id").val();

$("#msgbox2").html('<img src="https://www.omicsonline.org/images/wait.gif" alt="Wait" width="128" height="15"/><br> <font color="green">Loading Please wait....</font>').fadeIn(1000);
//check the username exists or not from ajax
$.post("https://www.omicsonline.org/leave-message-postings.php", {
journal: journal,
art_id: art_id,
lm_name: lm_name,
lm_email: lm_email,
lm_phone: lm_phone,
lm_country: lm_country,
lm_ri: lm_ri,
lm_msg: lm_msg,
captcha1: captcha1,
rand: Math.random()
}, function (data) {
if (data == 'yes') //if correct login detail
{
$("#msgbox2").fadeTo(200, 0.8, function () //start fading the messagebox
	{
		$(this).html('<font color="green">Thank you for submitting your message. Our team will get back soon.</font>').addClass('messageboxok').fadeTo(3000, 1,
			//add message and change the class of the box and start fading
			//$(this).html('Logging in ,Please wait...').addClass('messageboxok').fadeTo(900,1,
			function () {
				//redirect to secure page
				$(this).html('<font color="green">Thank you for submitting your message. Our team will get back soon.</font>').addClass('messageboxerror').fadeTo(900, 1);
								document.location = 'https://www.omicsonline.org/open-access/molecular-markers-in-phylogenetic-studiesa-review-2329-9002-2-131.php?aid=30965';
			}
		);

	});
} else {
$("#msgbox2").fadeTo(200, 0.1, function () //start fading the messagebox
	{
		//add message and change the class of the box and start fading
		$(this).html('<font color="red">The Validation code does not match..!</font>').addClass('messageboxerror').fadeTo(900, 1);
	});
}

});
return false; //not to post the  form physically
});
});
</script>


<div id="goog-gt-tt" class="skiptranslate" dir="ltr"><div style="padding: 8px;"><div><div class="logo"><img src="https://www.gstatic.com/images/branding/product/1x/translate_24dp.png" alt="Google Translate" width="20" height="20"></div></div></div><div class="top" style="padding: 8px; float: left; width: 100%;"><h1 class="title gray">Original text</h1></div><div class="middle" style="padding: 8px;"><div class="original-text"></div></div><div class="bottom" style="padding: 8px;"><div class="activity-links"><span class="activity-link">Contribute a better translation</span><span class="activity-link"></span></div><div class="started-activity-container"><hr style="color: #CCC; background-color: #CCC; height: 1px; border: none;"><div class="activity-root"></div></div></div><div class="status-message" style="display: none;"></div></div>

<div class="goog-te-spinner-pos"><div class="goog-te-spinner-animation"><svg xmlns="http://www.w3.org/2000/svg" class="goog-te-spinner" width="96px" height="96px" viewBox="0 0 66 66"><circle class="goog-te-spinner-path" fill="none" stroke-width="6" stroke-linecap="round" cx="33" cy="33" r="30"></circle></svg></div></div></body></html>