<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script>

        <title>Ribosomal Database Project: data and tools for high throughput rRNA analysis | Nucleic Acids Research | Oxford Academic</title>

        <script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>



  

    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
  
<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico">
  <link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json">
  <meta name="msapplication-config" content="/UI/app/img/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site-1131979791.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'nar';
            App.SiteURL = 'academic.oup.com/nar';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/nar/article/42/D1/D633/1063201","@type":"ScholarlyArticle","name":"Ribosomal Database Project: data and tools for high throughput rRNA analysis","about":["genes, rrna","ribosomes","ribosomal rna","taxonomic classification","dystonia 12","datasets","Mutagenesis","Polymorphism/mutation detection","Repair","Computational Methods","Chromatin and Epigenetics","Genomics","Synthetic Biology and Assembly Cloning","Nucleic acid structure","Protein-nucleic acid interaction","Recombination","Replication","RNA characterisation and manipulation","Microarray Technology","Targeted inhibition of gene function","Massively Parallel (Deep) Sequencing","Targeted gene modification","DNA-Mediated Cell Transformation and Nucleic Acids Transfer","Transcriptome Mapping - Monitoring Gene Expression","New Restriction Enzymes","Nucleic acid amplification","Nucleic acid modification","Protein-protein interaction","Recombinant DNA expression","Cell biology","Ribosomes and Protein Translation","Physical and Biochemical Characterisation of DNA","Nucleic Acid Enzymology","Miscellaneous/other"],"datePublished":"2013-11-27","isPartOf":{"@id":"https://academic.oup.com/nar/issue/42/D1","@type":"PublicationIssue","issueNumber":"D1","datePublished":"2014-01-01","isPartOf":{"@id":"https://academic.oup.com/nar","@type":"Periodical","name":"Nucleic Acids Research","issn":["1362-4962"]}},"url":"http://dx.doi.org/10.1093/nar/gkt1244","keywords":["genes, rrna","ribosomes","ribosomal rna","taxonomic classification","dystonia 12","datasets"],"inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/nar/article/42/D1/D633/1063201","author":[{"name":"Cole, James R.","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Wang, Qiong","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Fish, Jordan A.","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Chai, Benli","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"McGarrell, Donna M.","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Sun, Yanni","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Brown, C. Titus","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Porras-Alfaro, Andrea","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Kuske, Cheryl R.","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"},{"name":"Tiedje, James M.","affiliation":"Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA","@type":"Person"}],"description":"Abstract.  Ribosomal Database Project (RDP; http://rdp.cme.msu.edu/) provides the research community with aligned and annotated rRNA gene sequence data, along w","pageStart":"D633","pageEnd":"D642","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/Issue/42/D1/3/m_cover.gif?Expires=1631352597&Signature=XjDhDUAPTfs-ENvTGXOBdn3E0GqPXKYUpDYESNOvUFhO~ixeUXdLGs3IZEU-AHBXGxeGj25W4HB5mBpJLIL25AZYs5oRF4-zWFBAm~dkBRP-wJau7nGbqCKthlzfdQxTNQX5RqbNeRFljCuBkscFYBM2EWL0nheqbySxVoPpoRFAh5hhq57Ffu3j9Olw9tzUKxatEeEyqYGYxgFGuBM9At4KQGwCsu7qqrCArsfnP2Lpw8YF5eDTIjsXGXxS2OsrFPoU3B1DmwKJtG9V-ono2Kx9cXhkt~5IPGl5ogdLrabos4BMfZXnV0UlJVgyJLdKTS6xBNG-VNZo6tu0Y9mgbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Ribosomal Database Project: data and tools for high throughput rRNA analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/Issue/42/D1/3/m_cover.gif?Expires=1631352597&Signature=XjDhDUAPTfs-ENvTGXOBdn3E0GqPXKYUpDYESNOvUFhO~ixeUXdLGs3IZEU-AHBXGxeGj25W4HB5mBpJLIL25AZYs5oRF4-zWFBAm~dkBRP-wJau7nGbqCKthlzfdQxTNQX5RqbNeRFljCuBkscFYBM2EWL0nheqbySxVoPpoRFAh5hhq57Ffu3j9Olw9tzUKxatEeEyqYGYxgFGuBM9At4KQGwCsu7qqrCArsfnP2Lpw8YF5eDTIjsXGXxS2OsrFPoU3B1DmwKJtG9V-ono2Kx9cXhkt~5IPGl5ogdLrabos4BMfZXnV0UlJVgyJLdKTS6xBNG-VNZo6tu0Y9mgbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="Ribosomal Database Project: data and tools for high throughput rRNA analysis">
    <meta property="og:description" content="Abstract.  Ribosomal Database Project (RDP; http://rdp.cme.msu.edu/) provides the research community with aligned and annotated rRNA gene sequence data, along w">
    <meta property="og:url" content="https://academic.oup.com/nar/article/42/D1/D633/1063201">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/Issue/42/D1/3/m_cover.gif?Expires=1631352597&amp;Signature=XjDhDUAPTfs-ENvTGXOBdn3E0GqPXKYUpDYESNOvUFhO~ixeUXdLGs3IZEU-AHBXGxeGj25W4HB5mBpJLIL25AZYs5oRF4-zWFBAm~dkBRP-wJau7nGbqCKthlzfdQxTNQX5RqbNeRFljCuBkscFYBM2EWL0nheqbySxVoPpoRFAh5hhq57Ffu3j9Olw9tzUKxatEeEyqYGYxgFGuBM9At4KQGwCsu7qqrCArsfnP2Lpw8YF5eDTIjsXGXxS2OsrFPoU3B1DmwKJtG9V-ono2Kx9cXhkt~5IPGl5ogdLrabos4BMfZXnV0UlJVgyJLdKTS6xBNG-VNZo6tu0Y9mgbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2013-11-27">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Cole, James R."><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Wang, Qiong"><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Fish, Jordan A."><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Chai, Benli"><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="McGarrell, Donna M."><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Sun, Yanni"><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Brown, C. Titus"><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Porras-Alfaro, Andrea"><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Kuske, Cheryl R."><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_author" content="Tiedje, James M."><meta name="citation_author_institution" content="Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, 2Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, 3Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, 4Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and 5Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA"><meta name="citation_title" content="Ribosomal Database Project: data and tools for high throughput rRNA analysis"><meta name="citation_firstpage" content="D633"><meta name="citation_lastpage" content="D642"><meta name="citation_doi" content="10.1093/nar/gkt1244"><meta name="citation_keyword" content="dystonia 12"><meta name="citation_keyword" content="ribosomal rna"><meta name="citation_journal_title" content="Nucleic Acids Research"><meta name="citation_journal_abbrev" content="Nucleic Acids Res"><meta name="citation_volume" content="42"><meta name="citation_issue" content="D1"><meta name="citation_publication_date" content="2014/01/01"><meta name="citation_issn" content="0305-1048"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Compilation of small ribosomal subunit RNA structures; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=3025-3049; "><meta name="citation_reference" content="citation_title=Phylogeny of some Fusarium species, as determined by large-subunit rRNA sequence comparison; Mol. Biol. Evol.;  citation_year=1989;  citation_volume=6;  citation_pages=227-242; "><meta name="citation_reference" content="citation_title=Comprehensive comparison of structural characteristics in Eukaryotic cytoplasmic large subunit (23S-like) ribosomal RNA; J. Mol. Biol.;  citation_year=1996;  citation_volume=256;  citation_pages=701-719; "><meta name="citation_reference" content="citation_title=Accurate, rapid taxonomic classification of fungal large subunit rRNA genes; Appl. Environ. Microbiol.;  citation_year=2012;  citation_volume=78;  citation_pages=1523-1533; "><meta name="citation_reference" content="citation_title=The International Nucleotide Sequence Database Collaboration; citation_author=International Nucleotide Sequence Database Collaboration; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D21-D24; "><meta name="citation_reference" content="citation_title=Facing growth in the European Nucleotide Archive; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D30-D35; "><meta name="citation_reference" content="citation_title=Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications; Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=415-420; "><meta name="citation_reference" content="citation_title=Database resources of the National Center for Biotechnology Information; citation_author=NCBI Resource Coordinators; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D8-D20; "><meta name="citation_reference" content="citation_title=UCHIME improves sensitivity and speed of chimera detection; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2194-2200; "><meta name="citation_reference" content="citation_title=The NCBI Taxonomy database; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D136-D143; "><meta name="citation_reference" content="citation_title=BioProject and BioSample databases at NCBI: facilitating capture and organization of metadata; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D57-D63; "><meta name="citation_reference" content="citation_title=Infernal 1.1: 100-fold faster RNA homology searches; Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=2933-3935; "><meta name="citation_reference" content="citation_title=The comparative RNA web (CRW) site: an online database of comparative sequence and structure information for ribosomal, intron, and other RNAs; BMC Bioinformatics;  citation_year=2002;  citation_volume=3;  citation_pages=2"><meta name="citation_reference" content="citation_title=Ironing out the wrinkles in the rare biosphere through improved OTU clustering; Environ. Microbiol.;  citation_year=2010;  citation_volume=12;  citation_pages=1889-1898; "><meta name="citation_reference" content="citation_title=Microbial diversity in the deep sea and the underexplored “rare biosphere”; Proc. Natl Acad. Sci. U.S.A.;  citation_year=2006;  citation_volume=103;  citation_pages=12115-12120; "><meta name="citation_reference" content="citation_title=LPSN—List of Prokaryotic Names with Standing in Nomenclature; Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D613-D616; "><meta name="citation_reference" content="citation_title=Release LTPs104 of the All-Species Living Tree; Syst. Appl. Microbiol.;  citation_year=2011;  citation_volume=34;  citation_pages=169-170; "><meta name="citation_reference" content="citation_title=Taxonomic parameters revisited: tarnished gold standards; Microbiol. Today;  citation_year=2006;  citation_volume=33;  citation_pages=152-155; "><meta name="citation_reference" content="citation_title=The ribosomal database project; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=3021-3023; "><meta name="citation_reference" content="citation_title=The Ribosomal Database Project (RDP-II): sequences and tools for high-throughput rRNA analysis; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D294-D296; "><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; "><meta name="citation_reference" content="citation_title=Naïve Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy; Appl. Environ. Microbiol.;  citation_year=2007;  citation_volume=73;  citation_pages=5261-5267; "><meta name="citation_reference" content="citation_title=Comparative analysis of pyrosequencing and a phylogenetic microarray for exploring microbial community structures in the human distal intestine; PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e6669"><meta name="citation_reference" content="citation_title=Bacterial community comparisons by taxonomy-supervised analysis independent of sequence alignment and clustering; Proc. Natl Acad. Sci. U.S.A.;  citation_year=2011;  citation_volume=108;  citation_pages=14637-14642; "><meta name="citation_reference" content="citation_title=Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys; ISME J.;  citation_year=2012;  citation_volume=6;  citation_pages=94-103; "><meta name="citation_reference" content="citation_title=The effect of training set on the classification of honey bee gut microbiota using the Naïve Bayesian Classifier; BMC Microbiol.;  citation_year=2012;  citation_volume=12;  citation_pages=221"><meta name="citation_reference" content="citation_title=A fast bit-vector algorithm for approximate string matching based on dynamic programming; J. ACM;  citation_year=1999;  citation_volume=46;  citation_pages=1-13; "><meta name="citation_reference" content="citation_title=Weighted neighbor joining: a likelihood-based approach to distance-based phylogeny reconstruction; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=189-197; "><meta name="citation_reference" content="citation_title=The Ribosomal Database Project: improved alignments and new tools for rRNA analysis; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D141-D145; "><meta name="citation_reference" content="citation_title=The Ribosomal Database Project: sequences and software for high-throughput rRNA analysis; citation_publisher=J. Wiley &amp; Sons, Inc., Hoboken, NJ; Handbook of Molecular Microbial Ecology I: Metagenomics and Complementary Approaches;  citation_year=2011;  citation_pages=313-324; "><meta name="citation_reference" content="citation_title=FunGene: the functional gene pipeline and repository; Front. Terr. Microbiol.;  citation_year=2013;  citation_volume=4;  citation_pages=291"><meta name="citation_reference" content="citation_title=The Biological Observation Matrix (BIOM) format or: how I learned to stop worrying and love the ome-ome; GigaScience;  citation_year=2012;  citation_volume=1;  citation_pages=7"><meta name="citation_reference" content="citation_title=Search and clustering orders of magnitude faster than BLAST; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2460-2461; "><meta name="citation_reference" content="citation_title=CD-HIT: accelerated for clustering the next generation sequencing data; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3150-3152; "><meta name="citation_reference" content="citation_title=Efficient algorithms for accurate hierarchical clustering of huge datasets: tackling the entire protein space; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=i41-i49; "><meta name="citation_reference" content="citation_title=ESPRIT: estimating species richness using large collections of 16S rRNA pyrosequences; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=e76"><meta name="citation_reference" content="citation_title=Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities; Appl. Environ. Microbiol.;  citation_year=2009;  citation_volume=75;  citation_pages=7537-7541; "><meta name="citation_reference" content="citation_title=PANDAseq: paired-end assembler for illumina sequences; BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=31"><meta name="citation_reference" content="citation_title=Development of a dual-index sequencing strategy and curation pipeline for analyzing amplicon sequence data on the MiSeq Illumina sequencing platform; Appl. Environ. Microbiol.;  citation_year=2013;  citation_volume=79;  citation_pages=5112-5120; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/nar/article-pdf/42/D1/D633/16952330/gkt1244.pdf"><meta name="description" content="Abstract.  Ribosomal Database Project (RDP; http://rdp.cme.msu.edu/) provides the research community with aligned and annotated rRNA gene sequence data, along w">    <script id="altmetric-embed-js" src="altmetric_badges-2288e2d46497462939299e96aa4d3daa.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019082901.js" async=""></script><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-4293a69f11403199e95492572e080de1.css"></head><body data-sitestyletemplate="Journal" theme-nar="" class="off-canvas pg_Article pg_article" data-sitename="nucleicacidsresearch"><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/nar/article-xml/42/D1/D633/1063201">



    <link rel="canonical" href="https://academic.oup.com/nar/article/42/D1/D633/1063201">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/nar_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/nar_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/nar_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/nar_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "nar/article/42/D1/D633/1063201");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&siteid=nar&authzrequired=false&doi=10.1093/nar/gkt1244';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5127" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="3000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="default" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f42%2fD1%2fD633%2f1063201" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Nucleic Acids Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  NAR Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f42%2fD1%2fD633%2f1063201" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384656">
            <a href="https://academic.oup.com/nar/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384653">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Section browse</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384657">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDChemical+Biology+and+Nucleic+Acid+Chemistry" class="nav-link ">Chemical Biology and Nucleic Acid Chemistry</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384658">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=COMPUTATIONAL+BIOLOGY" class="nav-link ">Computational Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384659">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Data+Resources+and+Analyses" class="nav-link ">Data Resources and Analyses</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384660">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Gene+Regulation,+Chromatin+and+Epigenetics" class="nav-link ">Gene Regulation, Chromatin and Epigenetics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384661">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genome+integrity,+repair+and+replication" class="nav-link ">Genome Integrity, Repair and Replication</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384662">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genomics" class="nav-link ">Genomics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384663">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=MOLECULAR+BIOLOGY" class="nav-link ">Molecular Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384664">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Nucleic+Acid+Enzymes" class="nav-link ">Nucleic Acid Enzymes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384665">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=RNAANDRNA+and+RNA-protein+complexes" class="nav-link ">RNA and RNA-protein complexes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384666">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=STRUCTURAL+BIOLOGY" class="nav-link ">Structural Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384667">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDSynthetic+Biology+and+Bioengineering" class="nav-link ">Synthetic Biology and Bioengineering</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384668">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Methods+Online" class="nav-link ">Methods Online</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384669">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=SURVEY%20@@@%20SUMMARY" class="nav-link ">Surveys and Summaries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384670">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Database+Issue" class="nav-link ">Database</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384671">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Web+Server+issue" class="nav-link ">Web Server</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384672">
            <a href="https://academic.oup.com/nar/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384654">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384673">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384674">
            <a href="http://mc.manuscriptcentral.com/nar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384675">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384676">
            <a href="https://academic.oup.com/nar/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384655">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384677">
            <a href="https://academic.oup.com/nar/pages/About" class="nav-link ">About Nucleic Acids Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384678">
            <a href="https://academic.oup.com/nar/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384679">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384680">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384681">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384682">
            <a href="https://academic.oup.com/nar/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384683">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384684">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/nar" class="journal-logo-container">
                    <img id="logo-NucleicAcidsResearch" class="journal-logo" src="nar_title248941275.svg" alt="Nucleic Acids Research">
                </a>

            <div class="society-logo-block">
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384656">
            <a href="https://academic.oup.com/nar/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384653">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Section browse</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384657">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDChemical+Biology+and+Nucleic+Acid+Chemistry" class="nav-link ">Chemical Biology and Nucleic Acid Chemistry</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384658">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=COMPUTATIONAL+BIOLOGY" class="nav-link ">Computational Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384659">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Data+Resources+and+Analyses" class="nav-link ">Data Resources and Analyses</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384660">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Gene+Regulation,+Chromatin+and+Epigenetics" class="nav-link ">Gene Regulation, Chromatin and Epigenetics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384661">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genome+integrity,+repair+and+replication" class="nav-link ">Genome Integrity, Repair and Replication</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384662">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genomics" class="nav-link ">Genomics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384663">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=MOLECULAR+BIOLOGY" class="nav-link ">Molecular Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384664">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Nucleic+Acid+Enzymes" class="nav-link ">Nucleic Acid Enzymes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384665">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=RNAANDRNA+and+RNA-protein+complexes" class="nav-link ">RNA and RNA-protein complexes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384666">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=STRUCTURAL+BIOLOGY" class="nav-link ">Structural Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384667">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDSynthetic+Biology+and+Bioengineering" class="nav-link ">Synthetic Biology and Bioengineering</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384668">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Methods+Online" class="nav-link ">Methods Online</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384669">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=SURVEY%20@@@%20SUMMARY" class="nav-link ">Surveys and Summaries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384670">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Database+Issue" class="nav-link ">Database</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384671">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Web+Server+issue" class="nav-link ">Web Server</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384672">
            <a href="https://academic.oup.com/nar/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384654">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384673">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384674">
            <a href="http://mc.manuscriptcentral.com/nar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384675">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384676">
            <a href="https://academic.oup.com/nar/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1384655">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384677">
            <a href="https://academic.oup.com/nar/pages/About" class="nav-link ">About Nucleic Acids Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384678">
            <a href="https://academic.oup.com/nar/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384679">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384680">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384681">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384682">
            <a href="https://academic.oup.com/nar/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384683">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1384684">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Nucleic Acids Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  NAR Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/nar/advanced-search?page=1&amp;fl_SiteID=5127&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="NAR Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/narjournals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5127" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="6232" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=45298" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/nar" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="nar" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/nar/issue/42/D1"><img id="issueImage" class="fb-featured-image" src="m_cover.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/nar/issue/42/D1">
                        <div class="volume trailing-comma">Volume 42</div>
                        <div class="issue">Issue D1</div>

            </a>

        </div>
        <div class="ii-pub-date">
1 January 2014        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="83627223">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627223">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627224">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627224">INTRODUCTION</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627229">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627229">RDP DATA COLLECTIONS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627243">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627243">TOOL DESCRIPTIONS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627293">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627293">USER SUPPORT</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627295">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627295">AVAILABILITY OF SUPPORTING DATA</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627297">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627297">FUNDING</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627300">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627300">ACKNOWLEDGEMENTS</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="83627302">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#83627302">REFERENCES</a>
            </div>
        </li>
<li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            <div class="comments-jumplink">
            <a class="js-jumplink scrollTo" href="#usercomments"><i class="icon-menu_comments"></i>Comments (0)</a>
        </div></div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/nar/article/42/D1/D625/1049480">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/nar/article/42/D1/D643/1061236">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Ribosomal Database Project: data and tools for high throughput rRNA analysis
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">James R. Cole</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="gkt1244-COR1" data-open="gkt1244-COR1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Tel: +1 517 353 3842; Fax: <div class="fax">+1 517 353 8957</div>; Email: <a href="mailto:colej@msu.edu" target="">colej@msu.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=James+R.+Cole">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cole%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Cole%20James%20R.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">James R. Cole</a><span class="delimiter"><i class="icon-general-mail"></i>, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qiong Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Qiong+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang%20Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wang%20Qiong%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Qiong Wang</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jordan A. Fish</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Jordan+A.+Fish">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fish%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Fish%20Jordan%20A.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Jordan A. Fish</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Benli Chai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Benli+Chai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chai%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Chai%20Benli%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Benli Chai</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Donna M. McGarrell</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Donna+M.+McGarrell">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=McGarrell%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22McGarrell%20Donna%20M.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Donna M. McGarrell</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanni Sun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Yanni+Sun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sun%20Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Sun%20Yanni%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Yanni Sun</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C. Titus Brown</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=C.+Titus+Brown">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Brown%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Brown%20C.%20Titus%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">C. Titus Brown</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrea Porras-Alfaro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Andrea+Porras-Alfaro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Porras-Alfaro%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Porras-Alfaro%20Andrea%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Andrea Porras-Alfaro</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Cheryl R. Kuske</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Cheryl+R.+Kuske">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuske%20C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kuske%20Cheryl%20R.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Cheryl R. Kuske</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">James M. Tiedje</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Center for Microbial Ecology, Michigan State University, East Lansing, MI 48824, USA, <sup>2</sup>Computer Science and Engineering, Michigan State University, East Lansing, MI 48824, USA, <sup>3</sup>Microbiology and Molecular Genetics, Michigan State University, East Lansing, MI 48824, USA, <sup>4</sup>Biological Sciences, Western Illinois University, Malcomb, IL 61455, USA and <sup>5</sup>Bioscience Division, Los Alamos National Laboratory, Los Alamos, NM 87545, USA</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=James+M.+Tiedje">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tiedje%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Tiedje%20James%20M.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">James M. Tiedje</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Nucleic Acids Research</em>, Volume 42, Issue D1, 1 January 2014, Pages D633–D642, <a href="https://doi.org/10.1093/nar/gkt1244">https://doi.org/10.1093/nar/gkt1244</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">27 November 2013</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">15 October 2013</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Revision received:</div>
                    <div class="wi-date">07 November 2013</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">08 November 2013</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">27 November 2013</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1063201" href="https://academic.oup.com/nar/article-pdf/42/D1/D633/16952330/gkt1244.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>James R. Cole, Qiong Wang, Jordan A. Fish, Benli Chai, Donna M. 
McGarrell, Yanni Sun, C. Titus Brown, Andrea Porras-Alfaro, Cheryl R. 
Kuske, James M. Tiedje,  Ribosomal Database Project: data and tools for 
high throughput rRNA analysis, <em>Nucleic Acids Research</em>, Volume 42, Issue D1, 1 January 2014, Pages D633–D642, <a href="https://doi.org/10.1093/nar/gkt1244">https://doi.org/10.1093/nar/gkt1244</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1063201&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1362-4962&amp;title=Ribosomal%20Database%20Project%3A%20data%20and%20tools%20for%20high%20throughput%20rRNA%20analysis&amp;publicationDate=2013-11-27&amp;volumeNum=42&amp;issueNum=D1&amp;author=Cole%2C%20James%20R.%3B%20Wang%2C%20Qiong&amp;startPage=D633&amp;endPage=D642&amp;contentId=10.1093%2Fnar%2Fgkt1244&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Nucleic Acids Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  NAR Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f42%2fD1%2fD633%2f1063201" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Nucleic Acids Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  NAR Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/nar/advanced-search?page=1&amp;fl_SiteID=5127&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="nar" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="83627223" id="83627223" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Ribosomal Database Project (RDP; <a class="link link-uri" href="http://rdp.cme.msu.edu/" target="">http://rdp.cme.msu.edu/</a>)
 provides the research community with aligned and annotated rRNA gene 
sequence data, along with tools to allow researchers to analyze their 
own rRNA gene sequences in the RDP framework. RDP data and tools are 
utilized in fields as diverse as human health, microbial ecology, 
environmental microbiology, nucleic acid chemistry, taxonomy and 
phylogenetics. In addition to aligned and annotated collections of 
bacterial and archaeal small subunit rRNA genes, RDP now includes a 
collection of fungal large subunit rRNA genes. RDP tools, including 
Classifier and Aligner, have been updated to work with this new fungal 
collection. The use of high-throughput sequencing to characterize 
environmental microbial populations has exploded in the past several 
years, and as sequence technologies have improved, the sizes of 
environmental datasets have increased. With release 11, RDP is providing
 an expanded set of tools to facilitate analysis of high-throughput 
data, including both single-stranded and paired-end reads. In addition, 
most tools are now available as open source packages for download and 
local use by researchers with high-volume needs or who would like to 
develop custom analysis pipelines.</p></section>                        <div class="article-metadata-panel clearfix"><div class="article-metadata-taxonomies">
                </div><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=IV.%20Viruses,%20bacteria,%20protozoa%20and%20fungi">IV. Viruses, bacteria, protozoa and fungi</a>
                </div></div>
                    <h2 scrollto-destination="83627224" id="83627224" class="section-title">INTRODUCTION</h2>
<p>Ribosomal Database Project (RDP) 11.1, released in October 2013 (<a class="link link-uri" href="http://rdp.cme.msu.edu/" target="">http://rdp.cme.msu.edu/</a>),
 contains 2 809 406 aligned and annotated bacterial and archaeal small 
subunit (SSU) rRNA gene sequences and 62 860 fungal large subunit (LSU) 
rRNA gene sequences. The majority of rRNA gene sequences in the RDP 
database are incomplete. Most of these are derived from sequencing PCR 
amplification products, whereas a small number of older entries derive 
from reverse transcriptase sequencing of isolated rRNA. Because PCR 
amplification makes use of primers to conserved regions internal to the 
genes, few of these sequences cover the 3′ and 5′ ends of the genes (<a reveal-id="gkt1244-F1" data-open="gkt1244-F1" class="link link-reveal link-table xref-fig">Figure 1</a>).
 Still, a diverse selection of complete gene sequences, mostly derived 
from genome sequencing, is available. Only a relatively small percentage
 of bacterial and archaeal sequences originate from organisms in 
culture; roughly 85% and 97%, respectively, of bacterial and archaeal 
sequences in RDP are from DNA directly isolated from environmental 
samples. </p>                        <a id="83627226" scrollto-destination="83627226"></a>
<div id="gkt1244-F1" data-id="gkt1244-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f1p.png" alt="Gene coverage: number of sequences from RDP release 11.1 covering the indicated positions on the reference sequence. (A) Bacterial SSU rRNA gene. Positions relative to Escherichia coli sequence GenBank accession J01695.1. Gray bars indicate variable regions (1). (B) Archaeal SSU rRNA gene. Positions relative to E. coli sequence GenBank accession J01695.1. (C) Fungal LSU rRNA gene. Positions relative to S. cerevisiae GenBank accession NC_001144.5 LSU gene. D1 and D2 indicate hypervariable regions initially used for discrimination among Fusarium spp. (2). The D2 region is among the most highly variable eukaryotic LSU regions in terms of both length and structure (3). Such high diversity may improve the performance of the RDP Classifier when discriminating between closely related genera. Gene coverage charts are available online and updated with each incremental RDP release." path-from-xml="gkt1244f1p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627226/gkt1244f1p.png" path-from-xml="gkt1244f1p.png" target="_blank">Open in new tab</a><a section="83627226" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f1p.png?Expires=1568374196&amp;Signature=RinFKKVpWENWpyITdo8vKQtgr065hI89VvInuV2NuH8m393hSMFmgzLcYlmJwgPBOWsOrXWyNGW5Rn7BChWQS2FbMTWyKXewHJRvG6WMOflHI0WA1qnLIxciBtZ-bdxTY2xZhA9gPHm4a3uJqpKHXKtjnaex3Zl6QlyKTNgsnxkZngwENvUif3rn1A741ckL08loOhingG6f5KDtQtixyxSICwFdB9qVUMKGJyFf-Gd89xi92IJnfWFu3ZF-4u-wyZlI6Wc%7EkWd-9%7EIu6ulBr5Y4sgOPIUU9gI5gqdppW3NP1B%7EyQThW0Ul1dzdvg11qYDFro6gpMphM37tbdel2Mg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627226&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f1p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Gene coverage: number of sequences from RDP release 11.1 covering the indicated positions on the reference sequence. <strong>(A)</strong> Bacterial SSU rRNA gene. Positions relative to <em>Escherichia coli</em> sequence GenBank accession J01695.1. Gray bars indicate variable regions (<a reveal-id="gkt1244-B1" data-open="gkt1244-B1" class="link link-ref link-reveal xref-bibr">1</a>). <strong>(B)</strong> Archaeal SSU rRNA gene. Positions relative to <em>E. coli</em> sequence GenBank accession J01695.1. <strong>(C)</strong> Fungal LSU rRNA gene. Positions relative to <em>S. cerevisiae</em> GenBank accession NC_001144.5 LSU gene. D1 and D2 indicate hypervariable regions initially used for discrimination among <em>Fusarium</em> spp. (<a reveal-id="gkt1244-B2" data-open="gkt1244-B2" class="link link-ref link-reveal xref-bibr">2</a>). The D2 region is among the most highly variable eukaryotic LSU regions in terms of both length and structure (<a reveal-id="gkt1244-B3" data-open="gkt1244-B3" class="link link-ref link-reveal xref-bibr">3</a>).
 Such high diversity may improve the performance of the RDP Classifier 
when discriminating between closely related genera. Gene coverage charts
 are available online and updated with each incremental RDP release.</p></div></div><div content-id="gkt1244-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f1p.png" alt="Gene coverage: number of sequences from RDP release 11.1 covering the indicated positions on the reference sequence. (A) Bacterial SSU rRNA gene. Positions relative to Escherichia coli sequence GenBank accession J01695.1. Gray bars indicate variable regions (1). (B) Archaeal SSU rRNA gene. Positions relative to E. coli sequence GenBank accession J01695.1. (C) Fungal LSU rRNA gene. Positions relative to S. cerevisiae GenBank accession NC_001144.5 LSU gene. D1 and D2 indicate hypervariable regions initially used for discrimination among Fusarium spp. (2). The D2 region is among the most highly variable eukaryotic LSU regions in terms of both length and structure (3). Such high diversity may improve the performance of the RDP Classifier when discriminating between closely related genera. Gene coverage charts are available online and updated with each incremental RDP release." path-from-xml="gkt1244f1p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627226/gkt1244f1p.png" path-from-xml="gkt1244f1p.png" target="_blank">Open in new tab</a><a section="83627226" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f1p.png?Expires=1568374196&amp;Signature=RinFKKVpWENWpyITdo8vKQtgr065hI89VvInuV2NuH8m393hSMFmgzLcYlmJwgPBOWsOrXWyNGW5Rn7BChWQS2FbMTWyKXewHJRvG6WMOflHI0WA1qnLIxciBtZ-bdxTY2xZhA9gPHm4a3uJqpKHXKtjnaex3Zl6QlyKTNgsnxkZngwENvUif3rn1A741ckL08loOhingG6f5KDtQtixyxSICwFdB9qVUMKGJyFf-Gd89xi92IJnfWFu3ZF-4u-wyZlI6Wc%7EkWd-9%7EIu6ulBr5Y4sgOPIUU9gI5gqdppW3NP1B%7EyQThW0Ul1dzdvg11qYDFro6gpMphM37tbdel2Mg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627226&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f1p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Gene coverage: number of sequences from RDP release 11.1 covering the indicated positions on the reference sequence. <strong>(A)</strong> Bacterial SSU rRNA gene. Positions relative to <em>Escherichia coli</em> sequence GenBank accession J01695.1. Gray bars indicate variable regions (<a reveal-id="gkt1244-B1" data-open="gkt1244-B1" class="link link-ref link-reveal xref-bibr">1</a>). <strong>(B)</strong> Archaeal SSU rRNA gene. Positions relative to <em>E. coli</em> sequence GenBank accession J01695.1. <strong>(C)</strong> Fungal LSU rRNA gene. Positions relative to <em>S. cerevisiae</em> GenBank accession NC_001144.5 LSU gene. D1 and D2 indicate hypervariable regions initially used for discrimination among <em>Fusarium</em> spp. (<a reveal-id="gkt1244-B2" data-open="gkt1244-B2" class="link link-ref link-reveal xref-bibr">2</a>). The D2 region is among the most highly variable eukaryotic LSU regions in terms of both length and structure (<a reveal-id="gkt1244-B3" data-open="gkt1244-B3" class="link link-ref link-reveal xref-bibr">3</a>).
 Such high diversity may improve the performance of the RDP Classifier 
when discriminating between closely related genera. Gene coverage charts
 are available online and updated with each incremental RDP release.</p></div></div><p>Over
 the past several years we have been approached by a number of 
researchers interested in using RDP tools for analysis of fungi in the 
environment. With the latest release, we are providing both an alignment
 of fungal 28S rRNA gene sequences and a fungal training set for the RDP
 Classifier leveraging a recently published phylogenetically consistent 
taxonomic mapping (<a reveal-id="gkt1244-B4" data-open="gkt1244-B4" class="link link-ref link-reveal xref-bibr">4</a>).
 For our new fungal alignment, the number of sequences covering 
positions in the 5′ end of the gene is much higher than for the 3′ end (<a reveal-id="gkt1244-F1" data-open="gkt1244-F1" class="link link-reveal link-table xref-fig">Figure 1</a>C).
 The 28S gene is much longer than the bacterial 16S gene, and many 
fungal researchers appear to find that sequencing 5′ regions of the 28S 
gene provides sufficient phylogenetic resolution for strain 
differentiation.</p><p>RDP offers tools for browsing and searching the 
data collections, for taxonomic classification and nearest-neighbor 
search, for primer-probe testing and for tree building. In addition, 
RDPipeline tools are specifically designed for processing high-volume 
amplicon sequence data. New tools have been designed with speed and 
capacity in mind, and most previously published tools have been updated 
to accommodate the recent changes of the sequencing technology. Many RDP
 tools are also available as open-source stand-alone packages.</p>                    <h2 scrollto-destination="83627229" id="83627229" class="section-title">RDP DATA COLLECTIONS</h2>
<p>RDP obtains most of its rRNA sequences from the International Nucleotide Sequence Database Collaboration (INSDC; <a reveal-id="gkt1244-B5" data-open="gkt1244-B5" class="link link-ref link-reveal xref-bibr">5</a>)
 databases. To prepare an RDP release, data files from the ‘standard’ 
dataclass and taxonomic divisions ‘Prokaryotes’, ‘Fungi’ and 
‘Environmental Samples’ are downloaded from the European Nucleotide 
Archive (ENA; <a reveal-id="gkt1244-B6" data-open="gkt1244-B6" class="link link-ref link-reveal xref-bibr">6</a>)
 ftp site. Records are examined for an ‘rRNA’ feature key of minimum 
length 500 bases (to allow sufficient context for taxonomic 
classification). If such record is found within an accession not in the 
RDP database, or within an existing accession but with a newer 
modification date, the sequence defined by the ‘rRNA’ feature is 
extracted. These new sequences are then filtered using a version of the 
RDP SeqMatch tool (described below) trained on a small hand-curated set 
of bacterial, archaeal, eukaryotic and mitochondrial SSU sequences and 
fungal LSU sequences. Only sequences having a best match to bacterial, 
archaeal or fungal with an S_ab score of at least 0.3 are saved. The 
original INSDC annotations, including structured comments such as 
Genomic Standards Consortium MIxS-compliant comments (<a reveal-id="gkt1244-B7" data-open="gkt1244-B7" class="link link-ref link-reveal xref-bibr">7</a>),
 are also captured. Many organism names in the INSDC records are not up 
to date. We obtain the most recent validly published synonym from 
Bacterial Nomenclature Up-to-Date (<a class="link link-uri" href="http://www.dsmz.de/" target="">http://www.dsmz.de/</a>).</p><p>Each
 sequence is aligned and classified using the corresponding RDP Aligner 
and Classifier (see below). Any sequence with a negative alignment bit 
saved score is discarded. The sequence is assigned to the lowest taxon 
in the RDP taxonomy with classification bootstrap confidence of 80% or 
above. Sequences passing this quality filtering are then subjected to 
the following tagging process. Sequences from type strains are tagged as
 ‘type’. Any sequence with accession listed in the bioproject.xml file 
(available from NCBI ftp site; <a reveal-id="gkt1244-B8" data-open="gkt1244-B8" class="link link-ref link-reveal xref-bibr">8</a>) is tagged as a genome project sequence. All 16S rRNA gene sequences are screened for chimeric sequences using UCHIME (<a reveal-id="gkt1244-B9" data-open="gkt1244-B9" class="link link-ref link-reveal xref-bibr">9</a>) in reference mode. Positive UCHIME results are tagged as ‘suspect’ sequences. Next, the NCBI taxonomic assignment (<a reveal-id="gkt1244-B10" data-open="gkt1244-B10" class="link link-ref link-reveal xref-bibr">10</a>)
 is determined using the taxonomy ID in the db_xref qualifier obtained 
from the INSDC annotation. Any sequence assigned under an NCBI taxon 
with name containing ‘environmental’, ‘uncultured’ or ‘uncultivated’ is 
tagged as an ‘uncultured’ sequence. For each release, a set of flat 
files containing the entire sequence collection for each of the three 
genes are available for download in aligned or unaligned FASTA, and 
annotated GenBank formats. With each release, RDP provides the resource 
files for the NCBI LinkOut service. This allows researchers to jump 
directly to RDP sequence records from the corresponding records in the 
NCBI’s Nucleotide and BioProject databases (<a reveal-id="gkt1244-B11" data-open="gkt1244-B11" class="link link-ref link-reveal xref-bibr">11</a>).</p>                    <h3 scrollto-destination="83627232" id="83627232" class="section-title">Alignments</h3>
<p>The sequences in the RDP database are aligned using Infernal, a stochastic context-free grammar-based aligner (<a reveal-id="gkt1244-B12" data-open="gkt1244-B12" class="link link-ref link-reveal xref-bibr">12</a>).
 This aligner has several advantages: it incorporates 
secondary-structure information into the alignment process; as a 
model-based aligner, new sequences can be easily added to a pre-existing
 alignment; it is fast enough for very large numbers of sequences. The 
bacterial and archaeal aligners were trained using secondary structure 
information from the Comparative RNA Web Site (CRW; <a reveal-id="gkt1244-B13" data-open="gkt1244-B13" class="link link-ref link-reveal xref-bibr">13</a>)
 and training alignments we developed with 2591 bacterial and 144 
archaeal full-length sequences mostly from sequenced genomes, 
respectively. The bacterial training sequences have greater coverage (27
 phyla) than those used for RDP release 10 (16 phyla). Many rRNA genes 
in genome sequences are annotated with incorrect start or stop 
positions. We adjusted these to produce consistent endpoints for the 
training set. We optimized the Infernal aligner parameters, particularly
 the relative entropy, to provide improved handling for partial 
sequences. Models and training sequences are available for download from
 the RDP website.</p><p>The V6 region is especially hard to place into a
 multiple sequence alignment because much of the region is not conserved
 in size, sequence or secondary structure; however, the high diversity 
of the V6 region makes it a very common amplification target. Available 
tools often do not attempt to produce a multiple sequence alignment for 
amplicons of this region, but instead score pairwise alignments to a set
 of reference sequences for analysis (<a reveal-id="gkt1244-B14" data-open="gkt1244-B14" class="link link-ref link-reveal xref-bibr">14</a>).
 The tuned Infernal 1.1 aligner is able to correctly align the less 
hypervariable positions in the short region amplified by commonly used 
V6 primers (<a reveal-id="gkt1244-B15" data-open="gkt1244-B15" class="link link-ref link-reveal xref-bibr">15</a>), producing an alignment for this region matching that produced with full-length sequences (<a reveal-id="gkt1244-F2" data-open="gkt1244-F2" class="link link-reveal link-table xref-fig">Figure 2</a>). </p>                        <a id="83627235" scrollto-destination="83627235"></a>
<div id="gkt1244-F2" data-id="gkt1244-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f2p.png" alt="Multiple sequence alignment of partial bacterial 16S rRNA sequences corresponding to the region between common V6 variable region amplification primers (15). Uppercase columns correspond to modeled positions. Lowercase columns correspond to regions where hypervariability in size and structure preclude assignment of homologous residues. These columns are normally ‘masked out’ before phylogenetic analysis. (A) Using the new RDP 11 alignment model. This matches the alignment for this region obtained with full-length sequences. (B) Using the RDP 10 alignment model. The alignment of the full-length sequences is almost identical in this V6 region between the two models, except one G-U pair in RDP 11 appears as inserts in the RDP 10 alignment. Bases highlighted in green color are canonical base pairs matching the conserved secondary structure. From top to bottom, the GenBank accessions are AB006164, AB006178, AB021164, AB015577, AB003932 and AB004715." path-from-xml="gkt1244f2p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627235/gkt1244f2p.png" path-from-xml="gkt1244f2p.png" target="_blank">Open in new tab</a><a section="83627235" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f2p.png?Expires=1568374196&amp;Signature=LsKdcByIW0VMfg56mZugK2VC7IBXjPJbM2AEouFHeHkG2y8hLpVd0kxWJbxSp%7ENlVxsO1NBN9ll0ZhLn05E7iKLH41WQzS2au3EpMyiCp5HQh6Ek2ZHR8rljdYbcmRnyHblYHOFQEllfU0gzx8FC21Y8EwVtWgkPBwPHtGh2ToXxGg2rgAA3gFf2QngzubOPWx8fIOV8LyQw6DIFtuq6QaPmRkBhmW5BwxEMAoPNtrpDKMWKtyVBE0MnVLtsBMdGXW0XxMFV2T7pj5si7pPd10EsoJcr--GODaTTeUNxvOTWkcl59waVfsc8rUN1sHPZxKkPxwQzZM6iRk-cwOfzpA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627235&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f2p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple
 sequence alignment of partial bacterial 16S rRNA sequences 
corresponding to the region between common V6 variable region 
amplification primers (<a reveal-id="gkt1244-B15" data-open="gkt1244-B15" class="link link-ref link-reveal xref-bibr">15</a>).
 Uppercase columns correspond to modeled positions. Lowercase columns 
correspond to regions where hypervariability in size and structure 
preclude assignment of homologous residues. These columns are normally 
‘masked out’ before phylogenetic analysis. <strong>(A)</strong> Using the new RDP 11 alignment model. This matches the alignment for this region obtained with full-length sequences. <strong>(B)</strong>
 Using the RDP 10 alignment model. The alignment of the full-length 
sequences is almost identical in this V6 region between the two models, 
except one G-U pair in RDP 11 appears as inserts in the RDP 10 
alignment. Bases highlighted in green color are canonical base pairs 
matching the conserved secondary structure. From top to bottom, the 
GenBank accessions are AB006164, AB006178, AB021164, AB015577, AB003932 
and AB004715.</p></div></div><div content-id="gkt1244-F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f2p.png" alt="Multiple sequence alignment of partial bacterial 16S rRNA sequences corresponding to the region between common V6 variable region amplification primers (15). Uppercase columns correspond to modeled positions. Lowercase columns correspond to regions where hypervariability in size and structure preclude assignment of homologous residues. These columns are normally ‘masked out’ before phylogenetic analysis. (A) Using the new RDP 11 alignment model. This matches the alignment for this region obtained with full-length sequences. (B) Using the RDP 10 alignment model. The alignment of the full-length sequences is almost identical in this V6 region between the two models, except one G-U pair in RDP 11 appears as inserts in the RDP 10 alignment. Bases highlighted in green color are canonical base pairs matching the conserved secondary structure. From top to bottom, the GenBank accessions are AB006164, AB006178, AB021164, AB015577, AB003932 and AB004715." path-from-xml="gkt1244f2p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627235/gkt1244f2p.png" path-from-xml="gkt1244f2p.png" target="_blank">Open in new tab</a><a section="83627235" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f2p.png?Expires=1568374196&amp;Signature=LsKdcByIW0VMfg56mZugK2VC7IBXjPJbM2AEouFHeHkG2y8hLpVd0kxWJbxSp%7ENlVxsO1NBN9ll0ZhLn05E7iKLH41WQzS2au3EpMyiCp5HQh6Ek2ZHR8rljdYbcmRnyHblYHOFQEllfU0gzx8FC21Y8EwVtWgkPBwPHtGh2ToXxGg2rgAA3gFf2QngzubOPWx8fIOV8LyQw6DIFtuq6QaPmRkBhmW5BwxEMAoPNtrpDKMWKtyVBE0MnVLtsBMdGXW0XxMFV2T7pj5si7pPd10EsoJcr--GODaTTeUNxvOTWkcl59waVfsc8rUN1sHPZxKkPxwQzZM6iRk-cwOfzpA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627235&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f2p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple
 sequence alignment of partial bacterial 16S rRNA sequences 
corresponding to the region between common V6 variable region 
amplification primers (<a reveal-id="gkt1244-B15" data-open="gkt1244-B15" class="link link-ref link-reveal xref-bibr">15</a>).
 Uppercase columns correspond to modeled positions. Lowercase columns 
correspond to regions where hypervariability in size and structure 
preclude assignment of homologous residues. These columns are normally 
‘masked out’ before phylogenetic analysis. <strong>(A)</strong> Using the new RDP 11 alignment model. This matches the alignment for this region obtained with full-length sequences. <strong>(B)</strong>
 Using the RDP 10 alignment model. The alignment of the full-length 
sequences is almost identical in this V6 region between the two models, 
except one G-U pair in RDP 11 appears as inserts in the RDP 10 
alignment. Bases highlighted in green color are canonical base pairs 
matching the conserved secondary structure. From top to bottom, the 
GenBank accessions are AB006164, AB006178, AB021164, AB015577, AB003932 
and AB004715.</p></div></div><p>We also tested this new alignment model 
by comparing the alignment produced using this model with the CRW 
bacterial seed alignment, which is hand-curated to match secondary 
structure. The Infernal alignment placed 92.7% of the bases in alignable
 positions (columns), the remainder corresponded to positions not 
conserved in the bacterial rRNA structure. We found that in 99.3% of the
 cases, pairs of residues in an alignable column of the Infernal 
alignment were mapped together to a column in the CRW alignment.</p><p>The
 fungal alignment uses a model built with 183 LSU sequences from 
complete fungal genomes plus the CRW fungal set, and covers four major 
fungal phyla: Ascomycota, Basidiomycota, Chytridiomycota and 
Blastocladiomycota. To develop the training model, we used a combination
 of the CRW general eukaryotic conservation model and <em>Saccharomyces cerevisiae</em>
 secondary structure model. In the large ribosomal subunit, the 5.8S and
 28S molecules form a common secondary structure and the training model 
included the combined 5.8S and 28S gene sequences. (The Internal 
Transcribed Spacer ITS2 between 5.8S and 28S evolves too rapidly for 
global alignment and is treated as an insert in our model.) This fungal 
LSU Aligner is especially useful for aligning sequences resulting from 
protocols that amplify and sequence from all or part of the 5.8S gene to
 the 5′ portion of the 28S gene while not compromising alignment of 
sequences of only the 28S gene.</p>                    <h3 scrollto-destination="83627238" id="83627238" class="section-title">Taxonomy</h3>
<p>RDP bases its bacterial and archaeal taxonomies on the taxonomic roadmaps published by Bergey’s Trust (<a class="link link-uri" href="http://www.bergeys.org/outlines.html" target="">http://www.bergeys.org/outlines.html</a>).
 As these are updated only at long intervals, we capture changes in 
taxonomic information and the publication of new species from the 
literature and from the List of Prokaryotic Names with Standing in 
Nomenclature website (<a reveal-id="gkt1244-B16" data-open="gkt1244-B16" class="link link-ref link-reveal xref-bibr">16</a>).
 We modify this taxonomy by adding clades for groups with few cultured 
relatives based on published informal taxonomies, where available. We 
compare this with the phylogenetic assessment from the All-Species 
Living Tree Project (<a reveal-id="gkt1244-B17" data-open="gkt1244-B17" class="link link-ref link-reveal xref-bibr">17</a>)
 and to our own assessment using the RDP Classifier. When there is a 
discrepancy between these sources, we conduct our own phylogenetic 
assessment by creating trees from the aligned sequences, including 
sequences from literature clades, and accept those clades best 
supported.</p><p>The fungal taxonomy used by RDP is the recently 
published taxonomy hand-developed using published phylogenies for 
different taxa and taxonomic databases (<a reveal-id="gkt1244-B4" data-open="gkt1244-B4" class="link link-ref link-reveal xref-bibr">4</a>) with updates. Because rRNA genes are too slowly evolving to reliably separate the validly named species (<a reveal-id="gkt1244-B18" data-open="gkt1244-B18" class="link link-ref link-reveal xref-bibr">18</a>),
 genus is the lowest rank presented in the RDP taxonomy. Where 
available, both genus and specific epithet, along with strain 
identifiers, are maintained for each sequence, but are not used to group
 or sort sequences. For species where our phylogenetically informed 
taxonomic assessment differs from the formal nomenclatural genus portion
 of the species binomial, the phylogenetically incorrect (but valid) 
name is maintained with the sequence and will differ from the assigned 
taxonomic lineage.</p><p>Using the pairwise distances generated with the
 enhanced distance calculation tool included in the mcClust package 
(described below) and the sequences and taxonomic data available from 
the RDP database, we computed accumulation curves for the group size and
 intra-taxa distances at the genus, family, order, class and phylum 
level for each domain (<a reveal-id="gkt1244-F3" data-open="gkt1244-F3" class="link link-reveal link-table xref-fig">Figure 3</a>). </p>                        <a id="83627242" scrollto-destination="83627242"></a>
<div id="gkt1244-F3" data-id="gkt1244-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f3p.png" alt="Accumulation curves showing (A) taxon size and (B) intra-taxon distance. All aligned sequences in RDP release 11.1 in each of the three RDP collections were clustered as described. The average distance between pairs of sequences in a taxon is shown in (B). The shape of the phylum curves, and to a lesser extent class curves, for archaea and fungi, are likely influenced by the small number of taxa and the skewed representation of sequences in these taxa." path-from-xml="gkt1244f3p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627242/gkt1244f3p.png" path-from-xml="gkt1244f3p.png" target="_blank">Open in new tab</a><a section="83627242" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f3p.png?Expires=1568374196&amp;Signature=vDqHWCDkYx-J0xYWGm60H53yqKA2WXHnUpcwgxu6H4lTjSGQgUEn%7ER4HjBTbCRi64ahd8%7EdSUBce5D6aqxGI3nf%7EWE6PQfbautFpLFV3MkKlzNOzODQ0Hp06tshIensfd55edPPK2p9STxQ0oXHrEErr28zNQxdpnmZwW5UPEz-Os2N1YioJ-RWl0JC2R28PXwzZjdQv8Qs7X8qGhBWZsjZd8mghFYFDKsL3g%7E8%7Ewqt-Dv1lctzwkctTdTN%7EWMFRDQilGBEjVL9Ok26sz2UYZa53cwfZjyAc9XYADf3NMJYgewPauZp-5fMpwtT3GWFDY7T5wHUAGTgynUBv4kJw5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627242&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f3p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Accumulation curves showing <strong>(A)</strong> taxon size and <strong>(B)</strong>
 intra-taxon distance. All aligned sequences in RDP release 11.1 in each
 of the three RDP collections were clustered as described. The average 
distance between pairs of sequences in a taxon is shown in (B). The 
shape of the phylum curves, and to a lesser extent class curves, for 
archaea and fungi, are likely influenced by the small number of taxa and
 the skewed representation of sequences in these taxa.</p></div></div><div content-id="gkt1244-F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f3p.png" alt="Accumulation curves showing (A) taxon size and (B) intra-taxon distance. All aligned sequences in RDP release 11.1 in each of the three RDP collections were clustered as described. The average distance between pairs of sequences in a taxon is shown in (B). The shape of the phylum curves, and to a lesser extent class curves, for archaea and fungi, are likely influenced by the small number of taxa and the skewed representation of sequences in these taxa." path-from-xml="gkt1244f3p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627242/gkt1244f3p.png" path-from-xml="gkt1244f3p.png" target="_blank">Open in new tab</a><a section="83627242" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f3p.png?Expires=1568374196&amp;Signature=vDqHWCDkYx-J0xYWGm60H53yqKA2WXHnUpcwgxu6H4lTjSGQgUEn%7ER4HjBTbCRi64ahd8%7EdSUBce5D6aqxGI3nf%7EWE6PQfbautFpLFV3MkKlzNOzODQ0Hp06tshIensfd55edPPK2p9STxQ0oXHrEErr28zNQxdpnmZwW5UPEz-Os2N1YioJ-RWl0JC2R28PXwzZjdQv8Qs7X8qGhBWZsjZd8mghFYFDKsL3g%7E8%7Ewqt-Dv1lctzwkctTdTN%7EWMFRDQilGBEjVL9Ok26sz2UYZa53cwfZjyAc9XYADf3NMJYgewPauZp-5fMpwtT3GWFDY7T5wHUAGTgynUBv4kJw5A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627242&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f3p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Accumulation curves showing <strong>(A)</strong> taxon size and <strong>(B)</strong>
 intra-taxon distance. All aligned sequences in RDP release 11.1 in each
 of the three RDP collections were clustered as described. The average 
distance between pairs of sequences in a taxon is shown in (B). The 
shape of the phylum curves, and to a lesser extent class curves, for 
archaea and fungi, are likely influenced by the small number of taxa and
 the skewed representation of sequences in these taxa.</p></div></div>                    <h2 scrollto-destination="83627243" id="83627243" class="section-title">TOOL DESCRIPTIONS</h2>
<p>The RDP website provides an interactive interface to the RDP database
 and tools. RDP tools that accept sequence input from researchers do so 
via file upload or direct entry into a text field on the tool page. 
Recognized sequence file formats include FASTA, FASTQ, GenBank and EMBL 
formats. Most tools display results in a taxonomic hierarchy view that 
allows interactive browsing. Results are saved in session until the 
start of a new task, allowing researchers to switch between tools 
without losing their results. From most tools, RDP sequences can be 
selected and saved to a SeqCart, which in turn can be used by other RDP 
tools as input, or downloaded as aligned or unaligned sequence files, or
 as a distance matrix from the download page. Batch loading into the 
SeqCart is possible by uploading a file containing a list of INSDC 
accession numbers or RDP identifiers. Most tools are available as 
open-source command-line versions from the RDP GitHub repository (<a class="link link-uri" href="http://github.com/rdpstaff/" target="">http://github.com/rdpstaff/</a>).</p>                    <h4 scrollto-destination="83627246" id="83627246" class="section-title">RDP Browsers</h4>
<p>RDP Browsers provide interactive web interfaces to RDP’s sequence 
collections. The Hierarchy Browser enables researchers to navigate, 
search and select sequences from the RDP collections displayed in either
 the RDP or NCBI taxonomy. Data set options allow researchers to examine
 subsets of sequence records based on any combination of the following 
options: type or non-type strain sequences, uncultured or isolated 
organisms, partial or near-full-length sequences and suspect quality or 
good quality sequences. A search feature allows researchers to enter a 
word or words to be matched against sequence annotation. Advanced search
 features include Boolean logic, regular expressions and ability to 
limit search to a specified annotation field. The ‘display depth’ can be
 modified to control the number of ranks shown in the hierarchy. 
Researchers can add or remove sequences from their SeqCart by selecting 
individual sequences or entire taxa.</p><p>Two additional specialized 
browsers are available. The Genome Browser organizes rRNA sequences from
 genome projects and provides rRNA copy number and genome size plus 
link-outs to additional genome information hosted at other sites. 
Publication View organizes sequences by publications. Sequences for any 
individual publication can be displayed and selected in Hierarchy 
Browser.</p><p><em>my</em>RDP is an account-based workspace that allows 
researchers to upload and store their pre-publication sequences. The 
facility is meant for single sequences up to groups of several hundred. 
These can be partial or complete rRNA sequences assembled from genomes 
or metagenomes, or sequences assembled from low-volume sequencing of 
rRNA gene clone libraries. The RDP Amplicon Sequence Pipeline 
(RDPipeline) described below is better suited for new amplicon 
sequencing technologies such as Illumina MiSeq. Sequences are uploaded 
in sequence groups to <em>my</em>RDP, and these groupings are 
maintained. Upon uploading, sequences are automatically submitted for 
alignment and classification. These <em>my</em>RDP sequences can then be
 analyzed in combination with sequences from RDP’s collections using 
RDP’s tool suite. A special social network feature allows sequence 
groups to be shared with additional researchers (‘research buddies’) 
specified by sequence owners. This feature is especially useful for 
remote collaborations.</p>                    <h4 scrollto-destination="83627250" id="83627250" class="section-title">Sequence Match (SeqMatch)</h4>
<p>This is one of the most often used online RDP tools. It is a re-implementation of the original RDP SIMILARITY_RANK (<a reveal-id="gkt1244-B19" data-open="gkt1244-B19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="gkt1244-B20" data-open="gkt1244-B20" class="link link-ref link-reveal xref-bibr">20</a>).
 SeqMatch finds closest RDP sequences to a query based on the fraction 
of shared seven-base sequence fragments (words) between the query and 
reference sequences (S_ab score). SeqMatch works well on partial- and 
full-length sequences and is more accurate than BLAST (<a reveal-id="gkt1244-B21" data-open="gkt1244-B21" class="link link-ref link-reveal xref-bibr">21</a>) at identifying database sequences that are closely related to query rRNA sequences.</p><p>The online SeqMatch is a <em>k</em>-nearest neighbor classifier and displays each query sequence under the lowest common ancestor taxon consistent with the <em>k</em> top matches for that query. In Detail View, these top <em>k</em> matches are all presented in a taxonomic hierarchy display similar to the Hierarchy Browser.</p><p>The
 standalone SeqMatch is available from the RDP GitHub repository. It 
requires an input sequence file, a reference sequence file and optional 
S_ab cutoff and <em>k</em> value. The output file contains the following information for each <em>k</em> top matches to a query: query name, match sequence ID, orientation, S_ab score and the number of unique common 7-mers.</p>                    <h4 scrollto-destination="83627254" id="83627254" class="section-title">Classifier</h4>
<p>The RDP Classifier rapidly and accurately assigns sequences into taxa
 with bootstrap value, an estimate of confidence for each assignment (<a reveal-id="gkt1244-B22" data-open="gkt1244-B22" class="link link-ref link-reveal xref-bibr">22</a>).
 The RDP Classifier has several advantages over most other methods of 
classifying rRNA sequences, especially for large high-throughput 
sequencing datasets: high speed with minimum memory requirement, does 
not require alignment, works well for partial sequences and can be 
easily retrained with alternative taxonomy or for different genes. The 
online RDP Classifier is pre-trained for bacterial and archaeal 16S and 
for fungal 28S rRNA gene sequences (see ‘Taxonomy’ for more detail). The
 bacterial and archaeal 16S training set has been updated seven times 
since the first release to reflect changes in taxonomic opinions. The 
online tool takes input query sequences and a choice of training set. 
The results are shown in a taxonomic hierarchy view displaying all the 
taxon nodes with sequences assigned to them. Researcher can change the 
‘confidence threshold’ to choose a cutoff suitable for the dataset. For 
partial sequences, using a lower confidence cutoff has been shown to 
increase the classification coverage at genus rank with sufficient 
accuracy (<a reveal-id="gkt1244-B23" data-open="gkt1244-B23" class="link link-ref link-reveal xref-bibr">23</a>). A detailed view shows individual queries assigned to each taxon.</p><p>The
 current version of the Classifier incorporates a number of enhancements
 not covered in the initial publication. The bootstrap assignment 
strategy has been changed to avoid an over-prediction problem when 
multiple genera are tied for highest score during bootstrap trials. The 
Classifier now allows multiple sample inputs. Expanded output options 
include detailed classification assignment for each sequence and an 
output file with one column for each sample containing assignment counts
 for each taxon. The latter is in a format appropriate for beta 
diversity analysis and sample ordination, and produces results similar 
to those obtained from operational taxonomic unit (OTU) clustering-based
 methods (<a reveal-id="gkt1244-B24" data-open="gkt1244-B24" class="link link-ref link-reveal xref-bibr">24</a>).</p><p>The
 command-line Classifier (available from the RDP GitHub repository) 
provides extensive support for retraining, allowing researchers to 
rapidly test the consistency of their training sets and to flag possible
 errors in their custom taxonomies. There is no requirement for a 
uniform number of taxonomic ranks and uncommon ranks are correctly 
handled. The classification speed is proportional to the number of 
genera, not the number of training sequences. This allows custom 
training sets with very large numbers of sequences. However, a larger 
training set with less accurate assignments or taxonomic irregularities 
will not necessarily work well—the testing tool can help validate a new 
training set. These features have allowed researchers to retrain the RDP
 Classifier on a broader range of sequences, including ones from 
environmental clades (<a reveal-id="gkt1244-B25" data-open="gkt1244-B25" class="link link-ref link-reveal xref-bibr">25</a>), on honeybee gut specific 16S rRNA sequences (<a reveal-id="gkt1244-B26" data-open="gkt1244-B26" class="link link-ref link-reveal xref-bibr">26</a>) and on fungal LSU sequences (<a reveal-id="gkt1244-B4" data-open="gkt1244-B4" class="link link-ref link-reveal xref-bibr">4</a>).</p>                    <h4 scrollto-destination="83627258" id="83627258" class="section-title">Library Compare</h4>
<p>RDP Library Compare (<a reveal-id="gkt1244-B22" data-open="gkt1244-B22" class="link link-ref link-reveal xref-bibr">22</a>)
 is used to investigate statistical differences between a pair of sample
 libraries. Instead of estimating overall difference between samples, 
LibCompare provides <em>P</em> values for determining statistical 
significance of abundance differences for individual taxa. This tool 
first uses the RDP Classifier to assign sequences to taxa. Depending on 
the abundance of sequences assigned to each taxon, one of two 
statistical tests is used to compute a <em>P</em> value to determine if a taxon is differentially represented in the two libraries.</p><p>The
 standalone Library Compare is available as part of the RDP Classifier 
package. It produces an output containing assignment detail result for 
each query, and a tab-delimited file containing comparison results 
sorted by <em>P</em> value. Each line contains the <em>P</em> value, taxon rank, taxon name and the number of assignments from each sample.</p>                    <h4 scrollto-destination="83627261" id="83627261" class="section-title">Probe Match</h4>
<p>Probe Match (<a reveal-id="gkt1244-B20" data-open="gkt1244-B20" class="link link-ref link-reveal xref-bibr">20</a>)
 performs a search against a sequence dataset for matches to the entered
 oligonucleotide sequences (primer). This tool implements a fast 
bit-vector algorithm for approximate substring matching (<a reveal-id="gkt1244-B27" data-open="gkt1244-B27" class="link link-ref link-reveal xref-bibr">27</a>).
 The online Probe Match takes primer sequences in Standard IUPAC code 
(allowing degenerate bases). There is an option to check a pair of 
primers in tandem, effectively testing <em>in silico</em> PCR. 
Researchers choose which of the three RDP sequence collections to 
search: Bacteria, Archaea or Fungi. Researchers can also limit the 
search to only the sequences containing a specified region of the 
molecule. This does not limit the search to that region, but by removing
 partial sequences missing the expected target site, it gives a more 
accurate estimate of primer coverage.</p><p>The standalone Probe Match 
general-purpose search engine (available from the RDP GitHub repository)
 requires an input sequence file and one or more primer sequences. Input
 sequences can be from any genes and of any length, but primers must not
 be longer than 64 characters. Multiple primers can be used at the same 
time, but for each sequence only the result for the best matching primer
 is reported. The output file contains sequence IDs that match at least 
one of the primer(s) within the specified distance and the detailed 
information of the match.</p>                    <h4 scrollto-destination="83627264" id="83627264" class="section-title">RDP Aligner</h4>
<p>RDP offers two ways for researchers to align sequences. Any bacterial or archaeal 16S gene sequences uploaded to a researcher’s <em>my</em>RDP
 account are automatically aligned. Researchers can also align bacterial
 and archaeal 16S as well as fungal 28S sequences using the Aligner on 
the RDPipeline website. The online Aligner uses the same Infernal 
alignment models used to process the RDP database sequences (see 
‘Alignment’ section above). The Aligner has been updated to Infernal 
version 1.1. This version is 7.5 times faster than the previous version 
used with RDP release 10. Since the standalone Infernal does not check 
the orientation of sequences; the online RDP Aligner first checks the 
orientation of each sequence and reverse-complements if necessary before
 aligning. The RDPipeline contains a suite of tools for further 
processing aligned sequence sets (see below). The Infernal 1.1 models 
used in RDP release 11, as well as the Infernal 0.81 models used in RDP 
release 10 are available from the RDP GitHub repository.</p>                    <h4 scrollto-destination="83627266" id="83627266" class="section-title">Tree Builder</h4>
<p>Tree Builder constructs a phylogenetic tree from sequences selected from the RDP collections and <em>my</em>RDP in any combination. This tool uses the Weighbor (<a reveal-id="gkt1244-B28" data-open="gkt1244-B28" class="link link-ref link-reveal xref-bibr">28</a>)
 weighted neighbor joining method with Jukes–Cantor corrected distances 
calculated from the RDP alignment. Bootstrap confidence estimates for 
each tree node are calculated from 100 bootstrap resamplings of the 
alignment columns. Normally an outgroup sequence should be included to 
allow the tree to be correctly rooted. The resulting tree is displayed 
in a Java applet that allows interactive exploratory manipulations, such
 as selecting nodes, and swapping branches. The trees can be downloaded 
in standard Newick format as well as in PS/PDF file formats.</p>                    <h4 scrollto-destination="83627268" id="83627268" class="section-title">Assignment Generator</h4>
<p>The Assignment Generator provides support for a 16S rRNA gene analysis lesson plan (<a reveal-id="gkt1244-B29" data-open="gkt1244-B29" class="link link-ref link-reveal xref-bibr">29</a>).
 It introduces comparative 16S rRNA gene analysis through a realistic 
bioinformatics exercise (unique sequences, common research tasks) that 
is easy to manage, distribute and evaluate using tools on RDP’s website.
 An instructor can generate a set of unique assignments for the entire 
class by specifying the number of students in the class, the number of 
sequences for each student and the choice of dataset (among bacteria, 
bacteria and archaea, or medically important bacteria). The tool 
provides: (i) a unique set of sequences for each student that are 
derived from the RDP sequence collections in a way that preserves 
secondary structure, (ii) a set of directions describing assignments for
 the students and (iii) an evaluation key for the instructor with the 
expected results for each student. This tool has been used in classes of
 sizes up to more than 500 students.</p>                    <h3 scrollto-destination="83627270" id="83627270" class="section-title">RDPipeline for high-throughput amplicon analysis</h3>
<p>The RDPipeline performs several common processing steps in 
taxonomy-dependent (using the RDP Classifier), and taxonomy-independent 
(using hierarchical clustering) analysis of large datasets. The 
RDPipeline is a new tool suite designed to replace our previous 
Pyrosequencing Pipeline (<a reveal-id="gkt1244-B30" data-open="gkt1244-B30" class="link link-ref link-reveal xref-bibr">30</a>),
 offering extended processing and analysis tools reflecting recent 
shifts in amplicon sequencing technologies and techniques. Researchers 
can utilize the tools in the RDPipeline tool suite in one of two ways. 
For researchers processing a moderate amount of sequences, we offer 
online versions of the RDPipeline tools. For researchers involved in 
high-volume sequencing projects, or who would like to incorporate some 
of our tools into their local custom workflow, we offer all the tools 
that make up the RDPipeline on the RDP GitHub repository.</p><p>The online RDPipeline has integrated support for <em>my</em>RDP
 accounts. All submitted jobs are viewable from the ‘my jobs’ page. 
Analysis results are stored for up to 2 weeks but job history remains 
available. The job history lists the type of each job, current status, 
submission, start and completion times and supplied processing 
parameters. For long running jobs, an email containing a direct link to 
download the results is sent when processing has completed. All 
RDPipeline tools accept compressed files. Any compressed file will be 
expanded upon upload and the contained files are treated as the input to
 the tool. All RDPipeline tools have extended input validation checks on
 uploaded files. Before processing begins, a summary is displayed 
showing the files detected and any files not used because of unexpected 
file type.</p><p>‘Initial processing’ prepares raw sequences from a 
sequencing facility for analysis. It is a multi-step process that 
includes sorting the raw reads by sample tag, trimming off tag and 
primer regions and removing sequences of low quality. Input file can be a
 single file or a compressed file containing multiple sequence files. 
For paired-end data, it uses our Assembler (described below; <a reveal-id="gkt1244-F4" data-open="gkt1244-F4" class="link link-reveal link-table xref-fig">Figure 4</a>) to assemble overlapping paired reads as the first step. We recommend researchers analyzing paired-end data use a read <em>Q</em>
 score cutoff of around 25–27 to filter out low-quality assembled reads.
 Summary statistics are included in the download for each tag, including
 the number of sequences filtered by each filter and a histogram of 
sequence lengths after filtering. </p>                        <a id="83627274" scrollto-destination="83627274"></a>
<div id="gkt1244-F4" data-id="gkt1244-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f4p.png" alt="Comparing per base error rates for three paired-end read assembly tools. The error rates were calculated using assembled reads filtered by either read Q score (Assembler and original PANDAseq; 38) or delta Q score (mothur; 39). Recommended read Q score of 27 for Assembler and base Q score (deltaq) of 6 for mothur are marked. (A) Sample M_20130714 and (B) Sample M_20130819." path-from-xml="gkt1244f4p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627274/gkt1244f4p.png" path-from-xml="gkt1244f4p.png" target="_blank">Open in new tab</a><a section="83627274" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f4p.png?Expires=1568374196&amp;Signature=oWU5obVdu1NG02AcJhljKPx8ikgCvwcdlsMaMU77MdkTyG0JzsNx5Dm9wDFoUQ8%7EEF-OmCXzp0R-t6T0aOseG8UhxhauzTrhWEIXs-CaLsEz5oGAmiPA9l0Nd6udgV96dPsf43XVKUHh0d2cc7iz9s9Zm7ldekNevTUIh6844mLRE-Iv8JgFjGeviDJ2LkLZdjN9HZu6W4B0WNvX7a%7EKj9xOJi6TCpbxhOkUDVKsmi5YEFqCW0NlmnwkTEASN-ZDVVEPDL0n%7E2hx4xjT0d4JPTrDRcapdU0QKNlQrZTY0lpy0xRJ%7EiEnZ-FpXLoFjk2s%7EdrUOi8lAzWxAw9A6ZaDrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627274&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f4p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparing
 per base error rates for three paired-end read assembly tools. The 
error rates were calculated using assembled reads filtered by either 
read <em>Q</em> score (Assembler and original PANDAseq; 38) or delta <em>Q</em> score (mothur; 39). Recommended read <em>Q</em> score of 27 for Assembler and base <em>Q</em> score (deltaq) of 6 for mothur are marked. <strong>(A)</strong> Sample M_20130714 and <strong>(B)</strong> Sample M_20130819.</p></div></div><div content-id="gkt1244-F4" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="m_gkt1244f4p.png" alt="Comparing per base error rates for three paired-end read assembly tools. The error rates were calculated using assembled reads filtered by either read Q score (Assembler and original PANDAseq; 38) or delta Q score (mothur; 39). Recommended read Q score of 27 for Assembler and base Q score (deltaq) of 6 for mothur are marked. (A) Sample M_20130714 and (B) Sample M_20130819." path-from-xml="gkt1244f4p.png"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/83627274/gkt1244f4p.png" path-from-xml="gkt1244f4p.png" target="_blank">Open in new tab</a><a section="83627274" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244f4p.png?Expires=1568374196&amp;Signature=oWU5obVdu1NG02AcJhljKPx8ikgCvwcdlsMaMU77MdkTyG0JzsNx5Dm9wDFoUQ8%7EEF-OmCXzp0R-t6T0aOseG8UhxhauzTrhWEIXs-CaLsEz5oGAmiPA9l0Nd6udgV96dPsf43XVKUHh0d2cc7iz9s9Zm7ldekNevTUIh6844mLRE-Iv8JgFjGeviDJ2LkLZdjN9HZu6W4B0WNvX7a%7EKj9xOJi6TCpbxhOkUDVKsmi5YEFqCW0NlmnwkTEASN-ZDVVEPDL0n%7E2hx4xjT0d4JPTrDRcapdU0QKNlQrZTY0lpy0xRJ%7EiEnZ-FpXLoFjk2s%7EdrUOi8lAzWxAw9A6ZaDrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=83627274&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5127" class="download-slide" path-from-xml="gkt1244f4p.png" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparing
 per base error rates for three paired-end read assembly tools. The 
error rates were calculated using assembled reads filtered by either 
read <em>Q</em> score (Assembler and original PANDAseq; 38) or delta <em>Q</em> score (mothur; 39). Recommended read <em>Q</em> score of 27 for Assembler and base <em>Q</em> score (deltaq) of 6 for mothur are marked. <strong>(A)</strong> Sample M_20130714 and <strong>(B)</strong> Sample M_20130819.</p></div></div>                    <h4 scrollto-destination="83627275" id="83627275" class="section-title">Alignment</h4>
<p>This tool allows researchers to align up to 1 000 000 
Bacterial/Archaeal 16S or Fungal 28S sequences at a time with Infernal 
1.1 using the RDP alignment models. Uploaded sequences for all genes are
 checked for orientation and reverse complemented when necessary. Each 
alignment job result also includes alignment position and length 
statistics as well as a summary histogram of read alignment start and 
end positions relative to the alignment model.</p>                    <h4 scrollto-destination="83627277" id="83627277" class="section-title">Clustering</h4>
<p>The complete linkage clustering tool (<a reveal-id="gkt1244-B29" data-open="gkt1244-B29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="gkt1244-B30" data-open="gkt1244-B30" class="link link-ref link-reveal xref-bibr">30</a>)
 allows users to upload aligned sequences to be clustered as the first 
step in taxonomy-independent analysis. Sequence files can be clustered 
together with each file treated as a sample, or files can be clustered 
separately. The online clustering tool is limited to 150 000 unique 
sequences per job. For clustering very large datasets, we provide a 
modified version of mcClust (<a reveal-id="gkt1244-B31" data-open="gkt1244-B31" class="link link-ref link-reveal xref-bibr">31</a>)
 for download (see below). This new version distributes distance 
calculations among a compute cluster and incorporates algorithmic 
changes that lower the time complexity and speed up clustering.</p>                    <h4 scrollto-destination="83627279" id="83627279" class="section-title">Ecological measures</h4>
<p>The cluster file obtained from Clustering or mcClust can be used to 
compute five common ecological measures for their samples. Alpha 
diversity can be estimated using Shannon or Chao1 and beta diversity can
 be measured using the Jaccard or Sørensen indices. Researchers can also
 assess sequencing depth using the rarefaction tool.</p>                    <h4 scrollto-destination="83627281" id="83627281" class="section-title">Sequencer run quality checks</h4>
<p>The RDPipeline includes two tools, Defined Community Analysis and Chimera Check for assessing the quality of sequence runs (<a reveal-id="gkt1244-B31" data-open="gkt1244-B31" class="link link-ref link-reveal xref-bibr">31</a>). The latter is powered by UCHIME (<a reveal-id="gkt1244-B9" data-open="gkt1244-B9" class="link link-ref link-reveal xref-bibr">9</a>).
 For researchers who include a defined community sample in their 
sequencer run, the Defined Community Analysis Tool calculates the 
observed error rates based on the known gene sequences of the organisms 
in the defined community.</p>                    <h4 scrollto-destination="83627283" id="83627283" class="section-title">Additional tools</h4>
<p>Researchers can use the ‘Cluster File Format Conversion’ tool to 
convert RDP Cluster files to an OTU table format, suitable for input to R
 and estimateS, or to the BIOM format (<a reveal-id="gkt1244-B32" data-open="gkt1244-B32" class="link link-ref link-reveal xref-bibr">32</a>).
 The ‘Alignment Merger’ tool allows researchers to merge alignment files
 created independently. The ‘Sequence Selector’ tool allows researchers 
to upload a set of sequence files and a separate file containing a list 
of IDs. A file is returned either containing only the sequences 
specified, or excluding them, depending on option selected. The 
‘Representative Sequence Selector’ tool allows researchers to upload a 
cluster file and retrieve a ‘representative sequence’ from each cluster,
 defined as the sequence with least sum of squared distances to all 
other sequences in the cluster.</p>                    <h4 scrollto-destination="83627285" id="83627285" class="section-title">mcClust enhancements</h4>
<p>Hierarchical sequence clustering methods that worked well for 
thousands of amplicon sequences often fail with the increased output of 
the latest sequencing technologies. Exact clustering methods require all
 pairwise distances for the input sequences and thus scale on the order 
of O(<em>n</em><sup>2</sup>). Many clustering implementations, in addition to requiring O(<em>n</em><sup>2</sup>) computational time, also have a memory complexity of O(<em>n</em><sup>2</sup>)
 as they store all distances in memory. Nonetheless, clustering methods 
remain an important tool in rRNA sequence analyses, and several groups 
have attempted to solve the scaling issues facing sequence clustering. 
One approach is to adopt an approximate clustering method, as in USEARCH
 (<a reveal-id="gkt1244-B33" data-open="gkt1244-B33" class="link link-ref link-reveal xref-bibr">33</a>) and CD-HIT (<a reveal-id="gkt1244-B34" data-open="gkt1244-B34" class="link link-ref link-reveal xref-bibr">34</a>), which use heuristics to limit the number of pairwise comparisons calculated. Another approach proposed by Loewenstein <em>et al.</em> (<a reveal-id="gkt1244-B35" data-open="gkt1244-B35" class="link link-ref link-reveal xref-bibr">35</a>)
 focuses on limiting the memory complexity of average linkage clustering
 by storing the distances on disk. To utilize disk storage for the 
pairwise distances, they must be in sorted order. With a general purpose
 sorting algorithm this increases the time complexity to O(<em>n</em><sup>2</sup>log <em>n</em><sup>2</sup>).</p><p>Several
 previously published complete linkage algorithm implementations take 
advantage of on disk storage of distances to limit the memory 
requirements for clustering (<a reveal-id="gkt1244-B31" data-open="gkt1244-B31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="gkt1244-B36" data-open="gkt1244-B36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="gkt1244-B37" data-open="gkt1244-B37" class="link link-ref link-reveal xref-bibr">37</a>).
 These implementations still require all pairwise distances (or at least
 all pairwise distances up to a maximum distance cutoff), and more 
importantly require sorting of all these distances. We propose a 
distance calculation tool with the goal of being efficient in the 
distance calculations, allowing the distance matrix computation to be 
parallelized and using an alternative sorting method to reduce the time 
complexity back to O(<em>n</em><sup>2</sup>) (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244_Supplementary_Data.zip?Expires=1568374196&amp;Signature=E4hhu8wlP24sz1EQTP-QYRYKdnLF7sNqw7yvDkpygRo9u%7E7LDlEYrziyaAkd-xJGhJKzFnSaJtV5wg1hZwvN4d9bHizS8Dpg3vT4YFDIVMOr3ZIwUuphBbHAAlbhOuhN59WEsX5slB0w87KvhmOjv3aCAKBVvXe2v9702JmrGeUVbzFj%7ETvaPotVVdOyufd-j6FO3c1wKKx6KeBXgBEB%7EFp2OeCcAlhIwfX-BVxN917TM4Jo%7Elaj8ceSzp9NK2zKe7s5Vlm9fsWSPz05nSmxCOGJySVXFzSlWYWwgeie7JakfxEVRiHqV-0HB4slUXxs4r5pUZMZ24aFqkP1Brcn4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span>).</p><p>The distance calculation tool is implemented in Java 1.6 and has been integrated as a tool into the mcClust package (<a reveal-id="gkt1244-B31" data-open="gkt1244-B31" class="link link-ref link-reveal xref-bibr">31</a>) available on the RDP GitHub repository.</p>                    <h4 scrollto-destination="83627289" id="83627289" class="section-title">Assembler for paired-end reads</h4>
<p>Compared to single-stranded Illumina reads, assembled paired-end 
reads can provide longer sequences with lower error rates. However, 
newly developed paired-end assembly tools have limitations. We have 
extended the existing PANDAseq (<a reveal-id="gkt1244-B38" data-open="gkt1244-B38" class="link link-ref link-reveal xref-bibr">38</a>)
 paired-end reads assembly program. Our modified PANDAseq (Assembler) 
performs a modified statistical analysis using the sequencer supplied 
quality (<em>Q</em>) scores to find the most likely overlap, computes assembled <em>Q</em> scores for the read overlap region and handles more complex overlap layouts (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244_Supplementary_Data.zip?Expires=1568374196&amp;Signature=E4hhu8wlP24sz1EQTP-QYRYKdnLF7sNqw7yvDkpygRo9u%7E7LDlEYrziyaAkd-xJGhJKzFnSaJtV5wg1hZwvN4d9bHizS8Dpg3vT4YFDIVMOr3ZIwUuphBbHAAlbhOuhN59WEsX5slB0w87KvhmOjv3aCAKBVvXe2v9702JmrGeUVbzFj%7ETvaPotVVdOyufd-j6FO3c1wKKx6KeBXgBEB%7EFp2OeCcAlhIwfX-BVxN917TM4Jo%7Elaj8ceSzp9NK2zKe7s5Vlm9fsWSPz05nSmxCOGJySVXFzSlWYWwgeie7JakfxEVRiHqV-0HB4slUXxs4r5pUZMZ24aFqkP1Brcn4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span> for details).</p><p>We
 have tested Assembler using two defined community samples from two 
different MiSeq runs. Both runs passed the Illumina MiSeq quality 
standards but the basic per base error rates of these two samples are 
quite different (0.17% for sample M_20130714 and 0.7% for sample 
M_20130819 after assembly). Both are within the reported error rate 
range for paired-end MiSeq amplicon data (0.28–1.08%) (<a reveal-id="gkt1244-B39" data-open="gkt1244-B39" class="link link-ref link-reveal xref-bibr">39</a>). Using an overall read <em>Q</em>
 score quality filter to remove low quality sequences, we tested the 
Assembler against the paired-end assembler and quality filter built into
 mothur (<a reveal-id="gkt1244-B39" data-open="gkt1244-B39" class="link link-ref link-reveal xref-bibr">39</a>), another amplicon analysis program. Assembler slightly outperformed mothur on the high-quality dataset (<a reveal-id="gkt1244-F4" data-open="gkt1244-F4" class="link link-reveal link-table xref-fig">Figure 4</a>A), and significantly outperformed on the average-quality dataset (<a reveal-id="gkt1244-F4" data-open="gkt1244-F4" class="link link-reveal link-table xref-fig">Figure 4</a>B). In both datasets, Assembler outperformed the original PANDAseq when scored in a similar manner (although such <em>Q</em> score based filtering was not a goal of that implementation). Using a read <em>Q</em>
 score of 27 decreases the error rates to 0.05% and 0.16% for M_20130714
 and M_20130819, respectively, and was effective in selectively removing
 reads with a high number of errors (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244_Supplementary_Data.zip?Expires=1568374196&amp;Signature=E4hhu8wlP24sz1EQTP-QYRYKdnLF7sNqw7yvDkpygRo9u%7E7LDlEYrziyaAkd-xJGhJKzFnSaJtV5wg1hZwvN4d9bHizS8Dpg3vT4YFDIVMOr3ZIwUuphBbHAAlbhOuhN59WEsX5slB0w87KvhmOjv3aCAKBVvXe2v9702JmrGeUVbzFj%7ETvaPotVVdOyufd-j6FO3c1wKKx6KeBXgBEB%7EFp2OeCcAlhIwfX-BVxN917TM4Jo%7Elaj8ceSzp9NK2zKe7s5Vlm9fsWSPz05nSmxCOGJySVXFzSlWYWwgeie7JakfxEVRiHqV-0HB4slUXxs4r5pUZMZ24aFqkP1Brcn4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S3</a></span>). The Assembler is integrated into Initial Processing and is available on the RDP GitHub repository.</p><p>All
 three programs can be run with multiple threads but were limited to a 
single thread in our testing. On an AMD Opteron 8384 quad-core 2.7 GHz 
processor, it took Assembler 1.4 h to assemble over 16 million reads 
from one MiSeq run. The original PANDAseq took 20 min, while mothur took
 21.3 h to assemble the same set of data using its recommended analysis 
protocol, on the same system.</p>                    <h2 scrollto-destination="83627293" id="83627293" class="section-title">USER SUPPORT</h2>
<p>RDP’s mission includes user support. RDP online tools are each 
supplied with a help page as a quick reference for its functionality, 
algorithm and how-tos. An RDP Wiki provides an updated searchable 
repository for answers to commonly asked questions compiled from 
previous user communications with RDP staff. Workflow tutorials guide 
researchers through common task-oriented processes, provide sample data 
and introduce researchers to the best practices for NGS data analysis. 
For command-line tools, step-by-step instructions and sample data files 
are provided on the RDP GitHub repository. Support questions can be 
emailed to <a href="mailto:rdpstaff@msu.edu" target="">rdpstaff@msu.edu</a>. Telephone support is available (+1 517 432 4998).</p>                    <h2 scrollto-destination="83627295" id="83627295" class="section-title">AVAILABILITY OF SUPPORTING DATA</h2>
<p>The sequence data from this study have been submitted to the ENA Short Read Archive (<a class="link link-uri" href="http://www.ebi.ac.uk/ena/" target="">http://www.ebi.ac.uk/ena/</a>) under accession no. PRJEB4878.</p>                    <h2 scrollto-destination="83627297" id="83627297" class="section-title">FUNDING</h2>
<p>Office of Science (Biological and Environmental Research), US 
Department of Energy [DE-FG02-99ER62848]. Additional support came from 
the Office of Science (Biological and Environmental Research), US 
Department of Energy [DE-SC0004601] and Bioenergy Center 
[DE-FC02-07ER64494]; the US National Institute of Environmental Health 
Sciences Superfund Research Program [P42 ES004911]; the National Science
 Foundation [DBI-0328255]; the US Department of Agriculture National 
Institute of Food and Agriculture National Research Initiative 
[2008-35107-04542]; the National Institute of Health Research Project 
[U01 HL098961] and Human Microbiome Project Demonstration Project [UH3 
DK083993]. Funding for open access charge: US Department of Energy.</p><p><em>Conflict of interest statement.</em> None declared.</p>                    <h2 scrollto-destination="83627300" id="83627300" class="backacknowledgements-title">ACKNOWLEDGEMENTS</h2>
<p>We thank Gareth W. Griffith for providing the Fungal LSU training 
data for phylum Neocallimastigomycota. We thank several individuals for 
their past contributions: Robin Gutell (and his colleagues), Ryan 
Farris, Sidique Kulam, Adithya M. Bandela, Chuck Parker, Paul Saxman, 
Bonnie Maidak, Tim Lilburn, Niels Larsen, George Garrity, Tom Macke, 
Michael J. McCaughey, Ross Overbeek, Sakti Pramanik, Scott Dawson, Mitch
 L. Sogin and Gary Olsen. We dedicate this publication to Carl Woese, 
whose insight made this work possible.</p>                    <h2 scrollto-destination="83627302" id="83627302" class="backreferences-title">REFERENCES</h2>
<div class="ref-list"><div content-id="gkt1244-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B1" class="ref-content "><div class="label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neefs</div> <div class="given-names">JM</div></div>,&nbsp; <div class="name"><div class="surname">Van de Peer</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">De Rijk</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Chapelle</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">De Wachter</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Compilation of small ribosomal subunit RNA structures</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">3025</div>-<div class="lpage">3049</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Compilation%20of%20small%20ribosomal%20subunit%20RNA%20structures&amp;author=JM%20Neefs&amp;author=Y%20Van%20de%20Peer&amp;author=P%20De%20Rijk&amp;author=S%20Chapelle&amp;author=R%20De%20Wachter&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=3025-3049" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/21.13.3025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.13.3025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F21.13.3025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8332525" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Compilation%20of%20small%20ribosomal%20subunit%20RNA%20structures&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B2" class="ref-content "><div class="label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guadet</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Julien</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Lafay</div> <div class="given-names">JF</div></div>,&nbsp; <div class="name"><div class="surname">Brygoo</div> <div class="given-names">Y</div></div>.&nbsp;</div><div class="article-title">Phylogeny of some <em>Fusarium</em> species, as determined by large-subunit rRNA sequence comparison</div>,&nbsp;<div class="source">Mol. Biol. Evol.</div>&nbsp;,&nbsp;<div class="year">1989</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">227</div>-<div class="lpage">242</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20of%20some%20Fusarium%20species%2C%20as%20determined%20by%20large-subunit%20rRNA%20sequence%20comparison&amp;author=J%20Guadet&amp;author=J%20Julien&amp;author=JF%20Lafay&amp;author=Y%20Brygoo&amp;publication_year=1989&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=6&amp;pages=227-242" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2622333" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Phylogeny%20of%20some%20Fusarium%20species%2C%20as%20determined%20by%20large-subunit%20rRNA%20sequence%20comparison&amp;aulast=Guadet&amp;date=1989&amp;spage=227&amp;epage=242&amp;volume=6" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Phylogeny%20of%20some%20Fusarium%20species%2C%20as%20determined%20by%20large-subunit%20rRNA%20sequence%20comparison&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B3" class="ref-content "><div class="label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnare</div> <div class="given-names">MN</div></div>,&nbsp; <div class="name"><div class="surname">Damberger</div> <div class="given-names">SH</div></div>,&nbsp; <div class="name"><div class="surname">Gray</div> <div class="given-names">MW</div></div>,&nbsp; <div class="name"><div class="surname">Gutell</div> <div class="given-names">RR</div></div>.&nbsp;</div><div class="article-title">Comprehensive comparison of structural characteristics in Eukaryotic cytoplasmic large subunit (23S-like) ribosomal RNA</div>,&nbsp;<div class="source">J. Mol. Biol.</div>&nbsp;,&nbsp;<div class="year">1996</div>,&nbsp;vol.&nbsp;<div class="volume">256</div>&nbsp;(pg.&nbsp;<div class="fpage">701</div>-<div class="lpage">719</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20comparison%20of%20structural%20characteristics%20in%20Eukaryotic%20cytoplasmic%20large%20subunit%20%2823S-like%29%20ribosomal%20RNA&amp;author=MN%20Schnare&amp;author=SH%20Damberger&amp;author=MW%20Gray&amp;author=RR%20Gutell&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=256&amp;pages=701-719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1006/jmbi.1996.0119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1006%2Fjmbi.1996.0119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8642592" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comprehensive%20comparison%20of%20structural%20characteristics%20in%20Eukaryotic%20cytoplasmic%20large%20subunit%20%2823S-like%29%20ribosomal%20RNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B4" class="ref-content "><div class="label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">K-L</div></div>,&nbsp; <div class="name"><div class="surname">Porras-Alfaro</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Kuske</div> <div class="given-names">CR</div></div>,&nbsp; <div class="name"><div class="surname">Eichorst</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Xie</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">Accurate, rapid taxonomic classification of fungal large subunit rRNA genes</div>,&nbsp;<div class="source">Appl. Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">78</div>&nbsp;(pg.&nbsp;<div class="fpage">1523</div>-<div class="lpage">1533</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accurate%2C%20rapid%20taxonomic%20classification%20of%20fungal%20large%20subunit%20rRNA%20genes&amp;author=K-L%20Liu&amp;author=A%20Porras-Alfaro&amp;author=CR%20Kuske&amp;author=S%20Eichorst&amp;author=G%20Xie&amp;publication_year=2012&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=78&amp;pages=1523-1533" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.06826-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.06826-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.06826-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22194300" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accurate%2C%20rapid%20taxonomic%20classification%20of%20fungal%20large%20subunit%20rRNA%20genes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B5" class="ref-content "><div class="label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Cochrane</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Karsch-Mizrachi</div> <div class="given-names">I</div></div>.&nbsp;</div><div class="collab">International Nucleotide Sequence Database Collaboration</div><div class="article-title">The International Nucleotide Sequence Database Collaboration</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">41</div>&nbsp;(pg.&nbsp;<div class="fpage">D21</div>-<div class="lpage">D24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20International%20Nucleotide%20Sequence%20Database%20Collaboration&amp;author=Y%20Nakamura&amp;author=G%20Cochrane&amp;author=I%20Karsch-Mizrachi&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D21-D24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks1084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks1084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23180798" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20International%20Nucleotide%20Sequence%20Database%20Collaboration&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B6" class="ref-content "><div class="label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cochrane</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Alako</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Amid</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Bower</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Cerdeño-Tárraga</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Cleland</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Gibson</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Goodgame</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Jang</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Kay</div> <div class="given-names">S</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Facing growth in the European Nucleotide Archive</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">41</div>&nbsp;(pg.&nbsp;<div class="fpage">D30</div>-<div class="lpage">D35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Facing%20growth%20in%20the%20European%20Nucleotide%20Archive&amp;author=G%20Cochrane&amp;author=B%20Alako&amp;author=C%20Amid&amp;author=L%20Bower&amp;author=A%20Cerde%C3%B1o-T%C3%A1rraga&amp;author=I%20Cleland&amp;author=R%20Gibson&amp;author=N%20Goodgame&amp;author=M%20Jang&amp;author=S%20Kay&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D30-D35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks1175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks1175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23203883" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Facing%20growth%20in%20the%20European%20Nucleotide%20Archive&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B7" class="ref-content "><div class="label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yilmaz</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Kottmann</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Field</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Amaral-Zettler</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Gilbert</div> <div class="given-names">JA</div></div>,&nbsp; <div class="name"><div class="surname">Karsch-Mizrachi</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Johnston</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Cochrane</div> <div class="given-names">G</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Minimum information about a marker gene sequence (MIMARKS) and minimum information about any (x) sequence (MIxS) specifications</div>,&nbsp;<div class="source">Nat. Biotechnol.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">29</div>&nbsp;(pg.&nbsp;<div class="fpage">415</div>-<div class="lpage">420</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Minimum%20information%20about%20a%20marker%20gene%20sequence%20%28MIMARKS%29%20and%20minimum%20information%20about%20any%20%28x%29%20sequence%20%28MIxS%29%20specifications&amp;author=P%20Yilmaz&amp;author=R%20Kottmann&amp;author=D%20Field&amp;author=R%20Knight&amp;author=JR%20Cole&amp;author=L%20Amaral-Zettler&amp;author=JA%20Gilbert&amp;author=I%20Karsch-Mizrachi&amp;author=A%20Johnston&amp;author=G%20Cochrane&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=415-420" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nbt.1823" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1823" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnbt.1823"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21552244" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Minimum%20information%20about%20a%20marker%20gene%20sequence%20%28MIMARKS%29%20and%20minimum%20information%20about%20any%20%28x%29%20sequence%20%28MIxS%29%20specifications&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B8" class="ref-content "><div class="label">8</div><div class="citation element-citation"><div class="collab">NCBI Resource Coordinators</div><div class="article-title">Database resources of the National Center for Biotechnology Information</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">41</div>&nbsp;(pg.&nbsp;<div class="fpage">D8</div>-<div class="lpage">D20</div>)<div class="citation-links"><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gks1189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgks1189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193264" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Database%20resources%20of%20the%20National%20Center%20for%20Biotechnology%20Information&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B9" class="ref-content "><div class="label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>,&nbsp; <div class="name"><div class="surname">Haas</div> <div class="given-names">BJ</div></div>,&nbsp; <div class="name"><div class="surname">Clemente</div> <div class="given-names">JC</div></div>,&nbsp; <div class="name"><div class="surname">Quince</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">UCHIME improves sensitivity and speed of chimera detection</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;(pg.&nbsp;<div class="fpage">2194</div>-<div class="lpage">2200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;author=RC%20Edgar&amp;author=BJ%20Haas&amp;author=JC%20Clemente&amp;author=C%20Quince&amp;author=R%20Knight&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2194-2200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B10" class="ref-content "><div class="label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Federhen</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">The NCBI Taxonomy database</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">40</div>&nbsp;(pg.&nbsp;<div class="fpage">D136</div>-<div class="lpage">D143</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20NCBI%20Taxonomy%20database&amp;author=S%20Federhen&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D136-D143" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1178" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1178" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1178"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22139910" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20NCBI%20Taxonomy%20database&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B11" class="ref-content "><div class="label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrett</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Clark</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Gevorgyan</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Gorelenkov</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Gribov</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Karsch-Mizrachi</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Kimelman</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Pruitt</div> <div class="given-names">KD</div></div>,&nbsp; <div class="name"><div class="surname">Resenchuk</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">BioProject and BioSample databases at NCBI: facilitating capture and organization of metadata</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">40</div>&nbsp;(pg.&nbsp;<div class="fpage">D57</div>-<div class="lpage">D63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=BioProject%20and%20BioSample%20databases%20at%20NCBI%3A%20facilitating%20capture%20and%20organization%20of%20metadata&amp;author=T%20Barrett&amp;author=K%20Clark&amp;author=R%20Gevorgyan&amp;author=V%20Gorelenkov&amp;author=E%20Gribov&amp;author=I%20Karsch-Mizrachi&amp;author=M%20Kimelman&amp;author=KD%20Pruitt&amp;author=S%20Resenchuk&amp;author=T%20Tatusova&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D57-D63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkr1163" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1163" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkr1163"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22139929" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:BioProject%20and%20BioSample%20databases%20at%20NCBI%3A%20facilitating%20capture%20and%20organization%20of%20metadata&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B12" class="ref-content "><div class="label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nawrocki</div> <div class="given-names">EP</div></div>,&nbsp; <div class="name"><div class="surname">Eddy</div> <div class="given-names">SR</div></div>.&nbsp;</div><div class="article-title">Infernal 1.1: 100-fold faster RNA homology searches</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">29</div>&nbsp;(pg.&nbsp;<div class="fpage">2933</div>-<div class="lpage">3935</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Infernal%201.1%3A%20100-fold%20faster%20RNA%20homology%20searches&amp;author=EP%20Nawrocki&amp;author=SR%20Eddy&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=2933-3935" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24008419" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Infernal%201.1%3A%20100-fold%20faster%20RNA%20homology%20searches&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B13" class="ref-content "><div class="label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannone</div> <div class="given-names">JJ</div></div>,&nbsp; <div class="name"><div class="surname">Subramanian</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Schnare</div> <div class="given-names">MN</div></div>,&nbsp; <div class="name"><div class="surname">Collett</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">D'Souza</div> <div class="given-names">LM</div></div>,&nbsp; <div class="name"><div class="surname">Du</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Feng</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Lin</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Madabusi</div> <div class="given-names">LV</div></div>,&nbsp; <div class="name"><div class="surname">Müller</div> <div class="given-names">KM</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">The
 comparative RNA web (CRW) site: an online database of comparative 
sequence and structure information for ribosomal, intron, and other RNAs</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">3</div>&nbsp;pg.&nbsp;<div class="fpage">2</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20comparative%20RNA%20web%20%28CRW%29%20site%3A%20an%20online%20database%20of%20comparative%20sequence%20and%20structure%20information%20for%20ribosomal%2C%20intron%2C%20and%20other%20RNAs&amp;author=JJ%20Cannone&amp;author=S%20Subramanian&amp;author=MN%20Schnare&amp;author=JR%20Collett&amp;author=LM%20D%27Souza&amp;author=Y%20Du&amp;author=B%20Feng&amp;author=N%20Lin&amp;author=LV%20Madabusi&amp;author=KM%20M%C3%BCller&amp;publication_year=2002&amp;journal=BMC%20Bioinformatics&amp;volume=3&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-3-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-3-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-3-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11869452" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20comparative%20RNA%20web%20%28CRW%29%20site%3A%20an%20online%20database%20of%20comparative%20sequence%20and%20structure%20information%20for%20ribosomal%2C%20intron%2C%20and%20other%20RNAs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B14" class="ref-content "><div class="label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huse</div> <div class="given-names">SM</div></div>,&nbsp; <div class="name"><div class="surname">Welch</div> <div class="given-names">DM</div></div>,&nbsp; <div class="name"><div class="surname">Morrison</div> <div class="given-names">HG</div></div>,&nbsp; <div class="name"><div class="surname">Sogin</div> <div class="given-names">ML</div></div>.&nbsp;</div><div class="article-title">Ironing out the wrinkles in the rare biosphere through improved OTU clustering</div>,&nbsp;<div class="source">Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">1889</div>-<div class="lpage">1898</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Ironing%20out%20the%20wrinkles%20in%20the%20rare%20biosphere%20through%20improved%20OTU%20clustering&amp;author=SM%20Huse&amp;author=DM%20Welch&amp;author=HG%20Morrison&amp;author=ML%20Sogin&amp;publication_year=2010&amp;journal=Environ.%20Microbiol.&amp;volume=12&amp;pages=1889-1898" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1462-2920.2010.02193.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2010.02193.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1462-2920.2010.02193.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20236171" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Ironing%20out%20the%20wrinkles%20in%20the%20rare%20biosphere%20through%20improved%20OTU%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B15" class="ref-content "><div class="label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sogin</div> <div class="given-names">ML</div></div>,&nbsp; <div class="name"><div class="surname">Morrison</div> <div class="given-names">HG</div></div>,&nbsp; <div class="name"><div class="surname">Huber</div> <div class="given-names">JA</div></div>,&nbsp; <div class="name"><div class="surname">Mark Welch</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Huse</div> <div class="given-names">SM</div></div>,&nbsp; <div class="name"><div class="surname">Neal</div> <div class="given-names">PR</div></div>,&nbsp; <div class="name"><div class="surname">Arrieta</div> <div class="given-names">JM</div></div>,&nbsp; <div class="name"><div class="surname">Herndl</div> <div class="given-names">GJ</div></div>.&nbsp;</div><div class="article-title">Microbial diversity in the deep sea and the underexplored “rare biosphere”</div>,&nbsp;<div class="source">Proc. Natl Acad. Sci. U.S.A.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">103</div>&nbsp;(pg.&nbsp;<div class="fpage">12115</div>-<div class="lpage">12120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Microbial%20diversity%20in%20the%20deep%20sea%20and%20the%20underexplored%20%E2%80%9Crare%20biosphere%E2%80%9D&amp;author=ML%20Sogin&amp;author=HG%20Morrison&amp;author=JA%20Huber&amp;author=D%20Mark%20Welch&amp;author=SM%20Huse&amp;author=PR%20Neal&amp;author=JM%20Arrieta&amp;author=GJ%20Herndl&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.S.A.&amp;volume=103&amp;pages=12115-12120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.0605127103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0605127103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0605127103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16880384" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Microbial%20diversity%20in%20the%20deep%20sea%20and%20the%20underexplored%20%E2%80%9Crare%20biosphere%E2%80%9D&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B16" class="ref-content "><div class="label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parte</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">LPSN—List of Prokaryotic Names with Standing in Nomenclature</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2014</div>,&nbsp;vol.&nbsp;<div class="volume">42</div>&nbsp;(pg.&nbsp;<div class="fpage">D613</div>-<div class="lpage">D616</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=LPSN%E2%80%94List%20of%20Prokaryotic%20Names%20with%20Standing%20in%20Nomenclature&amp;author=A%20Parte&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D613-D616" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkt1111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkt1111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24243842" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:LPSN%E2%80%94List%20of%20Prokaryotic%20Names%20with%20Standing%20in%20Nomenclature&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B17" class="ref-content "><div class="label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Munoz</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Yarza</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Ludwig</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Euzéby</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Amann</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Schleifer</div> <div class="given-names">KH</div></div>,&nbsp; <div class="name"><div class="surname">Glöckner</div> <div class="given-names">FO</div></div>,&nbsp; <div class="name"><div class="surname">Rosselló-Móra</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Release LTPs104 of the All-Species Living Tree</div>,&nbsp;<div class="source">Syst. Appl. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">34</div>&nbsp;(pg.&nbsp;<div class="fpage">169</div>-<div class="lpage">170</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Release%20LTPs104%20of%20the%20All-Species%20Living%20Tree&amp;author=R%20Munoz&amp;author=P%20Yarza&amp;author=W%20Ludwig&amp;author=J%20Euz%C3%A9by&amp;author=R%20Amann&amp;author=KH%20Schleifer&amp;author=FO%20Gl%C3%B6ckner&amp;author=R%20Rossell%C3%B3-M%C3%B3ra&amp;publication_year=2011&amp;journal=Syst.%20Appl.%20Microbiol.&amp;volume=34&amp;pages=169-170" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.syapm.2011.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.syapm.2011.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.syapm.2011.03.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21497273" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Release%20LTPs104%20of%20the%20All-Species%20Living%20Tree&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B18" class="ref-content "><div class="label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stackebrandt</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Ebers</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Taxonomic parameters revisited: tarnished gold standards</div>,&nbsp;<div class="source">Microbiol. Today</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">152</div>-<div class="lpage">155</div>)<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Taxonomic%20parameters%20revisited%3A%20tarnished%20gold%20standards&amp;aulast=Stackebrandt&amp;date=2006&amp;spage=152&amp;epage=155&amp;volume=33" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Taxonomic%20parameters%20revisited%3A%20tarnished%20gold%20standards&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B19" class="ref-content "><div class="label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larsen</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Olsen</div> <div class="given-names">GJ</div></div>,&nbsp; <div class="name"><div class="surname">Maidak</div> <div class="given-names">BL</div></div>,&nbsp; <div class="name"><div class="surname">McCaughey</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">Overbeek</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Macke</div> <div class="given-names">TJ</div></div>,&nbsp; <div class="name"><div class="surname">Marsh</div> <div class="given-names">TL</div></div>,&nbsp; <div class="name"><div class="surname">Woese</div> <div class="given-names">CR</div></div>.&nbsp;</div><div class="article-title">The ribosomal database project</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;(pg.&nbsp;<div class="fpage">3021</div>-<div class="lpage">3023</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20ribosomal%20database%20project&amp;author=N%20Larsen&amp;author=GJ%20Olsen&amp;author=BL%20Maidak&amp;author=MJ%20McCaughey&amp;author=R%20Overbeek&amp;author=TJ%20Macke&amp;author=TL%20Marsh&amp;author=CR%20Woese&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=3021-3023" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/21.13.3021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.13.3021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F21.13.3021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8332524" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20ribosomal%20database%20project&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B20" class="ref-content "><div class="label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Chai</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Farris</div> <div class="given-names">RJ</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Kulam</div> <div class="given-names">SA</div></div>,&nbsp; <div class="name"><div class="surname">McGarrell</div> <div class="given-names">DM</div></div>,&nbsp; <div class="name"><div class="surname">Garrity</div> <div class="given-names">GM</div></div>,&nbsp; <div class="name"><div class="surname">Tiedje</div> <div class="given-names">JM</div></div>.&nbsp;</div><div class="article-title">The Ribosomal Database Project (RDP-II): sequences and tools for high-throughput rRNA analysis</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">D294</div>-<div class="lpage">D296</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Ribosomal%20Database%20Project%20%28RDP-II%29%3A%20sequences%20and%20tools%20for%20high-throughput%20rRNA%20analysis&amp;author=JR%20Cole&amp;author=B%20Chai&amp;author=RJ%20Farris&amp;author=Q%20Wang&amp;author=SA%20Kulam&amp;author=DM%20McGarrell&amp;author=GM%20Garrity&amp;author=JM%20Tiedje&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D294-D296" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gki038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608200" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Ribosomal%20Database%20Project%20%28RDP-II%29%3A%20sequences%20and%20tools%20for%20high-throughput%20rRNA%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B21" class="ref-content "><div class="label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">SF</div></div>,&nbsp; <div class="name"><div class="surname">Madden</div> <div class="given-names">TL</div></div>,&nbsp; <div class="name"><div class="surname">Schäffer</div> <div class="given-names">AA</div></div>,&nbsp; <div class="name"><div class="surname">Zhang</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Miller</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Lipman</div> <div class="given-names">DJ</div></div>.&nbsp;</div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">25</div>&nbsp;(pg.&nbsp;<div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=SF%20Altschul&amp;author=TL%20Madden&amp;author=AA%20Sch%C3%A4ffer&amp;author=J%20Zhang&amp;author=Z%20Zhang&amp;author=W%20Miller&amp;author=DJ%20Lipman&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B22" class="ref-content "><div class="label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Garrity</div> <div class="given-names">GM</div></div>,&nbsp; <div class="name"><div class="surname">Tiedje</div> <div class="given-names">JM</div></div>,&nbsp; <div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>.&nbsp;</div><div class="article-title">Naïve Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</div>,&nbsp;<div class="source">Appl. Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">73</div>&nbsp;(pg.&nbsp;<div class="fpage">5261</div>-<div class="lpage">5267</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Na%C3%AFve%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;author=Q%20Wang&amp;author=GM%20Garrity&amp;author=JM%20Tiedje&amp;author=JR%20Cole&amp;publication_year=2007&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=73&amp;pages=5261-5267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.00062-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00062-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.00062-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Na%C3%AFve%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B23" class="ref-content "><div class="label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Claesson</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">O'Sullivan</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Nikkilä</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Marchesi</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Smidt</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">de Vos</div> <div class="given-names">WM</div></div>,&nbsp; <div class="name"><div class="surname">Ross</div> <div class="given-names">RP</div></div>,&nbsp; <div class="name"><div class="surname">O'Toole</div> <div class="given-names">PW</div></div>.&nbsp;</div><div class="article-title">Comparative
 analysis of pyrosequencing and a phylogenetic microarray for exploring 
microbial community structures in the human distal intestine</div>,&nbsp;<div class="source">PLoS One</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">4</div>&nbsp;pg.&nbsp;<div class="fpage">e6669</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20pyrosequencing%20and%20a%20phylogenetic%20microarray%20for%20exploring%20microbial%20community%20structures%20in%20the%20human%20distal%20intestine&amp;author=MJ%20Claesson&amp;author=O%20O%27Sullivan&amp;author=Q%20Wang&amp;author=J%20Nikkil%C3%A4&amp;author=JR%20Marchesi&amp;author=H%20Smidt&amp;author=WM%20de%20Vos&amp;author=RP%20Ross&amp;author=PW%20O%27Toole&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e6669" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0006669" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0006669" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0006669"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19693277" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Comparative%20analysis%20of%20pyrosequencing%20and%20a%20phylogenetic%20microarray%20for%20exploring%20microbial%20community%20structures%20in%20the%20human%20distal%20intestine&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B24" class="ref-content "><div class="label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sul</div> <div class="given-names">WJ</div></div>,&nbsp; <div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Jesus</div> <div class="given-names">EC</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Farris</div> <div class="given-names">RJ</div></div>,&nbsp; <div class="name"><div class="surname">Fish</div> <div class="given-names">JA</div></div>,&nbsp; <div class="name"><div class="surname">Tiedje</div> <div class="given-names">JM</div></div>.&nbsp;</div><div class="article-title">Bacterial community comparisons by taxonomy-supervised analysis independent of sequence alignment and clustering</div>,&nbsp;<div class="source">Proc. Natl Acad. Sci. U.S.A.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">108</div>&nbsp;(pg.&nbsp;<div class="fpage">14637</div>-<div class="lpage">14642</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Bacterial%20community%20comparisons%20by%20taxonomy-supervised%20analysis%20independent%20of%20sequence%20alignment%20and%20clustering&amp;author=WJ%20Sul&amp;author=JR%20Cole&amp;author=EC%20Jesus&amp;author=Q%20Wang&amp;author=RJ%20Farris&amp;author=JA%20Fish&amp;author=JM%20Tiedje&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.S.A.&amp;volume=108&amp;pages=14637-14642" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.1111435108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1111435108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.1111435108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873204" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Bacterial%20community%20comparisons%20by%20taxonomy-supervised%20analysis%20independent%20of%20sequence%20alignment%20and%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B25" class="ref-content "><div class="label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Werner</div> <div class="given-names">JJ</div></div>,&nbsp; <div class="name"><div class="surname">Koren</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">DeSantis</div> <div class="given-names">TZ</div></div>,&nbsp; <div class="name"><div class="surname">Walters</div> <div class="given-names">WA</div></div>,&nbsp; <div class="name"><div class="surname">Caporaso</div> <div class="given-names">JG</div></div>,&nbsp; <div class="name"><div class="surname">Angenent</div> <div class="given-names">LT</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Ley</div> <div class="given-names">RE</div></div>.&nbsp;</div><div class="article-title">Impact of training sets on classification of high-throughput bacterial 16s rRNA gene surveys</div>,&nbsp;<div class="source">ISME J.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">94</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20training%20sets%20on%20classification%20of%20high-throughput%20bacterial%2016s%20rRNA%20gene%20surveys&amp;author=JJ%20Werner&amp;author=O%20Koren&amp;author=P%20Hugenholtz&amp;author=TZ%20DeSantis&amp;author=WA%20Walters&amp;author=JG%20Caporaso&amp;author=LT%20Angenent&amp;author=R%20Knight&amp;author=RE%20Ley&amp;publication_year=2012&amp;journal=ISME%20J.&amp;volume=6&amp;pages=94-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ismej.2011.82" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2011.82" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fismej.2011.82"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21716311" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Impact%20of%20training%20sets%20on%20classification%20of%20high-throughput%20bacterial%2016s%20rRNA%20gene%20surveys&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B26" class="ref-content "><div class="label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Newton</div> <div class="given-names">IL</div></div>,&nbsp; <div class="name"><div class="surname">Roeselers</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">The effect of training set on the classification of honey bee gut microbiota using the Naïve Bayesian Classifier</div>,&nbsp;<div class="source">BMC Microbiol.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;pg.&nbsp;<div class="fpage">221</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20training%20set%20on%20the%20classification%20of%20honey%20bee%20gut%20microbiota%20using%20the%20Na%C3%AFve%20Bayesian%20Classifier&amp;author=IL%20Newton&amp;author=G%20Roeselers&amp;publication_year=2012&amp;journal=BMC%20Microbiol.&amp;volume=12&amp;pages=221" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2180-12-221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-12-221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2180-12-221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23013113" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20effect%20of%20training%20set%20on%20the%20classification%20of%20honey%20bee%20gut%20microbiota%20using%20the%20Na%C3%AFve%20Bayesian%20Classifier&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B27" class="ref-content "><div class="label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myers</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">A fast bit-vector algorithm for approximate string matching based on dynamic programming</div>,&nbsp;<div class="source">J. ACM</div>&nbsp;,&nbsp;<div class="year">1999</div>,&nbsp;vol.&nbsp;<div class="volume">46</div>&nbsp;(pg.&nbsp;<div class="fpage">1</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20fast%20bit-vector%20algorithm%20for%20approximate%20string%20matching%20based%20on%20dynamic%20programming&amp;author=G%20Myers&amp;publication_year=1999&amp;journal=J.%20ACM&amp;volume=46&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1145/316542.316550" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1145%2F316542.316550" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1145%2F316542.316550"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20fast%20bit-vector%20algorithm%20for%20approximate%20string%20matching%20based%20on%20dynamic%20programming&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B28" class="ref-content "><div class="label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bruno</div> <div class="given-names">WJ</div></div>,&nbsp; <div class="name"><div class="surname">Socci</div> <div class="given-names">ND</div></div>,&nbsp; <div class="name"><div class="surname">Halpern</div> <div class="given-names">AL</div></div>.&nbsp;</div><div class="article-title">Weighted neighbor joining: a likelihood-based approach to distance-based phylogeny reconstruction</div>,&nbsp;<div class="source">Mol. Biol. Evol.</div>&nbsp;,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">17</div>&nbsp;(pg.&nbsp;<div class="fpage">189</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Weighted%20neighbor%20joining%3A%20a%20likelihood-based%20approach%20to%20distance-based%20phylogeny%20reconstruction&amp;author=WJ%20Bruno&amp;author=ND%20Socci&amp;author=AL%20Halpern&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=189-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a026231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10666718" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Weighted%20neighbor%20joining%3A%20a%20likelihood-based%20approach%20to%20distance-based%20phylogeny%20reconstruction&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B29" class="ref-content "><div class="label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Cardenas</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Fish</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Chai</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Farris</div> <div class="given-names">RJ</div></div>,&nbsp; <div class="name"><div class="surname">Kulam-Syed-Mohideen</div> <div class="given-names">AS</div></div>,&nbsp; <div class="name"><div class="surname">McGarrell</div> <div class="given-names">DM</div></div>,&nbsp; <div class="name"><div class="surname">Marsh</div> <div class="given-names">TL</div></div>,&nbsp; <div class="name"><div class="surname">Garrity</div> <div class="given-names">GM</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">The Ribosomal Database Project: improved alignments and new tools for rRNA analysis</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">37</div>&nbsp;(pg.&nbsp;<div class="fpage">D141</div>-<div class="lpage">D145</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Ribosomal%20Database%20Project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;author=JR%20Cole&amp;author=Q%20Wang&amp;author=E%20Cardenas&amp;author=J%20Fish&amp;author=B%20Chai&amp;author=RJ%20Farris&amp;author=AS%20Kulam-Syed-Mohideen&amp;author=DM%20McGarrell&amp;author=TL%20Marsh&amp;author=GM%20Garrity&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D141-D145" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkn879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn879"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19004872" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Ribosomal%20Database%20Project%3A%20improved%20alignments%20and%20new%20tools%20for%20rRNA%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B30" class="ref-content "><div class="label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Chai</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Tiedje</div> <div class="given-names">JM</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">de Bruijn</div> <div class="given-names">FJ</div></div>.&nbsp;</div><div class="article-title">The Ribosomal Database Project: sequences and software for high-throughput rRNA analysis</div>,&nbsp;<div class="source">Handbook of Molecular Microbial Ecology I: Metagenomics and Complementary Approaches</div>&nbsp;,&nbsp;<div class="year">2011</div><div class="publisher-loc">Hoboken, NJ</div><div class="publisher-name">J. Wiley &amp; Sons, Inc.</div>(pg.&nbsp;<div class="fpage">313</div>-<div class="lpage">324</div>)<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Handbook%20of%20Molecular%20Microbial%20Ecology%20I%3A%20Metagenomics%20and%20Complementary%20Approaches&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Handbook%20of%20Molecular%20Microbial%20Ecology%20I%3A%20Metagenomics%20and%20Complementary%20Approaches&amp;aulast=Cole&amp;date=2011&amp;spage=313&amp;epage=324" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Handbook%20of%20Molecular%20Microbial%20Ecology%20I%3A%20Metagenomics%20and%20Complementary%20Approaches&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Handbook%20of%20Molecular%20Microbial%20Ecology%20I%3A%20Metagenomics%20and%20Complementary%20Approaches">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B31" class="ref-content "><div class="label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fish</div> <div class="given-names">JA</div></div>,&nbsp; <div class="name"><div class="surname">Chai</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Wang</div> <div class="given-names">Q</div></div>,&nbsp; <div class="name"><div class="surname">Yanni</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Brown</div> <div class="given-names">CT</div></div>,&nbsp; <div class="name"><div class="surname">Tiedje</div> <div class="given-names">JM</div></div>,&nbsp; <div class="name"><div class="surname">Cole</div> <div class="given-names">JR</div></div>.&nbsp;</div><div class="article-title">FunGene: the functional gene pipeline and repository</div>,&nbsp;<div class="source">Front. Terr. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">4</div>&nbsp;pg.&nbsp;<div class="fpage">291</div>&nbsp;<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=FunGene%3A%20the%20functional%20gene%20pipeline%20and%20repository&amp;aulast=Fish&amp;date=2013&amp;spage=291&amp;volume=4" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:FunGene%3A%20the%20functional%20gene%20pipeline%20and%20repository&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B32" class="ref-content "><div class="label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McDonald</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Clemente</div> <div class="given-names">JC</div></div>,&nbsp; <div class="name"><div class="surname">Kuczynski</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Rideout</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Stombaugh</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Wendel</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Wilke</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Huse</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Hufnagle</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Meyer</div> <div class="given-names">F</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">The Biological Observation Matrix (BIOM) format or: how I learned to stop worrying and love the ome-ome</div>,&nbsp;<div class="source">GigaScience</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">1</div>&nbsp;pg.&nbsp;<div class="fpage">7</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20Biological%20Observation%20Matrix%20%28BIOM%29%20format%20or%3A%20how%20I%20learned%20to%20stop%20worrying%20and%20love%20the%20ome-ome&amp;author=D%20McDonald&amp;author=JC%20Clemente&amp;author=J%20Kuczynski&amp;author=JR%20Rideout&amp;author=J%20Stombaugh&amp;author=D%20Wendel&amp;author=A%20Wilke&amp;author=S%20Huse&amp;author=J%20Hufnagle&amp;author=F%20Meyer&amp;publication_year=2012&amp;journal=GigaScience&amp;volume=1&amp;pages=7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/2047-217X-1-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F2047-217X-1-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587224" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20Biological%20Observation%20Matrix%20%28BIOM%29%20format%20or%3A%20how%20I%20learned%20to%20stop%20worrying%20and%20love%20the%20ome-ome&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B33" class="ref-content "><div class="label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>.&nbsp;</div><div class="article-title">Search and clustering orders of magnitude faster than BLAST</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">2460</div>-<div class="lpage">2461</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;author=RC%20Edgar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2460-2461" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B34" class="ref-content "><div class="label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Niu</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Zhu</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Wu</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Li</div> <div class="given-names">W</div></div>.&nbsp;</div><div class="article-title">CD-HIT: accelerated for clustering the next generation sequencing data</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">28</div>&nbsp;(pg.&nbsp;<div class="fpage">3150</div>-<div class="lpage">3152</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=CD-HIT%3A%20accelerated%20for%20clustering%20the%20next%20generation%20sequencing%20data&amp;author=L%20Fu&amp;author=B%20Niu&amp;author=Z%20Zhu&amp;author=S%20Wu&amp;author=W%20Li&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3150-3152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bts565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbts565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:CD-HIT%3A%20accelerated%20for%20clustering%20the%20next%20generation%20sequencing%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B35" class="ref-content "><div class="label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loewenstein</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Portugaly</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Fromer</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Linial</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Efficient algorithms for accurate hierarchical clustering of huge datasets: tackling the entire protein space</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">24</div>&nbsp;(pg.&nbsp;<div class="fpage">i41</div>-<div class="lpage">i49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Efficient%20algorithms%20for%20accurate%20hierarchical%20clustering%20of%20huge%20datasets%3A%20tackling%20the%20entire%20protein%20space&amp;author=Y%20Loewenstein&amp;author=E%20Portugaly&amp;author=M%20Fromer&amp;author=M%20Linial&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=i41-i49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btn174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtn174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18586742" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Efficient%20algorithms%20for%20accurate%20hierarchical%20clustering%20of%20huge%20datasets%3A%20tackling%20the%20entire%20protein%20space&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B36" class="ref-content "><div class="label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Cai</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Liu</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Yu</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Farrell</div> <div class="given-names">ML</div></div>,&nbsp; <div class="name"><div class="surname">McKendree</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Farmerie</div> <div class="given-names">W</div></div>.&nbsp;</div><div class="article-title">ESPRIT: estimating species richness using large collections of 16S rRNA pyrosequences</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">37</div>&nbsp;pg.&nbsp;<div class="fpage">e76</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ESPRIT%3A%20estimating%20species%20richness%20using%20large%20collections%20of%2016S%20rRNA%20pyrosequences&amp;author=Y%20Sun&amp;author=Y%20Cai&amp;author=L%20Liu&amp;author=F%20Yu&amp;author=ML%20Farrell&amp;author=W%20McKendree&amp;author=W%20Farmerie&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=e76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/nar/gkp285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkp285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19417062" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ESPRIT%3A%20estimating%20species%20richness%20using%20large%20collections%20of%2016S%20rRNA%20pyrosequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B37" class="ref-content "><div class="label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schloss</div> <div class="given-names">PD</div></div>,&nbsp; <div class="name"><div class="surname">Westcott</div> <div class="given-names">SL</div></div>,&nbsp; <div class="name"><div class="surname">Ryabin</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Hall</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Hartmann</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Hollister</div> <div class="given-names">EB</div></div>,&nbsp; <div class="name"><div class="surname">Lesniewski</div> <div class="given-names">RA</div></div>,&nbsp; <div class="name"><div class="surname">Oakley</div> <div class="given-names">BB</div></div>,&nbsp; <div class="name"><div class="surname">Parks</div> <div class="given-names">DH</div></div>,&nbsp; <div class="name"><div class="surname">Robinson</div> <div class="given-names">CJ</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Introducing
 mothur: open-source, platform-independent, community-supported software
 for describing and comparing microbial communities</div>,&nbsp;<div class="source">Appl. Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">75</div>&nbsp;(pg.&nbsp;<div class="fpage">7537</div>-<div class="lpage">7541</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Introducing%20mothur%3A%20open-source%2C%20platform-independent%2C%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;author=PD%20Schloss&amp;author=SL%20Westcott&amp;author=T%20Ryabin&amp;author=JR%20Hall&amp;author=M%20Hartmann&amp;author=EB%20Hollister&amp;author=RA%20Lesniewski&amp;author=BB%20Oakley&amp;author=DH%20Parks&amp;author=CJ%20Robinson&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=75&amp;pages=7537-7541" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.01541-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01541-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.01541-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Introducing%20mothur%3A%20open-source%2C%20platform-independent%2C%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B38" class="ref-content "><div class="label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Masella</div> <div class="given-names">AP</div></div>,&nbsp; <div class="name"><div class="surname">Bartram</div> <div class="given-names">AK</div></div>,&nbsp; <div class="name"><div class="surname">Truszkowski</div> <div class="given-names">JM</div></div>,&nbsp; <div class="name"><div class="surname">Brown</div> <div class="given-names">DG</div></div>,&nbsp; <div class="name"><div class="surname">Neufeld</div> <div class="given-names">JD</div></div>.&nbsp;</div><div class="article-title">PANDAseq: paired-end assembler for illumina sequences</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">13</div>&nbsp;pg.&nbsp;<div class="fpage">31</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PANDAseq%3A%20paired-end%20assembler%20for%20illumina%20sequences&amp;author=AP%20Masella&amp;author=AK%20Bartram&amp;author=JM%20Truszkowski&amp;author=DG%20Brown&amp;author=JD%20Neufeld&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-13-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-13-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22333067" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PANDAseq%3A%20paired-end%20assembler%20for%20illumina%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="gkt1244-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="gkt1244-B39" class="ref-content "><div class="label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozich</div> <div class="given-names">JJ</div></div>,&nbsp; <div class="name"><div class="surname">Westcott</div> <div class="given-names">SL</div></div>,&nbsp; <div class="name"><div class="surname">Baxter</div> <div class="given-names">NT</div></div>,&nbsp; <div class="name"><div class="surname">Highlander</div> <div class="given-names">SK</div></div>,&nbsp; <div class="name"><div class="surname">Schloss</div> <div class="given-names">PD</div></div>.&nbsp;</div><div class="article-title">Development
 of a dual-index sequencing strategy and curation pipeline for analyzing
 amplicon sequence data on the MiSeq Illumina sequencing platform</div>,&nbsp;<div class="source">Appl. Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">79</div>&nbsp;(pg.&nbsp;<div class="fpage">5112</div>-<div class="lpage">5120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20dual-index%20sequencing%20strategy%20and%20curation%20pipeline%20for%20analyzing%20amplicon%20sequence%20data%20on%20the%20MiSeq%20Illumina%20sequencing%20platform&amp;author=JJ%20Kozich&amp;author=SL%20Westcott&amp;author=NT%20Baxter&amp;author=SK%20Highlander&amp;author=PD%20Schloss&amp;publication_year=2013&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=79&amp;pages=5112-5120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1128/AEM.01043-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01043-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.01043-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23793624" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Development%20of%20a%20dual-index%20sequencing%20strategy%20and%20curation%20pipeline%20for%20analyzing%20amplicon%20sequence%20data%20on%20the%20MiSeq%20Illumina%20sequencing%20platform&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">© The Author(s) 2013. Published by Oxford University Press.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution License 
(http://creativecommons.org/licenses/by/3.0/), which permits 
unrestricted reuse, distribution, and reproduction in any medium, 
provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=1063201&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5127">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/42/D1/10.1093_nar_gkt1244/3/gkt1244_Supplementary_Data.zip?Expires=1568374196&amp;Signature=E4hhu8wlP24sz1EQTP-QYRYKdnLF7sNqw7yvDkpygRo9u%7E7LDlEYrziyaAkd-xJGhJKzFnSaJtV5wg1hZwvN4d9bHizS8Dpg3vT4YFDIVMOr3ZIwUuphBbHAAlbhOuhN59WEsX5slB0w87KvhmOjv3aCAKBVvXe2v9702JmrGeUVbzFj%7ETvaPotVVdOyufd-j6FO3c1wKKx6KeBXgBEB%7EFp2OeCcAlhIwfX-BVxN917TM4Jo%7Elaj8ceSzp9NK2zKe7s5Vlm9fsWSPz05nSmxCOGJySVXFzSlWYWwgeie7JakfxEVRiHqV-0HB4slUXxs4r5pUZMZ24aFqkP1Brcn4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

    <div class="comment-body__wrap">
        <div id="usercomments" class="comment-body__title">
            <h2>Comments</h2>
        </div>
        <div class="comment-body__number">
            0 Comments
        </div>
        <div class="submitCommentPlaceHolder"><div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
            <button id="comment-modal-opener" class="btn comment-body__button" data-reveal="">Add comment</button>
    <div id="divCommentModal" class="comment-form__modal medium reveal-modal" data-reveal="">

        <button class="comment-form__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close comment form modal</span>
        </button>

        <div class="preloader-wrap hide">
            <div class="preloader">
                <div class="spinner"></div>
            </div>
        </div>

        <div id="TermsAndConditionsContainer" class="comment-form__wrap" data-reveal=""></div>

            <div id="termsAgreeRow" class="comment-form__row--last terms-conditions">
                <input id="termsAndConditionsCheckbox" class="comment-form__input comment-form__checkbox" title="Terms and Conditions" required="required" type="checkbox">
                <label class="comment-form__required-statement required" for="termsAndConditionsCheckbox">I agree to the terms and conditions.</label>
                <label class="lblTermsAndConditions errors comment-form__error-label" for="termsAndConditionsCheckbox">
                    You must accept the terms and conditions.
                </label>
            </div>

        <div id="termsButtonRow" class="termsButtonRow comment-form__row">
            <button id="btnUserAgreeTerms" class="btnUserAgreeTerms btn btn-grey">Add comment</button>
            <button href="#" class="btnModalCancel btn btn-grey">Cancel</button>
        </div>


        <div id="divUserCommentForm" class="comment-form__wrap">

            <div class="comment-form__title">Submit a comment</div>



                <div class="comment-form__row">
                    <label class="comment-form__input-label required" for="Author">Name</label>

                    <input class="comment-form__input comment-form__textbox required" dataauthorvalue="" id="Author" maxlength="2000" name="Author" required="required" type="text">
                </div>


                <div class="comment-form__row">
                    <label class="comment-form__input-label required" for="AffInstitution">Affiliations</label>
                    <input class="comment-form__input comment-form__textbox required" id="AffInstitution" maxlength="200" name="AffInstitution" required="required" type="text">
                </div>



            <div class="comment-form__row">
                <label class="comment-form__input-label required" for="Title">Comment title</label>
                <input class="comment-form__input comment-form__textbox required" id="Title" maxlength="250" name="Title" required="required" type="text">
            </div>

            <div class="comment-form__row">
                <label class="comment-form__input-label required" for="Comment">Comment</label>
                <textarea class="comment-form__input comment-form__textarea required" cols="20" id="Comment" maxlength="@Model.Config.CommentBodyCharLimit" name="Comment" required="required" rows="2"></textarea>
                </div>


                <div class="captchaAlert comment-form__error-label">
                    You have entered an invalid code
                </div>

                <div class="submitRow comment-form__row">
                    <button id="btnUserCommentSubmit" class="btnUserCommentSubmit btn btn-grey">Submit</button>
                    <button href="#" class="btnModalCancel btn btn-grey">Cancel</button>
                </div>

            </div><!-- /.comment-form__wrap -->

            <div class="content-form__response comment-form__row">
                
                <div class="content-form__response--success">
                    <div class="alertmsg content-form__response-text" id="displayAlert">
                        Thank you for submitting a comment on this 
article. Your comment will be reviewed and published at the journal's 
discretion. Please check for further notifications by email.
                    </div>
                    <button class="btnCommentModalExit content-form__response-button btn btn-grey">Close</button>
                </div>
                
                            
                <div id="divError" class="errors content-form__response--error"></div>
            </div><!-- /.comment-form__response -->

        </div><!-- /.comment-form__modal -->
                        <script type="text/javascript">

                            /*******************************************************************************
                             * JS here is only being used to assign variables from values in the model
                             * logic should be implemented in external JS files, like client.script.js
                             *******************************************************************************/

                            var SCM = SCM || {};
                            SCM.UserCommentView = SCM.UserCommentView || {};

                            SCM.UserCommentView.articleId = '1063201';
                            SCM.UserCommentView.isUserLoggedIn = 'False';
                            SCM.UserCommentView.divId = 'WidgetNoAccessModalSmallOUP_UserComment_Article';
                            SCM.UserCommentView.bShowDisclosure = 'false';
                            SCM.UserCommentView.isNotModerated = 'false';
                            SCM.UserCommentView.showAffiliations = 'true';
                            SCM.UserCommentView.InstanceName = 'OUP_UserComment_Article';
                        </script>
 
    </div></div>
        
        
    </div>
 
    </div>
    

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">7,298</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000331139800093&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">1,219</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/nar/gkt1244" data-uuid="8f2012fe-cabb-ca53-f4dc-2029f278eb27"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=1944011" style="display:inline-block;">
        <img alt="Article has an altmetric score of 17" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1063201"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                <div class="userAlert alertType-33">
                    <a role="link" aria-label="Subject alert" data-userloggedin="False" data-alerttype="33">Subject alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
                <h3 class="article-widget-head">More on this topic</h3>
    <div class="widget-related-content">
            <div class="article-title"><a href="https://academic.oup.com/nar/article/40/W1/W82/1073944">TaxMan: a server to trim rRNA reference databases and inspect taxonomic coverage</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/44/11/5022/2468320">Phylogeny-aware identification and correction of taxonomically mislabeled sequences</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/45/W1/W509/3782595">Interactive microbial distribution analysis using BioAtlas</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/36/18/e120/1070009">Accurate taxonomy assignments from 16S rRNA sequences produced by highly parallel pyrosequencers</a></div>
    </div>
 
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000331139800093&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fnar%2farticle%2f42%2fD1%2fD633%2f1063201">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-RelatedPubMed widget-instance-Article_RightRailB0Article_RightRail_RelatedPubMed">
        <h3 class="article-widget-head">Related articles in PubMed</h3>
<div class="widget-related-pubmed">
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/31408831" id="article_ArticleTitle" target="">Small molecule HDAC inhibitors: Promising agents for breast cancer treatment.</a></div>
                <!-- <div class="citation"><span>Bioorg Chem</span> 2019;</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/31408790" id="article_ArticleTitle" target="">Oscillatory connectivity as a diagnostic marker of dementia due to Alzheimer's disease.</a></div>
                <!-- <div class="citation"><span>Clin Neurophysiol</span> 2019;</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/31408766" id="article_ArticleTitle" target="">Positioning
 Quebec ORF5 sequences of porcine reproductive and respiratory syndrome 
virus (PRRSV) within Canada and worldwide diversity.</a></div>
                <!-- <div class="citation"><span>Infect Genet Evol</span> 2019.</div> -->
            </div>
        </div>
</div>
<!-- <div class="article-widget-foot">
    <div class="view-more">
            <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=((ribosomes)+OR+(dystonia+AND+12)+OR+(taxonomy)+OR+(ribosomal+AND+rna))" id="Config_ViewMoreLink" class="view-more-link" target="_blank">View More</a>
    </div>
</div> -->
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000331139800093&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (1219)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F42%2FD1%2FD633%2F1063201" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/nar/crossref-citedby/1063201">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz774/5568216?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   regBase: whole genome base-wise 
aggregation and functional prediction for human non-coding regulatory 
variants
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz767/5568215?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   At least two molecules of the RNA 
helicase Has1 are simultaneously present in pre-ribosomes during 
ribosome biogenesis
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz773/5568213?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Dynamical ensemble of the active 
state and transition state mimic for the RNA-cleaving 8–17 DNAzyme in 
solution
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz787/5568212?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Stage-specific combinations of 
opposing poly(A) modifying enzymes guide gene expression during early 
oogenesis
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gkz783/5568211?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Translational recoding: canonical translation mechanisms reinterpreted
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="Ribosomal Database Project: data and tools for high throughput rRNA analysis | Nucleic Acids Research | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/nar/gkt1244");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/nar_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/nar/pages/About">About Nucleic Acids Research</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/nar/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/nar/pages/Policies">Policies</a>
                            </li>
                            <li class="link-4">
                                <a href="https://academic.oup.com/nar/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/nar_open">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="https://www.linkedin.com/in/nar-nucleic-acids-research-5297133b">LinkedIn</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-4">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-NucleicAcidsResearch" class="journal-footer-affiliations-logo" src="nar_f1248941275.svg" alt="Nucleic Acids Research">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1362-4962</li>
                                            <li>Print ISSN 0305-1048</li>
                    <li>Copyright © 2019 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh54" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1568293448876&amp;tmr=load%3D1568293448846%26core%3D1568293448870%26main%3D1568293448874%26ifr%3D1568293448880&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F42%2FD1%2FD633%2F1063201&amp;href=https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F42%2FD1%2FD633%2F1063201&amp;dt=Ribosomal%20Database%20Project%3A%20data%20and%20tools%20for%20high%20throughput%20rRNA%20analysis&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5d7a4248ec46a36d&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DRibosomal%2520Database%2520Project%253A%2520data%2520and%2520tools%2520for%2520high%2520throughput%2520rRNA%2520analysis%26description%3DAbstract.%2520%2520Ribosomal%2520Database%2520Project%2520%28RDP%253B%2520http%253A%252F%252Frdp.cme.msu.edu%252F%29%2520provides%2520the%2520research%2520community%2520with%2520aligned%2520and%2520annotated%2520rRNA%2520gene%2520sequence%2520data%252C%2520along%2520w%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fnar%252Farticle%252F42%252FD1%252FD633%252F1063201%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fnar%252FIssue%252F42%252FD1%252F3%252Fm_cover.gif%253FExpires%253D1631352597%2526Signature%253DXjDhDUAPTfs-ENvTGXOBdn3E0GqPXKYUpDYESNOvUFhO%7EixeUXdLGs3IZEU-AHBXGxeGj25W4HB5mBpJLIL25AZYs5oRF4-zWFBAm%7EdkBRP-wJau7nGbqCKthlzfdQxTNQX5RqbNeRFljCuBkscFYBM2EWL0nheqbySxVoPpoRFAh5hhq57Ffu3j9Olw9tzUKxatEeEyqYGYxgFGuBM9At4KQGwCsu7qqrCArsfnP2Lpw8YF5eDTIjsXGXxS2OsrFPoU3B1DmwKJtG9V-ono2Kx9cXhkt%7E5IPGl5ogdLrabos4BMfZXnV0UlJVgyJLdKTS6xBNG-VNZo6tu0Y9mgbg__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2013-11-27%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.27.7-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f42%2fD1%2fD633%2f1063201">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>

    <script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>




<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="gkt1244.js"></script></body></html>