<!DOCTYPE html>
<html class="js" style="--font-family-sans: Europa; --font-family-serif: Noto Serif;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <title>MUSCLE: a multiple sequence alignment method with reduced time and space complexity | BMC Bioinformatics | Full Text</title>

        
            

    <meta name="citation_abstract" content="In a previous paper, we introduced MUSCLE, a new program for creating multiple alignments of protein sequences, giving a brief summary of the algorithm and showing MUSCLE to achieve the highest scores reported to date on four alignment accuracy benchmarks. Here we present a more complete discussion of the algorithm, describing several previously unpublished techniques that improve biological accuracy and / or computational complexity. We introduce a new option, MUSCLE-fast, designed for high-throughput applications. We also describe a new protocol for evaluating objective functions that align two profiles. We compare the speed and accuracy of MUSCLE with CLUSTALW, Progressive POA and the MAFFT script FFTNS1, the fastest previously published program known to the author. Accuracy is measured using four benchmarks: BAliBASE, PREFAB, SABmark and SMART. We test three variants that offer highest accuracy (MUSCLE with default settings), highest speed (MUSCLE-fast), and a carefully chosen compromise between the two (MUSCLE-prog). We find MUSCLE-fast to be the fastest algorithm on all test sets, achieving average alignment accuracy similar to CLUSTALW in times that are typically two to three orders of magnitude less. MUSCLE-fast is able to align 1,000 sequences of average length 282 in 21 seconds on a current desktop computer. MUSCLE offers a range of options that provide improved speed and / or alignment accuracy compared with currently available programs. MUSCLE is freely available at 
                    http://www.drive5.com/muscle
                    
                  .">

    <meta name="description" content="In a previous paper, we introduced MUSCLE, a new program for creating multiple alignments of protein sequences, giving a brief summary of the algorithm and showing MUSCLE to achieve the highest scores reported to date on four alignment accuracy benchmarks. Here we present a more complete discussion of the algorithm, describing several previously unpublished techniques that improve biological accuracy and / or computational complexity. We introduce a new option, MUSCLE-fast, designed for high-throughput applications. We also describe a new protocol for evaluating objective functions that align two profiles. We compare the speed and accuracy of MUSCLE with CLUSTALW, Progressive POA and the MAFFT script FFTNS1, the fastest previously published program known to the author. Accuracy is measured using four benchmarks: BAliBASE, PREFAB, SABmark and SMART. We test three variants that offer highest accuracy (MUSCLE with default settings), highest speed (MUSCLE-fast), and a carefully chosen compromise between the two (MUSCLE-prog). We find MUSCLE-fast to be the fastest algorithm on all test sets, achieving average alignment accuracy similar to CLUSTALW in times that are typically two to three orders of magnitude less. MUSCLE-fast is able to align 1,000 sequences of average length 282 in 21 seconds on a current desktop computer. MUSCLE offers a range of options that provide improved speed and / or alignment accuracy compared with currently available programs. MUSCLE is freely available at 
                    http://www.drive5.com/muscle
                    
                  .">

    <meta name="journal_id" content="12859">

    <meta name="dc.title" content="MUSCLE: a multiple sequence alignment method with reduced time and space complexity">

    <meta name="dc.source" content="BMC Bioinformatics 2004 5:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2004-08-19">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2004 Edgar; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="In a previous paper, we introduced MUSCLE, a new program for creating multiple alignments of protein sequences, giving a brief summary of the algorithm and showing MUSCLE to achieve the highest scores reported to date on four alignment accuracy benchmarks. Here we present a more complete discussion of the algorithm, describing several previously unpublished techniques that improve biological accuracy and / or computational complexity. We introduce a new option, MUSCLE-fast, designed for high-throughput applications. We also describe a new protocol for evaluating objective functions that align two profiles. We compare the speed and accuracy of MUSCLE with CLUSTALW, Progressive POA and the MAFFT script FFTNS1, the fastest previously published program known to the author. Accuracy is measured using four benchmarks: BAliBASE, PREFAB, SABmark and SMART. We test three variants that offer highest accuracy (MUSCLE with default settings), highest speed (MUSCLE-fast), and a carefully chosen compromise between the two (MUSCLE-prog). We find MUSCLE-fast to be the fastest algorithm on all test sets, achieving average alignment accuracy similar to CLUSTALW in times that are typically two to three orders of magnitude less. MUSCLE-fast is able to align 1,000 sequences of average length 282 in 21 seconds on a current desktop computer. MUSCLE offers a range of options that provide improved speed and / or alignment accuracy compared with currently available programs. MUSCLE is freely available at 
                    http://www.drive5.com/muscle
                    
                  .">

    <meta name="prism.issn" content="1471-2105">

    <meta name="prism.publicationName" content="BMC Bioinformatics">

    <meta name="prism.publicationDate" content="2004-08-19">

    <meta name="prism.volume" content="5">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="113">

    <meta name="prism.copyright" content="2004 Edgar; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113">

    <meta name="prism.doi" content="doi:10.1186/1471-2105-5-113">

    <meta name="citation_pdf_url" content="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-113">

    <meta name="citation_fulltext_html_url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113">

    <meta name="citation_journal_title" content="BMC Bioinformatics">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2105">

    <meta name="citation_title" content="MUSCLE: a multiple sequence alignment method with reduced time and space complexity">

    <meta name="citation_volume" content="5">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2004/12">

    <meta name="citation_online_date" content="2004/08/19">

    <meta name="citation_firstpage" content="113">

    <meta name="citation_article_type" content="Software">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/1471-2105-5-113">

    <meta name="DOI" content="10.1186/1471-2105-5-113">

    <meta name="citation_doi" content="10.1186/1471-2105-5-113">

    <meta name="dc.creator" content="Robert C Edgar">

    <meta name="dc.subject" content="Bioinformatics">

    <meta name="dc.subject" content="Microarrays">

    <meta name="dc.subject" content="Computational Biology/Bioinformatics">

    <meta name="dc.subject" content="Computer Appl. in Life Sciences">

    <meta name="dc.subject" content="Combinatorial Libraries">

    <meta name="dc.subject" content="Algorithms">

    <meta name="citation_reference" content="citation_journal_title=Pharmacogenomics; citation_title=Recent progress in multiple sequence alignment: a survey; citation_author=C Notredame; citation_volume=3; citation_issue=1; citation_publication_date=2002; citation_pages=131-144; citation_doi=10.1517/14622416.3.1.131; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; citation_author=RC Edgar; citation_volume=32; citation_issue=5; citation_publication_date=2004; citation_pages=1792-1797; citation_doi=10.1093/nar/gkh340; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=J Comput Biol; citation_title=On the complexity of multiple sequence alignment; citation_author=L Wang, T Jiang; citation_volume=1; citation_issue=4; citation_publication_date=1994; citation_pages=337-348; citation_doi=10.1089/cmb.1994.1.337; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Adv in Math; citation_title=Some biological sequence metrics; citation_author=MS Waterman, TF Smith, WA Beyer; citation_volume=20; citation_publication_date=1976; citation_pages=367-387; citation_doi=10.1016/0001-8708(76)90202-4; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment; citation_author=CE Lawrence, SF Altschul, MS Boguski, JS Liu, AF Neuwald, JC Wootton; citation_volume=262; citation_issue=5131; citation_publication_date=1993; citation_pages=208-214; citation_doi=10.1126/science.8211139; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=The alignment of sets of sequences and the construction of phyletic trees: an integrated method; citation_author=P Hogeweg, B Hesper; citation_volume=20; citation_issue=2; citation_publication_date=1984; citation_pages=175-186; citation_doi=10.1007/BF02257378; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=J Mol Evol; citation_title=Progressive sequence alignment as a prerequisite to correct phylogenetic trees; citation_author=DF Feng, RF Doolittle; citation_volume=25; citation_issue=4; citation_publication_date=1987; citation_pages=351-360; citation_doi=10.1007/BF02603120; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; citation_author=JD Thompson, DG Higgins, TJ Gibson; citation_volume=22; citation_issue=22; citation_publication_date=1994; citation_pages=4673-4680; citation_doi=10.1093/nar/22.22.4673; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=T-Coffee: A novel method for fast and accurate multiple sequence alignment; citation_author=C Notredame, DG Higgins, J Heringa; citation_volume=302; citation_issue=1; citation_publication_date=2000; citation_pages=205-217; citation_doi=10.1006/jmbi.2000.4042; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=BAliBASE (Benchmark Alignment dataBASE): enhancements for repeats, transmembrane sequences and circular permutations; citation_author=A Bahr, JD Thompson, JC Thierry, O Poch; citation_volume=29; citation_issue=1; citation_publication_date=2001; citation_pages=323-326; citation_doi=10.1093/nar/29.1.323; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=BAliBASE: a benchmark alignment database for the evaluation of multiple alignment programs; citation_author=JD Thompson, F Plewniak, O Poch; citation_volume=15; citation_issue=1; citation_publication_date=1999; citation_pages=87-88; citation_doi=10.1093/bioinformatics/15.1.87; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=Comprehensive study on iterative algorithms of multiple sequence alignment; citation_author=M Hirosawa, Y Totoki, M Hoshida, M Ishikawa; citation_volume=11; citation_issue=1; citation_publication_date=1995; citation_pages=13-18; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Significant improvement in accuracy of multiple protein sequence alignments by iterative refinement as assessed by reference to structural alignments; citation_author=O Gotoh; citation_volume=264; citation_issue=4; citation_publication_date=1996; citation_pages=823-838; citation_doi=10.1006/jmbi.1996.0679; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform; citation_author=K Katoh, K Misawa, K Kuma, T Miyata; citation_volume=30; citation_issue=14; citation_publication_date=2002; citation_pages=3059-3066; citation_doi=10.1093/nar/gkf436; citation_id=CR14">

    <meta name="citation_reference" content="citation_title=Align-m–a new algorithm for multiple alignment of highly divergent sequences; citation_inbook_title=Bioinformatics; citation_publication_date=2004; citation_id=CR15; citation_author=I Van Walle; citation_author=I Lasters; citation_author=L Wyns">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=SMART: a web-based tool for the study of genetically mobile domains; citation_author=J Schultz, RR Copley, T Doerks, CP Ponting, P Bork; citation_volume=28; citation_issue=1; citation_publication_date=2000; citation_pages=231-234; citation_doi=10.1093/nar/28.1.231; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=SMART: identification and annotation of domains from signalling and extracellular protein sequences; citation_author=CP Ponting, J Schultz, F Milpetz, P Bork; citation_volume=27; citation_issue=1; citation_publication_date=1999; citation_pages=229-232; citation_doi=10.1093/nar/27.1.229; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci U S A; citation_title=SMART, a simple modular architecture research tool: identification of signaling domains; citation_author=J Schultz, F Milpetz, P Bork, CP Ponting; citation_volume=95; citation_issue=11; citation_publication_date=1998; citation_pages=5857-5864; citation_doi=10.1073/pnas.95.11.5857; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Alignment-free sequence comparison-a review; citation_author=S Vinga, J Almeida; citation_volume=19; citation_issue=4; citation_publication_date=2003; citation_pages=513-523; citation_doi=10.1093/bioinformatics/btg005; citation_id=CR19">

    <meta name="citation_reference" content="citation_inbook_title=A model of evolutionary change in proteins in Atlas of protein sequence and structure,; citation_publication_date=1978; citation_id=CR20; citation_author=MO Dayhoff; citation_author=RM Schwartz; citation_author=BC Orcutt; citation_publisher=National Biomedical Research Foundation">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Local homology recognition and distance measures in linear time using compressed amino acid alphabets; citation_author=RC Edgar; citation_volume=32; citation_issue=1; citation_publication_date=2004; citation_pages=380-385; citation_doi=10.1093/nar/gkh180; citation_id=CR21">

    <meta name="citation_reference" content="citation_inbook_title=The neutral theory of molecular evolution; citation_publication_date=1983; citation_id=CR22; citation_author=M Kimura">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; citation_author=N Saitou, M Nei; citation_volume=4; citation_issue=4; citation_publication_date=1987; citation_pages=406-425; citation_id=CR23">

    <meta name="citation_reference" content="citation_inbook_title=Numerical taxonomy; citation_publication_date=1973; citation_id=CR24; citation_author=PHA Sneath; citation_author=RR Sokal; citation_publisher=Freeman">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=Improved sensitivity of profile searches through the use of sequence weights and gap excision; citation_author=JD Thompson, DG Higgins, TJ Gibson; citation_volume=10; citation_issue=1; citation_publication_date=1994; citation_pages=19-29; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Position-based sequence weights; citation_author=S Henikoff, JG Henikoff; citation_volume=243; citation_issue=4; citation_publication_date=1994; citation_pages=574-578; citation_doi=10.1016/0022-2836(94)90032-9; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; citation_author=SF Altschul, TL Madden, AA Schaffer, J Zhang, Z Zhang, W Miller, DJ Lipman; citation_volume=25; citation_issue=17; citation_publication_date=1997; citation_pages=3389-3402; citation_doi=10.1093/nar/25.17.3389; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Volume changes in protein evolution; citation_author=M Gerstein, EL Sonnhammer, C Chothia; citation_volume=236; citation_issue=4; citation_publication_date=1994; citation_pages=1067-1078; citation_doi=10.1016/0022-2836(94)90012-4; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=A weighting system and algorithm for aligning many phylogenetically related sequences; citation_author=O Gotoh; citation_volume=11; citation_issue=5; citation_publication_date=1995; citation_pages=543-551; citation_id=CR29">

    <meta name="citation_reference" content="citation_title=A comparison of scoring functions for protein sequence profile alignment; citation_inbook_title=Bioinformatics; citation_publication_date=2004; citation_id=CR30; citation_author=RC Edgar; citation_author=K Sjolander">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=Dirichlet mixtures: a method for improved detection of weak but significant protein sequence homology; citation_author=K Sjolander, K Karplus, M Brown, R Hughey, A Krogh, IS Mian, D Haussler; citation_volume=12; citation_issue=4; citation_publication_date=1996; citation_pages=327-345; citation_id=CR31">

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Amino acid substitution matrices from an information theoretic perspective; citation_author=SF Altschul; citation_volume=219; citation_issue=3; citation_publication_date=1991; citation_pages=555-565; citation_doi=10.1016/0022-2836(91)90193-A; citation_id=CR32">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=The rapid generation of mutation data matrices from protein sequences; citation_author=DT Jones, WR Taylor, JM Thornton; citation_volume=8; citation_issue=3; citation_publication_date=1992; citation_pages=275-282; citation_id=CR33">

    <meta name="citation_reference" content="citation_journal_title=Mol Biol Evol; citation_title=Estimating amino acid substitution models: a comparison of Dayhoff's estimator, the resolvent approach and a maximum likelihood method; citation_author=T Muller, R Spang, M Vingron; citation_volume=19; citation_issue=1; citation_publication_date=2002; citation_pages=8-13; citation_doi=10.1093/oxfordjournals.molbev.a003985; citation_id=CR34">

    <meta name="citation_reference" content="citation_title=Improving profile-profile alignment via log average scoring; citation_inbook_title=In: Algorithms in Bioinformatics, First International Workshop, WABI 2001; citation_publication_date=2001; citation_pages=11-26; citation_id=CR35; citation_author=N von Ohsen; citation_author=R Zimmer; citation_publisher=Springer-Verlag">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=A comprehensive comparison of multiple sequence alignment programs; citation_author=JD Thompson, F Plewniak, O Poch; citation_volume=27; citation_issue=13; citation_publication_date=1999; citation_pages=2682-2690; citation_doi=10.1093/nar/27.13.2682; citation_id=CR36">

    <meta name="citation_reference" content="citation_journal_title=Comput Appl Biosci; citation_title=Optimal alignments in linear space; citation_author=EW Myers, W Miller; citation_volume=4; citation_issue=1; citation_publication_date=1988; citation_pages=11-17; citation_id=CR37">

    <meta name="citation_reference" content="citation_inbook_title=Inferring Phylogenies; citation_publication_date=2004; citation_id=CR38; citation_author=J Felsenstein; citation_publisher=Sinauer Associates">

    <meta name="citation_reference" content="citation_journal_title=Proteins; citation_title=Large-scale comparison of protein sequence alignment algorithms with structure alignments; citation_author=JM Sauder, JW Arthur, RL Dunbrack; citation_volume=40; citation_issue=1; citation_publication_date=2000; citation_pages=6-22; citation_doi=10.1002/(SICI)1097-0134(20000701)40:1&lt;6::AID-PROT30&gt;3.0.CO;2-7; citation_id=CR39">

    <meta name="citation_reference" content="citation_inbook_title=Biological sequence analysis; citation_publication_date=1998; citation_id=CR40; citation_author=R Durbin; citation_author=S Eddy; citation_author=A Krogh; citation_author=G Mitchison">

    <meta name="citation_reference" content="citation_inbook_title=Introduction to computational biology; citation_publication_date=1997; citation_id=CR41; citation_author=J Setubal; citation_author=J Meidanis; citation_publisher=Brooks/Cole">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=RASCAL: rapid scanning and correction of multiple sequence alignments; citation_author=JD Thompson, JC Thierry, O Poch; citation_volume=19; citation_issue=9; citation_publication_date=2003; citation_pages=1155-1161; citation_doi=10.1093/bioinformatics/btg133; citation_id=CR42">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=NCBI Reference Sequence project: update and current status; citation_author=KD Pruitt, T Tatusova, DR Maglott; citation_volume=31; citation_issue=1; citation_publication_date=2003; citation_pages=34-37; citation_doi=10.1093/nar/gkg111; citation_id=CR43">

    <meta name="citation_reference" content="citation_title=COACH: profile-profile alignment of protein families using hidden Markov models; citation_inbook_title=Bioinformatics; citation_publication_date=2004; citation_id=CR44; citation_author=RC Edgar; citation_author=K Sjolander">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Touring protein fold space with Dali/FSSP; citation_author=L Holm, C Sander; citation_volume=26; citation_issue=1; citation_publication_date=1998; citation_pages=316-319; citation_doi=10.1093/nar/26.1.316; citation_id=CR45">

    <meta name="citation_reference" content="citation_journal_title=J Mol Biol; citation_title=Within the twilight zone: a sensitive profile-profile comparison tool based on information theory; citation_author=G Yona, M Levitt; citation_volume=315; citation_issue=5; citation_publication_date=2002; citation_pages=1257-1275; citation_doi=10.1006/jmbi.2001.5293; citation_id=CR46">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=Searching databases of conserved sequence regions by aligning protein multiple-alignments; citation_author=S Pietrokovski; citation_volume=24; citation_issue=19; citation_publication_date=1996; citation_pages=3836-3845; citation_doi=10.1093/nar/24.19.3836; citation_id=CR47">

    <meta name="citation_reference" content="citation_title=Combining partial order alignment and progressive multiple sequence alignment increases alignment speed and scalability to very large alignment problems; citation_inbook_title=Bioinformatics; citation_publication_date=2004; citation_id=CR48; citation_author=C Grasso; citation_author=C Lee">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=The Protein Data Bank; citation_author=HM Berman, J Westbrook, Z Feng, G Gilliland, TN Bhat, H Weissig, IN Shindyalov, PE Bourne; citation_volume=28; citation_issue=1; citation_publication_date=2000; citation_pages=235-242; citation_doi=10.1093/nar/28.1.235; citation_id=CR49">

    <meta name="citation_author" content="Robert C Edgar">

    <meta name="citation_author_institution" content="Department of Plant and Microbial Biology, University of California, Berkeley, USA">


        

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">


        <script type="text/javascript" src="https://consumer.krxd.net/consent/set/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;dc=1&amp;al=1&amp;tg=1&amp;cd=0&amp;sh=0&amp;re=0&amp;callback=Krux.ns.macmillan.kxjsonp_consent_set_1"></script><script type="text/javascript" src="https://consumer.krxd.net/consent/get/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;callback=Krux.ns.macmillan.kxjsonp_consent_get_0"></script><script async="" src="controltag.c41d96148598462657526acd569e1aa2"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script type="text/javascript" async="" src="MathJax.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>
        <link rel="stylesheet" media="screen" href="basic-b69f4da053.css">
<link rel="stylesheet" media="print" href="print-7866383638.css">

<link rel="stylesheet" class="js-ctm" href="main-7c1188aaa2.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        <script src="polyfill.js" defer="defer"></script>
        <script src="app-5ac4f41c91.js" defer="defer"></script>
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {
                window.airbrakeErrors.push(error);
            }
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcbioinformatics.biomedcentral.com',
            siteWithPath: 'bmcbioinformatics.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcbioinformatics',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1471-2105-5-113',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
            
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/1471-2105-5-113","articleType":"Software","peerReviewType":"Closed","supplement":null},"contentInfo":{"imprint":"BioMed Central","title":"MUSCLE: a multiple sequence alignment method with reduced time and space complexity","publishedAt":1092873600000,"publishedAtDate":"2004-08-19","author":["Robert C Edgar"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"bmcbioinformatics.biomedcentral.com","volume":"5","issue":"1","title":"BMC Bioinformatics","type":"SERIES","journalID":12859,"gaCode":"UA-62065396-3","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Software","publishingSegment":"BMC-series Biology"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        
        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><script type="text/javascript" src="jquery_002.js"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle a { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle a { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style>
        
    
        
    


        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113">
    


        

        
            
        <meta property="og:url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-5-113">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Bioinformatics">
        <meta property="og:title" content="MUSCLE: a multiple sequence alignment method with reduced time and space complexity">
        <meta property="og:description" content="In a previous paper, we introduced MUSCLE, a new program for creating multiple alignments of protein sequences, giving a brief summary of the algorithm and showing MUSCLE to achieve the highest scores reported to date on four alignment accuracy benchmarks. Here we present a more complete discussion of the algorithm, describing several previously unpublished techniques that improve biological accuracy and / or computational complexity. We introduce a new option, MUSCLE-fast, designed for high-throughput applications. We also describe a new protocol for evaluating objective functions that align two profiles. We compare the speed and accuracy of MUSCLE with CLUSTALW, Progressive POA and the MAFFT script FFTNS1, the fastest previously published program known to the author. Accuracy is measured using four benchmarks: BAliBASE, PREFAB, SABmark and SMART. We test three variants that offer highest accuracy (MUSCLE with default settings), highest speed (MUSCLE-fast), and a carefully chosen compromise between the two (MUSCLE-prog). We find MUSCLE-fast to be the fastest algorithm on all test sets, achieving average alignment accuracy similar to CLUSTALW in times that are typically two to three orders of magnitude less. MUSCLE-fast is able to align 1,000 sequences of average length 282 in 21 seconds on a current desktop computer. MUSCLE offers a range of options that provide improved speed and / or alignment accuracy compared with currently available programs. MUSCLE is freely available at http://www.drive5.com/muscle .">
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12859.jpg">
    
        
    <script type="text/javascript" src="jquery.js"></script><script src="figshare.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_265.js" async=""></script></head>

    <body class="journal journal-fulltext"><div id="MathJax_Message" style="display: none;"></div>
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"></path></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"></circle><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"></path><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"></path></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcbioinformatics/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1471-2105-5-113;kwrd=Global Alignment,Profile Function,Amino Acid Type,Fractional Identity,Multiple Sequence Alignment Program;pmc=L15001,B12050,I23050,L17004,M14018,M14018;"><div id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0" title="3rd party ad content" name="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img src="logo-bmc-white-series-1da6525f4e.svg" alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link" data-track="click" data-track-category="Account" data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" placeholder="Search all BMC articles" type="text">
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--bmc-bioinformatics ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcbioinformatics.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Bioinformatics</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcbioinformatics.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec3" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Implementation">Implementation</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec50" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results">Results</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec53" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusions">Conclusions</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec54" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Availability and requirements">Availability and requirements</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Software</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">MUSCLE: a multiple sequence alignment method with reduced time and space complexity</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Robert&nbsp;C&nbsp;Edgar</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:bob@drive5.com" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Bioinformatics</span></span><span class="ArticleCitation_Year">2004</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">5</strong>:113</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/1471-2105-5-113">https://doi.org/10.1186/1471-2105-5-113</a></p><p class="Copyright">© &nbsp;Edgar; licensee BioMed Central Ltd.&nbsp;2004</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>25&nbsp;March&nbsp;2004</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>19&nbsp;August&nbsp;2004</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">19&nbsp;August&nbsp;2004</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">In a previous paper, we introduced 
MUSCLE, a new program for creating multiple alignments of protein 
sequences, giving a brief summary of the algorithm and showing MUSCLE to
 achieve the highest scores reported to date on four alignment accuracy 
benchmarks. Here we present a more complete discussion of the algorithm,
 describing several previously unpublished techniques that improve 
biological accuracy and / or computational complexity. We introduce a 
new option, MUSCLE-fast, designed for high-throughput applications. We 
also describe a new protocol for evaluating objective functions that 
align two profiles.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">We compare the speed and accuracy of 
MUSCLE with CLUSTALW, Progressive POA and the MAFFT script FFTNS1, the 
fastest previously published program known to the author. Accuracy is 
measured using four benchmarks: BAliBASE, PREFAB, SABmark and SMART. We 
test three variants that offer highest accuracy (MUSCLE with default 
settings), highest speed (MUSCLE-fast), and a carefully chosen 
compromise between the two (MUSCLE-prog). We find MUSCLE-fast to be the 
fastest algorithm on all test sets, achieving average alignment accuracy
 similar to CLUSTALW in times that are typically two to three orders of 
magnitude less. MUSCLE-fast is able to align 1,000 sequences of average 
length 282 in 21 seconds on a current desktop computer.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusions</h3>
                <p class="Para">MUSCLE offers a range of options that 
provide improved speed and / or alignment accuracy compared with 
currently available programs. MUSCLE is freely available at <span class="ExternalRef"><a href="http://www.drive5.com/muscle"><span class="RefSource">http://www.drive5.com/muscle</span></a></span>.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Global Alignment</li><li class="c-keywords__item" itemprop="about">Profile Function</li><li class="c-keywords__item" itemprop="about">Amino Acid Type</li><li class="c-keywords__item" itemprop="about">Fractional Identity</li><li class="c-keywords__item" itemprop="about">Multiple Sequence Alignment Program</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Multiple
 alignments of protein sequences are important in many applications, 
including phylogenetic tree estimation, secondary structure prediction 
and critical residue identification. Many multiple sequence alignment 
(MSA) algorithms have been proposed; for a recent review, see [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span>].
 Two attributes of MSA programs are of primary importance to the user: 
biological accuracy and computational complexity (i.e., time and memory 
requirements). Complexity is of increasing relevance due to the rapid 
growth of sequence databases, which now contain enough representatives 
of larger protein families to exceed the capacity of most current 
programs. Obtaining biologically accurate alignments is also a 
challenge, as the best methods sometimes fail to align readily apparent 
conserved motifs [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].
 We recently introduced MUSCLE, a new MSA program that provides 
significant improvements in both accuracy and speed, giving only a 
summary of the algorithm [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].
 Here, we describe the MUSCLE algorithm more fully and analyze its 
complexity. We introduce a new option designed for high-throughput 
applications, MUSCLE-fast. We also describe a new method for evaluating 
objective functions for profile-profile alignment, the iterated step in 
the MUSCLE algorithm.</p><section id="Sec2" class="Section2 RenderAsSection2">
                <h3 class="Heading">Current methods</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">While
 multiple alignment and phylogenetic tree reconstruction have 
traditionally been considered separately, the most natural formulation 
of the computational problem is to define a model of sequence evolution 
that assigns probabilities to all possible elementary sequence edits and
 then to seek an optimal directed graph in which edges represents edits 
and terminal nodes are the observed sequences. This graph makes the 
history explicit (it can be interpreted as a phylogenetic tree) and 
implies an alignment. No tractable method for finding an optimal graph 
is known for biologically realistic models, and simplification is 
therefore required. A common heuristic is to seek a multiple alignment 
that maximizes the SP score (the summed alignment score of each sequence
 pair), which is NP complete [<span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>]. It can be achieved by dynamic programming with time and space complexity O(<em xmlns="" class="EmphasisTypeItalic">L</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">N</em>
                  </sup>) in the sequence length <em xmlns="" class="EmphasisTypeItalic">L</em> and number of sequences <em xmlns="" class="EmphasisTypeItalic">N</em> [<span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>], and is practical only for very small <em xmlns="" class="EmphasisTypeItalic">N</em>. Stochastic methods such as Gibbs sampling can be used to search for a maximum objective score [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>], but have not been widely adopted. A more popular strategy is the progressive method [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>] (Figure <span class="InternalRef"><a href="#Fig1">1</a></span>),
 which first estimates a phylogenetic tree. A profile (a multiple 
alignment treated as a sequence by regarding each column as a symbol) is
 then constructed for each node in the binary tree. If the node is a 
leaf, the profile is the corresponding sequence; otherwise its profile 
is produced by a pair-wise alignment of the profiles of its child nodes 
(Figure <span class="InternalRef"><a href="#Fig2">2</a></span>). Current
 progressive algorithms are typically practical for up to a few hundred 
sequences on desktop computers, the best-known of which is CLUSTALW [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>]. A variant of the progressive approach is used by T-Coffee [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>],
 which builds a library of both local and global alignments of every 
pair of sequences and uses a library-based score for aligning two 
profiles. On the BAliBASE benchmark [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>],
 T-Coffee achieves the best results reported prior to MUSCLE, but has a 
high time and space complexity that limits the number of sequences it 
can align to typically around one hundred. In our experience, errors in 
progressive alignments can often be attributed to one of the following 
issues: sub-optimal branching order in the tree, scoring parameters that
 are not optimal for a particular set of sequences (especially gap 
penalties), and inappropriate boundary conditions (e.g., seeking a 
global alignment of proteins having different domain organizations). 
Misalignments are sometimes readily apparent, motivating further 
processing (<em xmlns="" class="EmphasisTypeItalic">refinement</em>). One approach is to use a progressive alignment as the initial state of a stochastic search for a maximum objective score (<em xmlns="" class="EmphasisTypeItalic">stochastic refinement</em>).
 Alternatively, pairs of profiles can be extracted from the progressive 
alignment and re-aligned, keeping the results only when an objective 
score is improved (<em xmlns="" class="EmphasisTypeItalic">horizontal refinement</em>) [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>].<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig1_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig1_HTML.jpg" alt="Figure 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig1_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig1_HTML.jpg" alt="Figure 1"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Progressive alignment.</strong>
 Sequences are assigned to the leaves of a binary tree. At each internal
 (i.e., non-leaf) node, the two child profiles are aligned using 
profile-profile alignment (see Figure 2). Indels introduced at each node
 are indicated by shaded background.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig2_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig2_HTML.jpg" alt="Figure 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig2_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig2_HTML.jpg" alt="Figure 2"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 2</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Profile-profile alignment.</strong>
 Two profiles (multiple sequence alignments) X and Y are aligned to each
 other such that columns from X and Y are preserved in the result. 
Columns of indels (gray background) are inserted as needed in order to 
align the columns to each other. The score for aligning a pair of 
columns is determined by the profile function, which should assign a 
high score to pairs of columns containing similar amino acids.</p>
                      </div></figcaption></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec3" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Implementation" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Implementation">Implementation</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Implementation" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The basic strategy used by MUSCLE is similar to that used by PRRP [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>] and MAFFT [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. A progressive alignment is built, to which horizontal refinement is then applied.</p><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Algorithm overview</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">MUSCLE has three stages. At the completion of each stage, a multiple alignment is available and the algorithm can be terminated.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">Stage 1: draft progressive</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The first stage builds a progressive alignment.</p>
                <section id="Sec6" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Similarity measure</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The similarity of each pair of sequences is computed, either using <em xmlns="" class="EmphasisTypeItalic">k</em>-mer counting or by constructing a global alignment of the pair and determining the fractional identity.</p>
                </section>
                <section id="Sec7" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Distance estimate</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A triangular distance matrix is computed from the pair-wise similarities.</p>
                </section>
                <section id="Sec8" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Tree construction</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A tree is constructed from the distance matrix using UPGMA or neighbor-joining, and a root is identified.</p>
                </section>
                <section id="Sec9" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Progressive alignment</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A
 progressive alignment is built by following the branching order of the 
tree, yielding a multiple alignment of all input sequences at the root.</p>
                </section>
              </section><section id="Sec10" class="Section2 RenderAsSection2">
                <h3 class="Heading">Stage 2: improved progressive</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 second stage attempts to improve the tree and builds a new progressive 
alignment according to this tree. This stage may be iterated.</p>
                <section id="Sec11" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Similarity measure</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 similarity of each pair of sequences is computed using fractional 
identity computed from their mutual alignment in the current multiple 
alignment.</p>
                </section>
                <section id="Sec12" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Tree construction</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A tree is constructed by computing a Kimura distance matrix and applying a clustering method to this matrix.</p>
                </section>
                <section id="Sec13" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Tree comparison</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 previous and new trees are compared, identifying the set of internal 
nodes for which the branching order has changed. If Stage 2 has executed
 more than once, and the number of changed nodes has not decreased, the 
process of improving the tree is considered to have converged and 
iteration terminates.</p>
                </section>
                <section id="Sec14" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Progressive alignment</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A
 new progressive alignment is built. The existing alignment is retained 
of each subtree for which the branching order is unchanged; new 
alignments are created for the (possibly empty) set of changed nodes. 
When the alignment at the root is completed, the algorithm may 
terminate, return to step 2.1 or go to Stage 3.</p>
                </section>
              </section><section id="Sec15" class="Section2 RenderAsSection2">
                <h3 class="Heading">Stage 3: refinement</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The third stage performs iterative refinement using a variant of tree-dependent restricted partitioning [<span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>].</p>
                <section id="Sec16" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Choice of bipartition</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">An
 edge is deleted from the tree, dividing the sequences into two disjoint
 subsets (a bipartition). Edges are visiting in order of decreasing 
distance from the root.</p>
                </section>
                <section id="Sec17" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Profile extraction</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 profile (multiple alignment) of each subset is extracted from the 
current multiple alignment. Columns containing no residues (i.e., indels
 only) are discarded.</p>
                </section>
                <section id="Sec18" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Re-alignment</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The two profiles obtained in step 3.2 are re-aligned to each other using profile-profile alignment.</p>
                </section>
                <section id="Sec19" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Accept/reject</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 SP score of the multiple alignment implied by the new profile-profile 
alignment is computed. If the score increases, the new alignment is 
retained, otherwise it is discarded. If all edges have been visited 
without a change being retained, or if a user-defined maximum number of 
iterations has been reached, the algorithm is terminated, otherwise it 
returns to step 3.1. Visiting edges in order of decreasing distance from
 the root has the effect of first re-aligning individual sequences, then
 closely related groups</p>
                </section>
              </section><section id="Sec20" class="Section2 RenderAsSection2">
                <h3 class="Heading">Algorithm elements</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 the following, we describe the elements of the MUSCLE algorithm. In 
several cases, alternative versions of these elements were implemented 
in order to investigate their relative performance and to offer 
different trade-offs between accuracy, speed and memory use. Most of 
these alternatives are made available to the user via command-line 
options. Four benchmark datasets have been used to evaluate options and 
parameters in MUSCLE: BAliBASE [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>, <span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span>], SABmark [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>], SMART [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span><span>, <span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>, </span><span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>] and our own benchmark, PREFAB [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].</p>
              </section><section id="Sec21" class="Section2 RenderAsSection2">
                <h3 class="Heading">Objective score</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 its refinement stage, MUSCLE seeks to maximize an objective score, i.e.
 a function that maps a multiple sequence alignment to a real number 
which is designed to give larger values to better alignments. MUSCLE 
uses the <em xmlns="" class="EmphasisTypeItalic">sum-of-pairs</em> (SP) 
score, defined to be the sum over pairs of sequences of their alignment 
scores. The alignment score of a pair of sequences is computed as the 
sum of substitution matrix scores for each aligned pair of residues, 
plus gap penalties. Gaps require special consideration (Figure <span class="InternalRef"><a href="#Fig3">3</a></span>). We use the term <em xmlns="" class="EmphasisTypeItalic">indel</em> for the symbol that indicates a gap in a column (typically a dash '-'), reserving the term <em xmlns="" class="EmphasisTypeItalic">gap</em>
 for a maximal contiguous series of indels. The gap penalty contribution
 to SP for a pair of sequences is computed by discarding all columns in 
which both sequences have an indel, then applying an affine penalty <em xmlns="" class="EmphasisTypeItalic">g</em> + <em xmlns="" class="EmphasisTypeItalic">λe</em> for each remaining gap where <em xmlns="" class="EmphasisTypeItalic">g</em> is the per-gap penalty, <em xmlns="" class="EmphasisTypeItalic">λ</em> is the gap length (number of indels in the gap), and <em xmlns="" class="EmphasisTypeItalic">e</em> is the gap-length penalty (sometimes called the extension penalty).<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig3_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig3_HTML.jpg" alt="Figure 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig3_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig3_HTML.jpg" alt="Figure 3"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 3</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Gap penalties in the SP score</strong> This figure shows a multiple alignment of three sequences <em class="EmphasisTypeItalic">s</em>, <em class="EmphasisTypeItalic">t</em> and <em class="EmphasisTypeItalic">u</em>.
 The SP score is the sum over all pairs of sequences of their pairwise 
alignment score. The contribution to the SP score from the pair (<em class="EmphasisTypeItalic">s</em>, <em class="EmphasisTypeItalic">t</em>)
 is computed by discarding columns in which both sequences have indels 
(arrows). Such indels are said to be external with respect to the pair. 
Gaps in the remaining columns (gray background) are assessed affine 
penalties <em class="EmphasisTypeItalic">g</em> + <em class="EmphasisTypeItalic">λe</em> where <em class="EmphasisTypeItalic">g</em> is the per-gap penalty, <em class="EmphasisTypeItalic">λ</em> is the gap length, and <em class="EmphasisTypeItalic">e</em> is the gap extension penalty.</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec22" class="Section2 RenderAsSection2">
                <h3 class="Heading">Progressive alignment</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Progressive
 alignment requires a rooted binary tree in which each sequence is 
assigned to a leaf. The tree is created by clustering a triangular 
matrix containing a distance measure for each pair of sequences. The 
branching order of the tree is followed in postfix order (i.e., children
 are visited before their parent). At each internal node, 
profile-profile alignment is used to align the existing alignments of 
the two child subtrees, and the new alignment is assigned to that node. A
 multiple alignment of all input sequences is produced at the root node 
(Figure <span class="InternalRef"><a href="#Fig1">1</a></span>).</p>
              </section><section id="Sec23" class="Section2 RenderAsSection2">
                <h3 class="Heading">Similarity measures</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We use the term <em xmlns="" class="EmphasisTypeItalic">similarity</em>
 for a measure on a pair of sequences that indicates their degree of 
evolutionary divergence (the sequences are assumed to be related). 
MUSCLE uses two types of similarity measure: the fractional identity <em xmlns="" class="EmphasisTypeItalic">D</em> computed from a global alignment of the two sequences, and measures obtained by <em xmlns="" class="EmphasisTypeItalic">k</em>-mer counting. A <em xmlns="" class="EmphasisTypeItalic">k</em>-mer is a contiguous subsequence of length <em xmlns="" class="EmphasisTypeItalic">k</em>, also known as a word or <em xmlns="" class="EmphasisTypeItalic">k</em>-tuple. Related sequences tend to have more <em xmlns="" class="EmphasisTypeItalic">k</em>-mers in common than expected by chance, provided that <em xmlns="" class="EmphasisTypeItalic">k</em> is not too large and the divergence is not too great. Many sequence comparison methods based on <em xmlns="" class="EmphasisTypeItalic">k</em>-mer counting have been proposed in the literature; for a review, see [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>]. The primary motivation for these measures is improved speed as no alignment is required. MAFFT uses <em xmlns="" class="EmphasisTypeItalic">k</em>-mer
 counting in a compressed alphabet (i.e., an alphabet in which symbols 
denote classes that may contain two or more residue types) to compute 
its initial distance measure. The alphabet used in MAFFT is taken from [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>],
 and is one of the options implemented in MUSCLE. Trivially, identity is
 higher or equal in a compressed alphabet; it cannot be reduced. If the 
alphabet is chosen such that there are high probabilities of intra-class
 substitution and low probabilities of inter-class substitution, then we
 might expect that detectable identity (and hence the number of 
conserved <em xmlns="" class="EmphasisTypeItalic">k</em>-mers) could be 
usefully extended to greater evolutionary distances while limiting the 
increase in matches due to chance. We have previously shown [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] that <em xmlns="" class="EmphasisTypeItalic">k</em>-mer
 similarities correlate well with fractional identity, although we 
failed to find evidence that compressed alphabets have superior 
performance to the standard alphabet at lower identities. We define the 
following similarity measure between sequences X and Y:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">F</em> = Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">τ</em>
                  </sub>min [<em xmlns="" class="EmphasisTypeItalic">n</em><sub>X</sub>(<em xmlns="" class="EmphasisTypeItalic">τ</em>), <em xmlns="" class="EmphasisTypeItalic">n</em><sub>Y</sub>(<em xmlns="" class="EmphasisTypeItalic">τ</em>) ] / [min (<em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub>, <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub>) - <em xmlns="" class="EmphasisTypeItalic">k</em> + 1 ]. &nbsp;&nbsp;&nbsp; (1)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Here <em xmlns="" class="EmphasisTypeItalic">τ</em> is a <em xmlns="" class="EmphasisTypeItalic">k</em>-mer, <em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub>, <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub> are the sequence lengths, and <em xmlns="" class="EmphasisTypeItalic">n</em><sub>X</sub>(<em xmlns="" class="EmphasisTypeItalic">τ</em>) and <em xmlns="" class="EmphasisTypeItalic">n</em><sub>Y</sub>(<em xmlns="" class="EmphasisTypeItalic">τ</em>) are the number of times <em xmlns="" class="EmphasisTypeItalic">τ</em>
 occurs in X and Y respectively. This definition can be motivated by 
considering an alignment of X to Y and defining the similarity to be the
 fraction of <em xmlns="" class="EmphasisTypeItalic">k</em>-mers that are conserved between the two sequences. The denominator of <em xmlns="" class="EmphasisTypeItalic">F</em> is the maximum number of <em xmlns="" class="EmphasisTypeItalic">k</em>-mers that could be aligned. Note that if a given <em xmlns="" class="EmphasisTypeItalic">k</em>-mer
 occurs more often in one sequence than the other, the excess cannot be 
conserved, hence the minimum in the numerator. The definition of <em xmlns="" class="EmphasisTypeItalic">F</em> is an approximation in which it is assumed that (after correcting for excesses) common <em xmlns="" class="EmphasisTypeItalic">k</em>-mers are always alignable to each other. MUSCLE also implements a binary approximation <em xmlns="" class="EmphasisTypeItalic">F</em><sup>Binary</sup>, so-called because it reduces the <em xmlns="" class="EmphasisTypeItalic">k</em>-mer count to a present / absent bit:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">F</em><sup>Binary</sup> = Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">τ</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">δ</em><sub>XY</sub>(<em xmlns="" class="EmphasisTypeItalic">τ</em>) / [min (<em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub>, <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub>) - <em xmlns="" class="EmphasisTypeItalic">k</em> + 1 ]. &nbsp;&nbsp;&nbsp; (2)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Here, <em xmlns="" class="EmphasisTypeItalic">δ</em><sub>XY</sub>(<em xmlns="" class="EmphasisTypeItalic">τ</em>) is 1 if <em xmlns="" class="EmphasisTypeItalic">τ</em> is present in both sequences, 0 otherwise. As multiple instances of a given <em xmlns="" class="EmphasisTypeItalic">k</em>-mer in one sequence are relatively rare, this is often a good approximation to <em xmlns="" class="EmphasisTypeItalic">F</em>.
 The binary approximation enables a significant speed improvement as the
 size of the count vector for a given sequence can be reduced by an 
order of magnitude. This allows the count vector for every sequence to 
be retained in memory, and pairs of vectors to be compared efficiently 
using bit-wise instructions. When using an integer count, there may be 
insufficient memory to store all count vectors, making it necessary to 
re-compute counts several times for a given sequence.</p>
              </section><section id="Sec24" class="Section2 RenderAsSection2">
                <h3 class="Heading">Distance measures</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Given
 a similarity value, we wish to estimate an additive distance measure. 
An additive measure distance measure d(A, B) between two sequences A and
 B satisfies d(A, B) = d(A, C) + d(C, B) for any third sequence C, 
assuming that A, B and C are all related. Ideal but generally unknowable
 is the <em xmlns="" class="EmphasisTypeItalic">mutation distance</em>, 
i.e. the number of mutations that occurred on the historical path 
between the sequences. The historical path through the phylogenetic tree
 extends from one sequence to the other via their most recent common 
ancestor. The mutation distance is trivially additive. The fractional 
identity <em xmlns="" class="EmphasisTypeItalic">D</em> is often used as a similarity measure; for closely related sequences 1 - <em xmlns="" class="EmphasisTypeItalic">D</em>
 is a good approximation to a mutation distance (it is exact assuming 
substitution at a single site to be the only allowed type of mutation 
and that no position mutates more than once). As sequences diverge, 
there is an increasing probability of multiple mutations at a single 
site. To correct for this, we use the following distance estimate [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span>]:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">d</em><sub>Kimura</sub> = -log<sub>e</sub> (1 - <em xmlns="" class="EmphasisTypeItalic">D</em> - <em xmlns="" class="EmphasisTypeItalic">D</em><sup>2</sup>/5) &nbsp;&nbsp;&nbsp; (3)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For <em xmlns="" class="EmphasisTypeItalic">D</em> ≤ 0.25 we use a lookup table taken from the CLUSTALW source code. For <em xmlns="" class="EmphasisTypeItalic">k</em>-mer measures, we use:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">d</em><sub>kmer</sub> = 1 - <em xmlns="" class="EmphasisTypeItalic">F</em>. &nbsp;&nbsp;&nbsp; (4)</p>
              </section><section id="Sec25" class="Section2 RenderAsSection2">
                <h3 class="Heading">Tree construction</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Given a distance matrix, a binary tree is constructed by clustering. Two methods are implemented: neighbor-joining [<span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>], and UPGMA [<span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>].
 MUSCLE implements three variants of UPGMA that differ in their 
assignment of distances to a new cluster. Consider two clusters 
(subtrees) <em xmlns="" class="EmphasisTypeItalic">L</em> and <em xmlns="" class="EmphasisTypeItalic">R</em> to be merged into a new cluster <em xmlns="" class="EmphasisTypeItalic">P</em>, which becomes the parent of <em xmlns="" class="EmphasisTypeItalic">L</em> and <em xmlns="" class="EmphasisTypeItalic">R</em> in the binary tree. Average linkage assigns this distance to a third cluster <em xmlns="" class="EmphasisTypeItalic">C</em>:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">d</em><sup>Avg</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">PC</em>
                  </sub>= (<em xmlns="" class="EmphasisTypeItalic">d</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">LC</em>
                  </sub>+ <em xmlns="" class="EmphasisTypeItalic">d</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">RC</em>
                  </sub>)/2. &nbsp;&nbsp;&nbsp; (5)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We can take the minimum rather than the average:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">d</em><sup>Min</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">PC</em>
                  </sub>= min [<em xmlns="" class="EmphasisTypeItalic">d</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">LC</em>
                  </sub>, <em xmlns="" class="EmphasisTypeItalic">d</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">RC</em>
                  </sub>]. &nbsp;&nbsp;&nbsp; (6)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Following MAFFT, we also implemented a weighted mixture of minimum and average linkage:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">d</em><sup>Mix</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">PC</em>
                  </sub>= (1 - <em xmlns="" class="EmphasisTypeItalic">s</em>) <em xmlns="" class="EmphasisTypeItalic">d</em><sup>Min</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">PC</em>
                  </sub>+ <em xmlns="" class="EmphasisTypeItalic">s d</em><sup>Avg</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">PC</em>
                  </sub>, &nbsp;&nbsp;&nbsp; (7)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">where <em xmlns="" class="EmphasisTypeItalic">s</em>
 is a parameter set to 0.1 by default. Clustering produces a pseudo-root
 (the last node created). We implemented two other methods for 
determining a root: minimizing the average branch weight [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>], as used by CLUSTALW, and locating the root at the center of the longest span.</p>
              </section><section id="Sec26" class="Section2 RenderAsSection2">
                <h3 class="Heading">Sequence weighting</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Conventional
 wisdom holds that sequences should be weighted to correct for the 
effects of biased sampling from a family of related proteins; however, 
there is no consensus on how such weights should be computed. MUSCLE 
implements the following sequence weighting schemes: none (all sequences
 have equal weight), Henikoff [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>], PSI-BLAST [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>] (a variant of Henikoff), CLUSTALW's, GSC [<span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>], and the three-way method [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>].
 We found the use of weighting to give a small improvement in benchmark 
accuracy results, e.g. approximately 1% on BAliBASE, but saw little 
difference between the alternative schemes. The CLUSTALW method enables a
 significant reduction in complexity (described later), and is therefore
 the default choice.</p>
              </section><section id="Sec27" class="Section2 RenderAsSection2">
                <h3 class="Heading">Profile functions</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 order to apply pair-wise alignment methods to profiles, a scoring 
function must be defined for a pair of profile positions, i.e. a pair of
 multiple alignment columns. This function is the profile analog of a 
substitution matrix; see for example [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>]. We use the following notation. Let <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em> be amino acid types, <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>the background probability of <em xmlns="" class="EmphasisTypeItalic">i</em>, <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>the joint probability of <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em> being aligned to each other, <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>the substitution matrix score, <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>the observed frequency of <em xmlns="" class="EmphasisTypeItalic">i</em> in column <em xmlns="" class="EmphasisTypeItalic">x</em> of the first profile, <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>G</sub> the observed frequency of gaps in that column, and <em xmlns="" class="EmphasisTypeItalic">α</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>the estimated probability of observing <em xmlns="" class="EmphasisTypeItalic">i</em> in position <em xmlns="" class="EmphasisTypeItalic">x</em> in the family. (Similarly for position <em xmlns="" class="EmphasisTypeItalic">y</em> in the second profile). Estimated probabilities <em xmlns="" class="EmphasisTypeItalic">α</em> are derived from the observed frequencies <em xmlns="" class="EmphasisTypeItalic">f</em>, typically by adding heuristic pseudo-counts or by using Bayesian methods such as Dirichlet mixture priors [<span class="CitationRef"><a href="#CR31" data-toggle="citation">31</a></span>].
 A commonly used profile function is the sequence-weighted sum of 
substitution matrix scores for each pair of letters, selecting one from 
each column (PSP, for profile SP):</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">PSP<sup>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sup>= Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>. &nbsp;&nbsp;&nbsp; (8)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Note that <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>= log (<em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>/ <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub>) [<span class="CitationRef"><a href="#CR32" data-toggle="citation">32</a></span>], so</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">PSP<sup>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sup>= Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub>log (<em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>/ <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub>). &nbsp;&nbsp;&nbsp; (9)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">PSP
 is the function used by CLUSTALW and MAFFT. It is a natural choice when
 attempting to maximize the SP objective score: if gap penalties are 
neglected, maximizing PSP maximizes SP under the constraint that columns
 in each profile are preserved. (This follows from the observation that 
the contribution to SP from a pair of sequences in the same profile is 
the same for all alignments allowed under the constraint). MUSCLE 
implements PSP functions based on the 200 PAM matrix of [<span class="CitationRef"><a href="#CR33" data-toggle="citation">33</a></span>] and the 240 PAM VTML matrix [<span class="CitationRef"><a href="#CR34" data-toggle="citation">34</a></span>]. In addition to PSP, MUSCLE implements a function we call the <em xmlns="" class="EmphasisTypeItalic">log-expectation</em> (LE) score. LE is a modified version of the log-average (LA) profile function that was proposed on theoretical grounds [<span class="CitationRef"><a href="#CR35" data-toggle="citation">35</a></span>]:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">LA<sup>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sup>= log Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">α</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">α</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>/ <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub>. &nbsp;&nbsp;&nbsp; (10)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">LE is defined as follows:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">LE<sup>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sup>= (1 - <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">G</em>
                  </sub>) (1 - <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>G</sub>) log Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>Σ <sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>/ <em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">p</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub>. &nbsp;&nbsp;&nbsp; (11)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The MUSCLE LE function uses probabilities computed from VTML 240. Note that estimated probabilities <em xmlns="" class="EmphasisTypeItalic">α</em> in LA are replaced by observed frequencies <em xmlns="" class="EmphasisTypeItalic">f</em> in LE. The factor (1 - <em xmlns="" class="EmphasisTypeItalic">f</em><sub>G</sub>) is the <em xmlns="" class="EmphasisTypeItalic">occupancy</em> of a column. Frequencies <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>must be normalized to sum to one if indels are 
present (otherwise the logarithm becomes increasingly negative with 
increasing numbers of gaps even when aligning conserved or similar 
residues). The occupancy factors are introduced to encourage more highly
 occupied columns (i.e., those with fewer gaps) to align, and are found 
to significantly improve accuracy. We avoid these complications in the 
PSP score by computing frequencies in a 21-letter alphabet (amino acids +
 indel), and by defining the substitution score of an amino acid to an 
indel to be zero. This has the desired effect of down-weighting column 
pairs with low occupancies, and can also be motivated by consideration 
of the SP function. If gap penalties are ignored, then this definition 
of PSP preserves the optimization of SP under the fixed-column 
constraint by correctly accounting for the reduced number of residue 
pairs in columns containing gaps.</p>
              </section><section id="Sec28" class="Section2 RenderAsSection2">
                <h3 class="Heading">Gap penalties</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We call the first indel in a gap its <em xmlns="" class="EmphasisTypeItalic">gap-open</em>; the last its <em xmlns="" class="EmphasisTypeItalic">gap-close</em>. Consider an alignment of two profiles X and Y, and a gap of length <em xmlns="" class="EmphasisTypeItalic">λ</em> in X in which the gap-open is aligned to position <em xmlns="" class="EmphasisTypeItalic">y</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">o</em>
                  </sub>in Y and the gap-close to <em xmlns="" class="EmphasisTypeItalic">y</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">c</em>
                  </sub>. The penalty for this gap is <em xmlns="" class="EmphasisTypeItalic">b</em>(<em xmlns="" class="EmphasisTypeItalic">y</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">o</em>
                  </sub>) + <em xmlns="" class="EmphasisTypeItalic">t</em>(<em xmlns="" class="EmphasisTypeItalic">y</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">c</em>
                  </sub>) + <em xmlns="" class="EmphasisTypeItalic">λe</em>, where <em xmlns="" class="EmphasisTypeItalic">b</em> and <em xmlns="" class="EmphasisTypeItalic">t</em> are costs for opening and closing a gap that vary according to the position in Y, and <em xmlns="" class="EmphasisTypeItalic">e</em>
 is a length cost (sometimes called a gap extension penalty) that does 
not vary by position. A fixed length cost allows a minor optimization of
 the scoring scheme [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>]. Consider a global alignment of sequences X and Y having lengths <em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub> and <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub>. If a constant <em xmlns="" class="EmphasisTypeItalic">δ</em> (the <em xmlns="" class="EmphasisTypeItalic">center</em>) is added to each substitution matrix score and <em xmlns="" class="EmphasisTypeItalic">δ</em>/2 is added for each gapped position, this adds the constant value <em xmlns="" class="EmphasisTypeItalic">δ</em>(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub> + <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub>)/2
 to the score of any possible alignment, and the set of optimal 
alignments is therefore unchanged. Given a scoring scheme with 
substitution matrix <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>and extension penalty <em xmlns="" class="EmphasisTypeItalic">e</em>, we can thus choose <em xmlns="" class="EmphasisTypeItalic">δ</em>/2 = <em xmlns="" class="EmphasisTypeItalic">e</em> and instead use <em xmlns="" class="EmphasisTypeItalic">S'</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>= <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">ij</em>
                  </sub>+ 2<em xmlns="" class="EmphasisTypeItalic">e</em> and <em xmlns="" class="EmphasisTypeItalic">e'</em> = 0 to obtain the same alignment. The constant 2<em xmlns="" class="EmphasisTypeItalic">e</em>
 can be added to the substitution matrix at compile time, and no 
explicit extension penalty is then needed in the recursion relations. 
MUSCLE uses this optimization for the PSP function, but not for LE 
(where the center must be added at execution time after taking the 
logarithm). Let <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">o</em>
                  </sub>be the number of gap-opens in column <em xmlns="" class="EmphasisTypeItalic">y</em> in Y and <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">c</em>
                  </sub>be the number of gap-closes in column <em xmlns="" class="EmphasisTypeItalic">y</em>. MUSCLE computes <em xmlns="" class="EmphasisTypeItalic">b</em> and <em xmlns="" class="EmphasisTypeItalic">t</em> as follows (Figure <span class="InternalRef"><a href="#Fig4">4</a></span>):<figure xmlns="" class="Figure" id="Fig4"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig4_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig4_HTML.jpg" alt="Figure 4" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig4_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig4_HTML.jpg" alt="Figure 4"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 4</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Position-specific gap penalties.</strong> An alignment of two profiles X and Y. Gaps in sequences <em class="EmphasisTypeItalic">t</em> and <em class="EmphasisTypeItalic">u</em> are embedded in X. Y contains a single sequence <em class="EmphasisTypeItalic">w</em>. The gap in <em class="EmphasisTypeItalic">w</em>
 (gray background) is inserted to align the profiles and is not part of 
Y. Consider the SP score for this alignment. We need not consider pairs 
of sequences in X as their scores are unchanged under all possible 
alignments of X to Y, leaving the inter-profile pairs (<em class="EmphasisTypeItalic">s</em>, <em class="EmphasisTypeItalic">w</em>), (<em class="EmphasisTypeItalic">t</em>, <em class="EmphasisTypeItalic">w</em>), (<em class="EmphasisTypeItalic">u</em>, <em class="EmphasisTypeItalic">w</em>) and (<em class="EmphasisTypeItalic">v</em>, <em class="EmphasisTypeItalic">w</em>). Note that there is no gap penalty for the pairs (<em class="EmphasisTypeItalic">u</em>, <em class="EmphasisTypeItalic">w</em>) and (<em class="EmphasisTypeItalic">v</em>, <em class="EmphasisTypeItalic">w</em>) as these pairs do not have gaps relative to each other. The remaining pairs (<em class="EmphasisTypeItalic">t</em>, <em class="EmphasisTypeItalic">w</em>) and (<em class="EmphasisTypeItalic">u</em>, <em class="EmphasisTypeItalic">w</em>) are assessed a penalty <em class="EmphasisTypeItalic">g</em> + 3<em class="EmphasisTypeItalic">e</em>
 for the gap in Y. The total over all pairs of open or close penalties 
due to a gap in Y is thus reduced in proportion to the fraction of 
sequences in X having a gap with the same open or close position. We 
incorporate this into the PSP score by using position-specific gap 
penalties <em class="EmphasisTypeItalic">b</em>(<em class="EmphasisTypeItalic">x</em>) and <em class="EmphasisTypeItalic">t</em>(<em class="EmphasisTypeItalic">x</em>). For example, <em class="EmphasisTypeItalic">b</em>(<em class="EmphasisTypeItalic">x</em>)
 in column 4 of X is half the default value because half of the 
sequences in X open a gap in that column. Note that there is no open 
penalty at the N-terminal and no close penalty at the C-terminal. This 
causes terminal gaps to receive half the penalty of internal gaps.</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">b</em>(<em xmlns="" class="EmphasisTypeItalic">y</em>) = <em xmlns="" class="EmphasisTypeItalic">g</em>/2 (1 - <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">o</em>
                  </sub>) (1 + <em xmlns="" class="EmphasisTypeItalic">h</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">w</em>
                  </sub>(<em xmlns="" class="EmphasisTypeItalic">y</em>)<em xmlns="" class="EmphasisTypeItalic">H</em>), &nbsp;&nbsp;&nbsp; (12)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">t</em>(<em xmlns="" class="EmphasisTypeItalic">y</em>) = <em xmlns="" class="EmphasisTypeItalic">g</em>/2 (1 - <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">c</em>
                  </sub>) (1 + <em xmlns="" class="EmphasisTypeItalic">h</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">w</em>
                  </sub>(<em xmlns="" class="EmphasisTypeItalic">y</em>)<em xmlns="" class="EmphasisTypeItalic">H</em>). &nbsp;&nbsp;&nbsp; (13)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Here, <em xmlns="" class="EmphasisTypeItalic">g</em> is a parameter that can be considered a default per-gap penalty, <em xmlns="" class="EmphasisTypeItalic">h</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">w</em>
                  </sub>(<em xmlns="" class="EmphasisTypeItalic">y</em>) is 1 if <em xmlns="" class="EmphasisTypeItalic">y</em> falls within a window of <em xmlns="" class="EmphasisTypeItalic">w</em> consecutive hydrophobic residues or zero otherwise, and <em xmlns="" class="EmphasisTypeItalic">H</em> is a tunable parameter. By default, <em xmlns="" class="EmphasisTypeItalic">w</em> = 5, <em xmlns="" class="EmphasisTypeItalic">H</em> = 1.2. The factor <em xmlns="" class="EmphasisTypeItalic">g</em>/2 (1 - <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">o</em>
                  </sub>) is motivated by considering the SP score of the alignment. The gap penalty contribution to SP for a pair of sequences (<em xmlns="" class="EmphasisTypeItalic">A</em> <span class="stix">∈</span> Y, <em xmlns="" class="EmphasisTypeItalic">B</em> <span class="stix">∈</span> X) is computed by discarding all columns in which both sequences have an indel, then applying an affine penalty <em xmlns="" class="EmphasisTypeItalic">g</em> + <em xmlns="" class="EmphasisTypeItalic">λe</em> for each remaining gap. It is convenient here to consider that half of the per-gap penalty <em xmlns="" class="EmphasisTypeItalic">g</em> is applied to the open position and half to the close position. Suppose a gap <em xmlns="" class="EmphasisTypeItalic">G</em> is inserted into X such that the gap-open is aligned to position <em xmlns="" class="EmphasisTypeItalic">y</em> in Y. If a sequence <em xmlns="" class="EmphasisTypeItalic">s</em> <span class="stix">∈</span> Y has a gap-open at <em xmlns="" class="EmphasisTypeItalic">y</em>, then the SP score includes no open penalty for <em xmlns="" class="EmphasisTypeItalic">G</em> induced by any pair (<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">t</em>) : <em xmlns="" class="EmphasisTypeItalic">t</em> <span class="stix">∈</span> X. The multiplier (1 - <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">o</em>
                  </sub>) therefore corrects the gap-open contribution 
to the SP score due to pre-existing gaps in Y. (It should be noted that 
even with this correction, there are other issues related to gaps and 
PSP still does not exactly optimize SP under the fixed-column 
constraint). The increased penalty in hydrophobic windows is designed to
 discourage gaps in buried core regions where insertions and deletions 
are less frequent. Note that MUSCLE treats open and close positions 
symmetrically, in contrast to CLUSTALW, which treats the open position 
specially and may therefore tend to produce, in word processing terms, 
left-aligned gaps with a ragged right margin.</p>
              </section><section id="Sec29" class="Section2 RenderAsSection2">
                <h3 class="Heading">Terminal gaps</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A <em xmlns="" class="EmphasisTypeItalic">terminal gap</em>
 is one that opens at the N-terminal position of the sequence to which 
it is aligned or closes at the C-terminal; as opposed to an <em xmlns="" class="EmphasisTypeItalic">internal gap</em>. It has been suggested [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>, <span class="CitationRef"><a href="#CR36" data-toggle="citation">36</a></span>]
 that global methods have intrinsic difficulties with long deletions or 
insertions. We believe that these difficulties are often due to the 
choice of penalties for terminal gaps. CLUSTALW, which charges no 
penalty for terminal gaps, tends to fail to open a needed internal gap 
and thus fail to align terminal motifs; MAFFT, which charges the same 
penalty for terminal and internal gaps, sometimes aligns small numbers 
of residues to a terminal by inserting an unnatural internal gap. By 
default, MUSCLE penalizes terminal gaps with half the penalty of an 
internal gap. This is done by setting <em xmlns="" class="EmphasisTypeItalic">b</em>(1), the open penalty at the C-terminal, and <em xmlns="" class="EmphasisTypeItalic">t</em>(<em xmlns="" class="EmphasisTypeItalic">L</em>), the close penalty at the N-terminal, to zero (Figure <span class="InternalRef"><a href="#Fig4">4</a></span>).
 The option of always applying full penalties, as in MAFFT, is also 
provided. We found that the compromise of a half penalty for terminal 
gaps gave good results for a wide range of input data, but that further 
improvements could sometimes by achieved by the following technique. If 
the length ratio of the two profiles to be aligned exceeds a threshold 
(by default, 20%), then MUSCLE constructs four different alignments in 
which gaps at both, one or neither terminals are fully penalized. A <em xmlns="" class="EmphasisTypeItalic">conservation score</em>
 is defined by subtracting all gap penalties (both internal and 
terminal) from the alignment score, leaving a sum over profile functions
 only. The alignment with the highest conservation score is used.</p>
              </section><section id="Sec30" class="Section2 RenderAsSection2">
                <h3 class="Heading">Tree comparison</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 progressive alignment, two subtrees will produce identical alignments 
if they have the same set of sequences at their leaves and the same 
branching orders (topologies). We exploit this observation to optimize 
the progressive alignment in Stage 2 of MUSCLE, which begins by 
constructing a new tree. Unchanged subtrees are identified, and their 
alignments are retained (Figure <span class="InternalRef"><a href="#Fig5">5</a></span>).
 A progressive alignment of the changed subtrees is constructed, 
producing the same alignment at the root that would be obtained starting
 from the leaves. Tree comparison is performed by the following 
algorithm. Consider two trees <em xmlns="" class="EmphasisTypeItalic">A</em> and <em xmlns="" class="EmphasisTypeItalic">B</em> with identical sets of <em xmlns="" class="EmphasisTypeItalic">N</em> leaves. Leaves are identified by consecutive integers (<em xmlns="" class="EmphasisTypeItalic">ids</em>) 1 through <em xmlns="" class="EmphasisTypeItalic">N</em>.
 Call a pair of nodes, one from each tree, equivalent if they are the 
same leaf or they are internal nodes and their children are equivalent. 
The left/right position of a child is not considered; in other words, 
subtree rotations are allowed (because they do not change the results of
 a progressive alignment). Traverse <em xmlns="" class="EmphasisTypeItalic">A</em> in prefix order (children before their parent), assigning internal nodes ids <em xmlns="" class="EmphasisTypeItalic">N</em> + 1 through 2<em xmlns="" class="EmphasisTypeItalic">N</em> in the order visited. When visiting an internal node <em xmlns="" class="EmphasisTypeItalic">P</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sup>, take the ids of its two child nodes <em xmlns="" class="EmphasisTypeItalic">L</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sup>and <em xmlns="" class="EmphasisTypeItalic">R</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sup>and use them as indexes into a lookup table pointing to nodes in <em xmlns="" class="EmphasisTypeItalic">B</em>. If (a) <em xmlns="" class="EmphasisTypeItalic">L</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sup>is equivalent to a node <em xmlns="" class="EmphasisTypeItalic">L</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sup>in <em xmlns="" class="EmphasisTypeItalic">B</em> and <em xmlns="" class="EmphasisTypeItalic">R</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sup>is equivalent to a node <em xmlns="" class="EmphasisTypeItalic">R</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sup>, and (b) <em xmlns="" class="EmphasisTypeItalic">L</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sup>and <em xmlns="" class="EmphasisTypeItalic">R</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sup>have the same parent <em xmlns="" class="EmphasisTypeItalic">P</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sup>, then assign <em xmlns="" class="EmphasisTypeItalic">P</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">B</em>
                  </sup>the same id as <em xmlns="" class="EmphasisTypeItalic">P</em><sup>
                    <em xmlns="" class="EmphasisTypeItalic">A</em>
                  </sup>, to which it is equivalent. When the traversal is complete, a node <em xmlns="" class="EmphasisTypeItalic">b</em> in <em xmlns="" class="EmphasisTypeItalic">B</em> is equivalent to some node in <em xmlns="" class="EmphasisTypeItalic">A</em> if and only if <em xmlns="" class="EmphasisTypeItalic">b</em> has an id. This procedure is O(<em xmlns="" class="EmphasisTypeItalic">N</em>) time and space.<figure xmlns="" class="Figure" id="Fig5"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig5_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig5_HTML.jpg" alt="Figure 5" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig5_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig5_HTML.jpg" alt="Figure 5"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 5</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Tree comparison.</strong>
 Two trees are compared in order to identify those nodes that have the 
same branching orders within subtree rotation (white). If a progressive 
alignment has been created using to the old tree, then alignments at 
these nodes can be retained as the same result would be produced at 
those nodes by the new tree. New alignments are needed at the changed 
(black) nodes only.</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec31" class="Section2 RenderAsSection2">
                <h3 class="Heading">Defaults, optimizations and complexity analysis</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We now discuss the default choices of algorithm elements in the MUSCLE program and analyze their complexity.</p>
              </section><section id="Sec32" class="Section2 RenderAsSection2">
                <h3 class="Heading">Complexity of CLUSTALW</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">It
 is instructive to consider the complexity of CLUSTALW. This is of 
intrinsic interest as CLUSTALW is currently the most widely used MSA 
program and, to the best of our knowledge, its complexity has not 
previously been stated correctly in the literature. It is also useful as
 a baseline for motivating some of the optimizations used in MUSCLE. The
 CLUSTALW algorithm can be described by the same steps as Stage 1 above.
 The similarity measure is the fractional identity computed from a 
global alignment, clustering is done by neighbor-joining. Global 
alignment of a pair of sequences or profiles is computed using the 
Myers-Miller linear space algorithm [<span class="CitationRef"><a href="#CR37" data-toggle="citation">37</a></span>] which is O(<em xmlns="" class="EmphasisTypeItalic">L</em>) space and O(<em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup>) time in the typical sequence length <em xmlns="" class="EmphasisTypeItalic">L</em>. Given <em xmlns="" class="EmphasisTypeItalic">N</em> sequences and thus <em xmlns="" class="EmphasisTypeItalic">N</em>(<em xmlns="" class="EmphasisTypeItalic">N</em> - 1)/2 = O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup>) pairs, it is therefore O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup><em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup>) time and O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup> + <em xmlns="" class="EmphasisTypeItalic">L</em>) space to construct the distance matrix. The neighbor-joining implementation is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup>) space and O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>4</sup>) time, at least up to CLUSTALW 1.82, although O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>3</sup>) time is possible; see e.g. [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]. A single iteration of progressive alignment computes a profile of each subtree from its multiple alignment, which is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sub>P</sub><em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) time and space in the number of sequences in the profile <em xmlns="" class="EmphasisTypeItalic">N</em><sub>P</sub> and the profile length <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>, then uses Myers-Miller to align the profiles in O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) space and O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup>) time. There are <em xmlns="" class="EmphasisTypeItalic">N</em> - 1 internal nodes in a rooted binary tree and hence O(<em xmlns="" class="EmphasisTypeItalic">N</em>) iterations. It is often assumed that <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> is O(<em xmlns="" class="EmphasisTypeItalic">L</em>),
 i.e. that O(0) gaps are introduced in each iteration. However, we often
 observe the alignment length to grow approximately linearly, i.e. that 
O(1) gaps are added per iteration. For example, taking the average over 
all iterations in all alignments in version 3 of the PREFAB benchmark, 
Stage 1 of MUSCLE adds 2.8 gaps per iteration to the longer profile. It 
is therefore more realistic to assume that <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> is O(<em xmlns="" class="EmphasisTypeItalic">L</em> + <em xmlns="" class="EmphasisTypeItalic">N</em>), making one iteration of progressive alignment O(<em xmlns="" class="EmphasisTypeItalic">NL</em> + <em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup>) in both space and time. This analysis is summarized in Table <span class="InternalRef"><a href="#Tab1">1</a></span>.<figure xmlns="" id="Tab1" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 1</span>
                        <p xmlns="" class="SimplePara">Complexity of CLUSTALW. Here we show the big-O asymptotic complexity of the elements of CLUSTALW as a function of <em class="EmphasisTypeItalic">L</em>, the typical sequence length, and <em class="EmphasisTypeItalic">N</em>, the number of sequences, retaining the highest-order terms in <em class="EmphasisTypeItalic">N</em> with <em class="EmphasisTypeItalic">L</em> fixed and vice versa.</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Step</p></th><th><p xmlns="" class="SimplePara">O(Space)</p></th><th><p xmlns="" class="SimplePara">O(Time)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">Distance matrix</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em></p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                              <em class="EmphasisTypeItalic">L</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Neighbor joining</p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">4</sup>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Progressive (one iteration)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">NL</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> + <em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> = <em class="EmphasisTypeItalic">NL</em> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">NL</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> + <em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Progressive (total)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">NL</em> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">3</sup> + <em class="EmphasisTypeItalic">NL</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TOTAL</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">4</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr></tbody></table></div></figure></div>
              </section><section id="Sec33" class="Section2 RenderAsSection2">
                <h3 class="Heading">Initial distance measure</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">One
 might expect (a) that a more accurate distance measure would lead to a 
more accurate final alignment due to an improved tree, and (b) that 
errors due to a less accurate distance measure might be eliminated by 
allowing Stage 2 to iterate more times. Neither of these expectations is
 supported by our test results (unpublished). Allowing Stage 2 to 
iterate more than once with the goal of further improving the tree gave 
no significant improvement with any distance measure. Possibly, the tree
 is biased towards the MSA that was used to estimate it, and the MSA is 
biased by the tree used to create it, making it hard to achieve 
improvements. The most accurate measure on a pair of sequences is 
presumably the fractional identity <em xmlns="" class="EmphasisTypeItalic">D</em> computed from a global alignment, but use of <em xmlns="" class="EmphasisTypeItalic">D</em>
 in step 1.1 does not improve average accuracy on benchmark tests. The 
6-class Dayhoff alphabet used by MAFFT proved to give slightly higher 
benchmark accuracy scores, despite the fact that other alphabets were 
found to correlate better with <em xmlns="" class="EmphasisTypeItalic">D</em> [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]. We also found that the use of the binary approximation <em xmlns="" class="EmphasisTypeItalic">F</em><sup>Binary</sup>
 gave slightly reduced accuracy scores even when Stage 2 was allowed to 
iterate. The default choice in MUSCLE is therefore to use the Dayhoff 
alphabet in step 1.1 and to execute Stage 2 once only. While the impact 
on the average accuracy of the final alignment due to the different 
options is not understood, we observe that a better alignment of a pair 
of sequences is often obtained from a multiple alignment than from a 
pair-wise alignment, due to the presence of intermediate sequences 
having higher identities. It is therefore plausible that <em xmlns="" class="EmphasisTypeItalic">D</em> obtained from the multiple alignment in step 2.1 may be more accurate than <em xmlns="" class="EmphasisTypeItalic">D</em>
 obtained from a pair-wise alignment in step 1.1, and this may be 
relatively insensitive to the method used to create the tree for Stage 
1. But this leaves unexplained why <em xmlns="" class="EmphasisTypeItalic">k</em>-mer counting appears to be as good as or better than <em xmlns="" class="EmphasisTypeItalic">D</em> in Stage 1. Computing <em xmlns="" class="EmphasisTypeItalic">F</em> from a pair of sequences is O(<em xmlns="" class="EmphasisTypeItalic">L</em>) time and O(1) space, so for all pairs the similarity calculation is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup><em xmlns="" class="EmphasisTypeItalic">L</em>), compared with O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup><em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup>) in CLUSTALW. For a typical <em xmlns="" class="EmphasisTypeItalic">L</em> around 250, combined with an order of magnitude improvement due to the simplicity of <em xmlns="" class="EmphasisTypeItalic">k</em>-mer
 counting compared with dynamic programming, this typically gives a 
three orders of magnitude speed improvement for computing the distance 
matrix in MUSCLE compared with CLUSTALW. The default strategy is 
therefore well justified as a speed optimization, and has the added 
bonus of providing a small improvement in accuracy.</p>
              </section><section id="Sec34" class="Section2 RenderAsSection2">
                <h3 class="Heading">Clustering</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">MUSCLE
 implements both UPGMA and neighbor-joining. We found UPGMA to give 
slightly better benchmark scores than neighbor-joining; UPGMA is 
therefore the default option. We expect neighbor-joining to give a 
better estimate of the correct evolutionary tree (see e.g. [<span class="CitationRef"><a href="#CR38" data-toggle="citation">38</a></span>]). However, it is well-known that alignment accuracy decreases with lower sequence identity (see e.g. [<span class="CitationRef"><a href="#CR39" data-toggle="citation">39</a></span>]).
 It follows that given a set of profiles, the two that can be aligned 
most accurately will tend to be the pair with the highest identity, i.e.
 at the shortest evolutionary distance. This is exactly the pair 
selected by the nearest-neighbor criterion in UPGMA. By contrast, 
neighbor-joining selects a pair of evolutionary neighbors, i.e. a pair 
having a common ancestor. When mutation rates are variable, the 
evolutionary neighbor may not be the nearest neighbor (Figure <span class="InternalRef"><a href="#Fig6">6</a></span>).
 This explains why a nearest-neighbor tree may be superior to the true 
evolutionary tree for guiding a progressive alignment. Neighbor-joining 
is naively O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>4</sup>) time, although this can be reduced to O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>3</sup>). UPGMA is naively O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>3</sup>) time as the minimum of an <em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup> matrix must be found in each of <em xmlns="" class="EmphasisTypeItalic">N</em> - 1 iterations. However, this can be reduced to O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup>)
 time by maintaining a vector of pointers to the minimum value in each 
row of the matrix. We are again fortunate to find that the most accurate
 method is also the fastest.<figure xmlns="" class="Figure" id="Fig6"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig6_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig6_HTML.jpg" alt="Figure 6" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig6_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig6_HTML.jpg" alt="Figure 6"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 6</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Neighbor-joining and UPGMA trees for progressive alignment.</strong>
 Here we show the same set of four sequences and the order in which they
 will be aligned according to a neighbor-joining tree (above) and a 
UPGMA tree (below). Notice that <em class="EmphasisTypeItalic">t</em> and <em class="EmphasisTypeItalic">u</em> are the most closely related pair, but (<em class="EmphasisTypeItalic">s</em>, <em class="EmphasisTypeItalic">t</em>) and (<em class="EmphasisTypeItalic">u</em>, <em class="EmphasisTypeItalic">v</em>) are evolutionary neighbors. With neighbor joining, <em class="EmphasisTypeItalic">t</em> and <em class="EmphasisTypeItalic">u</em> are not aligned to each other until the root, in contrast to UPGMA, which aligns <em class="EmphasisTypeItalic">s</em> and <em class="EmphasisTypeItalic">t</em> as the first pair.</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig7"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig7_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig7_HTML.jpg" alt="Figure 7" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig7_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig7_HTML.jpg" alt="Figure 7"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 7</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Additive profiles.</strong>
 The profile functions in MUSCLE require amino acid frequencies for each
 column. Here we show the alignment of two profiles X and Y, giving a 
new profile Z. Note that the count <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">Z</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>for amino acid <em class="EmphasisTypeItalic">i</em> in a given column of Z is the sum of the counts in the child profiles, i.e. <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">Z</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>= <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">X</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>+ <em class="EmphasisTypeItalic">n</em><sup xmlns="http://www.w3.org/1999/xhtml">Y</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>. In terms of frequencies, this becomes <em class="EmphasisTypeItalic">f</em> <sup xmlns="http://www.w3.org/1999/xhtml">Z</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>= <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">X</sup><em class="EmphasisTypeItalic">f</em> <sup xmlns="http://www.w3.org/1999/xhtml">X</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>/<em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">Z</sup> + <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">Y</sup><em class="EmphasisTypeItalic">f</em> <sup xmlns="http://www.w3.org/1999/xhtml">Y</sup><sub xmlns="http://www.w3.org/1999/xhtml">
                            <em xmlns="" class="EmphasisTypeItalic">i</em>
                          </sub>/<em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">Z</sup>, where <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">X</sup>, <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">Y</sup>, <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">Z</sup>
 are the number of sequences in X, Y and Z respectively. Therefore, 
given a suitable sequence weighting scheme, it is possible to compute 
frequencies in Z from the frequencies in X and Y. This avoids the step 
of building an explicit multiple alignment for Z in order to compute 
frequencies, as done in CLUSTALW and MAFFT.</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec35" class="Section2 RenderAsSection2">
                <h3 class="Heading">Dynamic programming</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The textbook algorithm for pair-wise alignment with affine penalties employs three dynamic programming matrices; see e.g. [<span class="CitationRef"><a href="#CR40" data-toggle="citation">40</a></span>, <span class="CitationRef"><a href="#CR41" data-toggle="citation">41</a></span>].
 A more time-and space-efficient implementation can be achieved using 
linear space for the recursion relations and a single matrix for 
trace-back (Kazutaka Katoh, personal communication). Consider sequences X
 and Y length <em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub>, <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub>. We use the following notation: X<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>is the <em xmlns="" class="EmphasisTypeItalic">x</em> th letter in X, X<sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup>the first <em xmlns="" class="EmphasisTypeItalic">x</em> letters in X, <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>the substitution score (or profile function) for aligning X<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>to Y<sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>, <em xmlns="" class="EmphasisTypeItalic">b</em><sup>X</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>the score for a gap-open in Y that is aligned to X<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>, <em xmlns="" class="EmphasisTypeItalic">t</em><sup>X</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>the score for a gap-close aligned to X<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>, <em xmlns="" class="EmphasisTypeItalic">U</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>the set of all alignments of X<sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup>to Y<sup>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sup>, M<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>the score of the best alignment in <em xmlns="" class="EmphasisTypeItalic">U</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>ending in a match (i.e., X<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>and Y<sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>are aligned), D<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>the score of the best alignment ending in a delete relative to X (X<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>is aligned to an indel) and I<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>the score of the best alignment ending in an insert (Y<sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>is aligned to an indel). A match is preceded by either a match, delete or insert, so:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">M<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>= <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>+ max { M<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>, D<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>+ <em xmlns="" class="EmphasisTypeItalic">t</em><sup>X</sup><sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1</sub>, I<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>+ <em xmlns="" class="EmphasisTypeItalic">t</em><sup>Y</sup><sub><em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>} &nbsp;&nbsp;&nbsp; (14)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We assume that a center parameter has been added to <em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>such that the gap extension penalty is zero. By considering all possible lengths for the final gap,</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">D<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>= max(<em xmlns="" class="EmphasisTypeItalic">k</em>&lt;<em xmlns="" class="EmphasisTypeItalic">x</em>) [M<sub>
                    <em xmlns="" class="EmphasisTypeItalic">ky</em>
                  </sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>X</sup><sub><em xmlns="" class="EmphasisTypeItalic">k</em>+1</sub>]. &nbsp;&nbsp;&nbsp; (15)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Here, <em xmlns="" class="EmphasisTypeItalic">k</em> is the last position in X that is aligned to a letter in Y. Extract the special case of a gap of length 1:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">D<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>= max { max(<em xmlns="" class="EmphasisTypeItalic">k</em>&lt;<em xmlns="" class="EmphasisTypeItalic">x</em>-1) [M<sub>
                    <em xmlns="" class="EmphasisTypeItalic">ky</em>
                  </sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>X</sup><sub><em xmlns="" class="EmphasisTypeItalic">k</em>+1</sub>], M<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em></sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>X</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>}. &nbsp;&nbsp;&nbsp; (16)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Hence,</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">D<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>= max { D<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em></sub>, M<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em></sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>X</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>}. &nbsp;&nbsp;&nbsp; (17)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Similarly,</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">I<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>= max { I<sub><em xmlns="" class="EmphasisTypeItalic">xy</em>-1</sub>, M<sub><em xmlns="" class="EmphasisTypeItalic">xy</em>-1</sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>Y</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>}. &nbsp;&nbsp;&nbsp; (18)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Let the outer loop iterate over increasing <em xmlns="" class="EmphasisTypeItalic">x</em> and the inner loop over increasing <em xmlns="" class="EmphasisTypeItalic">y</em>. For fixed <em xmlns="" class="EmphasisTypeItalic">x</em>, define vectors M<sup>
                    <em xmlns="" class="EmphasisTypeItalic">curr</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>= M<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>, M<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>= M<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em></sub>, D<sup>
                    <em xmlns="" class="EmphasisTypeItalic">curr</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>= D<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>, D<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>= D<sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1<em xmlns="" class="EmphasisTypeItalic">y</em></sub>; for fixed <em xmlns="" class="EmphasisTypeItalic">x</em>, <em xmlns="" class="EmphasisTypeItalic">y</em> define scalars I<sup>
                    <em xmlns="" class="EmphasisTypeItalic">curr</em>
                  </sup>= I<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>, I<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup>= I<sub><em xmlns="" class="EmphasisTypeItalic">xy</em>-1</sub>. Now we can re-write (14), (17) and (18) to obtain the following recursion relations:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">M<sup>
                    <em xmlns="" class="EmphasisTypeItalic">curr</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>= S<sub>
                    <em xmlns="" class="EmphasisTypeItalic">xy</em>
                  </sub>+ max { M<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub><em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>, D<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub><em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>+ <em xmlns="" class="EmphasisTypeItalic">t</em><sup>X</sup><sub><em xmlns="" class="EmphasisTypeItalic">x</em>-1</sub>, I<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub><em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>+ <em xmlns="" class="EmphasisTypeItalic">t</em><sup>Y</sup><sub><em xmlns="" class="EmphasisTypeItalic">y</em>-1</sub>} &nbsp;&nbsp;&nbsp; (19)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">D<sup>
                    <em xmlns="" class="EmphasisTypeItalic">curr</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>= max { D<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>, M<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>X</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>} &nbsp;&nbsp;&nbsp; (20)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">I<sup>
                    <em xmlns="" class="EmphasisTypeItalic">curr</em>
                  </sup>= max { I<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup>, M<sup>
                    <em xmlns="" class="EmphasisTypeItalic">prev</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>+ <em xmlns="" class="EmphasisTypeItalic">b</em><sup>Y</sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">y</em>
                  </sub>}. &nbsp;&nbsp;&nbsp; (21)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">An <em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub> × <em xmlns="" class="EmphasisTypeItalic">L</em><sub>Y</sub> matrix is needed for the trace-back that produces the final alignment.</p>
                <section id="Sec36" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Inner loop</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 inner-most dynamic programming loop, which computes the profile 
function, deserves careful optimization. We will consider the case of 
PSP; similar optimizations are possible for LE. PSP = Σ<sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>Σ<sub>
                      <em xmlns="" class="EmphasisTypeItalic">j</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                      <em xmlns="" class="EmphasisTypeItalic">x</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">j</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">ij</em>
                    </sub>= Σ<sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                      <em xmlns="" class="EmphasisTypeItalic">x</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">W</em><sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>, where <em xmlns="" class="EmphasisTypeItalic">W</em><sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>= Σ<sub>
                      <em xmlns="" class="EmphasisTypeItalic">j</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">j</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">S</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">ij</em>
                    </sub>. The vector <em xmlns="" class="EmphasisTypeItalic">W</em><sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>is used <em xmlns="" class="EmphasisTypeItalic">L</em><sub>X</sub>
 times, and it therefore pays to compute it once and cache it. Observe 
that a typical profile column contains &lt;&lt; 20 different amino 
acids. We sort the frequencies in decreasing order; the summation Σ<sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                      <em xmlns="" class="EmphasisTypeItalic">x</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub><em xmlns="" class="EmphasisTypeItalic">W</em><sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>is terminated if a frequency <em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                      <em xmlns="" class="EmphasisTypeItalic">x</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>= 0 is encountered. This typically reduces the
 time spent in the summation, especially when sequences are closely 
related. As with <em xmlns="" class="EmphasisTypeItalic">W</em><sup>
                      <em xmlns="" class="EmphasisTypeItalic">y</em>
                    </sup><sub>
                      <em xmlns="" class="EmphasisTypeItalic">i</em>
                    </sub>, the sort order is computed once and cached. 
Observe that the roles of the two profiles are not symmetrical. It is 
most efficient to choose X, for which frequency sort orders are 
computed, to be the profile with the lowest amino acid diversity when 
averaged over columns. With this choice, the summation terminates 
earlier on average then if the other profile is identified as X. Note 
that out of <em xmlns="" class="EmphasisTypeItalic">N</em> - 1 iterations of progressive alignment, a minimum of <span class="InlineEquation" id="IEq1">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_IEq1_HTML.gif"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_IEq1_HTML.gif" alt="" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_IEq1_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_IEq1_HTML.gif" alt=""></noscript></a>
                      </span> and maximum of <em xmlns="" class="EmphasisTypeItalic">N</em>
 - 1 profile-profile alignments will include at least one profile 
containing one sequence only, and in the refinement phase exactly <em xmlns="" class="EmphasisTypeItalic">N</em> of the 2<em xmlns="" class="EmphasisTypeItalic">N</em>
 - 1 edges in the tree terminate on a leaf. At least half of all 
profile-profile alignments created in the MUSCLE algorithm therefore 
include a profile of one sequence only. Special cases where one or both 
profiles is a single sequence can be handled in separate subroutines, 
saving overhead due to unneeded loops that are guaranteed to execute 
once only. This optimization is especially useful for the LE function as
 it enables the logarithm to be incorporated into the <em xmlns="" class="EmphasisTypeItalic">W</em> vector.</p>
                </section>
              </section><section id="Sec37" class="Section2 RenderAsSection2">
                <h3 class="Heading">Diagonal finding</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Many
 alignment algorithms are optimized for speed, typically at some expense
 in average accuracy, by using fast methods to identify regions of high 
similarity between two sequences, which appear as diagonals in the 
similarity matrix. The alignment path is then constrained to include 
these diagonals, reducing the area of the dynamic programming matrix 
that must be computed. MAFFT uses the fast Fourier transform to find 
diagonals. MUSCLE uses a different technique which we have previously 
shown [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>] have comparable sensitivity and to be significantly faster. We use a compressed alphabet to find <em xmlns="" class="EmphasisTypeItalic">k</em>-mers
 in common between two sequences, then attempt to extend the match. In 
the case of diagonal identification we found compressed alphabets to 
significantly out-perform the standard amino acid alphabet [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>].
 Currently, MUSCLE uses 6-mers in the Dayhoff alphabet for diagonal 
finding, as for the initial distance measure, though other alphabets are
 known to give slightly better performance [<span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>].
 A candidate diagonal is rejected if there is any overlap (i.e., if a 
single position in one of the sequences appears in two or more 
diagonals) or if it is less than a minimum length (default 24). The ends
 of the diagonal are deleted (by default, the first and last five 
positions) as they are less reliable. Despite these heuristics, we find 
the use of diagonal-finding to reduce average accuracy and to give only 
modest improvements in speed for typical input data; this option is 
therefore disabled by default. Similar results are seen in MAFFT; the 
most accurate MAFFT script is NWNSI [<span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>], in which diagonal-finding is also disabled.</p>
              </section><section id="Sec38" class="Section2 RenderAsSection2">
                <h3 class="Heading">Additive profiles</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Both
 the PSP and LE profile functions are defined in terms of amino acid 
frequencies and position-specific gap penalties. The data structure 
representing a profile is a vector of length <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>
 in which each element contains frequencies for each amino acid type and
 a few additional values related to gaps. We call this data structure a <em xmlns="" class="EmphasisTypeItalic">profile vector</em>, as distinct from a <em xmlns="" class="EmphasisTypeItalic">profile matrix</em>, an explicit <em xmlns="" class="EmphasisTypeItalic">N</em> × <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> multiple alignment containing letters and indels. For <em xmlns="" class="EmphasisTypeItalic">N</em>
 &gt; 20, using profile vectors reduces the cost of computing the 
profile function compared with profile matrices, and is therefore 
preferred for use in dynamic programming. In CLUSTALW and MAFFT, the 
implementation of progressive alignment builds a profile matrix at each 
internal node of the tree, which is used to compute a profile vector. 
This procedure is O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) = O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup> + <em xmlns="" class="EmphasisTypeItalic">NL</em>) in time and space, becoming expensive for large <em xmlns="" class="EmphasisTypeItalic">N</em>.
 Observe that the count of a given amino acid in a column in the parent 
matrix is the sum of the counts in the two child columns that are 
aligned at that position (Figure <span class="InternalRef"><a href="#Fig7">7</a></span>).
 With a suitable sequence weighting scheme, it is therefore possible to 
compute the amino acid frequencies of the parent profile vector from the
 frequencies in the two child profile vectors and the alignment path. 
This is an O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) procedure in both time and space, giving a significant advantage for <em xmlns="" class="EmphasisTypeItalic">N</em>
 &gt;&gt; 20. Three issues must be addressed to fully implement this 
idea: the sequence weighting scheme, inclusion of occupancy factors and 
position-specific gap penalties, and construction of a profile matrix 
(i.e., the final multiple alignment) at the root node.</p>
                <section id="Sec39" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Sequence weighting</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 the frequencies in the parent profile vector to be a linear combination
 of the child frequencies, the weight assigned to a sequence must be the
 same in the child and parent profiles. This requirement is not 
satisfied, for example, by the Henikoff or PSI-BLAST schemes, which 
compute weights based on a multiple alignment. We therefore choose the 
CLUSTALW scheme, which computes a fixed weight for each sequence from 
edge lengths in the tree.</p>
                </section>
                <section id="Sec40" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Gap representation</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To compute gap penalties, we need the frequencies <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">o</em>
                    </sub>of gap opens and <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">c</em>
                    </sub>of gap closes in each position. In the case of the LE profile function, we additionally require the gap frequency <em xmlns="" class="EmphasisTypeItalic">f</em><sub>G</sub>. These can be accommodated by storing <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">o</em>
                    </sub>, <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">c</em>
                    </sub>and <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">e</em>
                    </sub>in the profile vector, where <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">e</em>
                    </sub>is the frequency of gap-extensions in the 
column (meaning that indels are found in a given sequence in the column,
 the preceding column and in the following column; i.e., a gap-close is 
not counted as an extension). These three <em xmlns="" class="EmphasisTypeItalic">occupancy frequencies</em> are sufficient for computing the profile function and the position-specific gap penalties <em xmlns="" class="EmphasisTypeItalic">b</em> and <em xmlns="" class="EmphasisTypeItalic">t</em>. Note that we can compute the frequency <em xmlns="" class="EmphasisTypeItalic">f</em><sub>G</sub> of indels, as needed for the occupancy factor in the profile function, as follows:</p>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">f</em><sub>G</sub> = <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">o</em>
                    </sub>+ <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">c</em>
                    </sub>+ <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">e</em>
                    </sub>. &nbsp;&nbsp;&nbsp; (22)</p>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Now
 consider the problem of computing the occupancy frequencies in the 
parent profile vector, given only the child occupancy frequencies and 
the trace-back path for the alignment. Consider first a diagonal edge in
 the path, i.e. an edge that does not open or extend a gap, following 
another diagonal edge. In this case, the occupancy frequencies are 
computed similarly to amino acid frequencies (as a sum in which a child 
frequency is weighted according to the total weight of the sequences in 
its profile). For horizontal or vertical edges, i.e. edges that open or 
extend gaps, the parent occupancy frequencies can be computed by 
considering the effect of the new column of indels (Figure <span class="InternalRef"><a href="#Fig8">8</a></span>). It is straightforward to work through all cases and show that the three frequencies <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">o</em>
                    </sub>, <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">c</em>
                    </sub>and <em xmlns="" class="EmphasisTypeItalic">f</em><sub>
                      <em xmlns="" class="EmphasisTypeItalic">e</em>
                    </sub>are sufficient for their values in the parent profile vector to be computed in O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) time from the child profile vectors and alignment path.<figure xmlns="" class="Figure" id="Fig8"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig8_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig8_HTML.jpg" alt="Figure 8" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig8_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig8_HTML.jpg" alt="Figure 8"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 8</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Occupancy frequencies in additive profiles.</strong> Here we show an alignment of profiles X and Y giving Z. A column C of indels (shaded background) has been inserted at position <em class="EmphasisTypeItalic">x</em> in order to align X to Y. To compute the number of gap-extensions in column <em class="EmphasisTypeItalic">x</em>
 of Z, three cases must be considered: (1) a gap-extension in the 
corresponding column of Y, (2) a gap-open in the preceding column of X, 
and (3) a gap-extension in the preceding column of X. By enumerating all
 such cases, it is straightforward to compute the occupancy frequencies 
in Z from the occupancy frequencies of X and Y, plus the alignment path.</p>
                        </div></figcaption></figure></div>
                </section>
                <section id="Sec41" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Construction of the root alignment</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">By avoiding the use of profile matrices, the complexity of a single progressive alignment iteration is reduced from O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> <sup>2</sup> + <em xmlns="" class="EmphasisTypeItalic">NL</em><sub>p</sub>) space and O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup> + <em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) time to O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup>) = O(<em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup> + <em xmlns="" class="EmphasisTypeItalic">NL</em>) space and time. The <em xmlns="" class="EmphasisTypeItalic">NL</em>
 term in the time complexity is now due only to the increase in profile 
length, and is therefore typically much smaller than before. The root 
alignment is constructed by storing the alignment path produced at each 
internal node. For each input sequence, the path to the root is 
followed, inserting the gaps induced by each alignment path at each 
internal node. This procedure is O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub> log <em xmlns="" class="EmphasisTypeItalic">N</em>) = O(<em xmlns="" class="EmphasisTypeItalic">N</em> <sup>2</sup> log <em xmlns="" class="EmphasisTypeItalic">N</em> + <em xmlns="" class="EmphasisTypeItalic">NL</em> log <em xmlns="" class="EmphasisTypeItalic">N</em>) time, and requires O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) = O(<em xmlns="" class="EmphasisTypeItalic">NL</em> + <em xmlns="" class="EmphasisTypeItalic">N</em> <sup>2</sup>) space for storage of the paths. This is expensive for large <em xmlns="" class="EmphasisTypeItalic">N</em>, and we therefore optimize this step by using a device we call an <em xmlns="" class="EmphasisTypeItalic">e-string</em>, a type of edit string.</p>
                </section>
                <section id="Sec42" class="Section3 RenderAsSection3">
                  <h4 class="Heading">E-strings</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">An
 alignment path can be considered as an operator on a pair of sequences 
that inserts indels into those sequences such that their lengths become 
equal. Conventionally, an alignment path is represented as a vector of 
three symbols representing edges in the graph, say M, D and I (for 
match, delete and insert, i.e. a diagonal, horizontal or vertical edge).
 Note that indels in one sequence are inserted only by Ds, indels in the
 other are inserted only by Is. Define an e-string <em xmlns="" class="EmphasisTypeItalic">e</em> to be a vector of |<em xmlns="" class="EmphasisTypeItalic">e</em>| integers interpreted as an operator that inserts indels into a string by scanning it from left to right (Figure <span class="InternalRef"><a href="#Fig9">9</a></span>). A positive integer <em xmlns="" class="EmphasisTypeItalic">n</em> means skip <em xmlns="" class="EmphasisTypeItalic">n</em> letters of the string; a negative integer -<em xmlns="" class="EmphasisTypeItalic">n</em> means insert <em xmlns="" class="EmphasisTypeItalic">n</em>
 indels at the current position. We require the vector to be in its 
shortest form, so signs always alternate. We represent an alignment path
 as a pair of e-strings, one for each sequence, assigned to the 
appropriate edges in the tree. We will typically find that |<em xmlns="" class="EmphasisTypeItalic">e</em>|, the length of the e-string, is much less than <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>,
 the length of the alignment path. Now consider the effect of applying 
two consecutive e-strings ("multiplying" them). This can be expressed as
 a third e-string, which can be efficiently computed in O(|<em xmlns="" class="EmphasisTypeItalic">e</em>|)
 time from the multiplicands. For each leaf (input sequence), the 
product is computed of e-strings on the path to the root (Figure <span class="InternalRef"><a href="#Fig10">10</a></span>).
 The final e-string obtained at the root is then applied to the 
sequence. This method does not reduce the big-O time or space 
complexity, but is much faster than a naive implementation.<figure xmlns="" class="Figure" id="Fig9"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig9_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig9_HTML.jpg" alt="Figure 9" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig9_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig9_HTML.jpg" alt="Figure 9"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 9</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">E-strings.</strong> (1) The effect of the e-string operator &lt;3,-1,2&gt; on the sequence MQTIF. A positive number <em class="EmphasisTypeItalic">n</em> skips <em class="EmphasisTypeItalic">n</em> letters, a negative number <em class="EmphasisTypeItalic">-n</em> insert <em class="EmphasisTypeItalic">n</em>
 indels. (2) The effect of applying two successive e-strings. In the 
last line, the result is expressed as a new e-string applied to the 
original string. (3) We define multiplication on two e-strings as 
yielding the e-string that is equivalent to applying the two e-strings 
in order. (4) An alignment path is conventionally represented as a 
vector of edge types (<em class="EmphasisTypeItalic">M</em>, <em class="EmphasisTypeItalic">D</em> and <em class="EmphasisTypeItalic">I</em>). In this example, <em class="EmphasisTypeItalic">MDMIMM</em>,
 shown above a pairwise alignment, is the path that generates that 
alignment. The alignment can also be generated by a pair of e-strings 
(shown to the right). An alignment path is therefore equivalent to a 
pair of e-strings.</p>
                        </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig10"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig10_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig10_HTML.jpg" alt="Figure 10" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig10_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig10_HTML.jpg" alt="Figure 10"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 10</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Root alignment construction.</strong>
 Here we show the same progressive alignment as Figure 1. Each edge in 
the tree is labeled with the e-string for its side of the alignment at 
the parent node. The e-string needed to insert indels into a sequence in
 the root alignment can be determined by multiplying e-strings along the
 path to the root. For example, for sequence LSF, the root e-string is 
&lt;3,-1,1&gt;*&lt;1,-1,2&gt; = &lt;1,-1,1,-1,1&gt;.</p>
                        </div></figcaption></figure></div>
                </section>
                <section id="Sec43" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Brenner's method</h4>
                  <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Steven
 Brenner (personal communication) observed that a multiple alignment can
 be alternatively be obtained by aligning each sequence to the root 
profile. This requires O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub><sup>2</sup>) time, giving a lower asymptotic complexity in <em xmlns="" class="EmphasisTypeItalic">N</em> at the expense of an additional factor of <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>.
 This method gives opportunities for errors relative to the "exact" 
e-string solution (when a sequence misaligns to its copy in the 
profile), but can also lead to improvements by allowing the sequence to 
correctly align to conserved motifs that were not apparent when the 
sequence was added. (Note the resemblance to the refinement stage, which
 begins by re-aligning individual sequences to the rest). The chances 
for error are reduced by constraining the alignment to forbid gaps in 
the root profile. Our tests show that this method gives comparable 
average accuracy to the e-string solution but to be slower for up to at 
least a few thousand sequences, and e-strings are therefore used by 
default.</p>
                </section>
              </section><section id="Sec44" class="Section2 RenderAsSection2">
                <h3 class="Heading">Refinement complexity</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In
 the following, we assume that an explicit multiple alignment (profile 
matrix) of all sequences is maintained, and determine the complexity of 
each step in Stage 3. Step 3.1 determines the bipartition induced by 
deleting an edge from the tree. This is O(<em xmlns="" class="EmphasisTypeItalic">N</em>)
 time, and sufficiently fast that there is little motivation for further
 optimization. Step 3.2 extracts profiles for the two partitions from 
the current multiple alignment and computes their profile vectors, which
 is O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) time and space. Step 3.3 performs profile-profile alignment, which is O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup>) time and space. Step 3.4 computes the SP score, which is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup><em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) time and O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) space (discussed in more detail shortly). A single iteration of Stage 3 is thus O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup><em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> + <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup>) time and O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub> + <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup>) space. There are O(<em xmlns="" class="EmphasisTypeItalic">N</em>) edges in the tree, so executing this process for all edges is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>3</sup><em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> + <em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub><sup>2</sup>) time and O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub> + <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub><sup>2</sup>) space, which is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>4</sup> + <em xmlns="" class="EmphasisTypeItalic">N</em><sup>3</sup><em xmlns="" class="EmphasisTypeItalic">L</em> + <em xmlns="" class="EmphasisTypeItalic">NL</em><sup>2</sup>) time and O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup> + <em xmlns="" class="EmphasisTypeItalic">NL</em> + <em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup>)
 space. Assuming that a fixed maximum number of iterations of Stage 3 is
 imposed, this is also the total complexity of refinement. We now 
consider optimizations of the refinement stage.</p>
              </section><section id="Sec45" class="Section2 RenderAsSection2">
                <h3 class="Heading">Anchor columns</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">A multiple alignment can be divided vertically at high-confidence (<em xmlns="" class="EmphasisTypeItalic">anchor</em>) columns. Each vertical block is then refined separately, improving speed and reducing space due to the O(<em xmlns="" class="EmphasisTypeItalic">L</em><sup>2</sup>) factor in dynamic programming. This strategy has been used by several previous algorithms, including PRRP [<span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>], RASCAL [<span class="CitationRef"><a href="#CR42" data-toggle="citation">42</a></span>]
 and MAFFT. In MUSCLE, the following criteria are used to identify 
anchor columns. The profile function (LE or PSP) must exceed a 
threshold, the averaged profile function over a window around the 
position must exceed a (lower) threshold, and the column may not contain
 a gap. In addition, the contribution to the averaged score from a 
single column has a ceiling, reducing skew in the averaged score due to 
exceptionally high-scoring columns. These heuristics are designed to 
avoid anchor columns that have high scores but are either artifacts 
(similar residues found by chance in unrelated regions) or are too close
 to variable regions. When performing a profile-profile alignment, each 
anchor column and its two immediate neighbors (which form the boundaries
 of vertical blocks) are required to remain aligned; i.e., terminal gaps
 are forbidden except at the true terminals. Introducing this constraint
 overcomes a small degradation in average alignment quality that is 
otherwise observed. This implies that the degradation is sometimes due 
to cases where a well-conserved region is divided into two parts by an 
anchor column, one of which becomes short enough that it misaligns to a 
similar short motif elsewhere.</p>
              </section><section id="Sec46" class="Section2 RenderAsSection2">
                <h3 class="Heading">SP score</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Notice that computation of the SP score dominates the time complexity of refinement and of MUSCLE overall, introducing O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>4</sup>) and O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>3</sup><em xmlns="" class="EmphasisTypeItalic">L</em>) terms. We are therefore motivated to consider optimizations of this step. We first consider the contribution SP<sup>a</sup> to the SP score from amino acids; gap penalties require special treatment. Let <em xmlns="" class="EmphasisTypeItalic">s</em> and <em xmlns="" class="EmphasisTypeItalic">t</em> be sequences, <em xmlns="" class="EmphasisTypeItalic">x</em> be a column, <em xmlns="" class="EmphasisTypeItalic">s</em> [<em xmlns="" class="EmphasisTypeItalic">x</em>] be the amino acid of sequence <em xmlns="" class="EmphasisTypeItalic">s</em> in column <em xmlns="" class="EmphasisTypeItalic">x</em>, and <em xmlns="" class="EmphasisTypeItalic">S</em>(<em xmlns="" class="EmphasisTypeItalic">i</em>, <em xmlns="" class="EmphasisTypeItalic">j</em>) be the substitution score of amino acids <em xmlns="" class="EmphasisTypeItalic">i</em> and <em xmlns="" class="EmphasisTypeItalic">j</em>. It is convenient to impose an (arbitrary) ordering on the sequences and amino acid types. Then,</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">SP<sup>a</sup> = Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub>Σ<sub><em xmlns="" class="EmphasisTypeItalic">t</em> &gt;<em xmlns="" class="EmphasisTypeItalic">s</em></sub><em xmlns="" class="EmphasisTypeItalic">S</em>(<em xmlns="" class="EmphasisTypeItalic">s</em> [<em xmlns="" class="EmphasisTypeItalic">x</em>], <em xmlns="" class="EmphasisTypeItalic">t</em> [<em xmlns="" class="EmphasisTypeItalic">x</em>]). &nbsp;&nbsp;&nbsp; (23)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Define <em xmlns="" class="EmphasisTypeItalic">δ</em>(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">i</em>, <em xmlns="" class="EmphasisTypeItalic">x</em>) = 1 if <em xmlns="" class="EmphasisTypeItalic">s</em> [<em xmlns="" class="EmphasisTypeItalic">x</em>] = <em xmlns="" class="EmphasisTypeItalic">i</em>, 0 otherwise, and <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>[<em xmlns="" class="EmphasisTypeItalic">x</em>] = Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">δ</em>(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">i</em>, <em xmlns="" class="EmphasisTypeItalic">x</em>). We say <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>[<em xmlns="" class="EmphasisTypeItalic">x</em>] is the <em xmlns="" class="EmphasisTypeItalic">count</em> of amino acid type <em xmlns="" class="EmphasisTypeItalic">i</em> in column <em xmlns="" class="EmphasisTypeItalic">x</em>. We can now transform the sum over pairs of sequences into a sum over pairs of amino acid types:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">SP<sup>a</sup> = Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">j</em>
                  </sub><em xmlns="" class="EmphasisTypeItalic">n</em><sub><em xmlns="" class="EmphasisTypeItalic">j</em>&gt;<em xmlns="" class="EmphasisTypeItalic">i</em></sub><em xmlns="" class="EmphasisTypeItalic">S</em>(<em xmlns="" class="EmphasisTypeItalic">i</em>, <em xmlns="" class="EmphasisTypeItalic">j</em>) + 1/2 Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sub>Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>(<em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub><sup>2</sup> - <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>) <em xmlns="" class="EmphasisTypeItalic">S</em>(<em xmlns="" class="EmphasisTypeItalic">i</em>, <em xmlns="" class="EmphasisTypeItalic">i</em>). &nbsp;&nbsp;&nbsp; (24)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Frequencies are computed as:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><em xmlns="" class="EmphasisTypeItalic">f</em> <sup>
                    <em xmlns="" class="EmphasisTypeItalic">x</em>
                  </sup><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>= <em xmlns="" class="EmphasisTypeItalic">n</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">i</em>
                  </sub>[<em xmlns="" class="EmphasisTypeItalic">x</em>]/ <em xmlns="" class="EmphasisTypeItalic">N</em>. &nbsp;&nbsp;&nbsp; (25)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Using frequencies,</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">
                  <div id="Equa" class="Equation"><div class="EquationContent"><div class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Equa_HTML.gif"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Equa_HTML.gif" alt="" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Equa_HTML.gif"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Equa_HTML.gif" alt=""></noscript></a>
                    </div></div></div>
                </div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">For
 simplicity, we have neglected sequence weighting; it is straightforward
 to show that (26) applies unchanged if weighting is used. Note that 
(23) is O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup><em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) but (25) and (26) are O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>). For <em xmlns="" class="EmphasisTypeItalic">N</em> &gt;&gt; 20, this is a substantial improvement. Let SP<sup>g</sup> be the contribution of gap penalties to SP, so SP = SP<sup>a</sup> + SP<sup>g</sup>. It is natural to seek an O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) expression for SP<sup>g</sup>
 analogous to (26), but to the best of our knowledge no solution is 
known. Note that in MUSCLE refinement, the absolute value of the SP 
score is not needed; rather, it suffices to determine the difference in 
the SP scores before and after re-aligning a pair of profiles. Let SP(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">t</em>) be the contribution to the SP score from a pair of sequences <em xmlns="" class="EmphasisTypeItalic">s</em> and <em xmlns="" class="EmphasisTypeItalic">t</em>, so SP = Σ<sub>
                    <em xmlns="" class="EmphasisTypeItalic">s</em>
                  </sub>Σ<sub><em xmlns="" class="EmphasisTypeItalic">t</em>&gt;<em xmlns="" class="EmphasisTypeItalic">s</em></sub>SP(<em xmlns="" class="EmphasisTypeItalic">s</em>,<em xmlns="" class="EmphasisTypeItalic">t</em>), and denote the two profiles by X and Y. Then we can decompose SP into intra-and inter-profile terms as follows:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">SP = Σ<sub><em xmlns="" class="EmphasisTypeItalic">s</em><span class="stix">∈</span>X</sub>Σ<sub><em xmlns="" class="EmphasisTypeItalic">t</em><span class="stix">∈</span>X:<em xmlns="" class="EmphasisTypeItalic">t</em>&gt;<em xmlns="" class="EmphasisTypeItalic">s</em></sub>SP(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">t</em>) + Σ<sub><em xmlns="" class="EmphasisTypeItalic">s</em><span class="stix">∈</span>Y</sub>Σ<sub><em xmlns="" class="EmphasisTypeItalic">t</em><span class="stix">∈</span>Y:<em xmlns="" class="EmphasisTypeItalic">t</em>&gt;<em xmlns="" class="EmphasisTypeItalic">s</em></sub>SP(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">t</em>) + Σ<sub><em xmlns="" class="EmphasisTypeItalic">s</em><span class="stix">∈</span>X</sub>Σ<sub><em xmlns="" class="EmphasisTypeItalic">t</em><span class="stix">∈</span>Y</sub>SP(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">t</em>) &nbsp;&nbsp;&nbsp; (27)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Note
 that the intra-profile terms are unchanged in any alignment that 
preserves the columns of the profile intact, which is true by definition
 in profile-profile alignment. This follows by noting that any indels 
added to align the profiles are guaranteed to be external gaps with 
respect to any pair of sequences in the same profile. It therefore 
suffices to compute the change in the inter-profile term:</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">SP<sub>XY</sub> = Σ<sub><em xmlns="" class="EmphasisTypeItalic">s</em><span class="stix">∈</span>X</sub>Σ<sub><em xmlns="" class="EmphasisTypeItalic">t</em><span class="stix">∈</span>Y</sub>SP(<em xmlns="" class="EmphasisTypeItalic">s</em>, <em xmlns="" class="EmphasisTypeItalic">t</em>). &nbsp;&nbsp;&nbsp; (28)</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">This
 reduces the average time by a factor of about two. We can further 
improve on this by noting that in the typical case, there are few or no 
changes to the alignment. This suggests computing the change in SP score
 by looking only at differences between the two alignments. Let <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub> be the alignment path before re-alignment and <em xmlns="" class="EmphasisTypeItalic">π</em><sub>+</sub> the path after re-alignment. The change in alignment can be specified as the set of edges in <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub> or <em xmlns="" class="EmphasisTypeItalic">π</em><sub>+</sub>, but not both; i.e., by considering a path to be a set of edges and taking the set symmetric difference Δ<em xmlns="" class="EmphasisTypeItalic">π</em> = (<em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub> <span class="stix">∪</span> <em xmlns="" class="EmphasisTypeItalic">π</em><sub>+</sub>) - (<em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub> ∩ <em xmlns="" class="EmphasisTypeItalic">π</em><sub>+</sub>). The path <em xmlns="" class="EmphasisTypeItalic">π</em><sub>+</sub> after re-alignment is available from the dynamic programming traceback. The path <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub> before re-alignment can be efficiently computed in O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>)
 time. Note that in order to construct the profile of a subset of 
sequences extracted from a multiple alignment, those columns that 
contain only indels in that subset must be deleted. The set of such 
columns in both profiles is therefore available as a side effect of 
profile construction, and this set immediately implies <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub>. It is a simple O(<em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub>) procedure to compute Δ<em xmlns="" class="EmphasisTypeItalic">π</em> from <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub> and <em xmlns="" class="EmphasisTypeItalic">π</em><sub>+</sub>. Note that SP<sup>a</sup> is a sum over columns, and there is a one-to-one correspondence between columns and edges in <em xmlns="" class="EmphasisTypeItalic">π</em>. The change in SP<sup>a</sup> can therefore be computed as a sum over columns in Δ<em xmlns="" class="EmphasisTypeItalic">π</em>, with a negative sign for edges from <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub>, reducing the time complexity from O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) to O(<em xmlns="" class="EmphasisTypeItalic">N</em>|Δ<em xmlns="" class="EmphasisTypeItalic">π</em>|). We now turn our attention to SP<sup>g</sup>. We say that a gap G <em xmlns="" class="EmphasisTypeItalic">intersects</em> Δ<em xmlns="" class="EmphasisTypeItalic">π</em> if and only if any indel in G is in a column in Δ<em xmlns="" class="EmphasisTypeItalic">π</em>, and denote by Γ the set of gaps that intersect Δ<em xmlns="" class="EmphasisTypeItalic">π</em>. If a gap does not intersect Δ<em xmlns="" class="EmphasisTypeItalic">π</em>, i.e. does not have an indel in a changed column, its contribution to SP<sup>g</sup> is unchanged. It therefore suffices to consider penalties for gaps in Γ, again with negative signs for edges from <em xmlns="" class="EmphasisTypeItalic">π</em><sub>-</sub>. The construction of Γ is straightforward in O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>) time. Finally, a sum over pairs in Γ is needed, reducing the O(<em xmlns="" class="EmphasisTypeItalic">N</em><sup>2</sup>) component to the smallest possible set of terms.</p>
                <section id="Sec47" class="Section3 RenderAsSection3">
                  <h4 class="Heading">Dimer approximation</h4>
                  <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We next describe an approximation to SP that can be computed in O(<em xmlns="" class="EmphasisTypeItalic">NL</em><sub>P</sub>)
 time. Define a two-symbol alphabet {X, -} in which X represents any 
amino acid and - is the indel symbol. There are four dimers in this 
alphabet: XX, X-, -X and --, which denote by no-gap, gap-open, gap-close
 and gap-extend respectively. Re-write a multiple alignment in terms of 
these dimers, adopting the convention that dimer <em xmlns="" class="EmphasisTypeItalic">ab</em> composed of symbol <em xmlns="" class="EmphasisTypeItalic">a</em> in column <em xmlns="" class="EmphasisTypeItalic">x</em>-1 and symbol <em xmlns="" class="EmphasisTypeItalic">b</em> in column <em xmlns="" class="EmphasisTypeItalic">x</em> is written in column <em xmlns="" class="EmphasisTypeItalic">x</em>. Now consider the contribution to SP<sup>g</sup> of an aligned pair of dimers, written as <em xmlns="" class="EmphasisTypeItalic">ab</em>↔<em xmlns="" class="EmphasisTypeItalic">cd</em>. Clearly XX↔X- adds a gap-open penalty; XX↔-X adds a gap-close (Figure <span class="InternalRef"><a href="#Fig11">11</a></span>).
 To avoid double-counting, we will include only the penalty contribution
 of indels in the second column. Then XX↔X- adds a per-gap penalty, but 
XX↔-X adds zero because the second column does not contain a gap. 
External indels must be discarded; so, for example, --↔-- adds zero. In 
fact, aligning two identical dimers always contributes zero because any 
indel in the second column is found in both sequences and is therefore 
external. The contribution of all possible pairs of dimers is 
unambiguous, with the exception of -X↔--, which can add a per-gap or 
extend penalty (Figure <span class="InternalRef"><a href="#Fig12">12</a></span>). We approximate this case by assigning it a penalty of <em xmlns="" class="EmphasisTypeItalic">tg</em>, where <em xmlns="" class="EmphasisTypeItalic">g</em> is the default per-gap penalty and <em xmlns="" class="EmphasisTypeItalic">t</em>
 is a tunable parameter, set to 0.2 by default. With this approximation,
 dimers can be treated like amino acids: the scores for each aligned 
pair of dimers forms a substitution matrix (Figure <span class="InternalRef"><a href="#Fig13">13</a></span>), and SP<sup>g</sup>
 can be computed by summing substitution scores over all pairs of 
sequences. We can now apply Equation 26, re-interpreting the frequency 
vectors <em xmlns="" class="EmphasisTypeItalic">f</em> as having 24 components (20 amino acids and four dimers), and compute the change in SP by considering only those columns in Δ<em xmlns="" class="EmphasisTypeItalic">π</em>.
 We find use of the dimer approximation to marginally reduce benchmark 
scores. By default, MUSCLE therefore uses the exact SP score for <em xmlns="" class="EmphasisTypeItalic">N</em> ≤ 100 and the dimer approximation for <em xmlns="" class="EmphasisTypeItalic">N</em> &gt; 100, where the higher time complexity of the exact score becomes more noticeable.<figure xmlns="" class="Figure" id="Fig11"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig11_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig11_HTML.jpg" alt="Figure 11" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig11_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig11_HTML.jpg" alt="Figure 11"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 11</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Dimers in the {X,-} alphabet.</strong> Gap penalties for the sequence pair (<em class="EmphasisTypeItalic">s</em>, <em class="EmphasisTypeItalic">u</em>)
 can be computed be considering all aligned pairs of dimers in the 
alphabet {X,-}, where X is any amino acid and - is the usual indel 
symbol. Four cases are highlighted. Note that an aligned pair of 
identical dimers never contribute a gap penalty as any indels in the 
dimers are necessarily external, as in the left-most example.</p>
                        </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig12"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig12_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig12_HTML.jpg" alt="Figure 12" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig12_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig12_HTML.jpg" alt="Figure 12"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 12</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Problem dimer pair.</strong>
 The aligned dimer pair -X ↔ -- causes a problem because its gap penalty
 contribution cannot be computed without additional information. Note 
that the first column of indels is external; after this column is 
discarded, different penalties may be needed, as these two examples 
show.</p>
                        </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig13"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                        <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig13_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig13_HTML.jpg" alt="Figure 13" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig13_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig13_HTML.jpg" alt="Figure 13"></noscript></a>
                      </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 13</span>
                          <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Dimer substitution matrix.</strong>
 This matrix specifies the contribution to the total gap penalty for a 
pair of sequences for each possible pair of aligned dimers. Here, <em class="EmphasisTypeItalic">g</em> is the per-gap penalty, <em class="EmphasisTypeItalic">e</em> is the gap-extension penalty. The problem case -X ↔ -- is approximated as <em class="EmphasisTypeItalic">tg</em>, where <em class="EmphasisTypeItalic">t</em> is a tunable parameter.</p>
                        </div></figcaption></figure></div>
                </section>
              </section><section id="Sec48" class="Section2 RenderAsSection2">
                <h3 class="Heading">Evaluation of profile functions</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We have previously attempted a systematic comparison of profile functions [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>].
 The methodology used in that work demanded careful optimization of 
affine gap parameters for each function. This proved to be 
time-consuming and tedious, and we therefore tried the following 
alternative approach, inspired by the notion that a good profile 
function should be good at discriminating correctly aligned pairs of 
profile positions from incorrectly aligned pairs. The protocol begins 
with a set of pair-wise structural alignments. With the sequence of each
 structure as a query, we used PSI-BLAST to search the NCBI 
non-redundant protein sequence database [<span class="CitationRef"><a href="#CR43" data-toggle="citation">43</a></span>],
 giving a multiple sequence alignment (profile) for each structure. Note
 that we use the term profile in this context to mean the sequence 
alignment produced by PSI-BLAST, not the scoring matrix. Using the 
structural alignments as a guide, we then created a database in which 
columns from the PSI-BLAST profiles were aligned to each other, giving a
 large set of pairs of alignment columns that we consider to be 
correctly aligned (the "true" database, although there are undoubtedly 
misaligned sequences and hence some incorrect pairs). By selecting the 
same number of pairs of columns at random from structures in different 
FSSP families, we created a similar ("false") database of unrelated 
pairs. A profile function was evaluated by computing the score of each 
pair of columns in both the true and false databases, and then sorting 
the results in order of increasing score. The results can be displayed 
by reviewing the sorted list and, for each score <em xmlns="" class="EmphasisTypeItalic">S</em> in the list, plotting the number of true pairs with score ≤ <em xmlns="" class="EmphasisTypeItalic">S</em> (<em xmlns="" class="EmphasisTypeItalic">x</em> axis) against the number of false pairs with score ≤ <em xmlns="" class="EmphasisTypeItalic">S</em> (<em xmlns="" class="EmphasisTypeItalic">y</em> axis); we call the resulting graph a <em xmlns="" class="EmphasisTypeItalic">discrimination plot</em>.
 Ideally, all true pairs would score higher than all false pairs, in 
which case the profile function would be a perfect discriminator and 
would always produce perfect alignments. A function that perfectly 
discriminates will appear as a Γ-shaped plot; a function that has no 
ability to discriminate will appear as a diagonal plot along the line <em xmlns="" class="EmphasisTypeItalic">x</em> = <em xmlns="" class="EmphasisTypeItalic">y</em>. If a function F has a discrimination plot that is always above another function G (i.e., D<sub>F</sub>(<em xmlns="" class="EmphasisTypeItalic">x</em>) &gt; D<sub>G</sub>(<em xmlns="" class="EmphasisTypeItalic">x</em>) <span class="stix">∀</span> <em xmlns="" class="EmphasisTypeItalic">x</em>, where D<sub>F</sub> is the discriminator plot for F as a function of <em xmlns="" class="EmphasisTypeItalic">x</em>),
 then F has a superior ability to discriminate true from false pairs 
compared with G. If the plots intersect, the situation is ambiguous and 
neither function is clearly superior. We used sets of structural 
alignments from [<span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>] (PP) and [<span class="CitationRef"><a href="#CR44" data-toggle="citation">44</a></span>]
 (PP2). PP contains 588 structure pairs with sequence identity ≤ 30%, 
z-score ≥ 15, RMSD ≤ 2.5Å and an alignment length of ≥ 50 positions. 
These criteria were designed to select pairs of structures with low 
sequence identity and high structural similarity. PP2 contains 500 pairs
 selected from the FSSP database [<span class="CitationRef"><a href="#CR45" data-toggle="citation">45</a></span>]
 with ≤ 30% sequence identity, z-score ≥ 8 and ≤ 12, RMSD ≤ 3.5Å and 
alignment length ≥ 50. The criteria for PP2 were designed to select 
challenging alignments with low sequence identity and relatively high 
structural divergence, leading to a high frequency of gaps and 
therefore, presumably, a stronger dependence on accurate identification 
of sequence similarity. Results on PP2 show the LE function to have 
higher discrimination than all other tested functions (historically, the
 LE function was designed by systematic trial and error using a wide 
range of different profile functions with feedback from discrimination 
plots). This is illustrated in Figure <span class="InternalRef"><a href="#Fig14">14</a></span>, in which the discrimination plot for LE on PP2 is compared with several other functions: PSP, LA, Yona and Levitt's [<span class="CitationRef"><a href="#CR46" data-toggle="citation">46</a></span>], LAMA [<span class="CitationRef"><a href="#CR47" data-toggle="citation">47</a></span>].
 Using PP, we again find that LE is superior to LA (not shown), but the 
comparison with PSP is ambiguous as the discrimination plots intersect 
(Figure <span class="InternalRef"><a href="#Fig15">15</a></span>). A 
major advantage of this approach is that no gap penalties are required, 
with the result that once the databases have been constructed, a new 
function can be tested in seconds rather than the days or weeks that 
were needed with the earlier methodology. However, some caveats are in 
order. We are using PSI-BLAST as a gold standard for creating profiles, 
but PSI-BLAST may introduce biases both due to its selection of 
sequences for inclusion in the profile and due to errors in alignments 
of those sequences to the query. If the profile function will be used to
 align PSI-BLAST profiles, then this is an appropriate experimental 
design. But in the case of multiple sequence alignment, where profiles 
are produced iteratively by the profile function itself, the results may
 not be directly applicable. We also note that any monotonic 
transformation of the profile function leaves the discriminator plot 
unchanged as it does not change the sort order of the scores. (A 
monotonic transformation is F' = <em xmlns="" class="EmphasisTypeItalic">m</em>(F) where <em xmlns="" class="EmphasisTypeItalic">m</em>(<em xmlns="" class="EmphasisTypeItalic">x</em>)
 is a monotonically increasing function). However, a monotonic 
transformation may change the alignments produced by a profile function,
 so we can regard high discrimination as a necessary but not sufficient 
condition for a good profile function. One can turn this into a virtue 
by noting that the discrimination plot allows the relative probability 
of true versus false to be determined from a score. It is therefore 
possible to numerically determine a log-odds function from the 
discrimination plot, which can be evaluated by table look-up. Using 
discrimination plots for PP2, we found the optimal transformation for LE
 to be close to linear, in contrast to other functions we tried, 
including PSP (results not shown). This observation further encouraged 
us to explore the performance of LE in an MSA algorithm. Testing on 
multiple alignment benchmarks we find LE to give superior results on 
BAliBASE, but statistically indistinguishable results on other databases
 (results not shown). MUSCLE therefore uses LE as the default choice as 
it sometimes gives better results but has not been observed to give 
lower average accuracy on any of our tests. It is also useful to 
introduce a method with a distinctively different scoring scheme as an 
alternative that may give better results on some input data and may 
provide unique features for incorporation into jury or consensus 
systems. One drawback of LE is its relatively slow performance due to 
the need to compute a logarithm for each cell of the dynamic programming
 matrix.<figure xmlns="" class="Figure" id="Fig14"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig14_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig14_HTML.jpg" alt="Figure 14" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig14_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig14_HTML.jpg" alt="Figure 14"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 14</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Discrimination plot for PP2.</strong> The <em class="EmphasisTypeItalic">x</em> axis is the number of true column pairs with scores ≤ <em class="EmphasisTypeItalic">S</em> for some value <em class="EmphasisTypeItalic">S</em>, as a fraction of the total number of true pairs; the <em class="EmphasisTypeItalic">y</em> axis is the number of false column pairs with scores ≤ <em class="EmphasisTypeItalic">S</em>,
 as a fraction of the total number of false pairs. The databases were 
constructed from the PP2 test set. Shown are discrimination plots for 
the log-expectation (LE), log-average (LA), Yona-Levitt (YL), LAMA, and 
profile sum of pairs (PSP) functions. The LE function shows higher 
discrimination over the entire range of scores than any other function 
we tested (complete results not shown). The poor performance of the 
"standard" PSP function is striking. PSP displays negative 
discrimination over some of its range where it falls below the diagonal 
(dashed line).</p>
                      </div></figcaption></figure><figure xmlns="" class="Figure" id="Fig15"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig15_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig15_HTML.jpg" alt="Figure 15" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig15_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig15_HTML.jpg" alt="Figure 15"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 15</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Discrimination plot for PP.</strong>
 This is similar to Figure 13, except that the database was generated 
from the PP test set. Here we see an ambiguous result as the 
discrimination plots for LE and PSP intersect.</p>
                      </div></figcaption></figure></div>
              </section><section id="Sec49" class="Section2 RenderAsSection2">
                <h3 class="Heading">Complexity of MUSCLE</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The complexity of MUSCLE is summarized in Table <span class="InternalRef"><a href="#Tab2">2</a></span>. We assume <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> = O(<em xmlns="" class="EmphasisTypeItalic">L</em> + <em xmlns="" class="EmphasisTypeItalic">N</em>), the e-string construction for the root alignment, and a fixed number of refinement iterations.<figure xmlns="" id="Tab2" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 2</span>
                        <p xmlns="" class="SimplePara">Complexity of MUSCLE. Here we show the big-O asymptotic complexity of the elements of MUSCLE as a function of <em class="EmphasisTypeItalic">L</em>, the typical sequence length, and <em class="EmphasisTypeItalic">N</em>, the number of sequences, retaining the highest-order terms in <em class="EmphasisTypeItalic">N</em> with <em class="EmphasisTypeItalic">L</em> fixed and vice versa.</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Step</p></th><th><p xmlns="" class="SimplePara">O(Space)</p></th><th><p xmlns="" class="SimplePara">O(Time)</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">K</em>-mer distance matrix</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em></p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                              <em class="EmphasisTypeItalic">L</em>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">UPGMA</p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                            </p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Progressive (one iteration)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = <em class="EmphasisTypeItalic">NL</em> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Progressive (root alignment)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">NL</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> = <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">NL</em></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">NL</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> log <em class="EmphasisTypeItalic">N</em> = <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> log <em class="EmphasisTypeItalic">N</em> + <em class="EmphasisTypeItalic">NL</em> log <em class="EmphasisTypeItalic">N</em></p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Progressive (<em class="EmphasisTypeItalic">N</em> iterations + root)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">NL</em> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">3</sup> + <em class="EmphasisTypeItalic">NL</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Refinement (one edge)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">NL</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> + <em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup><em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub> + <em class="EmphasisTypeItalic">L</em><sub xmlns="http://www.w3.org/1999/xhtml">P</sub><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> = <em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">3</sup><em class="EmphasisTypeItalic">+ L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">Refinement (<em class="EmphasisTypeItalic">N</em> edges)</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara">
                              <em class="EmphasisTypeItalic">N</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">4</sup>
                              <em class="EmphasisTypeItalic">+ NL</em>
                              <sup xmlns="http://www.w3.org/1999/xhtml">2</sup>
                            </p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">TOTAL</p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup> + <em class="EmphasisTypeItalic">L</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td><td><p xmlns="" class="SimplePara"><em class="EmphasisTypeItalic">N</em><sup xmlns="http://www.w3.org/1999/xhtml">4</sup> + <em class="EmphasisTypeItalic">NL</em><sup xmlns="http://www.w3.org/1999/xhtml">2</sup></p></td></tr></tbody></table></div></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec50" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Results" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Results">Results</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Results" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">MUSCLE
 offers a variety of options that offer different trade-offs between 
speed and accuracy. In the following, we report speed and accuracy 
results for three sets of options: (1) the full MUSCLE algorithm 
including Stages 1, 2 and 3 with default options; (2) Stages 1 and 2 
only, using default options (MUSCLE-prog); and (3) Stage 1 only using 
the fastest possible options (MUSCLE-fast), which are as follows: <em xmlns="" class="EmphasisTypeItalic">F</em><sup>Binary</sup> is used as a distance measure (Equation 2), the PSP profile function is used, and diagonal finding is enabled.</p><section id="Sec51" class="Section2 RenderAsSection2">
                <h3 class="Heading">Alignment accuracy</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In Tables <span class="InternalRef"><a href="#Tab3">3</a></span> and <span class="InternalRef"><a href="#Tab4">4</a></span>
 we report the speed and accuracy of MUSCLE v3.3, CLUSTALW v1.82, 
Progressive POA, a recently published method that is claimed to be 10 to
 30 times faster than CLUSTALW for large alignments [<span class="CitationRef"><a href="#CR48" data-toggle="citation">48</a></span>],
 and the MAFFT script FFTNS1 v3.82, the fastest previously published 
method known to us. On the advice of one of the authors of Progressive 
POA, we used command-line options selecting global alignment with 
truncated gap scoring (Catherine Grasso, personal communication). We 
report results both using distance matrices computed by BLAST 
(POA-blast) and using the distance method built into the program (POA). 
We use four sets of reference alignments: BAliBASE v2, PREFAB v3, 
SABmark v1.61, and a version of SMART from July 2000. The accuracy score
 is <em xmlns="" class="EmphasisTypeItalic">Q</em>, the number of 
residue pairs correctly aligned divided by the length of the reference 
alignment. For more discussion of the reference data, assessment 
methodology and a comparison of MUSCLE with T-Coffee and NWNSI, the most
 accurate MAFFT script, see [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].<figure xmlns="" id="Tab3" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 3</span>
                        <p xmlns="" class="SimplePara">Accuracy scores. The average accuracy, measured by the <em class="EmphasisTypeItalic">Q</em> score, is reported for each method on each set of reference alignments.</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Method</p></th><th><p xmlns="" class="SimplePara">PREFAB</p></th><th><p xmlns="" class="SimplePara">BAliBASE</p></th><th><p xmlns="" class="SimplePara">SABmark</p></th><th><p xmlns="" class="SimplePara">SMART</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">MUSCLE</p></td><td><p xmlns="" class="SimplePara">0.648</p></td><td><p xmlns="" class="SimplePara">0.896</p></td><td><p xmlns="" class="SimplePara">0.430</p></td><td><p xmlns="" class="SimplePara">0.856</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MUSCLE-prog</p></td><td><p xmlns="" class="SimplePara">0.634</p></td><td><p xmlns="" class="SimplePara">0.883</p></td><td><p xmlns="" class="SimplePara">0.427</p></td><td><p xmlns="" class="SimplePara">0.837</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">FFTNS1</p></td><td><p xmlns="" class="SimplePara">0.619</p></td><td><p xmlns="" class="SimplePara">0.844</p></td><td><p xmlns="" class="SimplePara">0.376</p></td><td><p xmlns="" class="SimplePara">0.815</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MUSCLE-fast</p></td><td><p xmlns="" class="SimplePara">0.616</p></td><td><p xmlns="" class="SimplePara">0.849</p></td><td><p xmlns="" class="SimplePara">0.396</p></td><td><p xmlns="" class="SimplePara">0.813</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CLUSTALW</p></td><td><p xmlns="" class="SimplePara">0.588</p></td><td><p xmlns="" class="SimplePara">0.860</p></td><td><p xmlns="" class="SimplePara">0.404</p></td><td><p xmlns="" class="SimplePara">0.823</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">POA-blast</p></td><td><p xmlns="" class="SimplePara">0.577</p></td><td><p xmlns="" class="SimplePara">0.839</p></td><td><p xmlns="" class="SimplePara">0.352</p></td><td><p xmlns="" class="SimplePara">0.788</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">POA</p></td><td><p xmlns="" class="SimplePara">0.576</p></td><td><p xmlns="" class="SimplePara">0.834</p></td><td><p xmlns="" class="SimplePara">0.280</p></td><td><p xmlns="" class="SimplePara">0.797</p></td></tr></tbody></table></div></figure><figure xmlns="" id="Tab4" class="FigureTable"><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Table 4</span>
                        <p xmlns="" class="SimplePara">CPU times. The 
total CPU time required to create all alignments in each test set, 
measured in seconds on a 2.5 GHz Pentium 4 desktop computer.</p>
                      </div></figcaption><div class="Table"><table xmlns="http://www.w3.org/1999/xhtml" border="1"><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><colgroup><col></colgroup><thead><tr class="header"><th><p xmlns="" class="SimplePara">Method</p></th><th><p xmlns="" class="SimplePara">PREFAB</p></th><th><p xmlns="" class="SimplePara">BAliBASE</p></th><th><p xmlns="" class="SimplePara">SABmark</p></th><th><p xmlns="" class="SimplePara">SMART</p></th></tr></thead><tbody><tr class="noclass"><td><p xmlns="" class="SimplePara">MUSCLE-fast</p></td><td><p xmlns="" class="SimplePara">540</p></td><td><p xmlns="" class="SimplePara">11</p></td><td><p xmlns="" class="SimplePara">45</p></td><td><p xmlns="" class="SimplePara">30</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">FFTNS1</p></td><td><p xmlns="" class="SimplePara">720</p></td><td><p xmlns="" class="SimplePara">16</p></td><td><p xmlns="" class="SimplePara">70</p></td><td><p xmlns="" class="SimplePara">46</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MUSCLE-prog</p></td><td><p xmlns="" class="SimplePara">3,000</p></td><td><p xmlns="" class="SimplePara">52</p></td><td><p xmlns="" class="SimplePara">429</p></td><td><p xmlns="" class="SimplePara">180</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">MUSCLE</p></td><td><p xmlns="" class="SimplePara">11,000</p></td><td><p xmlns="" class="SimplePara">81</p></td><td><p xmlns="" class="SimplePara">1,500</p></td><td><p xmlns="" class="SimplePara">560</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">POA-blast</p></td><td><p xmlns="" class="SimplePara">11,000</p></td><td><p xmlns="" class="SimplePara">90</p></td><td><p xmlns="" class="SimplePara">290</p></td><td><p xmlns="" class="SimplePara">670</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">CLUSTALW</p></td><td><p xmlns="" class="SimplePara">15,000</p></td><td><p xmlns="" class="SimplePara">160</p></td><td><p xmlns="" class="SimplePara">210</p></td><td><p xmlns="" class="SimplePara">480</p></td></tr><tr class="noclass"><td><p xmlns="" class="SimplePara">POA</p></td><td><p xmlns="" class="SimplePara">24,000</p></td><td><p xmlns="" class="SimplePara">130</p></td><td><p xmlns="" class="SimplePara">380</p></td><td><p xmlns="" class="SimplePara">880</p></td></tr></tbody></table></div></figure></div>
              </section><section id="Sec52" class="Section2 RenderAsSection2">
                <h3 class="Heading">Execution speed</h3>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">To
 compare speeds for a larger number of sequences, we created a test set 
by using PSI-BLAST to search the NCBI non-redundant protein sequence 
database for hits to dienoyl-coa isomerase (1dci in the Protein Data 
Bank [<span class="CitationRef"><a href="#CR49" data-toggle="citation">49</a></span>]),
 selecting the highest-scoring 1,000 sequences. This set of sequences 
had average length 282, maximum length 454 and average pair-wise 
identity 20%. We aligned randomly chosen subsets of from 200 to 1,000 
sequences with each program and noted the total execution time. In the 
case of 1,000 sequences, the resulting alignments had from 1,100 from 
1,400 columns, confirming that it is unrealistic to assume that <em xmlns="" class="EmphasisTypeItalic">L</em><sub>P</sub> is O(<em xmlns="" class="EmphasisTypeItalic">L</em>). Results are shown in Figure <span class="InternalRef"><a href="#Fig16">16</a></span>.
 We have previously shown that MUSCLE-prog is faster than FFTNS1 on a 
set of 5,000 sequences, for which we estimated that CLUSTALW would 
require approximately one year [<span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>].
 In this test, MUSCLE-fast is approximately 3× faster than FFTNS1 for 
200 sequences, and 5× faster for 1,000 sequences. This trend continues 
for larger numbers of sequences (complete results not shown), showing 
that MUSCLE-fast has lower asymptotic complexity, due largely to the use
 of additive profiles for progressive alignment compared with the 
profile matrices constructed by FFTNS1.<figure xmlns="" class="Figure" id="Fig16"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig16_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2004_Article_229_Fig16_HTML.jpg" alt="Figure 16" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig16_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_Article_229_Fig16_HTML.jpg" alt="Figure 16"></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 16</span>
                        <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Execution time as a function of</strong> <span class="EmphasisTypeBoldItalic">N</span> <strong class="EmphasisTypeBold">.</strong> This plot shows the execution time as a function of <em class="EmphasisTypeItalic">N</em>
 (number of sequences) for the tested alignment methods. Input data is 
from 200 to 1,000 sequences in increments of 200. Average sequence 
length is 282, maximum length 454.</p>
                      </div></figcaption></figure></div>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec53" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusions" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusions">Conclusions</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusions" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">MUSCLE
 demonstrates improvements in accuracy and reductions in computational 
complexity by exploiting a range of existing and new algorithmic 
techniques. While the design–typically for practical multiple sequence 
alignment tools–arguably lacks elegance and theoretical coherence, 
useful improvements were achieved through a number of factors. Most 
important of these were selection of heuristics, close attention to 
details of the implementation, and careful evaluation of the impact of 
different elements of the algorithm on speed and accuracy. MUSCLE 
enables high-throughput applications to achieve average accuracy 
comparable to the most accurate tools previously available, which we 
expect to be increasingly important in view of the continuing rapid 
growth in sequence data.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec54" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Availabilityandrequirements" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Availabilityandrequirements">Availability and requirements</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Availabilityandrequirements" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">MUSCLE
 is a command-line program written in a conservative subset of C++. At 
the time of writing, MUSCLE has been successfully ported to 32-bit 
Windows, 32-bit Intel architecture Linux, Solaris, Macintosh OSX and the
 64-bit HP Alpha Tru64 platform. MUSCLE is donated to the public domain.
 Source code and executable files are freely available at <span class="ExternalRef"><a href="http://www.drive5.com/muscle"><span class="RefSource">http://www.drive5.com/muscle</span></a></span>.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Authors' original submitted files for images</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authors’ original submitted files for images.<div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM1_ESM.doc">12859_2004_229_MOESM1_ESM.doc</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 1</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM2_ESM.doc">12859_2004_229_MOESM2_ESM.doc</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 2</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM3_ESM.pdf">12859_2004_229_MOESM3_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 3</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM4">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM4_ESM.pdf">12859_2004_229_MOESM4_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 4</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM5">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM5_ESM.doc">12859_2004_229_MOESM5_ESM.doc</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 5</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM6">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM6_ESM.doc">12859_2004_229_MOESM6_ESM.doc</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 6</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM7">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM7_ESM.pdf">12859_2004_229_MOESM7_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 7</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM8">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM8_ESM.pdf">12859_2004_229_MOESM8_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 8</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM9">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM9_ESM.pdf">12859_2004_229_MOESM9_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 9</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM10">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM10_ESM.pdf">12859_2004_229_MOESM10_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 10</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM11">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM11_ESM.pdf">12859_2004_229_MOESM11_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 11</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM12">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM12_ESM.pdf">12859_2004_229_MOESM12_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 12</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM13">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM13_ESM.pdf">12859_2004_229_MOESM13_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 13</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM14">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM14_ESM.tiff">12859_2004_229_MOESM14_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 14</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM15">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM15_ESM.tiff">12859_2004_229_MOESM15_ESM.tiff</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 15</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM16">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-5-113/MediaObjects/12859_2004_229_MOESM16_ESM.pdf">12859_2004_229_MOESM16_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 16</span>
                        </div></div>
                  </div></div>
              
            </aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Department of Plant and Microbial Biology, University of California, 461 Koshland Hall, Berkeley, CA&nbsp;94720-3102, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Notredame C: <strong class="EmphasisTypeBold">Recent progress in multiple sequence alignment: a survey.</strong> <em class="EmphasisTypeItalic">Pharmacogenomics</em> 2002, <strong class="EmphasisTypeBold">3</strong>(1):131–144.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1517/14622416.3.1.131"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11966409"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Recent%20progress%20in%20multiple%20sequence%20alignment%3A%20a%20survey&amp;author=C.%20Notredame&amp;journal=Pharmacogenomics&amp;volume=3&amp;issue=1&amp;pages=131-144&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edgar RC: <strong class="EmphasisTypeBold">MUSCLE: multiple sequence alignment with high accuracy and high throughput.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2004, <strong class="EmphasisTypeBold">32</strong>(5):1792–1797. 10.1093/nar/gkh340</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390337"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkh340"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15034147"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC.%20Edgar&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;issue=5&amp;pages=1792-1797&amp;publication_year=2004&amp;doi=10.1093%2Fnar%2Fgkh340"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wang L, Jiang T: <strong class="EmphasisTypeBold">On the complexity of multiple sequence alignment.</strong> <em class="EmphasisTypeItalic">J Comput Biol</em> 1994, <strong class="EmphasisTypeBold">1</strong>(4):337–348.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1089/cmb.1994.1.337"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8790475"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=On%20the%20complexity%20of%20multiple%20sequence%20alignment&amp;author=L.%20Wang&amp;author=T.%20Jiang&amp;journal=J%20Comput%20Biol&amp;volume=1&amp;issue=4&amp;pages=337-348&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Waterman MS, Smith TF, Beyer WA: <strong class="EmphasisTypeBold">Some biological sequence metrics.</strong> <em class="EmphasisTypeItalic">Adv in Math</em> 1976, <strong class="EmphasisTypeBold">20:</strong> 367–387.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0001-8708%2876%2990202-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Some%20biological%20sequence%20metrics&amp;author=MS.%20Waterman&amp;author=TF.%20Smith&amp;author=WA.%20Beyer&amp;journal=Adv%20in%20Math&amp;volume=20&amp;pages=367-387&amp;publication_year=1976"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lawrence CE, Altschul SF, Boguski MS, Liu JS, Neuwald AF, Wootton JC: <strong class="EmphasisTypeBold">Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment.</strong> <em class="EmphasisTypeItalic">Science</em> 1993, <strong class="EmphasisTypeBold">262</strong>(5131):208–214.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1126/science.8211139"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8211139"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Detecting%20subtle%20sequence%20signals%3A%20a%20Gibbs%20sampling%20strategy%20for%20multiple%20alignment&amp;author=CE.%20Lawrence&amp;author=SF.%20Altschul&amp;author=MS.%20Boguski&amp;author=JS.%20Liu&amp;author=AF.%20Neuwald&amp;author=JC.%20Wootton&amp;journal=Science&amp;volume=262&amp;issue=5131&amp;pages=208-214&amp;publication_year=1993"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hogeweg P, Hesper B: <strong class="EmphasisTypeBold">The alignment of sets of sequences and the construction of phyletic trees: an integrated method.</strong> <em class="EmphasisTypeItalic">J Mol Evol</em> 1984, <strong class="EmphasisTypeBold">20</strong>(2):175–186.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02257378"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6433036"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20alignment%20of%20sets%20of%20sequences%20and%20the%20construction%20of%20phyletic%20trees%3A%20an%20integrated%20method&amp;author=P.%20Hogeweg&amp;author=B.%20Hesper&amp;journal=J%20Mol%20Evol&amp;volume=20&amp;issue=2&amp;pages=175-186&amp;publication_year=1984"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Feng DF, Doolittle RF: <strong class="EmphasisTypeBold">Progressive sequence alignment as a prerequisite to correct phylogenetic trees.</strong> <em class="EmphasisTypeItalic">J Mol Evol</em> 1987, <strong class="EmphasisTypeBold">25</strong>(4):351–360.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1007/BF02603120"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3118049"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Progressive%20sequence%20alignment%20as%20a%20prerequisite%20to%20correct%20phylogenetic%20trees&amp;author=DF.%20Feng&amp;author=RF.%20Doolittle&amp;journal=J%20Mol%20Evol&amp;volume=25&amp;issue=4&amp;pages=351-360&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thompson JD, Higgins DG, Gibson TJ: <strong class="EmphasisTypeBold">CLUSTAL
 W: improving the sensitivity of progressive multiple sequence alignment
 through sequence weighting, position-specific gap penalties and weight 
matrix choice.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1994, <strong class="EmphasisTypeBold">22</strong>(22):4673–4680.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC308517"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/22.22.4673"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7984417"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=JD.%20Thompson&amp;author=DG.%20Higgins&amp;author=TJ.%20Gibson&amp;journal=Nucleic%20Acids%20Res&amp;volume=22&amp;issue=22&amp;pages=4673-4680&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Notredame C, Higgins DG, Heringa J: <strong class="EmphasisTypeBold">T-Coffee: A novel method for fast and accurate multiple sequence alignment.</strong> <em class="EmphasisTypeItalic">J Mol Biol</em> 2000, <strong class="EmphasisTypeBold">302</strong>(1):205–217. 10.1006/jmbi.2000.4042</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/jmbi.2000.4042"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10964570"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=T-Coffee%3A%20A%20novel%20method%20for%20fast%20and%20accurate%20multiple%20sequence%20alignment&amp;author=C.%20Notredame&amp;author=DG.%20Higgins&amp;author=J.%20Heringa&amp;journal=J%20Mol%20Biol&amp;volume=302&amp;issue=1&amp;pages=205-217&amp;publication_year=2000&amp;doi=10.1006%2Fjmbi.2000.4042"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Bahr A, Thompson JD, Thierry JC, Poch O: <strong class="EmphasisTypeBold">BAliBASE (Benchmark Alignment dataBASE): enhancements for repeats, transmembrane sequences and circular permutations.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2001, <strong class="EmphasisTypeBold">29</strong>(1):323–326. 10.1093/nar/29.1.323</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC29792"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/29.1.323"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11125126"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BAliBASE%20%28Benchmark%20Alignment%20dataBASE%29%3A%20enhancements%20for%20repeats%2C%20transmembrane%20sequences%20and%20circular%20permutations&amp;author=A.%20Bahr&amp;author=JD.%20Thompson&amp;author=JC.%20Thierry&amp;author=O.%20Poch&amp;journal=Nucleic%20Acids%20Res&amp;volume=29&amp;issue=1&amp;pages=323-326&amp;publication_year=2001&amp;doi=10.1093%2Fnar%2F29.1.323"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thompson JD, Plewniak F, Poch O: <strong class="EmphasisTypeBold">BAliBASE: a benchmark alignment database for the evaluation of multiple alignment programs.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 1999, <strong class="EmphasisTypeBold">15</strong>(1):87–88. 10.1093/bioinformatics/15.1.87</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/15.1.87"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10068696"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=BAliBASE%3A%20a%20benchmark%20alignment%20database%20for%20the%20evaluation%20of%20multiple%20alignment%20programs&amp;author=JD.%20Thompson&amp;author=F.%20Plewniak&amp;author=O.%20Poch&amp;journal=Bioinformatics&amp;volume=15&amp;issue=1&amp;pages=87-88&amp;publication_year=1999&amp;doi=10.1093%2Fbioinformatics%2F15.1.87"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hirosawa M, Totoki Y, Hoshida M, Ishikawa M: <strong class="EmphasisTypeBold">Comprehensive study on iterative algorithms of multiple sequence alignment.</strong> <em class="EmphasisTypeItalic">Comput Appl Biosci</em> 1995, <strong class="EmphasisTypeBold">11</strong>(1):13–18.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7796270"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comprehensive%20study%20on%20iterative%20algorithms%20of%20multiple%20sequence%20alignment&amp;author=M.%20Hirosawa&amp;author=Y.%20Totoki&amp;author=M.%20Hoshida&amp;author=M.%20Ishikawa&amp;journal=Comput%20Appl%20Biosci&amp;volume=11&amp;issue=1&amp;pages=13-18&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gotoh O: <strong class="EmphasisTypeBold">Significant
 improvement in accuracy of multiple protein sequence alignments by 
iterative refinement as assessed by reference to structural alignments.</strong> <em class="EmphasisTypeItalic">J Mol Biol</em> 1996, <strong class="EmphasisTypeBold">264</strong>(4):823–838. 10.1006/jmbi.1996.0679</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/jmbi.1996.0679"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8980688"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Significant%20improvement%20in%20accuracy%20of%20multiple%20protein%20sequence%20alignments%20by%20iterative%20refinement%20as%20assessed%20by%20reference%20to%20structural%20alignments&amp;author=O.%20Gotoh&amp;journal=J%20Mol%20Biol&amp;volume=264&amp;issue=4&amp;pages=823-838&amp;publication_year=1996&amp;doi=10.1006%2Fjmbi.1996.0679"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Katoh K, Misawa K, Kuma K, Miyata T: <strong class="EmphasisTypeBold">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2002, <strong class="EmphasisTypeBold">30</strong>(14):3059–3066. 10.1093/nar/gkf436</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC135756"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkf436"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12136088"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;issue=14&amp;pages=3059-3066&amp;publication_year=2002&amp;doi=10.1093%2Fnar%2Fgkf436"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Van Walle I, Lasters I, Wyns L: <strong class="EmphasisTypeBold">Align-m–a new algorithm for multiple alignment of highly divergent sequences.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 2004.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Align-m%E2%80%93a%20new%20algorithm%20for%20multiple%20alignment%20of%20highly%20divergent%20sequences&amp;author=I.%20Van%20Walle&amp;author=I.%20Lasters&amp;author=L.%20Wyns&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schultz J, Copley RR, Doerks T, Ponting CP, Bork P: <strong class="EmphasisTypeBold">SMART: a web-based tool for the study of genetically mobile domains.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2000, <strong class="EmphasisTypeBold">28</strong>(1):231–234. 10.1093/nar/28.1.231</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102444"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/28.1.231"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10592234"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SMART%3A%20a%20web-based%20tool%20for%20the%20study%20of%20genetically%20mobile%20domains&amp;author=J.%20Schultz&amp;author=RR.%20Copley&amp;author=T.%20Doerks&amp;author=CP.%20Ponting&amp;author=P.%20Bork&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;issue=1&amp;pages=231-234&amp;publication_year=2000&amp;doi=10.1093%2Fnar%2F28.1.231"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Ponting CP, Schultz J, Milpetz F, Bork P: <strong class="EmphasisTypeBold">SMART: identification and annotation of domains from signalling and extracellular protein sequences.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1999, <strong class="EmphasisTypeBold">27</strong>(1):229–232. 10.1093/nar/27.1.229</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC148142"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/27.1.229"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9847187"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SMART%3A%20identification%20and%20annotation%20of%20domains%20from%20signalling%20and%20extracellular%20protein%20sequences&amp;author=CP.%20Ponting&amp;author=J.%20Schultz&amp;author=F.%20Milpetz&amp;author=P.%20Bork&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;issue=1&amp;pages=229-232&amp;publication_year=1999&amp;doi=10.1093%2Fnar%2F27.1.229"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Schultz J, Milpetz F, Bork P, Ponting CP: <strong class="EmphasisTypeBold">SMART, a simple modular architecture research tool: identification of signaling domains.</strong> <em class="EmphasisTypeItalic">Proc Natl Acad Sci U S A</em> 1998, <strong class="EmphasisTypeBold">95</strong>(11):5857–5864. 10.1073/pnas.95.11.5857</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC34487"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.95.11.5857"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9600884"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=SMART%2C%20a%20simple%20modular%20architecture%20research%20tool%3A%20identification%20of%20signaling%20domains&amp;author=J.%20Schultz&amp;author=F.%20Milpetz&amp;author=P.%20Bork&amp;author=CP.%20Ponting&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;volume=95&amp;issue=11&amp;pages=5857-5864&amp;publication_year=1998&amp;doi=10.1073%2Fpnas.95.11.5857"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Vinga S, Almeida J: <strong class="EmphasisTypeBold">Alignment-free sequence comparison-a review.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 2003, <strong class="EmphasisTypeBold">19</strong>(4):513–523. 10.1093/bioinformatics/btg005</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg005"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12611807"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Alignment-free%20sequence%20comparison-a%20review&amp;author=S.%20Vinga&amp;author=J.%20Almeida&amp;journal=Bioinformatics&amp;volume=19&amp;issue=4&amp;pages=513-523&amp;publication_year=2003&amp;doi=10.1093%2Fbioinformatics%2Fbtg005"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Dayhoff MO, Schwartz RM, Orcutt BC: <strong class="EmphasisTypeBold">A model of evolutionary change in proteins in Atlas of protein sequence and structure,.</strong> <em class="EmphasisTypeItalic">(Edited by: Dayhoff MO, Ech RV).</em> Maryland: National Biomedical Research Foundation 1978.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=MO.%20Dayhoff&amp;author=RM.%20Schwartz&amp;author=BC.%20Orcutt&amp;publication_year=1978"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edgar RC: <strong class="EmphasisTypeBold">Local homology recognition and distance measures in linear time using compressed amino acid alphabets.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2004, <strong class="EmphasisTypeBold">32</strong>(1):380–385. 10.1093/nar/gkh180</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC373290"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkh180"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14729922"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Local%20homology%20recognition%20and%20distance%20measures%20in%20linear%20time%20using%20compressed%20amino%20acid%20alphabets&amp;author=RC.%20Edgar&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;issue=1&amp;pages=380-385&amp;publication_year=2004&amp;doi=10.1093%2Fnar%2Fgkh180"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kimura M: <strong class="EmphasisTypeBold">The neutral theory of molecular evolution.</strong> Cambridge University Press 1983.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/CBO9780511623486"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=M.%20Kimura&amp;publication_year=1983"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Saitou N, Nei M: <strong class="EmphasisTypeBold">The neighbor-joining method: a new method for reconstructing phylogenetic trees.</strong> <em class="EmphasisTypeItalic">Mol Biol Evol</em> 1987, <strong class="EmphasisTypeBold">4</strong>(4):406–425.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3447015"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;journal=Mol%20Biol%20Evol&amp;volume=4&amp;issue=4&amp;pages=406-425&amp;publication_year=1987"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sneath PHA, Sokal RR: <strong class="EmphasisTypeBold">Numerical taxonomy.</strong> San Francisco: Freeman 1973.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=PHA.%20Sneath&amp;author=RR.%20Sokal&amp;publication_year=1973"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thompson JD, Higgins DG, Gibson TJ: <strong class="EmphasisTypeBold">Improved sensitivity of profile searches through the use of sequence weights and gap excision.</strong> <em class="EmphasisTypeItalic">Comput Appl Biosci</em> 1994, <strong class="EmphasisTypeBold">10</strong>(1):19–29.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8193951"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Improved%20sensitivity%20of%20profile%20searches%20through%20the%20use%20of%20sequence%20weights%20and%20gap%20excision&amp;author=JD.%20Thompson&amp;author=DG.%20Higgins&amp;author=TJ.%20Gibson&amp;journal=Comput%20Appl%20Biosci&amp;volume=10&amp;issue=1&amp;pages=19-29&amp;publication_year=1994"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Henikoff S, Henikoff JG: <strong class="EmphasisTypeBold">Position-based sequence weights.</strong> <em class="EmphasisTypeItalic">J Mol Biol</em> 1994, <strong class="EmphasisTypeBold">243</strong>(4):574–578. 10.1016/0022-2836(94)90032-9</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0022-2836%2894%2990032-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7966282"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Position-based%20sequence%20weights&amp;author=S.%20Henikoff&amp;author=JG.%20Henikoff&amp;journal=J%20Mol%20Biol&amp;volume=243&amp;issue=4&amp;pages=574-578&amp;publication_year=1994&amp;doi=10.1016%2F0022-2836%2894%2990032-9"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ: <strong class="EmphasisTypeBold">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1997, <strong class="EmphasisTypeBold">25</strong>(17):3389–3402. 10.1093/nar/25.17.3389</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC146917"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/25.17.3389"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9254694"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=SF.%20Altschul&amp;author=TL.%20Madden&amp;author=AA.%20Schaffer&amp;author=J.%20Zhang&amp;author=Z.%20Zhang&amp;author=W.%20Miller&amp;author=DJ.%20Lipman&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;issue=17&amp;pages=3389-3402&amp;publication_year=1997&amp;doi=10.1093%2Fnar%2F25.17.3389"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gerstein M, Sonnhammer EL, Chothia C: <strong class="EmphasisTypeBold">Volume changes in protein evolution.</strong> <em class="EmphasisTypeItalic">J Mol Biol</em> 1994, <strong class="EmphasisTypeBold">236</strong>(4):1067–1078. 10.1016/0022-2836(94)90012-4</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0022-2836%2894%2990012-4"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8120887"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Volume%20changes%20in%20protein%20evolution&amp;author=M.%20Gerstein&amp;author=EL.%20Sonnhammer&amp;author=C.%20Chothia&amp;journal=J%20Mol%20Biol&amp;volume=236&amp;issue=4&amp;pages=1067-1078&amp;publication_year=1994&amp;doi=10.1016%2F0022-2836%2894%2990012-4"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gotoh O: <strong class="EmphasisTypeBold">A weighting system and algorithm for aligning many phylogenetically related sequences.</strong> <em class="EmphasisTypeItalic">Comput Appl Biosci</em> 1995, <strong class="EmphasisTypeBold">11</strong>(5):543–551.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8590178"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20weighting%20system%20and%20algorithm%20for%20aligning%20many%20phylogenetically%20related%20sequences&amp;author=O.%20Gotoh&amp;journal=Comput%20Appl%20Biosci&amp;volume=11&amp;issue=5&amp;pages=543-551&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edgar RC, Sjolander K: <strong class="EmphasisTypeBold">A comparison of scoring functions for protein sequence profile alignment.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 2004, <em class="EmphasisTypeItalic">in press</em>.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20scoring%20functions%20for%20protein%20sequence%20profile%20alignment&amp;author=RC.%20Edgar&amp;author=K.%20Sjolander&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR31" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sjolander K, Karplus K, Brown M, Hughey R, Krogh A, Mian IS, Haussler D: <strong class="EmphasisTypeBold">Dirichlet mixtures: a method for improved detection of weak but significant protein sequence homology.</strong> <em class="EmphasisTypeItalic">Comput Appl Biosci</em> 1996, <strong class="EmphasisTypeBold">12</strong>(4):327–345.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8902360"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Dirichlet%20mixtures%3A%20a%20method%20for%20improved%20detection%20of%20weak%20but%20significant%20protein%20sequence%20homology&amp;author=K.%20Sjolander&amp;author=K.%20Karplus&amp;author=M.%20Brown&amp;author=R.%20Hughey&amp;author=A.%20Krogh&amp;author=IS.%20Mian&amp;author=D.%20Haussler&amp;journal=Comput%20Appl%20Biosci&amp;volume=12&amp;issue=4&amp;pages=327-345&amp;publication_year=1996"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR32" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Altschul SF: <strong class="EmphasisTypeBold">Amino acid substitution matrices from an information theoretic perspective.</strong> <em class="EmphasisTypeItalic">J Mol Biol</em> 1991, <strong class="EmphasisTypeBold">219</strong>(3):555–565.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0022-2836%2891%2990193-A"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2051488"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Amino%20acid%20substitution%20matrices%20from%20an%20information%20theoretic%20perspective&amp;author=SF.%20Altschul&amp;journal=J%20Mol%20Biol&amp;volume=219&amp;issue=3&amp;pages=555-565&amp;publication_year=1991"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR33" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Jones DT, Taylor WR, Thornton JM: <strong class="EmphasisTypeBold">The rapid generation of mutation data matrices from protein sequences.</strong> <em class="EmphasisTypeItalic">Comput Appl Biosci</em> 1992, <strong class="EmphasisTypeBold">8</strong>(3):275–282.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1633570"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20rapid%20generation%20of%20mutation%20data%20matrices%20from%20protein%20sequences&amp;author=DT.%20Jones&amp;author=WR.%20Taylor&amp;author=JM.%20Thornton&amp;journal=Comput%20Appl%20Biosci&amp;volume=8&amp;issue=3&amp;pages=275-282&amp;publication_year=1992"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR34" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Muller T, Spang R, Vingron M: <strong class="EmphasisTypeBold">Estimating
 amino acid substitution models: a comparison of Dayhoff's estimator, 
the resolvent approach and a maximum likelihood method.</strong> <em class="EmphasisTypeItalic">Mol Biol Evol</em> 2002, <strong class="EmphasisTypeBold">19</strong>(1):8–13.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/oxfordjournals.molbev.a003985"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11752185"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Estimating%20amino%20acid%20substitution%20models%3A%20a%20comparison%20of%20Dayhoff%27s%20estimator%2C%20the%20resolvent%20approach%20and%20a%20maximum%20likelihood%20method&amp;author=T.%20Muller&amp;author=R.%20Spang&amp;author=M.%20Vingron&amp;journal=Mol%20Biol%20Evol&amp;volume=19&amp;issue=1&amp;pages=8-13&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR35" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">von Ohsen N, Zimmer R: <strong class="EmphasisTypeBold">Improving profile-profile alignment via log average scoring.</strong> <em class="EmphasisTypeItalic">In: Algorithms in Bioinformatics, First International Workshop, WABI 2001</em> <em class="EmphasisTypeItalic">(Edited by: Gascuel O, Moret BME).</em> Berlin: Springer-Verlag 2001, <strong class="EmphasisTypeBold">2149:</strong> 11–26.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Improving%20profile-profile%20alignment%20via%20log%20average%20scoring&amp;author=N.%20von%20Ohsen&amp;author=R.%20Zimmer&amp;pages=11-26&amp;publication_year=2001"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR36" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thompson JD, Plewniak F, Poch O: <strong class="EmphasisTypeBold">A comprehensive comparison of multiple sequence alignment programs.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1999, <strong class="EmphasisTypeBold">27</strong>(13):2682–2690. 10.1093/nar/27.13.2682</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC148477"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/27.13.2682"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10373585"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=A%20comprehensive%20comparison%20of%20multiple%20sequence%20alignment%20programs&amp;author=JD.%20Thompson&amp;author=F.%20Plewniak&amp;author=O.%20Poch&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;issue=13&amp;pages=2682-2690&amp;publication_year=1999&amp;doi=10.1093%2Fnar%2F27.13.2682"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR37" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Myers EW, Miller W: <strong class="EmphasisTypeBold">Optimal alignments in linear space.</strong> <em class="EmphasisTypeItalic">Comput Appl Biosci</em> 1988, <strong class="EmphasisTypeBold">4</strong>(1):11–17.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3382986"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Optimal%20alignments%20in%20linear%20space&amp;author=EW.%20Myers&amp;author=W.%20Miller&amp;journal=Comput%20Appl%20Biosci&amp;volume=4&amp;issue=1&amp;pages=11-17&amp;publication_year=1988"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR38" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Felsenstein J: <strong class="EmphasisTypeBold">Inferring Phylogenies.</strong> Sunderland, Massachusetts: Sinauer Associates 2004.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=J.%20Felsenstein&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR39" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Sauder JM, Arthur JW, Dunbrack RL Jr: <strong class="EmphasisTypeBold">Large-scale comparison of protein sequence alignment algorithms with structure alignments.</strong> <em class="EmphasisTypeItalic">Proteins</em> 2000, <strong class="EmphasisTypeBold">40</strong>(1):6–22. 10.1002/(SICI)1097-0134(20000701)40:1&lt;6::AID-PROT30&gt;3.0.CO;2-7</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1002/%28SICI%291097-0134%2820000701%2940%3A1%3C6%3A%3AAID-PROT30%3E3.0.CO%3B2-7"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10813826"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Large-scale%20comparison%20of%20protein%20sequence%20alignment%20algorithms%20with%20structure%20alignments&amp;author=JM.%20Sauder&amp;author=JW.%20Arthur&amp;author=RL.%20Dunbrack&amp;journal=Proteins&amp;volume=40&amp;issue=1&amp;pages=6-22&amp;publication_year=2000&amp;doi=10.1002%2F%28SICI%291097-0134%2820000701%2940%3A1%3C6%3A%3AAID-PROT30%3E3.0.CO%3B2-7"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR40" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Durbin R, Eddy S, Krogh A, Mitchison G: <strong class="EmphasisTypeBold">Biological sequence analysis.</strong> Cambridge University Press 1998.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1017/CBO9780511790492"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=R.%20Durbin&amp;author=S.%20Eddy&amp;author=A.%20Krogh&amp;author=G.%20Mitchison&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR41" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Setubal J, Meidanis J: <strong class="EmphasisTypeBold">Introduction to computational biology.</strong> Pacific Grove, California: Brooks/Cole 1997.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=J.%20Setubal&amp;author=J.%20Meidanis&amp;publication_year=1997"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR42" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thompson JD, Thierry JC, Poch O: <strong class="EmphasisTypeBold">RASCAL: rapid scanning and correction of multiple sequence alignments.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 2003, <strong class="EmphasisTypeBold">19</strong>(9):1155–1161. 10.1093/bioinformatics/btg133</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/btg133"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12801878"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=RASCAL%3A%20rapid%20scanning%20and%20correction%20of%20multiple%20sequence%20alignments&amp;author=JD.%20Thompson&amp;author=JC.%20Thierry&amp;author=O.%20Poch&amp;journal=Bioinformatics&amp;volume=19&amp;issue=9&amp;pages=1155-1161&amp;publication_year=2003&amp;doi=10.1093%2Fbioinformatics%2Fbtg133"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR43" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pruitt KD, Tatusova T, Maglott DR: <strong class="EmphasisTypeBold">NCBI Reference Sequence project: update and current status.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2003, <strong class="EmphasisTypeBold">31</strong>(1):34–37. 10.1093/nar/gkg111</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC165558"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/gkg111"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12519942"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=NCBI%20Reference%20Sequence%20project%3A%20update%20and%20current%20status&amp;author=KD.%20Pruitt&amp;author=T.%20Tatusova&amp;author=DR.%20Maglott&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;issue=1&amp;pages=34-37&amp;publication_year=2003&amp;doi=10.1093%2Fnar%2Fgkg111"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR44" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edgar RC, Sjolander K: <strong class="EmphasisTypeBold">COACH: profile-profile alignment of protein families using hidden Markov models.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 2004, <em class="EmphasisTypeItalic">in press</em>.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=COACH%3A%20profile-profile%20alignment%20of%20protein%20families%20using%20hidden%20Markov%20models&amp;author=RC.%20Edgar&amp;author=K.%20Sjolander&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR45" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Holm L, Sander C: <strong class="EmphasisTypeBold">Touring protein fold space with Dali/FSSP.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1998, <strong class="EmphasisTypeBold">26</strong>(1):316–319. 10.1093/nar/26.1.316</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC147193"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/26.1.316"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9399863"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Touring%20protein%20fold%20space%20with%20Dali%2FFSSP&amp;author=L.%20Holm&amp;author=C.%20Sander&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;issue=1&amp;pages=316-319&amp;publication_year=1998&amp;doi=10.1093%2Fnar%2F26.1.316"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR46" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Yona G, Levitt M: <strong class="EmphasisTypeBold">Within the twilight zone: a sensitive profile-profile comparison tool based on information theory.</strong> <em class="EmphasisTypeItalic">J Mol Biol</em> 2002, <strong class="EmphasisTypeBold">315</strong>(5):1257–1275. 10.1006/jmbi.2001.5293</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/jmbi.2001.5293"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11827492"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Within%20the%20twilight%20zone%3A%20a%20sensitive%20profile-profile%20comparison%20tool%20based%20on%20information%20theory&amp;author=G.%20Yona&amp;author=M.%20Levitt&amp;journal=J%20Mol%20Biol&amp;volume=315&amp;issue=5&amp;pages=1257-1275&amp;publication_year=2002&amp;doi=10.1006%2Fjmbi.2001.5293"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR47" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Pietrokovski S: <strong class="EmphasisTypeBold">Searching databases of conserved sequence regions by aligning protein multiple-alignments.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 1996, <strong class="EmphasisTypeBold">24</strong>(19):3836–3845. 10.1093/nar/24.19.3836</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC146152"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/24.19.3836"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8871566"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Searching%20databases%20of%20conserved%20sequence%20regions%20by%20aligning%20protein%20multiple-alignments&amp;author=S.%20Pietrokovski&amp;journal=Nucleic%20Acids%20Res&amp;volume=24&amp;issue=19&amp;pages=3836-3845&amp;publication_year=1996&amp;doi=10.1093%2Fnar%2F24.19.3836"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR48" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Grasso C, Lee C: <strong class="EmphasisTypeBold">Combining
 partial order alignment and progressive multiple sequence alignment 
increases alignment speed and scalability to very large alignment 
problems.</strong> <em class="EmphasisTypeItalic">Bioinformatics</em> 2004.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Combining%20partial%20order%20alignment%20and%20progressive%20multiple%20sequence%20alignment%20increases%20alignment%20speed%20and%20scalability%20to%20very%20large%20alignment%20problems&amp;author=C.%20Grasso&amp;author=C.%20Lee&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR49" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE: <strong class="EmphasisTypeBold">The Protein Data Bank.</strong> <em class="EmphasisTypeItalic">Nucleic Acids Res</em> 2000, <strong class="EmphasisTypeBold">28</strong>(1):235–242. 10.1093/nar/28.1.235</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC102472"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/nar/28.1.235"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10592235"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Protein%20Data%20Bank&amp;author=HM.%20Berman&amp;author=J.%20Westbrook&amp;author=Z.%20Feng&amp;author=G.%20Gilliland&amp;author=TN.%20Bhat&amp;author=H.%20Weissig&amp;author=IN.%20Shindyalov&amp;author=PE.%20Bourne&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;issue=1&amp;pages=235-242&amp;publication_year=2000&amp;doi=10.1093%2Fnar%2F28.1.235"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#EdgarlicenseeBioMedCentralLtd" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="EdgarlicenseeBioMedCentralLtd">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="EdgarlicenseeBioMedCentralLtd"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Edgar; licensee BioMed Central Ltd.&nbsp;2004<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published
 under license to BioMed Central Ltd. This is an open-access article 
distributed under the terms of the Creative Commons Attribution License (<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-5-113" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/1471-2105-5-113">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--secondary c-button--full-width c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/1471-2105-5-113" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-5-113?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    76427
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 3012 <a href="http://citations.springer.com/item?doi=10.1186/1471-2105-5-113" rel="noopener" target="_blank">more information</a>
            </li>
        
                                
    <li class="c-list-group__item">
        <p>Altmetric Attention Score: 10</p>
        <div class="altmetric">
            <a class="altmetric__link" href="http://www.altmetric.com/details.php?citation_id=223337&amp;domain=www.biomedcentral.com" target="_blank" rel="noopener" data-track="click" data-track-category="Article Page" data-track-action="Click Altmetric" data-track-label="10.1186/1471-2105-5-113">
                <img class="altmetric__img" src="a.png" alt="Altmetric Attention Score">
            </a>
        </div>
    </li>

                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-5-113">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-5-113">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-5-113">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-5-113">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click" data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/1471-2105-5-113">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-5-113">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/1471-2105-5-113" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2F1471-2105-5-113&amp;volume=5&amp;issue=1&amp;title=MUSCLE%3A+a+multiple+sequence+alignment+method+with+reduced+time+and+space+complexity&amp;author_name=Robert+C+Edgar&amp;start_page=1&amp;end_page=19">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Bioinformatics</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1471-2105</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:bmcbioinformatics@biomedcentral.com">bmcbioinformatics@biomedcentral.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1471-2105-5-113.png" alt="">

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com/">
                    <img src="logo-bmc-series-9977e4f82f.svg" alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral" class="u-text-inherit" data-track="click" data-track-action="Click Twitter link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral" class="u-text-inherit" data-track="click" data-track-action="Click Facebook link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/" class="u-text-inherit" data-track="click" data-track-action="Click Google Plus link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral" class="u-text-inherit" data-track="click" data-track-action="Click Weibo link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="https://www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo">
        <p data-test="copyright">© 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com/" data-track="click" data-track-action="Click Springer Nature link" data-track-category="Corporate footer" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

        

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>

    


<script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(23);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var d=[],e="",g=[],h=a.split(";");for(a=0;a<h.length;++a){void 0!==k&&(e=k);var b=h[a].split("\x3d");var k=b[0];k!==e&&(0<e.length&&0<d.length&&g.push([e,d]),d=[]);if(2===b.length&&""!==b[0]&&""!==b[1]){var c=b[1].split(",");for(b=0;b<c.length;++b)d.push(c[b])}}0<k.length&&0<d.length&&g.push([k,d]);return g}
function splitSizes(a){var d=[];if(null!==a){var e=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<e.length;++a){var g=e[a].split("x");var h=parseInt(g[0],10);var b=parseInt(g[1],10);2===g.length&&!isNaN(h)&&!isNaN(b)&&0<=h&&0<=b&&d.push([h,b])}}return d}function debounce(a,d){var e=null,g=null;return function(){var h=this,b=Number(new Date),k=arguments;e&&b<e+d?(clearTimeout(g),g=setTimeout(function(){e=b;a.apply(h,k)},d)):(e=b,a.apply(h,k))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,d){function e(f){var a="test";a=a.replace(/[\[\]]/g,"\\$\x26");a=new RegExp("[?\x26]"+a+"(\x3d([^\x26#]*)|\x26|#|$)");a=(a=a.exec(window.location.href))?a[2]?decodeURIComponent(a[2].replace(/\+/g," ")):"":null;a=a&&"ads"===a?"adtype\x3dtest;":"";f=f.getAttribute("data-gpt-targeting");var c=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var d=isLoggedIn();a&&-1===f.indexOf(a)&&(f+=a);c&&
(f+=c);return f+d+b}function g(a,c){for(var f=a.length,b=[];f--;)c(a[f],f)&&(b.push(a[f].slot),a.splice(f,1));b.length&&googletag.pubads().refresh(b);return a}function h(f){var c=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return g(f,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})}var b,k=getAdContainers(),c=[];d&&(b=d.dfppKeyValues?d.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var m=
0;k[m];++m){var l=k[m];splitKeys(e(l));c.push({divId:l.getAttribute("id"),adUnitPath:l.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(l.getAttribute("data-gpt-sizes")),targeting:splitKeys(e(l))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(27)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());
for(var b=0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(p){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=h(c)});var n=debounce(function(){googletag.cmd.push(function(){c=h(c);c.length||removeScrollEvent(n)})},250);addScrollEvent(n)})(window,
window.Krux);</script>

<script type="text/javascript" id="ga-data-track-event-listeners">function addEvent(b){document.addEventListener(b,function(a){a=a.target;var c='[data-track\x3d"'+b+'"]';a=google_tag_manager["GTM-TDGJHK"].macro(35)(a,c);void 0!==a&&dataLayer.push({event:"data-track",eventAction:a.getAttribute("data-track-action"),eventCategory:a.getAttribute("data-track-category"),eventLabel:a.getAttribute("data-track-label")||null,eventValue:a.getAttribute("data-track-value")||null})},!0)}(function(){addEvent("click");addEvent("submit")})();</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2197/generated/client.html?source=https://bmcbioinformatics.biomedcentral.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Fbmcbioinformatics.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>