<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="62aa2354-f7fe-40f4-ae56-6f36d7731538" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script async="" src="prum.js"></script><script src="iframe_api"></script><script type="text/javascript" src="moatframe.js"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";article:article:doi\:10.1111/mec.12413;requestedJournal:journal:1365294x;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;issue:issue:doi\:10.1111/mec.2013.22.issue-17;journal:journal:1365294x;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text">









<iframe style="position: absolute; top: -10000px; left: -1000px;"></iframe><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script src="263124174624252" async=""></script><script async="" src="fbevents.js"></script><script type="text/javascript">var $DoubleClickZone = "molec-eco_mec";var $DoubleClickSite =  "wly.ecology_000046";</script><script id="analyticDigitalData">digitalData = {"site":{"ip":"105.27.236.251","environment":"LIVE","website":"onlinelibrary.wiley.com","serverDate":"2020-02-17","server":"web119"},"page":{"subPage":"Full Text","page":"Article/Chapter View","pageGroup":"Publication Pages","title":"title","errorCode":200},"identities":[{"type":"BasicGroup","uuid":"be20949b-b372-471e-96a5-9963c7266a9d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000054870"}]},{"type":"BasicGroup","uuid":"98ba5e39-8ca0-4d0a-addd-395777fbf515","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000034166"}]},{"type":"InstitutionUser","uuid":"6e97ed1f-543d-493d-b458-2e63cb8fbde5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#KLISC"}]},{"type":"BasicGroup","uuid":"e623deab-c83b-41e3-84a2-d9f1d46c5f17"},{"type":"RegionUser","uuid":"53f1676b-6f75-446d-8831-5915e1edce59"},{"type":"InstitutionUser","uuid":"dab9a169-ec74-4502-8066-0f76387b0f4e","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASP"}]},{"type":"BasicGroup","uuid":"40e3f768-07c0-45e5-aa81-97aef6eb686c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000005407"}]},{"type":"BasicGroup","uuid":"c3199058-0f44-4f36-b385-3a7c0023d85c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"KENYA HINARI NPL"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"5196cf05-b385-4de3-91a2-2d5ffd18abc5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASPKEN"}]},{"type":"InstitutionUser","uuid":"480e4dba-ca42-475f-a7ca-5b6fbfad49f7","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000014668"}]}],"publication":{"item":{"title":"How to fail at species delimitation","doi":"10.1111/mec.12413","access":"free","earliestDate":"2013-07-16","formats":["Abstract","Cited by","First page image","Full text","PDF","References"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"group":{"doi":"10.1111/mec.2013.22.issue-17","access":"full","volume":"22","issue":"17","coverDate":"2013-09-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"series":{"title":"Molecular Ecology","doi":"10.1111/(ISSN)1365-294X","access":"no","code":"1365294x","issn":"0962-1083","eissn":"1365-294X","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"},{"taxonomyUri":"gpt-zonename","topicLabel":"molec-eco_MEC","topicUri":"molec-eco_mec"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.ecology_000046","topicUri":"wly.ecology_000046"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    






    <meta property="og:title" content="How to fail at species delimitation">


    <meta property="og:type" content="Article">


        <meta property="og:url" content="https://onlinelibrary.wiley.com/doi/10.1111/mec.12413">
    

        
        <meta property="og:image" content="https://onlinelibrary.wiley.com/cms/asset/d73e9a77-dfad-468b-9c64-463eaf0a5bcc/mec.2013.22.issue-17.cover.gif">
    

    <meta property="og:site_name" content="Wiley Online Library">



        

        <meta property="og:description" content="Species delimitation is the act of identifying species‐level biological diversity. In recent years, the field has witnessed a dramatic increase in the number of methods available for delimiting speci...">
    


    <meta name="viewport" content="width=device-width,initial-scale=1">




















    

    <meta name="citation_doi" content="10.1111/mec.12413">

    <meta name="dc.identifier" content="10.1111/mec.12413">




<meta name="citation_journal_title" content="Molecular Ecology">
<meta name="citation_title" content="How to fail at species delimitation">
<meta name="citation_keywords" content="coalescent">
<meta name="citation_keywords" content="phylogeography">
<meta name="citation_keywords" content="species delimitation">
<meta name="citation_volume" content="22">
<meta name="citation_issue" content="17">
<meta name="citation_firstpage" content="4369">
<meta name="citation_lastpage" content="4383">
<meta name="citation_publication_date" content="2013/09/01">
<meta name="citation_online_date" content="2013/07/16">
<meta name="citation_issn" content="1365-294X">
<meta name="citation_language" content="en">
<meta name="citation_author" content="Bryan C. Carstens">
<meta name="citation_author_institution" content="Department of Evolution, Ecology and Organismal Biology The Ohio State University 318&nbsp;W. 12th Avenue Columbus OH 43210‐1293 USA">
<meta name="citation_author" content="Tara A. Pelletier">
<meta name="citation_author_institution" content="Department of Evolution, Ecology and Organismal Biology The Ohio State University 318&nbsp;W. 12th Avenue Columbus OH 43210‐1293 USA">
<meta name="citation_author" content="Noah M. Reid">
<meta name="citation_author_institution" content="Department of Biological Sciences Louisiana State University Life Sciences Building Baton Rouge LA 70803 USA">
<meta name="citation_author" content="Jordan D. Satler">
<meta name="citation_author_institution" content="Department of Evolution, Ecology and Organismal Biology The Ohio State University 318&nbsp;W. 12th Avenue Columbus OH 43210‐1293 USA">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.12413">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1111/mec.12413">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1111/mec.12413">
<meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd"><meta name="citation_fulltext_world_readable" content="">
<meta name="article_references" content="Carstens, B.C., Pelletier, T.A., Reid, N.M. and Satler, J.D. (2013), How to fail at species delimitation. Mol Ecol, 22: 4369-4383. doi:10.1111/mec.12413">
<meta name="epdf_available" content="true">
<meta name="Description" content="Abstract Species delimitation is the act of identifying species‐level biological diversity. In recent years, the field has witnessed a dramatic increase in the number of methods available for delim...">


<title>How to fail at species delimitation - Carstens - 2013 - Molecular Ecology - Wiley Online Library</title>


















<link rel="stylesheet" type="text/css" href="ibwuproduct.css">


    <link rel="stylesheet" href="build-99caeefe4bd3068dd1ca.css"><link rel="stylesheet" href="print-99caeefe4bd3068dd1ca.css" media="print"><link rel="stylesheet" href="owl.asc">





<script async="" src="catchpoint-1549680351760.js"></script>
<meta name="referrer" content="origin-when-cross-origin">


<link rel="stylesheet" type="text/css" href="hub-1579617685677.css">


<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1552321492860.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* PERICLES-7186: distorted cover images */
.journal-info-container .cover-image__image img {
    min-height: unset;
}

/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents,
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}

/* TEMPORARY Fix heading on Topical Index widget for MRWs */
.topical-index h2 {
    margin: 15px 0;
    font-size: 1.75rem;
    line-height: 28px;
    color: #1c1d1e;
    font-family: Source Serif Pro,serif;
    font-weight: normal;
}

/* PERICLES-7288 */
.issue-supplementary-files, .cover-image__virtual-issue {
    margin-bottom: 15px;
}

/* PERICLES-8875 */
.journal-news .slide-item img {
    float: left;
    margin-right: 10px;
}

/* DO NOT REMOVE */
li[heading="Restore Points"].ng-isolate-scope,
li[heading="Disable Widgets"].ng-isolate-scope {
    display: none!important;
}

.modal-header .widgetOrigin {
    display: none;
}

/* PERICLES-10090 */
.cover-image__image img {
    max-width: 150px;
    max-height: 150px;
}

/* LIT-282003 */
.journal-info-container span.info_value:after {
    content: '\003B';
}

.journal-info-container span.info_value:last-child:after {
    content: '';
}

/* LIT-299510 */
h1.citation__title {
    font-size: 22px;
    font-size: 1.375rem;
}

@media (max-width: 991px) {
    h1.citation__title {
        font-size: 18px;
        font-size: 1.125rem;
    }
    h1.citation__title:after {
        content: "";
    }
}

/* LIT-295816 */
.search__item .item__body .meta__details a.publication_meta_serial:after {
    content: "";
    margin: 0;
}
.search__item .item__body .meta__details a.publication_meta_volume_issue:before {
    content: "|";
    margin-left: 4px;
    margin-right: 4px;
}
</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5c87b4fe64746d78ad006ae3.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script><script src="satellite-5aaec36164746d79c1001d14.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>

<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	if (!$DoubleClickSite) {
		switch("pericles") {
			case "afspubs":
				var $DoubleClickSite = "wly.aqua.fish_000017";
				break;
			case "aasldpubs":
  				var $DoubleClickSite = "wly.gastro.hep_000059";
  				break;
			case "headachejournal":
  				var $DoubleClickSite = "wly.neurology_000086";
  				break;
			default:
				var $DoubleClickSite = "undefined-site";
		};
  	};

	if (!$DoubleClickZone) {
		switch("pericles") {
			case "afspubs":
				var $DoubleClickZone = "AFS_HUB";
				break;
      			case "aasldpubs":
        			var $DoubleClickZone = "AASLD-pubs_HUB";
        			break;
      			case "headachejournal":
        			var $DoubleClickZone = "headache_HUB";
        			break;
			default:
				var $DoubleClickZone = "undefined-zone";
		};
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};


		gptPage = window.digitalData && digitalData.page && digitalData.page.page ? digitalData.page.page : "blank";
		gptSubPage = window.digitalData && digitalData.page && digitalData.page.subPage ? digitalData.page.subPage : "blank";


		function gptClean(gptText){
			if(typeof(gptText)=="string"){
				gptText = gptText.replace(/\&/g,'%26');


				return gptText.replace(/\^|\*|#|\$|\"|\'|\=|\,|\!|\+|\~|\;|\(|\)|\<|\>|\[|\]|\?|\%|\:|\@|\;|\/|\&|\||\{|\}|\\|/g,'');

			}
			return "blank";
		}


		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'doi' , '10.1111/mec.12413') // Publication DOI
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		) 	// Krux
			.setTargeting ( 'pbPage', gptClean(gptPage) )
			.setTargeting ( 'pbSubPage', gptClean(gptSubPage) );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>






    







    
        
    
































    <!-- Get whether TNSI is in live or test mode --> 
    
    <!-- Get the configured TNSI Javascript version number --> 
    
    <!-- Get the configured TNSI MerchantID --> 
    
    <!-- Add "TEST" to beginning of merchant ID when running in test mode -->
    

    <script id="TNSISessionJS" src="session.js"></script>












    <script src="casa.js" async=""></script>







    


<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "How to fail at species delimitation - Carstens - 2013 - Molecular Ecology - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker.js" async=""></script>
<script async="" type="text/javascript" src="epdf_linker_002.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js" as="script"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2020013001.js" async=""></script><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="satellite-5c58af5e64746d5111000925.js"></script><script src="satellite-5c87cfcf64746d7044002f7b.js"></script><script src="satellite-5cf7bd7564746d596c000241.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script src="js.js" async="true"></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><script src="satellite-5c87b56e64746d4e33003a7c.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1a038ba29949ad2fd2059.js"></script><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2667a43520a.css"></head>
<body class="pb-ui">

<div class="skip-links">
    <ul class="rlist">
        
                <li><a href="#article__content" tabindex="1">Skip to Article Content</a></li>
                <li><a href="#pane-pcw-detailscon" tabindex="1">Skip to Article Information</a></li>
            
    </ul>
</div>







    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader fixed-element">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        



        
    

</div></div></div>

        </div>
    




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img id="mainLogo" alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img id="mobileLogo" alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" aria-label="Search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper"><div class="tab clearfix"><input placeholder="Search" tabindex="13" class="fakeQSInput quickSearchFilter" type="search"><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" role="tablist" tabindex="10" class="search-within"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" selected="selected">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1">Anywhere</option></select><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="1365294x" type="hidden"><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    









    <div class="loginBar login-open">
        <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fmec.12413%3Fcasa_token%3D0GNu1GWTRJkAAAAA%253AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl&amp;aria-label=Log+in+or+Register" class="show-login" role="button" uri="/doi/full/10.1111/mec.12413?casa_token=0GNu1GWTRJkAAAAA%3AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl" aria-label="Log in or Register">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </a>
            
    </div>



        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    










        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet body-text">
        



        
        <main id="main-content" tabindex="-1">
    <!--begin pagefulltext-->
    <div class="page-body pagefulltext">
        <div data-pb-dropzone="main">
        



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        



        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fmec.12413%3Fcasa_token%3D0GNu1GWTRJkAAAAA%253AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl&pubId=50345552&placeholderId=1015&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_0__container__" style="border: 0pt none; width: 100%; height: 0%;"></div></div>
	</aside>
        </div>
    


        </div>
    




        
        
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.12413&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.12413%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2013.22.issue-17%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.12413&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.12413%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2013.22.issue-17%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.12413&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.12413%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2013.22.issue-17%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.12413&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.12413%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2013.22.issue-17%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div id="article__content" tabindex="-1" class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      <div id="journal-banner-text" class="journal-banner-text" style="font-size: 22px;"><a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage">Molecular Ecology</a></div><a href="https://onlinelibrary.wiley.com/toc/1365294x/2013/22/17" class="volume-issue">Volume 22, Issue 17</a><a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage" class="citation--logo"><img id="journal-banner-image" test="test-value" alt="Molecular Ecology" src="1365294x-1525974682817.jpg" class="journal-banner-image" style="display: none;"></a><script>var image = new Image();

image.onerror = function() {
  // if the image does not exist
  //document.getElementById("journal-banner-text").style.marginBottom = "15px";
  document.getElementById("journal-banner-text").style.fontSize = "22px";
  document.getElementById("journal-banner-image").style.display = "none";
}

image.src = "/pb-assets/journal-banners/1365294x-1525974682817.jpg";</script>
   </div>
   <div class="doi-access-container clearfix"><span class="primary-heading"><span class="primary-heading">Invited Reviews and Meta‐analyses</span></span>
      <div class="doi-access-wrapper"><div class="free-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Free Access</div></div></div></div></div>
   <h1 class="citation__title">How to fail at species delimitation</h1>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Carstens%2C+Bryan+C" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl" role="button"><span>Bryan C. Carstens<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" role="region" id="am1">
            <p class="author-type">Corresponding Author</p>
            <p>Department of Evolution, Ecology and Organismal Biology, 
The Ohio State University, 318&nbsp;W. 12th Avenue, Columbus, OH, 
43210‐1293 USA</p>Correspondence: Bryan C. Carstens, Fax: +1 (614) 292 2030; E‐mail: 
            <p><a title="
                            Link to email address
                        " href="mailto:carstens.12@osu.edu" class="corr-email">carstens.12@osu.edu</a></p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Carstens%2C+Bryan+C">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Pelletier%2C+Tara+A" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl" role="button"><span>Tara A. Pelletier</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" role="region" id="am2">
            <p>Department of Evolution, Ecology and Organismal Biology, 
The Ohio State University, 318&nbsp;W. 12th Avenue, Columbus, OH, 
43210‐1293 USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Pelletier%2C+Tara+A">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Reid%2C+Noah+M" class="author-name accordion-tabbed__control" data-id="am3" data-db-target-for="am3" aria-controls="am3" aria-haspopup="true" id="am3_Ctrl" role="button"><span>Noah M. Reid</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am3" aria-labelledby="am3_Ctrl" role="region" id="am3">
            <p>Department of Biological Sciences, Louisiana State University, Life Sciences Building, Baton Rouge, LA, 70803 USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Reid%2C+Noah+M">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Satler%2C+Jordan+D" class="author-name accordion-tabbed__control" data-id="am4" data-db-target-for="am4" aria-controls="am4" aria-haspopup="true" id="am4_Ctrl" role="button"><span>Jordan D. Satler</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am4" aria-labelledby="am4_Ctrl" role="region" id="am4">
            <p>Department of Evolution, Ecology and Organismal Biology, 
The Ohio State University, 318&nbsp;W. 12th Avenue, Columbus, OH, 
43210‐1293 USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Satler%2C+Jordan+D">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Carstens%2C+Bryan+C" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl" role="button"><span>Bryan C. Carstens<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" role="region" id="a1">
                  <p class="author-type">Corresponding Author</p>
                  <p>Department of Evolution, Ecology and Organismal 
Biology, The Ohio State University, 318&nbsp;W. 12th Avenue, Columbus, 
OH, 43210‐1293 USA</p>Correspondence: Bryan C. Carstens, Fax: +1 (614) 292 2030; E‐mail: 
                  <p><a title="
                            Link to email address
                        " href="mailto:carstens.12@osu.edu" class="corr-email">carstens.12@osu.edu</a></p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Carstens%2C+Bryan+C">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Pelletier%2C+Tara+A" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl" role="button"><span>Tara A. Pelletier</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" role="region" id="a2">
                  <p>Department of Evolution, Ecology and Organismal 
Biology, The Ohio State University, 318&nbsp;W. 12th Avenue, Columbus, 
OH, 43210‐1293 USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Pelletier%2C+Tara+A">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Reid%2C+Noah+M" class="author-name accordion-tabbed__control" data-id="a3" data-db-target-for="a3" aria-controls="a3" aria-haspopup="true" id="a3_Ctrl" role="button"><span>Noah M. Reid</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a3" aria-labelledby="a3_Ctrl" role="region" id="a3">
                  <p>Department of Biological Sciences, Louisiana State University, Life Sciences Building, Baton Rouge, LA, 70803 USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Reid%2C+Noah+M">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Satler%2C+Jordan+D" class="author-name accordion-tabbed__control" data-id="a4" data-db-target-for="a4" aria-controls="a4" aria-haspopup="true" id="a4_Ctrl" role="button"><span>Jordan D. Satler</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a4" aria-labelledby="a4_Ctrl" role="region" id="a4">
                  <p>Department of Evolution, Ecology and Organismal 
Biology, The Ohio State University, 318&nbsp;W. 12th Avenue, Columbus, 
OH, 43210‐1293 USA</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Satler%2C+Jordan+D">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections"><div class="epub-section"><span class="epub-state">First published: </span><span class="epub-date">16 July 2013</span></div>
      <div class="epub-section"><a class="epub-doi" aria-label="Digital Object Identifier" href="https://doi.org/10.1111/mec.12413">https://doi.org/10.1111/mec.12413</a></div>
      <div class="epub-section cited-by-count"><span>Citations: <a href="#citedby-section">455</a></span></div>
   </div>
</div></div><div class="fixed-element"><nav aria-label="Article navigation and tools" class="coolBar stickybar"><div class="stickybar__wrapper coolBar__wrapper clearfix"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><ul class="transplanted-clone"><li><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul></li><li><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/mec.12413" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/mec.12413" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></li></ul></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d22006893"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0002-title"><span> The parameter space relevant to species delimitation</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0008-title"><span> Species discovery approaches assign samples to groups without a priori information</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0010-title"><span> Species validation approaches require the user to assign samples to putative lineages</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0004-title"><span> Strategies for complex empirical investigations</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0005-title"><span> Potential shortcomings of validation approaches</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0006-title"><span> Incongruence across results</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0007-title"><span> How to approach species delimitation</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0012-title"><span> Integrating genetic and nongenetic sources of data</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0013-title"><span> Do researchers trust in the results of their species delimitation analyses?</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0014-title"><span> Conclusions</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#mec12413-sec-0015-title"><span> Acknowledgements</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/mec.12413" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/mec.12413" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1111%2Fmec.12413&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2Fmec.12413" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1111%2Fmec.12413" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1111%2Fmec.12413" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="addthis_widget.js"></script><div class="relative"><a id="share-access__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share-access__ctrl hidden"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1111/mec.12413/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i><span class="sr-only">Close modal</span></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email</a></li><li><a role="link" class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a role="link" class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>LinkedIn</a></li><li><a role="link" class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav></div><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body "><article>
   
   <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
      <h2 id="d22006893" class="article-section__header section__title main main">Abstract</h2>
      <div class="article-section__content en main">
         
         <p>Species delimitation is the act of identifying species‐level
 biological diversity. In recent years, the field has witnessed a 
dramatic increase in the number of methods available for delimiting 
species. However, most recent investigations only utilize a handful 
(i.e. 2–3) of the available methods, often for unstated reasons. Because
 the parameter space that is potentially relevant to species 
delimitation far exceeds the parameterization of any existing method, a 
given method necessarily makes a number of simplifying assumptions, any 
one of which could be violated in a particular system. We suggest that 
researchers should apply a wide range of species delimitation analyses 
to their data and place their trust in delimitations that are congruent 
across methods. Incongruence across the results from different methods 
is evidence of either a difference in the power to detect cryptic 
lineages across one or more of the approaches used to delimit species 
and could indicate that assumptions of one or more of the methods have 
been violated. In either case, the inferences drawn from species 
delimitation studies should be conservative, for in most contexts it is 
better to fail to delimit species than it is to falsely delimit entities
 that do not represent actual evolutionary lineages.</p>
         
      </div>
   </section>
   <div class="pb-dropzone" data-pb-dropzone="below-abstract-group">
      <!-- Empty dropzone -->
   </div>
   
   <section class="article-section article-section__full">
      <section class="article-section__content" id="mec12413-sec-0001">
         
         <p>In the broadest sense, species delimitation is the act of 
identifying species‐level biological diversity. As genetic data have 
become easier and less expensive to gather, the field of phylogeography 
has experienced an explosion in the number and variety of methodological
 approaches to species delimitation. In this study, we discuss the 
application of these methods to data collected from sexually reproducing
 organisms. Our aim is not to rank available methods or to evaluate 
their effectiveness under some narrow range of simulation conditions, as
 these exercises have been conducted previously (e.g. O'Meara <span><a href="#mec12413-bib-0064" class="bibLink tab-link" id="mec12413-bib-0064R" data-tab="pane-pcw-references">2010</a></span>; Yang &amp; Rannala <span><a href="#mec12413-bib-0093" class="bibLink tab-link" id="mec12413-bib-0093R" data-tab="pane-pcw-references">2010</a></span>; Ence &amp; Carstens <span><a href="#mec12413-bib-0021" class="bibLink tab-link" id="mec12413-bib-0021R" data-tab="pane-pcw-references">2011</a></span>; Rittmeyer &amp; Austin <span><a href="#mec12413-bib-0077" class="bibLink tab-link" id="mec12413-bib-0077R" data-tab="pane-pcw-references">2012</a></span>).
 These studies generally find that methods are relatively accurate in 
simple cases (i.e. when attempting to delimit 2–3 lineages) using modest
 amounts of data (e.g. ~10–20 loci). Rather, we argue <i>any</i> of the 
existing methods are likely to be incapable of accurately delimiting 
evolutionary lineages under some plausible set of conditions and that 
because of these limitations, researchers conducting species 
delimitation should analyse their data using a wide range of methods and
 place their trust in the observable congruence across the results. 
After presenting this argument, we broaden our discussion of species 
delimitation to place genetic approaches in a wider context that 
includes delimitation with nongenetic sources of data and various 
concepts of species.
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0002">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0002-title"> The parameter space relevant to species delimitation</h2>
         
         <p>A useful thought experiment to consider before discussing 
the existing methods for species delimitation is to imagine the perfect 
methodological approach for delimiting independent evolutionary lineages
 using genetic data. Existing models range from nonparametric (e.g. 
Wiens &amp; Penkrot <span><a href="#mec12413-bib-0091" class="bibLink tab-link" id="mec12413-bib-0091R" data-tab="pane-pcw-references">2002</a></span>) to highly parameterized models (e.g. Yang &amp; Rannala <span><a href="#mec12413-bib-0093" class="bibLink tab-link" id="mec12413-bib-0093R" data-tab="pane-pcw-references">2010</a></span>).
 An idealized method would probably implement a parametric approach, 
because most methods for species delimitation operate by fitting some 
model of the historical diversification to the data collected from some 
natural system. However, the parameter space relevant to species 
delimitation is large, and existing tools limit their exploration to a 
subset of the potential parameter space by assuming a number of 
simplifications, with each method making a different set of simplifying 
assumptions. While this is a strategy inherent to model‐based analysis, 
and not intended as a criticism, researchers should realize that all 
methods incorporate models that are imperfect imitations of the 
biological reality. At a minimum, this parameter space for species 
delimitation includes population genetic and phylogenetic parameters. 
Examples of population genetic parameters include gene flow and 
population size as well as other parameters that are perhaps not 
directly relevant to species delimitation, but which may influence 
patterns of genetic diversity (e.g. extrinsic rates of population size 
change, recombination). Phylogenetic parameters related to the pattern 
and timing of lineage diversification are also necessary because systems
 to which species delimitation methods are applied are inherently 
emergent phylogenetic systems, even though the empirical data from such 
systems may approach the lower limits of appropriateness for 
phylogenetic analysis (see Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec12413-fea-0001">1</a>).
 Furthermore, an ideal method would be agnostic with regard to the 
assignment of samples to population and as such would not require that 
the researcher provide this information. Many existing methods implement
 a coalescent model (e.g. Kingman <span><a href="#mec12413-bib-0046" class="bibLink tab-link" id="mec12413-bib-0046R" data-tab="pane-pcw-references">1982</a></span>)
 and thus also assume that the data meet the standard assumptions of 
this model (i.e. that the data are sampled from neutral loci that do not
 contain internal points of recombination, that populations are 
panmictic, etc.). The potential parameter space relevant to species 
delimitation is larger and far more complex than that considered by even
 the most heavily parameterized of existing methods. Given that there is
 a large gap between our <i>ideal method</i> and existing methods, how should researchers proceed when delimiting species?
         </p>
         
         <div class="feature" id="mec12413-fea-0001">
            <h3><span class="feature-label">Box 1. </span>Systems in which species delimitation methods are applied are emergent phylogenetic systems
            </h3>
            <section class="article-section__content Box " id="mec12413-sec-0003">
               
               <p>Phylogenetic analysis is traditionally concerned with 
estimating relationships among species, while population genetics is 
concerned with adaptation and understanding the population‐level forces 
that change allele frequencies. Since the modern synthesis, these fields
 largely developed in parallel, but without much interaction (Hull <span><a href="#mec12413-bib-0041" class="bibLink tab-link" id="mec12413-bib-0041R" data-tab="pane-pcw-references">1998</a></span>; Felsenstein <span><a href="#mec12413-bib-0023" class="bibLink tab-link" id="mec12413-bib-0023R" data-tab="pane-pcw-references">2004</a></span>).
 However, in recent decades, there has been increased activity at the 
interface of these disciplines. For example, divergence population 
genetics (Hey &amp; Nielsen <span><a href="#mec12413-bib-0035" class="bibLink tab-link" id="mec12413-bib-0035R" data-tab="pane-pcw-references">2004</a></span>)
 models divergence among populations, and a number of approaches 
concerned with estimating population divergence have been developed 
(Nielsen &amp; Wakeley <span><a href="#mec12413-bib-0060" class="bibLink tab-link" id="mec12413-bib-0060R" data-tab="pane-pcw-references">2001</a></span>; Hey &amp; Nielsen <span><a href="#mec12413-bib-0035" class="bibLink tab-link" id="mec12413-bib-0035R" data-tab="pane-pcw-references">2004</a></span>; Gutenkunst <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0030" class="bibLink tab-link" id="mec12413-bib-0030R" data-tab="pane-pcw-references">2009</a></span>). Similarly, there are phylogenetic approaches to estimating population‐level parameters (e.g. Heled &amp; Drummond <span><a href="#mec12413-bib-0033" class="bibLink tab-link" id="mec12413-bib-0033R" data-tab="pane-pcw-references">2008</a></span>).
 However, the most striking evidence of the merger between population 
genetics and phylogenetics has been the use of the multispecies 
coalescent model to estimate phylogeny (i.e. species trees, Edwards <span><a href="#mec12413-bib-0020" class="bibLink tab-link" id="mec12413-bib-0020R" data-tab="pane-pcw-references">2009</a></span>). Seminal work by Maddison (<span><a href="#mec12413-bib-0056" class="bibLink tab-link" id="mec12413-bib-0056R" data-tab="pane-pcw-references">1997</a></span>)
 convinced a generation of researchers that they could not overlook the 
inherent stochasticity of genealogies when estimating phylogeny at the 
shallowest levels of diversification, and in time the basic coalescent 
model of Kingman (<span><a href="#mec12413-bib-0046" class="bibLink tab-link" id="mec12413-bib-0046R" data-tab="pane-pcw-references">1982</a></span>) was expanded to include multiple species (e.g. Rannala &amp; Yang <span><a href="#mec12413-bib-0073" class="bibLink tab-link" id="mec12413-bib-0073R" data-tab="pane-pcw-references">2003</a></span>). As multilocus data became common, and as curious statistical results such as anomalous zones (Degnan &amp; Rosenberg <span><a href="#mec12413-bib-0016" class="bibLink tab-link" id="mec12413-bib-0016R" data-tab="pane-pcw-references">2006</a></span>)
 were discovered, the multispecies coalescent (i.e. species trees) 
became the default option for phylogeny estimation within 
phylogeographical investigations. Concurrently, new algorithms for 
detecting population genetic structure (Pritchard <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0068" class="bibLink tab-link" id="mec12413-bib-0068R" data-tab="pane-pcw-references">2000</a></span>; Huelsenbeck &amp; Andolfatto <span><a href="#mec12413-bib-0038" class="bibLink tab-link" id="mec12413-bib-0038R" data-tab="pane-pcw-references">2007</a></span>) were developed, and questions related to species delimitation returned to the forefront of systematics (Wiens <span><a href="#mec12413-bib-0090" class="bibLink tab-link" id="mec12413-bib-0090R" data-tab="pane-pcw-references">2007</a></span>).
 Three particular developments have enabled new approaches to species 
delimitation. First, coalescent‐based methods that model 
population‐level processes such as genetic drift in combination with 
migration (Beerli &amp; Felsenstein <span><a href="#mec12413-bib-0005" class="bibLink tab-link" id="mec12413-bib-0005R" data-tab="pane-pcw-references">2001</a></span>), expansion (Kuhner <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0050" class="bibLink tab-link" id="mec12413-bib-0050R" data-tab="pane-pcw-references">1998</a></span>), population divergence (Hey &amp; Nielsen <span><a href="#mec12413-bib-0036" class="bibLink tab-link" id="mec12413-bib-0036R" data-tab="pane-pcw-references">2007</a></span>) or combinations of these processes (Gutenkunst <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0030" class="bibLink tab-link" id="mec12413-bib-0030R" data-tab="pane-pcw-references">2009</a></span>)
 are widely applied to phylogeographical data. Second, phylogenetic 
methods that estimate phylogeny while allowing for the action of some of
 these processes (Maddison &amp; Knowles <span><a href="#mec12413-bib-0057" class="bibLink tab-link" id="mec12413-bib-0057R" data-tab="pane-pcw-references">2006</a></span>; Liu <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0055" class="bibLink tab-link" id="mec12413-bib-0055R" data-tab="pane-pcw-references">2007</a></span>; Kubatko <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0049" class="bibLink tab-link" id="mec12413-bib-0049R" data-tab="pane-pcw-references">2009</a></span>; Heled &amp; Drummond <span><a href="#mec12413-bib-0034" class="bibLink tab-link" id="mec12413-bib-0034R" data-tab="pane-pcw-references">2010</a></span>)
 are increasingly applied to low‐level phylogenetic systems. Third, the 
field is no longer data limited, as advances in DNA sequencing allow 
data from 1000s of loci to be collected from nearly any system 
(McCormack <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0058" class="bibLink tab-link" id="mec12413-bib-0058R" data-tab="pane-pcw-references">2013</a></span>).
 In summary, the systems in which researchers conduct species 
delimitation exist at the interface of traditional population genetic 
and phylogenetic analyses. As such, they borrow methods from each but 
have a unique set of challenges.
               </p>
               
            </section>
         </div>
         
         <p>A naive response to the above conundrum is to identify a 
single method that is demonstrably accurate in some simulation study and
 apply it alone to the data. Thus, one researcher might choose 
Structurama (Huelsenbeck <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0039" class="bibLink tab-link" id="mec12413-bib-0039R" data-tab="pane-pcw-references">2011</a></span>) based on the results presented by Rittmeyer &amp; Austin (<span><a href="#mec12413-bib-0077" class="bibLink tab-link" id="mec12413-bib-0077R" data-tab="pane-pcw-references">2012</a></span>),
 analyses their data using this method and delimits species based on the
 results. This course of action has two substantial shortcomings. First,
 to our knowledge, no simulation study has included every potentially 
useful method, so any evaluation of existing methods will necessarily be
 incomplete. Second, results from simulation studies are conditional on 
the specific attributes of the simulated data used in such studies (i.e.
 the assumed <i>θ</i>, number of variable sites, number of loci) and 
thus most relevant when tailored to specific systems. To illustrate why 
these details matter, consider the simulations presented by Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>) in an investigation into <i>Liolaemus</i> lizards. They reported that BPP (Yang &amp; Rannala <span><a href="#mec12413-bib-0093" class="bibLink tab-link" id="mec12413-bib-0093R" data-tab="pane-pcw-references">2010</a></span>)
 and a custom approximate Bayesian computation method were more accurate
 in their simulations than the method spedeSTEM (Ence &amp; Carstens <span><a href="#mec12413-bib-0021" class="bibLink tab-link" id="mec12413-bib-0021R" data-tab="pane-pcw-references">2011</a></span>). Camargo <i>et&nbsp;al</i>.
 also found spedeSTEM to be less accurate in their simulations than in 
those reported by the program authors. This discrepancy resulted from 
differences in the sample size and numbers of variable sites in the 
simulated data, which influenced the quality of the gene tree estimates 
that are input into spedeSTEM. Because these studies were based on the 
levels of variation in data collected from different empirical systems, 
the simulation results differed slightly. For a researcher trying to 
delimit species in non‐<i>Liolaemus</i> systems, the results from a simulation study such as Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>) should be viewed as a rough guideline and not be considered a directly transferable prediction of accuracy.
         </p>
         
         <p>The Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 study exemplifies a more sophisticated approach to delimiting species. 
By designing and conducting a simulation study that matches the 
characteristics of their <i>Liolaemus</i> lizard system, Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 establish an expectation of accuracy that provides them with some 
degree of confidence in their results. Because their simulations 
indicate that the methods used are generally able to delimit species 
using the amount of data that they have collected, they can confidently 
assert that their study has discovered independent evolutionary lineages
 within <i>Liolaemus</i>. Given that simulated data are far easier to 
generate than actual data, such an approach to species delimitation 
should be widely emulated by any empirical study that attempts to 
delimit species. Another aspect of their study is worth highlighting. 
Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 chose to focus on methods derived from the multispecies coalescent 
model (BPP and spedeSTEM), but because they were unwilling to assume 
that all shared polymorphism was due to incompletely sorted ancestral 
polymorphism (an assumption of the above methods), they also utilized 
approximate Bayesian computation (ABC; see Csilléry <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0013" class="bibLink tab-link" id="mec12413-bib-0013R" data-tab="pane-pcw-references">2011</a></span>) in an effort to model divergence with gene flow. While Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 delimit the southern and northern populations as separate lineages 
using all three approaches, their simulation study would have provided 
justification for choosing one set of results over the others in the 
event of incongruence. However, even though their simulations represent 
an additional level of rigour when compared to most empirical 
investigations, Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 limit their consideration to a restricted set of delimitation analyses 
and thus may not have identified the optimal method for their system. 
This is perhaps justifiable, because the focal system of Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 was clearly disjunct in two populations and sample assignment to 
putative lineages was unambiguous. In this way, their investigation 
represents the low‐hanging fruit of species delimitation investigations 
because sample assignment was not in question and because the system 
represented either one or two lineages. However, many focal systems are 
more complex.
         </p>
         
         <p>In systems where the populations are not clearly delineated,
 the assignment of samples to putative groups is essential. Sample 
assignment dramatically increases the difficulty in species delimitation
 (O'Meara <span><a href="#mec12413-bib-0064" class="bibLink tab-link" id="mec12413-bib-0064R" data-tab="pane-pcw-references">2010</a></span>)
 because it adds dramatically to the complexity of the algorithms used 
in species delimitation. As a result, methods for species delimitation 
have approached sample assignment from a number of directions (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec12413-fea-0002">2</a>). Genetic clustering approaches such as Structure (Pritchard <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0068" class="bibLink tab-link" id="mec12413-bib-0068R" data-tab="pane-pcw-references">2000</a></span>) and Structurama (Huelsenbeck <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0039" class="bibLink tab-link" id="mec12413-bib-0039R" data-tab="pane-pcw-references">2011</a></span>)
 operate by identifying the population assignments and level of 
clustering that minimizes Hardy–Weinberg disequilibrium. While 
clustering algorithms do not explicitly model population‐level 
parameters (i.e. <i>θ</i>, migration rates or population divergence), 
they are flexible in terms of the data that are required and have been 
applied to a wide range of systems. Notably, the ability of clustering 
algorithms to cluster samples does not alone offer compelling proof that
 the delimited entities have a history of phylogenetic divergence 
because the population structure is inferred without consideration of 
historical patterns of diversification, and there is not always a clear 
correspondence between a given level of clustering and the branching 
pattern of a species tree (Jackson &amp; Austin <span><a href="#mec12413-bib-0042" class="bibLink tab-link" id="mec12413-bib-0042R" data-tab="pane-pcw-references">2010</a></span>; Kalinowski <span><a href="#mec12413-bib-0043" class="bibLink tab-link" id="mec12413-bib-0043R" data-tab="pane-pcw-references">2010</a></span>).
 Furthermore, given enough data it may be possible to delimit extremely 
localized populations. For example, results from O'Dushlaine <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0062" class="bibLink tab-link" id="mec12413-bib-0062R" data-tab="pane-pcw-references">2010</a></span>)
 indicate that population structure among individual villages in Europe 
can be identified using ~300&nbsp;000 SNPs. While this is clearly more 
data than most existing species delimitation investigations utilize, it 
is likely that investigations into nonmodel systems will approach this 
number in the near future. Due to these potential issues, many authors 
use the results from Structure or Structurama as a starting point for 
delimitation investigations (e.g. Leaché &amp; Fujita <span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>), and in particular, they complement methods that explicitly model population divergence.
         </p>
         
         <div class="feature" id="mec12413-fea-0002">
            <h3><span class="feature-label">Box 2. </span>Some useful methods for species delimitation
            </h3>
            <section class="article-section__content Box " id="mec12413-sec-0008">
               
               <h2 class="article-section__title section__title section2" id="mec12413-sec-0008-title"> Species discovery approaches assign samples to groups without a priori information</h2>
               
               <p><b>Structurama</b> (Huelsenbeck <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0039" class="bibLink tab-link" id="mec12413-bib-0039R" data-tab="pane-pcw-references">2011</a></span>) implements the clustering algorithm first described by Pritchard <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0068" class="bibLink tab-link" id="mec12413-bib-0068R" data-tab="pane-pcw-references">2000</a></span>;
 for their program Structure) that clusters samples into populations by 
minimizing Hardy–Weinberg disequilibrium for a given partitioning level.
 Structurama includes the addition of reversible jump MCMC to identify 
the optimal partitioning level. Nearly any type of genetic data can be 
input into Structurama, and the program can assign individuals to 
population with or without the admixture. One shortcoming of genetic 
clustering approaches is that they do not assess the evolutionary 
divergence of population clusters (<a href="http://cteg.berkeley.edu/~structurama/index.html" class="linkBehavior">http://cteg.berkeley.edu/~structurama/index.html</a>).
               </p>
               
               <p><b>Gaussian Clustering</b> (Hausdorf &amp; Hennig <span><a href="#mec12413-bib-0032" class="bibLink tab-link" id="mec12413-bib-0032R" data-tab="pane-pcw-references">2010</a></span>)
 groups samples into populations using genotypic data by searching for 
clusters that can be attributed to being mixtures of normal allele 
frequency distributions. Like Structurama, the method is flexible in 
terms of the data that can be analysed. This approach is implemented in R
 using the prabclus (Hausdorf &amp; Hennig <span><a href="#mec12413-bib-0032" class="bibLink tab-link" id="mec12413-bib-0032R" data-tab="pane-pcw-references">2010</a></span>) and mclust (Fraley &amp; Raftery <span><a href="#mec12413-bib-0026" class="bibLink tab-link" id="mec12413-bib-0026R" data-tab="pane-pcw-references">2006</a></span>) packages. As in other clustering approaches, temporal divergence among putative groups is not explicitly estimated.
               </p>
               
               <p>The <b>general mixed Yule coalescent model</b> (GMYC; Pons <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0067" class="bibLink tab-link" id="mec12413-bib-0067R" data-tab="pane-pcw-references">2006</a></span>)
 takes an ultrametric genealogy estimated from a single genetic locus as
 input. The method attempts to model the transition point between 
cladogenesis and allele coalescence by utilizing the assumption that the
 former will occur at a rate far lower than the later. This results in a
 shift in the rate of branching of the genealogy that reflects the 
transition between species‐level processes (such as speciation and 
extinction) and population‐level processes (allele coalescence). Reid 
&amp; Carstens (<span><a href="#mec12413-bib-0075" class="bibLink tab-link" id="mec12413-bib-0075R" data-tab="pane-pcw-references">2012</a></span>)
 proposed a version of the GMYC that accounts for phylogenetic 
uncertainty gene tree estimates using a Bayesian analysis. Both 
implementations of the GMYC are likely to delimit well‐supported clades 
of haplotypes as independent lineages and as such may be prone to over 
delimitation (<a href="http://r-forge.r-project.org/projects/splits" class="linkBehavior">http://r-forge.r-project.org/projects/splits</a>, <a href="http://https//sites.google.com/site/noahmreid/home/software" class="linkBehavior">https://sites.google.com/site/noahmreid/home/software</a>).
               </p>
               
               <p>Choi &amp; Hey (<span><a href="#mec12413-bib-0012" class="bibLink tab-link" id="mec12413-bib-0012R" data-tab="pane-pcw-references">2011</a></span>)
 describe two new methods for jointly estimating population assignment 
along with the parameters of an isolation‐with‐migration model. <b>Joint demography and assignment</b> (JDA) is applicable to an island or two population models, while <b>joint demography and assignment of population tree</b>
 (JDAP) is applicable to more than two diverging populations. Each takes
 sequence data as input and is implemented within IMa2 (Hey &amp; 
Nielsen <span><a href="#mec12413-bib-0036" class="bibLink tab-link" id="mec12413-bib-0036R" data-tab="pane-pcw-references">2007</a></span>) (<a href="http://genfaculty.rutgers.edu/hey/software" class="linkBehavior">http://genfaculty.rutgers.edu/hey/software</a>).
               </p>
               
               <p>Unlike other methods described here, the unified model of Guillot <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0029" class="bibLink tab-link" id="mec12413-bib-0029R" data-tab="pane-pcw-references">2012</a></span>)
 can analyse nongenetic data (phenotypical, geographical, behavioural) 
in addition to genetic data. Their approach implemented a Bayesian 
clustering algorithm that assumes that each cluster in a geographical 
domain can be approximated by polygons that are centred around points 
generated by a Poisson process. Guillot <i>et&nbsp;al</i>.'s model is 
flexible in terms of the genetic data that it can utilize and capable of
 accurately delimiting species. Their model is available as an extension
 of the R GENELAND package (Guillot <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0028" class="bibLink tab-link" id="mec12413-bib-0028R" data-tab="pane-pcw-references">2005</a></span>) (<a href="http://www2.imm.dtu.dk/~gigu/Geneland/" class="linkBehavior">http://www2.imm.dtu.dk/~gigu/Geneland/</a>).
               </p>
               
               <p><b>O'Meara's heuristic method</b> (O'Meara <span><a href="#mec12413-bib-0064" class="bibLink tab-link" id="mec12413-bib-0064R" data-tab="pane-pcw-references">2010</a></span>)
 of species delimitation takes gene trees from multiple loci as input 
and operates under a similar assumption to the GMYC (namely that allelic
 coalescence occurs more rapidly than speciation). Provided that this 
assumption is true, the longest branches of gene trees are likely to 
represent species‐level differences, and thus, congruence across loci is
 indicative of both the species tree and the population assignments. 
O'Meara's method is implemented in the Brownie package (O'Meara <span><a href="#mec12413-bib-0063" class="bibLink tab-link" id="mec12413-bib-0063R" data-tab="pane-pcw-references">2008</a></span>).
 Because this method takes gene trees as input, its accuracy will likely
 be correlated with the nodal support values in the gene trees (<a href="http://www.brianomeara.info/brownie" class="linkBehavior">http://www.brianomeara.info/brownie</a>).
               </p>
               
               <p>Species delimitation analyses that use the 
multispecies coalescent model compare the probability of trees with 
differing numbers of OTUs to identify optimal partitions of the data 
(e.g. spedeSTEM, BPP). Salter <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0080" class="bibLink tab-link" id="mec12413-bib-0080R" data-tab="pane-pcw-references">2013</a></span>)
 extend this strategy to its maximum extent by calculating the 
probability of the phylogeny that treats individual samples as putative 
lineages. The putative lineages are then sequentially collapsed on the 
basis of which samples are most closely related, the probability of the 
species tree is recalculated, and information theory (Burnham &amp; 
Anderson <span><a href="#mec12413-bib-0008" class="bibLink tab-link" id="mec12413-bib-0008R" data-tab="pane-pcw-references">2002</a></span>) is used to identify the optimal model of lineage composition. Thus, <b>spedeSTEM discovery</b> can be used to simultaneously delimit evolutionary lineages and assign samples to these lineages (<a href="http://carstenslab.org.ohio-state.edu/software.html" class="linkBehavior">http://carstenslab.org.ohio-state.edu/software.html</a>).
               </p>
               
            </section>
            <section class="article-section__content Box " id="mec12413-sec-0010">
               
               <h2 class="article-section__title section__title section2" id="mec12413-sec-0010-title"> Species validation approaches require the user to assign samples to putative lineages</h2>
               
               <p>The popular validation method <b>BPP</b> (Yang &amp; Rannala <span><a href="#mec12413-bib-0093" class="bibLink tab-link" id="mec12413-bib-0093R" data-tab="pane-pcw-references">2010</a></span>) implements a reversible jump Markov chain Monte Carlo (rjMCMC) search of parameter space that includes <i>θ</i>,
 population divergence and estimated distributions of gene trees from 
multiple loci. The method takes sequence data as input and also requires
 the user to define the topology of the species tree. Given this 
information, the algorithm implemented in BPP then traverses the 
parameter space to compute the posterior probability of the proposed 
nodes of the species tree. While inaccurately specified guide trees can 
lead to false‐positive delimitations, the accuracy of BPP does not 
generally appear dependent on its ability to estimate gene trees. As 
this manuscript was in review, an improvement to the rjMCMC was 
described (Rannala &amp; Yang <span><a href="#mec12413-bib-0074" class="bibLink tab-link" id="mec12413-bib-0074R" data-tab="pane-pcw-references">2013</a></span>) (<a href="http://abacus.gene.ucl.ac.uk/software.html" class="linkBehavior">http://abacus.gene.ucl.ac.uk/software.html</a>).
               </p>
               
               <p>The validation approach <b>spedeSTEM</b> was developed to test species boundaries in a system with existing subspecies taxonomy (Carstens &amp; Dewey <span><a href="#mec12413-bib-0010" class="bibLink tab-link" id="mec12413-bib-0010R" data-tab="pane-pcw-references">2010</a></span>).
 The approach computes the probability of the gene trees given the 
species tree for all hierarchical permutations of lineage grouping, and 
therefore, complex cases such as that described by Carstens and Dewey 
(four species with 1–4 described subspecies) can be evaluated. Because 
the –lnL (ST|GTs) is computed directly by STEM (Kubatko <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0049" class="bibLink tab-link" id="mec12413-bib-0049R" data-tab="pane-pcw-references">2009</a></span>),
 rather than estimated, phylogenetic uncertainty in the species tree 
does not affect species delimitations. However, accuracy of spedeSTEM is
 dependent on the quality of the gene tree estimates (<a href="http://carstenslab.org.ohio-state.edu/software.html" class="linkBehavior">http://carstenslab.org.ohio-state.edu/software.html</a>).
               </p>
               
            </section>
         </div>
         
         <p>Knowles &amp; Carstens (<span><a href="#mec12413-bib-0047" class="bibLink tab-link" id="mec12413-bib-0047R" data-tab="pane-pcw-references">2007</a></span>) suggested that the multispecies coalescent model (Rannala &amp; Yang <span><a href="#mec12413-bib-0073" class="bibLink tab-link" id="mec12413-bib-0073R" data-tab="pane-pcw-references">2003</a></span>; Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec12413-fea-0001">1</a>)
 offers an important opportunity for species delimitation. Relative to 
the delimitation of evolutionary lineages, the important difference 
between species tree methods that implement this model and conventional 
methods of phylogenetic inference is the shift in what constitutes an 
operational taxonomic unit (OTU). Rather than using a single individual 
or several representative individuals as exemplars, the OTUs are 
explicitly evolutionary lineages with multiple samples contained within 
each lineage. Thus, species tree estimation methods model the membership
 of individuals to evolutionary lineages, in addition to the coalescent 
model of population processes used in the phylogeny inference. The 
species tree paradigm enables the relationships among lineages and the 
membership of individual samples in these lineages to be evaluated in a 
rigorous statistical framework (Edwards <span><a href="#mec12413-bib-0020" class="bibLink tab-link" id="mec12413-bib-0020R" data-tab="pane-pcw-references">2009</a></span>).
 The species tree approach to species delimitation represents a 
fundamental shift in how genetic data can be used to delimit lineages 
(de Queiroz <span><a href="#mec12413-bib-0072" class="bibLink tab-link" id="mec12413-bib-0072R" data-tab="pane-pcw-references">2007</a></span>).
 Rather than equating gene trees with a species tree or basing species 
status on some genetic threshold (e.g. Baker &amp; Bradley <span><a href="#mec12413-bib-0003" class="bibLink tab-link" id="mec12413-bib-0003R" data-tab="pane-pcw-references">2006</a></span>),
 the relationship between the gene trees and the lineage history is 
modelled probabilistically using coalescent theory (Hudson <span><a href="#mec12413-bib-0037" class="bibLink tab-link" id="mec12413-bib-0037R" data-tab="pane-pcw-references">1991</a></span>; Rosenberg <span><a href="#mec12413-bib-0079" class="bibLink tab-link" id="mec12413-bib-0079R" data-tab="pane-pcw-references">2002</a></span>).
 Adopting this explicit model‐based approach also helps researchers to 
circumvent pernicious problems that result when genetic thresholds are 
applied to genetic data—the detection biases that arise from the timing 
and method of speciation and failure of any threshold to take into 
account the stochastic variance associated with genetic processes.
         </p>
         
         <p>By far the most popular of the species tree–based methods 
for species delimitation is the Bayesian method BPP (Yang &amp; Rannala <span><a href="#mec12413-bib-0093" class="bibLink tab-link" id="mec12413-bib-0093R" data-tab="pane-pcw-references">2010</a></span>).
 It uses a reversible jump Markov chain Monte Carlo approach to 
calculate the posterior probabilities of competing models that contain 
greater or fewer lineages. BPP takes sequence data from multiple loci as
 input and simulates posterior distributions of gene trees and species 
delimitations from these data. BPP does not consider species tree 
uncertainty but rather requires users to input a topology (i.e. species 
tree) to guide the Markov chain. Another method derived from the 
multispecies coalescent is spedeSTEM (Ence &amp; Carstens <span><a href="#mec12413-bib-0021" class="bibLink tab-link" id="mec12413-bib-0021R" data-tab="pane-pcw-references">2011</a></span>), a maximum‐likelihood approach that uses STEM (Kubatko <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0049" class="bibLink tab-link" id="mec12413-bib-0049R" data-tab="pane-pcw-references">2009</a></span>)
 to calculate the probability of models that contain differing numbers 
of evolutionary lineages and then uses information theory (see Burnham 
&amp; Anderson <span><a href="#mec12413-bib-0008" class="bibLink tab-link" id="mec12413-bib-0008R" data-tab="pane-pcw-references">2002</a></span>)
 to rank these models. SpedeSTEM takes gene trees as input (i.e. a 
single point estimate per locus) and thus does not consider uncertainty 
in the gene tree estimates, but does not require the user to input a 
guide topology. Both BPP and spedeSTEM assume that all of the shared 
polymorphism is the result of unsorted ancestral polymorphism. If gene 
flow has led to shared polymorphism across lineages, the underlying 
species tree is likely to be difficult to infer (Eckert &amp; Carstens <span><a href="#mec12413-bib-0019" class="bibLink tab-link" id="mec12413-bib-0019R" data-tab="pane-pcw-references">2008</a></span>), and thus, the accuracy of the species delimitation is likely to decrease (e.g. Camargo <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>).
         </p>
         
         <p>Species delimitation methods can be categorized on the basis
 of whether sample assignment is required. Cases where the data are 
input without a priori partitioning have been described as discovery 
methods and those that require the samples to be partitioned prior to 
analysis as validation approaches (Ence &amp; Carstens <span><a href="#mec12413-bib-0021" class="bibLink tab-link" id="mec12413-bib-0021R" data-tab="pane-pcw-references">2011</a></span>).
 In this dichotomy, discovery methods can be applied to any system, 
whereas validation approaches are limited to systems where either 
populations are clearly delineated (e.g. Camargo <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>), where existing subspecific taxonomy can serve as the basis for lineage assignment (e.g. Carstens &amp; Dewey <span><a href="#mec12413-bib-0010" class="bibLink tab-link" id="mec12413-bib-0010R" data-tab="pane-pcw-references">2010</a></span>)
 or where other characters can be used to formulate a hypothesis for 
delimitation. While all existing validation approaches implement a 
multispecies coalescent model, some discovery approaches also estimate a
 species tree (e.g. O'Meara's heuristic method). Many authors have 
approached species delimitation in more complex systems by using 
combinations of these approaches to species delimitation, and we 
highlight several in the next section.
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0004">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0004-title"> Strategies for complex empirical investigations</h2>
         
         <p>Many species delimitation investigations focus on 
understudied systems where the existing taxonomy is poor, and often 
these investigations analyse data collected from a single genetic locus 
as a first pass at estimating species diversity. For example, Esselstyn <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0022" class="bibLink tab-link" id="mec12413-bib-0022R" data-tab="pane-pcw-references">2012</a></span>) describe one such system, the <i>Hipposideros</i>
 bats of the Philippines, where the most recent taxonomic work was 
conducted in 1963. Their approach was to collect data from a single 
mitochondrial locus and analyse them under the general mixed Yule 
coalescent (GMYC) model. This model, developed by Pons <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0067" class="bibLink tab-link" id="mec12413-bib-0067R" data-tab="pane-pcw-references">2006</a></span>),
 is related to lineage through time plots, but the GMYC models the 
transition in a particular gene tree between allelic coalescence and 
cladogenesis. Once this transition point is identified, individuals can 
be assigned to species on the basis of whether they coalesce with a 
given individual before or after this transition point. Esselstyn <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0022" class="bibLink tab-link" id="mec12413-bib-0022R" data-tab="pane-pcw-references">2012</a></span>)
 also conduct a simulation study prior to the analysis of their 
empirical data, in part to calibrate the prior distributions assumed in 
the analysis. Their analysis suggests that species diversity is 
dramatically underdescribed within Philippine <i>Hipposideros</i> bats, 
and they consider the lineages identified via their application of the 
GMYC as putative lineages and targets for future research.
         </p>
         
         <p>In another system where sample assignment to putative lineage was not clear, Leaché &amp; Fujita (<span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>)
 investigated forest geckos from Western Africa. They adopted a 
two‐stage approach to species delimitation, using a species discovery 
approach (in this case, Structurama) to assign individuals to putative 
groups and then subsequently validated these groups using BPP. Their 
approach alleviates substantial shortcomings of these methods when each 
is used independently, the inability of Structurama to model the pattern
 of population divergence and the requirement of BPP that samples are 
assigned to populations prior to the analysis. This pairing of a species
 discovery method with BPP has been followed by a number of authors 
(e.g. Barrett &amp; Freudenstein <span><a href="#mec12413-bib-0004" class="bibLink tab-link" id="mec12413-bib-0004R" data-tab="pane-pcw-references">2011</a></span>; Setiadi <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0083" class="bibLink tab-link" id="mec12413-bib-0083R" data-tab="pane-pcw-references">2011</a></span>).
 A particularly useful approach may be the pairing of discovery methods 
that model the species tree to identify putative lineages. For example, 
Niemiller <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0061" class="bibLink tab-link" id="mec12413-bib-0061R" data-tab="pane-pcw-references">2012</a></span>) use the heuristic method described by O'Meara (<span><a href="#mec12413-bib-0064" class="bibLink tab-link" id="mec12413-bib-0064R" data-tab="pane-pcw-references">2010</a></span>)
 to jointly estimate the number of species and the species tree topology
 in a system of cavefish from southeastern North America. O'Meara's 
method assumes that gene trees will tend to agree on interspecific 
history (as measured by gene tree parsimony score given a species tree) 
and disagree within species (as measured by lack of triplet overlap) and
 discovers putative lineages by minimizing the weighted sum of these 
across the possible species trees and assignments. Niemiller <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0061" class="bibLink tab-link" id="mec12413-bib-0061R" data-tab="pane-pcw-references">2012</a></span>) were able to use this combined approach to delimit multiple lineages of cavefish.
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0005">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0005-title"> Potential shortcomings of validation approaches</h2>
         
         <p>Validation approaches, such as BPP and spedeSTEM, are often 
given more weight by empirical investigations because they explicitly 
model the process of lineage diversification. However, there are 
shortcomings inherent to these methods, so results should be interpreted
 with caution. For example, spedeSTEM does not estimate <i>θ</i> when it computes the ML species tree (<i>θ</i> is supplied by the user) and assumes that this parameter is unchanging across the phylogeny. In practice, <i>θ</i>
 influences the absolute probability of the ST rather than the relative 
probability, and thus, AIC model probabilities are not influenced by 
imprecision in this parameter. The accuracy of spedeSTEM is dependent on
 the quality of the gene tree estimates, and while it is straightforward
 to assess the quality of such estimates (e.g. using bootstrap values or
 Bayesian posterior probabilities), identification of a single threshold
 of nodal support below which results from spedeSTEM are suspect 
requires a simulation study tailored to the specifics of a given system.
 Conversely, BPP simulates a posterior distribution of gene trees, 
estimates <i>θ</i> on each branch of the species tree and calculates the
 posterior probabilities of the species delimitations. One shortcoming 
of BPP, first illustrated by Leaché &amp; Fujita (<span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>),
 is its dependency on the accuracy of the guide tree. BPP requires that 
the user specifies the guide tree topology (i.e. the species tree), 
which is used by the program to guide the reversible jumps in the Markov
 chain. Leaché &amp; Fujita (<span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>)
 conducted simulations to explore the effect of inaccuracies in the 
guide tree on the species delimitations and demonstrate that if this 
guide topology is specified incorrectly, BPP is likely to delimit each 
of the putative lineages (i.e. each OTU of the species tree). This is 
due to the artificial increase in genetic divergence between sister 
lineages. This shortcoming can be partially mitigated by directly 
estimating the topology of the guide tree by grouping samples assigned 
to populations into OTUs for a species tree analysis. For example, 
Leaché and Fujita use *Beast (Heled &amp; Drummond <span><a href="#mec12413-bib-0034" class="bibLink tab-link" id="mec12413-bib-0034R" data-tab="pane-pcw-references">2010</a></span>)
 to directly estimate the species tree, which is subsequently used as a 
guide topology. In their case, they were able to produce a 
well‐supported estimate of the species tree topology when assuming the 
population assignments produced by Structurama. However, species trees 
with low levels of divergence are difficult to estimate, particularly 
when the membership of samples to lineages is unknown (O'Meara <span><a href="#mec12413-bib-0064" class="bibLink tab-link" id="mec12413-bib-0064R" data-tab="pane-pcw-references">2010</a></span>), and estimates of the species tree topology are not always clear (e.g. Carstens &amp; Satler <span><a href="#mec12413-bib-0011" class="bibLink tab-link" id="mec12413-bib-0011R" data-tab="pane-pcw-references">2013</a></span>).
 In addition, Bayesian approaches to species tree estimation generate a 
posterior distribution of species trees, and a consensus tree from this 
posterior distribution is often used as a guide topology for the 
subsequent BPP analysis (e.g. Leaché &amp; Fujita <span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>).
 It is important to recognize that there is no guarantee that the 
consensus tree will share the topology of the maximum‐likelihood 
estimate (MLE) of the species tree (Felsenstein <span><a href="#mec12413-bib-0023" class="bibLink tab-link" id="mec12413-bib-0023R" data-tab="pane-pcw-references">2004</a></span>) or that the posterior distribution of a Bayesian analysis will contain the MLE. Leaché &amp; Fujita (<span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>)
 caution readers to carefully consider the guide tree in a BPP analysis;
 if this topology does not reflect the true history of diversification, 
BPP analyses are prone to falsely delimiting species.
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0006">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0006-title"> Incongruence across results</h2>
         
         <p>Many species delimitation investigations report discordance across results from various methods (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec12413-tbl-0002">2</a>).
 One explanation might be that the assumptions of one or more methods 
are violated, leading to an incorrect result. However, the nature of the
 speciation process leads us to expect incongruence if two conditions 
are met: first, most researchers have followed Darwin (<span><a href="#mec12413-bib-0014" class="bibLink tab-link" id="mec12413-bib-0014R" data-tab="pane-pcw-references">1859</a></span>)
 in considering speciation as a gradual process, with exceptions for 
some particular modes such as allopolyploid speciation (e.g. Kim <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0045" class="bibLink tab-link" id="mec12413-bib-0045R" data-tab="pane-pcw-references">2008</a></span>).
 If speciation in most cases is gradual, then incongruent results would 
be expected if the methods applied had differing degrees of statistical 
power to detect independent lineages. The results of simulation studies 
such as that by Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
 suggest that this is the case, but until such studies are conducted for
 a broad range of empirical systems we cannot determine whether certain 
methods have better or worse statistical power in all cases or only 
under specific conditions. Clearly, researchers should proceed by 
conducting simulation testing and choosing methods that have 
complementary shortcomings.
         </p>
         
         <p>For example, spedeSTEM and BPP are complementary validation 
approaches because they use different strategies for simplifying the 
parameter space of species delimitation. BPP takes sequences as input 
and uses genealogical sampling and reversible jump MCMC to evaluate 
species delimitations given a guide tree. Conversely, spedeSTEM takes 
previously estimated gene trees as input, calculates the 
maximum‐likelihood species tree for species delimitation and identifies 
the best of many possible delimitations using information theory. 
Because spedeSTEM computes the likelihood of species trees (i.e. it does
 not estimate this parameter) for all possible permutations of the 
putative lineages, it is robust to phylogenetic error in the species 
tree. However, it is clear from simulation studies that inaccuracy in 
spedeSTEM is largely conservative; it fails to delimit what are in 
reality separate lineages (Ence &amp; Carstens <span><a href="#mec12413-bib-0021" class="bibLink tab-link" id="mec12413-bib-0021R" data-tab="pane-pcw-references">2011</a></span>),
 rather than falsely delimiting the putative lineages as independent, as
 may occur in BPP if the guide tree is mis‐specified. In this way, 
spedeSTEM and BPP are complementary approaches to species validation 
because each appears prone to failure in an opposite manner. When both 
programs are used to validate population assignments and the results are
 congruent, researchers can reasonably infer that BPP has not been 
mislead due to an inaccurately designated guide tree and that spedeSTEM 
has not failed to detect independent lineages due to poorly estimated 
gene trees. Conversely, incongruence in the results probably indicates 
that one of these problems is likely evident.
         </p>
         
         <p>Another potential shortcoming of species delimitation 
investigation is the limited sample sizes common to many multilocus 
investigations (see Fujita <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0027" class="bibLink tab-link" id="mec12413-bib-0027R" data-tab="pane-pcw-references">2012</a></span>).
 Before multilocus data became common, phylogeographical investigations 
often collected data from hundreds of samples (e.g. Avise <span><a href="#mec12413-bib-0002" class="bibLink tab-link" id="mec12413-bib-0002R" data-tab="pane-pcw-references">2000</a></span> and references therein). However, while the need for multilocus data for both phylogeography (e.g. Brumfield <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0006" class="bibLink tab-link" id="mec12413-bib-0006R" data-tab="pane-pcw-references">2003</a></span>) and species delimitation (e.g. Dupuis <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0018" class="bibLink tab-link" id="mec12413-bib-0018R" data-tab="pane-pcw-references">2012</a></span>)
 is by now firmly established, the addition of multiple loci to 
phylogeographical investigations has too often come at the expense of 
sampling. For the papers reviewed here, there is a negative trend 
between number of samples and the number of loci, but the correlation is
 not significant (<i>r&nbsp;</i>=<i>&nbsp;</i>−0.26; <i>P&nbsp;</i>=<i>&nbsp;</i>0.182).
 We attribute this phenomenon to the expense of collecting data using 
Sanger sequencing and anticipate that recently developed strategies for 
collecting phylogeographical data using high‐throughput sequencing (e.g.
 McCormack <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0058" class="bibLink tab-link" id="mec12413-bib-0058R" data-tab="pane-pcw-references">2013</a></span>)
 will enable researchers to collect hundreds of loci from hundreds of 
individual samples. At an absolute minimum, researchers should collect 
at least 10 samples from all putative lineages because this number leads
 to a reasonably high probability (&gt;90%) of sampling the deepest 
coalescent events in each population (Saunders <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0081" class="bibLink tab-link" id="mec12413-bib-0081R" data-tab="pane-pcw-references">1994</a></span>),
 thus ensuring that most meaningful genetic variation (at least to the 
question of species boundaries) is sampled. However, optimal sampling 
levels are highly dependent on both the study system and the analyses 
used to delimit species—another justification for simulation testing 
(e.g. Camargo <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>; Esselstyn <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0022" class="bibLink tab-link" id="mec12413-bib-0022R" data-tab="pane-pcw-references">2012</a></span>).
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0007">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0007-title"> How to approach species delimitation</h2>
         
         <p>A shortcoming of empirical investigations to date is that 
researchers limit the number and scope of their species delimitation 
analyses. In 2006, a symposium organized by John Wiens (sponsored by the
 Society of Systematic Biologists) at the Evolution Annual meeting 
highlighted the need for new approaches to species delimitation (see 
Wiens <span><a href="#mec12413-bib-0090" class="bibLink tab-link" id="mec12413-bib-0090R" data-tab="pane-pcw-references">2007</a></span>). Since then, dozens of novel methods have been developed (see Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec12413-fea-0002">2</a>).
 Despite this recent abundance of methods, researchers commonly present 
results from only a handful (mean 2.25&nbsp;±&nbsp;1.2; Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec12413-tbl-0001">1</a>). When viewed in the context of the difference in parameterization between an <i>ideal method</i>
 for species delimitation and existing methods, this is problematic 
because any existing method is forced to make a series of simplifying 
assumptions, any one of which could possibly be violated in a particular
 empirical system. Viewed in this light, incongruence in the results 
across methods is evidence of the individual shortcomings of one or more
 of approaches used to delimit species, given the data. By limiting the 
number and scope of the analyses applied to the data, researchers do not
 allow for the possibility that there are inherent differences in 
statistical power across analyses and thus limit their ability to 
identify incongruence across methods. Conversely, researchers who apply a
 wide range of species delimitation analyses to their data can have one 
of two outcomes. First, they may find that all of their results are 
generally in agreement, which strengthens their confidence in these 
results and the resulting species delimitations. Second, in cases where 
there is incongruence across results, researchers may be prompted to 
explore the possible causes of this incongruence and are likely to be 
conservative in the inferences drawn from the analysis. Either of these 
outcomes is preferable to conclusions about species limits that are 
based on a single or a limited number of analyses and thus more likely 
to be inaccurate. In our reading of the literature, most workers either 
state explicitly or assume that the goal of their work is to identify 
evolutionary lineages, essentially evolutionarily significant units <i>sensu</i> Moritz (<span><a href="#mec12413-bib-0059" class="bibLink tab-link" id="mec12413-bib-0059R" data-tab="pane-pcw-references">1994</a></span>). Species delimitation thus has important implications to the conservation of biodiversity.
         </p>
         
         <div class="article-table-content" id="mec12413-tbl-0001">
            <header class="article-table-caption"><span class="table-caption__label">Table 1.
            </span>Properties of recent species delimitation studies. 
Shown for each study are the number of assumed species prior to 
analysis, the number of inferred lineages after the analysis, the number
 of samples and number of loci used, as well as a list of the discovery 
and validation approaches used
            </header>
            
            
            
            <div class="article-table-content-wrapper">
               <table class="table article-section__table">
                  
                  
                  
                  
                  
                  
                  
                  
                  <thead>
                     
                     <tr>
                        
                        <th class="bottom-bordered-cell right-bordered-cell left-aligned">Reference</th>
                        
                        <th class="bottom-bordered-cell center-aligned">Existing species</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Species after delimitation</th>
                        
                        <th class="bottom-bordered-cell center-aligned">No. of samples</th>
                        
                        <th class="bottom-bordered-cell center-aligned">No. of loci</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Discovery approaches</th>
                        
                        <th class="bottom-bordered-cell left-aligned">Validation approaches</th>
                        
                     </tr>
                     
                  </thead>
                  
                  <tbody>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Avila <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0001" class="bibLink tab-link" id="mec12413-bib-0001R" data-tab="pane-pcw-references">2006</a></span>)
                        </td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">&gt;12</td>
                        
                        <td class="center-aligned">293</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">Statistical parsimony (NCA)</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Barrett &amp; Freudenstein (<span><a href="#mec12413-bib-0004" class="bibLink tab-link" id="mec12413-bib-0004R" data-tab="pane-pcw-references">2011</a></span>)
                        </td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="left-aligned">3</td>
                        
                        <td class="center-aligned">162</td>
                        
                        <td class="center-aligned">5</td>
                        
                        <td class="left-aligned">Morphological cluster analysis, PCA</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Burbrink <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0007" class="bibLink tab-link" id="mec12413-bib-0007R" data-tab="pane-pcw-references">2011</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">1</td>
                        
                        <td class="center-aligned">45</td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="left-aligned">Structurama</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="left-aligned">1</td>
                        
                        <td class="center-aligned">505</td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="left-aligned">None</td>
                        
                        <td class="left-aligned">spedeSTEM, BPP, ABC</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Carstens &amp; Dewey (<span><a href="#mec12413-bib-0010" class="bibLink tab-link" id="mec12413-bib-0010R" data-tab="pane-pcw-references">2010</a></span>)
                        </td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="left-aligned">7</td>
                        
                        <td class="center-aligned">42</td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">None</td>
                        
                        <td class="left-aligned">spedeSTEM, Bayes Factors</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Carstens &amp; Satler (<span><a href="#mec12413-bib-0011" class="bibLink tab-link" id="mec12413-bib-0011R" data-tab="pane-pcw-references">2013</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">2</td>
                        
                        <td class="center-aligned">82</td>
                        
                        <td class="center-aligned">21</td>
                        
                        <td class="left-aligned">Structurama, Gaussian Clustering</td>
                        
                        <td class="left-aligned">spedeSTEM, BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Duminil <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0017" class="bibLink tab-link" id="mec12413-bib-0017R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="left-aligned">Unknown</td>
                        
                        <td class="left-aligned">Unstated</td>
                        
                        <td class="center-aligned">103</td>
                        
                        <td class="center-aligned">7</td>
                        
                        <td class="left-aligned">Morphometric clustering; structure</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Esselstyn <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0022" class="bibLink tab-link" id="mec12413-bib-0022R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">13</td>
                        
                        <td class="left-aligned">18–19</td>
                        
                        <td class="center-aligned">413</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">GMYC</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Florio <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0024" class="bibLink tab-link" id="mec12413-bib-0024R" data-tab="pane-pcw-references">2012</a></span>);
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">2</td>
                        
                        <td class="center-aligned">111</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">Canonical variates analysis</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Flot <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0025" class="bibLink tab-link" id="mec12413-bib-0025R" data-tab="pane-pcw-references">2010</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned"></td>
                        
                        <td class="center-aligned">74</td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="left-aligned">Haplowebs</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Hamilton <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0031" class="bibLink tab-link" id="mec12413-bib-0031R" data-tab="pane-pcw-references">2011</a></span>)
                        </td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="left-aligned">3</td>
                        
                        <td class="center-aligned">147</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">Combo WP and barcoding gap, monophyly, GMYC</td>
                        
                        <td class="left-aligned"></td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Kelly <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0044" class="bibLink tab-link" id="mec12413-bib-0044R" data-tab="pane-pcw-references">2008</a></span>)
                        </td>
                        
                        <td class="center-aligned">39</td>
                        
                        <td class="left-aligned">1</td>
                        
                        <td class="center-aligned">114</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">WP</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Leaché &amp; Fujita (<span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">3</td>
                        
                        <td class="center-aligned">51</td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">Structure</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Leavitt <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0053" class="bibLink tab-link" id="mec12413-bib-0053R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">19</td>
                        
                        <td class="left-aligned">2</td>
                        
                        <td class="center-aligned">414</td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">Structure</td>
                        
                        <td class="left-aligned">BPP, mean genetic distance</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Leliart <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0054" class="bibLink tab-link" id="mec12413-bib-0054R" data-tab="pane-pcw-references">2009</a></span>)
                        </td>
                        
                        <td class="center-aligned">19</td>
                        
                        <td class="left-aligned">13</td>
                        
                        <td class="center-aligned">175</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">GMYC, statistical parsimony (NCA)—clades that exceed 95% cut off</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Niemiller <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0061" class="bibLink tab-link" id="mec12413-bib-0061R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">19</td>
                        
                        <td class="center-aligned">135</td>
                        
                        <td class="center-aligned">9</td>
                        
                        <td class="left-aligned">O'Meara clustering</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Pons <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0067" class="bibLink tab-link" id="mec12413-bib-0067R" data-tab="pane-pcw-references">2006</a></span>)
                        </td>
                        
                        <td class="center-aligned">24</td>
                        
                        <td class="left-aligned">54</td>
                        
                        <td class="center-aligned">468</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">Parsimony network, PAA, CHA, WP, GMYC</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Puillandre <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0069" class="bibLink tab-link" id="mec12413-bib-0069R" data-tab="pane-pcw-references">2009</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">4</td>
                        
                        <td class="center-aligned">44</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="left-aligned">Elliptic Fourier analysis on shape to the mollusc shell; qualitative phylogenetic evidence</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Puillandre <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0070" class="bibLink tab-link" id="mec12413-bib-0070R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">43</td>
                        
                        <td class="left-aligned">27</td>
                        
                        <td class="center-aligned">1000</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="left-aligned">GMYC, ABGD</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Rielly <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0076" class="bibLink tab-link" id="mec12413-bib-0076R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">3</td>
                        
                        <td class="center-aligned">50</td>
                        
                        <td class="center-aligned">14</td>
                        
                        <td class="left-aligned">Structurama</td>
                        
                        <td class="left-aligned">Monophyly</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Rosell <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0078" class="bibLink tab-link" id="mec12413-bib-0078R" data-tab="pane-pcw-references">2010</a></span>)
                        </td>
                        
                        <td class="center-aligned">23</td>
                        
                        <td class="left-aligned">Unstated</td>
                        
                        <td class="center-aligned">83</td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="left-aligned">Discriminant analysis on morphological data; qualitative phylogenetic analysis</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Salter <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0080" class="bibLink tab-link" id="mec12413-bib-0080R" data-tab="pane-pcw-references">2013</a></span>)
                        </td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">2</td>
                        
                        <td class="center-aligned">142</td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">Structure, Brownie heuristic, bGMYC, step up STEM</td>
                        
                        <td class="left-aligned">spedeSTEM, BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Setiadi <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0083" class="bibLink tab-link" id="mec12413-bib-0083R" data-tab="pane-pcw-references">2011</a></span>)
                        </td>
                        
                        <td class="center-aligned">4</td>
                        
                        <td class="left-aligned">11</td>
                        
                        <td class="center-aligned">683</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="left-aligned">Genealogical‐concordance criterion</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Stech <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0085" class="bibLink tab-link" id="mec12413-bib-0085R" data-tab="pane-pcw-references">2013</a></span>)
                        </td>
                        
                        <td class="center-aligned">9</td>
                        
                        <td class="left-aligned">9</td>
                        
                        <td class="center-aligned">70</td>
                        
                        <td class="center-aligned">2</td>
                        
                        <td class="left-aligned">None</td>
                        
                        <td class="left-aligned">Comparison with morphology</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Stielow <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0087" class="bibLink tab-link" id="mec12413-bib-0087R" data-tab="pane-pcw-references">2011</a></span>)
                        </td>
                        
                        <td class="left-aligned">Unknown</td>
                        
                        <td class="left-aligned">24</td>
                        
                        <td class="center-aligned">142</td>
                        
                        <td class="center-aligned">1</td>
                        
                        <td class="left-aligned">Qualitative phylogenetic analysis</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Weisrock <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0088" class="bibLink tab-link" id="mec12413-bib-0088R" data-tab="pane-pcw-references">2010</a></span>)
                        </td>
                        
                        <td class="left-aligned">Unknown</td>
                        
                        <td class="left-aligned">16</td>
                        
                        <td class="center-aligned">216</td>
                        
                        <td class="center-aligned">5</td>
                        
                        <td class="left-aligned">Structure; gsi; phylogeny estimation</td>
                        
                        <td class="left-aligned">None</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Welton <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0089" class="bibLink tab-link" id="mec12413-bib-0089R" data-tab="pane-pcw-references">2013</a></span>)
                        </td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">8</td>
                        
                        <td class="center-aligned">53</td>
                        
                        <td class="center-aligned">5</td>
                        
                        <td class="left-aligned">Structure</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                     <tr>
                        
                        <td class="right-bordered-cell left-aligned">Zhou <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0094" class="bibLink tab-link" id="mec12413-bib-0094R" data-tab="pane-pcw-references">2012</a></span>)
                        </td>
                        
                        <td class="center-aligned">3</td>
                        
                        <td class="left-aligned">1</td>
                        
                        <td class="center-aligned">394</td>
                        
                        <td class="center-aligned">6</td>
                        
                        <td class="left-aligned">None</td>
                        
                        <td class="left-aligned">BPP</td>
                        
                     </tr>
                     
                  </tbody>
                  
               </table>
            </div>
            
            
            <div class="article-section__table-source"></div>
         </div>
         
         <p>To illustrate the importance of basing species delimitations
 on a conservative consensus across a wide range of methods, consider a 
recent investigation by Salter <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0080" class="bibLink tab-link" id="mec12413-bib-0080R" data-tab="pane-pcw-references">2013</a></span>).
 They investigated species boundaries in a clade of trapdoor spiders 
from the west coast of North America using six different approaches to 
species delimitation (Fig.&nbsp;<a href="#mec12413-fig-0001">1</a>). Results from these approaches delimited between 3 and 18 lineages, and Satler <i>et&nbsp;al</i>.
 interpreted this incongruence in a conservative manner by recognizing 
three of these lineages as species. It is possible that additional 
lineages identified by some of the analyses represent species‐level 
diversity, in which case Satler <i>et&nbsp;al</i>. failed at species 
delimitation by being overly conservative in their interpretation of the
 results. However, each of the methods used by Satler <i>et&nbsp;al</i>.
 contain evolutionary models with inherent simplifications of parameter 
space, and because these simplifications are nonoverlapping in terms of 
their assumptions and the parameters estimated from the data, the use of
 multiple approaches forced Satler <i>et&nbsp;al</i>. into reconciling 
their results into unified conceptual (although not quantitative) 
evolutionary model that is the basis for the requisite interpretations 
pertaining to species assignment. This cautious approach to delimitation
 assumes that failing to delimit species is preferable to falsely 
delimiting entities that do not represent actual evolutionarily 
lineages, particularly when the goal of the analysis is species 
description (Box&nbsp;<a class="featureLink scrollableLink" title="Link to feature" href="#mec12413-fea-0003">3</a>).
 This approach is consistent with the spirit of integrative taxonomy, 
which argues that taxonomic inference should be based on congruence 
across analyses that utilize multiple sources of data (e.g. Padial <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0065" class="bibLink tab-link" id="mec12413-bib-0065R" data-tab="pane-pcw-references">2010</a></span>; Schlick‐Steiner <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0082" class="bibLink tab-link" id="mec12413-bib-0082R" data-tab="pane-pcw-references">2010</a></span>).
 Species delimitation is perhaps the product of phylogeographical 
research that has the most tangible implications to the world beyond 
academia and as a field we should exercise care and caution when drawing
 taxonomic conclusions from our investigations. At the same time, it is 
important to have confidence in the data and the methods used to analyse
 these data and to make taxonomic recommendations in systems where they 
are warranted.
         </p>
         
         <section class="article-section__inline-figure">
            <figure class="figure" id="mec12413-fig-0001"><a target="_blank" href="https://onlinelibrary.wiley.com/cms/asset/588b0a74-2c7e-4683-b7b3-b910d356ae5b/mec12413-fig-0001-m.jpg">
                  <picture>
                     <source srcset="/cms/asset/588b0a74-2c7e-4683-b7b3-b910d356ae5b/mec12413-fig-0001-m.jpg" media="(min-width: 1650px)"><img class="figure__image" src="mec12413-fig-0001-m.png" data-lg-src="/cms/asset/588b0a74-2c7e-4683-b7b3-b910d356ae5b/mec12413-fig-0001-m.jpg" alt="image" title="image"></picture></a><figcaption class="figure__caption">
                  <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                     <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=mec12413-fig-0001&amp;doi=10.1111%2Fmec.12413" class="ppt-figure-link">PowerPoint</a></div>
                  </div>
                  <div>Incongruent results from an empirical system. Six loci were used to delimit lineages in <i>Aliatypus</i>
 species complex, a group of trapdoor spiders from southern California 
(localities shown by dots in the inset map). Results from six methods 
(named at left) are depicted, with bars coloured to highlight congruence
 across methods. Numbers in parentheses represent the number of lineages
 delimited using each method. Also shown is an estimate of the species 
tree made using *Beast, with posterior probabilities shown under each 
node. Immediately above the species tree, thin coloured lines are used 
to identify the three species described by Satler <i>et&nbsp;al</i>. This figure is redrawn from Salter <i>et&nbsp;al</i>. (<span class="figureLink bibLink tab-link"><a href="#mec12413-bib-0080" class="bibLink tab-link" id="mec12413-bib-0080R" data-tab="pane-pcw-references">2013</a></span>).
                  </div>
               </figcaption>
            </figure>
         </section>
         
         <div class="feature" id="mec12413-fea-0003">
            <h3><span class="feature-label">Box 3. </span>Species concepts and species delimitation
            </h3>
            <section class="article-section__content Box " id="mec12413-sec-0011">
               
               <p>Many recent investigations into species limits either 
explicitly state or imply that a primary goal of the research is to 
identify independent evolutionary lineages, which are equivalent to 
metapopulation lineages through time (Simpson <span><a href="#mec12413-bib-0084" class="bibLink tab-link" id="mec12413-bib-0084R" data-tab="pane-pcw-references">1951</a></span>; Wiley <span><a href="#mec12413-bib-0092" class="bibLink tab-link" id="mec12413-bib-0092R" data-tab="pane-pcw-references">1978</a></span>; de Queiroz <span><a href="#mec12413-bib-0071" class="bibLink tab-link" id="mec12413-bib-0071R" data-tab="pane-pcw-references">2005</a></span>) and commonly interpreted as evolutionarily significant units (Moritz <span><a href="#mec12413-bib-0059" class="bibLink tab-link" id="mec12413-bib-0059R" data-tab="pane-pcw-references">1994</a></span>).
 Thus, the goal of most investigations that collect genetic data and 
utilize recently developed methods is to discover or validate units of 
biodiversity with their own unique evolutionary history. Much of this 
work is enabled by Kevin de Queiroz's reformulation of species concepts 
(de <span><a href="#mec12413-bib-0015" class="bibLink tab-link" id="mec12413-bib-0015R" data-tab="pane-pcw-references">1998</a></span>; de Queiroz <span><a href="#mec12413-bib-0071" class="bibLink tab-link" id="mec12413-bib-0071R" data-tab="pane-pcw-references">2005</a></span>, <span><a href="#mec12413-bib-0072" class="bibLink tab-link" id="mec12413-bib-0072R" data-tab="pane-pcw-references">2007</a></span>).
 Prior to this, species delimitation was directly tied to species 
concepts, to the extent that one researcher was required to assume a 
species concept and collect data that could directly speak to the 
criterion used to delimit species given this concept. Thus, 
morphological differentiation was required to delimit species under the 
morphological species concept, allelic coalescence was a requirement 
under the genealogical species concept (Baum &amp; Shaw <span><a href="#mec12413-bib-1001" class="bibLink tab-link" id="mec12413-bib-1001R" data-tab="pane-pcw-references">1995</a></span>), and reproductive isolation was a requirement under the biological species concept (Mayr <span><a href="#mec12413-bib-1002" class="bibLink tab-link" id="mec12413-bib-1002R" data-tab="pane-pcw-references">1942</a></span>).
 While this made species delimitation less ambiguous, it also produced a
 long‐standing argument that lead many researchers to ignore species 
delimitation as a potential research goal because such an exercise 
required the adoption of one of a number of competing species concepts 
and thus forced the researcher to what had become an interminable 
debate.
               </p>
               
               <p>de Queiroz offered a practical solution to this dilemma by redefining the criteria inherent to species concepts (de <span><a href="#mec12413-bib-0015" class="bibLink tab-link" id="mec12413-bib-0015R" data-tab="pane-pcw-references">1998</a></span>; de Queiroz <span><a href="#mec12413-bib-0071" class="bibLink tab-link" id="mec12413-bib-0071R" data-tab="pane-pcw-references">2005</a></span>, <span><a href="#mec12413-bib-0072" class="bibLink tab-link" id="mec12413-bib-0072R" data-tab="pane-pcw-references">2007</a></span>). Rather than treating criteria such as morphological differences, monophyly or reproductive isolation as <i>the</i> single indicator of species‐level differentiation, de Queiroz argued that <i>each</i>
 of these is a property of the evolutionary divergence of lineages. 
Because all species concepts (at least in de Queiroz's view) assume that
 species represent independent metapopulation lineages through time, the
 solution to the species problem is to treat the traditional criteria 
used to demarcate species as attributes that accumulate during the 
process of lineage diversification (de Queiroz <span><a href="#mec12413-bib-0071" class="bibLink tab-link" id="mec12413-bib-0071R" data-tab="pane-pcw-references">2005</a></span>). This generalized lineage concept (GLC) has been broadly adopted by recent investigations into species limits (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec12413-tbl-0002">2</a>) and indirectly promoted the development of recent approaches to species delimitation (e.g. Knowles &amp; Carstens <span><a href="#mec12413-bib-0047" class="bibLink tab-link" id="mec12413-bib-0047R" data-tab="pane-pcw-references">2007</a></span>).
 However, it is a mistake to characterize the entire field as adopting 
the GLC or to assume that the GLC is a prerequisite for species 
delimitation (e.g. Rosell <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0078" class="bibLink tab-link" id="mec12413-bib-0078R" data-tab="pane-pcw-references">2010</a></span>; Barrett &amp; Freudenstein <span><a href="#mec12413-bib-0004" class="bibLink tab-link" id="mec12413-bib-0004R" data-tab="pane-pcw-references">2011</a></span>; Duminil <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0017" class="bibLink tab-link" id="mec12413-bib-0017R" data-tab="pane-pcw-references">2012</a></span>).
 However, it is good practice to define some species concept when 
reporting investigation that includes a substantial species delimitation
 component, if only because this articulation enforces the need for a 
clear argument regarding the criteria used to recognize species.
               </p>
               
               <div class="article-table-content" id="mec12413-tbl-0002">
                  <header class="article-table-caption"><span class="table-caption__label">Table 2.
            </span>Strategies adopted by recent species delimitation studies. For the studies cited in Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#mec12413-tbl-0001">1</a>,
 we show the species concept and geographical context of the 
investigation and report if/how nongenetic data were used and how the 
authors handled incongruence across results, and the taxonomic 
implications of their study
                  </header>
                  
                  
                  
                  <div class="article-table-content-wrapper">
                     <table class="table article-section__table">
                        
                        
                        
                        
                        
                        
                        
                        <thead>
                           
                           <tr>
                              
                              <th class="bottom-bordered-cell right-bordered-cell left-aligned">Reference</th>
                              
                              <th class="bottom-bordered-cell left-aligned">Species concept</th>
                              
                              <th class="bottom-bordered-cell left-aligned">Geographical context</th>
                              
                              <th class="bottom-bordered-cell left-aligned">Nongenetic data used?</th>
                              
                              <th class="bottom-bordered-cell left-aligned">Interpretation of incongruence</th>
                              
                              <th class="bottom-bordered-cell left-aligned">Taxonomic treatment</th>
                              
                           </tr>
                           
                        </thead>
                        
                        <tbody>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Avila <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0001" class="bibLink tab-link" id="mec12413-bib-0001R" data-tab="pane-pcw-references">2006</a></span>)
                              </td>
                              
                              <td class="left-aligned">Inertial</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">n/a</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Barrett &amp; Freudenstein (<span><a href="#mec12413-bib-0004" class="bibLink tab-link" id="mec12413-bib-0004R" data-tab="pane-pcw-references">2011</a></span>)
                              </td>
                              
                              <td class="left-aligned">PSC</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Morphological</td>
                              
                              <td class="left-aligned">Conservative consensus</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Burbrink <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0007" class="bibLink tab-link" id="mec12413-bib-0007R" data-tab="pane-pcw-references">2011</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">ENM</td>
                              
                              <td class="left-aligned">Consensus</td>
                              
                              <td class="left-aligned">Elevate subspecies</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Camargo <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0009" class="bibLink tab-link" id="mec12413-bib-0009R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Results congruent</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Carstens &amp; Dewey (<span><a href="#mec12413-bib-0010" class="bibLink tab-link" id="mec12413-bib-0010R" data-tab="pane-pcw-references">2010</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Morphology as basis for validation</td>
                              
                              <td class="left-aligned">Favour one (spedeSTEM)</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Carstens &amp; Satler (<span><a href="#mec12413-bib-0011" class="bibLink tab-link" id="mec12413-bib-0011R" data-tab="pane-pcw-references">2013</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Conservative ‐ delimitations shared across results</td>
                              
                              <td class="left-aligned">Recommend new species</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Duminil <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0017" class="bibLink tab-link" id="mec12413-bib-0017R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">Morphological implied</td>
                              
                              <td class="left-aligned">Partly</td>
                              
                              <td class="left-aligned">Morphological</td>
                              
                              <td class="left-aligned">None offered</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Esselstyn <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0022" class="bibLink tab-link" id="mec12413-bib-0022R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Behaviour, echolocation</td>
                              
                              <td class="left-aligned">Attribute lack of congruence to small sample sizes.</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Florio <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0024" class="bibLink tab-link" id="mec12413-bib-0024R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry (w/ contact zone)</td>
                              
                              <td class="left-aligned">Morphology and ENM</td>
                              
                              <td class="left-aligned">Results congruent</td>
                              
                              <td class="left-aligned">New species described</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Flot <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0025" class="bibLink tab-link" id="mec12413-bib-0025R" data-tab="pane-pcw-references">2010</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Unclear</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">n/a</td>
                              
                              <td class="left-aligned">Recommend new species</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Hamilton <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0031" class="bibLink tab-link" id="mec12413-bib-0031R" data-tab="pane-pcw-references">2011</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Geographical</td>
                              
                              <td class="left-aligned">Consensus ‐ conservative</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Kelly <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0044" class="bibLink tab-link" id="mec12413-bib-0044R" data-tab="pane-pcw-references">2008</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Sympatry/allopatry?</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Results congruent</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Leaché &amp; Fujita (<span><a href="#mec12413-bib-0052" class="bibLink tab-link" id="mec12413-bib-0052R" data-tab="pane-pcw-references">2010</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Structure as basis for BPP</td>
                              
                              <td class="left-aligned">New species described</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Leavitt <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0053" class="bibLink tab-link" id="mec12413-bib-0053R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Not clear</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Existing taxonomy guide genetic analyses</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Leliart <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0054" class="bibLink tab-link" id="mec12413-bib-0054R" data-tab="pane-pcw-references">2009</a></span>)
                              </td>
                              
                              <td class="left-aligned">Phylogenetic</td>
                              
                              <td class="left-aligned">Mostly allopatric</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Consensus ‐ conservative</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Niemiller <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0061" class="bibLink tab-link" id="mec12413-bib-0061R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Conservative</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Pons <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0067" class="bibLink tab-link" id="mec12413-bib-0067R" data-tab="pane-pcw-references">2006</a></span>)
                              </td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Morphology, geography</td>
                              
                              <td class="left-aligned">Integrate PAA and GMYC</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Puillandre <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0069" class="bibLink tab-link" id="mec12413-bib-0069R" data-tab="pane-pcw-references">2009</a></span>)
                              </td>
                              
                              <td class="left-aligned">Morphological</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">Morphology</td>
                              
                              <td class="left-aligned">n/a</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Puillandre <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0070" class="bibLink tab-link" id="mec12413-bib-0070R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Bathymetric data, morphology</td>
                              
                              <td class="left-aligned">Klee diagrams, comparison with morphology</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Rielly <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0076" class="bibLink tab-link" id="mec12413-bib-0076R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC implied</td>
                              
                              <td class="left-aligned">Parapatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">General congruence across results</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Rosell <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0078" class="bibLink tab-link" id="mec12413-bib-0078R" data-tab="pane-pcw-references">2010</a></span>)
                              </td>
                              
                              <td class="left-aligned">Phylogenetic</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">Yes</td>
                              
                              <td class="left-aligned">Suggested convergence in morphology</td>
                              
                              <td class="left-aligned">No</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Salter <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0080" class="bibLink tab-link" id="mec12413-bib-0080R" data-tab="pane-pcw-references">2013</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Conservative consensus</td>
                              
                              <td class="left-aligned">New species described</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Setiadi <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0083" class="bibLink tab-link" id="mec12413-bib-0083R" data-tab="pane-pcw-references">2011</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Morphology</td>
                              
                              <td class="left-aligned">Favour morphology ‐ do not use BPP results</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Stech <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0085" class="bibLink tab-link" id="mec12413-bib-0085R" data-tab="pane-pcw-references">2013</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Sympatry/allopatry</td>
                              
                              <td class="left-aligned">Morphology</td>
                              
                              <td class="left-aligned">Reassign samples to species based on genetic data</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Stielow <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0087" class="bibLink tab-link" id="mec12413-bib-0087R" data-tab="pane-pcw-references">2011</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Unclear</td>
                              
                              <td class="left-aligned">Morphology</td>
                              
                              <td class="left-aligned">Authors favour results from morphology</td>
                              
                              <td class="left-aligned">New species described</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Weisrock <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0088" class="bibLink tab-link" id="mec12413-bib-0088R" data-tab="pane-pcw-references">2010</a></span>)
                              </td>
                              
                              <td class="left-aligned">GLC</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">No</td>
                              
                              <td class="left-aligned">Invoke incompletely lineage sorting</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Welton <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0089" class="bibLink tab-link" id="mec12413-bib-0089R" data-tab="pane-pcw-references">2013</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Mostly allopatric</td>
                              
                              <td class="left-aligned">Morphology and geography as basis for validation</td>
                              
                              <td class="left-aligned">Express caution in areas of incongruence</td>
                              
                              <td class="left-aligned">Recommend new species</td>
                              
                           </tr>
                           
                           <tr>
                              
                              <td class="right-bordered-cell left-aligned">Zhou <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0094" class="bibLink tab-link" id="mec12413-bib-0094R" data-tab="pane-pcw-references">2012</a></span>)
                              </td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">Allopatry</td>
                              
                              <td class="left-aligned">ENM</td>
                              
                              <td class="left-aligned">None</td>
                              
                              <td class="left-aligned">None</td>
                              
                           </tr>
                           
                        </tbody>
                        
                     </table>
                  </div>
                  
                  
                  
                  <div class="article-section__table-footnotes">
                     <ul>
                        <li id="mec12413-note-0001">
                           PSC: Phylogenetic species concept; GLC: General lineage concept; ENM: Ecological niche models.
                           
                        </li>
                     </ul>
                  </div>
                  <div class="article-section__table-source"></div>
               </div>
               
            </section>
         </div>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0012">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0012-title"> Integrating genetic and nongenetic sources of data</h2>
         
         <p>Inferences regarding species boundaries based on genetic 
data alone are likely inadequate, and species delimitation should be 
conducted with consideration of the life history, geographical 
distribution, morphology and behaviour (where applicable) of the focal 
system (Knowles &amp; Carstens <span><a href="#mec12413-bib-0047" class="bibLink tab-link" id="mec12413-bib-0047R" data-tab="pane-pcw-references">2007</a></span>; Schlick‐Steiner <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0082" class="bibLink tab-link" id="mec12413-bib-0082R" data-tab="pane-pcw-references">2010</a></span>).
 There are several approaches for this integration. Some researchers 
interpret phenotypical variation in the context of delimitations from 
genetic data (e.g. Setiadi <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0083" class="bibLink tab-link" id="mec12413-bib-0083R" data-tab="pane-pcw-references">2011</a></span>; Esselstyn <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0022" class="bibLink tab-link" id="mec12413-bib-0022R" data-tab="pane-pcw-references">2012</a></span>; Stech <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0085" class="bibLink tab-link" id="mec12413-bib-0085R" data-tab="pane-pcw-references">2013</a></span>)
 as a way to buttress the results of delimitation analysis. In some 
systems, morphological differences serve as the basis for taxonomic 
hypotheses that are validated using genetic data (e.g. Carstens &amp; 
Dewey <span><a href="#mec12413-bib-0010" class="bibLink tab-link" id="mec12413-bib-0010R" data-tab="pane-pcw-references">2010</a></span>; Welton <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0089" class="bibLink tab-link" id="mec12413-bib-0089R" data-tab="pane-pcw-references">2013</a></span>).
 When results from morphological and genetic data are incongruent, it is
 reasonable to exercise caution in species delimitation (e.g. Leliart <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0054" class="bibLink tab-link" id="mec12413-bib-0054R" data-tab="pane-pcw-references">2009</a></span>; Barrett &amp; Freudenstein <span><a href="#mec12413-bib-0004" class="bibLink tab-link" id="mec12413-bib-0004R" data-tab="pane-pcw-references">2011</a></span>) while allowing for the possibility of morphologically cryptic species (e.g. Salter <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0080" class="bibLink tab-link" id="mec12413-bib-0080R" data-tab="pane-pcw-references">2013</a></span>). While ecological niche modelling (i.e. Peterson <span><a href="#mec12413-bib-0066" class="bibLink tab-link" id="mec12413-bib-0066R" data-tab="pane-pcw-references">2001</a></span>; Hugall <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0040" class="bibLink tab-link" id="mec12413-bib-0040R" data-tab="pane-pcw-references">2002</a></span>)
 is generally underutilized in species delimitation, several researchers
 have combined these approaches. This is particularly appealing because 
the environmental modelling enables an assessment of the environmental 
differentiation between putative lineages (e.g. Florio <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0024" class="bibLink tab-link" id="mec12413-bib-0024R" data-tab="pane-pcw-references">2012</a></span>; Zhou <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0094" class="bibLink tab-link" id="mec12413-bib-0094R" data-tab="pane-pcw-references">2012</a></span>). For example, Zhou <i>et&nbsp;al</i>. (<span><a href="#mec12413-bib-0094" class="bibLink tab-link" id="mec12413-bib-0094R" data-tab="pane-pcw-references">2012</a></span>) demonstrated that the major lineages of <i>Rana</i>
 frogs in central China were environmentally differentiated, supporting 
their inference of a cryptic species in this clade. Species distribution
 models can also provide valuable information regarding historical 
changes in species range, and preliminary results from our laboratory 
indicate that hindcasted species distribution models are a better 
predictor of species boundaries in <i>Plethodon</i> salamanders than 
their current ranges demonstrate (TA Pelletier, C Crisafulli, S Wagner, 
AJ Zellmer, BC Carstens in preparation). Inferences about species limits
 are best made using an approach that integrates across many data types 
and analyses.
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0013">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0013-title"> Do researchers trust in the results of their species delimitation analyses?</h2>
         
         <p>Fewer than 30% of the studies reviewed here made taxonomic 
recommendations and only 25% describe new species. This could indicate a
 lack of confidence in the results of the species delimitation analyses 
(perhaps caused by a lack of adequate training in taxonomy), an 
inability to reconcile incongruence across methods, or it could be an 
implicit admission of the inadequacy of the data. These reluctances are 
exacerbated by selection pressures imposed by the academic system; 
species descriptions are not always viewed as equivalent to research 
papers, and it is optimal for researchers at the early stages of their 
career to publish in general interest journals such as Molecular Ecology
 rather than taxon‐focused journals. Nevertheless, formal species 
descriptions have a lasting impact that will ultimately exceed that of 
all but the most highly cited papers in general interest journals, and 
we encourage researchers to generate these descriptions.</p>
         
         <p>In our review of the literature, it also because clear that 
many researchers conduct species delimitation investigation without a 
clear statement of their goals and without defining the relevance of the
 identification of independent evolutionary lineages to their system. 
Some studies are apparently designed to document current species 
diversity, while others seek to identify independent lineages in order 
to better understand evolutionary processes. It is imperative on 
researchers to clearly state the goals of their investigations and 
incumbent on reviewers to demand that such statements are present. Such 
statements can also establish the appropriate level of conservativeness 
in delimitating species. For example, studies that seek to conduct 
taxonomic revisions and describe new species are less justified in 
favouring the results of a method such as the GMYC that tends to delimit
 a larger number of lineages than studies that seek to use the delimited
 lineages in downstream ecological analyses (e.g. Stevens <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0086" class="bibLink tab-link" id="mec12413-bib-0086R" data-tab="pane-pcw-references">2012</a></span>).
         </p>
         
      </section>
      <section class="article-section__content" id="mec12413-sec-0014">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0014-title"> Conclusions</h2>
         
         <p>Species delimitation is a vital enterprise within 
evolutionary biology; it bridges the historically independent 
disciplines of phylogenetics and population genetics and identifies the 
point when population‐level processes begin to produce phylogenetic 
patterns. Researchers have never had a better collection of 
methodological tools for delimiting species, and high‐throughput 
sequencing enables extremely large amounts of data to be collected from 
nearly any empirical system. The appropriate way to conduct a species 
delimitation investigation is to analyse these data with a wide variety 
of methods and to delimit lineages that are consistent across the 
results. If the discipline of phylogeography is to do our part to 
document and protect biodiversity, it is incumbent on researchers to 
design and conduct effective investigations into species boundaries 
(Fujita <i>et&nbsp;al</i>. <span><a href="#mec12413-bib-0027" class="bibLink tab-link" id="mec12413-bib-0027R" data-tab="pane-pcw-references">2012</a></span>), to trust in their data and results and to follow through on these results by proposing the requisite taxonomic revisions.
         </p>
         
      </section>
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="mec12413-sec-0015-title"> Acknowledgements</h2>
         
         <p>Funding for species delimitation work in the Carstens Lab 
has been provided by the NSF (DEB‐0918212; DEB‐1257784). We thank past 
and present members of the Carstens Lab, members of the <i>Nimbios</i> 
Species Delimitation Working Group and members of the Weisrock Lab for 
memorable conversations related to species delimitation. We thank A. 
Camargo for sharing data from simulations he conducted; L. Bernatchez 
for the opportunity to write this review; and K. de Queiroz and two 
anonymous reviewers for comments that improved this manuscript.
         </p>
         
         <ol></ol>
      </div>
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion">
            <h2>
               <div tabindex="0" role="button" class="accordion__control" aria-controls="idmec12413-bibl-0001" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fmec.12413&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fmec.12413%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2013.22.issue-17%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class="section__title" id="references-section-1">References</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
            </h2>
            <div class="accordion__content" style="display: none;" role="
                            region
                        " aria-labelledby="
                            references-section-1" id="mec12413-bibl-0001">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="mec12413-bib-0001"><span class="author">Avila LJ</span>, <span class="author">Morando M</span>, <span class="author">Sites JW Jr</span> (<span class="pubYear">2006</span>)  <span class="articleTitle">Congeneric phylogeography: hypothesizing species limits and evolutionary processes in Patagonian lizards of the <i>Liolaemus boulengeri</i> group (Squamata: Liolaemini)</span>. <i>Biological Journal of the Linnean Society</i>,  <span class="vol">89</span>,  <span class="pageFirst">241</span>– <span class="pageLast">275</span>.
                     <div class="extra-links" id="e_1_2_11_2_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.2006.00666.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000240557300003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=89&amp;publication_year=2006&amp;pages=241-275&amp;journal=Biological+Journal+of+the+Linnean+Society&amp;author=LJ+Avila&amp;author=M+Morando&amp;author=JW+Sites&amp;title=Congeneric+phylogeography%3A+hypothesizing+species+limits+and+evolutionary+processes+in+Patagonian+lizards+of+the+Liolaemus+boulengeri+group+%28Squamata%3A+Liolaemini%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0002"><span class="author">Avise JC</span> (<span class="pubYear">2000</span>)  <span class="bookTitle">Phylogeography: The History and Formation of Species</span>.  Harvard University Press,  <span class="publisherLocation">Cambridge, Massachusetts</span>.
                     <div class="extra-links" id="e_1_2_11_3_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.2003.01731.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2000&amp;author=JC+Avise&amp;title=Phylogeography%3A+The+History+and+Formation+of+Species" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0003"><span class="author">Baker RJ</span>, <span class="author">Bradley RD</span> (<span class="pubYear">2006</span>)  <span class="articleTitle">Speciation in mammals and the genetic species concept</span>. <i>Journal of Mammalogy</i>,  <span class="vol">87</span>,  <span class="pageFirst">643</span>– <span class="pageLast">662</span>.
                     <div class="extra-links" id="e_1_2_11_4_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1644%2F06-MAMM-F-038R2.1" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=19890476" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000240301800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=87&amp;publication_year=2006&amp;pages=643-662&amp;journal=Journal+of+Mammalogy&amp;author=RJ+Baker&amp;author=RD+Bradley&amp;title=Speciation+in+mammals+and+the+genetic+species+concept" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0004"><span class="author">Barrett CF</span>, <span class="author">Freudenstein JV</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">An integrative approach to delimiting species in a rare but widespread mycoheterotrophic orchid</span>. <i>Molecular Ecology</i>,  <span class="vol">20</span>,  <span class="pageFirst">2771</span>– <span class="pageLast">2786</span>.
                     <div class="extra-links" id="e_1_2_11_5_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05124.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21569137" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000292200700010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2011&amp;pages=2771-2786&amp;journal=Molecular+Ecology&amp;author=CF+Barrett&amp;author=JV+Freudenstein&amp;title=An+integrative+approach+to+delimiting+species+in+a+rare+but+widespread+mycoheterotrophic+orchid" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-1001"><span class="author">Baum DA</span>, <span class="author">Shaw KL</span> (<span class="pubYear">1995</span>)  <span class="chapterTitle">Genealogical perspectives on the species problem</span>. In:  <span class="bookTitle">Experimental and Molecular Approaches to Plant Biosystematics</span> (eds  <span class="editor">PC Hoch</span>,  <span class="editor">AG Stephenson</span>), pp.  <span class="pageFirst">289</span>– <span class="pageLast">303</span>. Monographs in Systematic Botany from the Missouri Botanical Garden,  <span class="publisherLocation">St Louis, Missouri</span>.
                     <div class="extra-links" id="e_1_2_11_6_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=A1995BD80F00020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1995&amp;pages=289-303&amp;author=DA+Baum&amp;author=KL+Shaw&amp;title=Experimental+and+Molecular+Approaches+to+Plant+Biosystematics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0005"><span class="author">Beerli P</span>, <span class="author">Felsenstein J</span> (<span class="pubYear">2001</span>)  <span class="articleTitle">Maximum
 likelihood estimation of a migration matrix and effective populations 
sizes in n subpopulations by using a coalescent approach</span>. <i>Proceedings National Academy Science USA</i>,  <span class="vol">98</span>,  <span class="pageFirst">4563</span>– <span class="pageLast">4568</span>.
                     <div class="extra-links" id="e_1_2_11_7_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1073%2Fpnas.081068098" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtVagtLY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=11287657" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000168059700055" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=98&amp;publication_year=2001&amp;pages=4563-4568&amp;journal=Proceedings+National+Academy+Science+USA&amp;author=P+Beerli&amp;author=J+Felsenstein&amp;title=Maximum+likelihood+estimation+of+a+migration+matrix+and+effective+populations+sizes+in+n+subpopulations+by+using+a+coalescent+approach" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0006"><span class="author">Brumfield RT</span>, <span class="author">Beerli P</span>, <span class="author">Nickerson DA</span>, <span class="author">Edwards SV</span> (<span class="pubYear">2003</span>)  <span class="articleTitle">The utility of single nucleotide polymorphisms in inferences of population history</span>. <i>Trends in Ecology and Evolution</i>,  <span class="vol">18</span>,  <span class="pageFirst">249</span>– <span class="pageLast">256</span>.
                     <div class="extra-links" id="e_1_2_11_8_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2003&amp;pages=249-256&amp;journal=Trends+in+Ecology+and+Evolution&amp;author=RT+Brumfield&amp;author=P+Beerli&amp;author=DA+Nickerson&amp;author=SV+Edwards&amp;title=The+utility+of+single+nucleotide+polymorphisms+in+inferences+of+population+history" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0007"><span class="author">Burbrink FT</span>, <span class="author">Yao H</span>, <span class="author">Ingrasci M</span>, <span class="author">Bryson RW Jr</span>, <span class="author">Guiher TJ</span>, <span class="author">Ruane S</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">Speciation at the Mogollon Rim in the Arizona Mountain Kingsnake (<i>Lampropeltis pyromelana</i>)</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">60</span>,  <span class="pageFirst">445</span>– <span class="pageLast">454</span>.
                     <div class="extra-links" id="e_1_2_11_9_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2011.05.009" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21627995" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000292580200015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=60&amp;publication_year=2011&amp;pages=445-454&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=FT+Burbrink&amp;author=H+Yao&amp;author=M+Ingrasci&amp;author=RW+Bryson&amp;author=TJ+Guiher&amp;author=S+Ruane&amp;title=Speciation+at+the+Mogollon+Rim+in+the+Arizona+Mountain+Kingsnake+%28Lampropeltis+pyromelana%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0008"><span class="author">Burnham KP</span>, <span class="author">Anderson DA</span> (<span class="pubYear">2002</span>)  <span class="bookTitle">Model Selection and Multimodel Inference: A Practical Information‐Theoretic Approach</span>,  <span class="edition">2nd edn</span>,  <span class="pageFirst">488</span> pp.  Springer‐Verlag,  <span class="publisherLocation">New York</span>.
                     <div class="extra-links" id="e_1_2_11_10_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2002&amp;pages=488&amp;author=KP+Burnham&amp;author=DA+Anderson&amp;title=Model+Selection+and+Multimodel+Inference%3A+A+Practical+Information%E2%80%90Theoretic+Approach" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0009"><span class="author">Camargo A</span>, <span class="author">Morando M</span>, <span class="author">Avila LJ</span>, <span class="author">Sites JW</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Species
 delimitation with ABC and other coalescent‐based methods: a test of 
accuracy with simulations and an empirical example with lizards of the <i>Liolaemus darwinii</i> complex (Squamata: Liolaemidae)</span>. <i>Evolution</i>,  <span class="vol">66</span>,  <span class="pageFirst">2834</span>– <span class="pageLast">2849</span>.
                     <div class="extra-links" id="e_1_2_11_11_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2012.01640.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22946806" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000308405100013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2012&amp;pages=2834-2849&amp;journal=Evolution&amp;author=A+Camargo&amp;author=M+Morando&amp;author=LJ+Avila&amp;author=JW+Sites&amp;title=Species+delimitation+with+ABC+and+other+coalescent%E2%80%90based+methods%3A+a+test+of+accuracy+with+simulations+and+an+empirical+example+with+lizards+of+the+Liolaemus+darwinii+complex+%28Squamata%3A+Liolaemidae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0010"><span class="author">Carstens BC</span>, <span class="author">Dewey TA</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Species delimitation using a combined coalescent and information‐theoretic approach: an example from North American <i>Myotis</i> bats</span>. <i>Systematic Biology</i>,  <span class="vol">59</span>,  <span class="pageFirst">400</span>– <span class="pageLast">414</span>.
                     <div class="extra-links" id="e_1_2_11_12_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fsysbio%2Fsyq024" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20547777" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000279316400003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=59&amp;publication_year=2010&amp;pages=400-414&amp;journal=Systematic+Biology&amp;author=BC+Carstens&amp;author=TA+Dewey&amp;title=Species+delimitation+using+a+combined+coalescent+and+information%E2%80%90theoretic+approach%3A+an+example+from+North+American+Myotis+bats" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0011"><span class="author">Carstens BC</span>, <span class="author">Satler JD</span> (<span class="pubYear">2013</span>)  <span class="articleTitle">The carnivorous plant described as <i>Sarracenia alata</i> contains two cryptic species</span>. <i>Biological Journal of the Linnean Society</i>, doi:<a target="_blank" class="accessionId" href="https://doi.org/10.1111/bij.12093" title="Link to external resource: 10.1111/bij.12093">10.1111/bij.12093</a>. in press.
                     <div class="extra-links" id="e_1_2_11_13_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/bij.12093" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000328956800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;journal=Biological+Journal+of+the+Linnean+Society&amp;author=BC+Carstens&amp;author=JD+Satler&amp;title=The+carnivorous+plant+described+as+Sarracenia+alata+contains+two+cryptic+species" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0012"><span class="author">Choi SC</span>, <span class="author">Hey J</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">Joint inference of population assignment and Demographic history</span>. <i>Genetics</i>,  <span class="vol">189</span>,  <span class="pageFirst">561</span>– <span class="pageLast">577</span>.
                     <div class="extra-links" id="e_1_2_11_14_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1534%2Fgenetics.111.129205" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21775468" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000296158500013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=189&amp;publication_year=2011&amp;pages=561-577&amp;journal=Genetics&amp;author=SC+Choi&amp;author=J+Hey&amp;title=Joint+inference+of+population+assignment+and+Demographic+history" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0013"><span class="author">Csilléry K</span>, <span class="author">Blum MGB</span>, <span class="author">Gaggiotti OE</span>, <span class="author">Francois O</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">Approximate Bayesian Computation (ABC) in practice</span>. <i>Trends in Ecology and Evolution</i>,  <span class="vol">25</span>,  <span class="pageFirst">410</span>– <span class="pageLast">418</span>.
                     <div class="extra-links" id="e_1_2_11_15_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.tree.2010.04.001" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000279531700007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=2011&amp;pages=410-418&amp;journal=Trends+in+Ecology+and+Evolution&amp;author=K+Csill%C3%A9ry&amp;author=MGB+Blum&amp;author=OE+Gaggiotti&amp;author=O+Francois&amp;title=Approximate+Bayesian+Computation+%28ABC%29+in+practice" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0014"><span class="author">Darwin C</span> (<span class="pubYear">1859</span>)  <span class="bookTitle">On the Origin of Species by Means of Natural Selection, or the Preservation of Favoured Races in the Struggle for Life</span>.  John Murray Press,  <span class="publisherLocation">London, UK</span>.
                     <div class="extra-links" id="e_1_2_11_16_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1859&amp;author=C+Darwin&amp;title=On+the+Origin+of+Species+by+Means+of+Natural+Selection%2C+or+the+Preservation+of+Favoured+Races+in+the+Struggle+for+Life" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0015"><span class="author">de Queiroz K</span> (<span class="pubYear">1998</span>)  <span class="chapterTitle">The general lineage concept of species, species criteria, and the process of speciation</span>. In:  <span class="bookTitle">Endless Forms: Species and Speciation</span> (eds.  <span class="editor">SJ Howard</span>,  <span class="editor">SH Berlocher</span>), pp.  <span class="pageFirst">57</span>– <span class="pageLast">78</span>.  Oxford University Press,  <span class="publisherLocation">New York</span>.
                     <div class="extra-links" id="e_1_2_11_17_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1998&amp;pages=57-78&amp;author=K+de+Queiroz&amp;title=Endless+Forms%3A+Species+and+Speciation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0016"><span class="author">Degnan JH</span>, <span class="author">Rosenberg NA</span> (<span class="pubYear">2006</span>)  <span class="articleTitle">Discordance of species trees with their most likely gene trees</span>. <i>PLoS Genetics</i>  <span class="vol">2 e68</span>,  <span class="pageFirst">762</span>– <span class="pageLast">768</span>.
                     <div class="extra-links" id="e_1_2_11_18_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000239494600013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2+e68&amp;publication_year=2006&amp;pages=762-768&amp;journal=PLoS+Genetics&amp;author=JH+Degnan&amp;author=NA+Rosenberg&amp;title=Discordance+of+species+trees+with+their+most+likely+gene+trees" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0017"><span class="author">Duminil J</span>, <span class="author">Kenfack D</span>, <span class="author">Viscosi V</span>, <span class="author">Grumiau L</span>, <span class="author">Hardy OJ</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Testing species delimitation in sympatric species complexes: the case of an African tropical tree, <i>Carapa</i> spp. (Meliaceae)</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">62</span>,  <span class="pageFirst">275</span>– <span class="pageLast">285</span>.
                     <div class="extra-links" id="e_1_2_11_19_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2011.09.020" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22019936" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000301020500024" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=62&amp;publication_year=2012&amp;pages=275-285&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=J+Duminil&amp;author=D+Kenfack&amp;author=V+Viscosi&amp;author=L+Grumiau&amp;author=OJ+Hardy&amp;title=Testing+species+delimitation+in+sympatric+species+complexes%3A+the+case+of+an+African+tropical+tree%2C+Carapa+spp.+%28Meliaceae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0018"><span class="author">Dupuis JR</span>, <span class="author">Roe AD</span>, <span class="author">Sperling FA</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Multi‐locus species delimitation in closely related animals and fungi: one marker is not enough</span>. <i>Molecular Ecology</i>  <span class="vol">21</span>,  <span class="pageFirst">4422</span>– <span class="pageLast">4436</span>.
                     <div class="extra-links" id="e_1_2_11_20_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05642.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22891635" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000308634300004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=4422-4436&amp;journal=Molecular+Ecology&amp;author=JR+Dupuis&amp;author=AD+Roe&amp;author=FA+Sperling&amp;title=Multi%E2%80%90locus+species+delimitation+in+closely+related+animals+and+fungi%3A+one+marker+is+not+enough" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0019"><span class="author">Eckert AJ</span>, <span class="author">Carstens BC</span> (<span class="pubYear">2008</span>)  <span class="articleTitle">Does
 gene flow destroy phylogenetic signal? The performance of three methods
 for estimating species phylogenies in the presence of gene flow</span>. <i>Molecular Phylogenetics &amp; Evolution</i>,  <span class="vol">49</span>,  <span class="pageFirst">832</span>– <span class="pageLast">842</span>.
                     <div class="extra-links" id="e_1_2_11_21_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2008.09.008" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVCntLzO" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18845264" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000261647900013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=49&amp;publication_year=2008&amp;pages=832-842&amp;journal=Molecular+Phylogenetics+%26+Evolution&amp;author=AJ+Eckert&amp;author=BC+Carstens&amp;title=Does+gene+flow+destroy+phylogenetic+signal%3F+The+performance+of+three+methods+for+estimating+species+phylogenies+in+the+presence+of+gene+flow" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0020"><span class="author">Edwards SV</span> (<span class="pubYear">2009</span>)  <span class="articleTitle">Is a new and general theory of molecular systematics emerging?</span> <i>Evolution</i>,  <span class="vol">63</span>,  <span class="pageFirst">1</span>– <span class="pageLast">19</span>.
                     <div class="extra-links" id="e_1_2_11_22_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2008.00549.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXisVKgtbs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=19146594" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000262469600001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=63&amp;publication_year=2009&amp;pages=1-19&amp;journal=Evolution&amp;author=SV+Edwards&amp;title=Is+a+new+and+general+theory+of+molecular+systematics+emerging%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0021"><span class="author">Ence DD</span>, <span class="author">Carstens BC</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">SpedeSTEM: a rapid and accurate method for species delimitation</span>. <i>Molecular Ecology Resources</i>,  <span class="vol">11</span>,  <span class="pageFirst">473</span>– <span class="pageLast">480</span>.
                     <div class="extra-links" id="e_1_2_11_23_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1755-0998.2010.02947.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21481205" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000289465500006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=11&amp;publication_year=2011&amp;pages=473-480&amp;journal=Molecular+Ecology+Resources&amp;author=DD+Ence&amp;author=BC+Carstens&amp;title=SpedeSTEM%3A+a+rapid+and+accurate+method+for+species+delimitation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0022"><span class="author">Esselstyn JA</span>, <span class="author">Evans BJ</span>, <span class="author">Sedlock JL</span>, <span class="author">Anwarali Khan FA</span>, <span class="author">Heaney LR</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Single‐locus
 species delimitation: a test of the mixed Yule‐coalescent model, with 
an empirical application to Philippine round‐leaf bats</span>. <i>Proceedings of the Royal Society of London B</i>,  <span class="vol">279</span>,  <span class="pageFirst">3678</span>– <span class="pageLast">3686</span>.
                     <div class="extra-links" id="e_1_2_11_24_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1098%2Frspb.2012.0705" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22764163" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000307780400007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=279&amp;publication_year=2012&amp;pages=3678-3686&amp;journal=Proceedings+of+the+Royal+Society+of+London+B&amp;author=JA+Esselstyn&amp;author=BJ+Evans&amp;author=JL+Sedlock&amp;author=FA+Anwarali+Khan&amp;author=LR+Heaney&amp;title=Single%E2%80%90locus+species+delimitation%3A+a+test+of+the+mixed+Yule%E2%80%90coalescent+model%2C+with+an+empirical+application+to+Philippine+round%E2%80%90leaf+bats" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0023"><span class="author">Felsenstein J</span> (<span class="pubYear">2004</span>)  <span class="bookTitle">Inferring Phylogenies</span>.  Sinauer Associates,  <span class="publisherLocation">Sunderland, Massachusetts</span>.
                     <div class="extra-links" id="e_1_2_11_25_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;author=J+Felsenstein&amp;title=Inferring+Phylogenies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0024"><span class="author">Florio AM</span>, <span class="author">Ingram CM</span>, <span class="author">Rakotondravony HA</span>, <span class="author">Louis EE Jr</span>, <span class="author">Raxworthy CJ</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Detecting cryptic speciation in the widespread and morphologically conservative carpet chameleon (<i>Furcifer lateralis</i>) of Madagascar</span>. <i>Journal of Evolutionary Biology</i>,  <span class="vol">25</span>,  <span class="pageFirst">1399</span>– <span class="pageLast">1414</span>.
                     <div class="extra-links" id="e_1_2_11_26_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2012.02528.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ASTN%3A280%3ADC%252BC38jgs1OrsQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22686488" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000305130800016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=2012&amp;pages=1399-1414&amp;journal=Journal+of+Evolutionary+Biology&amp;author=AM+Florio&amp;author=CM+Ingram&amp;author=HA+Rakotondravony&amp;author=EE+Louis&amp;author=CJ+Raxworthy&amp;title=Detecting+cryptic+speciation+in+the+widespread+and+morphologically+conservative+carpet+chameleon+%28Furcifer+lateralis%29+of+Madagascar" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0025"><span class="author">Flot J‐F</span>, <span class="author">Couloux A</span>, <span class="author">Tillier S</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Haplowebs
 as a graphical tool for delimiting species: a revival of Doyle's “field
 for recombination” approach and its application to the coral genus 
Pocillopora in Clipperton</span>. <i>BMC Evolutionary Biology</i>,  <span class="vol">10</span>,  <span class="pageFirst">372</span>.
                     <div class="extra-links" id="e_1_2_11_27_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1186%2F1471-2148-10-372" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21118572" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000287575700004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2010&amp;pages=372&amp;journal=BMC+Evolutionary+Biology&amp;author=J%E2%80%90F+Flot&amp;author=A+Couloux&amp;author=S+Tillier&amp;title=Haplowebs+as+a+graphical+tool+for+delimiting+species%3A+a+revival+of+Doyle%27s+%E2%80%9Cfield+for+recombination%E2%80%9D+approach+and+its+application+to+the+coral+genus+Pocillopora+in+Clipperton" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0026"><span class="author">Fraley C</span>, <span class="author">Raftery AE</span>.( <span class="pubYear">2006</span>).  <span class="otherTitle">MCLUST Version 3 for R: Normal mixture modeling and model‐based clustering</span>. Technical Report no. 504, Department of Statistics, University of Washington.
                     <div class="extra-links" id="e_1_2_11_28_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=Fraley+C%2C+Raftery+AE.%28+2006%29.+MCLUST+Version+3+for+R%3A+Normal+mixture+modeling+and+model%E2%80%90based+clustering.+Technical+Report+no.+504%2C+Department+of+Statistics%2C+University+of+Washington." target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0027"><span class="author">Fujita MK</span>, <span class="author">Leaché AD</span>, <span class="author">Burbrink FT</span>, <span class="author">McGuire JA</span>, <span class="author">Moritz C</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Coalescent‐based species delimitation in an integrative taxonomy</span>. <i>Trends in Ecology and Evolution</i>,  <span class="vol">27</span>,  <span class="pageFirst">480</span>– <span class="pageLast">488</span>.
                     <div class="extra-links" id="e_1_2_11_29_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.tree.2012.04.012" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22633974" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000308268300004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2012&amp;pages=480-488&amp;journal=Trends+in+Ecology+and+Evolution&amp;author=MK+Fujita&amp;author=AD+Leach%C3%A9&amp;author=FT+Burbrink&amp;author=JA+McGuire&amp;author=C+Moritz&amp;title=Coalescent%E2%80%90based+species+delimitation+in+an+integrative+taxonomy" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0028"><span class="author">Guillot G</span>, <span class="author">Mortier F</span>, <span class="author">Estoup A</span> (<span class="pubYear">2005</span>)  <span class="articleTitle">Geneland: a program for landscape genetics</span>. <i>Molecular Ecology Notes</i>,  <span class="vol">5</span>,  <span class="pageFirst">712</span>– <span class="pageLast">715</span>.
                     <div class="extra-links" id="e_1_2_11_30_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2005.01031.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVOhurvN" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2005&amp;pages=712-715&amp;journal=Molecular+Ecology+Notes&amp;author=G+Guillot&amp;author=F+Mortier&amp;author=A+Estoup&amp;title=Geneland%3A+a+program+for+landscape+genetics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0029"><span class="author">Guillot G</span>, <span class="author">Renaud S</span>, <span class="author">Ledevin R</span>, <span class="author">Michaux J</span>, <span class="author">Claude J</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">A unifying model for the analysis of phenotypic, genetic, and geographic data</span>. <i>Systematic Biology</i>,  <span class="vol">61</span>,  <span class="pageFirst">897</span>– <span class="pageLast">911</span>.
                     <div class="extra-links" id="e_1_2_11_31_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fsysbio%2Fsys038" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22398122" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000310376400001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=61&amp;publication_year=2012&amp;pages=897-911&amp;journal=Systematic+Biology&amp;author=G+Guillot&amp;author=S+Renaud&amp;author=R+Ledevin&amp;author=J+Michaux&amp;author=J+Claude&amp;title=A+unifying+model+for+the+analysis+of+phenotypic%2C+genetic%2C+and+geographic+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0030"><span class="author">Gutenkunst RN</span>, <span class="author">Hernandez RD</span>, <span class="author">Williamson SH</span>, <span class="author">Bustamante CD</span> (<span class="pubYear">2009</span>)  <span class="articleTitle">Inferring the joint demographic history of multiple populations from multidimensional SNP frequency data</span>. <i>PLoS Genetics</i>,  <span class="vol">5</span>,  <span class="pageFirst">1</span>– <span class="pageLast">11</span>.
                     <div class="extra-links" id="e_1_2_11_32_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1371%2Fjournal.pgen.1000695" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlCiu7jL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000272032100033" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2009&amp;pages=1-11&amp;journal=PLoS+Genetics&amp;author=RN+Gutenkunst&amp;author=RD+Hernandez&amp;author=SH+Williamson&amp;author=CD+Bustamante&amp;title=Inferring+the+joint+demographic+history+of+multiple+populations+from+multidimensional+SNP+frequency+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0031"><span class="author">Hamilton CA</span>, <span class="author">Formanowicz DR</span>, <span class="author">Bond JE</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">Species delimitation and phylogeography of <i>Aphonopelma hentzi</i> (Araneae, Mygalomorphae, Theraphosidae): cryptic diversity in North American Tarantulas</span>. <i>PLoS ONE</i>,  <span class="vol">6</span>,  <span class="pageFirst">e26207</span>.
                     <div class="extra-links" id="e_1_2_11_33_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1371%2Fjournal.pone.0026207" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVSgsbjI" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22022570" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000295976000087" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2011&amp;pages=e26207&amp;journal=PLoS+ONE&amp;author=CA+Hamilton&amp;author=DR+Formanowicz&amp;author=JE+Bond&amp;title=Species+delimitation+and+phylogeography+of+Aphonopelma+hentzi+%28Araneae%2C+Mygalomorphae%2C+Theraphosidae%29%3A+cryptic+diversity+in+North+American+Tarantulas" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0032"><span class="author">Hausdorf B</span>, <span class="author">Hennig C</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Species delimitation using dominant and codominant multilocus markers</span>. <i>Systematic Biology</i>,  <span class="vol">59</span>,  <span class="pageFirst">491</span>– <span class="pageLast">503</span>.
                     <div class="extra-links" id="e_1_2_11_34_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fsysbio%2Fsyq039" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Oqtb7F" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20693311" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000282817100001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=59&amp;publication_year=2010&amp;pages=491-503&amp;journal=Systematic+Biology&amp;author=B+Hausdorf&amp;author=C+Hennig&amp;title=Species+delimitation+using+dominant+and+codominant+multilocus+markers" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0033"><span class="author">Heled J</span>, <span class="author">Drummond AJ</span> (<span class="pubYear">2008</span>)  <span class="articleTitle">Bayesian inference of population size history from a multiple loci</span>. <i>BMC Evolutionary Biology</i>,  <span class="vol">8</span>,  <span class="pageFirst">289</span>.
                     <div class="extra-links" id="e_1_2_11_35_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1186%2F1471-2148-8-289" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2jtbrL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18947398" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000264070600001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2008&amp;pages=289&amp;journal=BMC+Evolutionary+Biology&amp;author=J+Heled&amp;author=AJ+Drummond&amp;title=Bayesian+inference+of+population+size+history+from+a+multiple+loci" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0034"><span class="author">Heled J</span>, <span class="author">Drummond AJ</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Bayesian inference of species trees from multilocus data</span>. <i>Molecular Biology and Evolution</i>,  <span class="vol">27</span>,  <span class="pageFirst">570</span>– <span class="pageLast">580</span>.
                     <div class="extra-links" id="e_1_2_11_36_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fmolbev%2Fmsp274" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitlart7s%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=19906793" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000274786900008" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=2010&amp;pages=570-580&amp;journal=Molecular+Biology+and+Evolution&amp;author=J+Heled&amp;author=AJ+Drummond&amp;title=Bayesian+inference+of+species+trees+from+multilocus+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0035"><span class="author">Hey J</span>, <span class="author">Nielsen R</span> (<span class="pubYear">2004</span>)  <span class="articleTitle">Multilocus methods for estimating population sizes, migration rates and divergence time, with applications to the divergence of <i>Drosophila pseudoobscura</i> and <i>D. persimilis</i></span>. <i>Genetics</i>,  <span class="vol">167</span>,  <span class="pageFirst">747</span>– <span class="pageLast">760</span>.
                     <div class="extra-links" id="e_1_2_11_37_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1534%2Fgenetics.103.024182" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD2cXms1KntL8%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=15238526" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000222650500017" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=167&amp;publication_year=2004&amp;pages=747-760&amp;journal=Genetics&amp;author=J+Hey&amp;author=R+Nielsen&amp;title=Multilocus+methods+for+estimating+population+sizes%2C+migration+rates+and+divergence+time%2C+with+applications+to+the+divergence+of+Drosophila+pseudoobscura+and+D.+persimilis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0036"><span class="author">Hey J</span>, <span class="author">Nielsen R</span> (<span class="pubYear">2007</span>)  <span class="articleTitle">Integration within the Felsenstein equation for improved Markov chain Monte Carlo methods in population genetics</span>. <i>Proceedings of the National Academy of Sciences USA</i>,  <span class="vol">104</span>,  <span class="pageFirst">2785</span>– <span class="pageLast">2790</span>.
                     <div class="extra-links" id="e_1_2_11_38_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1073%2Fpnas.0611164104" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXisVSgtbc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=17301231" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000244511200041" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=104&amp;publication_year=2007&amp;pages=2785-2790&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+USA&amp;author=J+Hey&amp;author=R+Nielsen&amp;title=Integration+within+the+Felsenstein+equation+for+improved+Markov+chain+Monte+Carlo+methods+in+population+genetics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0037"><span class="author">Hudson RR</span> (<span class="pubYear">1991</span>)  <span class="chapterTitle">Gene genealogies and the coalescent process</span>. In:  <span class="bookTitle">Oxford Surveys in Evolutionary Biology</span> (eds  <span class="editor">D Futuyma</span> &amp;  <span class="editor">J Antonovics</span>), pp.  <span class="pageFirst">1</span>– <span class="pageLast">44</span>.  Oxford University Press,  <span class="publisherLocation">New York</span>.
                     <div class="extra-links" id="e_1_2_11_39_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1991&amp;pages=1-44&amp;author=RR+Hudson&amp;title=Oxford+Surveys+in+Evolutionary+Biology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0038"><span class="author">Huelsenbeck JP</span>, <span class="author">Andolfatto P</span> (<span class="pubYear">2007</span>)  <span class="articleTitle">Inference of population structure under a Dirichlet process model</span>. <i>Genetics</i>,  <span class="vol">175</span>,  <span class="pageFirst">1787</span>– <span class="pageLast">1802</span>.
                     <div class="extra-links" id="e_1_2_11_40_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1534%2Fgenetics.106.061317" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmsFymu7c%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=17237522" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000246448800022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=175&amp;publication_year=2007&amp;pages=1787-1802&amp;journal=Genetics&amp;author=JP+Huelsenbeck&amp;author=P+Andolfatto&amp;title=Inference+of+population+structure+under+a+Dirichlet+process+model" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0039"><span class="author">Huelsenbeck JP</span>, <span class="author">Andolfatto P</span>, <span class="author">Huelsenbeck ET</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">Structurama: Bayesian inference of population structure</span>. <i>Evolutionary Bioinformatics</i>,  <span class="vol">7</span>,  <span class="pageFirst">55</span>– <span class="pageLast">59</span>.
                     <div class="extra-links" id="e_1_2_11_41_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.4137%2FEBO.S6761" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21698091" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000291563000002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2011&amp;pages=55-59&amp;journal=Evolutionary+Bioinformatics&amp;author=JP+Huelsenbeck&amp;author=P+Andolfatto&amp;author=ET+Huelsenbeck&amp;title=Structurama%3A+Bayesian+inference+of+population+structure" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0040"><span class="author">Hugall A</span>, <span class="author">Moritz C</span>, <span class="author">Moussalii A</span>, <span class="author">Stanisic J</span> (<span class="pubYear">2002</span>)  <span class="articleTitle">Reconciling
 paleodistribution models and comparative phylogeography in the Wet 
Tropics rainforest land snail Arosophia bellendenkerensis (Brazier 1875)</span>. <i>Proceedings of the National Academy of Sciences USA</i>,  <span class="vol">99</span>,  <span class="pageFirst">6112</span>– <span class="pageLast">6117</span>.
                     <div class="extra-links" id="e_1_2_11_42_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=99&amp;publication_year=2002&amp;pages=6112-6117&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+USA&amp;author=A+Hugall&amp;author=C+Moritz&amp;author=A+Moussalii&amp;author=J+Stanisic&amp;title=Reconciling+paleodistribution+models+and+comparative+phylogeography+in+the+Wet+Tropics+rainforest+land+snail+Arosophia+bellendenkerensis+%28Brazier+1875%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0041"><span class="author">Hull DL</span> (<span class="pubYear">1998</span>)  <span class="bookTitle">Science as a Process</span>.  University of Chicago Press,  <span class="publisherLocation">Chicago, Illinois</span>.
                     <div class="extra-links" id="e_1_2_11_43_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1998&amp;author=DL+Hull&amp;title=Science+as+a+Process" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0042"><span class="author">Jackson ND</span>, <span class="author">Austin CC</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">The combined effects of rivers and refugia generate extreme cryptic fragmentation within the common ground skink (<i>Scincella lateralis</i>)</span>. <i>Evolution</i>,  <span class="vol">64</span>,  <span class="pageFirst">409</span>– <span class="pageLast">428</span>.
                     <div class="extra-links" id="e_1_2_11_44_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2009.00840.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitlKqs78%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=19780813" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000273821700010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=64&amp;publication_year=2010&amp;pages=409-428&amp;journal=Evolution&amp;author=ND+Jackson&amp;author=CC+Austin&amp;title=The+combined+effects+of+rivers+and+refugia+generate+extreme+cryptic+fragmentation+within+the+common+ground+skink+%28Scincella+lateralis%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0043"><span class="author">Kalinowski ST</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">The
 computer program STRUCTURE does not reliably identify the main genetic 
clusters within species: simulations and implications for human 
population genetic structure</span>. <i>Heredity</i>,  <span class="vol">106</span>,  <span class="pageFirst">625</span>– <span class="pageLast">632</span>.
                     <div class="extra-links" id="e_1_2_11_45_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1038%2Fhdy.2010.95" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20683484" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000288703900015" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=106&amp;publication_year=2010&amp;pages=625-632&amp;journal=Heredity&amp;author=ST+Kalinowski&amp;title=The+computer+program+STRUCTURE+does+not+reliably+identify+the+main+genetic+clusters+within+species%3A+simulations+and+implications+for+human+population+genetic+structure" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0044"><span class="author">Kelly CMR</span>, <span class="author">Barker NP</span>, <span class="author">Villet MH</span>, <span class="author">Broadley DG</span>, <span class="author">Branch WR</span> (<span class="pubYear">2008</span>)  <span class="articleTitle">The snake family Psammophiidae (Reptilia: Serpentes): phylogenetics and species delimitation in the African sand snakes (<i>Psammophis Boie</i>, 1925) and allied genera</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">47</span>,  <span class="pageFirst">1045</span>– <span class="pageLast">1060</span>.
                     <div class="extra-links" id="e_1_2_11_46_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2008.03.025" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1cXmtlyhtLc%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18434208" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000257128900014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=47&amp;publication_year=2008&amp;pages=1045-1060&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=CMR+Kelly&amp;author=NP+Barker&amp;author=MH+Villet&amp;author=DG+Broadley&amp;author=WR+Branch&amp;title=The+snake+family+Psammophiidae+%28Reptilia%3A+Serpentes%29%3A+phylogenetics+and+species+delimitation+in+the+African+sand+snakes+%28Psammophis+Boie%2C+1925%29+and+allied+genera" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0045"><span class="author">Kim S‐T</span>, <span class="author">Sultan SE</span>, <span class="author">Donoghue MJ</span> (<span class="pubYear">2008</span>)  <span class="articleTitle">Allopolyploid speciation in Persicaria (Polygonaceae): insights from a low‐copy nuclear region</span>. <i>Proceedings of the National Academy of Sciences USA</i>,  <span class="vol">105</span>,  <span class="pageFirst">12370</span>– <span class="pageLast">12375</span>.
                     <div class="extra-links" id="e_1_2_11_47_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1073%2Fpnas.0805141105" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVOhtrjP" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18711129" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000258905700050" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=105&amp;publication_year=2008&amp;pages=12370-12375&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+USA&amp;author=S%E2%80%90T+Kim&amp;author=SE+Sultan&amp;author=MJ+Donoghue&amp;title=Allopolyploid+speciation+in+Persicaria+%28Polygonaceae%29%3A+insights+from+a+low%E2%80%90copy+nuclear+region" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0046"><span class="author">Kingman JFC</span> (<span class="pubYear">1982</span>)  <span class="articleTitle">One the genealogy of large populations</span>. <i>Journal of Applied Probability</i>,  <span class="vol">19</span>,  <span class="pageFirst">27</span>– <span class="pageLast">43</span>.
                     <div class="extra-links" id="e_1_2_11_48_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.2307%2F3213548" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=1982&amp;pages=27-43&amp;journal=Journal+of+Applied+Probability&amp;author=JFC+Kingman&amp;title=One+the+genealogy+of+large+populations" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0047"><span class="author">Knowles LL</span>, <span class="author">Carstens BC</span> (<span class="pubYear">2007</span>)  <span class="articleTitle">Delimiting species without monophyletic gene trees</span>. <i>Systematic Biology</i>,  <span class="vol">56</span>,  <span class="pageFirst">887</span>– <span class="pageLast">895</span>.
                     <div class="extra-links" id="e_1_2_11_49_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1080%2F10635150701701091" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18027282" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000252360100003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2007&amp;pages=887-895&amp;journal=Systematic+Biology&amp;author=LL+Knowles&amp;author=BC+Carstens&amp;title=Delimiting+species+without+monophyletic+gene+trees" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0049"><span class="author">Kubatko LS</span>, <span class="author">Carstens BC</span>, <span class="author">Knowles LL</span> (<span class="pubYear">2009</span>)  <span class="articleTitle">STEM: species tree estimation using maximum likelihood for gene trees under coalescence</span>. <i>Bioinformatics</i>,  <span class="vol">25</span>,  <span class="pageFirst">971</span>– <span class="pageLast">973</span>.
                     <div class="extra-links" id="e_1_2_11_50_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fbioinformatics%2Fbtp079" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjvVaitb4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=19211573" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000264651800025" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=2009&amp;pages=971-973&amp;journal=Bioinformatics&amp;author=LS+Kubatko&amp;author=BC+Carstens&amp;author=LL+Knowles&amp;title=STEM%3A+species+tree+estimation+using+maximum+likelihood+for+gene+trees+under+coalescence" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0050"><span class="author">Kuhner MK</span>, <span class="author">Yamato J</span>, <span class="author">Felsenstein J</span> (<span class="pubYear">1998</span>)  <span class="articleTitle">Maximum likelihood estimation of population growth rates based on the coalescent</span>. <i>Genetics</i>,  <span class="vol">149</span>,  <span class="pageFirst">429</span>– <span class="pageLast">434</span>.
                     <div class="extra-links" id="e_1_2_11_51_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ASTN%3A280%3ADyaK1c3ltVOjsw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=9584114" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000073672500034" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=149&amp;publication_year=1998&amp;pages=429-434&amp;journal=Genetics&amp;author=MK+Kuhner&amp;author=J+Yamato&amp;author=J+Felsenstein&amp;title=Maximum+likelihood+estimation+of+population+growth+rates+based+on+the+coalescent" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0052"><span class="author">Leaché AD</span>, <span class="author">Fujita MK</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Bayesian species delimitation in West African forest geckos (<i>Hemidactylus fasciatus</i>)</span>. <i>Proceedings of the Royal Society B</i>,  <span class="vol">277</span>,  <span class="pageFirst">3071</span>– <span class="pageLast">3077</span>.
                     <div class="extra-links" id="e_1_2_11_52_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1098%2Frspb.2010.0662" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20519219" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000282645700003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=277&amp;publication_year=2010&amp;pages=3071-3077&amp;journal=Proceedings+of+the+Royal+Society+B&amp;author=AD+Leach%C3%A9&amp;author=MK+Fujita&amp;title=Bayesian+species+delimitation+in+West+African+forest+geckos+%28Hemidactylus+fasciatus%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0053"><span class="author">Leavitt SD</span>, <span class="author">Johnson LA</span>, <span class="author">Goward T</span>, <span class="author">Clair LL</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Species
 delimitation in taxonomically difficult lichen‐forming fungi: an 
example from morphologically and chemically diverse Xanthoparmelia 
(Parmeliaceae) in North America</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">60</span>,  <span class="pageFirst">317</span>– <span class="pageLast">332</span>.
                     <div class="extra-links" id="e_1_2_11_53_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2011.05.012" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000292580200005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=60&amp;publication_year=2012&amp;pages=317-332&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=SD+Leavitt&amp;author=LA+Johnson&amp;author=T+Goward&amp;author=LL+Clair&amp;title=Species+delimitation+in+taxonomically+difficult+lichen%E2%80%90forming+fungi%3A+an+example+from+morphologically+and+chemically+diverse+Xanthoparmelia+%28Parmeliaceae%29+in+North+America" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0054"><span class="author">Leliart F</span>, <span class="author">Verbruggen H</span>, <span class="author">Wysor B</span>, <span class="author">De Clerk O</span> (<span class="pubYear">2009</span>)  <span class="articleTitle">DNA taxonomy in morphologically plastic taxa: Algorithmic species delimitation in the <i>Boodlea</i> complex (Chlorophyta: Cladophorales)</span>. <i>Molecular Phylogentics and Evolution</i>,  <span class="vol">53</span>,  <span class="pageFirst">122</span>– <span class="pageLast">133</span>.
                     <div class="extra-links" id="e_1_2_11_54_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2009.06.004" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD1MXptlyksL0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000269067900012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=53&amp;publication_year=2009&amp;pages=122-133&amp;journal=Molecular+Phylogentics+and+Evolution&amp;author=F+Leliart&amp;author=H+Verbruggen&amp;author=B+Wysor&amp;author=O+Clerk&amp;title=DNA+taxonomy+in+morphologically+plastic+taxa%3A+Algorithmic+species+delimitation+in+the+Boodlea+complex+%28Chlorophyta%3A+Cladophorales%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0055"><span class="author">Liu L</span>, <span class="author">Pearl DK</span>, <span class="author">Brumfield RT</span>, <span class="author">Edwards SV</span> (<span class="pubYear">2007</span>)  <span class="articleTitle">Estimating species trees using multiple‐allele DNA sequence data</span>. <i>Evolution</i>,  <span class="vol">62</span>,  <span class="pageFirst">2080</span>– <span class="pageLast">2091</span>.
                     <div class="extra-links" id="e_1_2_11_55_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2008.00414.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18462214" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000258261700020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=62&amp;publication_year=2007&amp;pages=2080-2091&amp;journal=Evolution&amp;author=L+Liu&amp;author=DK+Pearl&amp;author=RT+Brumfield&amp;author=SV+Edwards&amp;title=Estimating+species+trees+using+multiple%E2%80%90allele+DNA+sequence+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0056"><span class="author">Maddison WP</span> (<span class="pubYear">1997</span>)  <span class="articleTitle">Gene trees in species trees</span>. <i>Systematic Biology</i>,  <span class="vol">46</span>,  <span class="pageFirst">523</span>– <span class="pageLast">536</span>.
                     <div class="extra-links" id="e_1_2_11_56_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fsysbio%2F46.3.523" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=A1997YA42300007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=46&amp;publication_year=1997&amp;pages=523-536&amp;journal=Systematic+Biology&amp;author=WP+Maddison&amp;title=Gene+trees+in+species+trees" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-1002"><span class="author">Mayr E</span> (<span class="pubYear">1942</span>)  <span class="bookTitle">Systematics and the Origin of Species, from the Viewpoint of a Zoologist (No. 13)</span>.  Harvard University Press, Cambridge,  <span class="publisherLocation">Massachusetts</span>.
                     <div class="extra-links" id="e_1_2_11_57_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1942&amp;author=E+Mayr&amp;title=Systematics+and+the+Origin+of+Species%2C+from+the+Viewpoint+of+a+Zoologist+%28No.+13%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0057"><span class="author">Maddison WP</span>, <span class="author">Knowles LL</span> (<span class="pubYear">2006</span>)  <span class="articleTitle">Inferring phylogeny despite incomplete lineage sorting</span>. <i>Systematic Biology</i>,  <span class="vol">55</span>,  <span class="pageFirst">21</span>– <span class="pageLast">30</span>.
                     <div class="extra-links" id="e_1_2_11_58_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1080%2F10635150500354928" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=16507521" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000235622700002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=55&amp;publication_year=2006&amp;pages=21-30&amp;journal=Systematic+Biology&amp;author=WP+Maddison&amp;author=LL+Knowles&amp;title=Inferring+phylogeny+despite+incomplete+lineage+sorting" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0058"><span class="author">McCormack JE</span>, <span class="author">Hird SM</span>, <span class="author">Zellmer AJ</span>, <span class="author">Carstens BC</span>, <span class="author">Brumfield RT</span> (<span class="pubYear">2013</span>)  <span class="articleTitle">Applications of next‐generation sequencing to phylogeography and phylogenetics</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">66</span>,  <span class="pageFirst">526</span>– <span class="pageLast">538</span>.
                     <div class="extra-links" id="e_1_2_11_59_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2011.12.007" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFajtbs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22197804" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000315553300009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2013&amp;pages=526-538&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=JE+McCormack&amp;author=SM+Hird&amp;author=AJ+Zellmer&amp;author=BC+Carstens&amp;author=RT+Brumfield&amp;title=Applications+of+next%E2%80%90generation+sequencing+to+phylogeography+and+phylogenetics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0059"><span class="author">Moritz C</span> (<span class="pubYear">1994</span>)  <span class="articleTitle">Defining “Evolutionarily Significant Units” for conservation</span>. <i>Trends in Ecology and Evolution</i>,  <span class="vol">9</span>,  <span class="pageFirst">373</span>– <span class="pageLast">375</span>.
                     <div class="extra-links" id="e_1_2_11_60_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2F0169-5347%2894%2990057-4" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ASTN%3A280%3ADC%252BC3M7itFWhsA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21236896" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=A1994PE12200011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=1994&amp;pages=373-375&amp;journal=Trends+in+Ecology+and+Evolution&amp;author=C+Moritz&amp;title=Defining+%E2%80%9CEvolutionarily+Significant+Units%E2%80%9D+for+conservation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0060"><span class="author">Nielsen R</span>, <span class="author">Wakeley J</span> (<span class="pubYear">2001</span>)  <span class="articleTitle">Distinguishing migration from isolation: a Markov Chain Monte Carlo approach</span>. <i>Genetics</i>,  <span class="vol">158</span>,  <span class="pageFirst">885</span>– <span class="pageLast">896</span>.
                     <div class="extra-links" id="e_1_2_11_61_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltFSisbg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=11404349" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000169275600035" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=158&amp;publication_year=2001&amp;pages=885-896&amp;journal=Genetics&amp;author=R+Nielsen&amp;author=J+Wakeley&amp;title=Distinguishing+migration+from+isolation%3A+a+Markov+Chain+Monte+Carlo+approach" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0061"><span class="author">Niemiller ML</span>, <span class="author">Near TJ</span>, <span class="author">Fitzpatrick BM</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Delimiting species using multilocus data: diagnosing cryptic diversity in the southern cavefish <i>Typhlichthys subterraneus</i> (Teleostei: Amblyopsidae)</span>. <i>Evolution</i>,  <span class="vol">66</span>,  <span class="pageFirst">846</span>– <span class="pageLast">866</span>.
                     <div class="extra-links" id="e_1_2_11_62_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.2011.01480.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22380444" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000300931400018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=66&amp;publication_year=2012&amp;pages=846-866&amp;journal=Evolution&amp;author=ML+Niemiller&amp;author=TJ+Near&amp;author=BM+Fitzpatrick&amp;title=Delimiting+species+using+multilocus+data%3A+diagnosing+cryptic+diversity+in+the+southern+cavefish+Typhlichthys+subterraneus+%28Teleostei%3A+Amblyopsidae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0062"><span class="author">O'Dushlaine C</span>, <span class="author">McQuillan R</span>, <span class="author">Weale M</span> <i>et&nbsp;al</i>. (<span class="pubYear">2010</span>)  <span class="articleTitle">Genes predict village of origin in rural Europe. <i>European</i></span>. <i>Journal of Human Genetics</i>  <span class="vol">18</span>,  <span class="pageFirst">1269</span>– <span class="pageLast">1270</span>.
                     <div class="extra-links" id="e_1_2_11_63_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1038%2Fejhg.2010.92" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000283314700018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2010&amp;pages=1269-1270&amp;journal=Journal+of+Human+Genetics&amp;author=C+O%27Dushlaine&amp;author=R+McQuillan&amp;author=M+Weale&amp;title=Genes+predict+village+of+origin+in+rural+Europe.+European" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0063"><span class="author">O'Meara BC</span> (<span class="pubYear">2008</span>)  <span class="otherTitle">Using
 Trees: Myrmecocystus phylogeny and character evolution and new methods 
for investigating trait evolution and species delimitation</span>. PhD Dissertation, Nature Proceedings <a target="_blank" class="accessionId" href="https://doi.org/10.1038/npre.2008.2261.1" title="Link to external resource: http://dx.doi.org/10.1038/npre.2008.2261.1">http://dx.doi.org/10.1038/npre.2008.2261.1</a><div class="extra-links" id="e_1_2_11_64_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&amp;q=O%27Meara+BC+%282008%29+Using+Trees%3A+Myrmecocystus+phylogeny+and+character+evolution+and+new+methods+for+investigating+trait+evolution+and+species+delimitation.+PhD+Dissertation%2C+Nature+Proceedings+http%3A%2F%2Fdx.doi.org%2F10.1038%2Fnpre.2008.2261.1" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0064"><span class="author">O'Meara BC</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">New heuristic methods for joint species delimitation and species tree inference</span>. <i>Systematic Biology</i>,  <span class="vol">59</span>,  <span class="pageFirst">59</span>– <span class="pageLast">73</span>.
                     <div class="extra-links" id="e_1_2_11_65_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1093%2Fsysbio%2Fsyp077" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20525620" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000272689500005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=59&amp;publication_year=2010&amp;pages=59-73&amp;journal=Systematic+Biology&amp;author=BC+O%27Meara&amp;title=New+heuristic+methods+for+joint+species+delimitation+and+species+tree+inference" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0065"><span class="author">Padial JM</span>, <span class="author">Miralles A</span>, <span class="author">De la Riva I</span>, <span class="author">Vences M</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">The integrative future of taxonomy</span>. <i>Frontiers in Zoology</i>,  <span class="vol">7</span>,  <span class="pageFirst">1</span>– <span class="pageLast">14</span>.
                     <div class="extra-links" id="e_1_2_11_66_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20148101" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000274695300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=7&amp;publication_year=2010&amp;pages=1-14&amp;journal=Frontiers+in+Zoology&amp;author=JM+Padial&amp;author=A+Miralles&amp;author=I+Riva&amp;author=M+Vences&amp;title=The+integrative+future+of+taxonomy" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0066"><span class="author">Peterson AT</span> (<span class="pubYear">2001</span>)  <span class="articleTitle">Predicting species geographic distributions based on ecological niche modeling</span>. <i>Condor</i>,  <span class="vol">103</span>,  <span class="pageFirst">599</span>– <span class="pageLast">605</span>.
                     <div class="extra-links" id="e_1_2_11_67_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1650%2F0010-5422%282001%29103%5B0599%3APSGDBO%5D2.0.CO%3B2" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000170390200019" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=103&amp;publication_year=2001&amp;pages=599-605&amp;journal=Condor&amp;author=AT+Peterson&amp;title=Predicting+species+geographic+distributions+based+on+ecological+niche+modeling" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0067"><span class="author">Pons J</span>, <span class="author">Barraclough TG</span>, <span class="author">Gomez‐Zurita J</span> <i>et&nbsp;al</i>. (<span class="pubYear">2006</span>)  <span class="articleTitle">Sequence‐based species delimitation for the DNA taxonomy of undescribed insects</span>. <i>Systematic Biology</i>,  <span class="vol">55</span>,  <span class="pageFirst">595</span>– <span class="pageLast">609</span>.
                     <div class="extra-links" id="e_1_2_11_68_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1080%2F10635150600852011" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=16967577" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000239572300005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=55&amp;publication_year=2006&amp;pages=595-609&amp;journal=Systematic+Biology&amp;author=J+Pons&amp;author=TG+Barraclough&amp;author=J+Gomez%E2%80%90Zurita&amp;title=Sequence%E2%80%90based+species+delimitation+for+the+DNA+taxonomy+of+undescribed+insects" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0068"><span class="author">Pritchard JK</span>, <span class="author">Stephens M</span>, <span class="author">Donnelly P</span> (<span class="pubYear">2000</span>)  <span class="articleTitle">Inference of population structure using multilocus genotype data</span>. <i>Genetics</i>,  <span class="vol">155</span>,  <span class="pageFirst">945</span>– <span class="pageLast">959</span>.
                     <div class="extra-links" id="e_1_2_11_69_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2004.02396.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ASTN%3A280%3ADC%252BD3cvislKrtA%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=10835412" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000087475100039" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;journal=Genetics&amp;author=JK+Pritchard&amp;author=M+Stephens&amp;author=P+Donnelly&amp;title=Inference+of+population+structure+using+multilocus+genotype+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0069"><span class="author">Puillandre N</span>, <span class="author">Baylac M</span>, <span class="author">Boisselier M‐C</span>, <span class="author">Cruaud C</span>, <span class="author">Samadi S</span> (<span class="pubYear">2009</span>)  <span class="articleTitle">An integrative approach to species delimitation in Benthomangelia (Mollusca: Conoidea)</span>. <i>Biological Journal Linnean Society</i>,  <span class="vol">96</span>,  <span class="pageFirst">696</span>– <span class="pageLast">708</span>.
                     <div class="extra-links" id="e_1_2_11_70_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.2008.01143.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000263678300016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=96&amp;publication_year=2009&amp;pages=696-708&amp;journal=Biological+Journal+Linnean+Society&amp;author=N+Puillandre&amp;author=M+Baylac&amp;author=M%E2%80%90C+Boisselier&amp;author=C+Cruaud&amp;author=S+Samadi&amp;title=An+integrative+approach+to+species+delimitation+in+Benthomangelia+%28Mollusca%3A+Conoidea%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0070"><span class="author">Puillandre N</span>, <span class="author">Modica MV</span>, <span class="author">Zhang Y</span> <i>et&nbsp;al</i>. (<span class="pubYear">2012</span>)  <span class="articleTitle">Large‐scale species delimitation method for hyperdiverse groups</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>,  <span class="pageFirst">2671</span>– <span class="pageLast">2691</span>.
                     <div class="extra-links" id="e_1_2_11_71_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05559.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1Wlu7vE" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22494453" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000304389500012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=2671-2691&amp;journal=Molecular+Ecology&amp;author=N+Puillandre&amp;author=MV+Modica&amp;author=Y+Zhang&amp;title=Large%E2%80%90scale+species+delimitation+method+for+hyperdiverse+groups" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0071"><span class="author">de Queiroz K</span> (<span class="pubYear">2005</span>)  <span class="articleTitle">Ernst Mayr and the modern concept of species</span>. <i>Proceedings of the National Academy of Sciences USA</i>,  <span class="vol">102</span>,  <span class="pageFirst">6600</span>– <span class="pageLast">6607</span>.
                     <div class="extra-links" id="e_1_2_11_72_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1073%2Fpnas.0502030102" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD2MXktlaisLo%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=15851674" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000229023700014" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=102&amp;publication_year=2005&amp;pages=6600-6607&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+USA&amp;author=K+Queiroz&amp;title=Ernst+Mayr+and+the+modern+concept+of+species" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0072"><span class="author">de Queiroz K</span> (<span class="pubYear">2007</span>)  <span class="articleTitle">Species concepts and species delimitation</span>. <i>Systematic Biology</i>,  <span class="vol">56</span>,  <span class="pageFirst">879</span>– <span class="pageLast">886</span>.
                     <div class="extra-links" id="e_1_2_11_73_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1080%2F10635150701701083" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18027281" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000252360100002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2007&amp;pages=879-886&amp;journal=Systematic+Biology&amp;author=K+Queiroz&amp;title=Species+concepts+and+species+delimitation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0073"><span class="author">Rannala B</span>, <span class="author">Yang Z</span> (<span class="pubYear">2003</span>)  <span class="articleTitle">Bayes estimation of species divergence times and ancestral population sizes using DNA sequences from multiple loci</span>. <i>Genetics</i>,  <span class="vol">164</span>,  <span class="pageFirst">1645</span>– <span class="pageLast">1656</span>.
                     <div class="extra-links" id="e_1_2_11_74_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvF2nt74%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=12930768" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000185248000036" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=164&amp;publication_year=2003&amp;pages=1645-1656&amp;journal=Genetics&amp;author=B+Rannala&amp;author=Z+Yang&amp;title=Bayes+estimation+of+species+divergence+times+and+ancestral+population+sizes+using+DNA+sequences+from+multiple+loci" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0074"><span class="author">Rannala B</span>, <span class="author">Yang Z</span> (<span class="pubYear">2013</span>)  <span class="articleTitle">Improved reversible jump algorithms for Bayesian Species delimitation</span>. <i>Genetics</i>,  <span class="vol">194</span>,  <span class="pageFirst">245</span>– <span class="pageLast">253</span>.
                     <div class="extra-links" id="e_1_2_11_75_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1534%2Fgenetics.112.149039" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=23502678" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000318274900024" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=194&amp;publication_year=2013&amp;pages=245-253&amp;journal=Genetics&amp;author=B+Rannala&amp;author=Z+Yang&amp;title=Improved+reversible+jump+algorithms+for+Bayesian+Species+delimitation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0075"><span class="author">Reid NM</span>, <span class="author">Carstens BC</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Phylogenetic
 estimation error can decrease the accuracy species delimitation: a 
Bayesian implementation of the General Mixed Yule Coalescent model</span>. <i>BMC Evolutionary Biology</i>,  <span class="vol">12</span>,  <span class="pageFirst">196</span>.
                     <div class="extra-links" id="e_1_2_11_76_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1186%2F1471-2148-12-196" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=23031350" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000311513300001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2012&amp;pages=196&amp;journal=BMC+Evolutionary+Biology&amp;author=NM+Reid&amp;author=BC+Carstens&amp;title=Phylogenetic+estimation+error+can+decrease+the+accuracy+species+delimitation%3A+a+Bayesian+implementation+of+the+General+Mixed+Yule+Coalescent+model" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0076"><span class="author">Rielly SB</span>, <span class="author">Marks SB</span>, <span class="author">Jennings WB</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Defining evolutionary boundaries across parapatric ecomorphs of Black salamanders (<i>Aneides flavipunctatus</i>) with conservation implications</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>,  <span class="pageFirst">5745</span>– <span class="pageLast">5761</span>.
                     <div class="extra-links" id="e_1_2_11_77_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.12068" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=23094781" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000311399000012" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=5745-5761&amp;journal=Molecular+Ecology&amp;author=SB+Rielly&amp;author=SB+Marks&amp;author=WB+Jennings&amp;title=Defining+evolutionary+boundaries+across+parapatric+ecomorphs+of+Black+salamanders+%28Aneides+flavipunctatus%29+with+conservation+implications" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0077"><span class="author">Rittmeyer EN</span>, <span class="author">Austin CC</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">The effects of sampling on delimiting species from multi‐ locus sequence data</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">65</span>,  <span class="pageFirst">451</span>– <span class="pageLast">463</span>.
                     <div class="extra-links" id="e_1_2_11_78_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2012.06.031" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlOjur3L" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22820460" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000309636200009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=65&amp;publication_year=2012&amp;pages=451-463&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=EN+Rittmeyer&amp;author=CC+Austin&amp;title=The+effects+of+sampling+on+delimiting+species+from+multi%E2%80%90+locus+sequence+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0078"><span class="author">Rosell JA</span>, <span class="author">Olson ME</span>, <span class="author">Weeks A</span> <i>et&nbsp;al</i>. (<span class="pubYear">2010</span>)  <span class="articleTitle">Diversification in species complexes: tests of species origin and delimitation in the <i>Bursera simaruba</i> clade of tropical trees (Burseraceae)</span>. <i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">57</span>,  <span class="pageFirst">798</span>– <span class="pageLast">811</span>.
                     <div class="extra-links" id="e_1_2_11_79_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1016%2Fj.ympev.2010.08.004" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20723609" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000284176800026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=57&amp;publication_year=2010&amp;pages=798-811&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=JA+Rosell&amp;author=ME+Olson&amp;author=A+Weeks&amp;title=Diversification+in+species+complexes%3A+tests+of+species+origin+and+delimitation+in+the+Bursera+simaruba+clade+of+tropical+trees+%28Burseraceae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0079"><span class="author">Rosenberg NA</span> (<span class="pubYear">2002</span>)  <span class="articleTitle">The probability of topological concordance of gene trees and species trees</span>. <i>Theoretical Population Biology</i>,  <span class="vol">61</span>,  <span class="pageFirst">225</span>– <span class="pageLast">247</span>.
                     <div class="extra-links" id="e_1_2_11_80_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1006%2Ftpbi.2001.1568" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=11969392" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000175473300010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=61&amp;publication_year=2002&amp;pages=225-247&amp;journal=Theoretical+Population+Biology&amp;author=NA+Rosenberg&amp;title=The+probability+of+topological+concordance+of+gene+trees+and+species+trees" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0080"><span class="author">Salter JD</span>, <span class="author">Carstens BC</span>, <span class="author">Hedin M</span> (<span class="pubYear">2013</span>)  <span class="articleTitle">Multilocus species delimitation in a complex of morphologically conserved trapdoor spiders (Mygalomorphae, Antrodiaetidae, <i>Aliatypus</i>)</span>. <i>Systematic Biology</i>, in press.
                     <div class="extra-links" id="e_1_2_11_81_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;journal=Systematic+Biology&amp;author=JD+Salter&amp;author=BC+Carstens&amp;author=M+Hedin&amp;title=Multilocus+species+delimitation+in+a+complex+of+morphologically+conserved+trapdoor+spiders+%28Mygalomorphae%2C+Antrodiaetidae%2C+Aliatypus%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0081"><span class="author">Saunders IW</span>, <span class="author">Tavare S</span>, <span class="author">Watterson GA</span> (<span class="pubYear">1994</span>)  <span class="articleTitle">On the genealogy of nested subsamples from a haploid population</span>. <i>Advances in Applied Probability</i>,  <span class="vol">16</span>,  <span class="pageFirst">471</span>– <span class="pageLast">491</span>.
                     <div class="extra-links" id="e_1_2_11_82_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.2307%2F1427285" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=A1984TH15000002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1994&amp;pages=471-491&amp;journal=Advances+in+Applied+Probability&amp;author=IW+Saunders&amp;author=S+Tavare&amp;author=GA+Watterson&amp;title=On+the+genealogy+of+nested+subsamples+from+a+haploid+population" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0082"><span class="author">Schlick‐Steiner BC</span>, <span class="author">Steiner FM</span>, <span class="author">Seifert B</span>, <span class="author">Stauffer C</span>, <span class="author">Christian E</span>, <span class="author">Crozier RH</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Integrative Taxonomy: a multisource approach to exploring biodiversity</span>. <i>Annual Reviews of Entomology</i>,  <span class="vol">55</span>,  <span class="pageFirst">421</span>– <span class="pageLast">438</span>.
                     <div class="extra-links" id="e_1_2_11_83_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1146%2Fannurev-ento-112408-085432" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptVShtQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=19737081" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000273712100022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=55&amp;publication_year=2010&amp;pages=421-438&amp;journal=Annual+Reviews+of+Entomology&amp;author=BC+Schlick%E2%80%90Steiner&amp;author=FM+Steiner&amp;author=B+Seifert&amp;author=C+Stauffer&amp;author=E+Christian&amp;author=RH+Crozier&amp;title=Integrative+Taxonomy%3A+a+multisource+approach+to+exploring+biodiversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0083"><span class="author">Setiadi MI</span>, <span class="author">McGuire JA</span>, <span class="author">Brown RM</span> <i>et&nbsp;al</i>. (<span class="pubYear">2011</span>)  <span class="articleTitle">Adaptive radiation and ecological opportunity in Sulawesi and Philippine Fanged Frog (<i>Limnonectes</i>) Communities</span>. <i>American Naturalist</i>,  <span class="vol">178</span>,  <span class="pageFirst">221</span>– <span class="pageLast">240</span>.
                     <div class="extra-links" id="e_1_2_11_84_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1086%2F660830" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21750386" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000292917900010" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=178&amp;publication_year=2011&amp;pages=221-240&amp;journal=American+Naturalist&amp;author=MI+Setiadi&amp;author=JA+McGuire&amp;author=RM+Brown&amp;title=Adaptive+radiation+and+ecological+opportunity+in+Sulawesi+and+Philippine+Fanged+Frog+%28Limnonectes%29+Communities" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0084"><span class="author">Simpson GG</span> (<span class="pubYear">1951</span>)  <span class="articleTitle">The species concept</span>. <i>Evolution</i>,  <span class="vol">5</span>,  <span class="pageFirst">285</span>– <span class="pageLast">298</span>.
                     <div class="extra-links" id="e_1_2_11_85_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1558-5646.1951.tb02788.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=A1951XW58000001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=1951&amp;pages=285-298&amp;journal=Evolution&amp;author=GG+Simpson&amp;title=The+species+concept" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0085"><span class="author">Stech M</span>, <span class="author">Veldman S</span>, <span class="author">Larrain J</span> <i>et&nbsp;al</i>. (<span class="pubYear">2013</span>)  <span class="articleTitle">Molecular species delimitation in the <i>Racomitrium canescens</i> complex (Grimmiaceae) and implications for DNA barcoding of species complexes in Mosses</span>. <i>PLoS ONE</i>,  <span class="vol">8</span>,  <span class="pageFirst">e15614</span>.
                     <div class="extra-links" id="e_1_2_11_86_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1371%2Fjournal.pone.0053134" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVGksbk%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000314759400032" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=2013&amp;pages=e15614&amp;journal=PLoS+ONE&amp;author=M+Stech&amp;author=S+Veldman&amp;author=J+Larrain&amp;title=Molecular+species+delimitation+in+the+Racomitrium+canescens+complex+%28Grimmiaceae%29+and+implications+for+DNA+barcoding+of+species+complexes+in+Mosses" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0086"><span class="author">Stevens RD</span>, <span class="author">Gavilanez MM</span>, <span class="author">Tello JS</span>, <span class="author">Ray DA</span> (<span class="pubYear">2012</span>)  <span class="articleTitle">Phylogenetic structure illuminates the mechanistic role of environmental heterogeneity in community organization</span>. <i>Journal of Animal Ecology</i>,  <span class="vol">81</span>,  <span class="pageFirst">455</span>– <span class="pageLast">462</span>.
                     <div class="extra-links" id="e_1_2_11_87_1"><span>
                           <a href="https://besjournals.onlinelibrary.wiley.com/doi/10.1111/j.1365-2656.2011.01900.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21895648" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000300502600016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=81&amp;publication_year=2012&amp;pages=455-462&amp;journal=Journal+of+Animal+Ecology&amp;author=RD+Stevens&amp;author=MM+Gavilanez&amp;author=JS+Tello&amp;author=DA+Ray&amp;title=Phylogenetic+structure+illuminates+the+mechanistic+role+of+environmental+heterogeneity+in+community+organization" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0087"><span class="author">Stielow B</span>, <span class="author">Bratek Z</span>, <span class="author">Orczan AKI</span> <i>et&nbsp;al</i>. (<span class="pubYear">2011</span>)  <span class="articleTitle">Species delimitation in taxonomically difficult Fungi: the case of Hymenogaster</span>. <i>PLoS ONE</i>,  <span class="vol">6</span>,  <span class="pageFirst">e15614</span>.
                     <div class="extra-links" id="e_1_2_11_88_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1371%2Fjournal.pone.0015614" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1GltbY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=21311589" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000286661400009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=6&amp;publication_year=2011&amp;pages=e15614&amp;journal=PLoS+ONE&amp;author=B+Stielow&amp;author=Z+Bratek&amp;author=AKI+Orczan&amp;title=Species+delimitation+in+taxonomically+difficult+Fungi%3A+the+case+of+Hymenogaster" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0088"><span class="author">Weisrock DW</span>, <span class="author">Rasoloarison RM</span>, <span class="author">Fiorentino I</span> <i>et&nbsp;al</i>. (<span class="pubYear">2010</span>)  <span class="articleTitle">Delimiting species without nuclear monophyly in Madagascar's Mouse lemurs</span>. <i>PLoS ONE</i>,  <span class="vol">5</span>,  <span class="pageFirst">e9883</span>.
                     <div class="extra-links" id="e_1_2_11_89_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1371%2Fjournal.pone.0009883" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFehu7k%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20360988" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000276163200003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2010&amp;pages=e9883&amp;journal=PLoS+ONE&amp;author=DW+Weisrock&amp;author=RM+Rasoloarison&amp;author=I+Fiorentino&amp;title=Delimiting+species+without+nuclear+monophyly+in+Madagascar%27s+Mouse+lemurs" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0089"><span class="author">Welton L</span>, <span class="author">Siler C</span>, <span class="author">Oaks J</span>, <span class="author">Diesmos A</span>, <span class="author">Brown R</span> (<span class="pubYear">2013</span>)  <span class="articleTitle">Multilocus
 phylogeny and Bayes estimates of species boundaries reveal hidden 
evolutionary relationships and cryptic diversity in Southeast Asian 
monitor lizards</span>. <i>Molecular Ecology</i>, in press.
                     <div class="extra-links" id="e_1_2_11_90_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.12324" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=23710945" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000320942000009" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2013&amp;journal=Molecular+Ecology&amp;author=L+Welton&amp;author=C+Siler&amp;author=J+Oaks&amp;author=A+Diesmos&amp;author=R+Brown&amp;title=Multilocus+phylogeny+and+Bayes+estimates+of+species+boundaries+reveal+hidden+evolutionary+relationships+and+cryptic+diversity+in+Southeast+Asian+monitor+lizards" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0090"><span class="author">Wiens JJ</span> (<span class="pubYear">2007</span>)  <span class="articleTitle">Species delimitation: new approaches for discovering diversity</span>. <i>Systematic Biology</i>,  <span class="vol">56</span>,  <span class="pageFirst">875</span>– <span class="pageLast">878</span>.
                     <div class="extra-links" id="e_1_2_11_91_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1080%2F10635150701748506" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=18027280" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000252360100001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2007&amp;pages=875-878&amp;journal=Systematic+Biology&amp;author=JJ+Wiens&amp;title=Species+delimitation%3A+new+approaches+for+discovering+diversity" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0091"><span class="author">Wiens JJ</span>, <span class="author">Penkrot TA</span> (<span class="pubYear">2002</span>)  <span class="articleTitle">Delimiting species using DNA and morphological variation and discordant species limits in spiny lizards (Sceloporus)</span>. <i>Systematic Biology</i>,  <span class="vol">51</span>,  <span class="pageFirst">69</span>– <span class="pageLast">91</span>.
                     <div class="extra-links" id="e_1_2_11_92_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1080%2F106351502753475880" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=11943093" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000174121300006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2002&amp;pages=69-91&amp;journal=Systematic+Biology&amp;author=JJ+Wiens&amp;author=TA+Penkrot&amp;title=Delimiting+species+using+DNA+and+morphological+variation+and+discordant+species+limits+in+spiny+lizards+%28Sceloporus%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0092"><span class="author">Wiley EO</span> (<span class="pubYear">1978</span>)  <span class="articleTitle">The evolutionary species concept reconsidered</span>. <i>Systematic Zoology</i>,  <span class="vol">27</span>,  <span class="pageFirst">17</span>– <span class="pageLast">26</span>.
                     <div class="extra-links" id="e_1_2_11_93_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.2307%2F2412809" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=A1978EY54300002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=1978&amp;pages=17-26&amp;journal=Systematic+Zoology&amp;author=EO+Wiley&amp;title=The+evolutionary+species+concept+reconsidered" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0093"><span class="author">Yang Z</span>, <span class="author">Rannala B</span> (<span class="pubYear">2010</span>)  <span class="articleTitle">Bayesian species delimitation using multilocus sequence data</span>. <i>Proceedings of the National Academy of Sciences USA</i>,  <span class="vol">107</span>,  <span class="pageFirst">9264</span>– <span class="pageLast">9269</span>.
                     <div class="extra-links" id="e_1_2_11_94_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fmec.12413&amp;key=10.1073%2Fpnas.0913022107" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmslGrtLs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=20439743" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000277822600045" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=107&amp;publication_year=2010&amp;pages=9264-9269&amp;journal=Proceedings+of+the+National+Academy+of+Sciences+USA&amp;author=Z+Yang&amp;author=B+Rannala&amp;title=Bayesian+species+delimitation+using+multilocus+sequence+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="mec12413-bib-0094"><span class="author">Zhou W‐W</span>, <span class="author">Wen Y</span>, <span class="author">Fu J</span> <i>et&nbsp;al</i>. (<span class="pubYear">2012</span>)  <span class="articleTitle">Speciation in the <i>Rana chensinensis</i> species complex and its relationship to the uplift of the Qinghai–Tibetan Plateau</span>. <i>Molecular Ecology</i>,  <span class="vol">21</span>,  <span class="pageFirst">960</span>– <span class="pageLast">973</span>.
                     <div class="extra-links" id="e_1_2_11_95_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05411.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fmec.12413&amp;key=1%3ACAS%3A528%3ADC%252BC38XksVamt7k%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fmec.12413&amp;key=22221323" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fmec.12413&amp;key=000299416900013" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2012&amp;pages=960-973&amp;journal=Molecular+Ecology&amp;author=W%E2%80%90W+Zhou&amp;author=Y+Wen&amp;author=J+Fu&amp;title=Speciation+in+the+Rana+chensinensis+species+complex+and+its+relationship+to+the+uplift+of+the+Qinghai%E2%80%93Tibetan+Plateau" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
      <section class="article-section__content" id="mec12413-sec-0016">
         
         <p>Research in the Carstens Lab seeks to understand how 
biological diversity is generated using computational approaches. We 
investigate empirical systems by identifying the limits of evolutionary 
lineages, to evaluate the relative contributions of evolutionary 
processes and infer the ecological and environmental forces that have 
contributed to the formation of population genetic structure. All of the
 authors of this manuscript are interested in conducting research in 
intriguing natural systems and developing improved strategies for 
collecting and analysing data.</p>
         
      </section>
   </section>
   <div style="display:none">
      <ul class="custom-style-list noteList"></ul>
   </div>
   <section id="citedBy" class="article-section article-section__citedBy" data-source="https://onlinelibrary.wiley.com/action/ajaxShowCitedBy?doi=10.1111%2Fmec.12413">
      <div class="accordion article-accordion">
         <h2>
            <div tabindex="0" class="accordion__control" role="button" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
         </h2>
         <div class="accordion__content" role="region" aria-labelledby="citedby-section" id="idss15" style="display: none;"></div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md article-row-right fixed-element" style="top: -370px;"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/1365294x/2013/22/17"><img src="mec.jpg" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/1365294x/2013/22/17"><span>Volume</span><span class="val">22</span>, <span>Issue</span><span class="val">17</span></a></p><p>September 2013</p><p class="page-range"><span>Pages</span><span> 4369-4383</span></p>



        
        
</div>
</div>



        
        



        <div class="pb-ad">
            <div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fmec.12413%3Fcasa_token%3D0GNu1GWTRJkAAAAA%253AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl&pubId=50345552&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		</div>
        </div>
    

<div class="sticko__parent scrollThenFix" style="height: calc(-71px + 100vh);"><div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko  tab--flex tabs--xs"><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list cloned"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul><ul role="presentation" class="rlist tab__content sticko__child"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tabpanel" class="tab__pane article-row-right__panes active" aria-hidden="true"><div class="contentTab">
<section class="info-tab-wrapper">
<section class="metrics">
<h3 class="info-header">Metrics</h3>
<div class="section__body">
<div class="cited-by-count"><label>Citations: </label><span>455</span></div>
<div class="pb-dropzone" data-pb-dropzone="publicaitonContent-info-tab-metrics">
<script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
<div data-badge-popover="left" data-badge-type="1" data-doi="10.1111/mec.12413" data-hide-no-mentions="false" class="altmetric-embed" data-uuid="6914916c-b462-668e-8863-831ef341dcac"><a target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042" rel="popover" data-content="&lt;div&gt;    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042&amp;tab=blogs'&gt;
          Blogged by &lt;b&gt;1&lt;/b&gt;
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042&amp;tab=twitter'&gt;
          Tweeted by &lt;b&gt;13&lt;/b&gt;
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042&amp;tab=facebook'&gt;
          On &lt;b&gt;1&lt;/b&gt; Facebook pages
      &lt;/a&gt;
    &lt;/div&gt;

    &lt;div class='altmetric-embed readers' style='margin-top: 10px;'&gt;
          &lt;div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'&gt;
              &lt;b&gt;1157&lt;/b&gt; readers on Mendeley
          &lt;/div&gt;
          &lt;div class='altmetric-embed tip_citeulike'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;'&gt;
              &lt;b&gt;2&lt;/b&gt; readers on CiteULike
          &lt;/div&gt;
    &lt;/div&gt;
    &lt;div style='margin-top: 10px; text-align: center;'&gt;
        &lt;a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042'&gt;
            See more details
        &lt;/a&gt;
    &lt;/div&gt;
&lt;/div&gt;" style="display:inline-block;" data-badge-popover="left">
        <img alt="Article has an altmetric score of 17" src="https://d1uo4w7k31k5mn.cloudfront.net/v1/17.png" style="border:0; margin:0; max-width: none;" width="110px" height="20px">
</a></div>
</div>
</div>
</section>
<section class="details">
<h3 class="info-header">Details</h3>
<p class="copyright">© 2013 John Wiley &amp; Sons Ltd</p>
<p></p>
<ul class="cross-mark rlist">
</ul>
</section>
<section class="keywords">
<h3 class="info-header">Keywords</h3>
<ul class="rlist rlist--inline">
<li><a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=coalescent">coalescent</a></li>
<li><a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=phylogeography">phylogeography</a></li>
<li><a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=species%20delimitation">species delimitation</a></li>
</ul>
</section>
<section class="funding-information">
<h3 class="info-header">Funding Information</h3>
<ul class="unordered-list">
<li>NSF. Grant Numbers: DEB‐0918212, DEB‐1257784</li>
</ul>
</section>
<section class="publication-history">
<h3 class="info-header">Publication History</h3>
<ul class="rlist">
<li><label>Issue Online: </label>27 August 2013
</li>
<li><label>Version of Record online: </label>16 July 2013
</li>
<li><label>Manuscript accepted: </label>04 June 2013
</li>
<li><label>Manuscript revised: </label>31 May 2013
</li>
<li><label>Manuscript received: </label>04 April 2013
</li>
</ul>
</section>
</section>

</div><div class="emptyTab">



        
        



        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fmec.12413%3Fcasa_token%3D0GNu1GWTRJkAAAAA%253AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl&pubId=50345552&placeholderId=1013&productId=101018" target="_blank">
				<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	</div>
</aside>
        </div>
    

</div></li><li class="tab__spinner" style="display: none;"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a href="#" data-ux3-role="controller" role="button" class="figure-viewer__ctrl__close"><span class="icon-close_thin"><span class="sr-only">Close Figure Viewer</span></span></a></div><div class="figure-viewer__top__left"><a href="#" role="button" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"><span class="sr-only">Browse All Figures</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"><span class="sr-only">Return to Figure</span></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: calc(-56px + 100vh);"><div class="figure-viewer__cent__left"><a href="#" role="button" class="figure-viewer__ctrl__prev" style="display: none;"><span class="icon-arrow_l"><span class="sr-only">Previous Figure</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__next" style="display: none;"><span class="icon-arrow_r"><span class="sr-only">Next Figure</span></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
        </div>
    </div><!--end pagefulltext-->
<div class="w-slide"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main>

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>










        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h2 class="sr-only">
                Additional links
            </h2>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
  <h3>About Wiley Online Library</h3>
  <ul class="rlist">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help &amp; Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2019 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    




        
        <div class="footer-logo pull-right">
        
<a href="http://www.wiley.com/" title="Wiley Home Page" aria-label="Wiley"><img alt="Wiley Home Page" src="footer-logo-wiley-1510029248417.png"></a>

</div>
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        <!-- [Tag{id&#x3D;320, code&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;688, code&#x3D;ch00, name&#x3D;General Chemistry}, Tag{id&#x3D;1184, code&#x3D;ls2f, name&#x3D;Ecological Genetics}, Tag{id&#x3D;323, code&#x3D;ls40, name&#x3D;Evolution}, Tag{id&#x3D;1206, code&#x3D;ls43, name&#x3D;Molecular Evolution}, Tag{id&#x3D;1207, code&#x3D;ls41, name&#x3D;Evolutionary Biology}, Tag{id&#x3D;60, code&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;1180, code&#x3D;ls2g, name&#x3D;Molecular Ecology}, Tag{id&#x3D;93, code&#x3D;ch, name&#x3D;CHEMISTRY}, Tag{id&#x3D;317, code&#x3D;ls60, name&#x3D;Genetics}, Tag{id&#x3D;103071, code&#x3D;molec-eco_mec, name&#x3D;molec-eco_MEC}, Tag{id&#x3D;101738, code&#x3D;wly.ecology_000046, name&#x3D;wly.ecology_000046}] -->                          <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 




        
        
















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="loginPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="Cancel Login"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2 id="loginPopupHead">Log in to Wiley Online Library</h2>

            































<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        

        
        <input name="loginUri" value="/doi/full/10.1111/mec.12413?casa_token=0GNu1GWTRJkAAAAA%3AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl" type="hidden">
        
            




<input name="redirectUri" value="/doi/full/10.1111/mec.12413?casa_token=0GNu1GWTRJkAAAAA%3AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl" type="hidden">
<input name="loginUri" value="/doi/full/10.1111/mec.12413?casa_token=0GNu1GWTRJkAAAAA%3AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl" type="hidden">
            
        
        
        
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="username">Email or Customer ID</label>
                
        </div>
        
        
                <input id="username" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="text">
            
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                



















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        
        
                <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="password">
            
        <span class="password-eye-icon icon-eye hidden"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            
                            <span class="submit " disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom accessSubmit" name="submitButton" value="Log In" disabled="disabled" type="submit">
                            </span>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                        
                        NEW USER <span aria-hidden="true">&gt;</span>
                    </a>
                    
                        <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1111%2Fmec.12413%3Fcasa_token%3D0GNu1GWTRJkAAAAA%253AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl" class="linkBtn float-right instituation-login last-tab-item">
                            
                            INSTITUTIONAL LOGIN <span aria-hidden="true">&gt;</span>
                        </a><a class="fakeFocusoutElement" href="#"></a>
                    
                </div>
            

        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="changePasswordPopupHead">
    <div class="content">
        <a href="#" class="close cancel" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="changePasswordPopupHead">Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
                <a href="#" class="fakeFormLastElement last-tab-item"></a><a class="fakeFocusoutElement" href="#"></a>
            </div>
            <div class="success-template hidden">
                <h2>Password Changed Successfully</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        





<div class="popup registration-popup hidden" role="dialog" tabindex="-1" aria-labelledby="createAccountPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2 id="createAccountPopupHead">Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1111/mec.12413?casa_token=0GNu1GWTRJkAAAAA%3AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login last-tab-item">
                    
                    Returning user
                </a><a class="fakeFocusoutElement" href="#"></a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="resetPasswordPopupHead">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="resetPasswordPopupHead">Forgot your password?</h2>

            <p class="sub">Enter your email address below. </p>

        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        
        
                <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
            
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">

                <p>"Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>

            </div>
        </div>
    </div>
</div>





        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>




        
        




<div class="popup top-drawer request-username-drawer username-popup hidden" role="dialog" tabindex="-1" aria-labelledby="requestUsernamePopupHead">
    <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2 id="requestUsernamePopupHead">Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script><script src="satellite-5dacec9364746d6c440013e7.js"></script>
        </div>
    </div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh858" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.4507315700952257&amp;iit=1581928530317&amp;tmr=load%3D1581928528991%26core%3D1581928529013%26main%3D1581928530314%26ifr%3D1581928530320&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fmec.12413%3Fcasa_token%3D0GNu1GWTRJkAAAAA%253AuAgn5oH6OLfWjYWuZbMoOQ7q29H5gF3-W_xsc0JeflVc6zv0CluuJ-_WQ27MwPMuczmSicwp1xYJJ_cl&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fmec.12413&amp;dt=How%20to%20fail%20at%20species%20delimitation&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=description%2Csite_name%2Cimage%2Curl%2Ctype%3DArticle%2Ctitle&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5e4a5050a163eb73&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=title%3DHow%2520to%2520fail%2520at%2520species%2520delimitation%26type%3DArticle%26url%3Dhttps%253A%252F%252Fonlinelibrary.wiley.com%252Fdoi%252F10.1111%252Fmec.12413%26image%3Dhttps%253A%252F%252Fonlinelibrary.wiley.com%252Fcms%252Fasset%252Fd73e9a77-dfad-468b-9c64-463eaf0a5bcc%252Fmec.2013.22.issue-17.cover.gif%26site_name%3DWiley%2520Online%2520Library%26description%3DSpecies%2520delimitation%2520is%2520the%2520act%2520of%2520identifying%2520species%25E2%2580%2590level%2520biological%2520diversity.%2520In%2520recent%2520years%252C%2520the%2520field%2520has%2520witnessed%2520a%2520dramatic%2520increase%2520in%2520the%2520number%2520of%2520methods%2520available%2520for%2520delimiting%2520speci...&amp;csi=undefined&amp;rev=v8.28.3-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
<script src="build.js"></script><script src="main.js"></script>
<script type="text/javascript" src="ibwuproduct.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>
























    










    <script>
        $(document).ready(function () {
            var start = new Date();
            $(window).unload(function () {
                var end = new Date();
                $.ajax({
                    url: '/action/analytics',
                    method: 'post',
                    data: {
                        'Duration': end - start,
                        'DOI': digitalData.publication.item.doi,
                        'EventType': 'FullTextDuration'
                    },
                    async: false
                });
            });

           window.onafterprint = function () {
                $.ajax({
                    url: '/action/analytics',
                    method: 'post',
                    data: {
                        'DOI': digitalData.publication.item.doi,
                        'EventType': 'FullTextPrint'
                    }
                });
            };
        });
    </script>


<div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">Close crossmark popup</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe title="Crossmark service - Intentionally blank" aria-hidden="true" class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_wileypublishing_0" name="destination_publishing_iframe_wileypublishing_0_name" style="display: none; width: 0px; height: 0px;" src="https://wileypublishing.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fonlinelibrary.wiley.com"></iframe><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1111/mec.12413?callback=_altmetric.embed_callback&amp;domain=onlinelibrary.wiley.com&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=11-17"></script><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid="6914916c-b462-668e-8863-831ef341dcac" id="_altmetric_popover_el_6914916c-b462-668e-8863-831ef341dcac" style="margin: 0px; display: none;"><div class="altmetric_container" id="_altmetric_container"><div class="altmetric_arrow altmetric-left"></div><div id="_altmetric_popover_inner" class="altmetric-embed altmetric-popover-inner altmetric-floating altmetric-left" data-uuid="6914916c-b462-668e-8863-831ef341dcac"><div class="altmetric-embed altmetric-popover-content altmetric-floating"><div>    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042&amp;tab=blogs">
          Blogged by <b>1</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042&amp;tab=twitter">
          Tweeted by <b>13</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042&amp;tab=facebook">
          On <b>1</b> Facebook pages
      </a>
    </div>

    <div class="altmetric-embed readers" style="margin-top: 10px;">
          <div class="altmetric-embed tip_mendeley" style="padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;">
              <b>1157</b> readers on Mendeley
          </div>
          <div class="altmetric-embed tip_citeulike" style="padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;">
              <b>2</b> readers on CiteULike
          </div>
    </div>
    <div style="margin-top: 10px; text-align: center;">
        <a class="altmetric_details" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=1628042">
            See more details
        </a>
    </div>
</div></div></div></div></div></body><iframe src="https://injections.readcube.com/html/track_epdf.43d7bc64.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe></html>