<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="139e3b42-a383-4ab5-aa30-8c3e97eba0c9" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="https://cdn.krxd.net/userdata/get?pub=5c9d7251-7b10-4260-8df4-d859510ca122&amp;technographics=1&amp;callback=Krux.ns._default.kxjsonp_userdata"></script><script type="text/javascript" src="https://beacon.krxd.net/optout_check?callback=Krux.ns._default.kxjsonp_optOutCheck"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";article:article:doi\:10.1111/j.1365-3113.2006.00355.x;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;issue:issue:doi\:10.1111/sen.2007.32.issue-1;requestedJournal:journal:13653113;journal:journal:13653113b;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text">









<iframe style="position: absolute; top: -10000px; left: -1000px;"></iframe><script async="" src="prum.js"></script><script src="iframe_api"></script><script type="text/javascript" async="" src="conversion_async.js"></script><script async="" src="controltag.js"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script async="" src="fbevents.js"></script><script type="text/javascript">var $DoubleClickZone = "systematic-entom_syen";var $DoubleClickSite =  "wly.zoology_000120";</script><script>digitalData = {"site":{"ip":"105.27.236.251","environment":"LIVE","website":"onlinelibrary.wiley.com","serverDate":"2019-09-05","server":"web103"},"page":{"pageGroup":"Publication Pages","page":"Article/Chapter View","subPage":"Full Text","title":"title","errorCode":200},"identities":[{"type":"BasicGroup","uuid":"be20949b-b372-471e-96a5-9963c7266a9d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000054870"}]},{"type":"BasicGroup","uuid":"98ba5e39-8ca0-4d0a-addd-395777fbf515","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000034166"}]},{"type":"InstitutionUser","uuid":"6e97ed1f-543d-493d-b458-2e63cb8fbde5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#KLISC"}]},{"type":"BasicGroup","uuid":"e623deab-c83b-41e3-84a2-d9f1d46c5f17"},{"type":"RegionUser","uuid":"53f1676b-6f75-446d-8831-5915e1edce59"},{"type":"InstitutionUser","uuid":"dab9a169-ec74-4502-8066-0f76387b0f4e","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASP"}]},{"type":"BasicGroup","uuid":"40e3f768-07c0-45e5-aa81-97aef6eb686c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000005407"}]},{"type":"BasicGroup","uuid":"c3199058-0f44-4f36-b385-3a7c0023d85c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"KENYA HINARI NPL"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"5196cf05-b385-4de3-91a2-2d5ffd18abc5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASPKEN"}]},{"type":"InstitutionUser","uuid":"480e4dba-ca42-475f-a7ca-5b6fbfad49f7","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000014668"}]}],"publication":{"item":{"title":"A mitochondrial genome phylogeny of Diptera: whole genome sequence data accurately resolve relationships over broad timescales with high precision","doi":"10.1111/j.1365-3113.2006.00355.x","access":"full","earliestDate":"2006-10-17","formats":["Abstract","Cited by","First page image","Full text","PDF","References"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"AGRICULTURE","topicUri":"ag"},{"taxonomyUri":"global-subject-codes","topicLabel":"Entomology","topicUri":"lsc0"},{"taxonomyUri":"global-subject-codes","topicLabel":"General \u0026 Introductory Agriculture","topicUri":"ag00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Systematics","topicUri":"ls44"},{"taxonomyUri":"global-subject-codes","topicLabel":"Animal Science \u0026 Zoology","topicUri":"ls80"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"Behavior \u0026 Behavioral Ecology","topicUri":"ls2b"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biodiversity","topicUri":"ls2j"}]},"group":{"doi":"10.1111/sen.2007.32.issue-1","access":"full","volume":"32","issue":"1","coverDate":"2007-01-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"AGRICULTURE","topicUri":"ag"},{"taxonomyUri":"global-subject-codes","topicLabel":"Entomology","topicUri":"lsc0"},{"taxonomyUri":"global-subject-codes","topicLabel":"General \u0026 Introductory Agriculture","topicUri":"ag00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Systematics","topicUri":"ls44"},{"taxonomyUri":"global-subject-codes","topicLabel":"Animal Science \u0026 Zoology","topicUri":"ls80"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"Behavior \u0026 Behavioral Ecology","topicUri":"ls2b"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biodiversity","topicUri":"ls2j"}]},"series":{"title":"Systematic Entomology","doi":"10.1111/(ISSN)1365-3113","access":"no","code":"13653113","issn":"0307-6970","eissn":"1365-3113","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"AGRICULTURE","topicUri":"ag"},{"taxonomyUri":"global-subject-codes","topicLabel":"Entomology","topicUri":"lsc0"},{"taxonomyUri":"global-subject-codes","topicLabel":"General \u0026 Introductory Agriculture","topicUri":"ag00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Systematics","topicUri":"ls44"},{"taxonomyUri":"global-subject-codes","topicLabel":"Animal Science \u0026 Zoology","topicUri":"ls80"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"Behavior \u0026 Behavioral Ecology","topicUri":"ls2b"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biodiversity","topicUri":"ls2j"},{"taxonomyUri":"gpt-zonename","topicLabel":"systematic-entom_SYEN","topicUri":"systematic-entom_syen"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.zoology_000120","topicUri":"wly.zoology_000120"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    






    <meta property="og:title" content="A mitochondrial genome phylogeny of Diptera: whole genome sequence data accurately resolve relationships over broad timescales with high precision">


    <meta property="og:type" content="Article">


        <meta property="og:url" content="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-3113.2006.00355.x">
    

        
        <meta property="og:image" content="https://wol-prod-cdn.literatumonline.com/cms/attachment/583e73f2-9e19-415a-9ddb-c31fb0cca199/sen.2007.32.issue-1.cover.gif">
    

    <meta property="og:site_name" content="Wiley Online Library">



        

        <meta property="og:description" content="Abstract Mitochondrial genomes provide a promising new tool for understanding deep‐level insect phylogenetics, but have yet to be evaluated for their ability to resolve intraordinal relationships. We...">
    


    <meta name="viewport" content="width=device-width,initial-scale=1">




















    

    <meta name="citation_doi" content="10.1111/j.1365-3113.2006.00355.x">

    <meta name="dc.identifier" content="10.1111/j.1365-3113.2006.00355.x">




<meta name="citation_journal_title" content="Systematic Entomology">
<meta name="citation_title" content="A mitochondrial genome phylogeny of Diptera: whole genome sequence data accurately resolve relationships over broad timescales with high precision">
<meta name="citation_volume" content="32">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="40">
<meta name="citation_lastpage" content="59">
<meta name="citation_publication_date" content="2007/01/01">
<meta name="citation_online_date" content="2006/09/15">
<meta name="citation_issn" content="1365-3113">
<meta name="citation_language" content="en">
<meta name="citation_author" content="STEPHEN L. CAMERON">
<meta name="citation_author_institution" content="Department of Integrative Biology, Brigham Young University, Provo, Utah, U.S.A.">
<meta name="citation_author" content="CHRISTINE L. LAMBKIN">
<meta name="citation_author_institution" content="Australian National Insect Collection, CSIRO Entomology, Canberra, Australia">
<meta name="citation_author" content="STEPHEN C. BARKER">
<meta name="citation_author_institution" content="Parasitology Section, School of Molecular and Microbial Sciences, The University of Queensland, Brisbane, Australia">
<meta name="citation_author" content="MICHAEL F. WHITING">
<meta name="citation_author_institution" content="Department of Integrative Biology, Brigham Young University, Provo, Utah, U.S.A.">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-3113.2006.00355.x">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1365-3113.2006.00355.x">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-3113.2006.00355.x">
<meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd (10.1111)">
<meta name="article_references" content="CAMERON, S. L., LAMBKIN, C. L., BARKER, S. C. and WHITING, M. F. (2007), A mitochondrial genome phylogeny of Diptera: whole genome sequence data accurately resolve relationships over broad timescales with high precision. Systematic Entomology, 32: 40-59. doi:10.1111/j.1365-3113.2006.00355.x">
<meta name="epdf_available" content="true">
<meta name="Description" content="Abstract Mitochondrial genomes provide a promising new tool for understanding deep‐level insect phylogenetics, but have yet to be evaluated for their ability to resolve intraordinal relationships. ...">


<title>A mitochondrial genome phylogeny of Diptera: whole genome 
sequence data accurately resolve relationships over broad timescales 
with high precision - CAMERON - 2007 - Systematic Entomology - Wiley 
Online Library</title>






<link rel="stylesheet" type="text/css" href="hfpfproduct.css"><link rel="stylesheet" type="text/css" href="build.css">
<link rel="stylesheet" type="text/css" href="print.css" media="print">




<script async="" src="catchpoint-1549680351760.js"></script>
<meta name="referrer" content="origin-when-cross-origin">


<link rel="stylesheet" type="text/css" href="hub-1534777306460.css">


<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1552321492860.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* PERICLES-7186: distorted cover images */
.journal-info-container .cover-image__image img {
    min-height: unset;
}

/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents,
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}

/* TEMPORARY Fix heading on Topical Index widget for MRWs */
.topical-index h2 {
    margin: 15px 0;
    font-size: 1.75rem;
    line-height: 28px;
    color: #1c1d1e;
    font-family: Source Serif Pro,serif;
    font-weight: normal;
}

/* PERICLES-7288 */
.issue-supplementary-files, .cover-image__virtual-issue {
    margin-bottom: 15px;
}

/* PERICLES-8875 */
.journal-news .slide-item img {
    float: left;
    margin-right: 10px;
}

/* DO NOT REMOVE: HIDING SPECIFIC PB OPTIONS */
li[heading="Restore Points"].ng-isolate-scope,
li[heading="Disable Widgets"].ng-isolate-scope {
    display: none!important;
}

.modal-header .widgetOrigin {
    display: none;
}

/* PERICLES-10090: temporary workaround for problem with cover image widget */
.cover-image__image img {
    max-width: 150px;
    max-height: 150px;
}

/* LIT-282003: adding a delimiter to the isi rankings */
.journal-info-container span.info_value:after {
    content: '\003B';
}

.journal-info-container span.info_value:last-child:after {
    content: '';
}
</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5c87b4fe64746d78ad006ae3.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script><script src="satellite-5aaec36164746d79c1001d14.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>

<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	if (!$DoubleClickSite) {
		switch("pericles") {
			case "afspubs":
				var $DoubleClickSite = "wly.aqua.fish_000017";
				break;
			default:
				var $DoubleClickSite = "undefined-site";
		};
  	};

	if (!$DoubleClickZone) {
		switch("pericles") {
			case "afspubs":
				var $DoubleClickZone = "AFS_HUB";
				break;
			default:
				var $DoubleClickZone = "undefined-zone";
		};
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};
		
	
		gptPage = window.digitalData && digitalData.page && digitalData.page.page ? digitalData.page.page : "blank";
		gptSubPage = window.digitalData && digitalData.page && digitalData.page.subPage ? digitalData.page.subPage : "blank";

		
		function gptClean(gptText){
			if(typeof(gptText)=="string"){
				gptText = gptText.replace(/\&/g,'%26');
				
			
				return gptText.replace(/\^|\*|#|\$|\"|\'|\=|\,|\!|\+|\~|\;|\(|\)|\<|\>|\[|\]|\?|\%|\:|\@|\;|\/|\&|\||\{|\}|\\|/g,'');
				
			}
			return "blank";
		}

		
		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'doi' , '10.1111/j.1365-3113.2006.00355.x') // Publication DOI
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		) 	// Krux
			.setTargeting ( 'pbPage', gptClean(gptPage) )
			.setTargeting ( 'pbSubPage', gptClean(gptSubPage) );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>






    







    
        
    

















    <script src="casa.js" async=""></script>







    


<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "A mitochondrial genome phylogeny of Diptera: whole genome sequence data accurately resolve relationships over broad timescales with high precision - CAMERON - 2007 - Systematic Entomology - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker_002.js" async=""></script>
<style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script async="" type="text/javascript" src="epdf_linker.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019082701.js" async=""></script><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="satellite-5c58af5e64746d5111000925.js"></script><script src="satellite-5c87cfcf64746d7044002f7b.js"></script><script src="satellite-5cf7bd7564746d596c000241.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script src="js.js" async="true"></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><script src="satellite-5c87b56e64746d4e33003a7c.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-bd2db1bd449846a14796d4a8023cbb24.js"></script></head>
<body class="pb-ui">

<div class="skip-links">
    <ul class="rlist">
        
                <li><a href="#article__content" tabindex="1">Skip to Article Content</a></li>
                <li><a href="#pane-pcw-detailscon" tabindex="1">Skip to Article Information</a></li>
            
    </ul>
</div>







    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader fixed-element" style="position: fixed;">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        



        
    

</div></div></div>

        </div>
    




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library" tabindex="2"><img id="mainLogo" alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img id="mobileLogo" alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" aria-label="Search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper"><div class="tab clearfix"><input placeholder="Search" tabindex="13" class="fakeQSInput quickSearchFilter" type="search"><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" role="tablist" tabindex="10" class="search-within"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" selected="selected">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1">Anywhere</option></select><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="13653113" type="hidden"><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=13653113" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=13653113#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=13653113" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=13653113#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    









    <div class="loginBar login-open">
        <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x&amp;aria-label=Log+in+or+Register" class="show-login" role="button" uri="/doi/full/10.1111/j.1365-3113.2006.00355.x" aria-label="Log in or Register" tabindex="30">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </a>
            
    </div>



        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    










        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet body-text" style="padding-top: 71px;">
        



        
        <main id="main-content" tabindex="-1" style="">
    <!--begin pagefulltext-->
    <div class="page-body pagefulltext">
        <div data-pb-dropzone="main">
        



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        



        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x&pubId=59236166&placeholderId=1015&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.zoology_000120/systematic-entom_syen_0__container__" style="border: 0pt none; width: 100%; height: 0%;"></div></div>
	</aside>
        </div>
    


        </div>
    




        
        <style type="text/css">li.menu-item[data-menu-label="Accepted Articles"] {
  display: none;
}</style>
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-3113.2006.00355.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fsen.2007.32.issue-1%3BrequestedJournal%3Ajournal%3A13653113%3Bjournal%3Ajournal%3A13653113b%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h3" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-3113.2006.00355.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fsen.2007.32.issue-1%3BrequestedJournal%3Ajournal%3A13653113%3Bjournal%3Ajournal%3A13653113b%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h3" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-3113.2006.00355.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fsen.2007.32.issue-1%3BrequestedJournal%3Ajournal%3A13653113%3Bjournal%3Ajournal%3A13653113b%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h3&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-3113.2006.00355.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fsen.2007.32.issue-1%3BrequestedJournal%3Ajournal%3A13653113%3Bjournal%3Ajournal%3A13653113b%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h3&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div id="article__content" tabindex="-1" class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      <h1 id="journal-banner-text" style="font-size: 22px;"> <a href="https://onlinelibrary.wiley.com/journal/13653113" title="Systematic Entomology homepage">Systematic Entomology</a></h1><a href="https://onlinelibrary.wiley.com/toc/13653113/2007/32/1" class="volume-issue">Volume 32, Issue 1</a><a href="https://onlinelibrary.wiley.com/journal/13653113" title="Systematic Entomology homepage" class="citation--logo"><img id="journal-banner-image" test="test-value" alt="Systematic Entomology banner" src="13653113-1532540546857.jpg" class="journal-banner-image" style="display: none;"></a><script>var image = new Image();

image.onerror = function() {
  // if the image does not exist 
  //document.getElementById("journal-banner-text").style.marginBottom = "15px";
  document.getElementById("journal-banner-text").style.fontSize = "22px";
  document.getElementById("journal-banner-image").style.display = "none";
}

image.src = "https://wol-prod-cdn.literatumonline.com/pb-assets/journal-banners/13653113-1532540546857.jpg";</script>
   </div>
   <div class="doi-access-container clearfix">
      <div class="doi-access-wrapper"><div class="full-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Full Access</div></div></div></div>
   </div>
   <h2 class="citation__title">A mitochondrial genome phylogeny of 
Diptera: whole genome sequence data accurately resolve relationships 
over broad timescales with high precision</h2>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=CAMERON%2C+STEPHEN+L" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl" role="button"><span>STEPHEN L. CAMERON<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" role="region" id="am1">
            <p class="author-type">Corresponding Author</p>
            <p>Department of Integrative Biology, Brigham Young University, Provo, Utah, U.S.A.</p>Stephen L. Cameron, Department of Integrative Biology, Brigham Young University, Provo, UT 84602, U.S.A. E‐mail: 
            <p><a title="
                            Link to email address
                        " href="mailto:stephen_cameron@byu.edu" class="corr-email">stephen_cameron@byu.edu</a></p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=CAMERON%2C+STEPHEN+L">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=LAMBKIN%2C+CHRISTINE+L" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl" role="button"><span>CHRISTINE L. LAMBKIN</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" role="region" id="am2">
            <p>Australian National Insect Collection, CSIRO Entomology, Canberra, Australia</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=LAMBKIN%2C+CHRISTINE+L">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=BARKER%2C+STEPHEN+C" class="author-name accordion-tabbed__control" data-id="am3" data-db-target-for="am3" aria-controls="am3" aria-haspopup="true" id="am3_Ctrl" role="button"><span>STEPHEN C. BARKER</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am3" aria-labelledby="am3_Ctrl" role="region" id="am3">
            <p>Parasitology Section, School of Molecular and Microbial Sciences, The University of Queensland, Brisbane, Australia</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=BARKER%2C+STEPHEN+C">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=WHITING%2C+MICHAEL+F" class="author-name accordion-tabbed__control" data-id="am4" data-db-target-for="am4" aria-controls="am4" aria-haspopup="true" id="am4_Ctrl" role="button"><span>MICHAEL F. WHITING</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am4" aria-labelledby="am4_Ctrl" role="region" id="am4">
            <p>Department of Integrative Biology, Brigham Young University, Provo, Utah, U.S.A.</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=WHITING%2C+MICHAEL+F">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=CAMERON%2C+STEPHEN+L" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl" role="button"><span>STEPHEN L. CAMERON<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" role="region" id="a1">
                  <p class="author-type">Corresponding Author</p>
                  <p>Department of Integrative Biology, Brigham Young University, Provo, Utah, U.S.A.</p>Stephen L. Cameron, Department of Integrative Biology, Brigham Young University, Provo, UT 84602, U.S.A. E‐mail: 
                  <p><a title="
                            Link to email address
                        " href="mailto:stephen_cameron@byu.edu" class="corr-email">stephen_cameron@byu.edu</a></p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=CAMERON%2C+STEPHEN+L">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=LAMBKIN%2C+CHRISTINE+L" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl" role="button"><span>CHRISTINE L. LAMBKIN</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" role="region" id="a2">
                  <p>Australian National Insect Collection, CSIRO Entomology, Canberra, Australia</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=LAMBKIN%2C+CHRISTINE+L">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=BARKER%2C+STEPHEN+C" class="author-name accordion-tabbed__control" data-id="a3" data-db-target-for="a3" aria-controls="a3" aria-haspopup="true" id="a3_Ctrl" role="button"><span>STEPHEN C. BARKER</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a3" aria-labelledby="a3_Ctrl" role="region" id="a3">
                  <p>Parasitology Section, School of Molecular and Microbial Sciences, The University of Queensland, Brisbane, Australia</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=BARKER%2C+STEPHEN+C">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=WHITING%2C+MICHAEL+F" class="author-name accordion-tabbed__control" data-id="a4" data-db-target-for="a4" aria-controls="a4" aria-haspopup="true" id="a4_Ctrl" role="button"><span>MICHAEL F. WHITING</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a4" aria-labelledby="a4_Ctrl" role="region" id="a4">
                  <p>Department of Integrative Biology, Brigham Young University, Provo, Utah, U.S.A.</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=WHITING%2C+MICHAEL+F">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections">
      <div class="epub-section"><span class="epub-state">First published:&nbsp;</span><span class="epub-date">17 October 2006</span></div>
      <div class="epub-section"><a class="epub-doi" aria-label="Digital Object Identifier" href="https://doi.org/10.1111/j.1365-3113.2006.00355.x">https://doi.org/10.1111/j.1365-3113.2006.00355.x</a></div>
      <div class="epub-section cited-by-count"><span>Cited by: <a href="#citedby-section">138</a></span></div>
   </div>
   <div class="epub-sections">
      <div class="article-header__references-container no-truncate" id="fn1">
         <div class="header-note-content" aria-labelledby="header-note">
            <p>Many taxonomic groups recognized previously within 
Diptera are now known to be paraphyletic evolutionary grades. Throughout
 this paper, these terms have been retained to allow ready comparison 
with older literature and are designated by being placed in quotation 
marks.</p>
         </div>
      </div>
      <div class="article-header__references-container no-truncate" id="fn2">
         <div class="header-note-content" aria-labelledby="header-note">
            <p>Three additional dipteran mitochondrial genomes have become available recently on GenBank –<i> Dermatobia hominis </i>(Oestridae),<i> Haematobia irritans </i>(Muscidae) and<i> Aedes albopictus </i>(Culicidae)
 – but have not yet been published. We have not included these sequences
 so as not to pre‐empt other workers. Further, as close relatives of 
each have already been published, their inclusion is not critical to 
this study.
            </p>
         </div>
      </div>
      <div class="article-header__references-container no-truncate" id="fn3">
         <div class="header-note-content" aria-labelledby="header-note">
            <p>A secondary structure model for the mt rRNA genes of<i> Apis </i>will be published shortly (<a href="#b35"> Gillespie<i> et al.</i>, 2006</a>),
 and its application to other insect groups will no doubt greatly 
improve the alignments of insect mt rRNA genes for phylogenetic studies.
            </p>
         </div>
      </div>
   </div>
</div></div><nav class="coolBar stickybar"><div class="stickybar__wrapper coolBar__wrapper clearfix" style="width: 610px; top: 71px;"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><div class="transplanted-clone"><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1365-3113.2006.00355.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-3113.2006.00355.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></div></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d6309510"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss1-title"><span> Introduction</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss2-title"><span> Materials and methods</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss6-title"><span> Results</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss12-title"><span> Discussion</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss15-title"><span> Conclusion</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss16-title"><span> Acknowledgements</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><ul data-cb-group="Article" data-cb-group-icon="icon-toc" class="rlist coolBar__first"></ul><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1365-3113.2006.00355.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-3113.2006.00355.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2Fj.1365-3113.2006.00355.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1111%2Fj.1365-3113.2006.00355.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1111%2Fj.1365-3113.2006.00355.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" src="addthis_widget.js"></script><div class="relative"><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1111/j.1365-3113.2006.00355.x/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i><span class="sr-only">Close modal</span></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email</a></li><li><a role="link" class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a role="link" class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body "><article>
   
   
   <div class="abstract-group">
      <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
         <h2 id="d6309510" class="article-section__header section__title main abstractlang_en main">Abstract</h2>
         <div class="article-section__content en main">
            <p><b>Abstract</b> Mitochondrial genomes provide a promising
 new tool for understanding deep‐level insect phylogenetics, but have 
yet to be evaluated for their ability to resolve intraordinal 
relationships. We tested the utility of mitochondrial genome data for 
the resolution of relationships within Diptera, the insect order for 
which the most data are available. We sequenced an additional three 
genomes, from a syrphid, nemestrinid and tabanid, representing three 
additional dipteran clades, ‘aschiza’, non‐heteroneuran muscomorpha and 
‘basal brachyceran’, respectively. We assessed the influence of 
optimality criteria, gene inclusion/exclusion, data recoding and 
partitioning strategies on topology and nodal support within Diptera. 
Our consensus phylogeny of Diptera was largely consistent with previous 
phylogenetic hypotheses of the order, except that we did not recover a 
monophyletic Muscomorpha (Nesmestrinidae grouped with Tabanidae) or 
Acalyptratae (Drosophilidae grouped with Calliphoridae). The results 
were very robust to optimality criteria, as parsimony, likelihood and 
Bayesian approaches yielded very similar topologies, although nodal 
support varied. The addition of ribosomal and transfer RNA genes to the 
protein coding genes traditionally used in mitochondrial genome 
phylogenies improved the resolution and support, contrary to previous 
suggestions that these genes would evolve too quickly or prove too 
difficult to align to provide phylogenetic signal at deep nodes. 
Strategies to recode data, aimed at reducing homoplasy, resulted in a 
decrease in tree resolution and branch support. Bayesian analyses were 
highly sensitive to partitioning strategy: biologically realistic 
partitions into codon groups produced the best results. The implications
 of this study for dipteran systematics and the effective approaches to 
using mitochondrial genome data are discussed. Mitochondrial genomes 
resolve intraordinal relationships within Diptera accurately over very 
wide time ranges (1–200 million years ago) and genetic distances, 
suggesting that this may be an excellent data source for deep‐level 
studies within other, less studied, insect orders.</p>
            
         </div>
      </section>
   </div>
   <div class="pb-dropzone" data-pb-dropzone="below-abstract-group">
      <!-- Empty dropzone -->
   </div>
   
   <section class="article-section article-section__full">
      <div class="article-section__content" id="ss1">
         
         <h2 class="article-section__title section__title section1" id="ss1-title"> Introduction</h2>
         
         <p>Considerable effort is currently being expended to develop 
new classes of molecular data for use in insect systematics, with the 
view that many of the higher level relationships (inter‐ and 
intraordinal level) cannot be resolved satisfactorily with the current 
limited array of molecular markers which have been optimized for routine
 sequencing. Of the vast range of potential genes which could be used 
for phylogenetic inference, only the nuclear ribosomal RNA (rRNA) genes (<i>18S </i>and<i> 28S</i>), some of the histone subunits (principally<i> H3</i>), portions of the mitochondrial (mt) genes (<i>cox1</i>,<i> cox2</i>,<i> cytB</i>,<i> 16S </i>and<i> 12S</i>) and a few developmental genes, such as wingless (<i>Wg</i>) and Hox (<i>Hx</i>), are sufficiently conservative to be sequenced readily across all orders (<span title=""><a href="#b17" id="#b17R" class="bibLink tab-link" data-tab="pane-pcw-references">Caterino<i> et al.</i>, 2000</a></span>).
 Other genes, such as opsin, elongation factors and rudimentary (CAD), 
have been optimized for some orders, but have proven difficult to use 
across multiple orders (<span title=""><a href="#b62" id="#b62R" class="bibLink tab-link" data-tab="pane-pcw-references">Moulton &amp; Wiegmann, 2004</a></span>). New markers are being pursued to expand this array of tools to improve phylogenetic resolution between and within orders.
         </p>
         
         <p>One potential marker is the mt genome. With thirty‐seven 
genes [thirteen protein coding, two rRNAs and twenty‐two transfer RNAs 
(tRNAs)], and usually between 14 000 and 17 000 base pairs in size, the 
mt genome of Metazoa is the smallest extant genome. This size makes it 
technically tractable to sequence in its entirety, but it is still an 
order of magnitude larger than most of the single genes used in current 
insect phylogenetic analyses (<span title=""><a href="#b75" id="#b75R" class="bibLink tab-link" data-tab="pane-pcw-references">Saccone<i> et al.</i>, 1999</a></span>).
 Two approaches to the use of the mt genome have been proposed: 
uncovering shared genome rearrangements and use of the whole genome in 
sequence‐based phylogenies (e.g. <span title=""><a href="#b12%20#b72" id="#b12 #b72R" class="bibLink tab-link" data-tab="pane-pcw-references">Boore &amp; Brown, 1998; Rokas &amp; Holland, 2000</a></span>). Because some of the first insect mt genomes to be sequenced, i.e.<i> Apis mellifera</i>, Hymenoptera (<span title=""><a href="#b24" id="#b24R" class="bibLink tab-link" data-tab="pane-pcw-references">Crozier &amp; Crozier, 1993</a></span>),<i> Locusta migratoria</i>, Orthoptera (<span title=""><a href="#b30" id="#b30R" class="bibLink tab-link" data-tab="pane-pcw-references">Flook<i> et al.</i>, 1995a</a></span>) and<i> Heterodoxus macropus</i>, Phthiraptera (<span title=""><a href="#b77" id="#b77R" class="bibLink tab-link" data-tab="pane-pcw-references">Shao<i> et al.</i>, 2001</a></span>)
 had gene order rearrangements relative to the inferred ancestral mt 
gene order, early efforts focused on the use of ‘genome morphology’ (<span title=""><a href="#b28" id="#b28R" class="bibLink tab-link" data-tab="pane-pcw-references">Dowton<i> et al.</i>, 2002</a></span>)
 to resolve deep nodes in insect evolution. Subsequent sequencing effort
 has not supported this notion. Although genome rearrangements have 
shown signal within some orders, e.g. Orthoptera (<span title=""><a href="#b31" id="#b31R" class="bibLink tab-link" data-tab="pane-pcw-references">Flook<i> et al.</i>, 1995b</a></span>), Hymenoptera (<span title=""><a href="#b27%20#b29" id="#b27 #b29R" class="bibLink tab-link" data-tab="pane-pcw-references">Dowton &amp; Austin, 1999; Dowton<i> et al.</i>, 2003</a></span>) and Phthiraptera (<span title=""><a href="#b23" id="#b23R" class="bibLink tab-link" data-tab="pane-pcw-references">Covacin<i> et al.</i>, 2006</a></span>),
 there are insufficient rearrangements in most genomes to resolve 
phylogenies adequately. Simultaneous with this realization has been a 
shift in emphasis to using the mt genome sequence in phylogenetic 
reconstruction.
         </p>
         
         <p>The use of whole mt genome sequences in insect phylogenetics has ranged from resolving strains within the<i> Drosophila melanogaster </i>subgroup (<span title=""><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Ballard, 2000a</a></span>) to phylogenies of all arthropods (e.g. <span title=""><a href="#b65%20#b14" id="#b65 #b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a; Cameron<i> et al.</i>, 2004</a></span>). These phylogenies have produced some remarkable results, such as inferring Phthiraptera + Hymenoptera (<span title=""><a href="#b65" id="#b65R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a</a></span>), the polyphyly of Hexapoda (<span title=""><a href="#b65%20#b3" id="#b65 #b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a; Bae<i> et al.</i>, 2004</a></span>) and Orthoptera + Holometabola to the exclusion of Paraneoptera (<span title=""><a href="#b83" id="#b83R" class="bibLink tab-link" data-tab="pane-pcw-references">Stewart &amp; Beckenbach, 2003</a></span>).
 These results have highlighted the need for rigorous evaluation of the 
phylogenetic behaviour of mt genomes to increase confidence that results
 obtained from this marker reflect evolutionary history. <span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al. </i>(2004)</a></span>
 evaluated the effects of outgroup choice, data transformations (DNA to 
amino acids), restricted gene analyses and optimality criteria on the 
behaviour of mt genome phylogenies of the basal hexapod relationships, 
in particular the relationships of Collembola to Insecta. This study 
demonstrated that, although the relationships of the collembolans were 
very sensitive to variations in all of these factors, there was 
remarkable consistency in the intraordinal relationships of the 
representatives of Diptera, Lepidoptera and Coleoptera. This hinted that
 the mt genome might be particularly valuable in resolving phylogenetic 
relationships within insect orders. We test this proposition with the 
insect order for which the most mt genomes have been sequenced, Diptera.
         </p>
         
         <p>Diptera is the perfect group in which to test the utility of
 mt genomes in insect intraordinal phylogenetic reconstruction for 
several reasons. First, many genomes have already been sequenced, with 
eleven species in four families providing one of the largest 
representations of any invertebrate group in GenBank. Furthermore, these
 representatives are spread over three of the major groups within the 
order: ‘nematocera’,<a class="fNoteLink superscripted" href="#fn1">*</a>‘acalyptrata’
 and Schizophora. The only missing groups represent grades that lie 
between the ‘nematocerans’ and the schizophorans: the ‘basal 
brachycerans’ and ‘aschizans’. Thus sequencing only a few additional 
families provides a relatively complete representation of significant 
groups. By contrast, for most insect orders, only a single mt genome has
 been sequenced, and, for those orders in which multiple representatives
 are available, frequently these are very close relatives. For example, 
Lepidoptera, one of the better studied orders, is represented by five 
species from three genera in two families, but both families are members
 of the highly derived clade, Obtectomera (<span title=""><a href="#b101" id="#b101R" class="bibLink tab-link" data-tab="pane-pcw-references">Kristensen &amp; Skalski, 1999</a></span>).
 Secondly, the deeper level phylogeny of Diptera has received 
considerable attention in recent years to provide a large background of 
phylogenetic data against which mt genome‐derived trees can be compared.
 This background phylogenetic data from a variety of sources, including 
comprehensive morphological and molecular analyses utilizing multiple 
nuclear genes, are highly congruent between the different data sources (<span title=""><a href="#b96%20#b97" id="#b96 #b97R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates &amp; Wiegmann, 1999, 2005</a></span>).
 A range of deep‐level splits in the dipteran tree are well supported as
 monophyletic (e.g. Brachycera, Cyclorrhapha, Schizophora) and can be 
used for comparisons of the utility of mt genome data. A better 
understanding of the utility and limits of mt genome data will, in turn,
 allow the confident application of mt genome data to groups such as 
Coleoptera or Lepidoptera, in which there is much less consensus about 
phylogenetic relationships (<span title=""><a href="#b101%20#b89" id="#b101 #b89R" class="bibLink tab-link" data-tab="pane-pcw-references">Kristensen &amp; Skalski, 1999; Vogler, 2005</a></span>).
         </p>
         
         <p>Here, we present the results of phylogenetic analyses using 
the Diptera mt genome. We have sequenced representatives of three 
additional families, Tabanidae, Nemestrinidae and Syrphidae, which 
belong to three grades of dipteran evolution, ‘basal brachycera’, 
non‐heteroneuran muscomorphs and ‘aschiza’, respectively. Complete mt 
genomes of these taxa, previously unrepresented in GenBank, are pivotal 
to the evaluation of the ability of mt genomes to resolve dipteran 
phylogeny accurately because they bridge the phylogenetic gap between 
early branching ‘nematocerans’ and the more derived schizophorans. We 
evaluate the influence of data transformations and optimality criteria 
on the recovered topologies and nodal support, and estimate the 
timescales over which mt genome data are informative, to gain a better 
understanding of the utility of mt genome sequence phylogenies and the 
types of questions to which they may be confidently applied.</p>
         
      </div>
      <div class="article-section__content" id="ss2">
         
         <h2 class="article-section__title section__title section1" id="ss2-title"> Materials and methods</h2>
         
         <div class="article-section__sub-content" id="ss3">
            
            <h3 class="article-section__sub-title section2" id="ss3-title"> Mitochondrial genome sequencing</h3>
            
            <p>Specimens of the tabanid,<i> Cydistomyia duplonotata </i>(Ricardo), and nemestrinid,<i> Trichophthalma (Lichtwardtiomyia) punctata </i>(Macquart),
 were collected from Benarkin State Forest (near Blackbutt, south‐east 
Queensland, Australia) by S. Cameron, J. Nielsen, M. Rix and G. Svenson,
 on 8 December 2001, and of the syrphid,<i> Simosyrphus grandicornis </i>(Macquart),
 from Cedar Creek (Pine Rivers Shire, south‐east Queensland, Australia) 
by S. Cameron, on 3 October 2001. All specimens were snap frozen in 
liquid nitrogen and stored at −80° C in the insect tissue collection of 
the Department of Integrative Biology, Brigham Young University. Flies 
were identified by C. Lambkin and voucher specimens have been deposited 
in the Australian National Insect Collection, accession numbers: 
ANIC29‐009567 (<i>T. punctata</i>), ANIC29‐009568 (<i>S. grandicornis</i>) and ANIC29‐009569 (<i>C. duplonotata</i>).
            </p>
            
            <p>Whole genomic DNA was extracted from thoracic muscle 
tissue with the DNeasy Tissue Kit (Qiagen, Valencia, CA). Short regions 
of the<i> cox1</i>,<i> cox2</i>,<i> cytB</i>,<i> 12S </i>and<i> 16S </i>genes
 were amplified using general insect primers (Table S1, ‘Supplementary 
material’) and sequenced. These short sequenced regions were used to 
design specific primers which, in combination with general insect 
primers, allowed the amplification of the whole genome of each species 
by long polymerase chain reaction (PCR). The primer sequence and 
location for each long PCR are listed in Table S2 (‘Supplementary 
material’). Within each long PCR product, the full double‐stranded 
sequence was determined by primer walking (primers available from S. 
Cameron on request). Short PCRs were performed using Elongase 
(Invitrogen, Carlsbad, CA) with the following cycling conditions: 95 °C 
for 12 min; forty cycles of 94 °C for 1 min, 40 °C for 1 min and 72 °C 
for 1 min; and a final elongation of 72 °C for 7 min. Long PCRs were 
performed using Elongase with the following cycling conditions: 92 °C 
for 2 min; forty cycles of 92 °C for 30 s, 50 °C for 30 s and 68 °C for 
12 min; and a final run out step of 68 °C for 20 min. Sequencing was 
performed using ABI (Foster City, CA) BigDye version 3 dye terminator 
sequencing technology and run on an ABI 3770 or ABI 3740 capillary 
sequencer. Sequencing PCR conditions were twenty‐eight cycles of 94 °C 
for 10 s, 50 °C for 5 s and 60 °C for 4 min. An ambiguous section of the<i> T. punctata </i>mt genome between<i> nadh4 </i>and<i> nadh6 </i>was resolved by cloning this region using Topo‐TA cloning chemistry (Invitrogen).
            </p>
            
            <p>Raw sequence files were edited and assembled into contigs
 in Sequencher version 4 (GeneCodes Corporation, Ann Arbor, MI). tRNA 
analysis was conducted using tRNAscan‐SE (<span title=""><a href="#b57" id="#b57R" class="bibLink tab-link" data-tab="pane-pcw-references">Lowe &amp; Eddy, 1997</a></span>) employing invertebrate mt predictors and a <span class="smallCaps">cove</span> score cut‐off of unity. Reading frames between tRNAs were found in Sequencher and identified using translated <span class="smallCaps">blast</span> searches (<span class="smallCaps">blastx</span>) (<span title=""><a href="#b2" id="#b2R" class="bibLink tab-link" data-tab="pane-pcw-references">Altschul<i> et al.</i>, 1997</a></span>) as implemented by the NCBI website (<a href="http://www.ncbi.nlm.nih.gov/" class="linkBehavior">http://www.ncbi.nlm.nih.gov/</a>).
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss4">
            
            <h3 class="article-section__sub-title section2" id="ss4-title"> Testing regime</h3>
            
            <p>To test the limits and utility of the mt genome in 
accurate intraordinal reconstructions, we assessed variation in three 
areas. First, the influence of inference methodology was tested by 
performing replicate analyses under maximum parsimony (MP), maximum 
likelihood (ML) and Bayesian (BA) analyses. Consistency across methods 
was assessed by comparison of topological differences between the 
methods and of the strength of nodal support for those nodes in common. 
Secondly, effects of data partitioning were examined. Comparisons of 
topology and nodal support were performed within each of the three 
inference frameworks to test consistency between the three main data 
partitions – protein coding genes (PCG), rRNA genes (RIBO) and tRNA 
genes (TRAN) – and also in combined analyses which included all three 
partitions (ALL). The possible effect of substitution saturation at the 
third codon position of the protein coding genes was assessed by 
comparison of datasets including (DNA123) or excluding (DNA12) third 
codon positions, and of those in which the PCG partition had been 
recoded as amino acids (PROT); these comparisons were performed for both
 the PCG partition alone and the combined ALL partition. The optimal 
partitioning strategy for BA analysis was assessed by comparison of 
Bayes factors (<span title=""><a href="#b67" id="#b67R" class="bibLink tab-link" data-tab="pane-pcw-references">Nylander<i> et al.</i>, 2004</a></span>)
 for runs partitioned by gene, by codon and by codon and gene. Gene 
‘quality’ was assessed by comparison of homoplasy levels across gene 
partitions in MP analyses following our previous methods (<span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2004</a></span>;
 see below). Thirdly, the time range over which mt genomes provided 
reliable phylogenetic signal was assessed by comparison of maximum nodal
 age from fossil evidence and previous dating analyses. By assessing 
which nodes were stable and well supported across all inference and 
partition methods, we were able to calculate upper and lower dates for 
which we were confident that phylogenetic reconstructions were accurate.
 A synopsis of the analyses performed is included in <a href="#t1" class="scrollableLink">Table 1</a>.
            </p>
            
            <div class="article-table-content" id="t1">
               <header class="article-table-caption"><span class="table-caption__label">Table 1.
            </span>Analyses performed in this study.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">Dataset</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Subanalyses</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Parsimony (MP)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Likelihood (ML)</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Bayesian (BA)</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned">ALL‐DNA123</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned"></td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Gene partitions (GP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon partitions (CP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon &amp; gene partitions (CGP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">ALL‐DNA12</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned"></td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Gene partitions (GP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon partitions (CP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon &amp; gene partitions (CGP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">ALL‐PROT</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">PCG‐DNA123</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned"></td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Gene partitions (GP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon partitions (CP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon &amp; gene partitions (CGP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">PCG‐DNA12</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned"></td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Gene partitions (GP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon partitions (CP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Codon &amp; gene partitions (CGP)</td>
                           
                           <td colspan="2" class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">PCG‐PROT</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">RIBO</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">TRAN</td>
                           
                           <td class="left-aligned"></td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                           <td class="left-aligned">Yes</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               
               <div class="article-section__table-footnotes">
                  <ul>
                     <li id="t1_note93">
                        ALL, all gene partitions; DNA123, nucleotide 
sequences, all codon positions; DNA12, nucleotide sequences, first and 
second positions only; PCG, protein coding genes only; PROT, amino acid 
sequences; RIBO, ribosomal RNA genes only; TRAN, transfer RNA genes 
only.
                        
                     </li>
                  </ul>
               </div>
               <div class="article-section__table-source"></div>
            </div>
            
         </div>
         
         <div class="article-section__sub-content" id="ss5">
            
            <h3 class="article-section__sub-title section2" id="ss5-title"> Phylogenetic inference</h3>
            
            <p>All the currently available mt genomes of Holometabola were used (see <a href="#t2" class="scrollableLink">Table 2</a>),<a class="fNoteLink superscripted" href="#fn2">†</a>
 including multiple representatives of the Lepidoptera (five species in 
three families), Coleoptera (three species in three families) and 
Hymenoptera (three species in two families).<i> Tricholepidion </i>(Zygentoma) and<i> Triatoma </i>(Hemiptera) were used as outgroups.
            </p>
            
            <div class="article-table-content" id="t2">
               <header class="article-table-caption"><span class="table-caption__label">Table 2.
            </span>Taxon sampling and availability.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">Species</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Order</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Family</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Accession number</th>
                           
                           <th class="bottom-bordered-cell left-aligned">Availability</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Tricholepidion gertischi</i> 
                           </td>
                           
                           <td class="left-aligned">Zygentoma</td>
                           
                           <td class="left-aligned">Tricholepididae</td>
                           
                           <td class="left-aligned">NC005437</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b65" id="#b65R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al. </i>(2003a)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Triatoma dimidiata</i> 
                           </td>
                           
                           <td class="left-aligned">Hemiptera</td>
                           
                           <td class="left-aligned">Reduvidae</td>
                           
                           <td class="left-aligned">NC002609</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b26" id="#b26R" class="bibLink tab-link" data-tab="pane-pcw-references">Dotson &amp; Beard (2001)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Tribolium castaneum</i> 
                           </td>
                           
                           <td class="left-aligned">Coleoptera</td>
                           
                           <td class="left-aligned">Tenebrionidae</td>
                           
                           <td class="left-aligned">NC003081</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b32" id="#b32R" class="bibLink tab-link" data-tab="pane-pcw-references">Friedrich &amp; Muqim (2003)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Crioceris duodecimpuctata</i> 
                           </td>
                           
                           <td class="left-aligned">Coleoptera</td>
                           
                           <td class="left-aligned">Chrysomelidae</td>
                           
                           <td class="left-aligned">NC003372</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b83" id="#b83R" class="bibLink tab-link" data-tab="pane-pcw-references">Stewart &amp; Beckenbach (2003)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Pyrocoelia rufa</i> 
                           </td>
                           
                           <td class="left-aligned">Coleoptera</td>
                           
                           <td class="left-aligned">Lampyridae</td>
                           
                           <td class="left-aligned">NC003970</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Bae<i> et al. </i>(2004)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Antheraea pernyi</i> 
                           </td>
                           
                           <td class="left-aligned">Lepidoptera</td>
                           
                           <td class="left-aligned">Saturnidae</td>
                           
                           <td class="left-aligned">NC004622</td>
                           
                           <td class="left-aligned">Liu<i> et al.</i>, unpublished
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Bombyx mori</i> 
                           </td>
                           
                           <td class="left-aligned">Lepidoptera</td>
                           
                           <td class="left-aligned">Bombycidae</td>
                           
                           <td class="left-aligned">NC002355</td>
                           
                           <td class="left-aligned">Lee<i> et al.</i>, unpublished
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Bombyx mandarina</i> 
                           </td>
                           
                           <td class="left-aligned">Lepidoptera</td>
                           
                           <td class="left-aligned">Bombycidae</td>
                           
                           <td class="left-aligned">NC003395</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b98" id="#b98R" class="bibLink tab-link" data-tab="pane-pcw-references">Yukuhiro<i> et al. </i>(2002)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Ostrinia furnacalis</i> 
                           </td>
                           
                           <td class="left-aligned">Lepidoptera</td>
                           
                           <td class="left-aligned">Crambidae</td>
                           
                           <td class="left-aligned">NC003368</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b20" id="#b20R" class="bibLink tab-link" data-tab="pane-pcw-references">Coates<i> et al. </i>(2005)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Ostrinia nubialis</i> 
                           </td>
                           
                           <td class="left-aligned">Lepidoptera</td>
                           
                           <td class="left-aligned">Crambidae</td>
                           
                           <td class="left-aligned">NC003367</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b20" id="#b20R" class="bibLink tab-link" data-tab="pane-pcw-references">Coates<i> et al. </i>2005</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Anopheles quadrimaculatus</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Culicidae</td>
                           
                           <td class="left-aligned">NC000875</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b61" id="#b61R" class="bibLink tab-link" data-tab="pane-pcw-references">Mitchell<i> et al. </i>(1993)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Anopheles gambiae</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Culicidae</td>
                           
                           <td class="left-aligned">NC002084</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Beard<i> et al. </i>(1993)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Cydistomyia duplonotata</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Tabanidae</td>
                           
                           <td class="left-aligned">DQ866052</td>
                           
                           <td class="left-aligned">Present study</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Trichophthalma punctata</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Nemestrinidae</td>
                           
                           <td class="left-aligned">DQ866051</td>
                           
                           <td class="left-aligned">Present study</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Simosyrphus grandicornis</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Syrphidae</td>
                           
                           <td class="left-aligned">DQ866050</td>
                           
                           <td class="left-aligned">Present study</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Drosophila yakuba</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Drosophilidae</td>
                           
                           <td class="left-aligned">NC001322</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b18" id="#b18R" class="bibLink tab-link" data-tab="pane-pcw-references">Clary &amp; Woolstenholme (1985)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Drosophila melanogaster</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Drosophilidae</td>
                           
                           <td class="left-aligned">NC001709</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b54" id="#b54R" class="bibLink tab-link" data-tab="pane-pcw-references">Lewis<i> et al. </i>(1995)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Drosophila simulans</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Drosophilidae</td>
                           
                           <td class="left-aligned">NC005781</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Ballard (2000a)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Drosophila sechellia</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Drosophilidae</td>
                           
                           <td class="left-aligned">NC005780</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Ballard (2000a)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Drosophila mauritiania</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Drosophilidae</td>
                           
                           <td class="left-aligned">NC005719</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b6" id="#b6R" class="bibLink tab-link" data-tab="pane-pcw-references">Ballard (2000b)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Chrysomya putoria</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Calliphoridae</td>
                           
                           <td class="left-aligned">NC002697</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b45" id="#b45R" class="bibLink tab-link" data-tab="pane-pcw-references">Junqueira<i> et al. </i>(2004)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Cochliomyia hominovorax</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Calliphoridae</td>
                           
                           <td class="left-aligned">NC002660</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b52" id="#b52R" class="bibLink tab-link" data-tab="pane-pcw-references">Lessinger<i> et al. </i>(2000)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Ceratitis capitata</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Tephritidae</td>
                           
                           <td class="left-aligned">NC000857</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b82" id="#b82R" class="bibLink tab-link" data-tab="pane-pcw-references">Spanos<i> et al. </i>(2000)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Bactrocera oleae</i> 
                           </td>
                           
                           <td class="left-aligned">Diptera</td>
                           
                           <td class="left-aligned">Tephritidae</td>
                           
                           <td class="left-aligned">NC005333</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b66" id="#b66R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al. </i>(2003b)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Perga condei</i> 
                           </td>
                           
                           <td class="left-aligned">Hymenoptera</td>
                           
                           <td class="left-aligned">Pergidae</td>
                           
                           <td class="left-aligned">AY787816</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b16" id="#b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Castro &amp; Dowton (2005)</a></span> 
                           </td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Melipona bicolor</i> 
                           </td>
                           
                           <td class="left-aligned">Hymenoptera</td>
                           
                           <td class="left-aligned">Apidae</td>
                           
                           <td class="left-aligned">NC004529</td>
                           
                           <td class="left-aligned">Silvestre &amp; Arias, unpublished</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>Apis mellifera</i> 
                           </td>
                           
                           <td class="left-aligned">Hymenoptera</td>
                           
                           <td class="left-aligned">Apidae</td>
                           
                           <td class="left-aligned">NC001566</td>
                           
                           <td class="left-aligned"> <span title=""><a href="#b24" id="#b24R" class="bibLink tab-link" data-tab="pane-pcw-references">Crozier &amp; Crozier (1993)</a></span> 
                           </td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            
            <p>An amino acid alignment was generated in C<span class="smallCaps">lustal</span>W (<span title=""><a href="#b87" id="#b87R" class="bibLink tab-link" data-tab="pane-pcw-references">Thompson<i> et al.</i>, 1994</a></span>; implemented in <span class="smallCaps">mega</span>3; <span title=""><a href="#b49" id="#b49R" class="bibLink tab-link" data-tab="pane-pcw-references">Kumar<i> et al.</i>, 2004</a></span>) for each of the thirteen protein coding genes, and a DNA alignment was inferred from the amino acid alignment using <span class="smallCaps">mega</span>3 (<span title=""><a href="#b49" id="#b49R" class="bibLink tab-link" data-tab="pane-pcw-references">Kumar<i> et al.</i>, 2004</a></span>),
 which can translate between DNA and amino acid sequences within 
alignments. Amino acid alignments were made with the following 
parameters: pairwise alignment gap opening penalty = 10 and extension 
penalty = 0.1; multiple alignment gap opening penalty = 10 and extension
 penalty = 0.2; protein weight matrix = Gonnet; residue specific 
penalties: on; hydrophilic penalties: on; gap separation distance = 4; 
end gap separation: off; negative matrix = off; and delay divergent 
cut‐off = 30%. tRNA and rRNA genes were aligned in <span class="smallCaps">mega</span>3 using C<span class="smallCaps">lustal</span>W,
 and corrected by eye to account for their secondary structures using 
the following parameters: gap opening cost = 15; gap extension cost = 
6.66; DNA weight matrix = IUB; transition weight cost = 0.5; negative 
matrix = off; and delay divergent cut‐off = 30%. Alignments of 
individual genes were then concatenated in M<span class="smallCaps">ac</span>C<span class="smallCaps">lade</span> 4.06 (<span title=""><a href="#b59" id="#b59R" class="bibLink tab-link" data-tab="pane-pcw-references">Maddison &amp; Maddison, 2003</a></span>),
 and data partitions were delimited on the basis of each included gene 
and for each codon position. Each of the rRNA genes was included as a 
separate partition, but the tRNA genes were joined in a single partition
 for M<span class="smallCaps">odel</span>T<span class="smallCaps">est</span> and BA, as the number of variable sites within individual tRNAs was too few for accurate parameter calculations.
            </p>
            
            <p>Phylogenetic analysis was performed with <span class="smallCaps">paup</span> 4.0b10 (<span title=""><a href="#b84" id="#b84R" class="bibLink tab-link" data-tab="pane-pcw-references">Swofford, 2002</a></span>) for MP and ML analyses and with M<span class="smallCaps">r</span>B<span class="smallCaps">ayes</span> (BA) versions 3.0b4 and 3.1.1 (<span title=""><a href="#b43" id="#b43R" class="bibLink tab-link" data-tab="pane-pcw-references">Huelsenbeck &amp; Ronquist, 2001</a></span>) for BA analysis. Bootstrap support was calculated with <span class="smallCaps">paup</span> 4.0b10 with either 1000 (MP) or 100 (ML) replicates. Tree statistics were calculated in <span class="smallCaps">paup</span> 4.0b10. The relative contribution of each dataset partition to the combined topology was calculated using partitioned Bremer (<span title=""><a href="#b4" id="#b4R" class="bibLink tab-link" data-tab="pane-pcw-references">Baker &amp; DeSalle, 1997</a></span>), as implemented in T<span class="smallCaps">ree</span>R<span class="smallCaps">ot</span> version 2 (<span title=""><a href="#b81" id="#b81R" class="bibLink tab-link" data-tab="pane-pcw-references">Sorenson, 1999</a></span>).
 All BA analyses were run with unlinked partitions, appropriate models 
of molecular evolution selected for each partition and analyses run 
using four chains, for three million generations with sampling every 
1000 generations. Completed BA analyses were examined for asymptotic 
behaviour of each parameter and for total tree likelihood; trees 
collected prior to this asymptotic point were treated as burn‐in and 
discarded (generally the first 30 000–60 000 generations). Models for BA
 and ML were chosen using AIC as implemented in M<span class="smallCaps">odel</span>T<span class="smallCaps">est</span> (<span title=""><a href="#b70" id="#b70R" class="bibLink tab-link" data-tab="pane-pcw-references">Posada &amp; Crandall, 1998</a></span>),
 run independently for each partition used in BA and for combined 
partitions in ML analyses. ML and BA run files are available for all 
analyses on request.
            </p>
            
            <p>The distribution of homoplasy amongst the genes was estimated by gene tree/total tree comparisons (<span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2004</a></span>).
 The proportional size of each gene (length of the aligned gene 
partition divided by the total alignment length) was compared with its 
proportional contribution to tree length (tree length for that gene 
partition determined by T<span class="smallCaps">ree</span>R<span class="smallCaps">ot</span> divided by the total tree length for the combined analysis). Thus, the formulae is nl<sub>g</sub>/nl<sub>t</sub> : tl<sub>g</sub>/tl<sub>t</sub>, where nl<sub>g</sub> is the aligned length of a particular gene, nl<sub>t</sub> is the nucleotide length of the total alignment, tl<sub>g</sub> is the partitioned tree length for that gene and tl<sub>t</sub>
 is the total tree length. Replicate analyses were performed using total
 gene length or informative sites alone. If homoplasy is distributed 
randomly within the dataset, these proportions should be the same, 
whereas, if homoplasy is clustered in particular genes, these should 
contribute extra tree length to the combined analysis disproportionate 
to the length of the gene. In this way, the relative contributions of 
different data partitions can be compared directly.
            </p>
            
         </div>
         
      </div>
      <div class="article-section__content" id="ss6">
         
         <h2 class="article-section__title section__title section1" id="ss6-title"> Results</h2>
         
         <div class="article-section__sub-content" id="ss7">
            
            <h3 class="article-section__sub-title section2" id="ss7-title"> Genome sequences</h3>
            
            <p>The entire mt genomes of<i> Cydistomyia duplonotata </i>(tabanid),<i> Trichophthalma (Lichtwardtiomyia) punctata </i>(nemestrinid) and<i> Simosyrphus grandicornis </i>(syrphid)
 were sequenced by primer walking of multiple overlapping long PCR 
fragments. These genomes have been deposited in GenBank with the 
following accession numbers:<i> C. duplonotata </i>DQ866052,<i> T. punctata </i>DQ866051 and<i> S. grandicornis </i>DQ866050.
 The genomes are 16 247, 16 396 and 16 141 base pairs long, 
respectively. Each genome has the usual metazoan complement of thirteen 
protein coding genes, two rRNA genes and twenty‐two tRNA genes. Neither 
of the genome rearrangements previously found in Diptera, tRNAs RANSEF 
found in Culicidae (<span title=""><a href="#b8%20#b61" id="#b8 #b61R" class="bibLink tab-link" data-tab="pane-pcw-references">Beard<i> et al.</i>, 1993; Mitchell<i> et al.</i>, 1993</a></span>) or duplication of tRNAs I and Q found in Calliphoridae (<span title=""><a href="#b53" id="#b53R" class="bibLink tab-link" data-tab="pane-pcw-references">Lessinger<i> et al.</i>, 2004</a></span>) were present. Each of the three genomes has the plesiomorphic pancrustacean mt genome arrangement (<span title=""><a href="#b11" id="#b11R" class="bibLink tab-link" data-tab="pane-pcw-references">Boore, 1999</a></span>). <span class="smallCaps">cove</span> analysis failed to find the Arg tRNA for any of the three species and the Ser (UCN) tRNA in<i> Cydistomyia</i>;
 these genes were identified by aligning the region expected to be 
occupied by these tRNAs against homologous regions from other insects. 
In addition, <span class="smallCaps">cove</span> analysis misidentified 
the tRNA isoform of Ser (UCN) as Phe as a result of misplacement of the 
anticodon loop; this was identified properly by alignments of this tRNA 
amongst Diptera and other insects. Additional tRNAs were detected in the
 AT‐rich regions (= putative control region, origin of replication) of 
each species. However, as there appeared to be no evolutionary 
conservation of these regions across the taxa and the <span class="smallCaps">cove</span>
 scores were very low, it is probable that these are spurious tRNAs: 
regions of random sequence capable of folding into clover‐leaf 
structures similar to real tRNAs. Despite fairly large AT‐rich regions 
of 1376, 1597 and 1127 bp, respectively, large repetitive sequences were
 absent. Each of the thirteen protein coding genes had the regular start
 (I or M) and stop (TAG, TAA, TA or T) codons, except for<i> cox1 </i>in
 the tabanid and nemestrinid, which did not have an available start 
codon. The start codons were identified by comparison of alignments with
 other dipterans to be R and Q, respectively. Start codons for<i> cox1 </i>are frequently non‐standard in holometabolans, and so this finding is not particularly unusual (<span title=""><a href="#b52%20#b45" id="#b52 #b45R" class="bibLink tab-link" data-tab="pane-pcw-references">Lessinger<i> et al.</i>, 2000; Junqueira<i> et al.</i>, 2004</a></span>). Drosophilids utilize tetranucleotide start codons for<i> cox1 </i>(ATAA = S) and other dipteran groups appear to simply use the first in‐frame codon which is usually an R, S, T or Q.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss8">
            
            <h3 class="article-section__sub-title section2" id="ss8-title"> Combined Dipteran mitochondrial genome phylogeny</h3>
            
            <p>We performed thirty separate phylogenetic analyses to 
test the effects of the optimality criterion, data partitioning 
strategies and data coding approaches on mt genome phylogenies of 
Diptera. Tree statistics are presented in <a href="#t3" class="scrollableLink">Table 3</a>. The results of analysis of the ALL‐DNA12 dataset under the three optimality criteria are presented in <a href="#f1%20#f2%20#f3" class="scrollableLink">Figs 1–3</a>.
 As the majority of analyses recovered results which were largely 
congruent with the trees recovered using the maximal amount of genome 
sequence data, the results of other analyses are not presented. Tables 
S3–S6 (‘Supplementary material’) depict which nodes are congruent and 
the level of nodal support. A full complement of tree data (alignments 
plus tree figures) is available from the Whiting laboratory website (<a href="http://whitinglab.byu.edu/" class="linkBehavior">http://whitinglab.byu.edu/</a>).
 There is remarkable congruence between datasets and across optimality 
criteria, with most of the intraordinal nodes recovered from a majority 
of analyses. Holometabola is monophyletic in all except some PCG 
datasets. Interordinal relationships vary widely, with all possible 
combinations of the four holometabolous orders recovered in at least 
some analyses. The orders generally are monophyletic, except in the RIBO
 analyses, in which Coleoptera becomes a paraphyletic grade at the base 
of Holometabola and<i> Melipona </i>generally groups with the 
tephritids, rendering most dipteran clades non‐monophyletic. Diptera is 
monophyletic in all except two analyses: RIBO‐ML and RIBO‐BA. These 
relationships found in the RIBO datasets are probably a result of 
alignment artefacts caused by the large indels in these genes. 
Intraordinal relationships within Diptera are also broadly congruent 
across analyses. The major dipteran clades, Brachycera, Cyclorrhapha and
 Schizophora, are recovered in almost all analyses, and intrafamilial 
and congeneric relationships are universally recovered. Furthermore, the
 overall topology is broadly congruent with current consensus 
phylogenies of the Diptera (<span title=""><a href="#b92%20#b97" id="#b92 #b97R" class="bibLink tab-link" data-tab="pane-pcw-references">Wiegmann<i> et al.</i>, 2003; Yeates &amp; Wiegmann, 2005</a></span>).
 The topology differs in two areas, the ‘basal brachycera’ and the 
‘acalyptrates’. Our trees recover monophyly for the ‘basal 
brachycerans’, whereas the Tabanidae and Nemestrinidae are placed more 
usually in infraorders Tabanomorpha and Muscomorpha, respectively, with 
the nemestrinids as the earliest branch within Muscomorpha (<span title=""><a href="#b100%20#b92" id="#b100 #b92R" class="bibLink tab-link" data-tab="pane-pcw-references">Wiegmann<i> et al.</i>, 2000, 2003</a></span>; <span title=""><a href="#b95" id="#b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates, 2002</a></span>). By contrast, ‘acalyptrate’ monophyly is controversial, asserted by many authorities but lacking morphological support (<span title=""><a href="#b92" id="#b92R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates &amp; Wiegmann, 2005</a></span>).
            </p>
            
            <div class="article-table-content" id="t3">
               <header class="article-table-caption"><span class="table-caption__label">Table 3.
            </span>Tree statistics.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th rowspan="2" class="bottom-bordered-cell left-aligned">Dataset</th>
                           
                           <th colspan="5" class="bottom-bordered-cell center-aligned">Parsimony</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Likelihood</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Bayesian<a href="#t3n1" class="scrollableLink"><sup><i>a</i></sup></a></th>
                           
                        </tr>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">No. sites</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Length</th>
                           
                           <th class="bottom-bordered-cell center-aligned">CI</th>
                           
                           <th class="bottom-bordered-cell center-aligned">RI</th>
                           
                           <th class="bottom-bordered-cell center-aligned">RC</th>
                           
                           <th class="bottom-bordered-cell center-aligned">−ln L</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Log. likelihood</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned">ALL‐DNA123</td>
                           
                           <td class="center-aligned">15803</td>
                           
                           <td class="center-aligned">46461</td>
                           
                           <td class="center-aligned">0.42</td>
                           
                           <td class="center-aligned">0.49</td>
                           
                           <td class="center-aligned">0.21</td>
                           
                           <td class="center-aligned">197367.4</td>
                           
                           <td class="center-aligned">189742.8</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">ALL‐DNA12</td>
                           
                           <td class="center-aligned">11888</td>
                           
                           <td class="center-aligned">26854</td>
                           
                           <td class="center-aligned">0.48</td>
                           
                           <td class="center-aligned">0.56</td>
                           
                           <td class="center-aligned">0.27</td>
                           
                           <td class="center-aligned">122448.3</td>
                           
                           <td class="center-aligned">119137.0</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">ALL‐PROT</td>
                           
                           <td class="center-aligned">7956</td>
                           
                           <td class="center-aligned">24338</td>
                           
                           <td class="center-aligned">0.58</td>
                           
                           <td class="center-aligned">0.58</td>
                           
                           <td class="center-aligned">0.34</td>
                           
                           <td class="center-aligned">n/a</td>
                           
                           <td class="center-aligned">45706.6</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">PCG‐DNA123</td>
                           
                           <td class="center-aligned">11745</td>
                           
                           <td class="center-aligned">36879</td>
                           
                           <td class="center-aligned">0.41</td>
                           
                           <td class="center-aligned">0.48</td>
                           
                           <td class="center-aligned">0.19</td>
                           
                           <td class="center-aligned">154836.1</td>
                           
                           <td class="center-aligned">147642.3</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">PCG‐DNA12</td>
                           
                           <td class="center-aligned">7830</td>
                           
                           <td class="center-aligned">17270</td>
                           
                           <td class="center-aligned">0.48</td>
                           
                           <td class="center-aligned">0.56</td>
                           
                           <td class="center-aligned">0.27</td>
                           
                           <td class="center-aligned">79920.4</td>
                           
                           <td class="center-aligned">77579.7</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">PCG‐PROT</td>
                           
                           <td class="center-aligned">3898</td>
                           
                           <td class="center-aligned">14708</td>
                           
                           <td class="center-aligned">0.64</td>
                           
                           <td class="center-aligned">0.62</td>
                           
                           <td class="center-aligned">0.39</td>
                           
                           <td class="center-aligned">n/a</td>
                           
                           <td class="center-aligned">84781.4</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">RIBO</td>
                           
                           <td class="center-aligned">2391</td>
                           
                           <td class="center-aligned">6338</td>
                           
                           <td class="center-aligned">0.48</td>
                           
                           <td class="center-aligned">0.54</td>
                           
                           <td class="center-aligned">0.26</td>
                           
                           <td class="center-aligned">25736.7</td>
                           
                           <td class="center-aligned">25814.2</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">TRAN</td>
                           
                           <td class="center-aligned">1667</td>
                           
                           <td class="center-aligned">3161</td>
                           
                           <td class="center-aligned">0.51</td>
                           
                           <td class="center-aligned">0.58</td>
                           
                           <td class="center-aligned">0.30</td>
                           
                           <td class="center-aligned">14941.6</td>
                           
                           <td class="center-aligned">14992.0</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               
               <div class="article-section__table-footnotes">
                  <ul>
                     <li id="t3_note75">
                        ALL, all gene partitions; DNA123, nucleotide 
sequences, all codon positions; DNA12, nucleotide sequences, first and 
second positions only; PCG, protein coding genes only; PROT, amino acid 
sequences; RIBO, ribosomal RNA genes only; TRAN, transfer RNA genes 
only; CI, consistency index; RI, retention index; RC, rescaled 
consistency index.
                        
                     </li>
                     <li id="t3n1">
                        
                        <sup>
                           <i>a</i>
                           </sup>For the ALL and PCG datasets, the partition by codon and gene datasets is given as these are the most likely.
                        
                     </li>
                  </ul>
               </div>
               <div class="article-section__table-source"></div>
            </div>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f1"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/aa15aae7-bd42-4ec3-b09a-80c541858057/syen_355_f1.gif"><img class="figure__image" src="syen_355_f1.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f1&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>Parsimony cladogram inferred from the 
ALL‐DNA12 dataset. Support is indicated above each node: the top number 
is the bootstrap percentage, the bottom number the non‐partitioned 
Bremer support. Numbers below each node (circled) are common across 
datasets; partitioned Bremer supports for each gene are given in <a href="#t4" class="scrollableLink">Table S3</a>;
 support levels across different datasets are given in Table 7. CO, 
Coleoptera; DI, Diptera; HY, Hymenoptera; LE, Lepidoptera; ns, not 
supported.</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f2"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/24792058-730a-4f37-86ee-dba11ef20a41/syen_355_f2.gif"><img class="figure__image" src="syen_355_f2.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 2</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f2&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>Likelihood phenogram inferred from the 
ALL‐DNA12 dataset. Bootstrap percentages are indicated above each node. 
Numbers below each node (circled) are common across datasets; support 
levels across different datasets are given in Table S4. CO, Coleoptera; 
DI, Diptera; HY, Hymenoptera; LE, Lepidoptera.</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f3"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/ac6ebe4e-69d6-49d6-8f71-7915cf2a0c6c/syen_355_f3.gif"><img class="figure__image" src="syen_355_f3.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 3</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f3&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>Bayesian phenogram inferred from the 
ALL‐DNA12 dataset. Posterior probabilities are indicated above each 
node. Numbers below each node (circled) are common across datasets; 
support levels across different datasets are given in Tables S5 and S6. 
CO, Coleoptera; DI, Diptera; HY, Hymenoptera; LE, Lepidoptera.</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
         </div>
         
         <div class="article-section__sub-content" id="ss9">
            
            <h3 class="article-section__sub-title section2" id="ss9-title"> Optimality criteria</h3>
            
            <p>The major effect of the optimality criteria is seen at 
the interordinal level. MP divides Holometabola into two groups: 
Coleoptera and an unresolved trichotomy of Lepidoptera, equivalent for 
the included taxa to support for a monophyletic Antliophora. ML splits 
Holometabola into two groups: Coleoptera + Hymenoptera and Lepidoptera +
 Diptera. BA splits Diptera from the remaining Holometabola. It should 
be noted, however, that interordinal relationships vary much more 
between datasets than between criteria, and dataset is probably the more
 significant determining variable (see below). By contrast, optimality 
criteria have almost no influence within orders, and particularly the 
intraordinal relationships within Diptera, which are almost constant 
across the optimality frameworks for comparable datasets (schizophoran 
paraphyly in the ML‐DNA123 dataset is the only exception). With regard 
to nodal support, across datasets there was a predictable increase in 
support levels comparing results from MP to ML to BA. As previously 
noted by many workers (e.g. <span title=""><a href="#b51%20#b91%20#b14" id="#b51 #b91 #b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Leaché &amp; Reeder, 2002; Whittingham<i> et al.</i>, 2002; Cameron<i> et al.</i>, 2004</a></span>),
 BA posterior probabilities are inflated relative to both ML and MP 
bootstraps. Indeed, several datasets recovered topologies for which 
every node had a posterior probability of 1.0. Differences between the 
ML and MP bootstrapping can be related to the greater sensitivity of the
 latter to homoplasy. mt datasets, particularly nucleotide analyses 
which include the third codon position of protein coding genes, have 
fairly high levels of homoplasy. Given the common topology within 
Diptera, the differences in nodal support between the three optimality 
criteria probably can be interpreted as upper (BA), middle (ML) and 
lower (MP) confidence bounds to the inferred relationships.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss10">
            
            <h3 class="article-section__sub-title section2" id="ss10-title"> Effects of data partitioning</h3>
            
            <p>Data partitioning strategies had much larger effects than
 optimality criteria on both topology and support. Compared with the ALL
 analyses, the three datasets utilizing reduced numbers of genes (PCG, 
RIBO, TRAN) were less likely to recover the same topology and, for those
 nodes which were recovered in common, nodal support was usually 
reduced. The ALL datasets were a third (DNA123) to twice (PROT) as large
 as the PCG datasets (<a href="#t3" class="scrollableLink">Table 3</a>),
 and the magnitudes of the differences between the ALL and PCG analyses 
were related to this relative increase in data, i.e. ALL‐PROT differed 
from PCG‐PROT more than ALL‐DNA123 did from PCG‐DNA123 in both topology 
and nodal support (Tables S3–S6; ‘Supplementary material’). In line with
 the dogma of total evidence (<span title=""><a href="#b47" id="#b47R" class="bibLink tab-link" data-tab="pane-pcw-references">Kluge, 1989</a></span>),
 more data appear to yield more robust phylogenetic estimates. The 
surprising exception is in the comparison of the RIBO and TRAN datasets.
 The RIBO dataset is almost 50% larger than the TRAN dataset (2391 bp 
vs. 1667 bp), and yet the TRAN dataset consistently outperforms RIBO. 
The RIBO topologies are the worst of the analyses performed here. 
Coleoptera is a paraphyletic grade at the base of Holometabola.<i> Pyrocoelia </i>and<i> Melipona </i>have
 wildly incorrect positions in the tree branching within Lepidoptera and
 Diptera, respectively, and both positions actually receive reasonable 
nodal support. These problematic topologies may be a function of the 
unsophisticated alignment procedure used (C<span class="smallCaps">lustal</span>), which is known to struggle with ribosomal sequence (e.g. <span title=""><a href="#b99" id="#b99R" class="bibLink tab-link" data-tab="pane-pcw-references">Terry &amp; Whiting, 2005</a></span>),
 or long‐branchlike effects caused by base composition and rate 
heterogeneities within insect mt rRNA genes, or both. By contrast, the 
TRAN datasets, with less data, produced much more reasonable 
phylogenies, particularly for deep nodes. The orders were monophyletic 
and, with the exception of Hymenoptera, received significant nodal 
support in all analyses. The highly conserved structural qualities of 
the tRNAs, plus consistent annotational methodology by mt genomics 
researchers, ensuring that truly homologous regions are compared, have 
probably contributed to the phylogenetic stability of these genes. 
Conflict of the TRAN datasets lies within Diptera where the topology 
differs considerably from both the ALL datasets and previous 
expectation. The position of the syrphid is variable, either as first 
branch within Diptera (MP) or as sister to<i> Anopheles </i>(ML, BA). 
The positions of the tabanid and nemestrinid are also variable, a 
paraphyletic grade at the base of Schizophora (MP) or a monophyletic 
sister group to<i> Anopheles </i>+ syrphid. Relationships between the 
three Schizophoran families also vary, with Tephritidae + Calliphoridae 
rather than Drosophilidae + Calliphoridae as in ALL and PCG analyses. It
 is possible that the tRNA genes are simply not variable enough to 
accurately capture mid‐level dipteran relationships on their own, with 
stem sites supporting deep‐level interordinal relationships and loop 
sites supporting shallow relationships. Within Drosophilidae, the TRAN 
datasets are in perfect concordance with accepted relationships within 
the drosophilids (<span title=""><a href="#b74%20#b68%20#b55%20#b85" id="#b74 #b68 #b55 #b85R" class="bibLink tab-link" data-tab="pane-pcw-references">Russo<i> et al.</i>, 1995; O’Grady &amp; Kidwell, 2002; Lewis<i> et al.</i>, 2005; Symons &amp; Wertheim, 2005</a></span>) and with our larger PCG and ALL datasets.
            </p>
            
            <p>Comparisons of how individual genes or sets of genes 
perform in isolated analyses, such as those described, fail to account 
for hidden support (<i>sensu </i><span title=""><a href="#b33" id="#b33R" class="bibLink tab-link" data-tab="pane-pcw-references">Gatesy<i> et al.</i>, 1999</a></span>),
 whereby genes which, in isolation, support different topologies from 
those of combined analyses nonetheless actually support the combined 
analysis topology. The favoured method for assessing hidden support is 
comparison of partitioned Bremer support (= decay indices) across the 
combined topology. Partitioned Bremer values for each gene are given in <a href="#t4" class="scrollableLink">Table 4</a>.
 Across the genes, we see a typical mosaic of positive and negative 
values without a clear trend of particular genes driving the analysis 
or, conversely, consistently conflicting with the combined analysis. The
 one exception is the<i> l‐rrna </i>gene, which is negative for all but the most derived relationships. This is not reflected in the<i> s‐rrna </i>gene, which is positive for all but a single node (Tabanidae + Nemestrinidae), making it, together with<i> cox2</i>,
 the best predictor of combined topology in the mt genome. This suggests
 that the discrepancies between the RIBO and ALL datasets are driven 
either by the large ribosomal subunit, or support in the small subunit 
for the combined topology is very well hidden indeed. The tRNA genes are
 highly concordant with the combined topology, as is expected given the 
similarity in recovered topologies between the ALL and TRAN datasets. 
The only significantly negative value is for the monophyly of 
Cyclorrhapha as a result of the divergent tRNAs in the syrphid, which 
groups with strong support with<i> Anopheles</i>. It is also notable 
that the tRNA dataset frequently provides some of the strongest support 
for ordinal monophyly, for Holometabola and for other deep nodes, 
suggesting that it may be useful in resolving interordinal relationships
 within Insecta.
            </p>
            
            <div class="article-table-content" id="t4">
               <header class="article-table-caption"><span class="table-caption__label">Table 4.
            </span>Partitioned Bremer supports for the ALL‐DNA12 dataset. Numbers and letters refer to the nodes labelled in <a href="#f1" class="scrollableLink">Fig. 1</a>.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">Node</th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>atp6</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>atp8</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>cox1</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>cox2</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>cox3</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>cytB</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>nadh1</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>nadh2</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>nadh3</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell left-aligned"> <i>nadh4</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell left-aligned"> <i>nadh4L</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>nadh5</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>nadh6</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>s‐rrna</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned"> <i>l‐rrna</i> 
                           </th>
                           
                           <th class="bottom-bordered-cell center-aligned">tRNAs</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned"> 1. Holometabola</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">5.5</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">3.5</td>
                           
                           <td class="center-aligned">−0.5</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">−5</td>
                           
                           <td class="center-aligned">−4</td>
                           
                           <td class="left-aligned">12.5</td>
                           
                           <td class="left-aligned">1.5</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">2.5</td>
                           
                           <td class="center-aligned">48.5</td>
                           
                           <td class="center-aligned">29</td>
                           
                           <td class="center-aligned">41.5</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 2. Coleoptera</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">−8</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">−14</td>
                           
                           <td class="center-aligned">−13</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">9</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="left-aligned">12</td>
                           
                           <td class="left-aligned">−3</td>
                           
                           <td class="center-aligned">35</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">21</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">3</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 3.<i> Tribolium </i>+<i> Crioceris</i></td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="left-aligned">6</td>
                           
                           <td class="left-aligned">−1</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">12</td>
                           
                           <td class="center-aligned">−15</td>
                           
                           <td class="center-aligned">5</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Diptera + (Lepidoptera + Hymenoptera)</td>
                           
                           <td class="center-aligned">−4</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="left-aligned">0</td>
                           
                           <td class="left-aligned">−5</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">−8</td>
                           
                           <td class="center-aligned">28</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">−13</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Lepidoptera + Hymenoptera</td>
                           
                           <td class="center-aligned">−4</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="left-aligned">0</td>
                           
                           <td class="left-aligned">−5</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">−8</td>
                           
                           <td class="center-aligned">28</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">−13</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 4. Lepidoptera</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">12</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">15</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="left-aligned">15</td>
                           
                           <td class="left-aligned">7</td>
                           
                           <td class="center-aligned">42</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">27</td>
                           
                           <td class="center-aligned">50</td>
                           
                           <td class="center-aligned">54</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 5.<i> Antheraea </i>+<i> Ostrinia</i></td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">−3.5</td>
                           
                           <td class="center-aligned">−3</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">3.5</td>
                           
                           <td class="center-aligned">−9</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="left-aligned">−3.5</td>
                           
                           <td class="left-aligned">−0.5</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">21</td>
                           
                           <td class="center-aligned">−5.5</td>
                           
                           <td class="center-aligned">0.5</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 6.<i> Bombyx</i></td>
                           
                           <td class="center-aligned">13</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">14</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">20</td>
                           
                           <td class="center-aligned">20</td>
                           
                           <td class="center-aligned">22</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="left-aligned">11</td>
                           
                           <td class="left-aligned">7</td>
                           
                           <td class="center-aligned">28</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">33</td>
                           
                           <td class="center-aligned">32</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 7.<i> Ostrinia</i></td>
                           
                           <td class="center-aligned">15</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">9</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">11</td>
                           
                           <td class="center-aligned">11</td>
                           
                           <td class="center-aligned">22</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="left-aligned">40</td>
                           
                           <td class="left-aligned">10</td>
                           
                           <td class="center-aligned">39</td>
                           
                           <td class="center-aligned">22</td>
                           
                           <td class="center-aligned">12</td>
                           
                           <td class="center-aligned">45</td>
                           
                           <td class="center-aligned">37</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 8. Hymenoptera</td>
                           
                           <td class="center-aligned">15</td>
                           
                           <td class="center-aligned">9</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">−15</td>
                           
                           <td class="center-aligned">14</td>
                           
                           <td class="center-aligned">9</td>
                           
                           <td class="center-aligned">−5</td>
                           
                           <td class="left-aligned">18</td>
                           
                           <td class="left-aligned">9</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">−5</td>
                           
                           <td class="center-aligned">27</td>
                           
                           <td class="center-aligned">−26</td>
                           
                           <td class="center-aligned">14</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> 9. Apidae</td>
                           
                           <td class="center-aligned">40</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">63</td>
                           
                           <td class="center-aligned">23</td>
                           
                           <td class="center-aligned">51</td>
                           
                           <td class="center-aligned">42</td>
                           
                           <td class="center-aligned">70</td>
                           
                           <td class="center-aligned">−8</td>
                           
                           <td class="center-aligned">26</td>
                           
                           <td class="left-aligned">79</td>
                           
                           <td class="left-aligned">14</td>
                           
                           <td class="center-aligned">121</td>
                           
                           <td class="center-aligned">20</td>
                           
                           <td class="center-aligned">37</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">52</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">10. Diptera</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="left-aligned">8</td>
                           
                           <td class="left-aligned">−4</td>
                           
                           <td class="center-aligned">23</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">30</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">4</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">11. ‘nematocera’ (=<i> Anopheles</i>)
                           </td>
                           
                           <td class="center-aligned">11</td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">13</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">12</td>
                           
                           <td class="center-aligned">40</td>
                           
                           <td class="center-aligned">16</td>
                           
                           <td class="center-aligned">23</td>
                           
                           <td class="center-aligned">12</td>
                           
                           <td class="left-aligned">24</td>
                           
                           <td class="left-aligned">12</td>
                           
                           <td class="center-aligned">63</td>
                           
                           <td class="center-aligned">19</td>
                           
                           <td class="center-aligned">59</td>
                           
                           <td class="center-aligned">65</td>
                           
                           <td class="center-aligned">54</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">12. Brachycera</td>
                           
                           <td class="center-aligned">−8</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">−8</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="left-aligned">1</td>
                           
                           <td class="left-aligned">10</td>
                           
                           <td class="center-aligned">24</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">17</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">−2</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">13. Tabanidae + Nemestrinidae</td>
                           
                           <td class="center-aligned">−3</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">−5</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="left-aligned">−1</td>
                           
                           <td class="left-aligned">−1</td>
                           
                           <td class="center-aligned">16</td>
                           
                           <td class="center-aligned">−7</td>
                           
                           <td class="center-aligned">20</td>
                           
                           <td class="center-aligned">24</td>
                           
                           <td class="center-aligned">−6</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">14. Cyclorrhapha</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">−2</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="left-aligned">−4</td>
                           
                           <td class="left-aligned">1</td>
                           
                           <td class="center-aligned">−3</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">9</td>
                           
                           <td class="center-aligned">11</td>
                           
                           <td class="center-aligned">−11</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">15. Schizophora</td>
                           
                           <td class="center-aligned">−3</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">−6</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">−9</td>
                           
                           <td class="center-aligned">−3</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="left-aligned">11</td>
                           
                           <td class="left-aligned">−6</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">−9</td>
                           
                           <td class="center-aligned">29</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">−2</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">16. Tephritidae</td>
                           
                           <td class="center-aligned">8.5</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">13</td>
                           
                           <td class="center-aligned">11</td>
                           
                           <td class="center-aligned">0.5</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">28.5</td>
                           
                           <td class="center-aligned">7.5</td>
                           
                           <td class="left-aligned">14</td>
                           
                           <td class="left-aligned">1.5</td>
                           
                           <td class="center-aligned">26</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">46.5</td>
                           
                           <td class="center-aligned">51.5</td>
                           
                           <td class="center-aligned">3.5</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">17. Drosphilidae + Calliphoridae</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">−5</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="left-aligned">5</td>
                           
                           <td class="left-aligned">0</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">−3</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">−1</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">18. Calliphoridae</td>
                           
                           <td class="center-aligned">14</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">12</td>
                           
                           <td class="center-aligned">16</td>
                           
                           <td class="center-aligned">20</td>
                           
                           <td class="center-aligned">36</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="left-aligned">31</td>
                           
                           <td class="left-aligned">7</td>
                           
                           <td class="center-aligned">51</td>
                           
                           <td class="center-aligned">24</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">44</td>
                           
                           <td class="center-aligned">10</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">19.<i> Drosophila</i></td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">13</td>
                           
                           <td class="center-aligned">14</td>
                           
                           <td class="center-aligned">14</td>
                           
                           <td class="center-aligned">33</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="left-aligned">39</td>
                           
                           <td class="left-aligned">8</td>
                           
                           <td class="center-aligned">44</td>
                           
                           <td class="center-aligned">21</td>
                           
                           <td class="center-aligned">15</td>
                           
                           <td class="center-aligned">26</td>
                           
                           <td class="center-aligned">27</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">A. Node B +<i> D. melanogaster</i></td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="left-aligned">3</td>
                           
                           <td class="left-aligned">0</td>
                           
                           <td class="center-aligned">8</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">7</td>
                           
                           <td class="center-aligned">10</td>
                           
                           <td class="center-aligned">7</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">B. Node C +<i> D. mauritania</i></td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="left-aligned">0</td>
                           
                           <td class="left-aligned">0</td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">2</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">C.<i> D. simulans </i>+<i> D. sechellia</i></td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">2</td>
                           
                           <td class="center-aligned">−1</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="center-aligned">1</td>
                           
                           <td class="center-aligned">5</td>
                           
                           <td class="center-aligned">0</td>
                           
                           <td class="left-aligned">5</td>
                           
                           <td class="left-aligned">1</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">4</td>
                           
                           <td class="center-aligned">3</td>
                           
                           <td class="center-aligned">6</td>
                           
                           <td class="center-aligned">1</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               <div class="article-section__table-source"></div>
            </div>
            
            <p>A second method of testing whether genes should be 
included in combined analyses is treewide assessment of homoplasy, the 
rationale being that highly homoplasious genes may decrease the 
resolution by more than is gained from the increased dataset size. A 
comparison of the relative contributions of each gene partition to the 
overall tree length is given in <a href="#t5" class="scrollableLink">Table 5</a>.
 Differences between protein coding genes are negligible. The ribosomal 
genes are less homoplasious than expected in the DNA123 and PROT 
analyses, but not in the DNA12 analyses, suggesting that third codon 
positions may be a more important source of homoplasy than the ribosomal
 genes. The tRNA genes are consistently the least homoplasious gene 
partition and the magnitude of their influence is greatly increased for 
the PROT analyses, suggesting that, with the loss of information due to 
recoding genes as amino acids, the tRNAs take on greater significance. 
There is no justification for the exclusion of any mt gene and there is a
 very strong argument for the inclusion of at least the tRNA genes.
            </p>
            
            <div class="article-table-content" id="t5">
               <header class="article-table-caption"><span class="table-caption__label">Table 5.
            </span>Relative contributions made by each gene to the whole
 phylogeny, ALL and PCG datasets. Differences in proportional length of 
each aligned gene vs. tree steps and informative sites only vs. tree 
steps. Negative values indicate that a gene’s contribution to the 
overall tree length is less than expected based on its aligned length 
alone.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th rowspan="2" class="bottom-bordered-cell left-aligned">Gene</th>
                           
                           <th colspan="2" class="bottom-bordered-cell center-aligned">ALL‐DNA123</th>
                           
                           <th colspan="2" class="bottom-bordered-cell center-aligned">ALL‐DNA12</th>
                           
                           <th colspan="2" class="bottom-bordered-cell center-aligned">ALL‐PROT</th>
                           
                        </tr>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned">Total</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Informative</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Total</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Informative</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Total</th>
                           
                           <th class="bottom-bordered-cell center-aligned">Informative</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>atp6</i> 
                           </td>
                           
                           <td class="center-aligned">0.20%</td>
                           
                           <td class="center-aligned">0.06%</td>
                           
                           <td class="center-aligned">−0.30%</td>
                           
                           <td class="center-aligned">−0.21%</td>
                           
                           <td class="center-aligned">0.07%</td>
                           
                           <td class="center-aligned">−0.11%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>atp8</i> 
                           </td>
                           
                           <td class="center-aligned">0.36%</td>
                           
                           <td class="center-aligned">0.16%</td>
                           
                           <td class="center-aligned">0.47%</td>
                           
                           <td class="center-aligned">0.20%</td>
                           
                           <td class="center-aligned">0.75%</td>
                           
                           <td class="center-aligned">0.54%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cox1</i> 
                           </td>
                           
                           <td class="center-aligned">−1.48%</td>
                           
                           <td class="center-aligned">0.44%</td>
                           
                           <td class="center-aligned">−3.97%</td>
                           
                           <td class="center-aligned">−0.26%</td>
                           
                           <td class="center-aligned">−2.83%</td>
                           
                           <td class="center-aligned">−0.32%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cox2</i> 
                           </td>
                           
                           <td class="center-aligned">−0.07%</td>
                           
                           <td class="center-aligned">0.03%</td>
                           
                           <td class="center-aligned">−0.66%</td>
                           
                           <td class="center-aligned">−0.04%</td>
                           
                           <td class="center-aligned">−0.10%</td>
                           
                           <td class="center-aligned">−0.12%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cox3</i> 
                           </td>
                           
                           <td class="center-aligned">0.07%</td>
                           
                           <td class="center-aligned">0.07%</td>
                           
                           <td class="center-aligned">−0.81%</td>
                           
                           <td class="center-aligned">−0.31%</td>
                           
                           <td class="center-aligned">−0.06%</td>
                           
                           <td class="center-aligned">−0.05%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cytB</i> 
                           </td>
                           
                           <td class="center-aligned">−0.01%</td>
                           
                           <td class="center-aligned">0.18%</td>
                           
                           <td class="center-aligned">−1.29%</td>
                           
                           <td class="center-aligned">−0.22%</td>
                           
                           <td class="center-aligned">−0.31%</td>
                           
                           <td class="center-aligned">0.02%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh1</i> 
                           </td>
                           
                           <td class="center-aligned">−0.52%</td>
                           
                           <td class="center-aligned">−0.01%</td>
                           
                           <td class="center-aligned">−1.13%</td>
                           
                           <td class="center-aligned">−0.27%</td>
                           
                           <td class="center-aligned">0.55%</td>
                           
                           <td class="center-aligned">0.78%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh2</i> 
                           </td>
                           
                           <td class="center-aligned">1.89%</td>
                           
                           <td class="center-aligned">0.24%</td>
                           
                           <td class="center-aligned">2.22%</td>
                           
                           <td class="center-aligned">0.22%</td>
                           
                           <td class="center-aligned">3.64%</td>
                           
                           <td class="center-aligned">1.95%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh3</i> 
                           </td>
                           
                           <td class="center-aligned">0.56%</td>
                           
                           <td class="center-aligned">0.32%</td>
                           
                           <td class="center-aligned">0.40%</td>
                           
                           <td class="center-aligned">0.31%</td>
                           
                           <td class="center-aligned">0.84%</td>
                           
                           <td class="center-aligned">0.60%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh4</i> 
                           </td>
                           
                           <td class="center-aligned">1.10%</td>
                           
                           <td class="center-aligned">0.39%</td>
                           
                           <td class="center-aligned">0.62%</td>
                           
                           <td class="center-aligned">0.16%</td>
                           
                           <td class="center-aligned">2.51%</td>
                           
                           <td class="center-aligned">1.36%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh4L</i> 
                           </td>
                           
                           <td class="center-aligned">0.13%</td>
                           
                           <td class="center-aligned">−0.06%</td>
                           
                           <td class="center-aligned">0.25%</td>
                           
                           <td class="center-aligned">0.00%</td>
                           
                           <td class="center-aligned">0.73%</td>
                           
                           <td class="center-aligned">0.43%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh5</i> 
                           </td>
                           
                           <td class="center-aligned">1.44%</td>
                           
                           <td class="center-aligned">0.29%</td>
                           
                           <td class="center-aligned">1.04%</td>
                           
                           <td class="center-aligned">0.26%</td>
                           
                           <td class="center-aligned">3.36%</td>
                           
                           <td class="center-aligned">2.26%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh6</i> 
                           </td>
                           
                           <td class="center-aligned">1.38%</td>
                           
                           <td class="center-aligned">0.50%</td>
                           
                           <td class="center-aligned">1.61%</td>
                           
                           <td class="center-aligned">0.53%</td>
                           
                           <td class="center-aligned">2.40%</td>
                           
                           <td class="center-aligned">1.49%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>l‐rrna</i> 
                           </td>
                           
                           <td class="center-aligned">−0.91%</td>
                           
                           <td class="center-aligned">−0.44%</td>
                           
                           <td class="center-aligned">2.29%</td>
                           
                           <td class="center-aligned">0.88%</td>
                           
                           <td class="center-aligned">−2.46%</td>
                           
                           <td class="center-aligned">−2.36%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>s‐rrna</i> 
                           </td>
                           
                           <td class="center-aligned">−0.47%</td>
                           
                           <td class="center-aligned">−0.54%</td>
                           
                           <td class="center-aligned">1.40%</td>
                           
                           <td class="center-aligned">0.06%</td>
                           
                           <td class="center-aligned">−1.24%</td>
                           
                           <td class="center-aligned">−1.90%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">tRNAs</td>
                           
                           <td class="center-aligned">−3.68%</td>
                           
                           <td class="center-aligned">−1.63%</td>
                           
                           <td class="center-aligned">−2.15%</td>
                           
                           <td class="center-aligned">−1.31%</td>
                           
                           <td class="center-aligned">−7.86%</td>
                           
                           <td class="center-aligned">−4.57%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Gene</td>
                           
                           <td class="center-aligned">PCG‐DNA123</td>
                           
                           <td class="center-aligned"></td>
                           
                           <td class="center-aligned">PCG‐DNA12</td>
                           
                           <td class="center-aligned"></td>
                           
                           <td class="center-aligned">PCG‐PROT</td>
                           
                           <td class="center-aligned"></td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>atp6</i> 
                           </td>
                           
                           <td class="center-aligned">−0.11%</td>
                           
                           <td class="center-aligned">−0.11%</td>
                           
                           <td class="center-aligned">−0.33%</td>
                           
                           <td class="center-aligned">−0.36%</td>
                           
                           <td class="center-aligned">−0.95%</td>
                           
                           <td class="center-aligned">−0.98%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>atp8</i> 
                           </td>
                           
                           <td class="center-aligned">0.35%</td>
                           
                           <td class="center-aligned">0.14%</td>
                           
                           <td class="center-aligned">0.81%</td>
                           
                           <td class="center-aligned">0.34%</td>
                           
                           <td class="center-aligned">0.97%</td>
                           
                           <td class="center-aligned">0.65%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cox1</i> 
                           </td>
                           
                           <td class="center-aligned">−2.75%</td>
                           
                           <td class="center-aligned">0.19%</td>
                           
                           <td class="center-aligned">−5.86%</td>
                           
                           <td class="center-aligned">−0.46%</td>
                           
                           <td class="center-aligned">−7.26%</td>
                           
                           <td class="center-aligned">−1.68%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cox2</i> 
                           </td>
                           
                           <td class="center-aligned">−0.46%</td>
                           
                           <td class="center-aligned">−0.15%</td>
                           
                           <td class="center-aligned">−0.90%</td>
                           
                           <td class="center-aligned">−0.10%</td>
                           
                           <td class="center-aligned">−1.28%</td>
                           
                           <td class="center-aligned">−1.01%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cox3</i> 
                           </td>
                           
                           <td class="center-aligned">−0.35%</td>
                           
                           <td class="center-aligned">−0.12%</td>
                           
                           <td class="center-aligned">−1.11%</td>
                           
                           <td class="center-aligned">−0.54%</td>
                           
                           <td class="center-aligned">−1.39%</td>
                           
                           <td class="center-aligned">−1.00%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>cytB</i> 
                           </td>
                           
                           <td class="center-aligned">−0.66%</td>
                           
                           <td class="center-aligned">−0.10%</td>
                           
                           <td class="center-aligned">−1.81%</td>
                           
                           <td class="center-aligned">−0.43%</td>
                           
                           <td class="center-aligned">−2.44%</td>
                           
                           <td class="center-aligned">−1.29%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh1</i> 
                           </td>
                           
                           <td class="center-aligned">−1.21%</td>
                           
                           <td class="center-aligned">−0.27%</td>
                           
                           <td class="center-aligned">−1.54%</td>
                           
                           <td class="center-aligned">−0.46%</td>
                           
                           <td class="center-aligned">−0.84%</td>
                           
                           <td class="center-aligned">0.07%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh2</i> 
                           </td>
                           
                           <td class="center-aligned">1.75%</td>
                           
                           <td class="center-aligned">−0.10%</td>
                           
                           <td class="center-aligned">3.64%</td>
                           
                           <td class="center-aligned">0.23%</td>
                           
                           <td class="center-aligned">4.13%</td>
                           
                           <td class="center-aligned">1.37%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh3</i> 
                           </td>
                           
                           <td class="center-aligned">0.51%</td>
                           
                           <td class="center-aligned">0.29%</td>
                           
                           <td class="center-aligned">0.71%</td>
                           
                           <td class="center-aligned">0.47%</td>
                           
                           <td class="center-aligned">0.75%</td>
                           
                           <td class="center-aligned">0.44%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh4</i> 
                           </td>
                           
                           <td class="center-aligned">0.64%</td>
                           
                           <td class="center-aligned">0.08%</td>
                           
                           <td class="center-aligned">1.25%</td>
                           
                           <td class="center-aligned">0.17%</td>
                           
                           <td class="center-aligned">1.98%</td>
                           
                           <td class="center-aligned">0.37%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh4L</i> 
                           </td>
                           
                           <td class="center-aligned">0.00%</td>
                           
                           <td class="center-aligned">−0.16%</td>
                           
                           <td class="center-aligned">0.44%</td>
                           
                           <td class="center-aligned">−0.03%</td>
                           
                           <td class="center-aligned">0.68%</td>
                           
                           <td class="center-aligned">0.25%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh5</i> 
                           </td>
                           
                           <td class="center-aligned">0.90%</td>
                           
                           <td class="center-aligned">−0.13%</td>
                           
                           <td class="center-aligned">2.06%</td>
                           
                           <td class="center-aligned">0.40%</td>
                           
                           <td class="center-aligned">2.65%</td>
                           
                           <td class="center-aligned">1.31%</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned"> <i>nadh6</i> 
                           </td>
                           
                           <td class="center-aligned">1.42%</td>
                           
                           <td class="center-aligned">0.45%</td>
                           
                           <td class="center-aligned">2.62%</td>
                           
                           <td class="center-aligned">0.78%</td>
                           
                           <td class="center-aligned">2.99%</td>
                           
                           <td class="center-aligned">1.50%</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               
               <div class="article-section__table-footnotes">
                  <ul>
                     <li id="t5_note220">
                        ALL, all gene partitions; DNA123, nucleotide 
sequences, all codon positions; DNA12, nucleotide sequences, first and 
second positions only; PCG, protein coding genes only; PROT, amino acid 
sequences.
                        
                     </li>
                  </ul>
               </div>
               <div class="article-section__table-source"></div>
            </div>
            
            <p>In addition to partitioning by gene, we also examined the
 effects of partitioning within genes using the datasets DNA123, DNA12 
and PROT, which should represent progressively more conservative 
analyses which sacrifice variability to reduce homoplasy. Comparison of 
the three approaches demonstrates the principal differences related to 
interordinal relationships (particularly for the MP analyses) and 
differing levels of nodal support within the orders. The DNA123 datasets
 tended to have higher nodal support than either the DNA12 or PROT 
datasets for common nodes, suggesting that the signal in the third codon
 position remains important despite high levels of homoplasy (cf. <a href="#t3" class="scrollableLink">Table 3</a>,
 the DNA123 trees are almost twice the length of the DNA12 trees). This 
trend extended to quite deep nodes, suggesting that the third codon 
positions do not become saturated quite as quickly as commonly believed 
(e.g. <span title=""><a href="#b56" id="#b56R" class="bibLink tab-link" data-tab="pane-pcw-references">Lin &amp; Danforth, 2004</a></span>),
 although differences between interordinal relationships may be 
attributable to saturation. Further, the reduced nodal support and 
modest topological differences in some analyses between the DNA12 and 
PROT datasets conflicts with the widely held, but to the best of our 
knowledge never substantiated, notion that all differences between trees
 based on DNA vs. amino acids are due to third codon position variation.
 For ML and BA analyses, the outgroup hemipteran<i> Triatoma </i>tended 
to group within the ingroup close to Coleoptera in the PCG‐DNA12 and 
PCG‐PROT datasets. This did not occur in the ALL or PCG‐DNA123 datasets,
 which recovered holometabolan monophyly. Reducing the amount of data, 
through the elimination of third codon positions, recoding as amino 
acids or elimination of the rRNA and tRNA genes, apparently results in 
worse trees.
            </p>
            
            <p>For the BA analyses, there is a related issue of how best
 to partition the data for the fastest and most accurate estimation of 
phylogeny. We performed duplicate analyses of the ALL‐DNA123, ALL‐DNA12,
 PCG‐DNA123 and PCG‐DNA12 datasets using three partitioning approaches –
 partitioning by gene (GP), by codon (CP) and by codons within each gene
 (CGP) – resulting in thirteen, three and thirty‐nine partitions for the
 protein coding genes (for the ALL analyses, the rRNA and tRNA genes 
were included as an additional three partitions). Topologically, most 
differences were at the interordinal level. Partitioning had a greater 
influence on the DNA123 than the DNA12 datasets. In both ALL and PCG, GP
 favoured (Lepidoptera + Diptera) + (Coleoptera + Hymenoptera), whereas 
both CP and CGP favoured Diptera + (Lepidoptera + (Coleoptera + 
Hymenoptera)). The posterior probabilities for these alternative 
relationships were between 0.97 and 1.0 for ALL and 0.62 and 0.78 for 
PCG datasets, indicating that the partitioning strategy influences 
topology and that conflicting topologies can each enjoy significant 
support under different strategies. By contrast, the DNA12 topologies 
were identical for the three partitioning strategies in the ALL datasets
 and differed only in the location of<i> Triatoma </i>in the PCG 
datasets. This suggests that differences between the evolutionary 
dynamics of the first and second vs. third codon positions are more 
significant than differences between different mt genes. We calculated 
Bayes factors (<span title=""><a href="#b67" id="#b67R" class="bibLink tab-link" data-tab="pane-pcw-references">Nylander<i> et al.</i>, 2004</a></span>) comparing the three partition approaches (<a href="#t6" class="scrollableLink">Table 6</a>),
 showing that, for the DNA123 datasets, the difference between GP and CP
 was much greater than that between CP and CGP. For the ALL‐DNA12 
datasets, the difference between CP and CGP was much greater than that 
between GP and CP; however, the opposite was the case for PCG‐DNA12 
datasets. The magnitudes of each Bayes factor comparison were also 
extremely high, indicating highly significant differences. Collectively,
 the topological comparisons and the Bayes factors indicate that the 
correct partitioning strategy is very important to BA analyses of mt 
genome data, and that partitioning by codon (with or without additional 
gene‐based partitions) is the most biologically relevant way to 
partition these data.
            </p>
            
            <div class="article-table-content" id="t6">
               <header class="article-table-caption"><span class="table-caption__label">Table 6.
            </span>Bayes factors for comparison of partition strategy of protein coding genes.
               </header>
               
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th class="bottom-bordered-cell left-aligned"></th>
                           
                           <th class="bottom-bordered-cell center-aligned">ALL‐DNA123</th>
                           
                           <th class="bottom-bordered-cell center-aligned">ALL‐DNA12</th>
                           
                           <th class="bottom-bordered-cell center-aligned">PCG‐DNA123</th>
                           
                           <th class="bottom-bordered-cell center-aligned">PCG‐DNA12</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td class="left-aligned">Gene partitions (13)</td>
                           
                           <td class="center-aligned">−193964.88</td>
                           
                           <td class="center-aligned">−120417.77</td>
                           
                           <td class="center-aligned">−158292.09</td>
                           
                           <td class="center-aligned">−82926.63</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Codon partitions (3)</td>
                           
                           <td class="center-aligned">−191348.76</td>
                           
                           <td class="center-aligned">−120291.89</td>
                           
                           <td class="center-aligned">−149817.65</td>
                           
                           <td class="center-aligned">−78794.63</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Codon &amp; gene partitions (39)</td>
                           
                           <td class="center-aligned">−189742.81</td>
                           
                           <td class="center-aligned">−119137.02</td>
                           
                           <td class="center-aligned">−147642.29</td>
                           
                           <td class="center-aligned">−77579.68</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Bayes factor gene vs. codon</td>
                           
                           <td class="center-aligned">2616.12 (very strong)</td>
                           
                           <td class="center-aligned">125.88 (very strong)</td>
                           
                           <td class="center-aligned">8474.44 (very strong)</td>
                           
                           <td class="center-aligned">4132.00 (very strong)</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td class="left-aligned">Bayes factor codon vs. codon &amp; gene</td>
                           
                           <td class="center-aligned">1605.95 (very strong)</td>
                           
                           <td class="center-aligned">1154.87 (very strong)</td>
                           
                           <td class="center-aligned">2175.36 (very strong)</td>
                           
                           <td class="center-aligned">1214.95 (very strong)</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
               
               <div class="article-section__table-footnotes">
                  <ul>
                     <li id="t6_note30">
                        ALL, all gene partitions; DNA123, nucleotide 
sequences, all codon positions; DNA12, nucleotide sequences, first and 
second positions only; PCG, protein coding genes only.
                        
                     </li>
                  </ul>
               </div>
               <div class="article-section__table-source"></div>
            </div>
            
         </div>
         
         <div class="article-section__sub-content" id="ss11">
            
            <h3 class="article-section__sub-title section2" id="ss11-title"> Reliability of mitochondrial genome data</h3>
            
            <p>We can determine the relative timescales or ranges of 
molecular divergence over which mt genome data can be considered a 
reliable phylogenetic marker. For the ALL‐DNA12 datasets, which had the 
best resolution, best nodal support and the least artefacts, 
relationships were consistent within each order. It is unclear whether 
these data are reliable at the interordinal level; however, as Diptera 
is the order for which taxonomic coverage is the most inclusive, we set 
the upper reliable bound at the divergence of Culicidae from the 
remaining dipterans. The lower bound is set by relationships within the 
genus<i> Drosophila</i>. All our analyses recovered the same branching pattern within<i> Drosophila</i>,
 a pattern which is consistent with the current phylogenetic 
understanding of this genus, and so we conclude that mt genomes are 
reliable down to at least the resolution of species within this genus. 
Current fossil and molecular clock estimates for these two divergences 
are as follows: Culicidae from the remaining Diptera in the early 
Triassic, 250–230 million years ago (<span title=""><a href="#b10" id="#b10R" class="bibLink tab-link" data-tab="pane-pcw-references">Blagoderov<i> et al.</i>, 2002</a></span>; <span title=""><a href="#b50" id="#b50R" class="bibLink tab-link" data-tab="pane-pcw-references">Labandeira, 2005</a></span>), and<i> D. simulans </i>+<i> D. sechellia </i>at less than one million years ago (<span title=""><a href="#b74%20#b86" id="#b74 #b86R" class="bibLink tab-link" data-tab="pane-pcw-references">Russo<i> et al.</i>, 1995; Tamura<i> et al.</i>, 2004</a></span>).
 Alternatively, the calculation of divergences across these taxa (Table 
S7; ‘Supplementary material’) using corrected ML pairwise distances 
showed that these distances ranged from 0.01 (<i>D. simulans </i>vs.<i> D. sechellia</i>) to 0.47 (<i>A. quadrimaculatus </i>Culicidae vs.<i> T. punctata </i>Nemestrinidae).
 mt genome data are therefore a reliable phylogenetic marker over an 
extremely broad timescale (at least 1–200 million years ago) and across 
wide molecular divergences (0.1–0.47 ML distances).
            </p>
            
         </div>
         
      </div>
      <div class="article-section__content" id="ss12">
         
         <h2 class="article-section__title section__title section1" id="ss12-title"> Discussion</h2>
         
         <div class="article-section__sub-content" id="ss13">
            
            <h3 class="article-section__sub-title section2" id="ss13-title"> Dipteran phylogeny and mitochondrial genomics</h3>
            
            <p>Dipteran relationships are resolved robustly by mt 
genomes. Nodal support and topology are broadly consistent across 
analytical methods and are largely congruent with previous phylogenetic 
analyses of the order. Each of the four dipteran families represented by
 multiple exemplars (Culicidae, Tephritidae, Calliphoridae and 
Drosophilidae) are monophyletic. Species relationships within 
Drosophilidae are consistent with previous analyses based on multiple 
molecular markers (<span title=""><a href="#b74%20#b68%20#b71%20#b55%20#b85" id="#b74 #b68 #b71 #b55 #b85R" class="bibLink tab-link" data-tab="pane-pcw-references">Russo<i> et al.</i>, 1995; O’Grady &amp; Kidwell, 2002; Remsen &amp; O’Grady, 2002; Lewis<i> et al.</i>, 2005; Symons &amp; Wertheim, 2005</a></span>) or morphology (<span title=""><a href="#b39%20#b76" id="#b39 #b76R" class="bibLink tab-link" data-tab="pane-pcw-references">Grimaldi, 1990; Schawaroch, 2002</a></span>). Deeper relationships between dipteran families are also largely consistent with previous analyses (<span title=""><a href="#b97" id="#b97R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates &amp; Wiegmann, 2005</a></span>)
 and recover a monophyletic Brachycera, Cyclorrhapha and Schizophora. 
The only inconsistency with previous work concerns the monophyly of the 
acalyptrates and placement of the Nemestrinidae (<a href="#f4" class="scrollableLink">Fig. 4</a>).
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="f4"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/df80fd01-f44a-459b-94f1-b4686d226eb6/syen_355_f4.gif"><img class="figure__image" src="syen_355_f4.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 4</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f4&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x" class="ppt-figure-link">PowerPoint</a></div>
                     </div>
                     <div>
                        <p>Summarized relationships between dipteran families with major groups named. (A) Consensus of previous studies (redrawn after <span class="figureLink bibLink tab-link" title=""><a href="#b97" id="#b97R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates &amp; Wiegmann, 2005</a></span>). (B) This study. Note that Tabanoidea is<i> sensu </i><span class="figureLink bibLink tab-link" title=""><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Colless &amp; McAlpine (1989)</a></span>, Muscomorpha is<i> sensu </i><span class="figureLink bibLink tab-link" title=""><a href="#b93" id="#b93R" class="bibLink tab-link" data-tab="pane-pcw-references">Woodley (1989)</a></span> and Acalyptratae/Calyptratae are<i> sensu </i>McAlpine (1991). Numbers below each node (circled) are common across datasets (refer to other figures, <a href="#t4" class="scrollableLink">Table 4</a> and supplementary material Tables S3–S6).</p>
                        
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>Our phylogeny strongly supports the sister group 
relationship, Nemestrinidae + Tabanidae. It is highly doubtful, however,
 that this is the actual sister group relationship, but rather the 
result of limited taxon sampling within the ‘basal brachycera’. Our 
result is more likely indicative of a close relationship between 
Nemestrinidae and one of the non‐muscomorph brachyceran infraorders. 
Currently, Brachycera is divided into four infraorders, Xylophagomorpha,
 Stratiomyomorpha, Tabanomorpha and Muscomorpha (<span title=""><a href="#b95" id="#b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates, 2002</a></span>), with some evidence that the first three infraorders form a monophyletic group (<span title=""><a href="#b95%20#b92%20#b97" id="#b95 #b92 #b97R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates, 2002; Wiegmann<i> et al.</i>, 2003; Yeates &amp; Wiegmann, 2005</a></span>). Nemestrinidae usually is considered part of Muscomorpha<i> sensu </i><span title=""><a href="#b93" id="#b93R" class="bibLink tab-link" data-tab="pane-pcw-references">Woodley (1989)</a></span> (including Empidoidea, Asiloidea and Cyclorrhapha), although they have been included in Tabanoidea (<span title=""><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Colless &amp; McAlpine, 1991</a></span>). <span title=""><a href="#b93" id="#b93R" class="bibLink tab-link" data-tab="pane-pcw-references">Woodley (1989)</a></span>
 proposed three Muscomorpha synapomorphies: reduction of antennal 
flagellomeres to four segments or less, loss of tibial spurs and female 
cerci composed of a single segment. <span title=""><a href="#b79" id="#b79R" class="bibLink tab-link" data-tab="pane-pcw-references">Sinclair (1992)</a></span>,
 in a review of larval mandible characters, found that this character 
set neither added new support nor contradicted the monophyly of 
Muscomorpha. Two additional synapomorphies of Muscomorpha were proposed 
by <span title=""><a href="#b80" id="#b80R" class="bibLink tab-link" data-tab="pane-pcw-references">Sinclair<i> et al. </i>(1994)</a></span>
 from studies of male genitalia: base of the epandrium articulated on 
the gonocoxites (character 12) and gonostyli which move obliquely or in a
 dorsoventral direction (character 13). The first character state occurs
 also in Tabanidae and Xylomyidae, but its presence in these families is
 considered the result of convergent evolution (<span title=""><a href="#b80%20#b95" id="#b80 #b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Sinclair<i> et al.</i>, 1994; Yeates, 2002</a></span>).
 The second character, direction of gonostyli articulation, was scored 
incorrectly for Nemestrinidae, in which movement is horizontal as in 
‘nematocera’, Xylophagomorpha, Stratiomyomorpha and Tabanomorpha (<span title=""><a href="#b94%20#b95" id="#b94 #b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates, 1994, 2002</a></span>). <span title=""><a href="#b38" id="#b38R" class="bibLink tab-link" data-tab="pane-pcw-references">Griffiths (1994)</a></span> noted that each synapomorphy of Muscomorpha proposed by <span title=""><a href="#b93" id="#b93R" class="bibLink tab-link" data-tab="pane-pcw-references">Woodley (1989)</a></span>
 occurred also in Tabanomorpha. In Griffiths’ (1994) opinion, evaluation
 of an alternative placement within Tabanomorpha, as advocated by <span title=""><a href="#b63" id="#b63R" class="bibLink tab-link" data-tab="pane-pcw-references">Nagatomi (1992)</a></span>, required detailed morphological studies of the male genitalia in Nemestrinidae, as an existing study by <span title=""><a href="#b58" id="#b58R" class="bibLink tab-link" data-tab="pane-pcw-references">Mackerras (1925)</a></span>
 did not allow ready homologization of structures between nemestrinids 
and tabanomorphs. A contemporary detailed study of bombyliid phylogeny (<span title=""><a href="#b94" id="#b94R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates, 1994</a></span>),
 including a re‐evaluation of nemestrinid morphology, does not comment 
on their possible homologies with tabanomorphs, but nemestrinids do not 
group with the included tabanomorph outgroup family Rhagionidae. <span title=""><a href="#b95" id="#b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates (2002)</a></span>,
 in the first large‐scale cladistic analysis of the group, found strong 
morphological support for the monophyly of Muscomorpha with 
Nemestrinidae, the sister group of Heterodactyla (the remaining 
muscomorphs). Six synapomorphies were found for muscomorphan monophyly, 
five of which are refinements of <span title=""><a href="#b93" id="#b93R" class="bibLink tab-link" data-tab="pane-pcw-references">Woodley (1989)</a></span> and <span title=""><a href="#b80" id="#b80R" class="bibLink tab-link" data-tab="pane-pcw-references">Sinclair<i> et al. </i>(1994)</a></span>.
            </p>
            
            <p>
               </p><ul class="rlist hanging">
                  
                  <li><span class="number">1 </span>
                     
                     
                     <p>Character 27: reduction of antennal flagellomeres to less than four.</p>
                     
                  </li>
                  
                  <li><span class="number">2 </span>
                     
                     
                     <p>Character 53: foretibial spurs absent.</p>
                     
                  </li>
                  
                  <li><span class="number">3 </span>
                     
                     
                     <p>Character 54: hindtibial spurs absent.</p>
                     
                  </li>
                  
                  <li><span class="number">4 </span>
                     
                     
                     <p>Character 62: epandrium articulates on gonocoxites.</p>
                     
                  </li>
                  
                  <li><span class="number">5 </span>
                     
                     
                     <p>Character 80: lateral aedeagal apodemes large and external.</p>
                     
                  </li>
                  
                  <li><span class="number">6 </span>
                     
                     
                     <p>Character 98: female cerci one segmented.</p>
                     
                  </li>
                  
               </ul>
            <p></p>
            
            <p>However, echoing the concerns of <span title=""><a href="#b38%20#b95" id="#b38 #b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Griffiths (1994), Yeates (2002)</a></span>
 noted that most of these characters are at least partly homoplasious; 
antennal flagellomere reduction occurs in non‐muscomorphan Brachycera, 
foretibial spurs are also absent in Pantophthalmidae, Xylomyidae and 
Stratiomyidae, hindtibial spurs are also absent in Stratiomyidae, 
articulated epandria are also found in Tabanidae and Xylomyidae, and 
females with one segmented cerci are also found in some Tabanidae and 
Athericidae. Finally, the novel muscomorphan synapomorphy, lateral 
aedeagal apodemes, has been interpreted in a wide variety of ways and is
 highly variable across Brachycera (<span title=""><a href="#b38%20#b80" id="#b38 #b80R" class="bibLink tab-link" data-tab="pane-pcw-references">Griffiths, 1994; Sinclair<i> et al.</i>, 1994</a></span>). Most recently, <span title=""><a href="#b48" id="#b48R" class="bibLink tab-link" data-tab="pane-pcw-references">Krzeminski &amp; Krzeminski (2003)</a></span>
 suggested that, on the basis of similarities of fossil specimens to 
xylophagomorphs and tabanomorphs, the traditional placement of 
Nemestrinidae within Muscomorpha should be reassessed.
            </p>
            
            <p>Only one molecular study considering brachyceran relationships also includes a representative of the Nemestrinidae. <span title=""><a href="#b92" id="#b92R" class="bibLink tab-link" data-tab="pane-pcw-references">Wiegmann<i> et al. </i>(2003)</a></span>, in a combined analysis of the large rRNA subunit (LSU‐rRNA = 28S‐rRNA) and morphology (the <span title=""><a href="#b95" id="#b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates, 2002</a></span>
 matrix), found reasonable support (82% bootstrap) for the monophyly of 
Muscomorpha with Nemestrinidae as its earliest diverging branch. 
However, when morphology was excluded, the same topology was recovered 
but the nodal support became insignificant (less than 50% bootstrap 
support in MP and a posterior probability of 0.76 in BA analyses).
            </p>
            
            <p>The question of the monophyly of Muscomorpha (<i>sensu </i><span title=""><a href="#b93" id="#b93R" class="bibLink tab-link" data-tab="pane-pcw-references">Woodley, 1989</a></span>)
 is thus contentious. Morphological evidence favours the inclusion of 
nemestrinids in Muscomorpha, but most synapomorphies for this 
relationship require the postulation of convergent evolution of similar 
structures in other brachyceran infraorders. It is equally plausible 
that nemestrinids actually are related to one of these other groups, 
with the similarities of nemestrinids to heteroneurans being convergent.
 The only other molecular phylogenetic study on this question is clearly
 highly influenced by the morphological data partition, as LSU‐rRNA 
phylogenies alone do not recover significant support for muscomorphan 
monophyly (<span title=""><a href="#b92" id="#b92R" class="bibLink tab-link" data-tab="pane-pcw-references">Wiegmann<i> et al.</i>, 2003</a></span>).
 By contrast, we found Tabanidae + Nemestrinidae in almost every 
analysis performed, with the relationship significantly supported in 
almost all cases. The only instance in which Nemestrinidae + 
Cyclorrhapha was found was the MP‐TRAN dataset, without significant 
support. Additional research using mt genome phylogenies to test the 
monophyly and relationships of the brachyceran infraorders will probably
 be highly informative.
            </p>
            
            <p>The second area in which our phylogeny differs from 
traditional ideas of dipteran relationships is in recovering a 
paraphyletic Acalyptratae. Acalyptrates are recognized as a monophyletic
 group in most major synthetic treatments (<span title=""><a href="#b60%20#b96%20#b97" id="#b60 #b96 #b97R" class="bibLink tab-link" data-tab="pane-pcw-references">McAlpine, 1989; Yeates &amp; Wiegmann, 1999, 2005</a></span>),
 despite the acknowledgement that: ‘The search for convincing 
synapomorphies uniting acalyptrate families has been difficult’ (<span title=""><a href="#b96" id="#b96R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates &amp; Wiegmann, 1999: 416</a></span>).
 The nominate character of the group, reduction of the calypter, is 
difficult to interpret across many schizophoran families as a result of 
its wide variation in both Acalyptratae and Calyptratae (= Muscoidea). <span title=""><a href="#b37" id="#b37R" class="bibLink tab-link" data-tab="pane-pcw-references">Griffiths (1972)</a></span> argued against <span title=""><a href="#b41" id="#b41R" class="bibLink tab-link" data-tab="pane-pcw-references">Hennig’s (1958)</a></span>
 conception of acalyptrate monophyly on the basis of calypter 
variability, and stated that the other proposed character uniting 
acalyptrates, reduction of the pupal prothoracic spiracular horn, was 
both homoplastic and understudied in the group. Although arguing 
strongly against the monophyly of Acalyptratae, and also proposing 
calypter variability as useless for identification as a result of the 
inability of calypter morphology to even divide up schizophoran 
families, no proposal was made by <span title=""><a href="#b37" id="#b37R" class="bibLink tab-link" data-tab="pane-pcw-references">Griffiths (1972)</a></span> on how the five superfamilies within Schizophora were interrelated. By contrast, <span title=""><a href="#b60" id="#b60R" class="bibLink tab-link" data-tab="pane-pcw-references">McAlpine (1989)</a></span>
 strongly supported acalyptrate monophyly and proposed fourteen 
synapomorphies for the group. These proposals are yet to be tested 
extensively, and comprehensive phylogenetic treatment of the whole 
Schizophora to resolve the status of Acalyptratae has not been attempted
 (<span title=""><a href="#b96%20#b95" id="#b96 #b95R" class="bibLink tab-link" data-tab="pane-pcw-references">Yeates &amp; Wiegmann, 1999; 2005</a></span>).
            </p>
            
            <p>In our opinion, the majority of recent molecular 
phylogenetic analyses of Diptera have not included sufficient samples 
for a confident resolution of the acalyptrate question. In a study of 
lower cyclorrhaphan relationships, principally the relationships of 
aschizan families to Schizophora, using LSU‐rRNA, <span title=""><a href="#b22" id="#b22R" class="bibLink tab-link" data-tab="pane-pcw-references">Collins &amp; Wiegmann (2002)</a></span>
 found a paraphyletic Acalyptratae, Otitidae + (Lauxaniidae + (Sepsidae +
 Muscidae)); sampling, however, was limited to a single species from 
each of just four families, a gross under‐representation of schizophoran
 diversity. In <span title=""><a href="#b92" id="#b92R" class="bibLink tab-link" data-tab="pane-pcw-references">Wiegmann<i> et al. </i>(2003)</a></span>,
 Acalyptratae and Calyptratae were represented by one species each, 
which were, unsurprisingly, sister groups. Studying Eremoneura, <span title=""><a href="#b62" id="#b62R" class="bibLink tab-link" data-tab="pane-pcw-references">Moulton &amp; Wiegmann (2004)</a></span>
 included four schizophoran species representing four families, two each
 for calyptrates and acalyptrates, and recovered monophyly for both 
groups. The only study with substantial familial coverage within 
acalyptrates (<span title=""><a href="#b40" id="#b40R" class="bibLink tab-link" data-tab="pane-pcw-references">Han &amp; Ro, 2005</a></span>) included fourteen of the sixty‐two recognized families and seven of eleven superfamilies following the classification of <span title=""><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Colless &amp; McAlpine (1991)</a></span>.
 Their result was similar to that reported here, acalyptrate paraphyly 
with drosophilids grouping with calliphorids to the exclusion of 
tephritids, which is unsurprising given that this study exclusively used
 mt genes (<i>s‐rrna</i>,<i> l‐rrna </i>and<i> cox2</i>). Three previous mt genomic phylogenies have yielded a range of results on the question of schizophoran relationships. <span title=""><a href="#b64" id="#b64R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al. </i>(2001)</a></span> found the same relationships as ours, using all the protein coding genes. Reanalysis (<span title=""><a href="#b65" id="#b65R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a</a></span>)
 favoured Calliphoridae + Tephritidae, but omitted nine of the thirteen 
protein coding genes and found non‐significant support for any 
resolution within Schizophora. Support for acalyptrate monophyly also 
came from <span title=""><a href="#b45" id="#b45R" class="bibLink tab-link" data-tab="pane-pcw-references">Junqueira<i> et al. </i>(2004)</a></span>,
 but five of the thirteen protein coding genes were omitted from this 
analysis. This diversity of results from similar primary data highlights
 the dangers of arbitrary data disposal (discussed further below and cf.
 <span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2004</a></span>).
 Current mt genome sequences are inadequately diverse (two of sixty‐two 
families) to be considered sufficient to address the question of 
acalyptrate monophyly, but the strength of nodal support for most 
relationships within Diptera, and the consistency of the recovered 
relationships with respect to analytical criteria, suggest that mt 
genome data may be an excellent avenue of investigation for addressing 
long‐standing contentious issues in dipteran phylogeny, of which 
acalyptrate interrelationships are probably the most significant.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="ss14">
            
            <h3 class="article-section__sub-title section2" id="ss14-title"> Methodological approaches to mitochondrial phylogenomics of insects</h3>
            
            <p>Over a dozen studies have used the mt genome for insect phylogenetics on divergence scales as fine as between strains of<i> Drosophila simulans </i>(<span title=""><a href="#b6" id="#b6R" class="bibLink tab-link" data-tab="pane-pcw-references">Ballard, 2000b</a></span>) to as broad as deep‐level hexapod phylogeny (<span title=""><a href="#b65" id="#b65R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a</a></span>).
 A wide variety of analytical approaches have been taken, but 
comparisons between approaches for common datasets have not been 
extensively performed. Exceptions include examinations of the effects of
 gene exclusion and optimality criteria (<span title=""><a href="#b83%20#b46" id="#b83 #b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Stewart &amp; Beckenbach, 2003; Kim<i> et al.</i>, 2005</a></span>), of data coding strategies (<span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2004</a></span>) and of taxon exclusion (<span title=""><a href="#b16" id="#b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Castro &amp; Dowton, 2005</a></span>).
 A key feature of each analysis is the focus on the effects of varying 
analytical regimes on insect interordinal relationships. As the rate at 
which mt genomes are being sequenced increases, it is worthwhile to 
continue and extend these efforts, particularly concerning how 
methodological approaches may affect the accuracy of inferring 
intraordinal relationships. For that reason, now that sufficient 
representatives are available to examine the relationships of the major 
dipteran clades, we chose to address some of the factors that may affect
 mt genome phylogenies significantly – optimality criteria, data 
inclusion/exclusion, data transformations and partitioning strategies.
            </p>
            
            <p>Optimality criteria are a perennial source of argument 
about the accuracy of phylogenetic studies. Fortunately, optimality 
criteria do not seem to have a very significant effect on intraordinal 
relationships. For common datasets, the topology within Diptera 
recovered by different criteria is largely the same. The major 
differences lie in the strength of nodal support, with MP bootstrapping 
consistently giving the weakest and BA posterior probabilities the 
strongest support for nodes which are common across optimality criteria.
 This effect has repeatedly been noted (<span title=""><a href="#b51%20#b91%20#b14" id="#b51 #b91 #b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Leaché &amp; Reeder, 2002; Whittingham<i> et al.</i>, 2002; Cameron<i> et al.</i>, 2004</a></span>)
 in comparisons of bootstrapping with BA posterior probabilities. 
Bootstrapping is known to be sensitive to the level of homoplasy in a 
dataset (<span title=""><a href="#b42" id="#b42R" class="bibLink tab-link" data-tab="pane-pcw-references">Hillis &amp; Bull, 1993</a></span>). Given that the consistency index of the datasets analysed here never increased above 0.6 (<a href="#t5" class="scrollableLink">Table 5</a>),
 it is likely that bootstrapping underestimates nodal support for mt 
genome data. We have used a cut‐off of 70% bootstrap support for nodal 
significance in line with previous estimations based on smaller, less 
noisy, simulated datasets (<span title=""><a href="#b42" id="#b42R" class="bibLink tab-link" data-tab="pane-pcw-references">Hillis &amp; Bull, 1993</a></span>);
 however, it is possible that significance levels may need to be 
re‐examined for the much larger and noisier, phylogenomic datasets 
currently available. Conversely, we are suspicious that BA posterior 
probabilities overstate nodal significance, particularly in the light of
 our finding that the same dataset, when partitioned in different ways, 
gave alternative topologies for which every node received posterior 
probabilities of 1.0 (see the fuller discussion below). Despite the 
similar topological results from different optimality criteria, we still
 believe that it is valuable to perform replicate analyses under 
different schemes as a way of evaluating upper (BA) and lower (MP) 
confidence bounds to recovered topologies.
            </p>
            
            <p>The second major issue in phylogenetic analyses of mt genome sequences is data exclusion. Many studies have excluded one (e.g. <span title=""><a href="#b32" id="#b32R" class="bibLink tab-link" data-tab="pane-pcw-references">Friedrich &amp; Muqim, 2003</a></span>), two (<span title=""><a href="#b52" id="#b52R" class="bibLink tab-link" data-tab="pane-pcw-references">Lessinger<i> et al.</i>, 2000</a></span>) or as many as nine (<span title=""><a href="#b65" id="#b65R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a</a></span>)
 of the thirteen protein coding genes from phylogenetic analysis, often 
citing difficulties of alignment as justification. In our opinion, these
 concerns are overstated. Reducing the amount of available data resulted
 in a loss of resolution and weakening of nodal support in the present 
study. The ALL datasets were usually better resolved, better supported 
and gave the most reasonable results in the light of previous 
phylogenetic work within Diptera. Levels of homoplasy did not vary 
greatly between the protein coding genes, which mirrors our previous 
findings for a much wider phylogenetic study of mt genomes across all 
arthropods (<span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2004</a></span>).
 The pattern of Bremer support across the gene partitions did not 
suggest that any particular gene had a radically different history, and 
partitions which, in isolation (RIBO, TRAN), gave radically different 
topologies showed considerable levels of hidden support for the ALL 
topology. This is to be expected, as recombination is supposedly rare 
within mt genomes, the mt genes are evolving as a linked unit and any 
particular gene is unlikely to have an evolutionary history entirely at 
odds from the rest of the genome (<span title=""><a href="#b7" id="#b7R" class="bibLink tab-link" data-tab="pane-pcw-references">Ballard &amp; Rand, 2005</a></span>). The only major reason for phylogenetic ‘untrustworthiness’ is therefore differing levels of noise between genes and, indeed, <span title=""><a href="#b75" id="#b75R" class="bibLink tab-link" data-tab="pane-pcw-references">Saccone<i> et al. </i>(1999)</a></span>
 have demonstrated that substitution rates may be related to the 
physical position of individual genes within the mt genome. Our results,
 however, do not suggest that there is any correlation between gene 
location and phylogenetic behaviour, as no gene is consistently for or 
against the topology (<a href="#f1" class="scrollableLink">Fig. 1; Table 6</a><a href="#t6" class="scrollableLink">[link]</a>).
 Further, the difficulty of aligning protein coding genes can be 
alleviated in several ways. The omission of highly divergent genomes 
that are not critical to the phylogenetic questions under consideration,
 such as, in this instance, the louse and sternorrhynch hemipterans, and
 better balanced taxon selections which include more than just a few 
representatives of included clades, make automated alignment more 
consistent. Secondly, the use of amino acid translations for alignments 
before back‐translating for analysis results in more reasonable 
alignments than those generated by nucleotides alone.
            </p>
            
            <p>Related to the question of eliminating protein coding 
genes is the lack of attention which has been devoted to rRNA and tRNA 
genes. Prior to this study, rRNA genes had only been included three 
times (<span title=""><a href="#b83%20#b3" id="#b83 #b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Stewart &amp; Beckenbach, 2003; Bae<i> et al</i>., 2004</a></span>; <span title=""><a href="#b46" id="#b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Kim<i> et al.</i>, 2005</a></span>) and tRNAs once (<span title=""><a href="#b46" id="#b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Kim<i> et al.</i>, 2005</a></span>) in phylogenetic analyses of insects. <span title=""><a href="#b46" id="#b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Kim<i> et al. </i>(2005)</a></span>
 noted that, in comparisons of pairwise differences between taxa, the 
tRNAs had consistently lower distances than either the protein coding or
 rRNA genes. Similarly, we found that the tRNAs were much less 
homoplasious than expected from their combined length, and that the TRAN
 dataset gave comparable results to the ALL or PCG datasets for deep and
 shallow nodes. It should be noted, however, that we did not incorporate
 either paired site coding (<span title=""><a href="#b34" id="#b34R" class="bibLink tab-link" data-tab="pane-pcw-references">Gillespie, 2004</a></span>) within the MP analyses or paired site models, such as the doublet model, in M<span class="smallCaps">r</span>B<span class="smallCaps">ayes</span> (<span title=""><a href="#b73" id="#b73R" class="bibLink tab-link" data-tab="pane-pcw-references">Ronquist &amp; Huelsenbeck, 2003</a></span>);
 therefore, potentially, support from this partition may be inflated as 
informative stem sites are counted twice rather than once (<span title=""><a href="#b90" id="#b90R" class="bibLink tab-link" data-tab="pane-pcw-references">Wheeler &amp; Honeycutt, 1988</a></span>).
 Despite this, tRNA genes have been needlessly overlooked as a result of
 untested assumptions regarding the rates of nucleotide evolution in 
mitochondria, i.e. that tRNA genes are generally too fast. By contrast, 
the rRNA genes probably warrant more caution in their use. The RIBO 
datasets gave tree topologies that were wildly incongruent with the 
other datasets and with previously accepted dipteran phylogenies, 
particularly for the ML and BA analyses, and the large subunit RNA was 
highly homoplasious. <span title=""><a href="#b46" id="#b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Kim<i> et al. </i>(2005)</a></span>
 found that the rRNA genes had consistently much higher pairwise 
differences than the protein coding genes, suggesting that the rRNA 
genes may be very noisy. An additional difficulty with the use of the 
ribosomal genes is that they have not been consistently annotated across
 sequenced insect mt genomes. The large subunit is defined as all 
nucleotides between adjacent tRNAs, whereas the small subunit is 
similarly defined as all nucleotides between an adjacent tRNA and the 
origin of replication, which itself has not been experimentally defined 
for any insect other than<i> Drosophila </i>(<span title=""><a href="#b19" id="#b19R" class="bibLink tab-link" data-tab="pane-pcw-references">Clayton, 1992</a></span>).
 The development of functional methods of annotating mt rRNA genes based
 on secondary structure analyses, similar to those used to define tRNAs,
 is necessary before the rRNA genes can be confidently compared in 
phylogenetic analyses.<a class="fNoteLink superscripted" href="#fn3">‡</a></p>
            
            <p>The third major area of methodological variation between 
insect phylogenetic studies of the mt genome is how sequence data are 
treated in the analysis. Three approaches have been investigated, two of
 which are examined in this study. The first is the translation of 
nucleotides to amino acids, which has the advantage of reducing the 
noise associated with the saturated nucleotide sequence, but introduces 
other potential sources of noise due to redundancy in the genetic code 
(cf. <span title=""><a href="#b44" id="#b44R" class="bibLink tab-link" data-tab="pane-pcw-references">Inagaki<i> et al.</i>, 2004</a></span>). This approach was one of the first used and has been widely adopted (e.g. <span title=""><a href="#b64" id="#b64R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2001</a></span>, 2003a; <span title=""><a href="#b32%20#b14" id="#b32 #b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Friedrich &amp; Muqim, 2003; Cameron<i> et al.</i>, 2004</a></span>).
 The second approach has been to use nucleotides directly without 
translation, which has the advantage of using the maximal amount of 
data; however, justifiable concerns about the capacity of noise in 
saturated nucleotide datasets to overwhelm the signal remains. 
Nucleotide analyses have also been widely used, both in comparison with 
amino acid analyses (e.g. <span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2004</a></span>; present study) and as stand‐alone analyses (e.g. <span title=""><a href="#b3%20#b16%20#b46" id="#b3 #b16 #b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Bae<i> et al.</i>, 2004; Castro &amp; Dowton, 2005; Kim<i> et al.</i>, 2005</a></span>).
 The third and most recently proposed approach is the translation of 
portions of the nucleotide data into purines (A or G = R) or pyrimidines
 (C or T = Y) – R/Y coding – to reduce the effects of nucleotide 
compositional bias (<span title=""><a href="#b69" id="#b69R" class="bibLink tab-link" data-tab="pane-pcw-references">Phillips &amp; Penny, 2003</a></span>). <span title=""><a href="#b25" id="#b25R" class="bibLink tab-link" data-tab="pane-pcw-references">Delsuc<i> et al. </i>(2003)</a></span> took this approach to reanalyse the data of <span title=""><a href="#b65" id="#b65R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al. </i>(2003a)</a></span>
 supporting hexapod polyphyly, and claimed that R/Y coding supported a 
monophyletic Hexapoda. This result, however, was not even remotely 
significant (<i>P </i>= 0.57), and subsequent attempts to apply R/Y 
coding to questions of polyneopteran relationships resulted in an almost
 complete collapse of signal on recoding of nucleotides (<span title=""><a href="#b15" id="#b15R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al.</i>, 2006</a></span>).
 In the light of the limited usefulness of R/Y coding, we have not 
tested again in this study. A fourth approach of combining data 
translated at multiple levels, such as nucleotides with amino acids or 
nucleotides with RY and amino acids, has yet to be applied to mt genome 
data (e.g. <span title=""><a href="#b1" id="#b1R" class="bibLink tab-link" data-tab="pane-pcw-references">Agosti<i> et al.</i>, 1995</a></span>).
            </p>
            
            <p>The present study supported the use of nucleotides over 
amino acids for intraordinal phylogenies because, even though the 
topology within Diptera was largely the same for both data treatments, 
nodal support was higher for nucleotide datasets. This suggests that 
recoding nucleotide sequences as amino acids results in the loss of 
informative nucleotide variation, such as that due to silent 
substitutions at the third codon position. Concerns about third position
 saturation are probably overstated, as we saw little variation between 
the DNA12 and DNA123 datasets. Only once did inclusion of the third 
codon position result in an artefactual grouping, namely syrphids 
grouping within Schizophora in the ALL‐DNA123 dataset in the ML 
analysis, and even then this relationship received limited nodal 
support. By contrast, for interordinal relationships, data treatment 
appears to be very significant, resulting in differing relationships 
between the holometabolan orders for the different data treatments and, 
particularly, for the outgroup hemipteran<i> Triatoma</i>, which tended 
to nest deep inside Holometabola in some treatments. In general, 
however, the use of amino acid sequence for alignment balances concerns 
about nucleotide saturation against loss of signal. We have found that 
aligning protein coding genes directly in programs such as C<span class="smallCaps">lustal</span> results in very ‘gappy’ alignments and very poor trees (data not shown). The direct alignment of nucleotides using C<span class="smallCaps">lustal</span>X (<span title=""><a href="#b88" id="#b88R" class="bibLink tab-link" data-tab="pane-pcw-references">Thompson<i> et al.</i>, 1997</a></span>) has also been tried by <span title=""><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Bae<i> et al. </i>(2004)</a></span> and <span title=""><a href="#b46" id="#b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Kim<i> et al. </i>(2005)</a></span>.
 These papers reported unusual results relating to the early branching 
patterns [e.g. outgroup chelicerates and crustaceans within Insecta and 
Orthoptera + Holometabola to the exclusion of Paraneoptera (<span title=""><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Bae<i> et al.</i>, 2004</a></span>); and a polyphyletic Orthoptera with ensiferans forming the earliest branch within insects (<span title=""><a href="#b46" id="#b46R" class="bibLink tab-link" data-tab="pane-pcw-references">Kim<i> et al.</i>, 2005</a></span>)].
 This suggests that the approach of direct nucleotide alignments may be 
very sensitive to outgroup choice, particularly over long internodes. 
Interestingly, this mirrors the results of <span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Cameron<i> et al. </i>(2004)</a></span>
 relating to the influence of outgroup on deep‐level insect 
mitogenomics. Amino acid alignment with back‐translation to nucleotides 
produces the most reasonable results, but this procedure forces 
insertion–deletion events (indels) into neat three nucleotide sets, 
which is unrealistic biologically as it ignores the possibility of 
single or double nucleotide indels, both of which have been recorded (<span title=""><a href="#b9" id="#b9R" class="bibLink tab-link" data-tab="pane-pcw-references">Beckenbach<i> et al.</i>, 2005</a></span>) and inferred (<span title=""><a href="#b36" id="#b36R" class="bibLink tab-link" data-tab="pane-pcw-references">Grant &amp; D’Haese, 2004</a></span>) for mt genes.
            </p>
            
            <p>Although BA analysis is now a standard feature of mt phylogenomic studies of the insects (used in <span title=""><a href="#b65%20#b14%20#b16" id="#b65 #b14 #b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Nardi<i> et al.</i>, 2003a; Cameron<i> et al.</i>, 2004; Castro &amp; Dowton, 2005</a></span>),
 this is the first examination of the effect of data partitioning on BA 
analysis. Partitioning improves both the accuracy and speed of BA 
analyses, but over‐partitioning results in inaccuracies similar to those
 encountered in ML analysis in which models are over‐parameterized (e.g.
 <span title=""><a href="#b67" id="#b67R" class="bibLink tab-link" data-tab="pane-pcw-references">Nylander<i> et al.</i>, 2004</a></span>).
 It is also a potential area of concern because, as found here, the same
 dataset partitioned in different ways can support different topologies,
 with the differing nodes of each alternative topology supported by 
significant (0.9) to absolute (1.0) posterior probabilities. We examined
 three possible ways of partitioning the protein coding genes – CP, GP 
and CGP – resulting in three, thirteen and thirty‐nine partitions, 
respectively. Interestingly, differing partitioning strategies resulted 
in different topologies for the DNA123 datasets; however, these were 
only at the interordinal level and only significant for the ALL‐DNA123 
dataset. In addition, the trees from CP and CGP analyses were the same, 
whereas the GP trees differed for the DNA123 datasets, but all three 
analyses were basically the same for the DNA12 datasets. Allowing 
different parameter optimizations between genes was far less important 
than allowing them between codons, which is why analyses that included 
some form of codon partition were the same. The absence of this ‘codon 
effect’ for the DNA12 datasets shows that significant rate heterogeneity
 exists between first + second and third codon positions, which is to be
 expected given that the third codon position is typically mutationally 
fast as substitutions are usually silent. Although the partitioning 
strategy does not appear to have a significant effect on intraordinal 
topology, the highly significant differences in Bayes factors between 
the three approaches suggest that it is still important, especially if 
accurate calculation of branch lengths is important, such as for 
molecular clock calculations. From this perspective, partitioning by 
codon alone may not be enough. Partitioning by gene, however, has the 
potential drawback that some of the mt genes may be too small for 
accurate a priori determination of parameter values in programs such as M<span class="smallCaps">odel</span>T<span class="smallCaps">est</span> (it was for this reason that the tRNA genes were treated as a single partition in all analyses). For example, for<i> atp8</i>, M<span class="smallCaps">odel</span>T<span class="smallCaps">est</span>
 favoured the Jukes–Cantor model, an evolutionary model so simplistic as
 to be almost completely unrealistic, which may be the result of the 
small size of this gene (183 bp aligned, 61 bp for each codon 
partition). Therefore, it may be desirable to treat blocks of genes as 
partitions, particularly those that, in the vast majority of insect mt 
genomes, have partially overlapping coding regions, such as<i> atp8–atp6–cox3</i>,<i> nadh4L–nadh4 </i>and<i> nadh6–cytB</i>.
            </p>
            
         </div>
         
      </div>
      <div class="article-section__content" id="ss15">
         
         <h2 class="article-section__title section__title section1" id="ss15-title"> Conclusion</h2>
         
         <p>We have demonstrated the broad usefulness of mt genome data 
for resolving intraordinal relationships within insects by a pilot study
 of the best represented insect order, Diptera. We have also evaluated 
many of the analytical approaches that have been investigated for insect
 phylogenetic reconstruction, and can conclude that the following 
elements are important considerations in experimental/analytical design.</p>
         
         <p>
            </p><ul class="rlist hanging">
               
               <li><span class="number">1 </span>
                  
                  
                  <p>The effect of optimality criteria on topology and 
tree support should be assessed by performance of replicate analyses 
because, even for regions of topological congruence between methods, 
nodal support can vary widely.</p>
                  
               </li>
               
               <li><span class="number">2 </span>
                  
                  
                  <p>Analyses should use as much data as possible. There
 is no evidence of significant incongruence between mt genes and, if 
alignment of divergence sequences remains a concern, taxon exclusion is a
 better approach (e.g. see <span title=""><a href="#b16" id="#b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Castro &amp; Dowton, 2005</a></span> for the effects of excluding the highly divergent<i> Apis </i>and<i> Melipona </i>sequences).
                  </p>
                  
               </li>
               
               <li><span class="number">3 </span>
                  
                  
                  <p>The rRNA and tRNA genes should be included in 
future analyses of insect relationships. There is considerable 
phylogenetic signal in these genes, adding almost 25% extra data 
sequenced but discarded from most genomic studies. Difficulties with the
 alignment of the rRNA genes, particularly<i> l‐rrna</i>, can be improved by the adoption of more consistent functional genome annotation approaches.
                  </p>
                  
               </li>
               
               <li><span class="number">4 </span>
                  
                  
                  <p>Sequence recoding as amino acids is unnecessary, 
reduces the phylogenetic signal and results in artefactual 
relationships, placing outgroup sequences in the ingroup. Concerns about
 the saturation of third codon positions can be more readily addressed 
by the inclusion/exclusion of this data partition. Recoding sequences as
 amino acids is, however, very useful in the alignment phase of an 
analysis.</p>
                  
               </li>
               
               <li><span class="number">5 </span>
                  
                  
                  <p>BA analyses are extremely sensitive to partition 
strategy. Partitions based on codons give different results to those 
based on genes, but both are very well supported. Bayes factors suggest 
that codon‐based partitions are better than gene‐based partitions; 
however, over‐partitioning will probably result in poor parameter 
estimation by both M<span class="smallCaps">odel</span>T<span class="smallCaps">est</span> and M<span class="smallCaps">r</span>B<span class="smallCaps">ayes</span>.
                  </p>
                  
               </li>
               
            </ul>
         <p></p>
         
      </div>
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="ss16-title"> Acknowledgements</h2>
         
         <p>We thank Brian Wiegmann (North Carolina State University) 
and David Yeates (Australian National Insect Collection) for their 
advice and support of this research. Mark Dowton (University of 
Wollongong) and Joseph Gillespie (University of Maryland) provided 
valuable comments on the manuscript. This study was supported by NSF 
grants DEB0120718 (MFW) and DEB0444972 (SLC, MFW).</p>
         
         <ol></ol>
      </div>
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion">
            <h3>
               <div tabindex="0" role="button" class="accordion__control" aria-controls="idss17" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;pbContext=%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-3113.2006.00355.x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fsen.2007.32.issue-1%3BrequestedJournal%3Ajournal%3A13653113%3Bjournal%3Ajournal%3A13653113b%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h3&amp;onlyLog=true">
                  <h2 class="section__title" id="references-section-1">References</h2><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
            </h3>
            <div class="accordion__content" style="display: none;" role="
                            region
                        " aria-labelledby="
                            references-section-1" id="ss17">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="b1">
                     <span class="author">Agosti, D.</span>, 
<span class="author">Jacobs, D.</span> &amp; 
<span class="author">DeSalle, R</span>. (<span class="pubYear">1995</span>)  <span class="articleTitle">On combining protein sequences and nucleic acid sequences in phylogenetic analysis: the homeobox protein case</span>. 
<i>Cladistics</i>,  <span class="vol">12</span>,  <span class="pageFirst">65</span>– <span class="pageLast">82</span>.

                     <div class="extra-links" id="e_1_2_7_2_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.1996.tb00193.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1996WG61400005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=1995&amp;pages=65-82&amp;journal=Cladistics&amp;author=D.+Agosti&amp;author=D.+Jacobs&amp;author=R+DeSalle&amp;title=On+combining+protein+sequences+and+nucleic+acid+sequences+in+phylogenetic+analysis%3A+the+homeobox+protein+case" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b2">
                     <span class="author">Altschul, S.F.</span>, 
<span class="author">Madden, T.L.</span>, 
<span class="author">Schäffer, A.A.</span>, 
<span class="author">Zhang, J.</span>, 
<span class="author">Zhang, Z.</span>, 
<span class="author">Miller, W.</span> &amp; 
<span class="author">Lipman, D.J</span>. (<span class="pubYear">1997</span>)  <span class="articleTitle">Gapped BLAST and PSI‐BLAST: a new generation of protein database search programs</span>. 
<i>Nucleic Acids Research</i>,  <span class="vol">25</span>,  <span class="pageFirst">3389</span>– <span class="pageLast">3402</span>.

                     <div class="extra-links" id="e_1_2_7_3_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fnar%2F25.17.3389" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2sXlvFyhu7w%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=9254694" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1997XU79300002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=1997&amp;pages=3389-3402&amp;journal=Nucleic+Acids+Research&amp;author=S.F.+Altschul&amp;author=T.L.+Madden&amp;author=A.A.+Sch%C3%A4ffer&amp;author=J.+Zhang&amp;author=Z.+Zhang&amp;author=W.+Miller&amp;author=D.J+Lipman&amp;title=Gapped+BLAST+and+PSI%E2%80%90BLAST%3A+a+new+generation+of+protein+database+search+programs" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b3">
                     <span class="author">Bae, J.S.</span>, 
<span class="author">Kim, I.</span>, 
<span class="author">Sohn, H.D.</span> &amp; 
<span class="author">Jin, B.R</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">The mitochondrial genome of the firefly,<i> Pyrocoelia rufa</i>: complete DNA sequence, genome organization, and phylogenetic analysis with other insects</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">32</span>,  <span class="pageFirst">978</span>– <span class="pageLast">985</span>.

                     <div class="extra-links" id="e_1_2_7_4_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.ympev.2004.03.009" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmt1CntLg%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15288070" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000223329500025" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=32&amp;publication_year=2004&amp;pages=978-985&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=J.S.+Bae&amp;author=I.+Kim&amp;author=H.D.+Sohn&amp;author=B.R+Jin&amp;title=The+mitochondrial+genome+of+the+firefly%2C+Pyrocoelia+rufa%3A+complete+DNA+sequence%2C+genome+organization%2C+and+phylogenetic+analysis+with+other+insects" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b4">
                     <span class="author">Baker, R.H.</span> &amp; 
<span class="author">DeSalle, R</span>. (<span class="pubYear">1997</span>)  <span class="articleTitle">Multiple sources of character information and the phylogeny of the Hawaiian drosophilids</span>. 
<i>Systematic Biology</i>,  <span class="vol">46</span>,  <span class="pageFirst">654</span>– <span class="pageLast">673</span>.

                     <div class="extra-links" id="e_1_2_7_5_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fsysbio%2F46.4.654" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ASTN%3A280%3ADC%252BD383js12hsg%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11975337" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=46&amp;publication_year=1997&amp;pages=654-673&amp;journal=Systematic+Biology&amp;author=R.H.+Baker&amp;author=R+DeSalle&amp;title=Multiple+sources+of+character+information+and+the+phylogeny+of+the+Hawaiian+drosophilids" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b5">
                     <span class="author">Ballard, J.W.O</span>. (<span class="pubYear">2000a</span>)  <span class="articleTitle">Comparative genomics of mitochondrial DNA in members of the<i> Drosophila melanogaster </i>subgroup</span>. 
<i>Journal of Molecular Evolution</i>,  <span class="vol">51</span>,  <span class="pageFirst">48</span>– <span class="pageLast">63</span>.

                     <div class="extra-links" id="e_1_2_7_6_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2Fs002390010066" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlslyls7s%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10903372" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000088151700006" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2000a&amp;pages=48-63&amp;journal=Journal+of+Molecular+Evolution&amp;author=J.W.O+Ballard&amp;title=Comparative+genomics+of+mitochondrial+DNA+in+members+of+the+Drosophila+melanogaster+subgroup" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b6">
                     <span class="author">Ballard, J.W.O</span>. (<span class="pubYear">2000b</span>)  <span class="articleTitle">Comparative genomics of mitochondrial DNA in<i> Drosophila simulans</i></span>. 
<i>Journal of Molecular Evolution</i>,  <span class="vol">51</span>,  <span class="pageFirst">64</span>– <span class="pageLast">75</span>.

                     <div class="extra-links" id="e_1_2_7_7_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2Fs002390010067" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlslyls7g%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10903373" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000088151700007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2000b&amp;pages=64-75&amp;journal=Journal+of+Molecular+Evolution&amp;author=J.W.O+Ballard&amp;title=Comparative+genomics+of+mitochondrial+DNA+in+Drosophila+simulans" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b7">
                     <span class="author">Ballard, J.W.O.</span> &amp; 
<span class="author">Rand, D.M</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">The population biology of mitochondrial DNA and its phylogenetic implications</span>. 
<i>Annual Review of Ecology, Evolution and Systematics</i>,  <span class="vol">36</span>,  <span class="pageFirst">621</span>– <span class="pageLast">642</span>.

                     <div class="extra-links" id="e_1_2_7_8_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1146%2Fannurev.ecolsys.36.091704.175513" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000234684900026" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=2005&amp;pages=621-642&amp;journal=Annual+Review+of+Ecology%2C+Evolution+and+Systematics&amp;author=J.W.O.+Ballard&amp;author=D.M+Rand&amp;title=The+population+biology+of+mitochondrial+DNA+and+its+phylogenetic+implications" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b8">
                     <span class="author">Beard, C.B.</span>, 
<span class="author">Hamm, D.M.</span> &amp; 
<span class="author">Collins, F.H</span>. (<span class="pubYear">1993</span>)  <span class="articleTitle">The mitochondrial genome of the mosquito<i> Anopheles gambiae</i>: DNA sequence, genome organisation and comparisons with mitochondrial sequences of other insects</span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">2</span>,  <span class="pageFirst">103</span>– <span class="pageLast">124</span>.

                     <div class="extra-links" id="e_1_2_7_9_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2583.1993.tb00131.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2cXitV2rsL8%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=9087549" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=2&amp;publication_year=1993&amp;pages=103-124&amp;journal=Insect+Molecular+Biology&amp;author=C.B.+Beard&amp;author=D.M.+Hamm&amp;author=F.H+Collins&amp;title=The+mitochondrial+genome+of+the+mosquito+Anopheles+gambiae%3A+DNA+sequence%2C+genome+organisation+and+comparisons+with+mitochondrial+sequences+of+other+insects" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b9">
                     <span class="author">Beckenbach, A.T.</span>, 
<span class="author">Robson, S.K.A.</span> &amp; 
<span class="author">Crozier, R.H</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">Single nucleotide + 1 frameshifts in an apparently functional mitochondrial cytochrome b gene in ants of the genus<i> Polyrhachis</i></span>. 
<i>Journal of Molecular Evolution</i>,  <span class="vol">60</span>,  <span class="pageFirst">141</span>– <span class="pageLast">152</span>.

                     <div class="extra-links" id="e_1_2_7_10_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2Fs00239-004-0178-5" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXit1SntLY%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15785844" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000227070100001" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=60&amp;publication_year=2005&amp;pages=141-152&amp;journal=Journal+of+Molecular+Evolution&amp;author=A.T.+Beckenbach&amp;author=S.K.A.+Robson&amp;author=R.H+Crozier&amp;title=Single+nucleotide+%2B+1+frameshifts+in+an+apparently+functional+mitochondrial+cytochrome+b+gene+in+ants+of+the+genus+Polyrhachis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b10">
                     <span class="author">Blagoderov, V.A.</span>, 
<span class="author">Lukashevich, E.D.</span> &amp; 
<span class="author">Mostovski, M.D</span>. (<span class="pubYear">2002</span>)  <span class="chapterTitle">Order Diptera Linné, 1758</span>. 
 <span class="bookTitle">History of Insects</span> (ed. by
					 <span class="editor">A. P. Rasnitsyn</span> &amp; 
 <span class="editor">D. L. J. Quicke</span>), pp.  <span class="pageFirst">227</span>– <span class="pageLast">240</span>.  Kluwer Academic Publishers, Dordrecht.

                     <div class="extra-links" id="e_1_2_7_11_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2002&amp;pages=227-240&amp;author=V.A.+Blagoderov&amp;author=E.D.+Lukashevich&amp;author=M.D+Mostovski&amp;title=History+of+Insects" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b11">
                     <span class="author">Boore, J.L</span>. (<span class="pubYear">1999</span>)  <span class="articleTitle">Animal mitochondrial genomes</span>. 
<i>Nucleic Acids Research</i>,  <span class="vol">27</span>,  <span class="pageFirst">1767</span>– <span class="pageLast">1780</span>.

                     <div class="extra-links" id="e_1_2_7_12_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fnar%2F27.8.1767" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK1MXivVersbo%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10101183" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000079966900001" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=27&amp;publication_year=1999&amp;pages=1767-1780&amp;journal=Nucleic+Acids+Research&amp;author=J.L+Boore&amp;title=Animal+mitochondrial+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b12">
                     <span class="author">Boore, J.L.</span> &amp; 
<span class="author">Brown, W.M</span>. (<span class="pubYear">1998</span>)  <span class="articleTitle">Big trees from little genomes: mitochondrial gene order as a phylogenetic tool</span>. 
<i>Current Opinion in Genetics and Development</i>,  <span class="vol">8</span>,  <span class="pageFirst">668</span>– <span class="pageLast">674</span>.

                     <div class="extra-links" id="e_1_2_7_13_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS0959-437X%2898%2980035-X" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK1MXhs1yisQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=9914213" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000077614100009" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=1998&amp;pages=668-674&amp;journal=Current+Opinion+in+Genetics+and+Development&amp;author=J.L.+Boore&amp;author=W.M+Brown&amp;title=Big+trees+from+little+genomes%3A+mitochondrial+gene+order+as+a+phylogenetic+tool" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b14">
                     <span class="author">Cameron, S.L.</span>, 
<span class="author">Miller, K.B.</span>, 
<span class="author">D’Haese, C.A.</span>, 
<span class="author">Whiting, M.F.</span> &amp; 
<span class="author">Barker, S.C</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Mitochondrial genome data alone are not enough to unambiguously resolve the relationships of Entognatha, Insecta and Crustacea<i> sensu lato </i>(Arthropoda)</span>. 
<i>Cladistics</i>,  <span class="vol">20</span>,  <span class="pageFirst">534</span>– <span class="pageLast">557</span>.

                     <div class="extra-links" id="e_1_2_7_14_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.2004.00040.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000226965400003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2004&amp;pages=534-557&amp;journal=Cladistics&amp;author=S.L.+Cameron&amp;author=K.B.+Miller&amp;author=C.A.+D%E2%80%99Haese&amp;author=M.F.+Whiting&amp;author=S.C+Barker&amp;title=Mitochondrial+genome+data+alone+are+not+enough+to+unambiguously+resolve+the+relationships+of+Entognatha%2C+Insecta+and+Crustacea+sensu+lato+%28Arthropoda%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b15">
                     <span class="author">Cameron, S.L.</span>, 
<span class="author">Barker, S.C.</span> &amp; 
<span class="author">Whiting, M.F</span>. (<span class="pubYear">2006</span>)  <span class="articleTitle">Mitochondrial genomics and the new insect order Mantophasmatodea</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">38</span>,  <span class="pageFirst">274</span>– <span class="pageLast">279</span>.

                     <div class="extra-links" id="e_1_2_7_15_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.ympev.2005.09.020" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlSiu7jE" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=16321547" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000234461700023" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=2006&amp;pages=274-279&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=S.L.+Cameron&amp;author=S.C.+Barker&amp;author=M.F+Whiting&amp;title=Mitochondrial+genomics+and+the+new+insect+order+Mantophasmatodea" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b16">
                     <span class="author">Castro, L.R.</span> &amp; 
<span class="author">Dowton, M</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">The position of the Hymenoptera within the Holometabola as inferred from the mitochondrial genome of<i> Perga condei </i>(Hymenoptera: Symphyta: Pergidae)</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">34</span>,  <span class="pageFirst">469</span>– <span class="pageLast">479</span>.

                     <div class="extra-links" id="e_1_2_7_16_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.ympev.2004.11.005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXptlCisw%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15683922" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000227031600002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2005&amp;pages=469-479&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=L.R.+Castro&amp;author=M+Dowton&amp;title=The+position+of+the+Hymenoptera+within+the+Holometabola+as+inferred+from+the+mitochondrial+genome+of+Perga+condei+%28Hymenoptera%3A+Symphyta%3A+Pergidae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b17">
                     <span class="author">Caterino, M.S.</span>, 
<span class="author">Cho, S.</span> &amp; 
<span class="author">Sperling, F.A.H</span>. (<span class="pubYear">2000</span>)  <span class="articleTitle">The current state of insect molecular systematics: a thriving Tower of Babel</span>. 
<i>Annual Review of Entomology</i>,  <span class="vol">45</span>,  <span class="pageFirst">1</span>– <span class="pageLast">54</span>.

                     <div class="extra-links" id="e_1_2_7_17_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1146%2Fannurev.ento.45.1.1" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=45&amp;publication_year=2000&amp;pages=1-54&amp;journal=Annual+Review+of+Entomology&amp;author=M.S.+Caterino&amp;author=S.+Cho&amp;author=F.A.H+Sperling&amp;title=The+current+state+of+insect+molecular+systematics%3A+a+thriving+Tower+of+Babel" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b18">
                     <span class="author">Clary, D.O.</span> &amp; 
<span class="author">Wolstenholme, D.R</span>. (<span class="pubYear">1985</span>)  <span class="articleTitle">The mitochondrial DNA molecule of<i> Drosophila yakuba</i>: nucleotide sequence, gene organization, and genetic code</span>. 
<i>Journal of Molecular Evolution</i>,  <span class="vol">22</span>,  <span class="pageFirst">252</span>– <span class="pageLast">271</span>.

                     <div class="extra-links" id="e_1_2_7_18_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2FBF02099755" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaL28Xht1eqtb4%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=3001325" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1985AUU2900007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=1985&amp;pages=252-271&amp;journal=Journal+of+Molecular+Evolution&amp;author=D.O.+Clary&amp;author=D.R+Wolstenholme&amp;title=The+mitochondrial+DNA+molecule+of+Drosophila+yakuba%3A+nucleotide+sequence%2C+gene+organization%2C+and+genetic+code" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b19">
                     <span class="author">Clayton, D.A</span>. (<span class="pubYear">1992</span>)  <span class="articleTitle">Transcription and replication of animal mitochondrial DNAs</span>. 
<i>International Review of Cytology</i>,  <span class="vol">141</span>,  <span class="pageFirst">217</span>– <span class="pageLast">232</span>.

                     <div class="extra-links" id="e_1_2_7_19_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS0074-7696%2808%2962067-7" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK3sXkt1Wgs7w%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1452432" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1992ME11200007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=141&amp;publication_year=1992&amp;pages=217-232&amp;journal=International+Review+of+Cytology&amp;author=D.A+Clayton&amp;title=Transcription+and+replication+of+animal+mitochondrial+DNAs" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b20">
                     <span class="author">Coates, B.S.</span>, 
<span class="author">Sumerford, D.V.</span>, 
<span class="author">Hellmich, R.L.</span> &amp; 
<span class="author">Lewis, L.C</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">Partial mitochondrial genome sequences of<i> Ostrinia nubialis </i>and<i> Ostrinia furnicalis</i></span>. 
<i>International Journal of Biology Sciences</i>,  <span class="vol">1</span>,  <span class="pageFirst">13</span>– <span class="pageLast">18</span>.

                     <div class="extra-links" id="e_1_2_7_20_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.7150%2Fijbs.1.13" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjs1eltLs%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15951845" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000209398500002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=2005&amp;pages=13-18&amp;journal=International+Journal+of+Biology+Sciences&amp;author=B.S.+Coates&amp;author=D.V.+Sumerford&amp;author=R.L.+Hellmich&amp;author=L.C+Lewis&amp;title=Partial+mitochondrial+genome+sequences+of+Ostrinia+nubialis+and+Ostrinia+furnicalis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b21">
                     <span class="author">Colless, D.H.</span> &amp; 
<span class="author">McAlpine, D.K</span>. (<span class="pubYear">1991</span>)  <span class="chapterTitle">Diptera</span>. 
 <span class="bookTitle">The Insects of Australia</span> (ed. by CSIRO), pp.  <span class="pageFirst">717</span>– <span class="pageLast">786</span>.  Melbourne University Press, Melbourne.

                     <div class="extra-links" id="e_1_2_7_21_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1991&amp;pages=717-786&amp;author=D.H.+Colless&amp;author=D.K+McAlpine&amp;title=The+Insects+of+Australia" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b22">
                     <span class="author">Collins, K.P.</span> &amp; 
<span class="author">Wiegmann, B.M</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Phylogenetic relationships of the lower Cyclorrhapha (Diptera: Brachycera) based on 28S rDNA sequences</span>. 
<i>Insect Systematics and Evolution</i>,  <span class="vol">33</span>,  <span class="pageFirst">445</span>– <span class="pageLast">456</span>.

                     <div class="extra-links" id="e_1_2_7_22_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2002&amp;pages=445-456&amp;journal=Insect+Systematics+and+Evolution&amp;author=K.P.+Collins&amp;author=B.M+Wiegmann&amp;title=Phylogenetic+relationships+of+the+lower+Cyclorrhapha+%28Diptera%3A+Brachycera%29+based+on+28S+rDNA+sequences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b23">
                     <span class="author">Covacin, C.</span>, 
<span class="author">Shao, R.</span>, 
<span class="author">Cameron, S.L.</span> &amp; 
<span class="author">Barker, S.C</span>. (<span class="pubYear">2006</span>)  <span class="articleTitle">Extraordinary amounts of gene rearrangement in the mitochondrial genomes of lice (Insecta: Phthiraptera)</span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">15</span>,  <span class="pageFirst">63</span>– <span class="pageLast">68</span>.

                     <div class="extra-links" id="e_1_2_7_23_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2583.2005.00608.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD28XislKksb0%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=16469069" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000235256500007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2006&amp;pages=63-68&amp;journal=Insect+Molecular+Biology&amp;author=C.+Covacin&amp;author=R.+Shao&amp;author=S.L.+Cameron&amp;author=S.C+Barker&amp;title=Extraordinary+amounts+of+gene+rearrangement+in+the+mitochondrial+genomes+of+lice+%28Insecta%3A+Phthiraptera%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b24">
                     <span class="author">Crozier, R.H.</span> &amp; 
<span class="author">Crozier, Y.C</span>. (<span class="pubYear">1993</span>)  <span class="articleTitle">The mitochondrial genome of the honeybee<i> Apis mellifera</i>: complete sequence and genome organisation</span>. 
<i>Genetics</i>,  <span class="vol">133</span>,  <span class="pageFirst">97</span>– <span class="pageLast">117</span>.

                     <div class="extra-links" id="e_1_2_7_24_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK3sXisVyrtrc%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=8417993" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1993KE78300011" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=133&amp;publication_year=1993&amp;pages=97-117&amp;journal=Genetics&amp;author=R.H.+Crozier&amp;author=Y.C+Crozier&amp;title=The+mitochondrial+genome+of+the+honeybee+Apis+mellifera%3A+complete+sequence+and+genome+organisation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b25">
                     <span class="author">Delsuc, F.</span>, 
<span class="author">Phillips, M.J.</span> &amp; 
<span class="author">Penny, D</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">Comment on ‘Hexapod origins: Monophyletic or Paraphyletic’</span>. 
<i>Science</i>,  <span class="vol">301</span>, 1482d.

                     <div class="extra-links" id="e_1_2_7_25_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1126%2Fscience.1086558" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1KqtLY%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12970547" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000185255300028" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=301&amp;publication_year=2003&amp;journal=Science&amp;author=F.+Delsuc&amp;author=M.J.+Phillips&amp;author=D+Penny&amp;title=Comment+on+%E2%80%98Hexapod+origins%3A+Monophyletic+or+Paraphyletic%E2%80%99" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b26">
                     <span class="author">Dotson, E.M.</span> &amp; 
<span class="author">Beard, C.B</span>. (<span class="pubYear">2001</span>)  <span class="articleTitle">Sequence and organization of the mitochondrial genome of the Chagas disease vector,<i> Triatoma dimidiata</i></span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">10</span>,  <span class="pageFirst">205</span>– <span class="pageLast">215</span>.

                     <div class="extra-links" id="e_1_2_7_26_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2583.2001.00258.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlt1ahtbc%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11437912" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000169474600003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=10&amp;publication_year=2001&amp;pages=205-215&amp;journal=Insect+Molecular+Biology&amp;author=E.M.+Dotson&amp;author=C.B+Beard&amp;title=Sequence+and+organization+of+the+mitochondrial+genome+of+the+Chagas+disease+vector%2C+Triatoma+dimidiata" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b27">
                     <span class="author">Dowton, M.</span> &amp; 
<span class="author">Austin, A.D</span>. (<span class="pubYear">1999</span>)  <span class="articleTitle">Evolutionary dynamics of a mitochondrial rearrangement ‘hotspot’ in the Hymenoptera</span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">16</span>,  <span class="pageFirst">298</span>– <span class="pageLast">309</span>.

                     <div class="extra-links" id="e_1_2_7_27_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Foxfordjournals.molbev.a026111" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK1MXht1elt7k%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10028295" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000078518600015" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1999&amp;pages=298-309&amp;journal=Molecular+Biology+and+Evolution&amp;author=M.+Dowton&amp;author=A.D+Austin&amp;title=Evolutionary+dynamics+of+a+mitochondrial+rearrangement+%E2%80%98hotspot%E2%80%99+in+the+Hymenoptera" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b28">
                     <span class="author">Dowton, M.</span>, 
<span class="author">Castro, L.R.</span> &amp; 
<span class="author">Austin, A.D</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Mitochondrial gene rearrangements as phylogenetic characters in the invertebrates: the examination of genome ‘morphology’</span>. 
<i>Invertebrate Systematics</i>,  <span class="vol">16</span>,  <span class="pageFirst">345</span>– <span class="pageLast">356</span>.

                     <div class="extra-links" id="e_1_2_7_28_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1071%2FIS02003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000176970100001" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2002&amp;pages=345-356&amp;journal=Invertebrate+Systematics&amp;author=M.+Dowton&amp;author=L.R.+Castro&amp;author=A.D+Austin&amp;title=Mitochondrial+gene+rearrangements+as+phylogenetic+characters+in+the+invertebrates%3A+the+examination+of+genome+%E2%80%98morphology%E2%80%99" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b29">
                     <span class="author">Dowton, M.</span>, 
<span class="author">Castro, L.R.</span>, 
<span class="author">Campbell, S.L.</span>, 
<span class="author">Bargon, S.D.</span> &amp; 
<span class="author">Austin, A.D</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">Frequent mitochondrial gene rearrangements at the hymenopteran nad3nad5 junction</span>. 
<i>Journal of Molecular Evolution</i>,  <span class="vol">56</span>,  <span class="pageFirst">517</span>– <span class="pageLast">526</span>.

                     <div class="extra-links" id="e_1_2_7_29_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2Fs00239-002-2420-3" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXivFygsro%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12698290" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000182065300001" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=56&amp;publication_year=2003&amp;pages=517-526&amp;journal=Journal+of+Molecular+Evolution&amp;author=M.+Dowton&amp;author=L.R.+Castro&amp;author=S.L.+Campbell&amp;author=S.D.+Bargon&amp;author=A.D+Austin&amp;title=Frequent+mitochondrial+gene+rearrangements+at+the+hymenopteran+nad3nad5+junction" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b30">
                     <span class="author">Flook, P.K.</span>, 
<span class="author">Rowell, C.H.F.</span> &amp; 
<span class="author">Gellissen, G</span>. (<span class="pubYear">1995a</span>)  <span class="articleTitle">The sequence, organisation and evolution of the<i> Locusta migratoria </i>mitochondrial genome</span>. 
<i>Journal of Molecular Evolution</i>,  <span class="vol">41</span>,  <span class="pageFirst">928</span>– <span class="pageLast">941</span>.

                     <div class="extra-links" id="e_1_2_7_30_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2FBF00173173" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK28XivFOhsg%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=8587138" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1995TM93000030" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=41&amp;publication_year=1995a&amp;pages=928-941&amp;journal=Journal+of+Molecular+Evolution&amp;author=P.K.+Flook&amp;author=C.H.F.+Rowell&amp;author=G+Gellissen&amp;title=The+sequence%2C+organisation+and+evolution+of+the+Locusta+migratoria+mitochondrial+genome" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b31">
                     <span class="author">Flook, P.K.</span>, 
<span class="author">Rowell, C.H.F.</span> &amp; 
<span class="author">Gellissen, G</span>. (<span class="pubYear">1995b</span>)  <span class="articleTitle">Homoplastic rearrangements of insect mitochondrial tRNA genes</span>. 
<i>Naturwissenschaften</i>,  <span class="vol">82</span>,  <span class="pageFirst">336</span>– <span class="pageLast">337</span>.

                     <div class="extra-links" id="e_1_2_7_31_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2FBF01131531" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2MXntFKit7w%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1995RL35100006" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=82&amp;publication_year=1995b&amp;pages=336-337&amp;journal=Naturwissenschaften&amp;author=P.K.+Flook&amp;author=C.H.F.+Rowell&amp;author=G+Gellissen&amp;title=Homoplastic+rearrangements+of+insect+mitochondrial+tRNA+genes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b32">
                     <span class="author">Friedrich, M.</span> &amp; 
<span class="author">Muqim, N</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">Sequence and phylogenetic analysis of the complete mitochondrial genome of the flour beetle<i> Tribolium castanaeum</i></span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">26</span>,  <span class="pageFirst">502</span>– <span class="pageLast">512</span>.

                     <div class="extra-links" id="e_1_2_7_32_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS1055-7903%2802%2900335-4" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitVKntLY%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12644407" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000181885500013" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2003&amp;pages=502-512&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=M.+Friedrich&amp;author=N+Muqim&amp;title=Sequence+and+phylogenetic+analysis+of+the+complete+mitochondrial+genome+of+the+flour+beetle+Tribolium+castanaeum" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b33">
                     <span class="author">Gatesy, J.</span>, 
<span class="author">O’Grady, P.</span> &amp; 
<span class="author">Baker, R</span>. (<span class="pubYear">1999</span>)  <span class="articleTitle">Corroboration
 among data sets in simultaneous analysis: hidden support for 
phylogenetic relationships among higher level artiodactyl taxa</span>. 
<i>Cladistics</i>,  <span class="vol">15</span>,  <span class="pageFirst">271</span>– <span class="pageLast">313</span>.

                     <div class="extra-links" id="e_1_2_7_33_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.1999.tb00268.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000083723200005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=1999&amp;pages=271-313&amp;journal=Cladistics&amp;author=J.+Gatesy&amp;author=P.+O%E2%80%99Grady&amp;author=R+Baker&amp;title=Corroboration+among+data+sets+in+simultaneous+analysis%3A+hidden+support+for+phylogenetic+relationships+among+higher+level+artiodactyl+taxa" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b34">
                     <span class="author">Gillespie, J.J</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Characterizing
 regions of ambiguous alignment caused by the expansion and contraction 
of hairpin‐stem loops in ribosomal RNA molecules</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">33</span>,  <span class="pageFirst">936</span>– <span class="pageLast">943</span>.

                     <div class="extra-links" id="e_1_2_7_34_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.ympev.2004.08.004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpt1Wjtrc%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15522814" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000225206700030" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=33&amp;publication_year=2004&amp;pages=936-943&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=J.J+Gillespie&amp;title=Characterizing+regions+of+ambiguous+alignment+caused+by+the+expansion+and+contraction+of+hairpin%E2%80%90stem+loops+in+ribosomal+RNA+molecules" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b35">
                     <span class="author">Gillespie, J.J.</span>, 
<span class="author">Johnston, J.S.</span>, 
<span class="author">Cannone, J.J.</span> &amp; 
<span class="author">Gutell, R.R</span>. (<span class="pubYear">2006</span>)  <span class="articleTitle">Characteristics of the nuclear (18S, 5.8S, 28S, and 5S) and mitochondrial (12S and 16S) rDNA genes of</span> 
                     <i>Apis mellifera</i> 
(Insecta: Hymenoptera): Structure, organization, and retrotransposition.<i> Insect Molecular Biology</i> (in press).
                     <div class="extra-links" id="e_1_2_7_35_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2006&amp;journal=Apis+mellifera&amp;author=J.J.+Gillespie&amp;author=J.S.+Johnston&amp;author=J.J.+Cannone&amp;author=R.R+Gutell&amp;title=Characteristics+of+the+nuclear+%2818S%2C+5.8S%2C+28S%2C+and+5S%29+and+mitochondrial+%2812S+and+16S%29+rDNA+genes+of" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b36">
                     <span class="author">Grant, T.</span> &amp; 
<span class="author">D’Haese, C.A</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Insertions and deletions in the evolution of equal‐length DNA fragments</span>. 
<i>Cladistics</i>,  <span class="vol">20</span>,  <span class="pageFirst">84</span>.

                     <div class="extra-links" id="e_1_2_7_36_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000220447100038" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2004&amp;pages=84&amp;journal=Cladistics&amp;author=T.+Grant&amp;author=C.A+D%E2%80%99Haese&amp;title=Insertions+and+deletions+in+the+evolution+of+equal%E2%80%90length+DNA+fragments" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b37">
                     <span class="author">Griffiths, G.C.D</span>. (<span class="pubYear">1972</span>)  <span class="bookTitle">The Phylogenetic Classification of Diptera Cyclorrhapha with Special Reference to the Structure of the Male Postabdomen</span>.  <span class="editor">W. Junk</span>
                     					 N.V. Publishers, The Hague.

                     <div class="extra-links" id="e_1_2_7_37_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1972&amp;author=G.C.D+Griffiths&amp;title=The+Phylogenetic+Classification+of+Diptera+Cyclorrhapha+with+Special+Reference+to+the+Structure+of+the+Male+Postabdomen" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b38">
                     <span class="author">Griffiths, G.C.D</span>. (<span class="pubYear">1994</span>)  <span class="articleTitle">Relationships among the major subgroups of Brachycera (Diptera): a critical review</span>. 
<i>Canadian Entomologist</i>,  <span class="vol">126</span>,  <span class="pageFirst">861</span>– <span class="pageLast">880</span>.

                     <div class="extra-links" id="e_1_2_7_38_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.4039%2FEnt126861-3" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1994PC45600022" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=126&amp;publication_year=1994&amp;pages=861-880&amp;journal=Canadian+Entomologist&amp;author=G.C.D+Griffiths&amp;title=Relationships+among+the+major+subgroups+of+Brachycera+%28Diptera%29%3A+a+critical+review" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b39">
                     <span class="author">Grimaldi, D.A</span>. (<span class="pubYear">1990</span>)  <span class="articleTitle">A phylogenetic, revised classification of genera in the Drosophilidae (Diptera)</span>. 
<i>Bulletin of the American Museum of Natural History</i>,  <span class="vol">197</span>,  <span class="pageFirst">1</span>– <span class="pageLast">139</span>.

                     <div class="extra-links" id="e_1_2_7_39_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=197&amp;publication_year=1990&amp;pages=1-139&amp;journal=Bulletin+of+the+American+Museum+of+Natural+History&amp;author=D.A+Grimaldi&amp;title=A+phylogenetic%2C+revised+classification+of+genera+in+the+Drosophilidae+%28Diptera%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b40">
                     <span class="author">Han, H.‐Y.</span> &amp; 
<span class="author">Ro, K.‐E.</span> (<span class="pubYear">2005</span>)  <span class="articleTitle">Molecular
 phylogeny of the superfamily Tephritoidea (Insecta: Diptera): new 
evidence from the mitochondrial 12S, 16S and COII genes</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">34</span>,  <span class="pageFirst">416</span>– <span class="pageLast">430</span>.

                     <div class="extra-links" id="e_1_2_7_40_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.ympev.2004.10.017" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtFahs7zE" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15619452" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000226258600014" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=34&amp;publication_year=2005&amp;pages=416-430&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=H.%E2%80%90Y.+Han&amp;author=K.%E2%80%90E.+Ro&amp;title=Molecular+phylogeny+of+the+superfamily+Tephritoidea+%28Insecta%3A+Diptera%29%3A+new+evidence+from+the+mitochondrial+12S%2C+16S+and+COII+genes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b41">
                     <span class="author">Hennig, W</span>. (<span class="pubYear">1958</span>)  <span class="articleTitle">Die Familien der Diptera Schizophora und ihre phylogenetischen Verwandtschaftsbeziehungen</span>. 
<i>Beiträge zur Entomologie</i>,  <span class="vol">8</span>,  <span class="pageFirst">505</span>– <span class="pageLast">688</span>.

                     <div class="extra-links" id="e_1_2_7_41_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=8&amp;publication_year=1958&amp;pages=505-688&amp;journal=Beitr%C3%A4ge+zur+Entomologie&amp;author=W+Hennig&amp;title=Die+Familien+der+Diptera+Schizophora+und+ihre+phylogenetischen+Verwandtschaftsbeziehungen" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b42">
                     <span class="author">Hillis, D.M.</span> &amp; 
<span class="author">Bull, J.J</span>. (<span class="pubYear">1993</span>)  <span class="articleTitle">An empirical test of bootstrapping as a method for assessing confidence in phylogenetic analysis</span>. 
<i>Systematic Biology</i>,  <span class="vol">42</span>,  <span class="pageFirst">182</span>– <span class="pageLast">192</span>.

                     <div class="extra-links" id="e_1_2_7_42_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fsysbio%2F42.2.182" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1993LF55800006" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=42&amp;publication_year=1993&amp;pages=182-192&amp;journal=Systematic+Biology&amp;author=D.M.+Hillis&amp;author=J.J+Bull&amp;title=An+empirical+test+of+bootstrapping+as+a+method+for+assessing+confidence+in+phylogenetic+analysis" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b43">
                     <span class="author">Huelsenbeck, J.P.</span> &amp; 
<span class="author">Ronquist, F.R</span>. (<span class="pubYear">2001</span>)  <span class="articleTitle">MrBayes: Bayesian inference of phylogeny</span>. 
<i>Biometrics</i>,  <span class="vol">17</span>,  <span class="pageFirst">754</span>– <span class="pageLast">755</span>.

                     <div class="extra-links" id="e_1_2_7_43_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ASTN%3A280%3ADC%252BD3MvotV2isw%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=2001&amp;pages=754-755&amp;journal=Biometrics&amp;author=J.P.+Huelsenbeck&amp;author=F.R+Ronquist&amp;title=MrBayes%3A+Bayesian+inference+of+phylogeny" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b44">
                     <span class="author">Inagaki, Y.</span>, 
<span class="author">Simpson, A.G.B.</span>, 
<span class="author">Dacks, J.B.</span> &amp; 
<span class="author">Rojer, A.J</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Phylogenetic
 artifacts can be caused by leucine, serine and arginine codon usage 
heterogeneity: dinoflagellate plastid origins as a case study</span>. 
<i>Systematic Biology</i>,  <span class="vol">53</span>,  <span class="pageFirst">582</span>– <span class="pageLast">593</span>.

                     <div class="extra-links" id="e_1_2_7_44_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1080%2F10635150490468756" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15371248" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000223598500006" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=53&amp;publication_year=2004&amp;pages=582-593&amp;journal=Systematic+Biology&amp;author=Y.+Inagaki&amp;author=A.G.B.+Simpson&amp;author=J.B.+Dacks&amp;author=A.J+Rojer&amp;title=Phylogenetic+artifacts+can+be+caused+by+leucine%2C+serine+and+arginine+codon+usage+heterogeneity%3A+dinoflagellate+plastid+origins+as+a+case+study" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b45">
                     <span class="author">Junqueira, A.C.M.</span>, 
<span class="author">Lessinger, A.C.</span>, 
<span class="author">Torres, T.T.</span>, 
<span class="author">Rodrigues da Silva, F.</span>, 
<span class="author">Vettore, A.L.</span>, 
<span class="author">Arruda, P.</span> &amp; 
<span class="author">Azeredo Espin, A.M.L</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">The mitochondrial genome of the blowfly<i> Chrysomya chloropyga </i>(Diptera: Calliphoridae)</span>. 
<i>Gene</i>,  <span class="vol">339</span>,  <span class="pageFirst">7</span>– <span class="pageLast">15</span>.

                     <div class="extra-links" id="e_1_2_7_45_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.gene.2004.06.031" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsVKjtb8%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15363841" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000224173200002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=339&amp;publication_year=2004&amp;pages=7-15&amp;journal=Gene&amp;author=A.C.M.+Junqueira&amp;author=A.C.+Lessinger&amp;author=T.T.+Torres&amp;author=F.+Rodrigues+da+Silva&amp;author=A.L.+Vettore&amp;author=P.+Arruda&amp;author=A.M.L+Azeredo+Espin&amp;title=The+mitochondrial+genome+of+the+blowfly+Chrysomya+chloropyga+%28Diptera%3A+Calliphoridae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b46">
                     <span class="author">Kim, I.</span>, 
<span class="author">Cha, S.Y.</span>, 
<span class="author">Yoon, M.H.</span>, 
<span class="author">Hwang, J.S.</span>, 
<span class="author">Lee, S.M.</span>, 
<span class="author">Sohn, H.D.</span> &amp; 
<span class="author">Jin, B.R</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">The complete nucleotide sequence and gene organisation of the mitochondrial genome of the oriental mole cricket.<i> Gryllotalpa orientalis </i>(Orthoptera: Gryllotalpidae)</span>. 
<i>Gene</i>,  <span class="vol">353</span>,  <span class="pageFirst">155</span>– <span class="pageLast">168</span>.

                     <div class="extra-links" id="e_1_2_7_46_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.gene.2005.04.019" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvV2ntbo%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15950403" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000230616600002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=353&amp;publication_year=2005&amp;pages=155-168&amp;journal=Gene&amp;author=I.+Kim&amp;author=S.Y.+Cha&amp;author=M.H.+Yoon&amp;author=J.S.+Hwang&amp;author=S.M.+Lee&amp;author=H.D.+Sohn&amp;author=B.R+Jin&amp;title=The+complete+nucleotide+sequence+and+gene+organisation+of+the+mitochondrial+genome+of+the+oriental+mole+cricket.+Gryllotalpa+orientalis+%28Orthoptera%3A+Gryllotalpidae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b47">
                     <span class="author">Kluge, A.G</span>. (<span class="pubYear">1989</span>)  <span class="articleTitle">A concern for evidence and a phylogenetic hypothesis of relationships among<i> Epicrates </i>(Boidae: Serpentes)</span>. 
<i>Systematic Zoology</i>,  <span class="vol">38</span>,  <span class="pageFirst">7</span>– <span class="pageLast">25</span>.

                     <div class="extra-links" id="e_1_2_7_47_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.2307%2F2992432" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1989U086400002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=38&amp;publication_year=1989&amp;pages=7-25&amp;journal=Systematic+Zoology&amp;author=A.G+Kluge&amp;title=A+concern+for+evidence+and+a+phylogenetic+hypothesis+of+relationships+among+Epicrates+%28Boidae%3A+Serpentes%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b101">
                     <span class="author">Kristensen, N.P.</span> &amp; 
<span class="author">Skalski, A.W</span>. (<span class="pubYear">1999</span>)  <span class="otherTitle">Phylogeny and palaeontology. Handbuch der Zoologie Band IV Arthropoda: Insecta Part 35 Lepidoptera, Moths and Butterflies</span> (ed. by
					 <span class="editor">N. P. Kristensen</span>), pp.  <span class="pageFirst">7</span>– <span class="pageLast">25</span>.  Walter de Gruyter Publications, Berlin.

                     <div class="extra-links" id="e_1_2_7_48_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar?hl=en&amp;q=%0AKristensen%2C+N.P.+%26+%0ASkalski%2C+A.W.+%281999%29++Phylogeny+and+palaeontology.+Handbuch+der+Zoologie+Band+IV+Arthropoda%3A+Insecta+Part+35+Lepidoptera%2C+Moths+and+Butterflies+%28ed.+by%0A%09%09%09%09%09+N.+P.+Kristensen%29%2C+pp.++7%E2%80%93+25.++Walter+de+Gruyter+Publications%2C+Berlin.%0A" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b48">
                     <span class="author">Krzeminski, W.</span> &amp; 
<span class="author">Krzeminski, E</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">Triassic Diptera: descriptions, revisions and phylogenetic relations</span>. 
<i>Acta Zoologica Cracoviensia</i>,  <span class="vol">46S</span>,  <span class="pageFirst">153</span>– <span class="pageLast">184</span>.

                     <div class="extra-links" id="e_1_2_7_49_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=46S&amp;publication_year=2003&amp;pages=153-184&amp;journal=Acta+Zoologica+Cracoviensia&amp;author=W.+Krzeminski&amp;author=E+Krzeminski&amp;title=Triassic+Diptera%3A+descriptions%2C+revisions+and+phylogenetic+relations" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b49">
                     <span class="author">Kumar, S.</span>, 
<span class="author">Tamura, K.</span> &amp; 
<span class="author">Nei, M</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">MEGA3: integrated software for molecular evolutionary genetics analysis and sequence alignment</span>. 
<i>Briefings in Bioinformatics</i>,  <span class="vol">5</span>,  <span class="pageFirst">150</span>– <span class="pageLast">163</span>.

                     <div class="extra-links" id="e_1_2_7_50_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fbib%2F5.2.150" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXntFGqu7s%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15260895" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000223033300005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=2004&amp;pages=150-163&amp;journal=Briefings+in+Bioinformatics&amp;author=S.+Kumar&amp;author=K.+Tamura&amp;author=M+Nei&amp;title=MEGA3%3A+integrated+software+for+molecular+evolutionary+genetics+analysis+and+sequence+alignment" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b50">
                     <span class="author">Labandeira, C.C</span>. (<span class="pubYear">2005</span>)  <span class="chapterTitle">Fossil history and evolutionary ecology of Diptera and their associations with plants</span>. 
 <span class="bookTitle">The Evolutionary Biology of Flies</span> (ed. by
					 <span class="editor">D. K. Yeates</span> &amp; 
 <span class="editor">B. M. Wiegmann</span>), pp.  <span class="pageFirst">217</span>– <span class="pageLast">273</span>.  Columbia University Press, New York.

                     <div class="extra-links" id="e_1_2_7_51_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2005&amp;pages=217-273&amp;author=C.C+Labandeira&amp;title=The+Evolutionary+Biology+of+Flies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b51">
                     <span class="author">Leaché, A.D.</span> &amp; 
<span class="author">Reeder, T.W</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Molecular systematics of the eastern fence lizard<i> Sceloporus undulates</i>: a comparison of parsimony, likelihood and Bayesian approaches</span>. 
<i>Systematic Biology</i>,  <span class="vol">51</span>,  <span class="pageFirst">44</span>– <span class="pageLast">68</span>.

                     <div class="extra-links" id="e_1_2_7_52_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1080%2F106351502753475871" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11943092" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000174121300005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=51&amp;publication_year=2002&amp;pages=44-68&amp;journal=Systematic+Biology&amp;author=A.D.+Leach%C3%A9&amp;author=T.W+Reeder&amp;title=Molecular+systematics+of+the+eastern+fence+lizard+Sceloporus+undulates%3A+a+comparison+of+parsimony%2C+likelihood+and+Bayesian+approaches" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b52">
                     <span class="author">Lessinger, A.C.</span>, 
<span class="author">Junqueira, A.C.M.</span>, 
<span class="author">Lemos, T.A.</span>, 
<span class="author">Kemper, E.L.</span>, 
<span class="author">Rodrigues da Silva, F.</span>, 
<span class="author">Vettore, A.L.</span>, 
<span class="author">Arruda, P.</span> &amp; 
<span class="author">Azeredo Espin, A.M.L</span>. (<span class="pubYear">2000</span>)  <span class="articleTitle">The mitochondrial genome of the primary screwworm fly<i> Cochliomyia hominovorax </i>(Diptera: Calliphoridae)</span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">9</span>,  <span class="pageFirst">521</span>– <span class="pageLast">529</span>.

                     <div class="extra-links" id="e_1_2_7_53_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2583.2000.00215.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXot1GksLw%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11029671" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000089857100010" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2000&amp;pages=521-529&amp;journal=Insect+Molecular+Biology&amp;author=A.C.+Lessinger&amp;author=A.C.M.+Junqueira&amp;author=T.A.+Lemos&amp;author=E.L.+Kemper&amp;author=F.+Rodrigues+da+Silva&amp;author=A.L.+Vettore&amp;author=P.+Arruda&amp;author=A.M.L+Azeredo+Espin&amp;title=The+mitochondrial+genome+of+the+primary+screwworm+fly+Cochliomyia+hominovorax+%28Diptera%3A+Calliphoridae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b53">
                     <span class="author">Lessinger, A.C.</span>, 
<span class="author">Junqueira, A.C.M.</span>, 
<span class="author">Conte, F.F.</span> &amp; 
<span class="author">Azeredo Espin, A.M.L</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Analysis of a conserved duplicated tRNA gene in the mitochondrial genome of blowflies</span>. 
<i>Gene</i>,  <span class="vol">339</span>,  <span class="pageFirst">1</span>– <span class="pageLast">6</span>.

                     <div class="extra-links" id="e_1_2_7_54_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.gene.2004.06.041" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsVKjtb4%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15363840" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000224173200001" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=339&amp;publication_year=2004&amp;pages=1-6&amp;journal=Gene&amp;author=A.C.+Lessinger&amp;author=A.C.M.+Junqueira&amp;author=F.F.+Conte&amp;author=A.M.L+Azeredo+Espin&amp;title=Analysis+of+a+conserved+duplicated+tRNA+gene+in+the+mitochondrial+genome+of+blowflies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b54">
                     <span class="author">Lewis, D.L.</span>, 
<span class="author">Farr, C.L.</span> &amp; 
<span class="author">Kaguni, L.S</span>. (<span class="pubYear">1995</span>)  <span class="articleTitle"><i>Drosophila melanogaster </i>mitochondrial DNA: completion of the nucleotide sequence and evolutionary comparisons</span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">4</span>,  <span class="pageFirst">263</span>– <span class="pageLast">278</span>.

                     <div class="extra-links" id="e_1_2_7_55_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2583.1995.tb00032.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2MXhtVSitLrM" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=8825764" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1995TL36200006" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=1995&amp;pages=263-278&amp;journal=Insect+Molecular+Biology&amp;author=D.L.+Lewis&amp;author=C.L.+Farr&amp;author=L.S+Kaguni&amp;title=Drosophila+melanogaster+mitochondrial+DNA%3A+completion+of+the+nucleotide+sequence+and+evolutionary+comparisons" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b55">
                     <span class="author">Lewis, R.L.</span>, 
<span class="author">Beckenbach, A.T.</span> &amp; 
<span class="author">Mooers, A.Ø</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">The phylogeny of the subgroups within the<i> melanogaster </i>species group: Likelihood tests on<i> CO1 </i>and<i> COII </i>sequences and a Bayesian estimate of phylogeny</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">37</span>,  <span class="pageFirst">15</span>– <span class="pageLast">24</span>.

                     <div class="extra-links" id="e_1_2_7_56_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2Fj.ympev.2005.02.018" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVart7jM" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=16182148" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000232507400002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=37&amp;publication_year=2005&amp;pages=15-24&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=R.L.+Lewis&amp;author=A.T.+Beckenbach&amp;author=A.%C3%98+Mooers&amp;title=The+phylogeny+of+the+subgroups+within+the+melanogaster+species+group%3A+Likelihood+tests+on+CO1+and+COII+sequences+and+a+Bayesian+estimate+of+phylogeny" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b56">
                     <span class="author">Lin, C.P.</span> &amp; 
<span class="author">Danforth, B.N</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">How do insect nuclear and mitochondrial gene substitution patterns differ? Insights from Bayesian analysis of combined datasets</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">30</span>,  <span class="pageFirst">686</span>– <span class="pageLast">702</span>.

                     <div class="extra-links" id="e_1_2_7_57_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS1055-7903%2803%2900241-0" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXptFWiuw%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15012948" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000188958400016" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=30&amp;publication_year=2004&amp;pages=686-702&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=C.P.+Lin&amp;author=B.N+Danforth&amp;title=How+do+insect+nuclear+and+mitochondrial+gene+substitution+patterns+differ%3F+Insights+from+Bayesian+analysis+of+combined+datasets" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b57">
                     <span class="author">Lowe, T.M.</span> &amp; 
<span class="author">Eddy, S.R</span>. (<span class="pubYear">1997</span>)  <span class="articleTitle">tRNAscan‐SE: a program for improved detection of transfer RNA genes in genomic sequence</span>. 
<i>Nucleic Acids Research</i>,  <span class="vol">25</span>,  <span class="pageFirst">955</span>– <span class="pageLast">964</span>.

                     <div class="extra-links" id="e_1_2_7_58_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fnar%2F25.5.955" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2sXhvVahtrk%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=9023104" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1997WM30000005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=25&amp;publication_year=1997&amp;pages=955-964&amp;journal=Nucleic+Acids+Research&amp;author=T.M.+Lowe&amp;author=S.R+Eddy&amp;title=tRNAscan%E2%80%90SE%3A+a+program+for+improved+detection+of+transfer+RNA+genes+in+genomic+sequence" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b58">
                     <span class="author">Mackerras, I.M</span>. (<span class="pubYear">1925</span>)  <span class="articleTitle">The Nemestrinidae (Diptera) of the Australasian Region</span>. 
<i>Proceedings of the Linnean Society of New South Wales</i>,  <span class="vol">50</span>,  <span class="pageFirst">489</span>– <span class="pageLast">561</span>.

                     <div class="extra-links" id="e_1_2_7_59_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=50&amp;publication_year=1925&amp;pages=489-561&amp;journal=Proceedings+of+the+Linnean+Society+of+New+South+Wales&amp;author=I.M+Mackerras&amp;title=The+Nemestrinidae+%28Diptera%29+of+the+Australasian+Region" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b59">
                     <span class="author">Maddison, W.</span> &amp; 
<span class="author">Maddison, D</span>. (<span class="pubYear">2003</span>)  <span class="bookTitle">Macclade Version 4.06</span>.  Sinauer Associates, Sunderland, MA.

                     <div class="extra-links" id="e_1_2_7_60_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2003&amp;author=W.+Maddison&amp;author=D+Maddison&amp;title=Macclade+Version+4.06" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b60">
                     <span class="author">McAlpine, J.F</span>. (<span class="pubYear">1989</span>)  <span class="chapterTitle">Phylogeny and classification of the Muscomorpha</span>. 
 <span class="bookTitle">Manual of Nearctic Diptera</span>, Vol.  <span class="vol">3</span> (ed. by
					 <span class="editor">J. F. McAlpine</span> &amp; 
 <span class="editor">D. V. Wood</span>), pp.  <span class="pageFirst">1397</span>– <span class="pageLast">1518</span>. Research Branch,  Agriculture Canada, Ottawa.

                     <div class="extra-links" id="e_1_2_7_61_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1989&amp;pages=1397-1518&amp;author=J.F+McAlpine&amp;title=Manual+of+Nearctic+Diptera" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b61">
                     <span class="author">Mitchell, S.E.</span>, 
<span class="author">Cockburn, A.F.</span> &amp; 
<span class="author">Seawright, J.A</span>. (<span class="pubYear">1993</span>)  <span class="articleTitle">The mitochondrial genome of<i> Anopheles quadrimaculatus </i>species A: complete nucleotide sequence and genome organisarion</span>. 
<i>Genome</i>,  <span class="vol">36</span>,  <span class="pageFirst">1058</span>– <span class="pageLast">1073</span>.

                     <div class="extra-links" id="e_1_2_7_62_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1139%2Fg93-141" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2cXmslKqtbc%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=8112570" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1993MQ45900008" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=36&amp;publication_year=1993&amp;pages=1058-1073&amp;journal=Genome&amp;author=S.E.+Mitchell&amp;author=A.F.+Cockburn&amp;author=J.A+Seawright&amp;title=The+mitochondrial+genome+of+Anopheles+quadrimaculatus+species+A%3A+complete+nucleotide+sequence+and+genome+organisarion" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b62">
                     <span class="author">Moulton, J.K.</span> &amp; 
<span class="author">Wiegmann, B.M</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Evolution and phylogenetic utility of CAD (rudimentary) among Mesozoic‐aged Eremoneuran Diptera (Insecta)</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">31</span>,  <span class="pageFirst">363</span>– <span class="pageLast">378</span>.

                     <div class="extra-links" id="e_1_2_7_63_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS1055-7903%2803%2900284-7" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvFSksbw%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15019631" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000220536800029" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=31&amp;publication_year=2004&amp;pages=363-378&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=J.K.+Moulton&amp;author=B.M+Wiegmann&amp;title=Evolution+and+phylogenetic+utility+of+CAD+%28rudimentary%29+among+Mesozoic%E2%80%90aged+Eremoneuran+Diptera+%28Insecta%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b63">
                     <span class="author">Nagatomi, A</span>. (<span class="pubYear">1992</span>)  <span class="articleTitle">Notes on the phylogeny of various taxa of the orthorraphous Brachycera (Insecta: Diptera)</span>. 
<i>Zoological Science</i>,  <span class="vol">9</span>,  <span class="pageFirst">843</span>– <span class="pageLast">857</span>.

                     <div class="extra-links" id="e_1_2_7_64_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1992JP00300018" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=1992&amp;pages=843-857&amp;journal=Zoological+Science&amp;author=A+Nagatomi&amp;title=Notes+on+the+phylogeny+of+various+taxa+of+the+orthorraphous+Brachycera+%28Insecta%3A+Diptera%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b64">
                     <span class="author">Nardi, F.</span>, 
<span class="author">Carapelli, A.</span>, 
<span class="author">Fanciulli, P.P.</span>, 
<span class="author">Dallai, R.</span> &amp; 
<span class="author">Frati, F</span>. (<span class="pubYear">2001</span>)  <span class="articleTitle">The complete mitochondrial DNA sequence of the basal hexapod<i> Tetrodontophora bielanensis</i>: evidence for heteroplasmy and tRNA translocations</span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">18</span>,  <span class="pageFirst">1293</span>– <span class="pageLast">1304</span>.

                     <div class="extra-links" id="e_1_2_7_65_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Foxfordjournals.molbev.a003914" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXltVGru78%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11420368" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000169846400013" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2001&amp;pages=1293-1304&amp;journal=Molecular+Biology+and+Evolution&amp;author=F.+Nardi&amp;author=A.+Carapelli&amp;author=P.P.+Fanciulli&amp;author=R.+Dallai&amp;author=F+Frati&amp;title=The+complete+mitochondrial+DNA+sequence+of+the+basal+hexapod+Tetrodontophora+bielanensis%3A+evidence+for+heteroplasmy+and+tRNA+translocations" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b65">
                     <span class="author">Nardi, F.</span>, 
<span class="author">Spinsanti, G.</span>, 
<span class="author">Boore, J.L.</span>, 
<span class="author">Carapelli, A.</span>, 
<span class="author">Dallai, R.</span> &amp; 
<span class="author">Frati, F</span>. (<span class="pubYear">2003a</span>)  <span class="articleTitle">Hexapod origins: monophyletic or paraphyletic?</span> 
                     <i>Science</i>,  <span class="vol">299</span>,  <span class="pageFirst">1887</span>– <span class="pageLast">1889</span>.

                     <div class="extra-links" id="e_1_2_7_66_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1126%2Fscience.1078607" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitFKlu7c%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12649480" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000181669700044" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=299&amp;publication_year=2003a&amp;pages=1887-1889&amp;journal=Science&amp;author=F.+Nardi&amp;author=G.+Spinsanti&amp;author=J.L.+Boore&amp;author=A.+Carapelli&amp;author=R.+Dallai&amp;author=F+Frati&amp;title=Hexapod+origins%3A+monophyletic+or+paraphyletic%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b66">
                     <span class="author">Nardi, F.</span>, 
<span class="author">Carapelli, A.</span>, 
<span class="author">Dallai, R.</span> &amp; 
<span class="author">Frati, F</span>. (<span class="pubYear">2003b</span>)  <span class="articleTitle">The mitochondrial genome of the olive fly<i> Bactrocera oleae</i>: two haplotypes from distant geographical locations</span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">12</span>,  <span class="pageFirst">605</span>– <span class="pageLast">611</span>.

                     <div class="extra-links" id="e_1_2_7_67_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2583.2003.00445.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpvFejsLg%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=14986921" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000186648100007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2003b&amp;pages=605-611&amp;journal=Insect+Molecular+Biology&amp;author=F.+Nardi&amp;author=A.+Carapelli&amp;author=R.+Dallai&amp;author=F+Frati&amp;title=The+mitochondrial+genome+of+the+olive+fly+Bactrocera+oleae%3A+two+haplotypes+from+distant+geographical+locations" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b67">
                     <span class="author">Nylander, J.A.A.</span>, 
<span class="author">Ronquist, F.</span>, 
<span class="author">Huelsenbeck, J.P.</span> &amp; 
<span class="author">Nieves‐Aldrey, J.L</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Bayesian phylogenetic analysis of combined data</span>. 
<i>Systematic Biology</i>,  <span class="vol">53</span>,  <span class="pageFirst">47</span>– <span class="pageLast">67</span>.

                     <div class="extra-links" id="e_1_2_7_68_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1080%2F10635150490264699" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=14965900" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000188895000005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=53&amp;publication_year=2004&amp;pages=47-67&amp;journal=Systematic+Biology&amp;author=J.A.A.+Nylander&amp;author=F.+Ronquist&amp;author=J.P.+Huelsenbeck&amp;author=J.L+Nieves%E2%80%90Aldrey&amp;title=Bayesian+phylogenetic+analysis+of+combined+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b68">
                     <span class="author">O’Grady, P.M.</span> &amp; 
<span class="author">Kidwell, M.G</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Phylogeny of the subgenus<i> Sophophora </i>(Diptera: Drosphilidae) based on combined analysis of nuclear and mitochondrial sequences</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">22</span>,  <span class="pageFirst">442</span>– <span class="pageLast">453</span>.

                     <div class="extra-links" id="e_1_2_7_69_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1006%2Fmpev.2001.1053" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhs1ykt7c%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11884169" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000174357800012" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2002&amp;pages=442-453&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=P.M.+O%E2%80%99Grady&amp;author=M.G+Kidwell&amp;title=Phylogeny+of+the+subgenus+Sophophora+%28Diptera%3A+Drosphilidae%29+based+on+combined+analysis+of+nuclear+and+mitochondrial+sequences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b69">
                     <span class="author">Phillips, M.J.</span> &amp; 
<span class="author">Penny, D</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">The root of the mammalian tree inferred from whole mitochondrial genomes</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">28</span>,  <span class="pageFirst">171</span>– <span class="pageLast">185</span>.

                     <div class="extra-links" id="e_1_2_7_70_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS1055-7903%2803%2900057-5" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXls1eqt7g%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12878457" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000184530100002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=28&amp;publication_year=2003&amp;pages=171-185&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=M.J.+Phillips&amp;author=D+Penny&amp;title=The+root+of+the+mammalian+tree+inferred+from+whole+mitochondrial+genomes" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b70">
                     <span class="author">Posada, D.</span> &amp; 
<span class="author">Crandall, K.A</span>. (<span class="pubYear">1998</span>)  <span class="articleTitle">ModelTest: Testing the best‐fit model of nucleotide substitution</span>. 
<i>Bioinformatics</i>,  <span class="vol">14</span>,  <span class="pageFirst">817</span>– <span class="pageLast">818</span>.

                     <div class="extra-links" id="e_1_2_7_71_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fbioinformatics%2F14.9.817" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK1MXktlCltw%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=9918953" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000077489900010" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=14&amp;publication_year=1998&amp;pages=817-818&amp;journal=Bioinformatics&amp;author=D.+Posada&amp;author=K.A+Crandall&amp;title=ModelTest%3A+Testing+the+best%E2%80%90fit+model+of+nucleotide+substitution" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b71">
                     <span class="author">Remsen, J.</span> &amp; 
<span class="author">O’Grady, P.M</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Phylogeny of Drosophilinae (Diptera: Drosophilidae), with comments on combined analysis and character support</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">24</span>,  <span class="pageFirst">249</span>– <span class="pageLast">264</span>.

                     <div class="extra-links" id="e_1_2_7_72_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS1055-7903%2802%2900226-9" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12144760" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000177736800007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=2002&amp;pages=249-264&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=J.+Remsen&amp;author=P.M+O%E2%80%99Grady&amp;title=Phylogeny+of+Drosophilinae+%28Diptera%3A+Drosophilidae%29%2C+with+comments+on+combined+analysis+and+character+support" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b72">
                     <span class="author">Rokas, A.</span> &amp; 
<span class="author">Holland, P.W.H</span>. (<span class="pubYear">2000</span>)  <span class="articleTitle">Rare genomic changes as a tool for phylogenetics</span>. 
<i>Trends in Ecology and Evolution</i>,  <span class="vol">15</span>,  <span class="pageFirst">454</span>– <span class="pageLast">459</span>.

                     <div class="extra-links" id="e_1_2_7_73_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS0169-5347%2800%2901967-4" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ASTN%3A280%3ADC%252BC2sbjsVWitQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11050348" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000165122400015" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=15&amp;publication_year=2000&amp;pages=454-459&amp;journal=Trends+in+Ecology+and+Evolution&amp;author=A.+Rokas&amp;author=P.W.H+Holland&amp;title=Rare+genomic+changes+as+a+tool+for+phylogenetics" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b73">
                     <span class="author">Ronquist, F.</span> &amp; 
<span class="author">Huelsenbeck, J.P</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">MRBAYES 3: Bayesian phylogenetic inference under mixed models</span>. 
<i>Bioinformatics</i>,  <span class="vol">19</span>,  <span class="pageFirst">1572</span>– <span class="pageLast">1574</span>.

                     <div class="extra-links" id="e_1_2_7_74_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.0908-8857.2004.03297.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntlKms7k%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12912839" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000184878700016" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2003&amp;pages=1572-1574&amp;journal=Bioinformatics&amp;author=F.+Ronquist&amp;author=J.P+Huelsenbeck&amp;title=MRBAYES+3%3A+Bayesian+phylogenetic+inference+under+mixed+models" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b74">
                     <span class="author">Russo, C.A.M.</span>, 
<span class="author">Takezaki, N.</span> &amp; 
<span class="author">Nei, M</span>. (<span class="pubYear">1995</span>)  <span class="articleTitle">Molecular phylogeny and divergence times of drosophilid species</span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">12</span>,  <span class="pageFirst">391</span>– <span class="pageLast">404</span>.

                     <div class="extra-links" id="e_1_2_7_75_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2MXltF2ntb8%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=7739381" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1995QU32000004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=1995&amp;pages=391-404&amp;journal=Molecular+Biology+and+Evolution&amp;author=C.A.M.+Russo&amp;author=N.+Takezaki&amp;author=M+Nei&amp;title=Molecular+phylogeny+and+divergence+times+of+drosophilid+species" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b75">
                     <span class="author">Saccone, C.</span>, 
<span class="author">De Giorgi, C.</span>, 
<span class="author">Gissi, C.</span>, 
<span class="author">Pesole, G.</span> &amp; 
<span class="author">Reyes, A</span>. (<span class="pubYear">1999</span>)  <span class="articleTitle">Evolutionary genomics in Metazoa: the mitochondrial DNA as a model system</span>. 
<i>Gene</i>,  <span class="vol">238</span>,  <span class="pageFirst">195</span>– <span class="pageLast">209</span>.

                     <div class="extra-links" id="e_1_2_7_76_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS0378-1119%2899%2900270-X" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK1MXntFKnu7s%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10570997" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000083005400020" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=238&amp;publication_year=1999&amp;pages=195-209&amp;journal=Gene&amp;author=C.+Saccone&amp;author=C.+De+Giorgi&amp;author=C.+Gissi&amp;author=G.+Pesole&amp;author=A+Reyes&amp;title=Evolutionary+genomics+in+Metazoa%3A+the+mitochondrial+DNA+as+a+model+system" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b76">
                     <span class="author">Schawaroch, V</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Phylogeny of a paradigm lineage: the<i> Drosophila melanogaster </i>species group (Diptera: Drosophilidae)</span>. 
<i>Biological Journal of the Linnean Society</i>,  <span class="vol">76</span>,  <span class="pageFirst">21</span>– <span class="pageLast">37</span>.

                     <div class="extra-links" id="e_1_2_7_77_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1095-8312.2002.tb01711.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000175718400003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=76&amp;publication_year=2002&amp;pages=21-37&amp;journal=Biological+Journal+of+the+Linnean+Society&amp;author=V+Schawaroch&amp;title=Phylogeny+of+a+paradigm+lineage%3A+the+Drosophila+melanogaster+species+group+%28Diptera%3A+Drosophilidae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b77">
                     <span class="author">Shao, R.</span>, 
<span class="author">Campbell, N.J.H.</span> &amp; 
<span class="author">Barker, S.C</span>. (<span class="pubYear">2001</span>)  <span class="articleTitle">Numerous gene rearrangements in the mitochondrial genome of the wallaby louse,<i> Heterodoxus macropus </i>(Phthiraptera)</span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">18</span>,  <span class="pageFirst">858</span>– <span class="pageLast">865</span>.

                     <div class="extra-links" id="e_1_2_7_78_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Foxfordjournals.molbev.a003867" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3MXjtFykt7g%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11319269" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000168427400017" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2001&amp;pages=858-865&amp;journal=Molecular+Biology+and+Evolution&amp;author=R.+Shao&amp;author=N.J.H.+Campbell&amp;author=S.C+Barker&amp;title=Numerous+gene+rearrangements+in+the+mitochondrial+genome+of+the+wallaby+louse%2C+Heterodoxus+macropus+%28Phthiraptera%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b79">
                     <span class="author">Sinclair, B.J</span>. (<span class="pubYear">1992</span>)  <span class="articleTitle">A phylogenetic interpretation of the Brachycera (Diptera) based on the larval mandible and associated mouthpart structures</span>. 
<i>Systematic Entomology</i>,  <span class="vol">17</span>,  <span class="pageFirst">233</span>– <span class="pageLast">252</span>.

                     <div class="extra-links" id="e_1_2_7_79_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-3113.1992.tb00335.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1992JF89900003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=17&amp;publication_year=1992&amp;pages=233-252&amp;journal=Systematic+Entomology&amp;author=B.J+Sinclair&amp;title=A+phylogenetic+interpretation+of+the+Brachycera+%28Diptera%29+based+on+the+larval+mandible+and+associated+mouthpart+structures" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b80">
                     <span class="author">Sinclair, B.J.</span>, 
<span class="author">Cumming, J.M.</span> &amp; 
<span class="author">Wood, D.M</span>. (<span class="pubYear">1994</span>)  <span class="articleTitle">Homology and phylogenetic implications of male genitalia in Diptera – Lower Brachycera</span>. 
<i>Entomologica Scandinavica</i>,  <span class="vol">24</span>,  <span class="pageFirst">407</span>– <span class="pageLast">432</span>.

                     <div class="extra-links" id="e_1_2_7_80_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1163%2F187631293X00190" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1993NA42200004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=1994&amp;pages=407-432&amp;journal=Entomologica+Scandinavica&amp;author=B.J.+Sinclair&amp;author=J.M.+Cumming&amp;author=D.M+Wood&amp;title=Homology+and+phylogenetic+implications+of+male+genitalia+in+Diptera+%E2%80%93+Lower+Brachycera" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b81">
                     <span class="author">Sorenson, M.D.</span> (<span class="pubYear">1999</span>) TreeRot, Version 2.  Boston University, Boston, MA.

                     <div class="extra-links" id="e_1_2_7_81_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar?hl=en&amp;q=%0ASorenson%2C+M.D.+%281999%29+TreeRot%2C+Version+2.++Boston+University%2C+Boston%2C+MA.%0A" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b82">
                     <span class="author">Spanos, L.</span>, 
<span class="author">Koutroumbas, G.</span>, 
<span class="author">Kotsyfakis, M.</span> &amp; 
<span class="author">Louis, C</span>. (<span class="pubYear">2000</span>)  <span class="articleTitle">The mitochondrial genome of the Mediterranean fruit fly,<i> Ceratitis capitata</i></span>. 
<i>Insect Molecular Biology</i>,  <span class="vol">9</span>,  <span class="pageFirst">139</span>– <span class="pageLast">144</span>.

                     <div class="extra-links" id="e_1_2_7_82_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2583.2000.00165.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjsF2htL8%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10762421" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000086700200003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2000&amp;pages=139-144&amp;journal=Insect+Molecular+Biology&amp;author=L.+Spanos&amp;author=G.+Koutroumbas&amp;author=M.+Kotsyfakis&amp;author=C+Louis&amp;title=The+mitochondrial+genome+of+the+Mediterranean+fruit+fly%2C+Ceratitis+capitata" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b83">
                     <span class="author">Stewart, J.B.</span> &amp; 
<span class="author">Beckenbach, A.T</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">Phylogenetic and genomic analysis of the complete mitochondrial DNA sequence of the spotted asparagus beetle<i> Crioceris duodecimpunctata</i></span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">26</span>,  <span class="pageFirst">513</span>– <span class="pageLast">526</span>.

                     <div class="extra-links" id="e_1_2_7_83_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1016%2FS1055-7903%2802%2900421-9" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitVKntLc%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12644408" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000181885500014" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=26&amp;publication_year=2003&amp;pages=513-526&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=J.B.+Stewart&amp;author=A.T+Beckenbach&amp;title=Phylogenetic+and+genomic+analysis+of+the+complete+mitochondrial+DNA+sequence+of+the+spotted+asparagus+beetle+Crioceris+duodecimpunctata" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b84">
                     <span class="author">Swofford, D.L</span>. (<span class="pubYear">2002</span>)  <span class="bookTitle">Paup* Phylogenetic Analysis Using Parsimony (*and Other Methods)</span>,  <span class="edition">Version 4</span>.  Sinauer Associates, Sunderland, MA.

                     <div class="extra-links" id="e_1_2_7_84_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1007%2Fs00572-003-0284-z" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=14669095" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000226303000001" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2002&amp;author=D.L+Swofford&amp;title=Paup*+Phylogenetic+Analysis+Using+Parsimony+%28*and+Other+Methods%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b85">
                     <span class="author">Symons, M.R.E.</span> &amp; 
<span class="author">Wertheim, B</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">The mode of evolution of aggregation pheromones in<i> Drosophila </i>species</span>. 
<i>Journal of Evolutionary Biology</i>,  <span class="vol">18</span>,  <span class="pageFirst">1253</span>– <span class="pageLast">1263</span>.

                     <div class="extra-links" id="e_1_2_7_85_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1420-9101.2005.00971.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=16135121" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000231508900020" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2005&amp;pages=1253-1263&amp;journal=Journal+of+Evolutionary+Biology&amp;author=M.R.E.+Symons&amp;author=B+Wertheim&amp;title=The+mode+of+evolution+of+aggregation+pheromones+in+Drosophila+species" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b86">
                     <span class="author">Tamura, K.</span>, 
<span class="author">Subramanian, S.</span> &amp; 
<span class="author">Kumar, S</span>. (<span class="pubYear">2004</span>)  <span class="articleTitle">Temporal patterns of fruit fly (<i>Drosophila</i>) evolution revealed by mutation clocks</span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">21</span>,  <span class="pageFirst">36</span>– <span class="pageLast">44</span>.

                     <div class="extra-links" id="e_1_2_7_86_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fmolbev%2Fmsg236" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhvVKqsrg%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12949132" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000189149300004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2004&amp;pages=36-44&amp;journal=Molecular+Biology+and+Evolution&amp;author=K.+Tamura&amp;author=S.+Subramanian&amp;author=S+Kumar&amp;title=Temporal+patterns+of+fruit+fly+%28Drosophila%29+evolution+revealed+by+mutation+clocks" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b99">
                     <span class="author">Terry, M.D</span>. &amp; <span class="author">Whiting, M.F</span>. (<span class="pubYear">2005</span>)  <span class="articleTitle">Comparison of two alignment techniques within a single complex data set: POY versus Clustal</span>. 
<i>Cladistics</i>,  <span class="vol">21</span>,  <span class="pageFirst">272</span>– <span class="pageLast">281</span>.

                     <div class="extra-links" id="e_1_2_7_87_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1096-0031.2005.00063.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000230601000004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2005&amp;pages=272-281&amp;journal=Cladistics&amp;author=M.D+Terry&amp;author=M.F+Whiting&amp;title=Comparison+of+two+alignment+techniques+within+a+single+complex+data+set%3A+POY+versus+Clustal" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b87">
                     <span class="author">Thompson, J.D.</span>, 
<span class="author">Higgins, D.G.</span> &amp; 
<span class="author">Gibson, T.J</span>. (<span class="pubYear">1994</span>)  <span class="articleTitle">Clustal
 W: Improving the sensitivity of progressive multiple sequence alignment
 through sequence weighting, position specific gap penalties and weight 
matrix choice</span>. 
<i>Nucleic Acids Research</i>,  <span class="vol">22</span>,  <span class="pageFirst">4673</span>– <span class="pageLast">4680</span>.

                     <div class="extra-links" id="e_1_2_7_88_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fnar%2F22.22.4673" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK2MXitlSgu74%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=7984417" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1994PU19900018" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=1994&amp;pages=4673-4680&amp;journal=Nucleic+Acids+Research&amp;author=J.D.+Thompson&amp;author=D.G.+Higgins&amp;author=T.J+Gibson&amp;title=Clustal+W%3A+Improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting%2C+position+specific+gap+penalties+and+weight+matrix+choice" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b88">
                     <span class="author">Thompson, J.D.</span>, 
<span class="author">Gibson, T.J.</span>, 
<span class="author">Plewniak, F.</span>, 
<span class="author">Jeanmougin, F.</span> &amp; 
<span class="author">Higgins, D.G</span>. (<span class="pubYear">1997</span>)  <span class="articleTitle">The CLUSTAL–windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools</span>. 
<i>Nucleic Acids Research</i>,  <span class="vol">24</span>,  <span class="pageFirst">173</span>– <span class="pageLast">216</span>.

                     <div class="extra-links" id="e_1_2_7_89_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000080463000003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=24&amp;publication_year=1997&amp;pages=173-216&amp;journal=Nucleic+Acids+Research&amp;author=J.D.+Thompson&amp;author=T.J.+Gibson&amp;author=F.+Plewniak&amp;author=F.+Jeanmougin&amp;author=D.G+Higgins&amp;title=The+CLUSTAL%E2%80%93windows+interface%3A+flexible+strategies+for+multiple+sequence+alignment+aided+by+quality+analysis+tools" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b89">
                     <span class="author">Vogler, A.P</span>. (<span class="pubYear">2005</span>)  <span class="chapterTitle">Molecular systematics of Coleoptera: what has been achieved so far?</span> 
                      <span class="bookTitle">Handbuch der Zoologie, Band IV Arthropoda: Insecta Part 38 Coleoptera. Beetles</span> (ed. by
					 <span class="editor">N. P. Kristensen</span> &amp; 
 <span class="editor">R. G. Beutel</span>), pp.  <span class="pageFirst">17</span>– <span class="pageLast">22</span>.  Walter der Gruyter Press, Berlin.

                     <div class="extra-links" id="e_1_2_7_90_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2005&amp;pages=17-22&amp;author=A.P+Vogler&amp;title=Handbuch+der+Zoologie%2C+Band+IV+Arthropoda%3A+Insecta+Part+38+Coleoptera.+Beetles" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b90">
                     <span class="author">Wheeler, W.C.</span> &amp; 
<span class="author">Honeycutt, R.L</span>. (<span class="pubYear">1988</span>)  <span class="articleTitle">Paired sequence difference in ribosomal RNAs: evolutionary and phylogenetic implications</span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">5</span>,  <span class="pageFirst">90</span>– <span class="pageLast">96</span>.

                     <div class="extra-links" id="e_1_2_7_91_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaL1cXht1Krur0%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=3357414" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=A1988L618700007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=5&amp;publication_year=1988&amp;pages=90-96&amp;journal=Molecular+Biology+and+Evolution&amp;author=W.C.+Wheeler&amp;author=R.L+Honeycutt&amp;title=Paired+sequence+difference+in+ribosomal+RNAs%3A+evolutionary+and+phylogenetic+implications" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b91">
                     <span class="author">Whittingham, L.A.</span>, 
<span class="author">Slikas, B.</span>, 
<span class="author">Winker, D.W.</span> &amp; 
<span class="author">Sheldon, F.H</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Phylogeny of the tree swallow genus<i> Tachycineta </i>(Aves: Hirundinidae) by Bayesian analysis of mitochondrial DNA sequences</span>. 
<i>Molecular Phylogenetics and Evolution</i>,  <span class="vol">22</span>,  <span class="pageFirst">430</span>– <span class="pageLast">441</span>.

                     <div class="extra-links" id="e_1_2_7_92_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1006%2Fmpev.2001.1073" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD38Xhs1yktb4%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=11884168" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000174357800011" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=22&amp;publication_year=2002&amp;pages=430-441&amp;journal=Molecular+Phylogenetics+and+Evolution&amp;author=L.A.+Whittingham&amp;author=B.+Slikas&amp;author=D.W.+Winker&amp;author=F.H+Sheldon&amp;title=Phylogeny+of+the+tree+swallow+genus+Tachycineta+%28Aves%3A+Hirundinidae%29+by+Bayesian+analysis+of+mitochondrial+DNA+sequences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b100">
                     <span class="author">Wiegmann, B.M.</span>, 
<span class="author">Tsaur, S.‐C</span>., <span class="author">Webb, D.W.</span>, 
<span class="author">Yeates, D.K.</span> &amp; 
<span class="author">Cassel, B.K</span>. (<span class="pubYear">2000</span>)  <span class="articleTitle">Monophyly and relationships of the Tabanomorpha (Diptera: Bracycera) based on 28S ribosomal gene sequences</span>. 
<i>Annals of the Entomological Society of America</i>,  <span class="vol">93</span>,  <span class="pageFirst">1031</span>– <span class="pageLast">1038</span>.

                     <div class="extra-links" id="e_1_2_7_93_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1603%2F0013-8746%282000%29093%5B1031%3AMAROTT%5D2.0.CO%3B2" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXntlShur4%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000089632300004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=93&amp;publication_year=2000&amp;pages=1031-1038&amp;journal=Annals+of+the+Entomological+Society+of+America&amp;author=B.M.+Wiegmann&amp;author=S.%E2%80%90C+Tsaur&amp;author=D.W.+Webb&amp;author=D.K.+Yeates&amp;author=B.K+Cassel&amp;title=Monophyly+and+relationships+of+the+Tabanomorpha+%28Diptera%3A+Bracycera%29+based+on+28S+ribosomal+gene+sequences" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b92">
                     <span class="author">Wiegmann, B.M.</span>, 
<span class="author">Yeates, D.K.</span>, 
<span class="author">Thorne, J.L.</span> &amp; 
<span class="author">Kishino, H</span>. (<span class="pubYear">2003</span>)  <span class="articleTitle">Time flies, a new molecular time scale for brachyceran fly evolution without a clock</span>. 
<i>Systematic Biology</i>,  <span class="vol">52</span>,  <span class="pageFirst">745</span>– <span class="pageLast">756</span>.

                     <div class="extra-links" id="e_1_2_7_94_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Fsysbio%2F52.6.745" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=14668115" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000187185200002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=52&amp;publication_year=2003&amp;pages=745-756&amp;journal=Systematic+Biology&amp;author=B.M.+Wiegmann&amp;author=D.K.+Yeates&amp;author=J.L.+Thorne&amp;author=H+Kishino&amp;title=Time+flies%2C+a+new+molecular+time+scale+for+brachyceran+fly+evolution+without+a+clock" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b93">
                     <span class="author">Woodley, N.E</span>. (<span class="pubYear">1989</span>)  <span class="chapterTitle">Phylogeny and classification of the ‘Orthorraphous’ Bracycera</span>. 
 <span class="bookTitle">Manual of Nearctic Diptera</span>, Vol.  <span class="vol">3</span> (ed. by
					 <span class="editor">J. F. McAlpine</span> &amp; 
 <span class="editor">D. V. Wood</span>), pp.  <span class="pageFirst">1371</span>– <span class="pageLast">1395</span>. Research Branch,  Agriculture Canada, Ottawa.

                     <div class="extra-links" id="e_1_2_7_95_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=1989&amp;pages=1371-1395&amp;author=N.E+Woodley&amp;title=Manual+of+Nearctic+Diptera" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b94">
                     <span class="author">Yeates, D.K</span>. (<span class="pubYear">1994</span>)  <span class="articleTitle">Cladistics and classification of the Bombyliidae (Diptera: Asiloidae)</span>. 
<i>Bulletin of the American Museum of Natural History</i>,  <span class="vol">219</span>,  <span class="pageFirst">1</span>– <span class="pageLast">191</span>.

                     <div class="extra-links" id="e_1_2_7_96_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=219&amp;publication_year=1994&amp;pages=1-191&amp;journal=Bulletin+of+the+American+Museum+of+Natural+History&amp;author=D.K+Yeates&amp;title=Cladistics+and+classification+of+the+Bombyliidae+%28Diptera%3A+Asiloidae%29" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b95">
                     <span class="author">Yeates, D.K</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Relationships of extant lower Brachycera (Diptera): a quantitative synthesis of morphological characters</span>. 
<i>Zoologica Scripta</i>,  <span class="vol">31</span>,  <span class="pageFirst">105</span>– <span class="pageLast">121</span>.

                     <div class="extra-links" id="e_1_2_7_97_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.0300-3256.2001.00077.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000176475500010" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=31&amp;publication_year=2002&amp;pages=105-121&amp;journal=Zoologica+Scripta&amp;author=D.K+Yeates&amp;title=Relationships+of+extant+lower+Brachycera+%28Diptera%29%3A+a+quantitative+synthesis+of+morphological+characters" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b96">
                     <span class="author">Yeates, D.K.</span> &amp; 
<span class="author">Wiegmann, B.M</span>. (<span class="pubYear">1999</span>)  <span class="articleTitle">Congruence and controversy: towards a higher‐level phylogeny of Diptera</span>. 
<i>Annual Review of Entomology</i>,  <span class="vol">44</span>,  <span class="pageFirst">397</span>– <span class="pageLast">428</span>.

                     <div class="extra-links" id="e_1_2_7_98_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1146%2Fannurev.ento.44.1.397" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADyaK1MXpvFKgtA%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=15012378" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000078435900017" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=44&amp;publication_year=1999&amp;pages=397-428&amp;journal=Annual+Review+of+Entomology&amp;author=D.K.+Yeates&amp;author=B.M+Wiegmann&amp;title=Congruence+and+controversy%3A+towards+a+higher%E2%80%90level+phylogeny+of+Diptera" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b97">
                     <span class="author">Yeates, D.K.</span> &amp; 
<span class="author">Wiegmann, B.M</span>. (<span class="pubYear">2005</span>)  <span class="chapterTitle">Phylogeny and evolution of Diptera: recent insights and new perspectives</span>. 
 <span class="bookTitle">The Evolutionary Biology of Flies</span> (ed. by
					 <span class="editor">D. K. Yeates</span> &amp; 
 <span class="editor">B. M. Wiegmann</span>), pp.  <span class="pageFirst">14</span>– <span class="pageLast">44</span>.  Columbia University Press, New York.

                     <div class="extra-links" id="e_1_2_7_99_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2005&amp;pages=14-44&amp;author=D.K.+Yeates&amp;author=B.M+Wiegmann&amp;title=The+Evolutionary+Biology+of+Flies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b98">
                     <span class="author">Yukuhiro, K.</span>, 
<span class="author">Sezutsu, H.</span>, 
<span class="author">Itoh, H.</span>, 
<span class="author">Shimizu, K.</span> &amp; 
<span class="author">Banno, Y</span>. (<span class="pubYear">2002</span>)  <span class="articleTitle">Significant
 levels of sequence divergence and gene rearrangements have occurred 
between the mitochondrial genomes of the wild mulberry silkmoth,<i> Bombyx mandarina</i>, and its close relative, the domesticated silkmoth,<i> Bombyx mori</i></span>. 
<i>Molecular Biology and Evolution</i>,  <span class="vol">19</span>,  <span class="pageFirst">1385</span>– <span class="pageLast">1389</span>.

                     <div class="extra-links" id="e_1_2_7_100_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=10.1093%2Foxfordjournals.molbev.a004200" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=1%3ACAS%3A528%3ADC%252BD38XmtF2rs70%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=12140251" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x&amp;key=000177643900020" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=19&amp;publication_year=2002&amp;pages=1385-1389&amp;journal=Molecular+Biology+and+Evolution&amp;author=K.+Yukuhiro&amp;author=H.+Sezutsu&amp;author=H.+Itoh&amp;author=K.+Shimizu&amp;author=Y+Banno&amp;title=Significant+levels+of+sequence+divergence+and+gene+rearrangements+have+occurred+between+the+mitochondrial+genomes+of+the+wild+mulberry+silkmoth%2C+Bombyx+mandarina%2C+and+its+close+relative%2C+the+domesticated+silkmoth%2C+Bombyx+mori" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
   </section>
   <div style="display:none">
      <ul class="custom-style-list noteList"></ul>
   </div>
   <section id="citedBy" class="article-section article-section__citedBy" data-source="https://onlinelibrary.wiley.com/action/ajaxShowCitedBy?doi=10.1111%2Fj.1365-3113.2006.00355.x">
      <div class="accordion article-accordion">
         <h3><a href="#" class="accordion__control" role="button" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a></h3>
         <div class="accordion__content" role="region" aria-labelledby="citedby-section" id="idss15" style="display: none;"></div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md sticko__parent article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/13653113/2007/32/1"><img src="sen.gif" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/13653113/2007/32/1"><span>Volume</span><span class="val">32</span>, <span>Issue</span><span class="val">1</span></a></p><p>January 2007</p><p class="page-range"><span>Pages</span><span> 40-59</span></p>



        
        
</div>
</div>



        
        



        <div class="pb-ad">
            <div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script><div id="google_ads_iframe_/2215/wly.zoology_000120/systematic-entom_syen_1__container__" style="border: 0pt none; width: 300px; height: 250px;"></div>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x&pubId=59236166&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		</div>
        </div>
    

<div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko tab--flex tabs--xs js--sticko" style="width: 289.983px; top: 621.5px; position: static; height: 0px;"><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list cloned"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><ul role="presentation" class="rlist tab__content sticko__child" style="width: 289.983px; height: 0px; overflow-y: auto;"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tabpanel" class="tab__pane article-row-right__panes active" aria-hidden="true"><div class="contentTab"><!-- XSLT: InfoTab.xsl, Revision: 15213715, Interface: urn:atypon.com:wiley:interface:atypon-wiley-prime, was cached: Sat Aug 31 11:14:50 PDT 2019-->
<section class="info-tab-wrapper">
   <section class="metrics">
      <h3 class="info-header">Metrics</h3>
      <div class="section__body">
         <div class="cited-by-count"><label>Citations: </label><span>138</span></div>
         <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-info-tab-metrics">
            <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
<div data-badge-popover="left" data-badge-type="1" data-doi="10.1111/j.1365-3113.2006.00355.x" data-hide-no-mentions="false" class="altmetric-embed" data-uuid="a1dcfa31-1c0d-8c0b-2d9a-65676998eade"><a style="cursor: pointer; display: inline-block; background-image: url(&quot;https://d1uo4w7k31k5mn.cloudfront.net/v1/0.png&quot;); background-repeat: no-repeat; width: 110px; height: 20px;" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;doi=10.1111%2Fj.1365-3113.2006.00355.x" target="_self"> </a></div>
         </div>
      </div>
   </section>
   <section class="details">
      <h3 class="info-header">Details</h3>
      <p class="copyright">2006 The Royal Entomological Society</p>
      <p></p>
      <ul class="cross-mark rlist">
      </ul>
   </section>
   <section class="publication-history">
      <h3 class="info-header">Publication History</h3>
      <ul class="rlist">
         <li><label>Issue Online: </label>17 October 2006
         </li>
         <li><label>Version of Record online: </label>15 September 2006
         </li>
         <li><label>Editorial history: </label>Accepted 1 March 2006, First published online 15 September 2006
         </li>
      </ul>
   </section>
</section>
<!-- END OF XSLT -->
</div><div class="emptyTab">



        
        



        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script><div id="google_ads_iframe_/2215/wly.zoology_000120/systematic-entom_syen_2__container__" style="border: 0pt none; width: 300px; height: 250px;"></div>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x&pubId=59236166&placeholderId=1013&productId=101018" target="_blank">
				<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	</div>
</aside>
        </div>
    

</div></li><li class="tab__spinner" style="display: none;"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a href="#" data-ux3-role="controller" role="button" class="figure-viewer__ctrl__close"><span class="icon-close_thin"><span class="sr-only">Close Figure Viewer</span></span></a></div><div class="figure-viewer__top__left"><a href="#" role="button" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"><span class="sr-only">Browse All Figures</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"><span class="sr-only">Return to Figure</span></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: calc(100vh - 56px);"><div class="figure-viewer__cent__left"><a href="#" role="button" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"><span class="sr-only">Previous Figure</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"><span class="sr-only">Next Figure</span></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
        </div>
    </div><!--end pagefulltext-->
<div class="w-slide padding-top" style="top: 71px;"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main>

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>










        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h3 class="hidden accessibility-enabled">
                Additional links
            </h3>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
  <h3>About Wiley Online Library</h3>
  <ul class="rlist">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help &amp; Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2019 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    




        
        <div class="footer-logo pull-right">
        
<a href="http://www.wiley.com/" title="Wiley Home Page" aria-label="Wiley"><img alt="Wiley Home Page" src="footer-logo-wiley-1510029248417.png"></a>

</div>
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        <!-- [Tag{id&#x3D;320, code&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;81, code&#x3D;ag, name&#x3D;AGRICULTURE}, Tag{id&#x3D;321, code&#x3D;lsc0, name&#x3D;Entomology}, Tag{id&#x3D;561, code&#x3D;ag00, name&#x3D;General &amp; Introductory Agriculture}, Tag{id&#x3D;323, code&#x3D;ls40, name&#x3D;Evolution}, Tag{id&#x3D;1204, code&#x3D;ls44, name&#x3D;Systematics}, Tag{id&#x3D;327, code&#x3D;ls80, name&#x3D;Animal Science &amp; Zoology}, Tag{id&#x3D;1207, code&#x3D;ls41, name&#x3D;Evolutionary Biology}, Tag{id&#x3D;1193, code&#x3D;ls2b, name&#x3D;Behavior &amp; Behavioral Ecology}, Tag{id&#x3D;60, code&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;1183, code&#x3D;ls2j, name&#x3D;Biodiversity}, Tag{id&#x3D;103450, code&#x3D;systematic-entom_syen, name&#x3D;systematic-entom_SYEN}, Tag{id&#x3D;101813, code&#x3D;wly.zoology_000120, name&#x3D;wly.zoology_000120}] -->                            <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 




        
        
















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="loginPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="Cancel Login"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2 id="loginPopupHead">Log in to Wiley Online Library</h2>

            































<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        

        
        <input name="loginUri" value="/doi/full/10.1111/j.1365-3113.2006.00355.x" type="hidden">
        
            




<input name="redirectUri" value="/doi/full/10.1111/j.1365-3113.2006.00355.x" type="hidden">
<input name="loginUri" value="" type="hidden">
            
        
        
        
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="username">Email or Customer ID</label>
                
        </div>
        
        
                <input id="username" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="text">
            
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                



















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        
        
                <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="password">
            
        <span class="password-eye-icon icon-eye hidden"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            
                            <span class="submit " disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom accessSubmit" name="submitButton" value="Log In" disabled="disabled" type="submit">
                            </span>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                        
                        NEW USER <span aria-hidden="true">&gt;</span>
                    </a>
                    
                        <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x" class="linkBtn float-right instituation-login last-tab-item">
                            
                            INSTITUTIONAL LOGIN <span aria-hidden="true">&gt;</span>
                        </a><a class="fakeFocusoutElement" href="#"></a>
                    
                </div>
            

        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="changePasswordPopupHead">
    <div class="content">
        <a href="#" class="close cancel" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="changePasswordPopupHead">Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
                <a href="#" class="fakeFormLastElement last-tab-item"></a><a class="fakeFocusoutElement" href="#"></a>
            </div>
            <div class="success-template hidden">
                <h2>Password Changed Successfully</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        





<div class="popup registration-popup hidden" role="dialog" tabindex="-1" aria-labelledby="createAccountPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2 id="createAccountPopupHead">Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1111/j.1365-3113.2006.00355.x" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login last-tab-item">
                    
                    Returning user
                </a><a class="fakeFocusoutElement" href="#"></a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="resetPasswordPopupHead">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="resetPasswordPopupHead">Forgot your password?</h2>
          
            <p class="sub">Enter your email address below. </p>

        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        
        
                <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
            
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">

                <p>"Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>

            </div>
        </div>
    </div>
</div>





        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>




        
        




<div class="popup top-drawer request-username-drawer username-popup hidden" role="dialog" tabindex="-1" aria-labelledby="requestUsernamePopupHead">
    <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2 id="requestUsernamePopupHead">Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script>
        </div>
    </div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh600" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.335bb7dc7f89ace1b379cb85.html#rand=0.4507315700952257&amp;iit=1567694290554&amp;tmr=load%3D1567694285124%26core%3D1567694285172%26main%3D1567694290548%26ifr%3D1567694290558&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.1365-3113.2006.00355.x&amp;dt=A%20mitochondrial%20genome%20phylogeny%20of%20Diptera%3A%20whole%20genome%20sequence%20data%20accurately%20resolve%20relationships%20over%20broad%20timescales%20with%20high%20precision&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=description%2Csite_name%2Cimage%2Curl%2Ctype%3DArticle%2Ctitle&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5d711dcddcfe3691&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=title%3DA%2520mitochondrial%2520genome%2520phylogeny%2520of%2520Diptera%253A%2520whole%2520genome%2520sequence%2520data%2520accurately%2520resolve%2520relationships%2520over%2520broad%2520timescales%2520with%2520high%2520precision%26type%3DArticle%26url%3Dhttps%253A%252F%252Fonlinelibrary.wiley.com%252Fdoi%252F10.1111%252Fj.1365-3113.2006.00355.x%26image%3Dhttps%253A%252F%252Fwol-prod-cdn.literatumonline.com%252Fcms%252Fattachment%252F583e73f2-9e19-415a-9ddb-c31fb0cca199%252Fsen.2007.32.issue-1.cover.gif%26site_name%3DWiley%2520Online%2520Library%26description%3DAbstract%2520Mitochondrial%2520genomes%2520provide%2520a%2520promising%2520new%2520tool%2520for%2520understanding%2520deep%25E2%2580%2590level%2520insect%2520phylogenetics%252C%2520but%2520have%2520yet%2520to%2520be%2520evaluated%2520for%2520their%2520ability%2520to%2520resolve%2520intraordinal%2520relationships.%2520We...&amp;csi=undefined&amp;rev=v8.26.14-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_wileypublishing_0" name="destination_publishing_iframe_wileypublishing_0_name" style="display: none; width: 0px; height: 0px;" src="https://wileypublishing.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fonlinelibrary.wiley.com" class="aamIframeLoaded"></iframe>


<script type="text/javascript" src="main.js"></script>
<script type="text/javascript" src="hfpfproduct.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>






















    











<div class="kxhead" data-id="JMOONjNh" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20226"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20227"></span><span class="kxtag kxinvisible" data-id="20229"><script>
(function() {
  // adadvisor redirects to http://logger... so it's not https safe
  if (location.protocol == "http:") {
     var u = "https://aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e";
     (new Image()).src = u;
   }
})();
</script></span><span class="kxtag kxinvisible" data-id="20234"><script type="text/javascript">
var findPh = document.getElementsByTagName("h1")[0]
if(findPh != null) { var pageheader = document.getElementsByTagName("h1")[0].innerHTML }

// Using Omniture prop1 to produce page attribute Page Type
Krux('set', 'page_attr_omniture_prop1', typeof window.s =='object' && window.s['prop1']);

// Using Omniture prop2 to produce page attribute Page Subtype
Krux('set', 'page_attr_omniture_prop2', typeof window.s =='object' && window.s['prop2']);

// Using Omniture prop3 to produce page attribute Title
Krux('set', 'page_attr_omniture_prop3', typeof window.s =='object' && window.s['prop3']);

// Using Omniture prop4 to produce page attribute Subject Codes
Krux('set', 'page_attr_omniture_prop4', typeof window.s =='object' && window.s['prop4']);

// Using Omniture prop7 to produce page attribute Article Title
Krux('set', 'page_attr_omniture_prop7', typeof window.s =='object' && window.s['prop7']);

// Using Omniture prop8 to produce page attribute Journal Volume and Issue
Krux('set', 'page_attr_omniture_prop8', typeof window.s =='object' && window.s['prop8']);

// Using Omniture prop9 to produce page attribute Article Access State
Krux('set', 'page_attr_omniture_prop9', typeof window.s =='object' && window.s['prop9']);

// Using Omniture prop10 to produce page attribute Publication Access Status
Krux('set', 'page_attr_omniture_prop10', typeof window.s =='object' && window.s['prop10']);

// Using Omniture prop13 to produce page attribute Journal DOI
Krux('set', 'page_attr_omniture_prop13', typeof window.s =='object' && window.s['prop13']);

// Using Omniture prop14 to produce page attribute Article / Chapter DOI
Krux('set', 'page_attr_omniture_prop14', typeof window.s =='object' && window.s['prop14']);

// Using Omniture prop15 to produce page attribute Issue DOI
Krux('set', 'page_attr_omniture_prop15', typeof window.s =='object' && window.s['prop15']);

// Using Omniture prop16 to produce page attribute MRW Chapter
Krux('set', 'page_attr_omniture_prop16', typeof window.s =='object' && window.s['prop16']);

// Using Omniture prop20 to produce page attribute Site Search Query
Krux('set', 'page_attr_omniture_prop20', typeof window.s =='object' && window.s['prop20']);

// Using Omniture prop26 to produce page attribute Logged In or Out
Krux('set', 'user_attr_omniture_prop26', typeof window.s =='object' && window.s['prop26']);

// Using Omniture channel to produce page attribute Channel
Krux('set', 'page_attr_omniture_channel', typeof window.s =='object' && window.s['channel']);

// Using Omniture prop33 to produce page attribute Primary Subject Code
Krux('set', 'page_attr_omniture_prop33', typeof window.s =='object' && window.s['prop33']);

// Using pageheader to produce page attribute Page Header
Krux('scrape', { 'page_attr_wileyonlinelib_pageheader': {javascript: 'pageheader'}});

</script></span><span class="kxtag kxinvisible" data-id="20238"></span><span class="kxtag kxinvisible" data-id="20241"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20243"></span><span class="kxtag kxinvisible" data-id="20247"><script type="text/javascript">
(function() {
    (new Image()).src = '//aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e';
})();
</script></span><span class="kxtag kxinvisible" data-id="20248"><script>
(function(){
        var kuid = Krux('get', 'user');
        var prefix = window.location.protocol == 'https:' ? 'https:' : 'http:';
        if (kuid) {
           var url = prefix + '//pix.btrll.com/partner/868092.png';
           (new Image()).src = url;
        }
})();
</script></span><span class="kxtag kxinvisible" data-id="20251"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8da8b14d-5569-4bec-bcea-722864ee8d06&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20252"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bef9f122-393d-4c45-8d8d-32d8be7ac433&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iyzu39iyzud95&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20253"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';
 
            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
         
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8bf57916-aac8-4f01-a386-4baf103b3e1f&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=bckw15bckwu20&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20254"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };

        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20255"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=536f0daa-aaaa-4d9e-9a25-dde40646786a&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20256"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=7c6392c9-e878-492c-8b14-bf06e3828ebd&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rowp70rowpu60&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20257"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
        
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bb8ae0e2-9cd7-45b2-ad37-7737269627d8&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rsxs71rsxsk73&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span></div><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">Close crossmark popup</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe title="Crossmark service - Intentionally blank" aria-hidden="true" class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1111/j.1365-3113.2006.00355.x?callback=_altmetric.embed_callback&amp;domain=onlinelibrary.wiley.com&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=17-5"></script></body><iframe id="kx-proxy-JMOONjNh" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=JMOONjNh&amp;kxt=https%3A%2F%2Fonlinelibrary.wiley.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe><iframe src="https://injections.readcube.com/html/track_epdf.43d7bc64.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe></html>