<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script>

        <title>PopGenome: An Efficient Swiss Army Knife for Population 
Genomic Analyses in R | Molecular Biology and Evolution | Oxford 
Academic</title>

        <script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>




    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site-1495373344.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'mbe';
            App.SiteURL = 'academic.oup.com/mbe';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/mbe/article/31/7/1929/2925788","@type":"ScholarlyArticle","name":"PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R","datePublished":"2014-04-16","isPartOf":{"@id":"https://academic.oup.com/mbe/issue/31/7","@type":"PublicationIssue","issueNumber":"7","datePublished":"2014-07-01","isPartOf":{"@id":"https://academic.oup.com/mbe","@type":"Periodical","name":"Molecular Biology and Evolution","issn":["1537-1719"]}},"url":"http://dx.doi.org/10.1093/molbev/msu136","keywords":["population genomics","software","single-nucleotide polymorphisms"],"inLanguage":"en","copyrightHolder":"Society for Molecular Biology and Evolution","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/mbe/article/31/7/1929/2925788","author":[{"name":"Pfeifer, Bastian","affiliation":"Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany","@type":"Person"},{"name":"Wittelsbürger, Ulrich","affiliation":"Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany","@type":"Person"},{"name":"Ramos-Onsins, Sebastian E.","affiliation":"Centre for Research in Agricultural Genomics, Bellaterra, Spain","@type":"Person"},{"name":"Lercher, Martin J.","affiliation":"Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany  Cluster of Excellence on Plant Sciences, Düsseldorf, Germany","@type":"Person"}],"description":"Abstract.  Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they ","pageStart":"1929","pageEnd":"1936","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/31/7/0/m_mbe31_7.cover.gif?Expires=1637529725&Signature=0KKIT28wqF6rDpeLvs7UbXCv6VeTiA3ZIavhy1Z4ZmWHZBxFfNclEQs-FUJhSla3nDlOsVAC~Lpqse3WjW-gX1lIZTX0ovC5m9MUEBE1lD3oFdYbhyw1woth~Hn~kBQR7KxntRQfKfMPPJBN4EtusXN3QjlMezB9VpmedyrORlvZYtRWJLTyqo1fMaErc7zYj746N4W9wsq57dHMByfqa7LYOdxaZZoJk4UP1X2Bs5K2gfvUmGGa~9JF5CBxrYpXEiKif0aIyWk3vOhkqIyMos70aDrY3zLRF6R3NdswKTuruwfPvB-Pz6eV1Xe4~8kHRBBcqvzmXRCNH1mC0xCFbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/31/7/0/m_mbe31_7.cover.gif?Expires=1637529725&Signature=0KKIT28wqF6rDpeLvs7UbXCv6VeTiA3ZIavhy1Z4ZmWHZBxFfNclEQs-FUJhSla3nDlOsVAC~Lpqse3WjW-gX1lIZTX0ovC5m9MUEBE1lD3oFdYbhyw1woth~Hn~kBQR7KxntRQfKfMPPJBN4EtusXN3QjlMezB9VpmedyrORlvZYtRWJLTyqo1fMaErc7zYj746N4W9wsq57dHMByfqa7LYOdxaZZoJk4UP1X2Bs5K2gfvUmGGa~9JF5CBxrYpXEiKif0aIyWk3vOhkqIyMos70aDrY3zLRF6R3NdswKTuruwfPvB-Pz6eV1Xe4~8kHRBBcqvzmXRCNH1mC0xCFbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R">
    <meta property="og:description" content="Abstract.  Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they ">
    <meta property="og:url" content="https://academic.oup.com/mbe/article/31/7/1929/2925788">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/31/7/0/m_mbe31_7.cover.gif?Expires=1637529725&amp;Signature=0KKIT28wqF6rDpeLvs7UbXCv6VeTiA3ZIavhy1Z4ZmWHZBxFfNclEQs-FUJhSla3nDlOsVAC~Lpqse3WjW-gX1lIZTX0ovC5m9MUEBE1lD3oFdYbhyw1woth~Hn~kBQR7KxntRQfKfMPPJBN4EtusXN3QjlMezB9VpmedyrORlvZYtRWJLTyqo1fMaErc7zYj746N4W9wsq57dHMByfqa7LYOdxaZZoJk4UP1X2Bs5K2gfvUmGGa~9JF5CBxrYpXEiKif0aIyWk3vOhkqIyMos70aDrY3zLRF6R3NdswKTuruwfPvB-Pz6eV1Xe4~8kHRBBcqvzmXRCNH1mC0xCFbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2014-04-16">
<meta name="citation_author" content="Pfeifer, Bastian"><meta name="citation_author_institution" content="Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany"><meta name="citation_author" content="Wittelsbürger, Ulrich"><meta name="citation_author_institution" content="Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany"><meta name="citation_author" content="Ramos-Onsins, Sebastian E."><meta name="citation_author_institution" content="Centre for Research in Agricultural Genomics, Bellaterra, Spain"><meta name="citation_author" content="Lercher, Martin J."><meta name="citation_author_institution" content="Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany"><meta name="citation_author_institution" content="Cluster of Excellence on Plant Sciences, Düsseldorf, Germany"><meta name="citation_title" content="PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R"><meta name="citation_firstpage" content="1929"><meta name="citation_lastpage" content="1936"><meta name="citation_doi" content="10.1093/molbev/msu136"><meta name="citation_journal_title" content="Molecular Biology and Evolution"><meta name="citation_journal_abbrev" content="Mol Biol Evol"><meta name="citation_volume" content="31"><meta name="citation_issue" content="7"><meta name="citation_publication_date" content="2014/07/01"><meta name="citation_issn" content="0737-4038"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Frequency spectrum neutrality tests: one for all and all for one; Genetics;  citation_year=2009;  citation_volume=183;  citation_issue=1;  citation_pages=249-258; "><meta name="citation_reference" content="citation_title=ff: memory-efficient storage of large data on disk and fast access functions [R package version 2.2-11]. [cited 2013 Dec]; citation_text=Available from: http://CRAN.R-project.org/package=ff;  citation_year=2013; "><meta name="citation_reference" content="citation_title=PGEToolbox: a Matlab toolbox for population genetics and evolution; J Hered.;  citation_year=2008;  citation_volume=99;  citation_issue=4;  citation_pages=438-440; "><meta name="citation_reference" content="citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations; Nat Genet.;  citation_year=2011;  citation_volume=43;  citation_issue=10;  citation_pages=956-963; "><meta name="citation_reference" content="citation_title=MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_issue=16;  citation_pages=2064-2065; "><meta name="citation_reference" content="citation_title=Arlequin (version 3.0): an integrated software package for population genetics data analysis; Evol Bioinform Online.;  citation_year=2005;  citation_volume=1;  citation_pages=47-50; "><meta name="citation_reference" content="citation_title=Analysis of molecular variance inferred from metric distances among DNA haplotypes: application to human mitochondrial DNA restriction data; Genetics;  citation_year=1992;  citation_volume=131;  citation_issue=2;  citation_pages=479-491; "><meta name="citation_reference" content="citation_title=Hitchhiking under positive Darwinian selection; Genetics;  citation_year=2000;  citation_volume=155;  citation_issue=3;  citation_pages=1405-1413; "><meta name="citation_reference" content="citation_title=A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective; Genetics;  citation_year=2008;  citation_volume=180;  citation_issue=2;  citation_pages=977-993; "><meta name="citation_reference" content="citation_title=Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection; Genetics;  citation_year=1997;  citation_volume=147;  citation_issue=2;  citation_pages=915-925; "><meta name="citation_reference" content="citation_title=Statistical tests of neutrality of mutations; Genetics;  citation_year=1993;  citation_volume=133;  citation_issue=3;  citation_pages=693-709; "><meta name="citation_reference" content="citation_title=Understanding genetic variation and function—the applications of next generation sequencing; Semin Cell Dev Biol.;  citation_year=2012;  citation_volume=23;  citation_issue=2;  citation_pages=230-236; "><meta name="citation_reference" content="citation_title=Balancing selection and MHC; Genetica;  citation_year=1998;  citation_volume=104;  citation_issue=3;  citation_pages=207-214; "><meta name="citation_reference" content="citation_title=A new statistic for detecting genetic differentiation; Genetics;  citation_year=2000;  citation_volume=155;  citation_issue=4;  citation_pages=2011-2014; "><meta name="citation_reference" content="citation_title=A statistical test for detecting geographic subdivision; Mol Biol Evol.;  citation_year=1992;  citation_volume=9;  citation_issue=1;  citation_pages=138-151; "><meta name="citation_reference" content="citation_title=Statistical properties of the number of recombination events in the history of a sample of DNA sequences; Genetics;  citation_year=1985;  citation_volume=111;  citation_issue=1;  citation_pages=147-164; "><meta name="citation_reference" content="citation_title=Generating samples under a Wright–Fisher neutral model of genetic variation; Bioinformatics Appl Note.;  citation_year=2002;  citation_volume=18;  citation_issue=2;  citation_pages=337-338; "><meta name="citation_reference" content="citation_title=Estimation of levels of gene flow from DNA sequence data; Genetics;  citation_year=1992;  citation_volume=132;  citation_issue=2;  citation_pages=583-589; "><meta name="citation_reference" content="citation_title=adegenet: a R package for the multivariate analysis of genetic markers; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_issue=11;  citation_pages=1403-1405; "><meta name="citation_reference" content="citation_title=A test of neutrality based on interlocus associations; Genetics;  citation_year=1997;  citation_volume=146;  citation_pages=1197-1206; "><meta name="citation_reference" content="citation_title=Tabix: fast retrieval of sequence features from generic TAB-delimited files; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_issue=5;  citation_pages=718-719; "><meta name="citation_reference" content="citation_title=Adaptive protein evolution at the Adh locus in Drosophila; Nature;  citation_year=1991;  citation_volume=351;  citation_issue=6328;  citation_pages=652-654; "><meta name="citation_reference" content="citation_title=Analysis of gene diversity in subdivided populations; Proc Natl Acad Sci U S A.;  citation_year=1973;  citation_volume=70;  citation_issue=12;  citation_pages=3321-3323; "><meta name="citation_reference" content="citation_title=Mathematical model for studying genetic variation in terms of restriction endonucleases; Proc Natl Acad Sci U S A.;  citation_year=1979;  citation_volume=76;  citation_issue=10;  citation_pages=5269-5273; "><meta name="citation_reference" content="citation_title=Genomic scans for selective sweeps using SNP data; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1566-1575; "><meta name="citation_reference" content="citation_title=pegas: an R package for population genetics with an integrated-modular approach; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_issue=3;  citation_pages=419-420; "><meta name="citation_reference" content="citation_title=SweeD: likelihood-based detection of selective sweeps in thousands of genomes; Mol Biol Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2224-2234; "><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; Am J Hum Genet.;  citation_year=2007;  citation_volume=81;  citation_issue=3;  citation_pages=559-575; "><meta name="citation_reference" content="citation_title=Statistical properties of new neutrality tests against population growth; Mol Biol Evol.;  citation_year=2002;  citation_volume=19;  citation_issue=12;  citation_pages=2092-2100; "><meta name="citation_reference" content="citation_title=DNA variation at the rp49 gene region of Drosophila simulans: evolutionary inferences from an unusual haplotype structure; Genetics;  citation_year=2001;  citation_volume=158;  citation_issue=3;  citation_pages=1147-1155; "><meta name="citation_reference" content="citation_title=DnaSP, DNA polymorphism analyses by the coalescent and other methods; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_issue=18;  citation_pages=2496-2497; "><meta name="citation_reference" content="citation_title=Average number of nucleotide differences in a sample from a single subpopulation: a test for population subdivision; Genetics;  citation_year=1987;  citation_volume=117;  citation_issue=1;  citation_pages=149-153; "><meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism; Genetics;  citation_year=1989;  citation_volume=123;  citation_issue=3;  citation_pages=585-595; "><meta name="citation_reference" content="citation_title=R/parallel—speeding up bioinformatics analysis with R; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=390"><meta name="citation_reference" content="citation_title=VariScan: analysis of evolutionary patterns from large-scale DNA sequence polymorphism data; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_issue=11;  citation_pages=2791-2793; "><meta name="citation_reference" content="citation_title=Recombination and the power of statistical tests of neutrality; Genet Res.;  citation_year=1999;  citation_volume=74;  citation_issue=1;  citation_pages=65-79; "><meta name="citation_reference" content="citation_title=Statistical tests for detecting positive selection by utilizing high-frequency variants; Genetics;  citation_year=2006;  citation_volume=174;  citation_issue=3;  citation_pages=1431-1439; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/mbe/article-pdf/31/7/1929/13171354/msu136.pdf"><meta name="description" content="Abstract.  Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they ">    <script id="altmetric-embed-js" src="altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019111801.js" async=""></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2667a43520a.css"></head><body data-sitestyletemplate="Journal" theme-mbe="" class="off-canvas pg_Article pg_article" data-sitename="molecularbiologyandevolution"><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/mbe/article-xml/31/7/1929/2925788">



    <link rel="canonical" href="https://academic.oup.com/mbe/article/31/7/1929/2925788">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "mbe/article/31/7/1929/2925788");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    
    

    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=molbev&authzrequired=false&doi=10.1093/molbev/msu136';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5325" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f7%2f1929%2f2925788" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f7%2f1929%2f2925788" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429235">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429232">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">More content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429236">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429237">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%E2%80%93+Newest+First&amp;f_TocHeadingTitle=Fast+TracksANDFAST+TRACK" class="nav-link ">Fast Track</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429238">
            <a href="https://academic.oup.com/mbe/search-results?allJournals=1&amp;f_TocHeadingTitle=Perspective&amp;fl_SiteID=5325&amp;page=1&amp;sort=Date+%e2%80%93+Newest+First" class="nav-link ">Perspectives</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429239">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Discoveries&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Discoveries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429240">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=METHODS&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Methods</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429241">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Resources&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Resources</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429242">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;f_TocHeadingTitle=PROTOCOLSANDProtocol&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Protocols</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429243">
            <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=News" class="nav-link ">News</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429233">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429244">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429245">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429246">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429247">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429248">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429249">
            <a href="https://academic.oup.com/mbe/pages/Open_Access" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429250">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429251">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429234">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429252">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429253">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429254">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429255">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429256">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429257">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429258">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429259">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429260">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/mbe" class="journal-logo-container">
                    <img id="logo-MolecularBiologyandEvolution" class="journal-logo" src="mbe_title188616415.svg" alt="Molecular Biology and Evolution">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.smbe.org/smbe/" target="" class="society-logo-container">
                                <img id="logo-SocietyforMolecularBiologyandEvolution" class="society-logo" src="mbe_h1-1277340080.svg" alt="Society for Molecular Biology and Evolution">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429235">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429232">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">More content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429236">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429237">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%E2%80%93+Newest+First&amp;f_TocHeadingTitle=Fast+TracksANDFAST+TRACK" class="nav-link ">Fast Track</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429238">
            <a href="https://academic.oup.com/mbe/search-results?allJournals=1&amp;f_TocHeadingTitle=Perspective&amp;fl_SiteID=5325&amp;page=1&amp;sort=Date+%e2%80%93+Newest+First" class="nav-link ">Perspectives</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429239">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Discoveries&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Discoveries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429240">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=METHODS&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Methods</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429241">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Resources&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Resources</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429242">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;f_TocHeadingTitle=PROTOCOLSANDProtocol&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Protocols</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429243">
            <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=News" class="nav-link ">News</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429233">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429244">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429245">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429246">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429247">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429248">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429249">
            <a href="https://academic.oup.com/mbe/pages/Open_Access" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429250">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429251">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1429234">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429252">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429253">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429254">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429255">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429256">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429257">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429258">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429259">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1429260">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option>
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="SMBE Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/smbe" type="hidden">
    <input name="siteID" id="hfsiteID" value="5325" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5538" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=45641" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/mbe" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/mbe/issue/31/7"><img id="issueImage" class="fb-featured-image" src="m_mbe31_7.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/mbe/issue/31/7">
                        <div class="volume trailing-comma">Volume 31</div>
                        <div class="issue">Issue 7</div>

            </a>

        </div>
        <div class="ii-pub-date">
July 2014        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="74403028">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403028">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74403030">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403030">Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74403036">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403036">Description of PopGenome</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74403066">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403066">Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74403091">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403091">Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="74403096">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403096">Acknowledgments</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="74403098">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#74403098">References</a>
            </div>
        </li>
<li class="section-jump-link head-1"><div class="section-jump-link__link-wrap"><a class="jumplink js-jumplink scrollTo" href="#authorNotesSectionTitle">Author notes</a></div></li><li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/mbe/article/31/7/1923/2925773">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/mbe/article/31/7/1937/2925882">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bastian Pfeifer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Bastian+Pfeifer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pfeifer%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Pfeifer%20Bastian%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Bastian Pfeifer</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ulrich Wittelsbürger</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Ulrich+Wittelsb%c3%bcrger">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wittelsb%C3%BCrger%20U">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Wittelsb%C3%BCrger%20Ulrich%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Ulrich Wittelsbürger</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sebastian E. Ramos-Onsins</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>2</sup></div>Centre for Research in Agricultural Genomics, Bellaterra, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Sebastian+E.+Ramos-Onsins">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ramos-Onsins%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Ramos-Onsins%20Sebastian%20E.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Sebastian E. Ramos-Onsins</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin J. Lercher</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label"><sup>1</sup></div>Institute for Computer Science, Heinrich Heine University, Düsseldorf, Germany</div><div class="aff"><div class="label"><sup>3</sup></div>Cluster of Excellence on Plant Sciences, Düsseldorf, Germany</div>
        </div>


        <div class="info-author-correspondence">
            <strong>*Corresponding author:</strong> E-mail: <a href="mailto:lercher@cs.uni-duesseldorf.de" target="">lercher@cs.uni-duesseldorf.de</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Martin+J.+Lercher">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lercher%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Lercher%20Martin%20J.%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Martin J. Lercher</a><span class="delimiter"><i class="icon-general-mail"></i></span>
                        </span>

                            <div class="al-author-name al-author-footnotes">
                                <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="footnote-content"><strong>Associate editor:</strong> Juliette de Meaux</div></div></div> 
    </div>
                                </div>
                                <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                            </div>
                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Molecular Biology and Evolution</em>, Volume 31, Issue 7, July 2014, Pages 1929–1936, <a href="https://doi.org/10.1093/molbev/msu136">https://doi.org/10.1093/molbev/msu136</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">16 April 2014</div>
            </div>
        </div>
    </div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2925788" href="https://academic.oup.com/mbe/article-pdf/31/7/1929/13171354/msu136.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Bastian Pfeifer, Ulrich Wittelsbürger, Sebastian E. Ramos-Onsins,
 Martin J. Lercher,  PopGenome: An Efficient Swiss Army Knife for 
Population Genomic Analyses in R, <em>Molecular Biology and Evolution</em>, Volume 31, Issue 7, July 2014, Pages 1929–1936, <a href="https://doi.org/10.1093/molbev/msu136">https://doi.org/10.1093/molbev/msu136</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=2925788&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1537-1719&amp;title=PopGenome%3A%20An%20Efficient%20Swiss%20Army%20Knife%20for%20Population%20Genomic%20Analyses%20in%20R&amp;publicationDate=2014-04-16&amp;volumeNum=31&amp;issueNum=7&amp;author=Pfeifer%2C%20Bastian%3B%20Wittelsb%C3%BCrger%2C%20Ulrich&amp;startPage=1929&amp;endPage=1936&amp;contentId=10.1093%2Fmolbev%2Fmsu136&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f7%2f1929%2f2925788" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option>
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="74403028" id="74403028" class="abstract-title">Abstract</h2>
<section class="abstract"><p>Although many computer programs can perform
 population genetics calculations, they are typically limited in the 
analyses and data input formats they offer; few applications can process
 the large data sets produced by whole-genome resequencing projects. 
Furthermore, there is no coherent framework for the easy integration of 
new statistics into existing pipelines, hindering the development and 
application of new population genetics and genomics approaches. Here, we
 present PopGenome, a population genomics package for the R software 
environment (a de facto standard for statistical analyses). PopGenome 
can efficiently process genome-scale data as well as large sets of 
individual loci. It reads DNA alignments and single-nucleotide 
polymorphism (SNP) data sets in most common formats, including those 
used by the HapMap, 1000 human genomes, and 1001 Arabidopsis genomes 
projects. PopGenome also reads associated annotation files in GFF 
format, enabling users to easily define regions or classify SNPs based 
on their annotation; all analyses can also be applied to sliding 
windows. PopGenome offers a wide range of diverse population genetics 
analyses, including neutrality tests as well as statistics for 
population differentiation, linkage disequilibrium, and recombination. 
PopGenome is linked to Hudson’s MS and Ewing’s MSMS programs to assess 
statistical significance based on coalescent simulations. PopGenome’s 
integration in R facilitates effortless and reproducible downstream 
analyses as well as the production of publication-quality graphics. 
Developers can easily incorporate new analyses methods into the 
PopGenome framework. PopGenome and R are freely available from CRAN (<a class="link link-uri" href="http://cran.r-project.org/" target="">http://cran.r-project.org/</a>) for all major operating systems under the GNU General Public License.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;population genomics&quot;">population genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="software">software</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single-nucleotide polymorphisms&quot;">single-nucleotide polymorphisms</a></div><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=Resources">Resources</a>
                </div></div>
                    <h2 scrollto-destination="74403030" id="74403030" class="section-title">Introduction</h2>
<p>Recent sequencing technologies allow to map genetic variation across hundreds of individual genomes (<a reveal-id="msu136-B12" data-open="msu136-B12" class="link link-ref link-reveal xref-bibr">Harrison 2012</a>); notable examples are the 1000 genomes project in humans (1000genomes.org) and the 1001 genomes project in <em>Arabidopsis thaliana</em>
 (1001genomes.org). These technological developments have shifted the 
bottleneck in population genetics from data acquisition to data 
analysis.</p><p>Different software packages for population genetics (or 
population genomics) analyses typically have limited overlap in 
implemented statistics and accepted input formats. This diversity 
hampers both efficient data analysis and the quick dispersion of new 
statistical approaches. Many widely used software packages, such as 
DnaSP (<a reveal-id="msu136-B31" data-open="msu136-B31" class="link link-ref link-reveal xref-bibr">Rozas et al. 2003</a>),
 cannot handle the data formats developed for massive resequencing 
projects. Only few programs support the use of genomic annotation, and 
thus users interested in specific regions have to preprocess the data 
using other tools.</p><p>To address these issues, a new software tool for population genomic data analysis should: </p><ul class="bullet"><li><p>read data in a variety of input formats, including both traditional formats and those used by the major resequencing projects;</p></li><li><p>implement a comprehensive range of population genetics/genomics analyses and statistics;</p></li><li><p>read associated annotation files and allow to systematically select regions of interest;</p></li><li><p>be able to analyze individual loci, multiple loci, and sliding windows;</p></li><li><p>be open source and be easily extendable by the scientific community to incorporate new types of analyses;</p></li><li><p>be integrated with powerful numerical and graphical capabilities; and</p></li><li><p>be platform independent.</p></li></ul><p>We
 implemented these features in PopGenome, a package embedded in the 
freely available, platform-independent, statistical, and graphical 
computing environment R (<a class="link link-uri" href="http://cran.r-project.org/" target="">http://cran.r-project.org/</a>, last accessed April 30, 2014).</p>                    <h2 scrollto-destination="74403036" id="74403036" class="section-title">Description of PopGenome</h2>
                    <h3 scrollto-destination="74403037" id="74403037" class="section-title">Summary</h3>
<p>To fully exploit the capabilities of the R statistical and graphical 
environment, and to allow the creation of stable workflows (scripts), 
all processes in PopGenome are executed as command line functions. 
However, we anticipate that a graphical user interface implementing the 
most important functionalities will be available in the near future.</p><p>PopGenome
 is designed to facilitate the easy integration of virtually all major 
types of population genetics and population genomics analyses, and, as 
outlined below, its current version includes a large array of different 
statistics (see overview in <a reveal-id="msu136-T2" data-open="msu136-T2" class="link link-reveal link-table xref-fig">table 2</a>).
 The emphasis on extensibility was inspired by the way Bioconductor 
(bioconductor.org) has come to dominate the analysis of microarray data,
 where newly developed methods for specific tasks are easily integrated 
into a pre-existing larger framework, thereby obviating the need to 
recreate tasks shared with existing analysis pipelines. We hope that 
PopGenome may become the kernel of a similar paradigm in the analysis of
 population genomics data, enabling researchers to effortlessly 
implement and share new or modified statistics.</p><p>That PopGenome is 
embedded within the R framework not only facilitates the easy 
integration of extensions and stable workflows but also allows immediate
 and effortless postprocessing of analysis results with R’s powerful 
numerical and graphical capabilities.</p>                    <h3 scrollto-destination="74403041" id="74403041" class="section-title">Data Organization</h3>
<p>PopGenome can read data both as full alignments and in 
single-nucleotide polymorphism (SNP) formats such as those generated by 
large resequencing projects. PopGenome’s ability to simultaneously 
manage large numbers of loci, which allows for variation in sequence and
 population coverage, provides a convenient framework for multilocus 
analyses.</p><p>After reading data, PopGenome first converts it into a 
biallelic matrix, that is, a matrix whose rows correspond to sequences 
and whose columns correspond to SNP positions in the alignment. Entries 
are either 0, indicating the major allele, or 1, indicating the minor 
allele (with entries corresponding to unknown variants labeled NA). In 
PopGenome, this matrix is stored as part of a <em>GENOME</em> object, 
which contains additional data needed for downstream analyses; this 
includes information on missing data, as well as annotations for 
individual SNPs (e.g., if these are transitions/transversions, 
coding/noncoding, or synonymous/nonsynonymous in the case of coding 
sequences).</p><p>Large input files are split automatically into smaller
 chunks, with the resulting partial biallelic matrices stored on the 
hard disk. For this temporary storage, we use ff objects (<a reveal-id="msu136-B2" data-open="msu136-B2" class="link link-ref link-reveal xref-bibr">Adler et al. 2013</a>).
 These data structures are stored on disk but behave (almost) as if they
 were in RAM, by transparently mapping only a section (pagesize) in main
 memory. An ff object needs about 3 kB of RAM to store SNP data from 1 
million SNPs across 50 individuals. When analyzing larger data sets, 
PopGenome concatenates the partial biallelic matrices into a single 
temporary file.</p><p>This strategy allows to simultaneously process 
whole-chromosome or whole-genome SNP data from hundreds of individuals, 
as collected in the 1000 human genomes (1000genomes.org) and 1001 
Arabidopsis genomes (1001genomes.org) projects. To further speed up the 
reading process, we employ the R-package parallel (<a reveal-id="msu136-B34" data-open="msu136-B34" class="link link-ref link-reveal xref-bibr">Vera et al. 2008</a>) that facilitates parallel computations on computers with multiple cores/CPUs.</p><p>Most
 functions in PopGenome are implemented in C or C++ to speed up 
computations and to limit memory requirements. This also applies to the 
reading of genome-scale alignments and SNP data. Supported alignment 
formats include FASTA, NEXUS, MEGA, MAF, and Phylip. An almost de facto 
standard for whole-genome variation data is the Variant Call Format 
(VCF), used, among others, by the 1000 genomes project (1000genomes.org)
 and the UK10k project (uk10k.org). PopGenome can read large SNP data 
sets stored in this format very efficiently, using indexes created with 
Tabix (<a reveal-id="msu136-B22" data-open="msu136-B22" class="link link-ref link-reveal xref-bibr">Li 2011</a>).
 SNPs in defined regions can be extracted directly from the 
corresponding file without time-consuming search operations over the 
entire file, with input speeds exceeding 6,000 variant positions per 
second even on older desktop computers.</p><p>The implementations of 
PopGenome’s data access functions conform to a set of optimization 
guidelines to guarantee a minimal reading time. To the best of our 
knowledge, there is currently no general-purpose population genetics 
software capable of directly accessing VCF files with comparable speed. 
Typical times required to read large data sets are listed in <a reveal-id="msu136-T1" data-open="msu136-T1" class="link link-reveal link-table xref-fig">table 1</a>. </p>                        <a id="74403048" scrollto-destination="74403048"></a>
<div content-id="msu136-T1" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="msu136-T1" class="table-wrap-title" data-id="msu136-T1"><div class="label">Table 1.</div><div class="caption"><p>Times Required to Read Large Data Sets.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Data Set&nbsp;</th><th>Individuals&nbsp;</th><th>SNPs&nbsp;</th><th>Format&nbsp;</th><th>Time for Reading<a reveal-id="msu136-TF1" data-open="msu136-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="2">Arabidopsis (Chr 1)&nbsp;</td><td rowspan="2">80&nbsp;</td><td rowspan="2">1,200,000&nbsp;</td><td rowspan="2">SNP (1001 Genomes)&nbsp;</td><td>&lt;1 min<a reveal-id="msu136-TF2" data-open="msu136-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td>∼3 min&nbsp;</td></tr><tr><td>Human (Chr2: 100–150 Mb)&nbsp;</td><td>1,094&nbsp;</td><td>660,000&nbsp;</td><td>VCF (1000 Genomes)&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>3450 individual alignments&nbsp;</td><td>25&nbsp;</td><td>200,000&nbsp;</td><td>FASTA&nbsp;</td><td>∼15 s&nbsp;</td></tr></tbody></table></div><div content-id="msu136-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Data Set&nbsp;</th><th>Individuals&nbsp;</th><th>SNPs&nbsp;</th><th>Format&nbsp;</th><th>Time for Reading<a reveal-id="msu136-TF1" data-open="msu136-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="2">Arabidopsis (Chr 1)&nbsp;</td><td rowspan="2">80&nbsp;</td><td rowspan="2">1,200,000&nbsp;</td><td rowspan="2">SNP (1001 Genomes)&nbsp;</td><td>&lt;1 min<a reveal-id="msu136-TF2" data-open="msu136-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td>∼3 min&nbsp;</td></tr><tr><td>Human (Chr2: 100–150 Mb)&nbsp;</td><td>1,094&nbsp;</td><td>660,000&nbsp;</td><td>VCF (1000 Genomes)&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>3450 individual alignments&nbsp;</td><td>25&nbsp;</td><td>200,000&nbsp;</td><td>FASTA&nbsp;</td><td>∼15 s&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="msu136-TF1"><div class="fn"><p><sup>a</sup>Intel® Core™ i3-2130 CPU @ 3.40 GHz × 4, 8 GB RAM, with data stored in temporary files.</p></div></div><div content-id="msu136-TF2"><div class="fn"><p><sup>b</sup>Without temporary files, if sufficient RAM is available.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/74403048" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="msu136-T1" class="table-wrap-title" data-id="msu136-T1"><div class="label">Table 1.</div><div class="caption"><p>Times Required to Read Large Data Sets.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Data Set&nbsp;</th><th>Individuals&nbsp;</th><th>SNPs&nbsp;</th><th>Format&nbsp;</th><th>Time for Reading<a reveal-id="msu136-TF1" data-open="msu136-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="2">Arabidopsis (Chr 1)&nbsp;</td><td rowspan="2">80&nbsp;</td><td rowspan="2">1,200,000&nbsp;</td><td rowspan="2">SNP (1001 Genomes)&nbsp;</td><td>&lt;1 min<a reveal-id="msu136-TF2" data-open="msu136-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td>∼3 min&nbsp;</td></tr><tr><td>Human (Chr2: 100–150 Mb)&nbsp;</td><td>1,094&nbsp;</td><td>660,000&nbsp;</td><td>VCF (1000 Genomes)&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>3450 individual alignments&nbsp;</td><td>25&nbsp;</td><td>200,000&nbsp;</td><td>FASTA&nbsp;</td><td>∼15 s&nbsp;</td></tr></tbody></table></div><div content-id="msu136-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Data Set&nbsp;</th><th>Individuals&nbsp;</th><th>SNPs&nbsp;</th><th>Format&nbsp;</th><th>Time for Reading<a reveal-id="msu136-TF1" data-open="msu136-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td rowspan="2">Arabidopsis (Chr 1)&nbsp;</td><td rowspan="2">80&nbsp;</td><td rowspan="2">1,200,000&nbsp;</td><td rowspan="2">SNP (1001 Genomes)&nbsp;</td><td>&lt;1 min<a reveal-id="msu136-TF2" data-open="msu136-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>&nbsp;</td></tr><tr><td>∼3 min&nbsp;</td></tr><tr><td>Human (Chr2: 100–150 Mb)&nbsp;</td><td>1,094&nbsp;</td><td>660,000&nbsp;</td><td>VCF (1000 Genomes)&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>3450 individual alignments&nbsp;</td><td>25&nbsp;</td><td>200,000&nbsp;</td><td>FASTA&nbsp;</td><td>∼15 s&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="msu136-TF1"><div class="fn"><p><sup>a</sup>Intel® Core™ i3-2130 CPU @ 3.40 GHz × 4, 8 GB RAM, with data stored in temporary files.</p></div></div><div content-id="msu136-TF2"><div class="fn"><p><sup>b</sup>Without temporary files, if sufficient RAM is available.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/74403048" target="_blank">Open in new tab</a></div></div></div><p>PopGenome
 sessions can be saved on hard disk; thus, the conversion to the 
biallelic matrix needs to be performed only once per data set. The 
function region.as.fasta() can be used to export data of a specific 
region, a group of subsites, or the entire data set (i.e., all SNPs or 
even complete genome alignments) as a FASTA file. A parameter 
include.unknown indicates if unknown positions should be included in the
 analyses.</p>                    <h3 scrollto-destination="74403050" id="74403050" class="section-title">Implemented Methods</h3>
<p>To structure the rich landscape of population genetics and genomics 
analysis methods, PopGenome partitions the implemented methods into 
modules. Currently, PopGenome provides nine modules (<a reveal-id="msu136-T2" data-open="msu136-T2" class="link link-reveal link-table xref-fig">table 2</a>).
 All modules use the GENOME object created when the input data were 
read, and store their results in the same GENOME object. For analyses 
that require to distinguish between ancestral and derived alleles, 
outgroups can be specified. </p>                        <a id="74403052" scrollto-destination="74403052"></a>
<div content-id="msu136-T2" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="msu136-T2" class="table-wrap-title" data-id="msu136-T2"><div class="label">Table 2.</div><div class="caption"><p>Population Genetics Statistics Implemented in PopGenome’s Modules.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Module&nbsp;</th><th>Statistics&nbsp;</th></tr></thead><tbody><tr><td>Neutrality statistics&nbsp;</td><td>Tajima’s <em>D</em> (<a reveal-id="msu136-B33" data-open="msu136-B33" class="link link-ref link-reveal xref-bibr">Tajima 1989</a>), Fu and Li’s <em>F</em>* &amp; <em>D</em>* (<a reveal-id="msu136-B11" data-open="msu136-B11" class="link link-ref link-reveal xref-bibr">Fu and Li 1993</a>), Fay and Wu’s <em>H</em> (<a reveal-id="msu136-B8" data-open="msu136-B8" class="link link-ref link-reveal xref-bibr">Fay and Wu 2000</a>), Zeng’s <em>E</em> (<a reveal-id="msu136-B37" data-open="msu136-B37" class="link link-ref link-reveal xref-bibr">Zeng et al. 2006</a>), Strobeck’s <em>S</em> (<a reveal-id="msu136-B32" data-open="msu136-B32" class="link link-ref link-reveal xref-bibr">Strobeck 1987</a>), Achaz’s <em>Y</em> (<a reveal-id="msu136-B1" data-open="msu136-B1" class="link link-ref link-reveal xref-bibr">Achaz 2009</a>), Fu’s <em>F</em><sub>S</sub> (<a reveal-id="msu136-B10" data-open="msu136-B10" class="link link-ref link-reveal xref-bibr">Fu 1997</a>), Ramos-Onsins’ and Rozas’ <em>R</em>2 (<a reveal-id="msu136-B30" data-open="msu136-B30" class="link link-ref link-reveal xref-bibr">Ramos-Onsins and Rozas 2002</a>), as well as all corresponding theta values&nbsp;</td></tr><tr><td>Linkage disequilibrium&nbsp;</td><td>ZnS (<a reveal-id="msu136-B21" data-open="msu136-B21" class="link link-ref link-reveal xref-bibr">Kelly 1997</a>), B/Q (<a reveal-id="msu136-B36" data-open="msu136-B36" class="link link-ref link-reveal xref-bibr">Wall 1999</a>), ZA/ZZ (<a reveal-id="msu136-B38" data-open="msu136-B38" class="link link-ref link-reveal xref-bibr">Rozas et al. 2001</a>), and correlation coefficient <em>r<sup>2</sup></em> for each pair of SNPs within or between windows/regions&nbsp;</td></tr><tr><td>Recombination statistics&nbsp;</td><td>Four-gamete test (<a reveal-id="msu136-B16" data-open="msu136-B16" class="link link-ref link-reveal xref-bibr">Hudson and Kaplan 1985</a>)&nbsp;</td></tr><tr><td>Diversities&nbsp;</td><td>Nucleotide and haplotype diversity (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); (<a reveal-id="msu136-B25" data-open="msu136-B25" class="link link-ref link-reveal xref-bibr">Nei 1979</a>); see “Neutrality statistics” for a list of calculated Theta values&nbsp;</td></tr><tr><td>Selective sweeps&nbsp;</td><td>CL, CLR (<a reveal-id="msu136-B26" data-open="msu136-B26" class="link link-ref link-reveal xref-bibr">Nielsen et al. 2005</a>)&nbsp;</td></tr><tr><td>FST estimates&nbsp;</td><td><em>G</em><sub>ST</sub> (<a reveal-id="msu136-B24" data-open="msu136-B24" class="link link-ref link-reveal xref-bibr">Nei 1973</a>); <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B18" data-open="msu136-B18" class="link link-ref link-reveal xref-bibr">Hudson, Slatkin et al. 1992</a>); <em>G</em><sub>ST</sub>, <em>H</em><sub>ST</sub>, <em>K</em><sub>ST</sub> (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); <em>S</em><sub>nn</sub> (<a reveal-id="msu136-B14" data-open="msu136-B14" class="link link-ref link-reveal xref-bibr">Hudson 2000</a>); Phi<sub>ST</sub> (<a reveal-id="msu136-B7" data-open="msu136-B7" class="link link-ref link-reveal xref-bibr">Excoffier and Smouse 1992</a>)&nbsp;</td></tr><tr><td>MKT&nbsp;</td><td>McDonald–Kreitman test (<a reveal-id="msu136-B23" data-open="msu136-B23" class="link link-ref link-reveal xref-bibr">McDonald and Kreitman 1991</a>)&nbsp;</td></tr><tr><td>Mixed statistics&nbsp;</td><td>Site frequency spectrum; fixed and shared polymorphisms; biallelic structure&nbsp;</td></tr><tr><td>BayeScanR&nbsp;</td><td>Bayesian estimation of <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B9" data-open="msu136-B9" class="link link-ref link-reveal xref-bibr">Foll and Gaggiotti 2008</a>)&nbsp;</td></tr></tbody></table></div><div content-id="msu136-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>Module&nbsp;</th><th>Statistics&nbsp;</th></tr></thead><tbody><tr><td>Neutrality statistics&nbsp;</td><td>Tajima’s <em>D</em> (<a reveal-id="msu136-B33" data-open="msu136-B33" class="link link-ref link-reveal xref-bibr">Tajima 1989</a>), Fu and Li’s <em>F</em>* &amp; <em>D</em>* (<a reveal-id="msu136-B11" data-open="msu136-B11" class="link link-ref link-reveal xref-bibr">Fu and Li 1993</a>), Fay and Wu’s <em>H</em> (<a reveal-id="msu136-B8" data-open="msu136-B8" class="link link-ref link-reveal xref-bibr">Fay and Wu 2000</a>), Zeng’s <em>E</em> (<a reveal-id="msu136-B37" data-open="msu136-B37" class="link link-ref link-reveal xref-bibr">Zeng et al. 2006</a>), Strobeck’s <em>S</em> (<a reveal-id="msu136-B32" data-open="msu136-B32" class="link link-ref link-reveal xref-bibr">Strobeck 1987</a>), Achaz’s <em>Y</em> (<a reveal-id="msu136-B1" data-open="msu136-B1" class="link link-ref link-reveal xref-bibr">Achaz 2009</a>), Fu’s <em>F</em><sub>S</sub> (<a reveal-id="msu136-B10" data-open="msu136-B10" class="link link-ref link-reveal xref-bibr">Fu 1997</a>), Ramos-Onsins’ and Rozas’ <em>R</em>2 (<a reveal-id="msu136-B30" data-open="msu136-B30" class="link link-ref link-reveal xref-bibr">Ramos-Onsins and Rozas 2002</a>), as well as all corresponding theta values&nbsp;</td></tr><tr><td>Linkage disequilibrium&nbsp;</td><td>ZnS (<a reveal-id="msu136-B21" data-open="msu136-B21" class="link link-ref link-reveal xref-bibr">Kelly 1997</a>), B/Q (<a reveal-id="msu136-B36" data-open="msu136-B36" class="link link-ref link-reveal xref-bibr">Wall 1999</a>), ZA/ZZ (<a reveal-id="msu136-B38" data-open="msu136-B38" class="link link-ref link-reveal xref-bibr">Rozas et al. 2001</a>), and correlation coefficient <em>r<sup>2</sup></em> for each pair of SNPs within or between windows/regions&nbsp;</td></tr><tr><td>Recombination statistics&nbsp;</td><td>Four-gamete test (<a reveal-id="msu136-B16" data-open="msu136-B16" class="link link-ref link-reveal xref-bibr">Hudson and Kaplan 1985</a>)&nbsp;</td></tr><tr><td>Diversities&nbsp;</td><td>Nucleotide and haplotype diversity (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); (<a reveal-id="msu136-B25" data-open="msu136-B25" class="link link-ref link-reveal xref-bibr">Nei 1979</a>); see “Neutrality statistics” for a list of calculated Theta values&nbsp;</td></tr><tr><td>Selective sweeps&nbsp;</td><td>CL, CLR (<a reveal-id="msu136-B26" data-open="msu136-B26" class="link link-ref link-reveal xref-bibr">Nielsen et al. 2005</a>)&nbsp;</td></tr><tr><td>FST estimates&nbsp;</td><td><em>G</em><sub>ST</sub> (<a reveal-id="msu136-B24" data-open="msu136-B24" class="link link-ref link-reveal xref-bibr">Nei 1973</a>); <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B18" data-open="msu136-B18" class="link link-ref link-reveal xref-bibr">Hudson, Slatkin et al. 1992</a>); <em>G</em><sub>ST</sub>, <em>H</em><sub>ST</sub>, <em>K</em><sub>ST</sub> (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); <em>S</em><sub>nn</sub> (<a reveal-id="msu136-B14" data-open="msu136-B14" class="link link-ref link-reveal xref-bibr">Hudson 2000</a>); Phi<sub>ST</sub> (<a reveal-id="msu136-B7" data-open="msu136-B7" class="link link-ref link-reveal xref-bibr">Excoffier and Smouse 1992</a>)&nbsp;</td></tr><tr><td>MKT&nbsp;</td><td>McDonald–Kreitman test (<a reveal-id="msu136-B23" data-open="msu136-B23" class="link link-ref link-reveal xref-bibr">McDonald and Kreitman 1991</a>)&nbsp;</td></tr><tr><td>Mixed statistics&nbsp;</td><td>Site frequency spectrum; fixed and shared polymorphisms; biallelic structure&nbsp;</td></tr><tr><td>BayeScanR&nbsp;</td><td>Bayesian estimation of <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B9" data-open="msu136-B9" class="link link-ref link-reveal xref-bibr">Foll and Gaggiotti 2008</a>)&nbsp;</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/74403052" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="msu136-T2" class="table-wrap-title" data-id="msu136-T2"><div class="label">Table 2.</div><div class="caption"><p>Population Genetics Statistics Implemented in PopGenome’s Modules.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Module&nbsp;</th><th>Statistics&nbsp;</th></tr></thead><tbody><tr><td>Neutrality statistics&nbsp;</td><td>Tajima’s <em>D</em> (<a reveal-id="msu136-B33" data-open="msu136-B33" class="link link-ref link-reveal xref-bibr">Tajima 1989</a>), Fu and Li’s <em>F</em>* &amp; <em>D</em>* (<a reveal-id="msu136-B11" data-open="msu136-B11" class="link link-ref link-reveal xref-bibr">Fu and Li 1993</a>), Fay and Wu’s <em>H</em> (<a reveal-id="msu136-B8" data-open="msu136-B8" class="link link-ref link-reveal xref-bibr">Fay and Wu 2000</a>), Zeng’s <em>E</em> (<a reveal-id="msu136-B37" data-open="msu136-B37" class="link link-ref link-reveal xref-bibr">Zeng et al. 2006</a>), Strobeck’s <em>S</em> (<a reveal-id="msu136-B32" data-open="msu136-B32" class="link link-ref link-reveal xref-bibr">Strobeck 1987</a>), Achaz’s <em>Y</em> (<a reveal-id="msu136-B1" data-open="msu136-B1" class="link link-ref link-reveal xref-bibr">Achaz 2009</a>), Fu’s <em>F</em><sub>S</sub> (<a reveal-id="msu136-B10" data-open="msu136-B10" class="link link-ref link-reveal xref-bibr">Fu 1997</a>), Ramos-Onsins’ and Rozas’ <em>R</em>2 (<a reveal-id="msu136-B30" data-open="msu136-B30" class="link link-ref link-reveal xref-bibr">Ramos-Onsins and Rozas 2002</a>), as well as all corresponding theta values&nbsp;</td></tr><tr><td>Linkage disequilibrium&nbsp;</td><td>ZnS (<a reveal-id="msu136-B21" data-open="msu136-B21" class="link link-ref link-reveal xref-bibr">Kelly 1997</a>), B/Q (<a reveal-id="msu136-B36" data-open="msu136-B36" class="link link-ref link-reveal xref-bibr">Wall 1999</a>), ZA/ZZ (<a reveal-id="msu136-B38" data-open="msu136-B38" class="link link-ref link-reveal xref-bibr">Rozas et al. 2001</a>), and correlation coefficient <em>r<sup>2</sup></em> for each pair of SNPs within or between windows/regions&nbsp;</td></tr><tr><td>Recombination statistics&nbsp;</td><td>Four-gamete test (<a reveal-id="msu136-B16" data-open="msu136-B16" class="link link-ref link-reveal xref-bibr">Hudson and Kaplan 1985</a>)&nbsp;</td></tr><tr><td>Diversities&nbsp;</td><td>Nucleotide and haplotype diversity (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); (<a reveal-id="msu136-B25" data-open="msu136-B25" class="link link-ref link-reveal xref-bibr">Nei 1979</a>); see “Neutrality statistics” for a list of calculated Theta values&nbsp;</td></tr><tr><td>Selective sweeps&nbsp;</td><td>CL, CLR (<a reveal-id="msu136-B26" data-open="msu136-B26" class="link link-ref link-reveal xref-bibr">Nielsen et al. 2005</a>)&nbsp;</td></tr><tr><td>FST estimates&nbsp;</td><td><em>G</em><sub>ST</sub> (<a reveal-id="msu136-B24" data-open="msu136-B24" class="link link-ref link-reveal xref-bibr">Nei 1973</a>); <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B18" data-open="msu136-B18" class="link link-ref link-reveal xref-bibr">Hudson, Slatkin et al. 1992</a>); <em>G</em><sub>ST</sub>, <em>H</em><sub>ST</sub>, <em>K</em><sub>ST</sub> (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); <em>S</em><sub>nn</sub> (<a reveal-id="msu136-B14" data-open="msu136-B14" class="link link-ref link-reveal xref-bibr">Hudson 2000</a>); Phi<sub>ST</sub> (<a reveal-id="msu136-B7" data-open="msu136-B7" class="link link-ref link-reveal xref-bibr">Excoffier and Smouse 1992</a>)&nbsp;</td></tr><tr><td>MKT&nbsp;</td><td>McDonald–Kreitman test (<a reveal-id="msu136-B23" data-open="msu136-B23" class="link link-ref link-reveal xref-bibr">McDonald and Kreitman 1991</a>)&nbsp;</td></tr><tr><td>Mixed statistics&nbsp;</td><td>Site frequency spectrum; fixed and shared polymorphisms; biallelic structure&nbsp;</td></tr><tr><td>BayeScanR&nbsp;</td><td>Bayesian estimation of <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B9" data-open="msu136-B9" class="link link-ref link-reveal xref-bibr">Foll and Gaggiotti 2008</a>)&nbsp;</td></tr></tbody></table></div><div content-id="msu136-T2" class="table-modal"><table><thead name="thead" align="left"><tr><th>Module&nbsp;</th><th>Statistics&nbsp;</th></tr></thead><tbody><tr><td>Neutrality statistics&nbsp;</td><td>Tajima’s <em>D</em> (<a reveal-id="msu136-B33" data-open="msu136-B33" class="link link-ref link-reveal xref-bibr">Tajima 1989</a>), Fu and Li’s <em>F</em>* &amp; <em>D</em>* (<a reveal-id="msu136-B11" data-open="msu136-B11" class="link link-ref link-reveal xref-bibr">Fu and Li 1993</a>), Fay and Wu’s <em>H</em> (<a reveal-id="msu136-B8" data-open="msu136-B8" class="link link-ref link-reveal xref-bibr">Fay and Wu 2000</a>), Zeng’s <em>E</em> (<a reveal-id="msu136-B37" data-open="msu136-B37" class="link link-ref link-reveal xref-bibr">Zeng et al. 2006</a>), Strobeck’s <em>S</em> (<a reveal-id="msu136-B32" data-open="msu136-B32" class="link link-ref link-reveal xref-bibr">Strobeck 1987</a>), Achaz’s <em>Y</em> (<a reveal-id="msu136-B1" data-open="msu136-B1" class="link link-ref link-reveal xref-bibr">Achaz 2009</a>), Fu’s <em>F</em><sub>S</sub> (<a reveal-id="msu136-B10" data-open="msu136-B10" class="link link-ref link-reveal xref-bibr">Fu 1997</a>), Ramos-Onsins’ and Rozas’ <em>R</em>2 (<a reveal-id="msu136-B30" data-open="msu136-B30" class="link link-ref link-reveal xref-bibr">Ramos-Onsins and Rozas 2002</a>), as well as all corresponding theta values&nbsp;</td></tr><tr><td>Linkage disequilibrium&nbsp;</td><td>ZnS (<a reveal-id="msu136-B21" data-open="msu136-B21" class="link link-ref link-reveal xref-bibr">Kelly 1997</a>), B/Q (<a reveal-id="msu136-B36" data-open="msu136-B36" class="link link-ref link-reveal xref-bibr">Wall 1999</a>), ZA/ZZ (<a reveal-id="msu136-B38" data-open="msu136-B38" class="link link-ref link-reveal xref-bibr">Rozas et al. 2001</a>), and correlation coefficient <em>r<sup>2</sup></em> for each pair of SNPs within or between windows/regions&nbsp;</td></tr><tr><td>Recombination statistics&nbsp;</td><td>Four-gamete test (<a reveal-id="msu136-B16" data-open="msu136-B16" class="link link-ref link-reveal xref-bibr">Hudson and Kaplan 1985</a>)&nbsp;</td></tr><tr><td>Diversities&nbsp;</td><td>Nucleotide and haplotype diversity (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); (<a reveal-id="msu136-B25" data-open="msu136-B25" class="link link-ref link-reveal xref-bibr">Nei 1979</a>); see “Neutrality statistics” for a list of calculated Theta values&nbsp;</td></tr><tr><td>Selective sweeps&nbsp;</td><td>CL, CLR (<a reveal-id="msu136-B26" data-open="msu136-B26" class="link link-ref link-reveal xref-bibr">Nielsen et al. 2005</a>)&nbsp;</td></tr><tr><td>FST estimates&nbsp;</td><td><em>G</em><sub>ST</sub> (<a reveal-id="msu136-B24" data-open="msu136-B24" class="link link-ref link-reveal xref-bibr">Nei 1973</a>); <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B18" data-open="msu136-B18" class="link link-ref link-reveal xref-bibr">Hudson, Slatkin et al. 1992</a>); <em>G</em><sub>ST</sub>, <em>H</em><sub>ST</sub>, <em>K</em><sub>ST</sub> (<a reveal-id="msu136-B15" data-open="msu136-B15" class="link link-ref link-reveal xref-bibr">Hudson, Boos et al. 1992</a>); <em>S</em><sub>nn</sub> (<a reveal-id="msu136-B14" data-open="msu136-B14" class="link link-ref link-reveal xref-bibr">Hudson 2000</a>); Phi<sub>ST</sub> (<a reveal-id="msu136-B7" data-open="msu136-B7" class="link link-ref link-reveal xref-bibr">Excoffier and Smouse 1992</a>)&nbsp;</td></tr><tr><td>MKT&nbsp;</td><td>McDonald–Kreitman test (<a reveal-id="msu136-B23" data-open="msu136-B23" class="link link-ref link-reveal xref-bibr">McDonald and Kreitman 1991</a>)&nbsp;</td></tr><tr><td>Mixed statistics&nbsp;</td><td>Site frequency spectrum; fixed and shared polymorphisms; biallelic structure&nbsp;</td></tr><tr><td>BayeScanR&nbsp;</td><td>Bayesian estimation of <em>F</em><sub>ST</sub> (<a reveal-id="msu136-B9" data-open="msu136-B9" class="link link-ref link-reveal xref-bibr">Foll and Gaggiotti 2008</a>)&nbsp;</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/74403052" target="_blank">Open in new tab</a></div></div></div><p>As
 a default, all analyses packaged into one module are performed 
simultaneously when the module is executed. To accelerate calculations 
on large data sets, individual methods can be switched off using 
additional arguments. We plan to integrate more methods in the future, 
and welcome requests for the implementation of specific statistics. In 
the next release of PopGenome, we aim to incorporate methods for 
detecting recent selective sweeps, such as the algorithm implemented in 
the software SweeD (<a reveal-id="msu136-B28" data-open="msu136-B28" class="link link-ref link-reveal xref-bibr">Pavlidis et al. 2013</a>).</p><p>Apart
 from many “standard” statistics (e.g., neutrality and linkage 
disequilibrium statistics), PopGenome offers several tests for the 
detection of nonneutral evolution. So far, we have implemented the 
McDonald–Kreitmann test (<a reveal-id="msu136-B23" data-open="msu136-B23" class="link link-ref link-reveal xref-bibr">McDonald and Kreitman 1991</a>)
 and a wide range of FST measurements, including an implementation of a 
previously published method based on Bayesian statistics (<a reveal-id="msu136-B9" data-open="msu136-B9" class="link link-ref link-reveal xref-bibr">Foll and Gaggiotti 2008</a>). PopGenome also includes a calculation of <em>r</em><sup>2</sup> correlation coefficients and the corresponding <em>P</em>
 values (Fisher’s exact test) for interregion calculations. By 
concatenating the corresponding GENOME objects, statistics that rely on 
comparisons between regions can be calculated even when these are 
located on different chromosomes. Details of the implemented methods are
 given in <a reveal-id="msu136-T2" data-open="msu136-T2" class="link link-reveal link-table xref-fig">table 2</a> and in the PopGenome documentation.</p><p>PopGenome is fully integrated with two widely used coalescent simulation tools: Hudson’s MS (<a reveal-id="msu136-B17" data-open="msu136-B17" class="link link-ref link-reveal xref-bibr">Hudson 2002</a>), as well as Ewing’s MSMS (<a reveal-id="msu136-B5" data-open="msu136-B5" class="link link-ref link-reveal xref-bibr">Ewing and Hermisson 2010</a>),
 which incorporates selection. The PopGenome function MS() compares the 
statistics calculated for the observed data with corresponding data 
simulated by the coalescent method. PopGenome supports the full 
coalescent simulation capabilities of MS and MSMS. Parameters can be 
specified as vectors in the dedicated class “cs.stats,” and thus 
different models (mutation rates, migration rates, etc.) can be applied 
to different windows or regions of the genome. PopGenome’s MS() function
 stores the calculated statistics of coalescent simulations in a 
dedicated R object. Direct comparison to coalescent simulations is 
currently implemented for the modules Neutrality statistics, Linkage, 
and FST. If statistics from other modules need to be compared with 
coalescent simulations, PopGenome can directly read MS output files and 
then process these data using the method of interest (readMS()).</p><p>R
 is an efficient environment for large-scale computations. However, 
although most native R functions are implemented in C or Fortran, R 
itself is an interpreted and vector-oriented language. As a consequence,
 some types of calculations tend to be slow when applied to large 
objects. To avoid major bottlenecks, we implemented several specialized 
calculations in C++. PopGenome finishes the calculation of most 
statistics in minutes even for very large data sets (<a reveal-id="msu136-T3" data-open="msu136-T3" class="link link-reveal link-table xref-fig">table 3</a>). </p>                        <a id="74403057" scrollto-destination="74403057"></a>
<div content-id="msu136-T3" class="table-modal table-full-width-wrap" style="display: block;"><div class="table-wrap"><div id="msu136-T3" class="table-wrap-title" data-id="msu136-T3"><div class="label">Table 3.</div><div class="caption"><p>Calculation Speed for Haplotype and Nucleotide Diversity in Sliding Windows.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Sliding Window (nucleotides)&nbsp;</th><th>Running Time<a reveal-id="msu136-TF3" data-open="msu136-TF3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td>Arabidopsis (Chr 1)&nbsp;</td><td>Window size = 10,000&nbsp;</td><td>∼30 s&nbsp;</td></tr><tr><td>80 individuals&nbsp;</td><td>Jump size = 10,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>1,200,000 SNPs&nbsp;</td><td>Number of windows = 3,042&nbsp;</td><td>&nbsp;</td></tr><tr><td>Human (Chr 2: 100–150 Mb)&nbsp;</td><td>Window size = 1,000&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>1,094 individuals&nbsp;</td><td>Jump size = 1,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>660,000 SNPs&nbsp;</td><td>Number of windows = 50,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>3,450 alignments&nbsp;</td><td>3,450 windows (alignments)&nbsp;</td><td>∼7 s&nbsp;</td></tr><tr><td>25 individuals&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>5,086,953 sites&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>200,097 SNPs&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div content-id="msu136-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Sliding Window (nucleotides)&nbsp;</th><th>Running Time<a reveal-id="msu136-TF3" data-open="msu136-TF3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td>Arabidopsis (Chr 1)&nbsp;</td><td>Window size = 10,000&nbsp;</td><td>∼30 s&nbsp;</td></tr><tr><td>80 individuals&nbsp;</td><td>Jump size = 10,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>1,200,000 SNPs&nbsp;</td><td>Number of windows = 3,042&nbsp;</td><td>&nbsp;</td></tr><tr><td>Human (Chr 2: 100–150 Mb)&nbsp;</td><td>Window size = 1,000&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>1,094 individuals&nbsp;</td><td>Jump size = 1,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>660,000 SNPs&nbsp;</td><td>Number of windows = 50,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>3,450 alignments&nbsp;</td><td>3,450 windows (alignments)&nbsp;</td><td>∼7 s&nbsp;</td></tr><tr><td>25 individuals&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>5,086,953 sites&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>200,097 SNPs&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="msu136-TF3"><div class="fn"><p><sup>a</sup>Intel® Core™ i3-2130 CPU @ 3.40 GHz × 4, 8 GB RAM.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/74403057" target="_blank">Open in new tab</a></div></div></div><div class="table-full-width-wrap"><div class="table-wrap"><div id="msu136-T3" class="table-wrap-title" data-id="msu136-T3"><div class="label">Table 3.</div><div class="caption"><p>Calculation Speed for Haplotype and Nucleotide Diversity in Sliding Windows.</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Sliding Window (nucleotides)&nbsp;</th><th>Running Time<a reveal-id="msu136-TF3" data-open="msu136-TF3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td>Arabidopsis (Chr 1)&nbsp;</td><td>Window size = 10,000&nbsp;</td><td>∼30 s&nbsp;</td></tr><tr><td>80 individuals&nbsp;</td><td>Jump size = 10,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>1,200,000 SNPs&nbsp;</td><td>Number of windows = 3,042&nbsp;</td><td>&nbsp;</td></tr><tr><td>Human (Chr 2: 100–150 Mb)&nbsp;</td><td>Window size = 1,000&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>1,094 individuals&nbsp;</td><td>Jump size = 1,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>660,000 SNPs&nbsp;</td><td>Number of windows = 50,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>3,450 alignments&nbsp;</td><td>3,450 windows (alignments)&nbsp;</td><td>∼7 s&nbsp;</td></tr><tr><td>25 individuals&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>5,086,953 sites&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>200,097 SNPs&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div content-id="msu136-T3" class="table-modal"><table><thead name="thead" align="left"><tr><th>Data&nbsp;</th><th>Sliding Window (nucleotides)&nbsp;</th><th>Running Time<a reveal-id="msu136-TF3" data-open="msu136-TF3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>&nbsp;</th></tr></thead><tbody><tr><td>Arabidopsis (Chr 1)&nbsp;</td><td>Window size = 10,000&nbsp;</td><td>∼30 s&nbsp;</td></tr><tr><td>80 individuals&nbsp;</td><td>Jump size = 10,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>1,200,000 SNPs&nbsp;</td><td>Number of windows = 3,042&nbsp;</td><td>&nbsp;</td></tr><tr><td>Human (Chr 2: 100–150 Mb)&nbsp;</td><td>Window size = 1,000&nbsp;</td><td>∼5 min&nbsp;</td></tr><tr><td>1,094 individuals&nbsp;</td><td>Jump size = 1,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>660,000 SNPs&nbsp;</td><td>Number of windows = 50,000&nbsp;</td><td>&nbsp;</td></tr><tr><td>3,450 alignments&nbsp;</td><td>3,450 windows (alignments)&nbsp;</td><td>∼7 s&nbsp;</td></tr><tr><td>25 individuals&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>5,086,953 sites&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr><tr><td>200,097 SNPs&nbsp;</td><td>&nbsp;</td><td>&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="msu136-TF3"><div class="fn"><p><sup>a</sup>Intel® Core™ i3-2130 CPU @ 3.40 GHz × 4, 8 GB RAM.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/74403057" target="_blank">Open in new tab</a></div></div></div>                    <h3 scrollto-destination="74403058" id="74403058" class="section-title">Partitioning and Interpreting SNPs</h3>
<p>Users can restrict analyses to subregions specified either by genomic
 coordinates or positions in SNP files. When an annotation file in GFF 
format is present (GFF v2 or v3), PopGenome will automatically label 
SNPs located in genes, exons, coding regions, and UTRs. Other 
annotations of interest in the GFF file can be read using the function 
getgffinfo(). The user can apply the full range of implemented methods 
to all SNPs observed in a specific class (e.g., all exonic SNPs), or to 
each region specified in the GFF file separately (e.g., all introns 
individually).</p><p>If a GFF file is present, PopGenome can also 
classify synonymous and nonsynonymous sites; for SNP data formats, this 
additionally requires a reference genome in FASTA format. PopGenome 
stores the codons internally as numerical values, coded from a 
polynomial function as in the PGE Toolbox (<a reveal-id="msu136-B3" data-open="msu136-B3" class="link link-ref link-reveal xref-bibr">Cai 2008</a>).
 Based on the GFF file, the function get.codons() will provide 
information about the nature of amino acid changes resulting from the 
observed SNPs (encoded amino acids, charges, hydrophobicities, size, and
 polarity changes). Per default, PopGenome assumes the standard genetic 
code, but alternative codes can be specified.</p><p>Most multipurpose population genetics software tools are geared toward the analysis of discrete loci (<a reveal-id="msu136-B31" data-open="msu136-B31" class="link link-ref link-reveal xref-bibr">Rozas et al. 2003</a>; <a reveal-id="msu136-B6" data-open="msu136-B6" class="link link-ref link-reveal xref-bibr">Excoffier et al. 2005</a>),
 restricting their utility for the analysis of whole-genome SNP data 
sets. One widely used approach to apply population genetics methods to 
whole-genome data is the analysis of sliding windows (<a reveal-id="msu136-B38" data-open="msu136-B38" class="link link-ref link-reveal xref-bibr">Rozas et al. 2001</a>).
 In PopGenome, users can freely choose window and jump sizes for sliding
 windows, measured either in nucleotides or in numbers of SNPs. The 
underlying algorithm copies the information (mostly pointer) stored in 
the GENOME object to another object of the same class, where the data 
are reorganized into the specified windows. The full spectrum of 
PopGenome methods is thus available both for arbitrarily large sets of 
individual loci and for systematic genomic scans.</p>                    <h3 scrollto-destination="74403062" id="74403062" class="section-title">Easy Integration of New Methods</h3>
<p>To be used by the scientific community at large, a new algorithm 
ideally has to be implemented in a framework that allows its efficient 
application to data in the diverse file formats commonly used in 
different resequencing projects. PopGenome is geared toward making this 
task as easy as possible. All information required for the calculation 
of population genetics statistics, including the biallelic matrix as 
well as genomic annotation, is stored in a GENOME object, which new 
methods can directly access.</p><p>To further simplify the integration 
of new methods, we implemented the function create.PopGenome.method(), 
which generates a skeleton of a typical PopGenome function. New methods 
thus implemented are fully embedded in the PopGenome framework and can 
be applied to sliding windows or subsites in the same way as the 
existing modules. This approach frees developers of new population 
genetics or population genomics algorithms from the need to implement 
many auxiliary functions, such as efficient data input and output, data 
conversion, and region subsetting.</p><p>To enable PopGenome to work 
with additional data formats, users can write a simple parser that 
converts the data to a binary R object. The mechanism to integrate new 
methods or data formats is documented extensively in a tutorial, 
accessible by typing “vignette(“Integration_of_new_Methods”)” in R (see 
also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary file S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online).</p>                    <h2 scrollto-destination="74403066" id="74403066" class="section-title">Results</h2>
<p>To illustrate the usage of PopGenome, we show two exemplary analyses for human and <em>A. </em><em>thaliana</em> whole-genome SNP data. More details are found in the PopGenome documentation and in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary file S2</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online.</p>                    <h3 scrollto-destination="74403068" id="74403068" class="section-title">Diversity on <em>A. thaliana</em> Chromosome 1</h3>
<p>The 1001 genomes project (1000genomes.org) stores all SNP calls from one individual <em>A. thaliana</em> plant in one (.SNP) file. We downloaded .SNP files for 80 individuals (<a reveal-id="msu136-B4" data-open="msu136-B4" class="link link-ref link-reveal xref-bibr">Cao et al. 2011</a>)
 into a subdirectory named “Arabidopsis.” After starting R and loading 
the PopGenome library, we read in the data for chromosome 1 to analyze 
diversity and population differentiation: </p><blockquote><p>&gt; <em>library</em>(PopGenome)</p><p>&gt; genome &lt;- <em>readSNP</em>(“Arabidopsis,” CHR=1)</p></blockquote><p></p><p>We define the populations as a list of character vectors containing the individuals of each population: </p><blockquote><p>&gt; Central_Asia &lt;- <em>c</em>(“ICE127,” “ICE130,” “ICE134,” “ICE138,” “ICE150,” “ICE152,” “ICE153” , “Sha”)</p><p>&gt; … (analogous for the other populations) …</p><p>&gt; populations &lt;- <em>list</em>(Central_Asia, Caucasus, N_Europe, N_Africa, S_Italy, S_Russia, S_Tyrol, Swabia)</p></blockquote><p></p><p>The population definitions are now added to the GENOME object: </p><blockquote><p>&gt; genome &lt;- <em>set.populations</em>(genome, populations)</p></blockquote><p></p><p>We then transform these data into consecutive 10-kb-sliding windows: </p><blockquote><p>&gt; genome.slide &lt;- <em>sliding.window.transform</em>(genome, width=10000, jump=10000)</p></blockquote><p></p><p>The nucleotide and haplotype diversities for each window are calculated in the module <em>diversity.stats</em>. We store the results also in the GENOME object: </p><blockquote><p>&gt; genome.slide &lt;- <em>diversity.stats</em>(genome.slide)</p></blockquote><p></p><p>The “slot” of the <em>GENOME</em>
 object that stores the nucleotide diversities of the individual 
populations is called nuc.diversity.within. Slots of such objects are 
accessed by appending the slot name to the object name, separated by an @
 symbol: <a href="mailto:genome.slide@nuc.diversity.within" target="">genome.slide@nuc.diversity.within</a>.
 These data can be analyzed further using the built-in statistical and 
graphical capabilities of R. Here, we plot the sliding window nucleotide
 diversities (<a reveal-id="msu136-F1" data-open="msu136-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>A</em>) with a specialized function: </p><blockquote><p>&gt; <em>PopGplot</em>(<a href="mailto:genome.slide@nuc.diversity.within," target="">genome.slide@nuc.diversity.within,</a> colours)</p></blockquote>
 where colors is a vector listing the R names of the colors used for the
 eight populations in the figure. The haplotype diversity per 10-kb 
window (<a reveal-id="msu136-F1" data-open="msu136-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>B</em>), as well as Hudson’s fixation index, <em>F</em><sub>ST</sub> (<a reveal-id="msu136-F1" data-open="msu136-F1" class="link link-reveal link-table xref-fig">fig. 1</a><em>C</em>), is produced with similar function calls. <p></p>                        <a id="74403075" scrollto-destination="74403075"></a>
<div id="msu136-F1" data-id="msu136-F1" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_msu136f1p.jpeg" alt="Diversity statistics for Arabidopsis thaliana chromosome 1. Data from the 1001 genomes project website (1001genomes.org) was analyzed in consecutive 10-kb windows. (A) Nucleotide diversity, (B) haplotype diversity, (C) fixation index (Hudson’s FST), contrasting one population against all other individuals. Each line corresponds to one population (see legend in panel [A]). Lines were smoothed using spline interpolation. The black bars around 15-Mb mask the centromere." path-from-xml="msu136f1p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74403075/msu136f1p.jpeg" path-from-xml="msu136f1p.jpeg" target="_blank">Open in new tab</a><a section="74403075" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136f1p.jpeg?Expires=1577481724&amp;Signature=eoUApuRMppqugJdBfOAywU0SR4U9pFMF6j90JxO1o12yy3iIaQY-mkcH683U9-Ct%7EMu-tt5BFnpVDcvWRKV3Ld2h1vUeGVrVATUGEiFHqWSfHDkgnUK0FAM6QZTJDpRIiu6%7E-wXPM-unU1bl4McvF5pbaQ6YnDZw8x5LN5i4N6nTeDuwHYDKe3ytQ5XgDyyjV4p2T6YM4KqblnDLRbIwfmjZQ-Gvtmskt5Tr6W06hWU1%7EQNGJFE-62PZw%7EvuXK-3W8%7EjHTnu6nkTLffX-kyPebQI9eeUnf3doF9y-5xhR9yZe0qIszUTqIhEkpyCOmGZgKoMi95zjSCHM8cTVyEquQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74403075&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu136f1p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Diversity statistics for <em>Arabidopsis thaliana</em> chromosome 1. Data from the 1001 genomes project website (1001genomes.org) was analyzed in consecutive 10-kb windows. (<em>A</em>) Nucleotide diversity, (<em>B</em>) haplotype diversity, (<em>C</em>) fixation index (Hudson’s <em>F</em><sub>ST</sub>), contrasting one population against all other individuals. Each line corresponds to one population (see legend in panel [<em>A</em>]). Lines were smoothed using spline interpolation. The black bars around 15-Mb mask the centromere.</p></div></div><div content-id="msu136-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_msu136f1p.jpeg" alt="Diversity statistics for Arabidopsis thaliana chromosome 1. Data from the 1001 genomes project website (1001genomes.org) was analyzed in consecutive 10-kb windows. (A) Nucleotide diversity, (B) haplotype diversity, (C) fixation index (Hudson’s FST), contrasting one population against all other individuals. Each line corresponds to one population (see legend in panel [A]). Lines were smoothed using spline interpolation. The black bars around 15-Mb mask the centromere." path-from-xml="msu136f1p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74403075/msu136f1p.jpeg" path-from-xml="msu136f1p.jpeg" target="_blank">Open in new tab</a><a section="74403075" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136f1p.jpeg?Expires=1577481724&amp;Signature=eoUApuRMppqugJdBfOAywU0SR4U9pFMF6j90JxO1o12yy3iIaQY-mkcH683U9-Ct%7EMu-tt5BFnpVDcvWRKV3Ld2h1vUeGVrVATUGEiFHqWSfHDkgnUK0FAM6QZTJDpRIiu6%7E-wXPM-unU1bl4McvF5pbaQ6YnDZw8x5LN5i4N6nTeDuwHYDKe3ytQ5XgDyyjV4p2T6YM4KqblnDLRbIwfmjZQ-Gvtmskt5Tr6W06hWU1%7EQNGJFE-62PZw%7EvuXK-3W8%7EjHTnu6nkTLffX-kyPebQI9eeUnf3doF9y-5xhR9yZe0qIszUTqIhEkpyCOmGZgKoMi95zjSCHM8cTVyEquQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74403075&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu136f1p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Diversity statistics for <em>Arabidopsis thaliana</em> chromosome 1. Data from the 1001 genomes project website (1001genomes.org) was analyzed in consecutive 10-kb windows. (<em>A</em>) Nucleotide diversity, (<em>B</em>) haplotype diversity, (<em>C</em>) fixation index (Hudson’s <em>F</em><sub>ST</sub>), contrasting one population against all other individuals. Each line corresponds to one population (see legend in panel [<em>A</em>]). Lines were smoothed using spline interpolation. The black bars around 15-Mb mask the centromere.</p></div></div><p>The data thus displayed in <a reveal-id="msu136-F1" data-open="msu136-F1" class="link link-reveal link-table xref-fig">figure 1</a> indicate a lower level of diversity in the Central Asia population of <em>A. </em><em>thaliana</em> (dark blue lines) along the whole chromosome. As reported earlier (<a reveal-id="msu136-B4" data-open="msu136-B4" class="link link-ref link-reveal xref-bibr">Cao et al. 2011</a>),
 we observe a dip in diversity in the region around 20 Mb in all 
populations, suggesting a recent selective sweep in this region.</p><p>We
 find two additional, even stronger candidate regions for selective 
sweeps around position 8 and 22 Mb. Closer inspection of these two 
regions shows that they are devoid of polymorphisms between positions 
8765643 and 8831390, and between positions 21766562 and 21823063. This 
observation suggests additional recent, species-wide selective sweeps in
 these two regions. Furthermore, we find a strong decrease in diversity 
in plants from Asia and Southern Russia between genomic positions 
11870001 and 11900000, suggesting a population-specific selective sweep 
in this region.</p>                    <h3 scrollto-destination="74403078" id="74403078" class="section-title">Tajima’s <em>D</em> across Human MHC Exons</h3>
<p>As a second illustration of the PopGenome usage, we calculated Tajima’s <em>D</em>
 around the human MHC (Major Histocompatibility Complex) region. The 
1000 genomes project stores SNP calls from all examined individuals in 
gzipped VCF format, together with a Tabix (<a reveal-id="msu136-B22" data-open="msu136-B22" class="link link-ref link-reveal xref-bibr">Li 2011</a>)
 index (.tbi) file. To read data for the MHC region on human chromosome 6
 (including the corresponding annotation in the GFF file), we use the 
following line: </p><blockquote><p>&gt; genome &lt;- <em>readVCF</em>(“chr6.vcf.gz,” numcols=10000, tid=“6,” from=28000000, to=34000000, gffpath=“chr6.gff”)</p></blockquote>
 where “numcols” is the number of SNPs read in simultaneously, “tid” is a
 chromosome identifier, “from/to” delimit the nucleotide positions of 
the SNPs read in, and “gffpath” specifies the position of the GFF 
annotation file. Based on the annotations read from the GFF file and the
 chromosomal reference sequence in FASTA format, the next command labels
 positions in protein-coding regions as either synonymous or 
nonsynonymous: <blockquote><p>&gt; genome &lt;- set.synnonsyn(genome, ref.chr=“chr6.fas”)</p></blockquote><p></p><p>As
 in the Arabidopsis example, we define the populations via the vectors 
“Africa” and “Europe,” which contain the identifiers of the 
corresponding individuals: </p><blockquote><p>&gt; genome &lt;- <em>set.populations</em>(genome, <em>list</em>(Africa, Europe))</p></blockquote><p></p><p>Here,
 we want to calculate statistics for each exon individually, considering
 only nonsynonymous SNPs. To do this, we first split the region into 
individual loci, where each locus stores the SNP information from the 
coding sequence part contained in one exon, as annotated in the GFF 
file: </p><blockquote><p>&gt; genome.exons &lt;- <em>splitting.data</em>(genome, subsites=“coding”)</p></blockquote><p></p><p>Next, we calculate Tajima’s <em>D</em> across all nonsynonymous positions of each exon. This calculation is performed in the module “neutrality.stats”: </p><blockquote><p>&gt; genome.exons &lt;- <em>neutrality.stats</em>(genome.exons, subsites=”nonsyn”)</p></blockquote><p></p><p>We thus obtain the data displayed in <a reveal-id="msu136-F2" data-open="msu136-F2" class="link link-reveal link-table xref-fig">figure 2</a>. The high Tajima’s <em>D</em> values in some loci likely reflect balancing selection (<a reveal-id="msu136-B13" data-open="msu136-B13" class="link link-ref link-reveal xref-bibr">Hedrick 1998</a>), for example, due to frequency-dependent selection in pathogen recognition. </p>                        <a id="74403084" scrollto-destination="74403084"></a>
<div id="msu136-F2" data-id="msu136-F2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_msu136f2p.jpeg" alt="Tajima’s D calculated across nonsynonymous coding sites of exons in the human MHC region on chromosome 6. Each data point in (A) and (B) represents one exon; HLA type I and type II exons are shown in red. (A) Tajima’s D of a Tuscan population (117 individuals), plotted along chr. 6. (B) Comparison of Tajima’s D between a Tuscan (117 individuals) and a Yoruba (229 individuals) population. (C) Distribution (density curves) of the Tajima’s D values in (A) for MHC (red) and non-MHC exons (black). The blue curve displays the distribution of neutral values from coalescent simulations with Hudson’s MS based on all SNPs in the MHC region. Data from 1000genomes.org." path-from-xml="msu136f2p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74403084/msu136f2p.jpeg" path-from-xml="msu136f2p.jpeg" target="_blank">Open in new tab</a><a section="74403084" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136f2p.jpeg?Expires=1577481724&amp;Signature=xlLD22y%7EOyhJfRqTsjf7Sfdo%7EPBtvc1lxYLaHbzgYLBJZ-JuSR8ZSivsX%7EJoqRLNoXa0dd7CflJWGhInKuAKYVnnL2ekM9ZL6NHggbE4Q%7EgsU-mrXQ%7EtJU8pRmxuWnOPyuvnR8dvQP0nKDmDq1K9pC1MnaaLDh28zby6pWfc9Sz%7EUqLbg-4V%7E2FNJzCWWy513XN2PS9wtIaNyH8MmB8vpraobtHoSqEVyARoih0CtA22MSMolkNMaL33oLrmEq5ayDBxezNCrv6tcQEVEuuMGovrtMF7ubbS1l0QDdR6hKM4jlFS0qv1RTQGRi7tyUYpKZ%7E%7EhEAU3baj0R01%7EueJfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74403084&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu136f2p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Tajima’s <em>D</em> calculated across nonsynonymous coding sites of exons in the human MHC region on chromosome 6. Each data point in (<em>A</em>) and (<em>B</em>) represents one exon; HLA type I and type II exons are shown in red. (<em>A</em>) Tajima’s <em>D</em> of a Tuscan population (117 individuals), plotted along chr. 6. (<em>B</em>) Comparison of Tajima’s <em>D</em> between a Tuscan (117 individuals) and a Yoruba (229 individuals) population. (<em>C</em>) Distribution (density curves) of the Tajima’s <em>D</em> values in (<em>A</em>)
 for MHC (red) and non-MHC exons (black). The blue curve displays the 
distribution of neutral values from coalescent simulations with Hudson’s
 MS based on all SNPs in the MHC region. Data from 1000genomes.org.</p></div></div><div content-id="msu136-F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_msu136f2p.jpeg" alt="Tajima’s D calculated across nonsynonymous coding sites of exons in the human MHC region on chromosome 6. Each data point in (A) and (B) represents one exon; HLA type I and type II exons are shown in red. (A) Tajima’s D of a Tuscan population (117 individuals), plotted along chr. 6. (B) Comparison of Tajima’s D between a Tuscan (117 individuals) and a Yoruba (229 individuals) population. (C) Distribution (density curves) of the Tajima’s D values in (A) for MHC (red) and non-MHC exons (black). The blue curve displays the distribution of neutral values from coalescent simulations with Hudson’s MS based on all SNPs in the MHC region. Data from 1000genomes.org." path-from-xml="msu136f2p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74403084/msu136f2p.jpeg" path-from-xml="msu136f2p.jpeg" target="_blank">Open in new tab</a><a section="74403084" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136f2p.jpeg?Expires=1577481724&amp;Signature=xlLD22y%7EOyhJfRqTsjf7Sfdo%7EPBtvc1lxYLaHbzgYLBJZ-JuSR8ZSivsX%7EJoqRLNoXa0dd7CflJWGhInKuAKYVnnL2ekM9ZL6NHggbE4Q%7EgsU-mrXQ%7EtJU8pRmxuWnOPyuvnR8dvQP0nKDmDq1K9pC1MnaaLDh28zby6pWfc9Sz%7EUqLbg-4V%7E2FNJzCWWy513XN2PS9wtIaNyH8MmB8vpraobtHoSqEVyARoih0CtA22MSMolkNMaL33oLrmEq5ayDBxezNCrv6tcQEVEuuMGovrtMF7ubbS1l0QDdR6hKM4jlFS0qv1RTQGRi7tyUYpKZ%7E%7EhEAU3baj0R01%7EueJfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74403084&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu136f2p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Tajima’s <em>D</em> calculated across nonsynonymous coding sites of exons in the human MHC region on chromosome 6. Each data point in (<em>A</em>) and (<em>B</em>) represents one exon; HLA type I and type II exons are shown in red. (<em>A</em>) Tajima’s <em>D</em> of a Tuscan population (117 individuals), plotted along chr. 6. (<em>B</em>) Comparison of Tajima’s <em>D</em> between a Tuscan (117 individuals) and a Yoruba (229 individuals) population. (<em>C</em>) Distribution (density curves) of the Tajima’s <em>D</em> values in (<em>A</em>)
 for MHC (red) and non-MHC exons (black). The blue curve displays the 
distribution of neutral values from coalescent simulations with Hudson’s
 MS based on all SNPs in the MHC region. Data from 1000genomes.org.</p></div></div><p>We can use coalescent simulations to derive the expected neutral distribution of Tajima’s <em>D</em> values across the MHC region. For this, we first calculate Theta across all SNPs in the MHC region: </p><blockquote><p>&gt; genome &lt;- <em>neutrality.stats</em>(genome)</p></blockquote><p></p><p>We then use the genome object as input for a call to Hudson’s MS program: </p><blockquote><p>&gt; ms &lt;- <em>MS</em>(genome, thetaID=“Tajima,” neutrality=TRUE)</p></blockquote><p></p><p>If no additional parameters are specified for the MS simulations, PopGenome will use the standard neutral model (SNM).</p><p>The simulated Tajima’s <em>D</em> values are then extracted: </p><blockquote><p>&gt; <em>MS.get.stats</em>(ms)</p></blockquote><p></p><p><a reveal-id="msu136-F2" data-open="msu136-F2" class="link link-reveal link-table xref-fig">Figure 2</a><em>C</em> compares the distributions of expected Tajima’s <em>D</em>
 values under the SNM with values observed for Human Leukocyte Antigen 
(HLA, red) and non-HLA (black) exons. The distribution of Tajima’s <em>D</em>
 values is strongly shifted toward higher values in HLA exons compared 
with non-HLA exons, indicating strong balancing selection; a deviation 
from neutral expectations for HLA exons is supported by a comparison to 
the simulated data.</p><p>The highest Tajima’s <em>D</em> values—suggesting strong balancing selection—are seen in HLA type I and type II genes (marked in red in <a reveal-id="msu136-F2" data-open="msu136-F2" class="link link-reveal link-table xref-fig">fig. 2</a>). Tajima’s <em>D</em> values are correlated between the African (Yoruba) and European (Tuscany) populations (<a reveal-id="msu136-F2" data-open="msu136-F2" class="link link-reveal link-table xref-fig">fig. 2</a><em>B</em>; Spearman’s <em>R</em><sup>2 </sup>=
 0.33). One notable outlier is the coding exon 2 of the HLA-DPA1 gene, 
which shows evidence of purifying selection in Yoruba, but strong 
evidence of balancing selection in Tuscans.</p>                    <h2 scrollto-destination="74403091" id="74403091" class="section-title">Discussion</h2>
<p>Several computer programs for population genetics or population 
genomics analyses are publicly available. However, these tend to 
specialize on specific subsets of analyses (e.g., <a reveal-id="msu136-B35" data-open="msu136-B35" class="link link-ref link-reveal xref-bibr">Vilella 2005</a>; <a reveal-id="msu136-B38" data-open="msu136-B38" class="link link-ref link-reveal xref-bibr">Rozas et al. 2001</a>; <a reveal-id="msu136-B29" data-open="msu136-B29" class="link link-ref link-reveal xref-bibr">Purcell et al. 2007</a>; <a reveal-id="msu136-B20" data-open="msu136-B20" class="link link-ref link-reveal xref-bibr">Jombart 2008</a>; <a reveal-id="msu136-B27" data-open="msu136-B27" class="link link-ref link-reveal xref-bibr">Paradis 2010</a>) or cannot process whole-genome data (e.g., <a reveal-id="msu136-B31" data-open="msu136-B31" class="link link-ref link-reveal xref-bibr">Rozas et al. 2003</a>; <a reveal-id="msu136-B6" data-open="msu136-B6" class="link link-ref link-reveal xref-bibr">Excoffier et al. 2005</a>). <a reveal-id="msu136-F3" data-open="msu136-F3" class="link link-reveal link-table xref-fig">Figure 3</a> compares major features of PopGenome with five other widely used software packages. </p>                        <a id="74403093" scrollto-destination="74403093"></a>
<div id="msu136-F3" data-id="msu136-F3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_msu136f3p.jpeg" alt="Comparison of PopGenome with existing software for population genetics and population genomics analyses. Symbols reflect the breadth of the implemented functionalities: ++, broad; +, limited; −, nonexistent. Details on the criteria used for assignment to the breadth classes are given in supplementary table S1, Supplementary Material online." path-from-xml="msu136f3p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74403093/msu136f3p.jpeg" path-from-xml="msu136f3p.jpeg" target="_blank">Open in new tab</a><a section="74403093" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136f3p.jpeg?Expires=1577481724&amp;Signature=nP7pXdh8y9kPw5inAu17STGXm9o81v7M1ejfUuSMPRgjxFV84ZAyLt%7EQN7mlvrQYHbaR%7E3r%7ET6DCNwG62uopR3OiyuarfhfOU2Tj2OGr6GsDZBwkI%7EAajVH-4QkT7tFIRbCOejkgdB1b8M9F0143sYDFFc4B7qeZ7P7K32T-pBBPpLqpJ3YoIt-c1np1bW5q9iyxos-jZfn70OwdWLumVxi1nX-pD%7E1MZdqBxOqjEw13TMZcyir4jrFTkVYh3SAtU8PNcTefcuSfF%7ELLfJ206w3KgIBfM5bQI92U6052CG-5bf3XBDUcedQ%7EUDih5598DMvkQjAKOG4D1hcNMFe4CA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74403093&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu136f3p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison
 of PopGenome with existing software for population genetics and 
population genomics analyses. Symbols reflect the breadth of the 
implemented functionalities: ++, broad; +, limited; −, nonexistent. 
Details on the criteria used for assignment to the breadth classes are 
given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online.</p></div></div><div content-id="msu136-F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">ig</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_msu136f3p.jpeg" alt="Comparison of PopGenome with existing software for population genetics and population genomics analyses. Symbols reflect the breadth of the implemented functionalities: ++, broad; +, limited; −, nonexistent. Details on the criteria used for assignment to the breadth classes are given in supplementary table S1, Supplementary Material online." path-from-xml="msu136f3p.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/74403093/msu136f3p.jpeg" path-from-xml="msu136f3p.jpeg" target="_blank">Open in new tab</a><a section="74403093" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136f3p.jpeg?Expires=1577481724&amp;Signature=nP7pXdh8y9kPw5inAu17STGXm9o81v7M1ejfUuSMPRgjxFV84ZAyLt%7EQN7mlvrQYHbaR%7E3r%7ET6DCNwG62uopR3OiyuarfhfOU2Tj2OGr6GsDZBwkI%7EAajVH-4QkT7tFIRbCOejkgdB1b8M9F0143sYDFFc4B7qeZ7P7K32T-pBBPpLqpJ3YoIt-c1np1bW5q9iyxos-jZfn70OwdWLumVxi1nX-pD%7E1MZdqBxOqjEw13TMZcyir4jrFTkVYh3SAtU8PNcTefcuSfF%7ELLfJ206w3KgIBfM5bQI92U6052CG-5bf3XBDUcedQ%7EUDih5598DMvkQjAKOG4D1hcNMFe4CA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=74403093&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msu136f3p.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison
 of PopGenome with existing software for population genetics and 
population genomics analyses. Symbols reflect the breadth of the 
implemented functionalities: ++, broad; +, limited; −, nonexistent. 
Details on the criteria used for assignment to the breadth classes are 
given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary table S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577481724&amp;Signature=fRAMOBj3QFlZeq%7EKS8WPwz2yEqpNxdrQnpasf2GsuxjnE53G4r2ZNYdGjqyU7A81TBVaQUbd8hnpTrYr0VPfl0GwhEcp14WaDi2Dx2EMxtnFIFEwREYY0sBAyVKY03cd3hZCKwk3vkGuF6sWGaMFnV9DpMkNgehHSWciWDxTftuFoM6n08N8WOKf27KlyQvyuvX-yYumNT5qIichLiydLPieLMZqU5CT2pimZxv5y7mT9D9ZLJ2H4RjswNI9F2MTQoQFXOQeAcOSDJII3U4LGFaDmWxeB7EjtP2Y%7Ei1y1F9s%7EYBvLGaU64InJ0UrwOK0EiMCkDf%7EFoNizFDSS5Reyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> online.</p></div></div><p>PopGenome
 can not only read data in several common alignment formats but also 
understands the widest choice of SNP data formats; this includes data 
from the HapMap as well as the 1000 and 1001 genomes projects. 
PopGenome’s ability to work efficiently with temporary files allows 
calculations on very large SNP data sets. No other available program 
offers a similar combination of flexible data input options with a broad
 toolbox of population genetics and genomics statistics, including the 
ability to perform analyses in sliding windows.</p><p>PopGenome can read
 GFF annotation files, which permits high plasticity in the variability 
analysis of different regions of the genome. PopGenome can link this 
annotation automatically to the SNP data, which can thus be restricted 
to specific annotated features or feature groups (e.g., all introns vs. 
all exons). This feature also allows to discriminate synonymous and 
nonsynonymous codon positions in whole-genome SNP data sets, which is 
necessary, for example, for McDonald–Kreitman tests. Like adegenet/pegas
 (which are more limited in scope), PopGenome is fully integrated with 
the powerful graphical and data analysis capabilities of the R 
environment (<a class="link link-uri" href="http://cran.r-project.org/" target="">http://cran.r-project.org/</a>,
 last accessed April 30, 2014), thus simplifying downstream analyses and
 graphics as well as the development of stable work flows.</p>                    <h2 scrollto-destination="74403096" id="74403096" class="backacknowledgements-title">Acknowledgments</h2>
<p>The authors thank Juliette de Meaux, Laura Rose, Kristian Ulrich, 
Thorsten Klösges, and William Martin for helpful discussions. This work 
was supported by the Spanish Ministerio de Ciencia e Innovación Grant 
CGL2009-09346 to S.E.R.O. and by the German Research Foundation DFG 
grants EXC 1028 and CRC 680 to M.J.L.</p>                    <h2 scrollto-destination="74403098" id="74403098" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="msu136-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Achaz</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">Frequency spectrum neutrality tests: one for all and all for one</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">183</div>&nbsp;<div class="issue">1</div>(pg.&nbsp;<div class="fpage">249</div>-<div class="lpage">258</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Frequency%20spectrum%20neutrality%20tests%3A%20one%20for%20all%20and%20all%20for%20one&amp;author=G%20Achaz&amp;publication_year=2009&amp;journal=Genetics&amp;volume=183&amp;pages=249-258" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.109.104042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.104042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.109.104042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19546320" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Frequency%20spectrum%20neutrality%20tests%3A%20one%20for%20all%20and%20all%20for%20one&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adler</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Gläser</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Nenadic</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Oehlschlägel</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Zucchini</div> <div class="given-names">W</div></div>.&nbsp;</div><div class="article-title">ff: memory-efficient storage of large data on disk and fast access functions [R package version 2.2-11]. [cited 2013 Dec]</div>,&nbsp;<div class="year">2013</div>&nbsp;<div class="comment">Available from: <a class="link link-uri" href="http://cran.r-project.org/package=ff" target="">http://CRAN.R-project.org/package=ff</a></div><div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=&amp;aulast=Adler&amp;date=2013" href="#"></a></span></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cai</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">PGEToolbox: a Matlab toolbox for population genetics and evolution</div>,&nbsp;<div class="source">J Hered.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">99</div>&nbsp;<div class="issue">4</div>(pg.&nbsp;<div class="fpage">438</div>-<div class="lpage">440</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PGEToolbox%3A%20a%20Matlab%20toolbox%20for%20population%20genetics%20and%20evolution&amp;author=J%20Cai&amp;publication_year=2008&amp;journal=J%20Hered.&amp;volume=99&amp;pages=438-440" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/jhered/esm127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fesm127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fjhered%2Fesm127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18310616" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PGEToolbox%3A%20a%20Matlab%20toolbox%20for%20population%20genetics%20and%20evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Schneeberger</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Ossowski</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Günther</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Bender</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Fitz</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Koenig</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Lanz</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Stegle</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Lippert</div> <div class="given-names">C</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations</div>,&nbsp;<div class="source">Nat Genet.</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">43</div>&nbsp;<div class="issue">10</div>(pg.&nbsp;<div class="fpage">956</div>-<div class="lpage">963</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20multiple%20Arabidopsis%20thaliana%20populations&amp;author=J%20Cao&amp;author=K%20Schneeberger&amp;author=S%20Ossowski&amp;author=T%20G%C3%BCnther&amp;author=S%20Bender&amp;author=J%20Fitz&amp;author=D%20Koenig&amp;author=C%20Lanz&amp;author=O%20Stegle&amp;author=C%20Lippert&amp;publication_year=2011&amp;journal=Nat%20Genet.&amp;volume=43&amp;pages=956-963" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/ng.911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fng.911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874002" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Whole-genome%20sequencing%20of%20multiple%20Arabidopsis%20thaliana%20populations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Hermisson</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">MSMS: a coalescent simulation program including recombination, demographic structure and selection at a single locus</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;<div class="issue">16</div>(pg.&nbsp;<div class="fpage">2064</div>-<div class="lpage">2065</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MSMS%3A%20a%20coalescent%20simulation%20program%20including%20recombination%2C%20demographic%20structure%20and%20selection%20at%20a%20single%20locus&amp;author=G%20Ewing&amp;author=J%20Hermisson&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2064-2065" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq322" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq322" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq322"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20591904" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MSMS%3A%20a%20coalescent%20simulation%20program%20including%20recombination%2C%20demographic%20structure%20and%20selection%20at%20a%20single%20locus&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Excoffier</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Laval</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Schneider</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Arlequin (version 3.0): an integrated software package for population genetics data analysis</div>,&nbsp;<div class="source">Evol Bioinform Online.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">1</div>&nbsp;(pg.&nbsp;<div class="fpage">47</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Arlequin%20(version%203.0)%3A%20an%20integrated%20software%20package%20for%20population%20genetics%20data%20analysis&amp;aulast=Excoffier&amp;date=2005&amp;spage=47&amp;epage=50&amp;volume=1" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Arlequin%20%28version%203.0%29%3A%20an%20integrated%20software%20package%20for%20population%20genetics%20data%20analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Excoffier</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Smouse</div> <div class="given-names">PE</div></div>.&nbsp;</div><div class="article-title">Analysis
 of molecular variance inferred from metric distances among DNA 
haplotypes: application to human mitochondrial DNA restriction data</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">131</div>&nbsp;<div class="issue">2</div>(pg.&nbsp;<div class="fpage">479</div>-<div class="lpage">491</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20molecular%20variance%20inferred%20from%20metric%20distances%20among%20DNA%20haplotypes%3A%20application%20to%20human%20mitochondrial%20DNA%20restriction%20data&amp;author=L%20Excoffier&amp;author=PE%20Smouse&amp;publication_year=1992&amp;journal=Genetics&amp;volume=131&amp;pages=479-491" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1644282" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Analysis%20of%20molecular%20variance%20inferred%20from%20metric%20distances%20among%20DNA%20haplotypes%3A%20application%20to%20human%20mitochondrial%20DNA%20restriction%20data&amp;aulast=Excoffier&amp;date=1992&amp;spage=479&amp;epage=491&amp;volume=131&amp;issue=2" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analysis%20of%20molecular%20variance%20inferred%20from%20metric%20distances%20among%20DNA%20haplotypes%3A%20application%20to%20human%20mitochondrial%20DNA%20restriction%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fay</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Wu</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">Hitchhiking under positive Darwinian selection</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">155</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">1405</div>-<div class="lpage">1413</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Hitchhiking%20under%20positive%20Darwinian%20selection&amp;author=J%20Fay&amp;author=C%20Wu&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=1405-1413" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880498" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Hitchhiking%20under%20positive%20Darwinian%20selection&amp;aulast=Fay&amp;date=2000&amp;spage=1405&amp;epage=1413&amp;volume=155&amp;issue=3" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Hitchhiking%20under%20positive%20Darwinian%20selection&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foll</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Gaggiotti</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">A genome-scan method to identify selected loci appropriate for both dominant and codominant markers: a Bayesian perspective</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">180</div>&nbsp;<div class="issue">2</div>(pg.&nbsp;<div class="fpage">977</div>-<div class="lpage">993</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20genome-scan%20method%20to%20identify%20selected%20loci%20appropriate%20for%20both%20dominant%20and%20codominant%20markers%3A%20a%20Bayesian%20perspective&amp;author=M%20Foll&amp;author=O%20Gaggiotti&amp;publication_year=2008&amp;journal=Genetics&amp;volume=180&amp;pages=977-993" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.108.092221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.092221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.108.092221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18780740" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20genome-scan%20method%20to%20identify%20selected%20loci%20appropriate%20for%20both%20dominant%20and%20codominant%20markers%3A%20a%20Bayesian%20perspective&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">Y</div></div>.&nbsp;</div><div class="article-title">Statistical tests of neutrality of mutations against population growth, hitchhiking and background selection</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">147</div>&nbsp;<div class="issue">2</div>(pg.&nbsp;<div class="fpage">915</div>-<div class="lpage">925</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection&amp;author=Y%20Fu&amp;publication_year=1997&amp;journal=Genetics&amp;volume=147&amp;pages=915-925" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9335623" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection&amp;aulast=Fu&amp;date=1997&amp;spage=915&amp;epage=925&amp;volume=147&amp;issue=2" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Statistical%20tests%20of%20neutrality%20of%20mutations%20against%20population%20growth%2C%20hitchhiking%20and%20background%20selection&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Li</div> <div class="given-names">W</div></div>.&nbsp;</div><div class="article-title">Statistical tests of neutrality of mutations</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">133</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">693</div>-<div class="lpage">709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Statistical%20tests%20of%20neutrality%20of%20mutations&amp;author=Y%20Fu&amp;author=W%20Li&amp;publication_year=1993&amp;journal=Genetics&amp;volume=133&amp;pages=693-709" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8454210" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Statistical%20tests%20of%20neutrality%20of%20mutations&amp;aulast=Fu&amp;date=1993&amp;spage=693&amp;epage=709&amp;volume=133&amp;issue=3" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Statistical%20tests%20of%20neutrality%20of%20mutations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Understanding genetic variation and function—the applications of next generation sequencing</div>,&nbsp;<div class="source">Semin Cell Dev Biol.</div>&nbsp;,&nbsp;<div class="year">2012</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;<div class="issue">2</div>(pg.&nbsp;<div class="fpage">230</div>-<div class="lpage">236</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Understanding%20genetic%20variation%20and%20function%E2%80%94the%20applications%20of%20next%20generation%20sequencing&amp;author=R%20Harrison&amp;publication_year=2012&amp;journal=Semin%20Cell%20Dev%20Biol.&amp;volume=23&amp;pages=230-236" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.semcdb.2012.01.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.semcdb.2012.01.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.semcdb.2012.01.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22285423" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Understanding%20genetic%20variation%20and%20function%E2%80%94the%20applications%20of%20next%20generation%20sequencing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hedrick</div></div>.&nbsp;</div><div class="article-title">Balancing selection and MHC</div>,&nbsp;<div class="source">Genetica</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">104</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">207</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Balancing%20selection%20and%20MHC&amp;author=%20Hedrick&amp;publication_year=1998&amp;journal=Genetica&amp;volume=104&amp;pages=207-214" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1023/A:1026494212540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1026494212540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1023%2FA:1026494212540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10386384" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Balancing%20selection%20and%20MHC&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">A new statistic for detecting genetic differentiation</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">155</div>&nbsp;<div class="issue">4</div>(pg.&nbsp;<div class="fpage">2011</div>-<div class="lpage">2014</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20new%20statistic%20for%20detecting%20genetic%20differentiation&amp;author=R%20Hudson&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=2011-2014" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10924493" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20new%20statistic%20for%20detecting%20genetic%20differentiation&amp;aulast=Hudson&amp;date=2000&amp;spage=2011&amp;epage=2014&amp;volume=155&amp;issue=4" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20new%20statistic%20for%20detecting%20genetic%20differentiation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Boos</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Kaplan</div> <div class="given-names">N</div></div>.&nbsp;</div><div class="article-title">A statistical test for detecting geographic subdivision</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;<div class="issue">1</div>(pg.&nbsp;<div class="fpage">138</div>-<div class="lpage">151</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20test%20for%20detecting%20geographic%20subdivision&amp;author=R%20Hudson&amp;author=D%20Boos&amp;author=N%20Kaplan&amp;publication_year=1992&amp;journal=Mol%20Biol%20Evol.&amp;volume=9&amp;pages=138-151" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1552836" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20statistical%20test%20for%20detecting%20geographic%20subdivision&amp;aulast=Hudson&amp;date=1992&amp;spage=138&amp;epage=151&amp;volume=9&amp;issue=1" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20statistical%20test%20for%20detecting%20geographic%20subdivision&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Kaplan</div> <div class="given-names">N</div></div>.&nbsp;</div><div class="article-title">Statistical properties of the number of recombination events in the history of a sample of DNA sequences</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1985</div>,&nbsp;vol.&nbsp;<div class="volume">111</div>&nbsp;<div class="issue">1</div>(pg.&nbsp;<div class="fpage">147</div>-<div class="lpage">164</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Statistical%20properties%20of%20the%20number%20of%20recombination%20events%20in%20the%20history%20of%20a%20sample%20of%20DNA%20sequences&amp;author=R%20Hudson&amp;author=N%20Kaplan&amp;publication_year=1985&amp;journal=Genetics&amp;volume=111&amp;pages=147-164" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4029609" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Statistical%20properties%20of%20the%20number%20of%20recombination%20events%20in%20the%20history%20of%20a%20sample%20of%20DNA%20sequences&amp;aulast=Hudson&amp;date=1985&amp;spage=147&amp;epage=164&amp;volume=111&amp;issue=1" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Statistical%20properties%20of%20the%20number%20of%20recombination%20events%20in%20the%20history%20of%20a%20sample%20of%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Generating samples under a Wright–Fisher neutral model of genetic variation</div>,&nbsp;<div class="source">Bioinformatics Appl Note.</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">18</div>&nbsp;<div class="issue">2</div>(pg.&nbsp;<div class="fpage">337</div>-<div class="lpage">338</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Generating%20samples%20under%20a%20Wright%E2%80%93Fisher%20neutral%20model%20of%20genetic%20variation&amp;author=R%20Hudson&amp;publication_year=2002&amp;journal=Bioinformatics%20Appl%20Note.&amp;volume=18&amp;pages=337-338" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/18.2.337" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.2.337" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F18.2.337"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Generating%20samples%20under%20a%20Wright%E2%80%93Fisher%20neutral%20model%20of%20genetic%20variation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Slatkin</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Maddison</div> <div class="given-names">W</div></div>.&nbsp;</div><div class="article-title">Estimation of levels of gene flow from DNA sequence data</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">132</div>&nbsp;<div class="issue">2</div>(pg.&nbsp;<div class="fpage">583</div>-<div class="lpage">589</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20levels%20of%20gene%20flow%20from%20DNA%20sequence%20data&amp;author=R%20Hudson&amp;author=M%20Slatkin&amp;author=W%20Maddison&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=583-589" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1427045" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Estimation%20of%20levels%20of%20gene%20flow%20from%20DNA%20sequence%20data&amp;aulast=Hudson&amp;date=1992&amp;spage=583&amp;epage=589&amp;volume=132&amp;issue=2" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Estimation%20of%20levels%20of%20gene%20flow%20from%20DNA%20sequence%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jombart</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">adegenet: a R package for the multivariate analysis of genetic markers</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">24</div>&nbsp;<div class="issue">11</div>(pg.&nbsp;<div class="fpage">1403</div>-<div class="lpage">1405</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=adegenet%3A%20a%20R%20package%20for%20the%20multivariate%20analysis%20of%20genetic%20markers&amp;author=T%20Jombart&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1403-1405" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btn129" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn129" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtn129"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397895" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:adegenet%3A%20a%20R%20package%20for%20the%20multivariate%20analysis%20of%20genetic%20markers&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kelly</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">A test of neutrality based on interlocus associations</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">146</div>&nbsp;(pg.&nbsp;<div class="fpage">1197</div>-<div class="lpage">1206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20test%20of%20neutrality%20based%20on%20interlocus%20associations&amp;author=J%20Kelly&amp;publication_year=1997&amp;journal=Genetics&amp;volume=146&amp;pages=1197-1206" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9215920" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20test%20of%20neutrality%20based%20on%20interlocus%20associations&amp;aulast=Kelly&amp;date=1997&amp;spage=1197&amp;epage=1206&amp;volume=146" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20test%20of%20neutrality%20based%20on%20interlocus%20associations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">Tabix: fast retrieval of sequence features from generic TAB-delimited files</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2011</div>,&nbsp;vol.&nbsp;<div class="volume">27</div>&nbsp;<div class="issue">5</div>(pg.&nbsp;<div class="fpage">718</div>-<div class="lpage">719</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Tabix%3A%20fast%20retrieval%20of%20sequence%20features%20from%20generic%20TAB-delimited%20files&amp;author=H%20Li&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=718-719" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq671" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq671" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq671"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21208982" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Tabix%3A%20fast%20retrieval%20of%20sequence%20features%20from%20generic%20TAB-delimited%20files&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McDonald</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Kreitman</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Adaptive protein evolution at the Adh locus in <em>Drosophila</em></div>,&nbsp;<div class="source">Nature</div>&nbsp;,&nbsp;<div class="year">1991</div>,&nbsp;vol.&nbsp;<div class="volume">351</div>&nbsp;<div class="issue">6328</div>(pg.&nbsp;<div class="fpage">652</div>-<div class="lpage">654</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Adaptive%20protein%20evolution%20at%20the%20Adh%20locus%20in%20Drosophila&amp;author=J%20McDonald&amp;author=M%20Kreitman&amp;publication_year=1991&amp;journal=Nature&amp;volume=351&amp;pages=652-654" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/351652a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F351652a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F351652a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1904993" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Adaptive%20protein%20evolution%20at%20the%20Adh%20locus%20in%20Drosophila&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Analysis of gene diversity in subdivided populations</div>,&nbsp;<div class="source">Proc Natl Acad Sci U S A.</div>&nbsp;,&nbsp;<div class="year">1973</div>,&nbsp;vol.&nbsp;<div class="volume">70</div>&nbsp;<div class="issue">12</div>(pg.&nbsp;<div class="fpage">3321</div>-<div class="lpage">3323</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20gene%20diversity%20in%20subdivided%20populations&amp;author=M%20Nei&amp;publication_year=1973&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A.&amp;volume=70&amp;pages=3321-3323" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.70.12.3321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.70.12.3321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.70.12.3321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4519626" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Analysis%20of%20gene%20diversity%20in%20subdivided%20populations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Mathematical model for studying genetic variation in terms of restriction endonucleases</div>,&nbsp;<div class="source">Proc Natl Acad Sci U S A.</div>&nbsp;,&nbsp;<div class="year">1979</div>,&nbsp;vol.&nbsp;<div class="volume">76</div>&nbsp;<div class="issue">10</div>(pg.&nbsp;<div class="fpage">5269</div>-<div class="lpage">5273</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mathematical%20model%20for%20studying%20genetic%20variation%20in%20terms%20of%20restriction%20endonucleases&amp;author=M%20Nei&amp;publication_year=1979&amp;journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A.&amp;volume=76&amp;pages=5269-5273" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1073/pnas.76.10.5269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.76.10.5269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.76.10.5269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/291943" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mathematical%20model%20for%20studying%20genetic%20variation%20in%20terms%20of%20restriction%20endonucleases&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Williamson</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Kim</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Hubisz</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Clark</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Bustamante</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">Genomic scans for selective sweeps using SNP data</div>,&nbsp;<div class="source">Genome Res.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">15</div>&nbsp;(pg.&nbsp;<div class="fpage">1566</div>-<div class="lpage">1575</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genomic%20scans%20for%20selective%20sweeps%20using%20SNP%20data&amp;author=R%20Nielsen&amp;author=S%20Williamson&amp;author=Y%20Kim&amp;author=M%20Hubisz&amp;author=A%20Clark&amp;author=C%20Bustamante&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1566-1575" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1101/gr.4252305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4252305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.4252305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16251466" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genomic%20scans%20for%20selective%20sweeps%20using%20SNP%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paradis</div> <div class="given-names">E</div></div>.&nbsp;</div><div class="article-title">pegas: an R package for population genetics with an integrated-modular approach</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">419</div>-<div class="lpage">420</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=pegas%3A%20an%20R%20package%20for%20population%20genetics%20with%20an%20integrated-modular%20approach&amp;author=E%20Paradis&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=419-420" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btp696" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtp696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080509" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:pegas%3A%20an%20R%20package%20for%20population%20genetics%20with%20an%20integrated-modular%20approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pavlidis</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Zivkovic</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Alachiotis</div> <div class="given-names">N</div></div>.&nbsp;</div><div class="article-title">SweeD: likelihood-based detection of selective sweeps in thousands of genomes</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2013</div>,&nbsp;vol.&nbsp;<div class="volume">30</div>&nbsp;(pg.&nbsp;<div class="fpage">2224</div>-<div class="lpage">2234</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=SweeD%3A%20likelihood-based%20detection%20of%20selective%20sweeps%20in%20thousands%20of%20genomes&amp;author=P%20Pavlidis&amp;author=D%20Zivkovic&amp;author=A%20Stamatakis&amp;author=N%20Alachiotis&amp;publication_year=2013&amp;journal=Mol%20Biol%20Evol.&amp;volume=30&amp;pages=2224-2234" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/mst112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmst112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23777627" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:SweeD%3A%20likelihood-based%20detection%20of%20selective%20sweeps%20in%20thousands%20of%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Purcell</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Neale</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Todd-Brown</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Thomas</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Ferreira</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Bender</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Maller</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Sklar</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">de Bakker</div> <div class="given-names">PI</div></div>,&nbsp; <div class="name"><div class="surname">Daly</div> <div class="given-names">MJ</div></div>,&nbsp;et al.&nbsp;</div><div class="article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</div>,&nbsp;<div class="source">Am J Hum Genet.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">81</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">559</div>-<div class="lpage">575</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S%20Purcell&amp;author=B%20Neale&amp;author=K%20Todd-Brown&amp;author=L%20Thomas&amp;author=M%20Ferreira&amp;author=D%20Bender&amp;author=J%20Maller&amp;author=P%20Sklar&amp;author=PI%20de%20Bakker&amp;author=MJ%20Daly&amp;publication_year=2007&amp;journal=Am%20J%20Hum%20Genet.&amp;volume=81&amp;pages=559-575" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1086/519795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F519795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1086%2F519795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramos-Onsins</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Rozas</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Statistical properties of new neutrality tests against population growth</div>,&nbsp;<div class="source">Mol Biol Evol.</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">19</div>&nbsp;<div class="issue">12</div>(pg.&nbsp;<div class="fpage">2092</div>-<div class="lpage">2100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Statistical%20properties%20of%20new%20neutrality%20tests%20against%20population%20growth&amp;author=S%20Ramos-Onsins&amp;author=J%20Rozas&amp;publication_year=2002&amp;journal=Mol%20Biol%20Evol.&amp;volume=19&amp;pages=2092-2100" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a004034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a004034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446801" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Statistical%20properties%20of%20new%20neutrality%20tests%20against%20population%20growth&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B38" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozas</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Gullaud</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Blandin</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Aguade</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">DNA variation at the rp49 gene region of Drosophila simulans: evolutionary inferences from an unusual haplotype structure</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">2001</div>,&nbsp;vol.&nbsp;<div class="volume">158</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">1147</div>-<div class="lpage">1155</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DNA%20variation%20at%20the%20rp49%20gene%20region%20of%20Drosophila%20simulans%3A%20evolutionary%20inferences%20from%20an%20unusual%20haplotype%20structure&amp;author=J%20Rozas&amp;author=M%20Gullaud&amp;author=G%20Blandin&amp;author=M%20Aguade&amp;publication_year=2001&amp;journal=Genetics&amp;volume=158&amp;pages=1147-1155" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11454763" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=DNA%20variation%20at%20the%20rp49%20gene%20region%20of%20Drosophila%20simulans%3A%20evolutionary%20inferences%20from%20an%20unusual%20haplotype%20structure&amp;aulast=Rozas&amp;date=2001&amp;spage=1147&amp;epage=1155&amp;volume=158&amp;issue=3" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DNA%20variation%20at%20the%20rp49%20gene%20region%20of%20Drosophila%20simulans%3A%20evolutionary%20inferences%20from%20an%20unusual%20haplotype%20structure&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B31" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozas</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Sanchez-DelBarrio</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Messeguer</div> <div class="given-names">X</div></div>,&nbsp; <div class="name"><div class="surname">Rozas</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">DnaSP, DNA polymorphism analyses by the coalescent and other methods</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">19</div>&nbsp;<div class="issue">18</div>(pg.&nbsp;<div class="fpage">2496</div>-<div class="lpage">2497</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DnaSP%2C%20DNA%20polymorphism%20analyses%20by%20the%20coalescent%20and%20other%20methods&amp;author=J%20Rozas&amp;author=J%20Sanchez-DelBarrio&amp;author=X%20Messeguer&amp;author=R%20Rozas&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=2496-2497" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btg359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtg359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14668244" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DnaSP%2C%20DNA%20polymorphism%20analyses%20by%20the%20coalescent%20and%20other%20methods&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B32" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strobeck</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">Average number of nucleotide differences in a sample from a single subpopulation: a test for population subdivision</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1987</div>,&nbsp;vol.&nbsp;<div class="volume">117</div>&nbsp;<div class="issue">1</div>(pg.&nbsp;<div class="fpage">149</div>-<div class="lpage">153</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Average%20number%20of%20nucleotide%20differences%20in%20a%20sample%20from%20a%20single%20subpopulation%3A%20a%20test%20for%20population%20subdivision&amp;author=C%20Strobeck&amp;publication_year=1987&amp;journal=Genetics&amp;volume=117&amp;pages=149-153" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17246396" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Average%20number%20of%20nucleotide%20differences%20in%20a%20sample%20from%20a%20single%20subpopulation%3A%20a%20test%20for%20population%20subdivision&amp;aulast=Strobeck&amp;date=1987&amp;spage=149&amp;epage=153&amp;volume=117&amp;issue=1" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Average%20number%20of%20nucleotide%20differences%20in%20a%20sample%20from%20a%20single%20subpopulation%3A%20a%20test%20for%20population%20subdivision&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B33" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tajima</div> <div class="given-names">F</div></div>.&nbsp;</div><div class="article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">1989</div>,&nbsp;vol.&nbsp;<div class="volume">123</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">585</div>-<div class="lpage">595</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;author=F%20Tajima&amp;publication_year=1989&amp;journal=Genetics&amp;volume=123&amp;pages=585-595" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;aulast=Tajima&amp;date=1989&amp;spage=585&amp;epage=595&amp;volume=123&amp;issue=3" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B34" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vera</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Jansen</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Suppi</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">R/parallel—speeding up bioinformatics analysis with R</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">9</div>&nbsp;pg.&nbsp;<div class="fpage">390</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=R%2Fparallel%E2%80%94speeding%20up%20bioinformatics%20analysis%20with%20R&amp;author=G%20Vera&amp;author=R%20Jansen&amp;author=R%20Suppi&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=390" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2105-9-390" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-390" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-9-390"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18808714" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:R%2Fparallel%E2%80%94speeding%20up%20bioinformatics%20analysis%20with%20R&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B35" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vilella</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Blanco-Garcia</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Hutter</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Rozas</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">VariScan: analysis of evolutionary patterns from large-scale DNA sequence polymorphism data</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;<div class="issue">11</div>(pg.&nbsp;<div class="fpage">2791</div>-<div class="lpage">2793</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=VariScan%3A%20analysis%20of%20evolutionary%20patterns%20from%20large-scale%20DNA%20sequence%20polymorphism%20data&amp;author=J%20Vilella&amp;author=A%20Blanco-Garcia&amp;author=S%20Hutter&amp;author=J%20Rozas&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2791-2793" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/bti403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbti403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15814564" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:VariScan%3A%20analysis%20of%20evolutionary%20patterns%20from%20large-scale%20DNA%20sequence%20polymorphism%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B36" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wall</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Recombination and the power of statistical tests of neutrality</div>,&nbsp;<div class="source">Genet Res.</div>&nbsp;,&nbsp;<div class="year">1999</div>,&nbsp;vol.&nbsp;<div class="volume">74</div>&nbsp;<div class="issue">1</div>(pg.&nbsp;<div class="fpage">65</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Recombination%20and%20the%20power%20of%20statistical%20tests%20of%20neutrality&amp;author=J%20Wall&amp;publication_year=1999&amp;journal=Genet%20Res.&amp;volume=74&amp;pages=65-79" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1017/S0016672399003870" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672399003870" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1017%2FS0016672399003870"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Recombination%20and%20the%20power%20of%20statistical%20tests%20of%20neutrality&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msu136-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msu136-B37" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zeng</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Fu</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Shi</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Wu</div> <div class="given-names">C-I</div></div>.&nbsp;</div><div class="article-title">Statistical tests for detecting positive selection by utilizing high-frequency variants</div>,&nbsp;<div class="source">Genetics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">174</div>&nbsp;<div class="issue">3</div>(pg.&nbsp;<div class="fpage">1431</div>-<div class="lpage">1439</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Statistical%20tests%20for%20detecting%20positive%20selection%20by%20utilizing%20high-frequency%20variants&amp;author=K%20Zeng&amp;author=Y%20Fu&amp;author=S%20Shi&amp;author=C-I%20Wu&amp;publication_year=2006&amp;journal=Genetics&amp;volume=174&amp;pages=1431-1439" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1534/genetics.106.061432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.061432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1534%2Fgenetics.106.061432"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16951063" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Statistical%20tests%20for%20detecting%20positive%20selection%20by%20utilizing%20high-frequency%20variants&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id="FN1"><div class="fn"><div class="footnote-content"><strong>Associate editor:</strong> Juliette de Meaux</div></div></div><div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution Non-Commercial License 
(http://creativecommons.org/licenses/by-nc/3.0/), which permits 
non-commercial re-use, distribution, and reproduction in any medium, 
provided the original work is properly cited. For commercial re-use, 
please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2925788&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5325">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/31/7/10.1093_molbev_msu136/1/msu136_Supplementary_Data.zip?Expires=1577709648&amp;Signature=z9SlGCeCkuMUu-rjQWTmC1pUV3Xv-AscJekJiOrfqGdeWOklAd1Fh3nCzAP0GxpFEfhAWzZ696-PftkxBw1dyl2%7Et4ZkXDVFY%7ElWlUc-ZAw2O3OlcsLcRllT-WHNPV7FRSLV92KtQGn7QLhvQSXSlW24frytdKUIeHKwI6iQ26DnEdCkUHxSsezt-ff4lF-3%7EpJ59TjXgsx4SwI0RU9VLpRmUiUxDjdyoGU7K9h6KGXCqHp-HuZ5y257aP6DrbtosGsLqgj8eM7aiTC%7E6J%7EIK7dGJLFLCsoG1c%7E-HPaAB5tJxqvz6RlPsJ3-lK57H9IdheVd-ud4QZJ30%7EPz4JNC5w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">10,968</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339423800026&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">190</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/molbev/msu136" data-uuid="0318bc8d-6604-dee1-0ff4-7c9cd3fcb5a6"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=2286126" style="display:inline-block;">
        <img alt="Article has an altmetric score of 27" src="https://badges.altmetric.com/?size=128&amp;score=27&amp;types=tttttttf" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2925788"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339423800026&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fmbe%2farticle%2f31%2f7%2f1929%2f2925788">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339423800026&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (194)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F31%2F7%2F1929%2F2925788" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/mbe/crossref-citedby/2925788">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz275/5637232?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Gene duplication accelerates the pace of protein gain and loss from plant organelles
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz276/5637231?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Machine Learning Methods for Predicting 
Human-adaptative Influenza A Viruses Based on Viral Nucleotide 
Compositions
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz271/5626191?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Recurrent Collection of  Drosophila 
melanogaster  from Wild African Environments and Genomic Insights into 
Species History
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz234/5613640?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Dissecting the transcriptomic basis of phenotypic evolution in an aquatic keystone grazer
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz270/5625159?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Somatic mutations and genome stability maintenance in clonal coral colonies
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R | Molecular Biology and Evolution | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/molbev/msu136");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/mbe_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div>
</div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/mbe/pages/About">About Molecular Biology and Evolution</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/mbe/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/mbe/pages/Policies">Policies</a>
                                </li>
                                <li class="link-4">
                                    <a href="https://academic.oup.com/mbe/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://www.smbe.org/smbe/">Contact Us</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://twitter.com/OfficialSMBE">Twitter</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/mbe/subscribe">Purchase</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/mbe" target="">
                        <img id="footer-logo-MolecularBiologyandEvolution" class="journal-footer-affiliations-logo" src="mbe_f1-2062292105.svg" alt="Molecular Biology and Evolution">
                    </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1537-1719</li>
                                            <li>Print ISSN 0737-4038</li>
                    <li>Copyright © 2019 Society for Molecular Biology and Evolution</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh208" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1574685649484&amp;tmr=load%3D1574685649450%26core%3D1574685649476%26main%3D1574685649480%26ifr%3D1574685649488&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F31%2F7%2F1929%2F2925788&amp;href=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F31%2F7%2F1929%2F2925788&amp;dt=PopGenome%3A%20An%20Efficient%20Swiss%20Army%20Knife%20for%20Population%20Genomic%20Analyses%20in%20R&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5ddbcbd19eb8c563&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3DPopGenome%253A%2520An%2520Efficient%2520Swiss%2520Army%2520Knife%2520for%2520Population%2520Genomic%2520Analyses%2520in%2520R%26description%3DAbstract.%2520%2520Although%2520many%2520computer%2520programs%2520can%2520perform%2520population%2520genetics%2520calculations%252C%2520they%2520are%2520typically%2520limited%2520in%2520the%2520analyses%2520and%2520data%2520input%2520formats%2520they%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fmbe%252Farticle%252F31%252F7%252F1929%252F2925788%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fmbe%252FIssue%252F31%252F7%252F0%252Fm_mbe31_7.cover.gif%253FExpires%253D1637529725%2526Signature%253D0KKIT28wqF6rDpeLvs7UbXCv6VeTiA3ZIavhy1Z4ZmWHZBxFfNclEQs-FUJhSla3nDlOsVAC%7ELpqse3WjW-gX1lIZTX0ovC5m9MUEBE1lD3oFdYbhyw1woth%7EHn%7EkBQR7KxntRQfKfMPPJBN4EtusXN3QjlMezB9VpmedyrORlvZYtRWJLTyqo1fMaErc7zYj746N4W9wsq57dHMByfqa7LYOdxaZZoJk4UP1X2Bs5K2gfvUmGGa%7E9JF5CBxrYpXEiKif0aIyWk3vOhkqIyMos70aDrY3zLRF6R3NdswKTuruwfPvB-Pz6eV1Xe4%7E8kHRBBcqvzmXRCNH1mC0xCFbg__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2014-04-16&amp;csi=undefined&amp;rev=v8.28.1-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f31%2f7%2f1929%2f2925788">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>

    <script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="msu136.js"></script></body></html>