_altmetric.embed_callback({"title":"PopGenome: An Efficient Swiss Army Knife for Population Genomic Analyses in R","doi":"10.1093/molbev/msu136","pmid":"24739305","isbns":[],"altmetric_jid":"4f6fa62e3cf058f6100081dc","issns":["1537-1719","0737-4038"],"journal":"Molecular Biology and Evolution","cohorts":{"pub":20,"sci":28,"com":1},"abstract":"Although many computer programs can perform population genetics calculations, they are typically limited in the analyses and data input formats they offer; few applications can process the large data sets produced by whole-genome resequencing projects. Furthermore, there is no coherent framework for the easy integration of new statistics into existing pipelines, hindering the development and application of new population genetics and genomics approaches. Here, we present PopGenome, a population genomics package for the R software environment (a de facto standard for statistical analyses). PopGenome can efficiently process genome-scale data as well as large sets of individual loci. It reads DNA alignments and single-nucleotide polymorphism (SNP) data sets in most common formats, including those used by the HapMap, 1000 human genomes, and 1001 Arabidopsis genomes projects. PopGenome also reads associated annotation files in GFF format, enabling users to easily define regions or classify SNPs based on their annotation; all analyses can also be applied to sliding windows. PopGenome offers a wide range of diverse population genetics analyses, including neutrality tests as well as statistics for population differentiation, linkage disequilibrium, and recombination. PopGenome is linked to Hudson's MS and Ewing's MSMS programs to assess statistical significance based on coalescent simulations. PopGenome's integration in R facilitates effortless and reproducible downstream analyses as well as the production of publication-quality graphics. Developers can easily incorporate new analyses methods into the PopGenome framework. PopGenome and R are freely available from CRAN (http://cran.r-project.org/) for all major operating systems under the GNU General Public License.","abstract_source":"pubmed","context":{"all":{"count":13892828,"mean":8.218725194364,"rank":644586,"pct":95,"higher_than":13247935},"journal":{"count":3699,"mean":12.517048134127,"rank":376,"pct":89,"higher_than":3323},"similar_age_3m":{"count":190374,"mean":8.126954347518,"rank":10466,"pct":94,"higher_than":179899},"similar_age_journal_3m":{"count":79,"mean":12.826717948718,"rank":9,"pct":88,"higher_than":70}},"authors":["Bastian Pfeifer","Ulrich Wittelsb√ºrger","Sebastian E. Ramos-Onsins","Martin J. Lercher"],"type":"article","handles":[],"altmetric_id":2286126,"schema":"1.5.4","is_oa":false,"publisher_subjects":[{"name":"Genetics","scheme":"era"},{"name":"Evolutionary Biology","scheme":"era"},{"name":"Biochemistry And Cell Biology","scheme":"era"}],"cited_by_fbwalls_count":1,"cited_by_posts_count":52,"cited_by_tweeters_count":49,"cited_by_accounts_count":50,"last_updated":1475471296,"score":26.35,"history":{"1y":0,"6m":0,"3m":0,"1m":0,"1w":0,"6d":0,"5d":0,"4d":0,"3d":0,"2d":0,"1d":0,"at":26.35},"url":"http://dx.doi.org/10.1093/molbev/msu136","added_on":1397723391,"published_on":1397606400,"subjects":["molecularbiology"],"scopus_subjects":["Biochemistry, Genetics and Molecular Biology","Agricultural and Biological Sciences","Life Sciences"],"readers":{"citeulike":"2","mendeley":"508","connotea":"0"},"readers_count":510,"images":{"small":"https://badges.altmetric.com/?size=64&score=27&types=tttttttf","medium":"https://badges.altmetric.com/?size=100&score=27&types=tttttttf","large":"https://badges.altmetric.com/?size=180&score=27&types=tttttttf"},"details_url":"http://www.altmetric.com/details.php?citation_id=2286126"})