<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script>

        <title>Performance, Accuracy, and Web Server for Evolutionary 
Placement of Short Sequence Reads under Maximum Likelihood | Systematic 
Biology | Oxford Academic</title>

        <script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="/UI/app/js/jquery-2.2.4.js">\x3C/script>')</script>



  

    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <!-- Bookmark Icons -->
<link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png">
<link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32">
<link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16">
<link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json">
<link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5">
<link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico">
<meta name="msapplication-config" content="/UI/app/img/browserconfig.xml">
<meta name="theme-color" content="#002f65">

    



<link href="style.css" rel="stylesheet" type="text/css">


<link href="jquery.css" rel="stylesheet" type="text/css">


<link href="jscrollpane.css" rel="stylesheet" type="text/css">
<link href="theme.css" rel="stylesheet" type="text/css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site-2027664058.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'sysbio';
            App.SiteURL = 'academic.oup.com/sysbio';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/sysbio/article/60/3/291/1667010","@type":"ScholarlyArticle","name":"Performance, Accuracy, and Web Server for Evolutionary Placement of Short Sequence Reads under Maximum Likelihood","datePublished":"2011-03-23","isPartOf":{"@id":"https://academic.oup.com/sysbio/issue/60/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2011-05-01","isPartOf":{"@id":"https://academic.oup.com/sysbio","@type":"Periodical","name":"Systematic Biology","issn":["1076-836X"]}},"url":"http://dx.doi.org/10.1093/sysbio/syr010","keywords":["Maximum likelihood","metagenomics","phylogenetic placement","RAxML","short sequence reads"],"inLanguage":"en","copyrightHolder":"Society of Systematic Biologists","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/sysbio/article/60/3/291/1667010","author":[{"name":"Berger, Simon A.","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany","@type":"Person"},{"name":"Krompass, Denis","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany","@type":"Person"},{"name":"Stamatakis, Alexandros","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany","@type":"Person"}],"description":"Abstract.  We present an evolutionary placement algorithm (EPA) and a Web server for the rapid assignment of sequence fragments (short reads) to edges of a give","pageStart":"291","pageEnd":"302","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/60/3/0/m_sysbio60_3.cover.gif?Expires=1624442107&Signature=bvtaWs~xKVXspH9nhyTgb~elYj4NKgBjovHNSZYnjbhN0Ybu~QjS090RwpkzrqZyl9302fbV8AVhHQtyczIKNUXGZaC-KO1UDh4ajYJqG5L7IB6DYCoggAkpxzq5tRTDHpxIkK5EAy2DlKq8G79Hk5HQC7lKY5m0u69nbhmcVv~O2g~fX7AqHHiE3~8fK9pOcP0NlrmfsZOGN5MOoA6NvfZQObAXzKfzYCVbIV6r~v7PnVwgyZo5O7KHQBSKQVPJeZpyw0im3jMOTx8D0zs9AFOxTAvsyrU5UaZVnCvobTc9SGwdpOeduDj1zxi7V59YVRM8PKHoeJefUZuerdGLsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Performance, Accuracy, and Web Server for Evolutionary Placement of Short Sequence Reads under Maximum Likelihood","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/60/3/0/m_sysbio60_3.cover.gif?Expires=1624442107&Signature=bvtaWs~xKVXspH9nhyTgb~elYj4NKgBjovHNSZYnjbhN0Ybu~QjS090RwpkzrqZyl9302fbV8AVhHQtyczIKNUXGZaC-KO1UDh4ajYJqG5L7IB6DYCoggAkpxzq5tRTDHpxIkK5EAy2DlKq8G79Hk5HQC7lKY5m0u69nbhmcVv~O2g~fX7AqHHiE3~8fK9pOcP0NlrmfsZOGN5MOoA6NvfZQObAXzKfzYCVbIV6r~v7PnVwgyZo5O7KHQBSKQVPJeZpyw0im3jMOTx8D0zs9AFOxTAvsyrU5UaZVnCvobTc9SGwdpOeduDj1zxi7V59YVRM8PKHoeJefUZuerdGLsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="Performance, Accuracy, and Web Server for Evolutionary Placement of Short Sequence Reads under Maximum Likelihood">
    <meta property="og:description" content="Abstract.  We present an evolutionary placement algorithm (EPA) and a Web server for the rapid assignment of sequence fragments (short reads) to edges of a give">
    <meta property="og:url" content="https://academic.oup.com/sysbio/article/60/3/291/1667010">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/Issue/60/3/0/m_sysbio60_3.cover.gif?Expires=1624442107&amp;Signature=bvtaWs~xKVXspH9nhyTgb~elYj4NKgBjovHNSZYnjbhN0Ybu~QjS090RwpkzrqZyl9302fbV8AVhHQtyczIKNUXGZaC-KO1UDh4ajYJqG5L7IB6DYCoggAkpxzq5tRTDHpxIkK5EAy2DlKq8G79Hk5HQC7lKY5m0u69nbhmcVv~O2g~fX7AqHHiE3~8fK9pOcP0NlrmfsZOGN5MOoA6NvfZQObAXzKfzYCVbIV6r~v7PnVwgyZo5O7KHQBSKQVPJeZpyw0im3jMOTx8D0zs9AFOxTAvsyrU5UaZVnCvobTc9SGwdpOeduDj1zxi7V59YVRM8PKHoeJefUZuerdGLsQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2011-03-23">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Berger, Simon A."><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany"><meta name="citation_author" content="Krompass, Denis"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany"><meta name="citation_author" content="Stamatakis, Alexandros"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany"><meta name="citation_title" content="Performance, Accuracy, and Web Server for Evolutionary Placement of Short Sequence Reads under Maximum Likelihood"><meta name="citation_firstpage" content="291"><meta name="citation_lastpage" content="302"><meta name="citation_doi" content="10.1093/sysbio/syr010"><meta name="citation_journal_title" content="Systematic Biology"><meta name="citation_journal_abbrev" content="Syst Biol"><meta name="citation_volume" content="60"><meta name="citation_issue" content="3"><meta name="citation_publication_date" content="2011/05/01"><meta name="citation_issn" content="1063-5157"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1225-1231; "><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389"><meta name="citation_reference" content="Accuracy of morphology-based phylogenetic fossil placement under maximum likelihood. Proceedings of IEEE/ACS International Conference on Computer Systems and Applications (AICCSA-10); 2010 May 16–18; Hammamet, Tunisia: IEEE Computer Society. p. 1–8;  citation_year=2010; "><meta name="citation_reference" content="citation_title=Scaling of accuracy in extremely large phylogenetic trees; citation_publisher=World Scientific, River Edge (NJ); Pacific Symposium on Biocomputing 2001;  citation_year=2000;  citation_pages=547-558; "><meta name="citation_reference" content="citation_title=Phymm and PhymmBL: metagenomic phylogenetic classification with interpolated Markov models; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=673-676; "><meta name="citation_reference" content="citation_title=A detailed analysis of 16S ribosomal RNA gene segments for the diagnosis of pathogenic bacteria; J. Microbiol. Methods;  citation_year=2007;  citation_volume=69;  citation_pages=330-339; "><meta name="citation_reference" content="citation_title=NAST: a multiple sequence alignment server for comparative analysis of 16S rRNA genes; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W394-W399; "><meta name="citation_reference" content="citation_title=Profile hidden Markov models; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=755-763; "><meta name="citation_reference" content="citation_title=Muscle: multiple sequence alignment with high accuracy and high throughput; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1792-1797; "><meta name="citation_reference" content="citation_title=Evolutionary trees from DNA sequences: a maximum likelihood approach; J. Mol. Evol.;  citation_year=1981;  citation_volume=17;  citation_pages=368-376; "><meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using the bootstrap; Evolution;  citation_year=1985;  citation_volume=39;  citation_pages=783-791; "><meta name="citation_reference" content="The influence of sex, handedness, and washing on the diversity of hand surface bacteria. Proc. Natl. Acad. Sci. U.S.A;  citation_year=2008;  citation_volume=105;  citation_pages=17994-17999; "><meta name="citation_reference" content="citation_title=Inferring pattern and process: maximum-likelihood implementation of a nonhomogeneous model of DNA sequence evolution for phylogenetic analysis; Mol. Biol. Evol.;  citation_year=1998;  citation_volume=15;  citation_pages=871-879; "><meta name="citation_reference" content="citation_title=Methanogenic communities in permafrost-affected soils of the laptev sea coast, siberian arctic, characterized by 16s rRNA gene fingerprints; FEMS Microbiol. Ecol;  citation_year=2007;  citation_volume=59;  citation_pages=476-488; "><meta name="citation_reference" content="citation_title=Error detecting and error correcting codes. Bell Syst; Tech. J.;  citation_year=1950;  citation_volume=26;  citation_pages=147-160; "><meta name="citation_reference" content="citation_title=phyloXML: XML for evolutionary biology and comparative genomics; BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=356"><meta name="citation_reference" content="citation_title=An empirical test of bootstrapping as a method for assessing confidence in phylogenetic analysis; Syst. Biol.;  citation_year=1993;  citation_volume=42;  citation_pages=182"><meta name="citation_reference" content="citation_title=SeqVis: visualization of compositional heterogeneity in large alignments of nucleotides; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2162-2163; "><meta name="citation_reference" content="citation_title=MEGAN analysis of metagenomic data; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=377-386; "><meta name="citation_reference" content="citation_title=Estimation of phylogeny and invariant sites under the general Markov model of nucleotide sequence evolution; Syst. Biol.;  citation_year=2007;  citation_volume=56;  citation_pages=155-162; "><meta name="citation_reference" content="citation_title=The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated; Syst. Biol.;  citation_year=2004;  citation_volume=53;  citation_pages=638-643; "><meta name="citation_reference" content="citation_title=MAFFT version 5: improvement in accuracy of multiple sequence alignment; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=511-518; "><meta name="citation_reference" content="citation_title=Quantitative phyletics and the evolution of anurans; Syst. Zool;  citation_year=1969;  citation_volume=18;  citation_pages=1-32; "><meta name="citation_reference" content="citation_title=The closest BLAST hit is often not the nearest neighbor; J. Mol. Evol.;  citation_year=2001;  citation_volume=52;  citation_pages=540-542; "><meta name="citation_reference" content="citation_title=Obesity alters gut microbial ecology; Proc. Natl. Acad. Sci. U.S.A;  citation_year=2005;  citation_volume=102;  citation_pages=11070-11075; "><meta name="citation_reference" content="citation_title=Unexpected diversity and complexity of the guerrero negro hypersaline microbial mat; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=3685-3695; "><meta name="citation_reference" content="citation_title=Worlds within worlds: evolution of the vertebrate gut microbiota; Nat. Rev. Microbiol.;  citation_year=2008;  citation_volume=6;  citation_pages=776-788; "><meta name="citation_reference" content="citation_title=UniFrac: a new phylogenetic method for comparing microbial communities; Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=8228-8235; "><meta name="citation_reference" content="citation_title=Arb: a software environment for sequence data; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1363-1371; "><meta name="citation_reference" content="citation_title=Pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=538"><meta name="citation_reference" content="citation_title=Accurate phylogenetic classification of variable-length DNA fragments; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=63-72; "><meta name="citation_reference" content="citation_title=Sequence-length requirements for phylogenetic methods; citation_text=(Lecture Notes in Computer Science; 2452); citation_publisher=Springer, Berlin; Proceedings of Second International Workshop on Algorithms in Bioinformatics (WABI 02) 2002;  citation_year=2002;  citation_pages=343-356; "><meta name="citation_reference" content="citation_title=Statistical assignment of DNA sequences using Bayesian phylogenetics; Syst. Biol.;  citation_year=2008;  citation_volume=57;  citation_pages=750-757; "><meta name="citation_reference" content="citation_title=Statistical approaches for DNA barcoding; Syst. Biol.;  citation_year=2006;  citation_volume=55;  citation_pages=162-169; "><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_issue=Database issue;  citation_pages=D61-D65; "><meta name="citation_reference" content="citation_title=Pyrosequencing sheds light on DNA sequencing; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=3-11; "><meta name="citation_reference" content="Phylogenetic models of rate heterogeneity: a high performance computing perspective. Proceedings of 20th IEEE/ACM International Parallel and Distributed Processing Symposium (IPDPS2006); 2006 Apr 25–29; Rhodes Island, Greece. IEEE Computer Society. p. 278;  citation_year=2006; "><meta name="citation_reference" content="citation_title=RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=2688-2690; "><meta name="citation_reference" content="citation_title=A rapid bootstrap algorithm for the RAxML Web servers; Syst. Biol.;  citation_year=2008;  citation_volume=57;  citation_pages=758-771; "><meta name="citation_reference" content="Evolutionary placement of short sequence reads on multi-core architectures. Proceedings of IEEE/ACS International Conference on Computer Systems and Applications (AICCSA-10); 2010 May 16–18; Hammamet, Tunisia. IEEE Computer Society. p. 1–8;  citation_year=2010; "><meta name="citation_reference" content="citation_title=RAxML-III: a fast program for maximum likelihood-based inference of large phylogenetic trees; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_issue=4;  citation_pages=456-463; "><meta name="citation_reference" content="citation_title=Inferring confidence sets of possibly misspecified gene trees; Proc. R. Soc. B Biol. Sci.;  citation_year=2002;  citation_volume=269;  citation_pages=137-142; "><meta name="citation_reference" content="citation_title=A core gut microbiome in obese and lean twins; Nature;  citation_year=2008;  citation_volume=457;  citation_pages=480-484; "><meta name="citation_reference" content="citation_title=Quantitative phylogenetic assessment of microbial communities in diverse environments; Science;  citation_year=2007;  citation_volume=315;  citation_pages=1126-1130; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/sysbio/article-pdf/60/3/291/24555170/syr010.pdf"><meta name="description" content="Abstract.  We present an evolutionary placement algorithm (EPA) and a Web server for the rapid assignment of sequence fragments (short reads) to edges of a give">    <link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019061701.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-71f30fc33cff2fd0a5de8e407f36bda8.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-f6c32e9d70390ae112e1f1b71326f9b4.css"><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax_LineBox {display: table!important}
.MathJax_LineBox span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head><body data-sitestyletemplate="Journal" theme-sysbio="" class="off-canvas pg_Article pg_article" data-sitename="systematicbiology" style="cursor: default;"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/sysbio/article-xml/60/3/291/1667010">



    <link rel="canonical" href="https://academic.oup.com/sysbio/article/60/3/291/1667010">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/sysbio_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/sysbio_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/sysbio_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/sysbio_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "sysbio/article/60/3/291/1667010");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&siteid=sysbio&authentication_method=IP&authzrequired=false&account_id=2783535&account_list=2783535,2785688&authnips=197.248.44.39&doi=10.1093/sysbio/syr010';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5349" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">International Institue of Tropical Agriculture (IITA) <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f60%2f3%2f291%2f1667010" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            International Institue of Tropical Agriculture (IITA)
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f60%2f3%2f291%2f1667010" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311923">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311924">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311921">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311925">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311926">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311927">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311928">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311922">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311929">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311930">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311931">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311932">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311933">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311934">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311935">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311936">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311937">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311938">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/sysbio" class="journal-logo-container">
                    <img id="logo-SystematicBiology" class="journal-logo" src="sysbio_title1175497244.svg" alt="Systematic Biology">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.systbio.org/" target="" class="society-logo-container">
                                <img id="logo-SocietyofSystematicBiologists" class="society-logo" src="sysbio_h1-360035926.svg" alt="Society of Systematic Biologists">
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311923">
            <a href="https://academic.oup.com/sysbio/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311924">
            <a href="https://academic.oup.com/sysbio/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311921">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311925">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311926">
            <a href="https://mc.manuscriptcentral.com/systbiol" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311927">
            <a href="https://academic.oup.com/sysbio/pages/General_Instructions" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311928">
            <a href="https://academic.oup.com/sysbio/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311922">
            <a href="https://academic.oup.com/sysbio/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311929">
            <a href="https://academic.oup.com/sysbio/pages/About" class="nav-link ">About Systematic Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311930">
            <a href="http://www.systbio.org/" class="nav-link ">About the Society of Systematic Biologists</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311931">
            <a href="https://academic.oup.com/sysbio/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311932">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311933">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311934">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311935">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311936">
            <a href="http://www.systbio.org/" class="nav-link ">Contact Us</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1311937">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1311938">
            <a href="https://academic.oup.com/sysbio/pages/membership_society_systematic_biologists" class="nav-link ">Join SSB</a>
            
        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5349" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=66628" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/sysbio" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="sysbio" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/sysbio/issue/60/3"><img id="issueImage" class="fb-featured-image" src="m_sysbio60_3.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/sysbio/issue/60/3">
                        <div class="volume trailing-comma">Volume 60</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
May 2011        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="115076852">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076852">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076863">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076863">E<span class="small-caps">VOLUTIONARY</span> P<span class="small-caps">LACEMENT</span> A<span class="small-caps">LGORITHM</span></a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076875">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076875">E<span class="small-caps">XPERIMENTAL</span> S<span class="small-caps">ETUP</span></a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076890">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076890">R<span class="small-caps">ESULTS AND</span> D<span class="small-caps">ISCUSSION</span></a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076922">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076922">EPA WEB SERVICE</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076925">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076925">C<span class="small-caps">ONCLUSION</span></a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076929">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076929">S<span class="small-caps">UPPLEMENTARY</span> M<span class="small-caps">ATERIAL</span></a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076931">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076931">F<span class="small-caps">UNDING</span></a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="115076935">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#115076935">References</a>
            </div>
        </li>
<li class="section-jump-link head-1"><div class="section-jump-link__link-wrap"><a class="jumplink js-jumplink scrollTo" href="#authorNotesSectionTitle">Author notes</a></div></li><li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/sysbio/article/60/3/276/1667313">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/sysbio/article/60/3/303/1665734">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Performance, Accuracy, and Web Server for Evolutionary Placement of Short Sequence Reads under Maximum Likelihood
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Simon A. Berger</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing 
Group, Heidelberg Institute for Theoretical Studies, 
Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Simon+A.+Berger">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Berger%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Berger%20Simon%20A.%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Simon A. Berger</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Denis Krompass</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing 
Group, Heidelberg Institute for Theoretical Studies, 
Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Denis+Krompass">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Krompass%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Krompass%20Denis%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Denis Krompass</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexandros Stamatakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing 
Group, Heidelberg Institute for Theoretical Studies, 
Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg, Germany</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>Correspondence to be sent 
to: The Exelixis Lab, Scientific Computing Group, Heidelberg Institute 
for Theoretical Studies, Schloss-Wolfsbrunnenweg 35, D-69118 Heidelberg,
 Germany; E-mail: <a href="mailto:alexandros.stamatakis@epfl.ch" target="">alexandros.stamatakis@epfl.ch</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/sysbio/search-results?f_Authors=Alexandros+Stamatakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stamatakis%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stamatakis%20Alexandros%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alexandros Stamatakis<i class="icon-general-mail"></i></a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Associate Editor: Lars Jermiin</div></div></div> 
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Systematic Biology</em>, Volume 60, Issue 3, May 2011, Pages 291–302, <a href="https://doi.org/10.1093/sysbio/syr010">https://doi.org/10.1093/sysbio/syr010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 March 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 March 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">08 June 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 January 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1667010" href="https://academic.oup.com/sysbio/article-pdf/60/3/291/24555170/syr010.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Simon A. Berger, Denis Krompass, Alexandros Stamatakis,  
Performance, Accuracy, and Web Server for Evolutionary Placement of 
Short Sequence Reads under Maximum Likelihood, <em>Systematic Biology</em>, Volume 60, Issue 3, May 2011, Pages 291–302, <a href="https://doi.org/10.1093/sysbio/syr010">https://doi.org/10.1093/sysbio/syr010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=1667010&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1076-836X&amp;title=Performance%2C%20Accuracy%2C%20and%20Web%20Server%20for%20Evolutionary%20Placement%20of%20Short%20Sequence%20Reads%20under%20Maximum%20Likelihood&amp;publicationDate=2011-03-23&amp;volumeNum=60&amp;issueNum=3&amp;author=Berger%2C%20Simon%20A.%3B%20Krompass%2C%20Denis&amp;startPage=291&amp;endPage=302&amp;contentId=10.1093%2Fsysbio%2Fsyr010&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Systematic Biology</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            International Institue of Tropical Agriculture (IITA)
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f60%2f3%2f291%2f1667010" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Systematic Biology</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/sysbio/advanced-search?page=1&amp;fl_SiteID=5349&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="sysbio" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="115076852" id="115076852" class="abstract-title">Abstract</h2>
<section class="abstract"><p>We present an evolutionary placement 
algorithm (EPA) and a Web server for the rapid assignment of sequence 
fragments (short reads) to edges of a given phylogenetic tree under the 
maximum-likelihood model. The accuracy of the algorithm is evaluated on 
several real-world data sets and compared with placement by pair-wise 
sequence comparison, using edit distances and BLAST. We introduce a slow
 and accurate as well as a fast and less accurate placement algorithm. 
For the slow algorithm, we develop additional heuristic techniques that 
yield almost the same run times as the fast version with only a small 
loss of accuracy. When those additional heuristics are employed, the run
 time of the more accurate algorithm is comparable with that of a simple
 BLAST search for data sets with a high number of short query sequences.
 Moreover, the accuracy of the EPA is significantly higher, in 
particular when the sample of taxa in the reference topology is sparse 
or inadequate. Our algorithm, which has been integrated into RAxML, 
therefore provides an equally fast but more accurate alternative to 
BLAST for tree-based inference of the evolutionary origin and 
composition of short sequence reads. We are also actively developing a 
Web server that offers a freely available service for computing read 
placements on trees using the EPA.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Maximum likelihood&quot;">Maximum likelihood</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenomics">metagenomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic placement&quot;">phylogenetic placement</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RAxML">RAxML</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;short sequence reads&quot;">short sequence reads</a></div><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/sysbio/search-results?f_TocHeadingTitle=Regular%20articles">Regular articles</a>
                </div></div>
<p>Identification of organisms from, for example, microbial communities 
increasingly relies on analysis of DNA extracted from soil or water 
samples containing many, often unknown, organisms rather than from the 
individual organisms. Recently, the advent of new DNA sequencing 
techniques (e.g., pyrosequencing; <a reveal-id="bib36" data-open="bib36" class="link link-ref link-reveal xref-bibr">Ronaghi 2001</a>)
 has increased the amount of sequence data available for identification 
and analysis of microbial communities by several orders of magnitude. 
This rapid increase in the amount of sequence data available poses new 
challenges for short-read sequence identification tools. We can no 
longer expect that the steady increase in computing power, according to 
Moore's law, is fast enough to handle this <em>flood of sequence data</em>.</p><p>Depending
 on the DNA sequencing method used, a single sequencing run can already 
generate more than 100,000 short-read sequences, which comprise sequence
 fragments with a length of ∼30–450 nucleotides (base pairs). Such 
sequencing runs can be carried out within about an hour. Besides rapid 
full-genome assembly, another important application is the sampling of 
microbial communities from, for example, permafrost-affected soils 
(Ganzert et al. 2007), vertebrate guts (Ley et al. 2005, Ley et al. 
2008, Turnbaugh et al. 2008), hypersaline mats (Ley et al. 2006), or on 
hands of humans (Fierer et al. 2008).</p><p>Given the large amount of 
short-read sequences that metagenomic studies of microbial communities 
often yield and the fact that the provenance of these often is unknown, 
the first step in studies of metagenomic data is to identify the 
biological origin of these reads. This assignment of short reads to 
known organisms then allows us to examine and compare microbial samples 
and communities (see Turnbaugh et al. 2008). For instance, 20% of the 
reads in one sample might be most closely related to a specific 
taxonomic group of bacteria, whereas in a different sample, only 5% may 
be associated with this group.</p><p>Here, we present a novel algorithm,
 the evolutionary placement algorithm (EPA), for rapid phylogenetic 
identification of anonymous query sequences (QSs), using a set of 
full-length reference sequences (RSs). The most straightforward approach
 to identify the origin of a QS is to use tools that are based on 
sequence similarity (e.g., BLAST; Altschul et al. 1997). However, the 
BLAST-based approach has an important limitation: It can yield 
misleading assignments of QS to RS if the sample of RS does not contain 
sequences that are sufficiently closely related to the QS (i.e., if the 
taxon sampling is sparse or inappropriate). Any approach based on 
pair-wise sequence similarity, like BLAST, will not unravel, but 
silently ignore, potential problems in the taxon sampling of the RS. For
 instance, given two RS <em>a</em> and <em>b</em>, a QS <em>q</em> may be identified by BLAST as being most closely related to <em>a</em>. In reality, <em>q</em> might be most closely related to a RS <em>c</em>, which is not included in the set of RS. Because this is a known problem (<a reveal-id="bib24" data-open="bib24" class="link link-ref link-reveal xref-bibr">Koski and Golding 2001</a>),
 recent studies of microbial communities have begun to employ 
phylogenetic methods for QS identification (von Mering et al. 2007), 
despite the significantly higher computational cost. This treatment of 
short sequence reads is related to phylogenetic tree reconstruction 
methods that employ stepwise addition of sequences (<a reveal-id="bib23" data-open="bib23" class="link link-ref link-reveal xref-bibr">Kluge and Farris 1969</a>),
 with the difference that each QS is individually placed in the 
phylogenetic reference tree (RT). If a QS connects to an internal edge 
of a RT comprising the RS (i.e., it is not located near a leaf of the 
tree), then this indicates that the sampling of the RS is insufficient 
to identify and characterize the diversity of the QS. This can be used 
as a means to identify parts of the tree in which taxon sampling is 
sparse and to guide sequencing efforts to improve the sampling.</p><p>To
 date, phylogeny-based identification of the provenance of anonymous 
reads is conducted as follows: The QSs are aligned with respect to a 
reference alignment (RA) for the RS and then inserted into the RT either
 via a complete de novo tree reconstruction, a constrained tree search, 
using the RT as a constraint or backbone, or a fast and/or approximate 
QS addition algorithm, such as ARB (Ludwig et al. 2004), which uses 
maximum parsimony (MP). For DNA barcoding, phylogeny-based Bayesian 
analysis methods have recently been proposed by Munch et al. (2008) as 
well as <a reveal-id="bib34" data-open="bib34" class="link link-ref link-reveal xref-bibr">Nielsen and Matz (2006)</a>; these, however, are applied to trees with significantly less taxa. Recently, <a reveal-id="bib5" data-open="bib5" class="link link-ref link-reveal xref-bibr">Brady and Salzberg (2009)</a>
 proposed the Phymm and PhymmBL algorithms for metagenomic phylogenetic 
classification and report an improved classification accuracy for 
simulated QS compared with BLAST for PhymmBL. Classification by Phymm is
 based on oligonucleotide composition, whereas PhymmBL uses a weighted 
combination of scores from Phymm and BLAST. These algorithms classify QS
 relative to a given database of unaligned bacterial genomes (the 
National Center for Biotechnology Information [NCBI] RefSeq database; 
Pruitt et al. 2007) and their phylogenetic labels as provided in the 
RefSeq database (i.e., from Phylum level down to Genus level). This is 
different to the EPA, which can be used with any set of aligned RS and 
places QS into a fully resolved bifurcating RT. Because of the different
 focus of Phymm and the EPA, it is currently impossible to directly 
compare their accuracy on the same data sets (e.g., multiple sequence 
alignments and fully resolved phylogenies are not provided by the NCBI 
RefSeq database). This also hinders direct comparison with other 
previous phylogenetic classification methods like PhyloPythia (McHardy 
et al. 2007), which is the only phylogenetic classifier that has been 
compared with Phymm (<a reveal-id="bib5" data-open="bib5" class="link link-ref link-reveal xref-bibr">Brady and Salzberg 2009</a>
 show that Phymm and BLAST substantially outperform PhyloPythia) and 
MEGAN (Hudson et al. 2007). However, it is possible to compare the 
relative performance of the different methods (Phymm, PhymmBL, and EPA) 
with placements/classifications obtained by using BLAST on data sets 
that are suitable for the respective methods. Therefore, we mainly 
compare the accuracy of the EPA with basic BLAST searches and discuss 
the analogous accuracy evaluation performed for Phymm/PhymmBL.</p><p>A 
very similar algorithm to the EPA, called pplacer, has been developed 
independently by Matsen et al. (2010). The execution times of pplacer 
are comparable with those of the EPA according to a joint performance 
study conducted by F. Matsen, A. Stamatakis, and S.A. Berger. A 
comparative study is included in Matsen et al. (2010).</p><p>The current
 standard approach for analysis of environmental reads yields a fully 
resolved bifurcating tree that often comprises more than 10,000 
sequences (Fierer et al. 2008, Turnbaugh et al. 2008). The alignments 
used to reconstruct these trees mostly comprise only a single gene, 
typically 16S or 18S rRNA. The reconstruction of such large trees with 
thousands of taxa, based on data from a single gene, is time consuming 
and hard because of the weak historical signal in the alignment, which 
results in a decreased reconstruction accuracy for trees with many but 
relatively short sequences (see Bininda-Emonds et al. 2000, Moret et al.
 2002). Moreover, in metagenomic data sets, a large number of QS will 
only have a length of ∼200–450 bp if a 454 sequencer is used. Thus, for 
identification of the provenance of short read QS, the lack of 
historical signal becomes even more prevalent and critical if a 
comprehensive tree is reconstructed. In order to solve the problems 
associated with the lack of signal and to significantly accelerate the 
analysis, we advocate a different approach that only computes the 
optimal insertion position for every QS in the RT with respect to its 
maximum-likelihood (ML) score.</p><p>We introduce a new algorithm for 
the phylogenetic placement of QS and thoroughly test the placement 
accuracy on 8 published data sets. We assess the impact of QS length on 
placement accuracy and conduct tests on short reads derived from 
original full-length sequences of the test data sets. Because 
phylogenetic placement is inherently more computationally intensive than
 BLAST-based placement, performance optimization is an important factor 
in the development of such an algorithm if it is to become a useful and 
fast alternative to BLAST. Therefore, we have devised several EPAs and 
heuristics with varying degrees of computational complexity.</p><p>The 
algorithm, which has been developed and tested in cooperation with 
microbial biologists, is already available in the latest open source 
code release of RAxML (<a reveal-id="bib37" data-open="bib37" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>b; version 7.2.7, released in August 2010, <a class="link link-uri" href="http://wwwkramer.in.tum.de/exelixis/" target="">http://wwwkramer.in.tum.de/exelixis/</a> software.html). The alpha version of the respective Web service is available at <a class="link link-uri" href="http://i12k-exelixis3.informatik.tu-muenchen.de/raxml" target="">http://i12k-exelixis3.informatik.tu-muenchen.de/raxml</a>.
 Our new approach represents a useful scalable and fast tool for 
evolutionarily sound identification of the provenance of environmental 
QS. The EPA and pplacer are currently the only algorithms that can 
perform the task described here. The parallelization of the EPA 
(Stamatakis et al. 2010) and the ability to conduct placements under all
 time-reversible substitution models and data types offered by RAxML is a
 unique feature of the EPA that helps it to scale well on large and 
diverse data sets. Pplacer can infer placements using either ML (like 
the EPA) or Bayesian posterior probabilities.</p>                            <h2 scrollto-destination="115076863" id="115076863" class="section-title">E<span class="small-caps">VOLUTIONARY</span> P<span class="small-caps">LACEMENT</span> A<span class="small-caps">LGORITHM</span></h2>
<p>The input for our EPA consists of a RT comprising the <em>r</em> RS and a large comprehensive alignment that contains the <em>r</em> RS and the <em>q</em>
 QS. The task of aligning several QS with respect to a given RS 
alignment can be accomplished with ARB (Ludwig et al. 2004), NAST 
(DeSantis et al. 2006), MUSCLE (<a reveal-id="bib9" data-open="bib9" class="link link-ref link-reveal xref-bibr">Edgar 2004</a>), and MAFFT (Katoh et al. 2005), or, as tested here, with HMMER (<a reveal-id="bib8" data-open="bib8" class="link link-ref link-reveal xref-bibr">Eddy 1998</a>).
 One key assumption is that the RT is biologically well established or 
that it has been obtained via a preceding thorough phylogenetic 
analysis. In a typical usage scenario, the EPA could be used for mapping
 distinct microbial samples, for instance, from time-series experiments 
or different locations/individuals, to the same, for example, microbial 
RT in order to compare communities. Because the same RT can be used in 
multiple placement runs, the inference of the RT is not part of the EPA.</p><p>Initially,
 the algorithm will read the RT and RA and mark all sequences not 
contained in the RT as QS. Thereafter, the ML model parameters and edge 
lengths on the RT will be optimized using the standard procedures 
implemented in RAxML (a description of the ML model can be found in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463705&amp;Signature=maIVDUzwT%7E2SCcMh1Eh%7Ej5oVy3f%7EmurtqKhDQr8zbSADUuBor8trjpKBG4Mm4ccBgd0jU--oacQ-99xNPAaA-Uf5ZddOkFEMkVAIjcu6wIyWx%7E0BcvUnOyEmw-I8ehBClWGvQV0japdTIVi3hyZ6DZhaRnjLSXHwAUiSTpFUDUcc6FTADVQCcTGXdlsAaEepki4DrJAB1bHUhNXGBslpyquJ23YbQF93gfc93iJZNd1XjkYyIhHg33M3Lij4MEskZp7bbXjZ2hmxCf3IJ-Rpl-w-qVEixMNF666mcCW8DxW6o5lMUM-Lrx1IuSy-NhAyTbGItxYFD8YS1rIWJ%7Ev51w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> [available from <a class="link link-uri" href="http://www.sysbio.oxfordjournals.org/" target="">http://www.sysbio.oxfordjournals.org/</a>]).</p><p>Once
 the model parameters and edge lengths have been optimized on the RT, 
the actual identification algorithm is invoked. It will visit the 2<em>r</em>
 − 3 edges of the RT by a preorder tree traversal, starting at an 
arbitrary edge of the tree leading to a tip (i.e., visit the current 
edge first and then recursively visit the two unvisited neighboring 
edges). At each edge, initially, the probability vectors of the RT to 
the left and the right will be recomputed (if they are not already 
oriented toward the current edge). Thereafter, the program will 
successively insert, and subsequently remove, one QS at a time into the 
current edge and compute the likelihood (henceforth denoted as the 
insertion score) of the respective tree containing <em>r</em> + 1 taxa. The insertion score will then be stored in a <em>q</em>×(2<em>r</em> − 3) table that keeps track of the insertion scores for all <em>q</em> QS into all 2<em>r</em>
 − 3 edges of the RT. In order to more rapidly compute the per-edge 
insertions of the QS, we use an approximation that is comparable with 
the lazy subtree rearrangement (LSR) moves that are deployed in the 
standard RAxML search algorithm (see Stamatakis et al. 2005). After 
inserting a QS into an edge of the RT, we would normally need to 
reoptimize all edge lengths of the <em>r</em> + 1 tipped tree in order 
to obtain the corresponding insertion score. Instead, we only optimize 
the three edges adjacent to the insertion node of the QS (see <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>)
 before computing the likelihood of the insertion. This approach rests 
on the same rationale that was used to justify the LSR moves. Our 
experimental results justify this approximation because it yields a high
 placement accuracy. We use two methods, like those used for the LSR 
moves, to reestimate the three edges adjacent to the insertion edge: A <em>fast</em> method and a <em>slow</em> method that uses the Newton–Raphson method. The fast method simply splits the insertion edge, <em>b</em><sub><em>r</em></sub>, in the RT into two parts, <em>b</em><sub><em>r</em>1</sub> and <em>b</em><sub><em>r</em>2</sub>, by setting <em>b</em><sub><em>r</em>1</sub> = <em>b</em><sub><em>r</em>2</sub> = <em>b</em><sub><em>r</em></sub>/2, and <em>b</em><sub><em>q</em></sub>
 = 0.9 (i.e., the edge leading to QS), where 0.9 is the default RAxML 
value to initialize edge lengths. These values were chosen empirically 
for good placement accuracy over varying input data. Note that, we used 
the slow method, that is, the most accurate placement method available 
in RAxML without the above approximations for the main accuracy 
evaluation presented here. Thereafter, we also conducted a separate 
comparison of the slow and fast methods. The slow method repeatedly 
applies the Newton–Raphson method to all three edges (<em>b</em><sub><em>r</em>1</sub>, <em>b</em><sub><em>r</em>2</sub>, and <em>b</em><sub><em>q</em></sub>) until no further application of the Newton–Raphson method is needed (i.e., when <em>ϵ</em> ≤ 0.00001, where <em>ϵ</em>
 is the edge length change between two invocations of the Newton–Raphson
 method). Alternatively, our algorithm can also use MP to prescore and 
order promising candidate insertion edges in order to further accelerate
 the placement process.</p>                        <a id="115076867" scrollto-destination="115076867"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 1</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f01_lw.gif" alt="Local optimization of edge lengths for the insertion of a QS into the RT." path-from-xml="sysbiosyr010f01_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076867/sysbiosyr010f01_lw.gif" path-from-xml="sysbiosyr010f01_lw.gif" target="_blank">View large</a><a section="115076867" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f01_lw.gif?Expires=1561463705&amp;Signature=eDc-E3t2B%7EhKAaS6vX4F-CWNyNfY4w72YpywSAhbkFksw8xFN7DkrwE0qfhDeMHVIoMAP9MgXZe6CITL1N5Vbj3wjQk6tux5N5Lr-Y7f4nSu0PtTMoZc%7EQEh9u7VNNs3h7iqrfVoFwhY-akgd3%7ERYLwZ3dTey9Q6eNVx3YTEuuOQdmcNgThS1Y0zveEiXXGGlMLDSNtXpE8zR8mfMB3OC8TqMxL3ynwVLGA3gUIrAYvN1VvyGm7WL4SJP-GRCIsitv%7E-J0j8Dpm6V7BXSYfss2zJkKpnb9HhxbzdjXo0dIjBzsBZ9pM6AFtdDcC1beZAhOqXrcT-f5fpHjeFNqSVqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076867&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f01_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Local optimization of edge lengths for the insertion of a QS into the RT.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 1</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f01_lw.gif" alt="Local optimization of edge lengths for the insertion of a QS into the RT." path-from-xml="sysbiosyr010f01_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076867/sysbiosyr010f01_lw.gif" path-from-xml="sysbiosyr010f01_lw.gif" target="_blank">View large</a><a section="115076867" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f01_lw.gif?Expires=1561463705&amp;Signature=eDc-E3t2B%7EhKAaS6vX4F-CWNyNfY4w72YpywSAhbkFksw8xFN7DkrwE0qfhDeMHVIoMAP9MgXZe6CITL1N5Vbj3wjQk6tux5N5Lr-Y7f4nSu0PtTMoZc%7EQEh9u7VNNs3h7iqrfVoFwhY-akgd3%7ERYLwZ3dTey9Q6eNVx3YTEuuOQdmcNgThS1Y0zveEiXXGGlMLDSNtXpE8zR8mfMB3OC8TqMxL3ynwVLGA3gUIrAYvN1VvyGm7WL4SJP-GRCIsitv%7E-J0j8Dpm6V7BXSYfss2zJkKpnb9HhxbzdjXo0dIjBzsBZ9pM6AFtdDcC1beZAhOqXrcT-f5fpHjeFNqSVqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076867&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f01_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Local optimization of edge lengths for the insertion of a QS into the RT.</p></div></div><p>The
 output of this procedure consists of the RT, enhanced by assignments of
 the QS to edges of the RT. Each QS is attached to the edge that yielded
 the best insertion score for the specific QS. Hence, the algorithm will
 return a multifurcating tree if two or more QS are assigned to the same
 edge. An example is depicted in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Figure 2</a>.</p>                        <a id="115076869" scrollto-destination="115076869"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 2</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f02_lw.gif" alt="Evolutionary identification of three QSs (QS0, QS1, QS2) using a 4-taxon RT." path-from-xml="sysbiosyr010f02_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076869/sysbiosyr010f02_lw.gif" path-from-xml="sysbiosyr010f02_lw.gif" target="_blank">View large</a><a section="115076869" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f02_lw.gif?Expires=1561463705&amp;Signature=ni1w10Yr%7E3hae%7Ek6lpOPq4SN32BqRSyzoqhk6hai94seac%7EToD8zri22dne9prdhYivXnJ15HFfnmnsFwBqC559SSjOe20poAe073WoxAdDL0R4wMUXJn5hsSMpLEja0rs-ddsRBVQJsSIxcG-dixRIYIghtdD4bHwRYXAYv%7EaNl3rGBzI0X9cWj3xqSDFUGmPX6oBUtwHO%7ETWL0pZoBUTdmc9l98VnMtxx%7EE8HkPyQbC043RPHt9s5EnLCkBmQkRzkE2gRoGjP6mvSfvbfHzLcqu%7ElM1TmtUOPORmSatUIsla5cdM6vTtfAgOAZUkx2XB7HTcm2S84IOkZbRoNDdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076869&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f02_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary identification of three QSs (QS<sub>0</sub>, QS<sub>1</sub>, QS<sub>2</sub>) using a 4-taxon RT.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 2</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f02_lw.gif" alt="Evolutionary identification of three QSs (QS0, QS1, QS2) using a 4-taxon RT." path-from-xml="sysbiosyr010f02_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076869/sysbiosyr010f02_lw.gif" path-from-xml="sysbiosyr010f02_lw.gif" target="_blank">View large</a><a section="115076869" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f02_lw.gif?Expires=1561463705&amp;Signature=ni1w10Yr%7E3hae%7Ek6lpOPq4SN32BqRSyzoqhk6hai94seac%7EToD8zri22dne9prdhYivXnJ15HFfnmnsFwBqC559SSjOe20poAe073WoxAdDL0R4wMUXJn5hsSMpLEja0rs-ddsRBVQJsSIxcG-dixRIYIghtdD4bHwRYXAYv%7EaNl3rGBzI0X9cWj3xqSDFUGmPX6oBUtwHO%7ETWL0pZoBUTdmc9l98VnMtxx%7EE8HkPyQbC043RPHt9s5EnLCkBmQkRzkE2gRoGjP6mvSfvbfHzLcqu%7ElM1TmtUOPORmSatUIsla5cdM6vTtfAgOAZUkx2XB7HTcm2S84IOkZbRoNDdA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076869&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f02_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Evolutionary identification of three QSs (QS<sub>0</sub>, QS<sub>1</sub>, QS<sub>2</sub>) using a 4-taxon RT.</p></div></div><p>The EPA algorithm can optionally use the nonparametric bootstrap (<a reveal-id="bib11" data-open="bib11" class="link link-ref link-reveal xref-bibr">Felsenstein 1985</a>) to account for uncertainty in the placement of the QS. An example for this is shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>.
 Thus, a QS might be placed repeatedly onto different edges of the RT 
with various levels of support. For the bootstrap procedure, we 
introduce additional heuristics to accelerate the insertion process. 
During the insertions onto the RT using the original alignment, we keep 
track of the insertion scores for all QS into all edges of the RT. For 
every QS, we can then sort the insertion edges by their scores and for 
each bootstrap replicate only conduct insertions for a specific QS into 
10% of the best-scoring insertion edges on the RT. This reduces the 
number of insertion scores to be computed per QS on each bootstrap 
replicate by 90% and therefore approximately yields a 10-fold speedup 
for the bootstrapping procedure. In a typical application scenario, one 
may determine the diversity of the environmental sample for every 
replicate using, for instance, UniFrac (<a reveal-id="bib28" data-open="bib28" class="link link-ref link-reveal xref-bibr">Lozupone and Knight 2005</a>), and then compute an average diversity over all replicates.</p>                        <a id="115076871" scrollto-destination="115076871"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 3</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f03_lw.gif" alt="Phylogenetic placement of three QSs (QS0, QS1, QS2) onto a 4-taxon RT with insertion support score." path-from-xml="sysbiosyr010f03_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076871/sysbiosyr010f03_lw.gif" path-from-xml="sysbiosyr010f03_lw.gif" target="_blank">View large</a><a section="115076871" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f03_lw.gif?Expires=1561463705&amp;Signature=BI6luY7FNlmlrDgSNsNRU2011vje%7EnsaTPOHiFqdOJ%7EzCPI6xBayfpT8PZaQ-6dEzuAGXuuvgcvRcT7LKC2iGnLq0QSZBmXTxWFZd-kQemFeUUeXloSqfXevrQepcP85wgRoM3-NK9OW%7EGPf6J1pHhUk55pq36RoBtCSaw3fBBlDSR0NHJzeRQxiIA6xaGOgeMrWiuEuCUKX88B68QfQn8LXMMDHehk2YGofUwfiBaYvqPr0AWhqGrBl-Rwx68hgaZnFNANC9e--7KTZWYB6zT1bY8JpTMZSneTWJxATxZ-5IzOr6CB1toIKTjBQUWNq6mk1SmW4Q7HM%7E7p2G0kGew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076871&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f03_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic placement of three QSs (QS<sub>0</sub>, QS<sub>1</sub>, QS<sub>2</sub>) onto a 4-taxon RT with insertion support score.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 3</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f03_lw.gif" alt="Phylogenetic placement of three QSs (QS0, QS1, QS2) onto a 4-taxon RT with insertion support score." path-from-xml="sysbiosyr010f03_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076871/sysbiosyr010f03_lw.gif" path-from-xml="sysbiosyr010f03_lw.gif" target="_blank">View large</a><a section="115076871" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f03_lw.gif?Expires=1561463705&amp;Signature=BI6luY7FNlmlrDgSNsNRU2011vje%7EnsaTPOHiFqdOJ%7EzCPI6xBayfpT8PZaQ-6dEzuAGXuuvgcvRcT7LKC2iGnLq0QSZBmXTxWFZd-kQemFeUUeXloSqfXevrQepcP85wgRoM3-NK9OW%7EGPf6J1pHhUk55pq36RoBtCSaw3fBBlDSR0NHJzeRQxiIA6xaGOgeMrWiuEuCUKX88B68QfQn8LXMMDHehk2YGofUwfiBaYvqPr0AWhqGrBl-Rwx68hgaZnFNANC9e--7KTZWYB6zT1bY8JpTMZSneTWJxATxZ-5IzOr6CB1toIKTjBQUWNq6mk1SmW4Q7HM%7E7p2G0kGew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076871&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f03_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic placement of three QSs (QS<sub>0</sub>, QS<sub>1</sub>, QS<sub>2</sub>) onto a 4-taxon RT with insertion support score.</p></div></div><p>As
 a faster alternative to the nonparametric bootstrap, the insertion 
scores can also be directly used to compute placement uncertainty. von 
Mering et al. (2007) used expected likelihood weights (ELW; <a reveal-id="bib42" data-open="bib42" class="link link-ref link-reveal xref-bibr">Strimmer and Rambaut 2002</a>)
 to assign QS to an area of a tree with a certain confidence. Methods 
for calculating a placement uncertainty using ELW are already 
implemented in the EPA and pplacer.</p><p>In order to improve the 
runtime of the slow insertion method, we developed two heuristics that 
rely on the fast scoring approach (described above) or MP scores, 
respectively. Given those prescoring techniques, the number of insertion
 positions considered for the thorough, but slow insertion process, can 
be reduced to a fraction of promising candidate edges. The proportion of
 insertion edges suggested by the rapid prescoring heuristics for 
analysis under the slow insertion method is determined by a user-defined
 parameter <em>f</em><em>h</em>. As part of our performance evaluation, we tested the ML- and MP-based heuristics with regard to this parameter setting.</p><p>It
 is a known problem (Jermiin et al. 2004) that compositional 
heterogeneity can bias phylogenetic inference because it implies that 
the sequences cannot have evolved under the same stationary, reversible,
 and homogeneous conditions (assumed by all the available 
time-reversible substitution models). In the case of the EPA, this can 
be problematic for the placement of QS onto potentially very short edges
 of the RT. Different approaches exist to resolve those problems by 
using more sophisticated models (e.g., <a reveal-id="bib13" data-open="bib13" class="link link-ref link-reveal xref-bibr">Galtier and Gouy 1998</a>,
 Jayaswal et al. 2007). Nonetheless, time reversibility is required to 
accommodate the high computational demands of large-scale tree and EPA 
inferences, in particular with respect to computing the likelihood on 
trees (the “pulley principle”; <a reveal-id="bib10" data-open="bib10" class="link link-ref link-reveal xref-bibr">Felsenstein 1981</a>).
 Therefore, we recommend that users of EPA test their alignment of RS 
for evidence that the sequences have not evolved under time-reversible 
conditions (e.g., using methods published by Ababneh et al. 2006 and Ho 
et al. 2006) before using the EPA. If the data have not evolved under 
time-reversible conditions, then the conclusions drawn from using the 
EPA should be given with this caveat in mind. These considerations do 
not affect our accuracy assessment because trees of full-length 
sequences and placements have been inferred under the same model and the
 potential errors that may occur because of inappropriate model 
selection affect both tree construction and placement.</p>                            <h2 scrollto-destination="115076875" id="115076875" class="section-title">E<span class="small-caps">XPERIMENTAL</span> S<span class="small-caps">ETUP</span></h2>
                            <h3 scrollto-destination="115076876" id="115076876" class="section-title">Data Sets</h3>
<p>To test the accuracy of the EPA and competing approaches, we used 
eight RAs of nucleotides or amino acids from 140 up to 1604 sequences. 
The experimental data span a broad range of organisms and include rbcL 
genes (D500), small subunit rRNA (D150, D218, D714, D855, and D1604), 
fungal DNA (D628), and amino acid sequences from <em>Papillomaviridae</em>
 (D140). For each set, we computed the most likely tree and obtained 
bootstrap support (BS; Stamatakis et al. 2008) values for the internal 
edges; this ML tree was denoted the RT. The data sets are available at <a class="link link-uri" href="http://wwwkramer.in.tum.de/exelixis/epaData.tar.bz2" target="">http://wwwkramer.in.tum.de/exelixis/epaData.tar.bz2</a>.
 The selection of the data sets and data types per se is not important, 
as long as QS with well-supported positions can be extracted from them. 
It should be noted that the number of data sets that could be assessed 
was limited by the excessive computational requirements of the 
leave-one-out experiments described in the following sections. The 
question we intend to answer by these experiments is this: can the EPA 
place a QS of reduced length onto approximately the same position from 
which the full-length QS was pruned? We specifically did not include 
real metagenomic data sets in our study because the phylogenetic 
positions of the QS are unknown. Moreover, real metagenomic data sets do
 not allow for comparing the placement accuracy, or inferred placement 
position, between full-length and short read sequences, whereas using a 
full-length sequence alignment and emulating short reads allows for such
 a comparison. Therefore, we emulated metagenomic data sets using 
real-world alignments with ML trees and BS support, which is as close as
 one can get to reality for assessing placement accuracy using real 
sequence data. To analyze real metagenomic data, we used the parallel 
version of the EPA, which can be applied to very large real metagenomic 
data sets (4874 RS and 100,627 QS; Stamatakis et al. 2010).</p>                            <h3 scrollto-destination="115076878" id="115076878" class="section-title">Generation of QS</h3>
<p>To evaluate the accuracy of our algorithm, we pruned one candidate QS
 at a time from the existing ML trees before reinserting the QS into the
 tree. We only pruned and reinserted those QS that were associated with 
high BS scores in the RT in order to assess placement accuracy for taxa 
whose position in the original tree is reliable. A candidate QS is 
considered to have high BS, when the BS of either one of the two edges 
to which the taxon is attached is ≥ 75% and the other one leads to a 
neighboring tip (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4a</a>),
 or if both of these edges have a BS ≥ 75% (Fig. 4b). The 75% threshold 
reflects the typical empirical cutoff that is widely used in 
phylogenetics (<a reveal-id="bib17" data-open="bib17" class="link link-ref link-reveal xref-bibr">Hillis and Bull 1993</a>).
 For each QS, a reduced RT is derived by pruning the respective tip from
 the original tree. The QS associated with that taxon is then placed 
onto the reduced tree (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4c</a>) with our EPA algorithm.</p>                        <a id="115076880" scrollto-destination="115076880"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 4</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f04_lw.gif" alt="Illustration of the criterion for the QS selection and experimental setup. (a) Candidate QS belongs to subtree of size 2 that is connected to the tree by a well-supported edge. It has one other tip as direct neighbor. QS with this property will be referred to as outer QS. (b) Candidate QS is connected to the tree by two well-supported edges. QS with this property will be referred to as inner QS. (c) Experimental setting: reinsert shortened candidate QS in to pruned RT. We use three different ways of generating QS with desirable features: contiguous subsequences, paired-end reads, and random gaps." path-from-xml="sysbiosyr010f04_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076880/sysbiosyr010f04_lw.gif" path-from-xml="sysbiosyr010f04_lw.gif" target="_blank">View large</a><a section="115076880" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f04_lw.gif?Expires=1561463705&amp;Signature=xo29PfHytlNeDF7wv5A640SpBhlxqpBEtjRNu5wjVZh2jrURc9m-DnEuJRJOe5FD2yDylBITEKQMo4bB5hNcxVluXCW%7Ecg7QmgXr3NHDylak4i8pU6ChBvy24npUB3zV7v4nUaE8LDTj4VPwIdYoxdIZXSeFr1e6ASn5qQvV1z7EnehemYSuCcG1hm3ue381RCSto7H3uX2pJvkqoimgrDo0wKYe-k4YbbyOFfoxaPXVTdN8GM6%7EwITTZrJnLDY9hkWXyIMiQuj59iNdq955C7%7E0BWuTt6SP4R59h5ZAhGL7eiWjmTale2kY0jlWOdSqAEEpbyCREy4VBB3AZ5VBmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076880&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f04_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration
 of the criterion for the QS selection and experimental setup. (a) 
Candidate QS belongs to subtree of size 2 that is connected to the tree 
by a well-supported edge. It has one other tip as direct neighbor. QS 
with this property will be referred to as outer QS. (b) Candidate QS is 
connected to the tree by two well-supported edges. QS with this property
 will be referred to as inner QS. (c) Experimental setting: reinsert 
shortened candidate QS in to pruned RT. We use three different ways of 
generating QS with desirable features: contiguous subsequences, 
paired-end reads, and random gaps.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 4</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f04_lw.gif" alt="Illustration of the criterion for the QS selection and experimental setup. (a) Candidate QS belongs to subtree of size 2 that is connected to the tree by a well-supported edge. It has one other tip as direct neighbor. QS with this property will be referred to as outer QS. (b) Candidate QS is connected to the tree by two well-supported edges. QS with this property will be referred to as inner QS. (c) Experimental setting: reinsert shortened candidate QS in to pruned RT. We use three different ways of generating QS with desirable features: contiguous subsequences, paired-end reads, and random gaps." path-from-xml="sysbiosyr010f04_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076880/sysbiosyr010f04_lw.gif" path-from-xml="sysbiosyr010f04_lw.gif" target="_blank">View large</a><a section="115076880" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f04_lw.gif?Expires=1561463705&amp;Signature=xo29PfHytlNeDF7wv5A640SpBhlxqpBEtjRNu5wjVZh2jrURc9m-DnEuJRJOe5FD2yDylBITEKQMo4bB5hNcxVluXCW%7Ecg7QmgXr3NHDylak4i8pU6ChBvy24npUB3zV7v4nUaE8LDTj4VPwIdYoxdIZXSeFr1e6ASn5qQvV1z7EnehemYSuCcG1hm3ue381RCSto7H3uX2pJvkqoimgrDo0wKYe-k4YbbyOFfoxaPXVTdN8GM6%7EwITTZrJnLDY9hkWXyIMiQuj59iNdq955C7%7E0BWuTt6SP4R59h5ZAhGL7eiWjmTale2kY0jlWOdSqAEEpbyCREy4VBB3AZ5VBmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076880&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f04_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration
 of the criterion for the QS selection and experimental setup. (a) 
Candidate QS belongs to subtree of size 2 that is connected to the tree 
by a well-supported edge. It has one other tip as direct neighbor. QS 
with this property will be referred to as outer QS. (b) Candidate QS is 
connected to the tree by two well-supported edges. QS with this property
 will be referred to as inner QS. (c) Experimental setting: reinsert 
shortened candidate QS in to pruned RT. We use three different ways of 
generating QS with desirable features: contiguous subsequences, 
paired-end reads, and random gaps.</p></div></div><p>In our test data 
sets, the QSs were always derived from the full-length sequences in the 
RA. In a typical application scenario, however, the placement algorithm 
will have to cope with QSs that are significantly shorter than the 
full-length sequences in the RA. Hence, we carry out a systematic 
assessment of the placement accuracy depending on the length of the QS 
by artificially shortening the full-length sequence in question via 
insertion of gaps. We deployed three distinct methods to produce a QS 
that are ordered according to increasing biological realism.</p><p>A 
first method to produce QS involved randomly replacing existing 
characters by gaps. Although this method arguably does not reflect a 
real usage scenario, it provides a means to systematically assess the 
placement of QS over a wide variation of “virtual read lengths” while 
minimizing the influence of the position-specific placement accuracy 
variation. Position-specific effects on placement accuracy have 
previously been identified as a problem by Chakravorty et al. (2007). 
Multiple placement runs were conducted for QS with the relative 
proportion of nongap characters set to 10%, 20%, …, 90%, up to the full 
sequence length. Because the sequences have been extracted from the 
original multiple alignment, the remaining nongap characters are still 
aligned to the RA. Because the proportion of gaps is calculated relative
 to the length of the RA, the maximum proportion of available nongap 
characters is alignment dependent. We emphasize that, mathematically, 
the introduction of random gaps does not influence the calculation of 
the likelihood function because the used models assume independence 
between sites. The results from these experiments show the qualitative 
relationship between QS length and placement accuracy, in a way that is 
not feasible with the more realistic QS generation method described 
below, because of the high computational requirements of these 
evaluations.</p><p>The second method to artificially shorten candidate 
QS involved randomly sampling contiguous subsequences from the 
full-length sequence in question. This method to produce QS closely 
reflects the main EPA application scenario. Typically, a large number of
 short sequence reads generated by next generation sequencing methods 
from unknown positions will need to be placed onto a RT. For every 
full-length sequence in question, we sampled 20 QS with uniformly 
distributed positions and normally distributed lengths (mean length: 
200±60 bp for nucleotides; 70±20 for amino acids). This roughly 
corresponds to the read lengths generated by current high throughput 
sequencing technologies. We sampled 20 QS per full-length sequence to 
minimize the aforementioned influence of position-specific bias on the 
placement of QS. Because of the high computational requirements of 
placing 20 QS per full-length sequence, we did not repeat this 
evaluation for different mean read lengths. To assess the relationship 
between read length and placement accuracy, we conducted the random gap 
evaluation. Because a method is needed to align the short reads to the 
RA in a typical analysis using EPA, we also assessed placement accuracy 
using realigned QS and compared it with the QS placement accuracy 
induced by using the original alignment. We used HMMER to realign the QS
 to a profile hidden Markov model (HMM) of the RA (sequence-to-profile 
alignments with MUSCLE and MAFFT resulted in a slightly inferior 
placement accuracy; data not shown). Because the realignment procedure 
is not an integral part of the EPA, and future developments could 
potentially improve the realignment quality, we present results for the 
EPA with and without HMMER realignment.</p><p>A third method to produce QS involved generating these such that they correspond to paired-end reads (see <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4c</a>;
 in this experiment, we excluded data set D140 because it is a multigene
 alignment of amino acid sequences). Thus, in contrast to the previous 
method, the position of the extracted subsequences within the gene 
remains fixed. This modification reflects another real-world scenario of
 the EPA because paired-end sequencing is a widely used technique. Once 
again, we conducted our placement accuracy assessment on paired-end 
reads that were artificially generated from the full-length sequence in 
question by replacing all characters in the middle of the sequence by 
gaps. The artificial paired-end reads were 2×50 bp and 2×100 bp in 
length.</p>                            <h3 scrollto-destination="115076885" id="115076885" class="section-title">Comparison with Placements Based on Pair-Wise Sequence Similarity</h3>
<p>We conducted our evaluation of the accuracy of the EPA by comparison 
with a typical application scenario in which appropriate sequence-based 
search tools such as BLAST are used to assign a QS to the most similar 
RS. In this setting, a QS will always be assigned to one of the terminal
 edges of the RT. As mentioned above, for the EPA tests, we can choose 
to reuse the alignment information from the original multiple sequence 
alignment from which the QS were generated. With BLAST we do not have 
this option, so all QS will effectively be realigned against the RS. For
 this reason, we compare BLAST against the EPA with and without previous
 QS realignment using HMMER. For all tests involving BLAST, we removed 
all gaps from the multiple alignment and built a BLAST database for each
 data set. We also removed all gaps from the candidate QS and 
concatenated the two ends of the artificial paired-end reads into one 
sequence. Searches with those sequences were conducted against the 
corresponding BLAST database. The default parameters of the BLAST 
program from the NCBI C Toolkit were used for character match/mismatch 
(scores 1 and − 3) and gaps (nonaffine gap penalty of − 1). The default 
values from the NCBI BLAST Web site with affine gap penalties were also 
tested but produced slightly worse placement results and higher run 
times than the default settings. Using BLAST has the disadvantage that 
the information stemming from the RA that is present in the QS cannot be
 used, so for fairness, the BLAST placements should be mainly compared 
with EPA placements with previous QS realignment. For the tests on 
random gap QS, we did not use BLAST for comparison because it is not 
well suited for aligning such QS. The gap model as well as the local 
alignment algorithm rely on contiguous sequence stretches of certain 
lengths, which were not present in the random gap QS. For the random gap
 evaluation, we used a custom distance measure to calculate the 
pair-wise sequence similarities as an alternative to BLAST. It is 
defined as follows: For the two aligned sequences in question, we count 
the number of positions, where two different nongap characters are 
aligned with each other. This measure corresponds to the <a reveal-id="bib15" data-open="bib15" class="link link-ref link-reveal xref-bibr">Hamming (1950)</a>
 distance, where a gap is a placeholder for any character. Placements 
derived from this distance measure will be referred to as sequence-based
 nearest-neighbor (SEQ-NN) placements.</p>                            <h3 scrollto-destination="115076887" id="115076887" class="section-title">Accuracy Measures</h3>
<p>To quantify placement accuracy, we used two distance measures based 
on the topology and edge lengths of the original ML tree. In all cases, 
we considered an original edge and an insertion edge. The original edge 
is the one corresponding to the sequence used to generate the QS and 
into which it should ideally be reinserted. The insertion edge is 
computed by the EPA. To quantify the distance between the original edge 
and the insertion edge, we use the following two distance measures: The 
node distance (ND) is the unweighted path length in the original RT 
between the two edges. This corresponds to the number of nodes located 
on the path that connects the two edges (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5a</a>)
 and represents an absolute distance measure. The second measure is the 
sum of edge lengths on the path connecting the two edges. This measure 
also includes 50% of the length of the insertion edge and 50% of the 
length of the original edge (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5a</a>).
 For comparability between different trees and in order to obtain a 
relative measure, we normalize the edge path length by dividing it by 
the maximum tree diameter (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5b</a>).
 The maximum diameter is the edge path of maximum length between two 
taxa in the RT. This distance measure is henceforth denoted as 
normalized edge distance (NED%).</p>                        <a id="115076889" scrollto-destination="115076889"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 5</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f05_lw.gif" alt="Illustration of the tree-based distance measures. a) Example tree with two edges (original and insertion edge) highlighted. There are two nodes on the path, so the ND is 2. The edge distance corresponds to the length of the connecting path, where of the two end edges only half of the edge length is used. b) Tree diameter that is used to normalize the edge distance." path-from-xml="sysbiosyr010f05_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076889/sysbiosyr010f05_lw.gif" path-from-xml="sysbiosyr010f05_lw.gif" target="_blank">View large</a><a section="115076889" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f05_lw.gif?Expires=1561463705&amp;Signature=NPMLdRc6z6pqkaMz1iONyPkHJFce4yYiDNCUJDnWWPObaBZadKTuDXmPGf26-XkTQ6xTZpGiHNFwjvBd7rzi8nAqZaZp1NFWj90P-7uHvFWFp9DNgRfaRvNwuN1Mj7QBXZP06JJdUYDuX3zb4A%7EcvXZ5eOeAD%7EqAC8MYZapRSQn77f-RDa4NmpVV1xtrw2ROjRJRDDVW-9Ggh-ZhJehSKu3OO3Y8S%7EX9xcO%7E2CUS3L-2lzeNpJ51a-aZv0kuqQdwt8iX1PKG4qLL3nHIBLOyWYkC0zAkmxQhyRsRuc54Jb3RfYS7%7E8DT8WlHO85-U4D43RluxoLa1PDXR7mOOXVn%7Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076889&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f05_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration
 of the tree-based distance measures. a) Example tree with two edges 
(original and insertion edge) highlighted. There are two nodes on the 
path, so the ND is 2. The edge distance corresponds to the length of the
 connecting path, where of the two end edges only half of the edge 
length is used. b) Tree diameter that is used to normalize the edge 
distance.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 5</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f05_lw.gif" alt="Illustration of the tree-based distance measures. a) Example tree with two edges (original and insertion edge) highlighted. There are two nodes on the path, so the ND is 2. The edge distance corresponds to the length of the connecting path, where of the two end edges only half of the edge length is used. b) Tree diameter that is used to normalize the edge distance." path-from-xml="sysbiosyr010f05_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076889/sysbiosyr010f05_lw.gif" path-from-xml="sysbiosyr010f05_lw.gif" target="_blank">View large</a><a section="115076889" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f05_lw.gif?Expires=1561463705&amp;Signature=NPMLdRc6z6pqkaMz1iONyPkHJFce4yYiDNCUJDnWWPObaBZadKTuDXmPGf26-XkTQ6xTZpGiHNFwjvBd7rzi8nAqZaZp1NFWj90P-7uHvFWFp9DNgRfaRvNwuN1Mj7QBXZP06JJdUYDuX3zb4A%7EcvXZ5eOeAD%7EqAC8MYZapRSQn77f-RDa4NmpVV1xtrw2ROjRJRDDVW-9Ggh-ZhJehSKu3OO3Y8S%7EX9xcO%7E2CUS3L-2lzeNpJ51a-aZv0kuqQdwt8iX1PKG4qLL3nHIBLOyWYkC0zAkmxQhyRsRuc54Jb3RfYS7%7E8DT8WlHO85-U4D43RluxoLa1PDXR7mOOXVn%7Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076889&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f05_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration
 of the tree-based distance measures. a) Example tree with two edges 
(original and insertion edge) highlighted. There are two nodes on the 
path, so the ND is 2. The edge distance corresponds to the length of the
 connecting path, where of the two end edges only half of the edge 
length is used. b) Tree diameter that is used to normalize the edge 
distance.</p></div></div>                            <h2 scrollto-destination="115076890" id="115076890" class="section-title">R<span class="small-caps">ESULTS AND</span> D<span class="small-caps">ISCUSSION</span></h2>
                            <h3 scrollto-destination="115076891" id="115076891" class="section-title">Placement Accuracy for Random Gap QS</h3>
<p>To assess the influence of QS length and at the same time reduce the 
impact of positional variability on placement accuracy, we tested the 
accuracy of the EPA on random-gap sequences of various lengths. 
Placements were carried out on the 8 data sets for varying artificial 
read lengths. <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Figure 6</a>
 provides a detailed plot of the accuracy as a function of the 
proportion of gaps, averaged over all candidate QS from all data sets 
(respective plots for the individual data sets can be found in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>).
 As a measure of accuracy, we use the distance between the placement 
position and the true position from which the respective QS was 
originally pruned. Therefore, a lower distance indicates higher 
accuracy. For each placement method, we show the distances for three 
subsets of candidate QS: the disjunct sets of outer QS and inner QS (see
 Fig. 4), as well as the complete set of QS (all QS).</p>                        <a id="115076893" scrollto-destination="115076893"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 6</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f06_lw.gif" alt="Placement accuracy for QS with artificially introduced random gaps. a) Average ND and b) NED (between insertion positions and real positions)." path-from-xml="sysbiosyr010f06_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076893/sysbiosyr010f06_lw.gif" path-from-xml="sysbiosyr010f06_lw.gif" target="_blank">View large</a><a section="115076893" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f06_lw.gif?Expires=1561463706&amp;Signature=RwgBziUULpiUX1rQWmjrxx4DqpnkXY7tPDwW4D7GYN-xyInlAYnU4oENJ9GeymN-Wf4s-iRT81KQZC0oNMBbbQg5cTWYnGXAC1NggpHrJqSm4O7Ot4GPg64gCeR%7E7O2n%7EENFOpLfl3a%7Eb2Gz7YCBcdE0o1lBltYE7VCk0Kteus7dpnfwWLIbrGrmXqeNn2agsOvBpYrQRvTy2f9XVZ-x6WhvyXiwtNL4gXJsQPuPIyq1d0lsmuAY0GA-24OIGxyCQA2Up--mMejaMxZJe3ldVdxskde0uZLlF3GQrpvmJP7wo3dZAbjoW%7EbYjeGoXb0Btbs%7E%7EblhnBJTucDhjTcKmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076893&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f06_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Placement
 accuracy for QS with artificially introduced random gaps. a) Average ND
 and b) NED (between insertion positions and real positions).</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 6</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f06_lw.gif" alt="Placement accuracy for QS with artificially introduced random gaps. a) Average ND and b) NED (between insertion positions and real positions)." path-from-xml="sysbiosyr010f06_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076893/sysbiosyr010f06_lw.gif" path-from-xml="sysbiosyr010f06_lw.gif" target="_blank">View large</a><a section="115076893" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f06_lw.gif?Expires=1561463706&amp;Signature=RwgBziUULpiUX1rQWmjrxx4DqpnkXY7tPDwW4D7GYN-xyInlAYnU4oENJ9GeymN-Wf4s-iRT81KQZC0oNMBbbQg5cTWYnGXAC1NggpHrJqSm4O7Ot4GPg64gCeR%7E7O2n%7EENFOpLfl3a%7Eb2Gz7YCBcdE0o1lBltYE7VCk0Kteus7dpnfwWLIbrGrmXqeNn2agsOvBpYrQRvTy2f9XVZ-x6WhvyXiwtNL4gXJsQPuPIyq1d0lsmuAY0GA-24OIGxyCQA2Up--mMejaMxZJe3ldVdxskde0uZLlF3GQrpvmJP7wo3dZAbjoW%7EbYjeGoXb0Btbs%7E%7EblhnBJTucDhjTcKmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076893&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f06_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Placement
 accuracy for QS with artificially introduced random gaps. a) Average ND
 and b) NED (between insertion positions and real positions).</p></div></div><p>As
 expected, the general trend is that placement accuracy increases with 
the QS lengths. For all three QS subsets, the EPA achieves higher 
placement accuracies than SEQ-NN. Generally, the distances of the EPA 
placements are at least two times lower than for SEQ-NN. For SEQ-NN, the
 placement accuracy is considerably lower for the inner QS subset 
compared with the other QS subsets. Placement based on pair-wise 
sequence similarity is harder for inner QS than for the outer QS because
 the candidate QS do not have direct neighbors in the RT. This decrease 
of placement accuracy is independent of the QS lengths. For the EPA, the
 accuracy is more uniform across the three QS subsets. Only for short 
QS, there is a slight accuracy decrease for inner QS (this effect is 
more pronounced for the NED% measure). With increasing QS length, the 
EPA placements become almost equally accurate for the three QS subsets. 
It is worth noting that, on average, the EPA correctly places almost all
 QS from all three subsets, when they contain &lt; 50% gaps. The results
 suggest that there is a steady increase in accuracy for increased QS 
lengths up to the “perfect” placement on our test sets. This is 
particularly promising because read lengths will further increase.</p><p>The
 EPA placements on the inner QS compared with the outer QS are 
especially encouraging because this subset represents a worst-case 
scenario with respect to taxon sampling in the RT. In contrast to 
SEQ-NN, the original QS position has negligible impact on the placement 
accuracy. The results on this subset are indicative for the performance 
on data sets with a sparse or inadequate taxon sampling. Because it is 
hard to determine an adequate taxon sampling a priori for an unknown 
microbial community, our approach therefore can be used to appropriately
 adapt the taxon sampling.</p>                            <h3 scrollto-destination="115076896" id="115076896" class="section-title">Placement Accuracy for Randomly Selected Subsequences</h3>
<p><a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>
 provides the placement accuracy (according to the ND measure) for the 
uniformly sampled contiguous subsequences of normally distributed 
lengths (mean: 200±60 bp and 70±20 amino acid residues). The table 
values represent distances between the placement positions and the true 
positions from which the QS have been pruned. An ND of 0 represents a 
perfect placement, larger values indicate larger placement errors. As in
 the previous section, we show separate results for outer QS, inner QS, 
and all QS. In the second column, we show the placement accuracy of the 
EPA in terms of ND. For data set D140, this means that, averaged over 
all QS, the placements calculated by the EPA are within 0.51 nodes of 
the original placements. In the next column, we provide the average ND 
of the EPA for the case when the QS have been realigned to the RA using 
HMMER prior to the placement run. For D140, the average ND is 0.59, so 
the additional realignment step results in EPA placements that are on 
average 1.16 times further away from their original position than EPA 
placements without QS realignment. The third column gives the average ND
 for a BLAST-based approach. For D140, the value of 0.91 corresponds to 
placements that are on average 1.78 and 1.53 times further away from the
 true position than the EPA and EPA–RA placements. A corresponding table
 using the NED% measure is provided in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>. In general, the results are comparable with the ND results.</p><p>The
 values for D140 reflect the general trends that apply to most of the 
data sets. The EPA placements are on average more accurate than the 
BLAST placements, by factors between 1.12 and 2.06. Except for the two 
smallest (in terms of number of taxa) nucleotide data sets, the 
advantage of the EPA over alternative methods improves for the inner QS.
 For the smallest data sets, the number of inner QS (see <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>,
 remember that we only select well-supported leaves as candidate QS) is 
very small; thus, this variation may be attributed to random effects. 
For the outer QS, the advantage over BLAST is less pronounced. For two 
of the larger data sets (D1604 and D755), the absolute accuracy of the 
EPA is approximately equal for outer QS and inner QS, although there is a
 larger accuracy decrease for BLAST. The realignment using HMMER (see <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>
 columns EPA and EPA–RA) before placement by the EPA has only a small 
negative impact on placement accuracy. The realignment step decreases 
the accuracy of the EPA with respect to the two distance measures by 
factor 1.03–1.2. For one data set only (D628), the combination of EPA 
and HMMER was found to be slightly less accurate than placement with 
BLAST. We conclude that profile-HMMs as implemented in HMMER offer a 
useful method for the addition of short reads to a RA in this scenario, 
although there is still room to improve the QS alignment on certain data
 sets.</p>                        <a id="115076899" scrollto-destination="115076899"></a>
<div content-id="tbl1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label">T<span class="small-caps">ABLE</span> 1</div><div class="caption"><p>Data sets used for evaluation of the EPA</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>Type&nbsp;</td><td>Length&nbsp;</td><td>Number of taxa&nbsp;</td><td>Number of QS&nbsp;</td><td>Number of inner QS&nbsp;</td></tr></thead><tbody><tr><td>D140&nbsp;</td><td>AA&nbsp;</td><td char=".">1104&nbsp;</td><td char=".">140&nbsp;</td><td char=".">95&nbsp;</td><td char=".">9&nbsp;</td></tr><tr><td>D150&nbsp;</td><td>N&nbsp;</td><td char=".">1269&nbsp;</td><td char=".">150&nbsp;</td><td char=".">66&nbsp;</td><td char=".">10&nbsp;</td></tr><tr><td>D218&nbsp;</td><td>N&nbsp;</td><td char=".">2294&nbsp;</td><td char=".">218&nbsp;</td><td char=".">80&nbsp;</td><td char=".">14&nbsp;</td></tr><tr><td>D500&nbsp;</td><td>N&nbsp;</td><td char=".">1398&nbsp;</td><td char=".">500&nbsp;</td><td char=".">205&nbsp;</td><td char=".">29&nbsp;</td></tr><tr><td>D628&nbsp;</td><td>N&nbsp;</td><td char=".">1228&nbsp;</td><td char=".">628&nbsp;</td><td char=".">210&nbsp;</td><td char=".">20&nbsp;</td></tr><tr><td>D714&nbsp;</td><td>N&nbsp;</td><td char=".">1241&nbsp;</td><td char=".">714&nbsp;</td><td char=".">293&nbsp;</td><td char=".">61&nbsp;</td></tr><tr><td>D855&nbsp;</td><td>N&nbsp;</td><td char=".">1436&nbsp;</td><td char=".">855&nbsp;</td><td char=".">344&nbsp;</td><td char=".">48&nbsp;</td></tr><tr><td>D1604&nbsp;</td><td>N&nbsp;</td><td char=".">1276&nbsp;</td><td char=".">1604&nbsp;</td><td char=".">541&nbsp;</td><td char=".">83&nbsp;</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>Type&nbsp;</td><td>Length&nbsp;</td><td>Number of taxa&nbsp;</td><td>Number of QS&nbsp;</td><td>Number of inner QS&nbsp;</td></tr></thead><tbody><tr><td>D140&nbsp;</td><td>AA&nbsp;</td><td char=".">1104&nbsp;</td><td char=".">140&nbsp;</td><td char=".">95&nbsp;</td><td char=".">9&nbsp;</td></tr><tr><td>D150&nbsp;</td><td>N&nbsp;</td><td char=".">1269&nbsp;</td><td char=".">150&nbsp;</td><td char=".">66&nbsp;</td><td char=".">10&nbsp;</td></tr><tr><td>D218&nbsp;</td><td>N&nbsp;</td><td char=".">2294&nbsp;</td><td char=".">218&nbsp;</td><td char=".">80&nbsp;</td><td char=".">14&nbsp;</td></tr><tr><td>D500&nbsp;</td><td>N&nbsp;</td><td char=".">1398&nbsp;</td><td char=".">500&nbsp;</td><td char=".">205&nbsp;</td><td char=".">29&nbsp;</td></tr><tr><td>D628&nbsp;</td><td>N&nbsp;</td><td char=".">1228&nbsp;</td><td char=".">628&nbsp;</td><td char=".">210&nbsp;</td><td char=".">20&nbsp;</td></tr><tr><td>D714&nbsp;</td><td>N&nbsp;</td><td char=".">1241&nbsp;</td><td char=".">714&nbsp;</td><td char=".">293&nbsp;</td><td char=".">61&nbsp;</td></tr><tr><td>D855&nbsp;</td><td>N&nbsp;</td><td char=".">1436&nbsp;</td><td char=".">855&nbsp;</td><td char=".">344&nbsp;</td><td char=".">48&nbsp;</td></tr><tr><td>D1604&nbsp;</td><td>N&nbsp;</td><td char=".">1276&nbsp;</td><td char=".">1604&nbsp;</td><td char=".">541&nbsp;</td><td char=".">83&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note:
 The columns contain (from left to right): the name of the data; the 
type of the data (N: nucleotides and AA: amino acids); and the length of
 the data (i.e., number of sites in the alignment); for definitions of 
QS and inner QS, see the main text).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076899" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label">T<span class="small-caps">ABLE</span> 1</div><div class="caption"><p>Data sets used for evaluation of the EPA</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>Type&nbsp;</td><td>Length&nbsp;</td><td>Number of taxa&nbsp;</td><td>Number of QS&nbsp;</td><td>Number of inner QS&nbsp;</td></tr></thead><tbody><tr><td>D140&nbsp;</td><td>AA&nbsp;</td><td char=".">1104&nbsp;</td><td char=".">140&nbsp;</td><td char=".">95&nbsp;</td><td char=".">9&nbsp;</td></tr><tr><td>D150&nbsp;</td><td>N&nbsp;</td><td char=".">1269&nbsp;</td><td char=".">150&nbsp;</td><td char=".">66&nbsp;</td><td char=".">10&nbsp;</td></tr><tr><td>D218&nbsp;</td><td>N&nbsp;</td><td char=".">2294&nbsp;</td><td char=".">218&nbsp;</td><td char=".">80&nbsp;</td><td char=".">14&nbsp;</td></tr><tr><td>D500&nbsp;</td><td>N&nbsp;</td><td char=".">1398&nbsp;</td><td char=".">500&nbsp;</td><td char=".">205&nbsp;</td><td char=".">29&nbsp;</td></tr><tr><td>D628&nbsp;</td><td>N&nbsp;</td><td char=".">1228&nbsp;</td><td char=".">628&nbsp;</td><td char=".">210&nbsp;</td><td char=".">20&nbsp;</td></tr><tr><td>D714&nbsp;</td><td>N&nbsp;</td><td char=".">1241&nbsp;</td><td char=".">714&nbsp;</td><td char=".">293&nbsp;</td><td char=".">61&nbsp;</td></tr><tr><td>D855&nbsp;</td><td>N&nbsp;</td><td char=".">1436&nbsp;</td><td char=".">855&nbsp;</td><td char=".">344&nbsp;</td><td char=".">48&nbsp;</td></tr><tr><td>D1604&nbsp;</td><td>N&nbsp;</td><td char=".">1276&nbsp;</td><td char=".">1604&nbsp;</td><td char=".">541&nbsp;</td><td char=".">83&nbsp;</td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>Type&nbsp;</td><td>Length&nbsp;</td><td>Number of taxa&nbsp;</td><td>Number of QS&nbsp;</td><td>Number of inner QS&nbsp;</td></tr></thead><tbody><tr><td>D140&nbsp;</td><td>AA&nbsp;</td><td char=".">1104&nbsp;</td><td char=".">140&nbsp;</td><td char=".">95&nbsp;</td><td char=".">9&nbsp;</td></tr><tr><td>D150&nbsp;</td><td>N&nbsp;</td><td char=".">1269&nbsp;</td><td char=".">150&nbsp;</td><td char=".">66&nbsp;</td><td char=".">10&nbsp;</td></tr><tr><td>D218&nbsp;</td><td>N&nbsp;</td><td char=".">2294&nbsp;</td><td char=".">218&nbsp;</td><td char=".">80&nbsp;</td><td char=".">14&nbsp;</td></tr><tr><td>D500&nbsp;</td><td>N&nbsp;</td><td char=".">1398&nbsp;</td><td char=".">500&nbsp;</td><td char=".">205&nbsp;</td><td char=".">29&nbsp;</td></tr><tr><td>D628&nbsp;</td><td>N&nbsp;</td><td char=".">1228&nbsp;</td><td char=".">628&nbsp;</td><td char=".">210&nbsp;</td><td char=".">20&nbsp;</td></tr><tr><td>D714&nbsp;</td><td>N&nbsp;</td><td char=".">1241&nbsp;</td><td char=".">714&nbsp;</td><td char=".">293&nbsp;</td><td char=".">61&nbsp;</td></tr><tr><td>D855&nbsp;</td><td>N&nbsp;</td><td char=".">1436&nbsp;</td><td char=".">855&nbsp;</td><td char=".">344&nbsp;</td><td char=".">48&nbsp;</td></tr><tr><td>D1604&nbsp;</td><td>N&nbsp;</td><td char=".">1276&nbsp;</td><td char=".">1604&nbsp;</td><td char=".">541&nbsp;</td><td char=".">83&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Note:
 The columns contain (from left to right): the name of the data; the 
type of the data (N: nucleotides and AA: amino acids); and the length of
 the data (i.e., number of sites in the alignment); for definitions of 
QS and inner QS, see the main text).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076899" target="_blank">View Large</a></div></div>                        <a id="115076900" scrollto-destination="115076900"></a>
<div content-id="tbl2" class="table-modal" style="display: block;"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label">T<span class="small-caps">ABLE</span> 2</div><div class="caption"><p>Accuracy on randomly sampled short subsequences in terms of ND from the original position</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>EPA-RA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.49&nbsp;</td><td char=".">0.58&nbsp;</td><td char=".">0.82&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">1.2&nbsp;</td><td char=".">1.96&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.7&nbsp;</td><td char=".">2.01&nbsp;</td><td char=".">3.66&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">1.37&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">2.44&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">2.69&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.71&nbsp;</td><td char=".">1.82&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.87&nbsp;</td><td char=".">2.97&nbsp;</td><td char=".">3.53&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.26&nbsp;</td><td char=".">2.45&nbsp;</td><td char=".">2.87&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.74&nbsp;</td><td char=".">0.71&nbsp;</td><td char=".">1.81&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">3.09&nbsp;</td><td char=".">3.1&nbsp;</td><td char=".">4.62&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">2.24&nbsp;</td><td char=".">2.66&nbsp;</td><td char=".">3.81&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">2.05&nbsp;</td><td char=".">2.37&nbsp;</td><td char=".">4.16&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">3.28&nbsp;</td><td char=".">3.65&nbsp;</td><td char=".">4.04&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.78&nbsp;</td><td char=".">1.93&nbsp;</td><td char=".">3.51&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">3.68&nbsp;</td><td char=".">3.74&nbsp;</td><td char=".">4.91&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">2.38&nbsp;</td><td char=".">3.86&nbsp;</td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>EPA-RA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.49&nbsp;</td><td char=".">0.58&nbsp;</td><td char=".">0.82&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">1.2&nbsp;</td><td char=".">1.96&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.7&nbsp;</td><td char=".">2.01&nbsp;</td><td char=".">3.66&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">1.37&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">2.44&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">2.69&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.71&nbsp;</td><td char=".">1.82&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.87&nbsp;</td><td char=".">2.97&nbsp;</td><td char=".">3.53&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.26&nbsp;</td><td char=".">2.45&nbsp;</td><td char=".">2.87&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.74&nbsp;</td><td char=".">0.71&nbsp;</td><td char=".">1.81&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">3.09&nbsp;</td><td char=".">3.1&nbsp;</td><td char=".">4.62&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">2.24&nbsp;</td><td char=".">2.66&nbsp;</td><td char=".">3.81&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">2.05&nbsp;</td><td char=".">2.37&nbsp;</td><td char=".">4.16&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">3.28&nbsp;</td><td char=".">3.65&nbsp;</td><td char=".">4.04&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.78&nbsp;</td><td char=".">1.93&nbsp;</td><td char=".">3.51&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">3.68&nbsp;</td><td char=".">3.74&nbsp;</td><td char=".">4.91&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">2.38&nbsp;</td><td char=".">3.86&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Notes:
 The second column (EPA) shows the average ND of the EPA placements 
(using slow insertions under the GTR + Γ model) for the data sets in 
question. The third column (EPA-RA) shows the average ND for the EPA 
with previous realignment using HMMER. The last column (BLAST) shows the
 average ND for a BLAST-based approach.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076900" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label">T<span class="small-caps">ABLE</span> 2</div><div class="caption"><p>Accuracy on randomly sampled short subsequences in terms of ND from the original position</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>EPA-RA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.49&nbsp;</td><td char=".">0.58&nbsp;</td><td char=".">0.82&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">1.2&nbsp;</td><td char=".">1.96&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.7&nbsp;</td><td char=".">2.01&nbsp;</td><td char=".">3.66&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">1.37&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">2.44&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">2.69&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.71&nbsp;</td><td char=".">1.82&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.87&nbsp;</td><td char=".">2.97&nbsp;</td><td char=".">3.53&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.26&nbsp;</td><td char=".">2.45&nbsp;</td><td char=".">2.87&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.74&nbsp;</td><td char=".">0.71&nbsp;</td><td char=".">1.81&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">3.09&nbsp;</td><td char=".">3.1&nbsp;</td><td char=".">4.62&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">2.24&nbsp;</td><td char=".">2.66&nbsp;</td><td char=".">3.81&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">2.05&nbsp;</td><td char=".">2.37&nbsp;</td><td char=".">4.16&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">3.28&nbsp;</td><td char=".">3.65&nbsp;</td><td char=".">4.04&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.78&nbsp;</td><td char=".">1.93&nbsp;</td><td char=".">3.51&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">3.68&nbsp;</td><td char=".">3.74&nbsp;</td><td char=".">4.91&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">2.38&nbsp;</td><td char=".">3.86&nbsp;</td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>EPA-RA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.49&nbsp;</td><td char=".">0.58&nbsp;</td><td char=".">0.82&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">1.2&nbsp;</td><td char=".">1.96&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.7&nbsp;</td><td char=".">2.01&nbsp;</td><td char=".">3.66&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">1.37&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">2.44&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">2.69&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.71&nbsp;</td><td char=".">1.82&nbsp;</td><td char=".">2.36&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.87&nbsp;</td><td char=".">2.97&nbsp;</td><td char=".">3.53&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.26&nbsp;</td><td char=".">2.45&nbsp;</td><td char=".">2.87&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    D140&nbsp;</td><td char=".">0.74&nbsp;</td><td char=".">0.71&nbsp;</td><td char=".">1.81&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">3.09&nbsp;</td><td char=".">3.1&nbsp;</td><td char=".">4.62&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">2.24&nbsp;</td><td char=".">2.66&nbsp;</td><td char=".">3.81&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">2.05&nbsp;</td><td char=".">2.37&nbsp;</td><td char=".">4.16&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">3.28&nbsp;</td><td char=".">3.65&nbsp;</td><td char=".">4.04&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.78&nbsp;</td><td char=".">1.93&nbsp;</td><td char=".">3.51&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">3.68&nbsp;</td><td char=".">3.74&nbsp;</td><td char=".">4.91&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">2.38&nbsp;</td><td char=".">3.86&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Notes:
 The second column (EPA) shows the average ND of the EPA placements 
(using slow insertions under the GTR + Γ model) for the data sets in 
question. The third column (EPA-RA) shows the average ND for the EPA 
with previous realignment using HMMER. The last column (BLAST) shows the
 average ND for a BLAST-based approach.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076900" target="_blank">View Large</a></div></div><p>In <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a>,
 we show the placement precision of the EPA and BLAST in terms of 
percentage of QS placed within a certain ND of their true position. Up 
to an ND threshold of 10, the EPA with HMMER realignment (column EPA-RA)
 outperforms BLAST by 2.2–8%. As before, the EPA without realignment has
 slightly higher accuracy compared with EPA–RA. For the inner QS subset,
 the difference between BLAST and EPA–RA is larger, reaching 19.7% for 
an ND threshold of 1 (by definition, BLAST cannot correctly place inner 
QS, thus ND will be at least 1). <a reveal-id="bib5" data-open="bib5" class="link link-ref link-reveal xref-bibr">Brady and Salzberg (2009)</a>
 evaluated Phymm/PhymmBL in a similar experimental setup, by measuring 
the classification accuracy of Phymm, PhymmBL, and BLAST at different 
taxonomic levels. PhymmBL achieved an accuracy improvement of ∼6% for 
PhymmBL over BLAST on simulated QS.</p>                        <a id="115076902" scrollto-destination="115076902"></a>
<div content-id="tbl3" class="table-modal" style="display: block;"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label">T<span class="small-caps">ABLE</span> 3</div><div class="caption"><p>Percentage of QS placed correctly within a certain ND (max(ND)) of the original position over all data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Max(ND)&nbsp;</td><td>EPA (%)&nbsp;</td><td>EPA-RA (%)&nbsp;</td><td>BLAST (%)&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">58.4&nbsp;</td><td char=".">56.4&nbsp;</td><td char=".">56.3&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">71.1&nbsp;</td><td char=".">69.5&nbsp;</td><td char=".">64.1&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">77.6&nbsp;</td><td char=".">76.0&nbsp;</td><td char=".">69.5&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">87.9&nbsp;</td><td char=".">86.8&nbsp;</td><td char=".">82.6&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.6&nbsp;</td><td char=".">93.9&nbsp;</td><td char=".">91.9&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">37.0&nbsp;</td><td char=".">35.1&nbsp;</td><td char=".">0.0&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">64.3&nbsp;</td><td char=".">63.4&nbsp;</td><td char=".">43.7&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">74.7&nbsp;</td><td char=".">73.1&nbsp;</td><td char=".">56.9&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">86.5&nbsp;</td><td char=".">85.8&nbsp;</td><td char=".">77.9&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.1&nbsp;</td><td char=".">93.6&nbsp;</td><td char=".">90.5&nbsp;</td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><td>Max(ND)&nbsp;</td><td>EPA (%)&nbsp;</td><td>EPA-RA (%)&nbsp;</td><td>BLAST (%)&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">58.4&nbsp;</td><td char=".">56.4&nbsp;</td><td char=".">56.3&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">71.1&nbsp;</td><td char=".">69.5&nbsp;</td><td char=".">64.1&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">77.6&nbsp;</td><td char=".">76.0&nbsp;</td><td char=".">69.5&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">87.9&nbsp;</td><td char=".">86.8&nbsp;</td><td char=".">82.6&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.6&nbsp;</td><td char=".">93.9&nbsp;</td><td char=".">91.9&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">37.0&nbsp;</td><td char=".">35.1&nbsp;</td><td char=".">0.0&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">64.3&nbsp;</td><td char=".">63.4&nbsp;</td><td char=".">43.7&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">74.7&nbsp;</td><td char=".">73.1&nbsp;</td><td char=".">56.9&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">86.5&nbsp;</td><td char=".">85.8&nbsp;</td><td char=".">77.9&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.1&nbsp;</td><td char=".">93.6&nbsp;</td><td char=".">90.5&nbsp;</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076902" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label">T<span class="small-caps">ABLE</span> 3</div><div class="caption"><p>Percentage of QS placed correctly within a certain ND (max(ND)) of the original position over all data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>Max(ND)&nbsp;</td><td>EPA (%)&nbsp;</td><td>EPA-RA (%)&nbsp;</td><td>BLAST (%)&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">58.4&nbsp;</td><td char=".">56.4&nbsp;</td><td char=".">56.3&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">71.1&nbsp;</td><td char=".">69.5&nbsp;</td><td char=".">64.1&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">77.6&nbsp;</td><td char=".">76.0&nbsp;</td><td char=".">69.5&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">87.9&nbsp;</td><td char=".">86.8&nbsp;</td><td char=".">82.6&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.6&nbsp;</td><td char=".">93.9&nbsp;</td><td char=".">91.9&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">37.0&nbsp;</td><td char=".">35.1&nbsp;</td><td char=".">0.0&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">64.3&nbsp;</td><td char=".">63.4&nbsp;</td><td char=".">43.7&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">74.7&nbsp;</td><td char=".">73.1&nbsp;</td><td char=".">56.9&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">86.5&nbsp;</td><td char=".">85.8&nbsp;</td><td char=".">77.9&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.1&nbsp;</td><td char=".">93.6&nbsp;</td><td char=".">90.5&nbsp;</td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><td>Max(ND)&nbsp;</td><td>EPA (%)&nbsp;</td><td>EPA-RA (%)&nbsp;</td><td>BLAST (%)&nbsp;</td></tr></thead><tbody><tr><td colspan="4">Outer QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">58.4&nbsp;</td><td char=".">56.4&nbsp;</td><td char=".">56.3&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">71.1&nbsp;</td><td char=".">69.5&nbsp;</td><td char=".">64.1&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">77.6&nbsp;</td><td char=".">76.0&nbsp;</td><td char=".">69.5&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">87.9&nbsp;</td><td char=".">86.8&nbsp;</td><td char=".">82.6&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.6&nbsp;</td><td char=".">93.9&nbsp;</td><td char=".">91.9&nbsp;</td></tr><tr><td colspan="4">Inner QS&nbsp;</td></tr><tr><td>    0&nbsp;</td><td char=".">37.0&nbsp;</td><td char=".">35.1&nbsp;</td><td char=".">0.0&nbsp;</td></tr><tr><td>    1&nbsp;</td><td char=".">64.3&nbsp;</td><td char=".">63.4&nbsp;</td><td char=".">43.7&nbsp;</td></tr><tr><td>    2&nbsp;</td><td char=".">74.7&nbsp;</td><td char=".">73.1&nbsp;</td><td char=".">56.9&nbsp;</td></tr><tr><td>    5&nbsp;</td><td char=".">86.5&nbsp;</td><td char=".">85.8&nbsp;</td><td char=".">77.9&nbsp;</td></tr><tr><td>    10&nbsp;</td><td char=".">94.1&nbsp;</td><td char=".">93.6&nbsp;</td><td char=".">90.5&nbsp;</td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076902" target="_blank">View Large</a></div></div><p>As
 previously noted here, there can be position-specific effects that can 
influence the placement accuracy depending on which part of the gene is 
used to generate QS. To minimize the influence of this position-specific
 bias, we sampled multiple QS (at different gene regions) from every 
input sequence and report averages in this study. This broad sample of 
QS along the gene can also be used to plot the site-specific mean 
placement accuracy over the length of the RA. Accuracy plots for the 
data sets in this study are available in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>.
 This approach can also be used, a priori, on a full-length sequence 
alignment to determine appropriate gene regions for short-read 
generation.</p>                            <h3 scrollto-destination="115076904" id="115076904" class="section-title">Placement Accuracy for Paired-End Reads</h3>
<p><a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>
 provides the overall results of the experiments with virtual paired-end
 reads of length 2×100 bp (the results for 2×50 bp reads are provided in
 the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>).
 Similar to the previous section, the placement accuracies are given in 
terms of ND and NED%, averaged over all QS. For D150, the EPA places the
 QS within 1.26 nodes of their true position, whereas for BLAST, the 
average ND of 3.67 corresponds to placements that are 2.91 times further
 away from the original position relative to the EPA. The table also 
contains corresponding values for the NED% measure, which indicate 
similar results as the ND measure. As in the previous section, the 
general trend is similar for all data sets. The accuracy of the EPA 
placements are on average 1.58–5.87 times better than for BLAST.</p>                        <a id="115076906" scrollto-destination="115076906"></a>
<div content-id="tbl4" class="table-modal" style="display: block;"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label">T<span class="small-caps">ABLE</span> 4</div><div class="caption"><p>Accuracy of the placement of 2 × 100 bp paired-end reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>&nbsp;</td><td colspan="2">ND<hr>&nbsp;</td><td colspan="2">NED%<hr>&nbsp;</td></tr><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="5">Outer QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">3.38&nbsp;</td><td char=".">0.63&nbsp;</td><td char=".">1.76&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.44&nbsp;</td><td char=".">4.77&nbsp;</td><td char=".">3.6&nbsp;</td><td char=".">8.45&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">0.84&nbsp;</td><td char=".">4.88&nbsp;</td><td char=".">1.75&nbsp;</td><td char=".">7.35&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.56&nbsp;</td><td char=".">1.83&nbsp;</td><td char=".">0.83&nbsp;</td><td char=".">1.83&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">3.59&nbsp;</td><td char=".">1.77&nbsp;</td><td char=".">4.78&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.0&nbsp;</td><td char=".">5.98&nbsp;</td><td char=".">1.11&nbsp;</td><td char=".">3.35&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.58&nbsp;</td><td char=".">3.67&nbsp;</td><td char=".">0.79&nbsp;</td><td char=".">1.65&nbsp;</td></tr><tr><td colspan="5">Inner QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.9&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">5.64&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">4.43&nbsp;</td><td char=".">5.21&nbsp;</td><td char=".">8.16&nbsp;</td><td char=".">9.38&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.59&nbsp;</td><td char=".">9.41&nbsp;</td><td char=".">3.15&nbsp;</td><td char=".">12.24&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.85&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">0.55&nbsp;</td><td char=".">1.13&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.66&nbsp;</td><td char=".">4.9&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.76&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.54&nbsp;</td><td char=".">2.04&nbsp;</td><td char=".">4.61&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.57&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">1.69&nbsp;</td><td char=".">4.84&nbsp;</td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><td>&nbsp;</td><td colspan="2">ND<hr>&nbsp;</td><td colspan="2">NED%<hr>&nbsp;</td></tr><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="5">Outer QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">3.38&nbsp;</td><td char=".">0.63&nbsp;</td><td char=".">1.76&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.44&nbsp;</td><td char=".">4.77&nbsp;</td><td char=".">3.6&nbsp;</td><td char=".">8.45&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">0.84&nbsp;</td><td char=".">4.88&nbsp;</td><td char=".">1.75&nbsp;</td><td char=".">7.35&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.56&nbsp;</td><td char=".">1.83&nbsp;</td><td char=".">0.83&nbsp;</td><td char=".">1.83&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">3.59&nbsp;</td><td char=".">1.77&nbsp;</td><td char=".">4.78&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.0&nbsp;</td><td char=".">5.98&nbsp;</td><td char=".">1.11&nbsp;</td><td char=".">3.35&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.58&nbsp;</td><td char=".">3.67&nbsp;</td><td char=".">0.79&nbsp;</td><td char=".">1.65&nbsp;</td></tr><tr><td colspan="5">Inner QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.9&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">5.64&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">4.43&nbsp;</td><td char=".">5.21&nbsp;</td><td char=".">8.16&nbsp;</td><td char=".">9.38&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.59&nbsp;</td><td char=".">9.41&nbsp;</td><td char=".">3.15&nbsp;</td><td char=".">12.24&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.85&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">0.55&nbsp;</td><td char=".">1.13&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.66&nbsp;</td><td char=".">4.9&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.76&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.54&nbsp;</td><td char=".">2.04&nbsp;</td><td char=".">4.61&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.57&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">1.69&nbsp;</td><td char=".">4.84&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Notes:
 The values given are the ND and the normalized edge distance (NED %). 
The methods used are the EPA (slow insertions under the GTR + Γ model) 
and BLAST-based nearest neighbor.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076906" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label">T<span class="small-caps">ABLE</span> 4</div><div class="caption"><p>Accuracy of the placement of 2 × 100 bp paired-end reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><td>&nbsp;</td><td colspan="2">ND<hr>&nbsp;</td><td colspan="2">NED%<hr>&nbsp;</td></tr><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="5">Outer QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">3.38&nbsp;</td><td char=".">0.63&nbsp;</td><td char=".">1.76&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.44&nbsp;</td><td char=".">4.77&nbsp;</td><td char=".">3.6&nbsp;</td><td char=".">8.45&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">0.84&nbsp;</td><td char=".">4.88&nbsp;</td><td char=".">1.75&nbsp;</td><td char=".">7.35&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.56&nbsp;</td><td char=".">1.83&nbsp;</td><td char=".">0.83&nbsp;</td><td char=".">1.83&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">3.59&nbsp;</td><td char=".">1.77&nbsp;</td><td char=".">4.78&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.0&nbsp;</td><td char=".">5.98&nbsp;</td><td char=".">1.11&nbsp;</td><td char=".">3.35&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.58&nbsp;</td><td char=".">3.67&nbsp;</td><td char=".">0.79&nbsp;</td><td char=".">1.65&nbsp;</td></tr><tr><td colspan="5">Inner QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.9&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">5.64&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">4.43&nbsp;</td><td char=".">5.21&nbsp;</td><td char=".">8.16&nbsp;</td><td char=".">9.38&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.59&nbsp;</td><td char=".">9.41&nbsp;</td><td char=".">3.15&nbsp;</td><td char=".">12.24&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.85&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">0.55&nbsp;</td><td char=".">1.13&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.66&nbsp;</td><td char=".">4.9&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.76&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.54&nbsp;</td><td char=".">2.04&nbsp;</td><td char=".">4.61&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.57&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">1.69&nbsp;</td><td char=".">4.84&nbsp;</td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><td>&nbsp;</td><td colspan="2">ND<hr>&nbsp;</td><td colspan="2">NED%<hr>&nbsp;</td></tr><tr><td>Data&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td><td>EPA&nbsp;</td><td>BLAST&nbsp;</td></tr></thead><tbody><tr><td colspan="5">Outer QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.14&nbsp;</td><td char=".">3.38&nbsp;</td><td char=".">0.63&nbsp;</td><td char=".">1.76&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">1.44&nbsp;</td><td char=".">4.77&nbsp;</td><td char=".">3.6&nbsp;</td><td char=".">8.45&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">0.84&nbsp;</td><td char=".">4.88&nbsp;</td><td char=".">1.75&nbsp;</td><td char=".">7.35&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.56&nbsp;</td><td char=".">1.83&nbsp;</td><td char=".">0.83&nbsp;</td><td char=".">1.83&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.31&nbsp;</td><td char=".">3.59&nbsp;</td><td char=".">1.77&nbsp;</td><td char=".">4.78&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.0&nbsp;</td><td char=".">5.98&nbsp;</td><td char=".">1.11&nbsp;</td><td char=".">3.35&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.58&nbsp;</td><td char=".">3.67&nbsp;</td><td char=".">0.79&nbsp;</td><td char=".">1.65&nbsp;</td></tr><tr><td colspan="5">Inner QS&nbsp;</td></tr><tr><td>    D150&nbsp;</td><td char=".">1.9&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">2.23&nbsp;</td><td char=".">5.64&nbsp;</td></tr><tr><td>    D218&nbsp;</td><td char=".">4.43&nbsp;</td><td char=".">5.21&nbsp;</td><td char=".">8.16&nbsp;</td><td char=".">9.38&nbsp;</td></tr><tr><td>    D500&nbsp;</td><td char=".">1.59&nbsp;</td><td char=".">9.41&nbsp;</td><td char=".">3.15&nbsp;</td><td char=".">12.24&nbsp;</td></tr><tr><td>    D628&nbsp;</td><td char=".">0.85&nbsp;</td><td char=".">2.95&nbsp;</td><td char=".">0.55&nbsp;</td><td char=".">1.13&nbsp;</td></tr><tr><td>    D714&nbsp;</td><td char=".">1.66&nbsp;</td><td char=".">4.9&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.76&nbsp;</td></tr><tr><td>    D855&nbsp;</td><td char=".">2.9&nbsp;</td><td char=".">7.54&nbsp;</td><td char=".">2.04&nbsp;</td><td char=".">4.61&nbsp;</td></tr><tr><td>    D1604&nbsp;</td><td char=".">1.57&nbsp;</td><td char=".">5.3&nbsp;</td><td char=".">1.69&nbsp;</td><td char=".">4.84&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Notes:
 The values given are the ND and the normalized edge distance (NED %). 
The methods used are the EPA (slow insertions under the GTR + Γ model) 
and BLAST-based nearest neighbor.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/115076906" target="_blank">View Large</a></div></div><p><a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Figure 7</a>
 provides histograms for the accuracy distribution of individual 
placements computed by the EPA and BLAST for 2×100 bp paired-end reads 
on data set D855. Respective histograms for all data sets on 2×100 bp 
and 2×50 bp reads are available in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>.
 The plots suggest that the placement error for both methods follows an 
approximate power law distribution. The placements obtained via the EPA 
are, on average, closer to the true position and yield smaller maximum 
placement errors than BLAST.</p>                        <a id="115076908" scrollto-destination="115076908"></a>
<div id="fig7" data-id="fig7" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 7</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f07_ht.gif" alt="Histogram showing the placement accuracy, based on NDs, for the placement of 2 × 100 bp paired-end reads from D855, using outer a) and inner b) QS." path-from-xml="sysbiosyr010f07_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076908/sysbiosyr010f07_ht.gif" path-from-xml="sysbiosyr010f07_ht.gif" target="_blank">View large</a><a section="115076908" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f07_ht.gif?Expires=1561463706&amp;Signature=3gXyIvXeclAH8eLezAUHWBwgsEcfcoqmGHpVFsfmgXt9p8GidyYrTMoD-sZ5ggX0h9sK5s9ljwj8%7Eoi9AVylv9yEBYIq3M-jmtCtBn5Mq2IhBQ1JwD5E4TMA7kStF7ICpBm8A%7E1pu0kq043FCwrCqKUk%7Eq%7E3UNLUq80YF2I8HiulSLQn7n0PjC7j2yZ3UYxX8WFOGhpEoWhtWlu0IxbGxau9kqrYZLou2UyD56tbv68%7EAfPMT%7EwFcaheCbULQQINPyumX6PfSNCOGY2W9sMR6Jl5ETaFyuoQhJfn3QFD5Bi3m-1rKF3LswFx5mKY2oxZUm1Lg4L6uCyHon1DCUG%7EXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076908&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f07_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram
 showing the placement accuracy, based on NDs, for the placement of 2 × 
100 bp paired-end reads from D855, using outer a) and inner b) QS.</p></div></div><div content-id="fig7" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 7</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f07_ht.gif" alt="Histogram showing the placement accuracy, based on NDs, for the placement of 2 × 100 bp paired-end reads from D855, using outer a) and inner b) QS." path-from-xml="sysbiosyr010f07_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076908/sysbiosyr010f07_ht.gif" path-from-xml="sysbiosyr010f07_ht.gif" target="_blank">View large</a><a section="115076908" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f07_ht.gif?Expires=1561463706&amp;Signature=3gXyIvXeclAH8eLezAUHWBwgsEcfcoqmGHpVFsfmgXt9p8GidyYrTMoD-sZ5ggX0h9sK5s9ljwj8%7Eoi9AVylv9yEBYIq3M-jmtCtBn5Mq2IhBQ1JwD5E4TMA7kStF7ICpBm8A%7E1pu0kq043FCwrCqKUk%7Eq%7E3UNLUq80YF2I8HiulSLQn7n0PjC7j2yZ3UYxX8WFOGhpEoWhtWlu0IxbGxau9kqrYZLou2UyD56tbv68%7EAfPMT%7EwFcaheCbULQQINPyumX6PfSNCOGY2W9sMR6Jl5ETaFyuoQhJfn3QFD5Bi3m-1rKF3LswFx5mKY2oxZUm1Lg4L6uCyHon1DCUG%7EXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076908&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f07_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram
 showing the placement accuracy, based on NDs, for the placement of 2 × 
100 bp paired-end reads from D855, using outer a) and inner b) QS.</p></div></div><p><a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a>
 highlights that the EPA consistently outperforms BLAST-based placements
 for paired-end reads and that placements are approximately twice as 
accurate on average. Generally, the placement accuracy for paired-end 
reads of lengths 2×100 bp is better than was expected from the test with
 randomly selected contiguous subsequences of mean length 200 bp in the 
previous section. One contributing factor is that many of the 
subsequences in the previous experiment were significantly shorter than 
200 bp because we use normally distributed lengths. There also appears 
to exist a positive effect, generated by selecting subsequences from two
 distinct regions of the gene (the start and the end) from the original 
QS; in combination, phylogenetic information from both ends, may contain
 a stronger historical signal than a single, randomly selected 
subsequence.</p>                            <h3 scrollto-destination="115076910" id="115076910" class="section-title">Impact of Placement Algorithms and Substitution Models on Accuracy</h3>
<p>All preceding computational experiments were carried out using the 
most thorough (slow) version of the EPA under the generalized time 
reversible + Γ and Whelan and Goldman + Γ models. In this mode, the EPA 
optimizes edge lengths via the Newton–Raphson method for every possible 
insertion edge on the RT. As previously mentioned, we also devised a 
fast version of the EPA where this optimization is deactivated for QS 
insertions. These heuristics can speed up the EPA by one order of 
magnitude when a large amount of QS is being placed onto a RT. An 
additional speedup by a factor of 3–4 can be achieved by using the GTR +
 CAT or PROT + CAT approximations (<a reveal-id="bib37" data-open="bib37" class="link link-ref link-reveal xref-bibr">Stamatakis 2006</a>a) of rate heterogeneity.</p><p><a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Figure 8</a>
 shows the impact of EPA heuristics and rate heterogeneity models on 
placement accuracy for all QS over all data sets (analogous plots for 
the individual data sets are available in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>).
 For the slow insertion method, there is practically no difference in 
placement accuracy between the Γ model and the CAT approximation. For 
the fast insertion method, there is a notable decrease in placement 
accuracy for the CAT as well as the Γ models. The decrease is more 
pronounced for the outer QS, whereas for the inner QS, the effect of 
using the fast insertion method is less pronounced. As already 
mentioned, correct placement of the inner QS is harder than placement of
 the outer QS, which have direct neighbors in the RT. The results of 
this experiment show that the slow version of the EPA that includes edge
 length optimization can produce better placement results than the fast 
version, especially when QS are placed on inner edges of the RT.</p>                        <a id="115076913" scrollto-destination="115076913"></a>
<div id="fig8" data-id="fig8" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 8</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f08_lw.gif" alt="Average ND for different versions of the EPA (fast/slow insertions) algorithm and model types (GTR + Γ, GTR + CAT) on inner QS and outer QS from all data sets." path-from-xml="sysbiosyr010f08_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076913/sysbiosyr010f08_lw.gif" path-from-xml="sysbiosyr010f08_lw.gif" target="_blank">View large</a><a section="115076913" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f08_lw.gif?Expires=1561463706&amp;Signature=QneNANCfd1JV59sBnYs4ZJ1fieUxPS0hG%7EVWVf5peQOBiHApCDm-HVBfOLGGe6ysNx9k5XLkiTYDwU59gtiPRjw%7EnZLFL5xphfs1bfcpf1juT-uPprkKshanVfhW9zgIVpkKX%7E98MvCzCtJzQkWq9kcMUMLwig82cXVnfFhoBjvWa4xC-2jblmBP4WZSUXFsM3Cyk%7EQv7yyO2-aN4xtm1%7EaegomLgO6h%7E67zfViZV0X%7E2BZ7FsSOgxr6aj0tlSH6XIBzCXq2c9JMML-EeI0rOLAbbcjxcYLH4e5j5GKXmVG32mZ%7ENw8iV5%7EwFQtNiY8RhFTiH4DmRFRqP-hQvwY07Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076913&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f08_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Average
 ND for different versions of the EPA (fast/slow insertions) algorithm 
and model types (GTR + Γ, GTR + CAT) on inner QS and outer QS from all 
data sets.</p></div></div><div content-id="fig8" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 8</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f08_lw.gif" alt="Average ND for different versions of the EPA (fast/slow insertions) algorithm and model types (GTR + Γ, GTR + CAT) on inner QS and outer QS from all data sets." path-from-xml="sysbiosyr010f08_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076913/sysbiosyr010f08_lw.gif" path-from-xml="sysbiosyr010f08_lw.gif" target="_blank">View large</a><a section="115076913" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f08_lw.gif?Expires=1561463706&amp;Signature=QneNANCfd1JV59sBnYs4ZJ1fieUxPS0hG%7EVWVf5peQOBiHApCDm-HVBfOLGGe6ysNx9k5XLkiTYDwU59gtiPRjw%7EnZLFL5xphfs1bfcpf1juT-uPprkKshanVfhW9zgIVpkKX%7E98MvCzCtJzQkWq9kcMUMLwig82cXVnfFhoBjvWa4xC-2jblmBP4WZSUXFsM3Cyk%7EQv7yyO2-aN4xtm1%7EaegomLgO6h%7E67zfViZV0X%7E2BZ7FsSOgxr6aj0tlSH6XIBzCXq2c9JMML-EeI0rOLAbbcjxcYLH4e5j5GKXmVG32mZ%7ENw8iV5%7EwFQtNiY8RhFTiH4DmRFRqP-hQvwY07Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076913&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f08_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Average
 ND for different versions of the EPA (fast/slow insertions) algorithm 
and model types (GTR + Γ, GTR + CAT) on inner QS and outer QS from all 
data sets.</p></div></div>                            <h3 scrollto-destination="115076914" id="115076914" class="section-title">Heuristics for Slow Insertions</h3>
<p>As shown in the previous section, the loss of accuracy induced by the
 fast insertion method is minimal. Nonetheless, a slight accuracy 
improvement can be attained by the slow insertion method, especially 
regarding the more precise edge length estimate at the insertion 
position that can be used for postanalysis purposes. Using the rapid 
insertion edge prescoring heuristics already described, it is possible 
to accelerate the slow insertion algorithm with little to no impact on 
placement accuracy. Here, we evaluate the accuracy trade-offs associated
 with these heuristics. We also provide run-time measurements for the 
EPA and BLAST.</p><p>In contrast to the previous accuracy assessments, 
we do not test the placement of one QS at a time onto an existing RT 
from which the QS has been previously pruned. Instead, we randomly split
 the alignments into two subsets that each comprise 50% of the taxa. The
 first subset is used to infer a best-known ML tree with RAxML onto 
which the remaining taxa (of the second subset) are placed via the EPA. 
Here, we assume the slow EPA placements to be the true placements. In 
this experiment, we reduce the length of the QS to 50% nongap 
characters. The nongap characters are a contiguous sequence fragment 
that starts at the beginning of the respective sequence; that is, the QS
 represent roughly the first half of the gene.</p><p><a reveal-id="fig9" data-open="fig9" class="link link-reveal link-table xref-fig">Figure 9</a>
 shows the accuracy on the largest data set D1604 (placement of 802 QS 
onto a RT with 802 RS). The fraction of insertion edges considered for 
the slow insertion phase is controlled by the parameter <em>fh</em>. In the plot the accuracy of the heuristics for values of <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-1-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mi&gt;f&lt;/mi&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;4&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;8&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;16&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;32&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;64&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;128&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo&gt;,&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;256&lt;/mn&gt;&lt;/mfrac&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-1" style="width: 16.615em; display: inline-block;"><span style="display: inline-block; position: relative; width: 12.554em; height: 0px; font-size: 132%;"><span style="position: absolute; clip: rect(1.228em, 1012.55em, 2.841em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.06em;"></span></span><span class="mi" id="MathJax-Span-5" style="font-family: MathJax_Math; font-style: italic;">h</span><span class="mo" id="MathJax-Span-6" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-7" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 0.474em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-8" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.363em, 1000.33em, 4.167em, -1000em); top: -3.614em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-9" style="font-size: 70.7%; font-family: MathJax_Main;">4</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.47em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.474em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-10" style="font-family: MathJax_Main;">,</span><span class="mfrac" id="MathJax-Span-11" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.474em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-12" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.371em, 1000.32em, 4.182em, -1000em); top: -3.622em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-13" style="font-size: 70.7%; font-family: MathJax_Main;">8</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.47em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.474em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-14" style="font-family: MathJax_Main;">,</span><span class="mfrac" id="MathJax-Span-15" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.827em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-16" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.371em, 1000.68em, 4.182em, -1000em); top: -3.622em; left: 50%; margin-left: -0.354em;"><span class="mn" id="MathJax-Span-17" style="font-size: 70.7%; font-family: MathJax_Main;">16</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.83em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.827em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-18" style="font-family: MathJax_Main;">,</span><span class="mfrac" id="MathJax-Span-19" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.827em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-20" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.371em, 1000.67em, 4.182em, -1000em); top: -3.622em; left: 50%; margin-left: -0.354em;"><span class="mn" id="MathJax-Span-21" style="font-size: 70.7%; font-family: MathJax_Main;">32</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.83em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.827em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-22" style="font-family: MathJax_Main;">,</span><span class="mfrac" id="MathJax-Span-23" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 0.827em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-24" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.363em, 1000.69em, 4.182em, -1000em); top: -3.614em; left: 50%; margin-left: -0.354em;"><span class="mn" id="MathJax-Span-25" style="font-size: 70.7%; font-family: MathJax_Main;">64</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.83em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.827em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-26" style="font-family: MathJax_Main;">,</span><span class="mfrac" id="MathJax-Span-27" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.181em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-28" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.371em, 1001.03em, 4.182em, -1000em); top: -3.622em; left: 50%; margin-left: -0.53em;"><span class="mn" id="MathJax-Span-29" style="font-size: 70.7%; font-family: MathJax_Main;">128</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1001.18em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 1.181em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-30" style="font-family: MathJax_Main;">,</span><span class="mfrac" id="MathJax-Span-31" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.181em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-32" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.371em, 1001.03em, 4.182em, -1000em); top: -3.622em; left: 50%; margin-left: -0.53em;"><span class="mn" id="MathJax-Span-33" style="font-size: 70.7%; font-family: MathJax_Main;">256</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1001.18em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 1.181em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.607em; border-left: 0px solid; width: 0px; height: 1.843em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mi>h</mi><mo>=</mo><mfrac><mn>1</mn><mn>4</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>8</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>16</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>32</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>64</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>128</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>256</mn></mfrac></mrow></math></span></span><script type="math/mml" id="MathJax-Element-1"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mi>h</mi><mo>=</mo><mfrac><mn>1</mn><mn>4</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>8</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>16</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>32</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>64</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>128</mn></mfrac><mo>,</mo><mfrac><mn>1</mn><mn>256</mn></mfrac></mrow></math></script></span>
 is shown (i.e., on this data set ∼400, 200, 100, 50, 25, 12, 6 out of 
1601 possible insertion edges are considered). For the lowest <em>fh</em>
 values, there is a visible decrease in placement accuracy (sharp rise 
of the curves on the left side of the plot). For 50 or more insertion 
edges, the accuracy remains virtually constant. The results suggest 
that, on this data set, it is sufficient to more thoroughly analyze only
 50 out of 1601 <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-2-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mrow&gt;&lt;mo&gt;(&lt;/mo&gt;&lt;mi&gt;f&lt;/mi&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;/mrow&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;32&lt;/mn&gt;&lt;/mfrac&gt;&lt;mo stretchy=&quot;false&quot;&gt;)&lt;/mo&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-34" style="width: 5.685em; display: inline-block;"><span style="display: inline-block; position: relative; width: 4.275em; height: 0px; font-size: 132%;"><span style="position: absolute; clip: rect(1.228em, 1004.18em, 2.833em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-35"><span class="mrow" id="MathJax-Span-36"><span class="mrow" id="MathJax-Span-37"><span class="mo" id="MathJax-Span-38" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-39" style="font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.06em;"></span></span><span class="mi" id="MathJax-Span-40" style="font-family: MathJax_Math; font-style: italic;">h</span><span class="mo" id="MathJax-Span-41" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span></span><span class="mfrac" id="MathJax-Span-42" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 0.827em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-43" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.371em, 1000.67em, 4.182em, -1000em); top: -3.622em; left: 50%; margin-left: -0.354em;"><span class="mn" id="MathJax-Span-44" style="font-size: 70.7%; font-family: MathJax_Main;">32</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.83em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.827em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span><span class="mo" id="MathJax-Span-45" style="font-family: MathJax_Main;">)</span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.597em; border-left: 0px solid; width: 0px; height: 1.833em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mo>(</mo><mi>f</mi><mi>h</mi><mo>=</mo></mrow><mfrac><mn>1</mn><mn>32</mn></mfrac><mo stretchy="false">)</mo></mrow></math></span></span><script type="math/mml" id="MathJax-Element-2"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mo>(</mo><mi>f</mi><mi>h</mi><mo>=</mo></mrow><mfrac><mn>1</mn><mn>32</mn></mfrac><mo stretchy="false">)</mo></mrow></math></script></span> candidate insertion edges proposed by the heuristics to obtain the best possible accuracy (even for <span class="inline-formula no-formula-id"><span class="MathJax_Preview" style="color: inherit; display: none;"></span><span class="MathJax" id="MathJax-Element-3-Frame" tabindex="0" style="position: relative;" data-mathml="&lt;math xmlns=&quot;http://www.w3.org/1998/Math/MathML&quot;&gt;&lt;mrow xmlns=&quot;&quot;&gt;&lt;mi&gt;f&lt;/mi&gt;&lt;mi&gt;h&lt;/mi&gt;&lt;mo&gt;=&lt;/mo&gt;&lt;mfrac&gt;&lt;mn&gt;1&lt;/mn&gt;&lt;mn&gt;64&lt;/mn&gt;&lt;/mfrac&gt;&lt;/mrow&gt;&lt;/math&gt;" role="presentation"><nobr aria-hidden="true"><span class="math" id="MathJax-Span-46" style="width: 4.656em; display: inline-block;"><span style="display: inline-block; position: relative; width: 3.517em; height: 0px; font-size: 132%;"><span style="position: absolute; clip: rect(1.228em, 1003.52em, 2.841em, -1000em); top: -2.273em; left: 0em;"><span class="mrow" id="MathJax-Span-47"><span class="mrow" id="MathJax-Span-48"><span class="mi" id="MathJax-Span-49" style="font-family: MathJax_Math; font-style: italic;">f<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.06em;"></span></span><span class="mi" id="MathJax-Span-50" style="font-family: MathJax_Math; font-style: italic;">h</span><span class="mo" id="MathJax-Span-51" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mfrac" id="MathJax-Span-52" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 0.827em; height: 0px; margin-right: 0.12em; margin-left: 0.12em;"><span style="position: absolute; clip: rect(3.371em, 1000.3em, 4.167em, -1000em); top: -4.416em; left: 50%; margin-left: -0.177em;"><span class="mn" id="MathJax-Span-53" style="font-size: 70.7%; font-family: MathJax_Main;">1</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(3.363em, 1000.69em, 4.182em, -1000em); top: -3.614em; left: 50%; margin-left: -0.354em;"><span class="mn" id="MathJax-Span-54" style="font-size: 70.7%; font-family: MathJax_Main;">64</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; clip: rect(0.852em, 1000.83em, 1.245em, -1000em); top: -1.302em; left: 0em;"><span style="display: inline-block; overflow: hidden; vertical-align: 0em; border-top: 1.3px solid; width: 0.827em; height: 0px;"></span><span style="display: inline-block; width: 0px; height: 1.082em;"></span></span></span></span></span></span><span style="display: inline-block; width: 0px; height: 2.273em;"></span></span></span><span style="display: inline-block; overflow: hidden; vertical-align: -0.607em; border-left: 0px solid; width: 0px; height: 1.843em;"></span></span></nobr><span class="MJX_Assistive_MathML" role="presentation"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mi>h</mi><mo>=</mo><mfrac><mn>1</mn><mn>64</mn></mfrac></mrow></math></span></span><script type="math/mml" id="MathJax-Element-3"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>f</mi><mi>h</mi><mo>=</mo><mfrac><mn>1</mn><mn>64</mn></mfrac></mrow></math></script>⁠</span>,
 there is only a very small deviation from the reference placements). 
Another important result is that the MP heuristics produce equally 
accurate placements as the ML heuristics, for all, except the smallest 
values of <em>fh</em>.</p>                        <a id="115076918" scrollto-destination="115076918"></a>
<div id="fig9" data-id="fig9" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IGURE</span> 9</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f09_lw.gif" alt="Accuracy of the EPA as a function of the number of edges considered for slow insertion after heuristic filtering." path-from-xml="sysbiosyr010f09_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076918/sysbiosyr010f09_lw.gif" path-from-xml="sysbiosyr010f09_lw.gif" target="_blank">View large</a><a section="115076918" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f09_lw.gif?Expires=1561463706&amp;Signature=Vs8AR4653PEm0dUShCrZVvSHH9evjfEPKM1BkTU722LnANTfc0lvEnRsouKTuhBZkhi6zRhJ%7EbIMPQ4QDnLS9xnyxpw4JxJEwIwWO6zX8zGy7TCCirZcvug9slwUnYFZkE1DFbrB9qyLA1-2guewYGszbTjDhhVy3s3EDarZjE3CrVG5qawFUKv2AZ5CFxwXZ86YPmz8tZ6VB3pEtySHfzxKCaPBr2RbhV1hJzFFdaiIccFFwiw0NyaT4oAIC-GjrSOBVQJaIU%7EzJ8ZQQIHTGGvCvb0zoQz1ztNLUITWwEAkRBWz0HFhszrPuxGbVne%7E-z8F1FBSFDzw0vPApB965A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076918&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f09_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Accuracy of the EPA as a function of the number of edges considered for slow insertion after heuristic filtering.</p></div></div><div content-id="fig9" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IGURE</span> 9</div><div class="graphic-wrap"><img class="content-image" src="m_sysbiosyr010f09_lw.gif" alt="Accuracy of the EPA as a function of the number of edges considered for slow insertion after heuristic filtering." path-from-xml="sysbiosyr010f09_lw.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/115076918/sysbiosyr010f09_lw.gif" path-from-xml="sysbiosyr010f09_lw.gif" target="_blank">View large</a><a section="115076918" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/sysbiosyr010f09_lw.gif?Expires=1561463706&amp;Signature=Vs8AR4653PEm0dUShCrZVvSHH9evjfEPKM1BkTU722LnANTfc0lvEnRsouKTuhBZkhi6zRhJ%7EbIMPQ4QDnLS9xnyxpw4JxJEwIwWO6zX8zGy7TCCirZcvug9slwUnYFZkE1DFbrB9qyLA1-2guewYGszbTjDhhVy3s3EDarZjE3CrVG5qawFUKv2AZ5CFxwXZ86YPmz8tZ6VB3pEtySHfzxKCaPBr2RbhV1hJzFFdaiIccFFwiw0NyaT4oAIC-GjrSOBVQJaIU%7EzJ8ZQQIHTGGvCvb0zoQz1ztNLUITWwEAkRBWz0HFhszrPuxGbVne%7E-z8F1FBSFDzw0vPApB965A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=115076918&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5349" class="download-slide" path-from-xml="sysbiosyr010f09_lw.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Accuracy of the EPA as a function of the number of edges considered for slow insertion after heuristic filtering.</p></div></div><p>We conclude that the MP heuristics with a parameter setting of <em>f</em><em>h</em> = 1/32 (using the Γ model for <em>thorough</em>
 insertions) are sufficient for achieving placement accuracy comparable 
with the reference placement but with computational requirements (290 s)
 that are in the same order of magnitude as a simple BLAST search (216 
s) of the QS against the sequences in the RS. Realignment of the QS to 
the RA takes 224 s using HMMER on this data set. Thus, the combined run 
time of HMMER and the EPA is ∼2.5 times higher than a simple BLAST 
search but still within the same order of magnitude.</p><p>The lowest 
run time (113 s) was achieved by using the CAT model for slow insertions
 at the expense of a slight loss in accuracy (i.e., on average, the ND 
increased by ∼0.1). Based on the results in the previous section, we 
expect the accuracy difference between the CAT approximation and the Γ 
model of rate heterogeneity to be negligible in a real-world scenario.</p><p>The
 differences in accuracy between the fast and slow insertion methods as 
well as between the Γ and CAT models are generally larger than in the 
previous sections. This is not surprising, given that the setup of this 
experiment was not designed to measure the insertion accuracy relative 
to an assumed correct position, but the deviation between our best, yet 
slowest, method and less accurate accelerated methods. Here, we do not 
constrain the experiment to QS with high support values in the RT, but 
chose QS at random, which may introduce a certain loss of precision to 
this evaluation. In addition, the RT (comprising 50% of the taxa in the 
original RA) is smaller than in the previous evaluations and thus more 
sparsely sampled. Nonetheless, the deviation between the fast and slow 
EPA versions amounts to less than half a node on average and the general
 finding that slow insertions under CAT are more accurate than fast 
insertions under Γ is consistent with previous experiments.</p>                            <h2 scrollto-destination="115076922" id="115076922" class="section-title">EPA WEB SERVICE</h2>
<p>An alpha version of a Web server that offers the EPA algorithm is freely available at <a class="link link-uri" href="http://i12k-exelixis3.informatik.tu-muenchen.de/raxml" target="">http://i12k-exelixis3.informatik.tu-muenchen.de/raxml</a>
 and will be continuously developed and improved. The server runs on a 
dedicated machine with 24 AMD cores and 128GB of main memory.</p><p>Users
 can upload RTs and RAs and chose to align QS to the RA with HMMER or 
upload an alignment that already contains the QS. When the QS are to be 
aligned by the server, they can also be clustered using UCLUST (<a class="link link-uri" href="http://www.drive5.com/usearch/" target="">http://www.drive5.com/usearch/</a>)
 prior to alignment with HMMER. The UCLUST option can be used to reduce 
the number of reads that will subsequently be placed and aligned. 
Finally, the Web server also offers a JAVA-based result visualization 
tool that uses the Archeopteryx framework (<a reveal-id="bib16" data-open="bib16" class="link link-ref link-reveal xref-bibr">Han and Zmasek 2009</a>) and provides a simple visualization of the read distribution over the RT.</p>                            <h2 scrollto-destination="115076925" id="115076925" class="section-title">C<span class="small-caps">ONCLUSION</span></h2>
<p>We have presented an accurate and scalable approach for 
tree-dependent sequence comparison and compared its accuracy with 
sequence comparison-based methods. A tree-dependent approach has 
methodological advantages over standard, pair-wise, similarity-based 
comparative approaches and the EPA is freely available for download as 
open source code and as a Web service. We demonstrate that our approach 
may be substantially more accurate than standard techniques used to 
analyze, for example, microbial communities. More importantly, we 
demonstrate that achieving improved accuracy does not require longer 
inference times and that our approach is as fast as a simple BLAST-based
 search when using additional heuristics.</p><p>The EPA also is 
relatively straightforward to parallelize by applying a multigrain 
parallelization technique (Stamatakis et al. 2010). On a multicore 
system with 32 cores and 64 GB of main memory, we were able to place 
100,627 QS in parallel into a RT with 4874 taxa within 1.5 h. The 
application of the EPA is not limited to molecular data only. <a reveal-id="bib3" data-open="bib3" class="link link-ref link-reveal xref-bibr">Berger and Stamatakis (2010)</a> have used the EPA for the placement of fossil taxa onto a (molecular) RT of extant species.</p><p>A
 major challenge that remains to be solved consists in aligning the QS 
to a given RA. Throughout this paper, we have assumed that such an 
alignment was given or that such an alignment can conveniently be 
generated by aligning the individual QS to a profile-HMM of the RA. 
Ideally, one would like to simultaneously place and align the QS to the 
respective insertion edge. We are currently working on a tree-dependent 
alignment procedure for short read QS, which can be closely integrated 
with the placement process. Initial tests show that with tree-dependent 
realignment, it is possible to achieve higher EPA placement accuracy 
than with realignment using profile-HMMs. Another challenge consists of 
dealing with the matter of assigning short-read sequences to a RA when 
there is compositional heterogeneity across the sequences. This is a 
particularly important issue if sequences have acquired the same 
nucleotide or amino acid composition independently. A final challenge 
consists in developing appropriate visualization tools and metrics for 
analyzing distributions of reads on trees that have been computed by the
 EPA or pplacer programs.</p>                            <h2 scrollto-destination="115076929" id="115076929" class="section-title">S<span class="small-caps">UPPLEMENTARY</span> M<span class="small-caps">ATERIAL</span></h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span>.</p>                            <h2 scrollto-destination="115076931" id="115076931" class="section-title">F<span class="small-caps">UNDING</span></h2>
<p>This work was partially funded under the auspices of the Emmy–Noether
 program by the German Science Foundation (DFG). Grant numbers: DFG STA 
860/2 (S. A. Berger) and DFG STA 860/3 (D. Krompass).</p><p>The authors 
would like to thank Rob Knight, Steven Kembel, Micah Hamady, Christian 
von Mering, and Manuel Stark for useful discussions on algorithm design 
and for providing test data sets. We also thank F. Matsen for fruitful 
discussions on pplacer and EPA. We wish to acknowledge the constructive 
suggestions made by an anonymous reviewer.</p>                            <h2 scrollto-destination="115076935" id="115076935" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="bib1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ababneh</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Jermiin</div> <div class="given-names">LS</div></div>,&nbsp; <div class="name"><div class="surname">Ma</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Robinson</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Matched-pairs tests of homogeneity with applications to homologous nucleotide sequences</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">1225</div>-<div class="lpage">1231</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Matched-pairs%20tests%20of%20homogeneity%20with%20applications%20to%20homologous%20nucleotide%20sequences&amp;author=F%20Ababneh&amp;author=LS%20Jermiin&amp;author=C%20Ma&amp;author=J%20Robinson&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1225-1231" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl064" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl064" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16492684" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Madden</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Zhang</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Miller</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Lipman</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">25</div>&nbsp;pg.&nbsp;<div class="fpage">3389</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S%20Altschul&amp;author=T%20Madden&amp;author=A%20Schaffer&amp;author=J%20Zhang&amp;author=Z%20Zhang&amp;author=W%20Miller&amp;author=D%20Lipman&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berger</div> <div class="given-names">SA</div></div>,&nbsp; <div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div>,&nbsp;<div class="source">Accuracy
 of morphology-based phylogenetic fossil placement under maximum 
likelihood. Proceedings of IEEE/ACS International Conference on Computer
 Systems and Applications (AICCSA-10); 2010 May 16–18; Hammamet, 
Tunisia: IEEE Computer Society. p. 1–8</div>&nbsp;,&nbsp;<div class="year">2010</div></div></div></div></div><div content-id="bib4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bininda-Emonds</div> <div class="given-names">ORP</div></div>,&nbsp; <div class="name"><div class="surname">Brady</div> <div class="given-names">SG</div></div>,&nbsp; <div class="name"><div class="surname">Sanderson</div> <div class="given-names">MJ</div></div>,&nbsp; <div class="name"><div class="surname">Kim</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Altman</div> <div class="given-names">RB</div></div>,&nbsp; <div class="name"><div class="surname">Dunker</div> <div class="given-names">AK</div></div>,&nbsp; <div class="name"><div class="surname">Hunter</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Lauderdale</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Klein</div> <div class="given-names">TE</div></div>.&nbsp;</div><div class="article-title">Scaling of accuracy in extremely large phylogenetic trees</div>,&nbsp;<div class="source">Pacific Symposium on Biocomputing 2001</div>&nbsp;,&nbsp;<div class="year">2000</div><div class="publisher-loc">River Edge (NJ)</div><div class="publisher-name">World Scientific</div>(pg.&nbsp;<div class="fpage">547</div>-<div class="lpage">558</div>)</div></div></div></div><div content-id="bib5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brady</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Salzberg</div> <div class="given-names">SL</div></div>.&nbsp;</div><div class="article-title">Phymm and PhymmBL: metagenomic phylogenetic classification with interpolated Markov models</div>,&nbsp;<div class="source">Nat. Methods</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">673</div>-<div class="lpage">676</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phymm%20and%20PhymmBL%3A%20metagenomic%20phylogenetic%20classification%20with%20interpolated%20Markov%20models&amp;author=A%20Brady&amp;author=SL%20Salzberg&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=673-676" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1358" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1358" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.1358"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19648916" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chakravorty</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Helb</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Burday</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Connell</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Alland</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">A detailed analysis of 16S ribosomal RNA gene segments for the diagnosis of pathogenic bacteria</div>,&nbsp;<div class="source">J. Microbiol. Methods</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">69</div>&nbsp;(pg.&nbsp;<div class="fpage">330</div>-<div class="lpage">339</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20detailed%20analysis%20of%2016S%20ribosomal%20RNA%20gene%20segments%20for%20the%20diagnosis%20of%20pathogenic%20bacteria&amp;author=S%20Chakravorty&amp;author=D%20Helb&amp;author=M%20Burday&amp;author=N%20Connell&amp;author=D%20Alland&amp;publication_year=2007&amp;journal=J.%20Microbiol.%20Methods&amp;volume=69&amp;pages=330-339" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mimet.2007.02.005" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2007.02.005" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.mimet.2007.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17391789" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DeSantis</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Keller</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Brodie</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Larsen</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Piceno</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Phan</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Andersen</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">NAST: a multiple sequence alignment server for comparative analysis of 16S rRNA genes</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">34</div>&nbsp;(pg.&nbsp;<div class="fpage">W394</div>-<div class="lpage">W399</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NAST%3A%20a%20multiple%20sequence%20alignment%20server%20for%20comparative%20analysis%20of%2016S%20rRNA%20genes&amp;author=T%20DeSantis&amp;author=P%20Hugenholtz&amp;author=K%20Keller&amp;author=E%20Brodie&amp;author=N%20Larsen&amp;author=Y%20Piceno&amp;author=R%20Phan&amp;author=G%20Andersen&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W394-W399" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl244" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl244" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl244"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845035" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Profile hidden Markov models</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">755</div>-<div class="lpage">763</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Profile%20hidden%20Markov%20models&amp;author=S%20Eddy&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=755-763" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.9.755" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.9.755" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F14.9.755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918945" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>.&nbsp;</div><div class="article-title">Muscle: multiple sequence alignment with high accuracy and high throughput</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">32</div>&nbsp;(pg.&nbsp;<div class="fpage">1792</div>-<div class="lpage">1797</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Muscle%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1792-1797" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Evolutionary trees from DNA sequences: a maximum likelihood approach</div>,&nbsp;<div class="source">J. Mol. Evol.</div>&nbsp;,&nbsp;<div class="year">1981</div>,&nbsp;vol.&nbsp;<div class="volume">17</div>&nbsp;(pg.&nbsp;<div class="fpage">368</div>-<div class="lpage">376</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20trees%20from%20DNA%20sequences%3A%20a%20maximum%20likelihood%20approach&amp;author=J%20Felsenstein&amp;publication_year=1981&amp;journal=J.%20Mol.%20Evol.&amp;volume=17&amp;pages=368-376" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01734359" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01734359" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF01734359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7288891" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Confidence limits on phylogenies: an approach using the bootstrap</div>,&nbsp;<div class="source">Evolution</div>&nbsp;,&nbsp;<div class="year">1985</div>,&nbsp;vol.&nbsp;<div class="volume">39</div>&nbsp;(pg.&nbsp;<div class="fpage">783</div>-<div class="lpage">791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;author=J%20Felsenstein&amp;publication_year=1985&amp;journal=Evolution&amp;volume=39&amp;pages=783-791" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1558-5646.1985.tb00420.x" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1558-5646.1985.tb00420.x" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1558-5646.1985.tb00420.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28561359" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fierer</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Hamady</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Lauber</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">R</div></div>.&nbsp;</div>,&nbsp;<div class="source">The influence of sex, handedness, and washing on the diversity of hand surface bacteria. Proc. Natl. Acad. Sci. U.S.A</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">105</div>&nbsp;(pg.&nbsp;<div class="fpage">17994</div>-<div class="lpage">17999</div>)</div></div></div></div><div content-id="bib13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galtier</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Gouy</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Inferring
 pattern and process: maximum-likelihood implementation of a 
nonhomogeneous model of DNA sequence evolution for phylogenetic analysis</div>,&nbsp;<div class="source">Mol. Biol. Evol.</div>&nbsp;,&nbsp;<div class="year">1998</div>,&nbsp;vol.&nbsp;<div class="volume">15</div>&nbsp;(pg.&nbsp;<div class="fpage">871</div>-<div class="lpage">879</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20pattern%20and%20process%3A%20maximum-likelihood%20implementation%20of%20a%20nonhomogeneous%20model%20of%20DNA%20sequence%20evolution%20for%20phylogenetic%20analysis&amp;author=N%20Galtier&amp;author=M%20Gouy&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=15&amp;pages=871-879" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025991" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025991" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a025991"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9656487" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ganzert</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Jurgens</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Munster</div> <div class="given-names">U</div></div>,&nbsp; <div class="name"><div class="surname">Wagner</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">Methanogenic
 communities in permafrost-affected soils of the laptev sea coast, 
siberian arctic, characterized by 16s rRNA gene fingerprints</div>,&nbsp;<div class="source">FEMS Microbiol. Ecol</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">59</div>&nbsp;(pg.&nbsp;<div class="fpage">476</div>-<div class="lpage">488</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methanogenic%20communities%20in%20permafrost-affected%20soils%20of%20the%20laptev%20sea%20coast%2C%20siberian%20arctic%2C%20characterized%20by%2016s%20rRNA%20gene%20fingerprints&amp;author=L%20Ganzert&amp;author=G%20Jurgens&amp;author=U%20Munster&amp;author=D%20Wagner&amp;publication_year=2007&amp;journal=FEMS%20Microbiol.%20Ecol&amp;volume=59&amp;pages=476-488" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2007.59.issue-2" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2007.59.issue-2" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Ffem.2007.59.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16978241" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamming</div> <div class="given-names">RW</div></div>.&nbsp;</div><div class="article-title">Error detecting and error correcting codes. Bell Syst</div>,&nbsp;<div class="source">Tech. J.</div>&nbsp;,&nbsp;<div class="year">1950</div>,&nbsp;vol.&nbsp;<div class="volume">26</div>&nbsp;(pg.&nbsp;<div class="fpage">147</div>-<div class="lpage">160</div>)</div></div></div></div><div content-id="bib16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Zmasek</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">phyloXML: XML for evolutionary biology and comparative genomics</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2009</div>,&nbsp;vol.&nbsp;<div class="volume">10</div>&nbsp;pg.&nbsp;<div class="fpage">356</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=phyloXML%3A%20XML%20for%20evolutionary%20biology%20and%20comparative%20genomics&amp;author=M%20Han&amp;author=C%20Zmasek&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=356" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-356" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-356" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-10-356"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19860910" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hillis</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Bull</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">An empirical test of bootstrapping as a method for assessing confidence in phylogenetic analysis</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">42</div>&nbsp;pg.&nbsp;<div class="fpage">182</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20empirical%20test%20of%20bootstrapping%20as%20a%20method%20for%20assessing%20confidence%20in%20phylogenetic%20analysis&amp;author=D%20Hillis&amp;author=J%20Bull&amp;publication_year=1993&amp;journal=Syst.%20Biol.&amp;volume=42&amp;pages=182" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/42.2.182" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F42.2.182" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F42.2.182"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ho</div> <div class="given-names">JWK</div></div>,&nbsp; <div class="name"><div class="surname">Adams</div> <div class="given-names">CE</div></div>,&nbsp; <div class="name"><div class="surname">Lew</div> <div class="given-names">JB</div></div>,&nbsp; <div class="name"><div class="surname">Matthews</div> <div class="given-names">TJ</div></div>,&nbsp; <div class="name"><div class="surname">Ng</div> <div class="given-names">CC</div></div>,&nbsp; <div class="name"><div class="surname">Shahabi-Sirjani</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Tan</div> <div class="given-names">LH</div></div>,&nbsp; <div class="name"><div class="surname">Zhao</div> <div class="given-names">Y</div></div>,&nbsp; <div class="name"><div class="surname">Easteal</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Wilson</div> <div class="given-names">SR</div></div>,&nbsp; <div class="name"><div class="surname">Jermiin</div> <div class="given-names">LS</div></div>.&nbsp;</div><div class="article-title">SeqVis: visualization of compositional heterogeneity in large alignments of nucleotides</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">2162</div>-<div class="lpage">2163</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SeqVis%3A%20visualization%20of%20compositional%20heterogeneity%20in%20large%20alignments%20of%20nucleotides&amp;author=JWK%20Ho&amp;author=CE%20Adams&amp;author=JB%20Lew&amp;author=TJ%20Matthews&amp;author=CC%20Ng&amp;author=A%20Shahabi-Sirjani&amp;author=LH%20Tan&amp;author=Y%20Zhao&amp;author=S%20Easteal&amp;author=SR%20Wilson&amp;author=LS%20Jermiin&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2162-2163" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl283" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl283" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16766557" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">D</div></div>,&nbsp; <div class="name"><div class="surname">Auch</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Qi</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Schuster</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">MEGAN analysis of metagenomic data</div>,&nbsp;<div class="source">Genome Res.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">17</div>&nbsp;(pg.&nbsp;<div class="fpage">377</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGAN%20analysis%20of%20metagenomic%20data&amp;author=D%20Hudson&amp;author=A%20Auch&amp;author=J%20Qi&amp;author=S%20Schuster&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=377-386" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5969107" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5969107" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.5969107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17255551" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jayaswal</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Robinson</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Jermiin</div> <div class="given-names">LS</div></div>.&nbsp;</div><div class="article-title">Estimation of phylogeny and invariant sites under the general Markov model of nucleotide sequence evolution</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">56</div>&nbsp;(pg.&nbsp;<div class="fpage">155</div>-<div class="lpage">162</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20phylogeny%20and%20invariant%20sites%20under%20the%20general%20Markov%20model%20of%20nucleotide%20sequence%20evolution&amp;author=V%20Jayaswal&amp;author=J%20Robinson&amp;author=LS%20Jermiin&amp;publication_year=2007&amp;journal=Syst.%20Biol.&amp;volume=56&amp;pages=155-162" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150701247921" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150701247921" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150701247921"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17454972" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jermiin</div> <div class="given-names">LS</div></div>,&nbsp; <div class="name"><div class="surname">Ho</div> <div class="given-names">SYW</div></div>,&nbsp; <div class="name"><div class="surname">Ababneh</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Robinson</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Larkum</div> <div class="given-names">AWD</div></div>.&nbsp;</div><div class="article-title">The biasing effect of compositional heterogeneity on phylogenetic estimates may be underestimated</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">53</div>&nbsp;(pg.&nbsp;<div class="fpage">638</div>-<div class="lpage">643</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biasing%20effect%20of%20compositional%20heterogeneity%20on%20phylogenetic%20estimates%20may%20be%20underestimated&amp;author=LS%20Jermiin&amp;author=SYW%20Ho&amp;author=F%20Ababneh&amp;author=J%20Robinson&amp;author=AWD%20Larkum&amp;publication_year=2004&amp;journal=Syst.%20Biol.&amp;volume=53&amp;pages=638-643" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150490468648" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150490468648" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150490468648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15371251" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katoh</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Kuma</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Toh</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Miyata</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="article-title">MAFFT version 5: improvement in accuracy of multiple sequence alignment</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">33</div>&nbsp;(pg.&nbsp;<div class="fpage">511</div>-<div class="lpage">518</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%20version%205%3A%20improvement%20in%20accuracy%20of%20multiple%20sequence%20alignment&amp;author=K%20Katoh&amp;author=K%20Kuma&amp;author=H%20Toh&amp;author=T%20Miyata&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=511-518" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki198" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki198" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgki198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15661851" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kluge</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Farris</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Quantitative phyletics and the evolution of anurans</div>,&nbsp;<div class="source">Syst. Zool</div>&nbsp;,&nbsp;<div class="year">1969</div>,&nbsp;vol.&nbsp;<div class="volume">18</div>&nbsp;(pg.&nbsp;<div class="fpage">1</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20phyletics%20and%20the%20evolution%20of%20anurans&amp;author=A%20Kluge&amp;author=J%20Farris&amp;publication_year=1969&amp;journal=Syst.%20Zool&amp;volume=18&amp;pages=1-32" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2412407" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2412407" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F2412407"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koski</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Golding</div> <div class="given-names">G</div></div>.&nbsp;</div><div class="article-title">The closest BLAST hit is often not the nearest neighbor</div>,&nbsp;<div class="source">J. Mol. Evol.</div>&nbsp;,&nbsp;<div class="year">2001</div>,&nbsp;vol.&nbsp;<div class="volume">52</div>&nbsp;(pg.&nbsp;<div class="fpage">540</div>-<div class="lpage">542</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20closest%20BLAST%20hit%20is%20often%20not%20the%20nearest%20neighbor&amp;author=L%20Koski&amp;author=G%20Golding&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol.&amp;volume=52&amp;pages=540-542" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010184" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010184" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2Fs002390010184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11443357" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">RE</div></div>,&nbsp; <div class="name"><div class="surname">Bäckhed</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Lozupone</div> <div class="given-names">CA</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">RD</div></div>,&nbsp; <div class="name"><div class="surname">Gordon</div> <div class="given-names">JI</div></div>.&nbsp;</div><div class="article-title">Obesity alters gut microbial ecology</div>,&nbsp;<div class="source">Proc. Natl. Acad. Sci. U.S.A</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">102</div>&nbsp;(pg.&nbsp;<div class="fpage">11070</div>-<div class="lpage">11075</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Obesity%20alters%20gut%20microbial%20ecology&amp;author=RE%20Ley&amp;author=F%20B%C3%A4ckhed&amp;author=P%20Turnbaugh&amp;author=CA%20Lozupone&amp;author=RD%20Knight&amp;author=JI%20Gordon&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=102&amp;pages=11070-11075" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0504978102" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0504978102" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.0504978102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16033867" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib26" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">RE</div></div>,&nbsp; <div class="name"><div class="surname">Harris</div> <div class="given-names">JK</div></div>,&nbsp; <div class="name"><div class="surname">Wilcox</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Spear</div> <div class="given-names">JR</div></div>,&nbsp; <div class="name"><div class="surname">Miller</div> <div class="given-names">SR</div></div>,&nbsp; <div class="name"><div class="surname">Bebout</div> <div class="given-names">BM</div></div>,&nbsp; <div class="name"><div class="surname">Maresca</div> <div class="given-names">JA</div></div>,&nbsp; <div class="name"><div class="surname">Bryant</div> <div class="given-names">DA</div></div>,&nbsp; <div class="name"><div class="surname">Sogin</div> <div class="given-names">ML</div></div>,&nbsp; <div class="name"><div class="surname">Pace</div> <div class="given-names">NR</div></div>.&nbsp;</div><div class="article-title">Unexpected diversity and complexity of the guerrero negro hypersaline microbial mat</div>,&nbsp;<div class="source">Appl. Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">72</div>&nbsp;(pg.&nbsp;<div class="fpage">3685</div>-<div class="lpage">3695</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unexpected%20diversity%20and%20complexity%20of%20the%20guerrero%20negro%20hypersaline%20microbial%20mat&amp;author=RE%20Ley&amp;author=JK%20Harris&amp;author=J%20Wilcox&amp;author=JR%20Spear&amp;author=SR%20Miller&amp;author=BM%20Bebout&amp;author=JA%20Maresca&amp;author=DA%20Bryant&amp;author=ML%20Sogin&amp;author=NR%20Pace&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=3685-3695" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.72.5.3685-3695.2006" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.72.5.3685-3695.2006" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.72.5.3685-3695.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16672518" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib27" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">RE</div></div>,&nbsp; <div class="name"><div class="surname">Lozupone</div> <div class="given-names">CA</div></div>,&nbsp; <div class="name"><div class="surname">Hamady</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">RD</div></div>,&nbsp; <div class="name"><div class="surname">Gordon</div> <div class="given-names">JI</div></div>.&nbsp;</div><div class="article-title">Worlds within worlds: evolution of the vertebrate gut microbiota</div>,&nbsp;<div class="source">Nat. Rev. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">6</div>&nbsp;(pg.&nbsp;<div class="fpage">776</div>-<div class="lpage">788</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worlds%20within%20worlds%3A%20evolution%20of%20the%20vertebrate%20gut%20microbiota&amp;author=RE%20Ley&amp;author=CA%20Lozupone&amp;author=M%20Hamady&amp;author=RD%20Knight&amp;author=JI%20Gordon&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=6&amp;pages=776-788" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1978" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1978" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnrmicro1978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18794915" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib28" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lozupone</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">UniFrac: a new phylogenetic method for comparing microbial communities</div>,&nbsp;<div class="source">Appl. Environ. Microbiol.</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">71</div>&nbsp;(pg.&nbsp;<div class="fpage">8228</div>-<div class="lpage">8235</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniFrac%3A%20a%20new%20phylogenetic%20method%20for%20comparing%20microbial%20communities&amp;author=C%20Lozupone&amp;author=R%20Knight&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=8228-8235" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.12.8228-8235.2005" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.12.8228-8235.2005" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1128%2FAEM.71.12.8228-8235.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16332807" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib29" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ludwig</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Strunk</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Westram</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Richter</div> <div class="given-names">L</div></div>,&nbsp; <div class="name"><div class="surname">Meier</div> <div class="given-names">H</div></div>,&nbsp; <div class="name"><div class="surname">Yadhukumar</div> <div class="given-names">BA</div></div>,&nbsp; <div class="name"><div class="surname">Lai</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Steppi</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Jobb</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Forster</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Brettske</div> <div class="given-names">I</div></div>,&nbsp; <div class="name"><div class="surname">Gerber</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Ginhart</div> <div class="given-names">AW</div></div>,&nbsp; <div class="name"><div class="surname">Gross</div> <div class="given-names">O</div></div>,&nbsp; <div class="name"><div class="surname">Grumann</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Hermann</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Jost</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Konig</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Liss</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Lussmann</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">May</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Nonhoff</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Reichel</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Strehlow</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Stuckmann</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Vilbig</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Lenke</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Ludwig</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Bode</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Schleifer</div> <div class="given-names">KH</div></div>.&nbsp;</div><div class="article-title">Arb: a software environment for sequence data</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">32</div>&nbsp;(pg.&nbsp;<div class="fpage">1363</div>-<div class="lpage">1371</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arb%3A%20a%20software%20environment%20for%20sequence%20data&amp;author=W%20Ludwig&amp;author=O%20Strunk&amp;author=R%20Westram&amp;author=L%20Richter&amp;author=H%20Meier&amp;author=BA%20Yadhukumar&amp;author=T%20Lai&amp;author=S%20Steppi&amp;author=G%20Jobb&amp;author=W%20Forster&amp;author=I%20Brettske&amp;author=S%20Gerber&amp;author=AW%20Ginhart&amp;author=O%20Gross&amp;author=S%20Grumann&amp;author=S%20Hermann&amp;author=R%20Jost&amp;author=A%20Konig&amp;author=T%20Liss&amp;author=R%20Lussmann&amp;author=M%20May&amp;author=B%20Nonhoff&amp;author=B%20Reichel&amp;author=R%20Strehlow&amp;author=A%20Stamatakis&amp;author=N%20Stuckmann&amp;author=A%20Vilbig&amp;author=M%20Lenke&amp;author=T%20Ludwig&amp;author=A%20Bode&amp;author=KH%20Schleifer&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1363-1371" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh293" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh293" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkh293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14985472" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib30" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsen</div> <div class="given-names">F</div></div>,&nbsp; <div class="name"><div class="surname">Kodner</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Armbrust</div> <div class="given-names">EV</div></div>.&nbsp;</div><div class="article-title">Pplacer: linear time maximum-likelihood and Bayesian phylogenetic placement of sequences onto a fixed reference tree</div>,&nbsp;<div class="source">BMC Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2010</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;pg.&nbsp;<div class="fpage">538</div>&nbsp;<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pplacer%3A%20linear%20time%20maximum-likelihood%20and%20Bayesian%20phylogenetic%20placement%20of%20sequences%20onto%20a%20fixed%20reference%20tree&amp;author=F%20Matsen&amp;author=R%20Kodner&amp;author=EV%20Armbrust&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=538" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-538" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-538" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2105-11-538"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21034504" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib31" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McHardy</div> <div class="given-names">AC</div></div>,&nbsp; <div class="name"><div class="surname">Martin</div> <div class="given-names">HG</div></div>,&nbsp; <div class="name"><div class="surname">Aristotelis</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Rigoutsos</div> <div class="given-names">I</div></div>.&nbsp;</div><div class="article-title">Accurate phylogenetic classification of variable-length DNA fragments</div>,&nbsp;<div class="source">Nat. Methods</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">4</div>&nbsp;(pg.&nbsp;<div class="fpage">63</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20phylogenetic%20classification%20of%20variable-length%20DNA%20fragments&amp;author=AC%20McHardy&amp;author=HG%20Martin&amp;author=T%20Aristotelis&amp;author=P%20Hugenholtz&amp;author=I%20Rigoutsos&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=63-72" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth976" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth976" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17179938" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib32" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moret</div> <div class="given-names">BME</div></div>,&nbsp; <div class="name"><div class="surname">Roshan</div> <div class="given-names">U</div></div>,&nbsp; <div class="name"><div class="surname">Warnow</div> <div class="given-names">T</div></div>.&nbsp;</div><div class="person-group"><div class="name"><div class="surname">Guigo</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Gusfield</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">Sequence-length requirements for phylogenetic methods</div>,&nbsp;<div class="source">Proceedings of Second International Workshop on Algorithms in Bioinformatics (WABI 02) 2002</div>&nbsp;,&nbsp;<div class="year">2002</div><div class="publisher-loc">Berlin</div><div class="publisher-name">Springer</div>(pg.&nbsp;<div class="fpage">343</div>-<div class="lpage">356</div>)&nbsp;<div class="comment">(Lecture Notes in Computer Science; 2452)</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-length%20requirements%20for%20phylogenetic%20methods&amp;author=BME%20Moret&amp;author=U%20Roshan&amp;author=T%20Warnow&amp;author=R%20Guigo&amp;author=D%20Gusfield&amp;publication_year=2002&amp;book=Proceedings%20of%20Second%20International%20Workshop%20on%20Algorithms%20in%20Bioinformatics%20%28WABI%2002%29%202002" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/3-540-45784-4" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F3-540-45784-4" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2F3-540-45784-4"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib33" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Munch</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Boomsma</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Huelsenbeck</div> <div class="given-names">JP</div></div>,&nbsp; <div class="name"><div class="surname">Willerslev</div> <div class="given-names">E</div></div>,&nbsp; <div class="name"><div class="surname">Nielsen</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">Statistical assignment of DNA sequences using Bayesian phylogenetics</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">57</div>&nbsp;(pg.&nbsp;<div class="fpage">750</div>-<div class="lpage">757</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20assignment%20of%20DNA%20sequences%20using%20Bayesian%20phylogenetics&amp;author=K%20Munch&amp;author=W%20Boomsma&amp;author=JP%20Huelsenbeck&amp;author=E%20Willerslev&amp;author=R%20Nielsen&amp;publication_year=2008&amp;journal=Syst.%20Biol.&amp;volume=57&amp;pages=750-757" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150802422316" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150802422316" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150802422316"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18853361" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib34" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielsen</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Matz</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Statistical approaches for DNA barcoding</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">55</div>&nbsp;(pg.&nbsp;<div class="fpage">162</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20approaches%20for%20DNA%20barcoding&amp;author=R%20Nielsen&amp;author=M%20Matz&amp;publication_year=2006&amp;journal=Syst.%20Biol.&amp;volume=55&amp;pages=162-169" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150500431239" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150500431239" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150500431239"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16507534" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib35" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Maglott</div> <div class="given-names">D</div></div>.&nbsp;</div><div class="article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>,&nbsp;<div class="source">Nucleic Acids Res.</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">35</div>&nbsp;<div class="issue">Database issue</div>(pg.&nbsp;<div class="fpage">D61</div>-<div class="lpage">D65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%20%28RefSeq%29%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K%20Pruitt&amp;author=T%20Tatusova&amp;author=D%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D61-D65" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkl842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130148" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib36" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronaghi</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Pyrosequencing sheds light on DNA sequencing</div>,&nbsp;<div class="source">Genome Res.</div>&nbsp;,&nbsp;<div class="year">2001</div>,&nbsp;vol.&nbsp;<div class="volume">11</div>&nbsp;(pg.&nbsp;<div class="fpage">3</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pyrosequencing%20sheds%20light%20on%20DNA%20sequencing&amp;author=M%20Ronaghi&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=3-11" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.11.1.3" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.11.1.3" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.11.1.3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11156611" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib37" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div>,&nbsp;<div class="source">Phylogenetic
 models of rate heterogeneity: a high performance computing perspective.
 Proceedings of 20th IEEE/ACM International Parallel and Distributed 
Processing Symposium (IPDPS2006); 2006 Apr 25–29; Rhodes Island, Greece.
 IEEE Computer Society. p. 278</div>&nbsp;,&nbsp;<div class="year">2006</div></div></div></div></div><div content-id="bib38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib38" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa and mixed models</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">22</div>&nbsp;(pg.&nbsp;<div class="fpage">2688</div>-<div class="lpage">2690</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAxML-VI-HPC%3A%20maximum%20likelihood-based%20phylogenetic%20analyses%20with%20thousands%20of%20taxa%20and%20mixed%20models&amp;author=A%20Stamatakis&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=2688-2690" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl446" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl446" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtl446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928733" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib39" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Hoover</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Rougemont</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">A rapid bootstrap algorithm for the RAxML Web servers</div>,&nbsp;<div class="source">Syst. Biol.</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">57</div>&nbsp;(pg.&nbsp;<div class="fpage">758</div>-<div class="lpage">771</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20bootstrap%20algorithm%20for%20the%20RAxML%20Web%20servers&amp;author=A%20Stamatakis&amp;author=P%20Hoover&amp;author=J%20Rougemont&amp;publication_year=2008&amp;journal=Syst.%20Biol.&amp;volume=57&amp;pages=758-771" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150802429642" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150802429642" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150802429642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18853362" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib40" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Komornik</div> <div class="given-names">Z</div></div>,&nbsp; <div class="name"><div class="surname">Berger</div> <div class="given-names">SA</div></div>.&nbsp;</div>,&nbsp;<div class="source">Evolutionary
 placement of short sequence reads on multi-core architectures. 
Proceedings of IEEE/ACS International Conference on Computer Systems and
 Applications (AICCSA-10); 2010 May 16–18; Hammamet, Tunisia. IEEE 
Computer Society. p. 1–8</div>&nbsp;,&nbsp;<div class="year">2010</div></div></div></div></div><div content-id="bib41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib41" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Ludwig</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Meier</div> <div class="given-names">H</div></div>.&nbsp;</div><div class="article-title">RAxML-III: a fast program for maximum likelihood-based inference of large phylogenetic trees</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">21</div>&nbsp;<div class="issue">4</div>(pg.&nbsp;<div class="fpage">456</div>-<div class="lpage">463</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAxML-III%3A%20a%20fast%20program%20for%20maximum%20likelihood-based%20inference%20of%20large%20phylogenetic%20trees&amp;author=A%20Stamatakis&amp;author=T%20Ludwig&amp;author=H%20Meier&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=456-463" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti191" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti191" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbti191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608047" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib42" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strimmer</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Rambaut</div> <div class="given-names">A</div></div>.&nbsp;</div><div class="article-title">Inferring confidence sets of possibly misspecified gene trees</div>,&nbsp;<div class="source">Proc. R. Soc. B Biol. Sci.</div>&nbsp;,&nbsp;<div class="year">2002</div>,&nbsp;vol.&nbsp;<div class="volume">269</div>&nbsp;(pg.&nbsp;<div class="fpage">137</div>-<div class="lpage">142</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20confidence%20sets%20of%20possibly%20misspecified%20gene%20trees&amp;author=K%20Strimmer&amp;author=A%20Rambaut&amp;publication_year=2002&amp;journal=Proc.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;volume=269&amp;pages=137-142" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2001.1862" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2001.1862" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frspb.2001.1862"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib43" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Hamady</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Yatsunenko</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Cantarel</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Duncan</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Ley</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Sogin</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Jones</div> <div class="given-names">W</div></div>,&nbsp; <div class="name"><div class="surname">Roe</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Affourtit</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Egholm</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Henrissat</div> <div class="given-names">B</div></div>,&nbsp; <div class="name"><div class="surname">Heath</div> <div class="given-names">AC</div></div>,&nbsp; <div class="name"><div class="surname">Knight</div> <div class="given-names">R</div></div>,&nbsp; <div class="name"><div class="surname">Gordon</div> <div class="given-names">JI</div></div>.&nbsp;</div><div class="article-title">A core gut microbiome in obese and lean twins</div>,&nbsp;<div class="source">Nature</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">457</div>&nbsp;(pg.&nbsp;<div class="fpage">480</div>-<div class="lpage">484</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20core%20gut%20microbiome%20in%20obese%20and%20lean%20twins&amp;author=P%20Turnbaugh&amp;author=M%20Hamady&amp;author=T%20Yatsunenko&amp;author=B%20Cantarel&amp;author=A%20Duncan&amp;author=R%20Ley&amp;author=M%20Sogin&amp;author=W%20Jones&amp;author=B%20Roe&amp;author=J%20Affourtit&amp;author=M%20Egholm&amp;author=B%20Henrissat&amp;author=AC%20Heath&amp;author=R%20Knight&amp;author=JI%20Gordon&amp;publication_year=2008&amp;journal=Nature&amp;volume=457&amp;pages=480-484" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07540" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07540" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnature07540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19043404" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib44" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Von Mering</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P</div></div>,&nbsp; <div class="name"><div class="surname">Raes</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Tringe</div> <div class="given-names">SG</div></div>,&nbsp; <div class="name"><div class="surname">Doerks</div> <div class="given-names">T</div></div>,&nbsp; <div class="name"><div class="surname">Jensen</div> <div class="given-names">LJ</div></div>,&nbsp; <div class="name"><div class="surname">Ward</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Bork</div> <div class="given-names">P</div></div>.&nbsp;</div><div class="article-title">Quantitative phylogenetic assessment of microbial communities in diverse environments</div>,&nbsp;<div class="source">Science</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">315</div>&nbsp;(pg.&nbsp;<div class="fpage">1126</div>-<div class="lpage">1130</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20phylogenetic%20assessment%20of%20microbial%20communities%20in%20diverse%20environments&amp;author=C%20Von%20Mering&amp;author=P%20Hugenholtz&amp;author=J%20Raes&amp;author=SG%20Tringe&amp;author=T%20Doerks&amp;author=LJ%20Jensen&amp;author=N%20Ward&amp;author=P%20Bork&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=1126-1130" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1133420" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1133420" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1133420"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17272687" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
                    <h2 class="authorNotes-section-title" id="authorNotesSectionTitle" scrollto-destination="authorNotesSectionTitle">Author notes</h2>
<div content-id=""><div class="fn"><div class="footnote-content">Associate Editor: Lars Jermiin</div></div></div><div class="copyright copyright-statement">© The Author(s) 2011. Published by Oxford University Press, on behalf of the Society of Systematic Biologists.</div><div class="license"><div class="license-p">This
 is an Open Access article distributed under the terms of the Creative 
Commons Attribution Non-Commercial License 
(http://creativecommons.org/licenses/by-nc/2.5), which permits 
unrestricted non-commercial use, distribution, and reproduction in any 
medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget --> 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=1667010&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5349">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/sysbio/60/3/10.1093_sysbio_syr010/3/syr010_supplementary_data.zip?Expires=1561463706&amp;Signature=KAC04f2AhfOGx3xnSUZPbDBUcyVTD0HBJLbv9UNs1KuBjL62y3AEdvkWCMyPNBJjTZAjpURVHO31srLxGCcGvvpR70ZmVEBRf9stceiO5xY7ZJvgEUsscXLtJKgK1zbdH%7EKaoj2Po32iDce6y4SwtmSzWb0cj3IptlUQbsIRQKYh7k-E7kZSYJ%7EL4lgKcAJ-E48SrL8Fud2Vg-g-y-38y-E6qJo4HcUSLPhDNfkLy3LruiG0gNTSlqgUp8jymmCngJla2-PDodNi4TN%7E%7EE1z8GDdi5%7EGMQea2FQ2tTSzwqzQOKDdM0f7Oq5m63aRzerro3IITk5gu%7EE2X6TCCo4CFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">4,499</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000289805600004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">217</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/sysbio/syr010" data-uuid="79c63709-56ba-e417-a2f8-a8e46d1f1203"><a target="" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=24454107" style="display:inline-block;">
        <img alt="Article has an altmetric score of 9" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1667010"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000289805600004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fsysbio%2farticle%2f60%2f3%2f291%2f1667010">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000289805600004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (217)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F60%2F3%2F291%2F1667010" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/sysbio/CrossRef-CitedBy/1667010">CrossRef</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz044/5521411?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Phylogenomics Uncovers Confidence and
 Conflict in the Rapid Radiation of Australo-Papuan Rodents
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz042/5513370?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   The Multispecies Coalescent Over-splits Species in the Case of Geographically Widespread Taxa
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz028/5484767?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   An Annealed Sequential Monte Carlo Method for Bayesian Phylogenetics
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz041/5511414?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Mean and Variance of Phylogenetic Trees
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/sysbio/advance-article/doi/10.1093/sysbio/syz043/5511413?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Monophyly, Taxon Sampling, and the 
Nature of Ranks in the Classification of Orb-weaving Spiders (Araneae: 
Araneoidea)
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="Performance, Accuracy, and Web Server for Evolutionary Placement of Short Sequence Reads under Maximum Likelihood | Systematic Biology | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/sysbio/syr010");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/sysbio_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/sysbio/pages/About">About Systematic Biology</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/sysbio/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/sysbio/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.systbio.org/">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/systbiol">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/sysbio/subscribe">Purchase</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-5">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-SystematicBiology" class="journal-footer-affiliations-logo" src="sysbio_f1-178201843.svg" alt="Systematic Biology">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1076-836X</li>
                                            <li>Print ISSN 1063-5157</li>
                    <li>Copyright © 2019 Society of Systematic Biologists</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh64" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.6072316804029534&amp;iit=1561377977452&amp;tmr=load%3D1561377977416%26core%3D1561377977446%26main%3D1561377977448%26ifr%3D1561377977456&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F60%2F3%2F291%2F1667010&amp;href=https%3A%2F%2Facademic.oup.com%2Fsysbio%2Farticle%2F60%2F3%2F291%2F1667010&amp;dt=Performance%2C%20Accuracy%2C%20and%20Web%20Server%20for%20Evolutionary%20Placement%20of%20Short%20Sequence%20Reads%20under%20Maximum%20Likelihood&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5d10bcb92dc04be6&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DPerformance%252C%2520Accuracy%252C%2520and%2520Web%2520Server%2520for%2520Evolutionary%2520Placement%2520of%2520Short%2520Sequence%2520Reads%2520under%2520Maximum%2520Likelihood%26description%3DAbstract.%2520%2520We%2520present%2520an%2520evolutionary%2520placement%2520algorithm%2520%28EPA%29%2520and%2520a%2520Web%2520server%2520for%2520the%2520rapid%2520assignment%2520of%2520sequence%2520fragments%2520%28short%2520reads%29%2520to%2520edges%2520of%2520a%2520give%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fsysbio%252Farticle%252F60%252F3%252F291%252F1667010%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fsysbio%252FIssue%252F60%252F3%252F0%252Fm_sysbio60_3.cover.gif%253FExpires%253D1624442107%2526Signature%253DbvtaWs%7ExKVXspH9nhyTgb%7EelYj4NKgBjovHNSZYnjbhN0Ybu%7EQjS090RwpkzrqZyl9302fbV8AVhHQtyczIKNUXGZaC-KO1UDh4ajYJqG5L7IB6DYCoggAkpxzq5tRTDHpxIkK5EAy2DlKq8G79Hk5HQC7lKY5m0u69nbhmcVv%7EO2g%7EfX7AqHHiE3%7E8fK9pOcP0NlrmfsZOGN5MOoA6NvfZQObAXzKfzYCVbIV6r%7Ev7PnVwgyZo5O7KHQBSKQVPJeZpyw0im3jMOTx8D0zs9AFOxTAvsyrU5UaZVnCvobTc9SGwdpOeduDj1zxi7V59YVRM8PKHoeJefUZuerdGLsQ__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2011-03-23%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.4.10-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5349&amp;returnUrl=%2fsysbio%2farticle%2f60%2f3%2f291%2f1667010">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>
<script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>




<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="syr010.js"></script><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_Size4,sans-serif;"></div></div></body></html>