<!DOCTYPE html>
<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
      <meta name="citation_pii" content="S0169534710000662">
<meta name="citation_issn" content="0169-5347">
<meta name="citation_volume" content="25">
<meta name="citation_lastpage" content="418">
<meta name="citation_issue" content="7">
<meta name="citation_publisher" content="Elsevier Current Trends">
<meta name="citation_firstpage" content="410">
<meta name="citation_journal_title" content="Trends in Ecology &amp; Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.tree.2010.04.001">
<meta name="dc.identifier" content="10.1016/j.tree.2010.04.001">
<meta name="citation_article_type" content="Review article">
<meta property="og:description" content="Understanding the forces that influence natural variation within and among populations has been a major objective of evolutionary biologists for decad…">
<meta property="og:image" content="https://ars.els-cdn.com/content/image/Dcellpress.gif">
<meta name="citation_title" content="Approximate Bayesian Computation (ABC) in practice">
<meta property="og:title" content="Approximate Bayesian Computation (ABC) in practice">
<meta name="citation_publication_date" content="2010/07/01">
<meta name="citation_online_date" content="2010/05/18">
<meta name="citation_pdf_url" content="https://www.sciencedirect.com/science/article/pii/S0169534710000662/pdfft?md5=19d73a51c65f4cabb36fdc807fa2ac4f&amp;pid=1-s2.0-S0169534710000662-main.pdf">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR">
      <title>Approximate Bayesian Computation (ABC) in practice - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S0169534710000662">
      <meta property="og:type" content="article">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel="shortcut icon" href="https://sdfestaticassets-eu-west-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://sdfestaticassets-eu-west-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="arp.css">
      <link rel="dns-prefetch" href="https://w.usabilla.com/">
      <link rel="dns-prefetch" href="https://www.deepdyve.com/">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <iframe src="javascript:false" title="" style="width: 0px; height: 0px; border: 0px none; display: none;"></iframe><script>
        window.pageTargeting = {"region":"eu-west-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Trends in Ecology & Evolution"};
        window.pageData = {"content":[{"entitlementType":"package","format":"MIME-XHTML","id":"sd:article:pii:S0169534710000662","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:rev","publicationType":"journal","issn":"0169-5347","volumeNumber":"25","issueNumber":"7"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productAppVersion":"Full","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1540822739910,"loadTime":""},"visitor":{"accessType":"ae:ANON_IP","accountId":"ae:262591","accountName":"ae:International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"ae:12289512","ipAddress":"197.248.44.39","appSessionId":"50fde1736906d84eda69c8a17bce301e14e8gxrqa"}};
        window.arp = {
          config: {"reduxLogging":false,"assetsBaseUrl":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com/prod/938ff3bae6dc95b874f4e26b3f27d0639fb57223","cdnAssetsHost":"https://sdfestaticassets-eu-west-1.sciencedirectassets.com","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","googleMapsApiKey":"AIzaSyAV0ZB1XWtRlzJ06pJAW74Oan87lQVgBHY","bosUrl":"https://feedback.recs.d.elsevier.com/raw/events","bosTimeOut":60000,"strictMode":false,"enableSocialSharing":false,"facebookAppId":"269707397153187","enableGoogleScholarLinks":false},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
      </script>
      
      <!-- begin mPulse embed code -->
      <script>
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
    <script src="satellite-5b3b560664746d57b70018c6.js"></script><script src="s-code-contents-9c0358adbc3b5986e210099b3bf1d427fc5bd286.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_266.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link id="plx-css-summary" type="text/css" rel="stylesheet" href="https://d39af2mgp1pqhg.cloudfront.net/summary.css"><script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script><script type="text/javascript" src="//d39af2mgp1pqhg.cloudfront.net/extjs/xss.js"></script></head>
    <body><div style="display: none;" id="lightningjs-usabilla_live"><div><iframe id="lightningjs-frame-usabilla_live" frameborder="0"></iframe></div></div>
      <a class="sr-only sr-only-focusable" href="#app">Skip to main content</a>
      <!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://service.elsevier.com/app/answers/detail/a_id/9831">this support page</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
      <div data-iso-key="_0"><div class="App" id="app" data-reactroot=""><div class="page"><section><div id="header"><div class="u-bg-white" style="overflow:visible" role="banner"><div class="els-header" style="min-height:80px"><a id="els-main-title-link" href="https://www.sciencedirect.com/"><svg id="els-header-wordmark" class="u-fill-orange u-margin-l-top u-margin-s-left u-margin-l-left-from-sm" viewBox="-3334 3439.4 163 26" style="width:163px;height:24px"><title>ScienceDirect</title><g transform="scale(.8125,.8125)"><path d="M-4099.8,4240.4c0-1.8,1-3.6,4.4-3.6c1.7,0,3.7,0.5,5.5,1.6l0.2-3.1c-1.7-0.7-3.5-1.1-5.6-1.1 c-5.5,0-7.8,2.9-7.8,6.4c0,6.6,10.4,7.2,10.4,11.7c0,1.8-1.4,3.6-4.6,3.6c-2,0-4-0.7-5.6-1.6l-0.4,3.1c1.8,0.9,4.2,1.2,6.1,1.2 c5,0,7.9-2.9,7.9-6.5C-4089.4,4245.8-4099.8,4244.9-4099.8,4240.4"></path><path id="c" d="M-4080.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4083.1,4244.3-4080.6,4243.1-4080.3,4242.9"></path><path id="i" d="M-4068,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-4066,4234.1-4067,4233.1-4068,4233.1 M-4069.5,4258.1h3v-17.7h-3V4258.1z"></path><path id="e" d="M-4057.1,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4058.7,4244.4-4057.4,4243.3-4057.1,4243.1z M-4047.5,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4 c0,5.8,3.5,9.2,7.9,9.2c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-4047.5,4248.9"></path><path d="M-4034.8,4240c-2.6,0-4.3,1.3-5.7,3.1l-0.5-2.6h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.5v12.8h3v-11.9 c0.8-1.1,2.5-2.9,2.9-3.1c0.3-0.2,1.5-0.5,2.5-0.5c2.7,0,2.9,1.4,3,4c0,1.4,0,3.7,0,3.7c0,3.5-0.1,7.6-0.1,7.6h3 c0,0,0.1-5.3,0.1-8.2c0-1.8,0-3.5-0.1-5.3C-4029.5,4241-4031.6,4240-4034.8,4240"></path><path d="M-3982.5,4255.6h-4.4v-18.6h4.8c6.4,0,8.2,5.2,8.2,9.2C-3973.8,4252.2-3976.5,4255.6-3982.5,4255.6z M-3981.6,4234.6h-8.4v23.5h8.1c8.6,0,11.5-6.7,11.5-11.9C-3970.4,4240.9-3973.2,4234.6-3981.6,4234.6"></path><path d="M-3950.5,4240c-1.9,0-3.4,1.7-4,3.2l-0.5-2.8h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.4v12.8h3v-11.2 c0.6-1.5,2-4.4,3.7-4.4c1.1,0,1.2,1.2,1.2,1.4l2.5-0.7v-0.2c0,0,0-0.2-0.1-0.5C-3947.4,4240.9-3948.5,4240-3950.5,4240"></path><path d="M-3903.5,4255.2c-1.1,0.4-2,0.8-3,0.8c-1.4,0-1.9-0.8-1.9-2.8v-10.4h4.6v-2.3h-4.6v-4.7h-2.9v4.7h-3.2v2.3h3.2 v11.4c0,3.1,1.6,4.4,3.9,4.4c1.4,0,3-0.5,4.1-0.9L-3903.5,4255.2"></path><g><path d="M-3923.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-3926.1,4244.3-3923.6,4243.1-3923.3,4242.9"></path></g><g><path d="M-3941.6,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-3943.2,4244.4-3941.8,4243.3-3941.6,4243.1z M-3932,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3932,4248.9"></path></g><g><path d="M-3964.5,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-3962.5,4234.1-3963.4,4233.1-3964.5,4233.1 M-3965.9,4258.1h3v-17.7h-3V4258.1z"></path></g><g><path d="M-4004.4,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4006,4244.4-4004.6,4243.3-4004.4,4243.1z M-3994.7,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3994.7,4248.9"></path></g><g><path d="M-4019.1,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4021.8,4244.3-4019.4,4243.1-4019.1,4242.9"></path></g></g></svg></a><div class="move-right u-display-inline-block"><nav class="u-clr-grey8 u-show-from-md"><div class="u-display-inline-block" style="margin-top:29px"><span><span class="u-margin-l-right"><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/browse/journals-and-books" id="els-header-navigation-section-journals-and-books" style="border-bottom:"><span class="anchor-text">Journals &amp; Books</span></a></span><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS0169534710000662" id="els-header-navigation-section-register"><span class="anchor-text">Register</span></a></span><a class="anchor qa-no-js-fallback-link anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS0169534710000662" id="els-header-user-sign-in"><span class="anchor-text">Sign in</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a><span><a class="anchor u-margin-l-hor help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="help" title="Help" aria-label="Help" target="_blank"><span class="anchor-text"><svg focusable="false" viewBox="0 0 114 128" style="margin-top:-10px" width="21.375" height="24" class="icon icon-help"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></span></a></span></div></nav></div><div class="u-hide-from-md move-right"><ul><li style="list-style:none;display:inline-block;margin-top:32px" class="u-margin-s-hor u-margin-l-hor-from-sm"><div class="hamburger-button"><button class="button-link button-link-primary" aria-label="Mobile menu" aria-expanded="false" type="button"><svg viewBox="0 0 40 18" width="40" height="18" y="52"><path d="M0 16h40v2H0zm0-8h40v2H0zm0-8h40v2H0z"></path></svg><span class="button-link-text"></span></button></div></li></ul><div style="bottom:0;left:0;position:fixed;top:0;width:100%;z-index:70;opacity:.8" class="u-bg-grey1 u-display-none"></div><div id="mobile-menu" style="overflow:auto;position:fixed;width:288px;right:0;top:0;z-index:1000;height:100%" aria-label="Mobile menu" class="u-bg-grey7 u-clr-grey1 u-display-none" role="navigation"><div class="u-bg-black panel-s"></div><div class="u-bg-grey8 panel-s"><ul><li style="list-style:none"><a class="anchor journals-and-books-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/browse/journals-and-books" style="border-bottom:" id="mobile-journals-and-books-link"><span class="anchor-text">Journals &amp; Books</span></a></li><li style="list-style:none"><a class="anchor u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS0169534710000662" id="mobile-register-link"><span class="anchor-text">Register</span></a></li></ul></div><div class="panel-s u-bg-grey7"><ul class="text-xs"></ul><ul class="u-margin-s-top text-s"><li style="list-style:none"><a class="anchor anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS0169534710000662" id="mobile-sign-in-out-link" rel="nofollow"><span class="anchor-text">Sign In</span></a></li><span><a class="anchor u-padding-xs-top text-s help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="mobile-help" title="Help" aria-label="Help" target="_blank"><span class="anchor-text">Help</span></a></span></ul></div></div><div class="move-bottom move-center"></div></div></div></div></div><div class="Article" id="mathjax-container"><div class="sticky-outer-wrapper"><div class="sticky-inner-wrapper" style="position: relative; z-index: 1; transform: translate3d(0px, 0px, 0px);"><div class="Toolbar" role="region" aria-label="download options and search"><div class="toolbar-container"><div class="u-show-from-lg col-lg-6">&nbsp;</div><div class="buttons pull-left pad-left"><button class="button show-toc-button button-anchor u-hide-from-lg u-margin-s-right" type="button"><svg focusable="false" viewBox="0 0 104 128" width="19.5" height="24" class="icon icon-list"><path d="m2e1 95a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm14 55h68v1e1h-68zm0-3e1h68v1e1h-68zm0-3e1h68v1e1h-68z"></path></svg><span class="button-text"><span>Outline</span></span></button><div class="PdfDownloadButton"><div class="popover download-pdf-popover" id="download-pdf-popover"><div id="popover-trigger-download-pdf-popover"><button id="pdfLink" class="button button-anchor u-padding-0-left" role="button" aria-haspopup="true" aria-expanded="false" aria-controls="Download PDF options" type="button"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor pdf-icon"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="button-text"><span class="pdf-download-label u-show-inline-from-lg">Download PDF</span><span class="pdf-download-label-short u-hide-from-lg">Download</span></span></button></div></div></div><div class="ExportCitation" role="region" id="export-citation"><div class="popover export-citation-popover" id="export-citation-popover"><div id="popover-trigger-export-citation-popover"><button class="button button-anchor" role="button" aria-haspopup="true" aria-expanded="false" type="button"><span class="button-text">Export</span></button></div></div></div></div><div class="quick-search-container pull-right pad-right u-show-from-md"><form id="quick-search" class="QuickSearch u-margin-xs-right" action="/search/advanced" method="get"><input class="query" aria-label="Search ScienceDirect" name="qs" placeholder="Search ScienceDirect" type="text"><button class="button button-primary" type="submit" aria-label="Submit search"><span class="button-text"><svg focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><a class="advanced-search-link" href="https://www.sciencedirect.com/search/advanced">Advanced</a><input name="origin" value="article" type="hidden"><input name="zone" value="qSearch" type="hidden"></form></div></div></div></div></div><div class="article-wrapper grid row"><div class="u-show-from-lg col-lg-6"><div class="TableOfContents u-margin-l-bottom" lang="en"><div class="Outline" id="toc-outline"><h2>Outline</h2><ol class="u-padding-s-bottom"><li><a href="#aep-section-id20" title="Inference with simulations in evolutionary genetics">Inference with simulations in evolutionary genetics</a></li><li><a href="#aep-section-id21" title="The ABC of Approximate Bayesian Computation">The ABC of Approximate Bayesian Computation</a></li><li><a href="#aep-section-id22" title="Bayesian data analysis: building, fitting, and improving the model">Bayesian data analysis: building, fitting, and improving the model</a></li><li><a href="#aep-section-id26" title="Choice and dimension of summary statistics">Choice and dimension of summary statistics</a></li><li><a href="#aep-section-id27" title="The future of ABC">The future of ABC</a></li><li><a href="#aep-section-id31" title="Conclusions">Conclusions</a></li><li><a href="#aep-acknowledgment-id32" title="Acknowledgements">Acknowledgements</a></li><li><a href="#aep-bibliography-id33" title="References">References</a></li><li><a href="#aep-glossary-id35" title="Glossary">Glossary</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show full outline</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2>Figures (4)</h2><ol><li><a href="#fig2"><div><img alt="Figure I. The bottleneck model of D" src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b1.sml" style="max-width: 219px; max-height: 124px;"></div></a></li><li><a href="#fig3"><div><img alt="Figure I. Estimation of the biodiversity number, θ, for communities of tropical trees in…" src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b2.sml" style="max-width: 112px; max-height: 164px;"></div></a></li><li><a href="#fig4"><div><img alt="Figure I. The main steps of an ABC analysis" src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b4.sml" style="max-width: 132px; max-height: 163px;"></div></a></li><li><a href="#fig1"><div><img alt="Figure 1. Linear regression adjustment in the ABC algorithm" src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1.sml" style="max-width: 219px; max-height: 123px;"></div></a></li></ol><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2>Tables (1)</h2><ol class="u-padding-s-bottom"><li><a href="#tbl1" title="Posterior estimates of Ne"><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table I</a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right" role="main" lang="en"><noscript>JavaScript is disabled on your browser. Please enable JavaScript to use all the features on this page.</noscript><div class="Publication" id="publication"><div class="publication-brand"><a class="publication-brand-title-link" title="Go to Trends in Ecology &amp; Evolution on ScienceDirect" href="https://www.sciencedirect.com/science/journal/01695347"><img class="article-branding u-show-from-sm" src="B01695347.svg" alt="Trends in Ecology &amp; Evolution"><h2 class="u-hide-from-sm publication-title-link">Trends in Ecology &amp; Evolution</h2></a><div class="text-xs"><a title="Go to table of contents for this volume/issue" href="https://www.sciencedirect.com/science/journal/01695347/25/7">Volume 25, Issue 7</a>, <!-- -->July 2010<!-- -->, Pages 410-418</div></div><div class="publication-cover u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/01695347/25/7"><img class="publication-cover-image" src="Dcellpress.gif" alt="Journal home page for Trends in Ecology &amp; Evolution"></a></div></div><h1 class="Head"><div class="article-dochead"><span>Review</span></div><span class="title-text">Approximate Bayesian Computation (ABC) in practice</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><a class="author size-m workspace-trigger" name="baep-author-id14" href="#%21"><span class="content"><span class="text given-name">Katalin</span><span class="text surname">Csilléry</span><span class="author-ref" id="baff1"><sup>1</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id15" href="#%21"><span class="content"><span class="text given-name">Michael G.B.</span><span class="text surname">Blum</span><span class="author-ref" id="baff1"><sup>1</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id16" href="#%21"><span class="content"><span class="text given-name">Oscar E.</span><span class="text surname">Gaggiotti</span><span class="author-ref" id="baff2"><sup>2</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id17" href="#%21"><span class="content"><span class="text given-name">Olivier</span><span class="text surname">François</span><span class="author-ref" id="baff1"><sup>1</sup></span></span></a></div></div></div><button class="show-hide-details"><svg viewBox="0 0 9 9" class="icon-expand"><path d="M5 7H4V5H2V4h2V2h1v2h2v1H5z"></path><path d="M0 0v9h9V0zm1 1h7v7H1z"></path></svg>Show more</button></div><div class="DoiLink" id="doi-link"><a class="doi" href="https://doi.org/10.1016/j.tree.2010.04.001" target="_blank" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier">https://doi.org/10.1016/j.tree.2010.04.001</a><a class="rights-and-content" target="_blank" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S0169534710000662&amp;orderBeanReset=true">Get rights and content</a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts" id="abstracts"><div class="abstract author" id="aep-abstract-id18"><div id="aep-abstract-sec-id19"><p>Understanding
 the forces that influence natural variation within and among 
populations has been a major objective of evolutionary biologists for 
decades. Motivated by the growth in computational power and data 
complexity, modern approaches to this question make intensive use of 
simulation methods. Approximate Bayesian Computation (ABC) is one of 
these methods. Here we review the foundations of ABC, its recent 
algorithmic developments, and its applications in evolutionary biology 
and ecology. We argue that the use of ABC should incorporate all aspects
 of Bayesian data analysis: formulation, fitting, and improvement of a 
model. ABC can be a powerful tool to make inferences with complex models
 if these principles are carefully applied.</p></div></div></div><ul id="issue-navigation" class="issue-navigation u-margin-s-bottom u-bg-grey1"><li class="previous move-left u-padding-s-ver u-padding-s-left"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S0169534710000856"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-left"><path d="m1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text"><strong>Previous </strong><span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span></a></li><li class="next move-right u-padding-s-ver u-padding-s-right"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S0169534710000832"><span class="button-alternative-text"><strong>Next </strong><span class="extra-detail-1">article</span><span class="extra-detail-2"> in issue</span></span><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Body" id="body"><div><section id="aep-section-id20"><h2>Inference with simulations in evolutionary genetics</h2><p>Natural
 populations have complex demographic histories: their sizes and ranges 
change over time, leading to fission and fusion processes that leave 
signatures on their genetic composition <a name="bbib1" href="#bib1" class="workspace-trigger">[1]</a>.
 One ‘promise’ of biology is that molecular data will help us uncover 
the complex demographic and adaptive processes that have acted on 
natural populations. The widespread availability of different molecular 
markers and increased computer power has fostered the development of 
sophisticated statistical methods that have begun to fulfill this 
expectation. Most of these techniques are based on the concept of 
likelihood, a function that describes the probability of the data given a
 parameter.</p><p>Current approaches derive likelihoods based on classical population genetics or coalescent theory <a name="bbib2" href="#bib2" class="workspace-trigger">2</a>, <a name="bbib3" href="#bib3" class="workspace-trigger">3</a>. In recent years, likelihood-based inference has frequently been undertaken using Markov chain Monte Carlo (MCMC) techniques <a name="bbib2" href="#bib2" class="workspace-trigger">2</a>, <a name="bbib4" href="#bib4" class="workspace-trigger">4</a>.
 Many of these methods are, however, limited by the difficulty of 
computing the likelihood function, thus restricting their use to simple 
evolutionary scenarios and molecular models. Additionally, even with 
ever-increasing computational power, these techniques cannot keep up 
with the demands of the large amounts of data generated by recently 
developed, high-throughput DNA sequencing technologies. Both of these 
factors have stimulated the development of new methods that approximate 
the likelihood <a name="bbib4" href="#bib4" class="workspace-trigger">[4]</a>.</p><p>One of the most recent approaches is Approximate Bayesian Computation (ABC <a name="bbib5" href="#bib5" class="workspace-trigger">[5]</a>).
 ABC approaches bypass exact likelihood calculations by using summary 
statistics and simulations. Summary statistics are values calculated 
from the data to represent the maximum amount of information in the 
simplest possible form. Their use goes back to the origins of population
 genetics when, for instance, Sewall Wright <a name="bbib6" href="#bib6" class="workspace-trigger">[6]</a>
 devised fixation indices to describe correlations among alleles sampled
 at hierarchically organized levels of a population. The use of 
simulations, both as artificial experiments in evolution and inference 
tools, also has a long tradition in population genetics <a name="bbib7" href="#bib7" class="workspace-trigger">[7]</a>.</p></section><section id="aep-section-id21"><h2>The ABC of Approximate Bayesian Computation</h2><p>ABC has its roots in the rejection algorithm, a simple technique to generate samples from a probability distribution <a name="bbib8" href="#bib8" class="workspace-trigger">8</a>, <a name="bbib9" href="#bib9" class="workspace-trigger">9</a>.
 The basic rejection algorithm consists of simulating large numbers of 
datasets under a hypothesized evolutionary scenario. The parameters of 
the scenario are not chosen deterministically, but sampled from a 
probability distribution. The data generated by simulation are then 
reduced to summary statistics, and the sampled parameters are accepted 
or rejected on the basis of the distance between the simulated and the 
observed summary statistics. The sub-sample of accepted values contains 
the fitted parameter values, and allows us to evaluate uncertainty on 
parameters given the observed statistics.</p><div><p>The applications of ABC are often based on improved versions of the basic rejection scheme <a name="bbib5" href="#bib5" class="workspace-trigger">5</a>, <a name="bbib10" href="#bib10" class="workspace-trigger">10</a>, <a name="bbib11" href="#bib11" class="workspace-trigger">11</a>, <a name="bbib12" href="#bib12" class="workspace-trigger">12</a>, <a name="bbib13" href="#bib13" class="workspace-trigger">13</a>, and have already yielded valuable insights into various questions in evolutionary biology and ecology (<a name="btb1" href="#tb1" class="workspace-trigger">Box 1</a>, <a name="btb2" href="#tb2" class="workspace-trigger">Box 2</a>). Examples include the estimation of various demographic parameters such as: the effective population size <a name="bbib14" href="#bib14" class="workspace-trigger">[14]</a>; the detection and timing of past demographic events (e.g. growth or decline of populations <a name="bbib9" href="#bib9" class="workspace-trigger">9</a>, <a name="bbib15" href="#bib15" class="workspace-trigger">15</a>, <a name="bbib16" href="#bib16" class="workspace-trigger">16</a>, <a name="bbib17" href="#bib17" class="workspace-trigger">17</a>, <a name="bbib18" href="#bib18" class="workspace-trigger">18</a>, <a name="bbib19" href="#bib19" class="workspace-trigger">19</a>, <a name="bbib20" href="#bib20" class="workspace-trigger">20</a>, <a name="bbib21" href="#bib21" class="workspace-trigger">21</a>, <a name="bbib22" href="#bib22" class="workspace-trigger">22</a>); or the rate of spread of pathogens <a name="bbib23" href="#bib23" class="workspace-trigger">23</a>, <a name="bbib24" href="#bib24" class="workspace-trigger">24</a>. Other applications have compared alternative models of evolution in humans <a name="bbib25" href="#bib25" class="workspace-trigger">25</a>, <a name="bbib26" href="#bib26" class="workspace-trigger">26</a>, <a name="bbib27" href="#bib27" class="workspace-trigger">27</a>, <a name="bbib28" href="#bib28" class="workspace-trigger">28</a>, <a name="bbib29" href="#bib29" class="workspace-trigger">29</a>, <a name="bbib30" href="#bib30" class="workspace-trigger">30</a>, <a name="bbib31" href="#bib31" class="workspace-trigger">31</a>, <a name="bbib32" href="#bib32" class="workspace-trigger">32</a>, <a name="bbib33" href="#bib33" class="workspace-trigger">33</a>, <a name="bbib34" href="#bib34" class="workspace-trigger">34</a>, <a name="bbib35" href="#bib35" class="workspace-trigger">35</a>, <a name="bbib36" href="#bib36" class="workspace-trigger">36</a>; inferred admixture proportions <a name="bbib37" href="#bib37" class="workspace-trigger">37</a>, <a name="bbib38" href="#bib38" class="workspace-trigger">38</a>, <a name="bbib39" href="#bib39" class="workspace-trigger">39</a>; migration rates <a name="bbib40" href="#bib40" class="workspace-trigger">40</a>, <a name="bbib41" href="#bib41" class="workspace-trigger">41</a>; mutation rates <a name="bbib9" href="#bib9" class="workspace-trigger">[9]</a>; rates of recombination and gene conversion <a name="bbib42" href="#bib42" class="workspace-trigger">42</a>, <a name="bbib43" href="#bib43" class="workspace-trigger">43</a>, <a name="bbib44" href="#bib44" class="workspace-trigger">44</a>; the strength of positive selection <a name="bbib45" href="#bib45" class="workspace-trigger">[45]</a>; the influence of selection on gene regulation in humans <a name="bbib46" href="#bib46" class="workspace-trigger">[46]</a>; or the age of an allele <a name="bbib27" href="#bib27" class="workspace-trigger">27</a>, <a name="bbib47" href="#bib47" class="workspace-trigger">47</a>.
 ABC has also been used to make inferences at an inter-specific level, 
for example: dating the divergence between closely related species <a name="bbib48" href="#bib48" class="workspace-trigger">48</a>, <a name="bbib49" href="#bib49" class="workspace-trigger">49</a>, <a name="bbib50" href="#bib50" class="workspace-trigger">50</a>, <a name="bbib51" href="#bib51" class="workspace-trigger">51</a>, <a name="bbib52" href="#bib52" class="workspace-trigger">52</a>; making inferences about the evolution of polyploidy <a name="bbib53" href="#bib53" class="workspace-trigger">[53]</a>; and identifying a biodiversity hotspot in the Brazilian forest <a name="bbib54" href="#bib54" class="workspace-trigger">[54]</a>. In ecology, ABC has been used to infer parameters of the neutral theory of biodiversity in tropical forests (<a name="bbib55" href="#bib55" class="workspace-trigger">[55]</a>, <a name="btb2" href="#tb2" class="workspace-trigger">Box 2</a>).</p><div class="article-textbox" id="tb1" aria-label="" role="region"><span class="label">Box 1</span><span><p>Inferring the demographic history of <em>Drosophila melanogaster</em></p></span><div><section id="aep-section-id5"><h3>Selection or demography?</h3><p>Understanding
 the relative importance of selective and demographic processes raises 
considerable interest, however disentangling the effects of the two 
processes poses a serious challenge. This is because selection and 
demography may result in similar patterns of nucleotide variability. 
Developing models that take into account demographic processes is 
essential to identify regions of the genome under selection. For <em>Drosophila melanogaster</em>, Thornton and Andolfatto <a name="bbib16" href="#bib16" class="workspace-trigger">[16]</a>
 propose a simple model of the demographic history and, using ABC and 
Bayesian evaluation of the goodness of fit, they show that their model 
can predict the majority of the observed patterns of molecular 
variability without invoking selective processes.</p></section><section id="aep-section-id6"><h3>The bottleneck model</h3><div><p>In the cosmopolitan <em>D. melanogaster</em>,
 molecular variation is substantially reduced outside of Africa. This 
suggests an African origin and a reduction in population size when 
colonizing other continents. The population model includes a reduction 
in population size, and then for simplicity a recovery to the same 
effective population size as before (bottleneck, <a name="bfig2" href="#fig2" class="workspace-trigger">Figure I</a>).
 Timing of the bottleneck can provide an estimate of the date of 
colonization of Europe, from where the authors have obtained samples of 
molecular data.</p><figure class="figure" id="fig2"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b1.jpg" alt="" height="212"><ol class="links-for-figure"><li><a class="anchor download-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b1.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text">Download full-size image</span></a></li></ol></span><div class="captions"><span><p><span class="label">Figure I</span>. The bottleneck model of <em>D. melanogaster</em> colonizing Europe. Image of <em>D. melanogaster</em> reproduced with permission from Nicolas Gompel (<a href="http://www.ibdml.univ-mrs.fr/equipes/BP_NG/" target="_blank">http://www.ibdml.univ-mrs.fr/equipes/BP_NG/</a>).</p></span></div></figure></div></section><section id="aep-section-id7"><h3>Dating the colonization event</h3><p>Biogeographical studies suggest that <em>D. melanogaster</em> colonized the rest of the world only after the last glaciation period (about 6,000–10,000 years ago) <a name="bbib101" href="#bib101" class="workspace-trigger">[101]</a>. Using patterns of variability observed at 115 loci scattered across the X chromosome in a European population of <em>D. melanogaster</em>,
 Thornton and Andolfatto estimate the start of the bottleneck to be 
16,000 years before the present, associated with a 95% credible interval
 of 9,000–43,000 years <a name="bbib16" href="#bib16" class="workspace-trigger">[16]</a>. Thus, molecular data suggest that the colonization of Europe might have occurred before the last glaciation period.</p></section></div></div><div class="article-textbox" id="tb2" aria-label="" role="region"><span class="label">Box 2</span><span><p>Inferring the parameters of the neutral theory of biodiversity</p></span><div><section id="aep-section-id8"><h3>Hubbell's theory</h3><p>In community ecology, the unified neutral theory of biodiversity (UNTB) <a name="bbib102" href="#bib102" class="workspace-trigger">[102]</a>
 states that all individuals in a regional community are governed by the
 same rates of birth, death and dispersal. The UNTB is a stochastic, 
individual-based model that gives quantitative predictions for species 
abundance data. The theory has a practical merit for the evaluation of 
biodiversity: it provides a fundamental biodiversity number. The 
biodiversity number can be interpreted as the size of an ideal community
 that best approximates the real, sampled community <a name="bbib103" href="#bib103" class="workspace-trigger">[103]</a>.</p></section><section id="aep-section-id9"><h3>Inference of the fundamental biodiversity number</h3><div><p>Inference
 under the UNTB encounters a serious theoretical issue: species 
abundance data generally leads to two distinct and equally likely 
estimates of the biodiversity number. One solution is to use <em>a-priori</em> information to discard the unrealistic estimate of the biodiversity number. Jabot and Chave <a name="bbib55" href="#bib55" class="workspace-trigger">[55]</a>
 provide an elegant alternative solution to the estimation of the 
biodiversity number by combining phylogenetic information with ABC. They
 find that the phylogenetic relationships between the species living in 
the local community convey useful information for parameter inference. 
Exploiting the information contained in ‘tree imbalance’ statistics 
leads to a unique, most-likely estimate of the biodiversity number (<a name="bfig3" href="#fig3" class="workspace-trigger">Figure I</a>).</p><figure class="figure" id="fig3"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b2.jpg" alt="" height="549"><ol class="links-for-figure"><li><a class="anchor download-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b2.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text">Download full-size image</span></a></li></ol></span><div class="captions"><span><p><span class="label">Figure I</span>. Estimation of the biodiversity number, <em>θ,</em>
 for communities of tropical trees in Barro Colorado Island (Panama) 
with ABC. While the variation of the species evenness (measured by 
Shannon diversity, <em>H</em>) leads to two likely values of the biodiversity number (<em>θ</em><sub>1</sub> and <em>θ</em><sub>2</sub>), the use of a measure of phylogenetic tree balance, <em>B1</em>, leads to a unique most-likely biodiversity number (<em>θ</em><sub>2</sub>). Photograph reproduced with permission from Christian Ziegler (<a href="http://www.naturphoto.de/" target="_blank">http://www.naturphoto.de</a>), and figure reproduced with permission from Wiley-Blackwell <a name="bbib55" href="#bib55" class="workspace-trigger">[55]</a>.</p></span></div></figure></div></section><section id="aep-section-id10"><h3>Application of ABC to tropical rainforests</h3><p>Using ABC, Jabot and Chave <a name="bbib55" href="#bib55" class="workspace-trigger">[55]</a>
 give an estimate of the biodiversity number of the regional pool for 
two tropical-forest tree plots in Panama and Columbia. The biodiversity 
numbers are one order of magnitude greater than those computed by 
maximum-likelihood methods that do not use phylogenetic information. One
 explanation for those larger values is that the Central American plots 
extend over areas of the size of the entire Neotropical ecozone. The two
 forest plots in Central America would then be connected <em>via</em> dispersal to a large ecozone including Central and South America.</p></section></div></div></div><div><p>Although
 the use of ABC is widespread in many areas of evolutionary biology, it 
has become contentious in the field of statistical phylogeography <a name="bbib56" href="#bib56" class="workspace-trigger">56</a>, <a name="bbib57" href="#bib57" class="workspace-trigger">57</a> (<a name="btb3" href="#tb3" class="workspace-trigger">Box 3</a>).
 The main objections to ABC are that inference is limited to a finite 
set of phylogeographical models, and that models hypothesized in ABC 
studies are complex. As a consequence, conclusions from ABC could be 
influenced by subjective inclusion of evolutionary scenarios and 
implicit model assumptions that are not foreseen by the modeler. A 
recent article addresses these concerns in great detail, and points out 
that they are general criticisms of model-based approaches and are not 
specific to ABC <a name="bbib58" href="#bib58" class="workspace-trigger">[58]</a>.
 We acknowledge that difficulties can arise at several levels if using a
 simulation-based method such as ABC, thus the principal aim of this 
review is to encourage good practice in the use of the method. Here, we 
review some elementary Bayesian principles and emphasize some facets of 
Bayesian analysis that are often neglected in ABC studies. Then we 
highlight ways to improve inference with ABC via application of these 
principles.</p><div class="article-textbox" id="tb3" aria-label="" role="region"><span class="label">Box 3</span><span><p>Controversy surrounding ABC</p></span><div><p>The ABC approach has been vigorously criticized by Templeton <a name="bbib56" href="#bib56" class="workspace-trigger">56</a>, <a name="bbib57" href="#bib57" class="workspace-trigger">57</a> and forcefully defended by a group of statistical geneticists <a name="bbib58" href="#bib58" class="workspace-trigger">[58]</a>.
 The purpose of this Box is, therefore, to acknowledge the existence of 
this controversy and to provide a brief comment on the criticisms that 
we deem most relevant.</p><p>The issue that is most thoroughly discussed is the testing of complex phylogeographic models <em>via</em>
 computer simulation. According to Templeton, the impossibility of 
including an exhaustive set of alternative hypotheses in the testing 
procedure forces us to choose only one subset, introducing a great deal 
of subjectivity in the process. Moreover, in this situation one runs the
 risk of not including the true model in the restricted set of models, 
in which case the results of the test would be meaningless. According to
 Beaumont et al. <a name="bbib58" href="#bib58" class="workspace-trigger">[58]</a>,
 this situation is applicable to all model-based methods whether they 
rely on simulations or not. Although in principle this is a potentially 
important problem, in reality scientific arguments often revolve around a
 limited number of hypotheses or scenarios without the need to consider 
an infinite set of alternative models. The issue then becomes the 
possibility of model mis-specification, something that is also addressed
 by Templeton but again is not restricted to ABC approaches. This 
problem can in part be addressed by using the many statistical 
techniques for assessing the fit of a model <a name="bbib58" href="#bib58" class="workspace-trigger">[58]</a>.
 Models can always be improved and refined by other authors, allowing an
 open discussion that can greatly increase our understanding of the 
problem being studied. This is the way scientific progress is made.</p></div></div></div></section><section id="aep-section-id22"><h2>Bayesian data analysis: building, fitting, and improving the model</h2><div><p>The
 three main steps of Bayesian analysis are formulating the model, 
fitting the model to data, and improving the model by checking its fit 
and comparing it with other models <a name="bbib59" href="#bib59" class="workspace-trigger">[59]</a> (<a name="btb4" href="#tb4" class="workspace-trigger">Box 4</a>).
 When formulating a model, we use our experience and prior knowledge, 
and sometimes resort to established theories. This step is often 
mathematical in essence because it encompasses explicit definitions for 
the likelihood (or a generating mechanism) and the prior distribution. 
These quantities summarize background information about the data and the
 parameter values. Fitting the model is at the heart of Bayesian data 
analysis and, in modern approaches, it is often carried out with Monte 
Carlo algorithms. The aim of this second step is to calculate the 
posterior probability distribution and a credibility interval for the 
parameter of interest, starting with its prior distribution, and 
updating it based on the data <a name="bbib2" href="#bib2" class="workspace-trigger">[2]</a>.
 Models are always approximations of reality, so evaluating their 
goodness-of-fit and improving them is the third step of a Bayesian 
analysis. In this step, we often need to confront two or more models 
that differ in their level of complexity. The three steps of a Bayesian 
analysis are strongly inter-dependent and should be considered as a 
unified approach, with a possibility of cycling through the three 
stages.</p><div class="article-textbox" id="tb4" aria-label="" role="region"><span class="label">Box 4</span><span><p>Tutorial on ABC: estimating the effective population size (<em>N</em><sub><em>e</em></sub>)</p></span><div><p>Estimating
 the effective population size from molecular data is of interest to 
many evolutionary biologists. Here, we propose a sample of 40 haploid 
individuals genotyped at 20 unlinked microsatellite loci.</p><section id="aep-section-id11"><h3>Models</h3><p>Before
 seeing the data, three candidate demographic scenarios are 
hypothesized: constant population size, bottleneck, and divergence 
(where an ancestral population splits into two sub-populations of equal 
size). Uniform prior distributions are assumed. Simulations are carried 
out with Hudson's coalescent sampler, ms <a name="bbib88" href="#bib88" class="workspace-trigger">[88]</a>.</p></section><section id="aep-section-id12"><h3>Inference and model choice</h3><div><p>Model fitting is based on two classic summary statistics: genetic diversity and the Garza–Williamson statistic <a name="bbib104" href="#bib104" class="workspace-trigger">[104]</a>.
 Both measures are known to be sensitive to historical variation in 
population size. Estimation of the present effective population size, <em>N</em><sub><em>e</em></sub>, is carried out according to the algorithm of Beaumont et al. <a name="bbib5" href="#bib5" class="workspace-trigger">[5]</a>.
 Additional model parameters such as the divergence time or the duration
 and severity of the bottleneck are considered to be ‘nuisance 
parameters’ and estimates of <em>N</em><sub><em>e</em></sub> are 
averaged over these variables. Estimates obtained under the bottleneck 
and divergence models are close to the value we used for generating the 
example dataset (<em>N</em><sub><em>e</em></sub> = 600, <a name="btbl1" href="#tbl1" class="workspace-trigger">Table I</a>). Posterior model probabilities computed by multinomial logistic regression <a name="bbib66" href="#bib66" class="workspace-trigger">[66]</a>
 reveal that the bottleneck model is the most supported by the data 
(61%), but the divergence model also receives considerable support 
(38%).</p><div class="tables frame-all rowsep-0 colsep-0" id="tbl1"><div class="captions"><span><p><span class="label">Table I</span>. Posterior estimates of <em>N</em><sub><em>e</em></sub></p></span></div><div class="groups"><table><thead class="valign-top"><tr><th scope="col" class="rowsep-1 align-left">Model</th><th scope="col" class="rowsep-1 align-left">Posterior median</th><th scope="col" class="rowsep-1 align-left">95% credible interval</th></tr></thead><tbody><tr><td class="align-left">Constant population size</td><td class="align-left">3274</td><td class="align-left">3974, 4746</td></tr><tr><td class="align-left">Bottleneck</td><td class="align-left">588</td><td class="align-left">238, 1065</td></tr><tr><td class="align-left">Divergence</td><td class="align-left">550</td><td class="align-left">236, 1310</td></tr></tbody></table></div></div></div></section><section id="aep-section-id13"><h3>Model checking and model averaging</h3><div><p>Replicates
 of the data under the three models were simulated using the posterior 
estimates of the parameters. The distributions of the replicated data 
(posterior predictive distributions) were compared with the observed 
data in terms of the expected heterozygosity, a ‘test’ statistic that 
was not used during the model-fitting stage. We find that the observed 
value of heterozygosity lay within the tails of the posterior predictive
 distribution under the bottleneck and divergence models, but well 
outside under the constant population size model. We decided to discard 
the constant population size scenario, and estimated <em>N</em><sub><em>e</em></sub> by weighting the posterior values obtained from the bottleneck and divergence models (<a name="bfig4" href="#fig4" class="workspace-trigger">Figure I</a>).</p><figure class="figure" id="fig4"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b4.jpg" alt="" height="464"><ol class="links-for-figure"><li><a class="anchor download-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1b4.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text">Download full-size image</span></a></li></ol></span><div class="captions"><span><p><span class="label">Figure I</span>. The main steps of an ABC analysis.</p></span></div></figure></div></section></div></div></div><section id="aep-section-id23"><h3>Model building</h3><p>Two
 intimately linked questions arise when formulating or improving models.
 First, why do we want models? Second, how complex should the models be?
 Models can be used towards two distinct ends: explanation and 
prediction <a name="bbib60" href="#bib60" class="workspace-trigger">[60]</a>.
 Although we might be concerned with prediction when investigating the 
decrease of biodiversity or the consequences of global change <a name="bbib61" href="#bib61" class="workspace-trigger">[61]</a>,
 evolutionary models tend to be explanatory, i.e. they are used to help 
describe the evolutionary processes that have generated the data. There 
are often several potential explanatory models of a phenomenon, and 
model formulation is not restricted to hypothesizing a unique scenario. 
Often, many different explanatory models can be proposed, with the main 
objective of finding the most parsimonious explanation. As Einstein 
nicely stated, “models should be as simple as possible, but not more so”
 <a name="bbib62" href="#bib62" class="workspace-trigger">[62]</a>.</p></section><section id="aep-section-id24"><h3>Model fitting</h3><div><p>ABC
 algorithms can be classified into three broad categories. The first 
class of algorithms relies on the basic rejection algorithm <a name="bbib8" href="#bib8" class="workspace-trigger">8</a>, <a name="bbib9" href="#bib9" class="workspace-trigger">9</a>.
 Technical improvements of this basic scheme correct for the discrepancy
 between the simulated and the observed statistics by using local linear
 or non-linear regression techniques (<a name="bfig1" href="#fig1" class="workspace-trigger">Figure 1</a>) <a name="bbib5" href="#bib5" class="workspace-trigger">5</a>, <a name="bbib10" href="#bib10" class="workspace-trigger">10</a>, <a name="bbib63" href="#bib63" class="workspace-trigger">63</a>.
 A second class of methods, ABC-MCMC algorithms, explore the parameter 
space iteratively using the distance between the simulated and the 
observed summary statistics to update the current parameter values <a name="bbib11" href="#bib11" class="workspace-trigger">11</a>, <a name="bbib64" href="#bib64" class="workspace-trigger">64</a>.
 In this way, parameter values that produce simulations close to the 
observed data are visited preferentially. As in standard MCMC methods, 
ABC-MCMC requires that the Markov chain converges to its stationary 
state, a condition that can be difficult to verify <a name="bbib4" href="#bib4" class="workspace-trigger">[4]</a>. A third class of algorithms is inspired by Sequential Monte Carlo methods (SMC) <a name="bbib65" href="#bib65" class="workspace-trigger">[65]</a>.
 SMC-ABC algorithms approximate the posterior distribution by using a 
large set of randomly chosen parameter values called ‘particles’. These 
particles are propagated over time by simple sampling mechanisms or 
rejected if they generate data that match the observation poorly. 
Ongoing work is seeking to improve the parameter space exploration and 
develop efficient sampling strategies that drive particles toward 
regions of high posterior probability mass <a name="bbib12" href="#bib12" class="workspace-trigger">12</a>, <a name="bbib13" href="#bib13" class="workspace-trigger">13</a>, <a name="bbib66" href="#bib66" class="workspace-trigger">66</a>.</p><figure class="figure" id="fig1"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1.jpg" alt="" height="459"><ol class="links-for-figure"><li><a class="anchor download-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S0169534710000662-gr1.jpg" target="_blank" download="" title="Download full-size image"><span class="anchor-text">Download full-size image</span></a></li></ol></span><div class="captions"><span><p><span class="label">Figure 1</span>. Linear regression adjustment in the ABC algorithm. In ABC, we repeatedly sample a parameter value, <em>θ</em><sub><em>i</em></sub>, from its prior distribution to simulate a dataset, <em>y</em><sub><em>i</em></sub>, under a model. Then, from the simulated data, we compute the value of a summary statistic, <em>S(y</em><sub><em>i</em></sub><em>)</em>, and compare it with the value of the summary statistic in the observed data, <em>S(y</em><sub><em>0</em></sub><em>)</em>, using a distance measure. If the distance between <em>S(y</em><sub><em>0</em></sub><em>)</em> and <em>S(y</em><sub><em>i</em></sub><em>)</em> is less than <em>ɛ</em> (the so-called ‘tolerance’), the parameter value, <em>θ</em><sub><em>i</em></sub>, is accepted. The plot shows how the accepted values of <em>θ</em><sub><em>i</em></sub> (points in orange) are adjusted according to a linear transform, <em>θ</em><sub><em>i</em></sub><sup><em>*</em></sup><em>= θ</em><sub><em>i</em></sub><em>– b(S(y) – S(y</em><sub><em>0</em></sub><em>))</em> (green arrow), where <em>b</em>
 is the slope of the regression line. After adjustment, the new 
parameter values (green histogram) form a sample from the posterior 
distribution.</p></span></div></figure></div></section><section id="aep-section-id25"><h3>Model improvement</h3><p>Comparing models and evaluating their goodness-of-fit are fundamental steps in the modeling and inference process (<a name="btb4" href="#tb4" class="workspace-trigger">Box 4</a>).
 Model comparison is often based on a decision theoretic framework, the 
objective of which is to choose models receiving high posterior support.
 In ABC studies, the posterior probability of a given model can be 
approximated by the proportion of accepted simulations given the model <a name="bbib9" href="#bib9" class="workspace-trigger">9</a>, <a name="bbib28" href="#bib28" class="workspace-trigger">28</a>, by logistic regression estimates <a name="bbib25" href="#bib25" class="workspace-trigger">25</a>, <a name="bbib67" href="#bib67" class="workspace-trigger">67</a>, or updated through a sequential Monte Carlo algorithm <a name="bbib66" href="#bib66" class="workspace-trigger">66</a>, <a name="bbib68" href="#bib68" class="workspace-trigger">68</a>. When comparing two models, often the Bayes factor is reported <a name="bbib9" href="#bib9" class="workspace-trigger">9</a>, <a name="bbib18" href="#bib18" class="workspace-trigger">18</a>, <a name="bbib69" href="#bib69" class="workspace-trigger">69</a>. In this process, model choice does not imply the selection of a single ‘best’ model <a name="bbib60" href="#bib60" class="workspace-trigger">[60]</a>.
 Different mechanisms could lead to the same data patterns, so several 
models could explain the data equally well. For example, both a weak 
population bottleneck and population subdivision with migration produce 
gene genealogies having long internal branches <a name="bbib70" href="#bib70" class="workspace-trigger">[70]</a>. Thus, both demographic models can provide equally good explanations for an observed pattern of genetic variation <a name="bbib71" href="#bib71" class="workspace-trigger">[71]</a>.
 Instead of focusing on a single model, we should consider the 
plausibility of each alternative model, and eventually weight parameter 
estimates over several models <a name="bbib72" href="#bib72" class="workspace-trigger">[72]</a> (see <a name="btb4" href="#tb4" class="workspace-trigger">Box 4</a>).</p><p>The
 objective of model checking is to understand the ways in which models 
do not fit the data. The current practice in Bayesian data analysis is 
oriented towards graphical checks whereby data simulated from the fitted
 model are compared with the observation through ‘test’ statistics 
(so-called ‘posterior predictive checks’, see <a name="btb4" href="#tb4" class="workspace-trigger">Box 4</a>).
 Specifying the test statistics amounts to deciding which aspects of the
 model are relevant to criticism. Posterior predictive checks were 
applied recently to coalescent models in ABC <a name="bbib16" href="#bib16" class="workspace-trigger">16</a>, <a name="bbib20" href="#bib20" class="workspace-trigger">20</a>, <a name="bbib36" href="#bib36" class="workspace-trigger">36</a>. Another example of model checking is given by Schaffner and colleagues <a name="bbib73" href="#bib73" class="workspace-trigger">[73]</a>,
 who calibrated a model of human evolutionary history to generate data 
that closely resembled empirical data in terms of allele frequencies, 
linkage disequilibrium, and population differentiation.</p></section></section><section id="aep-section-id26"><h2>Choice and dimension of summary statistics</h2><p>Carrying
 out inference based on summary statistics instead of the full dataset 
inevitably implies discarding potentially useful information. More 
specifically, if a summary statistic is not sufficient for the parameter
 of interest, the posterior distribution computed with this statistic 
would not be equal to the posterior distribution computed with the full 
dataset <a name="bbib4" href="#bib4" class="workspace-trigger">[4]</a>. 
Many areas of evolutionary biology focus on developing informative 
statistics. An example of a recently developed statistic is the extended
 haplotype homozygosity that aims to detect ongoing positive selection 
using genomic data <a name="bbib74" href="#bib74" class="workspace-trigger">[74]</a>.
 The choice of summary statistics is crucial, and is closely linked to 
the particular inference questions addressed. In fact, ABC can be 
limited by the availability of informative statistics for any particular
 model parameter <a name="bbib75" href="#bib75" class="workspace-trigger">[75]</a>.
 For example, the number of polymorphic sites is useful in population 
genetics for the estimation of scaled mutation rates, but it says little
 about the demographic history <a name="bbib75" href="#bib75" class="workspace-trigger">[75]</a>. As another example, F<sub>ST</sub>,
 a well-known measure of population differentiation, is informative for 
estimating the migration rate between two populations under a symmetric 
island model <a name="bbib6" href="#bib6" class="workspace-trigger">[6]</a>, but less informative when estimating the migration rate in a model of divergence with migration <a name="bbib76" href="#bib76" class="workspace-trigger">[76]</a>.</p><p>An
 intuitive approach to the lack of sufficient summary statistics in most
 problems is to increase the number of summary statistics, thereby 
increasing the amount of information available to the ABC algorithm <a name="bbib38" href="#bib38" class="workspace-trigger">[38]</a>. However, increasing the number of statistics can reduce the accuracy of the inference <a name="bbib5" href="#bib5" class="workspace-trigger">[5]</a>.
 The need to manipulate large numbers of summary statistics is amplified
 by the ongoing increase in the amount of available data. With hundreds 
of loci, one approach is to average summary statistics over loci. 
Another approach is to use the quantiles and the moments of the summary 
statistic distribution across the loci <a name="bbib18" href="#bib18" class="workspace-trigger">18</a>, <a name="bbib33" href="#bib33" class="workspace-trigger">33</a>, <a name="bbib48" href="#bib48" class="workspace-trigger">48</a> or the allele frequencies themselves <a name="bbib38" href="#bib38" class="workspace-trigger">[38]</a>,
 thereby increasing the dimensionality of the set of summary statistics.
 This might be a concern because ABC algorithms attempt to sample from a
 small, multidimensional sphere around the observed summary statistics, 
and the probability of accepting a simulation decreases exponentially as
 dimensionality increases. As a result, the number of summary statistics
 that can be handled with a reasonable number of simulations is limited 
in any ABC algorithm <a name="bbib5" href="#bib5" class="workspace-trigger">[5]</a>.
 To circumvent this problem, a method for selecting summary statistics 
has been proposed that scores summary statistics according to whether 
their inclusion in the analysis substantially improves the quality of 
inference <a name="bbib77" href="#bib77" class="workspace-trigger">[77]</a>.
 Alternative methods use dimension reduction techniques. Dimension 
reduction approaches are more robust to an increase in the number of 
summary statistics than the standard ABC rejection algorithm, and they 
lead to more accurate estimates. These methods encompass non-linear feed
 forward neural networks <a name="bbib10" href="#bib10" class="workspace-trigger">[10]</a> and partial least square regression <a name="bbib64" href="#bib64" class="workspace-trigger">[64]</a>.
 Neural networks and similar algorithms can also find combinations of 
summary statistics that contain the maximum information about the 
parameters of interest, and so are also useful in the selection of 
summary statistics <a name="bbib10" href="#bib10" class="workspace-trigger">[10]</a>.</p></section><section id="aep-section-id27"><h2>The future of ABC</h2><section id="aep-section-id28"><h3>Inference under complex models</h3><p>ABC
 is extremely flexible and relatively easy to implement, so inference 
can be carried out for many complex models in evolution and ecology as 
long as informative summary statistics are available and simulating data
 under the model is possible. Templeton <a name="bbib56" href="#bib56" class="workspace-trigger">56</a>, <a name="bbib57" href="#bib57" class="workspace-trigger">57</a>
 argues that the interpretation of why a complex model is preferred is 
highly subjective when using computer simulations. With the 
proliferation of highly complex models, Bayesian statisticians and 
evolutionary biologists have made efforts to reduce subjectivity by 
using hierarchical modeling techniques <a name="bbib48" href="#bib48" class="workspace-trigger">48</a>, <a name="bbib59" href="#bib59" class="workspace-trigger">59</a> and information theoretic measures of model selection <a name="bbib78" href="#bib78" class="workspace-trigger">78</a>, <a name="bbib79" href="#bib79" class="workspace-trigger">79</a>.
 Hierarchical models help in the understanding of parameter dependencies
 and remove implicit assumptions that are not foreseen by the modeler. 
With the development of highly structured hierarchical models, new model
 selection methods have come to the fore. These methods provide explicit
 evaluation of model complexity based on information theoretic measures 
such as Akaike's Information Criterion (AIC)<a name="bbib72" href="#bib72" class="workspace-trigger">72</a>, <a name="bbib79" href="#bib79" class="workspace-trigger">79</a>, <a name="bbib80" href="#bib80" class="workspace-trigger">80</a> or the Deviance Information Criterion (DIC)<a name="bbib78" href="#bib78" class="workspace-trigger">[78]</a>,
 which can be readily approximated in ABC approaches. Several aspects of
 Bayesian thinking have yet to be explored in ABC, but subjectivity can 
be reduced via careful application of all steps of Bayesian data 
analysis.</p></section><section id="aep-section-id29"><h3>Future applications</h3><p>The most widespread application of ABC is in making inferences about demographic history and local adaptation <a name="bbib81" href="#bib81" class="workspace-trigger">81</a>, <a name="bbib82" href="#bib82" class="workspace-trigger">82</a>, but applications outside evolutionary biology have already appeared for example in epidemiology <a name="bbib12" href="#bib12" class="workspace-trigger">12</a>, <a name="bbib23" href="#bib23" class="workspace-trigger">23</a>, <a name="bbib83" href="#bib83" class="workspace-trigger">83</a> and in systems biology <a name="bbib13" href="#bib13" class="workspace-trigger">13</a>, <a name="bbib68" href="#bib68" class="workspace-trigger">68</a>, <a name="bbib84" href="#bib84" class="workspace-trigger">84</a>. With the advent of high-throughput genotyping technologies to address evolutionary questions <a name="bbib85" href="#bib85" class="workspace-trigger">85</a>, <a name="bbib86" href="#bib86" class="workspace-trigger">86</a>,
 ABC applications to genome-wide data will arrive very soon. Genome-wide
 approaches present an increased level of complexity, and fast genome 
samplers will become increasingly important when dealing with haplotypic
 diversity and patterns of linkage disequilibrium shaped by meiotic 
processes and natural selection <a name="bbib87" href="#bib87" class="workspace-trigger">[87]</a>.
 In this context, it will become more and more important to develop 
simulation programs that capture some essential features of the 
biological problem, but which can also sample whole genomes. An example 
of ‘trading-off’ high accuracy for decreased run-time is given in the 
estimation of recombination rates from genome-wide haplotypic data based
 on the coalescent and its extensions <a name="bbib88" href="#bib88" class="workspace-trigger">[88]</a>. Various approximations to the coalescent allow for fast simulation of recombining haplotypes <a name="bbib89" href="#bib89" class="workspace-trigger">89</a>, <a name="bbib90" href="#bib90" class="workspace-trigger">90</a>.
 Although the mathematical ‘tricks’ used for improving speed might be 
unrealistic biologically, these models continue to accurately relate 
patterns of linkage disequilibrium to the underlying recombination 
process, which make them appropriate in an ABC inference framework.</p></section><section id="aep-section-id30"><h3>Software for ABC inferences</h3><p>ABC users can base their analysis on simulation programs such as SIMCOAL <a name="bbib92" href="#bib92" class="workspace-trigger">[92]</a>, ms <a name="bbib93" href="#bib93" class="workspace-trigger">[93]</a> or MaCS <a name="bbib91" href="#bib91" class="workspace-trigger">[91]</a>
 and use statistical software to carry out the model-fitting step. More 
recently, specific ABC software have been developed that carry out the 
data simulation as well as the rejection and regression steps such as 
msBayes (<a href="http://msbayes.sourceforge.net/" target="_blank">http://msbayes.sourceforge.net/</a>) <a name="bbib94" href="#bib94" class="workspace-trigger">[94]</a>, DIYABC (<a href="http://www1.montpellier.inra.fr/CBGP/diyabc/" target="_blank">http://www1.montpellier.inra.fr/CBGP/diyabc/</a>) <a name="bbib95" href="#bib95" class="workspace-trigger">[95]</a>, ONeSAMP (<a href="http://genomics.jun.alaska.edu/asp/" target="_blank">http://genomics.jun.alaska.edu/asp/</a>) <a name="bbib96" href="#bib96" class="workspace-trigger">[96]</a>, ABC4F (<a href="http://www-leca.ujf-grenoble.fr/logiciels.htm" target="_blank">http://www-leca.ujf-grenoble.fr/logiciels.htm</a>) <a name="bbib97" href="#bib97" class="workspace-trigger">[97]</a>, PopABC (<a href="http://code.google.com/p/popabc/" target="_blank">http://code.google.com/p/popabc/</a>) <a name="bbib98" href="#bib98" class="workspace-trigger">[98]</a> and 2BAD <a name="bbib99" href="#bib99" class="workspace-trigger">[99]</a>.
 Even though these ABC software packages greatly facilitate the 
inference step of the algorithm, users must check their models. As with 
the direction followed by MCMC programs, we predict that efficient ABC 
programs will be developed to address specific questions. For these 
programs, the accuracy of inferences could also be extensively validated
 by model-checking techniques <a name="bbib100" href="#bib100" class="workspace-trigger">[100]</a>.</p></section></section><section id="aep-section-id31"><h2>Conclusions</h2><p>Biology
 is a complex science, so it is inevitable that the observation of 
biological systems leads us to build complex models. However, the 
apparent ease of using an inference algorithm such as ABC should never 
hide the general difficulties of making inferences under complex models.
 The automatic process of inference is hampered by the model definition 
and model checking steps, which are case-dependent and highly 
user-interactive. ABC is far from being as ‘easy as 123’. Important <em>caveats</em>
 when using ABC algorithms can be summarized in the following points. 
First, credibility intervals obtained from ABC algorithms are 
potentially inflated due to the loss of information implied by the 
partial use of the data. This source of error should not be ignored, and
 users must be cautious about interpretations of their parameter 
estimates. Second, all models are wrong, thus model checking is a way to
 explore and understand differences between model and data, and to 
improve the fit between them <a name="bbib56" href="#bib56" class="workspace-trigger">[56]</a>.
 Third, there can be several models that explain the data equally well, 
and support from the data for one model does not imply that the model is
 true. If models have common parameters, weighting parameter estimates 
over several well-supported models can produce more robust estimates 
than estimates from a single model (<a name="btb4" href="#tb4" class="workspace-trigger">Box 4</a>). With these <em>caveats</em>
 in mind, we argue that ABC is an extremely useful tool to make 
inferences with complex models. We envisage that many aspects of the ABC
 algorithms will be further improved in the near future, such as dealing
 with high-dimensional sets of summary statistics, evaluating model 
complexity, and devising efficient ways for model checking in ABC.</p></section></div><section id="aep-acknowledgment-id32"><h2>Acknowledgements</h2><p>KC
 is funded by a postdoctoral fellowship from the Université Joseph 
Fourier (ABC MSTIC). OF was partially supported by a grant from the 
Agence Nationale de la Recherche (BLAN06-3146282 MAEV). MB and OF 
acknowledge the support of the Complex Systems Institute (IXXI). OEG 
acknowledges support from the EcoChange Project.</p></section></div><div class="related-content-links u-hide-from-md"><button class="button button-anchor" type="button"><span class="button-text">Recommended articles</span></button><button class="button button-anchor" type="button"><span class="button-text">Citing articles (<!-- -->436<!-- -->)</span></button></div><div class="Tail"></div><section class="bibliography" id="aep-bibliography-id33"><h2 class="section-title">References</h2><section class="bibliography-sec" id="aep-bibliography-sec-id34"><dl class="references"><dt class="label"><a href="#bbib1" id="ref-id-bib1">1</a></dt><dd class="reference"><div class="contribution">J.C. Avise<strong class="title">Molecular Markers, Natural History and Evolution</strong></div><div class="host"> (2nd edn), Sinauer Associates (2004)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib2" id="ref-id-bib2">2</a></dt><dd class="reference"><div class="contribution">B.A. Beaumont, B. Rannala<strong class="title">The Bayesian revolution in genetics</strong></div><div class="host">Nat. Rev. Genet., 5 (2004), pp. 251-261</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib3" id="ref-id-bib3">3</a></dt><dd class="reference"><div class="contribution">M.K. Kuhner<strong class="title">Coalescent genealogy samplers: windows into population history</strong></div><div class="host">Trends Ecol. Evol., 24 (2009), pp. 86-93</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib4" id="ref-id-bib4">4</a></dt><dd class="reference"><div class="contribution">P. Marjoram, S. Tavaré<strong class="title">Modern computational approaches for analysing molecular genetic variation data</strong></div><div class="host">Nat. Rev. Genet., 7 (2006), pp. 759-770</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib5" id="ref-id-bib5">5</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont, <em> et al.</em><strong class="title">Approximate Bayesian Computation in population genetics</strong></div><div class="host">Genetics, 162 (2002), pp. 2025-2035</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib6" id="ref-id-bib6">6</a></dt><dd class="reference"><div class="contribution">S. Wright<strong class="title">The genetical structure of populations</strong></div><div class="host">Ann. Eugen., 15 (1951), pp. 323-354</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib7" id="ref-id-bib7">7</a></dt><dd class="reference"><div class="contribution">L.L. Cavalli-Sforza, G. Zei<strong class="title">Experiments with an artificial population</strong></div><div class="host">J.F. Crow, J.V. Neel (Eds.), Proceedings of the Third International Congress of Human Genetics, John Hopkins Press (1967), pp. 473-478</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib8" id="ref-id-bib8">8</a></dt><dd class="reference"><div class="contribution">S. Tavaré, <em> et al.</em><strong class="title">Inferring coalescence times from DNA sequence data</strong></div><div class="host">Genetics, 145 (1997), pp. 505-518</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib9" id="ref-id-bib9">9</a></dt><dd class="reference"><div class="contribution">J.K. Pritchard, <em> et al.</em><strong class="title">Population growth of human Y chromosomes: a study of Y chromosome microsatellites</strong></div><div class="host">Mol. Biol. Evol., 16 (1999), pp. 1791-1798</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib10" id="ref-id-bib10">10</a></dt><dd class="reference"><div class="contribution">M.G.B. Blum, O. François<strong class="title">Non-linear regression models for Approximate Bayesian Computation</strong></div><div class="host">Stat. Comput., 20 (2010), pp. 63-73</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib11" id="ref-id-bib11">11</a></dt><dd class="reference"><div class="contribution">P. Marjoram, <em> et al.</em><strong class="title">Markov chain Monte Carlo without likelihoods</strong></div><div class="host">Proc. Natl. Acad. Sci. U. S. A., 100 (2003), pp. 15324-15328</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib12" id="ref-id-bib12">12</a></dt><dd class="reference"><div class="contribution">S.A. Sisson, <em> et al.</em><strong class="title">Sequential Monte Carlo without likelihoods</strong></div><div class="host">Proc. Natl. Acad. Sci. U. S. A., 104 (2007), pp. 1760-1765</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib13" id="ref-id-bib13">13</a></dt><dd class="reference"><div class="contribution">T. Toni, <em> et al.</em><strong class="title">Approximate Bayesian computation scheme for parameter inference and model selection in dynamical systems</strong></div><div class="host">J. R. Soc. Interface, 6 (2009), pp. 187-202</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib14" id="ref-id-bib14">14</a></dt><dd class="reference"><div class="contribution">D.A. Tallmon, <em> et al.</em><strong class="title">Comparative evaluation of a new effective population size estimator based on approximate Bayesian computation</strong></div><div class="host">Genetics, 167 (2004), pp. 977-988</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib15" id="ref-id-bib15">15</a></dt><dd class="reference"><div class="contribution">Y.L. Chan, <em> et al.</em><strong class="title">Bayesian estimation of the timing and severity of a population bottleneck from ancient DNA</strong></div><div class="host">PLoS Genet., 2 (2006), p. e59</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib16" id="ref-id-bib16">16</a></dt><dd class="reference"><div class="contribution">K.R. Thornton, P. Andolfatto<strong class="title">Approximate Bayesian inference reveals evidence for a recent, severe, bottleneck in a Netherlands population of <em>Drosophila melanogaster</em></strong></div><div class="host">Genetics, 172 (2006), pp. 1607-1619</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib17" id="ref-id-bib17">17</a></dt><dd class="reference"><div class="contribution">M. Pascual, <em> et al.</em><strong class="title">Introduction history of Drosophila subobscura in the New World: a microsatellite based survey using ABC methods</strong></div><div class="host">Mol. Ecol., 16 (2007), pp. 3069-3083</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib18" id="ref-id-bib18">18</a></dt><dd class="reference"><div class="contribution">O. François, <em> et al.</em><strong class="title">Demographic history of European populations of Arabidopsis thaliana</strong></div><div class="host">PLoS Genet., 4 (2008), p. e1000075</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib19" id="ref-id-bib19">19</a></dt><dd class="reference"><div class="contribution">J. Ross-Ibarra, <em> et al.</em><strong class="title">Patterns of polymorphism and demographic history in natural populations of Arabidopsis lyrata</strong></div><div class="host">PLoS ONE, 3 (2008), p. e2411</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib20" id="ref-id-bib20">20</a></dt><dd class="reference"><div class="contribution">P.K. Ingvarsson<strong class="title">Multilocus patterns of nucleotide polymorphism and the demographic history of Populus tremula</strong></div><div class="host">Genetics, 180 (2008), pp. 329-340</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib21" id="ref-id-bib21">21</a></dt><dd class="reference"><div class="contribution">L.Z. Gao, H. Innan<strong class="title">Non-independent
 domestication of the two rice subspecies, Oryza sativa subsp. indica 
and subsp. japonica, demonstrated by multilocus microsatellites</strong></div><div class="host">Genetics, 179 (2008), pp. 965-976</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib22" id="ref-id-bib22">22</a></dt><dd class="reference"><div class="contribution">T. Guillemaud, <em> et al.</em><strong class="title">Inferring introduction routes of invasive species using approximate Bayesian computation on microsatellite data</strong></div><div class="host">Heredity, 104 (2010), pp. 88-99</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib23" id="ref-id-bib23">23</a></dt><dd class="reference"><div class="contribution">M. Tanaka, <em> et al.</em><strong class="title">Estimating tuberculosis transmission parameters from genotype data using approximate Bayesian computation</strong></div><div class="host">Genetics, 173 (2006), pp. 1511-1520</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib24" id="ref-id-bib24">24</a></dt><dd class="reference"><div class="contribution">D. Shriner, <em> et al.</em><strong class="title">Evolution of intrahost HIV-1 genetic diversity during chronic infection</strong></div><div class="host">Evolution, 60 (2006), pp. 1165-1176</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib25" id="ref-id-bib25">25</a></dt><dd class="reference"><div class="contribution">N.J.R. Fagundes, <em> et al.</em><strong class="title">Statistical evaluation of alternative models of human evolution</strong></div><div class="host">Proc. Natl. Acad. Sci. U. S. A., 104 (2007), pp. 17614-17619</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib26" id="ref-id-bib26">26</a></dt><dd class="reference"><div class="contribution">M.P. Cox, <em> et al.</em><strong class="title">Testing
 for archaic hominin admixture on the X chromosome: model likelihoods 
for the modern human RRM2P4 region from summaries of genealogical 
topology under the structured coalescent</strong></div><div class="host">Genetics, 178 (2008), pp. 427-437</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib27" id="ref-id-bib27">27</a></dt><dd class="reference"><div class="contribution">P. Gerbault, <em> et al.</em><strong class="title">Impact of selection and demography on the diffusion of lactase persistence</strong></div><div class="host">PLoS ONE, 4 (2009), p. e6369</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib28" id="ref-id-bib28">28</a></dt><dd class="reference"><div class="contribution">E. Patin, <em> et al.</em><strong class="title">Inferring the demographic history of African farmers and Pygmy hunter–gatherers using a multilocus resequencing data set</strong></div><div class="host">PLoS Genet., 5 (2009), p. e1000448</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib29" id="ref-id-bib29">29</a></dt><dd class="reference"><div class="contribution">P. Verdu, <em> et al.</em><strong class="title">Origins and genetic diversity of Pygmy hunter-gatherers from western Central Africa</strong></div><div class="host">Curr. Biol., 19 (2009), pp. 312-318</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib30" id="ref-id-bib30">30</a></dt><dd class="reference"><div class="contribution">M. Bonhomme, <em> et al.</em><strong class="title">Origin and number of founders in an introduced insular primate: estimation from nuclear genetic data</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 1009-1019</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib31" id="ref-id-bib31">31</a></dt><dd class="reference"><div class="contribution">A. Estoup, <em> et al.</em><strong class="title">Genetic analysis of complex demographic scenarios: spatially expanding populations of the cane toad</strong></div><div class="host">Bufo marinus. Evolution, 58 (2004), pp. 2021-2036</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib32" id="ref-id-bib32">32</a></dt><dd class="reference"><div class="contribution">N. Miller, <em> et al.</em><strong class="title">Multiple transatlantic introductions of the Western corn rootworm</strong></div><div class="host">Science, 310 (2005), p. 992</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib33" id="ref-id-bib33">33</a></dt><dd class="reference"><div class="contribution">E.B. Rosenblum, <em> et al.</em><strong class="title">A multilocus perspective on colonization accompanied by selection and gene flow</strong></div><div class="host">Evolution, 61 (2007), pp. 2971-2985</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib34" id="ref-id-bib34">34</a></dt><dd class="reference"><div class="contribution">S. Neuenschwander, <em> et al.</em><strong class="title">Colonization
 history of the Swiss Rhine basin by the bullhead (Cottus gobio): 
inference under a Bayesian spatially explicit framework</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 757-772</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib35" id="ref-id-bib35">35</a></dt><dd class="reference"><div class="contribution">N. Ray, <em> et al.</em><strong class="title">A
 statistical evaluation of models for the initial settlement of the 
American continent emphasizes the importance of gene flow with Asia</strong></div><div class="host">Mol. Biol. Evol., 27 (2010), pp. 337-345</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib36" id="ref-id-bib36">36</a></dt><dd class="reference"><div class="contribution">S. Ghirotto, <em> et al.</em><strong class="title">Inferring genealogical processes from patterns of bronze-age and modern DNA variation in Sardinia</strong></div><div class="host">Mol. Biol. Evol., 27 (2010), pp. 875-886</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib37" id="ref-id-bib37">37</a></dt><dd class="reference"><div class="contribution">L. Excoffier, <em> et al.</em><strong class="title">Bayesian analysis of an admixture model with mutations and arbitrarily linked markers</strong></div><div class="host">Genetics, 169 (2005), pp. 1727-1738</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib38" id="ref-id-bib38">38</a></dt><dd class="reference"><div class="contribution">V.C. Sousa, <em> et al.</em><strong class="title">Approximate Bayesian computation without summary statistics: the case of admixture</strong></div><div class="host">Genetics, 181 (2009), pp. 1507-1519</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib39" id="ref-id-bib39">39</a></dt><dd class="reference"><div class="contribution">J.M. Cornuet, <em> et al.</em><strong class="title">Bayesian
 inference under complex evolutionary scenarios using microsatellite 
markers: multiple divergence and genetic admixture events in the honey 
bee Apis mellifera</strong></div><div class="host">C.L. Mahoney, D.A. Springer (Eds.), Genetic diversity, Nova Science Publishers Inc (2009), pp. 229-246</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib40" id="ref-id-bib40">40</a></dt><dd class="reference"><div class="contribution">G. Hamilton, <em> et al.</em><strong class="title">Bayesian estimation of recent migration rates after a spatial expansion</strong></div><div class="host">Genetics, 170 (2005), pp. 409-417</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib41" id="ref-id-bib41">41</a></dt><dd class="reference"><div class="contribution">J.S. Lopes, S. Boessenkool<strong class="title">The use of approximate Bayesian computation in conservation genetics and its application in a case study on yellow-eyed penguins</strong></div><div class="host">Conserv. Genet, 11 (2010), pp. 421-433</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib42" id="ref-id-bib42">42</a></dt><dd class="reference"><div class="contribution">I. Tiemann-Boege, <em> et al.</em><strong class="title">High-resolution recombination patterns in a region of human chromosome 21 measured by sperm typing</strong></div><div class="host">PLoS Genet., 2 (2006), p. e70</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib43" id="ref-id-bib43">43</a></dt><dd class="reference"><div class="contribution">B. Padhukasahasram, <em> et al.</em><strong class="title">Estimating recombination rates from single-nucleotide polymorphisms using summary statistics</strong></div><div class="host">Genetics, 174 (2006), pp. 1517-1528</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib44" id="ref-id-bib44">44</a></dt><dd class="reference"><div class="contribution">M. Touchon, <em> et al.</em><strong class="title">Organised genome dynamics in the Escherichia coli species results in highly diverse adaptive paths</strong></div><div class="host">PLoS Genet., 5 (2009), p. e1000344</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib45" id="ref-id-bib45">45</a></dt><dd class="reference"><div class="contribution">J.D. Jensen, <em> et al.</em><strong class="title">An approximate Bayesian estimator suggests strong, recurrent selective sweeps in Drosophila</strong></div><div class="host">PLoS Genet., 4 (2008), p. e1000198</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib46" id="ref-id-bib46">46</a></dt><dd class="reference"><div class="contribution">H. Quach, <em> et al.</em><strong class="title">Signatures of purifying and local positive selection in human miRNAs</strong></div><div class="host">Am. J. Hum. Genet., 84 (2009), pp. 316-327</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib47" id="ref-id-bib47">47</a></dt><dd class="reference"><div class="contribution">Y. Itan, <em> et al.</em><strong class="title">The origins of lactase persistence in Europe</strong></div><div class="host">PLoS Comput. Biol., 5 (2009), p. e1000491</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib48" id="ref-id-bib48">48</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, <em> et al.</em><strong class="title">Test for simultaneous divergence using approximate Bayesian computation</strong></div><div class="host">Evolution, 60 (2006), pp. 2435-2453</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib49" id="ref-id-bib49">49</a></dt><dd class="reference"><div class="contribution">C. Becquet, M. Przeworski<strong class="title">A new approach to estimate parameters of speciation models with application to apes</strong></div><div class="host">Genome Res., 17 (2007), pp. 1505-1519</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib50" id="ref-id-bib50">50</a></dt><dd class="reference"><div class="contribution">A. Leaché, <em> et al.</em><strong class="title">Two waves of diversification in mammals and reptiles of Baja California revealed by hierarchical Bayesian analysis</strong></div><div class="host">Biol. Lett., 3 (2007), pp. 646-650</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib51" id="ref-id-bib51">51</a></dt><dd class="reference"><div class="contribution">A.S. Putnam, <em> et al.</em><strong class="title">Discordant divergence times among Z chromosome regions between two ecologically distinct swallowtail butterfly species</strong></div><div class="host">Evolution, 61 (2007), pp. 912-927</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib52" id="ref-id-bib52">52</a></dt><dd class="reference"><div class="contribution">R.D. Wilkinson, S. Tavaré<strong class="title">Estimating primate divergence times by using conditioned birth-and-death processes</strong></div><div class="host">Theor. Popul. Biol., 75 (2009), pp. 278-285</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib53" id="ref-id-bib53">53</a></dt><dd class="reference"><div class="contribution">M. Jakobsson, <em> et al.</em><strong class="title">A recent unique origin of the allotetraploid species Arabidopsis suecica: evidence from nuclear DNA markers</strong></div><div class="host">Mol. Biol. Evol., 23 (2006), pp. 1217-1231</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib54" id="ref-id-bib54">54</a></dt><dd class="reference"><div class="contribution">A.C. Carnaval, <em> et al.</em><strong class="title">Stability predicts genetic diversity in the Brazilian Atlantic forest hotspot</strong></div><div class="host">Science, 323 (2009), pp. 785-789</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib55" id="ref-id-bib55">55</a></dt><dd class="reference"><div class="contribution">F. Jabot, J. Chave<strong class="title">Inferring
 the parameters of the neutral theory of biodiversity using phylogenetic
 information, and implications for tropical forests</strong></div><div class="host">Ecol. Lett., 12 (2009), pp. 239-248</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib56" id="ref-id-bib56">56</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Statistical hypothesis testing in intraspecific phylogeography: NCPA versus ABC</strong></div><div class="host">Mol. Ecol., 18 (2009), pp. 319-331</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib57" id="ref-id-bib57">57</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Coalescent-based, maximum likelihood inference in phylogeography</strong></div><div class="host">Mol. Ecol., 19 (2010), pp. 431-435</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib58" id="ref-id-bib58">58</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont, <em> et al.</em><strong class="title">In defence of model-based inference in phylogeography</strong></div><div class="host">Mol. Ecol., 19 (2010), pp. 436-446</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib59" id="ref-id-bib59">59</a></dt><dd class="reference"><div class="contribution">A. Gelman, <em> et al.</em><strong class="title">Bayesian Data Analysis</strong></div><div class="host"> (2nd edn), Chapman and Hall (2004)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib60" id="ref-id-bib60">60</a></dt><dd class="reference"><div class="contribution">B.D. Ripley<strong class="title">Selecting amongst large classes of models</strong></div><div class="host">N. Adams, <em> et al.</em> (Eds.), Methods and Models in Statistics: In Honour of Professor John Nelder, FRS, Imperial College Press (2004), pp. 155-170</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib61" id="ref-id-bib61">61</a></dt><dd class="reference"><div class="contribution">A. Purvis, <em> et al.</em><strong class="title">Predicting extinction risk in declining species</strong></div><div class="host">Proc. Biol. Sci., 267 (2000), pp. 1947-1952</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib62" id="ref-id-bib62">62</a></dt><dd class="reference"><div class="contribution">R.M. May<strong class="title">Uses and abuses of mathematics in biology</strong></div><div class="host">Science, 303 (2004), pp. 790-793</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib63" id="ref-id-bib63">63</a></dt><dd class="reference"><div class="contribution">C. Leuenberger, D. Wegmann<strong class="title">Bayesian computation and model selection without likelihoods</strong></div><div class="host">Genetics, 184 (2010), pp. 243-252</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib64" id="ref-id-bib64">64</a></dt><dd class="reference"><div class="contribution">D. Wegmann, <em> et al.</em><strong class="title">Efficient Approximate Bayesian Computation coupled with Markov chain Monte Carlo without likelihood</strong></div><div class="host">Genetics, 182 (2009), pp. 1207-1218</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib65" id="ref-id-bib65">65</a></dt><dd class="reference"><div class="contribution">J.S. Liu<strong class="title">Monte Carlo Strategies in Scientific Computing</strong></div><div class="host">Springer (2001)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib66" id="ref-id-bib66">66</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont, <em> et al.</em><strong class="title">Adaptive approximate Bayesian computation</strong></div><div class="host">Biometrika, 96 (2009), pp. 983-990</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib67" id="ref-id-bib67">67</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont<strong class="title">Joint determination of topology, divergence time, and immigration in population trees</strong></div><div class="host">S. Matsumura, <em> et al.</em> (Eds.), Simulation, Genetics, and Human Prehistory, McDonald Institute for Archaeological Research (2008), pp. 135-154</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib68" id="ref-id-bib68">68</a></dt><dd class="reference"><div class="contribution">T. Toni, M.P.H. Stumpf<strong class="title">Simulation-based model selection for dynamical systems in systems and population biology</strong></div><div class="host">Bioinformatics, 26 (2010), pp. 104-110</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib69" id="ref-id-bib69">69</a></dt><dd class="reference"><div class="contribution">A. Grelaud, <em> et al.</em><strong class="title">ABC likelihood-free methods for model choice in Gibbs random fields</strong></div><div class="host">Bayesian Anal., 4 (2009), pp. 317-336</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib70" id="ref-id-bib70">70</a></dt><dd class="reference"><div class="contribution">J. Hein, <em> et al.</em><strong class="title">Gene Genealogies, Variation, and Evolution</strong></div><div class="host">Oxford University Press (2005)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib71" id="ref-id-bib71">71</a></dt><dd class="reference"><div class="contribution">R. Nielsen, M.A. Beaumont<strong class="title">Statistical inferences in phylogeography</strong></div><div class="host">Mol. Ecol., 18 (2009), pp. 1034-1047</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib72" id="ref-id-bib72">72</a></dt><dd class="reference"><div class="contribution">J.B. Johnson, K.S. Omland<strong class="title">Model selection in ecology and evolution</strong></div><div class="host">Trends Ecol. Evol., 19 (2004), pp. 101-108</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib73" id="ref-id-bib73">73</a></dt><dd class="reference"><div class="contribution">S.F. Schaffner, <em> et al.</em><strong class="title">Calibrating a coalescent simulation of human genome sequence variation</strong></div><div class="host">Genome Res., 15 (2005), pp. 1576-1583</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib74" id="ref-id-bib74">74</a></dt><dd class="reference"><div class="contribution">P.C. Sabeti, <em> et al.</em><strong class="title">Detecting recent positive selection in the human genome from haplotype structure</strong></div><div class="host">Nature, 419 (2002), pp. 832-837</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib75" id="ref-id-bib75">75</a></dt><dd class="reference"><div class="contribution">J. Hey, C.A. Machado<strong class="title">The study of structured populations - new hope for a difficult and divided science</strong></div><div class="host">Nat. Rev. Genet., 4 (2003), pp. 535-543</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib76" id="ref-id-bib76">76</a></dt><dd class="reference"><div class="contribution">R. Nielsen, J. Wakeley<strong class="title">Distinguishing migration from isolation: a Markov chain Monte Carlo approach</strong></div><div class="host">Genetics, 158 (2001), pp. 885-896</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib77" id="ref-id-bib77">77</a></dt><dd class="reference"><div class="contribution">P. Joyce, P. Marjoram<strong class="title">Approximately sufficient statistics and Bayesian computation</strong></div><div class="host">Stat. Appl. Genet. Mol. Biol., 7 (2008)</div><div class="comment">26</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib78" id="ref-id-bib78">78</a></dt><dd class="reference"><div class="contribution">D.J. Spiegelhalter, <em> et al.</em><strong class="title">Bayesian measures of model complexity and fit</strong></div><div class="host">J. R. Stat. Soc. Series B Stat. Methodol., 64 (2002), pp. 583-639</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib79" id="ref-id-bib79">79</a></dt><dd class="reference"><div class="contribution">B.C. Carstens, <em> et al.</em><strong class="title">An information-theoretical approach to phylogeography</strong></div><div class="host">Mol. Ecol., 18 (2009), pp. 4270-4282</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib80" id="ref-id-bib80">80</a></dt><dd class="reference"><div class="contribution">H. Akaike<strong class="title">New look at statistical model identification</strong></div><div class="host">IEEE Trans. Automat. Control, 19 (1974), pp. 716-723</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib81" id="ref-id-bib81">81</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, <em> et al.</em><strong class="title">Phylogeography's past, present, and future: 10 years after Avise, 2000</strong></div><div class="host">Mol. Phylogenet. Evol., 54 (2010), pp. 291-301</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib82" id="ref-id-bib82">82</a></dt><dd class="reference"><div class="contribution">G. Segelbacher, <em> et al.</em><strong class="title">Applications of landscape genetics in conservation biology: concepts and challenges</strong></div><div class="host">Conserv. Genet., 11 (2001), pp. 375-385</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib83" id="ref-id-bib83">83</a></dt><dd class="reference"><div class="contribution">J.S. Lopes, M.A. Beaumont<strong class="title">ABC: A useful Bayesian tool for the analysis of population data</strong></div><div class="host">Infect. Genet. Evol (2009), <a href="https://doi.org/10.1016/j.meegid.2009.10.010" target="_blank">10.1016/j.meegid.2009.10.010</a></div><div class="host"><a href="http://ees.elsevier.com/meegid/" target="_blank">http://ees.elsevier.com/meegid/</a></div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib84" id="ref-id-bib84">84</a></dt><dd class="reference"><div class="contribution">O. Ratmann, <em> et al.</em><strong class="title">Model criticism based on likelihood-free inference</strong></div><div class="host">Proc. Natl. Acad. Sci. U. S. A., 106 (2009), pp. 10576-10581</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib85" id="ref-id-bib85">85</a></dt><dd class="reference"><div class="contribution">M. Jakobsson, <em> et al.</em><strong class="title">Genotype, haplotype, and copy number variation in worldwide human populations</strong></div><div class="host">Nature, 451 (2008), pp. 998-1003</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib86" id="ref-id-bib86">86</a></dt><dd class="reference"><div class="contribution">J. Li, <em> et al.</em><strong class="title">Worldwide human relationships inferred from genome-wide patterns of variation</strong></div><div class="host">Science, 319 (2008), pp. 1100-1104</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib87" id="ref-id-bib87">87</a></dt><dd class="reference"><div class="contribution">C.J. Hoggart, <em> et al.</em><strong class="title">Sequence-level population simulations over large genomic regions</strong></div><div class="host">Genetics, 177 (2007), pp. 1725-1731</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib88" id="ref-id-bib88">88</a></dt><dd class="reference"><div class="contribution">R.R. Hudson<strong class="title">Generating samples under a Wright-Fisher neutral model</strong></div><div class="host">Bioinformatics, 18 (2002), pp. 337-338</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib89" id="ref-id-bib89">89</a></dt><dd class="reference"><div class="contribution">C. Wiuf, J. Hein<strong class="title">Recombination as a point process along sequences</strong></div><div class="host">Theor. Popul. Biol., 55 (1999), pp. 248-259</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib90" id="ref-id-bib90">90</a></dt><dd class="reference"><div class="contribution">P. Marjoram, J. Wall<strong class="title">Fast “coalescent” simulation</strong></div><div class="host">BMC Genet., 7 (2006)</div><div class="comment">16</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib91" id="ref-id-bib91">91</a></dt><dd class="reference"><div class="contribution">G.K. Chen, <em> et al.</em><strong class="title">Fast and flexible simulation of DNA sequence data</strong></div><div class="host">Genome Res., 19 (2009), pp. 136-142</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib92" id="ref-id-bib92">92</a></dt><dd class="reference"><div class="contribution">G. Laval, L. Excoffier<strong class="title">SIMCOAL
 2.0: a program to simulate genomic diversity over large recombining 
regions in a subdivided population with a complex history</strong></div><div class="host">Bioinformatics, 20 (2004), pp. 2485-3248</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib93" id="ref-id-bib93">93</a></dt><dd class="reference"><div class="contribution">R.R. Hudson<strong class="title">Properties of a neutral allele model with intragenic recombination</strong></div><div class="host">Theor. Popul. Biol., 23 (1983), pp. 183-201</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib94" id="ref-id-bib94">94</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, <em> et al.</em><strong class="title">msBayes: pipeline for testing comparative phylogeographic histories using hierarchical approximate Bayesian computation</strong></div><div class="host">BMC Bioinformatics, 8 (2007)</div><div class="comment">268</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib95" id="ref-id-bib95">95</a></dt><dd class="reference"><div class="contribution">J.M. Cornuet, <em> et al.</em><strong class="title">Inferring population history with DIY ABC: a user-friendly approach to Approximate Bayesian Computation</strong></div><div class="host">Bioinformatics, 24 (2008), pp. 2713-2719</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib96" id="ref-id-bib96">96</a></dt><dd class="reference"><div class="contribution">D.A. Tallmon, <em> et al.</em><strong class="title">ONeSAMP: a program to estimate effective population size using approximate Bayesian computation</strong></div><div class="host">Mol. Ecol. Resour., 8 (2008), pp. 299-301</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib97" id="ref-id-bib97">97</a></dt><dd class="reference"><div class="contribution">M. Foll, <em> et al.</em><strong class="title">An
 approximate Bayesian computation approach to overcome biases that arise
 when using amplified fragment length polymorphism markers to study 
population structure</strong></div><div class="host">Genetics, 179 (2008), pp. 927-939</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib98" id="ref-id-bib98">98</a></dt><dd class="reference"><div class="contribution">J. Lopes, <em> et al.</em><strong class="title"><em>PopABC</em>: a program to infer historical demographic parameters</strong></div><div class="host">Bioinformatics, 25 (2009), pp. 2747-2749</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib99" id="ref-id-bib99">99</a></dt><dd class="reference"><div class="contribution">T.C. Bray, <em> et al.</em><strong class="title">2BAD: an application to estimate the parental contributions during two independent admixture events</strong></div><div class="host">Mol. Ecol. Resour., 10 (2009), pp. 538-541</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib100" id="ref-id-bib100">100</a></dt><dd class="reference"><div class="contribution">S. Cook, <em> et al.</em><strong class="title">Validation of software for Bayesian models using posterior quantiles</strong></div><div class="host">J. Comput. Graph. Stat., 15 (2006), pp. 675-692</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib101" id="ref-id-bib101">101</a></dt><dd class="reference"><div class="contribution">D. Lachaise, <em> et al.</em><strong class="title">Historical biogeography of the <em>Drosophila-melanogaster</em> species subgroup</strong></div><div class="host">Evol. Biol., 22 (1988), pp. 159-225</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib102" id="ref-id-bib102">102</a></dt><dd class="reference"><div class="contribution">S.P. Hubbell<strong class="title">The Unified Neutral Theory of Biodiversity and Biogeography</strong></div><div class="host">Princeton University Press (2001)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib103" id="ref-id-bib103">103</a></dt><dd class="reference"><div class="contribution">D. Alonso, <em> et al.</em><strong class="title">The merits of neutral theory</strong></div><div class="host">Trends Ecol. Evol., 21 (2006), pp. 451-457</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib104" id="ref-id-bib104">104</a></dt><dd class="reference"><div class="contribution">J.C. Garza, E.G. Williamson<strong class="title">Detection of reduction in population size using data from microsatellite loci</strong></div><div class="host">Mol. Ecol., 10 (2001), pp. 305-318</div><div class="ReferenceLinks" id="reference-links"></div></dd></dl></section></section><div id="aep-glossary-id35" class="Glossary"><h2 class="section-title">Glossary</h2><div class="glossary-section" id="aep-glossary-sec-id36"><dl><div><dt>Bayes factor</dt><dd class="u-margin-xxl-left">the
 ratio of probabilities of two models that is used to evaluate the 
relative support of one model in relation to another in Bayesian model 
comparison.</dd></div><div><dt>Bayesian statistics</dt><dd class="u-margin-xxl-left">a
 general framework for summarizing uncertainty (prior information) and 
making estimates and predictions using probability statements 
conditional on observed data and an assumed model.</dd></div><div><dt>Coalescent theory</dt><dd class="u-margin-xxl-left">a
 mathematical theory that describes the ancestral relationships of a 
sample of ‘individuals’ back to their common ancestor. Individuals may 
represent molecular marker loci, genes, or chromosomes depending on the 
context.</dd></div><div><dt>Credible interval</dt><dd class="u-margin-xxl-left">a
 posterior probability interval used in Bayesian statistics that can be 
directly constructed from the posterior distribution. For example, a 95%
 credible interval for the parameter θ means that the posterior 
probability that <em>θ</em> lies in the interval is 0.95.</dd></div><div><dt>Deviance Information Criterion (DIC)</dt><dd class="u-margin-xxl-left">an
 information theoretic measure used to determine if improvement in model
 fit justifies the use of a more complex model whereby model complexity 
is expressed with a quantity related to the number of parameters.</dd></div><div><dt>Dimension reduction</dt><dd class="u-margin-xxl-left">the
 mathematical process of transforming a number of possibly correlated 
variables into a smaller number of variables. A well-known example of 
such methods is principal component analysis, but other methods such as 
feed-forward neural networks (FFNNs) or partial least-squared regression
 (PLS) are used in the analysis of complex genetic data. FFNNs are 
flexible non-linear regression models. PLS is a regression method for 
constructing predictive models in the presence of many factors.</dd></div><div><dt>Effective population size (<em>N</em><sub><em>e</em></sub>)</dt><dd class="u-margin-xxl-left">the size of an idealized Wright–Fisher population that has the same level of genetic drift as the population in question.</dd></div><div><dt>Hierarchical models</dt><dd class="u-margin-xxl-left">models
 in which the parameters of prior distributions are estimated from data 
rather than using subjective information. Hierarchical models are 
central to modern Bayesian statistics and allow an objective approach to
 inference.</dd></div><div><dt>High-throughput genotyping</dt><dd class="u-margin-xxl-left">a
 common name for recently developed, next-generation sequencing 
technologies that provide massively parallel sequencing at low cost and 
without the requirement for large, automated facilities.</dd></div><div><dt>Markov Chain Monte Carlo (MCMC)</dt><dd class="u-margin-xxl-left">an
 iterative Bayesian statistical technique that generates samples from 
the posterior distribution. Well-designed MCMC algorithms converge to 
the posterior distribution, which is independent of the starting 
position.</dd></div><div><dt>Posterior distribution</dt><dd class="u-margin-xxl-left">the
 conditional distribution of the parameter given the data, which is 
proportional to the product of the likelihood and the prior 
distribution.</dd></div><div><dt>Posterior predictive distribution</dt><dd class="u-margin-xxl-left">the distribution of future observations conditional on the observed data.</dd></div><div><dt>Prior distribution</dt><dd class="u-margin-xxl-left">the distribution of parameter values before any data are examined.</dd></div><div><dt>Statistical phylogeography</dt><dd class="u-margin-xxl-left">an
 interdisciplinary field that aims to understand the processes 
underlying the spatial and temporal dimensions of genetic variation by 
combining information from genetic, ecological and paleontological data.</dd></div><div><dt>Sufficient statistics</dt><dd class="u-margin-xxl-left">a statistic is sufficient for a parameter, <em>θ</em>, if the probability of the data, given the statistic and <em>θ</em>, does not depend on <em>θ</em>. In other words, a sufficient statistic provides just as much information to estimate <em>θ</em> as would the full dataset.</dd></div></dl></div></div><div class="Copyright"><span class="copyright-line">Copyright © 2010 Elsevier Ltd. All rights reserved.</span></div></article><div class="u-show-from-md col-lg-6 col-md-8 pad-right"><aside class="RelatedContent" aria-label="Related content"><section class="SidePanel"><header id="recommended-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Recommended articles</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S0305440314002593"><h3 class="article-title ellipsis" id="recommended-articles-article0-title" title="An Approximate Bayesian Computation approach for inferring patterns of cultural evolutionary change"><span>An Approximate Bayesian Computation approach for inferring patterns of cultural evolutionary change</span></h3></a><div class="article-source ellipsis"><div class="source">Journal of Archaeological Science, Volume 50, 2014, pp. 160-170</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S0305440314002593/pdfft?md5=f77747f67dcb53087ca5ee5a33f6901f&amp;pid=1-s2.0-S0305440314002593-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article0-title" aria-controls="recommended-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S0022249612000272"><h3 class="article-title ellipsis" id="recommended-articles-article1-title" title="A tutorial on approximate Bayesian computation"><span>A tutorial on approximate Bayesian computation</span></h3></a><div class="article-source ellipsis"><div class="source">Journal of Mathematical Psychology, Volume 56, Issue 2, 2012, pp. 69-85</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S0022249612000272/pdfft?md5=17738b6e86bb20598b1a9fb0cc1e42c7&amp;pid=1-s2.0-S0022249612000272-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article1-title" aria-controls="recommended-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1567134809002251"><h3 class="article-title ellipsis" id="recommended-articles-article2-title" title="ABC: A useful Bayesian tool for the analysis of population data"><span>ABC: A useful Bayesian tool for the analysis of population data</span></h3></a><div class="article-source ellipsis"><div class="source">Infection, Genetics and Evolution, Volume 10, Issue 6, 2010, pp. 825-832</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1567134809002251/pdfft?md5=5683b81ec43da61b1b08e4418fc9d398&amp;pid=1-s2.0-S1567134809002251-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article2-title" aria-controls="recommended-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article2" aria-hidden="true"></div></li></ul></div><div class="pagination u-position-relative u-padding-xs-bottom"><span class="u-position-absolute"></span><span class="pagination-pages-label"><span class="pagination-nav u-margin-xs-hor pagination-current underline-page-number">1</span><span class="pagination-nav u-margin-xs-hor">2</span></span><span class="u-position-absolute"><button class="button-link button-link-secondary next-button" type="button"><span class="button-link-text">Next</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></button></span></div></div></section><section class="SidePanel"><header id="citing-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle button-link-primary" aria-expanded="false" type="button"><span class="button-link-text"><h2 class="section-title">Citing articles (436)</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="u-display-none" aria-hidden="true" aria-describedby="citing-articles-header"><div id="citing-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S0959437X17301764" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article0-title" title="Fine-tuning of Approximate Bayesian Computation for human population genomics">Fine-tuning of Approximate Bayesian Computation for human population genomics</h3></a><div class="article-source ellipsis">2018, Current Opinion in Genetics and Development</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S0959437X17301764/pdfft?md5=98115bb28fa0367a96d773c07b9409c6&amp;pid=1-s2.0-S0959437X17301764-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article0-title" aria-controls="citing-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S105579031830109X" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article1-title" title="Forest corridors between the central Andes and the southern Atlantic Forest enabled dispersal and peripatric diversification without niche divergence in a passerine">Forest
 corridors between the central Andes and the southern Atlantic Forest 
enabled dispersal and peripatric diversification without niche 
divergence in a passerine</h3></a><div class="article-source ellipsis">2018, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S105579031830109X/pdfft?md5=5fe367dcdd8b74e68a165828398f3645&amp;pid=1-s2.0-S105579031830109X-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article1-title" aria-controls="citing-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S0265931X18301735" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article2-title" title="Calibration of forest&lt;sup&gt;137&lt;/sup&gt;Cs cycling model ”FoRothCs” via approximate Bayesian computation based on 6-year observations from plantation forests in Fukushima">Calibration
 of forest&lt;sup&gt;137&lt;/sup&gt;Cs cycling model ”FoRothCs” via 
approximate Bayesian computation based on 6-year observations from 
plantation forests in Fukushima</h3></a><div class="article-source ellipsis">2018, Journal of Environmental Radioactivity</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S0265931X18301735/pdfft?md5=63eb672812401a58451a5b28ce82d228&amp;pid=1-s2.0-S0265931X18301735-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article2-title" aria-controls="citing-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article2" aria-hidden="true"></div></li></ul><a class="anchor side-panel-view-more" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-77954816556&amp;md5=46278b191bdcf588a3d141b3c2638a7" target="_blank"><span class="anchor-text">View more articles</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></div></div></section><section class="SidePanel hidden"><header id="metrics-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Article Metrics</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><a href="https://plu.mx/plum/a/?doi=10.1016/j.tree.2010.04.001" class="plumx-summary plum-sciencedirect-theme" data-pass-hidden-categories="true" data-hide-usage="true" data-orientation="vertical" data-hide-print="true" data-site="plum" data-on-success="onMetricsWidgetSuccess">View article metrics</a></div></section></aside></div></div><div></div></div><div id="footer"><div class="hor-line" style="border-color:#e9711c"></div><div class="panel-s u-padding-l-bottom u-bg-white u-clr-grey7" role="contentinfo"><a class="anchor move-left els-footer-elsevier anchor-has-inherit-color" href="https://www.elsevier.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg viewBox="-3345 3440.027 140.01 24.333" style="width:104px;height:30px"><title>Elsevier</title><path id="E" style="fill:#E9711C" d="M-3343.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3344L-3343.999,3461.698"></path><path style="fill:#E9711C" d="M-3325.448,3461.698c2.074-0.118,2.83-0.502,2.83-2.832v-13.511c0-2.33-0.757-2.715-2.83-2.832v-0.591h8.884 v0.591c-2.243,0-3.027,0.472-3.027,2.891v13.009c0,1.652,0.056,2.625,1.71,2.625h4.008c3,0,4.4-2,5.576-4.1l0.673,0.118 l-1.71,5.222h-16.114V3461.698"></path><path style="fill:#E9711C" d="M-3307.122,3456.27h0.561c1.12,2.596,2.886,5.4,5.94,5.4c2,0,3.672-1.3,3.672-3.334 c0-1.652-1.626-3.1-4.176-4.927c-3.28-2.36-5.41-3.746-5.41-6.43c0-3.422,2.55-5.517,5.633-5.517c2.214,0,3,0.944,4.204,0.944 c0.476,0,0.56-0.266,0.476-0.737h0.561l0.645,6.076h-0.561c-0.785-2.625-2.523-5.19-5.354-5.19c-1.737,0-3.138,1.356-3.138,3.185 c0,1.918,1.933,3.157,5.016,5.016c2.523,1.504,5,3.362,5,6.254c0,3.245-2.608,5.752-5.97,5.752c-2.02,0-4.148-0.855-4.68-0.855 c-0.336,0-0.7,0.207-0.756,0.649h-0.56l-1.094-6.282"></path><path style="fill:#E9711C" d="M-3293.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3294L-3293.999,3461.698"></path><path style="fill:#E9711C" d="M-3265.839,3462.524h-0.42l-5.41-12.538c-0.896-2.065-1.71-4.16-2.83-6.136 c-0.478-0.826-1.346-1.327-2.3-1.327v-0.591h8.323v0.591c-0.785,0-2.354,0-2.354,1.15c0,0.384,0.87,2.45,1.653,4.308l4.063,9.676 l4.877-11.359c0.588-1.356,0.757-2.094,0.757-2.713c0-0.618-0.673-0.974-2.13-1.062v-0.59h5.941v0.591 c-0.337,0.06-0.7,0.117-1.037,0.295c-1.066,0.56-2.13,3.57-2.635,4.749l-6.5,14.957"></path><path style="fill:#E9711C" d="M-3255.472,3461.698c2.24,0,3.025-0.473,3.025-2.892v-13.393c0-2.42-0.784-2.89-3.025-2.891v-0.59h9.078v0.591 c-2.24,0-3.025,0.472-3.025,2.891v13.393c0,2.42,0.784,2.892,3.025,2.892v0.59h-9.08v-0.59"></path><path id="E_2_" style="fill:#E9711C" d="M-3244.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3245L-3244.999,3461.698"></path><path style="fill:#E9711C" d="M-3206,3461.698c-1.26-0.354-1.71-0.68-2.466-1.623l-6.166-7.609c3.027-0.65,5.13-2.185,5.13-5.547 c0-4.75-4.26-4.986-7.764-4.986h-9.191v0.591c2.24,0,3.026,0.472,3.026,2.891v13.393c0,2.42-0.785,2.892-3.026,2.892v0.59h9.08 v-0.59c-2.242,0-3.027-0.473-3.027-2.892v-5.604h2.551l7.314,9.086h4.54L-3206,3461.698 M-3220.399,3444.499 c0-1.387,0.337-1.476,2.186-1.476c2.774,0,5.3,0.974,5.3,4.308c0,3.6-2.887,4.63-5.914,4.631h-1.569v-7.463H-3220.399z"></path></svg></span></a><div class="panel-s u-bg-white u-padding-0-hor-from-xs u-padding-s-hor-from-md u-padding-xs-ver text-xs u-clear-both-from-xs u-clear-none-from-md"><p class="u-margin-xs-bottom"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/solutions/sciencedirect" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">About ScienceDirect</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/customer/authenticate/manra" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Remote access</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science?_ob=ShoppingCartURL&amp;_method=display&amp;md5=3ff44acb300f01481824c54a2973d019" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Shopping cart</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Contact and support</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Terms and conditions</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/privacy-policy" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Privacy policy</span></a></p><p id="els-footer-cookie-message">We use cookies to help provide and enhance our service and tailor content and ads. By continuing you agree to the<!-- --> <a class="anchor u-margin-0-right" href="https://www.elsevier.com/solutions/sciencedirect/support/cookies" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">use of cookies</span></a>.</p><p id="els-footer-copyright">Copyright © <!-- -->2018<!-- --> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</p></div><div><span style="margin-top:-10px" class="u-position-relative move-bottom u-float-left-from-xs u-float-right-from-md move-right u-padding-0-hor"><a class="anchor els-footer-relx anchor-has-inherit-color" href="https://www.relx.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg style="width:118px;height:28px" viewBox="-3248 3424.938 334.44 53.422"><title>RELX Group</title><path style="fill:#E9711C" d="M-3227.285,3448.329c10.948,0,24.15-2.576,24.15-13.124c0-7.474-7.63-10.267-14.6-10.267 c-10.647,0-30.265,5.926-30.265,26.96c0,10.555,8.034,16.643,20.778,16.643c13.474,0,23.266-8.768,26.26-20.535 c-7.912,13.648-17.972,17.896-26.18,17.896c-10.19,0-14.077-7.568-14.077-14.05c0-18.56,14.406-24.798,23.494-24.798 c6.135,0,9.727,3.936,9.727,8.253c0,11.918-17.17,11.578-25.008,11.578c-0.803,0-3.274,0.02-3.76-0.003l-1.654,1.714 c3.59,0.508,8.554,1.497,13.944,4.223c11.204,5.656,21.745,20.59,35.7,20.59c5.062,0,6.38-1.543,7.335-2.893 c-6.834,4.354-18.286-3.197-24.46-9.04c-5.082-4.806-9.57-10.26-21.387-13.146"></path><g><path style="fill:#666666" d="M-3158.73,3468.502c-0.423,0-0.636-0.164-0.794-0.547l-6.885-14.511h-6.197c-0.21,0-0.316,0.11-0.316,0.33 v14.182c0,0.328-0.213,0.547-0.532,0.547h-5.189c-0.32,0-0.53-0.22-0.53-0.547v-36.139c0-0.33,0.21-0.55,0.53-0.549h13.928 c6.62,0,11.492,4.545,11.492,11.117c0,4.873-2.7,8.65-6.833,10.238l7.575,15.167c0.21,0.385,0,0.713-0.37,0.713L-3158.73,3468.502 L-3158.73,3468.502z M-3159.42,3442.384c0-3.232-2.223-5.31-5.506-5.311h-7.68c-0.21,0-0.316,0.108-0.316,0.328v9.912 c0,0.217,0.104,0.328,0.316,0.328h7.68C-3161.644,3447.641-3159.42,3445.561-3159.42,3442.384"></path><path style="fill:#666666" d="M-3146.95,3431.817c0-0.33,0.21-0.55,0.53-0.549h21.865c0.317,0,0.53,0.22,0.53,0.549v4.709 c0,0.33-0.212,0.548-0.53,0.548h-15.827c-0.212,0-0.318,0.108-0.318,0.328v9.089c0,0.22,0.106,0.328,0.318,0.328h12.726 c0.317,0,0.53,0.22,0.53,0.547v4.709c0,0.33-0.213,0.548-0.53,0.548h-12.726c-0.212,0-0.318,0.11-0.318,0.329v9.418 c0,0.22,0.106,0.328,0.318,0.328h15.827c0.317,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.212,0.547-0.53,0.547h-21.865 c-0.32,0-0.53-0.22-0.53-0.547C-3146.95,3467.956-3146.95,3431.817-3146.95,3431.817z"></path><path style="fill:#666666" d="M-3118.2,3431.817c0-0.33,0.212-0.55,0.53-0.549h5.191c0.317,0,0.53,0.22,0.53,0.549v30.553 c0,0.22,0.106,0.328,0.317,0.328h14.722c0.318,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.213,0.547-0.53,0.547h-20.76 c-0.317,0-0.53-0.22-0.53-0.547C-3118.2,3467.956-3118.2,3431.817-3118.2,3431.817z"></path><path style="fill:#666666" d="M-3052.46,3449.885c0-6.46,0.423-8.926,1.006-10.787c1.747-5.53,5.878-8.377,11.758-8.377 c5.88,0,9.532,3.012,11.28,6.9c0.107,0.274,0.107,0.548-0.21,0.712l-1.695,0.876c-0.265,0.11-0.53,0.055-0.688-0.22 c-1.802-3.396-4.45-5.312-8.74-5.312c-4.45,0-7.468,2.192-8.792,6.3c-0.475,1.422-0.845,3.722-0.845,9.909s0.37,8.486,0.845,9.91 c1.324,4.107,4.343,6.3,8.792,6.3c4.342,0,7.468-2.137,8.79-6.19c0.477-1.422,0.85-3.56,0.85-7.117c0-0.22-0.106-0.328-0.318-0.328 h-8.155c-0.318,0-0.53-0.22-0.53-0.548v-1.807c0-0.33,0.212-0.548,0.53-0.548h10.962c0.317,0,0.53,0.22,0.53,0.548v2.243 c0,3.725-0.423,6.572-0.954,8.27c-1.694,5.53-5.88,8.432-11.65,8.432c-5.88,0-10.01-2.848-11.758-8.38 C-3052.036,3458.811-3052.459,3456.345-3052.46,3449.885"></path><path style="fill:#666666" d="M-3021.43,3468.502c-0.317,0-0.53-0.22-0.53-0.547v-25.023c0-0.33,0.213-0.548,0.53-0.548h1.801 c0.316,0,0.53,0.22,0.53,0.548v3.338h0.052c0.953-2.572,3.23-4.434,6.78-4.434c2.013,0,3.866,0.712,5.084,1.807 c0.265,0.164,0.317,0.438,0.107,0.713l-1.06,1.531c-0.212,0.274-0.48,0.274-0.795,0.11c-1.164-0.767-2.436-1.313-4.024-1.313 c-4.45,0-6.144,3.996-6.144,8.815v14.456c0,0.328-0.213,0.547-0.53,0.547H-3021.43L-3021.43,3468.502z"></path><path style="fill:#666666" d="M-3004.54,3462.26c-0.53-1.752-0.847-3.668-0.847-6.844c0-3.12,0.316-5.037,0.847-6.79 c1.378-4.327,4.818-6.79,9.426-6.79c4.662,0,8.104,2.464,9.48,6.79c0.53,1.752,0.848,3.668,0.848,6.79 c0,3.176-0.317,5.092-0.848,6.844c-1.377,4.326-4.818,6.79-9.48,6.79C-2999.722,3469.05-3003.162,3466.585-3004.54,3462.26 M-2988.387,3461.33c0.475-1.48,0.688-3.066,0.688-5.914c0-2.794-0.213-4.38-0.688-5.86c-1.007-3.12-3.442-4.873-6.728-4.873 c-3.23,0-5.666,1.752-6.672,4.873c-0.477,1.48-0.688,3.065-0.688,5.86c0,2.848,0.212,4.436,0.688,5.914 c1.006,3.12,3.442,4.873,6.672,4.873C-2991.829,3466.203-2989.395,3464.451-2988.387,3461.33"></path><path style="fill:#666666" d="M-2963.66,3468.502c-0.317,0-0.528-0.22-0.528-0.547v-3.066h-0.053c-1.272,2.574-3.92,4.162-7.36,4.162 c-5.615,0-8.74-3.616-8.74-9.913v-16.206c0-0.33,0.213-0.548,0.53-0.548h1.801c0.317,0,0.53,0.22,0.53,0.548v15.44 c0,5.257,2.118,7.83,6.515,7.83c3.812,0,6.78-2.74,6.78-7.284v-15.987c0-0.33,0.21-0.548,0.528-0.548h1.802 c0.317,0,0.53,0.22,0.53,0.548v25.023c0,0.328-0.213,0.547-0.53,0.547h-1.802L-2963.66,3468.502z"></path><path style="fill:#666666" d="M-2955.29,3478.36c-0.317,0-0.53-0.22-0.53-0.549v-34.879c0-0.33,0.213-0.548,0.53-0.548h1.8 c0.317,0,0.53,0.22,0.53,0.548v3.176h0.053c1.218-2.465,3.496-4.272,7.68-4.272c4.45,0,7.256,2.08,8.58,6.242 c0.638,2.137,0.9,4.435,0.9,7.393c0,2.9-0.264,5.2-0.9,7.336c-1.323,4.164-4.13,6.242-8.58,6.242c-4.183,0-6.46-1.807-7.68-4.27 h-0.053v13.031c0,0.33-0.213,0.55-0.53,0.549L-2955.29,3478.36L-2955.29,3478.36z M-2939.35,3461.602 c0.53-1.643,0.688-3.832,0.688-6.13c0-2.355-0.158-4.545-0.688-6.188c-0.952-2.957-3.124-4.6-6.46-4.6 c-3.178,0-5.507,1.533-6.46,4.6c-0.477,1.423-0.69,3.34-0.69,6.188c0,2.847,0.212,4.71,0.69,6.13 c0.954,3.067,3.283,4.602,6.46,4.602C-2942.473,3466.204-2940.3,3464.561-2939.35,3461.602"></path><path style="fill:#666666" d="M-3070.48,3468.545c-0.424,0-0.634-0.164-0.85-0.547l-7.254-12.649h-0.106l-7.307,12.649 c-0.213,0.383-0.424,0.547-0.85,0.547h-5.72c-0.37,0-0.528-0.328-0.318-0.711l10.76-18.562l-9.96-17.248 c-0.21-0.385-0.05-0.712,0.32-0.712h5.721c0.424,0,0.636,0.163,0.847,0.548l6.513,11.278h0.106l6.513-11.278 c0.213-0.385,0.425-0.548,0.85-0.548h5.72c0.37,0,0.53,0.327,0.317,0.712l-9.956,17.248l10.75,18.562 c0.21,0.383,0.052,0.71-0.32,0.711h-5.771H-3070.48z"></path><path style="fill:#666666" d="M-2932.08,3443.951c-0.108,0-0.18-0.075-0.18-0.189v-10.385c0-0.075-0.036-0.113-0.11-0.113h-3.202 c-0.11,0-0.184-0.073-0.184-0.186v-1.6c0-0.11,0.074-0.186,0.184-0.186h8.746c0.11,0,0.183,0.075,0.183,0.186v1.6 c0,0.113-0.07,0.186-0.183,0.186h-3.201c-0.073,0-0.11,0.038-0.11,0.113v10.385c0,0.113-0.07,0.19-0.18,0.189H-2932.08 L-2932.08,3443.951z"></path><path style="fill:#666666" d="M-2924.47,3431.478c0-0.112,0.07-0.186,0.178-0.186h1.604c0.163,0,0.252,0.056,0.307,0.186l3.367,7.8h0.07 l3.315-7.8c0.052-0.13,0.144-0.186,0.305-0.186h1.583c0.11,0,0.182,0.074,0.182,0.186v12.286c0,0.11-0.07,0.186-0.182,0.186h-1.564 c-0.11,0-0.18-0.075-0.18-0.186v-7.744h-0.073l-2.593,5.957c-0.07,0.168-0.18,0.242-0.343,0.242h-1.044 c-0.16,0-0.27-0.074-0.343-0.242l-2.59-5.957h-0.072v7.744c0,0.11-0.072,0.186-0.18,0.186h-1.567c-0.106,0-0.178-0.075-0.178-0.186 v-12.286H-2924.47z"></path></g></svg></span></a></span></div></div></div></section></div></div></div>
<script type="application/json" data-iso-key="_0">{"userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.77 Safari/537.36","abstracts":{"content":[{"#name":"abstract","$":{"xmlns:ce":true,"class":"author","id":"aep-abstract-id18"},"$$":[{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id19"},"$$":[{"#name":"simple-para","$":{"view":"all"},"_":"Understanding the forces that influence natural variation within and among populations has been a major objective of evolutionary biologists for decades. Motivated by the growth in computational power and data complexity, modern approaches to this question make intensive use of simulation methods. Approximate Bayesian Computation (ABC) is one of these methods. Here we review the foundations of ABC, its recent algorithmic developments, and its applications in evolutionary biology and ecology. We argue that the use of ABC should incorporate all aspects of Bayesian data analysis: formulation, fitting, and improvement of a model. ABC can be a powerful tool to make inferences with complex models if these principles are carefully applied."}]}]}],"floats":[],"footnotes":[],"attachments":[]},"biographies":{},"combinedContentItems":{},"experiments":{},"rawtext":"","authors":{"content":[{"#name":"author-group","$":{"xmlns:ce":true},"$$":[{"#name":"author","$":{"id":"aep-author-id14"},"$$":[{"#name":"given-name","_":"Katalin"},{"#name":"surname","_":"Csilléry"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"1"}]},{"#name":"e-address","$":{"type":"email"},"_":"kati.csillery@gmail.com"}]},{"#name":"author","$":{"id":"aep-author-id15"},"$$":[{"#name":"given-name","_":"Michael G.B."},{"#name":"surname","_":"Blum"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"1"}]}]},{"#name":"author","$":{"id":"aep-author-id16"},"$$":[{"#name":"given-name","_":"Oscar E."},{"#name":"surname","_":"Gaggiotti"},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"2"}]}]},{"#name":"author","$":{"id":"aep-author-id17"},"$$":[{"#name":"given-name","_":"Olivier"},{"#name":"surname","_":"François"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"1"}]}]},{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"1"},{"#name":"textfn","_":"Laboratoire Techniques de l’Ingénierie Médicale et de la Complexité, Centre National de la Recherche Scientifique UMR5525, Université Joseph Fourier, 38706 La Tronche, France"}]},{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"2"},{"#name":"textfn","_":"Laboratoire d’Ecologie Alpine, Centre National de la Recherche Scientifique UMR5553, Université Joseph Fourier, 38041 Grenoble, France"}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"1"},{"#name":"textfn","_":"Laboratoire Techniques de l’Ingénierie Médicale et de la Complexité, Centre National de la Recherche Scientifique UMR5525, Université Joseph Fourier, 38706 La Tronche, France"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"2"},{"#name":"textfn","_":"Laboratoire d’Ecologie Alpine, Centre National de la Recherche Scientifique UMR5553, Université Joseph Fourier, 38041 Grenoble, France"}]}},"attachments":[],"correspondences":{},"scopusAuthorIds":{},"articles":{}},"body":{},"exam":{},"article":{"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Elsevier Current Trends","id":"88"},"isSpecialIssue":false,"altSmallCover":"https://ars.els-cdn.com/content/image/Dcellpress.gif","isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":51,"openArchiveStatus":false,"openArchiveArticleCount":0,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"titleImageUrl":"https://ars.els-cdn.com/content/image/B01695347.svg","issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S0169534710X00064-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S0169534710X00064/loc/DOWNSAMPLED200/image/gif/ca4241efae9f83f3dbaacc1d5331c18f/cov200h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S0169534710X00064/loc/DOWNSAMPLED200/image/gif/ca4241efae9f83f3dbaacc1d5331c18f/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"20581","pixel-height":"200","pixel-width":"153"},{"attachment-eid":"1-s2.0-S0169534710X00064-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S0169534710X00064/loc/DOWNSAMPLED/image/gif/168250f553b80483f1942468ea8c37ef/cov150h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S0169534710X00064/loc/DOWNSAMPLED/image/gif/168250f553b80483f1942468ea8c37ef/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"12835","pixel-height":"150","pixel-width":"115"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S01695347.gif","sourceOpenAccess":false,"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/Dcellpress.gif"},"pii":"S0169534710000662","dates":{"Available online":"18 May 2010","Revised":[],"Publication date":"1 July 2010"},"access":{},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"analyticsMetadata":{"accountId":"262591","accountName":"International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"12289512"},"cid":"271938","content-family":"serial","copyright-line":"Copyright © 2010 Elsevier Ltd. All rights reserved.","cover-date-years":["2010"],"cover-date-start":"2010-07-01","cover-date-text":"July 2010","document-subtype":"rev","document-type":"article","entitledToken":"1AD479B59D354C38FA62755684CC50913D9D9F8B1DD18B74C931B425B3310834942E4085CF8C9599","eid":"1-s2.0-S0169534710000662","doi":"10.1016/j.tree.2010.04.001","first-fp":"410","hub-eid":"1-s2.0-S0169534710X00064","issuePii":"S0169534710X00064","iss-first":"7","item-weight":"FULL-TEXT","language":"en","last-lp":"418","last-author":{"#name":"last-author","$":{"xmlns:dm":true},"$$":[{"#name":"author","$":{"xmlns:ce":true,"id":"aep-author-id17"},"$$":[{"#name":"given-name","_":"Olivier"},{"#name":"surname","_":"François"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"1"}]}]}]},"normalized-first-auth-initial":"K","normalized-first-auth-surname":"CSILLERY","pages":[{"last-page":"418","first-page":"410"}],"srctitle":"Trends in Ecology & Evolution","timestamp":"2015-05-14T05:31:34.878038-04:00","title":{"content":[{"#name":"dochead","$":{"xmlns:ce":true},"$$":[{"#name":"textfn","_":"Review"}]},{"#name":"title","$":{"xmlns:ce":true},"_":"Approximate Bayesian Computation (ABC) in practice"}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"25","vol-iss-suppl-text":"Volume 25, Issue 7","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":false,"blockFullTextForAnonymousAccess":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":false,"issn":"01695347","issn-primary-formatted":"0169-5347","useEnhancedReader":false,"isCorpReq":false,"pdfDownload":{"linkType":"DOWNLOAD","linkToPdf":"/science/article/pii/S0169534710000662/pdfft?md5=19d73a51c65f4cabb36fdc807fa2ac4f&pid=1-s2.0-S0169534710000662-main.pdf","isPdfFullText":false,"fileName":"1-s2.0-S0169534710000662-main.pdf"},"pdfUrlForCrawlers":"https://www.sciencedirect.com/science/article/pii/S0169534710000662/pdfft?md5=19d73a51c65f4cabb36fdc807fa2ac4f&pid=1-s2.0-S0169534710000662-main.pdf","indexTag":true,"volRange":"25","issRange":"7","userProfile":{"departmentName":"General Access","accessType":"IPRANGE","accountId":"262591","webUserId":"12289512","accountName":"International Institute of Tropical Agriculture","departmentId":"267018","userType":"NORMAL","hasMultipleOrganizations":false},"entitlementReason":"package","articleEntitlement":{"entitled":true,"usageInfo":"(12289512,U|267018,D|262591,A|34,P|2,PL)(SDFE,CON|50fde1736906d84eda69c8a17bce301e14e8gxrqa,SSO|ANON_IP,ACCESS_TYPE)"},"aipType":"none","downloadFullIssue":true,"headerConfig":{"helpUrl":"https://service.elsevier.com/app/home/supporthub/sciencedirect/","contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","orgName":"","webUserId":"12289512","libraryBanner":{},"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasMultiOrg":false,"userType":"IPRANGE","allowCart":false},"open-research":{},"titleString":"Approximate Bayesian Computation (ABC) in practice","onAbstractWhitelist":false,"isAbstract":false,"isContentVisible":false,"ajaxLinks":{"glossary":true,"citingArticles":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true}},"specialIssueArticles":{},"recommendations":{},"entitledRecommendations":{"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"citingArticles":{},"workspace":{"isOpen":false},"crossMark":{"isOpen":false},"userIdentity":{},"refersTo":{},"referredToBy":{},"downloadIssue":{"isOpen":false,"articles":[],"selected":[]},"references":{},"referenceLinks":{"internal":{},"external":{}},"glossary":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenRecommendations":true,"isOpenCitingArticles":false,"citingArticles":[false,false,false],"recommendations":[false,false,false,false,false,false],"specialIssueArticles":[false,false,false]},"banner":{"expanded":false},"transientError":{"isOpen":false},"tableOfContents":{"showEntitledTocLinks":true},"chapters":{"toc":[],"isLoading":false},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"metrics":{"isOpen":true},"signOut":{"isOpen":false},"issueNavigation":{"previous":{},"next":{}},"tail":{},"linkingHubLinks":[]}</script>
      
      <script src="satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c.js"></script><script src="satellite-565e008964746d4385002642.js"></script>
      <script src="babel-polyfill.js"></script>
      <script src="react.js"></script>
      <script src="react-dom.js"></script>
      <script src="arp.js" async=""></script>
      <!-- begin usabilla live embed code -->
      <script type="text/javascript">
        window.lightningjs||function(c){function g(b,d){d&&(d+=(/\?/.test(d)?"&":"?")+"lv=1");c[b]||function(){var i=window,h=document,j=b,g=h.location.protocol,l="load",k=0;(function(){function b(){a.P(l);a.w=1;c[j]("_load")}c[j]=function(){function m(){m.id=e;return c[j].apply(m,arguments)}var b,e=++k;b=this&&this!=i?this.id||0:0;(a.s=a.s||[]).push([e,b,arguments]);m.then=function(b,c,h){var d=a.fh[e]=a.fh[e]||[],j=a.eh[e]=a.eh[e]||[],f=a.ph[e]=a.ph[e]||[];b&&d.push(b);c&&j.push(c);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"//"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.frameBorder="0";f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
        window.usabilla_live = lightningjs.require("usabilla_live", "https://w.usabilla.com/eb1c14a91932.js");
        var customData = {};

        if(window.pageData && pageData.content && pageData.content[0]) {
          customData.entitlementType = pageData.content[0].entitlementType;
        }
        if(window.pageData && pageData.visitor) {
          customData.accessType = pageData.visitor.accessType;
          customData.accountId = pageData.visitor.accountId;
          customData.loginStatus = pageData.visitor.loginStatus;
        }
        usabilla_live("data", {"custom": customData });
      </script>
      <!-- end usabilla live embed code -->
      <script type="text/x-mathjax-config;executed=true">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="MathJax.js"></script>
      <script async="" src="gpt.js"></script>
    
  <div class="js-react-modal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><script src="widget-summary.js" async=""></script><iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_elsevier_0" name="destination_publishing_iframe_elsevier_0_name" style="display: none; width: 0px; height: 0px;" src="https://elsevier.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS0169534710000662%23bib4"></iframe></body></html>