<!DOCTYPE html>
<html class="no-js" lang="en"><head><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script>

        <title>ModelTest-NG: A New and Scalable Tool for the Selection 
of DNA and Protein Evolutionary Models | Molecular Biology and Evolution
 | Oxford Academic</title>

        <script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>




    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721281881/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637087210721125680/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site-1495373344.css" rel="stylesheet" type="text/css">
    


        <script>
            (function (w, d, s, l, i) {
            w[l] = w[l] || []; w[l].push({
                'gtm.start':
                new Date().getTime(), event: 'gtm.js'
            }); var f = d.getElementsByTagName(s)[0],
                j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
                    'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    


        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [1],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'mbe';
            App.SiteURL = 'academic.oup.com/mbe';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz189/5552155","@type":"ScholarlyArticle","name":"ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models","datePublished":"2019-08-21","isPartOf":{"@id":"https://academic.oup.com/mbe","@type":"Periodical","name":"Molecular Biology and Evolution","issn":["1537-1719"]},"url":"http://dx.doi.org/10.1093/molbev/msz189","keywords":["phylogenetic model selection","high-performance computing","efficient algorithms","phylogenetic inference"],"inLanguage":"en","copyrightHolder":"Society for Molecular Biology and Evolution","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz189/5552155","author":[{"name":"Darriba, Diego","affiliation":"Computer Architecture Group, Centro de investigación CITIC, Universidade da Coruña , Elviña, A Coruña, Spain   Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person"},{"name":"Posada, David","affiliation":"Department of Biochemistry, Genetics, and Immunology, University of Vigo , Vigo, Spain   Biomedical Research Center (CINBIO), University of Vigo , Vigo, Spain   Galicia Sur Health Research Institute , Vigo, Spain","@type":"Person","sameAs":"http://orcid.org/0000-0003-1407-3406"},{"name":"Kozlov, Alexey M","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person"},{"name":"Stamatakis, Alexandros","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany   Institute of Theoretical Informatics, Karlsruhe Institute of Technology , Karlsruhe, Germany","@type":"Person","sameAs":"http://orcid.org/0000-0003-0353-0691"},{"name":"Morel, Benoit","affiliation":"Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies , Heidelberg, Germany","@type":"Person"},{"name":"Flouri, Tomas","affiliation":"Department of Genetics, Evolution and Environment, University College London , London, United Kingdom","@type":"Person"}],"description":"Abstract.  ModelTest-NG is a reimplementation from scratch of jModelTest and ProtTest, two popular tools for selecting the best-fit nucleotide and amino acid su","pageStart":"msz189","thumbnailURL":"//oup.silverchair-cdn.com/UI/app/img/generic-cover.png","headline":"ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models","image":"//oup.silverchair-cdn.com/UI/app/img/generic-cover.png"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models">
    <meta property="og:description" content="Abstract.  ModelTest-NG is a reimplementation from scratch of jModelTest and ProtTest, two popular tools for selecting the best-fit nucleotide and amino acid su">
    <meta property="og:url" content="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz189/5552155">
    <meta property="og:image" content="//oup.silverchair-cdn.com/UI/app/img/generic-cover.png">
    <meta property="og:updated_time" content="2019-08-21">
<meta name="citation_author" content="Darriba, Diego"><meta name="citation_author_institution" content="Computer Architecture Group, Centro de investigación CITIC, Universidade da Coruña, Elviña, A Coruña, Spain"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Posada, David"><meta name="citation_author_institution" content="Department of Biochemistry, Genetics, and Immunology, University of Vigo, Vigo, Spain"><meta name="citation_author_institution" content="Biomedical Research Center (CINBIO), University of Vigo, Vigo, Spain"><meta name="citation_author_institution" content="Galicia Sur Health Research Institute, Vigo, Spain"><meta name="citation_author" content="Kozlov, Alexey M"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Stamatakis, Alexandros"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author_institution" content="Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany"><meta name="citation_author" content="Morel, Benoit"><meta name="citation_author_institution" content="Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Flouri, Tomas"><meta name="citation_author_institution" content="Department of Genetics, Evolution and Environment, University College London, London, United Kingdom"><meta name="citation_title" content="ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models"><meta name="citation_doi" content="10.1093/molbev/msz189"><meta name="citation_journal_title" content="Molecular Biology and Evolution"><meta name="citation_journal_abbrev" content="Mol Biol Evol"><meta name="citation_reference" content="citation_title=Model selection may not be a mandatory step for phylogeny reconstruction;  citation_journal_title=Nat Commun;  citation_year=2019;  citation_volume=10;  citation_issue=1;  citation_pages=934."><meta name="citation_reference" content="citation_title=A new look at the statistical model identification;  citation_journal_title=IEEE Trans Automat Contr;  citation_year=1974;  citation_volume=19;  citation_issue=6;  citation_pages=716-723; "><meta name="citation_reference" content="citation_title=Genome-wide heterogeneity of nucleotide substitution model fit;  citation_journal_title=Genome Biol Evol;  citation_year=2011;  citation_volume=3;  citation_pages=896-908; "><meta name="citation_reference" content="citation_title=Model misspecification and probabilistic tests of topology: evidence from empirical data sets;  citation_journal_title=Syst Biol;  citation_year=2002;  citation_volume=51;  citation_issue=3;  citation_pages=509-523; "><meta name="citation_reference" content="citation_title=The effects of nucleotide substitution model assumptions on estimates of nonparametric bootstrap support;  citation_journal_title=Mol Biol Evol;  citation_year=2002;  citation_volume=19;  citation_issue=4;  citation_pages=394-405; "><meta name="citation_reference" content="citation_title=ProtTest 3: fast selection of best-fit models of protein evolution;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_issue=8;  citation_pages=1164-1165; "><meta name="citation_reference" content="citation_title=jModelTest 2: more models, new heuristics and parallel computing;  citation_journal_title=Nat Methods;  citation_year=2012;  citation_volume=9;  citation_issue=8;  citation_pages=772-772; "><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood;  citation_journal_title=Syst Biol;  citation_year=2003;  citation_volume=52;  citation_issue=5;  citation_pages=696-704; "><meta name="citation_reference" content="citation_title=Does the choice of nucleotide substitution models matter topologically?;  citation_journal_title=BMC Bioinformatics;  citation_year=2016;  citation_volume=17;  citation_issue=1;  citation_pages=143."><meta name="citation_reference" content="citation_title=Modelfinder: fast model selection for accurate phylogenetic estimates;  citation_journal_title=Nat Methods;  citation_year=2017;  citation_volume=14;  citation_issue=6;  citation_pages=587."><meta name="citation_reference" content="citation_title=Efficient detection of repeating sites to accelerate phylogenetic likelihood calculations;  citation_journal_title=Syst Biol;  citation_year=2017;  citation_volume=66;  citation_issue=2;  citation_pages=205-217; "><meta name="citation_reference" content="citation_title=RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference;  citation_journal_title=Bioinformatics;  citation_year=2019; "><meta name="citation_reference" content="citation_title=Modeling protein evolution with several amino acid replacement matrices depending on site rates;  citation_journal_title=Mol Biol Evol;  citation_year=2012;  citation_volume=29;  citation_issue=10;  citation_pages=2921-2936; "><meta name="citation_reference" content="citation_title=SMS: Smart Model Selection in PhyML;  citation_journal_title=Mol Biol Evol;  citation_year=2017;  citation_volume=34;  citation_issue=9;  citation_pages=2422-2424; "><meta name="citation_reference" content="citation_title=The importance of proper model assumption in Bayesian phylogenetic;  citation_journal_title=Syst Biol;  citation_year=2004;  citation_volume=53;  citation_issue=2;  citation_pages=265-277; "><meta name="citation_reference" content="citation_title=Performance-based selection of likelihood models for phylogeny estimation;  citation_journal_title=Syst Biol;  citation_year=2003;  citation_volume=52;  citation_issue=5;  citation_pages=674-683; "><meta name="citation_reference" content="citation_title=IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies;  citation_journal_title=Mol Biol Evol;  citation_year=2015;  citation_volume=32;  citation_issue=1;  citation_pages=268-274; "><meta name="citation_reference" content="citation_title=jModelTest: phylogenetic model averaging;  citation_journal_title=Mol Biol Evol;  citation_year=2008;  citation_volume=25;  citation_issue=7;  citation_pages=1253-1256; "><meta name="citation_reference" content="citation_title=MODELTEST: testing the model of DNA substitution;  citation_journal_title=Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_issue=9;  citation_pages=817-818; "><meta name="citation_reference" content="citation_title=MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space;  citation_journal_title=Syst Biol;  citation_year=2012;  citation_volume=61;  citation_issue=3;  citation_pages=539-542; "><meta name="citation_reference" content="citation_title=Estimating the dimension of a model;  citation_journal_title=Ann Statist;  citation_year=1978;  citation_volume=6;  citation_issue=2;  citation_pages=461-464; "><meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_issue=9;  citation_pages=1312-1313; "><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland (MA;  citation_title=Paup*: phylogenetic analysis using parsimony (and other methods) version 4;  citation_year=2002; "><meta name="citation_reference" content="citation_title=Some probabilistic and statistical problems in the analysis of DNA sequences;  citation_journal_title=Lectures Math Life Sci;  citation_year=1986;  citation_volume=17;  citation_issue=2;  citation_pages=57-86; "><meta name="citation_reference" content="citation_title=A space-time process model for the evolution of DNA sequences;  citation_journal_title=Genetics;  citation_year=1995;  citation_volume=139;  citation_issue=2;  citation_pages=993-1005; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/mbe/advance-article-pdf/doi/10.1093/molbev/msz189/30026588/msz189.pdf"><meta name="description" content="Abstract.  ModelTest-NG is a reimplementation from scratch of jModelTest and ProtTest, two popular tools for selecting the best-fit nucleotide and amino acid su">    <script id="altmetric-embed-js" src="altmetric_badges-e99c269c07ff8ac628ce199348c2901d63f7ce6052a1.js"></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019111801.js" async=""></script><style></style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2.css"></head><body data-sitestyletemplate="Journal" theme-mbe="" class="off-canvas pg_Article pg_article" data-sitename="molecularbiologyandevolution" style="cursor: default;"><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
    <meta name="citation_xml_url" content="https://academic.oup.com/mbe/advance-article-xml/doi/10.1093/molbev/msz189/5552155">



    <link rel="canonical" href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz189/5552155">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/mbe_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "advance-article");
    googletag.pubads().setTargeting("jnlsurl", "mbe/advance-article/doi/10.1093/molbev/msz189/5552155");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    
    

    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=molbev&authentication_method=IP&authzrequired=false&account_id=2783535&account_list=2783535,2785688&authnips=197.248.44.39&doi=10.1093/molbev/msz189';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5325" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">International Institue of Tropical Agriculture (IITA) <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                                        </li>

                                </ul>

                            </div>
                        </div>
                    </li>                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2fadvance-article%2fdoi%2f10.1093%2fmolbev%2fmsz189%2f5552155" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            International Institue of Tropical Agriculture (IITA)
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2fadvance-article%2fdoi%2f10.1093%2fmolbev%2fmsz189%2f5552155" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432203">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432200">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">More content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432204">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432205">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%E2%80%93+Newest+First&amp;f_TocHeadingTitle=Fast+TracksANDFAST+TRACK" class="nav-link ">Fast Track</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432206">
            <a href="https://academic.oup.com/mbe/search-results?allJournals=1&amp;f_TocHeadingTitle=Perspective&amp;fl_SiteID=5325&amp;page=1&amp;sort=Date+%e2%80%93+Newest+First" class="nav-link ">Perspectives</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432207">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Discoveries&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Discoveries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432208">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=METHODS&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Methods</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432209">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Resources&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Resources</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432210">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;f_TocHeadingTitle=PROTOCOLSANDProtocol&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Protocols</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432211">
            <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=News" class="nav-link ">News</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432201">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432212">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432213">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432214">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432215">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432216">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432217">
            <a href="https://academic.oup.com/mbe/pages/Open_Access" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432218">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432219">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432202">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432220">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432221">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432222">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432223">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432224">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432225">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432226">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432227">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432228">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/mbe" class="journal-logo-container">
                    <img id="logo-MolecularBiologyandEvolution" class="journal-logo" src="mbe_title188616415.svg" alt="Molecular Biology and Evolution">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.smbe.org/smbe/" target="" class="society-logo-container">
                                <img id="logo-SocietyforMolecularBiologyandEvolution" class="society-logo" src="mbe_h1-1277340080.svg" alt="Society for Molecular Biology and Evolution">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432203">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432200">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">More content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432204">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432205">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%E2%80%93+Newest+First&amp;f_TocHeadingTitle=Fast+TracksANDFAST+TRACK" class="nav-link ">Fast Track</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432206">
            <a href="https://academic.oup.com/mbe/search-results?allJournals=1&amp;f_TocHeadingTitle=Perspective&amp;fl_SiteID=5325&amp;page=1&amp;sort=Date+%e2%80%93+Newest+First" class="nav-link ">Perspectives</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432207">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Discoveries&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Discoveries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432208">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=METHODS&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Methods</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432209">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;allJournals=1&amp;f_TocHeadingTitle=Resources&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Resources</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432210">
            <a href="https://academic.oup.com/mbe/search-results?sort=Date+%e2%80%93+Newest+First&amp;f_TocHeadingTitle=PROTOCOLSANDProtocol&amp;fl_SiteID=5325&amp;page=1" class="nav-link ">Protocols</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432211">
            <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=News" class="nav-link ">News</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432201">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432212">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432213">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432214">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432215">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432216">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432217">
            <a href="https://academic.oup.com/mbe/pages/Open_Access" class="nav-link ">Open Access</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432218">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432219">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1432202">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432220">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432221">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432222">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432223">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432224">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432225">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432226">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432227">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1432228">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option>
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="SMBE Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/smbe" type="hidden">
    <input name="siteID" id="hfsiteID" value="5325" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5538" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/mbe" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="163539885">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#163539885">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="163539896">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#163539896">Supplementary Material</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="163539898">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#163539898">Acknowledgments</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="163539900">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#163539900">References</a>
            </div>
        </li>
<li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-article-browse-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-article-browse-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
                    <span class="article-pubstate article-flag">Corrected proof</span>

        </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Diego Darriba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computer Architecture Group, Centro de investigación CITIC, Universidade da Coruña</div>, Elviña, A Coruña, Spain</div><div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>


        <div class="info-author-correspondence">
            <strong>Corresponding author:</strong> E-mail: <a href="mailto:diego.darriba@udc.es" target="">diego.darriba@udc.es</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Diego+Darriba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Darriba%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Darriba%20Diego%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Diego Darriba</a><span class="delimiter"><i class="icon-general-mail"></i>, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David Posada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Department of Biochemistry, Genetics, and Immunology, University of Vigo</div>, Vigo, Spain</div><div class="aff"><div class="insititution">Biomedical Research Center (CINBIO), University of Vigo</div>, Vigo, Spain</div><div class="aff"><div class="insititution">Galicia Sur Health Research Institute</div>, Vigo, Spain</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-1407-3406">
                <img class="orchid-icon" alt="" src="mini-icon.png">&nbsp;&nbsp;http://orcid.org/0000-0003-1407-3406
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=David+Posada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Posada%20D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Posada%20David%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">David Posada</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexey M Kozlov</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Alexey+M+Kozlov">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kozlov%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kozlov%20Alexey%20M%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Alexey M Kozlov</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alexandros Stamatakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div><div class="aff"><div class="insititution">Institute of Theoretical Informatics, Karlsruhe Institute of Technology</div>, Karlsruhe, Germany</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-0353-0691">
                <img class="orchid-icon" alt="" src="mini-icon.png">&nbsp;&nbsp;http://orcid.org/0000-0003-0353-0691
            </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Alexandros+Stamatakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stamatakis%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stamatakis%20Alexandros%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Alexandros Stamatakis</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Benoit Morel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Computational Molecular Evolution Group, Heidelberg Institute for Theoretical Studies</div>, Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Benoit+Morel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morel%20B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Morel%20Benoit%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Benoit Morel</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomas Flouri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Department of Genetics, Evolution and Environment, University College London</div>, London, United Kingdom</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Tomas+Flouri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Flouri%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Flouri%20Tomas%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">Tomas Flouri</a><span class="delimiter"></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Molecular Biology and Evolution</em>, msz189, <a href="https://doi.org/10.1093/molbev/msz189">https://doi.org/10.1093/molbev/msz189</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">21 August 2019</div>
            </div>
        </div>
    </div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5552155" href="https://academic.oup.com/mbe/advance-article-pdf/doi/10.1093/molbev/msz189/30026588/msz189.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Diego Darriba, David Posada, Alexey M Kozlov, Alexandros 
Stamatakis, Benoit Morel, Tomas Flouri,  ModelTest-NG: A New and 
Scalable Tool for the Selection of DNA and Protein Evolutionary Models, <em>Molecular Biology and Evolution</em>, , msz189, <a href="https://doi.org/10.1093/molbev/msz189">https://doi.org/10.1093/molbev/msz189</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=5552155&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1537-1719&amp;title=ModelTest-NG%3A%20A%20New%20and%20Scalable%20Tool%20for%20the%20Selection%20of%20DNA%20and%20Protein%20Evolutionary%20Models&amp;publicationDate=2019-08-21&amp;volumeNum=&amp;issueNum=&amp;author=Darriba%2C%20Diego%3B%20Posada%2C%20David&amp;startPage=1&amp;endPage=4&amp;contentId=10.1093%2Fmolbev%2Fmsz189&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            International Institue of Tropical Agriculture (IITA)
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

International Institue of Tropical Agriculture (IITA)
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2fadvance-article%2fdoi%2f10.1093%2fmolbev%2fmsz189%2f5552155" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option>
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="163539885" id="163539885" class="abstract-title">Abstract</h2>
<section class="abstract"><p>ModelTest-NG is a reimplementation from 
scratch of jModelTest and ProtTest, two popular tools for selecting the 
best-fit nucleotide and amino acid substitution models, respectively. 
ModelTest-NG is one to two orders of magnitude faster than jModelTest 
and ProtTest but equally accurate and introduces several new features, 
such as ascertainment bias correction, mixture, and free-rate models, or
 the automatic processing of single partitions. ModelTest-NG is 
available under a GNU GPL3 license at <a class="link link-uri" href="https://github.com/ddarriba/modeltest" target="">https://github.com/ddarriba/modeltest</a> , last accessed September 2, 2019.</p></section>                        <div class="article-metadata-panel clearfix"><div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic model selection&quot;">phylogenetic model selection</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-performance computing&quot;">high-performance computing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;efficient algorithms&quot;">efficient algorithms</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phylogenetic inference&quot;">phylogenetic inference</a></div><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=Brief%20Communication">Brief Communication</a>
                </div><div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Associate Editor:
                            <span class="al-author-name">
                                <span id="info-21200964" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keith Crandall</div>
            <div class="info-card-role">
                Associate Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Keith+Crandall">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Crandall%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Crandall%20Keith%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Keith Crandall</a>
                            </span>
                        <br>
                    </div>
            </div>
        </div></div>
<p>It is well known that the use of distinct probabilistic models of evolution can change the outcome of phylogenetic analyses (<a reveal-id="msz189-B4" data-open="msz189-B4" class="link link-ref link-reveal xref-bibr">Buckley 2002</a>; <a reveal-id="msz189-B5" data-open="msz189-B5" class="link link-ref link-reveal xref-bibr">Buckley and Cunningham 2002</a>; <a reveal-id="msz189-B15" data-open="msz189-B15" class="link link-ref link-reveal xref-bibr">Lemmon and Moriarty 2004</a>).
 Not surprisingly, a number of bioinformatic tools have been developed 
in the last 20 years for selecting the best-fit model for the data at 
hand (<a reveal-id="msz189-B19" data-open="msz189-B19" class="link link-ref link-reveal xref-bibr">Posada and Crandall 1998</a>; <a reveal-id="msz189-B18" data-open="msz189-B18" class="link link-ref link-reveal xref-bibr">Posada 2008</a>; <a reveal-id="msz189-B6" data-open="msz189-B6" class="link link-ref link-reveal xref-bibr">Darriba et&nbsp;al. 2011</a>, <a reveal-id="msz189-B7" data-open="msz189-B7" class="link link-ref link-reveal xref-bibr">2012</a>; <a reveal-id="msz189-B10" data-open="msz189-B10" class="link link-ref link-reveal xref-bibr">Kalyaanamoorthy et&nbsp;al. 2017</a>). Although <a reveal-id="msz189-B1" data-open="msz189-B1" class="link link-ref link-reveal xref-bibr">Abadi et&nbsp;al. (2019)</a>
 concluded that using a parameter-rich model for DNA data leads to very 
similar inferences as the best-fit models, they average over a number of
 benchmark multiple sequence alignments (MSAs). However, looking at 
individual MSA analyses we may observe in some circumstances substantial
 topological differences between trees inferred under the best-fit model
 and under a parameter-rich GTR (<a reveal-id="msz189-B24" data-open="msz189-B24" class="link link-ref link-reveal xref-bibr">Tavaré 1986</a>) model (<a reveal-id="msz189-B3" data-open="msz189-B3" class="link link-ref link-reveal xref-bibr">Arbiza et&nbsp;al. 2011</a>; <a reveal-id="msz189-B9" data-open="msz189-B9" class="link link-ref link-reveal xref-bibr">Hoff et&nbsp;al. 2016</a>).
 Nowadays, continuous advances in sequencing technologies have made 
possible the assemblage of large MSAs that require faster and more 
scalable tools. In particular, our tools jModelTest (<a reveal-id="msz189-B7" data-open="msz189-B7" class="link link-ref link-reveal xref-bibr">Darriba et&nbsp;al. 2012</a>) and ProtTest (<a reveal-id="msz189-B6" data-open="msz189-B6" class="link link-ref link-reveal xref-bibr">Darriba et&nbsp;al. 2011</a>),
 which are among the most popular tools for DNA and protein model 
selection, despite implementing high-performance computing algorithms 
for parallel execution with dynamic load balancing, still rely on PhyML (<a reveal-id="msz189-B8" data-open="msz189-B8" class="link link-ref link-reveal xref-bibr">Guindon and Gascuel 2003</a>)
 for calculating the maximum likelihood (ML) scores for the competing 
models. This step constitutes the by far most compute-intensive part, 
requiring &gt;99% of overall execution time. PhyML and hence jModelTest 
and ProtTest are relatively inefficient compared with more recent tools 
such as IQ-TREE (<a reveal-id="msz189-B17" data-open="msz189-B17" class="link link-ref link-reveal xref-bibr">Nguyen et&nbsp;al. 2015</a>). The model selection feature of IQ-TREE, called ModelFinder (<a reveal-id="msz189-B10" data-open="msz189-B10" class="link link-ref link-reveal xref-bibr">Kalyaanamoorthy et&nbsp;al. 2017</a>),
 is becoming increasingly popular due to its algorithmic and 
computational efficiency, the wide range of supported evolutionary 
models, and its user-friendliness. Another recently released tool for 
model selection is SMS (Smart Model Selection) (<a reveal-id="msz189-B14" data-open="msz189-B14" class="link link-ref link-reveal xref-bibr">Lefort et&nbsp;al. 2017</a>). SMS is based on PhyML and uses heuristic strategies to avoid evaluating the full set of candidate models.</p><p>Here,
 we introduce ModelTest-NG, a new program that outperforms its 
predecessors jModelTest and ProtTest in terms of speed. ModelTest-NG 
offers a completely redesigned graphical user interface and has several 
new capabilities. Its main features are as follows: </p><ul class="bullet"><li><p><em>Data and models supported</em>:
 ModelTest-NG supports both nucleotide and amino acid models. It uses 
statistical criteria for selecting the best-fit substitution models such
 as AIC <a reveal-id="msz189-B2" data-open="msz189-B2" class="link link-ref link-reveal xref-bibr">Akaike (1974)</a>, BIC <a reveal-id="msz189-B21" data-open="msz189-B21" class="link link-ref link-reveal xref-bibr">Schwarz (1978)</a>, and DT <a reveal-id="msz189-B16" data-open="msz189-B16" class="link link-ref link-reveal xref-bibr">Minin et&nbsp;al. (2003)</a>.
 It can select among all models included in jModelTest and ProtTest plus
 four other empirical amino acid replacement matrices and protein 
mixture models such as LG4M and LG4X (<a reveal-id="msz189-B13" data-open="msz189-B13" class="link link-ref link-reveal xref-bibr">Le et&nbsp;al. 2012</a>). ModelTest-NG can also assess the fit of a free-rate model (<a reveal-id="msz189-B25" data-open="msz189-B25" class="link link-ref link-reveal xref-bibr">Yang 1995</a>).</p></li><li><p><em>Partitioned MSAs</em>:
 ModelTest-NG can automatically perform model selection on single, 
nonoverlapping partitions, as specified by the user (e.g., on a per-gene
 basis, or by codon position).</p></li><li><p><em>Phylogenetic templates</em>: Users can select so-called templates for popular phylogenetic inference tools like RAxML (<a reveal-id="msz189-B22" data-open="msz189-B22" class="link link-ref link-reveal xref-bibr">Stamatakis 2014</a>), RAxML-NG (<a reveal-id="msz189-B12" data-open="msz189-B12" class="link link-ref link-reveal xref-bibr">Kozlov et&nbsp;al. 2019</a>), IQ-TREE, PhyML, PAUP (<a reveal-id="msz189-B23" data-open="msz189-B23" class="link link-ref link-reveal xref-bibr">Swofford 2002</a>), or MrBayes (<a reveal-id="msz189-B20" data-open="msz189-B20" class="link link-ref link-reveal xref-bibr">Ronquist et&nbsp;al. 2012</a>).
 When such a template is specified, ModelTest-NG will only evaluate 
models supported by the given tool and will print out the corresponding 
command line for phylogenetic reconstruction under the best-fit model.</p></li><li><p><em>Native implementation</em>:
 ModelTest-NG constitutes a full reimplementation of jModelTest and 
ProtTest in C++ that relies on a novel and efficient low-level 
implementation of the Phylogenetic Likelihood Library (PLL) (<a class="link link-uri" href="https://github.com/xflouris/libpll-2" target="">https://github.com/xflouris/libpll-2</a>;
 last accessed September 2, 2019). This library encapsulates all 
compute- and memory-intensive phylogenetic likelihood computations and 
fully leverages the capabilities of modern x86 processors by using the 
AVX and AVX2 vector instruction sets. PLL also incorporates a recent 
algorithmic technique for accelerating likelihood calculations (<a reveal-id="msz189-B11" data-open="msz189-B11" class="link link-ref link-reveal xref-bibr">Kobert et&nbsp;al. 2017</a>). All required numerical optimization routines are implemented in the pll-modules library (<a class="link link-uri" href="https://github.com/ddarriba/pll-modules" target="">https://github.com/ddarriba/pll-modules</a>; last accessed September 2, 2019).</p></li><li><p><em>Parallel execution</em>: ModelTest-NG can take advantage of multicore desktop computers and clusters using <em>P</em>Threads and MPI (Message Passing Interface) (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578337366&amp;Signature=WxasLR4jgVQmHAW81ZiHp1%7EpFChROyoOt6QyqpPDShFVE60NWrtO40NcJfFW01OuY3OHiD3DYM3n4Ci45cxMyfMYSmTpU2Pvn7%7EABrJyuYsY%7Eqm7PgK4%7EsjCalrVTY6ksHaaXLyaTA7G%7ErjerCvrPyXaToDEdPCcqVdJNG1GEam4hp7NfmgVQITFCmlatV45dnBVG8Mq6pQcY-uVtotTRfZS6sK0b%7ERwxxzZfo0aU%7EgR6U7CYkgK9nQob5NAeKVXnH6nwVuTzZ7sqUK9h44S8BUoxHJq3EFVGUWV7e1OV%7EaCuyb3O8sm2PxpjMWXF8p4%7EsK%7ECSeqIQsU-vqan08vGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary material, Supplementary Material</a></span> online for details).</p></li></ul><p></p><p>We
 benchmarked ModelTest-NG against jModelTest, ProtTest, and ModelFinder 
(part of IQ-TREE version 1.6.1) using simulated as well as empirical 
data sets. We measured model selection accuracy (i.e., how often the 
generating model is recovered) using the simulated data sets, as well as
 run times. In all cases, we used the default model selection parameter 
settings. The experimental setup is described in detail in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578337366&amp;Signature=WxasLR4jgVQmHAW81ZiHp1%7EpFChROyoOt6QyqpPDShFVE60NWrtO40NcJfFW01OuY3OHiD3DYM3n4Ci45cxMyfMYSmTpU2Pvn7%7EABrJyuYsY%7Eqm7PgK4%7EsjCalrVTY6ksHaaXLyaTA7G%7ErjerCvrPyXaToDEdPCcqVdJNG1GEam4hp7NfmgVQITFCmlatV45dnBVG8Mq6pQcY-uVtotTRfZS6sK0b%7ERwxxzZfo0aU%7EgR6U7CYkgK9nQob5NAeKVXnH6nwVuTzZ7sqUK9h44S8BUoxHJq3EFVGUWV7e1OV%7EaCuyb3O8sm2PxpjMWXF8p4%7EsK%7ECSeqIQsU-vqan08vGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary material, Supplementary Material</a></span> online, where we also further discuss the results.</p><p>ModelTest-NG
 found the true generating model for 81% of the simulated DNA MSAs 
(jModelTest: 81%, ModelFinder: 70%) and for 85% of the simulated protein
 MSAs (ProtTest: 85%, ModelFinder: 87%) (<a reveal-id="msz189-F1" data-open="msz189-F1" class="link link-reveal link-table xref-fig">fig.&nbsp;1</a>).
 In general, the larger the data in terms of number of taxa and number 
of sites, the better ModelTest-NG performs compared with the competing 
tools (see <a reveal-id="msz189-F1" data-open="msz189-F1" class="link link-reveal link-table xref-fig">fig.&nbsp;1</a>). </p>                        <a id="163539891" scrollto-destination="163539891"></a>
<div id="msz189-F1" data-id="msz189-F1" class="fig fig-section"><div class="label fig-label"><span class="small-caps">Fig</span>. 1</div><div class="graphic-wrap"><img class="content-image" src="m_msz189f1.png" alt="Model selection accuracy comparison between ModelTest-NG, jModelTest/ProtTest, and ModelFinder for simulated data (top) and LOESS curved fitted to a scatter plot of ModelTest-NG run times versus ModelFinder for empirical data (bottom), for DNA (left) and protein (right) MSAs. The dashed line represents equal run times." path-from-xml="msz189f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/163539891/msz189f1.tif" path-from-xml="msz189f1.tif" target="_blank">Open in new tab</a><a section="163539891" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189f1.png?Expires=1578337366&amp;Signature=1-ypdU4cj8uoXmN3Sk02U447QjUokkDgjp8Tm1kOCu8PB2tM%7E00-QOpeqWV-c%7ERKuoSqNpQqs%7EmWuG20xXXrsnMHJ0tvnSlU3OeblcLmheXew9WvAUTUJkXVDG3I%7EfPvK4WMQbFhmWDQTFh1vye5K1wo8-xS32SK2ek%7Et9DoSrzE5Z9oF-wDUINMdG2q023WOQc-fwQthN3RSXGuxKYOf2i19lpjXbRczAl7Qol2-m8WJwfHgTuLi2d-YKqShcAHe96IFcvQXSOjFVOZxEfZrMVU5YsEhAO1sZEqh53Bxf2UyYEkfCz%7Ej6AHVMv9iyeUSaUlay2F5uyAQC1PhKAnsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=163539891&amp;ar=5552155&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msz189f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Model
 selection accuracy comparison between ModelTest-NG, 
jModelTest/ProtTest, and ModelFinder for simulated data (top) and LOESS 
curved fitted to a scatter plot of ModelTest-NG run times versus 
ModelFinder for empirical data (bottom), for DNA (left) and protein 
(right) MSAs. The dashed line represents equal run times.</p></div></div><div content-id="msz189-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label"><span class="small-caps">Fig</span>. 1</div><div class="graphic-wrap"><img class="content-image" src="m_msz189f1.png" alt="Model selection accuracy comparison between ModelTest-NG, jModelTest/ProtTest, and ModelFinder for simulated data (top) and LOESS curved fitted to a scatter plot of ModelTest-NG run times versus ModelFinder for empirical data (bottom), for DNA (left) and protein (right) MSAs. The dashed line represents equal run times." path-from-xml="msz189f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/163539891/msz189f1.tif" path-from-xml="msz189f1.tif" target="_blank">Open in new tab</a><a section="163539891" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189f1.png?Expires=1578337366&amp;Signature=1-ypdU4cj8uoXmN3Sk02U447QjUokkDgjp8Tm1kOCu8PB2tM%7E00-QOpeqWV-c%7ERKuoSqNpQqs%7EmWuG20xXXrsnMHJ0tvnSlU3OeblcLmheXew9WvAUTUJkXVDG3I%7EfPvK4WMQbFhmWDQTFh1vye5K1wo8-xS32SK2ek%7Et9DoSrzE5Z9oF-wDUINMdG2q023WOQc-fwQthN3RSXGuxKYOf2i19lpjXbRczAl7Qol2-m8WJwfHgTuLi2d-YKqShcAHe96IFcvQXSOjFVOZxEfZrMVU5YsEhAO1sZEqh53Bxf2UyYEkfCz%7Ej6AHVMv9iyeUSaUlay2F5uyAQC1PhKAnsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=163539891&amp;ar=5552155&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="msz189f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Model
 selection accuracy comparison between ModelTest-NG, 
jModelTest/ProtTest, and ModelFinder for simulated data (top) and LOESS 
curved fitted to a scatter plot of ModelTest-NG run times versus 
ModelFinder for empirical data (bottom), for DNA (left) and protein 
(right) MSAs. The dashed line represents equal run times.</p></div></div><p>In
 terms of speed, on simulated DNA data, ModelTest-NG was 110.77 times 
faster than jModelTest but slower than ModelFinder (the latter was 1.59 
times faster). On empirical DNA data, ModelTest-NG yielded average 
speedups of 510.13 over jModelTest and of 1.24 over ModelFinder (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578337366&amp;Signature=WxasLR4jgVQmHAW81ZiHp1%7EpFChROyoOt6QyqpPDShFVE60NWrtO40NcJfFW01OuY3OHiD3DYM3n4Ci45cxMyfMYSmTpU2Pvn7%7EABrJyuYsY%7Eqm7PgK4%7EsjCalrVTY6ksHaaXLyaTA7G%7ErjerCvrPyXaToDEdPCcqVdJNG1GEam4hp7NfmgVQITFCmlatV45dnBVG8Mq6pQcY-uVtotTRfZS6sK0b%7ERwxxzZfo0aU%7EgR6U7CYkgK9nQob5NAeKVXnH6nwVuTzZ7sqUK9h44S8BUoxHJq3EFVGUWV7e1OV%7EaCuyb3O8sm2PxpjMWXF8p4%7EsK%7ECSeqIQsU-vqan08vGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary fig. S1</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578337366&amp;Signature=WxasLR4jgVQmHAW81ZiHp1%7EpFChROyoOt6QyqpPDShFVE60NWrtO40NcJfFW01OuY3OHiD3DYM3n4Ci45cxMyfMYSmTpU2Pvn7%7EABrJyuYsY%7Eqm7PgK4%7EsjCalrVTY6ksHaaXLyaTA7G%7ErjerCvrPyXaToDEdPCcqVdJNG1GEam4hp7NfmgVQITFCmlatV45dnBVG8Mq6pQcY-uVtotTRfZS6sK0b%7ERwxxzZfo0aU%7EgR6U7CYkgK9nQob5NAeKVXnH6nwVuTzZ7sqUK9h44S8BUoxHJq3EFVGUWV7e1OV%7EaCuyb3O8sm2PxpjMWXF8p4%7EsK%7ECSeqIQsU-vqan08vGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>
 online). On simulated protein data, ModelTest-NG yielded average 
speedups of 36.07 over ProtTest, and similar run times as ModelFinder. 
On empirical protein data, ModelTest-NG was 36.94 times faster than 
ProtTest, and 1.19 times faster than ModelFinder. Importantly, 
ModelTest-NG seems to scale better than ModelFinder and 
jModelTest/ProtTest on large MSAs.</p><p>To ensure a fairer comparison 
with SMS, which only considers a subset of the models, we conducted a 
separate set of experiments comparing only ModelTest-NG and SMS on 
empirical data. For DNA data, both tools selected the same model 80% of 
the time, while ModelTest-NG was 95.53 times faster. For protein data, 
ModelTest-NG selected the same model as SMS 86.7% of the time and was 
17.20 times faster.</p><p>The thoroughness of the model parameter 
optimization routines in ModelTest-NG can be controlled by the user. In 
additional experiments, we found that the more thoroughly we optimize 
the likelihood score the more accurate the selected model becomes (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578337366&amp;Signature=WxasLR4jgVQmHAW81ZiHp1%7EpFChROyoOt6QyqpPDShFVE60NWrtO40NcJfFW01OuY3OHiD3DYM3n4Ci45cxMyfMYSmTpU2Pvn7%7EABrJyuYsY%7Eqm7PgK4%7EsjCalrVTY6ksHaaXLyaTA7G%7ErjerCvrPyXaToDEdPCcqVdJNG1GEam4hp7NfmgVQITFCmlatV45dnBVG8Mq6pQcY-uVtotTRfZS6sK0b%7ERwxxzZfo0aU%7EgR6U7CYkgK9nQob5NAeKVXnH6nwVuTzZ7sqUK9h44S8BUoxHJq3EFVGUWV7e1OV%7EaCuyb3O8sm2PxpjMWXF8p4%7EsK%7ECSeqIQsU-vqan08vGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>
 online). Possibly, the slight loss of accuracy in ModelFinder in our 
simulations can be explained by a less thorough default model 
optimization setting than in ModelTest-NG.</p><p>ModelTest-NG represents
 a substantial improvement over our previous tools, jModelTest and 
ProtTest. Although being equally accurate, it is up to two orders of 
magnitude faster on empirical data. Compared with ModelFinder, we 
observed similar run times for empirical data sets, but ModelFinder was 
faster on synthetic MSAs, particularly on DNA data. However, the 
accuracy of ModelFinder on DNA data was substantially lower than for 
ModelTest-NG (70% vs. 81%, respectively). In future versions of 
ModelTest-NG, we intend to introduce new methods to dynamically 
determine the optimal speed/accuracy tradeoff for the data set at hand. 
ModelTest-NG is particularly well suited for analyzing large data sets.</p>                    <h2 scrollto-destination="163539896" id="163539896" class="section-title">Supplementary Material</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578337366&amp;Signature=WxasLR4jgVQmHAW81ZiHp1%7EpFChROyoOt6QyqpPDShFVE60NWrtO40NcJfFW01OuY3OHiD3DYM3n4Ci45cxMyfMYSmTpU2Pvn7%7EABrJyuYsY%7Eqm7PgK4%7EsjCalrVTY6ksHaaXLyaTA7G%7ErjerCvrPyXaToDEdPCcqVdJNG1GEam4hp7NfmgVQITFCmlatV45dnBVG8Mq6pQcY-uVtotTRfZS6sK0b%7ERwxxzZfo0aU%7EgR6U7CYkgK9nQob5NAeKVXnH6nwVuTzZ7sqUK9h44S8BUoxHJq3EFVGUWV7e1OV%7EaCuyb3O8sm2PxpjMWXF8p4%7EsK%7ECSeqIQsU-vqan08vGA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>Molecular Biology and Evolution</em> online.</p>                    <h2 scrollto-destination="163539898" id="163539898" class="backacknowledgements-title" data-legacy-id="ack1">Acknowledgments</h2>
<p>This work was supported by the Ministry of Economy and 
Competitiveness of Spain and FEDER funds of the EU (Project 
TIN2016-75845-P) and by the Galician Government (Xunta de Galicia) under
 the Consolidation Program of Competitive Research (ref. ED431C 
2017/04). Part of this work was funded by the Klaus Tschira Foundation 
and DFG grant STA-860/6.</p>                    <h2 scrollto-destination="163539900" id="163539900" class="backreferences-title" data-legacy-id="R1">References</h2>
<div class="ref-list"><div content-id="msz189-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B1" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abadi</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Azouri</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Pupko</div> <div class="given-names">T</div></div>, <div class="name string-name"><div class="surname">Mayrose</div> <div class="given-names">I.</div></div></div> <div class="year">2019</div>. <div class="article-title">Model selection may not be a mandatory step for phylogeny reconstruction</div>. <div class="source">Nat Commun</div>&nbsp;. <div class="volume">10</div>(<div class="issue">1</div>):<div class="fpage">934.</div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Model%20selection%20may%20not%20be%20a%20mandatory%20step%20for%20phylogeny%20reconstruction&amp;author=S%20Abadi&amp;author=D%20Azouri&amp;author=T%20Pupko&amp;author=I.%20Mayrose&amp;publication_year=2019&amp;journal=Nat%20Commun&amp;volume=10&amp;pages=934." target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/s41467-019-08822-w" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41467-019-08822-w" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fs41467-019-08822-w"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/30804347" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Model%20selection%20may%20not%20be%20a%20mandatory%20step%20for%20phylogeny%20reconstruction&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B2" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akaike</div> <div class="given-names">H.</div></div></div> <div class="year">1974</div>. <div class="article-title">A new look at the statistical model identification</div>. <div class="source">IEEE Trans Automat Contr</div>&nbsp;. <div class="volume">19</div>(<div class="issue">6</div>):<div class="fpage">716</div>–<div class="lpage">723</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20new%20look%20at%20the%20statistical%20model%20identification&amp;author=H.%20Akaike&amp;publication_year=1974&amp;journal=IEEE%20Trans%20Automat%20Contr&amp;volume=19&amp;pages=716-723" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1109/TAC.1974.1100705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTAC.1974.1100705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1109%2FTAC.1974.1100705"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20new%20look%20at%20the%20statistical%20model%20identification&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B3" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Arbiza</div> <div class="given-names">L</div></div>, <div class="name string-name"><div class="surname">Patricio</div> <div class="given-names">M</div></div>, <div class="name string-name"><div class="surname">Dopazo</div> <div class="given-names">H</div></div>, <div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div> <div class="year">2011</div>. <div class="article-title">Genome-wide heterogeneity of nucleotide substitution model fit</div>. <div class="source">Genome Biol Evol</div>&nbsp;. <div class="volume">3</div>:<div class="fpage">896</div>–<div class="lpage">908</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20heterogeneity%20of%20nucleotide%20substitution%20model%20fit&amp;author=L%20Arbiza&amp;author=M%20Patricio&amp;author=H%20Dopazo&amp;author=D.%20Posada&amp;publication_year=2011&amp;journal=Genome%20Biol%20Evol&amp;volume=3&amp;pages=896-908" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/gbe/evr080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevr080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fgbe%2Fevr080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21824869" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Genome-wide%20heterogeneity%20of%20nucleotide%20substitution%20model%20fit&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B4" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buckley</div> <div class="given-names">TR.</div></div></div> <div class="year">2002</div>. <div class="article-title">Model misspecification and probabilistic tests of topology: evidence from empirical data sets</div>. <div class="source">Syst Biol</div>&nbsp;. <div class="volume">51</div>(<div class="issue">3</div>):<div class="fpage">509</div>–<div class="lpage">523</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Model%20misspecification%20and%20probabilistic%20tests%20of%20topology%3A%20evidence%20from%20empirical%20data%20sets&amp;author=TR.%20Buckley&amp;publication_year=2002&amp;journal=Syst%20Biol&amp;volume=51&amp;pages=509-523" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150290069922" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150290069922" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150290069922"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12079647" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Model%20misspecification%20and%20probabilistic%20tests%20of%20topology%3A%20evidence%20from%20empirical%20data%20sets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B5" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buckley</div> <div class="given-names">TR</div></div>, <div class="name string-name"><div class="surname">Cunningham</div> <div class="given-names">CW.</div></div></div> <div class="year">2002</div>. <div class="article-title">The effects of nucleotide substitution model assumptions on estimates of nonparametric bootstrap support</div>. <div class="source">Mol Biol Evol</div>&nbsp;. <div class="volume">19</div>(<div class="issue">4</div>):<div class="fpage">394</div>–<div class="lpage">405</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20effects%20of%20nucleotide%20substitution%20model%20assumptions%20on%20estimates%20of%20nonparametric%20bootstrap%20support&amp;author=TR%20Buckley&amp;author=CW.%20Cunningham&amp;publication_year=2002&amp;journal=Mol%20Biol%20Evol&amp;volume=19&amp;pages=394-405" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a004094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a004094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11919280" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20effects%20of%20nucleotide%20substitution%20model%20assumptions%20on%20estimates%20of%20nonparametric%20bootstrap%20support&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B6" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darriba</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Taboada</div> <div class="given-names">GL</div></div>, <div class="name string-name"><div class="surname">Doallo</div> <div class="given-names">R</div></div>, <div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div> <div class="year">2011</div>. <div class="article-title">ProtTest 3: fast selection of best-fit models of protein evolution</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="volume">27</div>(<div class="issue">8</div>):<div class="fpage">1164</div>–<div class="lpage">1165</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ProtTest%203%3A%20fast%20selection%20of%20best-fit%20models%20of%20protein%20evolution&amp;author=D%20Darriba&amp;author=GL%20Taboada&amp;author=R%20Doallo&amp;author=D.%20Posada&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1164-1165" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btr088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtr088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21335321" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ProtTest%203%3A%20fast%20selection%20of%20best-fit%20models%20of%20protein%20evolution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B7" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darriba</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Taboada</div> <div class="given-names">GL</div></div>, <div class="name string-name"><div class="surname">Doallo</div> <div class="given-names">R</div></div>, <div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div> <div class="year">2012</div>. <div class="article-title">jModelTest 2: more models, new heuristics and parallel computing</div>. <div class="source">Nat Methods</div>&nbsp;. <div class="volume">9</div>(<div class="issue">8</div>):<div class="fpage">772</div>–<div class="lpage">772</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=jModelTest%202%3A%20more%20models%2C%20new%20heuristics%20and%20parallel%20computing&amp;author=D%20Darriba&amp;author=GL%20Taboada&amp;author=R%20Doallo&amp;author=D.%20Posada&amp;publication_year=2012&amp;journal=Nat%20Methods&amp;volume=9&amp;pages=772-772" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.2109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.2109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22847109" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:jModelTest%202%3A%20more%20models%2C%20new%20heuristics%20and%20parallel%20computing&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B8" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">O</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>. <div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>. <div class="source">Syst Biol</div>&nbsp;. <div class="volume">52</div>(<div class="issue">5</div>):<div class="fpage">696</div>–<div class="lpage">704</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=O%20Guindon&amp;author=S.%20Gascuel&amp;publication_year=2003&amp;journal=Syst%20Biol&amp;volume=52&amp;pages=696-704" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150390235520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B9" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoff</div> <div class="given-names">M</div></div>, <div class="name string-name"><div class="surname">Orf</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Riehm</div> <div class="given-names">B</div></div>, <div class="name string-name"><div class="surname">Darriba</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> <div class="year">2016</div>. <div class="article-title">Does the choice of nucleotide substitution models matter topologically?</div> <div class="source">BMC Bioinformatics</div>&nbsp; <div class="volume">17</div>(<div class="issue">1</div>):<div class="fpage">143.</div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Does%20the%20choice%20of%20nucleotide%20substitution%20models%20matter%20topologically%3F&amp;author=M%20Hoff&amp;author=S%20Orf&amp;author=B%20Riehm&amp;author=D%20Darriba&amp;author=A.%20Stamatakis&amp;publication_year=2016&amp;journal=BMC%20Bioinformatics&amp;volume=17&amp;pages=143." target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/s12859-016-0985-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12859-016-0985-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2Fs12859-016-0985-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27009141" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Does%20the%20choice%20of%20nucleotide%20substitution%20models%20matter%20topologically%3F&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B10" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kalyaanamoorthy</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Minh</div> <div class="given-names">BQ</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">TK</div></div>, <div class="name string-name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Jermiin</div> <div class="given-names">LS.</div></div></div> <div class="year">2017</div>. <div class="article-title">Modelfinder: fast model selection for accurate phylogenetic estimates</div>. <div class="source">Nat Methods</div>&nbsp;. <div class="volume">14</div>(<div class="issue">6</div>):<div class="fpage">587.</div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Modelfinder%3A%20fast%20model%20selection%20for%20accurate%20phylogenetic%20estimates&amp;author=S%20Kalyaanamoorthy&amp;author=BQ%20Minh&amp;author=TK%20Wong&amp;author=A%20von%20Haeseler&amp;author=LS.%20Jermiin&amp;publication_year=2017&amp;journal=Nat%20Methods&amp;volume=14&amp;pages=587." target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/nmeth.4285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.4285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2Fnmeth.4285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28481363" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Modelfinder%3A%20fast%20model%20selection%20for%20accurate%20phylogenetic%20estimates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B11" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobert</div> <div class="given-names">K</div></div>, <div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Flouri</div> <div class="given-names">T.</div></div></div> <div class="year">2017</div>. <div class="article-title">Efficient detection of repeating sites to accelerate phylogenetic likelihood calculations</div>. <div class="source">Syst Biol</div>&nbsp;. <div class="volume">66</div>(<div class="issue">2</div>):<div class="fpage">205</div>–<div class="lpage">217</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Efficient%20detection%20of%20repeating%20sites%20to%20accelerate%20phylogenetic%20likelihood%20calculations&amp;author=K%20Kobert&amp;author=A%20Stamatakis&amp;author=T.%20Flouri&amp;publication_year=2017&amp;journal=Syst%20Biol&amp;volume=66&amp;pages=205-217" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27576546" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Efficient%20detection%20of%20repeating%20sites%20to%20accelerate%20phylogenetic%20likelihood%20calculations&amp;aulast=Kobert&amp;date=2017&amp;spage=205&amp;epage=217&amp;volume=66&amp;issue=2" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Efficient%20detection%20of%20repeating%20sites%20to%20accelerate%20phylogenetic%20likelihood%20calculations&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B12" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kozlov</div> <div class="given-names">AM</div></div>, <div class="name string-name"><div class="surname">Darriba</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Flouri</div> <div class="given-names">T</div></div>, <div class="name string-name"><div class="surname">Morel</div> <div class="given-names">B</div></div>, <div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> Forthcoming <div class="year">2019</div>. <div class="article-title">RAxML-NG: a fast, scalable and user-friendly tool for maximum likelihood phylogenetic inference</div>. <div class="source">Bioinformatics</div>&nbsp; pii: btz305.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=RAxML-NG%3A%20a%20fast%2C%20scalable%20and%20user-friendly%20tool%20for%20maximum%20likelihood%20phylogenetic%20inference&amp;aulast=Kozlov&amp;date=2019" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML-NG%3A%20a%20fast%2C%20scalable%20and%20user-friendly%20tool%20for%20maximum%20likelihood%20phylogenetic%20inference&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B13" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le</div> <div class="given-names">SQ</div></div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">CC</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2012</div>. <div class="article-title">Modeling protein evolution with several amino acid replacement matrices depending on site rates</div>. <div class="source">Mol Biol Evol</div>&nbsp;. <div class="volume">29</div>(<div class="issue">10</div>):<div class="fpage">2921</div>–<div class="lpage">2936</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Modeling%20protein%20evolution%20with%20several%20amino%20acid%20replacement%20matrices%20depending%20on%20site%20rates&amp;author=SQ%20Le&amp;author=CC%20Dang&amp;author=O.%20Gascuel&amp;publication_year=2012&amp;journal=Mol%20Biol%20Evol&amp;volume=29&amp;pages=2921-2936" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/mss112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmss112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22491036" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Modeling%20protein%20evolution%20with%20several%20amino%20acid%20replacement%20matrices%20depending%20on%20site%20rates&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B14" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lefort</div> <div class="given-names">V</div></div>, <div class="name string-name"><div class="surname">Longueville</div> <div class="given-names">J-E</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2017</div>. <div class="article-title">SMS: Smart Model Selection in PhyML</div>. <div class="source">Mol Biol Evol</div>&nbsp;. <div class="volume">34</div>(<div class="issue">9</div>):<div class="fpage">2422</div>–<div class="lpage">2424</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=SMS%3A%20Smart%20Model%20Selection%20in%20PhyML&amp;author=V%20Lefort&amp;author=J-E%20Longueville&amp;author=O.%20Gascuel&amp;publication_year=2017&amp;journal=Mol%20Biol%20Evol&amp;volume=34&amp;pages=2422-2424" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msx149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsx149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsx149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28472384" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:SMS%3A%20Smart%20Model%20Selection%20in%20PhyML&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B15" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lemmon</div> <div class="given-names">AR</div></div>, <div class="name string-name"><div class="surname">Moriarty</div> <div class="given-names">EC.</div></div></div> <div class="year">2004</div>. <div class="article-title">The importance of proper model assumption in Bayesian phylogenetic</div>. <div class="source">Syst Biol</div>&nbsp;. <div class="volume">53</div>(<div class="issue">2</div>):<div class="fpage">265</div>–<div class="lpage">277</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20importance%20of%20proper%20model%20assumption%20in%20Bayesian%20phylogenetic&amp;author=AR%20Lemmon&amp;author=EC.%20Moriarty&amp;publication_year=2004&amp;journal=Syst%20Biol&amp;volume=53&amp;pages=265-277" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150490423520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150490423520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150490423520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15205052" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20importance%20of%20proper%20model%20assumption%20in%20Bayesian%20phylogenetic&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B16" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Minin</div> <div class="given-names">V</div></div>, <div class="name string-name"><div class="surname">Abdo</div> <div class="given-names">Z</div></div>, <div class="name string-name"><div class="surname">Joyce</div> <div class="given-names">P</div></div>, <div class="name string-name"><div class="surname">Sullivan</div> <div class="given-names">J.</div></div></div> <div class="year">2003</div>. <div class="article-title">Performance-based selection of likelihood models for phylogeny estimation</div>. <div class="source">Syst Biol</div>&nbsp;. <div class="volume">52</div>(<div class="issue">5</div>):<div class="fpage">674</div>–<div class="lpage">683</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Performance-based%20selection%20of%20likelihood%20models%20for%20phylogeny%20estimation&amp;author=V%20Minin&amp;author=Z%20Abdo&amp;author=P%20Joyce&amp;author=J.%20Sullivan&amp;publication_year=2003&amp;journal=Syst%20Biol&amp;volume=52&amp;pages=674-683" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150390235494" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235494" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150390235494"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530134" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Performance-based%20selection%20of%20likelihood%20models%20for%20phylogeny%20estimation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B17" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">L-T</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">HA</div></div>, <div class="name string-name"><div class="surname">von Haeseler</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Minh</div> <div class="given-names">BQ.</div></div></div> <div class="year">2015</div>. <div class="article-title">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies</div>. <div class="source">Mol Biol Evol</div>&nbsp;. <div class="volume">32</div>(<div class="issue">1</div>):<div class="fpage">268</div>–<div class="lpage">274</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;author=L-T%20Nguyen&amp;author=HA%20Schmidt&amp;author=A%20von%20Haeseler&amp;author=BQ.%20Minh&amp;publication_year=2015&amp;journal=Mol%20Biol%20Evol&amp;volume=32&amp;pages=268-274" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msu300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsu300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25371430" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:IQ-TREE%3A%20a%20fast%20and%20effective%20stochastic%20algorithm%20for%20estimating%20maximum-likelihood%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B18" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>. <div class="article-title">jModelTest: phylogenetic model averaging</div>. <div class="source">Mol Biol Evol</div>&nbsp;. <div class="volume">25</div>(<div class="issue">7</div>):<div class="fpage">1253</div>–<div class="lpage">1256</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=jModelTest%3A%20phylogenetic%20model%20averaging&amp;author=D.%20Posada&amp;publication_year=2008&amp;journal=Mol%20Biol%20Evol&amp;volume=25&amp;pages=1253-1256" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msn083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsn083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18397919" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:jModelTest%3A%20phylogenetic%20model%20averaging&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B19" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D</div></div>, <div class="name string-name"><div class="surname">Crandall</div> <div class="given-names">KA.</div></div></div> <div class="year">1998</div>. <div class="article-title">MODELTEST: testing the model of DNA substitution</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="volume">14</div>(<div class="issue">9</div>):<div class="fpage">817</div>–<div class="lpage">818</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MODELTEST%3A%20testing%20the%20model%20of%20DNA%20substitution&amp;author=D%20Posada&amp;author=KA.%20Crandall&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=817-818" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/14.9.817" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.9.817" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2F14.9.817"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9918953" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MODELTEST%3A%20testing%20the%20model%20of%20DNA%20substitution&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B20" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ronquist</div> <div class="given-names">F</div></div>, <div class="name string-name"><div class="surname">Teslenko</div> <div class="given-names">M</div></div>, <div class="name string-name"><div class="surname">Van Der Mark</div> <div class="given-names">P</div></div>, <div class="name string-name"><div class="surname">Ayres</div> <div class="given-names">DL</div></div>, <div class="name string-name"><div class="surname">Darling</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Höhna</div> <div class="given-names">S</div></div>, <div class="name string-name"><div class="surname">Larget</div> <div class="given-names">B</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">L</div></div>, <div class="name string-name"><div class="surname">Suchard</div> <div class="given-names">MA</div></div>, <div class="name string-name"><div class="surname">Huelsenbeck</div> <div class="given-names">JP.</div></div></div> <div class="year">2012</div>. <div class="article-title">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space</div>. <div class="source">Syst Biol</div>&nbsp;. <div class="volume">61</div>(<div class="issue">3</div>):<div class="fpage">539</div>–<div class="lpage">542</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MrBayes%203.2%3A%20efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space&amp;author=F%20Ronquist&amp;author=M%20Teslenko&amp;author=P%20Van%20Der%20Mark&amp;author=DL%20Ayres&amp;author=A%20Darling&amp;author=S%20H%C3%B6hna&amp;author=B%20Larget&amp;author=L%20Liu&amp;author=MA%20Suchard&amp;author=JP.%20Huelsenbeck&amp;publication_year=2012&amp;journal=Syst%20Biol&amp;volume=61&amp;pages=539-542" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/sys029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357727" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MrBayes%203.2%3A%20efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B21" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schwarz</div> <div class="given-names">G.</div></div></div> <div class="year">1978</div>. <div class="article-title">Estimating the dimension of a model</div>. <div class="source">Ann Statist</div>&nbsp;. <div class="volume">6</div>(<div class="issue">2</div>):<div class="fpage">461</div>–<div class="lpage">464</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Estimating%20the%20dimension%20of%20a%20model&amp;author=G.%20Schwarz&amp;publication_year=1978&amp;journal=Ann%20Statist&amp;volume=6&amp;pages=461-464" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1214/aos/1176344136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Faos%2F1176344136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1214%2Faos%2F1176344136"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Estimating%20the%20dimension%20of%20a%20model&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B22" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>. <div class="article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</div>. <div class="source">Bioinformatics</div>&nbsp; <div class="volume">30</div>(<div class="issue">9</div>):<div class="fpage">1312</div>–<div class="lpage">1313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1312-1313" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B23" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Swofford</div> <div class="given-names">DL.</div></div></div> <div class="year">2002</div>. <div class="source">Paup*: phylogenetic analysis using parsimony (and other methods) version 4</div>&nbsp;. <div class="publisher-loc">Sunderland (MA</div>): <div class="publisher-name">Sinauer Associates</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Paup*%3A%20phylogenetic%20analysis%20using%20parsimony%20%28and%20other%20methods%29%20version%204&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Paup*%3A%20phylogenetic%20analysis%20using%20parsimony%20(and%20other%20methods)%20version%204&amp;aulast=Swofford&amp;date=2002" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Paup*%3A%20phylogenetic%20analysis%20using%20parsimony%20%28and%20other%20methods%29%20version%204&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Paup*%3A%20phylogenetic%20analysis%20using%20parsimony%20%28and%20other%20methods%29%20version%204">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B24" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tavaré</div> <div class="given-names">S.</div></div></div> <div class="year">1986</div>. <div class="article-title">Some probabilistic and statistical problems in the analysis of DNA sequences</div>. <div class="source">Lectures Math Life Sci</div>&nbsp;. <div class="volume">17</div>(<div class="issue">2</div>):<div class="fpage">57</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Some%20probabilistic%20and%20statistical%20problems%20in%20the%20analysis%20of%20DNA%20sequences&amp;aulast=Tavar%C3%A9&amp;date=1986&amp;spage=57&amp;epage=86&amp;volume=17&amp;issue=2" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Some%20probabilistic%20and%20statistical%20problems%20in%20the%20analysis%20of%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="msz189-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="msz189-B25" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">1995</div>. <div class="article-title">A space-time process model for the evolution of DNA sequences</div>. <div class="source">Genetics</div>&nbsp; <div class="volume">139</div>(<div class="issue">2</div>):<div class="fpage">993</div>–<div class="lpage">1005</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20space-time%20process%20model%20for%20the%20evolution%20of%20DNA%20sequences&amp;author=Z.%20Yang&amp;publication_year=1995&amp;journal=Genetics&amp;volume=139&amp;pages=993-1005" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7713447" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=A%20space-time%20process%20model%20for%20the%20evolution%20of%20DNA%20sequences&amp;aulast=Yang&amp;date=1995&amp;spage=993&amp;epage=1005&amp;volume=139&amp;issue=2" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20space-time%20process%20model%20for%20the%20evolution%20of%20DNA%20sequences&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">© The Author(s) 2019. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=5552155&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5325">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/PAP/10.1093_molbev_msz189/2/msz189_supplementary_data.zip?Expires=1578380248&amp;Signature=4lRt%7Esyjqv6%7Ec1HlrCxEngjoCpPRtGZBEAqHJb8GV7bh5H4sFsySrGyLhqWD3rwUHxqfCl9o2qL20fOymKpPHq%7EQvFXGSPfZZJK0LxpXIV9CeazPIJ%7EoIWs3nBqiYSTyLmbSewIcNwoqsNA9BYPqbYFUYILk84f4ShECQS%7EngPvRFez8GTOHcxlaI3%7EqtqxXgF84QDXP9cGemDzOs%7Ebxzsr49YCq%7Emd36CzFkYREfXHYiuj0RFyM3-OCi-IiSCilvrfPCX6idBZ%7EBu5QHcpJ6VHPSsfppTiJjnZt5vk0fJFn6quz3JdFLyHyXOrWHHuCvdvnE9YzSCc2MLDK4mqUvw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">msz189_Supplementary_Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">725</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    

            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/molbev/msz189" data-uuid="0165d2af-11cc-f1c6-e008-8221d19df24c"><a target="" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=65242157" style="display:inline-block;">
        <img alt="Article has an altmetric score of 14" src="a.png" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5552155"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fmbe%2fadvance-article%2fdoi%2f10.1093%2fmolbev%2fmsz189%2f5552155">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fmbe%2Fadvance-article%2Fdoi%2F10.1093%2Fmolbev%2Fmsz189%2F5552155" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/mbe/crossref-citedby/5552155">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz283/5645182?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Mutations beget more mutations – Rapid 
evolution of mutation rate in response to the risk of runaway 
accumulation
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz274/5644344?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                The loci of behavioral evolution: 
evidence that  Fas2  and  tilB  underlie differences in pupation site 
choice behavior between  Drosophila melanogaster  and  D. simulans
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz269/5625936?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Visual opsin diversity in sharks and rays
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz275/5637232?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Gene duplication accelerates the pace of protein gain and loss from plant organelles
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
                <div class="widget-dynamic-entry">
                    



                    
                        <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz276/5637231?searchresult=1">
                                                        <div class="widget-dynamic-journal-title">
                                Machine Learning Methods for Predicting 
Human-adaptative Influenza A Viruses Based on Viral Nucleotide 
Compositions
                            </div>
                        </a>
                    <div class="widget-dynamic-journal-image-synopsis">
                            <div class="widget-dynamic-journal-synopsis">
                                
                            </div> <!-- /.widget-dynamic-journal-synopsis -->
                    </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="ModelTest-NG: A New and Scalable Tool for the Selection of DNA and Protein Evolutionary Models | Molecular Biology and Evolution | Oxford Academic" type="hidden">
<input id="hfArticleLeftNavStickyOffset" name="hfArticleLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="addthis_widget.js"></script>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/molbev/msz189");
            googletag.enableServices();
        });
    </script>

        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/mbe/pages/About">About Molecular Biology and Evolution</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/mbe/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/mbe/pages/Policies">Policies</a>
                                </li>
                                <li class="link-4">
                                    <a href="https://academic.oup.com/mbe/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://www.smbe.org/smbe/">Contact Us</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://twitter.com/OfficialSMBE">Twitter</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/mbe/subscribe">Purchase</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-5">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
                    <a href="https://academic.oup.com/mbe" target="">
                        <img id="footer-logo-MolecularBiologyandEvolution" class="journal-footer-affiliations-logo" src="mbe_f1-2062292105.svg" alt="Molecular Biology and Evolution">
                    </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1537-1719</li>
                                            <li>Print ISSN 0737-4038</li>
                    <li>Copyright © 2019 Society for Molecular Biology and Evolution</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh138" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1575382853664&amp;tmr=load%3D1575382853628%26core%3D1575382853656%26main%3D1575382853660%26ifr%3D1575382853668&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fmbe%2Fadvance-article%2Fdoi%2F10.1093%2Fmolbev%2Fmsz189%2F5552155&amp;href=https%3A%2F%2Facademic.oup.com%2Fmbe%2Fadvance-article%2Fdoi%2F10.1093%2Fmolbev%2Fmsz189%2F5552155&amp;dt=ModelTest-NG%3A%20A%20New%20and%20Scalable%20Tool%20for%20the%20Selection%20of%20DNA%20and%20Protein%20Evolutionary%20Models&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5de66f45e8d64a0c&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3DModelTest-NG%253A%2520A%2520New%2520and%2520Scalable%2520Tool%2520for%2520the%2520Selection%2520of%2520DNA%2520and%2520Protein%2520Evolutionary%2520Models%26description%3DAbstract.%2520%2520ModelTest-NG%2520is%2520a%2520reimplementation%2520from%2520scratch%2520of%2520jModelTest%2520and%2520ProtTest%252C%2520two%2520popular%2520tools%2520for%2520selecting%2520the%2520best-fit%2520nucleotide%2520and%2520amino%2520acid%2520su%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fmbe%252Fadvance-article%252Fdoi%252F10.1093%252Fmolbev%252Fmsz189%252F5552155%26image%3D%252F%252Foup.silverchair-cdn.com%252FUI%252Fapp%252Fimg%252Fgeneric-cover.png%26updated_time%3D2019-08-21&amp;csi=undefined&amp;rev=v8.28.1-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2fadvance-article%2fdoi%2f10.1093%2fmolbev%2fmsz189%2f5552155">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script type="text/javascript" src="MathJax.js"></script>

    <script type="text/javascript" src="footer.js"></script>


    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="msz189.js"></script></body></html>