<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="a6cc8659-de88-4f6a-b225-e64629d53cab" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";requestedJournal:journal:1365294x;page:string:Article/Chapter View;ctype:string:Journal Content;website:website:pericles;issue:issue:doi\:10.1111/mec.2012.21.issue-14;article:article:doi\:10.1111/j.1365-294X.2012.05622.x;journal:journal:1365294x;wgroup:string:Publication Websites;pageGroup:string:Publication Pages;subPage:string:Full Text">









<iframe style="position: absolute; top: -10000px; left: -1000px;"></iframe><script type="text/javascript" async="" src="conversion_async.js"></script><script async="" src="prum.js"></script><script src="iframe_api.js"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script async="" src="fbevents.js"></script><script type="text/javascript">var $DoubleClickZone = "molec-eco_mec";var $DoubleClickSite =  "wly.ecology_000046";</script><script>digitalData = {"site":{"ip":"105.27.236.251","environment":"LIVE","website":"onlinelibrary.wiley.com","serverDate":"2019-10-18","server":"web119"},"page":{"pageGroup":"Publication Pages","page":"Article/Chapter View","subPage":"Full Text","title":"title","errorCode":200},"identities":[{"type":"BasicGroup","uuid":"be20949b-b372-471e-96a5-9963c7266a9d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000054870"}]},{"type":"BasicGroup","uuid":"98ba5e39-8ca0-4d0a-addd-395777fbf515","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000034166"}]},{"type":"InstitutionUser","uuid":"6e97ed1f-543d-493d-b458-2e63cb8fbde5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#KLISC"}]},{"type":"BasicGroup","uuid":"e623deab-c83b-41e3-84a2-d9f1d46c5f17"},{"type":"ReferrerUser","uuid":"5a52f27c-b57e-4a2d-8de6-c4f2e0fa8465"},{"type":"ReferrerUser","uuid":"46dc6b9f-376c-41f5-b4df-901d91d26006"},{"type":"RegionUser","uuid":"53f1676b-6f75-446d-8831-5915e1edce59"},{"type":"InstitutionUser","uuid":"dab9a169-ec74-4502-8066-0f76387b0f4e","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASP"}]},{"type":"BasicGroup","uuid":"40e3f768-07c0-45e5-aa81-97aef6eb686c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000005407"}]},{"type":"BasicGroup","uuid":"c3199058-0f44-4f36-b385-3a7c0023d85c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"KENYA HINARI NPL"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"5196cf05-b385-4de3-91a2-2d5ffd18abc5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASPKEN"}]},{"type":"InstitutionUser","uuid":"480e4dba-ca42-475f-a7ca-5b6fbfad49f7","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000014668"}]}],"publication":{"item":{"title":"Trees and/or networks to display intraspecific DNA sequence variation?","doi":"10.1111/j.1365-294X.2012.05622.x","access":"free","earliestDate":"2012-05-18","formats":["Abstract","Cited by","First page image","Full text","PDF","References"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"group":{"doi":"10.1111/mec.2012.21.issue-14","access":"full","volume":"21","issue":"14","coverDate":"2012-07-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"}]},"series":{"title":"Molecular Ecology","doi":"10.1111/(ISSN)1365-294X","access":"no","code":"1365294x","issn":"0962-1083","eissn":"1365-294X","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"General Chemistry","topicUri":"ch00"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecological Genetics","topicUri":"ls2f"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolution","topicUri":"ls40"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Evolution","topicUri":"ls43"},{"taxonomyUri":"global-subject-codes","topicLabel":"Evolutionary Biology","topicUri":"ls41"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Molecular Ecology","topicUri":"ls2g"},{"taxonomyUri":"global-subject-codes","topicLabel":"CHEMISTRY","topicUri":"ch"},{"taxonomyUri":"global-subject-codes","topicLabel":"Genetics","topicUri":"ls60"},{"taxonomyUri":"gpt-zonename","topicLabel":"molec-eco_MEC","topicUri":"molec-eco_mec"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.ecology_000046","topicUri":"wly.ecology_000046"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    






    <meta property="og:title" content="Trees and/or networks to display intraspecific DNA sequence variation?">


    <meta property="og:type" content="Article">


        <meta property="og:url" content="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05622.x">
    

        
        <meta property="og:image" content="https://wol-prod-cdn.literatumonline.com/cms/attachment/5bccabe3-f673-4f05-8647-c009a8afb5d8/mec.2012.21.issue-14.cover.gif">
    

    <meta property="og:site_name" content="Wiley Online Library">



        

        <meta property="og:description" content="Phylogenetic trees and networks are both used in the scientific literature to display DNA sequence variation at the intraspecific level. Should we rather use trees or networks? I argue that the proce...">
    


    <meta name="viewport" content="width=device-width,initial-scale=1">




















    

    <meta name="citation_doi" content="10.1111/j.1365-294X.2012.05622.x">

    <meta name="dc.identifier" content="10.1111/j.1365-294X.2012.05622.x">




<meta name="citation_journal_title" content="Molecular Ecology">
<meta name="citation_title" content="Trees and/or networks to display intraspecific DNA sequence variation?">
<meta name="citation_keywords" content="haplotype networks">
<meta name="citation_keywords" content="intraspecific DNA sequences">
<meta name="citation_keywords" content="parsimony">
<meta name="citation_keywords" content="phylogenetic trees">
<meta name="citation_keywords" content="phylogeography">
<meta name="citation_volume" content="21">
<meta name="citation_issue" content="14">
<meta name="citation_firstpage" content="3385">
<meta name="citation_lastpage" content="3390">
<meta name="citation_publication_date" content="2012/07/01">
<meta name="citation_online_date" content="2012/05/18">
<meta name="citation_issn" content="1365-294X">
<meta name="citation_language" content="en">
<meta name="citation_author" content="PATRICK MARDULYN">
<meta name="citation_author_institution" content="Evolutionary Biology and Ecology, Université Libre de Bruxelles, CP 160/12, av FD Roosevelt 50, 1050 Brussels, Belgium; Fonds de la Recherche Scientifique – FNRS, Brussels, Belgium">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-294X.2012.05622.x">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1111/j.1365-294X.2012.05622.x">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-294X.2012.05622.x">
<meta name="citation_publisher" content="John Wiley &amp; Sons, Ltd (10.1111)"><meta name="citation_fulltext_world_readable" content="">
<meta name="article_references" content="MARDULYN, P.  (2012), Trees and/or networks to display intraspecific DNA sequence variation?. Molecular Ecology, 21: 3385-3390. doi:10.1111/j.1365-294X.2012.05622.x">
<meta name="epdf_available" content="true">
<meta name="Description" content="Abstract Phylogenetic trees and networks are both used in the scientific literature to display DNA sequence variation at the intraspecific level. Should we rather use trees or networks? I argue tha...">


<title>Trees and/or networks to display intraspecific DNA sequence 
variation? - MARDULYN - 2012 - Molecular Ecology - Wiley Online Library</title>






<link rel="stylesheet" type="text/css" href="hoqiproduct.css"><link rel="stylesheet" type="text/css" href="build.css">
<link rel="stylesheet" type="text/css" href="print.css" media="print">




<script async="" src="catchpoint-1549680351760.js"></script>
<meta name="referrer" content="origin-when-cross-origin">


<link rel="stylesheet" type="text/css" href="hub-1534777306460.css">


<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1552321492860.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* PERICLES-7186: distorted cover images */
.journal-info-container .cover-image__image img {
    min-height: unset;
}

/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents,
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}

/* TEMPORARY Fix heading on Topical Index widget for MRWs */
.topical-index h2 {
    margin: 15px 0;
    font-size: 1.75rem;
    line-height: 28px;
    color: #1c1d1e;
    font-family: Source Serif Pro,serif;
    font-weight: normal;
}

/* PERICLES-7288 */
.issue-supplementary-files, .cover-image__virtual-issue {
    margin-bottom: 15px;
}

/* PERICLES-8875 */
.journal-news .slide-item img {
    float: left;
    margin-right: 10px;
}

/* DO NOT REMOVE */
li[heading="Restore Points"].ng-isolate-scope,
li[heading="Disable Widgets"].ng-isolate-scope {
    display: none!important;
}

.modal-header .widgetOrigin {
    display: none;
}

/* PERICLES-10090 */
.cover-image__image img {
    max-width: 150px;
    max-height: 150px;
}

/* LIT-282003 */
.journal-info-container span.info_value:after {
    content: '\003B';
}

.journal-info-container span.info_value:last-child:after {
    content: '';
}

/* LIT-299510 */
h1.citation__title {
    font-size: 22px;
    font-size: 1.375rem;
}

@media (max-width: 991px) {
    h1.citation__title {
        font-size: 18px;
        font-size: 1.125rem;
    }
    h1.citation__title:after {
        content: "";
    }
}

/* LIT-295816 */
.search__item .item__body .meta__details a.publication_meta_serial:after {
    content: "";
    margin: 0;
}
.search__item .item__body .meta__details a.publication_meta_volume_issue:before {
    content: "|";
    margin-left: 4px;
    margin-right: 4px;
}
</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5c87b4fe64746d78ad006ae3.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script><script src="satellite-5aaec36164746d79c1001d14.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>

<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	if (!$DoubleClickSite) {
		switch("pericles") {
			case "afspubs":
				var $DoubleClickSite = "wly.aqua.fish_000017";
				break;
			case "aasldpubs":
  				var $DoubleClickSite = "wly.gastro.hep_000059";
  				break;
			case "headachejournal":
  				var $DoubleClickSite = "wly.neurology_000086";
  				break;
			default:
				var $DoubleClickSite = "undefined-site";
		};
  	};

	if (!$DoubleClickZone) {
		switch("pericles") {
			case "afspubs":
				var $DoubleClickZone = "AFS_HUB";
				break;
      			case "aasldpubs":
        			var $DoubleClickZone = "AASLD-pubs_HUB";
        			break;
      			case "headachejournal":
        			var $DoubleClickZone = "headache_HUB";
        			break;
			default:
				var $DoubleClickZone = "undefined-zone";
		};
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};


		gptPage = window.digitalData && digitalData.page && digitalData.page.page ? digitalData.page.page : "blank";
		gptSubPage = window.digitalData && digitalData.page && digitalData.page.subPage ? digitalData.page.subPage : "blank";


		function gptClean(gptText){
			if(typeof(gptText)=="string"){
				gptText = gptText.replace(/\&/g,'%26');


				return gptText.replace(/\^|\*|#|\$|\"|\'|\=|\,|\!|\+|\~|\;|\(|\)|\<|\>|\[|\]|\?|\%|\:|\@|\;|\/|\&|\||\{|\}|\\|/g,'');

			}
			return "blank";
		}


		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'doi' , '10.1111/j.1365-294X.2012.05622.x') // Publication DOI
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		) 	// Krux
			.setTargeting ( 'pbPage', gptClean(gptPage) )
			.setTargeting ( 'pbSubPage', gptClean(gptSubPage) );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019101401.js" async=""></script>






    







    
        
    

















    <script src="casa.js" async=""></script>







    


<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "Trees and/or networks to display intraspecific DNA sequence variation? - MARDULYN - 2012 - Molecular Ecology - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker.js" async=""></script>
<style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script async="" type="text/javascript" src="epdf_linker_002.js"></script><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="satellite-5c58af5e64746d5111000925.js"></script><script src="satellite-5c87cfcf64746d7044002f7b.js"></script><script src="satellite-5cf7bd7564746d596c000241.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script src="js.js" async="true"></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><script src="satellite-5c87b56e64746d4e33003a7c.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style></head>
<body class="pb-ui">

<div class="skip-links">
    <ul class="rlist">
        
                <li><a href="#article__content" tabindex="1">Skip to Article Content</a></li>
                <li><a href="#pane-pcw-detailscon" tabindex="1">Skip to Article Information</a></li>
            
    </ul>
</div>







    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader fixed-element" style="position: fixed;">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        



        
    

</div></div></div>

        </div>
    




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg padded-content">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library" tabindex="2"><img id="mainLogo" alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img id="mobileLogo" alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm hidden-md institution--block">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access by</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" aria-label="Search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper"><div class="tab clearfix"><input placeholder="Search" tabindex="13" class="fakeQSInput quickSearchFilter" type="search"><label for="search-within" class="sr-only">Search within</label><select name="search-within" id="search-within" role="tablist" tabindex="10" class="search-within"><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" selected="selected">This Journal</option><option value="#pane-67065c09-4a88-49cd-934c-ac707951d35c1">Anywhere</option></select><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="1365294x" type="hidden"><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Search" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" tabindex="9" class="autocomplete actualQSInput quickSearchFilter ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper quickSearchFilter"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x" title="" tabindex="12" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced?publication=1365294x#citation" title="" tabindex="12" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" tabindex="11" aria-label="Submit Search" class="btn quick-search__button icon-search"></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    









    <div class="loginBar login-open">
        <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x&amp;aria-label=Log+in+or+Register" class="show-login" role="button" uri="/doi/full/10.1111/j.1365-294X.2012.05622.x" aria-label="Log in or Register" tabindex="30">
            <span class="icon-profile_empty"></span>
            <span class="sign-in-label">Login / Register</span>
        </a>
            
    </div>



        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    










        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet body-text" style="padding-top: 71px;">
        



        
        <main id="main-content" tabindex="-1" style="">
    <!--begin pagefulltext-->
    <div class="page-body pagefulltext">
        <div data-pb-dropzone="main">
        



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        



        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_0__container__" style="border: 0pt none; width: 100%; height: 0%;"></div>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x&pubId=58310032&placeholderId=1015&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		</div>
	</aside>
        </div>
    


        </div>
    




        
        
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;pbContext=%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2012.21.issue-14%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-294X.2012.05622.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;pbContext=%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2012.21.issue-14%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-294X.2012.05622.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;pbContext=%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2012.21.issue-14%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-294X.2012.05622.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;pbContext=%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2012.21.issue-14%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-294X.2012.05622.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;showSubjects=true&amp;taxonomiesCodes=enter+your+taxonomy+codes+here&amp;displayCitedByLink=true&amp;displayAlmetricDropzone=true&amp;taxonomyUri=publication-features&amp;topicUri=hide-issue-metadata" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div id="article__content" tabindex="-1" class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone">



        
        
</div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      <div id="journal-banner-text" class="journal-banner-text" style="font-size: 22px;"> <a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage">Molecular Ecology</a></div><a href="https://onlinelibrary.wiley.com/toc/1365294x/2012/21/14" class="volume-issue">Volume 21, Issue 14</a><a href="https://onlinelibrary.wiley.com/journal/1365294x" title="Molecular Ecology homepage" class="citation--logo"><img id="journal-banner-image" test="test-value" alt="Molecular Ecology banner" src="1365294x-1525974682817.jpg" class="journal-banner-image" style="display: none;"></a><script>var image = new Image();

image.onerror = function() {
  // if the image does not exist
  //document.getElementById("journal-banner-text").style.marginBottom = "15px";
  document.getElementById("journal-banner-text").style.fontSize = "22px";
  document.getElementById("journal-banner-image").style.display = "none";
}

image.src = "https://wol-prod-cdn.literatumonline.com/pb-assets/journal-banners/1365294x-1525974682817.jpg";</script>
   </div>
   <div class="doi-access-container clearfix"><span class="primary-heading"><span class="primary-heading">NEWS AND VIEWS<a id="link_artCateg-note" class="scrollableLink" aria-label="Scrollable Link" href="#artCateg-note"><sup>†</sup></a></span></span>
      <div class="doi-access-wrapper"><div class="free-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Free Access</div></div></div></div></div>
   <h1 class="citation__title">Trees and/or networks to display intraspecific DNA sequence variation?</h1>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=MARDULYN%2C+PATRICK" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl" role="button"><span>PATRICK MARDULYN</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" role="region" id="am1">
            <p>Evolutionary Biology and Ecology, Université Libre de 
Bruxelles, CP 160/12, av FD Roosevelt 50, 1050 Brussels, Belgium; Fonds 
de la Recherche Scientifique – FNRS, Brussels, Belgium</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=MARDULYN%2C+PATRICK">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=MARDULYN%2C+PATRICK" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl" role="button"><span>PATRICK MARDULYN</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" role="region" id="a1">
                  <p>Evolutionary Biology and Ecology, Université Libre 
de Bruxelles, CP 160/12, av FD Roosevelt 50, 1050 Brussels, Belgium; 
Fonds de la Recherche Scientifique – FNRS, Brussels, Belgium</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=MARDULYN%2C+PATRICK">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections">
      <div class="epub-section"><span class="epub-state">First published:&nbsp;</span><span class="epub-date">18 May 2012</span></div>
      <div class="epub-section"><a class="epub-doi" aria-label="Digital Object Identifier" href="https://doi.org/10.1111/j.1365-294X.2012.05622.x">https://doi.org/10.1111/j.1365-294X.2012.05622.x</a></div>
      <div class="epub-section cited-by-count"><span>Cited by: <a href="#citedby-section">13</a></span></div>
   </div>
   <div class="article-header__correspondence-to"><i aria-hidden="true" class="at-icon-wrapper icon-mail_outline"></i>Patrick Mardulyn, Fax: 32 2 6502445; E‐mail: <a title="
                        Link to email address
                    " href="mailto:pmarduly@ulb.ac.be">pmarduly@ulb.ac.be</a></div>
   <div class="epub-sections">
      <div class="article-header__references-container no-truncate" id="artCateg-note" tabindex="0">
         <div class="header-note-content" aria-labelledby="header-note">
            <p>OPINION</p>
         </div>
      </div>
   </div>
</div></div><nav class="coolBar stickybar"><div class="stickybar__wrapper coolBar__wrapper clearfix" style="width: 610px; top: 71px;"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><div class="transplanted-clone"><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1365-294X.2012.05622.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-294X.2012.05622.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></div></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d36500407"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss2-title"><span> Trees or networks?</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss3-title"><span> Tree graph or network graph?</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss4-title"><span> Phylogenetic estimation algorithm or network‐building algorithm?</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss5-title"><span> Conclusion</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#ss6-title"><span> Acknowledgements</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><ul data-cb-group="Article" data-cb-group-icon="icon-toc" class="rlist coolBar__first"></ul><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/j.1365-294X.2012.05622.x" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink scroll-to-target pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/j.1365-294X.2012.05622.x" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2Fj.1365-294X.2012.05622.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1111%2Fj.1365-294X.2012.05622.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1111%2Fj.1365-294X.2012.05622.x" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" async="async" src="addthis_widget.js"></script><div class="relative"><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1111/j.1365-294X.2012.05622.x/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i><span class="sr-only">Close modal</span></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="https://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="https://onlinelibrary.wiley.com/researchers/tools-resources/sharing" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email</a></li><li><a role="link" class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a role="link" class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav><div id="accessDenialslot" class="accessDenialslot"></div><div class="article__body "><article>
   
   
   <div class="abstract-group">
      <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
         <h2 id="d36500407" class="article-section__header section__title main abstractlang_en main">Abstract</h2>
         <div class="article-section__content en main">
            
            <p>Phylogenetic trees and networks are both used in the 
scientific literature to display DNA sequence variation at the 
intraspecific level. Should we rather use trees or networks? I argue 
that the process of inferring the most parsimonious genealogical 
relationships among a set of DNA sequences should be dissociated from 
the problem of displaying this information in a graph. A network graph 
is probably more appropriate than a strict consensus tree if many 
alternative, equally most parsimonious, genealogies are to be included. 
Within the maximum parsimony framework, current phylogenetic inference 
and network‐building algorithms are both unable to guarantee the finding
 of all most parsimonious (MP) connections. In fact, each approach can 
find MP connections that the other does not. Although it should be 
possible to improve at least the maximum parsimony approach, current 
implementations of these algorithms are such that it is advisable to use
 both approaches to increase the probability of finding all possible MP 
connections among a set of DNA sequences.</p>
            
         </div>
      </section>
   </div>
   <div class="pb-dropzone" data-pb-dropzone="below-abstract-group">
      <!-- Empty dropzone -->
   </div>
   
   <section class="article-section article-section__full">
      <div class="article-section__content" id="ss1">
         
         <p>Studies exploring DNA sequence variation at the 
intraspecific level commonly use two types of graph to summarize the 
genetic data: phylogenetic trees and/or haplotype networks. For example,
 a quick survey of phylogeographic studies published in the scientific 
journal <i>Molecular Ecology</i> in 2011 reveals that among 45 studies, 
38% displayed DNA sequence variation using phylogenetic trees, 22% using
 haplotype networks and 40% using both trees and networks. Phylogenetic 
trees are connected graphs with no cycle (<span title=""><a href="#b16" id="#b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Huson <i>et&nbsp;al.</i> 2011</a></span>)
 and are traditionally used to present estimates of phylogenetic 
relationships among species. Phylogenetic networks are connected graphs 
with cycles. Although different types of network exist, I restrict the 
discussion here to what is commonly called haplotype or allele networks,
 generated by, for example, a median‐joining (<span title=""><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Bandelt <i>et&nbsp;al.</i> 1999</a></span>) or statistical parsimony (<span title=""><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Clement <i>et&nbsp;al.</i> 2000</a></span>)
 analysis, in which nodes represent different allelic sequences, joined 
by edges (branches) whose length is defined and shows the number of 
nucleotides that differ between them (e.g. <span title=""><a href="#b16" id="#b16R" class="bibLink tab-link" data-tab="pane-pcw-references">Huson <i>et&nbsp;al.</i> 2011</a></span>). Split networks (<span title=""><a href="#b15" id="#b15R" class="bibLink tab-link" data-tab="pane-pcw-references">Huson &amp; Bryant 2006</a></span>),
 built by combining each split (i.e. a partition of the sequences in two
 subsets) identified by the data, from either distances (<span title=""><a href="#b1" id="#b1R" class="bibLink tab-link" data-tab="pane-pcw-references">Bandelt &amp; Dress 1992</a></span>; <span title=""><a href="#b5" id="#b5R" class="bibLink tab-link" data-tab="pane-pcw-references">Bryant &amp; Moulton 2004</a></span>) or trees (<span title=""><a href="#b13" id="#b13R" class="bibLink tab-link" data-tab="pane-pcw-references">Holland &amp; Moulton 2003</a></span>; <span title=""><a href="#b14" id="#b14R" class="bibLink tab-link" data-tab="pane-pcw-references">Holland <i>et&nbsp;al.</i> 2004</a></span>),
 also offer an interesting tool to explore sequence data and are 
increasingly used to display intraspecific sequence variation (e.g. <span title=""><a href="#b6" id="#b6R" class="bibLink tab-link" data-tab="pane-pcw-references">Cassens <i>et&nbsp;al.</i> 2003</a></span>; <span title=""><a href="#b18" id="#b18R" class="bibLink tab-link" data-tab="pane-pcw-references">Marshall <i>et&nbsp;al.</i> 2009</a></span>; <span title=""><a href="#b4" id="#b4R" class="bibLink tab-link" data-tab="pane-pcw-references">Barrett &amp; Freudenstein 2011</a></span>).
 Their main purpose is to visualize the ambiguous phylogenetic signal 
present in a data set. They will, however, not be discussed here further
 because, as explained below, and unlike haplotype networks, they are 
not appropriate to summarize the information contained in several most 
parsimonious phylograms, which is the focus of this article.
         </p>
         
      </div>
      <div class="article-section__content" id="ss2">
         
         <h2 class="article-section__title section__title section1" id="ss2-title"> Trees or networks?</h2>
         
         <p>Should we favour the use of trees and/or the use of 
networks? Inferring a genealogy from a set of intraspecific DNA 
sequences can be done using (i) phylogenetic tree methods, that is, 
classic methods of phylogeny inference that use an optimality criterion 
to compare trees (e.g. maximum parsimony and maximum likelihood) or (ii)
 network methods whose algorithms directly generate a network graph from
 the sequences (e.g. median‐joining or statistical parsimony). While a 
network graph is typically tied to, and sometimes defined by, the 
algorithm that generates it, I suggest that to compare the pros and cons
 of trees and networks, it is useful to differentiate the algorithm used
 to infer the genealogical relationships among a sample of DNA sequences
 (i.e. the algorithm that connects the sequences together), from the 
graph used to display this information. The processes of (i) identifying
 the evolutionary (e.g. most parsimonious) connections among the 
sequences and (ii) drawing the graph displaying this information can be 
treated in principle independently from each other. In fact, under this 
rationale, <span title=""><a href="#b7" id="#b7R" class="bibLink tab-link" data-tab="pane-pcw-references">Cassens <i>et&nbsp;al.</i> (2005)</a></span>
 have proposed a method to create a network graph from the set of all 
most parsimonious trees, thereby combining a classic phylogeny inference
 method with an algorithm that constructs a network graph. Also, other 
algorithms have been designed previously for the purpose of inferring 
network graphs that include all most parsimonious trees, directly from 
the sequence data: the median network algorithm of <span title=""><a href="#b2" id="#b2R" class="bibLink tab-link" data-tab="pane-pcw-references">Bandelt <i>et&nbsp;al.</i> (1995)</a></span>, restricted to the analysis of strictly binary data (two character states), and an algorithm developed by <span title=""><a href="#b10" id="#b10R" class="bibLink tab-link" data-tab="pane-pcw-references">Fitch (1997)</a></span>,
 which in practice is restricted to a limited number of sequences. If we
 do adopt a parsimony framework, which is reasonable when dealing with 
closely related sequences (e.g. <span title=""><a href="#b12" id="#b12R" class="bibLink tab-link" data-tab="pane-pcw-references">Holder &amp; Lewis 2003</a></span>; <span title=""><a href="#b9" id="#b9R" class="bibLink tab-link" data-tab="pane-pcw-references">Felsenstein 2004</a></span>),
 as intraspecific sequences often are, we can rephrase the problem as 
follows: Which algorithm is best to infer all MP connections among a 
group of sequences, and which kind of graph is best to display them?
         </p>
         
      </div>
      <div class="article-section__content" id="ss3">
         
         <h2 class="article-section__title section__title section1" id="ss3-title"> Tree graph or network graph?</h2>
         
         <p>While there are conceptual differences between a graph 
displaying genetic variation at the intraspecific level and a tree 
delivering an estimate of a species phylogeny (<span title=""><a href="#b20" id="#b20R" class="bibLink tab-link" data-tab="pane-pcw-references">Posada &amp; Crandall 2001</a></span>),
 a tree graph can technically be used to display intraspecific DNA 
sequence variation. In fact, in a parsimony framework, a clear 
correspondence can be established between a strictly bifurcating 
phylogram (i.e. a tree whose branches lengths are proportional to the 
number of mutations that have occurred along them, as opposed to a 
cladogram that does not include branch length information) and a 
haplotype network, as already discussed in <span title=""><a href="#b7" id="#b7R" class="bibLink tab-link" data-tab="pane-pcw-references">Cassens <i>et&nbsp;al.</i> (2005)</a></span> and shown in <a href="#f1" class="scrollableLink">Fig.&nbsp;1</a>.
 Indeed, both types of graph display the same information: the genetic 
distances separating alleles, the frequency of each allele (as displayed
 by the size of each circle in <a href="#f1" class="scrollableLink">Fig.&nbsp;1b</a> and as the number of sequences in <a href="#f1" class="scrollableLink">Fig.&nbsp;1c</a>)
 and even some information over their geographic distribution (using pie
 charts in networks or identifying the geographic location of each 
sequence in phylograms). Within a parsimony framework, a haplotype 
network without cycle is thus equivalent to a phylogram. Note that in 
this context, the length of a branch separating two alleles in both 
types of graph simply corresponds to the number of sites that are 
different between the two sequences, which is probably a good estimation
 of the number of mutations separating both sequences from their common 
ancestor, for closely related sequences.
         </p>
         
         <section class="article-section__inline-figure">
            <figure class="figure" id="f1"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/7e79c858-c691-49dd-94f7-dc3956eef9bf/mec_5622_f1.gif"><img class="figure__image" src="mec_5622_f1.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                  <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                     <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f1&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x" class="ppt-figure-link">PowerPoint</a></div>
                  </div>
                  <div>
                     <p> Relationship between a gene genealogy, an 
estimated network graph and an estimated phylogram. (a) Gene genealogy 
displaying genealogical relationships among 20 sampled copies of a gene 
within a species. Branch length is proportional to time. Horizontal bars
 identify mutations along the branches. (b), Allele network showing the 
history of mutations that have occurred among the different alleles of 
the sampled sequences. Each allele is represented by a circle: its size 
is proportional to its frequency and its colour provides information 
about its geographic distribution. Each edge corresponds to a single 
mutation, and small white circles correspond to inferred alleles, absent
 from the data set. (c) phylogram showing the history of mutations that 
have occurred among the different alleles of the sampled sequences. (b) 
and (c) are equivalent in terms of information content and can easily be
 drawn from (a). However, (a) is usually unknown, and (b) and (c) are 
estimated from a set of sampled sequences.</p>
                     
                  </div>
               </figcaption>
            </figure>
         </section>
         
         <p>Ambiguous connections appear under the parsimony criterion 
when two or more alternative connections share the same length, that is,
 in the presence of homoplasy in the sequence data. One can deal with 
ambiguity, as is usually done in phylogenetic studies, by considering 
all equally parsimonious trees separately, which rapidly becomes 
impractical with increasing numbers of trees, or by building a strict 
consensus cladogram compatible with all most parsimonious (MP) trees, 
that is collapsing clades, thereby creating a tree that includes 
multifurcations. But this has the undesirable effect of decreasing the 
amount of information contained in the graph (<a href="#f2" class="scrollableLink">Fig.&nbsp;2d</a>),
 because (i) the resulting strict consensus tree can become compatible 
with many more trees than the initial set of MP trees that was used to 
build it and (ii) ambiguous information over branch length cannot be 
displayed. Another way of dealing with ambiguities is to draw a 
haplotype network that includes cycles (also called loops; <a href="#f2" class="scrollableLink">Fig.&nbsp;2c</a>).
 The main feature distinguishing networks from trees is indeed the 
possibility of including cycles to represent (i) reticulate evolution 
(e.g. recombinations, hybridisations, lateral gene transfers) or (ii) 
conflicting signal in the data. <span title=""><a href="#b19" id="#b19R" class="bibLink tab-link" data-tab="pane-pcw-references">Morrison (2005)</a></span>
 makes a clear distinction between a ‘true phylogenetic network’ that 
aims to display true reticulate evolutionary events, and a 
‘character‐display network’, a network graph that merely displays all 
connections among the sequences that are equally well supported, 
revealing conflicting genealogical signal. While the ability to 
represent reticulate evolution is an essential feature of network 
graphs, the discussion here will be limited to the second type of 
network that summarizes all ambiguous connections in a single figure. A 
haplotype network with cycles can also be compatible with more trees 
than the initial set of MP trees, although the number of compatible 
trees is usually smaller than with a strict consensus tree. Moreover, 
this number can be further decreased by identifying branches (edges) 
that may be deleted in each cycle, and others that cannot, to recreate 
all the individual trees compatible with the network (<a href="#f2" class="scrollableLink">Fig.&nbsp;2c</a>).
 In a parsimony framework, branches that can be discarded are those 
whose deletion results in a minimum length graph. Also, a network graph 
with cycles continues to display branch length information.
         </p>
         
         <section class="article-section__inline-figure">
            <figure class="figure" id="f2"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/e6b38791-9d7e-4812-ac2a-f6d9bfffe7b4/mec_5622_f2.gif"><img class="figure__image" src="mec_5622_f2.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                  <div class="figure__caption__header"><strong class="figure-title">Figure 2</strong> 
                     <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f2&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x" class="ppt-figure-link">PowerPoint</a></div>
                  </div>
                  <div>
                     <p> Comparison of the informativeness of networks 
and strict consensus trees. (a) Polymorphic sites for a set of 14 
sequences. (b) One of several most parsimonious phylograms associated 
with this set of sequences. (c) Network graph including all most 
parsimonious trees associated with this set of sequences. Each sequence 
in the data set is represented by a labelled circle; each branch 
represents one mutation; unlabelled circles are sequences not present in
 the data set. Dashed branches are those that may be deleted to recreate
 one of the MP phylogram used to create the graph. (d) Strict consensus 
cladogram of all most parsimonious trees associated with this set of 
sequences, as generated by PAUP* (Branch and Bound search).</p>
                     
                  </div>
               </figcaption>
            </figure>
         </section>
         
         <p>It should be noted that a split network cannot be used for 
the same purpose of displaying several MP phylograms into a single 
figure. A split network inferred from a set of MP trees (i.e. a 
consensus network; <span title=""><a href="#b13" id="#b13R" class="bibLink tab-link" data-tab="pane-pcw-references">Holland &amp; Moulton 2003</a></span>)
 is built by combining all the splits defined by this set of trees and 
loses the information of branch lengths contained in the initial 
phylograms.
         </p>
         
         <p>In conclusion, when alternative connections are equally 
parsimonious, a haplotype network graph appears more appropriate to 
summarize intraspecific DNA sequence variation, because it conveys more 
information than a strict consensus cladogram.</p>
         
      </div>
      <div class="article-section__content" id="ss4">
         
         <h2 class="article-section__title section__title section1" id="ss4-title"> Phylogenetic estimation algorithm or network‐building algorithm?</h2>
         
         <p>This does not mean, however, that network‐building methods are automatically better. As argued by <span title=""><a href="#b7" id="#b7R" class="bibLink tab-link" data-tab="pane-pcw-references">Cassens <i>et&nbsp;al.</i> (2005)</a></span>,
 several methods of phylogenetic inference are based on the use of an 
optimality criterion to explore the space of all possible trees, which 
confers an advantage to them over network construction methods that are 
entirely defined by the algorithm used to construct the network step by 
step, and do not consider an a posteriori comparison of the resulting 
network to alternative networks based on some kind of criterion. 
Moreover, recent comparisons of the performances of traditional 
phylogenetic algorithms with those from network construction methods 
appear to be in favour of the former. <span title=""><a href="#b23" id="#b23R" class="bibLink tab-link" data-tab="pane-pcw-references">Woolley <i>et&nbsp;al.</i> (2008)</a></span>
 have tested the performances of these methods by analysing DNA sequence
 data simulated under various parameter values of a classic coalescent 
model depicting the evolution of a panmictic population. They concluded 
that although most methods perform equally well when the DNA sequence 
substitution rate is low, maximum parsimony is more accurate than 
network construction algorithms at a higher rate of substitution. <span title=""><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Salzburger <i>et&nbsp;al.</i> (2011)</a></span>
 have tested these performances under a wider range of population 
models, featuring coalescent simulations under both a single panmictic 
population and a structured population with symmetric or asymmetric 
migration. They concluded that (i) all traditional phylogenetic methods 
outperform the statistical parsimony network construction algorithm 
implemented in the software TCS (<span title=""><a href="#b8" id="#b8R" class="bibLink tab-link" data-tab="pane-pcw-references">Clement <i>et&nbsp;al.</i> 2000</a></span>), and (ii) in general, maximum parsimony analyses performed slightly better than maximum likelihood.
         </p>
         
         <p>In the light of all of the above, it would seem logical to 
advise the use of the traditional maximum parsimony method to analyse 
intraspecific DNA sequences, and to combine all resulting MP trees in a 
network graph, using, for example, the algorithm UMP (Union of Maximum 
Parsimonious trees, <span title=""><a href="#b7" id="#b7R" class="bibLink tab-link" data-tab="pane-pcw-references">Cassens <i>et&nbsp;al.</i> 2005</a></span>).
 However, it is relatively easy to identify a set of sequences for which
 a median‐joining analysis will perform better than maximum parsimony. 
For example, <span title=""><a href="#b17" id="#b17R" class="bibLink tab-link" data-tab="pane-pcw-references">Mardulyn <i>et&nbsp;al.</i> (2009)</a></span>
 have described a simple example data set (only four sequences of three 
nucleotides long: AAA, ACA, GAT, GCT) for which the median‐joining or 
TCS algorithms performed better than maximum parsimony: a maximum 
parsimony PAUP* (<span title=""><a href="#b22" id="#b22R" class="bibLink tab-link" data-tab="pane-pcw-references">Swofford 2003</a></span>)
 search resulted in a single MP phylogram, while the median‐joining or 
TCS graph contained two MP phylograms, including the one found by the 
PAUP* analysis. The reason for the better performance of the 
network‐building algorithms in this example lies with the way PAUP* 
deals with most parsimonious ancestral states. Like most phylogeny 
estimation programs, when running a maximum parsimony search, it 
compares cladograms (not phylograms) using the parsimony criterion. This
 is fine if one is interested in estimating evolutionary relationships 
among well‐differentiated species, but a phylogram is more informative 
when considering intraspecific DNA sequence variation (<a href="#f3" class="scrollableLink">Fig.&nbsp;3</a> shows the compatibility relationships existing between some phylograms and cladograms). As already noted by <span title=""><a href="#b21" id="#b21R" class="bibLink tab-link" data-tab="pane-pcw-references">Salzburger <i>et&nbsp;al.</i> (2011)</a></span>,
 more than one phylogram (called a Fitch tree in their article) 
corresponds to a single MP cladogram, because several equally 
parsimonious reconstructions of ancestral sequences are usually 
possible. If the inference program is subsequently asked to estimate 
branch lengths for the inferred set of MP cladograms, to generate a set 
of MP phylograms, it will assign only one of several possible most 
parsimonious ancestral sequences for each interior node of a tree. 
Because all alternative reconstructions of ancestral states are not 
taken into account, some MP phylograms may not be considered. The 
generated set of MP phylograms will then sometimes represent a portion 
only of the complete set of all MP phylograms corresponding to the 
inferred set of MP cladograms.
         </p>
         
         <section class="article-section__inline-figure">
            <figure class="figure" id="f3"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/1efa84c9-9e05-4d0c-8e47-a60d97ac1a20/mec_5622_f3.gif"><img class="figure__image" src="mec_5622_f3.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                  <div class="figure__caption__header"><strong class="figure-title">Figure 3</strong> 
                     <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f3&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x" class="ppt-figure-link">PowerPoint</a></div>
                  </div>
                  <div>
                     <p> Arrows indicate compatibility between three 
example MP cladograms and four example MP phylograms. A cladogram can be
 translated into a phylogram by assigning one possible MP ancestral 
sequence to each interior node of the tree. For one cladogram, 
alternative MP ancestral sequences may generate different phylograms. 
Conversely, some phylograms are compatible with more than one cladogram.
 A cladogram is considered compatible with a phylogram if it can be 
transformed into this phylogram only by modifying its branch lengths, 
including assigning a length of zero to some branches.</p>
                     
                  </div>
               </figcaption>
            </figure>
         </section>
         
         <p>As a result, a standard MP phylogeny inference program 
cannot guarantee the finding of all MP solutions, even when implementing
 a Branch and Bound search (i.e. a search that guarantees the finding of
 all MP cladograms; <span title=""><a href="#b11" id="#b11R" class="bibLink tab-link" data-tab="pane-pcw-references">Hendy &amp; Penny 1982</a></span>). The median‐joining algorithm cannot guarantee that either (<span title=""><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Bandelt <i>et&nbsp;al.</i> 1999</a></span>).
 Although it applies the parsimony criterion locally, adding median 
vectors to an initial minimum spanning network to reduce the overall 
length of the graph, it does not explore the entire space of possible 
solutions. In fact, <a href="#f4" class="scrollableLink">Fig.&nbsp;4</a>
 shows a case where the analysis of 15 sequences by both the 
median‐joining and maximum parsimony methods result in two different 
sets of most parsimonious solutions, each set containing unique 
solutions that the other does not. Therefore, neither method appears 
ideal at this time, and it might be advisable to use both methods to 
maximize the probability of including all MP paths in the final network.
 Combining the results of both approaches appears trivial: it is 
sufficient to add the connections present in the network graph obtained 
with one approach to the network graph generated by the second approach,
 if absent.
         </p>
         
         <section class="article-section__inline-figure">
            <figure class="figure" id="f4"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/a76f38af-b517-406f-9580-6dc0f9589a7f/mec_5622_f4.gif"><img class="figure__image" src="mec_5622_f4.gif" alt="image" title="image"></a><figcaption class="figure__caption">
                  <div class="figure__caption__header"><strong class="figure-title">Figure 4</strong> 
                     <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=f4&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x" class="ppt-figure-link">PowerPoint</a></div>
                  </div>
                  <div>
                     <p> Maximum parsimony and median‐joining algorithms
 can both lead to different and unique most parsimonious solutions. (a) 
Polymorphic sites for a set of 15 DNA sequences. (b) A median‐joining 
network inferred from the set of sequences in A using the program 
network (<span class="figureLink bibLink tab-link" title=""><a href="#b3" id="#b3R" class="bibLink tab-link" data-tab="pane-pcw-references">Bandelt <i>et&nbsp;al.</i> 1999</a></span>; available at <a href="http://www.fluxus-engineering.com/sharenet.htm" class="linkBehavior">http://www.fluxus‐engineering.com/sharenet.htm</a>).
 (c) The two most parsimonious trees inferred from a Branch and Bound 
maximum parsimony search performed on the set of sequences in (a) using 
PAUP*. (d) Graph including all most parsimonious trees found in (b) and 
(c). Branches found in both (b) and (c) are in black. Branches unique to
 (b) are in grey. Dashed‐lines branches are unique to (c).</p>
                     
                  </div>
               </figcaption>
            </figure>
         </section>
         
      </div>
      <div class="article-section__content" id="ss5">
         
         <h2 class="article-section__title section__title section1" id="ss5-title"> Conclusion</h2>
         
         <p>In summary, while haplotype/allele networks (defined here as
 a special case of phylogenetic network, see above) are often more 
informative than phylogenetic strict consensus trees to display 
intraspecific DNA sequence variation, both maximum parsimony and 
network‐building algorithms are not guaranteed to find all most 
parsimonious phylograms for a set of sequences and can lead to 
alternative MP solutions. As already suggested in <span title=""><a href="#b17" id="#b17R" class="bibLink tab-link" data-tab="pane-pcw-references">Mardulyn <i>et&nbsp;al.</i> (2009)</a></span>,
 considering all MP ancestral sequences of a set of MP cladograms should
 guarantee the inference of all most parsimonious phylograms, at least 
when using an algorithm exploring the space of possible trees that 
guarantees the finding of all MP cladograms. Developing a program 
capable of generating the set of all MP phylograms corresponding to a 
set of MP cladograms would therefore considerably improve the maximum 
parsimony approach and might make it superior compared with 
network‐building algorithms, at least if we are willing to evaluate the 
performance of these algorithms, as I have assumed here, by measuring 
their ability to find all MP phylograms from a set of sequences. While 
the use of a criterion‐based method appears desirable, the tree space to
 explore in the case of large data sets may become easily too large for 
the use of a Branch and Bound algorithm, forcing the user to turn to 
heuristic strategies. These still need to be compared with 
network‐building algorithms such as the median‐joining method, before 
deciding whether one of these approaches performs better than the other 
in practice.
         </p>
         
      </div>
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="ss6-title"> Acknowledgements</h2>
         
         <p>The comments of two anonymous reviewers have significantly 
improved the clarity of the ideas presented here. An earlier version of 
the manuscript has also benefited from the careful reading of Simon 
Dellicour, Ceridwen Fraser, François Mayer and Maud Quinzin. This work 
was supported by grants from the Belgian Fonds de la Recherche 
Scientifique—FNRS.</p>
         
         <ol></ol>
      </div>
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion">
            <h2>
               <div tabindex="0" role="button" class="accordion__control" aria-controls="idss7" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;pbContext=%3BrequestedJournal%3Ajournal%3A1365294x%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwebsite%3Awebsite%3Apericles%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fmec.2012.21.issue-14%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fj.1365-294X.2012.05622.x%3Bjournal%3Ajournal%3A1365294x%3Bwgroup%3Astring%3APublication+Websites%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;accordionHeadingWrapper=h2&amp;onlyLog=true"><span class="section__title" id="references-section-1">References</span><i aria-hidden="true" class="icon-section_arrow_d"></i></div>
            </h2>
            <div class="accordion__content" style="display: none;" role="
                            region
                        " aria-labelledby="
                            references-section-1" id="ss7">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="b1">
                     <span class="author">Bandelt HJ</span>, 
<span class="author">Dress AW</span> (<span class="pubYear">1992</span>)  <span class="articleTitle">Split decomposition: a new and useful approach to phylogenetic analysis of distance data</span>. 
<i>Molecular phylogenetics and evolution</i>,  <span class="vol">1</span>,  <span class="pageFirst">242</span>– <span class="pageLast">252</span>.

                     <div class="extra-links" id="e_1_2_7_2_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1016%2F1055-7903%2892%2990021-8" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1342941" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000207481100009" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=1&amp;publication_year=1992&amp;pages=242-252&amp;journal=Molecular+phylogenetics+and+evolution&amp;author=HJ+Bandelt&amp;author=AW+Dress&amp;title=Split+decomposition%3A+a+new+and+useful+approach+to+phylogenetic+analysis+of+distance+data" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b2">
                     <span class="author">Bandelt HJ</span>, 
<span class="author">Forster P</span>, 
<span class="author">Sykes BC</span>, 
<span class="author">Richards MB</span> (<span class="pubYear">1995</span>)  <span class="articleTitle">Mitochondrial portraits of human populations using median networks</span>. 
<i>Genetics</i>,  <span class="vol">141</span>,  <span class="pageFirst">743</span>– <span class="pageLast">753</span>.

                     <div class="extra-links" id="e_1_2_7_3_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ASTN%3A280%3ADyaK287gvFKitQ%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=8647407" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=A1995RX01900026" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=141&amp;publication_year=1995&amp;pages=743-753&amp;journal=Genetics&amp;author=HJ+Bandelt&amp;author=P+Forster&amp;author=BC+Sykes&amp;author=MB+Richards&amp;title=Mitochondrial+portraits+of+human+populations+using+median+networks" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b3">
                     <span class="author">Bandelt HJ</span>, 
<span class="author">Forster P</span>, 
<span class="author">Röhl A</span> (<span class="pubYear">1999</span>)  <span class="articleTitle">Median‐joining networks for inferring intraspecific phylogenies</span>. 
<i>MMolecular biology and evolution</i>,  <span class="vol">16</span>,  <span class="pageFirst">37</span>– <span class="pageLast">48</span>.

                     <div class="extra-links" id="e_1_2_7_4_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1093%2Foxfordjournals.molbev.a026036" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADyaK1MXjvVGltA%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10331250" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000077903600005" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=1999&amp;pages=37-48&amp;journal=MMolecular+biology+and+evolution&amp;author=HJ+Bandelt&amp;author=P+Forster&amp;author=A+R%C3%B6hl&amp;title=Median%E2%80%90joining+networks+for+inferring+intraspecific+phylogenies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b4">
                     <span class="author">Barrett CF</span>, 
<span class="author">Freudenstein JV</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">An integrative approach to delimiting species in a rare but widespread mycoheterotrophic orchid</span>. 
<i>Molecular ecology</i>,  <span class="vol">20</span>,  <span class="pageFirst">2771</span>– <span class="pageLast">2786</span>.

                     <div class="extra-links" id="e_1_2_7_5_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05124.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=21569137" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000292200700010" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2011&amp;pages=2771-2786&amp;journal=Molecular+ecology&amp;author=CF+Barrett&amp;author=JV+Freudenstein&amp;title=An+integrative+approach+to+delimiting+species+in+a+rare+but+widespread+mycoheterotrophic+orchid" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b5">
                     <span class="author">Bryant D</span>, 
<span class="author">Moulton V</span> (<span class="pubYear">2004</span>)  <span class="articleTitle">Neighbor‐net: an agglomerative method for the construction of phylogenetic networks</span>. 
<i>Molecular biology and evolution</i>,  <span class="vol">21</span>,  <span class="pageFirst">255</span>– <span class="pageLast">265</span>.

                     <div class="extra-links" id="e_1_2_7_6_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1093%2Fmolbev%2Fmsh018" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXisFWmurs%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=14660700" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000220083300008" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2004&amp;pages=255-265&amp;journal=Molecular+biology+and+evolution&amp;author=D+Bryant&amp;author=V+Moulton&amp;title=Neighbor%E2%80%90net%3A+an+agglomerative+method+for+the+construction+of+phylogenetic+networks" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b6">
                     <span class="author">Cassens I</span>, 
<span class="author">Van Waerebeek K</span>, 
<span class="author">Best PB</span>
                     						<i>
              et&nbsp;al.
            </i> (<span class="pubYear">2003</span>)  <span class="articleTitle">The phylogeography of dusky dolphins (Lagenorhynchus obscurus): a critical examination of network methods and rooting procedures</span>. 
<i>Molecular ecology</i>,  <span class="vol">12</span>,  <span class="pageFirst">1781</span>– <span class="pageLast">1792</span>.

                     <div class="extra-links" id="e_1_2_7_7_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294X.2003.01876.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ASTN%3A280%3ADC%252BD3szjsF2jug%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=12803631" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000183514900008" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=12&amp;publication_year=2003&amp;pages=1781-1792&amp;journal=Molecular+ecology&amp;author=I+Cassens&amp;author=Waerebeek+K+Van&amp;author=PB+Best&amp;title=The+phylogeography+of+dusky+dolphins+%28Lagenorhynchus+obscurus%29%3A+a+critical+examination+of+network+methods+and+rooting+procedures" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b7">
                     <span class="author">Cassens I</span>, 
<span class="author">Mardulyn P</span>, 
<span class="author">Milinkovitch MC</span> (<span class="pubYear">2005</span>)  <span class="articleTitle">Evaluating
 intraspecific “network” construction methods using simulated sequence 
data: do existing algorithms outperform the global maximum parsimony 
approach?</span> 
                     <i>Systematic biology</i>,  <span class="vol">54</span>,  <span class="pageFirst">363</span>– <span class="pageLast">372</span>.

                     <div class="extra-links" id="e_1_2_7_8_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1080%2F10635150590945377" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=16012104" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000230302500002" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=54&amp;publication_year=2005&amp;pages=363-372&amp;journal=Systematic+biology&amp;author=I+Cassens&amp;author=P+Mardulyn&amp;author=MC+Milinkovitch&amp;title=Evaluating+intraspecific+%E2%80%9Cnetwork%E2%80%9D+construction+methods+using+simulated+sequence+data%3A+do+existing+algorithms+outperform+the+global+maximum+parsimony+approach%3F" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b8">
                     <span class="author">Clement M</span>, 
<span class="author">Posada D</span>, 
<span class="author">Crandall KA</span> (<span class="pubYear">2000</span>)  <span class="articleTitle">TCS: a computer program to estimate gene genealogies</span>. 
<i>Molecular ecology</i>,  <span class="vol">9</span>,  <span class="pageFirst">1657</span>– <span class="pageLast">1659</span>.

                     <div class="extra-links" id="e_1_2_7_9_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.2000.01020.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD3cXnvV2gtbw%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=11050560" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000089998600020" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=9&amp;publication_year=2000&amp;pages=1657-1659&amp;journal=Molecular+ecology&amp;author=M+Clement&amp;author=D+Posada&amp;author=KA+Crandall&amp;title=TCS%3A+a+computer+program+to+estimate+gene+genealogies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b9">
                     <span class="author">Felsenstein J</span> (<span class="pubYear">2004</span>)  <span class="bookTitle">Inferring Phylogenies</span>.  Sinauer Associates Inc., Sunderland, Massachusetts.

                     <div class="extra-links" id="e_1_2_7_10_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2004&amp;author=J+Felsenstein&amp;title=Inferring+Phylogenies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b10">
                     <span class="author">Fitch WM</span> (<span class="pubYear">1997</span>)  <span class="articleTitle">Networks and viral evolution</span>. 
<i>Journal of molecular evolution</i>,  <span class="vol">44</span>,  <span class="pageFirst">S65</span>– <span class="pageLast">S75</span>.

                     <div class="extra-links" id="e_1_2_7_11_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1007%2FPL00000059" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADyaK2sXhsFGrtb4%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=9071014" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=A1997WL08100011" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=44&amp;publication_year=1997&amp;pages=S65-S75&amp;journal=Journal+of+molecular+evolution&amp;author=WM+Fitch&amp;title=Networks+and+viral+evolution" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b11">
                     <span class="author">Hendy MD</span>, 
<span class="author">Penny D</span> (<span class="pubYear">1982</span>)  <span class="articleTitle">Branch and bound algorithms to determine minimal evolutionary trees</span>. 
<i>Mathematical biosciences</i>,  <span class="vol">59</span>,  <span class="pageFirst">277</span>– <span class="pageLast">290</span>.

                     <div class="extra-links" id="e_1_2_7_12_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1016%2F0025-5564%2882%2990027-X" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=A1982PF22200007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=59&amp;publication_year=1982&amp;pages=277-290&amp;journal=Mathematical+biosciences&amp;author=MD+Hendy&amp;author=D+Penny&amp;title=Branch+and+bound+algorithms+to+determine+minimal+evolutionary+trees" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b12">
                     <span class="author">Holder M</span>, 
<span class="author">Lewis PO</span> (<span class="pubYear">2003</span>)  <span class="articleTitle">Phylogeny estimation: traditional and Bayesian approaches</span>. 
<i>Nature reviews. Genetics</i>,  <span class="vol">4</span>,  <span class="pageFirst">275</span>– <span class="pageLast">284</span>.

                     <div class="extra-links" id="e_1_2_7_13_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1038%2Fnrg1044" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD3sXisFWqsLs%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=12671658" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000181931400013" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=4&amp;publication_year=2003&amp;pages=275-284&amp;journal=Nature+reviews.+Genetics&amp;author=M+Holder&amp;author=PO+Lewis&amp;title=Phylogeny+estimation%3A+traditional+and+Bayesian+approaches" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b13">
                     <span class="author">Holland B</span>, 
<span class="author">Moulton V</span> (<span class="pubYear">2003</span>)  <span class="chapterTitle">Consensus networks: a method for visualizing incompatibilities in collections of trees</span>. In:  <span class="bookTitle">Algorithms in bioinformatics, WABI 2003</span> (eds
					 <span class="editor">G Benson</span> and 
 <span class="editor">R Page</span>), pp.  <span class="pageFirst">165</span>– <span class="pageLast">176</span>.  Springer‐Verlag, Berlin, Germany.

                     <div class="extra-links" id="e_1_2_7_14_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1007%2F978-3-540-39763-2_13" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2003&amp;pages=165-176&amp;author=B+Holland&amp;author=V+Moulton&amp;title=Algorithms+in+bioinformatics%2C+WABI+2003" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b14">
                     <span class="author">Holland BR</span>, 
<span class="author">Huber KT</span>, 
<span class="author">Moulton V</span>, 
<span class="author">Lockhart PJ</span> (<span class="pubYear">2004</span>)  <span class="articleTitle">Using consensus networks to visualize contradictory evidence for species phylogeny</span>. 
<i>Molecular biology and evolution</i>,  <span class="vol">21</span>,  <span class="pageFirst">1459</span>– <span class="pageLast">1461</span>.

                     <div class="extra-links" id="e_1_2_7_15_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1093%2Fmolbev%2Fmsh145" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD2cXltlKlsb0%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=15084681" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000222334300032" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=21&amp;publication_year=2004&amp;pages=1459-1461&amp;journal=Molecular+biology+and+evolution&amp;author=BR+Holland&amp;author=KT+Huber&amp;author=V+Moulton&amp;author=PJ+Lockhart&amp;title=Using+consensus+networks+to+visualize+contradictory+evidence+for+species+phylogeny" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b15">
                     <span class="author">Huson DH</span>, 
<span class="author">Bryant D</span> (<span class="pubYear">2006</span>)  <span class="articleTitle">Application of phylogenetic networks in evolutionary studies</span>. 
<i>Molecular biology and evolution</i>,  <span class="vol">23</span>,  <span class="pageFirst">254</span>– <span class="pageLast">267</span>.

                     <div class="extra-links" id="e_1_2_7_16_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1093%2Fmolbev%2Fmsj030" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD28XntValsw%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=16221896" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000234718800004" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=23&amp;publication_year=2006&amp;pages=254-267&amp;journal=Molecular+biology+and+evolution&amp;author=DH+Huson&amp;author=D+Bryant&amp;title=Application+of+phylogenetic+networks+in+evolutionary+studies" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b16">
                     <span class="author">Huson DH</span>, 
<span class="author">Rupp R</span>, 
<span class="author">Scornavacca C</span> (<span class="pubYear">2011</span>)  <span class="bookTitle">Phylogenetic networks: concepts, algorithms and applications</span>.  Cambridge University Press, Cambridge, UK.

                     <div class="extra-links" id="e_1_2_7_17_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2011&amp;author=DH+Huson&amp;author=R+Rupp&amp;author=C+Scornavacca&amp;title=Phylogenetic+networks%3A+concepts%2C+algorithms+and+applications" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b17">
                     <span class="author">Mardulyn P</span>, 
<span class="author">Cassens I</span>, 
<span class="author">Milinkovitch MC</span> (<span class="pubYear">2009</span>)  <span class="chapterTitle">A comparison of methods for constructing evolutionary networks from intraspecific DNA sequences</span>. In:  <span class="bookTitle">Population Genetics for Animal Conservation</span> (eds
					 <span class="editor">G Bertorelle</span>, 
 <span class="editor">MD Bruford</span>, 
 <span class="editor">HC Hauffe</span>, 
 <span class="editor">A Rizzoli</span> and 
 <span class="editor">C Vernesi</span>), pp.  <span class="pageFirst">102</span>– <span class="pageLast">118</span>.  Cambridge University Press, Cambridge, UK.

                     <div class="extra-links" id="e_1_2_7_18_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2009&amp;pages=102-118&amp;author=P+Mardulyn&amp;author=I+Cassens&amp;author=MC+Milinkovitch&amp;title=Population+Genetics+for+Animal+Conservation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b18">
                     <span class="author">Marshall DC</span>, 
<span class="author">Hill KBR</span>, 
<span class="author">Fontaine KM</span>, 
<span class="author">Buckley TR</span>, 
<span class="author">Simon C</span> (<span class="pubYear">2009</span>)  <span class="articleTitle">Glacial refugia in a maritime temperate climate: cicada (Kikihia subalpina) mtDNA phylogeography in New Zealand</span>. 
<i>Molecular ecology</i>,  <span class="vol">18</span>,  <span class="pageFirst">1995</span>– <span class="pageLast">2009</span>.

                     <div class="extra-links" id="e_1_2_7_19_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2009.04155.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmsVektLg%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=19434813" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000265189400016" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=18&amp;publication_year=2009&amp;pages=1995-2009&amp;journal=Molecular+ecology&amp;author=DC+Marshall&amp;author=KBR+Hill&amp;author=KM+Fontaine&amp;author=TR+Buckley&amp;author=C+Simon&amp;title=Glacial+refugia+in+a+maritime+temperate+climate%3A+cicada+%28Kikihia+subalpina%29+mtDNA+phylogeography+in+New+Zealand" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b19">
                     <span class="author">Morrison DA</span> (<span class="pubYear">2005</span>)  <span class="articleTitle">Networks in phylogenetic analysis: new tools for population biology</span>. 
<i>International journal for parasitology</i>,  <span class="vol">35</span>,  <span class="pageFirst">567</span>– <span class="pageLast">582</span>.

                     <div class="extra-links" id="e_1_2_7_20_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1016%2Fj.ijpara.2005.02.007" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=15826648" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000228765900010" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=35&amp;publication_year=2005&amp;pages=567-582&amp;journal=International+journal+for+parasitology&amp;author=DA+Morrison&amp;title=Networks+in+phylogenetic+analysis%3A+new+tools+for+population+biology" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b20">
                     <span class="author">Posada D</span>, 
<span class="author">Crandall KA</span> (<span class="pubYear">2001</span>)  <span class="articleTitle">Intraspecific gene genealogies: trees grafting into networks</span>. 
<i>Trends in ecology &amp; evolution</i>,  <span class="vol">16</span>,  <span class="pageFirst">37</span>– <span class="pageLast">45</span>.

                     <div class="extra-links" id="e_1_2_7_21_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.0014-3820.2004.tb01678.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ASTN%3A280%3ADC%252BC2sbjvFOgtw%253D%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=11146143" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000168453700022" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=16&amp;publication_year=2001&amp;pages=37-45&amp;journal=Trends+in+ecology+%26+evolution&amp;author=D+Posada&amp;author=KA+Crandall&amp;title=Intraspecific+gene+genealogies%3A+trees+grafting+into+networks" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b21">
                     <span class="author">Salzburger W</span>, 
<span class="author">Ewing GB</span>, 
<span class="author">Von Haeseler A</span> (<span class="pubYear">2011</span>)  <span class="articleTitle">The performance of phylogenetic algorithms in estimating haplotype genealogies with migration</span>. 
<i>Molecular ecology</i>,  <span class="vol">20</span>,  <span class="pageFirst">1952</span>– <span class="pageLast">1963</span>.

                     <div class="extra-links" id="e_1_2_7_22_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05066.x" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=21457168" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000289522400015" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=20&amp;publication_year=2011&amp;pages=1952-1963&amp;journal=Molecular+ecology&amp;author=W+Salzburger&amp;author=GB+Ewing&amp;author=A+Von+Haeseler&amp;title=The+performance+of+phylogenetic+algorithms+in+estimating+haplotype+genealogies+with+migration" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b22">
                     <span class="author">Swofford DL</span> (<span class="pubYear">2003</span>)  <span class="bookTitle">PAUP*, phylogenetic analysis using parsimony (*and other methods), v. 4b10</span>.  Sinauer Associates,  <span class="otherTitle">Sunderland, Massachusetts</span>.

                     <div class="extra-links" id="e_1_2_7_23_1"><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;publication_year=2003&amp;author=DL+Swofford&amp;title=PAUP*%2C+phylogenetic+analysis+using+parsimony+%28*and+other+methods%29%2C+v.+4b10" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="b23">
                     <span class="author">Woolley SM</span>, 
<span class="author">Posada D</span>, 
<span class="author">Crandall KA</span> (<span class="pubYear">2008</span>)  <span class="articleTitle">A comparison of phylogenetic network methods using computer simulation</span>. 
<i>PLoS One</i>,  <span class="vol">3</span>,  <span class="pageFirst">e1913</span>.

                     <div class="extra-links" id="e_1_2_7_24_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=10.1371%2Fjournal.pone.0001913" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltleiu7k%253D" onclick="newWindow(this.href);return false" target="_blank" class="visitable">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=18398452" onclick="newWindow(this.href);return false" target="_blank" class="visitable">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fj.1365-294X.2012.05622.x&amp;key=000260795500003" onclick="newWindow(this.href);return false" target="_blank" class="visitable">Web of Science®</a></span><span><a class="google-scholar visitable" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=3&amp;publication_year=2008&amp;pages=e1913&amp;journal=PLoS+One&amp;author=SM+Woolley&amp;author=D+Posada&amp;author=KA+Crandall&amp;title=A+comparison+of+phylogenetic+network+methods+using+computer+simulation" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
      <div class="article-section__content" id="ss8">
         
         <p>The author has a general interest in the analysis of DNA 
sequence variation for studying evolution. Its current research focuses 
mainly on comparing multilocus phylogeographic patterns among different 
temperate and cold‐adapted herbivorous insects.</p>
         
      </div>
   </section>
   <div style="display:none">
      <ul class="custom-style-list noteList"></ul>
   </div>
   <section id="citedBy" class="article-section article-section__citedBy" data-source="https://onlinelibrary.wiley.com/action/ajaxShowCitedBy?doi=10.1111%2Fj.1365-294X.2012.05622.x">
      <div class="accordion article-accordion">
         <h2><a href="#" class="accordion__control" role="button" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a></h2>
         <div class="accordion__content" role="region" aria-labelledby="citedby-section" id="idss15" style="display: none;"></div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md sticko__parent article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/1365294x/2012/21/14"><img src="mec.gif" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/1365294x/2012/21/14"><span>Volume</span><span class="val">21</span>, <span>Issue</span><span class="val">14</span></a></p><p>July 2012</p><p class="page-range"><span>Pages</span><span> 3385-3390</span></p>



        
        
</div>
</div>



        
        



        <div class="pb-ad">
            <div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_1__container__" style="border: 0pt none; width: 300px; height: 250px;"></div>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x&pubId=58310032&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		</div>
        </div>
    

<div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko tab--flex tabs--xs js--sticko" style="width: 289.983px; top: 621.5px; position: static; height: 0px;"><ul data-mobile-toggle="slide" role="tablist" class="rlist tab__nav w-slide--list cloned"><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-multi-figure"></i><h2>Figures</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-chain"></i><h2>References</h2></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><h2>Related</h2></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" tabindex="-1" aria-selected="false"><i aria-hidden="true" class="icon-Icon_Information"></i><h2>Information</h2></a></li></ul><ul role="presentation" class="rlist tab__content sticko__child" style="width: 289.983px; height: 0px; overflow-y: auto;"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tabpanel" class="tab__pane article-row-right__panes empty" aria-hidden="true"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tabpanel" class="tab__pane article-row-right__panes active" aria-hidden="true"><div class="contentTab"></div><div class="emptyTab">



        
        



        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script><div id="google_ads_iframe_/2215/wly.ecology_000046/molec-eco_mec_2__container__" style="border: 0pt none; width: 300px; height: 250px;"></div>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x&pubId=58310032&placeholderId=1013&productId=101018" target="_blank">
				<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	</div>
</aside>
        </div>
    

</div></li><li class="tab__spinner"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a href="#" data-ux3-role="controller" role="button" class="figure-viewer__ctrl__close"><span class="icon-close_thin"><span class="sr-only">Close Figure Viewer</span></span></a></div><div class="figure-viewer__top__left"><a href="#" role="button" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"><span class="sr-only">Browse All Figures</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"><span class="sr-only">Return to Figure</span></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: calc(100vh - 56px);"><div class="figure-viewer__cent__left"><a href="#" role="button" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"><span class="sr-only">Previous Figure</span></span></a><a href="#" role="button" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"><span class="sr-only">Next Figure</span></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
        </div>
    </div><!--end pagefulltext-->
<div class="w-slide padding-top" style="top: 71px;"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div></main>

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">



        
        <style>
    .footer-links {
        text-align: center;
    }
    .footer-links h3 {
        font-weight: 300;
    }
    .footer-links a {
        font-weight: 600;
    }
    @media (min-width: 992px) {
        .footer-links {
            text-align: left;
        }
    }
</style>










        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h2 class="sr-only">
                Additional links
            </h2>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="footer-links mb-3">
  <h3>About Wiley Online Library</h3>
  <ul class="rlist">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Help &amp; Support</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="footer-links mb-3">
    <h3>Opportunities</h3>
    <ul class="rlist">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="footer-links mb-3">
    <h3>Connect with Wiley</h3>
    <ul class="rlist">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2019 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    




        
        <div class="footer-logo pull-right">
        
<a href="http://www.wiley.com/" title="Wiley Home Page" aria-label="Wiley"><img alt="Wiley Home Page" src="footer-logo-wiley-1510029248417.png"></a>

</div>
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        <!-- [Tag{id&#x3D;320, code&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;688, code&#x3D;ch00, name&#x3D;General Chemistry}, Tag{id&#x3D;1184, code&#x3D;ls2f, name&#x3D;Ecological Genetics}, Tag{id&#x3D;323, code&#x3D;ls40, name&#x3D;Evolution}, Tag{id&#x3D;1206, code&#x3D;ls43, name&#x3D;Molecular Evolution}, Tag{id&#x3D;1207, code&#x3D;ls41, name&#x3D;Evolutionary Biology}, Tag{id&#x3D;60, code&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;1180, code&#x3D;ls2g, name&#x3D;Molecular Ecology}, Tag{id&#x3D;93, code&#x3D;ch, name&#x3D;CHEMISTRY}, Tag{id&#x3D;317, code&#x3D;ls60, name&#x3D;Genetics}, Tag{id&#x3D;103071, code&#x3D;molec-eco_mec, name&#x3D;molec-eco_MEC}, Tag{id&#x3D;101738, code&#x3D;wly.ecology_000046, name&#x3D;wly.ecology_000046}] -->                          <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 




        
        
















<div class="popup login-popup hidden" role="dialog" tabindex="-1" aria-labelledby="loginPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="Cancel Login"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2 id="loginPopupHead">Log in to Wiley Online Library</h2>

            































<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        

        
        <input name="loginUri" value="/doi/full/10.1111/j.1365-294X.2012.05622.x" type="hidden">
        
            




<input name="redirectUri" value="/doi/full/10.1111/j.1365-294X.2012.05622.x" type="hidden">
<input name="loginUri" value="" type="hidden">
            
        
        
        
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="username">Email or Customer ID</label>
                
        </div>
        
        
                <input id="username" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="text">
            
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                



















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        
        
                <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" required="true" type="password">
            
        <span class="password-eye-icon icon-eye hidden"></span>
        
                
    </div>

                
                    
                        <div class="actions">
                            
                            <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">
                                Forgot password?
                            </a>
                        </div>
                    
                
            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            
                            <span class="submit " disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom accessSubmit" name="submitButton" value="Log In" disabled="disabled" type="submit">
                            </span>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            
                <div class="login-container clearfix form-btn">
                    <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                        
                        NEW USER <span aria-hidden="true">&gt;</span>
                    </a>
                    
                        <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x" class="linkBtn float-right instituation-login last-tab-item">
                            
                            INSTITUTIONAL LOGIN <span aria-hidden="true">&gt;</span>
                        </a><a class="fakeFocusoutElement" href="#"></a>
                    
                </div>
            

        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="changePasswordPopupHead">
    <div class="content">
        <a href="#" class="close cancel" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
        <div class="modal__header">
            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="changePasswordPopupHead">Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
                <a href="#" class="fakeFormLastElement last-tab-item"></a><a class="fakeFocusoutElement" href="#"></a>
            </div>
            <div class="success-template hidden">
                <h2>Password Changed Successfully</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        





<div class="popup registration-popup hidden" role="dialog" tabindex="-1" aria-labelledby="createAccountPopupHead">



    <div class="content">
        <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2 id="createAccountPopupHead">Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1111/j.1365-294X.2012.05622.x" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login last-tab-item">
                    
                    Returning user
                </a><a class="fakeFocusoutElement" href="#"></a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden" role="dialog" tabindex="-1" aria-labelledby="resetPasswordPopupHead">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
    <img src="logo-header-1526603583437.png" alt="">
</div>

            <h2 id="resetPasswordPopupHead">Forgot your password?</h2>
          
            <p class="sub">Enter your email address below. </p>

        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                






















    
    
        
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        
        
                <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
            
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">

                <p>"Please check your email for instructions on resetting your password.
                    If you do not receive an email within 10 minutes, your email address may not be registered,
                    and you may need to create a new Wiley Online Library account.</p>

            </div>
        </div>
    </div>
</div>





        
        <div><div class="pb-dropzone" data-pb-dropzone="col-0" title="col-0"></div></div>




        
        




<div class="popup top-drawer request-username-drawer username-popup hidden" role="dialog" tabindex="-1" aria-labelledby="requestUsernamePopupHead">
    <a href="#" class="close" role="button" aria-label="close dialog"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2 id="requestUsernamePopupHead">Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script>
        </div>
    </div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh840" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.4507315700952257&amp;iit=1571396728780&amp;tmr=load%3D1571396723276%26core%3D1571396723310%26main%3D1571396728774%26ifr%3D1571396728784&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fj.1365-294X.2012.05622.x&amp;dt=Trees%20and%2For%20networks%20to%20display%20intraspecific%20DNA%20sequence%20variation%3F&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=description%2Csite_name%2Cimage%2Curl%2Ctype%3DArticle%2Ctitle&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5da99c73ddd91b09&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=title%3DTrees%2520and%252For%2520networks%2520to%2520display%2520intraspecific%2520DNA%2520sequence%2520variation%253F%26type%3DArticle%26url%3Dhttps%253A%252F%252Fonlinelibrary.wiley.com%252Fdoi%252F10.1111%252Fj.1365-294X.2012.05622.x%26image%3Dhttps%253A%252F%252Fwol-prod-cdn.literatumonline.com%252Fcms%252Fattachment%252F5bccabe3-f673-4f05-8647-c009a8afb5d8%252Fmec.2012.21.issue-14.cover.gif%26site_name%3DWiley%2520Online%2520Library%26description%3DPhylogenetic%2520trees%2520and%2520networks%2520are%2520both%2520used%2520in%2520the%2520scientific%2520literature%2520to%2520display%2520DNA%2520sequence%2520variation%2520at%2520the%2520intraspecific%2520level.%2520Should%2520we%2520rather%2520use%2520trees%2520or%2520networks%253F%2520I%2520argue%2520that%2520the%2520proce...&amp;csi=undefined&amp;rev=v8.28.0-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>


<script type="text/javascript" src="main.js"></script>
<script type="text/javascript" src="hoqiproduct.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>
























    











<div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><button class="crossmark-reset crossmark-popup__btn-close"><span class="sr-only">Close crossmark popup</span></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe title="Crossmark service - Intentionally blank" aria-hidden="true" class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul></body><iframe src="https://injections.readcube.com/html/track_epdf.43d7bc64.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe></html>