<!DOCTYPE html>
<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><iframe src="javascript:false" title="" style="width: 0px; height: 0px; border: 0px none; display: none;"></iframe><script type="text/javascript" async="" src="geo2.js"></script>
      <meta name="citation_pii" content="S105579030900373X">
<meta name="citation_issn" content="1055-7903">
<meta name="citation_volume" content="54">
<meta name="citation_lastpage" content="301">
<meta name="citation_issue" content="1">
<meta name="citation_publisher" content="Academic Press">
<meta name="citation_firstpage" content="291">
<meta name="citation_journal_title" content="Molecular Phylogenetics and Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.ympev.2009.09.016">
<meta name="dc.identifier" content="10.1016/j.ympev.2009.09.016">
<meta name="citation_article_type" content="Review article">
<meta property="og:description" content="Approximately 20 years ago, Avise and colleagues proposed the integration of phylogenetics and population genetics for investigating the connection be…">
<meta name="citation_title" content="Phylogeography’s past, present, and future: 10 years after ">
<meta property="og:title" content="Phylogeography’s past, present, and future: 10 years after ">
<meta name="citation_publication_date" content="2010/01/01">
<meta name="citation_online_date" content="2009/09/13">
<meta name="citation_pdf_url" content="https://www.sciencedirect.com/science/article/pii/S105579030900373X/pdfft?md5=1c84d31246620be32e523c2db2a3578b&amp;pid=1-s2.0-S105579030900373X-main.pdf">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR">
      <title>Phylogeography’s past, present, and future: 10 years after  - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S105579030900373X">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel="shortcut icon" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://sdfestaticassets-us-east-1.sciencedirectassets.com/shared-assets/13/images/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="arp.css">
      <link rel="dns-prefetch" href="https://w.usabilla.com/">
      <link rel="dns-prefetch" href="https://www.deepdyve.com/">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <script>
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Molecular Phylogenetics and Evolution"};
        window.pageData = {"content":[{"entitlementType":"package","format":"MIME-XHTML","id":"sd:article:pii:S105579030900373X","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:rev","publicationType":"journal","issn":"1055-7903","volumeNumber":"54","issueNumber":"1"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1534171440145,"loadTime":""},"visitor":{"accessType":"ae:ANON_IP","accountId":"ae:262591","accountName":"ae:International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"ae:12289512","ipAddress":"197.248.44.39","appSessionId":"5d3924b233b2b641f72b0df14d0b6d817073gxrqa"}};
        window.arp = {
          config: {"reduxLogging":false,"assetsBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/79f04dedd9c655edb1e6b0d5d013df22def9d9af","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","googleMapsApiKey":"AIzaSyAV0ZB1XWtRlzJ06pJAW74Oan87lQVgBHY","enableSignInFromEmailModal":true,"bosUrl":"https://feedback.recs.d.elsevier.com/raw/events","bosTimeOut":60000,"strictMode":false},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
      </script>
      <script type="text/javascript" src="204774041.js"></script>
      <!-- begin mPulse embed code -->
      <script>
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
    <script src="satellite-5b3b560664746d57b70018c6.js"></script><script src="https://assets.adobedtm.com/376c5346e33126fdb6b2dbac81e307cbacfd7935/s-code-contents-9c0358adbc3b5986e210099b3bf1d427fc5bd286.js"></script><link rel="preload" href="https://adservice.google.com/adsid/integrator.js?domain=www.sciencedirect.com"><script type="text/javascript" src="https://adservice.google.com/adsid/integrator.js?domain=www.sciencedirect.com"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_238.js" async=""></script><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.3/config/MML_SVG.js?V=2.7.3"></script></head>
    <body><div style="display: none;" id="lightningjs-usabilla_live"><div><iframe id="lightningjs-frame-usabilla_live" frameborder="0"></iframe></div></div>
      <a class="sr-only sr-only-focusable" href="#app">Skip to main content</a>
      <!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://service.elsevier.com/app/answers/detail/a_id/9831">this support page</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
      <div data-iso-key="_0"><div class="App" id="app" role="application" data-reactroot=""><div class="page"><section><div id="header"><div class="u-bg-white" style="overflow:visible" role="banner"><div class="els-header" style="min-height:80px"><a id="els-main-title-link" href="https://www.sciencedirect.com/"><svg id="els-header-wordmark" class="u-fill-orange u-margin-l-top u-margin-s-left u-margin-l-left-from-sm" viewBox="-3334 3439.4 163 26" style="width:163px;height:24px"><title>ScienceDirect</title><g transform="scale(.8125,.8125)"><path d="M-4099.8,4240.4c0-1.8,1-3.6,4.4-3.6c1.7,0,3.7,0.5,5.5,1.6l0.2-3.1c-1.7-0.7-3.5-1.1-5.6-1.1 c-5.5,0-7.8,2.9-7.8,6.4c0,6.6,10.4,7.2,10.4,11.7c0,1.8-1.4,3.6-4.6,3.6c-2,0-4-0.7-5.6-1.6l-0.4,3.1c1.8,0.9,4.2,1.2,6.1,1.2 c5,0,7.9-2.9,7.9-6.5C-4089.4,4245.8-4099.8,4244.9-4099.8,4240.4"></path><path id="c" d="M-4080.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4083.1,4244.3-4080.6,4243.1-4080.3,4242.9"></path><path id="i" d="M-4068,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-4066,4234.1-4067,4233.1-4068,4233.1 M-4069.5,4258.1h3v-17.7h-3V4258.1z"></path><path id="e" d="M-4057.1,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4058.7,4244.4-4057.4,4243.3-4057.1,4243.1z M-4047.5,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4 c0,5.8,3.5,9.2,7.9,9.2c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-4047.5,4248.9"></path><path d="M-4034.8,4240c-2.6,0-4.3,1.3-5.7,3.1l-0.5-2.6h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.5v12.8h3v-11.9 c0.8-1.1,2.5-2.9,2.9-3.1c0.3-0.2,1.5-0.5,2.5-0.5c2.7,0,2.9,1.4,3,4c0,1.4,0,3.7,0,3.7c0,3.5-0.1,7.6-0.1,7.6h3 c0,0,0.1-5.3,0.1-8.2c0-1.8,0-3.5-0.1-5.3C-4029.5,4241-4031.6,4240-4034.8,4240"></path><path d="M-3982.5,4255.6h-4.4v-18.6h4.8c6.4,0,8.2,5.2,8.2,9.2C-3973.8,4252.2-3976.5,4255.6-3982.5,4255.6z M-3981.6,4234.6h-8.4v23.5h8.1c8.6,0,11.5-6.7,11.5-11.9C-3970.4,4240.9-3973.2,4234.6-3981.6,4234.6"></path><path d="M-3950.5,4240c-1.9,0-3.4,1.7-4,3.2l-0.5-2.8h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.4v12.8h3v-11.2 c0.6-1.5,2-4.4,3.7-4.4c1.1,0,1.2,1.2,1.2,1.4l2.5-0.7v-0.2c0,0,0-0.2-0.1-0.5C-3947.4,4240.9-3948.5,4240-3950.5,4240"></path><path d="M-3903.5,4255.2c-1.1,0.4-2,0.8-3,0.8c-1.4,0-1.9-0.8-1.9-2.8v-10.4h4.6v-2.3h-4.6v-4.7h-2.9v4.7h-3.2v2.3h3.2 v11.4c0,3.1,1.6,4.4,3.9,4.4c1.4,0,3-0.5,4.1-0.9L-3903.5,4255.2"></path><g><path d="M-3923.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-3926.1,4244.3-3923.6,4243.1-3923.3,4242.9"></path></g><g><path d="M-3941.6,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-3943.2,4244.4-3941.8,4243.3-3941.6,4243.1z M-3932,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3932,4248.9"></path></g><g><path d="M-3964.5,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-3962.5,4234.1-3963.4,4233.1-3964.5,4233.1 M-3965.9,4258.1h3v-17.7h-3V4258.1z"></path></g><g><path d="M-4004.4,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4006,4244.4-4004.6,4243.3-4004.4,4243.1z M-3994.7,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3994.7,4248.9"></path></g><g><path d="M-4019.1,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4021.8,4244.3-4019.4,4243.1-4019.1,4242.9"></path></g></g></svg></a><div class="move-right u-display-inline-block"><nav class="u-clr-grey8 u-show-from-md"><div class="u-display-inline-block" style="margin-top:29px"><span><span class="u-margin-l-right"><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/browse/journals-and-books" id="els-header-navigation-section-journals-and-books" style="border-bottom:"><span class="anchor-text">Journals &amp; Books</span></a></span><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS105579030900373X" id="els-header-navigation-section-register"><span class="anchor-text">Register</span></a></span><a class="anchor qa-no-js-fallback-link anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS105579030900373X" id="els-header-user-sign-in"><span class="anchor-text">Sign in</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a><span><a class="anchor u-margin-l-hor help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="help" title="Help" aria-label="Help" target="_blank"><span class="anchor-text"><svg focusable="false" viewBox="0 0 114 128" style="margin-top:-10px" width="21.375" height="24" class="icon icon-help"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></span></a></span></div></nav></div><div class="u-hide-from-md move-right"><ul><li style="list-style:none;display:inline-block;margin-top:32px" class="u-margin-s-hor u-margin-l-hor-from-sm"><div class="hamburger-button"><button class="button-link button-link-primary" aria-label="Mobile menu" aria-expanded="false" type="button"><svg viewBox="0 0 40 18" width="40" height="18" y="52"><path d="M0 16h40v2H0zm0-8h40v2H0zm0-8h40v2H0z"></path></svg><span class="button-link-text"></span></button></div></li></ul><div style="bottom:0;left:0;position:fixed;top:0;width:100%;z-index:70;opacity:.8" class="u-bg-grey1 u-display-none"></div><div id="mobile-menu" style="overflow:auto;position:fixed;width:288px;right:0;top:0;z-index:1000;height:100%" aria-label="Mobile menu" class="u-bg-grey7 u-clr-grey1 u-display-none" role="navigation"><div class="u-bg-black panel-s"></div><div class="u-bg-grey8 panel-s"><ul><li style="list-style:none"><a class="anchor journals-and-books-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/browse/journals-and-books" style="border-bottom:" id="mobile-journals-and-books-link"><span class="anchor-text">Journals &amp; Books</span></a></li><li style="list-style:none"><a class="anchor u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS105579030900373X" id="mobile-register-link"><span class="anchor-text">Register</span></a></li></ul></div><div class="panel-s u-bg-grey7"><ul class="text-xs"></ul><ul class="u-margin-s-top text-s"><li style="list-style:none"><a class="anchor anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS105579030900373X" id="mobile-sign-in-out-link" rel="nofollow"><span class="anchor-text">Sign In</span></a></li><span><a class="anchor u-padding-xs-top text-s help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="mobile-help" title="Help" aria-label="Help" target="_blank"><span class="anchor-text">Help</span></a></span></ul></div></div><div class="move-bottom move-center"></div></div></div></div></div><div class="Article" id="mathjax-container"><div class="sticky-outer-wrapper"><div class="sticky-inner-wrapper" style="position: relative; z-index: 1; transform: translate3d(0px, 0px, 0px);"><div class="Toolbar" role="region" aria-label="download options and search"><div class="toolbar-container"><div class="u-show-from-lg col-lg-6">&nbsp;</div><div class="buttons pull-left pad-left"><button class="show-toc-button u-hide-from-lg"><svg focusable="false" viewBox="0 0 104 128" width="19.5" height="24" class="icon icon-list"><path d="m2e1 95a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm14 55h68v1e1h-68zm0-3e1h68v1e1h-68zm0-3e1h68v1e1h-68z"></path></svg><span class="label">Outline</span></button><div class="PdfDownloadButton"><button class="button-link button-link-secondary" id="pdfLink" aria-controls="Download PDF options" type="button"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="button-link-text"><span class="pdf-download-label u-show-inline-from-lg">Download PDF</span><span class="pdf-download-label-short u-hide-from-lg">Download</span></span></button></div><div class="ExportCitation" role="region" id="export-citation"><button class="button ExportCitationButton button-anchor" type="button"><span class="button-text">Export</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div><div class="quick-search-container pull-right pad-right u-show-from-md"><form id="quick-search" class="QuickSearch u-margin-xs-right" action="/search/advanced" method="get"><input class="query" aria-label="Search ScienceDirect" name="qs" placeholder="Search ScienceDirect" type="text"><button class="button button-primary" type="submit" aria-label="Submit search"><span class="button-text"><svg focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><a class="advanced-search-link" href="https://www.sciencedirect.com/search/advanced">Advanced</a><input name="origin" value="article" type="hidden"><input name="zone" value="qSearch" type="hidden"></form></div></div></div></div></div><div class="article-wrapper grid row"><div class="u-show-from-lg col-lg-6"><div class="TableOfContents u-margin-l-bottom" lang="en"><div class="Outline" id="toc-outline"><h2>Outline</h2><ol class="u-padding-s-bottom"><li><a href="#aep-abstract-id10" title="Abstract">Abstract</a></li><li><a href="#aep-keywords-id12" title="Keywords">Keywords</a></li><li><a href="#aep-section-id13" title="1. Introduction– 9 years since Avise, 2000">1. Introduction– 9 years since Avise, 2000</a></li><li><a href="#aep-section-id14" title="2. What kinds of questions has phylogeography addressed?">2. What kinds of questions has phylogeography addressed?</a></li><li><a href="#aep-section-id17" title="3. The emergence of model-based methods in phylogeography">3. The emergence of model-based methods in phylogeography</a></li><li><a href="#aep-section-id21" title="4. Future directions for integrative comparative phylogeography">4. Future directions for integrative comparative phylogeography</a></li><li><a href="#aep-section-id27" title="5. Conclusion">5. Conclusion</a></li><li><a href="#aep-acknowledgment-id28" title="Acknowledgments">Acknowledgments</a></li><li><a href="#aep-bibliography-id29" title="References">References</a></li><li><a href="#aep-glossary-id31" title="Glossary">Glossary</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show full outline</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right" role="main" lang="en"><noscript>JavaScript is disabled on your browser. Please enable JavaScript to use all the features on this page.</noscript><div class="Publication" id="publication"><div class="publication-brand u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903"><img class="publication-brand-image" src="elsevier-non-solus.png" alt="Elsevier"></a></div><div class="publication-volume u-text-center"><h2 class="publication-title" id="publication-title"><a class="publication-title-link" title="Go to Molecular Phylogenetics and Evolution on ScienceDirect" href="https://www.sciencedirect.com/science/journal/10557903">Molecular Phylogenetics and Evolution</a></h2><div class="text-xs"><a title="Go to table of contents for this volume/issue" href="https://www.sciencedirect.com/science/journal/10557903/54/1">Volume 54, Issue 1</a>, <!-- -->January 2010<!-- -->, Pages 291-301</div></div><div class="publication-cover u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903/54/1"><img class="publication-cover-image" src="1-s2.gif" alt="Molecular Phylogenetics and Evolution"></a></div></div><h1 class="Head"><div class="article-dochead"><span>Review</span></div><span class="title-text">Phylogeography’s past, present, and future: 10 years after <a name="bbib9" href="#bib9" class="workspace-trigger">Avise, 2000</a></span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><a class="author size-m workspace-trigger" name="baep-author-id1" href="#%21"><span class="content"><span class="text given-name">M.J.</span><span class="text surname">Hickerson</span><span class="author-ref" id="baff1"><sup>a</sup></span><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id2" href="#%21"><span class="content"><span class="text given-name">B.C.</span><span class="text surname">Carstens</span><span class="author-ref" id="baff2"><sup>b</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id3" href="#%21"><span class="content"><span class="text given-name">J.</span><span class="text surname">Cavender-Bares</span><span class="author-ref" id="baff3"><sup>c</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id4" href="#%21"><span class="content"><span class="text given-name">K.A.</span><span class="text surname">Crandall</span><span class="author-ref" id="baff4"><sup>d</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id5" href="#%21"><span class="content"><span class="text given-name">C.H.</span><span class="text surname">Graham</span><span class="author-ref" id="baff5"><sup>e</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id6" href="#%21"><span class="content"><span class="text given-name">J.B.</span><span class="text surname">Johnson</span><span class="author-ref" id="baff4"><sup>d</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id7" href="#%21"><span class="content"><span class="text given-name">L.</span><span class="text surname">Rissler</span><span class="author-ref" id="baff6"><sup>f</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id8" href="#%21"><span class="content"><span class="text given-name">P.F.</span><span class="text surname">Victoriano</span><span class="author-ref" id="baff7"><sup>g</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a><a class="author size-m workspace-trigger" name="baep-author-id9" href="#%21"><span class="content"><span class="text given-name">A.D.</span><span class="text surname">Yoder</span><span class="author-ref" id="baff8"><sup>h</sup></span><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a></div></div></div><button class="show-hide-details"><svg viewBox="0 0 9 9" class="icon-expand"><path d="M5 7H4V5H2V4h2V2h1v2h2v1H5z"></path><path d="M0 0v9h9V0zm1 1h7v7H1z"></path></svg>Show more</button></div><div class="DoiLink" id="doi-link"><a class="doi" href="https://doi.org/10.1016/j.ympev.2009.09.016" target="_blank" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier">https://doi.org/10.1016/j.ympev.2009.09.016</a><a class="rights-and-content" target="_blank" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S105579030900373X&amp;orderBeanReset=true">Get rights and content</a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts" id="abstracts"><div class="abstract author" id="aep-abstract-id10" lang="en"><h2 class="section-title">Abstract</h2><div id="aep-abstract-sec-id11"><p>Approximately
 20 years ago, Avise and colleagues proposed the integration of 
phylogenetics and population genetics for investigating the connection 
between micro- and macroevolutionary phenomena. The new field was termed
 phylogeography. Since the naming of the field, the statistical rigor of
 phylogeography has increased, in large part due to concurrent advances 
in coalescent theory which enabled model-based parameter estimation and 
hypothesis testing. The next phase will involve phylogeography 
increasingly becoming the integrative and comparative multi-taxon 
endeavor that it was originally conceived to be. This exciting 
convergence will likely involve combining spatially-explicit multiple 
taxon coalescent models, genomic studies of natural selection, 
ecological niche modeling, studies of ecological speciation, community 
assembly and functional trait evolution. This ambitious synthesis will 
allow us to determine the causal links between geography, climate 
change, ecological interactions and the evolution and composition of 
taxa across whole communities and assemblages. Although such integration
 presents analytical and computational challenges that will only be 
intensified by the growth of genomic data in non-model taxa, the rapid 
development of “likelihood-free” approximate Bayesian methods should 
permit parameter estimation and hypotheses testing using complex 
evolutionary demographic models and genomic phylogeographic data. We 
first review the conceptual beginnings of phylogeography and its 
accomplishments and then illustrate how it evolved into a statistically 
rigorous enterprise with the concurrent rise of coalescent theory. 
Subsequently, we discuss ways in which model-based phylogeography can 
interface with various subfields to become one of the most integrative 
fields in all of ecology and evolutionary biology.</p></div></div></div><ul class="issue-navigation" id="issue-navigation"><li class="previous"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790309003741"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-left"><path d="m1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text"><span><strong>Previous </strong>article<span class="extra-detail"> in issue</span></span></span></a></li><li class="next"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790309002115"><span class="button-alternative-text"><strong>Next </strong>article<span class="extra-detail"> in issue</span></span><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords"><div id="aep-keywords-id12" class="keywords-section"><h2 class="section-title">Keywords</h2><div class="keyword"><span>Comparative phylogeography</span></div><div class="keyword"><span>Statistical phylogeography</span></div><div class="keyword"><span>Ecological niche modeling (ENM)</span></div><div class="keyword"><span>Approximate Bayesian computation (ABC)</span></div><div class="keyword"><span>Coalescent theory</span></div></div></div><div class="Body" id="body"><div><section id="aep-section-id13"><h2>1. Introduction– 9 years since <a name="bbib9" href="#bib9" class="workspace-trigger">Avise, 2000</a></h2><p>Interpreted
 literally, the term phylogeography simply means the phylogenetic 
analysis of organismal data in the context of the geographic 
distribution of the organism. Even a cursory look at the literature 
reveals, however, that the meaning of “<em>phylogeography</em>” and the 
field that it describes is considerably more subtle and far-reaching. 
The term that launched the field arrived with a flourish in 1987 with 
the seminal review paper by Avise and colleagues (<a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al., 1987</a>),
 which aimed to unite evolutionary biologists in the disparate fields of
 phylogenetics and population genetics. Reaching back to the work of <a name="bbib54" href="#bib54" class="workspace-trigger">Dobzhansky (1937)</a> and <a name="bbib152" href="#bib152" class="workspace-trigger">Mayr, 1963</a>, <a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al., 1987</a>
 asked for a formal acknowledgement that “microevolutionary processes 
operating within species can be extrapolated to explain 
macroevolutionary differences among species and higher taxa” (p. 489). 
From our present perspective this connection appears obvious, but at the
 time of the paper’s publication the application of explicitly 
historical terms and methods was unknown to many empiricists working at 
or below the species level. This dearth of historical perspective for 
population genetic analysis has changed radically in intervening years, 
due in part to the publication of Avise’s monumental book Phylogeography
 (2000) and the rise of coalescent theory.</p><p>As Avise conceived it, 
phylogeography is the phylogenetic analysis of geographically 
contextualized genetic data for testing hypotheses regarding the causal 
relationship among geographic phenomena, species distributions, and the 
mechanisms driving speciation. <a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al. (1987)</a>
 also implored that genetic data from multiple codistributed taxa could 
augment investigation of deep-seated questions about the geographic, 
geological, and/or climatological phenomena that have generated the 
observed distribution of biodiversity (i.e., comparative 
phylogeography). This approach offers the opportunity of a natural 
experiment where focal objects (codistributed populations), have been 
independently submitted to the same “natural” evolutionary treatments 
(geologic and climate-change scenarios).</p><p>Fundamental to the 
empirical development of phylogeography was analysis of mitochondrial 
data at the species level. Mitochondrial DNA (mtDNA) in animals was 
promoted as the molecular marker of choice due to its lack of 
recombination, putative neutrality, and smaller effective population 
size, and consequently a shorter expected time to reciprocal monophyly 
between geographic regions. But the most salient and revolutionary 
aspect of the nascent field was the practice of treating segments of 
mtDNA, drawn from individuals within and among populations, as the 
operational taxonomic units (OTUs) in a phylogenetic analysis. Although 
this direct link between spatial patterning of different clades within a
 gene genealogy and organismal lineages within species might be 
conceptually misleading under some demographic histories (<a name="bbib113" href="#bib113" class="workspace-trigger">Irwin, 2002</a>),
 it gave phylogeographic analysis its perceived power and appeal. In a 
practical sense, clades within species were often assumed to reflect the
 boundaries of populations without accounting for statistical 
uncertainty under an appropriate statistical model, and phylogeography 
began to thereby implicitly explore the history of clade-defined 
lineages within species.</p><p>While Avise and colleagues catalyzed the 
growth of phylogeography, mathematicians and theoretical population 
geneticists turned population genetics upside down by introducing the 
mathematical formulation of coalescent theory (<a name="bbib108" href="#bib108" class="workspace-trigger">Hudson, 1983</a>, <a name="bbib123" href="#bib123" class="workspace-trigger">Kingman, 1982a</a>, <a name="bbib124" href="#bib124" class="workspace-trigger">Kingman, 1982b</a>, <a name="bbib216" href="#bib216" class="workspace-trigger">Tajima, 1983</a>).
 Traditional population genetics theory was typically based on 
forward-looking diffusion equations that allowed one to predict 
frequencies of alleles in future generations based on modeling entire 
populations (<a name="bbib69" href="#bib69" class="workspace-trigger">Ewens, 1979</a>).
 However, coalescent theory elegantly formalized a powerful way of using
 only the sample of alleles such that gene genealogies are modeled 
backwards in time under virtually any complex demographic history in 
order to estimate phylogeographic parameters such as historical 
population sizes, divergence times, and migration rates given the 
stochastic timing of coalescent events (<a name="bbib235" href="#bib235" class="workspace-trigger">Wakeley, 2008</a>).
 Even so, it took some time before statistical coalescent model-based 
methods percolated into the empirical phylogeography literature. Until 
statistical phylogeographicmethods became more commonly used (<a name="bbib128" href="#bib128" class="workspace-trigger">Knowles and Maddison, 2002</a>),
 empiricists generally relied on equating geographical patterns of the 
gene genealogy with the demographic history, as implied by the word 
“phylogeography”. Yet in the context of the coalescent, a single 
genealogy is but one realization of a stochastic process and estimating a
 single geographically contextualized phylogeny of multiple unlinked 
loci is often a misleading exercise (<a name="bbib59" href="#bib59" class="workspace-trigger">Edwards, 2009</a>, <a name="bbib146" href="#bib146" class="workspace-trigger">Maddison, 1997</a>).
 Instead of using estimated gene genealogies to directly infer the 
demographic history, coalescent methods generally treat these 
genealogies as a transition parameter to obtain estimates of 
biogeographically informative demographic parameters such as divergence 
times and migration rates (<a name="bbib96" href="#bib96" class="workspace-trigger">Hey and Machado, 2003</a>).</p><p>To move beyond equating genealogical patterns with biogeographic processes, <a name="bbib9" href="#bib9" class="workspace-trigger">Avise (2000)</a>
 acknowledged coalescent theory to be the appropriate statistical and 
methodological framework for testing phylogeographic hypotheses. One of 
the insights from post-coalescent phylogeography was that the standard 
single locus mtDNA or cpDNA data sets that were most commonly collected 
in the 1990s were often insufficient for obtaining precise parameter 
estimates and that multi-locus data could dramatically improve the 
performance of analytical methods derived from the coalescent (<a name="bbib33" href="#bib33" class="workspace-trigger">Carling and Brumfield, 2007</a>, <a name="bbib60" href="#bib60" class="workspace-trigger">Edwards and Beerli, 2000</a>, <a name="bbib74" href="#bib74" class="workspace-trigger">Felsenstein, 2006</a>, <a name="bbib101" href="#bib101" class="workspace-trigger">Hickerson et al., 2006a</a>). Phylogeography has therefore expanded its focus to nuclear markers (<a name="bbib55" href="#bib55" class="workspace-trigger">Dolman and Moritz, 2006</a>, <a name="bbib87" href="#bib87" class="workspace-trigger">Hare and Avise, 1998</a>, <a name="bbib88" href="#bib88" class="workspace-trigger">Harlin-Cognato et al., 2007</a>, <a name="bbib111" href="#bib111" class="workspace-trigger">Hurt et al., 2009</a>, <a name="bbib112" href="#bib112" class="workspace-trigger">Ingvarsson, 2008</a>, <a name="bbib140" href="#bib140" class="workspace-trigger">Lee and Edwards, 2008</a>, <a name="bbib179" href="#bib179" class="workspace-trigger">Peters et al., 2008</a>, <a name="bbib196" href="#bib196" class="workspace-trigger">Rosenblum et al., 2007</a>) with careful attention paid to the resolution of individual nuclear haplotypes (<a name="bbib199" href="#bib199" class="workspace-trigger">Salem et al., 2005</a>).
 The coalescent has also become relevant and necessary for 
phylogeographers to estimate species-level phylogeny at low levels of 
divergence given data from multiple loci and multiple individuals per 
subspecies or populations (<a name="bbib61" href="#bib61" class="workspace-trigger">Edwards et al., 2007</a>, <a name="bbib131" href="#bib131" class="workspace-trigger">Kubatko et al., 2009</a>, <a name="bbib171" href="#bib171" class="workspace-trigger">Oliver, 2008</a>).</p><p>As
 intended by the architects of the field, phylogeography is rapidly 
becoming one of the most integrative fields in evolutionary biology, as 
different analytical tools have been developed, and of necessity, as the
 complexity of the hypotheses being addressed has intensified. For 
example, phylogeographic parameter estimates and model testing can be 
potentially combined with ecological niche models (<a name="bbib181" href="#bib181" class="workspace-trigger">Peterson et al., 2002</a>),
 studies of ecological speciation and radiation, tests of community 
assembly models, as well as spatial analysis of quantitative trait 
variation and natural selection. Thus, the field that was originally 
conceived to bridge two disciplines—phylogenetics and population 
genetics—has evolved into one of the most integrative disciplines in all
 of biology. In addition to the intial integration of historical and 
contemporary genetic analysis, phylogeography borrows from fields as 
diverse as geospatial analysis, geology, climatology, and computer 
science, just to name a few. In this review, we will highlight the 
field’s intitial focal areas, touch on analytical advances, and point to
 exciting future directions integrating the analytical advances with 
emerging tools to distinguish between different complex historical 
demographic models while uncovering spatial patterns of adaptation, 
selection, and community membership. Throughout, we suspect that the 
reader will observe the many ways that phylogeographic theory and 
methods are woven together by the conceptual thread first developed by <a name="bbib9" href="#bib9" class="workspace-trigger">Avise, 2000</a>, <a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al., 1987</a>.</p></section><section id="aep-section-id14"><h2>2. What kinds of questions has phylogeography addressed?</h2><section id="aep-section-id15"><h3>2.1. Single species phylogeographic studies</h3><p>Perhaps
 the greatest impact of phylogeographic approaches has been on the most 
basic of biological questions—what is a species? Species definitions are
 many and varied, but many of those in use today have some 
phylogeographic aspects either explicit or implicit in their definitions
 (<a name="bbib11" href="#bib11" class="workspace-trigger">Avise and Ball, 1990</a>, <a name="bbib50" href="#bib50" class="workspace-trigger">De Queiroz, 2007</a>, <a name="bbib219" href="#bib219" class="workspace-trigger">Templeton, 2001</a>).
 These can be incorporated into phylogeographic methods of species 
delimitation and provide operational practicality in the face of real 
data (<a name="bbib139" href="#bib139" class="workspace-trigger">Leavitt et al., 2007</a>, <a name="bbib160" href="#bib160" class="workspace-trigger">Morando et al., 2003</a>, <a name="bbib204" href="#bib204" class="workspace-trigger">Sites and Marshall, 2004</a>).
 However, species delimitation and/or DNA-barcoding methods will be most
 effective when incorporating coalescent stochasticity (<a name="bbib102" href="#bib102" class="workspace-trigger">Hickerson et al., 2006b</a>, <a name="bbib109" href="#bib109" class="workspace-trigger">Hudson and Coyne, 2002</a>, <a name="bbib127" href="#bib127" class="workspace-trigger">Knowles and Carstens, 2007</a>, <a name="bbib183" href="#bib183" class="workspace-trigger">Pons et al., 2006</a>, <a name="bbib194" href="#bib194" class="workspace-trigger">Rosenberg, 2003</a>, <a name="bbib195" href="#bib195" class="workspace-trigger">Rosenberg, 2007</a>) or using robust non-coalescent models (<a name="bbib158" href="#bib158" class="workspace-trigger">Meyer and Paulay, 2005</a>).</p><p>Phylogeographic approaches can also identify historical hybridization events, hybrid zones, occurrences of introgression (<a name="bbib83" href="#bib83" class="workspace-trigger">Gonzalez-Rodriguez et al., 2004</a>, <a name="bbib95" href="#bib95" class="workspace-trigger">Hewitt, 2001</a>, <a name="bbib214" href="#bib214" class="workspace-trigger">Swenson and Howard, 2005</a>)
 and the geographic determinants of isolation. Such insights can be used
 to generate allopatric speciation hypotheses that are subsequently 
tested with genetic data in taxa that span the putative isolating 
barrier. When allopatric speciation is strongly supported, new methods 
have been developed to test alternative allopatric speciation models 
such as peripatric speciation, colonization or vicariance (<a name="bbib100" href="#bib100" class="workspace-trigger">Hickerson and Meyer, 2008</a>) or vicariance with recurrent migration (<a name="bbib97" href="#bib97" class="workspace-trigger">Hey and Nielsen, 2004</a>).</p></section><section id="aep-section-id16"><h3>2.2. Multi-species phylogeographic studies</h3><p>By
 comparing historical patterns of gene flow and divergence among species
 that overlap in time and space, comparative phylogeography has helped 
elucidate the relative influence of shared earth history events on 
current patterns of biodiversity. Although Pleistocene climatic 
oscillations have been implicated as important contributors to 
allopatric divergence (<a name="bbib120" href="#bib120" class="workspace-trigger">Jordan, 1905</a>) and range expansion (<a name="bbib1" href="#bib1" class="workspace-trigger">Adams, 1905</a>)
 since the turn of the century, this was also one of the first results 
in the nascent stages of comparative phylogeographic research (<a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al., 1987</a>).
 Investigations have helped predict range shifts arising from future 
climate change by allowing insights into the dynamics and persistence of
 communities during Pleistocene climate cycling (<a name="bbib94" href="#bib94" class="workspace-trigger">Hewitt, 2000</a>, <a name="bbib92" href="#bib92" class="workspace-trigger">Hewitt, 1996</a>). While some tropical communities appear stable through historical climate-change cycles (<a name="bbib163" href="#bib163" class="workspace-trigger">Moritz et al., 2000</a>),
 other studies suggest that temperate and boreal communities experienced
 severe range fluctuations where codistributed taxa ranges shift 
congruently (<a name="bbib3" href="#bib3" class="workspace-trigger">Alsos et al., 2007</a>, <a name="bbib93" href="#bib93" class="workspace-trigger">Hewitt, 1999</a>) or independently (<a name="bbib37" href="#bib37" class="workspace-trigger">Carstens et al., 2005a</a>, <a name="bbib155" href="#bib155" class="workspace-trigger">McLachlan et al., 2005</a>, <a name="bbib237" href="#bib237" class="workspace-trigger">Wares and Cunningham, 2001</a>) in response to Pleistocene climate cycling.</p><p>Some well-developed regional study systems include regions such as the Australian wet tropics (<a name="bbib201" href="#bib201" class="workspace-trigger">Schneider et al., 1998</a>), southeastern North America (<a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al., 1987</a>, <a name="bbib206" href="#bib206" class="workspace-trigger">Soltis et al., 2006</a>), Baja California (<a name="bbib138" href="#bib138" class="workspace-trigger">Leaché et al., 2007</a>, <a name="bbib189" href="#bib189" class="workspace-trigger">Riddle et al., 2000</a>, <a name="bbib190" href="#bib190" class="workspace-trigger">Riginos, 2005</a>), the North Atlantic (<a name="bbib147" href="#bib147" class="workspace-trigger">Maggs et al., 2008</a>, <a name="bbib172" href="#bib172" class="workspace-trigger">Olsen et al., 2004</a>), the Indo-Pacific coral triangle (<a name="bbib13" href="#bib13" class="workspace-trigger">Barber et al., 2000</a>), the Pacific Northwest of North America (<a name="bbib30" href="#bib30" class="workspace-trigger">Brunsfeld et al., 2001</a>, <a name="bbib37" href="#bib37" class="workspace-trigger">Carstens et al., 2005a</a>), California (<a name="bbib137" href="#bib137" class="workspace-trigger">Lapointe and Rissler, 2005</a>), the Pacific Coast of North America (<a name="bbib45" href="#bib45" class="workspace-trigger">Cook et al., 2001</a>, <a name="bbib115" href="#bib115" class="workspace-trigger">Jacobs et al., 2004</a>), the neotropical rainforests (<a name="bbib31" href="#bib31" class="workspace-trigger">Burney and Brumfield, 2009</a>), East Africa (<a name="bbib76" href="#bib76" class="workspace-trigger">Fjeldsa and Bowie, 2008</a>) and Europe (<a name="bbib94" href="#bib94" class="workspace-trigger">Hewitt, 2000</a>, <a name="bbib240" href="#bib240" class="workspace-trigger">Weiss and Nuno, 2006</a>). Notably, southern hemispheric regions have received much less attention (<a name="bbib22" href="#bib22" class="workspace-trigger">Beheregaray, 2008</a>, <a name="bbib89" href="#bib89" class="workspace-trigger">Heckman et al., 2007</a>, <a name="bbib231" href="#bib231" class="workspace-trigger">Victoriano et al., 2008</a>, <a name="bbib249" href="#bib249" class="workspace-trigger">Yoder et al., 2005</a>).
 However, a key challenge for comparative phylogeography is the need for
 developing analytical tools that can be used to evaluate spatial and 
temporal congruence or incongruence in phylogeographic patterns across 
multiple species, regardless of the system that these tools are applied 
to.</p><p>Inferences from such regional studies have subsequently helped
 inform conservation priorities by identifying and delimiting areas with
 codistributed singular evolutionary histories in order to prioritize 
geographical units for biological conservation. To date, the operative 
criteria for defining conservation areas are richness, endemism and 
phylogenetic diversity (<a name="bbib28" href="#bib28" class="workspace-trigger">Brooks et al., 2002</a>, <a name="bbib135" href="#bib135" class="workspace-trigger">Lamoreux et al., 2006</a>, <a name="bbib173" href="#bib173" class="workspace-trigger">Orme et al., 2005</a>, <a name="bbib208" href="#bib208" class="workspace-trigger">Spathelf and Waite, 2007</a>).
 However, knowledge about where unique evolutionary events or processes 
occurred have complemented diversity information thereby allowing us to 
conserve not only extant biodiversity but the processes that generate 
this diversity (<a name="bbib162" href="#bib162" class="workspace-trigger">Moritz and Faith, 1998</a>).
 Comparative phylogeography thus allows reconstruction of concerted 
evolutionary changes in codistributed species that have been affected by
 past climate changes (e.g., glacial and precipitation cycling) in 
addition to informing conservation planning. This approach will be 
useful for predicting how climate change will genetically, 
demographically, and spatially influence regional biodiversity (<a name="bbib75" href="#bib75" class="workspace-trigger">Ferrier and Guisan, 2006</a>, <a name="bbib215" href="#bib215" class="workspace-trigger">Taberlet and Cheddadi, 2002</a>, <a name="bbib242" href="#bib242" class="workspace-trigger">Williams et al., 2007</a>).
 So far the applications of comparative phylogeography provide a 
powerful battery for understanding evolutionary history and 
strengthening conservation efforts, yet this exciting next phase will 
require the deployment of powerful coalescent model-based statistical 
methods.</p></section></section><section id="aep-section-id17"><h2>3. The emergence of model-based methods in phylogeography</h2><section id="aep-section-id18"><h3>3.1. Descriptive phylogeographic inference</h3><p>In
 the first wave of phylogeography that coincided with the increased use 
of the polymerase chain reaction, study conclusions were often directly 
based on qualitative interpretations of each taxon’s single locus gene 
genealogy such that the shape of phylogenies, the geographic 
distribution of lineages, and estimated dates of gene tree branching 
events, could be used <em>directly</em> to infer the demographic history
 of each taxon. Under this approach, direct interpretations from 
geographically contextualized gene genealogies (i.e., geographically 
restricted monophyly&nbsp;=&nbsp;isolation) were often combined with reporting geographically defined summary statistics (i.e., hierarchically partitioned <em>F</em>
 statistics). For example, estimates of gene flow or divergence times 
can be obtained from the geographic patterns in a gene genealogy (<a name="bbib205" href="#bib205" class="workspace-trigger">Slatkin and Maddison, 1989</a>) or summary statistics that partition genetic differences among and within populations (<a name="bbib166" href="#bib166" class="workspace-trigger">Nei and Li, 1979</a>, <a name="bbib246" href="#bib246" class="workspace-trigger">Wright, 1969</a>).
 Permutation tests on various summary statistics were also commonly used
 to test for demographic expansion and/or geographic structuring of 
genetic variation (<a name="bbib70" href="#bib70" class="workspace-trigger">Excoffier et al., 1992</a>, <a name="bbib78" href="#bib78" class="workspace-trigger">Fu and Li, 1993</a>, <a name="bbib217" href="#bib217" class="workspace-trigger">Tajima, 1989</a>).
 In combining inference from estimated gene genealogies or gene networks
 with permutations of spatially-explicit summary statistics, the nested 
clade phylogeographic analysis (NCPA) became the paradigmadic method 
associated with this era of phylogeography. By testing for an 
association of geography and patterns in the gene genealogy, followed by
 using an inference key to interpret the patterns as being linked with 
particular historical scenarios (<a name="bbib218" href="#bib218" class="workspace-trigger">Templeton, 1998</a>, <a name="bbib220" href="#bib220" class="workspace-trigger">Templeton, 2004</a>),
 NCPA provided a statistical framework for using gene genealogies to 
directly infer demographic history. However, coalescent simulation 
studies from independent researchers have revealed the single locus 
implementation of this method to be potentially problematic (<a name="bbib128" href="#bib128" class="workspace-trigger">Knowles and Maddison, 2002</a>, <a name="bbib176" href="#bib176" class="workspace-trigger">Panchal and Beaumont, 2007</a>, <a name="bbib182" href="#bib182" class="workspace-trigger">Petit and Grivet, 2002</a>).</p><p>While such simulation testing of NCPA has provoked debate (<a name="bbib17" href="#bib17" class="workspace-trigger">Beaumont, 2008b</a>, <a name="bbib81" href="#bib81" class="workspace-trigger">Garrick et al., 2008</a>, <a name="bbib126" href="#bib126" class="workspace-trigger">Knowles, 2008</a>, <a name="bbib220" href="#bib220" class="workspace-trigger">Templeton, 2004</a>, <a name="bbib221" href="#bib221" class="workspace-trigger">Templeton, 2009a</a>, <a name="bbib222" href="#bib222" class="workspace-trigger">Templeton, 2009b</a>),
 it is also a sign that the field is becoming a more statistically 
rigorous endeavor and that empiricists are coming to recognize that 
equating genealogical pattern with demographic and evolutionary 
processes can lead to over interpretation when ignoring coalescent 
stochasticity in the data (<a name="bbib8" href="#bib8" class="workspace-trigger">Arbogast et al., 2002</a>, <a name="bbib60" href="#bib60" class="workspace-trigger">Edwards and Beerli, 2000</a>, <a name="bbib113" href="#bib113" class="workspace-trigger">Irwin, 2002</a>).
 Along these lines, phylogeographic studies are increasingly using 
simulation-based statistical methods that employ an explicit 
parameterized coalescent model to estimate parameters as well as test 
alternative <em>a priori</em> historical hypotheses. In this new wave of studies, the demographic history is not <em>directly</em>
 interpreted from the gene genealogy and therefore the gene genealogy is
 not the central point of a phylogeographic analysis. Instead the gene 
genealogy is a transition variable for connecting data to demographic 
parameters under an explicit statistical coalescent model (<a name="bbib96" href="#bib96" class="workspace-trigger">Hey and Machado, 2003</a>).</p></section><section id="aep-section-id19"><h3>3.2. Model-based statistical phylogeographic inference</h3><p>Using statistical approaches based on coalescent models for parameter estimation and hypothesis testing has been described as <em>statistical phylogeography</em> (<a name="bbib128" href="#bib128" class="workspace-trigger">Knowles and Maddison, 2002</a>) and is philosophically consistent with a methodological approach first described by <a name="bbib41" href="#bib41" class="workspace-trigger">Chamberlin (1890)</a>
 (reprinted 1965). In scientific disciplines that investigate historical
 events that cannot be observed directly or replicated experimentally, 
scientists have long proceeded by considering competing hypotheses that 
serve as separate but plausible explanations for a given phenomenon (<a name="bbib41" href="#bib41" class="workspace-trigger">Chamberlin, 1890</a>
 (reprinted 1965)). Under this approach, one hypothesis is identified as
 more probable relative to others when all available data are more 
likely under the corresponding model. Chamberlin’s epistemological 
strategy fortuitously dovetails with the statistical phylogeographic 
approach whereby coalescent theory is used to build statistical models 
for hypothesis testing under a Bayesian and/or likelihood-based 
framework. Indeed, under the Bayesian/likelihood-based stratagem each 
competing hypotheses can be evaluated by fitting the data to each model 
relative to other models, by way of different decision theoretical 
methods (e.g., Bayes factors or likelihood ratio tests). Alternatively, 
under a null hypothesis framework, a model is treated as a null 
hypothesis whereby the probability of observing data more extreme than 
the observed data is calculated assuming the null hypothesis is true. 
Under either approach, any one competing model is never “true” but 
instead is a useful approximation that should capture the essential 
features of a demographic history and is most useful if it is somewhat 
robust to violation of model assumptions (<a name="bbib5" href="#bib5" class="workspace-trigger">Anderson, 2007</a>, <a name="bbib234" href="#bib234" class="workspace-trigger">Wakeley, 2004</a>).</p><p>These
 statistical phylogeographic approaches are greatly strengthened when a 
wide range of plausible models is considered. One way of informing the 
choice of plausible models is to generate competing models from external
 sources of data that provide clues about past populations, including 
where they were located, when they may have become isolated from other 
populations, or how they were affected by historical events such as 
glaciations, mountain orogeny, or habitat perturbation. External sources
 of data may come from packrat middens (<a name="bbib43" href="#bib43" class="workspace-trigger">Cognato et al., 2003</a>), fossils (<a name="bbib29" href="#bib29" class="workspace-trigger">Brunhoff et al., 2003</a>), paleo-environmental data (<a name="bbib227" href="#bib227" class="workspace-trigger">Tribsch and Schonswetter, 2003</a>), palynological data (<a name="bbib30" href="#bib30" class="workspace-trigger">Brunsfeld et al., 2001</a>) or even ancient DNA (<a name="bbib14" href="#bib14" class="workspace-trigger">Barnes et al., 2007</a>, <a name="bbib85" href="#bib85" class="workspace-trigger">Hadly et al., 2004</a>).
 However, these types of data are not widely available for all systems, 
and a more generalized approach to generating hypotheses is useful (<a name="bbib188" href="#bib188" class="workspace-trigger">Richards et al., 2007</a>).</p><p>Some
 models are relatively basic. For example, one might want to test two 
models, model A that posits that extant populations in the focal taxon 
arose from a single population that persisted since before the last 
glacial maximum (LGM), and model B that posits that extant populations 
descended from two isolated populations that both persisted since before
 the LGM. For example, in South America, different sets of independent 
evidence argue for two alternative histories. In one hypothesized 
history, LGM refugia are distributed mainly in the Coastal Range close 
to the Pacific (<a name="bbib91" href="#bib91" class="workspace-trigger">Heusser et al., 1988</a>). In a second hypothesized history suggested by pollen analyses (<a name="bbib150" href="#bib150" class="workspace-trigger">Markgraf et al., 1995</a>),
 multiple persistent isolated refugia during the LGM were located on 
both slopes of the Andes, or in fragmented areas within the ice shield (<a name="bbib2" href="#bib2" class="workspace-trigger">Allnutt et al., 1999</a>, <a name="bbib148" href="#bib148" class="workspace-trigger">Marchelli et al., 1998</a>, <a name="bbib177" href="#bib177" class="workspace-trigger">Pastorino and Gallo, 2002</a>, <a name="bbib185" href="#bib185" class="workspace-trigger">Premoli et al., 2000</a>, <a name="bbib184" href="#bib184" class="workspace-trigger">Premoli, 1997</a>).</p><p>In
 one commonly used model-based statistical approach, a summary statistic
 is calculated from simulated data sets under each model to obtain a 
distribution of the summary statistic under each respective model. In 
this scheme, the probabilities of both models are evaluated with respect
 to the summary statistic calculated from the empirical data (<a name="bbib125" href="#bib125" class="workspace-trigger">Knowles, 2001</a>). This simulation-based approach has become a useful method in statistical model-based phylogeography (<a name="bbib4" href="#bib4" class="workspace-trigger">Alter et al., 2007</a>, <a name="bbib20" href="#bib20" class="workspace-trigger">Becquet et al., 2007</a>, <a name="bbib24" href="#bib24" class="workspace-trigger">Berthier et al., 2006</a>, <a name="bbib27" href="#bib27" class="workspace-trigger">Boul et al., 2007</a>, <a name="bbib32" href="#bib32" class="workspace-trigger">Burridge et al., 2008</a>, <a name="bbib37" href="#bib37" class="workspace-trigger">Carstens et al., 2005a</a>, <a name="bbib51" href="#bib51" class="workspace-trigger">DeChaine and Martin, 2005</a>, <a name="bbib58" href="#bib58" class="workspace-trigger">Eckert et al., 2008</a>, <a name="bbib65" href="#bib65" class="workspace-trigger">Epps et al., 2005</a>, <a name="bbib99" href="#bib99" class="workspace-trigger">Hickerson and Cunningham, 2005</a>, <a name="bbib125" href="#bib125" class="workspace-trigger">Knowles, 2001</a>, <a name="bbib149" href="#bib149" class="workspace-trigger">Mardulyn and Milinkovitch, 2005</a>, <a name="bbib159" href="#bib159" class="workspace-trigger">Milot et al., 2000</a>, <a name="bbib164" href="#bib164" class="workspace-trigger">Moya et al., 2007</a>, <a name="bbib167" href="#bib167" class="workspace-trigger">Nettel and Dodd, 2007</a>, <a name="bbib178" href="#bib178" class="workspace-trigger">Pavoine and Bailly, 2007</a>, <a name="bbib209" href="#bib209" class="workspace-trigger">Spellman and Klicka, 2006</a>, <a name="bbib210" href="#bib210" class="workspace-trigger">Steele and Storfer, 2006</a>, <a name="bbib223" href="#bib223" class="workspace-trigger">Thalmann et al., 2007</a>, <a name="bbib232" href="#bib232" class="workspace-trigger">Vila et al., 2005</a>, <a name="bbib243" href="#bib243" class="workspace-trigger">Wilson, 2006</a>).</p><p>Another
 common statistical approach is to assume a single model and estimate 
parameters under the model using full likelihood/Bayesian approaches 
that make use of all of the data (<a name="bbib21" href="#bib21" class="workspace-trigger">Beerli and Felsenstein, 2001</a>, <a name="bbib97" href="#bib97" class="workspace-trigger">Hey and Nielsen, 2004</a>, <a name="bbib98" href="#bib98" class="workspace-trigger">Hey and Nielsen, 2007</a>, <a name="bbib132" href="#bib132" class="workspace-trigger">Kuhner, 2006</a>, <a name="bbib134" href="#bib134" class="workspace-trigger">Kuhner et al., 1998</a>).
 However, these full likelihood-based methods become intractable for 
complex phylogeographic models that contain many demographic parameters.
 Fortunately, there are model-based statistical approaches that 
circumvent this problem, such as composite likelihood which treats 
polymorphic site as unlinked and calculates the likelihood function 
accordingly (<a name="bbib168" href="#bib168" class="workspace-trigger">Nielsen and Beaumont, 2009</a>).</p><p>Another
 promising family of methods called approximate Bayesian computation 
(ABC) circumvents this problem by bypassing the computational 
difficulties of calculating the likelihood functions (<a name="bbib18" href="#bib18" class="workspace-trigger">Beaumont et al., 2002</a>, <a name="bbib186" href="#bib186" class="workspace-trigger">Pritchard et al., 1999</a>).
 These “likelihood-free” ABC methods can approximate the posterior 
distribution of parameters thereby accomplishing estimation of 
parameters under an array of complex biogeographic scenarios by 
simulating data from a coalescent model using parameter values that are 
randomly drawn from the <em>prior</em> distribution (<a name="bbib42" href="#bib42" class="workspace-trigger">Chan et al., 2006</a>, <a name="bbib46" href="#bib46" class="workspace-trigger">Cornuet et al., 2008</a>, <a name="bbib66" href="#bib66" class="workspace-trigger">Estoup et al., 2004</a>, <a name="bbib71" href="#bib71" class="workspace-trigger">Excoffier et al., 2005</a>, <a name="bbib117" href="#bib117" class="workspace-trigger">Jobin and Mountain, 2008</a>).
 Here, the posterior is approximated from simulated data sets that most 
closely match the observed data set using sets of summary statistics 
that are identically calculated from each.</p><p>The added flexibility 
and power of ABC is that one can also use it for hypothesis testing in 
addition to parameter estimation by treating a set of models as a 
categorical discrete parameter that is <em>estimated</em> such that model testing and parameter estimation are achieved at the same time (<a name="bbib16" href="#bib16" class="workspace-trigger">Beaumont, 2008a</a>, <a name="bbib35" href="#bib35" class="workspace-trigger">Carnaval et al., 2009</a>, <a name="bbib72" href="#bib72" class="workspace-trigger">Fagundes et al., 2007</a>, <a name="bbib77" href="#bib77" class="workspace-trigger">François et al., 2008</a>, <a name="bbib100" href="#bib100" class="workspace-trigger">Hickerson and Meyer, 2008</a>, <a name="bbib230" href="#bib230" class="workspace-trigger">Verdu et al., 2009</a>).
 While the computational shortcut in ABC does not make total use of the 
data and usually requires choosing summary statistics that are 
informative about parameters of interest (<a name="bbib207" href="#bib207" class="workspace-trigger">Sousa et al., 2009</a>) as well as choosing how to combine summary statistics (<a name="bbib86" href="#bib86" class="workspace-trigger">Hamilton et al., 2005</a>),
 side-stepping the need for an explicit likelihood frees up the 
practitioner to use sufficiently complex historical demographic models.</p></section><section id="aep-section-id20"><h3>3.3. Comparative phylogeographic inference</h3><p>Achieving
 comparative phylogeography’s central goals of testing hypotheses about 
how abiotic and/or ecological processes drive evolution within whole 
communities (<a name="bbib7" href="#bib7" class="workspace-trigger">Arbogast and Kenagy, 2001</a>, <a name="bbib9" href="#bib9" class="workspace-trigger">Avise, 2000</a>, <a name="bbib23" href="#bib23" class="workspace-trigger">Bermingham and Moritz, 1998</a>),
 methods for analysis are being developed that go beyond interpreting 
results from multiple single taxon analyses. One method statistically 
estimates levels of topological congruence across taxa and then 
assembles the genetic datasets from different taxa into a single 
supertree depicting geographic linkages (<a name="bbib137" href="#bib137" class="workspace-trigger">Lapointe and Rissler, 2005</a>).
 This tree can then be tested against potential drivers of lineage 
divergence (e.g., climate) to determine what factors are correlated with
 concordant genetic breaks.</p><p>Another method that is both coalescent
 model-based and combines intra-specific data sets into a single 
analysis is an ABC methods that employs a hierarchical Bayesian model (<a name="bbib103" href="#bib103" class="workspace-trigger">Hickerson et al., 2006c</a>). Hierarchical Bayesian models have been used with ABC to incorporate mutation rate variation across loci (<a name="bbib71" href="#bib71" class="workspace-trigger">Excoffier et al., 2005</a>, <a name="bbib186" href="#bib186" class="workspace-trigger">Pritchard et al., 1999</a>),
 and now this strategy has been used to allow for variability in 
demographic parameters across taxa with a hyper-prior. In this case, 
each taxon’s demographic parameters are drawn from a prior that is 
itself conditional on a hyper-prior. This allows estimating the degree 
of congruence across taxa in these demographic sub-parameters (e.g., 
level of congruence in divergence times or effective population sizes) 
and/or estimating the degree of congruence in historical demographic 
models (e.g., vicariance vs dispersal) via posterior estimates of 
hyper-parameters (<a name="bbib100" href="#bib100" class="workspace-trigger">Hickerson and Meyer, 2008</a>) and hyper-posterior Bayes factors (<a name="bbib121" href="#bib121" class="workspace-trigger">Kass and Raftery, 1995</a>).
 Thus hyper-parameter estimates can inform us about biogeographic 
processes across taxa while demographic parameters are allowed to vary 
independently within each codistributed taxon.</p><p>Comparative 
phylogeographic ABC methods are in their infancy, yet have so far been 
used to test for simultaneous divergence times across codistributed taxa
 in a variety of biogeographic settings (<a name="bbib103" href="#bib103" class="workspace-trigger">Hickerson et al., 2006c</a>, <a name="bbib138" href="#bib138" class="workspace-trigger">Leaché et al., 2007</a>, <a name="bbib225" href="#bib225" class="workspace-trigger">Topp and Winker, 2008</a>, <a name="bbib233" href="#bib233" class="workspace-trigger">Voje et al., 2009</a>).
 They have also been used to test for congruence in biogeographic 
scenarios across taxa such as vicariance or colonization in multiple 
gastropod taxa that span large portions of the Pacific ocean (<a name="bbib100" href="#bib100" class="workspace-trigger">Hickerson and Meyer, 2008</a>)
 as well as testing for congruence in forest refugia models across 
multiple frog taxa distributed along the South American coastal forest (<a name="bbib35" href="#bib35" class="workspace-trigger">Carnaval et al., 2009</a>).</p></section></section><section id="aep-section-id21"><h2>4. Future directions for integrative comparative phylogeography</h2><section id="aep-section-id22"><h3>4.1. Ecological niche models</h3><p>Integrating
 phylogeography with species range distribution models (i.e., ecological
 niche models; ENM; see glossary) is showing enormous promise for 
elucidating how isolation, speciation, and selection are directly or 
indirectly linked to abiotic factors (<a name="bbib130" href="#bib130" class="workspace-trigger">Kozak et al., 2008</a>). If environmental factors are implicated in divergence and speciation, such integration can aide in species delineation (<a name="bbib25" href="#bib25" class="workspace-trigger">Bond and Stockman, 2008</a>, <a name="bbib187" href="#bib187" class="workspace-trigger">Raxworthy et al., 2007</a>, <a name="bbib191" href="#bib191" class="workspace-trigger">Rissler and Apodaca, 2007</a>) and testing models of niche conservatism and niche divergence (<a name="bbib142" href="#bib142" class="workspace-trigger">Losos, 2008</a>, <a name="bbib238" href="#bib238" class="workspace-trigger">Warren et al., 2008</a>).
 Another useful application has been to compare phylogeographic 
historical parameter estimates with ancestral area predictions obtained 
from contemporary and paleo-ENMs (<a name="bbib110" href="#bib110" class="workspace-trigger">Hugall et al., 2002</a>, <a name="bbib180" href="#bib180" class="workspace-trigger">Peterson and Nyári, 2007</a>, <a name="bbib188" href="#bib188" class="workspace-trigger">Richards et al., 2007</a>, <a name="bbib197" href="#bib197" class="workspace-trigger">Ruegg and Hijmans, 2006</a>).</p><p>Comparative
 phylogeographic studies are starting to use ENMs from multiple taxa. At
 first such studies compared paleo-ENMs with phylogeographic parameter 
estimates across codistributed taxa to evaluate the extent to which 
codistributed taxa are spatially and temporally concordant. In one of 
the first of these integrative studies, Carstens and colleagues found 
that divergence time estimates were similar in a vole and a willow, 
which the authors attributed to a similar historic distribution and a 
shared response to climatic change at the end of the Pleistocene (<a name="bbib37" href="#bib37" class="workspace-trigger">Carstens et al., 2005a</a>). However, the paleo-ENM models for these two taxa were markedly different suggesting recent niche divergence (<a name="bbib36" href="#bib36" class="workspace-trigger">Carstens and Richards, 2007</a>).
 Another such study compared paleo-ENMs with phylogeographic inferences 
from 20 intra-specific mammal and bird studies from North America and 
found that 14 out of 20 inferences of late Pleistocene refugia were 
spatially correlated with inferences from ENM models (<a name="bbib236" href="#bib236" class="workspace-trigger">Waltari et al., 2007</a>).
 There is also promise in using hierarchical ABC models to explicitly 
combine comparative phylogeographic data sets with ENM predictions. One 
such study did this to test environmental niche model predictions that 
three codistributed frog taxa co-expanded from putative coastal forest 
refugia after the LGM (<a name="bbib35" href="#bib35" class="workspace-trigger">Carnaval et al., 2009</a>). In this case, the paleo-ENMs were directly used to construct an <em>a priori</em>
 model such that Bayesian hypothesis testing was used to compare the fit
 of the genetic data relative to an alternative historic demographic 
model across all three taxa. Another promising direction will be the 
development of spatially-explicit coalescent models for more direct 
integration between ENMs and phylogeographic inference (<a name="bbib15" href="#bib15" class="workspace-trigger">Barton et al., 2009</a>, <a name="bbib49" href="#bib49" class="workspace-trigger">Currat et al., 2004</a>).</p><p>Nonetheless,
 it important to acknowledge that paleo-ENMs only predict the potential 
ranges under the assumption of niche conservatism (<a name="bbib6" href="#bib6" class="workspace-trigger">Araújo and Guisan, 2006</a>),
 and therefore model-based phylogeographic inference can be a way to 
test competing ancestral range predictions constructed under the 
alternative assumptions of niche stasis and niche liability. Although 
ENMs are a recently developed application with various sources of error (<a name="bbib6" href="#bib6" class="workspace-trigger">Araújo and Guisan, 2006</a>, <a name="bbib143" href="#bib143" class="workspace-trigger">Lozier et al., 2009</a>, <a name="bbib212" href="#bib212" class="workspace-trigger">Stockwell and Peterson, 2002</a>), this enterprise is rapidly advancing and we are bound to see important advances in the near-future.</p></section><section id="aep-section-id23"><h3>4.2. Studies of natural selection</h3><p>Although
 studies of natural selection could be integral to phylogeographic 
inference, they are usually omitted, perhaps due to the field’s early 
emphasis on neutral genetic variation and many of the subsequent 
analytical approaches that were developed around such assumptions. 
However, it is clear that selection plays a central role in generating 
biodiversity (<a name="bbib80" href="#bib80" class="workspace-trigger">Funk et al., 2006</a>),
 and thus should be an important integrative component in future 
comparative phylogeographic studies. As genomic data become available 
for non-model organisms (<a name="bbib229" href="#bib229" class="workspace-trigger">Vera et al., 2008</a>), comparative phylogeographic studies will allow identification of different locus-specific divergent selection patterns (<a name="bbib63" href="#bib63" class="workspace-trigger">Ellegren and Sheldon, 2008</a>, <a name="bbib144" href="#bib144" class="workspace-trigger">Luikart et al., 2003</a>, <a name="bbib145" href="#bib145" class="workspace-trigger">MacCallum and Hill, 2006</a>) between pairs of codistributed taxa or taxa that co-occur along the same geographic gradient (<a name="bbib26" href="#bib26" class="workspace-trigger">Bonin et al., 2006</a>, <a name="bbib118" href="#bib118" class="workspace-trigger">Joost et al., 2007</a>, <a name="bbib119" href="#bib119" class="workspace-trigger">Joost et al., 2008</a>)
 while also testing various multi-taxa demographic historical scenarios.
 This could involve identifying cases of simultaneous expansion into 
novel environments accompanied by congruent or incongruent patterns of 
selection or drift across suites of loci (<a name="bbib82" href="#bib82" class="workspace-trigger">Gavrilets, 2004</a>).
 Such large-scale analysis involving multiple taxa, multiple individuals
 and 100’s of loci will indeed bring serious analytical challenges 
ranging from data management to testing complex highly parameterized 
models of selection and demography, yet methodological advances such as 
ABC or composite likelihood should allow such studies to be tractable in
 the near-future (<a name="bbib168" href="#bib168" class="workspace-trigger">Nielsen and Beaumont, 2009</a>).</p><p>There
 are also now substantial opportunities to integrate studies of natural 
selection via population genomics and multi-species phylogeography with 
studies of quantitative genetics (<a name="bbib211" href="#bib211" class="workspace-trigger">Stinchcombe and Hoekstra, 2008</a>). By comparing variation in quantitative traits with neutral genetic variation (<a name="bbib73" href="#bib73" class="workspace-trigger">Falconer and Mackay, 1996</a>), and incorporating classic quantitative genetics experiments (<a name="bbib245" href="#bib245" class="workspace-trigger">Wright, 1943</a>)
 drawing from populations across their geographic ranges and 
environmental gradients, investigators have another means of detecting 
selection in a biogeographic context. This could not only include using 
direct comparisons of quantitative trait variation with neutral genetic 
variation using Qst—Fst comparisons (<a name="bbib153" href="#bib153" class="workspace-trigger">McKay and Latta, 2002</a>), but also include looking at eco-physiological trait differentiation and selection across geographic or climatic gradients (<a name="bbib39" href="#bib39" class="workspace-trigger">Cavender-Bares, 2007</a>, <a name="bbib57" href="#bib57" class="workspace-trigger">Dudley, 1996</a>, <a name="bbib68" href="#bib68" class="workspace-trigger">Etterson, 2008</a>, <a name="bbib67" href="#bib67" class="workspace-trigger">Etterson, 2004</a>, <a name="bbib247" href="#bib247" class="workspace-trigger">Wright and Stanton, 2007</a>).
 Looking further ahead, model-based phylogeographic inference tools can 
be used to relate divergence times and migration with levels of 
ecologically driven selection at genetic loci and quantitative traits. 
Furthermore, ENMs can be used to quantify and test for environmental 
divergence between pairs of taxa that are hypothesized to have undergone
 divergent selection at quantitative traits, genetic loci as well as 
geographic isolation (<a name="bbib165" href="#bib165" class="workspace-trigger">Nakazato et al., 2008</a>, <a name="bbib238" href="#bib238" class="workspace-trigger">Warren et al., 2008</a>).</p></section><section id="aep-section-id24"><h3>4.3. Ecological speciation</h3><p>Not only should researchers be able to uncover the genetic determinants of species boundaries (<a name="bbib174" href="#bib174" class="workspace-trigger">Orr et al., 2004</a>) using genetic markers believed to be associated with reproductive isolation (<a name="bbib175" href="#bib175" class="workspace-trigger">Palopoli and Wu, 1994</a>, <a name="bbib248" href="#bib248" class="workspace-trigger">Wu and Davis, 1993</a>),
 there is now great promise for comparative phylogeographic studies to 
unravel processes behind ecologically driven speciation (<a name="bbib80" href="#bib80" class="workspace-trigger">Funk et al., 2006</a>, <a name="bbib157" href="#bib157" class="workspace-trigger">Mendelson and Shaw, 2005</a>, <a name="bbib198" href="#bib198" class="workspace-trigger">Rundle and Nosil, 2005</a>, <a name="bbib200" href="#bib200" class="workspace-trigger">Schluter, 2000</a>) and the genomic basis of selection driving these patterns of ecological speciation (<a name="bbib62" href="#bib62" class="workspace-trigger">Egan et al., 2008</a>, <a name="bbib192" href="#bib192" class="workspace-trigger">Rogers and Bernatchez, 2007</a>, <a name="bbib202" href="#bib202" class="workspace-trigger">Scluter, 2009</a>, <a name="bbib244" href="#bib244" class="workspace-trigger">Wood et al., 2008</a>).
 A ‘unified framework’ for testing ecological speciation hypotheses can 
be deployed using divergence time estimates across sister taxon-pairs (<a name="bbib79" href="#bib79" class="workspace-trigger">Funk, 1998</a>, <a name="bbib170" href="#bib170" class="workspace-trigger">Nosil et al., 2009</a>).
 Ecologically driven selection is widely found in nature, yet cases that
 entail full reproductive isolation are less common and depend on either
 the strength of the divergent selection at single traits or the number 
of traits at which selection is acting (<a name="bbib170" href="#bib170" class="workspace-trigger">Nosil et al., 2009</a>).
 When traits under divergent selection can be identified in multiple 
sister taxon-pairs that are co-occur along an environmental gradient or 
have co-expanded into a novel environment, one could use model-based 
methods such as hierarchical ABC to initially subdivide these 
sister-pairs into different temporal waves of isolation via colonization
 and/or geographic expansion (<a name="bbib138" href="#bib138" class="workspace-trigger">Leaché et al., 2007</a>).
 The resulting groupings of taxon-pairs defined by isolation times could
 then be used within a multiple regression framework to estimate 
strength of selection at genetic loci, phenotypic traits as well as 
estimate divergence or stasis at ecological niche dimensions between the
 sister taxon-pairs.</p><p>If timing of isolation is positively correlated with reproductive isolation (<a name="bbib80" href="#bib80" class="workspace-trigger">Funk et al., 2006</a>),
 one could then test if timing of isolation is also correlated with the 
multifarious selection hypothesis where the completeness of reproductive
 isolation is positively correlated with the numbers of loci subject to 
selection (<a name="bbib203" href="#bib203" class="workspace-trigger">Seehausen et al., 2008</a>).
 Alternatively, one could test if the time since isolation or subsequent
 gene flow is correlated with the magnitude of selection at single 
genetic loci across taxon-pairs and thereby more consistent with the 
“stronger selection” hypothesis (<a name="bbib170" href="#bib170" class="workspace-trigger">Nosil et al., 2009</a>).
 Furthermore, if subsequent gene flow rather then time is positively 
correlated with the magnitude of selection, then stronger selection with
 reinforcement can be implicated across a subset of taxon-pairs (<a name="bbib105" href="#bib105" class="workspace-trigger">Hoskin et al., 2005</a>, <a name="bbib106" href="#bib106" class="workspace-trigger">Howard, 1993</a>).
 These inferences of ecological speciation, selection and timing of 
isolation could then be compared to estimates of divergence along an 
array of environmental gradients or ecological niche dimensions using 
ENMs (<a name="bbib129" href="#bib129" class="workspace-trigger">Kozak and Wiens, 2007</a>, <a name="bbib203" href="#bib203" class="workspace-trigger">Seehausen et al., 2008</a>, <a name="bbib238" href="#bib238" class="workspace-trigger">Warren et al., 2008</a>).</p></section><section id="aep-section-id25"><h3>4.4. Integrating comparative phylogeography with studies of community assembly</h3><p>One
 of the original objectives of comparative phylogeography was to resolve
 deep-seated questions about how climate change drives community 
assembly and evolution within whole biotas (<a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al., 1987</a>). Although inter-specific phylogenetic data is increasingly being used to address questions of community assembly (<a name="bbib40" href="#bib40" class="workspace-trigger">Cavender-Bares et al., 2009</a>, <a name="bbib64" href="#bib64" class="workspace-trigger">Emerson and Gillespie, 2008</a>, <a name="bbib114" href="#bib114" class="workspace-trigger">Jabot and Chave, 2009</a>, <a name="bbib156" href="#bib156" class="workspace-trigger">McPeek, 2008</a>, <a name="bbib228" href="#bib228" class="workspace-trigger">Vamosi et al., 2009</a>, <a name="bbib239" href="#bib239" class="workspace-trigger">Webb et al., 2002</a>),
 using comparative phylogeographic data for such purposes has so far 
been handicapped because such studies rarely involve more than a handful
 of codistributed taxa. However, comparative phylogeographic datasets 
are bound to have explosive growth as collecting DNA sequence data 
across a wide diversity of taxa will soon scale up to the level of 
comprehensive ecosystem sampling. Such “community-scale” comparative 
phylogeographic data sets could potentially test classic biogeographic 
hypotheses (e.g., vicariance versus dispersal) at the community level (<a name="bbib34" href="#bib34" class="workspace-trigger">Carlquist, 1966</a>, <a name="bbib193" href="#bib193" class="workspace-trigger">Rosen, 1978</a>), as well as test controversial and fundamental hypotheses of community assembly such as Hubbell’s neutral theory (<a name="bbib107" href="#bib107" class="workspace-trigger">Hubbell, 2001</a>), Tilman’s stochastic competitive assembly model (<a name="bbib224" href="#bib224" class="workspace-trigger">Tilman, 2004</a>), and Diamond’s niche assembly rules (<a name="bbib52" href="#bib52" class="workspace-trigger">Diamond, 1975</a>, <a name="bbib84" href="#bib84" class="workspace-trigger">Gotelli and McCabe, 2002</a>).
 As comparative phylogeographic datasets grow to include &gt;100 
codistributed taxon-pairs, the computational advantages of the 
hierarchical ABC approach will be well suited to test community assembly
 models that are explicitly defined by temporal patterns of dispersal 
and speciation. Although species interactions may not be explicitly 
incorporated into such tets of community assembly models, hierarchical 
ABC could allow combining summary statistics from simulated 
phylogeographic data, phylogenetic data and species abundance data (<a name="bbib114" href="#bib114" class="workspace-trigger">Jabot and Chave, 2009</a>)
 to estimate parameters under various community assembly models as well 
as compare the relative likelihoods of these models. Another promising 
approach for potentially integrating comparative phylogeography with 
studies of community assembly is using fossil pollen data. One study 
used this approach to infer historical range expansion of major tree 
species after the last North American glaciation to understand the rate 
at which species can move in response to climate change and the extent 
to which communities reassemble after disturbance (<a name="bbib155" href="#bib155" class="workspace-trigger">McLachlan et al., 2005</a>).
 Finally, researchers will be able to combine these inferences with 
studies of selection, reproductive isolation, phenotypic evolution as 
well as inferences of niche conservatism and niche evolution using ENMs.</p></section><section id="aep-section-id26"><h3>4.5. Empirical systems</h3><p>Given
 this exciting opportunity to co-estimate divergent selection patterns 
and demographic histories across codistributed non-model taxa, what are 
some candidate empirical systems? In the same way that “model organisms”
 were carefully chosen for whole genomes, there are a plethora of 
interesting “model communities” that stand to benefit from this new wave
 in comparative phylogeography. At the broadest scale, such techniques 
could be deployed to investigate how latitudinal patterns in 
biodiversity arise. Selection driven by environmental gradients has been
 implicated as the cause of biogeographic diversity patterns in the 
tropics (<a name="bbib163" href="#bib163" class="workspace-trigger">Moritz et al., 2000</a>)
 and using emerging genomic tools for comparative phylogeography will 
provide a means to better test this controversial hypothesis as well as 
examine how selection drives biodiversity patterns in temperate regions 
that have experience cyclical range expansions throughout the 
Pleistocene. Such studies will be critical for identifying and 
preserving geographic patterns of endemism (<a name="bbib47" href="#bib47" class="workspace-trigger">Crandall et al., 2000</a>, <a name="bbib161" href="#bib161" class="workspace-trigger">Moritz, 2002</a>),
 just as determining whether populations are differentially adapted is 
an important issue in conservation biology and restoration ecology (<a name="bbib154" href="#bib154" class="workspace-trigger">McKay et al., 2005</a>, <a name="bbib241" href="#bib241" class="workspace-trigger">Wilkinson, 2001</a>).
 For instance, the geography of adaptive genetic variation may be more 
relevant for the conservation of locally adapted ecotypes than that of 
neutral variation (<a name="bbib44" href="#bib44" class="workspace-trigger">Conover et al., 2006</a>).</p><p>These
 new model-based comparative phylogeographic techniques could also be 
used to test patterns of co-speciation in multiple taxon-pairs of plant 
hosts and their insect mutualists or parasites. For example, 
phylogenetic data from multiple western Palearctic oak and gallwasp 
species suggests that host parasite relations are persistent and ancient
 (<a name="bbib213" href="#bib213" class="workspace-trigger">Stone et al., 2009</a>),
 and hierarchical ABC methods could be used to estimate temporal 
patterns of Pleistocene co-expansion from Asian into Europe. 
Subsequently, one could further investigate how timing of expansion in 
the host and parasite taxon-pairs relates to levels of selection, 
numbers of loci under selection, phenotypic divergence, and divergence 
along ecological niche dimensions. In studies of ecological speciation, 
excellent empirical systems for deploying hierarchical ABC methods to 
control for time when uncovering processes behind the ecological, 
physiological and environmental factors behind adaptive divergence and 
reproductive isolation include Andean tomato species (<a name="bbib165" href="#bib165" class="workspace-trigger">Nakazato et al., 2008</a>), stickleback stream-lake and ocean-freshwater taxon-pairs (<a name="bbib90" href="#bib90" class="workspace-trigger">Hendry and Taylor, 2004</a>), Trinidadian guppies (<a name="bbib48" href="#bib48" class="workspace-trigger">Crispo et al., 2006</a>), sister-pairs of walking-sticks living on different host plants (<a name="bbib169" href="#bib169" class="workspace-trigger">Nosil, 2007</a>), African cichlids (<a name="bbib203" href="#bib203" class="workspace-trigger">Seehausen et al., 2008</a>), <em>Gambusia</em> fishes (<a name="bbib136" href="#bib136" class="workspace-trigger">Langerhans et al., 2007</a>) and <em>Heliconius</em> butterflies (<a name="bbib116" href="#bib116" class="workspace-trigger">Jiggins et al., 2004</a>).</p></section></section><section id="aep-section-id27"><h2>5. Conclusion</h2><p>Nearly 10 years after <a name="bbib9" href="#bib9" class="workspace-trigger">Avise (2000)</a> and 20 years after <a name="bbib12" href="#bib12" class="workspace-trigger">Avise et al. (1987)</a>,
 phylogeography is entering a new and exciting phase. With regard to 
data analysis, the field is rapidly moving from descriptive methods and 
into using coalescent models for parameter estimation (<a name="bbib133" href="#bib133" class="workspace-trigger">Kuhner, 2008</a>), <em>a priori</em> model testing (<a name="bbib16" href="#bib16" class="workspace-trigger">Beaumont, 2008a</a>, <a name="bbib38" href="#bib38" class="workspace-trigger">Carstens et al., 2005b</a>, <a name="bbib72" href="#bib72" class="workspace-trigger">Fagundes et al., 2007</a>), the estimation of spatially-explicit demographic histories (<a name="bbib141" href="#bib141" class="workspace-trigger">Lemmon and Moriarty Lemmon, 2008</a>), and testing for temporal and/or spatial congruence across codistributed taxa (<a name="bbib35" href="#bib35" class="workspace-trigger">Carnaval et al., 2009</a>, <a name="bbib103" href="#bib103" class="workspace-trigger">Hickerson et al., 2006c</a>). Concurrent with these methodological advances will be new ways to visualize phylogeographic data (<a name="bbib122" href="#bib122" class="workspace-trigger">Kidd and Ritchie, 2006</a>), a rapid growth in genomic phylogeographic data (<a name="bbib226" href="#bib226" class="workspace-trigger">Townsend et al., 2008</a>, <a name="bbib229" href="#bib229" class="workspace-trigger">Vera et al., 2008</a>), expansion of taxa included in comparative phylogeographic data sets (<a name="bbib10" href="#bib10" class="workspace-trigger">Avise, 2008</a>) as well as advances allowing analysis of ancient DNA (<a name="bbib104" href="#bib104" class="workspace-trigger">Hofreiter, 2008</a>)
 such that researchers can potentially resolve if humans were the causal
 agents in the extinction of large mammals at the last glacial maximum 
rather than climate changes (<a name="bbib14" href="#bib14" class="workspace-trigger">Barnes et al., 2007</a>).</p><p>Nonetheless,
 deciding which set of models to evaluate and use for parameter 
estimation remains to be a substantial challenge given the bewildering 
number of possible histories underlying any phylogeographic data set. 
Moreover, recent understanding of Pleistocene and Holocene climatic 
history suggests short-lived and extreme climate fluctuations with 
consequent cyclical changes of habitat suitability occurred with high 
frequency on timescales of centuries and millennia (<a name="bbib53" href="#bib53" class="workspace-trigger">Ditlevsen et al., 1996</a>, <a name="bbib151" href="#bib151" class="workspace-trigger">Mayewski et al., 2004</a>).
 Although phylogeographic models incorporating sufficient spatial 
complexity and frequency of these recurrent range 
expansions/contractions will require further analytical innovations, the
 rapid development of flexible methods should rise to the task. However,
 it should be recognized that tools for complex model-based inference 
that can be easily used by empiricists such as DIY-ABC (<a name="bbib46" href="#bib46" class="workspace-trigger">Cornuet et al., 2008</a>) is presently limited by the paucity of software packages.</p><p>Coinciding
 with these methodological advances is the explosion of opportunities 
for integrating comparative phylogeographic data with other fields that 
are also rapidly advancing. This includes ENMs (<a name="bbib236" href="#bib236" class="workspace-trigger">Waltari et al., 2007</a>), spatial analysis of genomic signatures in natural selection (<a name="bbib118" href="#bib118" class="workspace-trigger">Joost et al., 2007</a>), spatial analysis of morphological and functional trait evolution, studies of ecological speciation (<a name="bbib202" href="#bib202" class="workspace-trigger">Scluter, 2009</a>), and studies of community assembly that make use of ABC (<a name="bbib114" href="#bib114" class="workspace-trigger">Jabot and Chave, 2009</a>).
 With this potential interdisciplinary synthesis, comparative 
phylogeography is poised to achieve what Avise and colleagues originally
 envisioned - resolving deep-seated puzzles about how climate, 
geography, and ecological interactions determine and interact with 
community composition and evolution.</p></section></div><section id="aep-acknowledgment-id28"><h2>Acknowledgments</h2><p>We thank J.W. Sites, and the organizers of a catalysis meeting at the <span id="GS1">National Evolutionary Synthesis Center</span> (NESCent) NSF #<a href="#GS1">EF-0423641</a>
 on Patagonia Phylogeography as well as thank M. Baker, T. Demos, K. 
Ilves, L. Knowles, A. Vogler and an anonymous reviewer for making 
constructive suggestions. This work was partly supported by the <span id="GS2">National Science Foundation’s</span> (NSF) Partnerships in <span id="GS3">International Research and Education</span> (PIRE) OISE <a href="#GS3">0530267</a> awarded to J. Johnson and Proyecto Fondecyt <a href="#GS3">1090664</a> awarded to P.F. Victoriano.</p></section></div><div class="related-content-links u-hide-from-md"><button class="button button-anchor" type="button"><span class="button-text">Recommended articles</span></button><button class="button button-anchor" type="button"><span class="button-text">Citing articles (<!-- -->297<!-- -->)</span></button></div><div class="Tail"></div><section class="bibliography" id="aep-bibliography-id29"><h2 class="section-title">References</h2><section class="bibliography-sec" id="aep-bibliography-sec-id30"><dl class="references"><dt class="label"><a href="#bbib1" id="ref-id-bib1">Adams, 1905</a></dt><dd class="reference"><div class="contribution">C.C. Adams<strong class="title">The postglacial dispersal of the North American biota</strong></div><div class="host">Biol. Bull., 9 (1905), pp. 53-71</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib2" id="ref-id-bib2">Allnutt et al., 1999</a></dt><dd class="reference"><div class="contribution">T.R. Allnutt, A.C. Newton, A. Lara, A. Premoli, J.J. Armesto, R. Vergara<strong class="title">Genetic variation in <em>Fitzroya cupressoides</em> (alerce), a threatened South American conifer</strong></div><div class="host">Mol. Ecol., 8 (1999), pp. 975-987</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib3" id="ref-id-bib3">Alsos et al., 2007</a></dt><dd class="reference"><div class="contribution">I.G. Alsos, P.B. Eidesen, D. Ehrich, I. Skrede, K. Westergaard, G.H. Jacobsen, J.Y. Landvik, P. Taberlet, C. Brochmann<strong class="title">Frequent long-distance plant colonization in the changing Arctic</strong></div><div class="host">Science, 316 (2007), pp. 1606-1609</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib4" id="ref-id-bib4">Alter et al., 2007</a></dt><dd class="reference"><div class="contribution">S.E. Alter, E. Rynes, S.R. Palumbi<strong class="title">DNA evidence for historic population size and past ecosystem impacts of gray whales</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 104 (2007), pp. 15162-15167</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib5" id="ref-id-bib5">Anderson, 2007</a></dt><dd class="reference"><div class="contribution">D.R. Anderson<strong class="title">Model Based Inference in the Life Sciences: A Primer on Evidence</strong></div><div class="host">Springer, New York (2007)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib6" id="ref-id-bib6">Araújo and Guisan, 2006</a></dt><dd class="reference"><div class="contribution">M.B. Araújo, A. Guisan<strong class="title">Five (or so) challenges for species distribution modeling</strong></div><div class="host">J. Biogeogr., 33 (2006), pp. 1677-1688</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib7" id="ref-id-bib7">Arbogast and Kenagy, 2001</a></dt><dd class="reference"><div class="contribution">B.S. Arbogast, G.J. Kenagy<strong class="title">Comparative phylogeography as an integrative approach to historical biogeography</strong></div><div class="host">J. Biogeogr., 28 (2001), pp. 819-825</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib8" id="ref-id-bib8">Arbogast et al., 2002</a></dt><dd class="reference"><div class="contribution">B.S. Arbogast, S.V. Edwards, J. Wakeley, P. Beerli, J.B. Slowinski<strong class="title">Estimating divergence times from molecular data on phylogenetic and population genetic timescales</strong></div><div class="host">Annu. Rev. Ecol. Syst., 33 (2002), pp. 707-740</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib9" id="ref-id-bib9">Avise, 2000</a></dt><dd class="reference"><div class="contribution">J.C. Avise<strong class="title">Phylogeography: The History and Formation of Species</strong></div><div class="host">Harvard University Press, Cambridge (2000)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib10" id="ref-id-bib10">Avise, 2008</a></dt><dd class="reference"><div class="contribution">J.C. Avise<strong class="title">Phylogeography: retrospect and prospect</strong></div><div class="host">J. Biogeogr., 36 (2008), pp. 3-15</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib11" id="ref-id-bib11">Avise and Ball, 1990</a></dt><dd class="reference"><div class="contribution">J.C. Avise, M.R. Ball Jr.<strong class="title">Principles of geneological concordance in species concepts and biological taxonomy</strong></div><div class="host">D.J. Futuyma, J. Antonovics (Eds.), Oxford Surveys in Evolutionary Biology, Oxford University Press, Oxford (1990), pp. 45-67</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib12" id="ref-id-bib12">Avise et al., 1987</a></dt><dd class="reference"><div class="contribution">J.C. Avise, J. Arnold, R.M. Ball, E. Bermingham, T. Lamb, J.E. Neigel, C.A. Reeb, N.C. Saunders<strong class="title">Intraspecific phylogeography: the mitochondrial DNA bridge between population genetics and systematics</strong></div><div class="host">Ann. Rev. Ecol. Syst., 18 (1987), pp. 489-522</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib13" id="ref-id-bib13">Barber et al., 2000</a></dt><dd class="reference"><div class="contribution">P.H. Barber, S.R. Palumbi, M.V. Erdmann, M.K. Moosa<strong class="title">Biogeography—a marine Wallace’s line?</strong></div><div class="host">Nature, 406 (2000), pp. 692-693</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib14" id="ref-id-bib14">Barnes et al., 2007</a></dt><dd class="reference"><div class="contribution">I. Barnes, B. Shapiro, T. Kuznetsova, A. Sher, D. Guthrie, A. Lister, M.G. Thomas<strong class="title">Genetic structure and extinction of the woolly mammoth <em>Mammuthus primigenius</em> (Blum.)</strong></div><div class="host">Curr. Biol., 17 (2007), pp. 1072-1075</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib15" id="ref-id-bib15">Barton et al., 2009</a></dt><dd class="reference"><span>Barton, N.H., Etheridge, A.M., Veber, A., 2009. A new model for evolution in a spatial continuum. arXiv arXiv:0904.0210v1.</span></dd><dt class="label"><a href="#bbib16" id="ref-id-bib16">Beaumont, 2008a</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont<strong class="title">Joint determination of topology, divergence time and immigration in population trees</strong></div><div class="host">S. Matsumura, P. Forster, C. Renfrew (Eds.), Simulations, Genetics and Human Prehistory, McDonald Institute for Archaeological Research, Cambridge (2008), pp. 135-154</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib17" id="ref-id-bib17">Beaumont, 2008b</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont<strong class="title">On the validity of nested clade phylogeographical analysis</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 2563-2565</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib18" id="ref-id-bib18">Beaumont et al., 2002</a></dt><dd class="reference"><div class="contribution">M.A. Beaumont, W. Zhang, D.J. Balding<strong class="title">Approximate Bayesian computation in population genetics</strong></div><div class="host">Genetics, 162 (2002), pp. 2025-2035</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib20" id="ref-id-bib20">Becquet et al., 2007</a></dt><dd class="reference"><span>Becquet,
 C., Patterson, N., Stone, A.C., Przeworski, M., Reich, D., 2007. 
Genetic Structure of Chimpanzee Populations. PLoS Genet. 3, e66. doi: <a href="https://doi.org/10.1371/journal.pgen.0030066" target="_blank">10.1371/journal.pgen.0030066</a>.</span></dd><dt class="label"><a href="#bbib21" id="ref-id-bib21">Beerli and Felsenstein, 2001</a></dt><dd class="reference"><div class="contribution">P. Beerli, J. Felsenstein<strong class="title">Maximum
 likelihood estimation of a migration matrix and effective population 
sizes in n subpopulations by using a coalescent approach</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 98 (2001), pp. 4563-4568</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib22" id="ref-id-bib22">Beheregaray, 2008</a></dt><dd class="reference"><div class="contribution">L.B. Beheregaray<strong class="title">Twenty years of phylogeography: the state of the field and the challenges for the Southern Hemisphere</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 3754-3774</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib23" id="ref-id-bib23">Bermingham and Moritz, 1998</a></dt><dd class="reference"><div class="contribution">E. Bermingham, C. Moritz<strong class="title">Comparative phylogeography: concepts and applications</strong></div><div class="host">Mol. Ecol., 7 (1998), pp. 367-369</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib24" id="ref-id-bib24">Berthier et al., 2006</a></dt><dd class="reference"><div class="contribution">P. Berthier, L. Excoffier, M. Ruedi<strong class="title">Recurrent replacement of mtDNA and cryptic hybridization between two sibling bat species <em>Myotis myotis</em> and <em>Myotis blythii</em></strong></div><div class="host">Proc. R. Soc. B, 273 (2006), pp. 3101-3109</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib25" id="ref-id-bib25">Bond and Stockman, 2008</a></dt><dd class="reference"><div class="contribution">J. Bond, A. Stockman<strong class="title">An
 integrative method for delimiting cohesion species: finding the 
population-species interface in a group of Californian trapdoor spiders 
with extreme genetic divergence and geographic structuring</strong></div><div class="host">Syst. Biol., 57 (2008), pp. 628-646</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib26" id="ref-id-bib26">Bonin et al., 2006</a></dt><dd class="reference"><div class="contribution">A. Bonin, P. Taberlet, C. Miaud, F. Pompano<strong class="title">Explorative genome scan to detect candidate loci for adaptation along a gradient of altitude in the common frog</strong></div><div class="host">Mol. Biol. Evol., 23 (2006), pp. 773-783</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib27" id="ref-id-bib27">Boul et al., 2007</a></dt><dd class="reference"><div class="contribution">K.E. Boul, W.C. Funk, C.R. Darst, D.C. Cannatella, M.J. Ryan<strong class="title">Sexual selection drives speciation in an Amazonian frog</strong></div><div class="host">Proc. R. Soc. B, 7 (2007), pp. 399-406</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib28" id="ref-id-bib28">Brooks et al., 2002</a></dt><dd class="reference"><div class="contribution">T.M. Brooks, R.A. Mittermeier, C.G. Mittermeier, G.A.B. Da Fonseca, A.B. Rylandas, W.R. Konstant, P. Flick, J. Pilgrim, S. Oldfield, G. Magin, C. Hilton-Taylor<strong class="title">Habitat loss and extinction in the hostspots of biodiversity</strong></div><div class="host">Conserv. Biol., 16 (2002), pp. 909-923</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib29" id="ref-id-bib29">Brunhoff et al., 2003</a></dt><dd class="reference"><div class="contribution">C. Brunhoff, K.E. Galbreath, V.B. Fedorov, J.A. Cook, M. Jaarola<strong class="title">Holartic phylogeography of the root vole (<em>Microtus oeconomus</em>): implications for late Quaternary biogeography of high latitudes</strong></div><div class="host">Mol. Ecol., 12 (2003), pp. 957-968</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib30" id="ref-id-bib30">Brunsfeld et al., 2001</a></dt><dd class="reference"><span>Brunsfeld,
 S.J., Sullivan, J., Soltis, D.E., Soltis, P.S., 2001. Comparative 
Phylogeography of Northwestern North America: A Synthesis. Blackwell 
Publishing.</span></dd><dt class="label"><a href="#bbib31" id="ref-id-bib31">Burney and Brumfield, 2009</a></dt><dd class="reference"><div class="contribution">C.W. Burney, R.T. Brumfield<strong class="title">Ecology predicts levels of genetic differentiation in Neotropical birds</strong></div><div class="host">Am. Nat., 174 (2009), pp. 358-368</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib32" id="ref-id-bib32">Burridge et al., 2008</a></dt><dd class="reference"><div class="contribution">C.P. Burridge, D. Craw, D.C. Jack, T.M. King, J.M. Waters<strong class="title">Does fish ecology predict dispersal across a river drainage divide?</strong></div><div class="host">Evolution, 62 (2008), pp. 1484-1499</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib33" id="ref-id-bib33">Carling and Brumfield, 2007</a></dt><dd class="reference"><span>Carling,
 M.D., Brumfield, R.T., 2007. Gene sampling strategies for multi-locus 
population estimates of genetic diversity (theta). PLoS One 2, e160. 
doi: 10.1371/journal.pone.0000160.</span></dd><dt class="label"><a href="#bbib34" id="ref-id-bib34">Carlquist, 1966</a></dt><dd class="reference"><div class="contribution">C. Carlquist<strong class="title">The biota of long-distance dispersal, I: principles of dispersal and evolution</strong></div><div class="host">Q. Rev. Biol., 41 (1966), pp. 247-270</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib35" id="ref-id-bib35">Carnaval et al., 2009</a></dt><dd class="reference"><div class="contribution">A. Carnaval, M.J. Hickerson, C.F.B. Haddad, M.T. Rodrigues, C. Moritz<strong class="title">Stability predicts genetic diversity in the Brazilian Atlantic Forest Hotspot</strong></div><div class="host">Science, 323 (2009), pp. 785-789</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib36" id="ref-id-bib36">Carstens and Richards, 2007</a></dt><dd class="reference"><div class="contribution">B.C. Carstens, C.L. Richards<strong class="title">Integrating coalescent and ecological niche modeling in comparative phylogeography</strong></div><div class="host">Evolution, 61 (2007), pp. 1439-1454</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib37" id="ref-id-bib37">Carstens et al., 2005a</a></dt><dd class="reference"><div class="contribution">B.C. Carstens, S.J. Brunsfeld, J.R. Demboski, J.D. Good, J. Sullivan<strong class="title">Investigating
 the evolutionary history of the Pacific Northwest mesic forest 
ecosystem: hypothesis testing within a comparative phylogeographic 
framework</strong></div><div class="host">Evolution, 59 (2005), pp. 1639-1652</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib38" id="ref-id-bib38">Carstens et al., 2005b</a></dt><dd class="reference"><div class="contribution">B.C. Carstens, J.D. Degenhardt, A.L. Stevensen, J. Sullivan<strong class="title">Accounting
 for coalescent stochasticity in testing phylogeographic hypotheses: 
modeling Pleistocene population structure in the Idaho Giant Salamander <em>Dicamptodon aterrimus</em></strong></div><div class="host">Mol. Ecol., 14 (2005), pp. 255-265</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib39" id="ref-id-bib39">Cavender-Bares, 2007</a></dt><dd class="reference"><div class="contribution">J. Cavender-Bares<strong class="title">Chilling
 and freezing stress in live oaks (Quercus section Virentes): intra- and
 interspecific variation in PS II sensitivity corresponds to latitude of
 origin</strong></div><div class="host">Photosynth. Res., 94 (2007), pp. 437-453</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib40" id="ref-id-bib40">Cavender-Bares et al., 2009</a></dt><dd class="reference"><div class="contribution">J. Cavender-Bares, K. Kozak, P. Fine, S. Kembel<strong class="title">The merging of community ecology and phylogenetic biology</strong></div><div class="host">Ecol. Lett., 12 (2009), pp. 693-715</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib41" id="ref-id-bib41">Chamberlin, 1890</a></dt><dd class="reference"><span>Chamberlin,
 T.C., 1890. The method of multiple working hypotheses. Science 15, 
92–96. Reprinted from: The method of multiple working hypotheses, 148, 
754–759 (1965).</span></dd><dt class="label"><a href="#bbib42" id="ref-id-bib42">Chan et al., 2006</a></dt><dd class="reference"><div class="contribution">Y.L. Chan, C.N.K. Anderson, E.A. Hadly<strong class="title">Bayesian estimation of the timing and severity of a population bottleneck from ancient DNA</strong></div><div class="host">PLoS Genet., 2 (2006), p. e59</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib43" id="ref-id-bib43">Cognato et al., 2003</a></dt><dd class="reference"><div class="contribution">A.I. Cognato, A.D. Harlin, M.L. Fisher<strong class="title">Genetic structure among Pinyon pine beetle populations (Scolytinae: <em>Ips confusus</em>)</strong></div><div class="host">Environ. Entomol., 32 (2003), pp. 1262-1270</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib44" id="ref-id-bib44">Conover et al., 2006</a></dt><dd class="reference"><div class="contribution">D.O. Conover, L.M. Clarke, S.B. Munch, G.N. Wagner<strong class="title">Spatial and temporal scales of adaptive divergence in marine fishes and the implications for conservation</strong></div><div class="host">Fish Biol., 21 (2006), p. 47</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib45" id="ref-id-bib45">Cook et al., 2001</a></dt><dd class="reference"><div class="contribution">J.A. Cook, A.L. Bidlack, C.J. Conroy, J.R. Demboski, M.A. Fleming, A.M. Runck, K.D. Stone, S.O. MacDonald<strong class="title">A phylogeographic perspective on endemism in the Alexander Archipelago of southeast Alaska</strong></div><div class="host">Biol. Conserv., 97 (2001), pp. 215-227</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib46" id="ref-id-bib46">Cornuet et al., 2008</a></dt><dd class="reference"><div class="contribution">J.-M. Cornuet, F. Santos, M.A. Beaumont, C.P. Robert, J.-M. Marin, D.J. Balding, T. Guillemaud, A. Estoup<strong class="title">Inferring population history with DIY ABC: a user-friendly approach to approximate Bayesian computation</strong></div><div class="host">Bioinformatics, 24 (2008), pp. 2713-2719</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib47" id="ref-id-bib47">Crandall et al., 2000</a></dt><dd class="reference"><div class="contribution">K.A. Crandall, O.R.P. Bininda-Emonds, G.M. Mace, R.K. Wayne<strong class="title">Considering evolutionary processes in conservation biology</strong></div><div class="host">Trends Ecol. Evol., 15 (2000), pp. 290-295</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib48" id="ref-id-bib48">Crispo et al., 2006</a></dt><dd class="reference"><div class="contribution">E. Crispo, P. Bentzen, D.N. Reznick, M.T. Kinnison, A.P. Hendry<strong class="title">The relative influence of natural selection and geography on gene flow in guppies</strong></div><div class="host">Mol. Ecol., 15 (2006), pp. 49-62</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib49" id="ref-id-bib49">Currat et al., 2004</a></dt><dd class="reference"><div class="contribution">M. Currat, N. Ray, L. Excoffier<strong class="title">Splatche: a program to simulate genetic diversity taking into account environmental heterogeneity</strong></div><div class="host">Mol. Ecol. Notes, 4 (2004), pp. 139-142</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib50" id="ref-id-bib50">De Queiroz, 2007</a></dt><dd class="reference"><div class="contribution">K. De Queiroz<strong class="title">Species concepts and species delimitation</strong></div><div class="host">Syst. Biol., 56 (2007), pp. 879-886</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib51" id="ref-id-bib51">DeChaine and Martin, 2005</a></dt><dd class="reference"><div class="contribution">E.G. DeChaine, A.P. Martin<strong class="title">Historical biogeography of two alpine butterflies in the Rocky Mountains: broad-scale concordance and local-scale discordance</strong></div><div class="host">J. Biogeogr., 32 (2005), pp. 1943-1956</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib52" id="ref-id-bib52">Diamond, 1975</a></dt><dd class="reference"><div class="contribution">J.M. Diamond<strong class="title">Assembly of species communities</strong></div><div class="host">M.L. Cody, J.M. Diamond (Eds.), Ecology and Evolution of Communities, Harvard, Belknap (1975), pp. 342-444</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib53" id="ref-id-bib53">Ditlevsen et al., 1996</a></dt><dd class="reference"><div class="contribution">P.D. Ditlevsen, H. Svensmark, S. Johnsen<strong class="title">Contrasting atmospheric and climatic dynamics of last-glacial and Holocene periods</strong></div><div class="host">Nature, 379 (1996), pp. 810-812</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib54" id="ref-id-bib54">Dobzhansky, 1937</a></dt><dd class="reference"><div class="contribution">T.G. Dobzhansky<strong class="title">Genetics and the Origin of Species</strong></div><div class="host">Columbia University, New York (1937)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib55" id="ref-id-bib55">Dolman and Moritz, 2006</a></dt><dd class="reference"><div class="contribution">G. Dolman, C. Moritz<strong class="title">A multilocus perspective on refugial isolation and divergence in rainforest skinks (<em>carlia)</em></strong></div><div class="host">Evolution, 60 (2006), pp. 573-582</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib57" id="ref-id-bib57">Dudley, 1996</a></dt><dd class="reference"><div class="contribution">S.A. Dudley<strong class="title">Differing selection on plant physiological traits in response to environmental water availability: a test of adaptive hypotheses</strong></div><div class="host">Evolution, 50 (1996), pp. 92-102</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib58" id="ref-id-bib58">Eckert et al., 2008</a></dt><dd class="reference"><div class="contribution">A.J. Eckert, B.T. Tearse, B.D. Hall<strong class="title">A
 phylogeographical analysis of the range disjunction for foxtail pine 
(Pinus balfouriana, Pinaceae): the role of Pleistocene glaciation</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 1983-1997</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib59" id="ref-id-bib59">Edwards, 2009</a></dt><dd class="reference"><div class="contribution">S.V. Edwards<strong class="title">Is a new and general theory of molecular systematics emerging?</strong></div><div class="host">Evolution, 63 (2009), pp. 1-19</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib60" id="ref-id-bib60">Edwards and Beerli, 2000</a></dt><dd class="reference"><div class="contribution">S.V. Edwards, P. Beerli<strong class="title">Perspective: gene divergence, population divergence, and the variance in coalescence time in phylogeographic studies</strong></div><div class="host">Evolution, 54 (2000), pp. 1839-1854</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib61" id="ref-id-bib61">Edwards et al., 2007</a></dt><dd class="reference"><div class="contribution">S.V. Edwards, L. Liu, D.K. Pearl<strong class="title">High-resolution species trees without concatenation</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 104 (2007), pp. 5936-5941</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib62" id="ref-id-bib62">Egan et al., 2008</a></dt><dd class="reference"><div class="contribution">S.P. Egan, P. Nosil, D.J. Funk<strong class="title">Selection
 and genomic differentiation during ecological speciation: isolating the
 contributions of host association via a comparative genome scan of <em>Neochlamisus Bebbianae</em> leaf beetles</strong></div><div class="host">Evolution, 62 (2008), pp. 1162-1181</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib63" id="ref-id-bib63">Ellegren and Sheldon, 2008</a></dt><dd class="reference"><div class="contribution">H. Ellegren, B.C. Sheldon<strong class="title">Genetic basis of fitness differences in natural populations</strong></div><div class="host">Nature, 452 (2008), pp. 169-175</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib64" id="ref-id-bib64">Emerson and Gillespie, 2008</a></dt><dd class="reference"><div class="contribution">B.C. Emerson, R.G. Gillespie<strong class="title">Phylogenetic analysis of community assembly and structure over space and time</strong></div><div class="host">Trends Ecol. Evol., 23 (2008), pp. 619-630</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib65" id="ref-id-bib65">Epps et al., 2005</a></dt><dd class="reference"><div class="contribution">C.W. Epps, P.J. Palsbøll, J.D. Wehausen, G.K. Roderick, R.R. Ramey II, D.R. McCullough<strong class="title">Highways block gene flow and cause a rapid decline in genetic diversity of desert bighorn sheep</strong></div><div class="host">Ecol. Lett., 8 (2005), pp. 1029-1038</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib66" id="ref-id-bib66">Estoup et al., 2004</a></dt><dd class="reference"><div class="contribution">A. Estoup, B.A. Beaumont, F. Sennedot, C. Moritz, J.-M. Cornuet<strong class="title">Genetic analysis of complex demographic scenarios: spatially expanding populations of the cane toad, <em>Bufo marinus</em></strong></div><div class="host">Evolution, 58 (2004), pp. 2021-2036</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib67" id="ref-id-bib67">Etterson, 2004</a></dt><dd class="reference"><div class="contribution">J.R. Etterson<strong class="title">Evolutionary potential of <em>Chamaecrista fasciculata</em> in relation to climate change I. Clinal patterns of selection along an environmental gradient in the Great Plains</strong></div><div class="host">Evolution, 58 (2004), pp. 1446-1458</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib68" id="ref-id-bib68">Etterson, 2008</a></dt><dd class="reference"><div class="contribution">J. Etterson<strong class="title">Evolution in response to climate change</strong></div><div class="host">Chuck Fox, S.B. Carroll (Eds.), Conservation Biology: Evolution in Action, Oxford University Press, New York (2008), pp. 147-165</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib69" id="ref-id-bib69">Ewens, 1979</a></dt><dd class="reference"><div class="contribution">W.J. Ewens<strong class="title">Mathematical Population Genetics</strong></div><div class="host">Springer-Verlag, Berlin (1979)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib70" id="ref-id-bib70">Excoffier et al., 1992</a></dt><dd class="reference"><div class="contribution">L. Excoffier, P.E. Smouse, J.M. Quattro<strong class="title">Analysis
 of molecular variance inferred from metric distances among DNA 
haplotypes: application to human mitochondrial DNA restriction data</strong></div><div class="host">Genetics, 131 (1992), pp. 479-491</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib71" id="ref-id-bib71">Excoffier et al., 2005</a></dt><dd class="reference"><div class="contribution">L. Excoffier, A. Estoup, J.-M. Cornuet<strong class="title">Bayesian analysis of an admixture model with mutations and arbitrarily linked markers</strong></div><div class="host">Genetics, 169 (2005), pp. 1727-1738</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib72" id="ref-id-bib72">Fagundes et al., 2007</a></dt><dd class="reference"><div class="contribution">N.J.R. Fagundes, N. Ray, M. Beaumont, S. Neuenschwander, F.M. Salzano, S.L. Bonatto, L. Excoffier<strong class="title">Statistical evaluation of alternative models of human evolution</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 104 (2007), pp. 17614-17619</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib73" id="ref-id-bib73">Falconer and Mackay, 1996</a></dt><dd class="reference"><div class="contribution">D.S. Falconer, T.F.C. Mackay<strong class="title">Introduction to Quantitative Genetics</strong></div><div class="host">Prentice Hall, New York (1996)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib74" id="ref-id-bib74">Felsenstein, 2006</a></dt><dd class="reference"><div class="contribution">J. Felsenstein<strong class="title">Accuracy of coalescent likelihood estimates: do we need more sites, more sequences, or more loci?</strong></div><div class="host">Mol. Biol. Evol., 23 (2006), pp. 691-700</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib75" id="ref-id-bib75">Ferrier and Guisan, 2006</a></dt><dd class="reference"><div class="contribution">S. Ferrier, A. Guisan<strong class="title">Spatial modelling of biodiversity at the community level</strong></div><div class="host">J. Appl. Ecol., 43 (2006), pp. 393-404</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib76" id="ref-id-bib76">Fjeldsa and Bowie, 2008</a></dt><dd class="reference"><div class="contribution">J. Fjeldsa, R.C.K. Bowie<strong class="title">New perspectives on the origin and diversification of Africa’s forest avifauna</strong></div><div class="host">Afr. J. Ecol., 46 (2008), pp. 235-247</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib77" id="ref-id-bib77">François et al., 2008</a></dt><dd class="reference"><span>François, O., Blum, M.G.B., Jakobsson, M., Rosenberg, N.A., 2008. Demographic history of european populations of <em>arabidopsis thaliana</em>. PLoS Genet. 4, e1000075. doi: 1000010.1001371/journal.pgen.1000075.</span></dd><dt class="label"><a href="#bbib78" id="ref-id-bib78">Fu and Li, 1993</a></dt><dd class="reference"><div class="contribution">Y.-X. Fu, W.-H. Li<strong class="title">Statistical tests of neutrality of mutations</strong></div><div class="host">Genetics, 133 (1993), pp. 693-709</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib79" id="ref-id-bib79">Funk, 1998</a></dt><dd class="reference"><div class="contribution">D.J. Funk<strong class="title">Isolating a role for natural selection in speciation: host adaptation and sexual isolation in <em>Neochlamisus bebbianae</em> leaf beetles</strong></div><div class="host">Evolution, 52 (1998), pp. 1744-1759</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib80" id="ref-id-bib80">Funk et al., 2006</a></dt><dd class="reference"><div class="contribution">D.J. Funk, P. Nosil, W.J. Etges<strong class="title">Ecological divergence exhibits consistently positive associations with reproductive isolation across disparate taxa</strong></div><div class="host">PNAS, 103 (2006), pp. 3209-3213</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib81" id="ref-id-bib81">Garrick et al., 2008</a></dt><dd class="reference"><div class="contribution">R.C. Garrick, R.J. Dyer, L.B. Beheregaray, P. Sunnucks<strong class="title">Babies and bathwater: a comment on the premature obituary for nested clade phylogeographical analysis</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 1401-1403</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib82" id="ref-id-bib82">Gavrilets, 2004</a></dt><dd class="reference"><div class="contribution">S. Gavrilets<strong class="title">Fitness Landscapes and the Origin of Species</strong></div><div class="host">Princeton University Press, Princeton, NJ (2004)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib83" id="ref-id-bib83">Gonzalez-Rodriguez et al., 2004</a></dt><dd class="reference"><div class="contribution">A. Gonzalez-Rodriguez, J.F. Bain, J.L. Golden, K. Oyama<strong class="title">Chloroplast
 DNA variation in the Quercus affinis-Q. Laurina complex in Mexico: 
geographical structure and associations with nuclear and morphological 
variation</strong></div><div class="host">Mol. Ecol., 13 (2004), pp. 3467-3476</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib84" id="ref-id-bib84">Gotelli and McCabe, 2002</a></dt><dd class="reference"><div class="contribution">N.J. Gotelli, D.J. McCabe<strong class="title">Species co-occurrence. a meta-analysis of J.M. Diamond’s assembly rules model</strong></div><div class="host">Ecology, 83 (2002), pp. 2091-2096</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib85" id="ref-id-bib85">Hadly et al., 2004</a></dt><dd class="reference"><div class="contribution">E.A. Hadly, U. Ramakrishnan, Y.L. Chan, M. van Tuinen, K. O’Keefe, P.A. Spaeth, C.J. Conroy<strong class="title">Genetic response to climatic change: insights from ancient DNA and phylochronology</strong></div><div class="host">PLoS Biol., 2 (2004), pp. 1600-1609</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib86" id="ref-id-bib86">Hamilton et al., 2005</a></dt><dd class="reference"><div class="contribution">G. Hamilton, M. Currat, N. Ray, G. Heckel, M. Beaumont, L. Excoffier<strong class="title">Bayesian estimation of recent migration rates after a spatial expansion</strong></div><div class="host">Genetics, 170 (2005), pp. 409-417</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib87" id="ref-id-bib87">Hare and Avise, 1998</a></dt><dd class="reference"><div class="contribution">M.P. Hare, J.C. Avise<strong class="title">Population structure in the American Oyster as inferred by nuclear gene genealogies</strong></div><div class="host">Mol. Biol. Evol., 15 (1998), pp. 119-128</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib88" id="ref-id-bib88">Harlin-Cognato et al., 2007</a></dt><dd class="reference"><div class="contribution">A. Harlin-Cognato, T. Markowitz, B. Wursig, R. Honeycutt<strong class="title">Multi-locus phylogeography of the dusky dolphin (<em>Lagenorhynchus obscurus</em>): passive dispersal via the west-wind drift or response to prey species and climate change?</strong></div><div class="host">BMC Evol. Biol., 7 (2007), p. 131</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib89" id="ref-id-bib89">Heckman et al., 2007</a></dt><dd class="reference"><div class="contribution">K.L. Heckman, C.L. Hanley, R. Rasoloarison, A.D. Yoder<strong class="title">Multiple nuclear loci reveal patterns of incomplete lineage sorting and complex species history within western mouse lemurs (<em>Microcebus</em>)</strong></div><div class="host">Mol. Phylogen. Evol., 43 (2007), pp. 353-367</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib90" id="ref-id-bib90">Hendry and Taylor, 2004</a></dt><dd class="reference"><div class="contribution">A.P. Hendry, E.B. Taylor<strong class="title">How
 much of the variation in adaptive divergence can be explained by gene 
flow? An evaluation using lake-stream stickleback pairs</strong></div><div class="host">Evolution, 58 (2004), pp. 2319-2331</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib91" id="ref-id-bib91">Heusser et al., 1988</a></dt><dd class="reference"><div class="contribution">C.J. Heusser, J. Rabassa, A. Brandant, R. Stuckenrath<strong class="title">Late-Holocene vegetation of the Andean Araucaria region, Province Neuquen, Argentina</strong></div><div class="host">Mt. Res. Dev., 8 (1988), pp. 53-63</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib92" id="ref-id-bib92">Hewitt, 1996</a></dt><dd class="reference"><div class="contribution">G.M. Hewitt<strong class="title">Some genetic consequences of ice ages, and their role in divergence and speciation</strong></div><div class="host">Biol. J. Linn. Soc., 58 (1996), pp. 247-276</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib93" id="ref-id-bib93">Hewitt, 1999</a></dt><dd class="reference"><div class="contribution">G.M. Hewitt<strong class="title">Post-glacial re-colonization of European biota</strong></div><div class="host">Biol. J. Linn. Soc., 68 (1999), pp. 87-112</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib94" id="ref-id-bib94">Hewitt, 2000</a></dt><dd class="reference"><div class="contribution">G. Hewitt<strong class="title">The genetic legacy of the quaternary ice ages</strong></div><div class="host">Nature, 405 (2000), pp. 907-913</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib95" id="ref-id-bib95">Hewitt, 2001</a></dt><dd class="reference"><div class="contribution">G.M. Hewitt<strong class="title">Speciation, hybrid zones and phylogeography or seeing genes in space and time</strong></div><div class="host">Mol. Ecol., 10 (2001), pp. 537-549</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib96" id="ref-id-bib96">Hey and Machado, 2003</a></dt><dd class="reference"><div class="contribution">J. Hey, C.A. Machado<strong class="title">The study of structured populations—new hope for a difficult and divided science</strong></div><div class="host">Nat. Rev. Genet., 4 (2003), pp. 535-543</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib97" id="ref-id-bib97">Hey and Nielsen, 2004</a></dt><dd class="reference"><div class="contribution">J. Hey, R. Nielsen<strong class="title">Multilocus methods for estimating population sizes, migration rates and divergence time, with applications to the divergence of <em>Drosophila pseudoobscura</em> and <em>D. persimilis</em></strong></div><div class="host">Genetics, 167 (2004), pp. 747-760</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib98" id="ref-id-bib98">Hey and Nielsen, 2007</a></dt><dd class="reference"><div class="contribution">J. Hey, R. Nielsen<strong class="title">Integration within the Felsenstein equation for improved Markov chain Monte Carlo methods in population genetics</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 104 (2007), pp. 2785-2790</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib99" id="ref-id-bib99">Hickerson and Cunningham, 2005</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, C.W. Cunningham<strong class="title">Contrasting quaternary histories in an ecologically divergent pair of low-dispersing intertidal fish (<em>Xiphister</em>) revealed by multi-locus DNA analysis</strong></div><div class="host">Evolution, 59 (2005), pp. 344-360</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib100" id="ref-id-bib100">Hickerson and Meyer, 2008</a></dt><dd class="reference"><span>Hickerson,
 M.J., Meyer, C., 2008. Testing comparative phylogeographic models of 
marine vicariance and dispersal using a hierarchical Bayesian approach. 
BMC Evol. Biol. doi: <a href="https://doi.org/10.1186/1471-2148-8-322" target="_blank">10.1186/1471-2148-8-322</a>.</span></dd><dt class="label"><a href="#bbib101" id="ref-id-bib101">Hickerson et al., 2006a</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, G. Dolman, C. Moritz<strong class="title">Comparative phylogeographic summary statistics for testing simultaneous vicariance across taxon-pairs</strong></div><div class="host">Mol. Ecol., 15 (2006), pp. 209-224</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib102" id="ref-id-bib102">Hickerson et al., 2006b</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, C. Meyer, C. Moritz<strong class="title">DNA-Barcoding will often fail to discover new animal species over broad parameter space</strong></div><div class="host">Syst. Biol., 55 (2006), pp. 729-739</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib103" id="ref-id-bib103">Hickerson et al., 2006c</a></dt><dd class="reference"><div class="contribution">M.J. Hickerson, E. Stahl, H.A. Lessios<strong class="title">Test for simultaneous divergence using approximate Bayesian computation</strong></div><div class="host">Evolution, 60 (2006), pp. 2435-2453</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib104" id="ref-id-bib104">Hofreiter, 2008</a></dt><dd class="reference"><div class="contribution">M. Hofreiter<strong class="title">Long DNA sequences and large data sets: investigating the Quaternary via ancient DNA</strong></div><div class="host">Q. Sci. Rev., 27 (2008), pp. 2586-2592</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib105" id="ref-id-bib105">Hoskin et al., 2005</a></dt><dd class="reference"><div class="contribution">C.J. Hoskin, M. Higgie, K.R. McDonald, C. Moritz<strong class="title">Reinforcement drives rapid allopatric speciation</strong></div><div class="host">Nature, 437 (2005), pp. 1353-1357</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib106" id="ref-id-bib106">Howard, 1993</a></dt><dd class="reference"><div class="contribution">D.J. Howard<strong class="title">Reinforcement: origin, dynamics, and fate of an evolutionary hypothesis</strong></div><div class="host">R.G. Harrison (Ed.), Hybrid Zones and the Evolutionary Process, Oxford University Press, New York (1993)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib107" id="ref-id-bib107">Hubbell, 2001</a></dt><dd class="reference"><div class="contribution">S.P. Hubbell<strong class="title">The Unified Neutral Theory of Biodiversity and Biogeography</strong></div><div class="host">Princeton University Press, Princeton, NJ (2001)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib108" id="ref-id-bib108">Hudson, 1983</a></dt><dd class="reference"><div class="contribution">R.R. Hudson<strong class="title">Properties of a neutral model with intragenic recombination</strong></div><div class="host">Theor. Popul. Biol., 23 (1983), pp. 183-201</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib109" id="ref-id-bib109">Hudson and Coyne, 2002</a></dt><dd class="reference"><div class="contribution">R.R. Hudson, J.A. Coyne<strong class="title">Mathematical consequences of the genealogical species concept</strong></div><div class="host">Evolution, 56 (2002), pp. 1557-1565</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib110" id="ref-id-bib110">Hugall et al., 2002</a></dt><dd class="reference"><div class="contribution">A. Hugall, C. Moritz, A. Moussalli, J. Stanisic<strong class="title">Reconciling paleodistribution models and comparative phylogeography in the Wet Tropics rainforest land snail <em>Gnarosophia bellendenkerensis</em> (Brazier 1875)</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 99 (2002), pp. 6112-6117</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib111" id="ref-id-bib111">Hurt et al., 2009</a></dt><dd class="reference"><div class="contribution">C. Hurt, A. Anker, N. Knowlton<strong class="title">A multilocus test of simultaneous divergence across the Isthmus of Panama using snapping shrimp in the genus <em>Alpheus</em></strong></div><div class="host">Evolution, 63 (2009), pp. 514-530</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib112" id="ref-id-bib112">Ingvarsson, 2008</a></dt><dd class="reference"><div class="contribution">P.K. Ingvarsson<strong class="title">Multilocus patterns of nucleotide polymorphism and the demographic history of <em>Populus tremula</em></strong></div><div class="host">Genetics, 180 (2008), pp. 329-340</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib113" id="ref-id-bib113">Irwin, 2002</a></dt><dd class="reference"><div class="contribution">D.E. Irwin<strong class="title">Phylogeographic breaks without geographic barriers</strong></div><div class="host">Evolution, 56 (2002), pp. 2383-2394</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib114" id="ref-id-bib114">Jabot and Chave, 2009</a></dt><dd class="reference"><div class="contribution">F. Jabot, J. Chave<strong class="title">Inferring
 the parameters of the neutral theory of biodiversity using phylogenetic
 information and implications for tropical forests</strong></div><div class="host">Ecol. Lett., 12 (2009), pp. 239-248</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib115" id="ref-id-bib115">Jacobs et al., 2004</a></dt><dd class="reference"><div class="contribution">D.K. Jacobs, T.A. Haney, K.D. Louie<strong class="title">Genes, diversity, and geologic processes on the Pacific coast</strong></div><div class="host">Annu. Rev. Earth Planet. Sci., 32 (2004), pp. 601-652</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib116" id="ref-id-bib116">Jiggins et al., 2004</a></dt><dd class="reference"><div class="contribution">C.D. Jiggins, C. Estrada, A. Rodrigues<strong class="title">Mimicry and the evolution of premating isolation in <em>Heliconius melpomene</em> (Linnaeus)</strong></div><div class="host">J. Evol. Biol., 17 (2004), pp. 680-691</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib117" id="ref-id-bib117">Jobin and Mountain, 2008</a></dt><dd class="reference"><div class="contribution">M.J. Jobin, J.L. Mountain<strong class="title">REJECTOR: software for population history inference from genetic data via a rejection algorithm</strong></div><div class="host">Bioinformatics, 24 (2008), pp. 2936-2937</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib118" id="ref-id-bib118">Joost et al., 2007</a></dt><dd class="reference"><div class="contribution">S.A.B. Joost, M.W. Bruford, L. Després, C. Conord, G. Erhardt, P. Taberlet<strong class="title">A spatial analysis method (SAM) to detect candidate loci for selection: towards a landscape genomics approach to adaptation</strong></div><div class="host">Mol. Ecol., 16 (2007), pp. 3955-3969</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib119" id="ref-id-bib119">Joost et al., 2008</a></dt><dd class="reference"><div class="contribution">S. Joost, M. Kalbermaten, A. Bonin<strong class="title">Spatial
 analysis method (SAM): a software tool combining molecular and 
environmental data to identify candidate loci for selection</strong></div><div class="host">Mol. Ecol. Resour., 8 (2008), p. 957</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib120" id="ref-id-bib120">Jordan, 1905</a></dt><dd class="reference"><div class="contribution">D.S. Jordan<strong class="title">The origin of species through isolation</strong></div><div class="host">Science, 22 (1905), pp. 545-562</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib121" id="ref-id-bib121">Kass and Raftery, 1995</a></dt><dd class="reference"><div class="contribution">R.E. Kass, A. Raftery<strong class="title">Bayesian factors</strong></div><div class="host">J. Am. Stat. Assoc., 90 (1995), pp. 773-795</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib122" id="ref-id-bib122">Kidd and Ritchie, 2006</a></dt><dd class="reference"><div class="contribution">D.M. Kidd, M.G. Ritchie<strong class="title">Phylogeographic information systems: putting the geography into phylogeography</strong></div><div class="host">J. Biogeogr., 33 (2006), pp. 1851-1865</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib123" id="ref-id-bib123">Kingman, 1982a</a></dt><dd class="reference"><div class="contribution">J.F.C. Kingman<strong class="title">On the genealogy of large populations</strong></div><div class="host">J. Appl. Probab., 19a (1982), pp. 27-43</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib124" id="ref-id-bib124">Kingman, 1982b</a></dt><dd class="reference"><div class="contribution">J.F.C. Kingman<strong class="title">The coalescent</strong></div><div class="host">Stoch. Process. Appl., 13 (1982), pp. 235-248</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib125" id="ref-id-bib125">Knowles, 2001</a></dt><dd class="reference"><div class="contribution">L.L. Knowles<strong class="title">Did the Pleistocene glaciations promote divergence? Tests of explicit refugial models in montane grasshopprers</strong></div><div class="host">Mol. Ecol., 10 (2001), pp. 691-702</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib126" id="ref-id-bib126">Knowles, 2008</a></dt><dd class="reference"><div class="contribution">L.L. Knowles<strong class="title">Why does a method that fails continue to be used?</strong></div><div class="host">Evolution, 62 (2008), pp. 2713-2717</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib127" id="ref-id-bib127">Knowles and Carstens, 2007</a></dt><dd class="reference"><div class="contribution">L.L. Knowles, B.C. Carstens<strong class="title">Delimiting species without monophyletic gene trees</strong></div><div class="host">Syst. Biol., 56 (2007), pp. 887-895</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib128" id="ref-id-bib128">Knowles and Maddison, 2002</a></dt><dd class="reference"><div class="contribution">L.L. Knowles, W.P. Maddison<strong class="title">Statistical phylogeography</strong></div><div class="host">Mol. Ecol., 11 (2002), pp. 2623-2635</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib129" id="ref-id-bib129">Kozak and Wiens, 2007</a></dt><dd class="reference"><div class="contribution">K.H. Kozak, J.J. Wiens<strong class="title">Climatic zonation drives latitudinal variation in speciation mechanisms</strong></div><div class="host">Proc. Biol. Sci., 274 (2007), pp. 2995-3003</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib130" id="ref-id-bib130">Kozak et al., 2008</a></dt><dd class="reference"><div class="contribution">K.H. Kozak, C.H. Graham, J.J. Wiens<strong class="title">Integrating GIS-based environmental data into evolutionary biology</strong></div><div class="host">Trends Ecol. Evol., 23 (2008), pp. 141-148</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib131" id="ref-id-bib131">Kubatko et al., 2009</a></dt><dd class="reference"><span>Kubatko,
 L., Carstens, B.C., Knowles, L.L., 2009. Species tree estimation using 
maximum likelihood for gene trees under coalescence. Bioinformatics. 
doi: <a href="https://doi.org/10.1093/bioinformatics/btp079" target="_blank">10.1093/bioinformatics/btp079</a>.</span></dd><dt class="label"><a href="#bbib132" id="ref-id-bib132">Kuhner, 2006</a></dt><dd class="reference"><div class="contribution">M.K. Kuhner<strong class="title">LAMARC 2.0: maximum likelihood and Bayesian estimation of population parameters</strong></div><div class="host">Bioinformatics, 22 (2006), pp. 768-770</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib133" id="ref-id-bib133">Kuhner, 2008</a></dt><dd class="reference"><div class="contribution">M.K. Kuhner<strong class="title">Coalescent genealogy samplers: windows into population history</strong></div><div class="host">Trends Ecol. Evol., 24 (2008), pp. 86-93</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib134" id="ref-id-bib134">Kuhner et al., 1998</a></dt><dd class="reference"><div class="contribution">M.K. Kuhner, J. Yamato, J. Felsenstein<strong class="title">Maximum likelihood estimation of population growth rates based on the coalescent</strong></div><div class="host">Genetics, 149 (1998), pp. 429-434</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib135" id="ref-id-bib135">Lamoreux et al., 2006</a></dt><dd class="reference"><div class="contribution">J.F. Lamoreux, J.C. Morrison, T.H. Ricketts, D.M. Olson, E. Dinerstein, M.W. McKnight, H.H. Shugart<strong class="title">Global tests of biodiversity concordance and the importance of endemism</strong></div><div class="host">Nature, 440 (2006), pp. 212-214</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib136" id="ref-id-bib136">Langerhans et al., 2007</a></dt><dd class="reference"><div class="contribution">R.B. Langerhans, M.E. Gifford, E.O. Joseph, D. Funk<strong class="title">Ecological speciation in gambusia fishes</strong></div><div class="host">Evolution, 61 (2007), pp. 2056-2074</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib137" id="ref-id-bib137">Lapointe and Rissler, 2005</a></dt><dd class="reference"><div class="contribution">F.-J. Lapointe, L.J. Rissler<strong class="title">Congruence, concensus, and the comparative phylogeography of codistributed species in California</strong></div><div class="host">Am. Nat., 166 (2005), pp. 290-299</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib138" id="ref-id-bib138">Leaché et al., 2007</a></dt><dd class="reference"><div class="contribution">A. Leaché, S.A. Crews, M.J. Hickerson<strong class="title">Did an ancient seaway across Baja California cause community isolation?</strong></div><div class="host">Biol. Lett., 3 (2007), pp. 646-650</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib139" id="ref-id-bib139">Leavitt et al., 2007</a></dt><dd class="reference"><div class="contribution">D.H. Leavitt, R.L. Bezy, K.A. Crandal, J.W. Sites Jr.<strong class="title">Multi-locus
 DNA sequence data reveal a history of deep cryptic vicariance and 
habitat-driven convergence in the desert night lizard <em>Xantusia vigilis</em> species complex (Squamata: Xantusiidae)</strong></div><div class="host">Mol. Ecol., 16 (2007), pp. 4455-4481</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib140" id="ref-id-bib140">Lee and Edwards, 2008</a></dt><dd class="reference"><div class="contribution">J.Y. Lee, S.V. Edwards<strong class="title">Divergence across Australia’s carpentarian barrier: statistical phylogeography of the red-backed fairy wren (<em>Malurus melanocephalus</em>)</strong></div><div class="host">Evolution, 62 (2008), pp. 3117-3134</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib141" id="ref-id-bib141">Lemmon and Moriarty Lemmon, 2008</a></dt><dd class="reference"><div class="contribution">A.R. Lemmon, E. Moriarty Lemmon<strong class="title">A likelihood framework for estimating phylogeographic history using geographically continuous genetic data</strong></div><div class="host">Syst. Biol., 57 (2008), pp. 544-561</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib142" id="ref-id-bib142">Losos, 2008</a></dt><dd class="reference"><div class="contribution">J.B. Losos<strong class="title">Phylogenetic
 niche conservatism, phylogenetic signal and the relationship between 
phylogenetic relatedness and ecological similarity among species</strong></div><div class="host">Ecol. Lett., 11 (2008), pp. 995-1003</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib143" id="ref-id-bib143">Lozier et al., 2009</a></dt><dd class="reference"><span>Lozier,
 J.D., Aniello, P., Hickerson, M.J., 2009. Predicting the distribution 
of Sasquatch in western North America: anything goes with ecological 
niche modeling. J. Biogeogr. doi: <a href="https://doi.org/10.1111/j.1365-2699.2009.02152.x" target="_blank">10.1111/j.1365-2699.2009.02152.x</a>.</span></dd><dt class="label"><a href="#bbib144" id="ref-id-bib144">Luikart et al., 2003</a></dt><dd class="reference"><div class="contribution">G. Luikart, P.R. England, D. Tallman, S. Jordan, P. Taberlet<strong class="title">The power and promise of population genomics: from genotyping to genome typing</strong></div><div class="host">Nat. Rev. Genet., 4 (2003), pp. 981-994</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib145" id="ref-id-bib145">MacCallum and Hill, 2006</a></dt><dd class="reference"><div class="contribution">C. MacCallum, E. Hill<strong class="title">Being positive about selection</strong></div><div class="host">PLoS Biol., 4 (2006), pp. 293-295</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib146" id="ref-id-bib146">Maddison, 1997</a></dt><dd class="reference"><div class="contribution">W.P. Maddison<strong class="title">Gene trees in species trees</strong></div><div class="host">Syst. Biol., 46 (1997), pp. 523-536</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib147" id="ref-id-bib147">Maggs et al., 2008</a></dt><dd class="reference"><div class="contribution">C.A. Maggs, R. Castilho, D. Foltz, C. Henzler, M.T. Jolly, J. Kelly, J. Olsen, K.E. Perez, W. Stam, R. Väinölä, F. Viard, J. Wares<strong class="title">Evaluating signatures of glacial refugia for North Atlantic benthic marine taxa</strong></div><div class="host">Ecology, 89 (2008), pp. S108-S122</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib148" id="ref-id-bib148">Marchelli et al., 1998</a></dt><dd class="reference"><div class="contribution">P. Marchelli, L. Gallo, F. Scholz, B. Ziegenhagen<strong class="title">Chloroplast DNA markers reveal a geographical divide across Argentinean southern beech <em>Nothofagus nervosa</em> (Phil.) Dim. et Mil. distribution area</strong></div><div class="host">Theor. Appl. Genet., 97 (1998), pp. 642-646</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib149" id="ref-id-bib149">Mardulyn and Milinkovitch, 2005</a></dt><dd class="reference"><div class="contribution">P. Mardulyn, M.C. Milinkovitch<strong class="title">Inferring contemporary levels of gene flow and demographic history in a local population of the leaf beetle <em>Gonioctena olivacea</em> from mitochondrial DNA sequence variation</strong></div><div class="host">Mol. Ecol., 14 (2005), pp. 1641-1653</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib150" id="ref-id-bib150">Markgraf et al., 1995</a></dt><dd class="reference"><div class="contribution">V. Markgraf, M. McGlone, G. Hope<strong class="title">Neogene paleoenvironmental and paleoclimatic change in southern temperate ecosystems: a southern perspective</strong></div><div class="host">Trends Ecol. Evol., 10 (1995), pp. 143-147</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib151" id="ref-id-bib151">Mayewski et al., 2004</a></dt><dd class="reference"><div class="contribution">P.A. Mayewski, E.E. Rohling, J.C. Stager, W. KarlÈn, K.A. Maasch, D.M. L, E.A. Meyerson, F. Gasse, S. van Kreveld, K. Holmgren, J. Lee-Thorp, G. Rosqvist, F. Rack, M. Staubwasser, R.R. Schneider, E.J. Steig<strong class="title">Holocene climate variability</strong></div><div class="host">Quat. Res., 62 (2004), pp. 243-255</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib152" id="ref-id-bib152">Mayr, 1963</a></dt><dd class="reference"><div class="contribution">E. Mayr<strong class="title">Animal Species and Evolution</strong></div><div class="host">Harvard University Press, Cambridge, MA (1963)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib153" id="ref-id-bib153">McKay and Latta, 2002</a></dt><dd class="reference"><div class="contribution">J.K. McKay, R.G. Latta<strong class="title">Adaptive population divergence. markers, QTL and traits</strong></div><div class="host">Trends Ecol. Evol., 17 (2002), pp. 285-291</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib154" id="ref-id-bib154">McKay et al., 2005</a></dt><dd class="reference"><div class="contribution">J.K. McKay, C.E. Christian, S. Harrison, K.J. Rice<strong class="title">How local is local?—A review of practical and conceptual issues in the genetics of restoration</strong></div><div class="host">Restor. Ecol., 13 (2005), pp. 432-440</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib155" id="ref-id-bib155">McLachlan et al., 2005</a></dt><dd class="reference"><div class="contribution">J.S. McLachlan, J.S. Clark, P.S. Manos<strong class="title">Molecular indicators of tree migration capacity under rapid climate change</strong></div><div class="host">Ecology, 86 (2005), pp. 2088-2098</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib156" id="ref-id-bib156">McPeek, 2008</a></dt><dd class="reference"><div class="contribution">M.A. McPeek<strong class="title">The ecological dynamics of clade diversification and community assembly</strong></div><div class="host">Am. Nat., 172 (2008), pp. E270-E284</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib157" id="ref-id-bib157">Mendelson and Shaw, 2005</a></dt><dd class="reference"><div class="contribution">T.C. Mendelson, K.L. Shaw<strong class="title">Rapid speciation in an arthropod: the likely force behind an explosion of new Hawaiian cricket species revealed</strong></div><div class="host">Nature, 433 (2005), pp. 375-376</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib158" id="ref-id-bib158">Meyer and Paulay, 2005</a></dt><dd class="reference"><div class="contribution">C.P. Meyer, G. Paulay<strong class="title">DNA barcoding: error rates based on comprehensive sampling</strong></div><div class="host">PLoS Biol., 3 (2005), p. e422</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib159" id="ref-id-bib159">Milot et al., 2000</a></dt><dd class="reference"><div class="contribution">M. Milot, H.L. Gibbs, K.A. Hobson<strong class="title">Phylogeography and genetic structure of northern populations of the yellow warbler (<em>Dendroica petechia</em>)</strong></div><div class="host">Mol. Ecol., 9 (2000), pp. 667-681</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib160" id="ref-id-bib160">Morando et al., 2003</a></dt><dd class="reference"><div class="contribution">M. Morando, L. Avila, J.W. Sites Jr.<strong class="title">Sampling
 strategies for delimiting species: genes, individuals, and populations 
in the Liolaemus elongatus-kriegi complex (Squamata: Liolaemidae) in 
Andean-Patagonian South America</strong></div><div class="host">Syst. Biol., 52 (2003), pp. 159-185</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib161" id="ref-id-bib161">Moritz, 2002</a></dt><dd class="reference"><div class="contribution">C. Moritz<strong class="title">Strategies to protect biological diversity and the evolutionary processes that sustain it</strong></div><div class="host">Syst. Biol., 51 (2002), pp. 238-254</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib162" id="ref-id-bib162">Moritz and Faith, 1998</a></dt><dd class="reference"><div class="contribution">C. Moritz, D. Faith<strong class="title">Comparative phylogeography and the identification of genetically divergent areas for conservation</strong></div><div class="host">Mol. Ecol., 7 (1998), pp. 419-429</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib163" id="ref-id-bib163">Moritz et al., 2000</a></dt><dd class="reference"><div class="contribution">C. Moritz, J.L. Patton, C.J. Schneider, T.B. Smith<strong class="title">Diversification of rainforest faunas: an integrated molecular approach</strong></div><div class="host">Annu. Rev. Ecol. Syst., 31 (2000), pp. 533-563</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib164" id="ref-id-bib164">Moya et al., 2007</a></dt><dd class="reference"><div class="contribution">O. Moya, H.-G. Contreras-Díaz, P. Oromí, W. Juan<strong class="title">Phylogeography
 of a ground beetle species in La Gomera (Canary Islands): the effects 
of landscape topology and population history</strong></div><div class="host">Heredity, 99 (2007), pp. 322-330</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib165" id="ref-id-bib165">Nakazato et al., 2008</a></dt><dd class="reference"><div class="contribution">T. Nakazato, M. Bogonovich, L.C. Moyle<strong class="title">Environmental factors predict adaptive phenotypic differentiation within and between two wild Andean tomatoes</strong></div><div class="host">Evolution, 62 (2008), pp. 774-792</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib166" id="ref-id-bib166">Nei and Li, 1979</a></dt><dd class="reference"><div class="contribution">M. Nei, W. Li<strong class="title">Mathematical model for studying variation in terms of restriction endonucleases</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 76 (1979), pp. 5269-5273</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib167" id="ref-id-bib167">Nettel and Dodd, 2007</a></dt><dd class="reference"><div class="contribution">A. Nettel, R.S. Dodd<strong class="title">Drifting propagules and receding swamps: genetic footprints of mangrove recolonization and dispersal along tropical coasts</strong></div><div class="host">Evolution, 61 (2007), pp. 958-971</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib168" id="ref-id-bib168">Nielsen and Beaumont, 2009</a></dt><dd class="reference"><div class="contribution">R. Nielsen, M.A. Beaumont<strong class="title">Statistical inferences in phylogeography</strong></div><div class="host">Mol. Ecol., 18 (2009), pp. 1034-1047</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib169" id="ref-id-bib169">Nosil, 2007</a></dt><dd class="reference"><div class="contribution">P. Nosil<strong class="title">Divergent host–plant adaptation and reproductive isolation between ecotypes of <em>Timema cristinae</em> walking-sticks</strong></div><div class="host">Am. Nat., 169 (2007), pp. 151-162</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib170" id="ref-id-bib170">Nosil et al., 2009</a></dt><dd class="reference"><div class="contribution">P. Nosil, L.J. Harmon, O. Seehausen<strong class="title">Ecological explanations for (incomplete) speciation</strong></div><div class="host">Trends Ecol. Evol., 24 (2009), pp. 145-156</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib171" id="ref-id-bib171">Oliver, 2008</a></dt><dd class="reference"><div class="contribution">J.C. Oliver<strong class="title">AUGIST: inferring species trees while accommodating gene tree uncertainty</strong></div><div class="host">Bioinformatics, 24 (2008), pp. 2932-2933</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib172" id="ref-id-bib172">Olsen et al., 2004</a></dt><dd class="reference"><div class="contribution">J.L. Olsen, W.T. Stam, J.A. Coyer, T.B.H. Reusch, M. Billingham, C. Bostrom, E. Calvert, H. Christie, S. Granger, R. La Lumiere, N. Milchakova, M.P. Oudot-Le Secq, G. Procaccini, B. Sanjabi, E. Serrao, J. Veldsink, S. Widdicombe, S. Wyllie-Echeverria<strong class="title">North Atlantic phylogeography and large-scale population differentiation of the seagrass <em>Zostera marina</em> L</strong></div><div class="host">Mol. Ecol., 13 (2004), pp. 1923-1941</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib173" id="ref-id-bib173">Orme et al., 2005</a></dt><dd class="reference"><div class="contribution">C.D.L. Orme, R.G. Davies, M. Burgess, F. Eigenbrod, N. Pickup, V.A. Olson, A.J. Webster, T. Ding, P.C. Rasmussen, R.S. Ridgely, A.J. Stattersfield, P.M. Bennett, T.M. Blackburn, K.J. Gaston, I.P.F. Owens<strong class="title">Global hotspots of species richness are not congruent with endemism or threat</strong></div><div class="host">Nature, 436 (2005), pp. 1016-1019</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib174" id="ref-id-bib174">Orr et al., 2004</a></dt><dd class="reference"><div class="contribution">H.A. Orr, J.P. Masly, D.C. Presgraves<strong class="title">Speciation genes</strong></div><div class="host">Curr. Opin. Genet. Dev., 14 (2004), pp. 675-679</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib175" id="ref-id-bib175">Palopoli and Wu, 1994</a></dt><dd class="reference"><div class="contribution">M. Palopoli, C.-I. Wu<strong class="title">Genetics of hybrid male sterility between <em>Drosophila sibling</em> species: a complex web of epistasis is revealed in interspecific studies</strong></div><div class="host">Genetics, 138 (1994), pp. 329-341</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib176" id="ref-id-bib176">Panchal and Beaumont, 2007</a></dt><dd class="reference"><div class="contribution">M. Panchal, B.A. Beaumont<strong class="title">The automation and evaluation of nested clade phylogeographic analysis</strong></div><div class="host">Evolution, 61 (2007), pp. 1466-1480</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib177" id="ref-id-bib177">Pastorino and Gallo, 2002</a></dt><dd class="reference"><div class="contribution">M.J. Pastorino, L.A. Gallo<strong class="title">Quaternary evolutionary history of <em>Austrocedrus chilensis</em>, a cypress native to the Andean–Patagonian forest</strong></div><div class="host">J. Biogeogr., 29 (2002), pp. 1167-1178</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib178" id="ref-id-bib178">Pavoine and Bailly, 2007</a></dt><dd class="reference"><span>Pavoine,
 S., Bailly, X., 2007. New analysis for consistency among markers in the
 study of genetic diversity: development and application to the 
description of bacterial diversity. BMC Evol. Biol. 7. doi: <a href="https://doi.org/10.1186/1471-2148-1187-1156" target="_blank">10.1186/1471-2148-1187-1156</a>.</span></dd><dt class="label"><a href="#bbib179" id="ref-id-bib179">Peters et al., 2008</a></dt><dd class="reference"><div class="contribution">J.L. Peters, Y.N. Zhuravlev, I. Fefelov, E.M. Humphries, K.E. Omland<strong class="title">Multilocus phylogeography of a Holarctic duck: colonization of North America from Eurasia by gadwall (<em>anas strepera</em>)</strong></div><div class="host">Evolution, 62 (2008), pp. 1469-1483</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib180" id="ref-id-bib180">Peterson and Nyári, 2007</a></dt><dd class="reference"><div class="contribution">A.T. Peterson, S. Nyári<strong class="title">Ecological niche conservatism and Pleistocene refugia in the thrush-like mourner</strong></div><div class="host">Evolution, 62 (2007), pp. 173-183</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib181" id="ref-id-bib181">Peterson et al., 2002</a></dt><dd class="reference"><div class="contribution">A.T. Peterson, J. Soberon, V. Sanchez-Cordero<strong class="title">Distributional prediction based on ecological niche modeling of primary occurrence data</strong></div><div class="host">J.M. Scott, P.J. Heglund, M. Morrison, J.B. Haufler, M.G. Raphael, W.A. Wall, F.B. Samson (Eds.), Predicting Species Occurrences: Issues of Scale and Accuracy, Island Press, Washington, DC (2002), pp. 617-623</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib182" id="ref-id-bib182">Petit and Grivet, 2002</a></dt><dd class="reference"><div class="contribution">R.J. Petit, D. Grivet<strong class="title">Optimal randomization strategies when testing the existence of a phylogeographic structure</strong></div><div class="host">Genetics, 161 (2002), pp. 469-471</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib183" id="ref-id-bib183">Pons et al., 2006</a></dt><dd class="reference"><div class="contribution">J. Pons, T.G. Barraclough, J. Gomez-Zurita, A. Cardoso, D.P. Duran, S. Hazell, S. Kamoun, W.D. Sumlin, A.P. Vogler<strong class="title">Sequence-based species delimitation for the DNA taxonomy of undescribed insects</strong></div><div class="host">Syst. Biol., 55 (2006), pp. 595-609</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib184" id="ref-id-bib184">Premoli, 1997</a></dt><dd class="reference"><div class="contribution">A.C. Premoli<strong class="title">Genetic variation in a geographically restricted and two widespread species of South American Nothofagus</strong></div><div class="host">J. Biogeogr., 24 (1997), pp. 883-892</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib185" id="ref-id-bib185">Premoli et al., 2000</a></dt><dd class="reference"><div class="contribution">A. Premoli, T. Kitzberger, T.T. Veblem<strong class="title">Isozyme variation and recent biogeographical history of the long-lived conifer <em>Fitzroya cupressoides</em></strong></div><div class="host">J. Biogeogr., 27 (2000), pp. 251-260</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib186" id="ref-id-bib186">Pritchard et al., 1999</a></dt><dd class="reference"><div class="contribution">J.K. Pritchard, M.T. Seielstad, A. Perez-Lenzacen, M.W. Feldman<strong class="title">Population growth of human Y chromosomes: a study of Y chromosome microsatellites</strong></div><div class="host">Mol. Biol. Evol., 16 (1999), pp. 1791-1798</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib187" id="ref-id-bib187">Raxworthy et al., 2007</a></dt><dd class="reference"><div class="contribution">C.J. Raxworthy, C. Ingram, N. Rabibisoa, R. Pearson<strong class="title">Applications of ecological niche modeling for species delimitation: a review and empirical evaluation using day geckos (<em>phelsuma</em>) from Madagascar</strong></div><div class="host">Syst. Biol., 56 (2007), pp. 907-923</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib188" id="ref-id-bib188">Richards et al., 2007</a></dt><dd class="reference"><div class="contribution">C.L. Richards, B.C. Carstens, L.L. Knowles<strong class="title">Distribution
 modeling and statistical phylogeography: an integrative framework for 
generating and testing alternative biogeographic hypotheses</strong></div><div class="host">J. Biogeogr., 34 (2007), pp. 1833-1845</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib189" id="ref-id-bib189">Riddle et al., 2000</a></dt><dd class="reference"><div class="contribution">B.R. Riddle, D.J. Hafner, L.F. Alexander, J.R. Jaeger<strong class="title">Cryptic vicariance in the historical assembly of a Baja California Peninsular Desert biota</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 97 (2000), pp. 14438-14443</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib190" id="ref-id-bib190">Riginos, 2005</a></dt><dd class="reference"><div class="contribution">C. Riginos<strong class="title">Cryptic vicariance in Gulf of California fishes parallels vicariant patterns found in Baja, California mammals and reptiles</strong></div><div class="host">Evolution, 59 (2005), pp. 2678-2690</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib191" id="ref-id-bib191">Rissler and Apodaca, 2007</a></dt><dd class="reference"><div class="contribution">L.J. Rissler, J.J. Apodaca<strong class="title">Adding
 more ecology into species delimitation: ecological niche models and 
phylogeography help define cryptic species in the black salamander (<em>Aneides flavipunctatus</em>)</strong></div><div class="host">Syst. Biol., 56 (2007), pp. 924-942</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib192" id="ref-id-bib192">Rogers and Bernatchez, 2007</a></dt><dd class="reference"><div class="contribution">S.M. Rogers, L. Bernatchez<strong class="title">The genetic architecture of ecological speciation and the association with signatures of selection in natural lake whitefish (<em>Coregonus</em> sp. <em>salmonidae</em>) species pairs</strong></div><div class="host">Mol. Biol. Evol., 24 (2007), pp. 1423-1438</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib193" id="ref-id-bib193">Rosen, 1978</a></dt><dd class="reference"><div class="contribution">D.E. Rosen<strong class="title">Vicariant patterns and historical explanation in biogeography</strong></div><div class="host">Syst. Zool., 27 (1978), pp. 159-188</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib194" id="ref-id-bib194">Rosenberg, 2003</a></dt><dd class="reference"><div class="contribution">N.A. Rosenberg<strong class="title">The
 shapes of neutral gene genealogies in two species: probabilities of 
monophyly, paraphyly, and polyphyly in a coalescent model</strong></div><div class="host">Evolution, 57 (2003), pp. 1465-1477</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib195" id="ref-id-bib195">Rosenberg, 2007</a></dt><dd class="reference"><div class="contribution">N.A. Rosenberg<strong class="title">Statistical tests for taxonomic distinctiveness from observations of monophyly</strong></div><div class="host">Evolution, 61 (2007), pp. 317-323</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib196" id="ref-id-bib196">Rosenblum et al., 2007</a></dt><dd class="reference"><div class="contribution">E. Rosenblum, M.J. Hickerson, C. Moritz<strong class="title">A multilocus perspective on colonization accompanied by selection and gene flow</strong></div><div class="host">Evolution, 61 (2007), pp. 2971-2985</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib197" id="ref-id-bib197">Ruegg and Hijmans, 2006</a></dt><dd class="reference"><div class="contribution">K.C. Ruegg, R.J. Hijmans<strong class="title">Climate change and the origin of migratory pathways in the Swainson’s thrush, <em>Catharus ustulatus</em></strong></div><div class="host">J. Biogeogr., 33 (2006), pp. 1172-1182</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib198" id="ref-id-bib198">Rundle and Nosil, 2005</a></dt><dd class="reference"><div class="contribution">H.D. Rundle, P. Nosil<strong class="title">Ecological speciation</strong></div><div class="host">Ecol. Lett., 8 (2005), pp. 336-352</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib199" id="ref-id-bib199">Salem et al., 2005</a></dt><dd class="reference"><div class="contribution">R.M. Salem, J. Wessel, N.J. Schork<strong class="title">A comprehensive literature review of haplotyping software and methods for use with unrelated individuals</strong></div><div class="host">Human Genomics., 2 (2005), pp. 39-66</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib200" id="ref-id-bib200">Schluter, 2000</a></dt><dd class="reference"><div class="contribution">D. Schluter<strong class="title">The Ecology of Adaptive Radiation</strong></div><div class="host">Oxford University Press, Oxford (2000)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib201" id="ref-id-bib201">Schneider et al., 1998</a></dt><dd class="reference"><div class="contribution">C.J. Schneider, M. Cunningham, C. Moritz<strong class="title">Comparative phylogeography and the history of endemic vertebrates in the Wet Tropics rainforests of Australia</strong></div><div class="host">Mol. Ecol., 7 (1998), pp. 487-498</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib202" id="ref-id-bib202">Scluter, 2009</a></dt><dd class="reference"><div class="contribution">D. Scluter<strong class="title">Evidence for ecological speciation and its alternative</strong></div><div class="host">Science, 323 (2009), pp. 737-741</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib203" id="ref-id-bib203">Seehausen et al., 2008</a></dt><dd class="reference"><div class="contribution">O. Seehausen, Y. Terai, I.S. Magalhaes, K.L. Carleton, H.D.J. Mrosso, R. Miyagi, I. van der Sluijs, M.V. Schneider, M.E. Maan, H. Tachida, H. Imai, N. Okada<strong class="title">Speciation through sensory drive in cichlid fish</strong></div><div class="host">Nature, 455 (2008), pp. 620-626</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib204" id="ref-id-bib204">Sites and Marshall, 2004</a></dt><dd class="reference"><div class="contribution">W. Sites, J.C. Marshall<strong class="title">Delimiting species: a renaissance issue in systematic biology</strong></div><div class="host">Trends Ecol. Evol., 18 (2004), pp. 462-470</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib205" id="ref-id-bib205">Slatkin and Maddison, 1989</a></dt><dd class="reference"><div class="contribution">M. Slatkin, W.P. Maddison<strong class="title">A cladistic measure of gene flow inferred from the phylogenies of alleles</strong></div><div class="host">Genetics, 123 (1989), pp. 603-613</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib206" id="ref-id-bib206">Soltis et al., 2006</a></dt><dd class="reference"><div class="contribution">D.E. Soltis, A. Morris, J.S. McLachlan, P.S. Manos, P.S. Soltis<strong class="title">Comparative phylogeography of unglaciated eastern North America</strong></div><div class="host">Mol. Ecol., 15 (2006), pp. 4261-4293</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib207" id="ref-id-bib207">Sousa et al., 2009</a></dt><dd class="reference"><div class="contribution">V.C. Sousa, M. Fritz, M.A. Beaumont, L. Chikhi<strong class="title">Approximate Bayesian computation without summary statistics: the case of admixture</strong></div><div class="host">Genetics, 181 (2009), pp. 1507-1519</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib208" id="ref-id-bib208">Spathelf and Waite, 2007</a></dt><dd class="reference"><div class="contribution">M. Spathelf, T.A. Waite<strong class="title">Will hotspots conserve extra primate and carnivore evolutionary history?</strong></div><div class="host">Divers. Distrib., 13 (2007), pp. 746-751</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib209" id="ref-id-bib209">Spellman and Klicka, 2006</a></dt><dd class="reference"><div class="contribution">G.M. Spellman, J. Klicka<strong class="title">Testing hypotheses of Pleistocene population history using coalescent simulations: phylogeography of the pygmy nuthatch (<em>Sitta pygmaea</em>)</strong></div><div class="host">Proc. R. Soc. B, 22 (2006), pp. 3057-3063</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib210" id="ref-id-bib210">Steele and Storfer, 2006</a></dt><dd class="reference"><div class="contribution">C.A. Steele, A. Storfer<strong class="title">Coalescent-based
 hypothesis testing supports multiple Pleistocene refugia in the Pacific
 Northwest for the Pacific giant salamander (<em>Dicamptodon tenebrosus</em>)</strong></div><div class="host">Mol. Ecol., 15 (2006), pp. 2477-2487</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib211" id="ref-id-bib211">Stinchcombe and Hoekstra, 2008</a></dt><dd class="reference"><div class="contribution">J.R. Stinchcombe, H.E. Hoekstra<strong class="title">Combining population genomics and quantitative genetics: finding the genes underlying ecologically important traits</strong></div><div class="host">Heredity, 100 (2008), pp. 158-170</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib212" id="ref-id-bib212">Stockwell and Peterson, 2002</a></dt><dd class="reference"><div class="contribution">D.R.B. Stockwell, A.T. Peterson<strong class="title">Effects of sample size on accuracy of species distribution models</strong></div><div class="host">Ecol. Model., 148 (2002), pp. 1-13</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib213" id="ref-id-bib213">Stone et al., 2009</a></dt><dd class="reference"><div class="contribution">G.N. Stone, A. Hernandez-Lopez, J.A. Nicholls, E. di Pierro, J. Pujade-Villar, G. Melika, J.M. Cook<strong class="title">Extreme host plant conservatism during at least 20 million years of host plant pursuit by oak gallwasps</strong></div><div class="host">Evolution, 63 (2009), pp. 854-869</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib214" id="ref-id-bib214">Swenson and Howard, 2005</a></dt><dd class="reference"><div class="contribution">N.G. Swenson, D.J. Howard<strong class="title">Clustering of contact zones, hybrid zones, and phylogeographic breaks in North America</strong></div><div class="host">Am. Nat., 166 (2005), pp. 581-591</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib215" id="ref-id-bib215">Taberlet and Cheddadi, 2002</a></dt><dd class="reference"><div class="contribution">P. Taberlet, R. Cheddadi<strong class="title">Quaternary refugia and persistence of biodiversity</strong></div><div class="host">Science, 297 (2002), pp. 2009-2010</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib216" id="ref-id-bib216">Tajima, 1983</a></dt><dd class="reference"><div class="contribution">F. Tajima<strong class="title">Evolutionary relationship of DNA sequences in finite populations</strong></div><div class="host">Genetics, 105 (1983), pp. 437-460</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib217" id="ref-id-bib217">Tajima, 1989</a></dt><dd class="reference"><div class="contribution">F. Tajima<strong class="title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</strong></div><div class="host">Genetics, 123 (1989), pp. 585-595</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib218" id="ref-id-bib218">Templeton, 1998</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Nested clade analyses of phylogeographic data: testing hypotheses about gene flow and population history</strong></div><div class="host">Mol. Ecol., 7 (1998), pp. 381-397</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib219" id="ref-id-bib219">Templeton, 2001</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Using phylogeographic analyses of gene trees to test species status and processes</strong></div><div class="host">Mol. Ecol., 10 (2001), pp. 779-791</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib220" id="ref-id-bib220">Templeton, 2004</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Statistical phylogeography: methods of evaluating and minimizing inference errors</strong></div><div class="host">Mol. Ecol., 13 (2004), pp. 789-810</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib221" id="ref-id-bib221">Templeton, 2009a</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Statistical
 hypothesis testing in intraspecific phylogeography: nested clade 
phylogeographical analysis vs. approximate Bayesian computation</strong></div><div class="host">Mol. Eovl., 18 (2009), pp. 319-331</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib222" id="ref-id-bib222">Templeton, 2009b</a></dt><dd class="reference"><div class="contribution">A.R. Templeton<strong class="title">Why does a method that fails continue to be used? The answer</strong></div><div class="host">Evolution, 63 (2009), pp. 807-812</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib223" id="ref-id-bib223">Thalmann et al., 2007</a></dt><dd class="reference"><div class="contribution">O. Thalmann, A. Fischer, F. Lankester, S. Pääbo, L. Vigilant<strong class="title">The complex evolutionary history of gorillas: insights from genomic data</strong></div><div class="host">Mol. Biol. Evol., 24 (2007), pp. 146-158</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib224" id="ref-id-bib224">Tilman, 2004</a></dt><dd class="reference"><div class="contribution">D. Tilman<strong class="title">Niche
 tradeoffs, neutrality, and community structure: a stochastic theory of 
resource competition, invasion, and community assembly</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 101 (2004), pp. 10854-10861</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib225" id="ref-id-bib225">Topp and Winker, 2008</a></dt><dd class="reference"><div class="contribution">C.M. Topp, K. Winker<strong class="title">Genetic patterns of differentiation among five landbird species on the Queen Charlotte Islands, British Columbia</strong></div><div class="host">Auk, 125 (2008), pp. 461-472</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib226" id="ref-id-bib226">Townsend et al., 2008</a></dt><dd class="reference"><div class="contribution">T.M. Townsend, R.E. Alegre, S.T. Kelley, J.J. Wiens, T.W. Reeder<strong class="title">Rapid development of multiple nuclear loci for phylogenetic analysis using genomic resources: an example from squamate reptiles</strong></div><div class="host">Mol. Phylogen. Evol., 47 (2008), pp. 129-142</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib227" id="ref-id-bib227">Tribsch and Schonswetter, 2003</a></dt><dd class="reference"><div class="contribution">A. Tribsch, P. Schonswetter<strong class="title">Patterns
 of endemism and comparative phylogeography confirm paleoenvironmental 
evidence for Pleistocene refugia in the Eastern Alps</strong></div><div class="host">Taxon, 477 (2003), p. 497</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib228" id="ref-id-bib228">Vamosi et al., 2009</a></dt><dd class="reference"><div class="contribution">S.M. Vamosi, S.B. Heard, J.C. Vamosi, C.O. Webb<strong class="title">Emerging patterns in the comparative analysis of phylogenetic community structure</strong></div><div class="host">Mol. Ecol., 18 (2009), pp. 572-592</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib229" id="ref-id-bib229">Vera et al., 2008</a></dt><dd class="reference"><span>Vera,
 C., Wheat, C.W., Fescemyer, H.W., Frilander, M.J., Crawford, D.L., 
Hanski, I., Marden, J.H., 2008. Rapid transcriptome characterization for
 a non-model organism using massively parallel 454 pyrosequencing. Mol. 
Ecol. 2371.</span></dd><dt class="label"><a href="#bbib230" id="ref-id-bib230">Verdu et al., 2009</a></dt><dd class="reference"><div class="contribution">P. Verdu, F. Austerlitz, A. Estoup, R. Vitalis, M. Georges, S. ThÈry, A. Froment, S. Le Bomin, A. Gessain, J. Hombert, L. Van der Veen, L. Quintana-Murci, S. Bahuchet, E. Heyer<strong class="title">Origins and genetic diversity of pygmy hunter-gatherers from western central Africa</strong></div><div class="host">Curr. Biol., 19 (2009), pp. 312-318</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib231" id="ref-id-bib231">Victoriano et al., 2008</a></dt><dd class="reference"><div class="contribution">P.F. Victoriano, J.C. Ortiz, E. Benavides, B.J. Adams, J.W. Sites Jr.<strong class="title">Comparative
 phylogeography of codistributed species of Chilean Liolaemus (Squamata:
 Tropiduridae) from the central-southern Andean range</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 2397-2416</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib232" id="ref-id-bib232">Vila et al., 2005</a></dt><dd class="reference"><div class="contribution">M. Vila, J.R. Vidal-Romaní, M. Björklund<strong class="title">The importance of time scale and multiple refugia: incipient speciation and admixture of lineages in the butterfly <em>Erebia triaria</em> (Nymphalidae)</strong></div><div class="host">Mol. Phylogen. Evol., 36 (2005), pp. 249-260</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib233" id="ref-id-bib233">Voje et al., 2009</a></dt><dd class="reference"><div class="contribution">K.L. Voje, C. Hemp, Y. Flagstad, G.-P. Saetre, N.C. Stenseth<strong class="title">Climatic change as an engine for speciation in flightless Orthoptera species inhabiting African mountains</strong></div><div class="host">Mol. Ecol., 18 (2009), pp. 93-108</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib234" id="ref-id-bib234">Wakeley, 2004</a></dt><dd class="reference"><div class="contribution">J. Wakeley<strong class="title">Recent trends in population genetics: More data! More math! Simple models?</strong></div><div class="host">J. Hered., 95 (2004), pp. 397-405</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib235" id="ref-id-bib235">Wakeley, 2008</a></dt><dd class="reference"><div class="contribution">J. Wakeley<strong class="title">Coalescent Theory</strong></div><div class="host">Roberts and Company Publishers, Greenwood Village, CO (2008)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib236" id="ref-id-bib236">Waltari et al., 2007</a></dt><dd class="reference"><span>Waltari,
 E., Hijmans, R.J., Peterson, A.T., Nyári, A.S., Perkins, S.L., 
Guralnick, R.P., 2007. Locating pleistocene refugia: comparing 
phylogeographic and ecological niche model predictions. PLoS One 2, 
e563. doi: 10.1371/journal.pone.0000563.</span></dd><dt class="label"><a href="#bbib237" id="ref-id-bib237">Wares and Cunningham, 2001</a></dt><dd class="reference"><div class="contribution">J.P. Wares, C.W. Cunningham<strong class="title">Phylogeography and historical ecology of the North Atlantic intertidal</strong></div><div class="host">Evolution, 55 (2001), pp. 2455-2469</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib238" id="ref-id-bib238">Warren et al., 2008</a></dt><dd class="reference"><div class="contribution">D.L. Warren, R.E. Glor, M. Turelli<strong class="title">Environmental niche equivalency versus conservatism: quantitative approaches to niche evolution</strong></div><div class="host">Evolution, 62 (2008), pp. 2868-2883</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib239" id="ref-id-bib239">Webb et al., 2002</a></dt><dd class="reference"><div class="contribution">C.O. Webb, D.D. Ackerly, M.A. McPeek, M.J. Donoghue<strong class="title">Phylogenies and community ecology</strong></div><div class="host">Annu. Rev. Ecol. Syst., 33 (2002), pp. 475-505</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib240" id="ref-id-bib240">Weiss and Nuno, 2006</a></dt><dd class="reference"><div class="contribution">S. Weiss, F. Nuno<strong class="title">Phylogeography of Southern European Refugia</strong></div><div class="host">Springer, Dordrecht, The Netherlands (2006)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib241" id="ref-id-bib241">Wilkinson, 2001</a></dt><dd class="reference"><div class="contribution">D. Wilkinson<strong class="title">Is local provenance important in habitat creation?</strong></div><div class="host">J. Appl. Ecol., 38 (2001), pp. 1371-1373</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib242" id="ref-id-bib242">Williams et al., 2007</a></dt><dd class="reference"><div class="contribution">J.W. Williams, S.T. Jackson, J.E. Kutzbach<strong class="title">Projected distributions of novel and disappearing climates by 2100 AD</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 104 (2007), pp. 5738-5742</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib243" id="ref-id-bib243">Wilson, 2006</a></dt><dd class="reference"><div class="contribution">A.B. Wilson<strong class="title">Genetic signature of recent glaciation on populations of a near-shore marine fish species (<em>Syngnathus leptorhynchus</em>)</strong></div><div class="host">Mol. Ecol., 15 (2006), pp. 1857-1871</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib244" id="ref-id-bib244">Wood et al., 2008</a></dt><dd class="reference"><div class="contribution">H.M. Wood, J.W. Graham, S. Humphrey, J. Rogers, R.K. Butlin<strong class="title">Sequence differentiation in regions identified by a genome scan for local adaptation</strong></div><div class="host">Mol. Ecol., 17 (2008), pp. 3123-3135</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib245" id="ref-id-bib245">Wright, 1943</a></dt><dd class="reference"><div class="contribution">S. Wright<strong class="title">Isolation by distance</strong></div><div class="host">Genetics, 28 (1943), pp. 114-138</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib246" id="ref-id-bib246">Wright, 1969</a></dt><dd class="reference"><div class="contribution">S. Wright<strong class="title">Evolution and the genetics of populations, vol. 2</strong></div><div class="host">The Theory of Gene Frequencies, University of Chicago Press, Chicago (1969)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib247" id="ref-id-bib247">Wright and Stanton, 2007</a></dt><dd class="reference"><div class="contribution">J. Wright, M. Stanton<strong class="title"><em>Collinsia sparsiflora</em>
 in serpentine and nonserpentine habitats: using F2 hybrids to detect 
the potential role of selection in ecotypic differentiation</strong></div><div class="host">New Phytol., 173 (2007), pp. 354-366</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib248" id="ref-id-bib248">Wu and Davis, 1993</a></dt><dd class="reference"><div class="contribution">C.-I. Wu, A.W. Davis<strong class="title">Evolution of postmating reproductive isolation: the composite nature of Haldane’s rule and its genetic bases</strong></div><div class="host">Am. Nat., 142 (1993), pp. 187-212</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib249" id="ref-id-bib249">Yoder et al., 2005</a></dt><dd class="reference"><div class="contribution">A.D. Yoder, L.E. Olson, C. Hanley, K.L. Heckman, R. Rasoloarison, A.L. Russell, J. Ranivo<strong class="title">A multidimensional approach for detecting species patterns in Malagasy vertebrates</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 102 (2005), pp. 6581-6586</div><div class="ReferenceLinks" id="reference-links"></div></dd></dl></section></section><div id="aep-glossary-id31" class="Glossary"><h2 class="section-title">Glossary</h2><div class="glossary-section" id="aep-glossary-sec-id32"><dl><div><dt><em>Approximate Bayesian computation</em></dt><dd class="u-margin-xxl-left">This
 provides approximations of posterior probability estimates of model 
parameters and/or models based on the closest between summary statistics
 of the observed data and data simulated across the prior distribution 
of model parameter space.</dd></div><div><dt><em>Coalescent theory</em></dt><dd class="u-margin-xxl-left">Probabilistic
 model underlying gene genealogies within a population or populations. 
Coalescent theory provides the statistical framework to build virtually 
any arbitrary complex historical demographic model with parameters such 
as migration rates, population sizes, divergence times, recombination 
and selection.</dd></div><div><dt><em>Community assembly</em></dt><dd class="u-margin-xxl-left">The dynamics and processes underlying community composition via invasion, colonization, dispersal, speciation, and extinction.</dd></div><div><dt><em>Ecological niche models</em></dt><dd class="u-margin-xxl-left">Alternatively
 referred to as bioclimactic modeling or species range distributional 
modeling, ecological niche models (ENMs) predict the occurrence of 
species on a landscape from geo-referenced site locality data and sets 
of spatially-explicit environmental data layers that are assumed to 
correlate with the species’ range (Peterson et al., 2002).</dd></div><div><dt><em>Gene genealogy</em></dt><dd class="u-margin-xxl-left">Inheritance
 relationship between sampled homologous gene copies that can be 
depicted as a branching tree that arises within a population or set of 
populations of individuals. Instead of using estimates of gene 
genealogies to directly infer a species phylogeny or species demographic
 history, coalescent model-based methods use gene genealogies as a 
transition variable for testing models and making model parameter 
estimates from data (Hey and Machado, 2003).</dd></div><div><dt><em>Hierarchical Bayesian model</em></dt><dd class="u-margin-xxl-left">A
 model in which a set of parameters are drawn from a prior distribution 
that is itself conditional on and drawn from a hyper-prior distribution.</dd></div><div><dt><em>Model/hypothesis testing</em></dt><dd class="u-margin-xxl-left">Decision
 theoretical approach to evaluate the fit of the data to a model. In 
Bayesian statistics the fit of the data to a model is evaluated relative
 to other models. In frequentist statistics, a decision is made about 
the null hypothesis; that is, if the null hypothesis is true, what is 
the probability of observing data that is more extreme than the observed
 data.</dd></div><div><dt><em>Parameter estimation</em></dt><dd class="u-margin-xxl-left">A
 method for approximating unknowable quantities (i.e., parameters) with 
data under a probabilistic model. In Bayesian statistics, the 
probabilistic model describes the prior distribution of the parameters. 
Bayesian posterior distribution estimates include the most likely values
 of parameters as well as the level of statistical uncertainty under the
 model. In population genetics and phylogeography, unobserved parameters
 often include effective population size, mutation rates, migration, 
recombination and magnitude of population size change.</dd></div><div><dt><em>Phylogeographic model</em></dt><dd class="u-margin-xxl-left">A
 probabilistic model that explicitly describes how demographic 
parameters affect DNA polymorphism data. Models are approximations of 
actual demographic history, but one must explicitly specify the 
connection between demography and genetic data through a model or set of
 models, in order to estimate parameters or statistically test 
alternative models differing in their parameters (i.e., hypothesis 
testing). Although any model will never be absolutely “true”, useful 
models contain the essential features of the demographic histories that 
are of interest.</dd></div><div><dt><em>Summary statistic</em></dt><dd class="u-margin-xxl-left">A
 mathematical function that condenses the observed data into a numerical
 metric. In population genetics and phylogeography, summary statistics 
quantify patterns in DNA polymorphism within and between populations and
 are most useful if they convey information about and correlate with 
unobserved parameters such as effective population size, mutation rates,
 migration rates, recombination rates and magnitude of population size 
change.</dd></div></dl></div></div><div class="Copyright"><span class="copyright-line">Copyright © 2009 Elsevier Inc. All rights reserved.</span></div></article><div class="u-show-from-md col-lg-6 col-md-8 pad-right"><aside class="RelatedContent" aria-label="Related content"><section class="SidePanel"><header id="recommended-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Recommended articles</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790313002467"><h3 class="article-title ellipsis" id="recommended-articles-article0-title" title="Integrating phylogenetics, phylogeography and population genetics through genomes and evolutionary theory"><span>Integrating phylogenetics, phylogeography and population genetics through genomes and evolutionary theory</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 69, Issue 3, 2013, pp. 1172-1185</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790313002467/pdfft?md5=98648c2b1611fc8e705b038d8790b058&amp;pid=1-s2.0-S1055790313002467-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article0-title" aria-controls="recommended-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S0169534715001093"><h3 class="article-title ellipsis" id="recommended-articles-article1-title" title="Post-molecular systematics and the future of phylogenetics"><span>Post-molecular systematics and the future of phylogenetics</span></h3></a><div class="article-source ellipsis"><div class="source">Trends in Ecology &amp; Evolution, Volume 30, Issue 7, 2015, pp. 384-389</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S0169534715001093/pdfft?md5=e123eb4dbc54dd4569f081632cb251cd&amp;pid=1-s2.0-S0169534715001093-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article1-title" aria-controls="recommended-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317304086"><h3 class="article-title ellipsis" id="recommended-articles-article2-title" title="The Near East as a cradle of biodiversity: A phylogeography of banded newts (genus Ommatotriton) reveals extensive inter- and intraspecific genetic differentiation"><span>The Near East as a cradle of biodiversity: A phylogeography of banded newts (genus <em>Ommatotriton</em>) reveals extensive inter- and intraspecific genetic differentiation</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 114, 2017, pp. 73-81</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317304086/pdfft?md5=81ed1177c9d957977bae4441e979a3bb&amp;pid=1-s2.0-S1055790317304086-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article2-title" aria-controls="recommended-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article2" aria-hidden="true"></div></li></ul></div><div class="pagination u-position-relative u-padding-xs-bottom"><span class="u-position-absolute"></span><span class="pagination-pages-label"><span class="pagination-nav u-margin-xs-hor pagination-current underline-page-number">1</span><span class="pagination-nav u-margin-xs-hor">2</span></span><span class="u-position-absolute"><button class="button-link button-link-secondary next-button" type="button"><span class="button-link-text">Next</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></button></span></div></div></section><section class="SidePanel"><header id="citing-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle button-link-primary" aria-expanded="false" type="button"><span class="button-link-text"><h2 class="section-title">Citing articles (297)</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="u-display-none" aria-hidden="true" aria-describedby="citing-articles-header"><div id="citing-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317307108" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article0-title" title="Synchronous diversification of parachuting frogs (Genus Rhacophorus) on Sumatra and Java">Synchronous diversification of parachuting frogs (Genus Rhacophorus) on Sumatra and Java</h3></a><div class="article-source ellipsis">2018, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317307108/pdfft?md5=5ed382600cf9f23014c9047f3044fa14&amp;pid=1-s2.0-S1055790317307108-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article0-title" aria-controls="citing-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317306036" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article1-title" title="Resolving taxonomic turbulence and uncovering cryptic diversity in the musk turtles (Sternotherus) using robust demographic modeling">Resolving
 taxonomic turbulence and uncovering cryptic diversity in the musk 
turtles (Sternotherus) using robust demographic modeling</h3></a><div class="article-source ellipsis">2018, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317306036/pdfft?md5=d5a27f5132d6a6e0d090ee37a3be187b&amp;pid=1-s2.0-S1055790317306036-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article1-title" aria-controls="citing-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S0167577X17302732" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article2-title" title="Allopatric divergence, demographic history, and conservation implications of an endangered conifer Cupressus chengiana in the eastern Qinghai-Tibet Plateau">Allopatric
 divergence, demographic history, and conservation implications of an 
endangered conifer Cupressus chengiana in the eastern Qinghai-Tibet 
Plateau</h3></a><div class="article-source ellipsis">2017, Tree Genetics and Genomes</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S0167577X17302732/pdfft?md5=549a3ee198877b1e5c5b6ba20a42affd&amp;pid=1-s2.0-S0167577X17302732-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article2-title" aria-controls="citing-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article2" aria-hidden="true"></div></li></ul><a class="anchor side-panel-view-more" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-70749117928&amp;md5=d8d2a433a44bde159c4c4d5da0e3c2a5" target="_blank"><span class="anchor-text">View more articles</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></div></div></section><section class="SidePanel hidden"><header id="metrics-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Article Metrics</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><a href="https://plu.mx/plum/a/?doi=10.1016/j.ympev.2009.09.016" class="plumx-summary plum-sciencedirect-theme" data-pass-hidden-categories="true" data-hide-usage="true" data-orientation="vertical" data-hide-print="true" data-site="plum" data-on-success="onMetricsWidgetSuccess">View article metrics</a></div></section></aside></div></div><div></div></div><div id="footer"><div class="hor-line" style="border-color:#e9711c"></div><div class="panel-s u-padding-l-bottom u-bg-white u-clr-grey7" role="contentinfo"><a class="anchor move-left els-footer-elsevier anchor-has-inherit-color" href="https://www.elsevier.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg viewBox="-3345 3440.027 140.01 24.333" style="width:104px;height:30px"><title>Elsevier</title><path id="E" style="fill:#E9711C" d="M-3343.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3344L-3343.999,3461.698"></path><path style="fill:#E9711C" d="M-3325.448,3461.698c2.074-0.118,2.83-0.502,2.83-2.832v-13.511c0-2.33-0.757-2.715-2.83-2.832v-0.591h8.884 v0.591c-2.243,0-3.027,0.472-3.027,2.891v13.009c0,1.652,0.056,2.625,1.71,2.625h4.008c3,0,4.4-2,5.576-4.1l0.673,0.118 l-1.71,5.222h-16.114V3461.698"></path><path style="fill:#E9711C" d="M-3307.122,3456.27h0.561c1.12,2.596,2.886,5.4,5.94,5.4c2,0,3.672-1.3,3.672-3.334 c0-1.652-1.626-3.1-4.176-4.927c-3.28-2.36-5.41-3.746-5.41-6.43c0-3.422,2.55-5.517,5.633-5.517c2.214,0,3,0.944,4.204,0.944 c0.476,0,0.56-0.266,0.476-0.737h0.561l0.645,6.076h-0.561c-0.785-2.625-2.523-5.19-5.354-5.19c-1.737,0-3.138,1.356-3.138,3.185 c0,1.918,1.933,3.157,5.016,5.016c2.523,1.504,5,3.362,5,6.254c0,3.245-2.608,5.752-5.97,5.752c-2.02,0-4.148-0.855-4.68-0.855 c-0.336,0-0.7,0.207-0.756,0.649h-0.56l-1.094-6.282"></path><path style="fill:#E9711C" d="M-3293.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3294L-3293.999,3461.698"></path><path style="fill:#E9711C" d="M-3265.839,3462.524h-0.42l-5.41-12.538c-0.896-2.065-1.71-4.16-2.83-6.136 c-0.478-0.826-1.346-1.327-2.3-1.327v-0.591h8.323v0.591c-0.785,0-2.354,0-2.354,1.15c0,0.384,0.87,2.45,1.653,4.308l4.063,9.676 l4.877-11.359c0.588-1.356,0.757-2.094,0.757-2.713c0-0.618-0.673-0.974-2.13-1.062v-0.59h5.941v0.591 c-0.337,0.06-0.7,0.117-1.037,0.295c-1.066,0.56-2.13,3.57-2.635,4.749l-6.5,14.957"></path><path style="fill:#E9711C" d="M-3255.472,3461.698c2.24,0,3.025-0.473,3.025-2.892v-13.393c0-2.42-0.784-2.89-3.025-2.891v-0.59h9.078v0.591 c-2.24,0-3.025,0.472-3.025,2.891v13.393c0,2.42,0.784,2.892,3.025,2.892v0.59h-9.08v-0.59"></path><path id="E_2_" style="fill:#E9711C" d="M-3244.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3245L-3244.999,3461.698"></path><path style="fill:#E9711C" d="M-3206,3461.698c-1.26-0.354-1.71-0.68-2.466-1.623l-6.166-7.609c3.027-0.65,5.13-2.185,5.13-5.547 c0-4.75-4.26-4.986-7.764-4.986h-9.191v0.591c2.24,0,3.026,0.472,3.026,2.891v13.393c0,2.42-0.785,2.892-3.026,2.892v0.59h9.08 v-0.59c-2.242,0-3.027-0.473-3.027-2.892v-5.604h2.551l7.314,9.086h4.54L-3206,3461.698 M-3220.399,3444.499 c0-1.387,0.337-1.476,2.186-1.476c2.774,0,5.3,0.974,5.3,4.308c0,3.6-2.887,4.63-5.914,4.631h-1.569v-7.463H-3220.399z"></path></svg></span></a><div class="panel-s u-bg-white u-padding-0-hor-from-xs u-padding-s-hor-from-md u-padding-xs-ver text-xs u-clear-both-from-xs u-clear-none-from-md"><p class="u-margin-xs-bottom"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/solutions/sciencedirect" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">About ScienceDirect</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/customer/authenticate/manra" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Remote access</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science?_ob=ShoppingCartURL&amp;_method=display&amp;md5=3ff44acb300f01481824c54a2973d019" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Shopping cart</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Contact and support</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Terms and conditions</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/privacy-policy" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Privacy policy</span></a></p><p id="els-footer-cookie-message">Cookies are used by this site. For more information, visit the<!-- --> <a class="anchor u-margin-0-right" href="https://www.elsevier.com/solutions/sciencedirect/support/cookies" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">cookies page</span></a>.</p><p id="els-footer-copyright">Copyright © <!-- -->2018<!-- --> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</p></div><div><span style="margin-top:-10px" class="u-position-relative move-bottom u-float-left-from-xs u-float-right-from-md move-right u-padding-0-hor"><a class="anchor els-footer-relx anchor-has-inherit-color" href="https://www.relx.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg style="width:118px;height:28px" viewBox="-3248 3424.938 334.44 53.422"><title>RELX Group</title><path style="fill:#E9711C" d="M-3227.285,3448.329c10.948,0,24.15-2.576,24.15-13.124c0-7.474-7.63-10.267-14.6-10.267 c-10.647,0-30.265,5.926-30.265,26.96c0,10.555,8.034,16.643,20.778,16.643c13.474,0,23.266-8.768,26.26-20.535 c-7.912,13.648-17.972,17.896-26.18,17.896c-10.19,0-14.077-7.568-14.077-14.05c0-18.56,14.406-24.798,23.494-24.798 c6.135,0,9.727,3.936,9.727,8.253c0,11.918-17.17,11.578-25.008,11.578c-0.803,0-3.274,0.02-3.76-0.003l-1.654,1.714 c3.59,0.508,8.554,1.497,13.944,4.223c11.204,5.656,21.745,20.59,35.7,20.59c5.062,0,6.38-1.543,7.335-2.893 c-6.834,4.354-18.286-3.197-24.46-9.04c-5.082-4.806-9.57-10.26-21.387-13.146"></path><g><path style="fill:#666666" d="M-3158.73,3468.502c-0.423,0-0.636-0.164-0.794-0.547l-6.885-14.511h-6.197c-0.21,0-0.316,0.11-0.316,0.33 v14.182c0,0.328-0.213,0.547-0.532,0.547h-5.189c-0.32,0-0.53-0.22-0.53-0.547v-36.139c0-0.33,0.21-0.55,0.53-0.549h13.928 c6.62,0,11.492,4.545,11.492,11.117c0,4.873-2.7,8.65-6.833,10.238l7.575,15.167c0.21,0.385,0,0.713-0.37,0.713L-3158.73,3468.502 L-3158.73,3468.502z M-3159.42,3442.384c0-3.232-2.223-5.31-5.506-5.311h-7.68c-0.21,0-0.316,0.108-0.316,0.328v9.912 c0,0.217,0.104,0.328,0.316,0.328h7.68C-3161.644,3447.641-3159.42,3445.561-3159.42,3442.384"></path><path style="fill:#666666" d="M-3146.95,3431.817c0-0.33,0.21-0.55,0.53-0.549h21.865c0.317,0,0.53,0.22,0.53,0.549v4.709 c0,0.33-0.212,0.548-0.53,0.548h-15.827c-0.212,0-0.318,0.108-0.318,0.328v9.089c0,0.22,0.106,0.328,0.318,0.328h12.726 c0.317,0,0.53,0.22,0.53,0.547v4.709c0,0.33-0.213,0.548-0.53,0.548h-12.726c-0.212,0-0.318,0.11-0.318,0.329v9.418 c0,0.22,0.106,0.328,0.318,0.328h15.827c0.317,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.212,0.547-0.53,0.547h-21.865 c-0.32,0-0.53-0.22-0.53-0.547C-3146.95,3467.956-3146.95,3431.817-3146.95,3431.817z"></path><path style="fill:#666666" d="M-3118.2,3431.817c0-0.33,0.212-0.55,0.53-0.549h5.191c0.317,0,0.53,0.22,0.53,0.549v30.553 c0,0.22,0.106,0.328,0.317,0.328h14.722c0.318,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.213,0.547-0.53,0.547h-20.76 c-0.317,0-0.53-0.22-0.53-0.547C-3118.2,3467.956-3118.2,3431.817-3118.2,3431.817z"></path><path style="fill:#666666" d="M-3052.46,3449.885c0-6.46,0.423-8.926,1.006-10.787c1.747-5.53,5.878-8.377,11.758-8.377 c5.88,0,9.532,3.012,11.28,6.9c0.107,0.274,0.107,0.548-0.21,0.712l-1.695,0.876c-0.265,0.11-0.53,0.055-0.688-0.22 c-1.802-3.396-4.45-5.312-8.74-5.312c-4.45,0-7.468,2.192-8.792,6.3c-0.475,1.422-0.845,3.722-0.845,9.909s0.37,8.486,0.845,9.91 c1.324,4.107,4.343,6.3,8.792,6.3c4.342,0,7.468-2.137,8.79-6.19c0.477-1.422,0.85-3.56,0.85-7.117c0-0.22-0.106-0.328-0.318-0.328 h-8.155c-0.318,0-0.53-0.22-0.53-0.548v-1.807c0-0.33,0.212-0.548,0.53-0.548h10.962c0.317,0,0.53,0.22,0.53,0.548v2.243 c0,3.725-0.423,6.572-0.954,8.27c-1.694,5.53-5.88,8.432-11.65,8.432c-5.88,0-10.01-2.848-11.758-8.38 C-3052.036,3458.811-3052.459,3456.345-3052.46,3449.885"></path><path style="fill:#666666" d="M-3021.43,3468.502c-0.317,0-0.53-0.22-0.53-0.547v-25.023c0-0.33,0.213-0.548,0.53-0.548h1.801 c0.316,0,0.53,0.22,0.53,0.548v3.338h0.052c0.953-2.572,3.23-4.434,6.78-4.434c2.013,0,3.866,0.712,5.084,1.807 c0.265,0.164,0.317,0.438,0.107,0.713l-1.06,1.531c-0.212,0.274-0.48,0.274-0.795,0.11c-1.164-0.767-2.436-1.313-4.024-1.313 c-4.45,0-6.144,3.996-6.144,8.815v14.456c0,0.328-0.213,0.547-0.53,0.547H-3021.43L-3021.43,3468.502z"></path><path style="fill:#666666" d="M-3004.54,3462.26c-0.53-1.752-0.847-3.668-0.847-6.844c0-3.12,0.316-5.037,0.847-6.79 c1.378-4.327,4.818-6.79,9.426-6.79c4.662,0,8.104,2.464,9.48,6.79c0.53,1.752,0.848,3.668,0.848,6.79 c0,3.176-0.317,5.092-0.848,6.844c-1.377,4.326-4.818,6.79-9.48,6.79C-2999.722,3469.05-3003.162,3466.585-3004.54,3462.26 M-2988.387,3461.33c0.475-1.48,0.688-3.066,0.688-5.914c0-2.794-0.213-4.38-0.688-5.86c-1.007-3.12-3.442-4.873-6.728-4.873 c-3.23,0-5.666,1.752-6.672,4.873c-0.477,1.48-0.688,3.065-0.688,5.86c0,2.848,0.212,4.436,0.688,5.914 c1.006,3.12,3.442,4.873,6.672,4.873C-2991.829,3466.203-2989.395,3464.451-2988.387,3461.33"></path><path style="fill:#666666" d="M-2963.66,3468.502c-0.317,0-0.528-0.22-0.528-0.547v-3.066h-0.053c-1.272,2.574-3.92,4.162-7.36,4.162 c-5.615,0-8.74-3.616-8.74-9.913v-16.206c0-0.33,0.213-0.548,0.53-0.548h1.801c0.317,0,0.53,0.22,0.53,0.548v15.44 c0,5.257,2.118,7.83,6.515,7.83c3.812,0,6.78-2.74,6.78-7.284v-15.987c0-0.33,0.21-0.548,0.528-0.548h1.802 c0.317,0,0.53,0.22,0.53,0.548v25.023c0,0.328-0.213,0.547-0.53,0.547h-1.802L-2963.66,3468.502z"></path><path style="fill:#666666" d="M-2955.29,3478.36c-0.317,0-0.53-0.22-0.53-0.549v-34.879c0-0.33,0.213-0.548,0.53-0.548h1.8 c0.317,0,0.53,0.22,0.53,0.548v3.176h0.053c1.218-2.465,3.496-4.272,7.68-4.272c4.45,0,7.256,2.08,8.58,6.242 c0.638,2.137,0.9,4.435,0.9,7.393c0,2.9-0.264,5.2-0.9,7.336c-1.323,4.164-4.13,6.242-8.58,6.242c-4.183,0-6.46-1.807-7.68-4.27 h-0.053v13.031c0,0.33-0.213,0.55-0.53,0.549L-2955.29,3478.36L-2955.29,3478.36z M-2939.35,3461.602 c0.53-1.643,0.688-3.832,0.688-6.13c0-2.355-0.158-4.545-0.688-6.188c-0.952-2.957-3.124-4.6-6.46-4.6 c-3.178,0-5.507,1.533-6.46,4.6c-0.477,1.423-0.69,3.34-0.69,6.188c0,2.847,0.212,4.71,0.69,6.13 c0.954,3.067,3.283,4.602,6.46,4.602C-2942.473,3466.204-2940.3,3464.561-2939.35,3461.602"></path><path style="fill:#666666" d="M-3070.48,3468.545c-0.424,0-0.634-0.164-0.85-0.547l-7.254-12.649h-0.106l-7.307,12.649 c-0.213,0.383-0.424,0.547-0.85,0.547h-5.72c-0.37,0-0.528-0.328-0.318-0.711l10.76-18.562l-9.96-17.248 c-0.21-0.385-0.05-0.712,0.32-0.712h5.721c0.424,0,0.636,0.163,0.847,0.548l6.513,11.278h0.106l6.513-11.278 c0.213-0.385,0.425-0.548,0.85-0.548h5.72c0.37,0,0.53,0.327,0.317,0.712l-9.956,17.248l10.75,18.562 c0.21,0.383,0.052,0.71-0.32,0.711h-5.771H-3070.48z"></path><path style="fill:#666666" d="M-2932.08,3443.951c-0.108,0-0.18-0.075-0.18-0.189v-10.385c0-0.075-0.036-0.113-0.11-0.113h-3.202 c-0.11,0-0.184-0.073-0.184-0.186v-1.6c0-0.11,0.074-0.186,0.184-0.186h8.746c0.11,0,0.183,0.075,0.183,0.186v1.6 c0,0.113-0.07,0.186-0.183,0.186h-3.201c-0.073,0-0.11,0.038-0.11,0.113v10.385c0,0.113-0.07,0.19-0.18,0.189H-2932.08 L-2932.08,3443.951z"></path><path style="fill:#666666" d="M-2924.47,3431.478c0-0.112,0.07-0.186,0.178-0.186h1.604c0.163,0,0.252,0.056,0.307,0.186l3.367,7.8h0.07 l3.315-7.8c0.052-0.13,0.144-0.186,0.305-0.186h1.583c0.11,0,0.182,0.074,0.182,0.186v12.286c0,0.11-0.07,0.186-0.182,0.186h-1.564 c-0.11,0-0.18-0.075-0.18-0.186v-7.744h-0.073l-2.593,5.957c-0.07,0.168-0.18,0.242-0.343,0.242h-1.044 c-0.16,0-0.27-0.074-0.343-0.242l-2.59-5.957h-0.072v7.744c0,0.11-0.072,0.186-0.18,0.186h-1.567c-0.106,0-0.178-0.075-0.178-0.186 v-12.286H-2924.47z"></path></g></svg></span></a></span></div></div></div></section></div></div></div>
<script type="application/json" data-iso-key="_0">{"userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36 Zotero/5.0","abstracts":{"content":[{"#name":"abstract","$":{"xmlns:ce":true,"class":"author","lang":"en","id":"aep-abstract-id10"},"$$":[{"#name":"section-title","_":"Abstract"},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id11"},"$$":[{"#name":"simple-para","$":{"view":"all"},"_":"Approximately 20 years ago, Avise and colleagues proposed the integration of phylogenetics and population genetics for investigating the connection between micro- and macroevolutionary phenomena. The new field was termed phylogeography. Since the naming of the field, the statistical rigor of phylogeography has increased, in large part due to concurrent advances in coalescent theory which enabled model-based parameter estimation and hypothesis testing. The next phase will involve phylogeography increasingly becoming the integrative and comparative multi-taxon endeavor that it was originally conceived to be. This exciting convergence will likely involve combining spatially-explicit multiple taxon coalescent models, genomic studies of natural selection, ecological niche modeling, studies of ecological speciation, community assembly and functional trait evolution. This ambitious synthesis will allow us to determine the causal links between geography, climate change, ecological interactions and the evolution and composition of taxa across whole communities and assemblages. Although such integration presents analytical and computational challenges that will only be intensified by the growth of genomic data in non-model taxa, the rapid development of “likelihood-free” approximate Bayesian methods should permit parameter estimation and hypotheses testing using complex evolutionary demographic models and genomic phylogeographic data. We first review the conceptual beginnings of phylogeography and its accomplishments and then illustrate how it evolved into a statistically rigorous enterprise with the concurrent rise of coalescent theory. Subsequently, we discuss ways in which model-based phylogeography can interface with various subfields to become one of the most integrative fields in all of ecology and evolutionary biology."}]}]}],"floats":[],"footnotes":[],"attachments":[]},"biographies":{},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:ja":true,"xmlns":true,"class":"keyword","id":"aep-keywords-id12"},"$$":[{"#name":"section-title","_":"Keywords"},{"#name":"keyword","$$":[{"#name":"text","_":"Comparative phylogeography"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Statistical phylogeography"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Ecological niche modeling (ENM)"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Approximate Bayesian computation (ABC)"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Coalescent theory"}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"experiments":{},"rawtext":"","authors":{"content":[{"#name":"author-group","$":{"xmlns:ce":true},"$$":[{"#name":"author","$":{"id":"aep-author-id1"},"$$":[{"#name":"given-name","_":"M.J."},{"#name":"surname","_":"Hickerson"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"type":"email"},"_":"michael.hickerson@qc.cuny.edu"}]},{"#name":"author","$":{"id":"aep-author-id2"},"$$":[{"#name":"given-name","_":"B.C."},{"#name":"surname","_":"Carstens"},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"e-address","$":{"type":"email"},"_":"carstens@lsu.edu"}]},{"#name":"author","$":{"id":"aep-author-id3"},"$$":[{"#name":"given-name","_":"J."},{"#name":"surname","_":"Cavender-Bares"},{"#name":"cross-ref","$":{"refid":"aff3"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]},{"#name":"e-address","$":{"type":"email"},"_":"cavender@umn.edu"}]},{"#name":"author","$":{"id":"aep-author-id4"},"$$":[{"#name":"given-name","_":"K.A."},{"#name":"surname","_":"Crandall"},{"#name":"cross-ref","$":{"refid":"aff4"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"e-address","$":{"type":"email"},"_":"keith.crandall@byu.edu"}]},{"#name":"author","$":{"id":"aep-author-id5"},"$$":[{"#name":"given-name","_":"C.H."},{"#name":"surname","_":"Graham"},{"#name":"cross-ref","$":{"refid":"aff5"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"e"}]}]},{"#name":"author","$":{"id":"aep-author-id6"},"$$":[{"#name":"given-name","_":"J.B."},{"#name":"surname","_":"Johnson"},{"#name":"cross-ref","$":{"refid":"aff4"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]},{"#name":"e-address","$":{"type":"email"},"_":"jerry.johnson@byu.edu"}]},{"#name":"author","$":{"id":"aep-author-id7"},"$$":[{"#name":"given-name","_":"L."},{"#name":"surname","_":"Rissler"},{"#name":"cross-ref","$":{"refid":"aff6"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"f"}]},{"#name":"e-address","$":{"type":"email"},"_":"Rissler@bama.ua.edu"}]},{"#name":"author","$":{"id":"aep-author-id8"},"$$":[{"#name":"given-name","_":"P.F."},{"#name":"surname","_":"Victoriano"},{"#name":"cross-ref","$":{"refid":"aff7"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"g"}]},{"#name":"e-address","$":{"type":"email"},"_":"pvictori@udec.cl"}]},{"#name":"author","$":{"id":"aep-author-id9"},"$$":[{"#name":"given-name","_":"A.D."},{"#name":"surname","_":"Yoder"},{"#name":"cross-ref","$":{"refid":"aff8"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"h"}]},{"#name":"e-address","$":{"type":"email"},"_":"anne.yoder@duke.edu"}]},{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Biology Department, Queens College, City University of New York, 65-30 Kissena Blvd., Flushing, NY 11367, USA"}]},{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Department of Biology, 202 Life Sciences Building, Louisiana State University, Baton Rouge, LA 70808, USA"}]},{"#name":"affiliation","$":{"id":"aff3"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Department of Ecology, Evolution and Behavior, University of Minnesota, Saint Paul, MN 55108, USA"}]},{"#name":"affiliation","$":{"id":"aff4"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Department of Biology, Brigham Young University, Provo, UT 84602, USA"}]},{"#name":"affiliation","$":{"id":"aff5"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","_":"Department of Ecology and Evolution, 650 Life Sciences Building, Stony Brook University, Stony Brook, NY 11789, USA"}]},{"#name":"affiliation","$":{"id":"aff6"},"$$":[{"#name":"label","_":"f"},{"#name":"textfn","_":"Department of Biological Sciences, Box 870345 MHB Hall, University of Alabama, Tuscaloosa, AL 35487, USA"}]},{"#name":"affiliation","$":{"id":"aff7"},"$$":[{"#name":"label","_":"g"},{"#name":"textfn","_":"Departamento de Zoología, Facultad de Ciencias Naturales y Oceanográficas, C.I.E.P. Universidad de Concepción, Casilla 160-C, Concepción, Chile"}]},{"#name":"affiliation","$":{"id":"aff8"},"$$":[{"#name":"label","_":"h"},{"#name":"textfn","_":"Department of Biology, Duke University, Durham, NC 27708, USA"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author. Fax: +1 718 997 3445."}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Biology Department, Queens College, City University of New York, 65-30 Kissena Blvd., Flushing, NY 11367, USA"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Department of Biology, 202 Life Sciences Building, Louisiana State University, Baton Rouge, LA 70808, USA"}]},"aff3":{"#name":"affiliation","$":{"id":"aff3"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Department of Ecology, Evolution and Behavior, University of Minnesota, Saint Paul, MN 55108, USA"}]},"aff4":{"#name":"affiliation","$":{"id":"aff4"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Department of Biology, Brigham Young University, Provo, UT 84602, USA"}]},"aff5":{"#name":"affiliation","$":{"id":"aff5"},"$$":[{"#name":"label","_":"e"},{"#name":"textfn","_":"Department of Ecology and Evolution, 650 Life Sciences Building, Stony Brook University, Stony Brook, NY 11789, USA"}]},"aff6":{"#name":"affiliation","$":{"id":"aff6"},"$$":[{"#name":"label","_":"f"},{"#name":"textfn","_":"Department of Biological Sciences, Box 870345 MHB Hall, University of Alabama, Tuscaloosa, AL 35487, USA"}]},"aff7":{"#name":"affiliation","$":{"id":"aff7"},"$$":[{"#name":"label","_":"g"},{"#name":"textfn","_":"Departamento de Zoología, Facultad de Ciencias Naturales y Oceanográficas, C.I.E.P. Universidad de Concepción, Casilla 160-C, Concepción, Chile"}]},"aff8":{"#name":"affiliation","$":{"id":"aff8"},"$$":[{"#name":"label","_":"h"},{"#name":"textfn","_":"Department of Biology, Duke University, Durham, NC 27708, USA"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author. Fax: +1 718 997 3445."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"body":{},"exam":{},"article":{"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":121,"openArchiveStatus":false,"openArchiveArticleCount":0,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1055790309X00126-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790309X00126/cover/DOWNSAMPLED200/image/gif/f2e8026b155a6438531f88702683a4d9/cov200h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790309X00126/cover/DOWNSAMPLED200/image/gif/f2e8026b155a6438531f88702683a4d9/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"11383","pixel-height":"200","pixel-width":"151"},{"attachment-eid":"1-s2.0-S1055790309X00126-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790309X00126/cover/DOWNSAMPLED/image/gif/b6bccd1066d8aa9d062604f63ba792c2/cov150h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790309X00126/cover/DOWNSAMPLED/image/gif/b6bccd1066d8aa9d062604f63ba792c2/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"7470","pixel-height":"150","pixel-width":"113"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S10557903.gif","sourceOpenAccess":false,"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1055790309X00126-cov150h.gif"},"pii":"S105579030900373X","dates":{"Available online":"13 September 2009","Received":"17 January 2009","Revised":["28 August 2009"],"Accepted":"9 September 2009","Publication date":"1 January 2010"},"access":{},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"analyticsMetadata":{"accountId":"262591","accountName":"International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"12289512"},"cid":"272322","content-family":"serial","copyright-line":"Copyright © 2009 Elsevier Inc. All rights reserved.","cover-date-years":["2010"],"cover-date-start":"2010-01-01","cover-date-text":"January 2010","document-subtype":"rev","document-type":"article","entitledToken":"48D19A91DAF120766FE4E18FF2ADB40C298723A8D2F4D49782D62B550F684E6B3751FE97F10E0A8B","eid":"1-s2.0-S105579030900373X","doi":"10.1016/j.ympev.2009.09.016","first-fp":"291","hub-eid":"1-s2.0-S1055790309X00126","issuePii":"S1055790309X00126","iss-first":"1","item-weight":"FULL-TEXT","language":"en","last-lp":"301","last-author":{"#name":"last-author","$":{"xmlns:dm":true},"$$":[{"#name":"author","$":{"xmlns:ce":true,"id":"aep-author-id9"},"$$":[{"#name":"given-name","_":"A.D."},{"#name":"surname","_":"Yoder"},{"#name":"cross-ref","$":{"refid":"aff8"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"h"}]},{"#name":"e-address","$":{"type":"email"},"_":"anne.yoder@duke.edu"}]}]},"normalized-first-auth-initial":"M","normalized-first-auth-surname":"HICKERSON","pages":[{"last-page":"301","first-page":"291"}],"srctitle":"Molecular Phylogenetics and Evolution","timestamp":"2015-05-15T04:50:19.793049-04:00","title":{"content":[{"#name":"dochead","$":{"xmlns:ce":true},"$$":[{"#name":"textfn","_":"Review"}]},{"#name":"title","$":{"xmlns:ce":true},"$$":[{"#name":"__text__","_":"Phylogeography’s past, present, and future: 10 years after "},{"#name":"cross-ref","$":{"refid":"bib9"},"_":"Avise, 2000"}]}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"54","vol-iss-suppl-text":"Volume 54, Issue 1","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":false,"issn":"10557903","issn-primary-formatted":"1055-7903","useEnhancedReader":false,"isCorpReq":false,"pdfDownload":{"linkType":"DOWNLOAD","linkToPdf":"/science/article/pii/S105579030900373X/pdfft?md5=1c84d31246620be32e523c2db2a3578b&pid=1-s2.0-S105579030900373X-main.pdf","isPdfFullText":false,"fileName":"1-s2.0-S105579030900373X-main.pdf"},"pdfUrlForCrawlers":"https://www.sciencedirect.com/science/article/pii/S105579030900373X/pdfft?md5=1c84d31246620be32e523c2db2a3578b&pid=1-s2.0-S105579030900373X-main.pdf","indexTag":true,"volRange":"54","issRange":"1","userProfile":{"departmentName":"General Access","accessType":"IPRANGE","accountId":"262591","webUserId":"12289512","accountName":"International Institute of Tropical Agriculture","departmentId":"267018","userType":"NORMAL","hasMultipleOrganizations":false},"entitlementReason":"package","articleEntitlement":{"entitled":true},"aipType":"none","downloadFullIssue":true,"headerConfig":{"helpUrl":"https://service.elsevier.com/app/home/supporthub/sciencedirect/","contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","orgName":"","webUserId":"12289512","libraryBanner":{},"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasMultiOrg":false,"userType":"IPRANGE","allowCart":false},"open-research":{},"titleString":"Phylogeography’s past, present, and future: 10 years after ","isAbstract":false,"onAbstractWhitelist":false,"isContentVisible":false,"ajaxLinks":{"glossary":true,"citingArticles":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true}},"specialIssueArticles":{},"recommendations":{},"entitledRecommendations":{"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"citingArticles":{},"workspace":{"isOpen":false},"crossMark":{"isOpen":false},"userIdentity":{},"refersTo":{},"referredToBy":{},"downloadIssue":{"isOpen":false,"articles":[],"selected":[]},"references":{},"referenceLinks":{"internal":{},"external":{}},"glossary":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenRecommendations":true,"isOpenCitingArticles":false,"citingArticles":[false,false,false],"recommendations":[false,false,false,false,false,false],"specialIssueArticles":[false,false,false]},"banner":{"expanded":false},"transientError":{"isOpen":false},"pdfDropdown":{},"exportCitation":{"isOpen":false},"tableOfContents":{"showEntitledTocLinks":true},"chapters":{"toc":[],"isLoading":false},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"metrics":{"isOpen":true},"signOut":{"isOpen":false},"issueNavigation":{"previous":{},"next":{}},"tail":{},"linkingHubLinks":[]}</script>
      
      <script src="satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c.js"></script><script src="satellite-565e008964746d4385002642.js"></script>
      <script src="babel-polyfill.js"></script>
      <script src="react.js"></script>
      <script src="react-dom.js"></script>
      <script src="arp.js" async=""></script>
      <!-- begin usabilla live embed code -->
      <script type="text/javascript">
        window.lightningjs||function(c){function g(b,d){d&&(d+=(/\?/.test(d)?"&":"?")+"lv=1");c[b]||function(){var i=window,h=document,j=b,g=h.location.protocol,l="load",k=0;(function(){function b(){a.P(l);a.w=1;c[j]("_load")}c[j]=function(){function m(){m.id=e;return c[j].apply(m,arguments)}var b,e=++k;b=this&&this!=i?this.id||0:0;(a.s=a.s||[]).push([e,b,arguments]);m.then=function(b,c,h){var d=a.fh[e]=a.fh[e]||[],j=a.eh[e]=a.eh[e]||[],f=a.ph[e]=a.ph[e]||[];b&&d.push(b);c&&j.push(c);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"//"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.frameBorder="0";f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
        window.usabilla_live = lightningjs.require("usabilla_live", "https://w.usabilla.com/eb1c14a91932.js");
        var customData = {};

        if(window.pageData && pageData.content && pageData.content[0]) {
          customData.entitlementType = pageData.content[0].entitlementType;
        }
        if(window.pageData && pageData.visitor) {
          customData.accessType = pageData.visitor.accessType;
          customData.accountId = pageData.visitor.accountId;
          customData.loginStatus = pageData.visitor.loginStatus;
        }
        usabilla_live("data", {"custom": customData });
      </script>
      <!-- end usabilla live embed code -->
      <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="MathJax.js"></script>
      <script async="" src="gpt.js"></script>
    
  <iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_elsevier_0" name="destination_publishing_iframe_elsevier_0_name" style="display: none; width: 0px; height: 0px;" src="https://elsevier.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fwww.sciencedirect.com%2Fscience%2Farticle%2Fpii%2FS105579030900373X%3Fvia%253Dihub" class="aamIframeLoaded"></iframe><div class="js-react-modal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><script src="https://d39af2mgp1pqhg.cloudfront.net/widget-summary.js" async=""></script></body></html>