<!DOCTYPE html>
<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><iframe src="javascript:false" title="" style="width: 0px; height: 0px; border: 0px none; display: none;"></iframe><script type="text/javascript" async="" src="geo2.js"></script>
      <meta name="citation_pii" content="S1055790309003480">
<meta name="citation_issn" content="1055-7903">
<meta name="citation_volume" content="53">
<meta name="citation_lastpage" content="947">
<meta name="citation_issue" content="3">
<meta name="citation_publisher" content="Academic Press">
<meta name="citation_firstpage" content="935">
<meta name="citation_journal_title" content="Molecular Phylogenetics and Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.ympev.2009.08.027">
<meta name="dc.identifier" content="10.1016/j.ympev.2009.08.027">
<meta name="citation_article_type" content="Full-length article">
<meta property="og:description" content="The Falkland biota are generally considered to be derived from the nearest continental source in Patagonian South America, yet they harbor many endemi…">
<meta name="citation_title" content="DNA taxonomy and phylogeography of beetles of the Falkland Islands (Islas Malvinas)">
<meta property="og:title" content="DNA taxonomy and phylogeography of beetles of the Falkland Islands (Islas Malvinas)">
<meta name="citation_publication_date" content="2009/12/01">
<meta name="citation_online_date" content="2009/08/31">
<meta name="citation_pdf_url" content="https://www.sciencedirect.com/science/article/pii/S1055790309003480/pdfft?md5=d64b9dd218c2e38401bfa46c6c97bd23&amp;pid=1-s2.0-S1055790309003480-main.pdf">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR">
      <title>DNA taxonomy and phylogeography of beetles of the Falkland Islands (Islas Malvinas) - ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S1055790309003480">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel="shortcut icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="arp.css">
      <link rel="dns-prefetch" href="https://w.usabilla.com/">
      <link rel="dns-prefetch" href="https://www.deepdyve.com/">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <script>
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Molecular Phylogenetics and Evolution"};
        window.pageData = {"content":[{"entitlementType":"package","format":"MIME-XHTML","id":"sd:article:pii:S1055790309003480","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"1055-7903","volumeNumber":"53","issueNumber":"3"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1531924675376,"loadTime":""},"visitor":{"accessType":"ae:ANON_IP","accountId":"ae:262591","accountName":"ae:International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"ae:12289512","ipAddress":"197.248.44.39","appSessionId":"fdf01f62447ee3481f3b4045afa46b236f2bgxrqa"}};
        window.arp = {
          config: {"reduxLogging":false,"assetsBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/d459717ce4bedcd05c18402c0b015aff6be02806","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","googleMapsApiKey":"AIzaSyAV0ZB1XWtRlzJ06pJAW74Oan87lQVgBHY","bosUrl":"https://feedback.recs.d.elsevier.com/raw/events","bosTimeOut":60000,"strictMode":false},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
      </script>
      <script type="text/javascript" src="204774041.js"></script>
      <!-- begin mPulse embed code -->
      <script>
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
    <script src="satellite-5b3b560664746d57b70018c6.js"></script><script src="https://assets.adobedtm.com/376c5346e33126fdb6b2dbac81e307cbacfd7935/s-code-contents-9c0358adbc3b5986e210099b3bf1d427fc5bd286.js"></script><link rel="preload" href="https://adservice.google.com/adsid/integrator.js?domain=www.sciencedirect.com"><script type="text/javascript" src="https://adservice.google.com/adsid/integrator.js?domain=www.sciencedirect.com"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_232.js" async=""></script><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.3/config/MML_SVG.js?V=2.7.3"></script></head>
    <body><div style="display: none;" id="lightningjs-usabilla_live"><div><iframe id="lightningjs-frame-usabilla_live" frameborder="0"></iframe></div></div>
      <a class="sr-only sr-only-focusable" href="#app">Skip to main content</a>
      <!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://service.elsevier.com/app/answers/detail/a_id/9831">this support page</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
      <div data-iso-key="_0"><div class="App" id="app" role="application" data-reactroot=""><div class="page"><section><div id="header"><div class="u-bg-white" style="overflow:visible" role="banner"><div class="els-header" style="min-height:80px"><a id="els-main-title-link" href="https://www.sciencedirect.com/"><svg id="els-header-wordmark" class="u-fill-orange u-margin-l-top u-margin-s-left u-margin-l-left-from-sm" viewBox="-3334 3439.4 163 26" style="width:163px;height:24px"><title>ScienceDirect</title><g transform="scale(.8125,.8125)"><path d="M-4099.8,4240.4c0-1.8,1-3.6,4.4-3.6c1.7,0,3.7,0.5,5.5,1.6l0.2-3.1c-1.7-0.7-3.5-1.1-5.6-1.1 c-5.5,0-7.8,2.9-7.8,6.4c0,6.6,10.4,7.2,10.4,11.7c0,1.8-1.4,3.6-4.6,3.6c-2,0-4-0.7-5.6-1.6l-0.4,3.1c1.8,0.9,4.2,1.2,6.1,1.2 c5,0,7.9-2.9,7.9-6.5C-4089.4,4245.8-4099.8,4244.9-4099.8,4240.4"></path><path id="c" d="M-4080.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4083.1,4244.3-4080.6,4243.1-4080.3,4242.9"></path><path id="i" d="M-4068,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-4066,4234.1-4067,4233.1-4068,4233.1 M-4069.5,4258.1h3v-17.7h-3V4258.1z"></path><path id="e" d="M-4057.1,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4058.7,4244.4-4057.4,4243.3-4057.1,4243.1z M-4047.5,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4 c0,5.8,3.5,9.2,7.9,9.2c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-4047.5,4248.9"></path><path d="M-4034.8,4240c-2.6,0-4.3,1.3-5.7,3.1l-0.5-2.6h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.5v12.8h3v-11.9 c0.8-1.1,2.5-2.9,2.9-3.1c0.3-0.2,1.5-0.5,2.5-0.5c2.7,0,2.9,1.4,3,4c0,1.4,0,3.7,0,3.7c0,3.5-0.1,7.6-0.1,7.6h3 c0,0,0.1-5.3,0.1-8.2c0-1.8,0-3.5-0.1-5.3C-4029.5,4241-4031.6,4240-4034.8,4240"></path><path d="M-3982.5,4255.6h-4.4v-18.6h4.8c6.4,0,8.2,5.2,8.2,9.2C-3973.8,4252.2-3976.5,4255.6-3982.5,4255.6z M-3981.6,4234.6h-8.4v23.5h8.1c8.6,0,11.5-6.7,11.5-11.9C-3970.4,4240.9-3973.2,4234.6-3981.6,4234.6"></path><path d="M-3950.5,4240c-1.9,0-3.4,1.7-4,3.2l-0.5-2.8h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.4v12.8h3v-11.2 c0.6-1.5,2-4.4,3.7-4.4c1.1,0,1.2,1.2,1.2,1.4l2.5-0.7v-0.2c0,0,0-0.2-0.1-0.5C-3947.4,4240.9-3948.5,4240-3950.5,4240"></path><path d="M-3903.5,4255.2c-1.1,0.4-2,0.8-3,0.8c-1.4,0-1.9-0.8-1.9-2.8v-10.4h4.6v-2.3h-4.6v-4.7h-2.9v4.7h-3.2v2.3h3.2 v11.4c0,3.1,1.6,4.4,3.9,4.4c1.4,0,3-0.5,4.1-0.9L-3903.5,4255.2"></path><g><path d="M-3923.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-3926.1,4244.3-3923.6,4243.1-3923.3,4242.9"></path></g><g><path d="M-3941.6,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-3943.2,4244.4-3941.8,4243.3-3941.6,4243.1z M-3932,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3932,4248.9"></path></g><g><path d="M-3964.5,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-3962.5,4234.1-3963.4,4233.1-3964.5,4233.1 M-3965.9,4258.1h3v-17.7h-3V4258.1z"></path></g><g><path d="M-4004.4,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4006,4244.4-4004.6,4243.3-4004.4,4243.1z M-3994.7,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3994.7,4248.9"></path></g><g><path d="M-4019.1,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4021.8,4244.3-4019.4,4243.1-4019.1,4242.9"></path></g></g></svg></a><div class="move-right u-display-inline-block"><nav class="u-clr-grey8 u-show-from-md"><div class="u-display-inline-block" style="margin-top:29px"><span><span class="u-margin-l-right"><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/science/journals" id="els-header-navigation-section-journal" style="border-bottom:"><span class="anchor-text">Journals</span></a><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/science/bookbshsrw" id="els-header-navigation-section-book" style="border-bottom:"><span class="anchor-text">Books</span></a></span><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790309003480" id="els-header-navigation-section-register"><span class="anchor-text">Register</span></a></span><a class="anchor qa-no-js-fallback-link anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790309003480" id="els-header-user-sign-in"><span class="anchor-text">Sign in</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a><li style="list-style:none;display:inline"><a class="anchor u-margin-l-hor help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="help" title="Help" target="_blank"><span class="anchor-text"><svg focusable="false" viewBox="0 0 114 128" style="margin-top:-10px" width="21.375" height="24" class="icon icon-help"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></span></a></li></div></nav></div><div class="u-hide-from-md move-right"><ul><li style="list-style:none;display:inline-block;margin-top:32px" class="u-margin-s-hor u-margin-l-hor-from-sm"><div class="hamburger-button"><button class="button-link button-link-primary" aria-label="Mobile menu" aria-expanded="false" type="button"><svg viewBox="0 0 40 18" width="40" height="18" y="52"><path d="M0 16h40v2H0zm0-8h40v2H0zm0-8h40v2H0z"></path></svg><span class="button-link-text"></span></button></div></li></ul><div style="bottom:0;left:0;position:fixed;top:0;width:100%;z-index:70;opacity:.8" class="u-bg-grey1 u-display-none"></div><div id="mobile-menu" style="overflow:auto;position:fixed;width:288px;right:0;top:0;z-index:1000;height:100%" aria-label="Mobile menu" class="u-bg-grey7 u-clr-grey1 u-display-none" role="navigation"><div class="u-bg-black panel-s"></div><div class="u-bg-grey8 panel-s"><ul><li style="list-style:none"><a class="anchor journals-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/science/journals" style="border-bottom:" id="mobile-journals-link"><span class="anchor-text">Journals</span></a></li><li style="list-style:none"><a class="anchor books-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/science/bookbshsrw" style="border-bottom:" id="mobile-books-link"><span class="anchor-text">Books</span></a></li><li style="list-style:none"><a class="anchor u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790309003480" id="mobile-register-link"><span class="anchor-text">Register</span></a></li></ul></div><div class="panel-s u-bg-grey7"><ul class="text-xs"></ul><ul class="u-margin-s-top text-s"><li style="list-style:none"><a class="anchor anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790309003480" id="mobile-sign-in-out-link" rel="nofollow"><span class="anchor-text">Sign In</span></a></li><li style="list-style:none;display:inline"><a class="anchor u-padding-xs-top text-s help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="mobile-help" title="Help" target="_blank"><span class="anchor-text">Help</span></a></li></ul></div></div><div class="move-bottom move-center"></div></div></div></div></div><div class="Article" id="mathjax-container"><div class="sticky-outer-wrapper"><div class="sticky-inner-wrapper" style="position: relative; z-index: 1; transform: translate3d(0px, 0px, 0px);"><div class="Toolbar" role="region" aria-label="download options and search"><div class="toolbar-container"><div class="u-show-from-lg col-lg-6">&nbsp;</div><div class="buttons pull-left pad-left"><button class="show-toc-button u-hide-from-lg"><svg focusable="false" viewBox="0 0 104 128" width="19.5" height="24" class="icon icon-list"><path d="m2e1 95a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm14 55h68v1e1h-68zm0-3e1h68v1e1h-68zm0-3e1h68v1e1h-68z"></path></svg><span class="label">Outline</span></button><div class="PdfDownloadButton"><button class="button-link button-link-secondary" id="pdfLink" aria-controls="Download PDF options" type="button"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="button-link-text"><span class="pdf-download-label u-show-inline-from-lg">Download PDF</span><span class="pdf-download-label-short u-hide-from-lg">Download</span></span></button></div><div class="ExportCitation" role="region" id="export-citation"><button class="button ExportCitationButton button-anchor" type="button"><span class="button-text">Export</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div><div class="pull-right pad-right u-show-from-md"><form id="quick-search" class="QuickSearch" action="/search/advanced" method="get"><input class="query" aria-label="Search ScienceDirect" name="qs" placeholder="Search ScienceDirect" type="text"><button class="button button-primary" type="submit" aria-label="Submit search"><span class="button-text"><svg focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><a class="advanced-search-link" href="https://www.sciencedirect.com/search/advanced">Advanced</a><input name="origin" value="article" type="hidden"><input name="zone" value="qSearch" type="hidden"></form></div></div></div></div></div><div class="article-wrapper grid row"><div class="u-show-from-lg col-lg-6"><div class="TableOfContents u-margin-l-bottom" lang="en"><div class="Outline" id="toc-outline"><h2>Outline</h2><ol class="u-padding-s-bottom"><li><a href="#aep-abstract-id11" title="Abstract">Abstract</a></li><li><a href="#aep-keywords-id13" title="Keywords">Keywords</a></li><li><a href="#aep-section-id14" title="1. Introduction">1. Introduction</a></li><li><a href="#sec1" title="2. Material and methods">2. Material and methods</a></li><li><a href="#aep-section-id18" title="3. Results">3. Results</a></li><li><a href="#aep-section-id22" title="4. Discussion">4. Discussion</a></li><li><a href="#aep-acknowledgment-id26" title="Acknowledgments">Acknowledgments</a></li><li><a href="#app1" title="Appendix A. Supplementary data">Appendix A. Supplementary data</a></li><li><a href="#aep-bibliography-id33" title="References">References</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show full outline</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2>Figures (8)</h2><ol><li><a href="#fig1"><div><img alt="Fig. 1. Map of the Falkland Islands showing the 10 sampling sites as described in Table…" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr1.sml" style="max-width: 210px; max-height: 164px;"></div></a></li><li><a href="#fig2"><div><img alt="Fig. 2. Bayesian trees of Carabidae (left) and Curculionidae (right)" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr2.sml" style="max-width: 146px; max-height: 164px;"></div></a></li><li><a href="#fig3"><div><img alt="Fig. 3. TCS networks for selected genera" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr3.sml" style="max-width: 75px; max-height: 164px;"></div></a></li><li><a href="#fig4"><div><img alt="Fig. 4. (a) Nucleotide diversity (π) in various species (GMYC groups), separate for…" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr4.sml" style="max-width: 202px; max-height: 164px;"></div></a></li><li><a href="#fig5"><div><img alt="Fig. 5. Dating of intra-generic splits using (a) MrBayes and r8s or (b) BEAST" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr5.sml" style="max-width: 148px; max-height: 164px;"></div></a></li><li><a href="#fig6"><div><img alt="Fig. 6. Clock-constrained trees of the families Carabidae (a) and Curculionidae (b)…" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr6.sml" style="max-width: 132px; max-height: 163px;"></div></a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show all figures</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2>Tables (3)</h2><ol class="u-padding-s-bottom"><li><a href="#tbl1" title="Collection sites. This list describes the main collecting sites with the grid references (see Fig. 1)."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 1</a></li><li><a href="#tbl2" title="Number of morphologically defined species per genus in comparison to GMYC clusters and TCS groups. Only genera with more than 10 sampled individuals are included. Analyses were based on two mtDNA gene..."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 2</a></li><li><a href="#tbl3" title="Bayes factor comparisons for selection of partitioning scheme when using (a) MrBayes or (b) BEAST. Five alternative partitioning schemes (P0–P4) were compared as described in the text. Numbers in brac..."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 3</a></li></ol><div class="PageDivider"></div></div><div class="Extras" id="toc-extras"><h2>Extras (1)</h2><ol class="u-padding-s-bottom"><li><a href="#aep-e-component-id31" title="Supplementary data"><svg focusable="false" viewBox="0 0 95 128" width="17.8125" height="24" class="icon icon-pdf-download"><path d="m82 108h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06h46.4v4e1h1e1v-5e1h-56.4c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-18h-1e1v8zm-25.94-39.4c-0.84-0.38-1.84-0.56-2.98-0.56h-9.04v23.6h5.94v-9h2.18c2.48 0 4.36-0.62 5.66-1.84s1.92-3.06 1.92-5.5c0-1.04-0.12-2-0.4-2.88-0.26-0.88-0.66-1.64-1.22-2.3s-1.22-1.14-2.06-1.52zm-3.12 8.94c-0.4 0.46-0.98 0.7-1.74 0.7h-1.22v-5.76h1.22c1.56 0 2.34 0.96 2.34 2.88 0 0.98-0.2 1.72-0.6 2.18zm21.84-8.52c-0.96-0.66-2.32-0.98-4.06-0.98h-8.72v23.6h8.72c1.74 0 3.1-0.32 4.06-0.98s1.7-1.52 2.18-2.62 0.78-2.34 0.88-3.76 0.16-2.9 0.16-4.44-0.06-3.02-0.16-4.44-0.4-2.68-0.88-3.76c-0.48-1.1-1.2-1.96-2.18-2.62zm-2.78 14.66c-0.06 0.96-0.18 1.72-0.38 2.24s-0.48 0.88-0.84 1.04-0.86 0.24-1.48 0.24h-1.32v-14.74h1.32c0.62 0 1.12 0.08 1.48 0.24s0.64 0.52 0.84 1.04 0.32 1.28 0.38 2.24c0.06 0.98 0.08 2.24 0.08 3.84s-0.02 2.9-0.08 3.86zm13.98-6.58v-4.02h7.74v-5.04h-13.7v23.6h5.96v-9.72h7.26v-4.82z"></path></svg>Supplementary data</a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right" role="main" lang="en"><noscript>JavaScript is disabled on your browser. Please enable JavaScript to use all the features on this page.</noscript><div class="Publication" id="publication"><div class="publication-brand u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903"><img class="publication-brand-image" src="elsevier-non-solus.png" alt="Elsevier"></a></div><div class="publication-volume u-text-center"><h2 class="publication-title" id="publication-title"><a class="publication-title-link" title="Go to Molecular Phylogenetics and Evolution on ScienceDirect" href="https://www.sciencedirect.com/science/journal/10557903">Molecular Phylogenetics and Evolution</a></h2><div class="text-xs"><a title="Go to table of contents for this volume/issue" href="https://www.sciencedirect.com/science/journal/10557903/53/3">Volume 53, Issue 3</a>, <!-- -->December 2009<!-- -->, Pages 935-947</div></div><div class="publication-cover u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903/53/3"><img class="publication-cover-image" src="1-s2.gif" alt="Molecular Phylogenetics and Evolution"></a></div></div><h1 class="Head"><span class="title-text">DNA taxonomy and phylogeography of beetles of the Falkland Islands (Islas Malvinas)</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><a class="author size-m workspace-trigger" name="baep-author-id7" href="#%21"><span class="content"><span class="text given-name">Anna</span><span class="text surname">Papadopoulou</span><span class="author-ref" id="baff1"><sup>a</sup></span><span class="author-ref" id="baff2"><sup>b</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id8" href="#%21"><span class="content"><span class="text given-name">Alex G.</span><span class="text surname">Jones</span><span class="author-ref" id="baff3"><sup>c</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id9" href="#%21"><span class="content"><span class="text given-name">Peter M.</span><span class="text surname">Hammond</span><span class="author-ref" id="baff1"><sup>a</sup></span></span></a><a class="author size-m workspace-trigger" name="baep-author-id10" href="#%21"><span class="content"><span class="text given-name">Alfried P.</span><span class="text surname">Vogler</span><span class="author-ref" id="baff1"><sup>a</sup></span><span class="author-ref" id="baff2"><sup>b</sup></span><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a></div></div></div><button class="show-hide-details"><svg viewBox="0 0 9 9" class="icon-expand"><path d="M5 7H4V5H2V4h2V2h1v2h2v1H5z"></path><path d="M0 0v9h9V0zm1 1h7v7H1z"></path></svg>Show more</button></div><div class="DoiLink" id="doi-link"><a class="doi" href="https://doi.org/10.1016/j.ympev.2009.08.027" target="_blank" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier">https://doi.org/10.1016/j.ympev.2009.08.027</a><a class="rights-and-content" target="_blank" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1055790309003480&amp;orderBeanReset=true">Get rights and content</a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts" id="abstracts"><div class="abstract author" id="aep-abstract-id11" lang="en"><h2 class="section-title">Abstract</h2><div id="aep-abstract-sec-id12"><p>The
 Falkland biota are generally considered to be derived from the nearest 
continental source in Patagonian South America, yet they harbor many 
endemic species whose taxonomy and evolutionary history remains 
insufficiently understood. Comprehensive sampling of Coleoptera over two
 field seasons from numerous sites across the Falkland archipelago 
produced representatives of 55 morphologically separable species, 
assigned to 35 genera and 13 families of Coleoptera. Partial 
mitochondrial Cytochrome Oxidase subunit I and 16S ribosomal RNA genes 
were sequenced for 283 individuals. These sequences formed 55 clusters 
under a Yule-Coalescent model that largely conformed to Linnean species 
while deep-level phylogenetic relationships were broadly congruent with 
the higher level classification. Detailed analysis of the most diverse 
families Carabidae and Curculionidae addressed the question about the 
age and persistence <em>in situ</em> of Falkland biota, showing that 
separation of sister species within genera based on molecular clock 
estimates pre-dated the Pleistocene in all cases. Intra-specific 
diversity of mtDNA haplotypes and nucleotide diversity were high in most
 species, while intra-population variation was equally high and showed 
local differentiation of populations, but there was no 
isolation-by-distance relationship. Taken together, these observations 
indicate that ancient endemics are unlikely to be due to the recent 
establishment from a source elsewhere, but have persisted <em>in situ</em>.
 The observed patterns differ greatly from those in climatically similar
 areas of the Northern Hemisphere. They do not support the view that 
postglacial ranges of insects near the limits of former glaciations are 
merely the result of redistribution due to changing climate.</p></div></div></div><ul class="issue-navigation" id="issue-navigation"><li class="previous"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790309003479"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-left"><path d="m1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text"><span><strong>Previous </strong>article<span class="extra-detail"> in issue</span></span></span></a></li><li class="next"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790309003509"><span class="button-alternative-text"><strong>Next </strong>article<span class="extra-detail"> in issue</span></span><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords"><div id="aep-keywords-id13" class="keywords-section"><h2 class="section-title">Keywords</h2><div class="keyword"><span>Southern Ocean islands</span></div><div class="keyword"><span>Species delimitation</span></div><div class="keyword"><span>Phylogeny</span></div><div class="keyword"><span>Historical biogeography</span></div><div class="keyword"><span>Molecular clock</span></div><div class="keyword"><span>Dispersal</span></div><div class="keyword"><span>Climate change</span></div></div></div><div class="Body" id="body"><div><section id="aep-section-id14"><h2>1. Introduction</h2><p>Surveys
 of variation in mitochondrial DNA are applied to an increasing number 
of taxonomic groups and biogeographical regions and continually improve 
the understanding of biological diversity (e.g., <a name="bbib2" href="#bib2" class="workspace-trigger">Avise, 2000</a>, <a name="bbib27" href="#bib27" class="workspace-trigger">Hebert and Gregory, 2005</a>, <a name="bbib28" href="#bib28" class="workspace-trigger">Hewitt, 1999</a>, <a name="bbib65" href="#bib65" class="workspace-trigger">Savolainen et al., 2005</a>).
 The subantarctic islands of the Southern Ocean remain poorly 
represented in these analyses but are of great interest to taxonomy, 
evolutionary biology and conservation (<a name="bbib10" href="#bib10" class="workspace-trigger">Chown and Convey, 2007</a>, <a name="bbib11" href="#bib11" class="workspace-trigger">Chown et al., 2008</a>).
 Southern Ocean islands have a unique fauna and flora consisting of 
simplified species assemblages whose composition is determined by a 
harsh climate and great distances to the nearest continents. Their 
insect fauna is dominated by a limited set of cold-adapted lineages (<a name="bbib38" href="#bib38" class="workspace-trigger">McDowall, 2005</a>), including many species of beetles in the families Curculionidae (weevils) and Carabidae (ground beetles) (<a name="bbib9" href="#bib9" class="workspace-trigger">Chown, 1994</a>, <a name="bbib54" href="#bib54" class="workspace-trigger">Posadas, 2008</a>).
 In the absence of woody plants on these islands, most species are 
either predatory or feed and scavenge on vegetal matter and tussock 
grass stocks. Beetle assemblages are mostly composed of heavily built, 
ground-dwelling species which are frequently flightless and slow-moving.
 These groups contain numerous endemics that are confined to portions of
 the Southern Ocean provinces, such as a radiation of weevils in the <em>Ectemnorhinus</em> group of genera in the Southern Indian Ocean (<a name="bbib9" href="#bib9" class="workspace-trigger">Chown, 1994</a>). In addition, many recent aliens make up the current fauna and flora which have great impact on the native species (<a name="bbib11" href="#bib11" class="workspace-trigger">Chown et al., 2008</a>, <a name="bbib20" href="#bib20" class="workspace-trigger">Ernsting, 1993</a>, <a name="bbib23" href="#bib23" class="workspace-trigger">Gaston et al., 2003</a>).</p><p>Among
 the Southern Ocean islands, the Falklands (Islas Malvinas) constitute 
the largest archipelago with over 780 isles and a combined area of 
approximately 12,000&nbsp;km<sup>2</sup> stretching from 51<sup>o</sup> to 52<sup>o</sup>55′S. The Falkland Islands are rich in biodiversity compared to other Southern Ocean islands (<a name="bbib24" href="#bib24" class="workspace-trigger">Gressitt, 1970</a>, <a name="bbib31" href="#bib31" class="workspace-trigger">Jones, 2004</a>, <a name="bbib32" href="#bib32" class="workspace-trigger">Jones et al., 2003</a>) but their fauna remains poorly documented (<a name="bbib49" href="#bib49" class="workspace-trigger">Oldfield, 1987</a>, <a name="bbib50" href="#bib50" class="workspace-trigger">Oldfield and Sheppard, 1997</a>, <a name="bbib59" href="#bib59" class="workspace-trigger">Procter and Fleming, 1999</a>).
 A taxonomic checklist of the species present in the collections and 
references of the Natural History Museum (London) was published in 1984 
and lists ca. 230 extant species (<a name="bbib62" href="#bib62" class="workspace-trigger">Robinson, 1984</a>).
 Coleoptera are represented by some 15 different families from a wide 
range of major lineages. The dominant Carabidae and Curculionidae 
include several groups of closely related species which are difficult to
 separate and whose taxonomic status is uncertain because of 
intra-specific variation. This variation among individuals and 
populations is a great problem for morphological species recognition, 
while recent conservation efforts (<a name="bbib31" href="#bib31" class="workspace-trigger">Jones, 2004</a>, <a name="bbib50" href="#bib50" class="workspace-trigger">Oldfield and Sheppard, 1997</a>, <a name="bbib75" href="#bib75" class="workspace-trigger">UKDTCF, 1996</a>) have raised the need for proper taxonomic information that also includes the frequently encountered larval stages.</p><p>Approximately 70% of the insect species described in the 1984 list were not then recorded elsewhere (<a name="bbib62" href="#bib62" class="workspace-trigger">Robinson, 1984</a>).
 Although this high number of presumed endemics may be inflated due to 
insufficient sampling in adjacent south temperate regions, a review of 
current knowledge on biogeographic relationships (<a name="bbib38" href="#bib38" class="workspace-trigger">McDowall, 2005</a>)
 showed that several groups of animals and plants exhibit a high 
proportion of species apparently unique to the islands. This analysis 
also revealed the overwhelming evidence for close affinities of the 
Falkland Islands flora and fauna with those of Patagonian South America (<a name="bbib38" href="#bib38" class="workspace-trigger">McDowall, 2005</a>),
 whereas only the most dispersive groups show affinities with other 
Southern Ocean islands or mainland Antarctica, including sea birds such 
as Southern Skuas (<em>Catharacta</em> spp.) (<a name="bbib61" href="#bib61" class="workspace-trigger">Ritz et al., 2008</a>). The Falklands and southern South America are 450&nbsp;km
 distant at the closest continental landfall in Tierra del Fuego. The 
high endemicity of the insect fauna could therefore be due to the 
long-term isolation of these biota, in particular for groups that are 
largely flightless, such as the 22 species of weevils known from the 
Falkland Islands of which only four are shared with the Fuegan or 
Valdivian provinces of southern South America (<a name="bbib54" href="#bib54" class="workspace-trigger">Posadas, 2008</a>).</p><p>The high degree of endemicity raises the question about the origin of the Falkland fauna and the role of dispersal (<a name="bbib7" href="#bib7" class="workspace-trigger">Buckland and Hammond, 1997</a>).
 Paleogeographic investigations established that the shelf region that 
makes up the Falklands was originally part of eastern South Africa from 
where it moved due to processes of plate tectonics that began with the 
break-up of Gondwanaland some 200&nbsp;Mya. The Falkland Islands crustal
 block was carried westward with the South American plate to which it 
has been linked since before the break-up of Africa and South America 
some 130&nbsp;Mya (<a name="bbib36" href="#bib36" class="workspace-trigger">Marshall, 1994</a>, <a name="bbib69" href="#bib69" class="workspace-trigger">Storey et al., 1999</a>, <a name="bbib74" href="#bib74" class="workspace-trigger">Trewin et al., 2002</a>).
 None of the animal and plant groups investigated to date appear to 
reflect these ancient geological affinities with Africa while the 
taxonomic relationships with the South American groups are strongly 
corroborated by phylogenetic studies (<a name="bbib46" href="#bib46" class="workspace-trigger">Morrone and Posadas, 2005</a>). Pangeographic analyses also support this major link, with minor ‘generalized tracks’ to other subantarctic islands (<a name="bbib46" href="#bib46" class="workspace-trigger">Morrone and Posadas, 2005</a>).
 Sea level changes during the Tertiary and in particular the most recent
 glacial cycles might have resulted in temporary land connection between
 these areas with a drop of sea levels of only 100–150&nbsp;m (<a name="bbib44" href="#bib44" class="workspace-trigger">Morrone, 1998</a>).
 However, intermittent cold periods throughout the last ice age have 
subjected the islands to periglacial conditions lasting thousands of 
years and, although never covered by an ice sheet, led to the formation 
of small mountain top glaciers during the most recent cold stage between
 ca. 26 and 13&nbsp;ka BP (<a name="bbib12" href="#bib12" class="workspace-trigger">Clapperton, 1990</a>, <a name="bbib13" href="#bib13" class="workspace-trigger">Clapperton and Sugden, 1976</a>, <a name="bbib26" href="#bib26" class="workspace-trigger">Hall, 1990</a>, <a name="bbib39" href="#bib39" class="workspace-trigger">Mercer, 1983</a>).
 Periglacial conditions might have been severe enough to cause the local
 extinction of most terrestrial species. Therefore it has been 
postulated that the current insect species found in the islands would 
have been colonized during the last 15–10&nbsp;ka BP (<a name="bbib7" href="#bib7" class="workspace-trigger">Buckland and Hammond, 1997</a>).</p><p>Recent activities to establish baseline data for invertebrate distribution on the Falklands (<a name="bbib31" href="#bib31" class="workspace-trigger">Jones, 2004</a>)
 have produced large numbers of specimens useful for DNA extraction and 
evolutionary analyses that might resolve the questions about the age and
 origin of the Falkland fauna. Here we provide sequences for a 
representative sample of all species of beetles encountered in two 
expeditions to the islands, using two mitochondrial gene regions. The 
application of recently developed coalescent-based methods for species 
delimitation using branching patterns in DNA-based phylogenetic trees (<a name="bbib22" href="#bib22" class="workspace-trigger">Fontaneto et al., 2007</a>, <a name="bbib53" href="#bib53" class="workspace-trigger">Pons et al., 2006</a>)
 provided independent tests of morphologically defined entities and the 
presence of unrecognized species. In addition, the geographic extent and
 topographic structure of the archipelago which is characterized by a 
highly indented coastline and major mountain ranges of &gt;700&nbsp;m 
above sea level on both main islands, East Falkland (Soledad) and West 
Falkland (Gran Malvina), might result in substantial population 
subdivision. The sequence analysis, together with existing morphological
 analyses conducted in adjacent South American sites, provides an 
evolutionary framework for the beetle assemblage of the Falkland 
Islands, estimating the age of lineages with calibrated molecular clocks
 and testing the possibility of <em>in situ</em> speciation on the islands.</p></section><section id="sec1"><h2>2. Material and methods</h2><section id="aep-section-id15"><h3>2.1. Sampling and DNA sequencing</h3><div><p>Samples
 were obtained as a part of an insect survey carried out by the Falkland
 Island Invertebrates Conservation Project (FIICP), during 
January–February 2004 and January–March 2005. Specimens were collected 
using a variety of methods including pitfall trapping, Tullgren funnel 
extraction, Malaise trapping and hand collection, then stored in 96% 
ethanol and kept refrigerated at 4&nbsp;°C. Specimens were sorted and 
identified to species against catalogues and the collection of the NHM. 
Preliminary identifications were maintained throughout the data analysis
 reported here, but may need to be re-assessed in the future. In total, 
sequence data were obtained for 260 adult beetles and 23 larvae. These 
specimens were collected across a diverse range of habitat types from 
numerous sites found within 16 different 10&nbsp;km<sup>2</sup> grid 
squares. Specimens collected from connecting grid squares were grouped 
together such that they represented 10 discrete geographic localities 
across the Falkland main islands and adjacent islets (<a name="btbl1" href="#tbl1" class="workspace-trigger">Table 1</a> and <a name="bfig1" href="#fig1" class="workspace-trigger">Fig. 1</a>).</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl1"><div class="captions"><span><p><span class="label">Table 1</span>. Collection sites. This list describes the main collecting sites with the grid references (see <a name="bfig1" href="#fig1" class="workspace-trigger">Fig. 1</a>).</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">Region</th><th scope="col" class="align-left">Site</th><th scope="col" class="align-left">Locality code</th><th scope="col" class="align-left">UT grid reference</th></tr></thead><tbody><tr class="valign-top"><td class="align-left" colspan="4"><em>East Falkland</em></td></tr><tr class="valign-top"><td class="align-left" rowspan="4">Mainland</td><td class="align-left">Stanley and Gypsy Cove</td><td class="align-left">EF1</td><td class="align-left">VC47</td></tr><tr class="valign-top"><td class="align-left">Cape Dolphin</td><td class="align-left">EF2</td><td class="align-left">UD61&nbsp;+&nbsp;UD71</td></tr><tr class="valign-top"><td class="align-left">Mt. Usborne</td><td class="align-left">EF3</td><td class="align-left">UC77</td></tr><tr class="valign-top"><td class="align-left">Lafonia</td><td class="align-left">EF4</td><td class="align-left">UC54&nbsp;+&nbsp;UC55&nbsp;+&nbsp;UC66</td></tr><tr class="valign-top"><td class="align-left" rowspan="2">Islands</td><td class="align-left">Pleasant Island</td><td class="align-left">EF5</td><td class="align-left">VC15</td></tr><tr class="valign-top"><td class="align-left">Bleaker Island</td><td class="align-left">EF6</td><td class="align-left">UC81</td></tr><tr class="valign-top"><td class="align-left" colspan="4"><br></td></tr><tr class="valign-top"><td class="align-left" colspan="4"><em>West Falkland</em></td></tr><tr class="valign-top"><td class="align-left">Mainland</td><td class="align-left">Hill Cove</td><td class="align-left">WF1</td><td class="align-left">TC88&nbsp;+&nbsp;TC89&nbsp;+&nbsp;TD90</td></tr><tr class="valign-top"><td class="align-left" rowspan="3">Islands</td><td class="align-left">Carcass Island</td><td class="align-left">WF2</td><td class="align-left">TD51</td></tr><tr class="valign-top"><td class="align-left">Grand Jason</td><td class="align-left">WF3</td><td class="align-left">TD43</td></tr><tr class="valign-top"><td class="align-left">Steeple Jason</td><td class="align-left">WF4</td><td class="align-left">TD03&nbsp;+&nbsp;TD04</td></tr></tbody></table></div></div><figure class="figure" id="fig1"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr1.jpg" alt="" height="519"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr1.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 1</span>. Map of the Falkland Islands showing the 10 sampling sites as described in <a name="btbl1" href="#tbl1" class="workspace-trigger">Table 1</a>.</p></span></div></figure></div><p>Non-destructive
 DNA extraction was performed using a Qiagen DNeasy kit. Amplification 
of two mtDNA regions was generally successful with well-established 
oligonucleotides. A 826&nbsp;bp fragment of Cytochrome Oxidase subunit 1 (<em>cox1</em>) was amplified using primer pair Pat (5′TCCAATGCACTAATCTGCCATATTA3′) and Jerry (5′CAACATTTATTTTGATTTTTTGG3′) (<a name="bbib66" href="#bib66" class="workspace-trigger">Simon et al., 1994</a>) and partial 16S ribosomal RNA (<em>rrnL</em>), adjacent tRNA leucine 2 (<em>trnL2</em>) and NADH Dehydroxygenase subunit 1 (<em>nad1</em>)
 were amplified using forward primer 16Sar (5′CGCCTGTTTAACAAAAACAT3′) 
and reverse primers ND1A (5′GGTCCCTTACGAATTTGAATATATCCT3′) or 16SB2 
(5′CTCCGGTTTGAACTCAGATCA3′) (<a name="bbib66" href="#bib66" class="workspace-trigger">Simon et al., 1994</a>).
 PCR fragments were sequenced in both directions on an ABI377 automated 
sequencer (Applied Biosystems). Sequence chromatograms were assembled 
and edited using the Sequencher 4.6 software (Gene Codes Corp., Ann 
Harbor, MI, USA). Sequences were aligned using ClustalW with the default
 gap opening and extension penalties (<a name="bbib72" href="#bib72" class="workspace-trigger">Thompson et al., 1994</a>). All sequences have been submitted to the EMBL Nucleotide Sequence Database under Accession Nos. <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FM994290" target="_blank">FM994290</a>–<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FM994548" target="_blank">FM994548</a> (<em>cox1</em>) and <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FM994683" target="_blank">FM994683</a>–<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;db=nucleotide&amp;doptcmdl=genbank&amp;term=FM994891" target="_blank">FM994891</a> (<em>rrnL</em>–<em>trnL2</em>–<em>nad1</em> fragment). A total of 17 sequences analyzed here had already been published in <a name="bbib101" href="#bib101" class="workspace-trigger">Hunt et al. (2007)</a> (Accession Nos. in <a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Table S1</a>).</p></section><section id="aep-section-id16"><h3>2.2. Phylogenetic analysis and species delimitation</h3><p>Bayesian analyses were performed in MrBayes 3.1.2 (<a name="bbib63" href="#bib63" class="workspace-trigger">Ronquist and Huelsenbeck, 2003</a>)
 for the full dataset and separately for the families Carabidae and 
Curculionidae, with two parallel runs using one cold and three 
incrementally heated Markov chains (<em>λ</em>&nbsp;=&nbsp;0.1) and sampling one in every 1000 trees. Two partitions were defined <em>a priori</em> corresponding to the protein coding regions (<em>cox1</em> and <em>nad1</em>) and the structural RNA gene regions (<em>rrnL</em> and <em>trnL2</em>),
 respectively, and a separate GTR+Γ+I model was applied to each 
partition. Moreover, in order to account for among-partition rate 
variation, a separate rate multiplier was assigned to each partition. 
Standard convergence diagnostics, as implemented in MrBayes (<em>sump</em>
 command) were checked to ensure that the Markov chain had reached 
stationarity. A total of 16 million generations was performed for the 
full data set and 5 million generations for the Carabidae and 
Curculionidae data sets. Trees were summarized using an ‘all-compatible’
 consensus after discarding a 50% burn-in.</p><p>Quantitative species 
delimitation was performed with the General Mixed Yule-Coalescent (GMYC)
 method that establishes the point of transition from slow to faster 
apparent branching rates of the gene tree expected at the species 
boundary (<a name="bbib22" href="#bib22" class="workspace-trigger">Fontaneto et al., 2007</a>, <a name="bbib53" href="#bib53" class="workspace-trigger">Pons et al., 2006</a>). The method optimizes a threshold age under a stochastic branching rate (Yule) model (<a name="bbib47" href="#bib47" class="workspace-trigger">Nee et al., 1992</a>, <a name="bbib79" href="#bib79" class="workspace-trigger">Yule, 1924</a>) and a separate process of branching for each species under a coalescent model (<a name="bbib30" href="#bib30" class="workspace-trigger">Hudson, 1991</a>, <a name="bbib76" href="#bib76" class="workspace-trigger">Wakeley, 2006</a>).
 Branches crossing the threshold define genetic clusters each obeying an
 independent coalescent process. The likelihood of this model is 
compared to a null model that the entire sample can be fitted by a 
single neutral coalescent model for all individuals. The GMYC model 
provides a credibility interval corresponding to the number of clusters 
at ±2log&nbsp;<em>L</em> units of the ML estimate. The analysis was 
carried out using the R package ‘splits’ (SPecies LImits by Threshold 
Statistics) available at <a href="http://r-forge.r-project.org/projects/splits/" target="_blank">http://r-forge.r-project.org/projects/splits/</a><em>.</em>
 We applied the GMYC method on Bayesian ‘all-compatible’ consensus trees
 including only unique haplotypes, which were obtained in MrBayes as 
described before and converted to ultrametric using penalized likelihood
 in r8s 1.7 (<a name="bbib64" href="#bib64" class="workspace-trigger">Sanderson, 2003</a>), with the optimal smoothing parameter determined by cross-validation of values between 0.01 and 1000.</p><p>All GMYC groups were numbered with a unique alphanumerical identifier (<a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Table S1</a>)
 from which a holotype specimen (DNA preparation) was selected to 
represent this sequence cluster, based on the most complete sequence 
(fewest missing data). Where the GMYC groups were congruent with the 
Linnean names, sequences were submitted to GenBank under that name. In 
cases of discrepancies between GMYC groups and Linnean names, or if none
 of the members of a GMYC group had been assigned a Linnean species 
identification, we appended the alphanumerical GMYC designation after 
the genus name for submission. This treatment is one of several 
possibilities to deal with the lack of one-to-one correspondence of 
mtDNA groups and Linnean names. The approach emphasizes the result of 
the current DNA-based analysis which generally provided strongly 
subdivided DNA-based groups that are open to further scrutiny in 
classical taxonomic analyses including the specimens used here, after 
which these alphanumerical designations may be exchanged for Linnean 
binomials.</p><p>Statistical parsimony analysis was performed with TCS 1.21 <a name="bbib100" href="#bib100" class="workspace-trigger">(Clement et al., 2000)</a>
 which partitions the data into independent networks of haplotypes 
connected by changes that are non-homoplastic with a 95% probability. 
The analysis was performed on all genera represented by at least 10 
individuals. The hierarchical levels of the resulting networks were 
established using specific nesting rules and separate TCS groups were 
defined using the 95% connection limit (<a name="bbib70" href="#bib70" class="workspace-trigger">Templeton et al., 1987</a>, <a name="bbib71" href="#bib71" class="workspace-trigger">Templeton and Sing, 1993</a>). Arlequin 3.11 (<a name="bbib21" href="#bib21" class="workspace-trigger">Excoffier et al., 2005</a>) was used to estimate within-species nucleotide diversity.</p></section><section id="aep-section-id17"><h3>2.3. Divergence time estimation and data partitioning</h3><p>In
 order to estimate divergence times between closely related GMYC groups,
 we analyzed two reduced datasets that comprised a single individual per
 group for the families Carabidae and Curculionidae, respectively. 
Analyses were performed in MrBayes as described before for the full 
datasets, using the same outgroup sequences and without enforcing a 
clock. Trees were again converted to ultrametric using penalized 
likelihood in r8s 1.7 using the ‘all-compatible’ consensus as an input 
for the cross-validation procedure in order to define the smoothing 
parameter, which was then applied to 2500 sampled trees from the 
stationary phase in order to obtain mean estimated ages and standard 
deviations. In the absence of fossil calibration points, we applied the 
widely accepted insect mitochondrial clock of 2.3% divergence per 
million year (<a name="bbib6" href="#bib6" class="workspace-trigger">Brower, 1994</a>),
 by fixing the mean substitution rate estimated by r8s to 0.0115 per 
site per My. For comparison, we also analyzed the two datasets in BEAST 
1.4.8 (<a name="bbib18" href="#bib18" class="workspace-trigger">Drummond et al., 2006</a>, <a name="bbib19" href="#bib19" class="workspace-trigger">Drummond and Rambaut, 2007</a>)
 under an uncorrelated lognormal relaxed clock and without an outgroup. 
The mean of the branch rates (ucld.mean) was fixed to 0.0115, a Yule 
tree prior was employed and the default options were used for all other 
prior and operator settings. Two independent runs of 20 million 
generations were conducted for each analysis, sampling every 2000th 
generation, the convergence and mixing of each MCMC chain was assessed 
by inspection of the trace plots and the Effective Sample Sizes using 
Tracer 1.4.1 (<a name="bbib19" href="#bib19" class="workspace-trigger">Drummond and Rambaut, 2007</a>)
 and samples from both runs were pooled after removing a 10% burn-in 
using LogCombiner 1.4.8. The posterior probabilities of the individual 
clades and the posterior estimates and 95% Highest Posterior Density 
limits of the node heights were summarized using TreeAnnotator 1.4.8 and
 visualized using FigTree 1.2. The means and standard errors of the node
 heights were summarized using Tracer and standard deviations were 
calculated by multiplying the standard errors by the square root of the 
Effective Sample Size in each case.</p><p>We assessed the effect of rate heterogeneity among genes and codon positions for molecular dating (<a name="bbib78" href="#bib78" class="workspace-trigger">Yang and Yoder, 2003</a>)
 by applying a range of different partitioning schemes and comparing 
their likelihood. The following partitioning schemes were compared: P0, 
no partitioning; P1, protein coding regions (<em>cox1</em> and <em>nad1</em>) versus structural RNA gene regions (<em>rrnL</em> and <em>trnL</em>2);
 P2, 3rd codon positions versus non-3rd codon positions; P3, 1st plus 
2nd codon positions versus 3rd codon positions versus structural RNA 
genes; P4, each codon position separately and non-protein coding DNA as a
 4th partition. A nucleotide substitution model applied to each 
partition was selected by the AIC criterion as implemented in 
MrModeltest 2.2 (<a name="bbib48" href="#bib48" class="workspace-trigger">Nylander, 2004</a>).
 The HKY+I nucleotide substitution model was applied to the 2nd codon 
position when considered alone (in partition scheme P4), and a separate 
GTR+Γ+I model to all other partitions. Bayes Factors were calculated in 
order to choose the preferred partition scheme, based on the natural 
logarithm of the harmonic mean of the sampled likelihood values at the 
stationary phase, which was estimated either using the <em>sump</em> 
command in MrBayes or, in the case of the BEAST searches, by Tracer 
1.4.1 with 1000 bootstrap replicates. We followed either the widely used
 cut-off values proposed by <a name="bbib33" href="#bib33" class="workspace-trigger">Kass and Raftery (1995)</a>, when comparing partition schemes that required equal number of free parameters, or the recommendations of <a name="bbib51" href="#bib51" class="workspace-trigger">Pagel and Meade (2004)</a> for penalizing over-parameterization, as applied by <a name="bbib40" href="#bib40" class="workspace-trigger">Miller et al. (in press)</a>, when partition schemes differed in total number of free parameters. In this case we calculated the ratio ln(Bayes Factor)/Δ<em>p</em> (Δ<em>p</em>&nbsp;=&nbsp;difference
 in number of total free parameters between alternative partition 
schemes) and we accepted additional free parameters only when this value
 was 10 or greater. The GTR+Γ+I model has 10 free parameters (5 for the 
substitution rate matrix, 3 for the estimated bases frequencies, 1 for 
the alpha parameter of the gamma distribution and 1 for the proportion 
of invariable sites) and the HKY+I has 5 free parameters (1 for the 
transition/transversion ratio, 3 for the base frequencies and 1 for the 
proportion of invariable sites), while each added partition adds an 
extra free parameter due to the partition-specific rate multiplier. P0 
(no partitioning) had 10 free parameters, partition schemes P1 and P2 
shared the same total number of free parameters (<em>p</em>&nbsp;=&nbsp;21),
 while P3 and P4 had a total of 32 and 38 free parameters, respectively.
 Estimated ln(Bayes Factors) were thus used to compare P1 versus P2 
while all other comparisons were based on the ratios ln(Bayes Factor)/Δ<em>p</em> (<a name="bbib51" href="#bib51" class="workspace-trigger">Pagel and Meade, 2004</a>).</p></section></section><section id="aep-section-id18"><h2>3. Results</h2><section id="aep-section-id19"><h3>3.1. Species encountered and phylogenetic relationships</h3><p>The
 collecting effort resulted in 55 morphologically recognizable species, 
plus three tentative groups which represented slight variants. These 
species could be assigned to 35 genera and 13 families of Coleoptera. 
The individuals encountered included several widely distributed species 
including several commensals likely introduced, such as the Australian 
Spider Beetle <em>Ptinus tectus</em> (Anobiidae) and the European rove beetle <em>Atheta amicula</em>
 (Staphylinidae), and others that are members of widespread genera but 
whose exact species identity and status as autochthonous to the 
Falklands were unclear, including four species of <em>Aridius</em> 
(Lathridiidae). The majority of samples was attributed to three 
families, Carabidae, Curculionidae and Perimylopidae. The latter family 
is endemic to the temperate Southern Hemisphere, while the members of 
Carabidae and Curculionidae also consisted of lineages with predominant 
distributions in subantarctic South America.</p><div><p>Sequences of 283 representative individuals for <em>cox1</em> and <em>rrnL</em> produced a Bayesian tree (<a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Fig. S1</a>)
 that grouped the members of each family as monophyletic. Deep 
relationships within Polyphaga were not resolved satisfactorily, given 
the sporadic taxon sampling of deep lineages and the use of mtDNA which 
is insufficient to resolve basal relationships in Coleoptera (<a name="bbib29" href="#bib29" class="workspace-trigger">Howland and Hewitt, 1995</a>).
 Bayesian analyses were conducted individually on sequences of the two 
most highly represented families, Carabidae and Curculionidae (<a name="bfig2" href="#fig2" class="workspace-trigger">Fig. 2</a>a
 and b). These trees show that species broadly grouped according to 
their morphological identification (see below) and lineages of closely 
related haplotypes frequently included sequences obtained from larvae 
for which no morphological identification was available. Likewise, taxon
 designations at the genus level were recovered as monophyletic groups, 
except for the genus <em>Antarctobius</em> which was paraphyletic for the single species of <em>Germaniellus</em>.
 The tribal and subfamily relationships were also as expected, with the 
notable exception of the paraphyly of the curculionid subfamily 
Entiminae (<a name="bfig2" href="#fig2" class="workspace-trigger">Fig. 2</a>).</p><figure class="figure" id="fig2"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr2.jpg" alt="" height="898"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr2.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 2</span>. 
Bayesian trees of Carabidae (left) and Curculionidae (right). 
Independent GMYC groups are marked with a square. Posterior 
probabilities from Bayesian analysis are given on the branches and 
branch length corresponds to inferred numbers of substitutions per site.
 Terminals are labeled according to morphological identification. See 
the tree in <a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Fig. S1</a> for the corresponding name codes of GMYC groups used in GenBank.</p></span></div></figure></div></section><section id="aep-section-id20"><h3>3.2. Species delimitation and geographic distribution of haplotypes</h3><div><p>Species
 delimitation using the GMYC methodology was performed separately for 
Carabidae, Curculionidae and all other families combined. This revealed a
 total of 53 (45–57) independent coalescence groups from the combined 
data which was reduced to 47 (39–53) groups when using the less variable
 <em>rrnL</em> gene separately, although this number was affected by 
partially missing data for this gene. Only one of the GMYC clusters from
 the combined analysis was not detected when using <em>cox1</em> data alone (<a name="btbl2" href="#tbl2" class="workspace-trigger">Table 2</a>).
 In no case were the combined-data GMYCs inconsistent with the 
single-gene analyses, i.e., differences were due to lack of resolution. 
Most larval specimens were easily associated with one of the GMYC 
groups, providing an unambiguous species identification for 20 larval 
specimens as belonging to 9 species. Three further specimens were 
assigned to GMYC groups not sampled in the adult stage; one of them was 
closely allied to the curculionid <em>Ca. biangulatus</em>, a second was
 deeply embedded in the curculionid subfamily Cyclominae, while the 
third was monophyletic with the Staphylinidae plus Byrrhidae in the 
all-species Bayesian analysis (<a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Fig. S1</a>) and closest to members of Scarabaeidae (<em>rrnL</em>; 86% identity, 4% gaps) and Staphylinidae (<em>cox1</em>; 84% identity) in Blastn searches (<em>E</em>&nbsp;=&nbsp;0.0).</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl2"><div class="captions"><span><p><span class="label">Table 2</span>. 
Number of morphologically defined species per genus in comparison to 
GMYC clusters and TCS groups. Only genera with more than 10 sampled 
individuals are included. Analyses were based on two mtDNA gene regions 
separately and in combination. Asterisks indicate cases where the 
results were affected by missing data for the <em>rrnL</em> region.</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">Genera</th><th scope="col" class="align-left">Morphological species</th><th scope="col" class="align-left">GMYC COI/16S/both</th><th scope="col" class="align-left">TCS networks COI/16S/both</th><th scope="col" class="align-left">Match morphology</th></tr></thead><tbody><tr class="valign-top"><td class="align-left"><em>Lissopterus</em></td><td class="align-left">2</td><td class="align-left">2/1/2</td><td class="align-left">2/1/2</td><td class="align-left">No</td></tr><tr class="valign-top"><td class="align-left"><em>Metius</em></td><td class="align-left">3</td><td class="align-left">3/3/3</td><td class="align-left">3/3/3</td><td class="align-left">Yes</td></tr><tr class="valign-top"><td class="align-left"><em>Trechisibus</em></td><td class="align-left">3</td><td class="align-left">3/3/3</td><td class="align-left">3/3/3</td><td class="align-left">Yes</td></tr><tr class="valign-top"><td class="align-left"><em>Oopterus</em></td><td class="align-left">1</td><td class="align-left">1/1/1</td><td class="align-left">1/1/1</td><td class="align-left">Yes</td></tr><tr class="valign-top"><td class="align-left"><em>Germainiellus</em></td><td class="align-left">1(+1)</td><td class="align-left">1/1/1</td><td class="align-left">1/1/1</td><td class="align-left">Yes</td></tr><tr class="valign-top"><td class="align-left"><em>Antarctobius</em></td><td class="align-left">3</td><td class="align-left">4/4/5</td><td class="align-left">5/4/5</td><td class="align-left">No</td></tr><tr class="valign-top"><td class="align-left"><em>Falklandius</em></td><td class="align-left">4</td><td class="align-left">2/2/2</td><td class="align-left">2/2/2</td><td class="align-left">No</td></tr><tr class="valign-top"><td class="align-left"><em>Puranius</em></td><td class="align-left">3</td><td class="align-left">3/1∗/3</td><td class="align-left">3/1∗/3</td><td class="align-left">Yes</td></tr><tr class="valign-top"><td class="align-left"><em>Malvinius</em></td><td class="align-left">2</td><td class="align-left">3/2∗/3</td><td class="align-left">5/2∗/5</td><td class="align-left">No</td></tr><tr class="valign-top"><td class="align-left"><em>Parahelops</em></td><td class="align-left">4</td><td class="align-left">5/2∗/5</td><td class="align-left">5/2∗/5</td><td class="align-left">No?</td></tr><tr class="valign-top"><td class="align-left"><em>Aridius</em></td><td class="align-left">4</td><td class="align-left">4/2∗/4</td><td class="align-left">4/2∗/4</td><td class="align-left">Yes</td></tr><tr class="valign-top"><td class="align-left"><em>Chalcosphaerium</em></td><td class="align-left">1</td><td class="align-left">1/1/1</td><td class="align-left">1/1/1</td><td class="align-left">Yes</td></tr></tbody></table></div></div></div><p>The
 GMYC groups were largely congruent with the morphological 
identifications, but with several exceptions. Complete consensus was 
obtained for <em>Metius</em> (3 species), <em>Trechisibus</em> (3 species), <em>Puranius</em> (3 species) and <em>Aridius</em>
 (4 species). The remaining five genera showed various degrees of 
incongruence. A single named species was separated into two or three 
GMYC groups in <em>Antarctobius vulsus</em> and <em>Parahelops quadricollis</em> (cryptic species). A more complicated situation was encountered in <em>Malvinius</em>, where one of the named species was paraphyletic for another species/GMYC group (species paraphyly); in <em>Falklandius</em> which showed three species collapsing into a single GMYC group (species polyphyly); and in <em>Lissopterus</em>
 showing some members of a Linnean species incorrectly assigned to a 
different GMYC group (conflict with named groups). Using GMYC group 
limits at the ±2log&nbsp;<em>L</em> high and low estimate did not improve the fit with the morphological groups.</p><div><p>Genera with &gt;10 representatives were also used for network construction using statistical parsimony analysis (<a name="bbib70" href="#bib70" class="workspace-trigger">Templeton et al., 1987</a>). These analyses produced nesting groups at between three and five hierarchical (<em>n</em>-step) levels when using the <em>cox1</em> gene (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a> and <a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Fig. S2</a>). TCS networks were fully congruent with GMYC groups, with the exception of <em>Antarctobius</em> and <em>Germaniellus</em>, where 1 and 2 additional TCS networks were encountered, respectively (<a name="btbl2" href="#tbl2" class="workspace-trigger">Table 2</a>).
 The TCS analysis did not resolve the morphology–mtDNA inconsistencies 
but revealed more detail about the substructure within these groups due 
to the hierarchical arrangement implied by the nested design. In the 
case of paraphyletic species within <em>Malvinus</em>, <em>M. nordenskioldi</em> corresponded to a tight group at the 2-step level distant from the much more diverse and paraphyletic <em>M. compressiventris</em> to which it was grouped only at the 4-step level (and additional networks) (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>a). In the case of the polyphyletic species of <em>Falklandius</em>,
 a single TCS network was comprised of three species, in agreement with 
the GMYC groups. A subset of haplotypes was identical in two species (<em>F. goliath</em> and <em>F. turbificatus</em>) or grouped two species at the 3- and 4-step levels (<em>F. goliath</em> and <em>F. turbificatus</em>; <em>F. goliath</em> and <em>F. kuscheli</em>) (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>b). It is also noteworthy that this multi-species network showed geographic structure, as <em>F. goliath</em> and <em>F. turbificatus</em> were confined to site W1, whereas <em>F. kuscheli</em>
 was limited to sites W2 and E4. However, this geographic arrangement 
was not reflected in the phylogenetic relationships of haplotypes, as 
the co-occurring haplotypes were from distant portions of the haplotype 
network (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>b), arguing against a scenario of vicariance that may have produced these haplotype subgroups. In the genus <em>Lissopterus</em> (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>c), one haplotype of <em>L. hyadesi</em> was shared with <em>L. quadrinotatus</em> and all <em>L. hyadesi</em>
 haplotypes were members of a single 1-step network. Each species was 
collected from four localities which were widely distributed throughout 
the West and East Falklands, while the haplotypes shared between both 
species were found in distant places, again arguing against vicariant 
scenarios of species splits. Haplotypes assigned to <em>L. quadrinotatus</em>
 were highly diverse, separating into two independent networks, and the 
co-occurring haplotypes were drawn from several divergent portions of 
the networks (e.g., at site E4 haplotypes from both 4-step and four 
3-step networks were present; <a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>c).</p><figure class="figure" id="fig3"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr3.jpg" alt="" height="1012"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr3.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 3</span>. 
TCS networks for selected genera. Each gray field corresponds to a 
single individual, connected by lines representing a single mutational 
step. Small circles represent hypothetical haplotypes. Nesting levels 
are represented with broken and continuous lines from lowest to highest 
nesting levels, with designations of nests given in each case. The 
letter codes within the gray fields correspond to the specimens number 
(only last three digits shown; see full specimen number in <a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Table S1</a>) and the locality according to <a name="bfig1" href="#fig1" class="workspace-trigger">Fig. 1</a>. The shape of the fields indicates the species designation according to morphological assessment. (a) <em>Malvinius</em>, (b) <em>Falklandius</em> and (c) <em>Lissopterus</em>.</p></span></div></figure></div><div><p>There
 was high intra-specific variation in most species, as measured by 
nucleotide diversity when including all species (GMYC groups) with 
&gt;10 individuals, except for the two members of <em>Metius</em> (Carabidae) (<a name="bfig4" href="#fig4" class="workspace-trigger">Fig. 4</a>).
 The intra-specific variation was high even in species that were 
encountered in a small number of populations only, and the level of 
variation was not correlated with the number of sampled populations per 
species (<em>r</em>&nbsp;=&nbsp;−0.412, <em>p</em>&nbsp;=&nbsp;0.23, for <em>cox1</em> and <em>r</em>&nbsp;=&nbsp;−0.192, <em>p</em>&nbsp;=&nbsp;0.59 for <em>rrnL</em>;
 Pearson’s product-moment correlation). This confirms that diversity 
within populations is generally high relative to the total variation 
observed in a given species, and that this high diversity is not 
generally structured along geographically separated populations.</p><figure class="figure" id="fig4"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr4.jpg" alt="" height="306"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr4.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 4</span>. (a) Nucleotide diversity (<em>π</em>) in various species (GMYC groups), separate for <em>cox1</em> and <em>rrnL</em> genes. The number behind each species name indicates the number of sites at which this species was encountered.</p></span></div></figure></div></section><section id="aep-section-id21"><h3>3.3. Dating the age of species divergence and data partitioning</h3><div><p>We
 estimated divergence times between congeneric sister GMYC clusters (3 
pairs in the family Carabidae and 5 in Curculionidae) based on the 
insect mitochondrial clock rate of 2.3% divergence per 10<sup>6</sup> years (<a name="bbib6" href="#bib6" class="workspace-trigger">Brower, 1994</a>)
 and using either penalized likelihood on 2500 sampled trees from the 
stationary phase of each MrBayes run or an uncorrelated relaxed clock in
 BEAST. We used Bayes Factors or the ratios ln(Bayes Factor)/Δ<em>p</em> to select among five data partitioning schemes (Section <a name="bsec1" href="#sec1" class="workspace-trigger">2</a>).
 The estimated harmonic means of the sampled likelihood values were 
consistently lower for the non-partitioned datasets and improved every 
time we increased the number of partitions (and free parameters) (<a name="btbl3" href="#tbl3" class="workspace-trigger">Table 3</a>). However, using the recommendations of <a name="bbib51" href="#bib51" class="workspace-trigger">Pagel and Meade (2004)</a>
 requiring at least a 10LnL increase per additional free parameter 
before accepting a more complicated model, the P2 partition scheme (3rd 
codons versus all other sites) was favored for both datasets (Carabidae 
ln&nbsp;BF<sub>(P2:P1)</sub>&nbsp;=&nbsp;264.89 in MrBayes or 206.84 in BEAST; Curculionidae ln&nbsp;BF<sub>(P2:P1)</sub>&nbsp;=&nbsp;320.12 in MrBayes or 294.26 in BEAST; while in all cases ln&nbsp;BF/Δ<em>p</em><sub>P3:P2</sub>, ln&nbsp;BF/Δ<em>p</em><sub>P4:P2</sub>&nbsp;&lt;&nbsp;10).</p><div class="tables frame-topbot rowsep-0 colsep-0" id="tbl3"><div class="captions"><span><p><span class="label">Table 3</span>. 
Bayes factor comparisons for selection of partitioning scheme when using
 (a) MrBayes or (b) BEAST. Five alternative partitioning schemes (P0–P4)
 were compared as described in the text. Numbers in brackets indicate 
the total number of free parameters required for each partitioning 
scheme. Below the diagonals: ln(Bayes Factor) (estimated as the 
difference of the natural logarithms of the harmonic mean of the 
likelihoods between two partition schemes), above the diagonal: ln(Bayes
 Factor)/Δ<em>p</em> (where Δ<em>p</em>&nbsp;=&nbsp;difference in total 
number of free parameters between two partition schemes). Asterisks 
indicate the selected partition scheme in each case. See <a name="bapp1" href="#app1" class="workspace-trigger">Table S3</a> for likelihood values.</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left"></th><th scope="col" class="align-left">P0 (10)</th><th scope="col" class="align-left">P1 (21)</th><th scope="col" class="align-left">P2 (21)</th><th scope="col" class="align-left">P3 (32)</th><th scope="col" class="align-left">P4 (38)</th></tr></thead><tbody><tr class="valign-top"><td class="align-left" colspan="6">(a) MrBayes</td></tr><tr class="valign-top"><td class="align-left" colspan="6"><em>Carabidae</em></td></tr><tr class="valign-top"><td class="align-left">P0</td><td class="align-left">—</td><td class="align-left">7.57</td><td class="align-left">24.08</td><td class="align-left">40.80</td><td class="align-left">45.33</td></tr><tr class="valign-top"><td class="align-left">P1</td><td class="align-left">83.29</td><td class="align-left">—</td><td class="align-left">n/a</td><td class="align-left">33.23</td><td class="align-left">24.43</td></tr><tr class="valign-top"><td class="align-left">P2∗</td><td class="align-left">264.89</td><td class="align-left">264.89</td><td class="align-left">—</td><td class="align-left">9.14</td><td class="align-left">8.85</td></tr><tr class="valign-top"><td class="align-left">P3</td><td class="align-left">448.77</td><td class="align-left">365.48</td><td class="align-left">100.59</td><td class="align-left">—</td><td class="align-left">8.30</td></tr><tr class="valign-top"><td class="align-left">P4</td><td class="align-left">498.58</td><td class="align-left">415.29</td><td class="align-left">150.40</td><td class="align-left">49.81</td><td class="align-left">—</td></tr><tr class="valign-top"><td class="align-left" colspan="6"><br></td></tr><tr class="valign-top"><td class="align-left" colspan="6"><em>Curculionidae</em></td></tr><tr class="valign-top"><td class="align-left">P0</td><td class="align-left">—</td><td class="align-left">9.50</td><td class="align-left">38.60</td><td class="align-left">45.68</td><td class="align-left">50.33</td></tr><tr class="valign-top"><td class="align-left">P1</td><td class="align-left">104.49</td><td class="align-left">—</td><td class="align-left">n/a</td><td class="align-left">36.18</td><td class="align-left">40.83</td></tr><tr class="valign-top"><td class="align-left">P2∗</td><td class="align-left">424.61</td><td class="align-left">320.12</td><td class="align-left">—</td><td class="align-left">7.08</td><td class="align-left">7.59</td></tr><tr class="valign-top"><td class="align-left">P3</td><td class="align-left">502.46</td><td class="align-left">397.97</td><td class="align-left">77.85</td><td class="align-left">—</td><td class="align-left">8.53</td></tr><tr class="valign-top"><td class="align-left">P4</td><td class="align-left">553.66</td><td class="align-left">449.17</td><td class="align-left">129.05</td><td class="align-left">51.20</td><td class="align-left">—</td></tr><tr class="valign-top"><td class="align-left" colspan="6"><br></td></tr><tr class="valign-top"><td class="align-left" colspan="6">(b) BEAST</td></tr><tr class="valign-top"><td class="align-left" colspan="6"><em>Carabidae</em></td></tr><tr class="valign-top"><td class="align-left">P0</td><td class="align-left">—</td><td class="align-left">53.38</td><td class="align-left">72.18</td><td class="align-left">79.25</td><td class="align-left">82.88</td></tr><tr class="valign-top"><td class="align-left">P1</td><td class="align-left">587.16</td><td class="align-left">—</td><td class="align-left">n/a</td><td class="align-left">25.87</td><td class="align-left">19.09</td></tr><tr class="valign-top"><td class="align-left">P2∗</td><td class="align-left">794.00</td><td class="align-left">206.84</td><td class="align-left">—</td><td class="align-left">7.07</td><td class="align-left">6.92</td></tr><tr class="valign-top"><td class="align-left">P3</td><td class="align-left">871.78</td><td class="align-left">284.62</td><td class="align-left">77.78</td><td class="align-left">—</td><td class="align-left">6.64</td></tr><tr class="valign-top"><td class="align-left">P4</td><td class="align-left">911.63</td><td class="align-left">324.47</td><td class="align-left">117.63</td><td class="align-left">39.85</td><td class="align-left">—</td></tr><tr class="valign-top"><td class="align-left" colspan="6"><br></td></tr><tr class="valign-top"><td class="align-left" colspan="6"><em>Curculionidae</em></td></tr><tr class="valign-top"><td class="align-left">P0</td><td class="align-left">—</td><td class="align-left">66.25</td><td class="align-left">93.00</td><td class="align-left">99.79</td><td class="align-left">103.62</td></tr><tr class="valign-top"><td class="align-left">P1</td><td class="align-left">728.76</td><td class="align-left">—</td><td class="align-left">n/a</td><td class="align-left">33.54</td><td class="align-left">24.18</td></tr><tr class="valign-top"><td class="align-left">P2∗</td><td class="align-left">1023.02</td><td class="align-left">294.26</td><td class="align-left">—</td><td class="align-left">6.79</td><td class="align-left">6.87</td></tr><tr class="valign-top"><td class="align-left">P3</td><td class="align-left">1097.75</td><td class="align-left">368.99</td><td class="align-left">74.72</td><td class="align-left">—</td><td class="align-left">7.02</td></tr><tr class="valign-top"><td class="align-left">P4</td><td class="align-left">1139.84</td><td class="align-left">411.08</td><td class="align-left">116.82</td><td class="align-left">42.10</td><td class="align-left">—</td></tr></tbody></table></div></div></div><div><p>The
 partitioning scheme had a major impact on the estimated divergence 
times. Estimated mean node ages for key nodes in the trees of 
Curculionidae and Carabidae (<a name="bfig5" href="#fig5" class="workspace-trigger">Fig. 5</a>)
 were generally younger for the non-partitioned data set and the least 
favored partitioning scheme P1, while P2, P3 and P4 which had increased 
marginal likelihoods, gave older mean ages for the nodes of interest (<a name="bfig5" href="#fig5" class="workspace-trigger">Fig. 5</a>).
 Differences in node ages under the preferred partitioning schemes P2–P4
 were not substantial when using penalized likelihood on MrBayes trees (<a name="bfig5" href="#fig5" class="workspace-trigger">Fig. 5</a>a) but were much greater in the BEAST analyses (<a name="bfig5" href="#fig5" class="workspace-trigger">Fig 5</a>b).
 There was also a general trend of increased standard deviations for the
 deeper nodes. This analysis placed the youngest interspecific splits 
(between <em>Lissopterus</em> LIM1 and LIM2) at 1.15&nbsp;±&nbsp;0.35 Mya (or 1.27&nbsp;±&nbsp;0.56 using BEAST). Most other splits between sister GMYC groups within genera were dated to be ancient, at a minimum of 4.17&nbsp;±&nbsp;1.88 Mya (or 2.75&nbsp;±&nbsp;0.92 using BEAST) at the node separating <em>Malvinius MAM-1 and M. MAM-2</em>, and up to 18.3&nbsp;±&nbsp;7.7&nbsp;Mya (or 17.6&nbsp;±&nbsp;4.8 using BEAST) at the node separating <em>Antarctobius ANM-1</em> and <em>A. falklandius</em> (<a name="bfig6" href="#fig6" class="workspace-trigger">Fig. 6</a>), while the age of the endemic genus <em>Malvinius</em> was estimated to be 12.14&nbsp;±&nbsp;4.3&nbsp;Mya (or 10.21&nbsp;±&nbsp;3.24 using BEAST).</p><figure class="figure" id="fig5"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr5.jpg" alt="" height="589"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr5.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 5</span>. 
Dating of intra-generic splits using (a) MrBayes and r8s or (b) BEAST. 
Estimated mean ages for the MRCAs of 8 pairs of sister GMYC groups (3 
carabids and 5 curculionids) are shown under five alternative 
partitioning schemes (P0–P4). The error bars correspond to 1 standard 
deviation, calculated across all the sampled trees from the stationary 
phase. The values used in the graph are provided in <a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Table S2</a>.</p></span></div></figure><figure class="figure" id="fig6"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr6.jpg" alt="" height="840"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-gr6.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Fig. 6</span>. 
Clock-constrained trees of the families Carabidae (a) and Curculionidae 
(b) based on the P2 partitioning scheme (see text). The insect standard 
mtDNA rate of 2.3% divergence per My was applied using penalized 
likelihood. The bars correspond to 95% confidence intervals.</p></span></div></figure></div></section></section><section id="aep-section-id22"><h2>4. Discussion</h2><section id="aep-section-id23"><h3>4.1. Genetic variation and species delimitation</h3><p>The
 terrestrial biota of Antarctica and surrounding Southern Ocean islands 
remain poorly understood evolutionarily. Here we combined taxonomic and 
DNA-based evolutionary analyses of an entire fauna of Coleoptera that 
were encountered during extensive field collecting. Ground-dwelling 
beetles in the families predominant in these habitats are notoriously 
variable morphologically, generating difficulties for taxonomy. Yet, we 
found an overall close match of morphologically defined species with 
mtDNA-derived GMYC groups. Notable disagreements (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>a–c)
 were primarily due to the presence of closely related haplotypes in 
individuals considered to be different species on morphological grounds.
 In addition, tentative separation of groups based on slight 
morphological differences, e.g., in the case of <em>G. salebrosus</em> and “<em>G. nr. salebrosus</em>”
 were not reflected by separation in mtDNA. This seems to confirm that 
the GMYC method is conservative in recognizing separated entities, in 
particular when using the <em>rrnL</em> gene with its 3–5× lower rate of variation compared to <em>cox1</em> (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>), due to the requirement for monophyly and for large numbers of character changes on the subtending branches (<a name="bbib53" href="#bib53" class="workspace-trigger">Pons et al., 2006</a>). While the <em>rrnL</em> sequences appear to provide insufficient variation for the detection of a shift in branching rate, <em>cox1</em> or both genes combined produced a sufficient number of character changes for the application of the method.</p><p>While
 species delimitation based on mtDNA and morphology was in broad 
agreement, both character systems also agree on high intra-species 
variation. This seems to be due mostly to high within-population 
variation, as nucleotide diversity was high also at the level of 
individual collecting sites. However, haplotype distribution was highly 
structured geographically. For example considering the prolific site W1,
 sets of closely related haplotypes co-occur but this site also contains
 divergent haplotypes that are also present at select other sites, 
suggesting a composite nature of local populations. Site E4 also 
exhibited several cases of highly divergent haplotypes that each were 
encountered at different secondary sites (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>b
 and c), i.e., did not show an obvious isolation-by-distance pattern, 
but whose distribution is consistent with deep coalescence of local 
haplotypes and low gene flow within the Falklands. However, greater 
numbers of individuals per site and species will have to be obtained 
before it is possible to conduct formal analysis of 
isolation-by-distance or tests of matrix correlation of genetic and 
geographic structure.</p><p>The three cases of mtDNA–morphology incongruence also have to be seen in this context. Specifically, the <em>F. goliath</em>/<em>F. turbificatus</em> or <em>L. hyadesi</em>/<em>L. quadrinotatus</em>
 pairs showed identical or closely related haplotypes, in addition to 
more distantly related haplotypes in either species. This might be 
evidence of hybridization between these species and might indicate gene 
flow among populations generally, contrary to the above conclusions of <em>in situ</em>
 accumulation of local variation. However, secondary hybridization is 
not supported by the geographic distribution of the haplotypes in 
question. For example, the <em>L. hyadesi</em> haplotype shared with <em>L. quadrinotatus</em> was found at two distant localities in West and East Falklands, but the <em>L. quadrinotatus</em> individual that shared this haplotype did not co-occur at these sites (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>c).
 This distribution argues against a classical hybridization scenario of 
allopatric ranges affected by partial gene flow of mtDNA, as has been 
observed with ground-dwelling beetles elsewhere (<a name="bbib25" href="#bib25" class="workspace-trigger">Gómez-Zurita and Vogler, 2003</a>). This possibility is less easily ruled out in <em>Falklandius</em> because the haplotype shared by the <em>F. goliath/F. turbificatus</em> pair does in fact co-occur at a particular site (site W1; <a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>a)
 but distant haplotypes were also present at this site which would still
 favor a scenario of ancestral alleles persisting in this population. 
Hence, there is no immediate evidence for hybridization to cause the 
sharing of haplotypes between species. Finally, morphological characters
 currently used to delimit these species may have to be re-assessed, 
while nuclear marker will have to be sequenced as well.</p></section><section id="aep-section-id24"><h3>4.2. Phylogenetics and dating of Falkland lineages</h3><p>In
 addition to questions about species limits and population structure, 
DNA profiling of the Falkland beetles also allowed a preliminary 
phylogenetic analysis. Good nodal support was obtained mostly at the 
level of genera and GMYC groups, while major families were also 
recovered as monophyletic clades (<a name="bapp1" href="#app1" class="workspace-trigger">Supplementary Fig. S1</a>),
 including the Curculionidae and Carabidae that were used for analyses 
of dating. For the weevils, which have been intensely studied 
taxonomically and biogeographically (<a name="bbib46" href="#bib46" class="workspace-trigger">Morrone and Posadas, 2005</a>, <a name="bbib54" href="#bib54" class="workspace-trigger">Posadas, 2008</a>), our sampling appears to be largely complete at the species level, with a few caveats. In <em>Antarctobius</em> it was not clear if the four species encountered could be mapped correctly on the five species given in <a name="bbib54" href="#bib54" class="workspace-trigger">Posadas (2008)</a>. In addition, an unidentified larva (BMNH 668310) may be <em>Cylindrorhinus lemniscatus</em> based on its close relationship to <em>C. caudiculatus</em>. This leaves only <em>Lanteriella microphtalma</em>, and <em>Morronia brevirostris</em> unsampled, the latter being a rarely collected species that was described only recently (<a name="bbib57" href="#bib57" class="workspace-trigger">Posadas and Ocampo, 2001</a>).
 An unidentified adult (BMNH 673228) may correspond to either of them, 
unless our survey encountered a hitherto unknown species. The 
phylogenetic reconstruction of all haplotypes (<a name="bfig2" href="#fig2" class="workspace-trigger">Fig. 2</a>) or single representatives per GMYC group (<a name="bfig6" href="#fig6" class="workspace-trigger">Fig. 6</a>) using Bayesian analysis resulted in similar topologies producing strong support for the monophyly of genera (except for <em>Antarctobius</em> that was paraphyletic for <em>Germaniellus</em>, and <em>Metius</em>
 that was polyphyletic in the BEAST analysis using a single individual 
although monophyletic in the analysis of all individuals; <a name="bfig2" href="#fig2" class="workspace-trigger">Fig. 2</a>).</p><p>The
 topology and age estimates based on the Bayesian analyses are of great 
interest to establish the duration of independent lineage evolution in 
the Falklands. These analyses in future will benefit from the inclusion 
of mainland lineages, as the terminal branches unique to the Falklands 
and hence the inferred length of separation might be greatly reduced 
depending on the position of the mainland species in the trees. Only 
four species of Curculionidae encountered in this study (<em>F. antarcticus</em>, <em>Fa. suffodens</em>, <em>C. caudiculatus</em> and <em>H. albolimbata</em>)
 are shared with the South American mainland, while all others are 
endemic to the Falklands. This includes the three endemic genera <em>Malvinius</em>, <em>Morronia</em> and <em>Lanteriella</em>,
 with two, one and one species each. These are presumably ancient 
components with no close relatives on the mainland. In other genera, the
 Falkland species may have closely related mainland species, and 
therefore the age of Falkland endemic lineages may be younger than the 
branch points obtained in our analysis. Cladistic morphological analyses
 have shown that in most cases, including the genera <em>Puranius</em> (<a name="bbib43" href="#bib43" class="workspace-trigger">Morrone, 1994</a>), <em>Antarctobius</em> (<a name="bbib56" href="#bib56" class="workspace-trigger">Posadas and Morrone, 2004</a>, <a name="bbib42" href="#bib42" class="workspace-trigger">Morrone, 1992</a>) and <em>Falklandius</em> (<a name="bbib45" href="#bib45" class="workspace-trigger">Morrone and Anderson, 1995</a>), the Falkland species are not monophyletic. An exception was the monophyly of the three closely related <em>F. turbificatus</em>, <em>F. goliath</em> and <em>F. kusheli</em> which were not clearly distinguishable in our analysis (<a name="bfig3" href="#fig3" class="workspace-trigger">Fig. 3</a>a), but were phylogenetically widely separated from <em>F. antarcticus</em> that has its closest relatives on the mainland (<a name="bbib45" href="#bib45" class="workspace-trigger">Morrone and Anderson, 1995</a>).
 In the morphology-based studies the Falkland endemics generally occupy 
basal nodes in the respective genus-level trees, as sisters to 
species-rich mainland lineages. This pattern in the tree topologies also
 led to the conclusion from dispersal-vicariance analyses that the 
Falkland Islands are the biogeographic sister area to the southern South
 American Magellanic Moorlands&nbsp;+&nbsp;Magellanic Forests provinces (<a name="bbib55" href="#bib55" class="workspace-trigger">Posadas and Morrone, 2003</a>). These published trees do not necessarily conform to our results, e.g., in the genus <em>Puranius</em> where <a name="bbib43" href="#bib43" class="workspace-trigger">Morrone’s (1994)</a> study finds <em>P. scaber</em> as the sister to all other species and distantly related to <em>P. championi</em> and <em>P. exsculpticollis</em>, and not as sister to <em>P. championi</em>
 suggested by mtDNA. Yet, in these studies the Falkland groups generally
 occupy basal branches in the genus-level trees, reflecting ancient 
lineage splits. This supports the great divergence of the Falkland 
species from their mainland relatives and suggests that estimates of the
 age of the Falkland lineages are not likely to be greatly reduced from 
incorporation of species from other areas into the tree.</p><p>This 
leaves the difficult issue of calibrating the absolute node ages. In the
 absence of dating points for the Falkland insects we here applied a 
widely used clock calibration of 2.3% My<sup>−1</sup> following <a name="bbib6" href="#bib6" class="workspace-trigger">Brower (1994)</a>.
 Our recent re-evaluation of this estimate based on a set of 
co-distributed tenebrionid beetle lineages and a well-established 
geological event (<a name="bbib52" href="#bib52" class="workspace-trigger">Papadopoulou et al., submitted for publication</a>) revealed a slightly higher rate of sequence divergence (2.69% My<sup>−1</sup>)
 for the same combination of gene regions and substitution models as 
those used here, which is a composite of a much faster rate for <em>cox1</em> (3.38%) and lower for <em>rrnL</em> (1.09%). A survey of 30 recent studies of insect mtDNA calibrations (<a name="bbib52" href="#bib52" class="workspace-trigger">Papadopoulou et al., submitted for publication</a>)
 demonstrated the great effect of allowing for among-site rate 
heterogeneity in the substitution models, as well as the age of the 
calibration point (the mtDNA rate of variation is greatly increased for 
ages of &lt;1&nbsp;My). Although <a name="bbib6" href="#bib6" class="workspace-trigger">Brower’s (1994)</a>
 estimate did not correct for rate variation and the calibration nodes 
were very recent, the fortuitous combination of various parameters 
resulted in a widely accepted estimate that is consistent with many 
evolutionary scenarios using this divergence rate for external 
calibrations. In addition, the use of the GMYC model in order to select a
 single representative per species-level entity for the dating 
procedure, provides an objective measure for removing the presumably 
high intra-specific rate of variation from consideration.</p><p>Moreover,
 we specifically assessed the effect of data partitioning on node ages 
using likelihood comparisons and we found a general increase in 
estimated ages with the number of partitions as shown also by other 
recent studies (<a name="bbib52" href="#bib52" class="workspace-trigger">Papadopoulou et al., submitted for publication</a>, <a name="bbib58" href="#bib58" class="workspace-trigger">Poux et al., 2008</a>, <a name="bbib73" href="#bib73" class="workspace-trigger">Torres-Carvajal and de Queiroz, 2009</a>). Data partitioning is known to have a great impact on estimated branch lengths in Bayesian analysis (<a name="bbib35" href="#bib35" class="workspace-trigger">Marshall et al., 2006</a>).
 Allowing for rate variation using a gamma distribution within a single 
DNA partition (P0) or estimating a gamma distribution per gene region 
(P1) has been shown to result in lower likelihood than partitioning by 
codon positions (<a name="bbib5" href="#bib5" class="workspace-trigger">Brandley et al., 2005</a>, <a name="bbib8" href="#bib8" class="workspace-trigger">Castoe et al., 2005</a>, <a name="bbib40" href="#bib40" class="workspace-trigger">Miller et al., in press</a>).
 Accordingly, in the MrBayes analyses we found that separating the 3rd 
codon position from all other sites (P2) improved the likelihood 
considerably and at the same time changed dramatically the inferred node
 ages. In contrast, separating the 1st and the 2nd positions from the 
non-protein coding sites (P3) and from each other (P4) resulted in 
relatively smaller improvements of the harmonic mean and accordingly 
smaller differences in estimated times. Had we followed the widely used 
cut-off value of BF&nbsp;&gt;&nbsp;10 (<a name="bbib33" href="#bib33" class="workspace-trigger">Kass and Raftery, 1995</a>),
 even between models with different number of parameters, we would have 
preferred the most parameter-rich partitioning scheme (P4) but 
inferences about node ages would have not been considerably different. 
However, in the BEAST analyses increasing the number of partitions 
consistently increased the inferred node ages (<a name="bfig5" href="#fig5" class="workspace-trigger">Fig. 5</a>b), even if the differences in likelihood among P2–P4 were similarly small (<a name="bapp1" href="#app1" class="workspace-trigger">Table S3</a>).</p><p>Accepting
 the 2.3% divergence as a good approximation of the molecular rate, the 
preferred partitioning scheme suggests a great age of the Falkland 
lineages, both for nodes defining the three genus-level endemics but 
also for nodes within genera. For example, the genus <em>Malvinus</em> where the basal internal split in this lineage entirely confined to the Falklands was estimated to 4.17&nbsp;±&nbsp;1.9&nbsp;Mya and an even older age for the genus as a whole (12.14&nbsp;±&nbsp;4.3&nbsp;Mya; <a name="bfig6" href="#fig6" class="workspace-trigger">Fig. 6</a>).
 Therefore, despite uncertainties about the molecular age estimates of 
the Falkland endemic lineages and their separation from lineages present
 on the mainland, the molecular analysis combined with information from 
species distribution and existing cladistic analyses (<a name="bbib38" href="#bib38" class="workspace-trigger">McDowall, 2005</a>, <a name="bbib54" href="#bib54" class="workspace-trigger">Posadas, 2008</a>) supports an ancient origin of the Falkland lineages that likely preceded the Pleistocene glaciations.</p></section><section id="aep-section-id25"><h3>4.3. Biogeographic origin</h3><p>This raises the thorny question about the <em>in situ</em>
 persistence of the Falkland ancient endemics. Coleopterists studying 
subarctic assemblages in the Northern Hemisphere and their response to 
recent glacial cycles (<a name="bbib7" href="#bib7" class="workspace-trigger">Buckland and Hammond, 1997</a>, <a name="bbib16" href="#bib16" class="workspace-trigger">Coope, 1979</a>) have been reluctant to ascribe the presence of endemics to <em>in situ</em> evolution, due to the large range shifts apparent in specimens from subfossil layers, e.g., the presence of <em>Aphodius holderei</em> today only known from Tibet in Late Pleistocene deposits from Britain (<a name="bbib15" href="#bib15" class="workspace-trigger">Coope, 1973</a>). However, this perspective was questioned by the finding of Pleistocene speciation <em>in situ</em> in refugial areas in the Mediterranean where populations persisted without apparent range shifts (<a name="bbib60" href="#bib60" class="workspace-trigger">Ribera and Vogler, 2004</a>).
 In the Falklands, transport across ocean seems possible, and is likely 
to explain the distribution of non-endemics, such as the two carabid 
species in the genus <em>Metius</em> which are widespread predators of weevils, including pest species on pasture grasses elsewhere in South America (<a name="bbib1" href="#bib1" class="workspace-trigger">Ahmad, 1977</a>). Their history of recent immigration may also be reflected by the low nucleotide diversity (<a name="bfig4" href="#fig4" class="workspace-trigger">Fig. 4</a>).
 However, this is an unlikely scenario for the flightless, 
ground-dwelling endemics, including most of the weevils. A theoretical 
possibility is that these species were transported recently on drift 
wood that regularly piles up on the shores of the Falkland archipelago (<a name="bbib7" href="#bib7" class="workspace-trigger">Buckland and Hammond, 1997</a>),
 but this requires that these lineages have gone extinct in the mainland
 since dispersal to the Falklands. Alternatively, species known only 
from the Falklands may not yet have been observed on the mainland and 
therefore been mistaken as endemics, but this seems equally unlikely 
given the numerous records in the widespread species (<a name="bbib54" href="#bib54" class="workspace-trigger">Posadas, 2008</a>).
 Sampling from mainland localities for mtDNA analysis and age estimates 
is now an obvious priority to confirm these conclusions.</p><p>However, 
lineage persistence in the islands should be considered as an important 
factor of biogeographic history. Fossil pollen studies from a Miocene to
 Early Pliocene site revealed the presence of forests dominated by <em>Podocarpus</em> (conifers) and <em>Nothofagus</em> (southern beeches) on the islands that resembled today’s forests in southern South America and the southwest Pacific (<a name="bbib4" href="#bib4" class="workspace-trigger">Birnie and Roberts, 1986</a>, <a name="bbib34" href="#bib34" class="workspace-trigger">Macphail and Cantrill, 2006</a>),
 while forests may have existed as recently as the onset of the last 
series of glacial cycles a few hundred thousand years ago (<a name="bbib68" href="#bib68" class="workspace-trigger">Stone et al., 2005</a>). The islands currently experience a cool-temperate oceanic climate, with a narrow mean temperature range from 9.1&nbsp;°C in February to 2.2&nbsp;°C in July (<a name="bbib37" href="#bib37" class="workspace-trigger">McAdam, 1980</a>).
 This climate supports an altitudinal succession of native vegetation 
types growing on predominantly peaty soils, from coastal ‘tussac 
grasslands’ through varieties of ‘oceanic heath’, and up to montane 
‘feldmark’ communities (<a name="bbib17" href="#bib17" class="workspace-trigger">Davies and McAdam, 1989</a>, <a name="bbib41" href="#bib41" class="workspace-trigger">Moore, 1968</a>). Pollen records indicate that these vegetation types have been constant for at least 9.3&nbsp;ka (<a name="bbib3" href="#bib3" class="workspace-trigger">Barrow, 1978</a>), and are similar to those that were present ca. 36–28&nbsp;ka (<a name="bbib14" href="#bib14" class="workspace-trigger">Clark et al., 1998</a>), prior to the last period of periglacial conditions between ca. 26 and 13&nbsp;ka (<a name="bbib12" href="#bib12" class="workspace-trigger">Clapperton, 1990</a>).
 Although there was considerable cooling and drying during the glacials 
and the widely found peat soils are likely to have been formed only with
 warming climates at around 16.5&nbsp;ka (<a name="bbib77" href="#bib77" class="workspace-trigger">Wilson et al., 2002</a>), the islands were never covered by an ice sheet, except for small mountain top glaciers (<a name="bbib12" href="#bib12" class="workspace-trigger">Clapperton, 1990</a>, <a name="bbib13" href="#bib13" class="workspace-trigger">Clapperton and Sugden, 1976</a>, <a name="bbib39" href="#bib39" class="workspace-trigger">Mercer, 1983</a>).
 Similarly, botanists concluded from an early time of investigation that
 there is ‘no reason to suppose that the Falklands were destitute of 
vegetation’ (<a name="bbib67" href="#bib67" class="workspace-trigger">Skottsberg, 1913</a>; as cited in <a name="bbib38" href="#bib38" class="workspace-trigger">McDowall, 2005</a>). This leaves the distinct possibility that species of extant Coleoptera also persisted in pockets during the last glacial.</p><p>If a scenario of ancient lineage persistence <em>in situ</em>
 is accepted, the age estimates from the current analysis also set the 
stage for the time of origin of the Falkland biota. While any biotic 
trace of a South African geological connection has been lost (<a name="bbib38" href="#bib38" class="workspace-trigger">McDowall, 2005</a>),
 the connection with the South American continent may have been 
established at any time since the geographic proximity had been first 
established, and could have been through trans-oceanic dispersal or via 
temporary land bridges at times of low eustatic sea level (<a name="bbib46" href="#bib46" class="workspace-trigger">Morrone and Posadas, 2005</a>). Based on the dating, these events would be pre-Pleistocene and possibly much older (&gt;10&nbsp;Mya),
 quite in contrast to the fauna in comparable settings in the Northern 
Hemisphere, such as Iceland which lacks endemic beetle species (see <a name="bbib7" href="#bib7" class="workspace-trigger">Buckland and Hammond, 1997</a>). Based on the basal phylogenetic position of the Falkland species with respect to the Patagonian relatives (<a name="bbib46" href="#bib46" class="workspace-trigger">Morrone and Posadas, 2005</a>),
 interchange with the mainland may have been infrequent and at an early 
stage of the existence of genus-level taxa. The current mtDNA database 
will be easily extended for tests of relationships among subantarctic 
lineages and improved evolutionary scenarios of mainland connections and
 their age, including the link to other subantarctic islands, e.g., 
testing the multiregional origin of the insect faunas of these islands 
versus the ‘Paninsulantarctica’ hypothesis of a common origin of the 
South Atlantic island fauna (<a name="bbib44" href="#bib44" class="workspace-trigger">Morrone, 1998</a>).
 These evolutionary reconstructions, combined with genetic profiling of 
populations in this postglacial landscape, will aid the biogeographic 
and paleoclimatic understanding of biotic responses to temperature 
change.</p></section></section></div><section id="aep-acknowledgment-id26"><h2>Acknowledgments</h2><p>This
 work was funded by the NHM Museum Research Fund (to A.P.V. and P.M.H.) 
and Darwin Initiative of the UK, in collaboration with Falklands 
Conservation, London. We are grateful to Jesus Gomez-Zurita for help 
with haplotype network analysis. Two anonymous reviewers made invaluable
 comments on an earlier version of the manuscript.</p></section><div class="Appendices"><section id="app1"><h2>Appendix A. Supplementary data</h2><p><span class="display"><figure class="figure" id="aep-figure-id27"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-fx1.jpg" alt="" height="3771"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-fx1.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Supplementary data</span>. Bayesian tree of all 283 individuals, terminals labelled according to the GMYC groups.</p></span></div></figure></span><span class="display"><figure class="figure" id="aep-figure-id29"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-fx2.jpg" alt="" height="1064"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-fx2.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span><p><span class="label">Supplementary data</span>. Supplementary Figure S2.</p></span></div></figure></span><span class="display"><div class="e-component e-component-mmc1" id="aep-e-component-id31"><div class="article-attachment"><a class="icon-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-mmc1.pdf" title="Download Acrobat PDF file (95KB)" target="_blank"><svg focusable="false" viewBox="0 0 95 128" width="17.8125" height="24" class="icon icon-pdf-download"><path d="m82 108h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06h46.4v4e1h1e1v-5e1h-56.4c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-18h-1e1v8zm-25.94-39.4c-0.84-0.38-1.84-0.56-2.98-0.56h-9.04v23.6h5.94v-9h2.18c2.48 0 4.36-0.62 5.66-1.84s1.92-3.06 1.92-5.5c0-1.04-0.12-2-0.4-2.88-0.26-0.88-0.66-1.64-1.22-2.3s-1.22-1.14-2.06-1.52zm-3.12 8.94c-0.4 0.46-0.98 0.7-1.74 0.7h-1.22v-5.76h1.22c1.56 0 2.34 0.96 2.34 2.88 0 0.98-0.2 1.72-0.6 2.18zm21.84-8.52c-0.96-0.66-2.32-0.98-4.06-0.98h-8.72v23.6h8.72c1.74 0 3.1-0.32 4.06-0.98s1.7-1.52 2.18-2.62 0.78-2.34 0.88-3.76 0.16-2.9 0.16-4.44-0.06-3.02-0.16-4.44-0.4-2.68-0.88-3.76c-0.48-1.1-1.2-1.96-2.18-2.62zm-2.78 14.66c-0.06 0.96-0.18 1.72-0.38 2.24s-0.48 0.88-0.84 1.04-0.86 0.24-1.48 0.24h-1.32v-14.74h1.32c0.62 0 1.12 0.08 1.48 0.24s0.64 0.52 0.84 1.04 0.32 1.28 0.38 2.24c0.06 0.98 0.08 2.24 0.08 3.84s-0.02 2.9-0.08 3.86zm13.98-6.58v-4.02h7.74v-5.04h-13.7v23.6h5.96v-9.72h7.26v-4.82z"></path></svg></a><a class="download-link" target="_blank" download="" title="Download Acrobat PDF file (95KB)" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790309003480-mmc1.pdf">Download Acrobat PDF file (95KB)</a><a class="help-link right" title="Help with pdf files (Opens in new window)" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank">Help with pdf files</a></div><div class="captions"><span><p><span class="label">Supplementary data</span>. Supplementary Tables S1–S3.</p></span></div></div></span></p></section></div></div><div class="related-content-links u-hide-from-md"><button class="button button-anchor" type="button"><span class="button-text">Recommended articles</span></button><button class="button button-anchor" type="button"><span class="button-text">Citing articles (<!-- -->24<!-- -->)</span></button></div><div class="Tail"></div><section class="bibliography" id="aep-bibliography-id33"><h2 class="section-title">References</h2><section class="bibliography-sec" id="aep-bibliography-sec-id34"><dl class="references"><dt class="label"><a href="#bbib1" id="ref-id-bib1">Ahmad, 1977</a></dt><dd class="reference"><div class="contribution">R. Ahmad<strong class="title">On the knowledge of <em>Hyperodes bonariensis</em> Kuschel (Col., Curculionidae) and its antagonists in Argentina</strong></div><div class="host">Anz. Schaedlingskde., Pflanzenschutz, Umweltschutz, 50 (1977), pp. 150-151</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1007/BF02173708">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-34250294113&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib2" id="ref-id-bib2">Avise, 2000</a></dt><dd class="reference"><div class="contribution">J.C. Avise<strong class="title">Phylogeography: The History and Formation of Species</strong></div><div class="host">Harvard University Press, Cambridge, MA (2000)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib3" id="ref-id-bib3">Barrow, 1978</a></dt><dd class="reference"><div class="contribution">C.J. Barrow<strong class="title">Post-glacial pollen diagrams from South Georgia (Sub-Antarctic) and West Falkland Island (South-Atlantic)</strong></div><div class="host">J. Biogeogr., 5 (1978), p. 251</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.2307/3038040">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0001105592&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib4" id="ref-id-bib4">Birnie and Roberts, 1986</a></dt><dd class="reference"><div class="contribution">J.F. Birnie, D.E. Roberts<strong class="title">Evidence of Tertiary forest in the Falkland Islands (Islas Malvinas)</strong></div><div class="host">Palaeogeogr. Palaeoclimatol. Palaeoecol., 55 (1986), pp. 45-53</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/0031018286901367">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/0031018286901367/pdf?md5=51c456236a95d1b337ba857661baeca6&amp;pid=1-s2.0-0031018286901367-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0022858202&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib5" id="ref-id-bib5">Brandley et al., 2005</a></dt><dd class="reference"><div class="contribution">M.C. Brandley, A. Schmitz, T.W. Reeder<strong class="title">Partitioned Bayesian analyses, partition choice, and the phylogenetic relationships of scincid lizards</strong></div><div class="host">Syst. Biol., 54 (2005), pp. 373-390</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/10635150590946808">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-9944255059&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib6" id="ref-id-bib6">Brower, 1994</a></dt><dd class="reference"><div class="contribution">A.V.Z. Brower<strong class="title">Rapid morphological radiation and convergence among races of the butterfly <em>Heliconius erato</em> inferred from patterns of mitochondrial DNA evolution</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 91 (1994), pp. 6491-6495</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1073/pnas.91.14.6491">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0028303111&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib7" id="ref-id-bib7">Buckland and Hammond, 1997</a></dt><dd class="reference"><div class="contribution">P.C. Buckland, P.M. Hammond<strong class="title">The origins of the biota of the Falkland Islands and South Georgia</strong></div><div class="host">Quatern. Proc., 5 (1997), pp. 59-69</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib8" id="ref-id-bib8">Castoe et al., 2005</a></dt><dd class="reference"><div class="contribution">T.A. Castoe, M.M. Sasa, C.L. Parkinson<strong class="title">Modeling
 nucleotide evolution at the mesoscale: the phylogeny of the Neotropical
 pitvipers of the Porthidium group (Viperidae: Crotalinae)</strong></div><div class="host">Mol. Phylogenet. Evol., 37 (2005), pp. 881-898</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S105579030500196X">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S105579030500196X/pdfft?md5=1b54bb842bf990047c1766fe1182e021&amp;pid=1-s2.0-S105579030500196X-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-27744467658&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib9" id="ref-id-bib9">Chown, 1994</a></dt><dd class="reference"><div class="contribution">S.L. Chown<strong class="title">Historical ecology of Sub-Antarctic weevils (Coleoptera, Curculionidae)—patterns and processes on isolated islands</strong></div><div class="host">J. Nat. Hist., 28 (1994), pp. 411-433</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/00222939400770191">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0028165268&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib10" id="ref-id-bib10">Chown and Convey, 2007</a></dt><dd class="reference"><div class="contribution">S.L. Chown, P. Convey<strong class="title">Spatial and temporal variability across life’s hierarchies in the terrestrial Antarctic</strong></div><div class="host">Philos. Trans. R. Soc. B, 362 (2007), pp. 2307-2331</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1098/rstb.2006.1949">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-39149099180&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib11" id="ref-id-bib11">Chown et al., 2008</a></dt><dd class="reference"><div class="contribution">S.L. Chown, J.E. Lee, J.D. Shaw<strong class="title">Conservation of Southern Ocean Islands: invertebrates as exemplars</strong></div><div class="host">J. Insect Conserv., 12 (2008), pp. 277-291</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1007/s10841-008-9151-8">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-44949260051&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib12" id="ref-id-bib12">Clapperton, 1990</a></dt><dd class="reference"><div class="contribution">C.M. Clapperton<strong class="title">Quaternary glaciations in the Southern Ocean and Antarctic Peninsula area</strong></div><div class="host">Quatern. Sci. Rev., 9 (1990), pp. 229-252</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/027737919090020B">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/027737919090020B/pdf?md5=f73133a0f6f6e84e1ea69c6d6eebcdc2&amp;pid=1-s2.0-027737919090020B-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0025662112&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib13" id="ref-id-bib13">Clapperton and Sugden, 1976</a></dt><dd class="reference"><div class="contribution">C.M. Clapperton, D.E. Sugden<strong class="title">Maximum extent of glaciers in part of West Falkland</strong></div><div class="host">J. Glaciol., 17 (1976), pp. 73-77</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1017/S0022143000030732">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0000949499&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib14" id="ref-id-bib14">Clark et al., 1998</a></dt><dd class="reference"><div class="contribution">R. Clark, U.M. Huber, P. Wilson<strong class="title">Late Pleistocene sediments and environmental change at Plaza Creek, Falkland Islands, South Atlantic</strong></div><div class="host">J. Quatern. Sci., 13 (1998), pp. 95-105</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1002/%28SICI%291099-1417%28199803/04%2913:2%3C95::AID-JQS351%3E3.0.CO;2-G">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0031816985&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib100" id="ref-id-bib100">Clement et al., 2000</a></dt><dd class="reference"><div class="contribution">M. Clement, D. Posada, K.A. Crandall<strong class="title">TCS: a computer program to estimate gene genealogies</strong></div><div class="host">Mol. Ecol., 9 (2000), pp. 1657-1659</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1046/j.1365-294x.2000.01020.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033771358&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib15" id="ref-id-bib15">Coope, 1973</a></dt><dd class="reference"><div class="contribution">G.R. Coope<strong class="title">Tibetan species of dung beetle from Late Pleistocene deposits in England</strong></div><div class="host">Nature, 245 (1973), pp. 335-336</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1038/245335a0">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0015896026&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib16" id="ref-id-bib16">Coope, 1979</a></dt><dd class="reference"><div class="contribution">G.R. Coope<strong class="title">Late Cenozoic fossil Coleoptera: evolution, biogeography, and ecology</strong></div><div class="host">Annu. Rev. Ecol. Syst., 10 (1979), pp. 249-267</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib17" id="ref-id-bib17">Davies and McAdam, 1989</a></dt><dd class="reference"><div class="contribution">T.H. Davies, J.H. McAdam<strong class="title">Wild Flowers of the Falkland Islands</strong></div><div class="host">Bluntisham Books, Bluntisham (1989)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib18" id="ref-id-bib18">Drummond et al., 2006</a></dt><dd class="reference"><div class="contribution">A.J. Drummond, S.Y.W. Ho, M.J. Phillips, A. Rambaut<strong class="title">Relaxed phylogenetics and dating with confidence</strong></div><div class="host">PLoS Biol., 4 (2006), pp. 699-710</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33344465800&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib19" id="ref-id-bib19">Drummond and Rambaut, 2007</a></dt><dd class="reference"><div class="contribution">A.J. Drummond, A. Rambaut<strong class="title">BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div class="host">BMC Evol. Biol., 7 (2007)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib20" id="ref-id-bib20">Ernsting, 1993</a></dt><dd class="reference"><div class="contribution">G. Ernsting<strong class="title">Observations on life cycle and feeding ecology of 2 recently introduced predatory beetle species at South Georgia, Sub-Antarctic</strong></div><div class="host">Polar Biol., 13 (1993), pp. 423-428</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1007/BF01681985">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0027846350&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib21" id="ref-id-bib21">Excoffier et al., 2005</a></dt><dd class="reference"><div class="contribution">L. Excoffier, G. Laval, S. Schneider<strong class="title">Arlequin ver. 3.0: an integrated software package for population genetics data analysis</strong></div><div class="host">Evol. Bioinf. Online, 1 (2005), pp. 47-50</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-36348960851&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib22" id="ref-id-bib22">Fontaneto et al., 2007</a></dt><dd class="reference"><div class="contribution">D. Fontaneto, E.A. Herniou, C. Boschetti, M. Caprioli, G. Melone, C. Ricci, T.G. Barraclough<strong class="title">Independently evolving species in asexual bdelloid rotifers</strong></div><div class="host">PLoS Biol., 5 (2007), pp. 914-921</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-34247401269&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib23" id="ref-id-bib23">Gaston et al., 2003</a></dt><dd class="reference"><div class="contribution">K.J. Gaston, A.G. Jones, C. Hanel, S.L. Chown<strong class="title">Rates of species introduction to a remote oceanic island</strong></div><div class="host">Proc. R. Soc. B, 270 (2003), pp. 1091-1098</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1098/rspb.2003.2332">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0037853407&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib24" id="ref-id-bib24">Gressitt, 1970</a></dt><dd class="reference"><div class="contribution">J.L. Gressitt<strong class="title">Subantarctic entomology and biogeography</strong></div><div class="host">Pac. Insects Monogr., 23 (1970), pp. 295-374</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0001738541&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib25" id="ref-id-bib25">Gómez-Zurita and Vogler, 2003</a></dt><dd class="reference"><div class="contribution">J. Gómez-Zurita, A.P. Vogler<strong class="title">Incongruent nuclear and mitochondrial phylogeographic patterns in the <em>Timarcha goettingensis</em> species complex (Coleoptera, Chrysomelidae)</strong></div><div class="host">J. Evol. Biol., 16 (2003), pp. 833-843</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1046/j.1420-9101.2003.00599.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0042740734&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib26" id="ref-id-bib26">Hall, 1990</a></dt><dd class="reference"><div class="contribution">K.J. Hall<strong class="title">Quaternary glaciations in the Southern Ocean: sector 0-degrees long.–180-degrees long.</strong></div><div class="host">Quatern. Sci. Rev., 9 (1990), pp. 217-228</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/0277379190900197">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/0277379190900197/pdf?md5=b4218f65627c6876250fadc2aba6b895&amp;pid=1-s2.0-0277379190900197-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0025584060&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib27" id="ref-id-bib27">Hebert and Gregory, 2005</a></dt><dd class="reference"><div class="contribution">P.D.N. Hebert, T.R. Gregory<strong class="title">The promise of DNA barcoding for taxonomy</strong></div><div class="host">Syst. Biol., 54 (2005), pp. 852-859</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/10635150500354886">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-31344432889&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib28" id="ref-id-bib28">Hewitt, 1999</a></dt><dd class="reference"><div class="contribution">G.M. Hewitt<strong class="title">Post-glacial re-colonization of European biota</strong></div><div class="host">Biol. J. Linn. Soc., 68 (1999), pp. 87-112</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S0024406699903322">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S0024406699903322/pdf?md5=20f61880edd8d36d043e24c3cc169c22&amp;pid=1-s2.0-S0024406699903322-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1006/bijl.1999.0332">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0032822582&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib29" id="ref-id-bib29">Howland and Hewitt, 1995</a></dt><dd class="reference"><div class="contribution">D.E. Howland, G.M. Hewitt<strong class="title">Phylogeny of the Coleoptera based on mitochondrial cytochrome oxidase I sequence data</strong></div><div class="host">Insect Mol. Biol., 4 (1995), pp. 203-215</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1111/j.1365-2583.1995.tb00026.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0029347201&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib30" id="ref-id-bib30">Hudson, 1991</a></dt><dd class="reference"><div class="contribution">R.R. Hudson<strong class="title">Gene genealogies and the coalescent process</strong></div><div class="host">Oxf. Surv. Evol. Biol., 7 (1991), pp. 1-44</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-85006298907&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib101" id="ref-id-bib101">Hunt et al., 2007</a></dt><dd class="reference"><div class="contribution">T. Hunt, J. Bergsten, Z. Levkanicova, A. Papadopoulou, O. St-John, R. Wild, P.M. Hammond, D. Ahrens, M. Balke, M.S. Caterino, J. Gómez-Zurita, I. Ribera, T.G. Barraclough, M. Bocakova, L. Bocak, A.P. Vogler<strong class="title">A comprehensive phylogeny of beetles reveals the evolutionary origins of a superradiation</strong></div><div class="host">Science, 318 (2007), pp. 1913-1916</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1126/science.1146954">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-37549050969&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib31" id="ref-id-bib31">Jones, 2004</a></dt><dd class="reference"><div class="contribution">A.G. Jones<strong class="title">Insects of the Falkland Islands</strong></div><div class="host">Falklands Conservation, London (2004)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib32" id="ref-id-bib32">Jones et al., 2003</a></dt><dd class="reference"><div class="contribution">A.G. Jones, S.L. Chown, P.G. Ryan, N.J.M. Gremmen, K.J. Gaston<strong class="title">A review of conservation threats on Gough Island: a case study for terrestrial conservation in the Southern Oceans</strong></div><div class="host">Biol. Conserv., 113 (2003), pp. 75-87</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S0006320702003518">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S0006320702003518/pdfft?md5=d9483f64d95b7206f2de51e88c195111&amp;pid=1-s2.0-S0006320702003518-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0038687016&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib33" id="ref-id-bib33">Kass and Raftery, 1995</a></dt><dd class="reference"><div class="contribution">R.E. Kass, A.E. Raftery<strong class="title">Bayes Factors</strong></div><div class="host">J. Am. Stat. Assoc., 90 (1995), pp. 773-795</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/01621459.1995.10476572">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84950934893&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib34" id="ref-id-bib34">Macphail and Cantrill, 2006</a></dt><dd class="reference"><div class="contribution">M. Macphail, D.J. Cantrill<strong class="title">Age and implications of the Forest Bed, Falkland Islands, southwest Atlantic Ocean: evidence from fossil pollen and spores</strong></div><div class="host">Palaeogeogr. Palaeoclimatol. Palaeoecol., 240 (2006), pp. 602-629</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S0031018206001611">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S0031018206001611/pdfft?md5=d5bc4e118393b3845e152efc0d252c55&amp;pid=1-s2.0-S0031018206001611-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33748919454&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib35" id="ref-id-bib35">Marshall et al., 2006</a></dt><dd class="reference"><div class="contribution">D.C. Marshall, C. Simon, T.R. Buckley<strong class="title">Accurate
 branch length estimation in partitioned Bayesian analyses requires 
accommodation of among-partition rate variation and attention to branch 
length priors</strong></div><div class="host">Syst. Biol., 55 (2006), pp. 993-1003</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/10635150601087641">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33845446297&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib36" id="ref-id-bib36">Marshall, 1994</a></dt><dd class="reference"><div class="contribution">J.E.A. Marshall<strong class="title">The Falkland Islands—a key element in Gondwana paleogeography</strong></div><div class="host">Tectonics, 13 (1994), pp. 499-514</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1029/93TC03468">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0028254127&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib37" id="ref-id-bib37">McAdam, 1980</a></dt><dd class="reference"><div class="contribution">J.H. McAdam<strong class="title">Tatter flags and climate in the Falkland Islands</strong></div><div class="host">Weather, 35 (1980), pp. 321-327</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1002/j.1477-8696.1980.tb04713.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84979124921&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib38" id="ref-id-bib38">McDowall, 2005</a></dt><dd class="reference"><div class="contribution">R.M. McDowall<strong class="title">Falkland Islands biogeography: converging trajectories in the South Atlantic Ocean</strong></div><div class="host">J. Biogeogr., 32 (2005), pp. 49-62</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-12744279691&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib39" id="ref-id-bib39">Mercer, 1983</a></dt><dd class="reference"><div class="contribution">J.H. Mercer<strong class="title">Cenozoic glaciation in the Southern Hemisphere</strong></div><div class="host">Annu. Rev. Earth Planet. Sci., 11 (1983), pp. 99-132</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1146/annurev.ea.11.050183.000531">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0020973270&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib40" id="ref-id-bib40">Miller et al., in press</a></dt><dd class="reference"><span>Miller,
 K.B., Bergsten, J., Whiting, M.F., in press. Phylogeny and 
classification of the tribe Hydaticini (Coleoptera: Dytiscidae): 
partition choice for Bayesian analysis with multiple nuclear and 
mitochondrial protein-coding genes. Zool. Scr. <a href="https://doi.org/10.1111/j.1463-6409.2009.00393.x" target="_blank">doi:10.1111/j.1463-6409.2009.00393.x</a>.</span></dd><dt class="label"><a href="#bbib41" id="ref-id-bib41">Moore, 1968</a></dt><dd class="reference"><span>Moore, D.M., 1968. The vascular flora of the Falkland Islands. British Antarctic Survey Reports, No. 60.</span></dd><dt class="label"><a href="#bbib42" id="ref-id-bib42">Morrone, 1992</a></dt><dd class="reference"><div class="contribution">J.J. Morrone<strong class="title">Revisión sistemática y análisis cladísitico del género <em>Antarctobius</em> Fairmaire (Coleoptera: Curculionidae)</strong></div><div class="host">Neotropica, 38 (1992), pp. 3-20</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0003131730&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib43" id="ref-id-bib43">Morrone, 1994</a></dt><dd class="reference"><div class="contribution">J.J. Morrone<strong class="title">Systematics,
 cladistics, and biogeography of the Andean weevil genera Macrostyphlus,
 Adioristidius, Puranius, and Amathynetoides, new genus (Coleoptera: 
Curculionidae)</strong></div><div class="host">Am. Mus. Novit., 3104 (1994)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib44" id="ref-id-bib44">Morrone, 1998</a></dt><dd class="reference"><div class="contribution">J.J. Morrone<strong class="title">On Udvardy’s Insulantarctica province: a test from the weevils (Coleoptera: Curculionoidea)</strong></div><div class="host">J. Biogeogr., 25 (1998), pp. 947-955</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1046/j.1365-2699.1998.00240.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0032434794&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib45" id="ref-id-bib45">Morrone and Anderson, 1995</a></dt><dd class="reference"><div class="contribution">J.J. Morrone, R.S. Anderson<strong class="title">The <em>Falklandius</em> generic group: cladistic analysis with description of new taxa (Coleoptera: Curculionidae: Rhytirrhinini)</strong></div><div class="host">Am. Mus. Novit., 3121 (1995)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib46" id="ref-id-bib46">Morrone and Posadas, 2005</a></dt><dd class="reference"><div class="contribution">J.J. Morrone, P. Posadas<strong class="title">Falklands: facts and fiction—reply</strong></div><div class="host">J. Biogeogr., 32 (2005), pp. 2183-2187</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1111/j.1365-2699.2005.01378.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33745330302&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib47" id="ref-id-bib47">Nee et al., 1992</a></dt><dd class="reference"><div class="contribution">S. Nee, A.O. Mooers, P.H. Harvey<strong class="title">Tempo and mode of evolution revealed from molecular phylogenies</strong></div><div class="host">Proc. Natl. Acad. Sci. USA, 89 (1992), pp. 8322-8326</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1073/pnas.89.17.8322">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0026756206&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib48" id="ref-id-bib48">Nylander, 2004</a></dt><dd class="reference"><div class="contribution">J.A.A. Nylander<strong class="title">MrModeltest v2. Program Distributed by the Author</strong></div><div class="host">Evolutionary Biology Centre, Uppsala University (2004)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib49" id="ref-id-bib49">Oldfield, 1987</a></dt><dd class="reference"><div class="contribution">S. Oldfield<strong class="title">Fragments of Paradise: A Guide for Conservation in the UK Dependant Territories</strong></div><div class="host">British Association of Nature Conservationists, Oxford (1987)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib50" id="ref-id-bib50">Oldfield and Sheppard, 1997</a></dt><dd class="reference"><div class="contribution">S. Oldfield, C. Sheppard<strong class="title">Conservation of biodiversity and research needs in the UK dependent territories</strong></div><div class="host">J. Appl. Ecol., 34 (1997), pp. 1111-1121</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.2307/2405225">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0030771016&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib51" id="ref-id-bib51">Pagel and Meade, 2004</a></dt><dd class="reference"><div class="contribution">M. Pagel, A. Meade<strong class="title">A phylogenetic mixture model for detecting pattern-heterogeneity in gene sequence or character-state data</strong></div><div class="host">Syst. Biol., 53 (2004), pp. 571-581</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/10635150490468675">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-4243140058&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib52" id="ref-id-bib52">Papadopoulou et al., submitted for publication</a></dt><dd class="reference"><span>Papadopoulou,
 A., Anastasiou, I., Vogler, A.P., submitted for publication. Revisiting
 the insect mitochondrial molecular clock: the mid-Aegean trench 
calibration. Mol. Biol. Evol..</span></dd><dt class="label"><a href="#bbib53" id="ref-id-bib53">Pons et al., 2006</a></dt><dd class="reference"><div class="contribution">J. Pons, T.G. Barraclough, J. Gomez-Zurita, A. Cardoso, D.P. Duran, S. Hazell, S. Kamoun, W.D. Sumlin, A.P. Vogler<strong class="title">Sequence-based species delimitation for the DNA taxonomy of undescribed insects</strong></div><div class="host">Syst. Biol., 55 (2006), pp. 595-609</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/10635150600852011">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-34548444940&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib54" id="ref-id-bib54">Posadas, 2008</a></dt><dd class="reference"><div class="contribution">P. Posadas<strong class="title">A
 preliminar overview of species composition and geographical 
distribution of Malvinian weevils (Insecta: Coleoptera: Curculionidae)</strong></div><div class="host">Zootaxa (2008), pp. 1-26</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-39749115639&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib55" id="ref-id-bib55">Posadas and Morrone, 2003</a></dt><dd class="reference"><div class="contribution">P. Posadas, J.J. Morrone<strong class="title">Biogeografía histórica de la familia Curculionidae (Coleoptera) en las subregiones Subantártica y Chilena Central</strong></div><div class="host">Rev. Soc. Entomol. Argent., 62 (2003), pp. 75-84</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-30844443024&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib56" id="ref-id-bib56">Posadas and Morrone, 2004</a></dt><dd class="reference"><div class="contribution">P. Posadas, J.J. Morrone<strong class="title">A new species of <em>Antarctobius</em> Fairmaire from Islas Malvinas (Coleoptera: Curculionidae: Cyclominae)</strong></div><div class="host">Insect Syst. Evol., 35 (2004), pp. 353-359</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1163/187631204788920194">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-10944242430&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib57" id="ref-id-bib57">Posadas and Ocampo, 2001</a></dt><dd class="reference"><div class="contribution">P. Posadas, F.C. Ocampo<strong class="title"><em>Morronia brevirostris</em>, a new genus and species of Entimini (Coleoptera: Curculionidae: Entiminae) from Islas Malvinas</strong></div><div class="host">Insect Syst. Evol., 32 (2001), pp. 147-153</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1163/187631201X00100">CrossRef</a></div></dd><dt class="label"><a href="#bbib58" id="ref-id-bib58">Poux et al., 2008</a></dt><dd class="reference"><div class="contribution">C. Poux, O. Madsen, J. Glos, W.W. de Jong, M. Vences<strong class="title">Molecular
 phylogeny and divergence times of Malagasy tenrecs: influence of data 
partitioning and taxon sampling on dating analyses</strong></div><div class="host">BMC Evol. Biol., 8 (2008)</div><div class="comment">Article No. 102</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib59" id="ref-id-bib59">Procter and Fleming, 1999</a></dt><dd class="reference"><div class="contribution">D. Procter, L.V. Fleming<strong class="title">Biodiversity: The UK Overseas Territories</strong></div><div class="host">Joint Nature Conservation Committee, Peterborough (1999)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib60" id="ref-id-bib60">Ribera and Vogler, 2004</a></dt><dd class="reference"><div class="contribution">I. Ribera, A.P. Vogler<strong class="title">Speciation of Iberian diving beetles in Pleistocene refugia (Coleoptera, Dytiscidae)</strong></div><div class="host">Mol. Ecol., 13 (2004), pp. 179-193</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1046/j.1365-294X.2003.02035.x">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-1642565349&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib61" id="ref-id-bib61">Ritz et al., 2008</a></dt><dd class="reference"><div class="contribution">M.S. Ritz, C. Millar, G.D. Miller, R.A. Phillips, P. Ryan, V. Sternkopf, D. Liebers-Helbig, H.U. Peter<strong class="title">Phylogeography
 of the southern skua complex-rapid colonization of the southern 
hemisphere during a glacial period and reticulate evolution</strong></div><div class="host">Mol. Phylogenet. Evol., 49 (2008), pp. 292-303</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S1055790308003680">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S1055790308003680/pdfft?md5=aea58ea8c9d09e8078cc591307cfb7b8&amp;pid=1-s2.0-S1055790308003680-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-52249117216&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib62" id="ref-id-bib62">Robinson, 1984</a></dt><dd class="reference"><div class="contribution">G. Robinson<strong class="title">Insects of the Falkland Islands</strong></div><div class="host">British Museum (Natural History), London (1984)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib63" id="ref-id-bib63">Ronquist and Huelsenbeck, 2003</a></dt><dd class="reference"><div class="contribution">F. Ronquist, J.P. Huelsenbeck<strong class="title">MrBayes 3: Bayesian phylogenetic inference under mixed models</strong></div><div class="host">Bioinformatics, 19 (2003), pp. 1572-1574</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1093/bioinformatics/btg180">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0041386108&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib64" id="ref-id-bib64">Sanderson, 2003</a></dt><dd class="reference"><div class="contribution">M.J. Sanderson<strong class="title">R8s: inferring absolute rates of molecular evolution, divergence times in the absence of a molecular clock</strong></div><div class="host">Bioinformatics, 19 (2003), pp. 301-302</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1093/bioinformatics/19.2.301">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0037314098&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib65" id="ref-id-bib65">Savolainen et al., 2005</a></dt><dd class="reference"><div class="contribution">V. Savolainen, R.S. Cowan, A.P. Vogler, G. Roderick, R. Lane<strong class="title">Towards writing the encyclopaedia of life: an introduction to DNA barcoding</strong></div><div class="host">Philos. Trans. R. Soc. B, 360 (2005), pp. 1805-1811</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1098/rstb.2005.1730">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-33645760502&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib66" id="ref-id-bib66">Simon et al., 1994</a></dt><dd class="reference"><div class="contribution">C. Simon, F. Frati, A. Beckenbach, B. Crespi, H. Liu, P. Flook<strong class="title">Evolution,
 weighting, and phylogenetic utility of mitochondrial gene sequences and
 a compilation of conserved polymerase chain reaction primers</strong></div><div class="host">Ann. Entomol. Soc. Am., 87 (1994), pp. 651-701</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1093/aesa/87.6.651">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0028251187&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib67" id="ref-id-bib67">Skottsberg, 1913</a></dt><dd class="reference"><div class="contribution">C. Skottsberg<strong class="title">Botanische
 Ergebnisse der Schwedischen Expedition nach Patagonien und dem 
Feuerland 1907–1909: III. A botanical survey of the Falkland Islands</strong></div><div class="host">Kungliga Svenska Vetenskapsakademiens Handlingar, 50 (1913), pp. 1-120</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-19544381347&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib68" id="ref-id-bib68">Stone et al., 2005</a></dt><dd class="reference"><div class="contribution">P. Stone, D. Aldiss, E. Edwards<strong class="title">Rocks and Fossils of the Falkland Islands</strong></div><div class="host">British Geological Survey for Department of Mineral Resources, Falkland Islands Government (2005)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib69" id="ref-id-bib69">Storey et al., 1999</a></dt><dd class="reference"><div class="contribution">B.C. Storey, M.L. Curtis, J.K. Ferris, M.A. Hunter, R.A. Livermore<strong class="title">Reconstruction and break-out model for the Falkland Islands within Gondwana</strong></div><div class="host">J. Afr. Earth Sci., 29 (1999), pp. 153-163</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S089953629900086X">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S089953629900086X/pdf?md5=20a3c03ecbd8002cfc651044241c577d&amp;pid=1-s2.0-S089953629900086X-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0033169156&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib70" id="ref-id-bib70">Templeton et al., 1987</a></dt><dd class="reference"><div class="contribution">A.R. Templeton, E. Boerwinkle, C.F. Sing<strong class="title">A
 cladistic analysis of phenotypic associations with haplotypes inferred 
from restriction endonuclease mapping: I. Basic theory and an analysis 
of alcohol dehydrogenase activity in <em>Drosophila</em></strong></div><div class="host">Genetics, 117 (1987), pp. 343-351</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0023425198&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib71" id="ref-id-bib71">Templeton and Sing, 1993</a></dt><dd class="reference"><div class="contribution">A.R. Templeton, C.F. Sing<strong class="title">A
 cladistic analysis of phenotypic associations with haplotypes inferred 
from restriction endonuclease mapping: 4. Nested analyses with cladogram
 uncertainty and recombination</strong></div><div class="host">Genetics, 134 (1993), pp. 659-669</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0027278852&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib72" id="ref-id-bib72">Thompson et al., 1994</a></dt><dd class="reference"><div class="contribution">J.D. Thompson, D.G. Higgins, T.J. Gibson<strong class="title">CLUSTAL
 W: improving the sensitivity of progressive multiple sequence alignment
 through sequence weighting, position specific gap penalties and weight 
matrix choice</strong></div><div class="host">Nucleic Acids Res., 22 (1994), pp. 4673-4680</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1093/nar/22.22.4673">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0001420679&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib73" id="ref-id-bib73">Torres-Carvajal and de Queiroz, 2009</a></dt><dd class="reference"><div class="contribution">O. Torres-Carvajal, K. de Queiroz<strong class="title">Phylogeny of hoplocercine lizards (Squamata: Iguania) with estimates of relative divergence times</strong></div><div class="host">Mol. Phylogenet. Evol., 50 (2009), pp. 31-43</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S1055790308004776">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S1055790308004776/pdfft?md5=aa976dec43dba78e1645bad182dad93b&amp;pid=1-s2.0-S1055790308004776-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-57749209369&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib74" id="ref-id-bib74">Trewin et al., 2002</a></dt><dd class="reference"><div class="contribution">N.H. Trewin, D.I.M. MacDonald, C.G.C. Thomas<strong class="title">Stratigraphy
 and sedimentology of the Permian of the Falkland Islands: 
lithostratigraphic and palaeoenvironmental links with South Africa</strong></div><div class="host">J. Geol. Soc., 159 (2002), pp. 5-19</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0036140451&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib75" id="ref-id-bib75">UKDTCF, 1996</a></dt><dd class="reference"><div class="contribution">UKDTCF<strong class="title">UK Dependant Territories: A Conservation Review</strong></div><div class="host">UK Dependant Territories Conservation Forum, Leighton Buzzard, UK (1996)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib76" id="ref-id-bib76">Wakeley, 2006</a></dt><dd class="reference"><div class="contribution">J. Wakeley<strong class="title">Coalescent Theory: An Introduction</strong></div><div class="host">Roberts &amp; Co., Greenwood Village, Colorado (2006)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bbib77" id="ref-id-bib77">Wilson et al., 2002</a></dt><dd class="reference"><div class="contribution">P. Wilson, R. Clark, J. Birnie, D.M. Moore<strong class="title">Late
 Pleistocene and Holocene landscape evolution and environmental change 
in the Lake Sulivan area, Falkland Islands, South Atlantic</strong></div><div class="host">Quatern. Sci. Rev., 21 (2002), pp. 1821-1840</div><div class="ReferenceLinks" id="reference-links"><a class="link" href="https://www.sciencedirect.com/science/article/pii/S0277379102000082">Article</a><a class="anchor pdf link" href="https://www.sciencedirect.com/science/article/pii/S0277379102000082/pdfft?md5=5aec63e9c145bc2f36fdce2d1868573a&amp;pid=1-s2.0-S0277379102000082-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0036720680&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib78" id="ref-id-bib78">Yang and Yoder, 2003</a></dt><dd class="reference"><div class="contribution">Z.H. Yang, A.D. Yoder<strong class="title">Comparison
 of likelihood and Bayesian methods for estimating divergence times 
using multiple gene loci and calibration points, with application to a 
radiation of cute-looking mouse lemur species</strong></div><div class="host">Syst. Biol., 52 (2003), pp. 705-716</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://doi.org/10.1080/10635150390235557">CrossRef</a><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-0242571576&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd><dt class="label"><a href="#bbib79" id="ref-id-bib79">Yule, 1924</a></dt><dd class="reference"><div class="contribution">G.U. Yule<strong class="title">A mathematical theory of evolution based on the conclusions of Dr. J.C. Willis, FRS</strong></div><div class="host">Philos. Trans. R. Soc. B, 213 (1924), pp. 21-87</div><div class="ReferenceLinks" id="reference-links"><a class="link" target="_blank" rel="noopener noreferrer" href="https://www.scopus.com/inward/record.url?eid=2-s2.0-84948192729&amp;partnerID=10&amp;rel=R3.0.0">View Record in Scopus</a></div></dd></dl></section></section><div class="Copyright"><span class="copyright-line">Copyright © 2009 Elsevier Inc. All rights reserved.</span></div></article><div class="u-show-from-md col-lg-6 col-md-8 pad-right"><aside class="RelatedContent" aria-label="Related content"><section class="SidePanel"><header id="recommended-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Recommended articles</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317300143"><h3 class="article-title ellipsis" id="recommended-articles-article0-title" title="Phylogeographic structure of Canthon cyanellus (Coleoptera: Scarabaeidae), a Neotropical dung beetle in the Mexican Transition Zone: Insights on its origin and the impacts of Pleistocene climatic fluctuations on population dynamics"><span>Phylogeographic structure of <em>Canthon cyanellus</em>
 (Coleoptera: Scarabaeidae), a Neotropical dung beetle in the Mexican 
Transition Zone: Insights on its origin and the impacts of Pleistocene 
climatic fluctuations on population dynamics</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 109, 2017, pp. 180-190</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317300143/pdfft?md5=df98551581a52c951214ddcb077990ae&amp;pid=1-s2.0-S1055790317300143-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article0-title" aria-controls="recommended-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790308004284"><h3 class="article-title ellipsis" id="recommended-articles-article1-title" title="Phylogeography of the bark beetle Dendroctonus mexicanus Hopkins (Coleoptera: Curculionidae: Scolytinae)"><span>Phylogeography of the bark beetle <em>Dendroctonus mexicanus</em> Hopkins (Coleoptera: Curculionidae: Scolytinae)</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 49, Issue 3, 2008, pp. 930-940</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790308004284/pdfft?md5=d71c8baa064becfb7605415caccd86c9&amp;pid=1-s2.0-S1055790308004284-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article1-title" aria-controls="recommended-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790309002620"><h3 class="article-title ellipsis" id="recommended-articles-article2-title" title="Phylogeography of the frog Leptodactylus validus (Amphibia: Anura): Patterns and timing of colonization events in the Lesser Antilles"><span>Phylogeography of the frog <em>Leptodactylus validus</em> (Amphibia: Anura): Patterns and timing of colonization events in the Lesser Antilles</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 53, Issue 2, 2009, pp. 571-579</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790309002620/pdfft?md5=949f61a0e329324fc58cdced161d3377&amp;pid=1-s2.0-S1055790309002620-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article2-title" aria-controls="recommended-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article2" aria-hidden="true"></div></li></ul></div><div class="pagination u-position-relative u-padding-xs-bottom"><span class="u-position-absolute"></span><span class="pagination-pages-label"><span class="pagination-nav u-margin-xs-hor pagination-current underline-page-number">1</span><span class="pagination-nav u-margin-xs-hor">2</span></span><span class="u-position-absolute"><button class="button-link button-link-secondary next-button" type="button"><span class="button-link-text">Next</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></button></span></div></div></section><section class="SidePanel"><header id="citing-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle button-link-primary" aria-expanded="false" type="button"><span class="button-link-text"><h2 class="section-title">Citing articles (24)</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="u-display-none" aria-hidden="true" aria-describedby="citing-articles-header"><div id="citing-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790314003133" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article0-title" title="A comprehensive multilocus phylogeny of the Neotropical cotingas (Cotingidae, Aves) with a comparative evolutionary analysis of breeding system and plumage dimorphism and a revised phylogenetic classification">A
 comprehensive multilocus phylogeny of the Neotropical cotingas 
(Cotingidae, Aves) with a comparative evolutionary analysis of breeding 
system and plumage dimorphism and a revised phylogenetic classification</h3></a><div class="article-source ellipsis">2014, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790314003133/pdfft?md5=fdf2771c157bddc28c266c1204c2a475&amp;pid=1-s2.0-S1055790314003133-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article0-title" aria-controls="citing-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790311000625" target="_self"><h3 class="article-title ellipsis" id="citing-articles-article1-title" title="Deep mtDNA subdivision within Linnean species in an endemic radiation of tiger beetles from New Zealand (genus Neocicindela)">Deep mtDNA subdivision within Linnean species in an endemic radiation of tiger beetles from New Zealand (genus Neocicindela)</h3></a><div class="article-source ellipsis">2011, Molecular Phylogenetics and Evolution</div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790311000625/pdfft?md5=bb99c348fe3b3e6702363be96d96b59f&amp;pid=1-s2.0-S1055790311000625-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article1-title" aria-controls="citing-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://doi.org/10.1007/s10682-018-9938-3" target="_blank"><h3 class="article-title ellipsis" id="citing-articles-article2-title" title="Echoes of the whispering land: interacting roles of vicariance and selection in shaping the evolutionary divergence of two Calceolaria (Calceolariaceae) species from Patagonia and Malvinas/Falkland Islands">Echoes
 of the whispering land: interacting roles of vicariance and selection 
in shaping the evolutionary divergence of two Calceolaria 
(Calceolariaceae) species from Patagonia and Malvinas/Falkland Islands</h3></a><div class="article-source ellipsis">2018, Evolutionary Ecology</div></div><div class="buttons"><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="citing-articles-article2-title" aria-controls="citing-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="citing-articles-article2" aria-hidden="true"></div></li></ul><a class="anchor side-panel-view-more" href="http://www.scopus.com/scopus/inward/citedby.url?partnerID=10&amp;rel=3.0.0&amp;eid=2-s2.0-74949130463&amp;md5=a2fe55499337c1cf24eae5a079f0912b" target="_blank"><span class="anchor-text">View more articles</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></div></div></section><section class="SidePanel hidden"><header id="metrics-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Article Metrics</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><a href="https://plu.mx/plum/a/?doi=10.1016/j.ympev.2009.08.027" class="plumx-summary plum-sciencedirect-theme" data-pass-hidden-categories="true" data-hide-usage="true" data-orientation="vertical" data-hide-print="true" data-site="plum" data-on-success="onMetricsWidgetSuccess">View article metrics</a></div></section></aside></div></div><div></div></div><div id="footer"><div class="hor-line" style="border-color:#e9711c"></div><div class="panel-s u-padding-l-bottom u-bg-white u-clr-grey7" role="contentinfo"><a class="anchor move-left els-footer-elsevier anchor-has-inherit-color" href="https://www.elsevier.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg viewBox="-3345 3440.027 140.01 24.333" style="width:104px;height:30px"><title>Elsevier</title><path id="E" style="fill:#E9711C" d="M-3343.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3344L-3343.999,3461.698"></path><path style="fill:#E9711C" d="M-3325.448,3461.698c2.074-0.118,2.83-0.502,2.83-2.832v-13.511c0-2.33-0.757-2.715-2.83-2.832v-0.591h8.884 v0.591c-2.243,0-3.027,0.472-3.027,2.891v13.009c0,1.652,0.056,2.625,1.71,2.625h4.008c3,0,4.4-2,5.576-4.1l0.673,0.118 l-1.71,5.222h-16.114V3461.698"></path><path style="fill:#E9711C" d="M-3307.122,3456.27h0.561c1.12,2.596,2.886,5.4,5.94,5.4c2,0,3.672-1.3,3.672-3.334 c0-1.652-1.626-3.1-4.176-4.927c-3.28-2.36-5.41-3.746-5.41-6.43c0-3.422,2.55-5.517,5.633-5.517c2.214,0,3,0.944,4.204,0.944 c0.476,0,0.56-0.266,0.476-0.737h0.561l0.645,6.076h-0.561c-0.785-2.625-2.523-5.19-5.354-5.19c-1.737,0-3.138,1.356-3.138,3.185 c0,1.918,1.933,3.157,5.016,5.016c2.523,1.504,5,3.362,5,6.254c0,3.245-2.608,5.752-5.97,5.752c-2.02,0-4.148-0.855-4.68-0.855 c-0.336,0-0.7,0.207-0.756,0.649h-0.56l-1.094-6.282"></path><path style="fill:#E9711C" d="M-3293.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3294L-3293.999,3461.698"></path><path style="fill:#E9711C" d="M-3265.839,3462.524h-0.42l-5.41-12.538c-0.896-2.065-1.71-4.16-2.83-6.136 c-0.478-0.826-1.346-1.327-2.3-1.327v-0.591h8.323v0.591c-0.785,0-2.354,0-2.354,1.15c0,0.384,0.87,2.45,1.653,4.308l4.063,9.676 l4.877-11.359c0.588-1.356,0.757-2.094,0.757-2.713c0-0.618-0.673-0.974-2.13-1.062v-0.59h5.941v0.591 c-0.337,0.06-0.7,0.117-1.037,0.295c-1.066,0.56-2.13,3.57-2.635,4.749l-6.5,14.957"></path><path style="fill:#E9711C" d="M-3255.472,3461.698c2.24,0,3.025-0.473,3.025-2.892v-13.393c0-2.42-0.784-2.89-3.025-2.891v-0.59h9.078v0.591 c-2.24,0-3.025,0.472-3.025,2.891v13.393c0,2.42,0.784,2.892,3.025,2.892v0.59h-9.08v-0.59"></path><path id="E_2_" style="fill:#E9711C" d="M-3244.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3245L-3244.999,3461.698"></path><path style="fill:#E9711C" d="M-3206,3461.698c-1.26-0.354-1.71-0.68-2.466-1.623l-6.166-7.609c3.027-0.65,5.13-2.185,5.13-5.547 c0-4.75-4.26-4.986-7.764-4.986h-9.191v0.591c2.24,0,3.026,0.472,3.026,2.891v13.393c0,2.42-0.785,2.892-3.026,2.892v0.59h9.08 v-0.59c-2.242,0-3.027-0.473-3.027-2.892v-5.604h2.551l7.314,9.086h4.54L-3206,3461.698 M-3220.399,3444.499 c0-1.387,0.337-1.476,2.186-1.476c2.774,0,5.3,0.974,5.3,4.308c0,3.6-2.887,4.63-5.914,4.631h-1.569v-7.463H-3220.399z"></path></svg></span></a><div class="panel-s u-bg-white u-padding-0-hor-from-xs u-padding-s-hor-from-md u-padding-xs-ver text-xs u-clear-both-from-xs u-clear-none-from-md"><p class="u-margin-xs-bottom"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/solutions/sciencedirect" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">About ScienceDirect</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science/activateaccess" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Remote access</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science?_ob=ShoppingCartURL&amp;_method=display&amp;md5=3ff44acb300f01481824c54a2973d019" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Shopping cart</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Contact and support</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Terms and conditions</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/privacy-policy" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Privacy policy</span></a></p><p id="els-footer-cookie-message">Cookies are used by this site. For more information, visit the<!-- --> <a class="anchor u-margin-0-right" href="https://www.elsevier.com/solutions/sciencedirect/support/cookies" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">cookies page</span></a>.</p><p id="els-footer-copyright">Copyright © <!-- -->2018<!-- --> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</p></div><div><span style="margin-top:-10px" class="u-position-relative move-bottom u-float-left-from-xs u-float-right-from-md move-right u-padding-0-hor"><a class="anchor els-footer-relx anchor-has-inherit-color" href="https://www.relx.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg style="width:118px;height:28px" viewBox="-3248 3424.938 334.44 53.422"><title>RELX Group</title><path style="fill:#E9711C" d="M-3227.285,3448.329c10.948,0,24.15-2.576,24.15-13.124c0-7.474-7.63-10.267-14.6-10.267 c-10.647,0-30.265,5.926-30.265,26.96c0,10.555,8.034,16.643,20.778,16.643c13.474,0,23.266-8.768,26.26-20.535 c-7.912,13.648-17.972,17.896-26.18,17.896c-10.19,0-14.077-7.568-14.077-14.05c0-18.56,14.406-24.798,23.494-24.798 c6.135,0,9.727,3.936,9.727,8.253c0,11.918-17.17,11.578-25.008,11.578c-0.803,0-3.274,0.02-3.76-0.003l-1.654,1.714 c3.59,0.508,8.554,1.497,13.944,4.223c11.204,5.656,21.745,20.59,35.7,20.59c5.062,0,6.38-1.543,7.335-2.893 c-6.834,4.354-18.286-3.197-24.46-9.04c-5.082-4.806-9.57-10.26-21.387-13.146"></path><g><path style="fill:#666666" d="M-3158.73,3468.502c-0.423,0-0.636-0.164-0.794-0.547l-6.885-14.511h-6.197c-0.21,0-0.316,0.11-0.316,0.33 v14.182c0,0.328-0.213,0.547-0.532,0.547h-5.189c-0.32,0-0.53-0.22-0.53-0.547v-36.139c0-0.33,0.21-0.55,0.53-0.549h13.928 c6.62,0,11.492,4.545,11.492,11.117c0,4.873-2.7,8.65-6.833,10.238l7.575,15.167c0.21,0.385,0,0.713-0.37,0.713L-3158.73,3468.502 L-3158.73,3468.502z M-3159.42,3442.384c0-3.232-2.223-5.31-5.506-5.311h-7.68c-0.21,0-0.316,0.108-0.316,0.328v9.912 c0,0.217,0.104,0.328,0.316,0.328h7.68C-3161.644,3447.641-3159.42,3445.561-3159.42,3442.384"></path><path style="fill:#666666" d="M-3146.95,3431.817c0-0.33,0.21-0.55,0.53-0.549h21.865c0.317,0,0.53,0.22,0.53,0.549v4.709 c0,0.33-0.212,0.548-0.53,0.548h-15.827c-0.212,0-0.318,0.108-0.318,0.328v9.089c0,0.22,0.106,0.328,0.318,0.328h12.726 c0.317,0,0.53,0.22,0.53,0.547v4.709c0,0.33-0.213,0.548-0.53,0.548h-12.726c-0.212,0-0.318,0.11-0.318,0.329v9.418 c0,0.22,0.106,0.328,0.318,0.328h15.827c0.317,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.212,0.547-0.53,0.547h-21.865 c-0.32,0-0.53-0.22-0.53-0.547C-3146.95,3467.956-3146.95,3431.817-3146.95,3431.817z"></path><path style="fill:#666666" d="M-3118.2,3431.817c0-0.33,0.212-0.55,0.53-0.549h5.191c0.317,0,0.53,0.22,0.53,0.549v30.553 c0,0.22,0.106,0.328,0.317,0.328h14.722c0.318,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.213,0.547-0.53,0.547h-20.76 c-0.317,0-0.53-0.22-0.53-0.547C-3118.2,3467.956-3118.2,3431.817-3118.2,3431.817z"></path><path style="fill:#666666" d="M-3052.46,3449.885c0-6.46,0.423-8.926,1.006-10.787c1.747-5.53,5.878-8.377,11.758-8.377 c5.88,0,9.532,3.012,11.28,6.9c0.107,0.274,0.107,0.548-0.21,0.712l-1.695,0.876c-0.265,0.11-0.53,0.055-0.688-0.22 c-1.802-3.396-4.45-5.312-8.74-5.312c-4.45,0-7.468,2.192-8.792,6.3c-0.475,1.422-0.845,3.722-0.845,9.909s0.37,8.486,0.845,9.91 c1.324,4.107,4.343,6.3,8.792,6.3c4.342,0,7.468-2.137,8.79-6.19c0.477-1.422,0.85-3.56,0.85-7.117c0-0.22-0.106-0.328-0.318-0.328 h-8.155c-0.318,0-0.53-0.22-0.53-0.548v-1.807c0-0.33,0.212-0.548,0.53-0.548h10.962c0.317,0,0.53,0.22,0.53,0.548v2.243 c0,3.725-0.423,6.572-0.954,8.27c-1.694,5.53-5.88,8.432-11.65,8.432c-5.88,0-10.01-2.848-11.758-8.38 C-3052.036,3458.811-3052.459,3456.345-3052.46,3449.885"></path><path style="fill:#666666" d="M-3021.43,3468.502c-0.317,0-0.53-0.22-0.53-0.547v-25.023c0-0.33,0.213-0.548,0.53-0.548h1.801 c0.316,0,0.53,0.22,0.53,0.548v3.338h0.052c0.953-2.572,3.23-4.434,6.78-4.434c2.013,0,3.866,0.712,5.084,1.807 c0.265,0.164,0.317,0.438,0.107,0.713l-1.06,1.531c-0.212,0.274-0.48,0.274-0.795,0.11c-1.164-0.767-2.436-1.313-4.024-1.313 c-4.45,0-6.144,3.996-6.144,8.815v14.456c0,0.328-0.213,0.547-0.53,0.547H-3021.43L-3021.43,3468.502z"></path><path style="fill:#666666" d="M-3004.54,3462.26c-0.53-1.752-0.847-3.668-0.847-6.844c0-3.12,0.316-5.037,0.847-6.79 c1.378-4.327,4.818-6.79,9.426-6.79c4.662,0,8.104,2.464,9.48,6.79c0.53,1.752,0.848,3.668,0.848,6.79 c0,3.176-0.317,5.092-0.848,6.844c-1.377,4.326-4.818,6.79-9.48,6.79C-2999.722,3469.05-3003.162,3466.585-3004.54,3462.26 M-2988.387,3461.33c0.475-1.48,0.688-3.066,0.688-5.914c0-2.794-0.213-4.38-0.688-5.86c-1.007-3.12-3.442-4.873-6.728-4.873 c-3.23,0-5.666,1.752-6.672,4.873c-0.477,1.48-0.688,3.065-0.688,5.86c0,2.848,0.212,4.436,0.688,5.914 c1.006,3.12,3.442,4.873,6.672,4.873C-2991.829,3466.203-2989.395,3464.451-2988.387,3461.33"></path><path style="fill:#666666" d="M-2963.66,3468.502c-0.317,0-0.528-0.22-0.528-0.547v-3.066h-0.053c-1.272,2.574-3.92,4.162-7.36,4.162 c-5.615,0-8.74-3.616-8.74-9.913v-16.206c0-0.33,0.213-0.548,0.53-0.548h1.801c0.317,0,0.53,0.22,0.53,0.548v15.44 c0,5.257,2.118,7.83,6.515,7.83c3.812,0,6.78-2.74,6.78-7.284v-15.987c0-0.33,0.21-0.548,0.528-0.548h1.802 c0.317,0,0.53,0.22,0.53,0.548v25.023c0,0.328-0.213,0.547-0.53,0.547h-1.802L-2963.66,3468.502z"></path><path style="fill:#666666" d="M-2955.29,3478.36c-0.317,0-0.53-0.22-0.53-0.549v-34.879c0-0.33,0.213-0.548,0.53-0.548h1.8 c0.317,0,0.53,0.22,0.53,0.548v3.176h0.053c1.218-2.465,3.496-4.272,7.68-4.272c4.45,0,7.256,2.08,8.58,6.242 c0.638,2.137,0.9,4.435,0.9,7.393c0,2.9-0.264,5.2-0.9,7.336c-1.323,4.164-4.13,6.242-8.58,6.242c-4.183,0-6.46-1.807-7.68-4.27 h-0.053v13.031c0,0.33-0.213,0.55-0.53,0.549L-2955.29,3478.36L-2955.29,3478.36z M-2939.35,3461.602 c0.53-1.643,0.688-3.832,0.688-6.13c0-2.355-0.158-4.545-0.688-6.188c-0.952-2.957-3.124-4.6-6.46-4.6 c-3.178,0-5.507,1.533-6.46,4.6c-0.477,1.423-0.69,3.34-0.69,6.188c0,2.847,0.212,4.71,0.69,6.13 c0.954,3.067,3.283,4.602,6.46,4.602C-2942.473,3466.204-2940.3,3464.561-2939.35,3461.602"></path><path style="fill:#666666" d="M-3070.48,3468.545c-0.424,0-0.634-0.164-0.85-0.547l-7.254-12.649h-0.106l-7.307,12.649 c-0.213,0.383-0.424,0.547-0.85,0.547h-5.72c-0.37,0-0.528-0.328-0.318-0.711l10.76-18.562l-9.96-17.248 c-0.21-0.385-0.05-0.712,0.32-0.712h5.721c0.424,0,0.636,0.163,0.847,0.548l6.513,11.278h0.106l6.513-11.278 c0.213-0.385,0.425-0.548,0.85-0.548h5.72c0.37,0,0.53,0.327,0.317,0.712l-9.956,17.248l10.75,18.562 c0.21,0.383,0.052,0.71-0.32,0.711h-5.771H-3070.48z"></path><path style="fill:#666666" d="M-2932.08,3443.951c-0.108,0-0.18-0.075-0.18-0.189v-10.385c0-0.075-0.036-0.113-0.11-0.113h-3.202 c-0.11,0-0.184-0.073-0.184-0.186v-1.6c0-0.11,0.074-0.186,0.184-0.186h8.746c0.11,0,0.183,0.075,0.183,0.186v1.6 c0,0.113-0.07,0.186-0.183,0.186h-3.201c-0.073,0-0.11,0.038-0.11,0.113v10.385c0,0.113-0.07,0.19-0.18,0.189H-2932.08 L-2932.08,3443.951z"></path><path style="fill:#666666" d="M-2924.47,3431.478c0-0.112,0.07-0.186,0.178-0.186h1.604c0.163,0,0.252,0.056,0.307,0.186l3.367,7.8h0.07 l3.315-7.8c0.052-0.13,0.144-0.186,0.305-0.186h1.583c0.11,0,0.182,0.074,0.182,0.186v12.286c0,0.11-0.07,0.186-0.182,0.186h-1.564 c-0.11,0-0.18-0.075-0.18-0.186v-7.744h-0.073l-2.593,5.957c-0.07,0.168-0.18,0.242-0.343,0.242h-1.044 c-0.16,0-0.27-0.074-0.343-0.242l-2.59-5.957h-0.072v7.744c0,0.11-0.072,0.186-0.18,0.186h-1.567c-0.106,0-0.178-0.075-0.178-0.186 v-12.286H-2924.47z"></path></g></svg></span></a></span></div></div></div></section></div></div></div>
<script type="application/json" data-iso-key="_0">{"userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36 Zotero/5.0","abstracts":{"content":[{"#name":"abstract","$":{"xmlns:ce":true,"class":"author","lang":"en","id":"aep-abstract-id11"},"$$":[{"#name":"section-title","_":"Abstract"},{"#name":"abstract-sec","$":{"id":"aep-abstract-sec-id12"},"$$":[{"#name":"simple-para","$":{"view":"all"},"$$":[{"#name":"__text__","_":"The Falkland biota are generally considered to be derived from the nearest continental source in Patagonian South America, yet they harbor many endemic species whose taxonomy and evolutionary history remains insufficiently understood. Comprehensive sampling of Coleoptera over two field seasons from numerous sites across the Falkland archipelago produced representatives of 55 morphologically separable species, assigned to 35 genera and 13 families of Coleoptera. Partial mitochondrial Cytochrome Oxidase subunit I and 16S ribosomal RNA genes were sequenced for 283 individuals. These sequences formed 55 clusters under a Yule-Coalescent model that largely conformed to Linnean species while deep-level phylogenetic relationships were broadly congruent with the higher level classification. Detailed analysis of the most diverse families Carabidae and Curculionidae addressed the question about the age and persistence "},{"#name":"italic","_":"in situ"},{"#name":"__text__","_":" of Falkland biota, showing that separation of sister species within genera based on molecular clock estimates pre-dated the Pleistocene in all cases. Intra-specific diversity of mtDNA haplotypes and nucleotide diversity were high in most species, while intra-population variation was equally high and showed local differentiation of populations, but there was no isolation-by-distance relationship. Taken together, these observations indicate that ancient endemics are unlikely to be due to the recent establishment from a source elsewhere, but have persisted "},{"#name":"italic","_":"in situ"},{"#name":"__text__","_":". The observed patterns differ greatly from those in climatically similar areas of the Northern Hemisphere. They do not support the view that postglacial ranges of insects near the limits of former glaciations are merely the result of redistribution due to changing climate."}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"biographies":{},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:ja":true,"xmlns":true,"class":"keyword","id":"aep-keywords-id13"},"$$":[{"#name":"section-title","_":"Keywords"},{"#name":"keyword","$$":[{"#name":"text","_":"Southern Ocean islands"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Species delimitation"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Phylogeny"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Historical biogeography"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Molecular clock"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Dispersal"}]},{"#name":"keyword","$$":[{"#name":"text","_":"Climate change"}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"experiments":{},"rawtext":"","authors":{"content":[{"#name":"author-group","$":{"xmlns:ce":true},"$$":[{"#name":"author","$":{"id":"aep-author-id7"},"$$":[{"#name":"given-name","_":"Anna"},{"#name":"surname","_":"Papadopoulou"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"author","$":{"id":"aep-author-id8"},"$$":[{"#name":"given-name","_":"Alex G."},{"#name":"surname","_":"Jones"},{"#name":"cross-ref","$":{"refid":"aff3"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"aep-author-id9"},"$$":[{"#name":"given-name","_":"Peter M."},{"#name":"surname","_":"Hammond"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"id":"aep-author-id10"},"$$":[{"#name":"given-name","_":"Alfried P."},{"#name":"surname","_":"Vogler"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cor1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"type":"email"},"_":"a.vogler@nhm.ac.uk"}]},{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Department of Entomology, Natural History Museum, London, UK"}]},{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Division of Biology, Imperial College London, Silwood Park Campus, Ascot, UK"}]},{"#name":"affiliation","$":{"id":"aff3"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"University Museum of Zoology Cambridge, Department of Zoology, Downing Street, Cambridge CB2 3EJ, UK"}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author. Address: Department of Entomology, Natural History Museum, London SW7 5BD, UK. Fax: +44 207 942 5229."}]}]}],"floats":[],"footnotes":[],"affiliations":{"aff1":{"#name":"affiliation","$":{"id":"aff1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Department of Entomology, Natural History Museum, London, UK"}]},"aff2":{"#name":"affiliation","$":{"id":"aff2"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Division of Biology, Imperial College London, Silwood Park Campus, Ascot, UK"}]},"aff3":{"#name":"affiliation","$":{"id":"aff3"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"University Museum of Zoology Cambridge, Department of Zoology, Downing Street, Cambridge CB2 3EJ, UK"}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding author. Address: Department of Entomology, Natural History Museum, London SW7 5BD, UK. Fax: +44 207 942 5229."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"body":{},"exam":{},"article":{"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":119,"openArchiveStatus":false,"openArchiveArticleCount":0,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1055790309X00114-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790309X00114/cover/DOWNSAMPLED200/image/gif/3e4696fe4da22030fdfd73a664e0c859/cov200h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790309X00114/cover/DOWNSAMPLED200/image/gif/3e4696fe4da22030fdfd73a664e0c859/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"11008","pixel-height":"200","pixel-width":"150"},{"attachment-eid":"1-s2.0-S1055790309X00114-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790309X00114/cover/DOWNSAMPLED/image/gif/2677a262e1cb361c2689697fd7dc4f4c/cov150h.gif","https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790309X00114/cover/DOWNSAMPLED/image/gif/2677a262e1cb361c2689697fd7dc4f4c/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"7256","pixel-height":"150","pixel-width":"113"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S10557903.gif","sourceOpenAccess":false,"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1055790309X00114-cov150h.gif"},"pii":"S1055790309003480","dates":{"Available online":"31 August 2009","Received":"6 March 2009","Revised":["15 August 2009"],"Accepted":"17 August 2009","Publication date":"1 December 2009"},"access":{},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"analyticsMetadata":{"accountId":"262591","accountName":"International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"12289512"},"cid":"272322","content-family":"serial","copyright-line":"Copyright © 2009 Elsevier Inc. All rights reserved.","cover-date-years":["2009"],"cover-date-start":"2009-12-01","cover-date-text":"December 2009","document-subtype":"fla","document-type":"article","entitledToken":"C377168A1A3123754EBE0D19BB585267E33177835C1E9B825B6948D844B7117A1CE8F32DC3C593E0","eid":"1-s2.0-S1055790309003480","doi":"10.1016/j.ympev.2009.08.027","first-fp":"935","hub-eid":"1-s2.0-S1055790309X00114","issuePii":"S1055790309X00114","iss-first":"3","item-weight":"FULL-TEXT","language":"en","last-lp":"947","last-author":{"#name":"last-author","$":{"xmlns:dm":true},"$$":[{"#name":"author","$":{"xmlns:ce":true,"id":"aep-author-id10"},"$$":[{"#name":"given-name","_":"Alfried P."},{"#name":"surname","_":"Vogler"},{"#name":"cross-ref","$":{"refid":"aff1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"aff2"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]},{"#name":"cross-ref","$":{"refid":"cor1"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"type":"email"},"_":"a.vogler@nhm.ac.uk"}]}]},"normalized-first-auth-initial":"A","normalized-first-auth-surname":"PAPADOPOULOU","pages":[{"last-page":"947","first-page":"935"}],"srctitle":"Molecular Phylogenetics and Evolution","timestamp":"2015-05-15T04:50:19.793049-04:00","title":{"content":[{"#name":"title","$":{"xmlns:ce":true},"_":"DNA taxonomy and phylogeography of beetles of the Falkland Islands (Islas Malvinas)"}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"53","vol-iss-suppl-text":"Volume 53, Issue 3","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":false,"issn":"10557903","issn-primary-formatted":"1055-7903","useEnhancedReader":false,"isCorpReq":false,"pdfDownload":{"linkType":"DOWNLOAD","linkToPdf":"/science/article/pii/S1055790309003480/pdfft?md5=d64b9dd218c2e38401bfa46c6c97bd23&pid=1-s2.0-S1055790309003480-main.pdf","isPdfFullText":false,"fileName":"1-s2.0-S1055790309003480-main.pdf"},"pdfUrlForCrawlers":"https://www.sciencedirect.com/science/article/pii/S1055790309003480/pdfft?md5=d64b9dd218c2e38401bfa46c6c97bd23&pid=1-s2.0-S1055790309003480-main.pdf","indexTag":true,"volRange":"53","issRange":"3","userProfile":{"departmentName":"General Access","accessType":"IPRANGE","accountId":"262591","webUserId":"12289512","accountName":"International Institute of Tropical Agriculture","departmentId":"267018","userType":"NORMAL","hasMultipleOrganizations":false},"entitlementReason":"package","articleEntitlement":{"entitled":true},"aipType":"none","downloadFullIssue":true,"headerConfig":{"helpUrl":"https://service.elsevier.com/app/home/supporthub/sciencedirect/","contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","orgName":"","webUserId":"12289512","libraryBanner":{},"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasMultiOrg":false,"userType":"IPRANGE","allowCart":false},"open-research":{},"titleString":"DNA taxonomy and phylogeography of beetles of the Falkland Islands (Islas Malvinas)","isAbstract":false,"onAbstractWhitelist":false,"isContentVisible":false,"ajaxLinks":{"citingArticles":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true}},"specialIssueArticles":{},"recommendations":{},"entitledRecommendations":{"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"citingArticles":{},"workspace":{"isOpen":false},"crossMark":{"isOpen":false},"userIdentity":{},"refersTo":{},"referredToBy":{},"downloadIssue":{"isOpen":false,"articles":[],"selected":[]},"references":{},"referenceLinks":{"internal":{},"external":{}},"glossary":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenRecommendations":true,"isOpenCitingArticles":false,"citingArticles":[false,false,false],"recommendations":[false,false,false,false,false,false],"specialIssueArticles":[false,false,false]},"banner":{"expanded":false},"transientError":{"isOpen":false},"pdfDropdown":{},"exportCitation":{"isOpen":false},"tableOfContents":{"showEntitledTocLinks":true},"chapters":{"toc":[],"isLoading":false},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"metrics":{"isOpen":true},"signOut":{"isOpen":false},"issueNavigation":{"previous":{},"next":{}},"tail":{}}</script>
      
      <script src="satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c.js"></script><script src="satellite-565e008964746d4385002642.js"></script>
      <script src="babel-polyfill.js"></script>
      <script src="react.js"></script>
      <script src="react-dom.js"></script>
      <script src="arp.js" async=""></script>
      <!-- begin usabilla live embed code -->
      <script type="text/javascript">
        window.lightningjs||function(c){function g(b,d){d&&(d+=(/\?/.test(d)?"&":"?")+"lv=1");c[b]||function(){var i=window,h=document,j=b,g=h.location.protocol,l="load",k=0;(function(){function b(){a.P(l);a.w=1;c[j]("_load")}c[j]=function(){function m(){m.id=e;return c[j].apply(m,arguments)}var b,e=++k;b=this&&this!=i?this.id||0:0;(a.s=a.s||[]).push([e,b,arguments]);m.then=function(b,c,h){var d=a.fh[e]=a.fh[e]||[],j=a.eh[e]=a.eh[e]||[],f=a.ph[e]=a.ph[e]||[];b&&d.push(b);c&&j.push(c);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"//"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.frameBorder="0";f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
        window.usabilla_live = lightningjs.require("usabilla_live", "https://w.usabilla.com/eb1c14a91932.js");
        var customData = {};

        if(window.pageData && pageData.content && pageData.content[0]) {
          customData.entitlementType = pageData.content[0].entitlementType;
        }
        if(window.pageData && pageData.visitor) {
          customData.accessType = pageData.visitor.accessType;
          customData.accountId = pageData.visitor.accountId;
          customData.loginStatus = pageData.visitor.loginStatus;
        }
        usabilla_live("data", {"custom": customData });
      </script>
      <!-- end usabilla live embed code -->
      <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="MathJax.js"></script>
      <script async="" src="gpt.js"></script>
    
  <div class="js-react-modal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><script src="https://d39af2mgp1pqhg.cloudfront.net/widget-summary.js" async=""></script></body></html>