<!DOCTYPE html>
<html class="pb-page readcubeEpdfEnabled" data-request-id="b8750552-de7a-4457-b6bc-722c8d1a5085" lang="en"><head data-pb-dropzone="head">
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><script type="text/javascript" src="https://cdn.krxd.net/userdata/get?pub=5c9d7251-7b10-4260-8df4-d859510ca122&amp;technographics=1&amp;callback=Krux.ns._default.kxjsonp_userdata"></script><script type="text/javascript" src="https://beacon.krxd.net/optout_check?callback=Krux.ns._default.kxjsonp_optOutCheck"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="pbContext" content=";journal:journal:13652699;article:article:doi\:10.1111/jbi.12786;page:string:Article/Chapter View;website:website:pericles;wgroup:string:Publication Websites;issue:issue:doi\:10.1111/jbi.2017.44.issue-2;pageGroup:string:Publication Pages;subPage:string:Full Text"><meta data-pb-head="head-widgets-start">
<script async="" src="prum.js"></script><script src="iframe_api"></script><script async="" src="controltag.93f35b9b9881afd24d882ee537cf5cdd"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="controltag"></script><script type="text/javascript">var $DoubleClickZone = "j-biogeog_jbi";var $DoubleClickSite =  "wly.geography_000066";</script><script>digitalData = {"site":{"ip":"197.248.44.39","environment":"LIVE","website":"onlinelibrary.wiley.com"},"page":{"pageGroup":"Publication Pages","page":"Article/Chapter View","subPage":"Full Text","title":"title","errorCode":200},"identities":[{"type":"BasicGroup","uuid":"be20949b-b372-471e-96a5-9963c7266a9d","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000054870"}]},{"type":"BasicGroup","uuid":"98ba5e39-8ca0-4d0a-addd-395777fbf515","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000034166"}]},{"type":"BasicGroup","uuid":"e623deab-c83b-41e3-84a2-d9f1d46c5f17"},{"type":"RegionUser","uuid":"53f1676b-6f75-446d-8831-5915e1edce59"},{"type":"InstitutionUser","uuid":"dab9a169-ec74-4502-8066-0f76387b0f4e","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASP"}]},{"type":"BasicGroup","uuid":"40e3f768-07c0-45e5-aa81-97aef6eb686c","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"CORE00000005407"}]},{"type":"BasicGroup","uuid":"197fe835-5b5f-4f4e-b8e9-a74d68491a84"},{"type":"InstitutionUser","uuid":"5196cf05-b385-4de3-91a2-2d5ffd18abc5","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"P#INASPKEN"}]},{"type":"InstitutionUser","uuid":"480e4dba-ca42-475f-a7ca-5b6fbfad49f7","customerRecords":[{"customerDomain":"ALM-CU","customerNumber":"EAL00000014668"}]}],"publication":{"item":{"title":"A world of sequences: can we use georeferenced nucleotide databases for a robust automated phylogeography?","doi":"10.1111/jbi.12786","access":"full","earliestDate":"2016-07-04","formats":["Abstract","Full text","PDF"],"topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biogeography","topicUri":"ls2l"},{"taxonomyUri":"global-subject-codes","topicLabel":"Population \u0026 Community Ecology","topicUri":"ls2d"},{"taxonomyUri":"global-subject-codes","topicLabel":"Animal Science \u0026 Zoology","topicUri":"ls80"},{"taxonomyUri":"global-subject-codes","topicLabel":"Conservation Science","topicUri":"ls2k"},{"taxonomyUri":"global-subject-codes","topicLabel":"Macroecology","topicUri":"ls2e"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biodiversity","topicUri":"ls2j"},{"taxonomyUri":"fundref-funders","topicLabel":"Max Planck Society","topicUri":"501100004189"}]},"group":{"doi":"10.1111/jbi.2017.44.issue-2","access":"full","volume":"44","issue":"2","coverDate":"2017-02-01","topics":[{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biogeography","topicUri":"ls2l"},{"taxonomyUri":"global-subject-codes","topicLabel":"Population \u0026 Community Ecology","topicUri":"ls2d"},{"taxonomyUri":"global-subject-codes","topicLabel":"Animal Science \u0026 Zoology","topicUri":"ls80"},{"taxonomyUri":"global-subject-codes","topicLabel":"Conservation Science","topicUri":"ls2k"},{"taxonomyUri":"global-subject-codes","topicLabel":"Macroecology","topicUri":"ls2e"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biodiversity","topicUri":"ls2j"}]},"series":{"title":"Journal of Biogeography","doi":"10.1111/(ISSN)1365-2699","access":"no","code":"13652699","issn":"0305-0270","eissn":"1365-2699","topics":[{"taxonomyUri":"gpt-zonename","topicLabel":"j-biogeog_JBI","topicUri":"j-biogeog_jbi"},{"taxonomyUri":"gpt-sitename","topicLabel":"wly.geography_000066","topicUri":"wly.geography_000066"},{"taxonomyUri":"global-subject-codes","topicLabel":"Ecology \u0026 Organismal Biology","topicUri":"ls20"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biogeography","topicUri":"ls2l"},{"taxonomyUri":"global-subject-codes","topicLabel":"Population \u0026 Community Ecology","topicUri":"ls2d"},{"taxonomyUri":"global-subject-codes","topicLabel":"Animal Science \u0026 Zoology","topicUri":"ls80"},{"taxonomyUri":"global-subject-codes","topicLabel":"Conservation Science","topicUri":"ls2k"},{"taxonomyUri":"global-subject-codes","topicLabel":"Macroecology","topicUri":"ls2e"},{"taxonomyUri":"global-subject-codes","topicLabel":"LIFE SCIENCES","topicUri":"ls"},{"taxonomyUri":"global-subject-codes","topicLabel":"Biodiversity","topicUri":"ls2j"}]}}}</script>






<meta charset="UTF-8">




<meta name="robots" content="noarchive">

















    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=1">







    









    

    <meta name="citation_doi" content="10.1111/jbi.12786">

    <meta name="dc.identifier" content="10.1111/jbi.12786">




<meta name="citation_journal_title" content="Journal of Biogeography">
<meta name="citation_title" content="A world of sequences: can we use georeferenced nucleotide databases for a robust automated phylogeography?">
<meta name="citation_keywords" content="biodiversity">
<meta name="citation_keywords" content="comparative phylogeography">
<meta name="citation_keywords" content="DNA barcoding">
<meta name="citation_keywords" content="GenBank">
<meta name="citation_keywords" content="geodata">
<meta name="citation_keywords" content="georeferencing">
<meta name="citation_keywords" content="metadata">
<meta name="citation_volume" content="44">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="475">
<meta name="citation_lastpage" content="486">
<meta name="citation_publication_date" content="2017/02/01">
<meta name="citation_online_date" content="04 July 2016">
<meta name="citation_issn" content="1365-2699">
<meta name="citation_language" content="en">
<meta name="citation_author" content="Paolo Gratton">
<meta name="citation_author_institution" content="Department of Primatology Max Planck Institute for Evolutionary Anthropology Deutscher Platz 6 04103 Leipzig Germany">
<meta name="citation_author" content="Silvio Marta">
<meta name="citation_author_institution" content="Dipartimento di Biologia Università di Roma “Tor Vergata” Via della Ricerca Scientifica 1 00133 Rome Italy">
<meta name="citation_author" content="Gaëlle Bocksberger">
<meta name="citation_author_institution" content="Department of Primatology Max Planck Institute for Evolutionary Anthropology Deutscher Platz 6 04103 Leipzig Germany">
<meta name="citation_author" content="Marten Winter">
<meta name="citation_author_institution" content="German Center for Integrative Biodiversity Research (iDiv) Deutscher Platz 5 04103 Leipzig Germany">
<meta name="citation_author" content="Emiliano Trucchi">
<meta name="citation_author_institution" content="Department for Botany and Biodiversity Research University of Vienna Rennweg 14 A‐1030 Vienna Austria">
<meta name="citation_author" content="Hjalmar Kühl">
<meta name="citation_author_institution" content="Department of Primatology Max Planck Institute for Evolutionary Anthropology Deutscher Platz 6 04103 Leipzig Germany">
<meta name="citation_abstract_html_url" content="https://onlinelibrary.wiley.com/doi/abs/10.1111/jbi.12786">
<meta name="citation_fulltext_html_url" content="https://onlinelibrary.wiley.com/doi/full/10.1111/jbi.12786">
<meta name="citation_pdf_url" content="https://onlinelibrary.wiley.com/doi/pdf/10.1111/jbi.12786">
<meta name="citation_publisher" content="Wiley/Blackwell (10.1111)">
<meta name="article_references" content="Gratton, P. , Marta, S. , Bocksberger, G. , Winter, M. , Trucchi, E.  and Kühl, H.  (2017), A world of sequences: can we use georeferenced nucleotide databases for a robust automated phylogeography?. J. Biogeogr., 44: 475-486. doi:10.1111/jbi.12786">
<meta name="epdf_available" content="true">


<title>A world of sequences: can we use georeferenced nucleotide 
databases for a robust automated phylogeography? - Gratton - 2017 - 
Journal of Biogeography - Wiley Online Library</title>




<meta data-pb-head="head-widgets-end">

<link rel="stylesheet" type="text/css" href="f8d8product.css"><link rel="stylesheet" type="text/css" href="build.css"><link rel="stylesheet" type="text/css" href="default.css" id="pb-css" data-pb-css-id="t-1-v0/default.css">




<meta name="referrer" content="origin-when-cross-origin">

<link rel="stylesheet" type="text/css" href="hub-1529318462147.css">
<link rel="stylesheet" type="text/css" href="iucr-1510734465460.css">
<link rel="stylesheet" type="text/css" href="readcube-1510565373110.css">
<link rel="stylesheet" type="text/css" href="dr-overrides-1525191732107.css">
<!-- link rel="stylesheet" type="text/css" href="/pb-assets/css/temporary-1525702334993.css" -->

<style>
/* TEMPORARY CSS HACK TO FIX LIT-220590 */
.icon-icon-Rss::before {
    content: "\e629";
}

/* TEMPORARY CSS TO FIX ALIGNMENT OF TOP AD IN RIGHT RAIL */
.publication-cover-info {
    margin-bottom: 20px;
}

.article-row-right > .pb-ad {
    margin-left: 0;
    margin-bottom: 10px;
}

/* TEMPORARY HACK FOR PERICLES-6763 */
.table-of-content .issue-item h2 {
    font-size: 16px;
}

.table-of-content .issue-item h2 a.issue-item__title {
    display: inline;
}

/* TEMPORARY CSS FOR PERICLES-7445 */

input.saml__consent__yes {
    background: #005274;
    font-size: 12px;
    font-size: .75rem;
    padding: 12px 20px;
    border-radius: 0;
    margin-top: 20px;
    margin-right: 10px;
    width: 100px;
}

input.saml__consent__yes:hover {
    background: #003951;
}

/* HACK TO CHANGE LABEL IN LOGIN FORM */
.login-popup .login-form label[for="login"]:before {
    content: "Email or Customer ID";
    color: black;
}

.login-popup .login-form label[for="login"] {
    color: white;
}

/* PERICLES-2166: Site-wide maintenance message */

.admin-message {
    background-color: #005274;
    color: white;
}

/* TEMPORARY CSS FOR STAY-CONNECTED SOCIAL MEDIA ICONS */
.stay-connected ul  {
    list-style: none;
    margin: 0;
    padding: 0;
}

.stay-connected i {
    margin-right: 10px;
    font-size: 25px;
    line-height: 55px;
}

.stay-connected i.icon-hubs-rg-icon,
.stay-connected i.icon-hubs-soundcloud-icon {
    font-size: 41px;
}

.stay-connected .icon-hubs-pinterest-icon:before {
    color: white;
}

.stay-connected .icon-hubs-pinterest-icon {
    background-color: #BD081C;
    border: solid #BD081C;
    border-width: 8px;
}

/* PERICLES-5833: TEMPORARY CSS TO ADJUST POSITION OF ADS */
.article-mpu-bottom .pb-ad {
    margin: 0 0 0 195px;
}


.article-mpu-top .pb-ad {
    padding-top: 20px;
    margin: 0 0 -35px 0;
}

.banner-wrapper .pb-ad {
    padding: 0;
}

/* TEMPORARY CSS FOR THE SOCIETY MEMBER LOGIN PAGE */
.society-member-login .widget__header,
.society-member-login .show-registration,
.society-member-login .show-request-reset-password {
    display: none;
}

.society-member-login .login-form {
    margin-top: 20px;
}

.society-member-login .widget__body .input-group+.input-group {
    margin-top: 15px;
}


/* TEMPORARY CSS TO HIDE FPI IN TOP CONTENT ACTION WIDGET */
.top-content .firstPageImage {
	display: none;
}

/* PERICLES-5518: TEMPORARY CSS TO FIX ANCHOR TAG OFFSET ON CUSTOM PAGES */
.custom-page-hub .main-content :target:before {
	content: "";
	display: block;
	height: 205px;
	margin: -205px 0 0;
}

.custom-page-pericles .main-content :target:before {
	content: "";
	display: block;
	height: 140px;
	margin: -140px 0 0;
}

/* PERICLES-4777: TEMPORARY CSS TO HIDE OPTIONS IN THE SIDEBAR ON MRW PAGES */
.journal-resources li#TableofContents, 
.journal-resources li#Topics,
.journal-resources li#ArticleTitlesAZ {
	display: none;
}

/* TEMPORARY FIX FOR UNWANTED MARGIN BELOW COVER IMAGES ON BOOK SERIES PAGES */
.remove-margin .cover-image {
	margin: 0;
}

/* TEMPORARY STYLE TO HIDE NUMBER OF DOWNLOADS ON ARTICLE PAGES */
.info-tab-wrapper .metrics .section__body div:first-child {
	display: none;
}

/* TEMPORARY FIX TO MAKE IT POSSIBLE TO EDIT RICH TEXT WIDGETS IN SLIDESHOWS */
.bs .modal-backdrop {
	display: none;	
}

/* TEMPORARY FIX FOR MENU POSITION */
.hubpage-menu nav {
	padding-bottom: 0;
	position: relative;
}

/* TEMPORARY FOR MARGIN AROUND ADS */
.advert.advert-leaderboard {
	margin: 15px 0;
}

.banner-wrapper {
	box-shadow: none;
}

/* TEMPORARY TO FIX OVERFLOW FOR P IN JOURNAL-NEWS SLIDESHOWS */
.journal-news .slide-item p {
	overflow: unset;
}

/* EDITOR HIGHLIGHTS CSS */
div.ancillary-content.relateditem div.content {
	height: 10em;
}
div.ancillary-content.listitem div.content {
	height: 4em;
}




</style>

<!-- The Adobe DTM script. Must come after the digitalData object -->
<script src="satelliteLib-c1ac050bbdc647f13a62578e71f0d817f8a6cbb5.js"></script><script src="satellite-5aac819d64746d762f005d2f.js"></script><script src="satellite-59cd501964746d5527006272.js"></script>

<!-- Google Search Console verification -->
<meta name="google-site-verification" content="ffz4i4DxIJCdGpRXe0XN6KG6o4FYI-nnFW9I_dg8piM">

<!-- KRUX Implementation-->
<script class="kxct" data-id="JMOONjNh" data-timing="async" data-version="1.9" type="text/javascript" xml:space="preserve">

	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments )
	}).q = []);

	( function () {
		var k , m , s , src;
		k = document.createElement ( 'script' );
		k.type 	= 'text/javascript';
		k.async = true;
		src = ( m = location.href.match ( /\bkxsrc=([^&]+)/ )) && decodeURIComponent ( m [ 1 ]);
		k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test ( src ) ? src : src === 'disable' ? '' : ( location.protocol === 'https:'?'https:':'http:' ) + '//cdn.krxd.net/controltag?confid=JMOONjNh';
		s = document.getElementsByTagName ( 'script' ) [ 0 ];
		s.parentNode.insertBefore ( k , s );
	} ());
</script>
	
<!-- KRUX Implementation cont'd -->
<script type="text/javascript">
	window.Krux || (( Krux = function () {
		Krux.q.push ( arguments );
	}).q = []);

	( function () {
		function retrieve ( n ) {
			var m , k = 'kx' + n;
			if ( window.localStorage ) {
				return window.localStorage [ k ] || '';
			} else if ( navigator.cookieEnabled ) {
				m = document.cookie.match ( k + '=([^;]*)' );
				return ( m && unescape ( m [ 1 ])) || '';
			} else {
				return '';
			}
		}

		Krux.user = retrieve ( 'user' );
		Krux.segments = retrieve ( 'segs' ) && retrieve ( 'segs' ).split ( ',' ) || [];
	}) ();
</script>

<!-- Grapeshot Implementation -->
<script>
	var gs_channels = 'default';
</script>

<script src="channels.cgi" async=""></script>

<!-- DoubleClick Implementation -->
<script type="text/javascript">
	var $WileyGooglePublisherID = 2215;

	switch("journal_13652699") {
		case "journal_23798068":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19343663":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19388969":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19349262":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_25725637":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19343647":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348533":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_25725602":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19342616":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_1934340x":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_21604762":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19483430":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19349300":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348258":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_1934368x":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_21612617":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348576":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19349289":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_19348290":
			var $DoubleClickSite = "wly.gn.life.sci_000061";
			break;
		case "journal_15518833":
			var $DoubleClickSite = "wly.civ.eng.con_000034";
			break;
		case "journal_25738461":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_15372197":
			var $DoubleClickSite = "wly.plant.sci_000100";
			break;
		case "journal_18832148":
			var $DoubleClickSite = "wly.cardio.dis_000029";
			break;
		case "journal_21576580":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_15494918":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_24750360":
			var $DoubleClickSite = "wly.geriat.ed_000067";
			break;
		case "journal_18780369":
			var $DoubleClickSite = "wly.nursing_000088";
			break;
		case "journal_19412444":
			var $DoubleClickSite = "wly.nutrition_000089";
			break;
		case "journal_17582652":
			var $DoubleClickSite = "wly.inf.dis.mi_000072";
			break;
		case "journal_25721143":
			var $DoubleClickSite = "wly.sport.med_000112";
			break;
		case "journal_15518701":
			var $DoubleClickSite = "wly.civ.eng.con_000034";
			break;
		case "journal_14781239":
			var $DoubleClickSite = "wly.gastro.hep_000059";
			break;
		case "journal_19383673":
			var $DoubleClickSite = "wly.basic.med_000026";
			break;
		case "journal_24758817":
			var $DoubleClickSite = "wly.cell.m.bio_000030";
			break;
		case "journal_24725390":
			var $DoubleClickSite = "wly.emerge.med_000050";
			break;
		case "journal_15589293":
			var $DoubleClickSite = "wly.indus.chem_000124";
			break;
		case "journal_15589307":
			var $DoubleClickSite = "wly.food.sci_000058";
			break;
		case "journal_15589331":
			var $DoubleClickSite = "wly.chemistry_000033";
			break;
		case "journal_18735924":
			var $DoubleClickSite = "wly.finan.inves_000057";
			break;
		case "journal_25770470":
			var $DoubleClickSite = "wly.elec.e.eng_000049";
			break;
		case "journal_25738615":
			var $DoubleClickSite = "wly.finan.inves_000057";
	};

	switch("journal_13652699") {
		case "journal_23798068":
			var $DoubleClickZone = "cp-plant-bio_CPPB";
			break;
		case "journal_19343663":
			var $DoubleClickZone = "cp-protein-sci_CPPS";
			break;
		case "journal_19388969":
			var $DoubleClickZone = "cp-stem-cell-bio_CPSC";
			break;
		case "journal_19349262":
			var $DoubleClickZone = "cp-toxicology_CPTX";
			break;
		case "journal_25725637":
			var $DoubleClickZone = "cp-mri_CPMR";
			break;
		case "journal_19343647":
			var $DoubleClickZone = "cp-molecular-bio_CPMB";
			break;
		case "jorunal_19348533":
			var $DoubleClickZone = "cp-microbio_CPMI";
			break;
		case "journal_25725602":
			var $DoubleClickZone = "cp-food-a-chem_CPFA";
			break;
		case "journal_19342616":
			var $DoubleClickZone = "cp-cell-bio_CPCL";
			break;
		case "journal_1934340x":
			var $DoubleClickZone = "cp-bioinfo_CPBI";
			break;
		case "journal_21604762":
			var $DoubleClickZone = "cp-chem-bio_CPCB";
			break;
		case "journal_19483430":
			var $DoubleClickZone = "cp-ess-lab-tech_CPLT";
			break;
		case "journal_19349300":
			var $DoubleClickZone = "cp-cytometry_CPCY";
			break;
		case "journal_19348258":
			var $DoubleClickZone = "cp-human-gen_CPHG";
			break;
		case "journal_1934368x":
			var $DoubleClickZone = "cp-immun_CPIM";
			break;
		case "journal_21612617":
			var $DoubleClickZone = "cp-mouse-b_CPMO";
			break;
		case "journal_19348576":
			var $DoubleClickZone = "cp-neurosci_CPNE";
			break;
		case "journal_19349289":
			var $DoubleClickZone = "cp-n-acid-chem_CPNA";
			break;
		case "journal_19348290":
			var $DoubleClickZone = "cp-pharma_CPPH";
			break;
		case "journal_15518833":
			var $DoubleClickZone = "jrnl-american-water-works-assc_AWWA";
			break;
		case "journal_25738461":
			var $DoubleClickZone = "adv-in-cell-and-gene_ACG2";
			break;
		case "journal_15372197":
			var $DoubleClickZone = "am-j-o-botany_AJB2";
			break;
		case "journal_18832148":
			var $DoubleClickZone = "j-o-arrhythmia_JOA3";
			break;
		case "journal_21576580":
			var $DoubleClickZone = "stem-cells-trans-medicine_STC3";
			break;
		case "journal_15494918":
			var $DoubleClickZone = "stem-cells_STEM";
			break;
		case "journal_24750360":
			var $DoubleClickZone = "aging-medicine_AGM2";
			break;
		case "journal_18780369":
			var $DoubleClickZone = "aorn_AORN";
			break;
		case "journal_19412444":
			var $DoubleClickZone = "j-o-parent-and-enteral-nutrition_JPEN";
			break;
		case "journal_17582652":
			var $DoubleClickZone = "j-international-aids-s_JIA2";
			break;
		case "journal_25721143":
			var $DoubleClickZone = "j-orthoped-res_JOR";
			break;
		case "journal_15518701":
			var $DoubleClickZone = "opflow_OPFL";
			break;
		case "journal_14781239":
			var $DoubleClickZone = "j-gastro-hepatology_JGH";
			break;
		case "journal_19383673":
			var $DoubleClickZone = "jrnl-leukocyte-biology_JLB";
			break;
		case "journal_24758817":
			var $DoubleClickZone = "peptide-science_PEP2";
			break;
		case "journal_24725390":
			var $DoubleClickZone = "academic-emer-med_acem_ET";
			break;
		case "journal_15589293":
			var $DoubleClickZone = "jrnl-surfact-detergents_JSDE";
			break;
		case "journal_15589307":
			var $DoubleClickZone = "lipids_LIPD";
			break;
		case "journal_15589331":
			var $DoubleClickZone = "j-amer-oil_chem";
			break;
		case "journal_18735924":
			var $DoubleClickZone = "rev-finan-econ_RFE";
			break;
		case "journal_25770470":
			var $DoubleClickZone = "j-quant-eng_QUE2";
			break;
		case "journal_25738615":
			var $DoubleClickZone = "Fin-plan-review";
	};

	if (!$DoubleClickSite) {
		var $DoubleClickSite = "undefined-site";
	};

	if (!$DoubleClickZone) {
		var $DoubleClickZone = "undefined-zone";
	};

	var $GooglePublisherCategory = "/" + $WileyGooglePublisherID + "/" + $DoubleClickSite + "/" + $DoubleClickZone;
</script>

<script>
	var googletag = googletag || {};

	googletag.cmd = googletag.cmd || [];

	( function () {

		var gads = document.createElement ( 'script' ) ,
		node = document.getElementsByTagName ( 'script' ) [ 0 ] ;

		gads.async = true;
		gads.src = document.location.protocol + '//www.googletagservices.com/tag/js/gpt.js';

		node.parentNode.insertBefore ( gads , node );

	}) ();
</script>

<script>
	googletag.cmd.push ( function () {

		var sizes = {
			leaderboard : googletag.sizeMapping ()
				.addSize ([ 300 , 100 ] , [ 300 , 50 ])
				.addSize ([ 740 , 100 ] , [ 728 , 90 ])
				.addSize ([ 0 , 0 ] , 'fluid')
				.build () ,
			rail : googletag.sizeMapping ()
				.addSize ([ 0 , 0 ] , [ 300 , 250 ])
				.build ()
		};

		googletag.pubads ()
			.setTargeting ( 'property' 	, 'value-of-property' 	) 	// Change and duplicate this line for as many additional keywords to send
			.setTargeting ( 'gs_cat' 	, gs_channels 		) 	// Grapeshot
			.setTargeting ( 'ksg' 		, Krux.segments 	) 	// Krux
			.setTargeting ( 'kuid' 		, Krux.user 		); 	// Krux

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-leaderboard' )
			.defineSizeMapping ( sizes.leaderboard )
			.addService ( googletag.pubads ())
			.setCollapseEmptyDiv ( true );
				
		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-1' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-1')
			.addService ( googletag.pubads ()) 
			.setCollapseEmptyDiv ( true );

		googletag.defineSlot ( $GooglePublisherCategory, 'fluid', 'advert-rail-2' )
			.defineSizeMapping ( sizes.rail )
			.setTargeting ( 'Position', 'advert-rail-2')
			.addService ( googletag.pubads ()) 
			.setCollapseEmptyDiv ( true );

		googletag.pubads ().enableSingleRequest ();
		googletag.enableServices ();
	});
	//]]>
</script>









    
        
    








    




<script>
    var digitalData = window.digitalData ? window.digitalData : {};
    digitalData.page.title = "A world of sequences: can we use georeferenced nucleotide databases for a robust automated phylogeography? - Gratton - 2017 - Journal of Biogeography - Wiley Online Library";
</script>

<script>
    tracking = {};

    // events which don't generate server side events!
    tracking.adtmOnlyEvents = ["readcube.share.createlink", "readcube.ecommerce.load", "readcube.ecommerce.new",
        "readcube.ecommerce.userinfo", "readcube.ecommerce.register", "readcube.ecommerce.billing", "readcube.ecommerce.final",
        "readcube.ecommerce.purchasefailed"];

    tracking.ajaxData = function (eventName) {
        return {
            url: "/readcube/log",
            data: {
                "doi": digitalData.publication.item.doi,
                event: eventName
            }
        }
    };

    tracking.trackEvent = function (eventName, eventData, callback) {

        if (!tracking.adtmOnlyEvents.indexOf(eventName) >= 0) {
            tracking.trackUsage(eventName, callback);
        }
        tracking.trackAnalytics(eventName, eventData);
    };

    tracking.trackUsage = function (eventName, callback) {
        pr = $.ajax(tracking.ajaxData(eventName));

        if (callback) {
            pr.always(callback);
        }

    };

    tracking.trackAnalytics = function (eventName, eventData) {
        digitalData = window.digitalData ? digitalData : {};
        digitalData.event = digitalData.event || [];
        digitalData.event.push({
            "eventName": eventName,
            "eventData": eventData
        });
    }
</script>


    <script src="epdf_linker.js"></script><script async="" type="text/javascript" src="epdf_linker_002.js"></script>
<style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_stumbleupon_badge,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><script src="satellite-5b4e5b3364746d482f001123.js"></script><script src="analytics.js"></script><script src="s-code-contents-7d04accfbc0deee918eb104cc0a954ffa836b5dd.js"></script><script src="satellite-5ad74ae364746d5a68016a67.js"></script><link href="style.css" type="text/css" rel="stylesheet"><style type="text/css">.fancybox-margin{margin-right:0px;}</style><link rel="preload" href="https://adservice.google.com/adsid/integrator.js?domain=onlinelibrary.wiley.com"><script type="text/javascript" src="https://adservice.google.com/adsid/integrator.js?domain=onlinelibrary.wiley.com"></script><script src="https://securepubads.g.doubleclick.net/gpt/pubads_impl_238.js" async=""></script><script id="altmetric-embed-js" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-2c47105b6381604898bbf8ae8a680350.css"></head>
<body class="pb-ui">

    <a href="#article__content" id="skipNavigationLink" class="skip" tabindex="0">Skip to Main Content</a>




<script type="text/javascript">

    if(true) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('article__content');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('article__content').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>    <div id="pb-page-content" data-ng-non-bindable="">
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="pageHeader" data-widget-id="62175478-b98a-43d5-90f2-1f2045ac1a7e" class="pageHeader" style="position: fixed;">
        



        
        <header class="page-header">
    <div data-pb-dropzone="main">
    









        <div data-widget-def="ux3-layout-widget" data-widget-id="bb5b5e2c-d3d6-4015-b586-eeade38f59d4" class="admin-message">
        



        
        <div class="container"><div class="row"><div>



        
        


        
    

</div></div></div>

        </div>
    




        
        




        
        <div class="container"><div class="row"><div class="contents col-sm-12 col-md-12">









        <div data-widget-def="UX3InstitutionBanner" data-widget-id="0a735273-4141-44d8-801a-2b89143bed55" class="institution-wrapper-mobile hidden-lg hidden-md padded-content">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access By</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    




        
        <div class="pull-left">









        <div data-widget-def="ux3-general-image" data-widget-id="ff262197-0fb8-4973-aaa2-2fceba4b5a61" class="logo-container hidden-xs hidden-sm hidden-md">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img alt="Wiley Online Library" src="logo-header-1526603583437.png"></a>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="9a75b789-224c-4745-833a-930cf6c36f35" class="logo-container-mobile hidden-lg hidden-md hidden-sm">
        



        
        <a href="https://onlinelibrary.wiley.com/" title="Wiley Online Library"><img alt="Wiley Online Library" src="mobilehublogo-1518781861020.png"></a>

        </div>
    










        <div data-widget-def="UX3InstitutionBanner" data-widget-id="14d0e2f8-27d3-42d0-a477-72015e14e818" class="institution-wrapper hidden-xs hidden-sm institution--block">
        



        
        <div class="institution"><a href="http://www.icipe.org/" title="ICIPE LIBRARY"><div class="institution-info-wrapper"><span class="institution__intro">Access Provided by</span><span class="institution__name">ICIPE LIBRARY</span></div></a></div>

        </div>
    
</div><div class="pull-right">









        <div data-widget-def="UX3QuickSearchWidget" data-widget-id="48175bb0-0f70-46e6-bb89-7e2781fdb8e4" class="header-quicksearch">
        



        
        <div class="quick-search-wrapper"><a href="#" title="search" class="quick-search__item"><i aria-hidden="true" class="icon-search"></i><span class="quick-search__item__arrow"></span></a><div data-linkto-page="true" class="quick-search quickSearchFormContainer"><div class="main-search-field-wrapper" style="overflow: hidden;"><div class="tab clearfix"><input placeholder="Enter Your Search Term" class="fakeQSInput" type="search"><div class="dropBlock" data-db-parent-of="dbTab-0"><a href="#" class="dropBlock__link" data-db-target-for="dbTab-0"><span>This Journal</span><i class="icon-arrow_d_n" aria-hidden="true"></i></a><ul role="tablist" class="rlist tab__nav dropblock--tab" data-db-target-of="dbTab-0"><li class="active"><a href="#pane-67065c09-4a88-49cd-934c-ac707951d35c0" role="tab" data-toggle="tab" id="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" aria-expanded="true" data-search-in="thisJournal">This Journal</a></li><li><a href="#pane-67065c09-4a88-49cd-934c-ac707951d35c1" role="tab" data-toggle="tab" id="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" aria-expanded="false" data-search-in="default">Anywhere</a></li></ul></div><ul role="tablist" class="rlist tab__content"><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c0" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c0-con" role="tab" class="tab__pane quick-search-pane__anywhere active"><form action="/action/doSearch" name="thisJournalQuickSearch" method="get" title="Quick Search"><div class="input-group option-0"><label for="searchField0" class="hiddenLabel">Search term</label><input name="AllField" id="searchField0" placeholder="Enter Your Search Term" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input name="SeriesKey" value="13652699" type="hidden"><div class="search-options-wrapper"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=13652699" title="" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced#citation" title="" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" class="btn quick-search__button icon-search"><span>Search</span></button></form></li><li id="pane-67065c09-4a88-49cd-934c-ac707951d35c1" aria-labelledby="pane-67065c09-4a88-49cd-934c-ac707951d35c1-con" role="tab" class="tab__pane quick-search-pane__anywhere"><form action="/action/doSearch" name="defaultQuickSearch" method="get" title="Quick Search"><div class="input-group option-1"><label for="searchField1" class="hiddenLabel">Search term</label><input name="AllField" id="searchField1" placeholder="Enter Your Search Term" onfocus="this.value = this.value;" data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" class="autocomplete actualQSInput ui-autocomplete-input" autocomplete="off" type="search"><span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><div class="search-options-wrapper"><a href="https://onlinelibrary.wiley.com/search/advanced?publication=13652699" title="" class="advanced">  Advanced Search</a><a href="https://onlinelibrary.wiley.com/search/advanced#citation" title="" class="citation">  Citation Search</a></div></div><button id="quick-search-btn" type="submit" title="Search" class="btn quick-search__button icon-search"><span>Search</span></button></form></li></ul></div></div></div></div>

        </div>
    










        <div data-widget-def="literatumNavigationLoginBar" data-widget-id="d384ca92-7137-45d6-b3c2-78c9c0a98c2b" class="login-bar-container">
        



        
        




    








<div class="loginBar login-open">
    <a href="https://onlinelibrary.wiley.com/action/showLogin?uri=%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786" class="show-login">
        <span class="icon-profile_empty"></span>
        <span class="sign-in-label">Login / Register</span>
    </a>
    
</div>


        </div>
    
</div>
</div></div></div>
    </div>
</header>


        </div>
    




        
        




        
        




        
        <!-- [Tag{id&#x3D;102705, uri&#x3D;j-biogeog_jbi, name&#x3D;j-biogeog_JBI}, Tag{id&#x3D;101752, uri&#x3D;wly.geography_000066, name&#x3D;wly.geography_000066}, Tag{id&#x3D;320, uri&#x3D;ls20, name&#x3D;Ecology &amp; Organismal Biology}, Tag{id&#x3D;1189, uri&#x3D;ls2l, name&#x3D;Biogeography}, Tag{id&#x3D;1190, uri&#x3D;ls2d, name&#x3D;Population &amp; Community Ecology}, Tag{id&#x3D;327, uri&#x3D;ls80, name&#x3D;Animal Science &amp; Zoology}, Tag{id&#x3D;1194, uri&#x3D;ls2k, name&#x3D;Conservation Science}, Tag{id&#x3D;1195, uri&#x3D;ls2e, name&#x3D;Macroecology}, Tag{id&#x3D;60, uri&#x3D;ls, name&#x3D;LIFE SCIENCES}, Tag{id&#x3D;1183, uri&#x3D;ls2j, name&#x3D;Biodiversity}] -->                      <div id="iucr-failure"> <p><i class="icon-notice"></i>The full text of this article hosted at iucr.org is unavailable due to technical difficulties.<a href="#" class="close-iucr-failure"><i class="icon-x"></i></a> </p> </div> 












        <div data-widget-def="pageBody" data-widget-id="72100436-7a82-49fc-933b-c6c9d8c42914" class="pageBody article-page-tablet" style="padding-top: 71px;">
        



        
        <!--begin pagefulltext-->
<div class="page-body pagefulltext">
    <div data-pb-dropzone="main">
    



        
        <div>









        <div data-widget-def="literatumAd" data-widget-id="0220a30b-90c0-4b51-8637-141edf12f86a" class="banner-wrapper">
        



        
        


        <div class="pb-ad">
            <aside role="complementary">
		<div id="advert-leaderboard" class="advert advert-leaderboard">
			<script>
				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-leaderboard' );
				});
			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1013&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D728x90%26amp%3Btile%3D1%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786&pubId=63162737&placeholderId=1015&productId=101018" target="_blank">
					<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=728x90&amp;tile=1&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_0" title="3rd party ad content" name="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="100%" frameborder="0" height="0"></iframe></div></div>
	</aside>
        </div>
    


        </div>
    




        
        <style type="text/css">li.menu-item[aria-label="Accepted Articles"] {
  display: none;
}</style>
</div>










        <section data-widget-def="ux3-layout-widget" data-widget-id="0f7bccba-8328-4dbd-a1fa-e84c526dd9ab">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">



        
        <div>



        
        <article data-figures="/action/ajaxShowFigures?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fjbi.12786&amp;pbContext=%3Bjournal%3Ajournal%3A13652699%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fjbi.12786%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fjbi.2017.44.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fjbi.12786&amp;pbContext=%3Bjournal%3Ajournal%3A13652699%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fjbi.12786%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fjbi.2017.44.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" data-related="/action/ajaxShowRecommended?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fjbi.12786&amp;pbContext=%3Bjournal%3Ajournal%3A13652699%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fjbi.12786%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fjbi.2017.44.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=global-subject-codes" data-details="/action/ajaxShowPubInfo?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fjbi.12786&amp;pbContext=%3Bjournal%3Ajournal%3A13652699%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fjbi.12786%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fjbi.2017.44.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;showSubjects=true&amp;taxonomiesCodes=global-subject-codes" data-mathjax="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.0/MathJax.js?config=TeX-AMS-MML_HTMLorMML"><div class="row article-row"><div class="col-sm-12 col-md-8 col-lg-8 article__content article-row-left"><div data-pb-dropzone="mrwBannerDropZone" class="mrw-banner-dropzone"></div><div class="article-citation"><div class="citation">
   <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-series-title">
      



        
        <div data-pb-dropzone="publicaitonContent-series-title" class="pb-dropzone">
  <h1><a href="https://onlinelibrary.wiley.com/journal/13652699" title="Journal of Biogeography homepage">Journal of Biogeography</a></h1>
  <a href="https://onlinelibrary.wiley.com/toc/13652699/44/2" class="volume-issue">Volume 44, Issue 2</a>
  <a href="https://onlinelibrary.wiley.com/journal/13652699" title="Journal of Biogeography homepage" class="citation--logo"><img class="journal-banner-image" test="test-value" alt="Journal of Biogeography banner" src="13652699-1501384677093.jpg"></a>
</div>

   </div>
   <div class="doi-access-container clearfix"><span class="primary-heading">Methodological Application</span>
      <div class="doi-access-wrapper"><div class="full-access access-type"> <i aria-hidden="true" class="icon-icon-lock_open"></i> <div class="doi-access"> <div class="doi-access" tabindex="0">Full Access</div></div></div></div></div>
   <h2 class="citation__title">A world of sequences: can we use georeferenced nucleotide databases for a robust automated
      phylogeography?
   </h2>
   <div class="rlist--inline loa comma visible-xs mobile-authors loa-authors-trunc">
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gratton%2C+Paolo" class="author-name accordion-tabbed__control" data-id="am1" data-db-target-for="am1" aria-controls="am1" aria-haspopup="true" id="am1_Ctrl"><span>Paolo Gratton<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am1" aria-labelledby="am1_Ctrl" id="am1">
            <p class="author-type">Corresponding Author</p>
            <p><span>E-mail address:</span><a title="
                Link to email address
            " href="mailto:paolo_gratton@eva.mpg.de">paolo_gratton@eva.mpg.de</a></p>
            <p>Department of Primatology, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany</p>
            <div class="bottom-info">Correspondence: Paolo Gratton, Department of Primatology, Max Planck Institute for
               Evolutionary Anthropology, Deutscher Platz 6, 04103 Leipzig, Germany.<br>E‐mail: <span>E-mail address:</span><a title="
                Link to email address
            " href="mailto:paolo_gratton@eva.mpg.de">paolo_gratton@eva.mpg.de</a><br></div><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gratton%2C+Paolo">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Marta%2C+Silvio" class="author-name accordion-tabbed__control" data-id="am2" data-db-target-for="am2" aria-controls="am2" aria-haspopup="true" id="am2_Ctrl"><span>Silvio Marta</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am2" aria-labelledby="am2_Ctrl" id="am2">
            <p>Dipartimento di Biologia, Università di Roma “Tor Vergata”, Rome, Italy</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Marta%2C+Silvio">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Bocksberger%2C+Ga%C3%ABlle" class="author-name accordion-tabbed__control" data-id="am3" data-db-target-for="am3" aria-controls="am3" aria-haspopup="true" id="am3_Ctrl"><span>Gaëlle Bocksberger</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am3" aria-labelledby="am3_Ctrl" id="am3">
            <p>Department of Primatology, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Bocksberger%2C+Ga%C3%ABlle">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Winter%2C+Marten" class="author-name accordion-tabbed__control" data-id="am4" data-db-target-for="am4" aria-controls="am4" aria-haspopup="true" id="am4_Ctrl"><span>Marten Winter</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am4" aria-labelledby="am4_Ctrl" id="am4">
            <p>German Center for Integrative Biodiversity Research (iDiv), Leipzig, Germany</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Winter%2C+Marten">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Trucchi%2C+Emiliano" class="author-name accordion-tabbed__control" data-id="am5" data-db-target-for="am5" aria-controls="am5" aria-haspopup="true" id="am5_Ctrl"><span>Emiliano Trucchi</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am5" aria-labelledby="am5_Ctrl" id="am5">
            <p>Department for Botany and Biodiversity Research, University of Vienna, Vienna, Austria</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Trucchi%2C+Emiliano">Search for more papers by this author</a></div>
      </div>
      <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=K%C3%BChl%2C+Hjalmar" class="author-name accordion-tabbed__control" data-id="am6" data-db-target-for="am6" aria-controls="am6" aria-haspopup="true" id="am6_Ctrl"><span>Hjalmar Kühl</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="am6" aria-labelledby="am6_Ctrl" id="am6">
            <p>Department of Primatology, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=K%C3%BChl%2C+Hjalmar">Search for more papers by this author</a></div>
      </div>
   </div>
   <div class="loa-wrapper loa-authors hidden-xs">
      <div id="sb-1" class="accordion">
         <div class="accordion-tabbed">
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gratton%2C+Paolo" class="author-name accordion-tabbed__control" data-id="a1" data-db-target-for="a1" aria-controls="a1" aria-haspopup="true" id="a1_Ctrl"><span>Paolo Gratton<i aria-hidden="true" class="icon-mail_outline"></i></span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a1" aria-labelledby="a1_Ctrl" id="a1" style="display: none;">
                  <p class="author-type">Corresponding Author</p>
                  <p><span>E-mail address:</span><a title="
                Link to email address
            " href="mailto:paolo_gratton@eva.mpg.de">paolo_gratton@eva.mpg.de</a></p>
                  <p>Department of Primatology, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany</p>
                  <div class="bottom-info">Correspondence: Paolo Gratton, Department of Primatology, Max Planck Institute for
                     Evolutionary Anthropology, Deutscher Platz 6, 04103 Leipzig, Germany.<br>E‐mail: <span>E-mail address:</span><a title="
                Link to email address
            " href="mailto:paolo_gratton@eva.mpg.de">paolo_gratton@eva.mpg.de</a><br></div><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Gratton%2C+Paolo">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Marta%2C+Silvio" class="author-name accordion-tabbed__control" data-id="a2" data-db-target-for="a2" aria-controls="a2" aria-haspopup="true" id="a2_Ctrl"><span>Silvio Marta</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a2" aria-labelledby="a2_Ctrl" id="a2" style="display: none;">
                  <p>Dipartimento di Biologia, Università di Roma “Tor Vergata”, Rome, Italy</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Marta%2C+Silvio">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Bocksberger%2C+Ga%C3%ABlle" class="author-name accordion-tabbed__control" data-id="a3" data-db-target-for="a3" aria-controls="a3" aria-haspopup="true" id="a3_Ctrl"><span>Gaëlle Bocksberger</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a3" aria-labelledby="a3_Ctrl" id="a3" style="display: none;">
                  <p>Department of Primatology, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Bocksberger%2C+Ga%C3%ABlle">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Winter%2C+Marten" class="author-name accordion-tabbed__control" data-id="a4" data-db-target-for="a4" aria-controls="a4" aria-haspopup="true" id="a4_Ctrl"><span>Marten Winter</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a4" aria-labelledby="a4_Ctrl" id="a4" style="display: none;">
                  <p>German Center for Integrative Biodiversity Research (iDiv), Leipzig, Germany</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Winter%2C+Marten">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Trucchi%2C+Emiliano" class="author-name accordion-tabbed__control" data-id="a5" data-db-target-for="a5" aria-controls="a5" aria-haspopup="true" id="a5_Ctrl"><span>Emiliano Trucchi</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a5" aria-labelledby="a5_Ctrl" id="a5" style="display: none;">
                  <p>Department for Botany and Biodiversity Research, University of Vienna, Vienna, Austria</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=Trucchi%2C+Emiliano">Search for more papers by this author</a></div>
            </div>
            <div class="accordion-tabbed__tab-mobile  accordion__closed"><a href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=K%C3%BChl%2C+Hjalmar" class="author-name accordion-tabbed__control" data-id="a6" data-db-target-for="a6" aria-controls="a6" aria-haspopup="true" id="a6_Ctrl"><span>Hjalmar Kühl</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="author-info accordion-tabbed__content" data-db-target-of="a6" aria-labelledby="a6_Ctrl" id="a6" style="display: none;">
                  <p>Department of Primatology, Max Planck Institute for Evolutionary Anthropology, Leipzig, Germany</p><a class="moreInfoLink" href="https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorStored=K%C3%BChl%2C+Hjalmar">Search for more papers by this author</a></div>
            </div>
         </div>
      </div>
   </div>
   <div class="epub-sections">
      <div class="epub-section"><span class="epub-state">First published:&nbsp;</span><span class="epub-date">04 July 2016</span></div>
      <div class="epub-section"><a class="epub-doi" href="https://doi.org/10.1111/jbi.12786">https://doi.org/10.1111/jbi.12786</a></div>
      <div class="epub-section cited-by-count"><span>Cited by: <a href="#citedby-section">9</a></span></div>
   </div>
   <div class="epub-sections">
      <div class="article-header__references-container no-truncate" id="jbi12786-note-1001">
         <p>Editor: Alexandre Antonelli</p>
      </div>
   </div>
</div></div><nav class="stickybar coolBar trans"><div class="stickybar__wrapper coolBar__wrapper clearfix" style="width: 690px; top: 0px;"><div class="rlist coolBar__zone"><div class="coolBar__section"><a href="#" data-db-target-for="article" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="article_Pop" role="button" id="article_Ctrl" class="coolBar__ctrl hidden-lg"><i class="icon-abstract"></i><span>About</span></a><div data-db-target-of="article" aria-labelledby="article_Ctrl" role="menu" id="article_Pop" class="coolBar__drop fixed rlist dropBlock__holder"><div data-target=".article-row-right .tab__nav, .coolBar--download" data-remove="false" data-targetclass="hidden-xs hidden-sm" data-toggle="transplant" data-direction="from" data-transplant="self" class="transplant showit"><div class="transplanted-clone"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><div class="coolBar__section coolBar--download readcubeEPdfLink"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbi.12786" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink pdfLink--is-hidden"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/jbi.12786" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div></div></div></div></div><div class="coolBar__section coolBar--sections"><a href="#" data-db-target-for="sections" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="sections_Pop" role="button" id="sections_Ctrl" class="coolBar__ctrl"><i class="icon-list-unordered"></i><span>Sections</span></a><div data-db-target-of="sections" aria-labelledby="sections_Ctrl" role="menu" id="sections_Pop" class="coolBar__drop dropBlock__holder rlist fixed"><ul class="rlist w-slide--list"><li role="menuitem"><a class="w-slide__hide" href="#d5177291"><span>Abstract</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0006-title"><span> Introduction</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0007-title"><span> Methods</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0012-title"><span> Results</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0017-title"><span> Discussion</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0018-title"><span> Acknowledgements</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#support-information-section"><span>
                  Supporting Information
                  </span></a></li><li role="menuitem"><a class="w-slide__hide" href="#references-section-1"><span>References</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0019-title"><span> Data accessibility</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#jbi12786-sec-0020-title"><span> Biosketches</span></a></li><li role="menuitem"><a class="w-slide__hide" href="#citedby-section"><span>Citing Literature</span></a></li></ul></div></div></div><ul data-cb-group="Article" data-cb-group-icon="icon-toc" class="rlist coolBar__first"></ul><div class="coolBar__second rlist"><div class="coolBar__section coolBar--download readcubeEPdfLink cloned"><a href="https://onlinelibrary.wiley.com/doi/epdf/10.1111/jbi.12786" title="Article ePDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="ePDF">PDF</a></div><div class="coolBar__section coolBar--download PdfLink pdfLink--is-hidden cloned"><a href="https://onlinelibrary.wiley.com/doi/pdf/10.1111/jbi.12786" title="Article PDF" class="coolBar__ctrl pdf-download"><img src="pdf-icon.png" alt="PDF">PDF</a></div>



        
        <div class="relative"><a href="#" data-db-target-for="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" role="button" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" class="article-tools__ctrl"><i aria-hidden="true" class="icon-Icon_Tools"></i><span>Tools</span></a><div data-db-target-of="0e406ce9-5c43-4e2d-b741-e0b43d1f1865" aria-labelledby="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Ctrl" role="menu" id="0e406ce9-5c43-4e2d-b741-e0b43d1f1865_Pop" class="article-tools__block fixed dropBlock__holder"><ul class="rlist w-slide--list"><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/rightsLink?doi=10.1111%2Fjbi.12786&amp;mode=" role="menuitem" target="_blank"><i aria-hidden="true" class="icon-permissions-key"></i><span>Request permission</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1111%2Fjbi.12786" role="menuitem" target="_self"><i aria-hidden="true" class="icon-export-icon"></i><span>Export citation</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/personalize/addFavoritePublication?doi=10.1111%2Fjbi.12786" role="menuitem" target="_self"><i aria-hidden="true" class="icon-favorites"></i><span>Add to favorites</span></a></li><li role="presentation" class="article-tool"><a href="https://onlinelibrary.wiley.com/action/addCitationAlert?doi=10.1111%2Fjbi.12786" role="menuitem" target="_self"><i aria-hidden="true" class="icon-Icon_Track-citations"></i><span>Track citation</span></a></li></ul></div></div>




        
        <!-- Go to www.addthis.com/dashboard to customize your tools --><script type="text/javascript" src="addthis_widget.js"></script><div class="relative"><a id="share__ctrl" href="#" data-db-target-for="dff42abe-4b17-4fe9-a9c5-625405cd1007" data-db-switch="icon-close_thin" aria-haspopup="true" aria-controls="share_Pop" role="button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i><span>Share</span></a><div data-db-target-of="dff42abe-4b17-4fe9-a9c5-625405cd1007" aria-labelledby="share__ctrl" role="menu" id="share_Pop" class="share__block dropBlock__holder fixed">



        
        <div data-readcube="/pb/widgets/readCubeP2PUrl/10.1111/jbi.12786/" class="readCube-sharing hidden"><p>Give access</p><a href="#" data-toggle="modal" data-target="#readCube" class="readCube-sharing__link"><i class="icon-users"></i>Share full text access</a></div><div class="ux-modal-container readCube-sharing__modal"><div id="readCube" class="modal"><div class="modal__dialog"><div class="modal__header"><button type="button" data-dismiss="modal" class="close"><i aria-hidden="true" class="icon-close_thin"></i></button><p>Share full text access</p></div><div class="modal__body"><form method="post"><fieldset><legend>Please review our <a href="http://onlinelibrary.wiley.com/termsAndConditions" target="_blank">Terms and Conditions of Use</a> and check box below to share full-text version of article.</legend><div class="input-group"><label for="terms-and-conditions" class="checkbox--primary"><input id="terms-and-conditions" value="yes" required="" name="terms-and-conditions" type="checkbox"><span class="label-txt">I have read and accept the Wiley Online Library Terms and Conditions of Use.</span></label></div></fieldset><hr class="separator"><fieldset><label>Shareable Link</label><p>Use the link below to share a full-text version of this article with your friends and colleagues. <a href="http://www.wiley.com/WileyCDA/Section/id-829333.html" target="_blank" class="emphasis">Learn more.</a></p><div class="shareable-link"><div class="shareable-link__field"><span id="shareable-link__text"></span></div></div><button type="submit" disabled="disabled" class="btn shareable-link__btn">Copy URL</button></fieldset></form></div></div></div></div>
<p>Share a link</p><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a class="addthis_button_email" target="_blank" title="Email" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-Email"></i>Email to a friend</a></li><li><a class="addthis_button_facebook" title="Facebook" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a class="addthis_button_twitter" title="Twitter" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-twitter-t"></i>Twitter</a></li><li><a class="addthis_button_linkedin" target="_blank" title="LinkedIn" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linkedin</a></li><li><a class="addthis_button_google_plusone_share" target="_blank" title="Google+" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-google-plus"></i>Google+</a></li><li><a class="addthis_button_reddit" target="_blank" title="Reddit" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-reddit-alien"></i>Reddit</a></li><li><a class="addthis_button_citeulike" target="_blank" title="CiteULike" href="#"><i aria-hidden="true" class="at-icon-wrapper icon-citeulike"></i>CiteULike</a></li><div class="atclear"></div></ul></div></div>
</div></div></nav><div class="article__body"><style>
            .inline-equation {
            overflow-x:auto;
            }
        </style><article>
   
   <section class="article-section article-section__abstract" data-lang="en" id="section-1-en" style="" lang="en">
      <h2 id="d5177291" class="article-section__header section__title main main">Abstract</h2>
      <div class="article-section__content en main">
         
         
         <div class="article-section__content" id="jbi12786-sec-0001">
            
            <h3 class="article-section__sub-title section1"> Aim</h3>
            
            <p>Comparative phylogeography across a large number of species allows investigating community‐level
               processes at regional and continental scales. An effective approach to such studies
               would involve automatic retrieval of georeferenced sequence data from nucleotide databases
               (a first step towards an ‘automated phylogeography’). It remains unclear if, despite
               repeated calls, georeferencing of nucleotide databases has increased in frequency,
               and if accumulated data allow for broad applications based on automated retrieval
               of sequence data and associated geographical information. Here, we investigated geographical
               information available in NCBI GenBank accessions for tetrapods, exploring temporal and geographical patterns in
               georeferencing, and quantifying data available for automated phylogeography.
            </p>
            
         </div>
         
         <div class="article-section__content" id="jbi12786-sec-0002">
            
            <h3 class="article-section__sub-title section1"> Location</h3>
            
            <p>Global.</p>
            
         </div>
         
         <div class="article-section__content" id="jbi12786-sec-0003">
            
            <h3 class="article-section__sub-title section1"> Methods</h3>
            
            <p>We developed Python and R scripts to (1) download metadata from GenBank (1,125,514
               accessions, &gt;&nbsp;20,000 species); (2) geocode accessions from associated metadata; (3)
               map originally georeferenced and geocoded accessions and plot their frequency against
               time; (4) assess the size of intraspecific sets of homologous sequences and compare
               their geographical extent with species ranges, thus evaluating their potential for
               phylogeographical analyses.
            </p>
            
         </div>
         
         <div class="article-section__content" id="jbi12786-sec-0004">
            
            <h3 class="article-section__sub-title section1"> Results</h3>
            
            <p>Only 6.2% of surveyed tetrapod GenBank submissions reported geographical coordinates,
               without increase in recent years. Our geocoding raised georeferenced accessions to
               15.1%. The geographical distribution of georeferenced accessions is patchy, and especially
               sparse in economically underdeveloped areas. Automatically retrievable informative
               data sets covering most of the range are available for very few species of wide‐ranging
               tetrapods.
            </p>
            
         </div>
         
         <div class="article-section__content" id="jbi12786-sec-0005">
            
            <h3 class="article-section__sub-title section1"> Main conclusions</h3>
            
            <p>Although geocoding offers a partial solution to the scarcity of direct georeferencing,
               the amount of data potentially useful for automated phylogeography is still limited.
               Strong underrepresentation of hard‐to‐access areas suggests that sampling logistics
               represent a main hindrance to global data availability. We propose that, besides enhancing
               georeferencing of genetic data, future research agendas should focus on collaborative
               efforts to sample genetic diversity in biodiversity‐rich tropical areas.
            </p>
            
         </div>
         
      </div>
   </section>
   
   <section class="article-section article-section__full">
      
      <div class="article-section__content" id="jbi12786-sec-0006">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0006-title"> Introduction</h2>
         
         <p>To what extent do species in a regional assemblage share long‐term demographic trends?
            What is the relative role of abiotic and biotic factors in shaping these trends? Ultimately,
            how much do local and regional communities represent stable, identifiable entities
            through time and space? Answers to these questions, and more, could emerge from a
            comparative analysis of the spatial distribution of genetic diversity of species,
            that is, from comparative phylogeography (Bermingham &amp; Moritz, <span><a href="#jbi12786-bib-0004" class="bibLink tab-link" data-tab="pane-pcw-references">1998</a></span>; Arbogast &amp; Kenagy, <span><a href="#jbi12786-bib-0003" class="bibLink tab-link" data-tab="pane-pcw-references">2001</a></span>; Hewitt, <span><a href="#jbi12786-bib-0017" class="bibLink tab-link" data-tab="pane-pcw-references">2004</a></span>; Hickerson <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0018" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>). Molecular genetic data are probably the most important source of information about
            demographic histories (e.g. Drummond <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0010" class="bibLink tab-link" data-tab="pane-pcw-references">2005</a></span>; Schiffels &amp; Durbin, <span><a href="#jbi12786-bib-0034" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>), and the analysis of spatial patterns of genetic diversity (and of gene genealogies
            in particular, i.e. phylogeography) has become the tool of choice to infer range dynamics
            (e.g. Hewitt, <span><a href="#jbi12786-bib-0016" class="bibLink tab-link" data-tab="pane-pcw-references">2000</a></span>).
         </p>
         
         <p>As the spatial distribution of biodiversity emerges from the sum of range dynamics
            of independently evolving lineages (species), statistical comparisons of phylogeographical
            patterns across species may play a paramount role in revealing how current patterns
            of biodiversity were shaped by dispersal, diversification and competition (Marske
            <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0023" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>). However, most work published so far in the field of comparative phylogeography
            consists of two broad categories of studies, each with their own limitations. On the
            one hand, original research papers, although often relying on well‐designed sampling
            schemes and rigorous statistical approaches, usually cover relatively small taxonomic
            or geographical scales (e.g. Carnaval <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0006" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>; Moritz <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0025" class="bibLink tab-link" data-tab="pane-pcw-references">2009</a></span>). On the other hand, review articles taking a larger scale approach typically consist
            of descriptive accounts of published results (e.g. Lorenzen <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0020" class="bibLink tab-link" data-tab="pane-pcw-references">2012</a></span>), while statistically rigorous meta‐analyses remain rare and hindered by the challenge
            of comparing results obtained from different methods and study designs (Dawson, <span><a href="#jbi12786-bib-0009" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>). While the scope of original research papers tends to be obviously constrained by
            the amount of data that can be produced within a single research project, the weakness
            of the meta‐analysis and review approaches often depends on analysing interpretations,
            or very simple summaries, of published data (not always with support from a solid
            statistical analysis), rather than the data themselves (e.g. Pyron &amp; Burbrink, <span><a href="#jbi12786-bib-0030" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; Shafer <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0036" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>). Recent, elegant efforts towards statistically robust meta‐analyses have been published.
            However, they are often restricted to relatively simple spatial systems and/or implied
            the tiresome manual compilation of data sets from only those studies that reported
            the appropriate summary statistics (e.g. Riginos <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0031" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>, <span><a href="#jbi12786-bib-0032" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>; Paz‐Vinas <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0028" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>).
         </p>
         
         <p>For comparative phylogeography to mature as a research field, large amounts of primary
            data should be readily available to be analysed within a consistent framework. According
            to this view, an approach to comparative phylogeography involving automated retrieval
            of sequence data from nucleotide databases has the potential to bridge the gap between
            ‘review/meta‐analysis’ and ‘single study’ approaches. This might represent a first
            step towards an ‘automated phylogeography’, that is, a set of tools for fully automated,
            self‐updating analyses of available published data (for an example of a recent effort
            towards automated analyses in the field of phylogenetics, see Antonelli <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0002" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>). These might include web‐based interfaces providing, for example, estimates of demographic
            trends for all species of mammals with available genetic data within a queried area,
            or maps of local intraspecific genealogical depth for a custom set of taxa. As most
            biological journals require that published nucleotide sequence data are deposited
            in INSDC databanks (International Nucleotide Sequence Database Collaboration: NCBI,
            DDBJ, EMBL, see Nakamura <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0026" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>), these represent the main resource for any research application based on DNA data
            retrieval. In 2005, NCBI GenBank introduced special fields for latitude and longitude,
            which allows for an immediate connection of nucleotide sequence data to the geographical
            location where samples were collected and opening the way for efficient automated
            retrieval of phylogeographically informative data. More recently, many journals have
            been asking that genetic data sets (usually, but not necessarily, including associated
            metadata) are made available as supplementary materials, or released in data repositories
            (e.g. <a href="http://datadryad.org/" class="linkBehavior">datadryad.org</a>). While this represented an important improvement towards reproducibility of analyses
            (see e.g. Mesirov, <span><a href="#jbi12786-bib-0024" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>), the diversity of sources and data formats makes this form of data storage rather
            unpractical as a resource for large‐scale data analyses (see Hardisty &amp; Roberts, <span><a href="#jbi12786-bib-0015" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>; Tedersoo <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0038" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>). Very recently, Weissenbacher <i>et&nbsp;al</i>. (<span><a href="#jbi12786-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>) reported an attempt to automatically link geographical information contained in
            published papers to databases accessions. However, the precision of such methods is
            still limited, and improvements in their efficiency ultimately depend on interfacing
            information extracted from papers with metadata in the primary nucleotide databases
            (Weissenbacher <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0039" class="bibLink tab-link" data-tab="pane-pcw-references">2015</a></span>). Therefore, direct georeferencing of DNA sequences deposited in INSDC databases,
            such as the NCBI GenBank, appears as the most promising resource for a global‐scale
            comparative phylogeography, and an exceptionally valuable resource for biogeography
            in general. Key aspects of biogeography, such as the estimation of global patterns
            of speciation/extinction rates, for example, would hugely benefit from readily available
            intraspecific georeferenced sequence data. Analyses of the geographical distribution
            of intraspecific genetic diversity can indicate where each species has persisted longest
            over time, a possible proxy for the region where species originated (see Losos &amp; Glor,
            <span><a href="#jbi12786-bib-0021" class="bibLink tab-link" data-tab="pane-pcw-references">2003</a></span>, for an argument on why regular phylogenetic inference may not be sufficient for
            this task). Moreover, georeferenced sequence data might also serve as a powerful archive
            of biodiversity in the face of global change, allowing, for example, to track medium‐term
            changes in allele frequencies and population structure.
         </p>
         
         <p>An appeal for generalized georeferencing of biodiversity related data (and GenBank
            sequences in particular) was launched in a <i>Nature</i> editorial article in 2008 (Anonymous, <span><a href="#jbi12786-bib-0001" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>; but see also Guralnick <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0014" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>), and was warmly welcomed by a substantial group of researchers and institutions
            (Anonymous, <span><a href="#jbi12786-bib-0001" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>) and reiterated, for example, in a more recent letter to <i>Science</i> (Marques <i>et al</i>., <span><a href="#jbi12786-bib-0022" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>). Since then, efforts were made to integrate nucleotide data and geographical information,
            for example, with the recent publication of the ‘Geographically tagged INSDC sequences’
            into the GBIF portal (<a href="http://www.gbif.org/" class="linkBehavior">www.gbif.org</a>). Another, very important, contribution to the construction of a publicly available
            framework linking biogeographical and nucleotide data came with the establishment,
            in 2004, of the Consortium for the Barcode of Life (CBOL). CBOL has made a substantial
            move in this direction by requiring that the geographical origin of samples is recorded
            in the Barcode Of Life Data system (BOLD). Earlier attempts at estimating the amount
            of INSDC data with sufficient level of geographical information were reported by Scotch
            <i>et&nbsp;al</i>. (<span><a href="#jbi12786-bib-0035" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>) on a subset of nucleotide sequences from RNA viruses and by Ryberg <i>et&nbsp;al</i>. (<span><a href="#jbi12786-bib-0033" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>) and Tedersoo <i>et&nbsp;al</i>. (<span><a href="#jbi12786-bib-0037" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>) on mycorrhizal fungi. However, these studies (1) focused on taxa with peculiar biogeographical
            features (dispersal of most well‐known RNA viruses is generally human‐mediated) and/or
            relatively limited taxonomic resolution and (2) did not explore temporal patterns
            in georeferenced sequence data and the completeness of geographical coverage for each
            species. Therefore, it remains unclear how much the calls for an integration of genetic
            and biogeographical data have been heeded, how much they translated into an increased
            frequency of nucleotide data georeferencing, and how much the accumulated data allow
            for a comparative phylogeography based on the automated retrieval of large multispecies
            data sets.
         </p>
         
         <p>While the importance of linking genetic and geographical data was being brought to
            the attention of the scientific community, the next‐Generation sequencing revolution
            (NGS; Ekblom &amp; Galindo, <span><a href="#jbi12786-bib-0012" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>) created new opportunities for a global description of spatial genetic structures
            across population and species. On the one hand, NGS techniques can be used to sequence
            many copies of a few genetic markers (e.g. via amplicon sequencing: e.g. Bybee <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0005" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>; Wielstra <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0040" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>) that are then stored in standard databases like the GenBank. On the other hand,
            the more typical NGS application is the production of millions of genome‐wide short
            sequence reads. Dedicated platforms, as the Short Reads Archive (SRA, <a href="http://www.ncbi.nlm.nih.gov/sra" class="linkBehavior">www.ncbi.nlm.nih.gov/sra</a>) allow storage and publicly sharing of huge amounts of data. As with non‐NGS INSDC
            databanks, SRA can store georeferencing information about each sample, but this is
            not required in order to upload sequence data. Moreover, analyses including several
            species/taxa would imply prohibitive computational loads, if raw data from several
            different studies must be processed and compared for each taxon. Last, and possibly
            more importantly, availability of NGS‐generated genomic data is still restricted to
            a relatively small number of species, usually with a small number of specimens on
            a limited geographical extent, so that the potential of this kind of data for comparative
            phylogeography and for tracking genetic diversity across space and time is still relatively
            limited.
         </p>
         
         <p>In this study, therefore, we quantified georeferencing across INSDC accessions available
            through the NCBI GenBank, excluding NGS‐dedicated platforms. We explored temporal
            trends (is georeferencing of sequence data becoming more/less common with time?),
            geographical patterns (is georeferencing more common in some regions than others?)
            and distribution across different sets of publications and genomic regions (which
            classes of genetic markers and research topics are contributing more georeferenced
            data?). Moreover, we estimated the amount of INSDC sequence data that can potentially
            be used for an automated approach to large‐scale comparative phylogeography (i.e.
            an approach that avoids the time‐consuming step of manually retrieving geographical
            information from the original publications). For our survey, we focused on terrestrial
            vertebrates (tetrapods), a large taxon for which abundant distributional and ecological
            data are available.
         </p>
         
         <p>To this end, we (1) searched and stored INSDC accessions metadata for a list of 32,542
            tetrapod species; (2) developed a custom geocoding pipeline to retrieve geographical
            coordinates from the GenBank textual ‘country’ field; (3) analysed temporal patterns
            in INSDC georeferencing across different subsets of data; (4) examined the geographical
            distribution of georeferenced accessions; (5) clustered georeferenced sequence by
            sequence similarity within species and evaluated the geographical coverage of alignable
            sequence clusters to quantify the amount of alignable data sets that could be automatically
            retrieved and used in phylogeographical studies (i.e. usable in an automated phylogeography).
         </p>
         
      </div>
      
      <div class="article-section__content" id="jbi12786-sec-0007">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0007-title"> Methods</h2>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0008">
            
            <h3 class="article-section__sub-title section2"> Obtaining sequence metadata from GenBank</h3>
            
            <p>We downloaded authoritative lists of 32,542 tetrapod species from class‐specific taxonomy
               databases available online (see Appendix&nbsp;S1 in Supporting Information for details).
               We then translated Linnean binomials into NCBI species taxonomy identifiers (taxonIDs)
               by submitting the lists of species to the NCBI Taxonomy name/id Status Report Page
               and obtained 21,262 unique species‐level taxonIDs. We used species taxonIDs to search
               the NCBI GenBank and collaborative databases adhering to the INSDC using a custom
               Python script, filtering out RNA sequences, sequences from whole genome shotguns and
               other data categories (see Appendix&nbsp;S1 for details). GenBank searches were performed
               separately for each class from 21 November 2014 (mammals) to 9 December 2014 (reptiles).
            </p>
            
            <p>For each accession that matched our search criteria, we stored the associated metadata
               (see Supplementary Methods for a list of stored fields). Humans (<i>Homo sapiens</i>) and a few highly synanthropic or domesticated species (listed in Appendix&nbsp;S1) were
               excluded from our search. The complete list of searched taxa is available in Appendix&nbsp;S2.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0009">
            
            <h3 class="article-section__sub-title section2"> Georeferencing and geocoding of GenBank data</h3>
            
            <p>We considered as ‘originally georeferenced’, those accessions that contained unambiguous
               geographical coordinates in the ‘lat_lon’ field in one of the following formats: decimal
               degrees NSEW; decimal degrees with ‘–’ to indicate S or W; degrees with decimal minutes.
               Although direct reporting of geographical coordinates is obviously the most convenient
               way to make geographical information available, many GenBank accessions do contain
               information on the geographical origin of the sequenced samples in the ‘country’ field,
               which may contain very different levels of detail about the sampling location (see
               Scotch <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0035" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>). We considered as potentially ‘informative’ those accessions whose ‘country’ field
               contained some additional information besides the country itself (i.e. contained at
               least one ‘:’ character – the almost universal separator that follows country names
               in GenBank ‘country’ fields – followed by a space, a number or a letter).
            </p>
            
            <p>In order to estimate the amount of geographical information that could be automatically
               retrieved from GenBank metadata, we attempted to geocode (assign geographical coordinates
               based on a textual description) the subset of ‘informative’ accessions using a custom
               pipeline written in R and relying on geographical information from two publicly available
               databases: Global Administrative Areas (GADM) and GeoNames. Our geocoding pipeline
               is described in more detail in Appendix&nbsp;S1. In short, we started by isolating the
               country name from the rest of the text string in each unique ‘country’ field and removing
               most text not referring to placenames. The resulting character strings were then split
               at common separators and each substring was searched for in the subset of GADM and
               GeoNames databases relative to the country of interest. We recorded whether each substring
               appeared as such in the names (and variant names) of the administrative units from
               level 1 to level 5 in the GADM database. When a substring matched more than one administrative
               levels, we conservatively chose to consider the highest level match. Any substring
               not matching in the GADM database was searched for (as an exact match or partial match)
               in the GeoNames database. If at least one match was found in GADM for the same record,
               the substring was only searched among those GeoNames placenames whose geographical
               coordinates fell within the extent of the polygon corresponding to the matched GADM
               administrative unit. When multiple matches occurred, we calculated distance matrices
               among all matched locations: if the maximum distance among matched locations was smaller
               than 100&nbsp;km, we assigned the substring to the centroid of matched locations, if it
               was larger, no coordinates were assigned. Accessions that were unambiguously assigned
               to an administrative unit smaller than 10,000&nbsp;km<sup>2</sup> or to a GeoNames toponym not representing an administrative feature were finally
               considered as successfully geocoded. In order to check the precision of our geocoding,
               we calculated the distance between our retrieved coordinates and the coordinates provided
               by the authors for those successfully geocoded unique ‘country’ fields that were originally
               georeferenced.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0010">
            
            <h3 class="article-section__sub-title section2"> Taxonomic, temporal and spatial patterns in GenBank georeferencing</h3>
            
            <p>We first assessed the relative frequency of original georeferencing across GenBank
               accessions for each tetrapod class. We then analysed the temporal variation in the
               relative frequency of georeferencing across GenBank published submissions (here a
               ‘submission’ was defined as the combination of publishing year, authors list, journal
               and title extracted from the GenBank metadata) in the global data set and in four
               classes of publications: publications whose accessions are mostly (&gt;&nbsp;75%) mitochondrial
               DNA (mtDNA); publications with a reference to ‘phylogeography’, ‘geographic structure’,
               or similar in the title (title contains the string ‘geograph’); publications with
               reference to DNA barcoding in the title (title contains the string ‘barcod’); publications
               whose sequences are all deposited in the BOLD database (<a href="http://www.boldsystems.org/searched" class="linkBehavior">http://www.boldsystems.org/searched</a> 29 January 2015), not counting BOLD sequences that were ‘mined from GenBank’. We
               also explored the spatial distribution of georeferenced (both original and geocoded)
               GenBank data by counting the number of GenBank accessions and the number of species
               with at least one sequence on a global 200&nbsp;×&nbsp;200&nbsp;km grid, and, for each class, we
               assessed the correlation between the number of originally georeferenced sequences
               in any map cell and tetrapod species richness in the same cell. Species richness layers
               were obtained by summing species ranges on the same 200&nbsp;×&nbsp;200&nbsp;km global grid (an approach
               similar to Jenkins <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0019" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>). Polygons of species ranges were obtained by IUCN (<a href="http://www.iucnredlist.org/" class="linkBehavior">www.iucnredlist.org</a>, mammals and amphibians) and BirdLife (<a href="http://www.birdlife.org/" class="linkBehavior">www.birdlife.org</a>, birds). We did not perform the same comparison for reptiles as, while IUCN and BirdLife
               spatial data are taxonomically comprehensive for mammals, amphibians and birds, no
               similar complete data set is currently available for reptiles. Last, we estimated
               the number of INSDC accessions and the relative frequency of georeferencing by country,
               based on unambiguous country names in the ‘country’ field.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0011">
            
            <h3 class="article-section__sub-title section2"> Evaluating data availability for automated phylogeography</h3>
            
            <p>To be potentially informative for phylogeography, a set of sequence data must (1)
               contain a number of gene copies from the same taxon (e.g. from the same species or
               cluster of closely related species); (2) represent a substantial portion of the range
               of the taxon of interest. In order to evaluate the amount of data potentially useful
               for automated phylogeography we (1) clustered our set of georeferenced GenBank accessions
               by sequence similarity and assessed the number of sequences in each cluster within
               each species (2) measured the spatial coverage of such within‐species clusters relative
               to the distribution range of the species.
            </p>
            
            <p>We obtained spatial data (shapefiles) for species ranges from the IUCN website (<a href="http://www.iucnredlist.org/technical-documents/spatial-data" class="linkBehavior">www.iucnredlist.org/technical-documents/spatial-data</a>, downloaded 11 November 2014; amphibians: 6312 species; birds: 10,254 species; mammals:
               5291 species, excluding marine mammals; reptiles: 3903 species). Therefore, we limited
               this analysis to those georeferenced GenBank accessions whose organism name (field
               ‘organism’) could be unambiguously matched to a Linnean binomial listed in the IUCN
               spatial data. More specifically, we first searched for exact matches between genus
               names appearing in the INSDC accession (first word of the field ‘organism’) and the
               genus name of the Linnean binomial as reported in the IUCN shapefiles. To account
               for typing errors, non‐matching genus names were then searched for partial matches,
               and the type of match was recorded (‘exact match’, ‘partial match’, ‘genus not present’).
               Once assigned a (putative) genus name, we searched for the second term of the field
               ‘organism’ within the species names of the selected genus. Again, to account for typing
               errors and changes in species name/gender (e.g. <i>Actitis macularia</i> versus <i>Actitis macularius</i>), non‐matching species names were searched for partial matches. Finally, all partial
               matches were manually checked for concordance, and 23 binomials with apparent mismatch
               were excluded.
            </p>
            
            <p>Selected accessions were clustered by sequence similarity within each genus, using
               a custom R script calling for the fast sequence clustering algorithm uclust (Edgar,
               <span><a href="#jbi12786-bib-0011" class="bibLink tab-link" data-tab="pane-pcw-references">2010</a></span>; implemented in UCLUSTQ 1.2.22). Sequences shorter than 200&nbsp;bp (1846 accessions)
               were excluded from clustering, as well as sequences longer than 5000&nbsp;bp (233 accessions).
               The latter choice aimed at avoiding clustering of unrelated sequences to a few, very
               long sequences (e.g. complete mtDNA: our data set contained 136 georeferenced complete
               mtDNA sequences; maximum for a single species&nbsp;=&nbsp;6). We set the minimum identity threshold
               for clustering at 0.80, and run uclust with the –optimal option. The latter choice
               prompts a clustering algorithm ensuring that each sequence is clustered together with
               its best match, and minimizes the number of recovered clusters given the chosen threshold.
               All sequence clusters that contained at least 20 sequences from the same species were
               retained as potentially interesting.
            </p>
            
            <p>To evaluate the spatial coverage of each cluster relative to the species range, we
               first rasterized the shapefiles for each range to a <i>c</i>.&nbsp;100&nbsp;×&nbsp;100&nbsp;km grid (Behrmann cylindrical equal area projection), we then partitioned
               the resulting cells in 20 clusters of neighbouring cells using a k‐means clustering
               (R function kmeans() {stats}with 1000 random starts and 1000 iterations) and, last,
               we counted the number of such clusters (corresponding to 20 Voronoi cells) that contained
               at least one sequence. All species whose ranges intersected &lt;&nbsp;20 cells (roughly 200,000&nbsp;km<sup>2</sup>) were excluded from this analysis.
            </p>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="jbi12786-sec-0012">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0012-title"> Results</h2>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0013">
            
            <h3 class="article-section__sub-title section2"> Obtaining sequence metadata from GenBank</h3>
            
            <p>Our search returned metadata for 1,125,514 accessions (see Fig.&nbsp;<a href="#jbi12786-fig-0001">1</a> for the distribution across classes), with 26,077 unique taxonIDs at the specific
               or subspecific level, representing 20,810 Linnean binomials (the exact number of species
               may vary due to taxonomic instability) and 4201 unique taxonIDs at the genus level.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="jbi12786-fig-0001"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/13c02aab-0d2e-442c-9313-e09c30efbcc2/jbi12786-fig-0001-m.jpg"><img class="figure__image" src="jbi12786-fig-0001-m.jpg"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 1</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=jbi12786-fig-0001&amp;doi=10.1111%2Fjbi.12786" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>Proportion of originally georeferenced data in a selection of 1,125,514 tetrapods
                        INSDC accessions. Length of bars and black numbers besides bars represent the relative
                        frequency of georeferencing across accessions for a given class. Bar depths and white
                        number within bars represent the total number of accessions for each class. Full color
                        version available online [Colour figure can be viewed at <a href="http://wileyonlinelibrary.com/" class="linkBehavior">wileyonlinelibrary.com</a>].
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="fajbi12786-fig-0001" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="fajbi12786-fig-0001" class="accordion__content" style="display: none;">Proportion of originally georeferenced data in a selection of 1,125,514 tetrapods
                           INSDC accessions. Length of bars and black numbers besides bars represent the relative
                           frequency of georeferencing across accessions for a given class. Bar depths and white
                           number within bars represent the total number of accessions for each class. Full color
                           version available online [Colour figure can be viewed at wileyonlinelibrary.com].
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
         </div>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0014">
            
            <h3 class="article-section__sub-title section2"> Georeferencing and geocoding of GenBank data</h3>
            
            <p>The geographical coordinates (‘lat_lon’) field was filled for 69,782 accession (6.20%
               of all retrieved accessions), with 161 of them containing non‐numeric strings. In
               the following, we consider as ‘originally georeferenced’ the 69,483 accessions (6.17%)
               containing consistently interpretable latitude and longitude in one of the three most
               common formats (decimal degrees NSEW: 68,855 accessions; decimal degrees with ‘–’
               to indicate S or W: 421 accessions; degrees with decimal minutes: 207 accessions).
               We note that, as we did not check for consistency of coordinates at this stage (e.g.
               we did not check that a point for a terrestrial animal did not fall in the ocean),
               it is not guaranteed that all of the originally georeferenced accessions are also
               correctly georeferenced.
            </p>
            
            <p>The ‘country’ field was populated in 379,200 accessions (33.7%), while 377,357 (33.5%)
               contained unambiguous country names (ambiguous names include, for example, names referring
               to dissolved political entities, such as ‘Yugoslavia’). A total of 270,476 (24.0%)
               of them were considered as ‘informative’ (39,813 unique ‘country’ fields, with 39,627
               of them containing a valid country name). According to our criteria, 23,238 unique
               ‘country’ fields (58.6% of the unique searched fields) were successfully geocoded
               (i.e. they were unambiguously assigned to an administrative unit smaller than 10,000&nbsp;km<sup>2</sup> or to a point placename). Our geocoding thus successfully assigned coordinates to
               101,082 accessions, thus raising the total amount of georeferenced accessions to 170,565
               (15.1% of the total retained accession). Among the successfully geocoded accessions,
               4624 were originally georeferenced, and were used to check geocoding precision. We
               found that assigned coordinates were within 100&nbsp;km of the original coordinates in
               90.3% of cases, and within 200&nbsp;km in 94.6% of cases.
            </p>
            
         </div>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0015">
            
            <h3 class="article-section__sub-title section2"> Taxonomic, temporal and spatial patterns in INSDC georeferencing</h3>
            
            <p>The relative frequency of original georeferencing varied across classes, being markedly
               lower in reptiles than in amphibians, birds or mammals (Fig.&nbsp;<a href="#jbi12786-fig-0001">1</a>).
            </p>
            
            <p>The analysed set of tetrapods GenBank accessions has been accumulating quasi‐exponentially
               in the last 25&nbsp;years (Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>a). Georeferenced accessions increased rapidly for <i>c</i>.&nbsp;5&nbsp;years after the introduction of the ‘lat_lon’ field in 2005 (the few georeferenced
               accessions pre‐dating 2005 result from post‐publishing additions), but, at least in
               the last 5–6&nbsp;years, their growth rate did not exceed that of the total data set. Georeferenced+geocoded
               accessions accumulated at a steeper rate than the total data before the year 2000,
               but then flattened out too. Temporal trends in the frequency of submissions whose
               accessions are mostly georeferenced is shown in Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>(b). The trend for the total data set reflects the accession‐wise growth pattern in
               Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>(a), with an increase in 2005–2010 and subsequent flattening out, at a rather low
               frequency <i>c</i>.&nbsp;5–6%. Mitochondrial DNA sequences (mtDNA) have been the workhorse for phylogeographical
               inference since the late 1980s, therefore, it would be reasonable to expect that papers
               employing mtDNA as the main marker (i.e. &gt;&nbsp;75% of related accessions) display a markedly
               higher frequency of georeferencing. Figure&nbsp;<a href="#jbi12786-fig-0002">2</a>(b) shows that this is not the case, as the values for mtDNA‐based submissions are
               just slightly above the total data set in the original+geocoded data, and essentially
               average in the originally georeferenced data. Even more unexpectedly, papers whose
               title contains explicit reference to geography also do not differ markedly from the
               total data set considered in this study (Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>b). As the Barcode of Life database (BOLD) strongly recommends georeferencing for
               all accessions deposited there, it is not surprising that the few submissions whose
               sequences are (mostly) deposited in BOLD have a very high georeferencing frequency
               (Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>b). Georeferencing frequency across papers referring to ‘barcode’ in the title is
               lower, but still markedly higher than in the global data set (Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>b). The georeferencing frequency of both barcode‐related sets of submissions apparently
               decreases with time, and particularly so for submissions referring to DNA barcode
               in their title (Fig.&nbsp;<a href="#jbi12786-fig-0002">2</a>b). Nonetheless, BOLD accessions, which represented only 3.4% of the analysed data,
               contributed a large share of the total georeferenced sequences (20.2%), and about
               half (47.3%) of the originally georeferenced accessions.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="jbi12786-fig-0002"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/4455e6ab-c9d2-4616-a337-681b79dc0983/jbi12786-fig-0002-m.jpg"><img class="figure__image" src="jbi12786-fig-0002-m.jpg"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 2</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=jbi12786-fig-0002&amp;doi=10.1111%2Fjbi.12786" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>Accumulation of georeferenced GenBank 
accessions and frequency of submissions with
                        mostly georeferenced data. (a) Growth of total 
and georeferenced NCBI GenBank accessions. Cumulative distribution of a 
selection of 934,556 published tetrapods
                        accessions (red, continuous line) by year of 
publication. Cumulative distribution
                        of accessions reporting an unambiguously 
interpretable ‘lat_lon’ field (originally
                        georeferenced: blue, dashed line) and accessions
 that were georeferenced, including
                        geocoded accessions (originally 
georeferenced&nbsp;+&nbsp;geocoded: blue, continuous line).
                        Data are presented on a logarithmic vertical 
scale. (b) Relative frequency of submission
                        that reported geographical coordinates for most 
of their accessions across time in
                        a selection of 13,144 unique published 
submissions to GenBank containing tetrapods
                        sequences. The proportion of published 
submissions whose sequences are georeferenced
                        for more than 50% is plotted against the 
publication year for the total data set (red)
                        and four subsets of data: submissions containing
 mostly (&gt;&nbsp;75%) mtDNA sequences (<i>N&nbsp;</i>=<i>&nbsp;</i>6187, blue); submission containing a reference to geography (string ‘geograph’) in
                        publication title (<i>N&nbsp;</i>=<i>&nbsp;</i>1347, green); submissions containing a reference to DNA barocoding (string ‘barcod’) in publication title (<i>N&nbsp;</i>=<i>&nbsp;</i>71, purple); submissions whose sequences are all deposited in the BOLD (Barcode Of Life Database) systems (<i>N&nbsp;</i>=<i>&nbsp;</i>39,
 orange). The area of circles is proportional to the logarithm of the 
sample size
                        for each year. Lines are LOESS trends since 2005
 (special ‘lat_lon’ field introduced in the GenBank). Full color
                        version available online [Colour figure can be 
viewed at <a href="http://wileyonlinelibrary.com/" class="linkBehavior">wileyonlinelibrary.com</a>].
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="fajbi12786-fig-0002" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="fajbi12786-fig-0002" class="accordion__content" style="display: none;">Accumulation of georeferenced GenBank accessions and frequency of submissions with
                           mostly georeferenced data. (a) Growth of total and georeferenced NCBI GenBank accessions.
                           Cumulative distribution of a selection of 934,556 published tetrapods accessions (red,
                           continuous line) by year of publication. Cumulative distribution of accessions reporting
                           an unambiguously interpretable ‘lat_lon’ field (originally georeferenced: blue, dashed
                           line) and accessions that were georeferenced, including geocoded accessions (originally
                           georeferenced&nbsp;+&nbsp;geocoded: blue, continuous line). Data are presented on a logarithmic
                           vertical scale. (b) Relative frequency of submission that reported geographical coordinates
                           for most of their accessions across time in a selection of 13,144 unique published
                           submissions to GenBank containing tetrapods sequences. The proportion of published
                           submissions whose sequences are georeferenced for more than 50% is plotted against
                           the publication year for the total data set (red) and four subsets of data: submissions
                           containing mostly (&gt;&nbsp;75%) mtDNA sequences (N&nbsp;=&nbsp;6187, blue); submission containing
                           a reference to geography (string ‘geograph’) in publication title (N&nbsp;=&nbsp;1347, green);
                           submissions containing a reference to DNA barocoding (string ‘barcod’) in publication
                           title (N&nbsp;=&nbsp;71, purple); submissions whose sequences are all deposited in the BOLD
                           (Barcode Of Life Database) systems (N&nbsp;=&nbsp;39, orange). The area of circles is proportional
                           to the logarithm of the sample size for each year. Lines are LOESS trends since 2005
                           (special ‘lat_lon’ field introduced in the GenBank). Full color version available
                           online [Colour figure can be viewed at wileyonlinelibrary.com].
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>Originally georeferenced sequences are sparsely distributed, with large areas of land
               lacking any data (especially in socio‐economically underdeveloped and/or scarcely
               inhabited regions) (Fig.&nbsp;<a href="#jbi12786-fig-0003">3</a>a). Our geocoding substantially increased the amount of mappable data, but did not
               alter the general pattern (Fig.&nbsp;<a href="#jbi12786-fig-0003">3</a>b). The same considerations apply for the number of species with mappable sequence
               data (Fig.&nbsp;<a href="#jbi12786-fig-0004">4</a>a,b). Moreover, data from the different taxonomic classes do not follow a very consistent
               pattern, with low to modest correlation coefficients (see Appendix&nbsp;S3). In all classes,
               we found weak (though significant) correlations between the number of sequences with
               original georeferencing to a given grid cell (see Fig.&nbsp;<a href="#jbi12786-fig-0003">3</a>a) and the number of species predicted in the same cell (Pearson's <i>r</i>, amphibians: 0.10, <i>P&nbsp;</i>=<i>&nbsp;</i>0.015; birds: 0.23, <i>P&nbsp;</i>&lt;<i>&nbsp;</i>0.001; mammals 0.28, <i>P&nbsp;</i>&lt;<i>&nbsp;</i>0.001), and between the number of species with sequences (originally) georeferenced
               to a given cell (see Fig.&nbsp;<a href="#jbi12786-fig-0004">4</a>a) and the number of species predicted in the cell (Pearson's <i>r</i>, amphibians: 0.28, birds: 0.22, mammals: 0.32; all <i>P&nbsp;</i>&lt;<i>&nbsp;</i>0.001).
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="jbi12786-fig-0003"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/fd1a636f-ed4f-4591-9f40-399934abb1ae/jbi12786-fig-0003-m.jpg"><img class="figure__image" src="jbi12786-fig-0003-m.jpg"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 3</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=jbi12786-fig-0003&amp;doi=10.1111%2Fjbi.12786" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>Spatial distribution of georeferenced sequences in a selection of 1,125,514 tetrapods
                        GenBank accessions. Maps show the number of sequences georeferenced to each cell of
                        a 200&nbsp;×&nbsp;200&nbsp;km grid. (a) Originally georeferenced sequences; (b) total georeferenced
                        sequences (original&nbsp;+&nbsp;geocoded); (c) sequences belonging to intraspecific clusters
                        with at least 20 sequences and whose geographical extent covers at least 30% of the
                        species range (see text for details). The colour scale is proportional to the logarithm
                        of the number of sequences. All maps are in the Lambert cylindrical equal area projection.
                        Full color version available online [Colour figure can be viewed at <a href="http://wileyonlinelibrary.com/" class="linkBehavior">wileyonlinelibrary.com</a>].
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="fajbi12786-fig-0003" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="fajbi12786-fig-0003" class="accordion__content" style="display: none;">Spatial distribution of georeferenced sequences in a selection of 1,125,514 tetrapods
                           GenBank accessions. Maps show the number of sequences georeferenced to each cell of
                           a 200&nbsp;×&nbsp;200&nbsp;km grid. (a) Originally georeferenced sequences; (b) total georeferenced
                           sequences (original&nbsp;+&nbsp;geocoded); (c) sequences belonging to intraspecific clusters
                           with at least 20 sequences and whose geographical extent covers at least 30% of the
                           species range (see text for details). The colour scale is proportional to the logarithm
                           of the number of sequences. All maps are in the Lambert cylindrical equal area projection.
                           Full color version available online [Colour figure can be viewed at wileyonlinelibrary.com].
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="jbi12786-fig-0004"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/df38a7de-30fb-423d-9747-5c2100116930/jbi12786-fig-0004-m.jpg"><img class="figure__image" src="jbi12786-fig-0004-m.jpg"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 4</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=jbi12786-fig-0004&amp;doi=10.1111%2Fjbi.12786" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>Spatial distribution of species with 
georeferenced sequences in a selection of 1,125,514
                        tetrapods GenBank accessions. Maps show the 
number of species with at least one sequence
                        georeferenced to each cell of a 
200&nbsp;×&nbsp;200&nbsp;km grid. (a) Originally georeferenced 
sequences;
                        (b) total georeferenced sequences 
(original&nbsp;+&nbsp;geocoded); (c) sequences belonging to
                        intraspecific clusters with at least 20 
sequences and whose geographical extent covers
                        at least 30% of the species range (see text for 
details). The colour scale is proportional
                        to the logarithm of the number of species. All 
maps are in the Lambert cylindrical
                        equal area projection. Full color version 
available online [Colour figure can be viewed
                        at <a href="http://wileyonlinelibrary.com/" class="linkBehavior">wileyonlinelibrary.com</a>].
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="fajbi12786-fig-0004" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="fajbi12786-fig-0004" class="accordion__content" style="display: none;">Spatial
 distribution of species with georeferenced sequences in a selection of 
1,125,514
                           tetrapods GenBank accessions. Maps show the 
number of species with at least one sequence
                           georeferenced to each cell of a 
200&nbsp;×&nbsp;200&nbsp;km grid. (a) Originally georeferenced 
sequences;
                           (b) total georeferenced sequences 
(original&nbsp;+&nbsp;geocoded); (c) sequences belonging to
                           intraspecific clusters with at least 20 
sequences and whose geographical extent covers
                           at least 30% of the species range (see text 
for details). The colour scale is proportional
                           to the logarithm of the number of species. 
All maps are in the Lambert cylindrical
                           equal area projection. Full color version 
available online [Colour figure can be viewed
                           at wileyonlinelibrary.com].
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <p>The number of INSDC accessions by country and the relative frequency of originally
               georeferenced data are shown in Fig.&nbsp;<a href="#jbi12786-fig-0005">5</a>. Most African countries have very few data, and there is no strong regional pattern
               in the relative frequency of georeferencing, as above average (red) and below average
               (blue) countries are scattered across every continent.
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="jbi12786-fig-0005"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/ac8e328e-f0a4-4e18-b945-1e724b3cf14a/jbi12786-fig-0005-m.jpg"><img class="figure__image" src="jbi12786-fig-0005-m.jpg"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 5</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=jbi12786-fig-0005&amp;doi=10.1111%2Fjbi.12786" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>Georeferencing by country in a selection of 1,125,514 tetrapods GenBank accessions.
                        In Fig.&nbsp;<a href="#jbi12786-fig-0005">5</a>(a) colour intensity is proportional to the number of GenBank accessions that can
                        be traced to each country (unambiguous country name appearing in the ‘country’ field)
                        divided by the country area (log scale). In Fig.&nbsp;<a href="#jbi12786-fig-0005">5</a>(b), colour scale indicates the proportion of originally georeferenced sequences among
                        those that can be tracked to each country (white indicates no data). All maps are
                        in the Lambert cylindrical equal area projection. Full color version available online
                        [Colour figure can be viewed at <a href="http://wileyonlinelibrary.com/" class="linkBehavior">wileyonlinelibrary.com</a>].
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="fajbi12786-fig-0005" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="fajbi12786-fig-0005" class="accordion__content" style="display: none;">Georeferencing by country in a selection of 1,125,514 tetrapods GenBank accessions.
                           In Fig.&nbsp;(a) colour intensity is proportional to the number of GenBank accessions that
                           can be traced to each country (unambiguous country name appearing in the ‘country’
                           field) divided by the country area (log scale). In Fig.&nbsp;(b), colour scale indicates
                           the proportion of originally georeferenced sequences among those that can be tracked
                           to each country (white indicates no data). All maps are in the Lambert cylindrical
                           equal area projection. Full color version available online [Colour figure can be viewed
                           at wileyonlinelibrary.com].
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
         </div>
         
         <div class="article-section__sub-content" id="jbi12786-sec-0016">
            
            <h3 class="article-section__sub-title section2"> Evaluating data availability for automated phylogeography</h3>
            
            <p>A total of 1327 clusters in 761 species and 435 genera (summing up to 86,122 accessions)
               contained at least 20 sequences from the same species and were retained as potentially
               interesting.
            </p>
            
            <p>The distribution of within‐species clusters containing at least 20 sequences according
               to the number of sequences in the cluster and the proportion of the species range
               covered are illustrated in Fig.&nbsp;<a href="#jbi12786-fig-0006">6</a> and Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#jbi12786-tbl-0001">1</a>. Very few intraspecific sets of automatically georeferenceable GenBank sequences
               contain a large number of sequences and provide a satisfactory coverage of the species
               range (Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#jbi12786-tbl-0001">1</a>). In fact, a reliable, range‐wide description of intraspecific phylogeographical
               patterns seems to be possible only for a handful of tetrapod species. Even when we
               considered a very permissive threshold (30% of the species range covered with at least
               one sequence), the spatial distribution of the potentially usable clusters is highly
               dispersed, with very few areas covered by more than a few clusters (Figs&nbsp;<a href="#jbi12786-fig-0004">4</a>c &amp; <a href="#jbi12786-fig-0005">5</a>c). Again considering a conservative threshold at 30% of the range, we can track the
               enrichment of certain categories of data along the process from georeferencing to
               assessment of potential for phylogeography. Mitochondrial DNA accounts for 70.1% of
               the retained sequences, increasing from 47.4% in the total data and 67.7% in the georeferenced
               (original&nbsp;+&nbsp;geocoded) data. Publications with a reference to geography in the title
               contribute 28.7% of the retained data (15.8% in the total data and 17.1% in the georeferenced
               set). BOLD data represent 21.1% of retained data (3.4% in the total data and 20.3%
               in the georeferenced data set).
            </p>
            
            <section class="article-section__inline-figure">
               <figure class="figure" id="jbi12786-fig-0006"><a target="_blank" href="https://wol-prod-cdn.literatumonline.com/cms/attachment/b38364f5-384f-4461-92cc-ac80f4befaf8/jbi12786-fig-0006-m.jpg"><img class="figure__image" src="jbi12786-fig-0006-m.jpg"></a><figcaption class="figure__caption">
                     <div class="figure__caption__header"><strong class="figure-title">Figure 6</strong> 
                        <div class="figure-extra"><a href="#" class="open-figure-link">Open in figure viewer</a><a href="https://onlinelibrary.wiley.com/action/downloadFigures?id=jbi12786-fig-0006&amp;doi=10.1111%2Fjbi.12786" class="ppt-figure-link">PowerPoint
                              </a></div>
                     </div>
                     <div>Classification of intraspecific sets (clusters) of putatively homologous GenBank sequences
                        according to the number of sequences and the proportion of species range covered (spatial
                        coverage). The number of clusters in each category is indicated, with colour scale
                        proportional to the logarithm of the number of clusters. Full color version available
                        online [Colour figure can be viewed at <a href="http://wileyonlinelibrary.com/" class="linkBehavior">wileyonlinelibrary.com</a>].
                     </div>
                     <div class="accordion visible-xs visible-sm visible-md"><a href="#" title="Single Accordion Controller" aria-expanded="false" aria-controls="fajbi12786-fig-0006" target="_self" class="accordion__control"><span>Caption</span><i aria-hidden="true" class="icon-keyboard_arrow_down"></i></a><div id="fajbi12786-fig-0006" class="accordion__content" style="display: none;">Classification of intraspecific sets (clusters) of putatively homologous GenBank sequences
                           according to the number of sequences and the proportion of species range covered (spatial
                           coverage). The number of clusters in each category is indicated, with colour scale
                           proportional to the logarithm of the number of clusters. Full color version available
                           online [Colour figure can be viewed at wileyonlinelibrary.com].
                        </div>
                     </div>
                  </figcaption>
               </figure>
            </section>
            
            <div class="article-table-content" id="jbi12786-tbl-0001">
               <header class="article-table-caption"><span class="table-caption__label">Table 1.
                     </span>Number of potentially informative intraspecific clusters contained in a globally surveyed
                  selection of 1,125,514 tetrapods INSDC accessions (see text for details) according
                  to the number of sequences contained and the proportion of the species range covered
                  (sequences: condition regarding the number of sequences contained in the cluster;
                  range: condition regarding the proportion of the species range covered by the sequences
                  contained in the cluster; clusters: number of clusters satisfying the conditions for
                  sequences and range; species and genera: number of species and genera represented
                  in the clusters that satisfy the conditions for sequences and range)
               </header>
               
               
               
               <div class="article-table-content-wrapper">
                  <table class="table article-section__table">
                     
                     
                     
                     
                     
                     
                     <thead>
                        
                        <tr>
                           
                           <th style="padding-left:em;" class="bottom-bordered-cell right-bordered-cell left-aligned">Sequences</th>
                           
                           <th style="padding-left:em;" class="bottom-bordered-cell center-aligned">Range</th>
                           
                           <th style="padding-left:em;" class="bottom-bordered-cell center-aligned">Clusters</th>
                           
                           <th style="padding-left:em;" class="bottom-bordered-cell center-aligned">Species</th>
                           
                           <th style="padding-left:em;" class="bottom-bordered-cell center-aligned">Genera</th>
                           
                        </tr>
                        
                     </thead>
                     
                     <tbody>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;20</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.3</td>
                           
                           <td style="padding-left:em;" class="center-aligned">611</td>
                           
                           <td style="padding-left:em;" class="center-aligned">332</td>
                           
                           <td style="padding-left:em;" class="center-aligned">221</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;50</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.3</td>
                           
                           <td style="padding-left:em;" class="center-aligned">308</td>
                           
                           <td style="padding-left:em;" class="center-aligned">165</td>
                           
                           <td style="padding-left:em;" class="center-aligned">123</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;100</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.3</td>
                           
                           <td style="padding-left:em;" class="center-aligned">155</td>
                           
                           <td style="padding-left:em;" class="center-aligned">89</td>
                           
                           <td style="padding-left:em;" class="center-aligned">66</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;20</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.5</td>
                           
                           <td style="padding-left:em;" class="center-aligned">299</td>
                           
                           <td style="padding-left:em;" class="center-aligned">140</td>
                           
                           <td style="padding-left:em;" class="center-aligned">107</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;50</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.5</td>
                           
                           <td style="padding-left:em;" class="center-aligned">202</td>
                           
                           <td style="padding-left:em;" class="center-aligned">91</td>
                           
                           <td style="padding-left:em;" class="center-aligned">75</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;100</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.5</td>
                           
                           <td style="padding-left:em;" class="center-aligned">104</td>
                           
                           <td style="padding-left:em;" class="center-aligned">52</td>
                           
                           <td style="padding-left:em;" class="center-aligned">44</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;20</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.8</td>
                           
                           <td style="padding-left:em;" class="center-aligned">65</td>
                           
                           <td style="padding-left:em;" class="center-aligned">29</td>
                           
                           <td style="padding-left:em;" class="center-aligned">25</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;50</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.8</td>
                           
                           <td style="padding-left:em;" class="center-aligned">61</td>
                           
                           <td style="padding-left:em;" class="center-aligned">27</td>
                           
                           <td style="padding-left:em;" class="center-aligned">24</td>
                           
                        </tr>
                        
                        <tr>
                           
                           <td style="padding-left:em;" class="right-bordered-cell left-aligned">≥&nbsp;100</td>
                           
                           <td style="padding-left:em;" class="center-aligned">≥&nbsp;0.8</td>
                           
                           <td style="padding-left:em;" class="center-aligned">28</td>
                           
                           <td style="padding-left:em;" class="center-aligned">17</td>
                           
                           <td style="padding-left:em;" class="center-aligned">15</td>
                           
                        </tr>
                        
                     </tbody>
                     
                  </table>
               </div>
               
               
            </div>
            
         </div>
         
      </div>
      
      <div class="article-section__content" id="jbi12786-sec-0017">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0017-title"> Discussion</h2>
         
         <p>Our analysis of GenBank (and associated INSDC databanks) accessions for over 20,000
            tetrapod species showed that, despite a quasi‐exponential growth of the absolute number
            of accessions that directly report geographical coordinates, such georeferenced accessions
            still represent a very small fraction (6.2%) of the available sequence data for terrestrial
            vertebrates. Moreover, and very importantly, despite the widespread recognition of
            the potential importance of linking genes to geography (e.g. Field, <span><a href="#jbi12786-bib-0013" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>; Marques <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0022" class="bibLink tab-link" data-tab="pane-pcw-references">2013</a></span>), the relative frequency of georeferencing across GenBank submissions has not been
            increasing in recent years. Although our INSDC search did not target any specific
            genomic region, and considered all kind of publications, including reports from physiology
            studies or gene‐expression experiments, this relative scarcity is not essentially
            due to the prevalence of submissions dealing with topics not related with biogeography.
            In fact, we found that the frequency of georeferencing is also very low across mitochondrial
            sequences (largely employed in phylogeography, and representing 47.5% of the accessions
            we surveyed) and among papers with a reference to geographyin their title (18.9% of
            the published accessions we surveyed). The general paucity of georeferenced data may
            well explain their patchy global distribution (Figs&nbsp;<a href="#jbi12786-fig-0003">3</a> &amp; <a href="#jbi12786-fig-0004">4</a>) and lack of consistency across taxonomic classes (see Appendix&nbsp;S3). Indeed, a large
            share of the available georeferenced accessions originate from a limited number of
            local initiatives (e.g. DNA barcoding of national faunal assemblages, Clare <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0007" class="bibLink tab-link" data-tab="pane-pcw-references">2007</a></span>). Differences in accessibility among regions, together with other geopolitical and
            socio‐economic factors, might influence the absolute distribution of georeferenced
            data. Indeed, scarcely populated circumpolar and dry regions have very little data,
            while data are very dense throughout the highly developed areas of North America and
            Europe, and are extremely scarce across vast economically underdeveloped areas of
            Africa and Asia (Figs&nbsp;<a href="#jbi12786-fig-0004">4</a>, <a href="#jbi12786-fig-0005">5</a> &amp; <a href="#jbi12786-fig-0006">6</a>a). On the other hand, the relative frequency of georeferencing across countries does
            not show any interpretable pattern, as countries with high and low relative georeferencing
            are spread across every continent (Fig.&nbsp;<a href="#jbi12786-fig-0005">5</a>). Interestingly, our findings about the relative frequency of INSDC data georeferencing
            and their geographical distribution in tetrapods are qualitatively very similar to
            previous reports from very different groups of organisms such as mycorrhizal fungi
            (Tedersoo <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0037" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>) and RNA viruses (Scotch <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0035" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>). This suggests that the trends we observed for tetrapods might not be characteristic
            to a special taxonomic group, but rather represent general features of INSDC databases.
         </p>
         
         <p>The reasons why so many INSDC data are submitted without explicit georeferencing cannot
            be assessed through an analysis like ours. Sensible hypotheses fall into a few general
            categories: (1) genuine lack of precise geographical information; (2) unwillingness
            to reveal sensitive data (e.g. for samples from threatened species or populations);
            (3) lack of interest and awareness about the potential importance of direct georeferencing
            of data deposited in nucleotide databases for large‐scale reanalysis of sequence data
            (in fact, publishing the relevant geographical information in appendices and supplementary
            tables is not equivalent to easy access) and/or lack of time: researchers are often
            pressed for time and tend to skip accurate annotation (see Ryberg <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0033" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>). Regarding the last points, the only viable solution lies in efforts to educate
            the growing community of biogeographers and ecologists about the need to provide readily
            accessible georeferenced data, to which we hope our study, may contribute. Moreover,
            databank policies might actively encourage best‐practices in metadata enrichment,
            for example by asking submitters to explain why some fields are left empty. Importantly,
            INSDC databanks could take action to open themselves to verifiable third‐party annotation,
            which are currently not encouraged (see Pennisi, <span><a href="#jbi12786-bib-0029" class="bibLink tab-link" data-tab="pane-pcw-references">2008</a></span>). On the other hand, we suggest that neither lack of precision or perceived sensitivity
            of the data should refrain researchers from making georeferenced data as available
            as possible. In fact, geographical coordinates may be often precisely inferred <i>a posteriori</i> even when sampling was carried out without positioning devices and, even when submitted
            at a relatively low level of precision, may still prove useful for many applications,
            provided that precision estimates are provided. The Barcode Of Life Data system (BOLD)
            proposes a good example in this respect, in containing a special field (‘coord_accuracy’)
            to this end. Unfortunately, though, very few BOLD accessions actually use it (only
            0.02% of 56,866 georeferenced tetrapod data excluding humans and sequences mined from
            GenBank). The BOLD system, while containing a proportionally low number of the GenBank
            accessions considered in this study (38,435 accessions on over one million), contributes
            significantly to the available georeferenced data (&gt;&nbsp;20%). Our results thus highlight
            the importance of this initiative towards the creation of a global archive of genetic
            diversity. However, we note that this contribution is somewhat limited by the relatively
            large proportion of unreleased sequences (28.0% of 63,480 tetrapod data in BOLD).
         </p>
         
         <p>We also showed that a relatively simple geocoding algorithm may allow to largely increase
            (<i>c</i>.&nbsp;3‐fold) the amount of nucleotide sequence data that can be placed on a map with
            an accuracy (&lt;&nbsp;10,000&nbsp;km<sup>2</sup> or 100&nbsp;km radius) that would allow acceptably precise description of phylogeographical
            patterns at a continental scale (Figs&nbsp;<a href="#jbi12786-fig-0002">2</a>, <a href="#jbi12786-fig-0004">4</a> &amp; <a href="#jbi12786-fig-0005">5</a>).
         </p>
         
         <p>Lastly, our classification of intraspecific data sets based on the number of samples
            and spatial coverage of the species range, clearly indicates that the number and geographical
            distribution of data sets potentially amenable to automated phylogeographical analyses
            (Figs&nbsp;<a href="#jbi12786-fig-0004">4</a>c, <a href="#jbi12786-fig-0005">5</a>c &amp; 6) is still insufficient for a satisfactory description of global patterns of
            genetic diversity. Although it is not possible to have an exact estimate of the amount
            of potentially informative data that is not available due to lack of georeferencing,
            our analyses allow for a more educated guess, whose result is rather sobering. As
            we managed to obtain some georeferencing for little less than one‐sixth of the considered
            data (15.1%), we can project that, if all INSDC data that we have considered in this
            study were georeferenced, the availability would be moved up by no more than a factor
            of 6. From Table&nbsp;<a class="tableLink scrollableLink" title="Link to table" href="#jbi12786-tbl-0001">1</a> we see that such an upgrade would bring the number of species with at least 100 sequences
            from at least 80% of the range to <i>c</i>.&nbsp;100. Although this would seem promising, Figs&nbsp;<a href="#jbi12786-fig-0004">4</a>(c) and <a href="#jbi12786-fig-0005">5</a>(c) suggest that, as we did not find a consistent geographical pattern in the relative
            frequency of georeferencing among countries (Fig.&nbsp;<a href="#jbi12786-fig-0006">6</a>), continental scale comparative phylogeography involving more than very few species
            would likely only be possible for North America and Europe, while, for other continents,
            the amount of accumulated data would be barely sufficient for exploratory analyses.
         </p>
         
         <p>This rather sobering picture might change quickly, though, if the next wave of biogeographical
            and phylogeographical studies will be able to seize the opportunities opened by NGS
            techniques, which produce millions of sequences at very low cost. In our survey, we
            did not consider most NGS‐generated genomic data, mostly because the availability
            of genomic data through taxonomic and geographical space is still particularly limited.
            Another key practical concern is the computational loads needed to analyse raw data
            from several different studies for comparative purposes. Storing available information
            as sets of processed data might represent a promising alternative. NGS archives (like
            the SRA) could encourage the submission of summary tables using formats like the variant
            call format (VCF; Danecek <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0008" class="bibLink tab-link" data-tab="pane-pcw-references">2011</a></span>). This standard format can be used to summarize genome‐wide diversity as SNPs, SSR,
            insertion/deletion, sorted along a reference genome or according to a <i>de novo</i> assembled set of contigs, from different sources of data (whole‐genome sequencing,
            SNP‐chip, RADseq). However, the integration of different data sets into effective
            analytical pipelines may still be challenging, because genomic data produced by different
            studies can be highly heterogeneous. We expect that novel, computationally affordable,
            approaches capable of exploiting the exponentially growing amount of genomic data
            will be developed soon, and that a standard set of genome‐level, multiple ‘barcode’
            markers might be eventually defined.
         </p>
         
         <p>Our results show that, at least for terrestrial vertebrates, geographically biased
            sampling represents the main limiting step to provide a global reach to genetic diversity
            assessments. We suggest that future research agendas may focus on (1) widespread georeferencing
            of newly produced data, possibly involving more careful quality control by agencies
            managing nucleotide databases and more direct linking between different databases
            (e.g. adding a GBIF voucher to GenBank sequences, when available); (2) concerted efforts
            towards annotation of existing databases (see, e.g. Nilsson <i>et&nbsp;al</i>., <span><a href="#jbi12786-bib-0027" class="bibLink tab-link" data-tab="pane-pcw-references">2014</a></span>) (3) use of NGS techniques to produce extensive data on multiple individuals of several
            species; (4) coordination of large‐scale collaborative efforts for biodiversity sampling,
            with a special attention to tropical biodiversity‐rich areas in general, and to Africa
            in particular.
         </p>
         
      </div>
      
      <div class="article-section__content">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0018-title"> Acknowledgements</h2>
         
         <p>This study was mainly supported by the Max Planck Society through the Pan African
            Programme (<a href="http://panafrican.eva.mpg.de/" class="linkBehavior">panafrican.eva.mpg.de</a>). The authors are grateful to Michael Hofreiter for his valuable support during the
            development of this study, to Katharine Ann Marske and an anonymous referee for their
            exceptionally constructive comments and their active contribution to improve the first
            draft of the manuscript and to Dylan Craven for revising language and writing style.
         </p>
         
         <ol></ol>
      </div>
      
      <section class="article-section article-section__supporting">
         <div class="accordion article-accordion"><a class="accordion__control" aria-controls="d5177290e309" aria-expanded="false"><span class="section__title" id="support-information-section">
                  Supporting Information
                  </span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;" id="d5177290e309" aria-hidden="false">
               <div class="support-info__table-wrapper article-table-content-wrapper">
                  
                  
                  
                  
                  <table class="support-info__table table article-section__table">
                     <thead>
                        <tr><th id="article-filename" scope="col">Filename</th>
                        <th id="article-description" scope="col">Description</th>
                     </tr></thead>
                     <tbody>
                        <tr id="xmlId">
                           <td headers="article-filename"><a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1111%2Fjbi.12786&amp;file=jbi12786-sup-0001-AppendixS1.docx">jbi12786-sup-0001-AppendixS1.docx</a>Word document, 853.2 KB
                           </td>
                           <td headers="article-description"><b>Appendix&nbsp;S1</b> Methods.
                           </td>
                        </tr>
                        <tr id="xmlId">
                           <td headers="article-filename"><a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1111%2Fjbi.12786&amp;file=jbi12786-sup-0002-AppendixS2.txt">jbi12786-sup-0002-AppendixS2.txt</a>plain text document, 1.3 MB
                           </td>
                           <td headers="article-description"><b>Appendix&nbsp;S2</b> List of searched taxa.
                           </td>
                        </tr>
                        <tr id="xmlId">
                           <td headers="article-filename"><a href="https://onlinelibrary.wiley.com/action/downloadSupplement?doi=10.1111%2Fjbi.12786&amp;file=jbi12786-sup-0003-AppendixS3.pdf">jbi12786-sup-0003-AppendixS3.pdf</a>PDF document, 198.8 KB
                           </td>
                           <td headers="article-description"><b>Appendix S3</b> Class‐wise geographical patterns of georeferencing.
                           </td>
                        </tr>
                     </tbody>
                  </table>
                  <p class="disclaimer_text">Please note: The publisher is not responsible for the content or functionality of
                     any supporting information supplied by the authors. Any queries (other than missing
                     content) should be directed to the corresponding author for the article.
                  </p>
               </div>
            </div>
         </div>
      </section>
      <section class="article-section article-section__references" id="references-section">
         <div class="accordion article-accordion"><a href="#" class="accordion__control" aria-controls="idjbi12786-bibl-0001" aria-expanded="false" data-references="/action/ajaxShowEnhancedAbstract?widgetId=5cf4c79f-0ae9-4dc5-96ce-77f62de7ada9&amp;ajax=true&amp;doi=10.1111%2Fjbi.12786&amp;pbContext=%3Bjournal%3Ajournal%3A13652699%3Barticle%3Aarticle%3Adoi%5C%3A10.1111%2Fjbi.12786%3Bpage%3Astring%3AArticle%2FChapter+View%3Bwebsite%3Awebsite%3Apericles%3Bwgroup%3Astring%3APublication+Websites%3Bissue%3Aissue%3Adoi%5C%3A10.1111%2Fjbi.2017.44.issue-2%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;onlyLog=true"><span class="section__title" id="references-section-1">References</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" style="display: none;" id="jbi12786-bibl-0001">
               <ul class="rlist separator">
                  
                  
                  <li data-bib-id="jbi12786-bib-0001">Anonymous (<span class="pubYear">2008</span>) <span class="articleTitle">A place for everything: more researchers must record the latitude and longitude of
                        their data</span>. <i>Nature</i>, <span class="vol">453</span>, 2. doi:<a target="_blank" class="accessionId" href="https://doi.org/10.1038/453002a" title="Link to external resource: 10.1038/453002a">10.1038/453002a</a>.
                     <div class="extra-links" id="e_1_2_6_2_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1038%2F453002a" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltlahtbs%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=18451818" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000255398800003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature&amp;publication_year=2008&amp;pages=Anonymous+%282008%29+A+place+for+everything%3A+more+researchers+must+record+the+latitude+and+longitude+of+their+data.+Nature%2C+453%2C+2.+doi%3A10.1038%2F453002a.&amp;journal=2&amp;issue=453&amp;title=A+place+for+everything%3A+more+researchers+must+record+the+latitude+and+longitude+of+their+data&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0002">Antonelli, A., 
Condamine, F.L., Hettling, H., Nilsson, K., Nilsson, R.H., Oxelman, B., 
Sanderson, M.J., Sauquet, H., Scharn, R., Silvestro, D., Töpel, M. &amp;
 Vos, R.A. (<span class="pubYear">2014</span>) <span class="articleTitle">SUPERSMART: ecology and evolution in the era of big data</span>. <i>PeerJ PrePrints</i>, <span class="vol">2</span>, e501v1. Available at: <a href="https://dx.doi.org/10.7287/peerj.preprints.501v1" class="linkBehavior">https://dx.doi.org/10.7287/peerj.preprints.501v1</a>. Last accessed 1 June 2016.
                     <div class="extra-links" id="e_1_2_6_3_1"><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=PeerJ+PrePrints&amp;publication_year=2014&amp;pages=Antonelli%2C+A.%2C+Condamine%2C+F.L.%2C+Hettling%2C+H.%2C+Nilsson%2C+K.%2C+Nilsson%2C+R.H.%2C+Oxelman%2C+B.%2C+Sanderson%2C+M.J.%2C+Sauquet%2C+H.%2C+Scharn%2C+R.%2C+Silvestro%2C+D.%2C+T%C3%B6pel%2C+M.+%26+Vos%2C+R.A.+%282014%29+SUPERSMART%3A+ecology+and+evolution+in+the+era+of+big+data.+PeerJ+PrePrints%2C+2%2C+e501v1.+Available+at%3A+https%3A%2F%2Fdx.doi.org%2F10.7287%2Fpeerj.preprints.501v1.+Last+accessed+1+June+2016.&amp;journal=e501v1&amp;issue=2&amp;author=A.+Antonelliauthor=F.L.+Condamineauthor=H.+Hettlingauthor=K.+Nilssonauthor=R.H.+Nilssonauthor=B.+Oxelmanauthor=M.J.+Sandersonauthor=H.+Sauquetauthor=R.+Scharnauthor=D.+Silvestroauthor=M.+T%C3%B6pelauthor=R.A.+Vos&amp;title=SUPERSMART%3A+ecology+and+evolution+in+the+era+of+big+data&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0003">Arbogast, B.S. &amp; Kenagy, G.J. (<span class="pubYear">2001</span>) <span class="articleTitle">Comparative phylogeography as an integrative approach to historical biogeography</span>. <i>Journal of Biogeography</i>, <span class="vol">28</span>, 819–825.
                     <div class="extra-links" id="e_1_2_6_4_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-2699.2001.00594.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADyaK2cXmt1KrsLw%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=8022810" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000172403100001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Journal+of+Biogeography&amp;publication_year=2001&amp;pages=Arbogast%2C+B.S.+%26+Kenagy%2C+G.J.+%282001%29+Comparative+phylogeography+as+an+integrative+approach+to+historical+biogeography.+Journal+of+Biogeography%2C+28%2C+819%E2%80%93825.&amp;journal=819-825&amp;issue=28&amp;author=B.S.+Arbogastauthor=G.J.+Kenagy&amp;title=Comparative+phylogeography+as+an+integrative+approach+to+historical+biogeography&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0004">Bermingham, E. &amp; Moritz, C. (<span class="pubYear">1998</span>) <span class="articleTitle">Comparative phylogeography: concepts and applications</span>. <i>Molecular Ecology</i>, <span class="vol">7</span>, 367–369.
                     <div class="extra-links" id="e_1_2_6_5_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1046/j.1365-294x.1998.00424.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADyaK1cXjs1Siu7o%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=9628004" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000073797200001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology&amp;publication_year=1998&amp;pages=Bermingham%2C+E.+%26+Moritz%2C+C.+%281998%29+Comparative+phylogeography%3A+concepts+and+applications.+Molecular+Ecology%2C+7%2C+367%E2%80%93369.&amp;journal=367-369&amp;issue=7&amp;author=E.+Berminghamauthor=C.+Moritz&amp;title=Comparative+phylogeography%3A+concepts+and+applications&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0005">Bybee, S.M., 
Bracken‐Grissom, H., Haynes, B.D., Hermansen, R.A., Byers, R.L., 
Clement, M.J., Udall, J.A., Wilcox, E.R. &amp; Crandall, K.A. (<span class="pubYear">2011</span>) <span class="articleTitle">Targeted amplicon sequencing (TAS): a scalable next‐gen approach to multilocus, multitaxa
                        phylogenetics</span>. <i>Genome Biology and Evolution</i>, <span class="vol">3</span>, 1312–1323.
                     <div class="extra-links" id="e_1_2_6_6_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1093%2Fgbe%2Fevr106" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC38XltF2mtbg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=22002916" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000301535100022" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Genome+Biology+and+Evolution&amp;publication_year=2011&amp;pages=Bybee%2C+S.M.%2C+Bracken%E2%80%90Grissom%2C+H.%2C+Haynes%2C+B.D.%2C+Hermansen%2C+R.A.%2C+Byers%2C+R.L.%2C+Clement%2C+M.J.%2C+Udall%2C+J.A.%2C+Wilcox%2C+E.R.+%26+Crandall%2C+K.A.+%282011%29+Targeted+amplicon+sequencing+%28TAS%29%3A+a+scalable+next%E2%80%90gen+approach+to+multilocus%2C+multitaxa+phylogenetics.+Genome+Biology+and+Evolution%2C+3%2C+1312%E2%80%931323.&amp;journal=1312-1323&amp;issue=3&amp;author=S.M.+Bybeeauthor=H.+Bracken%E2%80%90Grissomauthor=B.D.+Haynesauthor=R.A.+Hermansenauthor=R.L.+Byersauthor=M.J.+Clementauthor=J.A.+Udallauthor=E.R.+Wilcoxauthor=K.A.+Crandall&amp;title=Targeted+amplicon+sequencing+%28TAS%29%3A+a+scalable+next%E2%80%90gen+approach+to+multilocus%2C+multitaxa+phylogenetics&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0006">Carnaval, A.C., Hickerson, M.J., Haddad, C.F., Rodrigues, M.T. &amp; Moritz, C. (<span class="pubYear">2009</span>) <span class="articleTitle">Stability predicts genetic diversity in the Brazilian Atlantic forest hotspot</span>. <i>Science</i>, <span class="vol">323</span>, 785–789.
                     <div class="extra-links" id="e_1_2_6_7_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1126%2Fscience.1166955" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlersb0%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=19197066" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000263066800048" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Science&amp;publication_year=2009&amp;pages=Carnaval%2C+A.C.%2C+Hickerson%2C+M.J.%2C+Haddad%2C+C.F.%2C+Rodrigues%2C+M.T.+%26+Moritz%2C+C.+%282009%29+Stability+predicts+genetic+diversity+in+the+Brazilian+Atlantic+forest+hotspot.+Science%2C+323%2C+785%E2%80%93789.&amp;journal=785-789&amp;issue=323&amp;author=A.C.+Carnavalauthor=M.J.+Hickersonauthor=C.F.+Haddadauthor=M.T.+Rodriguesauthor=C.+Moritz&amp;title=Stability+predicts+genetic+diversity+in+the+Brazilian+Atlantic+forest+hotspot&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0007">Clare, E.L., Lim, B.K., Engstrom, M.D., Eger, J.L. &amp; Hebert, P.D. (<span class="pubYear">2007</span>) <span class="articleTitle">DNA barcoding of Neotropical bats: species identification and discovery within Guyana</span>. <i>Molecular Ecology Notes</i>, <span class="vol">7</span>, 184–190.
                     <div class="extra-links" id="e_1_2_6_8_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1471-8286.2006.01657.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkvVKrurY%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000244520600002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology+Notes&amp;publication_year=2007&amp;pages=Clare%2C+E.L.%2C+Lim%2C+B.K.%2C+Engstrom%2C+M.D.%2C+Eger%2C+J.L.+%26+Hebert%2C+P.D.+%282007%29+DNA+barcoding+of+Neotropical+bats%3A+species+identification+and+discovery+within+Guyana.+Molecular+Ecology+Notes%2C+7%2C+184%E2%80%93190.&amp;journal=184-190&amp;issue=7&amp;author=E.L.+Clareauthor=B.K.+Limauthor=M.D.+Engstromauthor=J.L.+Egerauthor=P.D.+Hebert&amp;title=DNA+barcoding+of+Neotropical+bats%3A+species+identification+and+discovery+within+Guyana&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0008">Danecek, P., 
Auton, A., Abecasis, G., Albers, C.A., Banks, E., DePristo, M.A., 
Handsaker, R.E., Lunter, G., Marth, G.T., Sherry, S.T., McVean, G. &amp;
 Durbin, R. (<span class="pubYear">2011</span>) <span class="articleTitle">The variant call format and VCFtools</span>. <i>Bioinformatics</i>, <span class="vol">27</span>, 2156–2158.
                     <div class="extra-links" id="e_1_2_6_9_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1093%2Fbioinformatics%2Fbtr330" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptFCqt7w%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=21653522" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000292778700023" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Bioinformatics&amp;publication_year=2011&amp;pages=Danecek%2C+P.%2C+Auton%2C+A.%2C+Abecasis%2C+G.%2C+Albers%2C+C.A.%2C+Banks%2C+E.%2C+DePristo%2C+M.A.%2C+Handsaker%2C+R.E.%2C+Lunter%2C+G.%2C+Marth%2C+G.T.%2C+Sherry%2C+S.T.%2C+McVean%2C+G.+%26+Durbin%2C+R.+%282011%29+The+variant+call+format+and+VCFtools.+Bioinformatics%2C+27%2C+2156%E2%80%932158.&amp;journal=2156-2158&amp;issue=27&amp;author=P.+Danecekauthor=A.+Autonauthor=G.+Abecasisauthor=C.A.+Albersauthor=E.+Banksauthor=M.A.+DePristoauthor=R.E.+Handsakerauthor=G.+Lunterauthor=G.T.+Marthauthor=S.T.+Sherryauthor=G.+McVeanauthor=R.+Durbin&amp;title=The+variant+call+format+and+VCFtools&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0009">Dawson, M.N. (<span class="pubYear">2014</span>) <span class="articleTitle">Natural experiments and meta‐analyses in comparative phylogeography</span>. <i>Journal of Biogeography</i>, <span class="vol">41</span>, 52–65.
                     <div class="extra-links" id="e_1_2_6_10_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/jbi.12190" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000328271500006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Journal+of+Biogeography&amp;publication_year=2014&amp;pages=Dawson%2C+M.N.+%282014%29+Natural+experiments+and+meta%E2%80%90analyses+in+comparative+phylogeography.+Journal+of+Biogeography%2C+41%2C+52%E2%80%9365.&amp;journal=52-65&amp;issue=41&amp;author=M.N.+Dawson&amp;title=Natural+experiments+and+meta%E2%80%90analyses+in+comparative+phylogeography&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0010">Drummond, A.J., Rambaut, A., Shapiro, B. &amp; Pybus, O.G. (<span class="pubYear">2005</span>) <span class="articleTitle">Bayesian coalescent inference of past population dynamics from molecular sequences</span>. <i>Molecular Biology and Evolution</i>, <span class="vol">22</span>, 1185–1192.
                     <div class="extra-links" id="e_1_2_6_11_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1093%2Fmolbev%2Fmsi103" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjvVKitb4%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=15703244" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000228700200004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Biology+and+Evolution&amp;publication_year=2005&amp;pages=Drummond%2C+A.J.%2C+Rambaut%2C+A.%2C+Shapiro%2C+B.+%26+Pybus%2C+O.G.+%282005%29+Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences.+Molecular+Biology+and+Evolution%2C+22%2C+1185%E2%80%931192.&amp;journal=1185-1192&amp;issue=22&amp;author=A.J.+Drummondauthor=A.+Rambautauthor=B.+Shapiroauthor=O.G.+Pybus&amp;title=Bayesian+coalescent+inference+of+past+population+dynamics+from+molecular+sequences&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0011">Edgar, R.C. (<span class="pubYear">2010</span>) <span class="articleTitle">Search and clustering orders of magnitude faster than BLAST</span>. <i>Bioinformatics</i>, <span class="vol">26</span>, 2460–2461.
                     <div class="extra-links" id="e_1_2_6_12_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1093%2Fbioinformatics%2Fbtq461" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1WhtbzM" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=20709691" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000282170000016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Bioinformatics&amp;publication_year=2010&amp;pages=Edgar%2C+R.C.+%282010%29+Search+and+clustering+orders+of+magnitude+faster+than+BLAST.+Bioinformatics%2C+26%2C+2460%E2%80%932461.&amp;journal=2460-2461&amp;issue=26&amp;author=R.C.+Edgar&amp;title=Search+and+clustering+orders+of+magnitude+faster+than+BLAST&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0012">Ekblom, R. &amp; Galindo, J. (<span class="pubYear">2011</span>) <span class="articleTitle">Applications of next generation sequencing in molecular ecology of non‐model organisms</span>. <i>Heredity</i>, <span class="vol">107</span>, 1–15.
                     <div class="extra-links" id="e_1_2_6_13_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1038%2Fhdy.2010.152" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ASTN%3A280%3ADC%252BC3Mngtleisg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=21139633" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000291635500001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Heredity&amp;publication_year=2011&amp;pages=Ekblom%2C+R.+%26+Galindo%2C+J.+%282011%29+Applications+of+next+generation+sequencing+in+molecular+ecology+of+non%E2%80%90model+organisms.+Heredity%2C+107%2C+1%E2%80%9315.&amp;journal=1-15&amp;issue=107&amp;author=R.+Ekblomauthor=J.+Galindo&amp;title=Applications+of+next+generation+sequencing+in+molecular+ecology+of+non%E2%80%90model+organisms&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0013">Field, D. (<span class="pubYear">2008</span>) <span class="articleTitle">Working together to put molecules on the map</span>. <i>Nature</i>, <span class="vol">453</span>, 978–978.
                     <div class="extra-links" id="e_1_2_6_14_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1038%2F453978b" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD1cXntlGjsbg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=18563126" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000256839900017" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature&amp;publication_year=2008&amp;pages=Field%2C+D.+%282008%29+Working+together+to+put+molecules+on+the+map.+Nature%2C+453%2C+978%E2%80%93978.&amp;journal=978-978&amp;issue=453&amp;author=D.+Field&amp;title=Working+together+to+put+molecules+on+the+map&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0014">Guralnick, R.P., Hill, A.W. &amp; Lane, M. (<span class="pubYear">2007</span>) <span class="articleTitle">Towards a collaborative, global infrastructure for biodiversity assessment</span>. <i>Ecology Letters</i>, <span class="vol">10</span>, 663–672.
                     <div class="extra-links" id="e_1_2_6_15_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1461-0248.2007.01063.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=17594421" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000247561100004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Ecology+Letters&amp;publication_year=2007&amp;pages=Guralnick%2C+R.P.%2C+Hill%2C+A.W.+%26+Lane%2C+M.+%282007%29+Towards+a+collaborative%2C+global+infrastructure+for+biodiversity+assessment.+Ecology+Letters%2C+10%2C+663%E2%80%93672.&amp;journal=663-672&amp;issue=10&amp;author=R.P.+Guralnickauthor=A.W.+Hillauthor=M.+Lane&amp;title=Towards+a+collaborative%2C+global+infrastructure+for+biodiversity+assessment&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0015">Hardisty, A. &amp; Roberts, D. (<span class="pubYear">2013</span>) <span class="articleTitle">A decadal view of biodiversity informatics: challenges and priorities</span>. <i>BMC Ecology</i>, <span class="vol">13</span>, 16.
                     <div class="extra-links" id="e_1_2_6_16_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1186%2F1472-6785-13-16" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=23587026" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000318501200001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=BMC+Ecology&amp;publication_year=2013&amp;pages=Hardisty%2C+A.+%26+Roberts%2C+D.+%282013%29+A+decadal+view+of+biodiversity+informatics%3A+challenges+and+priorities.+BMC+Ecology%2C+13%2C+16.&amp;journal=16&amp;issue=13&amp;author=A.+Hardistyauthor=D.+Roberts&amp;title=A+decadal+view+of+biodiversity+informatics%3A+challenges+and+priorities&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0016">Hewitt, G.M. (<span class="pubYear">2000</span>) <span class="articleTitle">The genetic legacy of the Quaternary ice ages</span>. <i>Nature</i>, <span class="vol">405</span>, 907–913.
                     <div class="extra-links" id="e_1_2_6_17_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1038%2F35016000" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD3cXks1Wmu78%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=10879524" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000087732700039" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature&amp;publication_year=2000&amp;pages=Hewitt%2C+G.M.+%282000%29+The+genetic+legacy+of+the+Quaternary+ice+ages.+Nature%2C+405%2C+907%E2%80%93913.&amp;journal=907-913&amp;issue=405&amp;author=G.M.+Hewitt&amp;title=The+genetic+legacy+of+the+Quaternary+ice+ages&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0017">Hewitt, G.M. (<span class="pubYear">2004</span>) <span class="articleTitle">The structure of biodiversity–insights from molecular phylogeography</span>. <i>Frontiers in Zoology</i>, <span class="vol">1</span>, 1–16.
                     <div class="extra-links" id="e_1_2_6_18_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1186%2F1742-9994-1-4" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=15679902" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Frontiers+in+Zoology&amp;publication_year=2004&amp;pages=Hewitt%2C+G.M.+%282004%29+The+structure+of+biodiversity%E2%80%93insights+from+molecular+phylogeography.+Frontiers+in+Zoology%2C+1%2C+1%E2%80%9316.&amp;journal=1-16&amp;issue=1&amp;author=G.M.+Hewitt&amp;title=The+structure+of+biodiversity%E2%80%93insights+from+molecular+phylogeography&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0018">Hickerson, M.J., 
Carstens, B.C., Cavender‐Bares, J., Crandall, K.A., Graham, C.H., 
Johnson, J.B., Rissler, L., Victoriano, P.F. &amp; Yoder, A.D. (<span class="pubYear">2010</span>) <span class="articleTitle">Phylogeography's past, present, and future: 10&nbsp;years after</span>. <i>Molecular Phylogenetics and Evolution</i>, <span class="vol">54</span>, 291–301.
                     <div class="extra-links" id="e_1_2_6_19_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1016%2Fj.ympev.2009.09.016" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ASTN%3A280%3ADC%252BD1MjpvFKntw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=19755165" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000273518500026" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Phylogenetics+and+Evolution&amp;publication_year=2010&amp;pages=Hickerson%2C+M.J.%2C+Carstens%2C+B.C.%2C+Cavender%E2%80%90Bares%2C+J.%2C+Crandall%2C+K.A.%2C+Graham%2C+C.H.%2C+Johnson%2C+J.B.%2C+Rissler%2C+L.%2C+Victoriano%2C+P.F.+%26+Yoder%2C+A.D.+%282010%29+Phylogeography%27s+past%2C+present%2C+and+future%3A+10%C2%A0years+after.+Molecular+Phylogenetics+and+Evolution%2C+54%2C+291%E2%80%93301.&amp;journal=291-301&amp;issue=54&amp;author=M.J.+Hickersonauthor=B.C.+Carstensauthor=J.+Cavender%E2%80%90Baresauthor=K.A.+Crandallauthor=C.H.+Grahamauthor=J.B.+Johnsonauthor=L.+Risslerauthor=P.F.+Victorianoauthor=A.D.+Yoder&amp;title=Phylogeography%27s+past%2C+present%2C+and+future%3A+10%C2%A0years+after&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0019">Jenkins, C.N., Pimm, S.L. &amp; Joppa, L.N. (<span class="pubYear">2013</span>) <span class="articleTitle">Global patterns of terrestrial vertebrate diversity and conservation</span>. <i>Proceedings of the National Academy of Sciences USA</i>, <span class="vol">110</span>, E2602–E2610.
                     <div class="extra-links" id="e_1_2_6_20_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1073%2Fpnas.1302251110" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Gls7vP" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=23803854" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000321827000011" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Proceedings+of+the+National+Academy+of+Sciences+USA&amp;publication_year=2013&amp;pages=Jenkins%2C+C.N.%2C+Pimm%2C+S.L.+%26+Joppa%2C+L.N.+%282013%29+Global+patterns+of+terrestrial+vertebrate+diversity+and+conservation.+Proceedings+of+the+National+Academy+of+Sciences+USA%2C+110%2C+E2602%E2%80%93E2610.&amp;journal=E2602-E2610&amp;issue=110&amp;author=C.N.+Jenkinsauthor=S.L.+Pimmauthor=L.N.+Joppa&amp;title=Global+patterns+of+terrestrial+vertebrate+diversity+and+conservation&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0020">Lorenzen, E.D., Heller, R. &amp; Siegismund, H.R. (<span class="pubYear">2012</span>) <span class="articleTitle">Comparative phylogeography of African savannah ungulates</span>. <i>Molecular Ecology</i>, <span class="vol">21</span>, 3656–3670.
                     <div class="extra-links" id="e_1_2_6_21_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2012.05650.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ASTN%3A280%3ADC%252BC38jitFOktg%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=22702960" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000306478800005" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology&amp;publication_year=2012&amp;pages=Lorenzen%2C+E.D.%2C+Heller%2C+R.+%26+Siegismund%2C+H.R.+%282012%29+Comparative+phylogeography+of+African+savannah+ungulates.+Molecular+Ecology%2C+21%2C+3656%E2%80%933670.&amp;journal=3656-3670&amp;issue=21&amp;author=E.D.+Lorenzenauthor=R.+Hellerauthor=H.R.+Siegismund&amp;title=Comparative+phylogeography+of+African+savannah+ungulates&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0021">Losos, J.B. &amp; Glor, R.E. (<span class="pubYear">2003</span>) <span class="articleTitle">Phylogenetic comparative methods and the geography of speciation</span>. <i>Trends in Ecology and Evolution</i>, <span class="vol">18</span>, 220–227.
                     <div class="extra-links" id="e_1_2_6_22_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1016%2FS0169-5347%2803%2900037-5" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=Can%27t+run+EXECUTE.+Wrong+input+line" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000183117100007" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Trends+in+Ecology+and+Evolution&amp;publication_year=2003&amp;pages=Losos%2C+J.B.+%26+Glor%2C+R.E.+%282003%29+Phylogenetic+comparative+methods+and+the+geography+of+speciation.+Trends+in+Ecology+and+Evolution%2C+18%2C+220%E2%80%93227.&amp;journal=220-227&amp;issue=18&amp;author=J.B.+Lososauthor=R.E.+Glor&amp;title=Phylogenetic+comparative+methods+and+the+geography+of+speciation&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0022">Marques, A.C., Maronna, M.M. &amp; Collins, A.G. (<span class="pubYear">2013</span>) <span class="articleTitle">Putting GenBank data on the map</span>. <i>Science</i>, <span class="vol">341</span>, 1341.
                     <div class="extra-links" id="e_1_2_6_23_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1126%2Fscience.341.6152.1341-a" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=24052287" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000324597200018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Science&amp;publication_year=2013&amp;pages=Marques%2C+A.C.%2C+Maronna%2C+M.M.+%26+Collins%2C+A.G.+%282013%29+Putting+GenBank+data+on+the+map.+Science%2C+341%2C+1341.&amp;journal=1341&amp;issue=341&amp;author=A.C.+Marquesauthor=M.M.+Maronnaauthor=A.G.+Collins&amp;title=Putting+GenBank+data+on+the+map&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0023">Marske, K.A., Rahbek, C. &amp; Nogués‐Bravo, D. (<span class="pubYear">2013</span>) <span class="articleTitle">Phylogeography: spanning the ecology‐evolution continuum</span>. <i>Ecography</i>, <span class="vol">36</span>, 1169–1181.
                     <div class="extra-links" id="e_1_2_6_24_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2013.00244.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000326327800003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Ecography&amp;publication_year=2013&amp;pages=Marske%2C+K.A.%2C+Rahbek%2C+C.+%26+Nogu%C3%A9s%E2%80%90Bravo%2C+D.+%282013%29+Phylogeography%3A+spanning+the+ecology%E2%80%90evolution+continuum.+Ecography%2C+36%2C+1169%E2%80%931181.&amp;journal=1169-1181&amp;issue=36&amp;author=K.A.+Marskeauthor=C.+Rahbekauthor=D.+Nogu%C3%A9s%E2%80%90Bravo&amp;title=Phylogeography%3A+spanning+the+ecology%E2%80%90evolution+continuum&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0024">Mesirov, J.P. (<span class="pubYear">2010</span>) <span class="articleTitle">Computer science. Accessible reproducible research</span>. <i>Science</i>, <span class="vol">327</span>, 415–416.
                     <div class="extra-links" id="e_1_2_6_25_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1126%2Fscience.1179653" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVOns7k%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=20093459" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000273783100020" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Science&amp;publication_year=2010&amp;pages=Mesirov%2C+J.P.+%282010%29+Computer+science.+Accessible+reproducible+research.+Science%2C+327%2C+415%E2%80%93416.&amp;journal=415-416&amp;issue=327&amp;author=J.P.+Mesirov&amp;title=Computer+science.+Accessible+reproducible+research&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0025">Moritz, C., 
Hoskin, C.J., MacKenzie, J.B., Phillips, B.L., Tonione, M., Silva, N., 
VanDerWal, J., Williams, S.E. &amp; Graham, C.H. (<span class="pubYear">2009</span>) <span class="articleTitle">Identification and dynamics of a cryptic suture zone in tropical rainforest</span>. <i>Proceedings of the Royal Society B: Biological Sciences</i>, <span class="vol">276</span>, 1235–1244.
                     <div class="extra-links" id="e_1_2_6_26_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1098%2Frspb.2008.1622" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ASTN%3A280%3ADC%252BD1M3jsFWmug%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=19203915" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000264351800006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Proceedings+of+the+Royal+Society+B%3A+Biological+Sciences&amp;publication_year=2009&amp;pages=Moritz%2C+C.%2C+Hoskin%2C+C.J.%2C+MacKenzie%2C+J.B.%2C+Phillips%2C+B.L.%2C+Tonione%2C+M.%2C+Silva%2C+N.%2C+VanDerWal%2C+J.%2C+Williams%2C+S.E.+%26+Graham%2C+C.H.+%282009%29+Identification+and+dynamics+of+a+cryptic+suture+zone+in+tropical+rainforest.+Proceedings+of+the+Royal+Society+B%3A+Biological+Sciences%2C+276%2C+1235%E2%80%931244.&amp;journal=1235-1244&amp;issue=276&amp;author=C.+Moritzauthor=C.J.+Hoskinauthor=J.B.+MacKenzieauthor=B.L.+Phillipsauthor=M.+Tonioneauthor=N.+Silvaauthor=J.+VanDerWalauthor=S.E.+Williamsauthor=C.H.+Graham&amp;title=Identification+and+dynamics+of+a+cryptic+suture+zone+in+tropical+rainforest&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0026">Nakamura, Y., Cochrane, G. &amp; Karsch‐Mizrachi, I. (<span class="pubYear">2013</span>) <span class="articleTitle">The international nucleotide sequence database collaboration</span>. <i>Nucleic Acids Research</i>, <span class="vol">41</span>, D21–D24.
                     <div class="extra-links" id="e_1_2_6_27_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1093%2Fnar%2Fgks1084" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvV2ktrzO" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=23180798" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000312893300003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nucleic+Acids+Research&amp;publication_year=2013&amp;pages=Nakamura%2C+Y.%2C+Cochrane%2C+G.+%26+Karsch%E2%80%90Mizrachi%2C+I.+%282013%29+The+international+nucleotide+sequence+database+collaboration.+Nucleic+Acids+Research%2C+41%2C+D21%E2%80%93D24.&amp;journal=D21-D24&amp;issue=41&amp;author=Y.+Nakamuraauthor=G.+Cochraneauthor=I.+Karsch%E2%80%90Mizrachi&amp;title=The+international+nucleotide+sequence+database+collaboration&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0027">Nilsson, R.H., Hyde, K.D., Pawłowska, J. <i>et&nbsp;al</i>. (<span class="pubYear">2014</span>) <span class="articleTitle">Improving ITS sequence data for identification of plant pathogenic fungi</span>. <i>Fungal Diversity</i>, <span class="vol">67</span>, 11–19.
                     <div class="extra-links" id="e_1_2_6_28_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1007%2Fs13225-014-0291-8" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000342427300002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Fungal+Diversity&amp;publication_year=2014&amp;pages=Nilsson%2C+R.H.%2C+Hyde%2C+K.D.%2C+Paw%C5%82owska%2C+J.+et%C2%A0al.+%282014%29+Improving+ITS+sequence+data+for+identification+of+plant+pathogenic+fungi.+Fungal+Diversity%2C+67%2C+11%E2%80%9319.&amp;journal=11-19&amp;issue=67&amp;author=R.H.+Nilssonauthor=K.D.+Hydeauthor=J.+Paw%C5%82owska&amp;title=Improving+ITS+sequence+data+for+identification+of+plant+pathogenic+fungi&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0028">Paz‐Vinas, I., Loot, G., Stevens, V.M. &amp; Blanchet, S. (<span class="pubYear">2015</span>) <span class="articleTitle">Evolutionary processes driving spatial patterns of intraspecific genetic diversity
                        in river ecosystems</span>. <i>Molecular Ecology</i>, <span class="vol">24</span>, 4586–4604.
                     <div class="extra-links" id="e_1_2_6_29_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/mec.13345" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ASTN%3A280%3ADC%252BC287jvVOluw%253D%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=26284462" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000361555900002" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology&amp;publication_year=2015&amp;pages=Paz%E2%80%90Vinas%2C+I.%2C+Loot%2C+G.%2C+Stevens%2C+V.M.+%26+Blanchet%2C+S.+%282015%29+Evolutionary+processes+driving+spatial+patterns+of+intraspecific+genetic+diversity+in+river+ecosystems.+Molecular+Ecology%2C+24%2C+4586%E2%80%934604.&amp;journal=4586-4604&amp;issue=24&amp;author=I.+Paz%E2%80%90Vinasauthor=G.+Lootauthor=V.M.+Stevensauthor=S.+Blanchet&amp;title=Evolutionary+processes+driving+spatial+patterns+of+intraspecific+genetic+diversity+in+river+ecosystems&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0029">Pennisi, E. (<span class="pubYear">2008</span>) <span class="articleTitle">Proposal to ‘wikify’ GenBank meets stiff resistance</span>. <i>Science</i>, <span class="vol">319</span>, 1598–1599.
                     <div class="extra-links" id="e_1_2_6_30_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1126%2Fscience.319.5870.1598" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFWmtrg%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=18356493" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000254172300003" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Science&amp;publication_year=2008&amp;pages=Pennisi%2C+E.+%282008%29+Proposal+to+%E2%80%98wikify%E2%80%99+GenBank+meets+stiff+resistance.+Science%2C+319%2C+1598%E2%80%931599.&amp;journal=1598-1599&amp;issue=319&amp;author=E.+Pennisi&amp;title=Proposal+to+%E2%80%98wikify%E2%80%99+GenBank+meets+stiff+resistance&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0030">Pyron, R.A. &amp; Burbrink, F.T. (<span class="pubYear">2010</span>) <span class="articleTitle">Hard and soft allopatry: physically and ecologically mediated modes of geographic
                        speciation</span>. <i>Journal of Biogeography</i>, <span class="vol">37</span>, 2005–2015.
                     <div class="extra-links" id="e_1_2_6_31_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-2699.2010.02336.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000282183900016" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Journal+of+Biogeography&amp;publication_year=2010&amp;pages=Pyron%2C+R.A.+%26+Burbrink%2C+F.T.+%282010%29+Hard+and+soft+allopatry%3A+physically+and+ecologically+mediated+modes+of+geographic+speciation.+Journal+of+Biogeography%2C+37%2C+2005%E2%80%932015.&amp;journal=2005-2015&amp;issue=37&amp;author=R.A.+Pyronauthor=F.T.+Burbrink&amp;title=Hard+and+soft+allopatry%3A+physically+and+ecologically+mediated+modes+of+geographic+speciation&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0031">Riginos, C., Douglas, K.E., Jin, Y., Shanahan, D.F. &amp; Treml, E.A. (<span class="pubYear">2011</span>) <span class="articleTitle">Effects of geography and life history traits on genetic differentiation in benthic
                        marine fishes</span>. <i>Ecography</i>, <span class="vol">34</span>, 566–575.
                     <div class="extra-links" id="e_1_2_6_32_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1600-0587.2010.06511.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=11856425" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000293031700004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Ecography&amp;publication_year=2011&amp;pages=Riginos%2C+C.%2C+Douglas%2C+K.E.%2C+Jin%2C+Y.%2C+Shanahan%2C+D.F.+%26+Treml%2C+E.A.+%282011%29+Effects+of+geography+and+life+history+traits+on+genetic+differentiation+in+benthic+marine+fishes.+Ecography%2C+34%2C+566%E2%80%93575.&amp;journal=566-575&amp;issue=34&amp;author=C.+Riginosauthor=K.E.+Douglasauthor=Y.+Jinauthor=D.F.+Shanahanauthor=E.A.+Treml&amp;title=Effects+of+geography+and+life+history+traits+on+genetic+differentiation+in+benthic+marine+fishes&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0032">Riginos, C., Buckley, Y.M., Blomberg, S.P. &amp; Treml, E.A. (<span class="pubYear">2014</span>) <span class="articleTitle">Dispersal capacity predicts both population genetic structure and species richness
                        in reef fishes</span>. <i>The American Naturalist</i>, <span class="vol">184</span>, 52–64.
                     <div class="extra-links" id="e_1_2_6_33_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1086%2F676505" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=24921600" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000337736700006" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=The+American+Naturalist&amp;publication_year=2014&amp;pages=Riginos%2C+C.%2C+Buckley%2C+Y.M.%2C+Blomberg%2C+S.P.+%26+Treml%2C+E.A.+%282014%29+Dispersal+capacity+predicts+both+population+genetic+structure+and+species+richness+in+reef+fishes.+The+American+Naturalist%2C+184%2C+52%E2%80%9364.&amp;journal=52-64&amp;issue=184&amp;author=C.+Riginosauthor=Y.M.+Buckleyauthor=S.P.+Blombergauthor=E.A.+Treml&amp;title=Dispersal+capacity+predicts+both+population+genetic+structure+and+species+richness+in+reef+fishes&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0033">Ryberg, M., Nilsson, R.H., Kristiansson, E., Töpel, M., Jacobsson, S. &amp; Larsson, E. (<span class="pubYear">2008</span>) <span class="articleTitle">Mining metadata from unidentified ITS sequences in GenBank: a case study in <i>Inocybe</i> (Basidiomycota)</span>. <i>BMC Evolutionary Biology</i>, <span class="vol">8</span>, 50.
                     <div class="extra-links" id="e_1_2_6_34_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1186%2F1471-2148-8-50" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1Ont7g%253D" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=18282272" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000254611600001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=BMC+Evolutionary+Biology&amp;publication_year=2008&amp;pages=Ryberg%2C+M.%2C+Nilsson%2C+R.H.%2C+Kristiansson%2C+E.%2C+T%C3%B6pel%2C+M.%2C+Jacobsson%2C+S.+%26+Larsson%2C+E.+%282008%29+Mining+metadata+from+unidentified+ITS+sequences+in+GenBank%3A+a+case+study+in+Inocybe+%28Basidiomycota%29.+BMC+Evolutionary+Biology%2C+8%2C+50.&amp;journal=50&amp;issue=8&amp;author=M.+Rybergauthor=R.H.+Nilssonauthor=E.+Kristianssonauthor=M.+T%C3%B6pelauthor=S.+Jacobssonauthor=E.+Larsson&amp;title=Mining+metadata+from+unidentified+ITS+sequences+in+GenBank%3A+a+case+study+in+Inocybe+%28Basidiomycota%29&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0034">Schiffels, S. &amp; Durbin, R. (<span class="pubYear">2014</span>) <span class="articleTitle">Inferring human population size and separation history from multiple genome sequences</span>. <i>Nature Genetics</i>, <span class="vol">46</span>, 919–925.
                     <div class="extra-links" id="e_1_2_6_35_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1038%2Fng.3015" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVais7jI" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=24952747" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000339704400025" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Nature+Genetics&amp;publication_year=2014&amp;pages=Schiffels%2C+S.+%26+Durbin%2C+R.+%282014%29+Inferring+human+population+size+and+separation+history+from+multiple+genome+sequences.+Nature+Genetics%2C+46%2C+919%E2%80%93925.&amp;journal=919-925&amp;issue=46&amp;author=S.+Schiffelsauthor=R.+Durbin&amp;title=Inferring+human+population+size+and+separation+history+from+multiple+genome+sequences&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0035">Scotch, M., Sarkar, I.N., Mei, C., Leaman, R., Cheung, K.H., Ortiz, P., Singraur, A. &amp; Gonzalez, G. (<span class="pubYear">2011</span>) <span class="articleTitle">Enhancing phylogeography by improving geographical information from GenBank</span>. <i>Journal of Biomedical Informatics</i>, <span class="vol">44</span>, S44–S47.
                     <div class="extra-links" id="e_1_2_6_36_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1016%2Fj.jbi.2011.06.005" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=21723960" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Journal+of+Biomedical+Informatics&amp;publication_year=2011&amp;pages=Scotch%2C+M.%2C+Sarkar%2C+I.N.%2C+Mei%2C+C.%2C+Leaman%2C+R.%2C+Cheung%2C+K.H.%2C+Ortiz%2C+P.%2C+Singraur%2C+A.+%26+Gonzalez%2C+G.+%282011%29+Enhancing+phylogeography+by+improving+geographical+information+from+GenBank.+Journal+of+Biomedical+Informatics%2C+44%2C+S44%E2%80%93S47.&amp;journal=S44-S47&amp;issue=44&amp;author=M.+Scotchauthor=I.N.+Sarkarauthor=C.+Meiauthor=R.+Leamanauthor=K.H.+Cheungauthor=P.+Ortizauthor=A.+Singraurauthor=G.+Gonzalez&amp;title=Enhancing+phylogeography+by+improving+geographical+information+from+GenBank&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0036">Shafer, A., Cullingham, C.I., Cote, S.D. &amp; Coltman, D.W. (<span class="pubYear">2010</span>) <span class="articleTitle">Of glaciers and refugia: a decade of study sheds new light on the phylogeography of
                        northwestern North America</span>. <i>Molecular Ecology</i>, <span class="vol">19</span>, 4589–4621.
                     <div class="extra-links" id="e_1_2_6_37_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2010.04828.x" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=20849561" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000283163700004" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology&amp;publication_year=2010&amp;pages=Shafer%2C+A.%2C+Cullingham%2C+C.I.%2C+Cote%2C+S.D.+%26+Coltman%2C+D.W.+%282010%29+Of+glaciers+and+refugia%3A+a+decade+of+study+sheds+new+light+on+the+phylogeography+of+northwestern+North+America.+Molecular+Ecology%2C+19%2C+4589%E2%80%934621.&amp;journal=4589-4621&amp;issue=19&amp;author=A.+Shaferauthor=C.I.+Cullinghamauthor=S.D.+Coteauthor=D.W.+Coltman&amp;title=Of+glaciers+and+refugia%3A+a+decade+of+study+sheds+new+light+on+the+phylogeography+of+northwestern+North+America&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0037">Tedersoo, L., 
Abarenkov, K., Nilsson, R.H., Schussler, A., Grelet, G.A., Kohout, P., 
Oja, J., Bonito, G.M., Veldre, V., Jairus, T., Ryberg, M., Larsson, U. 
&amp; Kõljalg, U. (<span class="pubYear">2011</span>) <span class="articleTitle">Tidying up international nucleotide sequence databases: ecological, geographical,
                        and sequence quality annotation of ITS sequences of mycorrhizal fungi</span>. <i>PLoS ONE</i>, <span class="vol">6</span>, e24904.
                     <div class="extra-links" id="e_1_2_6_38_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1371%2Fjournal.pone.0024940" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=21961048" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000295041700079" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=PLoS+ONE&amp;publication_year=2011&amp;pages=Tedersoo%2C+L.%2C+Abarenkov%2C+K.%2C+Nilsson%2C+R.H.%2C+Schussler%2C+A.%2C+Grelet%2C+G.A.%2C+Kohout%2C+P.%2C+Oja%2C+J.%2C+Bonito%2C+G.M.%2C+Veldre%2C+V.%2C+Jairus%2C+T.%2C+Ryberg%2C+M.%2C+Larsson%2C+U.+%26+K%C3%B5ljalg%2C+U.+%282011%29+Tidying+up+international+nucleotide+sequence+databases%3A+ecological%2C+geographical%2C+and+sequence+quality+annotation+of+ITS+sequences+of+mycorrhizal+fungi.+PLoS+ONE%2C+6%2C+e24904.&amp;journal=e24904&amp;issue=6&amp;author=L.+Tedersooauthor=K.+Abarenkovauthor=R.H.+Nilssonauthor=A.+Schusslerauthor=G.A.+Greletauthor=P.+Kohoutauthor=J.+Ojaauthor=G.M.+Bonitoauthor=V.+Veldreauthor=T.+Jairusauthor=M.+Rybergauthor=U.+Larssonauthor=U.+K%C3%B5ljalg&amp;title=Tidying+up+international+nucleotide+sequence+databases%3A+ecological%2C+geographical%2C+and+sequence+quality+annotation+of+ITS+sequences+of+mycorrhizal+fungi&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0038">Tedersoo, L., Ramirez, K.S., Nilsson, R.H., Kaljuvee, A., Kõljalg, U. &amp; Abarenkov, K. (<span class="pubYear">2015</span>) <span class="articleTitle">Standardizing metadata and taxonomic identification in metabarcoding studies</span>. <i>GigaScience</i>, <span class="vol">4</span>, 1–4.
                     <div class="extra-links" id="e_1_2_6_39_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1186%2Fs13742-015-0074-5" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=25838885" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000365666800001" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=GigaScience&amp;publication_year=2015&amp;pages=Tedersoo%2C+L.%2C+Ramirez%2C+K.S.%2C+Nilsson%2C+R.H.%2C+Kaljuvee%2C+A.%2C+K%C3%B5ljalg%2C+U.+%26+Abarenkov%2C+K.+%282015%29+Standardizing+metadata+and+taxonomic+identification+in+metabarcoding+studies.+GigaScience%2C+4%2C+1%E2%80%934.&amp;journal=1-4&amp;issue=4&amp;author=L.+Tedersooauthor=K.S.+Ramirezauthor=R.H.+Nilssonauthor=A.+Kaljuveeauthor=U.+K%C3%B5ljalgauthor=K.+Abarenkov&amp;title=Standardizing+metadata+and+taxonomic+identification+in+metabarcoding+studies&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0039">Weissenbacher, D., Tahsin, T., Beard, R., Figaro, M., Rivera, R., Scotch, M. &amp; Gonzalez, G. (<span class="pubYear">2015</span>) <span class="articleTitle">Knowledge‐driven geospatial location resolution for phylogeographic models of virus
                        migration</span>. <i>Bioinformatics</i>, <span class="vol">31</span>, i348–i356.
                     <div class="extra-links" id="e_1_2_6_40_1"><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=16&amp;doi=10.1111%2Fjbi.12786&amp;key=10.1093%2Fbioinformatics%2Fbtv259" onclick="newWindow(this.href);return false" target="_blank">Crossref</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1Cit7bL" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=26072502" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000356625700040" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Bioinformatics&amp;publication_year=2015&amp;pages=Weissenbacher%2C+D.%2C+Tahsin%2C+T.%2C+Beard%2C+R.%2C+Figaro%2C+M.%2C+Rivera%2C+R.%2C+Scotch%2C+M.+%26+Gonzalez%2C+G.+%282015%29+Knowledge%E2%80%90driven+geospatial+location+resolution+for+phylogeographic+models+of+virus+migration.+Bioinformatics%2C+31%2C+i348%E2%80%93i356.&amp;journal=i348-i356&amp;issue=31&amp;author=D.+Weissenbacherauthor=T.+Tahsinauthor=R.+Beardauthor=M.+Figaroauthor=R.+Riveraauthor=M.+Scotchauthor=G.+Gonzalez&amp;title=Knowledge%E2%80%90driven+geospatial+location+resolution+for+phylogeographic+models+of+virus+migration&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
                  <li data-bib-id="jbi12786-bib-0040">Wielstra, B., Duijm, E., Lagler, P., Lammers, Y., Meilink, W.R.M., Ziermann, J.M. &amp; Arntzen, J.W. (<span class="pubYear">2014</span>) <span class="articleTitle">Parallel tagged amplicon sequencing of transcriptome‐based genetic markers for Triturus
                        newts with the Ion Torrent next‐generation sequencing platform</span>. <i>Molecular Ecology Resources</i>, <span class="vol">14</span>, 1080–1089.
                     <div class="extra-links" id="e_1_2_6_41_1"><span>
                           <a href="https://onlinelibrary.wiley.com/doi/10.1111/1755-0998.12242" onclick="newWindow(this.href);return false" target="_blank">Wiley Online Library</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=32&amp;doi=10.1111%2Fjbi.12786&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlKlu7zE" onclick="newWindow(this.href);return false" target="_blank">CAS</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=8&amp;doi=10.1111%2Fjbi.12786&amp;key=24571307" onclick="newWindow(this.href);return false" target="_blank">PubMed</a></span><span>
                           <a href="https://onlinelibrary.wiley.com/servlet/linkout?suffix=null&amp;dbid=128&amp;doi=10.1111%2Fjbi.12786&amp;key=000340425100018" onclick="newWindow(this.href);return false" target="_blank">Web of Science®</a></span><span><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&amp;volume=Molecular+Ecology+Resources&amp;publication_year=2014&amp;pages=Wielstra%2C+B.%2C+Duijm%2C+E.%2C+Lagler%2C+P.%2C+Lammers%2C+Y.%2C+Meilink%2C+W.R.M.%2C+Ziermann%2C+J.M.+%26+Arntzen%2C+J.W.+%282014%29+Parallel+tagged+amplicon+sequencing+of+transcriptome%E2%80%90based+genetic+markers+for+Triturus+newts+with+the+Ion+Torrent+next%E2%80%90generation+sequencing+platform.+Molecular+Ecology+Resources%2C+14%2C+1080%E2%80%931089.&amp;journal=1080-1089&amp;issue=14&amp;author=B.+Wielstraauthor=E.+Duijmauthor=P.+Laglerauthor=Y.+Lammersauthor=W.R.M.+Meilinkauthor=J.M.+Ziermannauthor=J.W.+Arntzen&amp;title=Parallel+tagged+amplicon+sequencing+of+transcriptome%E2%80%90based+genetic+markers+for+Triturus+newts+with+the+Ion+Torrent+next%E2%80%90generation+sequencing+platform&amp;" target="_blank">Google Scholar</a></span>
                     </div>
                  </li>
                  
               </ul>
            </div>
         </div>
      </section>
      
      <div class="article-section__content" id="jbi12786-sec-0019">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0019-title"> Data accessibility</h2>
         
         <p>R and Python scripts developed for and used in this study are available at <a href="https://github.com/paolo-gratton/Gratton_et_al_JBiogeogr_2016" class="linkBehavior">https://github.com/paolo-gratton/Gratton_et_al_JBiogeogr_2016</a>.
         </p>
         
      </div>
      
      <div class="article-section__content" id="jbi12786-sec-0020">
         
         <h2 class="article-section__title section__title section1" id="jbi12786-sec-0020-title"> Biosketches</h2>
         
         <p><b>Paolo Gratton</b> is a population geneticist and biogeographer, and a current PostDoc fellow at the
            Max Planck Institute for Evolutionary Anthropology, Leipzig. His research interests
            focus on the mechanisms through which biodiversity (at the level of genes, populations,
            species, and communities) is generated and maintained by spatial and temporal environmental
            variation.
         </p>
         
         <p><b>Paolo Gratton</b>,<b> Hjalmar Kühl</b> and <b>Gaëlle Bocksberger</b> are currently developing a research project centred on using published genetic data
            to evaluate evidences for community‐level refugia in Sub‐Saharan Africa.
         </p>
         
         <p>Author contributions: P.G., G.B., H.K. and M.W. conceived the original idea for this
            study, which was further developed with the contribution of S.M. and E.T. S.M., P.G.
            and G.B. developed and performed the data analysis. P.G. led the writing, which was
            performed by all authors.
         </p>
         
      </div>
      
   </section>
   <section id="citedBy" class="article-section article-section__citedBy">
      <div class="accordion article-accordion"><a href="#" class="accordion__control" aria-controls="idss15" aria-expanded="false"><span class="section__title" id="citedby-section">Citing Literature</span><i aria-hidden="true" class="icon-section_arrow_d"></i></a><div class="accordion__content" id="idss15" style="display: none;"><h3 class="border-bottom">Number of times cited: 9</h3><ul class="rlist separator">
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Cynthia
            S. Parr and </span><span class="hlFld-ContribAuthor">Anne E. Thessen</span></span>,
            <span>Biodiversity Informatics</span>, <span class="seriesTitle"><span>Ecological
            Informatics</span></span><span class="seperator">, </span><span class="doi">10.1007/978-3-319-59928-1_17</span><span class="seperator">, </span><span class="page-range"> (375-399)</span><span class="seperator">,
            </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1007/978-3-319-59928-1_17">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Tasnia
            Tahsin, </span><span class="hlFld-ContribAuthor">Davy Weissenbacher, </span><span class="hlFld-ContribAuthor">Karen O’Connor, </span><span class="hlFld-ContribAuthor">Arjun
            Magge, </span><span class="hlFld-ContribAuthor">Matthew Scotch, </span><span class="hlFld-ContribAuthor">Graciela
            Gonzalez-Hernandez and </span><span class="hlFld-ContribAuthor">Jonathan Wren</span></span>,
            <span>GeoBoost: accelerating research involving the geospatial metadata of virus GenBank
            records</span><span class="seperator">, </span><span class="seriesTitle"><i>Bioinformatics</i></span><span class="seperator">, </span><span class="doi">10.1093/bioinformatics/btx799</span><span class="seperator">, </span><span class="volume"><b>34</b></span><span class="issue">,
            9</span><span class="seperator">, </span><span class="page-range"> (1606-1608)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/bioinformatics/btx799">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Wolf
            L. Eiserhardt, </span><span class="hlFld-ContribAuthor">Alexandre Antonelli, </span><span class="hlFld-ContribAuthor">Dominic J. Bennett, </span><span class="hlFld-ContribAuthor">Laura
            R. Botigué, </span><span class="hlFld-ContribAuthor">J. Gordon Burleigh, </span><span class="hlFld-ContribAuthor">Steven Dodsworth, </span><span class="hlFld-ContribAuthor">Brian
            J. Enquist, </span><span class="hlFld-ContribAuthor">Félix Forest, </span><span class="hlFld-ContribAuthor">Jan
            T. Kim, </span><span class="hlFld-ContribAuthor">Alexey M. Kozlov, </span><span class="hlFld-ContribAuthor">Ilia
            J. Leitch, </span><span class="hlFld-ContribAuthor">Brian S. Maitner, </span><span class="hlFld-ContribAuthor">Siavash Mirarab, </span><span class="hlFld-ContribAuthor">William
            H. Piel, </span><span class="hlFld-ContribAuthor">Oscar A. Pérez‐Escobar, </span><span class="hlFld-ContribAuthor">Lisa Pokorny, </span><span class="hlFld-ContribAuthor">Carsten
            Rahbek, </span><span class="hlFld-ContribAuthor">Brody Sandel, </span><span class="hlFld-ContribAuthor">Stephen
            A. Smith, </span><span class="hlFld-ContribAuthor">Alexandros Stamatakis, </span><span class="hlFld-ContribAuthor">Rutger A. Vos, </span><span class="hlFld-ContribAuthor">Tandy
            Warnow and </span><span class="hlFld-ContribAuthor">William J. Baker</span></span>,
            <span>A roadmap for global synthesis of the plant tree of life</span><span class="seperator">,
            </span><span class="seriesTitle"><i>American Journal of Botany</i></span><span class="seperator">,
            </span><span class="volume"><b></b></span><span class="issue">, </span><span class="seperator">,
            </span><span class="page-range"> (614-622)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1002/ajb2.1041">Wiley Online Library</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">J
            Ebersbach, </span><span class="hlFld-ContribAuthor">A N Muellner-Riehl, </span><span class="hlFld-ContribAuthor">A Favre, </span><span class="hlFld-ContribAuthor">J Paule,
            </span><span class="hlFld-ContribAuthor">G Winterfeld and </span><span class="hlFld-ContribAuthor">J
            Schnitzler</span></span>, <span>Driving forces behind evolutionary radiations: Saxifraga
            section Ciliatae (Saxifragaceae) in the region of the Qinghai–Tibet Plateau</span><span class="seperator">, </span><span class="seriesTitle"><i>Botanical Journal of the Linnean
            Society</i></span><span class="seperator">, </span><span class="doi">10.1093/botlinnean/box100</span><span class="seperator">, </span><span class="volume"><b>186</b></span><span class="issue">,
            3</span><span class="seperator">, </span><span class="page-range"> (304-320)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/botlinnean/box100">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Bryan
            C. Carstens, </span><span class="hlFld-ContribAuthor">Ariadna E. Morales, </span><span class="hlFld-ContribAuthor">Kathryn Field and </span><span class="hlFld-ContribAuthor">Tara
            A. Pelletier</span></span>, <span>A global analysis of bats using automated comparative
            phylogeography uncovers a surprising impact of Pleistocene glaciation</span><span class="seperator">, </span><span class="seriesTitle"><i>Journal of Biogeography</i></span><span class="seperator">, </span><span class="volume"><b>45</b></span><span class="issue">,
            8</span><span class="seperator">, </span><span class="page-range"> (1795-1805)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2018)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/jbi.13382">Wiley Online Library</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Paolo
            Gratton, </span><span class="hlFld-ContribAuthor">Silvio Marta, </span><span class="hlFld-ContribAuthor">Gaëlle
            Bocksberger, </span><span class="hlFld-ContribAuthor">Marten Winter, </span><span class="hlFld-ContribAuthor">Petr Keil, </span><span class="hlFld-ContribAuthor">Emiliano
            Trucchi and </span><span class="hlFld-ContribAuthor">Hjalmar Kühl</span></span>, <span>Which
            Latitudinal Gradients for Genetic Diversity?</span><span class="seperator">, </span><span class="seriesTitle"><i>Trends in Ecology &amp; Evolution</i></span><span class="seperator">,
            </span><span class="doi">10.1016/j.tree.2017.07.007</span><span class="seperator">,
            </span><span class="volume"><b>32</b></span><span class="issue">, 10</span><span class="seperator">,
            </span><span class="page-range"> (724-726)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1016/j.tree.2017.07.007">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Thanushi
            Eagalle, </span><span class="hlFld-ContribAuthor">M. Alex Smith and </span><span class="hlFld-ContribAuthor">Peter
            G. Kevan</span></span>, <span>Diversity of parasitoid and parasitic wasps across a
            latitudinal gradient: Using public DNA records to work within a taxonomic impediment</span><span class="seperator">, </span><span class="seriesTitle"><i>FACETS</i></span><span class="seperator">,
            </span><span class="doi">10.1139/facets-2016-0061</span><span class="seperator">,
            </span><span class="volume"><b>2</b></span><span class="issue">, 2</span><span class="seperator">,
            </span><span class="page-range"> (937-954)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1139/facets-2016-0061">Crossref</a></span></div></li>
            
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">Tasnia
            Tahsin, </span><span class="hlFld-ContribAuthor">Davy Weissenbacher, </span><span class="hlFld-ContribAuthor">Demetrius Jones-Shargani, </span><span class="hlFld-ContribAuthor">Daniel
            Magee, </span><span class="hlFld-ContribAuthor">Matteo Vaiente, </span><span class="hlFld-ContribAuthor">Graciela
            Gonzalez and </span><span class="hlFld-ContribAuthor">Matthew Scotch</span></span>,
            <span>Named entity linking of geospatial and host metadata in GenBank for advancing
            biomedical research</span><span class="seperator">, </span><span class="seriesTitle"><i>Database</i></span><span class="seperator">, </span><span class="doi">10.1093/database/bax093</span><span class="seperator">,
            </span><span class="volume"><b>2017</b></span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://doi.org/10.1093/database/bax093">Crossref</a></span></div></li>
            <li class="citedByEntry"><span class="entryAuthor"><span class="hlFld-ContribAuthor">H.
            Peter Linder</span></span>, <span>Phylogeography</span><span class="seperator">, </span><span class="seriesTitle"><i>Journal of Biogeography</i></span><span class="seperator">,
            </span><span class="volume"><b>44</b></span><span class="issue">, 2</span><span class="seperator">,
            </span><span class="page-range"> (243-244)</span><span class="seperator">, </span><span class="pub-date"><span class="pub-year">(2017)</span></span>.<div class="extra-links"><span><a href="https://onlinelibrary.wiley.com/doi/10.1111/jbi.12958">Wiley Online Library</a></span></div></li></ul>
         </div>
      </div>
   </section>
   
</article></div></div><div class="col-lg-4 col-md-4 col-sm-4 hidden-xs hidden-sm hidden-md sticko__parent article-row-right"><div class="publication-cover-info clearfix">



        
        <div class="cover-image-wrapper cover-image__image hasDetails"><a href="https://onlinelibrary.wiley.com/toc/13652699/2017/44/2"><img src="jbi.jpg" alt="Publication cover image"></a></div><div class="extra-info-wrapper cover-image__details"><p class="volume-issue"><a href="https://onlinelibrary.wiley.com/toc/13652699/2017/44/2"><span>Volume</span><span class="val">44</span>, <span>Issue</span><span class="val">2</span></a></p><p>February 2017</p><p class="page-range"><span>Pages</span><span> 475-486</span></p>



        
        
</div>
</div>



        
        


        <div class="pb-ad">
            		<div id="advert-rail-1" class="advert advert-rail">
			<script>

				googletag.cmd.push 		( function () {
					googletag.display 	( 'advert-rail-1' );
				});

			</script>

			<!-- No JavaScript support is a call to an image -->
			<noscript>
				<a href="/action/clickThrough?id=1011&url=https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786&pubId=63162737&placeholderId=1014&productId=101018" target="_blank">
					<img src="https://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
				</a>
			</noscript>
		<div id="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_1__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_1" title="3rd party ad content" name="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_1" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>
        </div>
    

<div class="tab tab--slide tab--flex sticko__md tabs--xs dynamic-sticko js--sticko" style="width: 329.983px; top: 81px; position: static; height: 0px;"><ul data-mobile-toggle="slide" class="rlist tab__nav w-slide--list cloned"><li role="presentation" class=""><a data-slide-clone="self" href="#pane-pcw-figures" aria-controls="#pane-pcw-figures" role="tab" data-toggle="tab" id="pane-pcw-figurescon" data-slide-target="#pane-pcw-figures" class="article-row-right__tabLinks w-slide__btn figures-tab empty" aria-expanded="false"><i aria-hidden="true" class="icon-multi-figure"></i><span>Figures</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-references" aria-controls="#pane-pcw-references" role="tab" data-toggle="tab" id="pane-pcw-referencescon" data-slide-target="#pane-pcw-references" class="article-row-right__tabLinks w-slide__btn references-tab empty"><i aria-hidden="true" class="icon-chain"></i><span>References</span></a></li><li role="presentation"><a data-slide-clone="self" href="#pane-pcw-related" aria-controls="#pane-pcw-related" role="tab" data-toggle="tab" id="pane-pcw-relatedcon" data-slide-target="#pane-pcw-related" class="article-row-right__tabLinks w-slide__btn related-tab empty"><i aria-hidden="true" class="icon-related-circles-extend-button"></i><span>Related</span></a></li><li role="presentation" class="active"><a data-slide-clone="self" href="#pane-pcw-details" aria-controls="#pane-pcw-details" role="tab" data-toggle="tab" id="pane-pcw-detailscon" data-slide-target="#pane-pcw-details" class="article-row-right__tabLinks w-slide__btn details-tab" aria-expanded="true"><i aria-hidden="true" class="icon-Icon_Information"></i><span>Information</span></a></li></ul><ul role="tablist" class="rlist tab__content sticko__child" style="height: 0px; overflow-y: auto;"><li id="pane-pcw-figures" aria-labelledby="pane-pcw-figurescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-references" aria-labelledby="pane-pcw-referencescon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-related" aria-labelledby="pane-pcw-relatedcon" role="tab" class="tab__pane article-row-right__panes empty"></li><li id="pane-pcw-details" aria-labelledby="pane-pcw-detailscon" role="tab" class="tab__pane article-row-right__panes active"><div class="contentTab"><!-- XSLT: InfoTab.xsl, Revision: 210794389, Interface: urn:atypon.com:wiley:interface:atypon-wiley-prime, was cached: Tue Aug 07 02:43:38 PDT 2018-->
<section class="info-tab-wrapper">
   <section class="metrics">
      <p class="info-header">Metrics</p>
      <div class="section__body">
         <div>
            <label>Downloads: </label>114
         </div>
         <div class="cited-by-count">
            <label>Citations: </label>9
         </div>
         <div class="pb-dropzone" data-pb-dropzone="publicaitonContent-info-tab-metrics">
            



        
        <script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script>
<div data-badge-popover="left" data-badge-type="1" data-doi="10.1111/jbi.12786" data-hide-no-mentions="false" class="altmetric-embed" data-uuid="3515e7dc-5b72-c6c4-888e-1f618ba8709b"><a target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700" rel="popover" data-content="&lt;div&gt;    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=blogs'&gt;
          Blogged by &lt;b&gt;1&lt;/b&gt;
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=twitter'&gt;
          Tweeted by &lt;b&gt;26&lt;/b&gt;
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=facebook'&gt;
          On &lt;b&gt;1&lt;/b&gt; Facebook pages
      &lt;/a&gt;
    &lt;/div&gt;
    &lt;div style='padding-left: 10px; line-height:18px; border-left: 16px solid #E065BB;'&gt;
      &lt;a class='link-to-altmetric-details-tab' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=google'&gt;
          Mentioned in &lt;b&gt;1&lt;/b&gt; Google+ posts
      &lt;/a&gt;
    &lt;/div&gt;

    &lt;div class='altmetric-embed readers' style='margin-top: 10px;'&gt;
          &lt;div class='altmetric-embed tip_mendeley'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;'&gt;
              &lt;b&gt;70&lt;/b&gt; readers on Mendeley            
          &lt;/div&gt;
          &lt;div class='altmetric-embed tip_citeulike'
               style='padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;'&gt;
              &lt;b&gt;1&lt;/b&gt; readers on CiteULike
          &lt;/div&gt;
    &lt;/div&gt;
    &lt;div style='margin-top: 10px; text-align: center;'&gt;
        &lt;a class='altmetric_details' target='_self' href='https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700'&gt;
            See more details
        &lt;/a&gt;
        |
        &lt;a href='javascript:void(0)' class='close-popover'&gt;
            Close this
        &lt;/a&gt;
    &lt;/div&gt;

    &lt;a class='altmetric_embed close-popover'
       style='display: block; position: absolute; top: 10px; right: 15px; font-size: 1.2em; font-weight: bold; text-decoration: none; color: black; padding-bottom: 2em; padding-left: 2em;'
       href='javascript:void(0)'&gt;
        ×
    &lt;/a&gt;
&lt;/div&gt;" style="display:inline-block;" data-badge-popover="left">
        <img alt="Article has an altmetric score of 24" src="https://d1uo4w7k31k5mn.cloudfront.net/v1/24.png" style="border:0; margin:0; max-width: none;" width="110px" height="20px">
</a></div> 


         </div>
      </div>
   </section>
   <section class="details">
      <p class="info-header">Details</p>
      <p class="copyright">© 2016 John Wiley &amp; Sons Ltd</p>
      <p></p>
      <ul class="cross-mark rlist"></ul>
   </section>
   <section class="keywords">
      <p class="info-header">Keywords</p>
      <ul class="rlist rlist--inline">
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=biodiversity">biodiversity</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=comparative%20phylogeography">comparative phylogeography</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=DNA%20barcoding">DNA barcoding</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=GenBank">GenBank</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=geodata">geodata</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=georeferencing">georeferencing</a>
         </li>
         <li>
            <a class="badge-type" href="https://onlinelibrary.wiley.com/action/doSearch?field1=Keyword&amp;text1=metadata">metadata</a>
         </li>
      </ul>
   </section>
   <section class="funding-information">
      <p class="info-header">Funding Information</p>
      <ul class="unordered-list">
         <li>Max Planck Society through the Pan African Programme</li>
      </ul>
   </section>
   <section class="publication-history">
      <p class="info-header">Publication History</p>
      <ul class="rlist">
         <li>
            <label>Issue Online: </label>25 January 2017
         </li>
         <li>
            <label>Version of Record online: </label>04 July 2016
         </li>
      </ul>
   </section>
</section>
<!-- END OF XSLT -->
</div><div class="emptyTab">



        
        


        <div class="pb-ad">
            <aside>
	<div id="advert-rail-2" class="advert advert-rail">
		<script>

			googletag.cmd.push 		( function () {
				googletag.display 	( 'advert-rail-2' );
			});

		</script>

		<!-- No JavaScript support is a call to an image -->
		<noscript>
			<a href="/action/clickThrough?id=1012&url=http%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fjump%3Fiu%3D%24googlePublisherCategory%26amp%3Bsz%3D160x600%7C160x320%7C160x160%7C120x600%26amp%3Btile%3D2%26amp%3Bc%3D%24random&loc=%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786&pubId=63162737&placeholderId=1013&productId=101018" target="_blank">
				<img src="http://pubads.g.doubleclick.net/gampad/ad?iu=$googlePublisherCategory&amp;sz=160x600|160x320|160x160|120x600&amp;tile=2&amp;c=$random">
			</a>
		</noscript>
	<div id="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_2__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_2" title="3rd party ad content" name="google_ads_iframe_/2215/wly.geography_000066/j-biogeog_jbi_2" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>
</aside>
        </div>
    

</div></li><li class="tab__spinner" style="display: none;"><img src="spinner.gif" id="spinner" style="width: 100%" alt=""></li></ul></div></div></div></article><script>var articleRef = document.querySelector('.article__body:not(.show-references) .article__references');
if (articleRef) { articleRef.style.display = "none"; }

</script><div id="figure-viewer" data-ux3-wrapper="figure-viewer" data-ux3-transformed-by="figureInit" data-ux3-role="parent" role="dialog" class="figure-viewer"><div class="figure-viewer__reg__top clearfix"><div class="figure-viewer__top__right"><a title="Close Figure Viewer" href="#" data-ux3-role="controller" class="figure-viewer__ctrl__close"><span class="icon-close_thin"></span></a></div><div class="figure-viewer__top__left"><a title="Browse All Figures" href="#" class="figure-viewer__ctrl__browse"><span class="icon-allfigures"></span></a><a title="Return to Figure" href="#" class="figure-viewer__ctrl__return is-hidden"><span class="icon-arrow-left"></span></a><button title="zoom out" class="figure-viewer__label__zoom icon-zoom-out zoom-out"></button><span tabindex="1" class="zoomSlider js__zoom-slider ui-slider"><input class="zoom-range" value="50" type="range"></span><button title="zoom in" class="figure-viewer__label__zoom icon-zoom-in zoom-in"></button></div></div><div class="figure-viewer__reg__center" style="height: 100%;"><div class="figure-viewer__cent__left"><a title="Previous Figure" href="#" class="figure-viewer__ctrl__prev"><span class="icon-arrow_l"></span></a><a title="Next Figure" href="#" class="figure-viewer__ctrl__next"><span class="icon-arrow_r"></span></a><div class="figure-viewer__hold__fig"><figure class="holder"></figure></div><div class="figure-viewer__hold__list clearfix container"></div></div><div class="figure-viewer__cent__right"><div class="figure-viewer__title"><a title="Open/Close Caption" href="#" class="figure-viewer__ctrl__caption"><span class="icon-doublearrow"></span><span class="figure-viewer__caption__label">Caption</span></a><span class="figure-viewer__title__text"></span></div><div class="figure-viewer__hold__figcap"></div></div></div></div>
</div>
</div></div></div>

        </section>
    
    </div>
</div><!--end pagefulltext-->

        </div>
    










        <div data-widget-def="pageFooter" data-widget-id="053994f3-7d80-4bb1-97a4-c67fbc4768cd" class="pageFooter">
        



        
        <footer class="page-footer">
    <div data-pb-dropzone="main">
    



        
        <div class="main-footer">









        <div data-widget-def="ux3-layout-widget" data-widget-id="2753bec1-a79d-4403-827b-6c788ddd6db3" class="footer-sections">
        



        
            <h3 class="hidden accessibility-enabled">
                Additional links
            </h3>
        
        <div class="container"><div class="row"><div class=" col-md-3">



        
        <div class="collapsible-menu">
  <h3 data-bind="toggle" class="collapsible-menu-header">About Wiley Online Library<span class="icon icon-chevron-down"></span></h3>
  <ul class="rlist collapsible-menu-body">
    <li><a href="https://www.wiley.com/privacy">Privacy Policy</a></li>
    <li><a href="https://onlinelibrary.wiley.com/terms-and-conditions">Terms of Use</a></li>
    <li><a href="https://onlinelibrary.wiley.com/cookies">Cookies</a></li>
    <li><a href="https://onlinelibrary.wiley.com/accessibility">Accessibility</a></li>
  </ul>
</div>
</div><div class=" col-md-3">



        
         <div class="collapsible-menu">
    <h3 data-bind="toggle" class="collapsible-menu-header">Help &amp; Support<span class="icon icon-chevron-down"></span></h3>
    <ul class="rlist collapsible-menu-body">
      <li><a href="https://hub.wiley.com/community/support/onlinelibrary">Contact Us</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
         <div class="collapsible-menu">
    <h3 data-bind="toggle" class="collapsible-menu-header">Opportunities<span class="icon icon-chevron-down"></span></h3>
    <ul class="rlist collapsible-menu-body">
      <li><a href="https://onlinelibrary.wiley.com/agents">Subscription Agents</a></li>
      <li><a href="https://onlinelibrary.wiley.com/advertisers">Advertisers &amp; Corporate Partners</a></li>
    </ul>
  </div>
</div><div class=" col-md-3">



        
        <div class="collapsible-menu">
    <h3 data-bind="toggle" class="collapsible-menu-header">Connect with Wiley<span class="icon icon-chevron-down"></span></h3>
    <ul class="rlist collapsible-menu-body">
      <li><a href="https://hub.wiley.com/community/exchanges/">The Wiley Network</a></li>
      <li><a href="http://newsroom.wiley.com/">Wiley Press Room</a></li>
    </ul>
  </div>
</div></div></div>

        </div>
    










        <div data-widget-def="ux3-layout-widget" data-widget-id="86584e25-e4db-4118-9550-9b4f5341a4ac" class="copyright-container">
        



        
        <div class="container"><div class="row"><div class=" col-md-12">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="bfbf6c5d-a650-4662-828f-5275fada9120" class="copyright">
        



        
        <div class="copyright pull-left">Copyright © 1999-2018 <a href="http://www.wiley.com/">John Wiley &amp; Sons, Inc</a>. All rights reserved</div>

        </div>
    










        <div data-widget-def="ux3-general-image" data-widget-id="063fa3e6-c0a3-4672-a2e6-62aa1b09ed11" class="footer-logo pull-right">
        



        
        <a href="http://www.wiley.com/" title=""><img alt="" src="footer-logo-wiley-1510029248417.png"></a>

        </div>
    
</div></div></div>

        </div>
    
</div>
    </div>
</footer>

        </div>
    




        
        














<div class="popup login-popup hidden">



    <div class="content">
        <a href="#" class="close"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
        </div>

        <div class="modal__body">
            <div class="ssoLogin">
                
            </div>

            <h2>Log in to Wiley Online Library</h2>

            



























<div class="login-form">
    

    

    
    <form action="/action/doLogin?societyURLCode=" class="bordered" method="post"><input name="id" value="67065c09-4a88-49cd-934c-ac707951d35c" type="hidden">
        





<input name="redirectUri" value="/doi/full/10.1111/jbi.12786" type="hidden">
<input name="loginUri" value="/doi/full/10.1111/jbi.12786" type="hidden">
        
            <input name="popup" value="true" type="hidden">
        
        
        
                


















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="login">Email or Customer ID</label>
                
        </div>
        <input id="login" class="login" name="login" size="15" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="text">
        
                    
                        <div class="actions">
                            
                        </div>
                    
                
    </div>


                
                
















    
    <div class="input-group">
        <div class="label">
            <label for="password">Password</label>
        </div>
        <input id="password" class="password" name="password" value="" autocomplete="off" placeholder="Enter your password" autocorrect="off" spellcheck="false" autocapitalize="off" type="password">
        <span class="password-eye-icon icon-eye hidden"></span>
        
                    
                        
                            <div class="actions">
                                <a href="https://onlinelibrary.wiley.com/action/requestResetPassword" class="link show-request-reset-password">Forgot password?</a>
                            </div>
                        
                    
                
    </div>

            
        
        <div class="loginExtraBeans-dropZone" data-pb-dropzone="loginExtraBeans">
            
        </div>

        
                <div class="align-end">
                    
                    
                            <div class="submit" disabled="disabled">
                                <input class="button btn submit primary no-margin-bottom" name="submit" value="Log In" disabled="disabled" type="submit">
                            </div>
                        
                </div></form>
    <div class="societySpecificInstructions">
        
    </div>
</div>

            <div class="login-container clearfix form-btn">
                <a href="https://onlinelibrary.wiley.com/action/registration" class="linkBtn float-left">
                    
                    NEW USER &gt;
                </a>

                
                    <a href="https://onlinelibrary.wiley.com/action/ssostart?redirectUri=%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786" class="linkBtn float-right instituation-login">
                        
                        INSTITUTIONAL LOGIN &gt;
                    </a>
                
            </div>
        </div>
    </div>
</div>




        
        






<div class="popup change-password-drawer hidden">
    <div class="content">
        <a href="#" class="close cancel"><i class="icon-close_thin"></i></a>
        <div class="modal__header">


            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>
            <h2>Change Password</h2>
        </div>
        <div class="modal__body">

            <div class="form">

                <form action="/action/changePassword" method="post"><div class="message error"></div>
                    <input name="submit" value="submit" type="hidden">
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-old">Old Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-old" class="old" name="old" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                    </div>
                    <div class="input-group">
                        <div class="label">
                            <label for="a589574e-bb98-4c6e-8fed-67365ff05357-new">New Password</label>
                        </div>
                        <input id="a589574e-bb98-4c6e-8fed-67365ff05357-new" class="pass-hint new" name="new" value="" autocomplete="off" type="password">
                        <span class="password-eye-icon icon-eye hidden"></span>
                        
                            







    


    


    


<div class="password-strength-indicator" data-min="10" data-max="32" data-strength="3">
    <span class="text too-short">Too Short</span>
    <span class="text weak">Weak</span>
    <span class="text medium">Medium</span>
    <span class="text strong">Strong</span>
    <span class="text very-strong">Very Strong</span>
    <span class="text too-long">Too Long</span>
</div>
                        
                    </div>
                    <input class="button primary submit" value="Submit" disabled="disabled" type="submit"></form>
            </div>
            <div class="success-template hidden">
                <h2>Congrats!</h2>
                <p>Your password has been changed</p>
            </div>
        </div>
    </div>
</div>





        
        <div></div>




        
        





<div class="popup registration-popup hidden">



    <div class="content">
        <a href="#" class="close"><i class="icon-tools_close"></i></a>
        <div class="modal__header">
            <div class="logoDropZone" data-pb-dropzone="logoDropZone">
                
            </div>

            <h2>Create a new account</h2>
        </div>

        <div class="modal__body">
            <form action="/action/registration" class="registration-form" method="post"><input name="redirectUri" value="/doi/full/10.1111/jbi.12786" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="4e647394-f751-4441-baa4-df426bca4b6e.email">Email or Customer ID</label>
                    </div>
                    <input id="4e647394-f751-4441-baa4-df426bca4b6e.email" class="email" name="email" size="15" type="text">
                </div>
                <div class="submit">
                    <input class="button submit primary" value="Register" disabled="disabled" type="submit">
                </div></form>
            <div class="center">
                <a href="https://onlinelibrary.wiley.com/action/showLogin" class="show-login">
                    
                    Returning user
                </a>
            </div>
        </div>
    </div>
</div>




        
        



<div class="popup top-drawer request-reset-password-drawer hidden">
    <div class="content">
        <div class="modal__header">
            <a href="#" class="cancel close"><i class="icon-close_thin"></i></a>

            <div class="header_logo">
                <img src="logo-header-1526603583437.png" alt="">
            </div>

            <h2>Forgot your password?</h2>
            <p class="sub">Enter your email address below. If your address has been previously registered, you will
                receive an email with instructions on how to reset your password. If you don't receive an email,
                you should register as a new user</p>
        </div>

        <div class="modal__body">

            









<div class="error"></div>


<form action="/action/requestResetPassword" class="request-reset-password-form" method="post"><input name="requestResetPassword" value="true" type="hidden">
    
            <div class="input-group">
                


















    
    
    <div class="input-group">
        <div class="label">

            
                    <label for="email">Email or Customer ID</label>
                
        </div>
        <input id="email" class="email" name="email" placeholder="Enter your email" autocorrect="off" spellcheck="false" autocapitalize="off" type="email">
        
    </div>

            </div>
        


    
            <div class="password-recaptcha-ajax"></div>
        

    <div class="form-btn">
        <input class="button btn primary submit" name="submit" value="RESET PASSWORD" disabled="disabled" type="submit">
    </div></form>
            

            <div class="message error"></div>

            <div class="success-template hidden">
                <p>Please check your email for your password reset instructions.</p>
            </div>
        </div>
    </div>
</div>





        
        




<div class="popup top-drawer request-username-drawer username-popup hidden">
    <a href="#" class="close"><i class="icon-close_thin"></i></a>
    <div class="content">
        <h2>Request Username</h2>
        <div class="form">
            <p>Can't sign in? Forgot your username?</p>
            <p class="sub">Enter your email address below and we will send you your username</p>
            <div class="message error"></div>
            <form action="/action/requestUsername" method="post"><input name="requestUsername" value="requestUsername" type="hidden">
                <div class="input-group">
                    <div class="label">
                        <label for="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email">Email or Customer ID</label>
                    </div>
                    <input id="ac834f24-aa07-4ad2-9d13-f77c843f21cb.email" class="email" name="email" size="15" type="text">
                </div>

                <div class="username-recaptcha-ajax">
                </div>

                <input class="button primary submit" name="submit" value="Submit" disabled="disabled" type="submit">
                <div class="center">
                    <a href="#" class="cancel">Close</a>
                </div></form>
        </div>
        <div class="success-template hidden note">
            <p>If the address matches an existing account you will receive an email with instructions to retrieve your username</p>
        </div>
    </div>
</div>




        
        <!-- Belongs at the bottom of the page -->

<script type="text/javascript">
if(window._satellite) { 
_satellite.pageBottom(); 
}
</script><script src="satellite-5aaec36164746d79c1001d14.js"></script>
        </div>
    </div><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh357" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.5941208592118427&amp;iit=1534165428438&amp;tmr=load%3D1534165426026%26core%3D1534165426130%26main%3D1534165428430%26ifr%3D1534165428472&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=onlinelibrary.wiley.com&amp;dr=&amp;du=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786&amp;href=https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786&amp;dt=A%20world%20of%20sequences%3A%20can%20we%20use%20georeferenced%20nucleotide%20databases%20for%20a%20robust%20automated%20phylogeography%3F%20-%20Gratton%20-%202017%20-%20Journal%20of%20Biogeography%20-%20Wiley%20Online%20Library&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=&amp;pc=men&amp;pub=xa-4faab26f2cff13a7&amp;ssl=1&amp;sid=5b7181b24841678e&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=&amp;csi=undefined&amp;rev=v8.3.26-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>


<script type="text/javascript" src="main.js"></script>
<script type="text/javascript" src="f8d8product.js"></script>
<script>var _prum=[['id','59e8fecb3847311aab7b23c6'],['mark','firstbyte',(new Date()).getTime()]];(function(){var s=document.getElementsByTagName('script')[0],p=document.createElement('script');p.async='async';p.src='//rum-static.pingdom.net/prum.min.js';s.parentNode.insertBefore(p,s);})();</script>

<iframe sandbox="allow-scripts allow-same-origin" title="Adobe ID Syncing iFrame" id="destination_publishing_iframe_wileypublishing_0" name="destination_publishing_iframe_wileypublishing_0_name" style="display: none; width: 0px; height: 0px;" src="https://wileypublishing.demdex.net/dest5.html?d_nsid=0#https%3A%2F%2Fonlinelibrary.wiley.com%2Fdoi%2Ffull%2F10.1111%2Fjbi.12786" class="aamIframeLoaded"></iframe><div id="crossmark-widget" style="display: none;"><div class="crossmark-reset crossmark-overlay"></div><div class="crossmark-reset crossmark-popup"><div class="crossmark-reset crossmark-popup__offset"><div class="crossmark-reset crossmark-popup__inner"><div class="crossmark-reset crossmark-popup__header"><a target="_blank" href="http://www.crossref.org/crossmark"><img class="crossmark-reset crossmark-popup__logo"></a><button class="crossmark-reset crossmark-popup__btn-close"></button></div><div class="crossmark-reset crossmark-popup__content-wrapper"><iframe class="crossmark-reset crossmark-popup__content"></iframe></div></div></div></div></div><div class="w-slide padding-top" style="top: 71px;"><div class="w-slide_head"><a href="#" class="w-slide__back"><i class=" icon-arrow_l" aria-hidden="true"></i>back</a><span class="w-slide__title"></span></div><div class="w-slide__content"></div></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-1" tabindex="0" style="display: none;"></ul><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all quickSearchAutocomplete" id="ui-id-2" tabindex="0" style="display: none;"></ul><script type="text/javascript" src="https://api.altmetric.com/v1/doi/10.1111/jbi.12786?callback=_altmetric.embed_callback&amp;domain=onlinelibrary.wiley.com&amp;key=3c130976ca2b8f2e88f8377633751ba1&amp;cache_until=16-13"></script><div class="altmetric-embed altmetric-popover altmetric-left" data-uuid="3515e7dc-5b72-c6c4-888e-1f618ba8709b" id="_altmetric_popover_el" style="margin: 0px; display: none;"><div class="altmetric_container" id="_altmetric_container"><div class="altmetric_arrow altmetric-left"></div><div id="_altmetric_popover_inner" class="altmetric-embed altmetric-popover-inner altmetric-floating altmetric-left" data-uuid="3515e7dc-5b72-c6c4-888e-1f618ba8709b"><div class="altmetric-embed altmetric-popover-content altmetric-floating"><div>    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #ffd140;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=blogs">
          Blogged by <b>1</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #74CFED;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=twitter">
          Tweeted by <b>26</b>
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #2445bd;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=facebook">
          On <b>1</b> Facebook pages
      </a>
    </div>
    <div style="padding-left: 10px; line-height:18px; border-left: 16px solid #E065BB;">
      <a class="link-to-altmetric-details-tab" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700&amp;tab=google">
          Mentioned in <b>1</b> Google+ posts
      </a>
    </div>

    <div class="altmetric-embed readers" style="margin-top: 10px;">
          <div class="altmetric-embed tip_mendeley" style="padding-left: 10px; line-height:18px; border-left: 16px solid #A60000;">
              <b>70</b> readers on Mendeley            
          </div>
          <div class="altmetric-embed tip_citeulike" style="padding-left: 10px; line-height:18px; border-left: 16px solid #BCD2EF;">
              <b>1</b> readers on CiteULike
          </div>
    </div>
    <div style="margin-top: 10px; text-align: center;">
        <a class="altmetric_details" target="_self" href="https://www.altmetric.com/details.php?domain=onlinelibrary.wiley.com&amp;citation_id=9241700">
            See more details
        </a>
        |
        <a href="javascript:void(0)" class="close-popover">
            Close this
        </a>
    </div>

    <a class="altmetric_embed close-popover" style="display: block; position: absolute; top: 10px; right: 15px; font-size: 1.2em; font-weight: bold; text-decoration: none; color: black; padding-bottom: 2em; padding-left: 2em;" href="javascript:void(0)">
        ×
    </a>
</div></div></div></div></div><div class="kxhead" data-id="JMOONjNh" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20226"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20227"></span><span class="kxtag kxinvisible" data-id="20229"><script>
(function() {
  // adadvisor redirects to http://logger... so it's not https safe
  if (location.protocol == "http:") {
     var u = "https://aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e";
     (new Image()).src = u;
   }
})();
</script></span><span class="kxtag kxinvisible" data-id="20234"><script type="text/javascript">
var findPh = document.getElementsByTagName("h1")[0]
if(findPh != null) { var pageheader = document.getElementsByTagName("h1")[0].innerHTML }

// Using Omniture prop1 to produce page attribute Page Type
Krux('set', 'page_attr_omniture_prop1', typeof window.s =='object' && window.s['prop1']);

// Using Omniture prop2 to produce page attribute Page Subtype
Krux('set', 'page_attr_omniture_prop2', typeof window.s =='object' && window.s['prop2']);

// Using Omniture prop3 to produce page attribute Title
Krux('set', 'page_attr_omniture_prop3', typeof window.s =='object' && window.s['prop3']);

// Using Omniture prop4 to produce page attribute Subject Codes
Krux('set', 'page_attr_omniture_prop4', typeof window.s =='object' && window.s['prop4']);

// Using Omniture prop7 to produce page attribute Article Title
Krux('set', 'page_attr_omniture_prop7', typeof window.s =='object' && window.s['prop7']);

// Using Omniture prop8 to produce page attribute Journal Volume and Issue
Krux('set', 'page_attr_omniture_prop8', typeof window.s =='object' && window.s['prop8']);

// Using Omniture prop9 to produce page attribute Article Access State
Krux('set', 'page_attr_omniture_prop9', typeof window.s =='object' && window.s['prop9']);

// Using Omniture prop10 to produce page attribute Publication Access Status
Krux('set', 'page_attr_omniture_prop10', typeof window.s =='object' && window.s['prop10']);

// Using Omniture prop13 to produce page attribute Journal DOI
Krux('set', 'page_attr_omniture_prop13', typeof window.s =='object' && window.s['prop13']);

// Using Omniture prop14 to produce page attribute Article / Chapter DOI
Krux('set', 'page_attr_omniture_prop14', typeof window.s =='object' && window.s['prop14']);

// Using Omniture prop15 to produce page attribute Issue DOI
Krux('set', 'page_attr_omniture_prop15', typeof window.s =='object' && window.s['prop15']);

// Using Omniture prop16 to produce page attribute MRW Chapter
Krux('set', 'page_attr_omniture_prop16', typeof window.s =='object' && window.s['prop16']);

// Using Omniture prop20 to produce page attribute Site Search Query
Krux('set', 'page_attr_omniture_prop20', typeof window.s =='object' && window.s['prop20']);

// Using Omniture prop26 to produce page attribute Logged In or Out
Krux('set', 'user_attr_omniture_prop26', typeof window.s =='object' && window.s['prop26']);

// Using Omniture channel to produce page attribute Channel
Krux('set', 'page_attr_omniture_channel', typeof window.s =='object' && window.s['channel']);

// Using Omniture prop33 to produce page attribute Primary Subject Code
Krux('set', 'page_attr_omniture_prop33', typeof window.s =='object' && window.s['prop33']);

// Using pageheader to produce page attribute Page Header
Krux('scrape', { 'page_attr_wileyonlinelib_pageheader': {javascript: 'pageheader'}});

</script></span><span class="kxtag kxinvisible" data-id="20238"></span><span class="kxtag kxinvisible" data-id="20241"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20243"></span><span class="kxtag kxinvisible" data-id="20247"><script type="text/javascript">
(function() {
    (new Image()).src = '//aa.agkn.com/adscores/g.js?sid=9212244187&_kdpid=2111c0af-fc3a-446f-ab07-63aa74fbde8e';
})();
</script></span><span class="kxtag kxinvisible" data-id="20248"><script>
(function(){
        var kuid = Krux('get', 'user');
        var prefix = window.location.protocol == 'https:' ? 'https:' : 'http:';
        if (kuid) {
           var url = prefix + '//pix.btrll.com/partner/868092.png';
           (new Image()).src = url;
        }
})();
</script></span><span class="kxtag kxinvisible" data-id="20251"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8da8b14d-5569-4bec-bcea-722864ee8d06&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20252"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bef9f122-393d-4c45-8d8d-32d8be7ac433&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iyzu39iyzud95&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20253"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';
 
            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
         
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=8bf57916-aac8-4f01-a386-4baf103b3e1f&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=bckw15bckwu20&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20254"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };

        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=d7158cb7-a851-4e3c-b7ab-cc9e815b2399&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=gwjn34gwjnh86&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20255"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=536f0daa-aaaa-4d9e-9a25-dde40646786a&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=iefs40iefsj26&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20256"><script>
    (function() {
        var kuid = Krux('get', 'user');
        if (kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=7c6392c9-e878-492c-8b14-bf06e3828ebd&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rowp70rowpu60&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span><span class="kxtag kxinvisible" data-id="20257"><script>
    (function() {
        var allowUserMatch = function() {
            var GDN_SITE_ID = '1282650';

            if (Krux('get', 'config').params.client_type === 'marketer') {
                try {
                    var params = Krux('require:marketer').getParams(
                            Krux('require:sizzle').find('script[src*="' + Krux('get', 'confid') + '"]')
                    );
                    return GDN_SITE_ID !== (params.siteid || params.kxsiteid);
                }
                catch (e) {
                    // In case we don't find the script tag or the URL parser fails, just allow matching.
                }
            }
            return true;
        };
        
        var kuid = Krux('get', 'user');
        if (allowUserMatch() && kuid) {
            var prefix = location.protocol == 'https:' ? "https:" : "http:";
            var kurl_params = encodeURIComponent("_kuid=" + kuid + "&_kdpid=bb8ae0e2-9cd7-45b2-ad37-7737269627d8&dlxid=<na_id>&dlxdata=<na_da>");
            var kurl = prefix + "//beacon.krxd.net/data.gif?" + kurl_params;
            var dlx_url = '//r.nexac.com/e/getdata.xgi?dt=br&pkey=rsxs71rsxsk73&ru=' + kurl;
            var i = new Image();
            i.src = dlx_url;
        }
    })();
</script></span></div></body><iframe src="https://injections.readcube.com/html/track_epdf.3307bb36.html?action=Link&amp;publisher=wiley&amp;status=OK" style="position: absolute; visibility: hidden; width: 0px; height: 0px;"></iframe><iframe id="kx-proxy-JMOONjNh" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=JMOONjNh&amp;kxt=https%3A%2F%2Fonlinelibrary.wiley.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>