<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" class="js jr js flexbox no-touch draganddrop csscolumns csstransforms csstransforms3d csstransitions boxflex jr-fs-12 jr-col-auto no-animate"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <!-- For pinger, set start time and add meta elements. -->
        <script type="text/javascript">var ncbi_startTime = new Date();</script>
        
        <!-- Logger begin -->
        <meta name="ncbi_db" content="books">
<meta name="ncbi_pdid" content="book-part">
<meta name="ncbi_acc" content="NBK179288">
<meta name="ncbi_domain" content="helpeutils">
<meta name="ncbi_report" content="reader">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_objectid" content="">
<meta name="ncbi_pcid" content="/NBK179288/?report=reader">
<meta name="ncbi_pagename" content="Entrez Direct: E-utilities on the UNIX Command Line - Entrez Programming Utilities Help - NCBI Bookshelf">
<meta name="ncbi_bookparttype" content="chapter">
<meta name="ncbi_app" content="bookshelf">
        <!-- Logger end -->
        
        <!--component id="Page" label="meta"/-->
        <script type="text/javascript" src="jr.js"> </script><title>Entrez Direct: E-utilities on the UNIX Command Line - Entrez Programming Utilities Help - NCBI Bookshelf</title>
<meta charset="utf-8">
<meta name="apple-mobile-web-app-capable" content="no">
<meta name="viewport" content="initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no">
<meta name="jr-col-layout" content="auto">
<meta name="jr-prev-unit" content="/books/n/helpeutils/chapter5/?report=reader">
<meta name="bk-toc-url" content="/books/n/helpeutils/?report=toc">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE">
<meta name="citation_inbook_title" content="Entrez Programming Utilities Help [Internet]">
<meta name="citation_title" content="Entrez Direct: E-utilities on the UNIX Command Line">
<meta name="citation_publisher" content="National Center for Biotechnology Information (US)">
<meta name="citation_date" content="2019/02/11">
<meta name="citation_author" content="Jonathan Kans">
<meta name="citation_fulltext_html_url" content="https://www.ncbi.nlm.nih.gov/books/NBK179288/">
<link rel="schema.DC" href="http://purl.org/DC/elements/1.0/">
<meta name="DC.Title" content="Entrez Direct: E-utilities on the UNIX Command Line">
<meta name="DC.Type" content="Text">
<meta name="DC.Publisher" content="National Center for Biotechnology Information (US)">
<meta name="DC.Contributor" content="Jonathan Kans">
<meta name="DC.Date" content="2019/02/11">
<meta name="DC.Identifier" content="https://www.ncbi.nlm.nih.gov/books/NBK179288/">
<meta name="description" content="Entrez Direct (EDirect) provides access to the NCBI's suite of interconnected databases (publication, sequence, structure, gene, variation, expression, etc.) from a UNIX terminal window. Functions take search terms from command-line arguments. Individual operations are combined to build multi-step queries. Record retrieval and formatting normally complete the process.">
<meta name="og:title" content="Entrez Direct: E-utilities on the UNIX Command Line">
<meta name="og:type" content="book">
<meta name="og:description" content="Entrez Direct (EDirect) provides access to the NCBI's suite of interconnected databases (publication, sequence, structure, gene, variation, expression, etc.) from a UNIX terminal window. Functions take search terms from command-line arguments. Individual operations are combined to build multi-step queries. Record retrieval and formatting normally complete the process.">
<meta name="og:url" content="https://www.ncbi.nlm.nih.gov/books/NBK179288/">
<meta name="og:site_name" content="NCBI Bookshelf">
<meta name="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/bookshelf/thumbs/th-helpeutils-lrg.png">
<meta name="twitter:card" content="summary">
<meta name="twitter:site" content="@ncbibooks">
<meta name="bk-non-canon-loc" content="/books/n/helpeutils/chapter6/?report=reader">
<link rel="canonical" href="https://www.ncbi.nlm.nih.gov/books/NBK179288/">
<link href="css.css" rel="stylesheet" type="text/css">
<link rel="stylesheet" href="libs.css">
<link rel="stylesheet" href="jr.css">
<meta name="format-detection" content="telephone=no">
<link rel="stylesheet" href="books.css" type="text/css">
<link rel="stylesheet" href="books_print.css" type="text/css" media="print">
<link rel="stylesheet" href="books_reader.css" type="text/css">
<style type="text/css">p a.figpopup{display:inline !important} .bk_tt {font-family: monospace}  .first-line-outdent .bk_ref {display: inline}  .body-content h2, .body-content .h2  {border-bottom: 1px solid #97B0C8} .body-content h2.inline {border-bottom: none} a.page-toc-label , .jig-ncbismoothscroll a {text-decoration:none;border:0 !important} .temp-labeled-list  .graphic {display:inline-block !important} .temp-labeled-list  img{width:100%}</style>

    <link rel="shortcut icon" href="https://www.ncbi.nlm.nih.gov/favicon.ico">
<meta name="ncbi_phid" content="CE8A34B7CC9D13D100000000008E0057.m_4">
<meta name="referrer" content="origin-when-cross-origin"><link type="text/css" rel="stylesheet" href="3849091.css"><script async="1" src="analytics.js"></script></head>
    <body tabindex="0">
        <!-- Book content! -->
        

<div id="jr" data-jr-path="/corehtml/pmc/jatsreader/br_1.2/"><div id="ZN_7ZDsgy7KdsRefc1"><!--qs--></div><div class="jr-unsupported"><table class="modal"><tbody><tr><td><span class="attn inline-block"></span><br>Your browser does not support the NLM PubReader view.<br>Go to <a href="https://www.ncbi.nlm.nih.gov/pmc/about/pr-browsers/">this page</a> to see a list of supported browsers<br>or return to the <br><a href="https://www.ncbi.nlm.nih.gov/books/NBK179288/?report=classic">regular view</a>.</td></tr></tbody></table></div><div id="jr-ui" class=""><nav id="jr-head"><div class="flexh tb"><div id="jr-tb1"><a id="jr-links-sw" class="" title="Links"><svg xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 70.6 85.3" style="enable-background:new 0 0 70.6 85.3;vertical-align:middle" xml:space="preserve" width="24" height="24">
								<style type="text/css">.st0{fill:#939598;}</style>
								<g>
									<path class="st0" d="M36,0C12.8,2.2-22.4,14.6,19.6,32.5C40.7,41.4-30.6,14,35.9,9.8"></path>
									<path class="st0" d="M34.5,85.3c23.2-2.2,58.4-14.6,16.4-32.5c-21.1-8.9,50.2,18.5-16.3,22.7"></path>
									<path class="st0" d="M34.7,37.1c66.5-4.2-4.8-31.6,16.3-22.7c42.1,17.9,6.9,30.3-16.4,32.5h1.7c-66.2,4.4,4.8,31.6-16.3,22.7           c-42.1-17.9-6.9-30.3,16.4-32.5"></path>
								</g>
							</svg> Books</a></div><div class="jr-rhead f1 flexh"><div class="head"><a href="https://www.ncbi.nlm.nih.gov/books/n/helpeutils/chapter5/?report=reader"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M75,30 c-80,60 -80,0 0,60 c-30,-60 -30,0 0,-60"></path><text x="20" y="28" textLength="60" style="font-size:25px">Prev</text></svg></a></div><div class="body"><div class="t">Entrez Direct: E-utilities on the UNIX Command Line</div><div class="j">Entrez Programming Utilities Help [Internet]</div></div><div class="tail"></div></div><div id="jr-tb2"><a id="jr-bkhelp-sw" class="btn wsprkl" title="Help with NLM PubReader">?</a><a id="jr-help-sw" class="btn wsprkl" title="Settings and typography in NLM PubReader"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" preserveAspectRatio="none"><path d="M462,283.742v-55.485l-29.981-10.662c-11.431-4.065-20.628-12.794-25.274-24.001  c-0.002-0.004-0.004-0.009-0.006-0.013c-4.659-11.235-4.333-23.918,0.889-34.903l13.653-28.724l-39.234-39.234l-28.72,13.652  c-10.979,5.219-23.68,5.546-34.908,0.889c-0.005-0.002-0.01-0.003-0.014-0.005c-11.215-4.65-19.933-13.834-24-25.273L283.741,50  h-55.484l-10.662,29.981c-4.065,11.431-12.794,20.627-24.001,25.274c-0.005,0.002-0.009,0.004-0.014,0.005  c-11.235,4.66-23.919,4.333-34.905-0.889l-28.723-13.653l-39.234,39.234l13.653,28.721c5.219,10.979,5.545,23.681,0.889,34.91  c-0.002,0.004-0.004,0.009-0.006,0.013c-4.649,11.214-13.834,19.931-25.271,23.998L50,228.257v55.485l29.98,10.661  c11.431,4.065,20.627,12.794,25.274,24c0.002,0.005,0.003,0.01,0.005,0.014c4.66,11.236,4.334,23.921-0.888,34.906l-13.654,28.723  l39.234,39.234l28.721-13.652c10.979-5.219,23.681-5.546,34.909-0.889c0.005,0.002,0.01,0.004,0.014,0.006  c11.214,4.649,19.93,13.833,23.998,25.271L228.257,462h55.484l10.595-29.79c4.103-11.538,12.908-20.824,24.216-25.525  c0.005-0.002,0.009-0.004,0.014-0.006c11.127-4.628,23.694-4.311,34.578,0.863l28.902,13.738l39.234-39.234l-13.66-28.737  c-5.214-10.969-5.539-23.659-0.886-34.877c0.002-0.005,0.004-0.009,0.006-0.014c4.654-11.225,13.848-19.949,25.297-24.021  L462,283.742z M256,331.546c-41.724,0-75.548-33.823-75.548-75.546s33.824-75.547,75.548-75.547  c41.723,0,75.546,33.824,75.546,75.547S297.723,331.546,256,331.546z"></path></svg></a><a id="jr-fip-sw" class="btn wsprkl" title="Find"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 550 600" preserveAspectRatio="none"><path fill="none" stroke="#000" stroke-width="36" stroke-linecap="round" style="fill:#FFF" d="m320,350a153,153 0 1,0-2,2l170,170m-91-117 110,110-26,26-110-110"></path></svg></a><a id="jr-rtoc-sw" class="btn wsprkl" title="Table of Contents"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M20,20h10v8H20V20zM36,20h44v8H36V20zM20,37.33h10v8H20V37.33zM36,37.33h44v8H36V37.33zM20,54.66h10v8H20V54.66zM36,54.66h44v8H36V54.66zM20,72h10v8 H20V72zM36,72h44v8H36V72z"></path></svg></a></div></div></nav><nav id="jr-dash" class="noselect"><nav id="jr-dash" class="noselect"><div id="jr-pi" class=""><a id="jr-pi-prev" class="hidden" title="Previous page"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M75,30 c-80,60 -80,0 0,60 c-30,-60 -30,0 0,-60"></path><text x="20" y="28" textLength="60" style="font-size:25px">Prev</text></svg></a><div class="pginfo">Page <i class="jr-pg-pn">0</i> of <i class="jr-pg-lp">0</i></div><a id="jr-pi-next" class="hidden" title="Next page"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M25,30c80,60 80,0 0,60 c30,-60 30,0 0,-60"></path><text x="20" y="28" textLength="60" style="font-size:25px">Next</text></svg></a></div><div id="jr-is-tb"><a id="jr-is-sw" class="btn wsprkl" title="Switch between Figures/Tables strip and Progress bar"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><rect x="10" y="40" width="20" height="20"></rect><rect x="40" y="40" width="20" height="20"></rect><rect x="70" y="40" width="20" height="20"></rect></svg></a></div><nav id="jr-istrip" class="istrip hidden"><a id="jr-is-prev" href="#" class="hidden" title="Previous"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M80,40 60,65 80,90 70,90 50,65 70,40z M50,40 30,65 50,90 40,90 20,65 40,40z"></path><text x="35" y="25" textLength="60" style="font-size:25px">Prev</text></svg></a><a id="jr-is-next" href="#" class="hidden" title="Next"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M20,40 40,65 20,90 30,90 50,65 30,40z M50,40 70,65 50,90 60,90 80,65 60,40z"></path><text x="15" y="25" textLength="60" style="font-size:25px">Next</text></svg></a></nav><nav id="jr-progress" class=""><div class="pr-text" style="display: none;"><input data-orig-type="range" class="pr-pn" style="display: none;" value="50" type="text">  <span class="pr-np"></span> of <span class="pr-lp"></span></div><div class="pr-slider"><div></div><a href="#" class="pr-handle"></a></div></nav></nav></nav><aside id="jr-links-p" class="hidden flexv" tabindex="0"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">NCBI Bookshelf</div></div><div class="cnt lol f1"><a href="https://www.ncbi.nlm.nih.gov/books/">Home</a><a href="https://www.ncbi.nlm.nih.gov/books/browse/">Browse All Titles</a><a class="btn share" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https://www.ncbi.nlm.nih.gov/books/NBK179288/"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 33 33" style="vertical-align:middle" width="24" height="24" preserveAspectRatio="none"><g><path d="M 17.996,32L 12,32 L 12,16 l-4,0 l0-5.514 l 4-0.002l-0.006-3.248C 11.993,2.737, 13.213,0, 18.512,0l 4.412,0 l0,5.515 l-2.757,0 c-2.063,0-2.163,0.77-2.163,2.209l-0.008,2.76l 4.959,0 l-0.585,5.514L 18,16L 17.996,32z"></path></g></svg> Share on Facebook</a><a class="btn share" target="_blank" href="https://twitter.com/intent/tweet?url=https://www.ncbi.nlm.nih.gov/books/NBK179288/&amp;text=Entrez%20Direct%3A%20E-utilities%20on%20the%20UNIX%20Command%20Line"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 33 33" style="vertical-align:middle" width="24" height="24"><g><path d="M 32,6.076c-1.177,0.522-2.443,0.875-3.771,1.034c 1.355-0.813, 2.396-2.099, 2.887-3.632 c-1.269,0.752-2.674,1.299-4.169,1.593c-1.198-1.276-2.904-2.073-4.792-2.073c-3.626,0-6.565,2.939-6.565,6.565 c0,0.515, 0.058,1.016, 0.17,1.496c-5.456-0.274-10.294-2.888-13.532-6.86c-0.565,0.97-0.889,2.097-0.889,3.301 c0,2.278, 1.159,4.287, 2.921,5.465c-1.076-0.034-2.088-0.329-2.974-0.821c-0.001,0.027-0.001,0.055-0.001,0.083 c0,3.181, 2.263,5.834, 5.266,6.438c-0.551,0.15-1.131,0.23-1.73,0.23c-0.423,0-0.834-0.041-1.235-0.118 c 0.836,2.608, 3.26,4.506, 6.133,4.559c-2.247,1.761-5.078,2.81-8.154,2.81c-0.53,0-1.052-0.031-1.566-0.092 c 2.905,1.863, 6.356,2.95, 10.064,2.95c 12.076,0, 18.679-10.004, 18.679-18.68c0-0.285-0.006-0.568-0.019-0.849 C 30.007,8.548, 31.12,7.392, 32,6.076z"></path></g></svg> Share on Twitter</a><a class="btn share" target="_blank" href="https://plus.google.com/share?url=https://www.ncbi.nlm.nih.gov/books/NBK179288/"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 33 33" style="vertical-align:middle" width="24" height="24"><g><path d="M 17.471,2c0,0-6.28,0-8.373,0C 5.344,2, 1.811,4.844, 1.811,8.138c0,3.366, 2.559,6.083, 6.378,6.083 c 0.266,0, 0.524-0.005, 0.776-0.024c-0.248,0.475-0.425,1.009-0.425,1.564c0,0.936, 0.503,1.694, 1.14,2.313 c-0.481,0-0.945,0.014-1.452,0.014C 3.579,18.089,0,21.050,0,24.121c0,3.024, 3.923,4.916, 8.573,4.916 c 5.301,0, 8.228-3.008, 8.228-6.032c0-2.425-0.716-3.877-2.928-5.442c-0.757-0.536-2.204-1.839-2.204-2.604 c0-0.897, 0.256-1.34, 1.607-2.395c 1.385-1.082, 2.365-2.603, 2.365-4.372c0-2.106-0.938-4.159-2.699-4.837l 2.655,0 L 17.471,2z M 14.546,22.483c 0.066,0.28, 0.103,0.569, 0.103,0.863c0,2.444-1.575,4.353-6.093,4.353 c-3.214,0-5.535-2.034-5.535-4.478c0-2.395, 2.879-4.389, 6.093-4.354c 0.75,0.008, 1.449,0.129, 2.083,0.334 C 12.942,20.415, 14.193,21.101, 14.546,22.483z M 9.401,13.368c-2.157-0.065-4.207-2.413-4.58-5.246 c-0.372-2.833, 1.074-5.001, 3.231-4.937c 2.157,0.065, 4.207,2.338, 4.58,5.171 C 13.004,11.189, 11.557,13.433, 9.401,13.368zM 26,8L 26,2L 24,2L 24,8L 18,8L 18,10L 24,10L 24,16L 26,16L 26,10L 32,10L 32,8 z"></path></g></svg> Share on Google+</a></div></aside><aside id="jr-rtoc-p" class="hidden flexv" tabindex="0"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Table of Content</div></div><div class="cnt lol f1"><ul><li><a href="https://www.ncbi.nlm.nih.gov/books/n/helpeutils/?report=reader">Title Information</a></li><li><a href="https://www.ncbi.nlm.nih.gov/books/n/helpeutils/toc/?report=reader">Table of Contents Page</a></li></ul></div></aside><aside id="jr-help-p" class="hidden flexv" tabindex="0"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Settings</div></div><div class="cnt f1"><div id="jr-typo-p" class="typo" tabindex="0"><div><a class="sf btn wsprkl">A-</a><a class="lf btn wsprkl">A+</a></div><div><a class="bcol-auto btn wsprkl on"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 100" preserveAspectRatio="none"><text x="10" y="70" style="font-size:60px;font-family: Trebuchet MS, ArialMT, Arial, sans-serif" textLength="180">AUTO</text></svg></a><a class="bcol-1 btn wsprkl"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M15,25 85,25zM15,40 85,40zM15,55 85,55zM15,70 85,70z"></path></svg></a><a class="bcol-2 btn wsprkl"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M5,25 45,25z M55,25 95,25zM5,40 45,40z M55,40 95,40zM5,55 45,55z M55,55 95,55zM5,70 45,70z M55,70 95,70z"></path></svg></a></div></div><div class="lol"><a class="" href="https://www.ncbi.nlm.nih.gov/books/NBK179288/?report=classic">Switch to classic view</a><a href="https://www.ncbi.nlm.nih.gov/books/NBK179288/pdf/Bookshelf_NBK179288.pdf">PDF (548K)</a><a href="https://www.ncbi.nlm.nih.gov/books/n/helpeutils/pdf/">PDF (1.8M)</a><a href="https://www.ncbi.nlm.nih.gov/books/NBK179288/?report=printable">Print View</a></div></div></aside><aside id="jr-bkhelp-p" class="hidden flexv" tabindex="0"><div class="tb sk-htbar flexh"><div><a class="jr-p-close btn wsprkl">Done</a></div><div class="title-text f1">Help</div></div><div class="cnt f1 lol"><a id="jr-helpobj-sw" data-path="/corehtml/pmc/jatsreader/br_1.2/" data-href="/corehtml/pmc/jatsreader/br_1.2/img/bookshelf/help.xml" href="">Help</a><a href="mailto:bookshelf@ncbi.nlm.nih.gov?subject=PubReader%20feedback%20%2F%20NBK179288%20%2F%20sid%3A8A1A6945BBF57001_0075SID%20%2F%20phid%3ACE8A34B7CC9D13D100000000008E0057.3">Send us feedback</a><a id="jr-about-sw" data-path="/corehtml/pmc/jatsreader/br_1.2/" data-href="/corehtml/pmc/jatsreader/br_1.2/img/bookshelf/about.xml" href="">About PubReader</a><a id="myncbi-link" data-path="/corehtml/pmc/jatsreader/br_1.2/" data-href="/corehtml/pmc/jatsreader/br_1.2/img/bookshelf/about.xml" href="https://www.ncbi.nlm.nih.gov/myncbi/?back_url=https://www.ncbi.nlm.nih.gov/books/NBK179288/?report=reader">My NCBI</a></div></aside><aside id="jr-objectbox" class="thidden hidden" tabindex="0"><div class="jr-objectbox-close wsprkl">✘</div><div class="jr-objectbox-inner cnt"><div class="jr-objectbox-drawer"></div></div></aside><nav id="jr-pm-left" class="hidden"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 800" preserveAspectRatio="none"><text font-stretch="ultra-condensed" x="800" y="-15" text-anchor="end" transform="rotate(90)" font-size="18" letter-spacing=".1em">Previous Page</text></svg></nav><nav id="jr-pm-right" class="hidden"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 800" preserveAspectRatio="none"><text font-stretch="ultra-condensed" x="800" y="-15" text-anchor="end" transform="rotate(90)" font-size="18" letter-spacing=".1em">Next Page</text></svg></nav><nav id="jr-fip" class="hidden" tabindex="0"><nav id="jr-fip-term-p"><input placeholder="search this page" id="jr-fip-term" autocorrect="off" autocomplete="off" type="search"><a id="jr-fip-mg" class="wsprkl btn" title="Find"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 550 600" preserveAspectRatio="none"><path fill="none" stroke="#000" stroke-width="36" stroke-linecap="round" style="fill:#FFF" d="m320,350a153,153 0 1,0-2,2l170,170m-91-117 110,110-26,26-110-110"></path></svg></a><a id="jr-fip-done" class="wsprkl btn" title="Dismiss find">✘</a></nav><nav id="jr-fip-info-p"><a id="jr-fip-prev" class="wsprkl btn" title="Jump to previuos match">◀</a><button id="jr-fip-matches">no matches yet</button><a id="jr-fip-next" class="wsprkl btn" title="Jump to next match">▶</a></nav></nav></div><div id="jr-epub-interstitial" class="hidden"></div><div id="jr-content" style="top: 56px; bottom: -56px;" tabindex="0"><article data-type="main" style="transform: none;"><div class="main-content lit-style" itemscope="itemscope" itemtype="http://schema.org/CreativeWork"><div class="meta-content fm-sec"><div class="fm-sec"><h1 id="_NBK179288_"><span class="title" itemprop="name">Entrez Direct: E-utilities on the UNIX Command Line</span></h1><p class="contribs">Kans J.</p><p class="fm-aai"><a href="#_NBK179288_pubdet_">Publication Details</a></p><p><em>Estimated reading time: 1 hour, 10 minutes</em></p></div></div><div class="jig-ncbiinpagenav body-content whole_rhythm" data-jigconfig="allHeadingLevels: ['h2'],smoothScroll: false" itemprop="text"><div id="chapter6.Getting_Started"><h2 id="_chapter6_Getting_Started_">Getting Started</h2><div id="chapter6.Introduction"><h3>Introduction</h3><p>Entrez
 Direct (EDirect) provides access to the NCBI's suite of interconnected 
databases (publication, sequence, structure, gene, variation, 
expression, etc.) from a UNIX terminal window. Functions take search 
terms from command-line arguments. Individual operations are combined to
 build multi-step queries. Record retrieval and formatting normally 
complete the process.</p><p>EDirect also includes an argument-driven 
function that simplifies the extraction of data from document summaries 
or other results that are returned in structured XML format. This can 
eliminate the need for writing custom software to answer ad hoc 
questions. Queries can move seamlessly between EDirect commands and UNIX
 utilities or scripts to perform actions that cannot be accomplished 
entirely within Entrez.</p></div><div id="chapter6.Installation"><h3>Installation</h3><p>EDirect
 will run on UNIX and Macintosh computers that have the Perl language 
installed, and under the Cygwin UNIX-emulation environment on Windows 
PCs. To install the EDirect software, copy the following commands and 
paste them into a terminal window:</p><pre>  <br>  cd ~<br>  /bin/bash<br>  perl -MNet::FTP -e \<br>    '$ftp = new Net::FTP("ftp.ncbi.nlm.nih.gov", Passive =&gt; 1);<br>     $ftp-&gt;login; $ftp-&gt;binary;<br>     $ftp-&gt;get("/entrez/entrezdirect/edirect.tar.gz");'<br>  gunzip -c edirect.tar.gz | tar xf -<br>  rm edirect.tar.gz<br>  builtin exit<br>  export PATH=${PATH}:$HOME/edirect &gt;&amp; /dev/null || setenv PATH "${PATH}:$HOME/edirect"<br>  ./edirect/setup.sh<br>  </pre><p>This
 downloads several scripts into an "edirect" folder in the user's home 
directory. The setup.sh script then downloads any missing Perl modules, 
and may print an additional command for updating the PATH environment 
variable in the user's configuration file. Copy that command, if 
present, and paste it into the terminal window to complete the 
installation process. The editing instructions will look something like:</p><pre>  <br>  echo "export PATH=\$PATH:\$HOME/edirect" &gt;&gt; $HOME/.bash_profile<br>  </pre></div><div id="chapter6.Entrez_Direct_Functions"><h3>Entrez Direct Functions</h3><p>Navigation functions support exploration within the Entrez databases:</p><ul><li class="half_rhythm"><div><b>esearch</b> performs a new Entrez search using terms in indexed fields.</div></li><li class="half_rhythm"><div><b>elink</b> looks up neighbors (within a database) or links (between databases).</div></li><li class="half_rhythm"><div><b>efilter</b> filters or restricts the results of a previous query.</div></li></ul><p>Records can be retrieved in specified formats or as document summaries:</p><ul><li class="half_rhythm"><div><b>efetch</b> downloads records or reports in a designated format.</div></li></ul><p>Desired fields from XML results can be extracted without writing a program:</p><ul><li class="half_rhythm"><div><b>xtract</b> converts EDirect XML output into a table of data values.</div></li></ul><p>Several additional functions are also provided:</p><ul><li class="half_rhythm"><div><b>einfo</b> obtains information on indexed fields in an Entrez database.</div></li><li class="half_rhythm"><div><b>epost</b> uploads unique identifiers (UIDs) or sequence accession numbers.</div></li><li class="half_rhythm"><div><b>nquire</b> sends a URL request to a web page or CGI service.</div></li></ul></div><div id="chapter6.Entering_Query_Commands"><h3>Entering Query Commands</h3><p>UNIX
 programs are run by typing the name of the program and then supplying 
any required or optional arguments on the command line. Argument names 
are letters or words that start with a dash ("-") character.</p><p>In 
order to begin an Entrez search, the user types "esearch" and then 
enters the required -db (database) and -query arguments. A query on 
unqualified search terms:</p><pre>  esearch -db pubmed -query "opsin gene conversion"</pre><p>constructs
 the appropriate Entrez Utilities (E-utilities) URL from the query terms
 and executes the search. EDirect handles many technical details behind 
the scenes (avoiding the learning curve normally required for 
E-utilities programming), and saves the results on the Entrez history 
server.</p></div><div id="chapter6.Constructing_MultiStep_Queries"><h3>Constructing Multi-Step Queries</h3><p>EDirect
 allows individual operations to be described separately, combining them
 into a multi-step query by using the vertical bar ("|") UNIX pipe 
symbol. Piping esearch to elink:</p><pre>  esearch -db pubmed -query "opsin gene conversion" | elink -related</pre><p>will look up related articles (precomputed PubMed neighbors) of the initial results.</p></div><div id="chapter6.Writing_Commands_on_Multiple_Li"><h3>Writing Commands on Multiple Lines</h3><p>A
 query can be continued on the next line by typing the backslash ("\") 
UNIX escape character immediately before pressing the Return key. 
Continuing the query links to all protein sequences published in the 
neighbor articles:</p><pre>  esearch -db pubmed -query "opsin gene conversion" | \<br>  elink -related | \<br>  elink -target protein</pre><p>The vertical bar pipe symbol also allows the query to continue on the next line.</p></div><div id="chapter6.Retrieving_PubMed_Reports"><h3>Retrieving PubMed Reports</h3><p>Piping PubMed query results to efetch and specifying the "abstract" format:</p><pre>  esearch -db pubmed -query "lycopene cyclase" |<br>  efetch -format abstract</pre><p>returns a set of reports that can be read by a person:</p><pre>  ...<br>  10. PLoS One. 2013;8(3):e58144. doi: 10.1371/journal.pone.0058144. Epub ...<br><br>  Levels of lycopene β-cyclase 1 modulate carotenoid gene expression and<br>  accumulation in Daucus carota.<br><br>  Moreno JC(1), Pizarro L, Fuentes P, Handford M, Cifuentes V, Stange C.<br><br>  Author information:<br>  (1)Departamento de Biología, Facultad de Ciencias, Universidad de Chile,<br>  Santiago, Chile.<br><br>  Plant carotenoids are synthesized and accumulated in plastids through a<br>  highly regulated pathway. Lycopene β-cyclase (LCYB) is a key enzyme<br>  involved directly in the synthesis of α-carotene and β-carotene through<br>  ...</pre><p>Using
 efetch -format "medline" instead produces a report that can be entered 
into common bibliographic management software packages:</p><pre>  ...<br>  PMID- 23555569<br>  OWN - NLM<br>  STAT- MEDLINE<br>  DA  - 20130404<br>  DCOM- 20130930<br>  LR  - 20131121<br>  IS  - 1932-6203 (Electronic)<br>  IS  - 1932-6203 (Linking)<br>  VI  - 8<br>  IP  - 3<br>  DP  - 2013<br>  TI  - Levels of lycopene beta-cyclase 1 modulate carotenoid gene expression<br>        and accumulation in Daucus carota.<br>  PG  - e58144<br>  LID - 10.1371/journal.pone.0058144 [doi]<br>  AB  - Plant carotenoids are synthesized and accumulated in plastids<br>        through a highly regulated pathway. Lycopene beta-cyclase (LCYB) is a<br>        key enzyme involved directly in the synthesis of alpha-carotene and<br>        ...</pre></div><div id="chapter6.Retrieving_Sequence_Reports"><h3>Retrieving Sequence Reports</h3><p>Nucleotide and protein records can be downloaded in FASTA format:</p><pre>  esearch -db protein -query "lycopene cyclase" |<br>  efetch -format fasta</pre><p>which consists of a definition line followed by the sequence:</p><pre>  ...<br>  &gt;gi|735882|gb|<a href="https://www.ncbi.nlm.nih.gov/protein/735882/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAA81880.1</a>| lycopene cyclase [Arabidopsis thaliana]<br>  MDTLLKTPNKLDFFIPQFHGFERLCSNNPYPSRVRLGVKKRAIKIVSSVVSGSAALLDLVPETKKENLDF<br>  ELPLYDTSKSQVVDLAIVGGGPAGLAVAQQVSEAGLSVCSIDPSPKLIWPNNYGVWVDEFEAMDLLDCLD<br>  TTWSGAVVYVDEGVKKDLSRPYGRVNRKQLKSKMLQKCITNGVKFHQSKVTNVVHEEANSTVVCSDGVKI<br>  QASVVLDATGFSRCLVQYDKPYNPGYQVAYGIIAEVDGHPFDVDKMVFMDWRDKHLDSYPELKERNSKIP<br>  TFLYAMPFSSNRIFLEETSLVARPGLRMEDIQERMAARLKHLGINVKRIEEDERCVIPMGGPLPVLPQRV<br>  VGIGGTAGMVHPSTGYMVARTLAAAPIVANAIVRYLGSPSSNSLRGDQLSAEVWRDLWPIERRRQREFFC<br>  FGMDILLKLDLDATRRFFDAFFDLQPHYWHGFLSSRLFLPELLVFGLSLFSHASNTSRLEIMTKGTVPLA<br>  KMINNLVQDRD<br>  ...</pre><p>Additional FASTA -format variants are fasta_cds_na, fasta_cds_aa, and gene_fasta.</p><p>Sequence
 records can also be obtained as GenBank (-format gb) or GenPept 
(-format gp) flatfiles, which have features annotating particular 
regions of the sequence:</p><pre>  ...<br>  LOCUS       <a href="https://www.ncbi.nlm.nih.gov/protein/735882/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAA81880</a>                 501 aa            linear   PLN ...<br>  DEFINITION  lycopene cyclase [Arabidopsis thaliana].<br>  ACCESSION   <a href="https://www.ncbi.nlm.nih.gov/protein/735882/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAA81880</a><br>  VERSION     <a href="https://www.ncbi.nlm.nih.gov/protein/735882/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAA81880.1</a>  GI:735882<br>  DBSOURCE    locus ATHLYC accession <a href="https://www.ncbi.nlm.nih.gov/nuccore/735881" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">L40176.1</a><br>  KEYWORDS    .<br>  SOURCE      Arabidopsis thaliana (thale cress)<br>    ORGANISM  Arabidopsis thaliana<br>              Eukaryota; Viridiplantae; Streptophyta; Embryophyta;<br>              Tracheophyta; Spermatophyta; Magnoliophyta; eudicotyledons;<br>              Brassicales; Brassicaceae; Camelineae; Arabidopsis.<br>  REFERENCE   1  (residues 1 to 501)<br>    AUTHORS   Scolnik,P.A. and Bartley,G.E.<br>    TITLE     Nucleotide sequence of lycopene cyclase (GenBank <a href="https://www.ncbi.nlm.nih.gov/nuccore/735881" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">L40176</a>) from<br>              Arabidopsis (PGR95-019)<br>    JOURNAL   Plant Physiol. 108 (3), 1343 (1995)<br>  ...<br>  FEATURES             Location/Qualifiers<br>       source          1..501<br>                       /organism="Arabidopsis thaliana"<br>                       /db_xref="taxon:3702"<br>       Protein         1..501<br>                       /product="lycopene cyclase"<br>       transit_peptide 1..80<br>       mat_peptide     81..501<br>                       /product="lycopene cyclase"<br>       CDS             1..501<br>                       /gene="LYC"<br>                       /coded_by="<a href="https://www.ncbi.nlm.nih.gov/nuccore/735881" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">L40176.1</a>:2..1507"<br>  ORIGIN<br>          1 mdtllktpnk ldffipqfhg ferlcsnnpy psrvrlgvkk raikivssvv sgsaalldlv<br>         61 petkkenldf elplydtsks qvvdlaivgg gpaglavaqq vseaglsvcs idpspkliwp<br>        121 nnygvwvdef eamdlldcld ttwsgavvyv degvkkdlsr pygrvnrkql kskmlqkcit<br>        181 ngvkfhqskv tnvvheeans tvvcsdgvki qasvvldatg fsrclvqydk pynpgyqvay<br>        241 giiaevdghp fdvdkmvfmd wrdkhldsyp elkernskip tflyampfss nrifleetsl<br>        301 varpglrmed iqermaarlk hlginvkrie edercvipmg gplpvlpqrv vgiggtagmv<br>        361 hpstgymvar tlaaapivan aivrylgsps snslrgdqls aevwrdlwpi errrqreffc<br>        421 fgmdillkld ldatrrffda ffdlqphywh gflssrlflp ellvfglslf shasntsrle<br>        481 imtkgtvpla kminnlvqdr d<br>  //<br>  ...</pre></div></div><div id="chapter6.Searching_and_Filtering"><h2 id="_chapter6_Searching_and_Filtering_">Searching and Filtering</h2><div id="chapter6.Restricting_Query_Results"><h3>Restricting Query Results</h3><p>The
 current results can be refined by further term searching in Entrez 
(useful in the protein database for limiting BLAST neighbors to a 
taxonomic subset):</p><pre>  esearch -db pubmed -query "opsin gene conversion" |<br>  elink -related |<br>  efilter -query "tetrachromacy"</pre><p>Results can also be filtered by time. For example, the following statements:</p><pre>  efilter -days 60 -datetype PDAT<br>  efilter -mindate 1990 -maxdate 1999 -datetype PDAT</pre><p>restrict results to articles published in the previous two months or in the 1990s, respectively.</p></div><div id="chapter6.Qualifying_Queries_by_Indexed_F"><h3>Qualifying Queries by Indexed Field</h3><p>Query
 terms in esearch or efilter can be qualified by entering an indexed 
field abbreviation in brackets. Boolean operators and parentheses can 
also be used in the query expression for more complex searches.</p><p>Commonly-used fields for PubMed queries include:</p><pre>  [AFFL]    Affiliation<br>  [ALL]     All Fields<br>  [AUTH]    Author<br>  [FAUT]    Author - First<br>  [LAUT]    Author - Last<br>  [PDAT]    Date - Publication<br>  [FILT]    Filter<br>  [JOUR]    Journal<br>  [LANG]    Language<br>  [MAJR]    MeSH Major Topic<br>  [SUBH]    MeSH Subheading<br>  [MESH]    MeSH Terms<br>  [PTYP]    Publication Type<br>  [WORD]    Text Word<br>  [TITL]    Title<br>  [TIAB]    Title/Abstract<br>  [UID]     UID</pre><p>and a qualified query looks like:</p><pre>  "Tager HS [AUTH] AND glucagon [TIAB]"</pre><p>Filters that limit search results to subsets of PubMed include:</p><pre>  humans [MESH]<br>  pharmacokinetics [MESH]<br>  chemically induced [SUBH]<br>  all child [FILT]<br>  english [FILT]<br>  freetext [FILT]<br>  has abstract [FILT]<br>  historical article [FILT]<br>  randomized controlled trial [FILT]<br>  clinical trial, phase ii [PTYP]<br>  review [PTYP]</pre><p>Sequence databases are indexed with a different set of search fields, including:</p><pre>  [ACCN]    Accession<br>  [ALL]     All Fields<br>  [AUTH]    Author<br>  [GPRJ]    BioProject<br>  [ECNO]    EC/RN Number<br>  [FKEY]    Feature key<br>  [FILT]    Filter<br>  [GENE]    Gene Name<br>  [JOUR]    Journal<br>  [KYWD]    Keyword<br>  [MLWT]    Molecular Weight<br>  [ORGN]    Organism<br>  [PACC]    Primary Accession<br>  [PROP]    Properties<br>  [PROT]    Protein Name<br>  [SQID]    SeqID String<br>  [SLEN]    Sequence Length<br>  [SUBS]    Substance Name<br>  [WORD]    Text Word<br>  [TITL]    Title<br>  [UID]     UID</pre><p>and a sample query in the protein database is:</p><pre>  "alcohol dehydrogenase [PROT] NOT (bacteria [ORGN] OR fungi [ORGN])"</pre><p>Additional examples of subset filters in sequence databases are:</p><pre>  mammalia [ORGN]<br>  mammalia [ORGN:noexp]<br>  cds [FKEY]<br>  lacz [GENE]<br>  beta galactosidase [PROT]<br>  protein snp [FILT]<br>  reviewed [FILT]<br>  country united kingdom glasgow [TEXT]<br>  biomol genomic [PROP]<br>  dbxref flybase [PROP]<br>  gbdiv phg [PROP]<br>  phylogenetic study [PROP]<br>  sequence from mitochondrion [PROP]<br>  src cultivar [PROP]<br>  srcdb refseq validated [PROP]<br>  150:200 [SLEN]</pre><p>(The calculated molecular weight (MLWT) field is only indexed for proteins (and structures), not nucleotides.)</p></div><div id="chapter6.Examining_Intermediate_Results"><h3>Examining Intermediate Results</h3><p>EDirect
 stores intermediate results on the Entrez history server. EDirect 
navigation functions produce a custom XML message with the relevant 
fields (database, web environment, query key, and record count) that can
 be read the next command in the pipeline.</p><p>The results of each 
step in a query can be examined to confirm expected behavior before 
adding the next step. The Count field in the ENTREZ_DIRECT object 
contains the number of records returned by the previous step. A good 
measure of query success is a reasonable (non-zero) count value. For 
example:</p><pre>  esearch -db protein -query "<a href="https://www.ncbi.nlm.nih.gov/protein/18410104/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_567004</a> [ACCN]" |<br>  elink -related |<br>  efilter -query "28000:30000 [MLWT]" |<br>  elink -target structure |<br>  efilter -query "0:2 [RESO]"</pre><p>produces:</p><pre>  &lt;ENTREZ_DIRECT&gt;<br>    &lt;Db&gt;structure&lt;/Db&gt;<br>    &lt;WebEnv&gt;NCID_1_545606712_172.16.22.25_5555_1348089299_358182861&lt;/WebEnv&gt;<br>    &lt;QueryKey&gt;7&lt;/QueryKey&gt;<br>    &lt;Count&gt;39&lt;/Count&gt;<br>    &lt;Step&gt;5&lt;/Step&gt;<br>  &lt;/ENTREZ_DIRECT&gt;</pre><p>with
 39 protein structures being within the specified molecular weight range
 and having the desired (X-ray crystallographic) atomic position 
resolution.</p><p>(The QueryKey value is 7 instead of 5 because each 
elink command obtains the record count by running a separate ESearch 
query immediately after the ELink operation.)</p></div><div id="chapter6.Combining_Independent_Queries"><h3>Combining Independent Queries</h3><p>Independent
 esearch, elink, and efilter operations can be performed and then 
combined at the end by using the history server's "#" convention to 
indicate query key numbers. (The steps to be combined must be in the 
same database.) Subsequent esearch commands can take a -db argument to 
override the database piped in from the previous step. (Piping the 
queries together is necessary for sharing the same history thread.) For 
example, the query:</p><pre>  esearch -db protein -query "amyloid* [PROT]" |<br>  elink -target pubmed |<br>  esearch -db gene -query "apo* [GENE]" |<br>  elink -target pubmed |<br>  esearch -query "(#3) AND (#6)" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element Id Title</pre><p>uses
 truncation searching (entering the beginning of a word followed by an 
asterisk) to return titles of papers with links to amyloid protein 
sequence and apolipoprotein gene records:</p><pre>  23962925    Genome analysis reveals insights into physiology and ...<br>  23959870    Low levels of copper disrupt brain amyloid-β homeostasis ...<br>  23371554    Genomic diversity and evolution of the head crest in the ...<br>  23251661    Novel genetic loci identified for the pathophysiology of ...<br>  ...</pre><p>The
 use of (#3) AND (#6) instead of (#2) AND (#4) above reflects the need 
for each elink command to execute a separate ESearch query, which 
increments the QueryKey, in order to obtain the record count. The -label
 argument can be used to get around this artifact. The label value is 
prefixed by a "#" symbol and placed in parentheses in the final search. 
Thus:</p><pre>  esearch -db structure -query "insulin [TITL]" |<br>  elink -target pubmed -label struc_cit |<br>  esearch -db protein -query "insulin [PROT]" |<br>  elink -target pubmed -label prot_cit |<br>  esearch -query "(#struc_cit) AND (#prot_cit)" |<br>  efetch -format uid</pre><p>will return:</p><pre>  15299880<br>  9235985<br>  9141131<br>  8421693<br>  ...</pre><p>without the need to keep track of the internal QueryKey values.</p></div></div><div id="chapter6.Structured_Data"><h2 id="_chapter6_Structured_Data_">Structured Data</h2><div id="chapter6.Advantages_of_XML_Format"><h3>Advantages of XML Format</h3><p>The
 ability to obtain Entrez records in structured XML format, and to 
easily extract the underlying data, allows the user to ask novel 
questions that are not addressed by existing analysis software.</p><p>The
 advantage of XML is that many pieces of information are in specific 
locations in a well-defined data hierarchy. Accessing individual units 
of data that are fielded by name, such as:</p><pre>  &lt;PubDate&gt;2013&lt;/PubDate&gt;<br>  &lt;Source&gt;PLoS One&lt;/Source&gt;<br>  &lt;Volume&gt;8&lt;/Volume&gt;<br>  &lt;Issue&gt;3&lt;/Issue&gt;<br>  &lt;Pages&gt;e58144&lt;/Pages&gt;</pre><p>requires
 matching the same general pattern, differing only by the element name. 
This is much simpler than parsing the units from a long, complex string:</p><pre>  1. PLoS One. 2013;8(3):e58144 ...</pre><p>The
 disadvantage of XML is that data extraction usually requires 
programming. But EDirect relies on the common pattern of XML value 
representation to provide a simplified approach to interpreting XML 
data.</p></div><div id="chapter6.Conversion_of_XML_Data_into_Tab"><h3>Conversion of XML Data into Tabular Form</h3><p>The
 xtract function uses command-line arguments to direct the selective 
conversion of XML data into a tab-delimited table. The -pattern argument
 divides the results into rows, while placement of data into columns is 
controlled by -element. A trivial example:</p><pre>  xtract -pattern ENTREZ_DIRECT -element Count</pre><p>will print the number of records in the current query.</p><p>Xtract
 provides control over data conversion with a divide-and-conquer 
strategy using separate arguments for element selection, path 
exploration, conditional processing, and report formatting.</p><p>Element
 selection finds every occurrence of each indicated item, printing 
values as they are encountered. Exploration control limits selection by 
context, presenting specified objects one at a time. Conditional 
processing filters by content, requiring presence (or absence) of a 
particular data value in order to continue. Finally, custom formatting 
can override the normal tabular layout of the default output.</p><p>The details and ramifications of this flexible approach are discussed in the remainder of this section.</p></div><div id="chapter6.Extraction_Arguments"><h3>Extraction Arguments</h3><p>Selection arguments (-element, -first, and -last) extract and print data values from the indicated element names:</p><pre>  -element Id -first Name Title</pre><p>Exploration
 arguments (-pattern, -group, -block, and -subset) limit data extraction
 to specified regions of the XML, visiting all relevant objects one at a
 time. This sets a context for data collection, eliminates the need to 
provide the full path to a data element, and uncouples the concept of 
"what to look for" from "where to find it":</p><pre>  -pattern DocumentSummary<br>    -block Author</pre><p>Each
 pattern can have multiple groups, each group can have multiple blocks, 
and each block can have multiple subsets. This design allows nested 
exploration of complex, hierarchical data to be controlled by a linear 
chain of command-line argument statements.</p><p>Conditional processing 
arguments restrict exploration statements by object name and value (-if 
and -unless) or item location (-position):</p><pre>  -if Source -equals "J Bacteriol"<br>  -position first</pre><p>These commands are issued immediately after an exploration argument.</p><p>Formatting
 arguments (-ret, -tab, -sep, -pfx, -sfx, and -def) allow extensive 
customization of the default row/column table presentation:</p><pre>  -pfx "\n[" -sfx "]\t" -sep " " -tab "" -ret "\n\n" -def "-"</pre><p>and apply to subsequent selection statements.</p><p>(The "\n" escape sequence indicates a line break, while "\t" specifies a tab character.)</p></div><div id="chapter6.XML_Document_Summaries"><h3>XML Document Summaries</h3><p>Entrez provides a document summary in structured XML format for every record. Piping a query to "efetch -format docsum":</p><pre>  esearch -db pubmed -query "Garber ED [AUTH] AND PNAS [JOUR]" |<br>  elink -related |<br>  efilter -query "mouse" |<br>  efetch -format docsum</pre><p>will generate an XML document summary set:</p><pre>  &lt;DocumentSummarySet status="OK"&gt;<br>    &lt;DbBuild&gt;Build150407-2207m.3&lt;/DbBuild&gt;<br>    &lt;DocumentSummary&gt;<br>      &lt;Id&gt;19650888&lt;/Id&gt;<br>      &lt;PubDate&gt;2009 Aug 3&lt;/PubDate&gt;<br>      &lt;EPubDate&gt;2009 Aug 3&lt;/EPubDate&gt;<br>      &lt;Source&gt;BMC Microbiol&lt;/Source&gt;<br>      &lt;Authors&gt;<br>        &lt;Author&gt;<br>          &lt;Name&gt;Cano V&lt;/Name&gt;<br>          &lt;AuthType&gt;Author&lt;/AuthType&gt;<br>          &lt;ClusterID&gt;&lt;/ClusterID&gt;<br>        &lt;/Author&gt;<br>        &lt;Author&gt;<br>          &lt;Name&gt;Moranta D&lt;/Name&gt;<br>          &lt;AuthType&gt;Author&lt;/AuthType&gt;<br>          ...</pre><p>Piping the document summary output to:</p><pre>  xtract -outline</pre><p>will give an indented overview of the XML structure hierarchy:</p><pre>  DbBuild<br>  DocumentSummary<br>    Id<br>    PubDate<br>    EPubDate<br>    Source<br>    Authors<br>      Author<br>        Name<br>        AuthType<br>        ClusterID<br>      Author<br>        Name<br>        ...</pre><p>The
 outline view presents a clear, uncluttered picture of the XML hierarchy
 that is useful in designing the appropriate command for actual data 
extraction. Copy and paste from the -outline output to xtract arguments 
can help avoid typographical errors. Thus:</p><pre>  esearch -db pubmed -query "Garber ED [AUTH] AND PNAS [JOUR]" |<br>  elink -related |<br>  efilter -query "mouse" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element Id SortFirstAuthor Title</pre><p>returns the PubMed identifier (PMID), first author name, and article title:</p><pre>  19650888    Cano V        Klebsiella pneumoniae triggers a cytotoxic ...<br>  19262028    Suto J        Metabolic consequence of congenital asplenia ...<br>  19248821    Fukumoto N    Hypoalgesic behaviors of P/Q-type voltage- ...<br>  18822497    Trishin AV    [Protective activity of secreted proteins of ...<br>  18582214    Singh A       Generation and characterization of monoclonal ...<br>  ...</pre><p>Using
 xtract "-synopsis" instead of -outline show the full path to each 
element. Piping those results to "sort-uniq-count" (see below) produces a
 table of unique path counts.</p></div><div id="chapter6.Processing_Results_with_UNIX_Ut"><h3>Processing Results with UNIX Utilities</h3><p>A tab-delimited table can be processed by many UNIX utilities. For example:</p><pre>  esearch -db pubmed -query "Garber ED [AUTH] AND PNAS [JOUR]" |<br>  elink -related |<br>  efilter -query "mouse" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element Id SortFirstAuthor Title |<br>  sort -t $'\t' -k 2,2f -k 3,3f</pre><p>sorts
 the results of the previous example by author name and then (if there 
are multiple publications by the same author) alphabetically by title:</p><pre>  17474906    Benghezal M     Inhibitors of bacterial virulence ...<br>  19650888    Cano V          Klebsiella pneumoniae triggers a cytotoxic ...<br>  17102561    Chatterjee S    How reliable are models for malaria vaccine ...<br>  17371870    Clements A      Secondary acylation of Klebsiella ...<br>  17142396    Fresno S        A second galacturonic acid transferase is ...<br>  16735743    Fresno S        The ionic interaction of Klebsiella ...<br>  ...</pre><p>Rather
 than always having to retype a series of common post-processing 
instructions, frequently used combinations of UNIX commands can be 
placed in a function, stored in an alias file (e.g., the user's 
.bash_profile), and executed by name. (The following two functions are 
now included as scripts with the EDirect software.) For example:</p><pre>  WordAtATime() {<br>    sed 's/[^a-zA-Z0-9]/ /g; s/^ *//' |<br>    tr 'A-Z' 'a-z' |<br>    fmt -w 1<br>  }<br>  alias word-at-a-time='WordAtATime'<br><br>  SortUniqCountRank() {<br>    sort -f |<br>    uniq -i -c |<br>    perl -pe 's/\s*(\d+)\s(.+)/$1\t$2/' |<br>    sort -t $'\t' -k 1,1nr -k 2f<br>  }<br>  alias sort-uniq-count-rank='SortUniqCountRank'</pre><p>Titles can be passed to a pair of these UNIX alias commands:</p><pre>  esearch -db pubmed -query "Casadaban transposition immunity" |<br>  elink -related |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element Title |<br>  word-at-a-time |<br>  sort-uniq-count-rank</pre><p>to generate a table of word occurrence counts, sorted by frequency:</p><pre>  296    of<br>  175    the<br>  114    transposition<br>  102    and<br>  94     in<br>  93     mu<br>  83     a<br>  61     dna<br>  61     tn3<br>  55     transposon<br>  ...</pre></div><div id="chapter6.Output_Format_Customization"><h3>Output Format Customization</h3><p>The
 line break between -pattern objects can be overridden with -ret, and 
the tab character between fields can be replaced by -tab.</p><p>The -sep
 argument is used to distinguish multiple elements of the same type and 
control their separation independently of the -tab argument. For 
example:</p><pre>  esearch -db gene -query "deuteranopia" |<br>  efetch -format xml |<br>  xtract -pattern Entrezgene \<br>    -element Gene-track_geneid Gene-ref_locus \<br>    -sep "|" -element Gene-ref_syn_E</pre><p>combines all synonyms for a gene into a single column, separated by vertical bars:</p><pre>  2652    OPN1MW    CBD|GCP|GOP|CBBM|COD5|OPN1MW1<br>  5956    OPN1LW    CBP|RCP|ROP|CBBM|COD5</pre><p>The
 -sep value also applies to unrelated -element items that are grouped 
with commas. Otherwise the -tab value delineates individual fields.</p><p>Groups or fields are preceded by the -pfx value and followed by the -sfx value, both of which are initially empty.</p><p>Missing data values can be marked by the -def argument. For example:</p><pre>  esearch -db pubmed -query "deuteranopia" |<br>  efetch -format xml |<br>  xtract -pattern PubmedArticle -def "-" \<br>    -first MedlineCitation/PMID Author/LastName Keyword</pre><p>inserts a dash in a column where the specified element is missing.</p></div><div id="chapter6.Pubmed_Article_XML_Records"><h3>Pubmed Article XML Records</h3><p>The PubmedArticle object (for -db pubmed) has a more detailed structure than the DocumentSummary:</p><pre>  esearch -db pubmed -query "tetrachromacy" |<br>  efetch -format xml |<br>  xtract -outline</pre><p>More information is fielded, including author names, dates, and the abstract:</p><pre>  PubmedArticle<br>    MedlineCitation<br>      PMID<br>      DateCompleted<br>        Year<br>        Month<br>        Day<br>      DateRevised<br>        Year<br>        Month<br>        Day<br>      Article<br>        Journal<br>          ISSN<br>          JournalIssue<br>            Volume<br>            Issue<br>            PubDate<br>              Year<br>              Month<br>              Day<br>          Title<br>          ISOAbbreviation<br>        ArticleTitle<br>        Pagination<br>          MedlinePgn<br>        Abstract<br>          AbstractText<br>          CopyrightInformation<br>        AuthorList<br>          Author<br>            LastName<br>            ForeName<br>            Initials<br>            AffiliationInfo<br>              Affiliation<br>          Author<br>            LastName<br>            ...</pre><p>Using this information to craft a new xtract statement:</p><pre>  esearch -db pubmed -query "tetrachromacy" |<br>  efetch -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID LastName</pre><p>results in a table of all authors for each record:</p><pre>  23393278    Sabbah       Troje        Gray        Hawryshyn<br>  20884587    Jordan       Deeb         Bosten      Mollon<br>  18230593    Koshitaka    Kinoshita    Vorobyev    Arikawa<br>  17685813    Wachtler     Doi          Lee         Sejnowski<br>  ...</pre><p>(Note
 that "-element MedlineCitation/PMID" uses the "Parent/Child" construct 
to prevent the display of additional PMID items that may occur later in 
CommentsCorrections objects.)</p><p>The -first or -last arguments can be used instead of -element, if appropriate.</p></div><div id="chapter6.Exploration_of_XML_Sets"><h3>Exploration of XML Sets</h3><p>Individual PubmedArticle objects can be retrieved directly by efetch:</p><pre>  efetch -db pubmed -id 20643751 -format xml</pre><p>The resulting XML has authors with separate fields for last name and initials:</p><pre>  ...<br>  &lt;AuthorList&gt;<br>    &lt;Author&gt;<br>      &lt;LastName&gt;Inamdar&lt;/LastName&gt;<br>      &lt;ForeName&gt;Arati A&lt;/ForeName&gt;<br>      &lt;Initials&gt;AA&lt;/Initials&gt;<br>    &lt;/Author&gt;<br>    &lt;Author&gt;<br>      &lt;LastName&gt;Masurekar&lt;/LastName&gt;<br>      &lt;ForeName&gt;Prakash&lt;/ForeName&gt;<br>      &lt;Initials&gt;P&lt;/Initials&gt;<br>    &lt;/Author&gt;<br>    &lt;Author&gt;<br>      &lt;LastName&gt;Bennett&lt;/LastName&gt;<br>      &lt;ForeName&gt;Joan Wennstrom&lt;/ForeName&gt;<br>      &lt;Initials&gt;JW&lt;/Initials&gt;<br>    &lt;/Author&gt;<br>  &lt;/AuthorList&gt;<br>  ...</pre><p>Without being given any guidance about context, an -element statement with "Initials" and "LastName" arguments:</p><pre>  efetch -db pubmed -id 1413997,6301692,781293 -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID \<br>    -element Initials LastName</pre><p>will
 explore the current record for each argument separately, and thus print
 all author initials followed by all author last names:</p><pre>  1413997    RK    CR           JS         Mortimer      Contopoulou    King<br>  6301692    MA    NR           Krasnow    Cozzarelli<br>  781293     MJ    Casadaban</pre><p>Inserting
 a -block statement redirects data exploration to consider each author 
one at a time. Subsequent -element statements only see the current 
author's values:</p><pre>  efetch -db pubmed -id 1413997,6301692,781293 -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID \<br>    -block Author -element Initials LastName</pre><p>which restores the correct association of initials and last name:</p><pre>  1413997    RK    Mortimer     CR    Contopoulou    JS    King<br>  6301692    MA    Krasnow      NR    Cozzarelli<br>  781293     MJ    Casadaban</pre><p>Adding
 a -sep statement to replace the normal tab between group members, and 
using a comma to combine the two arguments ("Initials,LastName") into a 
group:</p><pre>  efetch -db pubmed -id 1413997,6301692,781293 -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID \<br>    -block Author -sep " " -element Initials,LastName</pre><p>results in more desirable formatting of author names:</p><pre>  1413997    RK Mortimer     CR Contopoulou    JS King<br>  6301692    MA Krasnow      NR Cozzarelli<br>  781293     MJ Casadaban</pre><p>The first or last structured object can be selected by the -position statement:</p><pre>  efetch -db pubmed -id 1413997,6301692,781293 -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID \<br>    -block Author -position first -sep " " -element Initials,LastName</pre></div><div id="chapter6.Exploring_Separate_XML_Regions"><h3>Exploring Separate XML Regions</h3><p>Multiple
 -block statements can be used in a single xtract to explore different 
areas of the XML. This limits element extraction to the desired 
subregions, and allows disambiguation of fields with identical names.</p><p>Combining
 independent fields with commas allows them to be treated as sets. The 
tab that normally separates these can be replaced with a -sep argument:</p><pre>  efetch -db pubmed -id 6092233,4640931,4296474 -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID \<br>    -block AuthorList -sep "/" -element LastName "#Author" \<br>    -block PubDate -sep " " -element Year,Month MedlineDate |<br>  sort -t $'\t' -k 3,3n -k 2,2f</pre><p>This
 generates a table that allows easy parsing of author last names, counts
 the number of authors present, and prints the date each record was 
published, sorting the results by author count:</p><pre>  4296474    Friedmann                        1    1968 Apr<br>  4640931    Tager/Steiner                    2    1972 Dec<br>  6092233    Calderon/Contopoulou/Mortimer    3    1984 Jul-Aug</pre><p>(Note that the PubDate object can exist either in a structured form:</p><pre>  &lt;PubDate&gt;<br>    &lt;Year&gt;1968&lt;/Year&gt;<br>    &lt;Month&gt;Apr&lt;/Month&gt;<br>    &lt;Day&gt;25&lt;/Day&gt;<br>  &lt;/PubDate&gt;</pre><p>(with the Day field frequently absent), or in a string form:</p><pre>  &lt;PubDate&gt;<br>    &lt;MedlineDate&gt;1984 Jul-Aug&lt;/MedlineDate&gt;<br>  &lt;/PubDate&gt;</pre><p>but would not contain a mixture of both types, so the directive:</p><pre>  -element Year,Month MedlineDate</pre><p>will only contribute a single column to the output.)</p></div><div id="chapter6.Nested_Exploration_of_Subsets_W"><h3>Nested Exploration of Subsets Within XML Sets</h3><p>Medical Subject Headings (MeSH terms) in a record may be assigned subheadings:</p><pre>    ...<br>    &lt;MeshHeading&gt;<br>      &lt;DescriptorName&gt;RNA, Messenger&lt;/DescriptorName&gt;<br>      &lt;QualifierName&gt;genetics&lt;/QualifierName&gt;<br>    &lt;/MeshHeading&gt;<br>    &lt;MeshHeading&gt;<br>      &lt;DescriptorName&gt;Transcription, Genetic&lt;/DescriptorName&gt;<br>    &lt;/MeshHeading&gt;<br>    &lt;MeshHeading&gt;<br>      &lt;DescriptorName&gt;beta-Galactosidase&lt;/DescriptorName&gt;<br>      &lt;QualifierName&gt;genetics&lt;/QualifierName&gt;<br>      &lt;QualifierName&gt;metabolism&lt;/QualifierName&gt;<br>    &lt;/MeshHeading&gt;<br>  &lt;/MeshHeadingList&gt;<br>  ...</pre><p>Visiting
 each MeSH term with a -block statement, and adding a -subset statement 
within the -block, allows nested exploration of the subheadings for the 
current MeSH term:</p><pre>  efetch -db pubmed -id 6162838 -format xml |<br>  xtract -pattern PubmedArticle -tab "" -element MedlineCitation/PMID \<br>    -block MeshHeading -pfx "\n" -tab "" -element DescriptorName \<br>      -subset QualifierName -pfx " / " -tab "" -element QualifierName</pre><p>and creates a list of MeSH terms with associated subheadings:</p><pre>  6162838<br>  Base Sequence<br>  DNA, Recombinant<br>  Escherichia coli / genetics<br>  ...<br>  RNA, Messenger / genetics<br>  Transcription, Genetic<br>  beta-Galactosidase / genetics / metabolism</pre></div><div id="chapter6.Selection_of_Attributes"><h3>Selection of Attributes</h3><p>The MeSH term and subheading fields actually have major topic attributes:</p><pre>  ...<br>  &lt;MeshHeading&gt;<br>    &lt;DescriptorName MajorTopicYN="N"&gt;beta-Galactosidase&lt;/DescriptorName&gt;<br>    &lt;QualifierName MajorTopicYN="Y"&gt;genetics&lt;/QualifierName&gt;<br>    &lt;QualifierName MajorTopicYN="N"&gt;metabolism&lt;/QualifierName&gt;<br>  &lt;/MeshHeading&gt;<br>  ...</pre><p>that can be selected as "DescriptorName@MajorTopicYN" or "@MajorTopicYN":</p><pre>  efetch -db pubmed -id 6162838 -format xml |<br>  xtract -pattern PubmedArticle -tab "" -element MedlineCitation/PMID \<br>    -block MeshHeading -pfx "\n" -sep "|" -tab "" \<br>      -element DescriptorName@MajorTopicYN,DescriptorName \<br>      -subset QualifierName -pfx " / " -sep "|" -tab "" \<br>        -element "@MajorTopicYN,QualifierName"</pre><p>The major topic value is placed before each MeSH term or subheading:</p><pre>  6162838<br>  N|Base Sequence<br>  Y|DNA, Recombinant<br>  N|Escherichia coli / N|genetics<br>  ...<br>  N|RNA, Messenger / Y|genetics<br>  N|Transcription, Genetic<br>  N|beta-Galactosidase / Y|genetics / N|metabolism</pre><p>The results can be processed by the UNIX stream editor "sed":</p><pre>  sed -e 's/N|//g' -e 's/Y|/*/g'</pre><p>to display an asterisk for major ("starred" MeSH term) concepts:</p><pre>  6162838<br>  Base Sequence<br>  *DNA, Recombinant<br>  Escherichia coli / genetics<br>  ...<br>  RNA, Messenger / *genetics<br>  Transcription, Genetic<br>  beta-Galactosidase / *genetics / metabolism</pre></div><div id="chapter6.Recording_Values_in_Variables"><h3>Recording Values in Variables</h3><p>A
 value can be recorded in a variable and then displayed multiple times 
as needed. Variables are indicated by a hyphen followed by a string of 
capital letters or digits. The variable "-PMID" is referred to as 
"&amp;PMID" in an -element argument. For example:</p><pre>  efetch -db pubmed -id 1413997,6301692,781293 -format xml |<br>  xtract -pattern PubmedArticle -PMID MedlineCitation/PMID \<br>    -block Author -element "&amp;PMID" \<br>      -sep " " -tab "\n" -element Initials,LastName</pre><p>produces a list of authors, with the PMID in the first column of each row:</p><pre>  1413997    RK Mortimer<br>  1413997    CR Contopoulou<br>  1413997    JS King<br>  6301692    MA Krasnow<br>  6301692    NR Cozzarelli<br>  781293     MJ Casadaban</pre></div><div id="chapter6.Variable_Initialization"><h3>Variable Initialization</h3><p>Variables can be initialized with a literal value in parentheses:</p><pre>  efetch -db pubmed -id 1413997,6301692,781293 -format xml |<br>  xtract -pattern PubmedArticle -element MedlineCitation/PMID \<br>    -block Author -sep " " -tab "" \<br>      -element "&amp;COM" Initials,LastName -COM "(, )"</pre><p>This
 can be used as a placeholder to prevent missing data from shifting 
columns in a table, or to have additional control over output 
formatting:</p><pre>  1413997    RK Mortimer, CR Contopoulou, JS King<br>  6301692    MA Krasnow, NR Cozzarelli<br>  781293     MJ Casadaban</pre><p>All variables are reset when the next record is processed.</p></div><div id="chapter6.Conditional_Processing"><h3>Conditional Processing</h3><p>Xtract provides -if and -unless arguments that filter by element name or name plus data value. For example:</p><pre>  esearch -db pubmed -query "Cozzarelli NR [AUTH]" |<br>  efetch -format xml |<br>  xtract -pattern PubmedArticle -if "#Author" -eq 3 \<br>    -block Author -if LastName -is-not Cozzarelli \<br>      -sep ", " -tab "\n" -element LastName,Initials |<br>  sort | uniq</pre><p>will select papers with exactly 3 authors and print the coauthor names:</p><pre>  Ackerman, RS<br>  Adams, DE<br>  Alexandrov, AI<br>  Arimondo, PB<br>  Bauer, WR<br>  ...</pre><p>Multiple conditions are specified with -and and -or commands:</p><pre>  -if @score -equals 1 -or @score -starts-with 0.9</pre><p>The -else command can supply alternative -element or -lbl instructions to be run if the condition is not satisfied:</p><pre>  -if MapLocation -element MapLocation -else -lbl "\-"</pre><p>Parallel
 -if and -unless statements can be used to provide a more complex 
response to alternative conditions that includes nested exploration.</p></div></div><div id="chapter6.Sequence_Records"><h2 id="_chapter6_Sequence_Records_">Sequence Records</h2><div id="chapter6.NCBI_Data_Model_for_Sequence_Re"><h3>NCBI Data Model for Sequence Records</h3><p>The
 NCBI represents sequence records in a data model that is based on the 
central dogma of molecular biology. Sequences, including genomic DNA, 
messenger RNAs, and protein products, are "instantiated" with the actual
 sequence letters, and are assigned identifiers (e.g., accession 
numbers) for reference. Features carry information about the biology of a
 given region, with a location that refers to specific intervals on a 
particular sequence. Some features may also point to the product 
sequence of a particular transformation.</p><p><span class="graphic"><img src="chapter6-Image001.jpg" alt="Image chapter6-Image001.jpg"></span></p><p>A
 gene feature indicates the location of a heritable region of nucleic 
acid that confers a measurable phenotype. An mRNA feature on genomic DNA
 represents the exonic and untranslated regions of the message that 
remain after transcription and splicing. A coding region (CDS) feature 
has a product reference to the translated protein.</p><p>Since messenger
 RNA sequences are not always submitted with a genomic region, CDS 
features (which model the travel of ribosomes on transcript molecules) 
are traditionally annotated on the genomic sequence, with locations that
 encode the exonic intervals.</p><p>Features display specific biological
 annotation in qualifiers. For example, the name of a gene is shown in 
the /gene qualifier. A qualifier can be dynamically generated from 
underlying data for the convenience of the user. Thus, the sequence of a
 mature peptide may be extracted from the mat_peptide feature's location
 on the precursor protein and displayed in a /peptide qualifier, even if
 a mature peptide is not instantiated.</p></div><div id="chapter6.Sequence_Records_in_INSDSeq_XML"><h3>Sequence Records in INSDSeq XML</h3><p>Sequence records can be retrieved in an XML version of the GenBank or GenPept flatfile. The query:</p><pre>  efetch -db protein -id 26418308,26418074 -format gpc</pre><p>returns a set of INSDSeq objects:</p><pre>  &lt;INSDSet&gt;<br>    &lt;INSDSeq&gt;<br>      &lt;INSDSeq_locus&gt;<a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128</a>&lt;/INSDSeq_locus&gt;<br>      &lt;INSDSeq_length&gt;17&lt;/INSDSeq_length&gt;<br>      &lt;INSDSeq_moltype&gt;AA&lt;/INSDSeq_moltype&gt;<br>      &lt;INSDSeq_topology&gt;linear&lt;/INSDSeq_topology&gt;<br>      &lt;INSDSeq_division&gt;INV&lt;/INSDSeq_division&gt;<br>      &lt;INSDSeq_update-date&gt;03-JAN-2003&lt;/INSDSeq_update-date&gt;<br>      &lt;INSDSeq_create-date&gt;10-DEC-2002&lt;/INSDSeq_create-date&gt;<br>      &lt;INSDSeq_definition&gt;alpha-conotoxin ImI precursor, partial [Conus<br>         imperialis]&lt;/INSDSeq_definition&gt;<br>      &lt;INSDSeq_primary-accession&gt;<a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128</a>&lt;/INSDSeq_primary-accession&gt;<br>      &lt;INSDSeq_accession-version&gt;<a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128.1</a>&lt;/INSDSeq_accession-version&gt;<br>      &lt;INSDSeq_other-seqids&gt;<br>        &lt;INSDSeqid&gt;gb|<a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128.1</a>|&lt;/INSDSeqid&gt;<br>        &lt;INSDSeqid&gt;gi|26418308&lt;/INSDSeqid&gt;<br>      &lt;/INSDSeq_other-seqids&gt;<br>      &lt;INSDSeq_source&gt;Conus imperialis&lt;/INSDSeq_source&gt;<br>      &lt;INSDSeq_organism&gt;Conus imperialis&lt;/INSDSeq_organism&gt;<br>      &lt;INSDSeq_taxonomy&gt;Eukaryota; Metazoa; Lophotrochozoa; Mollusca;<br>         Gastropoda; Caenogastropoda; Hypsogastropoda; Neogastropoda;<br>         Conoidea; Conidae; Conus&lt;/INSDSeq_taxonomy&gt;<br>      &lt;INSDSeq_references&gt;<br>        &lt;INSDReference&gt;<br>        ...</pre><p>INSDSeq XML presents biological features and qualifiers (shown here in GenPept format):</p><pre>  FEATURES             Location/Qualifiers<br>       source          1..17<br>                       /organism="Conus imperialis"<br>                       /db_xref="taxon:35631"<br>                       /country="Philippines"<br>       Protein         &lt;1..17<br>                       /product="alpha-conotoxin ImI precursor"<br>       mat_peptide     5..16<br>                       /product="alpha-conotoxin ImI"<br>                       /note="the C-terminal glycine of the precursor is post<br>                       translationally removed"<br>                       /calculated_mol_wt=1357<br>                       /peptide="GCCSDPRCAWRC"<br>       CDS             1..17<br>                       /coded_by="<a href="https://www.ncbi.nlm.nih.gov/nuccore/26418307" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">AY159318.1</a>:&lt;1..54"<br>                       /note="nAChR antagonist"</pre><p>in a structured feature table:</p><pre>  ...<br>  &lt;INSDFeature&gt;<br>    &lt;INSDFeature_key&gt;mat_peptide&lt;/INSDFeature_key&gt;<br>    &lt;INSDFeature_location&gt;5..16&lt;/INSDFeature_location&gt;<br>    &lt;INSDFeature_intervals&gt;<br>      &lt;INSDInterval&gt;<br>        &lt;INSDInterval_from&gt;5&lt;/INSDInterval_from&gt;<br>        &lt;INSDInterval_to&gt;16&lt;/INSDInterval_to&gt;<br>        &lt;INSDInterval_accession&gt;<a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128.1</a>&lt;/INSDInterval_accession&gt;<br>      &lt;/INSDInterval&gt;<br>    &lt;/INSDFeature_intervals&gt;<br>    &lt;INSDFeature_quals&gt;<br>      &lt;INSDQualifier&gt;<br>        &lt;INSDQualifier_name&gt;product&lt;/INSDQualifier_name&gt;<br>        &lt;INSDQualifier_value&gt;alpha-conotoxin ImI&lt;/INSDQualifier_value&gt;<br>      &lt;/INSDQualifier&gt;<br>      &lt;INSDQualifier&gt;<br>        &lt;INSDQualifier_name&gt;note&lt;/INSDQualifier_name&gt;<br>        &lt;INSDQualifier_value&gt;the C-terminal glycine of the precursor is<br>           post translationally removed&lt;/INSDQualifier_value&gt;<br>      &lt;/INSDQualifier&gt;<br>      &lt;INSDQualifier&gt;<br>        &lt;INSDQualifier_name&gt;calculated_mol_wt&lt;/INSDQualifier_name&gt;<br>        &lt;INSDQualifier_value&gt;1357&lt;/INSDQualifier_value&gt;<br>      &lt;/INSDQualifier&gt;<br>      &lt;INSDQualifier&gt;<br>        &lt;INSDQualifier_name&gt;peptide&lt;/INSDQualifier_name&gt;<br>        &lt;INSDQualifier_value&gt;GCCSDPRCAWRC&lt;/INSDQualifier_value&gt;<br>      &lt;/INSDQualifier&gt;<br>    &lt;/INSDFeature_quals&gt;<br>  &lt;/INSDFeature&gt;<br>  ...</pre><p>Feature
 and qualifier names are indicated in data values, not XML element tags,
 and require -if and -equals to select the desired object and content. 
The xtract -insd argument simplifies this process, as shown below.</p></div><div id="chapter6.Generating_Qualifier_Extraction"><h3>Generating Qualifier Extraction Commands</h3><p>Because
 obtaining specific qualifier values from INSDSeq XML is somewhat more 
complex than previous cases, the xtract -insd argument can be used to 
generate extraction instructions.</p><p>Running xtract -insd in an 
isolated command prints a new xtract statement that can then be copied, 
edited if necessary, and pasted into other queries. Running the -insd 
command within a multi-step pipe dynamically executes the constructed 
query.</p><p>Providing an optional (complete/partial) location indication, a feature key, and then one or more qualifier names:</p><pre>  xtract -insd complete mat_peptide "%peptide" product peptide</pre><p>creates
 a new xtract statement that will produce a table of qualifier values 
from mature peptide features with complete locations. The statement 
starts with instructions to record the accession and find features of 
the indicated type:</p><pre>  xtract -pattern INSDSeq -ACCN INSDSeq_accession-version \<br>    -group INSDFeature -if INSDFeature_key -equals mat_peptide \<br>      -unless INSDFeature_partial5 -or INSDFeature_partial3 \<br>        -clr -pfx "\n" -element "&amp;ACCN"</pre><p>Each qualifier then generates custom extraction code that is appended to the growing query. For example:</p><pre>  -block INSDQualifier \<br>    -if INSDQualifier_name -equals peptide \<br>      -element INSDQualifier_value</pre><p>Incorporating the xtract -insd command in a query for marine snail venom peptides:</p><pre>  esearch -db pubmed -query "conotoxin" |<br>  elink -target protein |<br>  efilter -query "mat_peptide [FKEY]" |<br>  efetch -format gpc |<br>  xtract -insd complete mat_peptide "%peptide" product peptide</pre><p>produces a table with columns for accession number, calculated peptide length, product name, and peptide sequence:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/protein/514833168/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AGO59814.1</a>    32    del13b conotoxin       DCPTSCPTTCANGWECCKGYPCVRQHCSGCNH<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/28190704/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAO33169.1</a>    16    alpha-conotoxin GIC    GCCSHPACAGNNQHIC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/284022479/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB65788.1</a>    20    conotoxin Cal 16       LEMQGCVCNANAKFCCGEGR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128.1</a>    12    alpha-conotoxin ImI    GCCSDPRCAWRC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/6671346/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAF23167.1</a>    31    BeTX toxin             CRAEGTYCENDSQCCLNECCWGGCGHPCRHP<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/284022481/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB65789.1</a>    20    conotoxin Cal 16       LEMQGCVCNANAKFCCGEGR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/26420194/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78279.1</a>    21    conotoxin Vx-II        WIDPSHYCCCGGGCTDDCVNC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/158120313/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ABW16858.1</a>    15    marmophin              DWEYHAHPKPNSFWT<br>  ...</pre><p>Piping the results to a series of UNIX commands:</p><pre>  grep -i conotoxin |<br>  awk -F '\t' -v 'OFS=\t' '{if ( 10 &lt;= $2 &amp;&amp; $2 &lt;= 30 ) print}' |<br>  sort -t $'\t' -u -k 3,4 |<br>  sort -t $'\t' -k 2,2n -k 3,3f |<br>  cut -f 1,3- |<br>  column -s $'\t' -t</pre><p>filters
 by product name, limits the results to a specified range of peptide 
lengths, removes redundant accessions, sorts the table by peptide 
length, deletes the length column, and aligns the columns for cleaner 
printing:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/protein/26418308/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78128.1</a>  alpha-conotoxin ImI            GCCSDPRCAWRC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/26418074/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78127.1</a>  alpha-conotoxin ImII           ACCSDRRCRWRC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/283825360/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB43130.1</a>  conotoxin Cal 1a               KCCKRHHGCHPCGRK<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/283825362/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB43131.1</a>  conotoxin Cal 1b               LCCKRHHGCHPCGRT<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/28190704/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAO33169.1</a>  alpha-conotoxin GIC            GCCSHPACAGNNQHIC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/283825356/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB43128.1</a>  conotoxin Cal 5.1              DPAPCCQHPIETCCRR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/4885000/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAD31913.1</a>  alpha A conotoxin Tx2          PECCSHPACNVDHPEICR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/283825358/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB43129.1</a>  conotoxin Cal 5.2              MIQRSQCCAVKKNCCHVG<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/291360653/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADD97803.1</a>  conotoxin Cal 1.2              AGCCPTIMYKTGACRTNRCR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/284022481/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB65789.1</a>  conotoxin Cal 16               LEMQGCVCNANAKFCCGEGR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/4884998/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAD31912.1</a>  alpha A conotoxin Tx1          PECCSDPRCNSSHPELCGGRR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/26420194/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAN78279.1</a>  conotoxin Vx-II                WIDPSHYCCCGGGCTDDCVNC<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/283825350/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADB43125.1</a>  conotoxin Cal 14.2             GCPADCPNTCDSSNKCSPGFPG<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/291360651/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ADD97802.1</a>  conotoxin Cal 6.4              GCWLCLGPNACCRGSVCHDYCPR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/57898661/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">CAH64846.1</a>  four-loop conotoxin            CRPSGSPCGVTSICCGRCSRGKCT<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/4885004/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAD31915.1</a>  O-superfamily conotoxin TxO2   CYDSGTSCNTGNQCCSGWCIFVCL<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/4885006/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAD31916.1</a>  O-superfamily conotoxin TxO3   CYDGGTSCDSGIQCCSGWCIFVCF<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/4885014/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAD31920.1</a>  omega conotoxin SVIA mutant 1  CRPSGSPCGVTSICCGRCYRGKCT<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/4885016/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAD31921.1</a>  omega conotoxin SVIA mutant 2  CRPSGSPCGVTSICCGRCSRGKCT<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/91177676/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ABE27010.1</a>  conotoxin fe14.1               SPGSTICKMACRTGNGHKYPFCNCR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/91177678/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ABE27011.1</a>  conotoxin fe14.2               SSGSTVCKMMCRLGYGHLYPSCGCR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/91177670/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ABE27007.1</a>  conotoxin p114.1               GPGSAICNMACRLGQGHMYPFCNCN<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/91177672/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ABE27008.1</a>  conotoxin p114.2               GPGSAICNMACRLEHGHLYPFCHCR<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/91177674/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ABE27009.1</a>  conotoxin p114.3               GPGSAICNMACRLEHGHLYPFCNCD<br>  ...</pre><p>For records where a particular qualifier is missing:</p><pre>  esearch -db protein -query "RAG1 [GENE] AND Mus musculus [ORGN]" |<br>  efetch -format gpc |<br>  xtract -insd source organism strain |<br>  sort -t $'\t' -u -k 2,3</pre><p>a dash is inserted as a placeholder:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/protein/341941803/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">P15919.2</a>          Mus musculus               -<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/28975331/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">AAO61776.1</a>        Mus musculus               129/Sv<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/133778933/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_033045.2</a>       Mus musculus               C57BL/6<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/568915977/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">XP_006499075.1</a>    Mus musculus               C57BL/6J<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/148695708/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">EDL27655.1</a>        Mus musculus               mixed<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/55415895/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">BAD69530.1</a>        Mus musculus castaneus     -<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/55415897/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">BAD69531.1</a>        Mus musculus domesticus    BALB/c<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/55415899/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">BAD69532.1</a>        Mus musculus molossinus    MOA</pre></div></div><div id="chapter6.Sequence_Coordinates"><h2 id="_chapter6_Sequence_Coordinates_">Sequence Coordinates</h2><div id="chapter6.Gene_Positions"><h3>Gene Positions</h3><p>An
 understanding of sequence coordinate conventions is necessary in order 
to use gene positions to retrieve the corresponding chromosome subregion
 with efetch or with the UCSC browser.</p><p>Sequence records displayed 
in GenBank or GenPept formats use a "one-based" coordinate system, with 
sequence position numbers starting at "1":</p><pre>    1 catgccattc gttgagttgg aaacaaactt gccggctagc cgcatacccg cggggctgga<br>   61 gaaccggctg tgtgcggcca cagccaccat cctggacaaa cccgaagacg tgagtgaggg<br>  121 tcggcgagaa cttgtgggct agggtcggac ctcccaatga cccgttccca tccccaggga<br>  181 ccccactccc ctggtaacct ctgaccttcc gtgtcctatc ctcccttcct agatcccttc<br>  ...</pre><p>Under this convention, positions refer to the sequence letters themselves:</p><pre>  C   A   T   G   C   C   A   T   T   C<br>  1   2   3   4   5   6   7   8   9  10</pre><p>and
 the position of the last base or residue is equal to the length of the 
sequence. The ATG initiation codon above is at positions 2 through 4, 
inclusive.</p><p>For computer programs, however, using "zero-based" 
coordinates can simplify the arithmetic used for calculations on 
sequence positions. The ATG codon in the 0-based representation is at 
positions 1 through 3. (The UCSC browser uses a hybrid, half-open 
representation, where the start position is 0-based and the stop 
position is 1-based.)</p><p>Software at NCBI will typically convert 
positions to 0-based coordinates upon input, perform whatever 
calculations are desired, and then convert the results to a 1-based 
representation for display. These transformations are done by simply 
subtracting 1 from the 1-based value or adding 1 to the 0-based value.</p></div><div id="chapter6.Coordinate_Conversions"><h3>Coordinate Conversions</h3><p>Retrieving the docsum for a particular gene:</p><pre>  esearch -db gene -query "BRCA2 [GENE] AND human [ORGN]" |<br>  efetch -format docsum</pre><p>returns the chromosomal position of that gene in 0-based coordinates:</p><pre>  ...<br>  &lt;GenomicInfoType&gt;<br>    &lt;ChrLoc&gt;13&lt;/ChrLoc&gt;<br>    &lt;ChrAccVer&gt;<a href="https://www.ncbi.nlm.nih.gov/nuccore/568815585" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000013.11</a>&lt;/ChrAccVer&gt;<br>    &lt;ChrStart&gt;32315479&lt;/ChrStart&gt;<br>    &lt;ChrStop&gt;32399671&lt;/ChrStop&gt;<br>    &lt;ExonCount&gt;27&lt;/ExonCount&gt;<br>  &lt;/GenomicInfoType&gt;<br>  ...</pre><p>Piping the document summary to an xtract command:</p><pre>  xtract -pattern GenomicInfoType -element ChrAccVer ChrStart ChrStop</pre><p>obtains the accession and 0-based coordinate values:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815585" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000013.11</a>    32315479    32399671</pre><p>EFetch
 has -seq_start and -seq_stop arguments to retrieve a gene segment, but 
these expect the sequence subrange to be in 1-based coordinates.</p><p>To address this problem, two additional efetch arguments, -chr_start and -chr_stop, allow direct use of the 0-based coordinates:</p><pre>  efetch -db nuccore -format gb -id <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815585" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000013.11</a> \<br>    -chr_start 32315479 -chr_stop 32399671</pre><p>and eliminate the need for writing a UNIX shell command to increment the two values.</p><p>Xtract has numeric extraction commands to assist with coordinate conversion. Selecting fields with an -inc argument:</p><pre>  xtract -pattern GenomicInfoType -element ChrAccVer -inc ChrStart ChrStop</pre><p>obtains the accession and 0-based coordinates, then increments the positions to produce 1-based values:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815585" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000013.11</a>    32315480    32399672</pre><p>EDirect
 knows the policies for sequence positions in all relevant Entrez 
databases (e.g., gene, snp, dbvar), and provides additional shortcuts 
for converting these to other conventions. For example:</p><pre>  xtract -pattern GenomicInfoType -element ChrAccVer -1-based ChrStart ChrStop</pre><p>understands
 that gene ChrStart and ChrStop fields are 0-based, sees that the 
desired output is 1-based, and translates the command to convert 
coordinates using the -inc argument. Similarly:</p><pre>  -element ChrAccVer -ucsc-based ChrStart ChrStop</pre><p>leaves
 the 0-based start value unchanged but increments the original stop 
value to produce the half-open form that can be passed to the UCSC 
browser:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815585" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000013.11</a>    32315479    32399672</pre></div></div><div id="chapter6.Complex_Objects"><h2 id="_chapter6_Complex_Objects_">Complex Objects</h2><div id="chapter6.Heterogeneous_Data"><h3>Heterogeneous Data</h3><p>XML objects can contain a heterogeneous mix of components. For example:</p><pre>  efetch -db pubmed -id 21433338,17247418 -format xml</pre><p>returns a mixture of book and journal records:</p><pre>  &lt;PubmedArticleSet&gt;<br>    &lt;PubmedBookArticle&gt;<br>      &lt;BookDocument&gt;<br>      ...<br>      &lt;/PubmedBookData&gt;<br>    &lt;/PubmedBookArticle&gt;<br>    &lt;PubmedArticle&gt;<br>      &lt;MedlineCitation&gt;<br>      ...<br>      &lt;/PubmedData&gt;<br>    &lt;/PubmedArticle&gt;<br>  &lt;/PubmedArticleSet&gt;</pre><p>The
 "Parent/*" construct is used to visit the individual components, even 
though they may have different names. Piping the XML output to:</p><pre>  xtract -pattern "PubmedArticleSet/*" -element "*"</pre><p>separately prints the entirety of each XML component:</p><pre>  &lt;PubmedBookArticle&gt;&lt;BookDocument&gt; ... &lt;/PubmedBookData&gt;&lt;/PubmedBookArticle&gt;<br>  &lt;PubmedArticle&gt;&lt;MedlineCitation&gt; ... &lt;/PubmedData&gt;&lt;/PubmedArticle&gt;</pre><p>Use
 of the "Parent/Child" construct can isolate objects of the same name 
that differ by their location in the XML hierarchy. For example:</p><pre>  efetch -db pubmed -id 21433338,17247418 -format xml |<br>  xtract -pattern "PubmedArticleSet/*" \<br>    -group "BookDocument/AuthorList" -tab "\n" -element LastName \<br>    -group "Book/AuthorList" -tab "\n" -element LastName \<br>    -group "Article/AuthorList" -tab "\n" -element LastName</pre><p>writes separate lines for book/chapter authors, book editors, and article authors:</p><pre>  Fauci        Desrosiers<br>  Coffin       Hughes        Varmus<br>  Lederberg    Cavalli       Lederberg</pre><p>Simply exploring with individual arguments:</p><pre>  -group BookDocument -block AuthorList -element LastName</pre><p>would
 visit the editors (at BookDocument/Book/AuthorList) as well as the 
authors (at BookDocument/AuthorList), and print names in order of 
appearance in the XML:</p><pre>  Coffin    Hughes    Varmus    Fauci    Desrosiers</pre><p>(In
 this particular example the book author lists could be distinguished by
 using -if "@Type" -equals authors or -if "@Type" -equals editors, but 
exploring by "Parent/Child" is a general position-based approach.)</p></div><div id="chapter6.Recursive_Definitions"><h3>Recursive Definitions</h3><p>Certain
 XML objects returned by efetch are recursively defined, including Taxon
 in TaxaSet (-db taxonomy) and Gene-commentary in Entrezgene (-db gene).
 Thus, they can have nested objects with the same XML tag.</p><p>Retrieving a set of taxonomy records:</p><pre>  efetch -db taxonomy -id 9606,7227 -format xml</pre><p>produces XML with nested Taxon objects (marked below with line references) for each rank in the taxonomic lineage:</p><pre>    &lt;TaxaSet&gt;<br>1     &lt;Taxon&gt;<br>        &lt;TaxId&gt;9606&lt;/TaxId&gt;<br>        &lt;ScientificName&gt;Homo sapiens&lt;/ScientificName&gt;<br>        ...<br>        &lt;LineageEx&gt;<br>2         &lt;Taxon&gt;<br>            &lt;TaxId&gt;131567&lt;/TaxId&gt;<br>            &lt;ScientificName&gt;cellular organisms&lt;/ScientificName&gt;<br>            &lt;Rank&gt;no rank&lt;/Rank&gt;<br>3         &lt;/Taxon&gt;<br>4         &lt;Taxon&gt;<br>            &lt;TaxId&gt;2759&lt;/TaxId&gt;<br>            &lt;ScientificName&gt;Eukaryota&lt;/ScientificName&gt;<br>            &lt;Rank&gt;superkingdom&lt;/Rank&gt;<br>5         &lt;/Taxon&gt;<br>          ...<br>        &lt;/LineageEx&gt;<br>        ...<br>6     &lt;/Taxon&gt;<br>7     &lt;Taxon&gt;<br>        &lt;TaxId&gt;7227&lt;/TaxId&gt;<br>        &lt;ScientificName&gt;Drosophila melanogaster&lt;/ScientificName&gt;<br>        ...<br>8     &lt;/Taxon&gt;<br>    &lt;/TaxaSet&gt;</pre><p>Xtract
 tracks XML object nesting to determine that the &lt;Taxon&gt; start tag
 on line 1 is actually closed by the &lt;/Taxon&gt; stop tag on line 6, 
and not by the first &lt;/Taxon&gt; encountered on line 3.</p><p>When a recursive object is given to an exploration command, selection of data using the -element command:</p><pre>  efetch -db taxonomy -id 9606,7227,10090 -format xml |<br>  xtract -pattern Taxon \<br>    -element TaxId ScientificName GenbankCommonName Division</pre><p>does not examine fields in the internal objects, and returns information only for the main entries:</p><pre>  9606     Homo sapiens               human          Primates<br>  7227     Drosophila melanogaster    fruit fly      Invertebrates<br>  10090    Mus musculus               house mouse    Rodents</pre><p>The "*/Child" construct will skip past the outer start tag:</p><pre>  efetch -db taxonomy -id 9606,7227,10090 -format xml |<br>  xtract -pattern Taxon -block "*/Taxon" \<br>    -tab "\n" -element TaxId,ScientificName</pre><p>to visit the next level of nested objects individually:</p><pre>  131567    cellular organisms<br>  2759      Eukaryota<br>  33154     Opisthokonta<br>  ...</pre><p>Recursive objects can be fully explored with a double-star-slash prefix:</p><pre>  esearch -db gene -query "DMD [GENE] AND human [ORGN]" |<br>  efetch -format xml |<br>  xtract -pattern Entrezgene -block "**/Gene-commentary" \<br>    -tab "\n" -element Gene-commentary_type@value,Gene-commentary_accession</pre><p>which visits every child object regardless of nesting depth:</p><pre>  genomic    <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023</a><br>  mRNA       <a href="https://www.ncbi.nlm.nih.gov/nuccore/1034673705" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">XM_006724469</a><br>  peptide    <a href="https://www.ncbi.nlm.nih.gov/protein/578837832/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">XP_006724532</a><br>  mRNA       <a href="https://www.ncbi.nlm.nih.gov/nuccore/768032058" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">XM_011545467</a><br>  peptide    <a href="https://www.ncbi.nlm.nih.gov/protein/768032059/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">XP_011543769</a><br>  ...</pre></div></div><div id="chapter6.Advanced_Topics"><h2 id="_chapter6_Advanced_Topics_">Advanced Topics</h2><div id="chapter6.Storing_Common_Phrases_in_Alias"><h3>Storing Common Phrases in Alias Files</h3><p>Long
 or complicated search phrases can be saved in a file to avoid having to
 retype (or copy and paste) the full text for each query. Each line of 
the file has a shortcut keyword, a tab character, and the expanded 
search term. Shortcuts are referenced by placing them in parentheses 
after prefixing with a pound ("#") sign.</p><p>For example, given a file named "q_aliases" containing:</p><pre>  jour_filt    [MULT] AND ncbijournals [FILT]<br>  trans_imm    (transposition OR target) immunity</pre><p>the esearch line in:</p><pre>  esearch -alias q_aliases -db nlmcatalog -query "Science (#jour_filt)" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element ISOAbbreviation \<br>    -subset ISSNInfo -sep "|" -element issn,issntype</pre><p>will be expanded to:</p><pre>  esearch -db nlmcatalog -query "Science [MULT] AND ncbijournals [FILT]"</pre><p>with the query producing:</p><pre>  J. Zhejiang Univ. Sci.    1009-3095|Print    1009-3095|Linking<br>  Science (80- )            0193-4511|Print    0193-4511|Linking<br>  Science                   0036-8075|Print    1095-9203|Electronic    ...</pre><p>An alias file can also be read in a separate instruction at the beginning of a pipeline or script:</p><pre>  eproxy -alias q_aliases</pre><p>For
 maximum flexibility, separate eproxy commands can be piped together to 
load multiple shortcut files, as long as the shortcut strings are all 
unique.</p></div><div id="chapter6.Additional_EDirect_Options"><h3>Additional EDirect Options</h3><p>ESearch and EFilter can be given a -sort argument to specify the order of results when the records are retrieved:</p><pre>  esearch -db pubmed -query "opsin gene conversion" -sort "last author" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element Id LastAuthor PubDate Title</pre><p>ELink
 can return links to the citation list using "-name 
pubmed_pubmed_citedin", but only for publications with full text 
deposited in PubMed Central (PMC). For example, the query:</p><pre>  esearch -db pubmed -query "Beadle GW [AUTH]" |<br>  elink -related -name pubmed_pubmed_citedin |<br>  efetch -format docsum |<br>  xtract -pattern Author -element Name |<br>  sort-uniq-count-rank |<br>  head -n 10</pre><p>produces a ranked list of the ten most cited authors:</p><pre>  13    Beadle GW<br>  8     Ephrussi B<br>  8     Glass NL<br>  7     Hawley RS<br>  7     Mitchell MB<br>  7     PERKINS DD<br>  7     Tatum EL<br>  6     Mitchell HK<br>  6     YANOFSKY C<br>  5     Langley CH</pre><p>Similarly, "-name pubmed_pubmed_refs" returns an article's reference list, again for publications deposited in PMC.</p><p>ELink
 has several command modes, and these can be specified with the -cmd 
argument. When not using the default "neighbor_history" command, elink 
will return an eLinkResult XML object, with the links for each UID 
presented in separate blocks. For example:</p><pre>  esearch -db pubmed -query "Hoffmann PC [AUTH] AND dopamine [MAJR]" |<br>  elink -related -cmd neighbor |<br>  xtract -pattern LinkSetDb -element Id</pre><p>will show the original PMID in the first column and related article PMIDs in subsequent columns:</p><pre>  1504781    11754494    3815119    1684029    14614914    12128255    ...<br>  1684029    3815119     1504781    8097798    17161385    14755628    ...<br>  2572612    2903614     6152036    2905789    9483560     1352865     ...<br>  ...</pre><p>When
 the elink command "prlinks" is used with "ref" mode, it can obtain HTML
 containing or referencing full text articles directly from the 
publishers. The UNIX "xargs" command calls elink separately for each 
identifier:</p><pre>  epost -db pubmed -id 22966225,19880848 |<br>  efilter -query "freetext [FILT]" |<br>  efetch -format uid |<br>  xargs -n 1 elink -db pubmed -cmd prlinks -mode ref -http get -id</pre><p>The elink -batch flag will bypass the Entrez history mechanism for large queries.</p></div><div id="chapter6.Xtract_Special_Topics"><h3>Xtract Special Topics</h3><p>Self-closing tags of the standard form:</p><pre>  &lt;Na-strand/&gt;</pre><p>or alternative form:</p><pre>  &lt;Na-strand&gt;&lt;/Na-strand&gt;</pre><p>have no text content and thus cannot be selected with an -element command. If the tag contains an attribute:</p><pre>  &lt;Seq-interval_strand&gt;<br>    &lt;Na-strand value="plus"/&gt;<br>  &lt;/Seq-interval_strand&gt;</pre><p>it can be selected by matching on the specified value:</p><pre>  -group Seq-interval_strand \<br>    -block Seq-interval_strand -if Na-strand@value -equals plus -lbl "+" \<br>    -block Seq-interval_strand -if Na-strand@value -equals minus -lbl "-"</pre><p>The
 -pattern, -group, -block, and -subset commands provide a nested 
hierarchy of loop organizers for exploration of XML objects. Each 
pattern can contain multiple groups, each group can encompass multiple 
blocks, and each block can have multiple subsets.</p><p>Use of different
 argument names allows a linear representation of loop nesting, and 
provides sufficient flexibility to identify and extract arbitrary data 
from XML records in Entrez.</p><p>Sketching in pseudo code can clarify relative nesting levels. The extraction command:</p><pre>  xtract -pattern PubmedArticle \<br>    -block Author -element Initials,LastName \<br>    -block MeshHeading \<br>      -if QualifierName \<br>        -element DescriptorName \<br>        -subset QualifierName -element QualifierName</pre><p>could be represented as a computer program in pseudo code by:</p><pre>  for each Pubmed record {<br>    for each Author {<br>      print Initials LastName<br>    }<br>    for each MeSH term {<br>      if Subheadings are present {<br>        print Term Name<br>        for each Subheading {<br>          print Subheading Name<br>        }<br>      }<br>    }<br>  }</pre><p>Extra
 arguments (-division, -branch, -section, and -unit) are held in reserve
 to provide additional levels of organization, should the need arise in 
the future for processing complex, deeply-nested XML data. The full set 
of commands, in order of rank, are:</p><pre>  -pattern<br>  -division<br>  -group<br>  -branch<br>  -block<br>  -section<br>  -subset<br>  -unit</pre><p>Starting
 xtract exploration with -block, and expanding with -group and -subset, 
leaves additional level names that can be used wherever needed without 
having to redesign the entire command.</p></div><div id="chapter6.Querying_External_Web_Services"><h3>Querying External Web Services</h3><p>The
 EDirect nquire function can be used to obtain data from an arbitrary 
URL. Queries are built up from command-line arguments. For example:</p><pre>  nquire -url "https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi" \<br>    -db pubmed -term insulin</pre><p>reads the URL and then tag/value pairs to generate an E-utilities query:</p><pre>  https://eutils. ... .gov/entrez/eutils/esearch.fcgi?db=pubmed&amp;term=insulin</pre><p>Paths can be separated into components, which are combined with slashes, so:</p><pre>  -url https://eutils.ncbi.nlm.nih.gov entrez/eutils efetch.fcgi</pre><p>is converted to:</p><pre>  https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi</pre><p>Multiple values between tags are combined with commas. Thus:</p><pre>  -db nuccore -id <a href="https://www.ncbi.nlm.nih.gov/nuccore/1322283" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">U54469</a> <a href="https://www.ncbi.nlm.nih.gov/nuccore/42672" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">V00328</a> -rettype fasta</pre><p>is transformed into:</p><pre>  db=nuccore&amp;id=<a href="https://www.ncbi.nlm.nih.gov/nuccore/1322283" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">U54469</a>,<a href="https://www.ncbi.nlm.nih.gov/nuccore/42672" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">V00328</a>&amp;rettype=fasta</pre><p>A value that starts with a hyphen (or minus sign) can be distinguished from a tag by prefixing it with a backslash, so:</p><pre>  nquire -url http://api.geonames.org/countryCode -lat 41.796 -lng "\-87.577"</pre><p>will be sent as:</p><pre>  http://api.geonames.org/countryCode?lat=41.796&amp;lng=-87.577</pre><p>and will return "US" for coordinates within Chicago, which has a negative (western hemisphere) longitude value.</p><p>The -alias argument can read a file of shortcut keywords and URL aliases. The following aliases are always available:</p><pre>  ncbi_url      https://www.ncbi.nlm.nih.gov<br>  eutils_url    https://eutils.ncbi.nlm.nih.gov/entrez/eutils</pre><p>so the command:</p><pre>  nquire -url "(#eutils_url)" esearch.fcgi \<br>    -db gds -term "<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE22309" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=geo" target="_blank">GSE22309</a> [ACCN] AND gse [ETYP]" -retmax 200</pre><p>will run an ESearch query and return an eSearchResult XML object.</p><p>Raw
 XML with inconsistent line-wrapping and indentation can be reformatted 
for easier visual inspection of the data structure and content by piping
 it through:</p><pre>  xtract -format</pre></div></div><div id="chapter6.Automation"><h2 id="_chapter6_Automation_">Automation</h2><div id="chapter6.Entrez_Direct_Commands_Within_S"><h3>Entrez Direct Commands Within Scripts</h3><p>Taking
 an adventurous plunge into the world of programming, a shell script can
 be written when each output line of one step needs to be processed 
independently, instead of output being piped in its entirety to the next
 command. (The simplest shell script is merely a copy of a set of 
commands that are typed into the terminal for execution.)</p><p>In 
scripts, variables can be set to the results of a command by enclosing 
the statements in backtick ("`") characters. The variable name is 
prefixed by a dollar sign ("$") to use its value as an argument in 
another command. Comments start with a pound sign ("#") and are ignored.
 Quotation marks within quoted strings are entered by "escaping" with a 
backslash ("\"). Subroutines can be used to collect common code or 
simplify the organization of the script.</p><p>For example, executing a script file containing:</p><pre>  #!/bin/bash -norc<br><br>  parse_fields() {<br>    echo "$1" |<br>    xtract -pattern Field \<br>      -pfx "[" -sfx "]" -element Name \<br>      -pfx "" -sfx "" -element FullName Description |<br>    sort -t $'\t' -k 2,2f | column -s $'\t' -t<br>  }<br><br>  dbs=`einfo -dbs | sort`<br><br>  for db in $dbs<br>  do<br>    eix=`einfo -db $db`<br>    flds=`parse_fields "$eix"`<br><br>    echo "$db"<br>    echo ""<br>    echo "$flds"<br>    echo ""<br><br>    sleep 1<br>  done</pre><p>will obtain the list of Entrez databases:</p><pre>  annotinfo<br>  assembly<br>  bioproject<br>  ...</pre><p>and then return the abbreviations, names, and descriptions of indexed search fields, for each individual database:</p><pre>  ...<br>  mesh<br><br>  [ALL]   All Fields         All terms from all searchable fields<br>  [FILT]  Filter             Limits the records<br>  [MESH]  MeSH Terms         MeSH Terms<br>  [MHUI]  MeSH Unique ID     NLM MeSH Browser Unique ID<br>  [MULT]  Multi              Multi<br>  [PREV]  Previous Indexing  Previous Indexing<br>  [TYPE]  Record Type        Record type<br>  [REG]   Registry Number    Registry Number<br>  [NOTE]  Scope Note         Scope Note<br>  [ALSO]  See Also           See Also<br>  [SUBS]  Substance Name     Substance Name<br>  [WORD]  Text Word          Free text<br>  [TN]    Tree Number        Tree Number<br>  [UID]   UID                Unique number assigned to publication<br>  ...</pre><p>The shell script command:</p><pre>  sleep 1</pre><p>adds
 a one second delay between steps in a loop, and can be used to help 
prevent overuse of the Entrez servers by advanced scripts.</p></div><div id="chapter6.XargsSh_Loop"><h3>Xargs/Sh Loop</h3><p>Writing
 a script to loop through data can sometimes be avoided by creative use 
of the UNIX xargs and sh commands. Within the "sh -c" command string, 
the last name and initials arguments (passed in pairs by "xargs -n 2") 
are substituted at the "$0" and "$1" variables. All of the commands in 
the sh string are run separately on each name:</p><pre>  echo "Garber ED Casadaban MJ Mortimer RK" |<br>  xargs -n 2 sh -c 'esearch -db pubmed -query "$0 $1 [AUTH]" |<br>  xtract -pattern ENTREZ_DIRECT -lbl "$1 $0" -element Count'</pre><p>This produces PubMed article counts for each author:</p><pre>  ED Garber       35<br>  MJ Casadaban    46<br>  RK Mortimer     85</pre></div><div id="chapter6.While_Loop"><h3>While Loop</h3><p>A
 "while" loop can also be used to independently process lines of data. 
Given a file "organisms.txt" containing genus-species names, the UNIX 
"cat" command:</p><pre>  cat organisms.txt</pre><p>writes the contents of the file:</p><pre>  Arabidopsis thaliana<br>  Caenorhabditis elegans<br>  Danio rerio<br>  Drosophila melanogaster<br>  Escherichia coli<br>  Homo sapiens<br>  Mus musculus<br>  Saccharomyces cerevisiae</pre><p>This can be piped to a loop that reads one line at a time:</p><pre>  while read org<br>  do<br>    esearch -db taxonomy -query "$org [LNGE] AND family [RANK]" &lt; /dev/null |<br>    efetch -format docsum |<br>    xtract -pattern DocumentSummary -lbl "$org" \<br>      -element ScientificName Division<br>  done</pre><p>looking up the taxonomic family name and BLAST division for each organism:</p><pre>  Arabidopsis thaliana        Brassicaceae          eudicots<br>  Caenorhabditis elegans      Rhabditidae           nematodes<br>  Danio rerio                 Cyprinidae            bony fishes<br>  Drosophila melanogaster     Drosophilidae         flies<br>  Escherichia coli            Enterobacteriaceae    enterobacteria<br>  Homo sapiens                Hominidae             primates<br>  Mus musculus                Muridae               rodents<br>  Saccharomyces cerevisiae    Saccharomycetaceae    ascomycetes</pre><p>(The "&lt; /dev/null" input redirection construct prevents esearch from "draining" the remaining lines from stdin.)</p></div><div id="chapter6.For_Loop"><h3>For Loop</h3><p>The same results can be obtained with organism names embedded in a "for" loop:</p><pre>  for org in \<br>    "Arabidopsis thaliana" \<br>    "Caenorhabditis elegans" \<br>    "Danio rerio" \<br>    "Drosophila melanogaster" \<br>    "Escherichia coli" \<br>    "Homo sapiens" \<br>    "Mus musculus" \<br>    "Saccharomyces cerevisiae"<br>  do<br>    esearch -db taxonomy -query "$org [LNGE] AND family [RANK]" |<br>    efetch -format docsum |<br>    xtract -pattern DocumentSummary -lbl "$org" \<br>      -element ScientificName Division<br>  done</pre></div><div id="chapter6.File_Exploration"><h3>File Exploration</h3><p>A for loop can also be used to explore the computer's file system:</p><pre>  for i in *<br>  do<br>    if [ -f "$i" ]<br>    then<br>      echo $(basename "$i")<br>    fi<br>  done</pre><p>visiting
 each file within the current directory. The asterisk ("*") character 
indicates all files, and can be replaced by any pattern (e.g., "*.txt") 
to limit the file search. The if statement "-f" operator can be changed 
to "-d" to find directories instead of files, and "-s" selects files 
with size greater than zero.</p></div><div id="chapter6.Processing_in_Groups"><h3>Processing in Groups</h3><p>Because
 of technical limits in the Entrez link server, it may be necessary to 
perform an elink operation on a large set of records by using a function
 that splits unique identifiers or sequence accession numbers into 
smaller groups:</p><pre>  JoinIntoGroupsOf() {<br>    xargs -n "$@" echo |<br>    sed 's/ /,/g'<br>  }<br>  alias join-into-group-of='JoinIntoGroupsOf'</pre><p>The following example will process sequence records in groups of 200 accessions at a time:</p><pre>  ...<br>  efetch -format acc |<br>  join-into-groups-of 200 |<br>  xargs -n 1 sh -c 'epost -db nuccore -format acc -id "$0" |<br>  elink -target pubmed |<br>  efetch -format abstract'</pre></div></div><div id="chapter6.Local_Data_Cache"><h2 id="_chapter6_Local_Data_Cache_">Local Data Cache</h2><p>Entrez
 Direct users normally obtain selected data records with efetch, which 
makes calls to the efetch.fcgi network server. This solution works well 
when a few thousand records are needed, but it does not scale for much 
larger sets of data, where the time it takes to download becomes a 
limiting factor.</p><div id="chapter6.Random_Access_Archive"><h3>Random Access Archive</h3><p>As
 an alternative, the entire set of PubMed records can be obtained by 
file transfer protocol, and can be kept current by retrieving daily 
update files. The release files can be decompressed and piped through 
xtract for bulk analysis, but they do not provide random access to 
individual records.</p><p>Recent advances in computer file system 
technology now allow all of these published journal article citations to
 be repackaged as individual files on an inexpensive 500 gigabyte 
external solid state drive, using a hierarchy of folders to organize the
 28 million PubMed records. This approach uses pairs of digits in the 
PMID as nodes in the directory structure, providing direct and rapid 
access to any record. For example, PMID 12345678 would be stored (as a 
compressed XML file) at /Archive/12/34/56/12345678.xml.gz.</p><p>EDirect
 now includes an archive-pubmed script to fully automate the download, 
update, extraction, and storage process. The path to the dedicated solid
 state drive is passed to the script on the command-line:</p><pre>  archive-pubmed -path /Volumes/alexandria</pre><p>It
 can be rerun on a daily or weekly basis to keep the archive up to date.
 Once the archive is populated, PMIDs can be piped to the fetch-pubmed 
script in order to retrieve PubmedArticle XML records from the local 
data cache. fetch-pubmed requires the path to the Archive subdirectory 
in order to find the data. For example:</p><pre>  esearch -db pubmed -query "cancer AND regulation" |<br>  efetch -format uid |<br>  fetch-pubmed -path /Volumes/alexandria/Archive</pre><p>The
 -path argument is not needed by either script if an environment 
variable is set up in the user's .bash_profile configuration file:</p><pre>  export EDIRECT_PUBMED_MASTER=/Volumes/alexandria</pre><p>and EDIRECT_PUBMED_MASTER is assumed to be set in subsequent examples.</p></div><div id="chapter6.Local_Query_Index"><h3>Local Query Index</h3><p>A
 similar divide-and-conquer strategy can be used to create a local 
information retrieval index suitable for large data mining queries. A 
second new script, index-pubmed, automates this activity. (It is a 
superset of archive-pubmed, and can also be run at regular intervals.)</p><p>For
 selected PubMed text fields (the title and primary abstract), the 
indexing process deletes hyphens after specific prefixes, removes 
accents and diacritical marks, splits words at punctuation characters, 
corrects encoding artifacts, and spells out Greek letters for easier 
searching on scientific terms. It then prepares inverted indices with 
term positions, and uses them to build distributed term lists and 
postings files.</p><p>For example, the term list that includes "cancer" 
would be located at /Postings/c/a/n/c/canc.trm. A query on cancer thus 
only needs to load a very small subset of the total index.</p><p>This 
design supports efficient Boolean expression evaluation (e.g., 
"(literacy AND numeracy) NOT (adolescent OR child)"), unrestricted 
wildcard truncation (e.g., "therap*"), phrase queries (e.g., "selective 
serotonin reuptake inhibitor"), and proximity searches (e.g., "vitamin c
 ~ ~ common cold").</p></div><div id="chapter6.Local_Search_Commands"><h3>Local Search Commands</h3><p>The
 phrase-search script controls access to the local indices. The actual 
underlying work is done by the rchive program. Term counts are returned 
by -count (combining wildcard expansions into a single number) or by 
-counts (which expands wildcards and displays the individual terms and 
counts). Using -query returns the PMIDs that match the search 
expression.</p><p>For example, examining the term list with a truncated phrase:</p><pre>  phrase-search -count "catabolite repress*"</pre><p>translates to:</p><pre>  rchive -path /Volumes/alexandria/Postings -count "catabolite repress*"</pre><p>and returns the total counts of the individual words:</p><pre>  4325      catabolite<br>  104977    repress*</pre><p>Using -counts instead of -count returns the expanded terms and the individual postings counts:</p><pre>  4325     catabolite<br>  12002    repress<br>  1        repressa<br>  5        repressable<br>  1        repressae<br>  1        repressant<br>  2        repressants<br>  1        repressc<br>  1        represse<br>  26322    repressed<br>  1        repressedm<br>  ...</pre><p>Searching with a phrase:</p><pre>  phrase-search -query "selective serotonin reuptake inhibitor*" |<br>  fetch-pubmed</pre><p>generates
 a list of PMIDs for records that contain the phrase, and then retrieves
 the relevant PubmedArticle XML from the local archive. Appending the 
query with [STEM] evaluates the same expression using words that have 
been processed through the Porter2 stemming algorithm.</p></div></div><div id="chapter6.XML_Processing"><h2 id="_chapter6_XML_Processing_">XML Processing</h2><p>In
 addition to -first and -last commands, xtract has other -element 
variants that return transformed versions of the selected data values. 
It also has methods that make it easy to wrap extracted values in XML 
tags for further processing by xtract.</p><div id="chapter6.Data_Transformation"><h3>Data Transformation</h3><p>Numeric
 evaluation commands work on a set of data with the same tag names. For 
example, -num will count the number of named objects (with the -element 
"#Author" shortcut translated to -num Author). Integer values in XML 
objects can be added with -sum, and their arithmetic mean can be 
calculated with -avg. The number of characters in selected contents is 
returned by -len (with -element "%Title" implemented as -len Title).</p><p>Text
 and string commands work on individual string values. Sentences are 
split at punctuation marks with -words, or converted to upper case with 
-upper. The -year command returns the first four-digit token encountered
 in the data, so it works correctly with an integer Year or the 
semi-structured MedlineDate string. Applying -year to "PubDate/*" will 
examine the values of all internal objects, regardless of tag name, and 
is a general solution for obtaining the PubMed publication year.</p><p>The full set of -element variants can be seen by running xtract -help.</p></div><div id="chapter6.Sequential_Processing"><h3>Sequential Processing</h3><p>Data
 analysis frequently involves several sequential steps of text or 
numeric processing. Examining the growth of PubMed abstracts over time, 
for example, requires isolating individual words per abstract, counting 
the words per article, filtering the results by year, and then computing
 the average word count per year. The individual operations are all 
supported by xtract functions, so wrapping intermediate values in XML 
can eliminate the need for a separate script to perform the 
calculations.</p><p>Running an esearch query on articles published in a 
chosen journal, limiting the results to articles with abstracts, 
obtaining the individual PMIDs, and fetching the records from the local 
data cache:</p><pre>  esearch -db pubmed -query "PNAS [JOUR]" -pub abstract |<br>  efetch -format uid |<br>  fetch-pubmed</pre><p>returns
 an XML PubmedArticleSet containing just over 115,000 PubmedArticle 
records. The next step will be to extract the year of publication and 
the individual words from the article's abstract. The desired fields in 
the original XML are:</p><pre>  ...<br>  &lt;PubDate&gt;<br>    &lt;Year&gt;2018&lt;/Year&gt;<br>    ...<br>  &lt;/PubDate&gt;<br>  ...<br>  &lt;Abstract&gt;<br>    &lt;AbstractText&gt;Ammonia plays a key role in the ....&lt;/AbstractText&gt;<br>  &lt;/Abstract&gt;<br>  ...</pre><p>Piping the PubmedArticleSet to an initial version of the extraction commands:</p><pre>  xtract -stops -pattern PubmedArticle \<br>    -year "PubDate/*" \<br>    -pfc "\n" -sep "\n" -words Abstract/AbstractText</pre><p>prints the year and each individual abstract word on a separate line:</p><pre>  2018<br>  ammonia<br>  plays<br>  a<br>  key<br>  role<br>  in<br>  the<br>  neutralization<br>  of<br>  atmospheric<br>  acids<br>  ...</pre></div><div id="chapter6.Wrapping_Results_in_XML"><h3>Wrapping Results in XML</h3><p>Customization
 arguments make it possible to wrap intermediate values (e.g., 
individual words per abstract, and word counts per article) back into 
XML form, so they can be sent to another xtract command for the next 
processing step. Piping the PubmedArticleSet to:</p><pre>  xtract -stops -head "&lt;Set&gt;" -tail "&lt;/Set&gt;" -hd "&lt;Rec&gt;" -tl "&lt;/Rec&gt;" \<br>    -pattern PubmedArticle \<br>    -pfx "&lt;Year&gt;" -sfx "&lt;/Year&gt;" -year "PubDate/*" \<br>    -pfx "&lt;Abst&gt;" -sfx "&lt;/Abst&gt;" -sep "&lt;/Abst&gt;&lt;Abst&gt;" \<br>    -words Abstract/AbstractText</pre><p>allows extracted values to be written as structured XML fragments (shown reformatted for easier reading):</p><pre>  &lt;Set&gt;<br>    &lt;Rec&gt;<br>      &lt;Year&gt;2018&lt;/Year&gt;<br>      &lt;Abst&gt;ammonia&lt;/Abst&gt;<br>      &lt;Abst&gt;plays&lt;/Abst&gt;<br>      &lt;Abst&gt;a&lt;/Abst&gt;<br>      &lt;Abst&gt;key&lt;/Abst&gt;<br>      &lt;Abst&gt;role&lt;/Abst&gt;<br>      &lt;Abst&gt;in&lt;/Abst&gt;<br>      &lt;Abst&gt;the&lt;/Abst&gt;<br>      &lt;Abst&gt;neutralization&lt;/Abst&gt;<br>      &lt;Abst&gt;of&lt;/Abst&gt;<br>      &lt;Abst&gt;atmospheric&lt;/Abst&gt;<br>      &lt;Abst&gt;acids&lt;/Abst&gt;<br>      ...</pre><p>The
 -wrp argument is a convenience that sets all of the appropriate 
customization arguments, shown above, given just the XML object names:</p><pre>  xtract -stops -wrp Set,Rec -pattern PubmedArticle \<br>    -wrp "Year" -year "PubDate/*" \<br>    -wrp "Abst" -words Abstract/AbstractText</pre><p>making it easy to wrap the intermediate values.</p><p>The
 results are then piped to the next step, which uses -num to count the 
number of words that were in the abstract of each article:</p><pre>  xtract -wrp Set,Pub -pattern Rec \<br>    -wrp "Year" -element Year \<br>    -wrp "Num" -num Abst</pre><p>again wrapping the results as structured XML:</p><pre>  &lt;Set&gt;<br>    &lt;Pub&gt;&lt;Year&gt;2018&lt;/Year&gt;&lt;Num&gt;198&lt;/Num&gt;&lt;/Pub&gt;<br>    &lt;Pub&gt;&lt;Year&gt;2018&lt;/Year&gt;&lt;Num&gt;167&lt;/Num&gt;&lt;/Pub&gt;<br>    &lt;Pub&gt;&lt;Year&gt;2018&lt;/Year&gt;&lt;Num&gt;242&lt;/Num&gt;&lt;/Pub&gt;<br>    ...</pre><p>Redirecting the output with "&gt; countsByYear.xml" saves the results in a file, to be used for all subsequent processing.</p></div><div id="chapter6.Selection_of_XML_Subsets"><h3>Selection of XML Subsets</h3><p>The
 xtract -select argument allows record subsetting by data value, such as
 the year of publication. It acts as an "-if" statement, and is followed
 by conditionals, but with no -element, since its purpose is to pass 
along the entirety of all records that satisfy the condition.</p><p>This
 can be more convenient, and is significantly faster, than running a 
separate esearch query for each year, and it can look at data elements 
that are not indexed by Entrez. It is used in the loop below to limit 
the processed query results to one year at a time, passing the relevant 
subset to a second xtract command:</p><pre>  for yr in {1960..2018}<br>  do<br>    cat countsByYear.xml |<br>    xtract -wrp Raw -pattern Pub -select Year -eq "$yr" |<br>    xtract -pattern Raw -lbl "$yr" -avg Num<br>  done</pre><p>that applies -avg to the word counts in order to compute the average number of abstract words per article for the current year:</p><pre>  1969    121<br>  1970    119<br>  1971    126<br>  ...<br>  2016    207<br>  2017    206<br>  2018    205</pre><p>This result can be saved by redirecting to a file, or it can be piped to:</p><pre>  tee /dev/tty |<br>  xy-plot pnas.png</pre><p>to print the data to the terminal and then display the results in graphical format. The last step should be:</p><pre>  rm countsByYear.xml</pre><p>to remove the intermediate file.</p></div></div><div id="chapter6.Examples"><h2 id="_chapter6_Examples_">Examples</h2><p>Additional examples of using EDirect to answer ad hoc questions are shown in this section.</p><div id="chapter6.Author_Frequency"><h3>Author Frequency</h3><p>Who are the most prolific authors on rattlesnake phospholipase?</p><pre>  esearch -db pubmed -query \<br>    "crotalid venoms [MAJR] AND phospholipase [TIAB]" |<br>  efetch -format xml |<br>  xtract -pattern PubmedArticle \<br>    -block Author -sep " " -tab "\n" -element LastName,Initials  |<br>  sort-uniq-count-rank</pre><p>This search produces:</p><pre>  74    Lomonte B<br>  73    Gutiérrez JM<br>  49    Soares AM<br>  48    Marangoni S<br>  43    Giglio JR<br>  39    Bon C<br>  ...</pre></div><div id="chapter6.Publication_Distribution"><h3>Publication Distribution</h3><p>When were the most papers about Legionnaires disease published?</p><pre>  esearch -db pubmed -query "legionnaires disease [TITL]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element PubDate |<br>  cut -c 1-4 |<br>  sort-uniq-count-rank</pre><p>reports the number of selected papers per year:</p><pre>  173    1979<br>  102    1980<br>  96     1978<br>  92     1981<br>  66     1983<br>  ...</pre></div><div id="chapter6.Treatment_Locations"><h3>Treatment Locations</h3><p>What is the geographic distribution of sepsis treatment studies?</p><pre>  esearch -db pubmed -query \<br>    "sepsis/therapy [MESH] AND geographic locations [MESH]" |<br>  efetch -format xml |<br>  xtract -pattern PubmedArticle \<br>    -block MeshHeading -if DescriptorName@Type -equals Geographic \<br>      -tab "\n" -element DescriptorName |<br>  sort-uniq-count-rank</pre><p>returns the number of articles ranked by country (or region) of study:</p><pre>  567    United States<br>  207    Spain<br>  176    Great Britain<br>  156    Germany<br>  123    India<br>  118    Europe<br>  113    France<br>  100    Taiwan<br>  89     Japan<br>  83     Thailand<br>  75     Italy<br>  74     England<br>  ...</pre></div><div id="chapter6.Research_History"><h3>Research History</h3><p>What is the historic pattern of publication on diphtheria, pertussis, and tetanus?</p><pre>  #!/bin/bash<br>  result=""<br>  for disease in diphtheria pertussis tetanus<br>  do<br>    current=`for (( yr = 2010; yr &gt;= 1900; yr -= 10 ))<br>      do<br>        esearch -db pubmed -query "$disease [TITL] AND $yr:$((yr+9)) [PDAT]" |<br>        xtract -pattern ENTREZ_DIRECT -lbl "${yr}s" -element Count<br>      done`<br>    heading=`echo -e "${disease:0:4}" | tr [a-z] [A-Z]`<br>    current=`echo -e "Years\t$heading\n-----\t----\n$current"`<br>    if [ -n "$result" ]<br>    then<br>      result=`join -t $'\t' &lt;(echo "$result") &lt;(echo "$current")`<br>    else<br>      result=$current<br>    fi<br>  done<br>  echo "$result"</pre><p>gives per-decade counts of relevant papers for each disease:</p><pre>  Years    DIPH    PERT    TETA<br>  -----    ----    ----    ----<br>  2010s    577     1708    914<br>  2000s    892     1966    1344<br>  1990s    1150    2661    1615<br>  1980s    780     1746    1485<br>  1970s    749     698     1524<br>  1960s    1152    635     2086<br>  1950s    1226    491     1540<br>  1940s    452     173     239<br>  1930s    157     26      46<br>  1920s    128     5       21<br>  1910s    83      7       41<br>  1900s    93      3       28</pre></div><div id="chapter6.Protein_Homolog"><h3>Protein Homolog</h3><p>Is there a mammalian equivalent of lycopene cyclase?</p><pre>  esearch -db protein -query \<br>    "lycopene beta cyclase [PROT] AND tomato [ORGN]" |<br>  elink -related |<br>  efetch -format gpc |<br>  xtract -pattern INSDSeq -element INSDSeq_division |<br>  sort-uniq-count-rank</pre><p>In the resulting list of GenBank division codes:</p><pre>  905    BCT<br>  856    ENV<br>  609    PLN<br>  197    CON<br>  127    PAT<br>  2      SYN</pre><p>there
 are no similar sequences (protein neighbors) in the HUM, PRI, ROD, MAM,
 VRT, or INV divisions, so lycopene cyclase is not present in animals.</p></div><div id="chapter6.Longest_Sequences"><h3>Longest Sequences</h3><p>What are the longest known insulin precursor molecules?</p><pre>  esearch -db protein -query "insulin [PROT]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element Caption Slen Title |<br>  grep -v receptor | sort -k 2,2nr | head -n 5 | cut -f 1 |<br>  xargs -n 1 sh -c 'efetch -db protein -id "$0" -format gp &gt; "$0".gpf'</pre><p>Post-processing
 excludes the longer "insulin-like receptor" sequences and saves the 
GenPept results to individual files named by their sequence accessions:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/protein/307167790/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">EFN61235</a>.gpf<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/307199990/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">EFN80340</a>.gpf<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/344252373/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">EGW08477</a>.gpf<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/405950444/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">EKC18433</a>.gpf<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/432099190/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">ELK28555</a>.gpf</pre><p>using the right angle bracket ("&gt;") UNIX output redirection character.</p></div><div id="chapter6.Archaea_Enzyme"><h3>Archaea Enzyme</h3><p>Which archaebacteria have chloramphenicol acetyltransferase?</p><pre>  esearch -db protein -query \<br>    "chloramphenicol acetyltransferase [PROT] AND archaea [ORGN]" |<br>  efetch -format gpc |<br>  xtract -pattern INSDSeq -element INSDSeq_organism INSDSeq_definition |<br>  grep -i chloramphenicol | cut -f 1 | sort -f | uniq</pre><p>produces a list of organism names:</p><pre>  Methanobrevibacter ruminantium<br>  Methanobrevibacter smithii<br>  Methanosarcina acetivorans<br>  ...</pre></div><div id="chapter6.Structural_Similarity"><h3>Structural Similarity</h3><p>What archaea structures are similar to snake venom phospholipase?</p><pre>  esearch -db structure -query "crotalus [ORGN] AND phospholipase A2" |<br>  elink -related |<br>  efilter -query "archaea [ORGN]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary \<br>    -if PdbClass -equals Hydrolase \<br>      -element PdbDescr |<br>  sort -f | uniq -i</pre><p>This
 query uses geometric comparison (structure neighboring) to find 
proteins that are too divergent to be detected by sequence similarity 
with a BLAST search:</p><pre>  Crystal Structure Of Autoprocessed Form Of Tk-Subtilisin<br>  Crystal Structure Of Ca2 Site Mutant Of Pro-S324a<br>  Crystal Structure Of Ca3 Site Mutant Of Pro-S324a<br>  ...</pre></div><div id="chapter6.Taxonomy_Search"><h3>Taxonomy Search</h3><p>Which organisms contain an annotated RefSeq genome MatK gene?</p><pre>  esearch -db nuccore -query "MatK [GENE] AND NC_0:NC_999999999 [PACC]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element TaxId |<br>  sort -n | uniq |<br>  epost -db taxonomy |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element ScientificName |<br>  sort</pre><p>The
 first query obtains taxonomy UIDs from nucleotide document summaries 
and uploads them for separate retrieval from the taxonomy database:</p><pre>  Acidosasa purpurea<br>  Acorus americanus<br>  ...<br>  Zingiber spectabile<br>  Zygnema circumcarinatum</pre></div><div id="chapter6.Chromosome_Locations"><h3>Chromosome Locations</h3><p>Where are mammalian calmodulin genes located?</p><pre>  esearch -db gene -query "calmodulin [PFN] AND mammalia [ORGN]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -MAP "(-)" -MAP MapLocation \<br>    -element Id Name "&amp;MAP" ScientificName</pre><p>The MAP variable is initialized with a literal dash to prevent missing data from shifting columns in the table:</p><pre>  801       CALM1    14q32.11     Homo sapiens<br>  808       CALM3    19q13.32     Homo sapiens<br>  805       CALM2    2p21         Homo sapiens<br>  24242     Calm1    6q32         Rattus norvegicus<br>  12313     Calm1    12 E         Mus musculus<br>  326597    CALM     -            Bos taurus<br>  50663     Calm2    6q12         Rattus norvegicus<br>  24244     Calm3    1q21         Rattus norvegicus<br>  12315     Calm3    7 9.15 cM    Mus musculus<br>  12314     Calm2    17 E4        Mus musculus<br>  617095    CALM1    -            Bos taurus<br>  396838    CALM3    6            Sus scrofa<br>  ...</pre><p>The -else command can also be used to insert placeholders for missing data:</p><pre>  esearch -db gene -query "calmodulin [PFN] AND mammalia [ORGN]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary \<br>    -if MapLocation -element Id Name MapLocation ScientificName \<br>      -else -element Id Name -lbl "\-" -element ScientificName</pre><p>The -def command can achieve the same result for missing elements:</p><pre>  esearch -db gene -query "calmodulin [PFN] AND mammalia [ORGN]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary \<br>    -def "-" -element Id Name MapLocation ScientificName</pre></div><div id="chapter6.Exon_Counts"><h3>Exon Counts</h3><p>How many exons are in each dystrophin transcript variant?</p><pre>  esearch -db gene -query "DMD [GENE] AND human [ORGN]" |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary \<br>    -block GenomicInfoType -tab "\n" -element ChrAccVer,ChrStart,ChrStop |</pre><p>This search returns the chromosome accession and the (0-based) gene start and stop positions:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    33339608    31119221</pre><p>These are then passed to efetch in (0-based) -chr_start and -chr_stop arguments:</p><pre>  xargs -n 3 sh -c 'efetch -db nuccore -format gbc \<br>    -id "$0" -chr_start "$1" -chr_stop "$2"' |</pre><p>which
 converts them to (1-based) -seq_start and -seq_stop arguments and 
retrieves an INSDSeq XML subset record for the indicated region. That 
contains a number of alternatively-spliced dystrophin mRNA and CDS 
features.</p><p>Data extraction computes the number of intervals for 
each mRNA location (corresponding to individual exons or UTRs), and 
obtains the transcript sequence accession, transcript length, and 
product name from qualifiers:</p><pre>  xtract -insd complete mRNA "#INSDInterval" \<br>    transcript_id "%transcription" product |</pre><p>Final processing sorts by number of exons:</p><pre>  grep -i dystrophin |<br>  sed 's/dystrophin, transcript variant //g' |<br>  sort -k 2,2nr -k 4,4nr</pre><p>resulting in a table of exon counts and transcript lengths:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    79    <a href="https://www.ncbi.nlm.nih.gov/nuccore/238018047" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NM_004010.3</a>       14083    Dp427p2<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    79    <a href="https://www.ncbi.nlm.nih.gov/nuccore/238018043" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NM_000109.3</a>       14069    Dp427c<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    79    <a href="https://www.ncbi.nlm.nih.gov/nuccore/238018046" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NM_004009.3</a>       14000    Dp427p1<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    79    <a href="https://www.ncbi.nlm.nih.gov/nuccore/238018044" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NM_004006.2</a>       13993    Dp427m<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    78    <a href="https://www.ncbi.nlm.nih.gov/nuccore/578837828" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">XM_006724468.1</a>    13920    X1<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    78    <a href="https://www.ncbi.nlm.nih.gov/nuccore/578837831" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">XM_006724469.1</a>    13802    X2<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    77    <a href="https://www.ncbi.nlm.nih.gov/nuccore/578837833" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">XM_006724470.1</a>    13881    X3<br>  ...</pre></div><div id="chapter6.Genome_Range"><h3>Genome Range</h3><p>What genes are in a given range on the human Y chromosome?</p><pre>  esearch -db gene -query "Homo sapiens [ORGN] AND Y [CHR]" |<br>  efilter -status alive | efetch -format docsum |<br>  xtract -pattern DocumentSummary -NAME Name -DESC Description \<br>    -block GenomicInfoType -if ChrLoc -equals Y \<br>      -min ChrStart,ChrStop -element "&amp;NAME" "&amp;DESC" |<br>  sort -k 1,1n | cut -f 2- |<br>  between-two-genes ASMT IL3RA</pre><p>This
 query returns a table of gene names and descriptions, for the human "Y"
 chromosome, in the region between the ASMT and IL3RA genes:</p><pre>  IL3RA           interleukin 3 receptor subunit alpha<br>  LOC101928032    uncharacterized LOC101928032<br>  LOC101928055    uncharacterized LOC101928055<br>  SLC25A6         solute carrier family 25 member 6<br>  LOC105373102    uncharacterized LOC105373102<br>  LINC00106       long intergenic non-protein coding RNA 106<br>  ASMTL-AS1       ASMTL antisense RNA 1<br>  ASMTL           acetylserotonin O-methyltransferase-like<br>  P2RY8           purinergic receptor P2Y8<br>  AKAP17A         A-kinase anchoring protein 17A<br>  ASMT            acetylserotonin O-methyltransferase</pre><p>(The
 "-if ChrLoc -equals Y" test is necessary because certain genes (e.g., 
IL9R) are present in the pseudoautosomal regions common to both X and Y 
chromosomes:</p><pre>  ...<br>  &lt;GenomicInfo&gt;<br>    &lt;GenomicInfoType&gt;<br>      &lt;ChrLoc&gt;Y&lt;/ChrLoc&gt;<br>      &lt;ChrAccVer&gt;<a href="https://www.ncbi.nlm.nih.gov/nuccore/568815574" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000024.10</a>&lt;/ChrAccVer&gt;<br>      &lt;ChrStart&gt;57184100&lt;/ChrStart&gt;<br>      &lt;ChrStop&gt;57197336&lt;/ChrStop&gt;<br>      &lt;ExonCount&gt;10&lt;/ExonCount&gt;<br>    &lt;/GenomicInfoType&gt;<br>    &lt;GenomicInfoType&gt;<br>      &lt;ChrLoc&gt;X&lt;/ChrLoc&gt;<br>      &lt;ChrAccVer&gt;<a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>&lt;/ChrAccVer&gt;<br>      &lt;ChrStart&gt;155997580&lt;/ChrStart&gt;<br>      &lt;ChrStop&gt;156010816&lt;/ChrStop&gt;<br>      &lt;ExonCount&gt;10&lt;/ExonCount&gt;<br>    &lt;/GenomicInfoType&gt;<br>  &lt;/GenomicInfo&gt;<br>  ...</pre><p>with each gene copy annotated in its own GenomicInfoType block.)</p></div><div id="chapter6.Gene_Counts"><h3>Gene Counts</h3><p>How many genes are on each human chromosome?</p><pre>  for chr in {1..22} X Y MT<br>  do<br>    esearch -db gene -query "Homo sapiens [ORGN] AND $chr [CHR]" |<br>    efilter -query "alive [PROP] AND genetype protein coding [PROP]" |<br>    efetch -format docsum |<br>    xtract -pattern DocumentSummary -NAME Name \<br>      -block GenomicInfoType -if ChrLoc -equals "$chr" \<br>        -tab "\n" -element ChrLoc,"&amp;NAME" |<br>    sort | uniq | cut -f 1 | sort-uniq-count-rank<br>  done</pre><p>returns a count of unique protein-coding genes per chromosome:</p><pre>  2067    1<br>  1268    2<br>  1071    3<br>  755     4<br>  873     5<br>  1034    6<br>  935     7<br>  690     8<br>  801     9<br>  739     10<br>  1288    11<br>  1027    12<br>  335     13<br>  607     14<br>  608     15<br>  862     16<br>  1181    17<br>  277     18<br>  1402    19<br>  545     20<br>  248     21<br>  445     22<br>  844     X<br>  71      Y<br>  13      MT</pre><p>The
 range construct cannot be used for Roman numerals, so the equivalent 
query on Saccharomyces cerevisiae would need to explicitly list all 
chromosomes:</p><pre>  for chr in I II III IV V VI VII VIII IX X XI XII XIII XIV XV XVI MT</pre><p>Plastid genes can be selected with "source plastid [PROP]".</p></div><div id="chapter6.Complete_Genomes"><h3>Complete Genomes</h3><p>What complete genomes are available for Escherichia coli?</p><pre>  esearch -db assembly -query \<br>    "Escherichia coli [ORGN] AND representative [PROP]" |<br>  elink -target nuccore -name assembly_nuccore_refseq |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element AccessionVersion Slen Title |<br>  sed 's/,.*//' |<br>  sort -t $'\t' -k 2,2nr</pre><p>This
 search finds genomic assemblies and sorts the results by sequence 
length, allowing complete genomes to be easily distinguished from 
smaller plasmids:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/15829254" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_002695.1</a>    5498450    Escherichia coli O157:H7 str. Sakai chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/407479587" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_018658.1</a>    5273097    Escherichia coli O104:H4 str. 2011C-3493 ...<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/218703261" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_011751.1</a>    5202090    Escherichia coli UMN026 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/218698419" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_011750.1</a>    5132068    Escherichia coli IAI39 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/387615344" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_017634.1</a>    4747819    Escherichia coli O83:H1 str. NRG 857C chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/556503834" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000913.3</a>    4641652    Escherichia coli str. K-12 substr. MG1655<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/387615190" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_017659.1</a>    147060     Escherichia coli O83:H1 str. NRG 857C plasmid ...<br>  ...</pre><p>The sed command removes extraneous text in the title (e.g., complete genome, complete sequence, primary assembly) after a comma.</p><p>A similar query for humans, additionally filtering out scaffolds, contigs, and plasmids:</p><pre>  esearch -db assembly -query "Homo sapiens [ORGN] AND representative [PROP]" |<br>  elink -target nuccore -name assembly_nuccore_refseq |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -element AccessionVersion Slen Title |<br>  sed 's/,.*//' | grep -v scaffold | grep -v contig | grep -v plasmid | sort</pre><p>returns the assembled chromosome and mitochondrial sequence records:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815597" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000001.11</a>    248956422    Homo sapiens chromosome 1<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815596" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000002.12</a>    242193529    Homo sapiens chromosome 2<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815595" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000003.12</a>    198295559    Homo sapiens chromosome 3<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815594" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000004.12</a>    190214555    Homo sapiens chromosome 4<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815593" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000005.10</a>    181538259    Homo sapiens chromosome 5<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815592" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000006.12</a>    170805979    Homo sapiens chromosome 6<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815591" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000007.14</a>    159345973    Homo sapiens chromosome 7<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815590" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000008.11</a>    145138636    Homo sapiens chromosome 8<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815589" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000009.12</a>    138394717    Homo sapiens chromosome 9<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815588" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000010.11</a>    133797422    Homo sapiens chromosome 10<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815587" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000011.10</a>    135086622    Homo sapiens chromosome 11<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815586" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000012.12</a>    133275309    Homo sapiens chromosome 12<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815585" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000013.11</a>    114364328    Homo sapiens chromosome 13<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815584" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000014.9</a>     107043718    Homo sapiens chromosome 14<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815583" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000015.10</a>    101991189    Homo sapiens chromosome 15<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815582" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000016.10</a>    90338345     Homo sapiens chromosome 16<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815581" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000017.11</a>    83257441     Homo sapiens chromosome 17<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815580" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000018.10</a>    80373285     Homo sapiens chromosome 18<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815579" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000019.10</a>    58617616     Homo sapiens chromosome 19<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815578" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000020.11</a>    64444167     Homo sapiens chromosome 20<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815577" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000021.9</a>     46709983     Homo sapiens chromosome 21<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815576" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000022.11</a>    50818468     Homo sapiens chromosome 22<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815575" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000023.11</a>    156040895    Homo sapiens chromosome X<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815574" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000024.10</a>    57227415     Homo sapiens chromosome Y<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/251831106" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_012920.1</a>     16569        Homo sapiens mitochondrion</pre><p>This process can be automated to loop through a list of specified organisms:</p><pre>  for org in \<br>    "Agrobacterium tumefaciens" \<br>    "Bacillus anthracis" \<br>    "Escherichia coli" \<br>    "Neisseria gonorrhoeae" \<br>    "Pseudomonas aeruginosa" \<br>    "Shigella flexneri" \<br>    "Streptococcus pneumoniae"<br>  do<br>    esearch -db assembly -query "$org [ORGN]" |<br>    efilter -query "representative [PROP]" |<br>    elink -target nuccore -name assembly_nuccore_refseq |<br>    efetch -format docsum |<br>    xtract -pattern DocumentSummary -element AccessionVersion Slen Title |<br>    sed 's/,.*//' |<br>    grep -v -i -e scaffold -e contig -e plasmid -e sequence -e patch |<br>    sort -t $'\t' -k 2,2nr<br>  done</pre><p>which generates:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/222084201" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_011985.1</a>    4005130    Agrobacterium radiobacter K84 chromosome 1<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/222080781" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_011983.1</a>    2650913    Agrobacterium radiobacter K84 chromosome 2<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/49183039" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_005945.1</a>    5228663    Bacillus anthracis str. Sterne chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/30260195" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_003997.3</a>    5227293    Bacillus anthracis str. Ames chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/15829254" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_002695.1</a>    5498450    Escherichia coli O157:H7 str. Sakai chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/407479587" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_018658.1</a>    5273097    Escherichia coli O104:H4 str. 2011C-3493 ...<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/218703261" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_011751.1</a>    5202090    Escherichia coli UMN026 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/218698419" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_011750.1</a>    5132068    Escherichia coli IAI39 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/387615344" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_017634.1</a>    4747819    Escherichia coli O83:H1 str. NRG 857C chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/556503834" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000913.3</a>    4641652    Escherichia coli str. K-12 substr. MG1655<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/59800473" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_002946.2</a>    2153922    Neisseria gonorrhoeae FA 1090 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/110645304" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_002516.2</a>    6264404    Pseudomonas aeruginosa PAO1 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/344915202" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_004337.2</a>    4607202    Shigella flexneri 2a str. 301 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/194172857" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_003028.3</a>    2160842    Streptococcus pneumoniae TIGR4 chromosome<br>  <a href="https://www.ncbi.nlm.nih.gov/nuccore/15902044" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_003098.1</a>    2038615    Streptococcus pneumoniae R6 chromosome</pre></div><div id="chapter6.Amino_Acid_Composition"><h3>Amino Acid Composition</h3><p>What is the amino acid composition of human titin?</p><pre>  abbrev=( Ala Asx Cys Asp Glu Phe Gly His Ile \<br>           Xle Lys Leu Met Asn Pyl Pro Gln Arg \<br>           Ser Thr Sec Val Trp Xxx Tyr Glx )<br>  efetch -db protein -id "<a href="https://www.ncbi.nlm.nih.gov/protein/384872704/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">Q8WZ42</a>.4" -format gpc |<br>  xtract -pattern INSDSeq -element INSDSeq_sequence |<br>  tr A-Z a-z |<br>  sed 's/[^a-z]//g' |<br>  fold -w 1 |<br>  sort-uniq-count |<br>  while read num lttr<br>  do<br>    idx=$(printf %i "'$lttr'")<br>    ofs=$((idx-97))<br>    echo -e "${abbrev[$ofs]}\t$num"<br>  done |<br>  sort</pre><p>produces a table of residue counts using the three-letter amino acid abbreviations:</p><pre>  Ala    2084<br>  Arg    1640<br>  Asn    1111<br>  Asp    1720<br>  Cys    513<br>  Gln    942<br>  Glu    3193<br>  Gly    2066<br>  His    478<br>  Ile    2062<br>  Leu    2117<br>  Lys    2943<br>  Met    398<br>  Phe    908<br>  Pro    2517<br>  Ser    2463<br>  Thr    2546<br>  Trp    466<br>  Tyr    999<br>  Val    3184</pre></div><div id="chapter6.Amino_Acid_Substitutions"><h3>Amino Acid Substitutions</h3><p>What are the missense products of green-sensitive opsin?</p><pre>  ApplySNPs() {<br>    seq=""<br>    last=""<br><br>    while read rsid accn res pos<br>    do<br>      if [ "$accn" != "$last" ]<br>      then<br>        insd=$(efetch -db protein -id "$accn" -format gbc &lt; /dev/null)<br>        seq=$(echo $insd | xtract -pattern INSDSeq -element INSDSeq_sequence)<br>        last=$accn<br>      fi<br><br>      pfx=""<br>      sfx=""<br>      echo "&gt;$rsid [$accn $res@$pos]"<br>      if [ $pos -gt 1 ]<br>      then<br>        pfx=$(echo ${seq:0:$pos-1})<br>      fi<br>      if [ $pos -lt ${#seq} ]<br>      then<br>        sfx=$(echo ${seq:$pos})<br>      fi<br>      echo "$pfx$res$sfx" | fold -w 50<br>    done<br>  }<br><br>  esearch -db gene -query "CBD [GENE] AND human [ORGN]" |<br>  elink -target snp |<br>  efetch -format xml |<br>  xtract -pattern Rs -pfx "rs" -RSID Rs@rsId \<br>    -block FxnSet -if @fxnClass -equals missense \<br>      -sep "." -element "&amp;RSID" @protAcc,@protVer \<br>      @residue -tab "\n" -1-based @aaPosition |<br>  sort -t $'\t' -k 2,2 -k 4,4n -k 3,3f -k 1.3n | uniq |<br>  ApplySNPs</pre><p>The
 query returns an intermediate table of non-synonymous amino acid 
substitutions (with 0-based location coordinates) derived from single 
nucleotide polymorphisms:</p><pre>  <a href="https://www.ncbi.nlm.nih.gov/snp/?term=1238141906" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=snp" target="_blank">rs1238141906</a>    <a href="https://www.ncbi.nlm.nih.gov/protein/4503965/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_000504.1</a>    K    41<br>  <a href="https://www.ncbi.nlm.nih.gov/snp/?term=1189783086" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=snp" target="_blank">rs1189783086</a>    <a href="https://www.ncbi.nlm.nih.gov/protein/4503965/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_000504.1</a>    L    43<br>  <a href="https://www.ncbi.nlm.nih.gov/snp/?term=1284438666" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=snp" target="_blank">rs1284438666</a>    <a href="https://www.ncbi.nlm.nih.gov/protein/4503965/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_000504.1</a>    I    64<br>  <a href="https://www.ncbi.nlm.nih.gov/snp/?term=1223726997" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=snp" target="_blank">rs1223726997</a>    <a href="https://www.ncbi.nlm.nih.gov/protein/4503965/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_000504.1</a>    T    65<br>  ...</pre><p>The rows are then processed to produce protein sequences with the individual residue substitutions in upper case:</p><pre>  &gt;<a href="https://www.ncbi.nlm.nih.gov/snp/?term=1238141906" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=term&amp;targettype=snp" target="_blank">rs1238141906</a> [<a href="https://www.ncbi.nlm.nih.gov/protein/4503965/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_000504.1</a> K@41]<br>  maqqwslqrlagrhpqdsyedstqssiftytnsnstrgpfKgpnyhiapr<br>  wvyhltsvwmifvviasvftnglvlaatmkfkklrhplnwilvnlavadl<br>  aetviastisvvnqvygyfvlghpmcvlegytvslcgitglwslaiiswe<br>  ...</pre></div><div id="chapter6.3UTR_Sequences"><h3>3'UTR Sequences</h3><p>What are the 3' UTR sequences for lycopene cyclase?</p><pre>  ThreePrimeUTRs() {<br>    xtract -pattern INSDSeq -ACC INSDSeq_accession-version -SEQ INSDSeq_sequence \<br>      -group INSDFeature -if INSDFeature_key -equals CDS -PRD "(-)" \<br>        -block INSDQualifier -if INSDQualifier_name \<br>          -equals product -PRD INSDQualifier_value \<br>        -block INSDFeature -pfc "\n" -element "&amp;ACC" -rst \<br>          -last INSDInterval_to -element "&amp;SEQ" "&amp;PRD" |<br>    while read acc pos seq prd<br>    do<br>      if [ $pos -lt ${#seq} ]<br>      then<br>        echo -e "&gt;$acc 3'UTR: $((pos+1))..${#seq} $prd"<br>        echo "${seq:$pos}" | fold -w 50<br>      elif [ $pos -ge ${#seq} ]<br>      then<br>        echo -e "&gt;$acc NO 3'UTR"<br>      fi<br>    done<br>  }<br><br>  esearch -db nuccore -query "5.5.1.19 [ECNO]" |<br>  efilter -molecule mrna -source refseq |<br>  efetch -format gbc | ThreePrimeUTRs</pre><p>prints the sequences immediately following the CDS stop codon:</p><pre>  &gt;<a href="https://www.ncbi.nlm.nih.gov/nuccore/1036551385" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NM_001328461.1</a> 3'UTR: 1737..1871 lycopene beta cyclase, chloroplastic<br>  gatgaatatagagttactgtgttgtaagctaatcatcatactgatgcaag<br>  tgcattatcacatttacttctgctgatgattgttcataagattatgagtt<br>  agccatttatcaaaaaaaaaaaaaaaaaaaaaaaa<br>  &gt;<a href="https://www.ncbi.nlm.nih.gov/nuccore/941355939" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NM_001316759.1</a> 3'UTR: 1628..1690 lycopene beta cyclase, chloroplastic<br>  atccgagtaattcggaatcttgtccaattttatatagcctatattaatac<br>  ...</pre></div><div id="chapter6.Upstream_Sequences"><h3>Upstream Sequences</h3><p>What sequences are upstream of phenylalanine hydroxylase genes?</p><pre>  esearch -db nuccore -query "<a href="https://www.ncbi.nlm.nih.gov/nuccore/2462721" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">U49897</a> [ACCN]" |<br>  elink -target gene |<br>  elink -target homologene |<br>  elink -target gene |<br>  efetch -format docsum |<br>  xtract -pattern DocumentSummary -if GenomicInfoType -element Id \<br>    -block GenomicInfoType -element ChrAccVer ChrStart ChrStop |<br>  awk -F '\t' -v 'OFS=\t' '{print $1, $2, $3+1, $4+1}'</pre><p>obtains
 a series of homologous genes, converting the gene coordinates to 
1-based positions suitable for retrieving sequence regions:</p><pre>  5053      <a href="https://www.ncbi.nlm.nih.gov/nuccore/568815586" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000012.12</a>    102917603    102838326<br>  18478     <a href="https://www.ncbi.nlm.nih.gov/nuccore/372099100" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_000076.6</a>     87521795     87584137<br>  38871     <a href="https://www.ncbi.nlm.nih.gov/nuccore/671162317" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NT_037436.4</a>     7760453      7763166<br>  24616     <a href="https://www.ncbi.nlm.nih.gov/nuccore/666183999" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_005106.4</a>     28066639     28129772<br>  378962    <a href="https://www.ncbi.nlm.nih.gov/nuccore/685508238" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">NC_007115.6</a>     17420391     17402704<br>  ...</pre><p>Given a shell script named "upstream.sh":</p><pre>  #!/bin/bash -norc<br><br>  bases=1500<br>  if [ -n "$1" ]<br>  then<br>    bases=$1<br>  fi<br><br>  while read id accn start stop<br>  do<br>    if [[ $start -eq 0 || $stop -eq 0 || $start -eq $stop ]]<br>    then<br>      echo "Skipping $id due to ambiguous coordinates"<br>      continue<br>    fi<br>    if [ $start -gt $stop ]<br>    then<br>      stop=$(( start + bases ))<br>      start=$(( start + 1 ))<br>      strand=2<br>    else<br>      stop=$(( start - 1 ))<br>      start=$(( start - bases ))<br>      strand=1<br>    fi<br>    rslt=`efetch -db nuccore -id $accn -format fasta \<br>          -seq_start $start -seq_stop $stop -strand $strand &lt; /dev/null`<br>    echo "$rslt"<br>  done</pre><p>the data lines can be piped through:</p><pre>  upstream.sh 500</pre><p>to
 extract and print the 500 nucleotides immediately upstream of each 
gene. (Without the argument it will default to 1500 nucleotides.)</p></div><div id="chapter6.Author_Combinations"><h3>Author Combinations</h3><p>What are the authorship patterns among selected individuals?</p><p>The
 "coauthors.sh" script takes author name arguments to construct a custom
 data extraction command for analyzing research collaboration patterns:</p><pre>  #!/bin/bash -norc<br><br>  if [ "$#" -lt 2 ]<br>  then<br>    echo "Must supply at least two author names"<br>    exit 1<br>  fi<br><br>  query="xtract -pattern PubmedArticle -element MedlineCitation/PMID"<br><br>  # append a -block statement for each author argument<br>  for auth in "$@"<br>  do<br>    query=`echo "$query -block Author -if LastName -equals \"$auth\"" \<br>                "-sep \" \" -element LastName,Initials"`<br>  done<br><br>  query=`echo "$query | sort -t \$'\\t' -k 2f -k 1,1n"`<br><br>  if [ -t 0 ]<br>  then<br>    # stand-alone command, print constructed query for later use<br>    echo "$query"<br>  else<br>    # dynamically execute query on XML data piped to script<br>    res=`eval "$query"`<br>    echo "$res"<br>  fi</pre><p>If
 XML publication data are piped to the script, it will read the data and
 immediately execute the generated xtract query. Otherwise, if called as
 a stand-alone command, it will print the custom query instructions for 
later use.</p><p>Running the following command:</p><pre>  esearch -db pubmed -query "Casadaban MJ [AUTH] OR Berg CM [AUTH]" |<br>  efetch -format xml |<br>  ./coauthors.sh Casadaban Groisman Berg Garber |<br>  ./extract-fuse.pl pubmed &gt; author_patterns.htm</pre><p>first produces an internal result table of PMIDs grouped by author combination:</p><pre>  ...<br>  7635839    Casadaban MJ<br>  9634770    Casadaban MJ<br>  1827084    Casadaban MJ    Groisman EA<br>  2954879    Casadaban MJ    Groisman EA<br>  3020001    Casadaban MJ    Groisman EA<br>  3525518    Casadaban MJ    Groisman EA<br>  3542967    Casadaban MJ    Groisman EA<br>  6324195    Casadaban MJ    Groisman EA<br>  3301525    Casadaban MJ    Groisman EA    Berg CM</pre><p>The sorted lines are then piped to the "extract-fuse.pl" script:</p><pre>  #!/usr/bin/perl<br><br>  my $max = scalar @ARGV;<br>  if ( $max &lt; 1 ) {<br>    die "Need argument for database\n";<br>  }<br>  my $db = $ARGV[0];<br><br>  my $thisline = "";<br>  my $laststr = "";<br>  my $str = "";<br>  my $uid = "";<br>  my $uidlist = "";<br>  my $count = 0;<br>  my $base = "https://www.ncbi.nlm.nih.gov";<br><br>  my $pfx = "";<br>  while ($thisline = &lt;STDIN&gt;) {<br>    $thisline =~ s/\r//;<br>    $thisline =~ s/\n//;<br><br>    if ($thisline =~ /^([^\t]+)\t(.+)$/) {<br>      $uid = $1;<br>      $str = $2;<br><br>      if ( lc ($str) ne lc ($laststr) and $laststr ne "" ) {<br>        $laststr =~ s/\t/, /g;<br>        print "&lt;p&gt;(&lt;a href=\"$base/$db/$uidlist\"&gt;";<br>        print " $count &lt;/a&gt;) - $laststr&lt;/p&gt;\n";<br>        $pfx = "";<br>        $count = 0;<br>        $uidlist = "";<br>      }<br>      $laststr = $str;<br><br>      $uidlist .= "$pfx$uid";<br>      $pfx = ",";<br>      $count++;<br>    }<br>  }<br><br>  if ( $laststr ne "" ) {<br>    $laststr =~ s/\t/, /g;<br>    print "&lt;p&gt;(&lt;a href=\"$base/$db/$uidlist\"&gt;";<br>    print " $count &lt;/a&gt;) - $laststr&lt;/p&gt;\n";<br>  }</pre><p>which combines them into PubMed query URLs, one for each author pattern:</p><pre>  https://www.ncbi.nlm.nih.gov/pubmed/1827084,2954879,3020001,...</pre><p>Those
 are then wrapped, along with a record count, in the appropriate HTML 
tags for web display. If the resulting file is opened with a browser, it
 presents an argument-order-dependent view of author collaboration:</p><pre>  ( 55 ) - Berg CM<br><br>  ( 10 ) - Berg CM, Berg DE<br><br>  ( 1 ) - BERG CM, GARBER ED<br><br>  ( 6 ) - Berg DE, Berg CM<br><br>  ( 39 ) - Casadaban MJ<br><br>  ( 6 ) - Casadaban MJ, Groisman EA<br><br>  ( 1 ) - Casadaban MJ, Groisman EA, Berg CM</pre><p>Clicking
 on a hyperlinked record count number opens the document summary or 
individual article page, so the actual publications can be examined.</p></div><div id="chapter6.Indexed_Fields"><h3>Indexed Fields</h3><p>What date fields are indexed for PubMed?</p><pre>  einfo -db pubmed |<br>  xtract -pattern Field \<br>    -if IsDate -equals Y -and IsHidden -equals N \<br>      -pfx "[" -sfx "]" -element Name \<br>      -pfx "" -sfx "" -element FullName |<br>  sort -k 2f | expand</pre><p>This produces a list of field abbreviations and names filtered by index type:</p><pre>  [CDAT]  Date - Completion<br>  [CRDT]  Date - Create<br>  [EDAT]  Date - Entrez<br>  [MHDA]  Date - MeSH<br>  [MDAT]  Date - Modification<br>  [PDAT]  Date - Publication</pre></div><div id="chapter6.Digital_Object_Identifiers"><h3>Digital Object Identifiers</h3><p>How are digital object identifiers obtained from PubMed articles?</p><pre>  esearch -db pubmed -query "Rowley JD [AUTH]" |<br>  efetch -format xml |<br>  xtract -head '&lt;html&gt;&lt;body&gt;' -tail '&lt;/body&gt;&lt;/html&gt;' \<br>    -pattern PubmedArticle -PMID MedlineCitation/PMID \<br>    -block ArticleId -if @IdType -equals doi \<br>    -tab '\n' -pfx '&lt;p&gt;&lt;a href="http://dx.doi.org/' \<br>    -sep '"&gt;' -sfx '&lt;/a&gt;&lt;/p&gt;' -encode ArticleId,"&amp;PMID"</pre><p>extracts the DOIs and constructs the appropriate URL references:</p><pre>  &lt;html&gt;&lt;body&gt;<br>  &lt;p&gt;&lt;a href="http://dx.doi.org/10.1038/leu.2013.340"&gt;24496283&lt;/a&gt;&lt;/p&gt;<br>  &lt;p&gt;&lt;a href="http://dx.doi.org/10.1073/pnas.1310656110"&gt;23818607&lt;/a&gt;&lt;/p&gt;<br>  &lt;p&gt;&lt;a href="http://dx.doi.org/10.1126/science.1241318"&gt;23788787&lt;/a&gt;&lt;/p&gt;<br>  ...</pre><p>These intermediate lines are then piped through:</p><pre>  xtract -format</pre><p>to produce a minimal HTML document with clickable links:</p><pre>  &lt;?xml version="1.0"?&gt;<br>  &lt;!DOCTYPE html&gt;<br>  &lt;html&gt;<br>    &lt;body&gt;<br>      &lt;p&gt;<br>        &lt;a href="http://dx.doi.org/10.1038/leu.2013.340"&gt;24496283&lt;/a&gt;<br>      &lt;/p&gt;<br>      &lt;p&gt;<br>        &lt;a href="http://dx.doi.org/10.1073/pnas.1310656110"&gt;23818607&lt;/a&gt;<br>      &lt;/p&gt;<br>      ...<br>    &lt;/body&gt;<br>  &lt;/html&gt;</pre></div><div id="chapter6.Phrase_Searching"><h3>Phrase Searching</h3><p>Can phrase searching be simulated in Entrez?</p><p>The
 "entrez-phrase-search" script included with EDirect takes advantage of 
the fact that some short phrases are indexed in certain Entrez fields. 
Given an input phrase, the script generates overlapping pairs of 
adjacent words, separately queries on each pair to determine which are 
present in the pubmed title or abstract index, and keeps those that 
appear in at least 10 articles. Independent phrases are separated by a 
plus ("+") sign.</p><p>For example, running the following command:</p><pre>  entrez-phrase-search -db pubmed -field WORD \<br>    selective serotonin reuptake inhibitor + monoamine oxidase inhibitor</pre><p>will generate word pairs from each phrase and run a query on each pair. The individual term counts are:</p><pre>  11343    selective serotonin<br>  11892    serotonin reuptake<br>   6714    reuptake inhibitor<br>  21722    monoamine oxidase<br>   3680    oxidase inhibitor</pre><p>The
 combined query will return a search result with 36 articles, and these 
can then be retrieved by piping to efetch. The script in its current 
form will not match phrases with plurals (e.g., serotonin reuptake 
inhibitors) or hyphens (e.g., monoamine-oxidase inhibitor).</p></div><div id="chapter6.GeneProtein_Links"><h3>Gene-Protein Links</h3><p>What proteins are produced by a given gene?</p><p>Given a query in the gene database, the following commands:</p><pre>  esearch -db gene -query "beta galactosidase [PFN]" |<br>  elink -target protein -name gene_protein_refseq -cmd neighbor |<br>  xtract -pattern LinkSet -element Id</pre><p>will show the gene ID in the first column and linked RefSeq protein UIDs in subsequent columns.</p><p>Piping
 the results to a Perl script named "gene-protein-links.pl" will read 
the identifiers and run separate efetch queries on the gene and protein 
databases:</p><pre>  #!/usr/bin/perl<br><br>  while ($line = &lt;STDIN&gt;) {<br><br>    chomp ($line);<br>    @uids = split( /\t/, $line);<br>    $gene = $uids [0];<br>    $proteins = join (',', @uids [1..$#uids]);<br><br>    $symbol = $data = '';<br><br>    $cmd = "efetch -format docsum -db gene -id $gene | ";<br>    $cmd .= "xtract -pattern DocumentSummary -element Name CommonName";<br>    open (CMD, "$cmd|");<br>    while (&lt;CMD&gt;) {<br>      $symbol .= $_;<br>    }<br>    close CMD;<br><br>    if ($proteins ne "") {<br>      $cmd = "efetch -format docsum -db protein -id $proteins | ";<br>      $cmd .= "xtract -pattern DocumentSummary -element Caption Slen Title";<br>      open (CMD, "$cmd|");<br>      while (&lt;CMD&gt;) {<br>        $data .= $_;<br>      }<br>      close CMD;<br>    }<br><br>    print "$symbol$data\n";<br>  }</pre><p>printing the gene symbol and organism common name, followed by the protein accessions, lengths, and titles:</p><pre>  GLB1            human<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/208022658/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_001129074</a>    546    beta-galactosidase isoform c preproprotein ...<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/119372312/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_001073279</a>    647    beta-galactosidase isoform b [Homo sapiens]<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/1519245746/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_000395</a>       677    beta-galactosidase isoform a preproprotein ...<br><br>  Glb1            house mouse<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/6753190/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_033882</a>       647    beta-galactosidase preproprotein [Mus musculus]<br><br>  Glb1            Norway rat<br>  <a href="https://www.ncbi.nlm.nih.gov/protein/157824103/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">NP_001101662</a>    647    beta-galactosidase precursor [Rattus norvegicus]<br>  ...</pre></div><div id="chapter6.Bulk_Downloads"><h3>Bulk Downloads</h3><p>How can the entire set of GenBank records for mammals be obtained?</p><pre>  ftp-ls ftp.ncbi.nlm.nih.gov genbank |<br>  grep ".seq.gz" |<br>  grep -e gbmam -e gbpri -e gbrod |<br>  while read file<br>  do<br>    ftp-cp ftp.ncbi.nlm.nih.gov genbank "$file"<br>    gzcat "$file"<br>    rm "$file"<br>  done</pre><p>will
 use the ftp-ls and ftp-cp scripts (included with the EDirect software) 
to retrieve and print GenBank flatfiles for human, primate, rodent, and 
other mammals:</p><pre>  GBMAM1.SEQ          Genetic Sequence Data Bank<br>                          February 15 2015<br><br>                  NCBI-GenBank Flat File Release 206.0<br><br>                 Other Mammalian Sequences (Part 1)<br><br>     20709 loci,   155323216 bases, from    20709 reported sequences<br><br><br>  LOCUS       <a href="https://www.ncbi.nlm.nih.gov/nuccore/1783121" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">AB000170</a>                2732 bp    mRNA    linear   MAM ...<br>  DEFINITION  Sus scrofa mRNA for endopeptidase 24.16, complete cds.<br>  ACCESSION   <a href="https://www.ncbi.nlm.nih.gov/nuccore/1783121" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">AB000170</a><br>  VERSION     <a href="https://www.ncbi.nlm.nih.gov/nuccore/1783121" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=nuccore" target="_blank">AB000170.1</a>  GI:1783121<br>  KEYWORDS    endopeptidase 24.16 type M3; endopeptidase 24.16 type M1.<br>  SOURCE      Sus scrofa (pig)<br>    ORGANISM  Sus scrofa<br>              Eukaryota; Metazoa; Chordata; Craniata; Vertebrata;<br>              Euteleostomi; Mammalia; Eutheria; Laurasiatheria;<br>              Cetartiodactyla; Suina; Suidae; Sus.<br>  REFERENCE   1<br>    AUTHORS   Kato,A., Sugiura,N., Saruta,Y., Hosoiri,T., Yasue,H. and<br>              Hirose,S.<br>    TITLE     Targeting of endopeptidase 24.16 to different subcellular<br>              compartments by alternative promoter usage<br>    JOURNAL   J. Biol. Chem. 272 (24), 15313-15322 (1997)<br>     PUBMED   9182559<br>  REFERENCE   2  (bases 1 to 2732)<br>    AUTHORS   Hirose,S.<br>    TITLE     Direct Submission<br>    JOURNAL   Submitted (27-DEC-1996) Shigehisa Hirose, Tokyo Institute of<br>              Technology, Department of Biological Sciences; 4259<br>              Nagatsuta-cho, Midori-ku, Yokohama, Kanagawa 226-8501, Japan<br>  FEATURES             Location/Qualifiers<br>       source          1..2732<br>                       /organism="Sus scrofa"<br>                       /mol_type="mRNA"<br>                       /db_xref="taxon:9823"<br>                       /tissue_type="Liver"<br>                       /dev_stage="Adult"<br>                       /note="porcine"<br>       mRNA            1..2732<br>                       /note="corresponding to exon1,5-16 of this gene;<br>                       endopeptidase 24.16 type 1"<br>       CDS             175..2289<br>                       /standard_name="endopeptidase 24.16"<br>                       /note="oligopeptidase M :neurolysin :sBAP(soluble<br>                       angiotensin-binding protein) :MEP(microsomal<br>                       metalloendopeptidase)"<br>                       /codon_start=1<br>                       /product="endopeptidase 24.16 type M1"<br>                       /protein_id="<a href="https://www.ncbi.nlm.nih.gov/protein/1783122/?report=GenPept" class="bk_tag" ref="pagearea=body&amp;targetsite=entrez&amp;targetcat=link&amp;targettype=genpept" target="_blank">BAA19060.1</a>"<br>                       /db_xref="GI:1783122"<br>                       ...</pre><p>For systems with Aspera Connect installed, the asp-ls and asp-cp scripts can be used for faster retrieval:</p><pre>  asp-ls genbank |<br>  grep ".seq.gz" |<br>  grep -e gbmam -e gbpri -e gbrod |<br>  while read file<br>  do<br>    asp-cp genbank "$file"<br>    gzcat "$file"<br>    rm "$file"<br>  done</pre></div></div><div id="chapter6.Appendices"><h2 id="_chapter6_Appendices_">Appendices</h2><div id="chapter6.Setting_Contact_Address_and_Scr"><h3>Setting Contact Address and Script Name</h3><p>EDirect
 automatically obtains the user's e-mail address from the system, to 
have someone to notify in case a runaway script causes problems with an 
Entrez server, but if another contact address is desired (e.g., that of a
 system administrator or software developer) it can be explicitly set at
 the beginning of a pipeline or script:</p><pre>  econtact -email author_email_address -tool name_of_script</pre><p>That
 way the NCBI has information on who to contact if an infinite loop in a
 script accidentally abuses NCBI resources. (For convenience, the 
preferred e-mail address and software tool name can also be set in all 
E-utilities-calling operations.)</p></div><div id="chapter6.CommandLine_Arguments"><h3>Command-Line Arguments</h3><p>Arguments for the EDirect functions are listed below:</p><p>Use esearch to start a new Entrez search on indexed terms:</p><pre>  Query Specification<br><br>    -db          Database name<br>    -query       Query string<br><br>  Document Order<br><br>    -sort        Result presentation order<br><br>  Date Constraint<br><br>    -days        Number of days in the past<br>    -datetype    Date field abbreviation<br>    -mindate     Start of date range<br>    -maxdate     End of date range<br><br>  Limit by Field<br><br>    -field       Query words individually in field<br>    -pairs       Query overlapping word pairs<br><br>  Spell Check<br><br>    -spell       Correct misspellings in query<br><br>  Miscellaneous Arguments<br><br>    -label       Alias for query step</pre><p>The elink function looks up related articles or associated records:</p><pre>  Destination Database<br><br>    -related    Neighbors in same database<br>    -target     Links in different database<br>    -name       Link name (e.g., pubmed_protein_refseq)<br><br>  Direct Record Selection<br><br>    -db         Database name<br>    -id         Unique identifier(s)<br><br>  Advanced Control<br><br>    -cmd        Command type (returns eLinkResult XML)<br>    -mode       "ref" uses LinkOut provider's web site<br>    -holding    Name of LinkOut provider<br><br>  Batch Processing<br><br>    -batch      Bypass Entrez history mechanism<br><br>  Miscellaneous Arguments<br><br>    -label      Alias for query step</pre><p>Use efilter to restrict search or link results by indexed terms:</p><pre>  Query Specification<br><br>    -query       Query string<br><br>  Document Order<br><br>    -sort        Result presentation order<br><br>  Date Constraint<br><br>    -days        Number of days in the past<br>    -datetype    Date field abbreviation<br>    -mindate     Start of date range<br>    -maxdate     End of date range<br><br>  Limit by Field<br><br>    -field       Query words individually in field<br>    -pairs       Query overlapping word pairs<br><br>  Spell Check<br><br>    -spell       Correct misspellings in query<br><br>  Publication Filters<br><br>    -pub         abstract, clinical, english, free, historical,<br>                 journal, medline, preprint, published, review,<br>                 structured<br>    -journal     pnas, "j bacteriol", ...<br>    -released    last_week, last_month, last_year, prev_years<br><br>  Sequence Filters<br><br>    -country     usa:minnesota, united_kingdom, "pacific ocean", ...<br>    -feature     gene, mrna, cds, mat_peptide, ...<br>    -location    mitochondrion, chloroplast, plasmid, plastid<br>    -molecule    genomic, mrna, trna, rrna, ncrna<br>    -organism    animals, archaea, bacteria, eukaryotes, fungi,<br>                 human, insects, mammals, plants, prokaryotes,<br>                 protists, rodents, viruses<br>    -source      genbank, insd, pdb, pir, refseq, swissprot, tpa<br><br>  Gene Filters<br>    -status      alive<br>    -type        coding, pseudo<br><br>  SNP Filters<br><br>    -class       acceptor, donor, frameshift, indel, intron,<br>                 missense, nonsense, synonymous<br><br>  Miscellaneous Arguments<br><br>    -label       Alias for query step</pre><p>The record retrieval function is efetch:</p><pre>  Format Selection<br><br>    -format        Format of record or report<br>    -mode          text, xml, asn.1, json<br>    -style         withparts, conwithfeat<br><br>  Direct Record Selection<br><br>    -db            Database name<br>    -id            Unique identifier or accession number<br><br>  Sequence Range<br><br>    -seq_start     First sequence position to retrieve<br>    -seq_stop      Last sequence position to retrieve<br>    -strand        Strand of DNA to retrieve<br><br>  Gene Range<br><br>    -chr_start     Sequence range from 0-based coordinates<br>    -chr_stop        in gene docsum GenomicInfoType object<br><br>  Sequence Flags<br><br>    -complexity    0 = default, 1 = bioseq, 3 = nuc-prot set<br>    -extend        Extend sequence retrieval in both directions<br>    -extrafeat     Bit flag specifying extra features<br><br>  Miscellaneous<br><br>    -raw           Skip database-specific XML modifications<br>    -json          Convert adjusted XML output to JSON</pre><p>The xtract function is used for processing XML data:</p><pre>  Processing Flags<br><br>    -strict          Remove HTML and MathML tags<br>    -mixed           Allow mixed content XML<br><br>    -accent          Excise Unicode accents and diacritical marks<br>    -ascii           Unicode to numeric HTML character entities<br>    -compress        Compress runs of spaces<br><br>    -stops           Retain stop words in selected phrases<br><br>  Data Source<br><br>    -input           Read XML from file instead of stdin<br>    -transform       File of substitutions for -translate<br><br>  Exploration Argument Hierarchy<br><br>    -pattern         Name of record within set<br>    -group             Use of different argument<br>    -block               names allows command-line<br>    -subset                control of nested looping<br><br>  Exploration Constructs<br><br>    Object           DateRevised<br>    Parent/Child     Book/AuthorList<br>    Heterogeneous    "PubmedArticleSet/*"<br>    Exhaustive       "History/**"<br>    Nested           "*/Taxon"<br>    Recursive        "**/Gene-commentary"<br><br>  Conditional Execution<br><br>    -if              Element [@attribute] required<br>    -unless          Skip if element matches<br>    -and             All tests must pass<br>    -or              Any passing test suffices<br>    -else            Execute if conditional test failed<br>    -position        [first|last|outer|inner|even|odd|all]<br>    -select          Select record subset by conditions<br><br>  String Constraints<br><br>    -equals          String must match exactly<br>    -contains        Substring must be present<br>    -is-within       String must be present<br>    -starts-with     Substring must be at beginning<br>    -ends-with       Substring must be at end<br>    -is-not          String must not match<br><br>  Numeric Constraints<br><br>    -gt              Greater than<br>    -ge              Greater than or equal to<br>    -lt              Less than<br>    -le              Less than or equal to<br>    -eq              Equal to<br>    -ne              Not equal to<br><br>  Format Customization<br><br>    -ret             Override line break between patterns<br>    -tab             Replace tab character between fields<br>    -sep             Separator between group members<br>    -pfx             Prefix to print before group<br>    -sfx             Suffix to print after group<br>    -plg             Prologue to print once before elements<br>    -elg             Epilogue to print once after elements<br>    -rst             Reset -sep through -elg<br>    -clr             Clear queued tab separator<br>    -pfc             Preface combines -clr and -pfx<br>    -deq             Delete and replace queued tab separator<br>    -wrp             Wrap elements in XML object<br>    -def             Default placeholder for missing fields<br>    -lbl             Insert arbitrary text<br><br>  Element Selection<br><br>    -element         Print all items that match tag name<br>    -first           Only print value of first item<br>    -last            Only print value of last item<br>    -NAME            Record value in named variable<br><br>  -element Constructs<br><br>    Tag              Caption<br>    Group            Initials,LastName<br>    Parent/Child     MedlineCitation/PMID<br>    Recursive        "**/Gene-commentary_accession"<br>    Unrestricted     "PubDate/*"<br>    Attribute        DescriptorName@MajorTopicYN<br>    Range            MedlineDate[1:4]<br>    Substring        "Title[phospholipase | rattlesnake]"<br>    Object Count     "#Author"<br>    Item Length      "%Title"<br>    Element Depth    "^PMID"<br>    Variable         "&amp;NAME"<br><br>  Special -element Operations<br><br>    Parent Index     "+"<br>    Object Name      "?"<br>    XML Subtree      "*"<br>    Children         "$"<br>    Attributes       "@"<br><br>  Numeric Processing<br><br>    -num             Count<br>    -len             Length<br>    -sum             Sum<br>    -min             Minimum<br>    -max             Maximum<br>    -inc             Increment<br>    -dec             Decrement<br>    -sub             Difference<br>    -avg             Average<br>    -dev             Deviation<br>    -med             Median<br>    -bin             Binary<br>    -bit             Bit Count<br><br>  String Processing<br><br>    -encode          URL-encode &lt;, &gt;, &amp;, ", and ' characters<br>    -upper           Convert text to upper-case<br>    -lower           Convert text to lower-case<br>    -title           Capitalize initial letters of words<br>    -year            Extract first 4-digit year from string<br>    -translate       Substitute values with -transform table<br><br>  Text Processing<br><br>    -terms           Partition text at spaces<br>    -words           Split at punctuation marks<br>    -pairs           Adjacent informative words<br>    -reverse         Reverse words in string<br>    -letters         Separate individual letters<br>    -clauses         Break at phrase separators<br>    -indices         Index normalized words<br><br>  Sequence Processing<br><br>    -revcomp         Reverse-complement nucleotide sequence<br>    -nucleic         Subrange determines forward or revcomp<br><br>  Sequence Coordinates<br><br>    -0-based         Zero-Based<br>    -1-based         One-Based<br>    -ucsc-based      Half-Open<br><br>  Command Generator<br><br>    -insd            Generate INSDSeq extraction commands<br><br>  -insd Argument Order<br><br>    Descriptors      INSDSeq_sequence INSDSeq_definition INSDSeq_division<br>    Flags            [complete|partial]<br>    Feature(s)       CDS,mRNA<br>    Qualifiers       INSDFeature_key "#INSDInterval" gene product<br><br>  Miscellaneous<br><br>    -head            Print before everything else<br>    -tail            Print after everything else<br>    -hd              Print before each record<br>    -tl              Print after each record<br><br>  Reformatting<br><br>    -format          [copy|compact|flush|indent|expand]<br><br>  Modification<br><br>    -filter          Object<br>                       [retain|remove|encode|decode|shrink|expand|accent]<br>                         [content|cdata|comment|object|attributes|container]<br><br>  Validation<br><br>    -verify          Report XML data integrity problems<br><br>  Summary<br><br>    -outline         Display outline of XML structure<br>    -synopsis        Display count of unique XML paths<br><br>  Documentation<br><br>    -examples        Examples of EDirect and xtract usage</pre><p>The einfo function returns information on Entrez indexed fields:</p><pre>  Database Selection<br><br>    -db        Database name<br>    -dbs       Get all database names<br><br>  Data Summaries<br><br>    -fields    Print field names<br>    -links     Print link names</pre><p>Several additional functions are provided by EDirect:</p><pre>  epost<br><br>    -db        Database name<br>    -id        Unique identifier(s) or accession number(s)<br>    -format    uid or acc<br>    -input     Read from file instead of stdin<br>    -label     Alias for query step<br><br>  eproxy<br><br>    -alias     File of aliases<br>    -pipe      Read aliases from stdin<br><br>  econtact<br><br>    -email     Contact person's address<br>    -tool      Name of script or program<br><br>  nquire<br><br>    -get       Uses HTTP GET instead of POST<br>    -url       Base URL for external search</pre><p>In
 addition, -email and -tool are available in all E-utilities-calling 
functions to override default values, -http get will force the use of 
GET instead of POST, -alias will specify a file of shortcut keywords and
 query strings or URL sections, and -help will print the list of 
arguments for each function.</p><p>For debugging, -silent will suppress 
link failure retry messages, -verbose will display the 
&lt;ENTREZ_DIRECT&gt; field values at each step, -debug will print the 
internal URL query and XML results of each step, and -base will specify a
 particular server for quality assurance testing.</p></div><div id="chapter6.EFetch_Formats"><h3>EFetch Formats</h3><p>EFetch -format and -mode values for each database are shown below:</p><pre>  -db            -format            -mode    Report Type<br>  ___            _______            _____    ___________<br><br>  (all)<br>                 docsum                      DocumentSummarySet XML<br>                 docsum             json     DocumentSummarySet JSON<br>                 full                        Same as native except for mesh<br>                 uid                         Unique Identifier List<br>                 url                         Entrez URL<br>                 xml                         Same as -format full -mode xml<br><br>  bioproject<br>                 native                      BioProject Report<br>                 native             xml      RecordSet XML<br><br>  biosample<br>                 native                      BioSample Report<br>                 native             xml      BioSampleSet XML<br><br>  biosystems<br>                 native             xml      Sys-set XML<br><br>  gds<br>                 native             xml      RecordSet XML<br>                 summary                     Summary<br><br>  gene<br>                 full_report                 Detailed Report<br>                 gene_table                  Gene Table<br>                 native                      Gene Report<br>                 native             asn.1    Entrezgene ASN.1<br>                 native             xml      Entrezgene-Set XML<br>                 tabular                     Tabular Report<br><br>  homologene<br>                 alignmentscores             Alignment Scores<br>                 fasta                       FASTA<br>                 homologene                  Homologene Report<br>                 native                      Homologene List<br>                 native             asn.1    HG-Entry ASN.1<br>                 native             xml      Entrez-Homologene-Set XML<br><br>  mesh<br>                 full                        Full Record<br>                 native                      MeSH Report<br>                 native             xml      RecordSet XML<br><br>  nlmcatalog<br>                 native                      Full Record<br>                 native             xml      NLMCatalogRecordSet XML<br><br>  pmc<br>                 medline                     MEDLINE<br>                 native             xml      pmc-articleset XML<br><br>  pubmed<br>                 abstract                    Abstract<br>                 medline                     MEDLINE<br>                 native             asn.1    Pubmed-entry ASN.1<br>                 native             xml      PubmedArticleSet XML<br><br>  (sequences)<br>                 acc                         Accession Number<br>                 est                         EST Report<br>                 fasta                       FASTA<br>                 fasta              xml      TinySeq XML<br>                 fasta_cds_aa                FASTA of CDS Products<br>                 fasta_cds_na                FASTA of Coding Regions<br>                 ft                          Feature Table<br>                 gb                          GenBank Flatfile<br>                 gb                 xml      GBSet XML<br>                 gbc                xml      INSDSet XML<br>                 gene_fasta                  FASTA of Gene<br>                 gp                          GenPept Flatfile<br>                 gp                 xml      GBSet XML<br>                 gpc                xml      INSDSet XML<br>                 gss                         GSS Report<br>                 ipg                         Identical Protein Report<br>                 ipg                xml      IPGReportSet XML<br>                 native             text     Seq-entry ASN.1<br>                 native             xml      Bioseq-set XML<br>                 seqid                       Seq-id ASN.1<br><br>  snp<br>                 chr                         Chromosome Report<br>                 docset                      Summary<br>                 fasta                       FASTA<br>                 flt                         Flat File<br>                 native             asn.1    Rs ASN.1<br>                 native             xml      ExchangeSet XML<br>                 rsr                         RS Cluster Report<br>                 ssexemplar                  SS Exemplar List<br><br>  sra<br>                 native             xml      EXPERIMENT_PACKAGE_SET XML<br>                 runinfo            xml      SraRunInfo XML<br><br>  structure<br>                 mmdb                        Ncbi-mime-asn1 strucseq ASN.1<br>                 native                      MMDB Report<br>                 native             xml      RecordSet XML<br><br>  taxonomy<br>                 native                      Taxonomy List<br>                 native             xml      TaxaSet XML</pre></div><div id="chapter6.ESearch_Sort_Order"><h3>ESearch Sort Order</h3><p>ESearch -sort values for several databases are listed below:</p><pre>  -db            -sort<br>  ___            _____<br><br>  gene<br>                 Chromosome<br>                 Gene Weight<br>                 Name<br>                 Relevance<br><br>  geoprofiles<br>                 Default Order<br>                 Deviation<br>                 Mean Value<br>                 Outliers<br>                 Subgroup Effect<br><br>  pubmed<br>                 First Author<br>                 Journal<br>                 Last Author<br>                 Pub Date<br>                 Recently Added<br>                 Relevance<br>                 Title<br><br>  (sequences)<br>                 Accession<br>                 Date Modified<br>                 Date Released<br>                 Default Order<br>                 Organism Name<br>                 Taxonomy ID<br><br>  snp<br>                 Chromosome Base Position<br>                 Default Order<br>                 Heterozygosity<br>                 Organism<br>                 SNP_ID<br>                 Success Rate</pre></div><div id="chapter6.ELink_Commands"><h3>ELink Commands</h3><p>ELink -cmd options produce results as LinkSet XML:</p><pre>  -cmd              Result<br>  ____              ______<br><br>  neighbor          Neighbors or links<br><br>  neighbor_score    Neighbors with computed similarity scores<br><br>  acheck            All links available<br><br>  ncheck            Existence of neighbors<br><br>  lcheck            Existence of external links (LinkOuts)<br><br>  llinks            Non-library LinkOut providers<br><br>  llinkslib         All LinkOut providers<br><br>  prlinks           Primary LinkOut provider,<br>                    or URL for single UID with -mode ref</pre></div><div id="chapter6.EInfo_Data"><h3>EInfo Data</h3><p>EInfo field data contains status flags for several term list index properties:</p><pre>  &lt;Field&gt;<br>    &lt;Name&gt;ALL&lt;/Name&gt;<br>    &lt;FullName&gt;All Fields&lt;/FullName&gt;<br>    &lt;Description&gt;All terms from all searchable fields&lt;/Description&gt;<br>    &lt;TermCount&gt;138982028&lt;/TermCount&gt;<br>    &lt;IsDate&gt;N&lt;/IsDate&gt;<br>    &lt;IsNumerical&gt;N&lt;/IsNumerical&gt;<br>    &lt;SingleToken&gt;N&lt;/SingleToken&gt;<br>    &lt;Hierarchy&gt;N&lt;/Hierarchy&gt;<br>    &lt;IsHidden&gt;N&lt;/IsHidden&gt;<br>    &lt;IsTruncatable&gt;Y&lt;/IsTruncatable&gt;<br>    &lt;IsRangable&gt;N&lt;/IsRangable&gt;<br>  &lt;/Field&gt;</pre></div><div id="chapter6.UNIX_Utilities"><h3>UNIX Utilities</h3><p>Several useful classes of UNIX text processing filters, with selected arguments, are presented below:</p><p>Process by Contents:</p><pre>  sort    Sorts lines of text<br><br>    -f    Ignore case<br>    -n    Numeric comparison<br>    -r    Reverse result order<br><br>    -k    Field key (start,stop or first)<br>    -u    Unique lines with identical keys<br><br>    -b    Ignore leading blanks<br>    -s    Stable sort<br>    -t    Specify field separator<br><br>  uniq    Removes repeated lines<br><br>    -c    Count occurrences<br>    -i    Ignore case<br><br>    -f    Ignore first n fields<br>    -s    Ignore first n characters<br><br>    -d    Only output repeated lines<br>    -u    Only output non-repeated lines<br><br>  grep    Matches patterns using regular expressions<br><br>    -i    Ignore case<br>    -v    Invert search<br>    -w    Search expression as a word<br>    -x    Search expression as whole line<br><br>    -e    Specify individual pattern<br><br>    -c    Only count number of matches<br>    -n    Print line numbers</pre><p>Regular Expressions:</p><pre>  Characters<br><br>    .     Any single character (except newline)<br>    \w    Alphabetic [A-Za-z], numeric [0-9], or underscore (_)<br>    \s    Whitespace (space or tab)<br>    \     Escapes special characters<br>    []    Matches any enclosed characters<br><br>  Positions<br><br>    ^     Beginning of line<br>    $     End of line<br>    \b    Word boundary<br><br>  Repeat Matches<br><br>    ?     0 or 1<br>    *     0 or more<br>    +     1 or more<br>    {n}   Exactly n</pre><p>Modify Contents:</p><pre>  sed     Replaces text strings<br><br>    -e    Specify individual expression<br><br>  tr      Translates characters<br><br>    -d    Delete character<br><br>  rev     Reverses characters on line</pre><p>Format Contents:</p><pre>  column  Aligns columns by content width<br><br>    -s    Specify field separator<br>    -t    Create table<br><br>  expand  Aligns columns to specified positions<br><br>    -t    Tab positions<br><br>  fold    Wraps lines at a specific width<br><br>    -w    Line width</pre><p>Filter by Position:</p><pre>  cut     Removes parts of lines<br><br>    -c    Characters to keep<br>    -f    Fields to keep<br>    -d    Specify field separator<br><br>    -s    Suppress lines with no delimiters<br><br>  head    Prints first lines<br><br>    -n    Number of lines<br><br>  tail    Prints last lines<br><br>    -n    Number of lines</pre><p>Miscellaneous:</p><pre>  wc      Counts words, lines, or characters<br><br>    -c    Characters<br>    -l    Lines<br>    -w    Words<br><br>  xargs   Constructs arguments<br><br>    -n    Number of words per batch</pre><p>File Compression:</p><pre>  tar     Archive files<br><br>    -c    Create archive<br>    -f    Name of output file<br>    -z    Compress archive with gzip<br><br>  gzip    Compress file<br><br>    -k    Keep original file<br>    -9    Best compression<br><br>  unzip   Decompress .zip archive<br><br>    -p    Pipe to stdout<br><br>  gzcat   Decompress .gz archive and pipe to stdout</pre><p>Directory and File Navigation:</p><pre>  cd      Changes directory<br><br>    /     Root<br>    ~     Home<br>    .     Current<br>    ..    Parent<br>    -     Previous<br><br>  ls      Lists file names<br><br>    -1    One entry per line<br>    -a    Show files beginning with dot (.)<br>    -l    List in long format<br>    -R    Recursively explore subdirectories<br>    -S    Sort files by size<br>    -t    Sort by most recently modified<br><br>  pwd     Prints working directory path</pre><p>Additional documentation with detailed explanations and examples can be obtained by typing "man" followed by a command name.</p></div><div id="chapter6.Terminal_Keyboard_Shortcuts"><h3>Terminal Keyboard Shortcuts</h3><p>Control
 and escape sequences can be used within a terminal session to navigate 
through the command history and to move the cursor for editing the 
command currently being entered:</p><p>Command history:</p><pre>  Ctrl-n     Next command<br>  Ctrl-p     Previous command</pre><p>Move cursor forward:</p><pre>  Ctrl-e     To end of line<br>  Ctrl-f     By one character<br>  Esc-f      By one argument</pre><p>Move cursor backward:</p><pre>  Ctrl-a     To beginning of line<br>  Ctrl-b     By one character<br>  Esc-b      By one argument</pre><p>Delete:</p><pre>  Del        Previous character<br>  Ctrl-d     Next character<br>  Ctrl-k     To end of line<br>  Ctrl-u     Entire line<br>  Ctrl-w     Previous word<br>  Esc-Del    Previous argument<br>  Esc-d      Next argument</pre><p>Autocomplete:</p><pre>  Tab        Completes directory or file names</pre><p>Program control:</p><pre>  Ctrl-c     Quit running program<br>  ^x^y       Run last command replacing x with y</pre><p>(Note
 that Control sequences are typed by holding down Control, hitting the 
other key, and releasing Control, while Escape sequences are typed by 
hitting Escape and then hitting the other key.)</p></div></div><div id="chapter6.Release_Notes"><h2 id="_chapter6_Release_Notes_">Release Notes</h2><div id="chapter6.EDirect_Version_10_9_February_1"><h3>EDirect Version 10.9: February 1, 2019</h3><ul><li class="half_rhythm"><div>Xtract -insd supports a sub_sequence qualifier that uses -nucleic and produces upper-case sequence.</div></li><li class="half_rhythm"><div>Xtract now has an -is-within string conditional test.</div></li></ul></div><div id="chapter6.EDirect_Version_10_8_January_20"><h3>EDirect Version 10.8: January 20, 2019</h3><ul><li class="half_rhythm"><div>EDIRECT_DO_AUTO_ABBREV environment variable restores relaxed matching of command-line arguments.</div></li><li class="half_rhythm"><div>Efilter shortcut -journal added for -db pubmed.</div></li><li class="half_rhythm"><div>Efilter -pub last_* shortcuts moved to -released.</div></li><li class="half_rhythm"><div>Efilter -pub and -feature can take comma-separate list of choices.</div></li><li class="half_rhythm"><div>Transmute -docsum command added.</div></li><li class="half_rhythm"><div>Transmute -decode and -encode commands renamed to -unescape and -escape.</div></li><li class="half_rhythm"><div>Transmute -decode64 and -encode64 commands added.</div></li></ul></div><div id="chapter6.EDirect_Version_10_7_January_14"><h3>EDirect Version 10.7: January 14, 2019</h3><ul><li class="half_rhythm"><div>Xtract -nucleic uses bracketed range direction to determine whether to reverse complement the sequence.</div></li></ul></div><div id="chapter6.EDirect_Version_10_6_December_1"><h3>EDirect Version 10.6: December 13, 2018</h3><ul><li class="half_rhythm"><div>Local archive script creates command for saving data path in configuration file.</div></li><li class="half_rhythm"><div>Xtract -reverse returns -words output in reverse order.</div></li><li class="half_rhythm"><div>Efilter shortcut added for -db snp (e.g., -class missense).</div></li><li class="half_rhythm"><div>Efetch -format gbc (INSDSeq XML) supports -style withparts and -style conwithfeat.</div></li></ul></div><div id="chapter6.EDirect_Version_10_5_December_4"><h3>EDirect Version 10.5: December 4, 2018</h3><ul><li class="half_rhythm"><div>EDirect commands and pipelines support faster access with API keys.</div></li><li class="half_rhythm"><div>Xtract attributes can be delimited by quotation marks or apostrophes.</div></li><li class="half_rhythm"><div>Transmute -encode and -decode commands added.</div></li><li class="half_rhythm"><div>Simplified processing of local inverted index intermediate files.</div></li></ul></div><div id="chapter6.EDirect_Version_10_4_November_1"><h3>EDirect Version 10.4: November 13, 2018</h3><ul><li class="half_rhythm"><div>Rchive local indexing code refactored for faster performance.</div></li><li class="half_rhythm"><div>Xtract -deq deletes and replaces queued tab separator after the fact.</div></li><li class="half_rhythm"><div>Efilter -organism queries in [ORGN] field if argument is not in shortcut list.</div></li></ul></div><div id="chapter6.EDirect_Version_10_3_November_1"><h3>EDirect Version 10.3: November 1, 2018</h3><ul><li class="half_rhythm"><div>Rchive -invert, -merge, -promote, and -query steps make better use of multiple processor cores.</div></li><li class="half_rhythm"><div>New phrase-search script replaces local-phrase-search.</div></li></ul></div><div id="chapter6.EDirect_Version_10_2_October_15"><h3>EDirect Version 10.2: October 15, 2018</h3><ul><li class="half_rhythm"><div>Transmute -x2j joins -j2x to simplify the use of JSON-based services.</div></li><li class="half_rhythm"><div>Efetch -json converts adjusted XML output to JSON as a convenience.</div></li><li class="half_rhythm"><div>Xtract tag alphabet expanded to accommodate converted JSON data.</div></li><li class="half_rhythm"><div>Nquire -ftp takes server, directory, and filename arguments, sends data to stdout.</div></li></ul></div><div id="chapter6.EDirect_Version_10_1_October_9"><h3>EDirect Version 10.1: October 9, 2018</h3><ul><li class="half_rhythm"><div>Xtract -mixed improves support for mixed-content XML.</div></li></ul></div><div id="chapter6.EDirect_Version_10_0_September"><h3>EDirect Version 10.0: September 27, 2018</h3><ul><li class="half_rhythm"><div>Efilter can search for sequence records by sample collection location (e.g., -country "canada new brunswick").</div></li><li class="half_rhythm"><div>Xtract parsing code was refactored in preparation for improvements in handling mixed-content XML data.</div></li><li class="half_rhythm"><div>Added transmute script for format conversions (e.g., -j2x for JSON to XML).</div></li></ul></div><div id="chapter6.EDirect_Version_9_90_September"><h3>EDirect Version 9.90: September 17, 2018</h3><ul><li class="half_rhythm"><div>Normalized archive path for low-value PMIDs in preparation for incremental indexing.</div></li></ul></div><div id="chapter6.EDirect_Version_9_80_September"><h3>EDirect Version 9.80: September 4, 2018</h3><ul><li class="half_rhythm"><div>Xtract XML block reader can run on separate thread for improved performance on computers with surplus processor cores.</div></li><li class="half_rhythm"><div>Fixed bug in string cleanup when text starts with a non-ASCII Unicode character.</div></li><li class="half_rhythm"><div>Efetch regular expression pattern for detecting mixed-content tags was adjusted.</div></li></ul></div><div id="chapter6.EDirect_Version_9_70_August_22"><h3>EDirect Version 9.70: August 22, 2018</h3><ul><li class="half_rhythm"><div>Local archive builds parallel stemmed and non-stemmed indices of terms in the title and abstract.</div></li><li class="half_rhythm"><div>Rchive and local-phrase-search use -query for evaluation of non-stemmed terms, -search for evaluation using the stemmed index.</div></li></ul></div><div id="chapter6.EDirect_Version_9_60_August_9_2"><h3>EDirect Version 9.60: August 9, 2018</h3><ul><li class="half_rhythm"><div>Local archive script removes newlines inside PubMed text fields.</div></li><li class="half_rhythm"><div>Efetch adds missing newline at end of PubmedArticleSet XML.</div></li></ul></div><div id="chapter6.EDirect_Version_9_50_July_30_20"><h3>EDirect Version 9.50: July 30, 2018</h3><ul><li class="half_rhythm"><div>Local indexing scripts adjusted to accommodate projected range of PMID values.</div></li><li class="half_rhythm"><div>Fixed inconsistency in positional indexing of terms with embedded non-alphanumeric characters.</div></li><li class="half_rhythm"><div>EDIRECT_PUBMED_WORKING environment variable keeps local archive intermediate files on a separate volume.</div></li><li class="half_rhythm"><div>Rchive and local-phrase-search use -exact to round-trip ArticleTitle contents without interpretation as a query formula.</div></li></ul></div><div id="chapter6.EDirect_Version_9_40_July_18_20"><h3>EDirect Version 9.40: July 18, 2018</h3><ul><li class="half_rhythm"><div>Xtract handles misplaced spaces in attributes.</div></li><li class="half_rhythm"><div>Xtract -format repairs misplaced spaces in attributes.</div></li></ul></div><div id="chapter6.EDirect_Version_9_30_July_9_201"><h3>EDirect Version 9.30: July 9, 2018</h3><ul><li class="half_rhythm"><div>Local data indexing retains intermediate products, allows rapid streaming of non-redundant current records.</div></li><li class="half_rhythm"><div>Index preparation removes apostrophe in trailing 's possessives.</div></li><li class="half_rhythm"><div>Wildcard minimum varies with prefix-driven posting character depth.</div></li></ul></div><div id="chapter6.EDirect_Version_9_20_June_26_20"><h3>EDirect Version 9.20: June 26, 2018</h3><ul><li class="half_rhythm"><div>Portability and efficiency improvements to local data cache scripts.</div></li><li class="half_rhythm"><div>Xtract handles misplaced spaces in self-closing tags.</div></li></ul></div><div id="chapter6.EDirect_Version_9_10_June_18_20"><h3>EDirect Version 9.10: June 18, 2018</h3><ul><li class="half_rhythm"><div>Added Parent/* element exploration construct to xtract.</div></li><li class="half_rhythm"><div>Xtract -year reliably obtains the year from "PubDate/*".</div></li></ul></div><div id="chapter6.EDirect_Version_9_00_June_6_201"><h3>EDirect Version 9.00: June 6, 2018</h3><ul><li class="half_rhythm"><div>Fetch-pubmed -path supplies missing Archive directory if root path is given.</div></li><li class="half_rhythm"><div>Efetch cleanup of MathML markup properly handles parentheses.</div></li></ul></div><div id="chapter6.EDirect_Version_8_90_June_4_201"><h3>EDirect Version 8.90: June 4, 2018</h3><ul><li class="half_rhythm"><div>Xtract -transform and -translate allow data value substitution.</div></li><li class="half_rhythm"><div>Xtract -wrp simplifies wrapping of extracted values in XML tags.</div></li></ul></div><div id="chapter6.EDirect_Version_8_80_May_29_201"><h3>EDirect Version 8.80: May 29, 2018</h3><ul><li class="half_rhythm"><div>Efetch removes MathML tags from PubmedArticle XML contents, unless the -raw flag is used.</div></li></ul></div><div id="chapter6.EDirect_Version_8_70_May_14_201"><h3>EDirect Version 8.70: May 14, 2018</h3><ul><li class="half_rhythm"><div>Local phrase indexing now uses positional indices instead of adjacent overlapping word pairs.</div></li><li class="half_rhythm"><div>Xtract -select uses conditional expressions to filter records.</div></li></ul></div><div id="chapter6.EDirect_Version_8_60_April_26_2"><h3>EDirect Version 8.60: April 26, 2018</h3><ul><li class="half_rhythm"><div>Efetch -format uid pauses between groups, retries on failure.</div></li><li class="half_rhythm"><div>Fetch delay drops from 1/3 to 1/10 second if API key is used.</div></li><li class="half_rhythm"><div>Local phrase indexing uses smaller files to avoid memory contention.</div></li><li class="half_rhythm"><div>Phrase index removes hyphens from selected prefixes.</div></li></ul></div><div id="chapter6.EDirect_Version_8_50_April_13_2"><h3>EDirect Version 8.50: April 13, 2018</h3><ul><li class="half_rhythm"><div>Efetch markup tag removal modified after change in server.</div></li><li class="half_rhythm"><div>Xtract -phrase filter split into -require and -exclude commands.</div></li></ul></div><div id="chapter6.EDirect_Version_8_40_April_9_20"><h3>EDirect Version 8.40: April 9, 2018</h3><ul><li class="half_rhythm"><div>Efetch removes markup tags in all PubMed XML.</div></li><li class="half_rhythm"><div>Xtract without -strict prints warnings if markup tags are encountered.</div></li><li class="half_rhythm"><div>Xtract proximity search moved from -matches to -phrase.</div></li></ul></div><div id="chapter6.EDirect_Version_8_30_April_4_20"><h3>EDirect Version 8.30: April 4, 2018</h3><ul><li class="half_rhythm"><div>Xtract is now available for ARM processors.</div></li></ul></div><div id="chapter6.EDirect_Version_8_20_March_12_2"><h3>EDirect Version 8.20: March 12, 2018</h3><ul><li class="half_rhythm"><div>Minor changes to local record archiving scripts.</div></li></ul></div><div id="chapter6.EDirect_Version_8_10_March_2_20"><h3>EDirect Version 8.10: March 2, 2018</h3><ul><li class="half_rhythm"><div>Xtract -strict and -mixed support MathML element tags in PubmedArticle XML.</div></li></ul></div><div id="chapter6.EDirect_Version_8_00_February_2"><h3>EDirect Version 8.00: February 26, 2018</h3><ul><li class="half_rhythm"><div>Efetch -raw skips database-specific XML modifications.</div></li><li class="half_rhythm"><div>Added local-phrase-search script.</div></li><li class="half_rhythm"><div>Xtract -strict, -mixed, and -repair flag speed improvements.</div></li></ul></div><div id="chapter6.EDirect_Version_7_90_February_1"><h3>EDirect Version 7.90: February 1, 2018</h3><ul><li class="half_rhythm"><div>Minor change to installation commands for tcsh.</div></li></ul></div><div id="chapter6.EDirect_Version_7_80_January_12"><h3>EDirect Version 7.80: January 12, 2018</h3><ul><li class="half_rhythm"><div>Updated setup.sh script with additional error checking.</div></li></ul></div><div id="chapter6.EDirect_Version_7_70_December_2"><h3>EDirect Version 7.70: December 27, 2017</h3><ul><li class="half_rhythm"><div>Added archive-pubmed script to automate local record archiving.</div></li></ul></div><div id="chapter6.EDirect_Version_7_60_November_1"><h3>EDirect Version 7.60: November 15, 2017</h3><ul><li class="half_rhythm"><div>Epost -id numeric argument bug fixed.</div></li><li class="half_rhythm"><div>Xtract conditional tests can now use subrange specifiers.</div></li><li class="half_rhythm"><div>Xtract -strict and -mixed use separate -repair flag to normalize Unicode superscripts and subscripts.</div></li></ul></div><div id="chapter6.EDirect_Version_7_50_October_31"><h3>EDirect Version 7.50: October 31, 2017</h3><ul><li class="half_rhythm"><div>Setup instructions now work with the tcsh shell.</div></li><li class="half_rhythm"><div>API key value is taken from the NCBI_API_KEY environment variable.</div></li><li class="half_rhythm"><div>Efetch -format gb supports -style withparts and -style conwithfeat.</div></li><li class="half_rhythm"><div>Xtract supports optional element [min:max] substring extraction.</div></li><li class="half_rhythm"><div>Xtract -position supports [first|last|outer|inner|all] argument values.</div></li><li class="half_rhythm"><div>Added prepare-stash script for local record archive.</div></li></ul></div><div id="chapter6.EDirect_Version_7_40_September"><h3>EDirect Version 7.40: September 27, 2017</h3><ul><li class="half_rhythm"><div>Xtract -hash reports checksums for local record archiving.</div></li><li class="half_rhythm"><div>Initial support for API keys.</div></li></ul></div><div id="chapter6.EDirect_Version_7_30_September"><h3>EDirect Version 7.30: September 6, 2017</h3><ul><li class="half_rhythm"><div>Modified stash-pubmed script to work around Cygwin artifact.</div></li><li class="half_rhythm"><div>Removed unpack-pubmed script.</div></li><li class="half_rhythm"><div>Xtract -archive replaces -stash for local record archiving.</div></li><li class="half_rhythm"><div>Xtract -gzip allows compression of archived XML records.</div></li></ul></div><div id="chapter6.EDirect_Version_7_20_August_28"><h3>EDirect Version 7.20: August 28, 2017</h3><ul><li class="half_rhythm"><div>Added
 download-pubmed, download-sequence, unpack-pubmed, stash-pubmed, and 
fetch-pubmed scripts, for experimental local record storage.</div></li><li class="half_rhythm"><div>Xtract -flags [strict|mixed] added to support new local storage scripts.</div></li><li class="half_rhythm"><div>Removed obsolete, original Perl implementation of xtract.pl.</div></li></ul></div><div id="chapter6.EDirect_Version_7_10_August_10"><h3>EDirect Version 7.10: August 10, 2017</h3><ul><li class="half_rhythm"><div>Xtract -ascii converts non-ASCII Unicode to hexadecimal numeric character references.</div></li><li class="half_rhythm"><div>Setup script recognizes Cygwin running under the MinGW emulator.</div></li></ul></div><div id="chapter6.EDirect_Version_7_00_July_10_20"><h3>EDirect Version 7.00: July 10, 2017</h3><ul><li class="half_rhythm"><div>Xtract -mixed and -strict handle multiply-escaped HTML tags.</div></li><li class="half_rhythm"><div>Efetch removes normal and escaped HTML tags from PubMed fields.</div></li><li class="half_rhythm"><div>Esearch -field processes individual query terms using the designated field, also removing stop words.</div></li><li class="half_rhythm"><div>Esearch -pairs splits the query phrase into adjacent overlapping word pairs.</div></li></ul></div><div id="chapter6.EDirect_Version_6_90_July_5_201"><h3>EDirect Version 6.90: July 5, 2017</h3><ul><li class="half_rhythm"><div>Xtract -mixed replaces -relaxed, and -accent replaces -plain.</div></li><li class="half_rhythm"><div>Efetch uses larger chunks for -format uid, url, and acc.</div></li><li class="half_rhythm"><div>Esearch -log shows constructed URL and QueryTranslation result.</div></li></ul></div><div id="chapter6.EDirect_Version_6_80_June_8_201"><h3>EDirect Version 6.80: June 8, 2017</h3><ul><li class="half_rhythm"><div>Modified download instructions to use edirect.tar.gz archive.</div></li><li class="half_rhythm"><div>The ftp-cp script can now read from stdin without the need for xargs.</div></li><li class="half_rhythm"><div>Rerunning ftp-cp or asp-cp only attempts to download missing files.</div></li></ul></div><div id="chapter6.EDirect_Version_6_70_May_8_2017"><h3>EDirect Version 6.70: May 8, 2017</h3><ul><li class="half_rhythm"><div>Added asp-cp script for faster download of NCBI ftp files using Aspera Connect.</div></li><li class="half_rhythm"><div>Xtract -strict and -relaxed handle empty HTML tag variants (e.g., &lt;b/&gt; and &lt;sup/&gt;).</div></li></ul></div><div id="chapter6.EDirect_Version_6_60_April_25_2"><h3>EDirect Version 6.60: April 25, 2017</h3><ul><li class="half_rhythm"><div>Xtract
 -strict replaces -degloss to remove HTML &lt;i&gt;, &lt;b&gt;, 
&lt;u&gt;, &lt;sup&gt; and &lt;sub&gt; tags from XML contents.</div></li><li class="half_rhythm"><div>Xtract -relaxed allows HTML tags in XML contents, to support current PubMed ftp release files.</div></li><li class="half_rhythm"><div>Xtract -plain removes Unicode accents.</div></li><li class="half_rhythm"><div>The setup.sh script prints an error message if it cannot fetch missing Perl modules.</div></li></ul></div><div id="chapter6.EDirect_Version_6_50_March_6_20"><h3>EDirect Version 6.50: March 6, 2017</h3><ul><li class="half_rhythm"><div>Xtract -degloss replaces -html to remove HTML &lt;i&gt;, &lt;b&gt;, &lt;u&gt;, &lt;sup&gt; and &lt;sub&gt; tags.</div></li></ul></div><div id="chapter6.EDirect_Version_6_40_March_1_20"><h3>EDirect Version 6.40: March 1, 2017</h3><ul><li class="half_rhythm"><div>Epost detects accession.version input for sequence databases and sets -format acc.</div></li><li class="half_rhythm"><div>Xtract -html [remove|encode] converts &lt;i&gt; and &lt;b&gt; tags embedded in XML contents.</div></li></ul></div><div id="chapter6.EDirect_Version_6_30_February_1"><h3>EDirect Version 6.30: February 13, 2017</h3><ul><li class="half_rhythm"><div>Efetch -format docsum skips GI-less sequences without summaries.</div></li><li class="half_rhythm"><div>Xtract local indexing commands moved to -extras documentation.</div></li></ul></div><div id="chapter6.EDirect_Version_6_20_January_30"><h3>EDirect Version 6.20: January 30, 2017</h3><ul><li class="half_rhythm"><div>Xtract -limit and -index allow extraction of selected records from XML file.</div></li></ul></div><div id="chapter6.EDirect_Version_6_10_January_19"><h3>EDirect Version 6.10: January 19, 2017</h3><ul><li class="half_rhythm"><div>Added run-ncbi-converter script for processing ASN.1 release files.</div></li><li class="half_rhythm"><div>Xtract -format flush option added.</div></li><li class="half_rhythm"><div>Removed obsolete accession-dot-version conversion code.</div></li></ul></div><div id="chapter6.EDirect_Version_6_00_December_2"><h3>EDirect Version 6.00: December 27, 2016</h3><ul><li class="half_rhythm"><div>Efetch -format docsum removes eSummaryResult wrapper.</div></li><li class="half_rhythm"><div>Fixed content truncation bug when Xtract encounters very long sequences.</div></li></ul></div><div id="chapter6.EDirect_Version_5_90_December_2"><h3>EDirect Version 5.90: December 21, 2016</h3><ul><li class="half_rhythm"><div>Efetch and Elink readied for switch to accession-dot-version sequence identifier.</div></li><li class="half_rhythm"><div>Xtract -insd recognizes INSDInterval_iscomp@value and other boolean attributes.</div></li><li class="half_rhythm"><div>Xtract adds experimental phrase processing commands for word index preparation.</div></li></ul></div><div id="chapter6.EDirect_Version_5_80_December_1"><h3>EDirect Version 5.80: December 12, 2016</h3><ul><li class="half_rhythm"><div>Efilter adds shortcuts for -db gene (e.g., -status alive, -type coding).</div></li><li class="half_rhythm"><div>Xtract
 numeric conditional tests can use an element name for the second 
argument (e.g., -if ChrStop -lt ChrStart finds minus strand genes).</div></li></ul></div><div id="chapter6.EDirect_Version_5_70_November_3"><h3>EDirect Version 5.70: November 30, 2016</h3><ul><li class="half_rhythm"><div>Xtract
 -format takes an optional [compact|indent|expand] argument. Processing 
compact XML is about 15% faster than indent form. Using expand places 
each attribute on a separate line for ease of reading.</div></li></ul></div><div id="chapter6.EDirect_Version_5_60_November_2"><h3>EDirect Version 5.60: November 22, 2016</h3><ul><li class="half_rhythm"><div>Fixed bug in -datetype argument for Esearch and Efilter.</div></li><li class="half_rhythm"><div>Added optional argument to filter-stop-words script to indicate replacement.</div></li></ul></div><div id="chapter6.EDirect_Version_5_50_November_1"><h3>EDirect Version 5.50: November 16, 2016</h3><ul><li class="half_rhythm"><div>Efetch -id allows non-numeric accessions only for sequence databases.</div></li><li class="half_rhythm"><div>Xtract element selection no longer considers fields in recursive sub-objects.</div></li><li class="half_rhythm"><div>Xtract introduces a double-star "**/Object" construct to flatten recursive child objects for linear exploration.</div></li><li class="half_rhythm"><div>Xtract conditional tests ignore empty self-closing tags.</div></li><li class="half_rhythm"><div>Xtract -else simplifies insertion of a placeholder to indicate missing data.</div></li></ul></div><div id="chapter6.EDirect_Version_5_40_November_7"><h3>EDirect Version 5.40: November 7, 2016</h3><ul><li class="half_rhythm"><div>Added filter-stop-words and xy-plot scripts.</div></li></ul></div><div id="chapter6.EDirect_Version_5_30_October_31"><h3>EDirect Version 5.30: October 31, 2016</h3><ul><li class="half_rhythm"><div>Added support for ecitmatch utility.</div></li><li class="half_rhythm"><div>Added amino-acid-composition and between-two-genes scripts.</div></li><li class="half_rhythm"><div>The
 sort-uniq-count and sort-uniq-count-rank scripts take an optional 
argument (e.g., -n for numeric comparisons, -r to reverse order).</div></li></ul></div><div id="chapter6.EDirect_Version_5_20_October_26"><h3>EDirect Version 5.20: October 26, 2016</h3><ul><li class="half_rhythm"><div>Setup
 script no longer modifies the user's configuration file to update the 
PATH variable. Instead, it now prints customized instructions for the 
user to execute. The user may choose to run these commands, but is free 
to edit the .bash_profile file manually.</div></li><li class="half_rhythm"><div>Xtract deprecates -match and -avoid functions and the Element:Value conditional shortcut.</div></li><li class="half_rhythm"><div>Xtract -if and -unless commands use compound statements for conditional execution (e.g., -if Element -equals Value).</div></li><li class="half_rhythm"><div>Colon
 now separates namespace prefix from element name in xtract arguments 
(e.g., -block jats:abstract). Colon at start of element name matches any
 namespace prefix.</div></li><li class="half_rhythm"><div>Xtract -insd uses a dash as placeholder for missing field. Experimental -insdx command is deprecated.</div></li><li class="half_rhythm"><div>Precompiled
 versions of xtract are now provided for Darwin, Linux, and CYGWIN_NT 
platforms. The appropriate executable is downloaded by the setup script.</div></li></ul></div><div id="chapter6.EDirect_Version_5_10_October_13"><h3>EDirect Version 5.10: October 13, 2016</h3><ul><li class="half_rhythm"><div>Xtract
 adds -0-based, -1-based, and -ucsc numeric extraction/conversion 
commands for sequence positions from several Entrez databases.</div></li></ul></div><div id="chapter6.EDirect_Version_5_00_September"><h3>EDirect Version 5.00: September 26, 2016</h3><ul><li class="half_rhythm"><div>Efetch -format fasta removes blank lines between records.</div></li><li class="half_rhythm"><div>Xtract -insdx uses a dash to indicate a missing field.</div></li><li class="half_rhythm"><div>Xtract -insd no longer has blank lines between records.</div></li><li class="half_rhythm"><div>Xtract -input allows reading XML data from a file.</div></li></ul></div><div id="chapter6.EDirect_Version_4_90_September"><h3>EDirect Version 4.90: September 14, 2016</h3><ul><li class="half_rhythm"><div>Epost -input allows reading from an input file instead of using data piped through stdin.</div></li><li class="half_rhythm"><div>Efilter now supports the -sort argument.</div></li><li class="half_rhythm"><div>Xtract -filter can recover information in XML comments and CDATA blocks.</div></li></ul></div><div id="chapter6.EDirect_Version_4_80_August_9_2"><h3>EDirect Version 4.80: August 9, 2016</h3><ul><li class="half_rhythm"><div>Xtract -insd controlled vocabularies updated.</div></li></ul></div><div id="chapter6.EDirect_Version_4_70_August_4_2"><h3>EDirect Version 4.70: August 4, 2016</h3><ul><li class="half_rhythm"><div>Einfo -db request can also display -fields and -links data summaries.</div></li><li class="half_rhythm"><div>Einfo -dbs prints database names instead of eInfoResult XML.</div></li></ul></div><div id="chapter6.EDirect_Version_4_60_July_18_20"><h3>EDirect Version 4.60: July 18, 2016</h3><ul><li class="half_rhythm"><div>Elink -cmd acheck returns information on all available links for a record.</div></li><li class="half_rhythm"><div>Efilter -pub structured limits to articles with structured abstracts.</div></li></ul></div><div id="chapter6.EDirect_Version_4_50_July_1_201"><h3>EDirect Version 4.50: July 1, 2016</h3><ul><li class="half_rhythm"><div>Esearch and Efilter detect and report -query phrase quotation errors.</div></li><li class="half_rhythm"><div>Efilter -pub shortcut adds last_week, last_month, and last_year choices.</div></li><li class="half_rhythm"><div>Efetch sets -strand 2 for minus strand if -seq_start &gt; -seq_stop or if -chr_start &gt; -chr_stop.</div></li></ul></div><div id="chapter6.EDirect_Version_4_40_June_21_20"><h3>EDirect Version 4.40: June 21, 2016</h3><ul><li class="half_rhythm"><div>Transitioning to use of https for access to NCBI services.</div></li><li class="half_rhythm"><div>Epost -db assembly -format acc uses [ASAC] field instead of [ACCN].</div></li></ul></div><div id="chapter6.EDirect_Version_4_30_June_13_20"><h3>EDirect Version 4.30: June 13, 2016</h3><ul><li class="half_rhythm"><div>Efilter -pub preprint limits results to ahead-of-print articles.</div></li><li class="half_rhythm"><div>Xtract -pattern Parent/* construct can now process catenated XML files.</div></li></ul></div><div id="chapter6.EDirect_Version_4_20_May_24_201"><h3>EDirect Version 4.20: May 24, 2016</h3><ul><li class="half_rhythm"><div>Xtract command-line argument parsing improvements.</div></li><li class="half_rhythm"><div>Nquire -get supersedes -http get.</div></li></ul></div><div id="chapter6.EDirect_Version_4_10_May_3_2016"><h3>EDirect Version 4.10: May 3, 2016</h3><ul><li class="half_rhythm"><div>Xtract -format removes multi-line XML comments and CDATA blocks.</div></li></ul></div><div id="chapter6.EDirect_Version_4_00_April_4_20"><h3>EDirect Version 4.00: April 4, 2016</h3><ul><li class="half_rhythm"><div>Esearch adds -spell to correct known misspellings of biological terms in the query string.</div></li><li class="half_rhythm"><div>Efilter
 adds -spell to correct query misspellings, and -pub, -feature, 
-location, -molecule, -organism, and -source shortcuts. Run efilter 
-help to see the choices available for each argument.</div></li></ul></div><div id="chapter6.EDirect_Version_3_90_March_21_2"><h3>EDirect Version 3.90: March 21, 2016</h3><ul><li class="half_rhythm"><div>Code optimizations for increased Xtract speed.</div></li></ul></div><div id="chapter6.EDirect_Version_3_80_February_2"><h3>EDirect Version 3.80: February 29, 2016</h3><ul><li class="half_rhythm"><div>Xtract can distribute its work among available processor cores for additional speed.</div></li></ul></div><div id="chapter6.EDirect_Version_3_70_February_8"><h3>EDirect Version 3.70: February 8, 2016</h3><ul><li class="half_rhythm"><div>Xtract performance improvements.</div></li></ul></div><div id="chapter6.EDirect_Version_3_60_January_11"><h3>EDirect Version 3.60: January 11, 2016</h3><ul><li class="half_rhythm"><div>The setup.sh configuration script now downloads a precompiled Xtract executable for selected platforms.</div></li></ul></div><div id="chapter6.EDirect_Version_3_50_December_2"><h3>EDirect Version 3.50: December 27, 2015</h3><ul><li class="half_rhythm"><div>Xtract reports error for element:value construct outside of -match or -avoid arguments.</div></li></ul></div><div id="chapter6.EDirect_Version_3_40_December_2"><h3>EDirect Version 3.40: December 20, 2015</h3><ul><li class="half_rhythm"><div>Xtract -insd supports extraction from multiple features (e.g., CDS,mRNA).</div></li></ul></div><div id="chapter6.EDirect_Version_3_30_December_3"><h3>EDirect Version 3.30: December 3, 2015</h3><ul><li class="half_rhythm"><div>Efetch -format docsum can accept a single sequence accession number in the -id argument.</div></li></ul></div><div id="chapter6.EDirect_Version_3_20_November_3"><h3>EDirect Version 3.20: November 30, 2015</h3><ul><li class="half_rhythm"><div>Xtract supports -match conditional execution on values recorded in variables.</div></li></ul></div><div id="chapter6.EDirect_Version_3_10_November_1"><h3>EDirect Version 3.10: November 18, 2015</h3><ul><li class="half_rhythm"><div>Efetch
 adds -chr_start and -chr_stop arguments to specify sequence range from 
0-based coordinates in gene docsum GenomicInfoType object.</div></li></ul></div><div id="chapter6.EDirect_Version_3_00_October_30"><h3>EDirect Version 3.00: October 30, 2015</h3><ul><li class="half_rhythm"><div>Xtract
 rewritten in the Go programming language for speed. The setup.sh 
configuration script installs an older Perl version (2.99) if a local Go
 compiler is unavailable.</div></li><li class="half_rhythm"><div>Efetch -format docsum only decodes HTML entity numbers in select situations.</div></li></ul></div><div id="chapter6.EDirect_Version_2_90_October_15"><h3>EDirect Version 2.90: October 15, 2015</h3><ul><li class="half_rhythm"><div>Xtract warns on use of deprecated arguments -present, -absent, and -trim, in preparation for release of much faster version.</div></li></ul></div><div id="chapter6.EDirect_Version_2_80_September"><h3>EDirect Version 2.80: September 9, 2015</h3><ul><li class="half_rhythm"><div>Xtract uses the "*/Child" construct for nested exploration into recursive structures, replacing the -trim argument.</div></li></ul></div><div id="chapter6.EDirect_Version_2_70_July_14_20"><h3>EDirect Version 2.70: July 14, 2015</h3><ul><li class="half_rhythm"><div>Added entrez-phrase-search script to query on adjacent word pairs indexed in specific fields.</div></li></ul></div><div id="chapter6.EDirect_Version_2_60_June_23_20"><h3>EDirect Version 2.60: June 23, 2015</h3><ul><li class="half_rhythm"><div>Xtract -match and -avoid support "Parent/Child" construct for BLAST XML.</div></li></ul></div><div id="chapter6.EDirect_Version_2_50_April_9_20"><h3>EDirect Version 2.50: April 9, 2015</h3><ul><li class="half_rhythm"><div>Xtract capitalized -Pattern handles recursively-defined top-level objects.</div></li></ul></div><div id="chapter6.EDirect_Version_2_40_March_25_2"><h3>EDirect Version 2.40: March 25, 2015</h3><ul><li class="half_rhythm"><div>EDirect programs use the http_proxy environment variable to work behind firewalls.</div></li></ul></div><div id="chapter6.EDirect_Version_2_30_March_11_2"><h3>EDirect Version 2.30: March 11, 2015</h3><ul><li class="half_rhythm"><div>Cleaned up logic in setup.sh configuration script.</div></li><li class="half_rhythm"><div>EPost -format acc works properly on protein accessions.</div></li></ul></div><div id="chapter6.EDirect_Version_2_20_March_4_20"><h3>EDirect Version 2.20: March 4, 2015</h3><ul><li class="half_rhythm"><div>Xtract -match and -avoid recognize "@attribute" without element or value.</div></li></ul></div><div id="chapter6.EDirect_Version_2_10_February_3"><h3>EDirect Version 2.10: February 3, 2015</h3><ul><li class="half_rhythm"><div>Added ftp-ls and ftp-cp scripts for convenient access to the NCBI anonymous ftp server.</div></li></ul></div><div id="chapter6.EDirect_Version_2_00_August_28"><h3>EDirect Version 2.00: August 28, 2014</h3><ul><li class="half_rhythm"><div>Introduced copy-and-paste installation commands with setup.sh configuration script.</div></li></ul></div><div id="chapter6.EDirect_Version_1_90_August_8_2"><h3>EDirect Version 1.90: August 8, 2014</h3><ul><li class="half_rhythm"><div>Xtract -format combines multiple XML results into a single valid object.</div></li><li class="half_rhythm"><div>Improved suppression of 0-count failure messages with -silent flag in scripts.</div></li></ul></div><div id="chapter6.EDirect_Version_1_80_July_15_20"><h3>EDirect Version 1.80: July 15, 2014</h3><ul><li class="half_rhythm"><div>EPost -format acc accepts accessions in an -id argument on the command line.</div></li></ul></div><div id="chapter6.EDirect_Version_1_70_April_23_2"><h3>EDirect Version 1.70: April 23, 2014</h3><ul><li class="half_rhythm"><div>EFetch -format docsum decodes HTML entity numbers embedded in the text.</div></li></ul></div><div id="chapter6.EDirect_Version_1_60_April_3_20"><h3>EDirect Version 1.60: April 3, 2014</h3><ul><li class="half_rhythm"><div>Minor enhancements to xtract -insd.</div></li></ul></div><div id="chapter6.EDirect_Version_1_50_March_29_2"><h3>EDirect Version 1.50: March 29, 2014</h3><ul><li class="half_rhythm"><div>Esearch -sort specifies the order of results when records are retrieved.</div></li><li class="half_rhythm"><div>Xtract exploration arguments (e.g., -block) now work on self-closing tags with data in attributes.</div></li></ul></div><div id="chapter6.EDirect_Version_1_40_March_17_2"><h3>EDirect Version 1.40: March 17, 2014</h3><ul><li class="half_rhythm"><div>Xtract -format repairs XML line-wrapping and indentation.</div></li><li class="half_rhythm"><div>Implemented -help flag to display the list of command-line arguments for each function.</div></li></ul></div><div id="chapter6.EDirect_Version_1_30_March_3_20"><h3>EDirect Version 1.30: March 3, 2014</h3><ul><li class="half_rhythm"><div>Xtract
 -insd partial logic was corrected to examine both 5' and 3' partial 
flags, and the location indicator recognizes "+" or "complete" and "-" 
or "partial".</div></li></ul></div><div id="chapter6.EDirect_Version_1_20_February_2"><h3>EDirect Version 1.20: February 26, 2014</h3><ul><li class="half_rhythm"><div>Xtract
 -insd detects if it is part of an EDirect sequence record query, and 
dynamically executes the extraction request for specific qualifier 
values. When run in isolation it generates extraction instructions that 
can be incorporated (with modifications, if necessary) into other 
queries.</div></li></ul></div><div id="chapter6.EDirect_Version_1_10_February_1"><h3>EDirect Version 1.10: February 10, 2014</h3><ul><li class="half_rhythm"><div>ESummary
 was replaced by "efetch -format docsum" to provide a single command for
 all document retrieval. The esummary command will continue to work for 
those who prefer it, and to avoid breaking existing scripts.</div></li><li class="half_rhythm"><div>Xtract
 processes each -pattern object immediately upon receipt, eliminating 
the need for using xargs and sh to split document retrieval into smaller
 units.</div></li></ul></div><div id="chapter6.EDirect_Version_1_00_February_6"><h3>EDirect Version 1.00: February 6, 2014</h3><ul><li class="half_rhythm"><div>Initial public release.</div></li></ul></div></div><div id="chapter6.For_More_Information"><h2 id="_chapter6_For_More_Information_">For More Information</h2><div id="chapter6.Announcement_Mailing_List"><h3>Announcement Mailing List</h3><p>NCBI posts general announcements regarding the E-utilities to the <a href="https://www.ncbi.nlm.nih.gov/mailman/listinfo/utilities-announce/" ref="pagearea=body&amp;targetsite=external&amp;targetcat=link&amp;targettype=uri" target="_blank"><u>utilities-announce announcement mailing list</u></a>. This mailing list is an announcement list only; individual subscribers may <b>not</b>
 send mail to the list. Also, the list of subscribers is private and is 
not shared or used in any other way except for providing announcements 
to list members. The list receives about one posting per month. Please 
subscribe at the above link.</p></div><div id="chapter6.Getting_Help"><h3>Getting Help</h3><p>Please refer to the <a href="https://www.ncbi.nlm.nih.gov/books/n/helppubmed/pubmedhelp/?report=reader">PubMed</a> and <a href="https://www.ncbi.nlm.nih.gov/books/n/helpentrez/EntrezHelp/?report=reader">Entrez</a> help documents for more information about search queries, database indexing, field limitations and database content.</p><p>Suggestions, comments, and questions specifically relating to the EUtility programs may be sent to <u><a href="mailto:dev@null" data-email="vog.hin.mln.ibcn@seitilitue" class="oemail">vog.hin.mln.ibcn@seitilitue</a></u>.</p></div></div><div id="bk_toc_contnr"></div></div></div><div class="fm-sec"><h2 id="_NBK179288_pubdet_">Publication Details</h2><h3>Author Information</h3><p class="contrib-group"></p><h4>Authors</h4><span itemprop="author">Jonathan Kans</span>, PhD<sup><img src="corrauth.gif" alt="corresponding author"></sup>.<p></p><h4>Affiliations</h4><div class="affiliation">NCBI<div><a href="mailto:dev@null" data-email="vog.hin.mln.ibcn@snak" class="oemail">vog.hin.mln.ibcn@snak</a></div></div><div><sup><img src="corrauth.gif" alt="corresponding author"></sup>Corresponding author.</div><h3>Publication History</h3><p class="small">Created: <span itemprop="datePublished">April 23, 2013</span>; Last Update: <span itemprop="dateModified">February 11, 2019</span>.</p><h3>Copyright</h3><div><div class="half_rhythm"><a href="https://www.ncbi.nlm.nih.gov/books/about/copyright/">Copyright Notice</a></div></div><h3>Publisher</h3><p><a href="http://www.ncbi.nlm.nih.gov/" ref="pagearea=page-banner&amp;targetsite=external&amp;targetcat=link&amp;targettype=publisher" target="_blank">National Center for Biotechnology Information (US)</a>, Bethesda (MD)</p><h3>NLM Citation</h3><p>Kans
 J. Entrez Direct: E-utilities on the UNIX Command Line. 2013 Apr 23 
[Updated 2019 Feb 11]. In: Entrez Programming Utilities Help [Internet].
 Bethesda (MD): National Center for Biotechnology Information (US); 
2010-. <span class="bk_cite_avail"></span></p></div><div class="small-screen-prev"><a href="https://www.ncbi.nlm.nih.gov/books/n/helpeutils/chapter5/?report=reader"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" preserveAspectRatio="none"><path d="M75,30 c-80,60 -80,0 0,60 c-30,-60 -30,0 0,-60"></path><text x="20" y="28" textLength="60" style="font-size:25px">Prev</text></svg></a></div><div class="small-screen-next"></div></article></div><div id="jr-scripts"><script src="libs.js"> </script><script src="jr_002.js"> </script><script type="text/javascript" src="qs.js"> </script></div></div>




        <!-- Book content -->   
        
        <script type="text/javascript" src="InstrumentPageStarterJS.js"> </script><img alt="" src="stat.gif" style="display: none;">
    

<!-- 8A1A6945BBF57001_0075SID /projects/books/PBooks@5.22 portal103 v4.1.r584435 Thu, Apr 11 2019 12:42:06 -->

<script type="text/javascript" src="3968615.js" snapshot="books"></script>

<img alt="" src="stat_002.gif" style="display: none;"><script type="text/javascript" src="a.js"></script></body></html>