<!DOCTYPE html>
<html lang="en"><head><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace"><meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1"><meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge"><link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">
    <link href="style.css" rel="stylesheet" type="text/css">

    
    <link href="jquery.css" rel="stylesheet" type="text/css">

    
    <link href="jscrollpane.css" rel="stylesheet" type="text/css">
    <link href="theme.css" rel="stylesheet" type="text/css">
    
    <link href="css.css" rel="stylesheet" type="text/css">
        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-304756074.css" rel="stylesheet" type="text/css">
    <script async="" src="gtm.js"></script><script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [0] 
        };

        App.CurrentSubdomain = 'mbe';
    </script>

    
    <script type="text/javascript" src="jquery-2.js"></script>

    
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">

        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/mbe/article/27/2/221/970247","@type":"ScholarlyArticle","name":"SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building","datePublished":"2009-10-23","isPartOf":{"@id":"https://academic.oup.com/mbe/issue/27/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2010-02-01","isPartOf":{"@id":"https://academic.oup.com/mbe","@type":"Periodical","name":"Molecular Biology and Evolution","issn":["1537-1719"]}},"url":"http://dx.doi.org/10.1093/molbev/msp259","keywords":["SeaView","multiple sequence alignment","molecular phylogeny","PhyML","graphical user interface"],"inLanguage":"en","copyrightHolder":"Society for Molecular Biology and Evolution","copyrightYear":"2019","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/mbe/article/27/2/221/970247","author":[{"name":"Gouy, Manolo","affiliation":"1 Laboratoire de Biométrie et Biologie Evolutive, CNRS UMR 5558, Université Lyon 1, Université de Lyon, Villeurbanne, France","@type":"Person"},{"name":"Guindon, Stéphane","affiliation":"2 Méthodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS UMR 5506, Université Montpellier II, Montpellier, France   3 Department of Statistics, University of Auckland, Auckland, New Zealand","@type":"Person"},{"name":"Gascuel, Olivier","affiliation":"2 Méthodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS UMR 5506, Université Montpellier II, Montpellier, France","@type":"Person"}],"description":"Abstract.  We present SeaView version 4, a multiplatform program designed to facilitate multiple alignment and phylogenetic tree building from molecular sequenc","pageStart":"221","pageEnd":"224","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/27/2/0/m_mbe27_2.cover.gif?Expires=1619537602&Signature=sOBBMs5zxB3xYz8-zPpxJyV-Hlr1OE9epwdXn3vd9B8vXCLvJiURzoAgV~ApkdSSnj-xnuB4MLVOoMlZv3lg~KgB4ilRiQdl2nOvPsRWJh970w6cLJztMDSyL0IoE52ZtJeXQIHx68NG5BAn1i50TDX6rgdFKBh0HVFVs9hHf3VzWroNJuiUs8Mf-wiSWPt5XxdwXgReVzS48yeA9iSgc4TilKuc378ZhgrbFvV8ainNe7j~UVhnDmmVFUw4GJpgPEJSRSTWg~QkdzAuUVs4heAyVODQAV9kP3FuUCRXNR9hG~MhrHzkUSeadSYUqYx6MzD9O7BG5gdb8FIxJhXY9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/27/2/0/m_mbe27_2.cover.gif?Expires=1619537602&Signature=sOBBMs5zxB3xYz8-zPpxJyV-Hlr1OE9epwdXn3vd9B8vXCLvJiURzoAgV~ApkdSSnj-xnuB4MLVOoMlZv3lg~KgB4ilRiQdl2nOvPsRWJh970w6cLJztMDSyL0IoE52ZtJeXQIHx68NG5BAn1i50TDX6rgdFKBh0HVFVs9hHf3VzWroNJuiUs8Mf-wiSWPt5XxdwXgReVzS48yeA9iSgc4TilKuc378ZhgrbFvV8ainNe7j~UVhnDmmVFUw4GJpgPEJSRSTWg~QkdzAuUVs4heAyVODQAV9kP3FuUCRXNR9hG~MhrHzkUSeadSYUqYx6MzD9O7BG5gdb8FIxJhXY9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building">
    <meta property="og:description" content="Abstract.  We present SeaView version 4, a multiplatform program designed to facilitate multiple alignment and phylogenetic tree building from molecular sequenc">
    <meta property="og:url" content="https://academic.oup.com/mbe/article/27/2/221/970247">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/Issue/27/2/0/m_mbe27_2.cover.gif?Expires=1619537602&amp;Signature=sOBBMs5zxB3xYz8-zPpxJyV-Hlr1OE9epwdXn3vd9B8vXCLvJiURzoAgV~ApkdSSnj-xnuB4MLVOoMlZv3lg~KgB4ilRiQdl2nOvPsRWJh970w6cLJztMDSyL0IoE52ZtJeXQIHx68NG5BAn1i50TDX6rgdFKBh0HVFVs9hHf3VzWroNJuiUs8Mf-wiSWPt5XxdwXgReVzS48yeA9iSgc4TilKuc378ZhgrbFvV8ainNe7j~UVhnDmmVFUw4GJpgPEJSRSTWg~QkdzAuUVs4heAyVODQAV9kP3FuUCRXNR9hG~MhrHzkUSeadSYUqYx6MzD9O7BG5gdb8FIxJhXY9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2009-10-23">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Gouy, Manolo"><meta name="citation_author_institution" content="Laboratoire de Biométrie et Biologie Evolutive, CNRS UMR 5558, Université Lyon 1, Université de Lyon, Villeurbanne, France"><meta name="citation_author" content="Guindon, Stéphane"><meta name="citation_author_institution" content="Méthodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS UMR 5506, Université Montpellier II, Montpellier, France"><meta name="citation_author_institution" content="Department of Statistics, University of Auckland, Auckland, New Zealand"><meta name="citation_author" content="Gascuel, Olivier"><meta name="citation_author_institution" content="Méthodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS UMR 5506, Université Montpellier II, Montpellier, France"><meta name="citation_title" content="SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building"><meta name="citation_firstpage" content="221"><meta name="citation_lastpage" content="224"><meta name="citation_doi" content="10.1093/molbev/msp259"><meta name="citation_journal_title" content="Molecular Biology and Evolution"><meta name="citation_journal_abbrev" content="Mol Biol Evol"><meta name="citation_volume" content="27"><meta name="citation_issue" content="2"><meta name="citation_publication_date" content="2010/02/01"><meta name="citation_issn" content="0737-4038"><meta name="citation_publisher" content="Narnia"><meta name="citation_reference" content="citation_title=Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative; Syst Biol;  citation_year=2006;  citation_volume=55;  citation_pages=539-552; "><meta name="citation_reference" content="citation_title=Phylogeny.fr: robust phylogenetic analysis for the non-specialist; citation_author=(12 co-authors); Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=W465-W469; "><meta name="citation_reference" content="citation_title=ProbCons: probabilistic consistency-based multiple sequence alignment; Genome Res;  citation_year=2005;  citation_volume=15;  citation_pages=330-340; "><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput; Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=1792-1797; "><meta name="citation_reference" content="citation_title=Multiple sequences alignment editor (MASE); Trends Biochem Sci;  citation_year=1988;  citation_volume=13;  citation_pages=321-322; "><meta name="citation_reference" content="citation_title=Confidence limits on phylogenies: an approach using the bootstrap; Evolution;  citation_year=1985;  citation_volume=39;  citation_pages=783-791; "><meta name="citation_reference" content="citation_text=Distributed by the author. Seattle (WA): Department of Genome Sciences, University of Washington; PHYLIP (phylogeny inference package) version 3.52;  citation_year=1993; "><meta name="citation_reference" content="citation_title=SEAVIEW and PHYLO_WIN: two graphic tools for sequence alignment and molecular phylogeny; Comput Appl Biosci;  citation_year=1996;  citation_volume=12;  citation_pages=543-548; "><meta name="citation_reference" content="citation_title=BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data; Mol Biol Evol;  citation_year=1997;  citation_volume=14;  citation_pages=685-695; "><meta name="citation_reference" content="citation_title=Remote access to ACNUC nucleotide and protein sequence databases at PBIL; Biochimie;  citation_year=2008;  citation_volume=90;  citation_pages=555-562; "><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood; Syst Biol;  citation_year=2003;  citation_volume=52;  citation_pages=696-704; "><meta name="citation_reference" content="citation_title=CLUSTAL V: improved software for multiple sequence alignment; Comput Appl Biosci;  citation_year=1992;  citation_volume=8;  citation_pages=189-191; "><meta name="citation_reference" content="citation_title=Reconstructing evolutionary trees from DNA and protein sequences: paralinear distances; Proc Natl Acad Sci USA;  citation_year=1994;  citation_volume=91;  citation_pages=1455-1459; "><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0; citation_author=(13 co-authors); Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-2948; "><meta name="citation_reference" content="citation_title=Unbiased estimation of the rates of synonymous and nonsynonymous substitution; J Mol Evol;  citation_year=1993;  citation_volume=36;  citation_pages=96-99; "><meta name="citation_reference" content="citation_title=NEXUS: an extensible file format for systematic information; Syst Biol;  citation_year=1997;  citation_volume=46;  citation_pages=590-621; "><meta name="citation_reference" content="citation_title=Enhanced graphic matrix analysis of nucleic acid and protein sequences; Proc Natl Acad Sci USA;  citation_year=1981;  citation_volume=78;  citation_pages=7665-7669; "><meta name="citation_reference" content="citation_publisher=Columbia University Press, New York; Molecular evolutionary genetics;  citation_year=1987; "><meta name="citation_reference" content="citation_title=T-Coffee: a novel method for multiple sequence alignments; J Mol Biol;  citation_year=2000;  citation_volume=302;  citation_pages=205-217; "><meta name="citation_reference" content="citation_title=Improved tools for biological sequence comparison; Proc Natl Acad Sci USA;  citation_year=1988;  citation_volume=85;  citation_pages=2444-2448; "><meta name="citation_reference" content="citation_title=Theoretical foundation of the minimum-evolution method of phylogenetic inference; Mol Biol Evol;  citation_year=1993;  citation_volume=10;  citation_pages=1073-1095; "><meta name="citation_reference" content="citation_title=Tests of applicability of several substitution models for DNA sequence data; Mol Biol Evol;  citation_year=1995;  citation_volume=12;  citation_pages=131-151; "><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol Biol Evol;  citation_year=1987;  citation_volume=4;  citation_pages=406-425; "><meta name="citation_reference" content="citation_title=A note on the neighbor-joining algorithm of Saitou and Nei; Mol Biol Evol;  citation_year=1988;  citation_volume=5;  citation_pages=729-731; "><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) Software Version 4.0; Mol Biol Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1596-1599; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/mbe/article-pdf/27/2/221/2916333/msp259.pdf"><meta name="description" content="Abstract.  We present SeaView version 4, a multiplatform program designed to facilitate multiple alignment and phylogenetic tree building from molecular sequenc">

        <meta name="citation_xml_url" content="https://academic.oup.com/mbe/article-xml/27/2/221/970247">



    <link rel="canonical" href="https://academic.oup.com/mbe/article/27/2/221/970247">


    <script async="async" src="gpt.js"></script>
    <script>
      var googletag = googletag || {};
      googletag.cmd = googletag.cmd || [];
    </script>

    

    <script type="text/javascript">
      var gptAdSlots = [];
      googletag.cmd.push(function() {
        var pathChar = '/';
        
        var mapping_ad1 = googletag.sizeMapping().
        addSize([1024, 0], [[970, 90], [728, 90]]).
        addSize([768, 0], [728, 90]).
        addSize([0, 0], [320, 50]).build();
        gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
        defineSizeMapping(mapping_ad1).
        addService(googletag.pubads());
        
        var mapping_ad2 = googletag.sizeMapping().
	    addSize([0, 0], []).
        addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
        gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
        defineSizeMapping(mapping_ad2).
        addService(googletag.pubads());
        
        var mapping_ad3 = googletag.sizeMapping().
	    addSize([0, 0], []).
        addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
        gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
        defineSizeMapping(mapping_ad3).
        addService(googletag.pubads());
        
        var mapping_ad4 = googletag.sizeMapping().
	    addSize([0, 0], []).
        addSize([768, 0], [728, 90]).build();
        gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'mbe_Behind_Ad4', [728, 90], 'adBlockFooter').
        defineSizeMapping(mapping_ad4).
        addService(googletag.pubads());
        
        googletag.pubads().setTargeting("jnlspage", "article");
        googletag.pubads().setTargeting("jnlsurl", "mbe/article/27/2/221/970247");
        googletag.pubads().enableSingleRequest();
        googletag.pubads().collapseEmptyDivs();
        
      });
    </script>
    

    <script>
        var oupcookiepolicy_siteid = "journals";	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=molbev&authzrequired=false&doi=10.1093/molbev/msp259';
    </script>
<title>
	SeaView Version 4: A Multiplatform Graphical User Interface for 
Sequence Alignment and Phylogenetic Tree Building | Molecular Biology 
and Evolution | Oxford Academic
</title><script id="altmetric-embed-js" src="altmetric_badges-71f30fc33cff2fd0a5de8e407f36bda8.js"></script><style></style><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019041801.js" async=""></script><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-f6c32e9d70390ae112e1f1b71326f9b4.css"></head>

<body data-sitename="molecularbiologyandevolution" class="off-canvas pg_article" theme-mbe=""><div id="MathJax_Message" style="display: none;"></div>
    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./970247" id="webform">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUIODQwOTY1MjUPZBYCZg9kFgJmD2QWBGYPZBYCAg4PZBYCAgEPZBYCZg8WAh4EVGV4dAX3ODxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR291eSwgTWFub2xvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9pcmUgZGUgQmlvbcOpdHJpZSBldCBCaW9sb2dpZSBFdm9sdXRpdmUsIENOUlMgVU1SIDU1NTgsIFVuaXZlcnNpdMOpIEx5b24gMSwgVW5pdmVyc2l0w6kgZGUgTHlvbiwgVmlsbGV1cmJhbm5lLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHdWluZG9uLCBTdMOpcGhhbmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNw6l0aG9kZXMgZXQgQWxnb3JpdGhtZXMgcG91ciBsYSBCaW9pbmZvcm1hdGlxdWUsIExJUk1NLCBDTlJTIFVNUiA1NTA2LCBVbml2ZXJzaXTDqSBNb250cGVsbGllciBJSSwgTW9udHBlbGxpZXIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgU3RhdGlzdGljcywgVW5pdmVyc2l0eSBvZiBBdWNrbGFuZCwgQXVja2xhbmQsIE5ldyBaZWFsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2FzY3VlbCwgT2xpdmllciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik3DqXRob2RlcyBldCBBbGdvcml0aG1lcyBwb3VyIGxhIEJpb2luZm9ybWF0aXF1ZSwgTElSTU0sIENOUlMgVU1SIDU1MDYsIFVuaXZlcnNpdMOpIE1vbnRwZWxsaWVyIElJLCBNb250cGVsbGllciwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTZWFWaWV3IFZlcnNpb24gNDogQSBNdWx0aXBsYXRmb3JtIEdyYXBoaWNhbCBVc2VyIEludGVyZmFjZSBmb3IgU2VxdWVuY2UgQWxpZ25tZW50IGFuZCBQaHlsb2dlbmV0aWMgVHJlZSBCdWlsZGluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9tb2xiZXYvbXNwMjU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9Ik1vbGVjdWxhciBCaW9sb2d5IGFuZCBFdm9sdXRpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9Ik1vbCBCaW9sIEV2b2wiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjA3MzctNDAzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik5hcm5pYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcHJveGltYXRlIGxpa2VsaWhvb2QtcmF0aW8gdGVzdCBmb3IgYnJhbmNoZXM6IGEgZmFzdCwgYWNjdXJhdGUsIGFuZCBwb3dlcmZ1bCBhbHRlcm5hdGl2ZTsgU3lzdCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9NTM5LTU1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55LmZyOiByb2J1c3QgcGh5bG9nZW5ldGljIGFuYWx5c2lzIGZvciB0aGUgbm9uLXNwZWNpYWxpc3Q7IGNpdGF0aW9uX2F1dGhvcj0oMTIgY28tYXV0aG9ycyk7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9VzQ2NS1XNDY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9iQ29uczogcHJvYmFiaWxpc3RpYyBjb25zaXN0ZW5jeS1iYXNlZCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQ7IEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zMzAtMzQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NVVNDTEU6IG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIGhpZ2ggYWNjdXJhY3kgYW5kIGhpZ2ggdGhyb3VnaHB1dDsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xNzkyLTE3OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHNlcXVlbmNlcyBhbGlnbm1lbnQgZWRpdG9yIChNQVNFKTsgVHJlbmRzIEJpb2NoZW0gU2NpOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MzIxLTMyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uZmlkZW5jZSBsaW1pdHMgb24gcGh5bG9nZW5pZXM6IGFuIGFwcHJvYWNoIHVzaW5nIHRoZSBib290c3RyYXA7IEV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTc4My03OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RleHQ9RGlzdHJpYnV0ZWQgYnkgdGhlIGF1dGhvci4gU2VhdHRsZSAoV0EpOiBEZXBhcnRtZW50IG9mIEdlbm9tZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBXYXNoaW5ndG9uOyBQSFlMSVAgKHBoeWxvZ2VueSBpbmZlcmVuY2UgcGFja2FnZSkgdmVyc2lvbiAzLjUyOyAgY2l0YXRpb25feWVhcj0xOTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TRUFWSUVXIGFuZCBQSFlMT19XSU46IHR3byBncmFwaGljIHRvb2xzIGZvciBzZXF1ZW5jZSBhbGlnbm1lbnQgYW5kIG1vbGVjdWxhciBwaHlsb2dlbnk7IENvbXB1dCBBcHBsIEJpb3NjaTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTU0My01NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJJT05KOiBhbiBpbXByb3ZlZCB2ZXJzaW9uIG9mIHRoZSBOSiBhbGdvcml0aG0gYmFzZWQgb24gYSBzaW1wbGUgbW9kZWwgb2Ygc2VxdWVuY2UgZGF0YTsgTW9sIEJpb2wgRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTY4NS02OTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlbW90ZSBhY2Nlc3MgdG8gQUNOVUMgbnVjbGVvdGlkZSBhbmQgcHJvdGVpbiBzZXF1ZW5jZSBkYXRhYmFzZXMgYXQgUEJJTDsgQmlvY2hpbWllOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9NTU1LTU2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUsIGZhc3QsIGFuZCBhY2N1cmF0ZSBhbGdvcml0aG0gdG8gZXN0aW1hdGUgbGFyZ2UgcGh5bG9nZW5pZXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyBTeXN0IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz02OTYtNzA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMIFY6IGltcHJvdmVkIHNvZnR3YXJlIGZvciBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQ7IENvbXB1dCBBcHBsIEJpb3NjaTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTg5LTE5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdHJ1Y3RpbmcgZXZvbHV0aW9uYXJ5IHRyZWVzIGZyb20gRE5BIGFuZCBwcm90ZWluIHNlcXVlbmNlczogcGFyYWxpbmVhciBkaXN0YW5jZXM7IFByb2MgTmF0bCBBY2FkIFNjaSBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xNDU1LTE0NTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0YWwgVyBhbmQgQ2x1c3RhbCBYIHZlcnNpb24gMi4wOyBjaXRhdGlvbl9hdXRob3I9KDEzIGNvLWF1dGhvcnMpOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctMjk0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5iaWFzZWQgZXN0aW1hdGlvbiBvZiB0aGUgcmF0ZXMgb2Ygc3lub255bW91cyBhbmQgbm9uc3lub255bW91cyBzdWJzdGl0dXRpb247IEogTW9sIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz05Ni05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkVYVVM6IGFuIGV4dGVuc2libGUgZmlsZSBmb3JtYXQgZm9yIHN5c3RlbWF0aWMgaW5mb3JtYXRpb247IFN5c3QgQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTU5MC02MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuaGFuY2VkIGdyYXBoaWMgbWF0cml4IGFuYWx5c2lzIG9mIG51Y2xlaWMgYWNpZCBhbmQgcHJvdGVpbiBzZXF1ZW5jZXM7IFByb2MgTmF0bCBBY2FkIFNjaSBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz03NjY1LTc2Njk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2x1bWJpYSBVbml2ZXJzaXR5IFByZXNzLCBOZXcgWW9yazsgTW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VC1Db2ZmZWU6IGEgbm92ZWwgbWV0aG9kIGZvciBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzOyBKIE1vbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTMwMjsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0yMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIHRvb2xzIGZvciBiaW9sb2dpY2FsIHNlcXVlbmNlIGNvbXBhcmlzb247IFByb2MgTmF0bCBBY2FkIFNjaSBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNDQ0LTI0NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZW9yZXRpY2FsIGZvdW5kYXRpb24gb2YgdGhlIG1pbmltdW0tZXZvbHV0aW9uIG1ldGhvZCBvZiBwaHlsb2dlbmV0aWMgaW5mZXJlbmNlOyBNb2wgQmlvbCBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTA3My0xMDk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UZXN0cyBvZiBhcHBsaWNhYmlsaXR5IG9mIHNldmVyYWwgc3Vic3RpdHV0aW9uIG1vZGVscyBmb3IgRE5BIHNlcXVlbmNlIGRhdGE7IE1vbCBCaW9sIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMzEtMTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmVpZ2hib3Itam9pbmluZyBtZXRob2Q6IGEgbmV3IG1ldGhvZCBmb3IgcmVjb25zdHJ1Y3RpbmcgcGh5bG9nZW5ldGljIHRyZWVzOyBNb2wgQmlvbCBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MDYtNDI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdGUgb24gdGhlIG5laWdoYm9yLWpvaW5pbmcgYWxnb3JpdGhtIG9mIFNhaXRvdSBhbmQgTmVpOyBNb2wgQmlvbCBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz03MjktNzMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNDogTW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljcyBBbmFseXNpcyAoTUVHQSkgU29mdHdhcmUgVmVyc2lvbiA0LjA7IE1vbCBCaW9sIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTE1OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9tYmUvYXJ0aWNsZS1wZGYvMjcvMi8yMjEvMjkxNjMzMy9tc3AyNTkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIHByZXNlbnQgU2VhVmlldyB2ZXJzaW9uIDQsIGEgbXVsdGlwbGF0Zm9ybSBwcm9ncmFtIGRlc2lnbmVkIHRvIGZhY2lsaXRhdGUgbXVsdGlwbGUgYWxpZ25tZW50IGFuZCBwaHlsb2dlbmV0aWMgdHJlZSBidWlsZGluZyBmcm9tIG1vbGVjdWxhciBzZXF1ZW5jIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFfi8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj05NzAyNDcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVCZzaXRlSWQ9NTMyNR4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkYAcX9ozXdvEo2lsF7dZlqEX8KfU=" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5325" type="hidden">
        <input id="hdnAdDelaySeconds" value="5000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="sticky" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f27%2f2%2f221%2f970247" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f27%2f2%2f221%2f970247" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196058">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196059">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196056">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196060">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196061">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196062">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196063">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196064">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196065">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196066">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196057">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196067">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196068">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196069">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196070">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196071">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196072">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196073">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196074">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196075">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/mbe" class=" journal-logo-container">

            <img id="logo-MolecularBiologyandEvolution" class="journal-logo" src="mbe_title188616415.svg" alt="Molecular Biology and Evolution">

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.smbe.org/smbe/" target="" class="society-logo-container">
                                <img id="logo-SocietyforMolecularBiologyandEvolution" class="society-logo" src="mbe_h1-1277340080.svg" alt="Society for Molecular Biology and Evolution">
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196058">
            <a href="https://academic.oup.com/mbe/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196059">
            <a href="https://academic.oup.com/mbe/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196056">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196060">
            <a href="http://mc.manuscriptcentral.com/mbe" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196061">
            <a href="https://academic.oup.com/mbe/pages/manuscript_types" class="nav-link ">Manuscript Types</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196062">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Process" class="nav-link ">Editorial Process</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196063">
            <a href="https://academic.oup.com/mbe/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196064">
            <a href="https://academic.oup.com/mbe/pages/Authors_Page_Charges" class="nav-link ">Publication Charges</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196065">
            <a href="https://academic.oup.com/mbe/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196066">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1196057">
            <a href="https://academic.oup.com/mbe/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196067">
            <a href="https://academic.oup.com/mbe/pages/About" class="nav-link ">About Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196068">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">About the Society for Molecular Biology and Evolution</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196069">
            <a href="https://academic.oup.com/mbe/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196070">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196071">
            <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196072">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196073">
            <a href="https://academic.oup.com/mbe/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196074">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1196075">
            <a href="http://www.smbe.org/smbe/" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="SMBE Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/smbe" type="hidden">
    <input name="siteID" id="hfsiteID" value="5325" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5538" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=41619" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/mbe" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

        </section>
        <main id="main" class="js-main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/mbe/issue/27/2"><img id="issueImage" class="fb-featured-image" src="m_mbe27_2.gif" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/mbe/issue/27/2">
                        <div class="volume trailing-comma">Volume 27</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2010        </div>
        
    </div>

</div> 
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="16680099">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#16680099">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="16680117">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#16680117">References</a>
            </div>
        </li>
</ul>
 
    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/mbe/article/27/2/217/968234">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/mbe/article/27/2/225/967672">Next &gt;</a>
        </li>
</ul> 
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building
<i class="icon-availability_free" title="Free"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manolo Gouy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratoire de Biométrie et Biologie Evolutive, CNRS UMR 5558, Université Lyon 1, Université de Lyon, Villeurbanne, France</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div><strong>Corresponding author:</strong> E-mail: <a href="mailto:mgouy@biomserv.univ-lyon1.fr" target="">mgouy@biomserv.univ-lyon1.fr</a>.
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Manolo+Gouy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gouy%20M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Gouy%20Manolo%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Manolo Gouy<i class="icon-general-mail"></i></a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stéphane Guindon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Méthodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS UMR 5506, Université Montpellier II, Montpellier, France</div><div class="aff"><div class="label -label">3</div>Department of Statistics, University of Auckland, Auckland, New Zealand</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=St%c3%a9phane+Guindon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guindon%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Guindon%20St%C3%A9phane%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stéphane Guindon</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Olivier Gascuel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Méthodes et Algorithmes pour la Bioinformatique, LIRMM, CNRS UMR 5506, Université Montpellier II, Montpellier, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/mbe/search-results?f_Authors=Olivier+Gascuel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gascuel%20O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Gascuel%20Olivier%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Olivier Gascuel</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Molecular Biology and Evolution</em>, Volume 27, Issue 2, February 2010, Pages 221–224, <a href="https://doi.org/10.1093/molbev/msp259">https://doi.org/10.1093/molbev/msp259</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 October 2009</div>
                        </div>
                    </div>
                            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap">
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="970247" href="https://academic.oup.com/mbe/article-pdf/27/2/221/2916333/msp259.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Manolo Gouy, Stéphane Guindon, Olivier Gascuel,  SeaView Version 
4: A Multiplatform Graphical User Interface for Sequence Alignment and 
Phylogenetic Tree Building, <em>Molecular Biology and Evolution</em>, Volume 27, Issue 2, February 2010, Pages 221–224, <a href="https://doi.org/10.1093/molbev/msp259">https://doi.org/10.1093/molbev/msp259</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=0">Ris (Zotero)</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=1">EndNote</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=2">BibTex</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=3">Medlars</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=0">ProCite</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=3">RefWorks</a></li>
        <li><a href="https://academic.oup.com/Citation/Download?resourceId=970247&amp;resourceType=3&amp;citationFormat=0">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2019 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div> 
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
        <div class="module-widget">
    <i class="icon-menu_permissions">
        <span class="screenreader-text">Permissions Icon</span>
    </i>
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1537-1719&amp;title=SeaView%20Version%204%3A%20A%20Multiplatform%20Graphical%20User%20Interface%20for%20Sequence%20Alignment%20and%20Phylogenetic%20Tree%20Building&amp;publicationDate=2009-10-23&amp;volumeNum=27&amp;issueNum=2&amp;author=Gouy%2C%20Manolo%3B%20Guindon%2C%20St%C3%A9phane&amp;startPage=221&amp;endPage=224&amp;contentId=10.1093%2Fmolbev%2Fmsp259&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
</div>
 
    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                    <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Molecular Biology and Evolution</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f27%2f2%2f221%2f970247" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Molecular Biology and Evolution</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Parent">All  SMBE Journals</option><option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/mbe/advanced-search?page=1&amp;fl_SiteID=5325&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="mbe" type="hidden"> 
    </div>

                        </div>
                    </div>                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="16680099" id="16680099" class="abstract-title">Abstract</h2>
<section class="abstract"><p>We present SeaView version 4, a 
multiplatform program designed to facilitate multiple alignment and 
phylogenetic tree building from molecular sequence data through the use 
of a graphical user interface. SeaView version 4 combines all the 
functions of the widely used programs SeaView (in its previous versions)
 and Phylo_win, and expands them by adding network access to sequence 
databases, alignment with arbitrary algorithm, maximum-likelihood tree 
building with PhyML, and display, printing, and copy-to-clipboard of 
rooted or unrooted, binary or multifurcating phylogenetic trees. In 
relation to the wide present offer of tools and algorithms for 
phylogenetic analyses, SeaView is especially useful for teaching and for
 occasional users of such software. SeaView is freely available at <a class="link link-uri" href="http://pbil.univ-lyon1.fr/software/seaview" target="">http://pbil.univ-lyon1.fr/software/seaview</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="SeaView">SeaView</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;multiple sequence alignment&quot;">multiple sequence alignment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular phylogeny&quot;">molecular phylogeny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="PhyML">PhyML</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;graphical user interface&quot;">graphical user interface</a></div><p>Multiple
 alignment and phylogenetic tree reconstruction from molecular sequence 
data are key tasks for many molecular evolution analyses. They involve 
the sequential use of several programs that perform part of the complete
 procedure and often require series of tedious and error-prone data 
reformatting to transfer sequences and trees between these programs. The
 computer programs SeaView and Phylo_win pioneered the use of graphical 
user interfaces for performing multiple sequence alignment and 
phylogenetic tree reconstruction (<a reveal-id="bib8" data-open="bib8" class="link link-ref link-reveal xref-bibr">Galtier et al. 1996</a>).
 These programs have been widely used but were lacking access to 
recently developed methods for maximum-likelihood tree estimation. We 
present here SeaView version 4, a program that allows its users to 
perform the complete phylogenetic analysis of a set of homologous DNA or
 protein sequences, from network-based sequence extraction from public 
databases to tree building and display using up-to-date alignment and 
maximum-likelihood tree-building algorithms (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">fig. 1</a>).</p>                        <a id="16680102" scrollto-destination="16680102"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IG</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_molbiolevolmsp259f01_3c.gif" alt="The two major SeaView window types display sequence data and phylogenetic trees. Two of the displayed menus, two pilot multiple alignment algorithms and tree-building methods. Tree display tools allow printing, copy to clipboard, rerooting, zooming in and out, restricting display to a subtree, and the use of three alternative (squared, circular, and cladogram) tree-drawing formats." path-from-xml="molbiolevolmsp259f01_3c.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/16680102/molbiolevolmsp259f01_3c.gif" path-from-xml="molbiolevolmsp259f01_3c.gif" target="_blank">View large</a><a section="16680102" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/27/2/10.1093/molbev/msp259/2/molbiolevolmsp259f01_3c.gif?Expires=1556559203&amp;Signature=fcAepltaUc1amJcpbSh3fr4eljAXeGRsuMBjkMm1LBvJ%7EfLTxRcUapaxSWSgSBoDp%7EY0NIrIywET4AvplUyk7j6yw8zmbnNpQOTizsPizJnDV7FlwvvCW0glp4rJszcW5Jv9vGJzw0X520P9xgzC5rFT2kPoYXGl3mAwbOKVyn310cSszrAh0SJff-og1Y0y-QVo-KCKBQ-ymTHFxK1hE4lhZBQ8K1gTl-2b9M4ssqkqsxIVbnIyTfIC94DIYhbzbF0m8axbCppDvtDshFxegPSpRgNpc9iIw5bgMuX8U55Li2hkyW2WAAlc%7EGDsVfxEfXcojtA05oopmdJorPRRlw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=16680102&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="molbiolevolmsp259f01_3c.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>The
 two major SeaView window types display sequence data and phylogenetic 
trees. Two of the displayed menus, two pilot multiple alignment 
algorithms and tree-building methods. Tree display tools allow printing,
 copy to clipboard, rerooting, zooming in and out, restricting display 
to a subtree, and the use of three alternative (squared, circular, and 
cladogram) tree-drawing formats.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IG</span>. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_molbiolevolmsp259f01_3c.gif" alt="The two major SeaView window types display sequence data and phylogenetic trees. Two of the displayed menus, two pilot multiple alignment algorithms and tree-building methods. Tree display tools allow printing, copy to clipboard, rerooting, zooming in and out, restricting display to a subtree, and the use of three alternative (squared, circular, and cladogram) tree-drawing formats." path-from-xml="molbiolevolmsp259f01_3c.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/16680102/molbiolevolmsp259f01_3c.gif" path-from-xml="molbiolevolmsp259f01_3c.gif" target="_blank">View large</a><a section="16680102" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/27/2/10.1093/molbev/msp259/2/molbiolevolmsp259f01_3c.gif?Expires=1556559203&amp;Signature=fcAepltaUc1amJcpbSh3fr4eljAXeGRsuMBjkMm1LBvJ%7EfLTxRcUapaxSWSgSBoDp%7EY0NIrIywET4AvplUyk7j6yw8zmbnNpQOTizsPizJnDV7FlwvvCW0glp4rJszcW5Jv9vGJzw0X520P9xgzC5rFT2kPoYXGl3mAwbOKVyn310cSszrAh0SJff-og1Y0y-QVo-KCKBQ-ymTHFxK1hE4lhZBQ8K1gTl-2b9M4ssqkqsxIVbnIyTfIC94DIYhbzbF0m8axbCppDvtDshFxegPSpRgNpc9iIw5bgMuX8U55Li2hkyW2WAAlc%7EGDsVfxEfXcojtA05oopmdJorPRRlw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=16680102&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="molbiolevolmsp259f01_3c.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>The
 two major SeaView window types display sequence data and phylogenetic 
trees. Two of the displayed menus, two pilot multiple alignment 
algorithms and tree-building methods. Tree display tools allow printing,
 copy to clipboard, rerooting, zooming in and out, restricting display 
to a subtree, and the use of three alternative (squared, circular, and 
cladogram) tree-drawing formats.</p></div></div><p>SeaView can read and 
write the most widely used file formats defined for holding aligned or 
unaligned protein or nucleotide sequence data: Fasta (<a reveal-id="bib19" data-open="bib19" class="link link-ref link-reveal xref-bibr">Pearson and Lipman 1988</a>), interleaved Phylip (<a reveal-id="bib7" data-open="bib7" class="link link-ref link-reveal xref-bibr">Felsenstein 1993</a>), Clustal (<a reveal-id="bib12" data-open="bib12" class="link link-ref link-reveal xref-bibr">Higgins et al. 1992</a>), multiple sequence file of the Genetics Computer Group package, Nexus (<a reveal-id="bib15" data-open="bib15" class="link link-ref link-reveal xref-bibr">Maddison et al. 1997</a>), and Mase (<a reveal-id="bib5" data-open="bib5" class="link link-ref link-reveal xref-bibr">Faulkner and Jurka 1988</a>).
 The last two formats allow for much useful information besides sequence
 and name, that is, trees, species, and sites selections, sequence 
annotations. SeaView can also import sequence data from the major public
 sequence databases using a network access (<a reveal-id="bib10" data-open="bib10" class="link link-ref link-reveal xref-bibr">Gouy and Delmotte 2008</a>)
 to daily (for GenBank and EMBL) or weekly updated (for 
UniProt/SwissProt) databases. Imported sequences can be identified by 
name, accession number, or keyword and named either with their database 
identifier or using the species name of their organism of origin. 
SeaView can also directly import from nucleotide databases most feature 
table elements (e.g., coding sequence, rRNA, and non-codingRNA) and 
select those whose annotations contain a user-given character string (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">fig. 2</a>).</p>                        <a id="16680104" scrollto-destination="16680104"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IG</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_molbiolevolmsp259f02_ht.gif" alt="Database sequence import dialog. This example will import into SeaView the single coding sequence (CDS) from EMBL's entry AE000782 (Archaeoglobus fulgidus complete genome sequence) containing the string (gyrA) in its database annotation and will name it Archaeoglobus." path-from-xml="molbiolevolmsp259f02_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/16680104/molbiolevolmsp259f02_ht.gif" path-from-xml="molbiolevolmsp259f02_ht.gif" target="_blank">View large</a><a section="16680104" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/27/2/10.1093/molbev/msp259/2/molbiolevolmsp259f02_ht.gif?Expires=1556559203&amp;Signature=fzShiLdAde29RwuWrQ8drcLr9Dl9QBqNqG%7EvowMdl0jgrAXEAezS6a7pDaHG530bFv1iWRnFBoy4DCtplqCG6%7Ev8C6X0EuUR59B%7E6SXU-OPffEp988dhSUs2k%7EcBvDGmAYkKog0N8g--9enlCU3GlKsRsg3%7E4h9VJtoN9sxcCqLkbt51ADmHCc1TmaxnnbsZuBTCFKYRz1ea5s0UJ-Bb2GInWm%7EvBmUB3IFfG%7E2t4G9usa75%7Eq8fGQl6kpafnZxGF2whJMSZB9vtLT5pTIyyv4FT8p5bGkk8nr1ASOSywSlK8zCo8wMcL04fIwhIMrFcOIhn4ICasZHy8oFX4Qg2IA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=16680104&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="molbiolevolmsp259f02_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Database
 sequence import dialog. This example will import into SeaView the 
single coding sequence (CDS) from EMBL's entry AE000782 (<em>Archaeoglobus fulgidus</em> complete genome sequence) containing the string (gyrA) in its database annotation and will name it <em>Archaeoglobus</em>.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IG</span>. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_molbiolevolmsp259f02_ht.gif" alt="Database sequence import dialog. This example will import into SeaView the single coding sequence (CDS) from EMBL's entry AE000782 (Archaeoglobus fulgidus complete genome sequence) containing the string (gyrA) in its database annotation and will name it Archaeoglobus." path-from-xml="molbiolevolmsp259f02_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/16680104/molbiolevolmsp259f02_ht.gif" path-from-xml="molbiolevolmsp259f02_ht.gif" target="_blank">View large</a><a section="16680104" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/27/2/10.1093/molbev/msp259/2/molbiolevolmsp259f02_ht.gif?Expires=1556559203&amp;Signature=fzShiLdAde29RwuWrQ8drcLr9Dl9QBqNqG%7EvowMdl0jgrAXEAezS6a7pDaHG530bFv1iWRnFBoy4DCtplqCG6%7Ev8C6X0EuUR59B%7E6SXU-OPffEp988dhSUs2k%7EcBvDGmAYkKog0N8g--9enlCU3GlKsRsg3%7E4h9VJtoN9sxcCqLkbt51ADmHCc1TmaxnnbsZuBTCFKYRz1ea5s0UJ-Bb2GInWm%7EvBmUB3IFfG%7E2t4G9usa75%7Eq8fGQl6kpafnZxGF2whJMSZB9vtLT5pTIyyv4FT8p5bGkk8nr1ASOSywSlK8zCo8wMcL04fIwhIMrFcOIhn4ICasZHy8oFX4Qg2IA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=16680104&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="molbiolevolmsp259f02_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Database
 sequence import dialog. This example will import into SeaView the 
single coding sequence (CDS) from EMBL's entry AE000782 (<em>Archaeoglobus fulgidus</em> complete genome sequence) containing the string (gyrA) in its database annotation and will name it <em>Archaeoglobus</em>.</p></div></div><p>Nucleotide
 sequences can be translated to protein using any user- or 
database-assigned genetic code, so operations such as alignment and tree
 building can be performed at the nucleotide or the protein levels. 
Unaligned protein-coding DNA sequences can be translated to protein, 
aligned, and displayed back as DNA sequences, a procedure that yields 
more realistic coding sequence alignments than would result from 
nucleotide-level alignment. Protein-coding DNA sequences can also be 
displayed by assigning the same color to all synonymous codons of the 
corresponding amino acid. Alignments can alternatively be displayed in 
reference mode, that is, where only residues that differ from the 
homologous one in a reference sequence are shown. Several sequence 
alignments can be handled simultaneously and copy/paste and 
concatenation operations can be performed between them. As far as 
display is concerned, SeaView accepts large sequence numbers (tens of 
thousand) and long sequences. SeaView is able to handle any number of 
sequence and site sets. Such sets can be named and saved in the Nexus or
 Mase file formats for subsequent use, by tree-building algorithms for 
instance.</p><div class="block-child-p">SeaView relies on external 
programs to perform multiple sequence alignments. Two programs are 
initially available: ClustalW version 2 (<a reveal-id="bib13" data-open="bib13" class="link link-ref link-reveal xref-bibr">Larkin et al. 2007</a>) and Muscle (<a reveal-id="bib4" data-open="bib4" class="link link-ref link-reveal xref-bibr">Edgar 2004</a>).
 These programs are run with their default parameter values that have 
been chosen by their authors to perform well in most cases. When special
 parameter values are needed, they can be specified once using SeaView's
 user interface and reused for subsequent alignment operations. 
Alignment can be applied to all or to selected sequences or part of 
sequences. Profile alignment that aims at adding more sequences to a 
preexisting alignment can be done with both Muscle and ClustalW. SeaView
 is also able to drive any external sequence alignment program provided 
this program reads and outputs Fasta-formatted sequence data and can be 
run by a command line of the form “program_name arguments.” SeaView 
communicates with external alignment programs through a list of 
arguments that is initially defined by the user. This definition is made
 by entering once in a dialog box the list of arguments suitable for 
running this program, replacing the input file name by “%f.pir” and the 
output file name by “%f.out.” The external alignment algorithm becomes 
directly usable after that step. For example, SeaView's interface to 
T-Coffee (<a reveal-id="bib18" data-open="bib18" class="link link-ref link-reveal xref-bibr">Notredame et al. 2000</a>) corresponds to the following argument list<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="" alt="graphic" path-from-xml="molbiolevolmsp259fx1_ht"></div></div>which
 contains the arguments expected by T_Coffee to align a Fasta-formatted 
file and to output its Fasta-formatted results without reordering 
sequences. Likewise, SeaView's interface to Probcons (<a reveal-id="bib3" data-open="bib3" class="link link-ref link-reveal xref-bibr">Do et al. 2005</a>) is straightforward:<a id="" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id=""><img class="content-image" src="" alt="graphic" path-from-xml="molbiolevolmsp259fx2_ht"></div></div></div><p>SeaView
 is also a multiple sequence alignment editor that can be used to add or
 remove one or several gaps in one or several sequences simultaneously. A
 dot-plot analysis (<a reveal-id="bib16" data-open="bib16" class="link link-ref link-reveal xref-bibr">Maizel and Lenk 1981</a>)
 can be performed between any two sequences to visually check whether 
alignment algorithms missed regions with high sequence similarity.</p><p>SeaView relies on PhyML version 3 (<a reveal-id="bib11" data-open="bib11" class="link link-ref link-reveal xref-bibr">Guindon and Gascuel 2003</a>)
 for maximum-likelihood phylogenetic tree reconstruction. Here again, 
PhyML is used as an independent program. Thus, future updates to PhyML 
will be accessible to SeaView users as soon as they will have installed 
the revised program. Tree building can be applied to all or to selected 
sequences and to all or selected sequence sites. Most PhyML options can 
be set through the graphical interface, both for nucleotide and 
protein-level analyses (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">fig. 3</a>). Thus, branch support can be estimated either with the approximate likelihood-ratio test (<a reveal-id="bib1" data-open="bib1" class="link link-ref link-reveal xref-bibr">Anisimova and Gascuel 2006</a>) or by bootstrap resampling (<a reveal-id="bib6" data-open="bib6" class="link link-ref link-reveal xref-bibr">Felsenstein 1985</a>).</p>                        <a id="16680109" scrollto-destination="16680109"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label">F<span class="small-caps">IG</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_molbiolevolmsp259f03_ht.gif" alt="SeaView dialog for setting PhyML options applied to nucleotide sequences." path-from-xml="molbiolevolmsp259f03_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/16680109/molbiolevolmsp259f03_ht.gif" path-from-xml="molbiolevolmsp259f03_ht.gif" target="_blank">View large</a><a section="16680109" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/27/2/10.1093/molbev/msp259/2/molbiolevolmsp259f03_ht.gif?Expires=1556559203&amp;Signature=DO2KDZq97FFeR9kzfU-435QwtmxR0DQus-NIsAkci9Xor848cs7CgZLCNUypGUBXwD6n3H3WG1GCRRj6AbFvliF5RY1EV9jTBe5i%7ELKDHAbsG8uAb9tih3tjUGlXN4lrv1edwVHHzAaXfDQH-LVWIbbfvLrwtVo9IoyBhmUaSpyqvI9231r%7EDodsoX-st%7Ez2n92zhwetMfeWm9kD0dhYDl4tp4vsCflET1TfsVcoF1BlNTBdwu6o4Jn6Shy2oaDISoE%7Ewu1TMUillUkbP5BpCZDkjTg1nuXZkcPGHjCYoFUaG97XTF3v7CF0GVu%7EDfUph2rr78yMvTwncHoLh0n4Lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=16680109&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="molbiolevolmsp259f03_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>SeaView dialog for setting PhyML options applied to nucleotide sequences.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">F<span class="small-caps">IG</span>. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_molbiolevolmsp259f03_ht.gif" alt="SeaView dialog for setting PhyML options applied to nucleotide sequences." path-from-xml="molbiolevolmsp259f03_ht.gif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/16680109/molbiolevolmsp259f03_ht.gif" path-from-xml="molbiolevolmsp259f03_ht.gif" target="_blank">View large</a><a section="16680109" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/mbe/27/2/10.1093/molbev/msp259/2/molbiolevolmsp259f03_ht.gif?Expires=1556559203&amp;Signature=DO2KDZq97FFeR9kzfU-435QwtmxR0DQus-NIsAkci9Xor848cs7CgZLCNUypGUBXwD6n3H3WG1GCRRj6AbFvliF5RY1EV9jTBe5i%7ELKDHAbsG8uAb9tih3tjUGlXN4lrv1edwVHHzAaXfDQH-LVWIbbfvLrwtVo9IoyBhmUaSpyqvI9231r%7EDodsoX-st%7Ez2n92zhwetMfeWm9kD0dhYDl4tp4vsCflET1TfsVcoF1BlNTBdwu6o4Jn6Shy2oaDISoE%7Ewu1TMUillUkbP5BpCZDkjTg1nuXZkcPGHjCYoFUaG97XTF3v7CF0GVu%7EDfUph2rr78yMvTwncHoLh0n4Lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=16680109&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=&amp;siteId=5325" class="download-slide" path-from-xml="molbiolevolmsp259f03_ht.gif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>SeaView dialog for setting PhyML options applied to nucleotide sequences.</p></div></div><p>SeaView includes two distance-based tree reconstruction methods: Neighbor-Joining (<a reveal-id="bib22" data-open="bib22" class="link link-ref link-reveal xref-bibr">Saitou and Nei 1987</a>; <a reveal-id="bib23" data-open="bib23" class="link link-ref link-reveal xref-bibr">Studier and Keppler 1988</a>) and BioNJ (<a reveal-id="bib9" data-open="bib9" class="link link-ref link-reveal xref-bibr">Gascuel 1997</a>).
 These can be applied to various nucleotide and protein sequence 
pairwise distances and combined with bootstrap resampling for 
branch-support estimation. Nucleotide-level distances are observed 
divergence, Jukes and Cantor, Kimura's two-parameter, 
Hasegawa–Kishino–Yano (see <a reveal-id="bib21" data-open="bib21" class="link link-ref link-reveal xref-bibr">Rzhetsky and Nei 1995</a> for these first 4 distances), LogDet (Lake 1994), and Li's nonsynonymous (<em>K</em><sub>a</sub>) and synonymous (<em>K</em><sub>s</sub>) distances for protein-coding sequences (<a reveal-id="bib14" data-open="bib14" class="link link-ref link-reveal xref-bibr">Li 1993</a>). Protein-level distances are observed, Poisson and Kimura's (<a reveal-id="bib17" data-open="bib17" class="link link-ref link-reveal xref-bibr">Nei 1987</a>).
 Gap-containing sites are by default excluded from pairwise distance 
computations. Alternatively, sites that are gap-free in two sequences 
can be used to compute the distance for this sequence pair. The branch 
lengths of any user-given tree topology can be computed by minimization 
of the sum of squared differences between evolutionary and patristic 
distances (<a reveal-id="bib20" data-open="bib20" class="link link-ref link-reveal xref-bibr">Rzhetsky and Nei 1993</a>).</p><p>SeaView
 can also reconstruct maximum-parsimony phylogenetic trees using code 
extracted from Dnapars and Protpars programs (Phylip version 3.52, <a reveal-id="bib7" data-open="bib7" class="link link-ref link-reveal xref-bibr">Felsenstein 1993</a>).
 Parsimony computation can be combined with bootstrap resampling of 
sites and can be repeated a user-chosen number of times after randomly 
changing the input order of sequences. The parsimony score of any 
user-given tree can also be computed. SeaView completes parsimony 
analyses by computing the strict consensus of all equally parsimonious 
trees found.</p><p>When tree building is completed. SeaView draws the resulting phylogenetic tree on the screen (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">fig. 1</a>).
 Plotted trees can be displayed with or without branch lengths (as when 
computed by parsimony), with or without branch-support values 
(typically, bootstrap scores or approximate Likelihood Ratio Test 
probabilities), binary or multifurcating, rooted or unrooted. 
Phylogenetic trees are initially rooted at the point in the tree that 
minimizes the variance of root-to-tip distances, but they can also be 
plotted as unrooted trees using a circular display or as cladograms 
containing topological but no branch-length information. The user can 
change the tree root and exchange the order of the two child lineages of
 a node. Trees can be saved to Newick, PDF, or PostScript files, and, 
under the Microsoft Windows and Mac OS X environments, printed or copied
 to the clipboard for communication with graphical tools such as Office 
applications. Aligned sequences can be reordered following their 
corresponding tree, and sequences that belong to a subtree can be 
selected in the corresponding alignment. Several tools such as subtree 
display, pattern matching in sequence names, and vertical zoom help 
dealing with large trees. A graphical tree editor allows topological 
changes by combining two basic operations: clade displacement and clade 
suppression.</p><p>SeaView version 4 is freely available at <a class="link link-uri" href="http://pbil.univ-lyon1.fr/software/seaview" target="">http://pbil.univ-lyon1.fr/software/seaview</a> for four computer platforms (Microsoft Windows, Mac OS X, Linux, and SPARC/Solaris) and as source code.</p><p>Many
 software packages are available for multiple sequence alignment and 
phylogenetic tree reconstruction. SeaView is especially comparable with 
MEGA 4 that also provides an elaborate graphical user interface for 
multiple sequence alignment and distance or parsimony tree 
reconstruction and display (<a reveal-id="bib24" data-open="bib24" class="link link-ref link-reveal xref-bibr">Tamura et al. 2007</a>).
 SeaView is less versatile than MEGA for pairwise distance computations 
and lacks features such as neutrality or molecular tests but is unique 
in being available for all major computer platforms and in allowing 
maximum-likelihood tree reconstruction with PhyML. SeaView version 4 is 
especially valuable for teaching molecular phylogeny because of its 
availability at no fee for all users and because its user interface 
graphically expresses the conceptual steps involved in phylogenetic 
analyses. SeaView is also helpful for occasional users of phylogenetic 
tree reconstruction because it frees them from being confronted to many 
technical details concerning file formats and program options. SeaView 
thus pursues similar objectives to those of the phylogeny web server 
Phylogeny.fr (<a reveal-id="bib2" data-open="bib2" class="link link-ref link-reveal xref-bibr">Dereeper et al. 2008</a>)
 exploiting the user's computing resources. Because it performs, using 
PhyML, maximum-likelihood analyses at both nucleotide and protein 
levels, implements most current evolutionary models and computes 
statistical branch support, SeaView is also expected to be useful to 
seasoned phylogeneticists.</p><p>We are grateful to the Fast Light Toolkit team for its wonderful cross-platform graphical user interface toolkit (<a class="link link-uri" href="http://www.fltk.org/" target="">http://www.fltk.org</a>). We thank Nicolas Galtier for contributing code from the Phylo_win program.</p>                            <h2 scrollto-destination="16680117" id="16680117" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="bib1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib1" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anisimova</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative</div>,&nbsp;<div class="source">Syst Biol</div>&nbsp;,&nbsp;<div class="year">2006</div>,&nbsp;vol.&nbsp;<div class="volume">55</div>&nbsp;(pg.&nbsp;<div class="fpage">539</div>-<div class="lpage">552</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Approximate%20likelihood-ratio%20test%20for%20branches%3A%20a%20fast%2C%20accurate%2C%20and%20powerful%20alternative&amp;author=M%20Anisimova&amp;author=O%20Gascuel&amp;publication_year=2006&amp;journal=Syst%20Biol&amp;volume=55&amp;pages=539-552" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150600755453" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150600755453" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150600755453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16785212" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib2" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dereeper</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Guignon</div> <div class="given-names">V</div></div>,&nbsp; <div class="name"><div class="surname">Blanc</div> <div class="given-names">G</div></div>,&nbsp;et al.&nbsp;</div><div class="collab">(12 co-authors)</div><div class="article-title">Phylogeny.fr: robust phylogenetic analysis for the non-specialist</div>,&nbsp;<div class="source">Nucleic Acids Res</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">36</div>&nbsp;(pg.&nbsp;<div class="fpage">W465</div>-<div class="lpage">W469</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny.fr%3A%20robust%20phylogenetic%20analysis%20for%20the%20non-specialist&amp;author=A%20Dereeper&amp;author=V%20Guignon&amp;author=G%20Blanc&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=W465-W469" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn180" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn180" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkn180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18424797" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib3" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Do</div> <div class="given-names">CB</div></div>,&nbsp; <div class="name"><div class="surname">Mahabhashyam</div> <div class="given-names">MS</div></div>,&nbsp; <div class="name"><div class="surname">Brudno</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Batzoglou</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">ProbCons: probabilistic consistency-based multiple sequence alignment</div>,&nbsp;<div class="source">Genome Res</div>&nbsp;,&nbsp;<div class="year">2005</div>,&nbsp;vol.&nbsp;<div class="volume">15</div>&nbsp;(pg.&nbsp;<div class="fpage">330</div>-<div class="lpage">340</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ProbCons%3A%20probabilistic%20consistency-based%20multiple%20sequence%20alignment&amp;author=CB%20Do&amp;author=MS%20Mahabhashyam&amp;author=M%20Brudno&amp;author=S%20Batzoglou&amp;publication_year=2005&amp;journal=Genome%20Res&amp;volume=15&amp;pages=330-340" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2821705" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2821705" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1101%2Fgr.2821705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15687296" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib4" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">RC</div></div>.&nbsp;</div><div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>,&nbsp;<div class="source">Nucleic Acids Res</div>&nbsp;,&nbsp;<div class="year">2004</div>,&nbsp;vol.&nbsp;<div class="volume">32</div>&nbsp;(pg.&nbsp;<div class="fpage">1792</div>-<div class="lpage">1797</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=RC%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-1797" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib5" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Faulkner</div> <div class="given-names">DV</div></div>,&nbsp; <div class="name"><div class="surname">Jurka</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Multiple sequences alignment editor (MASE)</div>,&nbsp;<div class="source">Trends Biochem Sci</div>&nbsp;,&nbsp;<div class="year">1988</div>,&nbsp;vol.&nbsp;<div class="volume">13</div>&nbsp;(pg.&nbsp;<div class="fpage">321</div>-<div class="lpage">322</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20sequences%20alignment%20editor%20%28MASE%29&amp;author=DV%20Faulkner&amp;author=J%20Jurka&amp;publication_year=1988&amp;journal=Trends%20Biochem%20Sci&amp;volume=13&amp;pages=321-322" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0968-0004%2888%2990129-6" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0968-0004%2888%2990129-6" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2F0968-0004(88)90129-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3154283" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib6" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">Confidence limits on phylogenies: an approach using the bootstrap</div>,&nbsp;<div class="source">Evolution</div>&nbsp;,&nbsp;<div class="year">1985</div>,&nbsp;vol.&nbsp;<div class="volume">39</div>&nbsp;(pg.&nbsp;<div class="fpage">783</div>-<div class="lpage">791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20on%20phylogenies%3A%20an%20approach%20using%20the%20bootstrap&amp;author=J%20Felsenstein&amp;publication_year=1985&amp;journal=Evolution&amp;volume=39&amp;pages=783-791" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2408678" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2408678" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.2307%2F2408678"> </span></div>&nbsp;</div></div></div></div></div><div content-id="bib7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib7" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Felsenstein</div> <div class="given-names">J</div></div>.&nbsp;</div>,&nbsp;<div class="source">PHYLIP (phylogeny inference package) version 3.52</div>&nbsp;,&nbsp;<div class="year">1993</div>&nbsp;<div class="comment">Distributed by the author. Seattle (WA): Department of Genome Sciences, University of Washington</div></div></div></div></div><div content-id="bib8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib8" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galtier</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Gouy</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Gautier</div> <div class="given-names">C</div></div>.&nbsp;</div><div class="article-title">SEAVIEW and PHYLO_WIN: two graphic tools for sequence alignment and molecular phylogeny</div>,&nbsp;<div class="source">Comput Appl Biosci</div>&nbsp;,&nbsp;<div class="year">1996</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">543</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SEAVIEW%20and%20PHYLO_WIN%3A%20two%20graphic%20tools%20for%20sequence%20alignment%20and%20molecular%20phylogeny&amp;author=N%20Galtier&amp;author=M%20Gouy&amp;author=C%20Gautier&amp;publication_year=1996&amp;journal=Comput%20Appl%20Biosci&amp;volume=12&amp;pages=543-548" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9021275" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib9" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">BIONJ: an improved version of the NJ algorithm based on a simple model of sequence data</div>,&nbsp;<div class="source">Mol Biol Evol</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">14</div>&nbsp;(pg.&nbsp;<div class="fpage">685</div>-<div class="lpage">695</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BIONJ%3A%20an%20improved%20version%20of%20the%20NJ%20algorithm%20based%20on%20a%20simple%20model%20of%20sequence%20data&amp;author=O%20Gascuel&amp;publication_year=1997&amp;journal=Mol%20Biol%20Evol&amp;volume=14&amp;pages=685-695" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025808" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025808" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a025808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254330" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib10" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gouy</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Delmotte</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">Remote access to ACNUC nucleotide and protein sequence databases at PBIL</div>,&nbsp;<div class="source">Biochimie</div>&nbsp;,&nbsp;<div class="year">2008</div>,&nbsp;vol.&nbsp;<div class="volume">90</div>&nbsp;(pg.&nbsp;<div class="fpage">555</div>-<div class="lpage">562</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Remote%20access%20to%20ACNUC%20nucleotide%20and%20protein%20sequence%20databases%20at%20PBIL&amp;author=M%20Gouy&amp;author=S%20Delmotte&amp;publication_year=2008&amp;journal=Biochimie&amp;volume=90&amp;pages=555-562" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biochi.2007.07.003" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biochi.2007.07.003" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.biochi.2007.07.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17825976" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib11" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guindon</div> <div class="given-names">S</div></div>,&nbsp; <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O</div></div>.&nbsp;</div><div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>,&nbsp;<div class="source">Syst Biol</div>&nbsp;,&nbsp;<div class="year">2003</div>,&nbsp;vol.&nbsp;<div class="volume">52</div>&nbsp;(pg.&nbsp;<div class="fpage">696</div>-<div class="lpage">704</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=S%20Guindon&amp;author=O%20Gascuel&amp;publication_year=2003&amp;journal=Syst%20Biol&amp;volume=52&amp;pages=696-704" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150390235520" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib12" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>,&nbsp; <div class="name"><div class="surname">Bleasby</div> <div class="given-names">AJ</div></div>,&nbsp; <div class="name"><div class="surname">Fuchs</div> <div class="given-names">R</div></div>.&nbsp;</div><div class="article-title">CLUSTAL V: improved software for multiple sequence alignment</div>,&nbsp;<div class="source">Comput Appl Biosci</div>&nbsp;,&nbsp;<div class="year">1992</div>,&nbsp;vol.&nbsp;<div class="volume">8</div>&nbsp;(pg.&nbsp;<div class="fpage">189</div>-<div class="lpage">191</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20V%3A%20improved%20software%20for%20multiple%20sequence%20alignment&amp;author=DG%20Higgins&amp;author=AJ%20Bleasby&amp;author=R%20Fuchs&amp;publication_year=1992&amp;journal=Comput%20Appl%20Biosci&amp;volume=8&amp;pages=189-191" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1591615" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib25" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lake</div> <div class="given-names">JA</div></div>.&nbsp;</div><div class="article-title">Reconstructing evolutionary trees from DNA and protein sequences: paralinear distances</div>,&nbsp;<div class="source">Proc Natl Acad Sci USA</div>&nbsp;,&nbsp;<div class="year">1994</div>,&nbsp;vol.&nbsp;<div class="volume"> 91</div>&nbsp;(pg.&nbsp;<div class="fpage">1455</div>-<div class="lpage">1459</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20evolutionary%20trees%20from%20DNA%20and%20protein%20sequences%3A%20paralinear%20distances&amp;author=JA%20Lake&amp;publication_year=1994&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=91&amp;pages=1455-1459" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.4.1455" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.4.1455" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.91.4.1455"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8108430" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib13" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larkin</div> <div class="given-names">MA</div></div>,&nbsp; <div class="name"><div class="surname">Blackshields</div> <div class="given-names">G</div></div>,&nbsp; <div class="name"><div class="surname">Brown</div> <div class="given-names">NP</div></div>,&nbsp;et al.&nbsp;</div><div class="collab">(13 co-authors)</div><div class="article-title">Clustal W and Clustal X version 2.0</div>,&nbsp;<div class="source">Bioinformatics</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">23</div>&nbsp;(pg.&nbsp;<div class="fpage">2947</div>-<div class="lpage">2948</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=MA%20Larkin&amp;author=G%20Blackshields&amp;author=NP%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-2948" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib14" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">WH</div></div>.&nbsp;</div><div class="article-title">Unbiased estimation of the rates of synonymous and nonsynonymous substitution</div>,&nbsp;<div class="source">J Mol Evol</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">36</div>&nbsp;(pg.&nbsp;<div class="fpage">96</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unbiased%20estimation%20of%20the%20rates%20of%20synonymous%20and%20nonsynonymous%20substitution&amp;author=WH%20Li&amp;publication_year=1993&amp;journal=J%20Mol%20Evol&amp;volume=36&amp;pages=96-99" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02407308" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02407308" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02407308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8433381" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib15" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maddison</div> <div class="given-names">DR</div></div>,&nbsp; <div class="name"><div class="surname">Swofford</div> <div class="given-names">DL</div></div>,&nbsp; <div class="name"><div class="surname">Maddison</div> <div class="given-names">WP</div></div>.&nbsp;</div><div class="article-title">NEXUS: an extensible file format for systematic information</div>,&nbsp;<div class="source">Syst Biol</div>&nbsp;,&nbsp;<div class="year">1997</div>,&nbsp;vol.&nbsp;<div class="volume">46</div>&nbsp;(pg.&nbsp;<div class="fpage">590</div>-<div class="lpage">621</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NEXUS%3A%20an%20extensible%20file%20format%20for%20systematic%20information&amp;author=DR%20Maddison&amp;author=DL%20Swofford&amp;author=WP%20Maddison&amp;publication_year=1997&amp;journal=Syst%20Biol&amp;volume=46&amp;pages=590-621" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/46.4.590" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F46.4.590" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F46.4.590"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11975335" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib16" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maizel</div> <div class="given-names">JV</div>Jr</div>,&nbsp; <div class="name"><div class="surname">Lenk</div> <div class="given-names">RP</div></div>.&nbsp;</div><div class="article-title">Enhanced graphic matrix analysis of nucleic acid and protein sequences</div>,&nbsp;<div class="source">Proc Natl Acad Sci USA</div>&nbsp;,&nbsp;<div class="year">1981</div>,&nbsp;vol.&nbsp;<div class="volume">78</div>&nbsp;(pg.&nbsp;<div class="fpage">7665</div>-<div class="lpage">7669</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhanced%20graphic%20matrix%20analysis%20of%20nucleic%20acid%20and%20protein%20sequences&amp;author=JV%20Maizel&amp;author=RP%20Lenk&amp;publication_year=1981&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=78&amp;pages=7665-7669" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.78.12.7665" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.78.12.7665" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.78.12.7665"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6801656" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib17" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>.&nbsp;</div>,&nbsp;<div class="source">Molecular evolutionary genetics</div>&nbsp;,&nbsp;<div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Columbia University Press</div></div></div></div></div><div content-id="bib18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib18" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Notredame</div> <div class="given-names">C</div></div>,&nbsp; <div class="name"><div class="surname">Higgins</div> <div class="given-names">DG</div></div>,&nbsp; <div class="name"><div class="surname">Heringa</div> <div class="given-names">J</div></div>.&nbsp;</div><div class="article-title">T-Coffee: a novel method for multiple sequence alignments</div>,&nbsp;<div class="source">J Mol Biol</div>&nbsp;,&nbsp;<div class="year">2000</div>,&nbsp;vol.&nbsp;<div class="volume">302</div>&nbsp;(pg.&nbsp;<div class="fpage">205</div>-<div class="lpage">217</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=T-Coffee%3A%20a%20novel%20method%20for%20multiple%20sequence%20alignments&amp;author=C%20Notredame&amp;author=DG%20Higgins&amp;author=J%20Heringa&amp;publication_year=2000&amp;journal=J%20Mol%20Biol&amp;volume=302&amp;pages=205-217" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.4042" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.4042" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1006%2Fjmbi.2000.4042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10964570" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib19" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">WR</div></div>,&nbsp; <div class="name"><div class="surname">Lipman</div> <div class="given-names">DJ</div></div>.&nbsp;</div><div class="article-title">Improved tools for biological sequence comparison</div>,&nbsp;<div class="source">Proc Natl Acad Sci USA</div>&nbsp;,&nbsp;<div class="year">1988</div>,&nbsp;vol.&nbsp;<div class="volume">85</div>&nbsp;(pg.&nbsp;<div class="fpage">2444</div>-<div class="lpage">2448</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20tools%20for%20biological%20sequence%20comparison&amp;author=WR%20Pearson&amp;author=DJ%20Lipman&amp;publication_year=1988&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=85&amp;pages=2444-2448" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2444" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2444" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1073%2Fpnas.85.8.2444"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3162770" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib20" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rzhetsky</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Theoretical foundation of the minimum-evolution method of phylogenetic inference</div>,&nbsp;<div class="source">Mol Biol Evol</div>&nbsp;,&nbsp;<div class="year">1993</div>,&nbsp;vol.&nbsp;<div class="volume">10</div>&nbsp;(pg.&nbsp;<div class="fpage">1073</div>-<div class="lpage">1095</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Theoretical%20foundation%20of%20the%20minimum-evolution%20method%20of%20phylogenetic%20inference&amp;author=A%20Rzhetsky&amp;author=M%20Nei&amp;publication_year=1993&amp;journal=Mol%20Biol%20Evol&amp;volume=10&amp;pages=1073-1095" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8412650" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib21" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rzhetsky</div> <div class="given-names">A</div></div>,&nbsp; <div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">Tests of applicability of several substitution models for DNA sequence data</div>,&nbsp;<div class="source">Mol Biol Evol</div>&nbsp;,&nbsp;<div class="year">1995</div>,&nbsp;vol.&nbsp;<div class="volume">12</div>&nbsp;(pg.&nbsp;<div class="fpage">131</div>-<div class="lpage">151</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tests%20of%20applicability%20of%20several%20substitution%20models%20for%20DNA%20sequence%20data&amp;author=A%20Rzhetsky&amp;author=M%20Nei&amp;publication_year=1995&amp;journal=Mol%20Biol%20Evol&amp;volume=12&amp;pages=131-151" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a040182" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a040182" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Foxfordjournals.molbev.a040182"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7877488" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib22" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N</div></div>,&nbsp; <div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>.&nbsp;</div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>,&nbsp;<div class="source">Mol Biol Evol</div>&nbsp;,&nbsp;<div class="year">1987</div>,&nbsp;vol.&nbsp;<div class="volume">4</div>&nbsp;(pg.&nbsp;<div class="fpage">406</div>-<div class="lpage">425</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N%20Saitou&amp;author=M%20Nei&amp;publication_year=1987&amp;journal=Mol%20Biol%20Evol&amp;volume=4&amp;pages=406-425" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib23" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Studier</div> <div class="given-names">JA</div></div>,&nbsp; <div class="name"><div class="surname">Keppler</div> <div class="given-names">KJ</div></div>.&nbsp;</div><div class="article-title">A note on the neighbor-joining algorithm of Saitou and Nei</div>,&nbsp;<div class="source">Mol Biol Evol</div>&nbsp;,&nbsp;<div class="year">1988</div>,&nbsp;vol.&nbsp;<div class="volume">5</div>&nbsp;(pg.&nbsp;<div class="fpage">729</div>-<div class="lpage">731</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20note%20on%20the%20neighbor-joining%20algorithm%20of%20Saitou%20and%20Nei&amp;author=JA%20Studier&amp;author=KJ%20Keppler&amp;publication_year=1988&amp;journal=Mol%20Biol%20Evol&amp;volume=5&amp;pages=729-731" target="">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3221794" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div><div content-id="bib24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="bib24" class="ref-content "><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K</div></div>,&nbsp; <div class="name"><div class="surname">Dudley</div> <div class="given-names">J</div></div>,&nbsp; <div class="name"><div class="surname">Nei</div> <div class="given-names">M</div></div>,&nbsp; <div class="name"><div class="surname">Kumar</div> <div class="given-names">S</div></div>.&nbsp;</div><div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) Software Version 4.0</div>,&nbsp;<div class="source">Mol Biol Evol</div>&nbsp;,&nbsp;<div class="year">2007</div>,&nbsp;vol.&nbsp;<div class="volume">24</div>&nbsp;(pg.&nbsp;<div class="fpage">1596</div>-<div class="lpage">1599</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20%28MEGA%29%20Software%20Version%204.0&amp;author=K%20Tamura&amp;author=J%20Dudley&amp;author=M%20Nei&amp;author=S%20Kumar&amp;publication_year=2007&amp;journal=Mol%20Biol%20Evol&amp;volume=24&amp;pages=1596-1599" target="">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="">PubMed</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix" style="display: none;"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published 
by Oxford University Press on behalf of the Society for Molecular 
Biology and Evolution. All rights reserved. For permissions, please 
e-mail: journals.permissions@oxfordjournals.org</div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget --> 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/mbe/search-results?f_TocHeadingTitle=Letters">Letters</a>
        </div>

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=970247&amp;xsltPath=%7E/UI/app/XSLT&amp;siteId=5325" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div>
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">8,704</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000273704400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">2,653</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/molbev/msp259" data-uuid="7d743eb7-9906-06e3-2cc0-8397cc525788"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=105652" style="display:inline-block;">
        <img alt="Article has an altmetric score of 18" src="https://badges.altmetric.com/?size=128&amp;score=18&amp;types=aaaaabbt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="970247"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000273704400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fmbe%2farticle%2f27%2f2%2f221%2f970247">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000273704400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (2653)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F27%2F2%2F221%2F970247" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/mbe/CrossRef-CitedBy/970247">CrossRef</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz102/5481007?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Shared signature dynamics tempered by
 local fluctuations enables fold adaptability and specificity
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz104/5481006?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Aneuploidy enables cross-adaptation to unrelated drugs
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz098/5480303?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Transposable elements are important 
contributors to standing variation in gene expression in  Capsella 
grandiflora
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz090/5480301?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Additive traits lead to feeding 
advantage and reproductive isolation, promoting homoploid hybrid 
speciation
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
                       <div class="widget-dynamic-entry">
                        



                        
                           <a href="https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz100/5480300?searchresult=1">
                                                              <div class="widget-dynamic-journal-title">
                                   Social parasitism in the honeybee ( Apis mellifera ) is not controlled by a single SNP
                               </div>
                                                                                             <div class="widget-dynamic-journal-image-synopsis">
                                       <div class="widget-dynamic-journal-synopsis">                                           
                                            
                                       </div> <!-- /.widget-dynamic-journal-synopsis -->
                               </div>


                           </a>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

        </div> 
    </div>
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="SeaView Version 4: A Multiplatform Graphical User Interface for Sequence Alignment and Phylogenetic Tree Building" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    


        
        <script type="text/javascript" src="MathJax.js"></script>
        
        <script type="text/javascript" src="footer.js"></script>

        
    
    <script type="text/javascript" src="addthis_widget.js"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh311" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.6072316804029534&amp;iit=1556465606034&amp;tmr=load%3D1556465605992%26core%3D1556465606026%26main%3D1556465606030%26ifr%3D1556465606038&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F27%2F2%2F221%2F970247&amp;href=https%3A%2F%2Facademic.oup.com%2Fmbe%2Farticle%2F27%2F2%2F221%2F970247&amp;dt=SeaView%20Version%204%3A%20A%20Multiplatform%20Graphical%20User%20Interface%20for%20Sequence%20Alignment%20and%20Phylogenetic%20Tree%20Building&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5cc5c7c6bd1d1c0d&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DSeaView%2520Version%25204%253A%2520A%2520Multiplatform%2520Graphical%2520User%2520Interface%2520for%2520Sequence%2520Alignment%2520and%2520Phylogenetic%2520Tree%2520Building%26description%3DAbstract.%2520%2520We%2520present%2520SeaView%2520version%25204%252C%2520a%2520multiplatform%2520program%2520designed%2520to%2520facilitate%2520multiple%2520alignment%2520and%2520phylogenetic%2520tree%2520building%2520from%2520molecular%2520sequenc%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fmbe%252Farticle%252F27%252F2%252F221%252F970247%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fmbe%252FIssue%252F27%252F2%252F0%252Fm_mbe27_2.cover.gif%253FExpires%253D1619537602%2526Signature%253DsOBBMs5zxB3xYz8-zPpxJyV-Hlr1OE9epwdXn3vd9B8vXCLvJiURzoAgV%7EApkdSSnj-xnuB4MLVOoMlZv3lg%7EKgB4ilRiQdl2nOvPsRWJh970w6cLJztMDSyL0IoE52ZtJeXQIHx68NG5BAn1i50TDX6rgdFKBh0HVFVs9hHf3VzWroNJuiUs8Mf-wiSWPt5XxdwXgReVzS48yeA9iSgc4TilKuc378ZhgrbFvV8ainNe7j%7EUVhnDmmVFUw4GJpgPEJSRSTWg%7EQkdzAuUVs4heAyVODQAV9kP3FuUCRXNR9hG%7EMhrHzkUSeadSYUqYx6MzD9O7BG5gdb8FIxJhXY9A__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2009-10-23%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.4.3-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
    <script type="text/javascript" src="api.js"></script>
    <script type="text/javascript" src="chartist.js"></script>
    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/molbev/msp259");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/mbe_Behind_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/mbe/pages/About">About Molecular Biology and Evolution</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/mbe/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/mbe/pages/Policies">Policies</a>
                            </li>
                            <li class="link-4">
                                <a href="https://academic.oup.com/mbe/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-5">
                                <a href="http://www.smbe.org/smbe/">Contact Us</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
                            <li class="link-2">
                                <a href="https://twitter.com/OfficialSMBE">Twitter</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/mbe/subscribe">Purchase</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                            </li>
                            <li class="link-5">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/mbe" target="">
                <img id="footer-logo-MolecularBiologyandEvolution" class="journal-footer-affiliations-logo" src="mbe_f1-2062292105.svg" alt="Molecular Biology and Evolution">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1537-1719</li>
                                            <li>Print ISSN 0737-4038</li>
                    <li>Copyright © 2019 Society for Molecular Biology and Evolution</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2019 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
     
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5325&amp;returnUrl=%2fmbe%2farticle%2f27%2f2%2f221%2f970247">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ"/>
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="msp259.js"></script></body></html>