<!DOCTYPE html>
<html lang="en"><head><script type="text/javascript" async="" src="recaptcha__en.js"></script><script type="text/javascript" src="300lo.json"></script><script type="text/javascript" src="_ate.config_resp"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    <!-- Meta -->
    <meta charset="utf-8"><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=Edge">
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="https://academic.oup.com/UI/app/img/apple-touch-icon.png"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-32x32.png" sizes="32x32"><link rel="icon" type="image/png" href="https://academic.oup.com/UI/app/img/favicon-16x16.png" sizes="16x16"><link rel="manifest" href="https://academic.oup.com/UI/app/img/manifest.json"><link rel="mask-icon" href="https://academic.oup.com/UI/app/img/safari-pinned-tab.svg" color="#5bbad5"><link rel="shortcut icon" href="https://academic.oup.com/UI/app/img/favicon.ico"><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml"><meta name="theme-color" content="#002f65">

    <!-- Stylesheets -->
    
<link href="css.css" type="text/css" rel="stylesheet">
<link href="globalCSS.css" type="text/css" rel="stylesheet">
<link href="cookiePolicyCSS.css" type="text/css" rel="stylesheet">
<link href="chartist.css" type="text/css" rel="stylesheet">

        <link href="global.css" rel="stylesheet" type="text/css">
    
        <link href="Site-1131979791.css" rel="stylesheet" type="text/css">
    

    <!-- Scripts -->
    
<script async="" src="analytics.js"></script><script src="jquery-2.js" type="text/javascript"></script>

<!--     

 -->

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [0] 
        };

        App.CurrentSubdomain = 'nar';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/nar/article/30/14/3059/2904316","@type":"ScholarlyArticle","name":"MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform","about":["fourier analysis","amino acids","amino acid sequence","benchmarking","computer simulation","sequence alignment","polarity","heuristics","chest pain units","Synthetic Biology and Assembly Cloning","Mutagenesis","New Restriction Enzymes","Nucleic acid amplification","Nucleic acid modification","Nucleic acid structure","Polymorphism/mutation detection","Protein-nucleic acid interaction","Protein-protein interaction","Recombinant DNA expression","Recombination","Cell biology","Repair","Replication","RNA characterisation and manipulation","Computational Methods","Microarray Technology","Targeted inhibition of gene function","Ribosomes and Protein Translation","Massively Parallel (Deep) Sequencing","Targeted gene modification","Chromatin and Epigenetics","Physical and Biochemical Characterisation of DNA","DNA-Mediated Cell Transformation and Nucleic Acids Transfer","Nucleic Acid Enzymology","Genomics","Miscellaneous/other","Transcriptome Mapping - Monitoring Gene Expression"],"datePublished":"2002-07-15","isPartOf":{"@id":"https://academic.oup.com/nar/issue/30/14","@type":"PublicationIssue","issueNumber":"14","datePublished":"2002-07-15","isPartOf":{"@id":"https://academic.oup.com/nar","@type":"Periodical","name":"Nucleic Acids Research","issn":["1362-4962"]}},"url":"http://dx.doi.org/10.1093/nar/gkf436","keywords":["fourier analysis","amino acids","amino acid sequence","benchmarking","computer simulation","sequence alignment","polarity","heuristics","chest pain units"],"inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/nar/article/30/14/3059/2904316","author":[{"name":"Katoh, Kazutaka","@type":"Person"},{"name":"Misawa, Kazuharu","@type":"Person"},{"name":"Kuma, Kei‐ichi","@type":"Person"},{"name":"Miyata, Takashi","@type":"Person"}],"description":"Abstract.  A multiple sequence alignment program, MAFFT, has been developed. The CPU time is drastically reduced as compared with existing methods. MAFFT includ","pageStart":"3059","pageEnd":"3066","thumbnailURL":"https://academic.oup.com/UI/app/img/generic-cover.png","headline":"MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform","image":"https://academic.oup.com/UI/app/img/generic-cover.png"}
    </script>
    <meta property="og:type" content="article">
    <meta property="og:title" content="MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform | Nucleic Acids Research | Oxford Academic">
    <meta property="og:description" content="Abstract.  A multiple sequence alignment program, MAFFT, has been developed. The CPU time is drastically reduced as compared with existing methods. MAFFT includ">
    <meta property="og:url" content="https://academic.oup.com/nar/article/30/14/3059/2904316">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/Issue/46/18/2/m_cover.gif?Expires=2147483647&amp;Signature=eV~hm087cge5g8xBrr8Ad3gSMmcSfY8S8fYMMxVMyebMOzGEOd-Q8KiPx0XalwBTcA6jZy29jYCCVng~gYiA2bRURSHYiJJmFdS3MB~zxLd3wDRhaCxH2ogC-EC~NIicXdJr6ACFFc~6Slb3Z4F3GPnTUmF1TfmETtWJ9ZljeOrPJCq00DIOHPwcSP5C6wnJSfquTOINmA8u8gutkWbmGOwPFaTa37wgWdMQ3sPlXeaTD7x3QVAvxU1cvpttJcvurNH6ZD1KkAoWB3fbxyqXHIEHlDf-ZLeY2k321uAAnOvcvm5UKznRhGSheuGJ5tbEO3eOOn2s4QmfHPn3G8VWYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2002-07-15">
    <meta property="og:site_name" content="OUP Academic">
<meta name="citation_author" content="Katoh, Kazutaka"><meta name="citation_author" content="Misawa, Kazuharu"><meta name="citation_author" content="Kuma, Kei‐ichi"><meta name="citation_author" content="Miyata, Takashi"><meta name="citation_title" content="MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform"><meta name="citation_firstpage" content="3059"><meta name="citation_lastpage" content="3066"><meta name="citation_doi" content="10.1093/nar/gkf436"><meta name="citation_keyword" content="sequence alignment"><meta name="citation_journal_title" content="Nucleic Acids Research"><meta name="citation_journal_abbrev" content="Nucleic Acids Res"><meta name="citation_volume" content="30"><meta name="citation_issue" content="14"><meta name="citation_publication_date" content="2002/07/15"><meta name="citation_issn" content="0305-1048"><meta name="citation_publisher" content="Oxford University Press"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1970;  citation_volume=48;  citation_pages=443"><meta name="citation_reference" content=" citation_year=1983;  citation_pages=253"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1987;  citation_volume=25;  citation_pages=351"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1987;  citation_volume=198;  citation_pages=327"><meta name="citation_reference" content="Comput. Appl. Biosci.;  citation_year=1991;  citation_volume=7;  citation_pages=479"><meta name="citation_reference" content="Comput. Appl. Biosci.;  citation_year=1993;  citation_volume=9;  citation_pages=361"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=2682"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=2000;  citation_volume=302;  citation_pages=205"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=2369"><meta name="citation_reference" content="Proc. Natl Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2444"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=1982;  citation_volume=10;  citation_pages=133"><meta name="citation_reference" content="J. Comput. Biol.;  citation_year=2002;  citation_volume=9;  citation_pages=23"><meta name="citation_reference" content="Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=87"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1979;  citation_volume=12;  citation_pages=219"><meta name="citation_reference" content=" citation_year=1983; "><meta name="citation_reference" content="Science;  citation_year=1974;  citation_volume=185;  citation_pages=862"><meta name="citation_reference" content=" citation_year=1995; "><meta name="citation_reference" content="Comput. Appl. Biosci.;  citation_year=1995;  citation_volume=11;  citation_pages=543"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1995;  citation_volume=249;  citation_pages=816"><meta name="citation_reference" content="Comput. Appl. Biosci.;  citation_year=1992;  citation_volume=8;  citation_pages=275"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1980;  citation_volume=16;  citation_pages=111"><meta name="citation_reference" content=" citation_year=1978;  citation_pages=345"><meta name="citation_reference" content="University of Kansas Scientific Bulletin;  citation_year=1958;  citation_volume=28;  citation_pages=1409"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=S38"><meta name="citation_reference" content="Comput. Appl. Biosci.;  citation_year=1995;  citation_volume=11;  citation_pages=13"><meta name="citation_reference" content="Proc. Int. Conf. Intell. Syst. Mol. Biol.;  citation_year=1997;  citation_volume=5;  citation_pages=303"><meta name="citation_reference" content="Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=12098"><meta name="citation_reference" content="Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=211"><meta name="citation_reference" content="Protein Eng.;  citation_year=1992;  citation_volume=5;  citation_pages=35"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1996;  citation_volume=264;  citation_pages=823"><meta name="citation_reference" content="Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=173"><meta name="citation_reference" content="J. Mol. Evol.;  citation_year=1991;  citation_volume=32;  citation_pages=70"><meta name="citation_reference" content="Mol. Biol. Evol.;  citation_year=1994;  citation_volume=11;  citation_pages=571"><meta name="citation_reference" content="J. Mol. Biol.;  citation_year=1981;  citation_volume=147;  citation_pages=195"><meta name="citation_reference" content="Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=713"><meta name="citation_reference" content="Gene;  citation_year=2001;  citation_volume=270;  citation_pages=17"><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/nar/article-pdf/30/14/3059/9488148/gkf436.pdf"><meta name="description" content="Abstract.  A multiple sequence alignment program, MAFFT, has been developed. The CPU time is drastically reduced as compared with existing methods. MAFFT includ">
        <meta name="citation_xml_url" content="https://academic.oup.com/nar/article-xml/30/14/3059/2904316">



    <link rel="canonical" href="https://academic.oup.com/nar/article/30/14/3059/2904316">


        <!-- Start: GPT Async -->

<script async="async" src="gpt.js"></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type="text/javascript">
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'nar_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "nar/article/30/14/3059/2904316");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=nar&authzrequired=false&doi=10.1093/nar/gkf436';
    </script>
<title>
	MAFFT: a novel method for rapid multiple sequence alignment based on 
fast Fourier transform | Nucleic Acids Research | Oxford Academic
</title><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_265.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-6b74dfff5c7f701b334f3e44e411f88f.js"></script><style></style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_google_plusone,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.at-share-tbx-element .google_plusone_iframe_widget>span>div{vertical-align:top!important}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_google_plusone.wsb-social-share-button{padding-top:0}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-2c47105b6381604898bbf8ae8a680350.css"></head>

<body data-sitename="nucleicacidsresearch" class="off-canvas pg_article" theme-nar="">
    <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');ga('create', 'UA-78288099-3', {'sampleRate': 100});ga('send', 'pageview');</script>
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2904316" id="webform">
<div class="aspNetHidden">
<input name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZguPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRvaCwgS2F6dXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXNhd2EsIEthenVoYXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VtYSwgS2Vp4oCQaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1peWF0YSwgVGFrYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTUFGRlQ6IGEgbm92ZWwgbWV0aG9kIGZvciByYXBpZCBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYmFzZWQgb24gZmFzdCBGb3VyaWVyIHRyYW5zZm9ybSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMwNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjMwNjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL25hci9na2Y0MzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fa2V5d29yZCIgY29udGVudD0ic2VxdWVuY2UgYWxpZ25tZW50IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDIvMDcvMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMDMwNS0xMDQ4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NDQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3BhZ2VzPTI1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE5ODsgIGNpdGF0aW9uX3BhZ2VzPTMyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkNvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjY4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzAyOyAgY2l0YXRpb25fcGFnZXM9MjA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjM2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNDQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9Ik51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTEzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz04NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5Nzk7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25feWVhcj0xOTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTc0OyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPTg2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBjaXRhdGlvbl95ZWFyPTE5OTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkNvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTU0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjQ5OyAgY2l0YXRpb25fcGFnZXM9ODE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iIGNpdGF0aW9uX3llYXI9MTk3ODsgIGNpdGF0aW9uX3BhZ2VzPTM0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgS2Fuc2FzIFNjaWVudGlmaWMgQnVsbGV0aW47ICBjaXRhdGlvbl95ZWFyPTE5NTg7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNDA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPVMzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkNvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iUHJvYy4gSW50LiBDb25mLiBJbnRlbGwuIFN5c3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTMwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0xMjA5OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iUHJvdGVpbiBFbmcuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjY0OyAgY2l0YXRpb25fcGFnZXM9ODIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTcwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTU3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9MTk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz03MTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL25hci9hcnRpY2xlLXBkZi8zMC8xNC8zMDU5Lzk0ODgxNDgvZ2tmNDM2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBwcm9ncmFtLCBNQUZGVCwgaGFzIGJlZW4gZGV2ZWxvcGVkLiBUaGUgQ1BVIHRpbWUgaXMgZHJhc3RpY2FsbHkgcmVkdWNlZCBhcyBjb21wYXJlZCB3aXRoIGV4aXN0aW5nIG1ldGhvZHMuIE1BRkZUIGluY2x1ZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjkwNDMxNiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTRfFZhUQdeyBdSL4/SJCtiqrk/KQ==" type="hidden">
</div>

<div class="aspNetHidden">

	<input name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" type="hidden">
</div>
        </form>
        
        <input id="hdnSiteID" value="5127" type="hidden">
        <input id="hdnAdDelaySeconds" value="3000" type="hidden">
        <input id="hdnAdConfigurationTop" value="default" type="hidden">
        <input id="hdnAdConfigurationRightRail" value="default" type="hidden">

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad1_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad1_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad1_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="https://academic.oup.com/"><img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f30%2f14%2f3059%2f2904316" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Nucleic Acids Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f30%2f14%2f3059%2f2904316" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115593">
            <a href="https://academic.oup.com/nar/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115590">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Section browse</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115594">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDChemical+Biology+and+Nucleic+Acid+Chemistry" class="nav-link ">Chemical Biology and Nucleic Acid Chemistry</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115595">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=COMPUTATIONAL+BIOLOGY" class="nav-link ">Computational Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115596">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Data+Resources+and+Analyses" class="nav-link ">Data Resources and Analyses</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115597">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Gene+Regulation,+Chromatin+and+Epigenetics" class="nav-link ">Gene Regulation, Chromatin and Epigenetics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115598">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genome+integrity,+repair+and+replication" class="nav-link ">Genome Integrity, Repair and Replication</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115599">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genomics" class="nav-link ">Genomics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115600">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=MOLECULAR+BIOLOGY" class="nav-link ">Molecular Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115601">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Nucleic+Acid+Enzymes" class="nav-link ">Nucleic Acid Enzymes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115602">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=RNAANDRNA+and+RNA-protein+complexes" class="nav-link ">RNA and RNA-protein complexes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115603">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=STRUCTURAL+BIOLOGY" class="nav-link ">Structural Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115604">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDSynthetic+Biology+and+Bioengineering" class="nav-link ">Synthetic Biology and Bioengineering</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115605">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Methods+Online" class="nav-link ">Methods Online</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115606">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=SURVEY%20@@@%20SUMMARY" class="nav-link ">Surveys and Summaries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115607">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Database+Issue" class="nav-link ">Database</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115608">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Web+Server+issue" class="nav-link ">Web Server</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115609">
            <a href="https://academic.oup.com/nar/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115591">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115610">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115611">
            <a href="http://mc.manuscriptcentral.com/nar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115612">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115613">
            <a href="https://academic.oup.com/journals/pages/open_access" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115614">
            <a href="https://academic.oup.com/nar/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115592">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115615">
            <a href="https://academic.oup.com/nar/pages/About" class="nav-link ">About Nucleic Acids Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115616">
            <a href="https://academic.oup.com/nar/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115617">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115618">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115619">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115620">
            <a href="https://academic.oup.com/nar/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115621">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115622">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="https://academic.oup.com/nar" class=" journal-logo-container">

            <img id="logo-NucleicAcidsResearch" class="journal-logo" src="nar_title248941275.svg" alt="Nucleic Acids Research">

                </a>
            <div class="society-logo-block">
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115593">
            <a href="https://academic.oup.com/nar/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115590">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Section browse</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115594">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDChemical+Biology+and+Nucleic+Acid+Chemistry" class="nav-link ">Chemical Biology and Nucleic Acid Chemistry</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115595">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=COMPUTATIONAL+BIOLOGY" class="nav-link ">Computational Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115596">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Data+Resources+and+Analyses" class="nav-link ">Data Resources and Analyses</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115597">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Gene+Regulation,+Chromatin+and+Epigenetics" class="nav-link ">Gene Regulation, Chromatin and Epigenetics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115598">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genome+integrity,+repair+and+replication" class="nav-link ">Genome Integrity, Repair and Replication</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115599">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Genomics" class="nav-link ">Genomics</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115600">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=MOLECULAR+BIOLOGY" class="nav-link ">Molecular Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115601">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Nucleic+Acid+Enzymes" class="nav-link ">Nucleic Acid Enzymes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115602">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=RNAANDRNA+and+RNA-protein+complexes" class="nav-link ">RNA and RNA-protein complexes</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115603">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=STRUCTURAL+BIOLOGY" class="nav-link ">Structural Biology</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115604">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Synthetic+Biology+and+ChemistryANDSynthetic+Biology+and+Bioengineering" class="nav-link ">Synthetic Biology and Bioengineering</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115605">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Methods+Online" class="nav-link ">Methods Online</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115606">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=SURVEY%20@@@%20SUMMARY" class="nav-link ">Surveys and Summaries</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115607">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Database+Issue" class="nav-link ">Database</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115608">
            <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Web+Server+issue" class="nav-link ">Web Server</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115609">
            <a href="https://academic.oup.com/nar/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115591">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115610">
            <a href="https://academic.oup.com/nar/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115611">
            <a href="http://mc.manuscriptcentral.com/nar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115612">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115613">
            <a href="https://academic.oup.com/journals/pages/open_access" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115614">
            <a href="https://academic.oup.com/nar/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1115592">
            <a href="https://academic.oup.com/nar/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115615">
            <a href="https://academic.oup.com/nar/pages/About" class="nav-link ">About Nucleic Acids Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115616">
            <a href="https://academic.oup.com/nar/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115617">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115618">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115619">
            <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115620">
            <a href="https://academic.oup.com/nar/pages/Policies" class="nav-link ">Policies</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115621">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1115622">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Nucleic Acids Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/nar/advanced-search?page=1&amp;fl_SiteID=5127&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5127" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=95144" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/nar" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="nar" type="hidden">
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left mobile">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/nar/issue/30/14"><img id="issueFallbackImage" class="fb-featured-image" src="generic-cover.png" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/nar/issue/30/14">
                        <div class="volume trailing-comma">Volume 30</div>
                        <div class="issue">Issue 14</div>

            </a>

        </div>
        <div class="ii-pub-date">
15 July 2002        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="55547717">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547717">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="55547720">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547720">INTRODUCTION</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="55547725">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547725">METHODS</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="55547786">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547786">RESULTS</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="55547803">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547803">DISCUSSION</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="55547809">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547809">ACKNOWLEDGEMENTS</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="55547821">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#55547821">References</a>
                </div>
            </li>
    </ul>

    </div>

	                <div class="comments-jumplink">
        <a class="js-jumplink scrollTo" href="#usercomments"><i class="icon-menu_comments"></i>Comments (0)</a>
    </div></div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/nar/article/30/14/3052/2904331">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/nar/article/30/14/3067/2904302">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazutaka Katoh</div>
    </div>


                <div class="info-card-note">
                    <a reveal-id="COR1" data-open="COR1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Tel: +81 75 753
 4220; Fax: +81 75 753 4223; Email: miyata@biophys.kyoto‐u.ac.jp
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Kazutaka+Katoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katoh%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Katoh%20K%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazutaka Katoh<i class="icon-general-mail"></i></a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuharu Misawa</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Kazuharu+Misawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Misawa%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Misawa%20K%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuharu Misawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kei‐ichi Kuma</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Kei%e2%80%90ichi+Kuma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuma%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kuma%20K%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kei‐ichi Kuma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Miyata</div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/nar/search-results?f_Authors=Takashi+Miyata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyata%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Miyata%20T%22">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Miyata</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>Nucleic Acids Research</em>, Volume 30, Issue 14, 15 July 2002, Pages 3059–3066, <a href="https://doi.org/10.1093/nar/gkf436">https://doi.org/10.1093/nar/gkf436</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 July 2002</div>
                        </div>
                    </div>
                            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views" style="display: none;">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up" style="display: block;"></div>
                                    <li class="article-content-filter" data-content-filter="article-content" style="display: none;"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    
                                    
                                    
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2904316" href="https://academic.oup.com/nar/article-pdf/30/14/3059/9488148/gkf436.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <div class="modal-title">Citation</div>
    <p>Kazutaka Katoh, Kazuharu Misawa, Kei‐ichi Kuma, Takashi Miyata;  
MAFFT: a novel method for rapid multiple sequence alignment based on 
fast Fourier transform, <em>Nucleic Acids Research</em>, Volume 30, Issue 14, 15 July 2002, Pages 3059–3066, <a href="https://doi.org/10.1093/nar/gkf436">https://doi.org/10.1093/nar/gkf436</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=ris">RIS (Zotero)</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=ris">EndNote</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=bibtex">BibTex</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=txt">Medlars</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=ris">ProCite</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=txt">RefWorks</a></li>
        <li><a href="https://academic.oup.com/nar/downloadcitation/2904316?format=ris">Reference Manager</a></li>
    </ul>
    <hr>
    
    <p>© 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1362-4962&amp;title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;publicationDate=2002-07-15&amp;volumeNum=30&amp;issueNum=14&amp;author=Katoh%2C%20Kazutaka%3B%20Misawa%2C%20Kazuharu&amp;startPage=3059&amp;endPage=3066&amp;contentId=10.1093%2Fnar%2Fgkf436&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <div class="atclear"></div></ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Nucleic Acids Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f30%2f14%2f3059%2f2904316" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  Nucleic Acids Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/nar/advanced-search?page=1&amp;fl_SiteID=5127&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" value="nar" type="hidden">
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination="55547717" id="55547717" class="abstract-title">Abstract</h2>
<section class="abstract"><p>A multiple sequence alignment program, 
MAFFT, has been developed. The CPU time is drastically reduced as 
compared with existing methods. MAFFT includes two novel techniques. (i)
 Homo logous regions are rapidly identified by the fast Fourier 
transform (FFT), in which an amino acid sequence is converted to a 
sequence composed of volume and polarity values of each amino acid 
residue. (ii) We propose a simplified scoring system that performs well 
for reducing CPU time and increasing the accuracy of alignments even for
 sequences having large insertions or extensions as well as distantly 
related sequences of similar length. Two different heuristics, the 
progressive method (FFT‐NS‐2) and the iterative refinement method 
(FFT‐NS‐i), are implemented in MAFFT. The performances of FFT‐NS‐2 and 
FFT‐NS‐i were compared with other methods by computer simulations and 
benchmark tests; the CPU time of FFT‐NS‐2 is drastically reduced as 
compared with CLUSTALW with comparable accuracy. FFT‐NS‐i is over 100 
times faster than T‐COFFEE, when the number of input sequences exceeds 
60, without sacrificing the accuracy.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<p>Received April 8, 2002; Revised and Accepted May 24, 2002</p>                            <h2 scrollto-destination="55547720" id="55547720" class="section-title">INTRODUCTION</h2>
<p>Multiple sequence alignment is a basic tool in various aspects of 
molecular biological analyses ranging from detecting key functional 
residues to inferring the evolutionary history of a protein family. It 
is, however, difficult to align distantly related sequences correctly 
without manual inspections by expert knowledge. Many efforts have been 
made on the problems concerning the optimization of sequence alignment. 
Needleman and Wunsch (<a reveal-id="GKF436C1" data-open="GKF436C1" class="link link-ref link-reveal xref-default">1</a>)
 presented an algorithm for sequence comparison based on dynamic 
programming (DP), by which the optimal alignment between two sequences 
is obtained. The generalization of this algorithm to multiple sequence 
alignment (<a reveal-id="GKF436C2" data-open="GKF436C2" class="link link-ref link-reveal xref-default">2</a>)
 is not applicable to a practical alignment that consists of dozens or 
hundreds of sequences, since it requires huge CPU time proportional to <em>N</em><sup><em>K</em></sup>, where <em>K</em> is the number of sequences each with length <em>N</em>. To overcome this difficulty, various heuristic methods, including progressive methods (<a reveal-id="GKF436C3" data-open="GKF436C3" class="link link-ref link-reveal xref-default">3</a>) and iterative refinement methods (<a reveal-id="GKF436C4" data-open="GKF436C4" class="link link-ref link-reveal xref-default">4</a>–<a reveal-id="GKF436C6" data-open="GKF436C6" class="link link-ref link-reveal xref-default">6</a>),
 have been proposed to date. They are mostly based on various 
combinations of successive two‐dimensional DP, which takes CPU time 
proportional to <em>N</em><sup>2</sup>.</p><p>Even if these heuristic 
methods successfully provide the optimal alignments, there remains the 
problem of whether the optimal alignment really corresponds to the 
biologically correct one. The accuracy of resulting alignments is 
greatly affected by the scoring system. Thompson <em>et al.</em> (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>)
 developed a complicated scoring system in their program CLUSTALW, in 
which gap penalties and other parameters are carefully adjusted 
according to the features of input sequences, such as sequence 
divergence, length, local hydropathy and so on. Nevertheless, no 
existing scoring system is able to process correctly global alignments 
for various types of problems including large terminal extension of 
internal insertion (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>). Considerable improvements in the accuracy have recently been made in CLUSTALW (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>) version 1.8, the most popular alignment program with excellent portability and operativity, and T‐COFFEE (<a reveal-id="GKF436C9" data-open="GKF436C9" class="link link-ref link-reveal xref-default">9</a>), which provides alignments of the highest accuracy among known methods to date.</p><p>On
 the other hand, few improvements have been made successfully to reduce 
the CPU time, since the proposal of the progressive method by Feng and 
Doolittle (<a reveal-id="GKF436C3" data-open="GKF436C3" class="link link-ref link-reveal xref-default">3</a>).
 A high‐speed computer program applicable to large‐scale problems is 
becoming more important with the rapid increase in the number of protein
 and DNA sequences. In order to improve the speed of DP, it is effective
 to use highly homologous segments in the procedure of multiple sequence
 alignment (<a reveal-id="GKF436C10" data-open="GKF436C10" class="link link-ref link-reveal xref-default">10</a>). There are well‐known homology search programs, such as FASTA (<a reveal-id="GKF436C11" data-open="GKF436C11" class="link link-ref link-reveal xref-default">11</a>) and BLAST (<a reveal-id="GKF436C12" data-open="GKF436C12" class="link link-ref link-reveal xref-default">12</a>), based on string matching algorithms.</p><p>In
 this report, we developed a novel method for multiple sequence 
alignment based on the fast Fourier transform (FFT), which allows rapid 
detection of homologous segments. In spite of its great efficiency, FFT 
has rarely been used practically for detecting sequence similarities (<a reveal-id="GKF436C13" data-open="GKF436C13" class="link link-ref link-reveal xref-default">13</a>,<a reveal-id="GKF436C14" data-open="GKF436C14" class="link link-ref link-reveal xref-default">14</a>).
 We also propose an improved scoring system, which performs well even 
for sequences having large insertions or extensions as well as distantly
 related sequences of similar length. The efficiency (CPU time and 
accuracy) of the method was tested by computer simulations and the 
BAliBASE (<a reveal-id="GKF436C15" data-open="GKF436C15" class="link link-ref link-reveal xref-default">15</a>)
 benchmark tests in comparison with several existing methods. These 
tests showed that the CPU time has been drastically reduced, whereas the
 accuracy of the resulting alignments is comparable with that of the 
most accurate methods among existing ones.</p>                            <h2 scrollto-destination="55547725" id="55547725" class="section-title">METHODS</h2>
                            <h3 scrollto-destination="55547726" id="55547726" class="section-title">Group‐to‐group alignments by FFT</h3>
<p>The frequency of amino acid substitutions strongly depends on the 
difference of physico‐chemical properties, particularly volume and 
polarity, between the amino acid pair involved in the substitution (<a reveal-id="GKF436C16" data-open="GKF436C16" class="link link-ref link-reveal xref-default">16</a>).
 Substitutions between physico‐chemically similar amino acids tend to 
preserve the structure of proteins, and such neutral substitutions have 
been accumulated in molecules during evolution (<a reveal-id="GKF436C17" data-open="GKF436C17" class="link link-ref link-reveal xref-default">17</a>). It is therefore reasonable to consider that an amino acid <em>a</em> is assigned to a vector whose components are the volume value <em>v</em>(<em>a</em>) and the polarity value <em>p</em>(<em>a</em>) introduced by Grantham (<a reveal-id="GKF436C18" data-open="GKF436C18" class="link link-ref link-reveal xref-default">18</a>). We use the normalized forms of these values: <em>v̂</em>(<em>a</em>) = [<em>v</em>(<em>a</em>) – <em>v̄</em>]/σ<sub><em>v</em></sub> and <em>p̂</em>(<em>a</em>) = [ <em>p</em>(<em>a</em>) – <em>p̄</em>]/σ<sub><em>p</em></sub>, where an overbar denotes the average over 20 amino acids, and σ<sub><em>v</em></sub> and σ<sub><em>p</em></sub>
 denote the standard deviations of volume and polarity, respectively. An
 amino acid sequence is converted to a sequence of such vectors.</p><p></p><p><em>Calculation of the correlation between two amino acid sequences.</em> We define the correlation <em>c</em>(<em>k</em>) between two sequences of such vectors as </p><p><em>c</em>(<em>k</em>) = <em>c</em><sub><em>v</em></sub>(<em>k</em>) + <em>c</em><sub><em>p</em></sub>(<em>k</em>),<strong>1</strong></p><p>where <em>c</em><sub><em>v</em></sub>(<em>k</em>) and <em>c</em><sub><em>p</em></sub>(<em>k</em>)
 are, as defined below, the correlations of volume component and 
polarity component, respectively, between two amino acid sequences to be
 aligned. The correlation <em>c</em>(<em>k</em>) represents the degree of similarity of two sequences with the positional lag of <em>k</em> sites. The high value of <em>c</em>(<em>k</em>) indicates that the sequences may have homologous regions.</p><p>The correlation <em>c</em><sub><em>v</em></sub>(<em>k</em>) of volume component between sequence 1 and sequence 2 with the positional lag of <em>k</em> sites is defined as</p><p><img class="content-image" src="m_gkf436equ2.jpg" alt="graphic" path-from-xml="gkf436equ2.jpeg"></p><p>where <em>v̂</em><sub>1</sub>(<em>n</em>) and <em>v̂</em><sub>2</sub>(<em>n</em>) are the volume component of the <em>n</em>th site of sequence 1 with the length of <em>N</em> and that of sequence 2 with the length of <em>M</em>, respectively. Considering <em>N</em> ≃ <em>M</em> in many cases, equation <strong>2</strong> takes <em>O</em>(<em>N</em><sup>2</sup>) operations. The FFT reduces the CPU time of this calculation to <em>O</em>(<em>N</em> log <em>N</em>) (<a reveal-id="GKF436C19" data-open="GKF436C19" class="link link-ref link-reveal xref-default">19</a>). If <em>V</em><sub>1</sub>(<em>m</em>) and <em>V</em><sub>2</sub>(<em>m</em>) are the Fourier transform of <em>v̂</em><sub>1</sub>(<em>n</em>) and <em>v̂</em><sub>2</sub>(<em>n</em>), i.e.</p><p><em>v̂</em><sub>1</sub>(<em>n</em>) ⇔ <em>V</em><sub>1</sub>(<em>m</em>)<strong>3</strong></p><p><em>v̂</em><sub>2</sub>(<em>n</em>) ⇔ <em>V</em><sub>2</sub>(<em>m</em>),<strong>4</strong></p><p>it is known that the correlation <em>c</em><sub><em>v</em></sub>(<em>k</em>) is expressed as</p><p><em>c</em><sub><em>v</em></sub>(<em>k</em>) ⇔ <em>V</em><sub>1</sub><sup>*</sup>(<em>m</em>) · <em>V</em><sub>2</sub>(<em>m</em>),<strong>5</strong></p><p>where ⇔ represents transform pairs, and the asterisk denotes complex conjugation.</p><p>The correlation <em>c</em><sub><em>p</em></sub>(<em>k</em>) of polarity component between two sequences</p><p><img class="content-image" src="m_gkf436equ6.jpg" alt="graphic" path-from-xml="gkf436equ6.jpeg"></p><p>is calculated in the same manner.</p><p></p><p><em>Finding homologous segments.</em> If two sequences compared have homologous regions, the correlation <em>c</em>(<em>k</em>) has some peaks corresponding to these regions (Fig. <a reveal-id="GKF436F1" data-open="GKF436F1" class="link link-ref link-reveal xref-default">1</a>A). By the FFT analysis, however, we can know only the positional lag <em>k</em> of a homologous region in two sequences but not the position of the region. As shown in Figure <a reveal-id="GKF436F1" data-open="GKF436F1" class="link link-ref link-reveal xref-default">1</a>B,
 to determine the positions of the homologous region in each sequence, a
 sliding window analysis with the window size of 30 sites is carried 
out, in which the degree of local homologies is calculated for each of 
the highest 20 peaks in the correlation <em>c</em>(<em>k</em>). We 
identify a segment of 30 sites with score value exceeding a given 
threshold (0.7 per site in our program, see below for details of the 
scoring system) as a homologous segment. If two or more successive 
segments are identified as homologous segments, they are combined into 
one segment of larger length. If the length of the combined segment is 
longer than 150 sites, the segment is divided into several segments with
 150 sites each.</p><p></p><p><em>Dividing a homology matrix.</em> To 
obtain an alignment between two sequences, the homologous segments must 
be arranged consistently in both sequences. A matrix <em>S</em><sub><em>ij</em></sub>(1 ≤ <em>i</em>, <em>j</em> ≤ <em>n</em>, <em>n</em> is the number of homologous segments) is constructed in the following manner. If the <em>i</em>th homologous segment on sequence 1 corresponds to the <em>j</em>th homologous segment on sequence 2, <em>S</em><sub><em>ij</em></sub> has the score value of the homologous segment calculated above; otherwise <em>S</em><sub><em>ij</em></sub> is set to 0. By applying the standard DP procedure to matrix <em>S</em><sub><em>ij</em></sub>, we obtain the optimal path, which corresponds to the optimal arrangement of homologous segments. Figure <a reveal-id="GKF436F2" data-open="GKF436F2" class="link link-ref link-reveal xref-default">2</a>A
 shows an example in which five homologous segments exist. The order of 
segments in sequence 1 differs from that in sequence 2. The optimal path
 depends on <em>S</em><sub>23</sub> and <em>S</em><sub>32</sub>; if <em>S</em><sub>23</sub> &gt; <em>S</em><sub>32</sub>, the path with bold arrows is optimal.</p><p>Overall
 homology matrix is divided into some sub‐matrices at the boundary 
corresponding to the center of homologous segments as illustrated in 
Figure <a reveal-id="GKF436F2" data-open="GKF436F2" class="link link-ref link-reveal xref-default">2</a>B. As a result, the shaded area in Figure <a reveal-id="GKF436F2" data-open="GKF436F2" class="link link-ref link-reveal xref-default">2</a>B is excluded from the calculation. As many homologous segments are detected by FFT, the CPU time is reduced.</p><p></p><p><em>Extension to group‐to‐group alignments.</em> The procedure described above can be easily extended to group‐to‐group alignment by considering equations <strong>2</strong> and <strong>6</strong> as a special case with one sequence in each group. These equations are extended to group‐to‐group alignment by replacing <em>v̂</em><sub>1</sub>(<em>n</em>) with <em>v̂</em><sub>group1</sub>(<em>n</em>), which is the linear combination of the volume components of the members belonging to group 1:</p><p><img class="content-image" src="m_gkf436equ6a.jpg" alt="graphic" path-from-xml="gkf436equ6a.jpeg"></p><p>where <em>w</em><sub><em>i</em></sub> is the weighting factor for sequence <em>i</em>, which is calculated in the same manner as CLUSTALW (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>) for the progressive method, or in the same manner as Gotoh’s (<a reveal-id="GKF436C20" data-open="GKF436C20" class="link link-ref link-reveal xref-default">20</a>) weighting system for the iterative refinement method. Similarly, polarity component is calculated as:</p><p><img class="content-image" src="m_gkf436equ6b.jpg" alt="graphic" path-from-xml="gkf436equ6b.jpeg"></p><p>This
 method is applicable to nucleotide sequences by converting a sequence 
to a sequence of four‐dimensional vectors whose components are the 
frequencies of A, T, G and C at each column, instead of volume and 
polarity values. In this case, correlation between two nucleotide 
sequences is:</p><p><em>c</em>(<em>k</em>) = <em>c</em><sub>A</sub>(<em>k</em>) + <em>c</em><sub>T</sub>(<em>k</em>) + <em>c</em><sub>G</sub>(<em>k</em>) + <em>c</em><sub>C</sub>(<em>k</em>).</p>                            <h3 scrollto-destination="55547755" id="55547755" class="section-title">Scoring system</h3>
<p><em>Similarity matrix.</em> In order to increase the efficiency of 
alignment, the scoring system (similarity matrix and gap penalties) was 
also modified. Vogt <em>et al.</em> (<a reveal-id="GKF436C21" data-open="GKF436C21" class="link link-ref link-reveal xref-default">21</a>)
 suggested that the Needleman–Wunsch (NW) algorithm performs well with 
all‐positive matrices, in which all elements have positive values. 
CLUSTALW (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>) and other methods use such all‐positive matrices by default. Since Vogt <em>et al.</em> (<a reveal-id="GKF436C21" data-open="GKF436C21" class="link link-ref link-reveal xref-default">21</a>)
 examined only the cases in which members of each protein familiy are 
similar in length, it is not clear whether such all‐positive matrices 
are suitable to various alignment problems, particularly to those of 
different length. Accordingly, contrary to existing methods, we adopted a
 normalized similarity matrix <em>M̂</em><sub><em>ab </em></sub>(<em>a</em> and <em>b</em> are amino acids) that has both positive and negative values: </p><p><em>M̂</em><sub><em>ab</em></sub> = [(<em>M</em><sub><em>ab </em></sub>– <em>average</em>2)/(<em>average</em>1 – <em>average</em>2)] + <em>S</em><sup><em>a</em></sup>,<sup></sup><sup></sup><sup></sup><sup></sup><strong>7</strong></p><p>where <em>average</em>1 = Σ<sub><em>a</em></sub><em>f</em><sub><em>a</em></sub><em>M</em><sub><em>aa</em></sub>, <em>average</em>2 = Σ<em><sub>a,b</sub>f<sub>a</sub>f<sub>b</sub>M<sub>ab</sub></em>, <em>M</em><sub><em>ab </em></sub>is raw similarity matrix, <em>f</em><sub><em>a </em></sub>is the frequency of occurrence of amino acid <em>a</em>, and <em>S</em><sup><em>a</em></sup> is a parameter that functions as a gap extension penalty. Under this similarity matrix <em>M̂</em><sub><em>ab</em></sub>, the score per site between two random sequences is <em>S</em><sup><em>a</em></sup>, and the score per site between two identical sequences is 1.0 + <em>S</em><sup><em>a</em></sup>. If <em>S</em><sup><em>a</em></sup> is much smaller than unity, gaps are scored virtually equivalent to random amino acid sequences.</p><p>The default parameters of our program are: <em>M</em><sub><em>ab </em></sub>is the 200 PAM log‐odds matrix by Jones <em>et al.</em> (<a reveal-id="GKF436C22" data-open="GKF436C22" class="link link-ref link-reveal xref-default">22</a>), <em>f</em><sub><em>a</em></sub> is the frequency of occurrence for amino acid <em>a</em> calculated by Jones <em>et al.</em> (<a reveal-id="GKF436C22" data-open="GKF436C22" class="link link-ref link-reveal xref-default">22</a>), <em>S</em><sup><em>op</em></sup> (gap opening penalty, defined below) is 2.4 and <em>S</em><sup><em>a</em></sup> is 0.06, for amino acid sequences. For nucleic acid sequences, <em>M</em><sub><em>ab </em></sub>is the 200 PAM log‐odds matrix calculated from Kimura’s two parameter model (<a reveal-id="GKF436C23" data-open="GKF436C23" class="link link-ref link-reveal xref-default">23</a>) with transition/transversion ratio of 2.0, <em>f</em><sub><em>a</em></sub> is 0.25, <em>S</em><sup><em>op</em></sup> is 2.4 and <em>S</em><sup><em>a</em></sup> is 0.06.</p><p></p><p><em>Gap penalty.</em> Homology matrix <em>H</em>(<em>i</em>, <em>j</em>) between two amino acid sequences <em>A</em>(<em>i</em>) and <em>B</em>(<em>j</em>) is constructed from the similarity matrix as <em>H</em>(<em>i</em>, <em>j</em>) = <em>M̂</em><sub><em>A</em></sub><sub>(</sub><sub><em>i</em></sub><sub>)</sub><sub><em>B</em></sub><sub>(</sub><sub><em>i</em></sub><sub>)</sub>, where <em>i</em> and <em>j</em>
 are positions in sequences. When two groups of sequences are aligned, 
homology matrix between group 1 and group 2 is calculated as:</p><p><img class="content-image" src="m_gkf436equ7a.jpg" alt="graphic" path-from-xml="gkf436equ7a.jpeg"></p><p>where <em>A</em>(<em>n, i</em>) indicates the <em>i</em>th site of the <em>n</em>th sequence in group 1, <em>B</em>(<em>m, j</em>) is the <em>j</em>th site of the <em>m</em>th sequence in group 2, and <em>w</em><sub><em>n </em></sub>is the weighting factor, defined previously, for <em>n</em>th sequences.</p><p>In the NW algorithm (1), the optimal alignment between two groups of sequences is calculated as:</p><p><img class="content-image" src="m_gkf436equ7b.jpg" alt="graphic" path-from-xml="gkf436equ7b.jpeg"></p><p>where <em>P</em>(<em>i,j</em>) is the accumulated score for the optimal path from (1,1) to (<em>i</em>, <em>j</em>), and <em>G</em><sub>1</sub>(<em>i</em>, <em>x</em>) and <em>G</em><sub>2</sub>(<em>j</em>, <em>y</em>) are gap penalties defined below.</p><p>Each
 group of sequences may contain the gaps already introduced at previous 
steps. If a gap is newly introduced at the same position as one of such 
existing gaps, the new gap should not be penalized, because these new 
and existing gaps are probably resulting from a single insertion or 
deletion event. Gotoh (<a reveal-id="GKF436C6" data-open="GKF436C6" class="link link-ref link-reveal xref-default">6</a>) and Thompson <em>et al.</em> (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>)
 developed position‐specific gap penalties depending on the pattern of 
existing gaps. Our method used in this report is rather simpler than 
theirs:</p><p><em>G</em><sub>1</sub>(<em>i, x</em>) = <em>S</em><sup><em>op</em></sup> · {1 – [<em>g</em><sub>1</sub><sup><em>start</em></sup>(<em>x</em>) + <em>g</em><sub>1</sub><sup><em>end</em></sup>(<em>i</em>)]/2},</p><p>where <em>S</em><sup><em>op</em></sup> corresponds to a gap opening penalty, <em>g</em><sub>1</sub><sup><em>start</em></sup>(<em>x</em>) is the number of the gaps that start at the <em>x</em>th site, and <em>g</em><sub>1</sub><sup><em>end</em></sup>(<em>i</em>) is the number of the gaps that end at the <em>i</em>th site. That is,</p><p><img class="content-image" src="m_gkf436equ7c.jpg" alt="graphic" path-from-xml="gkf436equ7c.jpeg"></p><p>where <em>z</em><sub><em>m</em></sub>(<em>i</em>) = 1 and <em>a</em><sub><em>m</em></sub>(<em>i</em>) = 0, if the <em>i</em>th site of sequence <em>m</em> is a gap; otherwise <em>z</em><sub><em>m</em></sub>(<em>i</em>) = 0 and <em>a</em><sub><em>m</em></sub>(<em>i</em>) = 1; <em>w</em><sub><em>m</em></sub> is the weighting factor for sequence <em>m</em>. The other penalty <em>G</em><sub>2</sub>(<em>j</em>, <em>y</em>) is calculated in the same manner. Because this formulation is simpler than existing ones (<a reveal-id="GKF436C6" data-open="GKF436C6" class="link link-ref link-reveal xref-default">6</a>,<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>),
 the CPU time is considerably reduced, but the accuracy of resulting 
alignments is comparable with that by existing scoring systems (see 
Results).</p>                            <h3 scrollto-destination="55547772" id="55547772" class="section-title">Computer programs</h3>
<p>We have developed a program package MAFFT, which incorporates new 
techniques described above. The source code for the FFT algorithm has 
been taken from Press <em>et al.</em> (<a reveal-id="GKF436C19" data-open="GKF436C19" class="link link-ref link-reveal xref-default">19</a>). In MAFFT, the progressive method (<a reveal-id="GKF436C3" data-open="GKF436C3" class="link link-ref link-reveal xref-default">3</a>,<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>) (FFT‐NS‐1, FFT‐NS‐2) and the iterative refinement method (<a reveal-id="GKF436C4" data-open="GKF436C4" class="link link-ref link-reveal xref-default">4</a>–<a reveal-id="GKF436C6" data-open="GKF436C6" class="link link-ref link-reveal xref-default">6</a>) (FFT‐NS‐i) are implemented with some slight modifications described below.</p><p></p><p><em>FFT‐NS‐1.</em>
 Using the FFT algorithm and the normalized similarity matrix described 
above, input sequences are progressively aligned following the branching
 order of sequences in the guide tree. This method is hereafter referred
 to as FFT‐NS‐1. This method requires a guide tree based on the 
all‐pairwise comparison, whose CPU time is <em>O</em>(<em>K</em><sup>2</sup>), where <em>K</em> is the number of sequences. Rapid calculation of a distance matrix is important for the case of large <em>K</em>. Thus we adopted the method of Jones <em>et al.</em> (<a reveal-id="GKF436C22" data-open="GKF436C22" class="link link-ref link-reveal xref-default">22</a>) with two modifications; 20 amino acids are grouped into six physico‐chemical groups (<a reveal-id="GKF436C24" data-open="GKF436C24" class="link link-ref link-reveal xref-default">24</a>), and the number <em>T</em><sub><em>ij</em></sub> of 6‐tuples shared by sequence <em>i</em> and sequence <em>j</em> is counted. This value is converted to a distance <em>D</em><sub><em>ij</em></sub> between sequence <em>i</em> and sequence <em>j</em> as</p><p><em>D</em><sub><em>ij</em></sub> = 1 – [<em>T</em><sub><em>ij</em></sub>/min(<em>T</em><sub><em>ii</em></sub>, <em>T</em><sub><em>jj</em></sub>)].</p><p>The guide tree is constructed from this distance matrix using the UPGMA method (<a reveal-id="GKF436C25" data-open="GKF436C25" class="link link-ref link-reveal xref-default">25</a>).</p><p></p><p><em>FFT‐NS‐2.</em>
 Input sequences are realigned along the guide tree inferred from the 
alignment by FFT‐NS‐1. It is expected that more reliable alignments are 
obtained on the basis of more reliable guide trees (<a reveal-id="GKF436C26" data-open="GKF436C26" class="link link-ref link-reveal xref-default">26</a>). This method is referred to as FFT‐NS‐2.</p><p><em>FFT‐NS‐i.</em>
 An alignment obtained by FFT‐NS‐2 is subjected to further improvement, 
in which the alignment is divided into two groups and realigned (<a reveal-id="GKF436C4" data-open="GKF436C4" class="link link-ref link-reveal xref-default">4</a>–<a reveal-id="GKF436C6" data-open="GKF436C6" class="link link-ref link-reveal xref-default">6</a>). We employ a technique called tree‐dependent restricted partitioning (<a reveal-id="GKF436C27" data-open="GKF436C27" class="link link-ref link-reveal xref-default">27</a>).
 This process is repeated until no better scoring alignment is obtained 
in respect of the score described above. This method is referred to as 
FFT‐NS‐i.</p><p>To test the effect of the FFT algorithm or the 
normalized similarity matrix described above, we compared these three 
methods with several methods in which these newly developed techniques 
are not used.</p><p></p><p><em>NW‐NS‐1/NW‐NS‐2.</em> We examined a 
method that uses the standard NW algorithm, instead of the FFT 
algorithm, with the normalized similarity matrix described above. This 
method is referred to as NW‐NS‐1 or NW‐NS‐2. Concerning the guide trees,
 NW‐NS‐1 and NW‐NS‐2 are identical to FFT‐NS‐1 or FFT‐NS‐2, 
respectively.</p><p></p><p><em>NW‐AP‐2.</em> To test the effect of the 
normalized similarity matrix described above, we examined a method with 
conventional all‐positive similarity matrix (<a reveal-id="GKF436C21" data-open="GKF436C21" class="link link-ref link-reveal xref-default">21</a>), which is made positive by subtracting the smallest number in the matrix from all elements. This is equivalent to setting <em>S</em><sub><em>a</em></sub> in equation <strong>7</strong>
 to 0.82 for the similarity matrix we use. This method is referred to as
 NW‐AP‐2. Except for the similarity matrix, the procedure of NW‐AP‐2 is 
identical to that of NW‐NS‐2.</p>                            <h2 scrollto-destination="55547786" id="55547786" class="section-title">RESULTS</h2>
                            <h3 scrollto-destination="55547787" id="55547787" class="section-title">Computer simulations</h3>
<p>In order to evaluate the performance of the present methods, we have 
conducted computer simulations focusing on the CPU time and the 
accuracy. Using the sequences generated by a simulation program ROSE (<a reveal-id="GKF436C28" data-open="GKF436C28" class="link link-ref link-reveal xref-default">28</a>),
 the CPU times of the present methods and two existing methods, CLUSTALW
 version 1.82 and T‐COFFEE, were compared for the various length and the
 various numbers of sequences. Two types of sequence sets were used; one
 is composed of highly conserved sequences with ∼35–85% identities 
(average distance is 100 PAM), and the other is a group of distantly 
related sequences with ∼15–65% identities (average distance is 250 PAM).
 We also estimated the order of CPU time [<em>Y</em> of <em>O</em>(<em>X</em><sup><em>Y</em></sup>), where <em>X</em> is the length or the number of input sequences] by the power regression analysis.</p><p>Figure <a reveal-id="GKF436F3" data-open="GKF436F3" class="link link-ref link-reveal xref-default">3</a>
 shows the dependence of CPU time on sequence length. The regression 
coefficient of each method is also shown. The standard NW‐based methods,
 CLUSTALW and NW‐NS‐2, require the CPU time proportional to the square 
of sequence length (the regression coefficients are close to 2 for both 
methods) independently of the degrees of sequence similarities, as 
expected. In contrast, the CPU times of FFT‐based methods, FFT‐NS‐2 and 
FFT‐NS‐i, depend on the degree of similarities of input sequences; the 
CPU times of FFT‐NS‐2 and FFT‐NS‐i are virtually proportional to the 
sequence length for highly conserved sequences (regression coefficients 
are close to 1 in Fig. <a reveal-id="GKF436F3" data-open="GKF436F3" class="link link-ref link-reveal xref-default">3</a>A), whereas the CPU time of FFT‐NS‐2 is close to that of NW‐NS‐2 for distantly related sequences (Fig. <a reveal-id="GKF436F3" data-open="GKF436F3" class="link link-ref link-reveal xref-default">3</a>B).</p><p>Figure <a reveal-id="GKF436F4" data-open="GKF436F4" class="link link-ref link-reveal xref-default">4</a>A and B show the dependence of CPU times on the number (<em>K</em>) of input sequences. The time consumption of T‐COFFEE is <em>O</em>(<em>K</em><sup>3</sup>) for alignments of relatively large number of sequences, as Notredame <em>et al.</em> (<a reveal-id="GKF436C9" data-open="GKF436C9" class="link link-ref link-reveal xref-default">9</a>) estimated. CLUSTALW (default), which requires the all‐pairwise comparison by the standard NW algorithm, consumes <em>O</em>(<em>K</em><sup>2</sup>) CPU time. Other methods require CPU times of approximately <em>O</em>(<em>K</em>).</p><p>To
 test the accuracy, five newly developed methods, FFT‐NS‐1, FFT‐NS‐2, 
NW‐NS‐1, NW‐NS‐2 and FFT‐NS‐i, were applied to the sequences of various 
homology levels generated by ROSE (<a reveal-id="GKF436C28" data-open="GKF436C28" class="link link-ref link-reveal xref-default">28</a>).
 The accuracy of each method was measured by sum‐of‐pairs score, where a
 reconstructed alignment is compared with the simulated (‘correct’) 
alignment and the ratio of correctly aligned pairs is calculated from 
all possible pairs (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>). The simulations were repeated 100 times and averaged for each method (Fig. <a reveal-id="GKF436F5" data-open="GKF436F5" class="link link-ref link-reveal xref-default">5</a>).</p><p>The
 accuracy of FFT‐based methods (FFT‐NS‐1 and FFT‐NS‐2) is almost 
equivalent to that of standard NS‐based methods (NW‐NS‐1 and NW‐NS‐2). 
This result indicates that the FFT algorithm does not sacrifice the 
accuracy. FFT‐NS‐2 performs better than FFT‐NS‐1 as expected. FFT‐NS‐i 
has an advantage in accuracy over FFT‐NS‐1 and FFT‐NS‐2 for distantly 
related sequences.</p>                            <h3 scrollto-destination="55547793" id="55547793" class="section-title">Benchmarks using BAliBASE</h3>
<p>Thompson <em>et al.</em> (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>)
 have published a systematic comparison of widely distributed alignment 
programs using the BAliBASE benchmark alignment database (<a reveal-id="GKF436C15" data-open="GKF436C15" class="link link-ref link-reveal xref-default">15</a>),
 a database of ‘correct’ alignments based on three‐dimensional 
structural superimpositions. The BAliBASE database is categorized into 
five different types of references. The first category is made up of 
phylogenetically equidistant members of similar length. In the second 
category, each alignment contains up to three orphan sequences with a 
group of close relatives. The third category contains up to four 
distantly related groups, while the fourth and fifth categories involve 
long terminal and internal insertions, respectively. These references 
will be referred to as categories 1–5 hereafter.</p><p>We have applied 
four methods described in Methods, NW‐AP‐2, NW‐NS‐2, FFT‐NS‐2 and 
FFT‐NS‐i, to this database to compare their efficiencies with those of 
five existing methods, DIALIGN (<a reveal-id="GKF436C29" data-open="GKF436C29" class="link link-ref link-reveal xref-default">29</a>,<a reveal-id="GKF436C30" data-open="GKF436C30" class="link link-ref link-reveal xref-default">30</a>), PIMA (<a reveal-id="GKF436C31" data-open="GKF436C31" class="link link-ref link-reveal xref-default">31</a>), CLUSTALW (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>) version 1.82, PRRP (<a reveal-id="GKF436C32" data-open="GKF436C32" class="link link-ref link-reveal xref-default">32</a>) and T‐COFFEE (<a reveal-id="GKF436C9" data-open="GKF436C9" class="link link-ref link-reveal xref-default">9</a>). The sum‐of‐pairs scores (see above) and the column scores [the ratio of correctly aligned columns (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>)] were calculated and averaged in each category. Wilcoxon matched‐pair signed‐rank test and <em>t</em>‐test were carried out to test the significance of the difference in the accuracy of each method. These tests give <em>P</em>‐values, which is the probability that the observed differences may be due to chance.</p><p>Table <a reveal-id="GKF436TB1" data-open="GKF436TB1" class="link link-ref link-reveal xref-default">1</a>
 shows the results of this benchmark test together with the CPU time of 
each method for performing this test. Unlike the simulation above, 
FFT‐NS‐2 (FFT‐based method) takes CPU time almost equivalent to NW‐NS‐2.
 This is because the FFT algorithm is not efficient for distantly 
related sequences like these tests. NW‐NS‐2 takes less CPU time than 
CLUSTALW does, possibly because of the simple calculation procedure of 
the former. FFT‐NS‐i takes less CPU time than T‐COFFEE does.</p><p>The 
accuracy of NW‐AP‐2, which contains neither the improved scoring system 
described above nor the FFT algorithm, is comparable with that of the 
previous version (1.7) of CLUSTALW (data not shown). By using the 
improved scoring system shown in equation <strong>7</strong>, NW‐NS‐2 
and FFT‐NS‐2 perform considerably better than NW‐AP‐2. T‐COFFEE marked 
the highest average accuracy, but the accuracy of FFT‐NS‐i is comparable
 with that of T‐COFFEE. <em>P</em>‐values by Wilcoxon matched‐pair signed‐rank test are 0.13 for sum‐of‐pairs score and 0.43 for column score, and <em>P</em>‐values by <em>t</em>‐test are 0.10 for sum‐of‐pairs score and 0.23 for column score. Thus the difference is not significant.</p>                            <h3 scrollto-destination="55547798" id="55547798" class="section-title">Applications to the LSU rRNA and RNA polymerase sequences</h3>
<p>BAliBASE is biased toward alignments composed of a small number of 
short sequences; the number of sequences in each alignment is 9.2 and 
sequence length is 251.1 on average. To illustrate the power of our 
approach to practical sequence analyses, we selected two examples of 
relatively large data sets: the nucleotide sequences of LSU rRNA and the
 amino acid sequences of the RNA polymerase largest subunit.</p><p></p><p><em>LSU rRNA.</em> The Ribosomal Database Project (RDP‐II) (<a reveal-id="GKF436C33" data-open="GKF436C33" class="link link-ref link-reveal xref-default">33</a>)
 contains 72 LSU rRNA sequences from Bacteria, Archaea and Eucarya. This
 alignment was used as a reference alignment. We also use another 
reference alignment of 59 sequences in which fragment sequences were 
excluded from the full 72 sequences set (the reference alignments are 
available at http://www.biophys.kyoto‐u.ac.jp/∼katoh/align/example/lsu).
 The CPU times and the sum‐of‐pairs and column scores (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>)
 of NW‐AP‐2, NW‐NS‐2, FFT‐NS‐2 and FFT‐NS‐i were compared with those of 
two existing methods, CLUSTALW (version 1.82) and T‐COFFEE using these 
two data sets (Table <a reveal-id="GKF436TB2" data-open="GKF436TB2" class="link link-ref link-reveal xref-default">2</a>). The FFT‐based methods (FFT‐NS‐2 and FFT‐NS‐i) are efficient for such relatively large data sets.</p><p><em>The largest subunit of RNA polymerase.</em> We used a reference alignment of the largest subunit sequences of RNA polymerase by Iwabe <em>et al.</em> (<a reveal-id="GKF436C34" data-open="GKF436C34" class="link link-ref link-reveal xref-default">34</a>),
 which includes 11 highly conserved blocks. Two data sets, one (large) 
composed of 76 sequences and the other (small) composed of 24 sequences,
 were compiled. Both of them contain amino acid sequences from Bacteria,
 Archaea and three major classes (I, II and III) from Eucarya (the 
reference alignments are available at 
http://www.biophys.kyoto‐u.ac.jp/∼katoh/align/example/rpol). Table <a reveal-id="GKF436TB3" data-open="GKF436TB3" class="link link-ref link-reveal xref-default">3</a>
 shows the CPU time and the number of correctly detected conserved 
blocks of sequences by six methods: NW‐AP‐2, FFT‐NS‐2, NW‐NS‐2, 
FFT‐NS‐i, CLUSTALW version 1.82 and T‐COFFEE. T‐COFFEE, FFT‐NS‐2, 
FFT‐NS‐i and NW‐NS‐2 successfully detected all of the 11 blocks, 
although the CPU times differ for different methods. The CPU time of 
FFT‐NS‐2 (FFT‐based method) is about one‐third of that of NW‐NS‐2 
(standard NW‐based method).</p>                            <h2 scrollto-destination="55547803" id="55547803" class="section-title">DISCUSSION</h2>
<p>It has been supposed that appropriate alignment algorithm depends on the nature of the sequences to be aligned (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>,<a reveal-id="GKF436C35" data-open="GKF436C35" class="link link-ref link-reveal xref-default">35</a>);
 the NW algorithm produces accurate and reliable alignments for 
references 1, 2 and 3 in BAliBASE, whereas the Smith–Waterman (SW) 
algorithm (<a reveal-id="GKF436C36" data-open="GKF436C36" class="link link-ref link-reveal xref-default">36</a>),
 a method for detecting local homology, is successful for categories 4 
and 5. It may be quite impractical to select properly these different 
algorithms, depending on the nature of input sequences; actual sequence 
data contain various types of sequences, i.e. fragment sequences, fusion
 proteins, orphan sequences, over‐representation of some members and so 
on.</p><p>On the basis of such considerations, Notredame <em>et al.</em> (<a reveal-id="GKF436C9" data-open="GKF436C9" class="link link-ref link-reveal xref-default">9</a>)
 formulates a combination of NW and SW alignment procedures in T‐COFFEE.
 This attempt is successful in improving the accuracy at the sacrifice 
of the computational simplicity. Thus, this method may be applicable to 
short and small data sets like those in BAliBASE as Karplus and Hu (<a reveal-id="GKF436C37" data-open="GKF436C37" class="link link-ref link-reveal xref-default">37</a>)
 pointed out. In contrast, the present methods employ a simple NW 
algorithm (NW‐NS‐2) or a more rapid algorithm based on FFT (FFT‐NS‐2 and
 FFT‐NS‐i). Nevertheless, the BAliBASE benchmark tests show that the 
present methods with the normalized similarity matrix perform well also 
for categories 4 and 5. As a result, the accuracy of FFT‐NS‐i is 
comparable with that of T‐COFFEE. This result indicates that the 
accuracy of alignments can be considerably improved without complicating
 any computational process, contrary to the conventional thought that a 
combination of the NW and SW algorithms was necessary for computing 
high‐quality alignments (<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>,<a reveal-id="GKF436C9" data-open="GKF436C9" class="link link-ref link-reveal xref-default">9</a>,<a reveal-id="GKF436C35" data-open="GKF436C35" class="link link-ref link-reveal xref-default">35</a>). The improvement in accuracy was achieved simply by normalizing the similarity matrix.</p><p>This suggests the importance of parameter choice as Thompson <em>et al.</em> (<a reveal-id="GKF436C7" data-open="GKF436C7" class="link link-ref link-reveal xref-default">7</a>,<a reveal-id="GKF436C8" data-open="GKF436C8" class="link link-ref link-reveal xref-default">8</a>)
 pointed out. However, there is a large difference between their 
strategy and ours. The scoring system used in CLUSTALW is complicated 
and time consuming; many parameters in the scoring system dynamically 
vary depending on input sequences. In contrast, the present scoring 
system is simple; the similarity matrix is fixed for any input 
sequences, and even extension gap penalty is not explicitly contained in
 the DP algorithm. Nevertheless, the accuracy of NW‐NS‐2/FFT‐NS‐2 is 
comparable with that of CLUSTALW.</p><p>In all cases tested above, the 
present methods consume generally less CPU time than existing methods of
 comparable accuracy do. It is remarkable that the order of CPU time is 
reduced from <em>O</em>(<em>N</em><sup>2</sup>) to <em>O</em>(<em>N</em>) by the FFT algorithm for highly conserved sequences (Fig. <a reveal-id="GKF436F3" data-open="GKF436F3" class="link link-ref link-reveal xref-default">3</a>A), where <em>N</em>
 is sequence length. Such a rapid multiple alignment method is suitable 
for automated high‐throughput analysis of genomic sequences. At the same
 time, biologists’ expertise is still of particular importance and, 
consequently, a user‐friendly alignment workbench is required, which 
provides easy access to the various information collected by database 
searches, alignment analyses and the predictions obtained by 
non‐homology methods (<a reveal-id="GKF436C38" data-open="GKF436C38" class="link link-ref link-reveal xref-default">38</a>). The method presented here is also useful as a core component of such an integrated alignment workbench.</p><p>The
 MAFFT program package is freely available at 
http://www.biophys.kyoto‐u.ac.jp/∼katoh/programs/align/mafft. It has 
been tested on the Linux operating system. A graphical user interface, 
written by H. Suga, K. Katoh, Y. Yamawaki, K. Kuma, D. Hoshiyama, N. 
Iwabe and T. Miyata, on the X Window System is also available at 
http://www.biophys. kyoto‐u.ac.jp/∼katoh/programs/align/xced.</p>                            <h2 scrollto-destination="55547809" id="55547809" class="section-title">ACKNOWLEDGEMENTS</h2>
<p>We thank Drs N. Iwabe, H. Suga and D. Hoshiyama for helpful comments.
 This work was supported by grants from the Ministry of Education, 
Culture, Sports, Science and Technology of Japan.</p><p></p>                        <a id="55547813" scrollto-destination="55547813"></a>
<div id="GKF436F1" data-id="GKF436F1" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f1.jpg" alt="Figure 1. (A) A result of the FFT analysis. There are two peaks corresponding to two homologous blocks. (B) Sliding window analysis is carried out and the positions of homologous blocks are determined. Note that window size is 30 (see text) but the window size is set to 4 in (B) for simplicity." path-from-xml="gkf436f1.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547813/gkf436f1.jpeg" path-from-xml="gkf436f1.jpeg" target="_blank">View large</a><a section="55547813" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f1.jpeg?Expires=2147483647&amp;Signature=4%7EGj7Nx-lmECV1QMHVjCW-73YiERrqIvUxOkbDUTsJJFREdi8SI5wMOwD78mTlaaU7yoDuHrYKxc1vlFfnq6UohxIufAo9AGyA40jpeIFCvGQKyere51Gw-nllbc3o%7EVQ47Ii3fCWWsF4ptCAs67cF6Z8jLSoFSRIAgxUot7ZQ-ikx9KSIZ6bwB6%7El1cB5QyfC9FhMjzfSiDMuQg7ikcG87ZDmNij5LU02Bb5-wNy2csnKvOzmLYKmE5T2Wl0SqDgRzdE-F9c4SrzdilL8DXzG1-LpNZjrB3YJtCbhgQlp6qxQUm9on25fd-cMjEFd7epmF50C%7ENkrNHNOxWCIGuZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547813&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f1.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 1.</strong> (<strong>A</strong>) A result of the FFT analysis. There are two peaks corresponding to two homologous blocks. (<strong>B</strong>)
 Sliding window analysis is carried out and the positions of homologous 
blocks are determined. Note that window size is 30 (see text) but the 
window size is set to 4 in (B) for simplicity.</p></div></div><div content-id="GKF436F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f1.jpg" alt="Figure 1. (A) A result of the FFT analysis. There are two peaks corresponding to two homologous blocks. (B) Sliding window analysis is carried out and the positions of homologous blocks are determined. Note that window size is 30 (see text) but the window size is set to 4 in (B) for simplicity." path-from-xml="gkf436f1.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547813/gkf436f1.jpeg" path-from-xml="gkf436f1.jpeg" target="_blank">View large</a><a section="55547813" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f1.jpeg?Expires=2147483647&amp;Signature=4%7EGj7Nx-lmECV1QMHVjCW-73YiERrqIvUxOkbDUTsJJFREdi8SI5wMOwD78mTlaaU7yoDuHrYKxc1vlFfnq6UohxIufAo9AGyA40jpeIFCvGQKyere51Gw-nllbc3o%7EVQ47Ii3fCWWsF4ptCAs67cF6Z8jLSoFSRIAgxUot7ZQ-ikx9KSIZ6bwB6%7El1cB5QyfC9FhMjzfSiDMuQg7ikcG87ZDmNij5LU02Bb5-wNy2csnKvOzmLYKmE5T2Wl0SqDgRzdE-F9c4SrzdilL8DXzG1-LpNZjrB3YJtCbhgQlp6qxQUm9on25fd-cMjEFd7epmF50C%7ENkrNHNOxWCIGuZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547813&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f1.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 1.</strong> (<strong>A</strong>) A result of the FFT analysis. There are two peaks corresponding to two homologous blocks. (<strong>B</strong>)
 Sliding window analysis is carried out and the positions of homologous 
blocks are determined. Note that window size is 30 (see text) but the 
window size is set to 4 in (B) for simplicity.</p></div></div>                        <a id="55547814" scrollto-destination="55547814"></a>
<div id="GKF436F2" data-id="GKF436F2" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f2.jpg" alt="Figure 2. (A) An example of the segment‐level DP; (B) Reducing the area for DP on a homology matrix." path-from-xml="gkf436f2.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547814/gkf436f2.jpeg" path-from-xml="gkf436f2.jpeg" target="_blank">View large</a><a section="55547814" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f2.jpeg?Expires=2147483647&amp;Signature=IfGM6VKCCdQGkasax5j-DAIHlP40DBONY-PEDqhYh2G6OB5Trgsz7Md-8wAUbmzK93hWaErBK0c2WdlqKud6TIXT3dYMTZkCWX5%7E%7Eien0RvWZumiB3M4-FPRIZymmxuiueJLarUQrEEEPkWSW9R%7EqoTcjiYYGY3nR%7ETbjdNhMMDvWE99RfPRMXKWcYBgAzhA0PZ861t0ov7e10GG96vIyPRAPDXyktp7kQL%7ECkqRLxv9hF6ItPbZ6DjsLU6r6sJyS42W%7E0tUtf21getacHOrzXZSxzMLAMQe8z-7WvOMnokzDvsnlzMlsYoRBQ9Idr2oW%7EJgunnmxnJ4HSpZhr8goA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547814&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f2.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 2.</strong> (<strong>A</strong>) An example of the segment‐level DP; (<strong>B</strong>) Reducing the area for DP on a homology matrix.</p></div></div><div content-id="GKF436F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f2.jpg" alt="Figure 2. (A) An example of the segment‐level DP; (B) Reducing the area for DP on a homology matrix." path-from-xml="gkf436f2.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547814/gkf436f2.jpeg" path-from-xml="gkf436f2.jpeg" target="_blank">View large</a><a section="55547814" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f2.jpeg?Expires=2147483647&amp;Signature=IfGM6VKCCdQGkasax5j-DAIHlP40DBONY-PEDqhYh2G6OB5Trgsz7Md-8wAUbmzK93hWaErBK0c2WdlqKud6TIXT3dYMTZkCWX5%7E%7Eien0RvWZumiB3M4-FPRIZymmxuiueJLarUQrEEEPkWSW9R%7EqoTcjiYYGY3nR%7ETbjdNhMMDvWE99RfPRMXKWcYBgAzhA0PZ861t0ov7e10GG96vIyPRAPDXyktp7kQL%7ECkqRLxv9hF6ItPbZ6DjsLU6r6sJyS42W%7E0tUtf21getacHOrzXZSxzMLAMQe8z-7WvOMnokzDvsnlzMlsYoRBQ9Idr2oW%7EJgunnmxnJ4HSpZhr8goA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547814&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f2.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 2.</strong> (<strong>A</strong>) An example of the segment‐level DP; (<strong>B</strong>) Reducing the area for DP on a homology matrix.</p></div></div>                        <a id="55547815" scrollto-destination="55547815"></a>
<div id="GKF436F3" data-id="GKF436F3" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f3.jpg" alt="Figure 3. The plot of CPU time versus the average lengths of input sequences for three methods described in the text, FFT‐NS‐2, FFT‐NS‐i and NW‐NS‐2, and two existing methods, CLUSTALW and T‐COFFEE. The average percent identities among input sequences are ∼35–85% (A) and ∼15–65% (B). The number of sequences is 40. The regression coefficient calculated from the power regression analysis is shown for each method. For all cases, default parameters were used, except for CLUSTALW, in which both cases default setting (CLW18d) and ‘quicktree’ option (CLW18q) were examined. All of the calculations were performed on a Linux operating system (Intel Xeon 1.7 GHz with 1 GB of memory). The gcc version 2.96 compiler was used with the optimization option ‘‐O3’." path-from-xml="gkf436f3.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547815/gkf436f3.jpeg" path-from-xml="gkf436f3.jpeg" target="_blank">View large</a><a section="55547815" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f3.jpeg?Expires=2147483647&amp;Signature=VVCi7ywbzKaPvn9Pdzze81zoFyBuOgqm266mXoFvLR%7EPSJ4nd-NH918Kfvq4S1OnjpAZcaKLsb4njkrKVT0-pyD6RuGerqw3H5jvhRsYceSUkcSSu-9eQ9rELcQnLxvS-qKzuhrtidjt0TxjYBAxwOaC9PxFox4-eMXXRH2n6YaTcDcJfq19ZOX7niOOYTQ5rCxm7fCZLVAzKFGc3gG9BIsDkw1zFYwt-eyCG5uh9XCMVj9mqO5f5oK9KnqT9lB4FAICBuuRtLWkhyJnlI88Co5LKjXYVbD2hygLB7%7EjQhgKS-r0cHXFts0PjNXkddLi603kB%7EUlcCIzFw7JDLQS3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547815&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f3.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 3.</strong>
 The plot of CPU time versus the average lengths of input sequences for 
three methods described in the text, FFT‐NS‐2, FFT‐NS‐i and NW‐NS‐2, and
 two existing methods, CLUSTALW and T‐COFFEE. The average percent 
identities among input sequences are ∼35–85% (<strong>A</strong>) and ∼15–65% (<strong>B</strong>).
 The number of sequences is 40. The regression coefficient calculated 
from the power regression analysis is shown for each method. For all 
cases, default parameters were used, except for CLUSTALW, in which both 
cases default setting (CLW18d) and ‘quicktree’ option (CLW18q) were 
examined. All of the calculations were performed on a Linux operating 
system (Intel Xeon 1.7 GHz with 1 GB of memory). The gcc version 2.96 
compiler was used with the optimization option ‘‐O3’.</p></div></div><div content-id="GKF436F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f3.jpg" alt="Figure 3. The plot of CPU time versus the average lengths of input sequences for three methods described in the text, FFT‐NS‐2, FFT‐NS‐i and NW‐NS‐2, and two existing methods, CLUSTALW and T‐COFFEE. The average percent identities among input sequences are ∼35–85% (A) and ∼15–65% (B). The number of sequences is 40. The regression coefficient calculated from the power regression analysis is shown for each method. For all cases, default parameters were used, except for CLUSTALW, in which both cases default setting (CLW18d) and ‘quicktree’ option (CLW18q) were examined. All of the calculations were performed on a Linux operating system (Intel Xeon 1.7 GHz with 1 GB of memory). The gcc version 2.96 compiler was used with the optimization option ‘‐O3’." path-from-xml="gkf436f3.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547815/gkf436f3.jpeg" path-from-xml="gkf436f3.jpeg" target="_blank">View large</a><a section="55547815" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f3.jpeg?Expires=2147483647&amp;Signature=VVCi7ywbzKaPvn9Pdzze81zoFyBuOgqm266mXoFvLR%7EPSJ4nd-NH918Kfvq4S1OnjpAZcaKLsb4njkrKVT0-pyD6RuGerqw3H5jvhRsYceSUkcSSu-9eQ9rELcQnLxvS-qKzuhrtidjt0TxjYBAxwOaC9PxFox4-eMXXRH2n6YaTcDcJfq19ZOX7niOOYTQ5rCxm7fCZLVAzKFGc3gG9BIsDkw1zFYwt-eyCG5uh9XCMVj9mqO5f5oK9KnqT9lB4FAICBuuRtLWkhyJnlI88Co5LKjXYVbD2hygLB7%7EjQhgKS-r0cHXFts0PjNXkddLi603kB%7EUlcCIzFw7JDLQS3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547815&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f3.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 3.</strong>
 The plot of CPU time versus the average lengths of input sequences for 
three methods described in the text, FFT‐NS‐2, FFT‐NS‐i and NW‐NS‐2, and
 two existing methods, CLUSTALW and T‐COFFEE. The average percent 
identities among input sequences are ∼35–85% (<strong>A</strong>) and ∼15–65% (<strong>B</strong>).
 The number of sequences is 40. The regression coefficient calculated 
from the power regression analysis is shown for each method. For all 
cases, default parameters were used, except for CLUSTALW, in which both 
cases default setting (CLW18d) and ‘quicktree’ option (CLW18q) were 
examined. All of the calculations were performed on a Linux operating 
system (Intel Xeon 1.7 GHz with 1 GB of memory). The gcc version 2.96 
compiler was used with the optimization option ‘‐O3’.</p></div></div>                        <a id="55547816" scrollto-destination="55547816"></a>
<div id="GKF436F4" data-id="GKF436F4" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f4.jpg" alt="Figure 4. The plot of CPU time versus the number of input sequences for three methods described in the text, FFT‐NS‐2 and FFT‐NS‐i, and two existing methods, CLUSTALW and T‐COFFEE. The average percent identities among input sequences are ∼35–85% (A) and ∼15–65% (B). The average length of input sequences is 300. The regression coefficient calculated from the power regression analysis is shown for each method. For all cases, default parameters were used, except for CLUSTALW, in which both cases default setting (CLW18d) and ‘quicktree’ option (CLW18q) were examined. All of the calculations were performed on a Linux operating system (Intel Xeon 1.7 GHz with 1 GB of memory). The gcc version 2.96 compiler was used with the optimization option ‘‐O3’." path-from-xml="gkf436f4.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547816/gkf436f4.jpeg" path-from-xml="gkf436f4.jpeg" target="_blank">View large</a><a section="55547816" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f4.jpeg?Expires=2147483647&amp;Signature=yKQUbItbdmRQwhhSm0e%7E1bk4Vtq-tvbkyB3vgYQZPxNGARIynKKkVhEylpxCgyBRV0-b2r98%7E%7ET7eQTZFuPaZmGa1-ecyzK6fSZ8gcgj6O6XtfUuw4jlpbanyBFmwXmb6xs5RszC5vcLlHzEZI8d6alq0t9MrVMwx7M3j6S06bVOoQ4JYcj8vZ1jyrsdGd9wSNJVKe-vG9R2cyk7yiQ26lVilKsYnlHIVlShj6znWJRDe34Y57Uz%7EYkHsRWRlvX5b77%7EcUyMr5hjg1r8%7EmxqgAPrGcrZHZymNP-MbKrNZQgGWKYZEDS%7EgZ%7Ej6PlDrebl99is%7EJrAmVX8vweGjLWOwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547816&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f4.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 4.</strong>
 The plot of CPU time versus the number of input sequences for three 
methods described in the text, FFT‐NS‐2 and FFT‐NS‐i, and two existing 
methods, CLUSTALW and T‐COFFEE. The average percent identities among 
input sequences are ∼35–85% (<strong>A</strong>) and ∼15–65% (<strong>B</strong>).
 The average length of input sequences is 300. The regression 
coefficient calculated from the power regression analysis is shown for 
each method. For all cases, default parameters were used, except for 
CLUSTALW, in which both cases default setting (CLW18d) and ‘quicktree’ 
option (CLW18q) were examined. All of the calculations were performed on
 a Linux operating system (Intel Xeon 1.7 GHz with 1 GB of memory). The 
gcc version 2.96 compiler was used with the optimization option ‘‐O3’.</p></div></div><div content-id="GKF436F4" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f4.jpg" alt="Figure 4. The plot of CPU time versus the number of input sequences for three methods described in the text, FFT‐NS‐2 and FFT‐NS‐i, and two existing methods, CLUSTALW and T‐COFFEE. The average percent identities among input sequences are ∼35–85% (A) and ∼15–65% (B). The average length of input sequences is 300. The regression coefficient calculated from the power regression analysis is shown for each method. For all cases, default parameters were used, except for CLUSTALW, in which both cases default setting (CLW18d) and ‘quicktree’ option (CLW18q) were examined. All of the calculations were performed on a Linux operating system (Intel Xeon 1.7 GHz with 1 GB of memory). The gcc version 2.96 compiler was used with the optimization option ‘‐O3’." path-from-xml="gkf436f4.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547816/gkf436f4.jpeg" path-from-xml="gkf436f4.jpeg" target="_blank">View large</a><a section="55547816" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f4.jpeg?Expires=2147483647&amp;Signature=yKQUbItbdmRQwhhSm0e%7E1bk4Vtq-tvbkyB3vgYQZPxNGARIynKKkVhEylpxCgyBRV0-b2r98%7E%7ET7eQTZFuPaZmGa1-ecyzK6fSZ8gcgj6O6XtfUuw4jlpbanyBFmwXmb6xs5RszC5vcLlHzEZI8d6alq0t9MrVMwx7M3j6S06bVOoQ4JYcj8vZ1jyrsdGd9wSNJVKe-vG9R2cyk7yiQ26lVilKsYnlHIVlShj6znWJRDe34Y57Uz%7EYkHsRWRlvX5b77%7EcUyMr5hjg1r8%7EmxqgAPrGcrZHZymNP-MbKrNZQgGWKYZEDS%7EgZ%7Ej6PlDrebl99is%7EJrAmVX8vweGjLWOwA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547816&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f4.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 4.</strong>
 The plot of CPU time versus the number of input sequences for three 
methods described in the text, FFT‐NS‐2 and FFT‐NS‐i, and two existing 
methods, CLUSTALW and T‐COFFEE. The average percent identities among 
input sequences are ∼35–85% (<strong>A</strong>) and ∼15–65% (<strong>B</strong>).
 The average length of input sequences is 300. The regression 
coefficient calculated from the power regression analysis is shown for 
each method. For all cases, default parameters were used, except for 
CLUSTALW, in which both cases default setting (CLW18d) and ‘quicktree’ 
option (CLW18q) were examined. All of the calculations were performed on
 a Linux operating system (Intel Xeon 1.7 GHz with 1 GB of memory). The 
gcc version 2.96 compiler was used with the optimization option ‘‐O3’.</p></div></div>                        <a id="55547817" scrollto-destination="55547817"></a>
<div id="GKF436F5" data-id="GKF436F5" class="fig fig-section"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f5.jpg" alt="Figure 5. The plot of sum‐of‐pairs score (8) versus the average distance of input sequences for five methods, FFT‐NS‐1, FFT‐NS‐2, FFT‐NS‐i, NW‐NS‐1 and NW‐NS‐2. The number of input sequences is 40, and sequence lengths are 200 sites on average. Vertical lines indicate the standard deviations of the scores. For all cases, default parameters were used." path-from-xml="gkf436f5.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547817/gkf436f5.jpeg" path-from-xml="gkf436f5.jpeg" target="_blank">View large</a><a section="55547817" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f5.jpeg?Expires=2147483647&amp;Signature=4tjXn1UQSBhVDbIuBKDwvgLDh5PB0MDMeNrlfFGPbZ07LkfRnZMosTr2Nhl3u0kEEbr9kCupDo71mqTuSChAQgd4rwPacSqOTzcbk6POnMkChhBlBgXn-J6JmE198zJa14U8WppWmVuTe54urv0OIluW6HrTf5SY2A6rPj8JSJQ29fZw0Mr2iTOW-MDCBJsfsxLyKgok5RxdEKniZuPoXzdve9zQHq7R6WW-nUqQfKt82EmZJ46BXR2YqlPpY60ujWUunOV1rjlyy-CwoDTRBS0Sxq-eWqXX2qL0X7hHeKMepiXlZ81fuTrvZgLttRqZy1JTgEQphBVFuFs9L0J6pg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547817&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f5.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 5.</strong>
 The plot of sum‐of‐pairs score (8) versus the average distance of input
 sequences for five methods, FFT‐NS‐1, FFT‐NS‐2, FFT‐NS‐i, NW‐NS‐1 and 
NW‐NS‐2. The number of input sequences is 40, and sequence lengths are 
200 sites on average. Vertical lines indicate the standard deviations of
 the scores. For all cases, default parameters were used.</p></div></div><div content-id="GKF436F5" class="fig fig-modal reveal-modal" style="display: block;"><div class="graphic-wrap"><img class="content-image" src="m_gkf436f5.jpg" alt="Figure 5. The plot of sum‐of‐pairs score (8) versus the average distance of input sequences for five methods, FFT‐NS‐1, FFT‐NS‐2, FFT‐NS‐i, NW‐NS‐1 and NW‐NS‐2. The number of input sequences is 40, and sequence lengths are 200 sites on average. Vertical lines indicate the standard deviations of the scores. For all cases, default parameters were used." path-from-xml="gkf436f5.jpeg"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/55547817/gkf436f5.jpeg" path-from-xml="gkf436f5.jpeg" target="_blank">View large</a><a section="55547817" href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/nar/30/14/10.1093_nar_gkf436/2/gkf436f5.jpeg?Expires=2147483647&amp;Signature=4tjXn1UQSBhVDbIuBKDwvgLDh5PB0MDMeNrlfFGPbZ07LkfRnZMosTr2Nhl3u0kEEbr9kCupDo71mqTuSChAQgd4rwPacSqOTzcbk6POnMkChhBlBgXn-J6JmE198zJa14U8WppWmVuTe54urv0OIluW6HrTf5SY2A6rPj8JSJQ29fZw0Mr2iTOW-MDCBJsfsxLyKgok5RxdEKniZuPoXzdve9zQHq7R6WW-nUqQfKt82EmZJ46BXR2YqlPpY60ujWUunOV1rjlyy-CwoDTRBS0Sxq-eWqXX2qL0X7hHeKMepiXlZ81fuTrvZgLttRqZy1JTgEQphBVFuFs9L0J6pg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=55547817&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="gkf436f5.jpeg" target="">Download slide</a></div></div><div class="caption fig-caption"><p><strong>Figure 5.</strong>
 The plot of sum‐of‐pairs score (8) versus the average distance of input
 sequences for five methods, FFT‐NS‐1, FFT‐NS‐2, FFT‐NS‐i, NW‐NS‐1 and 
NW‐NS‐2. The number of input sequences is 40, and sequence lengths are 
200 sites on average. Vertical lines indicate the standard deviations of
 the scores. For all cases, default parameters were used.</p></div></div>                        <a id="55547818" scrollto-destination="55547818"></a>
<div content-id="GKF436TB1" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKF436TB1" class="table-wrap-title" data-id="GKF436TB1"><div class="label -label"><strong>Table 1.</strong></div><div class="caption"><p>Sum‐of‐pairs scores and column scores of various alignment methods for the BAliBASE benchmark tests</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method &nbsp;</td><td>CPU time (s)&nbsp;</td><td>Cat. 1 &nbsp;</td><td>Cat. 2 &nbsp;</td><td>Cat. 3 &nbsp;</td><td>Cat. 4 &nbsp;</td><td>Cat. 5 &nbsp;</td><td>Average1 &nbsp;</td><td>Average2 &nbsp;</td></tr><tr><td>Progressive methods&nbsp;</td></tr><tr><td> PIMA &nbsp;</td><td>1116 &nbsp;</td><td>0.825/0.737 &nbsp;</td><td>0.751/0.127 &nbsp;</td><td>0.525/0.262 &nbsp;</td><td>0.700/0.480 &nbsp;</td><td>0.788/0.555 &nbsp;</td><td>0.772/0.558 &nbsp;</td><td>0.718/0.432 &nbsp;</td></tr><tr><td> CLW18d &nbsp;</td><td>2202 &nbsp;</td><td>0.871/0.792 &nbsp;</td><td>0.856/0.329 &nbsp;</td><td>0.754/0.490 &nbsp;</td><td>0.745/0.417 &nbsp;</td><td>0.852/0.617 &nbsp;</td><td>0.844/0.639 &nbsp;</td><td>0.816/0.529 &nbsp;</td></tr><tr><td> CLW18q &nbsp;</td><td>1657 &nbsp;</td><td>0.871/0.790 &nbsp;</td><td>0.859/0.334 &nbsp;</td><td>0.763/0.473 &nbsp;</td><td>0.728/0.402 &nbsp;</td><td>0.887/0.709 &nbsp;</td><td>0.847/0.644 &nbsp;</td><td>0.824/0.542 &nbsp;</td></tr><tr><td> NW‐AP‐2 &nbsp;</td><td>250 &nbsp;</td><td>0.842/0.746 &nbsp;</td><td>0.833/0.268 &nbsp;</td><td>0.770/0.443 &nbsp;</td><td>0.703/0.311 &nbsp;</td><td>0.851/0.667 &nbsp;</td><td>0.821/0.593 &nbsp;</td><td>0.800/0.487 &nbsp;</td></tr><tr><td> NW‐NS‐2 &nbsp;</td><td>243 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td> FFT‐NS‐2 &nbsp;</td><td>227 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td>Iterative refinement methods and T‐COFFEE&nbsp;</td></tr><tr><td> DIALIGN2‐1 &nbsp;</td><td>18132 &nbsp;</td><td>0.792/0.681 &nbsp;</td><td>0.814/0.219 &nbsp;</td><td>0.673/0.327 &nbsp;</td><td>0.818/0.615 &nbsp;</td><td>0.938/0.840 &nbsp;</td><td>0.801/0.584 &nbsp;</td><td>0.807/0.536 &nbsp;</td></tr><tr><td> PRRP &nbsp;</td><td>9782 &nbsp;</td><td>0.871/0.793 &nbsp;</td><td>0.860/0.354 &nbsp;</td><td>0.823/0.569 &nbsp;</td><td>0.663/0.275 &nbsp;</td><td>0.885/0.742 &nbsp;</td><td>0.845/0.646 &nbsp;</td><td>0.820/0.547 &nbsp;</td></tr><tr><td> T‐COFFEE &nbsp;</td><td>12065 &nbsp;</td><td>0.876/0.797 &nbsp;</td><td>0.856/0.343 &nbsp;</td><td>0.777/0.497 &nbsp;</td><td>0.811/0.555 &nbsp;</td><td>0.961/0.901 &nbsp;</td><td>0.865/0.683 &nbsp;</td><td>0.856/0.619 &nbsp;</td></tr><tr><td> FFT‐NS‐i &nbsp;</td><td>1466 &nbsp;</td><td>0.864/0.787 &nbsp;</td><td>0.853/0.363 &nbsp;</td><td>0.789/0.518 &nbsp;</td><td>0.799/0.534 &nbsp;</td><td>0.956/0.835 &nbsp;</td><td>0.857/0.675 &nbsp;</td><td>0.852/0.607 &nbsp;</td></tr><tr><td>Number of alignments&nbsp;</td><td>&nbsp;</td><td>82&nbsp;</td><td>23&nbsp;</td><td>12&nbsp;</td><td>15&nbsp;</td><td>12&nbsp;</td><td>144&nbsp;</td><td>–&nbsp;</td></tr></tbody></table></div><div content-id="GKF436TB1" class="table-modal"><table><tbody><tr><td>Method &nbsp;</td><td>CPU time (s)&nbsp;</td><td>Cat. 1 &nbsp;</td><td>Cat. 2 &nbsp;</td><td>Cat. 3 &nbsp;</td><td>Cat. 4 &nbsp;</td><td>Cat. 5 &nbsp;</td><td>Average1 &nbsp;</td><td>Average2 &nbsp;</td></tr><tr><td>Progressive methods&nbsp;</td></tr><tr><td> PIMA &nbsp;</td><td>1116 &nbsp;</td><td>0.825/0.737 &nbsp;</td><td>0.751/0.127 &nbsp;</td><td>0.525/0.262 &nbsp;</td><td>0.700/0.480 &nbsp;</td><td>0.788/0.555 &nbsp;</td><td>0.772/0.558 &nbsp;</td><td>0.718/0.432 &nbsp;</td></tr><tr><td> CLW18d &nbsp;</td><td>2202 &nbsp;</td><td>0.871/0.792 &nbsp;</td><td>0.856/0.329 &nbsp;</td><td>0.754/0.490 &nbsp;</td><td>0.745/0.417 &nbsp;</td><td>0.852/0.617 &nbsp;</td><td>0.844/0.639 &nbsp;</td><td>0.816/0.529 &nbsp;</td></tr><tr><td> CLW18q &nbsp;</td><td>1657 &nbsp;</td><td>0.871/0.790 &nbsp;</td><td>0.859/0.334 &nbsp;</td><td>0.763/0.473 &nbsp;</td><td>0.728/0.402 &nbsp;</td><td>0.887/0.709 &nbsp;</td><td>0.847/0.644 &nbsp;</td><td>0.824/0.542 &nbsp;</td></tr><tr><td> NW‐AP‐2 &nbsp;</td><td>250 &nbsp;</td><td>0.842/0.746 &nbsp;</td><td>0.833/0.268 &nbsp;</td><td>0.770/0.443 &nbsp;</td><td>0.703/0.311 &nbsp;</td><td>0.851/0.667 &nbsp;</td><td>0.821/0.593 &nbsp;</td><td>0.800/0.487 &nbsp;</td></tr><tr><td> NW‐NS‐2 &nbsp;</td><td>243 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td> FFT‐NS‐2 &nbsp;</td><td>227 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td>Iterative refinement methods and T‐COFFEE&nbsp;</td></tr><tr><td> DIALIGN2‐1 &nbsp;</td><td>18132 &nbsp;</td><td>0.792/0.681 &nbsp;</td><td>0.814/0.219 &nbsp;</td><td>0.673/0.327 &nbsp;</td><td>0.818/0.615 &nbsp;</td><td>0.938/0.840 &nbsp;</td><td>0.801/0.584 &nbsp;</td><td>0.807/0.536 &nbsp;</td></tr><tr><td> PRRP &nbsp;</td><td>9782 &nbsp;</td><td>0.871/0.793 &nbsp;</td><td>0.860/0.354 &nbsp;</td><td>0.823/0.569 &nbsp;</td><td>0.663/0.275 &nbsp;</td><td>0.885/0.742 &nbsp;</td><td>0.845/0.646 &nbsp;</td><td>0.820/0.547 &nbsp;</td></tr><tr><td> T‐COFFEE &nbsp;</td><td>12065 &nbsp;</td><td>0.876/0.797 &nbsp;</td><td>0.856/0.343 &nbsp;</td><td>0.777/0.497 &nbsp;</td><td>0.811/0.555 &nbsp;</td><td>0.961/0.901 &nbsp;</td><td>0.865/0.683 &nbsp;</td><td>0.856/0.619 &nbsp;</td></tr><tr><td> FFT‐NS‐i &nbsp;</td><td>1466 &nbsp;</td><td>0.864/0.787 &nbsp;</td><td>0.853/0.363 &nbsp;</td><td>0.789/0.518 &nbsp;</td><td>0.799/0.534 &nbsp;</td><td>0.956/0.835 &nbsp;</td><td>0.857/0.675 &nbsp;</td><td>0.852/0.607 &nbsp;</td></tr><tr><td>Number of alignments&nbsp;</td><td>&nbsp;</td><td>82&nbsp;</td><td>23&nbsp;</td><td>12&nbsp;</td><td>15&nbsp;</td><td>12&nbsp;</td><td>144&nbsp;</td><td>–&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>Categories
 (Cat.) 1–5 correspond to five different categories of alignments 
described in the text. Two types of scores averaged across alignments of
 each category are shown in each column, separated by a slash (the 
average of sum‐of‐pairs scores/the average of column scores). Average1 
gives the average score across all the 144 alignments. Since the number 
of alignments differs for different categories ranging from 12 to 82, 
another type of average score (Average2), i.e. the score averaged across
 the five categories, was calculated for each method after Notredame <em>et al.</em>
 (9). Total CPU time for computing all 144 alignments is also shown for 
each method. For all cases, default parameters were used, except for 
CLUSTALW, in which both cases default setting (indicated by CLW18d) and 
‘quicktree’ option (CLW18q) were examined. All calculations were 
performed on a Sun Ultra/2 workstation (UltraSPARC 168 MHz with 128 MB 
of memory, Solaris 2.6). The gcc version 2.8.1 compiler was used with 
the optimization option ‘‐O3’, except for DIALIGN, for which the 
pre‐compiled version by the authors was used.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/55547818" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKF436TB1" class="table-wrap-title" data-id="GKF436TB1"><div class="label -label"><strong>Table 1.</strong></div><div class="caption"><p>Sum‐of‐pairs scores and column scores of various alignment methods for the BAliBASE benchmark tests</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method &nbsp;</td><td>CPU time (s)&nbsp;</td><td>Cat. 1 &nbsp;</td><td>Cat. 2 &nbsp;</td><td>Cat. 3 &nbsp;</td><td>Cat. 4 &nbsp;</td><td>Cat. 5 &nbsp;</td><td>Average1 &nbsp;</td><td>Average2 &nbsp;</td></tr><tr><td>Progressive methods&nbsp;</td></tr><tr><td> PIMA &nbsp;</td><td>1116 &nbsp;</td><td>0.825/0.737 &nbsp;</td><td>0.751/0.127 &nbsp;</td><td>0.525/0.262 &nbsp;</td><td>0.700/0.480 &nbsp;</td><td>0.788/0.555 &nbsp;</td><td>0.772/0.558 &nbsp;</td><td>0.718/0.432 &nbsp;</td></tr><tr><td> CLW18d &nbsp;</td><td>2202 &nbsp;</td><td>0.871/0.792 &nbsp;</td><td>0.856/0.329 &nbsp;</td><td>0.754/0.490 &nbsp;</td><td>0.745/0.417 &nbsp;</td><td>0.852/0.617 &nbsp;</td><td>0.844/0.639 &nbsp;</td><td>0.816/0.529 &nbsp;</td></tr><tr><td> CLW18q &nbsp;</td><td>1657 &nbsp;</td><td>0.871/0.790 &nbsp;</td><td>0.859/0.334 &nbsp;</td><td>0.763/0.473 &nbsp;</td><td>0.728/0.402 &nbsp;</td><td>0.887/0.709 &nbsp;</td><td>0.847/0.644 &nbsp;</td><td>0.824/0.542 &nbsp;</td></tr><tr><td> NW‐AP‐2 &nbsp;</td><td>250 &nbsp;</td><td>0.842/0.746 &nbsp;</td><td>0.833/0.268 &nbsp;</td><td>0.770/0.443 &nbsp;</td><td>0.703/0.311 &nbsp;</td><td>0.851/0.667 &nbsp;</td><td>0.821/0.593 &nbsp;</td><td>0.800/0.487 &nbsp;</td></tr><tr><td> NW‐NS‐2 &nbsp;</td><td>243 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td> FFT‐NS‐2 &nbsp;</td><td>227 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td>Iterative refinement methods and T‐COFFEE&nbsp;</td></tr><tr><td> DIALIGN2‐1 &nbsp;</td><td>18132 &nbsp;</td><td>0.792/0.681 &nbsp;</td><td>0.814/0.219 &nbsp;</td><td>0.673/0.327 &nbsp;</td><td>0.818/0.615 &nbsp;</td><td>0.938/0.840 &nbsp;</td><td>0.801/0.584 &nbsp;</td><td>0.807/0.536 &nbsp;</td></tr><tr><td> PRRP &nbsp;</td><td>9782 &nbsp;</td><td>0.871/0.793 &nbsp;</td><td>0.860/0.354 &nbsp;</td><td>0.823/0.569 &nbsp;</td><td>0.663/0.275 &nbsp;</td><td>0.885/0.742 &nbsp;</td><td>0.845/0.646 &nbsp;</td><td>0.820/0.547 &nbsp;</td></tr><tr><td> T‐COFFEE &nbsp;</td><td>12065 &nbsp;</td><td>0.876/0.797 &nbsp;</td><td>0.856/0.343 &nbsp;</td><td>0.777/0.497 &nbsp;</td><td>0.811/0.555 &nbsp;</td><td>0.961/0.901 &nbsp;</td><td>0.865/0.683 &nbsp;</td><td>0.856/0.619 &nbsp;</td></tr><tr><td> FFT‐NS‐i &nbsp;</td><td>1466 &nbsp;</td><td>0.864/0.787 &nbsp;</td><td>0.853/0.363 &nbsp;</td><td>0.789/0.518 &nbsp;</td><td>0.799/0.534 &nbsp;</td><td>0.956/0.835 &nbsp;</td><td>0.857/0.675 &nbsp;</td><td>0.852/0.607 &nbsp;</td></tr><tr><td>Number of alignments&nbsp;</td><td>&nbsp;</td><td>82&nbsp;</td><td>23&nbsp;</td><td>12&nbsp;</td><td>15&nbsp;</td><td>12&nbsp;</td><td>144&nbsp;</td><td>–&nbsp;</td></tr></tbody></table></div><div content-id="GKF436TB1" class="table-modal"><table><tbody><tr><td>Method &nbsp;</td><td>CPU time (s)&nbsp;</td><td>Cat. 1 &nbsp;</td><td>Cat. 2 &nbsp;</td><td>Cat. 3 &nbsp;</td><td>Cat. 4 &nbsp;</td><td>Cat. 5 &nbsp;</td><td>Average1 &nbsp;</td><td>Average2 &nbsp;</td></tr><tr><td>Progressive methods&nbsp;</td></tr><tr><td> PIMA &nbsp;</td><td>1116 &nbsp;</td><td>0.825/0.737 &nbsp;</td><td>0.751/0.127 &nbsp;</td><td>0.525/0.262 &nbsp;</td><td>0.700/0.480 &nbsp;</td><td>0.788/0.555 &nbsp;</td><td>0.772/0.558 &nbsp;</td><td>0.718/0.432 &nbsp;</td></tr><tr><td> CLW18d &nbsp;</td><td>2202 &nbsp;</td><td>0.871/0.792 &nbsp;</td><td>0.856/0.329 &nbsp;</td><td>0.754/0.490 &nbsp;</td><td>0.745/0.417 &nbsp;</td><td>0.852/0.617 &nbsp;</td><td>0.844/0.639 &nbsp;</td><td>0.816/0.529 &nbsp;</td></tr><tr><td> CLW18q &nbsp;</td><td>1657 &nbsp;</td><td>0.871/0.790 &nbsp;</td><td>0.859/0.334 &nbsp;</td><td>0.763/0.473 &nbsp;</td><td>0.728/0.402 &nbsp;</td><td>0.887/0.709 &nbsp;</td><td>0.847/0.644 &nbsp;</td><td>0.824/0.542 &nbsp;</td></tr><tr><td> NW‐AP‐2 &nbsp;</td><td>250 &nbsp;</td><td>0.842/0.746 &nbsp;</td><td>0.833/0.268 &nbsp;</td><td>0.770/0.443 &nbsp;</td><td>0.703/0.311 &nbsp;</td><td>0.851/0.667 &nbsp;</td><td>0.821/0.593 &nbsp;</td><td>0.800/0.487 &nbsp;</td></tr><tr><td> NW‐NS‐2 &nbsp;</td><td>243 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td> FFT‐NS‐2 &nbsp;</td><td>227 &nbsp;</td><td>0.849/0.761 &nbsp;</td><td>0.844/0.334 &nbsp;</td><td>0.779/0.486 &nbsp;</td><td>0.797/0.532 &nbsp;</td><td>0.951/0.826 &nbsp;</td><td>0.845/0.652 &nbsp;</td><td>0.844/0.588 &nbsp;</td></tr><tr><td>Iterative refinement methods and T‐COFFEE&nbsp;</td></tr><tr><td> DIALIGN2‐1 &nbsp;</td><td>18132 &nbsp;</td><td>0.792/0.681 &nbsp;</td><td>0.814/0.219 &nbsp;</td><td>0.673/0.327 &nbsp;</td><td>0.818/0.615 &nbsp;</td><td>0.938/0.840 &nbsp;</td><td>0.801/0.584 &nbsp;</td><td>0.807/0.536 &nbsp;</td></tr><tr><td> PRRP &nbsp;</td><td>9782 &nbsp;</td><td>0.871/0.793 &nbsp;</td><td>0.860/0.354 &nbsp;</td><td>0.823/0.569 &nbsp;</td><td>0.663/0.275 &nbsp;</td><td>0.885/0.742 &nbsp;</td><td>0.845/0.646 &nbsp;</td><td>0.820/0.547 &nbsp;</td></tr><tr><td> T‐COFFEE &nbsp;</td><td>12065 &nbsp;</td><td>0.876/0.797 &nbsp;</td><td>0.856/0.343 &nbsp;</td><td>0.777/0.497 &nbsp;</td><td>0.811/0.555 &nbsp;</td><td>0.961/0.901 &nbsp;</td><td>0.865/0.683 &nbsp;</td><td>0.856/0.619 &nbsp;</td></tr><tr><td> FFT‐NS‐i &nbsp;</td><td>1466 &nbsp;</td><td>0.864/0.787 &nbsp;</td><td>0.853/0.363 &nbsp;</td><td>0.789/0.518 &nbsp;</td><td>0.799/0.534 &nbsp;</td><td>0.956/0.835 &nbsp;</td><td>0.857/0.675 &nbsp;</td><td>0.852/0.607 &nbsp;</td></tr><tr><td>Number of alignments&nbsp;</td><td>&nbsp;</td><td>82&nbsp;</td><td>23&nbsp;</td><td>12&nbsp;</td><td>15&nbsp;</td><td>12&nbsp;</td><td>144&nbsp;</td><td>–&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>Categories
 (Cat.) 1–5 correspond to five different categories of alignments 
described in the text. Two types of scores averaged across alignments of
 each category are shown in each column, separated by a slash (the 
average of sum‐of‐pairs scores/the average of column scores). Average1 
gives the average score across all the 144 alignments. Since the number 
of alignments differs for different categories ranging from 12 to 82, 
another type of average score (Average2), i.e. the score averaged across
 the five categories, was calculated for each method after Notredame <em>et al.</em>
 (9). Total CPU time for computing all 144 alignments is also shown for 
each method. For all cases, default parameters were used, except for 
CLUSTALW, in which both cases default setting (indicated by CLW18d) and 
‘quicktree’ option (CLW18q) were examined. All calculations were 
performed on a Sun Ultra/2 workstation (UltraSPARC 168 MHz with 128 MB 
of memory, Solaris 2.6). The gcc version 2.8.1 compiler was used with 
the optimization option ‘‐O3’, except for DIALIGN, for which the 
pre‐compiled version by the authors was used.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/55547818" target="_blank">View Large</a></div></div>                        <a id="55547819" scrollto-destination="55547819"></a>
<div content-id="GKF436TB2" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKF436TB2" class="table-wrap-title" data-id="GKF436TB2"><div class="label -label"><strong>Table 2.</strong></div><div class="caption"><p>Comparison of several methods using the LSU rRNA sequences</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Sum‐of‐pairs score&nbsp;</td><td>Column score&nbsp;</td></tr><tr><td>72 sequences × 1305–5183 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1998&nbsp;</td><td>0.692&nbsp;</td><td>–&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 600.2&nbsp;</td><td>0.597&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 197.0&nbsp;</td><td>0.796&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 205.2&nbsp;</td><td>0.770&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 73.39&nbsp;</td><td>0.769&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 251.8&nbsp;</td><td>0.781&nbsp;</td><td>–&nbsp;</td></tr><tr><td>59 sequences × 2810–5183 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>35 860&nbsp;</td><td>0.806&nbsp;</td><td>0.559&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1523&nbsp;</td><td>0.754&nbsp;</td><td>0.411&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 395.6&nbsp;</td><td>0.643&nbsp;</td><td>0.315&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 153.7&nbsp;</td><td>0.823&nbsp;</td><td>0.482&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 159.8&nbsp;</td><td>0.793&nbsp;</td><td>0.463&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 51.09&nbsp;</td><td>0.794&nbsp;</td><td>0.468&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 181.7&nbsp;</td><td>0.817&nbsp;</td><td>0.552&nbsp;</td></tr></tbody></table></div><div content-id="GKF436TB2" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Sum‐of‐pairs score&nbsp;</td><td>Column score&nbsp;</td></tr><tr><td>72 sequences × 1305–5183 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1998&nbsp;</td><td>0.692&nbsp;</td><td>–&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 600.2&nbsp;</td><td>0.597&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 197.0&nbsp;</td><td>0.796&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 205.2&nbsp;</td><td>0.770&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 73.39&nbsp;</td><td>0.769&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 251.8&nbsp;</td><td>0.781&nbsp;</td><td>–&nbsp;</td></tr><tr><td>59 sequences × 2810–5183 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>35 860&nbsp;</td><td>0.806&nbsp;</td><td>0.559&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1523&nbsp;</td><td>0.754&nbsp;</td><td>0.411&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 395.6&nbsp;</td><td>0.643&nbsp;</td><td>0.315&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 153.7&nbsp;</td><td>0.823&nbsp;</td><td>0.482&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 159.8&nbsp;</td><td>0.793&nbsp;</td><td>0.463&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 51.09&nbsp;</td><td>0.794&nbsp;</td><td>0.468&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 181.7&nbsp;</td><td>0.817&nbsp;</td><td>0.552&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>The
 CPU time and the sum‐of‐pairs and column scores (8) of each method are 
shown for the full LSU data set (72 sequences) retrieved from RDP‐II 
(33), which includes some fragment sequences, and a subset composed of 
59 sequences without fragment sequences. In the case of the full LSU 
data set, the column score cannot be calculated because of fragment 
sequences, and the test of T‐COFFEE is aborted due to a memory shortage.
 The default parameters were used for all methods, except for CLUSTALW, 
in which default setting (CLW18d) and ‘quicktree’ option (CLW18q) were 
examined. All calculations were performed on a Linux operating system 
(Intel Xeon 2 GHz with 4 GB of memory). The gcc compiler (version 2.96) 
was used with the optimization option ‘‐O3’.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/55547819" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKF436TB2" class="table-wrap-title" data-id="GKF436TB2"><div class="label -label"><strong>Table 2.</strong></div><div class="caption"><p>Comparison of several methods using the LSU rRNA sequences</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Sum‐of‐pairs score&nbsp;</td><td>Column score&nbsp;</td></tr><tr><td>72 sequences × 1305–5183 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1998&nbsp;</td><td>0.692&nbsp;</td><td>–&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 600.2&nbsp;</td><td>0.597&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 197.0&nbsp;</td><td>0.796&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 205.2&nbsp;</td><td>0.770&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 73.39&nbsp;</td><td>0.769&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 251.8&nbsp;</td><td>0.781&nbsp;</td><td>–&nbsp;</td></tr><tr><td>59 sequences × 2810–5183 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>35 860&nbsp;</td><td>0.806&nbsp;</td><td>0.559&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1523&nbsp;</td><td>0.754&nbsp;</td><td>0.411&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 395.6&nbsp;</td><td>0.643&nbsp;</td><td>0.315&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 153.7&nbsp;</td><td>0.823&nbsp;</td><td>0.482&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 159.8&nbsp;</td><td>0.793&nbsp;</td><td>0.463&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 51.09&nbsp;</td><td>0.794&nbsp;</td><td>0.468&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 181.7&nbsp;</td><td>0.817&nbsp;</td><td>0.552&nbsp;</td></tr></tbody></table></div><div content-id="GKF436TB2" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Sum‐of‐pairs score&nbsp;</td><td>Column score&nbsp;</td></tr><tr><td>72 sequences × 1305–5183 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1998&nbsp;</td><td>0.692&nbsp;</td><td>–&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 600.2&nbsp;</td><td>0.597&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 197.0&nbsp;</td><td>0.796&nbsp;</td><td>–&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 205.2&nbsp;</td><td>0.770&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 73.39&nbsp;</td><td>0.769&nbsp;</td><td>–&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 251.8&nbsp;</td><td>0.781&nbsp;</td><td>–&nbsp;</td></tr><tr><td>59 sequences × 2810–5183 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>35 860&nbsp;</td><td>0.806&nbsp;</td><td>0.559&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td> 1523&nbsp;</td><td>0.754&nbsp;</td><td>0.411&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td> 395.6&nbsp;</td><td>0.643&nbsp;</td><td>0.315&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td> 153.7&nbsp;</td><td>0.823&nbsp;</td><td>0.482&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td> 159.8&nbsp;</td><td>0.793&nbsp;</td><td>0.463&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td> 51.09&nbsp;</td><td>0.794&nbsp;</td><td>0.468&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td> 181.7&nbsp;</td><td>0.817&nbsp;</td><td>0.552&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>The
 CPU time and the sum‐of‐pairs and column scores (8) of each method are 
shown for the full LSU data set (72 sequences) retrieved from RDP‐II 
(33), which includes some fragment sequences, and a subset composed of 
59 sequences without fragment sequences. In the case of the full LSU 
data set, the column score cannot be calculated because of fragment 
sequences, and the test of T‐COFFEE is aborted due to a memory shortage.
 The default parameters were used for all methods, except for CLUSTALW, 
in which default setting (CLW18d) and ‘quicktree’ option (CLW18q) were 
examined. All calculations were performed on a Linux operating system 
(Intel Xeon 2 GHz with 4 GB of memory). The gcc compiler (version 2.96) 
was used with the optimization option ‘‐O3’.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/55547819" target="_blank">View Large</a></div></div>                        <a id="55547820" scrollto-destination="55547820"></a>
<div content-id="GKF436TB3" class="table-modal" style="display: block;"><div class="table-wrap"><div id="GKF436TB3" class="table-wrap-title" data-id="GKF436TB3"><div class="label -label"><strong>Table 3.</strong></div><div class="caption"><p>Comparison of several methods using the largest subunit sequences of RNA polymerase</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Number of correctly aligned blocks&nbsp;</td></tr><tr><td>76 sequences × 1182–2890 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>675.5&nbsp;</td><td>10&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>159.4&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>54.95&nbsp;</td><td>8&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>59.30&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>18.15&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>173.1&nbsp;</td><td>11&nbsp;</td></tr><tr><td>24 sequences × 1206–2890 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>745.3&nbsp;</td><td>11&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>100.1&nbsp;</td><td>9&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>50.78&nbsp;</td><td>9&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>20.79&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>22.77&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>7.150&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>46.00&nbsp;</td><td>11&nbsp;</td></tr></tbody></table></div><div content-id="GKF436TB3" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Number of correctly aligned blocks&nbsp;</td></tr><tr><td>76 sequences × 1182–2890 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>675.5&nbsp;</td><td>10&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>159.4&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>54.95&nbsp;</td><td>8&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>59.30&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>18.15&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>173.1&nbsp;</td><td>11&nbsp;</td></tr><tr><td>24 sequences × 1206–2890 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>745.3&nbsp;</td><td>11&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>100.1&nbsp;</td><td>9&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>50.78&nbsp;</td><td>9&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>20.79&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>22.77&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>7.150&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>46.00&nbsp;</td><td>11&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>The
 CPU time and the number of correctly aligned blocks by each method are 
shown for a large data set composed of 76 sequences and a small data set
 composed of 24 sequences. In the case of the large data set, the test 
of T‐COFFEE was aborted due to a memory shortage. For all cases, default
 parameters were used, except for CLUSTALW, in which both cases default 
setting (CLW18d) and ‘quicktree’ option (CLW18q) were examined. All 
calculations were performed on a Linux operating system (Intel Xeon 1.7 
GHz with 1 GB of memory). The gcc compiler (version 2.96) was used with 
the optimization option ‘‐O3’.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/55547820" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="GKF436TB3" class="table-wrap-title" data-id="GKF436TB3"><div class="label -label"><strong>Table 3.</strong></div><div class="caption"><p>Comparison of several methods using the largest subunit sequences of RNA polymerase</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Number of correctly aligned blocks&nbsp;</td></tr><tr><td>76 sequences × 1182–2890 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>675.5&nbsp;</td><td>10&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>159.4&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>54.95&nbsp;</td><td>8&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>59.30&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>18.15&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>173.1&nbsp;</td><td>11&nbsp;</td></tr><tr><td>24 sequences × 1206–2890 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>745.3&nbsp;</td><td>11&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>100.1&nbsp;</td><td>9&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>50.78&nbsp;</td><td>9&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>20.79&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>22.77&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>7.150&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>46.00&nbsp;</td><td>11&nbsp;</td></tr></tbody></table></div><div content-id="GKF436TB3" class="table-modal"><table><tbody><tr><td>Method&nbsp;</td><td>CPU time (s)&nbsp;</td><td>Number of correctly aligned blocks&nbsp;</td></tr><tr><td>76 sequences × 1182–2890 sites&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>675.5&nbsp;</td><td>10&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>159.4&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>54.95&nbsp;</td><td>8&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>59.30&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>18.15&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>173.1&nbsp;</td><td>11&nbsp;</td></tr><tr><td>24 sequences × 1206–2890 sites&nbsp;</td></tr><tr><td> T‐COFFEE&nbsp;</td><td>745.3&nbsp;</td><td>11&nbsp;</td></tr><tr><td> CLW18d&nbsp;</td><td>100.1&nbsp;</td><td>9&nbsp;</td></tr><tr><td> CLW18q&nbsp;</td><td>50.78&nbsp;</td><td>9&nbsp;</td></tr><tr><td> NW‐AP‐2&nbsp;</td><td>20.79&nbsp;</td><td>10&nbsp;</td></tr><tr><td> NW‐NS‐2&nbsp;</td><td>22.77&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐2&nbsp;</td><td>7.150&nbsp;</td><td>11&nbsp;</td></tr><tr><td> FFT‐NS‐i&nbsp;</td><td>46.00&nbsp;</td><td>11&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><p>The
 CPU time and the number of correctly aligned blocks by each method are 
shown for a large data set composed of 76 sequences and a small data set
 composed of 24 sequences. In the case of the large data set, the test 
of T‐COFFEE was aborted due to a memory shortage. For all cases, default
 parameters were used, except for CLUSTALW, in which both cases default 
setting (CLW18d) and ‘quicktree’ option (CLW18q) were examined. All 
calculations were performed on a Linux operating system (Intel Xeon 1.7 
GHz with 1 GB of memory). The gcc compiler (version 2.96) was used with 
the optimization option ‘‐O3’.</p></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="https://academic.oup.com/view-large/55547820" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination="55547821" id="55547821" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="GKF436C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C1" class="ref-content "><div class="label -label">1.</div><div class="citation mixed-citation">Needleman,S.B. and Wunsch,C.D. (<div class="year">1970</div>) A general method applicable to the search for similarities in the amino acid sequence of two proteins. <div class="source">J. Mol. Biol.</div>&nbsp;, <div class="volume">48</div>, <div class="fpage">443</div>–453.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Needleman,S.B.+and+Wunsch,C.D.+%281970%29+A+general+method+applicable+to+the+search+for+similarities+in+the+amino+acid+sequence+of+two+proteins.+J.+Mol.+Biol.,+48,+443%E2%80%93453." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C2" class="ref-content "><div class="label -label">2.</div><div class="citation mixed-citation">Sankoff,D. and Cedergren,R.J. (<div class="year">1983</div>) Simultaneous comparison of three or more sequences related by a tree. In Sankoff,D. and Kruskal,J.B. (eds), <em>Time Warps, String Edits and Macromolecules: The Theory and Practice of Sequence Comparison.</em> Addison‐Wesley, London, UK, pp. <div class="fpage">253</div>–264.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sankoff,D.+and+Cedergren,R.J.+%281983%29+Simultaneous+comparison+of+three+or+more+sequences+related+by+a+tree.+In+Sankoff,D.+and+Kruskal,J.B.+%28eds%29,+Time+Warps,+String+Edits+and+Macromolecules:+The+Theory+and+Practice+of+Sequence+Comparison.+Addison%E2%80%90Wesley,+London,+UK,+pp.+253%E2%80%93264." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C3" class="ref-content "><div class="label -label">3.</div><div class="citation mixed-citation">Feng,D.F. and Doolittle,R.F. (<div class="year">1987</div>) Progressive sequence alignment as a prerequisite to correct phylogenetic trees. <div class="source">J. Mol. Evol.</div>&nbsp;, <div class="volume">25</div>, <div class="fpage">351</div>–360.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Feng,D.F.+and+Doolittle,R.F.+%281987%29+Progressive+sequence+alignment+as+a+prerequisite+to+correct+phylogenetic+trees.+J.+Mol.+Evol.,+25,+351%E2%80%93360." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C4" class="ref-content "><div class="label -label">4.</div><div class="citation mixed-citation">Barton,G.J. and Sternberg,M.J. (<div class="year">1987</div>) A strategy for the rapid multiple alignment of protein sequences. Confidence levels from tertiary structure comparisons. <div class="source">J. Mol. Biol.</div>&nbsp;, <div class="volume">198</div>, <div class="fpage">327</div>–337.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Barton,G.J.+and+Sternberg,M.J.+%281987%29+A+strategy+for+the+rapid+multiple+alignment+of+protein+sequences.+Confidence+levels+from+tertiary+structure+comparisons.+J.+Mol.+Biol.,+198,+327%E2%80%93337." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C5" class="ref-content "><div class="label -label">5.</div><div class="citation mixed-citation">Berger,M.P. and Munson,P.J. (<div class="year">1991</div>) A novel randomized iterative strategy for aligning multiple protein sequences. <div class="source">Comput. Appl. Biosci.</div>&nbsp;, <div class="volume">7</div>, <div class="fpage">479</div>–484.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Berger,M.P.+and+Munson,P.J.+%281991%29+A+novel+randomized+iterative+strategy+for+aligning+multiple+protein+sequences.+Comput.+Appl.+Biosci.,+7,+479%E2%80%93484." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C6" class="ref-content "><div class="label -label">6.</div><div class="citation mixed-citation">Gotoh,O. (<div class="year">1993</div>) Optimal alignment between groups of sequences and its application to multiple sequence alignment. <div class="source">Comput. Appl. Biosci.</div>&nbsp;, <div class="volume">9</div>, <div class="fpage">361</div>–370.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Gotoh,O.+%281993%29+Optimal+alignment+between+groups+of+sequences+and+its+application+to+multiple+sequence+alignment.+Comput.+Appl.+Biosci.,+9,+361%E2%80%93370." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C7" class="ref-content "><div class="label -label">7.</div><div class="citation mixed-citation">Thompson,J.D., Higgins,D.G. and Gibson,T.J. (<div class="year">1994</div>)
 CLUSTAL W: improving the sensitivity of progressive multiple sequence 
alignment through sequence weighting, position‐specific gap penalties 
and weight matrix choice. <div class="source">Nucleic Acids Res.</div>&nbsp;, <div class="volume">22</div>, <div class="fpage">4673</div>–4680.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Thompson,J.D.,+Higgins,D.G.+and+Gibson,T.J.+%281994%29+CLUSTAL+W:+improving+the+sensitivity+of+progressive+multiple+sequence+alignment+through+sequence+weighting,+position%E2%80%90specific+gap+penalties+and+weight+matrix+choice.+Nucleic+Acids+Res.,+22,+4673%E2%80%934680." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C8" class="ref-content "><div class="label -label">8.</div><div class="citation mixed-citation">Thompson,J.D., Plewniak,F. and Poch,O. (<div class="year">1999</div>) A comprehensive comparison of multiple sequence alignment programs. <div class="source">Nucleic Acids Res.</div>&nbsp;, <div class="volume">27</div>, <div class="fpage">2682</div>–2690.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Thompson,J.D.,+Plewniak,F.+and+Poch,O.+%281999%29+A+comprehensive+comparison+of+multiple+sequence+alignment+programs.+Nucleic+Acids+Res.,+27,+2682%E2%80%932690." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C9" class="ref-content "><div class="label -label">9.</div><div class="citation mixed-citation">Notredame,C., Higgins,D.G. and Heringa,J. (<div class="year">2000</div>) T‐Coffee: a novel method for fast and accurate multiple sequence alignment. <div class="source">J. Mol. Biol.</div>&nbsp;, <div class="volume">302</div>, <div class="fpage">205</div>–217.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Notredame,C.,+Higgins,D.G.+and+Heringa,J.+%282000%29+T%E2%80%90Coffee:+a+novel+method+for+fast+and+accurate+multiple+sequence+alignment.+J.+Mol.+Biol.,+302,+205%E2%80%93217." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C10" class="ref-content "><div class="label -label">10.</div><div class="citation mixed-citation">Delcher,A.L., Kasif,S., Fleischmann,R.D., Peterson,J., White,O. and Salzberg,S.L. (<div class="year">1999</div>) Alignment of whole genomes. <div class="source">Nucleic Acids Res.</div>&nbsp;, <div class="volume">27</div>, <div class="fpage">2369</div>–2376.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Delcher,A.L.,+Kasif,S.,+Fleischmann,R.D.,+Peterson,J.,+White,O.+and+Salzberg,S.L.+%281999%29+Alignment+of+whole+genomes.+Nucleic+Acids+Res.,+27,+2369%E2%80%932376." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C11" class="ref-content "><div class="label -label">11.</div><div class="citation mixed-citation">Pearson,W.R. and Lipman,D.J. (<div class="year">1988</div>) Improved tools for biological sequence comparison. <div class="source">Proc. Natl Acad. Sci. USA</div>&nbsp;, <div class="volume">85</div>, <div class="fpage">2444</div>–2448.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Pearson,W.R.+and+Lipman,D.J.+%281988%29+Improved+tools+for+biological+sequence+comparison.+Proc.+Natl+Acad.+Sci.+USA,+85,+2444%E2%80%932448." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C12" class="ref-content "><div class="label -label">12.</div><div class="citation mixed-citation">Altschul,S.F., Madden,T.L., Schaffer,A.A., Zhang,J., Zhang,Z., Miller,W. and Lipman,D.J. (<div class="year">1997</div>) Gapped BLAST and PSI‐BLAST: a new generation of protein database search programs. <div class="source">Nucleic Acids Res.</div>&nbsp;, <div class="volume">25</div>, <div class="fpage">3389</div>–3402.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Altschul,S.F.,+Madden,T.L.,+Schaffer,A.A.,+Zhang,J.,+Zhang,Z.,+Miller,W.+and+Lipman,D.J.+%281997%29+Gapped+BLAST+and+PSI%E2%80%90BLAST:+a+new+generation+of+protein+database+search+programs.+Nucleic+Acids+Res.,+25,+3389%E2%80%933402." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C13" class="ref-content "><div class="label -label">13.</div><div class="citation mixed-citation">Felsenstein,J., Sawyer,S. and Kochin,R. (<div class="year">1982</div>) An efficient method for matching nucleic acid sequences. <div class="source">Nucleic Acids Res.</div>&nbsp;, <div class="volume">10</div>, <div class="fpage">133</div>–139.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Felsenstein,J.,+Sawyer,S.+and+Kochin,R.+%281982%29+An+efficient+method+for+matching+nucleic+acid+sequences.+Nucleic+Acids+Res.,+10,+133%E2%80%93139." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C14" class="ref-content "><div class="label -label">14.</div><div class="citation mixed-citation">Rajasekaran,S., Jin,X. and Spouge,J.L. (<div class="year">2002</div>) The efficient computation of position‐specific match scores with the fast Fourier transform. <div class="source">J. Comput. Biol.</div>&nbsp;, <div class="volume">9</div>, <div class="fpage">23</div>–33.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Rajasekaran,S.,+Jin,X.+and+Spouge,J.L.+%282002%29+The+efficient+computation+of+position%E2%80%90specific+match+scores+with+the+fast+Fourier+transform.+J.+Comput.+Biol.,+9,+23%E2%80%9333." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C15" class="ref-content "><div class="label -label">15.</div><div class="citation mixed-citation">Thompson,J.D., Plewniak,F. and Poch,O. (<div class="year">1999</div>) BAliBASE: a benchmark alignment database for the evaluation of multiple alignment programs. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">15</div>, <div class="fpage">87</div>–88.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Thompson,J.D.,+Plewniak,F.+and+Poch,O.+%281999%29+BAliBASE:+a+benchmark+alignment+database+for+the+evaluation+of+multiple+alignment+programs.+Bioinformatics,+15,+87%E2%80%9388." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C16" class="ref-content "><div class="label -label">16.</div><div class="citation mixed-citation">Miyata,T., Miyazawa,S. and Yasunaga,T. (<div class="year">1979</div>) Two types of amino acid substitutions in protein evolution. <div class="source">J. Mol. Evol.</div>&nbsp;, <div class="volume">12</div>, <div class="fpage">219</div>–236.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Miyata,T.,+Miyazawa,S.+and+Yasunaga,T.+%281979%29+Two+types+of+amino+acid+substitutions+in+protein+evolution.+J.+Mol.+Evol.,+12,+219%E2%80%93236." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C17" class="ref-content "><div class="label -label">17.</div><div class="citation mixed-citation">Kimura,M. (<div class="year">1983</div>) <em>The Neutral Theory of Molecular Evolution</em>. Cambridge University Press, Cambridge, UK.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Kimura,M.+%281983%29+The+Neutral+Theory+of+Molecular+Evolution.+Cambridge+University+Press,+Cambridge,+UK." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C18" class="ref-content "><div class="label -label">18.</div><div class="citation mixed-citation">Grantham,R. (<div class="year">1974</div>) Amino acid difference formula to help explain protein evolution. <div class="source">Science</div>&nbsp;, <div class="volume">185</div>, <div class="fpage">862</div>–864.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Grantham,R.+%281974%29+Amino+acid+difference+formula+to+help+explain+protein+evolution.+Science,+185,+862%E2%80%93864." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C19" class="ref-content "><div class="label -label">19.</div><div class="citation mixed-citation">Press,W.H., Teukolsky,S.A., Vetterling,W.T. and Flannery,B.P. (<div class="year">1995</div>) <em>Numerical Recipes in C: The Art of Scientific Computing</em>, 2nd Edn. Cambridge University Press, Cambridge, UK.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Press,W.H.,+Teukolsky,S.A.,+Vetterling,W.T.+and+Flannery,B.P.+%281995%29+Numerical+Recipes+in+C:+The+Art+of+Scientific+Computing,+2nd+Edn.+Cambridge+University+Press,+Cambridge,+UK." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C20" class="ref-content "><div class="label -label">20.</div><div class="citation mixed-citation">Gotoh,O. (<div class="year">1995</div>) A weighting system and algorithm for aligning many phylogenetically related sequences. <div class="source">Comput. Appl. Biosci.</div>&nbsp;, <div class="volume">11</div>, <div class="fpage">543</div>–551.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Gotoh,O.+%281995%29+A+weighting+system+and+algorithm+for+aligning+many+phylogenetically+related+sequences.+Comput.+Appl.+Biosci.,+11,+543%E2%80%93551." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C21" class="ref-content "><div class="label -label">21.</div><div class="citation mixed-citation">Vogt,G., Etzold,T. and Argos,P. (<div class="year">1995</div>) An assessment of amino acid exchange matrices in aligning protein sequences: the twilight zone revisited. <div class="source">J. Mol. Biol.</div>&nbsp;, <div class="volume">249</div>, <div class="fpage">816</div>–831.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Vogt,G.,+Etzold,T.+and+Argos,P.+%281995%29+An+assessment+of+amino+acid+exchange+matrices+in+aligning+protein+sequences:+the+twilight+zone+revisited.+J.+Mol.+Biol.,+249,+816%E2%80%93831." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C22" class="ref-content "><div class="label -label">22.</div><div class="citation mixed-citation">Jones,D.T., Taylor,W.R. and Thornton,J.M. (<div class="year">1992</div>) The rapid generation of mutation data matrices from protein sequences. <div class="source">Comput. Appl. Biosci.</div>&nbsp;, <div class="volume">8</div>, <div class="fpage">275</div>–282.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Jones,D.T.,+Taylor,W.R.+and+Thornton,J.M.+%281992%29+The+rapid+generation+of+mutation+data+matrices+from+protein+sequences.+Comput.+Appl.+Biosci.,+8,+275%E2%80%93282." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C23" class="ref-content "><div class="label -label">23.</div><div class="citation mixed-citation">Kimura,M. (<div class="year">1980</div>) A simple method for estimating evolutionary rates of base substitutions through comparative studies of nucleotide sequences. <div class="source">J. Mol. Evol.</div>&nbsp;, <div class="volume">16</div>, <div class="fpage">111</div>–120.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Kimura,M.+%281980%29+A+simple+method+for+estimating+evolutionary+rates+of+base+substitutions+through+comparative+studies+of+nucleotide+sequences.+J.+Mol.+Evol.,+16,+111%E2%80%93120." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C24" class="ref-content "><div class="label -label">24.</div><div class="citation mixed-citation">Dayhoff,M.O., Schwartz,R.M. and Orcutt,B.C. (<div class="year">1978</div>) A model of evolutionary change in proteins. In Dayhoff,M.O. and Ech,R.V. (eds), <em>Atlas of Protein Sequence and Structure.</em> National Biomedical Research Foundation, MD, pp. <div class="fpage">345</div>–352.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Dayhoff,M.O.,+Schwartz,R.M.+and+Orcutt,B.C.+%281978%29+A+model+of+evolutionary+change+in+proteins.+In+Dayhoff,M.O.+and+Ech,R.V.+%28eds%29,+Atlas+of+Protein+Sequence+and+Structure.+National+Biomedical+Research+Foundation,+MD,+pp.+345%E2%80%93352." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C25" class="ref-content "><div class="label -label">25.</div><div class="citation mixed-citation">Sokal,R.R. and Michener,C.D. (<div class="year">1958</div>) A statistical mehod for evaluating systematic relationships. <div class="source">University of Kansas Scientific Bulletin</div>&nbsp;, <div class="volume">28</div>, <div class="fpage">1409</div>–1438.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Sokal,R.R.+and+Michener,C.D.+%281958%29+A+statistical+mehod+for+evaluating+systematic+relationships.+University+of+Kansas+Scientific+Bulletin,+28,+1409%E2%80%931438." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C26" class="ref-content "><div class="label -label">26.</div><div class="citation mixed-citation">Tateno,Y.,
 Ikeo,K., Imanishi,T., Watanabe,H., Endo,T., Yamaguchi,Y., Suzuki,Y., 
Takahashi,K., Tsunoyama,K., Kawai,M., Kawanishi,Y., Naitou,K. and 
Gojobori,T. (<div class="year">1997</div>) Evolutionary motif and its biological and structural significance. <div class="source">J. Mol. Evol.</div>&nbsp;, <div class="volume">44</div> (Suppl. 1), <div class="fpage">S38</div>–S43.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Tateno,Y.,+Ikeo,K.,+Imanishi,T.,+Watanabe,H.,+Endo,T.,+Yamaguchi,Y.,+Suzuki,Y.,+Takahashi,K.,+Tsunoyama,K.,+Kawai,M.,+Kawanishi,Y.,+Naitou,K.+and+Gojobori,T.+%281997%29+Evolutionary+motif+and+its+biological+and+structural+significance.+J.+Mol.+Evol.,+44+%28Suppl.+1%29,+S38%E2%80%93S43." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C27" class="ref-content "><div class="label -label">27.</div><div class="citation mixed-citation">Hirosawa,M., Totoki,Y., Hoshida,M. and Ishikawa,M. (<div class="year">1995</div>) Comprehensive study on iterative algorithms of multiple sequence alignment. <div class="source">Comput. Appl. Biosci.</div>&nbsp;, <div class="volume">11</div>, <div class="fpage">13</div>–18.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Hirosawa,M.,+Totoki,Y.,+Hoshida,M.+and+Ishikawa,M.+%281995%29+Comprehensive+study+on+iterative+algorithms+of+multiple+sequence+alignment.+Comput.+Appl.+Biosci.,+11,+13%E2%80%9318." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C28" class="ref-content "><div class="label -label">28.</div><div class="citation mixed-citation">Stoye,J., Evers,D. and Meyer,F. (<div class="year">1997</div>) Generating benchmarks for multiple sequence alignments and phylogenetic reconstructions. <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol.</div>&nbsp;, <div class="volume">5</div>, <div class="fpage">303</div>–306.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Stoye,J.,+Evers,D.+and+Meyer,F.+%281997%29+Generating+benchmarks+for+multiple+sequence+alignments+and+phylogenetic+reconstructions.+Proc.+Int.+Conf.+Intell.+Syst.+Mol.+Biol.,+5,+303%E2%80%93306." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C29" class="ref-content "><div class="label -label">29.</div><div class="citation mixed-citation">Morgenstern,B., Dress,A. and Werner,T. (<div class="year">1996</div>) Multiple DNA and protein sequence alignment based on segment‐to‐segment comparison. <div class="source">Proc. Natl Acad. Sci. USA</div>&nbsp;, <div class="volume">93</div>, <div class="fpage">12098</div>–12103.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Morgenstern,B.,+Dress,A.+and+Werner,T.+%281996%29+Multiple+DNA+and+protein+sequence+alignment+based+on+segment%E2%80%90to%E2%80%90segment+comparison.+Proc.+Natl+Acad.+Sci.+USA,+93,+12098%E2%80%9312103." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C30" class="ref-content "><div class="label -label">30.</div><div class="citation mixed-citation">Morgenstern,B. (<div class="year">1999</div>) DIALIGN2: improvement of the segment‐to‐segment approach to multiple sequence alignment. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">15</div>, <div class="fpage">211</div>–218.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Morgenstern,B.+%281999%29+DIALIGN2:+improvement+of+the+segment%E2%80%90to%E2%80%90segment+approach+to+multiple+sequence+alignment.+Bioinformatics,+15,+211%E2%80%93218." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C31" class="ref-content "><div class="label -label">31.</div><div class="citation mixed-citation">Smith,R.F. and Smith,T.F. (<div class="year">1992</div>)
 Pattern‐induced multi‐sequence alignment (PIMA) algorithm employing 
secondary structure‐dependent gap penalties for use in comparative 
protein modelling. <div class="source">Protein Eng.</div>&nbsp;, <div class="volume">5</div>, <div class="fpage">35</div>–41.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Smith,R.F.+and+Smith,T.F.+%281992%29+Pattern%E2%80%90induced+multi%E2%80%90sequence+alignment+%28PIMA%29+algorithm+employing+secondary+structure%E2%80%90dependent+gap+penalties+for+use+in+comparative+protein+modelling.+Protein+Eng.,+5,+35%E2%80%9341." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C32" class="ref-content "><div class="label -label">32.</div><div class="citation mixed-citation">Gotoh,O. (<div class="year">1996</div>)
 Significant improvement in accuracy of multiple protein sequence 
alignments by iterative refinement as assessed by reference to 
structural alignments. <div class="source">J. Mol. Biol.</div>&nbsp;, <div class="volume">264</div>, <div class="fpage">823</div>–838.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Gotoh,O.+%281996%29+Significant+improvement+in+accuracy+of+multiple+protein+sequence+alignments+by+iterative+refinement+as+assessed+by+reference+to+structural+alignments.+J.+Mol.+Biol.,+264,+823%E2%80%93838." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C33" class="ref-content "><div class="label -label">33.</div><div class="citation mixed-citation">Maidak,B.L.,
 Cole,J.R., Lilburn,T.G., Parker,C.T.,Jr, Saxman,P.R., Farris,R.J., 
Garrity,G.M., Olsen,G.J., Schmidt,T.M. and Tiedje,J.M. (<div class="year">2001</div>) The RDP‐II (ribosomal database project). <div class="source">Nucleic Acids Res.</div>&nbsp;, <div class="volume">29</div>, <div class="fpage">173</div>–174.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Maidak,B.L.,+Cole,J.R.,+Lilburn,T.G.,+Parker,C.T.,Jr,+Saxman,P.R.,+Farris,R.J.,+Garrity,G.M.,+Olsen,G.J.,+Schmidt,T.M.+and+Tiedje,J.M.+%282001%29+The+RDP%E2%80%90II+%28ribosomal+database+project%29.+Nucleic+Acids+Res.,+29,+173%E2%80%93174." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C34" class="ref-content "><div class="label -label">34.</div><div class="citation mixed-citation">Iwabe,N., Kuma,K., Kishino,H., Hasegawa,M. and Miyata,T. (<div class="year">1991</div>) Evolution of RNA polymerases and branching patterns of the three major groups of archaebacteria. <div class="source">J. Mol. Evol.</div>&nbsp;, <div class="volume">32</div>, <div class="fpage">70</div>–78.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Iwabe,N.,+Kuma,K.,+Kishino,H.,+Hasegawa,M.+and+Miyata,T.+%281991%29+Evolution+of+RNA+polymerases+and+branching+patterns+of+the+three+major+groups+of+archaebacteria.+J.+Mol.+Evol.,+32,+70%E2%80%9378." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C35" class="ref-content "><div class="label -label">35.</div><div class="citation mixed-citation">McClure,M.A., Vasi,T.K. and Fitch,W.M. (<div class="year">1994</div>) Comparative analysis of multiple protein‐sequence alignment methods. <div class="source">Mol. Biol. Evol.</div>&nbsp;, <div class="volume">11</div>, <div class="fpage">571</div>–592.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=McClure,M.A.,+Vasi,T.K.+and+Fitch,W.M.+%281994%29+Comparative+analysis+of+multiple+protein%E2%80%90sequence+alignment+methods.+Mol.+Biol.+Evol.,+11,+571%E2%80%93592." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C36" class="ref-content "><div class="label -label">36.</div><div class="citation mixed-citation">Smith,T.F. and Waterman,M.S. (<div class="year">1981</div>) Identification of common molecular subsequences. <div class="source">J. Mol. Biol.</div>&nbsp;, <div class="volume">147</div>, <div class="fpage">195</div>–197.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Smith,T.F.+and+Waterman,M.S.+%281981%29+Identification+of+common+molecular+subsequences.+J.+Mol.+Biol.,+147,+195%E2%80%93197." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C37" class="ref-content "><div class="label -label">37.</div><div class="citation mixed-citation">Karplus,K. and Hu,B. (<div class="year">2001</div>) Evaluation of protein multiple alignments by SAM‐T99 using the BAliBASE multiple alignment test set. <div class="source">Bioinformatics</div>&nbsp;, <div class="volume">17</div>, <div class="fpage">713</div>–720.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Karplus,K.+and+Hu,B.+%282001%29+Evaluation+of+protein+multiple+alignments+by+SAM%E2%80%90T99+using+the+BAliBASE+multiple+alignment+test+set.+Bioinformatics,+17,+713%E2%80%93720." target="">Google Scholar</a></div></div></div></div></div></div><div content-id="GKF436C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="GKF436C38" class="ref-content "><div class="label -label">38.</div><div class="citation mixed-citation">Lecompte,O., Thompson,J.D., Plewniak,F., Thierry,J. and Poch,O. (<div class="year">2001</div>) Multiple alignment of complete sequences (MACS) in the post‐genomic era. <div class="source">Gene</div>&nbsp;, <div class="volume">270</div>, <div class="fpage">17</div>–30.<div class="citation-links"><div class="google-scholar-ref-link"><a class="google-scholar-link" href="https://scholar.google.com/scholar?q=Lecompte,O.,+Thompson,J.D.,+Plewniak,F.,+Thierry,J.+and+Poch,O.+%282001%29+Multiple+alignment+of+complete+sequences+%28MACS%29+in+the+post%E2%80%90genomic+era.+Gene,+270,+17%E2%80%9330." target="">Google Scholar</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix" style="display: none;"></div>

        
<!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-taxonomies">
        </div>
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="https://academic.oup.com/nar/search-results?f_TocHeadingTitle=Article">Article</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt" style="display: none;">
                            <a href="https://oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2904316&amp;xsltPath=%7E/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

<div class="comment-body__wrap">
    <div id="usercomments" class="comment-body__title">
        <h2>Comments</h2>
    </div>
    <div class="comment-body__number">
        0 Comments
    </div>
    <div class="submitCommentPlaceHolder"><!-- UserComment.cshtml placed here by JS --><div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
            <button id="comment-modal-opener" class="btn comment-body__button" data-reveal="">Add comment</button>
    <div id="divCommentModal" class="comment-form__modal medium reveal-modal" data-reveal="">

        <button class="comment-form__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close comment form modal</span>
        </button>

        <div class="preloader-wrap hide">
            <div class="preloader">
                <div class="spinner"></div>
            </div>
        </div>

        <div id="TermsAndConditionsContainer" class="comment-form__wrap" data-reveal=""></div>

            <div id="termsAgreeRow" class="comment-form__row--last terms-conditions">
                <input id="termsAndConditionsCheckbox" class="comment-form__input comment-form__checkbox" title="Terms and Conditions" required="required" type="checkbox">
                <label class="comment-form__required-statement required" for="termsAndConditionsCheckbox">I agree to the terms and conditions.</label>
                <label class="lblTermsAndConditions errors comment-form__error-label" for="termsAndConditionsCheckbox">
                    You must accept the terms and conditions.
                </label>
            </div>

        <div id="termsButtonRow" class="termsButtonRow comment-form__row">
            <button id="btnUserAgreeTerms" class="btnUserAgreeTerms btn btn-grey">Add comment</button>
            <button href="#" class="btnModalCancel btn btn-grey">Cancel</button>
        </div>


        <div id="divUserCommentForm" class="comment-form__wrap">

            <div class="comment-form__title">Submit a comment</div>



                <div class="comment-form__row">
                    <label class="comment-form__input-label required" for="Author">Name</label>

                    <input class="comment-form__input comment-form__textbox required" dataauthorvalue="" id="Author" maxlength="2000" name="Author" required="required" type="text">
                </div>


                <div class="comment-form__row">
                    <label class="comment-form__input-label required" for="AffInstitution">Affiliations</label>
                    <input class="comment-form__input comment-form__textbox required" id="AffInstitution" maxlength="200" name="AffInstitution" required="required" type="text">
                </div>



            <div class="comment-form__row">
                <label class="comment-form__input-label required" for="Title">Comment title</label>
                <input class="comment-form__input comment-form__textbox required" id="Title" maxlength="250" name="Title" required="required" type="text">
            </div>

            <div class="comment-form__row">
                <label class="comment-form__input-label required" for="Comment">Comment</label>
                <textarea class="comment-form__input comment-form__textarea required" cols="20" id="Comment" maxlength="@Model.Config.CommentBodyCharLimit" name="Comment" required="required" rows="2"></textarea>
                </div>


                <div class="captchaAlert comment-form__error-label">
                    You have entered an invalid code
                </div>

                <div class="submitRow comment-form__row">
                    <button id="btnUserCommentSubmit" class="btnUserCommentSubmit btn btn-grey">Submit</button>
                    <button href="#" class="btnModalCancel btn btn-grey">Cancel</button>
                </div>

            </div><!-- /.comment-form__wrap -->

            <div class="content-form__response comment-form__row">
                
                <div class="content-form__response--success">
                    <div class="alertmsg content-form__response-text" id="displayAlert">
                        Thank you for submitting a comment on this 
article. Your comment will be reviewed and published at the journal's 
discretion. Please check for further notifications by email.
                    </div>
                    <button class="btnCommentModalExit content-form__response-button btn btn-grey">Close</button>
                </div>
                
                            
                <div id="divError" class="errors content-form__response--error"></div>
            </div><!-- /.comment-form__response -->

        </div><!-- /.comment-form__modal -->
                        <script type="text/javascript">

                            /*******************************************************************************
                             * JS here is only being used to assign variables from values in the model
                             * logic should be implemented in external JS files, like client.script.js
                             *******************************************************************************/

                            var SCM = SCM || {};
                            SCM.UserCommentView = SCM.UserCommentView || {};

                            SCM.UserCommentView.articleId = '2904316';
                            SCM.UserCommentView.isUserLoggedIn = 'False';
                            SCM.UserCommentView.divId = 'WidgetNoAccessModalSmallOUP_UserComment_Article';
                            SCM.UserCommentView.bShowDisclosure = 'false';
                            SCM.UserCommentView.isNotModerated = 'false';
                            SCM.UserCommentView.showAffiliations = 'true';
                            SCM.UserCommentView.InstanceName = 'OUP_UserComment_Article';
                        </script>

    </div></div>
    
    
</div><!-- /.comment-body__wrap -->

    </div>
    

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad2_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad2_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad2_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">13,810</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000177154300016&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">4,623</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/nar/gkf436" data-uuid="898d4697-c62b-4681-597d-4c25670fe2a0"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=2529748" style="display:inline-block;">
        <img alt="Article has an altmetric score of 14" src="https://badges.altmetric.com/?size=128&amp;score=14&amp;types=aaaaaatw" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2904316"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">×</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                <div class="userAlert alertType-33">
                    <a role="link" aria-label="Subject alert" data-userloggedin="False" data-alerttype="33">Subject alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
                <h3 class="article-widget-head">More on this topic</h3>
    <div class="widget-related-content">
            <div class="article-title"><a href="https://academic.oup.com/nar/article/27/13/2682/2376831">A comprehensive comparison of multiple sequence alignment programs</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/37/11/e83/1095196">MM-align: a quick algorithm for aligning multiple-chain protein complex structures using iterative dynamic programming</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/42/13/e106/1276564">RAMICS: trainable, high-speed and biologically relevant alignment of high-throughput sequencing reads to coding DNA</a></div>
            <div class="article-title"><a href="https://academic.oup.com/nar/article/45/D1/D170/2605730">Uniclust databases of clustered and deeply annotated protein sequences and alignments</a></div>
    </div>

    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000177154300016&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fnar%2farticle%2f30%2f14%2f3059%2f2904316">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-RelatedPubMed widget-instance-Article_RightRailB0Article_RightRail_RelatedPubMed">
        <h3 class="article-widget-head">Related articles in PubMed</h3>
<div class="widget-related-pubmed">
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30274016" id="article_ArticleTitle" target="">Performance
 of LC/MS/MS in Analyzing Multiple Amino Acids and Acylcarnitines in 
Dried Blood Spot of Very Low Birth Weight, Low Birth Weight, and Normal 
Weight Neonates.</a></div>
                <!-- <div class="citation"><span>Clin Lab</span> 2018;64(9):1333-1339.</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30273774" id="article_ArticleTitle" target="">Growth
 hormone and Insulin-like growth factor-I (IGF-I) modulate the 
expression of L-type amino acid transporters in the muscles of 
spontaneous dwarf rats and L6 and C2C12 myocytes.</a></div>
                <!-- <div class="citation"><span>Growth Horm IGF Res</span> 2018;</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30273738" id="article_ArticleTitle" target="">The
 effects of crocetin supplementation on the blastocyst outcome, 
transcriptomic and metabolic profile of inÂ&nbsp;vitro produced bovine 
embryos.</a></div>
                <!-- <div class="citation"><span>Theriogenology</span> 2018;</div> -->
            </div>
        </div>
        <div class="inner-content-wrap">
            <div class="inner-content">
                <div class="article-title">    <a href="http://www.ncbi.nlm.nih.gov/pubmed/30273658" id="article_ArticleTitle" target="">Map Challenge: Analysis using a Pair Comparison Method based on Fourier Shell Correlation.</a></div>
                <!-- <div class="citation"><span>J Struct Biol</span> 2018.</div> -->
            </div>
        </div>
</div>
<!-- <div class="article-widget-foot">
    <div class="view-more">
            <a href="http://www.ncbi.nlm.nih.gov/pubmed?term=((sequence+AND+alignment)+OR+(fourier+AND+analysis)+OR+(amino+AND+acids)+OR+(amino+AND+acid+AND+sequence))" id="Config_ViewMoreLink" class="view-more-link" target="_blank">View More</a>
    </div>
</div> -->
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000177154300016&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (4623)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F30%2F14%2F3059%2F2904316" target="">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href="https://academic.oup.com/nar/CrossRef-CitedBy/2904316">CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky934/5133625">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky934/5133625">Structural
 basis for recognition and repair of the 3′-phosphate by NExo, a base 
excision DNA repair nuclease from  Neisseria meningitidis</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky934/5133625">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky912/5133664">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky912/5133664">Substrate specificity of human apurinic/apyrimidinic endonuclease APE1 in the nucleotide incision repair pathway</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky912/5133664">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky917/5133667">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky917/5133667">CLIPick: a sensitive peak caller for expression-based deconvolution of HITS-CLIP signals</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky917/5133667">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky921/5133660">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky921/5133660">Modulation of  MKNK2  alternative splicing by splice-switching oligonucleotides as a novel approach for glioblastoma treatment</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky921/5133660">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/nar/advance-article/doi/10.1093/nar/gky924/5133661">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky924/5133661">Structural,
 functional and biological insights into the role of  Mycobacterium 
tuberculosis  VapBC11 toxin–antitoxin system: targeting a tRNase to 
tackle mycobacterial adaptation</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky924/5133661">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/nar_Behind_Ad3_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad3_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad3_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="300" frameborder="0" height="250"></iframe></div></div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" id="hfArticleTitle" value="MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform" type="hidden">

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/nar/gkf436");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script><div id="google_ads_iframe_/116097782/nar_Behind_Ad4_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/116097782/nar_Behind_Ad4_0" title="3rd party ad content" name="google_ads_iframe_/116097782/nar_Behind_Ad4_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="https://academic.oup.com/nar/pages/About">About Nucleic Acids Research</a>
                            </li>
                            <li class="link-2">
                                <a href="https://academic.oup.com/nar/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="https://academic.oup.com/nar/pages/Policies">Policies</a>
                            </li>
                            <li class="link-4">
                                <a href="https://academic.oup.com/nar/pages/General_Instructions">Author Guidelines</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/nar_open">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="https://www.linkedin.com/in/nar-nucleic-acids-research-5297133b">LinkedIn</a>
                            </li>
                            <li class="link-3">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-4">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?keywords=&amp;t732=470072&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="" target="">
                <img id="footer-logo-NucleicAcidsResearch" class="journal-footer-affiliations-logo" src="nar_f1248941275.svg" alt="Nucleic Acids Research">
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1362-4962</li>
                                            <li>Print ISSN 0305-1048</li>
                    <li>Copyright © 2018 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal="">
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5127&amp;returnUrl=%2fnar%2farticle%2f30%2f14%2f3059%2f2904316">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="footerJS" type="text/javascript"></script>
<script src="OupCookiePolicyJS" type="text/javascript"></script>
<script src="MathJax.js" type="text/javascript"></script>
<script src="chartist.js" type="text/javascript"></script>
<script src="ArticlePageJS" type="text/javascript"></script>
<script src="addthis_widget.js" type="text/javascript"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh886" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.e4e8af4de595fdb10ec1459d.html#rand=0.8296555419636504&amp;iit=1539806902744&amp;tmr=load%3D1539806902710%26core%3D1539806902738%26main%3D1539806902740%26ifr%3D1539806902746&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F30%2F14%2F3059%2F2904316&amp;href=https%3A%2F%2Facademic.oup.com%2Fnar%2Farticle%2F30%2F14%2F3059%2F2904316&amp;dt=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform%20%7C%20Nucleic%20Acids%20Research%20%7C%20Oxford%20Academic&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=site_name%2Cupdated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5bc796b60ea85b87&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=type%3Darticle%26title%3DMAFFT%253A%2520a%2520novel%2520method%2520for%2520rapid%2520multiple%2520sequence%2520alignment%2520based%2520on%2520fast%2520Fourier%2520transform%2520%257C%2520Nucleic%2520Acids%2520Research%2520%257C%2520Oxford%2520Academic%26description%3DAbstract.%2520%2520A%2520multiple%2520sequence%2520alignment%2520program%252C%2520MAFFT%252C%2520has%2520been%2520developed.%2520The%2520CPU%2520time%2520is%2520drastically%2520reduced%2520as%2520compared%2520with%2520existing%2520methods.%2520MAFFT%2520includ%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fnar%252Farticle%252F30%252F14%252F3059%252F2904316%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fnar%252FIssue%252F46%252F18%252F2%252Fm_cover.gif%253FExpires%253D2147483647%2526Signature%253DeV%7Ehm087cge5g8xBrr8Ad3gSMmcSfY8S8fYMMxVMyebMOzGEOd-Q8KiPx0XalwBTcA6jZy29jYCCVng%7EgYiA2bRURSHYiJJmFdS3MB%7EzxLd3wDRhaCxH2ogC-EC%7ENIicXdJr6ACFFc%7E6Slb3Z4F3GPnTUmF1TfmETtWJ9ZljeOrPJCq00DIOHPwcSP5C6wnJSfquTOINmA8u8gutkWbmGOwPFaTa37wgWdMQ3sPlXeaTD7x3QVAvxU1cvpttJcvurNH6ZD1KkAoWB3fbxyqXHIEHlDf-ZLeY2k321uAAnOvcvm5UKznRhGSheuGJ5tbEO3eOOn2s4QmfHPn3G8VWYA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2002-07-15%26site_name%3DOUP%2520Academic&amp;csi=undefined&amp;rev=v8.3.31-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style>
<script src="api.js" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->



<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="gkf436"></script></body></html>