<!DOCTYPE html>
<!--[if IE 9]><html class="ie9" lang="en"><![endif]-->
<html lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><iframe src="javascript:false" title="" style="width: 0px; height: 0px; border: 0px none; display: none;"></iframe><script type="text/javascript" async="" src="geo2.js"></script>
      <meta name="citation_pii" content="S1055790317305900">
<meta name="citation_issn" content="1055-7903">
<meta name="citation_volume" content="124">
<meta name="citation_lastpage" content="49">
<meta name="citation_publisher" content="Academic Press">
<meta name="citation_firstpage" content="37">
<meta name="citation_journal_title" content="Molecular Phylogenetics and Evolution">
<meta name="citation_type" content="JOUR">
<meta name="citation_doi" content="10.1016/j.ympev.2018.02.018">
<meta name="dc.identifier" content="10.1016/j.ympev.2018.02.018">
<meta name="citation_article_type" content="Full-length article">
<meta property="og:image" content="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-fx1.jpg">
<meta property="og:description" content="The family Curculionidae (Coleoptera), the “true” weevils, have diversified tightly linked to the evolution of flowering plants. Here, we aim to asses…">
<meta name="citation_title" content="Evolution of Trichobaris (Curculionidae) in relation to host plants: Geometric morphometrics, phylogeny and phylogeography">
<meta property="og:title" content="Evolution of Trichobaris (Curculionidae) in relation to host plants: Geometric morphometrics, phylogeny and phylogeography">
<meta name="citation_publication_date" content="2018/07/01">
<meta name="citation_online_date" content="2018/02/24">
<meta name="citation_pdf_url" content="https://www.sciencedirect.com/science/article/pii/S1055790317305900/pdfft?md5=2283f738c86c14daafaa5ea4c70b33c4&amp;pid=1-s2.0-S1055790317305900-main.pdf">
<meta name="robots" content="INDEX,FOLLOW,NOARCHIVE,NOODP,NOYDIR">
      <title>Evolution of Trichobaris (Curculionidae) in relation to 
host plants: Geometric morphometrics, phylogeny and phylogeography - 
ScienceDirect</title>
      <link rel="canonical" href="https://www.sciencedirect.com/science/article/pii/S1055790317305900">
      <meta name="viewport" content="initial-scale=1">
      <meta name="SDTech" content="Proudly brought to you by the SD Technology team in London, Dayton, and Amsterdam">
      <link rel="shortcut icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="icon" href="https://cdn.els-cdn.com/sd/favSD.ico" type="image/x-icon">
      <link rel="stylesheet" href="arp.css">
      <link rel="dns-prefetch" href="https://w.usabilla.com/">
      <link rel="dns-prefetch" href="https://www.deepdyve.com/">
      <link rel="dns-prefetch" href="https://smetrics.elsevier.com/">
      <script>
        window.pageTargeting = {"region":"us-east-1","platform":"sdtech","entitled":true,"crawler":"","journal":"Molecular Phylogenetics and Evolution"};
        window.pageData = {"content":[{"entitlementType":"package","format":"MIME-XHTML","id":"sd:article:pii:S1055790317305900","type":"sd:article:JL:scope-full","detail":"sd:article:subtype:fla","publicationType":"journal","issn":"1055-7903","volumeNumber":"124","suppl":"C"}],"page":{"businessUnit":"ELS:RP:ST","language":"en","name":"product:journal:article","noTracking":"false","productName":"SD","type":"CP-CA","environment":"prod","loadTimestamp":1531924536410,"loadTime":""},"visitor":{"accessType":"ae:ANON_IP","accountId":"ae:262591","accountName":"ae:International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"ae:12289512","ipAddress":"197.248.44.39","appSessionId":"fdf01f62447ee3481f3b4045afa46b236f2bgxrqa"}};
        window.arp = {
          config: {"reduxLogging":false,"assetsBaseUrl":"https://sdfestaticassets-us-east-1.sciencedirectassets.com/prod/d459717ce4bedcd05c18402c0b015aff6be02806","mediaBaseUrl":"https://ars.els-cdn.com/content/image/","googleMapsApiKey":"AIzaSyAV0ZB1XWtRlzJ06pJAW74Oan87lQVgBHY","bosUrl":"https://feedback.recs.d.elsevier.com/raw/events","bosTimeOut":60000,"strictMode":false},
          subscriptions: [],
          subscribe: function(cb) {
            var self = this;
            var i = this.subscriptions.push(cb) - 1;
            return function unsubscribe() {
              self.subscriptions.splice(i, 1);
            }
          },
        };
      </script>
      <script type="text/javascript" src="204774041.js"></script>
      <!-- begin mPulse embed code -->
      <script>
        (function(){
          if(window.BOOMR && window.BOOMR.version){return;}
          var dom,doc,where,iframe = document.createElement('iframe'),win = window;

          function boomerangSaveLoadTime(e) {
            win.BOOMR_onload=(e && e.timeStamp) || new Date().getTime();
          }
          if (win.addEventListener) {
            win.addEventListener("load", boomerangSaveLoadTime, false);
          } else if (win.attachEvent) {
            win.attachEvent("onload", boomerangSaveLoadTime);
          }

          iframe.src = "javascript:false";
          iframe.title = ""; iframe.role="presentation";
          (iframe.frameElement || iframe).style.cssText = "width:0;height:0;border:0;display:none;";
          where = document.getElementsByTagName('script')[0];
          where.parentNode.insertBefore(iframe, where);

          try {
            doc = iframe.contentWindow.document;
          } catch(e) {
            dom = document.domain;
            iframe.src="javascript:var d=document.open();d.domain='"+dom+"';void(0);";
            doc = iframe.contentWindow.document;
          }
          doc.open()._l = function() {
            var js = this.createElement("script");
            if(dom) this.domain = dom;
            js.id = "boomr-if-as";
            js.src = 'https://c.go-mpulse.net/boomerang/2FBN2-NKMGU-EJKY8-ZANKZ-SUJZF';
            BOOMR_lstart=new Date().getTime();
            this.body.appendChild(js);
          };
          doc.write('<body onload="document._l();">');
          doc.close();
        })();
      </script>
      <!-- end mPulse embed code -->
    <script src="satellite-5b3b560664746d57b70018c6.js"></script><script src="s-code-contents-9c0358adbc3b5986e210099b3bf1d427fc5bd286.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_232.js" async=""></script><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.3/config/MML_SVG.js?V=2.7.3"></script></head>
    <body><div style="display: none;" id="lightningjs-usabilla_live"><div><iframe id="lightningjs-frame-usabilla_live" frameborder="0"></iframe></div></div>
      <a class="sr-only sr-only-focusable" href="#app">Skip to main content</a>
      <!--[if lt IE 9]>
      <div id="ie8Warning" class="warning">
        <script>function ie8click() {
  const node = document.getElementById('ie8Warning');
  document.cookie = 'ie_warning_state=1';
  node.parentNode.removeChild(node);
}</script>
        <p>Please note that Internet Explorer version 8.x is not supported as of January 1, 2016.
        Please refer to <a href="https://service.elsevier.com/app/answers/detail/a_id/9831">this support page</a> for more information.</p>
        <a class="warning-close" onclick="ie8click()" title="Close IE warning">&times;</a>
      </div>
    <![endif]-->
      <div data-iso-key="_0"><div class="App" id="app" role="application" data-reactroot=""><div class="page"><section><div id="header"><div class="u-bg-white" style="overflow:visible" role="banner"><div class="els-header" style="min-height:80px"><a id="els-main-title-link" href="https://www.sciencedirect.com/"><svg id="els-header-wordmark" class="u-fill-orange u-margin-l-top u-margin-s-left u-margin-l-left-from-sm" viewBox="-3334 3439.4 163 26" style="width:163px;height:24px"><title>ScienceDirect</title><g transform="scale(.8125,.8125)"><path d="M-4099.8,4240.4c0-1.8,1-3.6,4.4-3.6c1.7,0,3.7,0.5,5.5,1.6l0.2-3.1c-1.7-0.7-3.5-1.1-5.6-1.1 c-5.5,0-7.8,2.9-7.8,6.4c0,6.6,10.4,7.2,10.4,11.7c0,1.8-1.4,3.6-4.6,3.6c-2,0-4-0.7-5.6-1.6l-0.4,3.1c1.8,0.9,4.2,1.2,6.1,1.2 c5,0,7.9-2.9,7.9-6.5C-4089.4,4245.8-4099.8,4244.9-4099.8,4240.4"></path><path id="c" d="M-4080.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4083.1,4244.3-4080.6,4243.1-4080.3,4242.9"></path><path id="i" d="M-4068,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-4066,4234.1-4067,4233.1-4068,4233.1 M-4069.5,4258.1h3v-17.7h-3V4258.1z"></path><path id="e" d="M-4057.1,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4058.7,4244.4-4057.4,4243.3-4057.1,4243.1z M-4047.5,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4 c0,5.8,3.5,9.2,7.9,9.2c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-4047.5,4248.9"></path><path d="M-4034.8,4240c-2.6,0-4.3,1.3-5.7,3.1l-0.5-2.6h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.5v12.8h3v-11.9 c0.8-1.1,2.5-2.9,2.9-3.1c0.3-0.2,1.5-0.5,2.5-0.5c2.7,0,2.9,1.4,3,4c0,1.4,0,3.7,0,3.7c0,3.5-0.1,7.6-0.1,7.6h3 c0,0,0.1-5.3,0.1-8.2c0-1.8,0-3.5-0.1-5.3C-4029.5,4241-4031.6,4240-4034.8,4240"></path><path d="M-3982.5,4255.6h-4.4v-18.6h4.8c6.4,0,8.2,5.2,8.2,9.2C-3973.8,4252.2-3976.5,4255.6-3982.5,4255.6z M-3981.6,4234.6h-8.4v23.5h8.1c8.6,0,11.5-6.7,11.5-11.9C-3970.4,4240.9-3973.2,4234.6-3981.6,4234.6"></path><path d="M-3950.5,4240c-1.9,0-3.4,1.7-4,3.2l-0.5-2.8h-2.8l0.2,1.4c0.1,0.9,0.2,2.1,0.2,3.4v12.8h3v-11.2 c0.6-1.5,2-4.4,3.7-4.4c1.1,0,1.2,1.2,1.2,1.4l2.5-0.7v-0.2c0,0,0-0.2-0.1-0.5C-3947.4,4240.9-3948.5,4240-3950.5,4240"></path><path d="M-3903.5,4255.2c-1.1,0.4-2,0.8-3,0.8c-1.4,0-1.9-0.8-1.9-2.8v-10.4h4.6v-2.3h-4.6v-4.7h-2.9v4.7h-3.2v2.3h3.2 v11.4c0,3.1,1.6,4.4,3.9,4.4c1.4,0,3-0.5,4.1-0.9L-3903.5,4255.2"></path><g><path d="M-3923.3,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-3926.1,4244.3-3923.6,4243.1-3923.3,4242.9"></path></g><g><path d="M-3941.6,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-3943.2,4244.4-3941.8,4243.3-3941.6,4243.1z M-3932,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3932,4248.9"></path></g><g><path d="M-3964.5,4233.1c-1.1,0-1.9,1.1-1.9,2.1c0,1.1,0.9,2.2,1.9,2.2s2-1.1,2-2.2 C-3962.5,4234.1-3963.4,4233.1-3964.5,4233.1 M-3965.9,4258.1h3v-17.7h-3V4258.1z"></path></g><g><path d="M-4004.4,4243.1c0.2-0.2,1.5-0.5,2.3-0.5c2.9,0,4.4,0.6,4.6,4.1h-8.8 C-4006,4244.4-4004.6,4243.3-4004.4,4243.1z M-3994.7,4248.9c0-6.1-1.7-8.9-7.1-8.9c-4.6,0-7.9,3.3-7.9,9.4c0,5.8,3.5,9.2,7.9,9.2 c3.3,0,5.1-0.7,6.7-1.7l-0.2-2.7c-1.1,0.9-3.5,1.8-5.5,1.8c-3.7,0-5.8-2.3-5.8-6.5v-0.6L-3994.7,4248.9"></path></g><g><path d="M-4019.1,4242.9c0.3-0.1,0.8-0.3,2-0.3c2,0,2.9,0.4,2.9,1.9h2.8c0-0.4,0-0.9,0-1.3 c-0.3-2.3-2.5-3.2-5.8-3.2c-3.6,0-7.9,2.7-7.9,9.2c0,6.2,3.3,9.4,7.8,9.4c2,0,4.1-0.4,5.9-1.6l-0.2-2.7c-1.2,1-3.4,1.8-4.8,1.8 c-2.5,0-5.4-2-5.4-7C-4021.8,4244.3-4019.4,4243.1-4019.1,4242.9"></path></g></g></svg></a><div class="move-right u-display-inline-block"><nav class="u-clr-grey8 u-show-from-md"><div class="u-display-inline-block" style="margin-top:29px"><span><span class="u-margin-l-right"><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/science/journals" id="els-header-navigation-section-journal" style="border-bottom:"><span class="anchor-text">Journals</span></a><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/science/bookbshsrw" id="els-header-navigation-section-book" style="border-bottom:"><span class="anchor-text">Books</span></a></span><a class="anchor qa-journals-and-books u-margin-l-right anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790317305900" id="els-header-navigation-section-register"><span class="anchor-text">Register</span></a></span><a class="anchor qa-no-js-fallback-link anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790317305900" id="els-header-user-sign-in"><span class="anchor-text">Sign in</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a><li style="list-style:none;display:inline"><a class="anchor u-margin-l-hor help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="help" title="Help" target="_blank"><span class="anchor-text"><svg focusable="false" viewBox="0 0 114 128" style="margin-top:-10px" width="21.375" height="24" class="icon icon-help"><path d="m57 8c-14.7 0-28.5 5.72-38.9 16.1-10.38 10.4-16.1 24.22-16.1 38.9 0 30.32 24.68 55 55 55 14.68 0 28.5-5.72 38.88-16.1 10.4-10.4 16.12-24.2 16.12-38.9 0-30.32-24.68-55-55-55zm0 1e1c24.82 0 45 20.18 45 45 0 12.02-4.68 23.32-13.18 31.82s-19.8 13.18-31.82 13.18c-24.82 0-45-20.18-45-45 0-12.02 4.68-23.32 13.18-31.82s19.8-13.18 31.82-13.18zm-0.14 14c-11.55 0.26-16.86 8.43-16.86 18v2h1e1v-2c0-4.22 2.22-9.66 8-9.24 5.5 0.4 6.32 5.14 5.78 8.14-1.1 6.16-11.78 9.5-11.78 20.5v6.6h1e1v-5.56c0-8.16 11.22-11.52 12-21.7 0.74-9.86-5.56-16.52-16-16.74-0.39-0.01-0.76-0.01-1.14 0zm-4.86 5e1v1e1h1e1v-1e1h-1e1z"></path></svg></span></a></li></div></nav></div><div class="u-hide-from-md move-right"><ul><li style="list-style:none;display:inline-block;margin-top:32px" class="u-margin-s-hor u-margin-l-hor-from-sm"><div class="hamburger-button"><button class="button-link button-link-primary" aria-label="Mobile menu" aria-expanded="false" type="button"><svg viewBox="0 0 40 18" width="40" height="18" y="52"><path d="M0 16h40v2H0zm0-8h40v2H0zm0-8h40v2H0z"></path></svg><span class="button-link-text"></span></button></div></li></ul><div style="bottom:0;left:0;position:fixed;top:0;width:100%;z-index:70;opacity:.8" class="u-bg-grey1 u-display-none"></div><div id="mobile-menu" style="overflow:auto;position:fixed;width:288px;right:0;top:0;z-index:1000;height:100%" aria-label="Mobile menu" class="u-bg-grey7 u-clr-grey1 u-display-none" role="navigation"><div class="u-bg-black panel-s"></div><div class="u-bg-grey8 panel-s"><ul><li style="list-style:none"><a class="anchor journals-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/science/journals" style="border-bottom:" id="mobile-journals-link"><span class="anchor-text">Journals</span></a></li><li style="list-style:none"><a class="anchor books-link u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/science/bookbshsrw" style="border-bottom:" id="mobile-books-link"><span class="anchor-text">Books</span></a></li><li style="list-style:none"><a class="anchor u-padding-xs-top anchor-has-inherit-color" href="https://www.sciencedirect.com/user/register?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790317305900" id="mobile-register-link"><span class="anchor-text">Register</span></a></li></ul></div><div class="panel-s u-bg-grey7"><ul class="text-xs"></ul><ul class="u-margin-s-top text-s"><li style="list-style:none"><a class="anchor anchor-has-inherit-color" href="https://www.sciencedirect.com/user/login?returnURL=%2Fscience%2Farticle%2Fpii%2FS1055790317305900" id="mobile-sign-in-out-link" rel="nofollow"><span class="anchor-text">Sign In</span></a></li><li style="list-style:none;display:inline"><a class="anchor u-padding-xs-top text-s help-link anchor-has-inherit-color" href="https://service.elsevier.com/app/home/supporthub/sciencedirect/" id="mobile-help" title="Help" target="_blank"><span class="anchor-text">Help</span></a></li></ul></div></div><div class="move-bottom move-center"></div></div></div></div></div><div class="Article" id="mathjax-container"><div class="sticky-outer-wrapper"><div class="sticky-inner-wrapper" style="position: relative; z-index: 1; transform: translate3d(0px, 0px, 0px);"><div class="Toolbar" role="region" aria-label="download options and search"><div class="toolbar-container"><div class="u-show-from-lg col-lg-6">&nbsp;</div><div class="buttons pull-left pad-left"><button class="show-toc-button u-hide-from-lg"><svg focusable="false" viewBox="0 0 104 128" width="19.5" height="24" class="icon icon-list"><path d="m2e1 95a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm0-3e1a9 9 0 0 1 -9 9 9 9 0 0 1 -9 -9 9 9 0 0 1 9 -9 9 9 0 0 1 9 9zm14 55h68v1e1h-68zm0-3e1h68v1e1h-68zm0-3e1h68v1e1h-68z"></path></svg><span class="label">Outline</span></button><div class="PdfDownloadButton"><button class="button-link button-link-secondary" id="pdfLink" aria-controls="Download PDF options" type="button"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="button-link-text"><span class="pdf-download-label u-show-inline-from-lg">Download PDF</span><span class="pdf-download-label-short u-hide-from-lg">Download</span></span></button></div><div class="ExportCitation" role="region" id="export-citation"><button class="button ExportCitationButton button-anchor" type="button"><span class="button-text">Export</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div></div><div class="pull-right pad-right u-show-from-md"><form id="quick-search" class="QuickSearch" action="/search/advanced" method="get"><input class="query" aria-label="Search ScienceDirect" name="qs" placeholder="Search ScienceDirect" type="text"><button class="button button-primary" type="submit" aria-label="Submit search"><span class="button-text"><svg focusable="false" viewBox="0 0 100 128" height="20" width="18.75" class="icon icon-search"><path d="m19.22 76.91c-5.84-5.84-9.05-13.6-9.05-21.85s3.21-16.01 9.05-21.85c5.84-5.83 13.59-9.05 21.85-9.05 8.25 0 16.01 3.22 21.84 9.05 5.84 5.84 9.05 13.6 9.05 21.85s-3.21 16.01-9.05 21.85c-5.83 5.83-13.59 9.05-21.84 9.05-8.26 0-16.01-3.22-21.85-9.05zm80.33 29.6l-26.32-26.32c5.61-7.15 8.68-15.9 8.68-25.13 0-10.91-4.25-21.17-11.96-28.88-7.72-7.71-17.97-11.96-28.88-11.96s-21.17 4.25-28.88 11.96c-7.72 7.71-11.97 17.97-11.97 28.88s4.25 21.17 11.97 28.88c7.71 7.71 17.97 11.96 28.88 11.96 9.23 0 17.98-3.07 25.13-8.68l26.32 26.32 7.03-7.03"></path></svg></span></button><a class="advanced-search-link" href="https://www.sciencedirect.com/search/advanced">Advanced</a><input name="origin" value="article" type="hidden"><input name="zone" value="qSearch" type="hidden"></form></div></div></div></div></div><div class="article-wrapper grid row"><div class="u-show-from-lg col-lg-6"><div class="TableOfContents u-margin-l-bottom" lang="en"><div class="Outline" id="toc-outline"><h2>Outline</h2><ol class="u-padding-s-bottom"><li><a href="#ab010" title="Highlights">Highlights</a></li><li><a href="#ab015" title="Abstract">Abstract</a></li><li><a href="#ab005" title="Graphical abstract">Graphical abstract</a></li><li><a href="#kg005" title="Keywords">Keywords</a></li><li><a href="#s0005" title="1. Introduction">1. Introduction</a></li><li><a href="#s0010" title="2. Material and methods">2. Material and methods</a></li><li><a href="#s0055" title="3. Results">3. Results</a></li><li><a href="#s0090" title="4. Discussion">4. Discussion</a></li><li><a href="#s0110" title="5. Conclusions">5. Conclusions</a></li><li><a href="#s0115" title="Acknowledgments">Acknowledgments</a></li><li><a href="#s0120" title="Conflict of interest">Conflict of interest</a></li><li><a href="#s0130" title="Appendix A. Supplementary material">Appendix A. Supplementary material</a></li><li><a href="#ec-research-data" title="Research Data">Research Data</a></li><li><a href="#bi005" title="References">References</a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show full outline</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Figures" id="toc-figures"><h2>Figures (10)</h2><ol><li><a href="#f0050"><div><img alt="Unlabelled figure" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-fx1.sml" style="max-width: 164px; max-height: 164px;"></div></a></li><li><a href="#f0005"><div><img alt="Fig. 1. Trichobaris sampling localities in USA and Mexico" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr1.sml" style="max-width: 219px; max-height: 148px;"></div></a></li><li><a href="#f0010"><div><img alt="Fig. 2. Landmarks (green dots) used to perform the morphogeometric analyses" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr2.sml" style="max-width: 208px; max-height: 164px;"></div></a></li><li><a href="#f0015"><div><img alt="Fig. 3. Analysis of Canonical Variance of the genus Trichobaris, obtained by 75…" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr3.sml" style="max-width: 219px; max-height: 153px;"></div></a></li><li><a href="#f0020"><div><img alt="Fig. 4. Analysis of Canonical Variance of the rostrum of species of genus Trichobaris,…" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr4.sml" style="max-width: 219px; max-height: 149px;"></div></a></li><li><a href="#f0025"><div><img alt="Fig. 5. Centroid size of Trichobaris species" src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr5.sml" style="max-width: 134px; max-height: 164px;"></div></a></li></ol><button class="button button-anchor" type="button"><span class="button-text">Show all figures</span><svg focusable="false" viewBox="0 0 92 128" height="20" width="17.25" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button><div class="PageDivider"></div></div><div class="Tables" id="toc-tables"><h2>Tables (2)</h2><ol class="u-padding-s-bottom"><li><a href="#t0005" title="Sampling sites of Trichobaris weevils on their host plant species in Mexico and USA. (*) COI gene sequence from previous publications (De-la-Mora et al., 2015; in press)."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 1</a></li><li><a href="#t0010" title="Genetic diversity for Trichobaris species using 663bp of COI gene."><svg focusable="false" viewBox="0 0 98 128" width="18.375" height="24" class="icon icon-table"><path d="m54 68h32v32h-32v-32zm-42 0h32v32h-32v-32zm0-42h32v32h-32v-32zm42 0h32v32h-32v-32zm-52 84h94v-94h-94v94z"></path></svg>Table 2</a></li></ol><div class="PageDivider"></div></div><div class="Extras" id="toc-extras"><h2>Extras (3)</h2><ol class="u-padding-s-bottom"><li><a href="#m0005" title="Supplementary data 1"><svg focusable="false" viewBox="0 0 94 128" width="17.625" height="24" class="icon icon-text-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-15.99-2.01-4e1 -17.64-32.36h-56.4zm0 1e1h46.4v22.36 17.64 4e1 2.01 5.99h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06zm-13.6 38v1e1h5e1v-1e1h-5e1zm0 2e1v1e1h5e1v-1e1h-5e1z"></path></svg>Supplementary data 1</a></li><li><a href="#m0010" title="Supplementary data 2"><svg focusable="false" viewBox="0 0 95 128" width="17.8125" height="24" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg>Supplementary data 2</a></li><li><a href="#m0015" title="Supplementary data 3"><svg focusable="false" viewBox="0 0 95 128" width="17.8125" height="24" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg>Supplementary data 3</a></li></ol><div class="PageDivider"></div></div></div></div><article class="col-lg-12 col-md-16 pad-left pad-right" role="main" lang="en"><noscript>JavaScript is disabled on your browser. Please enable JavaScript to use all the features on this page.</noscript><div class="Publication" id="publication"><div class="publication-brand u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903"><img class="publication-brand-image" src="elsevier-non-solus.png" alt="Elsevier"></a></div><div class="publication-volume u-text-center"><h2 class="publication-title" id="publication-title"><a class="publication-title-link" title="Go to Molecular Phylogenetics and Evolution on ScienceDirect" href="https://www.sciencedirect.com/science/journal/10557903">Molecular Phylogenetics and Evolution</a></h2><div class="text-xs"><a title="Go to table of contents for this volume/issue" href="https://www.sciencedirect.com/science/journal/10557903/124/supp/C">Volume 124</a>, <!-- -->July 2018<!-- -->, Pages 37-49</div></div><div class="publication-cover u-show-from-sm"><a href="https://www.sciencedirect.com/science/journal/10557903/124/supp/C"><img class="publication-cover-image" src="1-s2.gif" alt="Molecular Phylogenetics and Evolution"></a></div></div><h1 class="Head"><span class="title-text">Evolution of <em>Trichobaris</em> (Curculionidae) in relation to host plants: Geometric morphometrics, phylogeny and phylogeography</span></h1><div class="Banner" id="banner"><div class="wrapper truncated"><div class="AuthorGroups"><div class="author-group" id="author-group"><span class="sr-only">Author links open overlay panel</span><a class="author size-m workspace-trigger" name="bau005" href="#%21"><span class="content"><span class="text given-name">Marisol</span><span class="text surname">De-la-Mora</span><span class="author-ref" id="baf005"><sup>a</sup></span><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg></span></a><a class="author size-m workspace-trigger" name="bau010" href="#%21"><span class="content"><span class="text given-name">Daniel</span><span class="text surname">Piñero</span><span class="author-ref" id="baf005"><sup>a</sup></span></span></a><a class="author size-m workspace-trigger" name="bau015" href="#%21"><span class="content"><span class="text given-name">Ken</span><span class="text surname">Oyama</span><span class="author-ref" id="baf010"><sup>b</sup></span></span></a><a class="author size-m workspace-trigger" name="bau020" href="#%21"><span class="content"><span class="text given-name">Brian</span><span class="text surname">Farrell</span><span class="author-ref" id="baf015"><sup>c</sup></span></span></a><a class="author size-m workspace-trigger" name="bau025" href="#%21"><span class="content"><span class="text given-name">Susana</span><span class="text surname">Magallón</span><span class="author-ref" id="baf020"><sup>d</sup></span></span></a><a class="author size-m workspace-trigger" name="bau030" href="#%21"><span class="content"><span class="text given-name">Juan</span><span class="text surname">Núñez-Farfán</span><span class="author-ref" id="baf005"><sup>a</sup></span><svg focusable="false" viewBox="0 0 106 128" width="19.875" height="24" class="icon icon-person"><path d="m11.07 1.2e2l0.84-9.29c1.97-18.79 23.34-22.93 41.09-22.93 17.74 0 39.11 4.13 41.08 22.84l0.84 9.38h10.04l-0.93-10.34c-2.15-20.43-20.14-31.66-51.03-31.66s-48.89 11.22-51.05 31.73l-0.91 10.27h10.03m41.93-102.29c-9.72 0-18.24 8.69-18.24 18.59 0 13.67 7.84 23.98 18.24 23.98s18.24-10.31 18.24-23.98c0-9.9-8.52-18.59-18.24-18.59zm0 52.29c-15.96 0-28-14.48-28-33.67 0-15.36 12.82-28.33 28-28.33s28 12.97 28 28.33c0 19.19-12.04 33.67-28 33.67"></path></svg><svg focusable="false" viewBox="0 0 102 128" width="19.125" height="24" class="icon icon-envelope"><path d="m55.8 57.2c-1.78 1.31-5.14 1.31-6.9 0l-31.32-23.2h69.54l-31.32 23.19zm-55.8-24.78l42.94 32.62c2.64 1.95 6.02 2.93 9.4 2.93s6.78-0.98 9.42-2.93l40.24-30.7v-10.34h-102zm92 56.48l-18.06-22.74-8.04 5.95 17.38 21.89h-64.54l18.38-23.12-8.04-5.96-19.08 24.02v-37.58l-1e1 -8.46v61.1h102v-59.18l-1e1 8.46v35.62"></path></svg></span></a></div></div></div><button class="show-hide-details"><svg viewBox="0 0 9 9" class="icon-expand"><path d="M5 7H4V5H2V4h2V2h1v2h2v1H5z"></path><path d="M0 0v9h9V0zm1 1h7v7H1z"></path></svg>Show more</button></div><div class="DoiLink" id="doi-link"><a class="doi" href="https://doi.org/10.1016/j.ympev.2018.02.018" target="_blank" aria-label="Persistent link using digital object identifier" title="Persistent link using digital object identifier">https://doi.org/10.1016/j.ympev.2018.02.018</a><a class="rights-and-content" target="_blank" href="https://s100.copyright.com/AppDispatchServlet?publisherName=ELS&amp;contentID=S1055790317305900&amp;orderBeanReset=true">Get rights and content</a></div><section class="ReferencedArticles"></section><section class="ReferencedArticles"></section><div class="PageDivider"></div><div class="Abstracts" id="abstracts"><div class="abstract author-highlights" id="ab010" lang="en"><h2 class="section-title">Highlights</h2><div id="as010"><p id="sp0010"><dl class="list"><dt class="list-label">•</dt><dd class="list-description"><p id="p0005">The first molecular phylogeny of the genus <em>Trichobaris</em> (Coleoptera: Curculionidae) is herein constructed.</p></dd><dt class="list-label">•</dt><dd class="list-description"><p id="p0010">Genetic data are generated for two mitochondrial and two nuclear markers.</p></dd><dt class="list-label">•</dt><dd class="list-description"><p id="p0015">The evolution of weevil’s rostrum is related to the host plant part used (e.g., stems or fruits) to oviposit.</p></dd><dt class="list-label">•</dt><dd class="list-description"><p id="p0020">The haplotype phylogeny suggests a recent emergence of species through interaction with their host plants.</p></dd></dl></p></div></div><div class="abstract author" id="ab015" lang="en"><h2 class="section-title">Abstract</h2><div id="as015"><p id="sp0015">The
 family Curculionidae (Coleoptera), the “true” weevils, have diversified
 tightly linked to the evolution of flowering plants. Here, we aim to 
assess diversification at a lower taxonomic level. We analyze the 
evolution of the genus <em>Trichobaris</em> in association with their host plants. <em>Trichobaris</em> comprises eight to thirteen species; their larvae feed inside the fruits of <em>Datura</em>
 spp. or inside the stem of wild and cultivated species of Solanaceae, 
such as potato, tobacco and tomato. We ask the following questions: (1) 
does the rostrum of <em>Trichobaris</em> species evolve according to the
 plant tissue used to oviposit, i.e., shorter rostrum to dig in stems 
and longer to dig in fruits? and (2) does <em>Trichobaris</em> diversify mainly in relation to the use of <em>Datura</em> species? For the first question, we estimated the phylogeny of <em>Trichobaris</em>
 based on four gene sequences (nuclear 18S and 28S rRNA genes and 
mitochondrial 16S rRNA and COI genes). Then, we carried out 
morphogeometric analyses of the <em>Trichobaris</em> species using 75 
landmarks. For the second question, we calibrated a COI haplotype 
phylogeny using a constant rate of divergence to infer the 
diversification time of <em>Trichobaris</em> species, and we traced the 
host plant species on the haplotype network. We performed an ancestral 
state reconstruction analysis to infer recent colonization events and 
conserved associations with host plant species. We found that ancestral 
species in the <em>Trichobaris</em> phylogeny use the stem of <em>Solanum</em> plants for oviposition and display weak sexual dimorphism of rostrum size, whereas other, more recent species of <em>Trichobaris</em> display sexual dimorphism in rostrum size and use the fruits of <em>Datura</em> species, and a possible reversion to use the stem of Solanaceae was detected in one <em>Trichobaris</em> species. The use of <em>Datura</em> species by <em>Trichobaris</em> species is widely distributed on haplotype networks and restricted to <em>Trichobaris</em> species that originated ca. 5 ± 1.5 Ma. Given that the origin of <em>Trichobaris</em> is estimated to be ca. 6 ± 1.5 Ma, it is likely that <em>Datura</em> has played a role in its diversification.</p></div></div><div class="abstract graphical" id="ab005" lang="en"><h2 class="section-title">Graphical abstract</h2><div id="as005"><p id="sp0005"><span class="display"><figure class="figure" id="f0050"><span id="download-image"><img src="1-s2.jpg" alt="" height="245"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-fx1_lrg.jpg" target="__blank">Download high-res image (158KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-fx1.jpg" target="__blank">Download full-size image</a></li></ol></span></figure></span></p></div></div></div><ul class="issue-navigation" id="issue-navigation"><li class="previous"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790317305857"><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-left"><path d="m1 61l45-45 7 7-38 38 38 38-7 7z"></path></svg><span class="button-alternative-text"><span><strong>Previous </strong>article<span class="extra-detail"> in issue</span></span></span></a></li><li class="next"><a class="button-alternative button-alternative-tertiary" href="https://www.sciencedirect.com/science/article/pii/S1055790317306048"><span class="button-alternative-text"><strong>Next </strong>article<span class="extra-detail"> in issue</span></span><svg focusable="false" viewBox="0 0 54 128" width="32" height="32" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></a></li></ul><div class="Keywords"><div id="kg005" class="keywords-section"><h2 class="section-title">Keywords</h2><div id="k0005" class="keyword"><span><em>Datura</em></span></div><div id="k0010" class="keyword"><span>Colonization to host-plant</span></div><div id="k0015" class="keyword"><span>Ancestral host-plant</span></div><div id="k0020" class="keyword"><span><em>Trichobaris</em></span></div><div id="k0025" class="keyword"><span>Insect-plant interaction</span></div></div></div><div class="Body" id="body"><div><section id="s0005"><h2 id="st025">1. Introduction</h2><p id="p0025">It
 is rare to see the process of speciation in nature. The use of both 
phylogenetic and phylogeographic approaches can help in understanding 
the evolution of species. Ideally, one should sample all species of a 
taxonomic group, such as a genus, and test that all named species 
constitute evolutionary entities within that group to rule out the 
effects of other processes, as well develop explanations for observed 
patterns (<a name="bb0020" href="#b0020" class="workspace-trigger">Avise and Johns, 1999</a>, <a name="bb0185" href="#b0185" class="workspace-trigger">Holt and Jønsson, 2014</a>). The phylogenetic hypothesis provides an indirect record of the sequence of lineage-divergence events (<a name="bb0045" href="#b0045" class="workspace-trigger">Barraclough and Nee, 2001</a>)
 that can be complemented with phylogeographic analysis. This permits 
the recognition of the geographic and spatial genetic structure of 
populations within a species at early stages of differentiation (<a name="bb0015" href="#b0015" class="workspace-trigger">Avise, 2000</a>, <a name="bb0170" href="#b0170" class="workspace-trigger">Gavrilets, 2003</a>). These integrative studies can provide insights into how speciation proceeds (e.g., <a name="bb0110" href="#b0110" class="workspace-trigger">Chen et al., 2017</a>, <a name="bb0335" href="#b0335" class="workspace-trigger">Supple et al., 2014</a>).</p><p id="p0030">Speciation in phytophagous insects has long been associated with plant diversification (<a name="bb0150" href="#b0150" class="workspace-trigger">Ehrlich and Raven, 1964</a>, <a name="bb0155" href="#b0155" class="workspace-trigger">Farrell, 1998</a>). The majority of phytophagous species in the order Coleoptera belong to the superfamilies Chrysomeloidea and Curculionoidea (<a name="bb0175" href="#b0175" class="workspace-trigger">Grimaldi and Engel, 2005</a>). Curculionidae beetles, known as weevils, include approximately 62 000 described species (<a name="bb0265" href="#b0265" class="workspace-trigger">Oberprieler et al., 2007</a>). Weevil diversity is associated mainly with angiosperms (<a name="bb0240" href="#b0240" class="workspace-trigger">McKenna et al., 2009</a>) due to their ability to exploit different plant tissues (<a name="bb0230" href="#b0230" class="workspace-trigger">Marvaldi et al., 2002</a>).
 However, weevil diversity has been associated with presumed key 
innovations, such as the rostrum’s length and shape, endophagous larvae 
and geniculate antennae and, in some groups, their association with 
fungi (<a name="bb0265" href="#b0265" class="workspace-trigger">Oberprieler et al., 2007</a>, <a name="bb0235" href="#b0235" class="workspace-trigger">Matsubayashi et al., 2010</a>).</p><p id="p0035">Weevils
 are pests of many economically important plants, such as chestnut, 
sugarcane, banana, cotton, raspberry, avocado, mango, pecan, pepper, 
alfalfa and sunflower (<a name="bb0025" href="#b0025" class="workspace-trigger">Avtzis et al., 2013</a>, <a name="bb0220" href="#b0220" class="workspace-trigger">Kuroki and Kodama, 1987</a>, <a name="bb0225" href="#b0225" class="workspace-trigger">Lemic et al., 2016</a>, <a name="bb0315" href="#b0315" class="workspace-trigger">Shankar et al., 2015</a>, <a name="bb0040" href="#b0040" class="workspace-trigger">Barr et al., 2013</a>, <a name="bb0280" href="#b0280" class="workspace-trigger">Parra et al., 2009</a>, <a name="bb0100" href="#b0100" class="workspace-trigger">Castañeda-Vildózola et al., 2015</a>, <a name="bb0060" href="#b0060" class="workspace-trigger">Bierig, 1939</a>, <a name="bb0050" href="#b0050" class="workspace-trigger">Basio et al., 1994</a>, <a name="bb0250" href="#b0250" class="workspace-trigger">Mynhardt, 2006</a>, <a name="bb0095" href="#b0095" class="workspace-trigger">Capinera, 2005</a>, <a name="bb0190" href="#b0190" class="workspace-trigger">Iwase et al., 2015</a>, <a name="bb0105" href="#b0105" class="workspace-trigger">Charlet, 1983</a>).
 The historical relationship between weevils and their host plants has 
scarcely been explored at the macroevolutionary level. Determination of 
ancestral and recently colonized host plant species is also a topic that
 remains understudied (e.g., <a name="bb0190" href="#b0190" class="workspace-trigger">Iwase et al., 2015</a>, <a name="bb0215" href="#b0215" class="workspace-trigger">Kuester et al., 2012</a>).
 Some studies have shown the ecological and evolutionary consequences of
 host shifts for biological control and to sustain productivity (<a name="bb0270" href="#b0270" class="workspace-trigger">Oliveira et al., 2012</a>).</p><p id="p0040">At
 a microevolutionary level, the effect of the host plant species on the 
genetic variation of weevil species in a genus is, likewise, a poorly 
explored question (e.g., <a name="bb0180" href="#b0180" class="workspace-trigger">Hernández-Vera et al., 2010</a>, <a name="bb0210" href="#b0210" class="workspace-trigger">Kohyama et al., 2014</a>).
 In some instances, no clear relationship between the genetic variation 
of weevils and their host plants has been found. In the stenophagous 
capitulum weevil (<em>Larinus cynarae</em>), the primary distribution of its genetic diversity indicates a geographic division followed by branching of <em>L. cynarae</em>’s lineage into different host plants (<a name="bb0075" href="#b0075" class="workspace-trigger">Briese et al., 1996</a>). However, in the boll weevil, <em>Anthonomus grandis</em>, three morphological forms associated with species of <em>Gossypium</em> have been described<em>;</em> although some haplotypes appeared related to either wild or cultivated cotton (<a name="bb0040" href="#b0040" class="workspace-trigger">Barr et al., 2013</a>), most genetic differentiation is due to their geographical distribution (<a name="bb0215" href="#b0215" class="workspace-trigger">Kuester et al., 2012</a>). In an extreme case, genetic variation of alfalfa weevil, <em>Hypera postica</em>, is not associated with its host plant or with its geographic distribution (<a name="bb0190" href="#b0190" class="workspace-trigger">Iwase et al., 2015</a>).
 However, the interaction between weevils and their host plants can 
result in a clear association of genetic variation of both organisms, as
 exemplified by <em>Curculio hilgendorfi</em> and <em>Castanopsis sieboldi</em> (<a name="bb0010" href="#b0010" class="workspace-trigger">Aoki et al., 2011</a>), or as a shared phylogeographical pattern (<a name="bb0005" href="#b0005" class="workspace-trigger">Aoki et al., 2009</a>).</p><p id="p0045">Weevil species of the genus <em>Trichobaris</em>
 (Curculionidae: Baridinae) use plants in the family Solanaceae as food 
and for larval development inside stems or fruits. The female digs a 
hole in the stem or fruit to oviposit; hatching larvae feed inside until
 pupation; and once metamorphosis occurs, adults emerge to start a new 
life cycle (<a name="bb0085" href="#b0085" class="workspace-trigger">Cabrales-Vargas, 1991</a>). Twelve described species integrate the genus: <em>Trichobaris major</em><a name="bb0035" href="#b0035" class="workspace-trigger">Barber 1935</a>, <em>T. soror</em> Champion 1909, <em>T. pueblana</em> Casey 1920, <em>T. trinotata</em> Say 1831, <em>T. insolita</em> Casey 1892, <em>T. championi</em><a name="bb0035" href="#b0035" class="workspace-trigger">Barber 1935</a>, <em>T. mucorea</em> LeConte 1854, <em>T. bridwelli</em><a name="bb0035" href="#b0035" class="workspace-trigger">Barber 1935</a>, <em>T. compacta</em> Casey 1892, <em>T. pellicea</em> Boheman 1844, <em>T. texana</em> LeConte 1876 and <em>T. cylindrica</em> Casey 1892. Of these species, six develop into fruits of <em>Datura</em> species, three species into the stems of potato, tobacco and tomatillo (<em>Solanum tuberosum, Nicotiana attenuata</em> and <em>Physalis</em> spp., respectively), and four into the stems of wild species of <em>Solanum</em> (<em>S. eleagnifolium, S. rostratum, S. carolinense</em>) (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>). The life cycle of these weevils is closely associated with their host plants; for instance, the larvae of <em>T. bridwelli</em> cannot survive in a different host (<a name="bb0125" href="#b0125" class="workspace-trigger">Cuda and Burke, 1991</a>). Ever since <a name="bb0035" href="#b0035" class="workspace-trigger">Barber’s (1935)</a> monographic study, the genus has not been studied again in such depth. His work on <em>Trichobaris</em> includes specimens from a wide range of locations and detailed morphological descriptions. Nevertheless, <a name="bb0035" href="#b0035" class="workspace-trigger">Barber (1935)</a>
 noted issues not resolved as of yet, including the need for precise 
information on host plants and the relevance of body shape and size in 
delimiting morphological species because some named species were thought
 to be environmentally related variants. Thus, the study of speciation 
makes it necessary to identify and establish the species, their host 
plants and the determination of their evolutionary relationships.</p><p id="p0050">In this study, we follow two approximations to study the evolution of <em>Trichobaris</em> species with their host plants. Under a phylogenetic framework, we tested whether the rostrum of <em>Trichobaris</em> species evolved according to the plant tissue used to oviposit, i.e., short rostrum for dig the stem in ancestral species of <em>Trichobaris</em> and long rostrum for dig the fruit in derived species of <em>Trichobaris</em>.
 We analyze the general morphology and, in particular, the rostrum of 
this weevils and build a molecular phylogeny using four gene sequences. 
We also tested whether diversification of <em>Trichobaris</em> is due mainly to the use of <em>Datura</em> species. To do this, under a phylogeographic framework, we map the host distribution on the haplotype network of each <em>Trichobaris</em>
 species and reconstructed the ancestral host plant species on the 
calibrated COI haplotype phylogeny to infer the historical associations 
or new colonizations of host plants.</p></section><section id="s0010"><h2 id="st030">2. Material and methods</h2><section id="s0015"><h3 id="st035">2.1. Sampling and specimens examined</h3><div><p id="p0055">A total of 168 insects (including adult and larval stages) were collected in 33 localities across the <em>Trichobaris</em> distribution range in Mexico and the United States of America (USA) (<a name="bt0005" href="#t0005" class="workspace-trigger">Table 1</a>; <a name="bf0005" href="#f0005" class="workspace-trigger">Fig. 1</a>). All adult insects were identified using the key provided by <a name="bb0035" href="#b0035" class="workspace-trigger">Barber (1935)</a>. We collected almost all described species of <em>Trichobaris</em> Le Conte (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>). Collectively, we found weevils in eight different species of <em>Datura</em>, four wild species of Solanaceae (<em>Physallis</em> sp., <em>Solanum eleagnifolium, S. carolinense</em> and <em>S. rostratum</em>) and one cultivated species of Solanaceae (potato, <em>S. tuberosum</em>).</p><div class="tables frame-topbot rowsep-0 colsep-0" id="t0005"><div class="captions"><span id="cn0050"><p id="sp0065"><span class="label">Table 1</span>. Sampling sites of <em>Trichobaris</em> weevils on their host plant species in Mexico and USA. (<sup>*</sup>) COI gene sequence from previous publications (<a name="bb0135" href="#b0135" class="workspace-trigger">De-la-Mora et al., 2015</a>; in press).</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">Number</th><th scope="col" class="align-left">Host plant</th><th scope="col" class="align-left">State</th><th scope="col" class="align-left">Country</th><th scope="col" class="align-left">Sampling site</th><th scope="col" class="align-left">Coordinates</th><th scope="col" class="align-left"># insects DNA data</th><th scope="col" class="align-left"># insects Morphology data</th></tr></thead><tbody><tr class="valign-top"><td class="align-left">1</td><td class="align-left"><em>D. discolor</em></td><td class="align-left">Baja California</td><td class="align-left">Mexico</td><td class="align-left">129</td><td class="align-left">26°0′21.46″N, 111°20′35.34″W</td><td class="align-left">5</td><td class="align-left">7</td></tr><tr class="valign-top"><td class="align-left">2</td><td class="align-left"><em>D. inoxia</em></td><td class="align-left">Hidalgo</td><td class="align-left">Mexico</td><td class="align-left">505</td><td class="align-left">20°34′54.50″N, 99°33′50.58″W</td><td class="align-left">5</td><td class="align-left">2</td></tr><tr class="valign-top"><td class="align-left">3</td><td class="align-left"><em>D. inoxia</em></td><td class="align-left">Oaxaca</td><td class="align-left">Mexico</td><td class="align-left">516</td><td class="align-left">16°28′46.10″N, 96°13′3.51″W</td><td class="align-left">1</td><td class="align-left">5</td></tr><tr class="valign-top"><td class="align-left">4</td><td class="align-left"><em>D. pruinosa</em></td><td class="align-left">Oaxaca</td><td class="align-left">Mexico</td><td class="align-left">415</td><td class="align-left">16°40′2.60″N, 96°22′48.98″W</td><td class="align-left">1</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">5</td><td class="align-left"><em>D. quercifolia</em></td><td class="align-left">Hidalgo</td><td class="align-left">Mexico</td><td class="align-left">207</td><td class="align-left">20°19′3.79″N, 99°9′34.98″W</td><td class="align-left">20</td><td class="align-left">12</td></tr><tr class="valign-top"><td class="align-left">6</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Puebla</td><td class="align-left">Mexico</td><td class="align-left">119</td><td class="align-left">19°5′38.49″N, 98°24′34.89″W</td><td class="align-left">6</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">7</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Oaxaca</td><td class="align-left">Mexico</td><td class="align-left">104</td><td class="align-left">17°14′11″N, 96°25′53.77″W</td><td class="align-left">9</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">8</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Hidalgo</td><td class="align-left">Mexico</td><td class="align-left">106</td><td class="align-left">20°8′18.98″N, 98°55′21.38″W</td><td class="align-left">5</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">9</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Puebla</td><td class="align-left">Mexico</td><td class="align-left">110</td><td class="align-left">18°54′8.23″N, 98°26′21.15″W</td><td class="align-left">19</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">10</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Puebla</td><td class="align-left">Mexico</td><td class="align-left">109</td><td class="align-left">18°56′27.15″N, 98°6′53.46″W</td><td class="align-left">18</td><td class="align-left">24</td></tr><tr class="valign-top"><td class="align-left">11</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Oaxaca</td><td class="align-left">Mexico</td><td class="align-left">113</td><td class="align-left">16°55′11.92″N, 96°23′6.10″W</td><td class="align-left">20</td><td class="align-left">1</td></tr><tr class="valign-top"><td class="align-left">12</td><td class="align-left"><em>Physallis</em> sp.</td><td class="align-left">Hidalgo</td><td class="align-left">Mexico</td><td class="align-left">T18</td><td class="align-left">20°35′23.59′N, 99°37′11.22″W</td><td class="align-left">2</td><td class="align-left">2</td></tr><tr class="valign-top"><td class="align-left">13</td><td class="align-left"><em>Solanum eleagnifolium</em></td><td class="align-left">Tamaulipas</td><td class="align-left">Mexico</td><td class="align-left">722</td><td class="align-left">25°58′45.05″N, 98°5′45.66″W</td><td class="align-left">1</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">14</td><td class="align-left"><em>Solanum rostratum</em></td><td class="align-left">SanLuisPotosí</td><td class="align-left">Mexico</td><td class="align-left">620</td><td class="align-left">23°33′36.99″N, 100°37′41.69″W</td><td class="align-left">8</td><td class="align-left">1</td></tr><tr class="valign-top"><td class="align-left">15</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">California</td><td class="align-left">USA</td><td class="align-left">5SD</td><td class="align-left">33°46′4.14″N, 116°19′28.23″W</td><td class="align-left">1</td><td class="align-left">3</td></tr><tr class="valign-top"><td class="align-left">16</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">California</td><td class="align-left">USA</td><td class="align-left">8SD</td><td class="align-left">_____</td><td class="align-left">3</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">17</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">California</td><td class="align-left">USA</td><td class="align-left">20SD</td><td class="align-left">33°59′46.45″N, 116°34′43.69″W</td><td class="align-left">1</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">18</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">Arizona</td><td class="align-left">USA</td><td class="align-left">0Az</td><td class="align-left">32°35′32.79″N, 110°50′56.54″W</td><td class="align-left">3</td><td class="align-left">1</td></tr><tr class="valign-top"><td class="align-left">19</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">Arizona</td><td class="align-left">USA</td><td class="align-left">1Az</td><td class="align-left">32°36′50.10″N, 110°49′57.91″W</td><td class="align-left">3</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">20</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">Arizona</td><td class="align-left">USA</td><td class="align-left">2Az</td><td class="align-left">32°58′42.19″N, 110°46′8.08″W</td><td class="align-left">3</td><td class="align-left">3</td></tr><tr class="valign-top"><td class="align-left">21</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Arizona</td><td class="align-left">USA</td><td class="align-left">4Az</td><td class="align-left">32°3′48.57″N, 110°17′3.42″W</td><td class="align-left">1</td><td class="align-left">1</td></tr><tr class="valign-top"><td class="align-left">22</td><td class="align-left"><em>S. eleagnifolium</em></td><td class="align-left">Arizona</td><td class="align-left">USA</td><td class="align-left">5Az</td><td class="align-left">32°1′54.69″N, 110°18′40.51″W</td><td class="align-left">2</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">23</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">Arizona</td><td class="align-left">USA</td><td class="align-left">6Az</td><td class="align-left">33°5′25.83″N, 112°2′1.26″W</td><td class="align-left">3</td><td class="align-left">2</td></tr><tr class="valign-top"><td class="align-left">24</td><td class="align-left"><em>S. eleagnifolium</em></td><td class="align-left">Texas</td><td class="align-left">USA</td><td class="align-left">Tx1</td><td class="align-left">33°52′55.49″N, 98°28′53.04″W</td><td class="align-left">2</td><td class="align-left">1</td></tr><tr class="valign-top"><td class="align-left">25</td><td class="align-left"><em>S. eleagnifolium</em></td><td class="align-left">Texas</td><td class="align-left">USA</td><td class="align-left">Tx2</td><td class="align-left">32°54′27.50″N, 97°34′50.64″W</td><td class="align-left">1</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">26</td><td class="align-left"><em>S. carolinense</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">4VA</td><td class="align-left">37°37′22.4″N, 77°59′13.1″W</td><td class="align-left">4</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">27</td><td class="align-left"><em>S. tuberosum</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">4VA</td><td class="align-left">37°37′11.4″N, 77°59′18.8″W</td><td class="align-left">2</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">28</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">5VA</td><td class="align-left">37°30′38.5″N, 77°42′12.9″W</td><td class="align-left">2</td><td class="align-left">4</td></tr><tr class="valign-top"><td class="align-left">29</td><td class="align-left"><em>S. carolinense</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">1VA</td><td class="align-left">37°39′49.8″N, 77°53′30.1″W</td><td class="align-left">1</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">30</td><td class="align-left"><em>S. carolinense</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">2VA</td><td class="align-left">37°32′28.6″N, 77°53′36.8″W</td><td class="align-left">4</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">31</td><td class="align-left"><em>S. carolinense</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">3VA</td><td class="align-left">37°32′56.3″N, 77°55′01.8″W</td><td class="align-left">5</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">32</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Virginia</td><td class="align-left">USA</td><td class="align-left">3VA</td><td class="align-left">37°31′31.5″N, 77°52′11.5″W</td><td class="align-left">2</td><td class="align-left">_____</td></tr><tr class="valign-top"><td class="align-left">33</td><td class="align-left"><em>D. ceratocaula</em></td><td class="align-left">Durango</td><td class="align-left">Mexico</td><td class="align-left">Map</td><td class="align-left">25°50′1.35″N, 103°50′55.46″W</td><td class="align-left">_____</td><td class="align-left">5</td></tr><tr class="valign-top"><td class="align-left">34</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Michoacán</td><td class="align-left">Mexico</td><td class="align-left">102</td><td class="align-left">19°40′46.19″N, 101°15′12.68″W</td><td class="align-left">_____</td><td class="align-left">3</td></tr><tr class="valign-top"><td class="align-left">35</td><td class="align-left"><em>Datura</em> sp.</td><td class="align-left">Baja California</td><td class="align-left">Mexico</td><td class="align-left">Uru</td><td class="align-left">31°34′03.2″N, 116°25′19.6″W</td><td class="align-left">_____</td><td class="align-left">6</td></tr><tr class="valign-top"><td class="align-left">36</td><td class="align-left"><em>Datura</em> sp.</td><td class="align-left">Baja California</td><td class="align-left">Mexico</td><td class="align-left">SAN</td><td class="align-left">32°06′55.8″N, 166°30′02.8″W</td><td class="align-left">_____</td><td class="align-left">7</td></tr><tr class="valign-top"><td class="align-left">37</td><td class="align-left"><em>Solanum</em> sp.</td><td class="align-left">Michoacán</td><td class="align-left">Mexico</td><td class="align-left">630</td><td class="align-left">19°41′9.87″N, 101°13′46.56″W</td><td class="align-left">_____</td><td class="align-left">3</td></tr><tr class="valign-top"><td class="align-left">38</td><td class="align-left"><em>D. quercifolia</em></td><td class="align-left">Guanajuato</td><td class="align-left">Mexico</td><td class="align-left">230</td><td class="align-left">20°25′20.6″N, 99°58′37.9″W</td><td class="align-left">_____</td><td class="align-left">18</td></tr><tr class="valign-top"><td class="align-left">39</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Edo. de Mexico</td><td class="align-left">Mexico</td><td class="align-left">Teo</td><td class="align-left">19°40′48.75″N, 98°52′26.51″W</td><td class="align-left"><sup>*</sup></td><td class="align-left">2</td></tr><tr class="valign-top"><td class="align-left">40</td><td class="align-left"><em>D. stramonium</em></td><td class="align-left">Guerrero</td><td class="align-left">Mexico</td><td class="align-left">RC</td><td class="align-left">17°32′27.34 N, 99°28′19.27″W</td><td class="align-left"><sup>*</sup></td><td class="align-left">1</td></tr><tr class="valign-top"><td class="align-left">41</td><td class="align-left"><em>D. discolor</em></td><td class="align-left">Oaxaca</td><td class="align-left">Mexico</td><td class="align-left">314</td><td class="align-left">16°47′11.61″N, 96°12′42.34″W</td><td class="align-left"><sup>*</sup></td><td class="align-left">14</td></tr><tr class="valign-top"><td class="align-left">42</td><td class="align-left"><em>D. discolor</em></td><td class="align-left">BajaCaliforniaN.</td><td class="align-left">Mexico</td><td class="align-left">828</td><td class="align-left">32°11′28.14″N, 114°55′19.63″W</td><td class="align-left"><sup>*</sup></td><td class="align-left">7</td></tr><tr class="valign-top"><td class="align-left">43</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">California</td><td class="align-left">USA</td><td class="align-left">16SD</td><td class="align-left">34°11′19.90″N, 116°26′4.89″W</td><td class="align-left"><sup>*</sup></td><td class="align-left">2</td></tr><tr class="valign-top"><td class="align-left">44</td><td class="align-left"><em>D. wrightii</em></td><td class="align-left">California</td><td class="align-left">USA</td><td class="align-left">18SD</td><td class="align-left">34°14′9.57″N, 116°26′23.43″W</td><td class="align-left"><sup>*</sup></td><td class="align-left">2</td></tr><tr class="valign-top"><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left"></td><td class="align-left">Total</td><td class="align-left">163</td><td class="align-left">139</td></tr></tbody></table></div></div><figure class="figure" id="f0005"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr1.jpg" alt="" aria-describedby="cn0005" height="534"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr1_lrg.jpg" target="__blank">Download high-res image (159KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr1.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0005"><p id="sp0020"><span class="label">Fig. 1</span>. <em>Trichobaris</em> sampling localities in USA and Mexico. Labels represent sample sites as indicated in <a name="bt0005" href="#t0005" class="workspace-trigger">Table 1</a>.</p></span></div></figure></div><p id="p0060">We do not use all collected specimens of <em>Trichobaris</em>
 for the analysis because immature stages cannot be used to identify 
species. Larvae were used only if adults were not found in some 
localities. Due to their small size, non-destructive DNA extraction was 
not possible. Then, we randomly selected subsets of insects for 
morphology and genetic analysis. We used 139 insects for morphological 
analysis (<a name="bt0005" href="#t0005" class="workspace-trigger">Table 1</a>),
 156 insects for phylogenetic analysis (documenting geographical 
distribution and plant host), and 189 COI sequences generated in this 
study for phylogeographic analyses. In addition, we included the 
sequences obtained from two previous studies (<em>T. soror</em>, <a name="bb0135" href="#b0135" class="workspace-trigger">De-la-Mora et al., 2015</a>; and <em>T. compacta</em>,
 De-la-Mora et al., in revision), resulting in a total of 844 COI 
sequences that comprise 198 COI haplotypes used to build a calibrated 
phylogeny and reconstruct the ancestral host plant.</p></section><section id="s0020"><h3 id="st040">2.2. Geometric morphometrics</h3><p id="p0065">A total of 139 insects were identified (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>)
 and photographed from frontal (rostrum), lateral (body shape) and 
dorsal (pronotum) views. We oriented the images identically: the frontal
 (with the head at the middle of the photograph), the lateral always was
 the left side (with the mesopleuron placed in the middle) and the 
dorsal view, with the scutellum oriented towards the bottom and the head
 towards the top). We use two different imaging methods: a SteREO 
Discovery V8 Zeiss stereomicroscope combined with AxioVision software 
for image processing (Instituto de Ecología, UNAM) and a Canon EOS 6D 
camera mounted on StackShot automated macro rail ® Cognisys Inc., 
combined with Helicon Focus image processing software (Museum of 
Comparative Zoology, Harvard University).</p><div><p id="p0070"><em>Landmark analysis</em> – We assessed three images per specimen for a total of 79 landmarks: 8 frontal, 39 lateral and 28 dorsal (<a name="bf0010" href="#f0010" class="workspace-trigger">Fig. 2</a>). These were quantified from a set of two-dimensional coordinates with tpsDIG 1.4 software (<a name="bb0305" href="#b0305" class="workspace-trigger">Rohlf, 2004</a>).
 To remove differences due to scale, orientation and location from 
landmark configurations, we used a minimum Procrustes distance 
criterion, implemented in MorphoJ (<a name="bb0200" href="#b0200" class="workspace-trigger">Klingenberg, 2011</a>),
 to produce a set of partial Procrustes superimpositions of the 
specimens. The coordinates obtained were transformed into relative warp 
scores to generate a W matrix. By means of principal component analysis 
(PCA), we analyzed shape variation among morphotypes. To maximize 
individual differences according to species, we performed a canonical 
variate analysis followed by a discriminant analysis among pairs of 
species. Both analyses were implemented in MorphoJ (<a name="bb0200" href="#b0200" class="workspace-trigger">Klingenberg, 2011</a>).
 Centroid size for frontal (rostrum), lateral (body shape) and dorsal 
(pronotum) views were plotted using R (R Development Core Team 2011).</p><figure class="figure" id="f0010"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr2.jpg" alt="" aria-describedby="cn0010" height="526"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr2_lrg.jpg" target="__blank">Download high-res image (848KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr2.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0010"><p id="sp0025"><span class="label">Fig. 2</span>. 
Landmarks (green dots) used to perform the morphogeometric analyses. 
(For interpretation of the references to colour in this figure legend, 
the reader is referred to the web version of this article.)</p></span></div></figure></div></section><section id="s0025"><h3 id="st045">2.3. DNA extraction, amplification and sequencing</h3><p id="p0075">Genomic
 DNA was extracted from whole insects using the DNeasy Blood &amp; 
Tissue kit (Qiagen). Four gene sequences were used to build the <em>Trichobaris</em>
 phylogeny: two mitochondrial genes (CO1 and 16S rRNA) and two nuclear 
genes (18S and 28S rRNA). The primers used were COA3107 and COS2183N (<a name="bb0325" href="#b0325" class="workspace-trigger">Sota et al., 2004</a>), LR-J-12887F and LR-N-13398R (<a name="bb0320" href="#b0320" class="workspace-trigger">Simon et al., 1994</a>), 28S and 28SR, and Insect18S-midF and Insect18S-midR (from <a name="bb0205" href="#b0205" class="workspace-trigger">Kobayashi et al., 2012</a>)
 for the COI, 16S, 28S and 18S genes, respectively. All PCR reactions 
were performed in volumes of 19.8 μL: 2 μL of 10x buffer, 1 µL of 20x 
MgCl<sub>2</sub>, 0.8 µL of 10 mM dNTPs, 1 µL of 10 mM forward primer, 
1 µL of 10 mM reverse primer, 1 µL of Taq Amplificasa®, 1 µL of 50 mM 
DNA and 12 µL of H<sub>2</sub>O. The amplification conditions were the 
same for all gene sequences except for 16S rRNA, whose annealing 
temperature was 50 °C (<a name="bb0325" href="#b0325" class="workspace-trigger">Sota et al., 2004</a>).
 An initial period of 5 min at 95 °C followed by 35 cycles of 1 min at 
94 °C, 1.2 min at 55 °C and 1 min at 72 °C, with a final extension of 
7 min at 72 °C. Samples were sequenced at the University of Washington’s
 High-Throughput Genomics Center (ABI 3730XL sequencer and the ABI Prism
 kit Cycle Sequencing Big Dye Terminator Ready Reaction; Applied 
Biosystems).</p><p id="p0080">Sequence processing was performed with 
Sequencher® 5.0 software (Gene Codes Corporation). The alignment was 
carried out by the FFT-NS-1 progressive alignment method, using the 
online version of the software MAFFT v7 (<a href="http://mafft.cbrc.jp/alignment/server/" target="_blank">http://mafft.cbrc.jp/alignment/server/</a>).</p></section><section id="s0030"><h3 id="st050">2.4. Estimation of genetic variability and phylogenetic analysis</h3><p id="p0085">The amount of genetic variability of COI, 16S, 18S and 28S sequences was estimated with the software DNAsp ver. 5.1 (<a name="bb0310" href="#b0310" class="workspace-trigger">Rozas et al., 2003</a>). The total number of haplotypes, mutations, segregating sites (<em>S</em>), nucleotide diversity (<em>π</em>), haplotype diversity (<em>h</em>) and theta (<em>θ</em>) using <a name="bb0260" href="#b0260" class="workspace-trigger">Nei’s (1987)</a> equations were estimated for the mitochondrial (COI and 16S) and nuclear (18S and 28S) sequences.</p><p id="p0090">Phylogenetic analyses were conducted for each locus separately, as well as for the concatenated matrix. <em>Sitophilus oryzae</em>
 (Curculionidae) was specified as the outgroup. Different codon 
positions were treated as separate partitions only for the COI sequence.
 Phylogenies were estimated using Bayesian Inference with the BEAST2 
software. v2.4.5. (<a name="bb0080" href="#b0080" class="workspace-trigger">Bouckaert et al., 2014</a>).
 The analysis was executed using 100,000,000 MCMC, 10,000 “burn in” with
 the GTR + I molecular replacement model using a fixed substitution rate
 of 1 substitution/site/unit time. The MCMC convergence of the chains 
was evaluated considering the effective sample size scores, all above 
100, using Tracer v. 1.6 (<a name="bb0295" href="#b0295" class="workspace-trigger">Rambaut et al., 2014</a>), and the trees were summarized with TreeAnnotator v2.1.2 (<a name="bb0285" href="#b0285" class="workspace-trigger">Rambaut and Drummond, 2014</a>). Finally, the phylogenetic tree was visualized in FigTree v 1.4 (<a name="bb0290" href="#b0290" class="workspace-trigger">Rambaut, 2012</a>).</p></section><section id="s0035"><h3 id="st055">2.5. Ancestral plant tissue and rostrum evolution</h3><p id="p0095">Second
 and third phylogenies were built using just one sequence per species 
(including the four markers), randomly chosen from the previous 
phylogenetic clustering (defined in geometric morphometric analysis). 
For the reconstruction of the ancestral plant tissue used by <em>Trichobaris</em>,
 all species were included, except for the phylogeny built for rostrum 
evolution, where only species with geometric morphometric information 
available for the rostrum of both males and females were used. Both 
phylogenies were estimated using Bayesian Inference with the BEAST2 
software v2.4.5. (<a name="bb0080" href="#b0080" class="workspace-trigger">Bouckaert et al., 2014</a>).
 The analysis was executed using 100 000 000 MCMC, 10 000 “burn in” with
 the GTR + I molecular replacement model using a fixed substitution rate
 of 1 substitution/site/unit time. Here, the MCMC convergence of the 
chains was also evaluated considering the effective sample size scores 
(ESS), all above 100, using Tracer v. 1.6 (<a name="bb0295" href="#b0295" class="workspace-trigger">Rambaut et al., 2014</a>), and the trees were summarized with TreeAnnotator v2.1.2 (<a name="bb0285" href="#b0285" class="workspace-trigger">Rambaut and Drummond, 2014</a>).</p><p id="p0100">To reconstruct the ancestral plant tissue and the evolution of rostrum of <em>Trichobaris</em>, we reconstructed ancestral states with maximum likelihood using the “phytools” package (<a name="bb0300" href="#b0300" class="workspace-trigger">Revell, 2012</a>),
 implemented in R (R Development Core Team 2011). The plant tissue was 
coded as a discrete characteristic (F = fruit, S = stem and U = for 
uncertainty of the tissue for the outgroup). We used the ER model (equal
 rates) that assumes the same probability for rate index matrix and the 
ARD model (all rates different) that assumes the different probability. A
 likelihood ratio test (LRT) was conducted to test for the best fit of 
the two models to the data. For the rostrum evolution, we use the mean 
centroid size per species, per sex, and coded it as a continuous 
characteristic. Here, we apply a Brownian motion model for the evolution
 of the rostrum. The phylogenetic signal of the rostrum per sex was 
estimated using the <em>K</em>-statistic (<a name="bb0065" href="#b0065" class="workspace-trigger">Blomberg et al., 2003</a>) and λ (<a name="bb0275" href="#b0275" class="workspace-trigger">Pagel, 1999</a>), both with nsim = 1 000 in “phytools” (<a name="bb0300" href="#b0300" class="workspace-trigger">Revell, 2012</a>).</p></section><section id="s0040"><h3 id="st060">2.6. COI haplotype network and host plant</h3><p id="p0105">Using the gene variation at the COI gene sequence from two previous studies (<a name="bb0135" href="#b0135" class="workspace-trigger">De-la-Mora et al., 2015</a>
 and De-la-Mora et al., submitted) and the ones obtained in this study, 
we compared the measures of genetic diversity within and between species
 with the DNAsp software (<a name="bb0310" href="#b0310" class="workspace-trigger">Rozas et al., 2003</a>).
 The relationship between haplotypes and their host-plant distribution 
was explored with haplotype networks built in Network ver. 4.6.1.1 
software (<a name="bb0030" href="#b0030" class="workspace-trigger">Bandelt et al., 1999</a>) through a median-joining algorithm.</p></section><section id="s0045"><h3 id="st065">2.7. COI haplotype phylogeny</h3><p id="p0110">Using the COI haplotypes of all <em>Trichobaris</em> species, we built the haplotype tree with Bayesian Inference using BEAST software. ver. 1.4.7 (<a name="bb0145" href="#b0145" class="workspace-trigger">Drummond and Rambaut, 2007</a>).
 The analysis was executed using 10,000,000 MCMC, 1000 “burn in” with 
the GTR + I molecular replacement model. The phylogeny was calibrated 
with a 2% divergence per million years, previously reported for COI in 
coleopterans (<a name="bb0255" href="#b0255" class="workspace-trigger">Nakamine and Takeda, 2008</a>).
 The MCMC convergence of the chains was evaluated considering the 
effective sample size scores (ESS), all above 100, using Tracer v. 1.6 (<a name="bb0295" href="#b0295" class="workspace-trigger">Rambaut et al., 2014</a>), and the trees were summarized with TreeAnnotator v2.1.2 (<a name="bb0285" href="#b0285" class="workspace-trigger">Rambaut and Drummond, 2014</a>). The phylogenetic tree was visualized in FigTree v 1.4 (<a name="bb0290" href="#b0290" class="workspace-trigger">Rambaut, 2012</a>).</p></section><section id="s0050"><h3 id="st070">2.8. Ancestral host plant reconstruction analysis</h3><p id="p0115">The gene CO1 has been effective in recognizing insect haplotypes associated with host plants (<a name="bb0195" href="#b0195" class="workspace-trigger">Jurado-Rivera et al., 2009</a>).
 Here, we used the phylogenetic tree built from COI haplotypes instead 
of the species phylogeny because the former has greater resolution, 
allowing for comparison of terminals to host plants. We coded ambiguous 
haplotypes (present in two or more host plants) present only in the most
 abundant host plant. Ancestral host utilization was estimated with 
maximum likelihood using the “phytools” package (<a name="bb0300" href="#b0300" class="workspace-trigger">Revell, 2012</a>) implemented in R (R Development Core Team 2011). The eleven host plants occupied by <em>Trichobaris</em>
 species were considered a discrete character. The host plant of the 
outgroup species was required for the analysis, although this is an 
arbitrary assignment. We used a continuous-time Markov chain model (Mk 
model) (<a name="bb0330" href="#b0330" class="workspace-trigger">Suchard et al., 2001</a>)
 for trait evolution, assuming all transitions in the character state 
with the same probability. We did not attempt to estimate the 
phylogenetic signal for discrete characters (some methods exist but for 
binary characters; see <a name="bb0160" href="#b0160" class="workspace-trigger">Fritz and Purvis, 2010</a>).
 For ancestral host plant reconstruction, the geographic distribution of
 species using different hosts was also taken into account through the 
phylogeny of haplotypes. Different models of reconstruction were not 
tested because there is no empirical evidence or theoretical basis to 
suppose differential transition rates between states (host plants); for 
instance, some species of insects would follow secondary chemical 
compounds present in one species of plant and not in other. 
Additionally, it is methodologically complex to analyze 12 discrete 
states because of the large number of parameters involved.</p></section></section><section id="s0055"><h2 id="st075">3. Results</h2><section id="s0060"><h3 id="st080">3.1. Species and geometric morphometric analysis</h3><div><p id="p0120">From sampled insects and according to Barber’s descriptions, we identified <em>T. soror, T. major, T. compacta, T. mucorea, T. mucorea</em> var. <em>striatula</em> Casey 1920<em>, T. texana, T. cylindrica</em> and <em>T. trinotata.</em> A total of nine <em>Trichobaris</em> species were included in the geometric morphometric analysis, with eight species being distinguishable: <em>T. soror, T. major, T. mucorea, T. mucorea</em> var<em>. striatula, T. texana, T. cylindrica, T. trinotata</em> and <em>T. compacta</em> (<a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>)<em>.</em> Missing from Barber’s descriptions were <em>T. pueblana, T. compacta</em> var. <em>retrusa</em> and <em>T. bridwelli</em>. We found sexual dimorphism in the rostrum; <em>T. soror, T. major</em> and <em>T. trinotata</em> have females with longer and thinner rostrums among the other females of <em>Trichobaris</em> species (<a name="bf0020" href="#f0020" class="workspace-trigger">Fig. 4</a>). Here, <em>T. insolita</em> and <em>T. cylindrica</em> are not shown due to sample size (&lt;2); <a name="bf0025" href="#f0025" class="workspace-trigger">Figs. 5</a> and 1S Supplementary Material).</p><figure class="figure" id="f0015"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr3.jpg" alt="" aria-describedby="cn0015" height="550"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr3_lrg.jpg" target="__blank">Download high-res image (309KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr3.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0015"><p id="sp0030"><span class="label">Fig. 3</span>. Analysis of Canonical Variance of the genus <em>Trichobaris</em>,
 obtained by 75 landmarks from three photographs per specimen. Each 
individual is represented by a dot, coloured according to the species to
 which it belongs. The photograph is representative of each species. The
 axes are the canonical variate 1 and 2 that represent the changes in 
shape.</p></span></div></figure><figure class="figure" id="f0020"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr4.jpg" alt="" aria-describedby="cn0020" height="536"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr4_lrg.jpg" target="__blank">Download high-res image (355KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr4.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0020"><p id="sp0035"><span class="label">Fig. 4</span>. Analysis of Canonical Variance of the rostrum of species of genus <em>Trichobaris</em>,
 obtained by eight landmarks per specimen. Males (M) and females (F) of 
each species are represented by the same colour darkest and lightest, 
respectively. The axes are the canonical variate 1 and 2 that represent 
the changes in shape represented on the transformation gif outside of 
the graphic.</p></span></div></figure><figure class="figure" id="f0025"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr5.jpg" alt="" aria-describedby="cn0025" height="966"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr5_lrg.jpg" target="__blank">Download high-res image (337KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr5.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0025"><p id="sp0040"><span class="label">Fig. 5</span>. Centroid size of <em>Trichobaris</em> species. Panels: Upper) Dorsal view or <em>pronotum</em>. Middle) Lateral view or body shape. Lower) Rostrum by sex of <em>Trichobaris</em> species. F = females, M = Males.</p></span></div></figure></div></section><section id="s0065"><h3 id="st085">3.2. Estimation of genetic variability and phylogenetic relationships</h3><p id="p0125">We
 sequenced 663 bp of COI, 454 bp of 16S, 429 bp of 18S and 771 bp of 28S
 from 156 randomly selected insects. There was substantial variation in 
resolution among the four gene regions. For COI, we found 59 haplotypes,
 with <em>h</em> = 0.874, <em>π</em> = 0.052, <em>θ</em> = 0.064 and <em>S</em> = 26. For 16S, 26 haplotypes were found, with <em>h</em> = 0.762, π = 0.025, <em>θ</em> = 0.06 and <em>S</em> = 6. For the nuclear sequences 18S and 28S, we found 15 (<em>h</em> = 0.593, <em>π</em> = 0.005, <em>θ</em> = 0.020 and <em>S</em> = 3) and 18 (<em>h</em> = 0.518, <em>π</em> = 0.025, <em>θ </em>= 0.062 and <em>S</em> = 10)
 consensus sequences, respectively. GenBank accession numbers: 
KY984501-KY984658 for 28S, KY984659-KY984816 for 18S, KY984817-KY984974 
for 16S and KX359724-KX359774 for COI.</p><div><p id="p0130">The Bayesian phylogeny provides an estimate of the evolutionary relationships among <em>Trichobaris</em> species (<a name="bf0030" href="#f0030" class="workspace-trigger">Fig. 6</a>;
 for posterior probability of parameters and tracer, see Fig. S2 
Supplementary Material). Six species agree with the morphological 
groupings and species reported. <em>Trichobaris major</em> and <em>T. soror</em>
 belong to the same morphological group and are genetically 
indistinguishable. Because larvae were used for this analysis, 
identification of some species required further confirmation, namely, <em>T. pellicea</em>,
 which was identified on the basis of its distribution and host plant. 
In general, the phylogenetic tree shows high posterior probability 
support for eight <em>Trichobaris</em> species.</p><figure class="figure" id="f0030"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr6.jpg" alt="" aria-describedby="cn0030" height="460"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr6_lrg.jpg" target="__blank">Download high-res image (184KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr6.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0030"><p id="sp0045"><span class="label">Fig. 6</span>. Bayesian phylogeny of <em>Trichobaris</em>. Phylogeny of <em>Trichobaris</em>,
 estimated by Bayesian Inference using 18S and 28S nrDNA and the 
mitochondrial 16S and COI. The morphological clustering from canonical 
variance analysis is indicated with the circles after species names. (?)
 indicates that the species identity could not be corroborated by 
morphology. Red and green circles on tips are observed values of plant 
tissue used by weevils species to oviposit and circles at nodes are the 
ancestral states reconstructed by maximum likelihood. (For 
interpretation of the references to colour in this figure legend, the 
reader is referred to the web version of this article.)</p></span></div></figure></div></section><section id="s0070"><h3 id="st090">3.3. Host plant tissue and weevil’s rostrum evolution</h3><div><p id="p0135">We reconstructed the host plant tissue used by weevils on the phylogeny of <em>Trichobaris</em> (<a name="bf0030" href="#f0030" class="workspace-trigger">Fig. 6</a>;
 for posterior probability of parameters and tracer see Fig. S3 
Supplementary Material). Two clades use different plant tissues to 
oviposit: Clade 1 (<em>T. cylindrica</em> and <em>T. texana)</em> uses the stem, while clade 2 (<em>T. pellicea, T. mucorea</em> var. <em>striatula, T. compacta, T. mucorea, T. soror</em> and <em>T. trinotata</em>) uses the fruit, except <em>T. trinotata</em>. Models ER and ARD for the rate of transitions (LRT = 4.20: critical value = 11.7 for <em>p</em> = 0.05) were not significantly different. Rostrum varies from 1.32 to 2.24 mm for females and from 1.33 to 1.66 for males (<a name="bf0035" href="#f0035" class="workspace-trigger">Fig. 7</a>;
 for posterior probability of parameters and tracer see Fig. S4 
Supplementary Material). The phylogenetic signal for this trait was low 
in both sexes (females: <em>K</em>-statistic = 0.1382412, <em>p</em>-value = 0.373; λ = 6.843901e-05, <em>p</em>-value = 1; males: <em>K</em>-statistic = 0.2773729, <em>p-</em>value = 0.477; λ = 6.843901e-05, <em>p-</em>value = 1).</p><figure class="figure" id="f0035"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr7.jpg" alt="" aria-describedby="cn0035" height="339"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr7_lrg.jpg" target="__blank">Download high-res image (179KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr7.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0035"><p id="sp0050"><span class="label">Fig. 7</span>. Colour gradient projection of rostrum, observed and reconstructed size, onto the phylogeny of <em>Trichobaris</em> species. Estimated by maximum likelihood under a Brownian motion model of evolution.</p></span></div></figure></div></section><section id="s0075"><h3 id="st095">3.4. Phylogeographic analysis</h3><div><p id="p0140">We obtained eight haplotype networks of 189 haplotypes (<a name="bf0040" href="#f0040" class="workspace-trigger">Fig. 8</a>) using the variation of only the COI gene sequence (<a name="bt0010" href="#t0010" class="workspace-trigger">Table 2</a>) of 844 insects, including the samples used for the estimation of the phylogeny (see above).</p><figure class="figure" id="f0040"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr8.jpg" alt="" aria-describedby="cn0040" height="808"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr8_lrg.jpg" target="__blank">Download high-res image (573KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr8.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0040"><p id="sp0055"><span class="label">Fig. 8</span>. Haplotype network of <em>Trichobaris</em>
 species estimated using the COI gene (663 bp) using the Median joining 
algorithm. Black dots represent vectors and the numbers on branches 
indicate mutational steps (see Supplementary Material for a more 
detailed view, in Figs. S6–S11). <em>Trichobaris pellicea</em>? Indicates that this species was not corroborated by morphology.</p></span></div></figure><div class="tables frame-topbot rowsep-0 colsep-0" id="t0010"><div class="captions"><span id="cn0055"><p id="sp0070"><span class="label">Table 2</span>. Genetic diversity for <em>Trichobaris</em> species using 663 bp of COI gene.</p></span></div><div class="groups"><table><thead><tr class="rowsep-1 valign-top"><th scope="col" class="align-left">Species</th><th scope="col" class="align-left"># insects</th><th scope="col" class="align-left"># haplotypes</th><th scope="col" class="align-left">h</th></tr></thead><tbody><tr class="valign-top"><td class="align-left"><em>T. soror</em></td><td class="align-left">469</td><td class="align-left">99</td><td class="align-left">0.785</td></tr><tr class="valign-top"><td class="align-left"><em>T. trinotata</em></td><td class="align-left">21</td><td class="align-left">4</td><td class="align-left">0.271</td></tr><tr class="valign-top"><td class="align-left"><em>T. mucorea</em></td><td class="align-left">67</td><td class="align-left">16</td><td class="align-left">0.701</td></tr><tr class="valign-top"><td class="align-left"><em>T. compacta</em></td><td class="align-left">232</td><td class="align-left">49</td><td class="align-left">0.709</td></tr><tr class="valign-top"><td class="align-left"><em>T. mucorea var. striatula</em></td><td class="align-left">23</td><td class="align-left">13</td><td class="align-left">0.932</td></tr><tr class="valign-top"><td class="align-left"><em>T. texana</em></td><td class="align-left">7</td><td class="align-left">3</td><td class="align-left">0.761</td></tr><tr class="valign-top"><td class="align-left"><em>T. cylindrica</em></td><td class="align-left">4</td><td class="align-left">3</td><td class="align-left">0.833</td></tr><tr class="valign-top"><td class="align-left"><em>T. pellicea</em> (?)</td><td class="align-left">3</td><td class="align-left">2</td><td class="align-left">0.667</td></tr><tr class="valign-top"><td class="align-left">Total</td><td class="align-left">826</td><td class="align-left">189</td><td class="align-left">0.905</td></tr></tbody></table></div></div></div><p id="p0145">The haplotype network of <em>T. mucorea</em> (n = 67) shows 16 haplotypes (Fig. 6S Supplementary Material), most of them distributed only on <em>D. stramonium,</em> three on <em>D. inoxia</em> and one on three host plants: <em>D. pruinosa, D. discolor</em> and <em>D. stramonium</em> (Fig. S6 Supplementary Material). The haplotype network of <em>T. soror</em> (n = 469) shows 99 haplotypes, mostly on <em>D. stramonium,</em> but nine haplotypes are also found on <em>D. quercifolia.</em> The low-frequency haplotypes So92, So93 and So94 were found on <em>D. inoxia, D. pruinosa</em> and <em>Solanum carolinense</em> (Fig. S7 Supplementary Material). The haplotype network of <em>T. trinotata</em> (n = 21) shows four haplotypes with a shared distribution on <em>S. carolinense, S. tuberosum</em> and <em>D. stramonium</em> (Fig. S8 Supplementary Material). The haplotype network of <em>T. compacta</em> (n = 232) shows 49 haplotypes, most of which were sampled on <em>D. wrightii.</em> Five haplotypes are distributed on <em>D. reburra</em>, eight on <em>D. discolor</em>, two on <em>D. pruinosa</em> and two in <em>D. inoxia</em> (Fig. S9 Supplementary Material).</p><p id="p0150">The haplotype network of <em>T. mucorea</em> var. <em>striatula</em> (n = 23) shows 13 haplotypes mainly distributed on <em>D. wrightii</em> and four haplotypes on <em>D. discolor</em> (Fig. S10 Supplementary Material). The haplotype network of <em>T. texana</em> (n = 7) shows three haplotypes distributed on <em>S. rostratum</em> (Fig. S11 Supplementary Material). The haplotype network of <em>T. cylindrica</em> (n = 4) shows three haplotypes distributed on <em>S. eleagnifolium</em> (Fig. S11 Supplementary Material)<em>.</em> Finally, <em>T. pellicea</em> n = 3 shows two haplotypes distributed on <em>D. stramonium</em> (Fig. S11 Supplementary Material).</p></section><section id="s0080"><h3 id="st100">3.5. Haplotype calibrated phylogeny</h3><div><p id="p0155">Most clades that were inferred with high support based on the four loci (<a name="bf0030" href="#f0030" class="workspace-trigger">Fig. 6</a>) were also inferred and strongly supported by the COI haplotype phylogeny (<a name="bf0045" href="#f0045" class="workspace-trigger">Fig. 9</a>; for posterior probability of parameters and tracer see Fig. S5 Supplementary Material), except for <em>T. pellicea</em>, which appears in a different position. The time-calibrated haplotype phylogeny shows that most <em>Trichobaris</em> species are no older than 6.6 (±1.5) million years ago (Ma). Diversification of <em>T. soror, T. trinotata, T. mucorea, T. compacta</em> and <em>T. mucorea</em> var. <em>striatula</em> haplotypes is very recent (less than 0.5 Ma) (<a name="bf0045" href="#f0045" class="workspace-trigger">Fig. 9</a>).</p><figure class="figure" id="f0045"><span id="download-image"><img src="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr9.jpg" alt="" aria-describedby="cn0045" height="738"><ol class="links-for-figure"><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr9_lrg.jpg" target="__blank">Download high-res image (340KB)</a></li><li><a href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-gr9.jpg" target="__blank">Download full-size image</a></li></ol></span><div class="captions"><span id="cn0045"><p id="sp0060"><span class="label">Fig. 9</span>. Calibrated COI haplotype tree of <em>Trichobaris</em>
 species. Node bars represent the variation on age estimation. Blue 
numbers on branches indicate node support (posterior probability). The 
estimation of the ancestral host plant was carried out in each node of <em>Trichobaris</em>
 haplotype tree using maximum likelihood; the pie charts illustrate the 
relative likelihood of each of 13 possible host plants. The coloured box
 at the tips are the plants where the haplotype was collected; each <em>Trichobaris</em>
 species is indicated by lateral bars at right side. (For interpretation
 of the references to colour in this figure legend, the reader is 
referred to the web version of this article.)</p></span></div></figure></div></section><section id="s0085"><h3 id="st105">3.6. Estimation of ancestral host plant species</h3><p id="p0160">We estimated the ancestral host plant based on the COI haplotype phylogeny of <em>Trichobaris</em> species (<a name="bf0045" href="#f0045" class="workspace-trigger">Fig. 9</a>).
 Here, we indicated the ancestral host at eleven nodes from (0–10), 
whose estimated likelihood values were not conclusive for nodes 0, 1, 2 
and 4. Node 3 suggests with a 0.528 likelihood that <em>S. eleagnifolium</em> could be the ancestral host plant of <em>T. texana</em> and <em>T. cylindrica</em>. Likelihood values of 0.318 and 0.982 from nodes 5 and 7, respectively, support the hypothesis that <em>D. wrightii</em> was the ancestral host plant of <em>T. compacta</em> and <em>T. mucorea</em> var<em>. striatula</em>. Finally, nodes 6, 8, 9 and 10 suggest that <em>D. stramonium</em> was the ancestral host plant of <em>T. soror, T. trinotata</em> and <em>T. mucorea</em> (0.272, 0.784, 0.862 and 0.983, respectively, at each node).</p></section></section><section id="s0090"><h2 id="st110">4. Discussion</h2><p id="p0165">This study constitutes the first integrative analysis, using morphological, phylogenetic and phylogeographic approaches, of <em>Trichobaris</em>
 species to investigate their association with Solanaceae plants. Our 
results provide relevant insights to understand the evolution of <em>Trichobaris</em> and its relationships with host plants.</p><section id="s0095"><h3 id="st115">4.1. Trichobaris species and Barber’s key</h3><p id="p0170">Barber’s monographic account of <em>Trichobaris</em> (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>) recognizes female rostrum length, body shape and male genitalia as critical morphological characters to distinguish <em>Trichobaris</em> species. In Barber’s key, the female rostrum allows us to distinguish <em>T. soror, T. pueblana</em> and <em>T. major</em> from all other <em>Trichobaris</em> species. In our study, we found that male and female rostrum shape is an important character across <em>Trichobaris</em> species. The rostrum has been identified as a key innovation in the evolution of weevils (<a name="bb0265" href="#b0265" class="workspace-trigger">Oberprieler et al., 2007</a>), and even at a lower taxonomic level, the rostrum is an important trait in the evolution of this genus (<a name="bf0035" href="#f0035" class="workspace-trigger">Fig. 7</a>).</p><p id="p0175">Evidence indicates that <em>T. soror, T. major, T. trinotata, T. compacta, T. texana</em> and <em>T. cylindrica</em> differ in body shape from each other (<a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>). In species such as <em>T. insolita</em> and <em>T. championi</em>, the clustering of individuals had no statistical significance, although this might be a consequence of small sample sizes.</p><p id="p0180">We found that <em>T. soror</em> and <em>T. major</em> are morphologically distinguishable (<a name="bf0030" href="#f0030" class="workspace-trigger">Fig. 6</a>), although they do not show differences in COI haplotypes (<a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>). In contrast, <em>T. mucorea</em> and <em>T. mucorea</em> var<em>. striatula</em> show close morphological clustering (<a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>), but they are genetically very distant, similar to cryptic species; in fact, <em>T. mucorea</em> is closely related to <em>T. soror</em>, whereas <em>T. mucorea</em> var<em>. striatula</em> is closer to <em>T. compacta</em> (<a name="bf0015" href="#f0015" class="workspace-trigger">Fig. 3</a>).</p><p id="p0185">Barber’s species of <em>Trichobaris</em> not included in our phylogeny were <em>T. pueblana, T. championi, T. bridwelli</em> and <em>T. insolita.</em> Although we sampled several populations in the state of Puebla and in the reported host plant <em>(D. stramonium),</em> we did not find <em>T. pueblana</em>. <em>Trichobaris championi</em> is a crop pest of tomatillo (<em>Physalis ixocarpa</em>) in Mexico (<a name="bb0090" href="#b0090" class="workspace-trigger">Calyecac-Cortero et al., 2004</a>). Instead, the weevils collected in wild <em>Physalis</em> species belonged to <em>T. mucorea,</em> which has also been reported in the stems of <em>Nicotiana attenuata</em> (<a name="bb0140" href="#b0140" class="workspace-trigger">Diezel et al., 2011</a>). <em>Trichobaris bridwelli</em> was reported by Barber as the single weevil species that feeds upon the fruits of <em>D. stramonium</em> in the USA (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>); however, the species of <em>Trichobaris</em> that we collected on <em>D. stramonium</em> was <em>T. soror</em> (see below). Finally, a single <em>T. insolita</em> was reported from a single locality in Florida (USA) on a patch of plants of <em>Physalis sp</em>. (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>). We were not able to corroborate that finding.</p></section><section id="s0100"><h3 id="st120">4.2. Phylogenetic patterns and rostrum evolution</h3><p id="p0190">In <em>Curculio</em> species (<a name="bb0340" href="#b0340" class="workspace-trigger">Toju and Sota, 2009</a>, <a name="bb0130" href="#b0130" class="workspace-trigger">Davis, 2014</a>),
 an evolutionary trend to increase rostrum length in specialized 
relationships with their host plants to drill the fruits and oviposit 
has been documented (e.g., <a name="bb0345" href="#b0345" class="workspace-trigger">Toju, 2009</a>). In <em>Trichobaris</em>, we observed a tendency to increase rostrum size among weevil species that use fruits, instead of stems, to oviposit (<a name="bf0030" href="#f0030" class="workspace-trigger">Fig. 6</a>) in comparison with the ancestral clade (<em>T. texana</em> and <em>T. cylindrica</em>);
 however, this remains speculative at the moment. When the phylogenetic 
signal is high, closely related species exhibit similar traits, and this
 biological similarity decreases as the evolutionary distance between 
species increases. In our study, rostrum size had a low phylogenetic 
signal, perhaps due to the bias induced by small branches near the tips 
(e.g., <em>T. trinotata</em> and <em>T. soror</em>).</p></section><section id="s0105"><h3 id="st125">4.3. Phylogeographic patterns and host plant associations</h3><p id="p0195">Although our sampling of <em>Trichobaris</em> did not cover all the geographic distribution regions reported for the species in the USA’s midwest (<em>T. texana, T. bridwelli</em> and <em>T. trinotata;</em><a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>, <a name="bb0115" href="#b0115" class="workspace-trigger">Cuda and Burke, 1984</a>), we report the host plant of these species. We found that the insects collected on <em>D. stramonium</em> in the USA (locality 1VA, <a name="bf0005" href="#f0005" class="workspace-trigger">Fig. 1</a> and <a name="bt0005" href="#t0005" class="workspace-trigger">Table 1</a>) vary only in 2 or 3 mutational steps in the haplotype network of <em>T. soror</em> (haplotypes: So93, So94, So95 and So96; Fig. S7 Supplementary Material). Additionally, <em>T. bridwelli</em> is reported as the only weevil species associated with <em>D. stramonium</em> in the USA (<a name="bb0035" href="#b0035" class="workspace-trigger">Barber, 1935</a>). Thus, this suggests that <em>T. soror</em> is also distributed up to the western USA. The reported potato pest, <em>T. trinotata,</em> is associated with <em>S. tuberosum</em> and <em>S. carolinense</em> (<a name="bb0120" href="#b0120" class="workspace-trigger">Cuda and Burke, 1986</a>, <a name="bb0350" href="#b0350" class="workspace-trigger">Wise, 2007</a>), and our study supports these observations (Fig. S8 Supplementary Material).</p><p id="p0200">Mapping
 the host plant onto the haplotype network and the ancestral host 
estimation using the phylogeny reveal several relevant aspects (<a name="bf0040" href="#f0040" class="workspace-trigger">Fig. 8</a>, <a name="bf0045" href="#f0045" class="workspace-trigger">Fig. 9</a>). First, the use of <em>Solanum</em> plants by <em>Trichobaris</em> weevils could be older than the occupation of <em>Datura</em>, except for <em>S. carolinense</em> and <em>S. tuberosum</em>, whose estimated ancestral host is <em>D. stramonium</em>. Second, the use of different species of <em>Datura</em> by <em>T. compacta</em> may be more recent than its interaction with <em>D. wrightii</em>. Almost all beetles collected in <em>D. stramonium</em> were <em>T. soror</em>, which also uses <em>D. quercifolia. T. mucorea</em> is associated with <em>D. stramonium</em>, and <em>T. mucorea</em> var<em>. striatula</em> is associated with <em>D. wrightii</em> but has a clade within <em>D. discolor</em>. Inconclusive cases, possibly due to sample size, are <em>T. trinotata, T. texana, T. cylindrica</em> and <em>T. pellicea</em>. This means that historical associations and recent colonizations are both critical in explaining the relationship of <em>Trichobaris</em> with its host plants.</p><p id="p0205">A possible directionality on the evolution of resource use in weevils has been proposed (<a name="bb0230" href="#b0230" class="workspace-trigger">Malvardi et al., 2002</a>). For <em>Trichobaris</em> species, we found that the use of stems and fruits is bidirectional. In the case of <em>T. trinotata</em>, we found that this species develops in the stems of both <em>S. carolinense</em> and <em>S. tuberosum</em>. It is likely that its ancestor developed in <em>Datura</em> fruits (<a name="bf0030" href="#f0030" class="workspace-trigger">Fig. 6</a>).
 It has been suggested that host shifts in closely related insect 
species are more correlated with host plant defensive secondary 
chemistry than with plant phylogeny (<a name="bb0165" href="#b0165" class="workspace-trigger">Futuyma and Agrawal, 2009</a>: <a name="bb0055" href="#b0055" class="workspace-trigger">Becerra, 2015</a>). In the case of <em>T. soror</em>, it is known that there is spatial variation in the levels of infestation of <em>D. stramonium</em> (<a name="bb0070" href="#b0070" class="workspace-trigger">Borbolla, 2015</a>), likely associated with alkaloid production, namely, atropine and scopolamine (<a name="bb0245" href="#b0245" class="workspace-trigger">Miranda-Pérez et al., 2016</a>). Future studies are needed to investigate the role of secondary compound chemistry in the association of <em>Trichobaris</em> with its plant hosts.</p></section></section><section id="s0110"><h2 id="st130">5. Conclusions</h2><p id="p0210">Here, we present the phylogenetic relationships among <em>Trichobaris</em> species based on four gene sequences. Additionally, we used a geometric morphometric approach for <em>Trichobaris</em> species to assess their concordance with phylogenetic clades and to test the hypothesis that <em>T. texana</em> and <em>T. cylindrica</em>
 have weak sexual dimorphism in rostrum size and use the stem of 
Solanaceae, probably since the origin of the genus (6 ± 1.5 Ma), whereas
 other, more recent, species of <em>Trichobaris</em> display sexual dimorphism in rostrum size (<em>T. soror</em> and <em>T. trinotata</em>, mainly) and use the fruits of <em>Datura</em> species (with only one possible reversion: <em>T. trinotata</em>, to the stem of <em>Solanum</em>).</p><p id="p0215"><em>Trichobaris soror</em> and <em>T. major</em>
 belong to different morphological groups but to the same clade in the 
phylogeny, suggesting a recent divergence (less than 1 Ma). In contrast,
 <em>T. mucorea</em> and <em>T. mucorea</em> var. <em>striatula</em> are
 morphologically close but belong to different clades in the phylogeny, 
suggesting that they constitute different species. The latter warrants a
 formal description.</p><p id="p0220">The haplotype network and the calibrated phylogeny built with the COI gene sequence suggest that most species of <em>Trichobaris</em> have originated recently in interaction with their host plants. Thus, our results indicate that <em>Trichobaris</em> species diverged very recently (less than 5 Ma), mainly associated with <em>Solanum</em> and <em>Datura</em> species, and that such interactions perhaps spurred the evolution of the rostrum to oviposit in different tissues.</p></section></div><section id="s0115"><h2 id="st135">Acknowledgments</h2><p id="p0225">We
 would like to thank R. Tapia-López for her technical support. We are 
grateful to professor J. Hayden, who helped us collect insects, and to 
sisters from Belmead land for facilitating sampling in Virginia. M. 
De-la-Mora thanks O. Barrera-Moreno for their assistance during field 
sampling. Our sincere thanks also go to S. Il Kim and R. Pérez-de la 
Fuente for their assistance in taking photographs, at Harvard’s Museum 
of Comparative Zoology, and to R. Pérez Ishiwara for the same but at 
Instituto de Ecología, UNAM. Funding was provided by <span id="gp005">PAPIIT-UNAM</span> (<a href="#gp005">IN-212214</a>), and <span id="gp010">CONACyT</span> grant <a href="#gp010">81490</a>
 “Evolución de la defensa en plantas contra sus enemigos naturales”. M. 
De-la-Mora acknowledges the scholarship from CONACyT for graduate 
studies. This contribution constitutes a partial fulfilment of the Ph. 
D. Thesis of M. De-la-Mora.</p></section><section id="s0120"><h2 id="st140">Conflict of interest</h2><p id="p0230">None.</p></section><div class="Appendices"><section id="s0130"><h2 id="st150">Appendix A. Supplementary material</h2><p id="p0240"><span class="display"><div class="e-component e-component-mmc1" id="m0005"><div class="article-attachment"><a class="icon-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-mmc1.docx" title="Download Word document (7MB)" target="_blank"><svg focusable="false" viewBox="0 0 94 128" width="17.625" height="24" class="icon icon-text-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-15.99-2.01-4e1 -17.64-32.36h-56.4zm0 1e1h46.4v22.36 17.64 4e1 2.01 5.99h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06zm-13.6 38v1e1h5e1v-1e1h-5e1zm0 2e1v1e1h5e1v-1e1h-5e1z"></path></svg></a><a class="download-link" target="_blank" download="" title="Download Word document (7MB)" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-mmc1.docx">Download Word document (7MB)</a><a class="help-link right" title="Help with docx files (Opens in new window)" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank">Help with docx files</a></div><div class="captions"><span><p><span class="label">Supplementary data 1</span>. </p></span></div></div></span><span class="display"><div class="e-component e-component-mmc2" id="m0010"><div class="article-attachment"><a class="icon-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-mmc2.nwk" title="Download Phylogenetic Trees in Newick (nwk) file (8KB)" target="_blank"><svg focusable="false" viewBox="0 0 95 128" width="17.8125" height="24" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg></a><a class="download-link" target="_blank" download="" title="Download Phylogenetic Trees in Newick (nwk) file (8KB)" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-mmc2.nwk">Download Phylogenetic Trees in Newick (nwk) file (8KB)</a><a class="help-link right" title="Help with nwk files (Opens in new window)" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank">Help with nwk files</a></div><div class="captions"><span><p><span class="label">Supplementary data 2</span>. </p></span></div></div></span><span class="display"><div class="e-component e-component-mmc3" id="m0015"><div class="article-attachment"><a class="icon-link" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-mmc3.xml" title="Download XML file (244B)" target="_blank"><svg focusable="false" viewBox="0 0 95 128" width="17.8125" height="24" class="icon icon-document"><path d="m35.6 1e1c-5.38 0-10.62 1.92-14.76 5.4-9.1 7.68-18.84 20.14-18.84 32.1v70.5h9e1v-108zm0 1e1h46.4v88h-7e1v-49c0-6.08 4.92-11 11-11h17v-2e1h-6c-2.2 0-4 1.8-4 4v6h-7c-3.32 0-6.44 0.78-9.22 2.16 2.46-5.62 7.28-11.86 13.5-17.1 2.34-1.98 5.3-3.06 8.32-3.06z"></path></svg></a><a class="download-link" target="_blank" download="" title="Download XML file (244B)" href="https://ars.els-cdn.com/content/image/1-s2.0-S1055790317305900-mmc3.xml">Download XML file (244B)</a><a class="help-link right" title="Help with xml files (Opens in new window)" href="https://service.elsevier.com/app/answers/detail/a_id/19286/supporthub/sciencedirect/" target="_blank">Help with xml files</a></div><div class="captions"><span><p><span class="label">Supplementary data 3</span>. </p></span></div></div></span></p></section></div></div><div class="related-content-links u-hide-from-md"><button class="button button-anchor" type="button"><span class="button-text">Recommended articles</span></button><button class="button button-anchor" disabled="disabled" type="button"><span class="button-text">Citing articles (<!-- -->0<!-- -->)</span></button></div><div id="ec-research-data" class="ec-research-data"><h2>Research data for this article</h2><div class="ec-research-data-card ec-data-profile"><div class="ec-research-data-card-entry ec-dp"><div class="ec-research-data-card-text"></div><div class="ec-research-data-card-access"><svg focusable="false" viewBox="0 0 92 128" width="16" height="16" class="icon icon-lock ec-research-data-card-icon-small"><path d="m81 105c0 1.63-1.38 3-3 3h-64c-1.62 0-3-1.37-3-3v-36c0-1.63 1.38-3 3-3h9v-19c0-12.68 10.32-23 23-23s23 10.32 23 23v9h-38v1e1h47c1.62 0 3 1.37 3 3v36zm-2-48.95v-9.05c0-18.2-14.8-33-33-33s-33 14.8-33 33v9.05c-6.7 0.51-12 6.12-12 12.95v36c0 7.17 5.84 13 13 13h64c7.16 0 13-5.83 13-13v-36c0-6.83-5.3-12.44-12-12.95m-33 19.95c-4.98 0-9 4.03-9 9 0 3.12 1.58 5.87 4 7.48v7.52h1e1v-7.52c2.42-1.61 4-4.36 4-7.48 0-4.97-4.02-9-9-9"></path></svg><span class="ec-description">Data not available / Data will be made available on request</span></div></div><style>
.e-component-mmc3 {
  display: none;
}
</style></div><a href="https://www.elsevier.com/authors/author-services/research-data" class="ec-anchor ec-about" aria-label="About research data" target="_blank"><svg focusable="false" viewBox="0 0 100 128" width="24" height="24" preserveAspectRatio="xMinYMin meet" class="icon icon-info ec-anchor-icon left ec-about-icon"><path d="m5e1 14c-13.08 0-25.4 5.1-34.64 14.36-9.26 9.24-14.36 21.56-14.36 34.64 0 27.02 21.98 49 49 49 13.1 0 25.4-5.1 34.66-14.36 9.24-9.24 14.34-21.56 14.34-34.64 0-27.02-21.98-49-49-49zm0 9.6c21.72 0 39.4 17.68 39.4 39.4 0 10.52-4.1 20.42-11.54 27.86s-17.34 11.52-27.86 11.52c-21.72 0-39.4-17.66-39.4-39.38 0-10.52 4.1-20.42 11.54-27.86s17.34-11.54 27.86-11.54zm-5 14.4v1e1h1e1v-1e1h-1e1zm0 16v34h1e1v-34h-1e1z"></path></svg><span>About research data<svg focusable="false" viewBox="0 0 78 128" width="16" height="16" class="icon icon-arrow-up-right ec-anchor-icon right"><path d="m4 36h57.07l-59.5 59.5 7.07 7.08 59.36-59.36v56.78h1e1v-74h-74z"></path></svg></span></a></div><div class="Tail"></div><section class="bibliography" id="bi005"><h2 class="section-title">References</h2><section class="bibliography-sec" id="bs005"><dl class="references"><dt class="label"><a href="#bb0005" id="ref-id-b0005">Aoki et al., 2009</a></dt><dd class="reference" id="h0005"><div class="contribution">K. Aoki, M. Kato, N. Murakami<strong class="title">Phylogeographical
 patterns of a generalist acorn weevil: insight into the biogeographical
 history of broadleaved deciduous and evergreen forests</strong></div><div class="host">BMC Evol. Biol., 9 (2009), p. 103</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0010" id="ref-id-b0010">Aoki et al., 2011</a></dt><dd class="reference" id="h0010"><div class="contribution">K. Aoki, M. Kato, N. Murakami<strong class="title">Phylogeography
 of phytophagous weevils and plant species in broadleaved evergreen 
forests: a congruent genetic gap between western and eastern parts of 
Japan</strong></div><div class="host">Insects, 2 (2) (2011), pp. 128-150</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0015" id="ref-id-b0015">Avise, 2000</a></dt><dd class="reference" id="h0015"><div class="contribution">J.C. Avise<strong class="title">Phylogeography: The History and Formation of Species</strong></div><div class="host">Harvard University Press, Cambridge, MA (2000), p. 447</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0020" id="ref-id-b0020">Avise and Johns, 1999</a></dt><dd class="reference" id="h0020"><div class="contribution">J.C. Avise, G.C. Johns<strong class="title">Proposal for a standardized temporal scheme of biological classification for extant species</strong></div><div class="host">Proc. Natl. Acad. Sci., 96 (13) (1999), pp. 7358-7363</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0025" id="ref-id-b0025">Avtzis et al., 2013</a></dt><dd class="reference" id="h0025"><div class="contribution">D.N. Avtzis, C. Perlerou, S. Diamandis<strong class="title">Geographic distribution of chestnut feeding insects in Greece</strong></div><div class="host">J. Pest. Sci., 86 (2) (2013), pp. 185-191</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0030" id="ref-id-b0030">Bandelt et al., 1999</a></dt><dd class="reference" id="h0030"><div class="contribution">H.J. Bandelt, P. Forster, A. Röhl<strong class="title">Median-joining networks for inferring intraspecific phylogenies</strong></div><div class="host">Mol. Biol. Evol., 16 (1) (1999), pp. 37-48</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0035" id="ref-id-b0035">Barber, 1935</a></dt><dd class="reference" id="h0035"><span>Barber,
 H.S., 1935. The tobacco and solanum weevils of the genus Trichobaris. 
Miscellaneous Publication No. 226. United States Department of 
Agriculture, Washington DC, 28 p.</span></dd><dt class="label"><a href="#bb0040" id="ref-id-b0040">Barr et al., 2013</a></dt><dd class="reference" id="h0040"><div class="contribution">N. Barr, R. Ruiz-Arce, O. Obregón, R. De Leon, N. Foster, C. Reuter, D. Vacek<strong class="title">Molecular diagnosis of populational variants of <em>Anthonomus grandis</em> (Coleoptera: Curculionidae) in North America</strong></div><div class="host">J. Econ. Entomol., 106 (1) (2013), pp. 437-449</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0045" id="ref-id-b0045">Barraclough and Nee, 2001</a></dt><dd class="reference" id="h0045"><div class="contribution">T.G. Barraclough, S. Nee<strong class="title">Phylogenetics and speciation</strong></div><div class="host">Trends Ecol. Evol., 16 (7) (2001), pp. 391-399</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0050" id="ref-id-b0050">Basio et al., 1994</a></dt><dd class="reference" id="h0050"><div class="contribution">R.G. Basio, P.J. Johnson, D.R. Pua, H.T. Bergonia, C.C. Diloy, E.L. Villegas<strong class="title">Mango pulp weevil [Sternochetus frigidus (Fabr.)](Curculionidae, Coleoptera) found in Palawan [Philippines]. Philippine</strong></div><div class="host">Entomologist, 9 (1994)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0055" id="ref-id-b0055">Becerra, 2015</a></dt><dd class="reference" id="h0055"><div class="contribution">J.X. Becerra<strong class="title">Macroevolutionary and geographical intensification of chemical defense in plants driven by insect herbivore selection pressure</strong></div><div class="host">Curr. Opin. Insect. Sci., 8 (2015), pp. 15-21</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0060" id="ref-id-b0060">Bierig, 1939</a></dt><dd class="reference" id="h0060"><div class="contribution">A. Bierig<strong class="title">El picudo del aguacate. The avocado weevil</strong></div><div class="host">Revista del Centro Nacional de Agricultura (Costa Rica), 4 (6) (1939), pp. 355-357</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0065" id="ref-id-b0065">Blomberg et al., 2003</a></dt><dd class="reference" id="h0065"><div class="contribution">S.P. Blomberg, T. Garland Jr, A.R. Ives<strong class="title">Testing for phylogenetic signal in comparative data: behavioral traits are more labile</strong></div><div class="host">Evolution, 57 (4) (2003), pp. 717-745</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0070" id="ref-id-b0070">Borbolla, 2015</a></dt><dd class="reference" id="h0070"><span>Borbolla,
 M., 2015. Estructura genética de Trichobaris soror depredador de 
semillas de Datura stramonium. B.Sc. Thesis, Biology, Faculty of 
Sciences, UNAM, Mexico City.</span></dd><dt class="label"><a href="#bb0075" id="ref-id-b0075">Briese et al., 1996</a></dt><dd class="reference" id="h0075"><div class="contribution">D.T. Briese, C. Espiau, A. Pouchot-Lermans<strong class="title">Micro-evolution in the weevil genus <em>Larinus</em>: the formation of host biotypes and speciation</strong></div><div class="host">Mol. Ecol., 5 (4) (1996), pp. 531-545</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0080" id="ref-id-b0080">Bouckaert et al., 2014</a></dt><dd class="reference" id="h0080"><div class="contribution">R. Bouckaert, J. Heled, D. Kühnert, T. Vaughan, C.H. Wu, D. Xie, A.J. Drummond<strong class="title">BEAST 2: a software platform for Bayesian evolutionary analysis</strong></div><div class="host">PLoS Comput. Biol., 10 (4) (2014), p. e1003537</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0085" id="ref-id-b0085">Cabrales-Vargas, 1991</a></dt><dd class="reference" id="h0085"><span>Cabrales-Vargas,
 R.A., 1991. Demografía e historia natural de Datura stramonium L. en el
 Pedregal de San Angel con algunas implicaciones evolutivas. B.Sc. 
Thesis, Biology, Faculty of Sciences, UNAM, Mexico City.</span></dd><dt class="label"><a href="#bb0090" id="ref-id-b0090">Calyecac-Cortero et al., 2004</a></dt><dd class="reference" id="h0090"><div class="contribution">H.G. Calyecac-Cortero, J. Cibrián-Tovar, N. Bautista-Martinez, J. López-Collado<strong class="title">Comportamiento de alimentación, cortejo, cópula y oviposición de <em>Trichobaris champion</em>i Barber (Coleoptera: Curculionidae)</strong></div><div class="host">Agrociencia, 38 (2004), pp. 365-373</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0095" id="ref-id-b0095">Capinera, 2005</a></dt><dd class="reference" id="h0095"><span>Capinera,
 J.L., 2005. Pepper weevil, Anthonomus eugenii Cano (Insecta: 
Coleoptera: Curculionidae). University of Florida. IFAS Extension. 
EENY-278.</span></dd><dt class="label"><a href="#bb0100" id="ref-id-b0100">Castañeda-Vildózola et al., 2015</a></dt><dd class="reference" id="h0100"><div class="contribution">Á. Castañeda-Vildózola, O. Franco-Mora, J.C.R. Alemán, C. Ruiz-Montiel, J. Váldez-Carrasco, A. Equihua-Martínez<strong class="title">New distribution records of the small avocado seed weevil, <em>Conotrachelus</em> perseae Barber (Coleoptera: Curculionidae), in Mexico and notes on its biology</strong></div><div class="host">Coleopterists Bull., 69 (2) (2015), pp. 267-271</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0105" id="ref-id-b0105">Charlet, 1983</a></dt><dd class="reference" id="h0105"><div class="contribution">L.D. Charlet<strong class="title">Distribution and abundance of a stem weevil, <em>Cylindrocopturus adspersus</em> (Coleoptera: Curculionidae), in cultivated sunflower in the northern plains</strong></div><div class="host">Environ. Entomol., 12 (5) (1983), pp. 1526-1528</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0110" id="ref-id-b0110">Chen et al., 2017</a></dt><dd class="reference" id="h0110"><div class="contribution">W. Chen, Z. Zhong, W. Dai, Q. Fan, S. He<strong class="title">Phylogeographic structure, cryptic speciation and demographic history of the sharpbelly <em>(Hemiculter leucisculu</em>s), a freshwater habitat generalist from southern China</strong></div><div class="host">BMC Evol. Biol., 17 (1) (2017), p. 216</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0115" id="ref-id-b0115">Cuda and Burke, 1984</a></dt><dd class="reference" id="h0115"><div class="contribution">J.P. Cuda, H.R. Burke<strong class="title">Biology and impact of <em>Trichobaris texana</em> (Coleoptera: Curculionidae) on silverleaf nightshade, <em>Solanum elaeagnifoliun</em> in Central Texas</strong></div><div class="host">Proc. VI Int. Symp. Biol. Contr. Weeds, 19 (1984), p. 25</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0120" id="ref-id-b0120">Cuda and Burke, 1986</a></dt><dd class="reference" id="h0120"><div class="contribution">J.P. Cuda, H.R. Burke<strong class="title">Reproduction and development of the potato stalk borer, (Coleoptera: Curculionidae) with notes on field biology</strong></div><div class="host">J. Econ. Entomol., 79 (6) (1986), pp. 1548-1554</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0125" id="ref-id-b0125">Cuda and Burke, 1991</a></dt><dd class="reference" id="h0125"><div class="contribution">J.P. Cuda, H.R. Burke<strong class="title">Biology of <em>Trichobaris bridwelli</em> (Coleoptera: Curculionidae), a possible agent for the biological control of <em>Datura stramonium</em> (Solanaceae)</strong></div><div class="host">Environ. Entomol., 20 (3) (1991), pp. 899-908</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0130" id="ref-id-b0130">Davis, 2014</a></dt><dd class="reference" id="h0130"><span>Davis,
 S.R., 2014. Morphology, phylogeny, and evolutionary development in the 
weevils (Insecta: Coleoptera: Curculionoidea) (Doctoral Dissertation, 
University of Kansas).</span></dd><dt class="label"><a href="#bb0135" id="ref-id-b0135">De-la-Mora et al., 2015</a></dt><dd class="reference" id="h0135"><div class="contribution">M. De-la-Mora, D. Piñero, J. Núñez-Farfán<strong class="title">Phylogeography of specialist weevil <em>Trichobaris soror:</em> a seed predator of <em>Datura stramonium</em></strong></div><div class="host">Genetica, 143 (6) (2015), pp. 681-691</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0140" id="ref-id-b0140">Diezel et al., 2011</a></dt><dd class="reference" id="h0140"><div class="contribution">C. Diezel, D. Kessler, I.T. Baldwin<strong class="title">Pithy protection: <em>Nicotiana attenuata</em>’s jasmonic acid-mediated defenses are required to resist stem-boring weevil larvae</strong></div><div class="host">Plant Physiol., 155 (4) (2011), pp. 1936-1946</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0145" id="ref-id-b0145">Drummond and Rambaut, 2007</a></dt><dd class="reference" id="h0145"><div class="contribution">A.J. Drummond, A. Rambaut<strong class="title">BEAST: Bayesian evolutionary analysis by sampling trees</strong></div><div class="host">BMC Evol. Biol., 7 (1) (2007), p. 214</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0150" id="ref-id-b0150">Ehrlich and Raven, 1964</a></dt><dd class="reference" id="h0150"><div class="contribution">P.R. Ehrlich, P.H. Raven<strong class="title">Butterflies and plants: a study in coevolution</strong></div><div class="host">Evolution (1964), pp. 586-608</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0155" id="ref-id-b0155">Farrell, 1998</a></dt><dd class="reference" id="h0155"><div class="contribution">B.D. Farrell<strong class="title">“Inordinate fondness” explained: Why are there so many beetles?</strong></div><div class="host">Science, 281 (5376) (1998), pp. 555-559</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0160" id="ref-id-b0160">Fritz and Purvis, 2010</a></dt><dd class="reference" id="h0160"><div class="contribution">S.A. Fritz, A. Purvis<strong class="title">Selectivity in mammalian extinction risk and threat types: a new measure of phylogenetic signal strength in binary traits</strong></div><div class="host">Conserv. Biol., 24 (4) (2010), pp. 1042-1051</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0165" id="ref-id-b0165">Futuyma and Agrawal, 2009</a></dt><dd class="reference" id="h0165"><div class="contribution">D.J. Futuyma, A.A. Agrawal<strong class="title">Macroevolution and the biological diversity of plants and herbivores</strong></div><div class="host">Proc. Natl. Acad. Sci., 106 (43) (2009), pp. 18054-18061</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0170" id="ref-id-b0170">Gavrilets, 2003</a></dt><dd class="reference" id="h0170"><div class="contribution">S. Gavrilets<strong class="title">Perspective: models of speciation: what have we learned in 40 years?</strong></div><div class="host">Evolution, 57 (10) (2003), pp. 2197-2215</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0175" id="ref-id-b0175">Grimaldi and Engel, 2005</a></dt><dd class="reference" id="h0175"><div class="contribution">D. Grimaldi, M.S. Engel<strong class="title">Evolution of the Insects</strong></div><div class="host">Cambridge University Press, New York (2005), p. 755</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0180" id="ref-id-b0180">Hernández-Vera et al., 2010</a></dt><dd class="reference" id="h0180"><div class="contribution">G. Hernández-Vera, M. Mitrovic, J. Jovic, I. Tosevski, R. Caldara, A. Gassmann, B.C. Emerson<strong class="title">Host-associated genetic differentiation in a seed parasitic weevil <em>Rhinusa antirrhini</em> (Coleptera: Curculionidae) revealed by mitochondrial and nuclear sequence data</strong></div><div class="host">Mol. Ecol., 19 (2010), pp. 2286-2300</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0185" id="ref-id-b0185">Holt and Jønsson, 2014</a></dt><dd class="reference" id="h0185"><div class="contribution">B.G. Holt, K.A. Jønsson<strong class="title">Reconciling hierarchical taxonomy with molecular phylogenies</strong></div><div class="host">Syst. Biol., 63 (6) (2014), pp. 1010-1017</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0190" id="ref-id-b0190">Iwase et al., 2015</a></dt><dd class="reference" id="h0190"><div class="contribution">S.I. Iwase, K. Nakahira, M. Tuda, K. Kagoshima, M. Takagi<strong class="title">Host-plant dependent population genetics of the invading weevil <em>Hypera postica</em></strong></div><div class="host">Bull. Entomol. Res., 105 (01) (2015), pp. 92-100</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0195" id="ref-id-b0195">Jurado-Rivera et al., 2009</a></dt><dd class="reference" id="h0195"><div class="contribution">J.A. Jurado-Rivera, A.P. Vogler, C.A. Reid, E. Petitpierre, J. Gómez-Zurita<strong class="title">DNA barcoding insect–host plant associations</strong></div><div class="host">Proc. Royal Soc. London B: Biol. Sci., 276 (1657) (2009), pp. 639-648</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0200" id="ref-id-b0200">Klingenberg, 2011</a></dt><dd class="reference" id="h0200"><div class="contribution">C.P. Klingenberg<strong class="title">MorphoJ: an integrated software package for geometric morphometrics</strong></div><div class="host">Mol. Ecol. Resour., 11 (2) (2011), pp. 353-357</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0205" id="ref-id-b0205">Kobayashi et al., 2012</a></dt><dd class="reference" id="h0205"><div class="contribution">C. Kobayashi, Y. Okuyama, K. Kawazoe, M. Kato<strong class="title">The
 evolutionary history of maternal plant-manipulation and larval feeding 
behaviours in attelabid weevils (Coleoptera; Curculionoidea)</strong></div><div class="host">Mol. Phylogenet. Evol., 64 (2) (2012), pp. 318-330</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0210" id="ref-id-b0210">Kohyama et al., 2014</a></dt><dd class="reference" id="h0210"><div class="contribution">T.I. Kohyama, K. Matsumoto, H. Katakura<strong class="title">Deep phylogeographical structure and parallel host range evolution in the leaf beetle <em>Agelasa nigriceps</em></strong></div><div class="host">Mol. Ecol., 23 (2) (2014), pp. 421-434</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0215" id="ref-id-b0215">Kuester et al., 2012</a></dt><dd class="reference" id="h0215"><div class="contribution">A.P. Kuester, R.W. Jones, T.W. Sappington, K.S. Kim, N.B. Barr, R.L. Roehrdanz, P. Senechal, J.D. Nason<strong class="title">Population structure and genetic diversity of the boll weevil (Coleoptera: Curculionidae) on <em>Gossypium</em> in North America</strong></div><div class="host">Ann. Entomol. Soc. Am., 105 (6) (2012), pp. 902-916</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0220" id="ref-id-b0220">Kuroki and Kodama, 1987</a></dt><dd class="reference" id="h0220"><div class="contribution">K. Kuroki, T. Kodama<strong class="title">Life history and control of the chestnut weevil, <em>Curculio dentipes</em> Roelofs, on the chestnut cultivate, ganne tree</strong></div><div class="host">Bulletin of the Yamaguchi Agricultural Experiment Station (1987)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0225" id="ref-id-b0225">Lemic et al., 2016</a></dt><dd class="reference" id="h0225"><div class="contribution">D. Lemic, H.A. Benítez, T.A. Püschel, H.V. Gašparić, M. Šatvar, R. Bažok<strong class="title">Ecological
 morphology of the sugar beet weevil Croatian populations: Evaluating 
the role of environmental conditions on body shape</strong></div><div class="host">Zoologischer Anzeiger-A J. Comp. Zool., 260 (2016), pp. 25-32</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0230" id="ref-id-b0230">Marvaldi et al., 2002</a></dt><dd class="reference" id="h0230"><div class="contribution">A.E. Marvaldi, A.S. Sequeira, C.W. O'Brien, B.D. Farrell<strong class="title">Molecular and morphological phylogenetics of weevils (Coleoptera, Curculionoidea): do niche shifts accompany diversification?</strong></div><div class="host">Syst. Biol., 51 (5) (2002), pp. 761-785</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0235" id="ref-id-b0235">Matsubayashi et al., 2010</a></dt><dd class="reference" id="h0235"><div class="contribution">K.W. Matsubayashi, I. Ohshima, P. Nosil<strong class="title">Ecological speciation in phytophagous insects</strong></div><div class="host">Entomol. Exp. Appl., 134 (1) (2010), pp. 1-27</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0240" id="ref-id-b0240">McKenna et al., 2009</a></dt><dd class="reference" id="h0240"><div class="contribution">D.D. McKenna, A.S. Sequeira, A.E. Marvaldi, B.D. Farrell<strong class="title">Temporal lags and overlap in the diversification of weevils and flowering plants</strong></div><div class="host">Proc. Natl. Acad. Sci., 106 (17) (2009), pp. 7083-7088</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0245" id="ref-id-b0245">Miranda-Pérez et al., 2016</a></dt><dd class="reference" id="h0245"><div class="contribution">A. Miranda-Pérez, G. Castillo, J. Hernández-Cumplido, P.L. Valverde, M. Borbolla, L.L. Cruz, R. Tapia-López, J. Fornoni, C.M. Flores-Ortiz, J. Núñez-Farfán<strong class="title">Natural selection drives chemical resistance of <em>Datura stramonium</em></strong></div><div class="host">PeerJ, 4 (2016), p. e1898, <a href="https://doi.org/10.7717/peerj.1898" target="_blank">10.7717/peerj.1898</a></div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0250" id="ref-id-b0250">Mynhardt, 2006</a></dt><dd class="reference" id="h0250"><span>Mynhardt,
 G., 2006. Population genetics of the pecan weevil, Curculio caryae Horn
 (Coleoptera: Curculionidae), inferred from mitochondrial nucleotide 
data (Doctoral Dissertation, Texas A&amp;M University).</span></dd><dt class="label"><a href="#bb0255" id="ref-id-b0255">Nakamine and Takeda, 2008</a></dt><dd class="reference" id="h0255"><div class="contribution">H. Nakamine, M. Takeda<strong class="title">Molecular phylogenetic relationships of flightless beetles belonging to the genus <em>Mesechthistatus</em> Breuning, (Coleoptera: Cerambycidae) inferred from mitochondrial COI gene sequences</strong></div><div class="host">J. Insect Sci., 8 (1) (2008)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0260" id="ref-id-b0260">Nei, 1987</a></dt><dd class="reference" id="h0260"><div class="contribution">M. Nei<strong class="title">Molecular Evolutionary Genetics</strong></div><div class="host">Columbia University Press (1987)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0265" id="ref-id-b0265">Oberprieler et al., 2007</a></dt><dd class="reference" id="h0265"><div class="contribution">R.G. Oberprieler, A.E. Marvaldi, R.S. Anderson<strong class="title">Weevils, weevils, weevils everywhere</strong></div><div class="host">Zootaxa, 1668 (2007), pp. 491-520</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0270" id="ref-id-b0270">Oliveira et al., 2012</a></dt><dd class="reference" id="h0270"><div class="contribution">D.C. Oliveira, F.C. Almeida, P.M. O’Grady, M.A. Armella, R. DeSalle, W.J. Etges<strong class="title">Monophyly, divergence times, and evolution of host plant use inferred from a revised phylogeny of the <em>Drosophila repleta</em> species group</strong></div><div class="host">Mol. Phylogenet. Evol., 64 (3) (2012), pp. 533-544</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0275" id="ref-id-b0275">Pagel, 1999</a></dt><dd class="reference" id="h0275"><div class="contribution">M. Pagel<strong class="title">The maximum likelihood approach to reconstructing ancestral character states of discrete characters on phylogenies</strong></div><div class="host">Syst. Biol., 48 (3) (1999), pp. 612-622</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0280" id="ref-id-b0280">Parra et al., 2009</a></dt><dd class="reference" id="h0280"><div class="contribution">L Mutis Parra, A. Aguilera, R. Rebolledo, A. Quiroz<strong class="title">Estado del conocimiento sobre el cabrito del frambueso (cf), <em>Aegorhinus superciliosus</em> (Guérin) (Coleoptera: Curculionidae)</strong></div><div class="host">Idesia, 27 (1) (2009), pp. 57-65</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0285" id="ref-id-b0285">Rambaut and Drummond, 2014</a></dt><dd class="reference" id="h0285"><div class="contribution">A. Rambaut, A.J. Drummond<strong class="title">TreeAnnotator v2.1.2</strong></div><div class="host">University of Edinburgh, Institute of Evolutionary Biology, Edinburgh (2014)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0290" id="ref-id-b0290">Rambaut, 2012</a></dt><dd class="reference" id="h0290"><div class="contribution">A. Rambaut<strong class="title">FigTree v1. 4. Molecular Evolution, Phylogenetics and Epidemiology</strong></div><div class="host">University of Edinburgh, Institute of Evolutionary Biology, Edinburgh, UK (2012)</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0295" id="ref-id-b0295">Rambaut et al., 2014</a></dt><dd class="reference" id="h0295"><span>Rambaut, A., Suchard, M.A., Xie, D., Drummond, A.J., 2014. Tracer v1.6, Available from <a href="http://beast.bio.ed.ac.uk/Tracer" target="_blank">http://beast.bio.ed.ac.uk/Tracer</a>.</span></dd><dt class="label"><a href="#bb0300" id="ref-id-b0300">Revell, 2012</a></dt><dd class="reference" id="h0300"><div class="contribution">L.J. Revell<strong class="title">phytools: a R package for phylogenetic comparative biology (and other things)</strong></div><div class="host">Methods Ecol. Evol., 3 (2) (2012), pp. 217-223</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0305" id="ref-id-b0305">Rohlf, 2004</a></dt><dd class="reference" id="h0305"><span>Rohlf, F.J., 2004. TPS dig v. 1.4 Department of Ecology and Evolution, State University of New York, Stony Brook, NY. (<a href="http://life.bio.sunysb.edu/morph/" target="_blank">http://life.bio.sunysb.edu/morph/</a>).</span></dd><dt class="label"><a href="#bb0310" id="ref-id-b0310">Rozas et al., 2003</a></dt><dd class="reference" id="h0310"><div class="contribution">J. Rozas, J.C. Sánchez-DelBarrio, X. Messeguer, R. Rozas<strong class="title">DnaSP, DNA polymorphism analyses by the coalescent and other methods</strong></div><div class="host">Bioinformatics, 19 (18) (2003), pp. 2496-2497</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0315" id="ref-id-b0315">Shankar et al., 2015</a></dt><dd class="reference" id="h0315"><div class="contribution">P. Shankar, V.M. Kulkarni, L.S. Kumar<strong class="title">Male biased gene flow in banana pseudostem weevil (<em>Odoiporus longicollis</em> Oliver) as revealed by analysis of the COI-tRNALeu COII region</strong></div><div class="host">Genetica, 143 (1) (2015), pp. 85-92</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0320" id="ref-id-b0320">Simon et al., 1994</a></dt><dd class="reference" id="h0320"><div class="contribution">C. Simon, F. Frati, A. Beckenbach, B. Crespi, H. Liu, P. Flook<strong class="title">Evolution,
 weighting, and phylogenetic utility of mitochondrial gene sequences and
 a compilation of conserved polymerase chain reaction primers</strong></div><div class="host">Ann. Entomological Soc. Am., 87 (6) (1994), pp. 651-701</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0325" id="ref-id-b0325">Sota et al., 2004</a></dt><dd class="reference" id="h0325"><div class="contribution">T. Sota, M. Hayashi, D. Iwai<strong class="title">Phylogeography of the leaf beetle <em>Chrysolina virgata</em> in wetlands of Japan inferred from the distribution of mitochondrial haplotypes</strong></div><div class="host">Entomol. Sci., 7 (4) (2004), pp. 381-388</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0330" id="ref-id-b0330">Suchard et al., 2001</a></dt><dd class="reference" id="h0330"><div class="contribution">M.A. Suchard, R.E. Weiss, J.S. Sinsheimer<strong class="title">Bayesian selection of continuous-time Markov chain evolutionary models</strong></div><div class="host">Mol. Biol. Evol., 18 (6) (2001), pp. 1001-1013</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0335" id="ref-id-b0335">Supple et al., 2014</a></dt><dd class="reference" id="h0335"><div class="contribution">M. Supple, R. Papa, B. Counterman, W.O. McMillan<strong class="title">The genomics of an adaptive radiation: insights across the Heliconius speciation continuum</strong></div><div class="host">Ecological Genomics, Springer, Netherlands (2014), pp. 249-271</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0340" id="ref-id-b0340">Toju and Sota, 2009</a></dt><dd class="reference" id="h0340"><div class="contribution">H. Toju, T. Sota<strong class="title">Do arms races punctuate evolutionary stasis? Unified insights from phylogeny, phylogeography and microevolutionary processes</strong></div><div class="host">Mol. Ecol., 18 (18) (2009), pp. 3940-3954</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0345" id="ref-id-b0345">Toju, 2009</a></dt><dd class="reference" id="h0345"><div class="contribution">H. Toju<strong class="title">Natural
 selection drives the fine-scale divergence of a coevolutionary arms 
race involving a long-mouthed weevil and its obligate host plant</strong></div><div class="host">BMC Evol. Biol., 9 (1) (2009), p. 273</div><div class="ReferenceLinks" id="reference-links"></div></dd><dt class="label"><a href="#bb0350" id="ref-id-b0350">Wise, 2007</a></dt><dd class="reference" id="h0350"><div class="contribution">M.J. Wise<strong class="title">Evolutionary
 ecology of resistance to herbivory: an investigation of potential 
genetic constraints in the multiple-herbivore community of <em>Solanum carolinense</em></strong></div><div class="host">New Phytol., 175 (4) (2007), pp. 773-784</div><div class="ReferenceLinks" id="reference-links"></div></dd></dl></section></section><div class="Copyright"><span class="copyright-line">© 2018 Elsevier Inc. All rights reserved.</span></div></article><div class="u-show-from-md col-lg-6 col-md-8 pad-right"><aside class="RelatedContent" aria-label="Related content"><section class="SidePanel"><header id="recommended-articles-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Recommended articles</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="recommended-articles-header"><div id="recommended-articles"><ul><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317308485"><h3 class="article-title ellipsis" id="recommended-articles-article0-title" title="Using an integrated approach to identify cryptic species, divergence patterns and hybrid species in Asian ladies’ tresses orchids (Spiranthes, Orchidaceae)"><span>Using
 an integrated approach to identify cryptic species, divergence patterns
 and hybrid species in Asian ladies’ tresses orchids (<em>Spiranthes</em>, Orchidaceae)</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 124, 2018, pp. 106-121</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317308485/pdfft?md5=8902c1550d9d2411199b80030684edd7&amp;pid=1-s2.0-S1055790317308485-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article0-title" aria-controls="recommended-articles-article0" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article0" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790316304250"><h3 class="article-title ellipsis" id="recommended-articles-article1-title" title="Phylogeny, molecular dating and zoogeographic history of the titi monkeys (Callicebus, Pitheciidae) of eastern Brazil"><span>Phylogeny, molecular dating and zoogeographic history of the titi monkeys (<em>Callicebus</em>, Pitheciidae) of eastern Brazil</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 124, 2018, pp. 10-15</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790316304250/pdfft?md5=864ebb704a220ba762003c6fdbab6c38&amp;pid=1-s2.0-S1055790316304250-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article1-title" aria-controls="recommended-articles-article1" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article1" aria-hidden="true"></div></li><li class="SidePanelItem"><div class="sub-heading"><a href="https://www.sciencedirect.com/science/article/pii/S1055790317306097"><h3 class="article-title ellipsis" id="recommended-articles-article2-title" title="Spiroides shrubs on Qinghai-Tibetan Plateau: Multilocus phylogeography and palaeodistributional reconstruction of Spiraea alpina and S. Mongolica (Rosaceae)"><span>Spiroides shrubs on Qinghai-Tibetan Plateau: Multilocus phylogeography and palaeodistributional reconstruction of <em>Spiraea alpina</em> and <em>S. Mongolica</em> (Rosaceae)</span></h3></a><div class="article-source ellipsis"><div class="source">Molecular Phylogenetics and Evolution, Volume 123, 2018, pp. 137-148</div></div></div><div class="buttons"><a class="anchor side-panel-pdf-link" href="https://www.sciencedirect.com/science/article/pii/S1055790317306097/pdfft?md5=66f3f0dd9f70eb4b351e78e9fb9cb294&amp;pid=1-s2.0-S1055790317306097-main.pdf" target="_blank" rel="nofollow"><svg focusable="false" viewBox="0 0 32 32" width="24" height="24" class="icon icon-pdf-multicolor"><path d="M7 .362h17.875l6.763 6.1V31.64H6.948V16z" stroke="#000" stroke-width=".703" fill="#fff"></path><path d="M.167 2.592H22.39V9.72H.166z" stroke="#aaa" stroke-width=".315" fill="#da0000"></path><path fill="#fff9f9" d="M5.97 3.638h1.62c1.053 0 1.483.677 1.488 1.564.008.96-.6 1.564-1.492 1.564h-.644v1.66h-.977V3.64m.977.897v1.34h.542c.27 0 .596-.068.596-.673-.002-.6-.32-.667-.596-.667h-.542m3.8.036v2.92h.35c.933 0 1.223-.448 1.228-1.462.008-1.06-.316-1.45-1.23-1.45h-.347m-.977-.94h1.03c1.68 0 2.523.586 2.534 2.39.01 1.688-.607 2.4-2.534 2.4h-1.03V3.64m4.305 0h2.63v.934h-1.657v.894H16.6V6.4h-1.56v2.026h-.97V3.638"></path><path d="M19.462 13.46c.348 4.274-6.59 16.72-8.508 15.792-1.82-.85 1.53-3.317 2.92-4.366-2.864.894-5.394 3.252-3.837 3.93 2.113.895 7.048-9.25 9.41-15.394zM14.32 24.874c4.767-1.526 14.735-2.974 15.152-1.407.824-3.157-13.72-.37-15.153 1.407zm5.28-5.043c2.31 3.237 9.816 7.498 9.788 3.82-.306 2.046-6.66-1.097-8.925-4.164-4.087-5.534-2.39-8.772-1.682-8.732.917.047 1.074 1.307.67 2.442-.173-1.406-.58-2.44-1.224-2.415-1.835.067-1.905 4.46 1.37 9.065z" fill="#f91d0a"></path></svg><span class="anchor-text">Download PDF</span></a><button class="button-link button-link-secondary side-panel-details-toggle" aria-describedby="recommended-articles-article2-title" aria-controls="recommended-articles-article2" aria-expanded="false" type="button"><span class="button-link-text">View details</span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></div><div id="recommended-articles-article2" aria-hidden="true"></div></li></ul></div><div class="pagination u-position-relative u-padding-xs-bottom"><span class="u-position-absolute"></span><span class="pagination-pages-label"><span class="pagination-nav u-margin-xs-hor pagination-current underline-page-number">1</span><span class="pagination-nav u-margin-xs-hor">2</span></span><span class="u-position-absolute"><button class="button-link button-link-secondary next-button" type="button"><span class="button-link-text">Next</span><svg focusable="false" viewBox="0 0 54 128" width="10.125" height="24" class="icon icon-navigate-right"><path d="m1 99l38-38-38-38 7-7 45 45-45 45z"></path></svg></button></span></div></div></section><section class="SidePanel"><header id="citing-articles-header" class="side-panel-header"><h2 class="section-title">Citing articles (0)</h2></header><div class="u-display-none" aria-hidden="true" aria-describedby="citing-articles-header"></div></section><section class="SidePanel hidden"><header id="metrics-header" class="side-panel-header"><button class="button-link side-panel-toggle is-up button-link-primary" aria-expanded="true" type="button"><span class="button-link-text"><h2 class="section-title">Article Metrics</h2></span><svg focusable="false" viewBox="0 0 92 128" width="17.25" height="24" class="icon icon-navigate-down"><path d="m1 51l7-7 38 38 38-38 7 7-45 45z"></path></svg></button></header><div class="" aria-hidden="false" aria-describedby="metrics-header"><a href="https://plu.mx/plum/a/?doi=10.1016/j.ympev.2018.02.018" class="plumx-summary plum-sciencedirect-theme" data-pass-hidden-categories="true" data-hide-usage="true" data-orientation="vertical" data-hide-print="true" data-site="plum" data-on-success="onMetricsWidgetSuccess">View article metrics</a></div></section></aside></div></div><div></div></div><div id="footer"><div class="hor-line" style="border-color:#e9711c"></div><div class="panel-s u-padding-l-bottom u-bg-white u-clr-grey7" role="contentinfo"><a class="anchor move-left els-footer-elsevier anchor-has-inherit-color" href="https://www.elsevier.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg viewBox="-3345 3440.027 140.01 24.333" style="width:104px;height:30px"><title>Elsevier</title><path id="E" style="fill:#E9711C" d="M-3343.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3344L-3343.999,3461.698"></path><path style="fill:#E9711C" d="M-3325.448,3461.698c2.074-0.118,2.83-0.502,2.83-2.832v-13.511c0-2.33-0.757-2.715-2.83-2.832v-0.591h8.884 v0.591c-2.243,0-3.027,0.472-3.027,2.891v13.009c0,1.652,0.056,2.625,1.71,2.625h4.008c3,0,4.4-2,5.576-4.1l0.673,0.118 l-1.71,5.222h-16.114V3461.698"></path><path style="fill:#E9711C" d="M-3307.122,3456.27h0.561c1.12,2.596,2.886,5.4,5.94,5.4c2,0,3.672-1.3,3.672-3.334 c0-1.652-1.626-3.1-4.176-4.927c-3.28-2.36-5.41-3.746-5.41-6.43c0-3.422,2.55-5.517,5.633-5.517c2.214,0,3,0.944,4.204,0.944 c0.476,0,0.56-0.266,0.476-0.737h0.561l0.645,6.076h-0.561c-0.785-2.625-2.523-5.19-5.354-5.19c-1.737,0-3.138,1.356-3.138,3.185 c0,1.918,1.933,3.157,5.016,5.016c2.523,1.504,5,3.362,5,6.254c0,3.245-2.608,5.752-5.97,5.752c-2.02,0-4.148-0.855-4.68-0.855 c-0.336,0-0.7,0.207-0.756,0.649h-0.56l-1.094-6.282"></path><path style="fill:#E9711C" d="M-3293.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3294L-3293.999,3461.698"></path><path style="fill:#E9711C" d="M-3265.839,3462.524h-0.42l-5.41-12.538c-0.896-2.065-1.71-4.16-2.83-6.136 c-0.478-0.826-1.346-1.327-2.3-1.327v-0.591h8.323v0.591c-0.785,0-2.354,0-2.354,1.15c0,0.384,0.87,2.45,1.653,4.308l4.063,9.676 l4.877-11.359c0.588-1.356,0.757-2.094,0.757-2.713c0-0.618-0.673-0.974-2.13-1.062v-0.59h5.941v0.591 c-0.337,0.06-0.7,0.117-1.037,0.295c-1.066,0.56-2.13,3.57-2.635,4.749l-6.5,14.957"></path><path style="fill:#E9711C" d="M-3255.472,3461.698c2.24,0,3.025-0.473,3.025-2.892v-13.393c0-2.42-0.784-2.89-3.025-2.891v-0.59h9.078v0.591 c-2.24,0-3.025,0.472-3.025,2.891v13.393c0,2.42,0.784,2.892,3.025,2.892v0.59h-9.08v-0.59"></path><path id="E_2_" style="fill:#E9711C" d="M-3244.999,3461.698c2.24,0,3.026-0.473,3.026-2.892v-13.393c0-2.42-0.785-2.89-3.026-2.891v-0.59 h16.787l0.252,4.455h-0.56c-0.308-2.48-1.513-3.216-3.84-3.216h-5.325c-1.26,0-1.26,0.355-1.26,2.066v5.693h5.913 c1.934,0,2.522-0.826,2.718-2.803h0.56v6.844h-0.56c-0.168-1.946-0.813-2.802-2.718-2.802h-5.913v6.401 c0,1.858,0.11,2.476,1.4,2.476h5.914c2.522,0,4.092-1.62,4.82-3.952l0.532,0.207l-1.513,4.985H-3245L-3244.999,3461.698"></path><path style="fill:#E9711C" d="M-3206,3461.698c-1.26-0.354-1.71-0.68-2.466-1.623l-6.166-7.609c3.027-0.65,5.13-2.185,5.13-5.547 c0-4.75-4.26-4.986-7.764-4.986h-9.191v0.591c2.24,0,3.026,0.472,3.026,2.891v13.393c0,2.42-0.785,2.892-3.026,2.892v0.59h9.08 v-0.59c-2.242,0-3.027-0.473-3.027-2.892v-5.604h2.551l7.314,9.086h4.54L-3206,3461.698 M-3220.399,3444.499 c0-1.387,0.337-1.476,2.186-1.476c2.774,0,5.3,0.974,5.3,4.308c0,3.6-2.887,4.63-5.914,4.631h-1.569v-7.463H-3220.399z"></path></svg></span></a><div class="panel-s u-bg-white u-padding-0-hor-from-xs u-padding-s-hor-from-md u-padding-xs-ver text-xs u-clear-both-from-xs u-clear-none-from-md"><p class="u-margin-xs-bottom"><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/solutions/sciencedirect" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">About ScienceDirect</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science/activateaccess" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Remote access</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.sciencedirect.com/science?_ob=ShoppingCartURL&amp;_method=display&amp;md5=3ff44acb300f01481824c54a2973d019" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Shopping cart</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://service.elsevier.com/app/contact/supporthub/sciencedirect/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Contact and support</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/elsevier-website-terms-and-conditions" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Terms and conditions</span></a><wbr><a class="anchor u-margin-xs-right u-margin-s-right-from-sm u-margin-l-right-from-md anchor-has-inherit-color" href="https://www.elsevier.com/legal/privacy-policy" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">Privacy policy</span></a></p><p id="els-footer-cookie-message">Cookies are used by this site. For more information, visit the<!-- --> <a class="anchor u-margin-0-right" href="https://www.elsevier.com/solutions/sciencedirect/support/cookies" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text">cookies page</span></a>.</p><p id="els-footer-copyright">Copyright © <!-- -->2018<!-- --> Elsevier B.V. or its licensors or contributors. ScienceDirect ® is a registered trademark of Elsevier B.V.</p></div><div><span style="margin-top:-10px" class="u-position-relative move-bottom u-float-left-from-xs u-float-right-from-md move-right u-padding-0-hor"><a class="anchor els-footer-relx anchor-has-inherit-color" href="https://www.relx.com/" rel="nofollow" target="_blank" style="white-space:nowrap"><span class="anchor-text"><svg style="width:118px;height:28px" viewBox="-3248 3424.938 334.44 53.422"><title>RELX Group</title><path style="fill:#E9711C" d="M-3227.285,3448.329c10.948,0,24.15-2.576,24.15-13.124c0-7.474-7.63-10.267-14.6-10.267 c-10.647,0-30.265,5.926-30.265,26.96c0,10.555,8.034,16.643,20.778,16.643c13.474,0,23.266-8.768,26.26-20.535 c-7.912,13.648-17.972,17.896-26.18,17.896c-10.19,0-14.077-7.568-14.077-14.05c0-18.56,14.406-24.798,23.494-24.798 c6.135,0,9.727,3.936,9.727,8.253c0,11.918-17.17,11.578-25.008,11.578c-0.803,0-3.274,0.02-3.76-0.003l-1.654,1.714 c3.59,0.508,8.554,1.497,13.944,4.223c11.204,5.656,21.745,20.59,35.7,20.59c5.062,0,6.38-1.543,7.335-2.893 c-6.834,4.354-18.286-3.197-24.46-9.04c-5.082-4.806-9.57-10.26-21.387-13.146"></path><g><path style="fill:#666666" d="M-3158.73,3468.502c-0.423,0-0.636-0.164-0.794-0.547l-6.885-14.511h-6.197c-0.21,0-0.316,0.11-0.316,0.33 v14.182c0,0.328-0.213,0.547-0.532,0.547h-5.189c-0.32,0-0.53-0.22-0.53-0.547v-36.139c0-0.33,0.21-0.55,0.53-0.549h13.928 c6.62,0,11.492,4.545,11.492,11.117c0,4.873-2.7,8.65-6.833,10.238l7.575,15.167c0.21,0.385,0,0.713-0.37,0.713L-3158.73,3468.502 L-3158.73,3468.502z M-3159.42,3442.384c0-3.232-2.223-5.31-5.506-5.311h-7.68c-0.21,0-0.316,0.108-0.316,0.328v9.912 c0,0.217,0.104,0.328,0.316,0.328h7.68C-3161.644,3447.641-3159.42,3445.561-3159.42,3442.384"></path><path style="fill:#666666" d="M-3146.95,3431.817c0-0.33,0.21-0.55,0.53-0.549h21.865c0.317,0,0.53,0.22,0.53,0.549v4.709 c0,0.33-0.212,0.548-0.53,0.548h-15.827c-0.212,0-0.318,0.108-0.318,0.328v9.089c0,0.22,0.106,0.328,0.318,0.328h12.726 c0.317,0,0.53,0.22,0.53,0.547v4.709c0,0.33-0.213,0.548-0.53,0.548h-12.726c-0.212,0-0.318,0.11-0.318,0.329v9.418 c0,0.22,0.106,0.328,0.318,0.328h15.827c0.317,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.212,0.547-0.53,0.547h-21.865 c-0.32,0-0.53-0.22-0.53-0.547C-3146.95,3467.956-3146.95,3431.817-3146.95,3431.817z"></path><path style="fill:#666666" d="M-3118.2,3431.817c0-0.33,0.212-0.55,0.53-0.549h5.191c0.317,0,0.53,0.22,0.53,0.549v30.553 c0,0.22,0.106,0.328,0.317,0.328h14.722c0.318,0,0.53,0.22,0.53,0.549v4.709c0,0.328-0.213,0.547-0.53,0.547h-20.76 c-0.317,0-0.53-0.22-0.53-0.547C-3118.2,3467.956-3118.2,3431.817-3118.2,3431.817z"></path><path style="fill:#666666" d="M-3052.46,3449.885c0-6.46,0.423-8.926,1.006-10.787c1.747-5.53,5.878-8.377,11.758-8.377 c5.88,0,9.532,3.012,11.28,6.9c0.107,0.274,0.107,0.548-0.21,0.712l-1.695,0.876c-0.265,0.11-0.53,0.055-0.688-0.22 c-1.802-3.396-4.45-5.312-8.74-5.312c-4.45,0-7.468,2.192-8.792,6.3c-0.475,1.422-0.845,3.722-0.845,9.909s0.37,8.486,0.845,9.91 c1.324,4.107,4.343,6.3,8.792,6.3c4.342,0,7.468-2.137,8.79-6.19c0.477-1.422,0.85-3.56,0.85-7.117c0-0.22-0.106-0.328-0.318-0.328 h-8.155c-0.318,0-0.53-0.22-0.53-0.548v-1.807c0-0.33,0.212-0.548,0.53-0.548h10.962c0.317,0,0.53,0.22,0.53,0.548v2.243 c0,3.725-0.423,6.572-0.954,8.27c-1.694,5.53-5.88,8.432-11.65,8.432c-5.88,0-10.01-2.848-11.758-8.38 C-3052.036,3458.811-3052.459,3456.345-3052.46,3449.885"></path><path style="fill:#666666" d="M-3021.43,3468.502c-0.317,0-0.53-0.22-0.53-0.547v-25.023c0-0.33,0.213-0.548,0.53-0.548h1.801 c0.316,0,0.53,0.22,0.53,0.548v3.338h0.052c0.953-2.572,3.23-4.434,6.78-4.434c2.013,0,3.866,0.712,5.084,1.807 c0.265,0.164,0.317,0.438,0.107,0.713l-1.06,1.531c-0.212,0.274-0.48,0.274-0.795,0.11c-1.164-0.767-2.436-1.313-4.024-1.313 c-4.45,0-6.144,3.996-6.144,8.815v14.456c0,0.328-0.213,0.547-0.53,0.547H-3021.43L-3021.43,3468.502z"></path><path style="fill:#666666" d="M-3004.54,3462.26c-0.53-1.752-0.847-3.668-0.847-6.844c0-3.12,0.316-5.037,0.847-6.79 c1.378-4.327,4.818-6.79,9.426-6.79c4.662,0,8.104,2.464,9.48,6.79c0.53,1.752,0.848,3.668,0.848,6.79 c0,3.176-0.317,5.092-0.848,6.844c-1.377,4.326-4.818,6.79-9.48,6.79C-2999.722,3469.05-3003.162,3466.585-3004.54,3462.26 M-2988.387,3461.33c0.475-1.48,0.688-3.066,0.688-5.914c0-2.794-0.213-4.38-0.688-5.86c-1.007-3.12-3.442-4.873-6.728-4.873 c-3.23,0-5.666,1.752-6.672,4.873c-0.477,1.48-0.688,3.065-0.688,5.86c0,2.848,0.212,4.436,0.688,5.914 c1.006,3.12,3.442,4.873,6.672,4.873C-2991.829,3466.203-2989.395,3464.451-2988.387,3461.33"></path><path style="fill:#666666" d="M-2963.66,3468.502c-0.317,0-0.528-0.22-0.528-0.547v-3.066h-0.053c-1.272,2.574-3.92,4.162-7.36,4.162 c-5.615,0-8.74-3.616-8.74-9.913v-16.206c0-0.33,0.213-0.548,0.53-0.548h1.801c0.317,0,0.53,0.22,0.53,0.548v15.44 c0,5.257,2.118,7.83,6.515,7.83c3.812,0,6.78-2.74,6.78-7.284v-15.987c0-0.33,0.21-0.548,0.528-0.548h1.802 c0.317,0,0.53,0.22,0.53,0.548v25.023c0,0.328-0.213,0.547-0.53,0.547h-1.802L-2963.66,3468.502z"></path><path style="fill:#666666" d="M-2955.29,3478.36c-0.317,0-0.53-0.22-0.53-0.549v-34.879c0-0.33,0.213-0.548,0.53-0.548h1.8 c0.317,0,0.53,0.22,0.53,0.548v3.176h0.053c1.218-2.465,3.496-4.272,7.68-4.272c4.45,0,7.256,2.08,8.58,6.242 c0.638,2.137,0.9,4.435,0.9,7.393c0,2.9-0.264,5.2-0.9,7.336c-1.323,4.164-4.13,6.242-8.58,6.242c-4.183,0-6.46-1.807-7.68-4.27 h-0.053v13.031c0,0.33-0.213,0.55-0.53,0.549L-2955.29,3478.36L-2955.29,3478.36z M-2939.35,3461.602 c0.53-1.643,0.688-3.832,0.688-6.13c0-2.355-0.158-4.545-0.688-6.188c-0.952-2.957-3.124-4.6-6.46-4.6 c-3.178,0-5.507,1.533-6.46,4.6c-0.477,1.423-0.69,3.34-0.69,6.188c0,2.847,0.212,4.71,0.69,6.13 c0.954,3.067,3.283,4.602,6.46,4.602C-2942.473,3466.204-2940.3,3464.561-2939.35,3461.602"></path><path style="fill:#666666" d="M-3070.48,3468.545c-0.424,0-0.634-0.164-0.85-0.547l-7.254-12.649h-0.106l-7.307,12.649 c-0.213,0.383-0.424,0.547-0.85,0.547h-5.72c-0.37,0-0.528-0.328-0.318-0.711l10.76-18.562l-9.96-17.248 c-0.21-0.385-0.05-0.712,0.32-0.712h5.721c0.424,0,0.636,0.163,0.847,0.548l6.513,11.278h0.106l6.513-11.278 c0.213-0.385,0.425-0.548,0.85-0.548h5.72c0.37,0,0.53,0.327,0.317,0.712l-9.956,17.248l10.75,18.562 c0.21,0.383,0.052,0.71-0.32,0.711h-5.771H-3070.48z"></path><path style="fill:#666666" d="M-2932.08,3443.951c-0.108,0-0.18-0.075-0.18-0.189v-10.385c0-0.075-0.036-0.113-0.11-0.113h-3.202 c-0.11,0-0.184-0.073-0.184-0.186v-1.6c0-0.11,0.074-0.186,0.184-0.186h8.746c0.11,0,0.183,0.075,0.183,0.186v1.6 c0,0.113-0.07,0.186-0.183,0.186h-3.201c-0.073,0-0.11,0.038-0.11,0.113v10.385c0,0.113-0.07,0.19-0.18,0.189H-2932.08 L-2932.08,3443.951z"></path><path style="fill:#666666" d="M-2924.47,3431.478c0-0.112,0.07-0.186,0.178-0.186h1.604c0.163,0,0.252,0.056,0.307,0.186l3.367,7.8h0.07 l3.315-7.8c0.052-0.13,0.144-0.186,0.305-0.186h1.583c0.11,0,0.182,0.074,0.182,0.186v12.286c0,0.11-0.07,0.186-0.182,0.186h-1.564 c-0.11,0-0.18-0.075-0.18-0.186v-7.744h-0.073l-2.593,5.957c-0.07,0.168-0.18,0.242-0.343,0.242h-1.044 c-0.16,0-0.27-0.074-0.343-0.242l-2.59-5.957h-0.072v7.744c0,0.11-0.072,0.186-0.18,0.186h-1.567c-0.106,0-0.178-0.075-0.178-0.186 v-12.286H-2924.47z"></path></g></svg></span></a></span></div></div></div></section></div></div></div>
<script type="application/json" data-iso-key="_0">{"userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.99 Safari/537.36 Zotero/5.0","abstracts":{"content":[{"#name":"abstract","$":{"xmlns:ce":true,"class":"graphical","lang":"en","id":"ab005","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st005"},"_":"Graphical abstract"},{"#name":"abstract-sec","$":{"id":"as005","view":"all"},"$$":[{"#name":"simple-para","$":{"id":"sp0005","view":"all"},"$$":[{"#name":"display","$$":[{"#name":"figure","$":{"id":"f0050"},"$$":[{"#name":"link","$":{"xmlns:xlink":true,"locator":"fx1","id":"lk00050","role":"http://data.elsevier.com/vocabulary/ElsevierContentTypes/23.4","href":"pii:S1055790317305900/fx1"}}]}]}]}]}]},{"#name":"abstract","$":{"xmlns:ce":true,"class":"author-highlights","lang":"en","id":"ab010","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st010"},"_":"Highlights"},{"#name":"abstract-sec","$":{"id":"as010","view":"all"},"$$":[{"#name":"simple-para","$":{"id":"sp0010","view":"all"},"$$":[{"#name":"list","$":{"id":"l0005"},"$$":[{"#name":"list-item","$":{"id":"o0005"},"$$":[{"#name":"label","_":"•"},{"#name":"para","$":{"id":"p0005","view":"all"},"$$":[{"#name":"__text__","_":"The first molecular phylogeny of the genus "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" (Coleoptera: Curculionidae) is herein constructed."}]}]},{"#name":"list-item","$":{"id":"o0010"},"$$":[{"#name":"label","_":"•"},{"#name":"para","$":{"id":"p0010","view":"all"},"_":"Genetic data are generated for two mitochondrial and two nuclear markers."}]},{"#name":"list-item","$":{"id":"o0015"},"$$":[{"#name":"label","_":"•"},{"#name":"para","$":{"id":"p0015","view":"all"},"_":"The evolution of weevil’s rostrum is related to the host plant part used (e.g., stems or fruits) to oviposit."}]},{"#name":"list-item","$":{"id":"o0020"},"$$":[{"#name":"label","_":"•"},{"#name":"para","$":{"id":"p0020","view":"all"},"_":"The haplotype phylogeny suggests a recent emergence of species through interaction with their host plants."}]}]}]}]}]},{"#name":"abstract","$":{"xmlns:ce":true,"class":"author","lang":"en","id":"ab015","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st015"},"_":"Abstract"},{"#name":"abstract-sec","$":{"id":"as015","view":"all"},"$$":[{"#name":"simple-para","$":{"id":"sp0015","view":"all"},"$$":[{"#name":"__text__","_":"The family Curculionidae (Coleoptera), the “true” weevils, have diversified tightly linked to the evolution of flowering plants. Here, we aim to assess diversification at a lower taxonomic level. We analyze the evolution of the genus "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" in association with their host plants. "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" comprises eight to thirteen species; their larvae feed inside the fruits of "},{"#name":"italic","_":"Datura"},{"#name":"__text__","_":" spp. or inside the stem of wild and cultivated species of Solanaceae, such as potato, tobacco and tomato. We ask the following questions: (1) does the rostrum of "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" species evolve according to the plant tissue used to oviposit, i.e., shorter rostrum to dig in stems and longer to dig in fruits? and (2) does "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" diversify mainly in relation to the use of "},{"#name":"italic","_":"Datura"},{"#name":"__text__","_":" species? For the first question, we estimated the phylogeny of "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" based on four gene sequences (nuclear 18S and 28S rRNA genes and mitochondrial 16S rRNA and COI genes). Then, we carried out morphogeometric analyses of the "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" species using 75 landmarks. For the second question, we calibrated a COI haplotype phylogeny using a constant rate of divergence to infer the diversification time of "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" species, and we traced the host plant species on the haplotype network. We performed an ancestral state reconstruction analysis to infer recent colonization events and conserved associations with host plant species. We found that ancestral species in the "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" phylogeny use the stem of "},{"#name":"italic","_":"Solanum"},{"#name":"__text__","_":" plants for oviposition and display weak sexual dimorphism of rostrum size, whereas other, more recent species of "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" display sexual dimorphism in rostrum size and use the fruits of "},{"#name":"italic","_":"Datura"},{"#name":"__text__","_":" species, and a possible reversion to use the stem of Solanaceae was detected in one "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" species. The use of "},{"#name":"italic","_":"Datura"},{"#name":"__text__","_":" species by "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" species is widely distributed on haplotype networks and restricted to "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" species that originated ca. 5 ± 1.5 Ma. Given that the origin of "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" is estimated to be ca. 6 ± 1.5 Ma, it is likely that "},{"#name":"italic","_":"Datura"},{"#name":"__text__","_":" has played a role in its diversification."}]}]}]}],"floats":[],"footnotes":[],"attachments":[{"attachment-eid":"1-s2.0-S1055790317305900-fx1.sml","ucs-locator":"https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790317305900/fx1/THUMBNAIL/image/gif/3883413e71dcc19d170f8f37c3e17ad3/fx1.sml","file-basename":"fx1","abstract-attachment":"true","filename":"fx1.sml","extension":"sml","filesize":"12905","pixel-height":"164","pixel-width":"164","attachment-type":"IMAGE-THUMBNAIL"},{"attachment-eid":"1-s2.0-S1055790317305900-fx1.jpg","ucs-locator":"https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790317305900/fx1/DOWNSAMPLED/image/jpeg/edc6436cd38c18e278d3a41c597104fb/fx1.jpg","file-basename":"fx1","abstract-attachment":"true","filename":"fx1.jpg","extension":"jpg","filesize":"25776","pixel-height":"245","pixel-width":"245","attachment-type":"IMAGE-DOWNSAMPLED"},{"attachment-eid":"1-s2.0-S1055790317305900-fx1_lrg.jpg","ucs-locator":"https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790317305900/fx1/HIGHRES/image/jpeg/5f8c7bb728febfa895c7194d60bfd0fe/fx1_lrg.jpg","file-basename":"fx1","abstract-attachment":"true","filename":"fx1_lrg.jpg","extension":"jpg","filesize":"161778","pixel-height":"1085","pixel-width":"1085","attachment-type":"IMAGE-HIGH-RES"}]},"biographies":{},"combinedContentItems":{"content":[{"#name":"keywords","$$":[{"#name":"keywords","$":{"xmlns:ce":true,"xmlns:aep":true,"xmlns:xoe":true,"xmlns:mml":true,"xmlns:xs":true,"xmlns:xlink":true,"xmlns:xocs":true,"xmlns:tb":true,"xmlns:xsi":true,"xmlns:cals":true,"xmlns:sb":true,"xmlns:sa":true,"xmlns:ja":true,"xmlns":true,"class":"keyword","id":"kg005","view":"all"},"$$":[{"#name":"section-title","$":{"id":"st020"},"_":"Keywords"},{"#name":"keyword","$":{"id":"k0005"},"$$":[{"#name":"text","$$":[{"#name":"italic","_":"Datura"}]}]},{"#name":"keyword","$":{"id":"k0010"},"$$":[{"#name":"text","_":"Colonization to host-plant"}]},{"#name":"keyword","$":{"id":"k0015"},"$$":[{"#name":"text","_":"Ancestral host-plant"}]},{"#name":"keyword","$":{"id":"k0020"},"$$":[{"#name":"text","$$":[{"#name":"italic","_":"Trichobaris"}]}]},{"#name":"keyword","$":{"id":"k0025"},"$$":[{"#name":"text","_":"Insect-plant interaction"}]}]}]}],"floats":[],"footnotes":[],"attachments":[]},"experiments":{},"rawtext":"","authors":{"content":[{"#name":"author-group","$":{"xmlns:ce":true,"id":"ag005"},"$$":[{"#name":"author","$":{"id":"au005","author-id":"S1055790317305900-8754313dfb8d50d1c599b311bb381be6"},"$$":[{"#name":"given-name","_":"Marisol"},{"#name":"surname","_":"De-la-Mora"},{"#name":"cross-ref","$":{"refid":"af005","id":"c0015"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"c0020"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]}]},{"#name":"author","$":{"id":"au010","author-id":"S1055790317305900-b5e0a7336c8a8124472710f292e20984"},"$$":[{"#name":"given-name","_":"Daniel"},{"#name":"surname","_":"Piñero"},{"#name":"cross-ref","$":{"refid":"af005","id":"c0025"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]}]},{"#name":"author","$":{"orcid":"0000-0001-5829-8338","id":"au015","author-id":"S1055790317305900-2456981de2042435f768432565d8eda9"},"$$":[{"#name":"given-name","_":"Ken"},{"#name":"surname","_":"Oyama"},{"#name":"cross-ref","$":{"refid":"af010","id":"c0030"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"b"}]}]},{"#name":"author","$":{"id":"au020","author-id":"S1055790317305900-35e43a999147ae420138974bb0a2088c"},"$$":[{"#name":"given-name","_":"Brian"},{"#name":"surname","_":"Farrell"},{"#name":"cross-ref","$":{"refid":"af015","id":"c0035"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"c"}]}]},{"#name":"author","$":{"id":"au025","author-id":"S1055790317305900-90b2316a08a85b7ea2d6939a46f670f5"},"$$":[{"#name":"given-name","_":"Susana"},{"#name":"surname","_":"Magallón"},{"#name":"cross-ref","$":{"refid":"af020","id":"c0040"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"d"}]}]},{"#name":"author","$":{"id":"au030","author-id":"S1055790317305900-868465659fb54f1f522c108b748f50a5"},"$$":[{"#name":"given-name","_":"Juan"},{"#name":"surname","_":"Núñez-Farfán"},{"#name":"cross-ref","$":{"refid":"af005","id":"c0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"c0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"xmlns:xlink":true,"type":"email","href":"mailto:farfan@unam.mx","id":"em005"},"_":"farfan@unam.mx"}]},{"#name":"affiliation","$":{"id":"af005","affiliation-id":"S1055790317305900-7c6d951905cd26dd43d7d103edac59d1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Departamento de Ecología Evolutiva, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, Mexico"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Departamento de Ecología EvolutivaInstituto de EcologíaUniversidad Nacional Autónoma de México"},{"#name":"city","_":"Ciudad de México"},{"#name":"country","_":"Mexico"}]}]},{"#name":"affiliation","$":{"id":"af010","affiliation-id":"S1055790317305900-9f1a6894e456ab81b083123b77e5e6b7"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Escuela Nacional de Estudios Superiores, Universidad Nacional Autónoma de México, Campus Morelia, Michoacán, Mexico"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Escuela Nacional de Estudios SuperioresUniversidad Nacional Autónoma de México"},{"#name":"address-line","_":"Campus Morelia"},{"#name":"city","_":"Michoacán"},{"#name":"country","_":"Mexico"}]}]},{"#name":"affiliation","$":{"id":"af015","affiliation-id":"S1055790317305900-2ccde510c951d2c6e5a67115b287a50f"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Museum of Comparative Zoology, Department of Evolutionary Biology, Harvard University, Cambridge, MA, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Museum of Comparative Zoology"},{"#name":"organization","_":"Department of Evolutionary BiologyHarvard University"},{"#name":"city","_":"Cambridge"},{"#name":"state","_":"MA"},{"#name":"country","_":"USA"}]}]},{"#name":"affiliation","$":{"id":"af020","affiliation-id":"S1055790317305900-d035d09f4505e0ccf948b790d8ee24af"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Departamento de Botánica, Instituto de Biología, Universidad Nacional Autónoma de México, Ciudad de México, Mexico"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Departamento de BotánicaInstituto de BiologíaUniversidad Nacional Autónoma de México"},{"#name":"city","_":"Ciudad de México"},{"#name":"country","_":"Mexico"}]}]},{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding authors."}]}]}],"floats":[],"footnotes":[],"affiliations":{"af005":{"#name":"affiliation","$":{"id":"af005","affiliation-id":"S1055790317305900-7c6d951905cd26dd43d7d103edac59d1"},"$$":[{"#name":"label","_":"a"},{"#name":"textfn","_":"Departamento de Ecología Evolutiva, Instituto de Ecología, Universidad Nacional Autónoma de México, Ciudad de México, Mexico"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Departamento de Ecología EvolutivaInstituto de EcologíaUniversidad Nacional Autónoma de México"},{"#name":"city","_":"Ciudad de México"},{"#name":"country","_":"Mexico"}]}]},"af010":{"#name":"affiliation","$":{"id":"af010","affiliation-id":"S1055790317305900-9f1a6894e456ab81b083123b77e5e6b7"},"$$":[{"#name":"label","_":"b"},{"#name":"textfn","_":"Escuela Nacional de Estudios Superiores, Universidad Nacional Autónoma de México, Campus Morelia, Michoacán, Mexico"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Escuela Nacional de Estudios SuperioresUniversidad Nacional Autónoma de México"},{"#name":"address-line","_":"Campus Morelia"},{"#name":"city","_":"Michoacán"},{"#name":"country","_":"Mexico"}]}]},"af015":{"#name":"affiliation","$":{"id":"af015","affiliation-id":"S1055790317305900-2ccde510c951d2c6e5a67115b287a50f"},"$$":[{"#name":"label","_":"c"},{"#name":"textfn","_":"Museum of Comparative Zoology, Department of Evolutionary Biology, Harvard University, Cambridge, MA, USA"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Museum of Comparative Zoology"},{"#name":"organization","_":"Department of Evolutionary BiologyHarvard University"},{"#name":"city","_":"Cambridge"},{"#name":"state","_":"MA"},{"#name":"country","_":"USA"}]}]},"af020":{"#name":"affiliation","$":{"id":"af020","affiliation-id":"S1055790317305900-d035d09f4505e0ccf948b790d8ee24af"},"$$":[{"#name":"label","_":"d"},{"#name":"textfn","_":"Departamento de Botánica, Instituto de Biología, Universidad Nacional Autónoma de México, Ciudad de México, Mexico"},{"#name":"affiliation","$":{"xmlns:sa":true},"$$":[{"#name":"organization","_":"Departamento de BotánicaInstituto de BiologíaUniversidad Nacional Autónoma de México"},{"#name":"city","_":"Ciudad de México"},{"#name":"country","_":"Mexico"}]}]}},"correspondences":{"cor1":{"#name":"correspondence","$":{"id":"cor1"},"$$":[{"#name":"label","_":"⁎"},{"#name":"text","_":"Corresponding authors."}]}},"attachments":[],"scopusAuthorIds":{},"articles":{}},"body":{},"exam":{},"article":{"publication-content":{"noElsevierLogo":false,"imprintPublisher":{"displayName":"Academic Press","id":"350"},"isSpecialIssue":false,"isSampleIssue":false,"transactionsBlocked":false,"publicationOpenAccess":{"oaStatus":"","oaArticleCount":119,"openArchiveStatus":false,"openArchiveArticleCount":0,"openAccessStartDate":"","oaAllowsAuthorPaid":true},"issue-cover":{"attachment":[{"attachment-eid":"1-s2.0-S1055790318X00050-cov200h.gif","file-basename":"cov200h","extension":"gif","filename":"cov200h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790318X00050/cover/DOWNSAMPLED200/image/gif/2ef0763602dfbf37e73a3db97206196a/cov200h.gif","https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790318X00050/cover/DOWNSAMPLED200/image/gif/2ef0763602dfbf37e73a3db97206196a/cov200h.gif"],"attachment-type":"IMAGE-COVER-H200","filesize":"18209","pixel-height":"200","pixel-width":"150"},{"attachment-eid":"1-s2.0-S1055790318X00050-cov150h.gif","file-basename":"cov150h","extension":"gif","filename":"cov150h.gif","ucs-locator":["https://s3-eu-west-1.amazonaws.com/prod-ucs-content-store-eu-west/content/pii:S1055790318X00050/cover/DOWNSAMPLED/image/gif/172946d8eeb302a62e31e4e77c9686d2/cov150h.gif","https://s3.amazonaws.com/prod-ucs-content-store-us-east/content/pii:S1055790318X00050/cover/DOWNSAMPLED/image/gif/172946d8eeb302a62e31e4e77c9686d2/cov150h.gif"],"attachment-type":"IMAGE-COVER-H150","filesize":"14405","pixel-height":"150","pixel-width":"113"}]},"smallCoverUrl":"https://ars.els-cdn.com/content/image/S10557903.gif","sourceOpenAccess":false,"publicationCoverImageUrl":"https://ars.els-cdn.com/content/image/1-s2.0-S1055790318X00050-cov150h.gif"},"pii":"S1055790317305900","dates":{"Available online":"24 February 2018","Received":"10 August 2017","Revised":["31 January 2018"],"Accepted":"16 February 2018","Publication date":"1 July 2018"},"access":{},"crawlerInformation":{"canCrawlPDFContent":false,"isCrawler":false},"analyticsMetadata":{"accountId":"262591","accountName":"International Institute of Tropical Agriculture","loginStatus":"anonymous","userId":"12289512"},"cid":"272322","content-family":"serial","copyright-line":"© 2018 Elsevier Inc. All rights reserved.","cover-date-years":["2018"],"cover-date-start":"2018-07-01","cover-date-text":"July 2018","document-subtype":"fla","document-type":"article","entitledToken":"517C89CDEF44F6A5E0A599E699387D23D71687F671870F48F13141BE6EDD25572A2EFA024FF2C27E","eid":"1-s2.0-S1055790317305900","doi":"10.1016/j.ympev.2018.02.018","first-fp":"37","hub-eid":"1-s2.0-S1055790318X00050","issuePii":"S1055790318X00050","item-weight":"FULL-TEXT","language":"en","last-lp":"49","last-author":{"#name":"last-author","$":{"xmlns:dm":true},"$$":[{"#name":"author","$":{"xmlns:ce":true,"id":"au030","author-id":"S1055790317305900-868465659fb54f1f522c108b748f50a5"},"$$":[{"#name":"given-name","_":"Juan"},{"#name":"surname","_":"Núñez-Farfán"},{"#name":"cross-ref","$":{"refid":"af005","id":"c0045"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"a"}]},{"#name":"cross-ref","$":{"refid":"cor1","id":"c0050"},"$$":[{"#name":"sup","$":{"loc":"post"},"_":"⁎"}]},{"#name":"e-address","$":{"xmlns:xlink":true,"type":"email","href":"mailto:farfan@unam.mx","id":"em005"},"_":"farfan@unam.mx"}]}]},"normalized-first-auth-initial":"M","normalized-first-auth-surname":"DELAMORA","pages":[{"last-page":"49","first-page":"37"}],"srctitle":"Molecular Phylogenetics and Evolution","suppl":"C","timestamp":"2018-04-12T10:21:22.199566Z","title":{"content":[{"#name":"title","$":{"xmlns:ce":true,"id":"tm005"},"$$":[{"#name":"__text__","_":"Evolution of "},{"#name":"italic","_":"Trichobaris"},{"#name":"__text__","_":" (Curculionidae) in relation to host plants: Geometric morphometrics, phylogeny and phylogeography"}]}],"floats":[],"footnotes":[],"attachments":[]},"vol-first":"124","vol-iss-suppl-text":"Volume 124","userSettings":{"forceAbstract":false,"creditCardPurchaseAllowed":false,"disableWholeIssueDownload":false,"preventTransactionalAccess":false,"preventDocumentDelivery":true},"contentType":"JL","crossmark":true,"issn":"10557903","issn-primary-formatted":"1055-7903","useEnhancedReader":true,"isCorpReq":false,"pdfDownload":{"linkType":"DOWNLOAD","linkToPdf":"/science/article/pii/S1055790317305900/pdfft?md5=2283f738c86c14daafaa5ea4c70b33c4&pid=1-s2.0-S1055790317305900-main.pdf","isPdfFullText":false,"fileName":"1-s2.0-S1055790317305900-main.pdf"},"pdfUrlForCrawlers":"https://www.sciencedirect.com/science/article/pii/S1055790317305900/pdfft?md5=2283f738c86c14daafaa5ea4c70b33c4&pid=1-s2.0-S1055790317305900-main.pdf","indexTag":true,"volRange":"124","issRange":"","userProfile":{"departmentName":"General Access","accessType":"IPRANGE","accountId":"262591","webUserId":"12289512","accountName":"International Institute of Tropical Agriculture","departmentId":"267018","userType":"NORMAL","hasMultipleOrganizations":false},"entitlementReason":"package","articleEntitlement":{"entitled":true},"aipType":"none","downloadFullIssue":true,"headerConfig":{"helpUrl":"https://service.elsevier.com/app/home/supporthub/sciencedirect/","contactUrl":"https://service.elsevier.com/app/contact/supporthub/sciencedirect/","userName":"","orgName":"","webUserId":"12289512","libraryBanner":{},"shib_regUrl":"","tick_regUrl":"","recentInstitutions":[],"canActivatePersonalization":false,"hasMultiOrg":false,"userType":"IPRANGE","allowCart":false},"open-research":{},"titleString":"Evolution of Trichobaris (Curculionidae) in relation to host plants: Geometric morphometrics, phylogeny and phylogeography","isAbstract":false,"onAbstractWhitelist":false,"isContentVisible":false,"ajaxLinks":{"citingArticles":true,"references":true,"referredToBy":true,"toc":true,"body":true,"recommendations":true}},"specialIssueArticles":{},"recommendations":{},"entitledRecommendations":{"isOpen":false,"articles":[],"selected":[],"currentPage":1,"totalPages":1},"citingArticles":{},"workspace":{"isOpen":false},"crossMark":{"isOpen":false},"userIdentity":{},"refersTo":{},"referredToBy":{},"downloadIssue":{"isOpen":false,"articles":[],"selected":[]},"references":{},"referenceLinks":{"internal":{},"external":{}},"glossary":{},"relatedContent":{"isModal":false,"isOpenSpecialIssueArticles":false,"isOpenRecommendations":true,"isOpenCitingArticles":false,"citingArticles":[false,false,false],"recommendations":[false,false,false,false,false,false],"specialIssueArticles":[false,false,false]},"banner":{"expanded":false},"transientError":{"isOpen":false},"pdfDropdown":{},"exportCitation":{"isOpen":false},"tableOfContents":{"showEntitledTocLinks":true},"chapters":{"toc":[],"isLoading":false},"enrichedContent":{"tableOfContents":false,"researchData":{"hasResearchData":false,"dataProfile":{},"openData":{},"mendeleyData":{},"databaseLinking":{}},"geospatialData":{"attachments":[]},"interactiveCaseInsights":{},"virtualMicroscope":{}},"metrics":{"isOpen":true},"signOut":{"isOpen":false},"issueNavigation":{"previous":{},"next":{}},"tail":{}}</script>
      
      <script src="satelliteLib-b7cfe8df39a4e5eec5536bba80e13f4b6fa0dd7c.js"></script><script src="satellite-565e008964746d4385002642.js"></script>
      <script src="babel-polyfill.js"></script>
      <script src="react.js"></script>
      <script src="react-dom.js"></script>
      <script src="arp.js" async=""></script>
      <!-- begin usabilla live embed code -->
      <script type="text/javascript">
        window.lightningjs||function(c){function g(b,d){d&&(d+=(/\?/.test(d)?"&":"?")+"lv=1");c[b]||function(){var i=window,h=document,j=b,g=h.location.protocol,l="load",k=0;(function(){function b(){a.P(l);a.w=1;c[j]("_load")}c[j]=function(){function m(){m.id=e;return c[j].apply(m,arguments)}var b,e=++k;b=this&&this!=i?this.id||0:0;(a.s=a.s||[]).push([e,b,arguments]);m.then=function(b,c,h){var d=a.fh[e]=a.fh[e]||[],j=a.eh[e]=a.eh[e]||[],f=a.ph[e]=a.ph[e]||[];b&&d.push(b);c&&j.push(c);h&&f.push(h);return m};return m};var a=c[j]._={};a.fh={};a.eh={};a.ph={};a.l=d?d.replace(/^\/\//,(g=="https:"?g:"http:")+"//"):d;a.p={0:+new Date};a.P=function(b){a.p[b]=new Date-a.p[0]};a.w&&b();i.addEventListener?i.addEventListener(l,b,!1):i.attachEvent("on"+l,b);var q=function(){function b(){return["<head></head><",c,' onload="var d=',n,";d.getElementsByTagName('head')[0].",d,"(d.",g,"('script')).",i,"='",a.l,"'\"></",c,">"].join("")}var c="body",e=h[c];if(!e)return setTimeout(q,100);a.P(1);var d="appendChild",g="createElement",i="src",k=h[g]("div"),l=k[d](h[g]("div")),f=h[g]("iframe"),n="document",p;k.style.display="none";e.insertBefore(k,e.firstChild).id=o+"-"+j;f.frameBorder="0";f.id=o+"-frame-"+j;/MSIE[ ]+6/.test(navigator.userAgent)&&(f[i]="javascript:false");f.allowTransparency="true";l[d](f);try{f.contentWindow[n].open()}catch(s){a.domain=h.domain,p="javascript:var d="+n+".open();d.domain='"+h.domain+"';",f[i]=p+"void(0);"}try{var r=f.contentWindow[n];r.write(b());r.close()}catch(t){f[i]=p+'d.write("'+b().replace(/"/g,String.fromCharCode(92)+'"')+'");d.close();'}a.P(2)};a.l&&setTimeout(q,0)})()}();c[b].lv="1";return c[b]}var o="lightningjs",k=window[o]=g(o);k.require=g;k.modules=c}({});
        window.usabilla_live = lightningjs.require("usabilla_live", "https://w.usabilla.com/eb1c14a91932.js");
        var customData = {};

        if(window.pageData && pageData.content && pageData.content[0]) {
          customData.entitlementType = pageData.content[0].entitlementType;
        }
        if(window.pageData && pageData.visitor) {
          customData.accessType = pageData.visitor.accessType;
          customData.accountId = pageData.visitor.accountId;
          customData.loginStatus = pageData.visitor.loginStatus;
        }
        usabilla_live("data", {"custom": customData });
      </script>
      <!-- end usabilla live embed code -->
      <script type="text/x-mathjax-config">
        MathJax.Hub.Config({
          displayAlign: 'left',
          "fast-preview": {
            disabled: true
          },
          CommonHTML: { linebreaks: { automatic: true } },
          PreviewHTML: { linebreaks: { automatic: true } },
          'HTML-CSS': { linebreaks: { automatic: true } },
          SVG: {
            scale: 90,
            linebreaks: { automatic: true }
          }
        });
      </script>
      <script async="" src="MathJax.js"></script>
      <script async="" src="gpt.js"></script>
    
  <div class="js-react-modal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><div class="ReactModalPortal"></div><script src="widget-summary.js" async=""></script></body></html>