<!DOCTYPE html>
<html class="js" style="--font-family-sans: Europa; --font-family-serif: Noto Serif;" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
        <title>msBayes: Pipeline for testing comparative phylogeographic
 histories using hierarchical approximate Bayesian computation | BMC 
Bioinformatics | Full Text</title>

        

    <meta name="citation_abstract" content="Although testing for simultaneous divergence (vicariance) across different population-pairs that span the same barrier to gene flow is of central importance to evolutionary biology, researchers often equate the gene tree and population/species tree thereby ignoring stochastic coalescent variance in their conclusions of temporal incongruence. In contrast to other available phylogeographic software packages, msBayes is the only one that analyses data from multiple species/population pairs under a hierarchical model. msBayes employs approximate Bayesian computation (ABC) under a hierarchical coalescent model to test for simultaneous divergence (TSD) in multiple co-distributed population-pairs. Simultaneous isolation is tested by estimating three hyper-parameters that characterize the degree of variability in divergence times across co-distributed population pairs while allowing for variation in various within population-pair demographic parameters (sub-parameters) that can affect the coalescent. msBayes is a software package consisting of several C and R programs that are run with a Perl " front-end".="" the="" method="" reasonably="" distinguishes="" simultaneous="" isolation="" from="" temporal="" incongruence="" in="" divergence="" of="" co-distributed="" population="" pairs,="" even="" with="" sparse="" sampling="" individuals.="" because="" estimate="" step="" is="" decoupled="" simulation="" step,="" one="" can="" rapidly="" evaluate="" different="" abc="" acceptance="" rejection="" conditions="" and="" choice="" summary="" statistics.="" given="" complex="" idiosyncratic="" nature="" testing="" multi-species="" biogeographic="" hypotheses,="" we="" envision="" msbayes="" as="" a="" powerful="" flexible="" tool="" for="" tackling="" wide="" array="" difficult="" research="" questions="" that="" use="" genetic="" data="" multiple="" species.="" pipeline="" available="" download="" at="" http:="" msbayes.sourceforge.net="" under="" an="" open="" source="" license="" (gnu="" public="" license).="" comprised="" several="" c="" r="" programs="" are="" run="" perl="" "front-end"="" runs="" on="" linux,="" mac="" os-x,="" most="" posix="" systems.="" although="" current="" implementation="" single="" locus="" per="" species-pair,="" future="" implementations="" will="" allow="" analysis="" multi-loci="" species="" pair."="">

    <meta name="description" content="Although testing for simultaneous divergence (vicariance) across different population-pairs that span the same barrier to gene flow is of central importance to evolutionary biology, researchers often equate the gene tree and population/species tree thereby ignoring stochastic coalescent variance in their conclusions of temporal incongruence. In contrast to other available phylogeographic software packages, msBayes is the only one that analyses data from multiple species/population pairs under a hierarchical model. msBayes employs approximate Bayesian computation (ABC) under a hierarchical coalescent model to test for simultaneous divergence (TSD) in multiple co-distributed population-pairs. Simultaneous isolation is tested by estimating three hyper-parameters that characterize the degree of variability in divergence times across co-distributed population pairs while allowing for variation in various within population-pair demographic parameters (sub-parameters) that can affect the coalescent. msBayes is a software package consisting of several C and R programs that are run with a Perl " front-end".="" the="" method="" reasonably="" distinguishes="" simultaneous="" isolation="" from="" temporal="" incongruence="" in="" divergence="" of="" co-distributed="" population="" pairs,="" even="" with="" sparse="" sampling="" individuals.="" because="" estimate="" step="" is="" decoupled="" simulation="" step,="" one="" can="" rapidly="" evaluate="" different="" abc="" acceptance="" rejection="" conditions="" and="" choice="" summary="" statistics.="" given="" complex="" idiosyncratic="" nature="" testing="" multi-species="" biogeographic="" hypotheses,="" we="" envision="" msbayes="" as="" a="" powerful="" flexible="" tool="" for="" tackling="" wide="" array="" difficult="" research="" questions="" that="" use="" genetic="" data="" multiple="" species.="" pipeline="" available="" download="" at="" http:="" msbayes.sourceforge.net="" under="" an="" open="" source="" license="" (gnu="" public="" license).="" comprised="" several="" c="" r="" programs="" are="" run="" perl="" "front-end"="" runs="" on="" linux,="" mac="" os-x,="" most="" posix="" systems.="" although="" current="" implementation="" single="" locus="" per="" species-pair,="" future="" implementations="" will="" allow="" analysis="" multi-loci="" species="" pair."="">

    <meta name="journal_id" content="12859">

    <meta name="dc.title" content="msBayes: Pipeline for testing comparative phylogeographic histories using hierarchical approximate Bayesian computation">

    <meta name="dc.source" content="BMC Bioinformatics 2007 8:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="BioMed Central">

    <meta name="dc.date" content="2007-07-26">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2007 Hickerson et al; licensee BioMed Central Ltd.">

    <meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0">

    <meta name="dc.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="dc.description" content="Although testing for simultaneous divergence (vicariance) across different population-pairs that span the same barrier to gene flow is of central importance to evolutionary biology, researchers often equate the gene tree and population/species tree thereby ignoring stochastic coalescent variance in their conclusions of temporal incongruence. In contrast to other available phylogeographic software packages, msBayes is the only one that analyses data from multiple species/population pairs under a hierarchical model. msBayes employs approximate Bayesian computation (ABC) under a hierarchical coalescent model to test for simultaneous divergence (TSD) in multiple co-distributed population-pairs. Simultaneous isolation is tested by estimating three hyper-parameters that characterize the degree of variability in divergence times across co-distributed population pairs while allowing for variation in various within population-pair demographic parameters (sub-parameters) that can affect the coalescent. msBayes is a software package consisting of several C and R programs that are run with a Perl &quot;front-end&quot;. The method reasonably distinguishes simultaneous isolation from temporal incongruence in the divergence of co-distributed population pairs, even with sparse sampling of individuals. Because the estimate step is decoupled from the simulation step, one can rapidly evaluate different ABC acceptance/rejection conditions and the choice of summary statistics. Given the complex and idiosyncratic nature of testing multi-species biogeographic hypotheses, we envision msBayes as a powerful and flexible tool for tackling a wide array of difficult research questions that use population genetic data from multiple co-distributed species. The msBayes pipeline is available for download at 
                    http://msbayes.sourceforge.net/
                    
                   under an open source license (GNU Public License). The msBayes pipeline is comprised of several C and R programs that are run with a Perl &quot;front-end&quot; and runs on Linux, Mac OS-X, and most POSIX systems. Although the current implementation is for a single locus per species-pair, future implementations will allow analysis of multi-loci data per species pair.">

    <meta name="prism.issn" content="1471-2105">

    <meta name="prism.publicationName" content="BMC Bioinformatics">

    <meta name="prism.publicationDate" content="2007-07-26">

    <meta name="prism.volume" content="8">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="268">

    <meta name="prism.copyright" content="2007 Hickerson et al; licensee BioMed Central Ltd.">

    <meta name="prism.rightsAgent" content="reprints@biomedcentral.com">

    <meta name="prism.url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-268">

    <meta name="prism.doi" content="doi:10.1186/1471-2105-8-268">

    <meta name="citation_pdf_url" content="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-268">

    <meta name="citation_fulltext_html_url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-268">

    <meta name="citation_journal_title" content="BMC Bioinformatics">

    <meta name="citation_publisher" content="BioMed Central">

    <meta name="citation_issn" content="1471-2105">

    <meta name="citation_title" content="msBayes: Pipeline for testing comparative phylogeographic histories using hierarchical approximate Bayesian computation">

    <meta name="citation_volume" content="8">

    <meta name="citation_issue" content="1">

    <meta name="citation_publication_date" content="2007/12">

    <meta name="citation_online_date" content="2007/07/26">

    <meta name="citation_firstpage" content="268">

    <meta name="citation_article_type" content="Software">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="dc.identifier" content="doi:10.1186/1471-2105-8-268">

    <meta name="DOI" content="10.1186/1471-2105-8-268">

    <meta name="citation_doi" content="10.1186/1471-2105-8-268">

    <meta name="dc.creator" content="Michael J Hickerson">

    <meta name="dc.creator" content="Eli Stahl">

    <meta name="dc.creator" content="Naoki Takebayashi">

    <meta name="dc.subject" content="Bioinformatics">

    <meta name="dc.subject" content="Microarrays">

    <meta name="dc.subject" content="Computational Biology/Bioinformatics">

    <meta name="dc.subject" content="Computer Appl. in Life Sciences">

    <meta name="dc.subject" content="Combinatorial Libraries">

    <meta name="dc.subject" content="Algorithms">

    <meta name="citation_reference" content="citation_inbook_title=Phylogeography: The history and formation of species; citation_publication_date=2000; citation_id=CR1; citation_author=JC Avise; citation_publisher=Harvard University Press">

    <meta name="citation_reference" content="citation_journal_title=Annu Rev Ecol Syst; citation_title=Origin and the evolution of the eastern North American distinct distributions of flowering plants; citation_author=J Wen; citation_volume=30; citation_publication_date=1999; citation_pages=421-455; citation_doi=10.1146/annurev.ecolsys.30.1.421; citation_id=CR2">

    <meta name="citation_reference" content="citation_title=The first stage of speciation as seen in organisms separated by the Isthmus Panama; citation_inbook_title=Endless Forms: species and speciation; citation_publication_date=1998; citation_pages=186-201; citation_id=CR3; citation_author=HA Lessios; citation_publisher=Oxford University Press">

    <meta name="citation_reference" content="citation_journal_title=Trends Ecol Evol; citation_title=Phylogenetics and speciation; citation_author=TG Barraclough, S Nee; citation_volume=16; citation_issue=7; citation_publication_date=2001; citation_pages=391-399; citation_doi=10.1016/S0169-5347(01)02161-9; citation_id=CR4">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Statistical phylogeography; citation_author=LL Knowles, WP Maddison; citation_volume=11; citation_issue=12; citation_publication_date=2002; citation_pages=2623-2635; citation_doi=10.1046/j.1365-294X.2002.01637.x; citation_id=CR5">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=High-resolution species trees without concatination; citation_author=SV Edwards, L Liu, DK Pearl; citation_volume=104; citation_publication_date=2007; citation_pages=5936-5941; citation_doi=10.1073/pnas.0607004104; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Bayes Estimation of Species Divergence Times and Ancestral Population Sizes Using DNA Sequences From Multiple Loci; citation_author=B Rannala, ZH Yang; citation_volume=164; citation_publication_date=2003; citation_pages=1645-1656; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Integration within the Felsenstein equation for improved Markov chain Monte Carlo methods in population genetics; citation_author=J Hey, R Nielsen; citation_volume=104; citation_publication_date=2007; citation_pages=2785-2790; citation_doi=10.1073/pnas.0611164104; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Perspective: Gene divergence, population divergence, and the variance in coalescence time in phylogeographic studies; citation_author=SV Edwards, P Beerli; citation_volume=54; citation_issue=6; citation_publication_date=2000; citation_pages=1839-1854; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Test for simultaneous divergence using approximate Bayesian computation; citation_author=MJ Hickerson, E Stahl, HA Lessios; citation_volume=60; citation_publication_date=2006; citation_pages=2435-2453; citation_doi=10.1111/j.0014-3820.2006.tb01880.x; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Approximate Bayesian inference reveals evidence for a recent, severe bottleneck in a Netherlands population of Drosophila melanogaster; citation_author=K Thornton, P Andolfatto; citation_volume=172; citation_publication_date=2005; citation_pages=1607-1619; citation_doi=10.1534/genetics.105.048223; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Bayesian analysis of an admixture model with mutations and arbitrarily linked markers; citation_author=L Excoffier, A Estoup, J-M Cornuet; citation_volume=169; citation_publication_date=2005; citation_pages=1727-1738; citation_doi=10.1534/genetics.104.036236; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=Genetic analysis of complex demographic scenarios: spatially expanding populations of the cane toad, Bufo marinus; citation_author=A Estoup, BA Beaumont, F Sennedot, C Moritz, J-M Cornuet; citation_volume=58; citation_publication_date=2004; citation_pages=2021-2036; citation_doi=10.1111/j.0014-3820.2004.tb00487.x; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Comparative evaluation of a new effective population size estimator based on approximate Bayesian computation; citation_author=DA Tallmon, G Luikart, BA Beaumont; citation_volume=167; citation_publication_date=2004; citation_pages=977-988; citation_doi=10.1534/genetics.103.026146; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Approximate Bayesian computation in population genetics; citation_author=MA Beaumont, W Zhang, DJ Balding; citation_volume=162; citation_publication_date=2002; citation_pages=2025-2035; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Inference of population history using a likelihood approach; citation_author=G Weiss, A von Haeseler; citation_volume=149; citation_publication_date=1998; citation_pages=1539-1546; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Theor Popul Biol; citation_title=On the number of segregating sites in genetic models without recombination; citation_author=GA Watterson; citation_volume=7; citation_publication_date=1975; citation_pages=256-276; citation_doi=10.1016/0040-5809(75)90020-9; citation_id=CR17">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Mathematical model for studying variation in terms of restriction endonucleases; citation_author=M Nei, W Li; citation_volume=76; citation_publication_date=1979; citation_pages=5269-5273; citation_doi=10.1073/pnas.76.10.5269; citation_id=CR18">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Comparative phylogeographic summary statistics for testing simultaneous vicariance across taxon-pairs; citation_author=MJ Hickerson, G Dolman, C Moritz; citation_volume=15; citation_publication_date=2006; citation_pages=209-224; citation_doi=10.1111/j.1365-294X.2005.02718.x; citation_id=CR19">

    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Generating samples under a Wright-Fisher neutral model of genetic variation; citation_author=RR Hudson; citation_volume=18; citation_publication_date=2002; citation_pages=337-338; citation_doi=10.1093/bioinformatics/18.2.337; citation_id=CR20">

    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=Multilocus methods for estimating population sizes, migration rates and divergence time, with applications to the divergence of Drosophila pseudoobscura and D. persimilis; citation_author=J Hey, R Nielsen; citation_volume=167; citation_publication_date=2004; citation_pages=747-760; citation_doi=10.1534/genetics.103.024182; citation_id=CR21">

    <meta name="citation_reference" content="citation_journal_title=Nat Rev Genet; citation_title=The Bayesian revolution in genetics; citation_author=MA Beaumont, B Rannala; citation_volume=5; citation_publication_date=2004; citation_pages=251-261; citation_doi=10.1038/nrg1318; citation_id=CR22">

    <meta name="citation_reference" content="citation_title=Estimation with quadradic loss; citation_inbook_title=Proceedings of the Fourth Berkeley Symposium on Mathematical Statistics and Probability: 1960; citation_publication_date=1960; citation_id=CR23; citation_author=W James; citation_author=C Stein; citation_publisher=University of California Press">

    <meta name="citation_reference" content="citation_inbook_title=Bayesian Data Analysis; citation_publication_date=1995; citation_id=CR24; citation_author=A Gelman; citation_author=JB Carlin; citation_author=HS Stern; citation_author=DB Rubin; citation_publisher=Chapman and Hall/CRC">

    <meta name="citation_reference" content="citation_journal_title=Proc Natl Acad Sci USA; citation_title=Reconciling paleodistribution models and comparative phylogeography in the Wet Tropics rainforest land snail Gnarosophia bellendenkerensis (Brazier 1875); citation_author=A Hugall, C Moritz, A Moussalli, J Stanisic; citation_volume=99; citation_issue=9; citation_publication_date=2002; citation_pages=6112-6117; citation_doi=10.1073/pnas.092538699; citation_id=CR25">

    <meta name="citation_reference" content="citation_journal_title=Journal of the American Statistical Association; citation_title=Bayesian factors; citation_author=RE Kass, A Raftery; citation_volume=90; citation_publication_date=1995; citation_pages=773-795; citation_doi=10.1080/01621459.1995.10476572; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Mol Ecol; citation_title=Evolutionary and statistical properties of three genetic distances; citation_author=ST Kalinowski; citation_volume=11; citation_issue=8; citation_publication_date=2002; citation_pages=1263-1273; citation_doi=10.1046/j.1365-294X.2002.01520.x; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=Ann Rev Ecol Syst; citation_title=Gene flow in natural populations; citation_author=M Slatkin; citation_volume=16; citation_publication_date=1985; citation_pages=393-430; citation_doi=10.1146/annurev.es.16.110185.002141; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Theor Popul Biol; citation_title=The variance of pairwise nucleotide differences in two populations with migration; citation_author=J Wakeley; citation_volume=49; citation_publication_date=1996; citation_pages=39-57; citation_doi=10.1006/tpbi.1996.0002; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=Theor Popul Biol; citation_title=Distinguishing migration from isolation using the variance of pairwise differences; citation_author=J Wakeley; citation_volume=49; citation_publication_date=1996; citation_pages=369-386; citation_doi=10.1006/tpbi.1996.0018; citation_id=CR30">

    <meta name="citation_author" content="Michael J Hickerson">

    <meta name="citation_author_institution" content="Biology Department, Queens College, Flushing, USA">

    <meta name="citation_author" content="Eli Stahl">

    <meta name="citation_author_institution" content="Department of Biology, University of Massachusetts Dartmouth, North Dartmouth, USA">

    <meta name="citation_author" content="Naoki Takebayashi">

    <meta name="citation_author_institution" content="Institute of Arctic Biology and Department of Biology and Wildlife, University of Alaska, Fairbanks, USA">



        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">

        <!-- For IE 9 and below -->
<!--[if IE]><link rel="shortcut icon" href=/static/images/favicons//favicon.ico><![endif]-->
<!-- Touch Icons - iOS and Android 2.1+ -->
<link rel="apple-touch-icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">
<!-- Firefox, Chrome, Safari, IE 11+ and Opera -->
<link rel="icon" href="https://bmcbioinformatics.biomedcentral.com/static/images/favicons/bmc/favicon-3f5c56eaaa.png">


        <script type="text/javascript" src="https://consumer.krxd.net/consent/set/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;dc=1&amp;al=1&amp;tg=1&amp;cd=0&amp;sh=0&amp;re=0&amp;callback=Krux.ns.macmillan.kxjsonp_consent_set_1"></script><script type="text/javascript" src="https://consumer.krxd.net/consent/get/bd339c69-af54-4a21-b4f1-654bcfcd83ca?idt=device&amp;dt=kxcookie&amp;callback=Krux.ns.macmillan.kxjsonp_consent_get_0"></script><script async="" src="//cdn.krxd.net/ctjs/controltag.js.135427d411750943a27d8118ae946121"></script><script type="text/javascript" async="" src="MathJax.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="KDqylSLE.js"></script><script async="" src="gtm.js"></script><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script>
        <script>(function(d){if (sessionStorage.fontsLoaded){d.style.setProperty('--font-family-sans', 'Europa');d.style.setProperty('--font-family-serif', 'Noto Serif')}})(document.documentElement);</script>

        <link rel="stylesheet" media="screen" href="basic-a31ca85bc2.css">
<link rel="stylesheet" media="print" href="print-3a6240f456.css">

<link rel="stylesheet" class="js-ctm" href="main-84414d3d47.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">

        

        <script src="app-bundle-8d61e6ef37.js" defer="defer"></script>
        
        <script>
            window.airbrakeErrors = [];
            window.onerror = function(error) {window.airbrakeErrors.push(error);}
        </script>
        
    <script type="text/javascript">
        config = {
            env: 'live',
            site: 'bmcbioinformatics.biomedcentral.com',
            siteWithPath: 'bmcbioinformatics.biomedcentral.com' + window.location.pathname,
            twitterHashtag: 'bmcbioinformatics',
            cmsPrefix: 'https://studio-cms.springernature.com/studio/',
            
            doi: '10.1186/1471-2105-8-268',
            figshareScriptUrl: 'https://widgets.figshare.com/static/figshare.js',
            publisherBrand: 'BioMed Central'
        };
    </script>

        
    <script type="text/javascript">
        dataLayer = [{"content":{"article":{"doi":"10.1186/1471-2105-8-268","articleType":"Software","peerReviewType":"Closed","supplement":null,"keywords":["Root Mean Square Error","Posterior Distribution","Approximate Bayesian Computation","Population Pair","Simultaneous Divergence"]},"contentInfo":{"imprint":"BioMed Central","title":"msBayes: Pipeline for testing comparative phylogeographic histories using hierarchical approximate Bayesian computation","publishedAt":1185408000000,"publishedAtDate":"2007-07-26","author":["Michael J Hickerson","Eli Stahl","Naoki Takebayashi"],"collection":[]},"attributes":{"deliveryPlatform":"oscar","template":"rebrand","cms":null,"copyright":{"creativeCommonsType":null,"openAccess":false},"environment":"live"},"journal":{"siteKey":"bmcbioinformatics.biomedcentral.com","volume":"8","issue":"1","title":"BMC Bioinformatics","type":"SERIES","journalID":12859,"gaCode":"UA-62065396-3","section":[]},"category":{"pmc":{"primarySubject":"Life Sciences"},"contentType":"Software","publishingSegment":"BMC-series Biology"}},"session":{"authentication":{"authenticationID":[]}},"version":"1.0.0","page":{"category":{"pageType":"article"}}}]

    </script>

        <script src="7d6f4b4f-77fb-43b9-b429-eb1e1eeac6aa.js" id="onetrust-bmc-control" charset="UTF-8"></script><script type="text/javascript" src="jquery.js"></script><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color: #F8F8F8 !important }#optanon ul#optanon-menu li.menu-item-selected { background-color: #C5F6F9 !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #1B3051 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #1B3051 !important; border-color: #1B3051 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle a { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle a { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #FFFFFF !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #FFFFFF !important }.optanon-alert-box-wrapper { background-color:#EFF6FC !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#333333 !important }</style>
        
    
        
    

        
    
        
            
                <!-- Google Tag Manager -->
                <script>
                    (function (w, d, s, l, i) {
                    w[l] = w[l] || [];
                    w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                    var f = d.getElementsByTagName(s)[0],
                            j = d.createElement(s),
                            dl = l != 'dataLayer' ? '&l=' + l : '';
                    j.async = true;
                    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                    f.parentNode.insertBefore(j, f);
                })(window, document, 'script', 'dataLayer', 'GTM-TDGJHK');
                </script>
                <!-- End Google Tag Manager -->
            
        
    


        
    
    <link rel="canonical" href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-268">
    

        
        
        <meta property="og:url" content="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-268">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="BMC Bioinformatics">
        <meta property="og:title" content="msBayes: Pipeline for testing comparative phylogeographic histories using hierarchical approximate Bayesian computation">
        <meta property="og:description" content="Although testing for simultaneous divergence (vicariance) across different population-pairs that span the same barrier to gene flow is of central importance to evolutionary biology, researchers often equate the gene tree and population/species tree thereby ignoring stochastic coalescent variance in their conclusions of temporal incongruence. In contrast to other available phylogeographic software packages, msBayes is the only one that analyses data from multiple species/population pairs under a hierarchical model. msBayes employs approximate Bayesian computation (ABC) under a hierarchical coalescent model to test for simultaneous divergence (TSD) in multiple co-distributed population-pairs. Simultaneous isolation is tested by estimating three hyper-parameters that characterize the degree of variability in divergence times across co-distributed population pairs while allowing for variation in various within population-pair demographic parameters (sub-parameters) that can affect the coalescent. msBayes is a software package consisting of several C and R programs that are run with a Perl &quot;front-end&quot;. The method reasonably distinguishes simultaneous isolation from temporal incongruence in the divergence of co-distributed population pairs, even with sparse sampling of individuals. Because the estimate step is decoupled from the simulation step, one can rapidly evaluate different ABC acceptance/rejection conditions and the choice of summary statistics. Given the complex and idiosyncratic nature of testing multi-species biogeographic hypotheses, we envision msBayes as a powerful and flexible tool for tackling a wide array of difficult research questions that use population genetic data from multiple co-distributed species. The msBayes pipeline is available for download at http://msbayes.sourceforge.net/ under an open source license (GNU Public License). The msBayes pipeline is comprised of several C and R programs that are run with a Perl &quot;front-end&quot; and runs on Linux, Mac OS-X, and most POSIX systems. Although the current implementation is for a single locus per species-pair, future implementations will allow analysis of multi-loci data per species pair.">
        <meta property="og:image" content="https://images.springer.com/sgw/journals/medium/12859.jpg">
    
    <script type="text/javascript" src="jquery_002.js"></script><script src="figshare.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_270.js" async=""></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>

    <body class="journal journal-fulltext"><div id="MathJax_Message" style="display: none;"></div>
        
    
        
            
                <!-- Google Tag Manager (noscript) -->
                <noscript>
                    <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TDGJHK"
                            height="0" width="0" style="display:none;visibility:hidden"></iframe>
                </noscript>
                <!-- End Google Tag Manager (noscript) -->
            
        
    


        

        

    <div class="u-vh-full">
        <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg"><defs><filter id="a" filterUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01"><feColorMatrix values="1 0 0 0 0 0 1 0 0 0 0 0 1 0 0 0 0 0 1 0"></feColorMatrix></filter></defs><symbol id="icon-arrow-down" viewBox="0 0 16 8"><path d="M8 6l6-6h2L8 8 0 0h2l6 6"></path></symbol><symbol id="icon-arrow-left-bullet" viewBox="0 0 8 16"><path d="M3 8l5 5v3L0 8l8-8v3L3 8z"></path></symbol><symbol id="icon-arrow-right-bullet" viewBox="0 0 8 16"><path d="M5 8L0 3V0l8 8-8 8v-3l5-5z"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 32 64"><path d="M0 0v64l32-32z"></path></symbol><symbol id="icon-close" viewBox="0 0 23.998 23.999"><path d="M20.998 0l-8.999 9L3 0 0 3l8.999 9L0 20.999l3 3L11.999 15l8.999 8.999 3-3L14.999 12l8.999-9z"></path></symbol><symbol id="icon-community" viewBox="0 0 60.643 31.01"><mask maskUnits="userSpaceOnUse" x="0" y="0" width="60.643" height="31.01" id="b"><path fill="#FFF" d="M0 31.01h60.643V0H0z" filter="url(#a)"></path></mask><path mask="url(#b)" d="M60.643 25.497L50.005 14.86a7.925 7.925 0 1 0-7.669 0l-6.847 6.847-1.334-1.334a7.925 7.925 0 0 0-3.834-14.861 7.925 7.925 0 0 0-3.834 14.861l-1.334 1.334-6.847-6.847a7.925 7.925 0 1 0-7.669 0L0 25.497h21.362L15.85 31.01h28.943l-5.514-5.513h21.364z"></path></symbol><symbol id="icon-download" viewBox="-301 390 9 14"><path d="M-301 395.6l4.5 5.1 4.5-5.1h-3V390h-3v5.6h-3zm0 6.5h9v1.9h-9z"></path></symbol><symbol id="icon-ext-link" viewBox="0 0 16 16"><path fill="#676767" d="M12.9 16H3.1C1.4 16 0 14.6 0 12.9V3.2C0 1.4 1.4 0 3.1 0h3.7v1H3.1C2 1 1 2 1 3.2v9.7C1 14 2 15 3.1 15h9.7c1.2 0 2.1-1 2.1-2.1V8.7h1v4.2c.1 1.7-1.3 3.1-3 3.1z"></path><path fill="#676767" d="M12.8 2.5l.7.7-9 8.9-.7-.7 9-8.9z"></path><path fill="#676767" d="M9.7 0L16 6.2V0z"></path></symbol><symbol id="icon-info-bordered" viewBox="470.812 270.868 18 18"><path d="M479.812 270.868c-4.972 0-9 4.029-9 9s4.028 9 9 9c4.971 0 9-4.029 9-9s-4.029-9-9-9zm0 16.875a7.875 7.875 0 0 1-7.875-7.875 7.875 7.875 0 1 1 15.75 0 7.875 7.875 0 0 1-7.875 7.875z"></path><path d="M479.284 279.586c.089-.238-.024-.361-.13-.361-.489 0-1.123 1.16-1.359 1.16-.093 0-.173-.092-.173-.174 0-.238.581-.801.751-.971.526-.506 1.217-.895 1.979-.895.567 0 1.174.346.703 1.639l-.952 2.604c-.079.199-.224.531-.224.746 0 .092.054.186.159.186.395 0 1.121-1.135 1.304-1.135.067 0 .158.086.158.199 0 .385-1.542 2.043-2.874 2.043-.477 0-.804-.225-.804-.732 0-.641.447-1.734.538-1.963l.924-2.346zm.727-3.41c0-.586.498-1.066 1.082-1.066.527 0 .91.357.91.906 0 .615-.501 1.068-1.098 1.068-.541-.002-.894-.361-.894-.908z"></path></symbol><symbol id="icon-opr" viewBox="0 0 18 18"><path d="M.9 2.5c-.2-.2-.4-.3-.6-.2-.3.2-.3.4-.3.6 0 .8 0 1.7.1 2.5 0 .5.3.7.7.6.9 0 1.7-.2 2.5-.3h.2c.5-.2.6-.5.3-.8-.2-.2-.4-.4-.6-.5-.1-.1-.2-.2-.4-.3.5-.6 1-1.1 1.7-1.5 4.6-3 10.9-.5 12 4.9.9 4.4-2.2 8.6-6.7 9.2-3.7.5-7.2-1.6-8.4-5.1 0-.1-.1-.2-.1-.3-.1-.3-.5-.5-.8-.4-.3.1-.5.4-.4.8.4 1.3 1.1 2.4 2 3.4 1.4 1.5 3.1 2.4 5.1 2.8 3 .5 5.6-.2 7.8-2.2 2.1-1.9 3-4.2 3-6.9-.1-4-3.1-7.6-7.1-8.4-3.3-.9-6.2 0-8.6 2.3-.2.1-.3.3-.5.5-.3-.2-.6-.5-.9-.7z"></path><path d="M13 4.7c-2.6 1.6-4.5 3.6-5.3 4.6L5.6 7.7l-.9.7L8.3 12c.6-1.6 2.6-4.6 4.9-6.8l-.2-.5z"></path></symbol><symbol id="icon-quotation" viewBox="0 0.333 55.932 35.619"><path d="M43.738.333L31.545 12.207v23.745h24.387V12.207H43.738zm-31.545 0L0 12.207v23.745h24.386V12.207H12.193z"></path></symbol><symbol id="icon-remove" viewBox="-296 388 18 18"><path d="M-291.7 396.1h9v2h-9z"></path><path d="M-287 405.5c-4.7 0-8.5-3.8-8.5-8.5s3.8-8.5 8.5-8.5 8.5 3.8 8.5 8.5-3.8 8.5-8.5 8.5zm0-16c-4.1 0-7.5 3.4-7.5 7.5s3.4 7.5 7.5 7.5 7.5-3.4 7.5-7.5-3.4-7.5-7.5-7.5z"></path></symbol><symbol id="icon-ribbon" viewBox="0 0 36 72"><path d="M0 0v72h.053L18 54.054 35.945 72H36V0H0zm25.8 11.4l3.001 3L15 28.2l-7.801-7.8 3-3L15 22.201 25.8 11.4z"></path></symbol><symbol id="icon-rss" viewBox="0 0 24 24"><g fill="#F26822"><ellipse cx="3.305" cy="20.702" rx="3.306" ry="3.298"></ellipse><path d="M15.978 24h-4.684c0-6.224-5.057-11.27-11.294-11.27V8.058c8.824 0 15.978 7.137 15.978 15.942z"></path><path d="M19.2 23.95C19.2 13.366 10.604 4.79 0 4.79V0c13.255 0 24 10.723 24 23.95h-4.8z"></path></g></symbol><symbol id="icon-search" viewBox="0 0 16 16"><path d="M6.276 12.257C2.815 12.257 0 9.508 0 6.128 0 2.749 2.815 0 6.276 0c3.462 0 6.277 2.749 6.277 6.128.002 3.38-2.815 6.129-6.277 6.129zm0-10.772c-2.622 0-4.755 2.083-4.755 4.643 0 2.562 2.133 4.644 4.755 4.644 2.623 0 4.756-2.082 4.756-4.644 0-2.56-2.134-4.643-4.756-4.643z"></path><path d="M16 13.751L13.641 16l-4.875-4.82 2.36-2.248L16 13.751z"></path></symbol><symbol id="icon-speech-bubble" viewBox="0 0 40 54"><path d="M0 0v40h6.667v13.334L20 40h20V0z"></path></symbol><symbol id="icon-error" viewBox="2.002 0 14 14"><path d="M16.002 11.949L13.951 14 9.002 9.051 4.053 14l-2.051-2.051L6.951 7 2.002 2.05 4.053 0l4.949 4.95L13.951 0l2.051 2.049L11.053 7l4.949 4.949z"></path></symbol><symbol id="icon-info" viewBox="5.502 0 7 14"><path d="M8.165 5.923c.146-.384-.04-.577-.209-.577-.783 0-1.798 1.856-2.177 1.856-.15 0-.277-.147-.277-.279 0-.383.933-1.282 1.203-1.557.846-.811 1.949-1.434 3.174-1.434.908 0 1.879.555 1.125 2.63l-1.526 4.167c-.126.32-.358.854-.358 1.198 0 .148.088.298.255.298.632 0 1.796-1.818 2.091-1.818.105 0 .252.134.252.321C11.717 11.345 9.245 14 7.11 14c-.763 0-1.289-.361-1.289-1.176 0-1.027.717-2.778.865-3.145l1.479-3.756zm1.144-4.211C9.309.771 10.111 0 11.045 0c.845 0 1.457.577 1.457 1.456 0 .983-.804 1.709-1.757 1.709-.866-.001-1.436-.578-1.436-1.453z"></path></symbol><symbol id="icon-success" viewBox="0 0 16 14"><path d="M15.592 0c-4.867 2.984-8.4 6.751-9.987 8.641L1.717 5.595 0 6.979l6.717 6.832c1.157-2.961 4.817-8.748 9.287-12.86L15.592 0z"></path></symbol><symbol id="icon-warning" viewBox="7.002 0 2.789 14"><path d="M7.002 11.211h2.789V14H7.002v-2.789zM7.002 0v3.436l.741 5.326h1.326l.723-5.326V0h-2.79z"></path></symbol><symbol id="icon-facebook-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path><path d="M483.025 280.48l.32-2.477h-2.453v-1.582c0-.715.199-1.207 1.227-1.207h1.311v-2.213a17.753 17.753 0 0 0-1.907-.098c-1.894 0-3.186 1.154-3.186 3.271V278h-2.142v2.477h2.142v6.354h2.557v-6.354l2.131.003z"></path></symbol><symbol id="icon-facebook" viewBox="0 0 24 24"><circle fill="#4D67A4" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M8.396 10.143h2.137V8.165s-.092-1.292.892-2.274c.979-.979 2.298-.887 4.177-.724v2.21h-1.387s-.586-.013-.861.303c-.271.315-.24.764-.24.875v1.588h2.41l-.311 2.458h-2.116v6.336h-2.56v-6.345H8.396v-2.449z"></path></symbol><symbol id="icon-gplus-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M480.5 273.117c.123.076.26.185.416.334.146.152.289.343.432.562.135.211.258.459.359.746.086.286.129.619.129 1.004-.012.7-.164 1.262-.465 1.683a5.374 5.374 0 0 1-.461.562c-.186.176-.375.35-.58.527a2.65 2.65 0 0 0-.322.41.908.908 0 0 0-.176.553c0 .205.059.375.18.508.102.129.199.238.299.336l.662.545a6.48 6.48 0 0 1 1.086 1.119c.293.415.445.955.459 1.625 0 .949-.42 1.789-1.256 2.521-.869.76-2.123 1.15-3.762 1.177-1.373-.015-2.4-.308-3.074-.879-.682-.534-1.023-1.172-1.023-1.917 0-.359.111-.766.336-1.211.215-.443.602-.837 1.172-1.174a6.126 6.126 0 0 1 1.998-.725 15.415 15.415 0 0 1 1.723-.173 3.996 3.996 0 0 1-.377-.596 1.347 1.347 0 0 1-.193-.74c0-.174.023-.317.074-.433a8.83 8.83 0 0 1 .121-.338 5.973 5.973 0 0 1-.633.035c-1.043-.012-1.838-.339-2.381-.979-.574-.603-.861-1.3-.861-2.097 0-.961.404-1.832 1.217-2.616.559-.46 1.135-.757 1.736-.898a8.494 8.494 0 0 1 1.67-.182h3.916l-1.209.704h-1.207v.005l-.005.002zm.754 11.276c0-.498-.162-.932-.484-1.302-.346-.348-.881-.778-1.609-1.29a4.045 4.045 0 0 0-.439-.02c-.102-.016-.354 0-.768.039a8.141 8.141 0 0 0-1.244.275c-.1.036-.24.096-.42.17a2.713 2.713 0 0 0-.553.355 1.985 1.985 0 0 0-.459.592c-.145.25-.215.551-.215.901 0 .687.311 1.253.932 1.698.596.444 1.404.68 2.432.688.928-.014 1.629-.217 2.117-.612.476-.385.71-.886.71-1.494zm-2.729-5.743a1.843 1.843 0 0 0 1.289-.556c.164-.249.271-.506.318-.767a7.32 7.32 0 0 0 .041-.659c0-.772-.195-1.552-.594-2.336a2.611 2.611 0 0 0-.73-.921 1.894 1.894 0 0 0-1.064-.352 1.723 1.723 0 0 0-1.324.64c-.299.437-.438.92-.428 1.457 0 .709.209 1.444.623 2.211.201.356.459.659.775.912.323.248.686.371 1.094.371zm9.698.077h-2.055v-2.056h-.996v2.056h-2.053v.998h2.053v2.052h.996v-2.052h2.055z"></path></symbol><symbol id="icon-gplus" viewBox="0 0 24 24"><circle fill="#D84A36" cx="12" cy="12" r="12"></circle><g fill="#FFF"><path d="M12.012 12.111c-.304-.33-.07-.842.155-1.078.226-.241.639-.56.829-.74.193-.184.66-.605.674-1.868 0 0 .112-1.139-1.127-2.06h.977l1.038-.593h-3.507s-2.083-.067-3.085 1.207c-1 1.276-.966 2.871.114 3.77 1.08.898 2.371.648 2.371.648s-.197.422-.163.775c.036.353.421.849.504.965 0 0-2.386-.011-3.551 1.033-1.165 1.053-1.29 3.097 1.021 3.803 2.312.707 5.303-.055 5.906-2.277 0 0 .262-.908-.189-1.683-.459-.775-1.663-1.571-1.967-1.902zm-2.555-1.83c-.744-1-1.308-2.953 0-3.783 0 0 .955-.579 1.801.266.846.846 1.082 2.277.928 3.003-.15.727-.563.897-.739 1.008-.185.111-1.249.504-1.99-.494zm3.668 5.481c-.035.393-.083 1.254-1.491 1.664-1.409.408-3.826-.264-3.757-1.914 0 0 .006-.698.449-1.089.445-.396 1.094-.794 3.029-.819 0 0 .643.473 1.072.809.427.332.733.952.698 1.349z"></path><path d="M16.323 11.055V9.803h-.752v1.252h-1.255v.748h1.255v1.256h.752v-1.256h1.248v-.748z"></path></g></symbol><symbol id="icon-linkedin" viewBox="0 0 24 24"><circle fill="#0178B5" cx="12" cy="12" r="12"></circle><g fill="#FFF"><circle cx="8.022" cy="8.043" r="1.256"></circle><path d="M6.929 10.246h2.17v6.967h-2.17zm3.533 6.967h2.157v-3.594s-.078-1.627 1.219-1.627c1.301 0 1.211 1.231 1.211 1.635v3.586h2.183v-3.775s.129-2.745-1.472-3.191c-1.605-.445-2.661.071-3.219.985v-.985h-2.079v6.966z"></path></g></symbol><symbol id="icon-reddit" viewBox="0 0 24 24"><circle fill="#BCBCBC" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M4.661 9.741c.941 0 1.703.761 1.703 1.704 0 .938-.762 1.705-1.703 1.705s-1.704-.767-1.704-1.705c0-.943.763-1.704 1.704-1.704zm13.844 0a1.704 1.704 0 1 1 .001 3.409 1.704 1.704 0 0 1-.001-3.409z"></path><path fill="#FFF" d="M11.736 8.732c4.285 0 7.762 2.283 7.762 5.104 0 2.812-3.477 5.1-7.762 5.1-4.288 0-7.762-2.285-7.762-5.1.001-2.82 3.474-5.104 7.762-5.104z"></path><path fill="#010101" d="M11.736 19.262c-4.461 0-8.088-2.437-8.088-5.426 0-2.994 3.626-5.431 8.088-5.431 4.457 0 8.087 2.437 8.087 5.431 0 2.989-3.63 5.426-8.087 5.426zm0-10.205c-4.104 0-7.438 2.145-7.438 4.779 0 2.633 3.334 4.773 7.438 4.773 4.098 0 7.437-2.142 7.437-4.773 0-2.636-3.339-4.779-7.437-4.779z"></path><circle fill="#FFF" cx="18.014" cy="5.765" r="1.385"></circle><path fill="#010101" d="M18.014 7.456a1.694 1.694 0 0 1-1.695-1.691c0-.934.764-1.694 1.695-1.694s1.693.762 1.693 1.694c0 .934-.762 1.691-1.693 1.691zm0-2.734c-.574 0-1.043.469-1.043 1.043s.469 1.041 1.043 1.041a1.042 1.042 0 0 0 0-2.084zM3.818 13.275a2.068 2.068 0 0 1-1.068-1.811 2.074 2.074 0 0 1 3.538-1.466l-.458.462a1.412 1.412 0 0 0-1.007-.42 1.424 1.424 0 0 0-.688 2.669l-.317.566zm15.866 0l-.316-.566a1.43 1.43 0 0 0 .73-1.245c0-.784-.635-1.424-1.42-1.424-.381 0-.736.149-1.008.42l-.459-.462a2.074 2.074 0 1 1 2.473 3.277z"></path><path fill="#F04B23" d="M9.165 11.604c.704 0 1.272.564 1.272 1.27a1.273 1.273 0 1 1-1.272-1.27zm5.321 0a1.274 1.274 0 1 1-1.275 1.27c0-.704.57-1.27 1.275-1.27z"></path><path fill="#010101" d="M11.79 8.915a.327.327 0 0 1-.307-.44l1.472-4.181 3.616.862c.177.044.283.22.238.395a.32.32 0 0 1-.391.239l-3.045-.729-1.275 3.632a.334.334 0 0 1-.308.222zm-.069 8.376c-2.076 0-2.908-.939-2.943-.982a.332.332 0 0 1 .035-.461.334.334 0 0 1 .459.033c.019.021.713.758 2.449.758 1.766 0 2.539-.763 2.549-.767a.324.324 0 1 1 .469.446c-.04.043-.954.973-3.018.973z"></path></symbol><symbol id="icon-twitter-bordered" viewBox="463.812 263.868 32 32"><path d="M486.416 276.191a5.622 5.622 0 0 1-1.554.429 2.718 2.718 0 0 0 1.19-1.502 5.456 5.456 0 0 1-1.72.657 2.71 2.71 0 0 0-1.979-.854 2.711 2.711 0 0 0-2.642 3.326 7.681 7.681 0 0 1-5.586-2.831 2.714 2.714 0 0 0 .839 3.618 2.748 2.748 0 0 1-1.227-.339v.031a2.71 2.71 0 0 0 2.174 2.656 2.735 2.735 0 0 1-1.229.049 2.726 2.726 0 0 0 2.531 1.883 5.442 5.442 0 0 1-4.01 1.123 7.672 7.672 0 0 0 4.155 1.215c4.983 0 7.71-4.129 7.71-7.711 0-.115-.004-.232-.006-.351a5.41 5.41 0 0 0 1.354-1.399z"></path><path d="M479.812 263.868c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14 14 6.269 14 14-6.267 14-14 14z"></path></symbol><symbol id="icon-twitter" viewBox="0 0 24 24"><circle fill="#26A7DF" cx="12" cy="12" r="12"></circle><path fill="#FFF" d="M5.903 6.768s2.176 2.883 5.953 3.082c0 0-.51-1.702 1.058-3.004 1.568-1.305 3.508-.2 3.879.292 0 0 1.104-.172 1.875-.691 0 0-.252.995-1.18 1.594 0 0 1.086-.146 1.596-.439 0 0-.533.853-1.447 1.503 0 0 .381 3.595-2.709 6.57-3.09 2.971-8.176 2.352-10.012.797 0 0 2.495.277 4.249-1.219 0 0-2.074-.037-2.671-1.973 0 0 1.036.08 1.215-.119 0 0-2.232-.539-2.254-2.873 0 0 .68.339 1.297.359 0-.001-2.153-1.557-.849-3.879z"></path></symbol><symbol id="icon-weibo-bordered" viewBox="463.812 263.868 32 32"><path d="M479.812 263.868c-8.838 0-16 7.163-16 16s7.162 16 16 16c8.837 0 16-7.163 16-16s-7.163-16-16-16zm0 30c-7.732 0-14-6.269-14-14s6.268-14 14-14c7.731 0 14 6.269 14 14s-6.267 14-14 14z"></path><path d="M478.552 285.348c-2.616.261-4.876-.926-5.044-2.649-.167-1.722 1.814-3.33 4.433-3.588 2.609-.263 4.871.926 5.041 2.647.165 1.721-1.818 3.331-4.43 3.59m5.23-5.718c-.226-.065-.374-.109-.259-.403.25-.639.276-1.188.005-1.581-.515-.734-1.915-.693-3.521-.021 0 0-.508.224-.378-.181.247-.798.209-1.468-.178-1.852-.87-.878-3.194.032-5.183 2.027-1.489 1.494-2.357 3.082-2.357 4.453 0 2.619 3.354 4.213 6.631 4.213 4.297 0 7.154-2.504 7.154-4.493.001-1.198-1.007-1.881-1.914-2.162"></path><path fill="#020202" d="M486.637 274.833a4.176 4.176 0 0 0-3.982-1.291.608.608 0 0 0-.465.72.604.604 0 0 0 .72.466 2.968 2.968 0 0 1 2.827.92 3 3 0 0 1 .625 2.918.602.602 0 0 0 .39.762.603.603 0 0 0 .763-.391v-.001a4.218 4.218 0 0 0-.878-4.103"></path><path fill="#020202" d="M485.041 276.276a2.037 2.037 0 0 0-1.938-.63.518.518 0 0 0-.396.621.517.517 0 0 0 .617.398c.336-.071.702.03.947.307s.312.649.207.979a.52.52 0 0 0 .336.654.523.523 0 0 0 .657-.336 2.038 2.038 0 0 0-.43-1.993m-6.347 5.951c-.09.156-.293.233-.451.166-.151-.062-.204-.235-.115-.389.093-.155.284-.229.44-.168.157.056.214.235.126.391m-.832 1.074c-.253.405-.795.58-1.202.396-.403-.186-.521-.655-.27-1.051.248-.39.771-.566 1.176-.393.413.17.543.636.296 1.048m.95-2.864c-1.244-.326-2.65.294-3.19 1.396-.553 1.119-.021 2.369 1.236 2.775 1.303.42 2.84-.225 3.374-1.436.526-1.183-.132-2.402-1.42-2.735"></path></symbol><symbol id="icon-weibo" viewBox="4 4 24 24"><circle fill="#C8E8F9" cx="16" cy="16" r="12"></circle><path fill="#FFF" d="M9.098 18.194c0 1.981 2.574 3.593 5.757 3.593 3.178 0 5.756-1.611 5.756-3.593 0-1.989-2.578-3.601-5.756-3.601-3.183.001-5.757 1.612-5.757 3.601"></path><path fill="#DF0A21" d="M14.991 21.496c-2.817.278-5.244-.996-5.428-2.85-.183-1.855 1.95-3.586 4.767-3.862 2.813-.278 5.243.996 5.428 2.849.18 1.855-1.957 3.584-4.767 3.863m5.628-6.155c-.24-.067-.404-.116-.279-.432.27-.688.299-1.278.004-1.699-.551-.791-2.062-.748-3.789-.022 0-.001-.548.241-.406-.191.266-.859.224-1.577-.191-1.993-.937-.942-3.437.037-5.577 2.18-1.602 1.61-2.533 3.317-2.533 4.789 0 2.817 3.606 4.533 7.13 4.533 4.626 0 7.701-2.696 7.701-4.834.001-1.292-1.085-2.027-2.06-2.331"></path><path fill="#F4992C" d="M23.689 10.182a4.492 4.492 0 0 0-4.283-1.39.651.651 0 1 0 .269 1.275 3.2 3.2 0 0 1 3.045.99 3.224 3.224 0 0 1 .672 3.138.647.647 0 0 0 .418.823.647.647 0 0 0 .816-.421v-.003a4.51 4.51 0 0 0-.937-4.412"></path><path fill="#F4992C" d="M21.973 11.736a2.183 2.183 0 0 0-2.086-.679.56.56 0 0 0-.428.667.554.554 0 0 0 .662.43v.001c.361-.075.754.034 1.018.33.266.299.34.698.227 1.054a.563.563 0 0 0 .359.706.563.563 0 0 0 .709-.362 2.2 2.2 0 0 0-.461-2.147"></path><path fill="#13110C" d="M15.143 18.139c-.1.17-.315.251-.483.179-.168-.07-.222-.255-.128-.42.097-.167.311-.248.479-.182.165.062.228.252.132.423m-.896 1.155c-.271.434-.854.625-1.295.425-.433-.195-.561-.707-.285-1.128.269-.426.832-.609 1.265-.427.443.183.58.688.315 1.13m1.024-3.084c-1.34-.351-2.852.32-3.434 1.506-.597 1.206-.021 2.546 1.33 2.984 1.402.453 3.055-.243 3.629-1.543.569-1.275-.142-2.587-1.525-2.947"></path></symbol></svg>
</div>

        <a class="u-visually-hidden u-visually-hidden-focus" href="#main-content">
    <span class="c-banner c-banner--info">Skip to content</span>
</a>

        
            
    <div class="adsbox c-ad c-ad--LB1">
        <div class="c-ad__inner">
            <p class="c-ad__label">Advertisement</p>
            <div id="div-gpt-ad-LB1" data-gpt-unitpath="/270604982/bmc/bmcbioinformatics/articles" data-gpt-sizes="728x90,970x90" data-gpt-targeting="pos=LB1;doi=10.1186/1471-2105-8-268;kwrd=Root Mean Square Error,Posterior Distribution,Approximate Bayesian Computation,Population Pair,Simultaneous Divergence;pmc=L15001,B12050,I23050,L17004,M14018,M14018;"><div id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0__container__" style="border: 0pt none;"><iframe id="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0" title="3rd party ad content" name="google_ads_iframe_/270604982/bmc/bmcbioinformatics/articles_0" scrolling="no" marginwidth="0" marginheight="0" style="border: 0px none; vertical-align: bottom;" srcdoc="" width="728" frameborder="0" height="90"></iframe></div></div>
        </div>
    </div>

        
         
    <div id="membership-message-loader-desktop" class="placeholder" data-placeholder="/placeholder/v1/membership/message"></div>


<header class="c-popup-search ctx-publisher-header" id="top" data-test="publisher-header">
    <nav class="c-navbar">
        <div class="u-container">
            
            <div class="basic-white-logo c-navbar__logo u-margin-right-xxl" itemscope="" itemtype="http://schema.org/Organization" data-test="navbar-logo-header">
                <link itemprop="parentOrganization" href="#parentOrganization">
                <div class="c-logo">
    <a href="https://www.biomedcentral.com/" itemprop="url">
        <img src="logo-bmc-white-series-1da6525f4e.svg" alt="BMC" itemprop="logo">
        <div class="c-logo__strapline">
            <img src="logo-bmc-white-strapline-sn-e7b1d6e67f.svg" alt="Part of Springer Nature">
        </div>
    </a>
</div>
            </div>
            <button type="button" class="c-navbar__link c-dropdown js-publisher-menu-button u-hidden-at-lg" id="publisher-header-menu-button" data-toggle="collapse" data-test="publisher-header-menu-toggle" data-toggle-linked="publisher-header" data-target="publisher-header-search publisher-header-menu" aria-controls="publisher-header-menu publisher-header-search" aria-expanded="false">
                <span class="c-dropdown__toggle">Menu</span>
            </button>
            <div class="c-navbar__content c-collapse" id="publisher-header-menu">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


                <ul class="c-navbar__nav">
                    <li class="c-navbar__item">
                        <a data-replace="true" data-placeholder="/placeholder/v1/username" class="c-navbar__link" href="https://www.biomedcentral.com/login" data-test="login-link" data-track="click" data-track-category="Account" data-track-action="Click Login Link">
                            <span class="u-js-shown">Login</span>
                            <span class="u-js-hidden">My Account</span>
                        </a>
                    </li>
                    <li class="c-navbar__item">
                        <button type="button" class="c-navbar__link js-publisher-search-button u-hidden-at-lt-lg" data-toggle="collapse" data-test="header-search-button" data-target="publisher-header-search publisher-header-menu publisher-header-menu-button" aria-controls="publisher-header-search publisher-header-menu" aria-expanded="false">
                            <span class="c-flex c-flex--align-center">
                                Search
                                <svg class="c-icon u-margin-left-xs" width="14" height="14" aria-hidden="true">
                                    <use xlink:href="#icon-search"></use>
                                </svg>
                            </span>
                        </button>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="c-popup-search__content c-collapse js-publisher-search-bar" id="publisher-header-search">
        <div class="u-container">
            <div class="c-popup-search__container">
                <div class="ctx-search">
    <form role="search" class="c-form-field c-form-field--inline" method="GET" action="//www.biomedcentral.com/search" data-track="submit" data-track-category="Search and Results" data-track-action="Submit search" data-dynamic-track-label="" data-track-label="" data-test="global-search" novalidate="">
        <label for="publisherSearch" class="u-visually-hidden">Search all BMC articles</label>
        <input id="publisherSearch" class="c-form-field__input js-publisher-search-input" autocomplete="off" role="textbox" data-test="search-input" name="query" placeholder="Search all BMC articles" type="text">
        <button class="c-button c-button--secondary" type="submit" data-test="search-submit-button">
    <span class="u-visually-hidden">Search</span>
    <svg class="c-icon" width="16" height="16" aria-hidden="true">
        <use xlink:href="#icon-search"></use>
    </svg>
</button>


        <input name="searchType" value="publisherSearch" type="hidden">
    </form>
</div>

            </div>
        </div>
    </div>
</header>


        
            <header class="c-journal-header c-journal-header--bmc-bioinformatics ctx-journal-header">
                <div class="u-container">
                    <div class="c-journal-header__inner ">
                        
                        <div class="c-journal-title" id="journalTitle">
                            <a href="https://bmcbioinformatics.biomedcentral.com/">
    
        
    
    <span class="c-journal-title__text ">BMC Bioinformatics</span>
    
</a>
                        </div>
                        
                    </div>
                </div>
                <div class="c-navbar">
                    <div class="u-container">
                        
                            <button type="button" class="c-navbar__link c-dropdown u-hidden-at-lg" data-toggle="collapse" data-test="journal-header-menu" data-toggle-linked="journal-header" data-target="journal-menu" aria-controls="journal-menu" aria-expanded="false">
                                    <span class="c-dropdown__toggle">Menu</span>
                            </button>
                            <div class="c-navbar__content c-collapse" id="journal-menu">
                                <nav class="c-navbar__nav">
                                    <ul class="c-navbar__nav c-navbar__nav--journal" role="menu" data-test="site-navigation">
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/">Home</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/about">About</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link c-navbar__link--is-shown" href="https://bmcbioinformatics.biomedcentral.com/articles">Articles</a>
                                            </li>
                                        
                                            <li class="c-navbar__item" role="menuitem">
                                                <a class="c-navbar__link" href="https://bmcbioinformatics.biomedcentral.com/submission-guidelines">Submission Guidelines</a>
                                            </li>
                                        
                                    </ul>
                                </nav>
                            </div>
                        
                    </div>
                </div>
                <div class="c-journal-header__identity c-journal-header__identity--default">
                    
                </div>
            </header>
            
        

        <div class="u-container u-margin-top-xl u-margin-bottom-xl" id="main-content">
            <div class="c-content-layout c-content-layout--fulltext" data-component="sticky-container">
                <div class="c-content-layout__nav u-hidden-at-lt-xl">
                    <nav xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="ArticleNavigation" aria-labelledby="toc-menu-title" data-component="sticky"><h3 id="toc-menu-title" class="u-visually-hidden" data-test="toc-title">Table of Contents</h3><ol class="c-nav c-nav--stacked" role="menu" id="tocMenu" aria-expanded="true" data-article-toc=""><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Abs1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Abstract" data-test="abstract-link">Abstract</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Background">Background</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec2" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Implementation">Implementation</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec3" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Results and Discussion">Results and Discussion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec6" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Conclusion">Conclusion</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Sec7" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Availability and requirements">Availability and requirements</a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Decs" data-test="declarations-link" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="Declarations">
                  Declarations
                </a></li><li role="menuitem" class="c-nav__item"><a class="c-nav__link" href="#Bib1" data-track-category="Table of Contents" data-track="click" data-track-action="Link Clicked" data-track-label="References">References</a></li></ol></nav>
                </div>

                <main class="c-content-layout__main">
                    <article itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                        
                        
    <div class="FulltextWrapper">
        <ul class="c-article-identifiers u-reset-list">
            <li class="c-article-identifiers__item" data-test="article-category">Software</li>
            <li class="c-article-identifiers__item" data-test="open-access">
                <span class="u-text-open-access">Open Access</span>
            </li>
            
            
        </ul>
    </div>

                        <div class="FulltextWrapper"><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="MainTitleSection"><h1 xmlns="http://www.w3.org/1999/xhtml" class="ArticleTitle" itemprop="headline" lang="en">msBayes: Pipeline for testing comparative phylogeographic histories using hierarchical approximate Bayesian computation</h1></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AuthorNames" data-author-affiliations="true"><ul class="u-listReset"><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Michael&nbsp;J&nbsp;Hickerson</span><sup><a href="#Aff1" class="AffiliationID">1</a></sup><a href="mailto:michael.hickerson@qc.cuny.edu" class="EmailAuthor" title="Email author"><span class="ContactIcon" aria-hidden="true"></span><span class="u-srOnly">Email author</span></a>, </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Eli&nbsp;Stahl</span><sup><a href="#Aff2" class="AffiliationID">2</a></sup> and </li><li class="Author hasAffil" data-jumpto="Aff" data-test="author-name"><span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AuthorName" itemprop="author" itemscope="" itemtype="http://schema.org/Person">Naoki&nbsp;Takebayashi</span><sup><a href="#Aff3" class="AffiliationID">3</a></sup></li></ul></div><div xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="ArticleContextInformation"><div class="ContextInformation"><div class="ContextInformationJournalTitles"><span class="ArticleCitation"><span itemscope="" itemtype="http://schema.org/Periodical" itemid="#periodical"><span xmlns="http://www.w3.org/1999/xhtml" class="JournalTitle">BMC Bioinformatics</span></span><span class="ArticleCitation_Year">2007</span><span class="ArticleCitation_Volume"><strong itemprop="isPartOf" itemtype="http://schema.org/PublicationVolume"><link itemprop="isPartOf" href="#periodical">8</strong>:268</span></span></div><p class="ArticleDOI"><a itemprop="sameAs" class="u-text-inherit" href="https://doi.org/10.1186/1471-2105-8-268">https://doi.org/10.1186/1471-2105-8-268</a></p><p class="Copyright">© &nbsp;Hickerson et al; licensee BioMed Central Ltd.&nbsp;2007</p></div></div><ul class="ArticleHistory u-reset-list"><li class="History HistoryReceived"><strong>Received: </strong><span>06&nbsp;April&nbsp;2007</span></li><li class="History HistoryAccepted"><strong>Accepted: </strong><span>26&nbsp;July&nbsp;2007</span></li><li class="History HistoryOnlineDate"><strong>Published: </strong><span itemprop="datePublished">26&nbsp;July&nbsp;2007</span></li></ul></div>
                        
                        
                        
    

                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Abstract Section1 RenderAsSection1 c-section" id="Abs1" lang="en"><h2 class="c-section__heading"><a href="#Abstract" data-test="abstract-heading" data-disable-click-event-default="">Abstract</a></h2><div class="c-section__content" id="Abstract"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec1">
                <h3 xmlns="" class="Heading">Background</h3>
                <p class="Para">Although testing for simultaneous 
divergence (vicariance) across different population-pairs that span the 
same barrier to gene flow is of central importance to evolutionary 
biology, researchers often equate the gene tree and population/species 
tree thereby ignoring stochastic coalescent variance in their 
conclusions of temporal incongruence. In contrast to other available 
phylogeographic software packages, msBayes is the only one that analyses
 data from multiple species/population pairs under a hierarchical model.</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec2">
                <h3 xmlns="" class="Heading">Results</h3>
                <p class="Para">msBayes employs approximate Bayesian 
computation (ABC) under a hierarchical coalescent model to test for 
simultaneous divergence (TSD) in multiple co-distributed 
population-pairs. Simultaneous isolation is tested by estimating three 
hyper-parameters that characterize the degree of variability in 
divergence times across co-distributed population pairs while allowing 
for variation in various within population-pair demographic parameters 
(sub-parameters) that can affect the coalescent. msBayes is a software 
package consisting of several C and R programs that are run with a Perl 
"front-end".</p>
              </div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="AbstractSection" id="ASec3">
                <h3 xmlns="" class="Heading">Conclusion</h3>
                <p class="Para">The method reasonably distinguishes 
simultaneous isolation from temporal incongruence in the divergence of 
co-distributed population pairs, even with sparse sampling of 
individuals. Because the estimate step is decoupled from the simulation 
step, one can rapidly evaluate different ABC acceptance/rejection 
conditions and the choice of summary statistics. Given the complex and 
idiosyncratic nature of testing multi-species biogeographic hypotheses, 
we envision msBayes as a powerful and flexible tool for tackling a wide 
array of difficult research questions that use population genetic data 
from multiple co-distributed species. The msBayes pipeline is available 
for download at <span class="ExternalRef"><a href="http://msbayes.sourceforge.net/"><span class="RefSource">http://msbayes.sourceforge.net/</span></a></span>
 under an open source license (GNU Public License). The msBayes pipeline
 is comprised of several C and R programs that are run with a Perl 
"front-end" and runs on Linux, Mac OS-X, and most POSIX systems. 
Although the current implementation is for a single locus per 
species-pair, future implementations will allow analysis of multi-loci 
data per species pair.</p>
              </div></div></section><section xmlns:func="http://oscar.fig.bmc.com" class="KeywordGroup Section1 RenderAsSection1 c-section" lang="en"><h2 xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Keywords" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Keywords">Keywords</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Keywords"><ul class="c-keywords"><li class="c-keywords__item" itemprop="about">Root Mean Square Error</li><li class="c-keywords__item" itemprop="about">Posterior Distribution</li><li class="c-keywords__item" itemprop="about">Approximate Bayesian Computation</li><li class="c-keywords__item" itemprop="about">Population Pair</li><li class="c-keywords__item" itemprop="about">Simultaneous Divergence</li></ul></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec1" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Background" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Background">Background</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Background" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Testing
 for simultaneous divergence (vicariance) across different 
population-pairs that span the same historical barrier to gene flow is 
of central importance to evolutionary biology, biogeography and 
community ecology [<span class="CitationRef"><a href="#CR1" data-toggle="citation">1</a></span><span>, <span class="CitationRef"><a href="#CR2" data-toggle="citation">2</a></span>, <span class="CitationRef"><a href="#CR3" data-toggle="citation">3</a></span>, </span><span class="CitationRef"><a href="#CR4" data-toggle="citation">4</a></span>].
 Such inferences inform processes underlying speciation, community 
composition, range delineation, and the ecological consequences of 
climatic changes. Estimating a population divergence time with an 
appropriate statistical model [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>] can be accomplished in a variety of ways [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span><span>, <span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>, </span><span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>],
 yet analyzing comparative phylogeographic data with multiple 
co-occurring species pairs that vary with respect to demographic 
parameters and pairwise coalescent times is less straightforward.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Instead
 of conducting an independent analysis on every population-pair and 
testing the hypothesis of temporal concordance based on this set of 
independent parameter estimates of divergence time, the hierarchical 
model employed by msBayes follows the suggestion of [<span class="CitationRef"><a href="#CR9" data-toggle="citation">9</a></span>]
 by concurrently estimating three hyper-parameters that characterize the
 mean, variability and number of different divergence events across a 
set of population-pairs. The model employed in msBayes allows estimation
 of these hyper-parameters across a multi-species data set while 
explicitly incorporating uncertainty and variation in the sub-parameters
 that independently describe each population-pair's demographic history 
(divergence time, current, ancestral and founding effective population 
sizes), post-divergence migration rate and recombination rate. The 
msBayes software pipeline is based on the introduction of the 
approximate Bayesian computation (ABC) method for sampling from the 
hyper-posterior distribution for testing for simultaneous divergence [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>].
 We review the important features here. Although the current 
implementation is for a single locus per species-pair, future 
implementations will allow analysis of multi-loci data per 
species/population pair.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">In contrast to previous ABC-like models [<span class="CitationRef"><a href="#CR11" data-toggle="citation">11</a></span><span>, <span class="CitationRef"><a href="#CR12" data-toggle="citation">12</a></span>, <span class="CitationRef"><a href="#CR13" data-toggle="citation">13</a></span>, <span class="CitationRef"><a href="#CR14" data-toggle="citation">14</a></span>, </span><span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>],
 our TSD is accomplished by implementing a hierarchical Bayesian model 
in which the sub-parameters (Φ; within population-pair parameters) are 
conditional on "hyper-parameters" (<em xmlns="" class="EmphasisTypeItalic">ϕ</em>) that describe the variability of Φ among the <em xmlns="" class="EmphasisTypeItalic">Y</em>
 population-pairs. For example, divergence times (Φ) can vary across a 
set of population pairs conditional on the set of hyper-parameters (<em xmlns="" class="EmphasisTypeItalic">ϕ</em>) that varies according to their hyper-prior distribution. Instead of explicitly calculating the likelihood expression P(Data | <em xmlns="" class="EmphasisTypeItalic">ϕ</em>,Φ) to get a posterior distribution, we sample from the posterior distribution P((<em xmlns="" class="EmphasisTypeItalic">ϕ</em>,Φ) | Data) by simulating the data <em xmlns="" class="EmphasisTypeItalic">K</em> times under the coalescent model using candidate parameters drawn from the prior distribution P(<em xmlns="" class="EmphasisTypeItalic">ϕ</em>,Φ). A summary statistic vector <strong xmlns="" class="EmphasisTypeBold">D</strong>
 for each simulated dataset is then compared to the observed summary 
statistic vector in order to generate random observations from the joint
 posterior distribution <em xmlns="" class="EmphasisTypeItalic">f</em>(<em xmlns="" class="EmphasisTypeItalic">ϕ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">i</em>
                </sub>,Φ<sub>
                  <em xmlns="" class="EmphasisTypeItalic">i</em>
                </sub>|<strong xmlns="" class="EmphasisTypeBold">D</strong><sub>
                  <em xmlns="" class="EmphasisTypeItalic">i</em>
                </sub>) by way of a rejection/acceptance algorithm [<span class="CitationRef"><a href="#CR16" data-toggle="citation">16</a></span>] followed by an optional weighted local regression step [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>].
 Loosely speaking, hyper-parameter values are accepted and used to 
construct the posterior distribution with probabilities proportional to 
the similarity between the summary statistic vector from the observed 
data and the summary statistic vector calculated from simulated data.</p><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The hierarchical model consists of ancestral populations that split at divergence times <em xmlns="" class="EmphasisTypeItalic">T</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">Y</em>
                </sub>= {<em xmlns="" class="EmphasisTypeItalic">τ</em><sub>1</sub>...<em xmlns="" class="EmphasisTypeItalic">τ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">Y</em>
                </sub>} in the past. The hyper-parameter set, <em xmlns="" class="EmphasisTypeItalic">ϕ</em> quantifies the degree of variability in these <em xmlns="" class="EmphasisTypeItalic">Y</em> divergence times across the <em xmlns="" class="EmphasisTypeItalic">Y</em> ancestral populations and their <em xmlns="" class="EmphasisTypeItalic">Y</em> descendent population pairs: (1) Ψ, the number of possible divergence times (1 ≤ Ψ ≤ <em xmlns="" class="EmphasisTypeItalic">Y</em>); (2) E(<em xmlns="" class="EmphasisTypeItalic">τ</em>), the mean divergence time; and (3) Ω, the ratio of variance to the mean in these <em xmlns="" class="EmphasisTypeItalic">Y</em> divergence times, Var(<em xmlns="" class="EmphasisTypeItalic">τ</em>)/E(<em xmlns="" class="EmphasisTypeItalic">τ</em>). The sub-parameters for the <em xmlns="" class="EmphasisTypeItalic">i</em>-th population-pair (Φ<sub>
                  <em xmlns="" class="EmphasisTypeItalic">i</em>
                </sub>) are allowed to vary independently across <em xmlns="" class="EmphasisTypeItalic">Y</em> population pairs and include divergence time (<em xmlns="" class="EmphasisTypeItalic">τ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">i</em>
                </sub>), current population sizes, ancestral population 
sizes, post-divergence founding population sizes, durations of 
post-divergence population growth, recombination rates, and 
post-divergence migration rates. The multiple population-pair splitting 
model is depicted in Figure <span class="InternalRef"><a href="#Fig1">1</a></span>. Each divergence time parameter <em xmlns="" class="EmphasisTypeItalic">τ</em> is scaled by 2<em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">AVE</em>
                </sub>generations, where <em xmlns="" class="EmphasisTypeItalic">N</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">AVE</em>
                </sub>is the parametric expectation of <em xmlns="" class="EmphasisTypeItalic">N</em> (effective population size) across <em xmlns="" class="EmphasisTypeItalic">Y</em> population pairs given the prior distribution.<figure xmlns="" class="Figure" id="Fig1"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig1_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2007_Article_1640_Fig1_HTML.jpg" alt="Figure 1" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig1_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig1_HTML.jpg" alt="Figure 1"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 1</span>
                      <p xmlns="" class="SimplePara"><strong class="EmphasisTypeBold">Depiction of the multiple population-pair divergence model used for the ABC estimates of Ψ, E(</strong> <span class="EmphasisTypeBoldItalic">τ</span> <strong class="EmphasisTypeBold">), and Ω</strong>.
 (A): The white lines depict a gene tree with TMRCA being the time to 
the gene sample's most recent common ancestor, and the black tree 
containing the gene tree is the population/species tree. (B): Parameters
 in the multiple population-pair divergence model. The population 
mutation parameter, <em class="EmphasisTypeItalic">θ</em>, is 2<em class="EmphasisTypeItalic">Nμ</em> where 2<em class="EmphasisTypeItalic">N</em>
 is the summed haploid effective female population size of each pair of 
daughter populations (μ is the per gene per generation mutation rate). 
The time since isolation of each population pair is denoted by <em class="EmphasisTypeItalic">τ</em> (in units of 2<em class="EmphasisTypeItalic">N</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">AVE</em>
                        </sub>generations, where <em class="EmphasisTypeItalic">N</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">AVE</em>
                        </sub>is the parametric expectation of <em class="EmphasisTypeItalic">N</em> across <em class="EmphasisTypeItalic">Y</em> population pairs given the prior distribution). Population mutation parameters for daughter populations <em class="EmphasisTypeItalic">a</em> and <em class="EmphasisTypeItalic">b</em> are <em class="EmphasisTypeItalic">θ</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">a</em>
                        </sub>and <em class="EmphasisTypeItalic">θ</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">b</em>
                        </sub>, whereas <em class="EmphasisTypeItalic">θ</em> '<sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">a</em>
                        </sub>and <em class="EmphasisTypeItalic">θ'</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">b</em>
                        </sub>are the population mutation parameters for the sizes of daughter populations <em class="EmphasisTypeItalic">a</em> and <em class="EmphasisTypeItalic">b</em> at the time of divergence until <em class="EmphasisTypeItalic">τ'</em> (length of bottleneck). The daughter populations <em class="EmphasisTypeItalic">θ</em> '<sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">a</em>
                        </sub>and <em class="EmphasisTypeItalic">θ'</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">b</em>
                        </sub>then grow exponentially to sizes <em class="EmphasisTypeItalic">θ</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">a</em>
                        </sub>and <em class="EmphasisTypeItalic">θ</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">b</em>
                        </sub>. The population mutation parameter for each ancestral population is depicted as <em class="EmphasisTypeItalic">θ</em><sub xmlns="http://www.w3.org/1999/xhtml">
                          <em xmlns="" class="EmphasisTypeItalic">A</em>
                        </sub>. The migration rate between each pair of daughter populations is depicted as <em class="EmphasisTypeItalic">M</em>
 (number of effective migrants per generation). (C): Example of four 
population-pairs where parameters in (B) are drawn from uniform priors.</p>
                    </div></figcaption></figure></div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The summary statistic vector <strong xmlns="" class="EmphasisTypeBold">D</strong> employed in msBayes currently consists of up to six summary statistics collected from each of the <em xmlns="" class="EmphasisTypeItalic">Y</em> population pairs (<em xmlns="" class="EmphasisTypeItalic">π</em>,<em xmlns="" class="EmphasisTypeItalic">θ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">W</em>
                </sub>, Var(<em xmlns="" class="EmphasisTypeItalic">π</em> - <em xmlns="" class="EmphasisTypeItalic">θ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">W</em>
                </sub>), <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">net</em>
                </sub>, <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">b</em>
                </sub>, and <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">w</em>
                </sub>). This includes <em xmlns="" class="EmphasisTypeItalic">π</em>, the average number of pairwise differences among all sequences within each population pair, <em xmlns="" class="EmphasisTypeItalic">θ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">W</em>
                </sub>the number of segregating sites within each population pair normalized for sample size, [<span class="CitationRef"><a href="#CR17" data-toggle="citation">17</a></span>], Var(<em xmlns="" class="EmphasisTypeItalic">π</em> - <em xmlns="" class="EmphasisTypeItalic">θ</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">W</em>
                </sub>) in each population pair, and <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">net</em>
                </sub>, Nei and Li's net nucleotide divergence between each pair of populations [<span class="CitationRef"><a href="#CR18" data-toggle="citation">18</a></span>]. This last summary statistic is the difference (<em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">b</em>
                </sub>- <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">w</em>
                </sub>) where <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">b</em>
                </sub>is the average pairwise differences between each population pair and <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                  <em xmlns="" class="EmphasisTypeItalic">w</em>
                </sub>is the average pairwise differences within a 
sister pair of descendent populations. The default setting includes the 
first four aforementioned summary statistics because they were found to 
be a least correlated subset of a larger group [<span class="CitationRef"><a href="#CR19" data-toggle="citation">19</a></span>], however, future versions of msBayes will allow users to choose other summary statistics.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">An extensive simulation study was conducted in [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]
 to evaluate the performance of our hierarchical ABC model. Because 
comparative phylogeographic studies are often conducted on multi-species
 data sets that include rare taxa from which it is difficult to obtain 
samples from many individuals, we extend the previous evaluation to 
explore the effectiveness of msBayes in conducting a TSD given small 
sample sizes (≤ 5 individuals per population pair).</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec2" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Implementation" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Implementation">Implementation</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Implementation" data-test="collapse-section"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">After
 preparation of a sample size file and the input files from DNA sequence
 data, running msBayes is a three step process that includes: (<strong xmlns="" class="EmphasisTypeBold">A</strong>) calculating the observed summary statistic vector from the DNA sequence input files and the sample size file; (<strong xmlns="" class="EmphasisTypeBold">B</strong>) running coalescent simulations of the DNA sequence data using parameters drawn from the hyper-prior (<em xmlns="" class="EmphasisTypeItalic">ϕ</em>) and prior (Φ); and (<strong xmlns="" class="EmphasisTypeBold">C</strong>) sampling from the posterior distribution and obtaining posterior estimates of Ψ, E(<em xmlns="" class="EmphasisTypeItalic">τ</em>), and Ω across the <em xmlns="" class="EmphasisTypeItalic">Y</em> population pairs (Figure <span class="InternalRef"><a href="#Fig2">2</a></span>).<figure xmlns="" class="Figure" id="Fig2"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                    <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig2_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2007_Article_1640_Fig2_HTML.jpg" alt="Figure 2" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig2_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig2_HTML.jpg" alt="Figure 2"></img></noscript></a>
                  </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 2</span>
                      <p xmlns="" class="SimplePara">Flowchart describing operation of msBayes.</p>
                    </div></figcaption></figure></div><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><strong xmlns="" class="EmphasisTypeBold">Step A</strong>
 is accomplished by a command-line Perl program (obsSumStats.pl) which 
uses two C programs to calculate the observed summary statistic vector 
file. Specifically, the user runs obsSumStats.pl after collecting 
separate aligned DNA sequence files from each population-pair in FASTA 
format, and constructing an additional text file that describes the 
samples sizes, length of genes and transition/transversion rate ratios.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><strong xmlns="" class="EmphasisTypeBold">Step B</strong> iteratively simulates data sets under the hierarchical model by: <strong xmlns="" class="EmphasisTypeBold">1.)</strong> randomly drawing hyper-parameters and sub-parameters from the hyper-prior and sub-prior distributions; <strong xmlns="" class="EmphasisTypeBold">2.)</strong> using these hyper-parameters and sub-parameters to simulate finite sites DNA sequence data from <em xmlns="" class="EmphasisTypeItalic">Y</em> population-pairs; and <strong xmlns="" class="EmphasisTypeBold">3.)</strong> calculating a summary statistic vector <strong xmlns="" class="EmphasisTypeBold">D</strong> from the simulated data set of <em xmlns="" class="EmphasisTypeItalic">Y</em>
 population-pairs. This is accomplished with several C programs that are
 run with a Perl "front-end" (msbayes.pl) that either prompts the user 
for the upper-bounds of various priors and the number of iterations to 
simulate or alternatively uses a batch configuration file with 
equivalent information. The first C program draws hyper-parameters and 
sub-parameters from their hyper-prior and sub-prior distributions. These
 parameters are then fed into several C programs that simulate 
finite-sites DNA sequence data using msarbpopQH a modified version of 
Hudson's classic coalescent simulator ms [<span class="CitationRef"><a href="#CR20" data-toggle="citation">20</a></span>],
 which incorporates finite sites mutation and arbitrary population 
structure and dynamics. Another set of C programs calculates a summary 
statistic vector (<strong xmlns="" class="EmphasisTypeBold">D</strong>) for every simulated data set of <em xmlns="" class="EmphasisTypeItalic">Y</em> population pairs.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para"><strong xmlns="" class="EmphasisTypeBold">Step C</strong>
 is accomplished by our command-line user-interface program 
(acceptRej.pl). This Perl program internally uses R for the calculation.
 The algorithim is a simple extension of the original R scripts which 
are kindly provided by M. Beaumont [<span class="CitationRef"><a href="#CR15" data-toggle="citation">15</a></span>].
 This step does the rejection/acceptance sampling and local regression 
to produce the approximate sample of the posterior distribution. This 
third step uses the output of <strong xmlns="" class="EmphasisTypeBold">step B</strong>
 as the input and produces an output file that contains multiple 
graphical depictions of the posterior distributions and a text output 
file with various summaries of the posterior distributions (estimates of
 Ψ, E(<em xmlns="" class="EmphasisTypeItalic">τ</em>), and Ω across the <em xmlns="" class="EmphasisTypeItalic">Y</em> population pairs). The user can choose which summary statistics to include within <strong xmlns="" class="EmphasisTypeBold">D</strong>
 (the summary statistic vector), choose the proportion of accepted draws
 from the prior, and can optionally choose to perform simple rejection 
sampling without the additional local regression step.</p><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">We
 distribute msBayes as C source code and pre-compiled binaries that run 
on Linux or Mac OS X operating systems. The msBayes package also 
includes the R functions, and Perl scripts, as well as 
installation/running instructions.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec3" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#ResultsandDiscussion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="ResultsandDiscussion">Results and Discussion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="ResultsandDiscussion" data-test="collapse-section"><section id="Sec4" class="Section2 RenderAsSection2">
                <h3 class="Heading">Performance of estimator with small sample sizes</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">At
 the present time, there are no other available coalescent-based tools 
for analyzing multiple population pairs simultaneously to yield 
hyper-parameter estimates. Although IM and IMa are most similar to 
msBayes [<span class="CitationRef"><a href="#CR8" data-toggle="citation">8</a></span>, <span class="CitationRef"><a href="#CR21" data-toggle="citation">21</a></span>]
 because they estimate divergence times and population sizes from single
 pairs of populations under a coalescent model, these do not employ a 
hierarchical model and therefore can only do so one pair at a time. The 
program MCMCcoal can estimate divergence times of a known phylogeny 
under a coalescent model, but can only use the separate divergence time 
estimates to test for phylogeographic congruence [<span class="CitationRef"><a href="#CR7" data-toggle="citation">7</a></span>]. The program BEST [<span class="CitationRef"><a href="#CR6" data-toggle="citation">6</a></span>]
 infers a species phylogeny and demographic parameters (e.g. divergence 
times and population sizes) using a population coalescent model, but 
likewise can only use the individual divergence time estimates to test 
for phylogeographic congruence across a multi-species dataset. On the 
other hand, the hierarchical model employed in msBayes not only can 
estimate hyper-parameters but also comes with the benefit of additional 
information gained from the "borrowing strength" across datasets [<span class="CitationRef"><a href="#CR22" data-toggle="citation">22</a></span><span>, <span class="CitationRef"><a href="#CR23" data-toggle="citation">23</a></span>, </span><span class="CitationRef"><a href="#CR24" data-toggle="citation">24</a></span>].
 In this case, the resulting emergent multi-species hyper-estimates use 
more of the information than the sum of their parts (within species-pair
 estimates).</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Although the hierarchical ABC model employed in msBayes was extensively evaluated in [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>],
 the behavior of the ABC estimator given minimal sampling of individuals
 was not examined. Because comparative phylogeographic studies are often
 conducted on multi-species data sets that include rare taxa from which 
it is difficult to obtain samples from many individuals, we evaluate how
 low sample sizes can affect inference. To this end, we explored the 
performance in scenarios where ≤ 5 per population pair were sampled from
 each of 10 population pairs. We created 1,000 simulated data sets under
 each of two different histories: (1) simultaneous divergence history 
and (2) variable divergence history among population pairs. In the 
simultaneous divergence history (true Ω = Var(<em xmlns="" class="EmphasisTypeItalic">τ</em>)/E(<em xmlns="" class="EmphasisTypeItalic">τ</em>) = 0), all ten population pairs arose from ancestral populations at <em xmlns="" class="EmphasisTypeItalic">τ</em> = 1.8 before the present. In the variable divergence history (true Ω = 0.1), two population pairs arose at <em xmlns="" class="EmphasisTypeItalic">τ</em> = 1.0 and eight population pairs arose at <em xmlns="" class="EmphasisTypeItalic">τ</em>
 = 2.0 before the present. We simulated these two histories with small 
sample sizes (2–5 individual per population-pair) and with larger sample
 sizes (20 individuals per population pair; 10 per descendent 
population). The simulated data sets consisted of haploid mtDNA samples 
from ten population pairs that were 550–600 base pairs in length. From 
each of the four sets of 1,000 simulated data sets, we used msBayes to 
obtain 1,000 ABC estimates of the hyper-parameter, Ω, with the goal of 
assessing the effects of sample sizes on the root mean square error 
(RMSE) of the ABC Ω estimator. Each estimate of Ω was obtained from the 
mode of 1,000 accepted draws (after the local regression step) from 
500,000 random draws from the hyper-prior, as these conditions were 
found to be optimal in [<span class="CitationRef"><a href="#CR10" data-toggle="citation">10</a></span>]. For the larger sample sizes we use four classes of summary statistics (<em xmlns="" class="EmphasisTypeItalic">π</em>, <em xmlns="" class="EmphasisTypeItalic">θ</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">W</em>
                  </sub>, Var(<em xmlns="" class="EmphasisTypeItalic">π</em> - <em xmlns="" class="EmphasisTypeItalic">θ</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">W</em>
                  </sub>) and <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">net</em>
                  </sub>), while for the smaller sample sizes we only use <em xmlns="" class="EmphasisTypeItalic">π</em><sub>
                    <em xmlns="" class="EmphasisTypeItalic">b</em>
                  </sub>to avoid null or n.a.n. values (not a number) that are yielded when only one sample is collected from a descendent population.</p>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 simulation analysis demonstrated that msBayes can usually distinguish 
simultaneous divergence from temporal incongruence in divergence, even 
with sparse sampling of individuals. The estimates of Ω were not 
markedly improved by sampling 20 individuals per population pair (10 
each population) when compared to sampling 2–5 individuals per 
population pair (1–3 each population; Figure. <span class="InternalRef"><a href="#Fig3">3</a></span>).
 However, Ω is being overestimated under both sample sizes and this 
upward bias is stronger with larger sample sizes when true Ω = 1. 
Therefore, simultaneous divergence is easier to correctly reject with 
larger sample sizes. Root mean square error (RMSE) for estimating Ω was 
&lt; 0.12 when the true history was simultaneous divergence (Ω = 0), and
 RMSE was &lt; 0.18 when the true history involved 2 different 
divergence events across 10 population pairs (Ω = 0.1). It is 
encouraging that one can obtain fair estimates with so few samples per 
population pair and that two samples per population pair can be analyzed
 by msBayes.<figure xmlns="" class="Figure" id="Fig3"><div xmlns="http://www.w3.org/1999/xhtml" class="MediaObject">
                      <a target="_blank" rel="noopener" href="https://media.springernature.com/full/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig3_HTML.jpg"><img class="c-lazy-image u-js-shown" src="12859_2007_Article_1640_Fig3_HTML.jpg" alt="Figure 3" data-lazy-load="true" data-src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig3_HTML.jpg"><noscript><img src="https://media.springernature.com/lw785/springer-static/image/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_Article_1640_Fig3_HTML.jpg" alt="Figure 3"></img></noscript></a>
                    </div><figcaption class="Caption" lang="en"><div xmlns="http://www.w3.org/1999/xhtml" class="CaptionContent"><span class="CaptionNumber">Figure 3</span>
                        <p xmlns="" class="SimplePara">Performance of 
estimator. Panels A through D each depict frequency histograms of 1,000 Ω
 estimates given 1,000 datasets simulated under either of two 
constrained histories. The simulated histories in panels A and C involve
 simultaneous divergence across ten population pairs (Ω = 0.0; all <em class="EmphasisTypeItalic">τ</em>
 = 1.8), whereas panels B and D are from histories involving two 
different divergence events across the 10 population pairs (Ω = 0.1; two
 splitting at <em class="EmphasisTypeItalic">τ</em> = 1.0 and eight splitting at <em class="EmphasisTypeItalic">τ</em>
 = 2.0). Panels A and B are using small sample sizes (≤ 5 individuals 
per population pair), whereas panels C and D are using samples of 10 
individuals per population pair. The actual sample sizes used for panels
 A and B are species pair 1: 1, 2; pair 2: 3, 2; pair 3: 1, 1; pair 4: 
2, 2; pair 5: 2, 3; pair 6: 2, 1; pair 7: 1, 1; pair 8: 1, 3; pair 9: 3,
 1; pair 10: 2, 1.</p>
                      </div></figcaption></figure></div>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">An
 attractive benefit of an ABC method such as msBayes is that one can 
perform this estimator evaluation relatively quickly. Simulating data 
from parameters drawn from the prior is only done once per set of 
conditions (sample size/history) and can be done in approximately 5 
hours per population pair on a 2 GHz linux computer. The computational 
time can be further reduced as the simulations can be run parallel on 
multiple processors. Because the acceptance/rejection step is decoupled 
from simulating the prior, multiple estimates from a series of simulated
 datasets can be accomplished without re-simulating the prior each time.
 The acceptance/rejection step for a single estimate is accomplished in 
one second to well under a minute such that 1,000 estimates can be 
obtained from 1,000 data sets simulated from fixed known parameter 
values in under an hour to within 24 hours on a single processor.</p>
              </section><section id="Sec5" class="Section2 RenderAsSection2">
                <h3 class="Heading">General use and future development</h3>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 most important aspect of msBayes is that its flexible and modular 
nature will allow us and others to add in new features. This 
characteristic is essential for a phylogeographic software tool because 
phylogeographic studies are highly idiosyncratic. Using population 
genetic data to test how climate and/or geological changes result in 
biogeographic shifts, speciation, extinction and consequent changes in 
ecological interactions can involve a wide array of hypotheses and 
models that conform to no generality with regards to model complexity, 
parameterization and sampling. We therefore anticipate making several 
extensions to msBayes, and will encourage other bioinformaticians to 
make versions that suite particular difficult research questions. 
Furthermore, phylogeographic studies are most powerful when combined 
with independent evidence (or hypotheses) about past habitat 
distributions that are generated from other types of historic data and 
ecological distribution models [<span class="CitationRef"><a href="#CR25" data-toggle="citation">25</a></span>].
 Particular historical hypotheses can then be directly parameterized by 
paleo-distribution models and tested with genetic data within the 
msBayes framework using Bayes factors [<span class="CitationRef"><a href="#CR26" data-toggle="citation">26</a></span>].</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">One
 feature we plan to include in future versions of msBayes is an option 
to simulate from the prior after constraining the number of divergence 
events per <em xmlns="" class="EmphasisTypeItalic">Y</em> population 
pairs to one fixed number. This will then allow getting estimates for 
when these different isolation events took place as well as estimating 
which population pairs originated at either of the particular divergence
 events. Other upcoming features to be included are: 1.) multiple loci 
per population pair by expanding the summary statistic vector and adding
 additional hyper-parameters controlling mutation rate variation across 
loci; 2.) having more summary statistics available; 3.) allowing 
analysis of only one population pair at a time; 4.) testing 
multi-species colonization hypotheses; 5.) three or more population 
models (as opposed to two population models); 6.) allowing 
microsatellite data and 7.) automating the simulation testing procedure 
used to obtain estimator bias.</p>
                <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">It
 should be noted that migration could hinder the ability of this method 
to correctly infer simultaneous divergence. Moderate migration in a 
subset of species/population pairs could cause the method to incorrectly
 support temporal discordance in divergence when the true history 
involved temporal congruence because migration can erase the genetic 
signal of isolation [<span class="CitationRef"><a href="#CR27" data-toggle="citation">27</a></span>, <span class="CitationRef"><a href="#CR28" data-toggle="citation">28</a></span>].
 Although the Bayesian support for temporal concordance in divergence 
times would likely weaken if this happens in a subset of 
species/population pairs, we will explore using the summary statistic 
Var(<em xmlns="" class="EmphasisTypeItalic">π</em>) as a means to tease apart migration from isolation as suggested in [<span class="CitationRef"><a href="#CR29" data-toggle="citation">29</a></span>, <span class="CitationRef"><a href="#CR30" data-toggle="citation">30</a></span>].</p>
              </section></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec6" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Conclusion" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Conclusion">Conclusion</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Conclusion" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The
 msBayes software pipeline will increasingly become an important tool as
 the field of comparative phylogeography progresses to become a more 
rigorous and statistical enterprise [<span class="CitationRef"><a href="#CR5" data-toggle="citation">5</a></span>].
 The program can obtain hyper-parameter estimates using hierarchical 
models in a reasonable amount of time without having the problems 
associated with convergence and mixing found in MCMC methods (Markov 
chain Monte Carlo). Because the estimation step is decoupled from the 
simulation step, one can quickly evaluate different ABC 
acceptance/rejection conditions and the choice of summary statistics. 
The method can reasonably distinguish biogeographic congruence from 
temporal incongruence, even with sparse sampling of individuals. Given 
the complex and idiosyncratic nature of testing multi-species 
biogeographic hypotheses, we envision msBayes as a powerful and flexible
 tool that is open for modification when faced with particularly 
difficult research questions. Finally, due to its flexible and modular 
design, msBayes will be a well-suited tool for the heterogeneous data 
sets that are emerging and being combined to test complex historical 
hypotheses.</p></div></section>
            <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" id="Sec7" class="Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Availabilityandrequirements" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Availabilityandrequirements">Availability and requirements</a></h2><div class="js-CollapseSection CollapseSection c-section__content" id="Availabilityandrequirements" data-test="collapse-section"><p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">The installation instructions, documentation, source code and precompiled binary for msBayes are all available for download at <span class="ExternalRef"><a href="http://msbayes.sourceforge.net/"><span class="RefSource">http://msbayes.sourceforge.net/</span></a></span>
 under an open source license (GNU Public License). The msBayes pipeline
 is comprised of several C and R programs that are run with a Perl 
"front-end" and runs on Linux, Mac OS-X, and most POSIX systems.</p></div></section>
          <section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="AbbreviationGroup Section1 RenderAsSection1 c-section"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#ListofAbbreviationsused" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="ListofAbbreviationsused">List of Abbreviations used</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="ListofAbbreviationsused"><div class="DefinitionList">
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>ABC:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Approximate Bayesian Computation</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>TSD:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Test of simultaneous divergence</p>
                  </dd></dl>
                <dl class="DefinitionListEntry"><dt class="Term"><dfn>mtDNA:&nbsp;</dfn></dt><dd class="Description">
                    <p xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Mitochondrial DNA</p>
                  </dd></dl>
              </div></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" data-test="declarations-section" id="Decs"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Declarations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Declarations">Declarations</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="Declarations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Acknowledgments"><h3 xmlns="" class="Heading">Acknowledgements</h3><p xmlns="" class="SimplePara">We
 thank A. Lancaster and three anonymous reviewers for making helpful 
suggestions and E. Andersson for handling the manuscript. We thank M. 
Beaumont for kindly providing R scripts and critically useful 
discussions. We thank D. Hudson for permission to use E. Stahl's finite 
sites version of his ms coalescent simulator under GNU Public License. 
We thank J. McGuire and C. Moritz for use of the linux parallel 
computing cluster housed at the Museum of Vertebrate zoology (University
 of California, Berkeley). We thank MBI (Mathematical Biosciences 
Institute) for hosting the workshop on Phylogenetics and Phylogeography.
 Support for M. J. Hickerson was provided by a NSF post-doctoral 
fellowship in interdisciplinary informatics. N. Takebayashi was 
supported by NIH Grant Number 2P20RR16466 from the INBRE program of the 
National Center for Research Resources and NSF DEB-0640520. E. Stahl, 
was supported by Sloan/DOE Fellowship award DE-FG02-00ER62993.</p></div><aside class="Appendix" id="App1">
              
                <h3 class="Heading">Authors' original submitted files for images</h3><div id="widgetContainer" data-test="figshare-container"></div>
                <div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Para">Below are the links to the authors’ original submitted files for images.<div class="MediaObject" id="MOESM1">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_1640_MOESM1_ESM.pdf">12859_2007_1640_MOESM1_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 1</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM2">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_1640_MOESM2_ESM.pdf">12859_2007_1640_MOESM2_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 2</span>
                        </div></div>
                  </div><div class="MediaObject" id="MOESM3">
                    <div xmlns="" class="esm-item "><div xmlns="http://www.w3.org/1999/xhtml" class="caption-container"><a class="filename" href="https://static-content.springer.com/esm/art%3A10.1186%2F1471-2105-8-268/MediaObjects/12859_2007_1640_MOESM3_ESM.pdf">12859_2007_1640_MOESM3_ESM.pdf</a>
                          <span xmlns="" class="SimplePara">Authors’ original file for figure 3</span>
                        </div></div>
                  </div></div>
              
            </aside><aside xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ArticleNote ArticleNoteMisc"><h3 xmlns="" class="Heading">Authors' contributions</h3><p xmlns="" class="SimplePara">MJH
 developed the idea for using ABC within a hierarchical model to analyze
 multiple population pairs simultaneously. ES developed the finite sites
 version of D. Hudson's classic coalescent simulator (ms). MJH and NT 
developed C, R, and Perl routines and modified pre-existing R and C 
routines to comprise an ABC algorithm that makes use of a Hierarchical 
model. NT extensively developed the C and Perl routines that comprise 
the user version of msBayes now available. NT and MJH maintains the 
msBayes website and NT developed the installation configurations and 
precompiled binaries. All authors read and approved the final version of
 the manuscript.</p></aside></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section Affiliations" id="Aff"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#Author-Affiliations" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="Author-Affiliations">Authors’ Affiliations</a></h2><div class="js-CollapseSection CollapseSection c-section__content c-section__content" data-test="collapse-section" id="Author-Affiliations"><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff1"><span class="AffiliationNumber">(1)</span><div class="AffiliationText">Biology Department, Queens College, CUNY, 65-30 Kissena Blvd, Flushing, NY&nbsp;11367-1597, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff2"><span class="AffiliationNumber">(2)</span><div class="AffiliationText">Department of Biology, University of Massachusetts Dartmouth, 285 Old Westport Rd, North Dartmouth, MA&nbsp;02747, USA</div></div><div xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="Affiliation" id="Aff3"><span class="AffiliationNumber">(3)</span><div class="AffiliationText">Institute
 of Arctic Biology and Department of Biology and Wildlife, University of
 Alaska, 311 Irving I Bldg, Fairbanks, AK&nbsp;99775, USA</div></div></div></section></div>
                        <div class="FulltextWrapper"><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section SectionTypeMaterialsAndMethods" id="Bib1"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#References" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="References">References</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="References"><ol class="BibliographyWrapper"><li class="Citation"><cite class="CitationContent" id="CR1" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Avise JC: Phylogeography: The history and formation of species. Cambridge: Harvard University Press; 2000.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=JC.%20Avise&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR2" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wen
 J: Origin and the evolution of the eastern North American distinct 
distributions of flowering plants. Annu Rev Ecol Syst 1999, 30: 421–455.
 10.1146/annurev.ecolsys.30.1.421</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.ecolsys.30.1.421"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Origin%20and%20the%20evolution%20of%20the%20eastern%20North%20American%20distinct%20distributions%20of%20flowering%20plants&amp;author=J.%20Wen&amp;journal=Annu%20Rev%20Ecol%20Syst&amp;volume=30&amp;pages=421-455&amp;publication_year=1999&amp;doi=10.1146%2Fannurev.ecolsys.30.1.421"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR3" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Lessios
 HA: The first stage of speciation as seen in organisms separated by the
 Isthmus Panama. In Endless Forms: species and speciation. Edited by: 
Howard DJ, Berlocher S. Oxford, U.K.: Oxford University Press; 
1998:186–201.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20first%20stage%20of%20speciation%20as%20seen%20in%20organisms%20separated%20by%20the%20Isthmus%20Panama&amp;author=HA.%20Lessios&amp;pages=186-201&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR4" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Barraclough TG, Nee S: Phylogenetics and speciation. Trends Ecol Evol 2001, 16(7):391–399. 10.1016/S0169-5347(01)02161-9</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/S0169-5347%2801%2902161-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11403872"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Phylogenetics%20and%20speciation&amp;author=TG.%20Barraclough&amp;author=S.%20Nee&amp;journal=Trends%20Ecol%20Evol&amp;volume=16&amp;issue=7&amp;pages=391-399&amp;publication_year=2001&amp;doi=10.1016%2FS0169-5347%2801%2902161-9"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR5" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Knowles LL, Maddison WP: Statistical phylogeography. Mol Ecol 2002, 11(12):2623–2635. 10.1046/j.1365-294X.2002.01637.x</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294X.2002.01637.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12453245"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Statistical%20phylogeography&amp;author=LL.%20Knowles&amp;author=WP.%20Maddison&amp;journal=Mol%20Ecol&amp;volume=11&amp;issue=12&amp;pages=2623-2635&amp;publication_year=2002&amp;doi=10.1046%2Fj.1365-294X.2002.01637.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR6" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edwards
 SV, Liu L, Pearl DK: High-resolution species trees without 
concatination. Proc Natl Acad Sci USA 2007, 104: 5936–5941. 
10.1073/pnas.0607004104</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1851595"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0607004104"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17392434"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=High-resolution%20species%20trees%20without%20concatination&amp;author=SV.%20Edwards&amp;author=L.%20Liu&amp;author=DK.%20Pearl&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=104&amp;pages=5936-5941&amp;publication_year=2007&amp;doi=10.1073%2Fpnas.0607004104"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR7" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Rannala
 B, Yang ZH: Bayes Estimation of Species Divergence Times and Ancestral 
Population Sizes Using DNA Sequences From Multiple Loci. Genetics 2003, 
164: 1645–1656.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1462670"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12930768"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bayes%20Estimation%20of%20Species%20Divergence%20Times%20and%20Ancestral%20Population%20Sizes%20Using%20DNA%20Sequences%20From%20Multiple%20Loci&amp;author=B.%20Rannala&amp;author=ZH.%20Yang&amp;journal=Genetics&amp;volume=164&amp;pages=1645-1656&amp;publication_year=2003"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR8" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hey
 J, Nielsen R: Integration within the Felsenstein equation for improved 
Markov chain Monte Carlo methods in population genetics. Proc Natl Acad 
Sci USA 2007, 104: 2785–2790. 10.1073/pnas.0611164104</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1815259"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.0611164104"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17301231"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Integration%20within%20the%20Felsenstein%20equation%20for%20improved%20Markov%20chain%20Monte%20Carlo%20methods%20in%20population%20genetics&amp;author=J.%20Hey&amp;author=R.%20Nielsen&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=104&amp;pages=2785-2790&amp;publication_year=2007&amp;doi=10.1073%2Fpnas.0611164104"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR9" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Edwards
 SV, Beerli P: Perspective: Gene divergence, population divergence, and 
the variance in coalescence time in phylogeographic studies. Evolution 
2000, 54(6):1839–1854.</span><span class="Occurrences"><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11209764"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Perspective%3A%20Gene%20divergence%2C%20population%20divergence%2C%20and%20the%20variance%20in%20coalescence%20time%20in%20phylogeographic%20studies&amp;author=SV.%20Edwards&amp;author=P.%20Beerli&amp;journal=Evolution&amp;volume=54&amp;issue=6&amp;pages=1839-1854&amp;publication_year=2000"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR10" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hickerson
 MJ, Stahl E, Lessios HA: Test for simultaneous divergence using 
approximate Bayesian computation. Evolution 2006, 60: 2435–2453.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.0014-3820.2006.tb01880.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17263107"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Test%20for%20simultaneous%20divergence%20using%20approximate%20Bayesian%20computation&amp;author=MJ.%20Hickerson&amp;author=E.%20Stahl&amp;author=HA.%20Lessios&amp;journal=Evolution&amp;volume=60&amp;pages=2435-2453&amp;publication_year=2006"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR11" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Thornton
 K, Andolfatto P: Approximate Bayesian inference reveals evidence for a 
recent, severe bottleneck in a Netherlands population of Drosophila 
melanogaster. Genetics 2005, 172: 1607–1619. 10.1534/genetics.105.048223</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.105.048223"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16299396"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Approximate%20Bayesian%20inference%20reveals%20evidence%20for%20a%20recent%2C%20severe%20bottleneck%20in%20a%20Netherlands%20population%20of%20Drosophila%20melanogaster&amp;author=K.%20Thornton&amp;author=P.%20Andolfatto&amp;journal=Genetics&amp;volume=172&amp;pages=1607-1619&amp;publication_year=2005&amp;doi=10.1534%2Fgenetics.105.048223"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR12" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Excoffier
 L, Estoup A, Cornuet J-M: Bayesian analysis of an admixture model with 
mutations and arbitrarily linked markers. Genetics 2005, 169: 1727–1738.
 10.1534/genetics.104.036236</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1449551"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.104.036236"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15654099"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bayesian%20analysis%20of%20an%20admixture%20model%20with%20mutations%20and%20arbitrarily%20linked%20markers&amp;author=L.%20Excoffier&amp;author=A.%20Estoup&amp;author=J-M.%20Cornuet&amp;journal=Genetics&amp;volume=169&amp;pages=1727-1738&amp;publication_year=2005&amp;doi=10.1534%2Fgenetics.104.036236"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR13" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Estoup
 A, Beaumont BA, Sennedot F, Moritz C, Cornuet J-M: Genetic analysis of 
complex demographic scenarios: spatially expanding populations of the 
cane toad, Bufo marinus . Evolution 2004, 58: 2021–2036.</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.0014-3820.2004.tb00487.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15521459"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20complex%20demographic%20scenarios%3A%20spatially%20expanding%20populations%20of%20the%20cane%20toad%2C%20Bufo%20marinus&amp;author=A.%20Estoup&amp;author=BA.%20Beaumont&amp;author=F.%20Sennedot&amp;author=C.%20Moritz&amp;author=J-M.%20Cornuet&amp;journal=Evolution&amp;volume=58&amp;pages=2021-2036&amp;publication_year=2004"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR14" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Tallmon
 DA, Luikart G, Beaumont BA: Comparative evaluation of a new effective 
population size estimator based on approximate Bayesian computation. 
Genetics 2004, 167: 977–988. 10.1534/genetics.103.026146</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1470910"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.103.026146"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15238546"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20evaluation%20of%20a%20new%20effective%20population%20size%20estimator%20based%20on%20approximate%20Bayesian%20computation&amp;author=DA.%20Tallmon&amp;author=G.%20Luikart&amp;author=BA.%20Beaumont&amp;journal=Genetics&amp;volume=167&amp;pages=977-988&amp;publication_year=2004&amp;doi=10.1534%2Fgenetics.103.026146"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR15" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Beaumont MA, Zhang W, Balding DJ: Approximate Bayesian computation in population genetics. Genetics 2002, 162: 2025–2035.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1462356"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12524368"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Approximate%20Bayesian%20computation%20in%20population%20genetics&amp;author=MA.%20Beaumont&amp;author=W.%20Zhang&amp;author=DJ.%20Balding&amp;journal=Genetics&amp;volume=162&amp;pages=2025-2035&amp;publication_year=2002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR16" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Weiss G, von Haeseler A: Inference of population history using a likelihood approach. Genetics 1998, 149: 1539–1546.</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1460236"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9649540"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20history%20using%20a%20likelihood%20approach&amp;author=G.%20Weiss&amp;author=A.%20von%20Haeseler&amp;journal=Genetics&amp;volume=149&amp;pages=1539-1546&amp;publication_year=1998"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR17" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Watterson
 GA: On the number of segregating sites in genetic models without 
recombination. Theor Popul Biol 1975, 7: 256–276. 
10.1016/0040-5809(75)90020-9</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1016/0040-5809%2875%2990020-9"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1145509"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=On%20the%20number%20of%20segregating%20sites%20in%20genetic%20models%20without%20recombination&amp;author=GA.%20Watterson&amp;journal=Theor%20Popul%20Biol&amp;volume=7&amp;pages=256-276&amp;publication_year=1975&amp;doi=10.1016%2F0040-5809%2875%2990020-9"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR18" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Nei
 M, Li W: Mathematical model for studying variation in terms of 
restriction endonucleases. Proc Natl Acad Sci USA 1979, 76: 5269–5273. 
10.1073/pnas.76.10.5269</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC413122"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.76.10.5269"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=291943"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Mathematical%20model%20for%20studying%20variation%20in%20terms%20of%20restriction%20endonucleases&amp;author=M.%20Nei&amp;author=W.%20Li&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=76&amp;pages=5269-5273&amp;publication_year=1979&amp;doi=10.1073%2Fpnas.76.10.5269"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR19" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hickerson
 MJ, Dolman G, Moritz C: Comparative phylogeographic summary statistics 
for testing simultaneous vicariance across taxon-pairs. Mol Ecol 2006, 
15: 209–224. 10.1111/j.1365-294X.2005.02718.x</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1111/j.1365-294X.2005.02718.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16367841"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Comparative%20phylogeographic%20summary%20statistics%20for%20testing%20simultaneous%20vicariance%20across%20taxon-pairs&amp;author=MJ.%20Hickerson&amp;author=G.%20Dolman&amp;author=C.%20Moritz&amp;journal=Mol%20Ecol&amp;volume=15&amp;pages=209-224&amp;publication_year=2006&amp;doi=10.1111%2Fj.1365-294X.2005.02718.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR20" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hudson
 RR: Generating samples under a Wright-Fisher neutral model of genetic 
variation. Bioinformatics 2002, 18: 337–338. 
10.1093/bioinformatics/18.2.337</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1093/bioinformatics/18.2.337"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11847089"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Generating%20samples%20under%20a%20Wright-Fisher%20neutral%20model%20of%20genetic%20variation&amp;author=RR.%20Hudson&amp;journal=Bioinformatics&amp;volume=18&amp;pages=337-338&amp;publication_year=2002&amp;doi=10.1093%2Fbioinformatics%2F18.2.337"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR21" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hey
 J, Nielsen R: Multilocus methods for estimating population sizes, 
migration rates and divergence time, with applications to the divergence
 of Drosophila pseudoobscura and D. persimilis . Genetics 2004, 167: 
747–760. 10.1534/genetics.103.024182</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1470901"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1534/genetics.103.024182"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15238526"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Multilocus%20methods%20for%20estimating%20population%20sizes%2C%20migration%20rates%20and%20divergence%20time%2C%20with%20applications%20to%20the%20divergence%20of%20Drosophila%20pseudoobscura%20and%20D.%20persimilis&amp;author=J.%20Hey&amp;author=R.%20Nielsen&amp;journal=Genetics&amp;volume=167&amp;pages=747-760&amp;publication_year=2004&amp;doi=10.1534%2Fgenetics.103.024182"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR22" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Beaumont MA, Rannala B: The Bayesian revolution in genetics. Nat Rev Genet 2004, 5: 251–261. 10.1038/nrg1318</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1038/nrg1318"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15131649"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20Bayesian%20revolution%20in%20genetics&amp;author=MA.%20Beaumont&amp;author=B.%20Rannala&amp;journal=Nat%20Rev%20Genet&amp;volume=5&amp;pages=251-261&amp;publication_year=2004&amp;doi=10.1038%2Fnrg1318"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR23" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">James
 W, Stein C: Estimation with quadradic loss. In Proceedings of the 
Fourth Berkeley Symposium on Mathematical Statistics and Probability: 
1960. Berkeley, CA: University of California Press; 1960.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Estimation%20with%20quadradic%20loss&amp;author=W.%20James&amp;author=C.%20Stein&amp;publication_year=1960"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR24" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Gelman A, Carlin JB, Stern HS, Rubin DB: Bayesian Data Analysis. London, UK: Chapman and Hall/CRC; 1995.</span><span class="Occurrences"><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?&amp;author=A.%20Gelman&amp;author=JB.%20Carlin&amp;author=HS.%20Stern&amp;author=DB.%20Rubin&amp;publication_year=1995"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR25" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Hugall
 A, Moritz C, Moussalli A, Stanisic J: Reconciling paleodistribution 
models and comparative phylogeography in the Wet Tropics rainforest land
 snail Gnarosophia bellendenkerensis (Brazier 1875). Proc Natl Acad Sci 
USA 2002, 99(9):6112–6117. 10.1073/pnas.092538699</span><span class="Occurrences"><span class="Occurrence OccurrencePMCID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC122911"><span><span>PubMed Central</span></span></a></span><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1073/pnas.092538699"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11972064"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Reconciling%20paleodistribution%20models%20and%20comparative%20phylogeography%20in%20the%20Wet%20Tropics%20rainforest%20land%20snail%20Gnarosophia%20bellendenkerensis%20%28Brazier%201875%29&amp;author=A.%20Hugall&amp;author=C.%20Moritz&amp;author=A.%20Moussalli&amp;author=J.%20Stanisic&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=99&amp;issue=9&amp;pages=6112-6117&amp;publication_year=2002&amp;doi=10.1073%2Fpnas.092538699"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR26" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kass RE, Raftery A: Bayesian factors. Journal of the American Statistical Association 1995, 90: 773–795. 10.2307/2291091</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1080/01621459.1995.10476572"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Bayesian%20factors&amp;author=RE.%20Kass&amp;author=A.%20Raftery&amp;journal=Journal%20of%20the%20American%20Statistical%20Association&amp;volume=90&amp;pages=773-795&amp;publication_year=1995&amp;doi=10.2307%2F2291091"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR27" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Kalinowski
 ST: Evolutionary and statistical properties of three genetic distances.
 Mol Ecol 2002, 11(8):1263–1273. 10.1046/j.1365-294X.2002.01520.x</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1046/j.1365-294X.2002.01520.x"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12144649"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Evolutionary%20and%20statistical%20properties%20of%20three%20genetic%20distances&amp;author=ST.%20Kalinowski&amp;journal=Mol%20Ecol&amp;volume=11&amp;issue=8&amp;pages=1263-1273&amp;publication_year=2002&amp;doi=10.1046%2Fj.1365-294X.2002.01520.x"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR28" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Slatkin M: Gene flow in natural populations. Ann Rev Ecol Syst 1985, 16: 393–430. 10.1146/annurev.ecolsys.16.1.393</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1146/annurev.es.16.110185.002141"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Gene%20flow%20in%20natural%20populations&amp;author=M.%20Slatkin&amp;journal=Ann%20Rev%20Ecol%20Syst&amp;volume=16&amp;pages=393-430&amp;publication_year=1985&amp;doi=10.1146%2Fannurev.ecolsys.16.1.393"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR29" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wakeley
 J: The variance of pairwise nucleotide differences in two populations 
with migration. Theor Popul Biol 1996, 49: 39–57. 10.1006/tpbi.1996.0002</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/tpbi.1996.0002"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8813013"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=The%20variance%20of%20pairwise%20nucleotide%20differences%20in%20two%20populations%20with%20migration&amp;author=J.%20Wakeley&amp;journal=Theor%20Popul%20Biol&amp;volume=49&amp;pages=39-57&amp;publication_year=1996&amp;doi=10.1006%2Ftpbi.1996.0002"><span><span>Google Scholar</span></span></a></span></span></cite></li><li class="Citation"><cite class="CitationContent" id="CR30" itemprop="citation" itemscope="" itemtype="http://schema.org/ScholarlyArticle"><span itemprop="headline">Wakeley
 J: Distinguishing migration from isolation using the variance of 
pairwise differences. Theor Popul Biol 1996, 49: 369–386. 
10.1006/tpbi.1996.0018</span><span class="Occurrences"><span class="Occurrence OccurrenceDOI"><a target="_blank" rel="noopener" href="https://doi.org/10.1006/tpbi.1996.0018"><span><span>View Article</span></span></a></span><span class="Occurrence OccurrencePID"><a target="_blank" rel="noopener nofollow" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8693431"><span><span>PubMed</span></span></a></span><span class="Occurrence OccurrenceGS"><a target="_blank" rel="noopener" href="http://scholar.google.com/scholar_lookup?title=Distinguishing%20migration%20from%20isolation%20using%20the%20variance%20of%20pairwise%20differences&amp;author=J.%20Wakeley&amp;journal=Theor%20Popul%20Biol&amp;volume=49&amp;pages=369-386&amp;publication_year=1996&amp;doi=10.1006%2Ftpbi.1996.0018"><span><span>Google Scholar</span></span></a></span></span></cite></li></ol></div></section><section xmlns="" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:meta="http://www.springer.com/app/meta" class="Section1 RenderAsSection1 c-section" id="CopyrightMessage"><h2 class="Heading js-ToggleCollapseSection c-section__heading" data-sticky-update="true"><a href="#HickersonetallicenseeBioMedCentralLtd" data-toggle="collapse" data-toggle-disable-clickoff="" data-target="HickersonetallicenseeBioMedCentralLtd">Copyright</a></h2><div class="js-CollapseSection CollapseSection c-section__content" data-test="collapse-section" id="HickersonetallicenseeBioMedCentralLtd"><div xmlns="http://www.w3.org/1999/xhtml" class="ArticleCopyright">©&nbsp;Hickerson et al; licensee BioMed Central Ltd.&nbsp;2007<div class="CopyrightComment">
                <p xmlns="" class="SimplePara">This article is published
 under license to BioMed Central Ltd. This is an Open Access article 
distributed under the terms of the Creative Commons Attribution License (<span xmlns:func="http://oscar.fig.bmc.com" xmlns="http://www.w3.org/1999/xhtml" class="ExternalRef"><a href="http://creativecommons.org/licenses/by/2.0"><span class="RefSource">http://creativecommons.org/licenses/by/2.0</span></a></span>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p>
              </div></div></div></section></div>
                        


                    </article>
                </main>

                <div class="c-content-layout__side u-text-sm">
                    <aside>
                        <div class="p-separator">
                            
        <p class="u-margin-bottom-sm">
            <a id="articlePdf" data-test="download-pdf" class="c-button c-button--primary c-button--full-width" target="_blank" rel="noopener" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/1471-2105-8-268" data-track="click" data-track-category="PDF Download" data-track-action="Click download article PDF" data-track-label="10.1186/1471-2105-8-268">Download PDF</a>
        </p>
    
                            
                            
    <button type="button" class="c-button c-button--secondary c-button--full-width c-button--expandable" data-toggle="collapse" data-target="export-citations" data-test="export-citations-button" data-sticky-update="true" aria-controls="export-citations" aria-expanded="false">
        <span class="c-dropdown__toggle">Export citations</span>
    </button>
    <div id="export-citations" class="c-export-citations" data-test="export-citations-content">
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Papers, Zotero, Reference Manager, RefWorks (<strong>.RIS</strong>)</h4>
            <ul class="c-export-citations__list" id="articleReferences">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks article citations" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=refman&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks references" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=refman&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Papers, Zotero, Reference Manager, RefWorks citations and references" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=refman&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">EndNote (<strong>.ENW</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote article citations" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=endnote&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote references" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=endnote&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download EndNote citations and references" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=endnote&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
        <div class="c-export-citations__section">
            <h4 class="c-export-citations__title">Mendeley, JabRef (<strong>.BIB</strong>)</h4>
            <ul class="c-export-citations__list">
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef article citations" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=bibtex&amp;flavour=citation">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Citations
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef references" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=bibtex&amp;flavour=references">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>References
                </a></li>
                <li><a data-track="click" data-track-category="Article Page" data-track-action="Download Mendeley, JabRef citations and references" data-track-label="10.1186/1471-2105-8-268" href="http://citation-needed.springer.com/v2/references/10.1186/1471-2105-8-268?format=bibtex&amp;flavour=full">
                    <svg class="c-export-citations__icon" width="12" height="12" aria-hidden="true">
                        <use xlink:href="#icon-download"></use>
                    </svg>
                    <span class="u-visually-hidden">Download </span>Both
                </a></li>
            </ul>
        </div>
    </div>

                        </div>
                        
    
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Metrics</h3>
                            <ul class="c-list-group c-list-group--md">
                                
            <li class="c-list-group__item">
                Article accesses:
                
                    8465
                
            </li>
        
                                
            <li class="c-list-group__item">
                Citations: 81 <a href="http://citations.springer.com/item?doi=10.1186/1471-2105-8-268" rel="noopener" target="_blank">more information</a>
            </li>
        
                                
    <li class="c-list-group__item">
        <p>Altmetric Attention Score: 3</p>
        <div class="altmetric">
            <a class="altmetric__link" href="http://www.altmetric.com/details.php?citation_id=31283462&amp;domain=www.biomedcentral.com" target="_blank" rel="noopener" data-track="click" data-track-category="Article Page" data-track-action="Click Altmetric" data-track-label="10.1186/1471-2105-8-268">
                <img class="altmetric__img" src="a.png" alt="Altmetric Attention Score">
            </a>
        </div>
    </li>

                            </ul>
                        </div>
                        
    <div id="sociaMediaShare" class="p-separator u-js-shown">
        <h3 class="p-section-title">Share this article</h3>
        <ul id="shareButtons" class="c-social-links">
            <li class="c-social-links__item">
                <button id="shareOnTwitter" class="js-btnShareOnTwitter" data-track="click" data-track-category="Article Page" data-track-action="Share Twitter" data-track-label="10.1186/1471-2105-8-268">
                    <span class="u-visually-hidden">Share on Twitter</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-twitter"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnFacebook" class="js-btnShareOnFacebook" data-track="click" data-track-category="Article Page" data-track-action="Share Facebook" data-track-label="10.1186/1471-2105-8-268">
                    <span class="u-visually-hidden">Share on Facebook</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-facebook"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnLinkedIn" class="js-btnShareOnLinkedIn" data-track="click" data-track-category="Article Page" data-track-action="Share LinkedIn" data-track-label="10.1186/1471-2105-8-268">
                    <span class="u-visually-hidden">Share on LinkedIn</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-linkedin"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnWeibo" class="js-btnShareOnWeibo" data-track="click" data-track-category="Article Page" data-track-action="Share Weibo" data-track-label="10.1186/1471-2105-8-268">
                    <span class="u-visually-hidden">Share on Weibo</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-weibo"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnGooglePlus" class="js-btnShareOnGplus" data-track="click" data-track-category="Article Page" data-track-action="Share Google+" data-track-label="10.1186/1471-2105-8-268">
                    <span class="u-visually-hidden">Share on Google Plus</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-gplus"></use>
                    </svg>
                </button>
            </li>
            <li class="c-social-links__item">
                <button id="shareOnReddit" class="js-btnShareOnReddit" data-track="click" data-track-category="Article Page" data-track-action="Share Reddit" data-track-label="10.1186/1471-2105-8-268">
                    <span class="u-visually-hidden">Share on Reddit</span>
                    <svg class="c-icon c-social-links__icon" width="24" height="24" aria-hidden="true">
                        <use xlink:href="#icon-reddit"></use>
                    </svg>
                </button>
            </li>
        </ul>
    </div>
 
                        
    
 
                        <div class="p-separator">
                            <h3 class="p-section-title">Other Actions</h3>
                            <ul class="c-list-group c-list-group--bordered c-list-group c-list-group--md">
                                <li class="c-list-group__item">
                                    
    <a data-track="click" data-track-category="Article Page" data-track-action="Order article reprints" data-track-label="10.1186/1471-2105-8-268" href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&amp;page=0&amp;journal=738&amp;doi=10.1186%2F1471-2105-8-268&amp;volume=8&amp;issue=1&amp;title=msBayes%3A+Pipeline+for+testing+comparative+phylogeographic+histories+using+hierarchical+approximate+Bayesian+computation&amp;author_name=Michael+J+Hickerson&amp;start_page=1&amp;end_page=7">Order reprint</a>

                                </li>
                            </ul>
                        </div>
                    </aside>
                    
    <div id="FulltextAd" class="adsbox c-ad c-ad--skyscraper">
        <div class="c-ad__inner" data-component="sticky" data-test="fulltext-ad">
            <p class="c-ad__label">Advertisement</p>
            <div class="c-ad__content">
                <span id="pbgrd-43415139"></span>
            </div>
        </div>
    </div>

                </div>
            </div>
        </div>


        
            <div class="c-journal-footer">
                <div class="c-journal-footer__inner">
                    <div class="c-journal-footer__summary">
                        <h4 class="c-journal-title c-journal-title--footer">
                            
                            <span class="c-journal-title__text">BMC Bioinformatics</span>
                        </h4>
                        <p class="c-journal-footer__issn">ISSN: 1471-2105</p>
                    </div>
                    
                        <div class="c-journal-footer__contact">
                            <h4 class="c-journal-footer__contact-title ">Contact us</h4>
                            <ul class="c-journal-footer__contact-list">
                                
                                    <li class="c-journal-footer__contact-item">Submission enquiries: <a href="mailto:bmcbioinformatics@biomedcentral.com">bmcbioinformatics@biomedcentral.com</a></li>
                                
                                
                                    <li class="c-journal-footer__contact-item">General enquiries: <a href="mailto:info@biomedcentral.com">info@biomedcentral.com</a></li>
                                
                            </ul>
                        </div>
                    
                </div>
            </div>
        
        
    <img rel="nofollow" class="tracker" style="display:none" src="1471-2105-8-268.png" alt="">

         
    <footer>
        <div class="ctx-publisher-footer">
    <div class="c-navbar">
        <div class="u-container">
            <div class="c-navbar__logo u-margin-right-xl">
                <a href="https://www.biomedcentral.com/">
                    <img src="logo-bmc-series-9977e4f82f.svg" alt="BMC logo" data-test="navbar-logo-footer">

                </a>
            </div>
            <div class="c-navbar__content">
                
    <ul class="c-navbar__nav" data-test="publisher-navigation">
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/journals">
                Explore journals
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/getpublished">
                Get published
            </a>
        </li>
        
        <li class="c-navbar__item">
            <a class="c-navbar__link" href="https://www.biomedcentral.com/about">
                About BMC
            </a>
        </li>
        
    </ul>


            </div>
        </div>
    </div>
    <div class="c-publisher-footer" data-test="publisher-footer">
        <div class="u-container">
            
            <div class="c-flex c-flex--wrap c-flex--space-between" role="navigation" data-test="publisher-footer-menu">
                <div class="c-flex" aria-label="Publisher footer links">
                    
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://blogs.biomedcentral.com/">Read more on our blogs</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/login">Receive BMC newsletters</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/account">Manage article alerts</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://authorservices.springernature.com/go/10BMC">Language editing for authors</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="http://authorservices.springernature.com/scientific-editing/">Scientific editing for authors</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/policies">Policies</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/accessibility">Accessibility</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/press-centre">Press center</a>
                                    </li>
                                
                            </ul>
                        
                            <ul class="c-list-group c-list-group--sm u-margin-right-lg u-margin-bottom-md">
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/contact-us">Contact us</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://biomedcentral.typeform.com/to/VLXboo">Leave feedback</a>
                                    </li>
                                
                                    <li class="c-list-group__item">
                                        <a class="u-text-inherit" href="https://www.biomedcentral.com/about/jobs">Careers</a>
                                    </li>
                                
                            </ul>
                        
                    
                </div>
                <div class="u-margin-bottom-lg">
                    <h3 id="social-menu" class="u-text-sm u-reset-margin u-text-normal">Follow BMC</h3>
                    <ul class="c-flex u-reset-list" data-test="footer-social-links">
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://twitter.com/biomedcentral" class="u-text-inherit" data-track="click" data-track-action="Click Twitter link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Twitter page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-twitter-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://www.facebook.com/BioMedCentral" class="u-text-inherit" data-track="click" data-track-action="Click Facebook link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Facebook page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-facebook-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="https://plus.google.com/106226337984152901734/" class="u-text-inherit" data-track="click" data-track-action="Click Google Plus link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Google Plus page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-gplus-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                            <li class="u-margin-top-xs u-margin-right-xs">
                                <a href="http://www.weibo.com/biomedcentral" class="u-text-inherit" data-track="click" data-track-action="Click Weibo link" data-track-category="Publisher footer">
                                    <span class="u-visually-hidden">BMC Weibo page</span>
                                    <svg class="c-icon" width="24" height="24" aria-hidden="true">
                                        <use xlink:href="#icon-weibo-bordered"></use>
                                    </svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </div>
            <p class="u-reset-margin">
                By using this website, you agree to our
                <a class="u-text-inherit" href="https://www.biomedcentral.com/terms-and-conditions">Terms and Conditions</a>,
                <a class="u-text-inherit" href="https://www.biomedcentral.com/privacy-statement">Privacy
                    statement</a> and
                <a class="u-text-inherit" href="https://www.biomedcentral.com/cookies" data-test="cookie-link">Cookies</a> policy.
                
                    <a class="optanon-toggle-display u-text-inherit" href="javascript:void(0);">Manage the cookies</a> we use in the preference centre.
                
            </p>
        </div>
    </div>
</div>


        <div class="c-sn-footer c-sn-footer--neutral">
    <div class="u-container">
        <img src="logo-springernature-8bd4e8d82f.svg" class="c-sn-footer__logo" alt="Springer Nature Logo" itemprop="logo">
        <p data-test="copyright">© 2018 BioMed Central Ltd unless otherwise stated. Part of
            <a href="http://www.springernature.com/" data-track="click" data-track-action="Click Springer Nature link" data-track-category="Corporate footer" itemscope="" itemtype="http://schema.org/Organization" itemid="#parentOrganization">Springer Nature</a>.
        </p>
    </div>
</div>

    </footer>

    </div>

    
    <script type="text/javascript" src="bmc.js" async="async"></script>
    <noscript>
	<img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>
    <!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script id="springerlink-event-tracker" src="eventTracker.txt"></script>
<script type="text/javascript">
    (function(){
        var dataLayer = window.dataLayer[0];
        var pageType = dataLayer.page.category.pageType;
        var doi = dataLayer.content.article.doi;
        var imprint = dataLayer.content.contentInfo.imprint;
        var publisher;

        if (pageType) {
            pageType = pageType.charAt(0).toUpperCase() + pageType.substr(1);
        }

        if (imprint === 'BioMed Central') { publisher = 'BMC' }
        else if (imprint === 'SpringerOpen') { publisher = 'SpringerOpen' }
        else { throw new Error('Event Tracker Error: The publisher has an unexpected value') }

        new EventTracker({'platform':publisher}).sendEvent('display',{'content_type':pageType,'doi':doi});
    })();
</script><iframe src="https://event-tracker.springernature.com" title="EventTracker" id="event-tracker" style="display:none; visibility:hidden" aria-hidden="true" tabindex="-1" width="0" height="0"></iframe>


    


<script type="text/javascript" id="polyfill-matches">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="gpt-control" src="gpt.js"></script><script type="text/javascript" id="krux-control-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var a=document.createElement("script");a.type="text/javascript";a.async=!0;a.src=("https:"===location.protocol?"https:":"http:")+"//cdn.krxd.net/controltag/KDqylSLE.js";var b=document.getElementsByTagName("script")[0];b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="krux-interchange-bmc">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""})();</script><script type="text/javascript" id="krux-consent">var allowed=google_tag_manager["GTM-TDGJHK"].macro(23);allowed=!0===allowed?1:0;Krux("consent:set",{dc:allowed,al:allowed,tg:allowed,cd:!1,sh:!1,re:!1},function(a,b){});</script><script type="text/javascript" id="gpt-retrieve-ads">function splitKeys(a){var d=[],e="",g=[],h=a.split(";");for(a=0;a<h.length;++a){void 0!==k&&(e=k);var b=h[a].split("\x3d");var k=b[0];k!==e&&(0<e.length&&0<d.length&&g.push([e,d]),d=[]);if(2===b.length&&""!==b[0]&&""!==b[1]){var c=b[1].split(",");for(b=0;b<c.length;++b)d.push(c[b])}}0<k.length&&0<d.length&&g.push([k,d]);return g}
function splitSizes(a){var d=[];if(null!==a){var e=0<=a.indexOf("|")?a.split("|"):a.split(",");for(a=0;a<e.length;++a){var g=e[a].split("x");var h=parseInt(g[0],10);var b=parseInt(g[1],10);2===g.length&&!isNaN(h)&&!isNaN(b)&&0<=h&&0<=b&&d.push([h,b])}}return d}function debounce(a,d){var e=null,g=null;return function(){var h=this,b=Number(new Date),k=arguments;e&&b<e+d?(clearTimeout(g),g=setTimeout(function(){e=b;a.apply(h,k)},d)):(e=b,a.apply(h,k))}}
function addScrollEvent(a){window.addEventListener?window.addEventListener("scroll",a,!1):window.attachEvent("onscroll",a)}function removeScrollEvent(a){window.removeEventListener?window.removeEventListener("scroll",a,!1):window.detachEvent("scroll",a)}function getAdContainers(){return document.querySelectorAll("div[data-gpt-unitpath]")}function isLoggedIn(){return 0<=document.cookie.indexOf("OSCAR_SESSION_COOKIE")?"logged\x3dy;":"logged\x3dn;"}
(function(a,d){function e(f){var a="test";a=a.replace(/[\[\]]/g,"\\$\x26");a=new RegExp("[?\x26]"+a+"(\x3d([^\x26#]*)|\x26|#|$)");a=(a=a.exec(window.location.href))?a[2]?decodeURIComponent(a[2].replace(/\+/g," ")):"":null;a=a&&"ads"===a?"adtype\x3dtest;":"";f=f.getAttribute("data-gpt-targeting");var c=document.querySelector("[data-ad-targeting-search-terms]")?"search\x3d"+document.querySelector("[data-ad-targeting-search-terms]").innerText+";":!1;var d=isLoggedIn();a&&-1===f.indexOf(a)&&(f+=a);c&&
(f+=c);return f+d+b}function g(a,c){for(var f=a.length,b=[];f--;)c(a[f],f)&&(b.push(a[f].slot),a.splice(f,1));b.length&&googletag.pubads().refresh(b);return a}function h(f){var c=Math.max(document.documentElement.clientHeight,a.innerHeight||0);return g(f,function(a){a=document.getElementById(a.divId);a=a.getBoundingClientRect();a=a.top-300;return c>a})}var b,k=getAdContainers(),c=[];d&&(b=d.dfppKeyValues?d.dfppKeyValues:"");a.googletag=a.googletag||{};a.googletag.cmd=a.googletag.cmd||[];for(var m=
0;k[m];++m){var l=k[m];splitKeys(e(l));c.push({divId:l.getAttribute("id"),adUnitPath:l.getAttribute("data-gpt-unitpath"),sizeArray:splitSizes(l.getAttribute("data-gpt-sizes")),targeting:splitKeys(e(l))})}googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["GTM-TDGJHK"].macro(27)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices()});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());
for(var b=0,d=c[a].targeting.length;b<d;++b)2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]&&c[a].slot.setTargeting(c[a].targeting[b][0],c[a].targeting[b][1])}catch(p){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)});googletag.cmd.push(function(){c=h(c)});var n=debounce(function(){googletag.cmd.push(function(){c=h(c);c.length||removeScrollEvent(n)})},250);addScrollEvent(n)})(window,
window.Krux);</script>

<script type="text/javascript" id="ga-data-track-event-listeners">function addEvent(b){document.addEventListener(b,function(a){a=a.target;var c='[data-track\x3d"'+b+'"]';a=google_tag_manager["GTM-TDGJHK"].macro(35)(a,c);void 0!==a&&dataLayer.push({event:"data-track",eventAction:a.getAttribute("data-track-action"),eventCategory:a.getAttribute("data-track-category"),eventLabel:a.getAttribute("data-track-label")||null,eventValue:a.getAttribute("data-track-value")||null})},!0)}(function(){addEvent("click");addEvent("submit")})();</script><script type="text/javascript" id="crossmark-script" src="widget.js"></script><script type="text/javascript" id="recommended-script" src="entry-point"></script></body><iframe id="kx-proxy-KDqylSLE" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=KDqylSLE&amp;kxt=https%3A%2F%2Fbmcbioinformatics.biomedcentral.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>