<!DOCTYPE html>
<html class="no-js" lang="en"><head><script type="text/javascript" src="300lo.js"></script><script type="text/javascript" src="_ate.js"></script><script type="text/javascript" src="moatframe.js"></script><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">

        <title>Multi-rate Poisson tree processes for single-locus 
species delimitation under maximum likelihood and Markov chain Monte 
Carlo | Bioinformatics | Oxford Academic</title>

        <script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="recaptcha__en.js"></script><script async="" src="gtm.js"></script><script src="jquery.js" type="text/javascript"></script>
<script>window.jQuery || document.write('<script src="//oup.silverchair-cdn.com/UI/app/vendor/jquery-2.2.4.js">\x3C/script>')</script>
<script src="jquery-migrate-1.js"></script>



    
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">

    
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=Edge">

<!-- Bookmark Icons -->
  <link rel="apple-touch-icon" sizes="180x180" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/apple-touch-icon.png">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/favicon-32x32.png" sizes="32x32">
  <link rel="icon" type="image/png" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/favicon-16x16.png" sizes="16x16">
  <link rel="mask-icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842209231/safari-pinned-tab.svg" color="#5bbad5">
  <link rel="shortcut icon" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/favicon.ico">
  <link rel="manifest" href="https://oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/manifest.json">
  <meta name="msapplication-config" content="//oup.silverchair-cdn.com/UI/app/img/v-637171821842052952/browserconfig.xml">
  <meta name="theme-color" content="#002f65">


    




<link rel="stylesheet" type="text/css" href="icons.css">

<link rel="stylesheet" type="text/css" href="style.css">

        <link rel="stylesheet" type="text/css" href="jquery.css">
        <link rel="stylesheet" type="text/css" href="jscrollpane.css">
        <link rel="stylesheet" type="text/css" href="theme.css">

    <link rel="stylesheet" type="text/css" href="css.css">


        <link href="global.css" rel="stylesheet" type="text/css">
                <link href="Site2065275106.css" rel="stylesheet" type="text/css">

        <script>
        (function (w, d, s, l, i) {
          w[l] = w[l] || []; w[l].push({
            'gtm.start':
              new Date().getTime(), event: 'gtm.js'
          }); var f = d.getElementsByTagName(s)[0],
            j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
            'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
        })(window, document, 'script', 'dataLayer', 'GTM-W6DD7HV');
        </script>

    

        <script type="text/javascript">
            var App = App || {};
            App.LoginUserInfo = {
                isInstLoggedIn: [0],
                isIndividualLoggedIn: [0]
            };

            App.CurrentSubdomain = 'bioinformatics';
            App.SiteURL = 'academic.oup.com/bioinformatics';
        </script>
    
    
    <link href="chartist.css" rel="stylesheet" type="text/css">
    <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/bioinformatics/article/33/11/1630/2929345","@type":"ScholarlyArticle","name":"Multi-rate Poisson tree processes for single-locus species delimitation under maximum likelihood and Markov chain Monte Carlo","datePublished":"2017-01-20","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/bioinformatics/issue/33/11","@type":"PublicationIssue","issueNumber":"11","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/bioinformatics/bioinformatics","@type":"Periodical","name":"Bioinformatics","issn":["1460-2059"]}},"url":"http://dx.doi.org/10.1093/bioinformatics/btx025","inLanguage":"en","copyrightHolder":"Oxford University Press","copyrightYear":"2020","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/bioinformatics/article/33/11/1630/2929345","author":[{"name":"Kapli, P","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany","@type":"Person"},{"name":"Lutteropp, S","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany  Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany","@type":"Person"},{"name":"Zhang, J","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany","@type":"Person"},{"name":"Kobert, K","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany","@type":"Person"},{"name":"Pavlidis, P","affiliation":"Foundation for Research and Technology, Hellas Institute of Computer Science GR, Heraklion, Crete, Greece","@type":"Person"},{"name":"Stamatakis, A","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany  Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany","@type":"Person"},{"name":"Flouri, T","affiliation":"The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany  Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany","@type":"Person"}],"description":"AbstractMotivation.  In recent years, molecular species delimitation has become a routine approach for quantifying and classifying biodiversity. Barcoding metho","pageStart":"1630","pageEnd":"1638","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/33/11/1/m_bioinfo_33_11cover.png?Expires=1645876432&Signature=dCQe1DRKWVCh5U6UeCI2R41kHFqzD0v29TiFTmCtQ2nSQyJPgwQGlgdRMr5y~kJIzQFeQN4jxRHpkOixKc0Oq2H1iOMXYcxs6o3RghRmAhQPiPNjZWK4xhEFmC-N-EorHPD~6NxhV48~Z-6VUxphdOI9ieBt3eCuzQe900NKhgk04CsdmvQfBOOtP1J0KVY9PxOjKp1Dz0MkYF0~-~tD9No549c0vLxxI53v-ghk9ccxC0tEV1bcX7m74zCUduytUFsxsaWJZui12SX6WVJEfYDgclPvqkWrDVp5ScxGdHUkczFNMkSDfCImnQ9lbHukEollLG3ayhVQa5pjCkvgjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Multi-rate Poisson tree processes for single-locus species delimitation under maximum likelihood and Markov chain Monte Carlo","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/33/11/1/m_bioinfo_33_11cover.png?Expires=1645876432&Signature=dCQe1DRKWVCh5U6UeCI2R41kHFqzD0v29TiFTmCtQ2nSQyJPgwQGlgdRMr5y~kJIzQFeQN4jxRHpkOixKc0Oq2H1iOMXYcxs6o3RghRmAhQPiPNjZWK4xhEFmC-N-EorHPD~6NxhV48~Z-6VUxphdOI9ieBt3eCuzQe900NKhgk04CsdmvQfBOOtP1J0KVY9PxOjKp1Dz0MkYF0~-~tD9No549c0vLxxI53v-ghk9ccxC0tEV1bcX7m74zCUduytUFsxsaWJZui12SX6WVJEfYDgclPvqkWrDVp5ScxGdHUkczFNMkSDfCImnQ9lbHukEollLG3ayhVQa5pjCkvgjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:site_name" content="OUP Academic">
    <meta property="og:type" content="article">
    <meta property="og:title" content="Multi-rate Poisson tree processes for single-locus species delimitation under maximum likelihood and Markov chain Monte Carlo">
    <meta property="og:description" content="AbstractMotivation.  In recent years, molecular species delimitation has become a routine approach for quantifying and classifying biodiversity. Barcoding metho">
    <meta property="og:url" content="https://academic.oup.com/bioinformatics/article/33/11/1630/2929345">
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/Issue/33/11/1/m_bioinfo_33_11cover.png?Expires=1645876432&amp;Signature=dCQe1DRKWVCh5U6UeCI2R41kHFqzD0v29TiFTmCtQ2nSQyJPgwQGlgdRMr5y~kJIzQFeQN4jxRHpkOixKc0Oq2H1iOMXYcxs6o3RghRmAhQPiPNjZWK4xhEFmC-N-EorHPD~6NxhV48~Z-6VUxphdOI9ieBt3eCuzQe900NKhgk04CsdmvQfBOOtP1J0KVY9PxOjKp1Dz0MkYF0~-~tD9No549c0vLxxI53v-ghk9ccxC0tEV1bcX7m74zCUduytUFsxsaWJZui12SX6WVJEfYDgclPvqkWrDVp5ScxGdHUkczFNMkSDfCImnQ9lbHukEollLG3ayhVQa5pjCkvgjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">
    <meta property="og:updated_time" content="2017-01-20">
<meta name="citation_author" content="Kapli, P"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Lutteropp, S"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author_institution" content="Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany"><meta name="citation_author" content="Zhang, J"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Kobert, K"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author" content="Pavlidis, P"><meta name="citation_author_institution" content="Foundation for Research and Technology, Hellas Institute of Computer Science GR, Heraklion, Crete, Greece"><meta name="citation_author" content="Stamatakis, A"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author_institution" content="Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany"><meta name="citation_author" content="Flouri, T"><meta name="citation_author_institution" content="The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany"><meta name="citation_author_institution" content="Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany"><meta name="citation_title" content="Multi-rate Poisson tree processes for single-locus species delimitation under maximum likelihood and Markov chain Monte Carlo"><meta name="citation_firstpage" content="1630"><meta name="citation_lastpage" content="1638"><meta name="citation_doi" content="10.1093/bioinformatics/btx025"><meta name="citation_journal_title" content="Bioinformatics"><meta name="citation_journal_abbrev" content="Bioinformatics"><meta name="citation_volume" content="33"><meta name="citation_issue" content="11"><meta name="citation_publication_date" content="2017/06/01"><meta name="citation_issn" content="1367-4803"><meta name="citation_publisher" content="Oxford Academic"><meta name="citation_reference" content="citation_title=A Bayesian analysis of the minimum AIC procedure;  citation_journal_title=Ann. Inst. Stat. Math;  citation_year=1978;  citation_volume=30;  citation_pages=9-14; "><meta name="citation_reference" content="citation_title=Incorporating historical museum specimens into molecular systematic and conservation genetics research;  citation_journal_title=Mol. Ecol. Notes;  citation_year=2006;  citation_volume=6;  citation_pages=1089-1092; "><meta name="citation_reference" content="citation_title=Availability of new Bayesian-delimited gecko names and the importance of character-based species descriptions;  citation_journal_title=Proc. R. Soc. Lond. B: Biol. Sci;  citation_year=2010;  citation_volume=278;  citation_pages=490-492; "><meta name="citation_reference" content="citation_title=Population size does not influence mitochondrial genetic diversity in animals;  citation_journal_title=Science;  citation_year=2006;  citation_volume=312;  citation_pages=570-572; "><meta name="citation_reference" content="citation_title=Molecular systematics: counting angels with DNA;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=421;  citation_pages=122-124; "><meta name="citation_reference" content="citation_title=Molecular taxonomics for biodiversity surveys: already a reality;  citation_journal_title=Trends Ecol. Evol;  citation_year=2003;  citation_volume=18;  citation_pages=268-269; "><meta name="citation_reference" content="citation_title=Defining operational taxonomic units using DNA barcode data;  citation_journal_title=Phil. Trans. R. Soc. B: Biol. Sci;  citation_year=2005;  citation_volume=360;  citation_pages=1935-1943; "><meta name="citation_reference" content="citation_publisher=Springer, New York;  citation_title=Model Selection and MultiModel Inference: A Practical Information-Theoretic Approach;  citation_year=2002; "><meta name="citation_reference" content="citation_title=DNA barcoding: a six-question tour to improve users’ awareness about the method;  citation_journal_title=Brief. Bioinform;  citation_year=2010;  citation_volume=11;  citation_pages=440-453; "><meta name="citation_reference" content="citation_title=Towards integrative taxonomy;  citation_journal_title=Biol. J. Linnean Soc;  citation_year=2005;  citation_volume=85;  citation_pages=407-415; "><meta name="citation_reference" content="citation_title=Species concepts and species delimitation;  citation_journal_title=Syst. Biol;  citation_year=2007;  citation_volume=56;  citation_pages=879-886; "><meta name="citation_reference" content="citation_title=Multi-locus species delimitation in closely related animals and fungi: one marker is not enough;  citation_journal_title=Mol. Ecol;  citation_year=2012;  citation_volume=21;  citation_pages=4422-4436; "><meta name="citation_reference" content="citation_title=Search and clustering orders of magnitude faster than BLAST;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2460-2461; "><meta name="citation_reference" content="citation_title=Single-locus species delimitation: a test of the mixed Yule–coalescent model, with an empirical application to Philippine round-leaf bats;  citation_journal_title=Proc. R. Soc. Lond. B: Biol. Sci;  citation_year=2012; "><meta name="citation_reference" content="citation_title=Delimiting species using single-locus data and the generalized mixed Yule coalescent approach: a revised method and evaluation on simulated data sets;  citation_journal_title=Syst. Biol;  citation_year=2013;  citation_volume=62;  citation_pages=707-724; "><meta name="citation_reference" content="citation_title=A rapid and scalable method for multilocus species delimitation using Bayesian model comparison and rooted triplets;  citation_journal_title=Syst. Biol;  citation_year=2016;  citation_volume=65;  citation_pages=759-771; "><meta name="citation_reference" content="citation_title=Species-level paraphyly and polyphyly: frequency, causes, and consequences, with insights from animal mitochondrial DNA;  citation_journal_title=Annu. Rev. Ecol. Evol. Syst;  citation_year=2003;  citation_volume=34;  citation_pages=397-423; "><meta name="citation_reference" content="citation_title=Clustering 16S rRNA for OTU prediction: a method of unsupervised Bayesian clustering;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=611-618; "><meta name="citation_reference" content="citation_title=The phylogeny of Anophelinae revisited: inferences about the origin and classification of Anopheles (Diptera: Culicidae);  citation_journal_title=Zool. Scr;  citation_year=2016;  citation_volume=45;  citation_pages=34-47; "><meta name="citation_reference" content="citation_title=Biological identifications through DNA barcodes;  citation_journal_title=Proc. R. Soc. Lond. B: Biol. Sci;  citation_year=2003;  citation_volume=270;  citation_pages=313-321; "><meta name="citation_reference" content="citation_title=Identification of birds through DNA barcodes;  citation_journal_title=PLoS Biol;  citation_year=2004;  citation_volume=2;  citation_pages=1657-1663; "><meta name="citation_reference" content="citation_title=Gene genealogies and the coalescent process;  citation_journal_title=Oxford Surv. Evol. Biol;  citation_year=1990;  citation_volume=7;  citation_pages=44."><meta name="citation_reference" content="citation_title=DISSECT: an assignment-free Bayesian discovery method for species delimitation under the multispecies coalescent;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=991-998; "><meta name="citation_reference" content="citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=772-780; "><meta name="citation_reference" content="citation_title=Exploring Genetic Divergence in a Species-Rich Insect Genus Using 2790 DNA Barcodes;  citation_journal_title=PLoS ONE;  citation_year=2015;  citation_volume=10;  citation_pages=1-24; "><meta name="citation_reference" content="citation_title=Gene trees in species trees;  citation_journal_title=Syst. Biol;  citation_year=1997;  citation_volume=46;  citation_pages=523-536; "><meta name="citation_reference" content="citation_title=The causes of mitochondrial DNA gene tree paraphyly in birds;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2010;  citation_volume=54;  citation_pages=647-650; "><meta name="citation_reference" content="citation_publisher=World Scientific, Singapore;  citation_title=Regression and Time Series Model Selection;  citation_year=1998; "><meta name="citation_reference" content="citation_title=Accelerated species inventory on Madagascar using coalescent-based models of species delineation;  citation_journal_title=Syst. Biol;  citation_year=2009;  citation_volume=58;  citation_pages=298-311; "><meta name="citation_reference" content="citation_title=Estimating divergence dates and substitution rates in the Drosophila phylogeny;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2012; "><meta name="citation_reference" content="citation_title=A possible method for the rapid assessment of biodiversity;  citation_journal_title=Conserv. Biol;  citation_year=1993;  citation_volume=7;  citation_pages=562-568; "><meta name="citation_reference" content="citation_title=Sequence-based species delimitation for the DNA taxonomy of undescribed insects;  citation_journal_title=Syst. Biol;  citation_year=2006;  citation_volume=55;  citation_pages=595-609; "><meta name="citation_reference" content="citation_title=ABGD, Automatic Barcode Gap Discovery for primary species delimitation;  citation_journal_title=Mol. Ecol;  citation_year=2012;  citation_volume=21;  citation_pages=1864-1877; "><meta name="citation_reference" content="citation_title=Mitochondrial phylogenomics and genetic relationships of closely related pine moth (Lasiocampidae: Dendrolimus) species in China, using whole mitochondrial genomes;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=1."><meta name="citation_reference" content="citation_title=A DNA-based registry for all animal species: The Barcode Index Number (BIN) System;  citation_journal_title=PloS One;  citation_year=2013;  citation_volume=8;  citation_pages=e66213."><meta name="citation_reference" content="citation_publisher=Butterworth-Heinemann, Newton, MA, USA;  citation_title=Information Retrieval;  citation_year=1979; "><meta name="citation_reference" content="citation_title=MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space;  citation_journal_title=Systematic Biology;  citation_year=2012;  citation_volume=61;  citation_pages=539-542; "><meta name="citation_reference" content="citation_title=DNA barcoding for effective biodiversity assessment of a hyperdiverse arthropod group: the ants of Madagascar;  citation_journal_title=Philos. Trans. R. Soc. B: Biol. Sci;  citation_year=2005;  citation_volume=360;  citation_pages=1825-1834; "><meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1312-1313; "><meta name="citation_reference" content="citation_publisher=Center for Evolutionary Functional Genomics: The Biodesign Institute, Tempe, AZ;  citation_title=MEGA 5: Molecular Evolutionary Genetics Analysis;  citation_year=2011;  citation_pages=85287-85301; "><meta name="citation_reference" content="citation_title=Effects of phylogenetic reconstruction method on the robustness of species delimitation using single-locus data;  citation_journal_title=Methods Ecol. Evol;  citation_year=2014;  citation_volume=5;  citation_pages=1086-1094; "><meta name="citation_reference" content="citation_title=A plea for DNA taxonomy;  citation_journal_title=Trends Ecol. Evol;  citation_year=2003;  citation_volume=18;  citation_pages=70-74; "><meta name="citation_reference" content="citation_title=DNA barcoding for ecologists;  citation_journal_title=Trends Ecol. Evol;  citation_year=2009;  citation_volume=24;  citation_pages=110-117; "><meta name="citation_reference" content="citation_title=Recent advances in DNA taxonomy;  citation_journal_title=J. Zool. Syst. Evol. Res;  citation_year=2007;  citation_volume=45;  citation_pages=1-10; "><meta name="citation_reference" content="citation_title=The perils of DNA barcoding and the need for integrative taxonomy;  citation_journal_title=Syst. Biol;  citation_year=2005;  citation_volume=54;  citation_pages=844-851; "><meta name="citation_reference" content="citation_title=Unguided species delimitation using DNA sequence data from multiple loci;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=3125-3135; "><meta name="citation_reference" content="citation_title=A mathematical theory of evolution, based on the conclusions of Dr. J. C. Willis, F.R.S;  citation_journal_title=Philos. Trans. R. Soc. Lond.. Ser. B: Containing Papers of a Biological Character;  citation_year=1925;  citation_volume=213;  citation_pages=21-87; "><meta name="citation_reference" content="citation_title=A general species delimitation method with applications to phylogenetic placements;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=2869-2876; "><meta name="citation_fulltext_world_readable" content=""><meta name="citation_pdf_url" content="https://academic.oup.com/bioinformatics/article-pdf/33/11/1630/25153573/btx025.pdf"><meta name="description" content="AbstractMotivation.  In recent years, molecular species delimitation has become a routine approach for quantifying and classifying biodiversity. Barcoding metho">    <link rel="preload" href="integrator.js" as="script"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2020022001.js" async=""></script><script id="altmetric-embed-js" src="altmetric_badges-75bc9437b4bcd96622a3f013e4e9519d1b65ea847ab6.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MathJax_Preview .MJXf-math {color: inherit!important}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style></style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style><style type="text/css">.at-icon{fill:#fff;border:0}.at-icon-wrapper{display:inline-block;overflow:hidden}a .at-icon-wrapper{cursor:pointer}.at-rounded,.at-rounded-element .at-icon-wrapper{border-radius:12%}.at-circular,.at-circular-element .at-icon-wrapper{border-radius:50%}.addthis_32x32_style .at-icon{width:2pc;height:2pc}.addthis_24x24_style .at-icon{width:24px;height:24px}.addthis_20x20_style .at-icon{width:20px;height:20px}.addthis_16x16_style .at-icon{width:1pc;height:1pc}#at16lb{display:none;position:absolute;top:0;left:0;width:100%;height:100%;z-index:1001;background-color:#000;opacity:.001}#at_complete,#at_error,#at_share,#at_success{position:static!important}.at15dn{display:none}#at15s,#at16p,#at16p form input,#at16p label,#at16p textarea,#at_share .at_item{font-family:arial,helvetica,tahoma,verdana,sans-serif!important;font-size:9pt!important;outline-style:none;outline-width:0;line-height:1em}* html #at15s.mmborder{position:absolute!important}#at15s.mmborder{position:fixed!important;width:250px!important}#at15s{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);float:none;line-height:1em;margin:0;overflow:visible;padding:5px;text-align:left;position:absolute}#at15s a,#at15s span{outline:0;direction:ltr;text-transform:none}#at15s .at-label{margin-left:5px}#at15s .at-icon-wrapper{width:1pc;height:1pc;vertical-align:middle}#at15s .at-icon{width:1pc;height:1pc}.at4-icon{display:inline-block;background-repeat:no-repeat;background-position:top left;margin:0;overflow:hidden;cursor:pointer}.addthis_16x16_style .at4-icon,.addthis_default_style .at4-icon,.at4-icon,.at-16x16{width:1pc;height:1pc;line-height:1pc;background-size:1pc!important}.addthis_32x32_style .at4-icon,.at-32x32{width:2pc;height:2pc;line-height:2pc;background-size:2pc!important}.addthis_24x24_style .at4-icon,.at-24x24{width:24px;height:24px;line-height:24px;background-size:24px!important}.addthis_20x20_style .at4-icon,.at-20x20{width:20px;height:20px;line-height:20px;background-size:20px!important}.at4-icon.circular,.circular .at4-icon,.circular.aticon{border-radius:50%}.at4-icon.rounded,.rounded .at4-icon{border-radius:4px}.at4-icon-left{float:left}#at15s .at4-icon{text-indent:20px;padding:0;overflow:visible;white-space:nowrap;background-size:1pc;width:1pc;height:1pc;background-position:top left;display:inline-block;line-height:1pc}.addthis_vertical_style .at4-icon,.at4-follow-container .at4-icon{margin-right:5px}html>body #at15s{width:250px!important}#at15s.atm{background:none!important;padding:0!important;width:10pc!important}#at15s_inner{background:#fff;border:1px solid #fff;margin:0}#at15s_head{position:relative;background:#f2f2f2;padding:4px;cursor:default;border-bottom:1px solid #e5e5e5}.at15s_head_success{background:#cafd99!important;border-bottom:1px solid #a9d582!important}.at15s_head_success a,.at15s_head_success span{color:#000!important;text-decoration:none}#at15s_brand,#at15sptx,#at16_brand{position:absolute}#at15s_brand{top:4px;right:4px}.at15s_brandx{right:20px!important}a#at15sptx{top:4px;right:4px;text-decoration:none;color:#4c4c4c;font-weight:700}#at15sptx:hover{text-decoration:underline}#at16_brand{top:5px;right:30px;cursor:default}#at_hover{padding:4px}#at_hover .at_item,#at_share .at_item{background:#fff!important;float:left!important;color:#4c4c4c!important}#at_share .at_item .at-icon-wrapper{margin-right:5px}#at_hover .at_bold{font-weight:700;color:#000!important}#at_hover .at_item{width:7pc!important;padding:2px 3px!important;margin:1px;text-decoration:none!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover{margin:0!important}#at_hover .at_item.athov,#at_hover .at_item:focus,#at_hover .at_item:hover,#at_share .at_item.athov,#at_share .at_item:hover{background:#f2f2f2!important;border:1px solid #e5e5e5;color:#000!important;text-decoration:none}.ipad #at_hover .at_item:focus{background:#fff!important;border:1px solid #fff}.at15t{display:block!important;height:1pc!important;line-height:1pc!important;padding-left:20px!important;background-position:0 0;text-align:left}.addthis_button,.at15t{cursor:pointer}.addthis_toolbox a.at300b,.addthis_toolbox a.at300m{width:auto}.addthis_toolbox a{margin-bottom:5px;line-height:initial}.addthis_toolbox.addthis_vertical_style{width:200px}.addthis_button_facebook_like .fb_iframe_widget{line-height:100%}.addthis_button_facebook_like iframe.fb_iframe_widget_lift{max-width:none}.addthis_toolbox a.addthis_button_counter,.addthis_toolbox a.addthis_button_facebook_like,.addthis_toolbox a.addthis_button_facebook_send,.addthis_toolbox a.addthis_button_facebook_share,.addthis_toolbox a.addthis_button_foursquare,.addthis_toolbox a.addthis_button_linkedin_counter,.addthis_toolbox a.addthis_button_pinterest_pinit,.addthis_toolbox a.addthis_button_tweet{display:inline-block}.addthis_toolbox span.addthis_follow_label{display:none}.addthis_toolbox.addthis_vertical_style span.addthis_follow_label{display:block;white-space:nowrap}.addthis_toolbox.addthis_vertical_style a{display:block}.addthis_toolbox.addthis_vertical_style.addthis_32x32_style a{line-height:2pc;height:2pc}.addthis_toolbox.addthis_vertical_style .at300bs{margin-right:4px;float:left}.addthis_toolbox.addthis_20x20_style span{line-height:20px}.addthis_toolbox.addthis_32x32_style span{line-height:2pc}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact,.addthis_toolbox.addthis_pill_combo_style a{float:left}.addthis_toolbox.addthis_pill_combo_style a.addthis_button_tweet{margin-top:-2px}.addthis_toolbox.addthis_pill_combo_style .addthis_button_compact .at15t_compact{margin-right:4px}.addthis_default_style .addthis_separator{margin:0 5px;display:inline}div.atclear{clear:both}.addthis_default_style .addthis_separator,.addthis_default_style .at4-icon,.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300bs,.addthis_default_style .at300m{float:left}.at300b img,.at300bo img{border:0}a.at300b .at4-icon,a.at300m .at4-icon{display:block}.addthis_default_style .at300b,.addthis_default_style .at300bo,.addthis_default_style .at300m{padding:0 2px}.at300b,.at300bo,.at300bs,.at300m{cursor:pointer}.addthis_button_facebook_like.at300b:hover,.addthis_button_facebook_like.at300bs:hover,.addthis_button_facebook_send.at300b:hover,.addthis_button_facebook_send.at300bs:hover{opacity:1}.addthis_20x20_style .at15t,.addthis_20x20_style .at300bs{overflow:hidden;display:block;height:20px!important;width:20px!important;line-height:20px!important}.addthis_32x32_style .at15t,.addthis_32x32_style .at300bs{overflow:hidden;display:block;height:2pc!important;width:2pc!important;line-height:2pc!important}.at300bs{overflow:hidden;display:block;background-position:0 0;height:1pc;width:1pc;line-height:1pc!important}.addthis_default_style .at15t_compact,.addthis_default_style .at15t_expanded{margin-right:4px}#at_share .at_item{width:123px!important;padding:4px;margin-right:2px;border:1px solid #fff}#at16p{background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);z-index:10000001;position:absolute;top:50%;left:50%;width:300px;padding:10px;margin:0 auto;margin-top:-185px;margin-left:-155px;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#5e5e5e}#at_share{margin:0;padding:0}#at16pt{position:relative;background:#f2f2f2;height:13px;padding:5px 10px}#at16pt a,#at16pt h4{font-weight:700}#at16pt h4{display:inline;margin:0;padding:0;font-size:9pt;color:#4c4c4c;cursor:default}#at16pt a{position:absolute;top:5px;right:10px;color:#4c4c4c;text-decoration:none;padding:2px}#at15sptx:focus,#at16pt a:focus{outline:thin dotted}#at15s #at16pf a{top:1px}#_atssh{width:1px!important;height:1px!important;border:0!important}.atm{width:10pc!important;padding:0;margin:0;line-height:9pt;letter-spacing:normal;font-family:arial,helvetica,tahoma,verdana,sans-serif;font-size:9pt;color:#444;background:url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABtJREFUeNpiZGBgaGAgAjAxEAlGFVJHIUCAAQDcngCUgqGMqwAAAABJRU5ErkJggg==);padding:4px}.atm-f{text-align:right;border-top:1px solid #ddd;padding:5px 8px}.atm-i{background:#fff;border:1px solid #d5d6d6;padding:0;margin:0;box-shadow:1px 1px 5px rgba(0,0,0,.15)}.atm-s{margin:0!important;padding:0!important}.atm-s a:focus{border:transparent;outline:0;transition:none}#at_hover.atm-s a,.atm-s a{display:block;text-decoration:none;padding:4px 10px;color:#235dab!important;font-weight:400;font-style:normal;transition:none}#at_hover.atm-s .at_bold{color:#235dab!important}#at_hover.atm-s a:hover,.atm-s a:hover{background:#2095f0;text-decoration:none;color:#fff!important}#at_hover.atm-s .at_bold{font-weight:700}#at_hover.atm-s a:hover .at_bold{color:#fff!important}.atm-s a .at-label{vertical-align:middle;margin-left:5px;direction:ltr}.at_PinItButton{display:block;width:40px;height:20px;padding:0;margin:0;background-image:url(//s7.addthis.com/static/t00/pinit00.png);background-repeat:no-repeat}.at_PinItButton:hover{background-position:0 -20px}.addthis_toolbox .addthis_button_pinterest_pinit{position:relative}.at-share-tbx-element .fb_iframe_widget span{vertical-align:baseline!important}#at16pf{height:auto;text-align:right;padding:4px 8px}.at-privacy-info{position:absolute;left:7px;bottom:7px;cursor:pointer;text-decoration:none;font-family:helvetica,arial,sans-serif;font-size:10px;line-height:9pt;letter-spacing:.2px;color:#666}.at-privacy-info:hover{color:#000}.body .wsb-social-share .wsb-social-share-button-vert{padding-top:0;padding-bottom:0}.body .wsb-social-share.addthis_counter_style .addthis_button_tweet.wsb-social-share-button{padding-top:40px}.body .wsb-social-share.addthis_counter_style .addthis_button_facebook_like.wsb-social-share-button{padding-top:21px}@media print{#at4-follow,#at4-share,#at4-thankyou,#at4-whatsnext,#at4m-mobile,#at15s,.at4,.at4-recommended{display:none!important}}@media screen and (max-width:400px){.at4win{width:100%}}@media screen and (max-height:700px) and (max-width:400px){.at4-thankyou-inner .at4-recommended-container{height:122px;overflow:hidden}.at4-thankyou-inner .at4-recommended .at4-recommended-item:first-child{border-bottom:1px solid #c5c5c5}}</style><style type="text/css">.at-branding-logo{font-family:helvetica,arial,sans-serif;text-decoration:none;font-size:10px;display:inline-block;margin:2px 0;letter-spacing:.2px}.at-branding-logo .at-branding-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAMAAAC67D+PAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRF////+GlNUkcc1QAAAB1JREFUeNpiYIQDBjQmAwMmkwEM0JnY1WIxFyDAABGeAFEudiZsAAAAAElFTkSuQmCC")}.at-branding-logo .at-branding-icon,.at-branding-logo .at-privacy-icon{display:inline-block;height:10px;width:10px;margin-left:4px;margin-right:3px;margin-bottom:-1px;background-repeat:no-repeat}.at-branding-logo .at-privacy-icon{background-image:url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAKCAMAAABR24SMAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABhQTFRF8fr9ot/xXcfn2/P5AKva////////AKTWodjhjAAAAAd0Uk5T////////ABpLA0YAAAA6SURBVHjaJMzBDQAwCAJAQaj7b9xifV0kUKJ9ciWxlzWEWI5gMF65KUTv0VKkjVeTerqE/x7+9BVgAEXbAWI8QDcfAAAAAElFTkSuQmCC")}.at-branding-logo span{text-decoration:none}.at-branding-logo .at-branding-addthis,.at-branding-logo .at-branding-powered-by{color:#666}.at-branding-logo .at-branding-addthis:hover{color:#333}.at-cv-with-image .at-branding-addthis,.at-cv-with-image .at-branding-addthis:hover{color:#fff}a.at-branding-logo:visited{color:initial}.at-branding-info{display:inline-block;padding:0 5px;color:#666;border:1px solid #666;border-radius:50%;font-size:10px;line-height:9pt;opacity:.7;transition:all .3s ease;text-decoration:none}.at-branding-info span{border:0;clip:rect(0 0 0 0);height:1px;margin:-1px;overflow:hidden;padding:0;position:absolute;width:1px}.at-branding-info:before{content:'i';font-family:Times New Roman}.at-branding-info:hover{color:#0780df;border-color:#0780df}</style><link id="altmetric-embed-css" rel="stylesheet" type="text/css" href="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed-a458e1093bac38cf5adc51eff02159de22f9e260b3636d0ecefba2667a43520a.css"><style type="text/css">.MathJax_Display {text-align: center; margin: 1em 0em; position: relative; display: block!important; text-indent: 0; max-width: none; max-height: none; min-width: 0; min-height: 0; width: 100%}
.MathJax .merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MathJax .MJX-monospace {font-family: monospace}
.MathJax .MJX-sans-serif {font-family: sans-serif}
#MathJax_Tooltip {background-color: InfoBackground; color: InfoText; border: 1px solid black; box-shadow: 2px 2px 5px #AAAAAA; -webkit-box-shadow: 2px 2px 5px #AAAAAA; -moz-box-shadow: 2px 2px 5px #AAAAAA; -khtml-box-shadow: 2px 2px 5px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true'); padding: 3px 4px; z-index: 401; position: absolute; left: 0; top: 0; width: auto; height: auto; display: none}
.MathJax {display: inline; font-style: normal; font-weight: normal; line-height: normal; font-size: 100%; font-size-adjust: none; text-indent: 0; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0; min-height: 0; border: 0; padding: 0; margin: 0}
.MathJax:focus, body :focus .MathJax {display: inline-table}
.MathJax.MathJax_FullWidth {text-align: center; display: table-cell!important; width: 10000em!important}
.MathJax img, .MathJax nobr, .MathJax a {border: 0; padding: 0; margin: 0; max-width: 5000em; max-height: 5000em; min-width: 0; min-height: 0; vertical-align: 0; line-height: normal; text-decoration: none}
img.MathJax_strut {border: 0!important; padding: 0!important; margin: 0!important; vertical-align: 0!important}
.MathJax span {display: inline; position: static; border: 0; padding: 0; margin: 0; vertical-align: 0; line-height: normal; text-decoration: none}
.MathJax nobr {white-space: nowrap!important}
.MathJax img {display: inline!important; float: none!important}
.MathJax * {transition: none; -webkit-transition: none; -moz-transition: none; -ms-transition: none; -o-transition: none}
.MathJax_Processing {visibility: hidden; position: fixed; width: 0; height: 0; overflow: hidden}
.MathJax_Processed {display: none!important}
.MathJax_ExBox {display: block!important; overflow: hidden; width: 1px; height: 60ex; min-height: 0; max-height: none}
.MathJax .MathJax_EmBox {display: block!important; overflow: hidden; width: 1px; height: 60em; min-height: 0; max-height: none}
.MathJax_LineBox {display: table!important}
.MathJax_LineBox span {display: table-cell!important; width: 10000em!important; min-width: 0; max-width: none; padding: 0; border: 0; margin: 0}
.MathJax .MathJax_HitBox {cursor: text; background: white; opacity: 0; filter: alpha(opacity=0)}
.MathJax .MathJax_HitBox * {filter: none; opacity: 1; background: transparent}
#MathJax_Tooltip * {filter: none; opacity: 1; background: transparent}
@font-face {font-family: MathJax_Blank; src: url('about:blank')}
.MathJax .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head><body data-sitestyletemplate="Journal" theme-bioinformatics="" class="off-canvas pg_Article pg_article" data-sitename="bioinformatics"><div style="visibility: hidden; overflow: hidden; position: absolute; top: 0px; height: 1px; width: auto; padding: 0px; border: 0px none; margin: 0px; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal;"><div id="MathJax_Hidden"></div></div><div id="MathJax_Message" style="display: none;"></div><div class="widget widget-Hypothesis widget-instance-OUP_Article_Hypothesis">
        <div id="hypothesis-container">
</div> 
    </div>
<meta name="citation_xml_url" content="https://academic.oup.com/bioinformatics/article-xml/33/11/1630/2929345">



    <link rel="canonical" href="https://academic.oup.com/bioinformatics/article/33/11/1630/2929345">




    
    


<script async="async" src="gpt.js"></script>
<script>
    var googletag = googletag || {};
    googletag.cmd = googletag.cmd || [];
</script>
    
        <script type="text/javascript">
                        var gptAdSlots = [];
                    googletag.cmd.push(function() {
            
            var mapping_ad1 = googletag.sizeMapping()
            .addSize([1024, 0], [[970, 90], [728, 90]])
            .addSize([768, 0], [728, 90])
            .addSize([0, 0], [320, 50])
            .build();
            gptAdSlots["ad1"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad1',
                [[970, 90], [728, 90], [320, 50]],
                'adBlockHeader')
            .defineSizeMapping(mapping_ad1)
            .addService(googletag.pubads());
            

            
            var mapping_ad2 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
        gptAdSlots["ad2"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad2',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyTop')
            .defineSizeMapping(mapping_ad2)
            .addService(googletag.pubads());
            

            
            var mapping_ad3 = googletag.sizeMapping()
            .addSize([0, 0], [])
            .addSize([768, 0], [[300, 250], [300, 600], [160, 600]])
            .build();
    gptAdSlots["ad3"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad3',
                [[300, 250], [160, 600], [300, 600]],
                'adBlockMainBodyBottom')
            .defineSizeMapping(mapping_ad3)
            .addService(googletag.pubads());
            

            
                var mapping_ad4 = googletag.sizeMapping()
                .addSize([0, 0], [])
                .addSize([768, 0], [728, 90])
                .build();
    gptAdSlots["ad4"] = googletag.defineSlot('/116097782/bioinformatics_Supplement_Ad4', [728, 90], 'adBlockFooter')
                .defineSizeMapping(mapping_ad4)
                .addService(googletag.pubads());
            
            googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "bioinformatics/article/33/11/1630/2929345");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
        });
        </script>
    


    


    



    



    


    <script>
        var oupcookiepolicy_siteid = "journals";        // the website id
        var oupcookiepolicy_messagetype = 'implied';                        // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";      // preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';    // the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website.By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/"> Find out more</a></noscript>
    <script type="text/javascript">
        var NTPT_PGEXTRA= 'event_type=full-text&supplier_tag=SC_Journals&object_type=Article&siteid=bioinformatics&authzrequired=false&doi=10.1093/bioinformatics/btx025';
    </script>









    
        <noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"
                    height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
        <a href="#main" class="skipnav">Skip to Main Content</a>
<input id="hdnSiteID" name="hdnSiteID" value="5139" type="hidden"><input id="hdnAdDelaySeconds" name="hdnAdDelaySeconds" value="5000" type="hidden"><input id="hdnAdConfigurationTop" name="hdnAdConfigurationTop" value="default" type="hidden"><input id="hdnAdConfigurationRightRail" name="hdnAdConfigurationRightRail" value="sticky" type="hidden">
    






<div class="master-container js-master-container">
    <section class="master-header row js-master-header vt-site-page-header">
    <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad1_0__container__" style="border: 0pt none; width: 970px; height: 90px;"></div></div>
 
    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">
            <div class="oup-header-logo">
                <a href="https://academic.oup.com/">
                    <img src="oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image">
                </a>
            </div>

            <ul class="oup-header-menu account-menu">

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search Menu</span></i>
                    </a>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </a>
                </li>

                
                <li class="oup-header-menu-item mobile">
                    <a href="javascript:;" class="mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </a>
                </li>

                
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div>
                </li>

                
                    <li class="oup-header-menu-item desktop"><a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f33%2f11%2f1630%2f2929345" class="register">Register</a></li>
                
            </ul>
        </div>
    </div>
<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <a href="javascript:;" id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f33%2f11%2f1630%2f2929345" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387512">
                <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link ">Issues</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387513">
                <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link ">Advance articles</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387510">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">Submit</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387514">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387515">
                <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link ">Submission Site</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387516">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Open Access</a>
                
            </li>
    </ul>

            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387517">
                <a href="https://academic.oup.com/bioinformatics/subscribe" class="nav-link ">Purchase</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387518">
                <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387511">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">About</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387519">
                <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link ">About Bioinformatics</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387520">
                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387521">
                <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387522">
                <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387523">
                <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387524">
                <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
                
            </li>
    </ul>

            </li>
    </ul>
    </div>

</div>


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                            <a href="https://academic.oup.com/bioinformatics" class="journal-logo-container">
                    <img id="logo-Bioinformatics" class="journal-logo" src="bioinformatics_title-205396838.svg" alt="Bioinformatics">
                </a>

            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.iscb.org/" target="" class="society-logo-container">
                                <img id="logo-InternationalSocietyforComputationalBiology" class="society-logo" src="bioinformatics_h11820233911.svg" alt="International Society for Computational Biology">
                            </a>
                                            </div>            </div> 
        </div>
    </div>
<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387512">
                <a href="https://academic.oup.com/bioinformatics/issue" class="nav-link ">Issues</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387513">
                <a href="https://academic.oup.com/bioinformatics/advance-articles" class="nav-link ">Advance articles</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387510">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">Submit</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387514">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Author Guidelines</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387515">
                <a href="http://mc.manuscriptcentral.com/bioinformatics" class="nav-link ">Submission Site</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387516">
                <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions" class="nav-link ">Open Access</a>
                
            </li>
    </ul>

            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387517">
                <a href="https://academic.oup.com/bioinformatics/subscribe" class="nav-link ">Purchase</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387518">
                <a href="https://academic.oup.com/my-account/email-alerts" class="nav-link ">Alerts</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1387511">
                <a href="https://academic.oup.com/bioinformatics/#" class="nav-link dummy-link">About</a>
                    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387519">
                <a href="https://academic.oup.com/bioinformatics/pages/About" class="nav-link ">About Bioinformatics</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387520">
                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=" class="nav-link ">Journals Career Network</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387521">
                <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387522">
                <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387523">
                <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_b" class="nav-link ">Self-Archiving Policy</a>
                
            </li>
            <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1387524">
                <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
                
            </li>
    </ul>

            </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <a href="javascript:;" class="navbar-search-close js_close-navsearch">Close</a>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Bioinformatics</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <a href="javascript:;" id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch"><span class="screenreader-text">Search Menu</span></a></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

    <input name="searchScope" id="hfSolrJournalID" value="" type="hidden">
    <input name="parentSiteName" id="hfParentSiteName" value="Journals" type="hidden">
    <input name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" type="hidden">
    <input name="siteID" id="hfsiteID" value="5139" type="hidden">
    <input name="parentSiteID" id="hfparentSiteID" value="5567" type="hidden">
    <input id="hfSolrJournalName" value="" type="hidden">
    <input id="hfSolrMaxAllowSearchChar" value="100" type="hidden">
    <input id="hfJournalShortName" value="" type="hidden">
    <input id="hfSearchPlaceholder" value="" type="hidden">
    <input name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" type="hidden">
    <input name="journalSiteScope" id="hfjournalSiteScope" value="Journals" type="hidden">
    <input name="parentSiteScope" id="hfparentSiteScope" value="Parent" type="hidden">
    <input name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" type="hidden">
    <input name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97383" type="hidden">
    <input name="issueSiteScope" id="hfIssueSiteScope" value="Issue" type="hidden">
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" value="advanced-search?page=1&amp;q=" type="hidden">    <input name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/bioinformatics" type="hidden">
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" value="bioinformatics" type="hidden"> 
    </div>

</section>

    <div id="main" class="content-main js-main ui-base">
        <section class="master-main row">
            <div class="center-inner-row no-overflow">
                


<div class="page-column-wrap">
<div id="InfoColumn" class="page-column page-column--left js-left-nav-col">
    <div class="mobile-content-topbar hide">
        <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
    </div>
    <div class="info-inner-wrap js-left-nav" style="top: 0px;">
        <button class="toggle-left-col__close btn-as-icon icon-general-close">
            <span class="screenreader-text">Close mobile search navigation</span>
        </button>
        <div class="responsive-nav-title">Article Navigation</div>
        <div class="info-widget-wrap">
    <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="https://academic.oup.com/bioinformatics/issue/33/11"><img id="issueImage" class="fb-featured-image" src="m_bioinfo_33_11cover.png" alt="Issue Cover"></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="https://academic.oup.com/bioinformatics/issue/33/11">
                        <div class="volume trailing-comma">Volume 33</div>
                        <div class="issue">Issue 11</div>

            </a>

        </div>
        <div class="ii-pub-date">
1 June 2017        </div>
        
    </div>

</div> 
    </div>

            <div class="content-nav">
    <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
        

<div class="contents-title">Article Contents</div>
<ul class="jumplink-list">
        <li class="section-jump-link head-1" link-destination="118736885">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736885">Abstract</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736886">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736886">1 Introduction</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736891">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736891">2 Methods</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736906">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736906">3 Experimental setup</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736920">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736920">4 Results</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736925">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736925">5 Discussion</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736934">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736934">6 Conclusions</a>
            </div>
        </li>
        <li class="section-jump-link head-1" link-destination="118736936">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736936">Acknowledgement</a>
            </div>
        </li>
        <li class="section-jump-link head-1 backReferenceLink" link-destination="118736941">
            <div class="section-jump-link__link-wrap">

                <a class="js-jumplink scrollTo" href="#118736941">References</a>
            </div>
        </li>
<li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li></ul>
 
    </div>

            </div>
    <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="https://academic.oup.com/bioinformatics/article/33/11/1621/2964791">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="https://academic.oup.com/bioinformatics/article/33/11/1639/2959846">Next &gt;</a>
        </li>
</ul> 
    </div>

        </div>
    </div>
</div>
<div class="sticky-toolbar js-sticky-toolbar"></div>
<div id="ContentColumn" class="page-column page-column--center">
    <div class="article-browse-top article-browse-mobile-nav js-mobile-nav">
        <div class="article-browse-mobile-nav-inner js-mobile-nav-inner">
            <button class="toggle-left-col toggle-left-col__article btn-as-link">
                Article Navigation
            </button>
        </div>
    </div>
    <div class="content-inner-wrap">
    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">
                    Multi-rate Poisson tree processes for single-locus 
species delimitation under maximum likelihood and Markov chain Monte 
Carlo
<i class="icon-availability_open" title="Open Access"></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">P Kapli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Email: <a href="mailto:paschalia.kapli@h-its.org" target="">paschalia.kapli@h-its.org</a> or <a href="mailto:alexandros.stamatakis@h-its.org" target="">alexandros.stamatakis@h-its.org</a> or <a href="mailto:tomas.flouri@h-its.org" target="">tomas.flouri@h-its.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=P+Kapli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kapli%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kapli%20P%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">P Kapli</a><span class="delimiter"><i class="icon-general-mail"></i>, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">S Lutteropp</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</div><div class="aff">Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=S+Lutteropp">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lutteropp%20S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Lutteropp%20S%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">S Lutteropp</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=J+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang%20J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Zhang%20J%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">J Zhang</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">K Kobert</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=K+Kobert">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobert%20K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Kobert%20K%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">K Kobert</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">P Pavlidis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Foundation for Research and Technology, Hellas Institute of Computer Science GR, Heraklion, Crete, Greece</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=P+Pavlidis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pavlidis%20P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Pavlidis%20P%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">P Pavlidis</a><span class="delimiter">, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A Stamatakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</div><div class="aff">Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Email: <a href="mailto:paschalia.kapli@h-its.org" target="">paschalia.kapli@h-its.org</a> or <a href="mailto:alexandros.stamatakis@h-its.org" target="">alexandros.stamatakis@h-its.org</a> or <a href="mailto:tomas.flouri@h-its.org" target="">tomas.flouri@h-its.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=A+Stamatakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stamatakis%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Stamatakis%20A%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">A Stamatakis</a><span class="delimiter"><i class="icon-general-mail"></i>, </span>
                        </span>
                        <span class="al-author-name-more">


                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">T Flouri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">The Exelixis Lab, Scientific Computing Group, Heidelberg Institute for Theoretical Studies, Heidelberg, Germany</div><div class="aff">Department of Informatics, Institute of Theoretical Informatics, Karlsruhe Institute of Technology, Karlsruhe, Germany</div>
        </div>


        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Email: <a href="mailto:paschalia.kapli@h-its.org" target="">paschalia.kapli@h-its.org</a> or <a href="mailto:alexandros.stamatakis@h-its.org" target="">alexandros.stamatakis@h-its.org</a> or <a href="mailto:tomas.flouri@h-its.org" target="">tomas.flouri@h-its.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=T+Flouri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Flouri%20T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Flouri%20T%22">Google Scholar</a>
            </div>

</div>
                            </span>



                            <a class="linked-name" href="javascript:;">T Flouri</a><span class="delimiter"><i class="icon-general-mail"></i></span>
                        </span>

                    </div>
                </div>
<div class="pub-history-wrap clearfix">

        <div class="pub-history-row clearfix">
            <div class="ww-citation-primary"><em>Bioinformatics</em>, Volume 33, Issue 11, 1 June 2017, Pages 1630–1638, <a href="https://doi.org/10.1093/bioinformatics/btx025">https://doi.org/10.1093/bioinformatics/btx025</a></div>
        </div>
        <div class="pub-history-row clearfix">
            <div class="ww-citation-date-wrap">
                <div class="citation-label">Published:</div>
                <div class="citation-date">20 January 2017</div>
            </div>
                <a href="javascript:;" class="ww-citation-history-wrap js-article-history st-article-history">
                    <span>Article history</span><i class="icon-general-arrow-filled-down arrow-icon"></i>
                </a>
        </div>
            <div class="history-entries-wrap js-history-entries-wrap">
                <div class="history-entry">
                    <div class="wi-state">Received:</div>
                    <div class="wi-date">28 July 2016</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Revision received:</div>
                    <div class="wi-date">27 December 2016</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Accepted:</div>
                    <div class="wi-date">17 January 2017</div>
                </div>
                <div class="history-entry">
                    <div class="wi-state">Published:</div>
                    <div class="wi-date">20 January 2017</div>
                </div>
        </div>
</div>
    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>

 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
         
    </div>


        <div class="article-body js-content-body">
            <div class="toolbar-wrap js-toolbar-wrap">
                <div class="toolbar-inner-wrap">
                    <ul id="Toolbar" role="navigation">
    <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2929345" href="https://academic.oup.com/bioinformatics/article-pdf/33/11/1630/25153573/btx025.pdf">
            <img src="pdf.svg" alt="pdf"><span class="pdf-link-text">PDF</span>
        </a>
    </li>
                                                    <li class="toolbar-item item-link item-split-view">
                                <a href="javascript:;" class="split-view js-split-view st-split-view" target="">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                        <li class="toolbar-item item-with-dropdown item-views">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                <i class="icon-menu_views"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Views</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up" style="display: block;"></div>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="article-content" style="display: none;">
                                    <a href="javascript:;"><span>Article contents</span></a>
                                </li>
                                <li class="article-content-filter js-article-content-filter" data-content-filter="figures-tables">
                                    <a href="javascript:;"><span>Figures &amp; tables</span></a>
                                </li>
                                
                                
                                <li class="article-content-filter js-article-content-filter" data-content-filter="supplementary-data">
                                    <a href="javascript:;"><span>Supplementary Data</span></a>
                                </li>
                            </ul>
                        </li>
                        <li class="toolbar-item">
    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        <a href="#" class="js-cite-button" data-reveal-id="getCitation" data-reveal=""><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal="">
    <h3 class="modal-title">Cite</h3>
        <div class="oxford-citation-text">
            <p>P Kapli, S Lutteropp, J Zhang, K Kobert, P Pavlidis, A 
Stamatakis, T Flouri,  Multi-rate Poisson tree processes for 
single-locus species delimitation under maximum likelihood and Markov 
chain Monte Carlo, <em>Bioinformatics</em>, Volume 33, Issue 11, 1 June 2017, Pages 1630–1638, <a href="https://doi.org/10.1093/bioinformatics/btx025">https://doi.org/10.1093/bioinformatics/btx025</a></p>
        </div>

    <div class="citation-download-wrap">
        <form action="/Citation/Download" method="get" id="citationModal">
            <input name="resourceId" value="2929345" type="hidden">
            <input name="resourceType" value="3" type="hidden">
            <label for="selectFormat" class="hide">Select Format</label>
            <select required="" name="citationFormat" class="citation-download-format" id="selectFormat">
                <option selected="selected" disabled="disabled">Select format</option>
                        <option value="0">.ris (Mendeley, Papers, Zotero)</option>
                        <option value="1">.enw (EndNote)</option>
                        <option value="2">.bibtex (BibTex)</option>
                        <option value="3">.txt (Medlars, RefWorks)</option>

            </select>
            <button class="btn citation-download-link disabled" type="submit">Download citation</button>
        </form>
    </div>

    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-tools">
    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
        <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1460-2059&amp;title=Multi-rate%20Poisson%20tree%20processes%20for%20single-locus%20species%20delimitation%20under%20maximum%20likelihood%20and%20Markov%20chain%20Monte%20Carlo&amp;publicationDate=2017-01-20&amp;volumeNum=33&amp;issueNum=11&amp;author=Kapli%2C%20P%3B%20Lutteropp%2C%20S&amp;startPage=1630&amp;endPage=1638&amp;contentId=10.1093%2Fbioinformatics%2Fbtx025&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="">
            <i class="icon-menu_permissions">
                <span class="screenreader-text">Permissions Icon</span>
            </i>
            Permissions
        </a>
    </div>
 
    </div>

                        </li>
                        <li class="toolbar-item item-with-dropdown item-share last">
                            <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                <i class="icon-menu_share"></i>
                                <div class="toolbar-label">
                                    <div class="toolbar-text">Share</div>
                                    <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </a>
                            <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content="" aria-label="submenu">
                                <div class="arrow-up"></div>
                                <li><a class="addthis_button_email at300b" target="_blank" title="Email" href="#"><span>Email</span></a></li>
                                <li><a class="addthis_button_twitter at300b" title="Twitter" href="#"><span>Twitter</span></a></li>
                                <li><a class="addthis_button_facebook at300b" title="Facebook" href="#"><span>Facebook</span></a></li>
                                <li><a class="addthis_button_compact at300m" href="#"><span>More</span></a></li>
                            <div class="atclear"></div></ul>
                        </li>
                    </ul>
                    <div class="toolbar-search">
    <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" selected="selected">This issue</option><option class="header-search-bar-filters-item" value="">All  Bioinformatics</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm" type="text">

                    <a href="javascript:;" id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                </div>
            </div>
        </div>
    </div>
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                            <li class="site-menu-item">
                    <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f33%2f11%2f1630%2f2929345" class="register nav-link">Register</a>
                </li>
        </ul>
    </div>
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div>

</div>



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <a href="javascript:;" class="navbar-search-close js_close-navsearch">Close</a>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
                                            <option class="header-search-bar-filters-item" value="Issue">This issue</option>
                                            <option class="header-search-bar-filters-item" value="" selected="selected">All  Bioinformatics</option>

                                </optgroup>
                                <optgroup label="Search beyond this journal">
                                            <option class="header-search-bar-filters-item" value="Journals">All  Journals</option>
                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" maxlength="255" id="MicrositeSearchTerm" title="search input" type="text">

                        <a href="javascript:;" id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"><span class="screenreader-text">Search</span></a>

                    </div>
                </div>
                <div class="navbar-search-advanced"><a href="https://academic.oup.com/bioinformatics/advanced-search?page=1&amp;fl_SiteID=5139&amp;SearchSourceType=1" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div>            <div class="navbar-search-collapsed"><a href="javascript:;" class="icon-menu_search js_expand-navsearch"><span class="screenreader-text">Search Menu</span></a></div>
    </div>
</div>

<input id="hfEnableOupOnlineProductsFeatures" name="hfEnableOupOnlineProductsFeatures" value="False" type="hidden">

<input id="routename" name="RouteName" value="bioinformatics" type="hidden"> 
    </div>

                    </div>
                </div>
            </div>
            <div id="ContentTab" class="content active">
    <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                    <h2 scrollto-destination="118736885" id="118736885" class="abstract-title">Abstract</h2>
<section class="abstract"><div class="sec"><div class="title">Motivation</div><p>In
 recent years, molecular species delimitation has become a routine 
approach for quantifying and classifying biodiversity. Barcoding methods
 are of particular importance in large-scale surveys as they promote 
fast species discovery and biodiversity estimates. Among those, 
distance-based methods are the most common choice as they scale well 
with large datasets; however, they are sensitive to similarity threshold
 parameters and they ignore evolutionary relationships. The recently 
introduced “Poisson Tree Processes” (PTP) method is a phylogeny-aware 
approach that does not rely on such thresholds. Yet, two weaknesses of 
PTP impact its accuracy and practicality when applied to large datasets;
 it does not account for divergent intraspecific variation and is slow 
for a large number of sequences.</p></div><div class="js-collapsableEnd"></div><div class="sec"><div class="title">Results</div><p>We
 introduce the multi-rate PTP (mPTP), an improved method that alleviates
 the theoretical and technical shortcomings of PTP. It incorporates 
different levels of intraspecific genetic diversity deriving from 
differences in either the evolutionary history or sampling of each 
species. Results on empirical data suggest that mPTP is superior to PTP 
and popular distance-based methods as it, consistently yields more 
accurate delimitations with respect to the taxonomy (i.e., identifies 
more taxonomic species, infers species numbers closer to the taxonomy). 
Moreover, mPTP does not require any similarity threshold as input. The 
novel dynamic programming algorithm attains a speedup of at least five 
orders of magnitude compared to PTP, allowing it to delimit species in 
large (meta-) barcoding data. In addition, Markov Chain Monte Carlo 
sampling provides a comprehensive evaluation of the inferred 
delimitation in just a few seconds for millions of steps, independently 
of tree size.</p></div><div class="js-collapsableEnd"></div><div class="sec"><div class="title">Availability and Implementation</div><p>mPTP is implemented in C and is available for download at <a class="link link-uri openInAnotherWindow" href="http://github.com/Pas-Kapli/mptp" target="_blank">http://github.com/Pas-Kapli/mptp</a> under the GNU Affero 3 license. A web-service is available at <a class="link link-uri openInAnotherWindow" href="http://mptp.h-its.org/" target="_blank">http://mptp.h-its.org</a>.</p></div><div class="js-collapsableEnd"></div><div class="sec"><div class="title">Supplementary information</div><p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>Bioinformatics</em> online.</p></div><div class="js-collapsableEnd"></div></section>                        <div class="article-metadata-panel clearfix"><div class="article-metadata-tocSections">
                    <div class="article-metadata-tocSections-title">Issue Section:</div>
                        <a href="https://academic.oup.com/bioinformatics/search-results?f_TocHeadingTitle=Phylogenetics">Phylogenetics</a>
                </div><div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Associate Editor:
                            <span class="al-author-name">
                                <span id="info-18233197" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alfonso Valencia</div>
            <div class="info-card-role">
                Associate Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="https://academic.oup.com/bioinformatics/search-results?f_Authors=Alfonso+Valencia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Valencia%20A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:%22Valencia%20Alfonso%22">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Alfonso Valencia</a>
                            </span>
                        <br>
                    </div>
            </div>
        </div></div>
                    <h2 scrollto-destination="118736886" id="118736886" class="section-title">1 Introduction</h2>
<p>Species are fundamental units of life and form the most common basis 
of comparison in evolutionary studies. Therefore, species delimitation 
is a critical task in systematic studies with potential implications in 
all subfields of biology that involve evolutionary relationships. In 
line with the species concept of <a reveal-id="btx025-B11" data-open="btx025-B11" class="link link-ref link-reveal xref-bibr">De Queiroz (2007)</a> and the integrative taxonomy approach (<a reveal-id="btx025-B10" data-open="btx025-B10" class="link link-ref link-reveal xref-bibr">Dayrat, 2005</a>),
 a reliable identification of a new species involves data from multiple 
sources (e.g., ecology, morphology, evolutionary history). Such an 
approach is necessary for meticulous comparative evolutionary studies 
but is tremendously difficult to apply, if at all possible, in large 
biodiversity studies (e.g., vast barcoding, environmental, microbial 
samples). In such studies, researchers use alternative units of 
comparison that are easier to delimit [Molecular Operational Taxonomic 
Units (<a reveal-id="btx025-B7" data-open="btx025-B7" class="link link-ref link-reveal xref-bibr">Blaxter <em>et al.</em>, 2005</a>), Recognizable Taxonomic Units (<a reveal-id="btx025-B31" data-open="btx025-B31" class="link link-ref link-reveal xref-bibr">Oliver and Beattie, 1993</a>)]
 rather than going through the cumbersome task of delimiting and 
describing full species. The correspondence of these units to real 
species remains ambiguous (<a reveal-id="btx025-B9" data-open="btx025-B9" class="link link-ref link-reveal xref-bibr">Casiraghi <em>et al.</em>, 2010</a>), especially in the absence of additional information. However, they are practical for biodiversity and <em>β</em>-diversity estimates (<a reveal-id="btx025-B43" data-open="btx025-B43" class="link link-ref link-reveal xref-bibr">Valentini <em>et al.</em>, 2009</a>).</p><p>With the introduction of DNA-barcoding (<a reveal-id="btx025-B20" data-open="btx025-B20" class="link link-ref link-reveal xref-bibr">Hebert <em>et al.</em>, 2003</a>) and the advances in coalescent models (<a reveal-id="btx025-B15" data-open="btx025-B15" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>; <a reveal-id="btx025-B46" data-open="btx025-B46" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2014</a>),
 genetic data became the most popular data source for delimiting 
species. Several algorithms and implementations exist for this purpose, 
most of which are inspired by the phylogenetic species concept (<a reveal-id="btx025-B15" data-open="btx025-B15" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>; <a reveal-id="btx025-B46" data-open="btx025-B46" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2014</a>; <a reveal-id="btx025-B48" data-open="btx025-B48" class="link link-ref link-reveal xref-bibr">Zhang <em>et al.</em>, 2013</a>) and the DNA barcoding concept (<a reveal-id="btx025-B18" data-open="btx025-B18" class="link link-ref link-reveal xref-bibr">Hao <em>et al.</em>, 2011</a>; <a reveal-id="btx025-B13" data-open="btx025-B13" class="link link-ref link-reveal xref-bibr">Edgar, 2010</a>; <a reveal-id="btx025-B33" data-open="btx025-B33" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>). These methods address different research questions (<a reveal-id="btx025-B9" data-open="btx025-B9" class="link link-ref link-reveal xref-bibr">Casiraghi <em>et al.</em>, 2010</a>);
 thus, the user needs to assess several factors before choosing the most
 appropriate method for a particular study. The “species-tree” 
approaches rely on multiple genetic loci (<a reveal-id="btx025-B23" data-open="btx025-B23" class="link link-ref link-reveal xref-bibr">Jones <em>et al.</em>, 2015</a>; <a reveal-id="btx025-B46" data-open="btx025-B46" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2014</a>) and account for potential species-tree/gene-tree incongruence (<a reveal-id="btx025-B26" data-open="btx025-B26" class="link link-ref link-reveal xref-bibr">Maddison, 1997</a>).
 Such methods are the most appropriate when the goal is to perform 
taxonomic revisions. However, most current implementations (<a reveal-id="btx025-B23" data-open="btx025-B23" class="link link-ref link-reveal xref-bibr">Jones <em>et al.</em>, 2015</a>; <a reveal-id="btx025-B46" data-open="btx025-B46" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2014</a>)
 are computationally demanding and can only be applied to small datasets
 of closely related taxa, becoming impractical with a growing number of 
samples and/or loci [see <a reveal-id="btx025-B16" data-open="btx025-B16" class="link link-ref link-reveal xref-bibr">Fujisawa <em>et al.</em> (2016)</a>
 for a recently introduced faster method]. Large-scale biodiversity 
(meta-) barcoding studies comprise hundreds or even thousands of samples
 of high evolutionary divergence. The goal of such studies often is to 
obtain β-diversity estimates (comparative studies of different 
treatments, ecological factors, etc.) or a rough estimate of the 
biodiversity for a given sample. Hence, a researcher may use 
distance-based methods (<a reveal-id="btx025-B13" data-open="btx025-B13" class="link link-ref link-reveal xref-bibr">Edgar, 2010</a>; <a reveal-id="btx025-B18" data-open="btx025-B18" class="link link-ref link-reveal xref-bibr">Hao <em>et al.</em>, 2011</a>; <a reveal-id="btx025-B33" data-open="btx025-B33" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>)
 that scale well on large datasets with respect to run times. However, 
these methods ignore the evolutionary relationships of the involved taxa
 and rely on not necessarily biologically meaningful <em>ad hoc</em> 
sequence similarity thresholds. Moreover, they are restricted to 
single-locus data, which reduces the accuracy of the delimitation (<a reveal-id="btx025-B12" data-open="btx025-B12" class="link link-ref link-reveal xref-bibr">Dupuis <em>et al.</em>, 2012</a>).</p><p>The General Mixed Yule Coalescent (GMYC; <a reveal-id="btx025-B15" data-open="btx025-B15" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>; <a reveal-id="btx025-B32" data-open="btx025-B32" class="link link-ref link-reveal xref-bibr">Pons <em>et al.</em>, 2006</a>) and the recently introduced Poisson Tree Processes (PTP; <a reveal-id="btx025-B48" data-open="btx025-B48" class="link link-ref link-reveal xref-bibr">Zhang <em>et al.</em>, 2013</a>)
 are two similar models that bridge the gap between “species-tree” and 
distance-based methods. While GMYC and PTP are also restricted to single
 loci, they do take the evolutionary relationships of the sequences into
 account. Since, at the same time, they are computationally inexpensive 
they can be deployed for analyzing large (meta-)barcoding samples. The 
GMYC method (<a reveal-id="btx025-B15" data-open="btx025-B15" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>) uses a speciation (<a reveal-id="btx025-B47" data-open="btx025-B47" class="link link-ref link-reveal xref-bibr">Yule, 1925</a>) and a neutral coalescent model (<a reveal-id="btx025-B22" data-open="btx025-B22" class="link link-ref link-reveal xref-bibr">Hudson, 1990</a>).
 It strives to maximize the likelihood score by separating/classifying 
the branches of an ultrametric tree (in units of absolute or relative 
ages) into two processes; within and between species. In contrast to 
GMYC, PTP models the branching processes based on the number of 
accumulated expected substitutions between subsequent speciation events.
 PTP tries to determine the transition point from a between- to a 
within-species process by assuming that a two parameter model—one 
parameter for the speciation and one for the coalescent process—best 
fits the data. The underlying assumption is that each substitution has a
 small probability of generating a branching event. Within species, 
branching events will be frequent whereas among species they will be 
more sparse. The probability of observing <em>n</em> speciations for <em>k</em>
 substitutions follows a Poisson process and therefore, the number of 
substitutions until the next speciation event can be modeled via an 
exponential distribution (<a reveal-id="btx025-B48" data-open="btx025-B48" class="link link-ref link-reveal xref-bibr">Zhang <em>et al.</em>, 2013</a>).
 Given that PTP directly uses substitutions, it does not require an 
ultrametric input tree, the inference of which can be time consuming and
 error prone. Thus, PTP often yields more accurate delimitations than 
GMYC (<a reveal-id="btx025-B41" data-open="btx025-B41" class="link link-ref link-reveal xref-bibr">Tang <em>et al.</em>, 2014</a>).</p><p>Here,
 we introduce a new algorithm and an improved model as well as 
implementation of PTP that alleviates previous shortcomings of the 
method. The initial PTP assumes one exponential distribution for the 
speciation events and only one for the coalescent events, across all 
species in the phylogeny. While the speciation rate can be assumed to be
 constant among closely related species, the intraspecific coalescent 
rate and consequently the genetic diversity may vary significantly even 
among sister species. This divergence in intraspecific variation can be 
attributed to factors such as population size and structure, population 
bottlenecks, selection, life cycle and mating systems (see <a reveal-id="btx025-B4" data-open="btx025-B4" class="link link-ref link-reveal xref-bibr">Bazin et al. (2006)</a>
 for further details). Additionally, sampling bias may also be 
responsible for observing different levels of intraspecific genetic 
diversity and it is already known to decrease the accuracy of PTP (<a reveal-id="btx025-B48" data-open="btx025-B48" class="link link-ref link-reveal xref-bibr">Zhang <em>et al.</em>, 2013</a>).
 To incorporate the potential divergence in intraspecific diversity, we 
propose the novel multi-rate PTP (mPTP) model. In contrast to PTP, it 
fits the branching events of <em>each</em> delimited species to a <em>distinct</em>
 exponential distribution. Thereby it can better accommodate the 
sampling- and population-specific characteristics of a broader range of 
empirical datasets. In addition, we develop and present a novel, 
dynamic-programming algorithm for the mPTP model. The implementation is 
several orders of magnitude faster than the original PTP and yields more
 accurate delimitations (almost instantly) on large datasets comprising 
thousands of taxa. Note that, the original PTP requires days of 
computation time to analyze such datasets. Finally, we provide a <em>Markov chain Monte Carlo</em> (MCMC) sampling approach that allows for inferring delimitation support values.</p>                    <h2 scrollto-destination="118736891" id="118736891" class="section-title">2 Methods</h2>
<p>The mPTP method takes as input a binary, rooted phylogenetic tree inferred with software such as RAxML (<a reveal-id="btx025-B39" data-open="btx025-B39" class="link link-ref link-reveal xref-bibr">Stamatakis, 2014</a>) or MrBayes (<a reveal-id="btx025-B37" data-open="btx025-B37" class="link link-ref link-reveal xref-bibr">Ronquist <em>et al.</em>, 2012</a>).
 Such software often infer unrooted trees; therefore, mPTP provides the 
option to root the input tree prior to the analysis either (a) 
implicitly on its longest terminal branch or (b) explicitly by providing
 an outgroup. In this section, we introduce the basic notation that is 
used throughout the manuscript, and provide a detailed description of 
the mPTP algorithm including the MCMC sampling method.</p><p>A binary rooted tree <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-1"><span class="MJXp-mrow" id="MJXp-Span-2"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-3">T</span><span class="MJXp-mo" id="MJXp-Span-4" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-5" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-6">V</span><span class="MJXp-mo" id="MJXp-Span-7" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-8">E</span><span class="MJXp-mo" id="MJXp-Span-9" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-1-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-1"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-2"><span class="mrow" id="MathJax-Span-3"><span class="mi" id="MathJax-Span-4" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span class="mo" id="MathJax-Span-5" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-6" style="font-family: MathJax_Main; padding-left: 0.278em;">(</span><span class="mi" id="MathJax-Span-7" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo" id="MathJax-Span-8" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-9" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mo" id="MathJax-Span-10" style="font-family: MathJax_Main;">)</span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-1"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>T</mi><mo>=</mo><mo stretchy="false">(</mo><mi>V</mi><mo>,</mo><mi>E</mi><mo stretchy="false">)</mo></mrow></math></script></span> is a connected acyclic graph where <em>V</em> is the set of nodes and <em>E</em> the set of branches (or <em>edges</em>), such that <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-10"><span class="MJXp-mrow" id="MJXp-Span-11"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-12">E</span><span class="MJXp-mo" id="MJXp-Span-13" style="margin-left: 0.333em; margin-right: 0.333em;">⊂</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-14">V</span><span class="MJXp-mo" id="MJXp-Span-15" style="margin-left: 0.267em; margin-right: 0.267em;">×</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-16">V</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-2-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-11"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-12"><span class="mrow" id="MathJax-Span-13"><span class="mi" id="MathJax-Span-14" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mo" id="MathJax-Span-15" style="font-family: MathJax_Main; padding-left: 0.278em;">⊂</span><span class="mi" id="MathJax-Span-16" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo" id="MathJax-Span-17" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mi" id="MathJax-Span-18" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-2"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>E</mi><mo>⊂</mo><mi>V</mi><mo>×</mo><mi>V</mi></mrow></math></script>⁠</span>. Each <em>inner node u</em> has a <em>degree</em> (number of branches for which <em>u</em> is an end-point) of 3 with the exception of the root node which has degree 2, while <em>leaves</em> (or <em>tips</em>) have degree 1. We use the notation <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-17"><span class="MJXp-mrow" id="MJXp-Span-18"><span class="MJXp-mo" id="MJXp-Span-19" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-20">u</span><span class="MJXp-mo" id="MJXp-Span-21" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-22">v</span><span class="MJXp-mo" id="MJXp-Span-23" style="margin-left: 0em; margin-right: 0em;">)</span><span class="MJXp-mo" id="MJXp-Span-24" style="margin-left: 0.333em; margin-right: 0.333em;">∈</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-25">E</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-3-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-19"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-20"><span class="mrow" id="MathJax-Span-21"><span class="mo" id="MathJax-Span-22" style="font-family: MathJax_Main;">(</span><span class="mi" id="MathJax-Span-23" style="font-family: MathJax_Math; font-style: italic;">u</span><span class="mo" id="MathJax-Span-24" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-25" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">v</span><span class="mo" id="MathJax-Span-26" style="font-family: MathJax_Main;">)</span><span class="mo" id="MathJax-Span-27" style="font-family: MathJax_Main; padding-left: 0.278em;">∈</span><span class="mi" id="MathJax-Span-28" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-3"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo stretchy="false">(</mo><mi>u</mi><mo>,</mo><mi>v</mi><mo stretchy="false">)</mo><mo>∈</mo><mi>E</mi></mrow></math></script></span> to denote a branch with end-points <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-26"><span class="MJXp-mrow" id="MJXp-Span-27"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-28">u</span><span class="MJXp-mo" id="MJXp-Span-29" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-30">v</span><span class="MJXp-mo" id="MJXp-Span-31" style="margin-left: 0.333em; margin-right: 0.333em;">∈</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-32">V</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-4-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-29"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-30"><span class="mrow" id="MathJax-Span-31"><span class="mi" id="MathJax-Span-32" style="font-family: MathJax_Math; font-style: italic;">u</span><span class="mo" id="MathJax-Span-33" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-34" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">v</span><span class="mo" id="MathJax-Span-35" style="font-family: MathJax_Main; padding-left: 0.278em;">∈</span><span class="mi" id="MathJax-Span-36" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-4"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>u</mi><mo>,</mo><mi>v</mi><mo>∈</mo><mi>V</mi></mrow></math></script>⁠</span>, and <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-42"><span class="MJXp-mrow" id="MJXp-Span-43"><span class="MJXp-mo" id="MJXp-Span-44" style="margin-left: 0em; margin-right: 0em;">ℓ</span><span class="MJXp-mo" id="MJXp-Span-45" style="margin-left: 0.111em; margin-right: 0.167em;">:</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-46">V</span><span class="MJXp-mo" id="MJXp-Span-47" style="margin-left: 0.267em; margin-right: 0.267em;">×</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-48">V</span><span class="MJXp-mo" id="MJXp-Span-49" style="margin-left: 0.333em; margin-right: 0.333em;">→</span><span class="MJXp-mo" id="MJXp-Span-50" style="margin-left: 0em; margin-right: 0em;">ℝ</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-5-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-37"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-38"><span class="mrow" id="MathJax-Span-39"><span class="mo" id="MathJax-Span-40" style="font-family: MathJax_Main;">ℓ</span><span class="mo" id="MathJax-Span-41" style="font-family: MathJax_Main; padding-left: 0.278em;">:</span><span class="mi" id="MathJax-Span-42" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo" id="MathJax-Span-43" style="font-family: MathJax_Main; padding-left: 0.222em;">×</span><span class="mi" id="MathJax-Span-44" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.222em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo" id="MathJax-Span-45" style="font-family: MathJax_Main; padding-left: 0.278em;">→</span><span class="mo" id="MathJax-Span-46" style="font-family: MathJax_AMS; padding-left: 0.278em;">R</span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-5"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>ℓ</mo><mo>:</mo><mi>V</mi><mo>×</mo><mi>V</mi><mo>→</mo><mo>ℝ</mo></mrow></math></script></span> to denote the associated branch length. Finally, we use <em>T<sub>u</sub></em> to denote the subtree rooted at node <em>u</em>.</p>                    <h3 scrollto-destination="118736894" id="118736894" class="section-title">2.1 mPTP heuristic algorithm</h3>
<div class="block-child-p">Let <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-51"><span class="MJXp-mrow" id="MJXp-Span-52"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-53">T</span><span class="MJXp-mo" id="MJXp-Span-54" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-55" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-56">V</span><span class="MJXp-mo" id="MJXp-Span-57" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-58">E</span><span class="MJXp-mo" id="MJXp-Span-59" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-6-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-47"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-48"><span class="mrow" id="MathJax-Span-49"><span class="mi" id="MathJax-Span-50" style="font-family: MathJax_Math; font-style: italic;">T<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.12em;"></span></span><span class="mo" id="MathJax-Span-51" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-52" style="font-family: MathJax_Main; padding-left: 0.278em;">(</span><span class="mi" id="MathJax-Span-53" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo" id="MathJax-Span-54" style="font-family: MathJax_Main;">,</span><span class="mi" id="MathJax-Span-55" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mo" id="MathJax-Span-56" style="font-family: MathJax_Main;">)</span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-6"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>T</mi><mo>=</mo><mo stretchy="false">(</mo><mi>V</mi><mo>,</mo><mi>E</mi><mo stretchy="false">)</mo></mrow></math></script></span> be a binary rooted phylogenetic tree with root node <em>r</em>. The optimization problem in the original PTP is to find a connected subgraph <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-60"><span class="MJXp-mrow" id="MJXp-Span-61"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-62">G</span><span class="MJXp-mo" id="MJXp-Span-63" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-64" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-msub" id="MJXp-Span-65"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-66" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-67" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-68" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-69"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-70" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-71" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-72" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-7-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-57"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-58"><span class="mrow" id="MathJax-Span-59"><span class="mi" id="MathJax-Span-60" style="font-family: MathJax_Math; font-style: italic;">G</span><span class="mo" id="MathJax-Span-61" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mo" id="MathJax-Span-62" style="font-family: MathJax_Main; padding-left: 0.278em;">(</span><span class="msub" id="MathJax-Span-63"><span style="display: inline-block; position: relative; width: 0.99em; height: 0px;"><span style="position: absolute; clip: rect(3.159em, 1000.77em, 4.189em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-64" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.583em;"><span class="mi" id="MathJax-Span-65" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span><span class="mo" id="MathJax-Span-66" style="font-family: MathJax_Main;">,</span><span class="msub" id="MathJax-Span-67" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.145em; height: 0px;"><span style="position: absolute; clip: rect(3.162em, 1000.76em, 4.167em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-68" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.738em;"><span class="mi" id="MathJax-Span-69" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span><span class="mo" id="MathJax-Span-70" style="font-family: MathJax_Main;">)</span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-7"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>G</mi><mo>=</mo><mo stretchy="false">(</mo><msub><mi>V</mi><mi>s</mi></msub><mo>,</mo><msub><mi>E</mi><mi>s</mi></msub><mo stretchy="false">)</mo></mrow></math></script></span> of <em>T</em>, where <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-82"><span class="MJXp-mrow" id="MJXp-Span-83"><span class="MJXp-msub" id="MJXp-Span-84"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-85" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-86" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-87" style="margin-left: 0.333em; margin-right: 0.333em;">⊆</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-88">V</span><span class="MJXp-mo" id="MJXp-Span-89" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mo" id="MJXp-Span-90" style="margin-left: 0em; margin-right: 0.222em;"> </span><span class="MJXp-msub" id="MJXp-Span-91"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-92" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-93" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-94" style="margin-left: 0.333em; margin-right: 0.333em;">⊆</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-95">E</span><span class="MJXp-mo" id="MJXp-Span-96" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mo" id="MJXp-Span-97" style="margin-left: 0em; margin-right: 0.222em;"> </span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-98">r</span><span class="MJXp-mo" id="MJXp-Span-99" style="margin-left: 0.333em; margin-right: 0.333em;">∈</span><span class="MJXp-msub" id="MJXp-Span-100"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-101" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-102" style="vertical-align: -0.4em;">s</span></span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-8-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-71"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-72"><span class="mrow" id="MathJax-Span-73"><span class="msub" id="MathJax-Span-74"><span style="display: inline-block; position: relative; width: 0.99em; height: 0px;"><span style="position: absolute; clip: rect(3.159em, 1000.77em, 4.189em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-75" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.583em;"><span class="mi" id="MathJax-Span-76" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span><span class="mo" id="MathJax-Span-77" style="font-family: MathJax_Main; padding-left: 0.278em;">⊆</span><span class="mi" id="MathJax-Span-78" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span class="mo" id="MathJax-Span-79" style="font-family: MathJax_Main;">,</span><span class="mo" id="MathJax-Span-80" style="font-family: MathJax_Main; padding-left: 0.167em;"> </span><span class="msub" id="MathJax-Span-81" style="padding-left: 0.167em;"><span style="display: inline-block; position: relative; width: 1.145em; height: 0px;"><span style="position: absolute; clip: rect(3.162em, 1000.76em, 4.167em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-82" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.738em;"><span class="mi" id="MathJax-Span-83" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span><span class="mo" id="MathJax-Span-84" style="font-family: MathJax_Main; padding-left: 0.278em;">⊆</span><span class="mi" id="MathJax-Span-85" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mo" id="MathJax-Span-86" style="font-family: MathJax_Main;">,</span><span class="mo" id="MathJax-Span-87" style="font-family: MathJax_Main; padding-left: 0.167em;"> </span><span class="mi" id="MathJax-Span-88" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.167em;">r</span><span class="mo" id="MathJax-Span-89" style="font-family: MathJax_Main; padding-left: 0.278em;">∈</span><span class="msub" id="MathJax-Span-90" style="padding-left: 0.278em;"><span style="display: inline-block; position: relative; width: 0.99em; height: 0px;"><span style="position: absolute; clip: rect(3.159em, 1000.77em, 4.189em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-91" style="font-family: MathJax_Math; font-style: italic;">V<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.186em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.583em;"><span class="mi" id="MathJax-Span-92" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-8"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>V</mi><mi>s</mi></msub><mo>⊆</mo><mi>V</mi><mo>,</mo><mo> </mo><msub><mi>E</mi><mi>s</mi></msub><mo>⊆</mo><mi>E</mi><mo>,</mo><mo> </mo><mi>r</mi><mo>∈</mo><msub><mi>V</mi><mi>s</mi></msub></mrow></math></script></span> such that (a) <em>G</em> is a binary tree, and (b) the likelihood of i.i.d branch lengths <em>E<sub>s</sub></em> and <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-103"><span class="MJXp-mrow" id="MJXp-Span-104"><span class="MJXp-msub" id="MJXp-Span-105"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-106" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-107" style="vertical-align: -0.4em;">c</span></span><span class="MJXp-mo" id="MJXp-Span-108" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-109">E</span><span class="MJXp-mi" id="MJXp-Span-110">∖</span><span class="MJXp-msub" id="MJXp-Span-111"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-112" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-113" style="vertical-align: -0.4em;">s</span></span></span></span></span><span class="MathJax MathJax_Processed" id="MathJax-Element-9-Frame" tabindex="0" style=""><nobr><span class="math" id="MathJax-Span-93"><span style="display: inline-block; position: relative; width: 0em; height: 0px; font-size: 132%;"><span style="position: absolute;"><span class="mrow" id="MathJax-Span-94"><span class="mrow" id="MathJax-Span-95"><span class="msub" id="MathJax-Span-96"><span style="display: inline-block; position: relative; width: 1.119em; height: 0px;"><span style="position: absolute; clip: rect(3.162em, 1000.76em, 4.167em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-97" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.738em;"><span class="mi" id="MathJax-Span-98" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">c</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span><span class="mo" id="MathJax-Span-99" style="font-family: MathJax_Main; padding-left: 0.278em;">=</span><span class="mi" id="MathJax-Span-100" style="font-family: MathJax_Math; font-style: italic; padding-left: 0.278em;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span class="mi" id="MathJax-Span-101" style="font-family: MathJax_AMS;">∖</span><span class="msub" id="MathJax-Span-102"><span style="display: inline-block; position: relative; width: 1.145em; height: 0px;"><span style="position: absolute; clip: rect(3.162em, 1000.76em, 4.167em, -1000em); top: -4.004em; left: 0em;"><span class="mi" id="MathJax-Span-103" style="font-family: MathJax_Math; font-style: italic;">E<span style="display: inline-block; overflow: hidden; height: 1px; width: 0.026em;"></span></span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span><span style="position: absolute; top: -3.854em; left: 0.738em;"><span class="mi" id="MathJax-Span-104" style="font-size: 70.7%; font-family: MathJax_Math; font-style: italic;">s</span><span style="display: inline-block; width: 0px; height: 4.004em;"></span></span></span></span></span></span></span></span></span></nobr></span><script type="math/mml" id="MathJax-Element-9"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>E</mi><mi>c</mi></msub><mo>=</mo><mi>E</mi><mi mathvariant="normal">∖</mi><msub><mi>E</mi><mi>s</mi></msub></mrow></math></script></span> fitting two distinct exponential distributions is maximized. Formally, we are interested in maximizing the likelihood <a id="E1" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-114"><span class="MJXp-mrow" id="MJXp-Span-115"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-116">L</span><span class="MJXp-mo" id="MJXp-Span-117" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mrow" id="MJXp-Span-118"><span class="MJXp-mo" id="MJXp-Span-119" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">(</span></span><span class="MJXp-mrow" id="MJXp-Span-120"><span class="MJXp-munder" id="MJXp-Span-121"><span class="MJXp-mo" id="MJXp-Span-122" style="margin-left: 0.111em; margin-right: 0.05em;">∏</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-123" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-124">x</span><span class="MJXp-mo" id="MJXp-Span-125">∈</span><span class="MJXp-msub" id="MJXp-Span-126"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-127" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-128" style="vertical-align: -0.4em;">s</span></span></span></span><span class="MJXp-mrow" id="MJXp-Span-129"><span class="MJXp-msub" id="MJXp-Span-130"><span class="MJXp-mrow" id="MJXp-Span-131" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-132"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-134" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-133">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-135" style="vertical-align: -0.4em;">s</span></span></span><span class="MJXp-msup" id="MJXp-Span-136"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-137" style="margin-right: 0.05em;">e</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-138" style="vertical-align: 0.5em;"><span class="MJXp-mo" id="MJXp-Span-139">−</span><span class="MJXp-msub" id="MJXp-Span-140"><span class="MJXp-mrow" id="MJXp-Span-141" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-142"><span><span class="MJXp-over"><span class=" MJXp-script" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-144" style="margin-right: 0px; margin-left: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-143">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-145" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-146">ℓ</span><span class="MJXp-mo" id="MJXp-Span-147">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-148">x</span><span class="MJXp-mo" id="MJXp-Span-149">)</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-150" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">)</span></span></span><span class="MJXp-mo" id="MJXp-Span-151" style="margin-left: 0.267em; margin-right: 0.267em;">×</span><span class="MJXp-mrow" id="MJXp-Span-152"><span class="MJXp-mo" id="MJXp-Span-153" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">(</span></span><span class="MJXp-mrow" id="MJXp-Span-154"><span class="MJXp-munder" id="MJXp-Span-155"><span class="MJXp-mo" id="MJXp-Span-156" style="margin-left: 0.111em; margin-right: 0.05em;">∏</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-157" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-158">x</span><span class="MJXp-mo" id="MJXp-Span-159">∈</span><span class="MJXp-msub" id="MJXp-Span-160"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-161" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-162" style="vertical-align: -0.4em;">c</span></span></span></span><span class="MJXp-mrow" id="MJXp-Span-163"><span class="MJXp-msub" id="MJXp-Span-164"><span class="MJXp-mrow" id="MJXp-Span-165" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-166"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-168" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-167">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-169" style="vertical-align: -0.4em;">c</span></span></span><span class="MJXp-msup" id="MJXp-Span-170"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-171" style="margin-right: 0.05em;">e</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-172" style="vertical-align: 0.5em;"><span class="MJXp-mo" id="MJXp-Span-173">−</span><span class="MJXp-msub" id="MJXp-Span-174"><span class="MJXp-mrow" id="MJXp-Span-175" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-176"><span><span class="MJXp-over"><span class=" MJXp-script" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-178" style="margin-right: 0px; margin-left: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-177">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-179" style="vertical-align: -0.4em;">c</span></span><span class="MJXp-mo" id="MJXp-Span-180">ℓ</span><span class="MJXp-mo" id="MJXp-Span-181">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-182">x</span><span class="MJXp-mo" id="MJXp-Span-183">)</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-184" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">)</span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-10-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-10"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>L</mi><mo>=</mo><mrow><mo stretchy="true">(</mo><mrow><munder><mo>∏</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>s</mi></msub></mrow></munder><mrow><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>s</mi></msub></mrow><msup><mi>e</mi><mrow><mo>−</mo><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>s</mi></msub><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo stretchy="true">)</mo></mrow><mo>×</mo><mrow><mo stretchy="true">(</mo><mrow><munder><mo>∏</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>c</mi></msub></mrow></munder><mrow><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>c</mi></msub></mrow><msup><mi>e</mi><mrow><mo>−</mo><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>c</mi></msub><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo stretchy="true">)</mo></mrow></mrow></math></script></div></div><div class="label">(1)</div></div> where <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-185"><span class="MJXp-mrow" id="MJXp-Span-186"><span class="MJXp-msub" id="MJXp-Span-187"><span class="MJXp-mrow" id="MJXp-Span-188" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-189"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-191" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-190">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-192" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-193" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-msup" id="MJXp-Span-194"><span class="MJXp-mrow" id="MJXp-Span-195" style="margin-right: 0.05em;"><span class="MJXp-mrow" id="MJXp-Span-196"><span class="MJXp-mo" id="MJXp-Span-197" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.689em;"><span style="font-size: 3.756em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">(</span></span><span class="MJXp-mrow" id="MJXp-Span-198"><span class="MJXp-mfrac" id="MJXp-Span-199" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mn" id="MJXp-Span-200">1</span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-201"><span class="MJXp-mo" id="MJXp-Span-202" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-203"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-204" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-205" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-206" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span></span></span></span></span></span></span><span class="MJXp-munder" id="MJXp-Span-207"><span class="MJXp-mo" id="MJXp-Span-208" style="margin-left: 0.111em; margin-right: 0.05em;">∑</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-209" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-210">x</span><span class="MJXp-mo" id="MJXp-Span-211">∈</span><span class="MJXp-msub" id="MJXp-Span-212"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-213" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-214" style="vertical-align: -0.4em;">s</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-215" style="margin-left: 0em; margin-right: 0em;">ℓ</span><span class="MJXp-mo" id="MJXp-Span-216" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-217">x</span><span class="MJXp-mo" id="MJXp-Span-218" style="margin-left: 0em; margin-right: 0em;">)</span></span><span class="MJXp-mo" id="MJXp-Span-219" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.689em;"><span style="font-size: 3.756em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">)</span></span></span></span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-220" style="vertical-align: 1.2em;"><span class="MJXp-mo" id="MJXp-Span-221">−</span><span class="MJXp-mn" id="MJXp-Span-222">1</span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-11-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-11"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>s</mi></msub><mo>=</mo><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mfrac><mn>1</mn><mrow><mo>|</mo><msub><mi>E</mi><mi>s</mi></msub><mo>|</mo></mrow></mfrac><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>s</mi></msub></mrow></munder><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math></script></span> and <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-223"><span class="MJXp-mrow" id="MJXp-Span-224"><span class="MJXp-msub" id="MJXp-Span-225"><span class="MJXp-mrow" id="MJXp-Span-226" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-227"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-229" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-228">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-230" style="vertical-align: -0.4em;">c</span></span><span class="MJXp-mo" id="MJXp-Span-231" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-msup" id="MJXp-Span-232"><span class="MJXp-mrow" id="MJXp-Span-233" style="margin-right: 0.05em;"><span class="MJXp-mrow" id="MJXp-Span-234"><span class="MJXp-mo" id="MJXp-Span-235" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.689em;"><span style="font-size: 3.756em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">(</span></span><span class="MJXp-mrow" id="MJXp-Span-236"><span class="MJXp-mfrac" id="MJXp-Span-237" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mn" id="MJXp-Span-238">1</span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-239"><span class="MJXp-mo" id="MJXp-Span-240" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-241"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-242" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-243" style="vertical-align: -0.4em;">c</span></span><span class="MJXp-mo" id="MJXp-Span-244" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span></span></span></span></span></span></span><span class="MJXp-munder" id="MJXp-Span-245"><span class="MJXp-mo" id="MJXp-Span-246" style="margin-left: 0.111em; margin-right: 0.05em;">∑</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-247" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-248">x</span><span class="MJXp-mo" id="MJXp-Span-249">∈</span><span class="MJXp-msub" id="MJXp-Span-250"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-251" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-252" style="vertical-align: -0.4em;">c</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-253" style="margin-left: 0em; margin-right: 0em;">ℓ</span><span class="MJXp-mo" id="MJXp-Span-254" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-255">x</span><span class="MJXp-mo" id="MJXp-Span-256" style="margin-left: 0em; margin-right: 0em;">)</span></span><span class="MJXp-mo" id="MJXp-Span-257" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.689em;"><span style="font-size: 3.756em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">)</span></span></span></span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-258" style="vertical-align: 1.2em;"><span class="MJXp-mo" id="MJXp-Span-259">−</span><span class="MJXp-mn" id="MJXp-Span-260">1</span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-12-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-12"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>c</mi></msub><mo>=</mo><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mfrac><mn>1</mn><mrow><mo>|</mo><msub><mi>E</mi><mi>c</mi></msub><mo>|</mo></mrow></mfrac><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>c</mi></msub></mrow></munder><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math></script></span> are maximum-likelihood (ML) estimates for the rate parameters.</div><div class="block-child-p">In the mPTP, we are interested in fitting the branch lengths of <em>each</em> maximal subtree of <em>T</em> (each species delimited in <em>T</em>) formed by branches from <em>E<sub>c</sub></em> to a <em>distinct</em> exponential distribution. Let <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-261"><span class="MJXp-mrow" id="MJXp-Span-262"><span class="MJXp-msub" id="MJXp-Span-263"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-264" style="margin-right: 0.05em;">T</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-265" style="vertical-align: -0.4em;">1</span></span><span class="MJXp-mo" id="MJXp-Span-266" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-267" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-msub" id="MJXp-Span-268"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-269" style="margin-right: 0.05em;">V</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-270" style="vertical-align: -0.4em;">1</span></span><span class="MJXp-mo" id="MJXp-Span-271" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-272"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-273" style="margin-right: 0.05em;">E</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-274" style="vertical-align: -0.4em;">1</span></span><span class="MJXp-mo" id="MJXp-Span-275" style="margin-left: 0em; margin-right: 0em;">)</span><span class="MJXp-mo" id="MJXp-Span-276" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-277"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-278" style="margin-right: 0.05em;">T</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-279" style="vertical-align: -0.4em;">2</span></span><span class="MJXp-mo" id="MJXp-Span-280" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-281" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-msub" id="MJXp-Span-282"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-283" style="margin-right: 0.05em;">V</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-284" style="vertical-align: -0.4em;">2</span></span><span class="MJXp-mo" id="MJXp-Span-285" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-286"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-287" style="margin-right: 0.05em;">E</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-288" style="vertical-align: -0.4em;">2</span></span><span class="MJXp-mo" id="MJXp-Span-289" style="margin-left: 0em; margin-right: 0em;">)</span><span class="MJXp-mo" id="MJXp-Span-290" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mo" id="MJXp-Span-291" style="margin-left: 0em; margin-right: 0em;">…</span><span class="MJXp-mo" id="MJXp-Span-292" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-293"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-294" style="margin-right: 0.05em;">T</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-295" style="vertical-align: -0.4em;">k</span></span><span class="MJXp-mo" id="MJXp-Span-296" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-297" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-msub" id="MJXp-Span-298"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-299" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-300" style="vertical-align: -0.4em;">k</span></span><span class="MJXp-mo" id="MJXp-Span-301" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-302"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-303" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-304" style="vertical-align: -0.4em;">k</span></span><span class="MJXp-mo" id="MJXp-Span-305" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-13-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-13"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>T</mi><mn>1</mn></msub><mo>=</mo><mo stretchy="false">(</mo><msub><mi>V</mi><mn>1</mn></msub><mo>,</mo><msub><mi>E</mi><mn>1</mn></msub><mo stretchy="false">)</mo><mo>,</mo><msub><mi>T</mi><mn>2</mn></msub><mo>=</mo><mo stretchy="false">(</mo><msub><mi>V</mi><mn>2</mn></msub><mo>,</mo><msub><mi>E</mi><mn>2</mn></msub><mo stretchy="false">)</mo><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>T</mi><mi>k</mi></msub><mo>=</mo><mo stretchy="false">(</mo><msub><mi>V</mi><mi>k</mi></msub><mo>,</mo><msub><mi>E</mi><mi>k</mi></msub><mo stretchy="false">)</mo></mrow></math></script></span> be the <em>maximal</em> subtrees of <em>T</em> formed exclusively by branches from <em>E<sub>c</sub></em> such that <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-306"><span class="MJXp-mrow" id="MJXp-Span-307"><span class="MJXp-msubsup" id="MJXp-Span-308"><span class="MJXp-mi" id="MJXp-Span-309" style="margin-right: 0.05em;">⊔</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-314">k</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mrow" id="MJXp-Span-310"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-311">i</span><span class="MJXp-mo" id="MJXp-Span-312">=</span><span class="MJXp-mn" id="MJXp-Span-313">1</span></span></span></span></span></span></span><span class="MJXp-msub" id="MJXp-Span-315"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-316" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-317" style="vertical-align: -0.4em;">i</span></span><span class="MJXp-mo" id="MJXp-Span-318" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-msub" id="MJXp-Span-319"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-320" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-321" style="vertical-align: -0.4em;">c</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-14-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-14"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi mathvariant="normal">⊔</mi><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></msubsup><msub><mi>E</mi><mi>i</mi></msub><mo>=</mo><msub><mi>E</mi><mi>c</mi></msub></mrow></math></script></span> and no pair <em>i</em>, <em>j</em> exists for which <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-322"><span class="MJXp-mrow" id="MJXp-Span-323"><span class="MJXp-msub" id="MJXp-Span-324"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-325" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-326" style="vertical-align: -0.4em;">i</span></span><span class="MJXp-mo" id="MJXp-Span-327" style="margin-left: 0.267em; margin-right: 0.267em;">∩</span><span class="MJXp-msub" id="MJXp-Span-328"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-329" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-330" style="vertical-align: -0.4em;">j</span></span><span class="MJXp-mo" id="MJXp-Span-331" style="margin-left: 0.333em; margin-right: 0.333em;">≠</span><span class="MJXp-mo" id="MJXp-Span-332" style="margin-left: 0em; margin-right: 0em;">∅</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-15-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-15"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>V</mi><mi>i</mi></msub><mo>∩</mo><msub><mi>V</mi><mi>j</mi></msub><mo>≠</mo><mo>∅</mo></mrow></math></script>⁠</span>. The task in mPTP is to maximize the likelihood <a id="E2" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-333"><span class="MJXp-mrow" id="MJXp-Span-334"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-335">L</span><span class="MJXp-mo" id="MJXp-Span-336" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mrow" id="MJXp-Span-337"><span class="MJXp-mo" id="MJXp-Span-338" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">(</span></span><span class="MJXp-mrow" id="MJXp-Span-339"><span class="MJXp-msubsup" id="MJXp-Span-340"><span class="MJXp-mo" id="MJXp-Span-341" style="margin-left: 0.111em; margin-right: 0.05em;">∏</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-346">k</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mrow" id="MJXp-Span-342"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-343">i</span><span class="MJXp-mo" id="MJXp-Span-344">=</span><span class="MJXp-mn" id="MJXp-Span-345">1</span></span></span></span></span></span></span><span class="MJXp-mrow" id="MJXp-Span-347"><span class="MJXp-munder" id="MJXp-Span-348"><span class="MJXp-mo" id="MJXp-Span-349" style="margin-left: 0.111em; margin-right: 0.05em;">∏</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-350" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-351">x</span><span class="MJXp-mo" id="MJXp-Span-352">∈</span><span class="MJXp-msub" id="MJXp-Span-353"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-354" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-355" style="vertical-align: -0.4em;">i</span></span></span></span><span class="MJXp-mrow" id="MJXp-Span-356"><span class="MJXp-msub" id="MJXp-Span-357"><span class="MJXp-mrow" id="MJXp-Span-358" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-359"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-361" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-360">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-362" style="vertical-align: -0.4em;">i</span></span></span></span><span class="MJXp-msup" id="MJXp-Span-363"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-364" style="margin-right: 0.05em;">e</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-365" style="vertical-align: 0.5em;"><span class="MJXp-mo" id="MJXp-Span-366">−</span><span class="MJXp-msub" id="MJXp-Span-367"><span class="MJXp-mrow" id="MJXp-Span-368" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-369"><span><span class="MJXp-over"><span class=" MJXp-script" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-371" style="margin-right: 0px; margin-left: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-370">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-372" style="vertical-align: -0.4em;">i</span></span><span class="MJXp-mo" id="MJXp-Span-373">ℓ</span><span class="MJXp-mo" id="MJXp-Span-374">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-375">x</span><span class="MJXp-mo" id="MJXp-Span-376">)</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-377" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">)</span></span></span><span class="MJXp-mo" id="MJXp-Span-378" style="margin-left: 0.267em; margin-right: 0.267em;">×</span><span class="MJXp-mrow" id="MJXp-Span-379"><span class="MJXp-mo" id="MJXp-Span-380" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">(</span></span><span class="MJXp-mrow" id="MJXp-Span-381"><span class="MJXp-munder" id="MJXp-Span-382"><span class="MJXp-mo" id="MJXp-Span-383" style="margin-left: 0.111em; margin-right: 0.05em;">∏</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-384" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-385">x</span><span class="MJXp-mo" id="MJXp-Span-386">∈</span><span class="MJXp-msub" id="MJXp-Span-387"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-388" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-389" style="vertical-align: -0.4em;">s</span></span></span></span><span class="MJXp-mrow" id="MJXp-Span-390"><span class="MJXp-msub" id="MJXp-Span-391"><span class="MJXp-mrow" id="MJXp-Span-392" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-393"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-395" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-394">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-396" style="vertical-align: -0.4em;">s</span></span></span><span class="MJXp-msup" id="MJXp-Span-397"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-398" style="margin-right: 0.05em;">e</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-399" style="vertical-align: 0.5em;"><span class="MJXp-mo" id="MJXp-Span-400">−</span><span class="MJXp-msub" id="MJXp-Span-401"><span class="MJXp-mrow" id="MJXp-Span-402" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-403"><span><span class="MJXp-over"><span class=" MJXp-script" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-405" style="margin-right: 0px; margin-left: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-404">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-406" style="vertical-align: -0.4em;">s</span></span><span class="MJXp-mo" id="MJXp-Span-407">ℓ</span><span class="MJXp-mo" id="MJXp-Span-408">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-409">x</span><span class="MJXp-mo" id="MJXp-Span-410">)</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-411" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">)</span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-16-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-16"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>L</mi><mo>=</mo><mrow><mo stretchy="true">(</mo><mrow><munderover><mo>∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>k</mi></munderover><mrow><munder><mo>∏</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>i</mi></msub></mrow></munder><mrow><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>i</mi></msub></mrow></mrow><msup><mi>e</mi><mrow><mo>−</mo><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>i</mi></msub><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo stretchy="true">)</mo></mrow><mo>×</mo><mrow><mo stretchy="true">(</mo><mrow><munder><mo>∏</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>s</mi></msub></mrow></munder><mrow><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>s</mi></msub></mrow><msup><mi>e</mi><mrow><mo>−</mo><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>s</mi></msub><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo stretchy="true">)</mo></mrow></mrow></math></script></div></div><div class="label">(2)</div></div> where <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-412"><span class="MJXp-mrow" id="MJXp-Span-413"><span class="MJXp-msub" id="MJXp-Span-414"><span class="MJXp-mrow" id="MJXp-Span-415" style="margin-right: 0.05em;"><span class="MJXp-mover" id="MJXp-Span-416"><span><span class="MJXp-over"><span class="" style="margin-bottom: -0.92em;"><span class="MJXp-mo" id="MJXp-Span-418" style="margin-left: 0px; margin-right: 0px;">ˆ</span></span><span class=""><span class="MJXp-mi MJXp-italic" id="MJXp-Span-417">λ</span></span></span></span></span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-419" style="vertical-align: -0.4em;">i</span></span><span class="MJXp-mo" id="MJXp-Span-420" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-msup" id="MJXp-Span-421"><span class="MJXp-mrow" id="MJXp-Span-422" style="margin-right: 0.05em;"><span class="MJXp-mrow" id="MJXp-Span-423"><span class="MJXp-mo" id="MJXp-Span-424" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.689em;"><span style="font-size: 3.756em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">(</span></span><span class="MJXp-mrow" id="MJXp-Span-425"><span class="MJXp-mfrac" id="MJXp-Span-426" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mn" id="MJXp-Span-427">1</span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-428"><span class="MJXp-mo" id="MJXp-Span-429" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-430"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-431" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-432" style="vertical-align: -0.4em;">i</span></span><span class="MJXp-mo" id="MJXp-Span-433" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span></span></span></span></span></span></span><span class="MJXp-munder" id="MJXp-Span-434"><span class="MJXp-mo" id="MJXp-Span-435" style="margin-left: 0.111em; margin-right: 0.05em;">∑</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-436" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-437">x</span><span class="MJXp-mo" id="MJXp-Span-438">∈</span><span class="MJXp-msub" id="MJXp-Span-439"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-440" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-441" style="vertical-align: -0.4em;">i</span></span></span></span><span class="MJXp-mo" id="MJXp-Span-442" style="margin-left: 0em; margin-right: 0em;">ℓ</span><span class="MJXp-mo" id="MJXp-Span-443" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-444">x</span><span class="MJXp-mo" id="MJXp-Span-445" style="margin-left: 0em; margin-right: 0em;">)</span></span><span class="MJXp-mo" id="MJXp-Span-446" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.689em;"><span style="font-size: 3.756em; margin-left: -0.17em;" class="MJXp-right MJXp-scale4">)</span></span></span></span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-447" style="vertical-align: 1.2em;"><span class="MJXp-mo" id="MJXp-Span-448">−</span><span class="MJXp-mn" id="MJXp-Span-449">1</span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-17-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-17"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mover accent="true"><mi>λ</mi><mo>^</mo></mover></mrow><mi>i</mi></msub><mo>=</mo><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mfrac><mn>1</mn><mrow><mo>|</mo><msub><mi>E</mi><mi>i</mi></msub><mo>|</mo></mrow></mfrac><munder><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>E</mi><mi>i</mi></msub></mrow></munder><mo>ℓ</mo><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math></script></span> for <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-450"><span class="MJXp-mrow" id="MJXp-Span-451"><span class="MJXp-mn" id="MJXp-Span-452">1</span><span class="MJXp-mo" id="MJXp-Span-453" style="margin-left: 0.333em; margin-right: 0.333em;">≤</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-454">i</span><span class="MJXp-mo" id="MJXp-Span-455" style="margin-left: 0.333em; margin-right: 0.333em;">≤</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-456">k</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-18-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-18"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>1</mn><mo>≤</mo><mi>i</mi><mo>≤</mo><mi>k</mi></mrow></math></script>⁠</span>. We refer to the <em>k</em> maximal subtree root nodes as <em>coalescent roots</em>.</div><p>Whether
 a polynomial-time algorithm exists to solve the two problems remains an
 open question. We assume that the problem is hard, and thus, we propose
 a greedy, dynamic-programming (DP) algorithm to solve it.</p><p>The algorithm visits all inner nodes of <em>T</em> in a bottom-up post-order traversal. For each node <em>u</em>, the DP computes an array of <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-457"><span class="MJXp-mrow" id="MJXp-Span-458"><span class="MJXp-mo" id="MJXp-Span-459" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-460"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-461" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-462" style="vertical-align: -0.4em;">u</span></span><span class="MJXp-mo" id="MJXp-Span-463" style="margin-left: 0.167em; margin-right: 0.167em; vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-mo" id="MJXp-Span-464" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mn" id="MJXp-Span-465">1</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-19-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-19"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>|</mo><msub><mi>E</mi><mi>u</mi></msub><mo>|</mo><mo>+</mo><mn>1</mn></mrow></math></script></span> scores, where <em>E<sub>u</sub></em> is the set of branches of subtree <em>T<sub>u</sub></em>. Entry <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-466"><span class="MJXp-mrow" id="MJXp-Span-467"><span class="MJXp-mn" id="MJXp-Span-468">0</span><span class="MJXp-mo" id="MJXp-Span-469" style="margin-left: 0.333em; margin-right: 0.333em;">≤</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-470">i</span><span class="MJXp-mo" id="MJXp-Span-471" style="margin-left: 0.333em; margin-right: 0.333em;">≤</span><span class="MJXp-mo" id="MJXp-Span-472" style="margin-left: 0.167em; margin-right: 0.167em; vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-473"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-474" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-475" style="vertical-align: -0.4em;">u</span></span><span class="MJXp-mo" id="MJXp-Span-476" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-20-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-20"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>0</mn><mo>≤</mo><mi>i</mi><mo>≤</mo><mo>|</mo><msub><mi>E</mi><mi>u</mi></msub><mo>|</mo></mrow></math></script></span> assumes that <em>T<sub>u</sub></em> contains <em>i</em> speciation branches. For the case <em>i</em> = 0, <em>T<sub>u</sub></em> is part of <em>a</em> coalescent process, while for <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-477"><span class="MJXp-mrow" id="MJXp-Span-478"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-479">i</span><span class="MJXp-mo" id="MJXp-Span-480" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mo" id="MJXp-Span-481" style="margin-left: 0.167em; margin-right: 0.167em; vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-482"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-483" style="margin-right: 0.05em;">E</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-484" style="vertical-align: -0.4em;">u</span></span><span class="MJXp-mo" id="MJXp-Span-485" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-21-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-21"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mo>|</mo><msub><mi>E</mi><mi>u</mi></msub><mo>|</mo></mrow></math></script>⁠</span>, <em>T<sub>u</sub></em> is part of <em>the</em> single speciation process. Each entry <em>i</em> &gt; 0 contains the maximization of <a href="#E2" class="sectionLink xref-sec">Equation (2)</a>, by considering only (i) the branches of subtree <em>T<sub>u</sub></em> and (ii) the set <em>S</em> of branches of <em>T</em> that have at least one end-point on the path from the root <em>r</em> to <em>u</em>. For this subset of branches, the speciation process consists of exactly <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-486"><span class="MJXp-mrow" id="MJXp-Span-487"><span class="MJXp-mo" id="MJXp-Span-488" style="margin-left: 0em; margin-right: 0em;">|</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-489">S</span><span class="MJXp-mo" id="MJXp-Span-490" style="margin-left: 0.167em; margin-right: 0.167em;">|</span><span class="MJXp-mo" id="MJXp-Span-491" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-492">i</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-22-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-22"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>|</mo><mi>S</mi><mo>|</mo><mo>+</mo><mi>i</mi></mrow></math></script></span> branches, and we only consider the coalescent processes inside <em>T<sub>u</sub></em>. We restrict ourselves to the set of branches <em>S</em> since it is infeasible to test all possible groupings of branches outside of <em>T<sub>u</sub></em>. For any <em>i</em> &gt; 0, it holds that the edges from <em>u</em> leading to its child nodes <em>v</em> and <em>w</em> belong to the speciation process, and, therefore, <em>S</em> is the smallest set of edges which, by definition, must be part of the speciation process.</p><p><a reveal-id="btx025-F1" data-open="btx025-F1" class="link link-reveal link-table xref-fig">Figure 1</a> illustrates the algorithm for a triplet of nodes <em>u</em>, <em>v</em>, <em>w</em>, and marks the set <em>S</em> with dashed lines. Entry 0 represents the null-model for <em>T<sub>u</sub></em>, that is, all branches of <em>T<sub>u</sub></em>
 belong to the coalescent process [or, equivalently, to the speciation 
process, as the two cases can not be distinguished by the (m)PTP model].
 The score (maximization of <a href="#E2" class="sectionLink xref-sec">Equation (2)</a>) for entry <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-493"><span class="MJXp-mrow" id="MJXp-Span-494"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-495">i</span><span class="MJXp-mo" id="MJXp-Span-496" style="margin-left: 0.333em; margin-right: 0.333em;">≥</span><span class="MJXp-mn" id="MJXp-Span-497">1</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-23-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-23"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>≥</mo><mn>1</mn></mrow></math></script></span> is computed from the information stored in the array entries <em>j</em> and <em>k</em> of the two child nodes <em>v</em> and <em>w</em>, by considering all combinations of <em>j</em> and <em>k</em> such that <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-498"><span class="MJXp-mrow" id="MJXp-Span-499"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-500">i</span><span class="MJXp-mo" id="MJXp-Span-501" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-502">j</span><span class="MJXp-mo" id="MJXp-Span-503" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-504">k</span><span class="MJXp-mo" id="MJXp-Span-505" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mn" id="MJXp-Span-506">2</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-24-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-24"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mi>j</mi><mo>+</mo><mi>k</mi><mo>+</mo><mn>2</mn></mrow></math></script></span> (plus two accounts for the two outgoing edges of <em>u</em>). Not all array entries are necessarily valid. For instance, entry <em>i</em> = 1 may be invalid given that node <em>u</em> has two out-going branches in the speciation process. Each entry <em>i</em> at node <em>u</em> stores the computed score, the sum of the speciation branch lengths <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-511"><span class="MJXp-mrow" id="MJXp-Span-512"><span class="MJXp-msubsup" id="MJXp-Span-513"><span class="MJXp-mo" id="MJXp-Span-514" style="margin-left: 0em; margin-right: 0.05em;">σ</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-516">i</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-515">u</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-25-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-25"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mo>σ</mo><mi>u</mi><mi>i</mi></msubsup></mrow></math></script></span> within <em>T<sub>u</sub></em>, the product <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-517"><span class="MJXp-mrow" id="MJXp-Span-518"><span class="MJXp-msubsup" id="MJXp-Span-519"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-520" style="margin-right: 0.05em;">L</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-522">i</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-521">u</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-26-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-26"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>L</mi><mi>u</mi><mi>i</mi></msubsup></mrow></math></script></span> of likelihoods of coalescent processes inside <em>T<sub>u</sub></em>, and pointers for storing which entries <em>j</em> and <em>k</em> were chosen for calculating a specific entry <em>i</em>. The first term of <a href="#E2" class="sectionLink xref-sec">Equation (2)</a> (coalescent process) is the product of <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-523"><span class="MJXp-mrow" id="MJXp-Span-524"><span class="MJXp-msubsup" id="MJXp-Span-525"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-526" style="margin-right: 0.05em;">L</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-528">j</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-527">v</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-27-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-27"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>L</mi><mi>v</mi><mi>j</mi></msubsup></mrow></math></script></span> and <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-529"><span class="MJXp-mrow" id="MJXp-Span-530"><span class="MJXp-msubsup" id="MJXp-Span-531"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-532" style="margin-right: 0.05em;">L</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-534">k</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-533">w</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-28-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-28"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>L</mi><mi>w</mi><mi>k</mi></msubsup></mrow></math></script>⁠</span>, while the second term (speciation process) is computed from the sums <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-535"><span class="MJXp-mrow" id="MJXp-Span-536"><span class="MJXp-msubsup" id="MJXp-Span-537"><span class="MJXp-mo" id="MJXp-Span-538" style="margin-left: 0em; margin-right: 0.05em;">σ</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-540">j</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-539">v</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-29-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-29"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mo>σ</mo><mi>v</mi><mi>j</mi></msubsup></mrow></math></script></span> and <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-541"><span class="MJXp-mrow" id="MJXp-Span-542"><span class="MJXp-msubsup" id="MJXp-Span-543"><span class="MJXp-mo" id="MJXp-Span-544" style="margin-left: 0em; margin-right: 0.05em;">σ</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-546">k</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-545">w</span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-30-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-30"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mo>σ</mo><mi>w</mi><mi>k</mi></msubsup></mrow></math></script></span> and the sum of branch lengths from <em>S</em>. We store the score and child node entry indices for the combination of <em>j</em> and <em>k</em> that maximize the score.</p>                        <a id="118736900" scrollto-destination="118736900"></a>
<div id="btx025-F1" data-id="btx025-F1" class="fig fig-section"><div class="label fig-label">Fig. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_btx025f1.png" alt="Visual representation of the mPTP dynamic programming algorithm. Each entry i at node u is computed from information stored at entries j and k of child nodes v and w, for all j and k such that i=j+k+2. The dashed branches denote the smallest set S of branches which, by definition, must be part of the speciation process, irrespective of the resolution of other subtrees outside Tu" path-from-xml="btx025f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/118736900/btx025f1.tif" path-from-xml="btx025f1.tif" target="_blank">Open in new tab</a><a section="118736900" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025f1.png?Expires=1585828431&amp;Signature=wO472OVxJuP28wjG4Fa20hokxYbHT03aq-gfpX4DMzu3TtT-olmwgPr12p4~pmJo~PS6EjCl-jQA0cIwdxZNyoxYglkkf7JKLkABPcS74Nzx-6Wh3X40ll1xTSJMRS1-C9U7s6HJjt-Vl8Q3BVw0loHr2rHYWEp1B4gppl-ajjsIwz0dF4B8wCA09G6gr36CQaT4G7I~vU-dsX52ccLd-j1vgMmzCHamG6ZQEOtdZDq5NLf709V6Ic~c4WSPDcPYlKYMM9yMzIQYj~-4SCfBhOxZo6Tqesg3FIptK7YlDWFWQPE3-vRbVwm8R1rILIStN3jAQaO0Xx-coMxEenzvrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=118736900&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btx025f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Visual representation of the mPTP dynamic programming algorithm. Each entry <em>i</em> at node <em>u</em> is computed from information stored at entries <em>j</em> and <em>k</em> of child nodes <em>v</em> and <em>w</em>, for all <em>j</em> and <em>k</em> such that <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-547"><span class="MJXp-mrow" id="MJXp-Span-548"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-549">i</span><span class="MJXp-mo" id="MJXp-Span-550" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-551">j</span><span class="MJXp-mo" id="MJXp-Span-552" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-553">k</span><span class="MJXp-mo" id="MJXp-Span-554" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mn" id="MJXp-Span-555">2</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-31-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-31"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mi>j</mi><mo>+</mo><mi>k</mi><mo>+</mo><mn>2</mn></mrow></math></script>⁠</span>. The dashed branches denote the smallest set <em>S</em>
 of branches which, by definition, must be part of the speciation 
process, irrespective of the resolution of other subtrees outside <em>T<sub>u</sub></em></p></div></div><div content-id="btx025-F1" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 1.</div><div class="graphic-wrap"><img class="content-image" src="m_btx025f1.png" alt="Visual representation of the mPTP dynamic programming algorithm. Each entry i at node u is computed from information stored at entries j and k of child nodes v and w, for all j and k such that i=j+k+2. The dashed branches denote the smallest set S of branches which, by definition, must be part of the speciation process, irrespective of the resolution of other subtrees outside Tu" path-from-xml="btx025f1.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/118736900/btx025f1.tif" path-from-xml="btx025f1.tif" target="_blank">Open in new tab</a><a section="118736900" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025f1.png?Expires=1585828431&amp;Signature=wO472OVxJuP28wjG4Fa20hokxYbHT03aq-gfpX4DMzu3TtT-olmwgPr12p4~pmJo~PS6EjCl-jQA0cIwdxZNyoxYglkkf7JKLkABPcS74Nzx-6Wh3X40ll1xTSJMRS1-C9U7s6HJjt-Vl8Q3BVw0loHr2rHYWEp1B4gppl-ajjsIwz0dF4B8wCA09G6gr36CQaT4G7I~vU-dsX52ccLd-j1vgMmzCHamG6ZQEOtdZDq5NLf709V6Ic~c4WSPDcPYlKYMM9yMzIQYj~-4SCfBhOxZo6Tqesg3FIptK7YlDWFWQPE3-vRbVwm8R1rILIStN3jAQaO0Xx-coMxEenzvrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=118736900&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btx025f1.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Visual representation of the mPTP dynamic programming algorithm. Each entry <em>i</em> at node <em>u</em> is computed from information stored at entries <em>j</em> and <em>k</em> of child nodes <em>v</em> and <em>w</em>, for all <em>j</em> and <em>k</em> such that <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-556"><span class="MJXp-mrow" id="MJXp-Span-557"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-558">i</span><span class="MJXp-mo" id="MJXp-Span-559" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-560">j</span><span class="MJXp-mo" id="MJXp-Span-561" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-562">k</span><span class="MJXp-mo" id="MJXp-Span-563" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mn" id="MJXp-Span-564">2</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-32-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-32"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mi>j</mi><mo>+</mo><mi>k</mi><mo>+</mo><mn>2</mn></mrow></math></script>⁠</span>. The dashed branches denote the smallest set <em>S</em>
 of branches which, by definition, must be part of the speciation 
process, irrespective of the resolution of other subtrees outside <em>T<sub>u</sub></em></p></div></div><p>Once the root node of <em>T</em> is processed, we have the likelihood values of exactly <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-565"><span class="MJXp-mrow" id="MJXp-Span-566"><span class="MJXp-mo" id="MJXp-Span-567" style="margin-left: 0em; margin-right: 0em;">|</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-568">E</span><span class="MJXp-mo" id="MJXp-Span-569" style="margin-left: 0.167em; margin-right: 0.167em;">|</span><span class="MJXp-mo" id="MJXp-Span-570" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-mn" id="MJXp-Span-571">1</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-33-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-33"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>|</mo><mi>E</mi><mo>|</mo><mo>+</mo><mn>1</mn></mrow></math></script></span> delimitations, one of which corresponds to the null-model (<span class="inline-formula">⁠<span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-572"><span class="MJXp-mrow" id="MJXp-Span-573"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-574">i</span><span class="MJXp-mo" id="MJXp-Span-575" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-576">0</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-34-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-34"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>i</mi><mo>=</mo><mn>0</mn></mrow></math></script>⁠</span>). We select the entry <em>i</em> that minimizes the <em>Akaike Information Criterion</em> score corrected for small sample size (<span class="inline-formula">⁠<span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-577"><span class="MJXp-mrow" id="MJXp-Span-578"><span class="MJXp-msub" id="MJXp-Span-579"><span class="MJXp-mrow" id="MJXp-Span-580" style="margin-right: 0.05em;"><span class="MJXp-mtext" id="MJXp-Span-581">AIC</span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-582" style="vertical-align: -0.4em;">c</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-35-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-35"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mtext>AIC</mtext></mrow><mi>c</mi></msub></mrow></math></script>⁠</span>; <a reveal-id="btx025-B28" data-open="btx025-B28" class="link link-ref link-reveal xref-bibr">McQuarrie and Tsai, 1998</a>). This way we penalize over-splitting caused by the increasing number of parameters (<span class="inline-formula">⁠<span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-583"><span class="MJXp-mrow" id="MJXp-Span-584"><span class="MJXp-msub" id="MJXp-Span-585"><span class="MJXp-mo" id="MJXp-Span-586" style="margin-left: 0em; margin-right: 0.05em;">λ</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-587" style="vertical-align: -0.4em;">1</span></span><span class="MJXp-mo" id="MJXp-Span-588" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-589"><span class="MJXp-mo" id="MJXp-Span-590" style="margin-left: 0em; margin-right: 0.05em;">λ</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-591" style="vertical-align: -0.4em;">2</span></span><span class="MJXp-mo" id="MJXp-Span-592" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mo" id="MJXp-Span-593" style="margin-left: 0em; margin-right: 0em;">…</span><span class="MJXp-mo" id="MJXp-Span-594" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-msub" id="MJXp-Span-595"><span class="MJXp-mo" id="MJXp-Span-596" style="margin-left: 0em; margin-right: 0.05em;">λ</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-597" style="vertical-align: -0.4em;">k</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-36-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-36"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mo>λ</mo><mn>1</mn></msub><mo>,</mo><msub><mo>λ</mo><mn>2</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mo>λ</mo><mi>k</mi></msub></mrow></math></script>⁠</span>). A lower <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-598"><span class="MJXp-mrow" id="MJXp-Span-599"><span class="MJXp-msub" id="MJXp-Span-600"><span class="MJXp-mrow" id="MJXp-Span-601" style="margin-right: 0.05em;"><span class="MJXp-mtext" id="MJXp-Span-602">AIC</span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-603" style="vertical-align: -0.4em;">c</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-37-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-37"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mtext>AIC</mtext></mrow><mi>c</mi></msub></mrow></math></script></span> value corresponds to a model which better explains the data. Once the best <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-604"><span class="MJXp-mrow" id="MJXp-Span-605"><span class="MJXp-msub" id="MJXp-Span-606"><span class="MJXp-mrow" id="MJXp-Span-607" style="margin-right: 0.05em;"><span class="MJXp-mtext" id="MJXp-Span-608">AIC</span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-609" style="vertical-align: -0.4em;">c</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-38-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-38"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mtext>AIC</mtext></mrow><mi>c</mi></msub></mrow></math></script></span> corrected entry <em>i</em>
 at the root has been determined, we perform a backtracking procedure 
using the stored child node pointers to retrieve the coalescent roots 
and hence obtain the species delimitation. The average asymptotic 
run-time complexity of the method is <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-610"><span class="MJXp-mrow" id="MJXp-Span-611"><span class="MJXp-mi MJXp-scr" id="MJXp-Span-612">O</span><span class="MJXp-mo" id="MJXp-Span-613" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-msup" id="MJXp-Span-614"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-615" style="margin-right: 0.05em;">n</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-616" style="vertical-align: 0.5em;">2</span></span><span class="MJXp-mo" id="MJXp-Span-617" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-39-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-39"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="script">O</mi><mo stretchy="false">(</mo><msup><mi>n</mi><mn>2</mn></msup><mo stretchy="false">)</mo></mrow></math></script></span> with a worst-case of <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-618"><span class="MJXp-mrow" id="MJXp-Span-619"><span class="MJXp-mi MJXp-scr" id="MJXp-Span-620">O</span><span class="MJXp-mo" id="MJXp-Span-621" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-msup" id="MJXp-Span-622"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-623" style="margin-right: 0.05em;">n</span><span class="MJXp-mn MJXp-script" id="MJXp-Span-624" style="vertical-align: 0.5em;">3</span></span><span class="MJXp-mo" id="MJXp-Span-625" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-40-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-40"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi mathvariant="script">O</mi><mo stretchy="false">(</mo><msup><mi>n</mi><mn>3</mn></msup><mo stretchy="false">)</mo></mrow></math></script>⁠</span>, where <em>n</em> is the number of nodes in <em>T</em> (for the proofs see Paragraph 1, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material SI</a></span>).</p>                    <h3 scrollto-destination="118736902" id="118736902" class="section-title">2.2 MCMC sampling</h3>
<div class="block-child-p">One way to assess the confidence of the 
best-scoring delimitation is to iterate through the finite set Ω of all 
possible delimitations, and compute a support value for every node <em>u</em> of the tree. This value is the sum of Akaike weights (<a reveal-id="btx025-B1" data-open="btx025-B1" class="link link-ref link-reveal xref-bibr">Akaike, 1978</a>; <a reveal-id="btx025-B8" data-open="btx025-B8" class="link link-ref link-reveal xref-bibr">Burnham and Anderson, 2002</a>) for all delimitations where <em>u</em>
 is part of the speciation process. Since this is infeasible for large 
trees, we deploy a discrete-time MCMC approach to approximate the 
probability distribution <em>π</em> on Ω that assigns an Akaike weight to each possible delimitation. Hence, by sampling according to the distribution <em>π</em>,
 we are able to approximate the support value for each node of the tree.
 At each step, we propose a new delimitation by designating a set of 
coalescent roots and compute its score. We denote the current 
delimitation as <em>θ</em> and the proposed delimitation, after applying a move to <em>θ</em>, as <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-626"><span class="MJXp-mrow" id="MJXp-Span-627"><span class="MJXp-mo" id="MJXp-Span-628" style="margin-left: 0em; margin-right: 0em;">θ</span><span class="MJXp-mo" id="MJXp-Span-629" style="margin-left: 0em; margin-right: 0.167em;">′</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-41-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-41"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>θ</mo><mo>′</mo></mrow></math></script>⁠</span>. We use the acceptance ratio <a id="E3" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-630"><span class="MJXp-mrow" id="MJXp-Span-631"><span class="MJXp-mo" id="MJXp-Span-632" style="margin-left: 0em; margin-right: 0em;">α</span><span class="MJXp-mo" id="MJXp-Span-633" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mfrac" id="MJXp-Span-634" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-635"><span class="MJXp-msubsup" id="MJXp-Span-636"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-637" style="margin-right: 0.05em;">L</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mo" id="MJXp-Span-639">′</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-638">c</span></span></span></span></span></span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-640"><span class="MJXp-msub" id="MJXp-Span-641"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-642" style="margin-right: 0.05em;">L</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-643" style="vertical-align: -0.4em;">c</span></span></span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-644" style="margin-left: 0.267em; margin-right: 0.267em;">×</span><span class="MJXp-mfrac" id="MJXp-Span-645" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-646"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-647">g</span><span class="MJXp-mo" id="MJXp-Span-648">(</span><span class="MJXp-mo" id="MJXp-Span-649">θ</span><span class="MJXp-mo" id="MJXp-Span-650">′</span><span class="MJXp-mo" id="MJXp-Span-651">→</span><span class="MJXp-mo" id="MJXp-Span-652">θ</span><span class="MJXp-mo" id="MJXp-Span-653">)</span></span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-654"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-655">g</span><span class="MJXp-mo" id="MJXp-Span-656">(</span><span class="MJXp-mo" id="MJXp-Span-657">θ</span><span class="MJXp-mo" id="MJXp-Span-658">→</span><span class="MJXp-mo" id="MJXp-Span-659">θ</span><span class="MJXp-mo" id="MJXp-Span-660">′</span><span class="MJXp-mo" id="MJXp-Span-661">)</span></span></span></span></span></span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-42-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-42"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>α</mo><mo>=</mo><mfrac><mrow><msubsup><mi>L</mi><mi>c</mi><mo>′</mo></msubsup></mrow><mrow><msub><mi>L</mi><mi>c</mi></msub></mrow></mfrac><mo>×</mo><mfrac><mrow><mi>g</mi><mo stretchy="false">(</mo><mo>θ</mo><mo>′</mo><mo>→</mo><mo>θ</mo><mo stretchy="false">)</mo></mrow><mrow><mi>g</mi><mo stretchy="false">(</mo><mo>θ</mo><mo>→</mo><mo>θ</mo><mo>′</mo><mo stretchy="false">)</mo></mrow></mfrac></mrow></math></script></div></div></div> to decide whether to keep the proposed delimitation <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-662"><span class="MJXp-mrow" id="MJXp-Span-663"><span class="MJXp-mo" id="MJXp-Span-664" style="margin-left: 0em; margin-right: 0em;">θ</span><span class="MJXp-mo" id="MJXp-Span-665" style="margin-left: 0em; margin-right: 0.167em;">′</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-43-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-43"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>θ</mo><mo>′</mo></mrow></math></script></span> or not. We assume that all delimitations are equally probable, and, therefore we use a uniform prior (cancelled out). If <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-666"><span class="MJXp-mrow" id="MJXp-Span-667"><span class="MJXp-mo" id="MJXp-Span-668" style="margin-left: 0em; margin-right: 0em;">α</span><span class="MJXp-mo" id="MJXp-Span-669" style="margin-left: 0.333em; margin-right: 0.333em;">≥</span><span class="MJXp-mn" id="MJXp-Span-670">1</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-44-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-44"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>α</mo><mo>≥</mo><mn>1</mn></mrow></math></script>⁠</span>, we accept the delimitation, otherwise we accept it with probability <em>α</em>. The term <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-671"><span class="MJXp-mrow" id="MJXp-Span-672"><span class="MJXp-msubsup" id="MJXp-Span-673"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-674" style="margin-right: 0.05em;">L</span><span class="MJXp-script-box" style="height: 1.86em; vertical-align: -0.64em;"><span class=" MJXp-script"><span><span style="margin-bottom: -0.25em;"><span class="MJXp-mo" id="MJXp-Span-676">′</span></span></span></span><span class=" MJXp-script"><span><span style="margin-top: -0.85em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-675">c</span></span></span></span></span></span><span class="MJXp-mo" id="MJXp-Span-677" style="margin-left: 0.111em; margin-right: 0.111em;">/</span><span class="MJXp-msub" id="MJXp-Span-678"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-679" style="margin-right: 0.05em;">L</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-680" style="vertical-align: -0.4em;">c</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-45-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-45"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msubsup><mi>L</mi><mi>c</mi><mo>′</mo></msubsup><mo>/</mo><msub><mi>L</mi><mi>c</mi></msub></mrow></math></script></span> is the ratio of the <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-681"><span class="MJXp-mrow" id="MJXp-Span-682"><span class="MJXp-msub" id="MJXp-Span-683"><span class="MJXp-mrow" id="MJXp-Span-684" style="margin-right: 0.05em;"><span class="MJXp-mtext" id="MJXp-Span-685">AIC</span></span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-686" style="vertical-align: -0.4em;">c</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-46-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-46"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mtext>AIC</mtext></mrow><mi>c</mi></msub></mrow></math></script></span>-corrected likelihood values of <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-687"><span class="MJXp-mrow" id="MJXp-Span-688"><span class="MJXp-mo" id="MJXp-Span-689" style="margin-left: 0em; margin-right: 0em;">θ</span><span class="MJXp-mo" id="MJXp-Span-690" style="margin-left: 0em; margin-right: 0.167em;">′</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-47-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-47"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>θ</mo><mo>′</mo></mrow></math></script></span> over <em>θ</em>, and <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-691"><span class="MJXp-mrow" id="MJXp-Span-692"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-693">g</span><span class="MJXp-mo" id="MJXp-Span-694" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-695">x</span><span class="MJXp-mo" id="MJXp-Span-696" style="margin-left: 0.333em; margin-right: 0.333em;">→</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-697">y</span><span class="MJXp-mo" id="MJXp-Span-698" style="margin-left: 0em; margin-right: 0em;">)</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-48-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-48"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>g</mi><mo stretchy="false">(</mo><mi>x</mi><mo>→</mo><mi>y</mi><mo stretchy="false">)</mo></mrow></math></script></span> is the transition probability from one delimitation, <em>x</em>, to any other delimitation <em>y</em>. Given <em>θ</em>, we propose <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-699"><span class="MJXp-mrow" id="MJXp-Span-700"><span class="MJXp-mo" id="MJXp-Span-701" style="margin-left: 0em; margin-right: 0em;">θ</span><span class="MJXp-mo" id="MJXp-Span-702" style="margin-left: 0em; margin-right: 0.167em;">′</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-49-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-49"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>θ</mo><mo>′</mo></mrow></math></script></span>
 by applying one of two possible moves with equal probability. One of 
the two moves selects a coalescent process at random and splits it into 
two new coalescent processes, by placing its root node (and its two 
out-going branches) into the speciation process. The other type of move 
performs the reverse; a randomly selected node <em>u</em>, whose two 
child nodes are coalescent roots, is removed from the speciation 
process, effectively merging the two coalescent processes into a new 
one. The method’s run-time is linear to the number of MCMC steps and 
independent of the tree size. A full description of the MCMC algorithm 
appears in Paragraph 3 of <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material SI</a></span>.</div><p>It
 is considered good practice that MCMC results are accompanied by a 
critical convergence assessment. Arguably, a good way of accomplishing 
this is to compare samples obtained from independent MCMC runs. We use 
the average standard deviation of delimitation support values (ASDDSV) 
for quantifying the similarity among such samples. Inspired by the 
average standard deviation of split frequencies (ASDSF) technique from 
phylogenetic inference (<a reveal-id="btx025-B37" data-open="btx025-B37" class="link link-ref link-reveal xref-bibr">Ronquist <em>et al.</em>, 2012</a>),
 we calculate the ASDDSV by averaging the standard deviation of per-node
 delimitation support values across multiple independent MCMC runs that 
explore the delimitation space. Each such MCMC run starts from a 
randomly generated delimitation. Similarly to ASDSF, ASDDSV approaches 
zero as runs converge to the same distribution of delimitations.</p><div class="block-child-p">ASDDSV
 is useful for monitoring that runs converge to the same distribution. 
To further assess the agreement between the sampled delimitations and 
the best scoring delimitation on large datasets, where visual inspection
 is impractical, we introduce the <em>average support value</em> (ASV), that is, <a id="E4" class="jumplink-placeholder">&nbsp;</a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-703"><span class="MJXp-mrow" id="MJXp-Span-704"><span class="MJXp-mfrac" id="MJXp-Span-705" style="vertical-align: 0.25em;"><span class="MJXp-box MJXp-script"><span class="MJXp-mn" id="MJXp-Span-706">1</span></span><span class="MJXp-box" style="margin-top: -0.9em;"><span class="MJXp-denom"><span><span class="MJXp-rule" style="height: 1em; border-top: medium none; border-bottom: 1px solid; margin: 0.1em 0px;"></span></span><span><span class="MJXp-box MJXp-script"><span class="MJXp-mrow" id="MJXp-Span-707"><span class="MJXp-mo" id="MJXp-Span-708" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-709"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-710" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-711" style="vertical-align: -0.4em;">T</span></span><span class="MJXp-mo" id="MJXp-Span-712" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-mo" id="MJXp-Span-713">+</span><span class="MJXp-mo" id="MJXp-Span-714" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span><span class="MJXp-msub" id="MJXp-Span-715"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-716" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-717" style="vertical-align: -0.4em;">R</span></span><span class="MJXp-mo" id="MJXp-Span-718" style="vertical-align: -0.244em;"><span style="font-size: 1.978em; margin-left: -0.013em;" class="MJXp-right MJXp-scale7">|</span></span></span></span></span></span></span></span><span class="MJXp-mrow" id="MJXp-Span-719"><span class="MJXp-mo" id="MJXp-Span-720" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">(</span></span><span class="MJXp-mrow" id="MJXp-Span-721"><span class="MJXp-munder" id="MJXp-Span-722"><span class="MJXp-mo" id="MJXp-Span-723" style="margin-left: 0.111em; margin-right: 0.05em;">∑</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-724" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-725">u</span><span class="MJXp-mo" id="MJXp-Span-726">∈</span><span class="MJXp-msub" id="MJXp-Span-727"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-728" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-729" style="vertical-align: -0.4em;">T</span></span></span></span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-730">f</span><span class="MJXp-mo" id="MJXp-Span-731" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-732">u</span><span class="MJXp-mo" id="MJXp-Span-733" style="margin-left: 0em; margin-right: 0em;">)</span><span class="MJXp-mo" id="MJXp-Span-734" style="margin-left: 0.267em; margin-right: 0.267em;">+</span><span class="MJXp-munder" id="MJXp-Span-735"><span class="MJXp-mo" id="MJXp-Span-736" style="margin-left: 0.111em; margin-right: 0.05em;">∑</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-737" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-738">u</span><span class="MJXp-mo" id="MJXp-Span-739">∈</span><span class="MJXp-msub" id="MJXp-Span-740"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-741" style="margin-right: 0.05em;">V</span><span class="MJXp-mi MJXp-italic MJXp-script" id="MJXp-Span-742" style="vertical-align: -0.4em;">R</span></span></span></span><span class="MJXp-mn" id="MJXp-Span-743">1</span><span class="MJXp-mo" id="MJXp-Span-744" style="margin-left: 0.267em; margin-right: 0.267em;">−</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-745">f</span><span class="MJXp-mo" id="MJXp-Span-746" style="margin-left: 0em; margin-right: 0em;">(</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-747">u</span><span class="MJXp-mo" id="MJXp-Span-748" style="margin-left: 0em; margin-right: 0em;">)</span></span><span class="MJXp-mo" id="MJXp-Span-749" style="margin-left: 0em; margin-right: 0em; vertical-align: -0.396em;"><span style="font-size: 2.582em; margin-left: -0.13em;" class="MJXp-right MJXp-scale5">)</span></span></span><span class="MJXp-mo" id="MJXp-Span-750" style="margin-left: 0em; margin-right: 0.222em;">,</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-50-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-50"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mfrac><mn>1</mn><mrow><mo>|</mo><msub><mi>V</mi><mi>T</mi></msub><mo>|</mo><mo>+</mo><mo>|</mo><msub><mi>V</mi><mi>R</mi></msub><mo>|</mo></mrow></mfrac><mrow><mo stretchy="true">(</mo><mrow><munder><mo>∑</mo><mrow><mi>u</mi><mo>∈</mo><msub><mi>V</mi><mi>T</mi></msub></mrow></munder><mi>f</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo><mo>+</mo><munder><mo>∑</mo><mrow><mi>u</mi><mo>∈</mo><msub><mi>V</mi><mi>R</mi></msub></mrow></munder><mn>1</mn><mo>−</mo><mi>f</mi><mo stretchy="false">(</mo><mi>u</mi><mo stretchy="false">)</mo></mrow><mo stretchy="true">)</mo></mrow><mo>,</mo></mrow></math></script></div></div></div> where <em>V<sub>T</sub></em> is the set of tip nodes within the speciation process, <em>V<sub>R</sub></em> is the set of coalescent roots, and <em>f</em>(<em>u</em>) the support value of node <em>u</em>. The closer ASV is to one, the better the support values agree with the ML delimitation.</div>                    <h2 scrollto-destination="118736906" id="118736906" class="section-title">3 Experimental setup</h2>
<p>For the evaluation of mPTP, we use empirical datasets that comprise a
 number of varying parameters (substitution rate, geographic range, 
genetic diversity etc.), which often occur in practice and are difficult
 to reproduce with simulations. We retrieved all datasets from the 
Barcode of Life Database (BOLD, <a class="link link-uri openInAnotherWindow" href="http://www.boldsystems.org/" target="_blank">http://www.boldsystems.org/</a>),
 the largest barcode library for eukaryotes. For each of the datasets, 
we inferred the putative species with mPTP and four other methods that 
model speciation on the basis of genetic distance (including the 
original PTP method), and assess their accuracy with respect to the 
current taxonomy (available in BOLD). Finally, we avoided comparisons 
with time-based delimitation methods (<a reveal-id="btx025-B15" data-open="btx025-B15" class="link link-ref link-reveal xref-bibr">Fujisawa and Barraclough, 2013</a>; <a reveal-id="btx025-B23" data-open="btx025-B23" class="link link-ref link-reveal xref-bibr">Jones <em>et al.</em>, 2015</a>; <a reveal-id="btx025-B46" data-open="btx025-B46" class="link link-ref link-reveal xref-bibr">Yang and Rannala, 2014</a>)
 which are time consuming and heavily dependent on the factorization 
accuracy of branch lengths into time and evolutionary rate.</p>                    <h3 scrollto-destination="118736908" id="118736908" class="section-title">3.1 Empirical datasets</h3>
<p>We retrieved 24 empirical genus level datasets (<a reveal-id="btx025-T1" data-open="btx025-T1" class="link link-reveal link-table xref-fig">Table 1</a>)
 that cover six of the most species rich animal Phyla in BOLD 
(Arthropoda, Annelida, Chordata, Echinodermata, Platyhelminthes and 
Cnidaria). This allows us to evaluate the efficiency of mPTP given a 
wide range of organisms with diverse biological traits and evolutionary 
histories. The majority of datasets (14 out of 24 datasets) belong to 
the Arthropoda which is the most species-rich animal group, and the most
 common group PTP has been applied to so far (in 61 out of 110 
citations). Within Arthropoda, we mainly focus on insects (8 out 14 
datasets), which represents over 90% of all animals. The rest of the 
empirical datasets spans the remaining five phyla. </p><div content-id="btx025-T1" class="table-modal" style="display: block;"><div class="table-wrap" content-id="btx025-T1"><div id="btx025-T1" data-id="btx025-T1" class="table-wrap-title"><div class="label">Table 1.</div><div class="caption"><p>Main characteristics of empirical test datasets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus&nbsp;</th><th>Phylum&nbsp;</th><th>NSp&nbsp;</th><th>NMSp&nbsp;</th><th>NSeq&nbsp;</th><th>AL&nbsp;</th><th>APD (%)&nbsp;</th></tr></thead><tbody><tr><td><em>Amynthas</em>&nbsp;</td><td>Annelida&nbsp;</td><td>43&nbsp;</td><td>28&nbsp;</td><td>265&nbsp;</td><td>1539&nbsp;</td><td>16.7&nbsp;</td></tr><tr><td><em>Phyllotreta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>16&nbsp;</td><td>14&nbsp;</td><td>195&nbsp;</td><td>659&nbsp;</td><td>11.8&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>84&nbsp;</td><td>60&nbsp;</td><td>399&nbsp;</td><td>1076&nbsp;</td><td>14.2&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>15&nbsp;</td><td>11&nbsp;</td><td>493&nbsp;</td><td>713&nbsp;</td><td>10.2&nbsp;</td></tr><tr><td><em>Digrammia</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>20&nbsp;</td><td>17&nbsp;</td><td>286&nbsp;</td><td>659&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><em>Carabus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>49&nbsp;</td><td>34&nbsp;</td><td>514&nbsp;</td><td>880&nbsp;</td><td>11.5&nbsp;</td></tr><tr><td><em>Bembidion</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>97&nbsp;</td><td>80&nbsp;</td><td>532&nbsp;</td><td>1495&nbsp;</td><td>12.9&nbsp;</td></tr><tr><td><em>Calcinus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>29&nbsp;</td><td>24&nbsp;</td><td>641&nbsp;</td><td>679&nbsp;</td><td>16.6&nbsp;</td></tr><tr><td><em>Xysticus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>33&nbsp;</td><td>861&nbsp;</td><td>1238&nbsp;</td><td>8.9&nbsp;</td></tr><tr><td><em>Gammarus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>53&nbsp;</td><td>27&nbsp;</td><td>755&nbsp;</td><td>1082&nbsp;</td><td>21.4&nbsp;</td></tr><tr><td><em>Clubiona</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>32&nbsp;</td><td>1127&nbsp;</td><td>1256&nbsp;</td><td>8.4&nbsp;</td></tr><tr><td><em>Culicoides</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>100&nbsp;</td><td>76&nbsp;</td><td>1252&nbsp;</td><td>1485&nbsp;</td><td>17.5&nbsp;</td></tr><tr><td><em>Balanus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>4&nbsp;</td><td>3&nbsp;</td><td>1775&nbsp;</td><td>1548&nbsp;</td><td>2.3&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>148&nbsp;</td><td>109&nbsp;</td><td>2303&nbsp;</td><td>1589&nbsp;</td><td>11.2&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>121&nbsp;</td><td>68&nbsp;</td><td>2741&nbsp;</td><td>1544&nbsp;</td><td>10.5&nbsp;</td></tr><tr><td><em>Anolis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>41&nbsp;</td><td>35&nbsp;</td><td>181&nbsp;</td><td>709&nbsp;</td><td>18.9&nbsp;</td></tr><tr><td><em>Coryphopterus</em>&nbsp;</td><td>Chordata&nbsp;</td><td>12&nbsp;</td><td>10&nbsp;</td><td>282&nbsp;</td><td>658&nbsp;</td><td>14.7&nbsp;</td></tr><tr><td><em>Myotis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>54&nbsp;</td><td>37&nbsp;</td><td>789&nbsp;</td><td>1542&nbsp;</td><td>11.7&nbsp;</td></tr><tr><td><em>Cyanea</em>&nbsp;</td><td>Cnidaria&nbsp;</td><td>5&nbsp;</td><td>4&nbsp;</td><td>92&nbsp;</td><td>1002&nbsp;</td><td>12.7&nbsp;</td></tr><tr><td><em>Ophiura</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>13&nbsp;</td><td>8&nbsp;</td><td>75&nbsp;</td><td>1605&nbsp;</td><td>16.8&nbsp;</td></tr><tr><td><em>Holothuria</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>18&nbsp;</td><td>12&nbsp;</td><td>355&nbsp;</td><td>1553&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><em>Mopalia</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>20&nbsp;</td><td>11&nbsp;</td><td>304&nbsp;</td><td>708&nbsp;</td><td>10.9&nbsp;</td></tr><tr><td><em>Rhagada</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>24&nbsp;</td><td>12&nbsp;</td><td>686&nbsp;</td><td>675&nbsp;</td><td>11.1&nbsp;</td></tr><tr><td><em>Echinococcus</em>&nbsp;</td><td>Platyhelminthes&nbsp;</td><td>7&nbsp;</td><td>5&nbsp;</td><td>316&nbsp;</td><td>1608&nbsp;</td><td>5.1&nbsp;</td></tr></tbody></table></div><div content-id="btx025-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus&nbsp;</th><th>Phylum&nbsp;</th><th>NSp&nbsp;</th><th>NMSp&nbsp;</th><th>NSeq&nbsp;</th><th>AL&nbsp;</th><th>APD (%)&nbsp;</th></tr></thead><tbody><tr><td><em>Amynthas</em>&nbsp;</td><td>Annelida&nbsp;</td><td>43&nbsp;</td><td>28&nbsp;</td><td>265&nbsp;</td><td>1539&nbsp;</td><td>16.7&nbsp;</td></tr><tr><td><em>Phyllotreta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>16&nbsp;</td><td>14&nbsp;</td><td>195&nbsp;</td><td>659&nbsp;</td><td>11.8&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>84&nbsp;</td><td>60&nbsp;</td><td>399&nbsp;</td><td>1076&nbsp;</td><td>14.2&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>15&nbsp;</td><td>11&nbsp;</td><td>493&nbsp;</td><td>713&nbsp;</td><td>10.2&nbsp;</td></tr><tr><td><em>Digrammia</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>20&nbsp;</td><td>17&nbsp;</td><td>286&nbsp;</td><td>659&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><em>Carabus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>49&nbsp;</td><td>34&nbsp;</td><td>514&nbsp;</td><td>880&nbsp;</td><td>11.5&nbsp;</td></tr><tr><td><em>Bembidion</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>97&nbsp;</td><td>80&nbsp;</td><td>532&nbsp;</td><td>1495&nbsp;</td><td>12.9&nbsp;</td></tr><tr><td><em>Calcinus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>29&nbsp;</td><td>24&nbsp;</td><td>641&nbsp;</td><td>679&nbsp;</td><td>16.6&nbsp;</td></tr><tr><td><em>Xysticus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>33&nbsp;</td><td>861&nbsp;</td><td>1238&nbsp;</td><td>8.9&nbsp;</td></tr><tr><td><em>Gammarus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>53&nbsp;</td><td>27&nbsp;</td><td>755&nbsp;</td><td>1082&nbsp;</td><td>21.4&nbsp;</td></tr><tr><td><em>Clubiona</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>32&nbsp;</td><td>1127&nbsp;</td><td>1256&nbsp;</td><td>8.4&nbsp;</td></tr><tr><td><em>Culicoides</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>100&nbsp;</td><td>76&nbsp;</td><td>1252&nbsp;</td><td>1485&nbsp;</td><td>17.5&nbsp;</td></tr><tr><td><em>Balanus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>4&nbsp;</td><td>3&nbsp;</td><td>1775&nbsp;</td><td>1548&nbsp;</td><td>2.3&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>148&nbsp;</td><td>109&nbsp;</td><td>2303&nbsp;</td><td>1589&nbsp;</td><td>11.2&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>121&nbsp;</td><td>68&nbsp;</td><td>2741&nbsp;</td><td>1544&nbsp;</td><td>10.5&nbsp;</td></tr><tr><td><em>Anolis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>41&nbsp;</td><td>35&nbsp;</td><td>181&nbsp;</td><td>709&nbsp;</td><td>18.9&nbsp;</td></tr><tr><td><em>Coryphopterus</em>&nbsp;</td><td>Chordata&nbsp;</td><td>12&nbsp;</td><td>10&nbsp;</td><td>282&nbsp;</td><td>658&nbsp;</td><td>14.7&nbsp;</td></tr><tr><td><em>Myotis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>54&nbsp;</td><td>37&nbsp;</td><td>789&nbsp;</td><td>1542&nbsp;</td><td>11.7&nbsp;</td></tr><tr><td><em>Cyanea</em>&nbsp;</td><td>Cnidaria&nbsp;</td><td>5&nbsp;</td><td>4&nbsp;</td><td>92&nbsp;</td><td>1002&nbsp;</td><td>12.7&nbsp;</td></tr><tr><td><em>Ophiura</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>13&nbsp;</td><td>8&nbsp;</td><td>75&nbsp;</td><td>1605&nbsp;</td><td>16.8&nbsp;</td></tr><tr><td><em>Holothuria</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>18&nbsp;</td><td>12&nbsp;</td><td>355&nbsp;</td><td>1553&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><em>Mopalia</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>20&nbsp;</td><td>11&nbsp;</td><td>304&nbsp;</td><td>708&nbsp;</td><td>10.9&nbsp;</td></tr><tr><td><em>Rhagada</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>24&nbsp;</td><td>12&nbsp;</td><td>686&nbsp;</td><td>675&nbsp;</td><td>11.1&nbsp;</td></tr><tr><td><em>Echinococcus</em>&nbsp;</td><td>Platyhelminthes&nbsp;</td><td>7&nbsp;</td><td>5&nbsp;</td><td>316&nbsp;</td><td>1608&nbsp;</td><td>5.1&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>NSp,
 number of species; NMSp, number of monophyletic species; NSeq, number 
of sequences; AL, alignment length; APD, average P-distance.</p></div></div></div></div></div><div class="table-wrap" content-id="btx025-T1"><div id="btx025-T1" data-id="btx025-T1" class="table-wrap-title"><div class="label">Table 1.</div><div class="caption"><p>Main characteristics of empirical test datasets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus&nbsp;</th><th>Phylum&nbsp;</th><th>NSp&nbsp;</th><th>NMSp&nbsp;</th><th>NSeq&nbsp;</th><th>AL&nbsp;</th><th>APD (%)&nbsp;</th></tr></thead><tbody><tr><td><em>Amynthas</em>&nbsp;</td><td>Annelida&nbsp;</td><td>43&nbsp;</td><td>28&nbsp;</td><td>265&nbsp;</td><td>1539&nbsp;</td><td>16.7&nbsp;</td></tr><tr><td><em>Phyllotreta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>16&nbsp;</td><td>14&nbsp;</td><td>195&nbsp;</td><td>659&nbsp;</td><td>11.8&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>84&nbsp;</td><td>60&nbsp;</td><td>399&nbsp;</td><td>1076&nbsp;</td><td>14.2&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>15&nbsp;</td><td>11&nbsp;</td><td>493&nbsp;</td><td>713&nbsp;</td><td>10.2&nbsp;</td></tr><tr><td><em>Digrammia</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>20&nbsp;</td><td>17&nbsp;</td><td>286&nbsp;</td><td>659&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><em>Carabus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>49&nbsp;</td><td>34&nbsp;</td><td>514&nbsp;</td><td>880&nbsp;</td><td>11.5&nbsp;</td></tr><tr><td><em>Bembidion</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>97&nbsp;</td><td>80&nbsp;</td><td>532&nbsp;</td><td>1495&nbsp;</td><td>12.9&nbsp;</td></tr><tr><td><em>Calcinus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>29&nbsp;</td><td>24&nbsp;</td><td>641&nbsp;</td><td>679&nbsp;</td><td>16.6&nbsp;</td></tr><tr><td><em>Xysticus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>33&nbsp;</td><td>861&nbsp;</td><td>1238&nbsp;</td><td>8.9&nbsp;</td></tr><tr><td><em>Gammarus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>53&nbsp;</td><td>27&nbsp;</td><td>755&nbsp;</td><td>1082&nbsp;</td><td>21.4&nbsp;</td></tr><tr><td><em>Clubiona</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>32&nbsp;</td><td>1127&nbsp;</td><td>1256&nbsp;</td><td>8.4&nbsp;</td></tr><tr><td><em>Culicoides</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>100&nbsp;</td><td>76&nbsp;</td><td>1252&nbsp;</td><td>1485&nbsp;</td><td>17.5&nbsp;</td></tr><tr><td><em>Balanus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>4&nbsp;</td><td>3&nbsp;</td><td>1775&nbsp;</td><td>1548&nbsp;</td><td>2.3&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>148&nbsp;</td><td>109&nbsp;</td><td>2303&nbsp;</td><td>1589&nbsp;</td><td>11.2&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>121&nbsp;</td><td>68&nbsp;</td><td>2741&nbsp;</td><td>1544&nbsp;</td><td>10.5&nbsp;</td></tr><tr><td><em>Anolis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>41&nbsp;</td><td>35&nbsp;</td><td>181&nbsp;</td><td>709&nbsp;</td><td>18.9&nbsp;</td></tr><tr><td><em>Coryphopterus</em>&nbsp;</td><td>Chordata&nbsp;</td><td>12&nbsp;</td><td>10&nbsp;</td><td>282&nbsp;</td><td>658&nbsp;</td><td>14.7&nbsp;</td></tr><tr><td><em>Myotis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>54&nbsp;</td><td>37&nbsp;</td><td>789&nbsp;</td><td>1542&nbsp;</td><td>11.7&nbsp;</td></tr><tr><td><em>Cyanea</em>&nbsp;</td><td>Cnidaria&nbsp;</td><td>5&nbsp;</td><td>4&nbsp;</td><td>92&nbsp;</td><td>1002&nbsp;</td><td>12.7&nbsp;</td></tr><tr><td><em>Ophiura</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>13&nbsp;</td><td>8&nbsp;</td><td>75&nbsp;</td><td>1605&nbsp;</td><td>16.8&nbsp;</td></tr><tr><td><em>Holothuria</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>18&nbsp;</td><td>12&nbsp;</td><td>355&nbsp;</td><td>1553&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><em>Mopalia</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>20&nbsp;</td><td>11&nbsp;</td><td>304&nbsp;</td><td>708&nbsp;</td><td>10.9&nbsp;</td></tr><tr><td><em>Rhagada</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>24&nbsp;</td><td>12&nbsp;</td><td>686&nbsp;</td><td>675&nbsp;</td><td>11.1&nbsp;</td></tr><tr><td><em>Echinococcus</em>&nbsp;</td><td>Platyhelminthes&nbsp;</td><td>7&nbsp;</td><td>5&nbsp;</td><td>316&nbsp;</td><td>1608&nbsp;</td><td>5.1&nbsp;</td></tr></tbody></table></div><div content-id="btx025-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus&nbsp;</th><th>Phylum&nbsp;</th><th>NSp&nbsp;</th><th>NMSp&nbsp;</th><th>NSeq&nbsp;</th><th>AL&nbsp;</th><th>APD (%)&nbsp;</th></tr></thead><tbody><tr><td><em>Amynthas</em>&nbsp;</td><td>Annelida&nbsp;</td><td>43&nbsp;</td><td>28&nbsp;</td><td>265&nbsp;</td><td>1539&nbsp;</td><td>16.7&nbsp;</td></tr><tr><td><em>Phyllotreta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>16&nbsp;</td><td>14&nbsp;</td><td>195&nbsp;</td><td>659&nbsp;</td><td>11.8&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>84&nbsp;</td><td>60&nbsp;</td><td>399&nbsp;</td><td>1076&nbsp;</td><td>14.2&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>15&nbsp;</td><td>11&nbsp;</td><td>493&nbsp;</td><td>713&nbsp;</td><td>10.2&nbsp;</td></tr><tr><td><em>Digrammia</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>20&nbsp;</td><td>17&nbsp;</td><td>286&nbsp;</td><td>659&nbsp;</td><td>6.2&nbsp;</td></tr><tr><td><em>Carabus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>49&nbsp;</td><td>34&nbsp;</td><td>514&nbsp;</td><td>880&nbsp;</td><td>11.5&nbsp;</td></tr><tr><td><em>Bembidion</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>97&nbsp;</td><td>80&nbsp;</td><td>532&nbsp;</td><td>1495&nbsp;</td><td>12.9&nbsp;</td></tr><tr><td><em>Calcinus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>29&nbsp;</td><td>24&nbsp;</td><td>641&nbsp;</td><td>679&nbsp;</td><td>16.6&nbsp;</td></tr><tr><td><em>Xysticus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>33&nbsp;</td><td>861&nbsp;</td><td>1238&nbsp;</td><td>8.9&nbsp;</td></tr><tr><td><em>Gammarus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>53&nbsp;</td><td>27&nbsp;</td><td>755&nbsp;</td><td>1082&nbsp;</td><td>21.4&nbsp;</td></tr><tr><td><em>Clubiona</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>35&nbsp;</td><td>32&nbsp;</td><td>1127&nbsp;</td><td>1256&nbsp;</td><td>8.4&nbsp;</td></tr><tr><td><em>Culicoides</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>100&nbsp;</td><td>76&nbsp;</td><td>1252&nbsp;</td><td>1485&nbsp;</td><td>17.5&nbsp;</td></tr><tr><td><em>Balanus</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>4&nbsp;</td><td>3&nbsp;</td><td>1775&nbsp;</td><td>1548&nbsp;</td><td>2.3&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>148&nbsp;</td><td>109&nbsp;</td><td>2303&nbsp;</td><td>1589&nbsp;</td><td>11.2&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>Arthropoda&nbsp;</td><td>121&nbsp;</td><td>68&nbsp;</td><td>2741&nbsp;</td><td>1544&nbsp;</td><td>10.5&nbsp;</td></tr><tr><td><em>Anolis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>41&nbsp;</td><td>35&nbsp;</td><td>181&nbsp;</td><td>709&nbsp;</td><td>18.9&nbsp;</td></tr><tr><td><em>Coryphopterus</em>&nbsp;</td><td>Chordata&nbsp;</td><td>12&nbsp;</td><td>10&nbsp;</td><td>282&nbsp;</td><td>658&nbsp;</td><td>14.7&nbsp;</td></tr><tr><td><em>Myotis</em>&nbsp;</td><td>Chordata&nbsp;</td><td>54&nbsp;</td><td>37&nbsp;</td><td>789&nbsp;</td><td>1542&nbsp;</td><td>11.7&nbsp;</td></tr><tr><td><em>Cyanea</em>&nbsp;</td><td>Cnidaria&nbsp;</td><td>5&nbsp;</td><td>4&nbsp;</td><td>92&nbsp;</td><td>1002&nbsp;</td><td>12.7&nbsp;</td></tr><tr><td><em>Ophiura</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>13&nbsp;</td><td>8&nbsp;</td><td>75&nbsp;</td><td>1605&nbsp;</td><td>16.8&nbsp;</td></tr><tr><td><em>Holothuria</em>&nbsp;</td><td>Echinodermata&nbsp;</td><td>18&nbsp;</td><td>12&nbsp;</td><td>355&nbsp;</td><td>1553&nbsp;</td><td>13.6&nbsp;</td></tr><tr><td><em>Mopalia</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>20&nbsp;</td><td>11&nbsp;</td><td>304&nbsp;</td><td>708&nbsp;</td><td>10.9&nbsp;</td></tr><tr><td><em>Rhagada</em>&nbsp;</td><td>Mollusca&nbsp;</td><td>24&nbsp;</td><td>12&nbsp;</td><td>686&nbsp;</td><td>675&nbsp;</td><td>11.1&nbsp;</td></tr><tr><td><em>Echinococcus</em>&nbsp;</td><td>Platyhelminthes&nbsp;</td><td>7&nbsp;</td><td>5&nbsp;</td><td>316&nbsp;</td><td>1608&nbsp;</td><td>5.1&nbsp;</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>NSp,
 number of species; NMSp, number of monophyletic species; NSeq, number 
of sequences; AL, alignment length; APD, average P-distance.</p></div></div></div></div><p></p><p>All datasets comprise the 5′ end of the common animal barcoding gene “Cytochrome Oxidase subunit I” (COI-5P) (<a reveal-id="btx025-B20" data-open="btx025-B20" class="link link-ref link-reveal xref-bibr">Hebert <em>et al.</em>, 2003</a>). The number of taxonomic species in the 24 datasets ranged from four (<em>Balanus</em>) to 148 (<em>Drosophila</em>), while the number of sequences from 75 (<em>Ophiura</em>) to 2741 (<em>Anopheles</em>). The number of sequences per species ranged between two and the extreme case of 1763 for <em>Balanus glandula</em>,
 reflecting situations where some species might be readily and others 
rarely available. Finally, the geographical distribution of the datasets
 also varied from the global scale (e.g., <em>Anopheles</em> and <em>Drosophila</em>) to the local scale (e.g., the <em>Anolis</em> samples originate from the islands and surrounding shores of the Caribbean sea).</p>                    <h3 scrollto-destination="118736911" id="118736911" class="section-title">3.2 Putative species delimitation</h3>
<p>The sequence files obtained from the BOLD database were preprocessed 
to remove three factors that interfere with the assessment of the 
delimitation methods. First, the efficiency of each method is measured 
by comparing the delimited to the taxonomic species (see Section 3.3). 
Therefore, it was necessary to remove sequences of ambiguous taxonomic 
assignment. Second, we removed duplicate sequences, using the “-f c” 
option of RAxML version 8.1.17 (<a reveal-id="btx025-B39" data-open="btx025-B39" class="link link-ref link-reveal xref-bibr">Stamatakis, 2014</a>),
 to avoid unnecessary computations. Finally, for all methods that infer 
the shifting point among speciation and coalescent based on the data 
(e.g., <a reveal-id="btx025-B33" data-open="btx025-B33" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>),
 it is important to assume adequate representation of both processes. 
Therefore, we removed singleton species that could be a potential source
 of error in this framework (i.e. the lack of multiple sequences per 
species will inevitably decrease the distinctiveness of the two 
evolutionary processes and consequently the identification of the 
shifting point from one to the other).</p><p>For each dataset, we 
inferred putative species using the two PTP models and we compared the 
results to three popular and well-established distance-based methods; 
ABGD (<a reveal-id="btx025-B33" data-open="btx025-B33" class="link link-ref link-reveal xref-bibr">Puillandre <em>et al.</em>, 2012</a>), Usearch (<a reveal-id="btx025-B13" data-open="btx025-B13" class="link link-ref link-reveal xref-bibr">Edgar, 2010</a>) and Crop (<a reveal-id="btx025-B18" data-open="btx025-B18" class="link link-ref link-reveal xref-bibr">Hao <em>et al.</em>, 2011</a>).
 The input format of the data and parameters differ among the five 
methods. The two PTP versions require a rooted phylogenetic tree. 
Therefore, we used Mafft v7.123 (<a reveal-id="btx025-B24" data-open="btx025-B24" class="link link-ref link-reveal xref-bibr">Katoh and Standley, 2013</a>)
 to align the sequences of each dataset, and subsequently RAxML under 
the default algorithm and the GTR + Γ model, to infer an ML tree. To 
root each tree, we chose outgroup taxa based on previously published 
phylogenies. For the genera without such prior phylogenetic knowledge, 
we selected a representative species of a genus belonging to the same 
family or tribe given the taxonomy in BOLD (NCBI Accession Numbers of 
the ingroup and the outgroup sequences are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material SII</a></span>).
 Moreover, we implemented a method that identifies and ignores branches 
that result from identical sequences, during the delimitation process 
(for more details see Section 2 and <a reveal-id="btx025-F1" data-open="btx025-F1" class="link link-reveal link-table xref-fig">Figs 1</a> and <a reveal-id="btx025-F2" data-open="btx025-F2" class="link link-reveal link-table xref-fig">2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material SI</a></span>).</p>                        <a id="118736914" scrollto-destination="118736914"></a>
<div id="btx025-F2" data-id="btx025-F2" class="fig fig-section"><div class="label fig-label">Fig. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_btx025f2.png" alt="Average performance over all datasets of the five delimitation methods (mPTP, PTP, Usearch, Crop and ABGD) for the (A) number of species, (B) F-scores and (C) number of RTS" path-from-xml="btx025f2.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/118736914/btx025f2.tif" path-from-xml="btx025f2.tif" target="_blank">Open in new tab</a><a section="118736914" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025f2.png?Expires=1585828431&amp;Signature=qTQJFk-Lzq6SR3qtxLrgOXiAv26ao8cJZD6sSjB-LbD-Y3~AEJxsUdQCukQqHAdp6dGonzT-l4~1IxBuhXG9bYZUTjBToKZ-HK~PGHn6uHyZuB8yC3ZiQnQ7CBggXMk8s9CrXnxUw6ZsKkSy75dCKvQXTqcFo1KJWugCtWKQeb9~lFGmzcbJflG3DjbPJKPFCXuSdQkgu1Q92gu5stod~olyhtpoAiXXl9TqqYLUKh9jRDlDSysLwGbFjvg4T21VpjIX8GEtIrP8cTDIVrBzRZL6xr1tYho9Fo4K0lZG4x~fnbc~CNwOfFPdKSgLTrb5CZvrdFZGVNolVxkQA0tyrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=118736914&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btx025f2.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Average
 performance over all datasets of the five delimitation methods (mPTP, 
PTP, Usearch, Crop and ABGD) for the (A) number of species, (B) <em>F</em>-scores and (C) number of RTS</p></div></div><div content-id="btx025-F2" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 2.</div><div class="graphic-wrap"><img class="content-image" src="m_btx025f2.png" alt="Average performance over all datasets of the five delimitation methods (mPTP, PTP, Usearch, Crop and ABGD) for the (A) number of species, (B) F-scores and (C) number of RTS" path-from-xml="btx025f2.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/118736914/btx025f2.tif" path-from-xml="btx025f2.tif" target="_blank">Open in new tab</a><a section="118736914" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025f2.png?Expires=1585828431&amp;Signature=qTQJFk-Lzq6SR3qtxLrgOXiAv26ao8cJZD6sSjB-LbD-Y3~AEJxsUdQCukQqHAdp6dGonzT-l4~1IxBuhXG9bYZUTjBToKZ-HK~PGHn6uHyZuB8yC3ZiQnQ7CBggXMk8s9CrXnxUw6ZsKkSy75dCKvQXTqcFo1KJWugCtWKQeb9~lFGmzcbJflG3DjbPJKPFCXuSdQkgu1Q92gu5stod~olyhtpoAiXXl9TqqYLUKh9jRDlDSysLwGbFjvg4T21VpjIX8GEtIrP8cTDIVrBzRZL6xr1tYho9Fo4K0lZG4x~fnbc~CNwOfFPdKSgLTrb5CZvrdFZGVNolVxkQA0tyrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=118736914&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btx025f2.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>Average
 performance over all datasets of the five delimitation methods (mPTP, 
PTP, Usearch, Crop and ABGD) for the (A) number of species, (B) <em>F</em>-scores and (C) number of RTS</p></div></div><p>For
 mPTP, we further assessed the confidence of the ML solution using MCMC 
sampling. For each dataset, we executed ten MCMC runs of <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-751"><span class="MJXp-mrow" id="MJXp-Span-752"><span class="MJXp-mn" id="MJXp-Span-753">2</span><span class="MJXp-mo" id="MJXp-Span-754" style="margin-left: 0.267em; margin-right: 0.267em;">×</span><span class="MJXp-msup" id="MJXp-Span-755"><span class="MJXp-mrow" id="MJXp-Span-756" style="margin-right: 0.05em;"><span class="MJXp-mn" id="MJXp-Span-757">10</span></span><span class="MJXp-mn MJXp-script" id="MJXp-Span-758" style="vertical-align: 0.5em;">7</span></span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-51-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-51"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mn>2</mn><mo>×</mo><msup><mrow><mn>10</mn></mrow><mn>7</mn></msup></mrow></math></script></span>
 steps, each starting from an initial random delimitation. We assess 
whether the independent runs have converged by calculating the ASDDSV. 
To obtain an overall support for the ML estimate, we computed the mean 
ASV over all 10 independent runs.</p><p>For each of the remaining three 
methods (ABGD, Usearch and Crop), we optimized a set of 
performance-critical parameters and chose the delimitation that 
recovered the highest number of taxonomic species. ABGD requires two 
parameters: (i) the prior limit to intraspecific diversity (<em>P</em>) and (ii) a proxy for the minimum barcoding gap among the inter- and intraspecific genetic distances (<em>X</em>). We ran ABGD with the aligned sequences and 400 parameter combinations. For <em>P</em>, we sampled 100 values from the range <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-759"><span class="MJXp-mrow" id="MJXp-Span-760"><span class="MJXp-mo" id="MJXp-Span-761" style="margin-left: 0em; margin-right: 0em;">〈</span><span class="MJXp-mn" id="MJXp-Span-762">0.001</span><span class="MJXp-mo" id="MJXp-Span-763" style="margin-left: 0em; margin-right: 0.222em;">,</span><span class="MJXp-mn" id="MJXp-Span-764">0.1</span><span class="MJXp-mo" id="MJXp-Span-765" style="margin-left: 0em; margin-right: 0em;">〉</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-52-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-52"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>〈</mo><mn>0.001</mn><mo>,</mo><mn>0.1</mn><mo>〉</mo></mrow></math></script></span> and for <em>X</em> we used the four values 0.05, 0.1, 0.15 and 0.2. The only critical parameter for Usearch is the fractional identity threshold (<em>id</em>) which defines the minimum similarity for the sequences of each cluster. We performed 50 consecutive runs, increasing the <em>id</em> value by 0.01 starting with 0.5. Finally, for Crop, we tried four combinations of the <em>l</em> and <em>u</em> parameters that correspond to similarity thresholds of 1%, 2%, 3%, and 5% as suggested by the authors (<a class="link link-uri openInAnotherWindow" href="http://code.google.com/p/crop-tingchenlab/" target="_blank">http://code.google.com/p/crop-tingchenlab/</a>). Another critical parameter for Crop is <em>z</em>, which specifies the maximum number of sequences to consider after the so-called initial “split and merge”. We set <em>z</em> to 100 which is considered a reasonable value for full-length barcoding genes.</p>                    <h3 scrollto-destination="118736917" id="118736917" class="section-title">3.3 Comparison of delimitation methods</h3>
<p>The evaluation of the five methods is based on three measures. First,
 the percentage of recovered taxonomic species (RTS), that is, the 
percentage of delimited species that match the “true species”. We 
consider the taxonomic species retrieved from BOLD to be the “true 
species”, and we deem the performance of the algorithm better when the 
number of matches to the taxonomic species is higher. Since we do not 
have the expertise to evaluate the taxonomy of each dataset, we could 
not assess the accuracy of each individual delimitation. However, by 
assuming that the closer a delimitation is to the current taxonomy the 
higher the probability to correspond to the real species, we gain some 
insight as to the relative accuracy among the different methods. The 
second measure is the <em>F</em>-score, also known as <em>F</em>-measure or <em>F</em>1-score (<a reveal-id="btx025-B36" data-open="btx025-B36" class="link link-ref link-reveal xref-bibr">Rijsbergen, 1979</a>), that is, the harmonic mean of <em>precision</em> and <em>recall</em>. In species delimitation, <em>precision</em> denotes the fraction of clustered sequences belonging to a single taxonomic species. The <em>recall</em> describes the fraction of sequences of a species that are clustered together. The <em>F</em>-score
 improves when decreasing (i) the number of species which are split into
 more than one groups and (ii) the number of taxonomic species lumped 
together into one group. The <em>F</em>-score ranges from 0 to 1, where 1
 indicates a perfect agreement among two delimitations. Finally, the 
third measure is simply the number of delimited species.</p><p>The 
accuracy of either PTP model in recovering the true species depends on 
whether these taxa form monophyletic clades. Thus, for PTP and mPTP, we 
also compare their performance by applying the above measures for 
monophyletic taxonomic species only. Additionally, we quantify how the 
percentage of monophyletic species correlates to the percentage of RTS 
for the five delimitation methods we test. Finally, we test whether the 
most complex model (mPTP) fits the data significantly better than the 
simpler one (PTP), by performing a Likelihood Ratio Test (LRT). The LRT 
comparison among PTP and mPTP is only possible when their likelihood 
scores are calculated on identical delimitations, in which case the PTP 
is nested in the mPTP delimitation. Such a comparison informs us whether
 the additional parameters in mPTP improve significantly the likelihood 
of the delimitation.</p>                    <h2 scrollto-destination="118736920" id="118736920" class="section-title">4 Results</h2>
<p>We analyzed a total of 17 219 COI sequences. The alignment length 
ranged from 658 bp to 1620 bp, while the proportion of variable 
nucleotides, measured by average P-Distance (MEGA v5.2; <a reveal-id="btx025-B40" data-open="btx025-B40" class="link link-ref link-reveal xref-bibr">Tamura <em>et al.</em>, 2011</a>) for each dataset, varied from 2.3% (<em>Balanus</em>) to 21.4% (<em>Gammarus</em>). <a reveal-id="btx025-T1" data-open="btx025-T1" class="link link-reveal link-table xref-fig">Table 1</a>
 presents the fraction of monophyletic taxonomic species in the 
phylogenies for each dataset. The fraction ranges from only 51% in <em>Gammarus</em>, the most variable of the datasets, to 94% (<em>Xysticus</em>). Out of the 400 combinations of parameters tested for ABGD, two recovered the highest number of species, with <em>P</em> = 0.010235 and <em>X</em> = 0.1 or 0.05. For Usearch, the delimitation that maximized the number of RTS was with the threshold value of <em>id</em> = 0.97. Finally, the best-scoring parameter combination for Crop (<em>l</em> = 1.0 and <em>u</em> = 1.5) corresponds to the 3% similarity threshold. <a reveal-id="btx025-T2" data-open="btx025-T2" class="link link-reveal link-table xref-fig">Table 2</a> presents the efficiency of the five delimitation methods for five of the datasets using three measures: percentage of RTS, <em>F</em>-score and number of species. A complete list of results for all 24 datasets is given in paragraph 4.1 in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material SI</a></span>. Overall, mPTP and ABGD scored best in terms of RTS percentage (59% and 57% on an average, respectively) and <em>F</em>-scores (0.828 and 0.819, respectively) compared with the other three methods (PTP: RTS = 53%, <em>F</em>-score = 0.776, Usearch: RTS = 53%, <em>F</em>-score = 0.79, Crop: RTS = 52%, <em>F</em>-score = 0.791) (<a reveal-id="btx025-F2" data-open="btx025-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>).
 The striking difference between the five methods is in the number of 
delimited species. The novel mPTP method delimited a total of 1190 
species which is the closest to the total number of taxonomic species 
(1041). In contrast, PTP inferred 2048 species which is almost twice 
this number. The other three methods yielded more conservative species 
numbers compared to PTP (Usearch: 1671, Crop: 1663, ABGD: 1412) that 
are, however, still notably higher than the mPTP estimates (<a reveal-id="btx025-F2" data-open="btx025-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>).
 In terms of likelihood, when we compared mPTP with PTP for a fixed 
delimitation, mPTP was always yielding at least as good scores as the 
PTP but higher in most cases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3 in Supplement I</a></span>).
 Based on the LRT results the increase in the likelihood was significant
 in all datasets except three for which both models resulted in 
identical or very similar scores. </p><div content-id="btx025-T2" class="table-modal" style="display: block;"><div class="table-wrap" content-id="btx025-T2"><div id="btx025-T2" data-id="btx025-T2" class="table-wrap-title"><div class="label">Table 2.</div><div class="caption"><p>Percentage of RTS, <em>F</em>-scores
 and number of delimited species for the five delimitation methods 
(mPTP, PTP, Usearch, Crop and ABGD) for five of the empirical datasets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>&nbsp;</th><th>mPTP&nbsp;</th><th>PTP&nbsp;</th><th>Usearch&nbsp;</th><th>Crop&nbsp;</th><th>ABGD&nbsp;</th></tr></thead><tbody><tr><td>Genus&nbsp;</td><td colspan="5"><strong><span class="underline">RTS (%)</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>51&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>41&nbsp;</td><td>39&nbsp;</td><td>30&nbsp;</td><td>31&nbsp;</td><td>40&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>64&nbsp;</td><td>60&nbsp;</td><td>57&nbsp;</td><td>55&nbsp;</td><td>60&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>47&nbsp;</td><td>51&nbsp;</td><td>41&nbsp;</td><td>34&nbsp;</td><td>53&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>60&nbsp;</td><td>27&nbsp;</td><td>47&nbsp;</td><td>53&nbsp;</td><td>47&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">F-score</span></strong>&nbsp;</td></tr><tr><td>Amynthas&nbsp;</td><td>0.784&nbsp;</td><td>0.638&nbsp;</td><td>0.649&nbsp;</td><td>0.674&nbsp;</td><td>0.673&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>0.787&nbsp;</td><td>0.704&nbsp;</td><td>0.730&nbsp;</td><td>0.681&nbsp;</td><td>0.730&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>0.839&nbsp;</td><td>0.844&nbsp;</td><td>0.836&nbsp;</td><td>0.832&nbsp;</td><td>0.850&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>0.728&nbsp;</td><td>0.559&nbsp;</td><td>0.747&nbsp;</td><td>0.729&nbsp;</td><td>0.765&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>0.882&nbsp;</td><td>0.717&nbsp;</td><td>0.812&nbsp;</td><td>0.852&nbsp;</td><td>0.828&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">Number of species</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>64&nbsp;</td><td>104&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td><td>90&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>126&nbsp;</td><td>218&nbsp;</td><td>193&nbsp;</td><td>154&nbsp;</td><td>118&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>85&nbsp;</td><td>100&nbsp;</td><td>95&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>139&nbsp;</td><td>444&nbsp;</td><td>183&nbsp;</td><td>217&nbsp;</td><td>157&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>21&nbsp;</td><td>38&nbsp;</td><td>26&nbsp;</td><td>20&nbsp;</td><td>24&nbsp;</td></tr></tbody></table></div><div content-id="btx025-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>&nbsp;</th><th>mPTP&nbsp;</th><th>PTP&nbsp;</th><th>Usearch&nbsp;</th><th>Crop&nbsp;</th><th>ABGD&nbsp;</th></tr></thead><tbody><tr><td>Genus&nbsp;</td><td colspan="5"><strong><span class="underline">RTS (%)</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>51&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>41&nbsp;</td><td>39&nbsp;</td><td>30&nbsp;</td><td>31&nbsp;</td><td>40&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>64&nbsp;</td><td>60&nbsp;</td><td>57&nbsp;</td><td>55&nbsp;</td><td>60&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>47&nbsp;</td><td>51&nbsp;</td><td>41&nbsp;</td><td>34&nbsp;</td><td>53&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>60&nbsp;</td><td>27&nbsp;</td><td>47&nbsp;</td><td>53&nbsp;</td><td>47&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">F-score</span></strong>&nbsp;</td></tr><tr><td>Amynthas&nbsp;</td><td>0.784&nbsp;</td><td>0.638&nbsp;</td><td>0.649&nbsp;</td><td>0.674&nbsp;</td><td>0.673&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>0.787&nbsp;</td><td>0.704&nbsp;</td><td>0.730&nbsp;</td><td>0.681&nbsp;</td><td>0.730&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>0.839&nbsp;</td><td>0.844&nbsp;</td><td>0.836&nbsp;</td><td>0.832&nbsp;</td><td>0.850&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>0.728&nbsp;</td><td>0.559&nbsp;</td><td>0.747&nbsp;</td><td>0.729&nbsp;</td><td>0.765&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>0.882&nbsp;</td><td>0.717&nbsp;</td><td>0.812&nbsp;</td><td>0.852&nbsp;</td><td>0.828&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">Number of species</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>64&nbsp;</td><td>104&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td><td>90&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>126&nbsp;</td><td>218&nbsp;</td><td>193&nbsp;</td><td>154&nbsp;</td><td>118&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>85&nbsp;</td><td>100&nbsp;</td><td>95&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>139&nbsp;</td><td>444&nbsp;</td><td>183&nbsp;</td><td>217&nbsp;</td><td>157&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>21&nbsp;</td><td>38&nbsp;</td><td>26&nbsp;</td><td>20&nbsp;</td><td>24&nbsp;</td></tr></tbody></table></div></div></div><div class="table-wrap" content-id="btx025-T2"><div id="btx025-T2" data-id="btx025-T2" class="table-wrap-title"><div class="label">Table 2.</div><div class="caption"><p>Percentage of RTS, <em>F</em>-scores
 and number of delimited species for the five delimitation methods 
(mPTP, PTP, Usearch, Crop and ABGD) for five of the empirical datasets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>&nbsp;</th><th>mPTP&nbsp;</th><th>PTP&nbsp;</th><th>Usearch&nbsp;</th><th>Crop&nbsp;</th><th>ABGD&nbsp;</th></tr></thead><tbody><tr><td>Genus&nbsp;</td><td colspan="5"><strong><span class="underline">RTS (%)</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>51&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>41&nbsp;</td><td>39&nbsp;</td><td>30&nbsp;</td><td>31&nbsp;</td><td>40&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>64&nbsp;</td><td>60&nbsp;</td><td>57&nbsp;</td><td>55&nbsp;</td><td>60&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>47&nbsp;</td><td>51&nbsp;</td><td>41&nbsp;</td><td>34&nbsp;</td><td>53&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>60&nbsp;</td><td>27&nbsp;</td><td>47&nbsp;</td><td>53&nbsp;</td><td>47&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">F-score</span></strong>&nbsp;</td></tr><tr><td>Amynthas&nbsp;</td><td>0.784&nbsp;</td><td>0.638&nbsp;</td><td>0.649&nbsp;</td><td>0.674&nbsp;</td><td>0.673&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>0.787&nbsp;</td><td>0.704&nbsp;</td><td>0.730&nbsp;</td><td>0.681&nbsp;</td><td>0.730&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>0.839&nbsp;</td><td>0.844&nbsp;</td><td>0.836&nbsp;</td><td>0.832&nbsp;</td><td>0.850&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>0.728&nbsp;</td><td>0.559&nbsp;</td><td>0.747&nbsp;</td><td>0.729&nbsp;</td><td>0.765&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>0.882&nbsp;</td><td>0.717&nbsp;</td><td>0.812&nbsp;</td><td>0.852&nbsp;</td><td>0.828&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">Number of species</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>64&nbsp;</td><td>104&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td><td>90&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>126&nbsp;</td><td>218&nbsp;</td><td>193&nbsp;</td><td>154&nbsp;</td><td>118&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>85&nbsp;</td><td>100&nbsp;</td><td>95&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>139&nbsp;</td><td>444&nbsp;</td><td>183&nbsp;</td><td>217&nbsp;</td><td>157&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>21&nbsp;</td><td>38&nbsp;</td><td>26&nbsp;</td><td>20&nbsp;</td><td>24&nbsp;</td></tr></tbody></table></div><div content-id="btx025-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>&nbsp;</th><th>mPTP&nbsp;</th><th>PTP&nbsp;</th><th>Usearch&nbsp;</th><th>Crop&nbsp;</th><th>ABGD&nbsp;</th></tr></thead><tbody><tr><td>Genus&nbsp;</td><td colspan="5"><strong><span class="underline">RTS (%)</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>51&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td><td>40&nbsp;</td><td>44&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>41&nbsp;</td><td>39&nbsp;</td><td>30&nbsp;</td><td>31&nbsp;</td><td>40&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>64&nbsp;</td><td>60&nbsp;</td><td>57&nbsp;</td><td>55&nbsp;</td><td>60&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>47&nbsp;</td><td>51&nbsp;</td><td>41&nbsp;</td><td>34&nbsp;</td><td>53&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>60&nbsp;</td><td>27&nbsp;</td><td>47&nbsp;</td><td>53&nbsp;</td><td>47&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">F-score</span></strong>&nbsp;</td></tr><tr><td>Amynthas&nbsp;</td><td>0.784&nbsp;</td><td>0.638&nbsp;</td><td>0.649&nbsp;</td><td>0.674&nbsp;</td><td>0.673&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>0.787&nbsp;</td><td>0.704&nbsp;</td><td>0.730&nbsp;</td><td>0.681&nbsp;</td><td>0.730&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>0.839&nbsp;</td><td>0.844&nbsp;</td><td>0.836&nbsp;</td><td>0.832&nbsp;</td><td>0.850&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>0.728&nbsp;</td><td>0.559&nbsp;</td><td>0.747&nbsp;</td><td>0.729&nbsp;</td><td>0.765&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>0.882&nbsp;</td><td>0.717&nbsp;</td><td>0.812&nbsp;</td><td>0.852&nbsp;</td><td>0.828&nbsp;</td></tr><tr><td>&nbsp;</td><td colspan="5"><strong><span class="underline">Number of species</span></strong>&nbsp;</td></tr><tr><td><em>Amynthas</em>&nbsp;</td><td>64&nbsp;</td><td>104&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td><td>90&nbsp;</td></tr><tr><td><em>Anopheles</em>&nbsp;</td><td>126&nbsp;</td><td>218&nbsp;</td><td>193&nbsp;</td><td>154&nbsp;</td><td>118&nbsp;</td></tr><tr><td><em>Atheta</em>&nbsp;</td><td>85&nbsp;</td><td>100&nbsp;</td><td>95&nbsp;</td><td>95&nbsp;</td><td>91&nbsp;</td></tr><tr><td><em>Drosophila</em>&nbsp;</td><td>139&nbsp;</td><td>444&nbsp;</td><td>183&nbsp;</td><td>217&nbsp;</td><td>157&nbsp;</td></tr><tr><td><em>Philodromus</em>&nbsp;</td><td>21&nbsp;</td><td>38&nbsp;</td><td>26&nbsp;</td><td>20&nbsp;</td><td>24&nbsp;</td></tr></tbody></table></div></div><p></p><p>When
 considering only monophyletic species, as one might expect, the 
recovery percentage increases substantially for both PTP models (82% and
 72% on an average for mPTP and PTP, respectively). Similarly, the 
accuracy of the delimitations is substantially higher for the 
monophyletic taxonomic species (average <em>F</em>-score 0.945 and 0.853
 for mPTP and PTP, respectively). These results indicate that polyphyly 
(we use the term polyphyly in referring to both paraphyly and polyphyly <em>sensu</em>, <a reveal-id="btx025-B17" data-open="btx025-B17" class="link link-ref link-reveal xref-bibr">Funk and Omland, 2003</a>) is a major contributing factor when taxonomic species are not recovered (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2 in Supplement I</a></span>).
 In particular, the RTS percentage of either PTP model is highly 
correlated with the percentage of monophyletic species in a dataset (<a reveal-id="btx025-F3" data-open="btx025-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The Pearson coefficient indicates that the correlation is stronger among the species recovered with mPTP (<span class="inline-formula">⁠<span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-766"><span class="MJXp-mrow" id="MJXp-Span-767"><span class="MJXp-msub" id="MJXp-Span-768"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-769" style="margin-right: 0.05em;">r</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-770" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-771">m</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-772">P</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-773">T</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-774">P</span></span></span><span class="MJXp-mo" id="MJXp-Span-775" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-776">0.75</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-53-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-53"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>r</mi><mrow><mi>m</mi><mi>P</mi><mi>T</mi><mi>P</mi></mrow></msub><mo>=</mo><mn>0.75</mn></mrow></math></script>⁠</span>, <em>P</em> value = 2.642e−05) than with PTP (<span class="inline-formula">⁠<span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-777"><span class="MJXp-mrow" id="MJXp-Span-778"><span class="MJXp-msub" id="MJXp-Span-779"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-780" style="margin-right: 0.05em;">r</span><span class="MJXp-mrow MJXp-script" id="MJXp-Span-781" style="vertical-align: -0.4em;"><span class="MJXp-mi MJXp-italic" id="MJXp-Span-782">P</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-783">T</span><span class="MJXp-mi MJXp-italic" id="MJXp-Span-784">P</span></span></span><span class="MJXp-mo" id="MJXp-Span-785" style="margin-left: 0.333em; margin-right: 0.333em;">=</span><span class="MJXp-mn" id="MJXp-Span-786">0.62</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-54-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-54"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>r</mi><mrow><mi>P</mi><mi>T</mi><mi>P</mi></mrow></msub><mo>=</mo><mn>0.62</mn></mrow></math></script>⁠</span>, <em>P</em> value = 0.001218). The correlation is also positive for the other three methods (<em>r<sub>ABGD</sub></em> = 0.63/<em>P</em> value = 0.0009327, <em>r<sub>Usearch</sub></em> = 0.5/<em>P</em> value = 0.01294, <em>r<sub>Crop</sub></em> = 0.63/<em>P</em>
 value = 0.0009421) and comparable with PTP but smaller than for mPTP. 
Similarly, the slope of the regression line was greater for mPTP than 
for the other methods, indicating a steeper linear relationship between 
the two variables (<a reveal-id="btx025-F3" data-open="btx025-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>).</p>                        <a id="118736923" scrollto-destination="118736923"></a>
<div id="btx025-F3" data-id="btx025-F3" class="fig fig-section"><div class="label fig-label">Fig. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_btx025f3.png" alt="For each method, we fit a regression line to the points of correspondence of the percentage of monophyletic species (x-axis) to the percentage of RTS (y-axis). The Pearson coefficient (r) is given for each correlation in the corresponding color" path-from-xml="btx025f3.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/118736923/btx025f3.tif" path-from-xml="btx025f3.tif" target="_blank">Open in new tab</a><a section="118736923" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025f3.png?Expires=1585828431&amp;Signature=kE4Vp0dGrXk2eGi7QIjhFHKlXHGlBhkzyLw0b3h7FSn7w0vAXSAGhhwTg~yQ4Oe5r7oQJtbLCNdcH6SRd7hm40SXuhQHXwauOrsWEehaC6n42K2eraqz9SwrIofQCa1fRUb8sA2HvnxNZo2Z6hTSRiPepU7zzx7liVjRPSyf0hYKUxNRGgNF2eLlbcwP9F6OPeM7NbPqi-54l8z9JgVUnPVg96fFADk0qmjZIhR56SxvRDVpi~VSzrnhEWBa0McXcKvv1NsOrEzUntgFaeW~zSVAxKzrMac9wMtBmgsIHdiK-8fA8qo4q0NQv0x3UYh8U5iunBV6c-4Mjc-45fWgZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=118736923&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btx025f3.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>For each method, we fit a regression line to the points of correspondence of the percentage of monophyletic species (<em>x</em>-axis) to the percentage of RTS (<em>y</em>-axis). The Pearson coefficient (<em>r</em>) is given for each correlation in the corresponding color</p></div></div><div content-id="btx025-F3" class="fig fig-modal reveal-modal" style="display: block;"><div class="label fig-label">Fig. 3.</div><div class="graphic-wrap"><img class="content-image" src="m_btx025f3.png" alt="For each method, we fit a regression line to the points of correspondence of the percentage of monophyletic species (x-axis) to the percentage of RTS (y-axis). The Pearson coefficient (r) is given for each correlation in the corresponding color" path-from-xml="btx025f3.tif"><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="https://academic.oup.com/view-large/figure/118736923/btx025f3.tif" path-from-xml="btx025f3.tif" target="_blank">Open in new tab</a><a section="118736923" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025f3.png?Expires=1585828431&amp;Signature=kE4Vp0dGrXk2eGi7QIjhFHKlXHGlBhkzyLw0b3h7FSn7w0vAXSAGhhwTg~yQ4Oe5r7oQJtbLCNdcH6SRd7hm40SXuhQHXwauOrsWEehaC6n42K2eraqz9SwrIofQCa1fRUb8sA2HvnxNZo2Z6hTSRiPepU7zzx7liVjRPSyf0hYKUxNRGgNF2eLlbcwP9F6OPeM7NbPqi-54l8z9JgVUnPVg96fFADk0qmjZIhR56SxvRDVpi~VSzrnhEWBa0McXcKvv1NsOrEzUntgFaeW~zSVAxKzrMac9wMtBmgsIHdiK-8fA8qo4q0NQv0x3UYh8U5iunBV6c-4Mjc-45fWgZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=118736923&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;imagename=&amp;siteId=5139" class="download-slide" path-from-xml="btx025f3.tif" target="">Download slide</a></div></div><div class="caption fig-caption"><p>For each method, we fit a regression line to the points of correspondence of the percentage of monophyletic species (<em>x</em>-axis) to the percentage of RTS (<em>y</em>-axis). The Pearson coefficient (<em>r</em>) is given for each correlation in the corresponding color</p></div></div><p>Regarding
 the confidence of the mPTP delimitations, all independent MCMC runs 
appear to converge with an ASDDSV below 0.01 for all datasets, except 
for <em>Drosophila</em> (one of the largest datasets), for which the ASDDSV was 0.048, but still in the acceptable range for assuming convergence (<a reveal-id="btx025-B37" data-open="btx025-B37" class="link link-ref link-reveal xref-bibr">Ronquist <em>et al.</em>, 2012</a>). The ASV with respect to the ML delimitation was very high for all datasets, ranging from 72% (<em>Phyllotreta</em>) to 100% (<em>Balanus</em>), indicating that the data support well the ML solution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3 in Supplement I</a></span>).
 The accumulated running time for all 10 independent runs (executed 
sequentially on an Intel Core i7-4500U CPU @ 1.80GHz) was less than 50 s
 on average across all datasets, which corresponds to <span class="inline-formula"><span class="MathJax_Preview" style="color: inherit;"><span class="MJXp-math" id="MJXp-Span-787"><span class="MJXp-mrow" id="MJXp-Span-788"><span class="MJXp-mo" id="MJXp-Span-789" style="margin-left: 0.333em; margin-right: 0.333em;">∼</span><span class="MJXp-mn" id="MJXp-Span-790">5</span></span></span></span><span class="MathJax MathJax_Processing" id="MathJax-Element-55-Frame" tabindex="0"></span><script type="math/mml" id="MathJax-Element-55"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mo>∼</mo><mn>5</mn></mrow></math></script></span> s per run. For a thorough run-time comparison between PTP and mPTP (including the ML method), refer to Paragraph 4.2 in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material SI</a></span>.</p>                    <h2 scrollto-destination="118736925" id="118736925" class="section-title">5 Discussion</h2>
<p>Molecular species delimitation has caused mixed reactions within the 
scientific community, from those highly enthusiastic about its potential
 to accelerate biodiversity cataloging (<a reveal-id="btx025-B5" data-open="btx025-B5" class="link link-ref link-reveal xref-bibr">Blaxter, 2003</a>; <a reveal-id="btx025-B6" data-open="btx025-B6" class="link link-ref link-reveal xref-bibr">Blaxter and Floyd, 2003</a>) to those very critical about its role in shaping modern systematics (<a reveal-id="btx025-B3" data-open="btx025-B3" class="link link-ref link-reveal xref-bibr">Bauer <em>et al.</em>, 2010</a>; <a reveal-id="btx025-B45" data-open="btx025-B45" class="link link-ref link-reveal xref-bibr">Will <em>et al.</em>, 2005</a>).
 The main argument between the two conflicting sides is whether 
molecular delimitation on its own is sufficient to justify taxonomic 
rearrangements (<a reveal-id="btx025-B45" data-open="btx025-B45" class="link link-ref link-reveal xref-bibr">Will <em>et al.</em>, 2005</a>).
 Integrative taxonomy alleviates this conflict as it, by definition, 
requires multiple levels of evidence taking into account various 
biodiversity characteristics of an organism to accept potential 
taxonomic changes. Within this framework, and in line with the 
independently evolving species concept (<a reveal-id="btx025-B11" data-open="btx025-B11" class="link link-ref link-reveal xref-bibr">De Queiroz, 2007</a>)
 and the phylogenetic species concept, molecular species delimitation 
using DNA-barcoding serves as an excellent tool in modern taxonomy (<a reveal-id="btx025-B42" data-open="btx025-B42" class="link link-ref link-reveal xref-bibr">Tautz <em>et al.</em>, 2003</a>; <a reveal-id="btx025-B44" data-open="btx025-B44" class="link link-ref link-reveal xref-bibr">Vogler and Monaghan, 2007</a>).
 It can be easily applied to a large range of organisms regardless of 
their life stage, gender or prior taxonomic knowledge, by a broad range 
of researchers. In addition, barcoding genes are easily amplified from 
small tissue samples even from poorly preserved historical samples (<a reveal-id="btx025-B2" data-open="btx025-B2" class="link link-ref link-reveal xref-bibr">Austin and Melville, 2006</a>).
 Furthermore, such an approach might represent the only feasible 
approach in biodiversity surveys, as they often comprise a large number 
of species, many of which might be unknown or not easily accessible. 
Hence, collecting ecological or morphological traits is often simply not
 feasible. For historical samples or samples from inaccessible areas 
(e.g., deep seas, deserts) barcoding methods are equally important, 
since the collection of life history traits might be similarly 
challenging. This and previous studies (<a reveal-id="btx025-B29" data-open="btx025-B29" class="link link-ref link-reveal xref-bibr">Monaghan <em>et al.</em>, 2009</a>; <a reveal-id="btx025-B14" data-open="btx025-B14" class="link link-ref link-reveal xref-bibr">Esselstyn <em>et al.</em>, 2012</a>; <a reveal-id="btx025-B35" data-open="btx025-B35" class="link link-ref link-reveal xref-bibr">Ratnasingham and Hebert, 2013</a>; <a reveal-id="btx025-B41" data-open="btx025-B41" class="link link-ref link-reveal xref-bibr">Tang <em>et al.</em>, 2014</a>)
 suggest that single-locus barcoding methods provide meaningful 
clusters, close to taxonomically acknowledged species. This makes them 
useful for approximate species estimation studies or when more thorough 
systematic research is practically impossible.</p>                    <h3 scrollto-destination="118736927" id="118736927" class="section-title">5.1 (m)PTP</h3>
<p>The molecular systematics of the example taxa vary from well-studied [e.g., <em>Anopheles</em> (<a reveal-id="btx025-B19" data-open="btx025-B19" class="link link-ref link-reveal xref-bibr">Harbach and Kitching, 2016</a>), <em>Drosophila</em> (<a reveal-id="btx025-B30" data-open="btx025-B30" class="link link-ref link-reveal xref-bibr">Obbard <em>et al.</em>, 2012</a>)] to scarcely studied (e.g., <em>Xysticus</em>, <em>Clubiona</em>).
 They further differ in the number of species, number of sequences per 
species, geographic ranges, and nucleotide divergences. Despite these 
differences, mPTP outperforms PTP and yields substantially smaller 
putative species numbers as well as delimitations that are closer to the
 current taxonomy. The assumption of mPTP that per-species branch 
lengths can be fit to distinct exponential distributions increases 
flexibility and adjustability to more realistic datasets. The divergence
 of intraspecific diversity patterns may either be due to population 
traits and processes (<a reveal-id="btx025-B4" data-open="btx025-B4" class="link link-ref link-reveal xref-bibr">Bazin <em>et al.</em>, 2006</a>)
 or the uneven sampling of the species (e.g., a highly sampled species 
from a single location compared to a species represented by one sample 
from multiple locations). The latter is known to decrease PTP accuracy (<a reveal-id="btx025-B48" data-open="btx025-B48" class="link link-ref link-reveal xref-bibr">Zhang <em>et al.</em>, 2013</a>).</p><p>The
 accuracy of (m)PTP strongly correlates with the proportion of 
monophyletic species in the underlying phylogeny. Polyphyletic species 
will either be delimited into smaller groups or delimited with other, 
nested species. Therefore, the recovery rate of taxonomic species is 
significantly higher when only considering the monophyletic species in 
each dataset. Among the 24 datasets, the number of monophyletic species 
ranged from 51% to 94%. Thus, the lack of monophyly is the primary 
contributing factor for not recovering taxonomic species with either PTP
 model. The average observed monophyly in our arthropod (76.4%) and the 
remaining invertebrate (64.7%) datasets corresponds well to available 
estimates for the same groups (73.5% and 61.4%, respectively) based on a
 large number of empirical studies (<a reveal-id="btx025-B17" data-open="btx025-B17" class="link link-ref link-reveal xref-bibr">Funk and Omland, 2003</a>).
 The same study reports that the most common reasons for polyphyly are 
inaccurate taxonomy, incomplete lineage sorting, and retrogressive 
hybridization (<a reveal-id="btx025-B17" data-open="btx025-B17" class="link link-ref link-reveal xref-bibr">Funk and Omland, 2003</a>; <a reveal-id="btx025-B27" data-open="btx025-B27" class="link link-ref link-reveal xref-bibr">McKay and Zink, 2010</a>).
 All three effects could apply to the selected datasets. Nevertheless, 
only the latter two are relevant to the efficiency of the algorithm <em>per se</em>,
 while the accuracy of the taxonomy is only relevant when it is used as a
 reference measure. Polyphyletic species affect both PTP versions in the
 same way. The reason for the improved delimitation accuracy of mPTP 
over PTP on monophyletic species is that the former accommodates 
different degrees of intraspecific genetic diversity within a phylogeny.</p>                    <h4 scrollto-destination="118736930" id="118736930" class="section-title">5.1.1 MCMC sampling</h4>
<p>The drawback of an ML approach is that it only provides a point 
estimate and no information on model uncertainty. The confidence about a
 given solution has a substantial impact in drawing reasonable 
conclusions. Therefore, we provide an MCMC method for assessing the 
plausibility of the ML solution. In phylogenies comprising hundreds of 
taxa, it is hard to obtain an overall support for a particular 
delimitation hypothesis by visual inspection of the tree. To alleviate 
this problem, at the end of an MCMC run we calculate the ASV for the ML 
solution. In our experiments, the ML delimitations were highly supported
 by the ASVs for all datasets, pointing towards a unimodal likelihood 
surface for our model (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3 in Supplement I</a></span>).
 Low ASVs may be interpreted as low confidence for the given 
delimitation scheme, either because another (multi-modal likelihood 
surface) or no delimitation scheme (flat-likelihood surface) is well 
supported. This also indicates a poor fit of the data to the model. The 
execution times of the MCMC sampling are almost negligible, regardless 
of tree size, and, therefore, we can thoroughly sample the delimitation 
space even for phylogenies comprising thousands of taxa. For the large 
phylogenies of our study (i.e., <em>Drosophila</em>, <em>Anopheles</em>),
 the PTP implementation required over 30 h for the ML optimization 
alone, and would require days or even months for the estimation of 
support values. Instead, mPTP required less than a minute for both the 
ML optimization and the support value estimation.</p>                    <h3 scrollto-destination="118736932" id="118736932" class="section-title">5.2 Distance-based methods</h3>
<p>Distance-based methods are easy to apply to large datasets as they 
need minimal preprocessing effort and computational time. Their major 
weakness is that they require either a threshold value or a combination 
of parameters associated with the threshold value, the sampling effort, 
or the search strategy. Empirical data show that certain similarity 
cut-offs (2–3%) correspond well to the species boundaries of several 
taxonomic groups (<a reveal-id="btx025-B20" data-open="btx025-B20" class="link link-ref link-reveal xref-bibr">Hebert <em>et al.</em>, 2003</a>, <a reveal-id="btx025-B21" data-open="btx025-B21" class="link link-ref link-reveal xref-bibr">Hebert <em>et al.</em>, 2004</a>; ; <a reveal-id="btx025-B38" data-open="btx025-B38" class="link link-ref link-reveal xref-bibr">Smith <em>et al.</em>, 2005</a>); however, these values are often far from optimal (<a reveal-id="btx025-B25" data-open="btx025-B25" class="link link-ref link-reveal xref-bibr">Lin <em>et al.</em>, 2015</a>). Selecting these parameters is not intuitive and they may only be evaluated <em>a posteriori</em>
 based on the expectations of the researcher. Furthermore, the empirical
 knowledge for threshold settings is tightly associated with barcoding 
genes, and it may not be as useful for other marker genes. Here, we 
optimized the parameters for three of the most popular distance methods 
(ABGD, Crop and Usearch) based on the RTS percentage. Despite our 
substantial effort to use optimal parameters for the given data, our 
results show that, on an average, mPTP performs better with respect to <em>F</em>-scores
 and RTS percentage. At the same time, it also delimited notably fewer 
species. ABGD accuracy was the closest to mPTP, while PTP, CROP, and 
Usearch performed notably worse. Besides accuracy, the greatest 
advantage of mPTP is that it consistently yields more accurate results 
without requiring the user to optimize any parameters/thresholds. 
Finally, in contrast to (m)PTP, distance methods ignore evolutionary 
relationships. Hence, there is no direct relation between monophyletic 
species and delimitation accuracy in similarity based tools. However, 
polyphyly often reflects recent speciation while reciprocal monophyly 
indicates that significant time since speciation has passed. 
Consequently, the barcoding gap should be less pronounced in datasets of
 recently diverged species. This justifies that the RTS fraction 
improves with the number of monophyletic species.</p>                    <h2 scrollto-destination="118736934" id="118736934" class="section-title">6 Conclusions</h2>
<p>We presented mPTP, a novel approach for single-locus delimitation 
that consistently provides faster and more accurate species estimates 
than PTP and other popular delimitation methods. As PTP, mPTP is mainly 
designed for analyzing barcoding loci, but can potentially also be 
applied to entire organelle phylogenies (e.g. mitochondria, <a reveal-id="btx025-B34" data-open="btx025-B34" class="link link-ref link-reveal xref-bibr">Qin <em>et al.</em>, 2015</a>).
 In contrast to methods based on sequence similarity, mPTP does not 
require any similarity threshold or other user-defined parameter as 
input. The limitations of mPTP are associated with processes that cannot
 be detected in either single-gene phylogenies (incomplete lineage 
sorting, hybridization) or in recent speciation events. The novel 
dynamic programming delimitation algorithm reduces computation time to a
 minimum and allows for almost instantaneous species delimitation on 
phylogenies with thousands of taxa. The MCMC sampling provides support 
values for the delimited species based on millions (or even billions) of
 MCMC generations in just a few seconds on a modern computer. The mPTP 
tool is available both, as a standalone package, and as a web service.</p>                    <h2 scrollto-destination="118736936" id="118736936" class="backacknowledgements-title">Acknowledgement</h2>
<p>The authors gratefully acknowledge the support of the Klaus Tschira Foundation.</p>                    <h3 scrollto-destination="118736938" id="118736938" class="section-title">Funding</h3>
<p>This work was supported by European Commission under the 7th Framework Programme [FP7-PEOPLE-2013-IEF EVOGREN (625057) to P.P.]</p><p><em>Conflict of Interest</em>: none declared.</p>                    <h2 scrollto-destination="118736941" id="118736941" class="backreferences-title">References</h2>
<div class="ref-list"><div content-id="btx025-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B1" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akaike</div> <div class="given-names">H.</div></div></div> (<div class="year">1978</div>) 
<div class="article-title">A Bayesian analysis of the minimum AIC procedure</div>. <div class="source ">Ann. Inst. Stat. Math</div>., <div class="volume">30</div>, <div class="fpage">9</div>–<div class="lpage">14</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20Bayesian%20analysis%20of%20the%20minimum%20AIC%20procedure&amp;author=H.%20Akaike&amp;publication_year=1978&amp;journal=Ann.%20Inst.%20Stat.%20Math&amp;volume=30&amp;pages=9-14" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1007/BF02480194" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02480194" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1007%2FBF02480194"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20Bayesian%20analysis%20of%20the%20minimum%20AIC%20procedure&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B2" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Austin</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Melville</div> <div class="given-names">J.</div></div></div> (<div class="year">2006</div>) 
<div class="article-title">Incorporating historical museum specimens into molecular systematic and conservation genetics research</div>. <div class="source ">Mol. Ecol. Notes</div>, <div class="volume">6</div>, <div class="fpage">1089</div>–<div class="lpage">1092</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Incorporating%20historical%20museum%20specimens%20into%20molecular%20systematic%20and%20conservation%20genetics%20research&amp;author=J.J.%20Austin&amp;author=J.%20Melville&amp;publication_year=2006&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=6&amp;pages=1089-1092" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/men.2006.6.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2006.6.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmen.2006.6.issue-4"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Incorporating%20historical%20museum%20specimens%20into%20molecular%20systematic%20and%20conservation%20genetics%20research&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B3" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bauer</div> <div class="given-names">A.M.</div></div></div> et al.&nbsp; (<div class="year">2010</div>). 
<div class="article-title">Availability of new Bayesian-delimited gecko names and the importance of character-based species descriptions</div>. <div class="source ">Proc. R. Soc. Lond. B: Biol. Sci</div>., <div class="volume">278</div>, <div class="fpage">490</div>–<div class="lpage">492</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Availability%20of%20new%20Bayesian-delimited%20gecko%20names%20and%20the%20importance%20of%20character-based%20species%20descriptions&amp;author=A.M.%20Bauer&amp;publication_year=2010&amp;journal=Proc.%20R.%20Soc.%20Lond.%20B%3A%20Biol.%20Sci&amp;volume=278&amp;pages=490-492" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rspb.2010.1330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2010.1330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frspb.2010.1330"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Availability%20of%20new%20Bayesian-delimited%20gecko%20names%20and%20the%20importance%20of%20character-based%20species%20descriptions&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B4" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bazin</div> <div class="given-names">E.</div></div></div> et al.&nbsp; (<div class="year">2006</div>) 
<div class="article-title">Population size does not influence mitochondrial genetic diversity in animals</div>. <div class="source ">Science</div>, <div class="volume">312</div>, <div class="fpage">570</div>–<div class="lpage">572</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Population%20size%20does%20not%20influence%20mitochondrial%20genetic%20diversity%20in%20animals&amp;author=E.%20Bazin&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=570-572" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1126/science.1122033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1122033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1126%2Fscience.1122033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16645093" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Population%20size%20does%20not%20influence%20mitochondrial%20genetic%20diversity%20in%20animals&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B5" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.</div></div></div> (<div class="year">2003</div>) 
<div class="article-title">Molecular systematics: counting angels with DNA</div>. <div class="source ">Nature</div>, <div class="volume">421</div>, <div class="fpage">122</div>–<div class="lpage">124</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20systematics%3A%20counting%20angels%20with%20DNA&amp;author=M.%20Blaxter&amp;publication_year=2003&amp;journal=Nature&amp;volume=421&amp;pages=122-124" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1038/421122a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F421122a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1038%2F421122a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12520286" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20systematics%3A%20counting%20angels%20with%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B6" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Floyd</div> <div class="given-names">R.</div></div></div> (<div class="year">2003</div>) 
<div class="article-title">Molecular taxonomics for biodiversity surveys: already a reality</div>. <div class="source ">Trends Ecol. Evol</div>., <div class="volume">18</div>, <div class="fpage">268</div>–<div class="lpage">269</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Molecular%20taxonomics%20for%20biodiversity%20surveys%3A%20already%20a%20reality&amp;author=M.%20Blaxter&amp;author=R.%20Floyd&amp;publication_year=2003&amp;journal=Trends%20Ecol.%20Evol&amp;volume=18&amp;pages=268-269" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0169-5347(03)00102-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(03)00102-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0169-5347(03)00102-2"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Molecular%20taxonomics%20for%20biodiversity%20surveys%3A%20already%20a%20reality&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B7" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.</div></div></div> et al.&nbsp; (<div class="year">2005</div>) 
<div class="article-title">Defining operational taxonomic units using DNA barcode data</div>. <div class="source ">Phil. Trans. R. Soc. B: Biol. Sci</div>., <div class="volume">360</div>, <div class="fpage">1935</div>–<div class="lpage">1943</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Defining%20operational%20taxonomic%20units%20using%20DNA%20barcode%20data&amp;author=M.%20Blaxter&amp;publication_year=2005&amp;journal=Phil.%20Trans.%20R.%20Soc.%20B%3A%20Biol.%20Sci&amp;volume=360&amp;pages=1935-1943" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rstb.2005.1725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2005.1725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frstb.2005.1725"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Defining%20operational%20taxonomic%20units%20using%20DNA%20barcode%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B8" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burnham</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">D.</div></div></div> (<div class="year">2002</div>). <div class="source ">Model Selection and MultiModel Inference: A Practical Information-Theoretic Approach</div>. 
<div class="publisher-name">Springer</div>, 
<div class="publisher-loc">New York</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Model%20Selection%20and%20MultiModel%20Inference%3A%20A%20Practical%20Information-Theoretic%20Approach&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Model%20Selection%20and%20MultiModel%20Inference%3A%20A%20Practical%20Information-Theoretic%20Approach&amp;aulast=Burnham&amp;date=2002" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Model%20Selection%20and%20MultiModel%20Inference%3A%20A%20Practical%20Information-Theoretic%20Approach&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Model%20Selection%20and%20MultiModel%20Inference%3A%20A%20Practical%20Information-Theoretic%20Approach">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B9" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Casiraghi</div> <div class="given-names">M.</div></div></div> et al.&nbsp; (<div class="year">2010</div>) 
<div class="article-title">DNA barcoding: a six-question tour to improve users’ awareness about the method</div>. <div class="source ">Brief. Bioinform</div>., <div class="volume">11</div>, <div class="fpage">440</div>–<div class="lpage">453</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DNA%20barcoding%3A%20a%20six-question%20tour%20to%20improve%20users%E2%80%99%20awareness%20about%20the%20method&amp;author=M.%20Casiraghi&amp;publication_year=2010&amp;journal=Brief.%20Bioinform&amp;volume=11&amp;pages=440-453" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bib/bbq003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbq003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbib%2Fbbq003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20156987" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DNA%20barcoding%3A%20a%20six-question%20tour%20to%20improve%20users%E2%80%99%20awareness%20about%20the%20method&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B10" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dayrat</div> <div class="given-names">B.</div></div></div> (<div class="year">2005</div>) 
<div class="article-title">Towards integrative taxonomy</div>. <div class="source ">Biol. J. Linnean Soc</div>., <div class="volume">85</div>, <div class="fpage">407</div>–<div class="lpage">415</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Towards%20integrative%20taxonomy&amp;author=B.%20Dayrat&amp;publication_year=2005&amp;journal=Biol.%20J.%20Linnean%20Soc&amp;volume=85&amp;pages=407-415" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/bij.2005.85.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fbij.2005.85.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fbij.2005.85.issue-3"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Towards%20integrative%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B11" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Queiroz</div> <div class="given-names">K.</div></div></div> (<div class="year">2007</div>) 
<div class="article-title">Species concepts and species delimitation</div>. <div class="source ">Syst. Biol</div>., <div class="volume">56</div>, <div class="fpage">879</div>–<div class="lpage">886</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Species%20concepts%20and%20species%20delimitation&amp;author=K.%20De%20Queiroz&amp;publication_year=2007&amp;journal=Syst.%20Biol&amp;volume=56&amp;pages=879-886" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150701701083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150701701083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150701701083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18027281" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Species%20concepts%20and%20species%20delimitation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B12" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dupuis</div> <div class="given-names">J.R.</div></div></div> et al.&nbsp; (<div class="year">2012</div>) 
<div class="article-title">Multi-locus species delimitation in closely related animals and fungi: one marker is not enough</div>. <div class="source ">Mol. Ecol</div>., <div class="volume">21</div>, <div class="fpage">4422</div>–<div class="lpage">4436</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Multi-locus%20species%20delimitation%20in%20closely%20related%20animals%20and%20fungi%3A%20one%20marker%20is%20not%20enough&amp;author=J.R.%20Dupuis&amp;publication_year=2012&amp;journal=Mol.%20Ecol&amp;volume=21&amp;pages=4422-4436" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1365-294X.2012.05642.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2012.05642.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1365-294X.2012.05642.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22891635" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Multi-locus%20species%20delimitation%20in%20closely%20related%20animals%20and%20fungi%3A%20one%20marker%20is%20not%20enough&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B13" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R.C.</div></div></div> (<div class="year">2010</div>) 
<div class="article-title">Search and clustering orders of magnitude faster than BLAST</div>. <div class="source ">Bioinformatics</div>, <div class="volume">26</div>, <div class="fpage">2460</div>–<div class="lpage">2461</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;author=R.C.%20Edgar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2460-2461" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B14" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Esselstyn</div> <div class="given-names">J.A.</div></div></div> et al.&nbsp; (<div class="year">2012</div>). 
<div class="article-title">Single-locus species delimitation: a test of 
the mixed Yule–coalescent model, with an empirical application to 
Philippine round-leaf bats</div>. <div class="source ">Proc. R. Soc. Lond. B: Biol. Sci</div>. doi:10.1098/rspb.2012.0705.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Single-locus%20species%20delimitation%3A%20a%20test%20of%20the%20mixed%20Yule%E2%80%93coalescent%20model%2C%20with%20an%20empirical%20application%20to%20Philippine%20round-leaf%20bats&amp;aulast=Esselstyn&amp;date=2012" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Single-locus%20species%20delimitation%3A%20a%20test%20of%20the%20mixed%20Yule%E2%80%93coalescent%20model%2C%20with%20an%20empirical%20application%20to%20Philippine%20round-leaf%20bats&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B15" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujisawa</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Barraclough</div> <div class="given-names">T.G.</div></div></div> (<div class="year">2013</div>) 
<div class="article-title">Delimiting species using single-locus data 
and the generalized mixed Yule coalescent approach: a revised method and
 evaluation on simulated data sets</div>. <div class="source ">Syst. Biol</div>., <div class="volume">62</div>, <div class="fpage">707</div>–<div class="lpage">724</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Delimiting%20species%20using%20single-locus%20data%20and%20the%20generalized%20mixed%20Yule%20coalescent%20approach%3A%20a%20revised%20method%20and%20evaluation%20on%20simulated%20data%20sets&amp;author=T.%20Fujisawa&amp;author=T.G.%20Barraclough&amp;publication_year=2013&amp;journal=Syst.%20Biol&amp;volume=62&amp;pages=707-724" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syt033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyt033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyt033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23681854" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Delimiting%20species%20using%20single-locus%20data%20and%20the%20generalized%20mixed%20Yule%20coalescent%20approach%3A%20a%20revised%20method%20and%20evaluation%20on%20simulated%20data%20sets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B16" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujisawa</div> <div class="given-names">T.</div></div></div> et al.&nbsp; (<div class="year">2016</div>) 
<div class="article-title">A rapid and scalable method for multilocus species delimitation using Bayesian model comparison and rooted triplets</div>. <div class="source ">Syst. Biol</div>., <div class="volume">65</div>, <div class="fpage">759</div>–<div class="lpage">771</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20and%20scalable%20method%20for%20multilocus%20species%20delimitation%20using%20Bayesian%20model%20comparison%20and%20rooted%20triplets&amp;author=T.%20Fujisawa&amp;publication_year=2016&amp;journal=Syst.%20Biol&amp;volume=65&amp;pages=759-771" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syw028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyw028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyw028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27055648" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20rapid%20and%20scalable%20method%20for%20multilocus%20species%20delimitation%20using%20Bayesian%20model%20comparison%20and%20rooted%20triplets&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B17" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Funk</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Omland</div> <div class="given-names">K.E.</div></div></div> (<div class="year">2003</div>) 
<div class="article-title">Species-level paraphyly and polyphyly: frequency, causes, and consequences, with insights from animal mitochondrial DNA</div>. <div class="source ">Annu. Rev. Ecol. Evol. Syst</div>., <div class="volume">34</div>, <div class="fpage">397</div>–<div class="lpage">423</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Species-level%20paraphyly%20and%20polyphyly%3A%20frequency%2C%20causes%2C%20and%20consequences%2C%20with%20insights%20from%20animal%20mitochondrial%20DNA&amp;author=D.J.%20Funk&amp;author=K.E.%20Omland&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Ecol.%20Evol.%20Syst&amp;volume=34&amp;pages=397-423" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1146/annurev.ecolsys.34.011802.132421" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ecolsys.34.011802.132421" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1146%2Fannurev.ecolsys.34.011802.132421"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Species-level%20paraphyly%20and%20polyphyly%3A%20frequency%2C%20causes%2C%20and%20consequences%2C%20with%20insights%20from%20animal%20mitochondrial%20DNA&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B18" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hao</div> <div class="given-names">X.</div></div></div> et al.&nbsp; (<div class="year">2011</div>) 
<div class="article-title">Clustering 16S rRNA for OTU prediction: a method of unsupervised Bayesian clustering</div>. <div class="source ">Bioinformatics</div>, <div class="volume">27</div>, <div class="fpage">611</div>–<div class="lpage">618</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Clustering%2016S%20rRNA%20for%20OTU%20prediction%3A%20a%20method%20of%20unsupervised%20Bayesian%20clustering&amp;author=X.%20Hao&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=611-618" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btq725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtq725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21233169" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Clustering%2016S%20rRNA%20for%20OTU%20prediction%3A%20a%20method%20of%20unsupervised%20Bayesian%20clustering&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B19" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harbach</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Kitching</div> <div class="given-names">I.J.</div></div></div> (<div class="year">2016</div>) 
<div class="article-title">The phylogeny of Anophelinae revisited: inferences about the origin and classification of Anopheles (Diptera: Culicidae)</div>. <div class="source ">Zool. Scr</div>., <div class="volume">45</div>, <div class="fpage">34</div>–<div class="lpage">47</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20phylogeny%20of%20Anophelinae%20revisited%3A%20inferences%20about%20the%20origin%20and%20classification%20of%20Anopheles%20(Diptera%3A%20Culicidae)&amp;author=R.E.%20Harbach&amp;author=I.J.%20Kitching&amp;publication_year=2016&amp;journal=Zool.%20Scr&amp;volume=45&amp;pages=34-47" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/zsc.12137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fzsc.12137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fzsc.12137"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20phylogeny%20of%20Anophelinae%20revisited%3A%20inferences%20about%20the%20origin%20and%20classification%20of%20Anopheles%20(Diptera%3A%20Culicidae)&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B20" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hebert</div> <div class="given-names">P.D.</div></div></div> et al.&nbsp; (<div class="year">2003</div>) 
<div class="article-title">Biological identifications through DNA barcodes</div>. <div class="source ">Proc. R. Soc. Lond. B: Biol. Sci</div>., <div class="volume">270</div>, <div class="fpage">313</div>–<div class="lpage">321</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Biological%20identifications%20through%20DNA%20barcodes&amp;author=P.D.%20Hebert&amp;publication_year=2003&amp;journal=Proc.%20R.%20Soc.%20Lond.%20B%3A%20Biol.%20Sci&amp;volume=270&amp;pages=313-321" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rspb.2002.2218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2002.2218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frspb.2002.2218"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Biological%20identifications%20through%20DNA%20barcodes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B21" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hebert</div> <div class="given-names">P.D.</div></div></div> et al.&nbsp; (<div class="year">2004</div>) 
<div class="article-title">Identification of birds through DNA barcodes</div>. <div class="source ">PLoS Biol</div>., <div class="volume">2</div>, <div class="fpage">1657</div>–<div class="lpage">1663</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20birds%20through%20DNA%20barcodes&amp;author=P.D.%20Hebert&amp;publication_year=2004&amp;journal=PLoS%20Biol&amp;volume=2&amp;pages=1657-1663" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pbio.0020312" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020312" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pbio.0020312"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Identification%20of%20birds%20through%20DNA%20barcodes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B22" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hudson</div> <div class="given-names">R.R.</div></div></div> (<div class="year">1990</div>) 
<div class="article-title">Gene genealogies and the coalescent process</div>. <div class="source ">Oxford Surv. Evol. Biol</div>., <div class="volume">7</div>, <div class="fpage">44.</div><div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Gene%20genealogies%20and%20the%20coalescent%20process&amp;aulast=Hudson&amp;date=1990&amp;spage=44.&amp;volume=7" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20genealogies%20and%20the%20coalescent%20process&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B23" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">G.</div></div></div> et al.&nbsp; (<div class="year">2015</div>) 
<div class="article-title">DISSECT: an assignment-free Bayesian discovery method for species delimitation under the multispecies coalescent</div>. <div class="source ">Bioinformatics</div>, <div class="volume">31</div>, <div class="fpage">991</div>–<div class="lpage">998</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DISSECT%3A%20an%20assignment-free%20Bayesian%20discovery%20method%20for%20species%20delimitation%20under%20the%20multispecies%20coalescent&amp;author=G.%20Jones&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=991-998" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25422051" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DISSECT%3A%20an%20assignment-free%20Bayesian%20discovery%20method%20for%20species%20delimitation%20under%20the%20multispecies%20coalescent&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B24" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Standley</div> <div class="given-names">D.M.</div></div></div> (<div class="year">2013</div>) 
<div class="article-title">MAFFT multiple sequence alignment software version 7: improvements in performance and usability</div>. <div class="source ">Mol. Biol. Evol</div>., <div class="volume">30</div>, <div class="fpage">772</div>–<div class="lpage">780</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;author=K.%20Katoh&amp;author=D.M.%20Standley&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=772-780" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/mst010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmst010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B25" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lin</div> <div class="given-names">X.</div></div></div> et al.&nbsp; (<div class="year">2015</div>) 
<div class="article-title">Exploring Genetic Divergence in a Species-Rich Insect Genus Using 2790 DNA Barcodes</div>. <div class="source ">PLoS ONE</div>, <div class="volume">10</div>, <div class="fpage">1</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Exploring%20Genetic%20Divergence%20in%20a%20Species-Rich%20Insect%20Genus%20Using%202790%20DNA%20Barcodes&amp;aulast=Lin&amp;date=2015&amp;spage=1&amp;epage=24&amp;volume=10" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Exploring%20Genetic%20Divergence%20in%20a%20Species-Rich%20Insect%20Genus%20Using%202790%20DNA%20Barcodes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B26" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maddison</div> <div class="given-names">W.P.</div></div></div> (<div class="year">1997</div>) 
<div class="article-title">Gene trees in species trees</div>. <div class="source ">Syst. Biol</div>., <div class="volume">46</div>, <div class="fpage">523</div>–<div class="lpage">536</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Gene%20trees%20in%20species%20trees&amp;author=W.P.%20Maddison&amp;publication_year=1997&amp;journal=Syst.%20Biol&amp;volume=46&amp;pages=523-536" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/46.3.523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2F46.3.523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2F46.3.523"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Gene%20trees%20in%20species%20trees&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B27" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McKay</div> <div class="given-names">B.D.</div></div>, <div class="name string-name"><div class="surname">Zink</div> <div class="given-names">R.M.</div></div></div> (<div class="year">2010</div>) 
<div class="article-title">The causes of mitochondrial DNA gene tree paraphyly in birds</div>. <div class="source ">Mol. Phylogenet. Evol</div>., <div class="volume">54</div>, <div class="fpage">647</div>–<div class="lpage">650</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20causes%20of%20mitochondrial%20DNA%20gene%20tree%20paraphyly%20in%20birds&amp;author=B.D.%20McKay&amp;author=R.M.%20Zink&amp;publication_year=2010&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=54&amp;pages=647-650" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.ympev.2009.08.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2009.08.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.ympev.2009.08.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19716428" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20causes%20of%20mitochondrial%20DNA%20gene%20tree%20paraphyly%20in%20birds&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B28" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McQuarrie</div> <div class="given-names">A.D.</div></div>, <div class="name string-name"><div class="surname">Tsai</div> <div class="given-names">C.L.</div></div></div> (<div class="year">1998</div>). <div class="source ">Regression and Time Series Model Selection</div>.: 
<div class="publisher-name">World Scientific</div>, 
<div class="publisher-loc">Singapore</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Regression%20and%20Time%20Series%20Model%20Selection&amp;author=A.D.%20McQuarrie&amp;author=C.L.%20Tsai&amp;publication_year=1998&amp;book=Regression%20and%20Time%20Series%20Model%20Selection" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1142/3573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1142%2F3573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1142%2F3573"> </span></div><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Regression%20and%20Time%20Series%20Model%20Selection&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Regression%20and%20Time%20Series%20Model%20Selection&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Regression%20and%20Time%20Series%20Model%20Selection">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B29" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Monaghan</div> <div class="given-names">M.T.</div></div></div> et al.&nbsp; (<div class="year">2009</div>) 
<div class="article-title">Accelerated species inventory on Madagascar using coalescent-based models of species delineation</div>. <div class="source ">Syst. Biol</div>., <div class="volume">58</div>, <div class="fpage">298</div>–<div class="lpage">311</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Accelerated%20species%20inventory%20on%20Madagascar%20using%20coalescent-based%20models%20of%20species%20delineation&amp;author=M.T.%20Monaghan&amp;publication_year=2009&amp;journal=Syst.%20Biol&amp;volume=58&amp;pages=298-311" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/syp027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyp027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsyp027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525585" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Accelerated%20species%20inventory%20on%20Madagascar%20using%20coalescent-based%20models%20of%20species%20delineation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B30" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Obbard</div> <div class="given-names">D.J.</div></div></div> et al.&nbsp; (<div class="year">2012</div>) 
<div class="article-title">Estimating divergence dates and substitution rates in the Drosophila phylogeny</div>. <div class="source ">Mol. Biol. Evol.</div>, page mss150.<div class="citation-links"><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=article&amp;atitle=Estimating%20divergence%20dates%20and%20substitution%20rates%20in%20the%20Drosophila%20phylogeny&amp;aulast=Obbard&amp;date=2012" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Estimating%20divergence%20dates%20and%20substitution%20rates%20in%20the%20Drosophila%20phylogeny&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B31" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oliver</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Beattie</div> <div class="given-names">A.J.</div></div></div> (<div class="year">1993</div>) 
<div class="article-title">A possible method for the rapid assessment of biodiversity</div>. <div class="source ">Conserv. Biol</div>., <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">568</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20possible%20method%20for%20the%20rapid%20assessment%20of%20biodiversity&amp;author=I.%20Oliver&amp;author=A.J.%20Beattie&amp;publication_year=1993&amp;journal=Conserv.%20Biol&amp;volume=7&amp;pages=562-568" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1046/j.1523-1739.1993.07030562.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1523-1739.1993.07030562.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1046%2Fj.1523-1739.1993.07030562.x"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20possible%20method%20for%20the%20rapid%20assessment%20of%20biodiversity&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B32" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pons</div> <div class="given-names">J.</div></div></div> et al.&nbsp; (<div class="year">2006</div>) 
<div class="article-title">Sequence-based species delimitation for the DNA taxonomy of undescribed insects</div>. <div class="source ">Syst. Biol</div>., <div class="volume">55</div>, <div class="fpage">595</div>–<div class="lpage">609</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20species%20delimitation%20for%20the%20DNA%20taxonomy%20of%20undescribed%20insects&amp;author=J.%20Pons&amp;publication_year=2006&amp;journal=Syst.%20Biol&amp;volume=55&amp;pages=595-609" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150600852011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150600852011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150600852011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16967577" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Sequence-based%20species%20delimitation%20for%20the%20DNA%20taxonomy%20of%20undescribed%20insects&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B33" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Puillandre</div> <div class="given-names">N.</div></div></div> et al.&nbsp; (<div class="year">2012</div>) 
<div class="article-title">ABGD, Automatic Barcode Gap Discovery for primary species delimitation</div>. <div class="source ">Mol. Ecol</div>., <div class="volume">21</div>, <div class="fpage">1864</div>–<div class="lpage">1877</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=ABGD%2C%20Automatic%20Barcode%20Gap%20Discovery%20for%20primary%20species%20delimitation&amp;author=N.%20Puillandre&amp;publication_year=2012&amp;journal=Mol.%20Ecol&amp;volume=21&amp;pages=1864-1877" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/j.1365-294X.2011.05239.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2011.05239.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fj.1365-294X.2011.05239.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21883587" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:ABGD%2C%20Automatic%20Barcode%20Gap%20Discovery%20for%20primary%20species%20delimitation&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B34" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qin</div> <div class="given-names">J.</div></div></div> et al.&nbsp; (<div class="year">2015</div>) 
<div class="article-title">Mitochondrial phylogenomics and genetic 
relationships of closely related pine moth (Lasiocampidae: Dendrolimus) 
species in China, using whole mitochondrial genomes</div>. <div class="source ">BMC Genomics</div>, <div class="volume">16</div>, <div class="fpage">1.</div><div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20phylogenomics%20and%20genetic%20relationships%20of%20closely%20related%20pine%20moth%20(Lasiocampidae%3A%20Dendrolimus)%20species%20in%20China%2C%20using%20whole%20mitochondrial%20genomes&amp;author=J.%20Qin&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=1." target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1186/1471-2164-16-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-16-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1186%2F1471-2164-16-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25553907" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Mitochondrial%20phylogenomics%20and%20genetic%20relationships%20of%20closely%20related%20pine%20moth%20(Lasiocampidae%3A%20Dendrolimus)%20species%20in%20China%2C%20using%20whole%20mitochondrial%20genomes&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B35" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ratnasingham</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hebert</div> <div class="given-names">P.D.</div></div></div> (<div class="year">2013</div>) 
<div class="article-title">A DNA-based registry for all animal species: The Barcode Index Number (BIN) System</div>. <div class="source ">PloS One</div>, <div class="volume">8</div>, <div class="fpage">e66213.</div><div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20DNA-based%20registry%20for%20all%20animal%20species%3A%20The%20Barcode%20Index%20Number%20(BIN)%20System&amp;author=S.%20Ratnasingham&amp;author=P.D.%20Hebert&amp;publication_year=2013&amp;journal=PloS%20One&amp;volume=8&amp;pages=e66213." target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1371/journal.pone.0066213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0066213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1371%2Fjournal.pone.0066213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23861743" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20DNA-based%20registry%20for%20all%20animal%20species%3A%20The%20Barcode%20Index%20Number%20(BIN)%20System&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B36" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rijsbergen</div> <div class="given-names">C.J.V.</div></div></div> (<div class="year">1979</div>). <div class="source ">Information Retrieval</div>, 2nd edn. 
<div class="publisher-name">Butterworth-Heinemann</div>, 
<div class="publisher-loc">Newton, MA, USA</div>.<div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=Information%20Retrieval&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=Information%20Retrieval&amp;aulast=Rijsbergen&amp;date=1979" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Information%20Retrieval&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=Information%20Retrieval">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B37" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ronquist</div> <div class="given-names">F.</div></div></div> et al.&nbsp; (<div class="year">2012</div>) 
<div class="article-title">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space</div>. <div class="source ">Systematic Biology</div>, <div class="volume">61</div>, <div class="fpage">539</div>–<div class="lpage">542</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=MrBayes%203.2%3A%20efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space&amp;author=F.%20Ronquist&amp;publication_year=2012&amp;journal=Systematic%20Biology&amp;volume=61&amp;pages=539-542" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/sysbio/sys029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fsysbio%2Fsys029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22357727" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MrBayes%203.2%3A%20efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B38" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smith</div> <div class="given-names">M.A.</div></div></div> et al.&nbsp; (<div class="year">2005</div>) 
<div class="article-title">DNA barcoding for effective biodiversity assessment of a hyperdiverse arthropod group: the ants of Madagascar</div>. <div class="source ">Philos. Trans. R. Soc. B: Biol. Sci</div>., <div class="volume">360</div>, <div class="fpage">1825</div>–<div class="lpage">1834</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DNA%20barcoding%20for%20effective%20biodiversity%20assessment%20of%20a%20hyperdiverse%20arthropod%20group%3A%20the%20ants%20of%20Madagascar&amp;author=M.A.%20Smith&amp;publication_year=2005&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B%3A%20Biol.%20Sci&amp;volume=360&amp;pages=1825-1834" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rstb.2005.1714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2005.1714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frstb.2005.1714"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DNA%20barcoding%20for%20effective%20biodiversity%20assessment%20of%20a%20hyperdiverse%20arthropod%20group%3A%20the%20ants%20of%20Madagascar&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B39" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> (<div class="year">2014</div>) 
<div class="article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</div>. <div class="source ">Bioinformatics</div>, <div class="volume">30</div>, <div class="fpage">1312</div>–<div class="lpage">1313</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1312-1313" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btu033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtu033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B40" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div></div> et al.&nbsp; (<div class="year">2011</div>) <div class="source ">MEGA 5: Molecular Evolutionary Genetics Analysis</div>. 
<div class="publisher-name">Center for Evolutionary Functional Genomics: The Biodesign Institute</div>, 
<div class="publisher-loc">Tempe, AZ</div>, pp. <div class="fpage">85287</div>–<div class="lpage">85301</div><div class="citation-links"><div class="google-preview-ref-link js-google-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.google.com/search?q=MEGA%205%3A%20Molecular%20Evolutionary%20Genetics%20Analysis&amp;btnG=Search+Books&amp;tbm=bks&amp;tbo=1" target="_blank">Google Preview</a></div><div class="xslopenurl" style="display:none"><span class="js-inst-open-url-holders-nodoi"><a class="openInAnotherWindow js-open-url-link" target="_blank" data-href-template="{targetURL}?sid=oup:orr&amp;genre=book&amp;title=MEGA%205%3A%20Molecular%20Evolutionary%20Genetics%20Analysis&amp;aulast=Tamura&amp;date=2011&amp;spage=85287&amp;epage=85301" href="#"></a></span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:MEGA%205%3A%20Molecular%20Evolutionary%20Genetics%20Analysis&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div><div class="copac-reference-ref-link js-copac-preview-ref-link" style="display:none" data-pubtype="book"><span class="inst-copac"><a class="openInAnotherWindow" target="_blank" href="http://copac.ac.uk/search?ti=MEGA%205%3A%20Molecular%20Evolutionary%20Genetics%20Analysis">COPAC</a></span></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B41" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">C.Q.</div></div></div> et al.&nbsp; (<div class="year">2014</div>) 
<div class="article-title">Effects of phylogenetic reconstruction method on the robustness of species delimitation using single-locus data</div>. <div class="source ">Methods Ecol. Evol</div>., <div class="volume">5</div>, <div class="fpage">1086</div>–<div class="lpage">1094</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20phylogenetic%20reconstruction%20method%20on%20the%20robustness%20of%20species%20delimitation%20using%20single-locus%20data&amp;author=C.Q.%20Tang&amp;publication_year=2014&amp;journal=Methods%20Ecol.%20Evol&amp;volume=5&amp;pages=1086-1094" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/mee3.2014.5.issue-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmee3.2014.5.issue-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fmee3.2014.5.issue-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25821577" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Effects%20of%20phylogenetic%20reconstruction%20method%20on%20the%20robustness%20of%20species%20delimitation%20using%20single-locus%20data&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B42" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div></div> et al.&nbsp; (<div class="year">2003</div>) 
<div class="article-title">A plea for DNA taxonomy</div>. <div class="source ">Trends Ecol. Evol</div>., <div class="volume">18</div>, <div class="fpage">70</div>–<div class="lpage">74</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20plea%20for%20DNA%20taxonomy&amp;author=D.%20Tautz&amp;publication_year=2003&amp;journal=Trends%20Ecol.%20Evol&amp;volume=18&amp;pages=70-74" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/S0169-5347(02)00041-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(02)00041-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2FS0169-5347(02)00041-1"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20plea%20for%20DNA%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B43" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Valentini</div> <div class="given-names">A.</div></div></div> et al.&nbsp; (<div class="year">2009</div>) 
<div class="article-title">DNA barcoding for ecologists</div>. <div class="source ">Trends Ecol. Evol</div>., <div class="volume">24</div>, <div class="fpage">110</div>–<div class="lpage">117</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=DNA%20barcoding%20for%20ecologists&amp;author=A.%20Valentini&amp;publication_year=2009&amp;journal=Trends%20Ecol.%20Evol&amp;volume=24&amp;pages=110-117" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1016/j.tree.2008.09.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2008.09.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1016%2Fj.tree.2008.09.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19100655" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:DNA%20barcoding%20for%20ecologists&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B44" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vogler</div> <div class="given-names">A.P.</div></div>, <div class="name string-name"><div class="surname">Monaghan</div> <div class="given-names">M.T.</div></div></div> (<div class="year">2007</div>) 
<div class="article-title">Recent advances in DNA taxonomy</div>. <div class="source ">J. Zool. Syst. Evol. Res</div>., <div class="volume">45</div>, <div class="fpage">1</div>–<div class="lpage">10</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Recent%20advances%20in%20DNA%20taxonomy&amp;author=A.P.%20Vogler&amp;author=M.T.%20Monaghan&amp;publication_year=2007&amp;journal=J.%20Zool.%20Syst.%20Evol.%20Res&amp;volume=45&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1111/jzs.2007.45.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjzs.2007.45.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1111%2Fjzs.2007.45.issue-1"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Recent%20advances%20in%20DNA%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B45" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Will</div> <div class="given-names">K.W.</div></div></div> et al.&nbsp; (<div class="year">2005</div>) 
<div class="article-title">The perils of DNA barcoding and the need for integrative taxonomy</div>. <div class="source ">Syst. Biol</div>., <div class="volume">54</div>, <div class="fpage">844</div>–<div class="lpage">851</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=The%20perils%20of%20DNA%20barcoding%20and%20the%20need%20for%20integrative%20taxonomy&amp;author=K.W.%20Will&amp;publication_year=2005&amp;journal=Syst.%20Biol&amp;volume=54&amp;pages=844-851" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1080/10635150500354878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150500354878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1080%2F10635150500354878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16243769" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:The%20perils%20of%20DNA%20barcoding%20and%20the%20need%20for%20integrative%20taxonomy&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B46" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Rannala</div> <div class="given-names">B.</div></div></div> (<div class="year">2014</div>) 
<div class="article-title">Unguided species delimitation using DNA sequence data from multiple loci</div>. <div class="source ">Mol. Biol. Evol</div>., <div class="volume">31</div>, <div class="fpage">3125</div>–<div class="lpage">3135</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=Unguided%20species%20delimitation%20using%20DNA%20sequence%20data%20from%20multiple%20loci&amp;author=Z.%20Yang&amp;author=B.%20Rannala&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=3125-3135" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/molbev/msu279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fmolbev%2Fmsu279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25274273" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:Unguided%20species%20delimitation%20using%20DNA%20sequence%20data%20from%20multiple%20loci&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B47" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yule</div> <div class="given-names">G.U.</div></div></div> (<div class="year">1925</div>) 
<div class="article-title">A mathematical theory of evolution, based on the conclusions of Dr. J. C. Willis, F.R.S</div>. <div class="source ">Philos. Trans. R. Soc. Lond.. Ser. B: Containing Papers of a Biological Character</div>, <div class="volume">213</div>, <div class="fpage">21</div>–<div class="lpage">87</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20mathematical%20theory%20of%20evolution%2C%20based%20on%20the%20conclusions%20of%20Dr.%20J.%20C.%20Willis%2C%20F.R.S&amp;author=G.U.%20Yule&amp;publication_year=1925&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond..%20Ser.%20B%3A%20Containing%20Papers%20of%20a%20Biological%20Character&amp;volume=213&amp;pages=21-87" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1098/rstb.1925.0002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1925.0002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1098%2Frstb.1925.0002"> </span></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20mathematical%20theory%20of%20evolution%2C%20based%20on%20the%20conclusions%20of%20Dr.%20J.%20C.%20Willis%2C%20F.R.S&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div><div content-id="btx025-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="btx025-B48" class="ref-content "><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div></div> et al.&nbsp; (<div class="year">2013</div>) 
<div class="article-title">A general species delimitation method with applications to phylogenetic placements</div>. <div class="source ">Bioinformatics</div>, <div class="volume">29</div>, <div class="fpage">2869</div>–<div class="lpage">2876</div>.<div class="citation-links"></div><div class="citation-links"><div class="google-scholar-ref-link"><a class="openInAnotherWindow" href="https://scholar.google.com/scholar_lookup?title=A%20general%20species%20delimitation%20method%20with%20applications%20to%20phylogenetic%20placements&amp;author=J.%20Zhang&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=2869-2876" target="_blank">Google Scholar</a></div><div class="crossref-doi js-ref-link"><a class="openInAnotherWindow" href="http://dx.doi.org/10.1093/bioinformatics/btt499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a class="openInAnotherWindow" href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtt499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetid="10.1093%2Fbioinformatics%2Fbtt499"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23990417" class="link link-pub-id openInAnotherWindow" target="_blank">PubMed</a></div><div class="worldcat-reference-ref-link js-worldcat-preview-ref-link" style="display:none"><a class="openInAnotherWindow" href="https://www.worldcat.org/search?q=ti:A%20general%20species%20delimitation%20method%20with%20applications%20to%20phylogenetic%20placements&amp;qt=advanced&amp;dblist=638" target="_blank">WorldCat</a></div>&nbsp;</div></div></div></div></div></div>        <!-- /foreach -->



        

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri openInAnotherWindow" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userhasaccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->

 
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
    

 
    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        
 
    </div>
    <div class="widget widget-ArticleOpenScienceBadges widget-instance-OUP_Article_OpenScienceBadge_Widget">
        
 
    </div>

                <div id="ContentTabFilteredView"></div>
                <div class="downloadImagesppt js-download-images-ppt st-download-images-ppt">
                    <a id="lnkDownloadAllImages" class="js-download-all-images-link" href="https://academic.oup.com/DownloadFile/DownloadImage.aspx?image=&amp;PPTtype=SlideSet&amp;ar=2929345&amp;xsltPath=~/UI/app/XSLT&amp;siteId=5139">Download all figures</a>
                </div>
    <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        
        <h2 scrollto-destination="supplementary-data" id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/bioinformatics/33/11/10.1093_bioinformatics_btx025/2/btx025_supp.zip?Expires=1585828431&amp;Signature=VsqSeFZ30EHAY7GOK80mjFM6tYpctZ3BKMCzjRmongoRs4zfOX3ioFa2rtdCF6L0au8q3pFO~47Jr4KNkeCgha8msA0KHemGeo1zm~mvmVxFtLZOmaBxc784hTiSqPma8XhX9Rlyaciy071~XCb0Zjd4SWpsGMcGlZhzRRgSpGNtRgcJ2oRTXoeCqIVSNF3tOfajiZjgPQCDBlr1r85fxvdFQPP3AqdZ6BEIDf-qhPnWkx-QjJfiaJK~UnyKiDDA5ghFjN4IeqPGGqA1lQvppL9nkSIP80nKzrsXkeTSbHWvCCMAMLkq06hM06VOaNeKnfQTg~J6io7qTuz2ETdkUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label">Supplementary Data</div></a> - zip file</div>
 
    </div>
                    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
         
    </div>

                <div class="comments">
    <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        

 
    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
         
    </div>

                </div>
            </div>
        </div>

    </div>
</div>
<div id="Sidebar" class="page-column page-column--right">
    <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad2_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="11"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        



    <div class="artmet-wrapper horizontal-artmet">

        <div class="artmet-condensed-wrap clearfix">

            <div class="artmet-condensed-stats clearfix">
                    <div class="artmet-item artmet-views">
                        <span class="artmet-number">3,016</span>
                        <span class="artmet-text">Views</span>
                    </div>

                <div class="artmet-item artmet-citations">
                    <span class="artmet-number">
                            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000402131500006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">122</a>
                    </span>
                    <span class="artmet-text">Citations</span>
                </div>
                    <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId" runat="server" class="altmetric-embed" data-badge-type="donut" data-hide-no-mentions="false" data-doi="10.1093/bioinformatics/btx025" data-uuid="1e823c4c-0932-86b0-8858-7273ed95ecfa"><a target="_self" href="https://www.altmetric.com/details.php?domain=academic.oup.com&amp;citation_id=21484836" style="display:inline-block;">
        <img alt="Article has an altmetric score of 10" src="https://badges.altmetric.com/?size=128&amp;score=10&amp;types=bbtttttt" style="border:0; margin:0; max-width: none;" width="64px" height="64px">
</a></div>
         <script type="text/javascript" src="embed.js"></script>
 
    </div>

                    </div>



            </div>

                <div class="artmet-modal-trigger-wrap clearfix">
                    <a class="artmet-modal-trigger" data-article-id="2929345"><i class="icon-metrics"></i><span>View Metrics</span></a>
                </div>
        </div>
            <div class="artmet-modal" id="MetricsModal">
                <div class="artmet-modal-contents">
                    <a class="artmet-close-modal">×</a>
                </div>
            </div>
    </div>

 
    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert" data-userloggedin="False" data-alerttype="1">Article activity alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts" data-userloggedin="False" data-alerttype="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert" data-userloggedin="False" data-alerttype="5">New issue alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic" data-userloggedin="False" data-additionalurl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal="">
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>
 
    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
         
    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
         
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000402131500006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fbioinformatics%2farticle%2f33%2f11%2f1630%2f2929345">Google Scholar</a>
                </li>
        </ul>
    </div>
 
    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <div class="rail-widget_wrap vt-articles-cited-by">
    <h3 class="article-cited-title">Citing articles via</h3>
    <div class="widget-links_wrap">
            <div class="article-cited-link-wrap web-of-science">
                <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000402131500006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="">Web of Science (122)</a>
            </div>
                    <div class="article-cited-link-wrap google-scholar-url">
                <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F33%2F11%2F1630%2F2929345" target="">Google Scholar</a>
            </div>
                    <div class="article-cited-link-wrap crossref-citedby">
                <a href="https://academic.oup.com/bioinformatics/crossref-citedby/2929345">Crossref</a>
            </div>
    </div>
</div> 
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa129/5758262?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
Connecting mathematical models to genomes: Joint estimation of model 
parameters and genome-wide marker effects on these parameters           
                     </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa126/5758261?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
Discovering and interpreting transcriptomic drivers of imaging traits using neural networks                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa127/5758260?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
Rhapsody: Predicting the pathogenicity of human missense variants                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa125/5758259?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
Composite Kernel Association Test (CKAT) for SNP-set Joint Assessment of
 Genotype and Genotype-by-treatment Interaction in Pharmacogenetics 
Studies                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
                <div class="widget-dynamic-entry">
                    

                                            <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa124/5756210?searchresult=1">
                                                            <div class="widget-dynamic-journal-title">
DeepMAsED: Evaluating the quality of metagenomic assemblies                                </div>
                        </a>
                        <div class="widget-dynamic-journal-image-synopsis">
                                <div class="widget-dynamic-journal-synopsis">
                                    
                                </div> <!-- /.widget-dynamic-journal-synopsis -->
                        </div>

                </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>
 
    </div>
    <div class="widget widget-JobTarget widget-instance-Article_RightRailB0Article_RightRail_JobTarget">
        <div class="wrapper">
    <div class="header">
        <a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/">Looking for your next opportunity?</a>
    </div>

        <div id="jobs" class="jobs-list">
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/specialist-physician-for-older-people/43778538/20102/">Specialist Physician for Older People</a></div>
                    <div class="job-location">
                        Whangarei, Northland
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/physician/53048008/20102/">Physician</a></div>
                    <div class="job-location">
                        Silver Spring, Maryland
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/sleep-medicine-opportunity/53031087/20102/">Sleep Medicine Opportunity</a></div>
                    <div class="job-location">
                        Columbus, Ohio
                    </div>
                </div>
                <div class="job">
                    <div class="job-position"><a href="https://medicine-and-health-careernetwork.oxfordjournals.org/job/assistant-professor-and-associatefull-professor/52019081/20102/">Assistant Professor and Associate/Full Professor</a></div>
                    <div class="job-location">
                        Hershey, Pennsylvania
                    </div>
                </div>
        </div>
        <div class="all-jobs"><a href="http://medicine-and-health-careernetwork.oxfordjournals.org/jobseeker/search/results/">View all jobs</a></div>

    <div class="sponsor-and-jobnetwork-links">
        <div class="job-sponsor"><a href="http://global.oup.com/"><img src="oup-logo_002.svg" alt="OXFORD University Press"></a></div>
        <div class="job-network"><a href="http://globaljobs-careernetwork.oxfordjournals.org/home/index.cfm?site_id=20101"><img src="journals-careers-network.svg" alt="Journals Career Network"></a></div>
    </div>
</div>
 
    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad3_0__container__" style="border: 0pt none; width: 300px; height: 250px;"></div></div>
 
    </div>

</div>

</div>
<input id="hfArticleTitle" name="hfArticleTitle" value="Multi-rate Poisson tree processes for single-locus species delimitation under maximum likelihood and Markov chain Monte Carlo | Bioinformatics | Oxford Academic" type="hidden">
<input id="hfLeftNavStickyOffset" name="hfLeftNavStickyOffset" value="29" type="hidden">


            </div><!-- /.center-inner-row no-overflow -->
        </section>
    </div>
    <div class="mobile-mask">
    </div>
    <section class="footer_wrap vt-site-footer">
        
    <script type="text/javascript" src="api.js"></script>
    
    
    <script type="text/javascript" src="chartist.js"></script>

    <script type="text/javascript" src="article-page.js"></script>

    <script type="text/javascript">
        googletag.cmd.push(function () {
            
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/bioinformatics/btx025");
            googletag.enableServices();
        });
    </script>



        


    <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter" style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
<div id="google_ads_iframe_/116097782/bioinformatics_Supplement_Ad4_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>
 
    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                                <li class="link-1">
                                    <a href="https://academic.oup.com/bioinformatics/pages/About">About Bioinformatics</a>
                                </li>
                                <li class="link-2">
                                    <a href="https://academic.oup.com/bioinformatics/pages/Editorial_Board">Editorial Board</a>
                                </li>
                                <li class="link-3">
                                    <a href="https://academic.oup.com/bioinformatics/pages/General_Instructions">Author Guidelines</a>
                                </li>
                                <li class="link-4">
                                    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                                </li>
                                <li class="link-5">
                                    <a href="https://twitter.com/oxfordjournals">Twitter</a>
                                </li>
</ul><ul>                                <li class="link-1">
                                    <a href="https://academic.oup.com/bioinformatics/subscribe">Purchase</a>
                                </li>
                                <li class="link-2">
                                    <a href="http://www.oxfordjournals.org/en/library-recommendation-form.html">Recommend to your Library</a>
                                </li>
                                <li class="link-3">
                                    <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                                </li>
                                <li class="link-4">
                                    <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org/jobseeker/search/results/?t730=&amp;t732=470053&amp;t731=&amp;t733=&amp;t735=&amp;t737=&amp;max=25&amp;site_id=20106&amp;search=">Journals Career Network</a>
                                </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1460-2059</li>
                                            <li>Print ISSN 1367-4803</li>
                    <li>Copyright © 2020 Oxford University Press</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->
 
    </div>

<div class="oup-footer">
    <div class="center-inner-row">
    <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="https://academic.oup.com/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="https://academic.oup.com/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="https://academic.oup.com/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="https://academic.oup.com/journals/pages/authors">Authors</a></li>
    <li><a href="https://academic.oup.com/journals/pages/librarians">Librarians</a></li>
    <li><a href="https://academic.oup.com/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="https://academic.oup.com/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="https://academic.oup.com/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. 
It furthers the University's objective of excellence in research, 
scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="oup-logo.svg" alt="Oxford University Press" width="150" height="42">
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright © 2020 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sitemap">Site Map</a></li>
    <li><a href="https://academic.oup.com/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
    .sf-facet-list .sf-facet label, .sf-facet-list .taxonomy-label-wrap label {font-size: .9375rem;}
    .issue-pagination-wrap .pagination-container {float: right;}
</style>
    
    
     
    </div>

    </div>
</div>
<div class="ss-ui-only">

</div>

    </section>
</div>



    



    


    <div id="revealModal" class="reveal-modal" data-reveal="">
        <div id="revealContent"></div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="NeedSubscription" class="reveal-modal small" data-reveal="">
        <div class="subscription-needed">
            <h5>This Feature Is Available To Subscribers Only</h5>
            <p><a href="https://academic.oup.com/sign-in">Sign In</a> or <a href="https://academic.oup.com/my-account/register?siteId=5139&amp;returnUrl=%2fbioinformatics%2farticle%2f33%2f11%2f1630%2f2929345">Create an Account</a></p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>
    <div id="noAccessReveal" class="reveal-modal tiny" data-reveal="">
        <p>This PDF is available to Subscribers Only</p>
        <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal=""><span>View Article Abstract &amp; Purchase Options</span></a>
        <div class="issue-purchase-modal">
            <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
        </div>
        <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
    </div>


    


<script src="MathJax.js"></script>


<script src="polyfill.js"></script>

    <script type="text/javascript" src="footer.js"></script>
    <script async="async" src="addthis_widget.js"></script>


    

    

    
    




    


    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script src="ntpagetag.js"></script>
    <noscript>
        <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ" />
    </noscript>


    <div class="ad-banner js-ad-riser ad-banner-riser">

    <div class="widget widget-AdBlock widget-instance-RiserAd">
        <input class="hfAdBlockInfo" data-divid="adBlockRiser" data-slotname="" data-targetname="" data-width="" data-height="" data-enabled-on-mobile="True" data-adprovider-typeid="1" type="hidden">

 
    </div>

    </div>





<div data-reveal="math-jax-load" id="math-jax-modal" class="reveal-modal" style="display: none;"></div><script type="text/javascript" src="btx025.js"></script><div style="visibility: hidden; height: 1px; width: 1px; position: absolute; top: -9999px; z-index: 100000;" id="_atssh"><iframe id="_atssh545" title="AddThis utility frame" style="height: 1px; width: 1px; position: absolute; top: 0px; z-index: 100000; border: 0px none; left: 0px;" src="https://s7.addthis.com/static/sh.f48a1a04fe8dbf021b4cda1d.html#rand=0.6072316804029534&amp;iit=1582804435643&amp;tmr=load%3D1582804435616%26core%3D1582804435637%26main%3D1582804435640%26ifr%3D1582804435646&amp;cb=0&amp;cdn=0&amp;md=0&amp;kw=&amp;ab=-&amp;dh=academic.oup.com&amp;dr=&amp;du=https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F33%2F11%2F1630%2F2929345&amp;href=https%3A%2F%2Facademic.oup.com%2Fbioinformatics%2Farticle%2F33%2F11%2F1630%2F2929345&amp;dt=Multi-rate%20Poisson%20tree%20processes%20for%20single-locus%20species%20delimitation%20under%20maximum%20likelihood%20and%20Markov%20chain%20Monte%20Carlo&amp;dbg=0&amp;cap=tc%3D0%26ab%3D0&amp;inst=1&amp;jsl=1&amp;prod=undefined&amp;lng=en&amp;ogt=updated_time%2Cimage%2Curl%2Cdescription%2Ctitle%2Ctype%3Darticle%2Csite_name&amp;pc=men&amp;pub=xa-5265518246c10183&amp;ssl=1&amp;sid=5e57add348a391f0&amp;srf=0.01&amp;ver=300&amp;xck=0&amp;xtr=0&amp;og=site_name%3DOUP%2520Academic%26type%3Darticle%26title%3DMulti-rate%2520Poisson%2520tree%2520processes%2520for%2520single-locus%2520species%2520delimitation%2520under%2520maximum%2520likelihood%2520and%2520Markov%2520chain%2520Monte%2520Carlo%26description%3DAbstractMotivation.%2520%2520In%2520recent%2520years%252C%2520molecular%2520species%2520delimitation%2520has%2520become%2520a%2520routine%2520approach%2520for%2520quantifying%2520and%2520classifying%2520biodiversity.%2520Barcoding%2520metho%26url%3Dhttps%253A%252F%252Facademic.oup.com%252Fbioinformatics%252Farticle%252F33%252F11%252F1630%252F2929345%26image%3Dhttps%253A%252F%252Foup.silverchair-cdn.com%252Foup%252Fbackfile%252FContent_public%252FJournal%252Fbioinformatics%252FIssue%252F33%252F11%252F1%252Fm_bioinfo_33_11cover.png%253FExpires%253D1645876432%2526Signature%253DdCQe1DRKWVCh5U6UeCI2R41kHFqzD0v29TiFTmCtQ2nSQyJPgwQGlgdRMr5y~kJIzQFeQN4jxRHpkOixKc0Oq2H1iOMXYcxs6o3RghRmAhQPiPNjZWK4xhEFmC-N-EorHPD~6NxhV48~Z-6VUxphdOI9ieBt3eCuzQe900NKhgk04CsdmvQfBOOtP1J0KVY9PxOjKp1Dz0MkYF0~-~tD9No549c0vLxxI53v-ghk9ccxC0tEV1bcX7m74zCUduytUFsxsaWJZui12SX6WVJEfYDgclPvqkWrDVp5ScxGdHUkczFNMkSDfCImnQ9lbHukEollLG3ayhVQa5pjCkvgjA__%2526Key-Pair-Id%253DAPKAIE5G5CRDK6RD3PGA%26updated_time%3D2017-01-20&amp;csi=undefined&amp;rev=v8.28.3-wp&amp;ct=1&amp;xld=1&amp;xd=1"></iframe></div><style id="service-icons-0"></style><div style="position: absolute; width: 0px; height: 0px; overflow: hidden; padding: 0px; border: 0px none; margin: 0px;"><div id="MathJax_Font_Test" style="position: absolute; visibility: hidden; top: 0px; left: 0px; width: auto; padding: 0px; border: 0px none; margin: 0px; white-space: nowrap; text-align: left; text-indent: 0px; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; font-size: 40px; font-weight: normal; font-style: normal; font-size-adjust: none; font-family: MathJax_AMS, sans-serif;"></div></div></body></html>