<!DOCTYPE html>
<html class="js svg" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Automated high throughput animal CO1 metabarcode classification | Scientific Reports</title>
    
        
<link rel="preload" href="https://www.nature.com/static/fonts/Lora-Regular.8861b0072d.woff2" as="font" type="font/woff2" crossorigin="">


<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=2.5,user-scalable=yes">


<script async="" src="https://connect.facebook.net/en_US/fbevents.js"></script><script async="" src="//static.ads-twitter.com/uwt.js"></script><script type="text/javascript" src="https://cdn.krxd.net/userdata/get?pub=bd339c69-af54-4a21-b4f1-654bcfcd83ca&amp;technographics=1&amp;callback=Krux.ns.macmillan.kxjsonp_userdata"></script><script type="text/javascript" src="https://beacon.krxd.net/optout_check?callback=Krux.ns.macmillan.kxjsonp_optOutCheck"></script><script type="text/javascript" src="bd339c69-af54-4a21-b4f1-654bcfcd83ca_002"></script><script type="text/javascript" src="bd339c69-af54-4a21-b4f1-654bcfcd83ca"></script><script type="text/javascript" async="" src="linkid.js"></script><script async="" src="controltag.js"></script><script async="true" type="text/javascript" src="event" data-owner="criteo-tag"></script><script async="" src="gpt.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script type="text/javascript" async="" src="controltag"></script><script type="text/javascript" async="" src="entry-point"></script><script type="text/javascript" async="" defer="defer" src="tag.js"></script><script type="text/javascript" async="" src="beacon.js"></script><script type="text/javascript" async="" src="792a5a21-3d99-4432-8f70-957d389bf2b9.js"></script><script type="text/javascript" async="" src="ld.js"></script><script async="" src="93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea0.js"></script><script async="" src="gtm.js"></script><script>
    (function(e){var t=e.documentElement,n=e.implementation;t.className='js';if(n&&n.hasFeature('http://www.w3.org/TR/SVG11/feature#Image','1.1')){t.className+=' svg'}})(document)
</script>
<script>window.abTestSharedArticleRenderer = true;</script>


    <script id="mosaic-loader">
        (function (win, doc, bundles) {
            if (!doc.querySelector) { return; }

            var script = function(attrs) {
                var s = doc.createElement('script');
                for (var attr in attrs) {
                    if (attrs.hasOwnProperty(attr)) {
                        s[attr] = attrs[attr];
                    }
                }
                return s;
            };

            var insertAfter = function(node, dest) {
                dest.parentNode.insertBefore(node, dest.nextSibling);
            };

            win.Component = win.Component || {};
            win.Component.Loader = {
                init: function() {
                    this.main = script({src: '/static/js/nature.3cfebae16c.js', id: 'mosaic', defer: true});
                    insertAfter(this.main, doc.getElementById('mosaic-loader'));
                },
                loadPageBundles: function() {
                    var lastId = this.main.id;
                    for (var i = 0; bundles[i]; ++i) {
                        if (doc.querySelector(bundles[i].match)) {
                            insertAfter(script({src: bundles[i].src, id: 'bundle' + i, defer: true}), doc.getElementById(lastId));
                            lastId = 'bundle' + i;
                        }
                    }
                }
            };
            win.Component.Loader.init();

        })(window, document, [
            {match: 'div[data-pan-container]', src: '/static/js/pan-zoom.3e0716c442.js'},
            {match: 'math,span.mathjax-tex', src: '/static/js/math.f961232688.js'},
            {match: 'div.c-article-metrics__wrapper', src: '/static/js/graphing.09371cfbe8.js'},
            {match: 'body', src: 'https://verify.nature.com/verify/nature.min.js'}
        ]);
    </script><script src="nature_002.js" id="mosaic" defer="defer"></script><script src="nature.js" id="bundle3" defer="defer"></script>



    <link rel="apple-touch-icon" sizes="114x114" href="https://www.nature.com/static/images/favicons/nature/favicon-114x114.1431047068.png">
    <link rel="apple-touch-icon" sizes="144x144" href="https://www.nature.com/static/images/favicons/nature/favicon-144x144.3e61d1f755.png">
    <link rel="apple-touch-icon" sizes="180x180" href="https://www.nature.com/static/images/favicons/nature/favicon-180x180.5789768d66.png">
    <link rel="icon" type="image/png" href="https://www.nature.com/static/images/favicons/nature/favicon-16x16.bfd0e25c74.png" sizes="16x16">
    <link rel="icon" type="image/png" href="https://www.nature.com/static/images/favicons/nature/favicon-32x32.1c4a03347a.png" sizes="32x32">
    <link rel="icon" type="image/png" href="https://www.nature.com/static/images/favicons/nature/favicon-96x96.14ff9c7f48.png" sizes="96x96">
    <link rel="icon" type="image/png" href="https://www.nature.com/static/images/favicons/nature/favicon-194x194.5789768d66.png" sizes="194x194">
    <link rel="shortcut icon" href="https://www.nature.com/static/images/favicons/nature/favicon.14515a9f85.ico">
    <meta name="msapplication-TileColor" content="#940720">
    <meta name="msapplication-TileImage" content="/static/images/favicons/nature/favicon-144x144.3e61d1f755.png">
    <meta name="theme-color" content="#940720">


<link href="grade-c.css" rel="stylesheet">



<link id="mustard" data-test="open-access-css" href="nature-open-access.css" rel="stylesheet" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">




<link rel="stylesheet" type="text/css" href="article.css" media="only screen and (-webkit-min-device-pixel-ratio:0) and (min-color-index:0), (-ms-high-contrast: none), only all and (min--moz-device-pixel-ratio:0) and (min-resolution: 3e1dpcm)">
<link rel="stylesheet" type="text/css" href="article-print.css" media="print">




    
    <style>
         .background-brand-primary {
              background-color: #cedde4;
              color: #222;
         }

         .background-brand-secondary {
              background-color: #F1F9FC;
              color: #222;
         }

         .background-brand-secondary-pdf {
              background-color: #036599;
              background-size: 22px auto;
              background-position: 80% 50%;
              color: #fff;
         }

         .background-brand-secondary-pdf a {
              color: #fff;
         }

         .background-brand-tertiary {
              background-color: #FBFEFF;
              color: #222;
         }

         .background-gradient-brand-primary {
              background-image: linear-gradient(to right, #cedde4, #cedde4 50%, transparent 50%);
         }

         .background-gradient-brand-secondary-tertiary {
              background-image: linear-gradient(to right, #F1F9FC, #F1F9FC 50%, #FBFEFF 50%);
         }

         .background-gradient-brand-gray {
              background-image: linear-gradient(to right, #e0e0e0, #e0e0e0 50%, transparent 50%);
         }

         .background-gradient-brand-gray-light {
              background-image: linear-gradient(to right, #eee, #eee 50%, transparent 50%);
         }

         

         

         

         .nav-border {
              border-bottom: 4px solid #F1F9FC;
         }

         /* less than or equal to 875px */
         @media only screen and (max-width: 54.688em) {
              .banner {
                  background-color: #cedde4;
              }

              .nav-border .menu-cell li {
                  border-bottom: 1px solid #F1F9FC;
              }
         }

         .sticky-header .inner-banner,
         .header-primary-color {
              background-color: #cedde4;
         }

         
             .menu-button,
             .menu-button-clone,
             .header-submit-button {
                 border: 1px solid #222;
                 border: 1px solid rgba(34,34,34,.75);
                 color: #222;
             }
             .nature-research-logo {
                 background-image: url("/static/images/product-logos/nature-research-logo-dark.2dec4e5439.svg"), none;
             }

             .icon-rotate.tools-menu-button-icon:after,
             .menu-button-icon:after {
                 background-image: url("/static/images/icons/icon-arrow-down-12x7-gray-dark.d2d673ff2e.svg"), none;
             }

             .header-search-button {
                 padding-right: 25px;
                 background-image: url("/static/images/icons/icon-search-24x24-gray.d711eca805.svg"), none;
             }
         
     </style>
     


<!-- SpringerLink's event tracker, as per: https://github.com/springernature/springerlink-event-tracker -->

<script type="text/javascript">
document.addEventListener('accessdetailsloaded', function(e) {
    var dL, doi, pageType, eventObject, i, script, loginEventData;

    for (i = 0; i < window.dataLayer.length; i++) {
        if (window.dataLayer[i]["content"] && window.dataLayer[i]["page"]) {
            dL = window.dataLayer[i];
        }
    }

    if (!dL
        || !dL.content.article
        || !(doi = dL.content.article.doi) && doi.length > 0
        || (pageType = dL.page.category.pageType.toLowerCase()) !== "article"
    ) {return false;}

    pageType = pageType.charAt(0).toUpperCase() + pageType.substring(1);

    eventObject = {'content_type': pageType, 'doi': doi};

    loginEventData = e.detail;
    if (loginEventData && loginEventData.business_partner_id) {
        eventObject['business_partner_ids'] = loginEventData.business_partner_id;
    }

    script = document.createElement('script');
    script.id = 'springerlink-event-tracker';
    script.src = 'https://event-tracker.springernature.com/dist/eventTracker.js';
    script.onload = function() {
        new EventTracker({'platform': 'Nature'}).sendEvent('display', eventObject);
    };
    document.head.appendChild(script);
});
</script>
<script>
    dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"biodiversity;classification-and-taxonomy;data-processing","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Scientific Reports","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41598-018-22505-4"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Teresita M. Porter","Mehrdad Hajibabaei"],"publishedAt":1520553600,"publishedAtString":"2018-03-09","title":"Automated high throughput animal CO1 metabarcode classification","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"srep","title":"scientific reports","volume":"8","issue":"1"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"ab_test_nature_rebrand_150","active":false},{"name":"ab_test_subscribe_button","active":true},{"name":"ab_test_briefing_new_signup_box","active":true},{"name":"ab_test_best_available_version","active":false},{"name":"ab_test_magazine_article_native_ads","active":true},{"name":"ab_test_ncomms_banner","active":false}]},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true}];
</script>
<!-- Google Tag Manager -->
<script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
        new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
})(window,document,'script','dataLayer','GTM-NWDMT9Q');</script>
<!-- End Google Tag Manager -->


    <meta name="description" content="Article">

<meta name="robots" content="noarchive">
<meta name="access" content="Yes">
<meta name="WT.cg_s" content="Article">
<meta name="WT.z_bandiera_abtest" content="a">
<meta name="WT.page_categorisation" content="Article_HTML">

    <meta name="WT.template" content="oscar">
    <meta name="WT.z_cg_type" content="Nature Research Journals">
    <meta name="WT.cg_n" content="Scientific Reports">
    <meta name="dc.rights" content="©2019 Macmillan Publishers Limited. All Rights Reserved.">
    <meta name="prism.issn" content="2045-2322">


<link rel="search" href="http://www.nature.com/search">
<link rel="search" href="http://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="http://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">


    
    

    <meta name="journal_id" content="41598">

    <meta name="dc.title" content="Automated high throughput animal CO1 metabarcode classification">

    <meta name="dc.source" content="Scientific Reports 2018 8:1">

    <meta name="dc.format" content="text/html">

    <meta name="dc.publisher" content="Nature Publishing Group">

    <meta name="dc.date" content="2018-03-09">

    <meta name="dc.type" content="OriginalPaper">

    <meta name="dc.language" content="En">

    <meta name="dc.copyright" content="2018 The Author(s)">

    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">

    <meta name="dc.description" content="We introduce a method for assigning names to CO1 metabarcode sequences with confidence scores in a rapid, high-throughput manner. We compiled nearly 1 million CO1 barcode sequences appropriate for classifying arthropods and chordates. Compared to our previous Insecta classifier, the current classifier has more than three times the taxonomic coverage, including outgroups, and is based on almost five times as many reference sequences. Unlike other popular rDNA metabarcoding markers, we show that classification performance is similar across the length of the CO1 barcoding region. We show that the RDP classifier can make taxonomic assignments about 19 times faster than the popular top BLAST hit method and reduce the false positive rate from nearly 100% to 34%. This is especially important in large-scale biodiversity and biomonitoring studies where datasets can become very large and the taxonomic assignment problem is not trivial. We also show that reference databases are becoming more representative of current species diversity but that gaps still exist. We suggest that it would benefit the field as a whole if all investigators involved in metabarocoding studies, through collaborations with taxonomic experts, also planned to barcode representatives of their local biota as a part of their projects.">

    <meta name="prism.issn" content="2045-2322">

    <meta name="prism.publicationName" content="Scientific Reports">

    <meta name="prism.publicationDate" content="2018-03-09">

    <meta name="prism.volume" content="8">

    <meta name="prism.number" content="1">

    <meta name="prism.section" content="OriginalPaper">

    <meta name="prism.startingPage" content="1">

    <meta name="prism.endingPage" content="10">

    <meta name="prism.copyright" content="2018 The Author(s)">

    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">

    <meta name="prism.url" content="https://www.nature.com/articles/s41598-018-22505-4">

    <meta name="prism.doi" content="doi:10.1038/s41598-018-22505-4">

    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41598-018-22505-4.pdf">

    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41598-018-22505-4">

    <meta name="citation_journal_title" content="Scientific Reports">

    <meta name="citation_journal_abbrev" content="Sci Rep">

    <meta name="citation_publisher" content="Nature Publishing Group">

    <meta name="citation_issn" content="2045-2322">

    <meta name="citation_title" content="Automated high throughput animal CO1 metabarcode classification">

    <meta name="citation_volume" content="8">

    <meta name="citation_issue" content="1">

    <meta name="citation_online_date" content="2018/03/09">

    <meta name="citation_firstpage" content="1">

    <meta name="citation_lastpage" content="10">

    <meta name="citation_article_type" content="Article">

    <meta name="citation_fulltext_world_readable" content="">

    <meta name="citation_language" content="en">

    <meta name="dc.identifier" content="doi:10.1038/s41598-018-22505-4">

    <meta name="DOI" content="10.1038/s41598-018-22505-4">

    <meta name="citation_doi" content="10.1038/s41598-018-22505-4">

    <meta name="description" content="We introduce a method for assigning names to CO1 metabarcode sequences with confidence scores in a rapid, high-throughput manner. We compiled nearly 1 million CO1 barcode sequences appropriate for classifying arthropods and chordates. Compared to our previous Insecta classifier, the current classifier has more than three times the taxonomic coverage, including outgroups, and is based on almost five times as many reference sequences. Unlike other popular rDNA metabarcoding markers, we show that classification performance is similar across the length of the CO1 barcoding region. We show that the RDP classifier can make taxonomic assignments about 19 times faster than the popular top BLAST hit method and reduce the false positive rate from nearly 100% to 34%. This is especially important in large-scale biodiversity and biomonitoring studies where datasets can become very large and the taxonomic assignment problem is not trivial. We also show that reference databases are becoming more representative of current species diversity but that gaps still exist. We suggest that it would benefit the field as a whole if all investigators involved in metabarocoding studies, through collaborations with taxonomic experts, also planned to barcode representatives of their local biota as a part of their projects.">

    <meta name="dc.creator" content="Teresita M. Porter">

    <meta name="dc.creator" content="Mehrdad Hajibabaei">

    <meta name="dc.subject" content="Biodiversity">

    <meta name="dc.subject" content="Classification and taxonomy">

    <meta name="dc.subject" content="Data processing">

    <meta name="citation_reference" content="citation_journal_title=Cladistics; citation_title=Impediments to taxonomy and users of taxonomy: accessibility and impact evaluation; citation_author=MC Ebach, AG Valdecasas, QD Wheeler; citation_volume=27; citation_publication_date=2011; citation_pages=550-557; citation_doi=10.1111/j.1096-0031.2011.00348.x; citation_id=CR1">

    <meta name="citation_reference" content="citation_journal_title=Proc. R. Soc. B Biol. Sci.; citation_title=Biological identifications through DNA barcodes; citation_author=PDN Hebert, A Cywinska, SL Ball, JR deWaard; citation_volume=270; citation_publication_date=2003; citation_pages=313-321; citation_doi=10.1098/rspb.2002.2218; citation_id=CR2">

    <meta name="citation_reference" content="citation_journal_title=Mol. Ecol.; citation_title=Towards next-generation biodiversity assessment using DNA metabarcoding; citation_author=P Taberlet, E Coissac, F Pompanon, C Brochmann, E Willerslev; citation_volume=21; citation_publication_date=2012; citation_pages=2045-2050; citation_doi=10.1111/j.1365-294X.2012.05470.x; citation_id=CR3">

    <meta name="citation_reference" content="citation_journal_title=Mol. Ecol.; citation_title=Environmental DNA metabarcoding: transforming how we survey animal and plant communities; citation_author=K Deiner; citation_volume=26; citation_publication_date=2017; citation_pages=5872-5895; citation_doi=10.1111/mec.14350; citation_id=CR4">

    <meta name="citation_reference" content="GRDI-EcoBiomics. Metagenomics Based Ecosystem Biomonitoring (GRDI-EcoBiomics) project, Government of Canada, Genomics R&amp;D Initiative, Year-End Performance Report for Shared Priority Projects (2016–2017). http://grdi-irdg.collaboration.gc.ca/eng/reports/annual/2015_2016.html (2016).">

    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci.; citation_title=Simultaneous assessment of the macrobiome and microbiome in a bulk sample of tropical arthropods through DNA metasystematics; citation_author=J Gibson; citation_volume=111; citation_publication_date=2014; citation_pages=8007-8012; citation_doi=10.1073/pnas.1406468111; citation_id=CR6">

    <meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.; citation_title=The Closest BLAST Hit Is Often Not the Nearest Neighbor; citation_author=LB Koski, GB Golding; citation_volume=52; citation_publication_date=2001; citation_pages=540-542; citation_doi=10.1007/s002390010184; citation_id=CR7">

    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=Comparative performances of DNA barcoding across insect orders; citation_author=M Virgilio, T Backeljau, B Nevado, M Meyer; citation_volume=11; citation_publication_date=2010; citation_doi=10.1186/1471-2105-11-206; citation_id=CR8">

    <meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.; citation_title=Rapid and accurate taxonomic classification of insect (class Insecta) cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve Bayesian classifier; citation_author=TM Porter; citation_volume=14; citation_publication_date=2014; citation_pages=929-942; citation_id=CR9">

    <meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.; citation_title=Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy; citation_author=Q Wang, GM Garrity, JM Tiedje, JR Cole; citation_volume=73; citation_publication_date=2007; citation_pages=5261-5267; citation_doi=10.1128/AEM.00062-07; citation_id=CR10">

    <meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.; citation_title=Accurate, Rapid Taxonomic Classification of Fungal Large-Subunit rRNA Genes; citation_author=K-L Liu, A Porras-Alfaro, CR Kuske, SA Eichorst, G Xie; citation_volume=78; citation_publication_date=2012; citation_pages=1523-1533; citation_doi=10.1128/AEM.06826-11; citation_id=CR11">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Ribosomal Database Project: data and tools for high throughput rRNA analysis; citation_author=JR Cole; citation_volume=42; citation_publication_date=2014; citation_pages=D633-D642; citation_doi=10.1093/nar/gkt1244; citation_id=CR12">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=ARB: a software environment for sequence data; citation_author=W Ludwig; citation_volume=32; citation_publication_date=2004; citation_pages=1363-1371; citation_doi=10.1093/nar/gkh293; citation_id=CR13">

    <meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.; citation_title=Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB; citation_author=TZ DeSantis; citation_volume=72; citation_publication_date=2006; citation_pages=5069-5072; citation_doi=10.1128/AEM.03006-05; citation_id=CR14">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB; citation_author=E Pruesse; citation_volume=35; citation_publication_date=2007; citation_pages=7188-7196; citation_doi=10.1093/nar/gkm864; citation_id=CR15">

    <meta name="citation_reference" content="citation_journal_title=New Phytol.; citation_title=The UNITE database for molecular identification of fungi – recent updates and future perspectives; citation_author=K Abarenkov; citation_volume=186; citation_publication_date=2010; citation_pages=281-285; citation_doi=10.1111/j.1469-8137.2009.03160.x; citation_id=CR16">

    <meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.; citation_title=ITS2 Database V: Twice as Much; citation_author=MJ Ankenbrand, A Keller, M Wolf, J Schultz, F Förster; citation_volume=32; citation_publication_date=2015; citation_pages=3030-3032; citation_doi=10.1093/molbev/msv174; citation_id=CR17">

    <meta name="citation_reference" content="Ratnasingham, S. &amp; Hebert, P. D. BOLD: The Barcode of Life Data System (
                    http://www.barcodinglife.org
                    
                  ). Mol. Ecol. Notes 7, 355–364 (2007).">

    <meta name="citation_reference" content="Porter, T. M. &amp; Hajibabaei, M. Scaling up: A guide to high throughput genomic approaches for biodiversity analysis. Mol. Ecol. (In press).">

    <meta name="citation_reference" content="citation_journal_title=Freshw. Biol.; citation_title=AusRivAS: using macroinvertebrates to assess ecological condition of rivers in Western Australia; citation_author=MJ Smith; citation_volume=41; citation_publication_date=1999; citation_pages=269-282; citation_doi=10.1046/j.1365-2427.1999.00430.x; citation_id=CR20">

    <meta name="citation_reference" content="McDermott, H., Paull, T. &amp; Strachan, S. CABIN Canadian Aquatic Biomonitoring Network, Laboratory Methods: Processing, Taxonomy, and Quality Control of Benthic Macroinvertebrate Samples. (Environment Canada, 2014).">

    <meta name="citation_reference" content="Elbrecht, V. &amp; Leese, F. Validation and Development of COI Metabarcoding Primers for Freshwater Macroinvertebrate Bioassessment. Front. Environ. Sci. 5, (2017).">

    <meta name="citation_reference" content="citation_journal_title=PLOS ONE; citation_title=Large-Scale Biomonitoring of Remote and Threatened Ecosystems via High-Throughput Sequencing; citation_author=JF Gibson; citation_volume=10; citation_publication_date=2015; citation_pages=e0138432; citation_doi=10.1371/journal.pone.0138432; citation_id=CR23">

    <meta name="citation_reference" content="Shokralla, S. et al. Environmental DNA Barcode Sequence Capture: Targeted, PCR-free Sequence Capture for Biodiversity Analysis from Bulk Environmental Samples. bioRxiv 087437 (2016).">

    <meta name="citation_reference" content="NCBI. BLAST Frequently Asked Questions. https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=FAQ#expect (2018).">

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Robust Detection of Rare Species Using Environmental DNA: The Importance of Primer Specificity; citation_author=TM Wilcox; citation_volume=8; citation_publication_date=2013; citation_pages=e59520; citation_doi=10.1371/journal.pone.0059520; citation_id=CR26">

    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Preserving Accuracy in GenBank; citation_author=MI Bidartondo; citation_volume=319; citation_publication_date=2008; citation_pages=1616-1616; citation_doi=10.1126/science.319.5870.1616a; citation_id=CR27">

    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective; citation_author=RH Nilsson; citation_volume=1; citation_publication_date=2006; citation_pages=e59; citation_doi=10.1371/journal.pone.0000059; citation_id=CR28">

    <meta name="citation_reference" content="citation_journal_title=Microbes Environ.; citation_title=A Comprehensive, Automatically Updated Fungal ITS Sequence Dataset for Reference-Based Chimera Control in Environmental Sequencing Efforts; citation_author=RH Nilsson; citation_volume=30; citation_publication_date=2015; citation_pages=145-150; citation_doi=10.1264/jsme2.ME14121; citation_id=CR29">

    <meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. B Biol. Sci.; citation_title=DNA barcodes for biosecurity: invasive species identification; citation_author=KF Armstrong, SL Ball; citation_volume=360; citation_publication_date=2005; citation_pages=1813-1823; citation_doi=10.1098/rstb.2005.1713; citation_id=CR30">

    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; citation_author=SF Altschul; citation_volume=25; citation_publication_date=1997; citation_pages=17; citation_doi=10.1093/nar/25.17.3389; citation_id=CR31">

    <meta name="citation_reference" content="Shi, Z. Y. et al. FuzzyID2: A software package for large dataset species identification via barcoding and metabarcoding using Hidden Markov Models and fuzzy set methods. Mol. Ecol. Resour. (In press).">

    <meta name="citation_reference" content="citation_journal_title=Syst. Biol.; citation_title=A Step Toward Barcoding Life: A Model-Based, Decision-Theoretic Method to Assign Genes to Preexisting Species Groups; citation_author=Z Abdo, GB Golding; citation_volume=56; citation_publication_date=2007; citation_pages=44-56; citation_doi=10.1080/10635150601167005; citation_id=CR33">

    <meta name="citation_reference" content="citation_journal_title=Mol. Phylogenet. Evol.; citation_title=Assigning sequences to species in the absence of large interspecific differences; citation_author=M Lou, GB Golding; citation_volume=56; citation_publication_date=2010; citation_pages=187-194; citation_doi=10.1016/j.ympev.2010.01.002; citation_id=CR34">

    <meta name="citation_reference" content="citation_journal_title=Syst. Biol.; citation_title=Inferring Species Membership Using DNA Sequences with Back-Propagation Neural Networks; citation_author=AB Zhang, DS Sikes, C Muster, SQ Li; citation_volume=57; citation_publication_date=2008; citation_pages=202-215; citation_doi=10.1080/10635150802032982; citation_id=CR35">

    <meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.; citation_title=Classification of Nucleotide Sequences Using Support Vector Machines; citation_author=T-K Seo; citation_volume=71; citation_publication_date=2010; citation_pages=250-267; citation_doi=10.1007/s00239-010-9380-9; citation_id=CR36">

    <meta name="citation_reference" content="Sayers, E. W. Ebot. http://www.ncbi.nlm.nih.gov/Class/PowerTools/eutils/course.html (2018)">

    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=The Bioperl toolkit: Perl modules for the life sciences; citation_author=JE Stajich; citation_volume=12; citation_publication_date=2002; citation_pages=1611-1618; citation_doi=10.1101/gr.361602; citation_id=CR38">

    <meta name="citation_reference" content="Maddison, W. P. &amp; Maddison, D. R. Mesquite version 3.10. http://mesquiteproject.org (2015).">

    <meta name="citation_reference" content="citation_journal_title=EMBnet J.; citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads; citation_author=M Martin; citation_volume=17; citation_publication_date=2011; citation_pages=10-12; citation_doi=10.14806/ej.17.1.200; citation_id=CR40">

    <meta name="citation_author" content="Teresita M. Porter">

    <meta name="citation_author_institution" content="The Centre for Biodiversity Genomics &amp; Department of Integrative Biology, University of Guelph, Guelph, Canada">

    <meta name="citation_author_institution" content="Great Lakes Forestry Centre, Natural Resources Canada, Sault Ste. Marie, Canada">

    <meta name="citation_author" content="Mehrdad Hajibabaei">

    <meta name="citation_author_institution" content="The Centre for Biodiversity Genomics &amp; Department of Integrative Biology, University of Guelph, Guelph, Canada">

    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">

    <meta name="twitter:card" content="summary">

    <meta name="twitter:title" content="Automated high throughput animal CO1 metabarcode classification">

    <meta name="twitter:site" content="@SciReports">

    <meta name="twitter:description" content="Automated high throughput animal CO1 metabarcode classification">

    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig1_HTML.jpg">

    <meta name="twitter:image:alt" content="Content cover image">

    <meta name="WT.z_cc_license_type" content="cc_by">

    <meta name="WT.z_primary_atype" content="Research">

    <meta name="WT.z_subject_term" content="Biodiversity;Classification and taxonomy;Data processing">

    <meta name="WT.z_subject_term_id" content="biodiversity;classification-and-taxonomy;data-processing">


    
        <meta property="og:url" content="https://www.nature.com/articles/s41598-018-22505-4">
        <meta property="og:type" content="article">
        <meta property="og:site_name" content="Scientific Reports">
        <meta property="og:title" content="Automated high throughput animal CO1 metabarcode classification">
        
        <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig1_HTML.jpg">
    
<script async="" type="text/javascript" src="inject.js"></script><script type="text/javascript" async="" src="nature_status.json"></script><link type="text/css" rel="stylesheet" href="nature_checkout.css"><link type="text/css" href="optanon.css" rel="stylesheet"><style>#optanon ul#optanon-menu li { background-color:  !important }#optanon ul#optanon-menu li.menu-item-selected { background-color:  !important }#optanon #optanon-popup-wrapper .optanon-white-button-middle { background-color: #006699 !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle { background-color: #006699 !important; border-color: #006699 !important; }#optanon #optanon-popup-wrapper .optanon-white-button-middle button { color: #ffffff !important }.optanon-alert-box-wrapper .optanon-alert-box-button-middle button { color: #ffffff !important }#optanon #optanon-popup-bottom { background-color: #F2F2F2 !important }#optanon.modern #optanon-popup-top, #optanon.modern #optanon-popup-body-left-shading { background-color: #F2F2F2 !important }.optanon-alert-box-wrapper { background-color:#222222 !important }.optanon-alert-box-wrapper .optanon-alert-box-bg p { color:#FFFFFF !important }#optanon-popup-body-left-shading {
background-color:red!important;
}</style><script id="springerlink-event-tracker" src="eventTracker.asc"></script><script class="kxct" data-id="Jn7TaPwq" data-timing="async" data-version="3.0">window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]);(function(){var k=document.createElement("script");k.type="text/javascript";k.async=true;var m,src=(m=location.href.match(/\bkxsrc=([^&]+)/))&&decodeURIComponent(m[1]);k.src = /^https?:\/\/([a-z0-9_\-\.]+\.)?krxd\.net(:\d{1,5})?\//i.test(src) ? src : src === "disable" ? "" :(location.protocol==="https:"?"https:":"http:")+"//cdn.krxd.net/controltag?confid=Jn7TaPwq";var s=document.getElementsByTagName("script")[0];s.parentNode.insertBefore(k,s);}());</script><script src="beacon_002.js"></script><link rel="preload" href="integrator.js"><script type="text/javascript" src="integrator.js"></script><script src="pubads_impl_2019090901.js" async=""></script></head>
<!--[if IE 9]><body class="ie9 article-page"><![endif]-->
<!--[if gt IE 9]><!--><body class="article-page"><div class="optanon-alert-box-wrapper  " role="alertdialog" aria-labelledby="alert-box-title" aria-describedby="alert-box-message" style="top: 0px;"><div class="optanon-alert-box-bottom-top"></div><div class="optanon-alert-box-bg"><div class="optanon-alert-box-logo"> </div><div class="optanon-alert-box-body"><p class="optanon-alert-box-title legacy-banner-title sr-only" id="alert-box-title" role="heading" aria-level="2">Cookie Notice</p><p class="banner-content" id="alert-box-message">We
 use cookies to personalise content and ads, to provide social media 
features and to analyse our traffic. We also share information about 
your use of our site with our social media, advertising and analytics 
partners in accordance with our <a href="https://www.nature.com/info/privacy.html" class="banner-policy-link" tabindex="1" aria-label="Privacy Policy">Privacy Policy</a>. You can manage your preferences in 'Manage Cookies'.</p></div><div class="optanon-clearfix"></div><div class="optanon-alert-box-button-container"><div class="optanon-alert-box-button optanon-button-close"><div class="optanon-alert-box-button-middle"><button class="optanon-alert-box-close" aria-label="Close">Close</button></div></div><div class="optanon-alert-box-button optanon-button-allow"><div class="optanon-alert-box-button-middle accept-cookie-container"><button class="optanon-allow-all accept-cookies-button" title="OK" aria-label="OK" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Accept Cookies');" tabindex="3">OK</button></div></div><div class="optanon-alert-box-button optanon-button-more"><div class="optanon-alert-box-button-middle"><button class="optanon-toggle-display cookie-settings-button" title="Manage Cookies" aria-label="Manage Cookies" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Banner Open Preferences');" tabindex="2">Manage Cookies</button></div></div></div><div class="optanon-clearfix optanon-alert-box-bottom-padding"></div></div></div><div id="optanon" class="modern"><div id="optanon-popup-bg"></div><div id="optanon-popup-wrapper" role="dialog" aria-modal="true" tabindex="-1" lang="en-GB"><div id="optanon-popup-top"><button onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Close Button');" aria-label="Close" class="optanon-close-link optanon-close optanon-close-ui" title="Close"><div id="optanon-close" style="background: url(https://cdn.cookielaw.org/skins/4.1.0/default_flat_top_two_button_black/v2/images/optanon-pop-up-close.png);width:34px;height:34px;"></div></button></div><div id="optanon-popup-body"><div id="optanon-popup-body-left"><div id="optanon-popup-body-left-shading"></div><div id="optanon-branding-top-logo" style="background-image: url('https://cdn.cookielaw.org/logos/5138/5138:nature.com/nature-red.png') !important;"></div><ul id="optanon-menu" aria-label="Navigation Menu" role="tablist"><li class="menu-item-on menu-item-about" title="Your Privacy"><p class="preference-menu-item"><button role="tab" aria-selected="true" aria-controls="tab-0" id="Your-Privacy">Your Privacy</button></p></li><li class="menu-item-necessary menu-item-on" title="Strictly Necessary Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-1" id="Strictly-Necessary">Strictly Necessary Cookies</button></p></li><li class="menu-item-on menu-item-performance" title="Performance &amp; Analytics Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-2" id="Performance-&amp;">Performance &amp; Analytics Cookies</button></p></li><li class="menu-item-on menu-item-functional" title="Functional Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-3" id="Functional-Cookies">Functional Cookies</button></p></li><li class="menu-item-on menu-item-advertising" title="Targeting Cookies"><p class="preference-menu-item"><button role="tab" aria-selected="false" aria-controls="tab-4" id="Targeting-Cookies">Targeting Cookies</button></p></li><li class="menu-item-moreinfo menu-item-off" title="More Information"><p class="preference-menu-item"><a target="_blank" aria-label="More Information" href="http://www.nature.com/info/cookies.html" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Cookie Policy');">More Information</a></p></li></ul></div><div id="optanon-popup-body-right"><p role="heading" aria-level="2" class="legacy-preference-banner-title h2" aria-label="Privacy Preference Centre">Privacy Preference Centre</p><div class="vendor-header-container"><p class="header-3" role="heading" aria-level="3"></p><div id="optanon-popup-more-info-bar"><div class="optanon-status"><div class="optanon-status-editable"><form><span class="fieldset"><p><input value="check" id="chkMain" checked="checked" class="legacy-group-status optanon-status-checkbox" type="checkbox"><label for="chkMain">Active</label></p></span></form></div><div class="optanon-status-always-active optanon-status-on"><p>Always Active</p></div></div></div></div><div class="optanon-main-info-text" role="tabpanel"></div></div><div class="optanon-bottom-spacer"></div></div><div id="optanon-popup-bottom"> <a href="https://onetrust.com/poweredbyonetrust" target="_blank"><div id="optanon-popup-bottom-logo" style="background: url(https://cdn.cookielaw.org/skins/4.1.0/default_flat_top_two_button_black/v2/images/cookie-collective-top-bottom.png);width:155px;height:35px;" title="powered by OneTrust"></div></a><div class="optanon-button-wrapper optanon-save-settings-button optanon-close optanon-close-consent"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><button title="Save Settings" aria-label="Save Settings" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Save Settings');">Save Settings</button></div><div class="optanon-white-button-right"></div></div><div class="optanon-button-wrapper optanon-allow-all-button optanon-allow-all" style="display: none;"><div class="optanon-white-button-left"></div><div class="optanon-white-button-middle"><button title="Allow All" aria-label="Allow All" onclick="Optanon.TriggerGoogleAnalyticsEvent('OneTrust Cookie Consent', 'Preferences Allow All');">Allow All</button></div><div class="optanon-white-button-right"></div></div></div></div></div><!--<![endif]-->
<!-- Google Tag Manager (noscript) -->
<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NWDMT9Q"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<!-- End Google Tag Manager (noscript) -->
<div role="banner" class="position-relative cleared z-index-50" data-test="top-containers">
    <div class="hide-print container background-gray-dark">
    <div class="content mq1200-padded">
        <a href="#content" id="skiplink" class="skiplink contrast-text js-no-scroll" data-track="click" data-track-action="skip to content" data-track-category="header" data-track-label="link">Skip to main content</a>
    </div>
</div>



<div class="container cleared container-type-banner-advert inline-group" data-container-type="banner-advert">
    <div class="hide-print pt10 pb10 mq875-pa0 position-relative z-index-100 clear-float cleared leaderboard-or-billboard-container">
        <div class="leaderboard-or-billboard-inner ad-with-label">
            
                
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-gpt-unitpath="/285/scientific_reports/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41598-018-22505-4;doi=10.1038/s41598-018-22505-4;subjmeta=114,1314,1386,158,631,670,704;kwrd=Biodiversity,Classification and taxonomy,Data processing">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/scientific_reports/article&amp;sz=728x90&amp;c=-1234712035&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41598-018-22505-4%26doi%3D10.1038/s41598-018-22505-4%26subjmeta%3D114,1314,1386,158,631,670,704%26kwrd%3DBiodiversity,Classification and taxonomy,Data processing">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/scientific_reports/article&amp;sz=728x90&amp;c=-1234712035&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41598-018-22505-4%26doi%3D10.1038/s41598-018-22505-4%26subjmeta%3D114,1314,1386,158,631,670,704%26kwrd%3DBiodiversity,Classification and taxonomy,Data processing"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    <div id="google_ads_iframe_/270604982/nature/srep/articles_0__container__" style="border: 0pt none; width: 728px; height: 90px;"></div></div>

            
        </div>
    </div>
</div>

<div class="hide-print text-orange content grade-c-show"><p>Thank you for visiting nature.com. You are using a browser version with
    limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off
    compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site
    without styles and JavaScript.</p></div>

    
        
            <div class="c-banner">
                <div class="content cleared mq1200-padded">
                    <p>Help us improve our products.<a href="https://mailchi.mp/bcbfa719e0ab/snuxresearch" data-track="click" data-track-action="click" data-track-category="feedback banner top of page" data-track-label="ux recruitment link"> Sign up to take part.</a></p>
                </div>
            </div>
        
    
    
    <div class="hide-print container background-bluegray contrast-text text14 strong lower mq640-hide position-relative z-index-100" data-test="header-breadcrumbs">
        <div class="content cleared mq1200-padded">
            <div class="breadcrumbs pin-left">
                    <ol class="ma0 cleared clean-list inline-list"><li id="breadcrumb0" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb1"><a href="https://www.nature.com/" itemprop="url" class="icon icon-right icon-arrow-right-6x10-white pr15" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="title">nature</span></a></li><li id="breadcrumb1" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb2" class="ml6"><a href="https://www.nature.com/srep" itemprop="url" class="icon icon-right icon-arrow-right-6x10-white pr15" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:scientific reports"><span itemprop="title">scientific reports</span></a></li><li id="breadcrumb2" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb" itemref="breadcrumb3" class="ml6"><a href="https://www.nature.com/srep/articles?type=article" itemprop="url" class="icon icon-right icon-arrow-right-6x10-white pr15" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="title">articles</span></a></li><li id="breadcrumb3" itemscope="itemscope" itemtype="http://data-vocabulary.org/Breadcrumb"><a href="https://www.nature.com/articles/s41598-018-22505-4" itemprop="url" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:article" class="ml6"><span itemprop="title">article</span></a></li></ol>
            </div>
            
                <p class="pin-right box-sizing header-primary-color nature-research-logo ma0 pa6 hide-text grade-c-hide">A Nature Research Journal</p>
            
        </div>
    </div>


    
    
        <div class="hide-print container js-header-container mb20 position-relative z-index-50" data-ui="header-container" data-container-type="header">
            <div class="cleared clear hide-print position-relative background-gradient-brand-primary js-header default-header z-index-100 composite-layer tighten-line-height">
                <div class="banner js-banner content mq1200-padded position-relative">
                    <div class="inner-banner cleared">
                        <div class="main-column small-header-main pin-left">
                            <div class="header-logo-container header-primary-color">
                                <a href="#menu" id="menu-button" class="js-header-menu-button menu-button js-no-scroll role-button" data-test="menu-button" role="button" tabindex="0" aria-haspopup="true" aria-controls="menu" data-track="click" data-track-action="open menu" data-track-category="header" data-track-label="link">
                                            <span class="menu-button-icon icon-rotate">
                                                <span class="menu-button-label">Menu</span>
                                            </span>
                                </a>
                                    <h1 class="inline-block"><a href="https://www.nature.com/srep" class="header-logo inline-block" data-track="click" data-track-action="home" data-track-category="header" data-track-label="image"><img alt="Scientific Reports" src="header-2c7da0708d3d7539cf612bcb0d72c930.svg" class="grade-c-invisible header-logo-primary js-svg" data-png="//media.springernature.com/full/nature-cms/uploads/product/srep/header-e3d497587e4888a8b21427cbb105e528.png"><img alt="Scientific Reports" src="header-2091eea0f18dbb1fee97bebe766793c1.svg" class="grade-c-invisible header-logo-secondary js-svg" data-png="//media.springernature.com/full/nature-cms/uploads/product/srep/header-63663207242d718880707605d7ea7eee.png"></a></h1>
                            </div>
                        </div>
                        <div class="position-absolute position-right small-header-side">
                            <div class="pin-right">
                              
                                <a href="#search-menu" data-component="tray-button" class="pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text mr1 role-button" data-test="search-link" data-track="click" data-track-category="header" data-track-action="open tray" data-track-label="button" role="button" tabindex="0" aria-haspopup="true" aria-controls="search-menu">
                                            <span class="icon icon-above icon-search-25x25-white icon-search-25x25-gray block text-center">
                                                <span class="block mt6 small-header-hide">Search</span>
                                            </span>
                                </a>
                                
                                  <a href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=288" class="mq640-hide pin-left inline-group-top pa10 small-header-icons background-bluegray text11 contrast-text mr1" data-test="ealert-link" data-track="click" data-track-action="ealert" data-track-category="header" data-track-label="link">
                                            <span class="icon icon-above icon-ealert-25x25-white icon-ealert-25x25-gray block text-center">
                                                <span class="block mt6 small-header-hide">E-alert</span>
                                            </span>
                                  </a>
                                
                                
                                  <a href="https://author-welcome.nature.com/" class="mq640-hide inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text mr1" data-test="submit-link" data-track="click" data-track-action="manuscript submission" data-track-category="header" data-track-label="link">
                                            <span class="icon icon-above icon-submit-25x25-white icon-submit-25x25-gray block text-center">
                                                <span class="block mt6 small-header-hide">Submit</span>
                                            </span>
                                  </a>
                                
                                
<a href="https://www.nature.com/nams/svc/myaccount" id="my-account" class="placeholder inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text" data-test="login-link" data-track="click" data-track-action="my account" data-track-category="header" data-track-label="link" style="display: none;">
            <span class="icon icon-above icon-login-25x25-white icon-login-25x25-gray block text-center">
                <span class="block mt6 small-header-hide">My Account</span>
            </span>
</a>
<a href="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41598-018-22505-4" id="login-button" style="display: block;" class="placeholder inline-block pin-left inline-group-top pa10 pr15 pl15 small-header-icons background-bluegray text11 contrast-text" data-test="login-link" data-track="click" data-track-action="login" data-track-category="header" data-track-label="link">
            <span class="icon icon-above icon-login-25x25-white icon-login-25x25-gray block text-center">
                <span class="block mt6 small-header-hide">Login</span>
            </span>
</a>




                              
                            </div>
                        </div><ul class="background-brand-secondary-pdf ma0 clean-list text14 pin-right icon icon-right hide icon-pdf-download"><li class="pin-left position-relative"><a data-track="click" data-track-action="download pdf" data-track-category="header" data-track-label="link" href="https://www.nature.com/articles/s41598-018-22505-4.pdf" class="pl20 pr30 pt10 inline-block block-link header-tools-link position-relative z-index-1"><span>PDF</span></a></li></ul>
                    <h2 class="pt10 pr20 mq640-pr5 background-brand-primary text14 unstyled nowrap overflow-ellipsis hide-overflow hide">Automated high throughput animal CO1 metabarcode classification</h2></div>
                </div>
            </div>
        <button type="button" data-action="close-on-focus" class="visually-hidden js-hide" tabindex="-1">Close menu</button><div id="search-menu" class="menu pt20 mb20 z-index-50 composite-layer background-white border-gray-medium border-bottom-2" data-component="tray" data-track-component="header" role="navigation" aria-hidden="true">
    <div class="menu-inner js-hide">
        <section>
            <h2 class="visually-hidden">Search</h2>
            <div class="hide-print content mb40 mq1200-padded position-relative" data-test="inline-search">
                <div class="pt30 cleared background-white sans-serif">
                    <div class="grid grid-8 grid-left-2 mq875-grid-12 mq875-ml0 mq640-pb30">
                        <form action="/search" method="get" role="search" autocomplete="off" class="standard-space-below" data-track="submit" data-track-action="search" data-track-label="form">
                            <label for="keywords" class="block strong">Article search</label>
                            <div class="position-relative">
                                <input id="keywords" class="border-gray border-all-1 equalize-line-height pa10 pr40 box-sizing grid-12" name="q" placeholder="Search by keywords or author" data-test="search-keywords" type="search">
                                <div class="position-absolute position-right position-top mt1 mr1">
                                    <button type="submit" class="icon icon-center search-btn kill-border" data-test="search-submit">
                                        Search
                                    </button>
                                </div>
                                <p class="mb0 mt4 text14"><a href="https://www.nature.com/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">Advanced search</a></p>
                            </div>
                        </form>
                        <h3 class="h3 strong mb4 border-gray-medium border-bottom-1 sans-serif">Quick links</h3>
                        <ul class="clean-list mb0 text14">
                            <li class="grid-ng grid-1of2 mb6 mt6"><a href="https://www.nature.com/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
                            <li class="grid-ng grid-1of2 mb6 mt6"><a href="https://www.nature.com/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
                            <li class="grid-ng grid-1of2"><a href="https://www.nature.com/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
                            <li class="grid-ng grid-1of2"><a href="https://www.nature.com/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
                        </ul>
                    </div>
                </div>
            </div>
        </section>
    </div>
</div><button type="button" data-action="close-on-focus" class="visually-hidden js-hide" tabindex="-1">Close menu</button><button type="button" data-action="close-on-focus" class="visually-hidden js-hide" tabindex="-1">Close menu</button><div class="hide-print js-header-menu menu mb20 z-index-50 composite-layer background-white border-bottom-2 border-gray-medium tighten-line-height" id="menu" data-track-component="menu" role="navigation" aria-labelledby="menu-button" aria-hidden="true">
    <div class="menu-inner content js-hide">
        <nav>
            <div class="cleared border-bottom-1 border-gray-medium ml20 mr20">
                <div class="grid-ng grid-1of4 mq875-grid-12">
                    <h3 class="h3 pa20 mq875-pa0 mq875-mt20">
                        Scientific Reports<span class="visually-hidden"> menu</span>
                    </h3>
                </div>
                
    

    
    <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
        <ul class="clean-list mb0">
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/articles" data-track="click" data-track-action="browse articles" data-track-label="link">Browse Articles</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/collections" data-track="click" data-track-action="browse collections" data-track-label="link">Browse Collections</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/browse-subjects" data-track="click" data-track-action="browse subjects" data-track-label="link">Browse Subjects</a></li>
            
        </ul>
    </div>
    

    
    <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
        <ul class="clean-list mb0">
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/about" data-track="click" data-track-action="about scientific reports" data-track-label="link">About Scientific Reports</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/author-instructions" data-track="click" data-track-action="author instructions" data-track-label="link">Author instructions</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/journal-policies" data-track="click" data-track-action="journal policies" data-track-label="link">Journal policies</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/guide-to-referees" data-track="click" data-track-action="guide to referees" data-track-label="link">Guide to referees</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/contact" data-track="click" data-track-action="contact" data-track-label="link">Contact</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/editorschoice" data-track="click" data-track-action="editor's choice" data-track-label="link">Editor's Choice</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/guestedited" data-track="click" data-track-action="guest edited collections" data-track-label="link">Guest Edited Collections</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/top100" data-track="click" data-track-action="scientific reports top 100 2017" data-track-label="link">Scientific Reports Top 100 2017</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/top10" data-track="click" data-track-action="scientific reports top 10 2018" data-track-label="link">Scientific Reports Top 10 2018</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/top100-2018" data-track="click" data-track-action="scientific reports top 100 2018" data-track-label="link">Scientific Reports Top 100 2018</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/ebm" data-track="click" data-track-action="editorial board highlights" data-track-label="link">Editorial Board Highlights</a></li>
            
            <li class="mb4 pr10"><a href="https://www.nature.com/srep/author-highlights" data-track="click" data-track-action="author highlights" data-track-label="link">Author Highlights</a></li>
            
        </ul>
    </div>
    

            </div>
            <div class="cleared ml20 mr20">
                <div class="cleared pb10">
    <div class="grid-ng grid-1of4 mq875-grid-12">
        <h3 class="serif pa20 mq875-pa0 mq875-mt20">Nature Research<span class="visually-hidden"> menu</span></h3>
    </div>
    <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
        <h4 class="mb4">Our Journals</h4>
        <ul class="clean-list mb0">
            <li class="mb4 pr10"><a href="https://www.nature.com/nature" data-track="click" data-track-action="nature" data-track-label="link">Nature</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/ncomms" data-track="click" data-track-action="nature communications" data-track-label="link">Nature Communications</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/nprot" data-track="click" data-track-action="nature protocols" data-track-label="link">Nature Protocols</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/srep" data-track="click" data-track-action="scientific reports" data-track-label="link">Scientific Reports</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/siteindex" data-track="click" data-track-action="journal a-z view all" data-track-label="link">View all journals</a></li>
        </ul>
    </div>
    <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
        <h4 class="mb4">Subjects</h4>
        <ul class="clean-list mb0">
            <li class="mb4 pr10"><a href="https://www.nature.com/subjects/biological-sciences" data-track="click" data-track-action="biological sciences" data-track-label="link">Biological Sciences</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/subjects/scientific-community-and-society" data-track="click" data-track-action="scientific community and society" data-track-label="link">Scientific Community &amp; Society</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/subjects/earth-and-environmental-sciences" data-track="click" data-track-action="earth and environmental sciences" data-track-label="link">Earth &amp; Environmental Sciences</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/subjects/health-sciences" data-track="click" data-track-action="health sciences" data-track-label="link">Health Sciences</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/subjects/physical-sciences" data-track="click" data-track-action="physical sciences" data-track-label="link">Physical Sciences</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/subjects" data-track="click" data-track-action="subjects view all" data-track-label="link">View all subjects</a></li>
        </ul>
    </div>
    <div class="grid-ng grid-1of4 text14 pa20 mq875-grid-4 mq640-grid-12">
        <h4 class="mb4">More</h4>
        <ul class="clean-list mb0">
            <li class="mb4 pr10"><a href="https://support.nature.com/support/home" data-track="click" data-track-action="subscriptions" data-track-label="link">Contact us</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/authors/index.html" data-track="click" data-track-action="authors and referees" data-track-label="link">Authors &amp; Referees</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/libraries/index.html" data-track="click" data-track-action="librarians" data-track-label="link">Librarians</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/advertising/index.html" data-track="click" data-track-action="advertisers" data-track-label="link">Advertisers</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/npg_/press_room/index.html" data-track="click" data-track-action="press" data-track-label="link">Press</a></li>
            <li class="mb4 pr10"><a href="https://www.nature.com/npg_/index_npg.html" data-track="click" data-track-action="about" data-track-label="link">About Nature Research</a></li>
        </ul>
    </div>
</div>

            </div>
        </nav>
    </div>
</div><button type="button" data-action="close-on-focus" class="visually-hidden js-hide" tabindex="-1">Close menu</button></div>
    

    
    

</div>



<div id="content" class="position-relative z-index-1">
    <div class="container cleared container-type-article" data-container-type="article">
        <div class="content position-relative cleared clear mq1200-padded" data-component="article-container">
            <article itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle" lang="en">
                <div class="full-width-print main-column pin-left js-main-column highlighter" role="main">
                    <div class="c-article-header">
                        <header>
                            <ul class="c-article-identifiers" data-test="article-identifier">
                                <li class="c-article-identifiers__item">Article</li>
                                
                                <li class="c-article-identifiers__item">
                                    <span class="c-article-identifiers__open">Open Access</span>
                                </li>
                                
                                <li class="c-article-identifiers__item"><a href="#article-info">Published: <time datetime="2018-03-09" itemprop="datePublished">09 March 2018</time></a></li>
                            </ul>

                            
                            <h1 class="c-article-title u-h1" data-test="article-title" data-article-title="" itemprop="name headline">Automated high throughput animal CO1 metabarcode classification</h1>
                            <ul class="c-author-list js-list-authors js-etal-collapsed" data-etal="25" data-etal-small="3" data-test="authors-list"><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-1" class="c-author-list__item-icon js-no-scroll" data-corresp-id="c1">Teresita M. Porter</a></span><span class="u-js-hide">&nbsp;
            <a class="js-orcid js-no-scroll" itemprop="url" href="http://orcid.org/0000-0002-0227-6874"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0227-6874</a></span><sup class="u-js-hide"><a href="#Aff1" class="js-no-scroll" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="University of Guelph"><meta itemprop="address" content="0000 0004 1936 8198, grid.34429.38, The Centre for Biodiversity Genomics &amp; Department of Integrative Biology, University of Guelph, 50 Stone Road East, Guelph, ON, N1G 2W1, Canada"></span>,<a href="#Aff2" class="js-no-scroll" tabindex="-1">2</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="Natural Resources Canada"><meta itemprop="address" content="0000 0001 2295 5236, grid.202033.0, Great Lakes Forestry Centre, Natural Resources Canada, 1219 Queen Street East, Sault Ste. Marie, ON, P6A 2E5, Canada"></span></sup> &amp; </li><li class="c-author-list__item" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span itemprop="name"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-category="article body" data-track-label="link" href="#auth-2" class="js-no-scroll">Mehrdad Hajibabaei</a></span><sup class="u-js-hide"><a href="#Aff1" class="js-no-scroll" tabindex="-1">1</a><span itemprop="affiliation" itemscope="itemscope" itemtype="http://schema.org/Organization" class="u-visually-hidden"><meta itemprop="name" content="University of Guelph"><meta itemprop="address" content="0000 0004 1936 8198, grid.34429.38, The Centre for Biodiversity Genomics &amp; Department of Integrative Biology, University of Guelph, 50 Stone Road East, Guelph, ON, N1G 2W1, Canada"></span></sup>&nbsp;</li></ul>
                            

                            <p class="c-article-info-details" data-container-section="info"><i data-test="journal-title">Scientific Reports</i><b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;8</b>, Article&nbsp;number:&nbsp;<span data-test="article-number">4226</span> (<span data-test="article-publication-year">2018</span>)
                <span class="c-article-info-details__divider u-hide-print"> | </span><a class="c-article-info-details__download-citation u-hide-print" data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/s41598-018-22505-4.ris">
                Download Citation
            </a></p>
                            
                        </header>

                        
    

                        
    

                        
                    </div>

                    <div data-article-body="true" data-track-component="article body" class="c-article-body">
                        <section aria-labelledby="Abs1" lang="en"><div class="c-article-section js-article-section" id="Abs1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content js-collapsible-section" id="Abs1-content"><p>We
 introduce a method for assigning names to CO1 metabarcode sequences 
with confidence scores in a rapid, high-throughput manner. We compiled 
nearly 1 million CO1 barcode sequences appropriate for classifying 
arthropods and chordates. Compared to our previous Insecta classifier, 
the current classifier has more than three times the taxonomic coverage,
 including outgroups, and is based on almost five times as many 
reference sequences. Unlike other popular rDNA metabarcoding markers, we
 show that classification performance is similar across the length of 
the CO1 barcoding region. We show that the RDP classifier can make 
taxonomic assignments about 19 times faster than the popular top BLAST 
hit method and reduce the false positive rate from nearly 100% to 34%. 
This is especially important in large-scale biodiversity and 
biomonitoring studies where datasets can become very large and the 
taxonomic assignment problem is not trivial. We also show that reference
 databases are becoming more representative of current species diversity
 but that gaps still exist. We suggest that it would benefit the field 
as a whole if all investigators involved in metabarocoding studies, 
through collaborations with taxonomic experts, also planned to barcode 
representatives of their local biota as a part of their projects.</p></div></div></section>

                        

                        
                            <section aria-labelledby="Sec1"><div class="c-article-section js-article-section" id="Sec1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec1">Introduction</h2><div class="c-article-section__content js-collapsible-section" id="Sec1-content"><p>Ecological
 investigations, such as environmental biomonitoring, require the 
identification of individual specimens. This is normally done for each 
individual specimen by comparing suites of morphological characters with
 those described in taxonomic keys. In comprehensive studies that span 
different taxonomic groups, such as vegetation, microbes, and animals, 
it can be difficult to access the broad array of expertise and keys 
needed for identification. In large-scale studies it can also be costly 
and time consuming to process large numbers of samples in a timely 
manner. There are also difficulties in identifying damaged, partial, or 
immature specimens that lack the appropriate morphological characters 
for identification. Impediments to the study of taxonomy and the users 
of taxonomy have spurred the development and application of DNA-based 
techniques, such as marker gene surveys, coupled with high throughput 
sequencing for biodiversity research<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ebach, M. C., Valdecasas, A. G. &amp; Wheeler, Q. D. Impediments to taxonomy and users of taxonomy: accessibility and impact evaluation. Cladistics 27, 550–557 (2011)." href="#ref-CR1" id="ref-link-section-d7940e322">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hebert, P. D. N., Cywinska, A., Ball, S. L. &amp; deWaard, J. R. Biological identifications through DNA barcodes. Proc. R. Soc. B Biol. Sci. 270, 313–321 (2003)." href="#ref-CR2" id="ref-link-section-d7940e322_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Taberlet, P., Coissac, E., Pompanon, F., Brochmann, C. &amp; Willerslev, E. Towards next-generation biodiversity assessment using DNA metabarcoding. Mol. Ecol. 21, 2045–2050 (2012)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR3" id="ref-link-section-d7940e325">3</a></sup>.</p><p>The
 advantage of a marker gene approach is that it does not require the 
isolation or identification of individual specimens. Instead, the method
 can be used to survey the community of organisms present in the 
environmental DNA (eDNA) extracted from soil, water, or passively 
collected biomass. A broad array of organisms can be targeted by 
choosing the appropriate marker such as 16S ribosomal DNA (rDNA) for 
prokaryotes, ITS or LSU rDNA for fungi, or rbcL and matK chloroplast DNA
 for plants. When the cytochrome c oxidase (CO1) gene became the target 
of marker gene surveys, the approach became known as metabarcoding<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Taberlet, P., Coissac, E., Pompanon, F., Brochmann, C. &amp; Willerslev, E. Towards next-generation biodiversity assessment using DNA metabarcoding. Mol. Ecol. 21, 2045–2050 (2012)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR3" id="ref-link-section-d7940e332">3</a></sup>.
 This approach has already been applied to agricultural, forestry, 
fishery, conservation, biodiversity, and biomonitoring programs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Deiner, K. et al. Environmental DNA metabarcoding: transforming how we survey animal and plant communities. Mol. Ecol. 26, 5872–5895 (2017)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR4" id="ref-link-section-d7940e336">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="GRDI-EcoBiomics. Metagenomics Based Ecosystem Biomonitoring (GRDI-EcoBiomics) project, Government of Canada, Genomics R&amp;D Initiative, Year-End Performance Report for Shared Priority Projects (2016–2017). http://grdi-irdg.collaboration.gc.ca/eng/reports/annual/2015_2016.html (2016)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR5" id="ref-link-section-d7940e339">5</a></sup>.
 Applied metabarcoding has become so popular because of the widespread 
availability of kits to extract eDNA from different substrates and the 
availability of high throughput sequencing.</p><p>With DNA 
metabarcoding, taxonomic assignment in new studies is shifted to 
computational algorithms. Indeed, the method is entirely reliant on 
having DNA barcode sequences from curated samples that have already been
 identified. So the method does not entirely relieve the field from the 
taxonomic impediment, but it does improve accessibility to existing 
named DNA barcode sequences that ideally come from curated specimens. 
Though there are many options available for making taxonomic 
assignments, most CO1 metabarcodes are still routinely assigned using 
the familiar top BLAST hit method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Gibson, J. et al. Simultaneous assessment of the macrobiome and microbiome in a bulk sample of tropical arthropods through DNA metasystematics. Proc. Natl. Acad. Sci. 111, 8007–8012 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR6" id="ref-link-section-d7940e346">6</a></sup>. This is not an ideal situation since the top BLAST hit method has already been shown to be misleading<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Koski, L. B. &amp; Golding, G. B. The Closest BLAST Hit Is Often Not the Nearest Neighbor. J. Mol. Evol. 52, 540–542 (2001)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR7" id="ref-link-section-d7940e350">7</a></sup>, has a high false positive rate of assignment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Virgilio, M., Backeljau, T., Nevado, B. &amp; De Meyer, M. Comparative performances of DNA barcoding across insect orders. BMC Bioinformatics 11, 206 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR8" id="ref-link-section-d7940e354">8</a></sup>, is slow, and provides no statistical measure of confidence for taxonomic assignments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Porter, T. M. et al. Rapid and accurate taxonomic classification of insect (class Insecta) cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve Bayesian classifier. Mol. Ecol. Resour. 14, 929–942 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR9" id="ref-link-section-d7940e358">9</a></sup>.</p><p>The Ribosomal Database Project (RDP) classifier uses a naïve Bayesian approach to make taxonomic assignments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Wang, Q., Garrity, G. M., Tiedje, J. M. &amp; Cole, J. R. Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy. Appl. Environ. Microbiol. 73, 5261–5267 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR10" id="ref-link-section-d7940e365">10</a></sup>.
 The RDP classifier was originally developed using prokaryote 16 S rDNA 
sequences. The classifier can be trained, however, to make taxonomic 
assignments using any DNA marker. For example, the tool can also be used
 to classify fungi based on their ITS or LSU rDNA sequences<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Liu, K.-L., Porras-Alfaro, A., Kuske, C. R., Eichorst, S. A. &amp; Xie, G. Accurate, Rapid Taxonomic Classification of Fungal Large-Subunit rRNA Genes. Appl. Environ. Microbiol. 78, 1523–1533 (2012)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR11" id="ref-link-section-d7940e369">11</a></sup>.
 One advantage of using the RDP classifier over the more widely used top
 BLAST hit method is speed. This method is much faster and can process 
large datasets from high throughput sequencing in a fraction of the time
 that it would take with BLAST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Porter, T. M. et al. Rapid and accurate taxonomic classification of insect (class Insecta) cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve Bayesian classifier. Mol. Ecol. Resour. 14, 929–942 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR9" id="ref-link-section-d7940e373">9</a></sup>.
 This is especially important in large-scale biodiversity and 
biomonitoring studies where data sets can become very large. 
Additionally, unlike BLAST, the RDP classifier was specifically 
developed to provide a measure of confidence for assignments at each 
rank in the taxonomic hierarchy. This is a key aspect of the method as 
it allows users to easily summarize results to more inclusive ranks, 
where necessary, as well as filter for good results based on bootstrap 
support values. The RDP classifier is open-source, well-documented, and 
has a long history of use in related fields<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Cole, J. R. et al. Ribosomal Database Project: data and tools for high throughput rRNA analysis. Nucleic Acids Res. 42, D633–D642 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR12" id="ref-link-section-d7940e377">12</a></sup>.</p><p>Although a variety of resources exist for taxonomically assigning prokaryotes and fungi<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Liu, K.-L., Porras-Alfaro, A., Kuske, C. R., Eichorst, S. A. &amp; Xie, G. Accurate, Rapid Taxonomic Classification of Fungal Large-Subunit rRNA Genes. Appl. Environ. Microbiol. 78, 1523–1533 (2012)." href="#ref-CR11" id="ref-link-section-d7940e385">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cole, J. R. et al. Ribosomal Database Project: data and tools for high throughput rRNA analysis. Nucleic Acids Res. 42, D633–D642 (2014)." href="#ref-CR12" id="ref-link-section-d7940e385_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ludwig, W. ARB: a software environment for sequence data. Nucleic Acids Res. 32, 1363–1371 (2004)." href="#ref-CR13" id="ref-link-section-d7940e385_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="DeSantis, T. Z. et al. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB. Appl. Environ. Microbiol. 72, 5069–5072 (2006)." href="#ref-CR14" id="ref-link-section-d7940e385_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pruesse, E. et al. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. Nucleic Acids Res. 35, 7188–7196 (2007)." href="#ref-CR15" id="ref-link-section-d7940e385_4">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Abarenkov, K. et al. The UNITE database for molecular identification of fungi – recent updates and future perspectives. New Phytol. 186, 281–285 (2010)." href="#ref-CR16" id="ref-link-section-d7940e385_5">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Ankenbrand, M. J., Keller, A., Wolf, M., Schultz, J. &amp; Förster, F. ITS2 Database V: Twice as Much. Mol. Biol. Evol. 32, 3030–3032 (2015)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR17" id="ref-link-section-d7940e388">17</a></sup>,
 most CO1 metabarcodes are still assigned using BLAST. The BOLD CO1 
database was designed as a curation and analysis tool for individual 
specimens and it is not suitable for the analysis of the large batches 
of CO1 metabarcodes generated by high throughput sequencing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Ratnasingham, S. &amp; Hebert, P. D. BOLD: The Barcode of Life Data System (
                    http://www.barcodinglife.org
                    
                  ). Mol. Ecol. Notes 7, 355–364 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR18" id="ref-link-section-d7940e392">18</a></sup>.
 To address that gap, we previously developed an Insecta CO1 training 
set that could be used with the RDP classifier. Unfortunately, it could 
not be used to identify non-insect animals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Porter, T. M. et al. Rapid and accurate taxonomic classification of insect (class Insecta) cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve Bayesian classifier. Mol. Ecol. Resour. 14, 929–942 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR9" id="ref-link-section-d7940e396">9</a></sup>.</p><p>The
 purpose of this study is to introduce the CO1 metabarcoding community 
to an alternative method for taxonomically assigning CO1 metabarcodes. 
(1) We compiled a comprehensive training set for the RDP classifier 
focusing on the Chordata and Arthropoda, the two largest groups of 
publicly available CO1 sequences. (2) We benchmark the performance of 
the classifier for a range of sequence lengths and taxa with a focus on 
groups important for freshwater biomonitoring. (3) We provide guidelines
 for bootstrap support cutoffs. (4) We show that the RDP classifier is 
faster than the top BLAST hit approach and has a lower false positive 
rate (FPR). (5) We also show the improvement in insect CO1 
classification and reference set coverage from 2013 to 2016. Altogether 
we show that this approach is a significant improvement over the widely 
used top BLAST hit method for CO1 metabarcode taxonomic assignments.</p></div></div></section><section aria-labelledby="Sec2"><div class="c-article-section js-article-section" id="Sec2-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content js-collapsible-section" id="Sec2-content"><p>In
 this study we introduce a new CO1 training set suitable for classifying
 the broad range of arthropods and chordates commonly found in 
metabarcode studies geared towards ecological assessments and 
biomonitoring. Improvements include more than three times the taxonomic 
coverage, outgroup taxa to flag other major eukaryote taxa, and a more 
comprehensive training set based on almost five times the number of 
sequences. The term ‘training set’ refers to the set of files produced 
after we trained the classifier to make CO1 taxonomic assignments. 
Training sets can be used directly with the RDP classifier and no 
further training is needed from the user. Our main results are the v1 
and v2 training sets that can be used to make genus- and species-rank 
taxonomic assignments, respectively. The taxonomic composition of the 
CO1 Eukaryote v1 training set is summarized in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab1">1</a> and in detail in Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S1</a>. A similar summary table is shown for the CO1 Eukaryote v2 training set in Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S2</a>.
 Outgroup taxa were also included to help sort non-Arthropod and 
non-Chordata taxa into broad groups such as fungi, diatoms, or 
nematodes.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 CO1 Eukaryote v1 training set summary.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://www.nature.com/articles/s41598-018-22505-4/tables/1"><span>Full size table</span></a></div></figure></div><p>The
 proportion of singletons in the dataset can indicate the presence of 
groups with low coverage and so is summarized for both the v1 and v2 
training sets in Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S3</a>.
 The proportion of singleton genera in the genus-trained classifier is 
23% compared with the proportion of singleton species in the 
species-trained classifier at 33%. In this study we define a false 
positive (FP) as an incorrect taxonomic assignment with a bootstrap 
support value greater than the cutoff. To avoid making FP assignments, 
the bootstrap cutoffs presented in this study should be treated as <i>minimum</i>
 cutoff values. Taxa for the training set were sampled to emphasize 
Arthropoda and Chordata since these were the best-represented eukaryote 
phyla in the GenBank nucleotide database. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Fig1">1</a>
 shows the proportion of correctly assigned sequences for a variety of 
query lengths at a variety of taxonomic ranks. Since the classifier is 
not meant to classify taxa not represented in the database, 
leave-one-out testing results from singletons were excluded from this 
figure and no bootstrap support cutoff was used. Classifier accuracy is 
highest at more inclusive taxonomic ranks, especially for fragments 
200 bp or longer.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1"><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 1</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig1_HTML.jpg?as=webp"><img aria-describedby="figure-1-desc" src="41598_2018_22505_Fig1_HTML.jpg" alt="figure1" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>The
 proportion of correct taxonomic assignments increases with more 
inclusive taxonomic ranks and longer CO1 sequences. Results summarize 
results from leave-one-out testing of the CO1 Eukaryote v1 training set.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/1" data-track-dest="link:Figure1 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>The
 receiver operator characteristic (ROC) curves for full length CO1 
barcode sequences identified to various taxonomic ranks and a range of 
fragment lengths at the genus rank is shown in Figure <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S1</a>.
 A ROC curve shows the relationship between the false positive rate 
(FPR) and the true positive rate (TPR) as the bootstrap support cutoff 
is tuned from 0 to 100%. The FPR represents the proportion of incorrect 
assignments with a high bootstrap support value out of all incorrect 
assignments. The TPR represents the proportion of correct assignments 
with a high bootstrap support value out of all correct assignments. In 
these figures, the bootstrap cutoff values are not directly shown on the
 plot, rather we show the resulting change in TPR and FPR. Generally, 
lower cutoffs result in lower TPRs (and FPRs) and higher cutoffs result 
in higher TPRs (and FPRs). The optimal place to be when choosing a 
cutoff value is in the top left quadrant where the TPR is high and the 
FPR is low. In this figure, the ROC curves are calculated from 
leave-one-out testing results where singletons were excluded. Points 
lying above the 50% line indicate results better than those obtained by 
chance. The high area-under-the-curve values indicate high true positive
 rates and good classifier performance across a wide range of bootstrap 
support values. In the top figure, the TPR for full length CO1 sequences
 at all taxonomic ranks is high indicating that most of the assignments 
are correctly assigned at any cutoff value. In the bottom figure, TPRs 
increases with longer CO1 fragment lengths.</p><p>Since classification 
performance varies with fragment size and taxonomic assignment rank, we 
have calculated a matrix of minimum bootstrap support value cutoffs to 
obtain 99% correct assignments during leave one out testing (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab2">2</a>).
 The assumption here, as well as when using the top BLAST hit method, is
 that the query sequence is actually represented in the database. 
Singletons were excluded from this analysis so cutoffs are based on 77% 
of the sequences in the original CO1 Eukaryote v1 training set. Also 
shown is the corresponding reduction in the proportion of classified 
sequences after applying the minimum bootstrap support cutoff values. A 
similar table for the Eukaryote v2 classifier trained to the species 
rank is shown in Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S4</a>.
 Generally as the amount of sequence information decreases with 
decreasing CO1 sequence length, higher bootstrap support cutoff values 
are needed to observe 99% correct assignments. Similarly, as assignments
 are made to increasingly specific ranks, higher cutoff values are 
required to observe 99% correct assignments.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b id="Tab2" data-test="table-caption">Table
 2 Bootstrap support cutoff values that produced at least 99% correct 
assignments during CO1 Eukaryote v1 leave-one-out testing.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://www.nature.com/articles/s41598-018-22505-4/tables/2"><span>Full size table</span></a></div></figure></div><p>Applying a bootstrap support cutoff can reduce the proportion of incorrect taxonomic assignments. Figure <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S2</a>
 shows the proportion of incorrect assignments for Arthropoda sequences 
both with and without using bootstrap support cutoffs. Leave one out 
testing results from singletons were included here to simulate the 
taxonomic assignment of sequences without congenerics in the database. 
The 70% bootstrap support cutoff value was selected for full length 
(500 bp+) CO1 sequences as shown in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab2">2</a>. When classifying Arthropoda sequences, 23% of which were known to have no congenerics in the training set (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S3</a>),
 applying a 70% bootstrap support cutoff at the genus rank reduced the 
misclassification rate for nearly all classes to ~1% while reducing the 
number of assigned sequences by ~3%. A similar analysis with Chordata is
 shown in Figure <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S3</a>.
 The proportion of incorrect assignments for all phyla, Arthropoda and 
Chordata classes, as well as for the orders in the large Insecta and 
Actinopteri groups are shown in the Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S5</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S9</a>.
 When we focus on groups that are particularly important in freshwater 
biomonitoring, we see that database sequences are highly skewed towards 
Diptera and that although the proportion of incorrect classification 
varies across groups the application of a bootstrap support cutoff 
reduces these rates to ~1% incorrect assignments (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab3">3</a>).
 One exception is for sequences in Megaloptera that have a higher 
proportion of incorrect assignments (1.7%) even after using a 70% 
bootstrap support cutoff at the genus rank for full length (500 bp+) CO1
 sequences. These tables show how database representation and 
misclassification rates can vary across taxonomic groups.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-3"><figure><figcaption class="c-article-table__figcaption"><b id="Tab3" data-test="table-caption">Table 3 Representation of freshwater biomonitoring taxa in the Eukaryote CO1v1 training set.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://www.nature.com/articles/s41598-018-22505-4/tables/3"><span>Full size table</span></a></div></figure></div><p>Classification
 performance may also vary for partial CO1 sequences whether they are 
sampled randomly from across the barcoding region (as in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Fig1">1</a>) or if they are anchored by CO1 primers (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Fig2">2</a>).
 The coverage of primer-anchored 200 bp sequences sampled from the 
dataset varies across the length of the barcoding region. Since primers 
are often trimmed before submission to GenBank, it was not surprising 
that the Folmer barcoding primers, and other primers designed near the 
5′ and 3′ end of the barcoding region, had especially low coverage in 
our training set (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Fig3">3</a>). The proportion of correct assignments of primer-anchored 200 bp sequences with and without 60% bootstrap support (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab2">2</a>)
 is also shown. Singletons were not included in this analysis. The 
proportion of correct assignments is especially high at the order to 
kingdom ranks. After applying the bootstrap support cutoff, the 
proportion of correct taxonomic assignments rose to ~99% across all 
primers at the genus, family, and order ranks. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Fig3">3</a>
 shows that there are minimal differences in CO1 classification 
performance across the barcoding region that comprises 6 alpha helices 
and 5 loop regions. This is in contrast with other popular metabarcode 
markers such as prokaryote 16 S and fungal ITS rDNA markers that show 
distinctive stem-loop secondary structure with peaks and troughs of 
classification accuracy across variable domains (loops) and conserved 
regions (stems), respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Wang, Q., Garrity, G. M., Tiedje, J. M. &amp; Cole, J. R. Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy. Appl. Environ. Microbiol. 73, 5261–5267 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR10" id="ref-link-section-d7940e1664">10</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 2</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig2_HTML.jpg?as=webp"><img aria-describedby="figure-2-desc" src="41598_2018_22505_Fig2_HTML.jpg" alt="figure2" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>CO1
 primers included in this study. Primer map of the CO1 barcoding region 
showing the relative position and direction of the primer-anchored 
200 bp fragments analyzed in this study. The CO1 helix regions that are 
embedded in the mitochondrial inner membrane are also shown for 
reference.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/2" data-track-dest="link:Figure2 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3"><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 3</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig3_HTML.jpg?as=webp"><img aria-describedby="figure-3-desc" src="41598_2018_22505_Fig3_HTML.jpg" alt="figure3" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>The
 proportion of correctly assigned primer-anchored 200 bp sequences can 
vary across the CO1 barcoding region before applying a bootstrap support
 cutoff. Primer names are prefixed with the outermost alignment position
 along the CO1 barcoding region and are arranged along the x-axis in the
 order that they would be encountered from the 5′ to 3′ end. Top panel: 
Coverage of primer-anchored 200 bp sequences in the CO1 Eukaryote v1 
training set. Middle panel: Proportion of correct taxonomic assignments.
 Bottom panel: Proportion of correct assignments after filtering by a 
60% bootstrap support cutoff at the genus rank. Note the differing scale
 on the y-axes.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/3" data-track-dest="link:Figure3 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>A
 comparison of taxonomic assignment outcomes using the top BLAST hit 
method and the RDP classifier with the CO1 Eukaryote v1 training set is 
shown for all primer-anchored 200 bp fragments in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab4">4</a> and Figure <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S4</a>.
 Using BLAST, no hits were returned for some queries because the expect 
value (e-value) was greater than the default cutoff of 10. In contrast, 
using the RDP classifier, a result was returned for every query. 
Assignment accuracy (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab4">4</a>)
 is highest for the top BLAST hit method, however, the FPR is ~3 times 
higher for BLAST than for the RDP classifier. This is significant 
because in this example, 397,820 taxonomic assignments are classified as
 ‘good’ based on the top BLAST hit metrics but they are actually 
incorrect. In general, using the RDP classifier with the CO1 Eukaryote 
v1 training set and the recommended minimum bootstrap support cutoff at 
the genus rank significantly reduces the FPR.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-4"><figure><figcaption class="c-article-table__figcaption"><b id="Tab4" data-test="table-caption">Table
 4 Taxonomic assignment outcomes at the genus rank from primer-anchored 
200 bp sequences using the top BLAST hit method compared with the RDP 
classifier and the CO1 Eukaryote v1 training set.</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-category="article body" data-track-label="button" rel="nofollow" href="https://www.nature.com/articles/s41598-018-22505-4/tables/4"><span>Full size table</span></a></div></figure></div><p>We
 also compared the time needed to make high-throughput sequence-based 
taxonomic assignments using the top BLAST hit method and the RDP 
classifier (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Fig4">4</a>).
 Using a single processor, making assignments using the RDP classifier 
with the CO1 Eukaryote v1 training set was on average ~19 times faster 
than using the top BLAST hit method. We did not consider the extra time 
needed to process tabular BLAST output into a usable format by adding 
taxonomic lineages and calculating query coverage.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4"><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-category="article body" data-track-label="image" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig4_HTML.jpg?as=webp"><img aria-describedby="figure-4-desc" src="41598_2018_22505_Fig4_HTML.jpg" alt="figure4" loading="lazy"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>The
 RDP classifier taxonomically assigns more queries per minute than the 
top BLAST hit method. The number of primer-anchored 200 bp query 
sequences taxonomically assigned per minute is compared using the top 
BLAST hit method against a locally installed copy of the nucleotide 
database and the RDP classifier 2.12 with the CO1 Eukaryote v1 training 
set.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="view figure" href="https://www.nature.com/articles/s41598-018-22505-4/figures/4" data-track-dest="link:Figure4 Full size image" rel="nofollow"><span>Full size image</span></a></div></figure></div><p>Compared
 with a 2013 training set, we found ~3 times more class Insecta 
reference sequences 500 bp + identified to the species rank (561,841 
versus 190,333) and one additional order ‘Zoraptera’ from GenBank (Table
 <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S10</a>).
 The group of top five Insecta orders with the greatest number of 
reference sequences has not changed from 2013 to 2016, though each group
 contains many more reference sequences today than 3 years ago (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S11</a>).
 The bottom five Insecta orders with the least number of reference 
sequences has changed slightly from 2013 and in the current training set
 includes Grylloblattodea (n = 1), Zoraptera (n = 2), undef_Insecta 
(n = 9), Mantophasmatodea (n = 30), and Dermaptera (n = 37) (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S12</a>).
 As expected, as the number of reference sequences in the database 
grows, the proportion of genus rank incorrect assignments decreases 
(Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S13</a>). Representation of various Insecta orders is shown in detail for CO1 sequences in the large class Insecta (Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S8</a>).
 To further reduce misclassification rates in class Insecta, reference 
sequences for the Grylloblattodea, Zoraptera, Lepidotrichidae, 
Lepismatidae, Nicoletiidae, Mantophasmatodea, and Dermaptera need to be 
added to public databases.</p></div></div></section><section aria-labelledby="Sec3"><div class="c-article-section js-article-section" id="Sec3-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec3">Discussion</h2><div class="c-article-section__content js-collapsible-section" id="Sec3-content"><p>Metabarcoding
 is widely used to survey bacteria using 16 S rDNA, fungi using ITS 
rDNA, plants using rbcL + matK cpDNA, as well as animals using CO1 rDNA 
markers. Bioinformatics, specifically taxonomic assignment methods, are a
 moving-target in this rapidly developing field. There are broad 
communities already devoted to developing 16 S and ITS rDNA resources<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Cole, J. R. et al. Ribosomal Database Project: data and tools for high throughput rRNA analysis. Nucleic Acids Res. 42, D633–D642 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR12" id="ref-link-section-d7940e1979">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="DeSantis, T. Z. et al. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB. Appl. Environ. Microbiol. 72, 5069–5072 (2006)." href="#ref-CR14" id="ref-link-section-d7940e1982">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pruesse, E. et al. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. Nucleic Acids Res. 35, 7188–7196 (2007)." href="#ref-CR15" id="ref-link-section-d7940e1982_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Abarenkov, K. et al. The UNITE database for molecular identification of fungi – recent updates and future perspectives. New Phytol. 186, 281–285 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR16" id="ref-link-section-d7940e1985">16</a></sup>.
 In the field of CO1 metabarcoding there is plenty of room for newer 
algorithms and method development. An advantage of CO1 metabarcoding is 
the ability to directly compare eDNA metabarcodes with real communities 
with tractable and quantifiable richness using morphology-based methods.
 Similar comparisons are a difficult feat for hyper-diverse microbial 
communities where base-line richness is often unknown. This provides an 
excellent foundation for benchmarking the performance of new methods.</p><p>CO1
 metabarcoding has been extensively compared with morphology-based 
biomonitoring methods across a range of applications and has been 
repeatedly shown to detect more and/or a complementary suite of taxa 
compared with traditional methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Deiner, K. et al. Environmental DNA metabarcoding: transforming how we survey animal and plant communities. Mol. Ecol. 26, 5872–5895 (2017)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR4" id="ref-link-section-d7940e1992">4</a></sup>.
 The continued interest and growing popularity of DNA metabarcoding in a
 diverse array of fields is driven by the scalability of this method 
when coupled with high throughput DNA sequencing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Porter, T. M. &amp; Hajibabaei, M. Scaling up: A guide to high throughput genomic approaches for biodiversity analysis. Mol. Ecol. (In press)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR19" id="ref-link-section-d7940e1996">19</a></sup>.
 Detection of taxa especially important for biomonitoring relies on 
standardized, representative, and reproducible field sampling methods 
such as those developed by the Canadian Aquatic Biomonitoring Network 
(CABIN) or the Australian River Assessment Scheme (AUSRIVAS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Smith, M. J. et al. AusRivAS: using macroinvertebrates to assess ecological condition of rivers in Western Australia. Freshw. Biol. 41, 269–282 (1999)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR20" id="ref-link-section-d7940e2000">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="McDermott, H., Paull, T. &amp; Strachan, S. CABIN Canadian Aquatic Biomonitoring Network, Laboratory Methods: Processing, Taxonomy, and Quality Control of Benthic Macroinvertebrate Samples. (Environment Canada, 2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR21" id="ref-link-section-d7940e2003">21</a></sup>.
 Improvement of lab methods such as primer development for PCR, the use 
of multiple markers to increase detection coverage, and the development 
of PCR-free methods are active areas of research<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Gibson, J. et al. Simultaneous assessment of the macrobiome and microbiome in a bulk sample of tropical arthropods through DNA metasystematics. Proc. Natl. Acad. Sci. 111, 8007–8012 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR6" id="ref-link-section-d7940e2007">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elbrecht, V. &amp; Leese, F. Validation and Development of COI Metabarcoding Primers for Freshwater Macroinvertebrate Bioassessment. Front. Environ. Sci. 5, (2017)." href="#ref-CR22" id="ref-link-section-d7940e2010">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gibson, J. F. et al. Large-Scale Biomonitoring of Remote and Threatened Ecosystems via High-Throughput Sequencing. PLOS ONE 10, e0138432 (2015)." href="#ref-CR23" id="ref-link-section-d7940e2010_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Shokralla, S. et al. Environmental DNA Barcode Sequence Capture: Targeted, PCR-free Sequence Capture for Biodiversity Analysis from Bulk Environmental Samples. bioRxiv 087437 (2016)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR24" id="ref-link-section-d7940e2013">24</a></sup>. Despite all this work to validate CO1 metabarcoding, a commonly overlooked area has been at the taxonomic assignment step.</p><p>Too
 often, it has been accepted without question that a generic top BLAST 
hit approach is sufficient to make high throughput CO1 taxonomic 
assignments. Unfortunately, the BLAST metrics commonly used for 
delimiting good taxonomic assignments such as percent identity, query 
coverage, bit score, e-value, or combinations thereof simply provides 
different measures of similarity to a top hit and a measure of random 
background noise in the database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="NCBI. BLAST Frequently Asked Questions. https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=FAQ#expect (2018)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR25" id="ref-link-section-d7940e2020">25</a></sup>.
 The RDP classifier, on the other hand, was developed specifically to 
make taxonomic assignments from marker gene sequences and provide a 
measure of confidence to assess how likely the assignment is to be 
correct<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Wang, Q., Garrity, G. M., Tiedje, J. M. &amp; Cole, J. R. Naive Bayesian Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial Taxonomy. Appl. Environ. Microbiol. 73, 5261–5267 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR10" id="ref-link-section-d7940e2024">10</a></sup>.
 With the top BLAST hit method, if there is no top BLAST hit that meets 
the user’s criteria for a good assignment, then no assignment can be 
made. With the RDP classifier, if there are no congenerics in the 
database or if the genus rank assignment has a low confidence score, it 
may still be possible to make an assignment to a more inclusive rank if 
there are, for example, confamilial sequences in the database.</p><p>The impact of different kinds of taxonomic assignment errors has been discussed in the literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Virgilio, M., Backeljau, T., Nevado, B. &amp; De Meyer, M. Comparative performances of DNA barcoding across insect orders. BMC Bioinformatics 11, 206 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR8" id="ref-link-section-d7940e2031">8</a></sup>.
 One particular concern is the effect of false positive taxonomic 
assignments. In this study, a false positive was defined as a sequence 
taxonomically assigned with high confidence even though it is wrong. 
This is especially significant when the cost of making a 
misidentification is high. A false positive assignment, for example, 
could lead investigators to over-estimate the presence or distribution 
of a rare threatened or endangered species<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Wilcox, T. M. et al. Robust Detection of Rare Species Using Environmental DNA: The Importance of Primer Specificity. PLoS ONE 8, e59520 (2013)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR26" id="ref-link-section-d7940e2035">26</a></sup>
 or the assignment may a create false alarm for an invasive or harmful 
species. In such cases, the RDP classifier is a more reliable tool to 
use than BLAST.</p><p>Another contributor to the false positive rate is 
high confidence taxonomic assignments made to incorrectly identified 
entries in public databases. The question of annotation accuracy in 
GenBank is a known issue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Bidartondo, M. I. Preserving Accuracy in GenBank. Science 319, 1616–1616 (2008)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR27" id="ref-link-section-d7940e2043">27</a></sup>.
 For example, it has been estimated that fungal ITS sequences in GenBank
 may be incorrectly identified to the species rank about 20% of the time<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Nilsson, R. H. et al. Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective. PLoS ONE 1, e59 (2006)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR28" id="ref-link-section-d7940e2047">28</a></sup>.
 It is reasonable to assume that similar issues also affect CO1 
sequences in GenBank. With respect to taxonomic assignment, one way to 
circumvent this problem is to summarize taxonomic assignments to the 
genus or other more inclusive ranks. With the top BLAST hit method, 
similarity statistics represent a single query-reference sequence 
comparison. In this situation, if the top BLAST hit record has an 
incorrect identification then the taxonomic assignment will be a 
false-positive if the BLAST metrics meet the user’s cutoff values. With 
the RDP classifier, k-mer frequency profiles are based on all the 
sequences for each unique genus in a genus-trained classifier. In this 
situation, a few erroneous k-mers and frequencies are unlikely to have a
 large effect if most of the taxa are correctly identified. For a taxon 
that is poorly-represented in the database by just one or a few 
sequences, however, the presence of a few misidentified sequences would 
have a greater effect on the RDP classifier making a false-positive more
 likely. For this reason, in cases were species-level taxonomic 
assignments are needed, the user needs to be aware of the database 
coverage of their taxa of interest. Generally, the responsibility for 
ensuring that high quality CO1 reference sequence datasets are available
 falls to the user-community. For example, the prokaryote 16 S rDNA 
SILVA and Greengenes reference sets as well as the fungal ITS UNITE rDNA
 database were developed and are maintained by researchers with a stake 
in the field and they provide a tremendous resource for the broader 
user-community<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="DeSantis, T. Z. et al. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB. Appl. Environ. Microbiol. 72, 5069–5072 (2006)." href="#ref-CR14" id="ref-link-section-d7940e2051">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pruesse, E. et al. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. Nucleic Acids Res. 35, 7188–7196 (2007)." href="#ref-CR15" id="ref-link-section-d7940e2051_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Abarenkov, K. et al. The UNITE database for molecular identification of fungi – recent updates and future perspectives. New Phytol. 186, 281–285 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR16" id="ref-link-section-d7940e2054">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Nilsson, R. H. et al. A Comprehensive, Automatically Updated Fungal ITS Sequence Dataset for Reference-Based Chimera Control in Environmental Sequencing Efforts. Microbes Environ. 30, 145–150 (2015)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR29" id="ref-link-section-d7940e2057">29</a></sup>. These researchers mine data from GenBank but also subject the data to 3<sup>rd</sup>
 party curation to provide better quality reference sets. For the animal
 CO1 community, GenBank entries with the BARCODE keyword indicate high 
quality records<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Ratnasingham, S. &amp; Hebert, P. D. BOLD: The Barcode of Life Data System (
                    http://www.barcodinglife.org
                    
                  ). Mol. Ecol. Notes 7, 355–364 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR18" id="ref-link-section-d7940e2063">18</a></sup>.</p><p>In
 this study a false negative was defined as a sequence correctly 
classified but with a confidence score below the threshold cutoff. The 
RDP classifier is more prone to FN’s than BLAST. Type II error also 
encompasses the outcome when a sequence cannot be classified because 
congeneric sequences are missing from the database. For example, this 
latter scenario is of particular concern in quarantine situations that 
could result in the introduction of parasites, pathogens, or invasive 
species<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Virgilio, M., Backeljau, T., Nevado, B. &amp; De Meyer, M. Comparative performances of DNA barcoding across insect orders. BMC Bioinformatics 11, 206 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR8" id="ref-link-section-d7940e2070">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Armstrong, K. F. &amp; Ball, S. L. DNA barcodes for biosecurity: invasive species identification. Philos. Trans. R. Soc. B Biol. Sci. 360, 1813–1823 (2005)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR30" id="ref-link-section-d7940e2073">30</a></sup>.
 In theory, in a quarantine situation where a limited suite of taxa is 
of interest, it should be easier to compile a representative database. 
As databases grow, high confidence assignments should improve and the 
rate of false negatives due to missing congenerics in the database 
should be reduced.</p><p>If every investigator involved in a 
metabarcoding study could further collaborate with taxonomic experts to 
identify a few representatives of their local biota, this could expedite
 the process of making public databases more representative. For 
example, previous work has shown that as few as 12% of described extant 
Insecta genera (8,679/72,618) are currently represented by full length 
(500 bp+) CO1 barcode sequences identified to the species rank in the 
GenBank nucleotide database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Porter, T. M. et al. Rapid and accurate taxonomic classification of insect (class Insecta) cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve Bayesian classifier. Mol. Ecol. Resour. 14, 929–942 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR9" id="ref-link-section-d7940e2080">9</a></sup>.
 Querying the database three years later, we found out that 22% of 
extant Insecta genera (16,285/72,618) are now represented by full length
 sequences identified to the species rank in the GenBank nucleotide 
database. At this rate of growth it could take 27 more years for all 
currently described extant Insecta genera to be represented by a full 
length CO1 barcode sequence in GenBank, not counting taxa still waiting 
to be described. This is only the tip of the metaphorical CO1 barcode 
iceberg, however, as the number of insect species is exponentially 
higher than genera and CO1 sequence representation in databases is 
expected to be even less at the species rank. This data gap could have 
significant implications to leverage the full potential of CO1 
metabarcoding in current studies. We suggest that an immediate way to 
improve the number of high confidence assignments for current studies is
 to sequence CO1 barcodes for common representatives of local biota to 
supplement existing databases. Even if they are not identified to the 
species rank, they will still represent local biota that can be targeted
 for further study if they prove to be of interest.</p><p>Developments 
in the field of CO1 taxonomic assignment are mostly geared to the 
assignment of single queries though some methods can assign batches of 
sequences at once. Methods range from tools that use HMM alignment 
followed by a linear search<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Ratnasingham, S. &amp; Hebert, P. D. BOLD: The Barcode of Life Data System (
                    http://www.barcodinglife.org
                    
                  ). Mol. Ecol. Notes 7, 355–364 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR18" id="ref-link-section-d7940e2087">18</a></sup>, Neighbor-joining analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Hebert, P. D. N., Cywinska, A., Ball, S. L. &amp; deWaard, J. R. Biological identifications through DNA barcodes. Proc. R. Soc. B Biol. Sci. 270, 313–321 (2003)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR2" id="ref-link-section-d7940e2091">2</a></sup>, BLAST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Altschul, S. F. et al. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25, 17 (1997)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR31" id="ref-link-section-d7940e2095">31</a></sup>, minimum distance and fuzzy set theory<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Shi, Z. Y. et al. FuzzyID2: A software package for large dataset species identification via barcoding and metabarcoding using Hidden Markov Models and fuzzy set methods. Mol. Ecol. Resour. (In press)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR32" id="ref-link-section-d7940e2099">32</a></sup>, the coalescent<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Abdo, Z. &amp; Golding, G. B. A Step Toward Barcoding Life: A Model-Based, Decision-Theoretic Method to Assign Genes to Preexisting Species Groups. Syst. Biol. 56, 44–56 (2007)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR33" id="ref-link-section-d7940e2103">33</a></sup>, segregating sites<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Lou, M. &amp; Golding, G. B. Assigning sequences to species in the absence of large interspecific differences. Mol. Phylogenet. Evol. 56, 187–194 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR34" id="ref-link-section-d7940e2108">34</a></sup>, neural networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Zhang, A. B., Sikes, D. S., Muster, C. &amp; Li, S. Q. Inferring Species Membership Using DNA Sequences with Back-Propagation Neural Networks. Syst. Biol. 57, 202–215 (2008)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR35" id="ref-link-section-d7940e2112">35</a></sup>, and support vector machines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Seo, T.-K. Classification of Nucleotide Sequences Using Support Vector Machines. J. Mol. Evol. 71, 250–267 (2010)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR36" id="ref-link-section-d7940e2116">36</a></sup>.
 Other than the first three methods, none of the alternative methods 
have caught on for CO1 taxonomic assignment most likely because the 
average user is not aware they exist or there is no portal to allow for 
easy implementation. The RDP naïve Bayesian classifier can be used to 
make taxonomic assignments in large batches. Compared to our previous 
Insecta CO1 classifier, the current CO1 Eukaryote training sets we 
describe here are suitable for classifying the broad range of arthropods
 and chordates commonly found in metabarcode studies geared towards 
ecological assessments and biomonitoring. Our method leverages the 
well-known RDP classifier and adapts it for use to classify animal CO1 
mtDNA. Aside from the capabilities demonstrated in this study, we 
believe the long history of this method, open-source availability, and 
clear documentation will help widespread application of this method for 
fast and accurate high throughput taxonomic assignments. Future work 
involves making this method available through a web portal.</p></div></div></section><section aria-labelledby="Sec4"><div class="c-article-section js-article-section" id="Sec4-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec4">Methods</h2><div class="c-article-section__content js-collapsible-section" id="Sec4-content"><p>Three
 sets of CO1 reference sequences were assembled: 1) Arthropoda, 2) 
Chordata, and 3) outgroup taxa as described below using Perl with 
BioPerl modules and the Ebot script<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Sayers, E. W. Ebot. http://www.ncbi.nlm.nih.gov/Class/PowerTools/eutils/course.html (2018)" href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR37" id="ref-link-section-d7940e2128">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Stajich, J. E. et al. The Bioperl toolkit: Perl modules for the life sciences. Genome Res. 12, 1611–1618 (2002)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR38" id="ref-link-section-d7940e2131">38</a></sup>.
 The following search terms were used to query the NCBI taxonomy 
database: 1) “Arthropoda”[ORGN] AND “species”[RANK] [Aug. 10, 2016], 2) 
“Chordata”[ORGN] AND “species”[RANK] [Aug. 24, 2016], and 3) “cellular 
organisms”[ORGN] AND “species”[RANK] NOT (“Arthropoda”[ORGN] OR 
“Chordata”[ORGN]) [Oct. 24, 2016]. A formatted taxon list was created 
using only taxa with complete binomial species names excluding the names
 containing sp., nr., aff., and cf. The NCBI nucleotide database was 
queried using the Entrez search term “cox1[gene] OR coxI[gene] OR 
CO1[gene] OR COI[gene] AND” the formatted taxon lists from above. For 
the outgroup taxa, the additional term “BARCODE”[keyword] was used. 
Sequences were retained if they were at least 500 bp and multiple 
sequences per species were retained when available. The associated 
taxonomic lineage was retrieved for each sequence. Human contaminant 
sequences were identified using BLAST and a custom database comprised of
 only human CO1 sequences. The taxonomic reports of hits with high query
 length coverage and high percent identity to known human sequences were
 individually explored, removed where necessary, and reported to NCBI. 
The Arthropoda, Chordata, and outgroup taxa were combined to create the 
CO1 Eukaryote v1 set trained to the genus rank and used with the RDP 
classifier v 2.12 for leave-one-out testing, cross-validation testing, 
and classifier training. A CO1 Eukaryote v2 training set was also 
created using the same sequences from above but was trained to the 
species rank. These training sets can be downloaded from <a href="https://github.com/terrimporter/CO1Classifier">https://github.com/terrimporter/CO1Classifier</a> and can only be used with the command-line version of the RDP classifier that can be downloaded separately from <a href="https://sourceforge.net/projects/rdp-classifier/">https://sourceforge.net/projects/rdp-classifier/</a>.</p><p>Since
 metabarcoding samples often contain partially degraded eDNAs, shorter 
fragments are often targeted to increase PCR and sequencing success. As a
 result, leave-one-out testing was performed for full length (500 bp+) 
CO1 sequences as well as for 400 bp, 200 bp, 100 bp, and 50 bp 
fragments. During leave-one-out testing, a sequence is removed from the 
dataset before it is classified. An assignment is scored as correct if 
the assignment matches the known taxonomy for the sequence. This 
assignment is made using a full set of 8 bp ‘words’ subsampled from the 
query sequence. Bootstrap support is assessed by randomly subsampling 
some of the 8 bp ‘words’ from the query sequence, making an assignment 
based on this set of 8 bp ‘words’, and repeating this procedure 100 
times. The proportion of times the original taxonomic assignment is 
recovered becomes the reported bootstrap support value for the 
assignment. The sequence is returned to the training set and the next 
sequence is removed, classified, and so on. The purpose of this type of 
testing is to assess classifier performance.</p><p>CO1 primers from the 
literature, especially those targeting invertebrates or developed 
especially for metabarcoding eDNA were compiled. Primers tested in this 
study and their references are shown in Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S14</a>. These primers were aligned against the <i>Drosophila yakuba</i> CO1 region obtained from GenBank accession X03240 using Mesquite v 3.10<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Maddison, W. P. &amp; Maddison, D. R. Mesquite version 3.10. http://mesquiteproject.org (2015)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR39" id="ref-link-section-d7940e2161">39</a></sup>. CO1 secondary structure features from <i>Bos taurus</i>
 were obtained from UniProt accession P00396. We used CUTADAPT v1.10 to 
retrieve primer-trimmed sequences using default settings (allowing up to
 a 10% mismatch in the primer sequence) from our CO1 training set in the
 same way that real raw sequence data would be processed with the 
default settings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. EMBnet J. 17, 10–12 (2011)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR40" id="ref-link-section-d7940e2168">40</a></sup>.
 These sequences were trimmed to 200 bp fragments to simulate the 
average length of an Illumina read after primer trimming and we assessed
 assignment accuracy and coverage using leave-one-out and 
cross-validation testing. For each primer, the RDP classifier was 
directly compared with the top BLAST hit method. Assignments were 
compared at the genus rank for each method. ‘Good’ assignments for the 
RDP classifier were defined according to Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="https://www.nature.com/articles/s41598-018-22505-4#Tab2">2</a>
 for 200 bp fragments at the genus rank, requiring a bootstrap 
proportion of 0.60 or greater. ‘Good’ assignments for the top BLAST hit 
method was defined by having a top BLAST hit with percent 
identity &gt;  = 95% and a top BLAST hit alignment that 
spans &gt;  = 85% of the original query sequence length (query 
coverage). We measured the proportion occurrence and rate of different 
types of taxonomic assignment outcomes as defined in Figure <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="https://www.nature.com/articles/s41598-018-22505-4#MOESM1">S5</a>.</p><p>We
 also compared how class Insecta sequence database composition and 
incorrect taxonomic assignment distribution across insect orders have 
changed over the past three years. This was done by comparing the 
proportion of incorrect assignments from class Insecta in the current 
CO1 Eukaryote v1 training set [August 2016] with the Insecta 
Genbank-Genus training set [March 2013] that both used the leave-one-out
 testing method provided by the RDP classifier tool<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Porter, T. M. et al. Rapid and accurate taxonomic classification of insect (class Insecta) cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve Bayesian classifier. Mol. Ecol. Resour. 14, 929–942 (2014)." href="https://www.nature.com/articles/s41598-018-22505-4#ref-CR9" id="ref-link-section-d7940e2182">9</a></sup>.</p><h3 class="c-article__sub-heading u-h3 c-article__sub-heading--divider" id="Sec5">Data availability</h3><p>The
 taxonomy and FASTA files used for training, the final trained sets 
ready to be used with the RDP classifier, as well as general usage 
instructions are available on GitHub <a href="https://github.com/terrimporter/CO1Classifier">https://github.com/terrimporter/CO1Classifier</a>.</p></div></div></section>
                        

                        <section aria-labelledby="Bib1"><div class="c-article-section js-article-section" id="Bib1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content js-collapsible-section" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references"><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">1.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR1">Ebach,
 M. C., Valdecasas, A. G. &amp; Wheeler, Q. D. Impediments to taxonomy 
and users of taxonomy: accessibility and impact evaluation. <i>Cladistics</i> <b>27</b>, 550–557 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1096-0031.2011.00348.x" aria-label="View reference 1">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 1 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Impediments%20to%20taxonomy%20and%20users%20of%20taxonomy%3A%20accessibility%20and%20impact%20evaluation&amp;journal=Cladistics&amp;volume=27&amp;pages=550-557&amp;publication_year=2011&amp;author=Ebach%2CMC&amp;author=Valdecasas%2CAG&amp;author=Wheeler%2CQD">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">2.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR2">Hebert, P. D. N., Cywinska, A., Ball, S. L. &amp; deWaard, J. R. Biological identifications through DNA barcodes. <i>Proc. R. Soc. B Biol. Sci.</i> <b>270</b>, 313–321 (2003).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXktVWiu7g%253D" aria-label="View reference 2 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1098%2Frspb.2002.2218" aria-label="View reference 2">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 2 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20identifications%20through%20DNA%20barcodes&amp;journal=Proc.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;volume=270&amp;pages=313-321&amp;publication_year=2003&amp;author=Hebert%2CPDN&amp;author=Cywinska%2CA&amp;author=Ball%2CSL&amp;author=deWaard%2CJR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">3.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR3">Taberlet,
 P., Coissac, E., Pompanon, F., Brochmann, C. &amp; Willerslev, E. 
Towards next-generation biodiversity assessment using DNA metabarcoding.
 <i>Mol. Ecol.</i> <b>21</b>, 2045–2050 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XptVGkt7w%253D" aria-label="View reference 3 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1365-294X.2012.05470.x" aria-label="View reference 3">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22486824" aria-label="View reference 3 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 3 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20next-generation%20biodiversity%20assessment%20using%20DNA%20metabarcoding&amp;journal=Mol.%20Ecol.&amp;volume=21&amp;pages=2045-2050&amp;publication_year=2012&amp;author=Taberlet%2CP&amp;author=Coissac%2CE&amp;author=Pompanon%2CF&amp;author=Brochmann%2CC&amp;author=Willerslev%2CE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">4.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR4">Deiner, K. <i>et al</i>. Environmental DNA metabarcoding: transforming how we survey animal and plant communities. <i>Mol. Ecol.</i> <b>26</b>, 5872–5895 (2017).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fmec.14350" aria-label="View reference 4">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28921802" aria-label="View reference 4 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 4 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Environmental%20DNA%20metabarcoding%3A%20transforming%20how%20we%20survey%20animal%20and%20plant%20communities&amp;journal=Mol.%20Ecol.&amp;volume=26&amp;pages=5872-5895&amp;publication_year=2017&amp;author=Deiner%2CK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">5.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR5">GRDI-EcoBiomics.
 Metagenomics Based Ecosystem Biomonitoring (GRDI-EcoBiomics) project, 
Government of Canada, Genomics R&amp;D Initiative, Year-End Performance 
Report for Shared Priority Projects (2016–2017). 
http://grdi-irdg.collaboration.gc.ca/eng/reports/annual/2015_2016.html 
(2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">6.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR6">Gibson, J. <i>et al</i>. Simultaneous assessment of the macrobiome and microbiome in a bulk sample of tropical arthropods through DNA metasystematics. <i>Proc. Natl. Acad. Sci.</i> <b>111</b>, 8007–8012 (2014).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014PNAS..111.8007G" aria-label="View reference 6 on ADS">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXnsFWlt7w%253D" aria-label="View reference 6 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1073%2Fpnas.1406468111" aria-label="View reference 6">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24808136" aria-label="View reference 6 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4050544" aria-label="View reference 6 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 6 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20assessment%20of%20the%20macrobiome%20and%20microbiome%20in%20a%20bulk%20sample%20of%20tropical%20arthropods%20through%20DNA%20metasystematics&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=111&amp;pages=8007-8012&amp;publication_year=2014&amp;author=Gibson%2CJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">7.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR7">Koski, L. B. &amp; Golding, G. B. The Closest BLAST Hit Is Often Not the Nearest Neighbor. <i>J. Mol. Evol.</i> <b>52</b>, 540–542 (2001).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2001JMolE..52..540K" aria-label="View reference 7 on ADS">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXksFOiurs%253D" aria-label="View reference 7 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1007%2Fs002390010184" aria-label="View reference 7">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11443357" aria-label="View reference 7 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 7 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Closest%20BLAST%20Hit%20Is%20Often%20Not%20the%20Nearest%20Neighbor&amp;journal=J.%20Mol.%20Evol.&amp;volume=52&amp;pages=540-542&amp;publication_year=2001&amp;author=Koski%2CLB&amp;author=Golding%2CGB">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">8.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR8">Virgilio, M., Backeljau, T., Nevado, B. &amp; De Meyer, M. Comparative performances of DNA barcoding across insect orders. <i>BMC Bioinformatics</i> <b>11</b>, 206 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1186%2F1471-2105-11-206" aria-label="View reference 8">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20420717" aria-label="View reference 8 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2885370" aria-label="View reference 8 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 8 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20performances%20of%20DNA%20barcoding%20across%20insect%20orders&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;publication_year=2010&amp;author=Virgilio%2CM&amp;author=Backeljau%2CT&amp;author=Nevado%2CB&amp;author=Meyer%2CM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">9.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR9">Porter, T. M. <i>et al</i>.
 Rapid and accurate taxonomic classification of insect (class Insecta) 
cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve
 Bayesian classifier. <i>Mol. Ecol. Resour.</i> <b>14</b>, 929–942 (2014).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhtlKlu7rL" aria-label="View reference 9 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4282328" aria-label="View reference 9 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 9 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20and%20accurate%20taxonomic%20classification%20of%20insect%20%28class%20Insecta%29%20cytochrome%20c%20oxidase%20subunit%201%20%28COI%29%20DNA%20barcode%20sequences%20using%20a%20na%C3%AFve%20Bayesian%20classifier&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=14&amp;pages=929-942&amp;publication_year=2014&amp;author=Porter%2CTM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">10.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR10">Wang,
 Q., Garrity, G. M., Tiedje, J. M. &amp; Cole, J. R. Naive Bayesian 
Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial
 Taxonomy. <i>Appl. Environ. Microbiol.</i> <b>73</b>, 5261–5267 (2007).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXpsleqtrc%253D" aria-label="View reference 10 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1128%2FAEM.00062-07" aria-label="View reference 10">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17586664" aria-label="View reference 10 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950982" aria-label="View reference 10 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 10 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Naive%20Bayesian%20Classifier%20for%20Rapid%20Assignment%20of%20rRNA%20Sequences%20into%20the%20New%20Bacterial%20Taxonomy&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=73&amp;pages=5261-5267&amp;publication_year=2007&amp;author=Wang%2CQ&amp;author=Garrity%2CGM&amp;author=Tiedje%2CJM&amp;author=Cole%2CJR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">11.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR11">Liu,
 K.-L., Porras-Alfaro, A., Kuske, C. R., Eichorst, S. A. &amp; Xie, G. 
Accurate, Rapid Taxonomic Classification of Fungal Large-Subunit rRNA 
Genes. <i>Appl. Environ. Microbiol.</i> <b>78</b>, 1523–1533 (2012).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XjtVSntLs%253D" aria-label="View reference 11 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1128%2FAEM.06826-11" aria-label="View reference 11">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22194300" aria-label="View reference 11 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294464" aria-label="View reference 11 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 11 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%2C%20Rapid%20Taxonomic%20Classification%20of%20Fungal%20Large-Subunit%20rRNA%20Genes&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=78&amp;pages=1523-1533&amp;publication_year=2012&amp;author=Liu%2CK-L&amp;author=Porras-Alfaro%2CA&amp;author=Kuske%2CCR&amp;author=Eichorst%2CSA&amp;author=Xie%2CG">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">12.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR12">Cole, J. R. <i>et al</i>. Ribosomal Database Project: data and tools for high throughput rRNA analysis. <i>Nucleic Acids Res.</i> <b>42</b>, D633–D642 (2014).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXoslGk" aria-label="View reference 12 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkt1244" aria-label="View reference 12">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24288368" aria-label="View reference 12 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 12 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Ribosomal%20Database%20Project%3A%20data%20and%20tools%20for%20high%20throughput%20rRNA%20analysis&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D633-D642&amp;publication_year=2014&amp;author=Cole%2CJR">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">13.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR13">Ludwig, W. ARB: a software environment for sequence data. <i>Nucleic Acids Res.</i> <b>32</b>, 1363–1371 (2004).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhvFSqu7k%253D" aria-label="View reference 13 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkh293" aria-label="View reference 13">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14985472" aria-label="View reference 13 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390282" aria-label="View reference 13 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 13 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ARB%3A%20a%20software%20environment%20for%20sequence%20data&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1363-1371&amp;publication_year=2004&amp;author=Ludwig%2CW">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">14.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR14">DeSantis, T. Z. <i>et al</i>. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB. <i>Appl. Environ. Microbiol.</i> <b>72</b>, 5069–5072 (2006).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28XnsVaqtLg%253D" aria-label="View reference 14 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1128%2FAEM.03006-05" aria-label="View reference 14">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16820507" aria-label="View reference 14 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1489311" aria-label="View reference 14 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 14 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Greengenes%2C%20a%20Chimera-Checked%2016S%20rRNA%20Gene%20Database%20and%20Workbench%20Compatible%20with%20ARB&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=5069-5072&amp;publication_year=2006&amp;author=DeSantis%2CTZ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">15.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR15">Pruesse, E. <i>et al</i>. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. <i>Nucleic Acids Res.</i> <b>35</b>, 7188–7196 (2007).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXhtFGmtg%253D%253D" aria-label="View reference 15 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2Fgkm864" aria-label="View reference 15">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17947321" aria-label="View reference 15 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2175337" aria-label="View reference 15 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 15 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=SILVA%3A%20a%20comprehensive%20online%20resource%20for%20quality%20checked%20and%20aligned%20ribosomal%20RNA%20sequence%20data%20compatible%20with%20ARB&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=7188-7196&amp;publication_year=2007&amp;author=Pruesse%2CE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">16.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR16">Abarenkov, K. <i>et al</i>. The UNITE database for molecular identification of fungi – recent updates and future perspectives. <i>New Phytol.</i> <b>186</b>, 281–285 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1111%2Fj.1469-8137.2009.03160.x" aria-label="View reference 16">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20409185" aria-label="View reference 16 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 16 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20UNITE%20database%20for%20molecular%20identification%20of%20fungi%20%E2%80%93%20recent%20updates%20and%20future%20perspectives&amp;journal=New%20Phytol.&amp;volume=186&amp;pages=281-285&amp;publication_year=2010&amp;author=Abarenkov%2CK">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">17.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR17">Ankenbrand, M. J., Keller, A., Wolf, M., Schultz, J. &amp; Förster, F. ITS2 Database V: Twice as Much. <i>Mol. Biol. Evol.</i> <b>32</b>, 3030–3032 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xht1KqtrjM" aria-label="View reference 17 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fmolbev%2Fmsv174" aria-label="View reference 17">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26248563" aria-label="View reference 17 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 17 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=ITS2%20Database%20V%3A%20Twice%20as%20Much&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=32&amp;pages=3030-3032&amp;publication_year=2015&amp;author=Ankenbrand%2CMJ&amp;author=Keller%2CA&amp;author=Wolf%2CM&amp;author=Schultz%2CJ&amp;author=F%C3%B6rster%2CF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">18.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR18">Ratnasingham, S. &amp; Hebert, P. D. BOLD: The Barcode of Life Data System (<a href="http://www.barcodinglife.org/">http://www.barcodinglife.org</a>). <i>Mol</i>. <i>Ecol</i>. <i>Note</i>s <b>7</b>, 355–364 (2007).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">19.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR19">Porter, T. M. &amp; Hajibabaei, M. Scaling up: A guide to high throughput genomic approaches for biodiversity analysis. <i>Mol</i>. <i>Ecol</i>. (In press).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">20.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR20">Smith, M. J. <i>et al</i>. AusRivAS: using macroinvertebrates to assess ecological condition of rivers in Western Australia. <i>Freshw. Biol.</i> <b>41</b>, 269–282 (1999).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1046%2Fj.1365-2427.1999.00430.x" aria-label="View reference 20">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 20 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=AusRivAS%3A%20using%20macroinvertebrates%20to%20assess%20ecological%20condition%20of%20rivers%20in%20Western%20Australia&amp;journal=Freshw.%20Biol.&amp;volume=41&amp;pages=269-282&amp;publication_year=1999&amp;author=Smith%2CMJ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">21.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR21">McDermott,
 H., Paull, T. &amp; Strachan, S. CABIN Canadian Aquatic Biomonitoring 
Network, Laboratory Methods: Processing, Taxonomy, and Quality Control 
of Benthic Macroinvertebrate Samples. (Environment Canada, 2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">22.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR22">Elbrecht,
 V. &amp; Leese, F. Validation and Development of COI Metabarcoding 
Primers for Freshwater Macroinvertebrate Bioassessment. <i>Front</i>. <i>Environ</i>. <i>Sci</i>. <b>5</b>, (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">23.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR23">Gibson, J. F. <i>et al</i>. Large-Scale Biomonitoring of Remote and Threatened Ecosystems via High-Throughput Sequencing. <i>PLOS ONE</i> <b>10</b>, e0138432 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0138432" aria-label="View reference 23">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26488407" aria-label="View reference 23 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4619546" aria-label="View reference 23 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 23 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-Scale%20Biomonitoring%20of%20Remote%20and%20Threatened%20Ecosystems%20via%20High-Throughput%20Sequencing&amp;journal=PLOS%20ONE&amp;volume=10&amp;publication_year=2015&amp;author=Gibson%2CJF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">24.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR24">Shokralla, S. <i>et al</i>.
 Environmental DNA Barcode Sequence Capture: Targeted, PCR-free Sequence
 Capture for Biodiversity Analysis from Bulk Environmental Samples. <i>bioRxiv</i> 087437 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">25.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR25">NCBI.
 BLAST Frequently Asked Questions. 
https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=FAQ#expect
 (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">26.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR26">Wilcox, T. M. <i>et al</i>. Robust Detection of Rare Species Using Environmental DNA: The Importance of Primer Specificity. <i>PLoS ONE</i> <b>8</b>, e59520 (2013).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PLoSO...859520W" aria-label="View reference 26 on ADS">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXlslCgur8%253D" aria-label="View reference 26 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0059520" aria-label="View reference 26">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23555689" aria-label="View reference 26 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3608683" aria-label="View reference 26 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 26 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20Detection%20of%20Rare%20Species%20Using%20Environmental%20DNA%3A%20The%20Importance%20of%20Primer%20Specificity&amp;journal=PLoS%20ONE&amp;volume=8&amp;publication_year=2013&amp;author=Wilcox%2CTM">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">27.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR27">Bidartondo, M. I. Preserving Accuracy in GenBank. <i>Science</i> <b>319</b>, 1616–1616 (2008).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2008Sci...319.1616B" aria-label="View reference 27 on ADS">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXktFWlu70%253D" aria-label="View reference 27 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1126%2Fscience.319.5870.1616a" aria-label="View reference 27">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18356505" aria-label="View reference 27 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 27 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Preserving%20Accuracy%20in%20GenBank&amp;journal=Science&amp;volume=319&amp;pages=1616-1616&amp;publication_year=2008&amp;author=Bidartondo%2CMI">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">28.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR28">Nilsson, R. H. <i>et al</i>. Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective. <i>PLoS ONE</i> <b>1</b>, e59 (2006).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006PLoSO...1...59N" aria-label="View reference 28 on ADS">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1371%2Fjournal.pone.0000059" aria-label="View reference 28">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17183689" aria-label="View reference 28 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1762357" aria-label="View reference 28 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 28 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Taxonomic%20Reliability%20of%20DNA%20Sequences%20in%20Public%20Sequence%20Databases%3A%20A%20Fungal%20Perspective&amp;journal=PLoS%20ONE&amp;volume=1&amp;publication_year=2006&amp;author=Nilsson%2CRH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">29.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR29">Nilsson, R. H. <i>et al</i>.
 A Comprehensive, Automatically Updated Fungal ITS Sequence Dataset for 
Reference-Based Chimera Control in Environmental Sequencing Efforts. <i>Microbes Environ.</i> <b>30</b>, 145–150 (2015).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1264%2Fjsme2.ME14121" aria-label="View reference 29">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25786896" aria-label="View reference 29 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4462924" aria-label="View reference 29 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 29 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Comprehensive%2C%20Automatically%20Updated%20Fungal%20ITS%20Sequence%20Dataset%20for%20Reference-Based%20Chimera%20Control%20in%20Environmental%20Sequencing%20Efforts&amp;journal=Microbes%20Environ.&amp;volume=30&amp;pages=145-150&amp;publication_year=2015&amp;author=Nilsson%2CRH">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">30.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR30">Armstrong, K. F. &amp; Ball, S. L. DNA barcodes for biosecurity: invasive species identification. <i>Philos. Trans. R. Soc. B Biol. Sci.</i> <b>360</b>, 1813–1823 (2005).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2MXhtlSjsrjN" aria-label="View reference 30 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1098%2Frstb.2005.1713" aria-label="View reference 30">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 30 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20barcodes%20for%20biosecurity%3A%20invasive%20species%20identification&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;volume=360&amp;pages=1813-1823&amp;publication_year=2005&amp;author=Armstrong%2CKF&amp;author=Ball%2CSL">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">31.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR31">Altschul, S. F. <i>et al</i>. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. <i>Nucleic Acids Res.</i> <b>25</b>, 17 (1997).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1093%2Fnar%2F25.17.3389" aria-label="View reference 31">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 31 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;publication_year=1997&amp;author=Altschul%2CSF">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">32.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR32">Shi, Z. Y. <i>et al</i>.
 FuzzyID2: A software package for large dataset species identification 
via barcoding and metabarcoding using Hidden Markov Models and fuzzy set
 methods. <i>Mol</i>. <i>Ecol</i>. <i>Resour</i>. (In press).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">33.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR33">Abdo,
 Z. &amp; Golding, G. B. A Step Toward Barcoding Life: A Model-Based, 
Decision-Theoretic Method to Assign Genes to Preexisting Species Groups.
 <i>Syst. Biol.</i> <b>56</b>, 44–56 (2007).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtVenurjL" aria-label="View reference 33 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150601167005" aria-label="View reference 33">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17366136" aria-label="View reference 33 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 33 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Step%20Toward%20Barcoding%20Life%3A%20A%20Model-Based%2C%20Decision-Theoretic%20Method%20to%20Assign%20Genes%20to%20Preexisting%20Species%20Groups&amp;journal=Syst.%20Biol.&amp;volume=56&amp;pages=44-56&amp;publication_year=2007&amp;author=Abdo%2CZ&amp;author=Golding%2CGB">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">34.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR34">Lou, M. &amp; Golding, G. B. Assigning sequences to species in the absence of large interspecific differences. <i>Mol. Phylogenet. Evol.</i> <b>56</b>, 187–194 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXmvVWgtrw%253D" aria-label="View reference 34 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1016%2Fj.ympev.2010.01.002" aria-label="View reference 34">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20067837" aria-label="View reference 34 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 34 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Assigning%20sequences%20to%20species%20in%20the%20absence%20of%20large%20interspecific%20differences&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=56&amp;pages=187-194&amp;publication_year=2010&amp;author=Lou%2CM&amp;author=Golding%2CGB">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">35.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR35">Zhang,
 A. B., Sikes, D. S., Muster, C. &amp; Li, S. Q. Inferring Species 
Membership Using DNA Sequences with Back-Propagation Neural Networks. <i>Syst. Biol.</i> <b>57</b>, 202–215 (2008).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXhsVymsr%252FK" aria-label="View reference 35 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1080%2F10635150802032982" aria-label="View reference 35">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18398766" aria-label="View reference 35 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 35 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20Species%20Membership%20Using%20DNA%20Sequences%20with%20Back-Propagation%20Neural%20Networks&amp;journal=Syst.%20Biol.&amp;volume=57&amp;pages=202-215&amp;publication_year=2008&amp;author=Zhang%2CAB&amp;author=Sikes%2CDS&amp;author=Muster%2CC&amp;author=Li%2CSQ">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">36.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR36">Seo, T.-K. Classification of Nucleotide Sequences Using Support Vector Machines. <i>J. Mol. Evol.</i> <b>71</b>, 250–267 (2010).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010JMolE..71..250S" aria-label="View reference 36 on ADS">ADS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXhtFOhsr7I" aria-label="View reference 36 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1007%2Fs00239-010-9380-9" aria-label="View reference 36">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20740280" aria-label="View reference 36 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 36 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Classification%20of%20Nucleotide%20Sequences%20Using%20Support%20Vector%20Machines&amp;journal=J.%20Mol.%20Evol.&amp;volume=71&amp;pages=250-267&amp;publication_year=2010&amp;author=Seo%2CT-K">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">37.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR37">Sayers, E. W. <i>Ebot</i>. http://www.ncbi.nlm.nih.gov/Class/PowerTools/eutils/course.html (2018)</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">38.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR38">Stajich, J. E. <i>et al</i>. The Bioperl toolkit: Perl modules for the life sciences. <i>Genome Res.</i> <b>12</b>, 1611–1618 (2002).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XotVKgtbY%253D" aria-label="View reference 38 on CAS">CAS</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.1101%2Fgr.361602" aria-label="View reference 38">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12368254" aria-label="View reference 38 on PubMed" rel="nofollow">PubMed</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC187536" aria-label="View reference 38 on PubMed Central" rel="nofollow">PubMed Central</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 38 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Bioperl%20toolkit%3A%20Perl%20modules%20for%20the%20life%20sciences&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1611-1618&amp;publication_year=2002&amp;author=Stajich%2CJE">
                        Google Scholar</a></li></ul></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">39.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR39">Maddison, W. P. &amp; Maddison, D. R. Mesquite version 3.10. http://mesquiteproject.org (2015).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" itemprop="citation" itemscope="itemscope" itemtype="http://schema.org/ScholarlyArticle"><span class="c-article-references__counter">40.</span><p class="c-article-references__text" itemprop="headline" id="ref-CR40">Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. <i>EMBnet J.</i> <b>17</b>, 10–12 (2011).</p><ul class="c-article-references__links u-hide-print"><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" href="https://doi.org/10.14806%2Fej.17.1.200" aria-label="View reference 40">Article</a></li><li><a data-track="click" data-track-action="outbound reference" data-track-category="article body" data-track-label="link" aria-label="Search for reference 40 on Google Scholar" href="http://scholar.google.com/scholar_lookup?&amp;title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;journal=EMBnet%20J.&amp;volume=17&amp;pages=10-12&amp;publication_year=2011&amp;author=Martin%2CM">
                        Google Scholar</a></li></ul></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-category="article body" data-track-label="link" href="https://www.nature.com/articles/s41598-018-22505-4-references.ris">
                    Download references
                </a></p></div></div></div></section><section aria-labelledby="Ack1"><div class="c-article-section js-article-section" id="Ack1-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content js-collapsible-section" id="Ack1-content"><p>We
 would like to acknowledge funding for T. Porter from the Government of 
Canada through the Genomics Research Development Initiative as well as 
office space and computational resources provided by the Hajibabaei lab 
at the Centre for Biodiversity Genomics, University of Guelph.</p></div></div></section><section aria-labelledby="author-information"><div class="c-article-section js-article-section" id="author-information-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content js-collapsible-section" id="author-information-content"><h3 class="c-article-author-information__subtitle u-h3" id="affiliations">Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><h4 class="c-article-author-affiliation__address u-h3">The
 Centre for Biodiversity Genomics &amp; Department of Integrative 
Biology, University of Guelph, 50 Stone Road East, Guelph, ON, N1G 2W1, 
Canada</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Teresita M. Porter</li><li class="c-article-author-affiliation__authors-item">&nbsp;&amp;&nbsp;Mehrdad Hajibabaei</li></ul></li><li id="Aff2"><h4 class="c-article-author-affiliation__address u-h3">Great Lakes Forestry Centre, Natural Resources Canada, 1219 Queen Street East, Sault Ste. Marie, ON, P6A 2E5, Canada</h4><ul class="c-article-author-affiliation__authors-list"><li class="c-article-author-affiliation__authors-item">Teresita M. Porter</li></ul></li></ol><div class="js-hide u-hide-print" data-test="author-info"><h3 class="c-article-author-information__subtitle u-h3">Authors</h3><ol class="c-article-author-authors-search"><li id="auth-1"><h3 class="c-article-author-authors-search__title u-h3 js-search-name">Search for Teresita M. Porter in:</h3><ul class="c-article-author-authors-search__list"><li class="c-article-author-authors-search__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Teresita%20M.+Porter">PubMed</a><span class="bullet"> • </span></li><li class="c-article-author-authors-search__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Teresita%20M.+Porter%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">
                        Google Scholar
                    </a></li></ul></li><li id="auth-2"><h3 class="c-article-author-authors-search__title u-h3 js-search-name">Search for Mehrdad Hajibabaei in:</h3><ul class="c-article-author-authors-search__list"><li class="c-article-author-authors-search__item"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mehrdad+Hajibabaei">PubMed</a><span class="bullet"> • </span></li><li class="c-article-author-authors-search__item"><a href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mehrdad+Hajibabaei%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en">
                        Google Scholar
                    </a></li></ul></li></ol></div><h3 class="c-article-author-information__subtitle u-h3" id="contributions">Contributions</h3><p>T. Porter conceived of the manuscript idea and conducted the analyses. T. Porter and M. Hajibabaei wrote the manuscript.</p><h3 class="c-article-author-information__subtitle u-h3" id="corresponding-author">Corresponding author</h3><p>Correspondence to
                <a id="corresp-c1" rel="nofollow" href="https://www.nature.com/articles/s41598-018-22505-4/email/correspondent/c1/new">Teresita M. Porter</a>.</p></div></div></section><section aria-labelledby="ethics"><div class="c-article-section js-article-section" id="ethics-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content js-collapsible-section" id="ethics-content">
              
                <h3 class="c-article__sub-heading u-h3">Competing Interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section aria-labelledby="additional-information"><div class="c-article-section js-article-section" id="additional-information-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content js-collapsible-section" id="additional-information-content"><p><b>Publisher's note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section aria-labelledby="Sec6"><div class="c-article-section js-article-section" id="Sec6-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="Sec6">Electronic supplementary material</h2><div class="c-article-section__content js-collapsible-section" id="Sec6-content">
                  
                <div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-category="article body" data-track-action="view supplementary info" data-track-label="link" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_MOESM1_ESM.docx" data-supp-info-image="">Supplementary Information</a></h3></div></div></div></section><section aria-labelledby="rightslink"><div class="c-article-section js-article-section" id="rightslink-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content js-collapsible-section" id="rightslink-content"><div class="c-article-license">
                <p><b>Open Access</b>  This article is licensed under a 
Creative Commons Attribution 4.0 International License, which permits 
use, sharing, adaptation, distribution and reproduction in any medium or
 format, as long as you give appropriate credit to the original 
author(s) and the source, provide a link to the Creative Commons 
license, and indicate if changes were made. The images or other third 
party material in this article are included in the article’s Creative 
Commons license, unless indicated otherwise in a credit line to the 
material. If material is not included in the article’s Creative Commons 
license and your intended use is not permitted by statutory regulation 
or exceeds the permitted use, you will need to obtain permission 
directly from the copyright holder. To view a copy of this license, 
visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license" itemprop="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              </div><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-category="article body" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?imprint=Nature&amp;oa=CC%20BY&amp;title=Automated%20high%20throughput%20animal%20CO1%20metabarcode%20classification&amp;author=Teresita%20M.%20Porter%20et%20al&amp;contentID=10.1038%2Fs41598-018-22505-4&amp;publication=Scientific%20Reports&amp;publicationDate=2018-03-09&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and Permissions</a></p></div></div></section><section aria-labelledby="article-info"><div class="c-article-section js-article-section" id="article-info-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content js-collapsible-section" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column"><a data-crossmark="10.1038/s41598-018-22505-4" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41598-018-22505-4" data-track="click" data-track-action="Click Crossmark" data-track-category="article body" data-track-label="link"><img alt="Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>" width="57" height="81"></a></div><div class="c-bibliographic-information__column"><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><h4 class="u-h4">Received</h4><p class="c-bibliographic-information__value"><time datetime="2017-11-16">16 November 2017</time></p></li><li class="c-bibliographic-information__list-item"><h4 class="u-h4">Accepted</h4><p class="c-bibliographic-information__value"><time datetime="2018-02-01">01 February 2018</time></p></li><li class="c-bibliographic-information__list-item"><h4 class="u-h4">Published</h4><p class="c-bibliographic-information__value"><time datetime="2018-03-09">09 March 2018</time></p></li><li class="c-bibliographic-information__list-item"><h4 class="u-h4"><abbr title="Digital Object Identifier">DOI</abbr></h4><p class="c-bibliographic-information__value"><a href="https://doi.org/10.1038/s41598-018-22505-4" data-track="click" data-track-action="view doi" data-track-category="article body" data-track-label="link" itemprop="sameAs">https://doi.org/10.1038/s41598-018-22505-4</a></p></li></ul><div data-component="share-box"><div class="c-article-share-box"><h3 class="c-article-share-box__title">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" id="get-share-url" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="get shareable link">Get shareable link</button>			<div class="js-no-share-url-container u-display-none" aria-hidden="true"><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div> 			<div class="js-share-url-container u-display-none" aria-hidden="true"><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="select share url"> 		</p><button class="js-copy-share-url c-article-share-box__button--link-like" id="copy-share-url" data-track="click" data-track-category="article body" data-track-label="button" data-track-action="copy share url">Copy to clipboard</button><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">Provided by the Springer Nature SharedIt content-sharing initiative</p></div> 		</div></div><div data-component="article-info-list"><h3 class="c-article__sub-heading">Subjects</h3><ul class="c-article-subject-list">
                <li class="c-article-subject-list__subject"><a href="https://www.nature.com/subjects/biodiversity" data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" itemprop="about">Biodiversity</a></li><li class="c-article-subject-list__subject"><a href="https://www.nature.com/subjects/classification-and-taxonomy" data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" itemprop="about">Classification and taxonomy</a></li><li class="c-article-subject-list__subject"><a href="https://www.nature.com/subjects/data-processing" data-track="click" data-track-action="view subject" data-track-category="article body" data-track-label="link" itemprop="about">Data processing</a></li>
            </ul></div></div></div></div></div></section>

                        
    <section aria-labelledby="further-reading">
      <div class="c-article-section js-article-section" id="further-reading-section">
        <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">Further reading</h2>
        <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
        <ul class="c-article-further-reading__list" id="further-reading-list">
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-1">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-category="article body" data-track-label="link:Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance" href="https://doi.org/10.1093/gigascience/giz092" id="title-link-1">
                  Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed" id="authors-1">
                  <li><span class="js-separator"></span><span>Alexander M Piper</span></li><li><span class="js-separator">, </span><span>Jana Batovska</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Noel O I Cogan</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>John Weiss</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>John Paul Cunningham</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Brendan C Rodoni</span></li><li class="c-author-list__show-more "><a href="javascript:;" class="js-etal js-no-scroll" title="Show all 7 authors" aria-label="Show all 7 authors for Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance">[…]</a></li><li><span class="js-separator u-js-hide">&nbsp;&amp;&nbsp;</span><span>Mark J Blacket</span></li><li class="c-author-list__show-less u-js-hide"><a href="javascript:;" class="js-etal js-no-scroll" aria-label="Show fewer authors for Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance">- Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-1">
                <i>GigaScience</i>
                (2019)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-2">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-category="article body" data-track-label="link:Global molecular variation of Paramecium jenningsi complex (Ciliophora, Protista): a starting point for further, detailed biogeography surveys" href="https://doi.org/10.1080/14772000.2019.1643424" id="title-link-2">
                  Global molecular variation of Paramecium jenningsi 
complex (Ciliophora, Protista): a starting point for further, detailed 
biogeography surveys
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed" id="authors-2">
                  <li><span class="js-separator"></span><span>Ewa PrzyboŚ</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Sebastian Tarcz</span></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-2">
                <i>Systematics and Biodiversity</i>
                (2019)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-3">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-category="article body" data-track-label="link:The taxonomic position of Asian Holopedium (Crustacea: Cladocera) confirmed by morphological and genetic analyses" href="https://doi.org/10.1007/s10201-019-00585-z" id="title-link-3">
                  The taxonomic position of Asian Holopedium (Crustacea:
 Cladocera) confirmed by morphological and genetic analyses
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed" id="authors-3">
                  <li><span class="js-separator"></span><span>Ayaka Yamamoto</span></li><li><span class="js-separator">, </span><span>Wataru Makino</span></li><li><span class="js-separator">&nbsp;&amp;&nbsp;</span><span>Jotaro Urabe</span></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-3">
                <i>Limnology</i>
                (2019)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-4">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-category="article body" data-track-label="link:Assessment of BOLD and GenBank – Their accuracy and reliability for the identification of biological materials" href="https://doi.org/10.1371/journal.pone.0217084" id="title-link-4">
                  Assessment of BOLD and GenBank – Their accuracy and reliability for the identification of biological materials
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed" id="authors-4">
                  <li><span class="js-separator"></span><span>Kelly A. Meiklejohn</span></li><li><span class="js-separator">, </span><span>Natalie Damaso</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>James M. Robertson</span></li><li class="c-author-list__show-more "><a href="javascript:;" class="js-etal js-no-scroll" title="Show all 4 authors" aria-label="Show all 4 authors for Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance">[…]</a></li><li><span class="js-separator u-js-hide">&nbsp;&amp;&nbsp;</span><span>Sebastian D. Fugmann</span></li><li class="c-author-list__show-less u-js-hide"><a href="javascript:;" class="js-etal js-no-scroll" aria-label="Show fewer authors for Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance">- Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-4">
                <i>PLOS ONE</i>
                (2019)
              </p>
            </li>
          
            <li class="c-article-further-reading__item js-ref-item" id="further-reading-item-5">
              <h3 class="c-article-further-reading__title">
                <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-category="article body" data-track-label="link:Beyond the visual: using metabarcoding to characterize the hidden reef cryptobiome" href="https://doi.org/10.1098/rspb.2018.2697" id="title-link-5">
                  Beyond the visual: using metabarcoding to characterize the hidden reef cryptobiome
                </a>
              </h3>
              
                <ul data-test="author-list" class="c-article-further-reading__authors-list js-list-authors-3 js-etal-collapsed" id="authors-5">
                  <li><span class="js-separator"></span><span>Susana Carvalho</span></li><li><span class="js-separator">, </span><span>Eva Aylagas</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Rodrigo Villalobos</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Yasser Kattan</span></li><li class="js-author-etal js-smaller-author-etal"><span class="js-separator">, </span><span>Michael Berumen</span></li><li class="c-author-list__show-more "><a href="javascript:;" class="js-etal js-no-scroll" title="Show all 6 authors" aria-label="Show all 6 authors for Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance">[…]</a></li><li><span class="js-separator u-js-hide">&nbsp;&amp;&nbsp;</span><span>John K. Pearman</span></li><li class="c-author-list__show-less u-js-hide"><a href="javascript:;" class="js-etal js-no-scroll" aria-label="Show fewer authors for Prospects and challenges of implementing DNA metabarcoding for high-throughput insect surveillance">- Show fewer authors</a></li></ul>
              
              <p class="c-article-further-reading__journal-title" id="journal-name-5">
                <i>Proceedings of the Royal Society B: Biological Sciences</i>
                (2019)
              </p>
            </li>
          
        </ul>
      </div>
    </div>
    </section>
  

                        
                            <section aria-labelledby="article-comments"><div class="c-article-section js-article-section" id="article-comments-section"><h2 class="c-article-section__title u-h2 js-section-title js-c-reading-companion-sections-item" id="article-comments">Comments</h2><div class="c-article-section__content js-collapsible-section" id="article-comments-content"><p>By submitting a comment you agree to abide by our <a href="https://www.nature.com/info/tandc.html">Terms</a> and <a href="https://www.nature.com/info/community-guidelines.html">Community Guidelines</a>. If you find something abusive or that does not comply with our terms or guidelines please flag it as inappropriate.</p><div id="inject-comments">
                                
    <div id="disqus_thread" class="pb20" data-object="disqus" data-disqus-s3="e30= 9d7698d6116a2b9ef1fbbbac3394604f2961f360 1568125767" data-disqus-key="ZZDdHrSs8UQRH0LAFbG4KmcMfQdvUwYDEyYczGZhznVSuj9QDfXFgWxFA1tz53jo" data-disqus-sso-name="Nature" data-disqus-sso-button="//www.nature.com/static/images/buttons/disqus-sso-login-button.png" data-disqus-sso-icon="//www.nature.com/static/images/favicons/nature/favicon.ico" data-disqus-sso-url="https://idp.nature.com/authorize/natureuser?client_id=grover&amp;redirect_uri=http%3A%2F%2Fwww.nature.com%2Farticles%2Fs41598-018-22505-4%2Fdisqus" data-disqus-sso-logout="https://www.nature.com/foxtrot/svc/logout?target=https%3A%2F%2Fwww.nature.com%2Farticles%2Fs41598-018-22505-4" data-disqus-shortname="srep" data-disqus-identifier="10.1038/s41598-018-22505-4" data-disqus-url="https://www.nature.com/articles/s41598-018-22505-4">
    </div>

                        </div></div></div></section>
                            
                        

                        <iframe style="width: 100%; height: 0px; bottom: 0px; right: 0px; z-index: 100000; padding: 0px; margin-top: 28px;" scrolling="no" src="https://recommended.springernature.com/v2344/generated/job-client.html?source=https://www.nature.com" name="job-client" id="job-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe>
                    </div>
            </div>

            <div class="c-article-extras u-hide-print" role="complementary" data-container-type="reading-companion" data-track-component="reading companion">
                
                    
    
        
        <div class="c-pdf-download u-clear-both">
            <a href="https://www.nature.com/articles/s41598-018-22505-4.pdf" class="c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-category="article body" data-track-label="link">
                <span>Download PDF</span>
            </a>
        </div>
        
    

                
                
    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <div class="c-article-metrics-bar">
                <h3 class="u-visually-hidden">Article metrics</h3>
                
                    <div class="c-article-metrics-bar__item">
                        <h4 class="c-article-metrics-bar__label">Accesses</h4>
                        <p class="c-article-metrics-bar__count">1038</p>
                    </div>
                
                
                    <div class="c-article-metrics-bar__item">
                        <h4 class="c-article-metrics-bar__label">Citations</h4>
                        <p class="c-article-metrics-bar__count">10</p>
                    </div>
                
                
                    
                        <div class="c-article-metrics-bar__item">
                            <h4 class="c-article-metrics-bar__label">Altmetric</h4>
                            <p class="c-article-metrics-bar__count">15</p>
                        </div>
                    
                    
                        <div class="c-article-metrics-bar__item">
                            <h4 class="u-visually-hidden">More details</h4>
                            <p class="c-article-metrics-bar__details"><a href="https://www.nature.com/articles/s41598-018-22505-4/metrics" data-track="click" data-track-action="view metrics" data-track-category="article body" data-track-label="link" rel="nofollow">Metrics</a></p>
                        </div>
                    
                
            </div>
        </div>
    


                
                    
                

                

                <div class="c-reading-companion">
                    <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" style="width: 308px;"><ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab">References</button></li></ul>
                        <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" tabindex="0"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Introduction">Introduction</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec3" class="c-reading-companion__section-item"><a href="#Sec3" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec4" class="c-reading-companion__section-item"><a href="#Sec4" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec6" class="c-reading-companion__section-item"><a href="#Sec6" data-track="click" data-track-action="section anchor" data-track-label="link:Electronic supplementary material">Electronic supplementary material</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:Further reading">Further reading</a></li><li id="rc-sec-article-comments" class="c-reading-companion__section-item"><a href="#article-comments" data-track="click" data-track-action="section anchor" data-track-label="link:Comments">Comments</a></li></ul></div>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-gpt-unitpath="/285/scientific_reports/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41598-018-22505-4;doi=10.1038/s41598-018-22505-4;subjmeta=114,1314,1386,158,631,670,704;kwrd=Biodiversity,Classification and taxonomy,Data processing">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/scientific_reports/article&amp;sz=300x250&amp;c=-854750673&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41598-018-22505-4%26doi%3D10.1038/s41598-018-22505-4%26subjmeta%3D114,1314,1386,158,631,670,704%26kwrd%3DBiodiversity,Classification and taxonomy,Data processing">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/scientific_reports/article&amp;sz=300x250&amp;c=-854750673&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41598-018-22505-4%26doi%3D10.1038/s41598-018-22505-4%26subjmeta%3D114,1314,1386,158,631,670,704%26kwrd%3DBiodiversity,Classification and taxonomy,Data processing"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                        <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig1">Figure 1</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig1_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig1_HTML.jpg" alt="figure1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41598-018-22505-4/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig2">Figure 2</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig2_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig2_HTML.jpg" alt="figure2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41598-018-22505-4/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig3">Figure 3</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig3_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig3_HTML.jpg" alt="figure3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41598-018-22505-4/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title" id="rc-Fig4">Figure 4</b></figcaption><picture><source type="image/webp" data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig4_HTML.jpg?as=webp"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41598-018-22505-4/MediaObjects/41598_2018_22505_Fig4_HTML.jpg" alt="figure4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41598-018-22505-4/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link">Full size image</a></p></figure></li></ul></div></div>
                        <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR1">Ebach,
 M. C., Valdecasas, A. G. &amp; Wheeler, Q. D. Impediments to taxonomy 
and users of taxonomy: accessibility and impact evaluation. <i>Cladistics</i> <b>27</b>, 550–557 (2011).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1111%2Fj.1096-0031.2011.00348.x" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Impediments%20to%20taxonomy%20and%20users%20of%20taxonomy%3A%20accessibility%20and%20impact%20evaluation&amp;journal=Cladistics&amp;volume=27&amp;pages=550-557&amp;publication_year=2011&amp;author=Ebach%2CMC&amp;author=Valdecasas%2CAG&amp;author=Wheeler%2CQD" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR2">Hebert, P. D. N., Cywinska, A., Ball, S. L. &amp; deWaard, J. R. Biological identifications through DNA barcodes. <i>Proc. R. Soc. B Biol. Sci.</i> <b>270</b>, 313–321 (2003).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3sXktVWiu7g%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1098%2Frspb.2002.2218" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20identifications%20through%20DNA%20barcodes&amp;journal=Proc.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;volume=270&amp;pages=313-321&amp;publication_year=2003&amp;author=Hebert%2CPDN&amp;author=Cywinska%2CA&amp;author=Ball%2CSL&amp;author=deWaard%2CJR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR3">Taberlet,
 P., Coissac, E., Pompanon, F., Brochmann, C. &amp; Willerslev, E. 
Towards next-generation biodiversity assessment using DNA metabarcoding.
 <i>Mol. Ecol.</i> <b>21</b>, 2045–2050 (2012).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XptVGkt7w%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1111%2Fj.1365-294X.2012.05470.x" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22486824" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20next-generation%20biodiversity%20assessment%20using%20DNA%20metabarcoding&amp;journal=Mol.%20Ecol.&amp;volume=21&amp;pages=2045-2050&amp;publication_year=2012&amp;author=Taberlet%2CP&amp;author=Coissac%2CE&amp;author=Pompanon%2CF&amp;author=Brochmann%2CC&amp;author=Willerslev%2CE" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR4">Deiner, K. <i>et al</i>. Environmental DNA metabarcoding: transforming how we survey animal and plant communities. <i>Mol. Ecol.</i> <b>26</b>, 5872–5895 (2017).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1111%2Fmec.14350" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28921802" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Environmental%20DNA%20metabarcoding%3A%20transforming%20how%20we%20survey%20animal%20and%20plant%20communities&amp;journal=Mol.%20Ecol.&amp;volume=26&amp;pages=5872-5895&amp;publication_year=2017&amp;author=Deiner%2CK" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR5">GRDI-EcoBiomics.
 Metagenomics Based Ecosystem Biomonitoring (GRDI-EcoBiomics) project, 
Government of Canada, Genomics R&amp;D Initiative, Year-End Performance 
Report for Shared Priority Projects (2016–2017). 
http://grdi-irdg.collaboration.gc.ca/eng/reports/annual/2015_2016.html 
(2016).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR6">Gibson, J. <i>et al</i>. Simultaneous assessment of the macrobiome and microbiome in a bulk sample of tropical arthropods through DNA metasystematics. <i>Proc. Natl. Acad. Sci.</i> <b>111</b>, 8007–8012 (2014).</p><ul class="c-reading-companion__reference-links"><li><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2014PNAS..111.8007G" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a></li><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXnsFWlt7w%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1073%2Fpnas.1406468111" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24808136" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4050544" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20assessment%20of%20the%20macrobiome%20and%20microbiome%20in%20a%20bulk%20sample%20of%20tropical%20arthropods%20through%20DNA%20metasystematics&amp;journal=Proc.%20Natl.%20Acad.%20Sci.&amp;volume=111&amp;pages=8007-8012&amp;publication_year=2014&amp;author=Gibson%2CJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR7">Koski, L. B. &amp; Golding, G. B. The Closest BLAST Hit Is Often Not the Nearest Neighbor. <i>J. Mol. Evol.</i> <b>52</b>, 540–542 (2001).</p><ul class="c-reading-companion__reference-links"><li><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2001JMolE..52..540K" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a></li><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD3MXksFOiurs%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1007%2Fs002390010184" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11443357" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Closest%20BLAST%20Hit%20Is%20Often%20Not%20the%20Nearest%20Neighbor&amp;journal=J.%20Mol.%20Evol.&amp;volume=52&amp;pages=540-542&amp;publication_year=2001&amp;author=Koski%2CLB&amp;author=Golding%2CGB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR8">Virgilio, M., Backeljau, T., Nevado, B. &amp; De Meyer, M. Comparative performances of DNA barcoding across insect orders. <i>BMC Bioinformatics</i> <b>11</b>, 206 (2010).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1186%2F1471-2105-11-206" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20420717" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2885370" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Comparative%20performances%20of%20DNA%20barcoding%20across%20insect%20orders&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;publication_year=2010&amp;author=Virgilio%2CM&amp;author=Backeljau%2CT&amp;author=Nevado%2CB&amp;author=Meyer%2CM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR9">Porter, T. M. <i>et al</i>.
 Rapid and accurate taxonomic classification of insect (class Insecta) 
cytochrome c oxidase subunit 1 (COI) DNA barcode sequences using a naïve
 Bayesian classifier. <i>Mol. Ecol. Resour.</i> <b>14</b>, 929–942 (2014).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXhtlKlu7rL" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4282328" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20and%20accurate%20taxonomic%20classification%20of%20insect%20%28class%20Insecta%29%20cytochrome%20c%20oxidase%20subunit%201%20%28COI%29%20DNA%20barcode%20sequences%20using%20a%20na%C3%AFve%20Bayesian%20classifier&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=14&amp;pages=929-942&amp;publication_year=2014&amp;author=Porter%2CTM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR10">Wang,
 Q., Garrity, G. M., Tiedje, J. M. &amp; Cole, J. R. Naive Bayesian 
Classifier for Rapid Assignment of rRNA Sequences into the New Bacterial
 Taxonomy. <i>Appl. Environ. Microbiol.</i> <b>73</b>, 5261–5267 (2007).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXpsleqtrc%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1128%2FAEM.00062-07" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17586664" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950982" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Naive%20Bayesian%20Classifier%20for%20Rapid%20Assignment%20of%20rRNA%20Sequences%20into%20the%20New%20Bacterial%20Taxonomy&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=73&amp;pages=5261-5267&amp;publication_year=2007&amp;author=Wang%2CQ&amp;author=Garrity%2CGM&amp;author=Tiedje%2CJM&amp;author=Cole%2CJR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR11">Liu,
 K.-L., Porras-Alfaro, A., Kuske, C. R., Eichorst, S. A. &amp; Xie, G. 
Accurate, Rapid Taxonomic Classification of Fungal Large-Subunit rRNA 
Genes. <i>Appl. Environ. Microbiol.</i> <b>78</b>, 1523–1533 (2012).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC38XjtVSntLs%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1128%2FAEM.06826-11" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22194300" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3294464" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%2C%20Rapid%20Taxonomic%20Classification%20of%20Fungal%20Large-Subunit%20rRNA%20Genes&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=78&amp;pages=1523-1533&amp;publication_year=2012&amp;author=Liu%2CK-L&amp;author=Porras-Alfaro%2CA&amp;author=Kuske%2CCR&amp;author=Eichorst%2CSA&amp;author=Xie%2CG" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR12">Cole, J. R. <i>et al</i>. Ribosomal Database Project: data and tools for high throughput rRNA analysis. <i>Nucleic Acids Res.</i> <b>42</b>, D633–D642 (2014).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC2cXoslGk" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1093%2Fnar%2Fgkt1244" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24288368" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Ribosomal%20Database%20Project%3A%20data%20and%20tools%20for%20high%20throughput%20rRNA%20analysis&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D633-D642&amp;publication_year=2014&amp;author=Cole%2CJR" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR13">Ludwig, W. ARB: a software environment for sequence data. <i>Nucleic Acids Res.</i> <b>32</b>, 1363–1371 (2004).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2cXhvFSqu7k%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1093%2Fnar%2Fgkh293" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14985472" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC390282" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=ARB%3A%20a%20software%20environment%20for%20sequence%20data&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1363-1371&amp;publication_year=2004&amp;author=Ludwig%2CW" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR14">DeSantis, T. Z. <i>et al</i>. Greengenes, a Chimera-Checked 16S rRNA Gene Database and Workbench Compatible with ARB. <i>Appl. Environ. Microbiol.</i> <b>72</b>, 5069–5072 (2006).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD28XnsVaqtLg%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1128%2FAEM.03006-05" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16820507" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1489311" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Greengenes%2C%20a%20Chimera-Checked%2016S%20rRNA%20Gene%20Database%20and%20Workbench%20Compatible%20with%20ARB&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=5069-5072&amp;publication_year=2006&amp;author=DeSantis%2CTZ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR15">Pruesse, E. <i>et al</i>. SILVA: a comprehensive online resource for quality checked and aligned ribosomal RNA sequence data compatible with ARB. <i>Nucleic Acids Res.</i> <b>35</b>, 7188–7196 (2007).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXhtFGmtg%253D%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1093%2Fnar%2Fgkm864" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17947321" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2175337" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=SILVA%3A%20a%20comprehensive%20online%20resource%20for%20quality%20checked%20and%20aligned%20ribosomal%20RNA%20sequence%20data%20compatible%20with%20ARB&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=7188-7196&amp;publication_year=2007&amp;author=Pruesse%2CE" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR16">Abarenkov, K. <i>et al</i>. The UNITE database for molecular identification of fungi – recent updates and future perspectives. <i>New Phytol.</i> <b>186</b>, 281–285 (2010).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1111%2Fj.1469-8137.2009.03160.x" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20409185" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20UNITE%20database%20for%20molecular%20identification%20of%20fungi%20%E2%80%93%20recent%20updates%20and%20future%20perspectives&amp;journal=New%20Phytol.&amp;volume=186&amp;pages=281-285&amp;publication_year=2010&amp;author=Abarenkov%2CK" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR17">Ankenbrand, M. J., Keller, A., Wolf, M., Schultz, J. &amp; Förster, F. ITS2 Database V: Twice as Much. <i>Mol. Biol. Evol.</i> <b>32</b>, 3030–3032 (2015).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC28Xht1KqtrjM" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1093%2Fmolbev%2Fmsv174" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26248563" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=ITS2%20Database%20V%3A%20Twice%20as%20Much&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=32&amp;pages=3030-3032&amp;publication_year=2015&amp;author=Ankenbrand%2CMJ&amp;author=Keller%2CA&amp;author=Wolf%2CM&amp;author=Schultz%2CJ&amp;author=F%C3%B6rster%2CF" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR18">Ratnasingham, S. &amp; Hebert, P. D. BOLD: The Barcode of Life Data System (<a href="http://www.barcodinglife.org/">http://www.barcodinglife.org</a>). <i>Mol</i>. <i>Ecol</i>. <i>Note</i>s <b>7</b>, 355–364 (2007).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR19">Porter, T. M. &amp; Hajibabaei, M. Scaling up: A guide to high throughput genomic approaches for biodiversity analysis. <i>Mol</i>. <i>Ecol</i>. (In press).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR20">Smith, M. J. <i>et al</i>. AusRivAS: using macroinvertebrates to assess ecological condition of rivers in Western Australia. <i>Freshw. Biol.</i> <b>41</b>, 269–282 (1999).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1046%2Fj.1365-2427.1999.00430.x" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=AusRivAS%3A%20using%20macroinvertebrates%20to%20assess%20ecological%20condition%20of%20rivers%20in%20Western%20Australia&amp;journal=Freshw.%20Biol.&amp;volume=41&amp;pages=269-282&amp;publication_year=1999&amp;author=Smith%2CMJ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR21">McDermott,
 H., Paull, T. &amp; Strachan, S. CABIN Canadian Aquatic Biomonitoring 
Network, Laboratory Methods: Processing, Taxonomy, and Quality Control 
of Benthic Macroinvertebrate Samples. (Environment Canada, 2014).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR22">Elbrecht,
 V. &amp; Leese, F. Validation and Development of COI Metabarcoding 
Primers for Freshwater Macroinvertebrate Bioassessment. <i>Front</i>. <i>Environ</i>. <i>Sci</i>. <b>5</b>, (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR23">Gibson, J. F. <i>et al</i>. Large-Scale Biomonitoring of Remote and Threatened Ecosystems via High-Throughput Sequencing. <i>PLOS ONE</i> <b>10</b>, e0138432 (2015).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1371%2Fjournal.pone.0138432" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26488407" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4619546" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Large-Scale%20Biomonitoring%20of%20Remote%20and%20Threatened%20Ecosystems%20via%20High-Throughput%20Sequencing&amp;journal=PLOS%20ONE&amp;volume=10&amp;publication_year=2015&amp;author=Gibson%2CJF" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR24">Shokralla, S. <i>et al</i>.
 Environmental DNA Barcode Sequence Capture: Targeted, PCR-free Sequence
 Capture for Biodiversity Analysis from Bulk Environmental Samples. <i>bioRxiv</i> 087437 (2016).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR25">NCBI.
 BLAST Frequently Asked Questions. 
https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=FAQ#expect
 (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR26">Wilcox, T. M. <i>et al</i>. Robust Detection of Rare Species Using Environmental DNA: The Importance of Primer Specificity. <i>PLoS ONE</i> <b>8</b>, e59520 (2013).</p><ul class="c-reading-companion__reference-links"><li><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2013PLoSO...859520W" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a></li><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3sXlslCgur8%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1371%2Fjournal.pone.0059520" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23555689" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3608683" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20Detection%20of%20Rare%20Species%20Using%20Environmental%20DNA%3A%20The%20Importance%20of%20Primer%20Specificity&amp;journal=PLoS%20ONE&amp;volume=8&amp;publication_year=2013&amp;author=Wilcox%2CTM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR27">Bidartondo, M. I. Preserving Accuracy in GenBank. <i>Science</i> <b>319</b>, 1616–1616 (2008).</p><ul class="c-reading-companion__reference-links"><li><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2008Sci...319.1616B" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a></li><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXktFWlu70%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1126%2Fscience.319.5870.1616a" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18356505" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Preserving%20Accuracy%20in%20GenBank&amp;journal=Science&amp;volume=319&amp;pages=1616-1616&amp;publication_year=2008&amp;author=Bidartondo%2CMI" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR28">Nilsson, R. H. <i>et al</i>. Taxonomic Reliability of DNA Sequences in Public Sequence Databases: A Fungal Perspective. <i>PLoS ONE</i> <b>1</b>, e59 (2006).</p><ul class="c-reading-companion__reference-links"><li><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2006PLoSO...1...59N" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a></li><li><a href="https://doi.org/10.1371%2Fjournal.pone.0000059" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17183689" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1762357" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Taxonomic%20Reliability%20of%20DNA%20Sequences%20in%20Public%20Sequence%20Databases%3A%20A%20Fungal%20Perspective&amp;journal=PLoS%20ONE&amp;volume=1&amp;publication_year=2006&amp;author=Nilsson%2CRH" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR29">Nilsson, R. H. <i>et al</i>.
 A Comprehensive, Automatically Updated Fungal ITS Sequence Dataset for 
Reference-Based Chimera Control in Environmental Sequencing Efforts. <i>Microbes Environ.</i> <b>30</b>, 145–150 (2015).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1264%2Fjsme2.ME14121" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25786896" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4462924" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Comprehensive%2C%20Automatically%20Updated%20Fungal%20ITS%20Sequence%20Dataset%20for%20Reference-Based%20Chimera%20Control%20in%20Environmental%20Sequencing%20Efforts&amp;journal=Microbes%20Environ.&amp;volume=30&amp;pages=145-150&amp;publication_year=2015&amp;author=Nilsson%2CRH" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR30">Armstrong, K. F. &amp; Ball, S. L. DNA barcodes for biosecurity: invasive species identification. <i>Philos. Trans. R. Soc. B Biol. Sci.</i> <b>360</b>, 1813–1823 (2005).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2MXhtlSjsrjN" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1098%2Frstb.2005.1713" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=DNA%20barcodes%20for%20biosecurity%3A%20invasive%20species%20identification&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;volume=360&amp;pages=1813-1823&amp;publication_year=2005&amp;author=Armstrong%2CKF&amp;author=Ball%2CSL" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR31">Altschul, S. F. <i>et al</i>. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. <i>Nucleic Acids Res.</i> <b>25</b>, 17 (1997).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.1093%2Fnar%2F25.17.3389" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;publication_year=1997&amp;author=Altschul%2CSF" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR32">Shi, Z. Y. <i>et al</i>.
 FuzzyID2: A software package for large dataset species identification 
via barcoding and metabarcoding using Hidden Markov Models and fuzzy set
 methods. <i>Mol</i>. <i>Ecol</i>. <i>Resour</i>. (In press).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR33">Abdo,
 Z. &amp; Golding, G. B. A Step Toward Barcoding Life: A Model-Based, 
Decision-Theoretic Method to Assign Genes to Preexisting Species Groups.
 <i>Syst. Biol.</i> <b>56</b>, 44–56 (2007).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD2sXhtVenurjL" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1080%2F10635150601167005" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17366136" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Step%20Toward%20Barcoding%20Life%3A%20A%20Model-Based%2C%20Decision-Theoretic%20Method%20to%20Assign%20Genes%20to%20Preexisting%20Species%20Groups&amp;journal=Syst.%20Biol.&amp;volume=56&amp;pages=44-56&amp;publication_year=2007&amp;author=Abdo%2CZ&amp;author=Golding%2CGB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR34">Lou, M. &amp; Golding, G. B. Assigning sequences to species in the absence of large interspecific differences. <i>Mol. Phylogenet. Evol.</i> <b>56</b>, 187–194 (2010).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXmvVWgtrw%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1016%2Fj.ympev.2010.01.002" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20067837" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Assigning%20sequences%20to%20species%20in%20the%20absence%20of%20large%20interspecific%20differences&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=56&amp;pages=187-194&amp;publication_year=2010&amp;author=Lou%2CM&amp;author=Golding%2CGB" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR35">Zhang,
 A. B., Sikes, D. S., Muster, C. &amp; Li, S. Q. Inferring Species 
Membership Using DNA Sequences with Back-Propagation Neural Networks. <i>Syst. Biol.</i> <b>57</b>, 202–215 (2008).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD1cXhsVymsr%252FK" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1080%2F10635150802032982" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18398766" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Inferring%20Species%20Membership%20Using%20DNA%20Sequences%20with%20Back-Propagation%20Neural%20Networks&amp;journal=Syst.%20Biol.&amp;volume=57&amp;pages=202-215&amp;publication_year=2008&amp;author=Zhang%2CAB&amp;author=Sikes%2CDS&amp;author=Muster%2CC&amp;author=Li%2CSQ" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR36">Seo, T.-K. Classification of Nucleotide Sequences Using Support Vector Machines. <i>J. Mol. Evol.</i> <b>71</b>, 250–267 (2010).</p><ul class="c-reading-companion__reference-links"><li><a href="http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010JMolE..71..250S" data-track="click" data-track-action="outbound reference" data-track-label="link">ADS</a></li><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BC3cXhtFOhsr7I" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1007%2Fs00239-010-9380-9" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20740280" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Classification%20of%20Nucleotide%20Sequences%20Using%20Support%20Vector%20Machines&amp;journal=J.%20Mol.%20Evol.&amp;volume=71&amp;pages=250-267&amp;publication_year=2010&amp;author=Seo%2CT-K" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR37">Sayers, E. W. <i>Ebot</i>. http://www.ncbi.nlm.nih.gov/Class/PowerTools/eutils/course.html (2018)</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR38">Stajich, J. E. <i>et al</i>. The Bioperl toolkit: Perl modules for the life sciences. <i>Genome Res.</i> <b>12</b>, 1611–1618 (2002).</p><ul class="c-reading-companion__reference-links"><li><a href="https://www.nature.com/articles/cas-redirect/1%3ACAS%3A528%3ADC%252BD38XotVKgtbY%253D" data-track="click" data-track-action="outbound reference" data-track-label="link">CAS</a></li><li><a href="https://doi.org/10.1101%2Fgr.361602" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12368254" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed</a></li><li><a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC187536" data-track="click" data-track-action="outbound reference" data-track-label="link">PubMed Central</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Bioperl%20toolkit%3A%20Perl%20modules%20for%20the%20life%20sciences&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1611-1618&amp;publication_year=2002&amp;author=Stajich%2CJE" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR39">Maddison, W. P. &amp; Maddison, D. R. Mesquite version 3.10. http://mesquiteproject.org (2015).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation" id="rc-ref-CR40">Martin, M. Cutadapt removes adapter sequences from high-throughput sequencing reads. <i>EMBnet J.</i> <b>17</b>, 10–12 (2011).</p><ul class="c-reading-companion__reference-links"><li><a href="https://doi.org/10.14806%2Fej.17.1.200" data-track="click" data-track-action="outbound reference" data-track-label="link">Article</a></li><li><a href="http://scholar.google.com/scholar_lookup?&amp;title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;journal=EMBnet%20J.&amp;volume=17&amp;pages=10-12&amp;publication_year=2011&amp;author=Martin%2CM" data-track="click" data-track-action="outbound reference" data-track-label="link">
                        Google Scholar</a></li></ul></li></ol></div></div>
                    </div>
                </div>
            </div>
        </article>
    </div>
</div>
</div>








<footer>
    

    <div class="container-footer mt40 cleared container-footer-full" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        
          <div class="content pt20 pr30 pl30">
            <div class="grid grid-12 last clear-float cleared mb15 text-gray-light text13 flex-box flex-wrap-reverse">
                <h2 aria-level="2" class="text13 strong emphasis ma0 sans-serif" itemprop="name">Scientific Reports</h2>
                <p class="pt0 pr10 pb0 pl10 ma0"><abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">2045-2322</span> (online)</p>
            </div>
          </div>
        

        <div class="grid grid-12 pl30 pr30 last" id="footer">
        <h2 aria-level="2" class="hide">nature.com sitemap</h2>
    </div>

    <div class="pl30 pr30 content" data-track-component="footer">

        <div class="grid grid-12 last clear-float cleared mb5 pt20 text-gray-light flex-wrap">
            <div class="grid grid-3 mq875-grid-12 just-mq640-last just-mq875-last mb20"><img alt="Nature Research" src="natureresearch-logo.svg" loading="lazy"></div>
            <ul class="grid grid-7 mq875-grid-8 mq640-grid-12 just-mq640-last clean-list mb10 mt10 text15 u-hide-print">
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12"><a href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" class="text-gray-light" data-track-label="link">About us</a></li>
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12 just-mq640-last just-mq480-last"><a href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link" class="text-gray-light">Press releases</a></li>
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12"><a href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link" class="text-gray-light">Press office</a></li>
                <li class="grid grid-3 mq875-grid-3 mq640-grid-6 mq480-grid-12 last"><a href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" class="text-gray-light" data-track-label="link">Contact us</a></li>
            </ul>
            <ul class="grid grid-2 mq875-grid-4 mq640-grid-12 last clean-list mb10 mt4 u-hide-print">
                <li class="grid grid-4 mq875-grid-3"><a href="https://www.facebook.com/nature/" data-track="click" data-track-action="facebook" data-track-label="link" class="text-gray-light"><img src="fb.svg" class="cleared text-footer-img" alt="Facebook"></a></li>
                <li class="grid grid-4 mq875-grid-3"><a href="https://twitter.com/nresearchnews?lang=en" data-track="click" data-track-action="twitter" data-track-label="link" class="text-gray-light"><img src="twitter.svg" class="cleared text-footer-img" alt="Twitter"></a></li>
                <li class="grid grid-4 mq875-grid-3 last"><a href="https://www.youtube.com/channel/UCvCLdSgYdSTpWcOgEJgi-ng" data-track="click" data-track-action="youtube" data-track-label="link" class="text-gray-light"><img src="youtube.svg" class="cleared text-footer-img" alt="Youtube"></a></li>
            </ul>
        </div>

        <div class="cleared z-index-1 pt30 pb20 border-top-1 border-gray text-footer-mobile-bottom hide-print u-hide-print">
            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
                <h3 class="text-footer-heading">Discover content</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.nature.com/siteindex/" data-track="click" data-track-action="journals a-z" data-track-label="link" class="text-gray-light">Journals A-Z</a></li>
                    <li class="pb4"><a href="https://www.nature.com/subjects/" data-track="click" data-track-action="article by subject" data-track-label="link" class="text-gray-light">Articles by subject</a></li>
                    <li class="pb4"><a href="https://nano.nature.com/" data-track="click" data-track-action="nano" data-track-label="link" class="text-gray-light">Nano</a></li>
                    <li class="pb4"><a href="https://www.nature.com/protocolexchange/" data-track="click" data-track-action="protocol exchange" data-track-label="link" class="text-gray-light">Protocol Exchange</a></li>
                    <li class="pb4"><a href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link" class="text-gray-light">Nature Index</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq640-last">
                <h3 class="text-footer-heading">Publish with us</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.nature.com/authors/author_resources/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link" class="text-gray-light">Guide to Authors</a></li>
                    <li class="pb4"><a href="https://www.nature.com/authors/peer_review/" data-track="click" data-track-action="guide to referees" data-track-label="link" class="text-gray-light">Guide to Referees</a></li>
                    <li class="pb4"><a href="https://www.nature.com/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link" class="text-gray-light">Editorial policies</a></li>
                    <li class="pb4"><a href="http://www.nature.com/openresearch/publishing-with-npg/" data-track="click" data-track-action="open access" data-track-label="link" class="text-gray-light">Open access</a></li>
                    <li class="pb4"><a href="https://www.nature.com/reprints/" data-track="click" data-track-action="reprints and permissions" data-track-label="link" class="text-gray-light">Reprints &amp; permissions</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq875-last">
                <h3 class="text-footer-heading">Researcher services</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link" class="text-gray-light">Research data</a></li>
                    <li class="pb4"><a href="https://authorservices.springernature.com/go/nr" data-track="click" data-track-action="language editing" data-track-label="link" class="text-gray-light">Language editing</a></li>
                    <li class="pb4"><a href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link" class="text-gray-light">Scientific editing</a></li>
                    <li class="pb4"><a href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link" class="text-gray-light">Nature Masterclasses</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/product/nature-research-academies/" data-track="click" data-track-action="nature research academies" data-track-label="link" class="text-gray-light">Nature Research Academies</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq875-clear just-mq640-last full-size-last just-mq1200-last just-mq875-last just-mq480-last">
                <h3 class="text-footer-heading">Libraries &amp; institutions</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link" class="text-gray-light">Librarian service &amp; tools</a></li>
                    <li class="pb4"><a href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link" class="text-gray-light">Librarian portal</a></li>
                    <li class="pb4"><a href="http://www.nature.com/openresearch/about-open-access/information-for-institutions/" data-track="click" data-track-action="open research" data-track-label="link" class="text-gray-light">Open research</a></li>
                </ul>
            </div>
        </div>

        <div class="cleared z-index-1 pt30 pb20 border-top-1 border-gray text-footer-mobile-top hide-print u-hide-print">
            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
                <h3 class="text-footer-heading">Advertising &amp; partnerships</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://partnerships.nature.com/" data-track="click" data-track-action="advertising" data-track-label="link" class="text-gray-light">Advertising</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/home/services/" data-track="click" data-track-action="partnerships and services" data-track-label="link" class="text-gray-light">Partnerships &amp; Services</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/home/resources/" data-track="click" data-track-action="media kits" data-track-label="link" class="text-gray-light">Media kits</a></li>
                    <li class="pb4"><a href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link" class="text-gray-light">Branded content</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12 just-mq640-last just-mq875-last">
                <h3 class="text-footer-heading">Career development</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="https://www.nature.com/naturecareers" data-track="click" data-track-action="nature careers" data-track-label="link" class="text-gray-light">Nature Careers</a></li>
                    <li class="pb4"><a href="https://www.nature.com/natureconferences/" data-track="click" data-track-action="nature conferences" data-track-label="link" class="text-gray-light">Nature<span class="visually-hidden"> </span> Conferences</a></li>
                    <li class="pb4"><a href="https://www.nature.com/natureevents/" data-track="click" data-track-action="nature events" data-track-label="link" class="text-gray-light">Nature<span class="visually-hidden"> </span> events</a></li>
                </ul>
            </div>

            <div class="grid grid-3 mq875-grid-4 mq640-grid-6 mq480-grid-12">
                <h3 class="text-footer-heading">Regional websites</h3>
                <ul class="clean-list ma0 mb6 text15">
                    <li class="pb4"><a href="http://www.naturechina.com/" data-track="click" data-track-action="nature china" data-track-label="link" class="text-gray-light">Nature China</a></li>
                    <li class="pb4"><a href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link" class="text-gray-light">Nature India</a></li>
                    <li class="pb4"><a href="https://www.natureasia.com/ja-jp/" data-track="click" data-track-action="nature japan" data-track-label="link" class="text-gray-light">Nature Japan</a></li>
                    <li class="pb4"><a href="https://www.natureasia.com/ko-kr/" data-track="click" data-track-action="nature korea" data-track-label="link" class="text-gray-light">Nature Korea</a></li>
                    <li class="pb4"><a href="https://www.nature.com/nmiddleeast/" data-track="click" data-track-action="nature middle east" data-track-label="link" class="text-gray-light">Nature Middle East</a></li>
                </ul>
            </div>
        </div>
    </div>

    <div class="container-footer-box cleared text14 border-gray-medium border-top-1">
        <div class="content">
            <img src="sn-logo.svg" alt="Springer Nature" loading="lazy" class="cleared">
            <p class="mb10 text-gray-light">© 2019 Springer Nature Publishing AG</p>
            <ul class="ma0 clean-list grid grid-12 hide-print u-hide-print">
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq480-last"><a href="https://www.nature.com/info/privacy.html" data-track="click" data-track-action="privacy policy" data-track-label="link" class="text-gray-light">Privacy Policy</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq640-last just-mq480-last"><a href="https://www.nature.com/info/cookies.html" data-track="click" data-track-action="use of cookies" data-track-label="link" class="text-gray-light">Use of cookies</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq875-last just-mq480-last"><a href="javascript:;" class="optanon-toggle-display text-gray-light" data-track="click" data-track-action="manage cookies" data-track-label="link">Manage cookies</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq640-last just-mq480-last"><a href="https://www.nature.com/info/legal_notice.html" data-track="click" data-track-action="legal notice" data-track-label="link" class="text-gray-light">Legal notice</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 just-mq480-last"><a href="https://www.nature.com/info/accessibility_statement.html" data-track="click" data-track-action="accessibility statement" data-track-label="link" class="text-gray-light">Accessibility statement</a></li>
                <li class="grid grid-2 mq875-grid-4 mq640-grid-6 mq480-grid-12 mb10 last"><a href="https://www.nature.com/info/tandc.html" data-track="click" data-track-action="terms and conditions" data-track-label="link" class="text-gray-light">Terms &amp; Conditions</a></li>
            </ul>
        </div>
    </div>


    </div>
</footer>




    
        
    






    
         
    



<script src="raven.js" crossorigin="anonymous"></script>
<script>
    if (Raven) {
        Raven.config('https://46e8b5cbb6a34c8ab1888c41c7f86c61@sentry.dc.springernature.pe/388').install();
    }
</script>


<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" border="0" width="0" height="0" style="display: none">
</noscript>



<script src="ping" async=""></script>
<img src="s41598-018-22505-4" alt="" class="visually-hidden" width="1" height="1">


<script type="text/javascript" id="">Bootstrapper=window.Bootstrapper||{};Bootstrapper.npg=Bootstrapper.npg||{};Bootstrapper.npg.utils=Bootstrapper.npg.utils||{};
Bootstrapper.npg.utils.Request=function(h,e){var f=function(){for(var a,b={},c=/([^&=]+)=?([^&]*)/g,d=h.location.search.substring(1);a=c.exec(d);)b[decodeURIComponent(a[1].replace(/\+/g," "))]=decodeURIComponent(a[2].replace(/\+/g," "));return b}(),g=function(){for(var a=document.cookie.split(/;\s*/),b,c=a.length,d={};c--;)b=a[c].split("\x3d"),d[b[0]]=b.slice(1).join("\x3d");return d}();return{get:function(a){return 1===arguments.length?f[a]||e:f},cookie:function(a){return 0===arguments.length?g:
1===arguments.length?g[a]||e:this.setCookie.apply(this,[].slice.call(arguments,0))},deleteCookies:function(a,b){for(var c=0;c<a.length;++c)this.deleteCookie(a[c],b)},deleteCookie:function(a,b){b=b||{};b.days=-1;this.setCookie(a,"",b)},setCookie:function(a,b,c){a=[a+"\x3d"+b];c=c||{};c.days&&(b=new Date,b.setTime(+b+864E5*c.days),a.push("expires\x3d"+b.toGMTString()));c.path&&a.push("path\x3d"+c.path);c.domain&&a.push("domain\x3d"+c.domain);document.cookie=a.join(";");return!0},deleteCookiesByPattern:function(a,
b){for(var c=new RegExp("(?:^|;|\\s)("+a+"[^\x3d]*)","g"),d=[],e;null!==(e=c.exec(document.cookie));)d.push(e[1]);this.deleteCookies(d,b)}}}(window);</script><script type="text/javascript" id="">Bootstrapper=window.Bootstrapper||{};Bootstrapper.npg=Bootstrapper.npg||{};Bootstrapper.npg.utils=Bootstrapper.npg.utils||{};Bootstrapper.npg.utils.Consent=function(){var d=unescape(Bootstrapper.npg.utils.Request.cookie("OptanonConsent")||""),b=(/groups=([^&]+)/.exec(d)||["",""])[1].split(",");return{STRICTLY_NECESSARY:1,PERFORMANCE:2,FUNCTIONAL:3,TARGETING:4,allow:function(c){if(c===this.STRICTLY_NECESSARY)return!0;for(var a=0;b[a];++a)if(b[a]===c+":0")return!1;return!0}}}();</script><script type="text/javascript" id="">(function(b,c){b.location.pathname.match(/^\/(nature_education|scitable|principles)(\/|$)/)||b.location.hostname.match("natureindex.com")||c.addEventListener("accessdetailsloaded",function(a){if(a=a.detail)b._pageMetaData=b._pageMetaData||{},b._pageMetaData.site_id=a.site_id&&a.site_id.join?a.site_id.join(";"):"",b._pageMetaData.user_id=a.registrant_id||"",b._pageMetaData.business_partner_id=a.business_partner_id&&a.business_partner_id.join?a.business_partner_id.join(";"):"";a=c.createElement("script");
a.className="kxct";a.setAttribute("data-id","Jn7TaPwq");a.setAttribute("data-timing","async");a.setAttribute("data-version","3.0");a.appendChild(c.createTextNode('window.Krux||((Krux\x3dfunction(){Krux.q.push(arguments)}).q\x3d[]);(function(){var k\x3ddocument.createElement("script");k.type\x3d"text/javascript";k.async\x3dtrue;var m,src\x3d(m\x3dlocation.href.match(/\\bkxsrc\x3d([^\x26]+)/))\x26\x26decodeURIComponent(m[1]);k.src \x3d /^https?:\\/\\/([a-z0-9_\\-\\.]+\\.)?krxd\\.net(:\\d{1,5})?\\//i.test(src) ? src : src \x3d\x3d\x3d "disable" ? "" :(location.protocol\x3d\x3d\x3d"https:"?"https:":"http:")+"//cdn.krxd.net/controltag?confid\x3dJn7TaPwq";var s\x3ddocument.getElementsByTagName("script")[0];s.parentNode.insertBefore(k,s);}());'));
c.head.appendChild(a);a=Bootstrapper.npg.utils.Consent;a=a.allow(a.TARGETING);Krux("consent:set",{dc:a,al:a,tg:a,cd:!1,sh:!1,re:!1},function(a,b){a&&console.error(a)})},!1)})(window,document);</script><script type="text/javascript" id="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail;var b={event:"update-access-details"};if(a){var c=a.registrant_id||"",d=a.site_id&&a.site_id.join?a.site_id.join(";"):"",e=a.business_partner_id&&a.business_partner_id.join?a.business_partner_id.join(";"):"";b.user={};b.user.profile={};b.user.profile.profileInfo={profileID:c||null,bpid:e||null};b.user.segment={registeredUser:!!c};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy=
{};b.session.authentication.legacy.webtrendsSiteID=d||null}window.dataLayer.push(b)},!1);</script><script type="text/javascript" id="">(function(a){if("function"===typeof window.CustomEvent)return!1;var b=function(b,c){c=c||{};var d=document.createEvent("CustomEvent");d.initCustomEvent(b,c.bubbles||!1,c.cancelable||!1,c.detail||a);return d};b.prototype=window.Event.prototype;window.CustomEvent=b})();
var parse=function(a,b){try{return 200===a?JSON.parse(b):null}catch(e){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},base=function(a){return-1===a.indexOf("www")?"https://www.nature.com":""};
if(-1!==window.location.hostname.indexOf(".nature.com")){var transport=new XMLHttpRequest;transport.open("GET",base(window.location.hostname)+"/platform/login/details",!0);transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script type="text/javascript" id="">function splitKeys(b){var f=[],e="",h=[],k=b.split(";");for(b=0;b<k.length;++b){void 0!==l&&(e=l);var c=k[b].split("\x3d");var l=c[0];l!==e&&(0<e.length&&0<f.length&&h.push([e,f]),f=[]);if(2===c.length&&""!==c[0]&&""!==c[1]){var n=c[1].split(",");for(c=0;c<n.length;++c)f.push(n[c])}}0<l.length&&0<f.length&&h.push([l,f]);return h}
function getScript(b,f){var e=document.createElement("script"),h=document.getElementsByTagName("script")[0];e.async=1;e.onload=e.onreadystatechange=function(b,c){if(c||!e.readyState||/loaded|complete/.test(e.readyState))e.onload=e.onreadystatechange=null,e=void 0,c||f&&f()};e.src=b;h.parentNode.insertBefore(e,h)}
function splitSizes(b){var f=[];if(null!==b){var e=-1!==b.indexOf("|")?b.split("|"):b.split(",");for(b=0;b<e.length;++b){var h=e[b].split("x");var k=parseInt(h[0],10);var c=parseInt(h[1],10);2===h.length&&!isNaN(k)&&!isNaN(c)&&0<=k&&0<=c&&f.push([k,c])}}return f}function hasClass(b,f){var e,h=b.className.split(/\s+/);var k=0;for(e=h.length;k<e;++k)if(h[k]===f)return!0;return!1}
function getAdContainers(){if("function"===typeof document.getElementsByClassName)return document.getElementsByClassName("div-gpt-ad");var b,f=[],e=document.getElementsByTagName("div");for(b=0;e[b];++b)hasClass(e[b],"div-gpt-ad")&&f.push(e[b]);return f}function debounce(b,f){var e=null,h=null;return function(){var k=this,c=+new Date,l=arguments;e&&c<e+f?(clearTimeout(h),h=setTimeout(function(){e=c;b.apply(k,l)},f)):(e=c,b.apply(k,l))}}
function addResizeEvent(b){window.addEventListener?window.addEventListener("resize",b,!1):window.attachEvent("resize",b)}function addScrollEvent(b){window.addEventListener?window.addEventListener("scroll",b,!1):window.attachEvent("onscroll",b)}function removeScrollEvent(b){window.removeEventListener?window.removeEventListener("scroll",b,!1):window.detachEvent("scroll",b)}function serveAdsFor(b){return-1!==b.indexOf("/naturejobs")?!1:!0}
function initAds(b,f,e,h){var k=Bootstrapper.npg.utils,c=[],l,n=null;if("true"!==k.Request.get("hide_ads")&&serveAdsFor(b.location.pathname)){var r=function(a){var c=Math.max(f.clientHeight,b.innerHeight||0);return x(a,function(a){if(a.isOutOfPage||a.forceLoadOnInit)return!0;var d=document.getElementById(a.divId),m=d.getBoundingClientRect();m=m.top-300;m=c>m;var e=null!==d.offsetParent;a=a.sizeArray&&a.sizeArray.length&&a.sizeArray[0].length&&2===a.sizeArray[0][0];d=d.getAttribute("data-ad-type")?
-1===d.getAttribute("data-ad-type").indexOf("top"):!1;var f=770>b.innerWidth;return m&&e&&!1===f||a||m&&e&&f&&d})},x=function(a,b){for(var d=a.length,c=[];d--;)b(a[d],d)&&(c.push(a[d].slot),a.splice(d,1));c.length&&googletag.pubads().refresh(c);return a},q={"career feature":!0,"career news":!0,"career q\x26a":!0,"career brief":!0,"career column":!0,spotlight:!0,"career guide":!0,"technology feature":!0,"nature careers podcast":!0},y=function(){var a=b.dataLayer?google_tag_manager["GTM-NWDMT9Q"].macro(1):null;if(a)return a;
if(-1!==b.location.hostname.indexOf("guide.labanimal"))return a=b.location.pathname,a=-1!==a.indexOf("categ")?"products":-1!==a.indexOf("supplier")?"suppliers":"homepage","laban/guide.labanimal/"+a;a=b.location.pathname.replace(/^\/+/,"").split("/");var c=b.location.hostname.split(".").slice(1).join(".");return a.length&&""!==a[0]?a[0]:c},z=function(a){var c="/270604982";0!==a.indexOf("/")&&(a="/"+a);0===a.indexOf("/285/")&&(a=a.replace(/^\/285\/[^\/]+/,c+"/nature/"+y()));a=a.replace("/nature/laban",
"/laban");a=a.replace("/nature/nature.com/index","/nature/nature/homepage");a=a.replace("/collections/collections","/collections");a=a.replace("/search/search_results","/nature/search");a=a.replace(/\/article$/,"/articles");a=a.replace(/\/nature\/authors\/.*/,"/nature/nature/authors");-1!==b.location.hostname.indexOf("blogs")&&(a=a.replace(/\/nature\/.*/,"/nature/nature/blogs"));-1!==b.location.hostname.indexOf("natureindex")&&(a=a.replace(/\/nature\/.*/,"/nature/nature_index"),"/"===b.location.pathname&&
(a+="/homepage"));window.dataLayer&&q[google_tag_manager["GTM-NWDMT9Q"].macro(2)]&&(a=a.replace(/\/articles$/,"/naturecareers"));a:{if(/^\/nature\/articles\/?$/.test(window.location.pathname)){var d=(/^.*?(?:\?|&)type=([^&]+)/.exec(window.location.search)||["",""])[1];if(q[d.replace(/-/g," ").replace(/ and /,"\x26")]){d=!0;break a}}d=!1}d&&(a=a.replace(/\/article-list$/,"/naturecareers"));(d=document.querySelector('meta[name\x3d"brandedcontent"]'))&&"true"===d.getAttribute("content")&&(a=c+"/nature/brandedcontent");return a},
A=function(a){for(var c={},d=0;a[d];++d){var g=a[d].size;var e="2x2"===a[d].size?window.dataLayer&&"core media"===google_tag_manager["GTM-NWDMT9Q"].macro(3)&&-1!==b.location.pathname.indexOf("/articles/")?3:0:0;c[g]={count:e,name:a[d].name}}return c}([{size:"728x90",name:"LB"},{size:"300x250",name:"MPU"},{size:"160x600",name:"SKY"},{size:"970x250",name:"BB"},{size:"2x2",name:"NATIVE"},{size:"300x100",name:"REC"},{size:"180x150",name:"EVENTS"},{size:"160x60",name:"TILE"}]),p=function(a,b,d,c){for(var g=!1,e=0;a[e];++e)a[e][0]===
b&&null!==d&&(a[e][1]=c?a[e][1].concat(d):[d],g=!0);g||null===d||a.push([b,[d]]);return a},t=function(a,b,d){for(var c=0;a[c];++c)a[c][0]===b&&(a[c][0]=d);return a},B=function(a,c){var d=a;var g=e.dfppKeyValues&&k.Consent.allow(k.Consent.TARGETING)?";"+e.dfppKeyValues:"";var m=k.Request.get("test");m=m&&"ads"===m?";adtype\x3dtest":"";d=d.getAttribute("data-gpt-targeting");m&&-1===d.indexOf(m)&&(d+=m);d+=h.registrant_id?";logged\x3dy":";logged\x3dn";h.business_partner_id&&(d+=";bpid\x3d"+h.business_partner_id.join(","));
0===b.location.pathname.indexOf("/collections/")&&(m=document.querySelector("span.hero-title-inner"))&&(d+=";sponsored\x3d"+m.innerText.replace(/^\s+/,"").replace(/\s+$/,"").replace(/\s+/g,"_").replace(/\W+/g,""));g=d+g;g=t(splitKeys(g),"artid","articleid");g=t(g,"kw","search");d=a.getAttribute("data-gpt-sizes");d=(d=A[d])?d.name+ ++d.count:null;g=p(g,"pos",d);g=p(g,"tile",c+1);d=b.location.pathname.split("/");d=3===d.length&&"subjects"===d[1]?d[2]:null;g=p(g,"subject",d);g=p(g,"article",window.dataLayer&&
q[google_tag_manager["GTM-NWDMT9Q"].macro(4)]?"naturecareers":null);d=window.dataLayer?google_tag_manager["GTM-NWDMT9Q"].macro(5):null;g=p(g,"collectionID",d);return g=p(g,"type",window.dataLayer&&"core media"===google_tag_manager["GTM-NWDMT9Q"].macro(6)?"fronthalf":null,!0)},u=function(a){for(var c=[],d=0;a[d];++d){var b=a[d];c.push({divId:b.getAttribute("id"),adUnitPath:z(b.getAttribute("data-gpt-unitpath")),sizeArray:splitSizes(b.getAttribute("data-gpt-sizes")),targeting:B(b,d),isOutOfPage:hasClass(b,"out-of-page"),forceLoadOnInit:!1,refreshed:!1})}return c},
v=function(){googletag.cmd.push(function(){googletag.pubads().setRequestNonPersonalizedAds(k.Consent.allow(k.Consent.TARGETING)?0:1);googletag.pubads().disableInitialLoad();googletag.enableServices();var a={};googletag.pubads().addEventListener("slotRenderEnded",function(b){var c=b.slot&&b.slot.getSlotElementId?b.slot.getSlotElementId():null,e;a[c]=!b.isEmpty;(a["div-gpt-ad-native-2"]||a["div-gpt-ad-native-1"])&&(e=document.querySelector(".c-paid-content"))&&e.classList.remove("hide");"div-gpt-ad-billboard-2"===
c&&!1===b.isEmpty&&(e=document.getElementById(c),e.parentNode.parentNode.classList.add("pb40"),e.parentNode.parentNode.classList.remove("pb20"),e.parentNode.parentNode.classList.remove("hide"));c&&b.isEmpty&&(e=document.getElementById(c),e.parentNode.parentNode.classList.remove("pb20"),e.parentNode.classList.remove("ad-with-label"))})});googletag.cmd.push(function(){for(var a=0;c[a];++a)try{c[a].slot=c[a].isOutOfPage?googletag.defineOutOfPageSlot(c[a].adUnitPath,c[a].divId).addService(googletag.pubads()):
googletag.defineSlot(c[a].adUnitPath,c[a].sizeArray,c[a].divId).addService(googletag.pubads());for(var b=0,d=c[a].targeting.length;b<d;++b)if(2===c[a].targeting[b].length&&""!==c[a].targeting[b][0]&&""!==c[a].targeting[b][1]){if("pos"===c[a].targeting[b][0]&&0===c[a].targeting[b][1][0].indexOf("BB")){l=c[a].slot;var e=googletag.sizeMapping().addSize([970,250],[3,3]).addSize([770,100],[4,4]).addSize([0,0],[5,5]).build();l.defineSizeMapping(e);l.setCollapseEmptyDiv(!0,!1)}c[a].slot.setTargeting(c[a].targeting[b][0],
c[a].targeting[b][1])}}catch(C){console.log("failed to create slot for",c[a])}});googletag.cmd.push(function(){for(var a=0;c[a];++a)googletag.display(c[a].divId)})},w=function(){n&&removeScrollEvent(n);googletag.cmd.push(function(){c=r(c)});n=debounce(function(){googletag.cmd.push(function(){c=r(c);c.length||(removeScrollEvent(n),n=null)})},250);addScrollEvent(n)};getScript("//www.googletagservices.com/tag/js/gpt.js",function(){b.googletag=b.googletag||{};b.googletag.cmd=b.googletag.cmd||[];c=u(getAdContainers());
v();w();var a=969<b.innerWidth,e=769<b.innerWidth&&!a,d=770>b.innerWidth,f=debounce(function(){var c=b.innerWidth;970<=c&&!a?(a=!0,d=e=!1,googletag.cmd.push(function(){googletag.pubads().refresh([l])})):770>c&&!d?(e=a=!1,d=!0,googletag.cmd.push(function(){googletag.pubads().refresh([l])})):970>c&&769<c&&!e&&(a=!1,e=!0,d=!1,googletag.cmd.push(function(){googletag.pubads().refresh([l])}))},250);addResizeEvent(f);document.addEventListener("refreshads",function(){googletag.destroySlots();c=u(getAdContainers());
v();w()},!1)})}}-1===window.location.hostname.indexOf("nature.com")&&("complete"===document.readyState||"loaded"===document.readyState||"interactive"===document.readyState?initAds(window,document.documentElement,window.Krux||{},{}):document.addEventListener("load",function(){initAds(window,document.documentElement,window.Krux||{},{})}));document.addEventListener("accessdetailsloaded",function(b){initAds(window,document.documentElement,window.Krux||{},b.detail||{})},!1);</script>
<script type="text/javascript" id="">window.location.pathname.match(/^\/(nature_education|scitable|principles)(\/|$)/)||(window.Krux||((Krux=function(){Krux.q.push(arguments)}).q=[]),function(){function c(a){a="kxmacmillan_"+a;try{var b=window.localStorage}catch(e){b=null}return b?b[a]||"":navigator.cookieEnabled?(b=document.cookie.match(a+"\x3d([^;]*)"))&&unescape(b[1])||"":""}Krux.user=c("user");Krux.segments=c("segs")?c("segs").split(","):[];var a=[];Krux.user&&a.push("kuid\x3d"+Krux.user);for(var d=0;d<Krux.segments.length;d++)a.push("ksg\x3d"+
Krux.segments[d]);Krux.dfppKeyValues=a.length?a.join(";")+";":""}());</script><script type="text/javascript" id="">var filterMeta=function(b){for(var c=[],a=0;b[a];++a)b[a].name&&c.push(b[a]);return c},translateMeta=function(b){var c={"WT.cg_n":"product_name","WT.site_id":"site_id","WT.site_id_name":"site_id_name","WT.registrant_id":"user_id"};return c[b]||b},getMeta=function(b){var c=filterMeta(document.getElementsByTagName("meta")||[]),a=document.getElementsByTagName("body")||[],f=function(){for(var a=c.length,d={},e;a--;)if(e=translateMeta(c[a].name))d[e]?("string"===typeof d[e]&&(d[e]=[d[e]]),d[e].push(c[a].content)):
d[e]=c[a].content||"";d.keywords&&(d.keywords=d.keywords.replace(/,/g,";"));b(d)};a.length?f():c.length&&f()};getMeta(function(b){window._pageMetaData=b});</script><script type="text/javascript" id="">function getRandomInt(a,b){return Math.floor(Math.random()*(b-a+1))+a}var oneOutOf=10;if(0===getRandomInt(0,oneOutOf)&&"www.nature.com"===window.location.hostname&&"/"===window.location.pathname){_prum=[["id","58bd3858754054efc9152d3f"],["mark","firstbyte",(new Date).getTime()]];var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//rum-static.pingdom.net/prum.min.js";s.parentNode.insertBefore(p,s)};</script><script type="text/javascript" id="">function getRandomInt(a,b){return Math.floor(Math.random()*(b-a+1))+a}var oneOutOf=10;if(0===getRandomInt(0,oneOutOf)&&"www.nature.com"===window.location.hostname&&"/"===window.location.pathname){_prum=[["id","58bd3858754054efc9152d3f"],["mark","firstbyte",(new Date).getTime()]];var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//rum-static.pingdom.net/prum.min.js";s.parentNode.insertBefore(p,s)};</script><script type="text/javascript" id="">function getRandomInt(a,b){return Math.floor(Math.random()*(b-a+1))+a}var oneOutOf=10;if(0===getRandomInt(0,oneOutOf)){var path=/^\/subjects/;if("www.nature.com"===window.location.hostname&&window.location.pathname.match(path)){_prum=[["id","58bd36ee7540540ec8152d3f"],["mark","firstbyte",(new Date).getTime()]];var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//rum-static.pingdom.net/prum.min.js";s.parentNode.insertBefore(p,s)}};</script><script type="text/javascript" id="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="//scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script type="text/javascript" id="">var images=document.images,sendNielson=!0;if(-1<images.length){var i=0;for(i;i<images.length;i++)images[i].src&&-1<images[i].src.indexOf("secure-us.imrworldwide.com")&&(sendNielson=!1)}sendNielson&&function(){var a=new Image(1,1);a.onerror=a.onload=function(){a.onerror=a.onload=null};a.src=["//secure-us.imrworldwide.com/cgi-bin/m?ci\x3dus-504674h\x26cg\x3d0\x26cc\x3d1\x26si\x3d",escape(window.location.href),"\x26rp\x3d",escape(document.referrer),"\x26ts\x3dcompact\x26rnd\x3d",(new Date).getTime()].join("")}();</script><script type="text/javascript" id="">(function(){var a=document,b=a.getElementsByTagName("script")[0];s=a.createElement("script");s.type="text/javascript";s.async=!0;s.src="//cdn.cookielaw.org/consent/792a5a21-3d99-4432-8f70-957d389bf2b9.js";b.parentNode.insertBefore(s,b)})();function OptanonWrapper(){};</script><script type="text/javascript" id="">var c_id="14617931",_comscore=window._comscore=_comscore||[];_comscore.push({c1:"2",c2:c_id});(function(){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;var c="https:"==document.location.protocol?"https://sb":"http://b";a.src=c+".scorecardresearch.com/beacon.js";b.parentNode.insertBefore(a,b)})();</script><script type="text/javascript" id="">(function(){if(window.location.hostname.match(/^(.*-?www.)?nature.com/)){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;a.defer=!0;a.src="//get.s-onetag.com/c692e01a-b65d-4aa9-aff9-bee34f082a39/tag.min.js";b.parentNode.insertBefore(a,b)}})();</script><script type="text/javascript" id="">(function(){var a=document,b=a.getElementsByTagName("script")[0];a=a.createElement("script");a.type="text/javascript";a.async=!0;a.src="https://recommended.springernature.com/latest/entry-point.js";b.parentNode.insertBefore(a,b)})();</script><iframe src="https://event-tracker.springernature.com" title="EventTracker" id="event-tracker" style="display:none; visibility:hidden" aria-hidden="true" tabindex="-1" width="0" height="0"></iframe><iframe style="width: 0px; height: 0px; bottom: 0px; right: 0px; z-index: 100000; position: fixed;" scrolling="no" src="https://recommended.springernature.com/v2344/generated/client.html?source=https://www.nature.com" name="uptodate-client" id="uptodate-client" title="Personalised recommendations" allowtransparency="true" frameborder="0"></iframe><div class="kxhead" data-id="Jn7TaPwq" style="display:none !important;"><span class="kxtag kxinvisible" data-id="20636"><script>
// this tag is intentionally blank
</script></span><span class="kxtag kxinvisible" data-id="20637"></span><span class="kxtag kxinvisible" data-id="20638"></span><span class="kxtag kxinvisible" data-id="20641"><script>
(function() {

    var pageMetaData = window._pageMetaData;
    Krux('scrape', {
        'page_attr_nature_url_path_1': {
            'url_path': '1'
        },
        'page_attr_nature_url_path_2': {
            'url_path': '2'
        },
        'page_attr_nature_url_path_3': {
            'url_path': '3'
        },
        'page_attr_nature_product_name': {
            'js_global': '_pageMetaData.product_name'
        },
        'page_attr_nature_site_id': {
            'js_global': '_pageMetaData.site_id'
        },
        'page_attr_nature_site_id_name': {
            'js_global': '_pageMetaData.site_id_name'
        }
    });
    if (Krux('get', 'page_attr_nature_site_id') && Krux('get', 'page_attr_nature_site_id').length > 1) {
        Krux('set', 'user_attr_nature_site_account_user', 'true');
    } else {
        Krux('set', 'user_attr_nature_site_account_user', 'false');
    }

})();
</script></span><span class="kxtag kxinvisible" data-id="20647"><script>
(function() {

    Krux('scrape', {
        'page_attr_Citation_Title': {
            'js_global': '_pageMetaData.citation_title'
        },
        'page_attr_Citation_DOI': {
            'js_global': '_pageMetaData.citation_doi'
        },
        'page_attr_Citation_Publication_Date': {
            'js_global': '_pageMetaData.citation_publication_date'
        },
        'page_attr_Citation_Author': {
            'js_global': '_pageMetaData.citation_author'
        }
    });

    if (window._pageMetaData && window._pageMetaData["WT.cg_s"]) {
        Krux('set', 'page_attr_Article_Type', _pageMetaData["WT.cg_s"]);

    } else if (window._pageMetaData && window._pageMetaData["prism.section"]) {
        Krux('set', 'page_attr_Article_Type', _pageMetaData["prism.section"]);
    }

    if (window._pageMetaData && window._pageMetaData.access === 'No') {
        Krux('set', 'user_attr_Access_Denied', _pageMetaData.product_name);
    }

    if (window._pageMetaData && window._pageMetaData["WT.z_subject_term"]) {
        Krux('set', 'page_attr_Article_Keywords', _pageMetaData["WT.z_subject_term"]);
    } else if (window._pageMetaData && window._pageMetaData.keywords) {
        Krux('scrape', {
            'page_attr_Article_Keywords': {
                'js_global': '_pageMetaData.keywords'
            }
        });
    } else {
        Krux('scrape', {
            'page_attr_Article_Keywords': {
                meta_name: 'keywords'
            }
        });
    }

})();
</script></span><span class="kxtag kxinvisible" data-id="20648"><script>
(function() {

    // Using Meta DCSext.job_id to produce page attribute dcsext_job_id
    Krux('scrape', {
        'page_attr_Job_ID': {
            meta_name: 'DCSext.job_id'
        },
        'page_attr_Employer_ID': {
            meta_name: 'DCSext.employer_id'
        },
        'page_attr_Job_Employer': {
            meta_name: 'Job Employer'
        },
        'page_attr_Job_Type': {
            meta_name: 'Job Type'
        }
    });

    var job_location = Krux('scrape', {
        'page_attr_Job_Location': {
            meta_name: 'Job Location'
        }
    });
    // meta attribute Job Location is always in format "city, country"; client only wants to ingest country
    if (job_location && job_location.page_attr_Job_Location && job_location.page_attr_Job_Location.split(',')[1]) Krux('set', 'page_attr_Job_Location', job_location.page_attr_Job_Location.split(',')[1].trim());

})();
</script></span><span class="kxtag kxinvisible" data-id="20651"><script type="text/javascript">Krux('social.init');</script></span><span class="kxtag kxinvisible" data-id="20655"><script>
(function(){
	var domain, path1, path2, key1, key2;
	var href = window.location.href;
	// pull in domain name
	domain = Krux('get', 'domain');
	// pull in path 1 & 2 of domain
	path1 = Krux('scrape.url_path' , '1');
	path2 = Krux('scrape.url_path' , '2');
	// only if domain is 'www.naturejobs.com'
	if (domain.match(/www\.nature\.com/g)){
		// and if both url path 1 is 'naturejobs' and url path 2 is 'science'
		if (path1.match(/naturejobs/g) && path2.match(/science/g)){
			// scrape keywords as naturejobs attributes
			Krux('scrape',{
				'page_attr_naturejobs_what':{'url_param':'q'},
				'page_attr_naturejobs_where':{'url_param':'where'}
			});
		}
	}
})();
</script></span><span class="kxtag kxinvisible" data-id="36836"><script>
(function() {
    /* Selective Attribute DataLayer Library Tag */
    var _, allAttr, allowedList, attr, attributes, dataLayerIngester, dataObj,
        isAllowed, keepCase, libUtil, omitKeys, pageAttr, prefix, toSet, trim,
        userAttr, util, value,
        hasProp = {}.hasOwnProperty;
    _ = Krux('require:underscore');
    util = Krux('require:util');
    libUtil = Krux('require:util.library-tag');
    dataLayerIngester = Krux('require:scrape').ingestDataLayer;

    /* Safe copy of dataLayer object */
    dataObj = Krux('scrape.javascript', 'dataLayer');

    /* String trimming helper function */
    trim = function(attr) {
        return ("" + attr).replace(/^\s+|\s+$/g, '');
    };

    /* Attribute configs */
    pageAttr = _.map('content.authorization.status,content.category.legacy.webtrendsSubjectTerms,content.category.contentType,content.contentInfo.authors,content.article.doi,content.contentInfo.publishedAtString,content.contentInfo.title,content.category.journal.title,page.category.pageType,page.attributes.template,window.location.pathname,window.location.pathname,window.location.pathname,content.jobs.jobTitle,content.jobs.jobLocation,content.jobs.employer,content.category.journal.title,content.journal.title'.split(','), trim);
    userAttr = _.map('user.profile.profileInfo.bpid,segment.registeredUser,gaClientId'.split(','), trim);

    /* Create a array of attributes striping any empty strings */
    allAttr = _.without(pageAttr.concat(userAttr), '');

    /* Configuration settings */
    keepCase = 'false' === 'true';
    omitKeys = 'none'.split(',');

    /* Resolve Prefix */
    prefix = libUtil.resolvePrefix('text', 'new_dl_',
        'undefined');

    /* Function to varify if attribute should be used */
    isAllowed = function(value, whitelist) {
        var i, len, str, x;
        str = "" + value;
        if (!((value != null) && str.length > 0)) {
            return false;
        }
        for (i = 0, len = whitelist.length; i < len; i++) {
            x = whitelist[i];
            if (value.match(x) != null) {
                return true;
            }
        }
        return false;
    };

    /* Get a full list of attributes usting the dataLayer tool */
    attributes = dataLayerIngester(dataObj, {
        omitKeys: libUtil.removeFalsyStrings(omitKeys.concat(libUtil.EXCLUDE_KEYS_CONFIG)),
        omitValues: libUtil.EXCLUDE_VALUES_CONFIG,
        caseSensitive: keepCase,
        useFullPath: 'true' === 'true',
        useLastValue: 'false' === 'true',
        customDelimited: [/./],
        altDelimiter: ',',
        userKeys: _.map(userAttr, function(exp) {
            return new RegExp("(^|\\.)" + exp + "$");
        }),
        optimizeNames: true
    });

    /* Only set Attributes in the allowed list */
    allowedList = _.map(allAttr, function(name) {
        return new RegExp("(_attr_|_attr_" + prefix + "|\\.)" + (keepCase ?
            name : libUtil.normalizeAttrName(name, {
                removeDot: false
            })) + "$");
    });
    toSet = {};
    for (attr in attributes) {
        if (!hasProp.call(attributes, attr)) continue;
        value = attributes[attr];
        if (isAllowed(attr, allowedList)) {
            toSet[attr] = value;
        }
    }
    toSet = Krux('prefix:attr', toSet, prefix);
    Krux('set', toSet);
}).call();
</script></span><span class="kxtag kxinvisible" data-id="38361"><script>

(function() { 

	var pageMetaData = window._pageMetaData; 
	Krux('scrape', { 
		'page_attr_url_path_1': { 
			'url_path': '1' 
		}, 
		'page_attr_url_path_2': { 
			'url_path': '2' 
		}, 
			'page_attr_url_path_3': { 
				'url_path': '3'
		}, 
			'page_attr_product_name': { 
				'js_global': '_pageMetaData.product_name'
		}, 
			'page_attr_site_id': { 
				'js_global': '_pageMetaData.site_id'
		}, 
			'page_attr_site_id_name': { 
				'js_global': '_pageMetaData.site_id_name' 
		}, 
			'user_attr_user_id': { 'js_global': '_pageMetaData.user_id' 
		} 
	}); 
	if (Krux('get', 'page_attr_site_id') && Krux('get', 'page_attr_site_id').length > 1) 	{ 
		Krux('set', 'user_attr_site_account_user', 'true'); 
	} else { 
		Krux('set', 'user_attr_site_account_user', 'false'); 
	} 
})(); 
</script></span></div><div id="criteo-tags-div" style="display: none;"><iframe style="display: none;" src="//dis.us.criteo.com/dis/dis.aspx?p=60881&amp;cb=26275895182&amp;ref=&amp;sc_r=1920x1080&amp;sc_d=24" title="Criteo DIS iframe" width="0" height="0"></iframe></div><script type="text/javascript" id="">!function(d,e,f,a,b,c){d.twq||(a=d.twq=function(){a.exe?a.exe.apply(a,arguments):a.queue.push(arguments)},a.version="1.1",a.queue=[],b=e.createElement(f),b.async=!0,b.src="//static.ads-twitter.com/uwt.js",c=e.getElementsByTagName(f)[0],c.parentNode.insertBefore(b,c))}(window,document,"script");twq("init","o1unr");twq("track","PageView");</script><script type="text/javascript" id="">-1==document.location.href.search("appspot.com")&&-1==document.referrer.search("appspot.com")&&!function(b,e,f,g,a,c,d){b.fbq||(a=b.fbq=function(){a.callMethod?a.callMethod.apply(a,arguments):a.queue.push(arguments)},b._fbq||(b._fbq=a),a.push=a,a.loaded=!0,a.version="2.0",a.queue=[],c=e.createElement(f),c.async=!0,c.src=g,d=e.getElementsByTagName(f)[0],d.parentNode.insertBefore(c,d))}(window,document,"script","https://connect.facebook.net/en_US/fbevents.js");fbq("init","1229240860577415");
fbq("track","PageView");var segs=google_tag_manager["GTM-NWDMT9Q"].macro(172);fbq("trackCustom","SciRep_inMarket",{t4fs93hlz:segs});</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=1229240860577415&amp;ev=PageView&amp;noscript=1"></noscript></body><iframe id="kx-proxy-Jn7TaPwq" src="https://cdn.krxd.net/partnerjs/xdi/proxy.3d2100fd7107262ecb55ce6847f01fa5.html#%21kxcid=Jn7TaPwq&amp;kxt=https%3A%2F%2Fwww.nature.com&amp;kxcl=cdn&amp;kxp=" style="display: none; visibility: hidden; height: 0; width: 0;"></iframe></html>